
<html lang="en"     class="pb-page"  data-request-id="734f8148-d423-473a-8b1d-8780b231fba3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-18;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c00567"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rational Design of Suprastat: A Novel Selective Histone Deacetylase 6 Inhibitor with the Ability to Potentiate Immunotherapy in Melanoma Models" /></meta><meta name="dc.Creator" content="Satish  Noonepalle" /></meta><meta name="dc.Creator" content="Sida  Shen" /></meta><meta name="dc.Creator" content="Jakub  Ptáček" /></meta><meta name="dc.Creator" content="Maurício T.  Tavares" /></meta><meta name="dc.Creator" content="Guiping  Zhang" /></meta><meta name="dc.Creator" content="Jan  Stránský" /></meta><meta name="dc.Creator" content="Jiří  Pavlíček" /></meta><meta name="dc.Creator" content="Glaucio M.  Ferreira" /></meta><meta name="dc.Creator" content="Melissa  Hadley" /></meta><meta name="dc.Creator" content="Guido  Pelaez" /></meta><meta name="dc.Creator" content="Cyril  Bařinka" /></meta><meta name="dc.Creator" content="Alan P.  Kozikowski" /></meta><meta name="dc.Creator" content="Alejandro  Villagra" /></meta><meta name="dc.Description" content="Selective inhibition of histone deacetylase 6 (HDAC6) is being recognized as a therapeutic approach for cancers. In this study, we designed a new HDAC6 inhibitor, named Suprastat, using in silico s..." /></meta><meta name="Description" content="Selective inhibition of histone deacetylase 6 (HDAC6) is being recognized as a therapeutic approach for cancers. In this study, we designed a new HDAC6 inhibitor, named Suprastat, using in silico s..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 20, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00567" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00567" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00567" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00567" /></link>
        
    
    

<title>Rational Design of Suprastat: A Novel Selective Histone Deacetylase 6 Inhibitor with the Ability to Potentiate Immunotherapy in Melanoma Models | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00567" /></meta><meta property="og:title" content="Rational Design of Suprastat: A Novel Selective Histone Deacetylase 6 Inhibitor with the Ability to Potentiate Immunotherapy in Melanoma Models" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0012.jpeg" /></meta><meta property="og:description" content="Selective inhibition of histone deacetylase 6 (HDAC6) is being recognized as a therapeutic approach for cancers. In this study, we designed a new HDAC6 inhibitor, named Suprastat, using in silico simulations. X-ray crystallography and molecular dynamics simulations provide strong evidence to support the notion that the aminomethyl and hydroxyl groups in the capping group of Suprastat establish significant hydrogen bond interactions, either direct or water-mediated, with residues D460, N530, and S531, which play a vital role in regulating the deacetylase function of the enzyme and which are absent in other isoforms. In vitro characterization of Suprastat demonstrates subnanomolar HDAC6 inhibitory potency and a hundred- to a thousand-fold HDAC6 selectivity over the other HDAC isoforms. In vivo studies reveal that a combination of Suprastat and anti-PD1 immunotherapy enhances antitumor immune response, mediated by a decrease of protumoral M2 macrophages and increased infiltration of antitumor CD8+ effector and memory T-cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00567"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00567">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00567&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00567&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00567&amp;href=/doi/10.1021/acs.jmedchem.0c00567" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 10246-10262</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00489" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00728" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rational Design of Suprastat: A Novel Selective Histone Deacetylase 6 Inhibitor with the Ability to Potentiate Immunotherapy in Melanoma Models</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Satish Noonepalle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Satish Noonepalle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Satish++Noonepalle">Satish Noonepalle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sida Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sida Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, University of Illinois at Chicago, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sida++Shen">Sida Shen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0295-2545" title="Orcid link">http://orcid.org/0000-0002-0295-2545</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jakub Ptáček</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jakub Ptáček</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Structural Biology, Institute of Biotechnology of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jakub++Pt%C3%A1%C4%8Dek">Jakub Ptáček</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maurício T. Tavares</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maurício T. Tavares</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, University of Illinois at Chicago, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maur%C3%ADcio+T.++Tavares">Maurício T. Tavares</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guiping Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guiping Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, University of Illinois at Chicago, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guiping++Zhang">Guiping Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9818-4773" title="Orcid link">http://orcid.org/0000-0001-9818-4773</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jan Stránský</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Stránský</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre of Molecular Structure, Institute of Biotechnology of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Str%C3%A1nsk%C3%BD">Jan Stránský</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiří Pavlíček</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiří Pavlíček</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre of Molecular Structure, Institute of Biotechnology of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ji%C5%99%C3%AD++Pavl%C3%AD%C4%8Dek">Jiří Pavlíček</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Glaucio M. Ferreira</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Glaucio M. Ferreira</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, SP 05508-000, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Glaucio+M.++Ferreira">Glaucio M. Ferreira</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Melissa Hadley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Melissa Hadley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Melissa++Hadley">Melissa Hadley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guido Pelaez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guido Pelaez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guido++Pelaez">Guido Pelaez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cyril Bařinka</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cyril Bařinka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Structural Biology, Institute of Biotechnology of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#e4a79d968d88caa685968d8a8f85a48d8690ca878597ca879e"><span class="__cf_email__" data-cfemail="591a202b3035771b382b3037323819303b2d773a382a773a23">[email protected]</span></a>. Phone: +420-325-873-777.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cyril++Ba%C5%99inka">Cyril Bařinka</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2751-3060" title="Orcid link">http://orcid.org/0000-0003-2751-3060</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alan P. Kozikowski</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alan P. Kozikowski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bright Minds Biosciences, Toronto, ON M5H 3V9, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6b0a070a052b0919020c031f0602050f1809020445080406"><span class="__cf_email__" data-cfemail="b0d1dcd1def0d2c2d9d7d8c4ddd9ded4c3d2d9df9ed3dfdd">[email protected]</span></a>. Phone: +1-773-793-5866.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alan+P.++Kozikowski">Alan P. Kozikowski</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Alejandro Villagra</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alejandro Villagra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#610017080d0d00061300210616144f040514"><span class="__cf_email__" data-cfemail="2445524d484845435645644353510a414051">[email protected]</span></a>. Phone: +1-202-994-9547.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alejandro++Villagra">Alejandro Villagra</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9346-8355" title="Orcid link">http://orcid.org/0000-0001-9346-8355</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00567&amp;href=/doi/10.1021%2Facs.jmedchem.0c00567" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 10246–10262</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 20, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 April 2020</li><li><span class="item_label"><b>Published</b> online</span>20 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00567" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00567</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10246%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSatish%2BNoonepalle%252C%2BSida%2BShen%252C%2BJakub%2BPt%25C3%25A1%25C4%258Dek%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D18%26contentID%3Dacs.jmedchem.0c00567%26title%3DRational%2BDesign%2Bof%2BSuprastat%253A%2BA%2BNovel%2BSelective%2BHistone%2BDeacetylase%2B6%2BInhibitor%2Bwith%2Bthe%2BAbility%2Bto%2BPotentiate%2BImmunotherapy%2Bin%2BMelanoma%2BModels%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10262%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00567"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2022</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00567" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rational Design of Suprastat: A Novel Selective Histone Deacetylase 6 Inhibitor with the Ability to Potentiate Immunotherapy in Melanoma Models&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Satish&quot;,&quot;last_name&quot;:&quot;Noonepalle&quot;},{&quot;first_name&quot;:&quot;Sida&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Jakub&quot;,&quot;last_name&quot;:&quot;Ptáček&quot;},{&quot;first_name&quot;:&quot;Maurício&quot;,&quot;last_name&quot;:&quot;T. Tavares&quot;},{&quot;first_name&quot;:&quot;Guiping&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Stránský&quot;},{&quot;first_name&quot;:&quot;Jiří&quot;,&quot;last_name&quot;:&quot;Pavlíček&quot;},{&quot;first_name&quot;:&quot;Glaucio&quot;,&quot;last_name&quot;:&quot;M. Ferreira&quot;},{&quot;first_name&quot;:&quot;Melissa&quot;,&quot;last_name&quot;:&quot;Hadley&quot;},{&quot;first_name&quot;:&quot;Guido&quot;,&quot;last_name&quot;:&quot;Pelaez&quot;},{&quot;first_name&quot;:&quot;Cyril&quot;,&quot;last_name&quot;:&quot;Bařinka&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;P. Kozikowski&quot;},{&quot;first_name&quot;:&quot;Alejandro&quot;,&quot;last_name&quot;:&quot;Villagra&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;10246-10262&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00567&quot;},&quot;abstract&quot;:&quot;Selective inhibition of histone deacetylase 6 (HDAC6) is being recognized as a therapeutic approach for cancers. In this study, we designed a new HDAC6 inhibitor, named Suprastat, using in silico simulations. X-ray crystallography and molecular dynamics simulations provide strong evidence to support the notion that the aminomethyl and hydroxyl groups in the capping group of Suprastat establish significant hydrogen bond interactions, either direct or water-mediated, with residues D460, N530, and S531, which play a vital role in regulating the deacetylase function of the enzyme and which are absent in other isoforms. In vitro characterization of Suprastat demonstrates subnanomolar HDAC6 inhibitory potency and a hundred- to a thousand-fold HDAC6 selectivity over the other HDAC isoforms. In vivo studies reveal that a combination of Suprastat and anti-PD1 immunotherapy enhances antitumor immune response, mediated by a decrease of protumoral M2 macrophages and increased infiltration of antitumor CD8+ effector and m&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00567&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00567" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00567&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00567" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00567&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00567" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00567&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00567&amp;href=/doi/10.1021/acs.jmedchem.0c00567" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00567" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00567" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00567%26sid%3Dliteratum%253Aachs%26pmid%3D32815366%26genre%3Darticle%26aulast%3DNoonepalle%26date%3D2020%26atitle%3DRational%2BDesign%2Bof%2BSuprastat%253A%2BA%2BNovel%2BSelective%2BHistone%2BDeacetylase%2B6%2BInhibitor%2Bwith%2Bthe%2BAbility%2Bto%2BPotentiate%2BImmunotherapy%2Bin%2BMelanoma%2BModels%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D18%26spage%3D10246%26epage%3D10262%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291081" title="Immunotherapy">Immunotherapy</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/jmcmar.2020.63.issue-18/20200924/jmcmar.2020.63.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Selective inhibition of histone deacetylase 6 (HDAC6) is being recognized as a therapeutic approach for cancers. In this study, we designed a new HDAC6 inhibitor, named Suprastat, using <i>in silico</i> simulations. X-ray crystallography and molecular dynamics simulations provide strong evidence to support the notion that the aminomethyl and hydroxyl groups in the capping group of Suprastat establish significant hydrogen bond interactions, either direct or water-mediated, with residues D460, N530, and S531, which play a vital role in regulating the deacetylase function of the enzyme and which are absent in other isoforms. <i>In vitro</i> characterization of Suprastat demonstrates subnanomolar HDAC6 inhibitory potency and a hundred- to a thousand-fold HDAC6 selectivity over the other HDAC isoforms. <i>In vivo</i> studies reveal that a combination of Suprastat and anti-PD1 immunotherapy enhances antitumor immune response, mediated by a decrease of protumoral M2 macrophages and increased infiltration of antitumor CD8+ effector and memory T-cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Reversible acetylation of lysine side chains on the surface of enzymes and other proteins is modulated by histone acetyltransferases (HATs) and histone deacetylases (HDACs). Protein lysine acetylation or deacetylation serves as a key regulatory pathway for various cellular processes, such as transcription, cell cycle, and cellular metabolism.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> HDACs have been demonstrated to be effective targets for the treatment of cancer, neurological diseases, and immune disorders.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Up to date, 11 zinc ion (Zn<sup>2+</sup>)-dependent HDACs (classes I, II, and IV) and seven nicotinamide adenine dinucleotide (NAD<sup>+</sup>)-dependent sirtuins (SIRTs) (class III) have been identified. Unlike other members, HDAC6 in class IIb is unique due to its ability to deacetylate a number of nonhistone proteins and its preferred substrates, such as α-tubulin and HSP-90.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> HDAC6 inhibitors (HDAC6is), ricolinostat (ACY-1215), citarinostat (ACY-241), and KA2507 are being evaluated in clinical trials for various types of cancers through either monotherapy or a combination approach,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> although they are only partially selective HDAC6is. Therefore, HDAC6is have emerged as a promising approach for cancer therapy.</div><div class="NLM_p">Immunomodulatory properties of HDAC6is have deemed them as therapeutic agents for cancer immunotherapy. It has been reported that HDAC6 interacts with the transcription factor STAT3, which is a primary regulator of immune responses in the tumor microenvironment,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and that it regulates STAT3-mediated gene expression.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In antigen-presenting cells (APCs), such as macrophages and dendritic cells, selective inhibition of HDAC6 leads to a decreased production of immunosuppressive cytokine IL-10, thereby retaining the proinflammatory state of APCs.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In melanoma tumor cells, HDAC6 inhibition leads to a decreased level of the immunosuppressive molecule PD-L1 by affecting the recruitment and activation of STAT3.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Furthermore, using the syngeneic murine melanoma mouse model, we recently demonstrated that a combination therapy composed of selective HDAC6is and PD-1 antibody leads to significantly improved effects on tumor growth compared to a single therapy,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> thereby underscoring the immunomodulatory capability of selective HDAC6is.</div><div class="NLM_p">The structure of one HDAC6i typically contains the following: (a) a zinc-binding group (ZBG) establishing essential coordination with the Zn<sup>2+</sup> ion in the active site, (b) a linker region filling the hydrophobic space between the catalytic site and the outer surface, and (c) a capping group (cap) interacting with residues on the surface. The phenylhydroxamate-based HDAC6is feature the critical ZBG and linker for selective and potent HDAC6 inhibition that originated from the invention of tubastatin A (TubA, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref9 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref9 ref14 ref15 ref16 ref17 ref18">(9,14−18)</a> HDAC6is are largely composed of arylhydroxamate-based analogues.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> However, the recent development of oxadiazole-based selective HDAC6is by pharmaceutical companies<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> may provide a new opportunity to refine the drug-like properties (e.g., metabolic stability, permeability, and mutagenicity)<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> of HDAC6is. Crystallographic studies of several ligands complexed with <i>Danio rerio</i> HDAC6 (zHDAC6) have revealed that most selective HDAC6is containing phenylhydroxamate exhibit an unusual monodentate Zn<sup>2+</sup> coordination geometry.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> On the other hand, a typical canonical bidentate Zn<sup>2+</sup> coordination was observed in a number of zHDAC6 crystals complexing with HDACis bearing alkylhydroxamate or capless arylhydroxamate.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> In addition, the phenylhydroxamate moiety is sandwiched by F583 and F643 in the hydrophobic tunnel, establishing a double π stacking interaction.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structures of zHDAC6 complexes with selective HDAC6is. (A) Tubastatin A (yellow; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6THV">6THV</a>). (B) RTS-V5 (green; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CW8">6CW8</a>). (C) ACY-1083 (pink; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGM">5WGM</a>). (D) HPB (blue; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGK">5WGK</a>). (E) SS-208 (cyan; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R0K">6R0K</a>). (F) CBZ (purple; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZU">6PZU</a>). Selective amino acid residues in HDAC6 are indicated in stick representation with carbon atoms in gray, oxygen atoms in red, and nitrogen atoms in blue colors, whereas the active site zinc ion and water molecules are indicated as silver and red spheres, respectively. IC<sub>50</sub> values for HDAC1 and HDAC6, along with selectivity indices, are indicated for each HDAC6 inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Among zinc-dependent HDACs, only HDAC6 contains a large open basin approximately 14 Å wide. Thus, a lot of selective HDAC6is consist of a bulky and rigid cap to occupy the broad rim of the pocket.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Polycyclic aromatic rings are often considered as useful caps in selective HDAC6is that can form robust hydrophobic engagements with the L1 loop pocket defined by key residues H463, P464, F583, and L712, which is considered as the selectivity-determining area.<a onclick="showRef(event, 'ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27">(25−27)</a> The crystal structure of the enzyme in complex with TubA (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6THV">6THV</a>) is shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A as an example.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Recent findings provide additional examples, such as resminostat (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZR">6PZR</a>), whose cap engages with another pocket defined by the L2 loop.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Moreover, the bifurcated cap of the dual HDAC/proteasome inhibitor RTS-V5 can occupy both L1 and L2 pockets and interact with the nearby residues G640–N645 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CW8">6CW8</a>).<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a></div><div class="NLM_p">In addition to hydrophobic and π stacking interactions, HDAC6 crystals in complex with selective HDAC6is also indicate that capping or linker groups form additional hydrogen bonds with the residue S531 inside the pocket: (a) The NH group on the linker of pyrimidinylhydroxamate-based ACY-1083 shows a direct hydrogen-bonding interaction to the hydroxyl group of S531 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGM">5WGM</a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (b) The hydroxyl group on the <i>n</i>-propyl tail chain of HPB forms a water-mediated hydrogen bond with S531 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGK">5WGK</a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (c) The carbonyl group on the amide connecting unit of isoxazole-3-hydroxamate-based SS-208 interacts with S531 at a distance of 3.3 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R0K">6R0K</a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Moreover, hydrogen-bonding interactions with other residues (e.g., D460 and F643) are also observed in zHDAC6 crystal complexes: (a) Ricolinostat establishes water-mediated hydrogen-bonding interactions with D460 and S531 through its aminopyrimidinyl core and amide linkage, respectively (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGL">5WGL</a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (b) zHDAC6 in complex with a Cbz-protected dipeptide (Leu-Ala) capped HDAC6i CBZ (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZU">6PZU</a>) reveals that the phenyl ring of the Cbz protecting group forms a carbon–hydrogen bond with D460. In contrast, hydrogen bond interactions mediated by water molecules are observed with the imidazole ring of H614 and the amino group of F643.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Noteworthy, S531 is critical for substrate recognition and engages with the NH group of the <i>N</i>-acetyllysine moiety through a hydrogen bond interaction.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> It was also found that a single mutation (S531A) in sequence encoding zHDAC6-CD2 decreased the catalytic efficiency by 258-fold.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Moreover, it was reported by the Matthias group that double mutations (W459A and D460A) and other single mutations (N530A, N530D, or S531A) significantly reduced α-tubulin deacetylation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> It should be noted that N530 of zHDAC6 corresponds to D567 in the human ortholog. Surprisingly, the N530D substitution in zHDAC6 led to a substantial decrease in the deacetylation efficacy of α-tubulin. At the same time, it has been reported by Hai and Christianson that the N530A substitution in zHDAC6 and D567A substitution in hHDAC6 did not significantly influence HDAC6 deacetylase activity using a fluorogenic substrate derived from histone H4.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Apparently, the residue might play an important role in the interaction interface with proteinaceous substrates (e.g., tubulin) but not small peptides. The detailed understanding of the N530 (D567) role in substrate recognition warrants further studies.</div><div class="NLM_p last">Overall, we hypothesized that establishing specific hydrogen-bonding interactions with these key residues, which play crucial roles in regulating the deacetylation function of HDAC6 and which are absent in other isoforms, might lead to enhanced potency and excellent HDAC6 isoform selectivity.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Currently, a typical strategy in the discovery of selective HDAC6is relies on modifications of aromatic capping groups to strengthen hydrophobic interactions with residues of the L1 loop pocket.<a onclick="showRef(event, 'ref9 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref9 ref16 ref17">(9,16,17)</a> As an alternative approach, we designed new analogues based on the nexturastat A (NextA) scaffold by incorporating polar groups to interact with key residues of the HDAC6 pocket. Integrated structural and biological characterization demonstrates that one analogue, named Suprastat, represents a new generation of selective HDAC6i.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17165" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17165" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design and Synthesis of Suprastat (<b>6a</b>)</h3><div class="NLM_p">NextA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, left) comprises a classical phenylhydroxamate zinc-binding core linked to a urea-based cap featuring a phenyl ring and an <i>n</i>-butyl side chain, which exhibits good affinity and selectivity for HDAC6.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The HDAC6/NextA crystal (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) reveals that there is no direct enzyme–inhibitor hydrogen-bonding interaction between the cap and the pocket. Instead, the steric complementarity between the bulky cap and the 3D contour of the pocket drives important hydrophobic interactions. NextA exhibited minimal antiproliferative activity against a variety of human cancer cell lines and induced G1 cell cycle arrest without eliciting apoptosis.<a onclick="showRef(event, 'ref26 ref34'); return false;" href="javascript:void(0);" class="ref ref26 ref34">(26,34)</a> Moreover, it was found that NextA downregulated the level of the immunosuppressive molecule PD-L1 (<i>CD274</i>) in melanoma cells and additionally impaired melanoma tumor growth in immunocompetent mice mediated by increased tumor-specific immunogenic signals.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Recent findings further demonstrated that a combination treatment employing NextA along with anti-PD1 immune blockade resulted in enhanced T-cell infiltration coupled with a decrease of protumor M2 macrophages in the tumor microenvironment, leading to significant tumor growth inhibition and a higher survival rate compared to monotherapy.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Given the well-understood molecular basis for its selective inhibition of HDAC6 and its efficacy in established murine melanoma models, we selected NextA as a parent scaffold for designing the next generation of HDAC6is.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In silico</i> design of Suprastat by NextA derivatization. (A) Structures of phenylhydroxamate-based nexturastat A (left) and Suprastat (right). (B) Molecular docking of Suprastat (orange) based on the HDAC6/nexturastat A complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>; silver) with an indication of the additional interactions created by the aminomethyl and hydroxybutyl polar groups with residues D460 and F643 at the rim of the catalytic pocket of HDAC6, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We reasoned that the derivatization of the NextA cap with polar groups could further increase potency and selectivity via potential interactions with the residues at the rim region of the HDAC6 catalytic core. Molecular docking studies using the HDAC6/NextA complex as a template were performed for a series of newly designed NextA derivatives. We examined their capacity to extend the interaction interface of the inhibitor cap with the HDAC6 rim residues. The most promising derivative, named Suprastat, contains an aminomethyl group at the <i>para</i> position of the phenyl ring and a hydroxyl group at the end of the <i>n-</i>butyl chain attached to the proximal urea nitrogen (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, right). Based on the <i>in silico</i> predictions, the positively charged aminomethyl and hydroxybutyl branches of Suprastat interact with the carboxyl group of D460 and the main chain amino group of F643, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><div class="NLM_p">Moreover, we performed relative binding free energy calculations<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> on the HDAC6/NextA and HDAC6/Suprastat complexes. The results indicate that the HDAC6/Suprastat complex shows a much lower free energy of binding (Δ<i>G</i><sup>0</sup> = −67.25 kcal mol<sup>–1</sup>) compared to HDAC6/NextA (Δ<i>G</i><sup>0</sup> = −38.85 kcal mol<sup>–1</sup>), suggesting an improved affinity of Suprastat for the HDAC6 catalytic pocket. The enthalpy (Δ<i>H</i><sup>0</sup>) and entropy (Δ<i>S</i><sup>0</sup>) contribute to both Δ<i>G</i><sup>0</sup> values, determined by the Gibbs free energy equation (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The results suggest that the improved affinity of Suprastat is accompanied by a more substantial entropy gain (−<i>T</i>Δ<i>S</i><sup>0</sup>).</div><div class="NLM_p">The synthetic route to Suprastat was initiated by the preparation of a carbamate intermediate <b>2</b> from phenyl chloroformate and aniline <b>1</b> under K<sub>2</sub>CO<sub>3</sub>/acetone conditions (<a class="ref internalNav" href="#sch1" aria-label="Figure 1">Figure 1</a>A). On the other hand, methyl 4-formylbenzoate <b>3a</b> underwent a rapid reductive amination with 4-amino-1-butanol to provide intermediate <b>4a</b>. Subsequently, the combination reaction between <b>2</b> and <b>4a</b> under TEA/THF conditions afforded the key urea precursor <b>5a</b>, which was further converted to the final hydroxamate product <b>6a</b> (Suprastat) using aqueous hydroxylamine under basic conditions followed by TFA/THF to remove the Boc group. For evaluation together with Suprastat in the following biological experiments, analogue <b>6b</b> bearing an aminomethyl group and the original <i>n-</i>butyl chain attached to the proximal urea nitrogen was also prepared from <b>3a</b> and <i>n</i>-butylamine using the same synthetic route to Suprastat. The synthesis of <b>6c</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which contains a hydroxylbutyl side chain but no aromatic substituent, has been reported in our prior work.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Moreover, nonhydroxamate analogues <b>6d</b>–<b>6f</b> containing the same cap with Suprastat were prepared and evaluated to explore if additional hydrogen-bonding interactions would be able to retain activity without a hydroxamate ZBG, inspired by ketone/amide-based class I HDACis.<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37−39)</a> The carboxylic acid analogue <b>6d</b> was directly afforded from the urea ester <b>5a</b> through hydrolysis under basic condition and Boc deprotection. To synthesize the amide analogue <b>6e</b>, 4-formylbenzonitrile <b>3b</b> underwent the two-step reductive amination followed by the reaction with carbamate <b>2</b> to generate the intermediate urea <b>5c</b>. The nitrile group in <b>5c</b> was further converted to an amide group by treating with aqueous hydrogen peroxide solution under basic condition, and the final product <b>6e</b> was afforded through Boc deprotection as described above. The synthetic route to the boronic acid analogue <b>6f</b> was initiated with the reductive amination of 4-bromobenzaldehyde <b>3c</b> followed by urea formation using the same procedures as above to give the intermediate urea <b>5d</b>. The precursor <b>5d</b> underwent a coupling reaction with bis(pinacolato)diboron under KOAc/Pd(dppf)Cl<sub>2</sub> conditions. In the end, the desired boronic acid product <b>6f</b> was obtained through pinacol deprotection and Boc deprotection under NaIO<sub>4</sub>/NH<sub>4</sub>OAc and TFA/THF, respectively.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route to <b>6a</b> and <b>6b</b> and <b>6d</b>–<b>6f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phenyl chloroformate, K<sub>2</sub>CO<sub>3</sub>, acetone, rt, and 2 h; (b) (i) 4-amino-1-butanol for <b>4a</b>, <b>4c</b>, and <b>4d</b> and <i>n</i>-butylamine for <b>4b</b>, EtOH, reflux, and 2 h; and (ii) NaBH<sub>4</sub>, MeOH, 0 °C to rt, and 2 h; (c) <b>2</b>, TEA, THF, reflux, and 2 h; (d) for <b>6a</b> and <b>6b</b>: (i) aqueous NH<sub>2</sub>OH (50%), NaOH, THF/MeOH, 0 °C, and 15 min; and (ii) TFA, THF, rt, and 0.5 h; (e) for <b>6d</b>: (i) 1 N NaOH, THF/MeOH, rt, and overnight; and (ii) TFA, THF, rt, and 0.5 h; (f) for <b>6e</b>: (i) H<sub>2</sub>O<sub>2</sub> (30%), K<sub>2</sub>CO<sub>3</sub>, DMSO, rt, and 5 h; and (ii) TFA, THF, rt, and 0.5 h; and (g) for <b>6f</b>: (i) B<sub>2</sub>pin<sub>2</sub>, KOAc, Pd(dppf)Cl<sub>2</sub>, DMF, 80 °C, and overnight; (ii) NaIO<sub>4</sub>, NH<sub>4</sub>OAc, acetone/H<sub>2</sub>O, rt, and overnight; and (iii) TFA, THF, rt, and 0.5 h.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>In Vitro</i> HDAC Profiles of <b>6a</b>–<b>6c</b> and NextA<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0011.gif" alt="" id="gr10" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">Suprastat (<b>6a</b>)</th><th class="colsep0 rowsep0" align="center"><b>6b</b></th><th class="colsep0 rowsep0" align="center"><b>6c</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">NextA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R<sup>1</sup></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R<sup>2</sup></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Isoform</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SI<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SI</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SI</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SI</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC6</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±">0.9 ± 0.7</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±">0.5 ± 0.4</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±">1.6 ± 0.4</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC1</td><td class="colsep0 rowsep0" align="char" char="±">117 ± 10</td><td class="colsep0 rowsep0" align="left">293</td><td class="colsep0 rowsep0" align="char" char="±">80 ± 45</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="char" char="±">148 ± 9</td><td class="colsep0 rowsep0" align="left">296</td><td class="colsep0 rowsep0" align="char" char="±">151 ± 20</td><td class="colsep0 rowsep0" align="left">94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC2</td><td class="colsep0 rowsep0" align="char" char="±">176 ± 21</td><td class="colsep0 rowsep0" align="left">440</td><td class="colsep0 rowsep0" align="char" char="±">281 ± 28</td><td class="colsep0 rowsep0" align="left">312</td><td class="colsep0 rowsep0" align="char" char="±">268 ± 16</td><td class="colsep0 rowsep0" align="left">536</td><td class="colsep0 rowsep0" align="char" char="±">276 ± 96</td><td class="colsep0 rowsep0" align="left">173</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC3</td><td class="colsep0 rowsep0" align="char" char="±">352 ± 2</td><td class="colsep0 rowsep0" align="left">880</td><td class="colsep0 rowsep0" align="char" char="±">443 ± 108</td><td class="colsep0 rowsep0" align="left">492</td><td class="colsep0 rowsep0" align="char" char="±">581 ± 18</td><td class="colsep0 rowsep0" align="left">1160</td><td class="colsep0 rowsep0" align="char" char="±">1420 ± 145</td><td class="colsep0 rowsep0" align="left">887</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC4</td><td class="colsep0 rowsep0" align="char" char="±">8250 ± 1350</td><td class="colsep0 rowsep0" align="left">20,600</td><td class="colsep0 rowsep0" align="char" char="±">23,100 ± 226</td><td class="colsep0 rowsep0" align="left">25,700</td><td class="colsep0 rowsep0" align="char" char="±">14,000 ± 1380</td><td class="colsep0 rowsep0" align="left">27,900</td><td class="colsep0 rowsep0" align="char" char="±">14,800 ± 1700</td><td class="colsep0 rowsep0" align="left">9250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC5</td><td class="colsep0 rowsep0" align="char" char="±">3420 ± 373</td><td class="colsep0 rowsep0" align="left">8550</td><td class="colsep0 rowsep0" align="char" char="±">10,100 ± 162</td><td class="colsep0 rowsep0" align="left">11,200</td><td class="colsep0 rowsep0" align="char" char="±">9130 ± 119</td><td class="colsep0 rowsep0" align="left">18,300</td><td class="colsep0 rowsep0" align="char" char="±">6620 ± 2660</td><td class="colsep0 rowsep0" align="left">4140</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC7</td><td class="colsep0 rowsep0" align="char" char="±">1470 ± 56</td><td class="colsep0 rowsep0" align="left">3680</td><td class="colsep0 rowsep0" align="char" char="±">9000 ± 1110</td><td class="colsep0 rowsep0" align="left">10,000</td><td class="colsep0 rowsep0" align="char" char="±">1930 ± 70</td><td class="colsep0 rowsep0" align="left">3870</td><td class="colsep0 rowsep0" align="char" char="±">2430 ± 300</td><td class="colsep0 rowsep0" align="left">1520</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC8</td><td class="colsep0 rowsep0" align="char" char="±">498 ± 58</td><td class="colsep0 rowsep0" align="left">1250</td><td class="colsep0 rowsep0" align="char" char="±">614 ± 58</td><td class="colsep0 rowsep0" align="left">682</td><td class="colsep0 rowsep0" align="char" char="±">478 ± 97</td><td class="colsep0 rowsep0" align="left">956</td><td class="colsep0 rowsep0" align="char" char="±">988 ± 264</td><td class="colsep0 rowsep0" align="left">618</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC9</td><td class="colsep0 rowsep0" align="char" char="±">6270 ± 177</td><td class="colsep0 rowsep0" align="left">15,700</td><td class="colsep0 rowsep0" align="char" char="±">17,300 ± 4990</td><td class="colsep0 rowsep0" align="left">19,200</td><td class="colsep0 rowsep0" align="char" char="±">29,700 ± 4190</td><td class="colsep0 rowsep0" align="left">5950</td><td class="colsep0 rowsep0" align="char" char="±">2000 ± 770</td><td class="colsep0 rowsep0" align="left">1250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC11</td><td class="colsep0 rowsep0" align="char" char="±">>50,000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">>50,000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">>50,000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">10,600 ± 2200</td><td class="colsep0 rowsep0" align="left">6630</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the mean of two experiments ± SEM calculated by nonlinear regression analysis from experimental <i>v</i><sub>i</sub>/<i>v</i><sub>0</sub> values for each HDAC isoform.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last"><b>6c</b> was originally published as compound <b>7b</b> in ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">SI: HDAC6 selectivity index over other HDAC isoforms.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> <i>In Vitro</i> HDAC Potency Assessment</h3><div class="NLM_p">To evaluate the influence of each additional functional group on the potency and isoform selectivity, we tested the potency of <b>6a</b>–<b>6c</b> along with NextA against human HDAC1–HDAC9 and HDAC11 under optimized conditions <i>in vitro</i>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> It should be mentioned that our highly pure (>98%) HDAC10 preparations heterologously expressed in HEK 293T cells did not show any appreciable activity using either acetylspermidine<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> or RHKK(Ac)AMC (typically used by Reaction Biology Corp., Malvern, PA) as the substrate for HDAC10 profiling. Therefore, the IC<sub>50</sub> values of HDAC10 were not included. Results in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> suggest that <b>6a</b> and <b>6c</b> are more potent and selective HDAC6is (IC<sub>50</sub> = 0.4 vs 0.5 nM) with at least 290-fold selectivity over class I isoform HDAC1–HDAC3 and HDAC8 and a thousand-fold selectivity over class IIa isoform HDAC4, HDAC5, HDAC7, and HDAC9. Moreover, <b>6a</b>–<b>6c</b> did not show activity against the class IV isoform HDAC11 up to 50 μM. It shall be noted that experimental IC<sub>50</sub> values in the range of 0.2–0.4 nM for HDAC6 are at the limit of our assay that uses approximately 0.6 nM concentration of the full-length enzyme (as determined by absorbance measurements at 280 nm). IC<sub>50</sub> values for Suprastat and <b>6c</b> are thus at the limit of our assay, and these two inhibitors can theoretically have an even higher potency than reported here. Overall, the inhibition data suggest that the hydroxylbutyl chain contributes more prominently toward the increase in HDAC6 affinity and selectivity compared to the aminomethyl group. Additionally, the incorporation of the polar aminomethyl and hydroxylbutyl groups into the inhibitors <b>6a</b>–<b>6c</b> increases the number of heavy atoms but decreases their clog P (calculated by SwissADME<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a>) relative to NextA (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), which leads to significantly elevated lipophilic ligand efficiencies (LipE),<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> especially <b>6a</b> (LipE = 7.57 (<b>6a</b>) vs 6.19 (NextA)), although the ligand efficiencies (LE) of <b>6a</b> and <b>6b</b> are slightly lower.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Taken together, compared to NextA and related analogues, Suprastat (<b>6a</b>) bearing both hydroxybutyl and aminomethyl moieties showed improved potency against HDAC6 and excellent selectivity over the other HDAC isoforms. At the same time, it also exhibits the highest LipE value due to its significantly decreased clogP. On the other hand, the inhibitory potency of the nonhydroxamate-based analogues <b>6d</b>–<b>6f</b> composed of carboxylic acid, amide, or boronic acid as alternative ZBGs against HDAC6 was severely compromised (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Table S1</a>), which would indicate that the additional hydrogen-bonding interactions between the cap and HDAC6 pocket are not strong enough to maintain nanomolar potency when the critical hydroxamate–Zn<sup>2+</sup> coordination is absent.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Ligand Efficiency and <i>In Vitro</i> ADME Profiling of <b>6a</b>–<b>6c</b> and NextA</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">Suprastat (<b>6a</b>)</th><th class="colsep0 rowsep0" align="center"><b>6b</b></th><th class="colsep0 rowsep0" align="center"><b>6c</b></th><th class="colsep0 rowsep0" align="center">NextA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clog P<sub>o/w</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">1.23</td><td class="colsep0 rowsep0" align="left">2.21</td><td class="colsep0 rowsep0" align="left">1.82</td><td class="colsep0 rowsep0" align="left">2.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.47</td><td class="colsep0 rowsep0" align="left">0.47</td><td class="colsep0 rowsep0" align="left">0.50</td><td class="colsep0 rowsep0" align="left">0.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LipE<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">7.57</td><td class="colsep0 rowsep0" align="left">6.59</td><td class="colsep0 rowsep0" align="left">6.98</td><td class="colsep0 rowsep0" align="left">6.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PBS stability (<i>t</i><sub>1/2</sub>, h)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">>24</td><td class="colsep0 rowsep0" align="left">>24</td><td class="colsep0 rowsep0" align="left">>24</td><td class="colsep0 rowsep0" align="left">>24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SGF stability (<i>t</i><sub>1/2</sub>, h)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">>5</td><td class="colsep0 rowsep0" align="left">>5</td><td class="colsep0 rowsep0" align="left">>5</td><td class="colsep0 rowsep0" align="left">>5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human plasma stability (<i>t</i><sub>1/2</sub>, h)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">>5</td><td class="colsep0 rowsep0" align="left">>5</td><td class="colsep0 rowsep0" align="left">>5</td><td class="colsep0 rowsep0" align="left">>5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat liver microsomes (<i>t</i><sub>1/2</sub>, min)<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">177 ± 6</td><td class="colsep0 rowsep0" align="left">173 ± 71</td><td class="colsep0 rowsep0" align="left">198 ± 32</td><td class="colsep0 rowsep0" align="left">423 ± 104</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human plasma binding (%)<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">1.9 ± 3.5</td><td class="colsep0 rowsep0" align="left">47 ± 2.0</td><td class="colsep0 rowsep0" align="left">48 ± 0.9</td><td class="colsep0 rowsep0" align="left">89 ± 0.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">clog P<sub>o/w</sub> values were calculated by SwissADME<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (<a href="http://www.swissadme.ch/" class="extLink">http://www.swissadme.ch/</a>).</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">LE: ligand efficiency = 1.4 × pIC<sub>50</sub>/number of heavy atoms.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">LipE: lipophilic ligand efficiency = pIC<sub>50</sub> – clog P<sub>o/w</sub>.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Data were obtained from two independent experiments run in triplicates.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Data are presented as the mean ± standard error from two independent experiments run in triplicates.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">Data are presented as the mean ± standard error from two independent experiments run in duplicates.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> <i>In Vitro</i> ADME Profiling of <b>6a</b>–<b>6c</b></h3><div class="NLM_p last">As a part of the initial ADME profiling, we determined the stability of studied compounds in PBS, simulated gastric fluid (SGF), human plasma, and rat liver microsomes as well as protein binding in human plasma (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Overall, the stability of Suprastat (<b>6a</b>) is excellent, ranging from >24 h in PBS to a half-life (<i>t</i><sub>1/2</sub>) of 173 min in rat liver microsomes. In line with predicted physicochemical characteristics, plasma binding of Suprastat is much lower (2% of plasma protein-bound fraction), which is beneficial for elevating free drug concentration <i>in vivo</i>, compared to the more hydrophobic parent compound NextA (89%). In comparison, plasma-bound fractions of singly modified analogues <b>6b</b> and <b>6c</b> both are approximately 50%.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> <i>In Vitro</i> Characterization of <b>6a</b>–<b>6c</b> in Melanoma Cells</h3><div class="NLM_p">To assess the potency and isoform selectivity of <b>6a</b>–<b>6c</b> in cells, we performed <i>in vitro</i> analysis using the WM164 human melanoma cell line. WM164 cells are mutant for BRAF V600E, a mutation that is frequently seen in melanoma patients.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> WM164 cells were treated with <b>6a</b>–<b>6c</b> and NextA with a concentration ranging from 0.1 to 10 μM. Their abilities to increase the levels of acetylated α-tubulin (Ac-α-tubulin) were determined by immunoblot analysis and compared side-by-side. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–D shows an increase in Ac-α-tubulin with increasing concentrations of HDAC6is but with varying magnitudes (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). However, the increase in Ac-α-tubulin was also associated with a slight increase in the levels of acetylated histone H3 (Ac-H3) albeit at higher concentrations (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Treatment with Suprastat led to the highest Ac-α-tubulin levels ranging from 0.1 to 10 μM and the most significant elevation at 10 μM compared to other HDAC6is. At higher concentrations, we did observe a slight increase in the levels of Ac-H3 with Suprastat, but it was much lower than with other HDAC6is. These data thus demonstrate that Suprastat, containing both aminomethyl and hydroxylbutyl groups, exhibits better HDAC6 potency and selectivity when tested in live cells. Based on the concentration range for the α-tubulin/histone acetylation experiments, additional cytotoxicity assays were performed in SM1 murine melanoma cells. The results shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Figure S1</a> reveal that NextA and <b>6b</b> begin to induce cytotoxicity at a concentration of 10 μM, while Suprastat and <b>6c</b> were not cytotoxic up to 25 μM.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Suprastat is a highly selective HDAC6 inhibitor. WM164 human melanoma cells treated overnight with increasing concentrations ranging from 0.1 to 10 μM for HDAC6 inhibitors. (A) Suprastat (<b>6a</b>). (B) <b>6b</b>. (C) <b>6c</b>. (D) NextA. Immunoblot analysis was performed for Ac-α-tubulin and Ac-H3. Total α-tubulin and total H3 are loading controls. (E, F) Densitometric analyses of the Ac-α-tubulin and Ac-H3 bands, respectively. The immunoblots were repeated at least twice to confirm accuracy, and the best representation is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Functional Characterization of Suprastat</h3><div class="NLM_p">Immune cells, such as macrophages, are a major cellular component of the tumor microenvironment. Tumor-associated macrophages are often tumor-promoting by secreting anti-inflammatory cytokines such as TGFβ and IL-10. We previously established that HDAC6 forms a complex with STAT3, and either pharmacological inhibition or shRNA-mediated knockdown of HDAC6 decreased STAT3 recruitment at the IL10 promoter region in antigen-presenting cells.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In line with these experiments, as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, the treatment of mouse bone marrow-derived macrophages with 5 μM Suprastat resulted in decreased expression of <i>IL10</i> gene compared to vehicle-treated macrophages as determined by mRNA quantification. <i>IL10</i> gene expression is normalized to β-actin (<i>ACTB</i>) as the reference gene. A dose-dependent increase in the levels of Ac-α-tubulin was observed in RAW 264.7 macrophages (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Figure S2</a>) when they were treated with increasing concentrations (1, 5, and 10 μM) of Suprastat, indicating that Suprastat affects macrophages and melanoma cells in a similar fashion. Furthermore, as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, immunoblot analysis of lysates obtained from WM164 melanoma cells pretreated with either Suprastat or NextA followed by exposure to IL-6 cytokine (30 ng mL<sup>–1</sup>) for 20 min resulted in decreased Y705 phosphorylation of STAT3 compared to IL-6 alone. This result indicates that Suprastat, similar to NextA, mediates immunomodulatory effects by affecting HDAC6 interaction with the STAT3 transcription factor.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Suprastat modulates immune pathways through interaction with STAT3. (A) <i>IL10</i> gene expression was determined by quantitative PCR in bone marrow-derived macrophages exposed to interferon-gamma (20 ng mL<sup>–1</sup>) and LPS (100 ng mL<sup>–1</sup>). (B) WM164 murine melanoma cells were pretreated with 5 μM Suprastat or NextA followed by treatment with IL-6 (30 ng mL<sup>–1</sup>) for 20 min.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <i>In Vivo</i> Combination Study with Immunotherapy</h3><div class="NLM_p">Immunotherapy is emerging as a primary treatment modality for solid tumors; however, patients will often develop resistance, and currently, there is a need for combination therapies to increase the effectiveness of immunotherapy while overcoming resistance.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Using the syngeneic SM1 murine melanoma model, we previously demonstrated that pretreatment with NextA significantly decreased tumor size in C57BL/6 mice. These mice have an active immune system, which enables us to test immune checkpoint inhibitors such as anti-PD1 therapy. The combination of NextA and anti-PD1 treatment resulted in substantial control of tumor growth compared to single-arm therapies, suggesting that HDAC6 inhibition plays a significant role in enhancing antitumor immunity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Following a similar approach, C57BL/6 mice harboring SM1 melanoma tumors were administered 25 mg kg<sup>–1</sup> Suprastat intraperitoneally (IP) before starting the anti-PD1 immune checkpoint blockade therapy (15 mg kg<sup>–1</sup>, IP). The pretreatment modality was performed to precondition the tumor microenvironment (TME) in a manner conducive to eliciting an antitumor immune response. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, compared to the control (PBS) group, single-arm therapies with Suprastat similarly reduced the tumor burden as indicated by the tumor volume. However, we did not see a significant difference between Suprastat and anti-PD1 therapy. On the contrary, a combination of Suprastat and anti-PD1 therapy showed a substantial decrease in tumor burden compared to control and single therapy groups, suggesting that pretreatment with Suprastat enhances the antitumor immune response resulting from the anti-PD1 therapy. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B shows the tumor growth of each mouse in the respective treatment groups. It was noted that the Suprastat and anti-PD1 therapy combined group exhibited enhanced inhibitory effects on the tumor growth before day 14 compared to other groups. On the other hand, the combination of NextA and anti-PD1 therapy started to exhibit distinct antitumor effects after day 20 relative to single therapy groups in our previous studies,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> suggesting that Suprastat is capable of promoting the immunotherapy at an earlier stage.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Combination of Suprastat with anti-PD1 immunotherapy significantly decreases tumor burden in the SM1 murine melanoma model. (A) Cumulative tumor growth rates in C57BL/6 mice (<i>n</i> = 10) after treatment with Suprastat or anti-PD1 therapy or a combination of both. Tumor growth rates of treatment groups were compared to control groups treated with PBS. (B) Tumor growth rates of individual mice in respective treatment groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Immunomodulatory Properties of Suprastat</h3><div class="NLM_p">To understand the immunomodulatory properties of Suprastat, we performed a comprehensive immune cell phenotyping by flow cytometry. The number of F4/80+ CD80+ H2+ antitumor M1 macrophages as a percentage of CD45+ cells did not significantly change in all of the treatment groups (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). However, Suprastat significantly decreased F4/80+ CD206+ protumor M2 macrophages (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B), thus shifting the balance toward an antitumor immune response as indicated by the considerably higher M1/M2 ratio (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) in the Suprastat and combination groups relative to control and anti-PD1 groups. Interestingly, the enhanced M1/M2 ratio in the combination group was not observed in our prior work with NextA.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Analysis of lymphoid cells shows a significant increase in CD8+ effector T-cells and effector memory (EM) cells in all of the treated groups compared to the control group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D) in which the measured fold increases were more significant in all the groups than prior studies performed with NextA. However, we only observed an increase in the percentage of CD8+ central memory (CM) cells with the Suprastat-treated group, suggesting that it can enhance the effector memory function of CD8 T-cells for prolonged antitumor immune responses. Analysis of CD4+ T-cells did not show any significant changes in central memory (CM) or effector memory (EM) functions (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E). We also did not see a substantial change in immunosuppressive T-regs (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). Further analysis of natural killer (NK) cells demonstrated an increase in the anti-PD1 group and combination group but did have a positive trend in the Suprastat group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>G), which was not observed in previous combination studies using NextA.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> NK T-cells were significantly decreased in all treatment groups compared to the control group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>H), which may be speculated to a significant increase in CD8+ effector T-cells. Overall, the immune cell phenotyping indicates that Suprastat has improved immunomodulatory properties by decreasing protumoral M2 macrophages and increasing the infiltration of antitumor CD8+ effector T-cells and memory cells relative to parent compound NextA. These factors are, in turn, likely responsible for the enhanced immunomodulatory effects observed <i>in vivo</i> and the improved antitumor immune response in combination with anti-PD1 therapy.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Suprastat has immunomodulatory properties in the tumor microenvironment. (A–C) Macrophages, (D–F) T-cells, and (G, H) NK cells were stained with cell surface markers and analyzed by multicolor flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> X-ray Crystallography</h3><div class="NLM_p"><i>In vitro</i> and <i>in vivo</i> characterization demonstrates that Suprastat is a highly selective HDAC6 inhibitor with significant antitumor effects through its immunomodulatory properties. To corroborate our <i>in silico</i> models experimentally, we solved the crystal structure of the zHDAC6/Suprastat complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY">6TCY</a>) to the resolution limit of 1.60 Å. The complex crystallized in the <i>P</i>12<sub>1</sub>1 monoclinic space group with two monomers in the asymmetric unit. There are no major conformational differences between the two monomers, as documented by the RMSD of 0.12 Å for 314 corresponding Cα atoms. Interestingly though, the inhibitor can be modeled in two different conformations for each monomer in the asymmetric unit (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B). In the first and more populated inhibitor conformation, monodentate coordination is observed for the zinc atom with the hydroxamate N–O<sup>–</sup> group with a Zn<sup>2+</sup>···O<sup>–</sup> distance of 2.24 Å. It is interesting to note that typical Zn<sup>2+</sup>···O<sup>–</sup> interatomic distances for monodentate HDAC6 complexes reported previously are shorter, falling into the range of 1.8–2.0 Å.<a onclick="showRef(event, 'ref23 ref24 ref25 ref46'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref46">(23−25,46)</a> At the same time, however, the Zn<sup>2+</sup>···O<sup>–</sup> interatomic distance in the NextA complex is 2.2 Å<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> that is virtually identical to the distance reported here. Clearly, the capping group of NextA (as well as its Suprastat derivative) might impose steric constraints on inhibitor positioning that leads to the somewhat atypical Zn<sup>2+</sup> coordination. In the less populated conformation, the hydroxamate moiety coordinates with the active site zinc ion in a bidentate fashion with interatomic distances of 2.25 and 2.34 Å for the hydroxamate N–O<sup>–</sup> and C═O groups, respectively.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Crystal structure of the zHDAC6/Suprastat complex. (A) Both monodentate (left) and bidentate (right) coordination of the active site Zn<sup>2+</sup> ion is observed in the complex. (B) The omit <i>F</i><sub>o</sub>–<i>F</i><sub>c</sub> difference electron density map for Suprastat is shown (green mesh) for the more prevalent monodentate binding mode (contoured at the 2σ level). (C, D) Nonpolar interactions between the phenylhydroxamate function of Suprastat and residues delineating the internal tunnel (C) and the capping group and the hydrophobic patch of the L1 loop (red surface representation; D). (E) Newly introduced polar/ionic contacts between the aminomethyl and hydroxylbutyl groups of Suprastat and D460 and N530, respectively (black dashed lines). (F) Superposition of binding modes of NextA (yellow; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>) and Suprastat (cyan) documenting minimal structural differences between the two structures. Both structures were superposed on corresponding 320 Cα atoms with an RMSD of 0.192 Å. Inhibitors are shown in stick representation, carbon atoms are colored cyan (Suprastat) or yellow (NextA), oxygens are red, and nitrogens are blue. The catalytic zinc ion and water molecules are depicted as purple and red spheres, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The phenyl rings of the linker fully overlap between the two conformers and are located in the internal tunnel delineated by the side chains of F583, F643, and H614 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). As for the capping group, the phenyl ring packs against a hydrophobic patch defined by the side chains of H463, P464, F583, and L712 of the L1 loop pocket (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D). As predicted by docking simulations, the positively charged aminomethyl group (p<i>K</i><sub>a</sub> = 9.46, calculated by MarvinSketch version 20.11) forms a salt bridge and a water-mediated hydrogen bond (3.0 Å) with the carboxylate group of D460 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E). On the other hand, contrary to predictions, the hydroxybutyl group extends away from the protein, and its distal hydroxyl group thus does not engage any residue of the HDAC6 rim directly but forms a solvent-mediated hydrogen bond with the side chain of N530 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E). A 1.99 Å resolution crystal structure of the zHDAC6/NextA complex was reported by the Matthias group in 2016 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Superposition of the two complexes reveals virtually identical positions of the capping groups and surrounding HDAC6 residues, suggesting that our derivatization strategy did not elicit any (additional) conformational strain on the binding pose of the inhibitor (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>F).</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Molecular Dynamics Simulations</h3><div class="NLM_p">X-ray crystallography is invaluable for the structure-assisted design of inhibitors yet, and crystal structures typically offer only a static snapshot of a single inhibitor binding pose disregarding the flexibility of inhibitors’ functional moieties and protein residues. Moreover, the protein/inhibitor interaction pattern can also be influenced by crystallographic contacts. In solution, an inhibitor can interact only with either water molecules or the target protein. In crystals (due to crystal packing), functional groups (that stick outside the HDAC6 pocket) can also interact “nonspecifically” with neighboring molecules from the crystal. Consequently, the interaction pattern can be influenced by such contact and differ from the solution; thus, one does simulations to model contacts in solution. To explore the flexibility and ensemble of interactions between the Suprastat capping group and HDAC6, we carried out molecular dynamics (MD) simulations using GROMACS software (2019). MD simulation results (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A–C) indicate the ability of the hydroxybutyl chain to engage in two hydrogen bond interactions with N530 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A) and S531 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C) at the rim of the catalytic pocket. At the same time, <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B shows the transition state between each interaction. The direct frequency of contact (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D) determined for amino acids within the vicinity of the hydroxybutyl chain and the aminomethyl group indicates that the interactions with N530 and S531 repeatedly occurred during the entire simulation length (100 ns) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Figure S3</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_002.mp4" class="ext-link">Video S1</a>). On the contrary, direct contact between the ligand and D460 was not observed. There is an indirect interaction between the aminomethyl group and D460 mediated by two water molecules. Unlike both bidentate and monodentate hydroxamate–Zn<sup>2+</sup> coordination modes were observed in the crystal complex, Suprastat mainly exhibited the bidentate coordination mode with Zn<sup>2+</sup> during the entire MD simulation process (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A–C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_002.mp4" class="ext-link">Video S1</a>). Overall, these findings suggest that the hydroxybutyl chain can directly engage in hydrogen-bonding interactions with the side chains of N530 and S531, which were not observed in the zHDAC6/Suprastat crystal complex. It should be noted that in the case of the monoclinic crystal form reported here, the capping group of the inhibitor comes to a short distance from a symmetry-related molecule, and both added polar functional groups engage with a symmetry-related HDAC6 molecule via solvent-mediated interactions. The observed crystallographic contacts may thus be responsible for differences in interaction patterns observed for the zHDAC6/Suprastat crystal complex as compared to MD simulations. Given the results from crystallographic studies and MD simulation, we conclude that Suprastat has a high ability to interact with the residues D460, N530, and S531 that play crucial roles in regulating the tubulin deacetylation function of HDAC6.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Representative poses of Suprastat bound to the zHDAC6-CD2 catalytic pocket after molecular dynamics (MD) simulations. (A) Flexible hydroxylbutyl chain engages in H-bonding interaction with N530. (B) Transition state between each stable conformer. (C) H-bonding interaction between Suprastat and S531. (A–C) Aminomethyl group establishes a consistent water-mediated interaction with D460 during the MD simulation process. (D) Direct contact frequency between the hydroxylbutyl chain and the aminomethyl group of Suprastat and the amino acids at the rim of zHDAC6-CD2 along with the MD simulation. Carbon atoms are depicted as gray. Nitrogen atoms are depicted as blue. Oxygen atoms are depicted as red. The Zn<sup>2+</sup> ion is represented as a gray sphere. Water molecules are represented as red spheres. The protein is depicted in gray. Hydrogen-bonding interactions are depicted as green dashed lines. Distances are reported in angstroms (Å). Images were generated in PyMOL 2.1 software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00185" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00185" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">HDAC6 is unique among the histone deacetylases in possessing two functional catalytic domains and a C-terminal ubiquitin-binding domain.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> HDAC6 is predominantly cytosolic due to the presence of a nuclear export signal, and it thereby regulates the acetylation status of numerous cytosolic proteins. Among the many effects of HDAC6, it appears to possess immunomodulatory properties through regulation of the expression of immunosuppressive molecules such as IL-10 and PD-L1, thus leading to enhanced antitumor activity when combined with anti-PD1 immune checkpoint blockade therapy. For these reasons, there is an increasing interest in the identification of potent and highly selective HDAC6 inhibitors. Suprastat was rationally designed based on the crystal structure of nexturastat A in complex with zHDAC6 wherein it was observed that the introduction of key functional groups would likely enhance HDAC6 potency and selectivity over other HDAC isoforms by establishing new hydrogen-bonding interactions with key residues in the HDAC6 pocket in addition to providing hydrophobic interactions between the cap and the rim region. In comparison with nexturastat A and related analogues, Suprastat shows improved HDAC6 activity, excellent isoform selectivity, and an enhanced ability to selectively elevate the levels of acetylated α-tubulin while having a less pronounced effect on histone acetylation in melanoma cells without cytotoxicity. The new polar functional groups on Suprastat decreases its lipophilicity, which leads to a higher lipophilic ligand efficiency and a lower plasma protein binding while maintaining good metabolic stability in different mediums. The <i>in vivo</i> combination therapy studies with an anti-PD1 antibody reveal that Suprastat significantly optimizes the therapeutic outcome through its promoted immunomodulatory properties relative to nexturastat A. Finally, we have utilized X-ray crystallography and molecular dynamics simulations to support our premise that the incorporation of the aminomethyl and hydroxyl groups into the capping group can enlist additional hydrogen-bonding interactions with the key amino acids that regulate the α-tubulin deacetylation function of HDAC6. This work thus offers another example of the rational drug design of an improved HDAC6 inhibitor through the embellishment of hydrogen-bonding interactions between the small molecule and its protein target. This work further underscores the ability of HDAC6 inhibitors to work in combination with anti-PD1 immune checkpoint blockade therapy to decrease tumor growth. To consider Suprastat as a drug candidate, genotoxicity studies (e.g., Ames test) will be further required to determine its mutagenic potential due to the hydroxamate moiety.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Moreover, pharmacokinetic/pharmacodynamic correlation studies of Suprastat will help understand the drug exposure and target engagement <i>in vivo</i> based on its promising antitumor efficacy, which we expect to perform in the next stage.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Information</h4><div class="NLM_p last"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were obtained on 400/101 and 500/126 MHz Bruker spectrometers, respectively, except where noted otherwise, using the solvent residual peak as the internal reference (chemical shifts: CDCl<sub>3</sub>, δ 7.26/77.16 and DMSO-<i>d</i><sub>6</sub>, 2.50/39.52). The following abbreviations for multiplicities were used: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br s = broad singlet. TLC plates (Merck silica gel 60 F<sub>254</sub>, 250 μm in thickness) were used to monitor reaction progress, and spots were visualized under UV (254 nm). High-resolution mass spectrometry (HRMS) was carried out using a Shimadzu IT-TOF instrument under the following conditions: column, ACE 3AQ (50 × 2.1 mm, id) and mobile phase, 5–100% acetonitrile/water containing 0.1% formic acid at a flow rate of 0.5 mL min<sup>–1</sup> for 4 min. Flash chromatography was performed on a CombiFlash Rf system (Teledyne ISCO) with silica gel cartridges. Preparative HPLC was used in the purification of all final compounds using a Shimadzu preparative LC under the following conditions: column, ACE 5AQ (150 × 21.2 mm, id); mobile phase, 5–100% acetonitrile/water containing 0.05% TFA at a flow rate of 17 mL min<sup>–1</sup> for 30 min; and UV detection at 254 and 280 nm. Analytical HPLC was carried out using an Agilent 1260 series instrument under the following conditions: column, ACE 3 (150 × 4.6 mm, id); mobile phase, 5–100% acetonitrile/water containing 0.05% TFA at a flow rate of 1.0 mL min<sup>–1</sup> for 25 min; and UV detection at 254 nm. The purity of all tested compounds (TFA salts) for <i>in vitro</i> biological studies was >95%. The purity of Suprastat (HCl salt) <i>in vivo</i> studies was >98%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Phenyl (4-(((<i>tert</i>-Butoxycarbonyl)amino)methyl)phenyl)carbamate (<b>2</b>)</h4><div class="NLM_p last">To a stirred solution of <i>tert</i>-butyl (4-aminobenzyl)carbamate (<b>1</b>, 500 mg, 2.25 mmol) and K<sub>2</sub>CO<sub>3</sub> (373 mg, 2.70 mmol) in acetone (15 mL) was added phenyl chloroformate (352 mg, 2.25 mmol) over 10 min. After stirring at room temperature for 2 h, the excess solid was filtered off. The filtrate was collected and concentrated under vacuum. The crude product was purified via flash chromatography (0–50% EtOAc/hexane) to afford <b>2</b> as a light yellow solid (700 mg; yield: 91%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.41–7.37 (m, 4H), 7.25–7.16 (m, 6H), 4.86 (s, 1H), 4.27 (d, <i>J</i> = 4.7 Hz, 2H), 1.46 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 156.1, 151.8, 150.7, 136.8, 134.6, 129.5 (2C), 128.4 (2C), 125.8, 121.8 (2C), 119.1 (2C), 79.7, 44.3, 28.5 (3C).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Methyl 4-(((4-Hydroxybutyl)amino)methyl)benzoate (<b>4a</b>)</h4><div class="NLM_p last"><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">(i)</td><td colspan="5" valign="top" class="paragraph"><p class="last">A solution of 4-amino-1-butanol (0.28 mL, 6.10 mmol) and methyl-4-formylbenzoate (<b>3a</b>, 500 mg, 3.05 mmol) in EtOH (25 mL) was heated to reflux for 2 h. After cooling to room temperature, the resulting mixture was concentrated, and the crude product was used directly in the next step.</p></td></tr><tr class="li1"><td valign="top">(ii)</td><td colspan="5" valign="top" class="paragraph"><p class="last">To a stirred solution of the crude product in MeOH (50 mL), sodium borohydride (116 mg, 3.10 mmol) was added over 10 min at 0 °C. The resulting mixture was allowed to warm to room temperature and stirred at room temperature for 2 h. Then, the reaction was quenched with water, and the mixture was extracted with EtOAc (15 mL × 3). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum to afford <b>4a</b> as a colorless oil. The product was used directly in the next step without further purification (630 mg; yield: 87% over two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.98 (d, <i>J</i> = 8.1 Hz, 2H), 7.36 (d, <i>J</i> = 8.0 Hz, 2H), 3.88 (s, 3H), 3.82 (s, 2H), 3.58 (t, <i>J</i> = 5.0 Hz, 2H), 3.07 (br s, 2H, NH + OH), 2.67 (t, <i>J</i> = 5.4 Hz, 2H), 1.71–1.53 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 167.0, 144.6, 130.0 (2C), 129.2, 128.3 (2C), 62.7, 53.6, 52.2, 49.4, 32.2, 28.4.</p></td></tr></table></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Methyl 4-((Butylamino)methyl)benzoate (<b>4b</b>)</h4><div class="NLM_p last"><b>4b</b> was synthesized from <i>n</i>-butylamine (0.31 mL, 6.10 mmol) and methyl-4-formylbenzoate (<b>3a</b>, 500 mg, 3.05 mmol) using a procedure similar to that described for the synthesis of <b>4a</b> and was obtained as a colorless oil (500 mg; yield: 74% over two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.99 (d, <i>J</i> = 8.3 Hz, 2H), 7.39 (d, <i>J</i> = 8.3 Hz, 2H), 3.90 (s, 3H), 3.84 (s, 2H), 2.75–2.51 (m, 2H), 1.53–1.45 (m, 2H), 1.39–1.30 (m, 2H), 0.91 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 166.7, 145.9, 129.5 (2C), 128.5, 127.7 (2C), 53.5, 51.7, 49.0, 32.1, 20.3, 13.8.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-(((4-Hydroxybutyl)amino)methyl)benzonitrile (<b>4c</b>)</h4><div class="NLM_p last"><b>4c</b> was synthesized from 4-amino-1-butanol (0.10 mL, 1.0 mmol) and 4-formylbenzonitrile (<b>3b</b>, 131 mg, 1.0 mmol) using a procedure similar to that described for the synthesis of <b>4a</b>. <b>4c</b> was obtained as a colorless oil (120 mg; yield: 59% over two steps) and used directly to the next step without characterization.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-((4-Bromobenzyl)amino)butan-1-ol (<b>4d</b>)</h4><div class="NLM_p last"><b>4d</b> was synthesized from 4-amino-1-butanol (0.276 mL, 3 mmol) and 4-bromobenzaldehyde (<b>3c</b>, 550 mg, 3.0 mmol) using a procedure similar to that described for the synthesis of <b>4a</b>. <b>4d</b> was obtained as a colorless oil (0.53 g; yield: 68% over two steps) and used directly to the next step without characterization.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Methyl 4-((3-(4-(((<i>tert</i>-Butoxycarbonyl)amino)methyl)phenyl)-1-(4-hydroxybutyl)-ureido)methyl)benzoate (<b>5a</b>)</h4><div class="NLM_p last">To a stirred solution of <b>4a</b> (284 mg, 1.2 mmol) and <b>2</b> (350 mg, 1.0 mmol) in THF (10 mL) was added TEA (0.28 mL, 2.0 mmol). The resulting mixture was heated to reflux for 2 h. Then, the reaction was cooled to room temperature and quenched with water (10 mL), and the mixture was extracted with EtOAc (10 mL × 3). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The residue was purified via flash chromatography (0–50% EtOAc/hexane) to afford <b>5a</b> as a colorless oil (420 mg; yield: 87%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.94 (d, <i>J</i> = 8.1 Hz, 2H), 7.65 (br s, 1H), 7.29 (d, <i>J</i> = 8.1 Hz, 4H), 7.05 (d, <i>J</i> = 8.1 Hz, 2H), 5.10 (br s, 1H), 4.56 (s, 2H), 4.14 (s, 2H), 3.87 (s, 3H), 3.65 (d, <i>J</i> = 4.9 Hz, 2H), 3.53 (br s, 1H, OH), 3.31 (d, <i>J</i> = 6.2 Hz, 2H), 1.64 (s, 2H), 1.53–1.45 (m, 2H), 1.40 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 167.0, 156.2, 156.1, 143.7, 138.7, 133.2, 130.0 (2C), 129.2, 127.8 (2C), 127.3 (2C), 120.3 (2C), 79.5, 62.5, 52.2, 49.9, 46.9, 44.1, 28.4 (3C), 27.6, 25.4.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Methyl 4-((3-(4-(((<i>tert</i>-Butoxycarbonyl)amino)methyl)phenyl)-1-butylureido)-methyl)benzoate (<b>5b</b>)</h4><div class="NLM_p last"><b>5b</b> was synthesized from <b>4b</b> (265 mg, 1.2 mmol) and <b>2</b> (350 mg, 1.0 mmol) using a procedure similar to that described for the synthesis of <b>5a</b> and was obtained as a colorless oil (400 mg; yield: 85%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.98 (d, <i>J</i> = 8.2 Hz, 2H), 7.33 (d, <i>J</i> = 8.1 Hz, 2H), 7.24 (d, <i>J</i> = 8.4 Hz, 2H), 7.12 (d, <i>J</i> = 8.0 Hz, 2H), 6.52 (d, <i>J</i> = 6.6 Hz, 1H), 4.92 (s, 1H), 4.60 (s, 2H), 4.19 (d, <i>J</i> = 5.1 Hz, 2H), 3.89 (s, 3H), 3.40–3.23 (m, 2H, OH), 1.99 (t, <i>J</i> = 10.2 Hz, 1H), 1.59 (dd, <i>J</i> = 14.7, 7.2 Hz, 2H), 1.42 (s, 9H), 1.32 (dd, <i>J</i> = 14.9, 7.4 Hz, 2H), 0.91 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 166.9, 156.0, 155.5, 143.3, 138.2, 133.7, 130.2 (2C), 129.4, 128.1 (2C), 127.1, 120.3 (2C), 79.4, 52.2, 50.4, 47.7, 44.2, 30.5, 28.5 (3C), 20.2, 13.9.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Methyl <i>tert</i>-Butyl (4-(3-(4-Cyanobenzyl)-3-(4-hydroxybutyl)ureido)benzyl)carbamate (<b>5c</b>)</h4><div class="NLM_p last"><b>5c</b> was synthesized from <b>4c</b> (120 mg, 0.58 mmol) and <b>2</b> (200 mg, 0.58 mmol) using a procedure similar to that described for the synthesis of <b>5a</b> and was obtained as a colorless oil (80 mg; yield: 30%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.80 (s, 1H), 7.57 (d, <i>J</i> = 7.9 Hz, 2H), 7.43–7.29 (m, 4H), 7.07 (d, <i>J</i> = 7.9 Hz, 2H), 5.01 (s, 1H), 4.58 (s, 2H), 4.16 (d, <i>J</i> = 4.5 Hz, 2H), 3.72 (s, 2H), 3.35–3.32 (m, 2H), 3.27 (br s, 1H), 1.69 (s, 2H), 1.54 (s, 2H), 1.42 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 156.0, 155.9, 144.1, 138.6, 133.1, 132.3 (2C), 128.0 (2C), 127.7 (2C), 120.1 (2C), 118.7, 110.9, 79.4, 62.6, 49.8, 46.8, 44.1, 28.3 (3C), 27.0, 25.5.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>tert</i>-Butyl (4-(3-(4-Bromobenzyl)-3-(4-hydroxybutyl)ureido)benzyl)carbamate (<b>5d</b>)</h4><div class="NLM_p last"><b>5d</b> was synthesized from <b>4d</b> (520 mg, 2 mmol) and <b>2</b> (680 mg, 2 mmol) using a procedure similar to that described for the synthesis of <b>5a</b> and was obtained as a colorless oil (960 mg, 95%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.35 (s, 1H), 7.52 (d, <i>J</i> = 8.3 Hz, 2H), 7.37 (d, <i>J</i> = 8.1 Hz, 2H), 7.29 (br t, 1H), 7.22 (d, <i>J</i> = 8.3 Hz, 2H), 7.08 (d, <i>J</i> = 8.1 Hz, 2H), 4.52 (s, 1H), 4.48 (br t, 2H), 4.03 (d, <i>J</i> = 5.6 Hz, 2H), 3.39–3.37 (m, 2H), 3.29–3.27 (m, 2H), 1.58–1.46 (m, 2H), 1.45–1.35 (m, 2H), 1.36 (s, 9H).</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-((3-(4-(Aminomethyl)phenyl)-1-(4-hydroxybutyl)ureido)methyl)-<i>N</i>-hydroxy-benzamide (<b>6a</b>)</h4><div class="NLM_p last"><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">(i)</td><td colspan="5" valign="top" class="paragraph"><p class="last">In a round-bottom flask, NaOH (68 mg, 1.69 mmol) was dissolved in 50% aqueous NH<sub>2</sub>OH (0.7 mL, approximately 50 equiv) at 0 °C. A solution of <b>5a</b> (200 mg, 0.43 mmol) in 1/1 THF/MeOH (2/2 mL) was added dropwise, and stirring was continued for 30 min while warming to room temperature. The solution was neutralized with 2 N HCl and extracted with EtOAc (10 mL × 3). The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude product was used directly in the next step without further purification.</p></td></tr><tr class="li1"><td valign="top">(ii)</td><td colspan="5" valign="top" class="paragraph"><p class="last">The crude intermediate was dissolved in THF (2 mL) followed by the addition of TFA (3 mL) at room temperature. The resulting mixture was stirred at room temperature for 0.5 h, and then the excess solvent was removed under vacuum. The crude product was purified via preparative HPLC and lyophilized to afford <b>6a</b> as a white powder (62 mg; TFA salt; purity: 95%; yield: 29% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.17 (br s, 1H), 8.51 (s, 1H), 8.07 (br s, 3H, NH<sub>3</sub><sup>+</sup>), 7.72 (d, <i>J</i> = 8.7 Hz, 2H), 7.50 (d, <i>J</i> = 8.7 Hz, 2H), 7.32 (d, <i>J</i> = 6.6 Hz, 2H), 7.30 (d, <i>J</i> = 7.0 Hz, 2H), 4.61 (s, 2H), 3.94 (q, <i>J</i> = 5.8 Hz, 3H), 3.39 (t, <i>J</i> = 6.4 Hz, 2H), 3.33 (t, <i>J</i> = 7.4 Hz, 2H), 1.54 (ddd, <i>J</i> = 9.2, 6.4, 2.5 Hz, 2H), 1.41 (dt, <i>J</i> = 8.7, 6.5 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.0, 155.1, 142.2, 140.8, 131.4, 129.1 (2C), 127.01 (2C), 126.97 (2C), 126.9, 119.7 (2C), 60.6, 49.0, 46.3, 42.0, 29.4, 24.6. ESI HRMS calcd for C<sub>20</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>: [M – H]<sup>−</sup>, <i>m</i>/<i>z</i> 385.1881; found, 385.1871.</p></td></tr></table></div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-((3-(4-(Aminomethyl)phenyl)-1-butylureido)methyl)-<i>N</i>-hydroxybenzamide (<b>6b</b>)</h4><div class="NLM_p last"><b>6b</b> was synthesized from <b>5b</b> (420 mg, 0.87 mmol) using a procedure similar to that described for the synthesis of <b>6a</b> and was obtained as a white solid (160 mg; TFA salt; purity: >99%; yield: 38% over two steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.17 (br s, 1H), 9.00 (br s, 1H), 8.48 (s, 1H), 8.01 (br s, 3H, NH<sub>3</sub><sup>+</sup>), 7.72 (d, <i>J</i> = 8.2 Hz, 2H), 7.50 (d, <i>J</i> = 8.6 Hz, 2H), 7.32 (d, <i>J</i> = 6.4 Hz, 2H), 7.30 (d, <i>J</i> = 6.8 Hz, 2H), 4.61 (s, 2H), 3.93 (t, <i>J</i> = 5.6 Hz, 2H), 3.30–3.26 (m, 2H, overlapping with a water peak), 1.53–1.40 (m, 2H), 1.31–1.16 (m, 2H), 0.86 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.0, 155.1, 142.2, 140.8, 131.4, 129.1 (2C), 127.0, 126.9 (4C), 119.8 (2C), 49.0, 46.2, 42.0, 29.9, 19.4, 13.8. ESI HRMS calcd for C<sub>20</sub>H<sub>25</sub>N<sub>4</sub>O<sub>3</sub>: [M – H]<sup>−</sup>, <i>m</i>/<i>z</i> 369.1932; found, 369.1923.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-((3-(4-(Aminomethyl)phenyl)-1-(4-hydroxybutyl)ureido)methyl)benzoic Acid (<b>6d</b>)</h4><div class="NLM_p last">To a stirred solution of <b>5a</b> (62 mg, 0.13 mmol) in THF/MeOH (1/1 mL) was added 1 N NaOH solution (1 mL). Then, the resulting mixture was stirred at room temperature overnight. After the completion of the reaction detected by HPLC, the reaction was acidified by 2 N HCl and extracted with THF (15 mL × 3). The combined organic extracts were washed with H<sub>2</sub>O and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude product was dissolved in THF (2 mL) followed by the addition of TFA (3 mL) at room temperature. The resulting mixture was stirred at room temperature for additional 0.5 h, and then the excess solvent was removed under vacuum. The crude product was purified via preparative HPLC and lyophilized to afford <b>6d</b> as a white powder (40 mg; TFA salt; purity: 95%; yield: 63% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.92 (br s, 1 H), 8.53 (s, 1H), 8.07 (br s, 3H, NH<sub>3</sub><sup>+</sup>), 7.91 (d, <i>J</i> = 8.3 Hz, 2H), 7.51 (d, <i>J</i> = 8.6 Hz, 2H), 7.37 (d, <i>J</i> = 8.2 Hz, 2H), 7.30 (d, <i>J</i> = 8.7 Hz, 2H), 4.65 (s, 2H), 3.94 (q, <i>J</i> = 5.6 Hz, 2H), 3.39 (t, <i>J</i> = 6.4 Hz, 2H), 3.34 (t, <i>J</i> = 7.5 Hz, 2H), 1.54 (tt, <i>J</i> = 8.2, 6.3 Hz, 2H), 1.45–1.32 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.2, 155.2, 144.2, 140.8, 129.5 (2C), 129.4, 129.1 (2C), 127.1 (2C), 127.0, 119.8 (2C), 60.6, 49.1, 46.4, 42.0, 29.4, 24.6. ESI HRMS calcd for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>4</sub>: [M – H]<sup>−</sup>, <i>m</i>/<i>z</i> 370.1772; found, 370.1769.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-((3-(4-(Aminomethyl)phenyl)-1-(4-hydroxybutyl)ureido)methyl)benzamide (<b>6e</b>)</h4><div class="NLM_p last">To a stirred solution of <b>5c</b> (80 mg, 0.18 mmol) in DMSO (2 mL) were added K<sub>2</sub>CO<sub>3</sub> (2.4 mg, 0.018 mmol) and H<sub>2</sub>O<sub>2</sub> (30%, 0.2 mL) at room temperature. The resulting mixture was stirred for 5 h. After completion of the reaction, the solution was quenched with water (5 mL) and extracted with EtOAc (10 mL × 3). The organic layers were separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude product was dissolved in THF (2 mL) followed by the addition of TFA (3 mL) at room temperature. The resulting mixture was stirred at room temperature overnight, and then the excess solvent was removed under vacuum. The crude product was purified via preparative HPLC and lyophilized to afford <b>6e</b> as a white powder (30 mg; TFA salt; purity: 98%; yield: 35% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.51 (s, 1H), 8.06 (br s, 3H, NH<sub>3</sub><sup>+</sup>), 7.92 (br s, 1H), 7.84 (d, <i>J</i> = 8.4 Hz, 2H), 7.51 (d, <i>J</i> = 8.6 Hz, 2H), 7.32–7.29 (m, 5H, one proton of CONH<sub>2</sub> overlaps with the signals from the phenyl ring), 4.62 (s, 2H), 3.94 (q, <i>J</i> = 5.8 Hz, 3H), 3.39 (t, <i>J</i> = 6.4 Hz, 2H), 3.33 (t, <i>J</i> = 7.5 Hz, 2H), 1.54 (ddd, <i>J</i> = 12.1, 8.8, 6.4 Hz, 2H), 1.45–1.34 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.6, 155.1, 142.3, 140.9, 133.0, 129.1 (2C), 129.0, 127.6 (2C), 126.8 (2C), 126.7, 119.7 (2C), 60.6, 49.0, 42.0, 29.4, 24.6. ESI HRMS calcd for C<sub>20</sub>H<sub>27</sub>N<sub>4</sub>O<sub>3</sub>: [M + H]<sup>+</sup>, <i>m</i>/<i>z</i> 371.2078; found, 371.2087.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (4-((3-(4-(Aminomethyl)phenyl)-1-(4-hydroxybutyl)ureido)methyl)phenyl)boronic Acid (<b>6f</b>)</h4><div class="NLM_p last"><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">(i)</td><td colspan="5" valign="top" class="paragraph"><p class="last">To a stirred solution of <b>5d</b> (510 mg, 1.0 mmol), bis(pinacolato)diboron (280 mg, 1.1 mmol), and potassium acetate (290 mg, 3 mmol) in DMF (5 mL) was added Pd(dppf)Cl<sub>2</sub> at room temperature under an argon atmosphere. After stirring at 80 °C overnight, the reaction was cooled to room temperature, and the excess solid was filtered off. The filtrate was collected, quenched with saturated NaHCO<sub>3</sub> aqueous solution (20 mL), and extracted with EtOAc (20 mL × 3). The combined organic layers were washed with 10% LiCl aqueous solution and brine (30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude was purified via flash chromatography (0–50% EtOAc/hexane) to afford the pinacol ester intermediate as a brown solid (0.25 g; yield: 44%).</p></td></tr><tr class="li1"><td valign="top">(ii)</td><td colspan="5" valign="top" class="paragraph"><p class="last">To a solution of the pinacol ester intermediate (240 mg, 0.44 mmol) in acetone/water (2/1, 9 mL) were added NaIO<sub>4</sub> (280 mg, 1.32 mmol) and NH<sub>4</sub>OAc (100 mg, 1.32 mmol) at room temperature. After stirring at room temperature overnight, the reaction mixture was quenched with water and extracted with EtOAc (20 mL × 3). The combined organic layers were washed with brine (30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude product was used directly to the next step without further purification.</p></td></tr><tr class="li1"><td valign="top">(iii)</td><td colspan="5" valign="top" class="paragraph"><p class="last">The crude product was dissolved in THF (1 mL) followed by the addition of TFA (1 mL) at room temperature. The resulting mixture was stirred at room temperature for 0.5 h, and then the excess solvent was removed under vacuum. The crude product was purified via preparative HPLC and lyophilized to afford <b>6f</b> as a white powder (190 mg; TFA salt; purity: 99%; yield: 39% over three steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.49 (s, 1H), 8.06 (br s, 3H, NH<sub>3</sub><sup>+</sup>), 7.74 (d, <i>J</i> = 7.5 Hz, 2H), 7.50 (d, <i>J</i> = 8.2 Hz, 2H), 7.30 (d, <i>J</i> = 8.2 Hz, 2H), 7.21 (d, <i>J</i> = 7.6 Hz, 2H), 4.57 (s, 2H), 3.95 (s, 2H), 3.48 (t, <i>J</i> = 5.7 Hz, 2H), 3.31 (t, <i>J</i> = 6.7 Hz, 2H), 1.61–1.57 (m, 2H), 1.46–1.43 (m, 2H). ESI HRMS calcd for C<sub>19</sub>H<sub>27</sub>BN<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup>: <i>m</i>/<i>z</i> 372.2089; found, 372.2098.</p></td></tr></table></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Molecular Docking Studies</h3><div class="NLM_p last">Docking models of the ligand bound to HDAC6 were developed using the Molecular Operating Environment (MOE) computational suite’s Builder utility. The energy minimization of ligands was conducted in the gas phase using the force field MMFF94X followed by the conformational search protocol to generate structural conformation databases. The zHDAC6-NextA crystal structure (PDB entry: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>; resolution: 1.99 Å) used as a template was obtained from the Protein Data Bank. The receptor preparation step was initiated with the removal of solvent except a single water molecule (HOH2316) close to the Zn<sup>2+</sup> ion that was kept for modeling studies. Hydrogens were then placed, while ionization states were assigned throughout the system. Finally, ligands and binding sites were isolated in 3D, and then a molecular surface was drawn around the binding site to visualize the space available for docked ligands. Ligand placement employed the alpha triangle method with Affinity dG scoring generating 300 data points that were further refined using the induced fit method with GBVI/WSA dG scoring to obtain the top 50 docking results. The docking result of each ligand was analyzed for selection of the best docking pose, based on the score and reported X-ray structures. All renderings were then performed in PyMOL.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Ligand–Protein Binding Free Energy Calculations</h3><div class="NLM_p last">Ligand–protein binding free energy calculations were carried out in the MOPAC 2016 package (Stewart Computational Chemistry). The free energy calculations were performed for the empty HDAC6-CD2 catalytic pocket (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>), HDAC6-CD2 bound to nexturastat A (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>), and the HDAC6-CD2/Suprastat complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY">6TCY</a>). Free energy calculations for both isolated ligands (nexturastat A and Suprastat) were also determined. The molecular geometries of each structure were optimized by the PM7 semiempirical method. Water molecules, zinc ions, and other ions were not considered for the calculation. A combination of molecular mechanics energies, polar and nonpolar solvation energies, and entropy were considered for the resulting binding free energy. The binding free energy calculations were performed considering both the ligand and protein in aqueous solution using a dielectric constant (ε) of 78.4. The enthalpy (Δ<i>H</i><sup>0</sup>) and entropy (Δ<i>S</i><sup>0</sup>) contribute to the Δ<i>G</i><sup>0</sup> value of NextA and Suprastat, also determined by the Gibbs free energy equation (Δ<i>G</i><sup>0</sup> = Δ<i>H</i><sup>0</sup> – <i>T</i>Δ<i>S</i><sup>0</sup>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Molecular Dynamics Simulation</h3><div class="NLM_p last">Molecular dynamics simulation was performed in GROMACS 2019<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> (fast, flexible, and free) for the HDAC6-CD2/Suprastat complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY">6TCY</a>). The simulation was carried out using the CHARMM force field.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> The simulated system comprised the protein–ligand complex, the Zn<sup>2+</sup> ion, a predefined water model (SPC)<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> as solvent, and counterions (Na<sup>+</sup> or Cl<sup>–</sup> set to neutralize the overall system charge). The system was treated in a triclinic box with periodic boundary conditions specifying the shape and the size of the box as 10 Å distance from the box edges to any atom of the protein. We used a time step of 1 fs, and the short-range Coulombic interactions were treated using a cutoff value of 8.0 Å by the short-range method. The long-range Coulombic interactions were handled via the smooth particle mesh Ewald method (PME).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Initially, the relaxation of the system was performed using both steepest descent and limited-memory Broyden–Fletcher–Goldfarb–Shanno algorithms in a hybrid manner. The simulations were performed under the isothermal–isobaric ensemble (NPT) for 5 ns implementing the Berendsen thermostat and barostat methods. The temperature was maintained at 310 K throughout the simulation using the Nosé–Hoover thermostat algorithm. The Martyna–Tobias–Klein barostat algorithm was employed to maintain 1 atm of pressure during calculations. After minimization and relaxation of the system, we proceeded with a single production step of 100 ns. The representative structure of Suprastat was selected by clustering the structures from the root-mean-square deviation (RSMD) values, using 0.5 Å as a cutoff. <a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Figure S2</a> (Supporting Information) represents the variation of the RMSD values along with the simulation and the root-mean-square fluctuation (RMSF) for the protein backbone. Interactions and distances were determined using the trajectory analysis of GROMACS. The current geometric criteria for protein–ligand H bonds are as follows: (i) a distance of 3.5 Å between donor (HBD) and acceptor (HBA) atoms, (ii) an angle of ≥120° for the HBD between donor–hydrogen–acceptor atoms (D–H···A), and (iii) an angle of ≥90° for the HBA between hydrogen–acceptor–bonded atoms (H···A–X). Similarly, protein–water or water–ligand H bonds have (iv) a distance of 3.0 Å between HBD and HBA atoms, (v) a donor angle of ≥110° between donor–hydrogen–acceptor atoms, and (vi) an acceptor angle of ≥90° between hydrogen–acceptor–bonded atoms. Nonspecific hydrophobic interactions are defined by hydrophobic side chains within 4.5 Å of aromatic or aliphatic carbons, and π–π interactions required two aromatic groups stacked face-to-face or face-to-edge, within 5.0 Å of distance.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Expression and Purification of Human HDAC1–HDAC9 and HDAC11</h3><div class="NLM_p last">Large-scale expression of human HDACs was carried out in HEK 293/T17 cells essentially as described previously.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> Briefly, transiently transfected cells were harvested 3 days post-transfection and the cell pellets were resuspended in a lysis buffer (50 mM Tris, 150 mM NaCl, 10 mM KCl, 2 mM MgCl<sub>2</sub>, 10% glycerol, 0.2% NP-40, and 2 Units mL<sup>–1</sup> benzonase at pH 8) supplemented with a cocktail of protease inhibitors (Roche, Basel, Switzerland). Cells were lysed by sonication (30 W; 3 × 20 s) on ice, and the cell lysate was cleared by centrifugation at 40,000<i>g</i> for 30 min at 4 °C. Recombinant fusion HDAC proteins were purified via Strep-Tactin affinity chromatography (IBA, Göttingen, Germany) with the elution buffer comprising 50 mM HEPES, 100 mM NaCl, 50 mM KCl, 10% glycerol, and 3 mM desthiobiotin at pH 7.5. Purified proteins were concentrated to 1 mg mL<sup>–1</sup>, aliquoted, flash-frozen in liquid nitrogen, and stored at −80 °C until further use.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Determination of Inhibitory Activity against Human HDAC1–HDAC9 and HDAC11</h3><div class="NLM_p last">IC<sub>50</sub> values in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> were determined using a fluorescence-based assay with 10 μM Ac-GAK(Ac)-AMC (HDAC1–HDAC3 and HDAC6) or 10 μM Boc-Lys(TFA)-AMC (HDAC4, HDAC5, HDAC7–HDAC9, and HDAC11) as a substrate.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Briefly, individual HDACs were preincubated with dilution series of tested inhibitors in a 384-well plate in the total volume of 40 μL for 10 min at 37 °C in a reaction buffer comprising 50 mM HEPES, 140 mM NaCl, 10 mM KCl, 1 mM TCEP, and 0.1% BSA at pH 7.4. Deacetylation reactions were started by the addition of 10 μL of a 10 μM substrate into the HDAC/inhibitor mixture. Following 30 min incubation at 37 °C, the reaction was terminated by the addition of 25 μL of trypsin solution (4 mg mL<sup>–1</sup>). Fluorescence development by trypsin was carried out at 37 °C for 15 and 60 min for the Ac-GAK(Ac)-AMC and Boc-Lys(TFA)-AMC substrate, respectively. Released aminomethyl coumarin was quantified using a CLARIOstar fluorimeter with the excitation and emission wavelengths set to 365 and 440 nm, respectively. Nonlinear regression analysis was employed to calculate IC<sub>50</sub> values using GraphPad Prism software. Fourteen-point IC<sub>50</sub> curves were generated using a three-fold inhibitor dilution series; inhibitor concentration ranges used are as follows: 100 μM to 63 pM for HDAC1–HDAC5, HDAC7–HDAC9, and HDAC11 and 3 μM to 1.88 pM for HDAC6. Reactions without the enzyme or the inhibitor were used to define 0 and 100% of the HDAC activity, respectively.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> zHDAC6 Expression and Purification</h3><div class="NLM_p last">The second catalytic domain of HDAC6 from <i>D. rerio</i> (zHDAC6; amino acids 440–798) was expressed and purified essentially as described previously.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Briefly, the synthetic gene encoding HDAC6 was recombined into a Gateway expression plasmid in frame with the TEV-cleavable His-MBP N-terminal tag. The fusion protein was expressed in <i>Escherichia coli</i> BL21-CodonPlus (DE3)-RIPL at 16 °C overnight. The purification protocol comprised the HisTrap HP affinity step (GE Healthcare, Chicago, IL, USA), removal of the tag by the TEV protease, affinity purification on amylose resin (New England Biolabs, Ipswitch, MA, USA), ion-exchange chromatography on HiTrap Q sepharose (GE Healthcare), and size-exclusion chromatography on a HiLoad Superdex 75 pg column (GE Healthcare; mobile phase: 50 mM HEPES, 100 mM KCl, 5% glycerol, and 1 mM TCEP at pH 7.5) as the final step. The purity of the final protein preparation was >98% as determined by SDS-PAGE, and purified zHDAC6-CD2 was concentrated to 10 mg mL<sup>–1</sup>, aliquoted, flash-frozen in liquid nitrogen, and stored at −80 °C until further use. The IC<sub>50</sub> value of NextA against zHDAC6 was determined to be 2.6 nM, which is comparable to its inhibitory affinity for hHDAC6 (IC<sub>50</sub> = 1.6 nM) shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Crystallization and Data Collection</h3><div class="NLM_p last">The zHDAC6 stock solution was mixed with 1/20 volume of the Suprastat solution (80 mM in DMSO), and the crystallization droplets were prepared by combining 1 μL of the complex solution with 1 μL of a reservoir solution containing 19% PEG 3350 (Sigma-Aldrich), 0.2 M KSCN (Hampton Research), and 0.1 M bis-Tris (Sigma-Aldrich) at pH 6.5. To bolster the nucleation step, droplets were streak-seeded using the seed stock prepared from crystals of the HDAC6/SAHA complex using a crystal crusher (Hampton Research). Crystals were grown by the hanging drop vapor diffusion method at 283 K. Diffraction quality crystals were vitrified in liquid nitrogen from the mother liquor supplemented with 20% (v/v) glycerol. Data collection was carried out at 100 K on the beamline 14.2 at the BESSY II synchrotron radiation source, Helmholtz-Zentrum Berlin, using a Pilatus3 2M detector.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The data were processed using DIALS,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> including scaling using dials.scale<i>.</i> The data quality indicators were calculated using AIMLESS and AUSPEX;<a onclick="showRef(event, 'ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref59 ref60">(59,60)</a> the data statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Table S2</a>. The structure was solved by molecular replacement using Phaser<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> in the CCP4 suite<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> with zHDAC6 structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEK">5EEK</a><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>; chain A) as a search model. The procedure resulted in a solution with an <i>R</i> of 36.6 and LLG of 22,589. Structure refinement involved iterative cycles of the model building using Coot<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> according to 2<i>mF</i><sub>o</sub>–<i>DF</i><sub>c</sub> and 2<i>mF</i><sub>o</sub>–<i>DF</i><sub>c</sub> Fourier maps and restrained refinement using REFMAC5.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a><i>R</i><sub>free</sub> was used as a cross-validation statistic. The presence of the ligand was confirmed using the polder map.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The structure was validated using the MolProbity program suite<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and a set of validation tools in Coot.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Metabolic Stability in Body Fluids and Rat Microsomes</h3><div class="NLM_p">Compounds were diluted to the initial concentration of 10 μM in the tested solution—PBS, simulated gastric fluid (SGF; 0.2% NaCl, 0.7% HCl, pH 1.2), or pooled human plasma (Innovative Research, Novi, MI, USA), all prewarmed to 37 °C. Samples were incubated at 37 °C, aliquots were aspirated at defined time points, and samples were processed and analyzed as described below.</div><div class="NLM_p">Rat liver microsomes were prepared according to a published protocol.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> The final microsomal preparation was diluted to 10 mg mL<sup>–1</sup> total protein content in 50 mM Tris (pH 7.5) supplemented with 10 mM EDTA and 20%, snap-frozen in liquid nitrogen, and stored at −80 °C until further use. The metabolic activity of our in-house rat liver microsomes was virtually identical to commercially available preparations (rat liver microsomes, RTMCPL, Life Technologies, CA, USA; <a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p last">Microsomes were diluted to the final concentration of 0.5 mg mL<sup>–1</sup> (total protein) in 0.1 M potassium phosphate buffer (pH 7.4) and preincubated for 10 min at 37 °C. Upon the addition of MgCl<sub>2</sub> (final concentration: 1.25 mM), the tested compounds were added to the final concentration of 10 μM and the reaction started by the addition of NADPH solution (final concentration: 1.8 mM). Aliquots were collected at defined time points, and samples were processed and analyzed as described below.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Plasma Protein Binding</h3><div class="NLM_p">An ultrafiltration (UF) protocol was used to determine compound binding to plasma proteins.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Tested compounds were diluted to 1 mM in water, and this stock solution was further diluted to the final concentration of 10 μM in PBS (nonspecific binding control) or pooled human plasma. The tested solution (200 μL) was transferred to a centrifugal ultrafiltration device (Merck Millipore, Burlington, MA, USA) and incubated at room temperature for 1 h. The sample (50 μL) was removed from the upper chamber (the input sample), and the assembled UF unit was centrifuged in a fixed angle rotor at 1000<i>g</i> at room temperature for 5 min. The filtrate (50 μL) was mixed with the equivalent volume of either pooled human plasma (nonspecific binding control samples) or PBS (plasma samples) to account for matrix effects.</div><div class="NLM_p">The plasma binding was calculated according to the following formula:</div><div class="NLM_p">nonspecific binding (NSB):<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_m001.gif" alt="" id="_i43" /></img></span></div><div class="NLM_p">free fraction (ff):<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_m002.gif" alt="" id="_i44" /></img></span></div><div class="NLM_p last">plasma bound (PB):<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_m003.gif" alt="" id="_i45" /></img></span>where <i>c</i><sub>B_in</sub> is the input concentration of the compound in PBS, <i>c</i><sub>B_fil</sub> is the filtrate concentration from PBS samples, <i>c</i><sub>P_in</sub> is the input concentration in plasma samples, and <i>c</i><sub>P_fil</sub> is the concentration in the filtrate from plasma samples.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Sample Processing and LC/MS-Based Quantification</h3><div class="NLM_p">Sample aliquots (stability or binding assays) were immediately mixed with three volumes of ice-cold acetonitrile containing losartan as an internal standard (13.3 ng mL<sup>–1</sup>), vortexed for 1 min, and centrifuged at 13,000<i>g</i> for 15 min. The supernatant was diluted with Milli-Q water 1/1, and compounds were quantified by LC/MS–MS.</div><div class="NLM_p last">Mass spectrometry quantification was carried out using an MS-coupled HPLC system (Shimadzu LCMS-8040, Shimadzu, Kyoto, Japan) equipped with an API electrospray ion source. Analytes were separated on a Luna Omega Polar C18 1.6 μm, 100 Å column, 50 × 2.1 mm (Phenomenex, Torrance, CA, USA) using a gradient of 5–70% CH<sub>3</sub>CN/H<sub>2</sub>O containing 0.1% formic acid at a flow rate 0.6 mL min<sup>–1</sup> over 4 min. Multiple reaction monitoring parameters were optimized for each analyte together with losartan as an internal standard, and the predominant CID fragment was used for quantification. Data were analyzed using GraphPad Prism software (GraphPad Software, San Diego, CA, USA) where the ratio of the analyte signal over the losartan signal was plotted against the incubation time, and data were fitted with an exponential one-phase decay equation to obtain a half-life of a tested compound.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Cell Culture and Antibodies</h3><div class="NLM_p last">The SM1 murine melanoma cells were obtained from Dr. A. Ribas at the University of California, Los Angeles. WM164 human melanoma cells were obtained from ATCC. The cells were cultured in an incubator in RPMI 1640, 1% penicillin–streptomycin, and 10% fetal bovine serum at 37 °C with 5% CO<sub>2</sub>. HDAC inhibitors including <b>6a</b>–<b>6c</b> and NextA were added at concentrations of 0.1, 0.5, 1, 2.5, 5, and 10 μM, and the incubation was conducted overnight. For STAT3 phosphorylation assays, cells were pretreated with or without HDAC6i (5 μM) overnight followed by treatment with recombinant human IL-6 (BioLegend) for 20 min. PBS was added as a control. RAW 264.7 macrophages were purchased from ATCC and cultured in DMEM supplemented with 10% FBS, 1% nonessential amino acids, and 2-mercaptoethanol (50 μM). RAW macrophages were treated with 1, 5, and 10 μM Suprastat overnight before collecting the lysates for the immunoblot assay. The cytotoxicity assay was performed using CellTox Green (Promega, Cat#G8731) following the manufacturer’s instructions, and fluorescence readings were obtained on a SpectraMax i3 (Molecular Devices) multimode plate reader at wavelengths EX 485 nm and EM 520 nm. SM1 cells were treated with compounds at various concentrations for 24 h to determine cytotoxicity.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Immunoblot Analysis</h3><div class="NLM_p last">The cells were harvested and lysed with RIPA buffer (Thermo Scientific, 89900) containing protease and phosphatase inhibitors (Thermo Scientific, 78440) by sonication in a Bioruptor (Diagenode) for 8 cycles of 30 s ON and 30 s OFF on high setting. To assess the expression of proteins, total protein samples were heat-denatured in SDS sample loading buffer, and 15–20 μg of protein was analyzed on 4–20% SDS-PAGE gels (Bio-Rad, 456-1093). Proteins were transferred onto low fluorescence PVDF membranes (Bio-Rad, 1704274) using a Trans-Blot Turbo transfer system (Bio-Rad). The membranes were blocked for 1 h with Odyssey blocking buffer (LI-COR, 927-40000) followed by incubation with primary antibodies (1/1000 dilution) at 4 °C. The membranes were washed in PBST buffer three times followed by incubation with near-infrared fluorophore-conjugated secondary antibodies (1/10,000 dilution) for 1 h at room temperature. The membranes were scanned on an Azure Biosystems c600 imager at near-infrared wavelengths. The images were analyzed and processed with Image Studio Lite software. The primary antibodies used are against phospho-STAT3 Y705 (Cell Signaling, 9145), α-tubulin (Cell Signaling, 3873), acetyl-α-tubulin (Cell Signaling, 3971), histone H3 (Cell Signaling, 3638), and acetyl-histone H3 (Cell Signaling, 9649).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Quantitative Analysis of Gene Expression</h3><div class="NLM_p last">Total RNA was isolated from cells following the manufacturer’s instructions of QIAzol lysis reagent (Qiagen, 79306). RNA quantification was done using a NanoDrop One spectrophotometer (NanoDrop Technologies). Samples with absorbance at 260/280 nm ratios over 1.9 were used for cDNA synthesis with the iScript cDNA synthesis kit (Bio-Rad, 1708891). Synthesized cDNA from 1 μg of total RNA was diluted 1/10 with nuclease-free water. The quantitative PCR analysis was performed using iQ SYBR Green Supermix (Bio-Rad, 1708882) on a CFX96 real-time system (Bio-Rad). Gene expression analysis was performed using the 2<sup>–ΔΔ<i>Ct</i></sup> method, and target mRNA levels were normalized to <i>ACTB</i> expression. Cycling conditions were used as per the manufacturer’s instructions. Single PCR product amplification was confirmed by melting curve analysis in all the experiments performed. The mouse <i>IL10</i> QuantiTect primer assay (Qiagen, QT00106169) was purchased from Qiagen. The sequence of primers used in the analysis is as follows: mouse ACTB forward: CATTGCTGACAGGATGCAGAAGG and reverse: TGCTGGAAGGTGGACAGTGAGG, which were synthesized from Invitrogen.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Mice</h3><div class="NLM_p">Animal experiments involving mice were performed in accordance with the protocol (#A354) approved by the Institutional Care and Use Committee (IACUC) at The George Washington University. Forty C57BL/6 female mice were purchased from the Charles River Laboratories (Wilmington, Massachusetts, USA). <i>In vivo</i> studies were performed using SM1 tumor cells that were passaged <i>in vivo</i> from mouse to mouse for a minimum of five times before tumor implantation. Mice were injected subcutaneously with 1.0 × 10<sup>6</sup><i>in viv</i>o passaged SM1 melanoma cells suspended in 100 μL of phosphate-buffered saline (PBS) (Corning, 21-040-CV). The pretreatment arm was started once the tumors were palpable, which was about 5 days post-tumor implantation. Cages were randomly assigned to different treatment groups, and mice were treated with Suprastat, anti-PD1 antibody, or vehicle control. Control mice were injected intraperitoneally with 100 μL of PBS as vehicle control, a 15 mg kg<sup>–1</sup> dose of anti-PD1 antibody (Bio X Cell, Clone RMP1-14), and 25 mg kg<sup>–1</sup> Suprastat. Mice were treated 5 days a week until tumors in the control group reached maximum size according to our IACUC protocol. Tumor volume measurements were taken on alternate days using caliper measurements and calculated using the formula <i>L</i> × <i>W</i><sup>2</sup>/2. All animal studies were performed with consideration for toxicity, and we routinely monitored for early signs of toxicity. Emphasis was given to mortality, body weight, and food consumption. At the endpoint, a post-mortem evaluation, including gross visual examination of organs such as the liver for hepatotoxicity, splenomegaly, and lung metastatic nodules, was done for each condition.</div><div class="NLM_p last">For macrophage isolation, bone marrow from 6 to 12 weeks old C57BL/6 mouse was used following an IACUC approved protocol. Briefly, femurs and tibia bones were isolated after removing the skeletal muscles. The bone marrow was flushed with RPMI complete medium supplemented with nonessential amino acids. A single-cell suspension of bone marrow was prepared with repeated pipetting and incubated with 20 ng mL<sup>–1</sup> mouse recombinant M-CSF (BioLegend) at 37 °C for 4 days to differentiate into macrophages. On day 3, macrophages were pretreated with 5 μM Suprastat or vehicle. On day 4, macrophages were treated with 5 μM Suprastat 1 h prior to polarization to M1 phenotype with 100 ng mL<sup>–1</sup> LPS (Sigma) and 20 ng mL<sup>–1</sup> recombinant mouse interferon-gamma (BioLegend).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Flow Cytometry</h3><div class="NLM_p last">Flow cytometry was performed following the protocol described previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Briefly, mice were euthanized following the IACUC protocol, and tumor cells were processed into a single-cell suspension for analysis by flow cytometry with tumor digestion buffer. The following antibodies were used to stain cell surface markers expressed by different immune cells. All the antibodies were purchased from BioLegend (San Diego, CA) unless otherwise specified. Myeloid cell surface markers are as follows: Brilliant Violet 421 antimouse/human CD11b (clone M1/70), APC antimouse CD80 (clone 16-10A1), PE/Cy7 antimouse CD206 (MMR) (clone C068C2), Brilliant Violet 650 antimouse CD11c (clone N418), APC/Fire 750 antimouse CD45.2 (clone 104), PE antimouse CD123 (clone 5B11), Brilliant Violet 605 antimouse Ly-6G/Ly-6C (Gr-1) (clone RB6-8C5), FITC antimouse H-2 (clone M1/42), Brilliant Violet 785 antimouse F4/80 (clone BM8), and Alexa Fluor 700 antimouse CD3 (clone 17A2). Lymphoid cell surface markers are as follows: PerCP/Cy5.5 antimouse CD3 (clone 17A2), Alexa Fluor 488 antimouse CD4 (clone GK1.5), APC/Fire 750 antimouse CD8a (clone 53-6.7), Brilliant Violet 421 antimouse CD25 (clone PC61), Brilliant Violet 785 antimouse CD45.2 (clone 104), Brilliant Violet 605 antimouse CD62L (clone MEL-14), Alexa Fluor 647 antimouse CD127 (IL-7Rα) (clone A7R34), PE antimouse/human CD44 (clone IM7), PE/Cy7 antimouse CD49b (pan-NK cells) (clone DX5), Brilliant Violet 421 antimouse CD16/32 (clone 93), and PE antimouse 4-1BB (clone 17B5). Multicolor flow data acquisition was performed on BD Celesta, and data analysis was performed with FlowJo software (version 10.3). Statistical analyses were performed with GraphPad Prism software (version 7.03).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00567" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73013" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73013" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00567?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00567</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary results for the cytotoxicity assay of <b>6a</b>–<b>6c</b> along with NextA (Figure S1); supplementary results for immunoblot analysis of Suprastat’s effects on α-tubulin acetylation in macrophages (Figure S2); supplementary figure for MD simulation of HDAC6 and the HDAC6/Suprastat complex (Figure S3); supplementary figure for the liver microsomal stability test (Figure S4); supplementary data on the HDAC enzymatic assay of <b>6d</b>–<b>6</b>f (Table S1); supplementary table for crystallization data collection and refinement statistics (Table S2); and <sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, and HPLC purity reports for compounds <b>6a</b>–<b>6c</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">MD simulations of the HDAC6/Suprastat complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_002.mp4" class="ext-link">MP4</a>)</p></li><li><p class="inline">Molecular formula strings of compounds <b>2</b>, <b>4a</b>–<b>4d</b>, <b>5a</b>–<b>5d</b>, <b>6b</b>–<b>6f</b>, Suprastat, and nexturastat including screening data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">A snapshot of the MD simulations of the HDAC6/Suprastat complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_004.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf">jm0c00567_si_001.pdf (1.68 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_002.mp4">jm0c00567_si_002.mp4 (1.33 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_003.csv">jm0c00567_si_003.csv (1.41 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_004.pdb">jm0c00567_si_004.pdb (439.02 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and corresponding structure factors for the zHDAC6-CD2/Suprastat complex have been deposited at the Protein Data Bank (PDB) as the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY">6TCY</a> entry. Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00567" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13030" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13030" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cyril Bařinka</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Structural Biology, Institute of Biotechnology
of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2751-3060" title="Orcid link">http://orcid.org/0000-0003-2751-3060</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#df9ca6adb6b3f19dbeadb6b1b4be9fb6bdabf1bcbeacf1bca5"><span class="__cf_email__" data-cfemail="66251f140f0a482407140f080d07260f04124805071548051c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan P. Kozikowski</span> - <span class="hlFld-Affiliation affiliation">Bright
Minds Biosciences, Toronto, ON M5H 3V9, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ff9e939e91bf9d8d9698978b9296919b8c9d9690d19c9092"><span class="__cf_email__" data-cfemail="0c6d606d624c6e7e656b6478616562687f6e6563226f6361">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alejandro Villagra</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9346-8355" title="Orcid link">http://orcid.org/0000-0001-9346-8355</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e283948b8e8e83859083a2859597cc878697"><span class="__cf_email__" data-cfemail="afced9c6c3c3cec8ddceefc8d8da81cacbda">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satish Noonepalle</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sida Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, University of Illinois at Chicago, Chicago, Illinois 60612, United States</span>; 
    <span>Present Address:
                        Departments of Chemistry, Center for Molecular Innovation and Drug Discovery and Center for Developmental Therapeutics, Northwestern University, Evanston, Illinois 60208, United States (S.S.)</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0295-2545" title="Orcid link">http://orcid.org/0000-0002-0295-2545</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jakub Ptáček</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Structural Biology, Institute of Biotechnology
of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maurício T. Tavares</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, University of Illinois at Chicago, Chicago, Illinois 60612, United States</span>; 
    <span>Present Address:
                        Department of Molecular Medicine, Scripps Research, Jupiter, Florida 33458, United States (M.T.T.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guiping Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, University of Illinois at Chicago, Chicago, Illinois 60612, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9818-4773" title="Orcid link">http://orcid.org/0000-0001-9818-4773</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Stránský</span> - <span class="hlFld-Affiliation affiliation">Centre
of Molecular Structure, Institute of Biotechnology
of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiří Pavlíček</span> - <span class="hlFld-Affiliation affiliation">Centre
of Molecular Structure, Institute of Biotechnology
of the Czech Academy of Sciences, BIOCEV, Prumyslova 595, Vestec 252 50, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Glaucio M. Ferreira</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, SP 05508-000, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Melissa Hadley</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guido Pelaez</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, The George Washington University, Washington, District of Columbia 20052, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.N. and S.S. contributed equally to this paper. A.V., A.P.K., and C.B. conceived the original idea, initiated the project, oversaw all the chemical, biological, crystallographic, ADMET as well as animal experimental designs/data analysis, and revised the manuscript. S.S. designed and synthesized compounds, oversaw all the experimental design, analyzed data, and wrote the manuscript with assistance from the other authors. S.N., M.H., and G.P. designed and performed all cellular and animal experimental designs, analyzed data, and contributed to the manuscript writing. J. Ptáček evaluated IC50 values of HDAC isoforms and generated ADMET data. J.S. and J. Pavlicek crystallized, solved, and refined the zHDAC6/Suprastat crystal complex. M.T.T and G.M.F. contributed to the molecular dynamics simulations. G.Z. contributed to the scale-up work of Suprastat.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40274" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40274" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work is funded by NIH R21 CA184612-01 and Melanoma Research Foundation CDA Grant Award (A.V.) and NIH R01NS079183, R43HD093464, and R41AG058283 (A.P.K.). Additionally, this work was in part supported by the CAS (RVO: 86652036), the Czech Science Foundation (15-19640S), and project BIOCEV (CZ.1.05/1.1.00/02.0109) from the ERDF (C.B.). We would like to acknowledge the important technical contributions and advice of Kimberlyn Acklin, MS, SCYM (ASCP), at The George Washington University Flow Cytometry Core Facility and Bethany Rentz, RVT, at The George Washington University Office of Animal Research. We thank Petra Baranova and Barbora Havlinova for their excellent technical assistance and Lucia Motlova for help with crystallization experiments. We also thank Dr. Werner Tueckmantel for proofreading the article and providing comments. We acknowledge the Helmholtz-Zentrum Berlin for the allocation of synchrotron radiation beamtime at the MX14.2 beamline and the support by the project CALIPSOplus (grant agreement 730872) from the EU Framework Programme for Research and Innovation HORIZON 2020 and CMS-Biocev (“Crystallization/Diffraction”) supported by MEYS CR (LM2018127).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Cbz</td><td class="NLM_def"><p class="first last">carboxylbenzyl</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">B<sub>2</sub>pin<sub>2</sub></td><td class="NLM_def"><p class="first last">bis(pinacolato)diboron</p></td></tr><tr><td class="NLM_term">Pd(dppf)Cl<sub>2</sub></td><td class="NLM_def"><p class="first last">[1,1′-bis(diphenylphosphino)ferrocene]-dichloropalladium(II)</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">IL-10</td><td class="NLM_def"><p class="first last">interleukin 10;</p></td></tr><tr><td class="NLM_term">PD-L1</td><td class="NLM_def"><p class="first last">programmed death-ligand 1</p></td></tr><tr><td class="NLM_term">PD-1</td><td class="NLM_def"><p class="first last">programmed cell death-1</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53406" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53406" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span> <span> </span><span class="NLM_article-title">Regulation of cellular metabolism by protein lysine acetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1004</span>, <span class="refDoi"> DOI: 10.1126/science.1179689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1126%2Fscience.1179689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20167786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1000-1004&author=S.+Zhaoauthor=W.+Xuauthor=W.+Jiangauthor=W.+Yuauthor=Y.+Linauthor=T.+Zhangauthor=J.+Yaoauthor=L.+Zhouauthor=Y.+Zengauthor=H.+Liauthor=Y.+Liauthor=J.+Shiauthor=W.+Anauthor=S.+M.+Hancockauthor=F.+Heauthor=L.+Qinauthor=J.+Chinauthor=P.+Yangauthor=X.+Chenauthor=Q.+Leiauthor=Y.+Xiongauthor=K.+L.+Guan&title=Regulation+of+cellular+metabolism+by+protein+lysine+acetylation&doi=10.1126%2Fscience.1179689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cellular metabolism by protein lysine acetylation</span></div><div class="casAuthors">Zhao, Shimin; Xu, Wei; Jiang, Wenqing; Yu, Wei; Lin, Yan; Zhang, Tengfei; Yao, Jun; Zhou, Li; Zeng, Yaxue; Li, Hong; Li, Yixue; Shi, Jiong; An, Wenlin; Hancock, Susan M.; He, Fuchu; Qin, Lunxiu; Chin, Jason; Yang, Pengyuan; Chen, Xian; Lei, Qunying; Xiong, Yue; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1000-1004</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Protein Lys acetylation has emerged as a key post-translational modification in cellular regulation, in particular through the modification of histones and nuclear transcription regulators.  Here, the authors show that acetylation of Lys residues is a prevalent modification in enzymes that catalyze intermediate metab.  Virtually every enzyme in glycolysis, gluconeogenesis, the tricarboxylic acid (TCA) cycle, the urea cycle, fatty acid metab., and glycogen metab. was found to be acetylated in human liver tissue.  The concn. of metabolic fuels, such as glucose, amino acids, and fatty acids, influenced the acetylation status of metabolic enzymes.  Acetylation activated enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase in fatty acid oxidn. and malate dehydrogenase in the TCA cycle, inhibited argininosuccinate lyase in the urea cycle, and destabilized phosphoenolpyruvate carboxykinase in gluconeogenesis.  Thus, this study revealed that acetylation plays a major role in metabolic regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbdnDsuudXV7Vg90H21EOLACvtfcHk0ljnuNIxO4qiHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D&md5=d2c3c897077b9285842a7c55e8f8737e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179689%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DAn%26aufirst%3DW.%26aulast%3DHancock%26aufirst%3DS.%2BM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26atitle%3DRegulation%2520of%2520cellular%2520metabolism%2520by%2520protein%2520lysine%2520acetylation%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1000%26epage%3D1004%26doi%3D10.1126%2Fscience.1179689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G. P.</span></span> <span> </span><span class="NLM_article-title">Acetylation of metabolic enzymes coordinates carbon source utilization and metabolic flux</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1004</span>– <span class="NLM_lpage">1007</span>, <span class="refDoi"> DOI: 10.1126/science.1179687</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1126%2Fscience.1179687" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20167787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1004-1007&author=Q.+Wangauthor=Y.+Zhangauthor=C.+Yangauthor=H.+Xiongauthor=Y.+Linauthor=J.+Yaoauthor=H.+Liauthor=L.+Xieauthor=W.+Zhaoauthor=Y.+Yaoauthor=Z.+B.+Ningauthor=R.+Zengauthor=Y.+Xiongauthor=K.+L.+Guanauthor=S.+Zhaoauthor=G.+P.+Zhao&title=Acetylation+of+metabolic+enzymes+coordinates+carbon+source+utilization+and+metabolic+flux&doi=10.1126%2Fscience.1179687"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation of metabolic enzymes coordinates carbon source utilization and metabolic flux</span></div><div class="casAuthors">Wang, Qijun; Zhang, Yakun; Yang, Chen; Xiong, Hui; Lin, Yan; Yao, Jun; Li, Hong; Xie, Lu; Zhao, Wei; Yao, Yufeng; Ning, Zhi-Bin; Zeng, Rong; Xiong, Yue; Guan, Kun-Liang; Zhao, Shimin; Zhao, Guo-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1004-1007</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation regulates many eukaryotic cellular processes, but its function in prokaryotes is largely unknown.  We demonstrated that central metab. enzymes in Salmonella were acetylated extensively and differentially in response to different C sources, concomitantly with changes in cell growth and metabolic flux.  The relative activities of key enzymes controlling the direction of glycolysis vs. gluconeogenesis and the branching between citrate cycle and glyoxylate bypass were all regulated by acetylation.  This modulation is mainly controlled by a pair of lysine acetyltransferase and deacetylase, whose expressions are coordinated with growth status.  Reversible acetylation of metabolic enzymes ensure that cells respond environmental changes via promptly sensing cellular energy status and flexibly altering reaction rates or directions.  It represents a metabolic regulatory mechanism conserved from bacteria to mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphUlI--SS1CbVg90H21EOLACvtfcHk0ljnuNIxO4qiHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbs%253D&md5=cd56ec28074baf0b6277b0936e4fdab2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179687&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179687%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DNing%26aufirst%3DZ.%2BB.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DG.%2BP.%26atitle%3DAcetylation%2520of%2520metabolic%2520enzymes%2520coordinates%2520carbon%2520source%2520utilization%2520and%2520metabolic%2520flux%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1004%26epage%3D1007%26doi%3D10.1126%2Fscience.1179687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">The growing landscape of lysine acetylation links metabolism and cell signalling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrm3841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnrm3841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25053359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtF2ktbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=536-550&author=C.+Choudharyauthor=B.+T.+Weinertauthor=Y.+Nishidaauthor=E.+Verdinauthor=M.+Mann&title=The+growing+landscape+of+lysine+acetylation+links+metabolism+and+cell+signalling&doi=10.1038%2Fnrm3841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The growing landscape of lysine acetylation links metabolism and cell signaling</span></div><div class="casAuthors">Choudhary, Chunaram; Weinert, Brian T.; Nishida, Yuya; Verdin, Eric; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-550</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lys residue acetylation is a conserved protein post-translational modification that links acetyl-CoA metab. and cellular signaling.  Recent advances in the identification and quantification of Lys residue acetylation by mass spectrometry have increased the understanding of Lys acetylation, implicating it in many biol. processes through the regulation of protein interactions, activity, and localization.  In addn., proteins are frequently modified by other types of acylations, such as formylation, butyrylation, propionylation, succinylation, malonylation, myristoylation, glutarylation, and crotonylation.  The intricate link between Lys acylation and cellular metab. has been clarified by the occurrence of several such metabolite-sensitive acylations and their selective removal by sirtuin deacylases.  These emerging findings point to new functions for different Lys acylations and deacylating enzymes and also highlight the mechanisms by which acetylation regulates various cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFJb5kUjj7arVg90H21EOLACvtfcHk0lj0YFqEMigjBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtF2ktbrN&md5=89703cd65c2cebe4a1474cf40f6d0be7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3841%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DNishida%26aufirst%3DY.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DThe%2520growing%2520landscape%2520of%2520lysine%2520acetylation%2520links%2520metabolism%2520and%2520cell%2520signalling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D536%26epage%3D550%26doi%3D10.1038%2Fnrm3841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span> <span> </span><span class="NLM_article-title">HDACs and HDAC inhibitors in cancer development and therapy</span>. <i>Cold Spring Harb. Perspect Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a026831</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1101%2Fcshperspect.a026831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27599530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpslCitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a026831&author=Y.+Liauthor=E.+Seto&title=HDACs+and+HDAC+inhibitors+in+cancer+development+and+therapy&doi=10.1101%2Fcshperspect.a026831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">HDACs and HDAC inhibitors in cancer development and therapy</span></div><div class="casAuthors">Li, Yixuan; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">a026831/1-a026831/35</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Over the last several decades, it has become clear that epigenetic abnormalities may be one of the hallmarks of cancer.  Posttranslational modifications of histones, for example, may play a crucial role in cancer development and progression by modulating gene transcription, chromatin remodeling, and nuclear architecture.  Histone acetylation, a well-studied posttranslational histone modification, is controlled by the opposing activities of histone acetyltransferases (HATs) and histone deacetylases (HDACs).  By removing acetyl groups, HDACs reverse chromatin acetylation and alter transcription of oncogenes and tumor suppressor genes.  In addn.,HDACs deacetylate numerous nonhistone cellular substrates that govern a wide array of biol. processes including cancer initiation and progression.  This review will discuss the role of HDACs in cancer and the therapeutic potential of HDAC inhibitors (HDACi) as emerging drugs in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnRL0Y7L3_s7Vg90H21EOLACvtfcHk0lj0YFqEMigjBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpslCitb4%253D&md5=1c647aa38027de1c4c92f74340d9041a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026831%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DHDACs%2520and%2520HDAC%2520inhibitors%2520in%2520cancer%2520development%2520and%2520therapy%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da026831%26doi%3D10.1101%2Fcshperspect.a026831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckschlager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiborova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrabeta, J.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase inhibitors as anticancer drugs</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1414</span>, <span class="refDoi"> DOI: 10.3390/ijms18071414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.3390%2Fijms18071414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28671573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1414&author=T.+Eckschlagerauthor=J.+Plchauthor=M.+Stiborovaauthor=J.+Hrabeta&title=Histone+deacetylase+inhibitors+as+anticancer+drugs&doi=10.3390%2Fijms18071414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors as anticancer drugs</span></div><div class="casAuthors">Eckschlager, Tomas; Plch, Johana; Stiborova, Marie; Hrabeta, Jan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1414/1-1414/25</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes.  Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT).  HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response.  Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc.  HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy.  HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma.  Other HDAC inhibitors are in clin. trials for the treatment of hematol. and solid malignancies.  The results of such studies are promising but further larger studies are needed.  Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance.  Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclin. and clin. studies, including the combination with other therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNVhoLBAHnrVg90H21EOLACvtfcHk0lj0YFqEMigjBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P&md5=6018c7709a09548214bfe34bd91107a9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fijms18071414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071414%26sid%3Dliteratum%253Aachs%26aulast%3DEckschlager%26aufirst%3DT.%26aulast%3DPlch%26aufirst%3DJ.%26aulast%3DStiborova%26aufirst%3DM.%26aulast%3DHrabeta%26aufirst%3DJ.%26atitle%3DHistone%2520deacetylase%2520inhibitors%2520as%2520anticancer%2520drugs%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1414%26doi%3D10.3390%2Fijms18071414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+deacetylases+and+their+inhibitors+in+cancer%2C+neurological+diseases+and+immune+disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0ligZYN29s_NCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520deacetylases%2520and%2520their%2520inhibitors%2520in%2520cancer%252C%2520neurological%2520diseases%2520and%2520immune%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">HDAC6 a new cellular stress surveillance factor</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.4161/cc.7.1.5186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.4161%2Fcc.7.1.5186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=18196966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVKntLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=7-10&author=P.+Matthiasauthor=M.+Yoshidaauthor=S.+Khochbin&title=HDAC6+a+new+cellular+stress+surveillance+factor&doi=10.4161%2Fcc.7.1.5186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 a new cellular stress surveillance factor</span></div><div class="casAuthors">Matthias, Patrick; Yoshida, Minoru; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-10</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Less than a decade has passed since histone deacetylase harboring (HDAC6) was first identified and regarded as an unusual histone deacetylase harboring 2 catalytic domains.  Early demonstration of its cytoplasmic localization, its ubiquitin-binding, and its tubulin-deacetylase activities took HDAC6 far away from everything known to involve other histone-deacetylases.  Recent discoveries confirmed the very unique functions of HDAC6 among deacetylases and pointed to this protein as a master regulator of cell response to cytotoxic assaults.  HDAC6 appears both as a sensor of stressful stimuli and as an effector, which, thanks to its wide range of activities, mediates and coordinates appropriate cell responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Tzz-Xi44N7Vg90H21EOLACvtfcHk0ligZYN29s_NCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVKntLY%253D&md5=6b0784dad7b4fac9a31e803548bc7a84</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4161%2Fcc.7.1.5186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.7.1.5186%26sid%3Dliteratum%253Aachs%26aulast%3DMatthias%26aufirst%3DP.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DHDAC6%2520a%2520new%2520cellular%2520stress%2520surveillance%2520factor%26jtitle%3DCell%2520Cycle%26date%3D2008%26volume%3D7%26spage%3D7%26epage%3D10%26doi%3D10.4161%2Fcc.7.1.5186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, J.</span></span> <span> </span><span class="NLM_article-title">Action mechanisms of histone deacetylase inhibitors in the treatment of hematological malignancies</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1543</span>– <span class="NLM_lpage">1549</span>, <span class="refDoi"> DOI: 10.1111/cas.13062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1111%2Fcas.13062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27554046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlykur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1543-1549&author=Y.+Imaiauthor=Y.+Maruauthor=J.+Tanaka&title=Action+mechanisms+of+histone+deacetylase+inhibitors+in+the+treatment+of+hematological+malignancies&doi=10.1111%2Fcas.13062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Action mechanisms of histone deacetylase inhibitors in the treatment of hematological malignancies</span></div><div class="casAuthors">Imai, Yoichi; Maru, Yoshiro; Tanaka, Junji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1543-1549</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Histone deacetylases (HDACs) critically regulate gene expression by detg. the acetylation status of histones.  In addn., studies have increasingly focused on the activities of HDACs, esp. involving non-histone proteins, and their various biol. effects.  Aberrant HDAC expression obsd. in several kinds of human tumors makes HDACs potential targets for cancer treatment.  Several preclin. studies have suggested that HDAC inhibitors exhibit some efficacy in the treatment of acute myelogenous leukemia (AML) with AML1-ETO, which mediates transcriptional repression through its interaction with a complex including HDAC1.  Recurrent mutations in epigenetic regulators are found in T-cell lymphomas (TCLs), and HDAC inhibitors and hypomethylating agents were shown to act cooperatively in the treatment of TCLs.  Preclin. modeling has suggested that persistent activation of the signal transducer and activator of transcription (STAT) signaling pathway could serve as a useful biomarker of resistance to HDAC inhibitor in patients with cutaneous TCL.  Panobinostat, a pan-HDAC inhibitor, in combination with bortezomib and dexamethasone, has achieved longer progression-free survival in patients with relapsed/refractory multiple myeloma (MM) than the placebo in combination with bortezomib and dexamethasone.  Panobinostat inhibited MM cell growth by degrading protein phosphatase 3 catalytic subunit α (PPP3CA), a catalytic subunit of calcineurin.  This degrdn. was suggested to be mediated by the blockade of the chaperone function of heat shock protein 90 (HSP90) due to HDAC6 inhibition.  Aberrant PPP3CA expression in advanced MM indicated a possible correlation between high PPP3CA expression and the pathogenesis of MM.  Furthermore, PPP3CA was suggested as a common target of panobinostat and bortezomib.  This article is protected by copyright.  All rights reserved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs8mVwuDgQ9LVg90H21EOLACvtfcHk0ligZYN29s_NCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlykur3O&md5=d97540b4f33bb4ce6c044717785f9954</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1111%2Fcas.13062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13062%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DY.%26aulast%3DMaru%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DJ.%26atitle%3DAction%2520mechanisms%2520of%2520histone%2520deacetylase%2520inhibitors%2520in%2520the%2520treatment%2520of%2520hematological%2520malignancies%26jtitle%3DCancer%2520Sci.%26date%3D2016%26volume%3D107%26spage%3D1543%26epage%3D1549%26doi%3D10.1111%2Fcas.13062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">A patent review of histone deacetylase 6 inhibitors in neurodegenerative diseases (2014-2019</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1080/13543776.2019.1708901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1080%2F13543776.2019.1708901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=31865813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVKnsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=121-136&author=S.+Shenauthor=A.+P.+Kozikowski&title=A+patent+review+of+histone+deacetylase+6+inhibitors+in+neurodegenerative+diseases+%282014-2019&doi=10.1080%2F13543776.2019.1708901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A patent review of histone deacetylase 6 inhibitors in neurodegenerative diseases (2014-2019)</span></div><div class="casAuthors">Shen, Sida; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-136</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase 6 (HDAC6) is unique in comparison with other zinc-dependent HDAC family members.  An increasing amt. of evidence from clin. and preclin. research demonstrates the potential of HDAC6 inhibition as an effective therapeutic approach for the treatment of cancer, autoimmune diseases, as well as neurol. disorders.  The recently disclosed crystal structures of HDAC6-ligand complexes offer further means for achieving pharmacophore refinement, thus further accelerating the pace of HDAC6 inhibitor discovery in the last few years.: This review summarizes the latest clin. status of HDAC6 inhibitors, discusses pharmacol. applications of selective HDAC6 inhibitors in neurodegenerative diseases, and describes the patent applications dealing with HDAC6 inhibitors from 2014-2019 that have not been reported in research articles.: Phenylhydroxamate has proven a very useful scaffold in the discovery of potent and selective HDAC6 inhibitors.  However, weaknesses of the hydroxamate function such as metabolic instability and mutagenic potential limit its application in the neurol. field, where long-term administration is required.  The recent invention of oxadiazole-based ligands by pharmaceutical companies may provide a new opportunity to optimize the druglike properties of HDAC6 inhibitors for the treatment of neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCWThC7SH8r7Vg90H21EOLACvtfcHk0lgzS38hJ_-Wfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVKnsb3J&md5=926d00ce3a0153953eb2f933599500b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1708901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1708901%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DA%2520patent%2520review%2520of%2520histone%2520deacetylase%25206%2520inhibitors%2520in%2520neurodegenerative%2520diseases%2520%25282014-2019%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2020%26volume%3D30%26spage%3D121%26epage%3D136%26doi%3D10.1080%2F13543776.2019.1708901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rébé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghiringhelli, F.</span></span> <span> </span><span class="NLM_article-title">STAT3, a master regulator of anti-tumor immune response</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1280</span>, <span class="refDoi"> DOI: 10.3390/cancers11091280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.3390%2Fcancers11091280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtVWmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1280&author=C.+R%C3%A9b%C3%A9author=F.+Ghiringhelli&title=STAT3%2C+a+master+regulator+of+anti-tumor+immune+response&doi=10.3390%2Fcancers11091280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3, a master regulator of anti-tumor immune response</span></div><div class="casAuthors">Rebe, Cedric; Ghiringhelli, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1280</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Immune cells in the tumor microenvironment regulate cancer growth.  Thus cancer progression is dependent on the activation or repression of transcription programs involved in the proliferation/activation of lymphoid and myeloid cells.  One of the main transcription factors involved in many of these pathways is the signal transducer and activator of transcription 3 (STAT3).  In this review we will focus on the role of STAT3 and its regulation, e.g., by phosphorylation or acetylation in immune cells and how it might impact immune cell function and tumor progression.  Moreover, we will review the ability of STAT3 to regulate checkpoint inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgk-poT_FrCrVg90H21EOLACvtfcHk0lgzS38hJ_-Wfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtVWmsbs%253D&md5=89d95d832347103f511110d1f23e720d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3390%2Fcancers11091280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11091280%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25A9b%25C3%25A9%26aufirst%3DC.%26aulast%3DGhiringhelli%26aufirst%3DF.%26atitle%3DSTAT3%252C%2520a%2520master%2520regulator%2520of%2520anti-tumor%2520immune%2520response%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D1280%26doi%3D10.3390%2Fcancers11091280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock-Klotz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span> <span> </span><span class="NLM_article-title">A novel role for histone deacetylase 6 in the regulation of the tolerogenic STAT3/IL-10 pathway in APCs</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">2850</span>– <span class="NLM_lpage">2862</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1302778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.4049%2Fjimmunol.1302778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25108026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2qur3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2014&pages=2850-2862&author=F.+Chengauthor=M.+Lienlafauthor=H.-W.+Wangauthor=P.+Perez-Villarroelauthor=C.+Leeauthor=K.+Woanauthor=J.+Rock-Klotzauthor=E.+Sahakianauthor=D.+Woodsauthor=J.+Pinilla-Ibarzauthor=J.+Kalinauthor=J.+Taoauthor=W.+Hancockauthor=A.+Kozikowskiauthor=E.+Setoauthor=A.+Villagraauthor=E.+M.+Sotomayor&title=A+novel+role+for+histone+deacetylase+6+in+the+regulation+of+the+tolerogenic+STAT3%2FIL-10+pathway+in+APCs&doi=10.4049%2Fjimmunol.1302778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Role for Histone Deacetylase 6 in the Regulation of the Tolerogenic STAT3/IL-10 Pathway in APCs</span></div><div class="casAuthors">Cheng, Fengdong; Lienlaf, Maritza; Wang, Hong-Wei; Perez-Villarroel, Patricio; Lee, Calvin; Woan, Karrune; Rock-Klotz, Jennifer; Sahakian, Eva; Woods, David; Pinilla-Ibarz, Javier; Kalin, Jay; Tao, Jianguo; Hancock, Wayne; Kozikowski, Alan; Seto, Edward; Villagra, Alejandro; Sotomayor, Eduardo M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2850-2862</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">APCs are crit. in T cell activation and in the induction of T cell tolerance.  Epigenetic modifications of specific genes in the APC play a key role in this process, and among them histone deacetylases (HDACs) have emerged as key participants.  HDAC6, one of the members of this family of enzymes, has been shown to be involved in regulation of inflammatory and immune responses.  In this study, to our knowledge we show for the first time that genetic or pharmacol. disruption of HDAC6 in macrophages and dendritic cells results in diminished prodn. of the immunosuppressive cytokine IL-10 and induction of inflammatory APCs that effectively activate Ag-specific naive T cells and restore the responsiveness of anergic CD4+ T cells.  Mechanistically, we have found that HDAC6 forms a previously unknown mol. complex with STAT3, assocn. that was detected in both the cytoplasmic and nuclear compartments of the APC.  By using HDAC6 recombinant mutants we identified the domain comprising amino acids 503-840 as being required for HDAC6 interaction with STAT3.  Furthermore, by re-chromatin immunopptn. we confirmed that HDAC6 and STAT3 are both recruited to the same DNA sequence within the Il10 gene promoter.  Of note, disruption of this complex by knocking down HDAC6 resulted in decreased STAT3 phosphorylation-but no changes in STAT3 acetylation-as well as diminished recruitment of STAT3 to the Il10 gene promoter region.  The addnl. demonstration that a selective HDAC6 inhibitor disrupts this STAT3/IL-10 tolerogenic axis points to HDAC6 as a novel mol. target in APCs to overcome immune tolerance and tips the balance toward T cell immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSLvMP_-UNx7Vg90H21EOLACvtfcHk0lgzS38hJ_-Wfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2qur3P&md5=4d547665a3108fcf168f353bba4f4646</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1302778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1302778%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.-W.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DWoan%26aufirst%3DK.%26aulast%3DRock-Klotz%26aufirst%3DJ.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DWoods%26aufirst%3DD.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DKalin%26aufirst%3DJ.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DHancock%26aufirst%3DW.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26atitle%3DA%2520novel%2520role%2520for%2520histone%2520deacetylase%25206%2520in%2520the%2520regulation%2520of%2520the%2520tolerogenic%2520STAT3%252FIL-10%2520pathway%2520in%2520APCs%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D193%26spage%3D2850%26epage%3D2862%26doi%3D10.4049%2Fjimmunol.1302778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pabon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montecino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Essential role of HDAC6 in the regulation of PD-L1 in melanoma</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">735</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2015.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.molonc.2015.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=26775640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=735-750&author=M.+Lienlafauthor=P.+Perez-Villarroelauthor=T.+Knoxauthor=M.+Pabonauthor=E.+Sahakianauthor=J.+Powersauthor=K.+V.+Woanauthor=C.+Leeauthor=F.+Chengauthor=S.+Dengauthor=K.+S.+M.+Smalleyauthor=M.+Montecinoauthor=A.+Kozikowskiauthor=J.+Pinilla-Ibarzauthor=A.+Sarnaikauthor=E.+Setoauthor=J.+Weberauthor=E.+M.+Sotomayorauthor=A.+Villagra&title=Essential+role+of+HDAC6+in+the+regulation+of+PD-L1+in+melanoma&doi=10.1016%2Fj.molonc.2015.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of HDAC6 in the regulation of PD-L1 in melanoma</span></div><div class="casAuthors">Lienlaf, M.; Perez-Villarroel, P.; Knox, T.; Pabon, M.; Sahakian, E.; Powers, J.; Woan, K. V.; Lee, C.; Cheng, F.; Deng, S.; Smalley, K. S. M.; Montecino, M.; Kozikowski, A.; Pinilla-Ibarz, J.; Sarnaik, A.; Seto, E.; Weber, J.; Sotomayor, E. M.; Villagra, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">735-750</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs), originally described as histone modifiers, have more recently been demonstrated to target a variety of other proteins unrelated to the chromatin environment.  In this context, our present work demonstrates that the pharmacol. or genetic abrogation of HDAC6 in primary melanoma samples and cell lines, down-regulates the expression of PD-L1, an important co-stimulatory mol. expressed in cancer cells, which activates the inhibitory regulatory pathway PD-1 in T-cells.  Our data suggests that this novel mechanism of PD-L1 regulation is mainly mediated by the influence of HDAC6 over the recruitment and activation of STAT3.  Addnl., we obsd. that selective HDAC6 inhibitors impairs tumor growth and reduce the in vivo expression of several inhibitory check-point mols. and other regulatory pathways involved in immunosurveillance.  Most importantly, these results provide a key pre-clin. rationale and justification to further study isotype selective HDAC6 inhibitors as potential immuno-modulatory agents in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnUp3ctcdjRLVg90H21EOLACvtfcHk0ljC8uVjvzq_NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtFWkuw%253D%253D&md5=6ead389dab6bd5f4fc621c452cd4e303</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2015.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2015.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DPabon%26aufirst%3DM.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DWoan%26aufirst%3DK.%2BV.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DMontecino%26aufirst%3DM.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DSarnaik%26aufirst%3DA.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DEssential%2520role%2520of%2520HDAC6%2520in%2520the%2520regulation%2520of%2520PD-L1%2520in%2520melanoma%26jtitle%3DMol.%2520Oncol.%26date%3D2016%26volume%3D10%26spage%3D735%26epage%3D750%26doi%3D10.1016%2Fj.molonc.2015.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonepalle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gracia-Hernandez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Selective HDAC6 inhibitors improve anti-PD-1 immune checkpoint blockade therapy by decreasing the anti-inflammatory phenotype of macrophages and down-regulation of immunosuppressive proteins in tumor cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">6136</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-42237-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fs41598-019-42237-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=30992475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FmtFOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=6136&author=T.+Knoxauthor=E.+Sahakianauthor=D.+Banikauthor=M.+Hadleyauthor=E.+Palmerauthor=S.+Noonepalleauthor=J.+Kimauthor=J.+Powersauthor=M.+Gracia-Hernandezauthor=V.+Oliveiraauthor=F.+Chengauthor=J.+Chenauthor=C.+Barinkaauthor=J.+Pinilla-Ibarzauthor=N.+H.+Leeauthor=A.+Kozikowskiauthor=A.+Villagra&title=Selective+HDAC6+inhibitors+improve+anti-PD-1+immune+checkpoint+blockade+therapy+by+decreasing+the+anti-inflammatory+phenotype+of+macrophages+and+down-regulation+of+immunosuppressive+proteins+in+tumor+cells&doi=10.1038%2Fs41598-019-42237-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selective HDAC6 inhibitors improve anti-PD-1 immune checkpoint blockade therapy by decreasing the anti-inflammatory phenotype of macrophages and down-regulation of immunosuppressive proteins in tumor cells</span></div><div class="casAuthors">Knox Tessa; Banik Debarati; Hadley Melissa; Palmer Erica; Noonepalle Satish; Kim Jennifer; Gracia-Hernandez Maria; Cheng Fengdong; Chen Jie; Lee Norman H; Villagra Alejandro; Sahakian Eva; Powers John; Pinilla-Ibarz Javier; Oliveira Vasco; Barinka Cyril; Kozikowski Alan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6136</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone deacetylases (HDACs) are involved in diverse cellular regulatory mechanisms including non-canonical functions outside the chromatin environment.  Several publications have demonstrated that selective HDAC inhibitors (HDACi) can influence tumor immunogenicity and the functional activity of specific immune cells.  In particular, the selective inhibition of HDAC6 has been reported to decrease tumor growth in several malignancies.  However, there is still no clarity about the cellular components mediating this effect.  In this study, we evaluated the HDAC6i Nexturastat A as a priming agent to facilitate the transition of the tumor microenvironment from "cold" to "hot", and potentially augment immune check-point blockade therapies.  This combination modality demonstrated to significantly reduce tumor growth in syngeneic melanoma tumor models.  Additionally, we observed a complete neutralization of the up-regulation of PD-L1 and other immunosuppressive pathways induced by the treatment with anti-PD-1 blockade.  This combination also showed profound changes in the tumor microenvironment such as enhanced infiltration of immune cells, increased central and effector T cell memory, and a significant reduction of pro-tumorigenic M2 macrophages.  The evaluation of individual components of the tumor microenvironment suggested that the in vivo anti-tumor activity of HDAC6i is mediated by its effect on tumor cells and tumor-associated macrophages, and not directly over T cells.  Overall, our results indicate that selective HDAC6i could be used as immunological priming agents to sensitize immunologically "cold" tumors and subsequently improve ongoing immune check-point blockade therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSl6RaBRQ4Kdw8IxFYvehhofW6udTcc2eYcPWH_Kxa7Rrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FmtFOrsg%253D%253D&md5=137a9261e021e60b002d9312608eba76</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-42237-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-42237-3%26sid%3Dliteratum%253Aachs%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DBanik%26aufirst%3DD.%26aulast%3DHadley%26aufirst%3DM.%26aulast%3DPalmer%26aufirst%3DE.%26aulast%3DNoonepalle%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DGracia-Hernandez%26aufirst%3DM.%26aulast%3DOliveira%26aufirst%3DV.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DN.%2BH.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DSelective%2520HDAC6%2520inhibitors%2520improve%2520anti-PD-1%2520immune%2520checkpoint%2520blockade%2520therapy%2520by%2520decreasing%2520the%2520anti-inflammatory%2520phenotype%2520of%2520macrophages%2520and%2520down-regulation%2520of%2520immunosuppressive%2520proteins%2520in%2520tumor%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D6136%26doi%3D10.1038%2Fs41598-019-42237-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brochier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vistoli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Rational design and simple chemistry yield a superior, neuroprotective HDAC6 inhibitor, tubastatin A</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">10842</span>– <span class="NLM_lpage">10846</span>, <span class="refDoi"> DOI: 10.1021/ja102758v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja102758v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=10842-10846&author=K.+V.+Butlerauthor=J.+Kalinauthor=C.+Brochierauthor=G.+Vistoliauthor=B.+Langleyauthor=A.+P.+Kozikowski&title=Rational+design+and+simple+chemistry+yield+a+superior%2C+neuroprotective+HDAC6+inhibitor%2C+tubastatin+A&doi=10.1021%2Fja102758v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design and Simple Chemistry Yield a Superior, Neuroprotective HDAC6 Inhibitor, Tubastatin A</span></div><div class="casAuthors">Butler, Kyle V.; Kalin, Jay; Brochier, Camille; Vistoli, Guilio; Langley, Brett; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10842-10846</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design combined with homol. modeling techniques were used to develop potent inhibitors of HDAC6 that display superior selectivity for the HDAC6 isoenzyme compared to other inhibitors.  These inhibitors can be assembled in a few synthetic steps, and thus are readily scaled up for in vivo studies.  An optimized compd. from this series, designated Tubastatin A, was tested in primary cortical neuron cultures in which it was found to induce elevated levels of acetylated α-tubulin, but not histone, consistent with its HDAC6 selectivity.  Tubastatin A also conferred dose-dependent protection in primary cortical neuron cultures against glutathione depletion-induced oxidative stress.  Importantly, when given alone at all concns. tested, this hydroxamate-contg. HDAC6-selective compd. displayed no neuronal toxicity, thus, forecasting the potential application of this agent and its analogs to neurodegenerative conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruEaJPWQfjH7Vg90H21EOLACvtfcHk0lgqF296vTSgwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D&md5=f0cc68bb68f94ad1ef48f88cc6b57129</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fja102758v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja102758v%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKalin%26aufirst%3DJ.%26aulast%3DBrochier%26aufirst%3DC.%26aulast%3DVistoli%26aufirst%3DG.%26aulast%3DLangley%26aufirst%3DB.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DRational%2520design%2520and%2520simple%2520chemistry%2520yield%2520a%2520superior%252C%2520neuroprotective%2520HDAC6%2520inhibitor%252C%2520tubastatin%2520A%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D10842%26epage%3D10846%26doi%3D10.1021%2Fja102758v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. A.</span></span> <span> </span><span class="NLM_article-title">Development and therapeutic implications of selective histone deacetylase 6 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6297</span>– <span class="NLM_lpage">6313</span>, <span class="refDoi"> DOI: 10.1021/jm4001659</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4001659" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslSntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6297-6313&author=J.+H.+Kalinauthor=J.+A.+Bergman&title=Development+and+therapeutic+implications+of+selective+histone+deacetylase+6+inhibitors&doi=10.1021%2Fjm4001659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Therapeutic Implications of Selective Histone Deacetylase 6 Inhibitors</span></div><div class="casAuthors">Kalin, Jay H.; Bergman, Joel A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6297-6313</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This Perspective provides an in depth look at the numerous disease states in which histone deacetylase 6 (HDAC6) has been implicated.  The physiol. pathways, protein-protein interactions, and non-histone substrates relating to different pathol. conditions are discussed with regard to HDAC6.  Furthermore, the compds. and methods used to modulate HDAC6 activity are profiled.  The latter half of this Perspective analyzes reported HDAC6 selective inhibitors in terms of structure, potency, and selectivity over the other HDAC isoforms with the intent of providing a comprehensive overview of the mol. tools available.  Potential obstacles and future directions of HDAC6 research are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvFD0xh4CLiLVg90H21EOLACvtfcHk0lgqF296vTSgwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslSntrc%253D&md5=e54e7e8b172aba337581341360c0685c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm4001659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4001659%26sid%3Dliteratum%253Aachs%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DBergman%26aufirst%3DJ.%2BA.%26atitle%3DDevelopment%2520and%2520therapeutic%2520implications%2520of%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6297%26epage%3D6313%26doi%3D10.1021%2Fjm4001659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Vreese, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Hooghe, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and applications of benzohydroxamic acid-based histone deacetylase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.ejmech.2017.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28453994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslOhur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2017&pages=174-195&author=R.+De%0AVreeseauthor=M.+D%E2%80%99Hooghe&title=Synthesis+and+applications+of+benzohydroxamic+acid-based+histone+deacetylase+inhibitors&doi=10.1016%2Fj.ejmech.2017.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and applications of benzohydroxamic acid-based histone deacetylase inhibitors</span></div><div class="casAuthors">De Vreese, Rob; D'hooghe, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">174-195</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">This paper provides an overview of the synthesis and biol. activity of the most representative benzohydroxamic acid-based histone deacetylase inhibitors published to date.  Benzohydroxamic acids comprise an important class of HDAC inhibitors, and recently several of these structures have been evaluated in clin. trials for the treatment of a variety of cancers.  In this overview, benzohydroxamic acids were divided in four different classes based on their reported selectivity towards zinc-dependent HDACs: a first and major class consists of HDAC6 selective inhibitors, a second class deals with pan-HDAC inhibitors, a third class comprises HDAC8 selective inhibitors and a fourth, minor class includes dual HDAC6/8 selective inhibitors.  Through this approach, structure-activity relationships were identified for each class, which could help future researchers in the design and development of novel benzohydroxamic acid-based HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyKAa1s9hoSbVg90H21EOLACvtfcHk0lgqF296vTSgwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslOhur0%253D&md5=769c343bdbb3fe610eff001afa79e0bb</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVreese%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Hooghe%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520applications%2520of%2520benzohydroxamic%2520acid-based%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D135%26spage%3D174%26epage%3D195%26doi%3D10.1016%2Fj.ejmech.2017.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. P.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the discovery of potent and selective HDAC6 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">1406</span>– <span class="NLM_lpage">1418</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.ejmech.2017.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=29133060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2ntbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2018&pages=1406-1418&author=X.+X.+Wangauthor=R.+Z.+Wanauthor=Z.+P.+Liu&title=Recent+advances+in+the+discovery+of+potent+and+selective+HDAC6+inhibitors&doi=10.1016%2Fj.ejmech.2017.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the discovery of potent and selective HDAC6 inhibitors</span></div><div class="casAuthors">Wang, Xiu-Xiu; Wan, Ren-Zhong; Liu, Zhao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1406-1418</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase HDAC6, a member of the class IIb HDAC family, is unique among HDAC enzymes in having two active catalytic domains, and has unique physiol. function.  In addn. to the modification of histone, HDAC6 targets specific substrates including α-tubulin and HSP90, and are involved in protein trafficking and degrdn., cell shape and migration.  Selective HDAC6 inhibitors are an emerging class of pharmaceuticals due to the involvement of HDAC6 in different pathways related to neurodegenerative diseases, cancer, and immunol.  Therefore, extensive investigations have been made in the discovery of selective HDAC6 inhibitors.  Based on their different zinc binding groups (ZBGs), in this review, HDAC6 inhibitors are grouped as hydroxamic acids, a sulfur contg. ZBG based derivs. and other ZBG-derived compds., and their enzymic inhibitory activity, selectivity and other biol. activities are introduced and summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdsNBEWGhpCrVg90H21EOLACvtfcHk0lgH7bqgdwm0vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2ntbrM&md5=f24fea971dbed247e1bb2568f85d1703</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%2BX.%26aulast%3DWan%26aufirst%3DR.%2BZ.%26aulast%3DLiu%26aufirst%3DZ.%2BP.%26atitle%3DRecent%2520advances%2520in%2520the%2520discovery%2520of%2520potent%2520and%2520selective%2520HDAC6%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D143%26spage%3D1406%26epage%3D1418%26doi%3D10.1016%2Fj.ejmech.2017.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faria
Freitas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuendet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, P.</span></span> <span> </span><span class="NLM_article-title">HDAC inhibitors: a 2013-2017 patent survey</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1080/13543776.2018.1459568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1080%2F13543776.2018.1459568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVSns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=365-381&author=M.+Faria%0AFreitasauthor=M.+Cuendetauthor=P.+Bertrand&title=HDAC+inhibitors%3A+a+2013-2017+patent+survey&doi=10.1080%2F13543776.2018.1459568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC inhibitors: a 2013-2017 patent survey</span></div><div class="casAuthors">Faria Freitas, Micaela; Cuendet, Muriel; Bertrand, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">365-381</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A Review.  Zinc-dependent histone deacetylases (HDAC) inhibitors represent an important class of biol. active compds. with four of them approved by the FDA.  A wide range of mols. has been reported for applications in several human diseases.: This review covers recent efforts in the synthesis and applications of HDAC inhibitors from 2013-2017.: HDAC inhibitors represent an important class of biol. active compds. for single or combination therapies.  The current synthetic methodologies are oriented towards selective HDAC isoforms to achieve better therapeutic effects.  Among the recent patents available, most of them focus on HDAC6 selective inhibitors.  Beside this search for isoform selectivity, the quest for zinc binding groups with better pharmacokinetic properties and high potency against HDACs only motivates medicinal chemists, as well as the design of inhibitors targeting HDACs and at the same time another biol. target.  If the major applications are for anticancer activity, one can note the emerging applications in neurol. or metabolic disorders or for the stimulation of the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgdOR23sC-crVg90H21EOLACvtfcHk0lgH7bqgdwm0vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVSns7w%253D&md5=0bed63ff4bcd3599758686197372f35b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F13543776.2018.1459568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2018.1459568%26sid%3Dliteratum%253Aachs%26aulast%3DFaria%2BFreitas%26aufirst%3DM.%26aulast%3DCuendet%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DP.%26atitle%3DHDAC%2520inhibitors%253A%2520a%25202013-2017%2520patent%2520survey%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2018%26volume%3D28%26spage%3D365%26epage%3D381%26doi%3D10.1080%2F13543776.2018.1459568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flipo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassonneville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez-Poulain, R.</span></span> <span> </span><span class="NLM_article-title">Hydroxamates: relationships between structure and plasma stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6790</span>– <span class="NLM_lpage">6802</span>, <span class="refDoi"> DOI: 10.1021/jm900648x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900648x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6790-6802&author=M.+Flipoauthor=J.+Chartonauthor=A.+Hocineauthor=S.+Dassonnevilleauthor=B.+Deprezauthor=R.+Deprez-Poulain&title=Hydroxamates%3A+relationships+between+structure+and+plasma+stability&doi=10.1021%2Fjm900648x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxamates: Relationships between Structure and Plasma Stability</span></div><div class="casAuthors">Flipo, Marion; Charton, Julie; Hocine, Akila; Dassonneville, Sandrine; Deprez, Benoit; Deprez-Poulain, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6790-6802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydroxamates are valuable tools for chem. biol. as well as interesting leads for medicinal chem.  Although many hydroxamates display nanomolar activities against metalloproteases, only three hydroxamates have reached the market, among which is the HDAC inhibitor vorinostat.  Failures in development are generally attributed to lack of selectivity, toxicity, or poor stability.  To help medicinal chemists with respect to plasma stability, the authors have performed the first and preliminary study on structure-plasma stability for hydroxamates.  The authors define some structural rules to predict or improve the plasma stability in the preclin. stage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk66ooWU1NhbVg90H21EOLACvtfcHk0liNu_84xmkEzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ&md5=069a87988fbeb68a248d006c7f6307a0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm900648x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900648x%26sid%3Dliteratum%253Aachs%26aulast%3DFlipo%26aufirst%3DM.%26aulast%3DCharton%26aufirst%3DJ.%26aulast%3DHocine%26aufirst%3DA.%26aulast%3DDassonneville%26aufirst%3DS.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DDeprez-Poulain%26aufirst%3DR.%26atitle%3DHydroxamates%253A%2520relationships%2520between%2520structure%2520and%2520plasma%2520stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6790%26epage%3D6802%26doi%3D10.1021%2Fjm900648x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szarics, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benoy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Den Bosch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eubanks, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jope, R. S.</span></span> <span> </span><span class="NLM_article-title">Brain penetrable histone deacetylase 6 inhibitor SW-100 ameliorates memory and learning impairments in a mouse model of Fragile X syndrome</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1679</span>– <span class="NLM_lpage">1695</span>, <span class="refDoi"> DOI: 10.1021/acschemneuro.8b00600</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.8b00600" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitl2jtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1679-1695&author=A.+P.+Kozikowskiauthor=S.+Shenauthor=M.+Pardoauthor=M.+T.+Tavaresauthor=D.+Szaricsauthor=V.+Benoyauthor=C.+A.+Zimprichauthor=Z.+Kutilauthor=G.+Zhangauthor=C.+Ba%C5%99inkaauthor=M.+B.+Robersauthor=L.+Van+Den+Boschauthor=J.+H.+Eubanksauthor=R.+S.+Jope&title=Brain+penetrable+histone+deacetylase+6+inhibitor+SW-100+ameliorates+memory+and+learning+impairments+in+a+mouse+model+of+Fragile+X+syndrome&doi=10.1021%2Facschemneuro.8b00600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Brain Penetrable Histone Deacetylase 6 Inhibitor SW-100 Ameliorates Memory and Learning Impairments in a Mouse Model of Fragile X Syndrome</span></div><div class="casAuthors">Kozikowski, Alan P.; Shen, Sida; Pardo, Marta; Tavares, Mauricio T.; Szarics, Dora; Benoy, Veronick; Zimprich, Chad A.; Kutil, Zsofia; Zhang, Guiping; Barinka, Cyril; Robers, Matthew B.; Van Den Bosch, Ludo; Eubanks, James H.; Jope, Richard S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1679-1695</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Disease-modifying therapies are needed for Fragile X Syndrome (FXS), as at present there are no effective treatments or cures.  Herein, we report on a tetrahydroquinoline-based selective histone deacetylase 6 (HDAC6) inhibitor SW-100, its pharmacol. and ADMET properties, and its ability to improve upon memory performance in a mouse model of FXS, Fmr1-/- mice.  This small mol. demonstrates good brain penetrance, low-nanomolar potency for the inhibition of HDAC6 (IC50 = 2.3 nM), with at least a thousand-fold selectivity over all other class I, II, and IV HDAC isoforms.  Moreover, through its inhibition of the α-tubulin deacetylase domain of HDAC6 (CD2), in cells SW-100 upregulates α-tubulin acetylation with no effect on histone acetylation and selectively restores the impaired acetylated α-tubulin levels in the hippocampus of Fmr1-/- mice.  Lastly, SW-100 ameliorates several memory and learning impairments in Fmr1-/- mice, thus modeling the intellectual deficiencies assocd. with FXS, and hence providing a strong rationale for pursuing HDAC6-based therapies for the treatment of this rare disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphc-8bfGiZh7Vg90H21EOLACvtfcHk0liNu_84xmkEzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitl2jtb3E&md5=b1e70a1a45522720d481985b3a4339f3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.8b00600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.8b00600%26sid%3Dliteratum%253Aachs%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DShen%26aufirst%3DS.%26aulast%3DPardo%26aufirst%3DM.%26aulast%3DTavares%26aufirst%3DM.%2BT.%26aulast%3DSzarics%26aufirst%3DD.%26aulast%3DBenoy%26aufirst%3DV.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DVan%2BDen%2BBosch%26aufirst%3DL.%26aulast%3DEubanks%26aufirst%3DJ.%2BH.%26aulast%3DJope%26aufirst%3DR.%2BS.%26atitle%3DBrain%2520penetrable%2520histone%2520deacetylase%25206%2520inhibitor%2520SW-100%2520ameliorates%2520memory%2520and%2520learning%2520impairments%2520in%2520a%2520mouse%2520model%2520of%2520Fragile%2520X%2520syndrome%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2019%26volume%3D10%26spage%3D1679%26epage%3D1695%26doi%3D10.1021%2Facschemneuro.8b00600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Why hydroxamates may not be the best histone deacetylase inhibitors--What some may have forgotten or would rather forget?</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fcmdc.201500486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=26603496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=15-21&author=S.+Shenauthor=A.+P.+Kozikowski&title=Why+hydroxamates+may+not+be+the+best+histone+deacetylase+inhibitors%2D%2DWhat+some+may+have+forgotten+or+would+rather+forget%3F&doi=10.1002%2Fcmdc.201500486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Why Hydroxamates May Not Be the Best Histone Deacetylase Inhibitors-What Some May Have Forgotten or Would Rather Forget?</span></div><div class="casAuthors">Shen, Sida; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-21</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Hydroxamate-based histone deacetylase inhibitors (HDACIs) have been approved as therapeutic agents by the US Food and Drug Administration for use in oncol. applications.  While the potential utility of such HDACIs in other areas of medicinal chem. is tremendous, there are significant concerns that "pan-HDAC inhibitors" may be too broadly acting and/or toxic for clin. use beyond oncol.  In addn. to the isoenzyme selectivity challenge, the potential mutagenicity of hydroxamate-contg. HDAC inhibitors represents a major hindrance in their application to other therapeutic areas.  Herein the authors report on the mutagenicity of known hydroxamates, discuss the mechanisms responsible for their genotoxicity, and review some of the current alternatives to hydroxamates.  The authors conclude that the hydroxamate group, while providing high-potency HDACIs, is not necessarily the best zinc-binding group for HDACI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplftHMFVoN6rVg90H21EOLACvtfcHk0liNu_84xmkEzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL&md5=5183d2c0a4c86fe67bea480a66bc5dc6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500486%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DWhy%2520hydroxamates%2520may%2520not%2520be%2520the%2520best%2520histone%2520deacetylase%2520inhibitors--What%2520some%2520may%2520have%2520forgotten%2520or%2520would%2520rather%2520forget%253F%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D15%26epage%3D21%26doi%3D10.1002%2Fcmdc.201500486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keusch, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melancon, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helquist, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span> <span> </span><span class="NLM_article-title">Structural insights into HDAC6 tubulin deacetylation and its selective inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnchembio.2140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27454931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=748-754&author=Y.+Miyakeauthor=J.+J.+Keuschauthor=L.+Wangauthor=M.+Saitoauthor=D.+Hessauthor=X.+Wangauthor=B.+J.+Melanconauthor=P.+Helquistauthor=H.+Gutauthor=P.+Matthias&title=Structural+insights+into+HDAC6+tubulin+deacetylation+and+its+selective+inhibition&doi=10.1038%2Fnchembio.2140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into HDAC6 tubulin deacetylation and its selective inhibition</span></div><div class="casAuthors">Miyake, Yasuyuki; Keusch, Jeremy J.; Wang, Longlong; Saito, Makoto; Hess, Daniel; Wang, Xiaoning; Melancon, Bruce J.; Helquist, Paul; Gut, Heinz; Matthias, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">748-754</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors report crystal structures of zebrafish histone deacetylase 6 (HDAC6) catalytic domains in tandem or as single domains in complex with the (R) and (S) enantiomers of trichostatin A (TSA) or with the HDAC6-specific inhibitor, nexturastat A.  The tandem domains formed, together with the inter-domain linker, an ellipsoid-shaped complex with pseudo-twofold symmetry.  The authors identified important active site differences between both catalytic domains and revealed the binding mode of HDAC6 selective inhibitors.  HDAC inhibition assays with (R)- and (S)-TSA showed that (R)-TSA was a broad-range inhibitor, whereas (S)-TSA had moderate selectivity for HDAC6.  The authors identified a uniquely positioned α-helix and a flexible Trp residue in the loop joining α-helixes H20 to H21 as crit. for deacetylation of the physiol. substrate, tubulin.  Using single-mol. measurements and biochem. assays, the authors demonstrated that HDAC6 catalytic domain 2 deacetylated α-tubulin Lys-40 in the lumen of microtubules, but that its preferred substrate was unpolymd. tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpERJRurS1cC7Vg90H21EOLACvtfcHk0ljfkr0bfZg55Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rM&md5=4abe529ab89615df33f54350a1a8c69a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2140%26sid%3Dliteratum%253Aachs%26aulast%3DMiyake%26aufirst%3DY.%26aulast%3DKeusch%26aufirst%3DJ.%2BJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSaito%26aufirst%3DM.%26aulast%3DHess%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMelancon%26aufirst%3DB.%2BJ.%26aulast%3DHelquist%26aufirst%3DP.%26aulast%3DGut%26aufirst%3DH.%26aulast%3DMatthias%26aufirst%3DP.%26atitle%3DStructural%2520insights%2520into%2520HDAC6%2520tubulin%2520deacetylation%2520and%2520its%2520selective%2520inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D748%26epage%3D754%26doi%3D10.1038%2Fnchembio.2140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahendran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breslow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Unusual zinc-binding mode of HDAC6-selective hydroxamate inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">13459</span>– <span class="NLM_lpage">13464</span>, <span class="refDoi"> DOI: 10.1073/pnas.1718823114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1073%2Fpnas.1718823114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=29203661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=13459-13464&author=N.+J.+Porterauthor=A.+Mahendranauthor=R.+Breslowauthor=D.+W.+Christianson&title=Unusual+zinc-binding+mode+of+HDAC6-selective+hydroxamate+inhibitors&doi=10.1073%2Fpnas.1718823114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Unusual zinc-binding mode of HDAC6-selective hydroxamate inhibitors</span></div><div class="casAuthors">Porter, Nicholas J.; Mahendran, Adaickapillai; Breslow, Ronald; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">13459-13464</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) regulate myriad cellular processes by catalyzing the hydrolysis of acetyl-L-lysine residues in histone and nonhistone proteins.  The Zn2+-dependent class IIb enzyme HDAC6 regulates microtubule function by deacetylating α-tubulin, which suppresses microtubule dynamics and leads to cell cycle arrest and apoptosis.  Accordingly, HDAC6 is a target for the development of selective inhibitors that might be useful in new therapeutic approaches for the treatment of cancer, neurodegenerative diseases, and other disorders.  Here, we present high-resoln. structures of catalytic domain 2 from Danio rerio HDAC6 (henceforth simply "HDAC6") complexed with compds. that selectively inhibit HDAC6 while maintaining nanomolar inhibitory potency: N-hydroxy-4-((N(2-hydroxyethyl)-2-phenylacetamido)methyl)-benzamide (HPB), ACY-1215 (Ricolinostat), and ACY-1083.  These structures reveal that an unusual monodentate Zn2+ coordination mode is exploited by sterically bulky HDAC6-selective phenylhydroxamate inhibitors.  We addnl. report the ultrahigh-resoln. structure of the HDAC6-trichostatin A complex, which reveals two Zn2+-binding conformers for the inhibitor: a major conformer (70%) with canonical bidentate hydroxamate-Zn2+ coordination geometry and a minor conformer (30%) with monodentate hydroxamate-Zn2+ coordination geometry, reflecting a free energy difference of only 0.5 kcal/mol.  The minor conformer is not visible in lower resoln. structure detns.  Structural comparisons of HDAC6-inhibitor complexes with class I HDACs suggest active site features that contribute to the isoenzyme selectivity obsd. in biochem. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0oUH1X_04p7Vg90H21EOLACvtfcHk0ljfkr0bfZg55Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsrbI&md5=07180756239eac4891642cf59a2e6992</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1718823114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1718823114%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DMahendran%26aufirst%3DA.%26aulast%3DBreslow%26aufirst%3DR.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DUnusual%2520zinc-binding%2520mode%2520of%2520HDAC6-selective%2520hydroxamate%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D13459%26epage%3D13464%26doi%3D10.1073%2Fpnas.1718823114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnchembio.2134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27454933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=741-747&author=Y.+Haiauthor=D.+W.+Christianson&title=Histone+deacetylase+6+structure+and+molecular+basis+of+catalysis+and+inhibition&doi=10.1038%2Fnchembio.2134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span></div><div class="casAuthors">Hai, Yang; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">741-747</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a crit. target for drug design because of its role in oncogenic transformation and cancer metastasis, and is unique among all histone deacetylases in that it contains tandem catalytic domains designated CD1 and CD2.  We now report the crystal structures of CD2 from Homo sapiens HDAC6 and of CD1 and CD2 from Danio rerio HDAC6.  We correlated these structures with activity measurements using 13 different substrates.  The catalytic activity of CD2 from both species exhibited broad substrate specificity, whereas that of CD1 was highly specific for substrates bearing C-terminal acetyllysine residues.  Crystal structures of substrate complexes yielded unprecedented snapshots of the catalytic mechanism.  Addnl., crystal structures of complexes with eight different inhibitors, including belinostat and panobinostat (currently used in cancer chemotherapy), the macrocyclic tetrapeptide HC toxin, and the HDAC6-specific inhibitor N-hydroxy-4-(2-((2-hydroxyethyl)(phenyl)amino)-2-oxoethyl)benzamide, revealed surprising new insight regarding changes in Zn2+ coordination and isoenzyme-specific inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCfziGBHBRLVg90H21EOLACvtfcHk0ljfkr0bfZg55Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ&md5=e167bc6f66e00923da4f2bced2b306b9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2134%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520deacetylase%25206%2520structure%2520and%2520molecular%2520basis%2520of%2520catalysis%2520and%2520inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D741%26epage%3D747%26doi%3D10.1038%2Fnchembio.2134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diedrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooker, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 6-selective inhibitors and the influence of capping groups on hydroxamate-zinc denticity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8054</span>– <span class="NLM_lpage">8060</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFansLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8054-8060&author=N.+J.+Porterauthor=J.+D.+Oskoauthor=D.+Diedrichauthor=T.+Kurzauthor=J.+M.+Hookerauthor=F.+K.+Hansenauthor=D.+W.+Christianson&title=Histone+deacetylase+6-selective+inhibitors+and+the+influence+of+capping+groups+on+hydroxamate-zinc+denticity&doi=10.1021%2Facs.jmedchem.8b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase 6-Selective Inhibitors and the Influence of Capping Groups on Hydroxamate-Zinc Denticity</span></div><div class="casAuthors">Porter, Nicholas J.; Osko, Jeremy D.; Diedrich, Daniela; Kurz, Thomas; Hooker, Jacob M.; Hansen, Finn K.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8054-8060</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four crystal structures are presented of histone deacetylase 6 (HDAC6) complexes with para-substituted phenylhydromaxamate inhibitors, including bulky peptoids.  These structures provide insight regarding the design of capping groups that confer selectivity for binding to HDAC6, specifically with regard to interactions in a pocket formed by the L1 loop.  Capping group interactions may also influence hydroxamate-Zn2+ coordination with monodentate or bidentate geometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq54PM6p6wpnrVg90H21EOLACvtfcHk0ljfkr0bfZg55Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFansLnE&md5=f1d227d7668d09e121e5967b2d65f1a5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01013%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DDiedrich%26aufirst%3DD.%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26aulast%3DHansen%26aufirst%3DF.%2BK.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520deacetylase%25206-selective%2520inhibitors%2520and%2520the%2520influence%2520of%2520capping%2520groups%2520on%2520hydroxamate-zinc%2520denticity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8054%26epage%3D8060%26doi%3D10.1021%2Facs.jmedchem.8b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ustinova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlicek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonepalle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of a new isoxazole-3-hydroxamate-based histone deacetylase 6 inhibitor SS-208 with antitumor activity in syngeneic melanoma mouse models</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8557</span>– <span class="NLM_lpage">8577</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00946</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00946" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFymtbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8557-8577&author=S.+Shenauthor=M.+Hadleyauthor=K.+Ustinovaauthor=J.+Pavlicekauthor=T.+Knoxauthor=S.+Noonepalleauthor=M.+T.+Tavaresauthor=C.+A.+Zimprichauthor=G.+Zhangauthor=M.+B.+Robersauthor=C.+Ba%C5%99inkaauthor=A.+P.+Kozikowskiauthor=A.+Villagra&title=Discovery+of+a+new+isoxazole-3-hydroxamate-based+histone+deacetylase+6+inhibitor+SS-208+with+antitumor+activity+in+syngeneic+melanoma+mouse+models&doi=10.1021%2Facs.jmedchem.9b00946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a New Isoxazole-3-hydroxamate-Based Histone Deacetylase 6 Inhibitor SS-208 with Antitumor Activity in Syngeneic Melanoma Mouse Models</span></div><div class="casAuthors">Shen, Sida; Hadley, Melissa; Ustinova, Kseniya; Pavlicek, Jiri; Knox, Tessa; Noonepalle, Satish; Tavares, Mauricio T.; Zimprich, Chad A.; Zhang, Guiping; Robers, Matthew B.; Barinka, Cyril; Kozikowski, Alan P.; Villagra, Alejandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8557-8577</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isoxazole is a five-membered heterocycle that is widely used in drug discovery endeavors.  Here, the authors report the design, synthesis, and structural and biol. characterization of SS-208, a novel HDAC6-selective inhibitor contg. the isoxazole-3-hydroxamate moiety as a zinc-binding group as well as a hydrophobic linker.  A crystal structure of the Danio rerio HDAC6/SS-208 complex reveals a bidentate coordination of the active-site zinc ion that differs from the preferred monodentate coordination obsd. for HDAC6 complexes with phenylhydroxamate-based inhibitors.  While SS-208 has minimal effects on the viability of murine SM1 melanoma cells in vitro, it significantly reduced in vivo tumor growth in a murine SM1 syngeneic melanoma mouse model.  These findings suggest that the antitumor activity of SS-208 is mainly mediated by immune-related antitumor activity as evidenced by the increased infiltration of CD8+ and NK+ T cells and the enhanced ratio of M1 and M2 macrophages in the tumor microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUagVGHIiRHbVg90H21EOLACvtfcHk0liiFHO8agTYPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFymtbzI&md5=53b4c45ce582d8752869430407d44649</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00946%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DHadley%26aufirst%3DM.%26aulast%3DUstinova%26aufirst%3DK.%26aulast%3DPavlicek%26aufirst%3DJ.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DNoonepalle%26aufirst%3DS.%26aulast%3DTavares%26aufirst%3DM.%2BT.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520a%2520new%2520isoxazole-3-hydroxamate-based%2520histone%2520deacetylase%25206%2520inhibitor%2520SS-208%2520with%2520antitumor%2520activity%2520in%2520syngeneic%2520melanoma%2520mouse%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8557%26epage%3D8577%26doi%3D10.1021%2Facs.jmedchem.9b00946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vögerl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological investigation of phenothiazine-based benzhydroxamic acids as selective histone deacetylase 6 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1166</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01090</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01090" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BB3cjhs1Sltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1138-1166&author=K.+V%C3%B6gerlauthor=N.+Ongauthor=J.+Sengerauthor=D.+Herpauthor=K.+Schmidtkunzauthor=M.+Marekauthor=M.+M%C3%BCllerauthor=K.+Bartelauthor=T.+B.+Shaikauthor=N.+J.+Porterauthor=D.+Robaaauthor=D.+W.+Christiansonauthor=C.+Romierauthor=W.+Sipplauthor=M.+Jungauthor=F.+Bracher&title=Synthesis+and+biological+investigation+of+phenothiazine-based+benzhydroxamic+acids+as+selective+histone+deacetylase+6+inhibitors&doi=10.1021%2Facs.jmedchem.8b01090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Investigation of Phenothiazine-Based Benzhydroxamic Acids as Selective Histone Deacetylase 6 Inhibitors</span></div><div class="casAuthors">Vogerl Katharina; Ong Nghia; Muller Martin; Bartel Karin; Bracher Franz; Senger Johanna; Herp Daniel; Schmidtkunz Karin; Jung Manfred; Marek Martin; Shaik Tajith B; Romier Christophe; Porter Nicholas J; Christianson David W; Robaa Dina; Sippl Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1138-1166</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The phenothiazine system was identified as a favorable cap group for potent and selective histone deacetylase 6 (HDAC6) inhibitors.  Here, we report the preparation and systematic variation of phenothiazines and their analogues containing a benzhydroxamic acid moiety as the zinc-binding group.  We evaluated their ability to selectively inhibit HDAC6 by a recombinant HDAC enzyme assay, by determining the protein acetylation levels in cells by western blotting (tubulin vs histone acetylation), and by assessing their effects on various cancer cell lines.  Structure-activity relationship studies revealed that incorporation of a nitrogen atom into the phenothiazine framework results in increased potency and selectivity for HDAC6 (more than 500-fold selectivity relative to the inhibition of HDAC1, HDAC4, and HDAC8), as rationalized by molecular modeling and docking studies.  The binding mode was confirmed by co-crystallization of the potent azaphenothiazine inhibitor with catalytic domain 2 from Danio rerio HDAC6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkSJzGKNxKWK69lAbTj9rGfW6udTcc2eZmloy2tR1f2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjhs1Sltw%253D%253D&md5=9d72da7b8f4bb63cd05ccc5f907d2dca</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01090%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25B6gerl%26aufirst%3DK.%26aulast%3DOng%26aufirst%3DN.%26aulast%3DSenger%26aufirst%3DJ.%26aulast%3DHerp%26aufirst%3DD.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%26aulast%3DBartel%26aufirst%3DK.%26aulast%3DShaik%26aufirst%3DT.%2BB.%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBracher%26aufirst%3DF.%26atitle%3DSynthesis%2520and%2520biological%2520investigation%2520of%2520phenothiazine-based%2520benzhydroxamic%2520acids%2520as%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1138%26epage%3D1166%26doi%3D10.1021%2Facs.jmedchem.8b01090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svoboda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavasin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motlova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinsey, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eubanks, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Structural and in vivo characterization of Tubastatin A, a widely used histone deacetylase 6 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00560</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00560" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFarsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=706-712&author=S.+Shenauthor=M.+Svobodaauthor=G.+Zhangauthor=M.+A.+Cavasinauthor=L.+Motlovaauthor=T.+A.+McKinseyauthor=J.+H.+Eubanksauthor=C.+Ba%C5%99inkaauthor=A.+P.+Kozikowski&title=Structural+and+in+vivo+characterization+of+Tubastatin+A%2C+a+widely+used+histone+deacetylase+6+inhibitor&doi=10.1021%2Facsmedchemlett.9b00560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and in Vivo Characterization of Tubastatin A, a Widely Used Histone Deacetylase 6 Inhibitor</span></div><div class="casAuthors">Shen, Sida; Svoboda, Michal; Zhang, Guangming; Cavasin, Maria A.; Motlova, Lucia; McKinsey, Timothy A.; Eubanks, James H.; Barinka, Cyril; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">706-712</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tubastatin A, a tetrahydro-γ-carboline-capped selective HDAC6 inhibitor (HDAC6i), was rationally designed 10 years ago, and has become the best investigated HDAC6i to date.  It shows efficacy in various neurol. disease animal models, as HDAC6 plays a crucial regulatory role in axonal transport deficits, protein aggregation, as well as oxidative stress.  In this work, we provide new insights into this HDAC6i by investigating the mol. basis of its interactions with HDAC6 through X-ray crystallog., detg. its functional capability to elevate the levels of acetylated α-tubulin in vitro and in vivo, correlating PK/PD profiles to det. EDs in plasma and brain, and finally assessing its therapeutic potential toward psychiatric diseases through use of the SmartCube screening platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi67K0VhCsP7Vg90H21EOLACvtfcHk0lgCGb9ew11bRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFarsr8%253D&md5=431ff199537a679fc09f19f95e528219</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00560%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DSvoboda%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DMotlova%26aufirst%3DL.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26aulast%3DEubanks%26aufirst%3DJ.%2BH.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DStructural%2520and%2520in%2520vivo%2520characterization%2520of%2520Tubastatin%2520A%252C%2520a%2520widely%2520used%2520histone%2520deacetylase%25206%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D706%26epage%3D712%26doi%3D10.1021%2Facsmedchemlett.9b00560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayana
Reddy, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooker, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Exploring structural determinants of inhibitor affinity and selectivity in complexes with histone deacetylase 6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01540</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit12jsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=295-308&author=J.+D.+Oskoauthor=N.+J.+Porterauthor=P.+A.+Narayana%0AReddyauthor=Y.+C.+Xiaoauthor=J.+Rokkaauthor=M.+Jungauthor=J.+M.+Hookerauthor=J.+M.+Salvinoauthor=D.+W.+Christianson&title=Exploring+structural+determinants+of+inhibitor+affinity+and+selectivity+in+complexes+with+histone+deacetylase+6&doi=10.1021%2Facs.jmedchem.9b01540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Structural Determinants of Inhibitor Affinity and Selectivity in Complexes with Histone Deacetylase 6</span></div><div class="casAuthors">Osko, Jeremy D.; Porter, Nicholas J.; Narayana Reddy, Poli Adi; Xiao, You-Cai; Rokka, Johanna; Jung, Manfred; Hooker, Jacob M.; Salvino, Joseph M.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">295-308</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of histone deacetylase 6 (HDAC6) has emerged as a promising therapeutic strategy for the treatment of cancer, chemotherapy-induced peripheral neuropathy, and neurodegenerative disease.  The recent X-ray crystal structure detn. of HDAC6 enables an understanding of structural features directing affinity and selectivity in the active site.  Here, we present the X-ray crystal structures of five HDAC6-inhibitor complexes that illuminate key mol. features of the inhibitor linker and capping groups that facilitate and differentiate binding to HDAC6.  In particular, arom. and heteroarom. linkers nestle within an arom. cleft defined by F583 and F643, and different arom. linkers direct the capping group toward shallow pockets defined by the L1 loop, the L2 loop, or somewhere in between these pockets.  These results expand our understanding of factors contributing to the selective inhibition of HDAC6, particularly regarding interactions that can be targeted in the region of the L2 pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBosfiCC-DubVg90H21EOLACvtfcHk0lgCGb9ew11bRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit12jsr3N&md5=0ebd3729b8a0ed03629c4ecca475b02e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01540%26sid%3Dliteratum%253Aachs%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DNarayana%2BReddy%26aufirst%3DP.%2BA.%26aulast%3DXiao%26aufirst%3DY.%2BC.%26aulast%3DRokka%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26aulast%3DSalvino%26aufirst%3DJ.%2BM.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DExploring%2520structural%2520determinants%2520of%2520inhibitor%2520affinity%2520and%2520selectivity%2520in%2520complexes%2520with%2520histone%2520deacetylase%25206%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D295%26epage%3D308%26doi%3D10.1021%2Facs.jmedchem.9b01540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krieger, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ressing, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkhardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of the first-in-class dual histone deacetylase-proteasome inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10299</span>– <span class="NLM_lpage">10309</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01487</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01487" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10299-10309&author=S.+Bhatiaauthor=V.+Kriegerauthor=M.+Grollauthor=J.+D.+Oskoauthor=N.+Ressingauthor=H.+Ahlertauthor=A.+Borkhardtauthor=T.+Kurzauthor=D.+W.+Christiansonauthor=J.+Hauerauthor=F.+K.+Hansen&title=Discovery+of+the+first-in-class+dual+histone+deacetylase-proteasome+inhibitor&doi=10.1021%2Facs.jmedchem.8b01487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the First-in-Class Dual Histone Deacetylase-Proteasome Inhibitor</span></div><div class="casAuthors">Bhatia, Sanil; Krieger, Viktoria; Groll, Michael; Osko, Jeremy D.; Ressing, Nina; Ahlert, Heinz; Borkhardt, Arndt; Kurz, Thomas; Christianson, David W.; Hauer, Julia; Hansen, Finn K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10299-10309</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual- or multitarget drugs have emerged as a promising alternative to combination therapies.  Proteasome inhibitors (PIs) possess synergistic activity with histone deacetylase (HDAC) inhibitors due to the simultaneous blockage of the ubiquitin degrdn. and aggresome pathways.  Here, the authors present the design, synthesis, binding modes, and anticancer properties of RTS-V5 as the first-in-class dual HDAC-proteasome ligand.  The inhibition of both targets was confirmed by biochem. and cellular assays as well as x-ray crystal structures of the 20S proteasome and HDAC6 complexed with RTS-V5.  Cytotoxicity assays with leukemia and multiple myeloma cell lines as well as therapy refractory primary patient-derived leukemia cells demonstrated that RTS-V5 possesses potent and selective anticancer activity.  The results will thus guide the structure-based optimization of dual HDAC-proteasome inhibitors for the treatment of hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwJMqcDlzXA7Vg90H21EOLACvtfcHk0lgCGb9ew11bRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K&md5=7d47e664bd77ac1bc5ebd39ea6eed02a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01487%26sid%3Dliteratum%253Aachs%26aulast%3DBhatia%26aufirst%3DS.%26aulast%3DKrieger%26aufirst%3DV.%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DRessing%26aufirst%3DN.%26aulast%3DAhlert%26aufirst%3DH.%26aulast%3DBorkhardt%26aufirst%3DA.%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DHauer%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DF.%2BK.%26atitle%3DDiscovery%2520of%2520the%2520first-in-class%2520dual%2520histone%2520deacetylase-proteasome%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10299%26epage%3D10309%26doi%3D10.1021%2Facs.jmedchem.8b01487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Structural determinants of affinity and selectivity in the binding of inhibitors to histone deacetylase 6</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127023</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.bmcl.2020.127023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=32067866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFGqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127023&author=J.+D.+Oskoauthor=D.+W.+Christianson&title=Structural+determinants+of+affinity+and+selectivity+in+the+binding+of+inhibitors+to+histone+deacetylase+6&doi=10.1016%2Fj.bmcl.2020.127023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of affinity and selectivity in the binding of inhibitors to histone deacetylase 6</span></div><div class="casAuthors">Osko, Jeremy D.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">127023</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review. Digest.  Histone deacetylase 6 (HDAC6) is assocd. with multiple neurol. disorders as well as aggressive cancers, making its selective inhibition highly desirable for therapeutic purposes.  The basic mol. design of an effective HDAC6 inhibitor consists of a zinc-binding group, a linker, and a capping group capable of making interactions at the mouth of the active site.  To date, more than 50 high-resoln. X-ray crystal structures of HDAC6-inhibitor complexes have been reported, many of which reveal intermol. interactions that contribute to isoenzyme affinity and selectivity.  Here, we review the key features of HDAC6 inhibitor design illuminated by these structural studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEHMdvmF9TPLVg90H21EOLACvtfcHk0ljHPrSLfx0cAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFGqs70%253D&md5=df46fa5e4720d8340d728e6269dadb90</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127023%26sid%3Dliteratum%253Aachs%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructural%2520determinants%2520of%2520affinity%2520and%2520selectivity%2520in%2520the%2520binding%2520of%2520inhibitors%2520to%2520histone%2520deacetylase%25206%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127023%26doi%3D10.1016%2Fj.bmcl.2020.127023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vergani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandrone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ripamonti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cellupica, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbiati, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caprini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocchio, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lattanzio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pezzuto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skorupska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomarico, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modena, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perego, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkühler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenazzi, A.</span></span> <span> </span><span class="NLM_article-title">Novel benzohydroxamate-based potent and selective histone deacetylase 6 (HDAC6) inhibitors bearing a pentaheterocyclic scaffold: Design, synthesis, and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10711</span>– <span class="NLM_lpage">10739</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01194</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01194" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFeqs77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10711-10739&author=B.+Verganiauthor=G.+Sandroneauthor=M.+Marchiniauthor=C.+Ripamontiauthor=E.+Cellupicaauthor=E.+Galbiatiauthor=G.+Capriniauthor=G.+Pavichauthor=G.+Porroauthor=I.+Rocchioauthor=M.+Lattanzioauthor=M.+Pezzutoauthor=M.+Skorupskaauthor=P.+Cordellaauthor=P.+Paganiauthor=P.+Pozziauthor=R.+Pomaricoauthor=D.+Modenaauthor=F.+Leoniauthor=R.+Peregoauthor=G.+Fossatiauthor=C.+Steink%C3%BChlerauthor=A.+Stevenazzi&title=Novel+benzohydroxamate-based+potent+and+selective+histone+deacetylase+6+%28HDAC6%29+inhibitors+bearing+a+pentaheterocyclic+scaffold%3A+Design%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Facs.jmedchem.9b01194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Benzohydroxamate-Based Potent and Selective Histone Deacetylase 6 (HDAC6) Inhibitors Bearing a Pentaheterocyclic Scaffold: Design, Synthesis, and Biological Evaluation</span></div><div class="casAuthors">Vergani, Barbara; Sandrone, Giovanni; Marchini, Mattia; Ripamonti, Chiara; Cellupica, Edoardo; Galbiati, Elisabetta; Caprini, Gianluca; Pavich, Gianfranco; Porro, Giulia; Rocchio, Ilaria; Lattanzio, Maria; Pezzuto, Marcello; Skorupska, Malgorzata; Cordella, Paola; Pagani, Paolo; Pozzi, Pietro; Pomarico, Roberta; Modena, Daniela; Leoni, Flavio; Perego, Raffaella; Fossati, Gianluca; Steinkuhler, Christian; Stevenazzi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10711-10739</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a peculiar HDAC isoform whose expression and functional alterations were correlated with a variety of pathologies such as autoimmune disorders, neurodegenerative diseases, and cancer.  It is primarily a cytoplasmic protein, and its deacetylase activity is focused mainly on non-histone substrates such as tubulin, heat shock protein (HSP)90, Foxp3 and cortactin, to name a few.  Selective inhibition of HDAC6 does not show cytotoxic effects in healthy cells, normally assocd. with the inhibition of Class I HDAC isoforms.  Here the authors describe the design and synthesis of a new class of potent and selective HDAC6 inhibitors that bear a pentaheterocyclic central core.  These compds. show a remarkably low toxicity both in vitro and in vivo and are able to increase the function of regulatory T cells (Tregs) at well tolerated concns., suggesting a potential clin. use for the treatment of degenerative, auto-immune diseases and organ transplantation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV3O384uJCkLVg90H21EOLACvtfcHk0ljHPrSLfx0cAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFeqs77I&md5=efc978e814193713983d2239a821cfd7</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01194%26sid%3Dliteratum%253Aachs%26aulast%3DVergani%26aufirst%3DB.%26aulast%3DSandrone%26aufirst%3DG.%26aulast%3DMarchini%26aufirst%3DM.%26aulast%3DRipamonti%26aufirst%3DC.%26aulast%3DCellupica%26aufirst%3DE.%26aulast%3DGalbiati%26aufirst%3DE.%26aulast%3DCaprini%26aufirst%3DG.%26aulast%3DPavich%26aufirst%3DG.%26aulast%3DPorro%26aufirst%3DG.%26aulast%3DRocchio%26aufirst%3DI.%26aulast%3DLattanzio%26aufirst%3DM.%26aulast%3DPezzuto%26aufirst%3DM.%26aulast%3DSkorupska%26aufirst%3DM.%26aulast%3DCordella%26aufirst%3DP.%26aulast%3DPagani%26aufirst%3DP.%26aulast%3DPozzi%26aufirst%3DP.%26aulast%3DPomarico%26aufirst%3DR.%26aulast%3DModena%26aufirst%3DD.%26aulast%3DLeoni%26aufirst%3DF.%26aulast%3DPerego%26aufirst%3DR.%26aulast%3DFossati%26aufirst%3DG.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26aulast%3DStevenazzi%26aufirst%3DA.%26atitle%3DNovel%2520benzohydroxamate-based%2520potent%2520and%2520selective%2520histone%2520deacetylase%25206%2520%2528HDAC6%2529%2520inhibitors%2520bearing%2520a%2520pentaheterocyclic%2520scaffold%253A%2520Design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10711%26epage%3D10739%26doi%3D10.1021%2Facs.jmedchem.9b01194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Selective histone deacetylase 6 inhibitors bearing substituted urea linkers inhibit melanoma cell growth</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9891</span>– <span class="NLM_lpage">9899</span>, <span class="refDoi"> DOI: 10.1021/jm301098e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301098e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9891-9899&author=J.+A.+Bergmanauthor=K.+Woanauthor=P.+Perez-Villarroelauthor=A.+Villagraauthor=E.+M.+Sotomayorauthor=A.+P.+Kozikowski&title=Selective+histone+deacetylase+6+inhibitors+bearing+substituted+urea+linkers+inhibit+melanoma+cell+growth&doi=10.1021%2Fjm301098e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Histone Deacetylase 6 Inhibitors Bearing Substituted Urea Linkers Inhibit Melanoma Cell Growth</span></div><div class="casAuthors">Bergman, Joel A.; Woan, Karrune; Perez-Villarroel, Patricio; Villagra, Alejandro; Sotomayor, Eduardo M.; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9891-9899</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The incidence of malignant melanoma has dramatically increased in recent years thus requiring the need for improved therapeutic strategies.  In our efforts to design selective histone deactylase inhibitors (HDACI), we discovered that the aryl urea 1 is a modestly potent yet nonselective inhibitor.  Structure-activity relationship studies revealed that adding substituents to the nitrogen atom of the urea so as to generate compds. bearing a branched linker group results in increased potency and selectivity for HDAC6.  Compd. 5g shows low nanomolar inhibitory potency against HDAC6 and a selectivity of ∼600-fold relative to the inhibition of HDAC1.  These HDACIs were evaluated for their ability to inhibit the growth of B16 melanoma cells with the most potent and selective HDAC6I being found to decrease tumor cell growth.  To the best of our knowledge, this work constitutes the first report of HDAC6-selective inhibitors that possess antiproliferative effects against melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTtPLqvwsrVg90H21EOLACvtfcHk0ljHPrSLfx0cAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ&md5=d6be4c0efb3922fc35d814ba00ae9ed1</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm301098e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301098e%26sid%3Dliteratum%253Aachs%26aulast%3DBergman%26aufirst%3DJ.%2BA.%26aulast%3DWoan%26aufirst%3DK.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DSelective%2520histone%2520deacetylase%25206%2520inhibitors%2520bearing%2520substituted%2520urea%2520linkers%2520inhibit%2520melanoma%2520cell%2520growth%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9891%26epage%3D9899%26doi%3D10.1021%2Fjm301098e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villaroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canales, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marante, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Celis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Targeting histone deacetylase 6 mediates a dual anti-melanoma effect: Enhanced antitumor immunity and impaired cell proliferation</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1447</span>– <span class="NLM_lpage">1457</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2015.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.molonc.2015.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25957812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=1447-1457&author=K.+V.+Woanauthor=M.+Lienlafauthor=P.+Perez-Villaroelauthor=C.+Leeauthor=F.+Chengauthor=T.+Knoxauthor=D.+M.+Woodsauthor=K.+Barriosauthor=J.+Powersauthor=E.+Sahakianauthor=H.+W.+Wangauthor=J.+Canalesauthor=D.+Maranteauthor=K.+S.+M.+Smalleyauthor=J.+Bergmanauthor=E.+Setoauthor=A.+Kozikowskiauthor=J.+Pinilla-Ibarzauthor=A.+Sarnaikauthor=E.+Celisauthor=J.+Weberauthor=E.+M.+Sotomayorauthor=A.+Villagra&title=Targeting+histone+deacetylase+6+mediates+a+dual+anti-melanoma+effect%3A+Enhanced+antitumor+immunity+and+impaired+cell+proliferation&doi=10.1016%2Fj.molonc.2015.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone deacetylase 6 mediates a dual anti-melanoma effect: Enhanced antitumor immunity and impaired cell proliferation</span></div><div class="casAuthors">Woan, K. V.; Lienlaf, M.; Perez-Villaroel, P.; Lee, C.; Cheng, F.; Knox, T.; Woods, D. M.; Barrios, K.; Powers, J.; Sahakian, E.; Wang, H. W.; Canales, J.; Marante, D.; Smalley, K. S. M.; Bergman, J.; Seto, E.; Kozikowski, A.; Pinilla-Ibarz, J.; Sarnaik, A.; Celis, E.; Weber, J.; Sotomayor, E. M.; Villagra, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1447-1457</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The median survival for metastatic melanoma is in the realm of 8-16 mo and there are few therapies that offer significant improvement in overall survival.  One of the recent advances in cancer treatment focuses on epigenetic modifiers to alter the survivability and immunogenicity of cancer cells.  Our group and others have previously demonstrated that pan-HDAC inhibitors induce apoptosis, cell cycle arrest and changes in the immunogenicity of melanoma cells.  Here we interrogated specific HDACs which may be responsible for this effect.  We found that both genetic abrogation and pharmacol. inhibition of HDAC6 decreases in vitro proliferation and induces G1 arrest of melanoma cell lines without inducing apoptosis.  Moreover, targeting this mol. led to an important upregulation in the expression of tumor assocd. antigens and MHC class I, suggesting a potential improvement in the immunogenicity of these cells.  Of note, this anti-melanoma activity was operative regardless of mutational status of the cells.  These effects translated into a pronounced delay of in vivo melanoma tumor growth which was, at least in part, dependent on intact immunity as evidenced by the restoration of tumor growth after CD4+ and CD8+ depletion.  Given our findings, we provide the initial rationale for the further development of selective HDAC6 inhibitors as potential therapeutic anti-melanoma agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDIQ5LKbZF8bVg90H21EOLACvtfcHk0lhb-T9bSJKaCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D&md5=7534df9db01f52a71f97f0e112db37cc</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2015.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2015.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DWoan%26aufirst%3DK.%2BV.%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DPerez-Villaroel%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DWoods%26aufirst%3DD.%2BM.%26aulast%3DBarrios%26aufirst%3DK.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DH.%2BW.%26aulast%3DCanales%26aufirst%3DJ.%26aulast%3DMarante%26aufirst%3DD.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DBergman%26aufirst%3DJ.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DSarnaik%26aufirst%3DA.%26aulast%3DCelis%26aufirst%3DE.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DTargeting%2520histone%2520deacetylase%25206%2520mediates%2520a%2520dual%2520anti-melanoma%2520effect%253A%2520Enhanced%2520antitumor%2520immunity%2520and%2520impaired%2520cell%2520proliferation%26jtitle%3DMol.%2520Oncol.%26date%3D2015%26volume%3D9%26spage%3D1447%26epage%3D1457%26doi%3D10.1016%2Fj.molonc.2015.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sixto-López, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correa-Basurto, J.</span></span> <span> </span><span class="NLM_article-title">Structural and energetic basis for the inhibitory selectivity of both catalytic domains of dimeric HDAC6</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">4701</span>– <span class="NLM_lpage">4720</span>, <span class="refDoi"> DOI: 10.1080/07391102.2018.1557560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1080%2F07391102.2018.1557560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=30558483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVaisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=4701-4720&author=Y.+Sixto-L%C3%B3pezauthor=M.+Belloauthor=J.+Correa-Basurto&title=Structural+and+energetic+basis+for+the+inhibitory+selectivity+of+both+catalytic+domains+of+dimeric+HDAC6&doi=10.1080%2F07391102.2018.1557560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and energetic basis for the inhibitory selectivity of both catalytic domains of dimeric HDAC6</span></div><div class="casAuthors">Sixto-Lopez, Yudibeth; Bello, Martiniano; Correa-Basurto, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure and Dynamics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4701-4720</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">HDAC6 is a protein involved in cancer, neurodegenerative disease and inflammatory disorders.  To date, the full three-dimensional (3D) structure of human HDAC6 has not been elucidated; however, there are some exptl. 3D structural homologs to HDAC6 that can be used as templates.  In this work, we utilized mol. modeling procedures to model both of the catalytic domains of HDAC6 connected by the linker region where DMB region is placed.  Once the 3D structure of human HDAC6 was obtained, it was structurally evaluated and submitted to docking and mol. dynamic (MD) simulations along with Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) method to explore the stability and the binding free energy properties of the HDAC6-ligand complexes.  In addn., its structural and energetic behavior was explored with each one of the catalytic domains in the mol. recognition of six selective HDAC6 inhibitors, HPOB, CAY10603, Nexturastat, Rocilinostat, Tubacin and Tubastatin A for DD2, and with the so-called 9-peptide which is DD1-HDAC6 selective substrate.  The use of the whole system (DD1-DMB-DD2) showed a tendency toward the ligand affinity of DD2, CAY10603> Tubacin > Rocilinostat > Nexturastat > HPOB > Tubastatin > 9-peptide, which is in line with exptl. reports.  However, 9-peptide showed a higher affinity for DD1, which agrees with exptl. reports elsewhere.  Principal component anal. provided important information about the structural changes linked to the mol. recognition process, whereas per-residue decompn. anal. revealed the energetic contribution of the key residues in the mol. binding and structural characteristics that could assist in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaDDSCE5cAdLVg90H21EOLACvtfcHk0lhb-T9bSJKaCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVaisLg%253D&md5=7029672cda1c077774128f90422209d3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F07391102.2018.1557560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2018.1557560%26sid%3Dliteratum%253Aachs%26aulast%3DSixto-L%25C3%25B3pez%26aufirst%3DY.%26aulast%3DBello%26aufirst%3DM.%26aulast%3DCorrea-Basurto%26aufirst%3DJ.%26atitle%3DStructural%2520and%2520energetic%2520basis%2520for%2520the%2520inhibitory%2520selectivity%2520of%2520both%2520catalytic%2520domains%2520of%2520dimeric%2520HDAC6%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2019%26volume%3D37%26spage%3D4701%26epage%3D4720%26doi%3D10.1080%2F07391102.2018.1557560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Synthesis and pharmacological evaluation of selective histone deacetylase 6 inhibitors in melanoma models</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1036</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00223</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00223" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1031-1036&author=M.+T.+Tavaresauthor=S.+Shenauthor=T.+Knoxauthor=M.+Hadleyauthor=Z.+Kutilauthor=C.+Ba%C5%99inkaauthor=A.+Villagraauthor=A.+P.+Kozikowski&title=Synthesis+and+pharmacological+evaluation+of+selective+histone+deacetylase+6+inhibitors+in+melanoma+models&doi=10.1021%2Facsmedchemlett.7b00223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Pharmacological Evaluation of Selective Histone Deacetylase 6 Inhibitors in Melanoma Models</span></div><div class="casAuthors">Tavares, Mauricio T.; Shen, Sida; Knox, Tessa; Hadley, Melissa; Kutil, Zsofia; Barinka, Cyril; Villagra, Alejandro; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1031-1036</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Only a handful of therapies offer significant improvement in the overall survival in cases of melanoma, a cancer whose incidence has continued to rise in the past 30 years.  In our effort to identify potent and isoform-selective histone deacetylase (HDAC) inhibitors as a therapeutic approach to melanoma, a series of new HDAC6 inhibitors based on the nexturastat A scaffold were prepd.  The new analogs bearing added hydrophilic substituents, so as to establish addnl. hydrogen bonding on the rim of the HDAC6 catalytic pocket, exhibit improved potency against HDAC6 and retain selectivity over HDAC1.  Compd. I exhibits antiproliferative effects on several types of melanoma and lymphoma cells.  Further studies indicates that I selectively increases acetylated tubulin levels in vitro and elicits an immune response through down-regulating cytokine IL-10.  A preliminary in vivo efficacy study indicates that I possesses improved capability to inhibit melanoma tumor growth and that this effect is based on the regulation of inflammatory and immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6cF7frE1ffLVg90H21EOLACvtfcHk0lgo_Ygv7pyQPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7zL&md5=305123b6d7ecc3e4273d5fa02565367f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00223%26sid%3Dliteratum%253Aachs%26aulast%3DTavares%26aufirst%3DM.%2BT.%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DHadley%26aufirst%3DM.%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DSynthesis%2520and%2520pharmacological%2520evaluation%2520of%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%2520in%2520melanoma%2520models%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1031%26epage%3D1036%26doi%3D10.1021%2Facsmedchemlett.7b00223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Francesco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz, O. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinzel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesiti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteagudo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pescatore, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdirame, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkühler, C.</span></span> <span> </span><span class="NLM_article-title">A novel series of potent and selective ketone histone deacetylase inhibitors with antitumor activity in vivo</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2350</span>– <span class="NLM_lpage">2353</span>, <span class="refDoi"> DOI: 10.1021/jm800079s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800079s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvFeisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2350-2353&author=P.+Jonesauthor=S.+Altamuraauthor=R.+De+Francescoauthor=O.+G.+Pazauthor=O.+Kinzelauthor=G.+Mesitiauthor=E.+Monteagudoauthor=G.+Pescatoreauthor=M.+Rowleyauthor=M.+Verdirameauthor=C.+Steink%C3%BChler&title=A+novel+series+of+potent+and+selective+ketone+histone+deacetylase+inhibitors+with+antitumor+activity+in+vivo&doi=10.1021%2Fjm800079s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A novel series of potent and selective ketone histone deacetylase inhibitors with antitumor activity in vivo</span></div><div class="casAuthors">Jones, Philip; Altamura, Sergio; De Francesco, Raffaele; Gonzalez Paz, Odalys; Kinzel, Olaf; Mesiti, Giuseppe; Monteagudo, Edith; Pescatore, Giovanna; Rowley, Michael; Verdirame, Maria; Steinkuhler, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2350-2353</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors offer a promising strategy for cancer therapy, and the first generation HDAC inhibitors are currently in the clinic.  Entirely novel ketone HDAC inhibitors have been developed from the cyclic tetrapeptide apicidin.  These compds. show class I subtype selectivity and levels of cellular activity comparable to clin. candidates.  A representative example has demonstrated tumor growth inhibition in a human colon HCT-116 carcinoma xenograft model comparable to known inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCTv5T7lu1nbVg90H21EOLACvtfcHk0lgo_Ygv7pyQPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvFeisr8%253D&md5=4701ba3c4a008d66664009b358c4fc9a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm800079s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800079s%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%26aulast%3DAltamura%26aufirst%3DS.%26aulast%3DDe%2BFrancesco%26aufirst%3DR.%26aulast%3DPaz%26aufirst%3DO.%2BG.%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DMesiti%26aufirst%3DG.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DRowley%26aufirst%3DM.%26aulast%3DVerdirame%26aufirst%3DM.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26atitle%3DA%2520novel%2520series%2520of%2520potent%2520and%2520selective%2520ketone%2520histone%2520deacetylase%2520inhibitors%2520with%2520antitumor%2520activity%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2350%26epage%3D2353%26doi%3D10.1021%2Fjm800079s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinzel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llauger-Bufi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pescatore, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteagudo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonsi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez Paz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiore, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkühler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I selective ketone histone deacetylase inhibitor with antitumor activity in vivo and optimized pharmacokinetic properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3453</span>– <span class="NLM_lpage">3456</span>, <span class="refDoi"> DOI: 10.1021/jm9004303</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9004303" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFWnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=3453-3456&author=O.+Kinzelauthor=L.+Llauger-Bufiauthor=G.+Pescatoreauthor=M.+Rowleyauthor=C.+Schultz-Fademrechtauthor=E.+Monteagudoauthor=M.+Fonsiauthor=O.+Gonzalez+Pazauthor=F.+Fioreauthor=C.+Steink%C3%BChlerauthor=P.+Jones&title=Discovery+of+a+potent+class+I+selective+ketone+histone+deacetylase+inhibitor+with+antitumor+activity+in+vivo+and+optimized+pharmacokinetic+properties&doi=10.1021%2Fjm9004303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Selective Ketone Histone Deacetylase Inhibitor with Antitumor Activity in Vivo and Optimized Pharmacokinetic Properties</span></div><div class="casAuthors">Kinzel, Olaf; Llauger-Bufi, Laura; Pescatore, Giovanna; Rowley, Michael; Schultz-Fademrecht, Carsten; Monteagudo, Edith; Fonsi, Massimiliano; Gonzalez Paz, Odalys; Fiore, Fabrizio; Steinkuhler, Christian; Jones, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3453-3456</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The optimization of a potent, class I selective ketone HDAC inhibitor is shown.  It possesses optimized pharmacokinetic properties in preclin. species, has a clean off-target profile, and is neg. in a microbial mutagenicity (Ames) test.  In a mouse xenograft model it shows efficacy comparable to that of vorinostat at a 10-fold reduced dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHRRFS9TSYV7Vg90H21EOLACvtfcHk0lgo_Ygv7pyQPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFWnsb0%253D&md5=8674adc0c3892942a1fc7b154cbc84b4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm9004303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9004303%26sid%3Dliteratum%253Aachs%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DLlauger-Bufi%26aufirst%3DL.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DRowley%26aufirst%3DM.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DFonsi%26aufirst%3DM.%26aulast%3DGonzalez%2BPaz%26aufirst%3DO.%26aulast%3DFiore%26aufirst%3DF.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520selective%2520ketone%2520histone%2520deacetylase%2520inhibitor%2520with%2520antitumor%2520activity%2520in%2520vivo%2520and%2520optimized%2520pharmacokinetic%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D3453%26epage%3D3456%26doi%3D10.1021%2Fjm9004303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bresciani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ontoria, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biancofiore, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cellucci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciammaichella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrigno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malancona, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteagudo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nizi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pace, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossetti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veneziano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span> <span> </span><span class="NLM_article-title">Improved selective Class I HDAC and novel selective HDAC3 inhibitors: beyond hydroxamic acids and benzamides</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00517</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00517" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlals7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=481-486&author=A.+Brescianiauthor=J.+M.+Ontoriaauthor=I.+Biancofioreauthor=A.+Cellucciauthor=A.+Ciammaichellaauthor=A.+Di+Marcoauthor=F.+Ferrignoauthor=A.+Franconeauthor=S.+Malanconaauthor=E.+Monteagudoauthor=E.+Niziauthor=P.+Paceauthor=S.+Ponziauthor=I.+Rossettiauthor=M.+Venezianoauthor=V.+Summaauthor=S.+Harper&title=Improved+selective+Class+I+HDAC+and+novel+selective+HDAC3+inhibitors%3A+beyond+hydroxamic+acids+and+benzamides&doi=10.1021%2Facsmedchemlett.8b00517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Improved selective class I HDAC and novel selective HDAC3 inhibitors: Beyond hydroxamic acids and benzamides</span></div><div class="casAuthors">Bresciani, Alberto; Ontoria, Jesus M.; Biancofiore, Ilaria; Cellucci, Antonella; Ciammaichella, Alina; Di Marco, Annalise; Ferrigno, Federica; Francone, Alessandra; Malancona, Savina; Monteagudo, Edith; Nizi, Emanuela; Pace, Paola; Ponzi, Simona; Rossetti, Ilaria; Veneziano, Maria; Summa, Vincenzo; Harper, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">481-486</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The application of class I HDAC inhibitors as cancer therapies is well established, but more recently their development for nononcol. indications has increased.  We report here on the generation of improved class I selective human HDAC inhibitors based on an ethylketone zinc binding group (ZBG) in place of the hydroxamic acid that features the majority of HDAC inhibitors.  We also describe a novel set of HDAC3 isoform selective inhibitors that show stronger potency and selectivity than the most commonly used HDAC3 selective tool compd. RGFP966.  These compds. are again based on an alternative ZBG with respect to the ortho-anilide that is featured in HDAC3 selective compds. reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIzwbKCL5A6bVg90H21EOLACvtfcHk0lgDFRnYvh5fzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlals7jO&md5=2882346cee38482adab18732afde3470</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00517%26sid%3Dliteratum%253Aachs%26aulast%3DBresciani%26aufirst%3DA.%26aulast%3DOntoria%26aufirst%3DJ.%2BM.%26aulast%3DBiancofiore%26aufirst%3DI.%26aulast%3DCellucci%26aufirst%3DA.%26aulast%3DCiammaichella%26aufirst%3DA.%26aulast%3DDi%2BMarco%26aufirst%3DA.%26aulast%3DFerrigno%26aufirst%3DF.%26aulast%3DFrancone%26aufirst%3DA.%26aulast%3DMalancona%26aufirst%3DS.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DNizi%26aufirst%3DE.%26aulast%3DPace%26aufirst%3DP.%26aulast%3DPonzi%26aufirst%3DS.%26aulast%3DRossetti%26aufirst%3DI.%26aulast%3DVeneziano%26aufirst%3DM.%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DHarper%26aufirst%3DS.%26atitle%3DImproved%2520selective%2520Class%2520I%2520HDAC%2520and%2520novel%2520selective%2520HDAC3%2520inhibitors%253A%2520beyond%2520hydroxamic%2520acids%2520and%2520benzamides%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D481%26epage%3D486%26doi%3D10.1021%2Facsmedchemlett.8b00517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinsky, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 10 structure and molecular function as a polyamine deacetylase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15368</span>, <span class="refDoi"> DOI: 10.1038/ncomms15368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fncomms15368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28516954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotFyisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15368&author=Y.+Haiauthor=S.+A.+Shinskyauthor=N.+J.+Porterauthor=D.+W.+Christianson&title=Histone+deacetylase+10+structure+and+molecular+function+as+a+polyamine+deacetylase&doi=10.1038%2Fncomms15368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 10 structure and molecular function as a polyamine deacetylase</span></div><div class="casAuthors">Hai, Yang; Shinsky, Stephen A.; Porter, Nicholas J.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15368</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cationic polyamines such as spermidine and spermine are crit. in all forms of life, as they regulate the function of biol. macromols.  Intracellular polyamine metab. is regulated by reversible acetylation and dysregulated polyamine metab. is assocd. with neoplastic diseases such as colon cancer, prostate cancer and neuroblastoma.  Here we report that histone deacetylase 10 (HDAC10) is a robust polyamine deacetylase, using recombinant enzymes from Homo sapiens (human) and Danio rerio (zebrafish).  The 2.85 Å-resoln. crystal structure of zebrafish HDAC10 complexed with a transition-state analog inhibitor reveals that a glutamate gatekeeper and a sterically constricted active site confer specificity for N8-acetylspermidine hydrolysis and disfavor acetyllysine hydrolysis.  Both HDAC10 and spermidine are known to promote cellular survival through autophagy.  Accordingly, this work sets a foundation for studying the chem. biol. of autophagy through the structure-based design of inhibitors that may also serve as new leads for cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa9laHMWjizLVg90H21EOLACvtfcHk0lgDFRnYvh5fzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotFyisrY%253D&md5=e192e4b219dcb0b46b05ece566f8a59d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms15368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15368%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DShinsky%26aufirst%3DS.%2BA.%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520deacetylase%252010%2520structure%2520and%2520molecular%2520function%2520as%2520a%2520polyamine%2520deacetylase%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15368%26doi%3D10.1038%2Fncomms15368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42717</span>, <span class="refDoi"> DOI: 10.1038/srep42717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fsrep42717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28256516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42717&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissADME%3A+a+free+web+tool+to+evaluate+pharmacokinetics%2C+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&doi=10.1038%2Fsrep42717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span></div><div class="casAuthors">Daina Antoine; Michielin Olivier; Zoete Vincent; Michielin Olivier; Michielin Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42717</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur.  Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited.  In that context, computer models constitute valid alternatives to experiments.  Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar.  Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch.  Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRl_Uk5dOMKs6qcneEtCi92fW6udTcc2ebG_DrcJ3x61bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D&md5=9715b8cb8a34b17c4c73ff69a5a8cc50</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fsrep42717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42717%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissADME%253A%2520a%2520free%2520web%2520tool%2520to%2520evaluate%2520pharmacokinetics%252C%2520drug-likeness%2520and%2520medicinal%2520chemistry%2520friendliness%2520of%2520small%2520molecules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42717%26doi%3D10.1038%2Fsrep42717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic efficiency as an important metric in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+efficiency+as+an+important+metric+in+drug+design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0ljUKAUx_X5KFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520efficiency%2520as%2520an%2520important%2520metric%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYpo81lIrrmQ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacketor, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comin-Anduix, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumeh, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minasyan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graeber, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span> <span> </span><span class="NLM_article-title">BRAF inhibitor vemurafenib improves the antitumor activity of adoptive cell immunotherapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3928</span>– <span class="NLM_lpage">3937</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-2837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1158%2F0008-5472.CAN-11-2837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=22693252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2rtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=3928-3937&author=R.+C.+Koyaauthor=S.+Mokauthor=N.+Otteauthor=K.+J.+Blacketorauthor=B.+Comin-Anduixauthor=P.+C.+Tumehauthor=A.+Minasyanauthor=N.+A.+Grahamauthor=T.+G.+Graeberauthor=T.+Chodonauthor=A.+Ribas&title=BRAF+inhibitor+vemurafenib+improves+the+antitumor+activity+of+adoptive+cell+immunotherapy&doi=10.1158%2F0008-5472.CAN-11-2837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF Inhibitor Vemurafenib Improves the Antitumor Activity of Adoptive Cell Immunotherapy</span></div><div class="casAuthors">Koya, Richard C.; Mok, Stephen; Otte, Nicholas; Blacketor, Kevin J.; Comin-Anduix, Begonya; Tumeh, Paul C.; Minasyan, Aspram; Graham, Nicholas A.; Graeber, Thomas G.; Chodon, Thinle; Ribas, Antoni</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3928-3937</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Targeted therapeutics that produce robust clin. responses are likely benefiting from inherent adjuvant effects that degrade immune escape, a feature that could be leveraged most effectively by combinatorial uses in settings such as that described here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg6bb5bmox1bVg90H21EOLACvtfcHk0ljUKAUx_X5KFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2rtr3J&md5=d8ec5438042feca327a3227c6d2a2381</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-2837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-2837%26sid%3Dliteratum%253Aachs%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DMok%26aufirst%3DS.%26aulast%3DOtte%26aufirst%3DN.%26aulast%3DBlacketor%26aufirst%3DK.%2BJ.%26aulast%3DComin-Anduix%26aufirst%3DB.%26aulast%3DTumeh%26aufirst%3DP.%2BC.%26aulast%3DMinasyan%26aufirst%3DA.%26aulast%3DGraham%26aufirst%3DN.%2BA.%26aulast%3DGraeber%26aufirst%3DT.%2BG.%26aulast%3DChodon%26aufirst%3DT.%26aulast%3DRibas%26aufirst%3DA.%26atitle%3DBRAF%2520inhibitor%2520vemurafenib%2520improves%2520the%2520antitumor%2520activity%2520of%2520adoptive%2520cell%2520immunotherapy%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D3928%26epage%3D3937%26doi%3D10.1158%2F0008-5472.CAN-11-2837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, M. W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, M. J.</span></span> <span> </span><span class="NLM_article-title">Resistance to PD1/PDL1 checkpoint inhibition</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2016.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.ctrv.2016.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27951441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKntLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2017&pages=71-81&author=J.+S.+O%E2%80%99Donnellauthor=G.+V.+Longauthor=R.+A.+Scolyerauthor=M.+W.+L.+Tengauthor=M.+J.+Smyth&title=Resistance+to+PD1%2FPDL1+checkpoint+inhibition&doi=10.1016%2Fj.ctrv.2016.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to PD1/PDL1 checkpoint inhibition</span></div><div class="casAuthors">O'Donnell, Jake S.; Long, Georgina V.; Scolyer, Richard A.; Teng, Michele W. L.; Smyth, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-81</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">For the first time in decades, patients with difficult-to-treat cancers such as advanced stage metastatic melanoma are being offered a glimpse of hope in the form of immunotherapies.  By targeting factors that foster the development and maintenance of an immunosuppressive microenvironment within tumors, these therapies release the brakes on the host's own immune system; allowing cure of disease.  Indeed, phase III clin. trials have revealed that therapies such as ipilimumab and pembrolizumab which target the CTLA4 and PD-1 immune checkpoints, resp., have raised the three-year survival of patients with melanoma to ∼70%, and overall survival (>5 years) to ∼30%.  Despite this unprecedented efficacy, many patients fail to respond, and more concerning, some patients who demonstrate encouraging initial responses to immunotherapy, can acquire resistance over time.  There is now an urgent need to identify mechanisms of resistance, to predict outcome and to identify targets for combination therapy.  Here, with the aim of guiding future combination trials that target specific resistance mechanisms to immunotherapies, we have summarised and discussed the current understanding of mechanisms promoting resistance to anti-PD1/PDL1 therapies, and how combination strategies which target these pathways might yield better outcomes for patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWZg7kf5N4R7Vg90H21EOLACvtfcHk0lgBb5D3WmZ6OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKntLzE&md5=61bfcb71c4dcc6a79be5aa6c06b30c97</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2016.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2016.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DTeng%26aufirst%3DM.%2BW.%2BL.%26aulast%3DSmyth%26aufirst%3DM.%2BJ.%26atitle%3DResistance%2520to%2520PD1%252FPDL1%2520checkpoint%2520inhibition%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2017%26volume%3D52%26spage%3D71%26epage%3D81%26doi%3D10.1016%2Fj.ctrv.2016.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Entropy as a driver of selectivity for inhibitor binding to histone deacetylase 6</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3916</span>– <span class="NLM_lpage">3924</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00367</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00367" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps1Onsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=3916-3924&author=N.+J.+Porterauthor=F.+F.+Wagnerauthor=D.+W.+Christianson&title=Entropy+as+a+driver+of+selectivity+for+inhibitor+binding+to+histone+deacetylase+6&doi=10.1021%2Facs.biochem.8b00367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Entropy as a Driver of Selectivity for Inhibitor Binding to Histone Deacetylase 6</span></div><div class="casAuthors">Porter, Nicholas J.; Wagner, Florence F.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3916-3924</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among the metal-dependent histone deacetylases, the class IIb isoenzyme HDAC6 is remarkable due to its role in the regulation of microtubule dynamics in the cytosol.  Selective inhibition of HDAC6 results in microtubule hyperacetylation, leading to cell cycle arrest and apoptosis, which is a validated strategy for cancer chemotherapy and the treatment of other disorders.  HDAC6 inhibitors generally consist of a Zn2+-binding group such as a hydroxamate, a linker, and a capping group; the capping group is a crit. determinant of isoenzyme selectivity.  Surprisingly, however, even "capless" inhibitors exhibit appreciable HDAC6 selectivity.  To probe the chem. basis for this selectivity, we now report high-resoln. crystal structures of HDAC6 complexed with capless cycloalkylhydroxamate inhibitors 1-4.  Each inhibitor hydroxamate group coordinates to the catalytic Zn2+ ion with canonical bidentate geometry.  Addnl., the olefin moieties of compds. 2 and 4 bind in an arom. crevice between the side chains of F583 and F643.  Reasoning that similar binding could be achieved in the representative class I isoenzyme HDAC8, we employed isothermal titrn. calorimetry to study the thermodn. of inhibitor binding.  These measurements indicate that the entropy of inhibitor binding is generally pos. for binding to HDAC6 and neg. for binding to HDAC8, resulting in up to 313-fold selectivity for binding to HDAC6 relative to HDAC8.  Thus, favorable binding entropy contributes to HDAC6 selectivity.  Notably, cyclohexenylhydroxamate 2 represents a promising lead for derivatization with capping groups that may further enhance its impressive 313-fold thermodn. selectivity for HDAC6 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5MmbdoXXUqrVg90H21EOLACvtfcHk0lgBb5D3WmZ6OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps1Onsrg%253D&md5=0da8abf0ee7f012db41c620101551442</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00367%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DEntropy%2520as%2520a%2520driver%2520of%2520selectivity%2520for%2520inhibitor%2520binding%2520to%2520histone%2520deacetylase%25206%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D3916%26epage%3D3924%26doi%3D10.1021%2Facs.biochem.8b00367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seigneurin-Berny, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemercier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Identification of components of the murine histone deacetylase 6 complex: link between acetylation and ubiquitination signaling pathways</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">8035</span>– <span class="NLM_lpage">8044</span>, <span class="refDoi"> DOI: 10.1128/MCB.21.23.8035-8044.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1128%2FMCB.21.23.8035-8044.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=11689694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD3MXotl2gtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=8035-8044&author=D.+Seigneurin-Bernyauthor=A.+Verdelauthor=S.+Curtetauthor=C.+Lemercierauthor=J.+Garinauthor=S.+Rousseauxauthor=S.+Khochbin&title=Identification+of+components+of+the+murine+histone+deacetylase+6+complex%3A+link+between+acetylation+and+ubiquitination+signaling+pathways&doi=10.1128%2FMCB.21.23.8035-8044.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of components of the murine histone deacetylase 6 complex: link between acetylation and ubiquitination signaling pathways</span></div><div class="casAuthors">Seigneurin-Berny, Daphne; Verdel, Andre; Curtet, Sandrine; Lemercier, Claudie; Garin, Jerome; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8035-8044</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The immunopurifn. of the endogenous cytoplasmic murine histone deacetylase 6 (mHDAC6), a member of the class II HDACs, from mouse testis cytosolic exts. allowed the identification of two assocd. proteins.  Both were mammalian homologues of yeast proteins known to interact with each other and involved in the ubiquitin signaling pathway: p97/VCP/Cdc48p, a homolog of yeast Cdc48p, and phospholipase A2-activating protein, a homolog of yeast UFD3 (ubiquitin fusion degrdn. protein 3).  Moreover, in the C-terminal region of mHDAC6, a conserved zinc finger-contg. domain named ZnF-UBP, also present in several ubiquitin-specific proteases, was discovered and was shown to mediate the specific binding of ubiquitin by mHDAC6.  By using a ubiquitin pull-down approach, nine major ubiquitin-binding proteins were identified in mouse testis cytosolic exts., and mHDAC6 was found to be one of them.  All of these findings strongly suggest that mHDAC6 could be involved in the control of protein ubiquitination.  The investigation of biochem. properties of the mHDAC6 complex in vitro further supported this hypothesis and clearly established a link between protein acetylation and protein ubiquitination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr46PPvjekckbVg90H21EOLACvtfcHk0lgBb5D3WmZ6OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXotl2gtLs%253D&md5=82eccf722f2702e987e74d2fb5138c2b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2FMCB.21.23.8035-8044.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.21.23.8035-8044.2001%26sid%3Dliteratum%253Aachs%26aulast%3DSeigneurin-Berny%26aufirst%3DD.%26aulast%3DVerdel%26aufirst%3DA.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DLemercier%26aufirst%3DC.%26aulast%3DGarin%26aufirst%3DJ.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520components%2520of%2520the%2520murine%2520histone%2520deacetylase%25206%2520complex%253A%2520link%2520between%2520acetylation%2520and%2520ubiquitination%2520signaling%2520pathways%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2001%26volume%3D21%26spage%3D8035%26epage%3D8044%26doi%3D10.1128%2FMCB.21.23.8035-8044.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadoul, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pabion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">HDAC6, at the crossroads between cytoskeleton and cell signaling by acetylation and ubiquitination</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5468</span>– <span class="NLM_lpage">5476</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210614</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fsj.onc.1210614" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=17694087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFersrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=5468-5476&author=C.+Boyaultauthor=K.+Sadoulauthor=M.+Pabionauthor=S.+Khochbin&title=HDAC6%2C+at+the+crossroads+between+cytoskeleton+and+cell+signaling+by+acetylation+and+ubiquitination&doi=10.1038%2Fsj.onc.1210614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6, at the crossroads between cytoskeleton and cell signaling by acetylation and ubiquitination</span></div><div class="casAuthors">Boyault, C.; Sadoul, K.; Pabion, M.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5468-5476</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase 6 (HDAC6) is a unique enzyme with specific structural and functional features.  It is actively or stably maintained in the cytoplasm and is the only member, within the HDAC family, that harbors a full duplication of its deacetylase homol. region followed by a specific ubiquitin-binding domain at the C-terminus end.  Accordingly, this deacetylase functions at the heart of a cellular regulatory mechanism capable of coordinating various cellular functions largely relying on the microtubule network.  Moreover, HDAC6 action as a regulator of HSP90 chaperone activity adds to the multifunctionality of the protein, and allows the authors to propose a crit. role for HDAC6 in mediating and coordinating various cellular events in response to different stressful stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9IYNSqbx5pbVg90H21EOLACvtfcHk0liFJ6REslS_vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFersrw%253D&md5=3796fc81a6b403bf2c6d08122d3e94c8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210614%26sid%3Dliteratum%253Aachs%26aulast%3DBoyault%26aufirst%3DC.%26aulast%3DSadoul%26aufirst%3DK.%26aulast%3DPabion%26aufirst%3DM.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DHDAC6%252C%2520at%2520the%2520crossroads%2520between%2520cytoskeleton%2520and%2520cell%2520signaling%2520by%2520acetylation%2520and%2520ubiquitination%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D5468%26epage%3D5476%26doi%3D10.1038%2Fsj.onc.1210614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenhof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van%0ADer+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0liFJ6REslS_vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guvench, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+Mackerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0liFJ6REslS_vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vermaas, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajkhorshid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmeyer, A.</span></span> <span> </span><span class="NLM_article-title">TopoGromacs: automated topology conversion from CHARMM to GROMACS within VMD</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00103</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00103" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFKrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1112-1116&author=J.+V.+Vermaasauthor=D.+J.+Hardyauthor=J.+E.+Stoneauthor=E.+Tajkhorshidauthor=A.+Kohlmeyer&title=TopoGromacs%3A+automated+topology+conversion+from+CHARMM+to+GROMACS+within+VMD&doi=10.1021%2Facs.jcim.6b00103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">TopoGromacs: Automated Topology Conversion from CHARMM to GROMACS within VMD</span></div><div class="casAuthors">Vermaas, Josh V.; Hardy, David J.; Stone, John E.; Tajkhorshid, Emad; Kohlmeyer, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1112-1116</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulation engines use a variety of different approaches for modeling mol. systems with force fields that govern their dynamics and describe their topol.  These different approaches introduce incompatibilities between engines, and previously published software bridges the gaps between many popular MD packages, such as between CHARMM and AMBER or GROMACS and LAMMPS.  While there are many structure building tools available that generate topologies and structures in CHARMM format, only recently have mechanisms been developed to convert their results into GROMACS input.  We present an approach to convert CHARMM-formatted topol. and parameters into a format suitable for simulation with GROMACS by expanding the functionality of TopoTools, a plugin integrated within the widely used mol. visualization and anal. software VMD.  The conversion process was diligently tested on a comprehensive set of biol. mols. in vacuo.  The resulting comparison between energy terms shows that the translation performed was lossless as the energies were unchanged for identical starting configurations.  By applying the conversion process to conventional benchmark systems that mimic typical modestly sized MD systems, we explore the effect of the implementation choices made in CHARMM, NAMD, and GROMACS.  The newly available automatic conversion capability breaks down barriers between simulation tools and user communities and allows users to easily compare simulation programs and leverage their unique features without the tedium of constructing a topol. twice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooVXPTAehkKrVg90H21EOLACvtfcHk0liFJ6REslS_vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFKrsrc%253D&md5=3dfea753660da654f8cdd3b5f2f40029</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00103%26sid%3Dliteratum%253Aachs%26aulast%3DVermaas%26aufirst%3DJ.%2BV.%26aulast%3DHardy%26aufirst%3DD.%2BJ.%26aulast%3DStone%26aufirst%3DJ.%2BE.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DKohlmeyer%26aufirst%3DA.%26atitle%3DTopoGromacs%253A%2520automated%2520topology%2520conversion%2520from%2520CHARMM%2520to%2520GROMACS%2520within%2520VMD%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1112%26epage%3D1116%26doi%3D10.1021%2Facs.jcim.6b00103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bjelkmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuendet, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">Implementation of the CHARMM force field in GROMACS: analysis of protein stability effects from correction maps, virtual interaction sites, and water models</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/ct900549r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900549r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=459-466&author=P.+Bjelkmarauthor=P.+Larssonauthor=M.+A.+Cuendetauthor=B.+Hessauthor=E.+Lindahl&title=Implementation+of+the+CHARMM+force+field+in+GROMACS%3A+analysis+of+protein+stability+effects+from+correction+maps%2C+virtual+interaction+sites%2C+and+water+models&doi=10.1021%2Fct900549r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Implementation of the CHARMM Force Field in GROMACS: Analysis of Protein Stability Effects from Correction Maps, Virtual Interaction Sites, and Water Models</span></div><div class="casAuthors">Bjelkmar, Par; Larsson, Per; Cuendet, Michel A.; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">459-466</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CHARMM27 is a widespread and popular force field for biomol. simulation, and several recent algorithms such as implicit solvent models have been developed specifically for it.  The authors have here implemented the CHARMM force field and all necessary extended functional forms in the GROMACS mol. simulation package, to make CHARMM-specific features available and to test them in combination with techniques for extended time steps, to make all major force fields available for comparison studies in GROMACS, and to test various solvent model optimizations, in particular the effect of Lennard-Jones interactions on hydrogens.  The implementation has full support both for CHARMM-specific features such as multiple potentials over the same dihedral angle and the grid-based energy correction map on the .vphi., ψ protein backbone dihedrals, as well as all GROMACS features such as virtual hydrogen interaction sites that enable 5 fs time steps.  The medium-to-long time effects of both the correction maps and virtual sites have been tested by performing a series of 100 ns simulations using different models for water representation, including comparisons between CHARMM and traditional TIP3P.  Including the correction maps improves sampling of near native-state conformations in the authors' systems, and to some extent it is even able to refine distorted protein conformations.  Finally, the authors show that this accuracy is largely maintained with a new implicit solvent implementation that works with virtual interaction sites, which enables performance in excess of 250 ns/day for a 900-atom protein on a quad-core desktop computer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzWFE8uOBhobVg90H21EOLACvtfcHk0linjI-ZM8Qp3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlKhsw%253D%253D&md5=728be20d72ab18e98268c1a48efcdcb1</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct900549r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900549r%26sid%3Dliteratum%253Aachs%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DCuendet%26aufirst%3DM.%2BA.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DImplementation%2520of%2520the%2520CHARMM%2520force%2520field%2520in%2520GROMACS%253A%2520analysis%2520of%2520protein%2520stability%2520effects%2520from%2520correction%2520maps%252C%2520virtual%2520interaction%2520sites%252C%2520and%2520water%2520models%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D459%26epage%3D466%26doi%3D10.1021%2Fct900549r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh ewald: an N Log(N) method for ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+ewald%3A+an+N+Log%28N%29+method+for+ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0linjI-ZM8Qp3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520ewald%253A%2520an%2520N%2520Log%2528N%2529%2520method%2520for%2520ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skultetyova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ustinova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlicek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikesova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havlinova, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubalek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansky, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span> <span> </span><span class="NLM_article-title">Human histone deacetylase 6 shows strong preference for tubulin dimers over assembled microtubules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11547</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-11739-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fs41598-017-11739-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28912522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BC1cboslOjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11547&author=L.+Skultetyovaauthor=K.+Ustinovaauthor=Z.+Kutilauthor=Z.+Novakovaauthor=J.+Pavlicekauthor=J.+Mikesovaauthor=D.+Traplauthor=P.+Baranovaauthor=B.+Havlinovaauthor=M.+Hubalekauthor=Z.+Lanskyauthor=C.+Barinka&title=Human+histone+deacetylase+6+shows+strong+preference+for+tubulin+dimers+over+assembled+microtubules&doi=10.1038%2Fs41598-017-11739-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Human histone deacetylase 6 shows strong preference for tubulin dimers over assembled microtubules</span></div><div class="casAuthors">Skultetyova Lubica; Ustinova Kseniya; Kutil Zsofia; Novakova Zora; Pavlicek Jiri; Mikesova Jana; Trapl Dalibor; Baranova Petra; Havlinova Barbora; Lansky Zdenek; Barinka Cyril; Skultetyova Lubica; Hubalek Martin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11547</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human histone deacetylase 6 (HDAC6) is the major deacetylase responsible for removing the acetyl group from Lys40 of α-tubulin (αK40), which is located lumenally in polymerized microtubules.  Here, we provide a detailed kinetic analysis of tubulin deacetylation and HDAC6/microtubule interactions using individual purified components.  Our data unequivocally show that free tubulin dimers represent the preferred HDAC6 substrate, with a K M value of 0.23 μM and a deacetylation rate over 1,500-fold higher than that of assembled microtubules.  We attribute the lower deacetylation rate of microtubules to both longitudinal and lateral lattice interactions within tubulin polymers.  Using TIRF microscopy, we directly visualized stochastic binding of HDAC6 to assembled microtubules without any detectable preferential binding to microtubule tips.  Likewise, indirect immunofluorescence microscopy revealed that microtubule deacetylation by HDAC6 is carried out stochastically along the whole microtubule length, rather than from the open extremities.  Our data thus complement prior studies on tubulin acetylation and further strengthen the rationale for the correlation between tubulin acetylation and microtubule age.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2XuDrQ2dPtsKL0FlyGHDofW6udTcc2eYGFPJjfXt-pbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cboslOjsw%253D%253D&md5=c1b7f71ae3bd1008ad4b8c925ea30599</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-11739-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-11739-3%26sid%3Dliteratum%253Aachs%26aulast%3DSkultetyova%26aufirst%3DL.%26aulast%3DUstinova%26aufirst%3DK.%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DPavlicek%26aufirst%3DJ.%26aulast%3DMikesova%26aufirst%3DJ.%26aulast%3DTrapl%26aufirst%3DD.%26aulast%3DBaranova%26aufirst%3DP.%26aulast%3DHavlinova%26aufirst%3DB.%26aulast%3DHubalek%26aufirst%3DM.%26aulast%3DLansky%26aufirst%3DZ.%26aulast%3DBarinka%26aufirst%3DC.%26atitle%3DHuman%2520histone%2520deacetylase%25206%2520shows%2520strong%2520preference%2520for%2520tubulin%2520dimers%2520over%2520assembled%2520microtubules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11547%26doi%3D10.1038%2Fs41598-017-11739-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meleshin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikesova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 11 is a fatty-acid deacylase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">685</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00942</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00942" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXoslCjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=685-693&author=Z.+Kutilauthor=Z.+Novakovaauthor=M.+Meleshinauthor=J.+Mikesovaauthor=M.+Schutkowskiauthor=C.+Barinka&title=Histone+deacetylase+11+is+a+fatty-acid+deacylase&doi=10.1021%2Facschembio.7b00942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase 11 Is a Fatty-Acid Deacylase</span></div><div class="casAuthors">Kutil, Zsofia; Novakova, Zora; Meleshin, Marat; Mikesova, Jana; Schutkowski, Mike; Barinka, Cyril</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">685-693</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 11 (HDAC11) is a sole member of the class IV HDAC subfamily with negligible intrinsic deacetylation activity.  Here, we report in vitro profiling of HDAC11 deacylase activities, and our data unequivocally show that the enzyme efficiently removes acyl moieties spanning 8-18 carbons from the side chain nitrogen of the lysine residue of a peptidic substrate.  Addnl., N-linked lipoic acid and biotin are removed by the enzyme, although with lower efficacy.  Catalytic efficiencies toward dodecanoylated and myristoylated peptides were 77 700 and 149 000 M-1 s-1, resp., making HDAC11 the most proficient fatty-acid deacylase of the HDAC family.  Interestingly, HDAC11 is strongly inhibited by free myristic, palmitic, and stearic acids with inhibition consts. of 6.5, 0.9, and 1.6 μM, resp.  At the same time, its deacylase activity is stimulated more than 2.5-fold by both palmitoyl-CoA and myristoyl-CoA, pointing toward metabolic control of the enzymic activity by fatty-acid metabolites.  Our data reveal novel enzymic activity of HDAC11 that can, in turn, facilitate the uncovering of addnl. biol. functions of the enzyme as well as the design of isoform-specific HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozNsptw7rk7bVg90H21EOLACvtfcHk0lgtzi3_O4k89g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXoslCjug%253D%253D&md5=07233a3bfed4f158c6d5752fc18a8c1f</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00942%26sid%3Dliteratum%253Aachs%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DMeleshin%26aufirst%3DM.%26aulast%3DMikesova%26aufirst%3DJ.%26aulast%3DSchutkowski%26aufirst%3DM.%26aulast%3DBarinka%26aufirst%3DC.%26atitle%3DHistone%2520deacetylase%252011%2520is%2520a%2520fatty-acid%2520deacylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D685%26epage%3D693%26doi%3D10.1021%2Facschembio.7b00942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Development of multi-functional histone deacetylase 6 degraders with potent anti-myeloma activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7042</span>– <span class="NLM_lpage">7057</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7042-7057&author=H.+Wuauthor=K.+Yangauthor=Z.+Zhangauthor=E.+D.+Leistenauthor=Z.+Liauthor=H.+Xieauthor=J.+Liuauthor=K.+A.+Smithauthor=Z.+Novakovaauthor=C.+Barinkaauthor=W.+Tang&title=Development+of+multi-functional+histone+deacetylase+6+degraders+with+potent+anti-myeloma+activity&doi=10.1021%2Facs.jmedchem.9b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Multifunctional Histone Deacetylase 6 Degraders with Potent Antimyeloma Activity</span></div><div class="casAuthors">Wu, Hao; Yang, Ka; Zhang, Zhongrui; Leisten, Eric D.; Li, Ziyuan; Xie, Haibo; Liu, Jin; Smith, Kerry A.; Novakova, Zora; Barinka, Cyril; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7042-7057</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) primarily catalyzes the removal of acetyl group from the side chain of acetylated lysine residues in cytoplasmic proteins such as α-tubulin and HSP90.  HDAC6 is involved in multiple disease-relevant pathways.  Based on the proteolysis targeting chimera strategy, we previously developed the first HDAC6 degrader by tethering a pan-HDAC inhibitor with cereblon (CRBN) E3 ubiquitin ligase ligand.  We herein report our new generation of multifunctional HDAC6 degraders by tethering selective HDAC6 inhibitor Nexturastat A with CRBN ligand that can synergize with HDAC6 degrdn. for the antiproliferation of multiple myeloma (MM).  This new class of degraders exhibited improved potency and selectivity for the degrdn. of HDAC6.  After the optimization of the linker length and linking positions, we discovered potent HDAC6 degraders with nanomolar DC50 and promising antiproliferation activity in multiple myeloma (MM) cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb58zf04V2GLVg90H21EOLACvtfcHk0lgtzi3_O4k89g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO&md5=7aab402404a5fce732b4b68fd76395d4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00516%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DK.%2BA.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520multi-functional%2520histone%2520deacetylase%25206%2520degraders%2520with%2520potent%2520anti-myeloma%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7042%26epage%3D7057%26doi%3D10.1021%2Facs.jmedchem.9b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">The MX beamlines BL<sub>14.1-3</sub> at BESSY II</span>. <i>J. Large-scale Res. Facil.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">47</span>, <span class="refDoi"> DOI: 10.17815/jlsrf-2-64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.17815%2Fjlsrf-2-64" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=47&author=M.+Gerlachauthor=U.+Muellerauthor=M.+S.+Weiss&title=The+MX+beamlines+BL14.1-3+at+BESSY+II&doi=10.17815%2Fjlsrf-2-64"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.17815%2Fjlsrf-2-64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.17815%252Fjlsrf-2-64%26sid%3Dliteratum%253Aachs%26aulast%3DGerlach%26aufirst%3DM.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DThe%2520MX%2520beamlines%2520BL14.1-3%2520at%2520BESSY%2520II%26jtitle%3DJ.%2520Large-scale%2520Res.%2520Facil.%26date%3D2016%26volume%3D2%26spage%3D47%26doi%3D10.17815%2Fjlsrf-2-64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">DIALS: implementation and evaluation of a new integration package</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/S2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=DIALS%3A+implementation+and+evaluation+of+a+new+integration+package&doi=10.1107%2FS2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0lj8fa0efzgAzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDIALS%253A%2520implementation%2520and%2520evaluation%2520of%2520a%2520new%2520integration%2520package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2FS2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/S090744491003982X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS090744491003982X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2FS090744491003982X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0lj8fa0efzgAzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS090744491003982X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744491003982X%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2FS090744491003982X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thorn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">AUSPEX: a graphical tool for X-ray diffraction data analysis</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1107/S205979831700969X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS205979831700969X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeqsrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=729-737&author=A.+Thornauthor=J.+Parkhurstauthor=P.+Emsleyauthor=R.+A.+Nichollsauthor=M.+Vollmarauthor=G.+Evansauthor=G.+N.+Murshudov&title=AUSPEX%3A+a+graphical+tool+for+X-ray+diffraction+data+analysis&doi=10.1107%2FS205979831700969X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AUSPEX: a graphical tool for X-ray diffraction data analysis</span></div><div class="casAuthors">Thorn, Andrea; Parkhurst, James; Emsley, Paul; Nicholls, Robert A.; Vollmar, Melanie; Evans, Gwyndaf; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">729-737</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">In this paper, AUSPEX, a new software tool for exptl. X-ray data anal., is presented.  Exploring the behavior of diffraction intensities and the assocd. estd. uncertainties facilitates the discovery of underlying problems and can help users to improve their data acquisition and processing in order to obtain better structural models.  The program enables users to inspect the distribution of obsd. intensities (or amplitudes) against resoln. as well as the assocd. estd. uncertainties (sigmas).  It is demonstrated how AUSPEX can be used to visually and automatically detect ice-ring artifacts in integrated X-ray diffraction data.  Such artifacts can hamper structure detn., but may be difficult to identify from the raw diffraction images produced by modern pixel detectors.  The anal. suggests that a significant portion of the data sets deposited in the PDB contain ice-ring artifacts.  Furthermore, it is demonstrated how other problems in exptl. X-ray data caused, for example, by scaling and data-conversion procedures can be detected by AUSPEX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_DReV2W1ibVg90H21EOLACvtfcHk0lj8fa0efzgAzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeqsrfE&md5=bde8571b0ed8c18cf74c8b3a157c5a1a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS205979831700969X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS205979831700969X%26sid%3Dliteratum%253Aachs%26aulast%3DThorn%26aufirst%3DA.%26aulast%3DParkhurst%26aufirst%3DJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DG.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAUSPEX%253A%2520a%2520graphical%2520tool%2520for%2520X-ray%2520diffraction%2520data%2520analysis%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2017%26volume%3D73%26spage%3D729%26epage%3D737%26doi%3D10.1107%2FS205979831700969X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljKGmZp6UewNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljKGmZp6UewNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljKGmZp6UewNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljKGmZp6UewNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebschner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Polder maps: improving OMIT maps by excluding bulk solvent</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">148</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1107/S2059798316018210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS2059798316018210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=148-157&author=D.+Liebschnerauthor=P.+V.+Afonineauthor=N.+W.+Moriartyauthor=B.+K.+Poonauthor=O.+V.+Sobolevauthor=T.+C.+Terwilligerauthor=P.+D.+Adams&title=Polder+maps%3A+improving+OMIT+maps+by+excluding+bulk+solvent&doi=10.1107%2FS2059798316018210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Polder maps: improving OMIT maps by excluding bulk solvent</span></div><div class="casAuthors">Liebschner, Dorothee; Afonine, Pavel V.; Moriarty, Nigel W.; Poon, Billy K.; Sobolev, Oleg V.; Terwilliger, Thomas C.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-157</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The crystallog. maps that are routinely used during the structure-soln. workflow are almost always model-biased because model information is used for their calcn.  As these maps are also used to validate the at. models that result from model building and refinement, this constitutes an immediate problem: anything added to the model will manifest itself in the map and thus hinder the validation.  OMIT maps are a common tool to verify the presence of atoms in the model.  The simplest way to compute an OMIT map is to exclude the atoms in question from the structure, update the corresponding structure factors and compute a residual map.  It is then expected that if these atoms are present in the crystal structure, the electron d. for the omitted atoms will be seen as pos. features in this map.  This, however, is complicated by the flat bulk-solvent model which is almost universally used in modern crystallog. refinement programs.  This model postulates const. electron d. at any voxel of the unit-cell vol. that is not occupied by the at. model.  Consequently, if the d. arising from the omitted atoms is weak then the bulk-solvent model may obscure it further.  A possible soln. to this problem is to prevent bulk solvent from entering the selected OMIT regions, which may improve the interpretative power of residual maps.  This approach is called a polder (OMIT) map.  Polder OMIT maps can be particularly useful for displaying weak densities of ligands, solvent mols., side chains, alternative conformations and residues both in terminal regions and in loops.  The tools described in this manuscript have been implemented and are available in PHENIX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwO8sssGmoxrVg90H21EOLACvtfcHk0lgf1ullZVdOQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu7w%253D&md5=0643653827bdba013d735703c941d434</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS2059798316018210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316018210%26sid%3Dliteratum%253Aachs%26aulast%3DLiebschner%26aufirst%3DD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DPoon%26aufirst%3DB.%2BK.%26aulast%3DSobolev%26aufirst%3DO.%2BV.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPolder%2520maps%253A%2520improving%2520OMIT%2520maps%2520by%2520excluding%2520bulk%2520solvent%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2017%26volume%3D73%26spage%3D148%26epage%3D157%26doi%3D10.1107%2FS2059798316018210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgf1ullZVdOQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knights, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stresser, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miners, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespi, C. L.</span></span> <span> </span><span class="NLM_article-title">In vitro drug metabolism using liver microsomes</span>. <i>Curr. Protoc. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1002/cpph.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fcpph.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27636111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BC2svhsVCisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2016&pages=7-8&author=K.+M.+Knightsauthor=D.+M.+Stresserauthor=J.+O.+Minersauthor=C.+L.+Crespi&title=In+vitro+drug+metabolism+using+liver+microsomes&doi=10.1002%2Fcpph.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Drug Metabolism Using Liver Microsomes</span></div><div class="casAuthors">Knights Kathleen M; Stresser David M; Miners John O; Crespi Charles L</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7.8.1-7.8.24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Knowledge of the metabolic stability of newly discovered drug candidates eliminated by metabolism is essential for predicting the pharmacokinetic (PK) parameters that underpin dosing and dosage frequency.  Further, characterization of the enzyme(s) responsible for metabolism (reaction phenotyping) allows prediction, at least at the qualitative level, of factors (including metabolic drug-drug interactions) likely to alter the clearance of both new chemical entities (NCEs) and established drugs.  Microsomes are typically used as the enzyme source for the measurement of metabolic stability and for reaction phenotyping because they express the major drug-metabolizing enzymes cytochrome P450 (CYP) and UDP-glucuronosyltransferase (UGT), along with others that contribute to drug metabolism.  Described in this unit are methods for microsome isolation, as well as for the determination of metabolic stability and metabolite formation (including kinetics). © 2016 by John Wiley & Sons, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS42TPy_5UT8q-9TkgmBqTqfW6udTcc2eaOKHCiOHU4Bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svhsVCisg%253D%253D&md5=506e94e8b6f81de2f05104c1b7879312</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fcpph.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcpph.9%26sid%3Dliteratum%253Aachs%26aulast%3DKnights%26aufirst%3DK.%2BM.%26aulast%3DStresser%26aufirst%3DD.%2BM.%26aulast%3DMiners%26aufirst%3DJ.%2BO.%26aulast%3DCrespi%26aufirst%3DC.%2BL.%26atitle%3DIn%2520vitro%2520drug%2520metabolism%2520using%2520liver%2520microsomes%26jtitle%3DCurr.%2520Protoc.%2520Pharmacol.%26date%3D2016%26volume%3D74%26spage%3D7%26epage%3D8%26doi%3D10.1002%2Fcpph.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barré, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamouard, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillement, J. P.</span></span> <span> </span><span class="NLM_article-title">Equilibrium dialysis, ultrafiltration, and ultracentrifugation compared for determining the plasma-protein-binding characteristics of valproic acid</span>. <i>Clin. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1093/clinchem/31.1.60</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1093%2Fclinchem%2F31.1.60" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=3917383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADyaL2MXovFKntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1985&pages=60-64&author=J.+Barr%C3%A9author=J.+M.+Chamouardauthor=G.+Houinauthor=J.+P.+Tillement&title=Equilibrium+dialysis%2C+ultrafiltration%2C+and+ultracentrifugation+compared+for+determining+the+plasma-protein-binding+characteristics+of+valproic+acid&doi=10.1093%2Fclinchem%2F31.1.60"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrium dialysis, ultrafiltration, and ultracentrifugation compared for determining the plasma-protein-binding characteristics of valproic acid</span></div><div class="casAuthors">Barre, Jerome; Chamouard, Jean Marie; Houin, Georges; Tillement, Jean Paul</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-4</span>CODEN:
                <span class="NLM_cas:coden">CLCHAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-9147</span>.
    </div><div class="casAbstract">Equil. dialysis, ultrafiltration, and ultracentrifugation were compared for their reliability and applicability in the study of binding of an anticonvulsant drug, valproic acid  [99-66-1], by plasma proteins.  Drug binding was studied with pooled serum and with solns. of human serum albumin at physiol. concns.  Binding characteristics such as no. of binding sites, affinity consts., and percent of binding were compared as measured by each method in the therapeutic range for valproic acid.  Results by ultracentrifugation differed from those by equil. dialysis and ultrafiltration, which agreed reasonably well with each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7LVSvezYF-rVg90H21EOLACvtfcHk0lhK1ye5cx-TcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXovFKntQ%253D%253D&md5=aca31b98ff28d80bfebf09a3513c7d95</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fclinchem%2F31.1.60&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fclinchem%252F31.1.60%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%25C3%25A9%26aufirst%3DJ.%26aulast%3DChamouard%26aufirst%3DJ.%2BM.%26aulast%3DHouin%26aufirst%3DG.%26aulast%3DTillement%26aufirst%3DJ.%2BP.%26atitle%3DEquilibrium%2520dialysis%252C%2520ultrafiltration%252C%2520and%2520ultracentrifugation%2520compared%2520for%2520determining%2520the%2520plasma-protein-binding%2520characteristics%2520of%2520valproic%2520acid%26jtitle%3DClin.%2520Chem.%26date%3D1985%26volume%3D31%26spage%3D60%26epage%3D64%26doi%3D10.1093%2Fclinchem%2F31.1.60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Sida Shen, Cristina Picci, Kseniya Ustinova, Veronick Benoy, Zsófia Kutil, Guiping Zhang, Maurício T. Tavares, Jiří Pavlíček, Chad A. Zimprich, Matthew B. Robers, Ludo Van Den Bosch, Cyril Bařinka, Brett Langley, <span class="NLM_string-name hlFld-ContribAuthor">Alan P. Kozikowski</span>. </span><span class="cited-content_cbyCitation_article-title">Tetrahydroquinoline-Capped Histone Deacetylase 6 Inhibitor SW-101 Ameliorates Pathological Phenotypes in a Charcot–Marie–Tooth Type 2A Mouse Model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4810-4840. <a href="https://doi.org/10.1021/acs.jmedchem.0c02210" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02210%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTetrahydroquinoline-Capped%252BHistone%252BDeacetylase%252B6%252BInhibitor%252BSW-101%252BAmeliorates%252BPathological%252BPhenotypes%252Bin%252Ba%252BCharcot%2525E2%252580%252593Marie%2525E2%252580%252593Tooth%252BType%252B2A%252BMouse%252BModel%26aulast%3DShen%26aufirst%3DSida%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21122020%26date%3D08042021%26volume%3D64%26issue%3D8%26spage%3D4810%26epage%3D4840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xin-Hui Zhang, Qin-Ma, Hui-Pan Wu, Mussa Yussuf Khamis, Yi-Han Li, Li-Ying Ma, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A Review of Progress in Histone Deacetylase 6 Inhibitors Research: Structural Specificity and Functional Diversity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1362-1391. <a href="https://doi.org/10.1021/acs.jmedchem.0c01782" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01782</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01782%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BReview%252Bof%252BProgress%252Bin%252BHistone%252BDeacetylase%252B6%252BInhibitors%252BResearch%25253A%252BStructural%252BSpecificity%252Band%252BFunctional%252BDiversity%26aulast%3DZhang%26aufirst%3DXin-Hui%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D12102020%26date%3D01022021%26volume%3D64%26issue%3D3%26spage%3D1362%26epage%3D1391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maurício  T Tavares</span>, <span class="hlFld-ContribAuthor ">Sida  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Recent innovative advances in the discovery of selective HDAC6 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 1017-1019. <a href="https://doi.org/10.4155/fmc-2021-0040" title="DOI URL">https://doi.org/10.4155/fmc-2021-0040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0040%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Binnovative%252Badvances%252Bin%252Bthe%252Bdiscovery%252Bof%252Bselective%252BHDAC6%252Binhibitors%26aulast%3DT%25C2%25A0Tavares%26aufirst%3DMaur%25C3%25ADcio%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D1017%26epage%3D1019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maurício Temotheo  Tavares</span>, <span class="hlFld-ContribAuthor ">Larissa Costa  de Almeida</span>, <span class="hlFld-ContribAuthor ">Thales  Kronenberger</span>, <span class="hlFld-ContribAuthor ">Glaucio  Monteiro Ferreira</span>, <span class="hlFld-ContribAuthor ">Thainá  Fujii de Divitiis</span>, <span class="hlFld-ContribAuthor ">Mônica  Franco Zannini Junqueira Toledo</span>, <span class="hlFld-ContribAuthor ">Neuza  Mariko Aymoto Hassimotto</span>, <span class="hlFld-ContribAuthor ">João  Agostinho Machado-Neto</span>, <span class="hlFld-ContribAuthor ">Letícia  Veras Costa-Lotufo</span>, <span class="hlFld-ContribAuthor ">Roberto  Parise-Filho</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationship and mechanistic studies for a series of cinnamyl hydroxamate histone deacetylase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>35 </em>, 116085. <a href="https://doi.org/10.1016/j.bmc.2021.116085" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116085</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116085%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Brelationship%252Band%252Bmechanistic%252Bstudies%252Bfor%252Ba%252Bseries%252Bof%252Bcinnamyl%252Bhydroxamate%252Bhistone%252Bdeacetylase%252Binhibitors%26aulast%3DTavares%26aufirst%3DMaur%25C3%25ADcio%2BTemotheo%26date%3D2021%26volume%3D35%26spage%3D116085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander C.  Chacon</span>, <span class="hlFld-ContribAuthor ">Alexa D.  Melucci</span>, <span class="hlFld-ContribAuthor ">Shuyang S.  Qin</span>, <span class="hlFld-ContribAuthor ">Peter A.  Prieto</span>. </span><span class="cited-content_cbyCitation_article-title">Thinking Small: Small Molecules as Potential Synergistic Adjuncts to Checkpoint Inhibition in Melanoma. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (6)
                                     , 3228. <a href="https://doi.org/10.3390/ijms22063228" title="DOI URL">https://doi.org/10.3390/ijms22063228</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22063228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22063228%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DThinking%252BSmall%25253A%252BSmall%252BMolecules%252Bas%252BPotential%252BSynergistic%252BAdjuncts%252Bto%252BCheckpoint%252BInhibition%252Bin%252BMelanoma%26aulast%3DChacon%26aufirst%3DAlexander%2BC.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D6%26spage%3D3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structures of zHDAC6 complexes with selective HDAC6is. (A) Tubastatin A (yellow; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6THV">6THV</a>). (B) RTS-V5 (green; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CW8">6CW8</a>). (C) ACY-1083 (pink; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGM">5WGM</a>). (D) HPB (blue; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGK">5WGK</a>). (E) SS-208 (cyan; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R0K">6R0K</a>). (F) CBZ (purple; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZU">6PZU</a>). Selective amino acid residues in HDAC6 are indicated in stick representation with carbon atoms in gray, oxygen atoms in red, and nitrogen atoms in blue colors, whereas the active site zinc ion and water molecules are indicated as silver and red spheres, respectively. IC<sub>50</sub> values for HDAC1 and HDAC6, along with selectivity indices, are indicated for each HDAC6 inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In silico</i> design of Suprastat by NextA derivatization. (A) Structures of phenylhydroxamate-based nexturastat A (left) and Suprastat (right). (B) Molecular docking of Suprastat (orange) based on the HDAC6/nexturastat A complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>; silver) with an indication of the additional interactions created by the aminomethyl and hydroxybutyl polar groups with residues D460 and F643 at the rim of the catalytic pocket of HDAC6, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route to <b>6a</b> and <b>6b</b> and <b>6d</b>–<b>6f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phenyl chloroformate, K<sub>2</sub>CO<sub>3</sub>, acetone, rt, and 2 h; (b) (i) 4-amino-1-butanol for <b>4a</b>, <b>4c</b>, and <b>4d</b> and <i>n</i>-butylamine for <b>4b</b>, EtOH, reflux, and 2 h; and (ii) NaBH<sub>4</sub>, MeOH, 0 °C to rt, and 2 h; (c) <b>2</b>, TEA, THF, reflux, and 2 h; (d) for <b>6a</b> and <b>6b</b>: (i) aqueous NH<sub>2</sub>OH (50%), NaOH, THF/MeOH, 0 °C, and 15 min; and (ii) TFA, THF, rt, and 0.5 h; (e) for <b>6d</b>: (i) 1 N NaOH, THF/MeOH, rt, and overnight; and (ii) TFA, THF, rt, and 0.5 h; (f) for <b>6e</b>: (i) H<sub>2</sub>O<sub>2</sub> (30%), K<sub>2</sub>CO<sub>3</sub>, DMSO, rt, and 5 h; and (ii) TFA, THF, rt, and 0.5 h; and (g) for <b>6f</b>: (i) B<sub>2</sub>pin<sub>2</sub>, KOAc, Pd(dppf)Cl<sub>2</sub>, DMF, 80 °C, and overnight; (ii) NaIO<sub>4</sub>, NH<sub>4</sub>OAc, acetone/H<sub>2</sub>O, rt, and overnight; and (iii) TFA, THF, rt, and 0.5 h.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Suprastat is a highly selective HDAC6 inhibitor. WM164 human melanoma cells treated overnight with increasing concentrations ranging from 0.1 to 10 μM for HDAC6 inhibitors. (A) Suprastat (<b>6a</b>). (B) <b>6b</b>. (C) <b>6c</b>. (D) NextA. Immunoblot analysis was performed for Ac-α-tubulin and Ac-H3. Total α-tubulin and total H3 are loading controls. (E, F) Densitometric analyses of the Ac-α-tubulin and Ac-H3 bands, respectively. The immunoblots were repeated at least twice to confirm accuracy, and the best representation is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Suprastat modulates immune pathways through interaction with STAT3. (A) <i>IL10</i> gene expression was determined by quantitative PCR in bone marrow-derived macrophages exposed to interferon-gamma (20 ng mL<sup>–1</sup>) and LPS (100 ng mL<sup>–1</sup>). (B) WM164 murine melanoma cells were pretreated with 5 μM Suprastat or NextA followed by treatment with IL-6 (30 ng mL<sup>–1</sup>) for 20 min.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Combination of Suprastat with anti-PD1 immunotherapy significantly decreases tumor burden in the SM1 murine melanoma model. (A) Cumulative tumor growth rates in C57BL/6 mice (<i>n</i> = 10) after treatment with Suprastat or anti-PD1 therapy or a combination of both. Tumor growth rates of treatment groups were compared to control groups treated with PBS. (B) Tumor growth rates of individual mice in respective treatment groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Suprastat has immunomodulatory properties in the tumor microenvironment. (A–C) Macrophages, (D–F) T-cells, and (G, H) NK cells were stained with cell surface markers and analyzed by multicolor flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Crystal structure of the zHDAC6/Suprastat complex. (A) Both monodentate (left) and bidentate (right) coordination of the active site Zn<sup>2+</sup> ion is observed in the complex. (B) The omit <i>F</i><sub>o</sub>–<i>F</i><sub>c</sub> difference electron density map for Suprastat is shown (green mesh) for the more prevalent monodentate binding mode (contoured at the 2σ level). (C, D) Nonpolar interactions between the phenylhydroxamate function of Suprastat and residues delineating the internal tunnel (C) and the capping group and the hydrophobic patch of the L1 loop (red surface representation; D). (E) Newly introduced polar/ionic contacts between the aminomethyl and hydroxylbutyl groups of Suprastat and D460 and N530, respectively (black dashed lines). (F) Superposition of binding modes of NextA (yellow; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I">5G0I</a>) and Suprastat (cyan) documenting minimal structural differences between the two structures. Both structures were superposed on corresponding 320 Cα atoms with an RMSD of 0.192 Å. Inhibitors are shown in stick representation, carbon atoms are colored cyan (Suprastat) or yellow (NextA), oxygens are red, and nitrogens are blue. The catalytic zinc ion and water molecules are depicted as purple and red spheres, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/medium/jm0c00567_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Representative poses of Suprastat bound to the zHDAC6-CD2 catalytic pocket after molecular dynamics (MD) simulations. (A) Flexible hydroxylbutyl chain engages in H-bonding interaction with N530. (B) Transition state between each stable conformer. (C) H-bonding interaction between Suprastat and S531. (A–C) Aminomethyl group establishes a consistent water-mediated interaction with D460 during the MD simulation process. (D) Direct contact frequency between the hydroxylbutyl chain and the aminomethyl group of Suprastat and the amino acids at the rim of zHDAC6-CD2 along with the MD simulation. Carbon atoms are depicted as gray. Nitrogen atoms are depicted as blue. Oxygen atoms are depicted as red. The Zn<sup>2+</sup> ion is represented as a gray sphere. Water molecules are represented as red spheres. The protein is depicted in gray. Hydrogen-bonding interactions are depicted as green dashed lines. Distances are reported in angstroms (Å). Images were generated in PyMOL 2.1 software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00567/20200918/images/large/jm0c00567_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00567&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71703" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71703" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span> <span> </span><span class="NLM_article-title">Regulation of cellular metabolism by protein lysine acetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1004</span>, <span class="refDoi"> DOI: 10.1126/science.1179689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1126%2Fscience.1179689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20167786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1000-1004&author=S.+Zhaoauthor=W.+Xuauthor=W.+Jiangauthor=W.+Yuauthor=Y.+Linauthor=T.+Zhangauthor=J.+Yaoauthor=L.+Zhouauthor=Y.+Zengauthor=H.+Liauthor=Y.+Liauthor=J.+Shiauthor=W.+Anauthor=S.+M.+Hancockauthor=F.+Heauthor=L.+Qinauthor=J.+Chinauthor=P.+Yangauthor=X.+Chenauthor=Q.+Leiauthor=Y.+Xiongauthor=K.+L.+Guan&title=Regulation+of+cellular+metabolism+by+protein+lysine+acetylation&doi=10.1126%2Fscience.1179689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cellular metabolism by protein lysine acetylation</span></div><div class="casAuthors">Zhao, Shimin; Xu, Wei; Jiang, Wenqing; Yu, Wei; Lin, Yan; Zhang, Tengfei; Yao, Jun; Zhou, Li; Zeng, Yaxue; Li, Hong; Li, Yixue; Shi, Jiong; An, Wenlin; Hancock, Susan M.; He, Fuchu; Qin, Lunxiu; Chin, Jason; Yang, Pengyuan; Chen, Xian; Lei, Qunying; Xiong, Yue; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1000-1004</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Protein Lys acetylation has emerged as a key post-translational modification in cellular regulation, in particular through the modification of histones and nuclear transcription regulators.  Here, the authors show that acetylation of Lys residues is a prevalent modification in enzymes that catalyze intermediate metab.  Virtually every enzyme in glycolysis, gluconeogenesis, the tricarboxylic acid (TCA) cycle, the urea cycle, fatty acid metab., and glycogen metab. was found to be acetylated in human liver tissue.  The concn. of metabolic fuels, such as glucose, amino acids, and fatty acids, influenced the acetylation status of metabolic enzymes.  Acetylation activated enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase in fatty acid oxidn. and malate dehydrogenase in the TCA cycle, inhibited argininosuccinate lyase in the urea cycle, and destabilized phosphoenolpyruvate carboxykinase in gluconeogenesis.  Thus, this study revealed that acetylation plays a major role in metabolic regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbdnDsuudXV7Vg90H21EOLACvtfcHk0li-Xl3V_Dkpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D&md5=d2c3c897077b9285842a7c55e8f8737e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179689%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DAn%26aufirst%3DW.%26aulast%3DHancock%26aufirst%3DS.%2BM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26atitle%3DRegulation%2520of%2520cellular%2520metabolism%2520by%2520protein%2520lysine%2520acetylation%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1000%26epage%3D1004%26doi%3D10.1126%2Fscience.1179689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G. P.</span></span> <span> </span><span class="NLM_article-title">Acetylation of metabolic enzymes coordinates carbon source utilization and metabolic flux</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1004</span>– <span class="NLM_lpage">1007</span>, <span class="refDoi"> DOI: 10.1126/science.1179687</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1126%2Fscience.1179687" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20167787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1004-1007&author=Q.+Wangauthor=Y.+Zhangauthor=C.+Yangauthor=H.+Xiongauthor=Y.+Linauthor=J.+Yaoauthor=H.+Liauthor=L.+Xieauthor=W.+Zhaoauthor=Y.+Yaoauthor=Z.+B.+Ningauthor=R.+Zengauthor=Y.+Xiongauthor=K.+L.+Guanauthor=S.+Zhaoauthor=G.+P.+Zhao&title=Acetylation+of+metabolic+enzymes+coordinates+carbon+source+utilization+and+metabolic+flux&doi=10.1126%2Fscience.1179687"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation of metabolic enzymes coordinates carbon source utilization and metabolic flux</span></div><div class="casAuthors">Wang, Qijun; Zhang, Yakun; Yang, Chen; Xiong, Hui; Lin, Yan; Yao, Jun; Li, Hong; Xie, Lu; Zhao, Wei; Yao, Yufeng; Ning, Zhi-Bin; Zeng, Rong; Xiong, Yue; Guan, Kun-Liang; Zhao, Shimin; Zhao, Guo-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1004-1007</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation regulates many eukaryotic cellular processes, but its function in prokaryotes is largely unknown.  We demonstrated that central metab. enzymes in Salmonella were acetylated extensively and differentially in response to different C sources, concomitantly with changes in cell growth and metabolic flux.  The relative activities of key enzymes controlling the direction of glycolysis vs. gluconeogenesis and the branching between citrate cycle and glyoxylate bypass were all regulated by acetylation.  This modulation is mainly controlled by a pair of lysine acetyltransferase and deacetylase, whose expressions are coordinated with growth status.  Reversible acetylation of metabolic enzymes ensure that cells respond environmental changes via promptly sensing cellular energy status and flexibly altering reaction rates or directions.  It represents a metabolic regulatory mechanism conserved from bacteria to mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphUlI--SS1CbVg90H21EOLACvtfcHk0li-Xl3V_Dkpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbs%253D&md5=cd56ec28074baf0b6277b0936e4fdab2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179687&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179687%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DNing%26aufirst%3DZ.%2BB.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DG.%2BP.%26atitle%3DAcetylation%2520of%2520metabolic%2520enzymes%2520coordinates%2520carbon%2520source%2520utilization%2520and%2520metabolic%2520flux%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1004%26epage%3D1007%26doi%3D10.1126%2Fscience.1179687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">The growing landscape of lysine acetylation links metabolism and cell signalling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrm3841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnrm3841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25053359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtF2ktbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=536-550&author=C.+Choudharyauthor=B.+T.+Weinertauthor=Y.+Nishidaauthor=E.+Verdinauthor=M.+Mann&title=The+growing+landscape+of+lysine+acetylation+links+metabolism+and+cell+signalling&doi=10.1038%2Fnrm3841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The growing landscape of lysine acetylation links metabolism and cell signaling</span></div><div class="casAuthors">Choudhary, Chunaram; Weinert, Brian T.; Nishida, Yuya; Verdin, Eric; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-550</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lys residue acetylation is a conserved protein post-translational modification that links acetyl-CoA metab. and cellular signaling.  Recent advances in the identification and quantification of Lys residue acetylation by mass spectrometry have increased the understanding of Lys acetylation, implicating it in many biol. processes through the regulation of protein interactions, activity, and localization.  In addn., proteins are frequently modified by other types of acylations, such as formylation, butyrylation, propionylation, succinylation, malonylation, myristoylation, glutarylation, and crotonylation.  The intricate link between Lys acylation and cellular metab. has been clarified by the occurrence of several such metabolite-sensitive acylations and their selective removal by sirtuin deacylases.  These emerging findings point to new functions for different Lys acylations and deacylating enzymes and also highlight the mechanisms by which acetylation regulates various cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFJb5kUjj7arVg90H21EOLACvtfcHk0lhp9TL9XLYVQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtF2ktbrN&md5=89703cd65c2cebe4a1474cf40f6d0be7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3841%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DNishida%26aufirst%3DY.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DThe%2520growing%2520landscape%2520of%2520lysine%2520acetylation%2520links%2520metabolism%2520and%2520cell%2520signalling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D536%26epage%3D550%26doi%3D10.1038%2Fnrm3841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span> <span> </span><span class="NLM_article-title">HDACs and HDAC inhibitors in cancer development and therapy</span>. <i>Cold Spring Harb. Perspect Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a026831</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1101%2Fcshperspect.a026831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27599530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpslCitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a026831&author=Y.+Liauthor=E.+Seto&title=HDACs+and+HDAC+inhibitors+in+cancer+development+and+therapy&doi=10.1101%2Fcshperspect.a026831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">HDACs and HDAC inhibitors in cancer development and therapy</span></div><div class="casAuthors">Li, Yixuan; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">a026831/1-a026831/35</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Over the last several decades, it has become clear that epigenetic abnormalities may be one of the hallmarks of cancer.  Posttranslational modifications of histones, for example, may play a crucial role in cancer development and progression by modulating gene transcription, chromatin remodeling, and nuclear architecture.  Histone acetylation, a well-studied posttranslational histone modification, is controlled by the opposing activities of histone acetyltransferases (HATs) and histone deacetylases (HDACs).  By removing acetyl groups, HDACs reverse chromatin acetylation and alter transcription of oncogenes and tumor suppressor genes.  In addn.,HDACs deacetylate numerous nonhistone cellular substrates that govern a wide array of biol. processes including cancer initiation and progression.  This review will discuss the role of HDACs in cancer and the therapeutic potential of HDAC inhibitors (HDACi) as emerging drugs in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnRL0Y7L3_s7Vg90H21EOLACvtfcHk0lhp9TL9XLYVQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpslCitb4%253D&md5=1c647aa38027de1c4c92f74340d9041a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026831%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DHDACs%2520and%2520HDAC%2520inhibitors%2520in%2520cancer%2520development%2520and%2520therapy%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da026831%26doi%3D10.1101%2Fcshperspect.a026831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckschlager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiborova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrabeta, J.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase inhibitors as anticancer drugs</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1414</span>, <span class="refDoi"> DOI: 10.3390/ijms18071414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.3390%2Fijms18071414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28671573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1414&author=T.+Eckschlagerauthor=J.+Plchauthor=M.+Stiborovaauthor=J.+Hrabeta&title=Histone+deacetylase+inhibitors+as+anticancer+drugs&doi=10.3390%2Fijms18071414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors as anticancer drugs</span></div><div class="casAuthors">Eckschlager, Tomas; Plch, Johana; Stiborova, Marie; Hrabeta, Jan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1414/1-1414/25</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes.  Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT).  HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response.  Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc.  HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy.  HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma.  Other HDAC inhibitors are in clin. trials for the treatment of hematol. and solid malignancies.  The results of such studies are promising but further larger studies are needed.  Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance.  Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclin. and clin. studies, including the combination with other therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNVhoLBAHnrVg90H21EOLACvtfcHk0lhp9TL9XLYVQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P&md5=6018c7709a09548214bfe34bd91107a9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fijms18071414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071414%26sid%3Dliteratum%253Aachs%26aulast%3DEckschlager%26aufirst%3DT.%26aulast%3DPlch%26aufirst%3DJ.%26aulast%3DStiborova%26aufirst%3DM.%26aulast%3DHrabeta%26aufirst%3DJ.%26atitle%3DHistone%2520deacetylase%2520inhibitors%2520as%2520anticancer%2520drugs%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1414%26doi%3D10.3390%2Fijms18071414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+deacetylases+and+their+inhibitors+in+cancer%2C+neurological+diseases+and+immune+disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0lgJ2fRsch-YpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520deacetylases%2520and%2520their%2520inhibitors%2520in%2520cancer%252C%2520neurological%2520diseases%2520and%2520immune%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">HDAC6 a new cellular stress surveillance factor</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.4161/cc.7.1.5186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.4161%2Fcc.7.1.5186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=18196966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVKntLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=7-10&author=P.+Matthiasauthor=M.+Yoshidaauthor=S.+Khochbin&title=HDAC6+a+new+cellular+stress+surveillance+factor&doi=10.4161%2Fcc.7.1.5186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 a new cellular stress surveillance factor</span></div><div class="casAuthors">Matthias, Patrick; Yoshida, Minoru; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-10</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Less than a decade has passed since histone deacetylase harboring (HDAC6) was first identified and regarded as an unusual histone deacetylase harboring 2 catalytic domains.  Early demonstration of its cytoplasmic localization, its ubiquitin-binding, and its tubulin-deacetylase activities took HDAC6 far away from everything known to involve other histone-deacetylases.  Recent discoveries confirmed the very unique functions of HDAC6 among deacetylases and pointed to this protein as a master regulator of cell response to cytotoxic assaults.  HDAC6 appears both as a sensor of stressful stimuli and as an effector, which, thanks to its wide range of activities, mediates and coordinates appropriate cell responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Tzz-Xi44N7Vg90H21EOLACvtfcHk0lgJ2fRsch-YpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVKntLY%253D&md5=6b0784dad7b4fac9a31e803548bc7a84</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4161%2Fcc.7.1.5186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.7.1.5186%26sid%3Dliteratum%253Aachs%26aulast%3DMatthias%26aufirst%3DP.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DHDAC6%2520a%2520new%2520cellular%2520stress%2520surveillance%2520factor%26jtitle%3DCell%2520Cycle%26date%3D2008%26volume%3D7%26spage%3D7%26epage%3D10%26doi%3D10.4161%2Fcc.7.1.5186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, J.</span></span> <span> </span><span class="NLM_article-title">Action mechanisms of histone deacetylase inhibitors in the treatment of hematological malignancies</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1543</span>– <span class="NLM_lpage">1549</span>, <span class="refDoi"> DOI: 10.1111/cas.13062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1111%2Fcas.13062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27554046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlykur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1543-1549&author=Y.+Imaiauthor=Y.+Maruauthor=J.+Tanaka&title=Action+mechanisms+of+histone+deacetylase+inhibitors+in+the+treatment+of+hematological+malignancies&doi=10.1111%2Fcas.13062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Action mechanisms of histone deacetylase inhibitors in the treatment of hematological malignancies</span></div><div class="casAuthors">Imai, Yoichi; Maru, Yoshiro; Tanaka, Junji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1543-1549</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Histone deacetylases (HDACs) critically regulate gene expression by detg. the acetylation status of histones.  In addn., studies have increasingly focused on the activities of HDACs, esp. involving non-histone proteins, and their various biol. effects.  Aberrant HDAC expression obsd. in several kinds of human tumors makes HDACs potential targets for cancer treatment.  Several preclin. studies have suggested that HDAC inhibitors exhibit some efficacy in the treatment of acute myelogenous leukemia (AML) with AML1-ETO, which mediates transcriptional repression through its interaction with a complex including HDAC1.  Recurrent mutations in epigenetic regulators are found in T-cell lymphomas (TCLs), and HDAC inhibitors and hypomethylating agents were shown to act cooperatively in the treatment of TCLs.  Preclin. modeling has suggested that persistent activation of the signal transducer and activator of transcription (STAT) signaling pathway could serve as a useful biomarker of resistance to HDAC inhibitor in patients with cutaneous TCL.  Panobinostat, a pan-HDAC inhibitor, in combination with bortezomib and dexamethasone, has achieved longer progression-free survival in patients with relapsed/refractory multiple myeloma (MM) than the placebo in combination with bortezomib and dexamethasone.  Panobinostat inhibited MM cell growth by degrading protein phosphatase 3 catalytic subunit α (PPP3CA), a catalytic subunit of calcineurin.  This degrdn. was suggested to be mediated by the blockade of the chaperone function of heat shock protein 90 (HSP90) due to HDAC6 inhibition.  Aberrant PPP3CA expression in advanced MM indicated a possible correlation between high PPP3CA expression and the pathogenesis of MM.  Furthermore, PPP3CA was suggested as a common target of panobinostat and bortezomib.  This article is protected by copyright.  All rights reserved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs8mVwuDgQ9LVg90H21EOLACvtfcHk0lgJ2fRsch-YpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlykur3O&md5=d97540b4f33bb4ce6c044717785f9954</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1111%2Fcas.13062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13062%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DY.%26aulast%3DMaru%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DJ.%26atitle%3DAction%2520mechanisms%2520of%2520histone%2520deacetylase%2520inhibitors%2520in%2520the%2520treatment%2520of%2520hematological%2520malignancies%26jtitle%3DCancer%2520Sci.%26date%3D2016%26volume%3D107%26spage%3D1543%26epage%3D1549%26doi%3D10.1111%2Fcas.13062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">A patent review of histone deacetylase 6 inhibitors in neurodegenerative diseases (2014-2019</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1080/13543776.2019.1708901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1080%2F13543776.2019.1708901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=31865813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVKnsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=121-136&author=S.+Shenauthor=A.+P.+Kozikowski&title=A+patent+review+of+histone+deacetylase+6+inhibitors+in+neurodegenerative+diseases+%282014-2019&doi=10.1080%2F13543776.2019.1708901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A patent review of histone deacetylase 6 inhibitors in neurodegenerative diseases (2014-2019)</span></div><div class="casAuthors">Shen, Sida; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-136</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase 6 (HDAC6) is unique in comparison with other zinc-dependent HDAC family members.  An increasing amt. of evidence from clin. and preclin. research demonstrates the potential of HDAC6 inhibition as an effective therapeutic approach for the treatment of cancer, autoimmune diseases, as well as neurol. disorders.  The recently disclosed crystal structures of HDAC6-ligand complexes offer further means for achieving pharmacophore refinement, thus further accelerating the pace of HDAC6 inhibitor discovery in the last few years.: This review summarizes the latest clin. status of HDAC6 inhibitors, discusses pharmacol. applications of selective HDAC6 inhibitors in neurodegenerative diseases, and describes the patent applications dealing with HDAC6 inhibitors from 2014-2019 that have not been reported in research articles.: Phenylhydroxamate has proven a very useful scaffold in the discovery of potent and selective HDAC6 inhibitors.  However, weaknesses of the hydroxamate function such as metabolic instability and mutagenic potential limit its application in the neurol. field, where long-term administration is required.  The recent invention of oxadiazole-based ligands by pharmaceutical companies may provide a new opportunity to optimize the druglike properties of HDAC6 inhibitors for the treatment of neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCWThC7SH8r7Vg90H21EOLACvtfcHk0lh9HNIHYrYrvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVKnsb3J&md5=926d00ce3a0153953eb2f933599500b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1708901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1708901%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DA%2520patent%2520review%2520of%2520histone%2520deacetylase%25206%2520inhibitors%2520in%2520neurodegenerative%2520diseases%2520%25282014-2019%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2020%26volume%3D30%26spage%3D121%26epage%3D136%26doi%3D10.1080%2F13543776.2019.1708901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rébé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghiringhelli, F.</span></span> <span> </span><span class="NLM_article-title">STAT3, a master regulator of anti-tumor immune response</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1280</span>, <span class="refDoi"> DOI: 10.3390/cancers11091280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.3390%2Fcancers11091280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtVWmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1280&author=C.+R%C3%A9b%C3%A9author=F.+Ghiringhelli&title=STAT3%2C+a+master+regulator+of+anti-tumor+immune+response&doi=10.3390%2Fcancers11091280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3, a master regulator of anti-tumor immune response</span></div><div class="casAuthors">Rebe, Cedric; Ghiringhelli, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1280</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Immune cells in the tumor microenvironment regulate cancer growth.  Thus cancer progression is dependent on the activation or repression of transcription programs involved in the proliferation/activation of lymphoid and myeloid cells.  One of the main transcription factors involved in many of these pathways is the signal transducer and activator of transcription 3 (STAT3).  In this review we will focus on the role of STAT3 and its regulation, e.g., by phosphorylation or acetylation in immune cells and how it might impact immune cell function and tumor progression.  Moreover, we will review the ability of STAT3 to regulate checkpoint inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgk-poT_FrCrVg90H21EOLACvtfcHk0lh9HNIHYrYrvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtVWmsbs%253D&md5=89d95d832347103f511110d1f23e720d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3390%2Fcancers11091280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11091280%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25A9b%25C3%25A9%26aufirst%3DC.%26aulast%3DGhiringhelli%26aufirst%3DF.%26atitle%3DSTAT3%252C%2520a%2520master%2520regulator%2520of%2520anti-tumor%2520immune%2520response%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D1280%26doi%3D10.3390%2Fcancers11091280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock-Klotz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span> <span> </span><span class="NLM_article-title">A novel role for histone deacetylase 6 in the regulation of the tolerogenic STAT3/IL-10 pathway in APCs</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">2850</span>– <span class="NLM_lpage">2862</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1302778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.4049%2Fjimmunol.1302778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25108026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2qur3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2014&pages=2850-2862&author=F.+Chengauthor=M.+Lienlafauthor=H.-W.+Wangauthor=P.+Perez-Villarroelauthor=C.+Leeauthor=K.+Woanauthor=J.+Rock-Klotzauthor=E.+Sahakianauthor=D.+Woodsauthor=J.+Pinilla-Ibarzauthor=J.+Kalinauthor=J.+Taoauthor=W.+Hancockauthor=A.+Kozikowskiauthor=E.+Setoauthor=A.+Villagraauthor=E.+M.+Sotomayor&title=A+novel+role+for+histone+deacetylase+6+in+the+regulation+of+the+tolerogenic+STAT3%2FIL-10+pathway+in+APCs&doi=10.4049%2Fjimmunol.1302778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Role for Histone Deacetylase 6 in the Regulation of the Tolerogenic STAT3/IL-10 Pathway in APCs</span></div><div class="casAuthors">Cheng, Fengdong; Lienlaf, Maritza; Wang, Hong-Wei; Perez-Villarroel, Patricio; Lee, Calvin; Woan, Karrune; Rock-Klotz, Jennifer; Sahakian, Eva; Woods, David; Pinilla-Ibarz, Javier; Kalin, Jay; Tao, Jianguo; Hancock, Wayne; Kozikowski, Alan; Seto, Edward; Villagra, Alejandro; Sotomayor, Eduardo M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2850-2862</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">APCs are crit. in T cell activation and in the induction of T cell tolerance.  Epigenetic modifications of specific genes in the APC play a key role in this process, and among them histone deacetylases (HDACs) have emerged as key participants.  HDAC6, one of the members of this family of enzymes, has been shown to be involved in regulation of inflammatory and immune responses.  In this study, to our knowledge we show for the first time that genetic or pharmacol. disruption of HDAC6 in macrophages and dendritic cells results in diminished prodn. of the immunosuppressive cytokine IL-10 and induction of inflammatory APCs that effectively activate Ag-specific naive T cells and restore the responsiveness of anergic CD4+ T cells.  Mechanistically, we have found that HDAC6 forms a previously unknown mol. complex with STAT3, assocn. that was detected in both the cytoplasmic and nuclear compartments of the APC.  By using HDAC6 recombinant mutants we identified the domain comprising amino acids 503-840 as being required for HDAC6 interaction with STAT3.  Furthermore, by re-chromatin immunopptn. we confirmed that HDAC6 and STAT3 are both recruited to the same DNA sequence within the Il10 gene promoter.  Of note, disruption of this complex by knocking down HDAC6 resulted in decreased STAT3 phosphorylation-but no changes in STAT3 acetylation-as well as diminished recruitment of STAT3 to the Il10 gene promoter region.  The addnl. demonstration that a selective HDAC6 inhibitor disrupts this STAT3/IL-10 tolerogenic axis points to HDAC6 as a novel mol. target in APCs to overcome immune tolerance and tips the balance toward T cell immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSLvMP_-UNx7Vg90H21EOLACvtfcHk0lh9HNIHYrYrvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2qur3P&md5=4d547665a3108fcf168f353bba4f4646</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1302778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1302778%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.-W.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DWoan%26aufirst%3DK.%26aulast%3DRock-Klotz%26aufirst%3DJ.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DWoods%26aufirst%3DD.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DKalin%26aufirst%3DJ.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DHancock%26aufirst%3DW.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26atitle%3DA%2520novel%2520role%2520for%2520histone%2520deacetylase%25206%2520in%2520the%2520regulation%2520of%2520the%2520tolerogenic%2520STAT3%252FIL-10%2520pathway%2520in%2520APCs%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D193%26spage%3D2850%26epage%3D2862%26doi%3D10.4049%2Fjimmunol.1302778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pabon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montecino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Essential role of HDAC6 in the regulation of PD-L1 in melanoma</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">735</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2015.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.molonc.2015.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=26775640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=735-750&author=M.+Lienlafauthor=P.+Perez-Villarroelauthor=T.+Knoxauthor=M.+Pabonauthor=E.+Sahakianauthor=J.+Powersauthor=K.+V.+Woanauthor=C.+Leeauthor=F.+Chengauthor=S.+Dengauthor=K.+S.+M.+Smalleyauthor=M.+Montecinoauthor=A.+Kozikowskiauthor=J.+Pinilla-Ibarzauthor=A.+Sarnaikauthor=E.+Setoauthor=J.+Weberauthor=E.+M.+Sotomayorauthor=A.+Villagra&title=Essential+role+of+HDAC6+in+the+regulation+of+PD-L1+in+melanoma&doi=10.1016%2Fj.molonc.2015.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of HDAC6 in the regulation of PD-L1 in melanoma</span></div><div class="casAuthors">Lienlaf, M.; Perez-Villarroel, P.; Knox, T.; Pabon, M.; Sahakian, E.; Powers, J.; Woan, K. V.; Lee, C.; Cheng, F.; Deng, S.; Smalley, K. S. M.; Montecino, M.; Kozikowski, A.; Pinilla-Ibarz, J.; Sarnaik, A.; Seto, E.; Weber, J.; Sotomayor, E. M.; Villagra, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">735-750</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs), originally described as histone modifiers, have more recently been demonstrated to target a variety of other proteins unrelated to the chromatin environment.  In this context, our present work demonstrates that the pharmacol. or genetic abrogation of HDAC6 in primary melanoma samples and cell lines, down-regulates the expression of PD-L1, an important co-stimulatory mol. expressed in cancer cells, which activates the inhibitory regulatory pathway PD-1 in T-cells.  Our data suggests that this novel mechanism of PD-L1 regulation is mainly mediated by the influence of HDAC6 over the recruitment and activation of STAT3.  Addnl., we obsd. that selective HDAC6 inhibitors impairs tumor growth and reduce the in vivo expression of several inhibitory check-point mols. and other regulatory pathways involved in immunosurveillance.  Most importantly, these results provide a key pre-clin. rationale and justification to further study isotype selective HDAC6 inhibitors as potential immuno-modulatory agents in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnUp3ctcdjRLVg90H21EOLACvtfcHk0lhiASwQZeOk3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtFWkuw%253D%253D&md5=6ead389dab6bd5f4fc621c452cd4e303</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2015.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2015.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DPabon%26aufirst%3DM.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DWoan%26aufirst%3DK.%2BV.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DMontecino%26aufirst%3DM.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DSarnaik%26aufirst%3DA.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DEssential%2520role%2520of%2520HDAC6%2520in%2520the%2520regulation%2520of%2520PD-L1%2520in%2520melanoma%26jtitle%3DMol.%2520Oncol.%26date%3D2016%26volume%3D10%26spage%3D735%26epage%3D750%26doi%3D10.1016%2Fj.molonc.2015.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonepalle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gracia-Hernandez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Selective HDAC6 inhibitors improve anti-PD-1 immune checkpoint blockade therapy by decreasing the anti-inflammatory phenotype of macrophages and down-regulation of immunosuppressive proteins in tumor cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">6136</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-42237-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fs41598-019-42237-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=30992475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FmtFOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=6136&author=T.+Knoxauthor=E.+Sahakianauthor=D.+Banikauthor=M.+Hadleyauthor=E.+Palmerauthor=S.+Noonepalleauthor=J.+Kimauthor=J.+Powersauthor=M.+Gracia-Hernandezauthor=V.+Oliveiraauthor=F.+Chengauthor=J.+Chenauthor=C.+Barinkaauthor=J.+Pinilla-Ibarzauthor=N.+H.+Leeauthor=A.+Kozikowskiauthor=A.+Villagra&title=Selective+HDAC6+inhibitors+improve+anti-PD-1+immune+checkpoint+blockade+therapy+by+decreasing+the+anti-inflammatory+phenotype+of+macrophages+and+down-regulation+of+immunosuppressive+proteins+in+tumor+cells&doi=10.1038%2Fs41598-019-42237-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selective HDAC6 inhibitors improve anti-PD-1 immune checkpoint blockade therapy by decreasing the anti-inflammatory phenotype of macrophages and down-regulation of immunosuppressive proteins in tumor cells</span></div><div class="casAuthors">Knox Tessa; Banik Debarati; Hadley Melissa; Palmer Erica; Noonepalle Satish; Kim Jennifer; Gracia-Hernandez Maria; Cheng Fengdong; Chen Jie; Lee Norman H; Villagra Alejandro; Sahakian Eva; Powers John; Pinilla-Ibarz Javier; Oliveira Vasco; Barinka Cyril; Kozikowski Alan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6136</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone deacetylases (HDACs) are involved in diverse cellular regulatory mechanisms including non-canonical functions outside the chromatin environment.  Several publications have demonstrated that selective HDAC inhibitors (HDACi) can influence tumor immunogenicity and the functional activity of specific immune cells.  In particular, the selective inhibition of HDAC6 has been reported to decrease tumor growth in several malignancies.  However, there is still no clarity about the cellular components mediating this effect.  In this study, we evaluated the HDAC6i Nexturastat A as a priming agent to facilitate the transition of the tumor microenvironment from "cold" to "hot", and potentially augment immune check-point blockade therapies.  This combination modality demonstrated to significantly reduce tumor growth in syngeneic melanoma tumor models.  Additionally, we observed a complete neutralization of the up-regulation of PD-L1 and other immunosuppressive pathways induced by the treatment with anti-PD-1 blockade.  This combination also showed profound changes in the tumor microenvironment such as enhanced infiltration of immune cells, increased central and effector T cell memory, and a significant reduction of pro-tumorigenic M2 macrophages.  The evaluation of individual components of the tumor microenvironment suggested that the in vivo anti-tumor activity of HDAC6i is mediated by its effect on tumor cells and tumor-associated macrophages, and not directly over T cells.  Overall, our results indicate that selective HDAC6i could be used as immunological priming agents to sensitize immunologically "cold" tumors and subsequently improve ongoing immune check-point blockade therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSl6RaBRQ4Kdw8IxFYvehhofW6udTcc2ebwoavyZkTlObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FmtFOrsg%253D%253D&md5=137a9261e021e60b002d9312608eba76</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-42237-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-42237-3%26sid%3Dliteratum%253Aachs%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DBanik%26aufirst%3DD.%26aulast%3DHadley%26aufirst%3DM.%26aulast%3DPalmer%26aufirst%3DE.%26aulast%3DNoonepalle%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DGracia-Hernandez%26aufirst%3DM.%26aulast%3DOliveira%26aufirst%3DV.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DN.%2BH.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DSelective%2520HDAC6%2520inhibitors%2520improve%2520anti-PD-1%2520immune%2520checkpoint%2520blockade%2520therapy%2520by%2520decreasing%2520the%2520anti-inflammatory%2520phenotype%2520of%2520macrophages%2520and%2520down-regulation%2520of%2520immunosuppressive%2520proteins%2520in%2520tumor%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D6136%26doi%3D10.1038%2Fs41598-019-42237-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brochier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vistoli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Rational design and simple chemistry yield a superior, neuroprotective HDAC6 inhibitor, tubastatin A</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">10842</span>– <span class="NLM_lpage">10846</span>, <span class="refDoi"> DOI: 10.1021/ja102758v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja102758v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=10842-10846&author=K.+V.+Butlerauthor=J.+Kalinauthor=C.+Brochierauthor=G.+Vistoliauthor=B.+Langleyauthor=A.+P.+Kozikowski&title=Rational+design+and+simple+chemistry+yield+a+superior%2C+neuroprotective+HDAC6+inhibitor%2C+tubastatin+A&doi=10.1021%2Fja102758v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design and Simple Chemistry Yield a Superior, Neuroprotective HDAC6 Inhibitor, Tubastatin A</span></div><div class="casAuthors">Butler, Kyle V.; Kalin, Jay; Brochier, Camille; Vistoli, Guilio; Langley, Brett; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10842-10846</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design combined with homol. modeling techniques were used to develop potent inhibitors of HDAC6 that display superior selectivity for the HDAC6 isoenzyme compared to other inhibitors.  These inhibitors can be assembled in a few synthetic steps, and thus are readily scaled up for in vivo studies.  An optimized compd. from this series, designated Tubastatin A, was tested in primary cortical neuron cultures in which it was found to induce elevated levels of acetylated α-tubulin, but not histone, consistent with its HDAC6 selectivity.  Tubastatin A also conferred dose-dependent protection in primary cortical neuron cultures against glutathione depletion-induced oxidative stress.  Importantly, when given alone at all concns. tested, this hydroxamate-contg. HDAC6-selective compd. displayed no neuronal toxicity, thus, forecasting the potential application of this agent and its analogs to neurodegenerative conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruEaJPWQfjH7Vg90H21EOLACvtfcHk0ljH1KiKy2J2oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D&md5=f0cc68bb68f94ad1ef48f88cc6b57129</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fja102758v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja102758v%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKalin%26aufirst%3DJ.%26aulast%3DBrochier%26aufirst%3DC.%26aulast%3DVistoli%26aufirst%3DG.%26aulast%3DLangley%26aufirst%3DB.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DRational%2520design%2520and%2520simple%2520chemistry%2520yield%2520a%2520superior%252C%2520neuroprotective%2520HDAC6%2520inhibitor%252C%2520tubastatin%2520A%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D10842%26epage%3D10846%26doi%3D10.1021%2Fja102758v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. A.</span></span> <span> </span><span class="NLM_article-title">Development and therapeutic implications of selective histone deacetylase 6 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6297</span>– <span class="NLM_lpage">6313</span>, <span class="refDoi"> DOI: 10.1021/jm4001659</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4001659" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslSntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6297-6313&author=J.+H.+Kalinauthor=J.+A.+Bergman&title=Development+and+therapeutic+implications+of+selective+histone+deacetylase+6+inhibitors&doi=10.1021%2Fjm4001659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Therapeutic Implications of Selective Histone Deacetylase 6 Inhibitors</span></div><div class="casAuthors">Kalin, Jay H.; Bergman, Joel A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6297-6313</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This Perspective provides an in depth look at the numerous disease states in which histone deacetylase 6 (HDAC6) has been implicated.  The physiol. pathways, protein-protein interactions, and non-histone substrates relating to different pathol. conditions are discussed with regard to HDAC6.  Furthermore, the compds. and methods used to modulate HDAC6 activity are profiled.  The latter half of this Perspective analyzes reported HDAC6 selective inhibitors in terms of structure, potency, and selectivity over the other HDAC isoforms with the intent of providing a comprehensive overview of the mol. tools available.  Potential obstacles and future directions of HDAC6 research are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvFD0xh4CLiLVg90H21EOLACvtfcHk0ljH1KiKy2J2oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslSntrc%253D&md5=e54e7e8b172aba337581341360c0685c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm4001659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4001659%26sid%3Dliteratum%253Aachs%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DBergman%26aufirst%3DJ.%2BA.%26atitle%3DDevelopment%2520and%2520therapeutic%2520implications%2520of%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6297%26epage%3D6313%26doi%3D10.1021%2Fjm4001659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Vreese, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Hooghe, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and applications of benzohydroxamic acid-based histone deacetylase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.ejmech.2017.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28453994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslOhur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2017&pages=174-195&author=R.+De%0AVreeseauthor=M.+D%E2%80%99Hooghe&title=Synthesis+and+applications+of+benzohydroxamic+acid-based+histone+deacetylase+inhibitors&doi=10.1016%2Fj.ejmech.2017.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and applications of benzohydroxamic acid-based histone deacetylase inhibitors</span></div><div class="casAuthors">De Vreese, Rob; D'hooghe, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">174-195</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">This paper provides an overview of the synthesis and biol. activity of the most representative benzohydroxamic acid-based histone deacetylase inhibitors published to date.  Benzohydroxamic acids comprise an important class of HDAC inhibitors, and recently several of these structures have been evaluated in clin. trials for the treatment of a variety of cancers.  In this overview, benzohydroxamic acids were divided in four different classes based on their reported selectivity towards zinc-dependent HDACs: a first and major class consists of HDAC6 selective inhibitors, a second class deals with pan-HDAC inhibitors, a third class comprises HDAC8 selective inhibitors and a fourth, minor class includes dual HDAC6/8 selective inhibitors.  Through this approach, structure-activity relationships were identified for each class, which could help future researchers in the design and development of novel benzohydroxamic acid-based HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyKAa1s9hoSbVg90H21EOLACvtfcHk0ljH1KiKy2J2oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslOhur0%253D&md5=769c343bdbb3fe610eff001afa79e0bb</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVreese%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Hooghe%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520applications%2520of%2520benzohydroxamic%2520acid-based%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D135%26spage%3D174%26epage%3D195%26doi%3D10.1016%2Fj.ejmech.2017.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. P.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the discovery of potent and selective HDAC6 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">1406</span>– <span class="NLM_lpage">1418</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.ejmech.2017.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=29133060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2ntbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2018&pages=1406-1418&author=X.+X.+Wangauthor=R.+Z.+Wanauthor=Z.+P.+Liu&title=Recent+advances+in+the+discovery+of+potent+and+selective+HDAC6+inhibitors&doi=10.1016%2Fj.ejmech.2017.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the discovery of potent and selective HDAC6 inhibitors</span></div><div class="casAuthors">Wang, Xiu-Xiu; Wan, Ren-Zhong; Liu, Zhao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1406-1418</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase HDAC6, a member of the class IIb HDAC family, is unique among HDAC enzymes in having two active catalytic domains, and has unique physiol. function.  In addn. to the modification of histone, HDAC6 targets specific substrates including α-tubulin and HSP90, and are involved in protein trafficking and degrdn., cell shape and migration.  Selective HDAC6 inhibitors are an emerging class of pharmaceuticals due to the involvement of HDAC6 in different pathways related to neurodegenerative diseases, cancer, and immunol.  Therefore, extensive investigations have been made in the discovery of selective HDAC6 inhibitors.  Based on their different zinc binding groups (ZBGs), in this review, HDAC6 inhibitors are grouped as hydroxamic acids, a sulfur contg. ZBG based derivs. and other ZBG-derived compds., and their enzymic inhibitory activity, selectivity and other biol. activities are introduced and summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdsNBEWGhpCrVg90H21EOLACvtfcHk0li8hviQ258bjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2ntbrM&md5=f24fea971dbed247e1bb2568f85d1703</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%2BX.%26aulast%3DWan%26aufirst%3DR.%2BZ.%26aulast%3DLiu%26aufirst%3DZ.%2BP.%26atitle%3DRecent%2520advances%2520in%2520the%2520discovery%2520of%2520potent%2520and%2520selective%2520HDAC6%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D143%26spage%3D1406%26epage%3D1418%26doi%3D10.1016%2Fj.ejmech.2017.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faria
Freitas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuendet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, P.</span></span> <span> </span><span class="NLM_article-title">HDAC inhibitors: a 2013-2017 patent survey</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1080/13543776.2018.1459568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1080%2F13543776.2018.1459568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVSns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=365-381&author=M.+Faria%0AFreitasauthor=M.+Cuendetauthor=P.+Bertrand&title=HDAC+inhibitors%3A+a+2013-2017+patent+survey&doi=10.1080%2F13543776.2018.1459568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC inhibitors: a 2013-2017 patent survey</span></div><div class="casAuthors">Faria Freitas, Micaela; Cuendet, Muriel; Bertrand, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">365-381</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A Review.  Zinc-dependent histone deacetylases (HDAC) inhibitors represent an important class of biol. active compds. with four of them approved by the FDA.  A wide range of mols. has been reported for applications in several human diseases.: This review covers recent efforts in the synthesis and applications of HDAC inhibitors from 2013-2017.: HDAC inhibitors represent an important class of biol. active compds. for single or combination therapies.  The current synthetic methodologies are oriented towards selective HDAC isoforms to achieve better therapeutic effects.  Among the recent patents available, most of them focus on HDAC6 selective inhibitors.  Beside this search for isoform selectivity, the quest for zinc binding groups with better pharmacokinetic properties and high potency against HDACs only motivates medicinal chemists, as well as the design of inhibitors targeting HDACs and at the same time another biol. target.  If the major applications are for anticancer activity, one can note the emerging applications in neurol. or metabolic disorders or for the stimulation of the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgdOR23sC-crVg90H21EOLACvtfcHk0li8hviQ258bjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVSns7w%253D&md5=0bed63ff4bcd3599758686197372f35b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F13543776.2018.1459568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2018.1459568%26sid%3Dliteratum%253Aachs%26aulast%3DFaria%2BFreitas%26aufirst%3DM.%26aulast%3DCuendet%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DP.%26atitle%3DHDAC%2520inhibitors%253A%2520a%25202013-2017%2520patent%2520survey%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2018%26volume%3D28%26spage%3D365%26epage%3D381%26doi%3D10.1080%2F13543776.2018.1459568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flipo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassonneville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deprez-Poulain, R.</span></span> <span> </span><span class="NLM_article-title">Hydroxamates: relationships between structure and plasma stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6790</span>– <span class="NLM_lpage">6802</span>, <span class="refDoi"> DOI: 10.1021/jm900648x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900648x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6790-6802&author=M.+Flipoauthor=J.+Chartonauthor=A.+Hocineauthor=S.+Dassonnevilleauthor=B.+Deprezauthor=R.+Deprez-Poulain&title=Hydroxamates%3A+relationships+between+structure+and+plasma+stability&doi=10.1021%2Fjm900648x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxamates: Relationships between Structure and Plasma Stability</span></div><div class="casAuthors">Flipo, Marion; Charton, Julie; Hocine, Akila; Dassonneville, Sandrine; Deprez, Benoit; Deprez-Poulain, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6790-6802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydroxamates are valuable tools for chem. biol. as well as interesting leads for medicinal chem.  Although many hydroxamates display nanomolar activities against metalloproteases, only three hydroxamates have reached the market, among which is the HDAC inhibitor vorinostat.  Failures in development are generally attributed to lack of selectivity, toxicity, or poor stability.  To help medicinal chemists with respect to plasma stability, the authors have performed the first and preliminary study on structure-plasma stability for hydroxamates.  The authors define some structural rules to predict or improve the plasma stability in the preclin. stage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk66ooWU1NhbVg90H21EOLACvtfcHk0li8hviQ258bjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GgsbnJ&md5=069a87988fbeb68a248d006c7f6307a0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm900648x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900648x%26sid%3Dliteratum%253Aachs%26aulast%3DFlipo%26aufirst%3DM.%26aulast%3DCharton%26aufirst%3DJ.%26aulast%3DHocine%26aufirst%3DA.%26aulast%3DDassonneville%26aufirst%3DS.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DDeprez-Poulain%26aufirst%3DR.%26atitle%3DHydroxamates%253A%2520relationships%2520between%2520structure%2520and%2520plasma%2520stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6790%26epage%3D6802%26doi%3D10.1021%2Fjm900648x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szarics, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benoy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Den Bosch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eubanks, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jope, R. S.</span></span> <span> </span><span class="NLM_article-title">Brain penetrable histone deacetylase 6 inhibitor SW-100 ameliorates memory and learning impairments in a mouse model of Fragile X syndrome</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1679</span>– <span class="NLM_lpage">1695</span>, <span class="refDoi"> DOI: 10.1021/acschemneuro.8b00600</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.8b00600" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitl2jtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1679-1695&author=A.+P.+Kozikowskiauthor=S.+Shenauthor=M.+Pardoauthor=M.+T.+Tavaresauthor=D.+Szaricsauthor=V.+Benoyauthor=C.+A.+Zimprichauthor=Z.+Kutilauthor=G.+Zhangauthor=C.+Ba%C5%99inkaauthor=M.+B.+Robersauthor=L.+Van+Den+Boschauthor=J.+H.+Eubanksauthor=R.+S.+Jope&title=Brain+penetrable+histone+deacetylase+6+inhibitor+SW-100+ameliorates+memory+and+learning+impairments+in+a+mouse+model+of+Fragile+X+syndrome&doi=10.1021%2Facschemneuro.8b00600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Brain Penetrable Histone Deacetylase 6 Inhibitor SW-100 Ameliorates Memory and Learning Impairments in a Mouse Model of Fragile X Syndrome</span></div><div class="casAuthors">Kozikowski, Alan P.; Shen, Sida; Pardo, Marta; Tavares, Mauricio T.; Szarics, Dora; Benoy, Veronick; Zimprich, Chad A.; Kutil, Zsofia; Zhang, Guiping; Barinka, Cyril; Robers, Matthew B.; Van Den Bosch, Ludo; Eubanks, James H.; Jope, Richard S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1679-1695</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Disease-modifying therapies are needed for Fragile X Syndrome (FXS), as at present there are no effective treatments or cures.  Herein, we report on a tetrahydroquinoline-based selective histone deacetylase 6 (HDAC6) inhibitor SW-100, its pharmacol. and ADMET properties, and its ability to improve upon memory performance in a mouse model of FXS, Fmr1-/- mice.  This small mol. demonstrates good brain penetrance, low-nanomolar potency for the inhibition of HDAC6 (IC50 = 2.3 nM), with at least a thousand-fold selectivity over all other class I, II, and IV HDAC isoforms.  Moreover, through its inhibition of the α-tubulin deacetylase domain of HDAC6 (CD2), in cells SW-100 upregulates α-tubulin acetylation with no effect on histone acetylation and selectively restores the impaired acetylated α-tubulin levels in the hippocampus of Fmr1-/- mice.  Lastly, SW-100 ameliorates several memory and learning impairments in Fmr1-/- mice, thus modeling the intellectual deficiencies assocd. with FXS, and hence providing a strong rationale for pursuing HDAC6-based therapies for the treatment of this rare disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphc-8bfGiZh7Vg90H21EOLACvtfcHk0ljzVZQtlj6G-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitl2jtb3E&md5=b1e70a1a45522720d481985b3a4339f3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.8b00600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.8b00600%26sid%3Dliteratum%253Aachs%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DShen%26aufirst%3DS.%26aulast%3DPardo%26aufirst%3DM.%26aulast%3DTavares%26aufirst%3DM.%2BT.%26aulast%3DSzarics%26aufirst%3DD.%26aulast%3DBenoy%26aufirst%3DV.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DVan%2BDen%2BBosch%26aufirst%3DL.%26aulast%3DEubanks%26aufirst%3DJ.%2BH.%26aulast%3DJope%26aufirst%3DR.%2BS.%26atitle%3DBrain%2520penetrable%2520histone%2520deacetylase%25206%2520inhibitor%2520SW-100%2520ameliorates%2520memory%2520and%2520learning%2520impairments%2520in%2520a%2520mouse%2520model%2520of%2520Fragile%2520X%2520syndrome%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2019%26volume%3D10%26spage%3D1679%26epage%3D1695%26doi%3D10.1021%2Facschemneuro.8b00600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Why hydroxamates may not be the best histone deacetylase inhibitors--What some may have forgotten or would rather forget?</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fcmdc.201500486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=26603496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=15-21&author=S.+Shenauthor=A.+P.+Kozikowski&title=Why+hydroxamates+may+not+be+the+best+histone+deacetylase+inhibitors%2D%2DWhat+some+may+have+forgotten+or+would+rather+forget%3F&doi=10.1002%2Fcmdc.201500486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Why Hydroxamates May Not Be the Best Histone Deacetylase Inhibitors-What Some May Have Forgotten or Would Rather Forget?</span></div><div class="casAuthors">Shen, Sida; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-21</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Hydroxamate-based histone deacetylase inhibitors (HDACIs) have been approved as therapeutic agents by the US Food and Drug Administration for use in oncol. applications.  While the potential utility of such HDACIs in other areas of medicinal chem. is tremendous, there are significant concerns that "pan-HDAC inhibitors" may be too broadly acting and/or toxic for clin. use beyond oncol.  In addn. to the isoenzyme selectivity challenge, the potential mutagenicity of hydroxamate-contg. HDAC inhibitors represents a major hindrance in their application to other therapeutic areas.  Herein the authors report on the mutagenicity of known hydroxamates, discuss the mechanisms responsible for their genotoxicity, and review some of the current alternatives to hydroxamates.  The authors conclude that the hydroxamate group, while providing high-potency HDACIs, is not necessarily the best zinc-binding group for HDACI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplftHMFVoN6rVg90H21EOLACvtfcHk0ljzVZQtlj6G-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2jsrbL&md5=5183d2c0a4c86fe67bea480a66bc5dc6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500486%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DWhy%2520hydroxamates%2520may%2520not%2520be%2520the%2520best%2520histone%2520deacetylase%2520inhibitors--What%2520some%2520may%2520have%2520forgotten%2520or%2520would%2520rather%2520forget%253F%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D15%26epage%3D21%26doi%3D10.1002%2Fcmdc.201500486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keusch, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melancon, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helquist, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span> <span> </span><span class="NLM_article-title">Structural insights into HDAC6 tubulin deacetylation and its selective inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnchembio.2140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27454931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=748-754&author=Y.+Miyakeauthor=J.+J.+Keuschauthor=L.+Wangauthor=M.+Saitoauthor=D.+Hessauthor=X.+Wangauthor=B.+J.+Melanconauthor=P.+Helquistauthor=H.+Gutauthor=P.+Matthias&title=Structural+insights+into+HDAC6+tubulin+deacetylation+and+its+selective+inhibition&doi=10.1038%2Fnchembio.2140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into HDAC6 tubulin deacetylation and its selective inhibition</span></div><div class="casAuthors">Miyake, Yasuyuki; Keusch, Jeremy J.; Wang, Longlong; Saito, Makoto; Hess, Daniel; Wang, Xiaoning; Melancon, Bruce J.; Helquist, Paul; Gut, Heinz; Matthias, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">748-754</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors report crystal structures of zebrafish histone deacetylase 6 (HDAC6) catalytic domains in tandem or as single domains in complex with the (R) and (S) enantiomers of trichostatin A (TSA) or with the HDAC6-specific inhibitor, nexturastat A.  The tandem domains formed, together with the inter-domain linker, an ellipsoid-shaped complex with pseudo-twofold symmetry.  The authors identified important active site differences between both catalytic domains and revealed the binding mode of HDAC6 selective inhibitors.  HDAC inhibition assays with (R)- and (S)-TSA showed that (R)-TSA was a broad-range inhibitor, whereas (S)-TSA had moderate selectivity for HDAC6.  The authors identified a uniquely positioned α-helix and a flexible Trp residue in the loop joining α-helixes H20 to H21 as crit. for deacetylation of the physiol. substrate, tubulin.  Using single-mol. measurements and biochem. assays, the authors demonstrated that HDAC6 catalytic domain 2 deacetylated α-tubulin Lys-40 in the lumen of microtubules, but that its preferred substrate was unpolymd. tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpERJRurS1cC7Vg90H21EOLACvtfcHk0ljzVZQtlj6G-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rM&md5=4abe529ab89615df33f54350a1a8c69a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2140%26sid%3Dliteratum%253Aachs%26aulast%3DMiyake%26aufirst%3DY.%26aulast%3DKeusch%26aufirst%3DJ.%2BJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSaito%26aufirst%3DM.%26aulast%3DHess%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMelancon%26aufirst%3DB.%2BJ.%26aulast%3DHelquist%26aufirst%3DP.%26aulast%3DGut%26aufirst%3DH.%26aulast%3DMatthias%26aufirst%3DP.%26atitle%3DStructural%2520insights%2520into%2520HDAC6%2520tubulin%2520deacetylation%2520and%2520its%2520selective%2520inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D748%26epage%3D754%26doi%3D10.1038%2Fnchembio.2140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahendran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breslow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Unusual zinc-binding mode of HDAC6-selective hydroxamate inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">13459</span>– <span class="NLM_lpage">13464</span>, <span class="refDoi"> DOI: 10.1073/pnas.1718823114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1073%2Fpnas.1718823114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=29203661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=13459-13464&author=N.+J.+Porterauthor=A.+Mahendranauthor=R.+Breslowauthor=D.+W.+Christianson&title=Unusual+zinc-binding+mode+of+HDAC6-selective+hydroxamate+inhibitors&doi=10.1073%2Fpnas.1718823114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Unusual zinc-binding mode of HDAC6-selective hydroxamate inhibitors</span></div><div class="casAuthors">Porter, Nicholas J.; Mahendran, Adaickapillai; Breslow, Ronald; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">13459-13464</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) regulate myriad cellular processes by catalyzing the hydrolysis of acetyl-L-lysine residues in histone and nonhistone proteins.  The Zn2+-dependent class IIb enzyme HDAC6 regulates microtubule function by deacetylating α-tubulin, which suppresses microtubule dynamics and leads to cell cycle arrest and apoptosis.  Accordingly, HDAC6 is a target for the development of selective inhibitors that might be useful in new therapeutic approaches for the treatment of cancer, neurodegenerative diseases, and other disorders.  Here, we present high-resoln. structures of catalytic domain 2 from Danio rerio HDAC6 (henceforth simply "HDAC6") complexed with compds. that selectively inhibit HDAC6 while maintaining nanomolar inhibitory potency: N-hydroxy-4-((N(2-hydroxyethyl)-2-phenylacetamido)methyl)-benzamide (HPB), ACY-1215 (Ricolinostat), and ACY-1083.  These structures reveal that an unusual monodentate Zn2+ coordination mode is exploited by sterically bulky HDAC6-selective phenylhydroxamate inhibitors.  We addnl. report the ultrahigh-resoln. structure of the HDAC6-trichostatin A complex, which reveals two Zn2+-binding conformers for the inhibitor: a major conformer (70%) with canonical bidentate hydroxamate-Zn2+ coordination geometry and a minor conformer (30%) with monodentate hydroxamate-Zn2+ coordination geometry, reflecting a free energy difference of only 0.5 kcal/mol.  The minor conformer is not visible in lower resoln. structure detns.  Structural comparisons of HDAC6-inhibitor complexes with class I HDACs suggest active site features that contribute to the isoenzyme selectivity obsd. in biochem. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0oUH1X_04p7Vg90H21EOLACvtfcHk0lhALIEi1x4ENQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsrbI&md5=07180756239eac4891642cf59a2e6992</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1718823114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1718823114%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DMahendran%26aufirst%3DA.%26aulast%3DBreslow%26aufirst%3DR.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DUnusual%2520zinc-binding%2520mode%2520of%2520HDAC6-selective%2520hydroxamate%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D13459%26epage%3D13464%26doi%3D10.1073%2Fpnas.1718823114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fnchembio.2134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27454933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=741-747&author=Y.+Haiauthor=D.+W.+Christianson&title=Histone+deacetylase+6+structure+and+molecular+basis+of+catalysis+and+inhibition&doi=10.1038%2Fnchembio.2134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span></div><div class="casAuthors">Hai, Yang; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">741-747</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a crit. target for drug design because of its role in oncogenic transformation and cancer metastasis, and is unique among all histone deacetylases in that it contains tandem catalytic domains designated CD1 and CD2.  We now report the crystal structures of CD2 from Homo sapiens HDAC6 and of CD1 and CD2 from Danio rerio HDAC6.  We correlated these structures with activity measurements using 13 different substrates.  The catalytic activity of CD2 from both species exhibited broad substrate specificity, whereas that of CD1 was highly specific for substrates bearing C-terminal acetyllysine residues.  Crystal structures of substrate complexes yielded unprecedented snapshots of the catalytic mechanism.  Addnl., crystal structures of complexes with eight different inhibitors, including belinostat and panobinostat (currently used in cancer chemotherapy), the macrocyclic tetrapeptide HC toxin, and the HDAC6-specific inhibitor N-hydroxy-4-(2-((2-hydroxyethyl)(phenyl)amino)-2-oxoethyl)benzamide, revealed surprising new insight regarding changes in Zn2+ coordination and isoenzyme-specific inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCfziGBHBRLVg90H21EOLACvtfcHk0lhALIEi1x4ENQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ&md5=e167bc6f66e00923da4f2bced2b306b9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2134%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520deacetylase%25206%2520structure%2520and%2520molecular%2520basis%2520of%2520catalysis%2520and%2520inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D741%26epage%3D747%26doi%3D10.1038%2Fnchembio.2134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diedrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooker, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 6-selective inhibitors and the influence of capping groups on hydroxamate-zinc denticity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8054</span>– <span class="NLM_lpage">8060</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFansLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8054-8060&author=N.+J.+Porterauthor=J.+D.+Oskoauthor=D.+Diedrichauthor=T.+Kurzauthor=J.+M.+Hookerauthor=F.+K.+Hansenauthor=D.+W.+Christianson&title=Histone+deacetylase+6-selective+inhibitors+and+the+influence+of+capping+groups+on+hydroxamate-zinc+denticity&doi=10.1021%2Facs.jmedchem.8b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase 6-Selective Inhibitors and the Influence of Capping Groups on Hydroxamate-Zinc Denticity</span></div><div class="casAuthors">Porter, Nicholas J.; Osko, Jeremy D.; Diedrich, Daniela; Kurz, Thomas; Hooker, Jacob M.; Hansen, Finn K.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8054-8060</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four crystal structures are presented of histone deacetylase 6 (HDAC6) complexes with para-substituted phenylhydromaxamate inhibitors, including bulky peptoids.  These structures provide insight regarding the design of capping groups that confer selectivity for binding to HDAC6, specifically with regard to interactions in a pocket formed by the L1 loop.  Capping group interactions may also influence hydroxamate-Zn2+ coordination with monodentate or bidentate geometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq54PM6p6wpnrVg90H21EOLACvtfcHk0lhALIEi1x4ENQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFansLnE&md5=f1d227d7668d09e121e5967b2d65f1a5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01013%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DDiedrich%26aufirst%3DD.%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26aulast%3DHansen%26aufirst%3DF.%2BK.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520deacetylase%25206-selective%2520inhibitors%2520and%2520the%2520influence%2520of%2520capping%2520groups%2520on%2520hydroxamate-zinc%2520denticity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8054%26epage%3D8060%26doi%3D10.1021%2Facs.jmedchem.8b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ustinova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlicek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonepalle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of a new isoxazole-3-hydroxamate-based histone deacetylase 6 inhibitor SS-208 with antitumor activity in syngeneic melanoma mouse models</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8557</span>– <span class="NLM_lpage">8577</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00946</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00946" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFymtbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8557-8577&author=S.+Shenauthor=M.+Hadleyauthor=K.+Ustinovaauthor=J.+Pavlicekauthor=T.+Knoxauthor=S.+Noonepalleauthor=M.+T.+Tavaresauthor=C.+A.+Zimprichauthor=G.+Zhangauthor=M.+B.+Robersauthor=C.+Ba%C5%99inkaauthor=A.+P.+Kozikowskiauthor=A.+Villagra&title=Discovery+of+a+new+isoxazole-3-hydroxamate-based+histone+deacetylase+6+inhibitor+SS-208+with+antitumor+activity+in+syngeneic+melanoma+mouse+models&doi=10.1021%2Facs.jmedchem.9b00946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a New Isoxazole-3-hydroxamate-Based Histone Deacetylase 6 Inhibitor SS-208 with Antitumor Activity in Syngeneic Melanoma Mouse Models</span></div><div class="casAuthors">Shen, Sida; Hadley, Melissa; Ustinova, Kseniya; Pavlicek, Jiri; Knox, Tessa; Noonepalle, Satish; Tavares, Mauricio T.; Zimprich, Chad A.; Zhang, Guiping; Robers, Matthew B.; Barinka, Cyril; Kozikowski, Alan P.; Villagra, Alejandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8557-8577</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isoxazole is a five-membered heterocycle that is widely used in drug discovery endeavors.  Here, the authors report the design, synthesis, and structural and biol. characterization of SS-208, a novel HDAC6-selective inhibitor contg. the isoxazole-3-hydroxamate moiety as a zinc-binding group as well as a hydrophobic linker.  A crystal structure of the Danio rerio HDAC6/SS-208 complex reveals a bidentate coordination of the active-site zinc ion that differs from the preferred monodentate coordination obsd. for HDAC6 complexes with phenylhydroxamate-based inhibitors.  While SS-208 has minimal effects on the viability of murine SM1 melanoma cells in vitro, it significantly reduced in vivo tumor growth in a murine SM1 syngeneic melanoma mouse model.  These findings suggest that the antitumor activity of SS-208 is mainly mediated by immune-related antitumor activity as evidenced by the increased infiltration of CD8+ and NK+ T cells and the enhanced ratio of M1 and M2 macrophages in the tumor microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUagVGHIiRHbVg90H21EOLACvtfcHk0lhQ_40X_wtO4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFymtbzI&md5=53b4c45ce582d8752869430407d44649</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00946%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DHadley%26aufirst%3DM.%26aulast%3DUstinova%26aufirst%3DK.%26aulast%3DPavlicek%26aufirst%3DJ.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DNoonepalle%26aufirst%3DS.%26aulast%3DTavares%26aufirst%3DM.%2BT.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520a%2520new%2520isoxazole-3-hydroxamate-based%2520histone%2520deacetylase%25206%2520inhibitor%2520SS-208%2520with%2520antitumor%2520activity%2520in%2520syngeneic%2520melanoma%2520mouse%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8557%26epage%3D8577%26doi%3D10.1021%2Facs.jmedchem.9b00946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vögerl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological investigation of phenothiazine-based benzhydroxamic acids as selective histone deacetylase 6 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1138</span>– <span class="NLM_lpage">1166</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01090</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01090" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BB3cjhs1Sltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1138-1166&author=K.+V%C3%B6gerlauthor=N.+Ongauthor=J.+Sengerauthor=D.+Herpauthor=K.+Schmidtkunzauthor=M.+Marekauthor=M.+M%C3%BCllerauthor=K.+Bartelauthor=T.+B.+Shaikauthor=N.+J.+Porterauthor=D.+Robaaauthor=D.+W.+Christiansonauthor=C.+Romierauthor=W.+Sipplauthor=M.+Jungauthor=F.+Bracher&title=Synthesis+and+biological+investigation+of+phenothiazine-based+benzhydroxamic+acids+as+selective+histone+deacetylase+6+inhibitors&doi=10.1021%2Facs.jmedchem.8b01090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Investigation of Phenothiazine-Based Benzhydroxamic Acids as Selective Histone Deacetylase 6 Inhibitors</span></div><div class="casAuthors">Vogerl Katharina; Ong Nghia; Muller Martin; Bartel Karin; Bracher Franz; Senger Johanna; Herp Daniel; Schmidtkunz Karin; Jung Manfred; Marek Martin; Shaik Tajith B; Romier Christophe; Porter Nicholas J; Christianson David W; Robaa Dina; Sippl Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1138-1166</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The phenothiazine system was identified as a favorable cap group for potent and selective histone deacetylase 6 (HDAC6) inhibitors.  Here, we report the preparation and systematic variation of phenothiazines and their analogues containing a benzhydroxamic acid moiety as the zinc-binding group.  We evaluated their ability to selectively inhibit HDAC6 by a recombinant HDAC enzyme assay, by determining the protein acetylation levels in cells by western blotting (tubulin vs histone acetylation), and by assessing their effects on various cancer cell lines.  Structure-activity relationship studies revealed that incorporation of a nitrogen atom into the phenothiazine framework results in increased potency and selectivity for HDAC6 (more than 500-fold selectivity relative to the inhibition of HDAC1, HDAC4, and HDAC8), as rationalized by molecular modeling and docking studies.  The binding mode was confirmed by co-crystallization of the potent azaphenothiazine inhibitor with catalytic domain 2 from Danio rerio HDAC6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkSJzGKNxKWK69lAbTj9rGfW6udTcc2eaukJK2sylPJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjhs1Sltw%253D%253D&md5=9d72da7b8f4bb63cd05ccc5f907d2dca</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01090%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25B6gerl%26aufirst%3DK.%26aulast%3DOng%26aufirst%3DN.%26aulast%3DSenger%26aufirst%3DJ.%26aulast%3DHerp%26aufirst%3DD.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%26aulast%3DBartel%26aufirst%3DK.%26aulast%3DShaik%26aufirst%3DT.%2BB.%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBracher%26aufirst%3DF.%26atitle%3DSynthesis%2520and%2520biological%2520investigation%2520of%2520phenothiazine-based%2520benzhydroxamic%2520acids%2520as%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1138%26epage%3D1166%26doi%3D10.1021%2Facs.jmedchem.8b01090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svoboda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavasin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motlova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinsey, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eubanks, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Structural and in vivo characterization of Tubastatin A, a widely used histone deacetylase 6 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00560</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00560" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFarsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=706-712&author=S.+Shenauthor=M.+Svobodaauthor=G.+Zhangauthor=M.+A.+Cavasinauthor=L.+Motlovaauthor=T.+A.+McKinseyauthor=J.+H.+Eubanksauthor=C.+Ba%C5%99inkaauthor=A.+P.+Kozikowski&title=Structural+and+in+vivo+characterization+of+Tubastatin+A%2C+a+widely+used+histone+deacetylase+6+inhibitor&doi=10.1021%2Facsmedchemlett.9b00560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and in Vivo Characterization of Tubastatin A, a Widely Used Histone Deacetylase 6 Inhibitor</span></div><div class="casAuthors">Shen, Sida; Svoboda, Michal; Zhang, Guangming; Cavasin, Maria A.; Motlova, Lucia; McKinsey, Timothy A.; Eubanks, James H.; Barinka, Cyril; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">706-712</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tubastatin A, a tetrahydro-γ-carboline-capped selective HDAC6 inhibitor (HDAC6i), was rationally designed 10 years ago, and has become the best investigated HDAC6i to date.  It shows efficacy in various neurol. disease animal models, as HDAC6 plays a crucial regulatory role in axonal transport deficits, protein aggregation, as well as oxidative stress.  In this work, we provide new insights into this HDAC6i by investigating the mol. basis of its interactions with HDAC6 through X-ray crystallog., detg. its functional capability to elevate the levels of acetylated α-tubulin in vitro and in vivo, correlating PK/PD profiles to det. EDs in plasma and brain, and finally assessing its therapeutic potential toward psychiatric diseases through use of the SmartCube screening platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi67K0VhCsP7Vg90H21EOLACvtfcHk0lhQ_40X_wtO4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFarsr8%253D&md5=431ff199537a679fc09f19f95e528219</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00560%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DS.%26aulast%3DSvoboda%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DMotlova%26aufirst%3DL.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26aulast%3DEubanks%26aufirst%3DJ.%2BH.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DStructural%2520and%2520in%2520vivo%2520characterization%2520of%2520Tubastatin%2520A%252C%2520a%2520widely%2520used%2520histone%2520deacetylase%25206%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D706%26epage%3D712%26doi%3D10.1021%2Facsmedchemlett.9b00560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayana
Reddy, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooker, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Exploring structural determinants of inhibitor affinity and selectivity in complexes with histone deacetylase 6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01540</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit12jsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=295-308&author=J.+D.+Oskoauthor=N.+J.+Porterauthor=P.+A.+Narayana%0AReddyauthor=Y.+C.+Xiaoauthor=J.+Rokkaauthor=M.+Jungauthor=J.+M.+Hookerauthor=J.+M.+Salvinoauthor=D.+W.+Christianson&title=Exploring+structural+determinants+of+inhibitor+affinity+and+selectivity+in+complexes+with+histone+deacetylase+6&doi=10.1021%2Facs.jmedchem.9b01540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Structural Determinants of Inhibitor Affinity and Selectivity in Complexes with Histone Deacetylase 6</span></div><div class="casAuthors">Osko, Jeremy D.; Porter, Nicholas J.; Narayana Reddy, Poli Adi; Xiao, You-Cai; Rokka, Johanna; Jung, Manfred; Hooker, Jacob M.; Salvino, Joseph M.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">295-308</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of histone deacetylase 6 (HDAC6) has emerged as a promising therapeutic strategy for the treatment of cancer, chemotherapy-induced peripheral neuropathy, and neurodegenerative disease.  The recent X-ray crystal structure detn. of HDAC6 enables an understanding of structural features directing affinity and selectivity in the active site.  Here, we present the X-ray crystal structures of five HDAC6-inhibitor complexes that illuminate key mol. features of the inhibitor linker and capping groups that facilitate and differentiate binding to HDAC6.  In particular, arom. and heteroarom. linkers nestle within an arom. cleft defined by F583 and F643, and different arom. linkers direct the capping group toward shallow pockets defined by the L1 loop, the L2 loop, or somewhere in between these pockets.  These results expand our understanding of factors contributing to the selective inhibition of HDAC6, particularly regarding interactions that can be targeted in the region of the L2 pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBosfiCC-DubVg90H21EOLACvtfcHk0lhj4Z0KpNZSjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit12jsr3N&md5=0ebd3729b8a0ed03629c4ecca475b02e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01540%26sid%3Dliteratum%253Aachs%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DNarayana%2BReddy%26aufirst%3DP.%2BA.%26aulast%3DXiao%26aufirst%3DY.%2BC.%26aulast%3DRokka%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26aulast%3DSalvino%26aufirst%3DJ.%2BM.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DExploring%2520structural%2520determinants%2520of%2520inhibitor%2520affinity%2520and%2520selectivity%2520in%2520complexes%2520with%2520histone%2520deacetylase%25206%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D295%26epage%3D308%26doi%3D10.1021%2Facs.jmedchem.9b01540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krieger, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ressing, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkhardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of the first-in-class dual histone deacetylase-proteasome inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10299</span>– <span class="NLM_lpage">10309</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01487</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01487" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10299-10309&author=S.+Bhatiaauthor=V.+Kriegerauthor=M.+Grollauthor=J.+D.+Oskoauthor=N.+Ressingauthor=H.+Ahlertauthor=A.+Borkhardtauthor=T.+Kurzauthor=D.+W.+Christiansonauthor=J.+Hauerauthor=F.+K.+Hansen&title=Discovery+of+the+first-in-class+dual+histone+deacetylase-proteasome+inhibitor&doi=10.1021%2Facs.jmedchem.8b01487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the First-in-Class Dual Histone Deacetylase-Proteasome Inhibitor</span></div><div class="casAuthors">Bhatia, Sanil; Krieger, Viktoria; Groll, Michael; Osko, Jeremy D.; Ressing, Nina; Ahlert, Heinz; Borkhardt, Arndt; Kurz, Thomas; Christianson, David W.; Hauer, Julia; Hansen, Finn K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10299-10309</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual- or multitarget drugs have emerged as a promising alternative to combination therapies.  Proteasome inhibitors (PIs) possess synergistic activity with histone deacetylase (HDAC) inhibitors due to the simultaneous blockage of the ubiquitin degrdn. and aggresome pathways.  Here, the authors present the design, synthesis, binding modes, and anticancer properties of RTS-V5 as the first-in-class dual HDAC-proteasome ligand.  The inhibition of both targets was confirmed by biochem. and cellular assays as well as x-ray crystal structures of the 20S proteasome and HDAC6 complexed with RTS-V5.  Cytotoxicity assays with leukemia and multiple myeloma cell lines as well as therapy refractory primary patient-derived leukemia cells demonstrated that RTS-V5 possesses potent and selective anticancer activity.  The results will thus guide the structure-based optimization of dual HDAC-proteasome inhibitors for the treatment of hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwJMqcDlzXA7Vg90H21EOLACvtfcHk0lhj4Z0KpNZSjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K&md5=7d47e664bd77ac1bc5ebd39ea6eed02a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01487%26sid%3Dliteratum%253Aachs%26aulast%3DBhatia%26aufirst%3DS.%26aulast%3DKrieger%26aufirst%3DV.%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DRessing%26aufirst%3DN.%26aulast%3DAhlert%26aufirst%3DH.%26aulast%3DBorkhardt%26aufirst%3DA.%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DHauer%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DF.%2BK.%26atitle%3DDiscovery%2520of%2520the%2520first-in-class%2520dual%2520histone%2520deacetylase-proteasome%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10299%26epage%3D10309%26doi%3D10.1021%2Facs.jmedchem.8b01487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Structural determinants of affinity and selectivity in the binding of inhibitors to histone deacetylase 6</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127023</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.bmcl.2020.127023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=32067866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFGqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127023&author=J.+D.+Oskoauthor=D.+W.+Christianson&title=Structural+determinants+of+affinity+and+selectivity+in+the+binding+of+inhibitors+to+histone+deacetylase+6&doi=10.1016%2Fj.bmcl.2020.127023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of affinity and selectivity in the binding of inhibitors to histone deacetylase 6</span></div><div class="casAuthors">Osko, Jeremy D.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">127023</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review. Digest.  Histone deacetylase 6 (HDAC6) is assocd. with multiple neurol. disorders as well as aggressive cancers, making its selective inhibition highly desirable for therapeutic purposes.  The basic mol. design of an effective HDAC6 inhibitor consists of a zinc-binding group, a linker, and a capping group capable of making interactions at the mouth of the active site.  To date, more than 50 high-resoln. X-ray crystal structures of HDAC6-inhibitor complexes have been reported, many of which reveal intermol. interactions that contribute to isoenzyme affinity and selectivity.  Here, we review the key features of HDAC6 inhibitor design illuminated by these structural studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEHMdvmF9TPLVg90H21EOLACvtfcHk0lhj4Z0KpNZSjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFGqs70%253D&md5=df46fa5e4720d8340d728e6269dadb90</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127023%26sid%3Dliteratum%253Aachs%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructural%2520determinants%2520of%2520affinity%2520and%2520selectivity%2520in%2520the%2520binding%2520of%2520inhibitors%2520to%2520histone%2520deacetylase%25206%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127023%26doi%3D10.1016%2Fj.bmcl.2020.127023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vergani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandrone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ripamonti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cellupica, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbiati, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caprini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocchio, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lattanzio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pezzuto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skorupska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomarico, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modena, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perego, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkühler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenazzi, A.</span></span> <span> </span><span class="NLM_article-title">Novel benzohydroxamate-based potent and selective histone deacetylase 6 (HDAC6) inhibitors bearing a pentaheterocyclic scaffold: Design, synthesis, and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10711</span>– <span class="NLM_lpage">10739</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01194</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01194" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFeqs77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10711-10739&author=B.+Verganiauthor=G.+Sandroneauthor=M.+Marchiniauthor=C.+Ripamontiauthor=E.+Cellupicaauthor=E.+Galbiatiauthor=G.+Capriniauthor=G.+Pavichauthor=G.+Porroauthor=I.+Rocchioauthor=M.+Lattanzioauthor=M.+Pezzutoauthor=M.+Skorupskaauthor=P.+Cordellaauthor=P.+Paganiauthor=P.+Pozziauthor=R.+Pomaricoauthor=D.+Modenaauthor=F.+Leoniauthor=R.+Peregoauthor=G.+Fossatiauthor=C.+Steink%C3%BChlerauthor=A.+Stevenazzi&title=Novel+benzohydroxamate-based+potent+and+selective+histone+deacetylase+6+%28HDAC6%29+inhibitors+bearing+a+pentaheterocyclic+scaffold%3A+Design%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Facs.jmedchem.9b01194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Benzohydroxamate-Based Potent and Selective Histone Deacetylase 6 (HDAC6) Inhibitors Bearing a Pentaheterocyclic Scaffold: Design, Synthesis, and Biological Evaluation</span></div><div class="casAuthors">Vergani, Barbara; Sandrone, Giovanni; Marchini, Mattia; Ripamonti, Chiara; Cellupica, Edoardo; Galbiati, Elisabetta; Caprini, Gianluca; Pavich, Gianfranco; Porro, Giulia; Rocchio, Ilaria; Lattanzio, Maria; Pezzuto, Marcello; Skorupska, Malgorzata; Cordella, Paola; Pagani, Paolo; Pozzi, Pietro; Pomarico, Roberta; Modena, Daniela; Leoni, Flavio; Perego, Raffaella; Fossati, Gianluca; Steinkuhler, Christian; Stevenazzi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10711-10739</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a peculiar HDAC isoform whose expression and functional alterations were correlated with a variety of pathologies such as autoimmune disorders, neurodegenerative diseases, and cancer.  It is primarily a cytoplasmic protein, and its deacetylase activity is focused mainly on non-histone substrates such as tubulin, heat shock protein (HSP)90, Foxp3 and cortactin, to name a few.  Selective inhibition of HDAC6 does not show cytotoxic effects in healthy cells, normally assocd. with the inhibition of Class I HDAC isoforms.  Here the authors describe the design and synthesis of a new class of potent and selective HDAC6 inhibitors that bear a pentaheterocyclic central core.  These compds. show a remarkably low toxicity both in vitro and in vivo and are able to increase the function of regulatory T cells (Tregs) at well tolerated concns., suggesting a potential clin. use for the treatment of degenerative, auto-immune diseases and organ transplantation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV3O384uJCkLVg90H21EOLACvtfcHk0lhYn1ByspHP9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFeqs77I&md5=efc978e814193713983d2239a821cfd7</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01194%26sid%3Dliteratum%253Aachs%26aulast%3DVergani%26aufirst%3DB.%26aulast%3DSandrone%26aufirst%3DG.%26aulast%3DMarchini%26aufirst%3DM.%26aulast%3DRipamonti%26aufirst%3DC.%26aulast%3DCellupica%26aufirst%3DE.%26aulast%3DGalbiati%26aufirst%3DE.%26aulast%3DCaprini%26aufirst%3DG.%26aulast%3DPavich%26aufirst%3DG.%26aulast%3DPorro%26aufirst%3DG.%26aulast%3DRocchio%26aufirst%3DI.%26aulast%3DLattanzio%26aufirst%3DM.%26aulast%3DPezzuto%26aufirst%3DM.%26aulast%3DSkorupska%26aufirst%3DM.%26aulast%3DCordella%26aufirst%3DP.%26aulast%3DPagani%26aufirst%3DP.%26aulast%3DPozzi%26aufirst%3DP.%26aulast%3DPomarico%26aufirst%3DR.%26aulast%3DModena%26aufirst%3DD.%26aulast%3DLeoni%26aufirst%3DF.%26aulast%3DPerego%26aufirst%3DR.%26aulast%3DFossati%26aufirst%3DG.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26aulast%3DStevenazzi%26aufirst%3DA.%26atitle%3DNovel%2520benzohydroxamate-based%2520potent%2520and%2520selective%2520histone%2520deacetylase%25206%2520%2528HDAC6%2529%2520inhibitors%2520bearing%2520a%2520pentaheterocyclic%2520scaffold%253A%2520Design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10711%26epage%3D10739%26doi%3D10.1021%2Facs.jmedchem.9b01194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Selective histone deacetylase 6 inhibitors bearing substituted urea linkers inhibit melanoma cell growth</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9891</span>– <span class="NLM_lpage">9899</span>, <span class="refDoi"> DOI: 10.1021/jm301098e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301098e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9891-9899&author=J.+A.+Bergmanauthor=K.+Woanauthor=P.+Perez-Villarroelauthor=A.+Villagraauthor=E.+M.+Sotomayorauthor=A.+P.+Kozikowski&title=Selective+histone+deacetylase+6+inhibitors+bearing+substituted+urea+linkers+inhibit+melanoma+cell+growth&doi=10.1021%2Fjm301098e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Histone Deacetylase 6 Inhibitors Bearing Substituted Urea Linkers Inhibit Melanoma Cell Growth</span></div><div class="casAuthors">Bergman, Joel A.; Woan, Karrune; Perez-Villarroel, Patricio; Villagra, Alejandro; Sotomayor, Eduardo M.; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9891-9899</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The incidence of malignant melanoma has dramatically increased in recent years thus requiring the need for improved therapeutic strategies.  In our efforts to design selective histone deactylase inhibitors (HDACI), we discovered that the aryl urea 1 is a modestly potent yet nonselective inhibitor.  Structure-activity relationship studies revealed that adding substituents to the nitrogen atom of the urea so as to generate compds. bearing a branched linker group results in increased potency and selectivity for HDAC6.  Compd. 5g shows low nanomolar inhibitory potency against HDAC6 and a selectivity of ∼600-fold relative to the inhibition of HDAC1.  These HDACIs were evaluated for their ability to inhibit the growth of B16 melanoma cells with the most potent and selective HDAC6I being found to decrease tumor cell growth.  To the best of our knowledge, this work constitutes the first report of HDAC6-selective inhibitors that possess antiproliferative effects against melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTtPLqvwsrVg90H21EOLACvtfcHk0lhYn1ByspHP9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ&md5=d6be4c0efb3922fc35d814ba00ae9ed1</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm301098e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301098e%26sid%3Dliteratum%253Aachs%26aulast%3DBergman%26aufirst%3DJ.%2BA.%26aulast%3DWoan%26aufirst%3DK.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DSelective%2520histone%2520deacetylase%25206%2520inhibitors%2520bearing%2520substituted%2520urea%2520linkers%2520inhibit%2520melanoma%2520cell%2520growth%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9891%26epage%3D9899%26doi%3D10.1021%2Fjm301098e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villaroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canales, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marante, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Celis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Targeting histone deacetylase 6 mediates a dual anti-melanoma effect: Enhanced antitumor immunity and impaired cell proliferation</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1447</span>– <span class="NLM_lpage">1457</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2015.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.molonc.2015.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=25957812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=1447-1457&author=K.+V.+Woanauthor=M.+Lienlafauthor=P.+Perez-Villaroelauthor=C.+Leeauthor=F.+Chengauthor=T.+Knoxauthor=D.+M.+Woodsauthor=K.+Barriosauthor=J.+Powersauthor=E.+Sahakianauthor=H.+W.+Wangauthor=J.+Canalesauthor=D.+Maranteauthor=K.+S.+M.+Smalleyauthor=J.+Bergmanauthor=E.+Setoauthor=A.+Kozikowskiauthor=J.+Pinilla-Ibarzauthor=A.+Sarnaikauthor=E.+Celisauthor=J.+Weberauthor=E.+M.+Sotomayorauthor=A.+Villagra&title=Targeting+histone+deacetylase+6+mediates+a+dual+anti-melanoma+effect%3A+Enhanced+antitumor+immunity+and+impaired+cell+proliferation&doi=10.1016%2Fj.molonc.2015.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone deacetylase 6 mediates a dual anti-melanoma effect: Enhanced antitumor immunity and impaired cell proliferation</span></div><div class="casAuthors">Woan, K. V.; Lienlaf, M.; Perez-Villaroel, P.; Lee, C.; Cheng, F.; Knox, T.; Woods, D. M.; Barrios, K.; Powers, J.; Sahakian, E.; Wang, H. W.; Canales, J.; Marante, D.; Smalley, K. S. M.; Bergman, J.; Seto, E.; Kozikowski, A.; Pinilla-Ibarz, J.; Sarnaik, A.; Celis, E.; Weber, J.; Sotomayor, E. M.; Villagra, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1447-1457</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The median survival for metastatic melanoma is in the realm of 8-16 mo and there are few therapies that offer significant improvement in overall survival.  One of the recent advances in cancer treatment focuses on epigenetic modifiers to alter the survivability and immunogenicity of cancer cells.  Our group and others have previously demonstrated that pan-HDAC inhibitors induce apoptosis, cell cycle arrest and changes in the immunogenicity of melanoma cells.  Here we interrogated specific HDACs which may be responsible for this effect.  We found that both genetic abrogation and pharmacol. inhibition of HDAC6 decreases in vitro proliferation and induces G1 arrest of melanoma cell lines without inducing apoptosis.  Moreover, targeting this mol. led to an important upregulation in the expression of tumor assocd. antigens and MHC class I, suggesting a potential improvement in the immunogenicity of these cells.  Of note, this anti-melanoma activity was operative regardless of mutational status of the cells.  These effects translated into a pronounced delay of in vivo melanoma tumor growth which was, at least in part, dependent on intact immunity as evidenced by the restoration of tumor growth after CD4+ and CD8+ depletion.  Given our findings, we provide the initial rationale for the further development of selective HDAC6 inhibitors as potential therapeutic anti-melanoma agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDIQ5LKbZF8bVg90H21EOLACvtfcHk0lif5vI5yah3UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D&md5=7534df9db01f52a71f97f0e112db37cc</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2015.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2015.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DWoan%26aufirst%3DK.%2BV.%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DPerez-Villaroel%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DWoods%26aufirst%3DD.%2BM.%26aulast%3DBarrios%26aufirst%3DK.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DH.%2BW.%26aulast%3DCanales%26aufirst%3DJ.%26aulast%3DMarante%26aufirst%3DD.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DBergman%26aufirst%3DJ.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DSarnaik%26aufirst%3DA.%26aulast%3DCelis%26aufirst%3DE.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DTargeting%2520histone%2520deacetylase%25206%2520mediates%2520a%2520dual%2520anti-melanoma%2520effect%253A%2520Enhanced%2520antitumor%2520immunity%2520and%2520impaired%2520cell%2520proliferation%26jtitle%3DMol.%2520Oncol.%26date%3D2015%26volume%3D9%26spage%3D1447%26epage%3D1457%26doi%3D10.1016%2Fj.molonc.2015.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sixto-López, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correa-Basurto, J.</span></span> <span> </span><span class="NLM_article-title">Structural and energetic basis for the inhibitory selectivity of both catalytic domains of dimeric HDAC6</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">4701</span>– <span class="NLM_lpage">4720</span>, <span class="refDoi"> DOI: 10.1080/07391102.2018.1557560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1080%2F07391102.2018.1557560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=30558483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVaisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=4701-4720&author=Y.+Sixto-L%C3%B3pezauthor=M.+Belloauthor=J.+Correa-Basurto&title=Structural+and+energetic+basis+for+the+inhibitory+selectivity+of+both+catalytic+domains+of+dimeric+HDAC6&doi=10.1080%2F07391102.2018.1557560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and energetic basis for the inhibitory selectivity of both catalytic domains of dimeric HDAC6</span></div><div class="casAuthors">Sixto-Lopez, Yudibeth; Bello, Martiniano; Correa-Basurto, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure and Dynamics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4701-4720</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">HDAC6 is a protein involved in cancer, neurodegenerative disease and inflammatory disorders.  To date, the full three-dimensional (3D) structure of human HDAC6 has not been elucidated; however, there are some exptl. 3D structural homologs to HDAC6 that can be used as templates.  In this work, we utilized mol. modeling procedures to model both of the catalytic domains of HDAC6 connected by the linker region where DMB region is placed.  Once the 3D structure of human HDAC6 was obtained, it was structurally evaluated and submitted to docking and mol. dynamic (MD) simulations along with Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) method to explore the stability and the binding free energy properties of the HDAC6-ligand complexes.  In addn., its structural and energetic behavior was explored with each one of the catalytic domains in the mol. recognition of six selective HDAC6 inhibitors, HPOB, CAY10603, Nexturastat, Rocilinostat, Tubacin and Tubastatin A for DD2, and with the so-called 9-peptide which is DD1-HDAC6 selective substrate.  The use of the whole system (DD1-DMB-DD2) showed a tendency toward the ligand affinity of DD2, CAY10603> Tubacin > Rocilinostat > Nexturastat > HPOB > Tubastatin > 9-peptide, which is in line with exptl. reports.  However, 9-peptide showed a higher affinity for DD1, which agrees with exptl. reports elsewhere.  Principal component anal. provided important information about the structural changes linked to the mol. recognition process, whereas per-residue decompn. anal. revealed the energetic contribution of the key residues in the mol. binding and structural characteristics that could assist in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaDDSCE5cAdLVg90H21EOLACvtfcHk0lif5vI5yah3UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVaisLg%253D&md5=7029672cda1c077774128f90422209d3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F07391102.2018.1557560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2018.1557560%26sid%3Dliteratum%253Aachs%26aulast%3DSixto-L%25C3%25B3pez%26aufirst%3DY.%26aulast%3DBello%26aufirst%3DM.%26aulast%3DCorrea-Basurto%26aufirst%3DJ.%26atitle%3DStructural%2520and%2520energetic%2520basis%2520for%2520the%2520inhibitory%2520selectivity%2520of%2520both%2520catalytic%2520domains%2520of%2520dimeric%2520HDAC6%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2019%26volume%3D37%26spage%3D4701%26epage%3D4720%26doi%3D10.1080%2F07391102.2018.1557560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Synthesis and pharmacological evaluation of selective histone deacetylase 6 inhibitors in melanoma models</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1036</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00223</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00223" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1031-1036&author=M.+T.+Tavaresauthor=S.+Shenauthor=T.+Knoxauthor=M.+Hadleyauthor=Z.+Kutilauthor=C.+Ba%C5%99inkaauthor=A.+Villagraauthor=A.+P.+Kozikowski&title=Synthesis+and+pharmacological+evaluation+of+selective+histone+deacetylase+6+inhibitors+in+melanoma+models&doi=10.1021%2Facsmedchemlett.7b00223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Pharmacological Evaluation of Selective Histone Deacetylase 6 Inhibitors in Melanoma Models</span></div><div class="casAuthors">Tavares, Mauricio T.; Shen, Sida; Knox, Tessa; Hadley, Melissa; Kutil, Zsofia; Barinka, Cyril; Villagra, Alejandro; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1031-1036</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Only a handful of therapies offer significant improvement in the overall survival in cases of melanoma, a cancer whose incidence has continued to rise in the past 30 years.  In our effort to identify potent and isoform-selective histone deacetylase (HDAC) inhibitors as a therapeutic approach to melanoma, a series of new HDAC6 inhibitors based on the nexturastat A scaffold were prepd.  The new analogs bearing added hydrophilic substituents, so as to establish addnl. hydrogen bonding on the rim of the HDAC6 catalytic pocket, exhibit improved potency against HDAC6 and retain selectivity over HDAC1.  Compd. I exhibits antiproliferative effects on several types of melanoma and lymphoma cells.  Further studies indicates that I selectively increases acetylated tubulin levels in vitro and elicits an immune response through down-regulating cytokine IL-10.  A preliminary in vivo efficacy study indicates that I possesses improved capability to inhibit melanoma tumor growth and that this effect is based on the regulation of inflammatory and immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6cF7frE1ffLVg90H21EOLACvtfcHk0lif5vI5yah3UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7zL&md5=305123b6d7ecc3e4273d5fa02565367f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00223%26sid%3Dliteratum%253Aachs%26aulast%3DTavares%26aufirst%3DM.%2BT.%26aulast%3DShen%26aufirst%3DS.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DHadley%26aufirst%3DM.%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DSynthesis%2520and%2520pharmacological%2520evaluation%2520of%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%2520in%2520melanoma%2520models%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1031%26epage%3D1036%26doi%3D10.1021%2Facsmedchemlett.7b00223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Francesco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz, O. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinzel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesiti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteagudo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pescatore, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdirame, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkühler, C.</span></span> <span> </span><span class="NLM_article-title">A novel series of potent and selective ketone histone deacetylase inhibitors with antitumor activity in vivo</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2350</span>– <span class="NLM_lpage">2353</span>, <span class="refDoi"> DOI: 10.1021/jm800079s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800079s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvFeisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2350-2353&author=P.+Jonesauthor=S.+Altamuraauthor=R.+De+Francescoauthor=O.+G.+Pazauthor=O.+Kinzelauthor=G.+Mesitiauthor=E.+Monteagudoauthor=G.+Pescatoreauthor=M.+Rowleyauthor=M.+Verdirameauthor=C.+Steink%C3%BChler&title=A+novel+series+of+potent+and+selective+ketone+histone+deacetylase+inhibitors+with+antitumor+activity+in+vivo&doi=10.1021%2Fjm800079s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A novel series of potent and selective ketone histone deacetylase inhibitors with antitumor activity in vivo</span></div><div class="casAuthors">Jones, Philip; Altamura, Sergio; De Francesco, Raffaele; Gonzalez Paz, Odalys; Kinzel, Olaf; Mesiti, Giuseppe; Monteagudo, Edith; Pescatore, Giovanna; Rowley, Michael; Verdirame, Maria; Steinkuhler, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2350-2353</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors offer a promising strategy for cancer therapy, and the first generation HDAC inhibitors are currently in the clinic.  Entirely novel ketone HDAC inhibitors have been developed from the cyclic tetrapeptide apicidin.  These compds. show class I subtype selectivity and levels of cellular activity comparable to clin. candidates.  A representative example has demonstrated tumor growth inhibition in a human colon HCT-116 carcinoma xenograft model comparable to known inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCTv5T7lu1nbVg90H21EOLACvtfcHk0ljlU0Aie4hHNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvFeisr8%253D&md5=4701ba3c4a008d66664009b358c4fc9a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm800079s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800079s%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%26aulast%3DAltamura%26aufirst%3DS.%26aulast%3DDe%2BFrancesco%26aufirst%3DR.%26aulast%3DPaz%26aufirst%3DO.%2BG.%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DMesiti%26aufirst%3DG.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DRowley%26aufirst%3DM.%26aulast%3DVerdirame%26aufirst%3DM.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26atitle%3DA%2520novel%2520series%2520of%2520potent%2520and%2520selective%2520ketone%2520histone%2520deacetylase%2520inhibitors%2520with%2520antitumor%2520activity%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2350%26epage%3D2353%26doi%3D10.1021%2Fjm800079s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinzel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llauger-Bufi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pescatore, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteagudo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonsi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez Paz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiore, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkühler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I selective ketone histone deacetylase inhibitor with antitumor activity in vivo and optimized pharmacokinetic properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3453</span>– <span class="NLM_lpage">3456</span>, <span class="refDoi"> DOI: 10.1021/jm9004303</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9004303" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFWnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=3453-3456&author=O.+Kinzelauthor=L.+Llauger-Bufiauthor=G.+Pescatoreauthor=M.+Rowleyauthor=C.+Schultz-Fademrechtauthor=E.+Monteagudoauthor=M.+Fonsiauthor=O.+Gonzalez+Pazauthor=F.+Fioreauthor=C.+Steink%C3%BChlerauthor=P.+Jones&title=Discovery+of+a+potent+class+I+selective+ketone+histone+deacetylase+inhibitor+with+antitumor+activity+in+vivo+and+optimized+pharmacokinetic+properties&doi=10.1021%2Fjm9004303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Selective Ketone Histone Deacetylase Inhibitor with Antitumor Activity in Vivo and Optimized Pharmacokinetic Properties</span></div><div class="casAuthors">Kinzel, Olaf; Llauger-Bufi, Laura; Pescatore, Giovanna; Rowley, Michael; Schultz-Fademrecht, Carsten; Monteagudo, Edith; Fonsi, Massimiliano; Gonzalez Paz, Odalys; Fiore, Fabrizio; Steinkuhler, Christian; Jones, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3453-3456</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The optimization of a potent, class I selective ketone HDAC inhibitor is shown.  It possesses optimized pharmacokinetic properties in preclin. species, has a clean off-target profile, and is neg. in a microbial mutagenicity (Ames) test.  In a mouse xenograft model it shows efficacy comparable to that of vorinostat at a 10-fold reduced dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHRRFS9TSYV7Vg90H21EOLACvtfcHk0ljlU0Aie4hHNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFWnsb0%253D&md5=8674adc0c3892942a1fc7b154cbc84b4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm9004303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9004303%26sid%3Dliteratum%253Aachs%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DLlauger-Bufi%26aufirst%3DL.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DRowley%26aufirst%3DM.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DFonsi%26aufirst%3DM.%26aulast%3DGonzalez%2BPaz%26aufirst%3DO.%26aulast%3DFiore%26aufirst%3DF.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520selective%2520ketone%2520histone%2520deacetylase%2520inhibitor%2520with%2520antitumor%2520activity%2520in%2520vivo%2520and%2520optimized%2520pharmacokinetic%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D3453%26epage%3D3456%26doi%3D10.1021%2Fjm9004303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bresciani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ontoria, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biancofiore, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cellucci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciammaichella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrigno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malancona, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteagudo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nizi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pace, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossetti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veneziano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span> <span> </span><span class="NLM_article-title">Improved selective Class I HDAC and novel selective HDAC3 inhibitors: beyond hydroxamic acids and benzamides</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00517</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00517" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlals7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=481-486&author=A.+Brescianiauthor=J.+M.+Ontoriaauthor=I.+Biancofioreauthor=A.+Cellucciauthor=A.+Ciammaichellaauthor=A.+Di+Marcoauthor=F.+Ferrignoauthor=A.+Franconeauthor=S.+Malanconaauthor=E.+Monteagudoauthor=E.+Niziauthor=P.+Paceauthor=S.+Ponziauthor=I.+Rossettiauthor=M.+Venezianoauthor=V.+Summaauthor=S.+Harper&title=Improved+selective+Class+I+HDAC+and+novel+selective+HDAC3+inhibitors%3A+beyond+hydroxamic+acids+and+benzamides&doi=10.1021%2Facsmedchemlett.8b00517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Improved selective class I HDAC and novel selective HDAC3 inhibitors: Beyond hydroxamic acids and benzamides</span></div><div class="casAuthors">Bresciani, Alberto; Ontoria, Jesus M.; Biancofiore, Ilaria; Cellucci, Antonella; Ciammaichella, Alina; Di Marco, Annalise; Ferrigno, Federica; Francone, Alessandra; Malancona, Savina; Monteagudo, Edith; Nizi, Emanuela; Pace, Paola; Ponzi, Simona; Rossetti, Ilaria; Veneziano, Maria; Summa, Vincenzo; Harper, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">481-486</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The application of class I HDAC inhibitors as cancer therapies is well established, but more recently their development for nononcol. indications has increased.  We report here on the generation of improved class I selective human HDAC inhibitors based on an ethylketone zinc binding group (ZBG) in place of the hydroxamic acid that features the majority of HDAC inhibitors.  We also describe a novel set of HDAC3 isoform selective inhibitors that show stronger potency and selectivity than the most commonly used HDAC3 selective tool compd. RGFP966.  These compds. are again based on an alternative ZBG with respect to the ortho-anilide that is featured in HDAC3 selective compds. reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIzwbKCL5A6bVg90H21EOLACvtfcHk0ljlU0Aie4hHNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlals7jO&md5=2882346cee38482adab18732afde3470</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00517%26sid%3Dliteratum%253Aachs%26aulast%3DBresciani%26aufirst%3DA.%26aulast%3DOntoria%26aufirst%3DJ.%2BM.%26aulast%3DBiancofiore%26aufirst%3DI.%26aulast%3DCellucci%26aufirst%3DA.%26aulast%3DCiammaichella%26aufirst%3DA.%26aulast%3DDi%2BMarco%26aufirst%3DA.%26aulast%3DFerrigno%26aufirst%3DF.%26aulast%3DFrancone%26aufirst%3DA.%26aulast%3DMalancona%26aufirst%3DS.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DNizi%26aufirst%3DE.%26aulast%3DPace%26aufirst%3DP.%26aulast%3DPonzi%26aufirst%3DS.%26aulast%3DRossetti%26aufirst%3DI.%26aulast%3DVeneziano%26aufirst%3DM.%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DHarper%26aufirst%3DS.%26atitle%3DImproved%2520selective%2520Class%2520I%2520HDAC%2520and%2520novel%2520selective%2520HDAC3%2520inhibitors%253A%2520beyond%2520hydroxamic%2520acids%2520and%2520benzamides%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D481%26epage%3D486%26doi%3D10.1021%2Facsmedchemlett.8b00517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinsky, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 10 structure and molecular function as a polyamine deacetylase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15368</span>, <span class="refDoi"> DOI: 10.1038/ncomms15368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fncomms15368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28516954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotFyisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15368&author=Y.+Haiauthor=S.+A.+Shinskyauthor=N.+J.+Porterauthor=D.+W.+Christianson&title=Histone+deacetylase+10+structure+and+molecular+function+as+a+polyamine+deacetylase&doi=10.1038%2Fncomms15368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 10 structure and molecular function as a polyamine deacetylase</span></div><div class="casAuthors">Hai, Yang; Shinsky, Stephen A.; Porter, Nicholas J.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15368</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cationic polyamines such as spermidine and spermine are crit. in all forms of life, as they regulate the function of biol. macromols.  Intracellular polyamine metab. is regulated by reversible acetylation and dysregulated polyamine metab. is assocd. with neoplastic diseases such as colon cancer, prostate cancer and neuroblastoma.  Here we report that histone deacetylase 10 (HDAC10) is a robust polyamine deacetylase, using recombinant enzymes from Homo sapiens (human) and Danio rerio (zebrafish).  The 2.85 Å-resoln. crystal structure of zebrafish HDAC10 complexed with a transition-state analog inhibitor reveals that a glutamate gatekeeper and a sterically constricted active site confer specificity for N8-acetylspermidine hydrolysis and disfavor acetyllysine hydrolysis.  Both HDAC10 and spermidine are known to promote cellular survival through autophagy.  Accordingly, this work sets a foundation for studying the chem. biol. of autophagy through the structure-based design of inhibitors that may also serve as new leads for cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa9laHMWjizLVg90H21EOLACvtfcHk0ljJtiIsggxHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotFyisrY%253D&md5=e192e4b219dcb0b46b05ece566f8a59d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms15368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15368%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DShinsky%26aufirst%3DS.%2BA.%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520deacetylase%252010%2520structure%2520and%2520molecular%2520function%2520as%2520a%2520polyamine%2520deacetylase%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15368%26doi%3D10.1038%2Fncomms15368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42717</span>, <span class="refDoi"> DOI: 10.1038/srep42717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fsrep42717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28256516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42717&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissADME%3A+a+free+web+tool+to+evaluate+pharmacokinetics%2C+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&doi=10.1038%2Fsrep42717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span></div><div class="casAuthors">Daina Antoine; Michielin Olivier; Zoete Vincent; Michielin Olivier; Michielin Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42717</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur.  Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited.  In that context, computer models constitute valid alternatives to experiments.  Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar.  Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch.  Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRl_Uk5dOMKs6qcneEtCi92fW6udTcc2eaWvpvmYQftKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D&md5=9715b8cb8a34b17c4c73ff69a5a8cc50</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fsrep42717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42717%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissADME%253A%2520a%2520free%2520web%2520tool%2520to%2520evaluate%2520pharmacokinetics%252C%2520drug-likeness%2520and%2520medicinal%2520chemistry%2520friendliness%2520of%2520small%2520molecules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42717%26doi%3D10.1038%2Fsrep42717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic efficiency as an important metric in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+efficiency+as+an+important+metric+in+drug+design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0ljJtiIsggxHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520efficiency%2520as%2520an%2520important%2520metric%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebDkZEm145r4rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacketor, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comin-Anduix, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumeh, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minasyan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graeber, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span> <span> </span><span class="NLM_article-title">BRAF inhibitor vemurafenib improves the antitumor activity of adoptive cell immunotherapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3928</span>– <span class="NLM_lpage">3937</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-2837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1158%2F0008-5472.CAN-11-2837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=22693252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2rtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=3928-3937&author=R.+C.+Koyaauthor=S.+Mokauthor=N.+Otteauthor=K.+J.+Blacketorauthor=B.+Comin-Anduixauthor=P.+C.+Tumehauthor=A.+Minasyanauthor=N.+A.+Grahamauthor=T.+G.+Graeberauthor=T.+Chodonauthor=A.+Ribas&title=BRAF+inhibitor+vemurafenib+improves+the+antitumor+activity+of+adoptive+cell+immunotherapy&doi=10.1158%2F0008-5472.CAN-11-2837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF Inhibitor Vemurafenib Improves the Antitumor Activity of Adoptive Cell Immunotherapy</span></div><div class="casAuthors">Koya, Richard C.; Mok, Stephen; Otte, Nicholas; Blacketor, Kevin J.; Comin-Anduix, Begonya; Tumeh, Paul C.; Minasyan, Aspram; Graham, Nicholas A.; Graeber, Thomas G.; Chodon, Thinle; Ribas, Antoni</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3928-3937</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Targeted therapeutics that produce robust clin. responses are likely benefiting from inherent adjuvant effects that degrade immune escape, a feature that could be leveraged most effectively by combinatorial uses in settings such as that described here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg6bb5bmox1bVg90H21EOLACvtfcHk0lioMoToOzvtuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2rtr3J&md5=d8ec5438042feca327a3227c6d2a2381</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-2837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-2837%26sid%3Dliteratum%253Aachs%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DMok%26aufirst%3DS.%26aulast%3DOtte%26aufirst%3DN.%26aulast%3DBlacketor%26aufirst%3DK.%2BJ.%26aulast%3DComin-Anduix%26aufirst%3DB.%26aulast%3DTumeh%26aufirst%3DP.%2BC.%26aulast%3DMinasyan%26aufirst%3DA.%26aulast%3DGraham%26aufirst%3DN.%2BA.%26aulast%3DGraeber%26aufirst%3DT.%2BG.%26aulast%3DChodon%26aufirst%3DT.%26aulast%3DRibas%26aufirst%3DA.%26atitle%3DBRAF%2520inhibitor%2520vemurafenib%2520improves%2520the%2520antitumor%2520activity%2520of%2520adoptive%2520cell%2520immunotherapy%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D3928%26epage%3D3937%26doi%3D10.1158%2F0008-5472.CAN-11-2837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, M. W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, M. J.</span></span> <span> </span><span class="NLM_article-title">Resistance to PD1/PDL1 checkpoint inhibition</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2016.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1016%2Fj.ctrv.2016.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27951441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKntLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2017&pages=71-81&author=J.+S.+O%E2%80%99Donnellauthor=G.+V.+Longauthor=R.+A.+Scolyerauthor=M.+W.+L.+Tengauthor=M.+J.+Smyth&title=Resistance+to+PD1%2FPDL1+checkpoint+inhibition&doi=10.1016%2Fj.ctrv.2016.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to PD1/PDL1 checkpoint inhibition</span></div><div class="casAuthors">O'Donnell, Jake S.; Long, Georgina V.; Scolyer, Richard A.; Teng, Michele W. L.; Smyth, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-81</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">For the first time in decades, patients with difficult-to-treat cancers such as advanced stage metastatic melanoma are being offered a glimpse of hope in the form of immunotherapies.  By targeting factors that foster the development and maintenance of an immunosuppressive microenvironment within tumors, these therapies release the brakes on the host's own immune system; allowing cure of disease.  Indeed, phase III clin. trials have revealed that therapies such as ipilimumab and pembrolizumab which target the CTLA4 and PD-1 immune checkpoints, resp., have raised the three-year survival of patients with melanoma to ∼70%, and overall survival (>5 years) to ∼30%.  Despite this unprecedented efficacy, many patients fail to respond, and more concerning, some patients who demonstrate encouraging initial responses to immunotherapy, can acquire resistance over time.  There is now an urgent need to identify mechanisms of resistance, to predict outcome and to identify targets for combination therapy.  Here, with the aim of guiding future combination trials that target specific resistance mechanisms to immunotherapies, we have summarised and discussed the current understanding of mechanisms promoting resistance to anti-PD1/PDL1 therapies, and how combination strategies which target these pathways might yield better outcomes for patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWZg7kf5N4R7Vg90H21EOLACvtfcHk0lioMoToOzvtuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKntLzE&md5=61bfcb71c4dcc6a79be5aa6c06b30c97</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2016.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2016.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DTeng%26aufirst%3DM.%2BW.%2BL.%26aulast%3DSmyth%26aufirst%3DM.%2BJ.%26atitle%3DResistance%2520to%2520PD1%252FPDL1%2520checkpoint%2520inhibition%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2017%26volume%3D52%26spage%3D71%26epage%3D81%26doi%3D10.1016%2Fj.ctrv.2016.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Entropy as a driver of selectivity for inhibitor binding to histone deacetylase 6</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3916</span>– <span class="NLM_lpage">3924</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00367</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00367" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps1Onsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=3916-3924&author=N.+J.+Porterauthor=F.+F.+Wagnerauthor=D.+W.+Christianson&title=Entropy+as+a+driver+of+selectivity+for+inhibitor+binding+to+histone+deacetylase+6&doi=10.1021%2Facs.biochem.8b00367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Entropy as a Driver of Selectivity for Inhibitor Binding to Histone Deacetylase 6</span></div><div class="casAuthors">Porter, Nicholas J.; Wagner, Florence F.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3916-3924</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among the metal-dependent histone deacetylases, the class IIb isoenzyme HDAC6 is remarkable due to its role in the regulation of microtubule dynamics in the cytosol.  Selective inhibition of HDAC6 results in microtubule hyperacetylation, leading to cell cycle arrest and apoptosis, which is a validated strategy for cancer chemotherapy and the treatment of other disorders.  HDAC6 inhibitors generally consist of a Zn2+-binding group such as a hydroxamate, a linker, and a capping group; the capping group is a crit. determinant of isoenzyme selectivity.  Surprisingly, however, even "capless" inhibitors exhibit appreciable HDAC6 selectivity.  To probe the chem. basis for this selectivity, we now report high-resoln. crystal structures of HDAC6 complexed with capless cycloalkylhydroxamate inhibitors 1-4.  Each inhibitor hydroxamate group coordinates to the catalytic Zn2+ ion with canonical bidentate geometry.  Addnl., the olefin moieties of compds. 2 and 4 bind in an arom. crevice between the side chains of F583 and F643.  Reasoning that similar binding could be achieved in the representative class I isoenzyme HDAC8, we employed isothermal titrn. calorimetry to study the thermodn. of inhibitor binding.  These measurements indicate that the entropy of inhibitor binding is generally pos. for binding to HDAC6 and neg. for binding to HDAC8, resulting in up to 313-fold selectivity for binding to HDAC6 relative to HDAC8.  Thus, favorable binding entropy contributes to HDAC6 selectivity.  Notably, cyclohexenylhydroxamate 2 represents a promising lead for derivatization with capping groups that may further enhance its impressive 313-fold thermodn. selectivity for HDAC6 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5MmbdoXXUqrVg90H21EOLACvtfcHk0lioMoToOzvtuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps1Onsrg%253D&md5=0da8abf0ee7f012db41c620101551442</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00367%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DN.%2BJ.%26aulast%3DWagner%26aufirst%3DF.%2BF.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DEntropy%2520as%2520a%2520driver%2520of%2520selectivity%2520for%2520inhibitor%2520binding%2520to%2520histone%2520deacetylase%25206%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D3916%26epage%3D3924%26doi%3D10.1021%2Facs.biochem.8b00367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seigneurin-Berny, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemercier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Identification of components of the murine histone deacetylase 6 complex: link between acetylation and ubiquitination signaling pathways</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">8035</span>– <span class="NLM_lpage">8044</span>, <span class="refDoi"> DOI: 10.1128/MCB.21.23.8035-8044.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1128%2FMCB.21.23.8035-8044.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=11689694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD3MXotl2gtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=8035-8044&author=D.+Seigneurin-Bernyauthor=A.+Verdelauthor=S.+Curtetauthor=C.+Lemercierauthor=J.+Garinauthor=S.+Rousseauxauthor=S.+Khochbin&title=Identification+of+components+of+the+murine+histone+deacetylase+6+complex%3A+link+between+acetylation+and+ubiquitination+signaling+pathways&doi=10.1128%2FMCB.21.23.8035-8044.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of components of the murine histone deacetylase 6 complex: link between acetylation and ubiquitination signaling pathways</span></div><div class="casAuthors">Seigneurin-Berny, Daphne; Verdel, Andre; Curtet, Sandrine; Lemercier, Claudie; Garin, Jerome; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8035-8044</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The immunopurifn. of the endogenous cytoplasmic murine histone deacetylase 6 (mHDAC6), a member of the class II HDACs, from mouse testis cytosolic exts. allowed the identification of two assocd. proteins.  Both were mammalian homologues of yeast proteins known to interact with each other and involved in the ubiquitin signaling pathway: p97/VCP/Cdc48p, a homolog of yeast Cdc48p, and phospholipase A2-activating protein, a homolog of yeast UFD3 (ubiquitin fusion degrdn. protein 3).  Moreover, in the C-terminal region of mHDAC6, a conserved zinc finger-contg. domain named ZnF-UBP, also present in several ubiquitin-specific proteases, was discovered and was shown to mediate the specific binding of ubiquitin by mHDAC6.  By using a ubiquitin pull-down approach, nine major ubiquitin-binding proteins were identified in mouse testis cytosolic exts., and mHDAC6 was found to be one of them.  All of these findings strongly suggest that mHDAC6 could be involved in the control of protein ubiquitination.  The investigation of biochem. properties of the mHDAC6 complex in vitro further supported this hypothesis and clearly established a link between protein acetylation and protein ubiquitination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr46PPvjekckbVg90H21EOLACvtfcHk0li5YdkwTcSRvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXotl2gtLs%253D&md5=82eccf722f2702e987e74d2fb5138c2b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2FMCB.21.23.8035-8044.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.21.23.8035-8044.2001%26sid%3Dliteratum%253Aachs%26aulast%3DSeigneurin-Berny%26aufirst%3DD.%26aulast%3DVerdel%26aufirst%3DA.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DLemercier%26aufirst%3DC.%26aulast%3DGarin%26aufirst%3DJ.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520components%2520of%2520the%2520murine%2520histone%2520deacetylase%25206%2520complex%253A%2520link%2520between%2520acetylation%2520and%2520ubiquitination%2520signaling%2520pathways%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2001%26volume%3D21%26spage%3D8035%26epage%3D8044%26doi%3D10.1128%2FMCB.21.23.8035-8044.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadoul, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pabion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">HDAC6, at the crossroads between cytoskeleton and cell signaling by acetylation and ubiquitination</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5468</span>– <span class="NLM_lpage">5476</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210614</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fsj.onc.1210614" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=17694087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFersrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=5468-5476&author=C.+Boyaultauthor=K.+Sadoulauthor=M.+Pabionauthor=S.+Khochbin&title=HDAC6%2C+at+the+crossroads+between+cytoskeleton+and+cell+signaling+by+acetylation+and+ubiquitination&doi=10.1038%2Fsj.onc.1210614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6, at the crossroads between cytoskeleton and cell signaling by acetylation and ubiquitination</span></div><div class="casAuthors">Boyault, C.; Sadoul, K.; Pabion, M.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5468-5476</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase 6 (HDAC6) is a unique enzyme with specific structural and functional features.  It is actively or stably maintained in the cytoplasm and is the only member, within the HDAC family, that harbors a full duplication of its deacetylase homol. region followed by a specific ubiquitin-binding domain at the C-terminus end.  Accordingly, this deacetylase functions at the heart of a cellular regulatory mechanism capable of coordinating various cellular functions largely relying on the microtubule network.  Moreover, HDAC6 action as a regulator of HSP90 chaperone activity adds to the multifunctionality of the protein, and allows the authors to propose a crit. role for HDAC6 in mediating and coordinating various cellular events in response to different stressful stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9IYNSqbx5pbVg90H21EOLACvtfcHk0li5YdkwTcSRvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFersrw%253D&md5=3796fc81a6b403bf2c6d08122d3e94c8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210614%26sid%3Dliteratum%253Aachs%26aulast%3DBoyault%26aufirst%3DC.%26aulast%3DSadoul%26aufirst%3DK.%26aulast%3DPabion%26aufirst%3DM.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DHDAC6%252C%2520at%2520the%2520crossroads%2520between%2520cytoskeleton%2520and%2520cell%2520signaling%2520by%2520acetylation%2520and%2520ubiquitination%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D5468%26epage%3D5476%26doi%3D10.1038%2Fsj.onc.1210614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenhof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van%0ADer+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0li5YdkwTcSRvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guvench, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+Mackerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0ljIEKqrz01bBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vermaas, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajkhorshid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmeyer, A.</span></span> <span> </span><span class="NLM_article-title">TopoGromacs: automated topology conversion from CHARMM to GROMACS within VMD</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00103</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00103" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFKrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1112-1116&author=J.+V.+Vermaasauthor=D.+J.+Hardyauthor=J.+E.+Stoneauthor=E.+Tajkhorshidauthor=A.+Kohlmeyer&title=TopoGromacs%3A+automated+topology+conversion+from+CHARMM+to+GROMACS+within+VMD&doi=10.1021%2Facs.jcim.6b00103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">TopoGromacs: Automated Topology Conversion from CHARMM to GROMACS within VMD</span></div><div class="casAuthors">Vermaas, Josh V.; Hardy, David J.; Stone, John E.; Tajkhorshid, Emad; Kohlmeyer, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1112-1116</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulation engines use a variety of different approaches for modeling mol. systems with force fields that govern their dynamics and describe their topol.  These different approaches introduce incompatibilities between engines, and previously published software bridges the gaps between many popular MD packages, such as between CHARMM and AMBER or GROMACS and LAMMPS.  While there are many structure building tools available that generate topologies and structures in CHARMM format, only recently have mechanisms been developed to convert their results into GROMACS input.  We present an approach to convert CHARMM-formatted topol. and parameters into a format suitable for simulation with GROMACS by expanding the functionality of TopoTools, a plugin integrated within the widely used mol. visualization and anal. software VMD.  The conversion process was diligently tested on a comprehensive set of biol. mols. in vacuo.  The resulting comparison between energy terms shows that the translation performed was lossless as the energies were unchanged for identical starting configurations.  By applying the conversion process to conventional benchmark systems that mimic typical modestly sized MD systems, we explore the effect of the implementation choices made in CHARMM, NAMD, and GROMACS.  The newly available automatic conversion capability breaks down barriers between simulation tools and user communities and allows users to easily compare simulation programs and leverage their unique features without the tedium of constructing a topol. twice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooVXPTAehkKrVg90H21EOLACvtfcHk0ljIEKqrz01bBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFKrsrc%253D&md5=3dfea753660da654f8cdd3b5f2f40029</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00103%26sid%3Dliteratum%253Aachs%26aulast%3DVermaas%26aufirst%3DJ.%2BV.%26aulast%3DHardy%26aufirst%3DD.%2BJ.%26aulast%3DStone%26aufirst%3DJ.%2BE.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DKohlmeyer%26aufirst%3DA.%26atitle%3DTopoGromacs%253A%2520automated%2520topology%2520conversion%2520from%2520CHARMM%2520to%2520GROMACS%2520within%2520VMD%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1112%26epage%3D1116%26doi%3D10.1021%2Facs.jcim.6b00103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bjelkmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuendet, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">Implementation of the CHARMM force field in GROMACS: analysis of protein stability effects from correction maps, virtual interaction sites, and water models</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/ct900549r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900549r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=459-466&author=P.+Bjelkmarauthor=P.+Larssonauthor=M.+A.+Cuendetauthor=B.+Hessauthor=E.+Lindahl&title=Implementation+of+the+CHARMM+force+field+in+GROMACS%3A+analysis+of+protein+stability+effects+from+correction+maps%2C+virtual+interaction+sites%2C+and+water+models&doi=10.1021%2Fct900549r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Implementation of the CHARMM Force Field in GROMACS: Analysis of Protein Stability Effects from Correction Maps, Virtual Interaction Sites, and Water Models</span></div><div class="casAuthors">Bjelkmar, Par; Larsson, Per; Cuendet, Michel A.; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">459-466</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CHARMM27 is a widespread and popular force field for biomol. simulation, and several recent algorithms such as implicit solvent models have been developed specifically for it.  The authors have here implemented the CHARMM force field and all necessary extended functional forms in the GROMACS mol. simulation package, to make CHARMM-specific features available and to test them in combination with techniques for extended time steps, to make all major force fields available for comparison studies in GROMACS, and to test various solvent model optimizations, in particular the effect of Lennard-Jones interactions on hydrogens.  The implementation has full support both for CHARMM-specific features such as multiple potentials over the same dihedral angle and the grid-based energy correction map on the .vphi., ψ protein backbone dihedrals, as well as all GROMACS features such as virtual hydrogen interaction sites that enable 5 fs time steps.  The medium-to-long time effects of both the correction maps and virtual sites have been tested by performing a series of 100 ns simulations using different models for water representation, including comparisons between CHARMM and traditional TIP3P.  Including the correction maps improves sampling of near native-state conformations in the authors' systems, and to some extent it is even able to refine distorted protein conformations.  Finally, the authors show that this accuracy is largely maintained with a new implicit solvent implementation that works with virtual interaction sites, which enables performance in excess of 250 ns/day for a 900-atom protein on a quad-core desktop computer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzWFE8uOBhobVg90H21EOLACvtfcHk0lgFKsW1ITdZUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlKhsw%253D%253D&md5=728be20d72ab18e98268c1a48efcdcb1</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct900549r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900549r%26sid%3Dliteratum%253Aachs%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DCuendet%26aufirst%3DM.%2BA.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DImplementation%2520of%2520the%2520CHARMM%2520force%2520field%2520in%2520GROMACS%253A%2520analysis%2520of%2520protein%2520stability%2520effects%2520from%2520correction%2520maps%252C%2520virtual%2520interaction%2520sites%252C%2520and%2520water%2520models%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D459%26epage%3D466%26doi%3D10.1021%2Fct900549r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh ewald: an N Log(N) method for ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+ewald%3A+an+N+Log%28N%29+method+for+ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lgFKsW1ITdZUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520ewald%253A%2520an%2520N%2520Log%2528N%2529%2520method%2520for%2520ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skultetyova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ustinova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlicek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikesova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havlinova, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubalek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansky, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span> <span> </span><span class="NLM_article-title">Human histone deacetylase 6 shows strong preference for tubulin dimers over assembled microtubules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11547</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-11739-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1038%2Fs41598-017-11739-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=28912522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BC1cboslOjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11547&author=L.+Skultetyovaauthor=K.+Ustinovaauthor=Z.+Kutilauthor=Z.+Novakovaauthor=J.+Pavlicekauthor=J.+Mikesovaauthor=D.+Traplauthor=P.+Baranovaauthor=B.+Havlinovaauthor=M.+Hubalekauthor=Z.+Lanskyauthor=C.+Barinka&title=Human+histone+deacetylase+6+shows+strong+preference+for+tubulin+dimers+over+assembled+microtubules&doi=10.1038%2Fs41598-017-11739-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Human histone deacetylase 6 shows strong preference for tubulin dimers over assembled microtubules</span></div><div class="casAuthors">Skultetyova Lubica; Ustinova Kseniya; Kutil Zsofia; Novakova Zora; Pavlicek Jiri; Mikesova Jana; Trapl Dalibor; Baranova Petra; Havlinova Barbora; Lansky Zdenek; Barinka Cyril; Skultetyova Lubica; Hubalek Martin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11547</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human histone deacetylase 6 (HDAC6) is the major deacetylase responsible for removing the acetyl group from Lys40 of α-tubulin (αK40), which is located lumenally in polymerized microtubules.  Here, we provide a detailed kinetic analysis of tubulin deacetylation and HDAC6/microtubule interactions using individual purified components.  Our data unequivocally show that free tubulin dimers represent the preferred HDAC6 substrate, with a K M value of 0.23 μM and a deacetylation rate over 1,500-fold higher than that of assembled microtubules.  We attribute the lower deacetylation rate of microtubules to both longitudinal and lateral lattice interactions within tubulin polymers.  Using TIRF microscopy, we directly visualized stochastic binding of HDAC6 to assembled microtubules without any detectable preferential binding to microtubule tips.  Likewise, indirect immunofluorescence microscopy revealed that microtubule deacetylation by HDAC6 is carried out stochastically along the whole microtubule length, rather than from the open extremities.  Our data thus complement prior studies on tubulin acetylation and further strengthen the rationale for the correlation between tubulin acetylation and microtubule age.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2XuDrQ2dPtsKL0FlyGHDofW6udTcc2eaHrMZ7AIhmc7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cboslOjsw%253D%253D&md5=c1b7f71ae3bd1008ad4b8c925ea30599</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-11739-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-11739-3%26sid%3Dliteratum%253Aachs%26aulast%3DSkultetyova%26aufirst%3DL.%26aulast%3DUstinova%26aufirst%3DK.%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DPavlicek%26aufirst%3DJ.%26aulast%3DMikesova%26aufirst%3DJ.%26aulast%3DTrapl%26aufirst%3DD.%26aulast%3DBaranova%26aufirst%3DP.%26aulast%3DHavlinova%26aufirst%3DB.%26aulast%3DHubalek%26aufirst%3DM.%26aulast%3DLansky%26aufirst%3DZ.%26aulast%3DBarinka%26aufirst%3DC.%26atitle%3DHuman%2520histone%2520deacetylase%25206%2520shows%2520strong%2520preference%2520for%2520tubulin%2520dimers%2520over%2520assembled%2520microtubules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11547%26doi%3D10.1038%2Fs41598-017-11739-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutil, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meleshin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikesova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylase 11 is a fatty-acid deacylase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">685</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00942</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00942" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXoslCjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=685-693&author=Z.+Kutilauthor=Z.+Novakovaauthor=M.+Meleshinauthor=J.+Mikesovaauthor=M.+Schutkowskiauthor=C.+Barinka&title=Histone+deacetylase+11+is+a+fatty-acid+deacylase&doi=10.1021%2Facschembio.7b00942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase 11 Is a Fatty-Acid Deacylase</span></div><div class="casAuthors">Kutil, Zsofia; Novakova, Zora; Meleshin, Marat; Mikesova, Jana; Schutkowski, Mike; Barinka, Cyril</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">685-693</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 11 (HDAC11) is a sole member of the class IV HDAC subfamily with negligible intrinsic deacetylation activity.  Here, we report in vitro profiling of HDAC11 deacylase activities, and our data unequivocally show that the enzyme efficiently removes acyl moieties spanning 8-18 carbons from the side chain nitrogen of the lysine residue of a peptidic substrate.  Addnl., N-linked lipoic acid and biotin are removed by the enzyme, although with lower efficacy.  Catalytic efficiencies toward dodecanoylated and myristoylated peptides were 77 700 and 149 000 M-1 s-1, resp., making HDAC11 the most proficient fatty-acid deacylase of the HDAC family.  Interestingly, HDAC11 is strongly inhibited by free myristic, palmitic, and stearic acids with inhibition consts. of 6.5, 0.9, and 1.6 μM, resp.  At the same time, its deacylase activity is stimulated more than 2.5-fold by both palmitoyl-CoA and myristoyl-CoA, pointing toward metabolic control of the enzymic activity by fatty-acid metabolites.  Our data reveal novel enzymic activity of HDAC11 that can, in turn, facilitate the uncovering of addnl. biol. functions of the enzyme as well as the design of isoform-specific HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozNsptw7rk7bVg90H21EOLACvtfcHk0lgNGnkti9fW1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXoslCjug%253D%253D&md5=07233a3bfed4f158c6d5752fc18a8c1f</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00942%26sid%3Dliteratum%253Aachs%26aulast%3DKutil%26aufirst%3DZ.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DMeleshin%26aufirst%3DM.%26aulast%3DMikesova%26aufirst%3DJ.%26aulast%3DSchutkowski%26aufirst%3DM.%26aulast%3DBarinka%26aufirst%3DC.%26atitle%3DHistone%2520deacetylase%252011%2520is%2520a%2520fatty-acid%2520deacylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D685%26epage%3D693%26doi%3D10.1021%2Facschembio.7b00942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Development of multi-functional histone deacetylase 6 degraders with potent anti-myeloma activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7042</span>– <span class="NLM_lpage">7057</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7042-7057&author=H.+Wuauthor=K.+Yangauthor=Z.+Zhangauthor=E.+D.+Leistenauthor=Z.+Liauthor=H.+Xieauthor=J.+Liuauthor=K.+A.+Smithauthor=Z.+Novakovaauthor=C.+Barinkaauthor=W.+Tang&title=Development+of+multi-functional+histone+deacetylase+6+degraders+with+potent+anti-myeloma+activity&doi=10.1021%2Facs.jmedchem.9b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Multifunctional Histone Deacetylase 6 Degraders with Potent Antimyeloma Activity</span></div><div class="casAuthors">Wu, Hao; Yang, Ka; Zhang, Zhongrui; Leisten, Eric D.; Li, Ziyuan; Xie, Haibo; Liu, Jin; Smith, Kerry A.; Novakova, Zora; Barinka, Cyril; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7042-7057</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) primarily catalyzes the removal of acetyl group from the side chain of acetylated lysine residues in cytoplasmic proteins such as α-tubulin and HSP90.  HDAC6 is involved in multiple disease-relevant pathways.  Based on the proteolysis targeting chimera strategy, we previously developed the first HDAC6 degrader by tethering a pan-HDAC inhibitor with cereblon (CRBN) E3 ubiquitin ligase ligand.  We herein report our new generation of multifunctional HDAC6 degraders by tethering selective HDAC6 inhibitor Nexturastat A with CRBN ligand that can synergize with HDAC6 degrdn. for the antiproliferation of multiple myeloma (MM).  This new class of degraders exhibited improved potency and selectivity for the degrdn. of HDAC6.  After the optimization of the linker length and linking positions, we discovered potent HDAC6 degraders with nanomolar DC50 and promising antiproliferation activity in multiple myeloma (MM) cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb58zf04V2GLVg90H21EOLACvtfcHk0lgNGnkti9fW1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO&md5=7aab402404a5fce732b4b68fd76395d4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00516%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DK.%2BA.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520multi-functional%2520histone%2520deacetylase%25206%2520degraders%2520with%2520potent%2520anti-myeloma%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7042%26epage%3D7057%26doi%3D10.1021%2Facs.jmedchem.9b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">The MX beamlines BL<sub>14.1-3</sub> at BESSY II</span>. <i>J. Large-scale Res. Facil.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">47</span>, <span class="refDoi"> DOI: 10.17815/jlsrf-2-64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.17815%2Fjlsrf-2-64" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=47&author=M.+Gerlachauthor=U.+Muellerauthor=M.+S.+Weiss&title=The+MX+beamlines+BL14.1-3+at+BESSY+II&doi=10.17815%2Fjlsrf-2-64"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.17815%2Fjlsrf-2-64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.17815%252Fjlsrf-2-64%26sid%3Dliteratum%253Aachs%26aulast%3DGerlach%26aufirst%3DM.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DThe%2520MX%2520beamlines%2520BL14.1-3%2520at%2520BESSY%2520II%26jtitle%3DJ.%2520Large-scale%2520Res.%2520Facil.%26date%3D2016%26volume%3D2%26spage%3D47%26doi%3D10.17815%2Fjlsrf-2-64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">DIALS: implementation and evaluation of a new integration package</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/S2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=DIALS%3A+implementation+and+evaluation+of+a+new+integration+package&doi=10.1107%2FS2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0lgNGnkti9fW1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDIALS%253A%2520implementation%2520and%2520evaluation%2520of%2520a%2520new%2520integration%2520package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2FS2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/S090744491003982X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS090744491003982X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2FS090744491003982X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0ljX5vBHCHhLAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS090744491003982X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744491003982X%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2FS090744491003982X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thorn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">AUSPEX: a graphical tool for X-ray diffraction data analysis</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1107/S205979831700969X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS205979831700969X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeqsrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=729-737&author=A.+Thornauthor=J.+Parkhurstauthor=P.+Emsleyauthor=R.+A.+Nichollsauthor=M.+Vollmarauthor=G.+Evansauthor=G.+N.+Murshudov&title=AUSPEX%3A+a+graphical+tool+for+X-ray+diffraction+data+analysis&doi=10.1107%2FS205979831700969X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AUSPEX: a graphical tool for X-ray diffraction data analysis</span></div><div class="casAuthors">Thorn, Andrea; Parkhurst, James; Emsley, Paul; Nicholls, Robert A.; Vollmar, Melanie; Evans, Gwyndaf; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">729-737</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">In this paper, AUSPEX, a new software tool for exptl. X-ray data anal., is presented.  Exploring the behavior of diffraction intensities and the assocd. estd. uncertainties facilitates the discovery of underlying problems and can help users to improve their data acquisition and processing in order to obtain better structural models.  The program enables users to inspect the distribution of obsd. intensities (or amplitudes) against resoln. as well as the assocd. estd. uncertainties (sigmas).  It is demonstrated how AUSPEX can be used to visually and automatically detect ice-ring artifacts in integrated X-ray diffraction data.  Such artifacts can hamper structure detn., but may be difficult to identify from the raw diffraction images produced by modern pixel detectors.  The anal. suggests that a significant portion of the data sets deposited in the PDB contain ice-ring artifacts.  Furthermore, it is demonstrated how other problems in exptl. X-ray data caused, for example, by scaling and data-conversion procedures can be detected by AUSPEX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_DReV2W1ibVg90H21EOLACvtfcHk0ljX5vBHCHhLAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeqsrfE&md5=bde8571b0ed8c18cf74c8b3a157c5a1a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS205979831700969X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS205979831700969X%26sid%3Dliteratum%253Aachs%26aulast%3DThorn%26aufirst%3DA.%26aulast%3DParkhurst%26aufirst%3DJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DG.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAUSPEX%253A%2520a%2520graphical%2520tool%2520for%2520X-ray%2520diffraction%2520data%2520analysis%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2017%26volume%3D73%26spage%3D729%26epage%3D737%26doi%3D10.1107%2FS205979831700969X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljX5vBHCHhLAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lin4HUHl47foQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lin4HUHl47foQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lin4HUHl47foQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebschner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Polder maps: improving OMIT maps by excluding bulk solvent</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">148</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1107/S2059798316018210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS2059798316018210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=148-157&author=D.+Liebschnerauthor=P.+V.+Afonineauthor=N.+W.+Moriartyauthor=B.+K.+Poonauthor=O.+V.+Sobolevauthor=T.+C.+Terwilligerauthor=P.+D.+Adams&title=Polder+maps%3A+improving+OMIT+maps+by+excluding+bulk+solvent&doi=10.1107%2FS2059798316018210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Polder maps: improving OMIT maps by excluding bulk solvent</span></div><div class="casAuthors">Liebschner, Dorothee; Afonine, Pavel V.; Moriarty, Nigel W.; Poon, Billy K.; Sobolev, Oleg V.; Terwilliger, Thomas C.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-157</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The crystallog. maps that are routinely used during the structure-soln. workflow are almost always model-biased because model information is used for their calcn.  As these maps are also used to validate the at. models that result from model building and refinement, this constitutes an immediate problem: anything added to the model will manifest itself in the map and thus hinder the validation.  OMIT maps are a common tool to verify the presence of atoms in the model.  The simplest way to compute an OMIT map is to exclude the atoms in question from the structure, update the corresponding structure factors and compute a residual map.  It is then expected that if these atoms are present in the crystal structure, the electron d. for the omitted atoms will be seen as pos. features in this map.  This, however, is complicated by the flat bulk-solvent model which is almost universally used in modern crystallog. refinement programs.  This model postulates const. electron d. at any voxel of the unit-cell vol. that is not occupied by the at. model.  Consequently, if the d. arising from the omitted atoms is weak then the bulk-solvent model may obscure it further.  A possible soln. to this problem is to prevent bulk solvent from entering the selected OMIT regions, which may improve the interpretative power of residual maps.  This approach is called a polder (OMIT) map.  Polder OMIT maps can be particularly useful for displaying weak densities of ligands, solvent mols., side chains, alternative conformations and residues both in terminal regions and in loops.  The tools described in this manuscript have been implemented and are available in PHENIX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwO8sssGmoxrVg90H21EOLACvtfcHk0lhCTzBCrqyvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu7w%253D&md5=0643653827bdba013d735703c941d434</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS2059798316018210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316018210%26sid%3Dliteratum%253Aachs%26aulast%3DLiebschner%26aufirst%3DD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DPoon%26aufirst%3DB.%2BK.%26aulast%3DSobolev%26aufirst%3DO.%2BV.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPolder%2520maps%253A%2520improving%2520OMIT%2520maps%2520by%2520excluding%2520bulk%2520solvent%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2017%26volume%3D73%26spage%3D148%26epage%3D157%26doi%3D10.1107%2FS2059798316018210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhCTzBCrqyvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knights, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stresser, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miners, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespi, C. L.</span></span> <span> </span><span class="NLM_article-title">In vitro drug metabolism using liver microsomes</span>. <i>Curr. Protoc. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1002/cpph.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1002%2Fcpph.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=27636111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A280%3ADC%252BC2svhsVCisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2016&pages=7-8&author=K.+M.+Knightsauthor=D.+M.+Stresserauthor=J.+O.+Minersauthor=C.+L.+Crespi&title=In+vitro+drug+metabolism+using+liver+microsomes&doi=10.1002%2Fcpph.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Drug Metabolism Using Liver Microsomes</span></div><div class="casAuthors">Knights Kathleen M; Stresser David M; Miners John O; Crespi Charles L</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7.8.1-7.8.24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Knowledge of the metabolic stability of newly discovered drug candidates eliminated by metabolism is essential for predicting the pharmacokinetic (PK) parameters that underpin dosing and dosage frequency.  Further, characterization of the enzyme(s) responsible for metabolism (reaction phenotyping) allows prediction, at least at the qualitative level, of factors (including metabolic drug-drug interactions) likely to alter the clearance of both new chemical entities (NCEs) and established drugs.  Microsomes are typically used as the enzyme source for the measurement of metabolic stability and for reaction phenotyping because they express the major drug-metabolizing enzymes cytochrome P450 (CYP) and UDP-glucuronosyltransferase (UGT), along with others that contribute to drug metabolism.  Described in this unit are methods for microsome isolation, as well as for the determination of metabolic stability and metabolite formation (including kinetics). © 2016 by John Wiley & Sons, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS42TPy_5UT8q-9TkgmBqTqfW6udTcc2eb23QGlWcfRCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svhsVCisg%253D%253D&md5=506e94e8b6f81de2f05104c1b7879312</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fcpph.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcpph.9%26sid%3Dliteratum%253Aachs%26aulast%3DKnights%26aufirst%3DK.%2BM.%26aulast%3DStresser%26aufirst%3DD.%2BM.%26aulast%3DMiners%26aufirst%3DJ.%2BO.%26aulast%3DCrespi%26aufirst%3DC.%2BL.%26atitle%3DIn%2520vitro%2520drug%2520metabolism%2520using%2520liver%2520microsomes%26jtitle%3DCurr.%2520Protoc.%2520Pharmacol.%26date%3D2016%26volume%3D74%26spage%3D7%26epage%3D8%26doi%3D10.1002%2Fcpph.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barré, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamouard, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillement, J. P.</span></span> <span> </span><span class="NLM_article-title">Equilibrium dialysis, ultrafiltration, and ultracentrifugation compared for determining the plasma-protein-binding characteristics of valproic acid</span>. <i>Clin. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1093/clinchem/31.1.60</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=10.1093%2Fclinchem%2F31.1.60" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=3917383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;key=1%3ACAS%3A528%3ADyaL2MXovFKntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1985&pages=60-64&author=J.+Barr%C3%A9author=J.+M.+Chamouardauthor=G.+Houinauthor=J.+P.+Tillement&title=Equilibrium+dialysis%2C+ultrafiltration%2C+and+ultracentrifugation+compared+for+determining+the+plasma-protein-binding+characteristics+of+valproic+acid&doi=10.1093%2Fclinchem%2F31.1.60"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrium dialysis, ultrafiltration, and ultracentrifugation compared for determining the plasma-protein-binding characteristics of valproic acid</span></div><div class="casAuthors">Barre, Jerome; Chamouard, Jean Marie; Houin, Georges; Tillement, Jean Paul</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-4</span>CODEN:
                <span class="NLM_cas:coden">CLCHAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-9147</span>.
    </div><div class="casAbstract">Equil. dialysis, ultrafiltration, and ultracentrifugation were compared for their reliability and applicability in the study of binding of an anticonvulsant drug, valproic acid  [99-66-1], by plasma proteins.  Drug binding was studied with pooled serum and with solns. of human serum albumin at physiol. concns.  Binding characteristics such as no. of binding sites, affinity consts., and percent of binding were compared as measured by each method in the therapeutic range for valproic acid.  Results by ultracentrifugation differed from those by equil. dialysis and ultrafiltration, which agreed reasonably well with each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7LVSvezYF-rVg90H21EOLACvtfcHk0liljTrKUXknbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXovFKntQ%253D%253D&md5=aca31b98ff28d80bfebf09a3513c7d95</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fclinchem%2F31.1.60&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fclinchem%252F31.1.60%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%25C3%25A9%26aufirst%3DJ.%26aulast%3DChamouard%26aufirst%3DJ.%2BM.%26aulast%3DHouin%26aufirst%3DG.%26aulast%3DTillement%26aufirst%3DJ.%2BP.%26atitle%3DEquilibrium%2520dialysis%252C%2520ultrafiltration%252C%2520and%2520ultracentrifugation%2520compared%2520for%2520determining%2520the%2520plasma-protein-binding%2520characteristics%2520of%2520valproic%2520acid%26jtitle%3DClin.%2520Chem.%26date%3D1985%26volume%3D31%26spage%3D60%26epage%3D64%26doi%3D10.1093%2Fclinchem%2F31.1.60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6THV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6THV','PDB','6THV'); return false;">PDB: 6THV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CW8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CW8','PDB','6CW8'); return false;">PDB: 6CW8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGM','PDB','5WGM'); return false;">PDB: 5WGM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGK','PDB','5WGK'); return false;">PDB: 5WGK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R0K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R0K','PDB','6R0K'); return false;">PDB: 6R0K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZU','PDB','6PZU'); return false;">PDB: 6PZU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PZR','PDB','6PZR'); return false;">PDB: 6PZR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGL','PDB','5WGL'); return false;">PDB: 5WGL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5G0I','PDB','5G0I'); return false;">PDB: 5G0I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY','PDB','6TCY'); return false;">PDB: 6TCY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEK','PDB','5EEK'); return false;">PDB: 5EEK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i52"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00567">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_62722"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00567?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00567</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary results for the cytotoxicity assay of <b>6a</b>–<b>6c</b> along with NextA (Figure S1); supplementary results for immunoblot analysis of Suprastat’s effects on α-tubulin acetylation in macrophages (Figure S2); supplementary figure for MD simulation of HDAC6 and the HDAC6/Suprastat complex (Figure S3); supplementary figure for the liver microsomal stability test (Figure S4); supplementary data on the HDAC enzymatic assay of <b>6d</b>–<b>6</b>f (Table S1); supplementary table for crystallization data collection and refinement statistics (Table S2); and <sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, and HPLC purity reports for compounds <b>6a</b>–<b>6c</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf">PDF</a>)</p></li><li><p class="inline">MD simulations of the HDAC6/Suprastat complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_002.mp4">MP4</a>)</p></li><li><p class="inline">Molecular formula strings of compounds <b>2</b>, <b>4a</b>–<b>4d</b>, <b>5a</b>–<b>5d</b>, <b>6b</b>–<b>6f</b>, Suprastat, and nexturastat including screening data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_003.csv">CSV</a>)</p></li><li><p class="inline">A snapshot of the MD simulations of the HDAC6/Suprastat complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_004.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_001.pdf">jm0c00567_si_001.pdf (1.68 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_002.mp4">jm0c00567_si_002.mp4 (1.33 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_003.csv">jm0c00567_si_003.csv (1.41 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00567/suppl_file/jm0c00567_si_004.pdb">jm0c00567_si_004.pdb (439.02 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and corresponding structure factors for the zHDAC6-CD2/Suprastat complex have been deposited at the Protein Data Bank (PDB) as the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TCY">6TCY</a> entry. Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00567&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-18%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00567" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00567" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0be7748c32367","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
