
<html lang="en"     class="pb-page"  data-request-id="86c17acc-3e9a-41df-9579-165b8d271820"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.1c00120"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Pyrrole-imidazole Polyamides as PD-L1 Expression Inhibitors and Their Anticancer Activity via Immune and Nonimmune Pathways" /></meta><meta name="dc.Creator" content="Ming  Wang" /></meta><meta name="dc.Creator" content="Xudong  Ma" /></meta><meta name="dc.Creator" content="Kang  Zhou" /></meta><meta name="dc.Creator" content="Huijuan  Mao" /></meta><meta name="dc.Creator" content="Jiachun  Liu" /></meta><meta name="dc.Creator" content="Xuqiong  Xiong" /></meta><meta name="dc.Creator" content="Xiaoyin  Zhao" /></meta><meta name="dc.Creator" content="Suresh  Narva" /></meta><meta name="dc.Creator" content="Yoshimasa  Tanaka" /></meta><meta name="dc.Creator" content="Yanling  Wu" /></meta><meta name="dc.Creator" content="Chuanxin  Guo" /></meta><meta name="dc.Creator" content="Hiroshi  Sugiyama" /></meta><meta name="dc.Creator" content="Wen  Zhang" /></meta><meta name="dc.Description" content="In recent years, PD-1 immune checkpoint inhibitors based on monoclonal antibodies have revolutionized cancer therapy, but there still exist unresolved issues, such as the high cost, the relatively ..." /></meta><meta name="Description" content="In recent years, PD-1 immune checkpoint inhibitors based on monoclonal antibodies have revolutionized cancer therapy, but there still exist unresolved issues, such as the high cost, the relatively ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 5, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00120" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00120" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00120" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00120" /></link>
        
    
    

<title>Discovery of Pyrrole-imidazole Polyamides as PD-L1 Expression Inhibitors and Their Anticancer Activity via Immune and Nonimmune Pathways | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00120" /></meta><meta property="og:title" content="Discovery of Pyrrole-imidazole Polyamides as PD-L1 Expression Inhibitors and Their Anticancer Activity via Immune and Nonimmune Pathways" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0008.jpeg" /></meta><meta property="og:description" content="In recent years, PD-1 immune checkpoint inhibitors based on monoclonal antibodies have revolutionized cancer therapy, but there still exist unresolved issues, such as the high cost, the relatively low response rates, and so on, compared with small-molecule drugs. Herein a type of pyrrole-imidazole (Py-Im) polyamide as a small-molecule DNA binder was designed and synthesized, which could competitively bind to the same double-stranded DNA stretch in the PD-L1 promoter region as the STAT3 binding site and thus downregulate PD-L1 expression. It was demonstrated that the Py-Im polyamides directly caused apoptosis in tumor cells and retarded cell migration in the absence of T cells through inhibiting the Akt/caspase-3 pathway. Also, in a coculture system, they enhanced the T-cell-mediated killing of tumor cells by the reversal of immune escape. Because such polyamides induced antitumor effects via both immune and nonimmune pathways, they could be further developed as promising PD-L1 gene-targeting antitumor drugs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00120"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00120">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/jctcce/largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00120&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00120&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00120&amp;href=/doi/10.1021/acs.jmedchem.1c00120" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 6021-6036</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00085" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00138" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Pyrrole-imidazole Polyamides as PD-L1 Expression Inhibitors and Their Anticancer Activity via Immune and Nonimmune Pathways</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ming Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ming Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ming++Wang">Ming Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xudong Ma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xudong Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xudong++Ma">Xudong Ma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kang Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kang Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kang++Zhou">Kang Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huijuan Mao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huijuan Mao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huijuan++Mao">Huijuan Mao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiachun Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiachun Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiachun++Liu">Jiachun Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xuqiong Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuqiong Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuqiong++Xiong">Xuqiong Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoyin Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoyin Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyin++Zhao">Xiaoyin Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Suresh Narva</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Suresh Narva</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Suresh++Narva">Suresh Narva</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0702-784X" title="Orcid link">http://orcid.org/0000-0003-0702-784X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yoshimasa Tanaka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yoshimasa Tanaka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Innovation in Immunoregulative Technology and Therapeutics, Graduate School of Medicine, Kyoto University, Kyoto 606-8501, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yoshimasa++Tanaka">Yoshimasa Tanaka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanling Wu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanling Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Molecular Immunology, Virus Inspection Department, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou 310051, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a7decbd0d2e7c4c3c489ddcd89c4c9"><span class="__cf_email__" data-cfemail="047d687371446760672a7e6e2a676a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanling++Wu">Yanling Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chuanxin Guo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chuanxin Guo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Graduate School of Science, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chuanxin++Guo">Chuanxin Guo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hiroshi Sugiyama</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hiroshi Sugiyama</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Graduate School of Science, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#b9d1caf9d2ccdad1dcd497d2c0d6cdd694cc97d8da97d3c9"><span class="__cf_email__" data-cfemail="83ebf0c3e8f6e0ebe6eeade8faecf7ecaef6ade2e0ade9f3">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hiroshi++Sugiyama">Hiroshi Sugiyama</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8923-5946" title="Orcid link">http://orcid.org/0000-0001-8923-5946</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Wen Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wen Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lab of Chemical Biology and Molecular Drug Design, College of Pharmaceutical Science, Zhejiang University of Technology, Hangzhou 310014, China</div><div class="loa-info-affiliations-info">Institute of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3245485a535c55040172485847461c5756471c515c"><span class="__cf_email__" data-cfemail="b1c6cbd9d0dfd68782f1cbdbc4c59fd4d5c49fd2df">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wen++Zhang">Wen Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3247-9672" title="Orcid link">http://orcid.org/0000-0002-3247-9672</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00120&amp;href=/doi/10.1021%2Facs.jmedchem.1c00120" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 6021–6036</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 5, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 January 2021</li><li><span class="item_label"><b>Published</b> online</span>5 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00120" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00120</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6021%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMing%2BWang%252C%2BXudong%2BMa%252C%2BKang%2BZhou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.1c00120%26title%3DDiscovery%2Bof%2BPyrrole-imidazole%2BPolyamides%2Bas%2BPD-L1%2BExpression%2BInhibitors%2Band%2BTheir%2BAnticancer%2BActivity%2Bvia%2BImmune%2Band%2BNonimmune%2BPathways%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6036%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00120"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">852</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00120" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Pyrrole-imidazole Polyamides as PD-L1 Expression Inhibitors and Their Anticancer Activity via Immune and Nonimmune Pathways&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ming&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xudong&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Kang&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Huijuan&quot;,&quot;last_name&quot;:&quot;Mao&quot;},{&quot;first_name&quot;:&quot;Jiachun&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xuqiong&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Xiaoyin&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Suresh&quot;,&quot;last_name&quot;:&quot;Narva&quot;},{&quot;first_name&quot;:&quot;Yoshimasa&quot;,&quot;last_name&quot;:&quot;Tanaka&quot;},{&quot;first_name&quot;:&quot;Yanling&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Chuanxin&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Hiroshi&quot;,&quot;last_name&quot;:&quot;Sugiyama&quot;},{&quot;first_name&quot;:&quot;Wen&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;6021-6036&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00120&quot;},&quot;abstract&quot;:&quot;In recent years, PD-1 immune checkpoint inhibitors based on monoclonal antibodies have revolutionized cancer therapy, but there still exist unresolved issues, such as the high cost, the relatively low response rates, and so on, compared with small-molecule drugs. Herein a type of pyrrole-imidazole (Py-Im) polyamide as a small-molecule DNA binder was designed and synthesized, which could competitively bind to the same double-stranded DNA stretch in the PD-L1 promoter region as the STAT3 binding site and thus downregulate PD-L1 expression. It was demonstrated that the Py-Im polyamides directly caused apoptosis in tumor cells and retarded cell migration in the absence of T cells through inhibiting the Akt/caspase-3 pathway. Also, in a coculture system, they enhanced the T-cell-mediated killing of tumor cells by the reversal of immune escape. Because such polyamides induced antitumor effects via both immune and nonimmune pathways, they could be further developed as promising PD-L1 gene-targeting antitumor drugs.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00120&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00120" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00120&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00120" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00120&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00120" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00120&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00120&amp;href=/doi/10.1021/acs.jmedchem.1c00120" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00120" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00120" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00120%26sid%3Dliteratum%253Aachs%26pmid%3D33949196%26genre%3Darticle%26aulast%3DWang%26date%3D2021%26atitle%3DDiscovery%2Bof%2BPyrrole-imidazole%2BPolyamides%2Bas%2BPD-L1%2BExpression%2BInhibitors%2Band%2BTheir%2BAnticancer%2BActivity%2Bvia%2BImmune%2Band%2BNonimmune%2BPathways%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D6021%26epage%3D6036%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291503" title="Organic polymers">Organic polymers</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0008.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In recent years, PD-1 immune checkpoint inhibitors based on monoclonal antibodies have revolutionized cancer therapy, but there still exist unresolved issues, such as the high cost, the relatively low response rates, and so on, compared with small-molecule drugs. Herein a type of pyrrole-imidazole (Py-Im) polyamide as a small-molecule DNA binder was designed and synthesized, which could competitively bind to the same double-stranded DNA stretch in the PD-L1 promoter region as the STAT3 binding site and thus downregulate PD-L1 expression. It was demonstrated that the Py-Im polyamides directly caused apoptosis in tumor cells and retarded cell migration in the absence of T cells through inhibiting the Akt/caspase-3 pathway. Also, in a coculture system, they enhanced the T-cell-mediated killing of tumor cells by the reversal of immune escape. Because such polyamides induced antitumor effects via both immune and nonimmune pathways, they could be further developed as promising PD-L1 gene-targeting antitumor drugs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88046" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88046" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Tumor immunotherapy refers to therapeutic strategies aimed at regulating the immune system to eradicate cancer.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Tumors suppress the effector functions of T lymphocytes by modulating the inhibitory pathways, called immune checkpoints.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Programmed death-1 (PD-1 or CD279) and its ligand PD-L1 (also termed B7-H1 or CD274), which are members of the B7 family of molecules, constitute the most promising immune checkpoint in cancer immunotherapy. The engagement of PD-1 with PD-L1 delivers a coinhibitory signal in T cells and regulates the production of cytokines such as interferons (IFNs).<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> The blockade of the PD-1/PD-L1 interaction therefore rejuvenates T cells and increases antitumor immunity. Antihuman PD-L1 monoclonal antibodies (mAbs), such as avelumab (Merck & Co.), atezolizumab (F. Hoffmann-La Roche), and durvalumab (Astrazeneca), are used for cancer treatments for non-small-cell lung cancer, urothelial carcinoma, and metastatic Merkel cell carcinoma. The mAbs specific for the PD-1 immune checkpoint are effective in some cancer patients in terms of progression-free survival, overall survival, and tolerability, but the high cost of these agents may affect the economic sustainability of the healthcare system, there exists immunogenic side effects, and some patients fail to respond to the treatment with such drugs based on clinical follow-up data. It is thus imperative to develop small-molecule modulators for the PD-1 immune checkpoint.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Meanwhile, more recent studies have shown that PD-L1 delivers an important intrinsic signal into tumors, which could enhance the cell proliferation and survival by inhibiting autophagy and mTOR activation in the absence of PD-1, but extensive studies have not been conducted so far.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a></div><div class="NLM_p">On the other hand, signal transducer and activator of transcription (STAT) plays a key role in signal transduction and transcriptional activation. The STAT family consists of six members, STAT1–STAT6, of which the gene encoding STAT3 is located on chromosome 17 q21.2.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> STAT3 is an important member of the signaling and transcriptional activator family and is broadly expressed in many different types of tissues and cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> When STAT3 is overactivated, it exhibits an oncogenic property and plays roles in the proliferation, differentiation, invasion, metastasis, antiapoptosis, and angiogenesis in tumor cells.<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8,11)</a> It was also demonstrated that the continual activation of STAT3 in tumor cells is associated with the abnormal expression of PD-L1 protein.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> The current studies have shown that STAT3 interacts with the promoter region of PD-L1 and regulates its expression,<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14−17)</a> which provides a feasible site for Py-Im polyamides target-regulating the PD-L1 expression.</div><div class="NLM_p">Studies on netropsin and distamycin, both of which are known as natural products, suggested that a certain class of small-molecule compounds can specifically recognize predetermined DNA sequences and interact with the minor groove of double-stranded DNA molecules.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Inspired by these two small-molecule compounds, Dervan and colleagues at the California Institute of Technology first developed pyrrole-imidazole (Py-Im) polyamides similar to the two natural small-molecule compounds.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This type of small-molecule compound, which mainly comprises <i>N</i>-methylpyrrole (Py), <i>N</i>-methylimidazole (Im) and aliphatic chain amino acid residues, can recognize and interact with DNA base pairs in the DNA minor groove with sequence specificity and has good cell membrane permeability.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> In addition, Dervan’s group revealed the rules of the recognition by Py-Im polyamides targeting the DNA sequences, that is, a Py/Py pair nonpreferentially binding to T·A and A·T, a Py/Im pair specifying C·G, and an Im/Py pair for G·C.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> Next, Zhang et al. reported that an (<i>S</i>)-β-hydroxy-γ-aminobutyric acid/β-alanine pair can discriminate T·A from A·T.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> To date, all four Watson–Crick base pairs, C·G, G·C, T·A, and A·T, in the DNA minor groove can be specifically recognized by the different recognition units based on the rules of recognition in the interaction between polyamides and DNA. On the basis of the affinity and specificity of this kind of molecule for B-DNA, Dervan’s group reported the effect of polyamides on the expression of the corresponding genes.<a onclick="showRef(event, 'ref21 ref22 ref26 ref29'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref26 ref29">(21,22,26,29)</a> Then, Fukuda and colleagues demonstrated that Py-Im polyamides could specifically interact with the TGF-β1 promoter and effectively alleviate a progressive kidney disease.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> More recently, Fukuda and Sugiyama, in collaboration with a Japanese pharmaceutical company, demonstrated that polyamide-targeted TGF-β1 gene therapy was effective in wound scar repair and predicted that the polyamide could be developed as a gene-targeted drug for clinical treatment.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a></div><div class="NLM_p last">In this work, we newly developed three Py-Im polyamides, of which two specifically interacted with the sequence of the target PD-L1 promoter and one was mismatched to the target according to the “polyamide/DNA recognition rule”. We then examined the ability and influence of their binding to the PD-L1 target gene sequence by monitoring the downstream gene expression and examined the mechanism by which the synthetic compounds modulated cancer cell apoptosis. Experimental results have shown that our synthetic matched Py-Im polyamides are able to downregulate the inducible PD-L1 expression by competitively inhibiting the STAT3 binding to PD-L1 promoter region, resulting in the poptosis of cancer cells not only by the nonimmune pathway, that is, Akt/caspase-3, but also by the immune pathway. This works sheds light on cost-effective cancer immunotherapy based on downregulating the expression of PD-L1 that reduces the PD-1 coinhibitory signaling and rejuvenates the T cells for effectively killing tumor cells.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34560" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34560" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Py-Im Polyamides Competitively Interact with the STAT3 Binding Site in the Promoter Region of the PD-L1 Gene at the Extracellular Level</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Design and Target Binding of Novel Polyamides to a PD-L1 Gene DNA Sequence</h4><div class="NLM_p">The structures of newly designed Py-Im polyamides <b>1</b>–<b>3</b> are depicted (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), and their synthetic strategy and characterization are shown (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Scheme S1</a>, <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The DNA sequences of the STAT3 binding site (SBS) in the PD-L1 promoter region are shown (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The polyamides examined in this study include two matched Py-Im polyamides <b>1</b> and <b>2</b> and a mismatched Py-Im polyamide (Mis-polyamide) to the PD-L1 28nt wild-type double-stranded DNA sequence (wt-ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Polyamides <b>1</b> and <b>2</b> can specifically recognize and bind to matched ds-DNA and Mis-polyamide cannot, according to “polyamide/DNA recognition rule”.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27 ref28">(23−28)</a> The possible binding modes of the Py-Im polyamides to the target DNA based on the “polyamide/DNA recognition rule” are illustrated (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Py-Im polyamides structures and (B) the ball-and-stick model of polyamides binding to their target sites. Target sites are the underlined DNA sequences of the STAT3 binding site (SBS) located at the upstream DNA (nucleotide base pairs −91 to −118) in the PD-L1 gene promoter region. ●, <i>N</i>-methylimidazole residue (Im); ○, <i>N</i>-methylpyrrole residue (Py); β or β-head: β-alanine residue; Dp, dimethylamino-propylamide; ⊂, γ or γ-turn (γ-aminobutanoic acid turn) residue between the two subunits; Ac, acetyl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Sequences of DNA Oligomers Used in This Study</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">oligomers</th><th class="colsep0 rowsep0" align="center">sequences</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">PD-L1 28nt-wt-ds-DNA (wt-ds-DNA)</td><td class="colsep0 rowsep0" align="left">5′-TTCCGCCGAT<u class="uu"><b>TTCACCGAA</b></u>GGTCAGGAA-3′</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3′-AAGGCGGCTA<u class="uu"><b>AAGTGGCTT</b></u>CCAGTCCTT-5′</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PD-L1 28nt-wt-ss-DNA (ss-DNA)</td><td class="colsep0 rowsep0" align="left">5′-TTCCGCCGAT<u class="uu"><b>TTCACCGAA</b></u>GGTCAGGAA-3′</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">PD-L1 28nt-mut-ds-DNA (mut-ds-DNA)</td><td class="colsep0 rowsep0" align="left">5′-TTCCGCCGAT<u class="uu"><b>T</b><i>G</i><b>C</b><i>G</i><b>C</b><i>T</i><b>G</b><i>G</i><b>A</b></u>GGTCAGGAA-3′</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3′-AAGGCGGCTA<u class="uu"><b>A</b><i>C</i><b>G</b><i>C</i><b>G</b><i>A</i><b>C</b><i>C</i><b>T</b></u>CCAGTCCTT-5′</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5′-biotin–ds-DNA</td><td class="colsep0 rowsep0" align="left">5′-biotin–TTCCGCCGAT<u class="uu"><b>TTCACCGAA</b></u>GGTCAGGAA-TATA-TTCCTGACC<u class="uu"><b>TTCGGTGAA</b></u>ATCGGCGGAA-3′</td></tr></tbody></table></div></div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Analysis of the Interaction between Py-Im Polyamides and Target DNAs by the Electrophoretic Mobility Shift Assay (EMSA)</h4><div class="NLM_p">We first examined whether the newly designed polyamides interact with SBS in the PD-L1 promoter at physiologically relevant concentrations (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The interaction between polyamides <b>1</b> and <b>2</b> and their matched DNA sequence (wt-ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) became more and more strong in the compound concentration-dependent mode (<8 μM) because a type of polyamide/ds-DNA complex was formed (red dashed arrow). At a concentration of 1 μM, such interaction was weak; when the concentration was 2 μM, it was more explicit. When the polyamide concentration reached 16 μM, the intensity of the wt-ds-DNA band and the polyamide/wt-ds-DNA complex band was markedly reduced. This phenomenon is called Beer’s law, in which the agglutination viscosity of the compound may damage the DNA when the concentration is too high.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> By contrast, no significant binding was observed between the Py-Im polyamides and the single-stranded DNA (ss-DNA) and between the Mis-polyamide and the SBS of the PD-L1 promoter.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. EMSA image (A) for Py-Im polyamides interacting with the PD-L1 promoter sequence and (B) for Py-Im polyamides and the STAT3 protein competitively binding to the SBS in the PD-L1 promoter region. (A) Lane 1: DNA standard Marker (M); Lane 2: the forward oligo of wt-ds-DNA alone ss-DNA) in buffer containing 0.5% DMSO; Lane 3: forward oligo + polyamide <b>1</b> (16 μM); Lane 4: forward oligo + polyamide <b>2</b> (16 μM); Lane 5: wt-ds-DNA alone; Lanes 6–10: wt-ds-DNA + polyamide <b>1</b> at concentrations of 1, 2, 4, 8, and 16 μM; Lanes 11–15: wt-ds-DNA + polyamide <b>2</b> at concentrations of 1, 2, 4, 8, and 16 μM; Lane 16: wt-ds-DNA + Mis-polyamide <b>3</b> (24 μM); Lane 17: mut-ds-DNA alone; Lane 18: mut-ds-DNA + polyamide <b>1</b> (16 μM); Lane 19: mut-ds-DNA + polyamide <b>2</b> (16 μM). (B) Lane 1: wt-ds-DNA alone; Lane 2: wt-ds-DNA + polyamide <b>1</b> (12 μM); Lane 3: wt-ds-DNA + polyamide <b>2</b> (12 μM); Lane 4: wt-ds-DNA + STAT3 protein (1 μL); Lanes 5–9: wt-ds-DNA + STAT3 protein + polyamide <b>1</b> at concentrations of 1, 2, 4, 8, and 12 μM; Lanes 10–14: wt-ds-DNA + STAT3 protein + polyamide <b>2</b> at concentrations of 1, 2, 4, 8, and 12 μM; Lane 15: wt-ds-DNA + Mis-polyamide <b>3</b> (24 μM); Lane 16: wt-ds-DNA + STAT3 protein + Mis-polyamide (24 μM). The final concentration of DNA was 2 μM for single-stranded DNA and 1 μM for double-stranded DNA, and the final volume of the protein was 1 μL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We next performed a competitive EMSA in the presence of the STAT3 protein to examine the binding ability of <b>1</b> and <b>2</b> to SBS (5′-TTCACCGAA-3′/5′-TTCGGTGAA-3′, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) in the promoter of PD-L1. When the STAT3 protein interacted with the 28nt double-stranded DNA (wt-ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) in the PD-L1 promoter, two lagging bands were observed in lanes 4–7 and 10 and 11 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), which was possibly due to two structurally different wt-ds-DNA/protein complexes. With the increase in Py-Im polyamides, the bands of wt-ds-DNA/protein complexes decreased, and those of wt-ds-DNA/polyamide complexes increased in a polyamide concentration-dependent manner, and the band of wt-ds-DNA/protein complexes apparently disappeared at a compound concentration of ≥8 μM. In contrast, as seen in lanes 15 and 16, the Mis-polyamide did not influence the binding of the STAT3 protein to the SBS. The results indicate that newly designed polyamides can strongly interact with the SBS to inhibit the interaction between the STAT3 protein and the target, strongly suggesting that polyamides might possess an ability to modulate the expression of PD-L1.</div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Analysis of <i>T</i><sub>m</sub> Values for PD-L1 Double-Stranded DNAs in the Presence/Absence of Py-Im Polyamides</h4><div class="NLM_p">To examine the thermal stability of the complex formed between the polyamides and target DNAs, the temperature (<i>T</i><sub>m</sub>/°C), at which half of the DNA was melted, was calculated by monitoring the ultraviolet (UV) absorbance at 260 nm (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Upon the interaction of polyamides <b>1</b> or <b>2</b> with the matched wt-ds-DNA of the PD-L1 promoter, an increase in the melting temperature of the wt-ds-DNA was observed (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S1A,B</a>). When the ratio of Py-Im polyamide <b>1</b> or <b>2</b> to the DNA concentration was 1:1, the melting temperature of <b>1</b> and <b>2</b> was increased by 8.2 and 7.0 °C, respectively, whereas there was almost no change in the case of the Mis-polyamide (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S1C</a>). When the ratio of polyamide to DNA concentration was >1, however, the melting temperature almost no longer changed, suggesting that the DNA was saturated with Py-Im polyamides when the ratio of polyamide to DNA concentration was 1. When Py-Im polyamide <b>1</b> was compared with <b>2</b>, the former was more effective in binding to DNA in a cell-free system, possibly because there were more hydrogen bonds between the longer Py-Im polyamide <b>1</b> and the DNA. The Mis-polyamide had no effect on the <i>T</i><sub>m</sub> of the wt-ds-DNA, indicating that the newly synthesized Py-Im polyamides <b>1</b> and <b>2</b> were highly specific and possessed strong affinity to the matched wt-ds-DNA. By contrast, no thermal shift was observed when mixing polyamides <b>1</b>/<b>2</b> and a mutant ds-DNA (mut-ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S1D,E</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <i>T</i><sub>m</sub> (°C) Values for the Interaction between Py-Im Polyamides and ds-DNA<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">wt-ds-DNA</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">mut-ds-DNA</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">ds-DNA/polyamide</th><th class="colsep0 rowsep0" align="center" char="±">polyamide <b>1</b></th><th class="colsep0 rowsep0" align="center" char="±">polyamide <b>2</b></th><th class="colsep0 rowsep0" align="center" char="±">Mis-polyamide</th><th class="colsep0 rowsep0" align="center" char="±">polyamide <b>1</b></th><th class="colsep0 rowsep0" align="center" char="±">polyamide <b>2</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>m</sub>/°C</td><td class="colsep0 rowsep0" align="left">1:0</td><td class="colsep0 rowsep0" align="char" char="±">70.0 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">70.0 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">70.0 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">73.4 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">73.4 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>ΔT</i><sub>m</sub>/°C</td><td class="colsep0 rowsep0" align="left">1:0.5</td><td class="colsep0 rowsep0" align="char" char="±">2.7 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">2.9 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.1 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>ΔT</i><sub>m</sub>/°C</td><td class="colsep0 rowsep0" align="left">1:1</td><td class="colsep0 rowsep0" align="char" char="±">8.2 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">7.0 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.3 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>ΔT</i><sub>m</sub>/°C</td><td class="colsep0 rowsep0" align="left">1:4</td><td class="colsep0 rowsep0" align="char" char="±">8.6 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">7.3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">0.7 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup><sup>a</sup></sup><p class="last">Values are expressed as mean ± SD (triplicates).</p></div></div></div></div><div id="sec2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Understanding the Mechanism Underlying the Interaction between Polyamides and the Target DNAs by Circular Dichroism and Surface Plasmon Resonance</h4><div class="NLM_p">To further analyze the binding properties of the novel polyamides to the target DNA sequences in the PD-L1 gene promoter region, including the affinity, specificity, and stoichiometry, circular dichroism (CD) and surface plasmon resonance (SPR) techniques were applied to examine the interaction properties of polyamides <b>1</b>–<b>3</b> with target DNAs (wt-ds-DNA, mut-ds-DNA, ss-DNA, and 5′-biotin–ds-DNA in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). CD and SPR techniques are, more importantly, regarded as practical methods for illuminating the interaction of small molecules with DNAs. In particular, CD spectroscopy has been extensively used for exploring the nature of structural changes of nuclei acids in the presence of binders.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38">(35−38)</a></div><div class="NLM_p">CD spectra of the wt-ds-DNA alone (dotted line) showed a palpable positive band at 283.5 to 284.0 nm and a weak negative band at 244.5–145.0 nm (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B), corresponding to a double-stranded DNA,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> whereas those of the three compounds alone were invisible due to their nonoptically active properties (dashed line). It was observed that the intensity of positive peaks at 283.5–287.5 nm presented an obvious change with a red shift of 3.0–3.5 nm and a drop of 1.0 to 1.1 in the ellipticity, and that of the negative band at 244.5–145.0 nm did slightly change but in the opposite direction, with the gradual titration of polyamide <b>1</b> or <b>2</b> into the wt-ds-DNA-containing solution at 2:1, 0.4:1, 0.6:1, 0.8:1, 1.0:1, and 2.0:1 ratios of compound/DNA. It is worth noting that there exists two different isoelliptic points at 340.0 nm (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and 325.0 nm (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), which indicate the two-state nature of the structural transition upon polyamide binding to wt-ds-DNA. More importantly, we found two types of slightly different strong induced CD (ICD) spectra of the polyamides from 300.0 to 394.0 nm with a wavelength of 340.0 nm in maximum ellipticity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and from 297.5 to 374.5 nm with a wavelength of 325.0 nm in maximum ellipticity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), both suggesting that there is a strong interaction between <b>1</b> or <b>2</b> and the target wt-ds-DNA. In addition, the CD signals at 340.0 nm plotted against the compound/DNA ratio showed an approximately 1:1 polyamide <b>1</b>–DNA complex, with the mode of interaction being a polyamide <b>1</b> binding to a wt-ds-DNA in the minor groove according to the previously discovered “polyamide/DNA recognition rule”, as shown in the inset of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A. Likewise, the inset in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B also demonstrated the stoichiometry of a 1:1 polyamide <b>2</b>–DNA complex. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S2A</a>, a negative-control Mut-polyamide (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which was expected not to recognize and interact with the 28nt wt-ds-DNA in the PD-L1 gene promoter sequence in the minor groove on the basis of the “polyamide/DNA recognition rule”, had no influence on the target wt-ds-DNA under the same experimental conditions, even when a higher concentration of Mut-polyamide at 8 μM was used.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Ability of two polyamides <b>1</b> and <b>2</b> to specifically interact with a 28nt double-stranded DNA sequence of PD-L1 gene promoter (wt-ds-DNA and 5′-biotin–ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) by (A,B) CD and (C,D) SPR. (E,F) Molecular modeling of the interaction between the polyamides and the target sequence of DNA 5′-TTCACCGAA-3′/5′-TTCGGTGAA-3′. CD spectra for the interaction of (A) <b>1</b> and (B) <b>2</b> with wt-ds-DNA. Solid arrows at 244.5–245.0 nm (A,B) indicate the effect of the concentration of <b>1</b> and <b>2</b> on the trough of wt-ds-DNA; solid arrows at 283.5–286.5 nm (A) and 284.0–287.5 nm (B)  indicate the effect of the concentration of <b>1</b> and <b>2</b> on the peak of wt-ds-DNA; dotted arrows near 300.0 nm (A) and 297.5 nm (B) indicate the isoelliptic points; solid arrows at 340.0 nm (A) and 325.0 nm (B) indicate the effect of <b>1</b> and <b>2</b> on the ICD peak at 300–394 nm (A) and 297.5–374.5 nm (B), respectively. All CD spectra were obtained from 225 to 400 nm for 28nt wt-ds-DNA in the promoter sequence of PD-L1 in 10 mM sodium cacodylate buffer, pH 7.4 containing 100 mM NaCl and 0.5% DMSO at compound/DNA ratios (<i>r</i>) of 0.0, 0.2, 0.4, 0.6, 0.8, 1.0, and 2.0 at ambient temperature. In the insets of panels A and B, the positive ICD signals at 340.0 nm for polyamide <b>1</b> and at 325.0 nm for polyamide <b>2</b> are plotted against the compound/DNA ratio, respectively. The black dotted curve represents DNA alone, and the black dashed line represents the compound alone. Monitoring of the interaction of <b>1</b> (C) and <b>2</b> (D) with 5′-biotin–ds-DNA by SPR sensorgram. The biotin-conjugated ds-DNA was immobilized on the surface of an SA sensor chip. The binding response was monitored with time. The SPR curves were fitted to a 1:1 interaction model with a mass-transfer effect. Experimental curves are depicted in color, and fitting curves are depicted in black. The association phase was run for 300 s until the steady state was confirmed, and the dissociation phase was run for 280 s for all experiments. The concentrations of <b>1</b> and <b>2</b> were 10, 20, 40, 60, 80, 100, 150, 250, 350, and 500 nM, respectively. All experiments were performed in triplicate. (E) Molecular models of the 1:1 complex of polyamides <b>1</b>/<b>2</b> and the ds-DNA, respectively, by computer-assisted molecular simulation and (F) the interactive binding energy calculated by energy minimization with the LibDock evaluation. The polyamide is depicted as a stick, N3 is in blue, A base is in blue, G is in yellow, and T is in red. In the Py-Im polyamides, hydrogen is shown in gray, nitrogen is in blue, oxygen us in red, and covalent bonds are in green. The DNA strand is represented as purple and green ribbons, and its surface rendering is set to be solid and transparent. The sodium ions and water molecules are not shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To examine the ability of <b>1</b> and <b>2</b> to selectively recognize and bind to matched ds-DNA (wt-ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) versus mutant ds-DNA (mut-ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) based on the “polyamide/DNA recognition rule” and single-stranded DNA (ss-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), the CD spectra of the 28nt mut-ds-DNA and ss-DNA were measured in the presence and absence of <b>1</b> and <b>2</b>. <b>1</b> and <b>2</b> had almost no influence on the CD behaviors of mut-ds-DNA and ss-DNA, but the CD spectra of the mut-ds-DNA and ss-DNA alone were both characteristic of double- and single-stranded DNA structures, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S2B–E</a>). The CD spectra of the mut-ds-DNA displayed almost an identically strong peak and trough at 283.0 to 283.5 nm and 244.0 to 245.5 nm (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S2B,C</a>), respectively, whereas those of ss-DNA had a strong peak at 282.0 nm and a relatively weak trough at 241.0 nm (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S2D,E</a>). All in all, the CD results demonstrate that the examined polyamides <b>1</b> and <b>2</b> can specifically recognize and strongly bind to 28nt matched ds-DNA in the PD-L1 gene promoter region with a 1:1 stoichiometry of polyamide/ds-DNA.</div><div class="NLM_p">SPR is a convenient and effective method for screening new drugs, especially for evaluating the binding natures of small molecules with target DNAs.<a onclick="showRef(event, 'ref27 ref28 ref38 ref40'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref38 ref40">(27,28,38,40)</a> In this study, the binding ability/affinity of the polyamides to the target ds<b>-</b>DNA sequence (5′-biotin–ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) was evaluated on the same sensor chip with the same immobilized 5′-biotin–ds-DNA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The <i>K</i><sub>D</sub> of polyamide <b>1</b> binding to ds<b>-</b>DNA was 8.28 (±0.56) × 10<sup>–9</sup> M, whose interaction was approximately six times stronger than that of polyamide <b>2</b> with a <i>K</i><sub>D</sub> of 4.99 (±0.47) × 10<sup>–8</sup> M. Both of the experimental SPR sensorgrams (curves in color) were very well fitted to a 1:1 global theoretical fitting model with mass transport (curves in black). Furthermore, binding of the polyamides <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C) and <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D) to the target DNA sequence led to saturation at relatively higher compound concentrations of 350 to 500 nM, implying a 1:1 stoichiometry of <b>1</b> or <b>2</b> to the target DNA. It is worth noting that polyamide <b>2</b> interacted swiftly with the target DNA to achieve a steady state at a concentration of 20 nM, whereas polyamide <b>1</b> attained a steady state at concentrations of not less than 80 nM, demonstrating that there was a more than five-fold difference in the binding rates of polyamide <b>1</b> with a <i>k</i><sub>a</sub> of 6.47 (±0.18) × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup> and <b>2</b> with a <i>k</i><sub>a</sub> of 1.28 (±0.14) × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup>. But interestingly, polyamide <b>1</b> was much more slowly dissociated from the sensor chip with a <i>k</i><sub>d</sub> of 1.06 (±0.23) × 10<sup>–2</sup> s<sup>–1</sup> than <b>2</b> with a <i>k</i><sub>d</sub> of 32.3 (±0.4) × 10<sup>–2</sup> s<sup>–1</sup>. In combination with the experimental results by EMSA and <i>T</i><sub>m</sub> analyses and our previous studies on polyamides, the rationale for the above extracellular results is possibly that the long polyamide <b>1</b> more slowly specifically recognized and bound to a long DNA sequence with nine base pairs (5′-TTCACC<b>G</b>AA-3′/5′-TTCGGTGAA-3′, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) and was more slowly dissociated from it than the short polyamide <b>2</b> that interacted with a short DNA sequence with six base pairs (5′-CACCGA-3′/5′-TCGGTG-3′, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) and was dissociated from it. By contrast, almost no interaction between Mis-polyamide (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and wt-ds-DNA was observed at lower concentrations. Although nonspecific binding was noted at higher concentrations, the mode of interaction and the precise parameters for the binding of Mis-polyamide to wt-ds-DNA could not be obtained, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S3</a>. Likewise, we could not obtain interaction kinetic parameters between three polyamides and ss-DNAs (forward oligo of ds-DNA and reverse oligo of ds-DNA), as almost no binding of all three polyamides (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) to ss-DNA was observed at experimental concentrations of 10–1000 nM. The experimental results are roughly consistent with those obtained in EMSA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), <i>T</i><sub>m</sub> value analysis (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), and CD (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Kinetic Constants of Py-Im Polyamides <b>1</b> and <b>2</b> to 5′-Biotin–ds-DNA (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) by SPR<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="×" /></col><col align="char" char="×" /></col><col align="char" char="×" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">DNA</th><th class="colsep0 rowsep0" align="center" char="×"><i>k</i><sub>a</sub> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="×"><i>k</i><sub>d</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="×"><i>K</i><sub>D</sub> (M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">polyamide <b>1</b></td><td class="colsep0 rowsep0" align="char" char="×">1.28 (±0.14)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a> × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="char" char="×">1.06 (±0.23) × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="char" char="×">8.28 (±0.56) × 10<sup>–9</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">polyamide <b>2</b></td><td class="colsep0 rowsep0" align="char" char="×">6.47 (±0.18) × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="char" char="×">32.3 (±0.4) × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="char" char="×">4.99 (±0.47) × 10<sup>–8</sup></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><i>k</i><sub>a</sub>, association rate constant (M<sup>–1</sup> s<sup>–1</sup>); <i>k</i><sub>d</sub>, dissociation rate constant (s<sup>–1</sup>); <i>K</i><sub>D</sub>, dissociation equilibrium constant (M); <i>K</i><sub>D</sub> = <i>k</i><sub>d</sub>/<i>k</i><sub>a</sub>.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Standard deviation is given in parentheses.</p></div></div></div><div class="NLM_p last">Taken together, the present results demonstrated that polyamides <b>1</b> and <b>2</b> exhibited high affinity to matched wt-ds-DNA at a polyamide/wt-ds-DNA ratio of 1:1, especially polyamide <b>1</b>. It is of note that a 2:1 binding motif of polyamides <b>1</b>/<b>2</b> to the ds-DNA was not observed unless exposed to high concentrations of DNA.</div></div><div id="sec2_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Docking Study on the Interaction of Py-Im Polyamides <b>1</b> and <b>2</b> with the Target DNA Sequence</h4><div class="NLM_p">Next, we further examined the mode and affinity of the interaction between Py-Im polyamides and the target ds-DNA, 5′-TTCACCGAA-3′/5′-TTCGGTGAA-3′ by using an <i>in silico</i> calculation based on the energy-minimized structures of the complex.<a onclick="showRef(event, 'ref27 ref41 ref38 ref42'); return false;" href="javascript:void(0);" class="ref ref27 ref41 ref38 ref42">(27,41,38,42)</a> The matched hairpin polyamide appeared to tightly interact with the DNA minor groove of the target ds-DNA in an antiparallel manner, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E. We then calculated the binding energy and LibDock score after the polyamides docked with the DNA sequence (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The <i><i>in silico</i></i> results coincided well with the previous experimental data, in which the calculated binding energy and the LibDock score of the Py-Im polyamide <b>1</b>/DNA complex were −680.61395 kcal/mol and 706.467, whereas those of the Py-Im polyamide <b>2</b>/DNA were −541.15002 kcal/mol and 505.427. In contrast, Mis-polyamide cannot strongly interact with the target ds-DNA in the DNA minor groove based on the docking results of the binding energy and the LibDock score (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S4</a>). The <i>in silico</i> calculation also suggested that there existed a strong interaction between polyamides <b>1</b> and <b>2</b> and their target DNA sequence, and the number of hydrogen bonds dominantly affected the polyamide affinity to the specific sequence of the DNA in the cell-free <i>in vitro</i> system.</div><div class="NLM_p last">In summary, the extracellular results demonstrated that the matched polyamides <b>1</b> and <b>2</b> strongly interacted with the STAT3 protein binding site in the promoter region of PD-L1 and could competitively inhibit the binding of the STAT3 protein to the DNA motif. Furthermore, at the extracellular level, polyamide <b>1</b> exhibits relatively stronger binding ability than <b>2</b> in the cell-free <i>in vitro</i> system.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Py-Im Polyamides Suppress PD-L1 Expression in Tumor Cells</h3><div class="NLM_p">In general, Py-Im polyamides not only show high specificity to certain DNA sequences but also have high cell permeability; that is, they can penetrate the cell membrane into the nucleus. Thus they may specifically regulate the expression of target genes through binding to the promoter of the genes. It was reported that IFN-γ could induce PD-L1 expression at a high level in HCC827 and MDA-MB-231 cells.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> Thus we first examined the effect of the newly synthesized Py-Im polyamides on the PD-L1 mRNA expression in the presence of IFN-γ in both cancer cells. Both Py-Im polyamides <b>1</b> and <b>2</b> concentration-dependently inhibited the PD-L1 mRNA expression in the two tumor cell lines, in which the inhibitory effect of the Py-Im polyamide <b>2</b> was more pronounced (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). Similarly, the PD-L1 protein expression was efficiently suppressed by the addition of <b>1</b> or <b>2</b> in a concentration-dependent manner, with polyamide <b>2</b> being more effective, as depicted (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D). By contrast, Mis-polyamide, which failed to recognize 5′-TTCACCGAA-3′/5′-TTCGGTGAA-3′ (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), showed essentially no effects on the expression of PD-L1 protein, consistent with the results of the cell-free <i>in vitro</i> system (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figures S1C, S2A, and S3</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Noticeably, polyamide <b>2</b> inhibited the PD-L1 expression at both the mRNA and protein levels more efficiently than <b>1</b> in tumor cells, whereas polyamide <b>1</b> interacted with the SBS of the PD-L1 promoter more potently than <b>2</b> in cell-free <i>in vitro</i> systems. This is possibly because polyamide <b>2</b> is shorter than <b>1</b>. It was reported that the cell permeability of polyamides was influenced by their size, and the shorter polyamides generally had higher cell permeability than the longer ones.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of Py-Im polyamides <b>1</b> and <b>2</b> on the PD-L1 gene expression in HCC827 and MDA-MB-231 cells. Level of PD-L1 mRNA expression in (A) HCC827 and (B) MDA-MB-231 cells. The tumor cells were treated with IFN-γ alone (50 ng/mL) or IFN-γ (50 ng/mL) + Py-Im polyamides (0, 1.25, 2.5, 5, and 10 μM) for 36 h, and the mRNA was analyzed by q-PCR. Western blot analysis of PD-L1 protein in (C) HCC827 and (D) MDA-MB-231 cells treated with the indicated polyamides. HCC827 and MDA-MB-231 cells were treated with IFN-γ alone (50 ng/mL) or IFN-γ (50 ng/mL) + Py-Im polyamide <b>1</b> or <b>2</b> (0, 1.25, 2.5, 5, and 10 μM). Data show the mean ± SD and are the result of three independent experiments (* <i>p</i> < 0.05, ** <i>p</i> < 0.01, *** <i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, we demonstrated that polyamide <b>2</b> could permeate into the MDA-MB-231 cell nucleus to interact with target DNA like a nucleic acid dye, Hoechst 33342, by the nucleus location experiment (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p last">Collectively, the newly synthesized Py-Im polyamides <b>1</b> and <b>2</b> can inhibit the expression of PD-L1 gene in tumor cells at both transcriptional and translational levels.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Py-Im Polyamides Promote T-Cell-Mediated Tumor Cell Killing and IL-2 Production</h3><div class="NLM_p">To examine the ability of the newly synthesized polyamides to kill tumor cells in the presence of T cells, we carried out assays on IL-2 secretion levels and T-cell-mediated tumor killing in coculture systems of tumor cells with Jurkat T cells. Several lines of evidence have shown that binding of PD-L1 expressed on tumor cells to PD-1 expressed on T cells reduces the T-cell activation and IL-2 production.<a onclick="showRef(event, 'ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48">(46−48)</a> For example, the coculture of melanoma cells expressing PD-L1 by IFN-γ with Jurkat T cells results in the impaired production of IL-2 in T cells.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In addition, the coculture of breast cancer cells, lung cancer cells, and the like with Jurkat T cells has the same phenomenon.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> Because the newly synthesized Py-Im polyamides quite reduced the PD-L1 expression in the two tumor cells, as previously described (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), we further explored whether the down-regulation of PD-L1 could influence the immune responses in the coculture system of T cells (Jurkat cells) and tumor cells, like HCC827 and MDA-MB-231 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). The amounts of IL-2, which was secreted from T cells into the culture supernatants in response to HCC827 cells treated with the Py-Im polyamides <b>1</b> and <b>2</b>, were increased by 28.20 (31.41%) and 51.17 pg/mL (57.00%), respectively, compared with those without compounds (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Similarly, the amounts in response to MDA-MB-231 were increased by 48.06 (33.00%) and 87.89 pg/mL (60.35%), respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The results demonstrated that addition of both polyamides <b>1</b> and <b>2</b> can promote the secretion of IL-2 in such a coculture system and that polyamide <b>2</b> was more effective in stimulating IL-2 secretion than <b>1</b> but less effective than a PD-1/PD-L1 inhibitor 2 (BMS202, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S6</a>) as a positive control, which enhanced the secretion of IL-2 by almost 100% from T cells in response to both tumor cells treated with the compound, whereas Mis-polyamide almost failed to enhance the secretion of IL-2 in both culture systems.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Promotion of T-cell-mediated tumor killing and IL-2 production by the Py-Im polyamides in coculture systems. IL-2 secretion levels in (A) Jurkat/HCC827 and (B) Jurkat/MDA-MB-231 coculture systems treated with 8 μM of the polyamide (<b>1</b> or <b>2</b> or Mis-polyamide) and PD-1/PD-L1 inhibitor 2 (BMS202 as a positive control; its structure is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S6</a>) are shown. Promotion of T-cell-mediated tumor killing in (C) Jurkat/HCC827 and (D) Jurkat/MDA-MB-231 coculture systems treated with 8 μM of <b>1</b> or <b>2</b> or Mis-polyamide and PD-1/PD-L1 inhibitor 2. Data show the mean ± SD and are the results of three independent experiments (* <i>p</i> < 0.05, ** <i>p</i> < 0.01, *** <i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we analyzed the effect of the Py-Im polyamides <b>1</b> and <b>2</b> on T-cell-mediated cytotoxicity against tumor cells. Tumor cell lines HCC827 and MDA-MB-231 were treated with <b>1</b> or <b>2</b> and challenged by T cells. Then, the viability of the tumor cells was determined by the standard CCK-8 assay. When HCC827 cells treated with <b>1</b> and <b>2</b> were incubated with T cells, 48.00 and 70.00% of tumor cells underwent apoptosis, respectively, relative to a control without any drug, and the decrease in the cell viability for HCC827 cells treated with BMS202 as a positive control was close to 70.87%. Similarly, 50.73 and 82.11% of <b>1-</b> and <b>2</b>-treated MDA-MB-231 cells were killed in the presence of T cells, respectively, relative to a control without any drugs, and both percentages were lower than that (87.58%) of BMS202-treated MDA-MB-231 cells. Again, polyamide <b>2</b> was more effective in the promotion of T-cell killing than <b>1</b>, but both polyamides were less effective than BMS202. Likewise, we investigated the influence of the studied polyamides on the cell viability of Jurkat cells alone, but almost no cytotoxicity was observed (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S7</a>). In addition, the Mis-polyamide had no effect on tested cells.</div><div class="NLM_p last">Taken together, Py-Im polyamides <b>1</b> and <b>2</b> could modulate the PD-1 immune checkpoints and promote the secretion of IL-2, leading to the death of tumor cells.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Py-Im Polyamides Induce Tumor Cell Proliferation Inhibition/Cytotoxicity via Another AKT/Caspapse-3 Pathway and Inhibit Cell Migration</h3><div class="NLM_p">In the above-used coculture systems, the Py-Im polyamides <b>1</b> and <b>2</b> possessed a killing effect on tumor cells via T cells. Subsequently, we explored whether both <b>1</b> and <b>2</b> could exhibit direct cytotoxicity against cancer cells via downregulating the PD-L1 expression in the absence of T cells. Direct toxic effects of the compounds on HCC827 and MDA-MB-231 cells were examined using the standard CCK-8 assay. The cytotoxicity of <b>1</b> and <b>2</b> against tumor cells increased with the increase in compound concentration, whereas the Mis-polyamide had almost no effect on the direct toxicity (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). It is noteworthy that the matched and mismatched Py-Im polyamides were essentially nontoxic to normal MRC-5 cells, even at the highest concentration of 32 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). The IC<sub>50</sub> values of polyamide <b>2</b> were 11.84 and 9.05 μM for HCC827 and MDA-MB-231 cells (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), respectively, which were more than two times lower than those of <b>1</b>, suggesting that polyamide <b>2</b> possessed stronger anticancer activity than <b>1</b>. This result is in accord with the influence of both polyamides <b>1</b> and <b>2</b> on PD-L1 expression in cancer cells on the whole (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cytotoxicity and migration inhibition of the Py-Im polyamides to cancer cells and the exploration of the mechanism for non-immune-mediated proliferation inhibition/apoptosis of cancer cells by Py-Im polyamides. Effect of the Py-Im polyamides on the viability of (A) HCC827, (B) MDA-MB-231, and (C) MRC-5 in the absence of immune T cells. The cell viability was determined using the CCK-8 assay. Cisplatin was used as a positive control. 1% DMSO was the reference solution, which contained 1% DMSO in RPMI 1640. (D) Effect of Py-Im polyamides <b>1</b> and <b>2</b> on the migration of tumor cells. Wound healing assays were employed to determine the inhibition of the tumor cell migration. 48 h after scratches, the migration of the tumor cells was observed. Western blot analysis of Akt, p-Akt (Ser473), and caspase-3 proteins in (E) HCC827 and (F) MDA-MB-231 cells treated with the indicated compounds. HCC827 and MDA-MB-231 cells were treated with the Py-Im polyamide <b>2</b> (10 μM) or Mis-polyamide (10 μM). Data are the mean ± SD and are the results of three independent experiments (* <i>p</i> < 0.05, ** <i>p</i> < 0.01, *** <i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. IC<sub>50</sub> (μM) of Py-Im Polyamides for HCC827 and MDA-MB-231 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell lines</th><th class="colsep0 rowsep0" align="center" char="±">polyamide <b>1</b></th><th class="colsep0 rowsep0" align="center" char="±">polyamide <b>2</b></th><th class="colsep0 rowsep0" align="center">Mis-polyamide</th><th class="colsep0 rowsep0" align="center" char="±">cisalpine</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCC827</td><td class="colsep0 rowsep0" align="char" char="±">23.23 ± 0.98</td><td class="colsep0 rowsep0" align="char" char="±">11.84 ± 0.53</td><td class="colsep0 rowsep0" align="left">≫200</td><td class="colsep0 rowsep0" align="char" char="±">8.41 ± 0.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-231</td><td class="colsep0 rowsep0" align="char" char="±">21.66 ± 0.87</td><td class="colsep0 rowsep0" align="char" char="±">9.05 ± 0.48</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="char" char="±">7.92 ± 0.52</td></tr></tbody></table></div></div><div class="NLM_p">More importantly, by comparing the results in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B with those obtained in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D, it was found that polyamides <b>1</b> and <b>2</b> at a concentration of 8 μM could increase the proliferation inhibition/apoptosis of HCC827 cells by 23.40 and 26.14%, respectively, and those of MDA-MB-231 cells by 27.76 and 30.84%, respectively, in the presence of Jurkat T cells, which was attributed to the immune antitumor effect. This suggests that the proliferation inhibition/apoptosis of cancer cells by <b>1</b> and <b>2</b> was achieved by both a nonimmunosuppressive pathway and an immunosuppressive pathway.</div><div class="NLM_p">Then, the effect of <b>1</b> and <b>2</b> on the migration of HCC827 and MDA-MB-231 cells was examined through a wound healing assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S8</a>). The tumor cell migration was inhibited in a polyamide concentration-dependent manner. A 50% inhibition rate was observed at 0.5 to 1.0 μM, and a 70% or greater inhibition rate was observed at a concentration of 2 μM. In particular, polyamide <b>2</b> showed an inhibition rate of 86.33% at 2 μM in MDA-MB-231 cells.</div><div class="NLM_p">It was reported that PD-L1 itself could modulate the proliferation of tumor cells in the absence of PD-1, which is related to the Akt/m-TOR pathway and autophagy.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> In addition, caspase-3 protein is closely linked to apoptosis and participates in its regulation.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> On the basis of such previous important results and analysis, we attempted to elucidate the mechanism by which the Py-Im polyamides could induce tumor cell apoptosis or inhibit proliferation by downregulating the expression of PD-L1 gene independently of the immune signaling. In our study, polyamide <b>2</b> was applied because it appeared to interact with the target DNA sequence more efficiently than <b>1</b>. Polyamide <b>2</b> significantly inhibited the expression of caspase-3 (pro-caspase-3) and the phosphorylation of Akt (P-Akt), which were closely associated with the expression of PD-L1 protein in cancer cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F).<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> In contrast, the Mis-polyamide showed no explicit effects on the signaling.</div><div class="NLM_p last">Collectively, these results demonstrated that the Py-Im polyamides inhibited the growth of tumor cells by suppressing the Akt/caspase-3 signaling as a new pathway independently of the modulation of PD-1 immune checkpoints.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61611" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61611" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">PD-L1 belongs to the B7 family of transmembrane proteins and negatively modulates the function of T cells, thereby attenuating the cytotoxicity of T cells against tumor cells and ultimately mediating tumor immune escape.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> PD-L1 is highly expressed in various tumor cells and thus is a target for the early diagnosis and immunotherapy of cancer.<a onclick="showRef(event, 'ref53 ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55 ref56">(53−56)</a> In this study, we demonstrated that the newly synthesized Py-Im polyamides predominately bound to the SBS of the PD-L1 promoter, thereby inhibiting the PD-L1 expression at the transcriptional and translational levels. In the coculture system, the Py-Im polyamides were capable of enhancing T-cell-mediated tumor killing and IL-2 production (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). In the tumor microenvironment, tumor cells show adaptive immune resistance by upregulating the PD-L1 expression inducible by IFN-γ.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> It was previously reported that the PD-1/PD-L1 interaction inhibited the secretion of IL-2 and IFN-γ.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> We thus hypothesized that the downregulation of the PD-L1 gene expression by the Py-Im polyamides attenuated the interaction between PD-1 and PD-L1, promoted the recovery of T-cell functions, and consequently elicited antitumor effects. Further direct evidence should, however, be required for the clearer elucidation of the PD-L1/PD-1 immune-mediated apoptosis mechanism by polyamides.</div><div class="NLM_p">It is intriguing that the Py-Im polyamides <b>1</b> and <b>2</b> exhibited a direct toxic effect on both tumor cells and inhibited cell proliferation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B) in the absence T cells. This finding prompted us to examine whether there exists another pathway to cause cancer cell apoptosis, as recent studies have shown that PD-L1 exerted intrinsic effects on tumors, in which PD-L1 could enhance the cell proliferation and survival by inhibiting autophagy and mTOR activation in the absence of PD-1.</div><div class="NLM_p">We thus examined the dynamics of p-Akt- (Ser473) and caspase-3-related proteins. Both Py-Im polyamides <b>1</b> and <b>2</b> could significantly inhibit the phosphorylation of Akt (Ser473) and reduce the expression of caspase-3, resulting in apoptosis. This implies that the Py-Im polyamides <b>1</b> and <b>2</b> inhibit the proliferation of cancer cells and promote apoptosis in the absence of T cells, possibly through the AKT signaling/caspase-3 pathway. But further studies should be required to clarify the mechanism by which the newly synthesized Py-Im polyamides enhance antitumor effects.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a></div><div class="NLM_p">It has been shown that STAT3 could regulate the transcription of PD-L1 by directly binding to its promoter.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> The PD-L1 expression is also known to be elevated with an oncogene ALK mutation, whereas the knockdown of STAT3 abrogates the expression.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> In addition, LMP1 positively regulates PD-L1 expression by promoting the phosphorylation of STAT3. On the contrary, the inhibition of STAT3 phosphorylation by the JAK3 inhibitor CP-690550 suppresses the LMP1-induced PD-L1 expression.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Blocking the STAT3 signaling pathway in tumor cells could therefore effectively inhibit the tumor cell proliferation, migration, and angiogenesis and promote apoptosis. To bind to the STAT3-binding site in the PD-L1 promoter, the designed compounds in this study could competitively inhibit the interaction of STAT3 with the DNA sequence. However, an investigation of the effect of the compounds on the phosphorylation of STAT3 and the downstream signaling leading to cell proliferation and migration is required.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></div><div class="NLM_p">The epithelial–mesenchymal transition (EMT) is a pivotal step in the early stages of tumor metastasis, and it accelerates the malignant transformation of tumors.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> It was demonstrated that PD-L1 is highly expressed in various malignant tumor tissues such as lung cancer and breast cancer, and its expression level is closely related to the clinical stage and prognosis of patients.<a onclick="showRef(event, 'ref53 ref62'); return false;" href="javascript:void(0);" class="ref ref53 ref62">(53,62)</a> A high level of PD-L1 expression promotes the EMT in normal skin epithelial cells and accelerates carcinogenesis.<a onclick="showRef(event, 'ref12 ref63'); return false;" href="javascript:void(0);" class="ref ref12 ref63">(12,63)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D, the matched polyamides <b>1</b> and <b>2</b> significantly inhibited tumor cell migration, which may be related to inhibition of the EMT.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a></div><div class="NLM_p">In summary, our study showed that small-molecule Py-Im polyamides could predominately bind to the SBS in the PD-L1 promoter, resulting in the downregulation of transcription and translation of PD-L1. Consequently, the compounds could regulate the PD-1/PD-L1 immune checkpoints, leading to the augmentation of anticancer effects. More strikingly, such small-molecule polyamides could also have a direct toxic effect on cancer cells, retard tumor cell migration, inhibit the expression of Akt/caspase-3, and promote apoptosis in the absence of T cells. The Py-Im polyamides therefore have the potential to be a prototype of therapeutics as PD-L1 inhibitors. A scheme for the proposed molecular mechanism by which small-molecule Py-Im polyamides specifically downregulate the expression of PD-L1 gene and promote antitumor effects via immune and nonimmune pathways is depicted (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Schematic of the molecular mechanism of Py-Im polyamides target-downregulating the expression of the PD-L1 gene and promoting tumor cell apoptosis via immune and nonimmune pathways.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General</h4><div class="NLM_p"><sup>1</sup>H NMR spectra were measured on a Bruker AVANCE III NMR spectrometer, in which chemical shifts (δ) were recorded in parts per million relative to residual solvent and coupling constants were measured in hertz (spin multiplicity: singlet (s), doublet (d), triplet (t), quartet (q), quintet (qu), multiplet (m), and broad (br)). UV spectra were obtained in HBS-EP buffer using a Shimadzu UV-2550 spectrophotometer. Mass spectra were measured using matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) mass spectrometry with a BioTOF II apparatus from Bruker Daltonics in a positive ionization mode, except where otherwise stated. High-performance liquid chromatography (HPLC) analysis was conducted on a Shimadzu LC-20AT HPLC system equipped with a ChemcoPak, Chemcoband, 5-ODS-H, 150 × 10 mm reversed-phase column using 0.1% TFA in water with acetonitrile as an eluent at a flow rate of 1.0 mL/min with a linear gradient elution of 0–100% acetonitrile in 30 min with detection at 254 nm, except where otherwise stated. Preparatory reversed-phase HPLC was conducted using a Shimadzu LC-6AD HPLC system equipped with a YMC-Pack R&D ODS, D-ODS-5-ST, S-5 μm, 150 × 20 mm reversed-phase column using 0.1% TFA in water with acetonitrile as an eluent at a flow rate of 3.0 mL/min under appropriate gradient elution conditions for the detection of optical density at 254 or 350 nm. Collected fractions were analyzed by electrospray ionization–time-of-flight mass spectrometry (ESI-TOFMass). A Millipore Milli-Q water purification system was used to prepare pure water with a conductivity at 18 MΩ, and all buffers were filtered through a 0.22 μm membrane and degassed before the SPR assay using a Biocore system.</div><div class="NLM_p last">Clear resin (0.52 mequiv/g) and <i>N</i>-{(dimethylamino)-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]-pyridino-1-ylmethylene}-<i>N</i>-methylmethanaminium hexafluorophosphate (HATU) were from Peptides International. 4-{[(9<i>H</i>-Fluoren-9-ylmethoxy)carbonyl]amino}-1-methyl-1<i>H</i>-imidazole-2-carboxylic acid (<b>4</b>, FmocPyCOOH, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Scheme S1</a>), 4-{[(9<i>H</i>-fluoren-9-ylmethoxy)carbonyl]amino}-1-methyl-1<i>H</i>-pyrrole-2-carboxylic acid (<b>5</b>, FmocImCOOH, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Scheme S1</a>), <i>N</i>,<i>N</i>-dimethylformamide (DMF), dimethyl sulfoxide (DMSO), piperidine, and pyridine were obtained from Wako, and <i>N</i>-[(9<i>H</i>-fluoren-9-ylmethoxy)carbonyl]-β-alanine (<b>6</b>, Fmoc-β-Ala-OH, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Scheme S1</a>) and 4-[(9<i>H</i>-fluoren-9-ylmethoxy)carbonyl]aminobutanoic acid (<b>7</b>, Fmoc-γ-Abu-COOH, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Scheme S1</a>) were from Novabiochem. <i>N</i>,<i>N</i>-Diisopropylethylamine (DIEA), acetic anhydride, and <i>N</i>,<i>N</i>-dimethyl-1,3-propanediamine were from Sigma-Aldrich. 1-Hydroxybenzotriazole (HOBt), dicyclohexylcarbodiimide (DCC), dichloromethane (DCM), trifluoroacetic acid (TFA), and acetic acid (HOAc) were from Acros Organics. All other materials and reagents were of highest quality and were purchased from standard suppliers.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Synthesis</h4><div class="NLM_p">Polyamides were semiautomatically synthesized using a Pioneer peptide synthesizer (Applied Biosystems, USA). A previously established protocol with a computer-assisted operation system was employed to synthesize the compounds on a 0.10 mmol scale (200 mg of clear resin, 0.52 mequiv/g) based on semimanual Fmoc chemistry.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> In brief, all polyamide solid-phase syntheses were performed for each cycle as follows: deblocking with 20% piperidine/DMF for 5 min, activating with 0.5 M HATU/DMF and 1.0 M DIEA/DMF for 2 min, coupling for 60 min, capping with 5% acetic anhydride/5% pyridine/DMF for 5 min for the minimization of side products, and washing with methanol. Coupling was conducted for 90 min in the case of coupling of any monomer to imidazole and coupling of imidazole itself; all other coupling reactions were carried out with single couple cycles. To swell the Fmoc-β-Ala-clear-acid resin (200 mg, 0.10 mmol), 3 mL of DMF was added to a 15 mL plastic centrifuge tube containing the resin. After being swollen for 30 min, the resin was loaded onto a reaction column. The synthetic procedure was regulated by a computer-aided program. Before and after solution transfers, all lines were emptied with argon.</div><div class="NLM_p">When the synthesis of the peptides was completed on the synthesizer, the resin was washed with methanol/DCM (1/1 v/v) (3 × 1 mL) to remove the unreacted reagents and dried under vacuum. Then, the resin was mixed with 5 mL of (dimethylamino)propylamine (Dp), which was incubated with continuous stirring at 55 °C overnight. To remove the resin, the reaction mixture was filtrated through a pad of celite, and the residual product was recovered by washing the resin with methanol/DCM (1/1 v/v). After the pH of the filtrate was adjusted to 4 to 5 with 10% TFA/DCM and the solvent was removed under vacuum, ∼0.3 mL of DMF was added to the residue. The polyamide samples were analyzed by analytical HPLC under monitoring at 254 nm and further purified by using preparatory HPLC with detection at 254 or 350 nm using the following eluent and conditions: 0.1% TFA/MeCN, linear gradient 0–100%, 0–20 min. After fractions were examined by <sup>1</sup>H NMR and TOF-Mass, the target products were lyophilized to yield the polyamides <b>1</b>–<b>3</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Scheme S1</a>) as white or light-yellow powders.</div><div class="NLM_p last">Polyamides <b>1</b>–<b>3</b> were obtained in a recovery of at least 8.1% and at most 16.1% by a single HPLC purification.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Dp-β-PyPy-ImPy-Py-β-PyPy-Py-γ-PyPy-Im-β-ImIm-Py-PyPy-Ac (<b>1</b>)</h4><div class="NLM_p last">Yield: 8.1% as a light-yellow powder. By analytical HPLC, <i>t</i><sub>R</sub> = 9.98 min, purity is ≥98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.50 (s, 1H, aromatic N<i>H</i>), 10.46 (s, 2H, aromatic N<i>H</i>), 10.42 (s, 1H, aromatic N<i>H</i>), 10.33 (s, 2H, aromatic N<i>H</i>), 10.26 (s, 1H, aromatic N<i>H</i>), 10.08 (s, 2H, aromatic N<i>H</i>), 9.86–9.78 (m, 3H, aromatic N<i>H</i>), 9.73 (s, 2H, aromatic N<i>H</i>), 9.62 (s, 2H, aromatic N<i>H</i>), 8.01–7.87 (m, 3H, aliphatic N<i>H</i>), 7.82–7.75 (m, 2H, aliphatic N<i>H</i>), 7.46 (s, 1H, aromatic C<i>H</i>), 7.43–7.31 (m, 4H, aromatic CH), 7.27–7.23 (m, 2H, aromatic C<i>H</i>), 7.20 (s, 1H, aromatic C<i>H</i>), 7.14–6.98 (m, 4H, aromatic C<i>H</i>), 6.96–6.89 (m, 6H, aromatic C<i>H</i>), 6.83–6.76 (m, 2H, aromatic C<i>H</i>), 6.73–6.67 (m, 4H, aromatic C<i>H</i>), 6.62–6.57 (m, 4H, aromatic C<i>H</i>), 3.98 (s, 3H, NC<i>H</i><sub>3</sub>), 3.95 (s, 3H, NC<i>H</i><sub>3</sub>), 3.92 (s, 3H, 2 × NC<i>H</i><sub>3</sub>), 3.88 (s, 3H, NC<i>H</i><sub>3</sub>), 3.85 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.81 (s, 3H, NC<i>H</i><sub>3</sub>), 3.77 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.74 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.70 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.65 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.57–3.50 (m, 4H, 2 × NHC<i>H</i><sub>2</sub>), 3.44 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.36–3.28 (m, 4H, 2 × NHC<i>H</i><sub>2</sub>), 2.65 (m, 2H, NMe<sub>2</sub>C<i>H</i><sub>2</sub>), 2.34 (t, <sup>3</sup>J<sub>H,H</sub> = 7.2 Hz, 2H, COC<i>H</i><sub>2</sub>), 2.28 (t, <sup>3</sup>J<sub>H,H</sub> = 6.8 Hz, 2H, COCH<sub>2</sub>), 2.22–2.15 (m, 4H, 2 × COC<i>H</i><sub>2</sub>), 2.04 (s, 6H, N(C<i>H</i><sub>3</sub>)<sub>2</sub>), 1.97 (s, 1H, COC<i>H</i><sub>3</sub>), 1.77 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 1.52 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>). TOF-Mass: <i>m</i>/<i>z</i> calcd for C<sub>112</sub>H<sub>131</sub>N<sub>42</sub>O<sub>21</sub>[M + H]<sup>+</sup>: 2400.0474; found: 2400.8130 [M + H]<sup>+</sup>, 2422.8110 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Dp-β-PyIm-Py-β-PyPy-γ-Im-β-ImIm-PyPy-Ac (<b>2</b>)</h4><div class="NLM_p last">Yield: 13.4% as an ashen powder. By analytical HPLC, <i>t</i><sub>R</sub> = 10.94 min, purity is ≥98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.43 (s, 1H, aromatic N<i>H</i>), 10.37 (s, 1H, aromatic N<i>H</i>), 10.30 (s, 2H, aromatic N<i>H</i>), 10.13 (s, 1H, aromatic N<i>H</i>), 9.97 (s, 2H, aromatic N<i>H</i>), 9.85 (s, 2H, aromatic N<i>H</i>), 9.71 (s, 1H, aromatic N<i>H</i>), 8.26–8.00 (m, 3H, aliphatic N<i>H</i>), 7.85–7.76 (m, 2H, aliphatic N<i>H</i>), 7.61 (s, 1H, aromatic C<i>H</i>), 7.50 (s, 1H, aromatic C<i>H</i>), 7.42 (s, 1H, aromatic C<i>H</i>), 7.30 (s, 1H, aromatic C<i>H</i>), 7.21–6.97 (m, 6H, aromatic C<i>H</i>), 6.89–6.82 (m, 2H, aromatic C<i>H</i>), 6.76–6.63 (m, 4H, aromatic C<i>H</i>), 3.99 (s, 3H, NC<i>H</i><sub>3</sub>), 3.94 (s, 3H, NC<i>H</i><sub>3</sub>), 3.90 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.85 (s, 3H, NCH<sub>3</sub>), 3.80 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.76 (s, 3H, NC<i>H</i><sub>3</sub>), 3.72 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.60 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.54 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.48 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.39–3.27 (m, 4H, 2 × NHC<i>H</i><sub>2</sub>), 2.61 (m, 2H, NMe<sub>2</sub>C<i>H</i><sub>2</sub>), 2.42 (d, <sup>3</sup>J<sub>H,H</sub> = 7.4 Hz, 2H, COC<i>H</i><sub>2</sub>), 2.31 (t, <sup>3</sup>J<sub>H,H</sub> = 6.8 Hz, 2H, COC<i>H</i><sub>2</sub>), 2.20–2.10 (m, 4H, 2 × COC<i>H</i><sub>2</sub>), 2.02 (s, 6H, N(C<i>H</i><sub>3</sub>)<sub>2</sub>), 1.97 (s, 1H, COC<i>H</i><sub>3</sub>), 1.77 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 1.53 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>). TOF-Mass: <i>m</i>/<i>z</i> calcd for C<sub>76</sub>H<sub>95</sub>N<sub>30</sub>O<sub>15</sub>[M + H]<sup>+</sup>: 1667.7593; found: 1667.5201 [M + H]<sup>+</sup>, 1689.4670 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Dp-β-Py-ImIm-β-PyPy-γ-Im-β-ImPy-PyPy-Ac (<b>3</b>)</h4><div class="NLM_p last">Yield: 16.1% as a light-yellow powder. By analytical HPLC, <i>t</i><sub>R</sub> = 11.97 min, purity is ≥98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.41 (s, 1H, aromatic N<i>H</i>), 10.36 (s, 1H, aromatic N<i>H</i>), 10.28 (s, 1H, aromatic N<i>H</i>), 10.14 (s, 2H, aromatic N<i>H</i>), 9.95 (s, 1H, aromatic N<i>H</i>), 9.85 (s, 2H, aromatic N<i>H</i>), 9.73 (s, 2H, aromatic N<i>H</i>), 8.28–8.03 (m, 3H, aliphatic N<i>H</i>), 7.87–7.74 (m, 2H, aliphatic N<i>H</i>), 7.63 (s, 1H, aromatic C<i>H</i>), 7.53 (s, 1H, aromatic C<i>H</i>), 7.42 (s, 1H, aromatic C<i>H</i>), 7.30 (s, 1H, aromatic C<i>H</i>), 7.23–7.01 (m, 6H, aromatic C<i>H</i>), 6.88–6.82 (m, 2H, aromatic C<i>H</i>), 6.77–6.63 (m, 4H, aromatic C<i>H</i>), 3.98 (s, 3H, NC<i>H</i><sub>3</sub>), 3.94 (s, 3H, NC<i>H</i><sub>3</sub>), 3.89 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.84 (s, 3H, NC<i>H</i><sub>3</sub>), 3.80 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.75 (s, 3H, NC<i>H</i><sub>3</sub>), 3.70 (s, 6H, 2 × NC<i>H</i><sub>3</sub>), 3.61 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.52 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.48 (m, 2H, NHC<i>H</i><sub>2</sub>), 3.40–3.25 (m, 4H, 2 × NHC<i>H</i><sub>2</sub>), 2.59 (m, 2H, NMe<sub>2</sub>C<i>H</i><sub>2</sub>), 2.40 (d, <sup>3</sup>J<sub>H,H</sub> = 7.4 Hz, 2H, COC<i>H</i><sub>2</sub>), 2.30 (t, <sup>3</sup>J<sub>H,H</sub> = 6.8 Hz, 2H, COC<i>H</i><sub>2</sub>), 2.22–2.11 (m, 4H, 2 × COC<i>H</i><sub>2</sub>), 2.02 (s, 6H, N(C<i>H</i><sub>3</sub>)<sub>2</sub>), 1.98 (s, 1H, COC<i>H</i><sub>3</sub>), 1.76 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 1.53 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>). TOF-Mass: <i>m</i>/<i>z</i> calcd for C<sub>76</sub>H<sub>95</sub>N<sub>30</sub>O<sub>15</sub>[M + H]<sup>+</sup>: 1667.7593; found: 1667.6080 [M + H]<sup>+</sup>, 1689.8750 [M + Na]<sup>+</sup>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> In Vitro Pharmacological Assays</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> DNA Oligomers, Cell Lines, and Inhibitors</h4><div class="NLM_p last">Oligomers/primers used in this study were obtained from Sangon (Shanghai, China). The DNA preparations were purified using HPLC, and the quality and identity were confirmed by HPLC–CE (capillary electrophoresis) and TOF-Mass, with the purities being ≥95%. A human non-small-cell lung cancer cell line (HCC827), a human breast cancer cell line (MDA-MB-231), a human fetal lung fibroblast cell line (MRC-5), and a human T-cell lymphoma (Jurkat) were obtained from the Cell Bank, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, China. All cell lines were cultured in RPMI-1640 or Dulbecco’s modified Eagle’s medium (DMEM, Thermo Scientific) supplemented with 10% fetal calf serum (FCS, Thermo Scientific) and 100 U/mL penicillin and 100 μg/mL streptomycin (Thermo Scientific). All cells were incubated in 12-well plates at a cell density of 1 × 10<sup>5</sup> cells/well at 37 °C with 5% CO<sub>2</sub>. For the cytokine-mediated expression of PD-L1, cells were pretreated with 50 ng/mL IFN-γ for 24 h at 37 °C with 5% CO<sub>2</sub>. Cisplatin with a purity of >99.0% was purchased from Sigma-Aldrich. PD-1/PD-L1 inhibitor <b>2</b> was obtained from Target Molecule (TargetMol).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Electrophoretic Mobility Shift Assay</h4><div class="NLM_p last">Native polyacrylamide gel (20%) electrophoresis in 1× Tris-base–boric-acid–EDTA (TBE) buffer was used for the EMSA. After the samples were loaded on the gels, the electrophoresis was run at 150 V for 4.5 h at 4 °C in circulating cooling water. Then, the gels were removed from the glass plates and immersed in 1× SYBR Gold (1:1000 dilution with 1× TE buffer) for 30 min and rinsed with ultrapure water. Finally, the gel images were photographed under a Bio-Rad gel imaging analyzer.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> UV Melting Assay</h4><div class="NLM_p last">A UV melting assay was performed on a UV-2550 spectrophotometer (Shimadzu) equipped with quartz cells with a path length of 10 mm and a thermoelectrically controlled cell holder in the presence or absence of newly designed and synthesized polyamides to measure the <i>T</i><sub>m</sub> values of the DNAs according to our previous work.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a> The final DNA concentration was 1 μM (double-stranded units) in the sample solution containing 100 mM Tris-HCl buffer containing 50 mM NaCl and 1 mM EDTA. As a reference, the buffer solution without DNA was subjected to the same analysis. The DNA preparations were denatured at 95 °C for 10 min, cooled to room temperature, and stored at 4 °C overnight. The samples were heated from 20 to 95 °C at a rate of 1 °C/min, and the absorbance at different ratios was measured.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Circular Dichroism</h4><div class="NLM_p last">For the CD assay, a Jasco J-815 CD spectropolarimeter equipped with a quartz cuvette with a path length of 5 mm was employed. CD spectra were measured at a scanning rate of 100 nm/min at a bandwidth of 5.0 nm, and the results of three independent experiments were averaged. Means and standard deviations were calculated. A sample of 2 μM ds-DNA or 3 μM ss-DNA was dissolved in buffer (pH 7.4) containing 10 mM sodium cacodylate and 100 mM NaCl and incubated for 24 h after annealing at 95 °C for further assays. The polyamides were gradually dropped into target DNA samples at room temperature, which had been degassed before use, at the indicated ratio of compound to DNA (r) in buffer (pH 7.4) containing 10 mM sodium cacodylate, 100 mM NaCl, and 0.5% DMSO. After 15 min to achieve equilibrium, CD spectra from 225 to 400 nm were recorded.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Surface Plasmon Resonance Assay</h4><div class="NLM_p last">A Biacore X100 system (Sweden) equipped with an SA sensor chip was used to measure the SPR. Tris-based buffer (10 mM Tris–HCl, pH 7.4, containing 10 mM KCl, 0.1 mM EDTA, 0.005% surfactant P20, and 0.5% DMSO (v/v)) was used as the flow solution, and the measurement was conducted at 25 °C.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> The processing of data was carried out with BIAevalution software. Flow solution containing sodium chloride (1 M) and sodium hydroxide (50 mM) was used to remove any unbound streptavidin from the surface of the sensor chip. After preconditioning of the sensor chip, its surface was further washed for 10 min to remove any trace of NaOH until the baseline became flat. Next, two annealed DNA oligos, 5′-biotin–TTCCGCCGATTTCACCGAAGGTCAGGAA-TATA-TTCCTGACCTTCGGTGAAATCGGCGGAA-3′ (5′-biotin–ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) were noncovalently immobilized on an SA sensor chip through a biotin–streptavidin interaction. After immobilization, the compounds were injected at a flow rate of 20 μL/min, and the equilibrium responses were determined relative to the baseline to obtain experimental data. Finally, the dissociation equilibrium constant (<i>K</i><sub>D</sub>, M), the association rate constant (<i>k</i><sub>a</sub>, M<sup>–1</sup> s<sup>–1</sup>), the dissociation rate constant (<i>k</i><sub>d</sub>, s<sup>–1</sup>), and the stoichiometry were calculated by fitting the sensorgram data to a theoretical model, a global 1:1 interaction model with a mass-transfer effect, using the BIAevaluation 4.1 program (Bia-core/GE Healthcare). Then, the affinity and specificity of binding for the polyamide compounds against the 28nt double-stranded PD-L1 DNA (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and single-stranded PD-L1 DNA were calculated.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Molecular Modeling Studies</h4><div class="NLM_p last">Discovery Studio 3.5 (Accelrys, San Diego, CA) software was used for molecular modeling. A double-stranded B-DNA composed of 5′-TTCACCGAA-3′/5′-TTCGGTGAA-3′ was constructed using CHARMM force-field parameters. The starting structure was built based on the structure of the ImPyPy-γ-PyPyPy-5′-d(CGCTAACAGGC)-3′/5′-d(GCCTGTTAGCG)-3′ complex revealed by NMR.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Periodic boundary conditions were used in explicit solvent simulation. The resulting orthorhombic water solvation model contained 0.145 mol/L salt (sodium cation and chloride anion). The minimum distance from the boundary was set to 7 Å. The entire system was minimized to a root mean square (RMS) of <0.001 kcal/mol Å without any restrictions.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Quantitative Real-Time Polymerase Chain Reaction Assay</h4><div class="NLM_p last">To quantify PD-L1 mRNA expression, total RNA was extracted from cells with TRIzol reagent (Invitrogen), and cDNA was synthesized using a first strand cDNA synthesis kit (TOYOBO) according to the manufacturer’s instructions. Quantitative real-time polymerase chain reaction (q-PCR) analysis was performed using the iTaq Universal SYBR Green Supermix (BIO RAD) and an ABI Prism 7900-HT sequence detection system (96-well, BIO RAD). The following primers were used for the amplification of PD-L1: forward primer 5′-GGACAAGCAGTGACCATCAAG-3′ and reverse primer 5′-CCCAGAATTACCAAGTGAGTCCT-3′. The following primers were used for the amplification of glyceraldehyde 3-phosphate dehydrogenase (GAPDH): forward primer 5′-AGAAGGCTGGGGCTCATTTG-3′ and reverse primer 5′-AGGGGCCATCCACAGTCTTC-3′. The experiments were conducted in triplicate. GAPDH was used as the internal control.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Western Blot Analysis</h4><div class="NLM_p last">Cells were lysed in 1× radioimmunoprecipitation assay (RIPA) buffer with phosphatase inhibitors. Cell lysates were cleared by centrifugation at 4 °C for 15 min at 13 200<i>g</i>. Protein was separated by sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE) and transferred onto a polyvinylidene fluoride (PVDF) membrane. After being blocked with 5% bovine serum albumin (BSA)/tris-buffered saline, 0.1% TWEEN 20 detergent (TBST) buffer for 2 h, membranes were incubated in primary antibodies overnight at 4 °C. The following primary antibodies were used in the dilution, as suggested by the supplier: anti-PD-L1 mAb, anti-Akt mAb, anti-P-Akt mAb, anticaspase-3 mAb (1:1000; Cell Signal Technology), and anti-GAPDH Ab (1:1000; Cell Signal Technology). Then, the membranes were incubated with secondary antibodies for 2 h at room temperature, immersed in BeyoECL Star (Beyotime) for 2 min, and photographed under a Bio-Rad gel imaging analyzer.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Wound Healing Assay</h4><div class="NLM_p last">MDA-MB-231 and HCC827 cells were cultured at a cell density of 1 × 10<sup>5</sup> cells/mL in DMEM and RPMI1640 medium containing growth factors for 24 h, respectively. An artificial gap was generated using a disposable 200 μL pipet chip on the confluent cell monolayer in a streaking motion. Polyamide compounds at the indicated concentrations were added to the medium of the streaked cell culture. The cells were cultured in serum-free medium for an additional 48 h, and the healing of the gap was photographed. To quantify the effect of the compounds on the wound healing, the % cell inhibitory rate was calculated by the following equation: cell inhibitory rate (%) = (1 – <i>D</i><sub>drug</sub>/<i>D</i><sub>control</sub>) × 100%,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> in which <i>D</i><sub>drug</sub> is the mean distance of cell migration in the sample containing drug and <i>D</i><sub>control</sub> is the mean distance of cell migration in the sample without the drug. The images of the initially disturbed monolayers were compared with the corresponding images of cells at the end of the incubation. Data are shown as the mean ± SD. Artificial lines fitting the cutting edges were drawn on the images of the original artificial gaps and on the images of cultures after incubation. Three different points were marked on each plate. Three independent experiments were done, and the representative images were shown.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Coculture Experiments and IL-2 Analysis by ELISA</h4><div class="NLM_p last">Jurkat T cells were cocultured with tumor cells, and the production of IL-2 was determined, as previously described.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> To analyze the effect of compounds on the tumor microenvironment, tumor cells were cocultured with PMA (phorbol 12-myristate 13-acetate, Sigma-Aldrich)-activated Jurkat T cells. The cells were cultured at an effector-to-target ratio of 5:1 for 24 or 36 h. IL-2 secreted from Jurkat cells into the medium was determined according to the manufacturer’s protocol (human IL-2 ELISA (enzyme-linked immunosorbent assay) kit, DAKEWE).</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> CCK-8 Assay</h4><div class="NLM_p last">The in vitro cytotoxic effect of novel polyamides on HCC827 and MDA-MB-231 and MRC-5 tumor cells was determined by using a Cell Counting Kit-8 assay kit (CCK-8) (Dojindo Molecular Technologies, Kumamoto, Japan) according to the manufacturer’s protocol. In brief, cells were maintained in the complete RPMI 1640 medium (Invitrogen Gibco) at 37 °C with 5% CO<sub>2</sub>. Cells in the logarithmic growth phase were harvested and resuspended in the medium. The cell suspensions (95 μL) were seeded in a 96-well plate, which was incubated for 4 h at 37 °C with 5% CO<sub>2</sub>. Then, 1 μL of a polyamide stock solution in DMSO was diluted with 4 μL of culture medium. After the 5 μL sample was added to the 96-well plate, the plate was cultured for an additional 36 h. Then, 10 μL of CCK-8 reagent was added to each well, and the plate was incubated at 37 °C for another 1 h. Finally, the optical density of each sample was measured at 450 nm using a BioTek Synergy H1 microplate reader. Cisplatin was used as a positive control. “1% DMSO” is a solution containing 1% DMSO in RPMI 1640 as a reference.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Statistical Analysis</h4><div class="NLM_p last">The Pearson R test, Spearman rank correlation test, and Student’s <i>t</i> test were performed using GraphPad Prism 6 (GraphPad Software). Data were expressed as the mean ± standard deviation (S.D.) of three independent experiments, and the difference (<i>P</i> < 0.05) was considered significant.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00120" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80281" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80281" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00120?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00120</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Schematic of the Fmoc solid-phase synthesis of the polyamide <b>1</b> and other biological results, including UV melting curves, CD spectra, a typical SPR sensorgram for Mis-polyamide, molecular docking of Mis-polyamide and the ds-DNA, distribution of fluorescein (Fluo)-labeled polyamide <b>2</b><i>in vitro</i>, structure of PD-1/PD-L1 inhibitor <b>2</b> (BMS202), effect of polyamides on Jurkat cell viability, and effect of polyamides <b>1</b> and <b>2</b> on the mobility of HCC827 and MDA-MB-231 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Antiproliferative activity of compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf">jm1c00120_si_001.pdf (0.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_002.csv">jm1c00120_si_002.csv (1.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_003.csv">jm1c00120_si_003.csv (0.96 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00120" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51665" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51665" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanling Wu</span> - <span class="hlFld-Affiliation affiliation">Lab
of Molecular Immunology, Virus Inspection Department, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou 310051, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#651c091210250601064b1f0f4b060b"><span class="__cf_email__" data-cfemail="b9c0d5ceccf9daddda97c3d397dad7">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hiroshi Sugiyama</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Graduate School of Science, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8923-5946" title="Orcid link">http://orcid.org/0000-0001-8923-5946</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f69e85b69d83959e939bd89d8f998299db83d89795d89c86"><span class="__cf_email__" data-cfemail="610912210a140209040c4f0a180e150e4c144f00024f0b11">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen Zhang</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3247-9672" title="Orcid link">http://orcid.org/0000-0002-3247-9672</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0275786a636c65343142786877762c6766772c616c"><span class="__cf_email__" data-cfemail="fa8d80929b949dccc9ba80908f8ed49f9e8fd49994">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ming Wang</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xudong Ma</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kang Zhou</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huijuan Mao</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiachun Liu</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuqiong Xiong</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyin Zhao</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suresh Narva</span> - <span class="hlFld-Affiliation affiliation">Lab
of Chemical Biology and Molecular Drug Design, College of Pharmaceutical
Science, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Drug Development & Chemical Biology, Zhejiang University of Technology, Hangzhou 310014, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0702-784X" title="Orcid link">http://orcid.org/0000-0003-0702-784X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yoshimasa Tanaka</span> - <span class="hlFld-Affiliation affiliation">Center
for Innovation in Immunoregulative Technology and Therapeutics, Graduate
School of Medicine, Kyoto University, Kyoto 606-8501, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chuanxin Guo</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Graduate School of Science, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</span>; 
    <span>Present Address:
                        C.G.: Shanghai Cell Therapy Group Co., Ltd., Building A, No. 75 Qianyang Road, Jiading Dist, Shanghai 210805, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.W., X.M., K.Z., and H.M. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44733" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44733" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Zhejiang University of Technology, China; the Zhejiang Provincial Center for Disease Control and Prevention, China; and Kyoto University, Japan for the research equipment and technical support for this research. We are grateful to Professor Tasuku Honjo from Kyoto University Graduate School of Medicine for critical reading of our manuscript and helpful advice. This work was supported by grants from the National Natural Science Foundation of China (nos. 21572207 and 21877101) and the Zhejiang Leading Innovation and Entrepreneurship Team (2018R01015).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PD-1</td><td class="NLM_def"><p class="first last">programmed cell death protein 1</p></td></tr><tr><td class="NLM_term">PD-L1</td><td class="NLM_def"><p class="first last">Programmed death ligand 1</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription-3</p></td></tr><tr><td class="NLM_term">IL-2</td><td class="NLM_def"><p class="first last">interleukin 2</p></td></tr><tr><td class="NLM_term">TGF-β1</td><td class="NLM_def"><p class="first last">transforming growth factor beta 1</p></td></tr><tr><td class="NLM_term">SBS</td><td class="NLM_def"><p class="first last">STAT3 binding site</p></td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">circular dichroism</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">ICD</td><td class="NLM_def"><p class="first last">induced circular dichroism</p></td></tr><tr><td class="NLM_term">HCC827</td><td class="NLM_def"><p class="first last">human non-small-cell lung cancer cell line</p></td></tr><tr><td class="NLM_term">MDA-MB-231</td><td class="NLM_def"><p class="first last">human breast cancer cell line</p></td></tr><tr><td class="NLM_term">IFN-γ</td><td class="NLM_def"><p class="first last">human interferon-γ</p></td></tr><tr><td class="NLM_term">BMS202</td><td class="NLM_def"><p class="first last">PD-1/PD-L1 interaction inhibitor 2</p></td></tr><tr><td class="NLM_term">CCK-8</td><td class="NLM_def"><p class="first last">Cell Counting Kit-8</p></td></tr><tr><td class="NLM_term">MRC-5</td><td class="NLM_def"><p class="first last">human fetal lung fibroblast cell line</p></td></tr><tr><td class="NLM_term">AKT</td><td class="NLM_def"><p class="first last">serine/threonine kinase</p></td></tr><tr><td class="NLM_term">mTOR</td><td class="NLM_def"><p class="first last">mechanistic target of rapamycin kinase</p></td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">anaplastic lymphoma kinase</p></td></tr><tr><td class="NLM_term">LMP1</td><td class="NLM_def"><p class="first last">latent membrane protein 1</p></td></tr><tr><td class="NLM_term">EMT</td><td class="NLM_def"><p class="first last">epithelial–mesenchymal transition</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34128" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34128" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Couzin-Frankel, J.</span></span> <span> </span><span class="NLM_article-title">Breakthrough of the year 2013. Cancer immunotherapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1126/science.342.6165.1432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1126%2Fscience.342.6165.1432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=24357284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2013&pages=1432-1433&author=J.+Couzin-Frankel&title=Breakthrough+of+the+year+2013.+Cancer+immunotherapy&doi=10.1126%2Fscience.342.6165.1432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer immunotherapy</span></div><div class="casAuthors">Couzin-Frankel, Jennifer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">6165</span>),
    <span class="NLM_cas:pages">1432-1433</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxASV1j-6kL7Vg90H21EOLACvtfcHk0liGWY8WJ3hRUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotF2qsg%253D%253D&md5=2e0995fd567c5b59f40c46981da0bef4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.342.6165.1432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.342.6165.1432%26sid%3Dliteratum%253Aachs%26aulast%3DCouzin-Frankel%26aufirst%3DJ.%26atitle%3DBreakthrough%2520of%2520the%2520year%25202013.%2520Cancer%2520immunotherapy%26jtitle%3DScience%26date%3D2013%26volume%3D342%26spage%3D1432%26epage%3D1433%26doi%3D10.1126%2Fscience.342.6165.1432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Topalian, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardoll, D. M.</span></span> <span> </span><span class="NLM_article-title">Immune checkpoint blockade: a common denominator approach to cancer therapy</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ccell.2015.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25858804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVaku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=450-461&author=S.+L.+Topalianauthor=C.+G.+Drakeauthor=D.+M.+Pardoll&title=Immune+checkpoint+blockade%3A+a+common+denominator+approach+to+cancer+therapy&doi=10.1016%2Fj.ccell.2015.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Immune Checkpoint Blockade: A Common Denominator Approach to Cancer Therapy</span></div><div class="casAuthors">Topalian, Suzanne L.; Drake, Charles G.; Pardoll, Drew M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The immune system recognizes and is poised to eliminate cancer but is held in check by inhibitory receptors and ligands.  These immune checkpoint pathways, which normally maintain self-tolerance and limit collateral tissue damage during anti-microbial immune responses, can be co-opted by cancer to evade immune destruction.  Drugs interrupting immune checkpoints, such as anti-CTLA-4, anti-PD-1, anti-PD-L1, and others in early development, can unleash anti-tumor immunity and mediate durable cancer regressions.  The complex biol. of immune checkpoint pathways still contains many mysteries, and the full activity spectrum of checkpoint-blocking drugs, used alone or in combination, is currently the subject of intense study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb0YUggJYBrLVg90H21EOLACvtfcHk0liGWY8WJ3hRUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVaku7k%253D&md5=e42c02d50881184b78630e5dd23f7f2e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DTopalian%26aufirst%3DS.%2BL.%26aulast%3DDrake%26aufirst%3DC.%2BG.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26atitle%3DImmune%2520checkpoint%2520blockade%253A%2520a%2520common%2520denominator%2520approach%2520to%2520cancer%2520therapy%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ccell.2015.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ceeraz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noelle, R. J.</span></span> <span> </span><span class="NLM_article-title">B7 family checkpoint regulators in immune regulation and disease</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">556</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1016/j.it.2013.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.it.2013.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=23954143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1KltbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=556-563&author=S.+Ceerazauthor=E.+C.+Nowakauthor=R.+J.+Noelle&title=B7+family+checkpoint+regulators+in+immune+regulation+and+disease&doi=10.1016%2Fj.it.2013.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">B7 family checkpoint regulators in immune regulation and disease</span></div><div class="casAuthors">Ceeraz, Sabrina; Nowak, Elizabeth C.; Noelle, Randolph J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">556-563</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fine-tuning the immune response and maintaining tolerance to self-antigens involves a complex network of co-stimulatory and co-inhibitory mols.  The recent FDA approval of ipilimumab, a monoclonal antibody blocking cytotoxic T lymphocyte antigen (CTLA)-4, demonstrates the impact of checkpoint regulators in disease.  This is reinforced by ongoing clin. trials targeting not only CTLA-4, but also the programmed death (PD)-1 and B7-H4 pathways in various disease states.  Recently, two new B7 family inhibitory ligands, V-domain Ig suppressor of T cell activation (VISTA) and B7-H6 were identified.  Here, we review recent understanding of B7 family members and their concerted regulation of the immune response to either self or foreign pathogens.  We also discuss clin. developments in targeting these pathways in different disease settings, and introduce VISTA as a putative therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpywmoIEHf3s7Vg90H21EOLACvtfcHk0lhZWj_WSiTJ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1KltbjJ&md5=395a88bdda0a83ec10fe4845d5fa76c9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2013.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2013.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DCeeraz%26aufirst%3DS.%26aulast%3DNowak%26aufirst%3DE.%2BC.%26aulast%3DNoelle%26aufirst%3DR.%2BJ.%26atitle%3DB7%2520family%2520checkpoint%2520regulators%2520in%2520immune%2520regulation%2520and%2520disease%26jtitle%3DTrends%2520Immunol.%26date%3D2013%26volume%3D34%26spage%3D556%26epage%3D563%26doi%3D10.1016%2Fj.it.2013.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karwacz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arce, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escors, D.</span></span> <span> </span><span class="NLM_article-title">PD-L1 co-stimulation contributes to ligand-induced T cell receptor down-modulation on CD8+ T cells</span>. <i>EMBO Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1002/emmm.201100165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Femmm.201100165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21739608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlersL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=581-592&author=K.+Karwaczauthor=C.+Bricogneauthor=D.+MacDonaldauthor=F.+Arceauthor=C.+L.+Bennettauthor=M.+Collinsauthor=D.+Escors&title=PD-L1+co-stimulation+contributes+to+ligand-induced+T+cell+receptor+down-modulation+on+CD8%2B+T+cells&doi=10.1002%2Femmm.201100165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1 co-stimulation contributes to ligand-induced T cell receptor down-modulation on CD8+ T cells</span></div><div class="casAuthors">Karwacz, Katarzyna; Bricogne, Christopher; MacDonald, Douglas; Arce, Frederick; Bennett, Clare L.; Collins, Mary; Escors, David</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">581-592</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">T cell receptor (TCR) down-modulation after antigen presentation is a fundamental process that regulates TCR signal transduction.  Current understanding of this process is that intrinsic TCR/CD28 signal transduction leads to TCR down-modulation.  Here, we show that the interaction between programmed cell death 1 ligand 1 (PD-L1) on dendritic cells (DCs) and programmed death 1 (PD-1) on CD8 T cells contributes to ligand-induced TCR down-modulation.  We provide evidence that this occurs via Casitas B-lymphoma (Cbl)-b E3 ubiquitin ligase up-regulation in CD8 T cells.  Interference with PD-L1/PD-1 signalling markedly inhibits TCR down-modulation leading to hyper-activated, proliferative CD8 T cells as assessed in vitro and in vivo in an arthritis model.  PD-L1 silencing accelerates anti-tumor immune responses and strongly potentiates DC anti-tumor capacities, when combined with mitogen-activated kinase (MAPK) modulators that promote DC activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsoJxavR3bpbVg90H21EOLACvtfcHk0lhZWj_WSiTJ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlersL%252FL&md5=4f7eefaa0d962c7118676ed6417d8bc3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Femmm.201100165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Femmm.201100165%26sid%3Dliteratum%253Aachs%26aulast%3DKarwacz%26aufirst%3DK.%26aulast%3DBricogne%26aufirst%3DC.%26aulast%3DMacDonald%26aufirst%3DD.%26aulast%3DArce%26aufirst%3DF.%26aulast%3DBennett%26aufirst%3DC.%2BL.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DEscors%26aufirst%3DD.%26atitle%3DPD-L1%2520co-stimulation%2520contributes%2520to%2520ligand-induced%2520T%2520cell%2520receptor%2520down-modulation%2520on%2520CD8%252B%2520T%2520cells%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D581%26epage%3D592%26doi%3D10.1002%2Femmm.201100165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marabelle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahmer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gettinger, S.</span></span> <span> </span><span class="NLM_article-title">Immune checkpoint modulation for non-small cell lung cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2256</span>– <span class="NLM_lpage">2262</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-2959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F1078-0432.CCR-14-2959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25979932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslOktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2256-2262&author=J.+C.+Soriaauthor=A.+Marabelleauthor=J.+R.+Brahmerauthor=S.+Gettinger&title=Immune+checkpoint+modulation+for+non-small+cell+lung+cancer&doi=10.1158%2F1078-0432.CCR-14-2959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Immune Checkpoint Modulation for Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Soria, Jean-Charles; Marabelle, Aurelien; Brahmer, Julie R.; Gettinger, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2256-2262</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Therapies targeting immune checkpoints have recently shown encouraging activity in patients with heavily pretreated advanced non-small cell lung cancer (NSCLC), independently of NSCLC histol. or mutational status, with low toxicity profiles when used as monotherapy.  Objective response rates of approx. 20% have been reported in patients with advanced NSCLC treated with antagonist antibodies targeting the immune checkpoint, programmed death 1 (PD-1) on activated T cells, or its primary ligand, programmed death ligand 1 (PD-L1) expressed within the tumor microenvironment.  Response rates appear to be higher in patients with tumor PD-L1 expression documented by immunohistochem., although responses have been appreciated in patients with reportedly PD-L1-neg. tumor specimens.  Antibodies directed against cytotoxic T-lymphocyte-assocd. protein 4 (CTLA-4), another immunosuppressive T-cell signaling mol., are also being evaluated in clin. trials, with one randomized phase II trial demonstrating improved immune-related progression-free survival in lung cancer patients when added to std. chemotherapy.  Addnl. clin. trials are combining anti-CTLA-4 antibodies with either anti-PD-1 or anti-PD-L1 antibodies.  Combinations of other immune checkpoint antagonists or agonist antibodies with anti-PD-1 or anti-PD-L1 antibodies are also being pursued.  Clin Cancer Res; 21(10); 2256-62. ©2015 AACR.  See all articles in this section, "Progress in Lung Cancer".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkSMBG3GpFQLVg90H21EOLACvtfcHk0lhZWj_WSiTJ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslOktbs%253D&md5=9bb87571496cc02a6172e82023b20f5b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-2959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-2959%26sid%3Dliteratum%253Aachs%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DMarabelle%26aufirst%3DA.%26aulast%3DBrahmer%26aufirst%3DJ.%2BR.%26aulast%3DGettinger%26aufirst%3DS.%26atitle%3DImmune%2520checkpoint%2520modulation%2520for%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2256%26epage%3D2262%26doi%3D10.1158%2F1078-0432.CCR-14-2959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sareddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandeswara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drerup, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murthy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thedieck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadlamudi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curiel, T. J.</span></span> <span> </span><span class="NLM_article-title">Tumor intrinsic PD-L1 signals regulate cell growth, pathogenesis, and autophagy in ovarian cancer and melanoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">6964</span>– <span class="NLM_lpage">6974</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-0258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F0008-5472.CAN-16-0258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27671674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFyntrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=6964-6974&author=C.+A.+Clarkauthor=H.+B.+Guptaauthor=G.+Sareddyauthor=S.+Pandeswaraauthor=S.+Laoauthor=B.+Yuanauthor=J.+M.+Drerupauthor=A.+Padronauthor=G.+J.+Conejo-Garciaauthor=K.+Murthyauthor=Y.+Liuauthor=M.+J.+Turkauthor=K.+Thedieckauthor=V.+Hurezauthor=R.+Liauthor=R.+Vadlamudiauthor=T.+J.+Curiel&title=Tumor+intrinsic+PD-L1+signals+regulate+cell+growth%2C+pathogenesis%2C+and+autophagy+in+ovarian+cancer+and+melanoma&doi=10.1158%2F0008-5472.CAN-16-0258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor-Intrinsic PD-L1 Signals Regulate Cell Growth, Pathogenesis, and Autophagy in Ovarian Cancer and Melanoma</span></div><div class="casAuthors">Clark, Curtis A.; Gupta, Harshita B.; Sareddy, Gangadhara; Pandeswara, Srilakshmi; Lao, Shunhua; Yuan, Bin; Drerup, Justin M.; Padron, Alvaro; Conejo-Garcia, Jose; Murthy, Kruthi; Liu, Yang; Turk, Mary Jo; Thedieck, Kathrin; Hurez, Vincent; Li, Rong; Vadlamudi, Ratna; Curiel, Tyler J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6964-6974</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PD-L1 antibodies produce efficacious clin. responses in diverse human cancers, but the basis for their effects remains unclear, leaving a gap in the understanding of how to rationally leverage therapeutic activity.  PD-L1 is widely expressed in tumor cells, but its contributions to tumor pathogenicity are incompletely understood.  In this study, we evaluated the hypothesis that PD-L1 exerts tumor cell-intrinsic signals that are crit. for pathogenesis.  Using RNAi methodol., we attenuated PD-L1 in the murine ovarian cell line ID8agg and the melanoma cell line B16 (termed PD-L1lo cells), which express basal PD-L1.  We obsd. that PD-L1lo cells proliferated more weakly than control cells in vitro.  As expected, PD-L1lo cells formed tumors in immunocompetent mice relatively more slowly, but unexpectedly, they also formed tumors more slowly in immunodeficient NSG mice.  RNA sequencing anal. identified a no. of genes involved in autophagy and mTOR signaling that were affected by PD-L1 expression.  In support of a functional role, PD-L1 attenuation augmented autophagy and blunted the ability of autophagy inhibitors to limit proliferation in vitro and in vivo in NSG mice.  PD-L1 attenuation also reduced mTORC1 activity and augmented the antiproliferative effects of the mTORC1 inhibitor rapamycin.  PD-L1lo cells were also relatively deficient in metastasis to the lung, and we found that anti-PD-L1 administration could block tumor cell growth and metastasis in NSG mice.  This therapeutic effect was obsd. with B16 cells but not ID8agg cells, illustrating tumor- or compartmental-specific effects in the therapeutic setting.  Overall, our findings extend understanding of PD-L1 functions, illustrate nonimmune effects of anti-PD-L1 immunotherapy, and suggest broader uses for PD-L1 as a biomarker for assessing cancer therapeutic responses.  Cancer Res; 76(23); 6964-74. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ6FcRFM_eY7Vg90H21EOLACvtfcHk0liCfRIWWgtePA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFyntrjP&md5=02dafdbfde9636f0272c53255c4e2d3c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-0258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-0258%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DC.%2BA.%26aulast%3DGupta%26aufirst%3DH.%2BB.%26aulast%3DSareddy%26aufirst%3DG.%26aulast%3DPandeswara%26aufirst%3DS.%26aulast%3DLao%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DDrerup%26aufirst%3DJ.%2BM.%26aulast%3DPadron%26aufirst%3DA.%26aulast%3DConejo-Garcia%26aufirst%3DG.%2BJ.%26aulast%3DMurthy%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DTurk%26aufirst%3DM.%2BJ.%26aulast%3DThedieck%26aufirst%3DK.%26aulast%3DHurez%26aufirst%3DV.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DVadlamudi%26aufirst%3DR.%26aulast%3DCuriel%26aufirst%3DT.%2BJ.%26atitle%3DTumor%2520intrinsic%2520PD-L1%2520signals%2520regulate%2520cell%2520growth%252C%2520pathogenesis%252C%2520and%2520autophagy%2520in%2520ovarian%2520cancer%2520and%2520melanoma%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D6964%26epage%3D6974%26doi%3D10.1158%2F0008-5472.CAN-16-0258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Windt, G. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonc, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, E. L.</span></span> <span> </span><span class="NLM_article-title">Metabolic competition in the tumor microenvironment is a driver of cancer progression</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.cell.2015.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26321679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVajs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2015&pages=1229-1241&author=C.+Changauthor=J.+Qiuauthor=D.+O%E2%80%99Sullivanauthor=M.+D.+Buckauthor=T.+Noguchiauthor=J.+D.+Curtisauthor=Q.+Chenauthor=M.+Gindinauthor=M.+M.+Gubinauthor=G.+J.+W.+van+der+Windtauthor=E.+Toncauthor=R.+D.+Schreiberauthor=E.+J.+Pearceauthor=E.+L.+Pearce&title=Metabolic+competition+in+the+tumor+microenvironment+is+a+driver+of+cancer+progression&doi=10.1016%2Fj.cell.2015.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Competition in the Tumor Microenvironment Is a Driver of Cancer Progression</span></div><div class="casAuthors">Chang, Chih-Hao; Qiu, Jing; O'Sullivan, David; Buck, Michael D.; Noguchi, Takuro; Curtis, Jonathan D.; Chen, Qiongyu; Gindin, Mariel; Gubin, Matthew M.; van der Windt, Gerritje J. W.; Tonc, Elena; Schreiber, Robert D.; Pearce, Edward J.; Pearce, Erika L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1229-1241</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Failure of T cells to protect against cancer is thought to result from lack of antigen recognition, chronic activation, and/or suppression by other cells.  Using a mouse sarcoma model, we show that glucose consumption by tumors metabolically restricts T cells, leading to their dampened mTOR activity, glycolytic capacity, and IFN-γ prodn., thereby allowing tumor progression.  We show that enhancing glycolysis in an antigenic "regressor" tumor is sufficient to override the protective ability of T cells to control tumor growth.  We also show that checkpoint blockade antibodies against CTLA-4, PD-1, and PD-L1, which are used clin., restore glucose in tumor microenvironment, permitting T cell glycolysis and IFN-γ prodn.  Furthermore, we found that blocking PD-L1 directly on tumors dampens glycolysis by inhibiting mTOR activity and decreasing expression of glycolysis enzymes, reflecting a role for PD-L1 in tumor glucose utilization.  Our results establish that tumor-imposed metabolic restrictions can mediate T cell hyporesponsiveness during cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSf2NZ5rM4LrVg90H21EOLACvtfcHk0liCfRIWWgtePA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVajs7%252FI&md5=cfc0a1c9d46d34259927a0130bd380a7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DD.%26aulast%3DBuck%26aufirst%3DM.%2BD.%26aulast%3DNoguchi%26aufirst%3DT.%26aulast%3DCurtis%26aufirst%3DJ.%2BD.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DGindin%26aufirst%3DM.%26aulast%3DGubin%26aufirst%3DM.%2BM.%26aulast%3Dvan%2Bder%2BWindt%26aufirst%3DG.%2BJ.%2BW.%26aulast%3DTonc%26aufirst%3DE.%26aulast%3DSchreiber%26aufirst%3DR.%2BD.%26aulast%3DPearce%26aufirst%3DE.%2BJ.%26aulast%3DPearce%26aufirst%3DE.%2BL.%26atitle%3DMetabolic%2520competition%2520in%2520the%2520tumor%2520microenvironment%2520is%2520a%2520driver%2520of%2520cancer%2520progression%26jtitle%3DCell%26date%3D2015%26volume%3D162%26spage%3D1229%26epage%3D1241%26doi%3D10.1016%2Fj.cell.2015.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrzeszczynska, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestell, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">STAT3 as an oneogene</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">303</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81959-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS0092-8674%2800%2981959-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10458605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1999&pages=295-303&author=J.+F.+Brombergauthor=M.+H.+Wrzeszczynskaauthor=G.+Devganauthor=Y.+Zhaoauthor=R.+G.+Pestellauthor=C.+Albaneseauthor=J.+E.+Darnell&title=STAT3+as+an+oneogene&doi=10.1016%2FS0092-8674%2800%2981959-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 as an oncogene</span></div><div class="casAuthors">Bromberg, Jacqueline F.; Wrzeszczynska, Melissa H.; Devgan, Geeta; Zhao, Yanxiang; Pestell, Richard G.; Albanese, Chris; Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">295-303</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">STATs are latent transcription factors that mediate cytokine- and growth factor-directed transcription.  In many human cancers and transformed cell lines, Stat3 is persistently activated, and in cell culture, active Stat3 is either required for transformation, enhances transformation, or blocks apoptosis.  Here, the substitution of two cysteine residues within the C-terminal loop of the SH2 domain of Stat3 produced a mol. that dimerizes spontaneously, binds to DNA, and activates transcription.  The Stat3-C mol. in immortalized fibroblasts causes cellular transformation scored by colony formation in soft agar and tumor formation in nude mice.  Thus, the activated Stat3 mol. by itself can mediate cellular transformation and the expts. focus attention on the importance of constitutive Stat3 activation in human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOPJVvUEeTbVg90H21EOLACvtfcHk0liCfRIWWgtePA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D&md5=f2e69e246bbfae15a3c5ee7b94340300</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981959-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981959-5%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%2BF.%26aulast%3DWrzeszczynska%26aufirst%3DM.%2BH.%26aulast%3DDevgan%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DPestell%26aufirst%3DR.%2BG.%26aulast%3DAlbanese%26aufirst%3DC.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DSTAT3%2520as%2520an%2520oneogene%26jtitle%3DCell%26date%3D1999%26volume%3D98%26spage%3D295%26epage%3D303%26doi%3D10.1016%2FS0092-8674%2800%2981959-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Decker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovarik, P.</span></span> <span> </span><span class="NLM_article-title">Serine phosphorylation of STATs</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2628</span>– <span class="NLM_lpage">2637</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fsj.onc.1203481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10851062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktVGhtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=2628-2637&author=T.+Deckerauthor=P.+Kovarik&title=Serine+phosphorylation+of+STATs&doi=10.1038%2Fsj.onc.1203481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Serine phosphorylation of STATs</span></div><div class="casAuthors">Decker, Thomas; Kovarik, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2628-2637</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 74 refs.  Tyrosine phosphorylation regulates the dimerization of STATs as an essential prerequisite for the establishment of a classical JAK-STAT signaling path.  However, most vertebrate STATs contain a second phosphorylation site within their C-termini.  The phosphorylated residue in this case is a serine contained within a P(M)SP motif, and in the majority of situations its mutation to alanine alters transcription factor activity.  This review addresses recent advances in understanding the regulation of STAT serine phosphorylation, as well as the kinases and other signal transducers implied in this process.  The biochem. and biol. consequences of STAT serine phosphorylation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3t2x7RXMunrVg90H21EOLACvtfcHk0ljWl2vYaLvNuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktVGhtrk%253D&md5=f5b49c8386069b7cef95702d3b52b90c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203481%26sid%3Dliteratum%253Aachs%26aulast%3DDecker%26aufirst%3DT.%26aulast%3DKovarik%26aufirst%3DP.%26atitle%3DSerine%2520phosphorylation%2520of%2520STATs%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D2628%26epage%3D2637%26doi%3D10.1038%2Fsj.onc.1203481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">The STATs of cancer-new molecular targets come of age</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/nrc1275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fnrc1275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=14964307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=97-105&author=H.+Yuauthor=R.+Jove&title=The+STATs+of+cancer-new+molecular+targets+come+of+age&doi=10.1038%2Fnrc1275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The STATs of cancer - new molecular targets come of age</span></div><div class="casAuthors">Yu, Hua; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-105</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Tumor cells acquire the ability to proliferate uncontrollably, resist apoptosis, sustain angiogenesis and evade immune surveillance.  STAT proteins - esp. STAT3 and STAT5 - regulate all of these processes and are persistently activated in a surprisingly large no. of human cancers.  Consequently, STAT proteins are emerging - unexpectedly - as ideal targets for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdnLoVoH4d1rVg90H21EOLACvtfcHk0ljWl2vYaLvNuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D&md5=3e342db4856310cac2465240e36d3226</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc1275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1275%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DThe%2520STATs%2520of%2520cancer-new%2520molecular%2520targets%2520come%2520of%2520age%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D97%26epage%3D105%26doi%3D10.1038%2Fnrc1275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koskela, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldfors, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Adrichem, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusanmäki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagström, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemente, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalkanen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almusa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edgren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepistö, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattila, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koistinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuittinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penttinen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saarela, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioniemi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porkka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughran, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span> <span> </span><span class="NLM_article-title">Somatic STAT3 mutations in large granular lymphocytic leukemia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>366</i></span>,  <span class="NLM_fpage">1905</span>– <span class="NLM_lpage">1913</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1114885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1056%2FNEJMoa1114885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22591296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38Xntlalsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2012&pages=1905-1913&author=H.+L.+Koskelaauthor=S.+Eldforsauthor=P.+Ellonenauthor=A.+J.+van+Adrichemauthor=H.+Kuusanm%C3%A4kiauthor=E.+I.+Anderssonauthor=S.+Lagstr%C3%B6mauthor=M.+J.+Clementeauthor=T.+Olsonauthor=S.+E.+Jalkanenauthor=M.+M.+Majumderauthor=H.+Almusaauthor=H.+Edgrenauthor=M.+Lepist%C3%B6author=P.+Mattilaauthor=K.+Guintaauthor=P.+Koistinenauthor=T.+Kuittinenauthor=K.+Penttinenauthor=A.+Parsonsauthor=J.+Knowlesauthor=J.+Saarelaauthor=K.+Wennerbergauthor=O.+Kallioniemiauthor=K.+Porkkaauthor=T.+P.+Loughranauthor=C.+A.+Heckmanauthor=J.+P.+Maciejewskiauthor=S.+Mustjoki&title=Somatic+STAT3+mutations+in+large+granular+lymphocytic+leukemia&doi=10.1056%2FNEJMoa1114885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Somatic STAT3 mutations in large granular lymphocytic leukemia</span></div><div class="casAuthors">Koskela, Hanna L. M.; Eldfors, Samuli; Ellonen, Pekka; van Adrichem, Arjan J.; Kuusanmaki, Heikki; Andersson, Emma I.; Lagstrom, Sonja; Clemente, Michael J.; Olson, Thomas; Jalkanen, Sari E.; Majumder, Muntasir Mamun; Almusa, Henrikki; Edgren, Henrik; Lepisto, Maija; Mattila, Pirkko; Guinta, Kathryn; Koistinen, Pirjo; Kuittinen, Taru; Penttinen, Kati; Parsons, Alun; Knowles, Jonathan; Saarela, Janna; Wennerberg, Krister; Kallioniemi, Olli; Porkka, Kimmo; Loughran, Thomas P., Jr.; Heckman, Caroline A.; Maciejewski, Jaroslaw P.; Mustjoki, Satu</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">366</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1905-1913</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">T-cell large granular lymphocytic leukemia is a rare lymphoproliferative disorder characterized by the expansion of clonal CD3+CD8+ cytotoxic T lymphocytes (CTLs) and often assocd. with autoimmune disorders and immune-mediated cytopenias.  We used next-generation exome sequencing to identify somatic mutations in CTLs from an index patient with large granular lymphocytic leukemia.  Targeted resequencing was performed in a well-characterized cohort of 76 patients with this disorder, characterized by clonal T-cell-receptor rearrangements and increased nos. of large granular lymphocytes.  Mutations in the signal transducer and activator of transcription 3 gene (STAT3) were found in 31 of 77 patients (40%) with large granular lymphocytic leukemia.  Among these 31 patients, recurrent mutational hot spots included Y640F in 13 (17%), D661V in 7 (9%), D661Y in 7 (9%), and N647I in 3 (4%).  All mutations were located in exon 21, encoding the Src homol. 2 (SH2) domain, which mediates the dimerization and activation of STAT protein.  The amino acid changes resulted in a more hydrophobic protein surface and were assocd. with phosphorylation of STAT3 and its localization in the nucleus.  In vitro functional studies showed that the Y640F and D661V mutations increased the transcriptional activity of STAT3.  In the affected patients, downstream target genes of the STAT3 pathway (IFNGR2, BCL2L1, and JAK2) were up-regulated.  Patients with STAT3 mutations presented more often with neutropenia and rheumatoid arthritis than did patients without these mutations.  The SH2 dimerization and activation domain of STAT3 is frequently mutated in patients with large granular lymphocytic leukemia; these findings suggest that aberrant STAT3 signaling underlies the pathogenesis of this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyekkHvePb9LVg90H21EOLACvtfcHk0ljWl2vYaLvNuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xntlalsr8%253D&md5=4282683e8c70804aaac1818ea4e278d3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1114885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1114885%26sid%3Dliteratum%253Aachs%26aulast%3DKoskela%26aufirst%3DH.%2BL.%26aulast%3DEldfors%26aufirst%3DS.%26aulast%3DEllonen%26aufirst%3DP.%26aulast%3Dvan%2BAdrichem%26aufirst%3DA.%2BJ.%26aulast%3DKuusanm%25C3%25A4ki%26aufirst%3DH.%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DLagstr%25C3%25B6m%26aufirst%3DS.%26aulast%3DClemente%26aufirst%3DM.%2BJ.%26aulast%3DOlson%26aufirst%3DT.%26aulast%3DJalkanen%26aufirst%3DS.%2BE.%26aulast%3DMajumder%26aufirst%3DM.%2BM.%26aulast%3DAlmusa%26aufirst%3DH.%26aulast%3DEdgren%26aufirst%3DH.%26aulast%3DLepist%25C3%25B6%26aufirst%3DM.%26aulast%3DMattila%26aufirst%3DP.%26aulast%3DGuinta%26aufirst%3DK.%26aulast%3DKoistinen%26aufirst%3DP.%26aulast%3DKuittinen%26aufirst%3DT.%26aulast%3DPenttinen%26aufirst%3DK.%26aulast%3DParsons%26aufirst%3DA.%26aulast%3DKnowles%26aufirst%3DJ.%26aulast%3DSaarela%26aufirst%3DJ.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DLoughran%26aufirst%3DT.%2BP.%26aulast%3DHeckman%26aufirst%3DC.%2BA.%26aulast%3DMaciejewski%26aufirst%3DJ.%2BP.%26aulast%3DMustjoki%26aufirst%3DS.%26atitle%3DSomatic%2520STAT3%2520mutations%2520in%2520large%2520granular%2520lymphocytic%2520leukemia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D366%26spage%3D1905%26epage%3D1913%26doi%3D10.1056%2FNEJMoa1114885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wölfle, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sähr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalpke, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heeg, K.</span></span> <span> </span><span class="NLM_article-title">PD-L1 expression on tolerogenic APCs is controlled by STAT-3</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1002/eji.201040979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Feji.201040979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21268011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC3M7ksVKrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=413-424&author=S.+J.+W%C3%B6lfleauthor=J.+Strebovskyauthor=H.+Bartzauthor=A.+S%C3%A4hrauthor=C.+Arnoldauthor=C.+Kaiserauthor=A.+H.+Dalpkeauthor=K.+Heeg&title=PD-L1+expression+on+tolerogenic+APCs+is+controlled+by+STAT-3&doi=10.1002%2Feji.201040979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1 expression on tolerogenic APCs is controlled by STAT-3</span></div><div class="casAuthors">Wolfle Sabine J; Strebovsky Julia; Bartz Holger; Sahr Aline; Arnold Caroline; Kaiser Claus; Dalpke Alexander H; Heeg Klaus</div><div class="citationInfo"><span class="NLM_cas:title">European journal of immunology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">413-24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">During infection, TLR agonists are released and trigger mature as well as differentiating innate immune cells.  Early encounter with TLR agonists (R848; LPS) blocks conventional differentiation of CD14(+) monocytes into immature dendritic cells (iDCs) resulting in a deviated phenotype.  We and others characterized these APCs (TLR-APC) by a retained expression of CD14 and a lack of CD1a.  Here, we show in addition, expression of programmed death ligand-1 (PD-L1).  TLR-APCs failed to induce T-cell proliferation and furthermore were able to induce CD25(+) Foxp3(+) T regulatory cells (Tregs).  Since PD-L1 is described as a key negative regulator and inducer of tolerance, we further analyzed its regulation.  PD-L1 expression was regulated in a MAPK/cytokine/STAT-3-dependent manner: high levels of IL-6 and IL-10 that signal via STAT-3 were produced by TLR-APCs.  Blocking of STAT-3 activation prevented PD-L1 expression.  Moreover, chromatin immunoprecipitation revealed direct binding of STAT-3 to the PD-L1 promoter.  Those findings indicate a pivotal role of STAT-3 in regulating PD-L1 expression.  MAPKs were indirectly engaged, as blocking of p38 and p44/42 MAPKs decreased IL-6 and IL-10 thus reducing STAT-3 activation and subsequent PD-L1 expression.  Hence, during DC differentiation TLR agonists induce a STAT-3-mediated expression of PD-L1 and favor the development of tolerogenic APCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqpucQi7Nbtr5XHngOXiV_fW6udTcc2ebtGcYD7tTtlLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M7ksVKrtQ%253D%253D&md5=cb53ad4a94a23a6c12269ff67b5d0cb3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Feji.201040979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201040979%26sid%3Dliteratum%253Aachs%26aulast%3DW%25C3%25B6lfle%26aufirst%3DS.%2BJ.%26aulast%3DStrebovsky%26aufirst%3DJ.%26aulast%3DBartz%26aufirst%3DH.%26aulast%3DS%25C3%25A4hr%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DC.%26aulast%3DKaiser%26aufirst%3DC.%26aulast%3DDalpke%26aufirst%3DA.%2BH.%26aulast%3DHeeg%26aufirst%3DK.%26atitle%3DPD-L1%2520expression%2520on%2520tolerogenic%2520APCs%2520is%2520controlled%2520by%2520STAT-3%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2011%26volume%3D41%26spage%3D413%26epage%3D424%26doi%3D10.1002%2Feji.201040979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagishita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeshita, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nokihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochiya, T.</span></span> <span> </span><span class="NLM_article-title">The Clinical Relevance of the miR-197/CKS1B/STAT3-mediated PD-L1 network in chemoresistant non-small-cell lung Cancer</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1038/mt.2015.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fmt.2015.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25597412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=717-727&author=Y.+Fujitaauthor=S.+Yagishitaauthor=K.+Hagiwaraauthor=Y.+Yoshiokaauthor=N.+Kosakaauthor=F.+Takeshitaauthor=T.+Fujiwaraauthor=K.+Tsutaauthor=H.+Nokiharaauthor=T.+Tamuraauthor=H.+Asamuraauthor=M.+Kawaishiauthor=K.+Kuwanoauthor=T.+Ochiya&title=The+Clinical+Relevance+of+the+miR-197%2FCKS1B%2FSTAT3-mediated+PD-L1+network+in+chemoresistant+non-small-cell+lung+Cancer&doi=10.1038%2Fmt.2015.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Clinical Relevance of the miR-197/CKS1B/STAT3-mediated PD-L1 Network in Chemoresistant Non-small-cell Lung Cancer</span></div><div class="casAuthors">Fujita, Yu; Yagishita, Shigehiro; Hagiwara, Keitaro; Yoshioka, Yusuke; Kosaka, Nobuyoshi; Takeshita, Fumitaka; Fujiwara, Tomohiro; Tsuta, Koji; Nokihara, Hiroshi; Tamura, Tomohide; Asamura, Hisao; Kawaishi, Makoto; Kuwano, Kazuyoshi; Ochiya, Takahiro</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">717-727</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Programmed cell death ligand-1 (PD-L1) has recently gained considerable attention for its role in tumor immune escape.  Here, we identify a miR-197/CKS1B/STAT3-mediated PD-L1 network in chemoresistant non-small-cell lung cancer (NSCLC), independent of immunoinhibitory signals.  MiR-197 is downregulated in platinum-resistant NSCLC specimens, resulting in the promotion of chemoresistance, tumorigenicity, and pulmonary metastasis in vitro and in vivo.  Mechanistic investigations reveal that a miR-197-mediated CKS1B/STAT3 axis exerts tumor progression regulated by various oncogenic genes (Bcl-2, c-Myc, and cyclin D1), and PD-L1 is a putative biomarker of this axis.  Furthermore, we demonstrate that a miR-197 mimic sensitizes PD-L1high drug-resistant cells to chemotherapy.  These results indicate that the biol. interaction between PD-L1 and chemoresistance occurs through the microRNA regulatory cascade.  More importantly, expression levels of miR-197 are inversely correlated with PD-L1 expression (n = 177; P = 0.026) and are assocd. with worse overall survival (P = 0.015).  Our discoveries suggest that the miR-197/CKS1B/STAT3-mediated network can drive tumor PD-L1 expression as a biomarker of this cascade, and miR-197 replacement therapy may be a potential treatment strategy for chemoresistant NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3_HY07pVy6LVg90H21EOLACvtfcHk0lglWkWRpTdk_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOqs7o%253D&md5=0f14ec8da56cb12e0361771d8ff50203</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fmt.2015.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2015.10%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DY.%26aulast%3DYagishita%26aufirst%3DS.%26aulast%3DHagiwara%26aufirst%3DK.%26aulast%3DYoshioka%26aufirst%3DY.%26aulast%3DKosaka%26aufirst%3DN.%26aulast%3DTakeshita%26aufirst%3DF.%26aulast%3DFujiwara%26aufirst%3DT.%26aulast%3DTsuta%26aufirst%3DK.%26aulast%3DNokihara%26aufirst%3DH.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DAsamura%26aufirst%3DH.%26aulast%3DKawaishi%26aufirst%3DM.%26aulast%3DKuwano%26aufirst%3DK.%26aulast%3DOchiya%26aufirst%3DT.%26atitle%3DThe%2520Clinical%2520Relevance%2520of%2520the%2520miR-197%252FCKS1B%252FSTAT3-mediated%2520PD-L1%2520network%2520in%2520chemoresistant%2520non-small-cell%2520lung%2520Cancer%26jtitle%3DMol.%2520Ther.%26date%3D2015%26volume%3D23%26spage%3D717%26epage%3D727%26doi%3D10.1038%2Fmt.2015.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horlad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikukawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komohara, Y.</span></span> <span> </span><span class="NLM_article-title">An IL-27/STAT3 axis induces expression of programmed cell death 1 ligands (PD-L1/2) on infiltrating macrophages in lymphoma</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1696</span>– <span class="NLM_lpage">1704</span>, <span class="refDoi"> DOI: 10.1111/cas.13065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1111%2Fcas.13065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27564404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC2szktlagsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1696-1704&author=H.+Horladauthor=C.+Maauthor=H.+Yanoauthor=C.+Panauthor=K.+Ohnishiauthor=Y.+Fujiwaraauthor=S.+Endoauthor=Y.+Kikukawaauthor=Y.+Okunoauthor=M.+Matsuokaauthor=M.+Takeyaauthor=Y.+Komohara&title=An+IL-27%2FSTAT3+axis+induces+expression+of+programmed+cell+death+1+ligands+%28PD-L1%2F2%29+on+infiltrating+macrophages+in+lymphoma&doi=10.1111%2Fcas.13065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An IL-27/Stat3 axis induces expression of programmed cell death 1 ligands (PD-L1/2) on infiltrating macrophages in lymphoma</span></div><div class="casAuthors">Horlad Hasita; Ma Chaoya; Yano Hiromu; Pan Cheng; Ohnishi Koji; Fujiwara Yukio; Takeya Motohiro; Komohara Yoshihiro; Endo Shinya; Kikukawa Yoshitaka; Okuno Yutaka; Matsuoka Masao; Matsuoka Masao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1696-1704</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Immune escape and tolerance in the tumor microenvironment are closely involved in tumor progression, and are caused by T-cell exhaustion and mediated by the inhibitory signaling of immune checkpoint molecules including programmed death-1 (PD-1), cytotoxic T-lymphocyte associated protein 4, and T-cell immunoglobulin and mucin domaincontaining molecule-3.  In the present study, we investigated the expression of the PD-1 ligand 1 (PD-L1) in a lymphoma microenvironment using paraffin-embedded tissue samples, and subsequently studied the detailed mechanism of upregulation of PD-L1 on macrophages using cultured human macrophages and lymphoma cell lines.  We found that macrophages in lymphoma tissues of almost all cases of adult T-cell leukemia/lymphoma (ATLL), follicular lymphoma and diffuse large B-cell lymphoma expressed PD-L1.  Cell culture studies showed that the conditioned medium of ATL-T and SLVL cell lines induced increased expression of PD-L1/2 on macrophages, and that this PD-L1/2 overexpression was dependent on activation of signal transducer and activator of transcription 3 (Stat3).  In vitro studies including cytokine array analysis showed that IL-27 (heterodimer of p28 and EBI3) induced overexpression of PD-L1/2 on macrophages via Stat3 activation.  Because lymphoma cell lines produced IL-27B (EBI3) but not IL-27p28, it was proposed that the IL-27p28 derived from macrophages and the IL-27B (EBI3) derived from lymphoma cells formed an IL-27 (heterodimer) that induced PD-L1/2 overexpression.  Although the significance of PD-L1/2 expressions on macrophages in lymphoma progression has never been clarified, an IL-27-Stat3 axis might be a target for immunotherapy for lymphoma patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvT7jxf980N4IRAdBLKY41fW6udTcc2ebtGcYD7tTtlLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szktlagsQ%253D%253D&md5=b0fa808fcc4f2e414ac24a22a40e7ca2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fcas.13065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13065%26sid%3Dliteratum%253Aachs%26aulast%3DHorlad%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DYano%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DOhnishi%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DEndo%26aufirst%3DS.%26aulast%3DKikukawa%26aufirst%3DY.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DMatsuoka%26aufirst%3DM.%26aulast%3DTakeya%26aufirst%3DM.%26aulast%3DKomohara%26aufirst%3DY.%26atitle%3DAn%2520IL-27%252FSTAT3%2520axis%2520induces%2520expression%2520of%2520programmed%2520cell%2520death%25201%2520ligands%2520%2528PD-L1%252F2%2529%2520on%2520infiltrating%2520macrophages%2520in%2520lymphoma%26jtitle%3DCancer%2520Sci.%26date%3D2016%26volume%3D107%26spage%3D1696%26epage%3D1704%26doi%3D10.1111%2Fcas.13065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdelhamed, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, Y.</span></span> <span> </span><span class="NLM_article-title">AKT-STAT3 pathway as a downstream target of EGFR signaling to regulate PD-L1 expression on NSCLC cells</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.7150/jca.14713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.7150%2Fjca.14713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27698894" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslWns7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1579-86&author=S.+Abdelhamedauthor=K.+Oguraauthor=S.+Yokoyamaauthor=I.+Saikiauthor=Y.+Hayakawa&title=AKT-STAT3+pathway+as+a+downstream+target+of+EGFR+signaling+to+regulate+PD-L1+expression+on+NSCLC+cells&doi=10.7150%2Fjca.14713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">AKT-STAT3 pathway as a downstream target of EGFR signaling to regulate PD-L1 expression on NSCLC cells</span></div><div class="casAuthors">Abdelhamed, Sherif; Ogura, Keisuke; Yokoyama, Satoru; Saiki, Ikuo; Hayakawa, Yoshihiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer (Wyoming, Australia)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1579-1586</span>CODEN:
                <span class="NLM_cas:coden">JCWAAL</span>;
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher Pty Ltd.</span>)
        </div><div class="casAbstract">While cancer development and progression can be controlled by cytotoxic T cells, it is also known that tumor-specific CD8+T cells become functionally impaired by acquiring a group of inhibitory receptors known as immune checkpoints.  Amongst those, programmed death-1 (PD-1) is one of the most recognized neg. regulators of T cell function.  In non-small lung cancers (NSCLCs), the aberrant activation of epidermal growth factor receptor (EGFR) is known to induce PD-L1 expression and further the treatment with gefitinib, a tyrosine kinase inhibitor (TKI) for EGFR, decrease the expression of PD-L1 on NSCLC.  Given the acquired resistance to gefitinib treatment frequently obsd. by developing secondary-site mutations limiting its efficacy, it is important to understand the downstream mechanism of activated-EGFR signaling for regulating PD-L1 in NSCLC.  In this study, we demonstrated that AKT-STAT3 pathway could be a potential target for regulating the surface expression of PD-L1 on NSCLCs with aberrant EGFR activity and, further, the inhibition of AKT or STAT3 activity could down-regulate the expression of PD-L1 even in gefitinib-resistant NSCLCs.  These results highlight an importance of AKT-STAT3 pathway as a promising target for potentiating anti-tumor immune responses by regulating PD-L1 expression on cancer cells with aberrant EGFR activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NBJ_KnblKbVg90H21EOLACvtfcHk0liviKorqir09w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslWns7k%253D&md5=097a1d3ced2f2109938357b257eb227c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.7150%2Fjca.14713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.14713%26sid%3Dliteratum%253Aachs%26aulast%3DAbdelhamed%26aufirst%3DS.%26aulast%3DOgura%26aufirst%3DK.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DSaiki%26aufirst%3DI.%26aulast%3DHayakawa%26aufirst%3DY.%26atitle%3DAKT-STAT3%2520pathway%2520as%2520a%2520downstream%2520target%2520of%2520EGFR%2520signaling%2520to%2520regulate%2520PD-L1%2520expression%2520on%2520NSCLC%2520cells%26jtitle%3DJ.%2520Cancer%26date%3D2016%26volume%3D7%26spage%3D1579%26epage%3D86%26doi%3D10.7150%2Fjca.14713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talukder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savage, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. B.</span></span> <span> </span><span class="NLM_article-title">JAK-STAT-mediated chronic inflammation impairscytotoxic T lymphocyte activation to decreaseanti-PD-1 immunotherapy efficacy in pancreaticcancer</span>. <i>Oncoimmunology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1291106</span> <span class="refDoi"> DOI: 10.1080/2162402X.2017.1291106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1080%2F2162402X.2017.1291106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28405527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2lsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=C.+Luauthor=A.+Talukderauthor=N.+M.+Savageauthor=N.+Singhauthor=K.+B.+Liu&title=JAK-STAT-mediated+chronic+inflammation+impairscytotoxic+T+lymphocyte+activation+to+decreaseanti-PD-1+immunotherapy+efficacy+in+pancreaticcancer&doi=10.1080%2F2162402X.2017.1291106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">JAK-STAT-mediated chronic inflammation impairs cytotoxic T lymphocyte activation to decrease anti-PD-1 immunotherapy efficacy in pancreatic cancer</span></div><div class="casAuthors">Lu, Chunwan; Talukder, Asif; Savage, Natasha M.; Singh, Nagendra; Liu, Kebin</div><div class="citationInfo"><span class="NLM_cas:title">OncoImmunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1291106/1-e1291106/15</span>CODEN:
                <span class="NLM_cas:coden">ONCOGX</span>;
        ISSN:<span class="NLM_cas:issn">2162-402X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Human pancreatic cancer does not respond to immune check point blockade immunotherapy.  One key feature of pancreatic cancer is the assocn. between its progression and chronic inflammation.  Emerging evidence supports a key role for the JAK-STAT pathway in pancreatic cancer inflammation.  We aimed at testing the hypothesis that sustained JAK-STAT signaling suppresses cytotoxic T lymphocyte (CTL) activation to counteract anti-PD-1 immunotherapy-induced CTL activity in pancreatic cancer.  We show that human pancreatic carcinomas express high level of PD-L1 and exhibit low level of CTL infiltration.  JAK-STAT inhibitor Ruxolitinib selectively inhibits STAT1 and STAT3 activation and increases CTL infiltration to induce a Tc1/Th1 immune response in the tumor microenvironment in an orthotopic pancreatic cancer mouse model.  Ruxilitinib-mediated tumor suppressive efficacy diminishes in T-cell-deficient mice.  Pancreatic tumor grows significantly faster in IFNγ-deficient mice.  However, neutralizing IFNγ does not alter tumor growth but diminishes Ruxolitinib-induced tumor suppression in vivo, indicating that lymphocytes and IFNγ are essential for Ruxolitinib-induced host antitumor immune response.  Both type I and type II interferons upregulate PD-L1 expression through the JAK-STAT signaling pathway in mouse pancreatic tumor cells.  Tumor cells respond to activated T cells by activating STAT3.  The inhibition of STAT3 downregulates immune suppressive cytokines prodn. by tumor cells, resulting in increased T cell activation and effector function.  Consequently, Ruxolitinib significantly improves the efficacy of anti-PD-1 immunotherapy.  Our data demonstrate that Ruxolitinib is effective in the inhibition of systemic inflammation in the tumor microenvironment and therefore upregulates CTL infiltration and activation to overcome pancreatic cancer resistance to anti-PD-1 immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrooTFsN2GfjbVg90H21EOLACvtfcHk0liviKorqir09w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2lsr4%253D&md5=5a73c10cf6b853bf6f0c1db3ebfe0f01</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F2162402X.2017.1291106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F2162402X.2017.1291106%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DC.%26aulast%3DTalukder%26aufirst%3DA.%26aulast%3DSavage%26aufirst%3DN.%2BM.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DK.%2BB.%26atitle%3DJAK-STAT-mediated%2520chronic%2520inflammation%2520impairscytotoxic%2520T%2520lymphocyte%2520activation%2520to%2520decreaseanti-PD-1%2520immunotherapy%2520efficacy%2520in%2520pancreaticcancer%26jtitle%3DOncoimmunology%26date%3D2017%26volume%3D6%26doi%3D10.1080%2F2162402X.2017.1291106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niidome, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komohara, Y.</span></span> <span> </span><span class="NLM_article-title">STAT3 inhibitor abrogates the expression of PD-1 ligands onlymphoma cell lines</span>. <i>J. Clin. Exp. Hematop.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.3960/jslrt.17006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.3960%2Fjslrt.17006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28496056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC1crmsVKnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=21-25&author=C.+Maauthor=H.+Horladauthor=C.+Panauthor=H.+Yanoauthor=K.+Ohnishiauthor=Y.+Fujiwaraauthor=M.+Matsuokaauthor=A.+Leeauthor=T.+Niidomeauthor=R.+Yamanakaauthor=M.+Takeyaauthor=Y.+Komohara&title=STAT3+inhibitor+abrogates+the+expression+of+PD-1+ligands+onlymphoma+cell+lines&doi=10.3960%2Fjslrt.17006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 inhibitor abrogates the expression of PD-1 ligands on lymphoma cell lines</span></div><div class="casAuthors">Ma Chaoya; Horlad Hasita; Pan Cheng; Yano Hiromu; Ohnishi Koji; Fujiwara Yukio; Takeya Motohiro; Komohara Yoshihiro; Matsuoka Masao; Matsuoka Masao; Lee Aeju; Lee Aeju; Niidome Takuro; Yamanaka Ryuya</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical and experimental hematopathology : JCEH</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent studies have indicated the significance of immune checkpoint molecules including programmed death-1 (PD-1), cytotoxic T-lymphocyte associated protein 4, and T-cell immunoglobulin and mucin domain-containing molecule-3 for anti-tumor immune responses.  We previously investigated PD-1 ligand 1/2 (PD-L1/2) expression in lymphoma cell lines, and found that PD-L1/2 is expressed on the adult T-cell leukemia/lymphoma (ATL-T) and B-cell lymphoma (SLVL) cell lines.  In the present study, we investigated whether the Stat3 inhibitor WP1066 abrogated PD-L1/2 expression in lymphoma cell lines.  Incubation with WP1066 inhibited lymphoma cell growth and induced cell apoptosis.  PD-L1/2 expression in the ATL-T, SLVL, and human brain malignant lymphoma (HKBML) cell lines was significantly abrogated by WP1066 treatment.  These data indicated that a Stat3 inhibitor abrogated PD-L1/2 expression in lymphoma cells.  Such an inhibitor is therefore considered to be useful for additional immunotherapy in patients with advanced lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTsbLe3_wWz3czK395ZObgVfW6udTcc2eYk-j58NyIHTbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crmsVKnsA%253D%253D&md5=ca280f9946b51f6ebc45e5536e51405e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3960%2Fjslrt.17006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3960%252Fjslrt.17006%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHorlad%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DYano%26aufirst%3DH.%26aulast%3DOhnishi%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DMatsuoka%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DNiidome%26aufirst%3DT.%26aulast%3DYamanaka%26aufirst%3DR.%26aulast%3DTakeya%26aufirst%3DM.%26aulast%3DKomohara%26aufirst%3DY.%26atitle%3DSTAT3%2520inhibitor%2520abrogates%2520the%2520expression%2520of%2520PD-1%2520ligands%2520onlymphoma%2520cell%2520lines%26jtitle%3DJ.%2520Clin.%2520Exp.%2520Hematop.%26date%3D2017%26volume%3D57%26spage%3D21%26epage%3D25%26doi%3D10.3960%2Fjslrt.17006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepers, U.</span></span> <span> </span><span class="NLM_article-title">Polyamides as artificial transcription factors: novel tools for molecular medicine?</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2472</span>– <span class="NLM_lpage">2475</span>, <span class="refDoi"> DOI: 10.1002/anie.200301745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Fanie.200301745" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkt1Whsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=2472-2475&author=K.+Schmitzauthor=U.+Schepers&title=Polyamides+as+artificial+transcription+factors%3A+novel+tools+for+molecular+medicine%3F&doi=10.1002%2Fanie.200301745"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of gene expression: Polyamides as artificial transcription factors: novel tools for molecular medicine?</span></div><div class="casAuthors">Schmitz, Katja; Schepers, Ute</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2472-2475</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Misregulation of gene expression often leads to severe defects in the development of an organism or to uncontrolled cell growth.  Artificial transcription factors can control the expression of the genes concerned to restore normal cell regulation.  Dervan et al. have synthesized several DNA-binding polyamides: (TALE (yellow) is a class of homeobox protein) that show potential for the treatment of diseases such as cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocgK5DT4upzbVg90H21EOLACvtfcHk0lgrDOMNUod62Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkt1Whsbg%253D&md5=57a38e41b03467b753b6138b52fa01f4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fanie.200301745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200301745%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitz%26aufirst%3DK.%26aulast%3DSchepers%26aufirst%3DU.%26atitle%3DPolyamides%2520as%2520artificial%2520transcription%2520factors%253A%2520novel%2520tools%2520for%2520molecular%2520medicine%253F%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2004%26volume%3D43%26spage%3D2472%26epage%3D2475%26doi%3D10.1002%2Fanie.200301745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">On the pairing rules for recognition in the minor groove of DNA by pyrrole-imidazole polyamides</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1016/S1074-5521(97)90243-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS1074-5521%2897%2990243-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9281524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1ansrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=569-578&author=S.+Whiteauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=On+the+pairing+rules+for+recognition+in+the+minor+groove+of+DNA+by+pyrrole-imidazole+polyamides&doi=10.1016%2FS1074-5521%2897%2990243-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">On the pairing rules for recognition in the minor groove of DNA by pyrrole-imidazole polyamides</span></div><div class="casAuthors">White, Sarah; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">569-578</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">Cell-permeable small mols. that target predetd. DNA sequences with high affinity and specificity have the potential to control gene expression.  A binary code has been developed to correlate DNA sequence with side-by-side pairings between N-methylpyrrole (Py) and N-methylimidazole (Im) carboxamides in the DNA minor groove.  We set out to det. the relative energetics of pairings of Im/Py, Py/Im, Im/Im, and Py/Py for targeting G·C and A·T base pairs.  A key specificity issue, which has not been previously addressed, is whether an Im/Im pair is energetically equiv. to an Im/Py pair for targeting G·C base pairs.  Equil. assocn. consts. were detd. at two five-base-pair sites for a series of four six-ring hairpin polyamides, to test the relative energetics of the four arom. amino-acid pairings opposite G·C and A·T base pairs in the central position.  We obsd. that a G·C base pair was effectively targeted with Im/Py but not Py/Im, Py/Py, or Im/Im.  The A·T base pair was effectively targeted with Py/Py but not Im/Py, Py/Im, or Im/Im.  An Im/Im pairing is energetically disfavored for the recognition of both A·T and G·C.  This specificity will create important limitations on undesirable slipped motifs that are available for unlinked dimers in the minor groove.  Baseline energetic parameters will thus be created which, using the predictability of the current pairing rules for specific mol. recognition of double-helical DNA, will guide further second-generation polyamide design for DNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS4iTwSAZYNrVg90H21EOLACvtfcHk0lgrDOMNUod62Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1ansrY%253D&md5=d9d60d5f85ef7c4d8d5038f744989467</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2897%2990243-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252897%252990243-X%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DOn%2520the%2520pairing%2520rules%2520for%2520recognition%2520in%2520the%2520minor%2520groove%2520of%2520DNA%2520by%2520pyrrole-imidazole%2520polyamides%26jtitle%3DChem.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D569%26epage%3D578%26doi%3D10.1016%2FS1074-5521%2897%2990243-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wade, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrksich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Design of peptides that bind in the minor groove of DNA at 5′-(A,T)5G(A,T)C(A,T) sequences by a dimeric side-by-side motif</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">8783</span>– <span class="NLM_lpage">8794</span>, <span class="refDoi"> DOI: 10.1021/ja00049a006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00049a006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK38XmtlCmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1992&pages=8783-8794&author=W.+S.+Wadeauthor=M.+Mrksichauthor=P.+B.+Dervan&title=Design+of+peptides+that+bind+in+the+minor+groove+of+DNA+at+5%E2%80%B2-%28A%2CT%295G%28A%2CT%29C%28A%2CT%29+sequences+by+a+dimeric+side-by-side+motif&doi=10.1021%2Fja00049a006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design of peptides that bind in the minor groove of DNA at 5'-(A,T)G(A,T)C(A,T)-3' sequences by a dimeric side-by-side motif</span></div><div class="casAuthors">Wade, Warren S.; Mrksich, Milan; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8783-94</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The designed peptides pyridine-2-carboxamide-netropsin (2-PyN) (I) and 1-methylimidazole-2-carboxamide-netropsin (2-ImN) (II) are crescent-shaped synthetic analogs of the natural products netropsin and distamycin A.  Footprinting expts. indicate that 2-PyN and 2-ImN bind the 5 base pair sequence 5'-TGTCA-3'.  Affinity cleaving data suggest that the complexes, 2-ImN.5'-TGTCA-3' and 2-PyN.5'-TGTCA-3', are composed of two equiv. orientations which disfavor a 1:1 model.  The footprinting and affinity cleaving data are in accord with a 2:1 complex where a novel side-by-side antiparallel dimer binds in the minor groove of double-helical DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8zsoompOt6rVg90H21EOLACvtfcHk0li8dEehpIVQ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XmtlCmsL4%253D&md5=b7823a69acc54a2d722927c873a8aedf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja00049a006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00049a006%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DW.%2BS.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DDesign%2520of%2520peptides%2520that%2520bind%2520in%2520the%2520minor%2520groove%2520of%2520DNA%2520at%25205%25E2%2580%25B2-%2528A%252CT%25295G%2528A%252CT%2529C%2528A%252CT%2529%2520sequences%2520by%2520a%2520dimeric%2520side-by-side%2520motif%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1992%26volume%3D114%26spage%3D8783%26epage%3D8794%26doi%3D10.1021%2Fja00049a006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melander, C.</span></span> <span> </span><span class="NLM_article-title">Programmable DNA-binding small molecules</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6101</span>– <span class="NLM_lpage">6114</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.bmc.2013.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=23665141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVGmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6101-6114&author=M.+S.+Blackledgeauthor=C.+Melander&title=Programmable+DNA-binding+small+molecules&doi=10.1016%2Fj.bmc.2013.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Programmable DNA-binding small molecules</span></div><div class="casAuthors">Blackledge, Meghan S.; Melander, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6101-6114</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Aberrant gene expression is responsible for a myriad of human diseases from infectious diseases to cancer.  Precise regulation of these genes via specific interactions with the DNA double helix could pave the way for novel therapeutics.  Pyrrole-imidazole polyamides are small mols. capable of binding to pre-detd. DNA sequences up to 16 base pairs with affinity and specificity comparable to natural transcription factors.  In the three decades since their development, great strides have been made relating to synthetic accessibility and improved sequence specificity and binding affinity.  This perspective presents a brief history of early seminal developments in the field and highlights recent reports of the utility of polyamides as both genetic modulators and mol. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnyvt5n21h47Vg90H21EOLACvtfcHk0li8dEehpIVQ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVGmurk%253D&md5=75d1c7ab474cde9bbfd15365f798ee24</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DBlackledge%26aufirst%3DM.%2BS.%26aulast%3DMelander%26aufirst%3DC.%26atitle%3DProgrammable%2520DNA-binding%2520small%2520molecules%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6101%26epage%3D6114%26doi%3D10.1016%2Fj.bmc.2013.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottesfeld, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neely, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauger, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Regulation of gene expression by small molecules</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1038/387202a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2F387202a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9144294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK2sXjtF2msrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=1997&pages=202-205&author=J.+M.+Gottesfeldauthor=L.+Neelyauthor=J.+W.+Traugerauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=Regulation+of+gene+expression+by+small+molecules&doi=10.1038%2F387202a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of gene expression by small molecules</span></div><div class="casAuthors">Gottesfeld, Joel M.; Neely, Laura; Trauger, John W.; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">6629</span>),
    <span class="NLM_cas:pages">202-205</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Small mols. that target specific DNA sequences have the potential to control gene expression.  Ligands designed for therapeutic application must bind any predetd. DNA sequence with high affinity and permeate living cells.  Synthetic polyamides contg. N-methylimidazole and N-methylpyrrole amino acids have an affinity and specificity for DNA comparable to naturally occurring DNA-binding proteins1.  We report here that an eight-ring polyamide targeted to a specific region of the transcription factor TFIIIA binding site interferes with 5S RNA gene expression in Xenopus kidney cells.  Our results indicate that pyrrole-imidazole polyamides are cell-permeable and can inhibit the transcription of specific genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUDTLP-1ZKgLVg90H21EOLACvtfcHk0li8dEehpIVQ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjtF2msrg%253D&md5=f3ebbe7bea63c4674894bb38550defce</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2F387202a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F387202a0%26sid%3Dliteratum%253Aachs%26aulast%3DGottesfeld%26aufirst%3DJ.%2BM.%26aulast%3DNeely%26aufirst%3DL.%26aulast%3DTrauger%26aufirst%3DJ.%2BW.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DRegulation%2520of%2520gene%2520expression%2520by%2520small%2520molecules%26jtitle%3DNature%26date%3D1997%26volume%3D387%26spage%3D202%26epage%3D205%26doi%3D10.1038%2F387202a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trauger, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Recognition of DNA by designed ligands at subnanomolar concentrations</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">561</span>, <span class="refDoi"> DOI: 10.1038/382559a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2F382559a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=8700233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK28XkvFGjtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=559-561&author=J.+W.+Traugerauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=Recognition+of+DNA+by+designed+ligands+at+subnanomolar+concentrations&doi=10.1038%2F382559a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of DNA by designed ligands at subnanomolar concentrations</span></div><div class="casAuthors">Trauger, John W.; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">6591</span>),
    <span class="NLM_cas:pages">559-561</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Small mols. that specifically bind with high affinity to any predetd. DNA sequence in the human genome would be useful tools in mol. biol. and potentially in human medicine.  Simple rules have been developed to control rationally the sequence specificity of minor-groove-binding polyamides contg. N-methylimidazole and N-methylpyrrole amino acids.  Two eight-ring pyrrole-imidazole polyamides differing in sequence by a single amino acid bind specifically to resp. six-base-pair target sites which differ in sequence by a single base pair.  Binding is obsd. at subnanomolar concns. of ligand.  The replacement of a single nitrogen atom with a C-H regulates affinity and specificity by two orders of magnitude.  The broad range of sequences that can be specifically targeted with pyrrole-imidazole polyamides, coupled with an efficient solid-phase synthesis methodol., identify a powerful class of small mols. for sequence-specific recognition of double-helical DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0_GQE1-LS77Vg90H21EOLACvtfcHk0li8dEehpIVQ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XkvFGjtLw%253D&md5=6b6c02141026961f8216226cf0afac6e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2F382559a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F382559a0%26sid%3Dliteratum%253Aachs%26aulast%3DTrauger%26aufirst%3DJ.%2BW.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DRecognition%2520of%2520DNA%2520by%2520designed%2520ligands%2520at%2520subnanomolar%2520concentrations%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D559%26epage%3D561%26doi%3D10.1038%2F382559a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Recognition of the four Watson-Crick base pairs in the DNA minor groove by synthetic ligands</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">468</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1038/35106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2F35106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9461213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1cXhtVemt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1998&pages=468-471&author=S.+Whiteauthor=J.+W.+Szewczykauthor=J.+M.+Turnerauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=Recognition+of+the+four+Watson-Crick+base+pairs+in+the+DNA+minor+groove+by+synthetic+ligands&doi=10.1038%2F35106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of the four Watson-Crick base pairs in the DNA minor groove by synthetic ligands</span></div><div class="casAuthors">White, Sarah; Szewczyk, Jason W.; Turner, James M.; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">6666</span>),
    <span class="NLM_cas:pages">468-471</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The design of synthetic ligands that read the information stored in the DNA double helix has been a long-standing goal at the interface of chem. and biol.  Cell-permeable small mols. that target predetd. DNA sequences offer a potential approach for the regulation of gene expression.  Oligodeoxynucleotides that recognize the major groove of double-helical DNA via triple-helix formation bind to a broad range of sequences with high affinity and specificity.  Although oligonucleotides and their analogs have been shown to interfere with gene expression, the triple-helix approach is limited to recognition of purines and suffers from poor cellular uptake.  The subsequent development of pairing rules for minor-groove binding polyamides contg. pyrrole (Py) and imidazole (Im) amino acids offers a second code to control sequence specificity.  An Im/Py pair distinguishes G⪆C from C⪆G and both of these from A·T/T·A base pairs.  A Py/Py pair specifies A,T from G,C but does not distinguish A·T from T·A.  To break this degeneracy, we have added a new arom. amino acid, 3-hydroxypyrrole (Hp), to the repertoire to test for pairings that discriminate A·T from T⪆a.  We find that replacement of a single hydrogen atom with a hydroxy group in a Hp/Py pairing regulates affinity and specificity by an order of magnitude.  By incorporation of this third amino acid, hydroxypyrrole-imidazole-pyrrole polyamides form four ring-pairings (Im/Py, Py/Im, Hp/Py and Py/Hp) which distinguish all Four Watson-Crick base pairs in the minor groove of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg-5qvdXyBvbVg90H21EOLACvtfcHk0lgILX_JmxUWjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtVemt78%253D&md5=38aa613c6a8afc2b933333f8779c392e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F35106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35106%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DJ.%2BW.%26aulast%3DTurner%26aufirst%3DJ.%2BM.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DRecognition%2520of%2520the%2520four%2520Watson-Crick%2520base%2520pairs%2520in%2520the%2520DNA%2520minor%2520groove%2520by%2520synthetic%2520ligands%26jtitle%3DNature%26date%3D1998%26volume%3D391%26spage%3D468%26epage%3D471%26doi%3D10.1038%2F35106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kielkopf, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">A structural basis for recognition of A·T and T·A base pairs in the minor groove of B-DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1126/science.282.5386.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1126%2Fscience.282.5386.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9756473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1cXmsFOjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1998&pages=111-115&author=C.+L.+Kielkopfauthor=S.+Whiteauthor=J.+W.+Szewczykauthor=J.+M.+Turnerauthor=E.+E.+Bairdauthor=P.+B.+Dervanauthor=D.+C.+Rees&title=A+structural+basis+for+recognition+of+A%C2%B7T+and+T%C2%B7A+base+pairs+in+the+minor+groove+of+B-DNA&doi=10.1126%2Fscience.282.5386.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A structural basis for recognition of A·T and T·A base pairs in the minor groove of B-DNA</span></div><div class="casAuthors">Kielkopf, Clara L.; White, Sarah; Sewczyk, Jason W.; Turner, James M.; Baird, Eldon E.; Dervan, Peter B.; Rees, Douglas C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">5386</span>),
    <span class="NLM_cas:pages">111-115</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Polyamide dimers contg. three types of arom. rings-pyrrole, imidazole, and hydroxypyrrole-afford a small-mol. recognition code that discriminates among all four Watson-Crick base pairs in the minor groove.  The crystal structure of a specific polyamide dimer-DNA complex establishes the structural basis for distinguishing T·A from A·T base pairs.  Specificity for the T·A base pair is achieved by means of distinct hydrogenbonds between pairs of substituted pyrroles on the ligand and the O2 of thymine and N3 of adenine.  In addn., shape-selective recognition of an asym. cleft between the thymine-O2 and the adenine-C2 was obsd.  Although hitherto similarities among the base pairs in the minor groove have been emphasized, the structure illustrates differences that allow specific minor groove recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuMGOkDl_rxbVg90H21EOLACvtfcHk0lgILX_JmxUWjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsFOjtbc%253D&md5=934706075f158b01006a993c81c17e14</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fscience.282.5386.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.282.5386.111%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DJ.%2BW.%26aulast%3DTurner%26aufirst%3DJ.%2BM.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DA%2520structural%2520basis%2520for%2520recognition%2520of%2520A%25C2%25B7T%2520and%2520T%25C2%25B7A%2520base%2520pairs%2520in%2520the%2520minor%2520groove%2520of%2520B-DNA%26jtitle%3DScience%26date%3D1998%26volume%3D282%26spage%3D111%26epage%3D115%26doi%3D10.1126%2Fscience.282.5386.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelson, B. S.</span></span> <span> </span><span class="NLM_article-title">Recognition of the DNA minor groove by pyrrloe-imidazole polyamides</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(03)00081-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS0959-440X%2803%2900081-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=12831879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvVKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=284-299&author=P.+B.+Dervanauthor=B.+S.+Edelson&title=Recognition+of+the+DNA+minor+groove+by+pyrrloe-imidazole+polyamides&doi=10.1016%2FS0959-440X%2803%2900081-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of the DNA minor groove by pyrrole-imidazole polyamides</span></div><div class="casAuthors">Dervan, Peter B.; Edelson, Benjamin S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">284-299</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Many diseases, such as cancer, are related to aberrant gene expression.  Regulating transcription by chem. methods could be important in human medicine.  Minor groove-binding polyamides offer one chem. approach to DNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ymr2dTEUT7Vg90H21EOLACvtfcHk0lgILX_JmxUWjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvVKnsrw%253D&md5=9f0a7b75f02d63ef70b61994acbe87d0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2803%2900081-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252803%252900081-2%26sid%3Dliteratum%253Aachs%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DEdelson%26aufirst%3DB.%2BS.%26atitle%3DRecognition%2520of%2520the%2520DNA%2520minor%2520groove%2520by%2520pyrrloe-imidazole%2520polyamides%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2003%26volume%3D13%26spage%3D284%26epage%3D299%26doi%3D10.1016%2FS0959-440X%2803%2900081-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span> <span> </span><span class="NLM_article-title">Discrimination of hairpin polyamides with an alpha-substituted-gamma-aminobutyric acid as a TG-reader in DNA minor groove</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">8766</span>– <span class="NLM_lpage">8776</span>, <span class="refDoi"> DOI: 10.1021/ja0580587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0580587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtVSisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=8766-8776&author=W.+Zhangauthor=T.+Bandoauthor=H.+Sugiyama&title=Discrimination+of+hairpin+polyamides+with+an+alpha-substituted-gamma-aminobutyric+acid+as+a+TG-reader+in+DNA+minor+groove&doi=10.1021%2Fja0580587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discrimination of Hairpin Polyamides with an α-Substituted-γ-aminobutyric Acid as a 5'-TG-3' Reader in DNA Minor Groove</span></div><div class="casAuthors">Zhang, Wen; Bando, Toshikazu; Sugiyama, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8766-8776</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pyrrole-imidazole (Py-Im) polyamides contg. stereospecifically α-amino- or α-hydroxyl-substituted γ-aminobutyric acid as a 5'-TG-3' recognition element were synthesized by machine-assisted Fmoc solid-phase synthesis.  Their binding properties to predetd. DNA sequences contg. a core binding site of 5'-TGCNCA-3'/3'-ACGN'GT-5' (N·N' = A·T, T·A, G·C, and C·G) were then systematically studied by surface plasmon resonance (SPR).  SPR results revealed that the pairing of stereospecifically α-amino-/α-hydroxyl-substituted γ-aminobutyric acids, (R or S)-α,γ-diaminobutyric acid (γRN or γSN) and (R or S)-α-hydroxyl-γ-aminobutyric acid (γRO or γSO), side-by-side with β-alanine (β) in such polyamides significantly influenced the DNA binding affinity and recognition specificity of hairpin polyamides in the DNA minor groove compared with β/β, β/γ, and γ/β pairings.  More importantly, the polyamide Ac-Im-γSO-ImPy-γ-ImPyβPy-β-Dp (β/γSO) favorably binds to a hairpin DNA contg. a core binding site of 5'-TGCNCA-3'/3'-ACGN'GT-5' (N·N' = A·T) with dissocn. equil. const. (KD) of 1.9×10-7 M over N·N' = T·A with KD = 3.7×10-6 M, with a 19-fold specificity.  By contrast, Ac-Im-γSN-ImPy-γ-ImPyβPy-β-Dp (β/γSN) binds to the above sequence with N·N' = A·T with KD = 8.7×10-7 M over N·N' = T·A with KD = 8.4×10-6 M, with a 9.6-fold specificity.  The results also show that the stereochem. of the α-substituent, as well as the α-substituent itself may greatly alter binding affinity and recognition selectivity of hairpin polyamides to different DNA sequences.  Further, we carried out mol. modeling studies on the binding by an energy minimization method, suggesting that α-hydroxyl is very close to N3 of the 3'-terminal G to induce the formation of hydrogen bonding between hydroxyl and N3 in the recognition event of the polyamide Ac-Im-γSO-ImPy-γ-ImPyβPy-β-Dp (β/γSO) to 5'-TGCNCA-3'/3'-ACGN'GT-5' (N·N' = A·T).  Therefore, SPR assays and mol. modeling studies collectively suggest that the (S)-α-hydroxyl-γ-aminobutyric acid (γSO) may act as a 5'-TG-3' recognition unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqusw2YO4yGrVg90H21EOLACvtfcHk0lillqahihV0OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtVSisb8%253D&md5=1fbbfdc9dc4d4db2a7bff0f84cf2c1f2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja0580587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0580587%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26atitle%3DDiscrimination%2520of%2520hairpin%2520polyamides%2520with%2520an%2520alpha-substituted-gamma-aminobutyric%2520acid%2520as%2520a%2520TG-reader%2520in%2520DNA%2520minor%2520groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D8766%26epage%3D8776%26doi%3D10.1021%2Fja0580587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Discrimination between T/A and A/T base pairs of pyrrole-imidazole polyamides substituted with chiral β-hydroxy-γ-aminobutyric acid/β-alanine pairs</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Fcbic.201100675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22113893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFaltLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=47-50&author=W.+Zhangauthor=S.+Jiangauthor=Y.+Wuauthor=C.+Guoauthor=H.+Zhangauthor=H.+Sugiyamaauthor=X.+Chen&title=Discrimination+between+T%2FA+and+A%2FT+base+pairs+of+pyrrole-imidazole+polyamides+substituted+with+chiral+%CE%B2-hydroxy-%CE%B3-aminobutyric+acid%2F%CE%B2-alanine+pairs&doi=10.1002%2Fcbic.201100675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discrimination between T/A and A/T Base Pairs of Pyrrole-Imidazole Polyamides Substituted with Chiral β-Hydroxy-γ-Aminobutyric Acid/β-Alanine Pairs</span></div><div class="casAuthors">Zhang, Wen; Jiang, Shi-Kun; Wu, Yan-Ling; Guo, Chuan-Xin; Zhang, Hong-Fei; Sugiyama, Hiroshi; Chen, Xing-Lai</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-50</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors synthesized five novel pyrrole-imidazole polyamides 1-5 contg. chiral β-hydroxy-γ-aminobutyric acid/β-alanine pairs, and examd. their selective recognition ability with a β/β pairing to Watson-Crick base pairs T·A, A·T, C·G, G·C in the DNA minor groove.  It was obsd. that no binding to 5'-TCCNCCA-3'/5'-TGGN'GGA-3' (N·N' = C·G, G·C) was obsd. at low concns. of polyamides 1-5, but there was large nonspecific binding at higher concns.  The authors obtained binding consts. for assocn. of 1-5 to 5'-TCCNCCA-3'/5'-TGGN'GGA-3' (N·N' = A·T, T·A), in addn. to thermodn. std. free-energy change (ΔG°), hydrogen bonding length, and interaction energy upon forming of the polyamide-DNA complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM6YJ1lmuGH7Vg90H21EOLACvtfcHk0lillqahihV0OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFaltLzE&md5=30e635f52d94d3b114d5cf47d90acb8b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100675%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DDiscrimination%2520between%2520T%252FA%2520and%2520A%252FT%2520base%2520pairs%2520of%2520pyrrole-imidazole%2520polyamides%2520substituted%2520with%2520chiral%2520%25CE%25B2-hydroxy-%25CE%25B3-aminobutyric%2520acid%252F%25CE%25B2-alanine%2520pairs%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D47%26epage%3D50%26doi%3D10.1002%2Fcbic.201100675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulin-Kerstien, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fechter, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelson, B. S.</span></span> <span> </span><span class="NLM_article-title">Regulation of gene expression by synthetic DNA-binding ligands</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1007/b100440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1007%2Fb100440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsF2isbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=2005&pages=1-31&author=P.+B.+Dervanauthor=A.+T.+Poulin-Kerstienauthor=E.+J.+Fechterauthor=B.+S.+Edelson&title=Regulation+of+gene+expression+by+synthetic+DNA-binding+ligands&doi=10.1007%2Fb100440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of gene expression by synthetic DNA-binding ligands</span></div><div class="casAuthors">Dervan, Peter B.; Poulin-Kerstien, Adam T.; Fechter, Eric J.; Edelson, Benjamin S.</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">DNA Binders and Related Subjects</span>),
    <span class="NLM_cas:pages">1-31</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">During the past 20 years, polyamides have evolved from the natural product distamycin to a new class of programmable heterocyclic oligomers that bind a broad repertoire of DNA sequences with high affinity and specificity.  This chapter details recent advances in this field of research, focusing on mol. recognition of DNA, and biol. applications such as modulating gene expression by small mols.  Work presented here represents efforts towards the modulation of specific cellular function by small mols. in an addressable fashion within the context of live cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEXxLDeer5krVg90H21EOLACvtfcHk0lillqahihV0OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsF2isbs%253D&md5=2a8f54b687c72f0c8e61138445b79b38</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fb100440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fb100440%26sid%3Dliteratum%253Aachs%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DPoulin-Kerstien%26aufirst%3DA.%2BT.%26aulast%3DFechter%26aufirst%3DE.%2BJ.%26aulast%3DEdelson%26aufirst%3DB.%2BS.%26atitle%3DRegulation%2520of%2520gene%2520expression%2520by%2520synthetic%2520DNA-binding%2520ligands%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2005%26volume%3D253%26spage%3D1%26epage%3D31%26doi%3D10.1007%2Fb100440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahira, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugishima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serie, K.</span></span> <span> </span><span class="NLM_article-title">Development of gene silencing pyrrole-imidazolepolyamide targeting the TGF-β1 promoter for treatment of progressive renal diseases</span>. <i>J. Am. Soc. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1681/ASN.2005060650</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1681%2FASN.2005060650" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=16371433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FksFOnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2006&pages=422-432&author=H.+Matsudaauthor=N.+Fukudaauthor=T.+Uenoauthor=Y.+Tahiraauthor=H.+Ayameauthor=W.+Zhangauthor=T.+Bandoauthor=H.+Sugiyamaauthor=S.+Saitoauthor=K.+Matsumotoauthor=H.+Mugishimaauthor=K.+Serie&title=Development+of+gene+silencing+pyrrole-imidazolepolyamide+targeting+the+TGF-%CE%B21+promoter+for+treatment+of+progressive+renal+diseases&doi=10.1681%2FASN.2005060650"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of gene silencing pyrrole-imidazole polyamide targeting the TGF-beta1 promoter for treatment of progressive renal diseases</span></div><div class="casAuthors">Matsuda Hiroyuki; Fukuda Noboru; Ueno Takahiro; Tahira Yoshiko; Ayame Hirohito; Zhang Wen; Bando Toshikazu; Sugiyama Hiroshi; Saito Satoshi; Matsumoto Koichi; Mugishima Hideo; Serie Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society of Nephrology : JASN</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-32</span>
        ISSN:<span class="NLM_cas:issn">1046-6673</span>.
    </div><div class="casAbstract">Pyrrole-imidazole (Py-Im) polyamides are nuclease-resistant novel compounds that inhibit gene expression by binding to the minor groove of DNA.  A Py-Im polyamide that targets rat TGF-beta1 was designed as a gene-silencing agent for progressive renal diseases, and the distribution and the effects of this polyamide on renal injury were examined in Dahl-salt sensitive (Dahl-S) rats.  For identification of transcription factor binding elements for activation of the rat TGF-beta1 gene, recombinant TGF-beta1 reporter plasmids were transfected into HEK-293 cells, and promoter activity was measured.  Py-Im polyamide was designed to the activator protein-1 binding site of the rat TGF-beta1 promoter.  This Py-Im polyamide showed strong, fast, and specific binding to the target DNA in gel mobility shift and Biacore assays.  Py-Im polyamide significantly inhibited TGF-beta1 promoter activity and expression of TGF-beta1 mRNA and protein in rat mesangial cells.  Intravenously administered fluorescein-labeled polyamide distributed to the kidney of rats.  Py-Im polyamide significantly inhibited expression of TGF-beta1 mRNA and protein in the renal cortex of Dahl-S rats and reduced the increase in urinary protein and albumin in Dahl-S rats independent of changes in blood pressure.  These results indicate that Py-Im polyamide that targets TGF-beta1 will be a novel gene-silencing agent for the TGF-beta1-associated diseases, including progressive renal diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSydrr4CeAj4vxQvRxYU81tfW6udTcc2ebiBZYeuH-HZbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28%252FksFOnsw%253D%253D&md5=b566cddae3baecfd8d9413a546d96266</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1681%2FASN.2005060650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1681%252FASN.2005060650%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DH.%26aulast%3DFukuda%26aufirst%3DN.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DTahira%26aufirst%3DY.%26aulast%3DAyame%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DSaito%26aufirst%3DS.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DMugishima%26aufirst%3DH.%26aulast%3DSerie%26aufirst%3DK.%26atitle%3DDevelopment%2520of%2520gene%2520silencing%2520pyrrole-imidazolepolyamide%2520targeting%2520the%2520TGF-%25CE%25B21%2520promoter%2520for%2520treatment%2520of%2520progressive%2520renal%2520diseases%26jtitle%3DJ.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2006%26volume%3D17%26spage%3D422%26epage%3D432%26doi%3D10.1681%2FASN.2005060650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Washio, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terui, T.</span></span> <span> </span><span class="NLM_article-title">Transcriptional inhibition of hypertrophicscars by a gene silencer, pyrrole-imidazole polyamide, targeting the TGF-β1 promoter</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">1987</span>– <span class="NLM_lpage">1995</span>, <span class="refDoi"> DOI: 10.1038/jid.2011.150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fjid.2011.150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21654833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOls7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2011&pages=1987-1995&author=H.+Washioauthor=N.+Fukudaauthor=H.+Matsudaauthor=H.+Nagaseauthor=T.+Watanabeauthor=Y.+Matsumotoauthor=T.+Terui&title=Transcriptional+inhibition+of+hypertrophicscars+by+a+gene+silencer%2C+pyrrole-imidazole+polyamide%2C+targeting+the+TGF-%CE%B21+promoter&doi=10.1038%2Fjid.2011.150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Inhibition of Hypertrophic Scars by a Gene Silencer, Pyrrole-Imidazole Polyamide, Targeting the TGF-β1 Promoter</span></div><div class="casAuthors">Washio, Hisayo; Fukuda, Noboru; Matsuda, Hiroyuki; Nagase, Hiroki; Watanabe, Takayoshi; Matsumoto, Yoshiaki; Terui, Tadashi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1987-1995</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Synthetic pyrrole-imidazole (PI) polyamides bind to the minor groove of double-helical DNA with high affinity and specificity, and inhibit the transcription of corresponding genes.  The authors examd. the effects of a transforming growth factor (TGF)-β1-targeted PI polyamide (Polyamide) on hypertrophic skin scars in rats.  Hypertrophic scars were created dorsally in rats by incisions.  FITC-labeled Polyamide was injected to investigate its distribution in the skin.  Expression of TGF-β1, connective tissue growth factor (CTGF), collagen type 1, and fibronectin mRNAs was evaluated by reverse transcription PCR anal.  The extent of fibrosis and the expression of TGF-β1 were evaluated histol. and immunohistochem.  Polyamide was distributed in almost all nuclei of skin cells.  Expression of TGF-β1 mRNA reached a peak at 3 days after skin incision.  Expression of CTGF and extracellular matrix mRNAs was increased continuously even after the peak induction of TGF-β1 mRNA.  Injection of Polyamide completely inhibited both the development of scars and the induction of growth factors and extracellular matrix mRNAs.  The treatment also markedly inhibited fibrotic changes and reduced the nos. of vimentin-pos. spindle-shaped fibroblasts.  Injection of Polyamide also reduced established hypertrophic scars in rats.  Thus, TGF-β1-targeted PI polyamide should be a feasible gene silencer for hypertrophic scars and keloids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1TvKWk0ahILVg90H21EOLACvtfcHk0lhOEWnwTau4cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOls7vM&md5=caadc49f10307d093967a6fb3c6564ce</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fjid.2011.150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2011.150%26sid%3Dliteratum%253Aachs%26aulast%3DWashio%26aufirst%3DH.%26aulast%3DFukuda%26aufirst%3DN.%26aulast%3DMatsuda%26aufirst%3DH.%26aulast%3DNagase%26aufirst%3DH.%26aulast%3DWatanabe%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DTerui%26aufirst%3DT.%26atitle%3DTranscriptional%2520inhibition%2520of%2520hypertrophicscars%2520by%2520a%2520gene%2520silencer%252C%2520pyrrole-imidazole%2520polyamide%252C%2520targeting%2520the%2520TGF-%25CE%25B21%2520promoter%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2011%26volume%3D131%26spage%3D1987%26epage%3D1995%26doi%3D10.1038%2Fjid.2011.150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soma, M.</span></span> <span> </span><span class="NLM_article-title">Preclinical Study of novel gene silencer Pyrrole-Imidazole Polyamide targeting human TGF-β1 promoter for hypertrophic scars in a common marmoset primate model</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0125295</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0125295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1371%2Fjournal.pone.0125295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25938472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWqtbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=J.+Igarashiauthor=N.+Fukudaauthor=T.+Inoueauthor=S.+Nakaiauthor=K.+Saitoauthor=K.+Fujiwaraauthor=H.+Matsudaauthor=T.+Uenoauthor=Y.+Matsumotoauthor=T.+Watanabeauthor=H.+Nagaseauthor=T.+Bandoauthor=H.+Sugiyamaauthor=T.+Itohauthor=M.+Soma&title=Preclinical+Study+of+novel+gene+silencer+Pyrrole-Imidazole+Polyamide+targeting+human+TGF-%CE%B21+promoter+for+hypertrophic+scars+in+a+common+marmoset+primate+model&doi=10.1371%2Fjournal.pone.0125295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical study of novel gene silencer pyrrole-imidazole polyamide targeting human TGF-β1 promoter for hypertrophic scars in a common marmoset primate model</span></div><div class="casAuthors">Igarashi, Jun; Fukuda, Noboru; Inoue, Takashi; Nakai, Shigeki; Saito, Kosuke; Fujiwara, Kyoko; Matsuda, Hiroyuki; Ueno, Takahiro; Matsumoto, Yoshiaki; Watanabe, Takayoshi; Nagase, Hiroki; Bando, Toshikazu; Sugiyama, Hiroshi; Itoh, Toshio; Soma, Masayoshi</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e0125295/1-e0125295/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">We report a preclin. study of a pyrrole-imidazole (PI) polyamide that targets the human transforming growth factor (hTGF)-β1 gene as a novel transcriptional gene silencer in a common marmoset primate model.  We designed and then synthesized PI polyamides to target the hTGF-β1 promoter.  We examd. effects of seven PI polyamides (GB1101-1107) on the expression of hTGF-β1 mRNA stimulated with phorbol 12-myristate 13-acetate (PMA) in human vascular smooth muscle cells.  GB1101, GB1105 and GB1106 significantly inhibited hTGF-β1 mRNA expression.  We examd. GB1101 as a PI polyamide to hTGF-β1 for hypertrophic scars in marmosets in vivo.  Injection of GB1101 completely inhibited hypertrophic scar formation at 35 days post-incision and inhibited cellular infiltration, TGF-β1 and vimentin staining, and epidermal thickness.  Mismatch polyamide did not affect hypertrophic scarring or histol. changes.  Epidermis was significantly thinner with GB1101 than with water and mismatch PI polyamides.  We developed the PI polyamides for practical ointment medicines for the treatment of hypertrophic scars.  FITC-labeled GB1101 with solbase most efficiently distributed in the nuclei of epidermal keratinocytes, completely suppressed hypertropic scarring at 42 days after incision, and considerably inhibited epidermal thickness and vimentin-pos. fibroblasts.  PI polyamides targeting hTGF-β1 promoter with solbase ointment will be practical medicines for treating hypertrophic scars after surgical operations and skin burns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi6uMbHEPZ2rVg90H21EOLACvtfcHk0lhOEWnwTau4cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWqtbvE&md5=f42751d121a52d415f17739096c15ade</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0125295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0125295%26sid%3Dliteratum%253Aachs%26aulast%3DIgarashi%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DN.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DNakai%26aufirst%3DS.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DK.%26aulast%3DMatsuda%26aufirst%3DH.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DWatanabe%26aufirst%3DT.%26aulast%3DNagase%26aufirst%3DH.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DSoma%26aufirst%3DM.%26atitle%3DPreclinical%2520Study%2520of%2520novel%2520gene%2520silencer%2520Pyrrole-Imidazole%2520Polyamide%2520targeting%2520human%2520TGF-%25CE%25B21%2520promoter%2520for%2520hypertrophic%2520scars%2520in%2520a%2520common%2520marmoset%2520primate%2520model%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0125295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, M. HR.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">DNA binding and nuclease activity of a water-soluble sulfonated manganese(III) corrole</span>. <i>Transition Met. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1007/s11243-013-9689-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1007%2Fs11243-013-9689-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVOnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=283-289&author=J.+Huangauthor=X.+Wangauthor=Y.+Zhangauthor=M.+HR.+Mahmoodauthor=Y.+Huangauthor=X.+Yingauthor=L.+Jiauthor=H.+Liu&title=DNA+binding+and+nuclease+activity+of+a+water-soluble+sulfonated+manganese%28III%29+corrole&doi=10.1007%2Fs11243-013-9689-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binding and nuclease activity of a water-soluble sulfonated manganese(III) corrole</span></div><div class="casAuthors">Huang, Jun-Teng; Wang, Xiang-Li; Zhang, Yang; Mahmood, Mian H. R.; Huang, Yan-Yuan; Ying, Xiao; Ji, Liang-Nian; Liu, Hai-Yang</div><div class="citationInfo"><span class="NLM_cas:title">Transition Metal Chemistry (Dordrecht, Netherlands)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-289</span>CODEN:
                <span class="NLM_cas:coden">TMCHDN</span>;
        ISSN:<span class="NLM_cas:issn">0340-4285</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The interaction of a water-sol. sulfonated Mn(III) corrole Mn(tpfc)(SO3Na)2 (3) [tpfc = 5,10,15-tris(pentafluorophenyl)corrole] with calf thymus DNA (ct-DNA) was studied by spectroscopic methods, and the nuclease activity of this complex also was examd. by agarose gel electrophoresis.  Mn(tpfc)(SO3Na)2 (3) exhibits weak aggregation tendency in buffer soln. and can bind to ct-DNA via an outside binding mode with a binding const. of 1.25 × 104 M-1.  The obsd. increase in Stern-Volmer quenching const. with increasing temp. indicates that the competition of the manganese corrole and ethidium bromide with ct-DNA is a dynamic process.  Moreover, the manganese corrole displays good chem. nuclease activity in the presence of hydrogen peroxide via oxidative cleavage of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOfDuwXgl_rrVg90H21EOLACvtfcHk0lgGjFivGbStww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVOnur8%253D&md5=dee78125bb327465806f2e276748320f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs11243-013-9689-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11243-013-9689-5%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMahmood%26aufirst%3DM.%2BHR.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DYing%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDNA%2520binding%2520and%2520nuclease%2520activity%2520of%2520a%2520water-soluble%2520sulfonated%2520manganese%2528III%2529%2520corrole%26jtitle%3DTransition%2520Met.%2520Chem.%26date%3D2013%26volume%3D38%26spage%3D283%26epage%3D289%26doi%3D10.1007%2Fs11243-013-9689-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, A.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span> <span> </span><span class="NLM_article-title">Following G-quartet formation by UV-spectroscopy</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(98)01043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS0014-5793%2898%2901043-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9755862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1cXmtlSmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=1998&pages=74-78&author=J.-L.+Mergnyauthor=A.-T.+Phanauthor=L.+Lacroix&title=Following+G-quartet+formation+by+UV-spectroscopy&doi=10.1016%2FS0014-5793%2898%2901043-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Following G-quartet formation by UV-spectroscopy</span></div><div class="casAuthors">Mergny, Jean-Louis; Phan, Anh-Tuan; Lacroix, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-78</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Oligodeoxynucleotides which include stretches of guanines form a well-known tetrameric structure.  We show that the recording of reversible absorbance changes at 295 nm allows to precisely monitor intramol. guanine (G)-quartet formation and dissocn.  Accurate Tm and thermodn. values could be easily extd. from the data, whereas classical recordings at 260 nm led to a much larger uncertainty and in extreme cases, to completely inaccurate measurements.  This inverted denaturation profile was obsd. for all G-quartet-forming oligonucleotides studied so far.  This technique is very useful in all cases where intramol. or intermol. quadruplex formation is suspected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_txGs3IpZNrVg90H21EOLACvtfcHk0lgGjFivGbStww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtlSmt78%253D&md5=8efa78a0b0dab38f7420d516fb653022</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2898%2901043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252898%252901043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMergny%26aufirst%3DJ.-L.%26aulast%3DPhan%26aufirst%3DA.-T.%26aulast%3DLacroix%26aufirst%3DL.%26atitle%3DFollowing%2520G-quartet%2520formation%2520by%2520UV-spectroscopy%26jtitle%3DFEBS%2520Lett.%26date%3D1998%26volume%3D435%26spage%3D74%26epage%3D78%26doi%3D10.1016%2FS0014-5793%2898%2901043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kypr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kejnovská, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renciuk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorlíčková, M.</span></span> <span> </span><span class="NLM_article-title">Circular dichroism and conformational polymorphism of DNA</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1725</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1093%2Fnar%2Fgkp026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=19190094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktV2ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=1713-1725&author=J.+Kyprauthor=I.+Kejnovsk%C3%A1author=D.+Renciukauthor=M.+Vorl%C3%AD%C4%8Dkov%C3%A1&title=Circular+dichroism+and+conformational+polymorphism+of+DNA&doi=10.1093%2Fnar%2Fgkp026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Circular dichroism and conformational polymorphism of DNA</span></div><div class="casAuthors">Kypr, Jaroslav; Kejnovska, Iva; Renciuk, Daniel; Vorlickova, Michaela</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1713-1725</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Here we review studies that provided important information about conformational properties of DNA using circular dichroic (CD) spectroscopy.  The conformational properties include the B-family of structures, A-form, Z-form, guanine quadruplexes, cytosine quadruplexes, triplexes and other less characterized structures.  CD spectroscopy is extremely sensitive and relatively inexpensive.  This fast and simple method can be used at low- as well as high-DNA concns. and with short- as well as long-DNA mols.  The samples can easily be titrated with various agents to cause conformational isomerizations of DNA.  The course of detected CD spectral changes makes possible to distinguish between gradual changes within a single DNA conformation and cooperative isomerizations between discrete structural states.  It enables measuring kinetics of the appearance of particular conformers and detn. of their thermodn. parameters.  In careful hands, CD spectroscopy is a valuable tool for mapping conformational properties of particular DNA mols.  Due to its numerous advantages, CD spectroscopy significantly participated in all basic conformational findings on DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtBf5jUZuhtbVg90H21EOLACvtfcHk0lgGjFivGbStww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktV2ru78%253D&md5=d03eafb37e5b3b6b6369198954b9999c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp026%26sid%3Dliteratum%253Aachs%26aulast%3DKypr%26aufirst%3DJ.%26aulast%3DKejnovsk%25C3%25A1%26aufirst%3DI.%26aulast%3DRenciuk%26aufirst%3DD.%26aulast%3DVorl%25C3%25AD%25C4%258Dkov%25C3%25A1%26aufirst%3DM.%26atitle%3DCircular%2520dichroism%2520and%2520conformational%2520polymorphism%2520of%2520DNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D1713%26epage%3D1725%26doi%3D10.1093%2Fnar%2Fgkp026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vorlíčková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kejnovská, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renčiuk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednářová, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motlová, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kypr, J.</span></span> <span> </span><span class="NLM_article-title">Circular dichroism and guanine quadruplexes</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2012.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ymeth.2012.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22450044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1ymt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=64-75&author=M.+Vorl%C3%AD%C4%8Dkov%C3%A1author=I.+Kejnovsk%C3%A1author=J.+Sagiauthor=D.+Ren%C4%8Diukauthor=K.+Bedn%C3%A1%C5%99ov%C3%A1author=J.+Motlov%C3%A1author=J.+Kypr&title=Circular+dichroism+and+guanine+quadruplexes&doi=10.1016%2Fj.ymeth.2012.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Circular dichroism and guanine quadruplexes</span></div><div class="casAuthors">Vorlickova, Michaela; Kejnovska, Iva; Sagi, Janos; Renciuk, Daniel; Bednarova, Klara; Motlova, Jitka; Kypr, Jaroslav</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-75</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  CD is remarkably sensitive to the conformational states of nucleic acids; therefore, CD spectroscopy was used to study most features of DNA and RNA structures.  Quadruplexes are among the significant noncanonical nucleic acids architectures that have received special attentions recently.  This article presents examples on the contribution of CD spectroscopy to the authors' knowledge of quadruplex structures and their polymorphism.  The examples were selected to demonstrate the potential of this simple method in the quadruplex field.  As CD spectroscopy detects only the global feature of a macromol., it should preferably be used in combination with other techniques.  On the other hand, CD spectroscopy, often as a pioneering approach, can reveal the formation of particular structural arrangements, to search for the conditions stabilizing the structures, to follow the transitions between various structural states, to explore kinetics of their appearance, to det. thermodn. parameters and also detect formation of higher order structures.  This article aims to show that CD spectroscopy is an important complementary technique to NMR spectroscopy and x-ray diffraction in quadruplex studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAwWkf_YGsV7Vg90H21EOLACvtfcHk0lhrlK3oYsiYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1ymt7s%253D&md5=65cb6aacfbaa31ec902248f834993282</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2012.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2012.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DVorl%25C3%25AD%25C4%258Dkov%25C3%25A1%26aufirst%3DM.%26aulast%3DKejnovsk%25C3%25A1%26aufirst%3DI.%26aulast%3DSagi%26aufirst%3DJ.%26aulast%3DRen%25C4%258Diuk%26aufirst%3DD.%26aulast%3DBedn%25C3%25A1%25C5%2599ov%25C3%25A1%26aufirst%3DK.%26aulast%3DMotlov%25C3%25A1%26aufirst%3DJ.%26aulast%3DKypr%26aufirst%3DJ.%26atitle%3DCircular%2520dichroism%2520and%2520guanine%2520quadruplexes%26jtitle%3DMethods%26date%3D2012%26volume%3D57%26spage%3D64%26epage%3D75%26doi%3D10.1016%2Fj.ymeth.2012.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis of 4,5-diazafluorene derivatives and their anticancer activity via targeting telomeric DNA G-quadruplex</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>178</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">499</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejmech.2019.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=31202994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1SrsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2019&pages=484-499&author=K.+Zhouauthor=J.+Liuauthor=X.+Xiongauthor=M.+Chengauthor=X.+Huauthor=S.+Narvaauthor=X.+Zhaoauthor=Y.+Wuauthor=W.+Zhang&title=Design%2C+synthesis+of+4%2C5-diazafluorene+derivatives+and+their+anticancer+activity+via+targeting+telomeric+DNA+G-quadruplex&doi=10.1016%2Fj.ejmech.2019.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis of 4,5-diazafluorene derivatives and their anticancer activity via targeting telomeric DNA G-quadruplex</span></div><div class="casAuthors">Zhou, Kang; Liu, Jiachun; Xiong, Xuqiong; Cheng, Mei; Hu, Xiaolin; Narva, Suresh; Zhao, Xiaoyin; Wu, Yanling; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">484-499</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In our work, 19 novel 4,5-diazafluorene derivs. (11a-d, 12a-d, 13a-d, 14a-c, 15c, 16a-c) bearing a 1,3-disubstituted pyrazol/thioxothiazolidinone or thioxothiazolidinone-oxadiazole moieties were designed, synthesized, preliminarily explored for their antitumor activities and in vitro mechanism.  All compds. showed different values of antiproliferative activity against A549, AGS, HepG2 and MCF-7 cell lines through CCK-8.  Esp., the compd. 14c exhibited the strongest activity and best selectivity against A549 cells with an IC50 1.13 μM and an SI value of 7.01 relative to MRC-5 cells, which was better than cisplatin (SI = 1.80) as a pos. control.  Exptl. results at extracellular level demonstrated that compds. 14a-c could strongly interact with the G-quadruplex(es) formed in a 26 nt telomeric G-rich DNA, in particular, the 14c exhibits quite strong binding affinity with an assocn. equil. const. (KA) of 7.04(±0.16) × 107 M-1 and more than 1000-fold specificity to G4-DNA over ds-DNA and Mut-DNA at the compd./G4-DNA ratio of 1:1.  Further trap assay ascertained that compds. 14a-c owned strong inhibitory ability of telomerase activity in A549 cells, suggesting that these compds. have great possibility to target telomeric G-quadruplexes and consequently indirectly inhibit the telomerase activity.  In addn., it is worthy of note that the remarkable inhibitory effects of 14a-c on the mobility of tested cancer cells were obsd. by wound healing assays.  Furthermore, mol. docking and UV-Vis spectral results unclose the rationale for the interaction of compds. with such G-quadruplex(es).  These results indicate that the growth and metastasis inhibition of cancer cells mediated by these 4,5-diazafluorene derivs. possibly result from their interaction with telomeric G-quadruplexes, suggesting that 4,5-diazafluorene derivs., esp. 14c, possess potential as anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61fInFRE5NrVg90H21EOLACvtfcHk0lhrlK3oYsiYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1SrsbrN&md5=be899fa13a48a9743475bc5a4c7da1df</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DNarva%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DDesign%252C%2520synthesis%2520of%25204%252C5-diazafluorene%2520derivatives%2520and%2520their%2520anticancer%2520activity%2520via%2520targeting%2520telomeric%2520DNA%2520G-quadruplex%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D178%26spage%3D484%26epage%3D499%26doi%3D10.1016%2Fj.ejmech.2019.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awadasseid, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and anti-cancer activity of pyrrole-imidazole polyamides through target-downregulation of c-kit gene expression</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>,  <span class="NLM_fpage">112704</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejmech.2020.112704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=32866755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&pages=112704&author=M.+Zhangauthor=J.+Liangauthor=S.+Jiangauthor=L.+Xuauthor=Y.+Wuauthor=A.+Awadasseidauthor=X.+Zhaoauthor=X.+Xiongauthor=H.+Sugiyamaauthor=W.+Zhang&title=Design%2C+synthesis+and+anti-cancer+activity+of+pyrrole-imidazole+polyamides+through+target-downregulation+of+c-kit+gene+expression&doi=10.1016%2Fj.ejmech.2020.112704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and anti-cancer activity of pyrrole-imidazole polyamides through target-downregulation of c-kit gene expression</span></div><div class="casAuthors">Zhang, Mi; Liang, Jing; Jiang, Shi-Kun; Xu, Ling; Wu, Yan-Ling; Awadasseid, Annoor; Zhao, Xiao-Yin; Xiong, Xu-Qiong; Sugiyama, Hiroshi; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112704</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Pyrrole-imidazole polyamide (PIP) can specifically bind in the B-DNA minor groove that has been used in several biol. applications, such as anti-cancer activity, gene expression and translation control, and visualization of complex genomic areas. c-kit is a family member of the Tyrosine Kinase (RTK) type III receptor and plays a vital role in tumor growth, proliferation, differentiation, and cell apoptosis; however, its mutations and overexpression induce tumor dysfunction.  Here, we designed and synthesized five matched PIPs that can recognize and bind to the DNA sequence in the oncogene c-kit promoter region, and evaluated their anti-cancer activity.  The RTCA assay findings revealed that the PIPs would prevent the proliferation of cancer cells A549 and SGC-7901.  EMSA assay showed that the PIPs were actively interacting with the c-kit gene target DNA.  RT-PCR and Western blot assays have an effect on expression levels of the c-kit gene in the presence of PIPs.  Flow cytometry and wound-healing assays showed that two of the PIPs would cause apoptosis of cancer cells and inhibit the migration of cells, resp.  Overall findings indicate that chloride of PIP (I) has a relatively significant intracellular and extracellular impact on the target, contributing to migration and proliferation redn., and cancer cell apoptosis.  In addn., PIP has a certain ability to evolve into c-kit gene-targeting drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJHTamfXa5z7Vg90H21EOLACvtfcHk0lhrlK3oYsiYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjL&md5=eefd1e911d49a683806b7cad8361093b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112704%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DAwadasseid%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DDesign%252C%2520synthesis%2520and%2520anti-cancer%2520activity%2520of%2520pyrrole-imidazole%2520polyamides%2520through%2520target-downregulation%2520of%2520c-kit%2520gene%2520expression%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26spage%3D112704%26doi%3D10.1016%2Fj.ejmech.2020.112704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span> <span> </span><span class="NLM_article-title">A proton-fuelled DNA nanomachine</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">5734</span>– <span class="NLM_lpage">5736</span>, <span class="refDoi"> DOI: 10.1002/anie.200352402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Fanie.200352402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=14661209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=5734-5736&author=D.+Liuauthor=S.+Balasubramanian&title=A+proton-fuelled+DNA+nanomachine&doi=10.1002%2Fanie.200352402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A proton-fueled DNA nanomachine</span></div><div class="casAuthors">Liu, Dongsheng; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">5734-5736</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A mol. machine based on the DNA i-motif structure is driven by changing pH.  At pH 5, oligonucleotide X* folds into the i-motif (closed state), and complementary strand Y adopts a random-coil structure.  At pH 8, X* unfolds and forms an extended duplex with Y.  This reversible process can be monitored by means of a rhodamine green label, whose fluorescence is quenched by a dabcyl moiety in the closed but not in the open state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgIDQPvx_h5bVg90H21EOLACvtfcHk0ljKmE3rwDpLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFahsr4%253D&md5=737e67368f61658bf5287f103ef53147</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.200352402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200352402%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DA%2520proton-fuelled%2520DNA%2520nanomachine%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26spage%3D5734%26epage%3D5736%26doi%3D10.1002%2Fanie.200352402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Biosensor-surface plasmon resonance: Quantitative analysis of small molecule–nucleic acid interactions</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2006.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ymeth.2006.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=17472897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=150-161&author=B.+Nguyenauthor=F.+A.+Taniousauthor=W.+D.+Wilson&title=Biosensor-surface+plasmon+resonance%3A+Quantitative+analysis+of+small+molecule%E2%80%93nucleic+acid+interactions&doi=10.1016%2Fj.ymeth.2006.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-surface plasmon resonance: quantitative analysis of small molecule-nucleic acid interactions</span></div><div class="casAuthors">Nguyen, Binh; Tanious, Farial A.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-161</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Surface plasmon resonance (SPR)-biosensor techniques directly provide essential information for the study and characterization of small mol.-nucleic acid interactions, and the use of these methods is steadily increasing.  The method is label-free and monitors the interactions in real time.  Both dynamic and steady-state information can be obtained for a wide range of reaction rates and binding affinities.  This article presents the basics of the SPR technique, provides suggestions for exptl. design, and illustrates data processing and anal. of results.  A specific example of the interaction of a well-known minor groove binding agent, netropsin, with DNA is evaluated by both kinetic and steady-state SPR methods.  Three different expts. are used to illustrate different approaches and anal. methods.  The three sets of results show the reproducibility of the binding consts. and agreement from both steady-state and kinetic analyses.  These expts. also show that reliable kinetic information can be obtained, even with difficult systems, if the exptl. conditions are optimized to minimize mass transport effects.  Limitations of the biosensor-SPR technique are also discussed to provide an awareness of the care needed to conduct a successful expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIMfX4ZwyTSLVg90H21EOLACvtfcHk0ljKmE3rwDpLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D&md5=9455fa47e2eefef19166b544873c9102</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2006.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2006.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DBiosensor-surface%2520plasmon%2520resonance%253A%2520Quantitative%2520analysis%2520of%2520small%2520molecule%25E2%2580%2593nucleic%2520acid%2520interactions%26jtitle%3DMethods%26date%3D2007%26volume%3D42%26spage%3D150%26epage%3D161%26doi%3D10.1016%2Fj.ymeth.2006.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Clairac, R. P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geierstanger, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrksich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wemmer, D. E.</span></span> <span> </span><span class="NLM_article-title">NMR characterization of hairpin polyamide complexed with the minor groove of DNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">7909</span>– <span class="NLM_lpage">7916</span>, <span class="refDoi"> DOI: 10.1021/ja964314r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja964314r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=7909-7916&author=R.+P.+L.+de%0AClairacauthor=B.+H.+Geierstangerauthor=M.+Mrksichauthor=P.+B.+Dervanauthor=D.+E.+Wemmer&title=NMR+characterization+of+hairpin+polyamide+complexed+with+the+minor+groove+of+DNA&doi=10.1021%2Fja964314r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja964314r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja964314r%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BClairac%26aufirst%3DR.%2BP.%2BL.%26aulast%3DGeierstanger%26aufirst%3DB.%2BH.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DWemmer%26aufirst%3DD.%2BE.%26atitle%3DNMR%2520characterization%2520of%2520hairpin%2520polyamide%2520complexed%2520with%2520the%2520minor%2520groove%2520of%2520DNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D7909%26epage%3D7916%26doi%3D10.1021%2Fja964314r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span> <span> </span><span class="NLM_article-title">Base pair recognition of the stereochemically alpha-substituted gamma-turn of pyrrole/imidazole hairpin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">14905</span>– <span class="NLM_lpage">14912</span>, <span class="refDoi"> DOI: 10.1021/ja064369l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja064369l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFCjtbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=14905-14912&author=W.+Zhangauthor=M.+Minoshimaauthor=H.+Sugiyama&title=Base+pair+recognition+of+the+stereochemically+alpha-substituted+gamma-turn+of+pyrrole%2Fimidazole+hairpin&doi=10.1021%2Fja064369l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Base Pair Recognition of the Stereochemically α-Substituted γ-Turn of Pyrrole/Imidazole Hairpin Polyamides</span></div><div class="casAuthors">Zhang, Wen; Minoshima, Masafumi; Sugiyama, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">14905-14912</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognition of the sequences 5'-NGCACA-3' (N = T, A, C, G) by pyrrole/imidazole polyamides with (R/S)-α-hydroxyl/α-amino-substituted γ-aminobutyric acid as a γ-turn was investigated.  Four novel polyamides, 2, 3, 4, and 5, including (R)-α-hydroxyl-γ-aminobutyric acid (γRO), (S)-α-hydroxyl-γ-aminobutyric acid (γSO), (R)-α,γ-diaminobutyric acid (γRN), and (S)-α,γ-diaminobutyric acid (γSN) residues, resp., were synthesized, and their binding affinity to T·A, A·T, G·C, and C·G base pairs at turn position was studied by the surface plasmon resonance (SPR) technique.  SPR data revealed that polyamide 3, AcImβImPy-γSO-ImPyβPy-β-Dp, with a γSO turn, possesses a marked binding preference for T·A over A·T with a 25-fold increase in specificity, despite low binding affinity relative to 2, with a γRO turn.  Similarly, AcImβImPy-γSN-ImPyβPy-β-Dp (5), with a γSN-turn, gives rise to a 8.7-fold increase in specificity for T·A over A·T.  Computer-assisted mol. modeling suggests that 3 binds more deeply in the minor groove of the T·A base pair relative to the A·T base pair, allowing hydrogen bonding to O2 of the thymine at the turn position, which explains the SPR results.  These results suggest that γSO and γSN may function as T-recognition units at the turn position, as well as a γ-turn in the discrimination of polyamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbh8Rp7yzV5bVg90H21EOLACvtfcHk0ljKmE3rwDpLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFCjtbzI&md5=7b6fd576cf04de6292ea760386de245d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja064369l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja064369l%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMinoshima%26aufirst%3DM.%26aulast%3DSugiyama%26aufirst%3DH.%26atitle%3DBase%2520pair%2520recognition%2520of%2520the%2520stereochemically%2520alpha-substituted%2520gamma-turn%2520of%2520pyrrole%252Fimidazole%2520hairpin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D14905%26epage%3D14912%26doi%3D10.1021%2Fja064369l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mormile, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, G.</span></span> <span> </span><span class="NLM_article-title">Endometriosis and susceptibility to cancer: the dark side of IFN-γ gene polymorphism?</span>. <i>J. Cancer Sci. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.4172/1948-5956.1000218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.4172%2F1948-5956.1000218" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=289-290&author=R.+Mormileauthor=G.+Vittori&title=Endometriosis+and+susceptibility+to+cancer%3A+the+dark+side+of+IFN-%CE%B3+gene+polymorphism%3F&doi=10.4172%2F1948-5956.1000218"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.4172%2F1948-5956.1000218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F1948-5956.1000218%26sid%3Dliteratum%253Aachs%26aulast%3DMormile%26aufirst%3DR.%26aulast%3DVittori%26aufirst%3DG.%26atitle%3DEndometriosis%2520and%2520susceptibility%2520to%2520cancer%253A%2520the%2520dark%2520side%2520of%2520IFN-%25CE%25B3%2520gene%2520polymorphism%253F%26jtitle%3DJ.%2520Cancer%2520Sci.%2520Ther.%26date%3D2013%26volume%3D5%26spage%3D289%26epage%3D290%26doi%3D10.4172%2F1948-5956.1000218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quandt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldkamp, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Specker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kribben, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzke, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, B.</span></span> <span> </span><span class="NLM_article-title">Increased percentages of PD-1 on CD4+ T cells is associated with higher INF-γ production and altered IL-17 production in patients with systemic lupus erythematosus</span>. <i>Scand. J. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.3109/03009742.2013.869830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.3109%2F03009742.2013.869830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25088926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC2cbos1yltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=307-313&author=S.+Dolffauthor=D.+Quandtauthor=T.+Feldkampauthor=C.+Junauthor=A.+Mitchellauthor=F.+Huaauthor=C.+Speckerauthor=A.+Kribbenauthor=O.+Witzkeauthor=B.+Wilde&title=Increased+percentages+of+PD-1+on+CD4%2B+T+cells+is+associated+with+higher+INF-%CE%B3+production+and+altered+IL-17+production+in+patients+with+systemic+lupus+erythematosus&doi=10.3109%2F03009742.2013.869830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Increased percentages of PD-1 on CD4+ T cells is associated with higher INF-γ production and altered IL-17 production in patients with systemic lupus erythematosus</span></div><div class="casAuthors">Dolff S; Quandt D; Feldkamp T; Jun C; Mitchell A; Hua F; Specker C; Kribben A; Witzke O; Wilde B</div><div class="citationInfo"><span class="NLM_cas:title">Scandinavian journal of rheumatology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-13</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Programmed death (PD)-1 is a cell death receptor that, upon stimulation, leads to apoptosis.  Previous studies have shown alteration of PD-1 expression on T cells and PD-1 genes in patients with systemic lupus erythematosus (SLE).  The aim of this study was to assess the expression of this receptor on effector T cells in patients with SLE.  METHOD:  In this study we enrolled 32 SLE patients and 31 healthy controls.  T cells from peripheral blood were analysed by flow cytometry for the expression of PD-1.  Interferon (IFN)-γ and interleukin (IL)-17-producing cells were investigated for the expression of this co-stimulatory marker.  RESULTS:  Percentages of CD4(+) T cells expressing PD-1 were significantly increased in patients with SLE compared to healthy controls.  The percentage of PD-1 expression was correlated with the production of INF-γ (r = 0.83, p < 0.0001).  We also investigated the production of IL-17 by PD-1(+) CD3(+) T cells.  Inactive patients (3.2 ± 1.2% vs. 5.9 ± 3.5%, p = 0.002) and patients without lupus nephritis (LN) (3.2 ± 1.5% vs. 5.9 ± 3.5%, p = 0.005) showed lower levels of IL-17 compared to healthy controls.  CONCLUSION:  We have demonstrated increased expression of PD-1 on CD4(+) T cells in SLE patients and an association between PD-1 expression on CD4(+) T cells and IFN-γ expression on CD3(+) T cells.  We have also shown that there is an altered subset of PD-1(+) T cells in inactive patients and patients without LN producing lower amounts of IL-17.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6qiH7W1cc5OdfST00W0BcfW6udTcc2eYmZCmMssRdCbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbos1yltA%253D%253D&md5=28ba97925c65dec563a7cdcc8890d413</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.3109%2F03009742.2013.869830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03009742.2013.869830%26sid%3Dliteratum%253Aachs%26aulast%3DDolff%26aufirst%3DS.%26aulast%3DQuandt%26aufirst%3DD.%26aulast%3DFeldkamp%26aufirst%3DT.%26aulast%3DJun%26aufirst%3DC.%26aulast%3DMitchell%26aufirst%3DA.%26aulast%3DHua%26aufirst%3DF.%26aulast%3DSpecker%26aufirst%3DC.%26aulast%3DKribben%26aufirst%3DA.%26aulast%3DWitzke%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DB.%26atitle%3DIncreased%2520percentages%2520of%2520PD-1%2520on%2520CD4%252B%2520T%2520cells%2520is%2520associated%2520with%2520higher%2520INF-%25CE%25B3%2520production%2520and%2520altered%2520IL-17%2520production%2520in%2520patients%2520with%2520systemic%2520lupus%2520erythematosus%26jtitle%3DScand.%2520J.%2520Rheumatol.%26date%3D2014%26volume%3D43%26spage%3D307%26epage%3D313%26doi%3D10.3109%2F03009742.2013.869830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Investigation of the Relative Cellular Permeability of DNA-binding Pyrrole-Imidazole Polyamides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4604</span>– <span class="NLM_lpage">4612</span>, <span class="refDoi"> DOI: 10.1021/jm9002999</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9002999" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslCnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4604-4612&author=B.+Liuauthor=T.+Kodadek&title=Investigation+of+the+Relative+Cellular+Permeability+of+DNA-binding+Pyrrole-Imidazole+Polyamides&doi=10.1021%2Fjm9002999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of the Relative Cellular Permeability of DNA-Binding Pyrrole-Imidazole Polyamides</span></div><div class="casAuthors">Liu, Bo; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4604-4612</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pyrrole-imidazole (Py-Im) polyamides are a group of chems. that are able to bind specifically to DNA sequences in vitro and in mammalian cells.  Using a cell based reporter assay, we investigated the size and linker affects on the cellular permeability of polyamides.  We found that the conventional β-alanine-3,3'-diamino-N-methyldipropylamine (βDa) linker strongly limited the cellular permeability.  We discovered that a short ethylene diamine (Et) linker displayed high cellular permeability.  With the improved Et linker, we found that the cellular permeability of polyamides was size-dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbf5cxbAOFWLVg90H21EOLACvtfcHk0ljKjhDh_Mjf1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXoslCnur8%253D&md5=c28171e290ee54529aa40d7bfc91569b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm9002999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9002999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DInvestigation%2520of%2520the%2520Relative%2520Cellular%2520Permeability%2520of%2520DNA-binding%2520Pyrrole-Imidazole%2520Polyamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4604%26epage%3D4612%26doi%3D10.1021%2Fjm9002999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alizadeh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niederkorn, J. Y.</span></span> <span> </span><span class="NLM_article-title">PD-L1: PD-1 interaction contributes to the functional suppression of T-cell responses to human uveal melanoma cells in vitro</span>. <i>Invest. Ophthalmol. Visual Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1167/iovs.07-1606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1167%2Fiovs.07-1606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=18296654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BD1czkvVertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2008&pages=2518-2525&author=W.+Yangauthor=P.+W.+Chenauthor=H.+Liauthor=H.+Alizadehauthor=J.+Y.+Niederkorn&title=PD-L1%3A+PD-1+interaction+contributes+to+the+functional+suppression+of+T-cell+responses+to+human+uveal+melanoma+cells+in+vitro&doi=10.1167%2Fiovs.07-1606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1: PD-1 interaction contributes to the functional suppression of T-cell responses to human uveal melanoma cells in vitro</span></div><div class="casAuthors">Yang Wanhua; Chen Peter W; Li Haochuan; Alizadeh Hassan; Niederkorn Jerry Y</div><div class="citationInfo"><span class="NLM_cas:title">Investigative ophthalmology & visual science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2518-25</span>
        ISSN:<span class="NLM_cas:issn">0146-0404</span>.
    </div><div class="casAbstract">PURPOSE:  To assess the expression of PD-L1 on human uveal melanomas and its potential to suppress T-cell function.  METHODS:  A panel of primary and metastatic uveal melanoma cell lines was evaluated for PD-L1 expression by RT-PCR and flow cytometric analysis.  Uveal melanoma-containing eyes were examined for PD-L1 expression by immunohistochemistry.  PD-L1 function was tested by coculturing IFN-gamma-pretreated uveal melanoma cells with activated Jurkat T cells for 48 hours and assessing T-cell production of IL-2 by ELISA.  RESULTS:  Five of the nine primary and one of the five metastatic uveal melanoma cell lines tested constitutively expressed PD-L1 protein at various levels.  However, all primary and metastatic uveal melanoma cell lines upregulated PD-L1 expression after stimulation with IFN-gamma.  Immunohistochemistry demonstrated that PD-L1 was not expressed by primary uveal melanomas in situ.  IL-2 production by activated Jurkat T cells was decreased significantly when the cells were cocultured with IFN-gamma-pretreated uveal melanoma cells.  More than 70% of IL-2 production was restored by addition of either anti-PD-L1 or anti-PD-1 antibody to the coculture assays (P < 0.01).  CONCLUSIONS:  Expression of PD-L1 by uveal melanoma cells regulates T-cell function by suppressing IL-2 production.  The results imply that the presence of IFN-gamma in the tumor local microenvironment promotes upregulation of PD-L1 expression by uveal melanoma, which may, in part, promote immune escape by impairing T-cell function.  The selective blockade of PD-L1 is a potential strategy in T-cell-based immunotherapy for uveal melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSA_-nZI1DHTpPq5LRczm8ffW6udTcc2eYmZCmMssRdCbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czkvVertQ%253D%253D&md5=dc40143aeb4a7a747cd16efc7776cc17</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1167%2Fiovs.07-1606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1167%252Fiovs.07-1606%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DP.%2BW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAlizadeh%26aufirst%3DH.%26aulast%3DNiederkorn%26aufirst%3DJ.%2BY.%26atitle%3DPD-L1%253A%2520PD-1%2520interaction%2520contributes%2520to%2520the%2520functional%2520suppression%2520of%2520T-cell%2520responses%2520to%2520human%2520uveal%2520melanoma%2520cells%2520in%2520vitro%26jtitle%3DInvest.%2520Ophthalmol.%2520Visual%2520Sci.%26date%3D2008%26volume%3D49%26spage%3D2518%26epage%3D2525%26doi%3D10.1167%2Fiovs.07-1606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, S.-Y</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ro, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ro, J.</span></span> <span> </span><span class="NLM_article-title">Prognostic implications of tumor-infiltrating lymphocytes in association with programmed death ligand 1 expression in early-stage breast cancer</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2015.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.clbc.2015.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26364145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSit7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=51-58&author=I.+H.+Parkauthor=S.-Y+Kongauthor=J.+Y.+Roauthor=Y.+Kwonauthor=J.+H.+Kangauthor=H.+J.+Moauthor=S.-Y.+Jungauthor=S.+Leeauthor=K.+S.+Leeauthor=H.-S.+Kangauthor=E.+Leeauthor=J.+Jooauthor=J.+Ro&title=Prognostic+implications+of+tumor-infiltrating+lymphocytes+in+association+with+programmed+death+ligand+1+expression+in+early-stage+breast+cancer&doi=10.1016%2Fj.clbc.2015.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic implications of tumor-infiltrating lymphocytes in association with programmed death ligand 1 expression in early-stage breast cancer</span></div><div class="casAuthors">Park, In Hae; Kong, Sun-Young; Ro, Jae Yoon; Kwon, Youngmee; Kang, Joo Hyun; Mo, Hye Jin; Jung, So-Youn; Lee, Seeyoun; Lee, Keun Seok; Kang, Han-Sung; Lee, Eunsook; Joo, Jungnam; Ro, Jungsil</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-58</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The immune system might influence breast cancer (BC) prognosis.  However, the relationship between programmed death ligand 1 (PD-L1) and tumor-infiltrating lymphocyte (TIL) profiles remains unclear with respect to BC subtypes.We investigated the relationship between TIL profiles for CD8+ and forkhead box P3-pos. (FOXP3+) and PD-L1 expression in primary tumor tissue using immunohistochem. and the clin. outcomes in 2 patient cohorts at the National Cancer Center: 256 patients diagnosed with early-stage BC from Jan. 2001 to Dec. 2005 and 77 hormone receptor (HR)-neg. BC patients diagnosed from Jan. 2006 to Dec. 2008.  Clin. data were collected, including HR status, human epidermal growth factor receptor 2 expression, disease-free survival, and overall survival (OS).The median patient age was 47 years (range, 28-78), and the median follow-up period was 9.8 years.  Of the 333 patients, 186 (55.9%) had HR-pos. and 125 (37.5%) had node-pos. BC.  We found a strong pos. correlation between CD8+ TILs and FOXP3+ TILs (P < .001).  CD8+ TILs were more abundant in tumors with low PD-L1 expression (P < .001), although no assocn. was found between FOXP3+ TILs and PD-L1 expression.  More CD8+ TILs were present in HR-neg. than in HR-pos. BC (P < .001), and PD-L1 expression was more frequent in HR-pos. BC (P < .001).  A greater no. of CD8+ TILs (increase in quartile) was strongly assocd. with OS (hazard ratio, 0.61; 95% confidence interval, 0.39-0.95; P = .03) only in HR-neg. BC when adjusted for various clin. factors.  PD-L1 expression and FOXP3+ TILs did not exhibit such assocns. Higher CD8+ lymphocyte infiltration was related to lower PD-L1 expression and higher FOXP3+ TIL infiltration in BC.  Higher CD8+ TIL expression was assocd. with prolonged survival only in those with HR-neg. BC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcTcRpS0nGPLVg90H21EOLACvtfcHk0li6c2DHTTx9Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSit7zF&md5=c524199fd1997df48b4d71d66851b7fd</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2015.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2015.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DI.%2BH.%26aulast%3DKong%26aufirst%3DS.-Y%26aulast%3DRo%26aufirst%3DJ.%2BY.%26aulast%3DKwon%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DJ.%2BH.%26aulast%3DMo%26aufirst%3DH.%2BJ.%26aulast%3DJung%26aufirst%3DS.-Y.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BS.%26aulast%3DKang%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DE.%26aulast%3DJoo%26aufirst%3DJ.%26aulast%3DRo%26aufirst%3DJ.%26atitle%3DPrognostic%2520implications%2520of%2520tumor-infiltrating%2520lymphocytes%2520in%2520association%2520with%2520programmed%2520death%2520ligand%25201%2520expression%2520in%2520early-stage%2520breast%2520cancer%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D51%26epage%3D58%26doi%3D10.1016%2Fj.clbc.2015.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span> <span> </span><span class="NLM_article-title">Adaptive immune resistance: how cancer protects from immune attack</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-0563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F2159-8290.CD-15-0563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26272491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVOqtrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=915-919&author=A.+Ribas&title=Adaptive+immune+resistance%3A+how+cancer+protects+from+immune+attack&doi=10.1158%2F2159-8290.CD-15-0563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Adaptive Immune Resistance: How Cancer Protects from Immune Attack</span></div><div class="casAuthors">Ribas, Antoni</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">915-919</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Adaptive immune resistance is a process in which the cancer changes its phenotype in response to a cytotoxic or proinflammatory immune response, thereby evading it.  This adaptive process is triggered by the specific recognition of cancer cells by T cells, which leads to the prodn. of immune-activating cytokines.  Cancers then hijack mechanisms developed to limit inflammatory and immune responses and protect themselves from the T-cell attack.  Inhibiting adaptive immune resistance is the mechanistic basis of responses to PD-1 or PD-L1-blocking antibodies, and may be of relevance for the development of other cancer immunotherapy strategies.  Significance: Several new immunotherapy strategies to treat cancer are based on inhibiting processes through which cancer adapts and evades from an immune response.  Recognizing the specific adaptive resistance mechanisms in each case is likely to allow the personalized development of immunotherapies tailored to block how a particular cancer protects itself from the immune system.  Cancer Discov; 5(9); 915-9. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWNhTGL5MdgbVg90H21EOLACvtfcHk0li6c2DHTTx9Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOqtrnL&md5=ba45d5b4e78b9a3fc0689756c0dffb62</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-0563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-0563%26sid%3Dliteratum%253Aachs%26aulast%3DRibas%26aufirst%3DA.%26atitle%3DAdaptive%2520immune%2520resistance%253A%2520how%2520cancer%2520protects%2520from%2520immune%2520attack%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D915%26epage%3D919%26doi%3D10.1158%2F2159-8290.CD-15-0563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coombs, M. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoskin, D. W.</span></span> <span> </span><span class="NLM_article-title">Apigenin inhibits the inducible expression of programmed death ligand 1 by human and mouse mammary carcinoma cells</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>380</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2016.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.canlet.2016.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27378243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFant73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=2016&pages=424-433&author=M.+R.+P.+Coombsauthor=M.+E.+Harrisonauthor=D.+W.+Hoskin&title=Apigenin+inhibits+the+inducible+expression+of+programmed+death+ligand+1+by+human+and+mouse+mammary+carcinoma+cells&doi=10.1016%2Fj.canlet.2016.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Apigenin inhibits the inducible expression of programmed death ligand 1 by human and mouse mammary carcinoma cells</span></div><div class="casAuthors">Coombs, Melanie R. Power; Harrison, Megan E.; Hoskin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">380</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">424-433</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Programmed death ligand 1 (PD-L1) is expressed by many cancer cell types, as well as by activated T cells and antigen-presenting cells.  Constitutive and inducible PD-L1 expression contributes to immune evasion by breast cancer (BC) cells.  We show here that the dietary phytochem. apigenin inhibited interferon (IFN)-γ-induced PD-L1 upregulation by triple-neg. MDA-MB-468 BC cells, HER2+ SK-BR-3 BC cells, and 4T1 mouse mammary carcinoma cells, as well as human mammary epithelial cells, but did not affect constitutive PD-L1 expression by triple-neg. MDA-MB-231 BC cells.  IFN-β-induced expression of PD-L1 by MDA-MB-468 cells was also inhibited by apigenin.  In addn., luteolin, the major metabolite of apigenin, inhibited IFN-γ-induced PD-L1 expression by MDA-MB-468 cells.  Apigenin-mediated inhibition of IFN-γ-induced PD-L1 expression by MDA-MB-468 and 4T1 cells was assocd. with reduced phosphorylation of STAT1, which was early and transient at Tyr701 and sustained at Ser727.  Apigenin-mediated inhibition of IFN-γ-induced PD-L1 expression by MDA-MB-468 cells also increased proliferation and interleukin-2 synthesis by PD-1-expressing Jurkat T cells that were co-cultured with MDA-MB-468 cells.  Apigenin therefore has the potential to increase the vulnerability of BC cells to T cell-mediated anti-tumor immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVpAiyZIgsLrVg90H21EOLACvtfcHk0li6c2DHTTx9Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFant73N&md5=4df618d6f5fa91c5dbd1dedd57b639d7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DCoombs%26aufirst%3DM.%2BR.%2BP.%26aulast%3DHarrison%26aufirst%3DM.%2BE.%26aulast%3DHoskin%26aufirst%3DD.%2BW.%26atitle%3DApigenin%2520inhibits%2520the%2520inducible%2520expression%2520of%2520programmed%2520death%2520ligand%25201%2520by%2520human%2520and%2520mouse%2520mammary%2520carcinoma%2520cells%26jtitle%3DCancer%2520Lett.%26date%3D2016%26volume%3D380%26spage%3D424%26epage%3D433%26doi%3D10.1016%2Fj.canlet.2016.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span> <span> </span><span class="NLM_article-title">Tumor cell-derived lactate induces TAZ-dependent upregulation of PD-L1 through GPR81 in human lung cancer cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">5829</span>– <span class="NLM_lpage">5839</span>, <span class="refDoi"> DOI: 10.1038/onc.2017.188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fonc.2017.188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28604752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslWgtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=5829-5839&author=J.+Fengauthor=H.+Yangauthor=Y.+Zhangauthor=H.+Weiauthor=Z.+Zhuauthor=B.+Zhuauthor=M.+Yangauthor=W.+Caoauthor=L.+Wangauthor=Z.+Wu&title=Tumor+cell-derived+lactate+induces+TAZ-dependent+upregulation+of+PD-L1+through+GPR81+in+human+lung+cancer+cells&doi=10.1038%2Fonc.2017.188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor cell-derived lactate induces TAZ-dependent upregulation of PD-L1 through GPR81 in human lung cancer cells</span></div><div class="casAuthors">Feng, J.; Yang, H.; Zhang, Y.; Wei, H.; Zhu, Z.; Zhu, B.; Yang, M.; Cao, W.; Wang, L.; Wu, Z.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">5829-5839</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The clin. success of immunotherapy that inhibits the neg. immune regulatory pathway programmed cell death protein 1/PD-1 ligand (PD-1/PD-L1) has initiated a new era in the treatment of metastatic cancer.  PD-L1 expression is upregulated in many solid tumors including lung cancer and functions predominantly in lactate-enriched tumor microenvironments.  Here, we provided evidence for PD-L1 induction in response to lactate stimulation in lung cancer cells.  Lactate-induced PD-L1 induction was mediated by its receptor GPR81.  The silencing of GPR81 signaling in lung cancer cells resulted in a decrease in PD-L1 protein levels and functional inactivation of PD-L1 promoter activity.  In addn., GPR81-mediated upregulation of PD-L1 in glucose-stimulated lung cancer cells that recapitulates the enhanced glycolysis in vivo was dependent on lactate dehydrogenase A (LDHA).  We also demonstrated that activation of GPR81 decreases intracellular cAMP levels and inhibits protein kinase A (PKA) activity, leading to activation of the transcriptional coactivator TAZ.  Interaction of TAZ with the transcription factor TEAD was essential for TAZ activation of PD-L1 and induction of its expression.  Furthermore, we found that lactate-induced activation of PD-L1 in tumor cells led to reduced prodn. of interferon-γ and induction of apoptosis of cocultured Jurkat T-cell leukemia cells.  Our findings reveal an unexpected role of lactate in contributing to tumor cell protection from cytotoxic T-cell targeting and establishes a direct connection between tumor cell metabolic reprograming and tumor evasion from the immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKwzZT3o-gKrVg90H21EOLACvtfcHk0ljfwCEGualWmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslWgtbk%253D&md5=e37888d8162b315bf5ad13d14e1122b7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fonc.2017.188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2017.188%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%26atitle%3DTumor%2520cell-derived%2520lactate%2520induces%2520TAZ-dependent%2520upregulation%2520of%2520PD-L1%2520through%2520GPR81%2520in%2520human%2520lung%2520cancer%2520cells%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D5829%26epage%3D5839%26doi%3D10.1038%2Fonc.2017.188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Quercetin nanoparticles induced autophagy and apoptosis through AKT/ERK/Caspase-3 signaling pathway in human neuroglioma cells: In vitro and in vivo</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2016.08.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.biopha.2016.08.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27621033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2rt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2016&pages=1-9&author=M.+Louauthor=L.+Zhangauthor=P.+Jiauthor=F.+Fengauthor=J.+Liuauthor=C.+Yangauthor=B.+Liauthor=L.+Wang&title=Quercetin+nanoparticles+induced+autophagy+and+apoptosis+through+AKT%2FERK%2FCaspase-3+signaling+pathway+in+human+neuroglioma+cells%3A+In+vitro+and+in+vivo&doi=10.1016%2Fj.biopha.2016.08.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Quercetin nanoparticles induced autophagy and apoptosis through AKT/ERK/Caspase-3 signaling pathway in human neuroglioma cells: In vitro and in vivo</span></div><div class="casAuthors">Lou, Miao; Zhang, Li-na; Ji, Pei-gang; Feng, Fu-qiang; Liu, Jing-hui; Yang, Chen; Li, Bao-fu; Wang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Neuroglioma is a complex neuroglial tumor involving dysregulation of many biol. pathways at multiple levels.  Quercetin is a potent cancer therapeutic agent presented in fruit and vegetables, preventing tumor proliferation, and is a well known cancer therapeutic agent and autophagy mediator.  Recent studies showed that drug delivery by nanoparticles have enhanced efficacy with reduced side effects.  In this regard, gold-quercetin into poly (DL-lactide-co-glycolide) nanoparticles was examd.  In the present study, quercetin nanoparticle induced cell autophagy and apoptosis in human neuroglioma cell was investigated.  Quercetin nanoparticle administrated to animals displayed suppressed role in tumor growth.  The cell viability was detd. through CCK8 assay.  Transmission electron microscopy was utilized to observe the formation of autophagosome.  The cell apoptosis was assessed by annexin V-PI staining.  The protein expression of cell autophagy regulators and tumor suppressors were analyzed via western blot and RT-PCR.  Treatment of human neuroglioma cell with quercetin nanoparticle induced cell death in a dose-and time-dependent manner.  The flow cytometry results showed that the proportion of the apoptosis cells had gained after quercetin nanoparticle treatment compared to untreatment group.  Moreover, the expression of activated PI3K/AKT and Bcl-2 were down-regulated upon quercetin nanoparticle treatment in human neuroglioma cells.  The expression level of LC3 and ERK as well as cytoplasm p53, cleaved Caspase-3 and PARP was pos. correlated with the concn. of quercetin nanoparticle.  In addn., p-mTOR and GAIP were obviously down-regulated by quercetin nanoparticle treatment in a dose-dependent manner.  These results indicated that quercetin nanoparticle could induce autophagy and apoptosis in human neuroglioma cells, the underlying mol. mechanisms, at least partly, through activation LC3/ERK/Caspase-3 and suppression AKT/mTOR signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDVZ0FbEamuLVg90H21EOLACvtfcHk0ljfwCEGualWmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2rt7fK&md5=2695d98cbbe32b7ad4abbbaf6d916917</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2016.08.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2016.08.055%26sid%3Dliteratum%253Aachs%26aulast%3DLou%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DP.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DQuercetin%2520nanoparticles%2520induced%2520autophagy%2520and%2520apoptosis%2520through%2520AKT%252FERK%252FCaspase-3%2520signaling%2520pathway%2520in%2520human%2520neuroglioma%2520cells%253A%2520In%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2016%26volume%3D84%26spage%3D1%26epage%3D9%26doi%3D10.1016%2Fj.biopha.2016.08.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saresella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rainone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Daghri, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clerici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabattoni, D.</span></span> <span> </span><span class="NLM_article-title">The PD-1/PD-L1 pathway in human pathology</span>. <i>Curr. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.2174/156652412799218903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.2174%2F156652412799218903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22300137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFWisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=259-267&author=M.+Saresellaauthor=V.+Rainoneauthor=N.+M.+Al-Daghriauthor=M.+Clericiauthor=D.+Trabattoni&title=The+PD-1%2FPD-L1+pathway+in+human+pathology&doi=10.2174%2F156652412799218903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The PD-1/PD-L1 pathway in human pathology</span></div><div class="casAuthors">Saresella, M.; Rainone, V.; Al-Daghri, N. M.; Clerici, M.; Trabattoni, D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Molecular Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">259-267</span>CODEN:
                <span class="NLM_cas:coden">CMMUBP</span>;
        ISSN:<span class="NLM_cas:issn">1566-5240</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  T-cell activation is dependent on signals delivered through the antigen-specific T-cell receptor and accessory receptors on T-cells.  Integration of signals through this family of costimulatory and inhibitory receptors and their ligands regulates the balance between T-cell activation, tolerance, and immunopathol.  Programmed death 1 (PD-1) and its ligands, PD-L1 and PD-L2, deliver inhibitory signals and exert a vital and diverse range of immunoregulatory roles in T-cell activation, tolerance, and immune-mediated tissue damage.  In this review, we revisit current understanding of the immunoregulatory functions of PD-1 and its ligands and their involvement in immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhq3Juj_4LM7Vg90H21EOLACvtfcHk0ljfwCEGualWmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFWisr4%253D&md5=2407cedf7cb6bfe52bef42cf2e69169c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.2174%2F156652412799218903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156652412799218903%26sid%3Dliteratum%253Aachs%26aulast%3DSaresella%26aufirst%3DM.%26aulast%3DRainone%26aufirst%3DV.%26aulast%3DAl-Daghri%26aufirst%3DN.%2BM.%26aulast%3DClerici%26aufirst%3DM.%26aulast%3DTrabattoni%26aufirst%3DD.%26atitle%3DThe%2520PD-1%252FPD-L1%2520pathway%2520in%2520human%2520pathology%26jtitle%3DCurr.%2520Mol.%2520Med.%26date%3D2012%26volume%3D12%26spage%3D259%26epage%3D267%26doi%3D10.2174%2F156652412799218903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsuliman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harazi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitwi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulbah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Tweigeri, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Alwan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghebeh, H.</span></span> <span> </span><span class="NLM_article-title">Bidirectional crosstalk between PD-L1 expression and epithelial to mesenchymal transition: significance in claudin-low breast cancer cells</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1186/s12943-015-0421-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1186%2Fs12943-015-0421-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26245467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC287gtFKkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=149-161&author=A.+Alsulimanauthor=D.+Colakauthor=O.+Al-Haraziauthor=H.+Fitwiauthor=A.+Tulbahauthor=T.+Al-Tweigeriauthor=M.+Al-Alwanauthor=H.+Ghebeh&title=Bidirectional+crosstalk+between+PD-L1+expression+and+epithelial+to+mesenchymal+transition%3A+significance+in+claudin-low+breast+cancer+cells&doi=10.1186%2Fs12943-015-0421-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Bidirectional crosstalk between PD-L1 expression and epithelial to mesenchymal transition: significance in claudin-low breast cancer cells</span></div><div class="casAuthors">Alsuliman Abdullah; Fitwi Hanaa; Al-Alwan Monther; Ghebeh Hazem; Colak Dilek; Al-Harazi Olfat; Tulbah Asma; Al-Tweigeri Taher; Al-Alwan Monther; Ghebeh Hazem</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">149</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The T-cell inhibitory molecule PD-L1 (B7-H1, CD274) is expressed on tumor cells of a subset of breast cancer patients.  However, the mechanism that regulates PD-L1 expression in this group of patients is still not well-identified.  METHODS:  We have used loss and gain of function gene manipulation approach, multi-parametric flow cytometry, large scale gene expression dataset analysis and immunohistochemistry of breast cancer tissue sections.  RESULTS:  Induction of epithelial to mesenchymal transition (EMT) in human mammary epithelial cells upregulated PD-L1 expression, which was dependent mainly on the activation of the PI3K/AKT pathway.  Interestingly, gene expression signatures available from large cohort of breast tumors showed a significant correlation between EMT score and the PD-L1 mRNA level (p < 0.001).  Strikingly, very strong association (p < 0.0001) was found between PD-L1 expression and claudin-low subset of breast cancer, which is known to have high EMT score.  On the protein level, significant correlation was found between PD-L1 expression and standard markers of EMT (p = 0.005) in 67 breast cancer patients.  Importantly, specific downregulation of PD-L1 in claudin-low breast cancer cells showed signs of EMT reversal as manifested by CD44 and Vimentin downregulation and CD24 upregulation.  CONCLUSIONS:  We have demonstrated a bidirectional effect between EMT status and PD-L1 expression especially in claudin-low subtype of breast cancer cells.  Our findings highlights the potential dual benefit of anti-PD-L1 particularly in this subset of breast cancer patients that will likely benefit more from anti-PD-L1 targeted therapy as well as in monitoring biological changes upon treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcROs3-ADOt9csuCWE5mhx1RfW6udTcc2ebF1MZCeu89l7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287gtFKkuw%253D%253D&md5=e12a6f1cea0ef041f4eaff4968b49a6c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs12943-015-0421-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-015-0421-2%26sid%3Dliteratum%253Aachs%26aulast%3DAlsuliman%26aufirst%3DA.%26aulast%3DColak%26aufirst%3DD.%26aulast%3DAl-Harazi%26aufirst%3DO.%26aulast%3DFitwi%26aufirst%3DH.%26aulast%3DTulbah%26aufirst%3DA.%26aulast%3DAl-Tweigeri%26aufirst%3DT.%26aulast%3DAl-Alwan%26aufirst%3DM.%26aulast%3DGhebeh%26aufirst%3DH.%26atitle%3DBidirectional%2520crosstalk%2520between%2520PD-L1%2520expression%2520and%2520epithelial%2520to%2520mesenchymal%2520transition%253A%2520significance%2520in%2520claudin-low%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Cancer%26date%3D2015%26volume%3D14%26spage%3D149%26epage%3D161%26doi%3D10.1186%2Fs12943-015-0421-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourque, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chernova, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitz, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malenkovich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okazaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, M. C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Engagement of the PD-1 immunoinhibitory receptor by a novel B7 family member leads to negative regulation of lymphocyte activation</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">1027</span>– <span class="NLM_lpage">1034</span>, <span class="refDoi"> DOI: 10.1084/jem.192.7.1027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1084%2Fjem.192.7.1027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=11015443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntFSlt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2000&pages=1027-1034&author=G.+J.+Freemanauthor=A.+J.+Longauthor=Y.+Iwaiauthor=K.+Bourqueauthor=T.+Chernovaauthor=H.+Nishimuraauthor=L.+J.+Fitzauthor=N.+Malenkovichauthor=T.+Okazakiauthor=M.+C.+Byrne&title=Engagement+of+the+PD-1+immunoinhibitory+receptor+by+a+novel+B7+family+member+leads+to+negative+regulation+of+lymphocyte+activation&doi=10.1084%2Fjem.192.7.1027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Engagement of the PD-1 immunoinhibitory receptor by a novel B7 family member leads to negative regulation of lymphocyte activation</span></div><div class="casAuthors">Freeman, Gordon J.; Long, Andrew J.; Iwai, Yoshiko; Bourque, Karen; Chernova, Tatyana; Nishimura, Hiroyuki; Fitz, Lori J.; Malenkovich, Nelly; Okazaki, Taku; Byrne, Michael C.; Horton, Heidi F.; Fouser, Lynette; Carter, Laura; Ling, Vincent; Bowman, Michael R.; Carreno, Beatriz M.; Collins, Mary; Wood, Clive R.; Honjo, Tasuku</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1027-1034</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">PD-1 is an immunoinhibitory receptor expressed by activated T cells, B cells, and myeloid cells.  Mice deficient in PD-1 exhibit a breakdown of peripheral tolerance and demonstrate multiple autoimmune features.  The authors report here that the ligand of PD-1 (PD-L1) is a member of the B7 gene family.  Engagement of PD-1 by PD-L1 leads to the inhibition of T cell receptor-mediated lymphocyte proliferation and cytokine secretion.  In addn., PD-1 signaling can inhibit at least suboptimal levels of CD28-mediated costimulation.  PD-L1 is expressed by antigen-presenting cells, including human peripheral blood monocytes stimulated with interferon γ, and activated human and murine dendritic cells.  In addn., PD-L1 is expressed in nonlymphoid tissues such as heart and lung.  The relative levels of inhibitory PD-L1 and costimulatory B7-1/B7-2 signals on antigen-presenting cells may det. the extent of T cell activation and consequently the threshold between tolerance and autoimmunity.  PD-L1 expression on nonlymphoid tissues and its potential interaction with PD-1 may subsequently det. the extent of immune responses at sites of inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZIeEjiecqBbVg90H21EOLACvtfcHk0lgjN_SEmnnuIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntFSlt7o%253D&md5=bc1499033282cd9dbfa396a4e2697f6e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1084%2Fjem.192.7.1027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.192.7.1027%26sid%3Dliteratum%253Aachs%26aulast%3DFreeman%26aufirst%3DG.%2BJ.%26aulast%3DLong%26aufirst%3DA.%2BJ.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DBourque%26aufirst%3DK.%26aulast%3DChernova%26aufirst%3DT.%26aulast%3DNishimura%26aufirst%3DH.%26aulast%3DFitz%26aufirst%3DL.%2BJ.%26aulast%3DMalenkovich%26aufirst%3DN.%26aulast%3DOkazaki%26aufirst%3DT.%26aulast%3DByrne%26aufirst%3DM.%2BC.%26atitle%3DEngagement%2520of%2520the%2520PD-1%2520immunoinhibitory%2520receptor%2520by%2520a%2520novel%2520B7%2520family%2520member%2520leads%2520to%2520negative%2520regulation%2520of%2520lymphocyte%2520activation%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2000%26volume%3D192%26spage%3D1027%26epage%3D1034%26doi%3D10.1084%2Fjem.192.7.1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoo, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.-C.</span></span> <span> </span><span class="NLM_article-title">Glycosylation and stabilization of programmed death ligand-1 suppresses T-cell activity</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">12632</span>, <span class="refDoi"> DOI: 10.1038/ncomms12632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fncomms12632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27572267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=12632&author=C.-W.+Liauthor=S.-O.+Limauthor=W.+Xiaauthor=H.-H.+Leeauthor=L.-C.+Chanauthor=C.-W.+Kuoauthor=K.-H.+Khooauthor=S.-S.+Changauthor=J.-H.+Chaauthor=T.+Kimauthor=J.+L.+Hsuauthor=Y.+Wuauthor=J.-M.+Hsuauthor=H.+Yamaguchiauthor=Q.+Dingauthor=Y.+Wangauthor=J.+Yaoauthor=C.-C.+Leeauthor=H.-J.+Wuauthor=A.+A.+Sahinauthor=J.+P.+Allisonauthor=D.+Yuauthor=G.+N.+Hortobagyiauthor=M.-C.+Hung&title=Glycosylation+and+stabilization+of+programmed+death+ligand-1+suppresses+T-cell+activity&doi=10.1038%2Fncomms12632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glycosylation and stabilization of programmed death ligand-1 suppresses T-cell activity</span></div><div class="casAuthors">Li, Chia-Wei; Lim, Seung-Oe; Xia, Weiya; Lee, Heng-Huan; Chan, Li-Chuan; Kuo, Chu-Wei; Khoo, Kay-Hooi; Chang, Shih-Shin; Cha, Jong-Ho; Kim, Taewan; Hsu, Jennifer L.; Wu, Yun; Hsu, Jung-Mao; Yamaguchi, Hirohito; Ding, Qingqing; Wang, Yan; Yao, Jun; Lee, Cheng-Chung; Wu, Hsing-Ju; Sahin, Aysegul A.; Allison, James P.; Yu, Dihua; Hortobagyi, Gabriel N.; Hung, Mien-Chie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12632</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Extracellular interaction between programmed death ligand-1 (PD-L1) and programmed cell death protein-1 (PD-1) leads to tumor-assocd. immune escape.  Here we show that the immunosuppression activity of PD-L1 is stringently modulated by ubiquitination and N-glycosylation.  We show that glycogen synthase kinase 3β (GSK3β) interacts with PD-L1 and induces phosphorylation-dependent proteasome degrdn. of PD-L1 by β-TrCP.  In-depth anal. of PD-L1 N192, N200 and N219 glycosylation suggests that glycosylation antagonizes GSK3β binding.  In this regard, only non-glycosylated PD-L1 forms a complex with GSK3β and β-TrCP.  We also demonstrate that epidermal growth factor (EGF) stabilizes PD-L1 via GSK3β inactivation in basal-like breast cancer.  Inhibition of EGF signalling by gefitinib destabilizes PD-L1, enhances antitumor T-cell immunity and therapeutic efficacy of PD-1 blockade in syngeneic mouse models.  Together, our results link ubiquitination and glycosylation pathways to the stringent regulation of PD-L1, which could lead to potential therapeutic strategies to enhance cancer immune therapy efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQnyTgbFXcD7Vg90H21EOLACvtfcHk0lgjN_SEmnnuIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbnJ&md5=35cab8a1af28193769fe685c0e1f37b7</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fncomms12632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12632%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.-W.%26aulast%3DLim%26aufirst%3DS.-O.%26aulast%3DXia%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DH.-H.%26aulast%3DChan%26aufirst%3DL.-C.%26aulast%3DKuo%26aufirst%3DC.-W.%26aulast%3DKhoo%26aufirst%3DK.-H.%26aulast%3DChang%26aufirst%3DS.-S.%26aulast%3DCha%26aufirst%3DJ.-H.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DHsu%26aufirst%3DJ.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DHsu%26aufirst%3DJ.-M.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DC.-C.%26aulast%3DWu%26aufirst%3DH.-J.%26aulast%3DSahin%26aufirst%3DA.%2BA.%26aulast%3DAllison%26aufirst%3DJ.%2BP.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DHung%26aufirst%3DM.-C.%26atitle%3DGlycosylation%2520and%2520stabilization%2520of%2520programmed%2520death%2520ligand-1%2520suppresses%2520T-cell%2520activity%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D12632%26doi%3D10.1038%2Fncomms12632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ihara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashiwa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukamizu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohmo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagatomo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kijima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tachibana, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumanogoh, A.</span></span> <span> </span><span class="NLM_article-title">Inhibitory roles of signal transducer and activator of transcription 3 in antitumor immunity during carcinogen-induced lung tumorigenesis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2990</span>– <span class="NLM_lpage">2999</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-4062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F0008-5472.CAN-11-4062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22659452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1yjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2990-2999&author=S.+Iharaauthor=H.+Kidaauthor=H.+Araseauthor=L.+P.+Tripathiauthor=Y.+Chenauthor=T.+Kimuraauthor=M.+Yoshidaauthor=Y.+Kashiwaauthor=H.+Hirataauthor=R.+Fukamizuauthor=R.+Inoueauthor=K.+Hasegawaauthor=S.+Goyaauthor=R.+Takahashiauthor=T.+Minamiauthor=K.+Tsujinoauthor=M.+Suzukiauthor=S.+Kohmoauthor=K.+Inoueauthor=I.+Nagatomoauthor=Y.+Takedaauthor=T.+Kijimaauthor=K.+Mizuguchiauthor=I.+Tachibanaauthor=A.+Kumanogoh&title=Inhibitory+roles+of+signal+transducer+and+activator+of+transcription+3+in+antitumor+immunity+during+carcinogen-induced+lung+tumorigenesis&doi=10.1158%2F0008-5472.CAN-11-4062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitory Roles of Signal Transducer and Activator of Transcription 3 in Antitumor Immunity during Carcinogen-Induced Lung Tumorigenesis</span></div><div class="casAuthors">Ihara, Shoichi; Kida, Hiroshi; Arase, Hisashi; Tripathi, Lokesh P.; Chen, Yi-An; Kimura, Tetsuya; Yoshida, Mitsuhiro; Kashiwa, Yozo; Hirata, Haruhiko; Fukamizu, Reiko; Inoue, Ruriko; Hasegawa, Kana; Goya, Sho; Takahashi, Ryo; Minami, Toshiyuki; Tsujino, Kazuyuki; Suzuki, Mayumi; Kohmo, Satoshi; Inoue, Koji; Nagatomo, Izumi; Takeda, Yoshito; Kijima, Takashi; Mizuguchi, Kenji; Tachibana, Isao; Kumanogoh, Atsushi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2990-2999</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Stat3 mediates a complex spectrum of cellular responses, including inflammation, cell proliferation, and apoptosis.  Although evidence exists in support of a pos. role for Stat3 in cancer, its role has remained somewhat controversial because of insufficient study of how its genetic deletion may affect carcinogenesis in various tissues.  In this study, we show using epithelium-specific knockout mice (Stat3Δ/Δ) that Stat3 blunts rather than supports antitumor immunity in carcinogen-induced lung tumorigenesis.  Although Stat3A/A mice did not show any lung defects in terms of proliferation, apoptosis, or angiogenesis, they exhibited reduced urethane-induced tumorigenesis and increased antitumor inflammation and natural killer (NK) cell immunity.  Comparative microarray anal. revealed an increase in Stat3Δ/Δ tumors in proinflammatory chemokine prodn. and a decrease in MHC class I antigen expression assocd. with NK cell recognition.  Consistent with these findings, human non-small cell lung cancer (NSCLC) cells in which Stat3 was silenced displayed an enhancement of proinflammatory chemokine prodn., reduced expression of MHC class I antigen, and increased susceptibility to NKcell-mediated cytotoxicity.  In addn., supernatants from Stat3-silenced NSCLC cells promoted monocyte migration.  Collectively, our findings argue that Stat3 exertsan inhibitory effect on antitumor NK cell immunity in the setting of carcinogen-induced tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK9a9e8642gLVg90H21EOLACvtfcHk0lgFWadoVHeGEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1yjurc%253D&md5=6376b221b272dac5b4907a0b52bc9e45</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-4062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-4062%26sid%3Dliteratum%253Aachs%26aulast%3DIhara%26aufirst%3DS.%26aulast%3DKida%26aufirst%3DH.%26aulast%3DArase%26aufirst%3DH.%26aulast%3DTripathi%26aufirst%3DL.%2BP.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DKashiwa%26aufirst%3DY.%26aulast%3DHirata%26aufirst%3DH.%26aulast%3DFukamizu%26aufirst%3DR.%26aulast%3DInoue%26aufirst%3DR.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DGoya%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DR.%26aulast%3DMinami%26aufirst%3DT.%26aulast%3DTsujino%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DKohmo%26aufirst%3DS.%26aulast%3DInoue%26aufirst%3DK.%26aulast%3DNagatomo%26aufirst%3DI.%26aulast%3DTakeda%26aufirst%3DY.%26aulast%3DKijima%26aufirst%3DT.%26aulast%3DMizuguchi%26aufirst%3DK.%26aulast%3DTachibana%26aufirst%3DI.%26aulast%3DKumanogoh%26aufirst%3DA.%26atitle%3DInhibitory%2520roles%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520in%2520antitumor%2520immunity%2520during%2520carcinogen-induced%2520lung%2520tumorigenesis%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2990%26epage%3D2999%26doi%3D10.1158%2F0008-5472.CAN-11-4062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Old, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, M. J.</span></span> <span> </span><span class="NLM_article-title">Cancer immunoediting: integrating immunity’s roles in cancer suppression and promotion</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>331</i></span>,  <span class="NLM_fpage">1565</span>– <span class="NLM_lpage">1570</span>, <span class="refDoi"> DOI: 10.1126/science.1203486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1126%2Fscience.1203486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21436444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=331&publication_year=2011&pages=1565-1570&author=R.+D.+Schreiberauthor=L.+J.+Oldauthor=M.+J.+Smyth&title=Cancer+immunoediting%3A+integrating+immunity%E2%80%99s+roles+in+cancer+suppression+and+promotion&doi=10.1126%2Fscience.1203486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Immunoediting: Integrating Immunity's Roles in Cancer Suppression and Promotion</span></div><div class="casAuthors">Schreiber, Robert D.; Old, Lloyd J.; Smyth, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">331</span>
        (<span class="NLM_cas:issue">6024</span>),
    <span class="NLM_cas:pages">1565-1570</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Understanding how the immune system affects cancer development and progression has been one of the most challenging questions in immunol.  Research over the past two decades has helped explain why the answer to this question has evaded the authors for so long.  The authors now appreciate that the immune system plays a dual role in cancer: It can not only suppress tumor growth by destroying cancer cells or inhibiting their outgrowth but also promote tumor progression either by selecting for tumor cells that are more fit to survive in an immunocompetent host or by establishing conditions within the tumor microenvironment that facilitate tumor outgrowth.  Here, the authors discuss a unifying conceptual framework called "cancer immunoediting," which integrates the immune system's dual host-protective and tumor-promoting roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpokt_z4WtBy7Vg90H21EOLACvtfcHk0lgFWadoVHeGEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbs%253D&md5=cebc33b67a44d78fa6e75af050114d4d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1126%2Fscience.1203486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1203486%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DR.%2BD.%26aulast%3DOld%26aufirst%3DL.%2BJ.%26aulast%3DSmyth%26aufirst%3DM.%2BJ.%26atitle%3DCancer%2520immunoediting%253A%2520integrating%2520immunity%25E2%2580%2599s%2520roles%2520in%2520cancer%2520suppression%2520and%2520promotion%26jtitle%3DScience%26date%3D2011%26volume%3D331%26spage%3D1565%26epage%3D1570%26doi%3D10.1126%2Fscience.1203486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marzec, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goradia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghunath, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paessler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wysocka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasik, M. A.</span></span> <span> </span><span class="NLM_article-title">Oncogenic kinase NPM/ALK induces through STAT3 expression of immunosuppressive protein CD274 (PD-L1, B7-H1)</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">20852</span>– <span class="NLM_lpage">20857</span>, <span class="refDoi"> DOI: 10.1073/pnas.0810958105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1073%2Fpnas.0810958105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=19088198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksFSntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=20852-20857&author=M.+Marzecauthor=Q.+Zhangauthor=A.+Goradiaauthor=P.+N.+Raghunathauthor=X.+Liuauthor=M.+Paesslerauthor=H.+Wangauthor=M.+Wysockaauthor=M.+Chengauthor=B.+A.+Ruggeriauthor=M.+A.+Wasik&title=Oncogenic+kinase+NPM%2FALK+induces+through+STAT3+expression+of+immunosuppressive+protein+CD274+%28PD-L1%2C+B7-H1%29&doi=10.1073%2Fpnas.0810958105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic kinase NPM/ALK induces through STAT3 expression of immunosuppressive protein CD274 (PD-L1, B7-H1)</span></div><div class="casAuthors">Marzec, Michal; Zhang, Qian; Goradia, Ami; Raghunath, Puthiyaveettil N.; Liu, Xiaobin; Paessler, Michele; Wang, Hong Yi; Wysocka, Maria; Cheng, Mangeng; Ruggeri, Bruce A.; Wasik, Mariusz A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">20852-20857</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The mechanisms of malignant cell transformation caused by the oncogenic, chimeric nucleophosmin (NPM)/anaplastic lymphoma kinase (ALK) remain only partially understood, with most of the previous studies focusing mainly on the impact of NPM/ALK on cell survival and proliferation.  Here we report that the NPM/ALK-carrying T cell lymphoma (ALK + TCL) cells strongly express the immunosuppressive cell-surface protein CD274 (PD-L1, B7-H1), as detd. on the mRNA and protein level.  The CD274 expression is strictly dependent on the expression and enzymic activity of NPM/ALK, as demonstrated by inhibition of the NPM/ALK function in ALK + TCL cells by the small mol. ALK inhibitor CEP-14083 and by documenting CD274 expression in IL-3-depleted BaF3 cells transfected with the wild-type NPM/ALK, but not the kinase-inactive NPM/ALK K210R mutant or empty vector alone.  NPM/ALK induces CD274 expression by activating its key signal transmitter, transcription factor STAT3.  STAT3 binds to the CD274 gene promoter in vitro and in vivo, as shown in the gel electromobility shift and chromatin immunopptn. assays, and is required for the PD-L1 gene expression, as demonstrated by siRNA-mediated STAT3 depletion.  These findings identify an addnl. cell-transforming property of NPM/ALK and describe a direct link between an oncoprotein and an immunosuppressive cell-surface protein.  These results also provide an addnl. rationale to therapeutically target NPM/ALK and STAT3 in ALK + TCL.  Finally, they suggest that future immunotherapeutic protocols for this type of lymphoma may need to include the inhibition of NPM/ALK and STAT3 to achieve optimal clin. efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh87qHOgigfrVg90H21EOLACvtfcHk0lgFWadoVHeGEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksFSntw%253D%253D&md5=1e85c07581e48e5aa0b9c7d685b57eef</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0810958105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0810958105%26sid%3Dliteratum%253Aachs%26aulast%3DMarzec%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DGoradia%26aufirst%3DA.%26aulast%3DRaghunath%26aufirst%3DP.%2BN.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DPaessler%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWysocka%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DRuggeri%26aufirst%3DB.%2BA.%26aulast%3DWasik%26aufirst%3DM.%2BA.%26atitle%3DOncogenic%2520kinase%2520NPM%252FALK%2520induces%2520through%2520STAT3%2520expression%2520of%2520immunosuppressive%2520protein%2520CD274%2520%2528PD-L1%252C%2520B7-H1%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D20852%26epage%3D20857%26doi%3D10.1073%2Fpnas.0810958105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, l.</span></span> <span> </span><span class="NLM_article-title">EBV-driven LMP1 and IFN-gamma up-regulate PD-L1 in nasopharyngeal carcinoma: implications for oncotargeted therapy</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">12189</span>– <span class="NLM_lpage">12202</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.18632%2Foncotarget.2608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25361008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC2M3kvVWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=12189-12202&author=W.+Fangauthor=J.+Zhangauthor=S.+Hongauthor=J.+Zhanauthor=N.+Chenauthor=T.+Qinauthor=Y.+Tangauthor=Y.+Zhangauthor=S.+Kangauthor=T.+Zhouauthor=X.+Wuauthor=W.+Liangauthor=Z.+Huauthor=Y.+Maauthor=Y.+Zhaoauthor=Y.+Tianauthor=Y.+Yangauthor=C.+Xueauthor=Y.+Yanauthor=X.+Houauthor=P.+Huangauthor=Y.+Huangauthor=H.+Zhaoauthor=l.+Zhang&title=EBV-driven+LMP1+and+IFN-gamma+up-regulate+PD-L1+in+nasopharyngeal+carcinoma%3A+implications+for+oncotargeted+therapy&doi=10.18632%2Foncotarget.2608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">EBV-driven LMP1 and IFN-γ up-regulate PD-L1 in nasopharyngeal carcinoma: Implications for oncotargeted therapy</span></div><div class="casAuthors">Fang Wenfeng; Hong Shaodong; Zhan Jianhua; Qin Tao; Zhang Yaxiong; Kang Shiyang; Zhou Ting; Wu Xuan; Liang Wenhua; Hu Zhihuang; Ma Yuxiang; Zhao Yuanyuan; Tian Ying; Yang Yunpeng; Xue Cong; Yan Yue; Hou Xue; Huang Peiyu; Huang Yan; Zhao Hongyun; Zhang Li; Zhang Jianwei; Chen Nan; Tang Yanna</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">12189-202</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PD-L1 expression is a feature of Epstein-Barr virus (EBV) associated malignancies such as nasopharyngeal carcinoma (NPC).  Here, we found that EBV-induced latent membrane protein 1 (LMP1) and IFN-γ pathways cooperate to regulate programmed cell death protein 1 ligand (PD-L1).  Expression of PD-L1 was higher in EBV positive NPC cell lines compared with EBV negative cell lines.  PD-L1 expression could be increased by exogenous and endogenous induction of LMP1 induced PD-L1.  In agreement, expression of PD-L1 was suppressed by knocking down LMP1 in EBV positive cell lines.  We further demonstrated that LMP1 up-regulated PD-L1 through STAT3, AP-1, and NF-κB pathways.  Besides, IFN-γ was independent of but synergetic with LMP1 in up-regulating PD-L1 in NPC.  Furthermore, we showed that PD-L1 was associated with worse disease-free survival in NPC patients.  These results imply that blocking both the LMP1 oncogenic pathway and PD-1/PD-L1 checkpoints may be a promising therapeutic approach for EBV positive NPC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRaEd7bN61FfO_Q9URG6Dx8fW6udTcc2eZlgZw1MoqdWrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3kvVWktg%253D%253D&md5=9e720a88e92ca72d470b12e9c5f6de94</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2608%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DS.%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DN.%26aulast%3DQin%26aufirst%3DT.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3Dl.%26atitle%3DEBV-driven%2520LMP1%2520and%2520IFN-gamma%2520up-regulate%2520PD-L1%2520in%2520nasopharyngeal%2520carcinoma%253A%2520implications%2520for%2520oncotargeted%2520therapy%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D12189%26epage%3D12202%26doi%3D10.18632%2Foncotarget.2608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gato-Cañas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuazo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arasanz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibañez-Vea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Hinojal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smerdou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martisova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arozarena, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellbrock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llopiz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarobe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breckpot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escors, D.</span></span> <span> </span><span class="NLM_article-title">PDL1 Signals through Conserved Sequence Motifs to Overcome Interferon-Mediated Cytotoxicity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1818</span>– <span class="NLM_lpage">1829</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.07.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.celrep.2017.07.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28834746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=1818-1829&author=M.+Gato-Ca%C3%B1asauthor=M.+Zuazoauthor=H.+Arasanzauthor=M.+Iba%C3%B1ez-Veaauthor=L.+Lorenzoauthor=G.+Fernandez-Hinojalauthor=R.+Veraauthor=C.+Smerdouauthor=E.+Martisovaauthor=I.+Arozarenaauthor=C.+Wellbrockauthor=D.+Llopizauthor=M.+Ruizauthor=P.+Sarobeauthor=K.+Breckpotauthor=G.+Kochanauthor=D.+Escors&title=PDL1+Signals+through+Conserved+Sequence+Motifs+to+Overcome+Interferon-Mediated+Cytotoxicity&doi=10.1016%2Fj.celrep.2017.07.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">PDL1 Signals through Conserved Sequence Motifs to Overcome Interferon-Mediated Cytotoxicity</span></div><div class="casAuthors">Gato-Canas, Maria; Zuazo, Miren; Arasanz, Hugo; Ibanez-Vea, Maria; Lorenzo, Laura; Fernandez-Hinojal, Gonzalo; Vera, Ruth; Smerdou, Cristian; Martisova, Eva; Arozarena, Imanol; Wellbrock, Claudia; Llopiz, Diana; Ruiz, Marta; Sarobe, Pablo; Breckpot, Karine; Kochan, Grazyna; Escors, David</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1818-1829</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">PDL1 blockade produces remarkable clin. responses, thought to occur by T cell reactivation through prevention of PDL1-PD1 T cell inhibitory interactions.  Here, we find that PDL1 cell-intrinsic signaling protects cancer cells from interferon (IFN) cytotoxicity and accelerates tumor progression.  PDL1 inhibited IFN signal transduction through a conserved class of sequence motifs that mediate crosstalk with IFN signaling.  Abrogation of PDL1 expression or antibody-mediated PDL1 blockade strongly sensitized cancer cells to IFN cytotoxicity through a STAT3/caspase-7-dependent pathway.  Moreover, somatic mutations found in human carcinomas within these PDL1 sequence motifs disrupted motif regulation, resulting in PDL1 mols. with enhanced protective activities from type I and type II IFN cytotoxicity.  Overall, our results reveal a mode of action of PDL1 in cancer cells as a first line of defense against IFN cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYq1BKCEgRMLVg90H21EOLACvtfcHk0ljF3S2SuW0ZfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLzL&md5=8ceac0a22cc56a0778bd1e4bdd3acf32</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.07.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.07.075%26sid%3Dliteratum%253Aachs%26aulast%3DGato-Ca%25C3%25B1as%26aufirst%3DM.%26aulast%3DZuazo%26aufirst%3DM.%26aulast%3DArasanz%26aufirst%3DH.%26aulast%3DIba%25C3%25B1ez-Vea%26aufirst%3DM.%26aulast%3DLorenzo%26aufirst%3DL.%26aulast%3DFernandez-Hinojal%26aufirst%3DG.%26aulast%3DVera%26aufirst%3DR.%26aulast%3DSmerdou%26aufirst%3DC.%26aulast%3DMartisova%26aufirst%3DE.%26aulast%3DArozarena%26aufirst%3DI.%26aulast%3DWellbrock%26aufirst%3DC.%26aulast%3DLlopiz%26aufirst%3DD.%26aulast%3DRuiz%26aufirst%3DM.%26aulast%3DSarobe%26aufirst%3DP.%26aulast%3DBreckpot%26aufirst%3DK.%26aulast%3DKochan%26aufirst%3DG.%26aulast%3DEscors%26aufirst%3DD.%26atitle%3DPDL1%2520Signals%2520through%2520Conserved%2520Sequence%2520Motifs%2520to%2520Overcome%2520Interferon-Mediated%2520Cytotoxicity%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D1818%26epage%3D1829%26doi%3D10.1016%2Fj.celrep.2017.07.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bill, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofori, G.</span></span> <span> </span><span class="NLM_article-title">The relevance of EMT in breast cancer metastasis: Correlation or causality?</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>589</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1587</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.febslet.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25979173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVaktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=1577-1587&author=R.+Billauthor=G.+Christofori&title=The+relevance+of+EMT+in+breast+cancer+metastasis%3A+Correlation+or+causality%3F&doi=10.1016%2Fj.febslet.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The relevance of EMT in breast cancer metastasis: Correlation or causality?</span></div><div class="casAuthors">Bill, Ruben; Christofori, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">589</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1577-1587</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although major progress has been achieved in treating breast cancer patients, metastatic breast cancer still remains a deadly disease.  A full understanding of the process of systemic cancer cell dissemination is therefore crit. to develop next generation therapies.  A plethora of exptl. data points toward a central role of an epithelial to mesenchymal transition (EMT) in the multistep cascade of metastasis formation.  However, in patients the data are based on correlative studies which often, but not always, tie the expression of EMT markers to cancer invasion, metastasis and poor clin. outcome.  Moreover, the notion that cancer cells are able to switch between different modes of migration asks for a thorough review of the actual relevance of EMT in cancer metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eoOwCOyzM7Vg90H21EOLACvtfcHk0lgls08_1I7CYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVaktro%253D&md5=78269348415035917bd165fae4553cdd</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DBill%26aufirst%3DR.%26aulast%3DChristofori%26aufirst%3DG.%26atitle%3DThe%2520relevance%2520of%2520EMT%2520in%2520breast%2520cancer%2520metastasis%253A%2520Correlation%2520or%2520causality%253F%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D1577%26epage%3D1587%26doi%3D10.1016%2Fj.febslet.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span> <span> </span><span class="NLM_article-title">PD-L1 expression as poor prognostic factor in patients with non-squamous non-small cell lung cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">58457</span>– <span class="NLM_lpage">58468</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.17022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.18632%2Foncotarget.17022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28938570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FhtlOjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=58457-58468&author=C.+Zhouauthor=J.+Tangauthor=H.+Sunauthor=X.+Zhengauthor=Z.+Liauthor=T.+Sunauthor=J.+Liauthor=S.+Wangauthor=X.+Zhouauthor=H.+Sunauthor=Z.+Chengauthor=H.+Zhangauthor=H.+Ma&title=PD-L1+expression+as+poor+prognostic+factor+in+patients+with+non-squamous+non-small+cell+lung+cancer&doi=10.18632%2Foncotarget.17022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1 expression as poor prognostic factor in patients with non-squamous non-small cell lung cancer</span></div><div class="casAuthors">Zhou Cuiling; Sun Huanhuan; Wang Shuncong; Zhou Xiuling; Cheng Zhibin; Zhang Hongyu; Ma Haiqing; Tang Jianjun; Zheng Xiaobin; Li Zhanyu; Sun Tiantian; Li Jie; Sun Hongliu</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">58457-58468</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  The role of programmed cell death ligand 1 (PD-L1) in non-small cell lung cancer (NSCLC), especially according to histologic type, remains controversial.  The purpose of this study was to assess PD-L1 expression and its association with overall survival (OS) and clinicopathologic characteristics in NSCLC.  MATERIALS AND METHODS:  Formalin-fixed paraffin-embedded specimens were obtained from 108 patients with surgically resected primary NSCLC.  PD-L1 expression was assessed via immunohistochemistry using a histochemistry score system.  The relationship between OS or clinicopathologic characteristics and PD-L1 expression was evaluated via the Kaplan-Meier method and Cox proportional hazards model, respectively.  RESULTS:  Of 108 NSCLC specimens, 44 had high PD-L1 expression, which was highly associated with histologic type (p = 0.003).  Patients without PD-L1 expression had remarkably longer OS than those with PD-L1 expression (median OS: 96 months vs. 33 months, p < 0.001).  In the subgroup analysis of non-squamous cell carcinoma, OS was more favorable in those without PD-L1 expression than in those with PD-L1 expression (median OS: 113 months vs. 37 months, p < 0.001).  Multivariate analysis revealed that PD-L1 expression (95% confidence interval 1.459-4.520, p < 0.001), male sex and higher tumor-node-metastasis stage were significantly correlated with shorter OS.  CONCLUSIONS:  This study demonstrated that PD-L1 expression is an independent prognostic factor for poor survival in NSCLC patients, especially those with non-squamous NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRcU4AE24nN2tFxYDaSSP8XfW6udTcc2ebkAwEfbQOBKrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FhtlOjug%253D%253D&md5=37fae906831a163f17e3205c84145d9b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.17022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.17022%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DH.%26atitle%3DPD-L1%2520expression%2520as%2520poor%2520prognostic%2520factor%2520in%2520patients%2520with%2520non-squamous%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D58457%26epage%3D58468%26doi%3D10.18632%2Foncotarget.17022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritprajak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashiguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, M.</span></span> <span> </span><span class="NLM_article-title">B7-H1 overexpression regulates epithelial -mesenchymal transition and accelerates carcinogenesis in skin</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1243</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-2217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F0008-5472.CAN-10-2217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21159661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFSjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=1235-1243&author=Y.+Caoauthor=L.+Zhangauthor=Y.+Kamimuraauthor=P.+Ritprajakauthor=M.+Hashiguchiauthor=S.+Hiroseauthor=M.+Azuma&title=B7-H1+overexpression+regulates+epithelial+-mesenchymal+transition+and+accelerates+carcinogenesis+in+skin&doi=10.1158%2F0008-5472.CAN-10-2217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">B7-H1 Overexpression Regulates Epithelial-Mesenchymal Transition and Accelerates Carcinogenesis in Skin</span></div><div class="casAuthors">Cao, Yu-Jia; Zhang, Lu; Kamimura, Yosuke; Ritprajak, Patcharee; Hashiguchi, Masaaki; Hirose, Sachiko; Azuma, Miyuki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1235-1243</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">B7-H1 (CD274) is a T-cell coinhibitory mol. that is also often induced on human carcinoma cells, where its expression has been implicated in immune escape.  Under inflammatory conditions, B7-H1 is also inducible in normal epithelial cells but little is known about its involvement in conversion of normal cells to tumor cells.  Here, we show that skin-specific expression of B7-H1 accelerates inflammatory carcinogenesis in a methylcholantrene (MCA)-induced model of squamous cell carcinoma (SCC).  Inflammatory responses induced by MCA or phorbol ester TPA were clearly inhibited in B7-H1 transgenic mice (B7-H1tg mice).  Antibody-mediated blockade of either B7-H1 or the related mol. PD-1 revealed that their ability to limit inflammation relied on ligand interactions made by B7-H1 or PD-1.  Skin keratinocytes derived from B7-H1tg mice exhibited constitutive redn. of E-cadherin, and SCC induced in B7-H1tg mice also showed loss of E-cadherin along with elevated expression of the transcription factors Slug and Twist that drive epithelial-mesenchymal transition (EMT).  Our results indicate that upregulation of B7-H1 in skin epithelial cells promotes EMT and accelerates carcinogenesis, revealing insights into the significance of B7-H1 overexpression on solid tumor cells and hinting at a close relationship between EMT and immune escape signaling pathways in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_SlDDyuBmp7Vg90H21EOLACvtfcHk0lhzydK8U18dqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFSjs7c%253D&md5=bbf41d72feef2fc94b6ba2b72e8cc56b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2217%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DKamimura%26aufirst%3DY.%26aulast%3DRitprajak%26aufirst%3DP.%26aulast%3DHashiguchi%26aufirst%3DM.%26aulast%3DHirose%26aufirst%3DS.%26aulast%3DAzuma%26aufirst%3DM.%26atitle%3DB7-H1%2520overexpression%2520regulates%2520epithelial%2520-mesenchymal%2520transition%2520and%2520accelerates%2520carcinogenesis%2520in%2520skin%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D1235%26epage%3D1243%26doi%3D10.1158%2F0008-5472.CAN-10-2217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Way, T.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C.-T.</span></span> <span> </span><span class="NLM_article-title">Emodin represses TWIST1-induced epithelial-mesenchymal transitions in head and neck squamous cell carcinoma cells by inhibiting the β-catenin and Akt pathways</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2013.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejca.2013.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=24157255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSmt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=366-378&author=T.-D.+Wayauthor=J.-T.+Huangauthor=C.-H.+Chouauthor=C.-H.+Huangauthor=M.-H.+Yangauthor=C.-T.+Ho&title=Emodin+represses+TWIST1-induced+epithelial-mesenchymal+transitions+in+head+and+neck+squamous+cell+carcinoma+cells+by+inhibiting+the+%CE%B2-catenin+and+Akt+pathways&doi=10.1016%2Fj.ejca.2013.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Emodin represses TWIST1-induced epithelial-mesenchymal transitions in head and neck squamous cell carcinoma cells by inhibiting the β-catenin and Akt pathways</span></div><div class="casAuthors">Way, Tzong-Der; Huang, Jhen-Ting; Chou, Chun-Hung; Huang, Chi-Hung; Yang, Muh-Hwa; Ho, Chi-Tang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">366-378</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Head and neck squamous cell carcinoma (HNSCC) is one of the leading causes of cancer deaths worldwide.  In recent studies, a crucial link has been discovered between the acquisition of metastatic traits and tumor-initiating abilities in cancer cells during the epithelial-mesenchymal transition (EMT).  Herein, we demonstrated that the ectopic expression of TWIST1, the EMT regulator, in HNSCC FaDu cells triggered EMT and resulted in the acquisition of a mesenchymal phenotype.  Moreover, FaDu-pFLAG-TWIST1 cancer cell populations that were induced to EMT displayed an increased proportion of cells with the CD44 marker, which is assocd. with tumor initiation.  Interestingly, we found that emodin treatment reduced the tumor-initiating abilities and inhibited cell migration and invasion in FaDu-pFLAG-TWIST1 cells.  Emodin directly inhibited TWIST1 expression, upregulated E-cadherin mRNA and protein expression, and downregulated vimentin mRNA and protein expression.  Moreover, we found that emodin inhibited TWIST1 binding to the E-cadherin promoter and repressed E-cadherin transcription activity.  We also found that emodin inhibited TWIST1-induced EMT by inhibiting the β-catenin and Akt pathways.  More interestingly, emodin significantly inhibited TWIST1-induced invasion in vivo.  Therefore, emodin might be applicable to anticancer therapy and could be a potential new therapeutic drug for HNSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXZ43nASnzrVg90H21EOLACvtfcHk0lhzydK8U18dqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSmt73M&md5=6a3905b32c13912e073acea500a1e6cb</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2013.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2013.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DWay%26aufirst%3DT.-D.%26aulast%3DHuang%26aufirst%3DJ.-T.%26aulast%3DChou%26aufirst%3DC.-H.%26aulast%3DHuang%26aufirst%3DC.-H.%26aulast%3DYang%26aufirst%3DM.-H.%26aulast%3DHo%26aufirst%3DC.-T.%26atitle%3DEmodin%2520represses%2520TWIST1-induced%2520epithelial-mesenchymal%2520transitions%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%2520cells%2520by%2520inhibiting%2520the%2520%25CE%25B2-catenin%2520and%2520Akt%2520pathways%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2014%26volume%3D50%26spage%3D366%26epage%3D378%26doi%3D10.1016%2Fj.ejca.2013.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and the interaction of 2-(1H-pyrazol-4-yl)-1H-imidazo[4,5-f][1,10]phenanthrolines with telomeric DNA as lung cancer inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejmech.2017.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28376371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVCit70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=36-49&author=J.+Liuauthor=M.+Chenauthor=Y.+Wangauthor=X.+Zhaoauthor=S.+Wangauthor=Y.+Wuauthor=W.+Zhang&title=Synthesis+and+the+interaction+of+2-%281H-pyrazol-4-yl%29-1H-imidazo%5B4%2C5-f%5D%5B1%2C10%5Dphenanthrolines+with+telomeric+DNA+as+lung+cancer+inhibitors&doi=10.1016%2Fj.ejmech.2017.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and the interaction of 2-(1H-pyrazol-4-yl)-1H-imidazo[4,5-f][1,10]phenanthrolines with telomeric DNA as lung cancer inhibitors</span></div><div class="casAuthors">Liu, Jiachun; Chen, Mei; Wang, Yanli; Zhao, Xiaoyin; Wang, Sijia; Wu, Yanling; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36-49</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A novel series of 2-(1H-pyrazol-4-yl)-1H-imidazo[4,5-f][1,10]phenanthrolines I [R1 = F, Cl, Br, CH3, 4-BrC6H4; R2 = H, CH3, Bn, 2,4-(O2N)2C6H3 etc.] was synthesized via Vilsmeier formylation of semicarbazones of substituted acetophenones to yield aryl substituted pyrazole carboxaldehydes which furher condensed with phendione.  These newly synthesized compds. were evaluated for their antitumor activity against lung adenocarcinoma by CCK-8 assay, electrophoretic mobility shift assay (EMSA), UV-melting study, wound healing assay and docking study.  These compds. showed good inhibitory activities against lung adenocarcinoma; esp. compd. I [R1 = Br, R2 = Bn] exhibited potential antiproliferative activity against A549 cell line with the half maximal inhibitory concn. (IC50) value of 1.48 μM, which was a more potent inhibitor than cisplatin (IC50 = 12.08 μM), and the max. cell inhibitory rate being up to 98.40 per cent.  Compds. I [R1 = F, Cl, Br, CH3; R2 = Bn] could strongly interact with telomeric DNA to stabilize G-quadruplex DNA with increased ΔTm values from 12.44 to 20.54°C at a ratio of DNA to compd. 1:10.  These results implied that growth inhibition of A549 cells mediated by these phenanthroline derivs. was pos. correlated to the fact their interaction with telomeric G-quadruplexs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz4vS_d_Fzr7Vg90H21EOLACvtfcHk0lil4e1GG5VXXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVCit70%253D&md5=6bb4fc4e5ae7c956d51d4e12ba91d910</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DSynthesis%2520and%2520the%2520interaction%2520of%25202-%25281H-pyrazol-4-yl%2529-1H-imidazo%255B4%252C5-f%255D%255B1%252C10%255Dphenanthrolines%2520with%2520telomeric%2520DNA%2520as%2520lung%2520cancer%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D133%26spage%3D36%26epage%3D49%26doi%3D10.1016%2Fj.ejmech.2017.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_label">(66) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He Wei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Formation and stabilization of the telomeric antiparallel G-quadruplex and inhibition of telomerase by novel benzothioxanthene derivatives with anti-tumor activity</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">13693</span>, <span class="refDoi"> DOI: 10.1038/srep13693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fsrep13693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26329134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC287ovVOitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=13693&author=W.+Zhangauthor=M.+Chenauthor=Y.+Ling+Wuauthor=Y.+Tanakaauthor=Y.+Juan+Jiauthor=S.+Lin+Zhangauthor=C.+He+Weiauthor=Y.+Xu&title=Formation+and+stabilization+of+the+telomeric+antiparallel+G-quadruplex+and+inhibition+of+telomerase+by+novel+benzothioxanthene+derivatives+with+anti-tumor+activity&doi=10.1038%2Fsrep13693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">6666</span><div class="casTitle"><span class="NLM_cas:atitle">Formation and stabilization of the telomeric antiparallel G-quadruplex and inhibition of telomerase by novel benzothioxanthene derivatives with anti-tumor activity</span></div><div class="casAuthors">Zhang Wen; Chen Min; Juan Ji Yan; Lin Zhang Su; He Wei Chuan; Ling Wu Yan; Tanaka Yoshimasa; Xu Yan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13693</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">G-quadruplexes formed in telomeric DNA sequences at human chromosome ends can be a novel target for the development of therapeutics for the treatment of cancer patients.  Herein, we examined the ability of six novel benzothioxanthene derivatives S1-S6 to induce the formation of and stabilize an antiparallel G-quadruplex by EMSA, UV-melting and CD techniques and the influence of S1-S6 on A549 and SGC7901 cells through real-time cell analysis, wound healing, trap assay methods.  Results show that six compounds could differentially induce 26 nt G-rich oligonucleotides to form the G-quadruplex with high selectivity vs C-rich DNA, mutated DNA and double-stranded DNA, stabilize it with high affinity, promote apoptosis and inhibit mobility and telomerase activity of A549 cells and SGC7901 cells.  Especially, S1, S3, S4 displayed stronger abilities, of which S3 was the most optimal with the maximum ΔTm value being up to 29.8 °C for G-quadruplex, the minimum IC50 value being 0.53 μM and the maximum cell inhibitory rate being up to 97.2%.  This study suggests that this type of compounds that induce the formation of and stabilize the telomeric antiparallel G-quadruplex, and consequently inhibit telomerase activity, leading to cell apoptosis, can be screened for the discovery of novel antitumor therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDnIOW2a-esJQHPYJuLg8TfW6udTcc2eaCdAQSP6-UMLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287ovVOitg%253D%253D&md5=16518ffc7648ab388149d1e22fcfe4f3</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fsrep13693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep13693%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DLing%2BWu%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DJuan%2BJi%26aufirst%3DY.%26aulast%3DLin%2BZhang%26aufirst%3DS.%26aulast%3DHe%2BWei%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DFormation%2520and%2520stabilization%2520of%2520the%2520telomeric%2520antiparallel%2520G-quadruplex%2520and%2520inhibition%2520of%2520telomerase%2520by%2520novel%2520benzothioxanthene%2520derivatives%2520with%2520anti-tumor%2520activity%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D13693%26doi%3D10.1038%2Fsrep13693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashcroft, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saraga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vimalachandran, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prime, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnogorac-Jurcevic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costello, E.</span></span> <span> </span><span class="NLM_article-title">Pancreatic cancer cells overexpress gelsolin family-capping proteins, which contribute to their cell motility</span>. <i>Gut</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1136/gut.2005.083691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1136%2Fgut.2005.083691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=16847067" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2007&pages=95-106&author=C.+C.+Thompsonauthor=F.+J.+Ashcroftauthor=S.+Patelauthor=G.+Saragaauthor=D.+Vimalachandranauthor=W.+Primeauthor=F.+Campbellauthor=A.+Dodsonauthor=R.+E.+Jenkinsauthor=N.+R.+Lemoineauthor=T.+Crnogorac-Jurcevicauthor=H.+Yinauthor=E.+Costello&title=Pancreatic+cancer+cells+overexpress+gelsolin+family-capping+proteins%2C+which+contribute+to+their+cell+motility&doi=10.1136%2Fgut.2005.083691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic cancer cells overexpress gelsolin family-capping proteins, which contribute to their cell motility</span></div><div class="casAuthors">Thompson, C. C.; Ashcroft, F. J.; Patel, S.; Saraga, G.; Vimalachandran, D.; Prime, W.; Campbell, F.; Dodson, A.; Jenkins, R. E.; Lemoine, N. R.; Crnogorac-Jurcevic, T.; Yin, H. L.; Costello, E.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-106</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Previously, proteomic methods were applied to characterize differentially expressed proteins in microdissected pancreatic ductal adenocarcinoma cells.  To report that CapG and a related protein, gelsolin, which have established roles in cell motility, are overexpressed in metastatic pancreatic cancer; and to describe their pattern of expression in pancreatic cancer tissue and their effect on cell motility in pancreatic cancer cell lines.  CapG was identified by mass spectrometry and immunoblotting.  CapG and gelsolin expression was assessed by immunohistochem. anal. on a pancreatic cancer tissue microarray and correlated with clin. and pathol. parameters.  CapG and gelsolin levels were reduced using RNA interface in Suit-2, Panc-1 and MiaPaCa-2 cells.  Cell motility was assessed using modified Boyden chamber or wound-healing assays.  Multiple isoforms of CapG were detected in pancreatic cancer tissue and cell lines.  Immunohistochem. anal. of benign (n = 44 patients) and malignant (n = 69) pancreatic ductal cells showed significantly higher CapG staining intensity in nuclear (p < 0.001) and cytoplasmic (p < 0.001) compartments of malignant cells.  Similarly, gelsolin immunostaining of benign (n = 24 patients) and malignant (n = 68 patients) pancreatic ductal cells showed higher expression in both compartments (both p < 0.001).  High nuclear CapG was assocd. with increased tumor size (p = 0.001).  High nuclear gelsolin was assocd. with reduced survival (p = 0.01).  Redn. of CapG or gelsolin expression in cell lines by RNAi was accompanied by significantly impaired motility.  Up regulation of these actin-capping proteins in pancreatic cancer and their ability to modulate cell motility in vitro suggest their potentially important role in pancreatic cancer cell motility and consequently dissemination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvU820K_DYDbVg90H21EOLACvtfcHk0lgNFiF_1eo3zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsLY%253D&md5=8f7c0bf71f5a6d28cd24d8018ce0ec54</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1136%2Fgut.2005.083691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.2005.083691%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DC.%2BC.%26aulast%3DAshcroft%26aufirst%3DF.%2BJ.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DSaraga%26aufirst%3DG.%26aulast%3DVimalachandran%26aufirst%3DD.%26aulast%3DPrime%26aufirst%3DW.%26aulast%3DCampbell%26aufirst%3DF.%26aulast%3DDodson%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DR.%2BE.%26aulast%3DLemoine%26aufirst%3DN.%2BR.%26aulast%3DCrnogorac-Jurcevic%26aufirst%3DT.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DCostello%26aufirst%3DE.%26atitle%3DPancreatic%2520cancer%2520cells%2520overexpress%2520gelsolin%2520family-capping%2520proteins%252C%2520which%2520contribute%2520to%2520their%2520cell%2520motility%26jtitle%3DGut%26date%3D2007%26volume%3D56%26spage%3D95%26epage%3D106%26doi%3D10.1136%2Fgut.2005.083691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitz, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussif, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, D.</span></span> <span> </span><span class="NLM_article-title">PD-1 inhibits T-cell receptor induced phosphorylation of the ZAP70/CD3ζ signalosome and downstream signaling to PKCθ</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>574</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2004.07.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.febslet.2004.07.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=15358536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFGmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=574&publication_year=2004&pages=37-41&author=K.+A.+Sheppardauthor=L.+J.+Fitzauthor=J.+M.+Leeauthor=C.+Benanderauthor=J.+A.+Georgeauthor=J.+Wootersauthor=Y.+Qiuauthor=J.+M.+Jussifauthor=L.+L.+Carterauthor=C.+R.+Woodauthor=D.+Chaudhary&title=PD-1+inhibits+T-cell+receptor+induced+phosphorylation+of+the+ZAP70%2FCD3%CE%B6+signalosome+and+downstream+signaling+to+PKC%CE%B8&doi=10.1016%2Fj.febslet.2004.07.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">PD-1 inhibits T-cell receptor induced phosphorylation of the ZAP70/CD3ζ signalosome and downstream signaling to PKCθ</span></div><div class="casAuthors">Sheppard, Kelly-Ann; Fitz, Lori J.; Lee, Julie M.; Benander, Christina; George, Judith A.; Wooters, Joe; Qiu, Yongchang; Jussif, Jason M.; Carter, Laura L.; Wood, Clive R.; Chaudhary, Divya</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">574</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">37-41</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Engagement of the immunoinhibitory receptor, programmed death-1 (PD-1) attenuates T-cell receptor (TCR)-mediated activation of IL-2 prodn. and T-cell proliferation.  Here, we demonstrate that PD-1 modulation of T-cell function involves inhibition of TCR-mediated phosphorylation of ZAP70 and assocn. with CD3ζ.  In addn., PD-1 signaling attenuates PKCθ activation loop phosphorylation in a cognate TCR signal.  PKCθ has been shown to be required for T-cell IL-2 prodn.  A phosphorylated PD-1 peptide, corresponding to the C-terminal immunoreceptor tyrosine-switch motif (ITSM), acts as a docking site in vitro for both SHP-2 and SHP-1, while the phosphorylated peptide contg. the N-terminal PD-1 immunoreceptor tyrosine based inhibitory motif (ITIM) assocs. only with SHP-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr85CRiewsx1LVg90H21EOLACvtfcHk0lgNFiF_1eo3zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFGmu7Y%253D&md5=2c9dbaa347d320423858ec102649d14f</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2004.07.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2004.07.083%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DK.%2BA.%26aulast%3DFitz%26aufirst%3DL.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DBenander%26aufirst%3DC.%26aulast%3DGeorge%26aufirst%3DJ.%2BA.%26aulast%3DWooters%26aufirst%3DJ.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DJussif%26aufirst%3DJ.%2BM.%26aulast%3DCarter%26aufirst%3DL.%2BL.%26aulast%3DWood%26aufirst%3DC.%2BR.%26aulast%3DChaudhary%26aufirst%3DD.%26atitle%3DPD-1%2520inhibits%2520T-cell%2520receptor%2520induced%2520phosphorylation%2520of%2520the%2520ZAP70%252FCD3%25CE%25B6%2520signalosome%2520and%2520downstream%2520signaling%2520to%2520PKC%25CE%25B8%26jtitle%3DFEBS%2520Lett.%26date%3D2004%26volume%3D574%26spage%3D37%26epage%3D41%26doi%3D10.1016%2Fj.febslet.2004.07.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0008.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Py-Im polyamides structures and (B) the ball-and-stick model of polyamides binding to their target sites. Target sites are the underlined DNA sequences of the STAT3 binding site (SBS) located at the upstream DNA (nucleotide base pairs −91 to −118) in the PD-L1 gene promoter region. ●, <i>N</i>-methylimidazole residue (Im); ○, <i>N</i>-methylpyrrole residue (Py); β or β-head: β-alanine residue; Dp, dimethylamino-propylamide; ⊂, γ or γ-turn (γ-aminobutanoic acid turn) residue between the two subunits; Ac, acetyl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. EMSA image (A) for Py-Im polyamides interacting with the PD-L1 promoter sequence and (B) for Py-Im polyamides and the STAT3 protein competitively binding to the SBS in the PD-L1 promoter region. (A) Lane 1: DNA standard Marker (M); Lane 2: the forward oligo of wt-ds-DNA alone ss-DNA) in buffer containing 0.5% DMSO; Lane 3: forward oligo + polyamide <b>1</b> (16 μM); Lane 4: forward oligo + polyamide <b>2</b> (16 μM); Lane 5: wt-ds-DNA alone; Lanes 6–10: wt-ds-DNA + polyamide <b>1</b> at concentrations of 1, 2, 4, 8, and 16 μM; Lanes 11–15: wt-ds-DNA + polyamide <b>2</b> at concentrations of 1, 2, 4, 8, and 16 μM; Lane 16: wt-ds-DNA + Mis-polyamide <b>3</b> (24 μM); Lane 17: mut-ds-DNA alone; Lane 18: mut-ds-DNA + polyamide <b>1</b> (16 μM); Lane 19: mut-ds-DNA + polyamide <b>2</b> (16 μM). (B) Lane 1: wt-ds-DNA alone; Lane 2: wt-ds-DNA + polyamide <b>1</b> (12 μM); Lane 3: wt-ds-DNA + polyamide <b>2</b> (12 μM); Lane 4: wt-ds-DNA + STAT3 protein (1 μL); Lanes 5–9: wt-ds-DNA + STAT3 protein + polyamide <b>1</b> at concentrations of 1, 2, 4, 8, and 12 μM; Lanes 10–14: wt-ds-DNA + STAT3 protein + polyamide <b>2</b> at concentrations of 1, 2, 4, 8, and 12 μM; Lane 15: wt-ds-DNA + Mis-polyamide <b>3</b> (24 μM); Lane 16: wt-ds-DNA + STAT3 protein + Mis-polyamide (24 μM). The final concentration of DNA was 2 μM for single-stranded DNA and 1 μM for double-stranded DNA, and the final volume of the protein was 1 μL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Ability of two polyamides <b>1</b> and <b>2</b> to specifically interact with a 28nt double-stranded DNA sequence of PD-L1 gene promoter (wt-ds-DNA and 5′-biotin–ds-DNA, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) by (A,B) CD and (C,D) SPR. (E,F) Molecular modeling of the interaction between the polyamides and the target sequence of DNA 5′-TTCACCGAA-3′/5′-TTCGGTGAA-3′. CD spectra for the interaction of (A) <b>1</b> and (B) <b>2</b> with wt-ds-DNA. Solid arrows at 244.5–245.0 nm (A,B) indicate the effect of the concentration of <b>1</b> and <b>2</b> on the trough of wt-ds-DNA; solid arrows at 283.5–286.5 nm (A) and 284.0–287.5 nm (B)  indicate the effect of the concentration of <b>1</b> and <b>2</b> on the peak of wt-ds-DNA; dotted arrows near 300.0 nm (A) and 297.5 nm (B) indicate the isoelliptic points; solid arrows at 340.0 nm (A) and 325.0 nm (B) indicate the effect of <b>1</b> and <b>2</b> on the ICD peak at 300–394 nm (A) and 297.5–374.5 nm (B), respectively. All CD spectra were obtained from 225 to 400 nm for 28nt wt-ds-DNA in the promoter sequence of PD-L1 in 10 mM sodium cacodylate buffer, pH 7.4 containing 100 mM NaCl and 0.5% DMSO at compound/DNA ratios (<i>r</i>) of 0.0, 0.2, 0.4, 0.6, 0.8, 1.0, and 2.0 at ambient temperature. In the insets of panels A and B, the positive ICD signals at 340.0 nm for polyamide <b>1</b> and at 325.0 nm for polyamide <b>2</b> are plotted against the compound/DNA ratio, respectively. The black dotted curve represents DNA alone, and the black dashed line represents the compound alone. Monitoring of the interaction of <b>1</b> (C) and <b>2</b> (D) with 5′-biotin–ds-DNA by SPR sensorgram. The biotin-conjugated ds-DNA was immobilized on the surface of an SA sensor chip. The binding response was monitored with time. The SPR curves were fitted to a 1:1 interaction model with a mass-transfer effect. Experimental curves are depicted in color, and fitting curves are depicted in black. The association phase was run for 300 s until the steady state was confirmed, and the dissociation phase was run for 280 s for all experiments. The concentrations of <b>1</b> and <b>2</b> were 10, 20, 40, 60, 80, 100, 150, 250, 350, and 500 nM, respectively. All experiments were performed in triplicate. (E) Molecular models of the 1:1 complex of polyamides <b>1</b>/<b>2</b> and the ds-DNA, respectively, by computer-assisted molecular simulation and (F) the interactive binding energy calculated by energy minimization with the LibDock evaluation. The polyamide is depicted as a stick, N3 is in blue, A base is in blue, G is in yellow, and T is in red. In the Py-Im polyamides, hydrogen is shown in gray, nitrogen is in blue, oxygen us in red, and covalent bonds are in green. The DNA strand is represented as purple and green ribbons, and its surface rendering is set to be solid and transparent. The sodium ions and water molecules are not shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of Py-Im polyamides <b>1</b> and <b>2</b> on the PD-L1 gene expression in HCC827 and MDA-MB-231 cells. Level of PD-L1 mRNA expression in (A) HCC827 and (B) MDA-MB-231 cells. The tumor cells were treated with IFN-γ alone (50 ng/mL) or IFN-γ (50 ng/mL) + Py-Im polyamides (0, 1.25, 2.5, 5, and 10 μM) for 36 h, and the mRNA was analyzed by q-PCR. Western blot analysis of PD-L1 protein in (C) HCC827 and (D) MDA-MB-231 cells treated with the indicated polyamides. HCC827 and MDA-MB-231 cells were treated with IFN-γ alone (50 ng/mL) or IFN-γ (50 ng/mL) + Py-Im polyamide <b>1</b> or <b>2</b> (0, 1.25, 2.5, 5, and 10 μM). Data show the mean ± SD and are the result of three independent experiments (* <i>p</i> < 0.05, ** <i>p</i> < 0.01, *** <i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Promotion of T-cell-mediated tumor killing and IL-2 production by the Py-Im polyamides in coculture systems. IL-2 secretion levels in (A) Jurkat/HCC827 and (B) Jurkat/MDA-MB-231 coculture systems treated with 8 μM of the polyamide (<b>1</b> or <b>2</b> or Mis-polyamide) and PD-1/PD-L1 inhibitor 2 (BMS202 as a positive control; its structure is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf" class="ext-link">Figure S6</a>) are shown. Promotion of T-cell-mediated tumor killing in (C) Jurkat/HCC827 and (D) Jurkat/MDA-MB-231 coculture systems treated with 8 μM of <b>1</b> or <b>2</b> or Mis-polyamide and PD-1/PD-L1 inhibitor 2. Data show the mean ± SD and are the results of three independent experiments (* <i>p</i> < 0.05, ** <i>p</i> < 0.01, *** <i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cytotoxicity and migration inhibition of the Py-Im polyamides to cancer cells and the exploration of the mechanism for non-immune-mediated proliferation inhibition/apoptosis of cancer cells by Py-Im polyamides. Effect of the Py-Im polyamides on the viability of (A) HCC827, (B) MDA-MB-231, and (C) MRC-5 in the absence of immune T cells. The cell viability was determined using the CCK-8 assay. Cisplatin was used as a positive control. 1% DMSO was the reference solution, which contained 1% DMSO in RPMI 1640. (D) Effect of Py-Im polyamides <b>1</b> and <b>2</b> on the migration of tumor cells. Wound healing assays were employed to determine the inhibition of the tumor cell migration. 48 h after scratches, the migration of the tumor cells was observed. Western blot analysis of Akt, p-Akt (Ser473), and caspase-3 proteins in (E) HCC827 and (F) MDA-MB-231 cells treated with the indicated compounds. HCC827 and MDA-MB-231 cells were treated with the Py-Im polyamide <b>2</b> (10 μM) or Mis-polyamide (10 μM). Data are the mean ± SD and are the results of three independent experiments (* <i>p</i> < 0.05, ** <i>p</i> < 0.01, *** <i>p</i> < 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/medium/jm1c00120_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Schematic of the molecular mechanism of Py-Im polyamides target-downregulating the expression of the PD-L1 gene and promoting tumor cell apoptosis via immune and nonimmune pathways.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00120/20210506/images/large/jm1c00120_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00120&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04753" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04753" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Couzin-Frankel, J.</span></span> <span> </span><span class="NLM_article-title">Breakthrough of the year 2013. Cancer immunotherapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1126/science.342.6165.1432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1126%2Fscience.342.6165.1432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=24357284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2013&pages=1432-1433&author=J.+Couzin-Frankel&title=Breakthrough+of+the+year+2013.+Cancer+immunotherapy&doi=10.1126%2Fscience.342.6165.1432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer immunotherapy</span></div><div class="casAuthors">Couzin-Frankel, Jennifer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">6165</span>),
    <span class="NLM_cas:pages">1432-1433</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxASV1j-6kL7Vg90H21EOLACvtfcHk0lj7oyMSu8QtSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotF2qsg%253D%253D&md5=2e0995fd567c5b59f40c46981da0bef4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.342.6165.1432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.342.6165.1432%26sid%3Dliteratum%253Aachs%26aulast%3DCouzin-Frankel%26aufirst%3DJ.%26atitle%3DBreakthrough%2520of%2520the%2520year%25202013.%2520Cancer%2520immunotherapy%26jtitle%3DScience%26date%3D2013%26volume%3D342%26spage%3D1432%26epage%3D1433%26doi%3D10.1126%2Fscience.342.6165.1432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Topalian, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardoll, D. M.</span></span> <span> </span><span class="NLM_article-title">Immune checkpoint blockade: a common denominator approach to cancer therapy</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ccell.2015.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25858804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVaku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=450-461&author=S.+L.+Topalianauthor=C.+G.+Drakeauthor=D.+M.+Pardoll&title=Immune+checkpoint+blockade%3A+a+common+denominator+approach+to+cancer+therapy&doi=10.1016%2Fj.ccell.2015.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Immune Checkpoint Blockade: A Common Denominator Approach to Cancer Therapy</span></div><div class="casAuthors">Topalian, Suzanne L.; Drake, Charles G.; Pardoll, Drew M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The immune system recognizes and is poised to eliminate cancer but is held in check by inhibitory receptors and ligands.  These immune checkpoint pathways, which normally maintain self-tolerance and limit collateral tissue damage during anti-microbial immune responses, can be co-opted by cancer to evade immune destruction.  Drugs interrupting immune checkpoints, such as anti-CTLA-4, anti-PD-1, anti-PD-L1, and others in early development, can unleash anti-tumor immunity and mediate durable cancer regressions.  The complex biol. of immune checkpoint pathways still contains many mysteries, and the full activity spectrum of checkpoint-blocking drugs, used alone or in combination, is currently the subject of intense study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb0YUggJYBrLVg90H21EOLACvtfcHk0lgiamjR8ta5_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVaku7k%253D&md5=e42c02d50881184b78630e5dd23f7f2e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DTopalian%26aufirst%3DS.%2BL.%26aulast%3DDrake%26aufirst%3DC.%2BG.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26atitle%3DImmune%2520checkpoint%2520blockade%253A%2520a%2520common%2520denominator%2520approach%2520to%2520cancer%2520therapy%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ccell.2015.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ceeraz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noelle, R. J.</span></span> <span> </span><span class="NLM_article-title">B7 family checkpoint regulators in immune regulation and disease</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">556</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1016/j.it.2013.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.it.2013.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=23954143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1KltbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=556-563&author=S.+Ceerazauthor=E.+C.+Nowakauthor=R.+J.+Noelle&title=B7+family+checkpoint+regulators+in+immune+regulation+and+disease&doi=10.1016%2Fj.it.2013.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">B7 family checkpoint regulators in immune regulation and disease</span></div><div class="casAuthors">Ceeraz, Sabrina; Nowak, Elizabeth C.; Noelle, Randolph J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">556-563</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fine-tuning the immune response and maintaining tolerance to self-antigens involves a complex network of co-stimulatory and co-inhibitory mols.  The recent FDA approval of ipilimumab, a monoclonal antibody blocking cytotoxic T lymphocyte antigen (CTLA)-4, demonstrates the impact of checkpoint regulators in disease.  This is reinforced by ongoing clin. trials targeting not only CTLA-4, but also the programmed death (PD)-1 and B7-H4 pathways in various disease states.  Recently, two new B7 family inhibitory ligands, V-domain Ig suppressor of T cell activation (VISTA) and B7-H6 were identified.  Here, we review recent understanding of B7 family members and their concerted regulation of the immune response to either self or foreign pathogens.  We also discuss clin. developments in targeting these pathways in different disease settings, and introduce VISTA as a putative therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpywmoIEHf3s7Vg90H21EOLACvtfcHk0lgiamjR8ta5_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1KltbjJ&md5=395a88bdda0a83ec10fe4845d5fa76c9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2013.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2013.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DCeeraz%26aufirst%3DS.%26aulast%3DNowak%26aufirst%3DE.%2BC.%26aulast%3DNoelle%26aufirst%3DR.%2BJ.%26atitle%3DB7%2520family%2520checkpoint%2520regulators%2520in%2520immune%2520regulation%2520and%2520disease%26jtitle%3DTrends%2520Immunol.%26date%3D2013%26volume%3D34%26spage%3D556%26epage%3D563%26doi%3D10.1016%2Fj.it.2013.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karwacz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arce, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escors, D.</span></span> <span> </span><span class="NLM_article-title">PD-L1 co-stimulation contributes to ligand-induced T cell receptor down-modulation on CD8+ T cells</span>. <i>EMBO Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1002/emmm.201100165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Femmm.201100165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21739608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlersL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=581-592&author=K.+Karwaczauthor=C.+Bricogneauthor=D.+MacDonaldauthor=F.+Arceauthor=C.+L.+Bennettauthor=M.+Collinsauthor=D.+Escors&title=PD-L1+co-stimulation+contributes+to+ligand-induced+T+cell+receptor+down-modulation+on+CD8%2B+T+cells&doi=10.1002%2Femmm.201100165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1 co-stimulation contributes to ligand-induced T cell receptor down-modulation on CD8+ T cells</span></div><div class="casAuthors">Karwacz, Katarzyna; Bricogne, Christopher; MacDonald, Douglas; Arce, Frederick; Bennett, Clare L.; Collins, Mary; Escors, David</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">581-592</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">T cell receptor (TCR) down-modulation after antigen presentation is a fundamental process that regulates TCR signal transduction.  Current understanding of this process is that intrinsic TCR/CD28 signal transduction leads to TCR down-modulation.  Here, we show that the interaction between programmed cell death 1 ligand 1 (PD-L1) on dendritic cells (DCs) and programmed death 1 (PD-1) on CD8 T cells contributes to ligand-induced TCR down-modulation.  We provide evidence that this occurs via Casitas B-lymphoma (Cbl)-b E3 ubiquitin ligase up-regulation in CD8 T cells.  Interference with PD-L1/PD-1 signalling markedly inhibits TCR down-modulation leading to hyper-activated, proliferative CD8 T cells as assessed in vitro and in vivo in an arthritis model.  PD-L1 silencing accelerates anti-tumor immune responses and strongly potentiates DC anti-tumor capacities, when combined with mitogen-activated kinase (MAPK) modulators that promote DC activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsoJxavR3bpbVg90H21EOLACvtfcHk0lgiamjR8ta5_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlersL%252FL&md5=4f7eefaa0d962c7118676ed6417d8bc3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Femmm.201100165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Femmm.201100165%26sid%3Dliteratum%253Aachs%26aulast%3DKarwacz%26aufirst%3DK.%26aulast%3DBricogne%26aufirst%3DC.%26aulast%3DMacDonald%26aufirst%3DD.%26aulast%3DArce%26aufirst%3DF.%26aulast%3DBennett%26aufirst%3DC.%2BL.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DEscors%26aufirst%3DD.%26atitle%3DPD-L1%2520co-stimulation%2520contributes%2520to%2520ligand-induced%2520T%2520cell%2520receptor%2520down-modulation%2520on%2520CD8%252B%2520T%2520cells%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D581%26epage%3D592%26doi%3D10.1002%2Femmm.201100165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marabelle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahmer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gettinger, S.</span></span> <span> </span><span class="NLM_article-title">Immune checkpoint modulation for non-small cell lung cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2256</span>– <span class="NLM_lpage">2262</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-2959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F1078-0432.CCR-14-2959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25979932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslOktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2256-2262&author=J.+C.+Soriaauthor=A.+Marabelleauthor=J.+R.+Brahmerauthor=S.+Gettinger&title=Immune+checkpoint+modulation+for+non-small+cell+lung+cancer&doi=10.1158%2F1078-0432.CCR-14-2959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Immune Checkpoint Modulation for Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Soria, Jean-Charles; Marabelle, Aurelien; Brahmer, Julie R.; Gettinger, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2256-2262</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Therapies targeting immune checkpoints have recently shown encouraging activity in patients with heavily pretreated advanced non-small cell lung cancer (NSCLC), independently of NSCLC histol. or mutational status, with low toxicity profiles when used as monotherapy.  Objective response rates of approx. 20% have been reported in patients with advanced NSCLC treated with antagonist antibodies targeting the immune checkpoint, programmed death 1 (PD-1) on activated T cells, or its primary ligand, programmed death ligand 1 (PD-L1) expressed within the tumor microenvironment.  Response rates appear to be higher in patients with tumor PD-L1 expression documented by immunohistochem., although responses have been appreciated in patients with reportedly PD-L1-neg. tumor specimens.  Antibodies directed against cytotoxic T-lymphocyte-assocd. protein 4 (CTLA-4), another immunosuppressive T-cell signaling mol., are also being evaluated in clin. trials, with one randomized phase II trial demonstrating improved immune-related progression-free survival in lung cancer patients when added to std. chemotherapy.  Addnl. clin. trials are combining anti-CTLA-4 antibodies with either anti-PD-1 or anti-PD-L1 antibodies.  Combinations of other immune checkpoint antagonists or agonist antibodies with anti-PD-1 or anti-PD-L1 antibodies are also being pursued.  Clin Cancer Res; 21(10); 2256-62. ©2015 AACR.  See all articles in this section, "Progress in Lung Cancer".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkSMBG3GpFQLVg90H21EOLACvtfcHk0ljOscjBhIfh6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslOktbs%253D&md5=9bb87571496cc02a6172e82023b20f5b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-2959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-2959%26sid%3Dliteratum%253Aachs%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DMarabelle%26aufirst%3DA.%26aulast%3DBrahmer%26aufirst%3DJ.%2BR.%26aulast%3DGettinger%26aufirst%3DS.%26atitle%3DImmune%2520checkpoint%2520modulation%2520for%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2256%26epage%3D2262%26doi%3D10.1158%2F1078-0432.CCR-14-2959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sareddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandeswara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drerup, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murthy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thedieck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadlamudi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curiel, T. J.</span></span> <span> </span><span class="NLM_article-title">Tumor intrinsic PD-L1 signals regulate cell growth, pathogenesis, and autophagy in ovarian cancer and melanoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">6964</span>– <span class="NLM_lpage">6974</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-0258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F0008-5472.CAN-16-0258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27671674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFyntrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=6964-6974&author=C.+A.+Clarkauthor=H.+B.+Guptaauthor=G.+Sareddyauthor=S.+Pandeswaraauthor=S.+Laoauthor=B.+Yuanauthor=J.+M.+Drerupauthor=A.+Padronauthor=G.+J.+Conejo-Garciaauthor=K.+Murthyauthor=Y.+Liuauthor=M.+J.+Turkauthor=K.+Thedieckauthor=V.+Hurezauthor=R.+Liauthor=R.+Vadlamudiauthor=T.+J.+Curiel&title=Tumor+intrinsic+PD-L1+signals+regulate+cell+growth%2C+pathogenesis%2C+and+autophagy+in+ovarian+cancer+and+melanoma&doi=10.1158%2F0008-5472.CAN-16-0258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor-Intrinsic PD-L1 Signals Regulate Cell Growth, Pathogenesis, and Autophagy in Ovarian Cancer and Melanoma</span></div><div class="casAuthors">Clark, Curtis A.; Gupta, Harshita B.; Sareddy, Gangadhara; Pandeswara, Srilakshmi; Lao, Shunhua; Yuan, Bin; Drerup, Justin M.; Padron, Alvaro; Conejo-Garcia, Jose; Murthy, Kruthi; Liu, Yang; Turk, Mary Jo; Thedieck, Kathrin; Hurez, Vincent; Li, Rong; Vadlamudi, Ratna; Curiel, Tyler J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6964-6974</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PD-L1 antibodies produce efficacious clin. responses in diverse human cancers, but the basis for their effects remains unclear, leaving a gap in the understanding of how to rationally leverage therapeutic activity.  PD-L1 is widely expressed in tumor cells, but its contributions to tumor pathogenicity are incompletely understood.  In this study, we evaluated the hypothesis that PD-L1 exerts tumor cell-intrinsic signals that are crit. for pathogenesis.  Using RNAi methodol., we attenuated PD-L1 in the murine ovarian cell line ID8agg and the melanoma cell line B16 (termed PD-L1lo cells), which express basal PD-L1.  We obsd. that PD-L1lo cells proliferated more weakly than control cells in vitro.  As expected, PD-L1lo cells formed tumors in immunocompetent mice relatively more slowly, but unexpectedly, they also formed tumors more slowly in immunodeficient NSG mice.  RNA sequencing anal. identified a no. of genes involved in autophagy and mTOR signaling that were affected by PD-L1 expression.  In support of a functional role, PD-L1 attenuation augmented autophagy and blunted the ability of autophagy inhibitors to limit proliferation in vitro and in vivo in NSG mice.  PD-L1 attenuation also reduced mTORC1 activity and augmented the antiproliferative effects of the mTORC1 inhibitor rapamycin.  PD-L1lo cells were also relatively deficient in metastasis to the lung, and we found that anti-PD-L1 administration could block tumor cell growth and metastasis in NSG mice.  This therapeutic effect was obsd. with B16 cells but not ID8agg cells, illustrating tumor- or compartmental-specific effects in the therapeutic setting.  Overall, our findings extend understanding of PD-L1 functions, illustrate nonimmune effects of anti-PD-L1 immunotherapy, and suggest broader uses for PD-L1 as a biomarker for assessing cancer therapeutic responses.  Cancer Res; 76(23); 6964-74. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ6FcRFM_eY7Vg90H21EOLACvtfcHk0ljOscjBhIfh6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFyntrjP&md5=02dafdbfde9636f0272c53255c4e2d3c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-0258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-0258%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DC.%2BA.%26aulast%3DGupta%26aufirst%3DH.%2BB.%26aulast%3DSareddy%26aufirst%3DG.%26aulast%3DPandeswara%26aufirst%3DS.%26aulast%3DLao%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DDrerup%26aufirst%3DJ.%2BM.%26aulast%3DPadron%26aufirst%3DA.%26aulast%3DConejo-Garcia%26aufirst%3DG.%2BJ.%26aulast%3DMurthy%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DTurk%26aufirst%3DM.%2BJ.%26aulast%3DThedieck%26aufirst%3DK.%26aulast%3DHurez%26aufirst%3DV.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DVadlamudi%26aufirst%3DR.%26aulast%3DCuriel%26aufirst%3DT.%2BJ.%26atitle%3DTumor%2520intrinsic%2520PD-L1%2520signals%2520regulate%2520cell%2520growth%252C%2520pathogenesis%252C%2520and%2520autophagy%2520in%2520ovarian%2520cancer%2520and%2520melanoma%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D6964%26epage%3D6974%26doi%3D10.1158%2F0008-5472.CAN-16-0258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Windt, G. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonc, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, E. L.</span></span> <span> </span><span class="NLM_article-title">Metabolic competition in the tumor microenvironment is a driver of cancer progression</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.cell.2015.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26321679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVajs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2015&pages=1229-1241&author=C.+Changauthor=J.+Qiuauthor=D.+O%E2%80%99Sullivanauthor=M.+D.+Buckauthor=T.+Noguchiauthor=J.+D.+Curtisauthor=Q.+Chenauthor=M.+Gindinauthor=M.+M.+Gubinauthor=G.+J.+W.+van+der+Windtauthor=E.+Toncauthor=R.+D.+Schreiberauthor=E.+J.+Pearceauthor=E.+L.+Pearce&title=Metabolic+competition+in+the+tumor+microenvironment+is+a+driver+of+cancer+progression&doi=10.1016%2Fj.cell.2015.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Competition in the Tumor Microenvironment Is a Driver of Cancer Progression</span></div><div class="casAuthors">Chang, Chih-Hao; Qiu, Jing; O'Sullivan, David; Buck, Michael D.; Noguchi, Takuro; Curtis, Jonathan D.; Chen, Qiongyu; Gindin, Mariel; Gubin, Matthew M.; van der Windt, Gerritje J. W.; Tonc, Elena; Schreiber, Robert D.; Pearce, Edward J.; Pearce, Erika L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1229-1241</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Failure of T cells to protect against cancer is thought to result from lack of antigen recognition, chronic activation, and/or suppression by other cells.  Using a mouse sarcoma model, we show that glucose consumption by tumors metabolically restricts T cells, leading to their dampened mTOR activity, glycolytic capacity, and IFN-γ prodn., thereby allowing tumor progression.  We show that enhancing glycolysis in an antigenic "regressor" tumor is sufficient to override the protective ability of T cells to control tumor growth.  We also show that checkpoint blockade antibodies against CTLA-4, PD-1, and PD-L1, which are used clin., restore glucose in tumor microenvironment, permitting T cell glycolysis and IFN-γ prodn.  Furthermore, we found that blocking PD-L1 directly on tumors dampens glycolysis by inhibiting mTOR activity and decreasing expression of glycolysis enzymes, reflecting a role for PD-L1 in tumor glucose utilization.  Our results establish that tumor-imposed metabolic restrictions can mediate T cell hyporesponsiveness during cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSf2NZ5rM4LrVg90H21EOLACvtfcHk0ljOscjBhIfh6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVajs7%252FI&md5=cfc0a1c9d46d34259927a0130bd380a7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DD.%26aulast%3DBuck%26aufirst%3DM.%2BD.%26aulast%3DNoguchi%26aufirst%3DT.%26aulast%3DCurtis%26aufirst%3DJ.%2BD.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DGindin%26aufirst%3DM.%26aulast%3DGubin%26aufirst%3DM.%2BM.%26aulast%3Dvan%2Bder%2BWindt%26aufirst%3DG.%2BJ.%2BW.%26aulast%3DTonc%26aufirst%3DE.%26aulast%3DSchreiber%26aufirst%3DR.%2BD.%26aulast%3DPearce%26aufirst%3DE.%2BJ.%26aulast%3DPearce%26aufirst%3DE.%2BL.%26atitle%3DMetabolic%2520competition%2520in%2520the%2520tumor%2520microenvironment%2520is%2520a%2520driver%2520of%2520cancer%2520progression%26jtitle%3DCell%26date%3D2015%26volume%3D162%26spage%3D1229%26epage%3D1241%26doi%3D10.1016%2Fj.cell.2015.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrzeszczynska, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestell, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">STAT3 as an oneogene</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">303</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81959-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS0092-8674%2800%2981959-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10458605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1999&pages=295-303&author=J.+F.+Brombergauthor=M.+H.+Wrzeszczynskaauthor=G.+Devganauthor=Y.+Zhaoauthor=R.+G.+Pestellauthor=C.+Albaneseauthor=J.+E.+Darnell&title=STAT3+as+an+oneogene&doi=10.1016%2FS0092-8674%2800%2981959-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 as an oncogene</span></div><div class="casAuthors">Bromberg, Jacqueline F.; Wrzeszczynska, Melissa H.; Devgan, Geeta; Zhao, Yanxiang; Pestell, Richard G.; Albanese, Chris; Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">295-303</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">STATs are latent transcription factors that mediate cytokine- and growth factor-directed transcription.  In many human cancers and transformed cell lines, Stat3 is persistently activated, and in cell culture, active Stat3 is either required for transformation, enhances transformation, or blocks apoptosis.  Here, the substitution of two cysteine residues within the C-terminal loop of the SH2 domain of Stat3 produced a mol. that dimerizes spontaneously, binds to DNA, and activates transcription.  The Stat3-C mol. in immortalized fibroblasts causes cellular transformation scored by colony formation in soft agar and tumor formation in nude mice.  Thus, the activated Stat3 mol. by itself can mediate cellular transformation and the expts. focus attention on the importance of constitutive Stat3 activation in human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOPJVvUEeTbVg90H21EOLACvtfcHk0liczreIeoV9rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D&md5=f2e69e246bbfae15a3c5ee7b94340300</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981959-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981959-5%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%2BF.%26aulast%3DWrzeszczynska%26aufirst%3DM.%2BH.%26aulast%3DDevgan%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DPestell%26aufirst%3DR.%2BG.%26aulast%3DAlbanese%26aufirst%3DC.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DSTAT3%2520as%2520an%2520oneogene%26jtitle%3DCell%26date%3D1999%26volume%3D98%26spage%3D295%26epage%3D303%26doi%3D10.1016%2FS0092-8674%2800%2981959-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Decker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovarik, P.</span></span> <span> </span><span class="NLM_article-title">Serine phosphorylation of STATs</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2628</span>– <span class="NLM_lpage">2637</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1203481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fsj.onc.1203481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10851062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3cXktVGhtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=2628-2637&author=T.+Deckerauthor=P.+Kovarik&title=Serine+phosphorylation+of+STATs&doi=10.1038%2Fsj.onc.1203481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Serine phosphorylation of STATs</span></div><div class="casAuthors">Decker, Thomas; Kovarik, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2628-2637</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 74 refs.  Tyrosine phosphorylation regulates the dimerization of STATs as an essential prerequisite for the establishment of a classical JAK-STAT signaling path.  However, most vertebrate STATs contain a second phosphorylation site within their C-termini.  The phosphorylated residue in this case is a serine contained within a P(M)SP motif, and in the majority of situations its mutation to alanine alters transcription factor activity.  This review addresses recent advances in understanding the regulation of STAT serine phosphorylation, as well as the kinases and other signal transducers implied in this process.  The biochem. and biol. consequences of STAT serine phosphorylation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3t2x7RXMunrVg90H21EOLACvtfcHk0liczreIeoV9rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXktVGhtrk%253D&md5=f5b49c8386069b7cef95702d3b52b90c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1203481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1203481%26sid%3Dliteratum%253Aachs%26aulast%3DDecker%26aufirst%3DT.%26aulast%3DKovarik%26aufirst%3DP.%26atitle%3DSerine%2520phosphorylation%2520of%2520STATs%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D2628%26epage%3D2637%26doi%3D10.1038%2Fsj.onc.1203481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">The STATs of cancer-new molecular targets come of age</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/nrc1275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fnrc1275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=14964307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=97-105&author=H.+Yuauthor=R.+Jove&title=The+STATs+of+cancer-new+molecular+targets+come+of+age&doi=10.1038%2Fnrc1275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The STATs of cancer - new molecular targets come of age</span></div><div class="casAuthors">Yu, Hua; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-105</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Tumor cells acquire the ability to proliferate uncontrollably, resist apoptosis, sustain angiogenesis and evade immune surveillance.  STAT proteins - esp. STAT3 and STAT5 - regulate all of these processes and are persistently activated in a surprisingly large no. of human cancers.  Consequently, STAT proteins are emerging - unexpectedly - as ideal targets for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdnLoVoH4d1rVg90H21EOLACvtfcHk0liczreIeoV9rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D&md5=3e342db4856310cac2465240e36d3226</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc1275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1275%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DThe%2520STATs%2520of%2520cancer-new%2520molecular%2520targets%2520come%2520of%2520age%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D97%26epage%3D105%26doi%3D10.1038%2Fnrc1275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koskela, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldfors, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Adrichem, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusanmäki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagström, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemente, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalkanen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almusa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edgren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepistö, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattila, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koistinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuittinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penttinen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saarela, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioniemi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porkka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughran, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckman, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span> <span> </span><span class="NLM_article-title">Somatic STAT3 mutations in large granular lymphocytic leukemia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>366</i></span>,  <span class="NLM_fpage">1905</span>– <span class="NLM_lpage">1913</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1114885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1056%2FNEJMoa1114885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22591296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38Xntlalsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2012&pages=1905-1913&author=H.+L.+Koskelaauthor=S.+Eldforsauthor=P.+Ellonenauthor=A.+J.+van+Adrichemauthor=H.+Kuusanm%C3%A4kiauthor=E.+I.+Anderssonauthor=S.+Lagstr%C3%B6mauthor=M.+J.+Clementeauthor=T.+Olsonauthor=S.+E.+Jalkanenauthor=M.+M.+Majumderauthor=H.+Almusaauthor=H.+Edgrenauthor=M.+Lepist%C3%B6author=P.+Mattilaauthor=K.+Guintaauthor=P.+Koistinenauthor=T.+Kuittinenauthor=K.+Penttinenauthor=A.+Parsonsauthor=J.+Knowlesauthor=J.+Saarelaauthor=K.+Wennerbergauthor=O.+Kallioniemiauthor=K.+Porkkaauthor=T.+P.+Loughranauthor=C.+A.+Heckmanauthor=J.+P.+Maciejewskiauthor=S.+Mustjoki&title=Somatic+STAT3+mutations+in+large+granular+lymphocytic+leukemia&doi=10.1056%2FNEJMoa1114885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Somatic STAT3 mutations in large granular lymphocytic leukemia</span></div><div class="casAuthors">Koskela, Hanna L. M.; Eldfors, Samuli; Ellonen, Pekka; van Adrichem, Arjan J.; Kuusanmaki, Heikki; Andersson, Emma I.; Lagstrom, Sonja; Clemente, Michael J.; Olson, Thomas; Jalkanen, Sari E.; Majumder, Muntasir Mamun; Almusa, Henrikki; Edgren, Henrik; Lepisto, Maija; Mattila, Pirkko; Guinta, Kathryn; Koistinen, Pirjo; Kuittinen, Taru; Penttinen, Kati; Parsons, Alun; Knowles, Jonathan; Saarela, Janna; Wennerberg, Krister; Kallioniemi, Olli; Porkka, Kimmo; Loughran, Thomas P., Jr.; Heckman, Caroline A.; Maciejewski, Jaroslaw P.; Mustjoki, Satu</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">366</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1905-1913</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">T-cell large granular lymphocytic leukemia is a rare lymphoproliferative disorder characterized by the expansion of clonal CD3+CD8+ cytotoxic T lymphocytes (CTLs) and often assocd. with autoimmune disorders and immune-mediated cytopenias.  We used next-generation exome sequencing to identify somatic mutations in CTLs from an index patient with large granular lymphocytic leukemia.  Targeted resequencing was performed in a well-characterized cohort of 76 patients with this disorder, characterized by clonal T-cell-receptor rearrangements and increased nos. of large granular lymphocytes.  Mutations in the signal transducer and activator of transcription 3 gene (STAT3) were found in 31 of 77 patients (40%) with large granular lymphocytic leukemia.  Among these 31 patients, recurrent mutational hot spots included Y640F in 13 (17%), D661V in 7 (9%), D661Y in 7 (9%), and N647I in 3 (4%).  All mutations were located in exon 21, encoding the Src homol. 2 (SH2) domain, which mediates the dimerization and activation of STAT protein.  The amino acid changes resulted in a more hydrophobic protein surface and were assocd. with phosphorylation of STAT3 and its localization in the nucleus.  In vitro functional studies showed that the Y640F and D661V mutations increased the transcriptional activity of STAT3.  In the affected patients, downstream target genes of the STAT3 pathway (IFNGR2, BCL2L1, and JAK2) were up-regulated.  Patients with STAT3 mutations presented more often with neutropenia and rheumatoid arthritis than did patients without these mutations.  The SH2 dimerization and activation domain of STAT3 is frequently mutated in patients with large granular lymphocytic leukemia; these findings suggest that aberrant STAT3 signaling underlies the pathogenesis of this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyekkHvePb9LVg90H21EOLACvtfcHk0lhJr7M77Iy1ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xntlalsr8%253D&md5=4282683e8c70804aaac1818ea4e278d3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1114885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1114885%26sid%3Dliteratum%253Aachs%26aulast%3DKoskela%26aufirst%3DH.%2BL.%26aulast%3DEldfors%26aufirst%3DS.%26aulast%3DEllonen%26aufirst%3DP.%26aulast%3Dvan%2BAdrichem%26aufirst%3DA.%2BJ.%26aulast%3DKuusanm%25C3%25A4ki%26aufirst%3DH.%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DLagstr%25C3%25B6m%26aufirst%3DS.%26aulast%3DClemente%26aufirst%3DM.%2BJ.%26aulast%3DOlson%26aufirst%3DT.%26aulast%3DJalkanen%26aufirst%3DS.%2BE.%26aulast%3DMajumder%26aufirst%3DM.%2BM.%26aulast%3DAlmusa%26aufirst%3DH.%26aulast%3DEdgren%26aufirst%3DH.%26aulast%3DLepist%25C3%25B6%26aufirst%3DM.%26aulast%3DMattila%26aufirst%3DP.%26aulast%3DGuinta%26aufirst%3DK.%26aulast%3DKoistinen%26aufirst%3DP.%26aulast%3DKuittinen%26aufirst%3DT.%26aulast%3DPenttinen%26aufirst%3DK.%26aulast%3DParsons%26aufirst%3DA.%26aulast%3DKnowles%26aufirst%3DJ.%26aulast%3DSaarela%26aufirst%3DJ.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DLoughran%26aufirst%3DT.%2BP.%26aulast%3DHeckman%26aufirst%3DC.%2BA.%26aulast%3DMaciejewski%26aufirst%3DJ.%2BP.%26aulast%3DMustjoki%26aufirst%3DS.%26atitle%3DSomatic%2520STAT3%2520mutations%2520in%2520large%2520granular%2520lymphocytic%2520leukemia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D366%26spage%3D1905%26epage%3D1913%26doi%3D10.1056%2FNEJMoa1114885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wölfle, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sähr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalpke, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heeg, K.</span></span> <span> </span><span class="NLM_article-title">PD-L1 expression on tolerogenic APCs is controlled by STAT-3</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1002/eji.201040979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Feji.201040979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21268011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC3M7ksVKrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=413-424&author=S.+J.+W%C3%B6lfleauthor=J.+Strebovskyauthor=H.+Bartzauthor=A.+S%C3%A4hrauthor=C.+Arnoldauthor=C.+Kaiserauthor=A.+H.+Dalpkeauthor=K.+Heeg&title=PD-L1+expression+on+tolerogenic+APCs+is+controlled+by+STAT-3&doi=10.1002%2Feji.201040979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1 expression on tolerogenic APCs is controlled by STAT-3</span></div><div class="casAuthors">Wolfle Sabine J; Strebovsky Julia; Bartz Holger; Sahr Aline; Arnold Caroline; Kaiser Claus; Dalpke Alexander H; Heeg Klaus</div><div class="citationInfo"><span class="NLM_cas:title">European journal of immunology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">413-24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">During infection, TLR agonists are released and trigger mature as well as differentiating innate immune cells.  Early encounter with TLR agonists (R848; LPS) blocks conventional differentiation of CD14(+) monocytes into immature dendritic cells (iDCs) resulting in a deviated phenotype.  We and others characterized these APCs (TLR-APC) by a retained expression of CD14 and a lack of CD1a.  Here, we show in addition, expression of programmed death ligand-1 (PD-L1).  TLR-APCs failed to induce T-cell proliferation and furthermore were able to induce CD25(+) Foxp3(+) T regulatory cells (Tregs).  Since PD-L1 is described as a key negative regulator and inducer of tolerance, we further analyzed its regulation.  PD-L1 expression was regulated in a MAPK/cytokine/STAT-3-dependent manner: high levels of IL-6 and IL-10 that signal via STAT-3 were produced by TLR-APCs.  Blocking of STAT-3 activation prevented PD-L1 expression.  Moreover, chromatin immunoprecipitation revealed direct binding of STAT-3 to the PD-L1 promoter.  Those findings indicate a pivotal role of STAT-3 in regulating PD-L1 expression.  MAPKs were indirectly engaged, as blocking of p38 and p44/42 MAPKs decreased IL-6 and IL-10 thus reducing STAT-3 activation and subsequent PD-L1 expression.  Hence, during DC differentiation TLR agonists induce a STAT-3-mediated expression of PD-L1 and favor the development of tolerogenic APCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqpucQi7Nbtr5XHngOXiV_fW6udTcc2eZLuY-ezbV_ZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M7ksVKrtQ%253D%253D&md5=cb53ad4a94a23a6c12269ff67b5d0cb3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Feji.201040979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201040979%26sid%3Dliteratum%253Aachs%26aulast%3DW%25C3%25B6lfle%26aufirst%3DS.%2BJ.%26aulast%3DStrebovsky%26aufirst%3DJ.%26aulast%3DBartz%26aufirst%3DH.%26aulast%3DS%25C3%25A4hr%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DC.%26aulast%3DKaiser%26aufirst%3DC.%26aulast%3DDalpke%26aufirst%3DA.%2BH.%26aulast%3DHeeg%26aufirst%3DK.%26atitle%3DPD-L1%2520expression%2520on%2520tolerogenic%2520APCs%2520is%2520controlled%2520by%2520STAT-3%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2011%26volume%3D41%26spage%3D413%26epage%3D424%26doi%3D10.1002%2Feji.201040979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagishita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeshita, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nokihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochiya, T.</span></span> <span> </span><span class="NLM_article-title">The Clinical Relevance of the miR-197/CKS1B/STAT3-mediated PD-L1 network in chemoresistant non-small-cell lung Cancer</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">727</span>, <span class="refDoi"> DOI: 10.1038/mt.2015.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fmt.2015.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25597412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=717-727&author=Y.+Fujitaauthor=S.+Yagishitaauthor=K.+Hagiwaraauthor=Y.+Yoshiokaauthor=N.+Kosakaauthor=F.+Takeshitaauthor=T.+Fujiwaraauthor=K.+Tsutaauthor=H.+Nokiharaauthor=T.+Tamuraauthor=H.+Asamuraauthor=M.+Kawaishiauthor=K.+Kuwanoauthor=T.+Ochiya&title=The+Clinical+Relevance+of+the+miR-197%2FCKS1B%2FSTAT3-mediated+PD-L1+network+in+chemoresistant+non-small-cell+lung+Cancer&doi=10.1038%2Fmt.2015.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Clinical Relevance of the miR-197/CKS1B/STAT3-mediated PD-L1 Network in Chemoresistant Non-small-cell Lung Cancer</span></div><div class="casAuthors">Fujita, Yu; Yagishita, Shigehiro; Hagiwara, Keitaro; Yoshioka, Yusuke; Kosaka, Nobuyoshi; Takeshita, Fumitaka; Fujiwara, Tomohiro; Tsuta, Koji; Nokihara, Hiroshi; Tamura, Tomohide; Asamura, Hisao; Kawaishi, Makoto; Kuwano, Kazuyoshi; Ochiya, Takahiro</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">717-727</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Programmed cell death ligand-1 (PD-L1) has recently gained considerable attention for its role in tumor immune escape.  Here, we identify a miR-197/CKS1B/STAT3-mediated PD-L1 network in chemoresistant non-small-cell lung cancer (NSCLC), independent of immunoinhibitory signals.  MiR-197 is downregulated in platinum-resistant NSCLC specimens, resulting in the promotion of chemoresistance, tumorigenicity, and pulmonary metastasis in vitro and in vivo.  Mechanistic investigations reveal that a miR-197-mediated CKS1B/STAT3 axis exerts tumor progression regulated by various oncogenic genes (Bcl-2, c-Myc, and cyclin D1), and PD-L1 is a putative biomarker of this axis.  Furthermore, we demonstrate that a miR-197 mimic sensitizes PD-L1high drug-resistant cells to chemotherapy.  These results indicate that the biol. interaction between PD-L1 and chemoresistance occurs through the microRNA regulatory cascade.  More importantly, expression levels of miR-197 are inversely correlated with PD-L1 expression (n = 177; P = 0.026) and are assocd. with worse overall survival (P = 0.015).  Our discoveries suggest that the miR-197/CKS1B/STAT3-mediated network can drive tumor PD-L1 expression as a biomarker of this cascade, and miR-197 replacement therapy may be a potential treatment strategy for chemoresistant NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3_HY07pVy6LVg90H21EOLACvtfcHk0lhJr7M77Iy1ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOqs7o%253D&md5=0f14ec8da56cb12e0361771d8ff50203</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fmt.2015.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2015.10%26sid%3Dliteratum%253Aachs%26aulast%3DFujita%26aufirst%3DY.%26aulast%3DYagishita%26aufirst%3DS.%26aulast%3DHagiwara%26aufirst%3DK.%26aulast%3DYoshioka%26aufirst%3DY.%26aulast%3DKosaka%26aufirst%3DN.%26aulast%3DTakeshita%26aufirst%3DF.%26aulast%3DFujiwara%26aufirst%3DT.%26aulast%3DTsuta%26aufirst%3DK.%26aulast%3DNokihara%26aufirst%3DH.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DAsamura%26aufirst%3DH.%26aulast%3DKawaishi%26aufirst%3DM.%26aulast%3DKuwano%26aufirst%3DK.%26aulast%3DOchiya%26aufirst%3DT.%26atitle%3DThe%2520Clinical%2520Relevance%2520of%2520the%2520miR-197%252FCKS1B%252FSTAT3-mediated%2520PD-L1%2520network%2520in%2520chemoresistant%2520non-small-cell%2520lung%2520Cancer%26jtitle%3DMol.%2520Ther.%26date%3D2015%26volume%3D23%26spage%3D717%26epage%3D727%26doi%3D10.1038%2Fmt.2015.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horlad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikukawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komohara, Y.</span></span> <span> </span><span class="NLM_article-title">An IL-27/STAT3 axis induces expression of programmed cell death 1 ligands (PD-L1/2) on infiltrating macrophages in lymphoma</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1696</span>– <span class="NLM_lpage">1704</span>, <span class="refDoi"> DOI: 10.1111/cas.13065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1111%2Fcas.13065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27564404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC2szktlagsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1696-1704&author=H.+Horladauthor=C.+Maauthor=H.+Yanoauthor=C.+Panauthor=K.+Ohnishiauthor=Y.+Fujiwaraauthor=S.+Endoauthor=Y.+Kikukawaauthor=Y.+Okunoauthor=M.+Matsuokaauthor=M.+Takeyaauthor=Y.+Komohara&title=An+IL-27%2FSTAT3+axis+induces+expression+of+programmed+cell+death+1+ligands+%28PD-L1%2F2%29+on+infiltrating+macrophages+in+lymphoma&doi=10.1111%2Fcas.13065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An IL-27/Stat3 axis induces expression of programmed cell death 1 ligands (PD-L1/2) on infiltrating macrophages in lymphoma</span></div><div class="casAuthors">Horlad Hasita; Ma Chaoya; Yano Hiromu; Pan Cheng; Ohnishi Koji; Fujiwara Yukio; Takeya Motohiro; Komohara Yoshihiro; Endo Shinya; Kikukawa Yoshitaka; Okuno Yutaka; Matsuoka Masao; Matsuoka Masao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1696-1704</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Immune escape and tolerance in the tumor microenvironment are closely involved in tumor progression, and are caused by T-cell exhaustion and mediated by the inhibitory signaling of immune checkpoint molecules including programmed death-1 (PD-1), cytotoxic T-lymphocyte associated protein 4, and T-cell immunoglobulin and mucin domaincontaining molecule-3.  In the present study, we investigated the expression of the PD-1 ligand 1 (PD-L1) in a lymphoma microenvironment using paraffin-embedded tissue samples, and subsequently studied the detailed mechanism of upregulation of PD-L1 on macrophages using cultured human macrophages and lymphoma cell lines.  We found that macrophages in lymphoma tissues of almost all cases of adult T-cell leukemia/lymphoma (ATLL), follicular lymphoma and diffuse large B-cell lymphoma expressed PD-L1.  Cell culture studies showed that the conditioned medium of ATL-T and SLVL cell lines induced increased expression of PD-L1/2 on macrophages, and that this PD-L1/2 overexpression was dependent on activation of signal transducer and activator of transcription 3 (Stat3).  In vitro studies including cytokine array analysis showed that IL-27 (heterodimer of p28 and EBI3) induced overexpression of PD-L1/2 on macrophages via Stat3 activation.  Because lymphoma cell lines produced IL-27B (EBI3) but not IL-27p28, it was proposed that the IL-27p28 derived from macrophages and the IL-27B (EBI3) derived from lymphoma cells formed an IL-27 (heterodimer) that induced PD-L1/2 overexpression.  Although the significance of PD-L1/2 expressions on macrophages in lymphoma progression has never been clarified, an IL-27-Stat3 axis might be a target for immunotherapy for lymphoma patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvT7jxf980N4IRAdBLKY41fW6udTcc2ea5mhHjq8AAc7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szktlagsQ%253D%253D&md5=b0fa808fcc4f2e414ac24a22a40e7ca2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fcas.13065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13065%26sid%3Dliteratum%253Aachs%26aulast%3DHorlad%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DYano%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DOhnishi%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DEndo%26aufirst%3DS.%26aulast%3DKikukawa%26aufirst%3DY.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DMatsuoka%26aufirst%3DM.%26aulast%3DTakeya%26aufirst%3DM.%26aulast%3DKomohara%26aufirst%3DY.%26atitle%3DAn%2520IL-27%252FSTAT3%2520axis%2520induces%2520expression%2520of%2520programmed%2520cell%2520death%25201%2520ligands%2520%2528PD-L1%252F2%2529%2520on%2520infiltrating%2520macrophages%2520in%2520lymphoma%26jtitle%3DCancer%2520Sci.%26date%3D2016%26volume%3D107%26spage%3D1696%26epage%3D1704%26doi%3D10.1111%2Fcas.13065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdelhamed, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, Y.</span></span> <span> </span><span class="NLM_article-title">AKT-STAT3 pathway as a downstream target of EGFR signaling to regulate PD-L1 expression on NSCLC cells</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.7150/jca.14713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.7150%2Fjca.14713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27698894" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslWns7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1579-86&author=S.+Abdelhamedauthor=K.+Oguraauthor=S.+Yokoyamaauthor=I.+Saikiauthor=Y.+Hayakawa&title=AKT-STAT3+pathway+as+a+downstream+target+of+EGFR+signaling+to+regulate+PD-L1+expression+on+NSCLC+cells&doi=10.7150%2Fjca.14713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">AKT-STAT3 pathway as a downstream target of EGFR signaling to regulate PD-L1 expression on NSCLC cells</span></div><div class="casAuthors">Abdelhamed, Sherif; Ogura, Keisuke; Yokoyama, Satoru; Saiki, Ikuo; Hayakawa, Yoshihiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer (Wyoming, Australia)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1579-1586</span>CODEN:
                <span class="NLM_cas:coden">JCWAAL</span>;
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher Pty Ltd.</span>)
        </div><div class="casAbstract">While cancer development and progression can be controlled by cytotoxic T cells, it is also known that tumor-specific CD8+T cells become functionally impaired by acquiring a group of inhibitory receptors known as immune checkpoints.  Amongst those, programmed death-1 (PD-1) is one of the most recognized neg. regulators of T cell function.  In non-small lung cancers (NSCLCs), the aberrant activation of epidermal growth factor receptor (EGFR) is known to induce PD-L1 expression and further the treatment with gefitinib, a tyrosine kinase inhibitor (TKI) for EGFR, decrease the expression of PD-L1 on NSCLC.  Given the acquired resistance to gefitinib treatment frequently obsd. by developing secondary-site mutations limiting its efficacy, it is important to understand the downstream mechanism of activated-EGFR signaling for regulating PD-L1 in NSCLC.  In this study, we demonstrated that AKT-STAT3 pathway could be a potential target for regulating the surface expression of PD-L1 on NSCLCs with aberrant EGFR activity and, further, the inhibition of AKT or STAT3 activity could down-regulate the expression of PD-L1 even in gefitinib-resistant NSCLCs.  These results highlight an importance of AKT-STAT3 pathway as a promising target for potentiating anti-tumor immune responses by regulating PD-L1 expression on cancer cells with aberrant EGFR activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NBJ_KnblKbVg90H21EOLACvtfcHk0lhWa-XnHHTMLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslWns7k%253D&md5=097a1d3ced2f2109938357b257eb227c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.7150%2Fjca.14713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.14713%26sid%3Dliteratum%253Aachs%26aulast%3DAbdelhamed%26aufirst%3DS.%26aulast%3DOgura%26aufirst%3DK.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DSaiki%26aufirst%3DI.%26aulast%3DHayakawa%26aufirst%3DY.%26atitle%3DAKT-STAT3%2520pathway%2520as%2520a%2520downstream%2520target%2520of%2520EGFR%2520signaling%2520to%2520regulate%2520PD-L1%2520expression%2520on%2520NSCLC%2520cells%26jtitle%3DJ.%2520Cancer%26date%3D2016%26volume%3D7%26spage%3D1579%26epage%3D86%26doi%3D10.7150%2Fjca.14713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talukder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savage, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. B.</span></span> <span> </span><span class="NLM_article-title">JAK-STAT-mediated chronic inflammation impairscytotoxic T lymphocyte activation to decreaseanti-PD-1 immunotherapy efficacy in pancreaticcancer</span>. <i>Oncoimmunology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1291106</span> <span class="refDoi"> DOI: 10.1080/2162402X.2017.1291106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1080%2F2162402X.2017.1291106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28405527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2lsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=C.+Luauthor=A.+Talukderauthor=N.+M.+Savageauthor=N.+Singhauthor=K.+B.+Liu&title=JAK-STAT-mediated+chronic+inflammation+impairscytotoxic+T+lymphocyte+activation+to+decreaseanti-PD-1+immunotherapy+efficacy+in+pancreaticcancer&doi=10.1080%2F2162402X.2017.1291106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">JAK-STAT-mediated chronic inflammation impairs cytotoxic T lymphocyte activation to decrease anti-PD-1 immunotherapy efficacy in pancreatic cancer</span></div><div class="casAuthors">Lu, Chunwan; Talukder, Asif; Savage, Natasha M.; Singh, Nagendra; Liu, Kebin</div><div class="citationInfo"><span class="NLM_cas:title">OncoImmunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1291106/1-e1291106/15</span>CODEN:
                <span class="NLM_cas:coden">ONCOGX</span>;
        ISSN:<span class="NLM_cas:issn">2162-402X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Human pancreatic cancer does not respond to immune check point blockade immunotherapy.  One key feature of pancreatic cancer is the assocn. between its progression and chronic inflammation.  Emerging evidence supports a key role for the JAK-STAT pathway in pancreatic cancer inflammation.  We aimed at testing the hypothesis that sustained JAK-STAT signaling suppresses cytotoxic T lymphocyte (CTL) activation to counteract anti-PD-1 immunotherapy-induced CTL activity in pancreatic cancer.  We show that human pancreatic carcinomas express high level of PD-L1 and exhibit low level of CTL infiltration.  JAK-STAT inhibitor Ruxolitinib selectively inhibits STAT1 and STAT3 activation and increases CTL infiltration to induce a Tc1/Th1 immune response in the tumor microenvironment in an orthotopic pancreatic cancer mouse model.  Ruxilitinib-mediated tumor suppressive efficacy diminishes in T-cell-deficient mice.  Pancreatic tumor grows significantly faster in IFNγ-deficient mice.  However, neutralizing IFNγ does not alter tumor growth but diminishes Ruxolitinib-induced tumor suppression in vivo, indicating that lymphocytes and IFNγ are essential for Ruxolitinib-induced host antitumor immune response.  Both type I and type II interferons upregulate PD-L1 expression through the JAK-STAT signaling pathway in mouse pancreatic tumor cells.  Tumor cells respond to activated T cells by activating STAT3.  The inhibition of STAT3 downregulates immune suppressive cytokines prodn. by tumor cells, resulting in increased T cell activation and effector function.  Consequently, Ruxolitinib significantly improves the efficacy of anti-PD-1 immunotherapy.  Our data demonstrate that Ruxolitinib is effective in the inhibition of systemic inflammation in the tumor microenvironment and therefore upregulates CTL infiltration and activation to overcome pancreatic cancer resistance to anti-PD-1 immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrooTFsN2GfjbVg90H21EOLACvtfcHk0lhWa-XnHHTMLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2lsr4%253D&md5=5a73c10cf6b853bf6f0c1db3ebfe0f01</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F2162402X.2017.1291106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F2162402X.2017.1291106%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DC.%26aulast%3DTalukder%26aufirst%3DA.%26aulast%3DSavage%26aufirst%3DN.%2BM.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DK.%2BB.%26atitle%3DJAK-STAT-mediated%2520chronic%2520inflammation%2520impairscytotoxic%2520T%2520lymphocyte%2520activation%2520to%2520decreaseanti-PD-1%2520immunotherapy%2520efficacy%2520in%2520pancreaticcancer%26jtitle%3DOncoimmunology%26date%3D2017%26volume%3D6%26doi%3D10.1080%2F2162402X.2017.1291106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niidome, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komohara, Y.</span></span> <span> </span><span class="NLM_article-title">STAT3 inhibitor abrogates the expression of PD-1 ligands onlymphoma cell lines</span>. <i>J. Clin. Exp. Hematop.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.3960/jslrt.17006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.3960%2Fjslrt.17006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28496056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC1crmsVKnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=21-25&author=C.+Maauthor=H.+Horladauthor=C.+Panauthor=H.+Yanoauthor=K.+Ohnishiauthor=Y.+Fujiwaraauthor=M.+Matsuokaauthor=A.+Leeauthor=T.+Niidomeauthor=R.+Yamanakaauthor=M.+Takeyaauthor=Y.+Komohara&title=STAT3+inhibitor+abrogates+the+expression+of+PD-1+ligands+onlymphoma+cell+lines&doi=10.3960%2Fjslrt.17006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 inhibitor abrogates the expression of PD-1 ligands on lymphoma cell lines</span></div><div class="casAuthors">Ma Chaoya; Horlad Hasita; Pan Cheng; Yano Hiromu; Ohnishi Koji; Fujiwara Yukio; Takeya Motohiro; Komohara Yoshihiro; Matsuoka Masao; Matsuoka Masao; Lee Aeju; Lee Aeju; Niidome Takuro; Yamanaka Ryuya</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical and experimental hematopathology : JCEH</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent studies have indicated the significance of immune checkpoint molecules including programmed death-1 (PD-1), cytotoxic T-lymphocyte associated protein 4, and T-cell immunoglobulin and mucin domain-containing molecule-3 for anti-tumor immune responses.  We previously investigated PD-1 ligand 1/2 (PD-L1/2) expression in lymphoma cell lines, and found that PD-L1/2 is expressed on the adult T-cell leukemia/lymphoma (ATL-T) and B-cell lymphoma (SLVL) cell lines.  In the present study, we investigated whether the Stat3 inhibitor WP1066 abrogated PD-L1/2 expression in lymphoma cell lines.  Incubation with WP1066 inhibited lymphoma cell growth and induced cell apoptosis.  PD-L1/2 expression in the ATL-T, SLVL, and human brain malignant lymphoma (HKBML) cell lines was significantly abrogated by WP1066 treatment.  These data indicated that a Stat3 inhibitor abrogated PD-L1/2 expression in lymphoma cells.  Such an inhibitor is therefore considered to be useful for additional immunotherapy in patients with advanced lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTsbLe3_wWz3czK395ZObgVfW6udTcc2eYcRZE4eKkcXbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crmsVKnsA%253D%253D&md5=ca280f9946b51f6ebc45e5536e51405e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3960%2Fjslrt.17006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3960%252Fjslrt.17006%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHorlad%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DYano%26aufirst%3DH.%26aulast%3DOhnishi%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DMatsuoka%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DNiidome%26aufirst%3DT.%26aulast%3DYamanaka%26aufirst%3DR.%26aulast%3DTakeya%26aufirst%3DM.%26aulast%3DKomohara%26aufirst%3DY.%26atitle%3DSTAT3%2520inhibitor%2520abrogates%2520the%2520expression%2520of%2520PD-1%2520ligands%2520onlymphoma%2520cell%2520lines%26jtitle%3DJ.%2520Clin.%2520Exp.%2520Hematop.%26date%3D2017%26volume%3D57%26spage%3D21%26epage%3D25%26doi%3D10.3960%2Fjslrt.17006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepers, U.</span></span> <span> </span><span class="NLM_article-title">Polyamides as artificial transcription factors: novel tools for molecular medicine?</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2472</span>– <span class="NLM_lpage">2475</span>, <span class="refDoi"> DOI: 10.1002/anie.200301745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Fanie.200301745" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkt1Whsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=2472-2475&author=K.+Schmitzauthor=U.+Schepers&title=Polyamides+as+artificial+transcription+factors%3A+novel+tools+for+molecular+medicine%3F&doi=10.1002%2Fanie.200301745"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of gene expression: Polyamides as artificial transcription factors: novel tools for molecular medicine?</span></div><div class="casAuthors">Schmitz, Katja; Schepers, Ute</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2472-2475</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Misregulation of gene expression often leads to severe defects in the development of an organism or to uncontrolled cell growth.  Artificial transcription factors can control the expression of the genes concerned to restore normal cell regulation.  Dervan et al. have synthesized several DNA-binding polyamides: (TALE (yellow) is a class of homeobox protein) that show potential for the treatment of diseases such as cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocgK5DT4upzbVg90H21EOLACvtfcHk0liYFMX8zaCbJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkt1Whsbg%253D&md5=57a38e41b03467b753b6138b52fa01f4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fanie.200301745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200301745%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitz%26aufirst%3DK.%26aulast%3DSchepers%26aufirst%3DU.%26atitle%3DPolyamides%2520as%2520artificial%2520transcription%2520factors%253A%2520novel%2520tools%2520for%2520molecular%2520medicine%253F%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2004%26volume%3D43%26spage%3D2472%26epage%3D2475%26doi%3D10.1002%2Fanie.200301745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">On the pairing rules for recognition in the minor groove of DNA by pyrrole-imidazole polyamides</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">578</span>, <span class="refDoi"> DOI: 10.1016/S1074-5521(97)90243-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS1074-5521%2897%2990243-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9281524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1ansrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=569-578&author=S.+Whiteauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=On+the+pairing+rules+for+recognition+in+the+minor+groove+of+DNA+by+pyrrole-imidazole+polyamides&doi=10.1016%2FS1074-5521%2897%2990243-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">On the pairing rules for recognition in the minor groove of DNA by pyrrole-imidazole polyamides</span></div><div class="casAuthors">White, Sarah; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">569-578</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">Cell-permeable small mols. that target predetd. DNA sequences with high affinity and specificity have the potential to control gene expression.  A binary code has been developed to correlate DNA sequence with side-by-side pairings between N-methylpyrrole (Py) and N-methylimidazole (Im) carboxamides in the DNA minor groove.  We set out to det. the relative energetics of pairings of Im/Py, Py/Im, Im/Im, and Py/Py for targeting G·C and A·T base pairs.  A key specificity issue, which has not been previously addressed, is whether an Im/Im pair is energetically equiv. to an Im/Py pair for targeting G·C base pairs.  Equil. assocn. consts. were detd. at two five-base-pair sites for a series of four six-ring hairpin polyamides, to test the relative energetics of the four arom. amino-acid pairings opposite G·C and A·T base pairs in the central position.  We obsd. that a G·C base pair was effectively targeted with Im/Py but not Py/Im, Py/Py, or Im/Im.  The A·T base pair was effectively targeted with Py/Py but not Im/Py, Py/Im, or Im/Im.  An Im/Im pairing is energetically disfavored for the recognition of both A·T and G·C.  This specificity will create important limitations on undesirable slipped motifs that are available for unlinked dimers in the minor groove.  Baseline energetic parameters will thus be created which, using the predictability of the current pairing rules for specific mol. recognition of double-helical DNA, will guide further second-generation polyamide design for DNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS4iTwSAZYNrVg90H21EOLACvtfcHk0liYFMX8zaCbJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1ansrY%253D&md5=d9d60d5f85ef7c4d8d5038f744989467</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2897%2990243-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252897%252990243-X%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DOn%2520the%2520pairing%2520rules%2520for%2520recognition%2520in%2520the%2520minor%2520groove%2520of%2520DNA%2520by%2520pyrrole-imidazole%2520polyamides%26jtitle%3DChem.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D569%26epage%3D578%26doi%3D10.1016%2FS1074-5521%2897%2990243-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wade, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrksich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Design of peptides that bind in the minor groove of DNA at 5′-(A,T)5G(A,T)C(A,T) sequences by a dimeric side-by-side motif</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">8783</span>– <span class="NLM_lpage">8794</span>, <span class="refDoi"> DOI: 10.1021/ja00049a006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00049a006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK38XmtlCmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1992&pages=8783-8794&author=W.+S.+Wadeauthor=M.+Mrksichauthor=P.+B.+Dervan&title=Design+of+peptides+that+bind+in+the+minor+groove+of+DNA+at+5%E2%80%B2-%28A%2CT%295G%28A%2CT%29C%28A%2CT%29+sequences+by+a+dimeric+side-by-side+motif&doi=10.1021%2Fja00049a006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design of peptides that bind in the minor groove of DNA at 5'-(A,T)G(A,T)C(A,T)-3' sequences by a dimeric side-by-side motif</span></div><div class="casAuthors">Wade, Warren S.; Mrksich, Milan; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8783-94</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The designed peptides pyridine-2-carboxamide-netropsin (2-PyN) (I) and 1-methylimidazole-2-carboxamide-netropsin (2-ImN) (II) are crescent-shaped synthetic analogs of the natural products netropsin and distamycin A.  Footprinting expts. indicate that 2-PyN and 2-ImN bind the 5 base pair sequence 5'-TGTCA-3'.  Affinity cleaving data suggest that the complexes, 2-ImN.5'-TGTCA-3' and 2-PyN.5'-TGTCA-3', are composed of two equiv. orientations which disfavor a 1:1 model.  The footprinting and affinity cleaving data are in accord with a 2:1 complex where a novel side-by-side antiparallel dimer binds in the minor groove of double-helical DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8zsoompOt6rVg90H21EOLACvtfcHk0lhqMywlwAC4aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XmtlCmsL4%253D&md5=b7823a69acc54a2d722927c873a8aedf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja00049a006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00049a006%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DW.%2BS.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DDesign%2520of%2520peptides%2520that%2520bind%2520in%2520the%2520minor%2520groove%2520of%2520DNA%2520at%25205%25E2%2580%25B2-%2528A%252CT%25295G%2528A%252CT%2529C%2528A%252CT%2529%2520sequences%2520by%2520a%2520dimeric%2520side-by-side%2520motif%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1992%26volume%3D114%26spage%3D8783%26epage%3D8794%26doi%3D10.1021%2Fja00049a006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melander, C.</span></span> <span> </span><span class="NLM_article-title">Programmable DNA-binding small molecules</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6101</span>– <span class="NLM_lpage">6114</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.bmc.2013.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=23665141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVGmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6101-6114&author=M.+S.+Blackledgeauthor=C.+Melander&title=Programmable+DNA-binding+small+molecules&doi=10.1016%2Fj.bmc.2013.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Programmable DNA-binding small molecules</span></div><div class="casAuthors">Blackledge, Meghan S.; Melander, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6101-6114</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Aberrant gene expression is responsible for a myriad of human diseases from infectious diseases to cancer.  Precise regulation of these genes via specific interactions with the DNA double helix could pave the way for novel therapeutics.  Pyrrole-imidazole polyamides are small mols. capable of binding to pre-detd. DNA sequences up to 16 base pairs with affinity and specificity comparable to natural transcription factors.  In the three decades since their development, great strides have been made relating to synthetic accessibility and improved sequence specificity and binding affinity.  This perspective presents a brief history of early seminal developments in the field and highlights recent reports of the utility of polyamides as both genetic modulators and mol. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnyvt5n21h47Vg90H21EOLACvtfcHk0lhqMywlwAC4aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVGmurk%253D&md5=75d1c7ab474cde9bbfd15365f798ee24</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DBlackledge%26aufirst%3DM.%2BS.%26aulast%3DMelander%26aufirst%3DC.%26atitle%3DProgrammable%2520DNA-binding%2520small%2520molecules%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6101%26epage%3D6114%26doi%3D10.1016%2Fj.bmc.2013.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottesfeld, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neely, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauger, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Regulation of gene expression by small molecules</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1038/387202a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2F387202a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9144294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK2sXjtF2msrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=1997&pages=202-205&author=J.+M.+Gottesfeldauthor=L.+Neelyauthor=J.+W.+Traugerauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=Regulation+of+gene+expression+by+small+molecules&doi=10.1038%2F387202a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of gene expression by small molecules</span></div><div class="casAuthors">Gottesfeld, Joel M.; Neely, Laura; Trauger, John W.; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">6629</span>),
    <span class="NLM_cas:pages">202-205</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Small mols. that target specific DNA sequences have the potential to control gene expression.  Ligands designed for therapeutic application must bind any predetd. DNA sequence with high affinity and permeate living cells.  Synthetic polyamides contg. N-methylimidazole and N-methylpyrrole amino acids have an affinity and specificity for DNA comparable to naturally occurring DNA-binding proteins1.  We report here that an eight-ring polyamide targeted to a specific region of the transcription factor TFIIIA binding site interferes with 5S RNA gene expression in Xenopus kidney cells.  Our results indicate that pyrrole-imidazole polyamides are cell-permeable and can inhibit the transcription of specific genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUDTLP-1ZKgLVg90H21EOLACvtfcHk0lhqMywlwAC4aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjtF2msrg%253D&md5=f3ebbe7bea63c4674894bb38550defce</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2F387202a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F387202a0%26sid%3Dliteratum%253Aachs%26aulast%3DGottesfeld%26aufirst%3DJ.%2BM.%26aulast%3DNeely%26aufirst%3DL.%26aulast%3DTrauger%26aufirst%3DJ.%2BW.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DRegulation%2520of%2520gene%2520expression%2520by%2520small%2520molecules%26jtitle%3DNature%26date%3D1997%26volume%3D387%26spage%3D202%26epage%3D205%26doi%3D10.1038%2F387202a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trauger, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Recognition of DNA by designed ligands at subnanomolar concentrations</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">561</span>, <span class="refDoi"> DOI: 10.1038/382559a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2F382559a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=8700233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK28XkvFGjtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=559-561&author=J.+W.+Traugerauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=Recognition+of+DNA+by+designed+ligands+at+subnanomolar+concentrations&doi=10.1038%2F382559a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of DNA by designed ligands at subnanomolar concentrations</span></div><div class="casAuthors">Trauger, John W.; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">6591</span>),
    <span class="NLM_cas:pages">559-561</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Small mols. that specifically bind with high affinity to any predetd. DNA sequence in the human genome would be useful tools in mol. biol. and potentially in human medicine.  Simple rules have been developed to control rationally the sequence specificity of minor-groove-binding polyamides contg. N-methylimidazole and N-methylpyrrole amino acids.  Two eight-ring pyrrole-imidazole polyamides differing in sequence by a single amino acid bind specifically to resp. six-base-pair target sites which differ in sequence by a single base pair.  Binding is obsd. at subnanomolar concns. of ligand.  The replacement of a single nitrogen atom with a C-H regulates affinity and specificity by two orders of magnitude.  The broad range of sequences that can be specifically targeted with pyrrole-imidazole polyamides, coupled with an efficient solid-phase synthesis methodol., identify a powerful class of small mols. for sequence-specific recognition of double-helical DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0_GQE1-LS77Vg90H21EOLACvtfcHk0lhqMywlwAC4aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XkvFGjtLw%253D&md5=6b6c02141026961f8216226cf0afac6e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2F382559a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F382559a0%26sid%3Dliteratum%253Aachs%26aulast%3DTrauger%26aufirst%3DJ.%2BW.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DRecognition%2520of%2520DNA%2520by%2520designed%2520ligands%2520at%2520subnanomolar%2520concentrations%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D559%26epage%3D561%26doi%3D10.1038%2F382559a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span> <span> </span><span class="NLM_article-title">Recognition of the four Watson-Crick base pairs in the DNA minor groove by synthetic ligands</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">468</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.1038/35106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2F35106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9461213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1cXhtVemt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=1998&pages=468-471&author=S.+Whiteauthor=J.+W.+Szewczykauthor=J.+M.+Turnerauthor=E.+E.+Bairdauthor=P.+B.+Dervan&title=Recognition+of+the+four+Watson-Crick+base+pairs+in+the+DNA+minor+groove+by+synthetic+ligands&doi=10.1038%2F35106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of the four Watson-Crick base pairs in the DNA minor groove by synthetic ligands</span></div><div class="casAuthors">White, Sarah; Szewczyk, Jason W.; Turner, James M.; Baird, Eldon E.; Dervan, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">6666</span>),
    <span class="NLM_cas:pages">468-471</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The design of synthetic ligands that read the information stored in the DNA double helix has been a long-standing goal at the interface of chem. and biol.  Cell-permeable small mols. that target predetd. DNA sequences offer a potential approach for the regulation of gene expression.  Oligodeoxynucleotides that recognize the major groove of double-helical DNA via triple-helix formation bind to a broad range of sequences with high affinity and specificity.  Although oligonucleotides and their analogs have been shown to interfere with gene expression, the triple-helix approach is limited to recognition of purines and suffers from poor cellular uptake.  The subsequent development of pairing rules for minor-groove binding polyamides contg. pyrrole (Py) and imidazole (Im) amino acids offers a second code to control sequence specificity.  An Im/Py pair distinguishes G⪆C from C⪆G and both of these from A·T/T·A base pairs.  A Py/Py pair specifies A,T from G,C but does not distinguish A·T from T·A.  To break this degeneracy, we have added a new arom. amino acid, 3-hydroxypyrrole (Hp), to the repertoire to test for pairings that discriminate A·T from T⪆a.  We find that replacement of a single hydrogen atom with a hydroxy group in a Hp/Py pairing regulates affinity and specificity by an order of magnitude.  By incorporation of this third amino acid, hydroxypyrrole-imidazole-pyrrole polyamides form four ring-pairings (Im/Py, Py/Im, Hp/Py and Py/Hp) which distinguish all Four Watson-Crick base pairs in the minor groove of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg-5qvdXyBvbVg90H21EOLACvtfcHk0lh65fDG2zTPKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtVemt78%253D&md5=38aa613c6a8afc2b933333f8779c392e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F35106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35106%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DJ.%2BW.%26aulast%3DTurner%26aufirst%3DJ.%2BM.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26atitle%3DRecognition%2520of%2520the%2520four%2520Watson-Crick%2520base%2520pairs%2520in%2520the%2520DNA%2520minor%2520groove%2520by%2520synthetic%2520ligands%26jtitle%3DNature%26date%3D1998%26volume%3D391%26spage%3D468%26epage%3D471%26doi%3D10.1038%2F35106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kielkopf, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">A structural basis for recognition of A·T and T·A base pairs in the minor groove of B-DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1126/science.282.5386.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1126%2Fscience.282.5386.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9756473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1cXmsFOjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1998&pages=111-115&author=C.+L.+Kielkopfauthor=S.+Whiteauthor=J.+W.+Szewczykauthor=J.+M.+Turnerauthor=E.+E.+Bairdauthor=P.+B.+Dervanauthor=D.+C.+Rees&title=A+structural+basis+for+recognition+of+A%C2%B7T+and+T%C2%B7A+base+pairs+in+the+minor+groove+of+B-DNA&doi=10.1126%2Fscience.282.5386.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A structural basis for recognition of A·T and T·A base pairs in the minor groove of B-DNA</span></div><div class="casAuthors">Kielkopf, Clara L.; White, Sarah; Sewczyk, Jason W.; Turner, James M.; Baird, Eldon E.; Dervan, Peter B.; Rees, Douglas C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">5386</span>),
    <span class="NLM_cas:pages">111-115</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Polyamide dimers contg. three types of arom. rings-pyrrole, imidazole, and hydroxypyrrole-afford a small-mol. recognition code that discriminates among all four Watson-Crick base pairs in the minor groove.  The crystal structure of a specific polyamide dimer-DNA complex establishes the structural basis for distinguishing T·A from A·T base pairs.  Specificity for the T·A base pair is achieved by means of distinct hydrogenbonds between pairs of substituted pyrroles on the ligand and the O2 of thymine and N3 of adenine.  In addn., shape-selective recognition of an asym. cleft between the thymine-O2 and the adenine-C2 was obsd.  Although hitherto similarities among the base pairs in the minor groove have been emphasized, the structure illustrates differences that allow specific minor groove recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuMGOkDl_rxbVg90H21EOLACvtfcHk0lh65fDG2zTPKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsFOjtbc%253D&md5=934706075f158b01006a993c81c17e14</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fscience.282.5386.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.282.5386.111%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DJ.%2BW.%26aulast%3DTurner%26aufirst%3DJ.%2BM.%26aulast%3DBaird%26aufirst%3DE.%2BE.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DA%2520structural%2520basis%2520for%2520recognition%2520of%2520A%25C2%25B7T%2520and%2520T%25C2%25B7A%2520base%2520pairs%2520in%2520the%2520minor%2520groove%2520of%2520B-DNA%26jtitle%3DScience%26date%3D1998%26volume%3D282%26spage%3D111%26epage%3D115%26doi%3D10.1126%2Fscience.282.5386.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelson, B. S.</span></span> <span> </span><span class="NLM_article-title">Recognition of the DNA minor groove by pyrrloe-imidazole polyamides</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(03)00081-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS0959-440X%2803%2900081-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=12831879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvVKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=284-299&author=P.+B.+Dervanauthor=B.+S.+Edelson&title=Recognition+of+the+DNA+minor+groove+by+pyrrloe-imidazole+polyamides&doi=10.1016%2FS0959-440X%2803%2900081-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of the DNA minor groove by pyrrole-imidazole polyamides</span></div><div class="casAuthors">Dervan, Peter B.; Edelson, Benjamin S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">284-299</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Many diseases, such as cancer, are related to aberrant gene expression.  Regulating transcription by chem. methods could be important in human medicine.  Minor groove-binding polyamides offer one chem. approach to DNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ymr2dTEUT7Vg90H21EOLACvtfcHk0lh65fDG2zTPKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvVKnsrw%253D&md5=9f0a7b75f02d63ef70b61994acbe87d0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2803%2900081-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252803%252900081-2%26sid%3Dliteratum%253Aachs%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DEdelson%26aufirst%3DB.%2BS.%26atitle%3DRecognition%2520of%2520the%2520DNA%2520minor%2520groove%2520by%2520pyrrloe-imidazole%2520polyamides%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2003%26volume%3D13%26spage%3D284%26epage%3D299%26doi%3D10.1016%2FS0959-440X%2803%2900081-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span> <span> </span><span class="NLM_article-title">Discrimination of hairpin polyamides with an alpha-substituted-gamma-aminobutyric acid as a TG-reader in DNA minor groove</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">8766</span>– <span class="NLM_lpage">8776</span>, <span class="refDoi"> DOI: 10.1021/ja0580587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0580587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtVSisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=8766-8776&author=W.+Zhangauthor=T.+Bandoauthor=H.+Sugiyama&title=Discrimination+of+hairpin+polyamides+with+an+alpha-substituted-gamma-aminobutyric+acid+as+a+TG-reader+in+DNA+minor+groove&doi=10.1021%2Fja0580587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discrimination of Hairpin Polyamides with an α-Substituted-γ-aminobutyric Acid as a 5'-TG-3' Reader in DNA Minor Groove</span></div><div class="casAuthors">Zhang, Wen; Bando, Toshikazu; Sugiyama, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8766-8776</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pyrrole-imidazole (Py-Im) polyamides contg. stereospecifically α-amino- or α-hydroxyl-substituted γ-aminobutyric acid as a 5'-TG-3' recognition element were synthesized by machine-assisted Fmoc solid-phase synthesis.  Their binding properties to predetd. DNA sequences contg. a core binding site of 5'-TGCNCA-3'/3'-ACGN'GT-5' (N·N' = A·T, T·A, G·C, and C·G) were then systematically studied by surface plasmon resonance (SPR).  SPR results revealed that the pairing of stereospecifically α-amino-/α-hydroxyl-substituted γ-aminobutyric acids, (R or S)-α,γ-diaminobutyric acid (γRN or γSN) and (R or S)-α-hydroxyl-γ-aminobutyric acid (γRO or γSO), side-by-side with β-alanine (β) in such polyamides significantly influenced the DNA binding affinity and recognition specificity of hairpin polyamides in the DNA minor groove compared with β/β, β/γ, and γ/β pairings.  More importantly, the polyamide Ac-Im-γSO-ImPy-γ-ImPyβPy-β-Dp (β/γSO) favorably binds to a hairpin DNA contg. a core binding site of 5'-TGCNCA-3'/3'-ACGN'GT-5' (N·N' = A·T) with dissocn. equil. const. (KD) of 1.9×10-7 M over N·N' = T·A with KD = 3.7×10-6 M, with a 19-fold specificity.  By contrast, Ac-Im-γSN-ImPy-γ-ImPyβPy-β-Dp (β/γSN) binds to the above sequence with N·N' = A·T with KD = 8.7×10-7 M over N·N' = T·A with KD = 8.4×10-6 M, with a 9.6-fold specificity.  The results also show that the stereochem. of the α-substituent, as well as the α-substituent itself may greatly alter binding affinity and recognition selectivity of hairpin polyamides to different DNA sequences.  Further, we carried out mol. modeling studies on the binding by an energy minimization method, suggesting that α-hydroxyl is very close to N3 of the 3'-terminal G to induce the formation of hydrogen bonding between hydroxyl and N3 in the recognition event of the polyamide Ac-Im-γSO-ImPy-γ-ImPyβPy-β-Dp (β/γSO) to 5'-TGCNCA-3'/3'-ACGN'GT-5' (N·N' = A·T).  Therefore, SPR assays and mol. modeling studies collectively suggest that the (S)-α-hydroxyl-γ-aminobutyric acid (γSO) may act as a 5'-TG-3' recognition unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqusw2YO4yGrVg90H21EOLACvtfcHk0liNwBFPcogDEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtVSisb8%253D&md5=1fbbfdc9dc4d4db2a7bff0f84cf2c1f2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja0580587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0580587%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26atitle%3DDiscrimination%2520of%2520hairpin%2520polyamides%2520with%2520an%2520alpha-substituted-gamma-aminobutyric%2520acid%2520as%2520a%2520TG-reader%2520in%2520DNA%2520minor%2520groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D8766%26epage%3D8776%26doi%3D10.1021%2Fja0580587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Discrimination between T/A and A/T base pairs of pyrrole-imidazole polyamides substituted with chiral β-hydroxy-γ-aminobutyric acid/β-alanine pairs</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Fcbic.201100675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22113893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFaltLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=47-50&author=W.+Zhangauthor=S.+Jiangauthor=Y.+Wuauthor=C.+Guoauthor=H.+Zhangauthor=H.+Sugiyamaauthor=X.+Chen&title=Discrimination+between+T%2FA+and+A%2FT+base+pairs+of+pyrrole-imidazole+polyamides+substituted+with+chiral+%CE%B2-hydroxy-%CE%B3-aminobutyric+acid%2F%CE%B2-alanine+pairs&doi=10.1002%2Fcbic.201100675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discrimination between T/A and A/T Base Pairs of Pyrrole-Imidazole Polyamides Substituted with Chiral β-Hydroxy-γ-Aminobutyric Acid/β-Alanine Pairs</span></div><div class="casAuthors">Zhang, Wen; Jiang, Shi-Kun; Wu, Yan-Ling; Guo, Chuan-Xin; Zhang, Hong-Fei; Sugiyama, Hiroshi; Chen, Xing-Lai</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-50</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors synthesized five novel pyrrole-imidazole polyamides 1-5 contg. chiral β-hydroxy-γ-aminobutyric acid/β-alanine pairs, and examd. their selective recognition ability with a β/β pairing to Watson-Crick base pairs T·A, A·T, C·G, G·C in the DNA minor groove.  It was obsd. that no binding to 5'-TCCNCCA-3'/5'-TGGN'GGA-3' (N·N' = C·G, G·C) was obsd. at low concns. of polyamides 1-5, but there was large nonspecific binding at higher concns.  The authors obtained binding consts. for assocn. of 1-5 to 5'-TCCNCCA-3'/5'-TGGN'GGA-3' (N·N' = A·T, T·A), in addn. to thermodn. std. free-energy change (ΔG°), hydrogen bonding length, and interaction energy upon forming of the polyamide-DNA complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM6YJ1lmuGH7Vg90H21EOLACvtfcHk0liNwBFPcogDEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFaltLzE&md5=30e635f52d94d3b114d5cf47d90acb8b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100675%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DDiscrimination%2520between%2520T%252FA%2520and%2520A%252FT%2520base%2520pairs%2520of%2520pyrrole-imidazole%2520polyamides%2520substituted%2520with%2520chiral%2520%25CE%25B2-hydroxy-%25CE%25B3-aminobutyric%2520acid%252F%25CE%25B2-alanine%2520pairs%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D47%26epage%3D50%26doi%3D10.1002%2Fcbic.201100675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulin-Kerstien, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fechter, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelson, B. S.</span></span> <span> </span><span class="NLM_article-title">Regulation of gene expression by synthetic DNA-binding ligands</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1007/b100440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1007%2Fb100440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsF2isbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=2005&pages=1-31&author=P.+B.+Dervanauthor=A.+T.+Poulin-Kerstienauthor=E.+J.+Fechterauthor=B.+S.+Edelson&title=Regulation+of+gene+expression+by+synthetic+DNA-binding+ligands&doi=10.1007%2Fb100440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of gene expression by synthetic DNA-binding ligands</span></div><div class="casAuthors">Dervan, Peter B.; Poulin-Kerstien, Adam T.; Fechter, Eric J.; Edelson, Benjamin S.</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">DNA Binders and Related Subjects</span>),
    <span class="NLM_cas:pages">1-31</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">During the past 20 years, polyamides have evolved from the natural product distamycin to a new class of programmable heterocyclic oligomers that bind a broad repertoire of DNA sequences with high affinity and specificity.  This chapter details recent advances in this field of research, focusing on mol. recognition of DNA, and biol. applications such as modulating gene expression by small mols.  Work presented here represents efforts towards the modulation of specific cellular function by small mols. in an addressable fashion within the context of live cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEXxLDeer5krVg90H21EOLACvtfcHk0liNwBFPcogDEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsF2isbs%253D&md5=2a8f54b687c72f0c8e61138445b79b38</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fb100440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fb100440%26sid%3Dliteratum%253Aachs%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DPoulin-Kerstien%26aufirst%3DA.%2BT.%26aulast%3DFechter%26aufirst%3DE.%2BJ.%26aulast%3DEdelson%26aufirst%3DB.%2BS.%26atitle%3DRegulation%2520of%2520gene%2520expression%2520by%2520synthetic%2520DNA-binding%2520ligands%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2005%26volume%3D253%26spage%3D1%26epage%3D31%26doi%3D10.1007%2Fb100440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahira, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugishima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serie, K.</span></span> <span> </span><span class="NLM_article-title">Development of gene silencing pyrrole-imidazolepolyamide targeting the TGF-β1 promoter for treatment of progressive renal diseases</span>. <i>J. Am. Soc. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1681/ASN.2005060650</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1681%2FASN.2005060650" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=16371433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FksFOnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2006&pages=422-432&author=H.+Matsudaauthor=N.+Fukudaauthor=T.+Uenoauthor=Y.+Tahiraauthor=H.+Ayameauthor=W.+Zhangauthor=T.+Bandoauthor=H.+Sugiyamaauthor=S.+Saitoauthor=K.+Matsumotoauthor=H.+Mugishimaauthor=K.+Serie&title=Development+of+gene+silencing+pyrrole-imidazolepolyamide+targeting+the+TGF-%CE%B21+promoter+for+treatment+of+progressive+renal+diseases&doi=10.1681%2FASN.2005060650"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of gene silencing pyrrole-imidazole polyamide targeting the TGF-beta1 promoter for treatment of progressive renal diseases</span></div><div class="casAuthors">Matsuda Hiroyuki; Fukuda Noboru; Ueno Takahiro; Tahira Yoshiko; Ayame Hirohito; Zhang Wen; Bando Toshikazu; Sugiyama Hiroshi; Saito Satoshi; Matsumoto Koichi; Mugishima Hideo; Serie Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society of Nephrology : JASN</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-32</span>
        ISSN:<span class="NLM_cas:issn">1046-6673</span>.
    </div><div class="casAbstract">Pyrrole-imidazole (Py-Im) polyamides are nuclease-resistant novel compounds that inhibit gene expression by binding to the minor groove of DNA.  A Py-Im polyamide that targets rat TGF-beta1 was designed as a gene-silencing agent for progressive renal diseases, and the distribution and the effects of this polyamide on renal injury were examined in Dahl-salt sensitive (Dahl-S) rats.  For identification of transcription factor binding elements for activation of the rat TGF-beta1 gene, recombinant TGF-beta1 reporter plasmids were transfected into HEK-293 cells, and promoter activity was measured.  Py-Im polyamide was designed to the activator protein-1 binding site of the rat TGF-beta1 promoter.  This Py-Im polyamide showed strong, fast, and specific binding to the target DNA in gel mobility shift and Biacore assays.  Py-Im polyamide significantly inhibited TGF-beta1 promoter activity and expression of TGF-beta1 mRNA and protein in rat mesangial cells.  Intravenously administered fluorescein-labeled polyamide distributed to the kidney of rats.  Py-Im polyamide significantly inhibited expression of TGF-beta1 mRNA and protein in the renal cortex of Dahl-S rats and reduced the increase in urinary protein and albumin in Dahl-S rats independent of changes in blood pressure.  These results indicate that Py-Im polyamide that targets TGF-beta1 will be a novel gene-silencing agent for the TGF-beta1-associated diseases, including progressive renal diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSydrr4CeAj4vxQvRxYU81tfW6udTcc2ea-sQRRW4zcLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28%252FksFOnsw%253D%253D&md5=b566cddae3baecfd8d9413a546d96266</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1681%2FASN.2005060650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1681%252FASN.2005060650%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DH.%26aulast%3DFukuda%26aufirst%3DN.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DTahira%26aufirst%3DY.%26aulast%3DAyame%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DSaito%26aufirst%3DS.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DMugishima%26aufirst%3DH.%26aulast%3DSerie%26aufirst%3DK.%26atitle%3DDevelopment%2520of%2520gene%2520silencing%2520pyrrole-imidazolepolyamide%2520targeting%2520the%2520TGF-%25CE%25B21%2520promoter%2520for%2520treatment%2520of%2520progressive%2520renal%2520diseases%26jtitle%3DJ.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2006%26volume%3D17%26spage%3D422%26epage%3D432%26doi%3D10.1681%2FASN.2005060650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Washio, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terui, T.</span></span> <span> </span><span class="NLM_article-title">Transcriptional inhibition of hypertrophicscars by a gene silencer, pyrrole-imidazole polyamide, targeting the TGF-β1 promoter</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">1987</span>– <span class="NLM_lpage">1995</span>, <span class="refDoi"> DOI: 10.1038/jid.2011.150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fjid.2011.150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21654833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOls7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2011&pages=1987-1995&author=H.+Washioauthor=N.+Fukudaauthor=H.+Matsudaauthor=H.+Nagaseauthor=T.+Watanabeauthor=Y.+Matsumotoauthor=T.+Terui&title=Transcriptional+inhibition+of+hypertrophicscars+by+a+gene+silencer%2C+pyrrole-imidazole+polyamide%2C+targeting+the+TGF-%CE%B21+promoter&doi=10.1038%2Fjid.2011.150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Inhibition of Hypertrophic Scars by a Gene Silencer, Pyrrole-Imidazole Polyamide, Targeting the TGF-β1 Promoter</span></div><div class="casAuthors">Washio, Hisayo; Fukuda, Noboru; Matsuda, Hiroyuki; Nagase, Hiroki; Watanabe, Takayoshi; Matsumoto, Yoshiaki; Terui, Tadashi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1987-1995</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Synthetic pyrrole-imidazole (PI) polyamides bind to the minor groove of double-helical DNA with high affinity and specificity, and inhibit the transcription of corresponding genes.  The authors examd. the effects of a transforming growth factor (TGF)-β1-targeted PI polyamide (Polyamide) on hypertrophic skin scars in rats.  Hypertrophic scars were created dorsally in rats by incisions.  FITC-labeled Polyamide was injected to investigate its distribution in the skin.  Expression of TGF-β1, connective tissue growth factor (CTGF), collagen type 1, and fibronectin mRNAs was evaluated by reverse transcription PCR anal.  The extent of fibrosis and the expression of TGF-β1 were evaluated histol. and immunohistochem.  Polyamide was distributed in almost all nuclei of skin cells.  Expression of TGF-β1 mRNA reached a peak at 3 days after skin incision.  Expression of CTGF and extracellular matrix mRNAs was increased continuously even after the peak induction of TGF-β1 mRNA.  Injection of Polyamide completely inhibited both the development of scars and the induction of growth factors and extracellular matrix mRNAs.  The treatment also markedly inhibited fibrotic changes and reduced the nos. of vimentin-pos. spindle-shaped fibroblasts.  Injection of Polyamide also reduced established hypertrophic scars in rats.  Thus, TGF-β1-targeted PI polyamide should be a feasible gene silencer for hypertrophic scars and keloids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1TvKWk0ahILVg90H21EOLACvtfcHk0ljWyhElR-FgvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOls7vM&md5=caadc49f10307d093967a6fb3c6564ce</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fjid.2011.150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2011.150%26sid%3Dliteratum%253Aachs%26aulast%3DWashio%26aufirst%3DH.%26aulast%3DFukuda%26aufirst%3DN.%26aulast%3DMatsuda%26aufirst%3DH.%26aulast%3DNagase%26aufirst%3DH.%26aulast%3DWatanabe%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DTerui%26aufirst%3DT.%26atitle%3DTranscriptional%2520inhibition%2520of%2520hypertrophicscars%2520by%2520a%2520gene%2520silencer%252C%2520pyrrole-imidazole%2520polyamide%252C%2520targeting%2520the%2520TGF-%25CE%25B21%2520promoter%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2011%26volume%3D131%26spage%3D1987%26epage%3D1995%26doi%3D10.1038%2Fjid.2011.150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soma, M.</span></span> <span> </span><span class="NLM_article-title">Preclinical Study of novel gene silencer Pyrrole-Imidazole Polyamide targeting human TGF-β1 promoter for hypertrophic scars in a common marmoset primate model</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0125295</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0125295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1371%2Fjournal.pone.0125295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25938472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWqtbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=J.+Igarashiauthor=N.+Fukudaauthor=T.+Inoueauthor=S.+Nakaiauthor=K.+Saitoauthor=K.+Fujiwaraauthor=H.+Matsudaauthor=T.+Uenoauthor=Y.+Matsumotoauthor=T.+Watanabeauthor=H.+Nagaseauthor=T.+Bandoauthor=H.+Sugiyamaauthor=T.+Itohauthor=M.+Soma&title=Preclinical+Study+of+novel+gene+silencer+Pyrrole-Imidazole+Polyamide+targeting+human+TGF-%CE%B21+promoter+for+hypertrophic+scars+in+a+common+marmoset+primate+model&doi=10.1371%2Fjournal.pone.0125295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical study of novel gene silencer pyrrole-imidazole polyamide targeting human TGF-β1 promoter for hypertrophic scars in a common marmoset primate model</span></div><div class="casAuthors">Igarashi, Jun; Fukuda, Noboru; Inoue, Takashi; Nakai, Shigeki; Saito, Kosuke; Fujiwara, Kyoko; Matsuda, Hiroyuki; Ueno, Takahiro; Matsumoto, Yoshiaki; Watanabe, Takayoshi; Nagase, Hiroki; Bando, Toshikazu; Sugiyama, Hiroshi; Itoh, Toshio; Soma, Masayoshi</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e0125295/1-e0125295/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">We report a preclin. study of a pyrrole-imidazole (PI) polyamide that targets the human transforming growth factor (hTGF)-β1 gene as a novel transcriptional gene silencer in a common marmoset primate model.  We designed and then synthesized PI polyamides to target the hTGF-β1 promoter.  We examd. effects of seven PI polyamides (GB1101-1107) on the expression of hTGF-β1 mRNA stimulated with phorbol 12-myristate 13-acetate (PMA) in human vascular smooth muscle cells.  GB1101, GB1105 and GB1106 significantly inhibited hTGF-β1 mRNA expression.  We examd. GB1101 as a PI polyamide to hTGF-β1 for hypertrophic scars in marmosets in vivo.  Injection of GB1101 completely inhibited hypertrophic scar formation at 35 days post-incision and inhibited cellular infiltration, TGF-β1 and vimentin staining, and epidermal thickness.  Mismatch polyamide did not affect hypertrophic scarring or histol. changes.  Epidermis was significantly thinner with GB1101 than with water and mismatch PI polyamides.  We developed the PI polyamides for practical ointment medicines for the treatment of hypertrophic scars.  FITC-labeled GB1101 with solbase most efficiently distributed in the nuclei of epidermal keratinocytes, completely suppressed hypertropic scarring at 42 days after incision, and considerably inhibited epidermal thickness and vimentin-pos. fibroblasts.  PI polyamides targeting hTGF-β1 promoter with solbase ointment will be practical medicines for treating hypertrophic scars after surgical operations and skin burns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi6uMbHEPZ2rVg90H21EOLACvtfcHk0ljWyhElR-FgvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWqtbvE&md5=f42751d121a52d415f17739096c15ade</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0125295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0125295%26sid%3Dliteratum%253Aachs%26aulast%3DIgarashi%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DN.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DNakai%26aufirst%3DS.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DK.%26aulast%3DMatsuda%26aufirst%3DH.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DWatanabe%26aufirst%3DT.%26aulast%3DNagase%26aufirst%3DH.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DSoma%26aufirst%3DM.%26atitle%3DPreclinical%2520Study%2520of%2520novel%2520gene%2520silencer%2520Pyrrole-Imidazole%2520Polyamide%2520targeting%2520human%2520TGF-%25CE%25B21%2520promoter%2520for%2520hypertrophic%2520scars%2520in%2520a%2520common%2520marmoset%2520primate%2520model%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0125295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, M. HR.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">DNA binding and nuclease activity of a water-soluble sulfonated manganese(III) corrole</span>. <i>Transition Met. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1007/s11243-013-9689-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1007%2Fs11243-013-9689-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVOnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=283-289&author=J.+Huangauthor=X.+Wangauthor=Y.+Zhangauthor=M.+HR.+Mahmoodauthor=Y.+Huangauthor=X.+Yingauthor=L.+Jiauthor=H.+Liu&title=DNA+binding+and+nuclease+activity+of+a+water-soluble+sulfonated+manganese%28III%29+corrole&doi=10.1007%2Fs11243-013-9689-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binding and nuclease activity of a water-soluble sulfonated manganese(III) corrole</span></div><div class="casAuthors">Huang, Jun-Teng; Wang, Xiang-Li; Zhang, Yang; Mahmood, Mian H. R.; Huang, Yan-Yuan; Ying, Xiao; Ji, Liang-Nian; Liu, Hai-Yang</div><div class="citationInfo"><span class="NLM_cas:title">Transition Metal Chemistry (Dordrecht, Netherlands)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-289</span>CODEN:
                <span class="NLM_cas:coden">TMCHDN</span>;
        ISSN:<span class="NLM_cas:issn">0340-4285</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The interaction of a water-sol. sulfonated Mn(III) corrole Mn(tpfc)(SO3Na)2 (3) [tpfc = 5,10,15-tris(pentafluorophenyl)corrole] with calf thymus DNA (ct-DNA) was studied by spectroscopic methods, and the nuclease activity of this complex also was examd. by agarose gel electrophoresis.  Mn(tpfc)(SO3Na)2 (3) exhibits weak aggregation tendency in buffer soln. and can bind to ct-DNA via an outside binding mode with a binding const. of 1.25 × 104 M-1.  The obsd. increase in Stern-Volmer quenching const. with increasing temp. indicates that the competition of the manganese corrole and ethidium bromide with ct-DNA is a dynamic process.  Moreover, the manganese corrole displays good chem. nuclease activity in the presence of hydrogen peroxide via oxidative cleavage of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOfDuwXgl_rrVg90H21EOLACvtfcHk0ljFMD-4VCEZOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVOnur8%253D&md5=dee78125bb327465806f2e276748320f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs11243-013-9689-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11243-013-9689-5%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMahmood%26aufirst%3DM.%2BHR.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DYing%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDNA%2520binding%2520and%2520nuclease%2520activity%2520of%2520a%2520water-soluble%2520sulfonated%2520manganese%2528III%2529%2520corrole%26jtitle%3DTransition%2520Met.%2520Chem.%26date%3D2013%26volume%3D38%26spage%3D283%26epage%3D289%26doi%3D10.1007%2Fs11243-013-9689-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, A.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span> <span> </span><span class="NLM_article-title">Following G-quartet formation by UV-spectroscopy</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(98)01043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2FS0014-5793%2898%2901043-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=9755862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADyaK1cXmtlSmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=1998&pages=74-78&author=J.-L.+Mergnyauthor=A.-T.+Phanauthor=L.+Lacroix&title=Following+G-quartet+formation+by+UV-spectroscopy&doi=10.1016%2FS0014-5793%2898%2901043-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Following G-quartet formation by UV-spectroscopy</span></div><div class="casAuthors">Mergny, Jean-Louis; Phan, Anh-Tuan; Lacroix, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-78</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Oligodeoxynucleotides which include stretches of guanines form a well-known tetrameric structure.  We show that the recording of reversible absorbance changes at 295 nm allows to precisely monitor intramol. guanine (G)-quartet formation and dissocn.  Accurate Tm and thermodn. values could be easily extd. from the data, whereas classical recordings at 260 nm led to a much larger uncertainty and in extreme cases, to completely inaccurate measurements.  This inverted denaturation profile was obsd. for all G-quartet-forming oligonucleotides studied so far.  This technique is very useful in all cases where intramol. or intermol. quadruplex formation is suspected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_txGs3IpZNrVg90H21EOLACvtfcHk0ljFMD-4VCEZOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtlSmt78%253D&md5=8efa78a0b0dab38f7420d516fb653022</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2898%2901043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252898%252901043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMergny%26aufirst%3DJ.-L.%26aulast%3DPhan%26aufirst%3DA.-T.%26aulast%3DLacroix%26aufirst%3DL.%26atitle%3DFollowing%2520G-quartet%2520formation%2520by%2520UV-spectroscopy%26jtitle%3DFEBS%2520Lett.%26date%3D1998%26volume%3D435%26spage%3D74%26epage%3D78%26doi%3D10.1016%2FS0014-5793%2898%2901043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kypr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kejnovská, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renciuk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorlíčková, M.</span></span> <span> </span><span class="NLM_article-title">Circular dichroism and conformational polymorphism of DNA</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1725</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1093%2Fnar%2Fgkp026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=19190094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktV2ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=1713-1725&author=J.+Kyprauthor=I.+Kejnovsk%C3%A1author=D.+Renciukauthor=M.+Vorl%C3%AD%C4%8Dkov%C3%A1&title=Circular+dichroism+and+conformational+polymorphism+of+DNA&doi=10.1093%2Fnar%2Fgkp026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Circular dichroism and conformational polymorphism of DNA</span></div><div class="casAuthors">Kypr, Jaroslav; Kejnovska, Iva; Renciuk, Daniel; Vorlickova, Michaela</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1713-1725</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Here we review studies that provided important information about conformational properties of DNA using circular dichroic (CD) spectroscopy.  The conformational properties include the B-family of structures, A-form, Z-form, guanine quadruplexes, cytosine quadruplexes, triplexes and other less characterized structures.  CD spectroscopy is extremely sensitive and relatively inexpensive.  This fast and simple method can be used at low- as well as high-DNA concns. and with short- as well as long-DNA mols.  The samples can easily be titrated with various agents to cause conformational isomerizations of DNA.  The course of detected CD spectral changes makes possible to distinguish between gradual changes within a single DNA conformation and cooperative isomerizations between discrete structural states.  It enables measuring kinetics of the appearance of particular conformers and detn. of their thermodn. parameters.  In careful hands, CD spectroscopy is a valuable tool for mapping conformational properties of particular DNA mols.  Due to its numerous advantages, CD spectroscopy significantly participated in all basic conformational findings on DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtBf5jUZuhtbVg90H21EOLACvtfcHk0ljFMD-4VCEZOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktV2ru78%253D&md5=d03eafb37e5b3b6b6369198954b9999c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp026%26sid%3Dliteratum%253Aachs%26aulast%3DKypr%26aufirst%3DJ.%26aulast%3DKejnovsk%25C3%25A1%26aufirst%3DI.%26aulast%3DRenciuk%26aufirst%3DD.%26aulast%3DVorl%25C3%25AD%25C4%258Dkov%25C3%25A1%26aufirst%3DM.%26atitle%3DCircular%2520dichroism%2520and%2520conformational%2520polymorphism%2520of%2520DNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D1713%26epage%3D1725%26doi%3D10.1093%2Fnar%2Fgkp026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vorlíčková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kejnovská, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renčiuk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednářová, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motlová, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kypr, J.</span></span> <span> </span><span class="NLM_article-title">Circular dichroism and guanine quadruplexes</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2012.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ymeth.2012.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22450044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1ymt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=64-75&author=M.+Vorl%C3%AD%C4%8Dkov%C3%A1author=I.+Kejnovsk%C3%A1author=J.+Sagiauthor=D.+Ren%C4%8Diukauthor=K.+Bedn%C3%A1%C5%99ov%C3%A1author=J.+Motlov%C3%A1author=J.+Kypr&title=Circular+dichroism+and+guanine+quadruplexes&doi=10.1016%2Fj.ymeth.2012.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Circular dichroism and guanine quadruplexes</span></div><div class="casAuthors">Vorlickova, Michaela; Kejnovska, Iva; Sagi, Janos; Renciuk, Daniel; Bednarova, Klara; Motlova, Jitka; Kypr, Jaroslav</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-75</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  CD is remarkably sensitive to the conformational states of nucleic acids; therefore, CD spectroscopy was used to study most features of DNA and RNA structures.  Quadruplexes are among the significant noncanonical nucleic acids architectures that have received special attentions recently.  This article presents examples on the contribution of CD spectroscopy to the authors' knowledge of quadruplex structures and their polymorphism.  The examples were selected to demonstrate the potential of this simple method in the quadruplex field.  As CD spectroscopy detects only the global feature of a macromol., it should preferably be used in combination with other techniques.  On the other hand, CD spectroscopy, often as a pioneering approach, can reveal the formation of particular structural arrangements, to search for the conditions stabilizing the structures, to follow the transitions between various structural states, to explore kinetics of their appearance, to det. thermodn. parameters and also detect formation of higher order structures.  This article aims to show that CD spectroscopy is an important complementary technique to NMR spectroscopy and x-ray diffraction in quadruplex studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAwWkf_YGsV7Vg90H21EOLACvtfcHk0ljFMD-4VCEZOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1ymt7s%253D&md5=65cb6aacfbaa31ec902248f834993282</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2012.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2012.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DVorl%25C3%25AD%25C4%258Dkov%25C3%25A1%26aufirst%3DM.%26aulast%3DKejnovsk%25C3%25A1%26aufirst%3DI.%26aulast%3DSagi%26aufirst%3DJ.%26aulast%3DRen%25C4%258Diuk%26aufirst%3DD.%26aulast%3DBedn%25C3%25A1%25C5%2599ov%25C3%25A1%26aufirst%3DK.%26aulast%3DMotlov%25C3%25A1%26aufirst%3DJ.%26aulast%3DKypr%26aufirst%3DJ.%26atitle%3DCircular%2520dichroism%2520and%2520guanine%2520quadruplexes%26jtitle%3DMethods%26date%3D2012%26volume%3D57%26spage%3D64%26epage%3D75%26doi%3D10.1016%2Fj.ymeth.2012.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis of 4,5-diazafluorene derivatives and their anticancer activity via targeting telomeric DNA G-quadruplex</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>178</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">499</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejmech.2019.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=31202994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1SrsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2019&pages=484-499&author=K.+Zhouauthor=J.+Liuauthor=X.+Xiongauthor=M.+Chengauthor=X.+Huauthor=S.+Narvaauthor=X.+Zhaoauthor=Y.+Wuauthor=W.+Zhang&title=Design%2C+synthesis+of+4%2C5-diazafluorene+derivatives+and+their+anticancer+activity+via+targeting+telomeric+DNA+G-quadruplex&doi=10.1016%2Fj.ejmech.2019.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis of 4,5-diazafluorene derivatives and their anticancer activity via targeting telomeric DNA G-quadruplex</span></div><div class="casAuthors">Zhou, Kang; Liu, Jiachun; Xiong, Xuqiong; Cheng, Mei; Hu, Xiaolin; Narva, Suresh; Zhao, Xiaoyin; Wu, Yanling; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">484-499</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In our work, 19 novel 4,5-diazafluorene derivs. (11a-d, 12a-d, 13a-d, 14a-c, 15c, 16a-c) bearing a 1,3-disubstituted pyrazol/thioxothiazolidinone or thioxothiazolidinone-oxadiazole moieties were designed, synthesized, preliminarily explored for their antitumor activities and in vitro mechanism.  All compds. showed different values of antiproliferative activity against A549, AGS, HepG2 and MCF-7 cell lines through CCK-8.  Esp., the compd. 14c exhibited the strongest activity and best selectivity against A549 cells with an IC50 1.13 μM and an SI value of 7.01 relative to MRC-5 cells, which was better than cisplatin (SI = 1.80) as a pos. control.  Exptl. results at extracellular level demonstrated that compds. 14a-c could strongly interact with the G-quadruplex(es) formed in a 26 nt telomeric G-rich DNA, in particular, the 14c exhibits quite strong binding affinity with an assocn. equil. const. (KA) of 7.04(±0.16) × 107 M-1 and more than 1000-fold specificity to G4-DNA over ds-DNA and Mut-DNA at the compd./G4-DNA ratio of 1:1.  Further trap assay ascertained that compds. 14a-c owned strong inhibitory ability of telomerase activity in A549 cells, suggesting that these compds. have great possibility to target telomeric G-quadruplexes and consequently indirectly inhibit the telomerase activity.  In addn., it is worthy of note that the remarkable inhibitory effects of 14a-c on the mobility of tested cancer cells were obsd. by wound healing assays.  Furthermore, mol. docking and UV-Vis spectral results unclose the rationale for the interaction of compds. with such G-quadruplex(es).  These results indicate that the growth and metastasis inhibition of cancer cells mediated by these 4,5-diazafluorene derivs. possibly result from their interaction with telomeric G-quadruplexes, suggesting that 4,5-diazafluorene derivs., esp. 14c, possess potential as anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61fInFRE5NrVg90H21EOLACvtfcHk0lgkffidr_bieg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1SrsbrN&md5=be899fa13a48a9743475bc5a4c7da1df</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DNarva%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DDesign%252C%2520synthesis%2520of%25204%252C5-diazafluorene%2520derivatives%2520and%2520their%2520anticancer%2520activity%2520via%2520targeting%2520telomeric%2520DNA%2520G-quadruplex%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D178%26spage%3D484%26epage%3D499%26doi%3D10.1016%2Fj.ejmech.2019.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awadasseid, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and anti-cancer activity of pyrrole-imidazole polyamides through target-downregulation of c-kit gene expression</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>,  <span class="NLM_fpage">112704</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejmech.2020.112704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=32866755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&pages=112704&author=M.+Zhangauthor=J.+Liangauthor=S.+Jiangauthor=L.+Xuauthor=Y.+Wuauthor=A.+Awadasseidauthor=X.+Zhaoauthor=X.+Xiongauthor=H.+Sugiyamaauthor=W.+Zhang&title=Design%2C+synthesis+and+anti-cancer+activity+of+pyrrole-imidazole+polyamides+through+target-downregulation+of+c-kit+gene+expression&doi=10.1016%2Fj.ejmech.2020.112704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and anti-cancer activity of pyrrole-imidazole polyamides through target-downregulation of c-kit gene expression</span></div><div class="casAuthors">Zhang, Mi; Liang, Jing; Jiang, Shi-Kun; Xu, Ling; Wu, Yan-Ling; Awadasseid, Annoor; Zhao, Xiao-Yin; Xiong, Xu-Qiong; Sugiyama, Hiroshi; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112704</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Pyrrole-imidazole polyamide (PIP) can specifically bind in the B-DNA minor groove that has been used in several biol. applications, such as anti-cancer activity, gene expression and translation control, and visualization of complex genomic areas. c-kit is a family member of the Tyrosine Kinase (RTK) type III receptor and plays a vital role in tumor growth, proliferation, differentiation, and cell apoptosis; however, its mutations and overexpression induce tumor dysfunction.  Here, we designed and synthesized five matched PIPs that can recognize and bind to the DNA sequence in the oncogene c-kit promoter region, and evaluated their anti-cancer activity.  The RTCA assay findings revealed that the PIPs would prevent the proliferation of cancer cells A549 and SGC-7901.  EMSA assay showed that the PIPs were actively interacting with the c-kit gene target DNA.  RT-PCR and Western blot assays have an effect on expression levels of the c-kit gene in the presence of PIPs.  Flow cytometry and wound-healing assays showed that two of the PIPs would cause apoptosis of cancer cells and inhibit the migration of cells, resp.  Overall findings indicate that chloride of PIP (I) has a relatively significant intracellular and extracellular impact on the target, contributing to migration and proliferation redn., and cancer cell apoptosis.  In addn., PIP has a certain ability to evolve into c-kit gene-targeting drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJHTamfXa5z7Vg90H21EOLACvtfcHk0lgkffidr_bieg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjL&md5=eefd1e911d49a683806b7cad8361093b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112704%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DAwadasseid%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DDesign%252C%2520synthesis%2520and%2520anti-cancer%2520activity%2520of%2520pyrrole-imidazole%2520polyamides%2520through%2520target-downregulation%2520of%2520c-kit%2520gene%2520expression%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26spage%3D112704%26doi%3D10.1016%2Fj.ejmech.2020.112704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span> <span> </span><span class="NLM_article-title">A proton-fuelled DNA nanomachine</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">5734</span>– <span class="NLM_lpage">5736</span>, <span class="refDoi"> DOI: 10.1002/anie.200352402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1002%2Fanie.200352402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=14661209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=5734-5736&author=D.+Liuauthor=S.+Balasubramanian&title=A+proton-fuelled+DNA+nanomachine&doi=10.1002%2Fanie.200352402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A proton-fueled DNA nanomachine</span></div><div class="casAuthors">Liu, Dongsheng; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">5734-5736</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A mol. machine based on the DNA i-motif structure is driven by changing pH.  At pH 5, oligonucleotide X* folds into the i-motif (closed state), and complementary strand Y adopts a random-coil structure.  At pH 8, X* unfolds and forms an extended duplex with Y.  This reversible process can be monitored by means of a rhodamine green label, whose fluorescence is quenched by a dabcyl moiety in the closed but not in the open state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgIDQPvx_h5bVg90H21EOLACvtfcHk0lgkffidr_bieg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFahsr4%253D&md5=737e67368f61658bf5287f103ef53147</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.200352402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200352402%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DA%2520proton-fuelled%2520DNA%2520nanomachine%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26spage%3D5734%26epage%3D5736%26doi%3D10.1002%2Fanie.200352402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Biosensor-surface plasmon resonance: Quantitative analysis of small molecule–nucleic acid interactions</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2006.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ymeth.2006.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=17472897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=150-161&author=B.+Nguyenauthor=F.+A.+Taniousauthor=W.+D.+Wilson&title=Biosensor-surface+plasmon+resonance%3A+Quantitative+analysis+of+small+molecule%E2%80%93nucleic+acid+interactions&doi=10.1016%2Fj.ymeth.2006.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-surface plasmon resonance: quantitative analysis of small molecule-nucleic acid interactions</span></div><div class="casAuthors">Nguyen, Binh; Tanious, Farial A.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-161</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Surface plasmon resonance (SPR)-biosensor techniques directly provide essential information for the study and characterization of small mol.-nucleic acid interactions, and the use of these methods is steadily increasing.  The method is label-free and monitors the interactions in real time.  Both dynamic and steady-state information can be obtained for a wide range of reaction rates and binding affinities.  This article presents the basics of the SPR technique, provides suggestions for exptl. design, and illustrates data processing and anal. of results.  A specific example of the interaction of a well-known minor groove binding agent, netropsin, with DNA is evaluated by both kinetic and steady-state SPR methods.  Three different expts. are used to illustrate different approaches and anal. methods.  The three sets of results show the reproducibility of the binding consts. and agreement from both steady-state and kinetic analyses.  These expts. also show that reliable kinetic information can be obtained, even with difficult systems, if the exptl. conditions are optimized to minimize mass transport effects.  Limitations of the biosensor-SPR technique are also discussed to provide an awareness of the care needed to conduct a successful expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIMfX4ZwyTSLVg90H21EOLACvtfcHk0li6HKVlgUEfOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D&md5=9455fa47e2eefef19166b544873c9102</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2006.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2006.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DBiosensor-surface%2520plasmon%2520resonance%253A%2520Quantitative%2520analysis%2520of%2520small%2520molecule%25E2%2580%2593nucleic%2520acid%2520interactions%26jtitle%3DMethods%26date%3D2007%26volume%3D42%26spage%3D150%26epage%3D161%26doi%3D10.1016%2Fj.ymeth.2006.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Clairac, R. P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geierstanger, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrksich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervan, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wemmer, D. E.</span></span> <span> </span><span class="NLM_article-title">NMR characterization of hairpin polyamide complexed with the minor groove of DNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">7909</span>– <span class="NLM_lpage">7916</span>, <span class="refDoi"> DOI: 10.1021/ja964314r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja964314r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=7909-7916&author=R.+P.+L.+de%0AClairacauthor=B.+H.+Geierstangerauthor=M.+Mrksichauthor=P.+B.+Dervanauthor=D.+E.+Wemmer&title=NMR+characterization+of+hairpin+polyamide+complexed+with+the+minor+groove+of+DNA&doi=10.1021%2Fja964314r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja964314r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja964314r%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BClairac%26aufirst%3DR.%2BP.%2BL.%26aulast%3DGeierstanger%26aufirst%3DB.%2BH.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DDervan%26aufirst%3DP.%2BB.%26aulast%3DWemmer%26aufirst%3DD.%2BE.%26atitle%3DNMR%2520characterization%2520of%2520hairpin%2520polyamide%2520complexed%2520with%2520the%2520minor%2520groove%2520of%2520DNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D7909%26epage%3D7916%26doi%3D10.1021%2Fja964314r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span> <span> </span><span class="NLM_article-title">Base pair recognition of the stereochemically alpha-substituted gamma-turn of pyrrole/imidazole hairpin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">14905</span>– <span class="NLM_lpage">14912</span>, <span class="refDoi"> DOI: 10.1021/ja064369l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja064369l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFCjtbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=14905-14912&author=W.+Zhangauthor=M.+Minoshimaauthor=H.+Sugiyama&title=Base+pair+recognition+of+the+stereochemically+alpha-substituted+gamma-turn+of+pyrrole%2Fimidazole+hairpin&doi=10.1021%2Fja064369l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Base Pair Recognition of the Stereochemically α-Substituted γ-Turn of Pyrrole/Imidazole Hairpin Polyamides</span></div><div class="casAuthors">Zhang, Wen; Minoshima, Masafumi; Sugiyama, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">14905-14912</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recognition of the sequences 5'-NGCACA-3' (N = T, A, C, G) by pyrrole/imidazole polyamides with (R/S)-α-hydroxyl/α-amino-substituted γ-aminobutyric acid as a γ-turn was investigated.  Four novel polyamides, 2, 3, 4, and 5, including (R)-α-hydroxyl-γ-aminobutyric acid (γRO), (S)-α-hydroxyl-γ-aminobutyric acid (γSO), (R)-α,γ-diaminobutyric acid (γRN), and (S)-α,γ-diaminobutyric acid (γSN) residues, resp., were synthesized, and their binding affinity to T·A, A·T, G·C, and C·G base pairs at turn position was studied by the surface plasmon resonance (SPR) technique.  SPR data revealed that polyamide 3, AcImβImPy-γSO-ImPyβPy-β-Dp, with a γSO turn, possesses a marked binding preference for T·A over A·T with a 25-fold increase in specificity, despite low binding affinity relative to 2, with a γRO turn.  Similarly, AcImβImPy-γSN-ImPyβPy-β-Dp (5), with a γSN-turn, gives rise to a 8.7-fold increase in specificity for T·A over A·T.  Computer-assisted mol. modeling suggests that 3 binds more deeply in the minor groove of the T·A base pair relative to the A·T base pair, allowing hydrogen bonding to O2 of the thymine at the turn position, which explains the SPR results.  These results suggest that γSO and γSN may function as T-recognition units at the turn position, as well as a γ-turn in the discrimination of polyamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbh8Rp7yzV5bVg90H21EOLACvtfcHk0li6HKVlgUEfOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFCjtbzI&md5=7b6fd576cf04de6292ea760386de245d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja064369l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja064369l%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMinoshima%26aufirst%3DM.%26aulast%3DSugiyama%26aufirst%3DH.%26atitle%3DBase%2520pair%2520recognition%2520of%2520the%2520stereochemically%2520alpha-substituted%2520gamma-turn%2520of%2520pyrrole%252Fimidazole%2520hairpin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D14905%26epage%3D14912%26doi%3D10.1021%2Fja064369l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mormile, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, G.</span></span> <span> </span><span class="NLM_article-title">Endometriosis and susceptibility to cancer: the dark side of IFN-γ gene polymorphism?</span>. <i>J. Cancer Sci. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.4172/1948-5956.1000218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.4172%2F1948-5956.1000218" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=289-290&author=R.+Mormileauthor=G.+Vittori&title=Endometriosis+and+susceptibility+to+cancer%3A+the+dark+side+of+IFN-%CE%B3+gene+polymorphism%3F&doi=10.4172%2F1948-5956.1000218"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.4172%2F1948-5956.1000218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F1948-5956.1000218%26sid%3Dliteratum%253Aachs%26aulast%3DMormile%26aufirst%3DR.%26aulast%3DVittori%26aufirst%3DG.%26atitle%3DEndometriosis%2520and%2520susceptibility%2520to%2520cancer%253A%2520the%2520dark%2520side%2520of%2520IFN-%25CE%25B3%2520gene%2520polymorphism%253F%26jtitle%3DJ.%2520Cancer%2520Sci.%2520Ther.%26date%3D2013%26volume%3D5%26spage%3D289%26epage%3D290%26doi%3D10.4172%2F1948-5956.1000218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quandt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldkamp, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Specker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kribben, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzke, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, B.</span></span> <span> </span><span class="NLM_article-title">Increased percentages of PD-1 on CD4+ T cells is associated with higher INF-γ production and altered IL-17 production in patients with systemic lupus erythematosus</span>. <i>Scand. J. Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.3109/03009742.2013.869830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.3109%2F03009742.2013.869830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25088926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC2cbos1yltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=307-313&author=S.+Dolffauthor=D.+Quandtauthor=T.+Feldkampauthor=C.+Junauthor=A.+Mitchellauthor=F.+Huaauthor=C.+Speckerauthor=A.+Kribbenauthor=O.+Witzkeauthor=B.+Wilde&title=Increased+percentages+of+PD-1+on+CD4%2B+T+cells+is+associated+with+higher+INF-%CE%B3+production+and+altered+IL-17+production+in+patients+with+systemic+lupus+erythematosus&doi=10.3109%2F03009742.2013.869830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Increased percentages of PD-1 on CD4+ T cells is associated with higher INF-γ production and altered IL-17 production in patients with systemic lupus erythematosus</span></div><div class="casAuthors">Dolff S; Quandt D; Feldkamp T; Jun C; Mitchell A; Hua F; Specker C; Kribben A; Witzke O; Wilde B</div><div class="citationInfo"><span class="NLM_cas:title">Scandinavian journal of rheumatology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-13</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Programmed death (PD)-1 is a cell death receptor that, upon stimulation, leads to apoptosis.  Previous studies have shown alteration of PD-1 expression on T cells and PD-1 genes in patients with systemic lupus erythematosus (SLE).  The aim of this study was to assess the expression of this receptor on effector T cells in patients with SLE.  METHOD:  In this study we enrolled 32 SLE patients and 31 healthy controls.  T cells from peripheral blood were analysed by flow cytometry for the expression of PD-1.  Interferon (IFN)-γ and interleukin (IL)-17-producing cells were investigated for the expression of this co-stimulatory marker.  RESULTS:  Percentages of CD4(+) T cells expressing PD-1 were significantly increased in patients with SLE compared to healthy controls.  The percentage of PD-1 expression was correlated with the production of INF-γ (r = 0.83, p < 0.0001).  We also investigated the production of IL-17 by PD-1(+) CD3(+) T cells.  Inactive patients (3.2 ± 1.2% vs. 5.9 ± 3.5%, p = 0.002) and patients without lupus nephritis (LN) (3.2 ± 1.5% vs. 5.9 ± 3.5%, p = 0.005) showed lower levels of IL-17 compared to healthy controls.  CONCLUSION:  We have demonstrated increased expression of PD-1 on CD4(+) T cells in SLE patients and an association between PD-1 expression on CD4(+) T cells and IFN-γ expression on CD3(+) T cells.  We have also shown that there is an altered subset of PD-1(+) T cells in inactive patients and patients without LN producing lower amounts of IL-17.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6qiH7W1cc5OdfST00W0BcfW6udTcc2eZKIQjWtAZaabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbos1yltA%253D%253D&md5=28ba97925c65dec563a7cdcc8890d413</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.3109%2F03009742.2013.869830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03009742.2013.869830%26sid%3Dliteratum%253Aachs%26aulast%3DDolff%26aufirst%3DS.%26aulast%3DQuandt%26aufirst%3DD.%26aulast%3DFeldkamp%26aufirst%3DT.%26aulast%3DJun%26aufirst%3DC.%26aulast%3DMitchell%26aufirst%3DA.%26aulast%3DHua%26aufirst%3DF.%26aulast%3DSpecker%26aufirst%3DC.%26aulast%3DKribben%26aufirst%3DA.%26aulast%3DWitzke%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DB.%26atitle%3DIncreased%2520percentages%2520of%2520PD-1%2520on%2520CD4%252B%2520T%2520cells%2520is%2520associated%2520with%2520higher%2520INF-%25CE%25B3%2520production%2520and%2520altered%2520IL-17%2520production%2520in%2520patients%2520with%2520systemic%2520lupus%2520erythematosus%26jtitle%3DScand.%2520J.%2520Rheumatol.%26date%3D2014%26volume%3D43%26spage%3D307%26epage%3D313%26doi%3D10.3109%2F03009742.2013.869830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Investigation of the Relative Cellular Permeability of DNA-binding Pyrrole-Imidazole Polyamides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4604</span>– <span class="NLM_lpage">4612</span>, <span class="refDoi"> DOI: 10.1021/jm9002999</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9002999" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslCnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4604-4612&author=B.+Liuauthor=T.+Kodadek&title=Investigation+of+the+Relative+Cellular+Permeability+of+DNA-binding+Pyrrole-Imidazole+Polyamides&doi=10.1021%2Fjm9002999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of the Relative Cellular Permeability of DNA-Binding Pyrrole-Imidazole Polyamides</span></div><div class="casAuthors">Liu, Bo; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4604-4612</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pyrrole-imidazole (Py-Im) polyamides are a group of chems. that are able to bind specifically to DNA sequences in vitro and in mammalian cells.  Using a cell based reporter assay, we investigated the size and linker affects on the cellular permeability of polyamides.  We found that the conventional β-alanine-3,3'-diamino-N-methyldipropylamine (βDa) linker strongly limited the cellular permeability.  We discovered that a short ethylene diamine (Et) linker displayed high cellular permeability.  With the improved Et linker, we found that the cellular permeability of polyamides was size-dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbf5cxbAOFWLVg90H21EOLACvtfcHk0lhVCx6FqKx65g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXoslCnur8%253D&md5=c28171e290ee54529aa40d7bfc91569b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm9002999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9002999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DInvestigation%2520of%2520the%2520Relative%2520Cellular%2520Permeability%2520of%2520DNA-binding%2520Pyrrole-Imidazole%2520Polyamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4604%26epage%3D4612%26doi%3D10.1021%2Fjm9002999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alizadeh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niederkorn, J. Y.</span></span> <span> </span><span class="NLM_article-title">PD-L1: PD-1 interaction contributes to the functional suppression of T-cell responses to human uveal melanoma cells in vitro</span>. <i>Invest. Ophthalmol. Visual Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1167/iovs.07-1606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1167%2Fiovs.07-1606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=18296654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BD1czkvVertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2008&pages=2518-2525&author=W.+Yangauthor=P.+W.+Chenauthor=H.+Liauthor=H.+Alizadehauthor=J.+Y.+Niederkorn&title=PD-L1%3A+PD-1+interaction+contributes+to+the+functional+suppression+of+T-cell+responses+to+human+uveal+melanoma+cells+in+vitro&doi=10.1167%2Fiovs.07-1606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1: PD-1 interaction contributes to the functional suppression of T-cell responses to human uveal melanoma cells in vitro</span></div><div class="casAuthors">Yang Wanhua; Chen Peter W; Li Haochuan; Alizadeh Hassan; Niederkorn Jerry Y</div><div class="citationInfo"><span class="NLM_cas:title">Investigative ophthalmology & visual science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2518-25</span>
        ISSN:<span class="NLM_cas:issn">0146-0404</span>.
    </div><div class="casAbstract">PURPOSE:  To assess the expression of PD-L1 on human uveal melanomas and its potential to suppress T-cell function.  METHODS:  A panel of primary and metastatic uveal melanoma cell lines was evaluated for PD-L1 expression by RT-PCR and flow cytometric analysis.  Uveal melanoma-containing eyes were examined for PD-L1 expression by immunohistochemistry.  PD-L1 function was tested by coculturing IFN-gamma-pretreated uveal melanoma cells with activated Jurkat T cells for 48 hours and assessing T-cell production of IL-2 by ELISA.  RESULTS:  Five of the nine primary and one of the five metastatic uveal melanoma cell lines tested constitutively expressed PD-L1 protein at various levels.  However, all primary and metastatic uveal melanoma cell lines upregulated PD-L1 expression after stimulation with IFN-gamma.  Immunohistochemistry demonstrated that PD-L1 was not expressed by primary uveal melanomas in situ.  IL-2 production by activated Jurkat T cells was decreased significantly when the cells were cocultured with IFN-gamma-pretreated uveal melanoma cells.  More than 70% of IL-2 production was restored by addition of either anti-PD-L1 or anti-PD-1 antibody to the coculture assays (P < 0.01).  CONCLUSIONS:  Expression of PD-L1 by uveal melanoma cells regulates T-cell function by suppressing IL-2 production.  The results imply that the presence of IFN-gamma in the tumor local microenvironment promotes upregulation of PD-L1 expression by uveal melanoma, which may, in part, promote immune escape by impairing T-cell function.  The selective blockade of PD-L1 is a potential strategy in T-cell-based immunotherapy for uveal melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSA_-nZI1DHTpPq5LRczm8ffW6udTcc2eZKIQjWtAZaabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czkvVertQ%253D%253D&md5=dc40143aeb4a7a747cd16efc7776cc17</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1167%2Fiovs.07-1606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1167%252Fiovs.07-1606%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DP.%2BW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAlizadeh%26aufirst%3DH.%26aulast%3DNiederkorn%26aufirst%3DJ.%2BY.%26atitle%3DPD-L1%253A%2520PD-1%2520interaction%2520contributes%2520to%2520the%2520functional%2520suppression%2520of%2520T-cell%2520responses%2520to%2520human%2520uveal%2520melanoma%2520cells%2520in%2520vitro%26jtitle%3DInvest.%2520Ophthalmol.%2520Visual%2520Sci.%26date%3D2008%26volume%3D49%26spage%3D2518%26epage%3D2525%26doi%3D10.1167%2Fiovs.07-1606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, S.-Y</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ro, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ro, J.</span></span> <span> </span><span class="NLM_article-title">Prognostic implications of tumor-infiltrating lymphocytes in association with programmed death ligand 1 expression in early-stage breast cancer</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2015.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.clbc.2015.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26364145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSit7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=51-58&author=I.+H.+Parkauthor=S.-Y+Kongauthor=J.+Y.+Roauthor=Y.+Kwonauthor=J.+H.+Kangauthor=H.+J.+Moauthor=S.-Y.+Jungauthor=S.+Leeauthor=K.+S.+Leeauthor=H.-S.+Kangauthor=E.+Leeauthor=J.+Jooauthor=J.+Ro&title=Prognostic+implications+of+tumor-infiltrating+lymphocytes+in+association+with+programmed+death+ligand+1+expression+in+early-stage+breast+cancer&doi=10.1016%2Fj.clbc.2015.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic implications of tumor-infiltrating lymphocytes in association with programmed death ligand 1 expression in early-stage breast cancer</span></div><div class="casAuthors">Park, In Hae; Kong, Sun-Young; Ro, Jae Yoon; Kwon, Youngmee; Kang, Joo Hyun; Mo, Hye Jin; Jung, So-Youn; Lee, Seeyoun; Lee, Keun Seok; Kang, Han-Sung; Lee, Eunsook; Joo, Jungnam; Ro, Jungsil</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-58</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The immune system might influence breast cancer (BC) prognosis.  However, the relationship between programmed death ligand 1 (PD-L1) and tumor-infiltrating lymphocyte (TIL) profiles remains unclear with respect to BC subtypes.We investigated the relationship between TIL profiles for CD8+ and forkhead box P3-pos. (FOXP3+) and PD-L1 expression in primary tumor tissue using immunohistochem. and the clin. outcomes in 2 patient cohorts at the National Cancer Center: 256 patients diagnosed with early-stage BC from Jan. 2001 to Dec. 2005 and 77 hormone receptor (HR)-neg. BC patients diagnosed from Jan. 2006 to Dec. 2008.  Clin. data were collected, including HR status, human epidermal growth factor receptor 2 expression, disease-free survival, and overall survival (OS).The median patient age was 47 years (range, 28-78), and the median follow-up period was 9.8 years.  Of the 333 patients, 186 (55.9%) had HR-pos. and 125 (37.5%) had node-pos. BC.  We found a strong pos. correlation between CD8+ TILs and FOXP3+ TILs (P < .001).  CD8+ TILs were more abundant in tumors with low PD-L1 expression (P < .001), although no assocn. was found between FOXP3+ TILs and PD-L1 expression.  More CD8+ TILs were present in HR-neg. than in HR-pos. BC (P < .001), and PD-L1 expression was more frequent in HR-pos. BC (P < .001).  A greater no. of CD8+ TILs (increase in quartile) was strongly assocd. with OS (hazard ratio, 0.61; 95% confidence interval, 0.39-0.95; P = .03) only in HR-neg. BC when adjusted for various clin. factors.  PD-L1 expression and FOXP3+ TILs did not exhibit such assocns. Higher CD8+ lymphocyte infiltration was related to lower PD-L1 expression and higher FOXP3+ TIL infiltration in BC.  Higher CD8+ TIL expression was assocd. with prolonged survival only in those with HR-neg. BC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcTcRpS0nGPLVg90H21EOLACvtfcHk0lhfyUZWAGbdQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSit7zF&md5=c524199fd1997df48b4d71d66851b7fd</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2015.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2015.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DI.%2BH.%26aulast%3DKong%26aufirst%3DS.-Y%26aulast%3DRo%26aufirst%3DJ.%2BY.%26aulast%3DKwon%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DJ.%2BH.%26aulast%3DMo%26aufirst%3DH.%2BJ.%26aulast%3DJung%26aufirst%3DS.-Y.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BS.%26aulast%3DKang%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DE.%26aulast%3DJoo%26aufirst%3DJ.%26aulast%3DRo%26aufirst%3DJ.%26atitle%3DPrognostic%2520implications%2520of%2520tumor-infiltrating%2520lymphocytes%2520in%2520association%2520with%2520programmed%2520death%2520ligand%25201%2520expression%2520in%2520early-stage%2520breast%2520cancer%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D51%26epage%3D58%26doi%3D10.1016%2Fj.clbc.2015.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span> <span> </span><span class="NLM_article-title">Adaptive immune resistance: how cancer protects from immune attack</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-0563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F2159-8290.CD-15-0563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26272491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVOqtrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=915-919&author=A.+Ribas&title=Adaptive+immune+resistance%3A+how+cancer+protects+from+immune+attack&doi=10.1158%2F2159-8290.CD-15-0563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Adaptive Immune Resistance: How Cancer Protects from Immune Attack</span></div><div class="casAuthors">Ribas, Antoni</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">915-919</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Adaptive immune resistance is a process in which the cancer changes its phenotype in response to a cytotoxic or proinflammatory immune response, thereby evading it.  This adaptive process is triggered by the specific recognition of cancer cells by T cells, which leads to the prodn. of immune-activating cytokines.  Cancers then hijack mechanisms developed to limit inflammatory and immune responses and protect themselves from the T-cell attack.  Inhibiting adaptive immune resistance is the mechanistic basis of responses to PD-1 or PD-L1-blocking antibodies, and may be of relevance for the development of other cancer immunotherapy strategies.  Significance: Several new immunotherapy strategies to treat cancer are based on inhibiting processes through which cancer adapts and evades from an immune response.  Recognizing the specific adaptive resistance mechanisms in each case is likely to allow the personalized development of immunotherapies tailored to block how a particular cancer protects itself from the immune system.  Cancer Discov; 5(9); 915-9. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWNhTGL5MdgbVg90H21EOLACvtfcHk0lhfyUZWAGbdQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOqtrnL&md5=ba45d5b4e78b9a3fc0689756c0dffb62</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-0563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-0563%26sid%3Dliteratum%253Aachs%26aulast%3DRibas%26aufirst%3DA.%26atitle%3DAdaptive%2520immune%2520resistance%253A%2520how%2520cancer%2520protects%2520from%2520immune%2520attack%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D915%26epage%3D919%26doi%3D10.1158%2F2159-8290.CD-15-0563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coombs, M. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoskin, D. W.</span></span> <span> </span><span class="NLM_article-title">Apigenin inhibits the inducible expression of programmed death ligand 1 by human and mouse mammary carcinoma cells</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>380</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2016.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.canlet.2016.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27378243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFant73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=2016&pages=424-433&author=M.+R.+P.+Coombsauthor=M.+E.+Harrisonauthor=D.+W.+Hoskin&title=Apigenin+inhibits+the+inducible+expression+of+programmed+death+ligand+1+by+human+and+mouse+mammary+carcinoma+cells&doi=10.1016%2Fj.canlet.2016.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Apigenin inhibits the inducible expression of programmed death ligand 1 by human and mouse mammary carcinoma cells</span></div><div class="casAuthors">Coombs, Melanie R. Power; Harrison, Megan E.; Hoskin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">380</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">424-433</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Programmed death ligand 1 (PD-L1) is expressed by many cancer cell types, as well as by activated T cells and antigen-presenting cells.  Constitutive and inducible PD-L1 expression contributes to immune evasion by breast cancer (BC) cells.  We show here that the dietary phytochem. apigenin inhibited interferon (IFN)-γ-induced PD-L1 upregulation by triple-neg. MDA-MB-468 BC cells, HER2+ SK-BR-3 BC cells, and 4T1 mouse mammary carcinoma cells, as well as human mammary epithelial cells, but did not affect constitutive PD-L1 expression by triple-neg. MDA-MB-231 BC cells.  IFN-β-induced expression of PD-L1 by MDA-MB-468 cells was also inhibited by apigenin.  In addn., luteolin, the major metabolite of apigenin, inhibited IFN-γ-induced PD-L1 expression by MDA-MB-468 cells.  Apigenin-mediated inhibition of IFN-γ-induced PD-L1 expression by MDA-MB-468 and 4T1 cells was assocd. with reduced phosphorylation of STAT1, which was early and transient at Tyr701 and sustained at Ser727.  Apigenin-mediated inhibition of IFN-γ-induced PD-L1 expression by MDA-MB-468 cells also increased proliferation and interleukin-2 synthesis by PD-1-expressing Jurkat T cells that were co-cultured with MDA-MB-468 cells.  Apigenin therefore has the potential to increase the vulnerability of BC cells to T cell-mediated anti-tumor immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVpAiyZIgsLrVg90H21EOLACvtfcHk0lhfyUZWAGbdQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFant73N&md5=4df618d6f5fa91c5dbd1dedd57b639d7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DCoombs%26aufirst%3DM.%2BR.%2BP.%26aulast%3DHarrison%26aufirst%3DM.%2BE.%26aulast%3DHoskin%26aufirst%3DD.%2BW.%26atitle%3DApigenin%2520inhibits%2520the%2520inducible%2520expression%2520of%2520programmed%2520death%2520ligand%25201%2520by%2520human%2520and%2520mouse%2520mammary%2520carcinoma%2520cells%26jtitle%3DCancer%2520Lett.%26date%3D2016%26volume%3D380%26spage%3D424%26epage%3D433%26doi%3D10.1016%2Fj.canlet.2016.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span> <span> </span><span class="NLM_article-title">Tumor cell-derived lactate induces TAZ-dependent upregulation of PD-L1 through GPR81 in human lung cancer cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">5829</span>– <span class="NLM_lpage">5839</span>, <span class="refDoi"> DOI: 10.1038/onc.2017.188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fonc.2017.188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28604752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslWgtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=5829-5839&author=J.+Fengauthor=H.+Yangauthor=Y.+Zhangauthor=H.+Weiauthor=Z.+Zhuauthor=B.+Zhuauthor=M.+Yangauthor=W.+Caoauthor=L.+Wangauthor=Z.+Wu&title=Tumor+cell-derived+lactate+induces+TAZ-dependent+upregulation+of+PD-L1+through+GPR81+in+human+lung+cancer+cells&doi=10.1038%2Fonc.2017.188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor cell-derived lactate induces TAZ-dependent upregulation of PD-L1 through GPR81 in human lung cancer cells</span></div><div class="casAuthors">Feng, J.; Yang, H.; Zhang, Y.; Wei, H.; Zhu, Z.; Zhu, B.; Yang, M.; Cao, W.; Wang, L.; Wu, Z.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">5829-5839</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The clin. success of immunotherapy that inhibits the neg. immune regulatory pathway programmed cell death protein 1/PD-1 ligand (PD-1/PD-L1) has initiated a new era in the treatment of metastatic cancer.  PD-L1 expression is upregulated in many solid tumors including lung cancer and functions predominantly in lactate-enriched tumor microenvironments.  Here, we provided evidence for PD-L1 induction in response to lactate stimulation in lung cancer cells.  Lactate-induced PD-L1 induction was mediated by its receptor GPR81.  The silencing of GPR81 signaling in lung cancer cells resulted in a decrease in PD-L1 protein levels and functional inactivation of PD-L1 promoter activity.  In addn., GPR81-mediated upregulation of PD-L1 in glucose-stimulated lung cancer cells that recapitulates the enhanced glycolysis in vivo was dependent on lactate dehydrogenase A (LDHA).  We also demonstrated that activation of GPR81 decreases intracellular cAMP levels and inhibits protein kinase A (PKA) activity, leading to activation of the transcriptional coactivator TAZ.  Interaction of TAZ with the transcription factor TEAD was essential for TAZ activation of PD-L1 and induction of its expression.  Furthermore, we found that lactate-induced activation of PD-L1 in tumor cells led to reduced prodn. of interferon-γ and induction of apoptosis of cocultured Jurkat T-cell leukemia cells.  Our findings reveal an unexpected role of lactate in contributing to tumor cell protection from cytotoxic T-cell targeting and establishes a direct connection between tumor cell metabolic reprograming and tumor evasion from the immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKwzZT3o-gKrVg90H21EOLACvtfcHk0lhV02rMWEdnxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslWgtbk%253D&md5=e37888d8162b315bf5ad13d14e1122b7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fonc.2017.188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2017.188%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%26atitle%3DTumor%2520cell-derived%2520lactate%2520induces%2520TAZ-dependent%2520upregulation%2520of%2520PD-L1%2520through%2520GPR81%2520in%2520human%2520lung%2520cancer%2520cells%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D5829%26epage%3D5839%26doi%3D10.1038%2Fonc.2017.188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Quercetin nanoparticles induced autophagy and apoptosis through AKT/ERK/Caspase-3 signaling pathway in human neuroglioma cells: In vitro and in vivo</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2016.08.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.biopha.2016.08.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27621033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2rt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2016&pages=1-9&author=M.+Louauthor=L.+Zhangauthor=P.+Jiauthor=F.+Fengauthor=J.+Liuauthor=C.+Yangauthor=B.+Liauthor=L.+Wang&title=Quercetin+nanoparticles+induced+autophagy+and+apoptosis+through+AKT%2FERK%2FCaspase-3+signaling+pathway+in+human+neuroglioma+cells%3A+In+vitro+and+in+vivo&doi=10.1016%2Fj.biopha.2016.08.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Quercetin nanoparticles induced autophagy and apoptosis through AKT/ERK/Caspase-3 signaling pathway in human neuroglioma cells: In vitro and in vivo</span></div><div class="casAuthors">Lou, Miao; Zhang, Li-na; Ji, Pei-gang; Feng, Fu-qiang; Liu, Jing-hui; Yang, Chen; Li, Bao-fu; Wang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Neuroglioma is a complex neuroglial tumor involving dysregulation of many biol. pathways at multiple levels.  Quercetin is a potent cancer therapeutic agent presented in fruit and vegetables, preventing tumor proliferation, and is a well known cancer therapeutic agent and autophagy mediator.  Recent studies showed that drug delivery by nanoparticles have enhanced efficacy with reduced side effects.  In this regard, gold-quercetin into poly (DL-lactide-co-glycolide) nanoparticles was examd.  In the present study, quercetin nanoparticle induced cell autophagy and apoptosis in human neuroglioma cell was investigated.  Quercetin nanoparticle administrated to animals displayed suppressed role in tumor growth.  The cell viability was detd. through CCK8 assay.  Transmission electron microscopy was utilized to observe the formation of autophagosome.  The cell apoptosis was assessed by annexin V-PI staining.  The protein expression of cell autophagy regulators and tumor suppressors were analyzed via western blot and RT-PCR.  Treatment of human neuroglioma cell with quercetin nanoparticle induced cell death in a dose-and time-dependent manner.  The flow cytometry results showed that the proportion of the apoptosis cells had gained after quercetin nanoparticle treatment compared to untreatment group.  Moreover, the expression of activated PI3K/AKT and Bcl-2 were down-regulated upon quercetin nanoparticle treatment in human neuroglioma cells.  The expression level of LC3 and ERK as well as cytoplasm p53, cleaved Caspase-3 and PARP was pos. correlated with the concn. of quercetin nanoparticle.  In addn., p-mTOR and GAIP were obviously down-regulated by quercetin nanoparticle treatment in a dose-dependent manner.  These results indicated that quercetin nanoparticle could induce autophagy and apoptosis in human neuroglioma cells, the underlying mol. mechanisms, at least partly, through activation LC3/ERK/Caspase-3 and suppression AKT/mTOR signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDVZ0FbEamuLVg90H21EOLACvtfcHk0lhV02rMWEdnxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2rt7fK&md5=2695d98cbbe32b7ad4abbbaf6d916917</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2016.08.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2016.08.055%26sid%3Dliteratum%253Aachs%26aulast%3DLou%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DP.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DQuercetin%2520nanoparticles%2520induced%2520autophagy%2520and%2520apoptosis%2520through%2520AKT%252FERK%252FCaspase-3%2520signaling%2520pathway%2520in%2520human%2520neuroglioma%2520cells%253A%2520In%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2016%26volume%3D84%26spage%3D1%26epage%3D9%26doi%3D10.1016%2Fj.biopha.2016.08.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saresella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rainone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Daghri, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clerici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabattoni, D.</span></span> <span> </span><span class="NLM_article-title">The PD-1/PD-L1 pathway in human pathology</span>. <i>Curr. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.2174/156652412799218903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.2174%2F156652412799218903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22300137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFWisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=259-267&author=M.+Saresellaauthor=V.+Rainoneauthor=N.+M.+Al-Daghriauthor=M.+Clericiauthor=D.+Trabattoni&title=The+PD-1%2FPD-L1+pathway+in+human+pathology&doi=10.2174%2F156652412799218903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The PD-1/PD-L1 pathway in human pathology</span></div><div class="casAuthors">Saresella, M.; Rainone, V.; Al-Daghri, N. M.; Clerici, M.; Trabattoni, D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Molecular Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">259-267</span>CODEN:
                <span class="NLM_cas:coden">CMMUBP</span>;
        ISSN:<span class="NLM_cas:issn">1566-5240</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  T-cell activation is dependent on signals delivered through the antigen-specific T-cell receptor and accessory receptors on T-cells.  Integration of signals through this family of costimulatory and inhibitory receptors and their ligands regulates the balance between T-cell activation, tolerance, and immunopathol.  Programmed death 1 (PD-1) and its ligands, PD-L1 and PD-L2, deliver inhibitory signals and exert a vital and diverse range of immunoregulatory roles in T-cell activation, tolerance, and immune-mediated tissue damage.  In this review, we revisit current understanding of the immunoregulatory functions of PD-1 and its ligands and their involvement in immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhq3Juj_4LM7Vg90H21EOLACvtfcHk0lhV02rMWEdnxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFWisr4%253D&md5=2407cedf7cb6bfe52bef42cf2e69169c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.2174%2F156652412799218903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156652412799218903%26sid%3Dliteratum%253Aachs%26aulast%3DSaresella%26aufirst%3DM.%26aulast%3DRainone%26aufirst%3DV.%26aulast%3DAl-Daghri%26aufirst%3DN.%2BM.%26aulast%3DClerici%26aufirst%3DM.%26aulast%3DTrabattoni%26aufirst%3DD.%26atitle%3DThe%2520PD-1%252FPD-L1%2520pathway%2520in%2520human%2520pathology%26jtitle%3DCurr.%2520Mol.%2520Med.%26date%3D2012%26volume%3D12%26spage%3D259%26epage%3D267%26doi%3D10.2174%2F156652412799218903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsuliman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harazi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitwi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulbah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Tweigeri, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Alwan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghebeh, H.</span></span> <span> </span><span class="NLM_article-title">Bidirectional crosstalk between PD-L1 expression and epithelial to mesenchymal transition: significance in claudin-low breast cancer cells</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1186/s12943-015-0421-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1186%2Fs12943-015-0421-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26245467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC287gtFKkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=149-161&author=A.+Alsulimanauthor=D.+Colakauthor=O.+Al-Haraziauthor=H.+Fitwiauthor=A.+Tulbahauthor=T.+Al-Tweigeriauthor=M.+Al-Alwanauthor=H.+Ghebeh&title=Bidirectional+crosstalk+between+PD-L1+expression+and+epithelial+to+mesenchymal+transition%3A+significance+in+claudin-low+breast+cancer+cells&doi=10.1186%2Fs12943-015-0421-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Bidirectional crosstalk between PD-L1 expression and epithelial to mesenchymal transition: significance in claudin-low breast cancer cells</span></div><div class="casAuthors">Alsuliman Abdullah; Fitwi Hanaa; Al-Alwan Monther; Ghebeh Hazem; Colak Dilek; Al-Harazi Olfat; Tulbah Asma; Al-Tweigeri Taher; Al-Alwan Monther; Ghebeh Hazem</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">149</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The T-cell inhibitory molecule PD-L1 (B7-H1, CD274) is expressed on tumor cells of a subset of breast cancer patients.  However, the mechanism that regulates PD-L1 expression in this group of patients is still not well-identified.  METHODS:  We have used loss and gain of function gene manipulation approach, multi-parametric flow cytometry, large scale gene expression dataset analysis and immunohistochemistry of breast cancer tissue sections.  RESULTS:  Induction of epithelial to mesenchymal transition (EMT) in human mammary epithelial cells upregulated PD-L1 expression, which was dependent mainly on the activation of the PI3K/AKT pathway.  Interestingly, gene expression signatures available from large cohort of breast tumors showed a significant correlation between EMT score and the PD-L1 mRNA level (p < 0.001).  Strikingly, very strong association (p < 0.0001) was found between PD-L1 expression and claudin-low subset of breast cancer, which is known to have high EMT score.  On the protein level, significant correlation was found between PD-L1 expression and standard markers of EMT (p = 0.005) in 67 breast cancer patients.  Importantly, specific downregulation of PD-L1 in claudin-low breast cancer cells showed signs of EMT reversal as manifested by CD44 and Vimentin downregulation and CD24 upregulation.  CONCLUSIONS:  We have demonstrated a bidirectional effect between EMT status and PD-L1 expression especially in claudin-low subtype of breast cancer cells.  Our findings highlights the potential dual benefit of anti-PD-L1 particularly in this subset of breast cancer patients that will likely benefit more from anti-PD-L1 targeted therapy as well as in monitoring biological changes upon treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcROs3-ADOt9csuCWE5mhx1RfW6udTcc2eb7nbvWyP0hZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287gtFKkuw%253D%253D&md5=e12a6f1cea0ef041f4eaff4968b49a6c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs12943-015-0421-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-015-0421-2%26sid%3Dliteratum%253Aachs%26aulast%3DAlsuliman%26aufirst%3DA.%26aulast%3DColak%26aufirst%3DD.%26aulast%3DAl-Harazi%26aufirst%3DO.%26aulast%3DFitwi%26aufirst%3DH.%26aulast%3DTulbah%26aufirst%3DA.%26aulast%3DAl-Tweigeri%26aufirst%3DT.%26aulast%3DAl-Alwan%26aufirst%3DM.%26aulast%3DGhebeh%26aufirst%3DH.%26atitle%3DBidirectional%2520crosstalk%2520between%2520PD-L1%2520expression%2520and%2520epithelial%2520to%2520mesenchymal%2520transition%253A%2520significance%2520in%2520claudin-low%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Cancer%26date%3D2015%26volume%3D14%26spage%3D149%26epage%3D161%26doi%3D10.1186%2Fs12943-015-0421-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourque, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chernova, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitz, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malenkovich, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okazaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, M. C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Engagement of the PD-1 immunoinhibitory receptor by a novel B7 family member leads to negative regulation of lymphocyte activation</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">1027</span>– <span class="NLM_lpage">1034</span>, <span class="refDoi"> DOI: 10.1084/jem.192.7.1027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1084%2Fjem.192.7.1027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=11015443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntFSlt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2000&pages=1027-1034&author=G.+J.+Freemanauthor=A.+J.+Longauthor=Y.+Iwaiauthor=K.+Bourqueauthor=T.+Chernovaauthor=H.+Nishimuraauthor=L.+J.+Fitzauthor=N.+Malenkovichauthor=T.+Okazakiauthor=M.+C.+Byrne&title=Engagement+of+the+PD-1+immunoinhibitory+receptor+by+a+novel+B7+family+member+leads+to+negative+regulation+of+lymphocyte+activation&doi=10.1084%2Fjem.192.7.1027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Engagement of the PD-1 immunoinhibitory receptor by a novel B7 family member leads to negative regulation of lymphocyte activation</span></div><div class="casAuthors">Freeman, Gordon J.; Long, Andrew J.; Iwai, Yoshiko; Bourque, Karen; Chernova, Tatyana; Nishimura, Hiroyuki; Fitz, Lori J.; Malenkovich, Nelly; Okazaki, Taku; Byrne, Michael C.; Horton, Heidi F.; Fouser, Lynette; Carter, Laura; Ling, Vincent; Bowman, Michael R.; Carreno, Beatriz M.; Collins, Mary; Wood, Clive R.; Honjo, Tasuku</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1027-1034</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">PD-1 is an immunoinhibitory receptor expressed by activated T cells, B cells, and myeloid cells.  Mice deficient in PD-1 exhibit a breakdown of peripheral tolerance and demonstrate multiple autoimmune features.  The authors report here that the ligand of PD-1 (PD-L1) is a member of the B7 gene family.  Engagement of PD-1 by PD-L1 leads to the inhibition of T cell receptor-mediated lymphocyte proliferation and cytokine secretion.  In addn., PD-1 signaling can inhibit at least suboptimal levels of CD28-mediated costimulation.  PD-L1 is expressed by antigen-presenting cells, including human peripheral blood monocytes stimulated with interferon γ, and activated human and murine dendritic cells.  In addn., PD-L1 is expressed in nonlymphoid tissues such as heart and lung.  The relative levels of inhibitory PD-L1 and costimulatory B7-1/B7-2 signals on antigen-presenting cells may det. the extent of T cell activation and consequently the threshold between tolerance and autoimmunity.  PD-L1 expression on nonlymphoid tissues and its potential interaction with PD-1 may subsequently det. the extent of immune responses at sites of inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZIeEjiecqBbVg90H21EOLACvtfcHk0lgv1Js4P22reA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntFSlt7o%253D&md5=bc1499033282cd9dbfa396a4e2697f6e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1084%2Fjem.192.7.1027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.192.7.1027%26sid%3Dliteratum%253Aachs%26aulast%3DFreeman%26aufirst%3DG.%2BJ.%26aulast%3DLong%26aufirst%3DA.%2BJ.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DBourque%26aufirst%3DK.%26aulast%3DChernova%26aufirst%3DT.%26aulast%3DNishimura%26aufirst%3DH.%26aulast%3DFitz%26aufirst%3DL.%2BJ.%26aulast%3DMalenkovich%26aufirst%3DN.%26aulast%3DOkazaki%26aufirst%3DT.%26aulast%3DByrne%26aufirst%3DM.%2BC.%26atitle%3DEngagement%2520of%2520the%2520PD-1%2520immunoinhibitory%2520receptor%2520by%2520a%2520novel%2520B7%2520family%2520member%2520leads%2520to%2520negative%2520regulation%2520of%2520lymphocyte%2520activation%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2000%26volume%3D192%26spage%3D1027%26epage%3D1034%26doi%3D10.1084%2Fjem.192.7.1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoo, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.-C.</span></span> <span> </span><span class="NLM_article-title">Glycosylation and stabilization of programmed death ligand-1 suppresses T-cell activity</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">12632</span>, <span class="refDoi"> DOI: 10.1038/ncomms12632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fncomms12632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=27572267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=12632&author=C.-W.+Liauthor=S.-O.+Limauthor=W.+Xiaauthor=H.-H.+Leeauthor=L.-C.+Chanauthor=C.-W.+Kuoauthor=K.-H.+Khooauthor=S.-S.+Changauthor=J.-H.+Chaauthor=T.+Kimauthor=J.+L.+Hsuauthor=Y.+Wuauthor=J.-M.+Hsuauthor=H.+Yamaguchiauthor=Q.+Dingauthor=Y.+Wangauthor=J.+Yaoauthor=C.-C.+Leeauthor=H.-J.+Wuauthor=A.+A.+Sahinauthor=J.+P.+Allisonauthor=D.+Yuauthor=G.+N.+Hortobagyiauthor=M.-C.+Hung&title=Glycosylation+and+stabilization+of+programmed+death+ligand-1+suppresses+T-cell+activity&doi=10.1038%2Fncomms12632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glycosylation and stabilization of programmed death ligand-1 suppresses T-cell activity</span></div><div class="casAuthors">Li, Chia-Wei; Lim, Seung-Oe; Xia, Weiya; Lee, Heng-Huan; Chan, Li-Chuan; Kuo, Chu-Wei; Khoo, Kay-Hooi; Chang, Shih-Shin; Cha, Jong-Ho; Kim, Taewan; Hsu, Jennifer L.; Wu, Yun; Hsu, Jung-Mao; Yamaguchi, Hirohito; Ding, Qingqing; Wang, Yan; Yao, Jun; Lee, Cheng-Chung; Wu, Hsing-Ju; Sahin, Aysegul A.; Allison, James P.; Yu, Dihua; Hortobagyi, Gabriel N.; Hung, Mien-Chie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12632</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Extracellular interaction between programmed death ligand-1 (PD-L1) and programmed cell death protein-1 (PD-1) leads to tumor-assocd. immune escape.  Here we show that the immunosuppression activity of PD-L1 is stringently modulated by ubiquitination and N-glycosylation.  We show that glycogen synthase kinase 3β (GSK3β) interacts with PD-L1 and induces phosphorylation-dependent proteasome degrdn. of PD-L1 by β-TrCP.  In-depth anal. of PD-L1 N192, N200 and N219 glycosylation suggests that glycosylation antagonizes GSK3β binding.  In this regard, only non-glycosylated PD-L1 forms a complex with GSK3β and β-TrCP.  We also demonstrate that epidermal growth factor (EGF) stabilizes PD-L1 via GSK3β inactivation in basal-like breast cancer.  Inhibition of EGF signalling by gefitinib destabilizes PD-L1, enhances antitumor T-cell immunity and therapeutic efficacy of PD-1 blockade in syngeneic mouse models.  Together, our results link ubiquitination and glycosylation pathways to the stringent regulation of PD-L1, which could lead to potential therapeutic strategies to enhance cancer immune therapy efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQnyTgbFXcD7Vg90H21EOLACvtfcHk0lj9IdBlC3l6HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbnJ&md5=35cab8a1af28193769fe685c0e1f37b7</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fncomms12632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12632%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.-W.%26aulast%3DLim%26aufirst%3DS.-O.%26aulast%3DXia%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DH.-H.%26aulast%3DChan%26aufirst%3DL.-C.%26aulast%3DKuo%26aufirst%3DC.-W.%26aulast%3DKhoo%26aufirst%3DK.-H.%26aulast%3DChang%26aufirst%3DS.-S.%26aulast%3DCha%26aufirst%3DJ.-H.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DHsu%26aufirst%3DJ.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DHsu%26aufirst%3DJ.-M.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DC.-C.%26aulast%3DWu%26aufirst%3DH.-J.%26aulast%3DSahin%26aufirst%3DA.%2BA.%26aulast%3DAllison%26aufirst%3DJ.%2BP.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DHung%26aufirst%3DM.-C.%26atitle%3DGlycosylation%2520and%2520stabilization%2520of%2520programmed%2520death%2520ligand-1%2520suppresses%2520T-cell%2520activity%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D12632%26doi%3D10.1038%2Fncomms12632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ihara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashiwa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukamizu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohmo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagatomo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kijima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tachibana, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumanogoh, A.</span></span> <span> </span><span class="NLM_article-title">Inhibitory roles of signal transducer and activator of transcription 3 in antitumor immunity during carcinogen-induced lung tumorigenesis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2990</span>– <span class="NLM_lpage">2999</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-4062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F0008-5472.CAN-11-4062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=22659452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1yjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2990-2999&author=S.+Iharaauthor=H.+Kidaauthor=H.+Araseauthor=L.+P.+Tripathiauthor=Y.+Chenauthor=T.+Kimuraauthor=M.+Yoshidaauthor=Y.+Kashiwaauthor=H.+Hirataauthor=R.+Fukamizuauthor=R.+Inoueauthor=K.+Hasegawaauthor=S.+Goyaauthor=R.+Takahashiauthor=T.+Minamiauthor=K.+Tsujinoauthor=M.+Suzukiauthor=S.+Kohmoauthor=K.+Inoueauthor=I.+Nagatomoauthor=Y.+Takedaauthor=T.+Kijimaauthor=K.+Mizuguchiauthor=I.+Tachibanaauthor=A.+Kumanogoh&title=Inhibitory+roles+of+signal+transducer+and+activator+of+transcription+3+in+antitumor+immunity+during+carcinogen-induced+lung+tumorigenesis&doi=10.1158%2F0008-5472.CAN-11-4062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitory Roles of Signal Transducer and Activator of Transcription 3 in Antitumor Immunity during Carcinogen-Induced Lung Tumorigenesis</span></div><div class="casAuthors">Ihara, Shoichi; Kida, Hiroshi; Arase, Hisashi; Tripathi, Lokesh P.; Chen, Yi-An; Kimura, Tetsuya; Yoshida, Mitsuhiro; Kashiwa, Yozo; Hirata, Haruhiko; Fukamizu, Reiko; Inoue, Ruriko; Hasegawa, Kana; Goya, Sho; Takahashi, Ryo; Minami, Toshiyuki; Tsujino, Kazuyuki; Suzuki, Mayumi; Kohmo, Satoshi; Inoue, Koji; Nagatomo, Izumi; Takeda, Yoshito; Kijima, Takashi; Mizuguchi, Kenji; Tachibana, Isao; Kumanogoh, Atsushi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2990-2999</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Stat3 mediates a complex spectrum of cellular responses, including inflammation, cell proliferation, and apoptosis.  Although evidence exists in support of a pos. role for Stat3 in cancer, its role has remained somewhat controversial because of insufficient study of how its genetic deletion may affect carcinogenesis in various tissues.  In this study, we show using epithelium-specific knockout mice (Stat3Δ/Δ) that Stat3 blunts rather than supports antitumor immunity in carcinogen-induced lung tumorigenesis.  Although Stat3A/A mice did not show any lung defects in terms of proliferation, apoptosis, or angiogenesis, they exhibited reduced urethane-induced tumorigenesis and increased antitumor inflammation and natural killer (NK) cell immunity.  Comparative microarray anal. revealed an increase in Stat3Δ/Δ tumors in proinflammatory chemokine prodn. and a decrease in MHC class I antigen expression assocd. with NK cell recognition.  Consistent with these findings, human non-small cell lung cancer (NSCLC) cells in which Stat3 was silenced displayed an enhancement of proinflammatory chemokine prodn., reduced expression of MHC class I antigen, and increased susceptibility to NKcell-mediated cytotoxicity.  In addn., supernatants from Stat3-silenced NSCLC cells promoted monocyte migration.  Collectively, our findings argue that Stat3 exertsan inhibitory effect on antitumor NK cell immunity in the setting of carcinogen-induced tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK9a9e8642gLVg90H21EOLACvtfcHk0ljJK4YvNAqAJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1yjurc%253D&md5=6376b221b272dac5b4907a0b52bc9e45</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-4062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-4062%26sid%3Dliteratum%253Aachs%26aulast%3DIhara%26aufirst%3DS.%26aulast%3DKida%26aufirst%3DH.%26aulast%3DArase%26aufirst%3DH.%26aulast%3DTripathi%26aufirst%3DL.%2BP.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DKashiwa%26aufirst%3DY.%26aulast%3DHirata%26aufirst%3DH.%26aulast%3DFukamizu%26aufirst%3DR.%26aulast%3DInoue%26aufirst%3DR.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DGoya%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DR.%26aulast%3DMinami%26aufirst%3DT.%26aulast%3DTsujino%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DKohmo%26aufirst%3DS.%26aulast%3DInoue%26aufirst%3DK.%26aulast%3DNagatomo%26aufirst%3DI.%26aulast%3DTakeda%26aufirst%3DY.%26aulast%3DKijima%26aufirst%3DT.%26aulast%3DMizuguchi%26aufirst%3DK.%26aulast%3DTachibana%26aufirst%3DI.%26aulast%3DKumanogoh%26aufirst%3DA.%26atitle%3DInhibitory%2520roles%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520in%2520antitumor%2520immunity%2520during%2520carcinogen-induced%2520lung%2520tumorigenesis%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2990%26epage%3D2999%26doi%3D10.1158%2F0008-5472.CAN-11-4062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Old, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, M. J.</span></span> <span> </span><span class="NLM_article-title">Cancer immunoediting: integrating immunity’s roles in cancer suppression and promotion</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>331</i></span>,  <span class="NLM_fpage">1565</span>– <span class="NLM_lpage">1570</span>, <span class="refDoi"> DOI: 10.1126/science.1203486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1126%2Fscience.1203486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21436444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=331&publication_year=2011&pages=1565-1570&author=R.+D.+Schreiberauthor=L.+J.+Oldauthor=M.+J.+Smyth&title=Cancer+immunoediting%3A+integrating+immunity%E2%80%99s+roles+in+cancer+suppression+and+promotion&doi=10.1126%2Fscience.1203486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Immunoediting: Integrating Immunity's Roles in Cancer Suppression and Promotion</span></div><div class="casAuthors">Schreiber, Robert D.; Old, Lloyd J.; Smyth, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">331</span>
        (<span class="NLM_cas:issue">6024</span>),
    <span class="NLM_cas:pages">1565-1570</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Understanding how the immune system affects cancer development and progression has been one of the most challenging questions in immunol.  Research over the past two decades has helped explain why the answer to this question has evaded the authors for so long.  The authors now appreciate that the immune system plays a dual role in cancer: It can not only suppress tumor growth by destroying cancer cells or inhibiting their outgrowth but also promote tumor progression either by selecting for tumor cells that are more fit to survive in an immunocompetent host or by establishing conditions within the tumor microenvironment that facilitate tumor outgrowth.  Here, the authors discuss a unifying conceptual framework called "cancer immunoediting," which integrates the immune system's dual host-protective and tumor-promoting roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpokt_z4WtBy7Vg90H21EOLACvtfcHk0ljJK4YvNAqAJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbs%253D&md5=cebc33b67a44d78fa6e75af050114d4d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1126%2Fscience.1203486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1203486%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DR.%2BD.%26aulast%3DOld%26aufirst%3DL.%2BJ.%26aulast%3DSmyth%26aufirst%3DM.%2BJ.%26atitle%3DCancer%2520immunoediting%253A%2520integrating%2520immunity%25E2%2580%2599s%2520roles%2520in%2520cancer%2520suppression%2520and%2520promotion%26jtitle%3DScience%26date%3D2011%26volume%3D331%26spage%3D1565%26epage%3D1570%26doi%3D10.1126%2Fscience.1203486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marzec, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goradia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghunath, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paessler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wysocka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasik, M. A.</span></span> <span> </span><span class="NLM_article-title">Oncogenic kinase NPM/ALK induces through STAT3 expression of immunosuppressive protein CD274 (PD-L1, B7-H1)</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">20852</span>– <span class="NLM_lpage">20857</span>, <span class="refDoi"> DOI: 10.1073/pnas.0810958105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1073%2Fpnas.0810958105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=19088198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksFSntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=20852-20857&author=M.+Marzecauthor=Q.+Zhangauthor=A.+Goradiaauthor=P.+N.+Raghunathauthor=X.+Liuauthor=M.+Paesslerauthor=H.+Wangauthor=M.+Wysockaauthor=M.+Chengauthor=B.+A.+Ruggeriauthor=M.+A.+Wasik&title=Oncogenic+kinase+NPM%2FALK+induces+through+STAT3+expression+of+immunosuppressive+protein+CD274+%28PD-L1%2C+B7-H1%29&doi=10.1073%2Fpnas.0810958105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic kinase NPM/ALK induces through STAT3 expression of immunosuppressive protein CD274 (PD-L1, B7-H1)</span></div><div class="casAuthors">Marzec, Michal; Zhang, Qian; Goradia, Ami; Raghunath, Puthiyaveettil N.; Liu, Xiaobin; Paessler, Michele; Wang, Hong Yi; Wysocka, Maria; Cheng, Mangeng; Ruggeri, Bruce A.; Wasik, Mariusz A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">20852-20857</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The mechanisms of malignant cell transformation caused by the oncogenic, chimeric nucleophosmin (NPM)/anaplastic lymphoma kinase (ALK) remain only partially understood, with most of the previous studies focusing mainly on the impact of NPM/ALK on cell survival and proliferation.  Here we report that the NPM/ALK-carrying T cell lymphoma (ALK + TCL) cells strongly express the immunosuppressive cell-surface protein CD274 (PD-L1, B7-H1), as detd. on the mRNA and protein level.  The CD274 expression is strictly dependent on the expression and enzymic activity of NPM/ALK, as demonstrated by inhibition of the NPM/ALK function in ALK + TCL cells by the small mol. ALK inhibitor CEP-14083 and by documenting CD274 expression in IL-3-depleted BaF3 cells transfected with the wild-type NPM/ALK, but not the kinase-inactive NPM/ALK K210R mutant or empty vector alone.  NPM/ALK induces CD274 expression by activating its key signal transmitter, transcription factor STAT3.  STAT3 binds to the CD274 gene promoter in vitro and in vivo, as shown in the gel electromobility shift and chromatin immunopptn. assays, and is required for the PD-L1 gene expression, as demonstrated by siRNA-mediated STAT3 depletion.  These findings identify an addnl. cell-transforming property of NPM/ALK and describe a direct link between an oncoprotein and an immunosuppressive cell-surface protein.  These results also provide an addnl. rationale to therapeutically target NPM/ALK and STAT3 in ALK + TCL.  Finally, they suggest that future immunotherapeutic protocols for this type of lymphoma may need to include the inhibition of NPM/ALK and STAT3 to achieve optimal clin. efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh87qHOgigfrVg90H21EOLACvtfcHk0li1LNWB4aUdBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksFSntw%253D%253D&md5=1e85c07581e48e5aa0b9c7d685b57eef</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0810958105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0810958105%26sid%3Dliteratum%253Aachs%26aulast%3DMarzec%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DGoradia%26aufirst%3DA.%26aulast%3DRaghunath%26aufirst%3DP.%2BN.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DPaessler%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWysocka%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DRuggeri%26aufirst%3DB.%2BA.%26aulast%3DWasik%26aufirst%3DM.%2BA.%26atitle%3DOncogenic%2520kinase%2520NPM%252FALK%2520induces%2520through%2520STAT3%2520expression%2520of%2520immunosuppressive%2520protein%2520CD274%2520%2528PD-L1%252C%2520B7-H1%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D20852%26epage%3D20857%26doi%3D10.1073%2Fpnas.0810958105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, l.</span></span> <span> </span><span class="NLM_article-title">EBV-driven LMP1 and IFN-gamma up-regulate PD-L1 in nasopharyngeal carcinoma: implications for oncotargeted therapy</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">12189</span>– <span class="NLM_lpage">12202</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.18632%2Foncotarget.2608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25361008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC2M3kvVWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=12189-12202&author=W.+Fangauthor=J.+Zhangauthor=S.+Hongauthor=J.+Zhanauthor=N.+Chenauthor=T.+Qinauthor=Y.+Tangauthor=Y.+Zhangauthor=S.+Kangauthor=T.+Zhouauthor=X.+Wuauthor=W.+Liangauthor=Z.+Huauthor=Y.+Maauthor=Y.+Zhaoauthor=Y.+Tianauthor=Y.+Yangauthor=C.+Xueauthor=Y.+Yanauthor=X.+Houauthor=P.+Huangauthor=Y.+Huangauthor=H.+Zhaoauthor=l.+Zhang&title=EBV-driven+LMP1+and+IFN-gamma+up-regulate+PD-L1+in+nasopharyngeal+carcinoma%3A+implications+for+oncotargeted+therapy&doi=10.18632%2Foncotarget.2608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">EBV-driven LMP1 and IFN-γ up-regulate PD-L1 in nasopharyngeal carcinoma: Implications for oncotargeted therapy</span></div><div class="casAuthors">Fang Wenfeng; Hong Shaodong; Zhan Jianhua; Qin Tao; Zhang Yaxiong; Kang Shiyang; Zhou Ting; Wu Xuan; Liang Wenhua; Hu Zhihuang; Ma Yuxiang; Zhao Yuanyuan; Tian Ying; Yang Yunpeng; Xue Cong; Yan Yue; Hou Xue; Huang Peiyu; Huang Yan; Zhao Hongyun; Zhang Li; Zhang Jianwei; Chen Nan; Tang Yanna</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">12189-202</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PD-L1 expression is a feature of Epstein-Barr virus (EBV) associated malignancies such as nasopharyngeal carcinoma (NPC).  Here, we found that EBV-induced latent membrane protein 1 (LMP1) and IFN-γ pathways cooperate to regulate programmed cell death protein 1 ligand (PD-L1).  Expression of PD-L1 was higher in EBV positive NPC cell lines compared with EBV negative cell lines.  PD-L1 expression could be increased by exogenous and endogenous induction of LMP1 induced PD-L1.  In agreement, expression of PD-L1 was suppressed by knocking down LMP1 in EBV positive cell lines.  We further demonstrated that LMP1 up-regulated PD-L1 through STAT3, AP-1, and NF-κB pathways.  Besides, IFN-γ was independent of but synergetic with LMP1 in up-regulating PD-L1 in NPC.  Furthermore, we showed that PD-L1 was associated with worse disease-free survival in NPC patients.  These results imply that blocking both the LMP1 oncogenic pathway and PD-1/PD-L1 checkpoints may be a promising therapeutic approach for EBV positive NPC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRaEd7bN61FfO_Q9URG6Dx8fW6udTcc2eZZH_eA1-QbgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3kvVWktg%253D%253D&md5=9e720a88e92ca72d470b12e9c5f6de94</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2608%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DS.%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DN.%26aulast%3DQin%26aufirst%3DT.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3Dl.%26atitle%3DEBV-driven%2520LMP1%2520and%2520IFN-gamma%2520up-regulate%2520PD-L1%2520in%2520nasopharyngeal%2520carcinoma%253A%2520implications%2520for%2520oncotargeted%2520therapy%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D12189%26epage%3D12202%26doi%3D10.18632%2Foncotarget.2608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gato-Cañas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuazo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arasanz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibañez-Vea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Hinojal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smerdou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martisova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arozarena, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellbrock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llopiz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarobe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breckpot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escors, D.</span></span> <span> </span><span class="NLM_article-title">PDL1 Signals through Conserved Sequence Motifs to Overcome Interferon-Mediated Cytotoxicity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1818</span>– <span class="NLM_lpage">1829</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.07.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.celrep.2017.07.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28834746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=1818-1829&author=M.+Gato-Ca%C3%B1asauthor=M.+Zuazoauthor=H.+Arasanzauthor=M.+Iba%C3%B1ez-Veaauthor=L.+Lorenzoauthor=G.+Fernandez-Hinojalauthor=R.+Veraauthor=C.+Smerdouauthor=E.+Martisovaauthor=I.+Arozarenaauthor=C.+Wellbrockauthor=D.+Llopizauthor=M.+Ruizauthor=P.+Sarobeauthor=K.+Breckpotauthor=G.+Kochanauthor=D.+Escors&title=PDL1+Signals+through+Conserved+Sequence+Motifs+to+Overcome+Interferon-Mediated+Cytotoxicity&doi=10.1016%2Fj.celrep.2017.07.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">PDL1 Signals through Conserved Sequence Motifs to Overcome Interferon-Mediated Cytotoxicity</span></div><div class="casAuthors">Gato-Canas, Maria; Zuazo, Miren; Arasanz, Hugo; Ibanez-Vea, Maria; Lorenzo, Laura; Fernandez-Hinojal, Gonzalo; Vera, Ruth; Smerdou, Cristian; Martisova, Eva; Arozarena, Imanol; Wellbrock, Claudia; Llopiz, Diana; Ruiz, Marta; Sarobe, Pablo; Breckpot, Karine; Kochan, Grazyna; Escors, David</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1818-1829</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">PDL1 blockade produces remarkable clin. responses, thought to occur by T cell reactivation through prevention of PDL1-PD1 T cell inhibitory interactions.  Here, we find that PDL1 cell-intrinsic signaling protects cancer cells from interferon (IFN) cytotoxicity and accelerates tumor progression.  PDL1 inhibited IFN signal transduction through a conserved class of sequence motifs that mediate crosstalk with IFN signaling.  Abrogation of PDL1 expression or antibody-mediated PDL1 blockade strongly sensitized cancer cells to IFN cytotoxicity through a STAT3/caspase-7-dependent pathway.  Moreover, somatic mutations found in human carcinomas within these PDL1 sequence motifs disrupted motif regulation, resulting in PDL1 mols. with enhanced protective activities from type I and type II IFN cytotoxicity.  Overall, our results reveal a mode of action of PDL1 in cancer cells as a first line of defense against IFN cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYq1BKCEgRMLVg90H21EOLACvtfcHk0liW2mIjonLNLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2ltLzL&md5=8ceac0a22cc56a0778bd1e4bdd3acf32</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.07.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.07.075%26sid%3Dliteratum%253Aachs%26aulast%3DGato-Ca%25C3%25B1as%26aufirst%3DM.%26aulast%3DZuazo%26aufirst%3DM.%26aulast%3DArasanz%26aufirst%3DH.%26aulast%3DIba%25C3%25B1ez-Vea%26aufirst%3DM.%26aulast%3DLorenzo%26aufirst%3DL.%26aulast%3DFernandez-Hinojal%26aufirst%3DG.%26aulast%3DVera%26aufirst%3DR.%26aulast%3DSmerdou%26aufirst%3DC.%26aulast%3DMartisova%26aufirst%3DE.%26aulast%3DArozarena%26aufirst%3DI.%26aulast%3DWellbrock%26aufirst%3DC.%26aulast%3DLlopiz%26aufirst%3DD.%26aulast%3DRuiz%26aufirst%3DM.%26aulast%3DSarobe%26aufirst%3DP.%26aulast%3DBreckpot%26aufirst%3DK.%26aulast%3DKochan%26aufirst%3DG.%26aulast%3DEscors%26aufirst%3DD.%26atitle%3DPDL1%2520Signals%2520through%2520Conserved%2520Sequence%2520Motifs%2520to%2520Overcome%2520Interferon-Mediated%2520Cytotoxicity%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D1818%26epage%3D1829%26doi%3D10.1016%2Fj.celrep.2017.07.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bill, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofori, G.</span></span> <span> </span><span class="NLM_article-title">The relevance of EMT in breast cancer metastasis: Correlation or causality?</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>589</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1587</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.febslet.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=25979173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVaktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=1577-1587&author=R.+Billauthor=G.+Christofori&title=The+relevance+of+EMT+in+breast+cancer+metastasis%3A+Correlation+or+causality%3F&doi=10.1016%2Fj.febslet.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The relevance of EMT in breast cancer metastasis: Correlation or causality?</span></div><div class="casAuthors">Bill, Ruben; Christofori, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">589</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1577-1587</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although major progress has been achieved in treating breast cancer patients, metastatic breast cancer still remains a deadly disease.  A full understanding of the process of systemic cancer cell dissemination is therefore crit. to develop next generation therapies.  A plethora of exptl. data points toward a central role of an epithelial to mesenchymal transition (EMT) in the multistep cascade of metastasis formation.  However, in patients the data are based on correlative studies which often, but not always, tie the expression of EMT markers to cancer invasion, metastasis and poor clin. outcome.  Moreover, the notion that cancer cells are able to switch between different modes of migration asks for a thorough review of the actual relevance of EMT in cancer metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eoOwCOyzM7Vg90H21EOLACvtfcHk0liW2mIjonLNLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVaktro%253D&md5=78269348415035917bd165fae4553cdd</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DBill%26aufirst%3DR.%26aulast%3DChristofori%26aufirst%3DG.%26atitle%3DThe%2520relevance%2520of%2520EMT%2520in%2520breast%2520cancer%2520metastasis%253A%2520Correlation%2520or%2520causality%253F%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D1577%26epage%3D1587%26doi%3D10.1016%2Fj.febslet.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span> <span> </span><span class="NLM_article-title">PD-L1 expression as poor prognostic factor in patients with non-squamous non-small cell lung cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">58457</span>– <span class="NLM_lpage">58468</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.17022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.18632%2Foncotarget.17022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28938570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FhtlOjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=58457-58468&author=C.+Zhouauthor=J.+Tangauthor=H.+Sunauthor=X.+Zhengauthor=Z.+Liauthor=T.+Sunauthor=J.+Liauthor=S.+Wangauthor=X.+Zhouauthor=H.+Sunauthor=Z.+Chengauthor=H.+Zhangauthor=H.+Ma&title=PD-L1+expression+as+poor+prognostic+factor+in+patients+with+non-squamous+non-small+cell+lung+cancer&doi=10.18632%2Foncotarget.17022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">PD-L1 expression as poor prognostic factor in patients with non-squamous non-small cell lung cancer</span></div><div class="casAuthors">Zhou Cuiling; Sun Huanhuan; Wang Shuncong; Zhou Xiuling; Cheng Zhibin; Zhang Hongyu; Ma Haiqing; Tang Jianjun; Zheng Xiaobin; Li Zhanyu; Sun Tiantian; Li Jie; Sun Hongliu</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">58457-58468</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  The role of programmed cell death ligand 1 (PD-L1) in non-small cell lung cancer (NSCLC), especially according to histologic type, remains controversial.  The purpose of this study was to assess PD-L1 expression and its association with overall survival (OS) and clinicopathologic characteristics in NSCLC.  MATERIALS AND METHODS:  Formalin-fixed paraffin-embedded specimens were obtained from 108 patients with surgically resected primary NSCLC.  PD-L1 expression was assessed via immunohistochemistry using a histochemistry score system.  The relationship between OS or clinicopathologic characteristics and PD-L1 expression was evaluated via the Kaplan-Meier method and Cox proportional hazards model, respectively.  RESULTS:  Of 108 NSCLC specimens, 44 had high PD-L1 expression, which was highly associated with histologic type (p = 0.003).  Patients without PD-L1 expression had remarkably longer OS than those with PD-L1 expression (median OS: 96 months vs. 33 months, p < 0.001).  In the subgroup analysis of non-squamous cell carcinoma, OS was more favorable in those without PD-L1 expression than in those with PD-L1 expression (median OS: 113 months vs. 37 months, p < 0.001).  Multivariate analysis revealed that PD-L1 expression (95% confidence interval 1.459-4.520, p < 0.001), male sex and higher tumor-node-metastasis stage were significantly correlated with shorter OS.  CONCLUSIONS:  This study demonstrated that PD-L1 expression is an independent prognostic factor for poor survival in NSCLC patients, especially those with non-squamous NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRcU4AE24nN2tFxYDaSSP8XfW6udTcc2eZ2o17UaZPrhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FhtlOjug%253D%253D&md5=37fae906831a163f17e3205c84145d9b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.17022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.17022%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DH.%26atitle%3DPD-L1%2520expression%2520as%2520poor%2520prognostic%2520factor%2520in%2520patients%2520with%2520non-squamous%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D58457%26epage%3D58468%26doi%3D10.18632%2Foncotarget.17022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritprajak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashiguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, M.</span></span> <span> </span><span class="NLM_article-title">B7-H1 overexpression regulates epithelial -mesenchymal transition and accelerates carcinogenesis in skin</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1243</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-2217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1158%2F0008-5472.CAN-10-2217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=21159661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFSjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=1235-1243&author=Y.+Caoauthor=L.+Zhangauthor=Y.+Kamimuraauthor=P.+Ritprajakauthor=M.+Hashiguchiauthor=S.+Hiroseauthor=M.+Azuma&title=B7-H1+overexpression+regulates+epithelial+-mesenchymal+transition+and+accelerates+carcinogenesis+in+skin&doi=10.1158%2F0008-5472.CAN-10-2217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">B7-H1 Overexpression Regulates Epithelial-Mesenchymal Transition and Accelerates Carcinogenesis in Skin</span></div><div class="casAuthors">Cao, Yu-Jia; Zhang, Lu; Kamimura, Yosuke; Ritprajak, Patcharee; Hashiguchi, Masaaki; Hirose, Sachiko; Azuma, Miyuki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1235-1243</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">B7-H1 (CD274) is a T-cell coinhibitory mol. that is also often induced on human carcinoma cells, where its expression has been implicated in immune escape.  Under inflammatory conditions, B7-H1 is also inducible in normal epithelial cells but little is known about its involvement in conversion of normal cells to tumor cells.  Here, we show that skin-specific expression of B7-H1 accelerates inflammatory carcinogenesis in a methylcholantrene (MCA)-induced model of squamous cell carcinoma (SCC).  Inflammatory responses induced by MCA or phorbol ester TPA were clearly inhibited in B7-H1 transgenic mice (B7-H1tg mice).  Antibody-mediated blockade of either B7-H1 or the related mol. PD-1 revealed that their ability to limit inflammation relied on ligand interactions made by B7-H1 or PD-1.  Skin keratinocytes derived from B7-H1tg mice exhibited constitutive redn. of E-cadherin, and SCC induced in B7-H1tg mice also showed loss of E-cadherin along with elevated expression of the transcription factors Slug and Twist that drive epithelial-mesenchymal transition (EMT).  Our results indicate that upregulation of B7-H1 in skin epithelial cells promotes EMT and accelerates carcinogenesis, revealing insights into the significance of B7-H1 overexpression on solid tumor cells and hinting at a close relationship between EMT and immune escape signaling pathways in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_SlDDyuBmp7Vg90H21EOLACvtfcHk0lh31snEHClibA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFSjs7c%253D&md5=bbf41d72feef2fc94b6ba2b72e8cc56b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2217%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DKamimura%26aufirst%3DY.%26aulast%3DRitprajak%26aufirst%3DP.%26aulast%3DHashiguchi%26aufirst%3DM.%26aulast%3DHirose%26aufirst%3DS.%26aulast%3DAzuma%26aufirst%3DM.%26atitle%3DB7-H1%2520overexpression%2520regulates%2520epithelial%2520-mesenchymal%2520transition%2520and%2520accelerates%2520carcinogenesis%2520in%2520skin%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D1235%26epage%3D1243%26doi%3D10.1158%2F0008-5472.CAN-10-2217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Way, T.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C.-T.</span></span> <span> </span><span class="NLM_article-title">Emodin represses TWIST1-induced epithelial-mesenchymal transitions in head and neck squamous cell carcinoma cells by inhibiting the β-catenin and Akt pathways</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2013.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejca.2013.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=24157255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSmt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=366-378&author=T.-D.+Wayauthor=J.-T.+Huangauthor=C.-H.+Chouauthor=C.-H.+Huangauthor=M.-H.+Yangauthor=C.-T.+Ho&title=Emodin+represses+TWIST1-induced+epithelial-mesenchymal+transitions+in+head+and+neck+squamous+cell+carcinoma+cells+by+inhibiting+the+%CE%B2-catenin+and+Akt+pathways&doi=10.1016%2Fj.ejca.2013.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Emodin represses TWIST1-induced epithelial-mesenchymal transitions in head and neck squamous cell carcinoma cells by inhibiting the β-catenin and Akt pathways</span></div><div class="casAuthors">Way, Tzong-Der; Huang, Jhen-Ting; Chou, Chun-Hung; Huang, Chi-Hung; Yang, Muh-Hwa; Ho, Chi-Tang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">366-378</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Head and neck squamous cell carcinoma (HNSCC) is one of the leading causes of cancer deaths worldwide.  In recent studies, a crucial link has been discovered between the acquisition of metastatic traits and tumor-initiating abilities in cancer cells during the epithelial-mesenchymal transition (EMT).  Herein, we demonstrated that the ectopic expression of TWIST1, the EMT regulator, in HNSCC FaDu cells triggered EMT and resulted in the acquisition of a mesenchymal phenotype.  Moreover, FaDu-pFLAG-TWIST1 cancer cell populations that were induced to EMT displayed an increased proportion of cells with the CD44 marker, which is assocd. with tumor initiation.  Interestingly, we found that emodin treatment reduced the tumor-initiating abilities and inhibited cell migration and invasion in FaDu-pFLAG-TWIST1 cells.  Emodin directly inhibited TWIST1 expression, upregulated E-cadherin mRNA and protein expression, and downregulated vimentin mRNA and protein expression.  Moreover, we found that emodin inhibited TWIST1 binding to the E-cadherin promoter and repressed E-cadherin transcription activity.  We also found that emodin inhibited TWIST1-induced EMT by inhibiting the β-catenin and Akt pathways.  More interestingly, emodin significantly inhibited TWIST1-induced invasion in vivo.  Therefore, emodin might be applicable to anticancer therapy and could be a potential new therapeutic drug for HNSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXZ43nASnzrVg90H21EOLACvtfcHk0lh31snEHClibA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSmt73M&md5=6a3905b32c13912e073acea500a1e6cb</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2013.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2013.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DWay%26aufirst%3DT.-D.%26aulast%3DHuang%26aufirst%3DJ.-T.%26aulast%3DChou%26aufirst%3DC.-H.%26aulast%3DHuang%26aufirst%3DC.-H.%26aulast%3DYang%26aufirst%3DM.-H.%26aulast%3DHo%26aufirst%3DC.-T.%26atitle%3DEmodin%2520represses%2520TWIST1-induced%2520epithelial-mesenchymal%2520transitions%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%2520cells%2520by%2520inhibiting%2520the%2520%25CE%25B2-catenin%2520and%2520Akt%2520pathways%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2014%26volume%3D50%26spage%3D366%26epage%3D378%26doi%3D10.1016%2Fj.ejca.2013.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and the interaction of 2-(1H-pyrazol-4-yl)-1H-imidazo[4,5-f][1,10]phenanthrolines with telomeric DNA as lung cancer inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.ejmech.2017.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=28376371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVCit70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=36-49&author=J.+Liuauthor=M.+Chenauthor=Y.+Wangauthor=X.+Zhaoauthor=S.+Wangauthor=Y.+Wuauthor=W.+Zhang&title=Synthesis+and+the+interaction+of+2-%281H-pyrazol-4-yl%29-1H-imidazo%5B4%2C5-f%5D%5B1%2C10%5Dphenanthrolines+with+telomeric+DNA+as+lung+cancer+inhibitors&doi=10.1016%2Fj.ejmech.2017.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and the interaction of 2-(1H-pyrazol-4-yl)-1H-imidazo[4,5-f][1,10]phenanthrolines with telomeric DNA as lung cancer inhibitors</span></div><div class="casAuthors">Liu, Jiachun; Chen, Mei; Wang, Yanli; Zhao, Xiaoyin; Wang, Sijia; Wu, Yanling; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36-49</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A novel series of 2-(1H-pyrazol-4-yl)-1H-imidazo[4,5-f][1,10]phenanthrolines I [R1 = F, Cl, Br, CH3, 4-BrC6H4; R2 = H, CH3, Bn, 2,4-(O2N)2C6H3 etc.] was synthesized via Vilsmeier formylation of semicarbazones of substituted acetophenones to yield aryl substituted pyrazole carboxaldehydes which furher condensed with phendione.  These newly synthesized compds. were evaluated for their antitumor activity against lung adenocarcinoma by CCK-8 assay, electrophoretic mobility shift assay (EMSA), UV-melting study, wound healing assay and docking study.  These compds. showed good inhibitory activities against lung adenocarcinoma; esp. compd. I [R1 = Br, R2 = Bn] exhibited potential antiproliferative activity against A549 cell line with the half maximal inhibitory concn. (IC50) value of 1.48 μM, which was a more potent inhibitor than cisplatin (IC50 = 12.08 μM), and the max. cell inhibitory rate being up to 98.40 per cent.  Compds. I [R1 = F, Cl, Br, CH3; R2 = Bn] could strongly interact with telomeric DNA to stabilize G-quadruplex DNA with increased ΔTm values from 12.44 to 20.54°C at a ratio of DNA to compd. 1:10.  These results implied that growth inhibition of A549 cells mediated by these phenanthroline derivs. was pos. correlated to the fact their interaction with telomeric G-quadruplexs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz4vS_d_Fzr7Vg90H21EOLACvtfcHk0lj7Y_EuaFzm5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVCit70%253D&md5=6bb4fc4e5ae7c956d51d4e12ba91d910</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DSynthesis%2520and%2520the%2520interaction%2520of%25202-%25281H-pyrazol-4-yl%2529-1H-imidazo%255B4%252C5-f%255D%255B1%252C10%255Dphenanthrolines%2520with%2520telomeric%2520DNA%2520as%2520lung%2520cancer%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D133%26spage%3D36%26epage%3D49%26doi%3D10.1016%2Fj.ejmech.2017.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_label">(66) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He Wei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Formation and stabilization of the telomeric antiparallel G-quadruplex and inhibition of telomerase by novel benzothioxanthene derivatives with anti-tumor activity</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">13693</span>, <span class="refDoi"> DOI: 10.1038/srep13693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1038%2Fsrep13693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=26329134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A280%3ADC%252BC287ovVOitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=13693&author=W.+Zhangauthor=M.+Chenauthor=Y.+Ling+Wuauthor=Y.+Tanakaauthor=Y.+Juan+Jiauthor=S.+Lin+Zhangauthor=C.+He+Weiauthor=Y.+Xu&title=Formation+and+stabilization+of+the+telomeric+antiparallel+G-quadruplex+and+inhibition+of+telomerase+by+novel+benzothioxanthene+derivatives+with+anti-tumor+activity&doi=10.1038%2Fsrep13693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">6666</span><div class="casTitle"><span class="NLM_cas:atitle">Formation and stabilization of the telomeric antiparallel G-quadruplex and inhibition of telomerase by novel benzothioxanthene derivatives with anti-tumor activity</span></div><div class="casAuthors">Zhang Wen; Chen Min; Juan Ji Yan; Lin Zhang Su; He Wei Chuan; Ling Wu Yan; Tanaka Yoshimasa; Xu Yan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13693</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">G-quadruplexes formed in telomeric DNA sequences at human chromosome ends can be a novel target for the development of therapeutics for the treatment of cancer patients.  Herein, we examined the ability of six novel benzothioxanthene derivatives S1-S6 to induce the formation of and stabilize an antiparallel G-quadruplex by EMSA, UV-melting and CD techniques and the influence of S1-S6 on A549 and SGC7901 cells through real-time cell analysis, wound healing, trap assay methods.  Results show that six compounds could differentially induce 26 nt G-rich oligonucleotides to form the G-quadruplex with high selectivity vs C-rich DNA, mutated DNA and double-stranded DNA, stabilize it with high affinity, promote apoptosis and inhibit mobility and telomerase activity of A549 cells and SGC7901 cells.  Especially, S1, S3, S4 displayed stronger abilities, of which S3 was the most optimal with the maximum ΔTm value being up to 29.8 °C for G-quadruplex, the minimum IC50 value being 0.53 μM and the maximum cell inhibitory rate being up to 97.2%.  This study suggests that this type of compounds that induce the formation of and stabilize the telomeric antiparallel G-quadruplex, and consequently inhibit telomerase activity, leading to cell apoptosis, can be screened for the discovery of novel antitumor therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDnIOW2a-esJQHPYJuLg8TfW6udTcc2eZoqTkqKLCJeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287ovVOitg%253D%253D&md5=16518ffc7648ab388149d1e22fcfe4f3</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fsrep13693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep13693%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DLing%2BWu%26aufirst%3DY.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DJuan%2BJi%26aufirst%3DY.%26aulast%3DLin%2BZhang%26aufirst%3DS.%26aulast%3DHe%2BWei%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DFormation%2520and%2520stabilization%2520of%2520the%2520telomeric%2520antiparallel%2520G-quadruplex%2520and%2520inhibition%2520of%2520telomerase%2520by%2520novel%2520benzothioxanthene%2520derivatives%2520with%2520anti-tumor%2520activity%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D13693%26doi%3D10.1038%2Fsrep13693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashcroft, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saraga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vimalachandran, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prime, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnogorac-Jurcevic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costello, E.</span></span> <span> </span><span class="NLM_article-title">Pancreatic cancer cells overexpress gelsolin family-capping proteins, which contribute to their cell motility</span>. <i>Gut</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1136/gut.2005.083691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1136%2Fgut.2005.083691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=16847067" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2007&pages=95-106&author=C.+C.+Thompsonauthor=F.+J.+Ashcroftauthor=S.+Patelauthor=G.+Saragaauthor=D.+Vimalachandranauthor=W.+Primeauthor=F.+Campbellauthor=A.+Dodsonauthor=R.+E.+Jenkinsauthor=N.+R.+Lemoineauthor=T.+Crnogorac-Jurcevicauthor=H.+Yinauthor=E.+Costello&title=Pancreatic+cancer+cells+overexpress+gelsolin+family-capping+proteins%2C+which+contribute+to+their+cell+motility&doi=10.1136%2Fgut.2005.083691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic cancer cells overexpress gelsolin family-capping proteins, which contribute to their cell motility</span></div><div class="casAuthors">Thompson, C. C.; Ashcroft, F. J.; Patel, S.; Saraga, G.; Vimalachandran, D.; Prime, W.; Campbell, F.; Dodson, A.; Jenkins, R. E.; Lemoine, N. R.; Crnogorac-Jurcevic, T.; Yin, H. L.; Costello, E.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-106</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Previously, proteomic methods were applied to characterize differentially expressed proteins in microdissected pancreatic ductal adenocarcinoma cells.  To report that CapG and a related protein, gelsolin, which have established roles in cell motility, are overexpressed in metastatic pancreatic cancer; and to describe their pattern of expression in pancreatic cancer tissue and their effect on cell motility in pancreatic cancer cell lines.  CapG was identified by mass spectrometry and immunoblotting.  CapG and gelsolin expression was assessed by immunohistochem. anal. on a pancreatic cancer tissue microarray and correlated with clin. and pathol. parameters.  CapG and gelsolin levels were reduced using RNA interface in Suit-2, Panc-1 and MiaPaCa-2 cells.  Cell motility was assessed using modified Boyden chamber or wound-healing assays.  Multiple isoforms of CapG were detected in pancreatic cancer tissue and cell lines.  Immunohistochem. anal. of benign (n = 44 patients) and malignant (n = 69) pancreatic ductal cells showed significantly higher CapG staining intensity in nuclear (p < 0.001) and cytoplasmic (p < 0.001) compartments of malignant cells.  Similarly, gelsolin immunostaining of benign (n = 24 patients) and malignant (n = 68 patients) pancreatic ductal cells showed higher expression in both compartments (both p < 0.001).  High nuclear CapG was assocd. with increased tumor size (p = 0.001).  High nuclear gelsolin was assocd. with reduced survival (p = 0.01).  Redn. of CapG or gelsolin expression in cell lines by RNAi was accompanied by significantly impaired motility.  Up regulation of these actin-capping proteins in pancreatic cancer and their ability to modulate cell motility in vitro suggest their potentially important role in pancreatic cancer cell motility and consequently dissemination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvU820K_DYDbVg90H21EOLACvtfcHk0lioRrViovoyPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2qsLY%253D&md5=8f7c0bf71f5a6d28cd24d8018ce0ec54</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1136%2Fgut.2005.083691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.2005.083691%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DC.%2BC.%26aulast%3DAshcroft%26aufirst%3DF.%2BJ.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DSaraga%26aufirst%3DG.%26aulast%3DVimalachandran%26aufirst%3DD.%26aulast%3DPrime%26aufirst%3DW.%26aulast%3DCampbell%26aufirst%3DF.%26aulast%3DDodson%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DR.%2BE.%26aulast%3DLemoine%26aufirst%3DN.%2BR.%26aulast%3DCrnogorac-Jurcevic%26aufirst%3DT.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DCostello%26aufirst%3DE.%26atitle%3DPancreatic%2520cancer%2520cells%2520overexpress%2520gelsolin%2520family-capping%2520proteins%252C%2520which%2520contribute%2520to%2520their%2520cell%2520motility%26jtitle%3DGut%26date%3D2007%26volume%3D56%26spage%3D95%26epage%3D106%26doi%3D10.1136%2Fgut.2005.083691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitz, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussif, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, D.</span></span> <span> </span><span class="NLM_article-title">PD-1 inhibits T-cell receptor induced phosphorylation of the ZAP70/CD3ζ signalosome and downstream signaling to PKCθ</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>574</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2004.07.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=10.1016%2Fj.febslet.2004.07.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=15358536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFGmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=574&publication_year=2004&pages=37-41&author=K.+A.+Sheppardauthor=L.+J.+Fitzauthor=J.+M.+Leeauthor=C.+Benanderauthor=J.+A.+Georgeauthor=J.+Wootersauthor=Y.+Qiuauthor=J.+M.+Jussifauthor=L.+L.+Carterauthor=C.+R.+Woodauthor=D.+Chaudhary&title=PD-1+inhibits+T-cell+receptor+induced+phosphorylation+of+the+ZAP70%2FCD3%CE%B6+signalosome+and+downstream+signaling+to+PKC%CE%B8&doi=10.1016%2Fj.febslet.2004.07.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">PD-1 inhibits T-cell receptor induced phosphorylation of the ZAP70/CD3ζ signalosome and downstream signaling to PKCθ</span></div><div class="casAuthors">Sheppard, Kelly-Ann; Fitz, Lori J.; Lee, Julie M.; Benander, Christina; George, Judith A.; Wooters, Joe; Qiu, Yongchang; Jussif, Jason M.; Carter, Laura L.; Wood, Clive R.; Chaudhary, Divya</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">574</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">37-41</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Engagement of the immunoinhibitory receptor, programmed death-1 (PD-1) attenuates T-cell receptor (TCR)-mediated activation of IL-2 prodn. and T-cell proliferation.  Here, we demonstrate that PD-1 modulation of T-cell function involves inhibition of TCR-mediated phosphorylation of ZAP70 and assocn. with CD3ζ.  In addn., PD-1 signaling attenuates PKCθ activation loop phosphorylation in a cognate TCR signal.  PKCθ has been shown to be required for T-cell IL-2 prodn.  A phosphorylated PD-1 peptide, corresponding to the C-terminal immunoreceptor tyrosine-switch motif (ITSM), acts as a docking site in vitro for both SHP-2 and SHP-1, while the phosphorylated peptide contg. the N-terminal PD-1 immunoreceptor tyrosine based inhibitory motif (ITIM) assocs. only with SHP-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr85CRiewsx1LVg90H21EOLACvtfcHk0lioRrViovoyPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFGmu7Y%253D&md5=2c9dbaa347d320423858ec102649d14f</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2004.07.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2004.07.083%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DK.%2BA.%26aulast%3DFitz%26aufirst%3DL.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DBenander%26aufirst%3DC.%26aulast%3DGeorge%26aufirst%3DJ.%2BA.%26aulast%3DWooters%26aufirst%3DJ.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DJussif%26aufirst%3DJ.%2BM.%26aulast%3DCarter%26aufirst%3DL.%2BL.%26aulast%3DWood%26aufirst%3DC.%2BR.%26aulast%3DChaudhary%26aufirst%3DD.%26atitle%3DPD-1%2520inhibits%2520T-cell%2520receptor%2520induced%2520phosphorylation%2520of%2520the%2520ZAP70%252FCD3%25CE%25B6%2520signalosome%2520and%2520downstream%2520signaling%2520to%2520PKC%25CE%25B8%26jtitle%3DFEBS%2520Lett.%26date%3D2004%26volume%3D574%26spage%3D37%26epage%3D41%26doi%3D10.1016%2Fj.febslet.2004.07.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i39"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00120">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_67224"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00120?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00120</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Schematic of the Fmoc solid-phase synthesis of the polyamide <b>1</b> and other biological results, including UV melting curves, CD spectra, a typical SPR sensorgram for Mis-polyamide, molecular docking of Mis-polyamide and the ds-DNA, distribution of fluorescein (Fluo)-labeled polyamide <b>2</b><i>in vitro</i>, structure of PD-1/PD-L1 inhibitor <b>2</b> (BMS202), effect of polyamides on Jurkat cell viability, and effect of polyamides <b>1</b> and <b>2</b> on the mobility of HCC827 and MDA-MB-231 cells (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Antiproliferative activity of compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_002.csv">CSV</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_001.pdf">jm1c00120_si_001.pdf (0.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_002.csv">jm1c00120_si_002.csv (1.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00120/suppl_file/jm1c00120_si_003.csv">jm1c00120_si_003.csv (0.96 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00120&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00120" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00120" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0aee4db901943","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
