
<html lang="en"     class="pb-page"  data-request-id="b67bdc81-63ac-4346-bf6e-0e188c97d8ed"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02055;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors" /></meta><meta name="dc.Creator" content="Yi  Zhang" /></meta><meta name="dc.Creator" content="Deqin  Rong" /></meta><meta name="dc.Creator" content="Bingbing  Li" /></meta><meta name="dc.Creator" content="Yuanxiang  Wang" /></meta><meta name="dc.Description" content="Epigenetic regulation of gene expression plays a critical role in various physiological processes, and epigenetic dysregulation is implicated in a number of diseases, prominently including cancer. ..." /></meta><meta name="Description" content="Epigenetic regulation of gene expression plays a critical role in various physiological processes, and epigenetic dysregulation is implicated in a number of diseases, prominently including cancer. ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 18, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02055" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02055" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02055" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02055" /></link>
        
    
    

<title>Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02055" /></meta><meta property="og:title" content="Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0020.jpeg" /></meta><meta property="og:description" content="Epigenetic regulation of gene expression plays a critical role in various physiological processes, and epigenetic dysregulation is implicated in a number of diseases, prominently including cancer. Epigenetic regulators have been validated as potential therapeutic targets, and significant progress has been made in the discovery and development of epigenetic-based inhibitors. However, successful epigenetic drug discovery is still facing challenges, including moderate selectivity, limited efficacy, and acquired drug resistance. Inspired by the advantages of covalent small-molecule inhibitors, targeted covalent inhibition has attracted increasing interest in epigenetic drug discovery. In this review, we comprehensively summarize the structure-based design and characterization of covalent inhibitors targeting epigenetic writers, readers, and erasers and highlight their potential benefits in enhancing selectivity across the enzyme family and improving in vivo efficacy. We also discuss the challenges and opportunities of covalent small-molecule inhibitors and hope to shed light on future epigenetic drug discovery." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02055"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02055">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02055&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02055&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02055&amp;href=/doi/10.1021/acs.jmedchem.0c02055" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 7900-7925</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02026" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c02265" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yi Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Guangdong Key Laboratory of Chiral Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi++Zhang">Yi Zhang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Deqin Rong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Deqin Rong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Guangdong Key Laboratory of Chiral Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Deqin++Rong">Deqin Rong</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Bingbing Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bingbing Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Guangdong Key Laboratory of Chiral Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bingbing++Li">Bingbing Li</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Yuanxiang Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuanxiang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Guangdong Key Laboratory of Chiral Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#0c7b6d626b757435394c616d6560227f757f7922696879226f62"><span class="__cf_email__" data-cfemail="99eef8f7fee0e1a0acd9f4f8f0f5b7eae0eaecb7fcfdecb7faf7">[emailÂ protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuanxiang++Wang">Yuanxiang Wang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5650-4396" title="Orcid link">https://orcid.org/0000-0001-5650-4396</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02055&amp;href=/doi/10.1021%2Facs.jmedchem.0c02055" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 7900â7925</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 18, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 November 2020</li><li><span class="item_label"><b>Published</b> online</span>18 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7900%26pageCount%3D26%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYi%2BZhang%252C%2BDeqin%2BRong%252C%2BBingbing%2BLi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.0c02055%26title%3DTargeting%2BEpigenetic%2BRegulators%2Bwith%2BCovalent%2BSmall-Molecule%2BInhibitors%26numPages%3D26%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7925%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02055"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3245</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02055" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yi&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Deqin&quot;,&quot;last_name&quot;:&quot;Rong&quot;},{&quot;first_name&quot;:&quot;Bingbing&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yuanxiang&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7900-7925&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02055&quot;},&quot;abstract&quot;:&quot;Epigenetic regulation of gene expression plays a critical role in various physiological processes, and epigenetic dysregulation is implicated in a number of diseases, prominently including cancer. Epigenetic regulators have been validated as potential therapeutic targets, and significant progress has been made in the discovery and development of epigenetic-based inhibitors. However, successful epigenetic drug discovery is still facing challenges, including moderate selectivity, limited efficacy, and acquired drug resistance. Inspired by the advantages of covalent small-molecule inhibitors, targeted covalent inhibition has attracted increasing interest in epigenetic drug discovery. In this review, we comprehensively summarize the structure-based design and characterization of covalent inhibitors targeting epigenetic writers, readers, and erasers and highlight their potential benefits in enhancing selectivity across the enzyme family and improving in vivo efficacy. We also discuss the challenges and opportuniti&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02055&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02055" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02055&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02055" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02055&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02055" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02055&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02055&amp;href=/doi/10.1021/acs.jmedchem.0c02055" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02055" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02055" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26pmid%3D33599482%26genre%3Darticle%26aulast%3DZhang%26date%3D2021%26atitle%3DTargeting%2BEpigenetic%2BRegulators%2Bwith%2BCovalent%2BSmall-Molecule%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Epigenetic regulation of gene expression plays a critical role in various physiological processes, and epigenetic dysregulation is implicated in a number of diseases, prominently including cancer. Epigenetic regulators have been validated as potential therapeutic targets, and significant progress has been made in the discovery and development of epigenetic-based inhibitors. However, successful epigenetic drug discovery is still facing challenges, including moderate selectivity, limited efficacy, and acquired drug resistance. Inspired by the advantages of covalent small-molecule inhibitors, targeted covalent inhibition has attracted increasing interest in epigenetic drug discovery. In this review, we comprehensively summarize the structure-based design and characterization of covalent inhibitors targeting epigenetic writers, readers, and erasers and highlight their potential benefits in enhancing selectivity across the enzyme family and improving in vivo efficacy. We also discuss the challenges and opportunities of covalent small-molecule inhibitors and hope to shed light on future epigenetic drug discovery.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span>Â Special Issue</h4><p class="last">Published as part of the <i>Journal of Medicinal Chemistry</i> special issue â<a href="/page/vi/epigenetics" class="ext-link">Epigenetics 2022</a>â.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82086" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82086" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetics refers to heritable phenotype changes in gene expression that do not involve variations in DNA sequence.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The main epigenetic factors include DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. Among others, histone proteins undergo post-translational modification (PTM) in different ways, including acetylation, methylation, phosphorylation, ubiquitylation, neddylation, and sumoylation. Epigenetic regulation is a dynamic and reversible process that is catalyzed by a wide range of proteins, which have been grouped into epigenetic writers, readers, and erasers according to their distinct functions.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Epigenetic writers, such as DNA methyltransferases (DNMTs), histone methyltransferases (HMTs), and histone acetyltransferases (HATs), catalyze the transfer of epigenetic marks into either DNA or histones. Epigenetic readers, such as bromodomain and extraterminal (BET) proteins that recognize and bind to acetylated lysine residues in histones H3 and H4, recognize or are recruited to a specific epigenetic mark. Epigenetic erasers, including histone demethylases (HDMs), histone deacetylases (HDACs), deubiquitinating enzymes (DUBs), and phosphatases, remove epigenetic marks.</div><div class="NLM_p">Epigenetic regulation of gene expression plays a critical role in various physiological processes, while epigenetic dysregulation is involved in a plethora of diseases, most prominently in cancer.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Epigenetic changes have been shown to be associated with cancer initiation and progression.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Therefore, the development and validation of drugs targeting epigenetic regulators for cancer therapy have attracted an increasing amount attention.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10">(5â10)</a> Indeed, several epigenetic drugs have been approved by the U.S. Food and Drug Administration (FDA) for the treatment of certain hematologic malignancies, such as DNMT inhibitors (azacitidine and decitabine) and HDAC inhibitors (vorinostat, romidepsin, belinostat, panobinostat, and chidamide). Furthermore, additional clinical trials involving inhibitors of epigenetic regulators are also ongoing.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, successful epigenetic drug discovery still faces challenges, such as moderate selectivity and limited efficacy. For instance, in view of the heterogeneity of proteins involved in epigenetic mechanisms, the selectivity issue should be considered within each enzyme family and across enzyme families. The challenge in selectivity optimization is derived from the high sequence identity and conserved catalytic domain across the enzyme family.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> On the other hand, discovering epigenetic drugs with high efficacy in vivo has been proven to be challenging because of the competition of drugs with high levels of internal cofactors in cells or competition with high-affinity complex interactions.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In addition, the occurrence of acquired resistance is also one of the major obstacles in epigenetic drug discovery. For instance, acquired resistance to DNMT and HDAC inhibitors has been reported.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> In summary, several challenges still exist in the area of developing selective and potent epigenetic drugs, including how to optimize the selectivity across the enzyme family, how to improve the in vivo efficacy by reducing competition with internal cofactors, and how to overcome the acquired drug resistance. Such challenges have hampered many studies on epigenetic functions and significantly delayed progress in epigenetic drug discovery.</div><div class="NLM_p">A look back at the history of drug discovery and development shows that the development of tyrosine kinase inhibitors (TKIs) also faced the above-mentioned three challenges in the early stage.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> First, the highly conserved ATP binding pockets among members of the kinase family present a formidable challenge for the development of selective TKIs. Second, the high concentration of ATP in cells and its binding affinity with kinases can negatively impact the engagement potency for TKIs. Third, clinically acquired resistance to TKIs generally occurs within 9â15 months. To overcome these challenges, targeted covalent inhibition has been applied in the development of potent and selective TKIs.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> Covalent targeting of a nonconserved nucleophilic residue, such as cysteine, near the highly conserved ATP binding pocket offers an opportunity to develop selective TKIs. Covalent binding of TKIs to kinases could also reduce competition with ATP to improve their in vivo efficacy. Additionally, covalent TKIs demonstrate prolonged duration of action and exhibit a reduced propensity for acquired resistance.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> On the basis of these encouraging benefits, seven covalent TKIs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) have been approved by the FDA to date, and many more covalent TKIs are undergoing clinical trials worldwide.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Covalent small-molecule TKIs approved by the FDA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Inspired by the benefits that covalent inhibitors display in the field of kinase drug discovery, targeted covalent inhibition has gained an increasing amount of attention in epigenetic drug discovery. Herein we comprehensively summarize recent advances in the development of covalent inhibitors targeting epigenetic regulators, including epigenetic writers, readers, and erasers, and highlight their potential benefits in enhancing selectivity within each enzyme family and across enzyme families and in improving in vivo efficacy. In the last part of the review, we will discuss the limitations in the development of epigenetic covalent inhibitors and hope to shed light on future drug discovery.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">2.  Targeting Epigenetic Writers with Covalent Small-Molecule Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85344" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85344" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic writers catalyze various chemical modifications on DNA molecules or histone proteins, such as methylation, acetylation, phosphorylation, ubiquitylation, neddylation, and sumoylation. In this section, we emphasize the recent progress in the discovery of covalent inhibitors targeting HMTs and DNMTs. In the case of HMTs, which are further subdivided into protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) on the basis of their target residues, we deal with each enzyme separately with its reported covalent inhibitors.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.1.  Targeting the MeninâMLL ProteinâProtein Interaction with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">Mixed-lineage leukemia (MLL) protein, also known as MLL1 or lysine-specific methyltransferase 2A (KMT2A), belongs to the SET1 family, which also consists of MLL2â4, SET1A, and SET1B, and is responsible for specific methylation of histone 3 lysine 4 residue (H3K4). MLL was reported to play a pivotal role in the maintenance of <i>HOX</i> gene expression patterns during hematopoiesis and embryonic development.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, unlike most other SET-domain-containing lysine methyltransferases, MLL itself displays poor methyltransferase activity, mainly because of its structural basis. Crystal structure studies have shown that the MLL protein exists in an open conformation, which is ineffective for the methylation process. The optimal catalytic activity of MLL requires additional partners in order to form a complex, including WD-repeat-containing protein 5 (WDR5), retinoblastoma-binding protein 5 (RBBP5), and ASH2-like histone lysine methyltransferase complex subunit (ASH2L), which are thought to orient the C-terminal SET domain of MLL to facilitate the methylation of H3K4.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p">On the other hand, clinical investigations revealed that the MLL gene rearrangement located at chromosome band 11q23 was observed in 5â10% of acute myeloid leukemia (AML) in adults and almost 70% of acute lymphoblastic leukemia (ALL) in infants.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This rearrangement leads to the expression of the MLL fusion protein by preservation of the N-terminal MLL fragment of approximately 1400 amino acids and fusion with one of over 70 partner genes. The N-terminal fragment retained in all MLL fusion proteins is involved in the proteinâprotein interaction (PPI) with menin, which is encoded by the <i>MEN1</i> gene and functions as an oncogenic cofactor of MLL fusion proteins. The PPIs between menin and the MLL fusion proteins upregulate the expression of target genes, including the <i>MEIS1</i> and <i>HOX</i> genes, both of which are essential for leukemogenesis in MLL leukemia. In addition, the PPI between menin and wild-type MLL is also critical for the development of solid tumors, including liver, brain, colon, and breast cancers. Therefore, blocking the meninâMLL interaction with small-molecule inhibitors represents a potential strategy for cancer treatment.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_p">Structural studies showed that MLL interacts with menin in the bivalent binding mode via two short motifs in the N-terminus of MLL, including menin binding motif 1 (MBM1) and menin binding motif 2 (MBM2). MBM1 contains MLL residues 4â15 and binds to the large central cavity on menin with high binding affinity (<i>K</i><sub>D</sub> = 56 nM), while MBM2 encompasses residues 23â43 and binds to menin with a <i>K</i><sub>D</sub> value of 1 Î¼M. Therefore, small-molecule inhibitors of the meninâMLL interaction were mainly designed to target the MBM1 pocket on menin.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> The first series of meninâMLL inhibitors were discovered via high-throughput screening (HTS) by Grembecka and co-workers, as exemplified by compound <b>1</b> (MI-1) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), which is characterized by a thienopyrimidine core structure.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Compound <b>1</b> inhibited the meninâMLL interaction with an IC<sub>50</sub> value of 1.9 Î¼M measured by fluorescence polarization (FP). Optimization of the hydrophobic group on the thienopyrimidine moiety led to the identification of several analogues with improved activities, represented by compounds <b>2</b> (MI-2) and <b>3</b> (MI-2-2) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) with IC<sub>50</sub> values of 0.446 and 0.046 Î¼M, respectively.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Analysis of the crystal structure of compound <b>3</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ4">4GQ4</a>) revealed that <b>3</b> binds to the MBM1 binding site on menin and competes with MLL. However, compound <b>3</b> displayed modest cellular activity against MLL-AF9-transformed murine bone marrow cells (BMCs), with a GI<sub>50</sub> value of 1.3 Î¼M and exhibited poor metabolic stability, with a half-life (<i>T</i><sub>1/2</sub>) of less than 2 min in mouse liver microsomes, making it unsuitable for in vivo characterization.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of compounds <b>1</b>â<b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In an effort to improve the cellular activity and drug-like properties of lead compounds, structure-based drug design combined with a medicinal chemistry campaign was employed to identify more potent meninâMLL inhibitors. This effort resulted in the identification of <b>4</b> (MI-136) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), which was designed by incorporating a 2-cyanoindole fragment into the thienopyrimidine core via a piperidine linker.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Compound <b>4</b> showed enhanced growth inhibition with a GI<sub>50</sub> value of 0.55 Î¼M against BMCs and exhibited an increase in <i>T</i><sub>1/2</sub> to 6.8 min.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Further improvement in the cellular activity and drug-like properties of <b>4</b> led to analogue <b>5</b> (MI-538) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), in which a pyrazol group is appended to the indole ring via a methylene linker. This compound displayed dramatically improved growth inhibitory activity (GI<sub>50</sub> = 0.083 Î¼M against BMCs) and metabolic stability (<i>T</i><sub>1/2</sub> = 14 min).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> However, compound <b>5</b> still suffered from modest metabolic stability and needed to be further optimized. With the aim of enhancing the metabolic stability of <b>5</b>, extensive exploration of substitutions on the indole nitrogen was conducted, resulting in <b>6</b> (MI-1481) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), which potently inhibited the meninâMLL interaction, with an IC<sub>50</sub> value of 3.6 nM, and demonstrated pronounced cell growth inhibition against BMCs, with a GI<sub>50</sub> value of 0.034 Î¼M. Interestingly, compound <b>6</b> possessed significantly improved metabolic stability, with a <i>T</i><sub>1/2</sub> of 59 min.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Unfortunately, <b>6</b> exhibited low oral bioavailability (<i>F</i> < 5%), mainly because of its increased molecular weight. It is noteworthy that Armstrong and co-workers identified the potent meninâMLL inhibitor <b>7</b> (VTP50469) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) by maintaining the key pyrimidine moiety embedded in <b>2</b>, which was capable of suppressing a subset of MLL fusion target genes and prolonging survival in a patient-derived xenograft (PDX) model of MLL-rearranged luekemia.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_p">The second series of meninâMLL inhibitors were also initially discovered by the Grembecka group and characterized by a hydroxymethylpiperidine core structure, as exemplified by compound <b>8</b> (MIV-3R) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), which inhibited the meninâMLL interaction with an IC<sub>50</sub> value of 270 nM as determined by the FP assay and bound to menin with a <i>K</i><sub>D</sub> value of 285 nM.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The cocrystal structure of <b>8</b> in complex with menin (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A) verified its direct binding to the MLL binding site on menin and revealed a water-mediated hydrogen bond between the hydroxyl group of <b>8</b> and Asp180, which provided a suitable site for further improving the potency and physicochemical properties. Replacement of the hydroxyl group with an amino group resulted in compound <b>9</b> (MIV-6R) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) possessing enhanced inhibitory activity (IC<sub>50</sub> = 56 nM) and stronger binding affinity to menin (<i>K</i><sub>D</sub> = 85 nM), mainly due to additional favorable electrostatic interactions between the amino group and Asp180.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> However, <b>9</b> showed moderate inhibition of cell proliferation in BMCs, with a GI<sub>50</sub> value of 1.1 Î¼M, and was prone to being metabolized at the amino group site.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of compounds <b>8</b>â<b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structures of meninâMLL inhibitors in complex with menin. (A) Cocrystal structure of <b>8</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG3">4OG3</a>). (B) Cocrystal structure of <b>9</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8">4OG8</a>). (C) Cocrystal structure of <b>12</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B41">6B41</a>). (D) Cocrystal structure of <b>14</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WNH">6WNH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the aforementioned progress, targeting the meninâMLL interaction by small-molecule inhibitors with greater in vivo potency and favorable drug-like properties is still challenging, mainly due to the large size of the binding site on menin and the lack of well-defined pockets. To address this issue, Wang and co-workers proposed targeted covalent inhibitors of the meninâMLL interaction to achieve high potency and favorable drug-like properties by.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Analysis of the cocrystal structure of <b>9</b> in complex with menin (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) revealed that the cyanophenyl group of <b>9</b> is located in proximity to the Cys329 residue within menin, providing a structural basis for the design of covalent meninâMLL inhibitors.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Prior to structure-based design of covalent meninâMLL inhibitors, further structural optimization of <b>9</b> was conducted to improve its metabolic stability. Extensive optimization of <b>9</b> by replacing the primary amine group with a metabolically stable nitrile group, constraining the flexible linker, and incorporating a reverse carbamate at the 2-position of the cyclopentyl ring resulted in the identification of <b>10</b> (M-470) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), which bound to menin with an IC<sub>50</sub> value of 7 nM.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Most strikingly, compound <b>10</b> also exhibited stronger cancer cell growth inhibition and more favorable metabolic stability than compound <b>9</b>, allowing it to serve as a good reversible meninâMLL inhibitor for covalent inhibitor discovery. Therefore, covalent meninâMLL inhibitor <b>11</b> was designed by replacing the cyclobutyl group in <b>10</b> with an azetidinyl group and installing an acrylamide group onto the nitrogen atom of the azetidinyl group.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Compound <b>11</b> demonstrated enhanced binding affinity to menin (IC<sub>50</sub> = 3 nM) and exhibited stronger cell growth inhibition (IC<sub>50</sub> = 6 nM) in the MV4;11 cell line than non-covalent <b>10</b> (IC<sub>50</sub> = 210 nM). Furthermore, mass spectrometry analyses confirmed that compound <b>11</b> formed a covalent complex with recombinant human menin protein.</div><div class="NLM_p last">In order to improve the solubility of <b>11</b>, a (dimethylamino)methyl group was attached to the acrylamide group, affording <b>12</b> (MI-525) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), which showed more potent antiproliferative effect against the MV4;11 cell line (IC<sub>50</sub> = 4 nM) compared with compound <b>11</b>.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Further evaluation of its activity and specificity in a variety of human acute leukemia cell lines with or without MLL fusions revealed that <b>12</b> significantly inhibited the cell growth of six leukemia cell lines harboring MLL fusions, with IC<sub>50</sub> values ranging from 2.3 to 31.3 nM, while it displayed much weaker activity, with IC<sub>50</sub> values of 900â6500 nM, in four leukemia cell lines lacking MLL fusion, suggesting its excellent cellular selectivity for MLL leukemia. On the other hand, the expression of the MLL target genes, such as <i>MEIS1</i> and <i>HOX</i>, was significantly downregulated in a dose-dependent manner after treatment with <b>12</b> in MV4;11 and MOLM-13 cells. To establish the binding mode of <b>12</b> with menin, a cocrystal structure of <b>12</b> in complex with menin was determined (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), which showed that <b>12</b> forms a covalent bond with the sulfur atom of Cys329 through its acrylamide group. The nitrile group of <b>12</b> is exposed to the solvent environment and is located in proximity to the negatively charged Asp180 and Glu359, indicating that replacement of the nitrile group with a positively charged group may further enhance the binding affinity and cellular potency of <b>12</b>. Accordingly, the nitrile group of <b>12</b> was replaced with a methylene group attached to an azetidine group, leading to compound <b>13</b>, which achieved IC<sub>50</sub> values of 3 and 8 nM, respectively, in inhibition of cell growth of the MV4;11 and MOLM-13 cell lines.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Further replacement of the dimethylamino group in compound <b>13</b> with a piperidine group resulted in <b>14</b> (M-808) <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) as a highly potent and efficacious covalent meninâMLL inhibitor.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><b>14</b> exhibited excellent cellular activity against the MV4;11 and MOLM-13 cell lines, with IC<sub>50</sub> values of 1 and 4 nM, respectively. The cocrystal structure of <b>14</b> in complex with menin (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D) also indicated the strong chargeâcharge interaction between the basic nitrogen atom of <b>14</b> and the carboxylic acid group of Asp180. More importantly, it displayed robust antitumor effects in an MV4;11 xenograft tumor model in mice with severe combined immunodeficiency (SCID) and achieved a maximum tumor growth inhibition (TGI) of 97% when dosed intravenously at 25 mg/kg every other day, three times per week, for a total of 11 doses. No significant weight change and no signs of drug-related toxicity were observed in treated mice.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.2.  Targeting Lysine Methyltransferase SETD8 with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">As a PKMT, SET-domain-containing protein 8 (SETD8), also known as SET8, PR-SET7, or lysine methyltransferase 5A (KMT5A), belongs to the SET-domain-containing family. SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 Lys20 (H4K20me1), which is associated with regulation of gene transcription, DNA damage response, chromatin condensation, cell-cycle progression, and development.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Apart from H4K20, SETD8 also catalyzes the methylation of non-histone substrates, including the tumor suppressor p53 and proliferating cell nuclear antigen (PCNA).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Monomethylation of p53 at Lys382 by SETD8 was reported to suppress p53-mediated transcriptional activation of highly responsive target genes such as p21 and PUMA, while genetic or pharmacological SETD8 inhibition led to activation of the p53 canonical pathway and rescued the pro-apoptotic function and cell-cycle arrest functions of p53 by decreasing p53K382me1 levels.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> SETD8 stabilizes the PCNA protein through monomethylation of Lys248, resulting in enhanced interaction between PCNA and the flap endonuclease FEN1. However, depletion of SETD8 is able to destabilize the expression of PCNA.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Overexpression of SETD8 is implicated in various types of cancer, including nonsmall cell and small cell lung cancer, bladder cancer, pancreatic cancer, hepatocellular carcinoma (HCC), and chronic myelogenous leukemia (CML).<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Furthermore, high expression of SETD8 is reported to be correlated with poor prognosis in cancer patients. These results indicate that SETD8 inhibitors could serve as valuable anticancer agents.</div><div class="NLM_p">Nauhoic acid A (<b>15</b>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), a marine natural product, was reported to inhibit SETD8 enzymatic activity with an IC<sub>50</sub> value of 6.5 Î¼M and was selective for SETD8 over 10 other methyltransferases.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Kinetic experiments revealed that it was competitive with the cofactor <i>S</i>-adenosylmethionine (SAM) and noncompetitive with respect to the peptide substrate. However, further cellular evaluation demonstrated that it only weakly inhibited cancer cell growth, including the U2OS, SUM159, and MDA-MB-436 cell lines with IC<sub>50</sub> values ranging from 45 to 85 Î¼M. Luo and co-workers identified several potent SETD inhibitors through HTS, as exemplified by compounds <b>16</b>â<b>18</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) with IC<sub>50</sub> values of 0.21, 0.50, and 0.70 Î¼M, respectively.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> A reduction in the H4K20me1 mark was observed in HEK293T cells within 24 h after treatment with <b>16</b>â<b>18</b> at doses of 1 and 5 Î¼M, and no other histone marks such as total H4 and H3, H4K20me2/3, and H3K9me were changed. These compounds feature a quinone motif as the core structure, which may form a covalent bond with a cysteine residue. Mechanism of action (MOA) studies revealed that <b>16</b>â<b>18</b> were able to inhibit SETD8 with sustained potency even after washout treatment, indicating an irreversible process. Cys311 of SETD8 was identified to be the residue covalently modified by <b>16</b> and <b>17</b>, while <b>18</b> may target other cysteine residues within SETD8. Later, Luo and co-workers reported compound <b>19</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) as an analogue of compounds <b>16</b>â<b>18</b> and resolved its cocrystal structure in complex with SETD8.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, compound <b>19</b> formed a covalent bond with Cys311 via substitution of the phenylthiol group by the sulfur atom of Cys311. However, it should be noted that the four compounds showed modest selectivity for SETD8, consistent with the fact that the quinone moiety is usually regarded as a feature of pan-assay interference compounds (PAINS).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of compounds <b>15</b>â<b>23</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Cocrystal structure of <b>19</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W1Y">5W1Y</a>). (B) Cocrystal structure of <b>21</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T5G">5T5G</a>). (C) Cocrystal structure of <b>22</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH7">5TH7</a>). (D) Cocrystal structure of <b>23</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOZ">6BOZ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In an effort to develop highly selective and potent SETD8 inhibitors, targeted covalent inhibition of SETD8 was exploited by Jin and co-workers.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Initially, compound <b>20</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) was identified as a non-covalent SETD8 inhibitor with an in vitro IC<sub>50</sub> value of 7.3 Î¼M via cross-screening with a quinazoline-based compound library.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> MOA studies revealed that it was competitive with the peptide substrate and noncompetitive with respect to the cofactor SAM. In addition, <b>20</b> was selective for SETD8 over 15 other methyltransferases. However, compound <b>20</b> displayed weak binding affinity with SETD8, as evidenced by isothermal titration calorimetry (ITC) and surface plasmon resonance (SPR), with <i>K</i><sub>D</sub> values of 18.3 and 36.0 Î¼M, respectively. Further optimization of compound <b>20</b> by replacement of the 7-methoxy group with a 7-aminoethoxy group afforded compound <b>21</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) with improved inhibitory potency (IC<sub>50</sub> = 1.9 Î¼M) and enhanced binding affinity to SETD8 (ITC <i>K</i><sub>D</sub> = 1.3 Î¼M).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Analysis of the cocrystal structure of <b>21</b> in complex with SETD8 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) revealed that the quinazoline group occupies the substrate binding pocket, consistent with the MOA results that <b>21</b> is competitive with the peptide substrate but noncompetitive with SAM. More interestingly, Cys311 of SETD8 was found to be close to the pyrrolidinopentyl group of <b>21</b>, providing an opportunity to develop covalent inhibitors of SETD8. Therefore, compound <b>22</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) was designed to modify Cys311 specifically by addition of an electrophilic acrylamide group to C4 of the quinazoline via a propyl linker. Compound <b>22</b> had an in vitro IC<sub>50</sub> of 804 nM and was selective for SETD8 over 28 other methyltransferases. Mass spectrometry analysis revealed that <b>22</b> covalently modified SETD8, yielding an adduct with mass equal to SETD8 plus <b>22</b>, but did not react with the SETD8 C311S mutant, demonstrating that <b>22</b> formed a covalent bond specifically with Cys311. <b>22</b> was unable to covalently modify other PKMTs such as PRC2, SMYD2, and SMYD3, further indicating its selectivity for SETD8. The cocrystal structure of compound <b>22</b> in complex with SETD8 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) clearly established that a covalent bond is formed between the acrylamide group of <b>22</b> and the sulfur atom of Cys311. Unlike the cocrystal structure of the <b>21</b>âSETD8 adduct, <b>22</b> moves out from the active site of SETD8 where it is attached, suggesting that the adduct is unstable in solution. Compound <b>22</b> still lacked cellular activity in suppressing the H4K20me1 mark of SETD8, mainly because of its poor membrane permeability (4.4 nm/s) and high efflux ratio. Upon further optimization of compound <b>22</b> by replacing its 7-methoxy group with a 7-aminoethoxy group, compound <b>23</b> was identified with the desired potency.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Its cocrystal structure in complex with SETD8 revealed a covalent bond similar to that observed with compound <b>22</b>. More interestingly, a hydrogen bond was formed between the newly incorporated amine group and the carbonyl of Asn339, which may contribute to the enhanced inhibitory activity against SETD8.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.3.  Targeting Arginine Methyltransferase PRMT5 with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">PRMTs catalyze the transfer of a methyl group from SAM to the guanidinium nitrogen atoms of arginine residues, releasing 1 equiv of <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH). In humans, the PRMTs can be classified into three types according to the main forms of methylated arginine: type I (PRMT1, PRMT2, PRMT3, PRMT4, PRMT6, and PRMT8), which catalyze the formation of Ï-<i>N</i><sup>G</sup>-monomethylarginines (MMAs) and Ï-<i>N</i><sup>G</sup>,<i>N</i><sup>G</sup>-asymmetric dimethylarginines (aDMAs); type II (PRMT5 and PRMT9), which catalyze the formation of Ï-<i>N</i><sup>G</sup>-MMAs and Ï-<i>N</i><sup>G</sup>,<i>N</i>â²<sup>G</sup>-symmetric dimethylarginines (sDMAs); and type III (PRMT7), which is only known to catalyze the formation of Ï-<i>N</i><sup>G</sup>-MMA.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><div class="NLM_p">Human PRMT family members share a conserved SAM-dependent methyltransferase domain that consists of the SAM binding site and the substrate binding site. However, they possess different motif structures outside of the methyltransferase domain. PRMT1 and PRMT6 contain the catalytic methyltransferase domain only, whereas PRMT2, PRMT3, PRMT4, PRMT5, PRMT8, and PRMT9 all have an N-terminal motif preceding the catalytic domain. Both PRMT7 and PRMT9 contain a duplicated methyltransferase domain, although the C-terminal domain in PRMT9 is suggested to be a pseudodomain. In general, type I PRMTs share higher sequence homology with each other, and PRMT9 is closer to type I in the primary sequence, while PRMT5 and PRMT7 are the most distant members of the family and are also quite different from each other.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">In PRMT5, a triosephosphate isomerase (TIM) barrel precedes the methyltransferase domain and forms a stable complex with MEP50, a WD domain containing cytosolic protein. Analysis of the cocrystal structure of the PRMT5âMEP50 complex revealed that the catalytic domain of PRMT5 adopts a canonical arginine methyltransferase tertiary structure, with a SAM binding domain containing the nucleotide binding Rossmann fold followed by a Î²-sandwich structure involved in substrate binding.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> As a major sDMA methyltransferase, PRMT5 is able to symmetrically dimethylate a wide spectrum of substrates, including histones and non-histone proteins. Histone substrates include H2A residue Arg3 (H2AR3), H4 residue Arg3 (H4R3), and H3 residues Arg2 (H3R2) and Arg8 (H3R8), which are associated with the regulation of gene transcription. A growing list of non-histone proteins have also been identified as PRMT5 substrates, including Sm proteins, nucleolin, RAF proteins, EGFR, the tumor suppressor p53, androgen receptor (AR), and others, whose functions vary from cell signaling to differentiation, mRNA splicing, etc.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Overexpression of PRMT5 has been observed in several human malignancies, including glioblastoma, leukemia/lymphoma, prostate cancer, and colorectal carcinoma, and is also associated with poor prognosis.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> In the past decade, PRMT5 has attracted increasing attention as an anticancer target.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div><div class="NLM_p">PRMT5 has two binding pockets, a SAM binding pocket and a substrate peptide binding pocket, which provide two options for the development of PRMT5 inhibitors. Indeed, many efforts have been made to discover PRMT5 inhibitors by targeting one of these two pockets.<a onclick="showRef(event, 'ref49 ref52 ref53 ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref49 ref52 ref53 ref54 ref55 ref56 ref57">(49,52â57)</a> Compound <b>24</b> (EPZ015666) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) is the first reported highly potent and orally bioavailable PRMT5 inhibitor, with an IC<sub>50</sub> of 22 nM in a biochemical assay.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Kinetic experiments showed that it was competitive with the peptide substrate (<i>K</i><sub>i</sub> = 5 Â± 0.3 nM) and uncompetitive with SAM. The cocrystal structure of <b>24</b> bound to the PRMT5âMEP50 complex (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61">4X61</a>) confirmed its binding in the substrate binding pocket.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a><b>24</b> represents a validated chemical probe for further study of PRMT5 biology and arginine methylation in cancer. Encouragingly, the optimized analogue <b>25</b> (GSK3326595 or EPZ015938) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) has already been advanced into a phase 1 clinical trial to assess its safety, pharmacokinetics (PK), and pharmacodynamics (PD) in patients with solid tumors and non-Hodgkin lymphoma (NCT02783300), myelodysplastic syndrome (MDS), and AML (NCT03614728).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures of compounds <b>24</b>â<b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the other hand, potent PRMT5 inhibitors occupying the SAM binding pocket were also developed, represented by <b>26</b> (LLY-283) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> As a SAM mimetic with the methionine moiety replaced by a phenyl group, <b>26</b> showed potent PRMT5 enzyme inhibitory activity, with an IC<sub>50</sub> value of 20 nM, and exhibited more than 100-fold selectivity for PRMT5 over other methyltransferases and non-epigenetic targets. The cocrystal structure of the PRMT5âMEP50 complex with <b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A) provided a structural basis for its selectivity and high potency and showed that the phenyl group of <b>26</b> occupies the position of the side chain of Phe327, a residue specific to PRMT5. Two more SAM mimetics, <b>27</b> (JNJ64619178) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>)<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> and PF-06939999<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> (structure not disclosed) have been advanced into clinical trials (NCT03573310 and NCT03854227, respectively) in patients with multiple cancer types.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Cocrystal structure of <b>26</b> in complex with PRMT5 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CKC">6CKC</a>). (B) Cocrystal structure of <b>29</b> in complex with PRMT5 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6K1S">6K1S</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although significant progress has been made in developing inhibitors of PRMT5 by targeting its SAM binding pocket, it should be noted that they suffer from competition with high concentrations of SAM in cells, a factor known to reduce in vivo efficacy. Irreversible binding of a covalent inhibitor to PRMT5 may reduce competition with SAM to improve its cellular activity. On the basis of this hypothesis, Luengo and co-workers from Preclude Therapeutics initiated a program to develop potent and selective covalent PRMT5 inhibitors using <b>26</b> as a starting point.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Further analysis of the cocrystal structure of <b>26</b> in complex with PRMT5 revealed that Cys449 in the SAM binding pocket is close to the amino group of <b>26</b>, providing a clue for the design and discovery of covalent PRMT5 inhibitors. On the other hand, Cys449 is unique to PRMT5 and corresponds to a serine residue in other PRMTs. The development of covalent inhibitors targeting PRMT5 Cys449 may not only improve their inhibitory activity but also enhance the specificity for PRMT5. Therefore, <b>26</b> was modified to hemiaminal <b>28</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), which is proposed to be converted to aldehyde <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) under physiological conditions. This then reacts with Cys449 to form a covalent adduct. Compared with <b>26</b>, hemiaminal <b>28</b> showed improved enzymatic inhibitory potency (IC<sub>50</sub> = 11 nM), and aldehyde <b>29</b> displayed comparable activity (IC<sub>50</sub> = 20 nM). More importantly, <b>28</b> was selective for PRMT5 over PRMT1 and PRMT4. Both compounds inhibited sDMA of SmD3 in Granta-519 cells in a dose-dependent manner, with IC<sub>50</sub> values of 12 and 22 nM, respectively. Furthermore, <b>28</b> and <b>29</b> demonstrated concentration-dependent antiproliferative activities in Granta-519 cells, with IC<sub>50</sub> values of 60 and 48 nM, respectively. The cocrystal structure of <b>29</b> bound to the PRMT5âMEP50 complex (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B) clearly showed that a vinyl thioether bond is formed between the thiol group of Cys449 and the aldehyde of compound <b>29</b>, followed by elimination of a molecule of H<sub>2</sub>O. In summary, the aldehyde was used to react with Cys449 of PRMT5, which provides a novel strategy to form covalent adducts. However, aldehydes are subject to metabolizing enzymes, such as aldehyde oxidases, cytochrome P450 enzymes, aldehyde dehydrogenases, etc. Therefore, the drug-like property of an aldehyde as a warhead to develop covalent inhibitors needs further evaluation. In addition, it should be noted that PRT534 (structure not disclosed),<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> a novel PRMT5 inhibitor developed by the same group, is being evaluated in a phase I clinical trial in patients with advanced cancers (NCT03886831), but whether it is a covalent inhibitor has not been disclosed.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.4.  Targeting Arginine Methyltransferase PRMT1 with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">PRMT1 is the predominant type I arginine methyltransferase and contributes to more than 90% of the global aDMA deposition in human cells.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Although the molecular basis for PRMT1 substrate recognition is less studied, a range of substrates have been identified, including H4R3, the DNA repair protein MRE11, p53 binding protein 1 (53BP1), ASH2L, the tumor suppressor BRCA1, and the transcription factor forkhead box O1 (FOXO1). In view of the remarkable diversity of substrate proteins, PRMT1 is implicated in a multitude of cellular processes, including gene transcription, DNA damage repair, RNA processing, and signal transduction.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> However, PRMT1 dysregulation is correlated with various types of human cancers. In addition, overexpression of PRMT1 has been shown to promote cancer proliferation and metastasis.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Furthermore, methylation of Arg198/200 in the EGFR extracellular domain by PRMT1 confers resistance to the EGFR monoclonal antibody cetuximab in triple-negative breast cancer (TNBC), and inhibition of PRMT1 sensitizes TNBC cells to cetuximab.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> Therefore, the development of selective and potent PRMT1 inhibitors would be highly desirable.</div><div class="NLM_p">Unlike the nonprocessive mechanism of PRMT5, which methylates two different guanidinium nitrogen atoms of the arginine residue symmetrically, PRMT1 asymmetrically dimethylates the same guanidinium nitrogen atom of the arginine in a processive mechanism.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Therefore, several potent PRMT1 inhibitors were designed to compete with the second methylation step of the arginine residue after the guanidine terminal nitrogen is monomethylated. These compounds are characterized by the presence of the ethylenediamino group, which mimics the conformation of the arginine substrate side chain, such as compounds <b>30</b>â<b>33</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).<a onclick="showRef(event, 'ref72 ref73 ref74 ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref72 ref73 ref74 ref75 ref76">(72â76)</a> For example, <b>30</b> displayed significant inhibitory activity against PRMT1 (IC<sub>50</sub> value = 30 nM) but was also highly potent for other type I PRMTs such as PRMT3 (IC<sub>50</sub> = 119 nM), PRMT4 (IC<sub>50</sub> = 83 nM), PRMT6 (IC<sub>50</sub> = 4 nM), and PRMT8 (IC<sub>50</sub> = 5 nM).<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> However, it was selective for type I PRMTs over all type II and type III PRMTs, 25 PKMTs, and other epigenetic regulators. Interestingly, replacing the terminal amino group of <b>30</b> with a hydroxyl group resulted in completely abolished inhibitory activity against all type I PRMTs, further establishing the initial design concept for ethylenediamino derivatives. Compound <b>33</b>, a close analogue of <b>30</b>, is also a pan-type I PRMT inhibitor and is currently in a phase 1 clinical trial (NCT03666988) for solid tumors and diffuse large B-cell lymphoma (DLBCL).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of compounds <b>30</b>â<b>37</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As discussed above, the reported reversible PRMT1 inhibitors lack selectivity across type I PRMT members and remain to be improved. On the other hand, a hyper-reactive cysteine (Cys101) is observed within the SAM binding pocket of PRMT1, which is absent in other PRMTs except PRMT8, providing an opportunity for the development of selective covalent PRMT1 inhibitors. Therefore, a high-throughput FP assay was developed by Mowen and co-workers to identify covalent PRMT1 inhibitors targeting the Cys101 residue.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Fortunately, the two novel compounds <b>34</b> and <b>35</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>) containing a nitroalkene fragment were identified in that campaign. Both <b>34</b> and <b>35</b> were specific for PRMT1 and PRMT8 over other SAM-dependent methyltransferases, presumably as a result of covalent modification of Cys101. It is important to note that both compounds lack thorough characterization in MOA and biochemical assays, thus limiting their use as chemical probes.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.5.  Targeting Arginine Methyltransferase PRMT6 with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">PRMT6 is the only known methyltransferase to methylate H3R2, a hallmark acting as a transcriptional repressor that antagonizes H3K4me3 catalyzed by MLL.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> In addition to H3R2, PRMT6 also methylates H3R42, H4R3, HIV-1 transactivator of transcription (HIV Tat), high-mobility group A protein 1a (HMGA1a), and DNA polymerase Î².<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> PRMT6 has been reported to regulate many cellular processes, including DNA repair, DNA replication, cell-cycle regulation, signal transduction, and transcription.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> However, overexpression or dysregulation of PRMT6 has also been shown to be implicated in diverse cancer types, including bladder, lung, melanoma, prostate, and colorectal cancer.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> Accordingly, PRMT6 has been regarded as a promising therapeutic target in anticancer drug discovery, and many efforts have been made to develop potent and selective inhibitors.<a onclick="showRef(event, 'ref73 ref82 ref83 ref84'); return false;" href="javascript:void(0);" class="ref ref73 ref82 ref83 ref84">(73,82â84)</a> Because of the highly conserved sequences as well as the conformational similarity of the methyltransferase domain shared among type I PRMTs, the reported PRMT6 inhibitors usually lack selectivity across type I PRMT members, as evidenced by the reversible inhibitors described in <a class="ref internalNav" href="#sec2_4" aria-label="section 2.4">section 2.4</a>. In 2016, Jin and co-workers reported the dual PRMT6 (IC<sub>50</sub> = 43 nM) and PRMT4 (IC<sub>50</sub> = 34 nM) inhibitor <b>36</b> (MS-049) (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>), which showed >30-fold selectivity over PRMT8 and >300-fold selectivity over PRMT1 and PRMT3 and without inhibition against PRMT5 and PRMT7.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> It also showed high selectivity against a panel of 37 other methyltransferases and demethylases and a broad range of non-epigenetic targets in two selectivity profiling panels (CEREP and PDSP). However, the development of specific PRMT6 inhibitors is still pressing and may provide valuable chemical probes for investigating the physiological and pathophysiological functions of PRMT6.</div><div class="NLM_p">Alignment of the PRMT protein sequences identified a unique Cys50 located in the substrate binding pocket of PRMT6, which provides a promising site for covalent inhibitor development. On the other hand, analysis of the cocrystal structure of PRMT6 in complex with <b>30</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A) revealed that the meta position of the phenyl ring is oriented toward the thiol group of Cys50 with a distance of 4.9 Ã, which can be bridged with four or five chemical bonds. Therefore, Jin and co-workers designed compound <b>37</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>) by adding an acrylamide group at the meta position of the phenyl group in <b>30</b>.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a><b>37</b> displayed much improved potency for PRMT6 in both biochemical and cellular assays compared with non-covalent inhibitor <b>30</b>. Kinetic studies demonstrated that <b>37</b> showed time-dependent activity against PRMT6 with a <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> value of 8.3 Ã 10<sup>4</sup> M<sup>â1</sup> s<sup>â1</sup>, suggesting the covalent inhibition mechanism. A mass-spectrometry-based assay revealed that compound <b>37</b> formed an adduct with PRMT6, while no adducts with other type I PRMTs could be observed under the same conditions. To precisely establish its binding mode with PRMT6, the cocrystal structure of <b>37</b> in complex with PRMT6 was determined and showed the formation of a CâS bond between the acrylamide group of <b>37</b> and the sulfur atom of Cys50 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B). Compound <b>37</b> was highly selective for PRMT6 over 22 PKMTs, three DNMTs, and type II and III PRMTs and exhibited moderate to good selectivity for PRMT6 over other type I PRMTs, with IC<sub>50</sub> values of 0.1, 3, 0.48, and 0.11 Î¼M, for PRMT1, PRMT3, PRMT4, and PRMT8, respectively. As the ethylenediamino group of <b>37</b> was still conserved in type I PRMT inhibitors as discussed above, intensive optimization of this group may further enhance the selectivity for PRMT6 over other type I PRMTs.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Cocrystal structure of <b>30</b> in complex with PRMT6 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R">5E8R</a>). (B) Cocrystal structure of <b>37</b> in complex with PRMT6 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">2.6.  Targeting DNA Methyltransferase with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">DNA methylation is a mammalian epigenetic mark catalyzed by DNMTs that involves the transfer of a methyl group from SAM to C-5 of cytosine in the context of cytosineâphosphateâguanine (CpG) islands. Five known mammalian DNMTs have been identified: DNMT1, DNMT2, DNMT3A, DNMT3B, and DNMT3L. Among them, DNMT1 methylates newly synthesized DNA strands to maintain DNA methylation, while DNMT3A and DNMT3B preferentially establish the initial CpG methylation pattern during embryonic development and cell differentiation as de novo methyltransferases. DNMT3L lacks the C-terminal catalytic domain but is required to stimulate the catalytic activity of DNMT3A/B by forming a complex. DNMT2 contains the catalytic domain but does not possess methyltransferase activity.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a></div><div class="NLM_p">Normal DNA methylation plays crucial roles in biological processes, such as gene expression control, genomic imprinting, cell differentiation, and embryonic development.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> However, aberrant DNA methylation associated with DNMT dysfunction has been implicated in many diseases, particularly cancer.<a onclick="showRef(event, 'ref88 ref89'); return false;" href="javascript:void(0);" class="ref ref88 ref89">(88,89)</a> For instance, DNMT1 or DNMT3A/B proteins have been found to be overexpressed in a variety of human cancers, such as prostate, hepatocarcinoma, colorectal, and breast cancers.,<a onclick="showRef(event, 'ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref90 ref91">(90,91)</a> In cancer cells, aberrant DNA methylation leads to global hypomethylation and promoter hypermethylation within CpG islands, resulting in genomic instability and inactivation of tumor suppressor genes. Inhibition of DNMTs can reactivate tumor suppressor genes and induce cancer cell differentiation and apoptosis.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a></div><div class="NLM_p">To date, two DNMT inhibitors, 5-azacytidine (<b>38</b>, azacitidine) and 5-aza-2â²-deoxycytidine (<b>39</b>, decitabine) (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>), have been approved by the FDA and European Medicines Agency (EMA) for the treatment of patients with MDS, AML, and chronic myelomonocytic leukemia (CMML).<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> MOA studies revealed that these inhibitors can be incorporated into DNA and then covalently trap DNMTs on DNA, leading to irreversible inhibition of these enzymes.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> As an example, after <b>38</b> is transported into cells by human concentrative nucleoside transporter-1, it is converted into 5-azacytidine-5â²-triphosphate by uridine cytidine kinase and then metabolized to 5-aza-2â²-deoxycytidine-5â²-triphosphate. As an analogue of 2â²-deoxycytidine-5â²-triphosphate, 5-aza-2â²-deoxycytidine-5â²-triphosphate functions as a substrate for the DNA replication machinery and is incorporated into DNA strands, where cytosine is displaced by azacytosine. Azacytosine-substituted DNA can also be recognized by DNMTs as a substrate. It is attacked at C-6 of the azacytosine ring by the catalytic cysteine residue of a DNMT to form a covalent CâS bond (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B). Then a methyl group is transferred from SAM to N-5 of the azacytosine ring. In the case of DNA containing cytosine (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A), which undergoes similar nucleophilic attack by DNMTs and subsequent methylation at C-5 of the cytosine ring, the CâS bond can be resolved by a Î²-elimination reaction to afford the methylated DNA and release the active DNMT. However, the CâS bond formed in the azacytosine-containing DNAâDNMT complex cannot be resolved through a Î²-elimination reaction because of the presence of N-5 of the azacytosine ring, resulting in covalent entrapment of the DNMT and blockage of its methyltransferase activity. In addition, the unresolved DNAâDNMT complex also impairs DNA function and induces the DNA damage response signaling pathway, leading to degradation of the trapped DNMT.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structures of compounds <b>38</b>â<b>45</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Trapping mechanism of azacytosine. (A) Scheme of DNA methylation. (B) Scheme of DNMT inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As a ribonucleoside, <b>38</b> can be incorporated into not only DNA but also RNA, whereas <b>39</b>, as a deoxyribonucleoside, can be incorporated only into DNA. Despite their potential in clinical response, both compounds suffer from poor bioavailability, chemical instability, low specificity, and high toxicity. To address these limitations, compounds <b>40</b> (zebularine) and <b>41</b> (SGI-110) (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>) were identified. <b>40</b> is a stable cytidine analogue with minimal cytotoxicity compared with <b>38</b> and <b>39</b>.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> MOA studies showed that when <b>40</b> is incorporated into DNA, it forms covalent complexes with DNMTs in a similar way as <b>38</b> and <b>39</b>.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> However, the covalent complexes of <b>40</b> with DNMTs are formed reversibly, in contrast to those of <b>38</b> and <b>39</b>. In addition, methylation of <b>40</b> was not observed, mainly because of the absence of the 4-amino group in <b>40</b>, which was thought to prevent activation of the cytosine C-5 and subsequent methylation. Such a lack of methylation instead decreased the dissociation of the DNAâDNMT complexes and further stabilized the binding of DNMTs to DNA. It is worth noting that a higher concentration of <b>40</b> was needed to achieve the same demethylation effect in cells as irreversible nucleoside analogues, hindering its clinical development. <b>41</b> was designed as a prodrug of <b>39</b> with the aim of enhancing its PK and PD properties.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> It was obtained by linking <b>39</b> to deoxyguanosine through a phosphodiester linker, which was supposed to prevent immediate deamination of <b>39</b> by cytidine deaminase. PK analysis revealed that <b>41</b> showed favorable profiles with a prolonged plasma half-life and lower peak plasma concentrations. Preclinical studies of <b>41</b> displayed that it was as effective as <b>39</b> on gene-specific and global hypomethylation both in vitro and in animal model systems.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a><b>41</b> is being evaluated in a range of clinical trials ranging from phase I to phase III in patients with myeloid malignancies or solid tumors.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a></div><div class="NLM_p">On the other hand, non-nucleoside DNMT inhibitors have also been developed by targeting the catalytic activity of these enzymes, as exemplified by compounds <b>42</b>â<b>45</b>.<a onclick="showRef(event, 'ref100 ref101 ref102 ref103 ref104 ref105 ref106'); return false;" href="javascript:void(0);" class="ref ref100 ref101 ref102 ref103 ref104 ref105 ref106">(100â106)</a> These compounds are not incorporated into DNA and function as non-covalent inhibitors of DNMTs, which are expected to show high selectivity for DNMTs and low side effects. Most of them bind to the catalytic pocket of DNMTs and are competitive with SAM or DNA. However, non-nucleoside inhibitors exhibit weaker cellular activity compared with nucleoside DNMT inhibitors, and none have been advanced into clinical trials, thus necessitating potency improvement.</div><div class="NLM_p last">Apart from the above-mentioned progress in the development of covalent small-molecule inhibitors targeting epigenetic writers, the natural product gambogenic acid was identified by Chen and co-workers<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> as a covalent inhibitor of enhancer of zeste homologue 2 (EZH2), which is the catalytic subunit of the polycomb repressive complex 2 (PRC2) complex and is responsible for the trimethylation of histone H3 at K27.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> As this compound inhibits multiple proteins and signaling pathways,<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> structure-based drug design remains to be performed to optimize its efficacy and selectivity against EZH2.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">3.  Targeting Epigenetic Readers with Covalent Small-Molecule Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain and extraterminal (BET) proteins are critical epigenetic readers<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> that recognize and bind acetylated lysine residues on histone tails through bromodomains (BDs). The BET family consists of four proteins: bromodomain-containing protein 2 (BRD2), BRD3, BRD4, and bromodomain testis-specific protein (BRDT). Each of these contains two tandem BDs, known as BD1 (N-terminus) and BD2 (C-terminus), with high similarity across family members.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> Under normal conditions, BET proteins are widely expressed and play critical roles in the regulation of gene transcription by recruiting transcriptional regulatory complexes to acetylated chromatin.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> However, dysregulation of BET proteins has been implicated in a variety of human cancers.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Inhibition of the interactions between BET proteins and acetylated lysine residues can block the transcription of target genes to induce cell apoptosis and represents a promising strategy for cancer treatment.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a></div><div class="NLM_p">Diverse small-molecule BET inhibitors have been developed to prevent the recognition interaction between BDs and acetylated histones and have advanced into different phases of human clinical trials, including compounds <b>46</b>â<b>55</b> (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>).<a onclick="showRef(event, 'ref115 ref116 ref117 ref118 ref119 ref120 ref121 ref122 ref123'); return false;" href="javascript:void(0);" class="ref ref115 ref116 ref117 ref118 ref119 ref120 ref121 ref122 ref123">(115â123)</a> As an example, compound <b>46</b> is the first reported BET inhibitor with a triazolodiazepine core structure to enter clinical trials. <b>46</b> occupies the acetylated lysine binding site within the BDs of BET proteins and thereby inhibits the binding of acetylated H4 to BET proteins with IC<sub>50</sub> values ranging from 92 to 112 nM, but it exhibits no inhibitory activity toward non-BET-family BDs. Like <b>46</b>, compounds <b>47</b>â<b>54</b> bind all eight BDs with similar affinities and are usually regarded as dual-bromodomain inhibitors. To better understand the contribution of each domain to therapeutic efficacy and improve the safety profile of their inhibitors, the development of BET-domain-selective inhibitors is highly desirable. Interestingly, compound <b>55</b> is reported to exhibit a selectivity of approximately 170-fold for BD2 over BD1.<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> Apart from the development of BD2-selective BET inhibitors,<a onclick="showRef(event, 'ref124 ref125 ref126 ref127'); return false;" href="javascript:void(0);" class="ref ref124 ref125 ref126 ref127">(124â127)</a> the discovery of BD1-selective BET inhibitors has also attracted dramatic attention and recently been reported.<a onclick="showRef(event, 'ref128 ref129 ref130 ref131 ref132'); return false;" href="javascript:void(0);" class="ref ref128 ref129 ref130 ref131 ref132">(128â132)</a> Despite the progress that has been made, the modest domain selectivity of these inhibitors limits their potential use in functional studies of the BD1 and BD2 domains.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Structures of compounds <b>46</b>â<b>58</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2018, Kharenko and co-workers from Zenith Epigenetics reported the design of covalent inhibitors of BRD4 BD1, aimed at improving the potency and domain selectivity.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> Compound <b>56</b> (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>), which is a potent BRD4 inhibitor with IC<sub>50</sub> values of 0.23 and 0.08 Î¼M for BD1 and BD2, respectively, was used as a starting parent non-covalent inhibitor. Analysis of the cocrystal structure of <b>56</b> in complex with BRD4 BD1 (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>A) revealed that although there is not a cysteine residue located close to the inhibitor, there is a methionine residue (Met149) adjacent to the benzyl group of <b>56</b> that may be used for the design of covalent BRD4 inhibitors. Inspired by the concept of epoxides with high reactivity toward methionine,<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> Kharenko and co-workers synthesized epoxide <b>57</b> (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>), in which an epoxide warhead is attached to the benzyl group of <b>56</b>, aimed at covalent targeting of Met149. The covalent inhibition of compound <b>57</b> against BRD4 BD1 was confirmed by a thermal shift assay, MALDI-TOF mass spectrometry, and the X-ray cocrystal structure. In the protein thermal denaturation assay, which measured the interaction of the inhibitor with BRD4 under various pH and temperature conditions, incubation of BRD4 BD1 with <b>57</b> for 30 min resulted in two distinct thermal transitions with Î<i>T</i><sub>m</sub> values of 7 and 22 Â°C, and the lower <i>T</i><sub>m</sub> was converted to the higher value after 4 h of incubation. In contrast, only one thermal transition with Î<i>T</i><sub>m</sub> = 5 Â°C was observed for <b>56</b> under the same conditions. The data are agreement with a two-step process of covalent inhibition by <b>57</b> in which a reversible proteinâinhibitor complex with a lower <i>T</i><sub>m</sub> is formed, followed by covalent bond formation with a higher <i>T</i><sub>m</sub>. On the other hand, incubation of BRD4 BD2 with <b>57</b> led to only one thermal transition with a <i>T</i><sub>m</sub> shift comparable to that of <b>56</b>, indicating that <b>57</b> covalently binds to BRD4 BD1. The thermal denaturation assay of <b>57</b> with non-BET bromodomain proteins carrying methionine at the position corresponding to Met149 within BRD4 BD1 established its selectivity for the BET proteins. In addition, MALDI-TOF mass spectrometry analyses demonstrated that a peak corresponding to the adduct with a mass equal to BRD4 BD1 plus <b>57</b> was observed. Furthermore, the X-ray cocrystal structure of BRD4 BD1 in complex with <b>57</b> (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>B) clearly showed that the epoxide group forms a covalent bond with the Î³-sulfur atom of Met149, as predicted. More interestingly, compound <b>58</b> (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>), the 3-fluoro analogue of <b>57</b>, displayed durable transcriptional and antiproliferative activity in AML MV4;11 based assays. To summarize, unlike canonical covalent inhibitors, which usually form covalent bonds with cysteine residues, this is the first study to design covalent inhibitors targeting methionine, offering a novel approach to covalent drug discovery. Unlike cysteine, the sulfur atom of methionine is not highly nucleophilic, and practical warheads targeting methionine besides epoxide remain to be extensively investigated. Furthermore, it is noteworthy that a BET inhibitor developed by Zenith Epigenetics, ZEN-3694 (structure not disclosed), is being evaluated in human clinical trials;<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a> whether it is a covalent inhibitor has not been disclosed.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. (A) Cocrystal structure of <b>56</b> in complex with BRD4 BD1 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZV">6CZV</a>). (B) Cocrystal structure of <b>57</b> in complex with BRD4 BD1 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZU">6CZU</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">4.  Targeting Epigenetic Erasers with Covalent Small-Molecule Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51087" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51087" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein lysine methylation can be reversed by lysine demethylases (KDMs), which are classified into two families according to their sequence conservation and catalytic MOA: the amine oxidase KDM1 family and the much larger jumonji-C (Jmjc) domain-containing lysine demethylase (Jmjc-KDM) family.<a onclick="showRef(event, 'ref8 ref136'); return false;" href="javascript:void(0);" class="ref ref8 ref136">(8,136)</a> Enzymes in the KDM1 family, including histone lysine-specific demethylase 1 (LSD1, also known as KDM1A) and LSD2 (also known as KDM1B), specifically remove a methyl group from mono- or dimethylated H3K4 and H3K9 (H3K4me1/2 and H3K9me1/2) in a flavin adenine dinucleotide (FAD)-dependent way. The Jmjc-KDMs, consisting of the KDM2 to KDM8 subfamilies as well as jumonji domain-containing protein 6 (JMJD6), catalyze the demethylation of mono-, di-, and trimethylated lysine residues through an Fe(II)/2-oxoglutarate (2-OG)-dependent oxidative mechanism. In this section, the reported covalent small-molecule inhibitors targeting KDMs are summarized.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">4.1.  Targeting Lysine Demethylase LSD1 with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">As the first KDM discovered, in 2004 by Shi and co-workers, LSD1 performs its demethylase activity using FAD as the cofactor.<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>, the catalytic process is initiated by hydride transfer from the <i>N</i><sup>Îµ</sup>-methyl group of methylated lysine onto FAD, resulting in the formation of unstable imine <b>I</b> and FADH<sup>â</sup>.<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> It is worth noting that the lone pair of electrons on the Îµ-N atom of lysine is required to motivate the hydride transfer, and therefore, the quaternary amine of trimethylated lysine (Kme3) is not suitable for this catalytic cascade. Subsequently, imine intermediate <b>I</b> is hydrated to afford N,O-hemiacetal <b>II</b>, which is cleaved to give demethylated lysine <b>III</b> and formaldehyde, and FADH<sup>â</sup> gets one more proton from water to afford FADH<sub>2</sub>. Further oxidization of FADH<sub>2</sub> by oxygen regenerates FAD for the next catalytic cycle.</div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. FAD-dependent histone demethylation by LSD1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">LSD1 plays distinct roles in regulation of gene transcription according to the various protein complexes that it forms. For instance, LSD1 is able to form a heterodimeric complex with a corepressor for RE1-silencing transcription factor (CoREST), which functions as a transcriptional repressor through demethylation of the transcriptional-activation-associated marks H3K4me1 and H3K4me2.<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> On the other hand, LSD1 can also bind to the androgen receptor (AR) and the estrogen receptor (ER) and then act as a coactivator of transcription through demethylation of the repressive marks H3K9me1 and H3K9me2.<a onclick="showRef(event, 'ref140 ref141'); return false;" href="javascript:void(0);" class="ref ref140 ref141">(140,141)</a> Aside from H3K4me1/2 and H3K9me1/2, LSD1 also demethylates non-histone substrates, including p53, signal transducer and activator of transcription 3 (STAT3), E2F transcription factor 1 (E2F1), and DNMTs.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> LSD1 and its downstream proteins are involved in a wide range of biological functions, such as embryonic development, stem cell maintenance, and differentiation. However, overexpression of LSD1 is reported to be implicated in a variety of human cancers, including AML, lymphoid neoplasm, ER-negative breast, colon, prostate, lung, and gastric cancers.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> Hence, LSD1 has been validated as a very attractive cancer therapeutic target, and dramatic efforts have been made to discover its inhibitors.<a onclick="showRef(event, 'ref144 ref145 ref146 ref147'); return false;" href="javascript:void(0);" class="ref ref144 ref145 ref146 ref147">(144â147)</a></div><div class="NLM_p">As a member of the FAD-dependent amine oxidase superfamily, LSD1 shares sequence homology and structural similarity with monoamine oxidases A and B (MAOA and MAOB), which are responsible for oxidative deamination of neurotransmitters such as dopamine and serotonin.<a onclick="showRef(event, 'ref148'); return false;" href="javascript:void(0);" class="ref ref148">(148)</a> Therefore, previously discovered MAO inhibitors were screened against LSD1 by Shiekhattar et al., and compound <b>59</b> (TCP) (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>), a MAO inhibitor used as an antidepressive drug, was identified to exhibit the best inhibitory activity toward LSD1, with an IC<sub>50</sub> value of <2 Î¼M.<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> As <b>59</b> is an irreversible inhibitor of MAOs via covalent modification of the cofactor FAD, a detailed study of its mechanism against LSD1 was conducted.<a onclick="showRef(event, 'ref150'); return false;" href="javascript:void(0);" class="ref ref150">(150)</a> For instance, kinetic analysis revealed that <b>59</b> inhibited LSD1 in a time- and concentration-dependent manner, indicating an irreversible covalent inhibition mechanism. UV absorbance spectroscopy analysis further demonstrated that FAD was covalently modified by <b>59</b>, as evidenced by the fact that the characteristic UV absorbance spectrum of FAD with two peaks (350 and 500 nm) was resolved into a single absorbance peak (415 nm) upon treatment with <b>59</b>. Furthermore, the adduct (MW = 920 Da) of <b>59</b> (MW = 133 Da) with FAD (MW = 787 Da) in LSD1 was isolated and verified by mass spectrometry. It should be noted that a minor peak with a mass of 902 Da was also observed, corresponding to dehydration of the <b>59</b>âFAD adduct. On the basis of the above analysis, a single-electron-transfer mechanism was proposed for the formation of the adduct. As shown in <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>, an electron from the primary amine of <b>59</b> is transferred to FAD, resulting in a cation radical (<a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>, pathway 1), which undergoes opening of the cyclopropyl ring to yield stable benzylic radical <b>65</b>. Conjunction of the benzylic radical with C-4a of FAD forms iminium ion <b>66</b>, which is hydrolyzed to afford aldehyde <b>67</b>. Then the aldehyde group condenses with N-5 of FAD to generate hemiaminal <b>68</b> containing a five-membered ring, dehydration of which produces compound <b>69</b> with a mass of 902 Da. It is worth noting that the covalent adduct between <b>59</b> and FAD in LSD1 is distinct from that between <b>59</b> and FAD in MAOB, which is proposed to be compound <b>71</b> (<a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>, pathway 2). To further understand the inhibitory mechanism of <b>59</b> against LSD1, the crystal structure of <b>59</b>-modified FAD in the LSD1âCoREST complex was determined by Yu and co-workers at 2.75 Ã resolution (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2UXX">2UXX</a>). This revealed that the hemiaminal fitted well with this electron density, while compound <b>70</b> did not fit at all.<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> It is interesting to note that compound <b>59</b> covalently inhibits LSD via formation of an adduct with the cofactor FAD, which is different from the generally defined covalent inhibitors, which form adducts with the target proteins. This brings fresh insight into the concept of targeted covalent inhibition. However, it is also important to note that FAD is a common cofactor in proteins and participates in many important chemical reactions in metabolism. The selectivity issue should be considered when covalent inhibitors are designed to form adducts with FDA.</div><figure id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Structures of compounds <b>59</b>â<b>64</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Catalytic mechanisms for LSD1 and MAOB inhibition with <b>59</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because <b>59</b> displayed moderate inhibitory potency and low selectivity against LSD1, significant efforts have been devoted to developing more potent and selective LSD1 inhibitors. Several promising candidates were identified via modification of the primary amine or phenyl group of <b>59</b>, including compounds <b>60</b>â<b>64</b> (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>), which have been advanced into clinical trials.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a> Compound <b>60</b> was designed by incorporating a 4-aminocyclohexyl group into the primary amine of <b>59</b>, aimed at improving the inhibitory potency and selectivity against LSD1. It exhibited stronger LSD1 inhibitory activity (IC<sub>50</sub> = 18 nM) and higher selectivity (more than 1000-fold over LSD2 and MAOA/B) than <b>59</b>.<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a> A binding kinetics analysis revealed that it bound rapidly and irreversibly to the FAD cofactor in LSD1 in a manner similar to that of <b>59</b>. <b>60</b> was able to induce the accumulation of H3K4me2 on LSD1 target genes and cause blast differentiation and reduce the leukemic stem cell capacity in THP-1 cells in a time- and concentration-dependent manner. The proliferation and colony formation of a panel of AML cell lines were significantly inhibited by <b>60</b>, especially in the THP1 and MV4;11 cell lines. <b>60</b> was also orally bioavailable and well-tolerated in mice. An in vivo efficacy study demonstrated that it reduced MV4;11 tumor growth in a mouse xenograft model and increased survival in a PDX model of T cell acute leukemia. Compound <b>60</b> was the first covalent LSD1 inhibitor to be advanced into a clinical trial in patients with relapsed or refractory acute leukemia (EudraCT 2013-002447-29), which revealed that its recommended dose was well-tolerated. One more phase I clinical trial involving <b>60</b> for relapsed, extensive-stage disease small cell lung cancer (SCLC) treatment was completed recently (NCT02913443), but the results were not posted.</div><div class="NLM_p">Compound <b>61</b> (GSK2879552) (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>) is another cyclopropylamine-containing LSD1 inhibitor discovered by Kruger and co-workers.<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> Various assays, such as kinetic characterization, characteristic UV absorbance, and the cocrystal structure, established covalent inhibition of LSD1 by <b>61</b> as a mechanism-based irreversible inactivator. <b>61</b> displayed high selectivity for LSD1 over MAOA/B, a panel of kinases, G protein-coupled receptors (GPCRs), nuclear receptors (NRs), and ion channels and transporters. The antiproliferative effect of <b>61</b> was screened against a panel of 165 cancer cell lines, of which SCLC and AML were identified to be the sensitive cell lines, with EC<sub>50</sub> values ranging from 2 to 240 nM. In addition, DNA hypomethylation was identified as a predictive biomarker of sensitivity to LSD1 inhibition by <b>61</b> in SCLC cell lines and primary samples. A further in vivo efficacy study of <b>61</b> revealed that 57% and 83% TGI were achieved in NCL-H526 and NCI-H1417 xenograft mouse models, respectively, when <b>61</b> was dosed at 1.5 mg/kg daily. Two phase I clinical trials to investigate the safety, tolerability, PK, and PD profiles of <b>61</b> in AML (NCT02177812) and relapsed/refractory SCLC (NCT02034123) were initiated in 2014 but were terminated in 2019 because of severe adverse effects. A phase I trial of <b>61</b> alone or in combination with all-trans retinoic acid (ATRA) in patients with high-risk MDS (NCT02929498) was also terminated.</div><div class="NLM_p last">Compounds <b>62</b> and <b>63</b> are potent, selective, and orally available covalent inhibitors of LSD1 developed by Incyte Corporation and Imago Bio-Sciences, respectively. A range of clinical trials involving these compounds alone or in combination with other agents are being conducted for the treatment of advanced solid tumors and malignancies.<a onclick="showRef(event, 'ref144'); return false;" href="javascript:void(0);" class="ref ref144">(144)</a> Compound <b>64</b> (ORY-2001) (<a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>) is a dual inhibitor of LSD1 and MAOB developed by Oryzon Genomics.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a> Unlike the above-discussed LSD1 inhibitors, which have been mainly evaluated for leukemias and other cancers, <b>64</b> was reported to prevent the development of cognitive impairment in mouse models. Treatment with <b>64</b> could downregulate the expression of inflammatory genes, including S100a9, and regulate the expression of genes correlated with neuroplasticity and behavior in the SAMP-8 mouse brain. An in vivo efficacy study demonstrated that <b>64</b> was able to reduce aggression and increase social behavior in SAMP-8 mice.<a onclick="showRef(event, 'ref156'); return false;" href="javascript:void(0);" class="ref ref156">(156)</a> This inhibitor was advanced into a phase IIa clinical trial in patients with mild to moderate Alzheimerâs disease in 2019 (NCT03867253).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">4.2.  Targeting Histone Lysine Demethylase KDM5 with Covalent Small-Molecule Inhibitors</h3><div class="NLM_p">In contrast to LSDs, which are unable to demethylate trimethylated lysines, JmjC-KDMs are capable of demethylating all three lysine methylation states utilizing Fe(II) and 2-OG as cofactors.<a onclick="showRef(event, 'ref157'); return false;" href="javascript:void(0);" class="ref ref157">(157)</a> The demethylation mechanism is proposed to involve hydroxylation of the methyl group of methylated lysine residues under the conditions of oxygen, Fe(II), and 2-OG to form an unstable hemiaminal, which is then hydrolyzed to yield a demethylated lysine substrate and formaldehyde. Among the JmjC-KDMs, the KDM5 subfamily (also known as the Jumonji/AT-rich interactive domain-containing protein 1 (JARID1) subfamily) consists of four members (A to D), which possess a high degree of sequence and domain homology.<a onclick="showRef(event, 'ref158'); return false;" href="javascript:void(0);" class="ref ref158">(158)</a> The four isoforms of KDM5 can all demethylate H3K4me2/3, which is a ubiquitous transcription-activating mark. It is important to note that KDM5AâD show distinct tissue distributions and cellular localizations, implying their diverse cellular functions.<a onclick="showRef(event, 'ref159'); return false;" href="javascript:void(0);" class="ref ref159">(159)</a> However, overexpression or dysregulation of KDM5s has been implicated in a variety of human cancers. For instance, increased expression levels of KDM5A were observed in hepatocellular carcinoma, gastric cancer, and lung cancer.<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a> KDM5B is reported to be upregulated in breast, bladder, prostate, colorectal, and lung cancers and malignant melanoma and is required for derived cancer cell proliferation and survival.<a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a> Inactivating mutations of KDM5C seem to be correlated with the development of renal carcinoma.<a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a> Being 85% identical to KDM5C, KDM5D has been associated with prostate cancer.<a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a> Because of the growing evidence for their oncogenic function, KDM5 enzymes have been recognized as promising drug targets for cancer therapy.<a onclick="showRef(event, 'ref164'); return false;" href="javascript:void(0);" class="ref ref164">(164)</a></div><div class="NLM_p">Several small-molecule KDM5 inhibitors have been identified in the past 5 years, as exemplified by compounds <b>72</b>â<b>77</b> (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>).<a onclick="showRef(event, 'ref165'); return false;" href="javascript:void(0);" class="ref ref165">(165)</a> In MOA studies, these compounds inhibited KDM5s by competing with the cofactor 2-OG and coordinating the Fe(II) ion to form one or two metalâligand bonds. <b>72</b> was developed by the Classon group as a highly potent KDM5 inhibitor with IC<sub>50</sub> values of 10, 3, and 14 nM for KDM5A, KDM5B, and KDM5C, respectively.<a onclick="showRef(event, 'ref166'); return false;" href="javascript:void(0);" class="ref ref166">(166)</a> More interestingly, it exhibited around 200-fold and 770-fold selectivity for KDM5A over KDM4C and KDM7B, respectively, and no measurable inhibitory activity against KDM2B, KDM3B, or KDM6A. Treatment of HeLa cells with <b>72</b> resulted in a dose-dependent accumulation of H3K4me3. Its cocrystal structure in complex with KDM5A<sub>12â797</sub> (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH">5CEH</a>) demonstrated that <b>72</b> occupies the 2-OG binding site and that its nitrile group coordinates to the active-site Fe(II) ion, thereby confirming its competitive MOA. On the basis of the cocrystal structure, further modification of <b>72</b> resulted in the identification of <b>73</b>, in which the phenyl group was replaced with a 1-(<i>tert</i>-butyl)-1<i>H</i>-pyrazol-4-yl group.<a onclick="showRef(event, 'ref167'); return false;" href="javascript:void(0);" class="ref ref167">(167)</a> Compound <b>73</b> was also selective for KDM5s (IC<sub>50</sub> = 15 nM for KDM5A, 4.7 nM for KDM5B, and 65.5 nM for KDM5C) over KDM1A, KDM2B, KDM3B, KDM4C, KDM6A, and KDM7B. Cell-based assays measuring the H3K4Me3 level in PC3 cells revealed that compound <b>73</b> demonstrated more potent activity (EC<sub>50</sub> = 0.34 Î¼M) than <b>72</b> (EC<sub>50</sub> = 5.2 Î¼M).</div><figure id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. Structures of compounds <b>72</b>â<b>83</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compounds <b>74</b> and <b>75</b> were developed by Blagg and co-workers as dual inhibitors of KDM4 (IC<sub>50</sub> = 31 and 17 nM, respectively) and KDM5 (IC<sub>50</sub> = 23 and 14 nM, respectively).<a onclick="showRef(event, 'ref168'); return false;" href="javascript:void(0);" class="ref ref168">(168)</a> The cocrystal structure of compound <b>75</b> in complex with KDM4A (<a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>C) demonstrated that it binds to Fe(II) in the active site through bidentate coordination of the pyridyl and thiazolyl nitrogen atoms. However, both compounds suffer from poor selectivity across KDM5s and low cellular potency. Compound <b>76</b> was characterized by the Cheng group as a KDM5A inhibitor. This compound is a racemic mixture, with the pure <i>S</i> enantiomer displaying more potent binding affinity (<i>K</i><sub>D</sub> = 60 nM) than the <i>R</i> enantiomer (<i>K</i><sub>D</sub> = 220 nM).<a onclick="showRef(event, 'ref169'); return false;" href="javascript:void(0);" class="ref ref169">(169)</a> The cocrystal structure of the <i>S</i> enantiomer bound with KDM5A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>) demonstrated that compound <b>76</b> binds to the KDM5A active site and coordinates to Fe(II) via the pyridine atom. However, this compound has low cell permeability, mainly due to the strong;y acidic nature of the carboxylate moiety. <b>77</b> was a highly selective inhibitor of the KDM5 subfamily discovered by Brennan and co-workers that showed potent inhibitory activity against KDM5AâD with IC<sub>50</sub> values less than 100 nM.<a onclick="showRef(event, 'ref170'); return false;" href="javascript:void(0);" class="ref ref170">(170)</a> However, this compound displays weak cellular activity, with an EC<sub>50</sub> value of 50 Î¼M assessed in an immunofluorescence assay using an overexpression system, possibly as a result of its low competition with the cofactor and its moderate cell permeability.</div><figure id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. (A) Cocrystal structure of <b>79</b> in complex with KDM5A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA">6DQA</a>). (B) Cocrystal structure of <b>80</b> in complex with KDM5A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB">6DQB</a>). (C) Cocrystal structure of <b>75</b> in complex with KDM4A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F3I">5F3I</a>). (D) Cocrystal structure of <b>82</b> in complex with KDM5B (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIN">6EIN</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In efforts to identify more potent and selective KDM5 subfamily inhibitors, several research groups have applied covalent inhibition strategies to target KDM5s. Sequence alignment of Jmjc-KDMs revealed that Cys481 of KDM5A is invariant among the KDM5 subfamily (corresponding to Cys497 in KDM5B, Cys502 in KDM5C, and Cys512 in KDM5D) but is absent in other Jmjc-KDM subfamilies, providing an opportunity to develop more potent and selective KDM5 subfamily inhibitors. Cheng and co-workers designed and synthesized irreversible KDM5 inhibitors that form a covalent interaction with Cys481 of KDM5A.<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a> They first replaced the pyrrolo[3,2-<i>b</i>]pyridine moiety of compound <b>76</b> with a thieno[3,2-<i>b</i>]pyridine moiety to obtain compound <b>78</b>, which demonstrated binding affinity (<i>K</i><sub>D</sub> = 0.26 Î¼M) comparable to that of compound <b>77</b> (<i>K</i><sub>D</sub> = 0.15 Î¼M) as assessed by ITC. Further replacement of pyrrolidine with piperidine and 2-chlorophenyl with 3-aminophenyl afforded compound <b>79</b>, whose cocrystal structure bound to KDM5A (<a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>A) demonstrated that the primary amine group is located close to Cys481 with a distance of 4.5 Ã. Therefore, a reactive acrylamide group, as exemplified by the dimethylamino-2-butenamide moiety, was attached to the primary amine of <b>79</b> to afford compound <b>80</b>, whose cocrystal structure in complex with KDM5A (<a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>B) clearly showed that a covalent bond is formed between the sulfur atom of Cys481 and the 2-butenamide group of <b>80</b>. Kinetic experiments revealed that compound <b>80</b> was noncompetitive with the cofactor 2-OG and that the inhibition of KDM5A by the non-covalent compound <b>79</b> was dependent on the 2-OG concentration, resulting in approximately 50-fold enhanced inhibition against KDM5A for compound <b>80</b> compared with compound <b>79</b>. More interestingly, increased selectivity of <b>80</b> for the KDM5 subfamily over KDM4A was also achieved compared with compound <b>79</b>.</div><div class="NLM_p">As discussed above, Cys481 of KDM5A is conserved in the KDM5 subfamily and corresponds to Cys497 of KDM5B. Therefore, another series of covalent KDM5 subfamily inhibitors targeting Cys497 of KDM5B were also designed by Brennan and co-workers, as exemplified by compound <b>81</b>, which was generated by incorporating 2-chloroacetamide into the pyrazolyl moiety of compound <b>73</b> via an ethylamine linker.<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> Covalent binding of compound <b>81</b> with KDM5B was confirmed by mass spectrometry labeling experiments, and a proteinâinhibitor adduct in the mass spectra was detected after incubation of <b>81</b> with KDM5B. Digestion of the adduct and subsequent LCâMS/MS analysis suggested that Cys497 was covalently modified. The 2-OG competition assay demonstrated that <b>81</b> was significantly more potent than the non-covalent precursor <b>73</b> at 100 Î¼M 2-OG by a factor of 37, indicating that addition of covalent inhibition against KDM5s reduces competition with 2-OG. Compound <b>81</b> showed equal potency against KDM5A, KDM5B, and KDM5D, with IC<sub>50</sub> values of 10, 7, and 38 nM, respectively, but was 81-fold less active against KDM5C. More importantly, <b>81</b> exhibited more than 500-fold selectivity for KDM5B over KDM4A/B and 253-fold selectivity over KDM4C and displayed greater selectivity (800-fold) over KDM3A. It is worth noting that compound <b>81</b> did not inhibit KDM2A and KDM6A, even at 100 Î¼M.</div><div class="NLM_p last">On the other hand, Cys480 is identified as a unique residue in KDM5B and corresponds to Ser464 in KDM5A, Ser485 in KDM5C, and Ser495 in KDM5D, providing an opportunity to develop selective covalent inhibitors of KDM5B over KDM5A/C/D and KDM4s.<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> Therefore, a series of KDM5B-selective covalent inhibitors based on the core structure of compound <b>74</b> were designed, as exemplified by compounds <b>82</b> and <b>83</b> (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>). The cocrystal structure of compound <b>82</b> in complex with KDM5B (<a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>D) clearly demonstrated that a covalent bond was formed between the sulfur atom of Cys480 and the acrylamide of compound <b>82</b>. Covalent inhibition of KDM5B by compound <b>83</b> was confirmed by a 2-OG competition assay, which indicated that <b>83</b> was noncompetitive with 2-OG, in contrast to the non-covalent inhibitor <b>74</b>. Mass spectrometry labeling experiments further revealed that compound <b>83</b> covalently modified KDM5B but not KDM4B or KDM5C under the same conditions. Selectivity screening showed that compound <b>83</b> demonstrated around 50-fold selectivity over KDM4A/B/C, while the non-covalent compound <b>74</b> functioned as a dual KDM4 and KDM5 inhibitor. Although <b>83</b> demonstrated moderate selectivity for KDM5B over KDM5C (38-fold), it still showed high inhibitory activity against KDM5A/D, with IC<sub>50</sub> values of 30 and 83 nM, respectively, indicating that its selectivity needs further improvement.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">5.  Summary and Perspective</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A large amount of evidence has shown that epigenetic regulators play critical roles in the regulation of gene transcription and are involved in many normal cellular processes. However, a growing body of clinical research has also revealed that overexpression or dysregulation of epigenetic regulators is implicated in a variety of human diseases, especially cancers. Consequently, discovering selective and potent inhibitors of epigenetic regulators has attracted increasing attention, and remarkable progress has been made in the past decade. Two DNMT inhibitors and five HDAC inhibitors have been approved for clinical uses, and many epigenetic-targeted inhibitors are also undergoing clinical trials ranging from phase I to phase III. However, the development of selective and potent inhibitors of epigenetic regulators also faces several challenges, including conserved catalytic domains across the same type of enzyme family, competition with internal cofactors, and the occurrence of acquired resistance. These challenges promote researchers to identify effective strategies to facilitate epigenetic drug discovery.</div><div class="NLM_p">Inspired by the advantages of covalent small-molecule inhibitors<a onclick="showRef(event, 'ref173'); return false;" href="javascript:void(0);" class="ref ref173">(173)</a> and especially the success in the development of covalent TKI inhibitors,<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a> targeted covalent inhibition has attracted increasing interest in epigenetic drug discovery. As we summarized in <a class="ref internalNav" href="#sec2" aria-label="sections 2">sections 2</a>â<a class="ref internalNav" href="#sec4" aria-label="4">4</a>, diverse covalent small-molecule inhibitors have been developed by targeting epigenetic writers, readers, and erasers, respectively, such as meninâMLL covalent inhibitors <b>11</b>â<b>14</b>, SETD8 covalent inhibitor <b>23</b>, PRMT5 covalent inhibitors <b>28</b> and <b>29</b>, PRMT6 covalent inhibitor <b>37</b>, BET covalent inhibitors <b>57</b> and <b>58</b>, and KDM5 covalent inhibitors <b>80</b>â<b>83</b>. Most of these covalent inhibitors were developed in a process of rational drug design and demonstrated improved target selectivity and enhanced in vivo efficacy compared with their non-covalent counterparts. However, it is important to note that many of these promising covalent epigenetic inhibitors are still being evaluated at the early stage and that few covalent epigenetic drugs have been approved for clinical treatments, unlike covalent TKI inhibitors, seven of which have been approved by the FDA. Although DNMT covalent inhibitors <b>38</b> and <b>39</b> have been approved for use in treating leukemia, both compounds are incorporated into DNA and covalently trap DNMTs, resulting in low specificity and high toxicity. Extensive efforts are still needed to advance selective and potent covalent epigenetic inhibitors into clinical trials to demonstrate their advantages. In addition, targeted covalent inhibition of epigenetic regulators is still focused on forming covalent bonds with cysteine residues, except for BET and LSD1 covalent inhibitors, which covalently modify methionine residues and the FAD cofactor, respectively. Structure-based design of covalent epigenetic inhibitors targeting other unique nucleophilic residues across the same enzyme family, including lysine, tyrosine, serine, threonine, and histidine, may provide a broad scope of epigenetic targets that are amenable to covalent inhibition.<a onclick="showRef(event, 'ref175 ref176'); return false;" href="javascript:void(0);" class="ref ref175 ref176">(175,176)</a> On the other hand, targeted covalent inhibition depends not only on the nucleophilic residue but also on the electrophilic warhead choice. As reviewed above, most of the covalent epigenetic inhibitors targeting cysteine residues use an acrylamide warhead or its derivatives. The development of novel electrophilic warheads beyond the common acrylamides may facilitate the discovery of potent and selective covalent inhibitors of epigenetic regulators.<a onclick="showRef(event, 'ref177 ref178 ref179 ref180'); return false;" href="javascript:void(0);" class="ref ref177 ref178 ref179 ref180">(177â180)</a> Finally, since no selective covalent epigenetic drug has been approved to date, overcoming the acquired resistance of non-covalent epigenetic drugs with covalent drugs has been less studied. Hence, the advantages of covalent epigenetic inhibitors in combating acquired drug resistance have not been demonstrated, but progress is expected in the discovery and development of covalent epigenetic inhibitors.</div><div class="NLM_p last">In conclusion, we have comprehensively summarized the structure-based design and characterization of covalent inhibitors of epigenetic regulators. We have highlighted the advantages of covalent epigenetic inhibitors in achieving superior selectivity and efficacy compared with their non-covalent counterparts. Many more studies are anticipated to reveal the potential of covalent epigenetic inhibitors in overcoming or delaying acquired drug resistance. We hope that this Perspective will shed light on future work toward the development of highly potent and selective covalent inhibitors of epigenetic regulators.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02055" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanxiang Wang</span> - <span class="hlFld-Affiliation affiliation">Guangdong Key Laboratory of Chiral
Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5650-4396" title="Orcid link">https://orcid.org/0000-0001-5650-4396</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5423353a332d2c6d611439353d387a272d27217a3130217a373a"><span class="__cf_email__" data-cfemail="1562747b726c6d2c205578747c793b666c66603b7071603b767b">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi Zhang</span> - <span class="hlFld-Affiliation affiliation">Guangdong Key Laboratory of Chiral
Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deqin Rong</span> - <span class="hlFld-Affiliation affiliation">Guangdong Key Laboratory of Chiral
Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bingbing Li</span> - <span class="hlFld-Affiliation affiliation">Guangdong Key Laboratory of Chiral
Molecule and Drug Discovery, School of Pharmaceutical Sciences, Sun Yat-Sen University, Guangzhou 510006, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through the contributions of all authors. All of the authors approved the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">This review focuses on progress in the discovery and development of covalent small-molecule inhibitors targeting epigenetic regulators, and we apologize for arbitrary omissions.<br /></br>The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26173" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26173" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Yi Zhang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=BIO-d7e2652-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yi Zhang</b> received his Bachelorâs degree in Medicinal Chemistry from Sun Yat-Sen University (SYSU) and is currently pursuing his Masterâs degree in the School of Pharmaceutical Sciences at the same university under the supervision of Prof. Yuanxiang Wang. His research currently focuses on the development of covalent inhibitors of protein arginine methyltransferases.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Deqin Rong</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=BIO-d7e2657-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Deqin Rong</b> obtained his Bachelorâs degree in medicinal chemistry from Henan Normal University. He is focusing on the development of covalent inhibitors targeting protein lysine methyltransferases under the guidance of Prof. Yuanxiang Wang in the School of Pharmaceutical Sciences at SYSU.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Bingbing Li</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=BIO-d7e2662-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Bingbing Li</b> received her Bachelorâs degree in medicinal chemistry from Southern Medical University and is currently pursuing her Masterâs degree in the School of Pharmaceutical Sciences at SYSU under the supervision of Prof. Yuanxiang Wang. Her research interests focus on the development of covalent inhibitors of protein lysine demethylases.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Yuanxiang Wang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=BIO-d7e2667-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yuanxiang Wang</b> is currently an Associate Professor in the School of Pharmaceutical Sciences at SYSU. He received a Ph.D. degree in Medicinal Chemistry from Shanghai Institute of Materia Medica, Chinese Academy of Sciences, in 2012 and completed postdoctoral training at the University of Arizona and the University of Utah from 2012 to 2017. In 2017, he joined SYSU as a faculty member. His group currently focuses on developing potent and selective inhibitors targeting epigenetic regulators.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The research in our lab was supported by the National Natural Science Foundation of China (Grant 21977128), the Fundamental Research Funds for the Central Universities (20ykpy117), and the Natural Science Foundation of Guangdong (Grant 2018A030313300).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">DNMT</td><td class="NLM_def"><p class="first last">DNA methyltransferase</p></td></tr><tr><td class="NLM_term">HMT</td><td class="NLM_def"><p class="first last">histone methyltransferase</p></td></tr><tr><td class="NLM_term">HAT</td><td class="NLM_def"><p class="first last">histone acetyltransferase</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal</p></td></tr><tr><td class="NLM_term">HDM</td><td class="NLM_def"><p class="first last">histone demethylase</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">DUB</td><td class="NLM_def"><p class="first last">deubiquitinating enzyme</p></td></tr><tr><td class="NLM_term">TKI</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitor</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed-lineage leukemia</p></td></tr><tr><td class="NLM_term">WDR5</td><td class="NLM_def"><p class="first last">WD repeat-containing protein 5</p></td></tr><tr><td class="NLM_term">RBBP5</td><td class="NLM_def"><p class="first last">retinoblastoma-binding protein 5</p></td></tr><tr><td class="NLM_term">ASH2L</td><td class="NLM_def"><p class="first last">ASH2-like histone lysine methyltransferase complex subunit</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">ALL</td><td class="NLM_def"><p class="first last">acute lymphoblastic leukemia</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">proteinâprotein interaction</p></td></tr><tr><td class="NLM_term">MBM1</td><td class="NLM_def"><p class="first last">menin binding motif 1</p></td></tr><tr><td class="NLM_term">MBM2</td><td class="NLM_def"><p class="first last">menin binding motif 2</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">BMC</td><td class="NLM_def"><p class="first last">bone marrow cell</p></td></tr><tr><td class="NLM_term">SCID</td><td class="NLM_def"><p class="first last">severe combined immunodeficiency</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term">SETD8</td><td class="NLM_def"><p class="first last">SET domain-containing protein 8</p></td></tr><tr><td class="NLM_term">KMT5A</td><td class="NLM_def"><p class="first last">lysine methyltransferase 5A</p></td></tr><tr><td class="NLM_term">PCNA</td><td class="NLM_def"><p class="first last">proliferating cell nuclear antigen</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">CML</td><td class="NLM_def"><p class="first last">chronic myelogenous leukemia</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr><tr><td class="NLM_term">MMA</td><td class="NLM_def"><p class="first last">monomethylarginine</p></td></tr><tr><td class="NLM_term">aDMA</td><td class="NLM_def"><p class="first last">asymmetric dimethylarginine</p></td></tr><tr><td class="NLM_term">sDMA</td><td class="NLM_def"><p class="first last">symmetric dimethylarginine</p></td></tr><tr><td class="NLM_term">TIM</td><td class="NLM_def"><p class="first last">triosephosphate isomerase</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">MDS</td><td class="NLM_def"><p class="first last">myelodysplastic syndrome</p></td></tr><tr><td class="NLM_term">53BP1</td><td class="NLM_def"><p class="first last">p53 binding protein 1</p></td></tr><tr><td class="NLM_term">FOXO1</td><td class="NLM_def"><p class="first last">transcriptional factor forkhead box O1</p></td></tr><tr><td class="NLM_term">TNBC</td><td class="NLM_def"><p class="first last">triple-negative breast cancer</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">HIV Tat</td><td class="NLM_def"><p class="first last">HIV-1 transactivator of transcription</p></td></tr><tr><td class="NLM_term">HMGA1a</td><td class="NLM_def"><p class="first last">high-mobility group A protein 1a</p></td></tr><tr><td class="NLM_term">CpG</td><td class="NLM_def"><p class="first last">cytosineâphosphateâguanine</p></td></tr><tr><td class="NLM_term">EMA</td><td class="NLM_def"><p class="first last">European Medicines Agency</p></td></tr><tr><td class="NLM_term">CMML</td><td class="NLM_def"><p class="first last">chronic myelomonocytic leukemia</p></td></tr><tr><td class="NLM_term">EZH2</td><td class="NLM_def"><p class="first last">enhancer of zeste homologue 2</p></td></tr><tr><td class="NLM_term">PRC2</td><td class="NLM_def"><p class="first last">polycomb repressive complex 2</p></td></tr><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">RDT</td><td class="NLM_def"><p class="first last">bromodomain testis-specific protein</p></td></tr><tr><td class="NLM_term">KDM</td><td class="NLM_def"><p class="first last">lysine demethylase</p></td></tr><tr><td class="NLM_term">Jmjc-KDM</td><td class="NLM_def"><p class="first last">jumonji-C domain-containing lysine demethylase</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine-specific demethylase 1</p></td></tr><tr><td class="NLM_term">2-OG</td><td class="NLM_def"><p class="first last">2-oxoglutarate</p></td></tr><tr><td class="NLM_term">JMJD6</td><td class="NLM_def"><p class="first last">jumonji domain-containing protein 6</p></td></tr><tr><td class="NLM_term">CoREST</td><td class="NLM_def"><p class="first last">corepressor for RE1-silencing transcription factor</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">E2F1</td><td class="NLM_def"><p class="first last">E2F transcription factor 1</p></td></tr><tr><td class="NLM_term">MAOA</td><td class="NLM_def"><p class="first last">monoamine oxidase A</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">NR</td><td class="NLM_def"><p class="first last">nuclear receptor</p></td></tr><tr><td class="NLM_term">ATRA</td><td class="NLM_def"><p class="first last">all-trans retinoic acid</p></td></tr><tr><td class="NLM_term">JARID1</td><td class="NLM_def"><p class="first last">jumonji/AT-rich interactive domain-containing protein 1</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67180" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67180" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 180 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, E.</span></span> <span> </span><span class="NLM_article-title">Advances in epigenetics link genetics to the environment and disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>571</i></span>,  <span class="NLM_fpage">489</span>â <span class="NLM_lpage">499</span>, <span class="refDoi">Â DOI: 10.1038/s41586-019-1411-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fs41586-019-1411-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31341302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVeqs7rO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2019&pages=489-499&author=G.+Cavalliauthor=E.+Heard&title=Advances+in+epigenetics+link+genetics+to+the+environment+and+disease&doi=10.1038%2Fs41586-019-1411-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in epigenetics link genetics to the environment and disease</span></div><div class="casAuthors">Cavalli, Giacomo; Heard, Edith</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">571</span>
        (<span class="NLM_cas:issue">7766</span>),
    <span class="NLM_cas:pages">489-499</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Epigenetic research has accelerated rapidly in the twenty-first century, generating justified excitement and hope, but also a degree of hype.  Here we review how the field has evolved over the last few decades and reflect on some of the recent advances that are changing our understanding of biol.  We discuss the interplay between epigenetics and DNA sequence variation as well as the implications of epigenetics for cellular memory and plasticity.  We consider the effects of the environment and both intergenerational and transgenerational epigenetic inheritance on biol., disease and evolution.  Finally, we present some new frontiers in epigenetics with implications for human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGOerze2PSHLVg90H21EOLACvtfcHk0lhLZ8uTO9cQmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVeqs7rO&md5=726fa6d235e33685dbb7a7f350eff7ff</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1411-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1411-0%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DG.%26aulast%3DHeard%26aufirst%3DE.%26atitle%3DAdvances%2520in%2520epigenetics%2520link%2520genetics%2520to%2520the%2520environment%2520and%2520disease%26jtitle%3DNature%26date%3D2019%26volume%3D571%26spage%3D489%26epage%3D499%26doi%3D10.1038%2Fs41586-019-1411-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span> <i>Epigenetics</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span>, <span class="NLM_string-name">Jenuwein, T.</span>, <span class="NLM_string-name">Reinberg, D.</span></span>, Eds.; <span class="NLM_publisher-name">Cold Spring Harbor Laboratory Press</span>: <span class="NLM_publisher-loc">Cold Spring Harbor, NY</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=C.+D.+Allis&author=T.+Jenuwein&author=D.+Reinberg&title=Epigenetics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DEpigenetics%26aulast%3DAllis%26aufirst%3DC.%2BD.%26pub%3DCold%2520Spring%2520Harbor%2520Laboratory%2520Press%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koldobskiy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶ndÃ¶r, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic modulators, modifiers and mediators in cancer aetiology and progression</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">284</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1038/nrg.2016.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrg.2016.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26972587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVOksLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=284-299&author=A.+P.+Feinbergauthor=M.+A.+Koldobskiyauthor=A.+G%C3%B6nd%C3%B6r&title=Epigenetic+modulators%2C+modifiers+and+mediators+in+cancer+aetiology+and+progression&doi=10.1038%2Fnrg.2016.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Disease mechanisms Epigenetic modulators, modifiers and mediators in cancer aetiology and progression</span></div><div class="casAuthors">Feinberg, Andrew P.; Koldobskiy, Michael A.; Gondor, Anita</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">284-299</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This year is the tenth anniversary of the publication in this journal of a model suggesting the existence of 'tumor progenitor genes'.  These genes are epigenetically disrupted at the earliest stages of malignancies, even before mutations, and thus cause altered differentiation throughout tumor evolution.  The past decade of discovery in cancer epigenetics has revealed a no. of similarities between cancer genes and stem cell reprogramming genes, widespread mutations in epigenetic regulators, and the part played by chromatin structure in cellular plasticity in both development and cancer.  In the light of these discoveries, we suggest here a framework for cancer epigenetics involving three types of genes: 'epigenetic mediators', corresponding to the tumor progenitor genes suggested earlier; 'epigenetic modifiers' of the mediators, which are frequently mutated in cancer; and 'epigenetic modulators' upstream of the modifiers, which are responsive to changes in the cellular environment and often linked to the nuclear architecture.  We suggest that this classification is helpful in framing new diagnostic and therapeutic approaches to cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowr2rnZ1iYq7Vg90H21EOLACvtfcHk0lj7JSJkmvncWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVOksLk%253D&md5=9437a1c1cde1a6745504b979cb6ab60f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2016.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2016.13%26sid%3Dliteratum%253Aachs%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DKoldobskiy%26aufirst%3DM.%2BA.%26aulast%3DG%25C3%25B6nd%25C3%25B6r%26aufirst%3DA.%26atitle%3DEpigenetic%2520modulators%252C%2520modifiers%2520and%2520mediators%2520in%2520cancer%2520aetiology%2520and%2520progression%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2016%26volume%3D17%26spage%3D284%26epage%3D299%26doi%3D10.1038%2Fnrg.2016.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S.</span></span> <span> </span><span class="NLM_article-title">Targeting the cancer epigenome for therapy</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">630</span>â <span class="NLM_lpage">641</span>, <span class="refDoi">Â DOI: 10.1038/nrg.2016.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrg.2016.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27629931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiu77O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=630-641&author=P.+A.+Jonesauthor=J.+P.+Issaauthor=S.+Baylin&title=Targeting+the+cancer+epigenome+for+therapy&doi=10.1038%2Fnrg.2016.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer epigenome for therapy</span></div><div class="casAuthors">Jones, Peter A.; Issa, Jean-Pierre J.; Baylin, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">630-641</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Next-generation sequencing has revealed that more than 50% of human cancers harbor mutations in enzymes that are involved in chromatin organization.  Tumor cells not only are activated by genetic and epigenetic alterations, but also routinely use epigenetic processes to ensure their escape from chemotherapy and host immune surveillance.  Hence, a growing emphasis of recent drug discovery efforts has been on targeting the epigenome, including DNA methylation and histone modifications, with several new drugs being tested and some already approved by the US Food and Drug Administration (FDA).  The future will see the increasing success of combining epigenetic drugs with other therapies.  As epigenetic drugs target the epigenome as a whole, these true 'genomic medicines' lessen the need for precision approaches to individualized therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5pfbPOnZMtbVg90H21EOLACvtfcHk0lj7JSJkmvncWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiu77O&md5=22d98e58a4637d61af34a382e92adace</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2016.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2016.93%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26aulast%3DBaylin%26aufirst%3DS.%26atitle%3DTargeting%2520the%2520cancer%2520epigenome%2520for%2520therapy%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2016%26volume%3D17%26spage%3D630%26epage%3D641%26doi%3D10.1038%2Fnrg.2016.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetics in cancer</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">187</span>â <span class="NLM_lpage">207</span>, <span class="refDoi">Â DOI: 10.1146/annurev-pharmtox-010716-105106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1146%2Fannurev-pharmtox-010716-105106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28992434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1altrrE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=187-207&author=R.+L.+Bennettauthor=J.+D.+Licht&title=Targeting+epigenetics+in+cancer&doi=10.1146%2Fannurev-pharmtox-010716-105106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Epigenetics in Cancer</span></div><div class="casAuthors">Bennett, Richard L.; Licht, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">187-207</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Alterations of genes regulating epigenetic processes are frequently found as cancer drivers and may cause widespread alterations of DNA methylation, histone modification patterns, or chromatin structure that disrupt normal patterns of gene expression.  Because of the inherent reversibility of epigenetic changes, inhibitors targeting these processes are promising anticancer strategies.  Small mols. targeting epigenetic regulators have been developed recently, and clin. trials of these agents are under way for hematol. malignancies and solid tumors.  In this review, we describe how the writers, readers, and erasers of epigenetic marks are dysregulated in cancer and summarize the development of therapies targeting these mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBHGapO_R7HbVg90H21EOLACvtfcHk0lj7JSJkmvncWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1altrrE&md5=49208408fd3d3c29f9e256c52c933446</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010716-105106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010716-105106%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DR.%2BL.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26atitle%3DTargeting%2520epigenetics%2520in%2520cancer%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2018%26volume%3D58%26spage%3D187%26epage%3D207%26doi%3D10.1146%2Fannurev-pharmtox-010716-105106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ã.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1596</span>â <span class="NLM_lpage">1629</span>, <span class="refDoi">Â DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+%C3%9C.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lhE652vK5hF5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ã.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>â <span class="NLM_lpage">1068</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lhE652vK5hF5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">673</span>â <span class="NLM_lpage">691</span>, <span class="refDoi">Â DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+deacetylases+and+their+inhibitors+in+cancer%2C+neurological+diseases+and+immune+disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0lhE652vK5hF5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520deacetylases%2520and%2520their%2520inhibitors%2520in%2520cancer%252C%2520neurological%2520diseases%2520and%2520immune%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkinson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Recent progress in histone demethylase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1308</span>â <span class="NLM_lpage">1329</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1308-1329&author=T.+E.+McAllisterauthor=K.+S.+Englandauthor=R.+J.+Hopkinsonauthor=P.+E.+Brennanauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Recent+progress+in+histone+demethylase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in Histone Demethylase Inhibitors</span></div><div class="casAuthors">McAllister, Tom E.; England, Katherine S.; Hopkinson, Richard J.; Brennan, Paul E.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1308-1329</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is increasing interest in targeting histone N-methyl-lysine demethylases (KDMs) with small mols. both for the generation of probes for target exploration and for therapeutic purposes.  Here we update on previous reviews on the inhibition of the lysine-specific demethylases (LSDs or KDM1s) and JmjC families of N-methyl-lysine demethylases (JmjC KDMs, KDM2-7), focusing on the academic and patent literature from 2014 to date.  We also highlight recent biochem., biol., and structural studies which are relevant to KDM inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWx0bnBT9cDrVg90H21EOLACvtfcHk0lgl8ldgeieJ0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE&md5=f744b65af58dff765cdab8f9fea85310</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01758%26sid%3Dliteratum%253Aachs%26aulast%3DMcAllister%26aufirst%3DT.%2BE.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DRecent%2520progress%2520in%2520histone%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1308%26epage%3D1329%26doi%3D10.1021%2Facs.jmedchem.5b01758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Chemical and biochemical perspectives of protein lysine methylation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">6656</span>â <span class="NLM_lpage">6705</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.8b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=6656-6705&author=M.+Luo&title=Chemical+and+biochemical+perspectives+of+protein+lysine+methylation&doi=10.1021%2Facs.chemrev.8b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Biochemical Perspectives of Protein Lysine Methylation</span></div><div class="casAuthors">Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6656-6705</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein lysine methylation is a distinct post-translational modification that causes minimal changes in the size and electrostatic status of lysine residues.  Lysine methylation plays essential roles in regulating fates and functions of target proteins in an epigenetic manner.  As a result, substrates and degrees (free vs. mono/di/tri) of protein lysine methylation are orchestrated within cells by balanced activities of protein lysine methyltransferases (PKMTs) and demethylases (KDMs).  Their dysregulation is often assocd. with neurol. disorders, developmental abnormalities or cancer.  Methyllysine-contg. proteins can be recognized by downstream effector proteins, which contain methyllysine reader domains, to relay their biol. functions.  While numerous efforts have been made to annotate biol. roles of protein lysine methylation, limited work has been done to uncover mechanisms assocd. with this modification at a mol. or at. level.  Given distinct biophys. and biochem. properties of methyllysine, this review will focus on chem. and biochem. aspects in addn., recognition, and removal of this post-translational mark.  Chem. and biophys. methods to profile PKMT substrates will be discussed along with classification of PKMT inhibitors for accurate perturbation of methyltransferase activities.  Semisynthesis of methyllysine-contg. proteins will also be covered given the crit. need for these reagents to unambiguously define functional roles of protein lysine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbSRWg3jP5bLVg90H21EOLACvtfcHk0lgl8ldgeieJ0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO&md5=d846db925be49839c5954f5fc34f17d9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00008%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DChemical%2520and%2520biochemical%2520perspectives%2520of%2520protein%2520lysine%2520methylation%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D6656%26epage%3D6705%26doi%3D10.1021%2Facs.chemrev.8b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic regulators for cancer therapy: Mechanisms and advances in clinical trials</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">62</span>, <span class="refDoi">Â DOI: 10.1038/s41392-019-0095-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fs41392-019-0095-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31871779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3MbjsFCisQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=62&author=Y.+Chengauthor=C.+Heauthor=M.+Wangauthor=X.+Maauthor=F.+Moauthor=S.+Yangauthor=J.+Hanauthor=X.+Wei&title=Targeting+epigenetic+regulators+for+cancer+therapy%3A+Mechanisms+and+advances+in+clinical+trials&doi=10.1038%2Fs41392-019-0095-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic regulators for cancer therapy: mechanisms and advances in clinical trials</span></div><div class="casAuthors">Cheng Yuan; He Cai; Wang Manni; Ma Xuelei; Mo Fei; Yang Shengyong; Han Junhong; Wei Xiawei</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetic alternations concern heritable yet reversible changes in histone or DNA modifications that regulate gene activity beyond the underlying sequence.  Epigenetic dysregulation is often linked to human disease, notably cancer.  With the development of various drugs targeting epigenetic regulators, epigenetic-targeted therapy has been applied in the treatment of hematological malignancies and has exhibited viable therapeutic potential for solid tumors in preclinical and clinical trials.  In this review, we summarize the aberrant functions of enzymes in DNA methylation, histone acetylation and histone methylation during tumor progression and highlight the development of inhibitors of or drugs targeted at epigenetic enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRhk-ixsypfJ0Q0lWbKiCflfW6udTcc2eYA97typTQMnrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbjsFCisQ%253D%253D&md5=b93a1408a20df29fea33c22487f4a074</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0095-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0095-0%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DX.%26atitle%3DTargeting%2520epigenetic%2520regulators%2520for%2520cancer%2520therapy%253A%2520Mechanisms%2520and%2520advances%2520in%2520clinical%2520trials%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D62%26doi%3D10.1038%2Fs41392-019-0095-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">107406</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2019.107406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.pharmthera.2019.107406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31521697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVOgsr3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2019&pages=107406&author=F.+Yeauthor=J.+Huangauthor=H.+Wangauthor=C.+Luoauthor=K.+Zhao&title=Targeting+epigenetic+machinery%3A+Emerging+novel+allosteric+inhibitors&doi=10.1016%2Fj.pharmthera.2019.107406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic machinery: Emerging novel allosteric inhibitors</span></div><div class="casAuthors">Ye, Fei; Huang, Jing; Wang, Hongbo; Luo, Cheng; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107406</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Epigenetics has emerged as an extremely exciting fast-growing area of biomedical research in post genome era.  Epigenetic dysfunction is tightly related with various diseases such as cancer and aging related degeneration, potentiating epigenetics modulators as important therapeutics targets.  Indeed, inhibitors of histone deacetylase and DNA methyltransferase have been approved for treating blood tumor malignancies, whereas inhibitors of histone methyltransferase and histone acetyl-lysine recognizer bromodomain are in clin. stage.  However, it remains a great challenge to discover potent and selective inhibitors by targeting catalytic site, as the same subfamily of epigenetic enzymes often share high sequence identity and very conserved catalytic core pocket.  It is well known that epigenetic modifications are usually carried out by multi-protein complexes, and activation of catalytic subunit is often tightly regulated by other interactive protein component, esp. in disease conditions.  Therefore, it is not unusual that epigenetic complex machinery may exhibit allosteric regulation site induced by protein-protein interactions.  Targeting allosteric site emerges as a compelling alternative strategy to develop epigenetic drugs with enhanced druggability and pharmacol. profiles.  In this review, we highlight recent progress in the development of allosteric inhibitors for epigenetic complexes through targeting protein-protein interactions.  We also summarized the status of clin. applications of those inhibitors.  Finally, we provide perspectives of future novel allosteric epigenetic machinery modulators emerging from otherwise undruggable single protein target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgODybjVaUN7Vg90H21EOLACvtfcHk0lh6auzFb632jA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVOgsr3I&md5=b799ad2dc59c390f7a15847a5d41ba56</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DTargeting%2520epigenetic%2520machinery%253A%2520Emerging%2520novel%2520allosteric%2520inhibitors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2019%26volume%3D204%26spage%3D107406%26doi%3D10.1016%2Fj.pharmthera.2019.107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: A success story for cofactor interference</span>. <i>Philos. Trans. R. Soc., B</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">20170069</span>, <span class="refDoi">Â DOI: 10.1098/rstb.2017.0069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1098%2Frstb.2017.0069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29685973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Cmsb7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2018&pages=20170069&author=A.+Ganesan&title=Epigenetic+drug+discovery%3A+A+success+story+for+cofactor+interference&doi=10.1098%2Frstb.2017.0069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: a success story for cofactor interference</span></div><div class="casAuthors">Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Philosophical Transactions of the Royal Society, B: Biological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">1748</span>),
    <span class="NLM_cas:pages">20170069/1-20170069/15</span>CODEN:
                <span class="NLM_cas:coden">PTRBAE</span>;
        ISSN:<span class="NLM_cas:issn">0962-8436</span>.
    
            (<span class="NLM_cas:orgname">Royal Society</span>)
        </div><div class="casAbstract">Within the past two decades, seven epigenetic drugs have received regulatory approval and numerous other candidates are currently in clin. trials.  Among the epigenetic targets are the writer and eraser enzymes that are, resp., responsible for the reversible introduction and removal of structural modifications in the nucleosome.  This reviewdiscusses the progress achieved in the design and development of inhibitors against the key writer and eraser pairs: DNA methyltransferases and Tet demethylases; lysine/arginine methyltransferases and lysine demethylases; and histone acetyltransferases and histone deacetylases.  A common theme for the successful inhibition of these enzymes in a potent and selective manner is the targeting of the cofactors present in the active site, namely zinc and iron cations, S-adenosylmethione, NAD, FAD and acetyl CoA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrohR1yq_2gzbVg90H21EOLACvtfcHk0lh6auzFb632jA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Cmsb7M&md5=9c9e20f8b16a73a1835320e82698e68a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1098%2Frstb.2017.0069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frstb.2017.0069%26sid%3Dliteratum%253Aachs%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520A%2520success%2520story%2520for%2520cofactor%2520interference%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520B%26date%3D2018%26volume%3D373%26spage%3D20170069%26doi%3D10.1098%2Frstb.2017.0069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unnikrishnan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papaemmanuil, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrakanthan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoms, J. A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tursky, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulasekararaj, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobiasson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karimi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellagatti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramakrishna, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthur, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curnow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warburton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncolato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boultwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, S. E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mufti, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellstrom-Lindberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkins, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pimanda, J. E.</span></span> <span> </span><span class="NLM_article-title">Integrative genomics identifies the molecular basis of resistance to azacitidine therapy in myelodysplastic syndromes</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">572</span>â <span class="NLM_lpage">585</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2017.06.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.celrep.2017.06.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28723562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurzM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=572-585&author=A.+Unnikrishnanauthor=E.+Papaemmanuilauthor=D.+Beckauthor=N.+P.+Deshpandeauthor=A.+Vermaauthor=A.+Kumariauthor=P.+S.+Wollauthor=L.+A.+Richardsauthor=K.+Knezevicauthor=V.+Chandrakanthanauthor=J.+A.+I.+Thomsauthor=M.+L.+Turskyauthor=Y.+Huangauthor=Z.+Aliauthor=J.+Olivierauthor=S.+Galbraithauthor=A.+G.+Kulasekararajauthor=M.+Tobiassonauthor=M.+Karimiauthor=A.+Pellagattiauthor=S.+R.+Wilsonauthor=R.+Lindemanauthor=B.+Youngauthor=R.+Ramakrishnaauthor=C.+Arthurauthor=R.+Starkauthor=P.+Crispinauthor=J.+Curnowauthor=P.+Warburtonauthor=F.+Roncolatoauthor=J.+Boultwoodauthor=K.+Lynchauthor=S.+E.+W.+Jacobsenauthor=G.+J.+Muftiauthor=E.+Hellstrom-Lindbergauthor=M.+R.+Wilkinsauthor=K.+L.+MacKenzieauthor=J.+W.+H.+Wongauthor=P.+J.+Campbellauthor=J.+E.+Pimanda&title=Integrative+genomics+identifies+the+molecular+basis+of+resistance+to+azacitidine+therapy+in+myelodysplastic+syndromes&doi=10.1016%2Fj.celrep.2017.06.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative genomics identifies the molecular basis of resistance to azacitidine therapy in myelodysplastic syndromes</span></div><div class="casAuthors">Unnikrishnan, Ashwin; Papaemmanuil, Elli; Beck, Dominik; Deshpande, Nandan P.; Verma, Arjun; Kumari, Ashu; Woll, Petter S.; Richards, Laura A.; Knezevic, Kathy; Chandrakanthan, Vashe; Thoms, Julie A. I.; Tursky, Melinda L.; Huang, Yizhou; Ali, Zara; Olivier, Jake; Galbraith, Sally; Kulasekararaj, Austin G.; Tobiasson, Magnus; Karimi, Mohsen; Pellagatti, Andrea; Wilson, Susan R.; Lindeman, Robert; Young, Boris; Ramakrishna, Raj; Arthur, Christopher; Stark, Richard; Crispin, Philip; Curnow, Jennifer; Warburton, Pauline; Roncolato, Fernando; Boultwood, Jacqueline; Lynch, Kevin; Jacobsen, Sten Eirik W.; Mufti, Ghulam J.; Hellstrom-Lindberg, Eva; Wilkins, Marc R.; MacKenzie, Karen L.; Wong, Jason W. H.; Campbell, Peter J.; Pimanda, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">572-585</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Myelodysplastic syndromes and chronic myelomonocytic leukemia are blood disorders characterized by ineffective hematopoiesis and progressive marrow failure that can transform into acute leukemia.  The DNA methyltransferase inhibitor 5-azacytidine (AZA) is the most effective pharmacol. option, but only â¼50% of patients respond.  A response only manifests after many months of treatment and is transient.  The reasons underlying AZA resistance are unknown, and few alternatives exist for non-responders.  Here, we show that AZA responders have more hematopoietic progenitor cells (HPCs) in the cell cycle.  Non-responder HPC quiescence is mediated by integrin Î±5 (ITGA5) signaling and their hematopoietic potential improved by combining AZA with an ITGA5 inhibitor.  AZA response is assocd. with the induction of an inflammatory response in HPCs in vivo.  By mol. bar coding and tracking individual clones, we found that, although AZA alters the sub-clonal contribution to different lineages, founder clones are not eliminated and continue to drive hematopoiesis even in complete responders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi31PH3PgdpLVg90H21EOLACvtfcHk0lh6auzFb632jA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurzM&md5=210500f9690bbf6683f02562b4ae0de3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.06.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.06.067%26sid%3Dliteratum%253Aachs%26aulast%3DUnnikrishnan%26aufirst%3DA.%26aulast%3DPapaemmanuil%26aufirst%3DE.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DDeshpande%26aufirst%3DN.%2BP.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DKumari%26aufirst%3DA.%26aulast%3DWoll%26aufirst%3DP.%2BS.%26aulast%3DRichards%26aufirst%3DL.%2BA.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DChandrakanthan%26aufirst%3DV.%26aulast%3DThoms%26aufirst%3DJ.%2BA.%2BI.%26aulast%3DTursky%26aufirst%3DM.%2BL.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DAli%26aufirst%3DZ.%26aulast%3DOlivier%26aufirst%3DJ.%26aulast%3DGalbraith%26aufirst%3DS.%26aulast%3DKulasekararaj%26aufirst%3DA.%2BG.%26aulast%3DTobiasson%26aufirst%3DM.%26aulast%3DKarimi%26aufirst%3DM.%26aulast%3DPellagatti%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DS.%2BR.%26aulast%3DLindeman%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DB.%26aulast%3DRamakrishna%26aufirst%3DR.%26aulast%3DArthur%26aufirst%3DC.%26aulast%3DStark%26aufirst%3DR.%26aulast%3DCrispin%26aufirst%3DP.%26aulast%3DCurnow%26aufirst%3DJ.%26aulast%3DWarburton%26aufirst%3DP.%26aulast%3DRoncolato%26aufirst%3DF.%26aulast%3DBoultwood%26aufirst%3DJ.%26aulast%3DLynch%26aufirst%3DK.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%2BW.%26aulast%3DMufti%26aufirst%3DG.%2BJ.%26aulast%3DHellstrom-Lindberg%26aufirst%3DE.%26aulast%3DWilkins%26aufirst%3DM.%2BR.%26aulast%3DMacKenzie%26aufirst%3DK.%2BL.%26aulast%3DWong%26aufirst%3DJ.%2BW.%2BH.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DPimanda%26aufirst%3DJ.%2BE.%26atitle%3DIntegrative%2520genomics%2520identifies%2520the%2520molecular%2520basis%2520of%2520resistance%2520to%2520azacitidine%2520therapy%2520in%2520myelodysplastic%2520syndromes%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D572%26epage%3D585%26doi%3D10.1016%2Fj.celrep.2017.06.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiotsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niimi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, S.</span></span> <span> </span><span class="NLM_article-title">Reduced expression of HDAC3 contributes to the resistance against HDAC inhibitor, Vorinostat (SAHA) in mature lymphoid malignancies</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">1342</span>, <span class="refDoi">Â DOI: 10.1182/blood.V120.21.1342.1342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1182%2Fblood.V120.21.1342.1342" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=1342&author=J.+Dingauthor=M.+Riauthor=T.+Naritaauthor=A.+Masakiauthor=F.+Moriauthor=A.+Itoauthor=S.+Kusumotoauthor=T.+Ishidaauthor=H.+Komatsuauthor=Y.+Shiotsuauthor=A.+Niimiauthor=S.+Iida&title=Reduced+expression+of+HDAC3+contributes+to+the+resistance+against+HDAC+inhibitor%2C+Vorinostat+%28SAHA%29+in+mature+lymphoid+malignancies&doi=10.1182%2Fblood.V120.21.1342.1342"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1182%2Fblood.V120.21.1342.1342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V120.21.1342.1342%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DRi%26aufirst%3DM.%26aulast%3DNarita%26aufirst%3DT.%26aulast%3DMasaki%26aufirst%3DA.%26aulast%3DMori%26aufirst%3DF.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DKusumoto%26aufirst%3DS.%26aulast%3DIshida%26aufirst%3DT.%26aulast%3DKomatsu%26aufirst%3DH.%26aulast%3DShiotsu%26aufirst%3DY.%26aulast%3DNiimi%26aufirst%3DA.%26aulast%3DIida%26aufirst%3DS.%26atitle%3DReduced%2520expression%2520of%2520HDAC3%2520contributes%2520to%2520the%2520resistance%2520against%2520HDAC%2520inhibitor%252C%2520Vorinostat%2520%2528SAHA%2529%2520in%2520mature%2520lymphoid%2520malignancies%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D1342%26doi%3D10.1182%2Fblood.V120.21.1342.1342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: The road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>â <span class="NLM_lpage">377</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+The+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lgiXucptrnF2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520The%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D377%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, A.</span></span> <span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: Balancing the benefits and risks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6243</span>â <span class="NLM_lpage">6262</span>, <span class="refDoi">Â DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+Balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0lgiXucptrnF2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520Balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span> <span> </span><span class="NLM_article-title">Targeted covalent inhibitors for drug design</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">13408</span>â <span class="NLM_lpage">13421</span>, <span class="refDoi">Â DOI: 10.1002/anie.201601091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201601091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=13408-13421&author=T.+A.+Baillie&title=Targeted+covalent+inhibitors+for+drug+design&doi=10.1002%2Fanie.201601091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Covalent Inhibitors for Drug Design</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">13408-13421</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In contrast to the traditional mechanism of drug action that relies on the reversible, noncovalent interaction of a ligand with its biol. target, a targeted covalent inhibitor (TCI) is designed such that the initial, reversible assocn. is followed by the formation of a covalent bond between an electrophile on the ligand and a nucleophilic center in the protein.  Although this approach offers a variety of potential benefits (high potency and extended duration of action), concerns over the possible toxicol. consequences of protein haptenization have hindered the development of the TCI concept.  Recently, approaches to mitigate the risk of serious adverse reactions to this new class of agent have emerged, thus stimulating interest in the field and leading to authorization of the first cadre of TCIs to be marketed.  The covalent inhibitor approach is rapidly gaining acceptance as a valuable tool in drug discovery, and is poised to make a major impact on the design of enzyme inhibitors and receptor modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEbjdFrctvNrVg90H21EOLACvtfcHk0lgiXucptrnF2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN&md5=18bed2ae08dcb4245a42b3070abc6354</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fanie.201601091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201601091%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DTargeted%2520covalent%2520inhibitors%2520for%2520drug%2520design%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D13408%26epage%3D13421%26doi%3D10.1002%2Fanie.201601091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Cysteinome of protein kinases as a target in drug development</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4372</span>â <span class="NLM_lpage">4385</span>, <span class="refDoi">Â DOI: 10.1002/anie.201707875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201707875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4372-4385&author=A.+Chaikuadauthor=P.+Kochauthor=S.+A.+Lauferauthor=S.+Knapp&title=The+Cysteinome+of+protein+kinases+as+a+target+in+drug+development&doi=10.1002%2Fanie.201707875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The Cysteinome of Protein Kinases as a Target in Drug Development</span></div><div class="casAuthors">Chaikuad, Apirat; Koch, Pierre; Laufer, Stefan A.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4372-4385</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through covalent bond formation represent a considerable fraction of the authors' repository of effective medicines but safety concerns and the complexity of developing covalent inhibitors has rendered covalent targeting a less attractive strategy for rational drug design.  The recent approval of four covalent kinase inhibitors and the development of highly potent covalent kinase probes with exceptional selectivity has raised significant interest in industry and academic research and validated the concept of covalent kinase targeting for clin. applications.  The abundance of cysteines at diverse positions in and around the kinase active site suggests that a large fraction of kinases can be targeted by covalent inhibitors.  Herein, the authors review recent developments of this rapidly growing area in kinase drug development and highlight the unique opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphHGXsIr-dpbVg90H21EOLACvtfcHk0lgtV9qEXAZyNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D&md5=311ba5ab53dc4a5dc1dc92ddd4b6d4c2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707875%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Cysteinome%2520of%2520protein%2520kinases%2520as%2520a%2520target%2520in%2520drug%2520development%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D4372%26epage%3D4385%26doi%3D10.1002%2Fanie.201707875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry strategies for the development of kinaseinhibitors targeting point mutations</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">10726</span>â <span class="NLM_lpage">10741</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00507</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00507" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpslymtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10726-10741&author=X.+Luauthor=J.+B.+Smaillauthor=K.+Ding&title=Medicinal+chemistry+strategies+for+the+development+of+kinaseinhibitors+targeting+point+mutations&doi=10.1021%2Facs.jmedchem.0c00507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal Chemistry Strategies for the Development of Kinase Inhibitors Targeting Point Mutations</span></div><div class="casAuthors">Lu, Xiaoyun; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10726-10741</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Clin. acquired resistance to small mol. kinase inhibitors (SMKIs) has become a major "unmet clin. need" in cancer therapy.  To date, there are six SMKIs to be approved for the treatment of cancer patients through targeting of clin. acquired resistance caused by on-target mutations.  These are mainly focused on the mutant kinases Bcr-Abl T315I, EGFR T790M, and ALK L1196M.  Herein, we summarize the major medicinal chem. strategies employed in the discovery of these representative SMKIs, such as avoiding steric hindrance, making addnl. interactions with mutated residues, and forming a covalent bond with an active site cysteine to override resistance obsd. for reversible inhibitors.  Addnl., we also briefly describe allosteric kinase inhibitors and proteolysis targeting chimera (PROTAC) as two other potential strategies while addressing future opportunities in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMa3W_5KO3W7Vg90H21EOLACvtfcHk0lgtV9qEXAZyNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpslymtr8%253D&md5=4e008f3c68b1442e89f79a5a1a2b3968</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00507%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DMedicinal%2520chemistry%2520strategies%2520for%2520the%2520development%2520of%2520kinaseinhibitors%2520targeting%2520point%2520mutations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D10726%26epage%3D10741%26doi%3D10.1021%2Facs.jmedchem.0c00507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">207</span>â <span class="NLM_lpage">218</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2005.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.cell.2005.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16239140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=207-218&author=A.+Yokoyamaauthor=T.+C.+Somervailleauthor=K.+S.+Smithauthor=O.+Rozenblatt-Rosenauthor=M.+Meyersonauthor=M.+L.+Cleary&title=The+menin+tumor+suppressor+protein+is+an+essential+oncogenic+cofactor+for+MLL-associated+leukemogenesis&doi=10.1016%2Fj.cell.2005.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span></div><div class="casAuthors">Yokoyama, Akihiko; Somervaille, Tim C. P.; Smith, Kevin S.; Rozenblatt-Rosen, Orit; Meyerson, Matthew; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-218</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Mixed-Lineage Leukemia (MLL) protein is a histone methyltransferase that is mutated in clin. and biol. distinctive subsets of acute leukemia.  MLL normally assocs. with a cohort of highly conserved cofactors to form a macromol. complex that includes menin, a product of the MEN1 tumor suppressor gene, which is mutated in heritable and sporadic endocrine tumors.  We demonstrate here that oncogenic MLL fusion proteins retain an ability to stably assoc. with menin through a high-affinity, amino-terminal, conserved binding motif and that this interaction is required for the initiation of MLL-mediated leukemogenesis.  Furthermore, menin is essential for maintenance of MLL-assocd. but not other oncogene induced myeloid transformation.  Acute genetic ablation of menin reverses aberrant Hox gene expression mediated by MLL-menin promoter-assocd. complexes, and specifically abrogates the differentiation arrest and oncogenic properties of MLL-transformed leukemic blasts.  These results demonstrate that a human oncoprotein is critically dependent on direct phys. interaction with a tumor suppressor protein for its oncogenic activity, validate a potential target for mol. therapy, and suggest central roles for menin in altered epigenetic functions underlying the pathogenesis of hematopoietic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MXmqsWp8BrVg90H21EOLACvtfcHk0lgtV9qEXAZyNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP&md5=d582f614f98d25ebaf2282e2915fd4b6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26aulast%3DSmith%26aufirst%3DK.%2BS.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DThe%2520menin%2520tumor%2520suppressor%2520protein%2520is%2520an%2520essential%2520oncogenic%2520cofactor%2520for%2520MLL-associated%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D123%26spage%3D207%26epage%3D218%26doi%3D10.1016%2Fj.cell.2005.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">713</span>â <span class="NLM_lpage">719</span>, <span class="refDoi">Â DOI: 10.1038/nsmb1128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+methyltransferase+activity+by+its+core+components&doi=10.1038%2Fnsmb1128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0liUKVaRq4pmMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520by%2520its%2520core%2520components%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719%26doi%3D10.1038%2Fnsmb1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">823</span>â <span class="NLM_lpage">833</span>, <span class="refDoi">Â DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0liB64ku8aXmNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Challenges and opportunities in targeting the meninâMLL interaction</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">447</span>â <span class="NLM_lpage">462</span>, <span class="refDoi">Â DOI: 10.4155/fmc.13.214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.4155%2Ffmc.13.214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=24635524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=447-462&author=T.+Cierpickiauthor=J.+Grembecka&title=Challenges+and+opportunities+in+targeting+the+menin%E2%80%93MLL+interaction&doi=10.4155%2Ffmc.13.214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and opportunities in targeting the menin-MLL interaction</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-462</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Menin is an essential co-factor of oncogenic MLL fusion proteins and the menin-MLL interaction is crit. for development of acute leukemia in vivo.  Targeting the menin-MLL interaction with small mols. represents an attractive strategy to develop new anticancer agents.  Recent developments, including detn. of menin crystal structure and development of potent small mol. and peptidomimetic inhibitors, demonstrate the feasibility of targeting the menin-MLL interaction.  On the other hand, biochem. and structural studies revealed that MLL binds to menin in a complex bivalent mode engaging two MLL motifs, and therefore inhibition of this protein-protein interaction represents a challenge.  This review summarizes the most recent achievements in targeting the menin-MLL interaction as well as discusses potential benefits of blocking menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovLrSLXU5DNbVg90H21EOLACvtfcHk0liB64ku8aXmNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D&md5=97777abe7b2fd8df3daf390d9b019721</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.214%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DChallenges%2520and%2520opportunities%2520in%2520targeting%2520the%2520menin%25E2%2580%2593MLL%2520interaction%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D447%26epage%3D462%26doi%3D10.4155%2Ffmc.13.214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of menin reveals binding site for mixed lineage leukemia (MLL) protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">31742</span>â <span class="NLM_lpage">31748</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.258186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.M111.258186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21757704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=31742-31748&author=M.+J.+Muraiauthor=M.+Chruszczauthor=G.+Reddyauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Crystal+structure+of+menin+reveals+binding+site+for+mixed+lineage+leukemia+%28MLL%29+protein&doi=10.1074%2Fjbc.M111.258186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Menin Reveals Binding Site for Mixed Lineage Leukemia (MLL) Protein</span></div><div class="casAuthors">Murai, Marcelo J.; Chruszcz, Maksymilian; Reddy, Gireesh; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">31742-31748, S31742/1-S31742/2</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Menin is a tumor suppressor protein that is encoded by the MEN1 (multiple endocrine neoplasia 1) gene and controls cell growth in endocrine tissues.  Importantly, menin also serves as a crit. oncogenic cofactor of MLL (mixed lineage leukemia) fusion proteins in acute leukemias.  Direct assocn. of menin with MLL fusion proteins is required for MLL fusion protein-mediated leukemogenesis in vivo, and this interaction has been validated as a new potential therapeutic target for development of novel anti-leukemia agents.  Here, we report the first crystal structure of menin homolog from Nematostella vectensis.  Due to a very high sequence similarity, the Nematostella menin is a close homolog of human menin, and these two proteins likely have very similar structures.  Menin is predominantly an Î±-helical protein with the protein core comprising three tetratricopeptide motifs that are flanked by two Î±-helical bundles and covered by a Î²-sheet motif.  A very interesting feature of menin structure is the presence of a large central cavity that is highly conserved between Nematostella and human menin.  By employing site-directed mutagenesis, we have demonstrated that this cavity constitutes the binding site for MLL.  Our data provide a structural basis for understanding the role of menin as a tumor suppressor protein and as an oncogenic co-factor of MLL fusion proteins.  It also provides essential structural information for development of inhibitors targeting the menin-MLL interaction as a novel therapeutic strategy in MLL-related leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZHi8naCLUvLVg90H21EOLACvtfcHk0liB64ku8aXmNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur7P&md5=5c3e41d9d5c1fa4d65e7c8718638551a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.258186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.258186%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520menin%2520reveals%2520binding%2520site%2520for%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D31742%26epage%3D31748%26doi%3D10.1074%2Fjbc.M111.258186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of the mixed lineage leukemia-menin interaction: implications for targeting mixed lineage leukemias</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">40690</span>â <span class="NLM_lpage">40698</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M110.172783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.M110.172783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20961854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFynsLrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=40690-40698&author=J.+Grembeckaauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=T.+Cierpicki&title=Molecular+basis+of+the+mixed+lineage+leukemia-menin+interaction%3A+implications+for+targeting+mixed+lineage+leukemias&doi=10.1074%2Fjbc.M110.172783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of the mixed lineage leukemia-menin interaction: implications for targeting mixed lineage leukemias</span></div><div class="casAuthors">Grembecka, Jolanta; Belcher, Amalia M.; Hartley, Thomas; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">40690-40698</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chromosomal translocations targeting the mixed lineage leukemia (MLL) gene result in MLL fusion proteins that are found in aggressive human acute leukemias.  Disruption of MLL by such translocations leads to overexpression of Hox genes, resulting in a blockage of hematopoietic differentiation that ultimately leads to leukemia.  Menin, which directly binds MLL, has been identified as an essential oncogenic co-factor required for the leukemogenic activity of MLL fusion proteins.  Here, we characterize the mol. basis of the MLL-menin interaction.  Using 13C-detected NMR expts., we have mapped the residues within the intrinsically unstructured fragment of MLL that are required for binding to menin.  Interestingly, we found that MLL interacts with menin with a nanomolar affinity (Kd â¼ 10 nM) through two motifs, MBM1 and MBM2 (menin binding motifs 1 and 2).  These motifs are located within the N-terminal 43-amino acid fragment of MLL, and the MBM1 represents a high affinity binding motif.  Using alanine scanning mutagenesis of MBM1, we found that the hydrophobic residues Phe9, Pro10, and Pro13 are most crit. for binding.  Furthermore, based on exchange-transferred nuclear Overhauser effect measurements, we established that MBM1 binds to menin in an extended conformation.  In a series of competition expts. we showed that a peptide corresponding to MBM1 efficiently dissocs. the menin-MLL complex.  Altogether, our work establishes the mol. basis of the menin interaction with MLL and MLL fusion proteins and provides the necessary foundation for development of small mol. inhibitors targeting this interaction in leukemias with MLL translocations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO3pwLp7kEn7Vg90H21EOLACvtfcHk0liB64ku8aXmNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFynsLrL&md5=79d3199a45d392803791b89ff2dc8f7a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.172783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.172783%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMolecular%2520basis%2520of%2520the%2520mixed%2520lineage%2520leukemia-menin%2520interaction%253A%2520implications%2520for%2520targeting%2520mixed%2520lineage%2520leukemias%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D40690%26epage%3D40698%26doi%3D10.1074%2Fjbc.M110.172783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorenson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">277</span>â <span class="NLM_lpage">284</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnchembio.773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=22286128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=277-284&author=J.+Grembeckaauthor=S.+Heauthor=A.+Shiauthor=T.+Purohitauthor=A.+G.+Munteanauthor=R.+J.+Sorensonauthor=H.+D.+Showalterauthor=M.+J.+Muraiauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=J.+L.+Hessauthor=T.+Cierpicki&title=Menin-MLL+inhibitors+reverse+oncogenic+activity+of+MLL+fusion+proteins+in+leukemia&doi=10.1038%2Fnchembio.773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span></div><div class="casAuthors">Grembecka, Jolanta; He, Shihan; Shi, Aibin; Purohit, Trupta; Muntean, Andrew G.; Sorenson, Roderick J.; Showalter, Hollis D.; Murai, Marcelo J.; Belcher, Amalia M.; Hartley, Thomas; Hess, Jay L.; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Translocations involving the mixed lineage leukemia (MLL) gene result in human acute leukemias with very poor prognosis.  The leukemogenic activity of MLL fusion proteins is critically dependent on their direct interaction with menin, a product of the multiple endocrine neoplasia (MEN1) gene.  Here we present what are to our knowledge the first small-mol. inhibitors of the menin-MLL fusion protein interaction that specifically bind menin with nanomolar affinities.  These compds. effectively reverse MLL fusion protein-mediated leukemic transformation by downregulating the expression of target genes required for MLL fusion protein oncogenic activity.  They also selectively block proliferation and induce both apoptosis and differentiation of leukemia cells harboring MLL translocations.  Identification of these compds. provides a new tool for better understanding MLL-mediated leukemogenesis and represents a new approach for studying the role of menin as an oncogenic cofactor of MLL fusion proteins.  Our findings also highlight a new therapeutic strategy for aggressive leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_e6VYyS5l7Vg90H21EOLACvtfcHk0lg9VS24J0YpVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D&md5=55eb9f353031dfef4c1a281b0ac28cba</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.773%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DA.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DSorenson%26aufirst%3DR.%2BJ.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMenin-MLL%2520inhibitors%2520reverse%2520oncogenic%2520activity%2520of%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D277%26epage%3D284%26doi%3D10.1038%2Fnchembio.773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of the bivalent Menin-MLL interaction by small molecules in leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4461</span>â <span class="NLM_lpage">4469</span>, <span class="refDoi">Â DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=22936661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&author=A.+Shiauthor=M.+J.+Muraiauthor=S.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+insights+into+inhibition+of+the+bivalent+Menin-MLL+interaction+by+small+molecules+in+leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span></div><div class="casAuthors">Shi, Aibin; Murai, Marcelo J.; He, Shihan; Lund, George; Hartley, Thomas; Purohit, Trupta; Reddy, Gireesh; Chruszcz, Maksymilian; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4461-4469</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Menin functions as a crit. oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in the development of acute leukemias, and inhibition of the menin interaction with MLL fusion proteins represents a very promising strategy to reverse their oncogenic activity.  MLL interacts with menin in a bivalent mode involving 2 N-terminal fragments of MLL.  In the present study, we reveal the first high-resoln. crystal structure of human menin in complex with a small-mol. inhibitor of the menin-MLL interaction, MI-2.  The structure shows that the compd. binds to the MLL pocket in menin and mimics the key interactions of MLL with menin.  Based on the menin-MI-2 structure, we developed MI-2-2, a compd. that binds to menin with low nanomolar affinity (Kd = 22nM) and very effectively disrupts the bivalent protein-protein interaction between menin and MLL.  MI-2-2 demonstrated specific and very pronounced activity in MLL leukemia cells, including inhibition of cell proliferation, down-regulation of Hoxa9 expression, and differentiation.  Our results provide the rational and essential structural basis to design next generation of inhibitors for effective targeting of the menin-MLL interaction in leukemia and demonstrate a proof of concept that inhibition of complex multivalent protein-protein interactions can be achieved by a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFC9mQS_4LVg90H21EOLACvtfcHk0lg9VS24J0YpVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J&md5=c1ab14816cf5ea3590fe03b0ed405ea5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520the%2520bivalent%2520Menin-MLL%2520interaction%2520by%2520small%2520molecules%2520in%2520leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the Menin-MLL interaction blocks progression of MLL leukemia in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">589</span>â <span class="NLM_lpage">602</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2015.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2015.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25817203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=589-602&author=D.+Borkinauthor=S.+Heauthor=H.+Miaoauthor=K.+Kempinskaauthor=J.+Pollockauthor=J.+Chaseauthor=T.+Purohitauthor=B.+Malikauthor=T.+Zhaoauthor=J.+Wangauthor=B.+Wenauthor=H.+Zongauthor=M.+Jonesauthor=G.+Danet-Desnoyersauthor=M.+L.+Guzmanauthor=M.+Talpazauthor=D.+L.+Bixbyauthor=D.+Sunauthor=J.+L.+Hessauthor=A.+G.+Munteanauthor=I.+Maillardauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+Menin-MLL+interaction+blocks+progression+of+MLL+leukemia+in+vivo&doi=10.1016%2Fj.ccell.2015.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of the Menin-MLL Interaction Blocks Progression of MLL Leukemia In Vivo</span></div><div class="casAuthors">Borkin, Dmitry; He, Shihan; Miao, Hongzhi; Kempinska, Katarzyna; Pollock, Jonathan; Chase, Jennifer; Purohit, Trupta; Malik, Bhavna; Zhao, Ting; Wang, Jingya; Wen, Bo; Zong, Hongliang; Jones, Morgan; Danet-Desnoyers, Gwenn; Guzman, Monica L.; Talpaz, Moshe; Bixby, Dale L.; Sun, Duxin; Hess, Jay L.; Muntean, Andrew G.; Maillard, Ivan; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">589-602</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chromosomal translocations affecting mixed lineage leukemia gene (MLL) result in acute leukemias resistant to therapy.  The leukemogenic activity of MLL fusion proteins is dependent on their interaction with menin, providing basis for therapeutic intervention.  Here we report the development of highly potent and orally bioavailable small-mol. inhibitors of the menin-MLL interaction, MI-463 and MI-503, and show their profound effects in MLL leukemia cells and substantial survival benefit in mouse models of MLL leukemia.  Finally, we demonstrate the efficacy of these compds. in primary samples derived from MLL leukemia patients.  Overall, we demonstrate that pharmacol. inhibition of the menin-MLL interaction represents an effective treatment for MLL leukemias in vivo and provide advanced mol. scaffold for clin. lead identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc2hESfSmR8bVg90H21EOLACvtfcHk0lg9VS24J0YpVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D&md5=1369b1556d139e9006c3fa7d1add49a0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DChase%26aufirst%3DJ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZong%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DBixby%26aufirst%3DD.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DMaillard%26aufirst%3DI.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520Menin-MLL%2520interaction%2520blocks%2520progression%2520of%2520MLL%2520leukemia%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D589%26epage%3D602%26doi%3D10.1016%2Fj.ccell.2015.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malik, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asangani, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CieÅlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prensner, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escara-Wilke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stender, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niknafs, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palanisamy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunju, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yocum, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellacheruvu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nesvizhskii, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanasekaran, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the MLL complex in castration-resistant prostate cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">344</span>â <span class="NLM_lpage">352</span>, <span class="refDoi">Â DOI: 10.1038/nm.3830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnm.3830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25822367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=344-352&author=R.+Malikauthor=A.+P.+Khanauthor=I.+A.+Asanganiauthor=M.+Cie%C5%9Blikauthor=J.+R.+Prensnerauthor=X.+Wangauthor=M.+K.+Iyerauthor=X.+Jiangauthor=D.+Borkinauthor=J.+Escara-Wilkeauthor=R.+Stenderauthor=Y.+M.+Wuauthor=Y.+S.+Niknafsauthor=X.+Jingauthor=Y.+Qiaoauthor=N.+Palanisamyauthor=L.+P.+Kunjuauthor=P.+M.+Krishnamurthyauthor=A.+K.+Yocumauthor=D.+Mellacheruvuauthor=A.+I.+Nesvizhskiiauthor=X.+Caoauthor=S.+M.+Dhanasekaranauthor=F.+Y.+Fengauthor=J.+Grembeckaauthor=T.+Cierpickiauthor=A.+M.+Chinnaiyan&title=Targeting+the+MLL+complex+in+castration-resistant+prostate+cancer&doi=10.1038%2Fnm.3830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the MLL complex in castration-resistant prostate cancer</span></div><div class="casAuthors">Malik, Rohit; Khan, Amjad P.; Asangani, Irfan A.; Cieslik, Marcin; Prensner, John R.; Wang, Xiaoju; Iyer, Matthew K.; Jiang, Xia; Borkin, Dmitry; Escara-Wilke, June; Stender, Rachell; Wu, Yi-Mi; Niknafs, Yashar S.; Jing, Xiaojun; Qiao, Yuanyuan; Palanisamy, Nallasivam; Kunju, Lakshmi P.; Krishnamurthy, Pranathi M.; Yocum, Anastasia K.; Mellacheruvu, Dattatreya; Nesvizhskii, Alexey I.; Cao, Xuhong; Dhanasekaran, Saravana M.; Feng, Felix Y.; Grembecka, Jolanta; Cierpicki, Tomasz; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">344-352</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Resistance to androgen deprivation therapies and increased androgen receptor (AR) activity are major drivers of castration-resistant prostate cancer (CRPC).  Although prior work has focused on targeting AR directly, co-activators of AR signaling, which may represent new therapeutic targets, are relatively underexplored.  Here we demonstrate that the mixed-lineage leukemia protein (MLL) complex, a well-known driver of MLL fusion-pos. leukemia, acts as a co-activator of AR signaling.  AR directly interacts with the MLL complex via the menin-MLL subunit.  Menin expression is higher in CRPC than in both hormone-naive prostate cancer and benign prostate tissue, and high menin expression correlates with poor overall survival of individuals diagnosed with prostate cancer.  Treatment with a small-mol. inhibitor of menin-MLL interaction blocks AR signaling and inhibits the growth of castration-resistant tumors in vivo in mice.  Taken together, this work identifies the MLL complex as a crucial co-activator of AR and a potential therapeutic target in advanced prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmvjojfsVQ97Vg90H21EOLACvtfcHk0ligm8xxSAe6wQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D&md5=31d39514cc6f6cf723146a70676f4021</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnm.3830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3830%26sid%3Dliteratum%253Aachs%26aulast%3DMalik%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DA.%2BP.%26aulast%3DAsangani%26aufirst%3DI.%2BA.%26aulast%3DCie%25C5%259Blik%26aufirst%3DM.%26aulast%3DPrensner%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DIyer%26aufirst%3DM.%2BK.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DEscara-Wilke%26aufirst%3DJ.%26aulast%3DStender%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DNiknafs%26aufirst%3DY.%2BS.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DPalanisamy%26aufirst%3DN.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DKrishnamurthy%26aufirst%3DP.%2BM.%26aulast%3DYocum%26aufirst%3DA.%2BK.%26aulast%3DMellacheruvu%26aufirst%3DD.%26aulast%3DNesvizhskii%26aufirst%3DA.%2BI.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DTargeting%2520the%2520MLL%2520complex%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D344%26epage%3D352%26doi%3D10.1038%2Fnm.3830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Property focused structure-based optimization of small molecule inhibitors of the protein-protein interaction between menin and mixed lineage leukemia (MLL)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">892</span>â <span class="NLM_lpage">913</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=892-913&author=D.+Borkinauthor=J.+Pollockauthor=K.+Kempinskaauthor=T.+Purohitauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=H.+Miaoauthor=S.+Shuklaauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Property+focused+structure-based+optimization+of+small+molecule+inhibitors+of+the+protein-protein+interaction+between+menin+and+mixed+lineage+leukemia+%28MLL%29&doi=10.1021%2Facs.jmedchem.5b01305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Property Focused Structure-Based Optimization of Small Molecule Inhibitors of the Protein-Protein Interaction between Menin and Mixed Lineage Leukemia (MLL)</span></div><div class="casAuthors">Borkin, Dmitry; Pollock, Jonathan; Kempinska, Katarzyna; Purohit, Trupta; Li, Xiaoqin; Wen, Bo; Zhao, Ting; Miao, Hongzhi; Shukla, Shirish; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-913</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of potent small mol. inhibitors of protein-protein interactions with optimized druglike properties represents a challenging task in lead optimization process.  Here, we report synthesis and structure-based optimization of new thienopyrimidine class of compds., which block the protein-protein interaction between menin and MLL fusion proteins that plays an important role in acute leukemias with MLL translocations.  We performed simultaneous optimization of both activity and druglike properties through systematic exploration of substituents introduced to the indole ring of lead compd. 1 (MI-136) to identify compds. suitable for in vivo studies in mice.  This work resulted in the identification of compd. 27 (MI-538), which showed significantly increased activity, selectivity, polarity, and pharmacokinetic profile over 1 and demonstrated a pronounced effect in a mouse model of MLL leukemia.  This study, which reports detailed structure-activity and structure-property relationships for the menin-MLL inhibitors, demonstrates challenges in optimizing inhibitors of protein-protein interactions for potential therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDD5t7XKKz_LVg90H21EOLACvtfcHk0ligm8xxSAe6wQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D&md5=f89b6eda72671f133a38181b3b23632d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01305%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DProperty%2520focused%2520structure-based%2520optimization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520protein-protein%2520interaction%2520between%2520menin%2520and%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D892%26epage%3D913%26doi%3D10.1021%2Facs.jmedchem.5b01305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klossowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linhares, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Complexity of blocking bivalent protein-protein interactions: Development of a highly Potent inhibitor of the menin-mixed-lineage leukemia interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4832</span>â <span class="NLM_lpage">4850</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4832-4850&author=D.+Borkinauthor=S.+Klossowskiauthor=J.+Pollockauthor=H.+Miaoauthor=B.+M.+Linharesauthor=K.+Kempinskaauthor=Z.+Jinauthor=T.+Purohitauthor=B.+Wenauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Complexity+of+blocking+bivalent+protein-protein+interactions%3A+Development+of+a+highly+Potent+inhibitor+of+the+menin-mixed-lineage+leukemia+interaction&doi=10.1021%2Facs.jmedchem.8b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Complexity of Blocking Bivalent Protein-Protein Interactions: Development of a Highly Potent Inhibitor of the Menin-Mixed-Lineage Leukemia Interaction</span></div><div class="casAuthors">Borkin, Dmitry; Klossowski, Szymon; Pollock, Jonathan; Miao, Hongzhi; Linhares, Brian M.; Kempinska, Katarzyna; Jin, Zhuang; Purohit, Trupta; Wen, Bo; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4832-4850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors.  Here, we report the development of a new generation of menin-MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compds.  This work resulted in compd. I (MI-1481), which showed very potent inhibition of the menin-MLL1 interaction (IC50 = 3.6 nM), representing the most potent reversible menin-MLL1 inhibitor reported to date.  The crystal structure of the menin-I complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of I.  Compd. I also demonstrates pronounced activity in MLL leukemia cells and in vivo in MLL leukemia models.  Thus, I is a valuable menin-MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_eephZ2UHnLVg90H21EOLACvtfcHk0li7pN4zyqe3dg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D&md5=8d12fa816af3a5aa191694d4bfe48802</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00071%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DKlossowski%26aufirst%3DS.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DLinhares%26aufirst%3DB.%2BM.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DComplexity%2520of%2520blocking%2520bivalent%2520protein-protein%2520interactions%253A%2520Development%2520of%2520a%2520highly%2520Potent%2520inhibitor%2520of%2520the%2520menin-mixed-lineage%2520leukemia%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4832%26epage%3D4850%26doi%3D10.1021%2Facs.jmedchem.8b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadrey, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eschle, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uckelmann, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braytee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claremon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gifford, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teicher, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pimanda, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGeehan, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lock, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">A meninâMLL inhibitor induces specific chromatin changes and eradicates disease in models of MLL-rearranged leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">660</span>â <span class="NLM_lpage">673</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2019.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2019.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31821784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOmtLrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=660-673&author=A.+V.+Krivtsovauthor=K.+Evansauthor=J.+Y.+Gadreyauthor=B.+K.+Eschleauthor=C.+Hattonauthor=H.+J.+Uckelmannauthor=K.+N.+Rossauthor=F.+Pernerauthor=S.+N.+Olsenauthor=T.+Pritchardauthor=L.+McDermottauthor=C.+D.+Jonesauthor=D.+Jingauthor=A.+Brayteeauthor=D.+Chaconauthor=E.+Earleyauthor=B.+M.+McKeeverauthor=D.+Claremonauthor=A.+J.+Giffordauthor=H.+J.+Leeauthor=B.+A.+Teicherauthor=J.+E.+Pimandaauthor=D.+Beckauthor=J.+A.+Perryauthor=M.+A.+Smithauthor=G.+M.+McGeehanauthor=R.+B.+Lockauthor=S.+A.+Armstrong&title=A+menin%E2%80%93MLL+inhibitor+induces+specific+chromatin+changes+and+eradicates+disease+in+models+of+MLL-rearranged+leukemia&doi=10.1016%2Fj.ccell.2019.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Menin-MLL Inhibitor Induces Specific Chromatin Changes and Eradicates Disease in Models of MLL-Rearranged Leukemia</span></div><div class="casAuthors">Krivtsov, Andrei V.; Evans, Kathryn; Gadrey, Jayant Y.; Eschle, Benjamin K.; Hatton, Charlie; Uckelmann, Hannah J.; Ross, Kenneth N.; Perner, Florian; Olsen, Sarah N.; Pritchard, Tara; McDermott, Lisa; Jones, Connor D.; Jing, Duohui; Braytee, Ali; Chacon, Diego; Earley, Eric; McKeever, Brian M.; Claremon, David; Gifford, Andrew J.; Lee, Heather J.; Teicher, Beverly A.; Pimanda, John E.; Beck, Dominik; Perry, Jennifer A.; Smith, Malcolm A.; McGeehan, Gerard M.; Lock, Richard B.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">660-673.e11</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Inhibition of the Menin (MEN1) and MLL (MLL1, KMT2A) interaction is a potential therapeutic strategy for MLL-rearranged (MLL-r) leukemia.  Structure-based design yielded the potent, highly selective, and orally bioavailable small-mol. inhibitor VTP50469.  Cell lines carrying MLL rearrangements were selectively responsive to VTP50469.  VTP50469 displaced Menin from protein complexes and inhibited chromatin occupancy of MLL at select genes.  Loss of MLL binding led to changes in gene expression, differentiation, and apoptosis.  Patient-derived xenograft (PDX) models derived from patients with either MLL-r acute myeloid leukemia or MLL-r acute lymphoblastic leukemia (ALL) showed dramatic redns. of leukemia burden when treated with VTP50469.  Multiple mice engrafted with MLL-r ALL remained disease free for more than 1 yr after treatment.  These data support rapid translation of this approach to clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgvJuHyOgm4LVg90H21EOLACvtfcHk0li7pN4zyqe3dg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOmtLrN&md5=31046dca120487279d1b9bd66d154c97</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DEvans%26aufirst%3DK.%26aulast%3DGadrey%26aufirst%3DJ.%2BY.%26aulast%3DEschle%26aufirst%3DB.%2BK.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DUckelmann%26aufirst%3DH.%2BJ.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DPerner%26aufirst%3DF.%26aulast%3DOlsen%26aufirst%3DS.%2BN.%26aulast%3DPritchard%26aufirst%3DT.%26aulast%3DMcDermott%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DC.%2BD.%26aulast%3DJing%26aufirst%3DD.%26aulast%3DBraytee%26aufirst%3DA.%26aulast%3DChacon%26aufirst%3DD.%26aulast%3DEarley%26aufirst%3DE.%26aulast%3DMcKeever%26aufirst%3DB.%2BM.%26aulast%3DClaremon%26aufirst%3DD.%26aulast%3DGifford%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA.%26aulast%3DPimanda%26aufirst%3DJ.%2BE.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DMcGeehan%26aufirst%3DG.%2BM.%26aulast%3DLock%26aufirst%3DR.%2BB.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DA%2520menin%25E2%2580%2593MLL%2520inhibitor%2520induces%2520specific%2520chromatin%2520changes%2520and%2520eradicates%2520disease%2520in%2520models%2520of%2520MLL-rearranged%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D660%26epage%3D673%26doi%3D10.1016%2Fj.ccell.2019.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">High-affinity small-molecule inhibitors of the menin-mixed lineage leukemia (MLL) interaction closely mimic a natural protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1543</span>â <span class="NLM_lpage">1556</span>, <span class="refDoi">Â DOI: 10.1021/jm401868d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401868d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1543-1556&author=S.+Heauthor=T.+J.+Senterauthor=J.+Pollockauthor=C.+Hanauthor=S.+K.+Upadhyayauthor=T.+Purohitauthor=R.+D.+Gogliottiauthor=C.+W.+Lindsleyauthor=T.+Cierpickiauthor=S.+R.+Staufferauthor=J.+Grembecka&title=High-affinity+small-molecule+inhibitors+of+the+menin-mixed+lineage+leukemia+%28MLL%29+interaction+closely+mimic+a+natural+protein-protein+interaction&doi=10.1021%2Fjm401868d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Small-Molecule Inhibitors of the Menin-Mixed Lineage Leukemia (MLL) Interaction Closely Mimic a Natural Protein-Protein Interaction</span></div><div class="casAuthors">He, Shihan; Senter, Timothy J.; Pollock, Jonathan; Han, Changho; Upadhyay, Sunil Kumar; Purohit, Trupta; Gogliotti, Rocco D.; Lindsley, Craig W.; Cierpicki, Tomasz; Stauffer, Shaun R.; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1543-1556</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction (PPI) between menin and mixed lineage leukemia (MLL) plays a crit. role in acute leukemias, and inhibition of this interaction represents a new potential therapeutic strategy for MLL leukemias.  We report development of a novel class of small-mol. inhibitors of the menin-MLL interaction, the hydroxy- and aminomethylpiperidine compds., which originated from HTS of â¼288000 small mols.  We detd. menin-inhibitor co-crystal structures and found that these compds. closely mimic all key interactions of MLL with menin.  Extensive crystallog. studies combined with structure-based design were applied for optimization of these compds., resulting in MIV-6R, which inhibits the menin-MLL interaction with IC50 = 56 nM.  Treatment with MIV-6 demonstrated strong and selective effects in MLL leukemia cells, validating specific mechanism of action.  Our studies provide novel and attractive scaffold as a new potential therapeutic approach for MLL leukemias and demonstrate an example of PPI amenable to inhibition by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNC4yHqMSn87Vg90H21EOLACvtfcHk0liCOzyswAvf9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D&md5=a580d1b2ea3f657c4abb31678f39548c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm401868d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401868d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSenter%26aufirst%3DT.%2BJ.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DS.%2BK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DHigh-affinity%2520small-molecule%2520inhibitors%2520of%2520the%2520menin-mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520interaction%2520closely%2520mimic%2520a%2520natural%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1543%26epage%3D1556%26doi%3D10.1021%2Fjm401868d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FernÃ¡ndez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the first-in-class, highly potent irreversible inhibitor targeting the Menin-MLL protein-protein interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1601</span>â <span class="NLM_lpage">1605</span>, <span class="refDoi">Â DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fern%C3%A1ndez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+first-in-class%2C+highly+potent+irreversible+inhibitor+targeting+the+Menin-MLL+protein-protein+interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Xu, Tianfeng; Zheng, Ke; Huang, Liyue; Stuckey, Jeanne; Chinnaswamy, Krishnapriya; Bernard, Denzil; Fernandez-Salas, Ester; Liu, Liu; Wang, Mi; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1605</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The structure-based design of M-525 (I) as the first-in-class, highly potent, irreversible small-mol. inhibitor of the menin-MLL interaction is presented.  M-525 targets cellular menin protein at sub-nanomolar concns. and achieves low nanomolar potencies in cell growth inhibition and in the suppression of MLL-regulated gene expression in MLL leukemia cells.  M-525 demonstrates high cellular specificity over non-MLL leukemia cells and is more than 30 times more potent than its corresponding reversible inhibitors.  Mass spectrometric anal. and co-crystal structure of M-525 in complex with menin firmly establish its mode of action.  A single administration of M-525 effectively suppresses MLL-regulated gene expression in tumor tissue.  An efficient procedure was developed to synthesize M-525.  This study demonstrates that irreversible inhibition of menin may be a promising therapeutic strategy for MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fbcCaeUFsLVg90H21EOLACvtfcHk0liCOzyswAvf9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D&md5=f7d189e2138b88a4d8c21c72586b7502</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFern%25C3%25A1ndez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520first-in-class%252C%2520highly%2520potent%2520irreversible%2520inhibitor%2520targeting%2520the%2520Menin-MLL%2520protein-protein%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawacki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of M-808 as a highly potent, covalent, small-molecule inhibitor of the Menin-MLL interaction with strong <i>in vivo</i> antitumor activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4997</span>â <span class="NLM_lpage">5010</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVClsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4997-5010&author=S.+Xuauthor=A.+Aguilarauthor=L.+Huangauthor=T.+Xuauthor=K.+Zhengauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=K.+Zawackiauthor=Z.+Liuauthor=K.+Chinnaswamyauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+M-808+as+a+highly+potent%2C+covalent%2C+small-molecule+inhibitor+of+the+Menin-MLL+interaction+with+strong+in+vivo+antitumor+activity&doi=10.1021%2Facs.jmedchem.0c00547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of M-808 as a Highly Potent, Covalent, Small-Molecule Inhibitor of the Menin-MLL Interaction with Strong In Vivo Antitumor Activity</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Huang, Liyue; Xu, Tianfeng; Zheng, Ke; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Zawacki, Kaitlin; Liu, Zhaomin; Chinnaswamy, Krishnapriya; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4997-5010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting the menin-MLL protein-protein interaction is a new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia).  We describe herein the structure-based optimization of a class of covalent menin inhibitors, which led to the discovery of M-808 (16)(I) as a highly potent and efficacious covalent menin inhibitor.  M-808 effectively inhibits leukemia cell growth at low nanomolar concns. and is capable of achieving partial tumor regression in an MV4;11 xenograft tumor model in mice at a well-tolerated dose schedule.  Detn. of the co-crystal structure of M-808 in complex with menin provides a structural basis for their high-affinity, covalent interactions.  M-808 represents a promising, covalent menin inhibitor for further optimization and evaluation toward developing a new therapy for the treatment of MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCh0XqIp3vpLVg90H21EOLACvtfcHk0liCOzyswAvf9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVClsrg%253D&md5=168b2c21b9222ee72120f6680e7f60db</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00547%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DZawacki%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520M-808%2520as%2520a%2520highly%2520potent%252C%2520covalent%252C%2520small-molecule%2520inhibitor%2520of%2520the%2520Menin-MLL%2520interaction%2520with%2520strong%2520in%2520vivo%2520antitumor%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4997%26epage%3D5010%26doi%3D10.1021%2Facs.jmedchem.0c00547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congdon, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokomori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J. C.</span></span> <span> </span><span class="NLM_article-title">Dynamic regulation of the PR-Set7 histone methyltransferase is required for normal cell cycle progression</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2531</span>â <span class="NLM_lpage">2542</span>, <span class="refDoi">Â DOI: 10.1101/gad.1984210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1101%2Fgad.1984210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20966048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSgu77O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2531-2542&author=S.+Wuauthor=W.+Wangauthor=X.+Kongauthor=L.+M.+Congdonauthor=K.+Yokomoriauthor=M.+W.+Kirschnerauthor=J.+C.+Rice&title=Dynamic+regulation+of+the+PR-Set7+histone+methyltransferase+is+required+for+normal+cell+cycle+progression&doi=10.1101%2Fgad.1984210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic regulation of the PR-Set7 histone methyltransferase is required for normal cell cycle progression</span></div><div class="casAuthors">Wu, Shumin; Wang, Weiping; Kong, Xiangduo; Congdon, Lauren M.; Yokomori, Kyoko; Kirschner, Marc W.; Rice, Judd C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2531-2542</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Although the PR-Set7/Set8/KMT5a histone H4 Lys 20 monomethyltransferase (H4K20me1) plays an essential role in mammalian cell cycle progression, esp. during G2/M, it remained unknown how PR-Set7 itself was regulated.  In this study, we discovered the mechanisms that govern the dynamic regulation of PR-Set7 during mitosis, and that perturbation of these pathways results in defective mitotic progression.  First, we found that PR-Set7 is phosphorylated at Ser 29 (S29) specifically by the cyclin-dependent kinase 1 (cdk1)/cyclinB complex, primarily from prophase through early anaphase, subsequent to global accumulation of H4K20me1.  While S29 phosphorylation did not affect PR-Set7 methyltransferase activity, this event resulted in the removal of PR-Set7 from mitotic chromosomes.  S29 phosphorylation also functions to stabilize PR-Set7 by directly inhibiting its interaction with the anaphase-promoting complex (APC), an E3 ubiquitin ligase.  The dephosphorylation of S29 during late mitosis by the Cdc14 phosphatases was required for APCcdh1-mediated ubiquitination of PR-Set7 and subsequent proteolysis.  This event is important for proper mitotic progression, as constitutive phosphorylation of PR-Set7 resulted in a substantial delay between metaphase and anaphase.  Collectively, we elucidated the mol. mechanisms that control PR-Set7 protein levels during mitosis, and demonstrated that its orchestrated regulation is important for normal mitotic progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV3w-MmV6YIrVg90H21EOLACvtfcHk0ljl_erc6JR-9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSgu77O&md5=4094cec7d8bdca00a92072ab4b99999d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1101%2Fgad.1984210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1984210%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DCongdon%26aufirst%3DL.%2BM.%26aulast%3DYokomori%26aufirst%3DK.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26atitle%3DDynamic%2520regulation%2520of%2520the%2520PR-Set7%2520histone%2520methyltransferase%2520is%2520required%2520for%2520normal%2520cell%2520cycle%2520progression%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2531%26epage%3D2542%26doi%3D10.1101%2Fgad.1984210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saloura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span> <span> </span><span class="NLM_article-title">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">110</span>â <span class="NLM_lpage">124</span>, <span class="refDoi">Â DOI: 10.1038/nrc3884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrc3884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25614009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=110-124&author=R.+Hamamotoauthor=V.+Salouraauthor=Y.+Nakamura&title=Critical+roles+of+non-histone+protein+lysine+methylation+in+human+tumorigenesis&doi=10.1038%2Fnrc3884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span></div><div class="casAuthors">Hamamoto, Ryuji; Saloura, Vassiliki; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-124</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several protein lysine methyltransferases and demethylases have been identified to have crit. roles in histone modification.  A large body of evidence has indicated that their dysregulation is involved in the development and progression of various diseases, including cancer, and these enzymes are now considered to be potential therapeutic targets.  Although most studies have focused on histone methylation, many reports have revealed that these enzymes also regulate the methylation dynamics of non-histone proteins such as p53, RB1 and STAT3 (signal transducer and activator of transcription 3), which have important roles in human tumorigenesis.  In this Review, we summarize the mol. functions of protein lysine methylation and its involvement in human cancer, with a particular focus on lysine methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0wn_Jk7jSGbVg90H21EOLACvtfcHk0ljl_erc6JR-9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D&md5=9d998a4efb3c1a069e14747341ceca5e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrc3884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3884%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DSaloura%26aufirst%3DV.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DCritical%2520roles%2520of%2520non-histone%2520protein%2520lysine%2520methylation%2520in%2520human%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D110%26epage%3D124%26doi%3D10.1038%2Fnrc3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kachirskaia, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appella, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">Modulation of p53 function by SET8-mediated methylation at lysine 382</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">636</span>â <span class="NLM_lpage">646</span>, <span class="refDoi">Â DOI: 10.1016/j.molcel.2007.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.molcel.2007.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=17707234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=636-646&author=X.+Shiauthor=I.+Kachirskaiaauthor=H.+Yamaguchiauthor=L.+E.+Westauthor=H.+Wenauthor=E.+W.+Wangauthor=S.+Duttaauthor=E.+Appellaauthor=O.+Gozani&title=Modulation+of+p53+function+by+SET8-mediated+methylation+at+lysine+382&doi=10.1016%2Fj.molcel.2007.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of p53 function by SET8-mediated methylation at lysine 382</span></div><div class="casAuthors">Shi, Xiaobing; Kachirskaia, Ioulia; Yamaguchi, Hiroshi; West, Lisandra E.; Wen, Hong; Wang, Evelyn W.; Dutta, Sucharita; Appella, Ettore; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">636-646</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Reversible covalent methylation of lysine residues on histone proteins constitutes a principal mol. mechanism that links chromatin states to diverse biol. outcomes.  Recently, lysine methylation has been obsd. on non-histone proteins, suggesting broad cellular roles for the enzymes generating and removing Me moieties.  Here we report that the lysine methyltransferase enzyme SET8/PR-Set7 regulates the tumor suppressor protein p53.  We find that SET8 specifically monomethylates p53 at lysine 382 (p53K382me1).  This methylation event robustly suppresses p53-mediated transcription activation of highly responsive target genes but has little influence on weak targets.  Further, depletion of SET8 augments the proapoptotic and checkpoint activation functions of p53, and accordingly, SET8 expression is downregulated upon DNA damage.  Together, our study identifies SET8 as a p53-modifying enzyme, identifies p53K382me1 as a regulatory posttranslational modification of p53, and begins to dissect how methylation may contribute to a dynamic posttranslational code that modulates distinct p53 functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR93io05Whx7Vg90H21EOLACvtfcHk0ljl_erc6JR-9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3E&md5=df158100369f875f859a4fdf296ac2fc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DX.%26aulast%3DKachirskaia%26aufirst%3DI.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DE.%2BW.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DModulation%2520of%2520p53%2520function%2520by%2520SET8-mediated%2520methylation%2520at%2520lysine%2520382%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D636%26epage%3D646%26doi%3D10.1016%2Fj.molcel.2007.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyokawa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maejima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dohmae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Field, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsunoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akasu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnuma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponder, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span> <span> </span><span class="NLM_article-title">Histone lysine methyltransferase SETD8 promotes carcinogenesis by deregulating PCNA expression</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3217</span>â <span class="NLM_lpage">3227</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-11-3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F0008-5472.CAN-11-3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=22556262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsFKktbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=3217-3227&author=M.+Takawaauthor=H.+S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=M.+Kogureauthor=Y.+Yamaneauthor=Y.+Iwaiauthor=K.+Maejimaauthor=K.+Uedaauthor=A.+Masudaauthor=N.+Dohmaeauthor=H.+I.+Fieldauthor=T.+Tsunodaauthor=T.+Kobayashiauthor=T.+Akasuauthor=M.+Sugiyamaauthor=S.+Ohnumaauthor=Y.+Atomiauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Histone+lysine+methyltransferase+SETD8+promotes+carcinogenesis+by+deregulating+PCNA+expression&doi=10.1158%2F0008-5472.CAN-11-3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methyltransferase SETD8 Promotes Carcinogenesis by Deregulating PCNA Expression</span></div><div class="casAuthors">Takawa, Masashi; Cho, Hyun-Soo; Hayami, Shinya; Toyokawa, Gouji; Kogure, Masaharu; Yamane, Yuka; Iwai, Yukiko; Maejima, Kazuhiro; Ueda, Koji; Masuda, Akiko; Dohmae, Naoshi; Field, Helen I.; Tsunoda, Tatsuhiko; Kobayashi, Takaaki; Akasu, Takayuki; Sugiyama, Masanori; Ohnuma, Shin-ichi; Atomi, Yutaka; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3217-3227</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although the physiol. significance of lysine methylation of histones is well known, whether lysine methylation plays a role in the regulation of nonhistone proteins has not yet been examd.  The histone lysine methyltransferase SETD8 is overexpressed in various types of cancer and seems to play a crucial role in S-phase progression.  Here, we show that SETD8 regulates the function of proliferating cell nuclear antigen (PCNA) protein through lysine methylation.  We found that SETD8 methylated PCNA on lysine 248, and either depletion of SETD8 or substitution of lysine 248 destabilized PCNA expression.  Mechanistically, lysine methylation significantly enhanced the interaction between PCNA and the flap endonuclease FEN1.  Loss of PCNA methylation retarded the maturation of Okazaki fragments, slowed DNA replication, and induced DNA damage, and cells expressing a methylation-inactive PCNA mutant were more susceptible to DNA damage.  An increase of methylated PCNA was found in cancer cells, and the expression levels of SETD8 and PCNA were correlated in cancer tissue samples.  Together, our findings reveal a function for lysine methylation on a nonhistone protein and suggest that aberrant lysine methylation of PCNA may play a role in human carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor3s_ED_aVQLVg90H21EOLACvtfcHk0ljrOFtAh5_jfg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsFKktbo%253D&md5=d1d7e9969329b06ea655c76b3fb93013</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-3701%26sid%3Dliteratum%253Aachs%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DAkasu%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DOhnuma%26aufirst%3DS.%26aulast%3DAtomi%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DHistone%2520lysine%2520methyltransferase%2520SETD8%2520promotes%2520carcinogenesis%2520by%2520deregulating%2520PCNA%2520expression%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D3217%26epage%3D3227%26doi%3D10.1158%2F0008-5472.CAN-11-3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milite, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rescigno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Progress in the development of lysine methyltransferase SETD8 inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1680</span>â <span class="NLM_lpage">1685</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201600272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fcmdc.201600272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27411844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKktrzF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1680-1685&author=C.+Militeauthor=A.+Feoliauthor=M.+Vivianoauthor=D.+Rescignoauthor=A.+Maiauthor=S.+Castellanoauthor=G.+Sbardella&title=Progress+in+the+development+of+lysine+methyltransferase+SETD8+inhibitors&doi=10.1002%2Fcmdc.201600272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the development of lysine methyltransferase SETD8 inhibitors</span></div><div class="casAuthors">Milite, Ciro; Feoli, Alessandra; Viviano, Monica; Rescigno, Donatella; Mai, Antonello; Castellano, Sabrina; Sbardella, Gianluca</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1680-1685</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  SETD8/SET8/Pr-SET7/KMT5A is the only known lysine methyltransferase that monomethylates lysine 20 of histone H4 (H4K20) in vivo.  The methyltransferase activity of SETD8 has been implicated in many essential cellular processes, including DNA replication, DNA damage response, transcription modulation, and cell cycle regulation.  In addn. to H4K20, SETD8 monomethylates non-histone substrates including proliferating cell nuclear antigen and p53.  During the past decade, different structural classes of inhibitors targeting various lysine methyltransferases have been designed and developed.  However, the development of SETD8 inhibitors is still in its infancy.  This review covers the progress made to date in inhibiting the activity of SETD8 by small mols., with an emphasis on their discovery, selectivity over other methyltransferases, and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqDsZ5YVk8ybVg90H21EOLACvtfcHk0ljrOFtAh5_jfg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKktrzF&md5=0e952b0788e4df4c4bd3626a808a4f66</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600272%26sid%3Dliteratum%253Aachs%26aulast%3DMilite%26aufirst%3DC.%26aulast%3DFeoli%26aufirst%3DA.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DRescigno%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DProgress%2520in%2520the%2520development%2520of%2520lysine%2520methyltransferase%2520SETD8%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D1680%26epage%3D1685%26doi%3D10.1002%2Fcmdc.201600272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milite, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rescigno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzano, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">The emerging role of lysine methyltransferase SETD8 in human diseases</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">102</span>, <span class="refDoi">Â DOI: 10.1186/s13148-016-0268-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1186%2Fs13148-016-0268-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27688818" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=102&author=C.+Militeauthor=A.+Feoliauthor=M.+Vivianoauthor=D.+Rescignoauthor=A.+Cianciulliauthor=A.+L.+Balzanoauthor=A.+Maiauthor=S.+Castellanoauthor=G.+Sbardella&title=The+emerging+role+of+lysine+methyltransferase+SETD8+in+human+diseases&doi=10.1186%2Fs13148-016-0268-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1186%2Fs13148-016-0268-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-016-0268-4%26sid%3Dliteratum%253Aachs%26aulast%3DMilite%26aufirst%3DC.%26aulast%3DFeoli%26aufirst%3DA.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DRescigno%26aufirst%3DD.%26aulast%3DCianciulli%26aufirst%3DA.%26aulast%3DBalzano%26aufirst%3DA.%2BL.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DThe%2520emerging%2520role%2520of%2520lysine%2520methyltransferase%2520SETD8%2520in%2520human%2520diseases%26jtitle%3DClin.%2520Epigenet.%26date%3D2016%26volume%3D8%26spage%3D102%26doi%3D10.1186%2Fs13148-016-0268-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalisay, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amphlett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneerat, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matainaho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, R. J.</span></span> <span> </span><span class="NLM_article-title">Nahuoic acid a produced by a streptomyces sp. isolated from a marine sediment is a selective SAM-competitive inhibitor of the histone methyltransferase SETD8</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">414</span>â <span class="NLM_lpage">417</span>, <span class="refDoi">Â DOI: 10.1021/ol303416k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol303416k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFej" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=414-417&author=D.+E.+Williamsauthor=D.+S.+Dalisayauthor=F.+Liauthor=J.+Amphlettauthor=W.+Maneeratauthor=M.+A.+Chavezauthor=Y.+A.+Wangauthor=T.+Matainahoauthor=W.+Yuauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=R.+J.+Andersen&title=Nahuoic+acid+a+produced+by+a+streptomyces+sp.+isolated+from+a+marine+sediment+is+a+selective+SAM-competitive+inhibitor+of+the+histone+methyltransferase+SETD8&doi=10.1021%2Fol303416k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Nahuoic acid A produced by a Streptomyces sp. isolated from a marine sediment is a selective SAM-competitive inhibitor of the histone methyltransferase SETD8</span></div><div class="casAuthors">Williams, David E.; Dalisay, Doralyn S.; Li, Fengling; Amphlett, James; Maneerat, Wisanu; Chavez, Miguel Angel Garcia; Wang, Yan Alexander; Matainaho, Teatulohi; Yu, Wenyu; Brown, Peter J.; Arrowsmith, Cheryl H.; Vedadi, Masoud; Andersen, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">414-417</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone lysine monomethyltransferase SETD8 is an epigenetic regulator of cell cycle progression.  Nahuoic acid A (I), a polyketide produced in culture by a Streptomyces sp. obtained from a tropical marine sediment, is the first known selective SAM-competitive inhibitor of SETD8.  The structure of I was elucidated by chem. transformation and detailed anal. of spectroscopic data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLuQ0zAEbeO7Vg90H21EOLACvtfcHk0liyU-CUk7wsfg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFej&md5=acf39ac06135084b15b7fbeedf23a202</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fol303416k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol303416k%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DDalisay%26aufirst%3DD.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DAmphlett%26aufirst%3DJ.%26aulast%3DManeerat%26aufirst%3DW.%26aulast%3DChavez%26aufirst%3DM.%2BA.%26aulast%3DWang%26aufirst%3DY.%2BA.%26aulast%3DMatainaho%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DR.%2BJ.%26atitle%3DNahuoic%2520acid%2520a%2520produced%2520by%2520a%2520streptomyces%2520sp.%2520isolated%2520from%2520a%2520marine%2520sediment%2520is%2520a%2520selective%2520SAM-competitive%2520inhibitor%2520of%2520the%2520histone%2520methyltransferase%2520SETD8%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26spage%3D414%26epage%3D417%26doi%3D10.1021%2Fol303416k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blum, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IbÃ¡Ã±ez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djaballah, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of SETD8 with cellular activity</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2471</span>â <span class="NLM_lpage">2478</span>, <span class="refDoi">Â DOI: 10.1021/cb500515r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500515r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2471-2478&author=G.+Blumauthor=G.+Ib%C3%A1%C3%B1ezauthor=X.+Raoauthor=D.+Shumauthor=C.+Raduauthor=H.+Djaballahauthor=J.+C.+Riceauthor=M.+Luo&title=Small-molecule+inhibitors+of+SETD8+with+cellular+activity&doi=10.1021%2Fcb500515r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of SETD8 with Cellular Activity</span></div><div class="casAuthors">Blum, Gil; Ibanez, Glorymar; Rao, Xiangjun; Shum, David; Radu, Constantin; Djaballah, Hakim; Rice, Judd C.; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2471-2478</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SETD8/SET8/Pr-SET7/KMT5A is the sole protein lysine methyltransferase (PKMT) known to monomethylate lysine 20 of histone H4 in vivo.  SETD8's methyltransferase activity has been implicated in many essential cellular processes including DNA replication, DNA damage response, transcription modulation, and cell cycle regulation.  Developing SETD8 inhibitors with cellular activity is a key step toward elucidating the diverse roles of SETD8 via convenient pharmacol. perturbation.  From the hits of a prior high throughput screen (HTS), SPS8I1-3 (NSC663284, BVT948, and ryuvidine) were validated as potent SETD8 inhibitors.  These compds. contain different structural motifs and inhibit SETD8 via distinct modes.  More importantly, these compds. show cellular activity by suppressing the H4K20me1 mark of SETD8 and recapitulate characteristic S/G2/M-phase cell cycle defects as obsd. for RNAi-mediated SETD8 knockdown.  The commonality of SPS8I1-3 against SETD8, together with their distinct structures and mechanisms for SETD8 inhibition, argues for the collective application of these compds. as SETD8 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquxqpOQLgFObVg90H21EOLACvtfcHk0liyU-CUk7wsfg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE&md5=87b7c415bf82577843fd5eeb09c2c746</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fcb500515r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500515r%26sid%3Dliteratum%253Aachs%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DIb%25C3%25A1%25C3%25B1ez%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DX.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DRadu%26aufirst%3DC.%26aulast%3DDjaballah%26aufirst%3DH.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520SETD8%2520with%2520cellular%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2471%26epage%3D2478%26doi%3D10.1021%2Fcb500515r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiewiora, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PittellaSilva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beauchamp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodera, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">The Dynamic Conformational Landscape of the Protein Methyltransferase SETD8</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e45403</span> <span class="refDoi">Â DOI: 10.7554/eLife.45403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.7554%2FeLife.45403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31081496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlals7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&author=S.+Chenauthor=R.+P.+Wiewioraauthor=F.+Mengauthor=N.+Babaultauthor=A.+Maauthor=W.+Yuauthor=K.+Qianauthor=H.+Huauthor=H.+Zouauthor=J.+Wangauthor=S.+Fanauthor=G.+Blumauthor=F.+PittellaSilvaauthor=K.+A.+Beauchampauthor=W.+Tempelauthor=H.+Jiangauthor=K.+Chenauthor=R.+Skeneauthor=Y.+G.+Zhengauthor=P.+J.+Brownauthor=J.+Jinauthor=C.+Luoauthor=J.+D.+Choderaauthor=M.+Luo&title=The+Dynamic+Conformational+Landscape+of+the+Protein+Methyltransferase+SETD8&doi=10.7554%2FeLife.45403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic conformational landscape of the protein methyltransferase SETD8</span></div><div class="casAuthors">Chen, Shi; Wiewiora, Rafal P.; Meng, Fanwang; Babault, Nicolas; Ma, Anqi; Yu, Wenyu; Qian, Kun; Hu, Hao; Zou, Hua; Wang, Junyi; Fan, Shijie; Blum, Gil; Pittella-Silva, Fabio; Beauchamp, Kyle A.; Tempel, Wolfram; Jiang, Hualiang; Chen, Kaixian; Skene, Robert J.; Zheng, Yujun George; Brown, Peter J.; Jin, Jian; Luo, Cheng; Chodera, John D.; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e45403/1-e45403/76</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Elucidating the conformational heterogeneity of proteins is essential for understanding protein function and developing exogenous ligands.  With the rapid development of exptl. and computational methods, it is of great interest to integrate these approaches to illuminate the conformational landscapes of target proteins.  SETD8 is a protein lysine methyltransferase (PKMT), which functions in vivo via the methylation of histone and nonhistone targets.  Utilizing covalent inhibitors and depleting native ligands to trap hidden conformational states, we obtained diverse X-ray structures of SETD8.  These structures were used to seed distributed atomistic mol. dynamics simulations that generated a total of six milliseconds of trajectory data.  Markov state models, built via an automated machine learning approach and corroborated exptl., reveal how slow conformational motions and conformational states are relevant to catalysis.  These findings provide mol. insight on enzymic catalysis and allosteric mechanisms of a PKMT via its detailed conformational landscape.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovPJl3tz4iu7Vg90H21EOLACvtfcHk0liyU-CUk7wsfg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlals7nL&md5=db09ad2c975d64615a594c391011a607</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.7554%2FeLife.45403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.45403%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWiewiora%26aufirst%3DR.%2BP.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DPittellaSilva%26aufirst%3DF.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DThe%2520Dynamic%2520Conformational%2520Landscape%2520of%2520the%2520Protein%2520Methyltransferase%2520SETD8%26jtitle%3DeLife%26date%3D2019%26volume%3D8%26doi%3D10.7554%2FeLife.45403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">PAINS in the assay: Chemical mechanisms of assay interference and promiscuous enzymatic inhibition observed during a sulfhydryl-scavenging HTS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2091</span>â <span class="NLM_lpage">2113</span>, <span class="refDoi">Â DOI: 10.1021/jm5019093</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5019093" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeisbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2091-2113&author=J.+L.+Dahlinauthor=J.+W.+Nissinkauthor=J.+M.+Strasserauthor=S.+Francisauthor=L.+Higginsauthor=H.+Zhouauthor=Z.+Zhangauthor=M.+A.+Walters&title=PAINS+in+the+assay%3A+Chemical+mechanisms+of+assay+interference+and+promiscuous+enzymatic+inhibition+observed+during+a+sulfhydryl-scavenging+HTS&doi=10.1021%2Fjm5019093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PAINS in the Assay: Chemical Mechanisms of Assay Interference and Promiscuous Enzymatic Inhibition Observed during a Sulfhydryl-Scavenging HTS</span></div><div class="casAuthors">Dahlin, Jayme L.; Nissink, J. Willem M.; Strasser, Jessica M.; Francis, Subhashree; Higgins, LeeAnn; Zhou, Hui; Zhang, Zhiguo; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2091-2113</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Significant resources in early drug discovery are spent unknowingly pursuing artifacts and promiscuous bioactive compds., while understanding the chem. basis for these adverse behaviors often goes unexplored in pursuit of lead compds.  Nearly all the hits from our recent sulfhydryl-scavenging high-throughput screen (HTS) targeting the histone acetyltransferase Rtt109 were such compds.  Herein, we characterize the chem. basis for assay interference and promiscuous enzymic inhibition for several prominent chemotypes identified by this HTS, including some pan-assay interference compds. (PAINS).  Protein mass spectrometry and ALARM NMR confirmed these compds. react covalently with cysteines on multiple proteins.  Unfortunately, compds. contg. these chemotypes have been published as screening actives in reputable journals and even touted as chem. probes or preclin. candidates.  Our detailed characterization and identification of such thiol-reactive chemotypes should accelerate triage of nuisance compds., guide screening library design, and prevent follow-up on undesirable chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4XTJapoezkrVg90H21EOLACvtfcHk0ljLsgl3z8a93w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeisbY%253D&md5=3f77702544cea2ec4c8184b60dd38b8a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm5019093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5019093%26sid%3Dliteratum%253Aachs%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DStrasser%26aufirst%3DJ.%2BM.%26aulast%3DFrancis%26aufirst%3DS.%26aulast%3DHiggins%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DPAINS%2520in%2520the%2520assay%253A%2520Chemical%2520mechanisms%2520of%2520assay%2520interference%2520and%2520promiscuous%2520enzymatic%2520inhibition%2520observed%2520during%2520a%2520sulfhydryl-scavenging%2520HTS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2091%26epage%3D2113%26doi%3D10.1021%2Fjm5019093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittella-Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a covalent inhibitor of the SET domain-containing protein 8 (SETD8) lysine methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9881</span>â <span class="NLM_lpage">9889</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01244</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01244" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9881-9889&author=K.+V.+Butlerauthor=A.+Maauthor=W.+Yuauthor=F.+Liauthor=W.+Tempelauthor=N.+Babaultauthor=F.+Pittella-Silvaauthor=J.+Shaoauthor=J.+Wangauthor=M.+Luoauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Structure-based+design+of+a+covalent+inhibitor+of+the+SET+domain-containing+protein+8+%28SETD8%29+lysine+methyltransferase&doi=10.1021%2Facs.jmedchem.6b01244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase</span></div><div class="casAuthors">Butler, Kyle V.; Ma, Anqi; Yu, Wenyu; Li, Fengling; Tempel, Wolfram; Babault, Nicolas; Pittella-Silva, Fabio; Shao, Jason; Wang, Junyi; Luo, Minkui; Vedadi, Masoud; Brown, Peter J.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9881-9889</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of protein lysine methyltransferases, including SET domain-contg. protein 8 (SETD8), are highly desired, as only a fraction of these enzymes are assocd. with high-quality inhibitors.  From our previously discovered SETD8 inhibitor, we developed a more potent analog and solved a cocrystal structure, which is the first crystal structure of SETD8 in complex with a small-mol. inhibitor.  This cocrystal structure allowed the design of a covalent inhibitor of SETD8 (MS453), which specifically modifies a cysteine residue near the inhibitor binding site, has an IC50 value of 804 nM, reacts with SETD8 with near-quant. yield, and is selective for SETD8 against 28 other methyltransferases.  We also solved the crystal structure of the covalent inhibitor in complex with SETD8.  This work provides at.-level perspective on the inhibition of SETD8 by small mols. and will help identify high-quality chem. probes of SETD8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjPurua1qCK7Vg90H21EOLACvtfcHk0ljLsgl3z8a93w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE&md5=b4d0b38dfe3a911633aad17642e78134</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DPittella-Silva%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DStructure-based%2520design%2520of%2520a%2520covalent%2520inhibitor%2520of%2520the%2520SET%2520domain-containing%2520protein%25208%2520%2528SETD8%2529%2520lysine%2520methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9881%26epage%3D9889%26doi%3D10.1021%2Facs.jmedchem.6b01244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleich, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective, substrate-competitive inhibitor of the lysine methyltransferase SETD8</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6822</span>â <span class="NLM_lpage">6833</span>, <span class="refDoi">Â DOI: 10.1021/jm500871s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500871s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6822-6833&author=A.+Maauthor=W.+Yuauthor=F.+Liauthor=R.+M.+Bleichauthor=J.+M.+Heroldauthor=K.+V.+Butlerauthor=J.+L.+Norrisauthor=V.+Korboukhauthor=A.+Tripathyauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=M.+Vedadiauthor=P.+J.+Brownauthor=J.+Jin&title=Discovery+of+a+selective%2C+substrate-competitive+inhibitor+of+the+lysine+methyltransferase+SETD8&doi=10.1021%2Fjm500871s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span></div><div class="casAuthors">Ma, Anqi; Yu, Wenyu; Li, Fengling; Bleich, Rachel M.; Herold, J. Martin; Butler, Kyle V.; Norris, Jacqueline L.; Korboukh, Victoria; Tripathy, Ashutosh; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Vedadi, Masoud; Brown, Peter J.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6822-6833</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20).  Monomethylation of H4K20 has been implicated in regulating diverse biol. processes including the DNA damage response.  In addn. to H4K20, SETD8 monomethylates nonhistone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression.  However, selective inhibitors of SETD8 are scarce.  The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor.  Here, the authors report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 I.  This small-mol. inhibitor is active in multiple biochem. assays.  Its affinity to SETD8 was confirmed by ITC (isothermal titrn. calorimetry) and SPR (surface plasmon resonance) studies.  Importantly, compd. I is selective for SETD8 over 15 other methyltransferases.  The authors also describe structure-activity relationships (SAR) of this series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR9y31FWBfY7Vg90H21EOLACvtfcHk0ljLsgl3z8a93w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI&md5=adca277f02033f96fe45a28087dca14e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500871s%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBleich%26aufirst%3DR.%2BM.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520selective%252C%2520substrate-competitive%2520inhibitor%2520of%2520the%2520lysine%2520methyltransferase%2520SETD8%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6822%26epage%3D6833%26doi%3D10.1021%2Fjm500871s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clancy, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Chemical biology of protein arginine modifications in epigenetic regulation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">5413</span>â <span class="NLM_lpage">5461</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.5b00003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=5413-5461&author=J.+Fuhrmannauthor=K.+W.+Clancyauthor=P.+R.+Thompson&title=Chemical+biology+of+protein+arginine+modifications+in+epigenetic+regulation&doi=10.1021%2Facs.chemrev.5b00003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology of Protein Arginine Modifications in Epigenetic Regulation</span></div><div class="casAuthors">Fuhrmann, Jakob; Clancy, Kathleen W.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5413-5461</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Here we aim to summarize the current knowledge surrounding the posttranslational modification of histone arginine residues, focusing on enzyme classes that catalyze the citrullination and methylation of arginine residues as well as noncanonical arginine modifications such as phosphorylation, ADP-ribosylation, and arginylation.  The major focus will be given to the protein arginine deiminases and protein arginine methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdzgTmC4C4fbVg90H21EOLACvtfcHk0lgIj60TiAfXuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D&md5=a6df90a99a606e1d0a7296fa95625202</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00003%26sid%3Dliteratum%253Aachs%26aulast%3DFuhrmann%26aufirst%3DJ.%26aulast%3DClancy%26aufirst%3DK.%2BW.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DChemical%2520biology%2520of%2520protein%2520arginine%2520modifications%2520in%2520epigenetic%2520regulation%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D5413%26epage%3D5461%26doi%3D10.1021%2Facs.chemrev.5b00003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira de Freitas, R.</span></span> <span> </span><span class="NLM_article-title">Structural biology and chemistry of protein arginine methyltransferases</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1779</span>â <span class="NLM_lpage">1788</span>, <span class="refDoi">Â DOI: 10.1039/C4MD00269E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1039%2FC4MD00269E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26693001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSiu7%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1779-1788&author=M.+Schapiraauthor=R.+Ferreira+de+Freitas&title=Structural+biology+and+chemistry+of+protein+arginine+methyltransferases&doi=10.1039%2FC4MD00269E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology and chemistry of protein arginine methyltransferases</span></div><div class="casAuthors">Schapira, Matthieu; Ferreira de Freitas, Renato</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1779-1788</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs), an emerging target class in drug discovery, can methylate histones and other substrates, and can be divided into three subgroups, based on the methylation pattern of the reaction product (monomethylation, sym. or asym. dimethylation).  Here, we review the growing body of structural information characterizing this protein family, including structures in complex with substrate-competitive and allosteric inhibitors.  We outline structural differences between type I, II and III enzymes and propose a model underlying class-specificity.  We analyze the structural plasticity and diversity of the substrate, cofactor and allosteric binding sites, and propose that the conformational dynamics of PRMTs can be exploited towards the discovery of allosteric inhibitors that would antagonize conformationally active states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFIT9cSgrvLVg90H21EOLACvtfcHk0lgIj60TiAfXuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSiu7%252FM&md5=4b092e8fd36283b625abb9738a496bad</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2FC4MD00269E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00269E%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26atitle%3DStructural%2520biology%2520and%2520chemistry%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1779%26epage%3D1788%26doi%3D10.1039%2FC4MD00269E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druzina, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheyi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungheim, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emtage, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the human PRMT5:MEP50 complex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17960</span>â <span class="NLM_lpage">17965</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1209814109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1073%2Fpnas.1209814109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23071334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17960-17965&author=S.+Antonysamyauthor=Z.+Bondayauthor=R.+M.+Campbellauthor=B.+Doyleauthor=Z.+Druzinaauthor=T.+Gheyiauthor=B.+Hanauthor=L.+N.+Jungheimauthor=Y.+Qianauthor=C.+Rauchauthor=M.+Russellauthor=J.+M.+Sauderauthor=S.+R.+Wassermanauthor=K.+Weichertauthor=F.+S.+Willardauthor=A.+Zhangauthor=S.+Emtage&title=Crystal+structure+of+the+human+PRMT5%3AMEP50+complex&doi=10.1073%2Fpnas.1209814109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human PRMT5:MEP50 complex</span></div><div class="casAuthors">Antonysamy, Stephen; Bonday, Zahid; Campbell, Robert M.; Doyle, Brandon; Druzina, Zhanna; Gheyi, Tarun; Han, Bomie; Jungheim, Louis N.; Qian, Yuewei; Rauch, Charles; Russell, Marijane; Sauder, J. Michael; Wasserman, Stephen R.; Wichert, Kenneth; Willard, Francis S.; Zhang, Aiping; Emtage, Spencer</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17960-17965, S17960/1-S17960/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play important roles in several cellular processes, including signaling, gene regulation, and transport of proteins and nucleic acids, to impact growth, differentiation, proliferation, and development.  PRMT5 sym. di-methylates the two-terminal Ï-guanidino nitrogens of arginine residues on substrate proteins.  PRMT5 acts as part of a multimeric complex in concert with a variety of partner proteins that regulate its function and specificity.  A core component of these complexes is the WD40 protein MEP50/WDR77/p44, which mediates interactions with binding partners and substrates.  We have detd. the crystal structure of human PRMT5 in complex with MEP50 (methylosome protein 50), bound to an S-adenosylmethionine analog and a peptide substrate derived from histone H4.  The structure of the surprising hetero-octameric complex reveals the close interaction between the seven-bladed Î²-propeller MEP50 and the N-terminal domain of PRMT5, and delineates the structural elements of substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcWBh5CVI7LVg90H21EOLACvtfcHk0lgIj60TiAfXuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL&md5=757af18cfd0787a02c88c840273109aa</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209814109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209814109%26sid%3Dliteratum%253Aachs%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DDoyle%26aufirst%3DB.%26aulast%3DDruzina%26aufirst%3DZ.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DJungheim%26aufirst%3DL.%2BN.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DSauder%26aufirst%3DJ.%2BM.%26aulast%3DWasserman%26aufirst%3DS.%2BR.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DWillard%26aufirst%3DF.%2BS.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DEmtage%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17960%26epage%3D17965%26doi%3D10.1073%2Fpnas.1209814109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span> <span> </span><span class="NLM_article-title">Targeting protein arginine methyltransferase 5 in disease</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2081</span>â <span class="NLM_lpage">2098</span>, <span class="refDoi">Â DOI: 10.4155/fmc-2017-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.4155%2Ffmc-2017-0089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29076773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=2081-2098&author=A.+Richters&title=Targeting+protein+arginine+methyltransferase+5+in+disease&doi=10.4155%2Ffmc-2017-0089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein arginine methyltransferase 5 in disease</span></div><div class="casAuthors">Richters, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2081-2098</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">PRMT5 catalyzes the mono- and sym. dimethylation of the arginine N-guanidine group of a wide variety of target proteins including histones, transcriptional elongation factors, kinases and tumor suppressors by utilizing the essential co-factor S-adenosylmethionine as Me source.  PRMT5 overexpression has been linked to the progression of various diseases, including cancer, and is oftentimes assocd. with a poor prognosis.  Therefore, PRMT5 is promoted as a valuable target for drug discovery approaches and was a subject matter in recent endeavors aiming for the development of specific PRMT5 inhibitors.  This review will embrace the significance of PRMT5 as therapeutic target with respect to its mol. interdependencies in disease states as well as its implication in drug development approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqverMGPRhuJbVg90H21EOLACvtfcHk0li3UfR7-zWDnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN&md5=6f8f3e27b9d7389e5b89ff084e6b691e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0089%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26atitle%3DTargeting%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520disease%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D2081%26epage%3D2098%26doi%3D10.4155%2Ffmc-2017-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">A patent review of arginine methyltransferase inhibitors (2010â2018)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">97</span>â <span class="NLM_lpage">114</span>, <span class="refDoi">Â DOI: 10.1080/13543776.2019.1567711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F13543776.2019.1567711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30640571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitF2murk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=97-114&author=X.+Liauthor=C.+Wangauthor=H.+Jiangauthor=C.+Luo&title=A+patent+review+of+arginine+methyltransferase+inhibitors+%282010%E2%80%932018%29&doi=10.1080%2F13543776.2019.1567711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A patent review of arginine methyltransferase inhibitors (2010-2018)</span></div><div class="casAuthors">Li, Xiao; Wang, Chen; Jiang, Hao; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-114</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are fundamental enzymes that specifically modify the arginine residues of versatile substrates in cells.  The aberrant expression and abnormal enzymic activity of PRMTs are assocd. with many human diseases, esp. cancer.  PRMTs are emerging as promising drug targets in both academia and industry.  This review summarizes the updated patented inhibitors targeting PRMTs from 2010 to 2018.  The authors illustrate the chem. structures, mol. mechanism of action, pharmacol. activities as well as the potential clin. application including combination therapy and biomarker-guided therapy.  PRMT inhibitors in clin. trials are also highlighted.  The authors provide a future perspective for further development of potent and selective PRMT inhibitors.  Although a no. of small mol. inhibitors of PRMTs with sufficient potency have been developed, the selectivity of most PRMT inhibitors remains to be improved.  Hence, novel approaches such as allosteric regulation need to be further studied to identify PRMT inhibitors.  So far, three PRMT inhibitors have entered clin. trials, including PRMT5 inhibitor GSK3326595 and JNJ-64619178 as well as PRMT1 inhibitor GSK3368715.  PRMT inhibitors with novel mechanism of action and good drug-like properties may shed new light on drug research and development progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPH62wM5Eh8bVg90H21EOLACvtfcHk0li3UfR7-zWDnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitF2murk%253D&md5=196b7ad8fa047aad498c984bd1b72b15</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1567711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1567711%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DA%2520patent%2520review%2520of%2520arginine%2520methyltransferase%2520inhibitors%2520%25282010%25E2%2580%25932018%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2019%26volume%3D29%26spage%3D97%26epage%3D114%26doi%3D10.1080%2F13543776.2019.1567711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindiapina, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Recent advances in targeting protein arginine methyltransferase enzymes in cancer therapy</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">527</span>â <span class="NLM_lpage">545</span>, <span class="refDoi">Â DOI: 10.1080/14728222.2018.1474203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F14728222.2018.1474203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29781349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BC1Mflt1GntQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=527-545&author=E.+Smithauthor=W.+Zhouauthor=P.+Shindiapinaauthor=S.+Sifauthor=C.+Liauthor=R.+A.+Baiocchi&title=Recent+advances+in+targeting+protein+arginine+methyltransferase+enzymes+in+cancer+therapy&doi=10.1080%2F14728222.2018.1474203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in targeting protein arginine methyltransferase enzymes in cancer therapy</span></div><div class="casAuthors">Smith Emily; Shindiapina Polina; Baiocchi Robert A; Zhou Wei; Li Chenglong; Sif Said; Li Chenglong</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on therapeutic targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">527-545</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Exploration in the field of epigenetics has revealed the diverse roles of the protein arginine methyltransferase (PRMT) family of proteins in multiple disease states.  These findings have led to the development of specific inhibitors and discovery of several new classes of drugs with potential to treat both benign and malignant conditions.  Areas covered: We provide an overview on the role of PRMT enzymes in healthy and malignant cells, highlighting the role of arginine methylation in specific pathways relevant to cancer pathogenesis.  Additionally, we describe structure and catalytic activity of PRMT and discuss the mechanisms of action of novel small molecule inhibitors of specific members of the arginine methyltransferase family.  Expert opinion: As the field of PRMT biology advances, it's becoming clear that this class of enzymes is highly relevant to maintaining normal physiologic processes as well and disease pathogenesis.  We discuss the potential impact of PRMT inhibitors as a broad class of drugs, including the pleiotropic effects, off target effects the need for more detailed PRMT-centric interactomes, and finally, the potential for targeting this class of enzymes in clinical development of experimental therapeutics for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkL15Wkj-iUfx2phytoz6_fW6udTcc2ebZIy6lYNplIrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mflt1GntQ%253D%253D&md5=413ff9fbe878fc14fcc95f6a133918ba</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1474203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1474203%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShindiapina%26aufirst%3DP.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DRecent%2520advances%2520in%2520targeting%2520protein%2520arginine%2520methyltransferase%2520enzymes%2520in%2520cancer%2520therapy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D527%26epage%3D545%26doi%3D10.1080%2F14728222.2018.1474203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>â <span class="NLM_lpage">6304</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 Î¼M) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 Î¼M.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lgRHf0MGnFd7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-substituted-<i>N</i>-(3-(3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide as potent and selective protein arginine methyltransferases 5 inhibitors: Design, synthesis and biological evaluation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">317</span>â <span class="NLM_lpage">333</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.12.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2018.12.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30605830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtl2nsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2019&pages=317-333&author=J.+Shaoauthor=K.+Zhuauthor=D.+Duauthor=Y+Zhangauthor=H.+Taoauthor=Z.+Chenauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=W.+Duan&title=Discovery+of+2-substituted-N-%283-%283%2C4-dihydroisoquinolin-2%281H%29-yl%29-2-hydroxypropyl%29-1%2C2%2C3%2C4-tetrahydroisoquinoline-6-carboxamide+as+potent+and+selective+protein+arginine+methyltransferases+5+inhibitors%3A+Design%2C+synthesis+and+biological+evaluation&doi=10.1016%2Fj.ejmech.2018.12.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-substituted-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)-2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide as potent and selective protein arginine methyltransferases 5 inhibitors: Design, synthesis and biological evaluation</span></div><div class="casAuthors">Shao, Jingwei; Zhu, Kongkai; Du, Daohai; Zhang, Yuanyuan; Tao, Hongrui; Chen, Zhifeng; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Duan, Wenhu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">317-333</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)-2-hydroxypropyl)amide derivs. targeting PRMT5 were designed with structure-based approach and synthesized.  Among them, compd. (S)-2-(4-bromobenzoyl)-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide showed potent and selective PRMT5 inhibition activity with an IC50 of 8.5 nM, which was approx. equiv. with the phase I clin. trial PRMT5 inhibitor GSK-3326595 (IC50 = 5.5 nM).  Compd. (S)-2-(4-bromobenzoyl)-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide also displayed pronounced anti-proliferative activity in MV4-11 cells (GI50 = 18 nM) and antitumor activity in MV4-11 mouse xenografts model.  This mol. served as an excellent tool compd. for probing the biol. function of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8r5tDk63g07Vg90H21EOLACvtfcHk0lgRHf0MGnFd7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtl2nsw%253D%253D&md5=d04aaa30e120c8e89d0f9778130cb800</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.12.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.12.065%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DDu%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY%26aulast%3DTao%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DW.%26atitle%3DDiscovery%2520of%25202-substituted-N-%25283-%25283%252C4-dihydroisoquinolin-2%25281H%2529-yl%2529-2-hydroxypropyl%2529-1%252C2%252C3%252C4-tetrahydroisoquinoline-6-carboxamide%2520as%2520potent%2520and%2520selective%2520protein%2520arginine%2520methyltransferases%25205%2520inhibitors%253A%2520Design%252C%2520synthesis%2520and%2520biological%2520evaluation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D164%26spage%3D317%26epage%3D333%26doi%3D10.1016%2Fj.ejmech.2018.12.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 (PRMT5) as an anticancer target and its inhibitor discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9429</span>â <span class="NLM_lpage">9441</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00598</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00598" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOmsLvJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9429-9441&author=Y.+Wangauthor=W.+Huauthor=Y.+Yuan&title=Protein+arginine+methyltransferase+5+%28PRMT5%29+as+an+anticancer+target+and+its+inhibitor+discovery&doi=10.1021%2Facs.jmedchem.8b00598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery</span></div><div class="casAuthors">Wang, Yuanxiang; Hu, Wenhao; Yuan, Yanqiu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9429-9441</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRMT5 is a major enzyme responsible for sym. dimethylation of arginine residues on both histone and non-histone proteins, regulating many biol. pathways in mammalian cells.  PRMT5 has been suggested as a therapeutic target in a variety of diseases including infectious disease, heart disease, and cancer.  Many PRMT5 inhibitors have been discovered in the past 5 years, and one entered clin. trial in 2015 for the treatment of solid tumor and mantle cell lymphoma (MCL).  The aim of this review is to summarize the current understanding of the roles of PRMT5 in cancer and the discovery of PRMT5 enzymic inhibitors.  By reviewing the structure-activity relationship (SAR) of known inhibitors of PRMT5, we hope to provide guidance for future drug designs and inhibitor optimization.  Opportunities and limitations of PRMT5 inhibitors for the treatment of cancer are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG2MM_2r6cc7Vg90H21EOLACvtfcHk0lgnC2NVlYZklw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOmsLvJ&md5=1fffdd9a78a5edbe4c31357d3e08e2d8</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00598%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DYuan%26aufirst%3DY.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520as%2520an%2520anticancer%2520target%2520and%2520its%2520inhibitor%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9429%26epage%3D9441%26doi%3D10.1021%2Facs.jmedchem.8b00598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>â <span class="NLM_lpage">437</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lgnC2NVlYZklw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter
Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Structure and property guided design in the identification of PRMT5 tool compound EPZ015666</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">162</span>â <span class="NLM_lpage">166</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=162-166&author=K.+W.+Duncanauthor=N.+Riouxauthor=P.+A.+Boriack-Sjodinauthor=M.+J.+Munchhofauthor=L.+A.+Reiterauthor=C.+R.+Majerauthor=L.+Jinauthor=L.+D.+Johnstonauthor=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=M.+Porter%0AScottauthor=R.+M.+Pollockauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=Structure+and+property+guided+design+in+the+identification+of+PRMT5+tool+compound+EPZ015666&doi=10.1021%2Facsmedchemlett.5b00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Property Guided Design in the Identification of PRMT5 Tool Compound EPZ015666</span></div><div class="casAuthors">Duncan, Kenneth W.; Rioux, Nathalie; Boriack-Sjodin, P. Ann; Munchhof, Michael J.; Reiter, Lawrence A.; Majer, Christina R.; Jin, Lei; Johnston, L. Danielle; Chan-Penebre, Elayne; Kuplast, Kristy G.; Porter Scott, Margaret; Pollock, Roy M.; Waters, Nigel J.; Smith, Jesse J.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-166</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent publication of a potent and selective inhibitor of protein methyltransferase 5 (PRMT5) provides the scientific community with in vivo-active tool compd. I (EPZ015666, GSK3235025) to probe the underlying pharmacol. of this key enzyme.  Herein, the authors report the design and optimization strategies employed on an initial hit compd. with poor in vitro clearance to yield in vivo tool compd. I and an addnl. potent in vitro tool mol. II (GSK3203591).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V7LiQgVZjrVg90H21EOLACvtfcHk0lgnC2NVlYZklw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L&md5=e7ac92c2d985da4994e9bb959adededf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00380%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DReiter%26aufirst%3DL.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DStructure%2520and%2520property%2520guided%2520design%2520in%2520the%2520identification%2520of%2520PRMT5%2520tool%2520compound%2520EPZ015666%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D162%26epage%3D166%26doi%3D10.1021%2Facsmedchemlett.5b00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerhart, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellner, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montes de Oca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span> <span> </span><span class="NLM_article-title">Activation of the p53-MDM4 regulatory axis defines the anti-tumour response to PRMT5 inhibition through its role in regulating cellular splicing</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">9711</span>, <span class="refDoi">Â DOI: 10.1038/s41598-018-28002-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fs41598-018-28002-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29946150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FitFGmsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9711&author=S.+V.+Gerhartauthor=W.+A.+Kellnerauthor=C.+Thompsonauthor=M.+B.+Pappalardiauthor=X.+P.+Zhangauthor=R.+Montes+de+Ocaauthor=E.+Penebreauthor=K.+Duncanauthor=A.+Boriack-Sjodinauthor=B.+Leauthor=C.+Majerauthor=M.+T.+McCabeauthor=C.+Carpenterauthor=N.+Johnsonauthor=R.+G.+Krugerauthor=O.+Barbash&title=Activation+of+the+p53-MDM4+regulatory+axis+defines+the+anti-tumour+response+to+PRMT5+inhibition+through+its+role+in+regulating+cellular+splicing&doi=10.1038%2Fs41598-018-28002-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the p53-MDM4 regulatory axis defines the anti-tumour response to PRMT5 inhibition through its role in regulating cellular splicing</span></div><div class="casAuthors">Gerhart Sarah V; Kellner Wendy A; Thompson Christine; Pappalardi Melissa B; Zhang Xi-Ping; Montes de Oca Rocio; Le BaoChau; McCabe Michael T; Carpenter Chris; Johnson Neil; Kruger Ryan G; Barbash Olena; Penebre Elayne; Duncan Kenneth; Boriack-Sjodin Ann; Majer Christina; Carpenter Chris</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9711</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Evasion of the potent tumour suppressor activity of p53 is one of the hurdles that must be overcome for cancer cells to escape normal regulation of cellular proliferation and survival.  In addition to frequent loss of function mutations, p53 wild-type activity can also be suppressed post-translationally through several mechanisms, including the activity of PRMT5.  Here we describe broad anti-proliferative activity of potent, selective, reversible inhibitors of protein arginine methyltransferase 5 (PRMT5) including GSK3326595 in human cancer cell lines representing both hematologic and solid malignancies.  Interestingly, PRMT5 inhibition activates the p53 pathway via the induction of alternative splicing of MDM4.  The MDM4 isoform switch and subsequent p53 activation are critical determinants of the response to PRMT5 inhibition suggesting that the integrity of the p53-MDM4 regulatory axis defines a subset of patients that could benefit from treatment with GSK3326595.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBwMOdegb-Q-_Q9URG6Dx8fW6udTcc2eZ_V4Fb5u8tybntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FitFGmsg%253D%253D&md5=f61573ec23278ba5a69df3da1531ec97</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-28002-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-28002-y%26sid%3Dliteratum%253Aachs%26aulast%3DGerhart%26aufirst%3DS.%2BV.%26aulast%3DKellner%26aufirst%3DW.%2BA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DZhang%26aufirst%3DX.%2BP.%26aulast%3DMontes%2Bde%2BOca%26aufirst%3DR.%26aulast%3DPenebre%26aufirst%3DE.%26aulast%3DDuncan%26aufirst%3DK.%26aulast%3DBoriack-Sjodin%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DMajer%26aufirst%3DC.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DN.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DBarbash%26aufirst%3DO.%26atitle%3DActivation%2520of%2520the%2520p53-MDM4%2520regulatory%2520axis%2520defines%2520the%2520anti-tumour%2520response%2520to%2520PRMT5%2520inhibition%2520through%2520its%2520role%2520in%2520regulating%2520cellular%2520splicing%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D9711%26doi%3D10.1038%2Fs41598-018-28002-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">612</span>â <span class="NLM_lpage">617</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 Â± 3 nM and 25 Â± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0lgxCsU5arx-aQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Nucleoside protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1264</span>â <span class="NLM_lpage">1269</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2019.03.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.bmcl.2019.03.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30956011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmslOit70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1264-1269&author=H.+Linauthor=J.+I.+Luengo&title=Nucleoside+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1016%2Fj.bmcl.2019.03.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoside protein arginine methyltransferase 5 (PRMT5) inhibitors</span></div><div class="casAuthors">Lin, Hong; Luengo, Juan I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1264-1269</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein Arginine Methyltransferase 5 (PRMT5) is known to sym. dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 selective inhibitors, GSK3326595, a substrate competitive inhibitor, and JNJ64619178, a SAM (S-adenosyl-L-methionine) mimetic/competitive inhibitor, have entered clinic trials for multiple cancer types.  This review focuses on the recent developments in SAM mimetic nucleoside PRMT5 inhibitors, their SAR and structural insight based on published co-crystal structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWzpjRzaVPBLVg90H21EOLACvtfcHk0lgtXFlSBhw33g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmslOit70%253D&md5=f5889f2c5e084dc231f865805a241da6</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.03.042%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DNucleoside%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1264%26epage%3D1269%26doi%3D10.1016%2Fj.bmcl.2019.03.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Brehmer, D.</span>; <span class="NLM_string-name">Beke, L.</span>; <span class="NLM_string-name">Boeckx, A.</span>; <span class="NLM_string-name">Diels, G. S. M.</span>; <span class="NLM_string-name">Gilissen, R. A. H. J.</span>; <span class="NLM_string-name">Lawson, E. C.</span>; <span class="NLM_string-name">Pande, V.</span>; <span class="NLM_string-name">Parade, M. C. B. C.</span>; <span class="NLM_string-name">Schepens, W. B. G.</span></span> <span> </span><span class="NLM_article-title">Novel 6â6 Bicyclic Aromatic Ring Substituted Nucleoside Analogues for Use as PRMT5 Inhibitors</span>. <span class="NLM_patent">WO 2017032840 A1</span>, August 25, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=T.+Wu&author=D.+Brehmer&author=L.+Beke&author=A.+Boeckx&author=G.+S.+M.+Diels&author=R.+A.+H.+J.+Gilissen&author=E.+C.+Lawson&author=V.+Pande&author=M.+C.+B.+C.+Parade&author=W.+B.+G.+Schepens&title=Novel+6%E2%80%936+Bicyclic+Aromatic+Ring+Substituted+Nucleoside+Analogues+for+Use+as+PRMT5+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26atitle%3DNovel%25206%25E2%2580%25936%2520Bicyclic%2520Aromatic%2520Ring%2520Substituted%2520Nucleoside%2520Analogues%2520for%2520Use%2520as%2520PRMT5%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J. H.</span>; <span class="NLM_string-name">McAlpine, I. J.</span>; <span class="NLM_string-name">Tran-Dube, M. B.</span>; <span class="NLM_string-name">Rui, E. Y.</span>; <span class="NLM_string-name">Wythes, M. J.</span>; <span class="NLM_string-name">Kumpf, R. A.</span>; <span class="NLM_string-name">McTigue, M. A.</span></span> <span> </span><span class="NLM_article-title">Substituted Nucleoside Derivatives Useful as Anticancer Agents</span>. <span class="NLM_patent">US 20160244475 A1</span>, February 17, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+H.+Tatlock&author=I.+J.+McAlpine&author=M.+B.+Tran-Dube&author=E.+Y.+Rui&author=M.+J.+Wythes&author=R.+A.+Kumpf&author=M.+A.+McTigue&title=Substituted+Nucleoside+Derivatives+Useful+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26atitle%3DSubstituted%2520Nucleoside%2520Derivatives%2520Useful%2520as%2520Anticancer%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1033</span>â <span class="NLM_lpage">1038</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.9b00074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1033-1038&author=H.+Linauthor=M.+Wangauthor=Y.+W.+Zhangauthor=S.+Tongauthor=R.+A.+Lealauthor=R.+Shettyauthor=K.+Vaddiauthor=J.+I.+Luengo&title=Discovery+of+potent+and+selective+covalent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1021%2Facsmedchemlett.9b00074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors</span></div><div class="casAuthors">Lin Hong; Wang Min; Zhang Yang W; Leal Raul A; Shetty Rupa; Vaddi Kris; Luengo Juan I; Tong Shuilong</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1033-1038</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is known to symmetrically dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 possesses a cysteine (C449) in the active site, unique to PRMT5.  Therefore, covalent PRMT5 inhibition is an attractive chemical approach.  Herein, we report an exciting discovery of a series of novel hemiaminals that under physiological conditions can be converted to aldehydes and react with C449 to form covalent adducts, which presumably undergo an unprecedented elimination to form the thiol-vinyl ethers, as indicated by electron density in the co-crystal structure of the PRMT5/MEP50 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdNDVSKB6iezW1zwg2yUK5fW6udTcc2ebO1KTwpdyxFbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D&md5=4b5b08dc3e3615144ece564f70bf3bd0</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BW.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DLeal%26aufirst%3DR.%2BA.%26aulast%3DShetty%26aufirst%3DR.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1033%26epage%3D1038%26doi%3D10.1021%2Facsmedchemlett.9b00074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J.</span>; <span class="NLM_string-name">Vaddi, K.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibitors of Protein Arginine Methyltransferase 5 (PRMT5)</span>. <span class="NLM_patent">WO 2017218802 A1</span>, June 15, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+Luengo&author=K.+Vaddi&title=Selective+Inhibitors+of+Protein+Arginine+Methyltransferase+5+%28PRMT5%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLuengo%26aufirst%3DJ.%26atitle%3DSelective%2520Inhibitors%2520of%2520Protein%2520Arginine%2520Methyltransferase%25205%2520%2528PRMT5%2529%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschman, H. R.</span></span> <span> </span><span class="NLM_article-title">PRMT1 is the predominant type I protein arginine methyltransferase in mammalian cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">7723</span>â <span class="NLM_lpage">7730</span>, <span class="refDoi">Â DOI: 10.1074/jbc.275.11.7723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.275.11.7723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10713084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVylsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=7723-7730&author=J.+Tangauthor=A.+Frankelauthor=R.+J.+Cookauthor=S.+Kimauthor=W.+K.+Paikauthor=K.+R.+Williamsauthor=S.+Clarkeauthor=H.+R.+Herschman&title=PRMT1+is+the+predominant+type+I+protein+arginine+methyltransferase+in+mammalian+cells&doi=10.1074%2Fjbc.275.11.7723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT1 is the predominant type I protein arginine methyltransferase in mammalian cells</span></div><div class="casAuthors">Tang, Jie; Frankel, Adam; Cook, Robert J.; Kim, Sangduk; Paik, Woon Ki; Williams, Kenneth R.; Clarke, Steven; Herschman, Harvey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7723-7730</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Type I protein arginine methyltransferases catalyze the formation of asym. Ï-NG,NG-dimethylarginine residues by transferring Me groups from S-adenosyl-L-methionine to guanidino groups of arginine residues in a variety of eucaryotic proteins.  The predominant type I enzyme activity is found in mammalian cells as a high mol. wt. complex (300-400 kDa).  In a previous study, this protein arginine methyltransferase activity was identified as an addnl. activity of 10-formyltetrahydrofolate dehydrogenase (FDH) protein.  However, immunodepletion of FDH activity in RAT1 cells and in murine tissue exts. with antibody to FDH does not diminish type I methyltransferase activity toward the methyl-accepting substrates glutathione S-transferase fibrillarin glycine arginine domain fusion protein or heterogeneous nuclear ribonucleoprotein A1.  Similarly, immunodepletion with anti-FDH antibody does not remove the endogenous methylating activity for hypomethylated proteins present in exts. from adenosine dialdehyde-treated RAT1 cells.  In contrast, anti-PRMT1 antibody can remove PRMT1 activity from RAT1 exts., murine tissue exts., and purified rat liver FDH prepns.  Tissue exts. from FDH(+/+), FDH(+/-), and FDH(-/-) mice have similar protein arginine methyltransferase activities but high, intermediate, and undetectable FDH activities, resp.  Recombinant glutathione S-transferase-PRMT1, but not purified FDH, can be cross-linked to the methyl-donor substrate S-adenosyl-L-methionine.  We conclude that PRMT1 contributes the major type I protein arginine methyltransferase enzyme activity present in mammalian cells and tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR-pUS71H91bVg90H21EOLACvtfcHk0licBO0lEy8tDQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVylsL0%253D&md5=5ddc959b9ce7f7c48715ac1dba77aaa9</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1074%2Fjbc.275.11.7723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.275.11.7723%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DR.%2BJ.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DPaik%26aufirst%3DW.%2BK.%26aulast%3DWilliams%26aufirst%3DK.%2BR.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26atitle%3DPRMT1%2520is%2520the%2520predominant%2520type%2520I%2520protein%2520arginine%2520methyltransferase%2520in%2520mammalian%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D7723%26epage%3D7730%26doi%3D10.1074%2Fjbc.275.11.7723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>â <span class="NLM_lpage">50</span>, <span class="refDoi">Â DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0li2QmpEBnVvEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. Q.</span></span> <span> </span><span class="NLM_article-title">PRMT1 promoted HCC growth and metastasis in vitro and in vivo via activating the STAT3 signalling pathway</span>. <i>Cell. Physiol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1643</span>â <span class="NLM_lpage">1654</span>, <span class="refDoi">Â DOI: 10.1159/000490983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1159%2F000490983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29945155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtl2hu7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=1643-1654&author=X.+P.+Zhangauthor=Y.+B.+Jiangauthor=C.+Q.+Zhongauthor=N.+Maauthor=E.+B.+Zhangauthor=F.+Zhangauthor=J.+J.+Liauthor=Y.+Z.+Dengauthor=K.+Wangauthor=D.+Xieauthor=S.+Q.+Cheng&title=PRMT1+promoted+HCC+growth+and+metastasis+in+vitro+and+in+vivo+via+activating+the+STAT3+signalling+pathway&doi=10.1159%2F000490983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT1 Promoted HCC Growth and Metastasis In Vitro and In Vivo via Activating the STAT3 Signalling Pathway</span></div><div class="casAuthors">Zhang, Xiu-Ping; Jiang, Ya-Bo; Zhong, Cheng-Qian; Ma, Ning; Zhang, Er-Bin; Zhang, Fan; Li, Jing-Jing; Deng, Yue-Zhen; Wang, Kang; Xie, Dong; Cheng, Shu-Qun</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1643-1654</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">Background/Aims: Although it has been widely accepted that protein arginine methyltransferase 1 (PRMT1) is a cancer-promoting gene in various cancers, the mechanism of PRMT1 in hepatocellular carcinoma (HCC) requires more exploration.  This study aimed to investigate the role of PRMT1 in HCC growth and metastasis.  Methods: We compared PRMT1 expression and clinicopathol. characteristics using paired HCC and adjacent noncancerous liver tissues from 210 patients and immunohistochem. analyses.  Cell proliferation, colony formation and migration were detd. in HCC cell lines with PRMT1 overexpression or downregulation through MTT, crystal violet and Boyden chamber assays.  Tumor growth was monitored in a xenograft model, and intrahepatic metastasis models were established.  Results: PRMT1 expression was greatly increased in clin. HCC samples and strongly assocd. with poor prognosis and recurrence; PRMT1 expression was also pos. correlated with microvascular invasion (P = 0.024), tumor differentiation (P = 0.014), tumor size (P = 0.002), and portal vein tumor thrombus (PVTT) (P = 0.028).  Cell proliferation, colony formation and migration in vitro were enhanced by PRMT1 upregulation and decreased by PRMT1 downregulation in HCC cell lines.  Moreover, low PRMT1 expression resulted in slow tumor growth and decreased tumor wt. in vivo, as well as tumor metastasis.  These phenotypes were assocd. with STAT3 signalling pathway activation.  Cryptotanshinone, a STAT3 inhibitor, inhibited STAT3 phosphorylation and reversed the HCC phenotype of PRMT1 expression.  Conclusions: We revealed a significant role for PRMT1 in HCC progression and metastasis in vitro and in vivo via STAT3 signalling pathway activation.  PRMT1 may be a potential novel prognostic biomarker and new therapeutic target for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWErzsvDb2GrVg90H21EOLACvtfcHk0li2QmpEBnVvEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2hu7rP&md5=324c21d3dd1a54d33402ae667b2cc8b6</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1159%2F000490983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000490983%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%2BP.%26aulast%3DJiang%26aufirst%3DY.%2BB.%26aulast%3DZhong%26aufirst%3DC.%2BQ.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DE.%2BB.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DDeng%26aufirst%3DY.%2BZ.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DS.%2BQ.%26atitle%3DPRMT1%2520promoted%2520HCC%2520growth%2520and%2520metastasis%2520in%2520vitro%2520and%2520in%2520vivo%2520via%2520activating%2520the%2520STAT3%2520signalling%2520pathway%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2018%26volume%3D47%26spage%3D1643%26epage%3D1654%26doi%3D10.1159%2F000490983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span> <span> </span><span class="NLM_article-title">The role of PRMT1 in EGFR methylation and signaling in MDA-MB-468 triple-negative breast cancer cells</span>. <i>Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">74</span>â <span class="NLM_lpage">80</span>, <span class="refDoi">Â DOI: 10.1007/s12282-017-0790-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1007%2Fs12282-017-0790-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28643125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BC1cjhtFamsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=74-80&author=K.+Nakaiauthor=W.+Xiaauthor=H.+W.+Liaoauthor=M.+Saitoauthor=M.+C.+Hungauthor=H.+Yamaguchi&title=The+role+of+PRMT1+in+EGFR+methylation+and+signaling+in+MDA-MB-468+triple-negative+breast+cancer+cells&doi=10.1007%2Fs12282-017-0790-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The role of PRMT1 in EGFR methylation and signaling in MDA-MB-468 triple-negative breast cancer cells</span></div><div class="casAuthors">Nakai Katsuya; Xia Weiya; Liao Hsin-Wei; Hung Mien-Chie; Yamaguchi Hirohito; Nakai Katsuya; Saito Mitsue</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer (Tokyo, Japan)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Epidermal growth factor receptor (EGFR) is often overexpressed in triple-negative breast cancer (TNBC).  However, clinical studies have shown that therapies against EGFR are not effective in patients with TNBC.  Recently, it has been reported that arginine 198/200 in EGFR extracellular domain is methylated by PRMT1 and that the methylation confers resistance to EGFR monoclonal antibody cetuximab in colorectal cancer cells.  To explore a potential mechanism underlying intrinsic resistance to anti-EGFR therapy in TNBC, we investigated the role of PRMT1 in EGFR methylation and signaling in MDA-MB-468 (468) TNBC cells.  METHODS:  We knocked down PRMT1 in 468 cells by shRNA, and subjected the cell lysates to Western blot analysis to examine EGFR activation and its downstream molecules.  We also evaluated cell proliferation and sphere formation of PRMT1-knockdown cells.  Finally, we examined the effects of pan-PRMT inhibitor, AMI-1, on cetuximab by colony formation and soft agar assays.  RESULTS:  EGFR methylation and activity was significantly reduced in PRMT1-knockdown cells compared to the parental cells.  Knockdown of PRMT1 also reduced cell proliferation and sphere formation.  Moreover, AMI-1 sensitized 468 cells to cetuximab.  CONCLUSION:  The results indicate that PRMT1 is critical for EGFR activity in 468 cells.  Our data also suggest that inhibition of PRMT1 sensitizes TNBC cells to cetuximab.  Thus, inhibition of PRMT1 may be an effective therapeutic strategy to overcome intrinsic resistance to cetuximab in TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSydnDN7IVXh0NHe_gFldTbfW6udTcc2eaOWDkfWQLvS7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjhtFamsQ%253D%253D&md5=d38db639657d3d981a01619c03b1d2dd</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1007%2Fs12282-017-0790-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12282-017-0790-z%26sid%3Dliteratum%253Aachs%26aulast%3DNakai%26aufirst%3DK.%26aulast%3DXia%26aufirst%3DW.%26aulast%3DLiao%26aufirst%3DH.%2BW.%26aulast%3DSaito%26aufirst%3DM.%26aulast%3DHung%26aufirst%3DM.%2BC.%26aulast%3DYamaguchi%26aufirst%3DH.%26atitle%3DThe%2520role%2520of%2520PRMT1%2520in%2520EGFR%2520methylation%2520and%2520signaling%2520in%2520MDA-MB-468%2520triple-negative%2520breast%2520cancer%2520cells%26jtitle%3DBreast%2520Cancer%26date%3D2018%26volume%3D25%26spage%3D74%26epage%3D80%26doi%3D10.1007%2Fs12282-017-0790-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Theoretical insights into catalytic mechanism of protein arginine methyltransferase 1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e72424</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0072424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1371%2Fjournal.pone.0072424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23977297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKmu7bK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=R.+Zhangauthor=X.+Liauthor=Z.+Liangauthor=K.+Zhuauthor=J.+Luauthor=X.+Kongauthor=S.+Ouyangauthor=L.+Liauthor=Y.+G.+Zhengauthor=C.+Luo&title=Theoretical+insights+into+catalytic+mechanism+of+protein+arginine+methyltransferase+1&doi=10.1371%2Fjournal.pone.0072424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Theoretical insights into catalytic mechanism of protein arginine methyltransferase 1</span></div><div class="casAuthors">Zhang, Ruihan; Li, Xin; Liang, Zhongjie; Zhu, Kongkai; Lu, Junyan; Kong, Xiangqian; Ouyang, Sisheng; Li, Lin; Zheng, Yujun George; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e72424</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 1 (PRMT1), the major arginine asym. dimethylation enzyme in mammals, is emerging as a potential drug target for cancer and cardiovascular disease.  Understanding the catalytic mechanism of PRMT1 will facilitate inhibitor design.  However, detailed mechanisms of the Me transfer process and substrate deprotonation of PRMT1 remain unclear.  Here, the authors present a theor. study on PRMT1-catalyzed arginine dimethylation by employing mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns.  Ternary complex models, composed of PRMT1, peptide substrate, and S-adenosyl-L-methionine (AdoMet) as cofactor, were constructed and verified by 30-ns MD simulation.  The snapshots selected from the MD trajectory were applied for the QM/MM calcn.  The typical SN2-favored transition states of the 1st and 2nd Me transfers were identified from the potential energy profile.  Deprotonation of the substrate, arginine, occurred immediately after Me transfer, and the carboxylate group of Glu-144 acted as proton acceptor.  Furthermore, natural bond orbital anal. and electrostatic potential calcn. showed that Glu-144 facilitated the charge redistribution during the reaction and reduced the energy barrier.  The authors propose the detailed mechanism of PRMT1-catalyzed asym. dimethylation, which increases insight on the small-mol. effector design, and enables further investigations into the physiol. function of this family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpznCqxjTezqrVg90H21EOLACvtfcHk0lhdnfnKdQE8UQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKmu7bK&md5=91656eee3ce77e35b56f01852261df98</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0072424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0072424%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DTheoretical%2520insights%2520into%2520catalytic%2520mechanism%2520of%2520protein%2520arginine%2520methyltransferase%25201%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0072424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela
Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A Potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">772</span>â <span class="NLM_lpage">781</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+Shenauthor=M.+M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+Liauthor=K.+V.+Butlerauthor=H.+U.+Kaniskanauthor=B.+A.+Speedauthor=C.+Dela%0ASenaauthor=A.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+Potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0lhdnfnKdQE8UQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520Potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of protein arginine methyltransferase 6 reduces reactive oxygen species production and attenuates aminoglycoside- and cisplatin-induced hair cell death</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">133</span>â <span class="NLM_lpage">150</span>, <span class="refDoi">Â DOI: 10.7150/thno.37362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.7150%2Fthno.37362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31903111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1OnsbzM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=133-150&author=Y.+Heauthor=W.+Liauthor=Z.+Zhengauthor=L.+Zhaoauthor=W.+Liauthor=Y.+Wangauthor=H.+Li&title=Inhibition+of+protein+arginine+methyltransferase+6+reduces+reactive+oxygen+species+production+and+attenuates+aminoglycoside-+and+cisplatin-induced+hair+cell+death&doi=10.7150%2Fthno.37362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of protein arginine methyltransferase 6 reduces reactive oxygen species production and attenuates aminoglycoside- and cisplatin-induced hair cell death</span></div><div class="casAuthors">He, Yingzi; Li, Wen; Zheng, Zhiwei; Zhao, Liping; Li, Wenyan; Wang, Yunfeng; Li, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-150</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Hair cells in the inner ear have been shown to be susceptible to ototoxicity from some beneficial pharmaceutical drugs, such as aminoglycosides and cisplatin.  Thus, there is great interest in discovering new targets or compds. that protect hair cells from these ototoxic drugs.  Epigenetic regulation is closely related to inner ear development; however, little is known about epigenetic regulation in the process of ototoxic drugs-induced hearing loss.  Methods: In this study, we investigated the role of protein arginine methyltransferase 6 (PRMT6) in aminoglycoside- and cisplatin-induced hair cell loss by using EPZ020411, a selective small mol. PRMT6 inhibitor, in vitro in neonatal mouse cochlear explants and in vivo in C57BL/6 mice.  We also took advantage of the HEI-OC1 cell line to evaluate the anti-apoptosis effects of PRMT6 knockdown on cisplatin-induced ototoxicity.  Apoptotic cells were identified using cleaved caspase-3 staining and TUNEL assay.  The levels of reactive oxygen species (ROS) were evaluated by DCFH-DA and cellROX green staining.  The mitochondrial membrane potential (ÎÎ¨m) were detd. by JC-1, TMRM, and rhodamine 123 staining.  Results: We found that EPZ020411 significantly alleviated neomycin- and cisplatin-induced cell apoptosis and increased hair cell survival.  Moreover, pretreatment with EPZ020411 could attenuate neomycin- and cisplatin-induced hearing loss in vivo.  Mechanistic studies revealed that inhibition of PRMT6 could reverse the increased expression of caspase-3 and cytochrome c translocation, mitochondrial dysfunction, increased accumulation of ROS, and activation of cell apoptosis after cisplatin injury.  Conclusions: Our findings suggested that PRMT6 might serve as a new therapeutic target to prevent hearing loss caused by aminoglycoside- and cisplatin-induced ototoxicity by preventing ROS formation and modulating the mitochondria-related damage and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofANKtj0EPF7Vg90H21EOLACvtfcHk0lhdnfnKdQE8UQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1OnsbzM&md5=777177eb5f3895a123b4b23864e87b9d</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.7150%2Fthno.37362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.37362%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DInhibition%2520of%2520protein%2520arginine%2520methyltransferase%25206%2520reduces%2520reactive%2520oxygen%2520species%2520production%2520and%2520attenuates%2520aminoglycoside-%2520and%2520cisplatin-induced%2520hair%2520cell%2520death%26jtitle%3DTheranostics%26date%3D2020%26volume%3D10%26spage%3D133%26epage%3D150%26doi%3D10.7150%2Fthno.37362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">9665</span>â <span class="NLM_lpage">9673</span>, <span class="refDoi">Â DOI: 10.1039/C4OB01591F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1039%2FC4OB01591F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25348815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=9665-9673&author=Y.+Xieauthor=R.+Zhouauthor=F.+Lianauthor=Y.+Liuauthor=L.+Chenauthor=Z.+Shiauthor=N.+Zhangauthor=M.+Zhengauthor=B.+Shenauthor=H.+Jiangauthor=Z.+Liangauthor=C.+Luo&title=Virtual+screening+and+biological+evaluation+of+novel+small+molecular+inhibitors+against+protein+arginine+methyltransferase+1+%28PRMT1%29&doi=10.1039%2FC4OB01591F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span></div><div class="casAuthors">Xie, Yiqian; Zhou, Ran; Lian, Fulin; Liu, Yan; Chen, Limin; Shi, Zhe; Zhang, Naixia; Zheng, Mingyue; Shen, Bairong; Jiang, Hualiang; Liang, Zhongjie; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">9665-9673</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methylation is a common post-translational modification which is crucial for a variety of biol. processes.  Dysregulation of protein arginine methyltransferases (PRMTs) activity has been implicated in cancer and other serious diseases.  Thus, small mol. inhibitors against PRMT have great potential for therapeutic development.  Herein, through the combination of virtual screening and bioassays, six small mol. compds. were identified as PRMT1 inhibitors.  Amongst them, the binding affinity of compds. DCLX069 and DCLX078 with PRMT1 was further validated by T1Ï and satn. transfer difference (STD) NMR expts.  Most important of all, both compds. effectively blocked cell proliferation in breast cancer, liver cancer and acute myeloid leukemia cell lines.  The binding mode anal. from mol. docking simulations theor. indicated that both inhibitors occupied the SAM binding pocket to exert the inhibitory effect.  Taken together, our compds. enriched the structural scaffolds as PRMT1 inhibitors and afforded clues for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrinhNjy5ctKbVg90H21EOLACvtfcHk0lhTDPILg1q8Lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F&md5=36cb4c315dad033dc9f10a1f21609ab4</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1039%2FC4OB01591F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01591F%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520novel%2520small%2520molecular%2520inhibitors%2520against%2520protein%2520arginine%2520methyltransferase%25201%2520%2528PRMT1%2529%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2014%26volume%3D12%26spage%3D9665%26epage%3D9673%26doi%3D10.1039%2FC4OB01591F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Development of potent type I protein arginine methyltransferase (PRMT) inhibitors of leukemia cell proliferation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8888</span>â <span class="NLM_lpage">8905</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01134</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01134" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8888-8905&author=C.+Wangauthor=H.+Jiangauthor=J.+Jinauthor=Y.+Xieauthor=Z.+Chenauthor=H.+Zhangauthor=F.+Lianauthor=Y.+C.+Liuauthor=C.+Zhangauthor=H.+Dingauthor=S.+Chenauthor=N.+Zhangauthor=Y.+Zhangauthor=H.+Jiangauthor=K.+Chenauthor=F.+Yeauthor=Z.+Yaoauthor=C.+Luo&title=Development+of+potent+type+I+protein+arginine+methyltransferase+%28PRMT%29+inhibitors+of+leukemia+cell+proliferation&doi=10.1021%2Facs.jmedchem.7b01134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Type I Protein Arginine Methyltransferase (PRMT) Inhibitors of Leukemia Cell Proliferation</span></div><div class="casAuthors">Wang, Chen; Jiang, Hao; Jin, Jia; Xie, Yiqian; Chen, Zhifeng; Zhang, Hao; Lian, Fulin; Liu, Yu-Chih; Zhang, Chenhua; Ding, Hong; Chen, Shijie; Zhang, Naixia; Zhang, Yuanyuan; Jiang, Hualiang; Chen, Kaixian; Ye, Fei; Yao, Zhiyi; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8888-8905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein Arginine Methyltransferases (PRMTs) are crucial players in diverse biol. processes, and dysregulation of PRMTs has been linked to various human diseases, esp. cancer.  Therefore, small mols. targeting PRMTs have profound impact for both academic functional studies and clin. disease treatment.  Here, the authors report the discovery of N1-(2-((2-chlorophenyl)thio)benzyl)-N1-methylethane-1,2-diamine (28d, DCPR049_12), a highly potent inhibitor of type I PRMTs that has good selectivity against a panel of other methyltransferases.  Compd. 28d effectively inhibits cell proliferation in several leukemia cell lines and reduces the cellular asym. arginine dimethylation levels.  Serving as an effective inhibitor, 28d demonstrates the mechanism of cell killing in both cell cycle arrest and apoptotic effect as well as downregulation of the pivotal mixed lineage leukemia (MLL) fusion target genes such as HOXA9 and MEIS1, which reflects the crit. roles of type I PRMTs in MLL leukemia.  These studies present 28d as a valuable inhibitor to investigate the role of type I PRMTs in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrddA_Xz0t9VLVg90H21EOLACvtfcHk0lhTDPILg1q8Lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN&md5=40953f4e3f8d78075d444451c67ff6e0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01134%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520potent%2520type%2520I%2520protein%2520arginine%2520methyltransferase%2520%2528PRMT%2529%2520inhibitors%2520of%2520leukemia%2520cell%2520proliferation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8888%26epage%3D8905%26doi%3D10.1021%2Facs.jmedchem.7b01134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedoriw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapurkar, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhart, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laraio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zappacosta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques-OâHagan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span> <span> </span><span class="NLM_article-title">Anti-tumor activity of the type I PRMT inhibitor, GSK3368715, synergizes with PRMT5 inhibition through MTAP loss</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">100</span>â <span class="NLM_lpage">114</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2019.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2019.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31257072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1OjsrjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=100-114&author=A.+Fedoriwauthor=S.+R.+Rajapurkarauthor=S.+O%E2%80%99Brienauthor=S.+V.+Gerhartauthor=L.+H.+Mitchellauthor=N.+D.+Adamsauthor=N.+Riouxauthor=T.+Lingarajauthor=S.+A.+Ribichauthor=M.+B.+Pappalardiauthor=N.+Shahauthor=J.+Laraioauthor=Y.+Liuauthor=M.+Butticelloauthor=C.+L.+Carpenterauthor=C.+Creasyauthor=S.+Korenchukauthor=M.+T.+McCabeauthor=C.+F.+McHughauthor=R.+Nagarajanauthor=C.+Wagnerauthor=F.+Zappacostaauthor=R.+Annanauthor=N.+O.+Conchaauthor=R.+A.+Thomasauthor=T.+K.+Hartauthor=J.+J.+Smithauthor=R.+A.+Copelandauthor=M.+P.+Moyerauthor=J.+Campbellauthor=K.+Sticklandauthor=J.+Millsauthor=S.+Jacques-O%E2%80%99Haganauthor=C.+Allainauthor=D.+Johnstonauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=N.+Watersauthor=K.+Swingerauthor=A.+Boriack-Sjodinauthor=T.+Rieraauthor=G.+Shapiroauthor=R.+Chesworthauthor=R.+K.+Prinjhaauthor=R.+G.+Krugerauthor=O.+Barbashauthor=H.+P.+Mohammad&title=Anti-tumor+activity+of+the+type+I+PRMT+inhibitor%2C+GSK3368715%2C+synergizes+with+PRMT5+inhibition+through+MTAP+loss&doi=10.1016%2Fj.ccell.2019.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-tumor Activity of the Type I PRMT Inhibitor, GSK3368715, Synergizes with PRMT5 Inhibition through MTAP Loss</span></div><div class="casAuthors">Fedoriw, Andrew; Rajapurkar, Satyajit R.; O'Brien, Shane; Gerhart, Sarah V.; Mitchell, Lorna H.; Adams, Nicholas D.; Rioux, Nathalie; Lingaraj, Trupti; Ribich, Scott A.; Pappalardi, Melissa B.; Shah, Niyant; Laraio, Jenny; Liu, Yan; Butticello, Michael; Carpenter, Chris L.; Creasy, Caretha; Korenchuk, Susan; McCabe, Michael T.; McHugh, Charles F.; Nagarajan, Raman; Wagner, Craig; Zappacosta, Francesca; Annan, Roland; Concha, Nestor O.; Thomas, Roberta A.; Hart, Timothy K.; Smith, Jesse J.; Copeland, Robert A.; Moyer, Mikel P.; Campbell, John; Stickland, Kim; Mills, James; Jacques-O'Hagan, Suzanne; Allain, Christina; Johnston, Danielle; Raimondi, Alejandra; Porter Scott, Margaret; Waters, Nigel; Swinger, Kerren; Boriack-Sjodin, Ann; Riera, Tom; Shapiro, Gideon; Chesworth, Richard; Prinjha, Rabinder K.; Kruger, Ryan G.; Barbash, Olena; Mohammad, Helai P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-114.e25</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Type I protein arginine methyltransferases (PRMTs) catalyze asym. dimethylation of arginines on proteins.  Type I PRMTs and their substrates have been implicated in human cancers, suggesting inhibition of type I PRMTs may offer a therapeutic approach for oncol.  The current report describes GSK3368715 (EPZ019997), a potent, reversible type I PRMT inhibitor with anti-tumor effects in human cancer models.  Inhibition of PRMT5, the predominant type II PRMT, produces synergistic cancer cell growth inhibition when combined with GSK3368715.  Interestingly, deletion of the methylthioadenosine phosphorylase gene (MTAP) results in accumulation of the metabolite 2-methylthioadenosine, an endogenous inhibitor of PRMT5, and correlates with sensitivity to GSK3368715 in cell lines.  These data provide rationale to explore MTAP status as a biomarker strategy for patient selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqtv1XXQ84BbVg90H21EOLACvtfcHk0lgCLTRSew7Jzw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1OjsrjM&md5=68faabf5206504d403441bff26531af6</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DFedoriw%26aufirst%3DA.%26aulast%3DRajapurkar%26aufirst%3DS.%2BR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DGerhart%26aufirst%3DS.%2BV.%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DAdams%26aufirst%3DN.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DShah%26aufirst%3DN.%26aulast%3DLaraio%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DCreasy%26aufirst%3DC.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DNagarajan%26aufirst%3DR.%26aulast%3DWagner%26aufirst%3DC.%26aulast%3DZappacosta%26aufirst%3DF.%26aulast%3DAnnan%26aufirst%3DR.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DThomas%26aufirst%3DR.%2BA.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCampbell%26aufirst%3DJ.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DMills%26aufirst%3DJ.%26aulast%3DJacques-O%25E2%2580%2599Hagan%26aufirst%3DS.%26aulast%3DAllain%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%26aulast%3DBoriack-Sjodin%26aufirst%3DA.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DShapiro%26aufirst%3DG.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26atitle%3DAnti-tumor%2520activity%2520of%2520the%2520type%2520I%2520PRMT%2520inhibitor%252C%2520GSK3368715%252C%2520synergizes%2520with%2520PRMT5%2520inhibition%2520through%2520MTAP%2520loss%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D100%26epage%3D114%26doi%3D10.1016%2Fj.ccell.2019.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K. A.</span></span> <span> </span><span class="NLM_article-title">Novel inhibitors for PRMT1 discovered by high-throughput screening using activity-based fluorescence polarization</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1198</span>â <span class="NLM_lpage">1204</span>, <span class="refDoi">Â DOI: 10.1021/cb300024c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300024c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFeltr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1198-1204&author=M.+B.+Dillonauthor=D.+A.+Bachovchinauthor=S.+J.+Brownauthor=M.+G.+Finnauthor=H.+Rosenauthor=B.+F.+Cravattauthor=K.+A.+Mowen&title=Novel+inhibitors+for+PRMT1+discovered+by+high-throughput+screening+using+activity-based+fluorescence+polarization&doi=10.1021%2Fcb300024c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitors for PRMT1 Discovered by High-Throughput Screening Using Activity-Based Fluorescence Polarization</span></div><div class="casAuthors">Dillon, Myles B. C.; Bachovchin, Daniel A.; Brown, Steven J.; Finn, M. G.; Rosen, Hugh; Cravatt, Benjamin F.; Mowen, Kerri A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1198-1204</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) catalyze the posttranslational methylation of arginine using S-adenosylmethionine (SAM) as a methyl-donor.  The PRMT family is widely expressed and has been implicated in biol. functions such as RNA splicing, transcriptional control, signal transduction, and DNA repair.  Therefore, specific inhibitors of individual PRMTs have potentially significant research and therapeutic value.  In particular, PRMT1 is responsible for >85% of arginine methyltransferase activity, but currently available inhibitors of PRMT1 lack specificity, efficacy, and bioavailability.  To address this limitation, we developed a high-throughput screening assay for PRMT1 that utilizes a hyper-reactive cysteine within the active site, which is lacking in almost all other PRMTs.  This assay, which monitors the kinetics of the fluorescence polarization signal increase upon PRMT1 labeling by a rhodamine-contg. cysteine-reactive probe, successfully identified two novel inhibitors selective for PRMT1 over other SAM-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyzeBUHtpySbVg90H21EOLACvtfcHk0lgCLTRSew7Jzw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFeltr4%253D&md5=a59bceb8643a97e72bd0508254695fea</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fcb300024c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300024c%26sid%3Dliteratum%253Aachs%26aulast%3DDillon%26aufirst%3DM.%2BB.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DMowen%26aufirst%3DK.%2BA.%26atitle%3DNovel%2520inhibitors%2520for%2520PRMT1%2520discovered%2520by%2520high-throughput%2520screening%2520using%2520activity-based%2520fluorescence%2520polarization%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1198%26epage%3D1204%26doi%3D10.1021%2Fcb300024c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesaroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuchlautz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ¼scher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span> <span> </span><span class="NLM_article-title">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">933</span>â <span class="NLM_lpage">937</span>, <span class="refDoi">Â DOI: 10.1038/nature06166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature06166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=17898714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=933-937&author=E.+Guccioneauthor=C.+Bassiauthor=F.+Casadioauthor=F.+Martinatoauthor=M.+Cesaroniauthor=H.+Schuchlautzauthor=B.+L%C3%BCscherauthor=B.+Amati&title=Methylation+of+histone+H3R2+by+PRMT6+and+H3K4+by+an+MLL+complex+are+mutually+exclusive&doi=10.1038%2Fnature06166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span></div><div class="casAuthors">Guccione, Ernesto; Bassi, Christian; Casadio, Fabio; Martinato, Francesca; Cesaroni, Matteo; Schuchlautz, Henning; Luescher, Bernhard; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">933-937</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Eukaryotic genomes are organized into active (euchromatic) and inactive (heterochromatic) chromatin domains.  Post-translational modifications of histones (or 'marks') are key in defining these functional states, particularly in promoter regions.  Mutual regulatory interactions between these marks-and the enzymes that catalyze them-contribute to the shaping of this epigenetic landscape, in a manner that remains to be fully elucidated.  We previously obsd. that asym. di-methylation of histone H3 arginine 2 (H3R2me2a) counter-correlates with di- and tri- methylation of H3 lysine 4 (H3K4me2, H3K4me3) on human promoters.  Here we show that the arginine methyltransferase PRMT6 catalyzes H3R2 di-methylation in vitro and controls global levels of H3R2me2a in vivo.  H3R2 methylation by PRMT6 was prevented by the presence of H3K4me3 on the H3 tail.  Conversely, the H3R2me2a mark prevented methylation of H3K4 as well as binding to the H3 tail by an ASH2/WDR5/MLL-family methyltransferase complex.  Chromatin immunopptn. showed that H3R2me2a was distributed within the body and at the 3' end of human genes, regardless of their transcriptional state, whereas it was selectively and locally depleted from active promoters, coincident with the presence of H3K4me3.  Hence, the mutual antagonism between H3R2 and H3K4 methylation, together with the assocn. of MLL-family complexes with the basal transcription machinery, may contribute to the localized patterns of H3K4 tri-methylation characteristic of transcriptionally poised or active promoters in mammalian genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWqE8pDpHmKLVg90H21EOLACvtfcHk0lhLyrxyol2Hag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK&md5=f4cc9371491088af7b74b614c90d6bd4</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnature06166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06166%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DMartinato%26aufirst%3DF.%26aulast%3DCesaroni%26aufirst%3DM.%26aulast%3DSchuchlautz%26aufirst%3DH.%26aulast%3DL%25C3%25BCscher%26aufirst%3DB.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DMethylation%2520of%2520histone%2520H3R2%2520by%2520PRMT6%2520and%2520H3K4%2520by%2520an%2520MLL%2520complex%2520are%2520mutually%2520exclusive%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D933%26epage%3D937%26doi%3D10.1038%2Fnature06166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vhuiyan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of arginine asymmetrical dimethylation by PRMT6</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>473</i></span>,  <span class="NLM_fpage">3049</span>â <span class="NLM_lpage">3063</span>, <span class="refDoi">Â DOI: 10.1042/BCJ20160537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1042%2FBCJ20160537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27480107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2016&pages=3049-3063&author=H.+Wuauthor=W.+Zhengauthor=M.+S.+Eramauthor=M.+Vhuiyanauthor=A.+Dongauthor=H.+Zengauthor=H.+Heauthor=P.+Brownauthor=A.+Frankelauthor=M.+Vedadiauthor=M.+Luoauthor=J.+Min&title=Structural+basis+of+arginine+asymmetrical+dimethylation+by+PRMT6&doi=10.1042%2FBCJ20160537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of arginine asymmetrical dimethylation by PRMT6</span></div><div class="casAuthors">Wu, Hong; Zheng, Weihong; Eram, Mohammad S.; Vhuiyan, Mynol; Dong, Aiping; Zeng, Hong; He, Hao; Brown, Peter; Frankel, Adam; Vedadi, Masoud; Luo, Minkui; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3049-3063</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">PRMT6 is a type I protein arginine methyltransferase, generating the asym. dimethylarginine mark on proteins such as histone H3R2.  Asym. dimethylation of histone H3R2 by PRMT6 acts as a repressive mark that antagonizes trimethylation of H3 lysine 4 by the MLL histone H3K4 methyltransferase.  PRMT6 is overexpressed in several cancer types, including prostate, bladder and lung cancers; therefore, it is of great interest to develop potent and selective inhibitors for PRMT6.  Here, we report the synthesis of a potent bisubstrate inhibitor GMS [6'-methyleneamine sinefungin, an analog of sinefungin (SNF)], and the crystal structures of human PRMT6 in complex, resp., with S-adenosyl- L-homocysteine (SAH) and the bisubstrate inhibitor GMS that shed light on the significantly improved inhibition effect of GMS on methylation activity of PRMT6 compared with SAH and an S-adenosyl-L-methionine competitive methyltransferase inhibitor SNF.  In addn., we also crystd. PRMT6 in complex with SAH and a short arginine-contg. peptide.  Based on the structural information here and available in the PDB database, we proposed a mechanism that can rationalize the distinctive arginine methylation product specificity of different types of arginine methyltransferases and pinpoint the structural determinant of such a specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo061VtGpp_KLVg90H21EOLACvtfcHk0lhLyrxyol2Hag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D&md5=dcacb06a69eb1a4856bea9b0d8458112</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160537%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DVhuiyan%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520arginine%2520asymmetrical%2520dimethylation%2520by%2520PRMT6%26jtitle%3DBiochem.%2520J.%26date%3D2016%26volume%3D473%26spage%3D3049%26epage%3D3063%26doi%3D10.1042%2FBCJ20160537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowhan, D. H.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2201</span>â <span class="NLM_lpage">2216</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkp1203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1093%2Fnar%2Fgkp1203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20047962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=2201-2216&author=M.+J.+Harrisonauthor=Y.+H.+Tangauthor=D.+H.+Dowhan&title=Protein+arginine+methyltransferase+6+regulates+multiple+aspects+of+gene+expression&doi=10.1093%2Fnar%2Fgkp1203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span></div><div class="casAuthors">Harrison, Matthew J.; Tang, Yue Hang; Dowhan, Dennis H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2201-2216</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It is well established that transcription and alternative splicing events are functionally coupled during gene expression.  Here, we report that protein arginine N-methyltransferase 6 (PRMT6) may play a key role in this coupling process by functioning as a transcriptional coactivator that can also regulate alternative splicing.  PRMT6 coactivates the progesterone, glucocorticoid and estrogen receptors in luciferase reporter assays in a hormone-dependent manner.  In addn., small interfering RNA (siRNA) oligonucleotide duplex knockdown of PRMT6 disrupts estrogen-stimulated transcription of endogenous GREB1 and progesterone receptor in MCF-7 breast cancer cells, demonstrating the importance of PRMT6 in hormone-dependent transcription.  In contrast, the regulation of alternative splicing by PRMT6 is hormone independent.  SiRNA knockdown of PRMT6 increases the exon inclusion:skipping ratio of alternatively spliced exons in endogenous vascular endothelial growth factor and spleen tyrosine kinase RNA transcripts in both the presence and absence of estrogen.  These results demonstrate that PRMT6 has a dual role in regulating gene expression and that these two functions can occur independently of each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA8_-JQGR0vbVg90H21EOLACvtfcHk0lhLyrxyol2Hag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D&md5=471281cfc866a6db8be4a7f8c9a009e5</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1203%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DM.%2BJ.%26aulast%3DTang%26aufirst%3DY.%2BH.%26aulast%3DDowhan%26aufirst%3DD.%2BH.%26atitle%3DProtein%2520arginine%2520methyltransferase%25206%2520regulates%2520multiple%2520aspects%2520of%2520gene%2520expression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D2201%26epage%3D2216%26doi%3D10.1093%2Fnar%2Fgkp1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarrold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. C.</span></span> <span> </span><span class="NLM_article-title">PRMTs and arginine methylation: Cancerâs best-kept secret?</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">993</span>â <span class="NLM_lpage">1009</span>, <span class="refDoi">Â DOI: 10.1016/j.molmed.2019.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.molmed.2019.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31230909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyns7jL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=993-1009&author=J.+Jarroldauthor=C.+C.+Davies&title=PRMTs+and+arginine+methylation%3A+Cancer%E2%80%99s+best-kept+secret%3F&doi=10.1016%2Fj.molmed.2019.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">PRMTs and Arginine Methylation: Cancer's Best-Kept Secret?</span></div><div class="casAuthors">Jarrold, James; Davies, Clare C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">993-1009</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Post-translational modification (PTM) of proteins is vital for increasing proteome diversity and maintaining cellular homeostasis.  If the writing, reading, and removal of modifications are not controlled, cancer can develop.  Arginine methylation is an understudied modification that is increasingly assocd. with cancer progression.  Consequently protein arginine methyltransferases (PRMTs), the writers of arginine methylation, have rapidly gained interest as novel drug targets.  However, for clin. success a deep mechanistic understanding of the biol. of PRMTs is required.  In this review we focus on advances made regarding the role of PRMTs in stem cell biol., epigenetics, splicing, immune surveillance and the DNA damage response, and highlight the rapid rise of specific inhibitors that are now in clin. trials for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVzzJJzCLCjbVg90H21EOLACvtfcHk0lhSCnedSjLRJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyns7jL&md5=49e14b56c1d3cb73e1776d0f09204bdb</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2019.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2019.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DJarrold%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DC.%2BC.%26atitle%3DPRMTs%2520and%2520arginine%2520methylation%253A%2520Cancer%25E2%2580%2599s%2520best-kept%2520secret%253F%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D993%26epage%3D1009%26doi%3D10.1016%2Fj.molmed.2019.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ã.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent, selective, and cell-active dual inhibitor of protein arginine methyltransferase 4 and protein arginine methyltransferase 6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9124</span>â <span class="NLM_lpage">9139</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9124-9139&author=Y.+Shenauthor=M.+M.+Szewczykauthor=M.+S.+Eramauthor=D.+Smilauthor=H.+%C3%9C.+Kaniskanauthor=R.+Ferreira+de+Freitasauthor=G.+Senisterraauthor=F.+Liauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+potent%2C+selective%2C+and+cell-active+dual+inhibitor+of+protein+arginine+methyltransferase+4+and+protein+arginine+methyltransferase+6&doi=10.1021%2Facs.jmedchem.6b01033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Shen, Yudao; Szewczyk, Magdalena M.; Eram, Mohammad S.; Smil, David; Kaniskan, H. Umit; Ferreira de Freitas, Renato; Senisterra, Guillermo; Li, Fengling; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9124-9139</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chem. tools for testing biol. and therapeutic hypotheses.  Based on the lead compd., a fragment-like inhibitor of type I PRMTs, the authors conducted structure-activity relationship (SAR) studies and explored three regions of this scaffold.  The studies led to the discovery of a potent, selective and cell-active dual inhibitor of PRMT4 and PRMT6, I (MS049).  As compared to the lead, I displayed much improved potency for PRMT4 and PRMT6 in both biochem. and cellular assays.  It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets.  The authors also developed II (MS049N), which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  Considering possible overlapping substrate specificity of PRMTs, I and II are valuable chem. tools for dissecting specific biol. functions and dysregulation of PRMT4 and PRMT6 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZENTB29LcELVg90H21EOLACvtfcHk0lhSCnedSjLRJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O&md5=2dcdad732107d0a64f7e317a05be1284</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01033%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520dual%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25204%2520and%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9124%26epage%3D9139%26doi%3D10.1021%2Facs.jmedchem.6b01033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchcock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1176</span>â <span class="NLM_lpage">1183</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0lhSCnedSjLRJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradei, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: Identification of the first PRMT6 tool compound</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">655</span>â <span class="NLM_lpage">659</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+Identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lik2Bz9AsnMuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520Identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabelian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class protein arginine methyltransferase 6 (PRMT6) covalent inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5477</span>â <span class="NLM_lpage">5487</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5477-5487&author=Y.+Shenauthor=F.+Liauthor=M.+M.+Szewczykauthor=L.+Halabelianauthor=K.+S.+Parkauthor=I.+Chauauthor=A.+Dongauthor=H.+Zengauthor=H.+Chenauthor=F.+Mengauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+first-in-class+protein+arginine+methyltransferase+6+%28PRMT6%29+covalent+inhibitor&doi=10.1021%2Facs.jmedchem.0c00406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor</span></div><div class="casAuthors">Shen, Yudao; Li, Fengling; Szewczyk, Magdalena M.; Halabelian, Levon; Park, Kwang-su; Chau, Irene; Dong, Aiping; Zeng, Hong; Chen, He; Meng, Fanye; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5477-5487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biol. processes assocd. with multiple cancers.  Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biol. and therapeutic hypotheses.  Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported.  Based on a cocrystal structure of PRMT6-MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, 4 (MS117)(I).  The covalent binding mode of compd. 4 to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure.  Compd. 4 did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases.  We also developed two structurally similar control compds., 5 (MS167) and 7 (MS168).  We provide these valuable chem. tools to the scientific community for further studying PRMT6 physiol. and pathophysiol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTU1W0CZjN17Vg90H21EOLACvtfcHk0lik2Bz9AsnMuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D&md5=cb244ab5a38d04320439c05048998b2f</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DHalabelian%26aufirst%3DL.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520protein%2520arginine%2520methyltransferase%25206%2520%2528PRMT6%2529%2520covalent%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5477%26epage%3D5487%26doi%3D10.1021%2Facs.jmedchem.0c00406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenthal, R. M.</span></span> <span> </span><span class="NLM_article-title">Mammalian DNA methyltransferases: A structural perspective</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">341</span>â <span class="NLM_lpage">350</span>, <span class="refDoi">Â DOI: 10.1016/j.str.2008.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.str.2008.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=18334209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=341-350&author=X.+Chengauthor=R.+M.+Blumenthal&title=Mammalian+DNA+methyltransferases%3A+A+structural+perspective&doi=10.1016%2Fj.str.2008.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian DNA Methyltransferases: A Structural Perspective</span></div><div class="casAuthors">Cheng, Xiaodong; Blumenthal, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">341-350</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The methylation of mammalian DNA, primarily at CpG dinucleotides, has long been recognized to play a major role in controlling gene expression, among other functions.  Given their importance, it is surprising how many basic questions remain to be answered about the proteins responsible for this methylation and for coordination with the parallel chromatin-marking system that operates at the level of histone modification.  This article reviews recent studies on, and discusses the resulting biochem. and structural insights into, the DNA nucleotide methyltransferase (Dnmt) proteins 1, 3a, 3a2, 3b, and 3L.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr80ixWBaUZh7Vg90H21EOLACvtfcHk0ljt2IhU-ld7Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsr0%253D&md5=7c0f3ce1acf8c9baac4a33deff36493c</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DBlumenthal%26aufirst%3DR.%2BM.%26atitle%3DMammalian%2520DNA%2520methyltransferases%253A%2520A%2520structural%2520perspective%26jtitle%3DStructure%26date%3D2008%26volume%3D16%26spage%3D341%26epage%3D350%26doi%3D10.1016%2Fj.str.2008.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jurkowska, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurkowski, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span> <span> </span><span class="NLM_article-title">Structure and function of mammalian DNA methyltransferases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">206</span>â <span class="NLM_lpage">222</span>, <span class="refDoi">Â DOI: 10.1002/cbic.201000195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fcbic.201000195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21243710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFWisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=206-222&author=R.+Z.+Jurkowskaauthor=T.+P.+Jurkowskiauthor=A.+Jeltsch&title=Structure+and+function+of+mammalian+DNA+methyltransferases&doi=10.1002%2Fcbic.201000195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of mammalian DNA methyltransferases</span></div><div class="casAuthors">Jurkowska, Renata Zofia; Jurkowski, Tomasz Piotr; Jeltsch, Albert</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-222</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  DNA methylation plays an important role in epigenetic signaling, having an impact on gene regulation, chromatin structure, development, and disease.  Here, the authors review the structure and function of mammalian DNA methyltransferases Dnmt1, Dnmt3a and Dnmt3b, including their domain structures, catalytic mechanisms, localization, regulation, post-translational modifications, and interaction with chromatin and other proteins, summarizing data obtained in genetic, cell biol., and enzymic studies.  The authors focus on the question of how the mol. and enzymic properties of these enzymes are connected to the dynamics of DNA methylation patterns and to the roles the enzymes play in the processes of de novo and maintenance DNA methylation.  Recent enzymic and genome-wide methylome data have led to a new model of genomic DNA methylation patterns based on the preservation of av. levels of DNA methylation in certain regions, rather than the methylation states of individual CG sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-58zM-psVXrVg90H21EOLACvtfcHk0ljt2IhU-ld7Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFWisw%253D%253D&md5=b1d52bb8bcc5efe6052aaeb0051bc1a6</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000195%26sid%3Dliteratum%253Aachs%26aulast%3DJurkowska%26aufirst%3DR.%2BZ.%26aulast%3DJurkowski%26aufirst%3DT.%2BP.%26aulast%3DJeltsch%26aufirst%3DA.%26atitle%3DStructure%2520and%2520function%2520of%2520mammalian%2520DNA%2520methyltransferases%26jtitle%3DChemBioChem%26date%3D2011%26volume%3D12%26spage%3D206%26epage%3D222%26doi%3D10.1002%2Fcbic.201000195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rondelet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wouters, J.</span></span> <span> </span><span class="NLM_article-title">Human DNA (cytosine-5)-methyltransferases: A functional and structural perspective for epigenetic cancer therapy</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">137</span>â <span class="NLM_lpage">147</span>, <span class="refDoi">Â DOI: 10.1016/j.biochi.2017.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.biochi.2017.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28600135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVantb7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=137-147&author=G.+Rondeletauthor=J.+Wouters&title=Human+DNA+%28cytosine-5%29-methyltransferases%3A+A+functional+and+structural+perspective+for+epigenetic+cancer+therapy&doi=10.1016%2Fj.biochi.2017.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Human DNA (cytosine-5)-methyltransferases: a functional and structural perspective for epigenetic cancer therapy</span></div><div class="casAuthors">Rondelet, Gregoire; Wouters, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137-147</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Epigenetic modifications modulate chromatin states to regulate gene expression.  Among them, DNA methylation and histone modifications play a crucial role in the establishment of the epigenome.  In cancer, these epigenetic events may act in concert to repress tumor suppressor genes or promote oncogenic pathways.  In the context of cancer initiation and progression, recruitment of DNA (cytosine-5)-methyltransferases to specific genomic regions is mainly mediated by histone epigenetic marks, transcription factors and co-regulators as part of a dynamic process.  Herein, we will review these mechanisms and present state-of-the-art of DNA methylation, treatment and development of epigenetic cancer therapies targeting this epigenetic modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4OMz-iVRrHrVg90H21EOLACvtfcHk0ljt2IhU-ld7Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVantb7E&md5=8cf0fb684fa94f1e058cfcb2986773bc</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2017.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2017.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DRondelet%26aufirst%3DG.%26aulast%3DWouters%26aufirst%3DJ.%26atitle%3DHuman%2520DNA%2520%2528cytosine-5%2529-methyltransferases%253A%2520A%2520functional%2520and%2520structural%2520perspective%2520for%2520epigenetic%2520cancer%2520therapy%26jtitle%3DBiochimie%26date%3D2017%26volume%3D139%26spage%3D137%26epage%3D147%26doi%3D10.1016%2Fj.biochi.2017.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">DNA methyltransferases: Emerging targets for the discovery of inhibitors as potent anticancer drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2323</span>â <span class="NLM_lpage">2331</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2019.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.drudis.2019.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31494187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslKntbvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2323-2331&author=J.+Yuauthor=T.+Xieauthor=Z.+Wangauthor=X.+Wangauthor=S.+Zengauthor=Y.+Kangauthor=T.+Hou&title=DNA+methyltransferases%3A+Emerging+targets+for+the+discovery+of+inhibitors+as+potent+anticancer+drugs&doi=10.1016%2Fj.drudis.2019.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferases: emerging targets for the discovery of inhibitors as potent anticancer drugs</span></div><div class="casAuthors">Yu, Jie; Xie, Tianli; Wang, Zhe; Wang, Xuwen; Zeng, Su; Kang, Yu; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2323-2331</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA methyltransferases (DNMTs) are a conserved family of cytosine methylases with crucial roles in epigenetic regulation.  They have been considered as promising therapeutic targets for the epigenetic treatment of cancer.  Therefore, DNMT inhibitors (DNMTis) have attracted considerable interest in recent years for the modulation of the aberrant DNA methylation pattern in a reversible way.  In this review, it provide a structure-based overview of the therapeutic importance of DNMTs against different cancer types, and then summarize recently investigated DNMTis as well as their inhibitory mechanisms, focusing on recent advances in the development of DNMTis with specificity and/or selectivity using computational approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIGX9s11xzIbVg90H21EOLACvtfcHk0ljt2IhU-ld7Ig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslKntbvF&md5=b191f9753cd011fc2129d918b1dde3a5</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DDNA%2520methyltransferases%253A%2520Emerging%2520targets%2520for%2520the%2520discovery%2520of%2520inhibitors%2520as%2520potent%2520anticancer%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D2323%26epage%3D2331%26doi%3D10.1016%2Fj.drudis.2019.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span> <span> </span><span class="NLM_article-title">DNA methyltransferase inhibitors and their therapeutic potential</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2448</span>â <span class="NLM_lpage">2457</span>, <span class="refDoi">Â DOI: 10.2174/1568026619666181120150122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2174%2F1568026619666181120150122" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=2448-2457&author=Z.+Zhouauthor=H.+Q.+Liauthor=F.+Liu&title=DNA+methyltransferase+inhibitors+and+their+therapeutic+potential&doi=10.2174%2F1568026619666181120150122"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.2174%2F1568026619666181120150122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026619666181120150122%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%2BQ.%26aulast%3DLiu%26aufirst%3DF.%26atitle%3DDNA%2520methyltransferase%2520inhibitors%2520and%2520their%2520therapeutic%2520potential%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2019%26volume%3D18%26spage%3D2448%26epage%3D2457%26doi%3D10.2174%2F1568026619666181120150122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pechalrieu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Etievant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">DNA methyltransferase inhibitors in cancer: From pharmacology to translational studies</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">13</span>, <span class="refDoi">Â DOI: 10.1016/j.bcp.2016.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.bcp.2016.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27956110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOjs7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=1-13&author=D.+Pechalrieuauthor=C.+Etievantauthor=P.+B.+Arimondo&title=DNA+methyltransferase+inhibitors+in+cancer%3A+From+pharmacology+to+translational+studies&doi=10.1016%2Fj.bcp.2016.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferase inhibitors in cancer: From pharmacology to translational studies</span></div><div class="casAuthors">Pechalrieu, Dany; Etievant, Chantal; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA methylation is a mammalian epigenetic mark that participates to define where and when genes are expressed, both in normal cells and in the context of diseases.  Like other epigenetic marks, it is reversible and can be modulated by chem. agents.  Because it plays an important role in cancer by silencing certain genes, such as tumor suppressor genes, it is a promising therapeutic target.  Two compds. are already approved to treat hematol. cancers, and many efforts have been carried out to discover new mols. that inhibit DNA methyltransferases, the enzymes responsible for DNA methylation.  Here, we analyze the mol. mechanisms and cellular pharmacol. of these inhibitors, pointing out the necessity for new pharmacol. models and paradigms.  The parameters of pharmacol. responses need to be redefined: the aim is cellular reprogramming rather than general cytotoxicity.  Thus, "epigenetic" rather than cytotoxic dosages are defined.  Another issue is the delay of the response: cellular reprogramming can take several generations to produce observable phenotypes.  Is this compatible with lab. scale expts.  Finally, it is important to consider the specificity for cancer cells compared to normal cells and the appearance of resistance.  We also discuss different techniques that are used and the selection of pharmacol. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj2Zvsz9F1krVg90H21EOLACvtfcHk0ljsNNEOa3kEdg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOjs7%252FP&md5=3a835b4c437050d8a09c35d8357c1c56</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DPechalrieu%26aufirst%3DD.%26aulast%3DEtievant%26aufirst%3DC.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DDNA%2520methyltransferase%2520inhibitors%2520in%2520cancer%253A%2520From%2520pharmacology%2520to%2520translational%2520studies%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D129%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.bcp.2016.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA methylation with small molecules: Whatâs next?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2569</span>â <span class="NLM_lpage">2583</span>, <span class="refDoi">Â DOI: 10.1021/jm500843d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500843d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2569-2583&author=A.+Erdmannauthor=L.+Halbyauthor=J.+Fahyauthor=P.+B.+Arimondo&title=Targeting+DNA+methylation+with+small+molecules%3A+What%E2%80%99s+next%3F&doi=10.1021%2Fjm500843d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA Methylation with Small Molecules: What's Next?</span></div><div class="casAuthors">Erdmann, Alexandre; Halby, Ludovic; Fahy, Jacques; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2569-2583</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA methylation is a mammalian epigenetic mark that is involved in defining where and when genes are expressed, both in normal cells and in the context of diseases.  Like other epigenetic marks, it is reversible and can be modulated by chem. agents.  Because it plays an important role in cancer by silencing certain genes, such as tumor suppressor genes, and by reactivating other regions, such as repeated elements, it is a promising therapeutic target.  Two compds. are already approved to treat hematol. cancers.  Many efforts have been carried out to discover new mols. that are able to efficiently inhibit DNA methylation in cancer cells.  We will briefly overview the foremost of these efforts by focusing on what we have learned to this point on non-nucleoside inhibitors and on what we consider to be the features of an ideal inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9LcXQUubTNbVg90H21EOLACvtfcHk0ljsNNEOa3kEdg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L&md5=cc469a073e82dc2cbe2d822cd8c02cce</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1021%2Fjm500843d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500843d%26sid%3Dliteratum%253Aachs%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DFahy%26aufirst%3DJ.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DTargeting%2520DNA%2520methylation%2520with%2520small%2520molecules%253A%2520What%25E2%2580%2599s%2520next%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2569%26epage%3D2583%26doi%3D10.1021%2Fjm500843d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Derissen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijnen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H.</span></span> <span> </span><span class="NLM_article-title">Concise drug review: Azacitidine and decitabine</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">619</span>â <span class="NLM_lpage">624</span>, <span class="refDoi">Â DOI: 10.1634/theoncologist.2012-0465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1634%2Ftheoncologist.2012-0465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23671007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXps1Cktbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=619-624&author=E.+J.+Derissenauthor=J.+H.+Beijnenauthor=J.+H.+Schellens&title=Concise+drug+review%3A+Azacitidine+and+decitabine&doi=10.1634%2Ftheoncologist.2012-0465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Concise drug review: azacitidine and decitabine</span></div><div class="casAuthors">Derissen, Ellen J. B.; Beijnen, Jos H.; Schellens, Jan H. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">619-624</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">This concise drug review summarizes the current state of treatment with the hypomethylating agents azacitidine and decitabine.  The hypomethylating agents azacitidine and decitabine (5-aza-2'-deoxycytidine) for the treatment of several specific forms of myelodysplastic syndromes (MDS), chronic myelomonocytic leukemia (CMML), and acute myeloid leukemia (AML) were discussed.  The introduction of the hypomethylating agents has been a major advancement in the treatment of patients with higher-risk MDS who are ineligible for allogeneic stem cell transplantation (allo-SCT).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo52eea6V6hobVg90H21EOLACvtfcHk0lhqk4rcrQzNQA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXps1Cktbo%253D&md5=131d4a854d1fc6d3986da9b5ae21e144</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2012-0465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2012-0465%26sid%3Dliteratum%253Aachs%26aulast%3DDerissen%26aufirst%3DE.%2BJ.%26aulast%3DBeijnen%26aufirst%3DJ.%2BH.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%26atitle%3DConcise%2520drug%2520review%253A%2520Azacitidine%2520and%2520decitabine%26jtitle%3DOncologist%26date%3D2013%26volume%3D18%26spage%3D619%26epage%3D624%26doi%3D10.1634%2Ftheoncologist.2012-0465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stresemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyko, F.</span></span> <span> </span><span class="NLM_article-title">Modes of action of the DNA methyltransferase inhibitors azacytidine and decitabine</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">8</span>â <span class="NLM_lpage">13</span>, <span class="refDoi">Â DOI: 10.1002/ijc.23607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fijc.23607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=18425818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtlantrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2008&pages=8-13&author=C.+Stresemannauthor=F.+Lyko&title=Modes+of+action+of+the+DNA+methyltransferase+inhibitors+azacytidine+and+decitabine&doi=10.1002%2Fijc.23607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Modes of action of the DNA methyltransferase inhibitors azacytidine and decitabine</span></div><div class="casAuthors">Stresemann, Carlo; Lyko, Frank</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-13</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  The cytosine analogs 5-azacytosine (azacytidine) and 2'-deoxy-5-azacytidine (decitabine) are the currently most advanced drugs for epigenetic cancer therapies.  These compds. function as DNA methyltransferase inhibitors and have shown substantial potency in reactivating epigenetically silenced tumor suppressor genes in vitro.  However, it has been difficult to define the mode of action of these drugs in patients and it appears that clin. responses are influenced both by epigenetic alterations and by apoptosis induction.  To maximize the clin. efficacy of azacytidine and decitabine it will be important to understand the mol. changes induced by these drugs.  In this review, we examine the pharmacol. properties of azanucleosides and their interactions with various cellular pathways.  Because azacytidine and decitabine are prodrugs, an understanding of the cellular mechanisms mediating transmembrane transport and metabolic activation will be critically important for optimizing patient responses.  We also discuss the mechanism of DNA methyltransferase inhibition and emphasize the need for the identification of predictive biomarkers for the further advancement of epigenetic therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfTNmdOvDjg7Vg90H21EOLACvtfcHk0lhqk4rcrQzNQA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtlantrg%253D&md5=25154a9967a675a3dd93bc3dd7f08191</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1002%2Fijc.23607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.23607%26sid%3Dliteratum%253Aachs%26aulast%3DStresemann%26aufirst%3DC.%26aulast%3DLyko%26aufirst%3DF.%26atitle%3DModes%2520of%2520action%2520of%2520the%2520DNA%2520methyltransferase%2520inhibitors%2520azacytidine%2520and%2520decitabine%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2008%26volume%3D123%26spage%3D8%26epage%3D13%26doi%3D10.1002%2Fijc.23607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsen, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquez, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selker, E. U.</span></span> <span> </span><span class="NLM_article-title">Inhibition of DNA methylation and reactivation of silenced genes by zebularine</span>. <i>J. Natl. Cancer Ins.t.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">399</span>â <span class="NLM_lpage">409</span>, <span class="refDoi">Â DOI: 10.1093/jnci/95.5.399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1093%2Fjnci%2F95.5.399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=12618505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVeqs74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2003&pages=399-409&author=J.+C.+Chengauthor=C.+B.+Matsenauthor=F.+A.+Gonzalesauthor=W.+Yeauthor=S.+Greerauthor=V.+E.+Marquezauthor=P.+A.+Jonesauthor=E.+U.+Selker&title=Inhibition+of+DNA+methylation+and+reactivation+of+silenced+genes+by+zebularine&doi=10.1093%2Fjnci%2F95.5.399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DNA methylation and reactivation of silenced genes by zebularine</span></div><div class="casAuthors">Cheng, Jonathan C.; Matsen, Cindy B.; Gonzales, Felicidad A.; Ye, Wei; Greer, Sheldon; Marquez, Victor E.; Jones, Peter A.; Selker, Eric U.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">399-409</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Gene silencing by abnormal methylation of promoter regions of regulatory genes is commonly assocd. with cancer.  Silenced tumor suppressor genes are obvious targets for reactivation by methylation inhibitors such as 5-azacytidine (5-Aza-CR) and 5-aza-2'-deoxycytidine (5-Aza-CdR).  However, both compds. are chem. unstable and toxic and neither can be given orally.  We characterized a new demethylating agent, zebularine [1-(beta-D-ribofuranosyl)-1,2-dihydropyrimidin-2-one], which is a chem. stable cytidine analog.  We tested the ability of zebularine to reactivate a silenced Neurospora crassa gene using a hygromycin gene reactivation assay.  We then analyzed the ability of zebularine to inhibit DNA methylation in C3H 10T1/2 Cl8 (10T1/2) mouse embryo cells as assayed by induction of a myogenic phenotype and in T24 human bladder carcinoma cells, using the methylation-sensitive single nucleotide primer extension (Ms-SNuPE) assay.  We also evaluated the effects of zebularine (administered orally or i.p.) on growth of EJ6 human bladder carcinoma cells grown in BALB/c nu/nu mice (five mice per group) and the in vivo reactivation of a methylated p16 gene in these cells.  All statistical tests were two-sided.  In N. crassa, zebularine inhibited DNA methylation and reactivated a gene previously silenced by methylation.  Zebularine induced the myogenic phenotype in 10T1/2 cells, which is a phenomenon unique to DNA methylation inhibitors.  Zebularine reactivated a silenced p16 gene and demethylated its promoter region in T24 bladder carcinoma cells in vitro and in tumors grown in mice.  Zebularine was only slightly cytotoxic to T24 cells in vitro (1 mM zebularine for 48 h decreased plating efficiency by 17% [95% confidence interval (CI) = 12.8% to 21.2%]) and to tumor-bearing mice (av. maximal wt. change in mice treated with 1000 mg/kg zebularine = 11% [95% CI = 4% to 19%]).  Compared with those in control mice, tumor vols. were statistically significantly reduced in mice treated with high-dose zebularine administered by i.p. injection (P<.001) or by oral gavage (P<.001).  Zebularine is a stable DNA demethylating agent and the first drug in its class able to reactivate an epigenetically silenced gene by oral administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuC5AqIpr-IbVg90H21EOLACvtfcHk0lhqk4rcrQzNQA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVeqs74%253D&md5=3c4016cb5147926157077807f569060d</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F95.5.399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F95.5.399%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DJ.%2BC.%26aulast%3DMatsen%26aufirst%3DC.%2BB.%26aulast%3DGonzales%26aufirst%3DF.%2BA.%26aulast%3DYe%26aufirst%3DW.%26aulast%3DGreer%26aufirst%3DS.%26aulast%3DMarquez%26aufirst%3DV.%2BE.%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DSelker%26aufirst%3DE.%2BU.%26atitle%3DInhibition%2520of%2520DNA%2520methylation%2520and%2520reactivation%2520of%2520silenced%2520genes%2520by%2520zebularine%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Ins.t.%26date%3D2003%26volume%3D95%26spage%3D399%26epage%3D409%26doi%3D10.1093%2Fjnci%2F95.5.399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickman, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurd, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornby, D. P.</span></span> <span> </span><span class="NLM_article-title">Zebularine: A novel DNA methylation inhibitor that forms a covalent complex with DNA methyltransferases</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">591</span>â <span class="NLM_lpage">599</span>, <span class="refDoi">Â DOI: 10.1016/S0022-2836(02)00676-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2FS0022-2836%2802%2900676-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=12206775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wmtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2002&pages=591-599&author=L.+Zhouauthor=X.+Chengauthor=B.+A.+Connollyauthor=M.+J.+Dickmanauthor=P.+J.+Hurdauthor=D.+P.+Hornby&title=Zebularine%3A+A+novel+DNA+methylation+inhibitor+that+forms+a+covalent+complex+with+DNA+methyltransferases&doi=10.1016%2FS0022-2836%2802%2900676-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Zebularine: A Novel DNA Methylation Inhibitor that Forms a Covalent Complex with DNA Methyltransferases</span></div><div class="casAuthors">Zhou, L.; Cheng, X.; Connolly, B. A.; Dickman, M. J.; Hurd, P. J.; Hornby, D. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">591-599</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Mechanism-based inhibitors of enzymes, which mimic reactive intermediates in the reaction pathway, have been deployed extensively in the anal. of metabolic pathways and as candidate drugs.  The inhibition of cytosine-[C5]-specific DNA methyltransferases (C5 MTases) by oligodeoxynucleotides contg. 5-azadeoxycytidine (AzadC) and 5-fluorodeoxycytidine (FdC) provides a well-documented example of mechanism-based inhibition of enzymes central to nucleic acid metab.  Here, we describe the interaction between the C5 MTase from Haemophilus haemolyticus (M.HhaI) and an oligodeoxynucleotide duplex contg. 2-H pyrimidinone, an analog often referred to as zebularine and known to give rise to high-affinity complexes with MTases.  X-ray crystallog. has demonstrated the formation of a covalent bond between M.HhaI and the 2-H pyrimidinone-contg. oligodeoxynucleotide.  This observation enables a comparison between the mechanisms of action of 2-H pyrimidinone with other mechanism-based inhibitors such as FdC.  This novel complex provides a mol. explanation for the mechanism of action of the anti-cancer drug zebularine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTq4bolT8QM7Vg90H21EOLACvtfcHk0lgVvFvH6bPRgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wmtbs%253D&md5=aeced0fbc4a800b82d117a0d1265042e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900676-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900676-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DConnolly%26aufirst%3DB.%2BA.%26aulast%3DDickman%26aufirst%3DM.%2BJ.%26aulast%3DHurd%26aufirst%3DP.%2BJ.%26aulast%3DHornby%26aufirst%3DD.%2BP.%26atitle%3DZebularine%253A%2520A%2520novel%2520DNA%2520methylation%2520inhibitor%2520that%2520forms%2520a%2520covalent%2520complex%2520with%2520DNA%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D321%26spage%3D591%26epage%3D599%26doi%3D10.1016%2FS0022-2836%2802%2900676-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bearss, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span> <span> </span><span class="NLM_article-title">S110, a 5-Aza-2â²-deoxycytidine-containing dinucleotide, is an effective DNA methylation inhibitor in vivo and can reduce tumor growth</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1443</span>â <span class="NLM_lpage">1450</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-09-1048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1535-7163.MCT-09-1048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20442312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslektbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1443-1450&author=J.+C.+Chuangauthor=S.+L.+Warnerauthor=D.+Vollmerauthor=H.+Vankayalapatiauthor=S.+Redkarauthor=D.+J.+Bearssauthor=X.+Qiuauthor=C.+B.+Yooauthor=P.+A.+Jones&title=S110%2C+a+5-Aza-2%E2%80%B2-deoxycytidine-containing+dinucleotide%2C+is+an+effective+DNA+methylation+inhibitor+in+vivo+and+can+reduce+tumor+growth&doi=10.1158%2F1535-7163.MCT-09-1048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">S110, a 5-Aza-2'-Deoxycytidine-Containing Dinucleotide, Is an Effective DNA Methylation Inhibitor In vivo and Can Reduce Tumor Growth</span></div><div class="casAuthors">Chuang, Jody C.; Warner, Steven L.; Vollmer, David; Vankayalapati, Hariprasad; Redkar, Sanjeev; Bearss, David J.; Qiu, Xiangning; Yoo, Christine B.; Jones, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1443-1450</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Methylation of CpG islands in promoter regions is often assocd. with gene silencing and aberrant DNA methylation occurs in most cancers, leading to the silencing of some tumor suppressor genes.  Reversal of this abnormal hypermethylation by DNA methylation inhibitors is effective in reactivating methylation-silenced tumor suppressor genes both in vitro and in vivo.  Several DNA methylation inhibitors have been well studied; the most potent among them is 5-aza-2'-deoxycytidine (5-Aza-CdR), which can induce myelosuppression in patients.  S110 is a dinucleotide consisting of 5-Aza-CdR followed by a deoxyguanosine, which we previously showed to be effective in vitro as a DNA methylation inhibitor while being less prone to deamination by cytidine deaminase, making it a promising alternative to 5-Aza-CdR.  Here, we show that S110 is better tolerated than 5-Aza-CdR in mice and is as effective in vivo in inducing p16 expression, reducing DNA methylation at the p16 promoter region, and retarding tumor growth in human xenograft.  We also show that S110 is effective by both i.p. and s.c. deliveries.  S110 therefore is a promising new agent that acts similarly to 5-Aza-CdR and has better stability and less toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMMN8iuoWq47Vg90H21EOLACvtfcHk0lgVvFvH6bPRgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslektbg%253D&md5=538700770fa7ae214fccf238726e8e7d</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-1048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-1048%26sid%3Dliteratum%253Aachs%26aulast%3DChuang%26aufirst%3DJ.%2BC.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DVollmer%26aufirst%3DD.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DYoo%26aufirst%3DC.%2BB.%26aulast%3DJones%26aufirst%3DP.%2BA.%26atitle%3DS110%252C%2520a%25205-Aza-2%25E2%2580%25B2-deoxycytidine-containing%2520dinucleotide%252C%2520is%2520an%2520effective%2520DNA%2520methylation%2520inhibitor%2520in%2520vivo%2520and%2520can%2520reduce%2520tumor%2520growth%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D1443%26epage%3D1450%26doi%3D10.1158%2F1535-7163.MCT-09-1048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daher-Reyes, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchan, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K. W. L.</span></span> <span> </span><span class="NLM_article-title">Guadecitabine (SGI-110): An investigational drug for the treatment of myelodysplastic syndrome and acute myeloid leukemia</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">835</span>â <span class="NLM_lpage">849</span>, <span class="refDoi">Â DOI: 10.1080/13543784.2019.1667331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F13543784.2019.1667331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31510809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVSgurfJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=835-849&author=G.+S.+Daher-Reyesauthor=B.+M.+Merchanauthor=K.+W.+L.+Yee&title=Guadecitabine+%28SGI-110%29%3A+An+investigational+drug+for+the+treatment+of+myelodysplastic+syndrome+and+acute+myeloid+leukemia&doi=10.1080%2F13543784.2019.1667331"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Guadecitabine (SGI-110): an investigational drug for the treatment of myelodysplastic syndrome and acute myeloid leukemia</span></div><div class="casAuthors">Daher-Reyes, Georgina S.; Merchan, Brayan M.; Yee, Karen W. L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">835-849</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : The incidence of acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) is increasing with the aging population.  Prognosis and overall survival (OS) remain poor in elderly patients and in those not eligible for intensive treatment.  Hypomethylating agents (HMAs) have played an important role in this group of patients but their efficacy is limited.: This article reviews the mechanism of action, pharmacol., safety profile and clin. efficacy of s.c. guadecitabine, a second-generation DNA methylation inhibitor in development for the treatment of AML and MDS.: Although guadecitabine did not yield improved complete remission (CR) rates and OS compared to the control arm in patients with treatment-naive AML who were ineligible for intensive chemotherapy, subgroup anal. in patients who received â¥4 cycles of therapy demonstrated superior outcomes in favor of guadecitabine.  Given its stability, ease of administration, safety profile and prolonged exposure time, guadecitabine would be the more appropriate HMA, replacing azacitidine and decitabine, to be used combination treatment regimens in patients with myeloid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS4NKimdcQ1bVg90H21EOLACvtfcHk0lh51fdNOdk45A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVSgurfJ&md5=09dcfa3ca3050d20aff1e69a6aa0dbc9</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1080%2F13543784.2019.1667331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543784.2019.1667331%26sid%3Dliteratum%253Aachs%26aulast%3DDaher-Reyes%26aufirst%3DG.%2BS.%26aulast%3DMerchan%26aufirst%3DB.%2BM.%26aulast%3DYee%26aufirst%3DK.%2BW.%2BL.%26atitle%3DGuadecitabine%2520%2528SGI-110%2529%253A%2520An%2520investigational%2520drug%2520for%2520the%2520treatment%2520of%2520myelodysplastic%2520syndrome%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2019%26volume%3D28%26spage%3D835%26epage%3D849%26doi%3D10.1080%2F13543784.2019.1667331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duchmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzykson, R.</span></span> <span> </span><span class="NLM_article-title">Clinical update on hypomethylating agents</span>. <i>Int. J. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">161</span>â <span class="NLM_lpage">169</span>, <span class="refDoi">Â DOI: 10.1007/s12185-019-02651-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1007%2Fs12185-019-02651-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31020568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FovFOmuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2019&pages=161-169&author=M.+Duchmannauthor=R.+Itzykson&title=Clinical+update+on+hypomethylating+agents&doi=10.1007%2Fs12185-019-02651-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical update on hypomethylating agents</span></div><div class="casAuthors">Duchmann Matthieu; Itzykson Raphael; Duchmann Matthieu; Itzykson Raphael</div><div class="citationInfo"><span class="NLM_cas:title">International journal of hematology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-169</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hypomethylating agents (HMAs), azacitidine and decitabine, are standards of care in higher-risk myelodysplastic syndromes and in acute myeloid leukemia patients ineligible for intensive therapy.  Over the last 10 years, research efforts have sought to better understand their mechanism of action, both at the molecular and cellular level.  These efforts have yet to robustly identify biomarkers for these agents.  The clinical activity of HMAs in myeloid neoplasms has been firmly established now but still remains of limited magnitude.  Besides optimized use at different stages of the disease, most of the expected clinical progress with HMAs will come from the development of second-generation compounds orally available and/or with improved pharmacokinetics, and from the search, so far mostly empirical, of HMA-based synergistic drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKdrpYE4wMEnthwrmtH4OBfW6udTcc2ebmGCc4_1pQ_rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FovFOmuw%253D%253D&md5=e1c1e7abf621defaf7af3caabc6871c4</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1007%2Fs12185-019-02651-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12185-019-02651-9%26sid%3Dliteratum%253Aachs%26aulast%3DDuchmann%26aufirst%3DM.%26aulast%3DItzykson%26aufirst%3DR.%26atitle%3DClinical%2520update%2520on%2520hypomethylating%2520agents%26jtitle%3DInt.%2520J.%2520Hematol.%26date%3D2019%26volume%3D110%26spage%3D161%26epage%3D169%26doi%3D10.1007%2Fs12185-019-02651-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghoshal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamage, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooke, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phiasivongsa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S. T.</span></span> <span> </span><span class="NLM_article-title">A new class of quinoline-based DNA hypomethylating agents reactivates tumor suppressor genes by blocking DNA methyltransferase 1 activity and inducing its degradation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">4277</span>â <span class="NLM_lpage">4285</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-08-3669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F0008-5472.CAN-08-3669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=19417133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFWltLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=4277-4285&author=J.+Dattaauthor=K.+Ghoshalauthor=W.+A.+Dennyauthor=S.+A.+Gamageauthor=D.+G.+Brookeauthor=P.+Phiasivongsaauthor=S.+Redkarauthor=S.+T.+Jacob&title=A+new+class+of+quinoline-based+DNA+hypomethylating+agents+reactivates+tumor+suppressor+genes+by+blocking+DNA+methyltransferase+1+activity+and+inducing+its+degradation&doi=10.1158%2F0008-5472.CAN-08-3669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">A New Class of Quinoline-Based DNA Hypomethylating Agents Reactivates Tumor Suppressor Genes by Blocking DNA Methyltransferase 1 Activity and Inducing Its Degradation</span></div><div class="casAuthors">Datta, Jharna; Ghoshal, Kalpana; Denny, William A.; Gamage, Swarna A.; Brooke, Darby G.; Phiasivongsa, Pasit; Redkar, Sanjeev; Jacob, Samson T.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4277-4285</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Reactivation of silenced tumor suppressor genes by 5-azacytidine (Vidaza) and its congener 5-aza-2'-deoxycytidine (decitabine) has provided an alternate approach to cancer therapy.  We have shown previously that these drugs selectively and rapidly induce degrdn. of the maintenance DNA methyltransferase (DNMT) 1 by a proteasomal pathway.  Because the toxicity of these compds. is largely due to their incorporation into DNA, it is crit. to explore novel, nonnucleoside compds. that can effectively reactivate the silenced genes.  Here, we report that a quinoline-based compd., designated SGI-1027, inhibits the activity of DNMT1, DNMT3A, and DNMT3B as well M. SssI with comparable IC50 (6-13 Î¼mol/L) by competing with S-adenosylmethionine in the methylation reaction.  Treatment of different cancer cell lines with SGI-1027 resulted in selective degrdn. of DNMT1 with minimal or no effects on DNMT3A and DNMT3B.  At a concn. of 2.5 to 5 Î¼mol/L (similar to that of decitabine), complete degrdn. of DNMT1 protein was achieved within 24 h without significantly affecting its mRNA level.  MG132 blocked SGI-1027-induced depletion of DNMT1, indicating the involvement of proteasomal pathway.  Prolonged treatment of RKO cells with SGI-1027 led to demethylation and reexpression of the silenced tumor suppressor genes P16, MLH1, and TIMP3.  Further, this compd. did not exhibit significant toxicity in a rat hepatoma (H4IIE) cell line.  This study provides a novel class of DNA hypomethylating agents that have the potential for use in epigenetic cancer therapy. [Cancer Res 2009;69(10):4277-85].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlNfytT2_KPbVg90H21EOLACvtfcHk0lh51fdNOdk45A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFWltLs%253D&md5=41a6321d36d19b46bc48171f230357a8</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3669%26sid%3Dliteratum%253Aachs%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DGhoshal%26aufirst%3DK.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DGamage%26aufirst%3DS.%2BA.%26aulast%3DBrooke%26aufirst%3DD.%2BG.%26aulast%3DPhiasivongsa%26aufirst%3DP.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%2BT.%26atitle%3DA%2520new%2520class%2520of%2520quinoline-based%2520DNA%2520hypomethylating%2520agents%2520reactivates%2520tumor%2520suppressor%2520genes%2520by%2520blocking%2520DNA%2520methyltransferase%25201%2520activity%2520and%2520inducing%2520its%2520degradation%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D4277%26epage%3D4285%26doi%3D10.1158%2F0008-5472.CAN-08-3669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asgatay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champion, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marloie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drujon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senamaud-Beaufort, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajavelu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schambel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lequin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoyan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guianvarcâh, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of analogues of <i>N</i>-phthaloyl-l-tryptophan (RG108) as inhibitors of DNA methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">421</span>â <span class="NLM_lpage">434</span>, <span class="refDoi">Â DOI: 10.1021/jm401419p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401419p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegsLjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=421-434&author=S.+Asgatayauthor=C.+Championauthor=G.+Marloieauthor=T.+Drujonauthor=C.+Senamaud-Beaufortauthor=A.+Ceccaldiauthor=A.+Erdmannauthor=A.+Rajaveluauthor=P.+Schambelauthor=A.+Jeltschauthor=O.+Lequinauthor=P.+Karoyanauthor=P.+B.+Arimondoauthor=D.+Guianvarc%E2%80%99h&title=Synthesis+and+evaluation+of+analogues+of+N-phthaloyl-l-tryptophan+%28RG108%29+as+inhibitors+of+DNA+methyltransferase+1&doi=10.1021%2Fjm401419p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Analogues of N-Phthaloyl-L-tryptophan (RG108) as Inhibitors of DNA Methyltransferase 1</span></div><div class="casAuthors">Asgatay, Saadia; Champion, Christine; Marloie, Gael; Drujon, Thierry; Senamaud-Beaufort, Catherine; Ceccaldi, Alexandre; Erdmann, Alexandre; Rajavelu, Arumugam; Schambel, Philippe; Jeltsch, Albert; Lequin, Olivier; Karoyan, Philippe; Arimondo, Paola B.; Guianvarc'h, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">421-434</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMT) are promising drug targets in cancer provided that new, more specific, and chem. stable inhibitors are discovered.  Among the non-nucleoside DNMT inhibitors, N-phthaloyl-L-tryptophan (RG108) was first identified as inhibitor of DNMT1.  Here, RG108 analogs were synthesized to understand its interaction with DNMT.  The indole, carboxylate, and phthalimide moieties were modified.  Homologated and conformationally constrained analogs were prepd.  The latter were synthesized from prolino-homotryptophan derivs. through a methodol. based amino-zinc-ene-enolate cyclization.  All compds. were tested for their ability to inhibit DNMT1 in vitro.  Among them, constrained compds. e.g., (2S,3R)-3-((1H-indol-3-yl)methyl)-1-(2-naphthoyl)pyrrolidine-2-carboxylic acid and NPys derivs. e.g., N2-[(3-nitro-2-pyridinyl)thio]-L-tryptophan were at least 10-fold more potent than the ref. compd.  The cytotoxicity on the tumor DU145 cell line of the most potent inhibitors was correlated to their inhibitory potency.  Finally, docking studies were conducted to understand their binding mode.  This study provides insights for the design of the next-generation of DNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxxOlrdGxBwrVg90H21EOLACvtfcHk0lhc7-e5k5QCew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegsLjM&md5=d35b180c01308c55c9f6df2f24334ec2</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fjm401419p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401419p%26sid%3Dliteratum%253Aachs%26aulast%3DAsgatay%26aufirst%3DS.%26aulast%3DChampion%26aufirst%3DC.%26aulast%3DMarloie%26aufirst%3DG.%26aulast%3DDrujon%26aufirst%3DT.%26aulast%3DSenamaud-Beaufort%26aufirst%3DC.%26aulast%3DCeccaldi%26aufirst%3DA.%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DRajavelu%26aufirst%3DA.%26aulast%3DSchambel%26aufirst%3DP.%26aulast%3DJeltsch%26aufirst%3DA.%26aulast%3DLequin%26aufirst%3DO.%26aulast%3DKaroyan%26aufirst%3DP.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DGuianvarc%25E2%2580%2599h%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520analogues%2520of%2520N-phthaloyl-l-tryptophan%2520%2528RG108%2529%2520as%2520inhibitors%2520of%2520DNA%2520methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D421%26epage%3D434%26doi%3D10.1021%2Fjm401419p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyko, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Constrained analogues of procaine as novel small molecule inhibitors of DNA methyltransferase-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2321</span>â <span class="NLM_lpage">2325</span>, <span class="refDoi">Â DOI: 10.1021/jm7015705</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7015705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlOjsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2321-2325&author=S.+Castellanoauthor=D.+Kuckauthor=M.+Salaauthor=E.+Novellinoauthor=F.+Lykoauthor=G.+Sbardella&title=Constrained+analogues+of+procaine+as+novel+small+molecule+inhibitors+of+DNA+methyltransferase-1&doi=10.1021%2Fjm7015705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Constrained Analogues of Procaine as Novel Small Molecule Inhibitors of DNA Methyltransferase-1</span></div><div class="casAuthors">Castellano, Sabrina; Kuck, Dirk; Sala, Marina; Novellino, Ettore; Lyko, Frank; Sbardella, Gianluca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2321-2325</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Constrained analogs of procaine were synthesized, and their inhibiting activity against DNMT1 was tested.  Among them, the most potent compd., deriv. 3b, was also able to induce a recognizable demethylation of chromosomal satellite repeats in HL60 human myeloid leukemia cells and thus represents a lead compd. for the development of a novel class of nonnucleoside DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsBswQtfKqrVg90H21EOLACvtfcHk0lhc7-e5k5QCew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlOjsL8%253D&md5=d922a4841968fa2c0c1efddf5102b5f4</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Fjm7015705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7015705%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DKuck%26aufirst%3DD.%26aulast%3DSala%26aufirst%3DM.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DLyko%26aufirst%3DF.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DConstrained%2520analogues%2520of%2520procaine%2520as%2520novel%2520small%2520molecule%2520inhibitors%2520of%2520DNA%2520methyltransferase-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2321%26epage%3D2325%26doi%3D10.1021%2Fjm7015705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identifying novel selective non-nucleoside DNA methyltransferase 1 inhibitors through docking-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9028</span>â <span class="NLM_lpage">9041</span>, <span class="refDoi">Â DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+Chenauthor=Y.+Wangauthor=W.+Zhouauthor=S.+Liauthor=J.+Pengauthor=Z.+Shiauthor=J.+Huauthor=Y.+C.+Liuauthor=H.+Dingauthor=Y.+Linauthor=L.+Liauthor=S.+Chengauthor=J.+Liuauthor=T.+Luauthor=H.+Jiangauthor=B.+Liuauthor=M.+Zhengauthor=C.+Luo&title=Identifying+novel+selective+non-nucleoside+DNA+methyltransferase+1+inhibitors+through+docking-based+virtual+screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0lhc7-e5k5QCew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520novel%2520selective%2520non-nucleoside%2520DNA%2520methyltransferase%25201%2520inhibitors%2520through%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ³pez-Vallejo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamborini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biochemical evaluation of Î´(2)-isoxazoline derivatives as DNA methyltransferase 1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7663</span>â <span class="NLM_lpage">7677</span>, <span class="refDoi">Â DOI: 10.1021/jm2010404</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2010404" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Orsb3K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7663-7677&author=S.+Castellanoauthor=D.+Kuckauthor=M.+Vivianoauthor=J.+Yooauthor=F.+L%C3%B3pez-Vallejoauthor=P.+Contiauthor=L.+Tamboriniauthor=A.+Pintoauthor=J.+L.+Medina-Francoauthor=G.+Sbardella&title=Synthesis+and+biochemical+evaluation+of+%CE%B4%282%29-isoxazoline+derivatives+as+DNA+methyltransferase+1+inhibitors&doi=10.1021%2Fjm2010404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biochemical Evaluation of Î2-Isoxazoline Derivatives as DNA Methyltransferase 1 Inhibitors</span></div><div class="casAuthors">Castellano, Sabrina; Kuck, Dirk; Viviano, Monica; Yoo, Jakyung; Lopez-Vallejo, Fabian; Conti, Paola; Tamborini, Lucia; Pinto, Andrea; Medina-Franco, Jose L.; Sbardella, Gianluca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7663-7677</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of Î2-isoxazoline constrained analogs of procaine/procainamide were prepd. and their inhibitory activity against DNA methyltransferase 1 (DNMT1) was tested.  Among them, compd. I is far more potent in vitro (IC50 = 150 Î¼M) than other non-nucleoside inhibitors and also exhibits a strong and dose-dependent antiproliferative effect against HCT116 human colon carcinoma cells.  The binding mode of I with the enzyme was also investigated by means of a simple competition assay as well as of docking simulations conducted using the recently published crystallog. structure of human DNMT1.  On the basis of the findings, we assessed that the mode of inhibition of I is consistent with a competition with the cofactor and propose it as a novel lead compd. for the development of non-nucleoside DNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFLFATuXaYW7Vg90H21EOLACvtfcHk0lh9iWpv3l-sAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Orsb3K&md5=6ea821ff418e46e0fe055776dfad904c</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fjm2010404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2010404%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DKuck%26aufirst%3DD.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DL%25C3%25B3pez-Vallejo%26aufirst%3DF.%26aulast%3DConti%26aufirst%3DP.%26aulast%3DTamborini%26aufirst%3DL.%26aulast%3DPinto%26aufirst%3DA.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DSynthesis%2520and%2520biochemical%2520evaluation%2520of%2520%25CE%25B4%25282%2529-isoxazoline%2520derivatives%2520as%2520DNA%2520methyltransferase%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7663%26epage%3D7677%26doi%3D10.1021%2Fjm2010404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnekenburger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwergel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosconati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirsch, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miele, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferretti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">Selective non-nucleoside inhibitors of human DNA methyltransferases active in cancer including in cancer stem cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">701</span>â <span class="NLM_lpage">713</span>, <span class="refDoi">Â DOI: 10.1021/jm4012627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4012627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=701-713&author=S.+Valenteauthor=Y.+Liuauthor=M.+Schnekenburgerauthor=C.+Zwergelauthor=S.+Cosconatiauthor=C.+Grosauthor=M.+Tardugnoauthor=D.+Labellaauthor=C.+Floreanauthor=S.+Mindenauthor=H.+Hashimotoauthor=Y.+Changauthor=X.+Zhangauthor=G.+Kirschauthor=E.+Novellinoauthor=P.+B.+Arimondoauthor=E.+Mieleauthor=E.+Ferrettiauthor=A.+Gulinoauthor=M.+Diederichauthor=X.+Chengauthor=A.+Mai&title=Selective+non-nucleoside+inhibitors+of+human+DNA+methyltransferases+active+in+cancer+including+in+cancer+stem+cells&doi=10.1021%2Fjm4012627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Non-nucleoside Inhibitors of Human DNA Methyltransferases Active in Cancer Including in Cancer Stem Cells</span></div><div class="casAuthors">Valente, Sergio; Yiwei, Liu; Schnekenburger, Michael; Zwergel, Clemens; Cosconati, Sandro; Gros, Christina; Tardugno, Maria; Labella, Donatella; Florean, Cristina; Minden, Steven; Hashimoto, Hideharu; Chang, Yanqi; Zhang, Xing; Kirsch, Gilbert; Novellino, Ettore; Arimondo, Paola B.; Miele, Evelina; Ferretti, Elisabetta; Gulino, Alberto; Diederich, Marc; Cheng, Xiaodong; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">701-713</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMTs) are important enzymes involved in epigenetic control of gene expression and represent valuable targets in cancer chemotherapy.  A no. of nucleoside DNMT inhibitors (DNMTi) have been studied in cancer, including in cancer stem cells, and two of them (azacytidine and decitabine) have been approved for treatment of myelodysplastic syndromes.  However, only a few non-nucleoside DNMTi have been identified so far, and even fewer have been validated in cancer.  Through a process of hit-to-lead optimization, we report here the discovery of compd. 5 as a potent non-nucleoside DNMTi that is also selective toward other AdoMet-dependent protein methyltransferases.  Compd. 5 was potent at single-digit micromolar concns. against a panel of cancer cells and was less toxic in peripheral blood mononuclear cells than two other compds. tested.  In mouse medulloblastoma stem cells, 5 inhibited cell growth, whereas related compd. 2 showed high cell differentiation.  To the best of our knowledge, 2 and 5 are the first non-nucleoside DNMTi tested in a cancer stem cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxOcRR02xFkLVg90H21EOLACvtfcHk0lh9iWpv3l-sAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D&md5=0749e33ad2324aa591342d7c7f23a955</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Fjm4012627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4012627%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSchnekenburger%26aufirst%3DM.%26aulast%3DZwergel%26aufirst%3DC.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DFlorean%26aufirst%3DC.%26aulast%3DMinden%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DKirsch%26aufirst%3DG.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DMiele%26aufirst%3DE.%26aulast%3DFerretti%26aufirst%3DE.%26aulast%3DGulino%26aufirst%3DA.%26aulast%3DDiederich%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DSelective%2520non-nucleoside%2520inhibitors%2520of%2520human%2520DNA%2520methyltransferases%2520active%2520in%2520cancer%2520including%2520in%2520cancer%2520stem%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D701%26epage%3D713%26doi%3D10.1021%2Fjm4012627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rilova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pechalrieu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhlel, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novosad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aussagues, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausseil, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleury, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guianvarcâh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferroud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Rational design of bisubstrate-type analogues as inhibitors of DNA methyltransferases in cancer cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4665</span>â <span class="NLM_lpage">4679</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntVGgtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4665-4679&author=L.+Halbyauthor=Y.+Menonauthor=E.+Rilovaauthor=D.+Pechalrieuauthor=V.+Massonauthor=C.+Fauxauthor=M.+A.+Bouhlelauthor=M.+H.+David-Cordonnierauthor=N.+Novosadauthor=Y.+Aussaguesauthor=A.+Samsonauthor=L.+Lacroixauthor=F.+Ausseilauthor=L.+Fleuryauthor=D.+Guianvarc%E2%80%99hauthor=C.+Ferroudauthor=P.+B.+Arimondo&title=Rational+design+of+bisubstrate-type+analogues+as+inhibitors+of+DNA+methyltransferases+in+cancer+cells&doi=10.1021%2Facs.jmedchem.7b00176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Bisubstrate-Type Analogues as Inhibitors of DNA Methyltransferases in Cancer Cells</span></div><div class="casAuthors">Halby, Ludovic; Menon, Yoann; Rilova, Elodie; Pechalrieu, Dany; Masson, Veronique; Faux, Celine; Bouhlel, Mohamed Amine; David-Cordonnier, Marie-Helene; Novosad, Natacha; Aussagues, Yannick; Samson, Arnaud; Lacroix, Laurent; Ausseil, Frederic; Fleury, Laurence; Guianvarc'h, Dominique; Ferroud, Clotilde; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4665-4679</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant DNA hypermethylation of promoter of tumor suppressor genes is commonly obsd. in cancer and its inhibition by small mols. is promising for their reactivation.  Here the authors designed bisubstrate analogs-based inhibitors, by mimicking each substrate, - the S-adenosyl-L-methionine and the deoxycytidine -, and linking them together.  This approach resulted in quinazoline-quinoline derivs. as potent inhibitors of DNMT3A and DNMT1, some showing certain isoform selectivity.  The authors highlighted the importance of (i) the nature and rigidity of the linker between the two moieties for inhibition, as (ii) the presence of the nitrogen on the quinoline group and (iii) of a hydrophobic group on the quinazoline.  The most potent inhibitors induced demethylation of CDKN2A promoter in colon carcinoma HCT116 cells and its reactivation after 7 days of treatment.  Furthermore, in a leukemia cells model system, the authors found a correlation between demethylation of the promoter induced by the treatment, chromatin opening at the promoter and the reactivation of a reporter gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpymGPl1Goj1bVg90H21EOLACvtfcHk0lgqZP-BcMrMJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntVGgtLY%253D&md5=91cf15839ca568c0f64987ff079df5e5</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00176%26sid%3Dliteratum%253Aachs%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DMenon%26aufirst%3DY.%26aulast%3DRilova%26aufirst%3DE.%26aulast%3DPechalrieu%26aufirst%3DD.%26aulast%3DMasson%26aufirst%3DV.%26aulast%3DFaux%26aufirst%3DC.%26aulast%3DBouhlel%26aufirst%3DM.%2BA.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.%2BH.%26aulast%3DNovosad%26aufirst%3DN.%26aulast%3DAussagues%26aufirst%3DY.%26aulast%3DSamson%26aufirst%3DA.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DAusseil%26aufirst%3DF.%26aulast%3DFleury%26aufirst%3DL.%26aulast%3DGuianvarc%25E2%2580%2599h%26aufirst%3DD.%26aulast%3DFerroud%26aufirst%3DC.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DRational%2520design%2520of%2520bisubstrate-type%2520analogues%2520as%2520inhibitors%2520of%2520DNA%2520methyltransferases%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4665%26epage%3D4679%26doi%3D10.1021%2Facs.jmedchem.7b00176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span> <span> </span><span class="NLM_article-title">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1243</span>â <span class="NLM_lpage">1260</span>, <span class="refDoi">Â DOI: 10.15252/embj.201694058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.15252%2Fembj.201694058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28320739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1243-1260&author=X.+Wangauthor=W.+Caoauthor=J.+Zhangauthor=M.+Yanauthor=Q.+Xuauthor=X.+Wuauthor=L.+Wanauthor=Z.+Zhangauthor=C.+Zhangauthor=X.+Qinauthor=M.+Xiaoauthor=D.+Yeauthor=Y.+Liuauthor=Z.+Hanauthor=S.+Wangauthor=L.+Maoauthor=W.+Weiauthor=W.+Chen&title=A+covalently+bound+inhibitor+triggers+EZH2+degradation+through+CHIP-mediated+ubiquitination&doi=10.15252%2Fembj.201694058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span></div><div class="casAuthors">Wang, Xu; Cao, Wei; Zhang, Jianjun; Yan, Ming; Xu, Qin; Wu, Xiangbing; Wan, Lixin; Zhang, Zhiyuan; Zhang, Chenping; Qin, Xing; Xiao, Meng; Ye, Dongxia; Liu, Yuyang; Han, Zeguang; Wang, Shaomeng; Mao, Li; Wei, Wenyi; Chen, Wantao</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1243-1260</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Enhancer of zeste homolog 2 (EZH2) has been characterized as a crit. oncogene and a promising drug target in human malignant tumors.  The current EZH2 inhibitors strongly suppress the enhanced enzymic function of mutant EZH2 in some lymphomas.  However, the recent identification of a PRC2- and methyltransferase-independent role of EZH2 indicates that a complete suppression of all oncogenic functions of EZH2 is needed.  Here, we report a unique EZH2-targeting strategy by identifying a gambogenic acid (GNA) deriv. as a novel agent that specifically and covalently bound to Cys668 within the EZH2-SET domain, triggering EZH2 degrdn. through COOH terminus of Hsp70-interacting protein (CHIP)-mediated ubiquitination.  This class of inhibitors significantly suppressed H3K27Me3 and effectively reactivated polycomb repressor complex 2 (PRC2)-silenced tumor suppressor genes.  Moreover, the novel inhibitors significantly suppressed tumor growth in an EZH2-dependent manner, and tumors bearing a non-GNA-interacting C668S-EZH2 mutation exhibited resistance to the inhibitors.  Together, our results identify the inhibition of the signaling pathway that governs GNA-mediated destruction of EZH2 as a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRQknSdWy5T7Vg90H21EOLACvtfcHk0lgqZP-BcMrMJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D&md5=b9b8a266537e2922648784d9cd4edf02</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.15252%2Fembj.201694058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembj.201694058%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26atitle%3DA%2520covalently%2520bound%2520inhibitor%2520triggers%2520EZH2%2520degradation%2520through%2520CHIP-mediated%2520ubiquitination%26jtitle%3DEMBO%2520J.%26date%3D2017%26volume%3D36%26spage%3D1243%26epage%3D1260%26doi%3D10.15252%2Fembj.201694058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiltz, G. E.</span></span> <span> </span><span class="NLM_article-title">Small molecule approaches for targeting the polycomb repressive complex 2 (PRC2) in cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">15344</span>â <span class="NLM_lpage">15370</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c01344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFSqu7rF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=15344-15370&author=M.+C.+Martinauthor=G.+Zengauthor=J.+Yuauthor=G.+E.+Schiltz&title=Small+molecule+approaches+for+targeting+the+polycomb+repressive+complex+2+%28PRC2%29+in+cancer&doi=10.1021%2Facs.jmedchem.0c01344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Approaches for Targeting the Polycomb Repressive Complex 2 (PRC2) in Cancer</span></div><div class="casAuthors">Martin, M. Cynthia; Zeng, Guihua; Yu, Jindan; Schiltz, Gary E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15344-15370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The polycomb repressive complex 2 (PRC2) is composed of three core subunits, enhancer of zeste 2 (EZH2), embryonic ectoderm development (EED), and suppressor of zeste 12 (SUZ12), along with a no. of accessory proteins.  It is the key enzymic protein complex that catalyzes histone H3 lysine 27 (H3K27) methylation to mediate epigenetic silencing of target genes.  PRC2 thus plays essential roles in maintaining embryonic stem cell identity and in controlling cellular differentiation.  Studies in the past decade have reported frequent overexpression or mutation of PRC2 in various cancers including prostate cancer and lymphoma.  Aberrant PRC2 function has been extensively studied and proven to contribute to a large no. of abnormal cellular processes, including those that lead to uncontrolled proliferation and tumorigenesis.  Significant efforts have recently been made to develop small mols. targeting PRC2 function for potential use as anticancer therapeutics.  In this review, we describe recent approaches to identify and develop small mols. that target PRC2.  These various strategies include the inhibition of the function of individual PRC2 core proteins, the disruption of PRC2 complex formation, and the degrdn. of its subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokmpM-lxiL4bVg90H21EOLACvtfcHk0lgqZP-BcMrMJw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFSqu7rF&md5=788620c0bb1dde734c26057707d8d825</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01344%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BC.%26aulast%3DZeng%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DSchiltz%26aufirst%3DG.%2BE.%26atitle%3DSmall%2520molecule%2520approaches%2520for%2520targeting%2520the%2520polycomb%2520repressive%2520complex%25202%2520%2528PRC2%2529%2520in%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D15344%26epage%3D15370%26doi%3D10.1021%2Facs.jmedchem.0c01344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banik, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harsha, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoloi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalduhsaki
Sailo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arfuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunnumakkara, A. B.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of gambogic acid, a caged xanthone, to target cancer</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>416</i></span>,  <span class="NLM_fpage">75</span>â <span class="NLM_lpage">86</span>, <span class="refDoi">Â DOI: 10.1016/j.canlet.2017.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.canlet.2017.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29246645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVegs7jL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=416&publication_year=2018&pages=75-86&author=K.+Banikauthor=C.+Harshaauthor=D.+Bordoloiauthor=B.+Lalduhsaki%0ASailoauthor=G.+Sethiauthor=H.+C.+Leongauthor=F.+Arfusoauthor=S.+Mishraauthor=L.+Wangauthor=A.+P.+Kumarauthor=A.+B.+Kunnumakkara&title=Therapeutic+potential+of+gambogic+acid%2C+a+caged+xanthone%2C+to+target+cancer&doi=10.1016%2Fj.canlet.2017.12.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of gambogic acid, a caged xanthone, to target cancer</span></div><div class="casAuthors">Banik, Kishore; Harsha, Choudhary; Bordoloi, Devivasha; Lalduhsaki Sailo, Bethsebie; Sethi, Gautam; Leong, Hin Chong; Arfuso, Frank; Mishra, Srishti; Wang, Lingzhi; Kumar, Alan P.; Kunnumakkara, Ajaikumar B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">416</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-86</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Natural compds. have enormous biol. and clin. activity against dreadful diseases such as cancer, as well as cardiovascular and neurodegenerative disorders.  In spite of the widespread research carried out in the field of cancer therapeutics, cancer is one of the most prevalent diseases with no perfect treatment till date.  Adverse side effects and the development of chemoresistance are the imperative limiting factors assocd. with conventional chemotherapeutics.  For this reason, there is an urgent need to find compds. that are highly safe and efficacious for the prevention and treatment of cancer.  Gambogic acid (GA) is a xanthone structure extd. from the dry, brownish gamboge resin secreted from the Garcinia hanburyi tree in Southeast Asia and has inherent anti-cancer properties.  In this review, the mol. mechanisms underlying the targets of GA that are liable for its effective anti-cancer activity are discussed that reveal the potential of GA as a pertinent candidate that can be appropriately developed and designed into a capable anti-cancer drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpQe0ZqukkY7Vg90H21EOLACvtfcHk0li7yEyUOLVTMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVegs7jL&md5=45a6d09c16c77e19d487d33bc006d734</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.12.014%26sid%3Dliteratum%253Aachs%26aulast%3DBanik%26aufirst%3DK.%26aulast%3DHarsha%26aufirst%3DC.%26aulast%3DBordoloi%26aufirst%3DD.%26aulast%3DLalduhsaki%2BSailo%26aufirst%3DB.%26aulast%3DSethi%26aufirst%3DG.%26aulast%3DLeong%26aufirst%3DH.%2BC.%26aulast%3DArfuso%26aufirst%3DF.%26aulast%3DMishra%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DA.%2BP.%26aulast%3DKunnumakkara%26aufirst%3DA.%2BB.%26atitle%3DTherapeutic%2520potential%2520of%2520gambogic%2520acid%252C%2520a%2520caged%2520xanthone%252C%2520to%2520target%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D416%26spage%3D75%26epage%3D86%26doi%3D10.1016%2Fj.canlet.2017.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cipriano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic reader domains by chemical biology</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">82</span>â <span class="NLM_lpage">94</span>, <span class="refDoi">Â DOI: 10.1016/j.cbpa.2020.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.cbpa.2020.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32739717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2020&pages=82-94&author=A.+Ciprianoauthor=G.+Sbardellaauthor=A.+Ciulli&title=Targeting+epigenetic+reader+domains+by+chemical+biology&doi=10.1016%2Fj.cbpa.2020.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic reader domains by chemical biology</span></div><div class="casAuthors">Cipriano, Alessandra; Sbardella, Gianluca; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-94</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Over the past years, growing interest toward post-translational modifications (PTMs) of histones and nonhistone proteins has prompted academia and industrial research groups to develop different approaches to better understand the link between PTMs and pathol. states.  Selective recognition of PTMs is carried out by reader modules, which mediate the biol. readout of epigenetic mechanisms.  Progress in medicinal chem. and chem. biol. has contributed to corroborate the role of reader domains in chromatin-binding proteins as potential therapeutic targets.  Here, we review the state-of-the-art of the most important small mols. developed to date, with a particular attention on contemporary chem. biol. approaches, including photoaffinity probes, cyclic peptides, bifunctional inhibitors, and PROTAC degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOYlHbr36l8rVg90H21EOLACvtfcHk0li7yEyUOLVTMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbbP&md5=5a804beab676445c967726108a33b9aa</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DCipriano%26aufirst%3DA.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520epigenetic%2520reader%2520domains%2520by%2520chemical%2520biology%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94%26doi%3D10.1016%2Fj.cbpa.2020.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spriano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">107631</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2020.107631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.pharmthera.2020.107631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32693114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2020&pages=107631&author=F.+Sprianoauthor=A.+Stathisauthor=F.+Bertoni&title=Targeting+BET+bromodomain+proteins+in+cancer%3A+The+example+of+lymphomas&doi=10.1016%2Fj.pharmthera.2020.107631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span></div><div class="casAuthors">Spriano, Filippo; Stathis, Anastasios; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107631</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Bromo- and Extra-Terminal domain (BET) family proteins act as ''readers'' of acetylated histones and they are important transcription regulators.  BRD2, BRD3, BRD4 and BRDT, part of the BET family, are important in different tumors, where upregulation or translocation often occurs.  The potential of targeting BET proteins as anti-cancer treatment originated with data obtained with a first series of compds., and there are now several data supporting BET inhibition in both solid tumors and hematol. malignancies.  Despite very pos. preclin. data in different tumor types, the clin. results have been so far moderate.  Using lymphoma as an example to review the data produced in the lab. and in the context of the early clin. trials, we discuss the modalities to make BET targeting more efficient both generating novel generation of compds. and by exploring the combination with small mols. affecting various signaling pathways, BCL2, or DNA damage response signaling, but also with addnl. epigenetic agents and with immunotherapy.  We also discuss the mechanisms of resistance and the toxicity profiles so far reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Jn61Q24_wrVg90H21EOLACvtfcHk0li7yEyUOLVTMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI&md5=cdb034e619849cb681f48c5b03f7cde8</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107631%26sid%3Dliteratum%253Aachs%26aulast%3DSpriano%26aufirst%3DF.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DTargeting%2520BET%2520bromodomain%2520proteins%2520in%2520cancer%253A%2520The%2520example%2520of%2520lymphomas%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D215%26spage%3D107631%26doi%3D10.1016%2Fj.pharmthera.2020.107631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: Epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>â <span class="NLM_lpage">356</span>, <span class="refDoi">Â DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+Epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhWKWLHC5F4Vg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520Epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>â <span class="NLM_lpage">4558</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lhWKWLHC5F4Vg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CÃ´tÃ©, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: Progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>â <span class="NLM_lpage">1298</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+Progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lhWKWLHC5F4Vg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520Progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooike, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugahara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daibata, M.</span></span> <span> </span><span class="NLM_article-title">Abstract C244: Development of the BET bromodomain inhibitor OTX015</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">C244</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.TARG-13-C244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1535-7163.TARG-13-C244" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=C244&author=J.+K.+Noelauthor=K.+Iwataauthor=S.+Ooikeauthor=K.+Sugaharaauthor=H.+Nakamuraauthor=M.+Daibata&title=Abstract+C244%3A+Development+of+the+BET+bromodomain+inhibitor+OTX015&doi=10.1158%2F1535-7163.TARG-13-C244"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-13-C244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-13-C244%26sid%3Dliteratum%253Aachs%26aulast%3DNoel%26aufirst%3DJ.%2BK.%26aulast%3DIwata%26aufirst%3DK.%26aulast%3DOoike%26aufirst%3DS.%26aulast%3DSugahara%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DH.%26aulast%3DDaibata%26aufirst%3DM.%26atitle%3DAbstract%2520C244%253A%2520Development%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520OTX015%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26volume%3D12%26spage%3DC244%26doi%3D10.1158%2F1535-7163.TARG-13-C244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorusso, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagey, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, S. B.</span></span> <span> </span><span class="NLM_article-title">Abstract A49: Clinically efficacy of the BET bromodomain inhibitor TEN-010 in an open-label substudy with patients with documented NUT-midline carcinoma (NMC)</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">A49</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.TARG-15-A49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1535-7163.TARG-15-A49" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=A49&author=G.+I.+Shapiroauthor=A.+Dowlatiauthor=P.+M.+Lorussoauthor=J.+P.+Ederauthor=A.+Andersonauthor=K.+T.+Doauthor=M.+H.+Kageyauthor=C.+Sirardauthor=J.+E.+Bradnerauthor=S.+B.+Landau&title=Abstract+A49%3A+Clinically+efficacy+of+the+BET+bromodomain+inhibitor+TEN-010+in+an+open-label+substudy+with+patients+with+documented+NUT-midline+carcinoma+%28NMC%29&doi=10.1158%2F1535-7163.TARG-15-A49"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-A49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-A49%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DLorusso%26aufirst%3DP.%2BM.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DAnderson%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DK.%2BT.%26aulast%3DKagey%26aufirst%3DM.%2BH.%26aulast%3DSirard%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLandau%26aufirst%3DS.%2BB.%26atitle%3DAbstract%2520A49%253A%2520Clinically%2520efficacy%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520TEN-010%2520in%2520an%2520open-label%2520substudy%2520with%2520patients%2520with%2520documented%2520NUT-midline%2520carcinoma%2520%2528NMC%2529%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DA49%26doi%3D10.1158%2F1535-7163.TARG-15-A49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>â <span class="NLM_lpage">1123</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgAW2WO5CyK0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a benzoisoxazoloazepine inhibitor (CPI-0610) of the bromodomain and extra- terminal (BET) family as a candidate for human clinical trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>â <span class="NLM_lpage">1339</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+Coteauthor=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.+C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+benzoisoxazoloazepine+inhibitor+%28CPI-0610%29+of+the+bromodomain+and+extra-+terminal+%28BET%29+family+as+a+candidate+for+human+clinical+trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-ÎºB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0lgAW2WO5CyK0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520benzoisoxazoloazepine%2520inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520bromodomain%2520and%2520extra-%2520terminal%2520%2528BET%2529%2520family%2520as%2520a%2520candidate%2520for%2520human%2520clinical%2520trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-(4-(2,4-nifluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a potent and orally available bromodomain and extraterminal domain (BET) family bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>â <span class="NLM_lpage">8384</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-nifluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfonamide+%28ABBV-075%2FMivebresib%29%2C+a+potent+and+orally+available+bromodomain+and+extraterminal+domain+%28BET%29+family+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lhCyIvmbpnjOw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-nifluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfonamide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520potent%2520and%2520orally%2520available%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">VÃ¡zquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civenni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zadic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbone, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catapano, C. V.</span></span> <span> </span><span class="NLM_article-title">Abstract 5080: BET inhibitors INCB054329 and INCB057643 display significant activity in androgen-independent prostate cancer models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">5080</span>, <span class="refDoi">Â DOI: 10.1158/1538-7445.AM2017-5080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1538-7445.AM2017-5080" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=5080&author=R.+V%C3%A1zquezauthor=G.+Civenniauthor=M.+Marchettiauthor=S.+Zadicauthor=P.+Liuauthor=B.+Ruggeriauthor=G.+M.+Carboneauthor=C.+V.+Catapano&title=Abstract+5080%3A+BET+inhibitors+INCB054329+and+INCB057643+display+significant+activity+in+androgen-independent+prostate+cancer+models&doi=10.1158%2F1538-7445.AM2017-5080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-5080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-5080%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez%26aufirst%3DR.%26aulast%3DCivenni%26aufirst%3DG.%26aulast%3DMarchetti%26aufirst%3DM.%26aulast%3DZadic%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DRuggeri%26aufirst%3DB.%26aulast%3DCarbone%26aufirst%3DG.%2BM.%26aulast%3DCatapano%26aufirst%3DC.%2BV.%26atitle%3DAbstract%25205080%253A%2520BET%2520inhibitors%2520INCB054329%2520and%2520INCB057643%2520display%2520significant%2520activity%2520in%2520androgen-independent%2520prostate%2520cancer%2520models%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D5080%26doi%3D10.1158%2F1538-7445.AM2017-5080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavai, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortolani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMalley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnelle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, V.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Abstract 5789: Discovery of clinical candidate BMS-986158, an oral BET inhibitor, for the treatment of cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">5789</span>, <span class="refDoi">Â DOI: 10.1158/1538-7445.AM2018-5789</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1538-7445.AM2018-5789" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=5789&author=A.+V.+Gavaiauthor=D.+Norrisauthor=D.+Tortolaniauthor=D.+O%E2%80%99Malleyauthor=Y.+Zhaoauthor=C.+Quesnelleauthor=P.+Gillauthor=W.+Vaccaroauthor=T.+Huynhauthor=V.+Ahuja&title=Abstract+5789%3A+Discovery+of+clinical+candidate+BMS-986158%2C+an+oral+BET+inhibitor%2C+for+the+treatment+of+cancer&doi=10.1158%2F1538-7445.AM2018-5789"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2018-5789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2018-5789%26sid%3Dliteratum%253Aachs%26aulast%3DGavai%26aufirst%3DA.%2BV.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DTortolani%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DQuesnelle%26aufirst%3DC.%26aulast%3DGill%26aufirst%3DP.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DAhuja%26aufirst%3DV.%26atitle%3DAbstract%25205789%253A%2520Discovery%2520of%2520clinical%2520candidate%2520BMS-986158%252C%2520an%2520oral%2520BET%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3D5789%26doi%3D10.1158%2F1538-7445.AM2018-5789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>â <span class="NLM_lpage">19759</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0likJuW57Oj3xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Gamal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hing, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blachly, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieselhuber, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goettl, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spevak, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carias, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantoy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanftner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nespi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MathÃ©, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span> <span> </span><span class="NLM_article-title">BRD4 profiling identifies critical chronic lymphocytic leukemia oncogenic circuits and reveals sensitivity to PLX51107, a novel structurally distinct BET inhibitor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">458</span>â <span class="NLM_lpage">477</span>, <span class="refDoi">Â DOI: 10.1158/2159-8290.CD-17-0902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F2159-8290.CD-17-0902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29386193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=458-477&author=H.+G.+Ozerauthor=D.+El-Gamalauthor=B.+Powellauthor=Z.+A.+Hingauthor=J.+S.+Blachlyauthor=B.+Harringtonauthor=S.+Mitchellauthor=N.+R.+Grieselhuberauthor=K.+Williamsauthor=T.+H.+Laiauthor=L.+Alinariauthor=R.+A.+Baiocchiauthor=L.+Brintonauthor=E.+Baskinauthor=M.+Cannonauthor=L.+Beaverauthor=V.+M.+Goettlauthor=D.+M.+Lucasauthor=J.+A.+Woyachauthor=D.+Sampathauthor=A.+M.+Lehmanauthor=L.+Yuauthor=J.+Zhangauthor=Y.+Maauthor=Y.+Zhangauthor=W.+Spevakauthor=S.+Shiauthor=P.+Seversonauthor=R.+Shellooeauthor=H.+Cariasauthor=G.+Tsangauthor=K.+Dongauthor=T.+Ewingauthor=A.+Marimuthuauthor=C.+Tantoyauthor=J.+Waltersauthor=L.+Sanftnerauthor=H.+Rezaeiauthor=M.+Nespiauthor=B.+Matusowauthor=G.+Habetsauthor=P.+Ibrahimauthor=C.+Zhangauthor=E.+A.+Math%C3%A9author=G.+Bollagauthor=J.+C.+Byrdauthor=R.+Lapalombella&title=BRD4+profiling+identifies+critical+chronic+lymphocytic+leukemia+oncogenic+circuits+and+reveals+sensitivity+to+PLX51107%2C+a+novel+structurally+distinct+BET+inhibitor&doi=10.1158%2F2159-8290.CD-17-0902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span></div><div class="casAuthors">Ozer, Hatice Gulcin; El-Gamal, Dalia; Powell, Ben; Hing, Zachary A.; Blachly, James S.; Harrington, Bonnie; Mitchell, Shaneice; Grieselhuber, Nicole R.; Williams, Katie; Lai, Tzung-Huei; Alinari, Lapo; Baiocchi, Robert A.; Brinton, Lindsey; Baskin, Elizabeth; Cannon, Matthew; Beaver, Larry; Goettl, Virginia M.; Lucas, David M.; Woyach, Jennifer A.; Sampath, Deepa; Lehman, Amy M.; Yu, Lianbo; Zhang, Jiazhong; Ma, Yan; Zhang, Ying; Spevak, Wayne; Shi, Songyuan; Severson, Paul; Shellooe, Rafe; Carias, Heidi; Tsang, Garson; Dong, Ken; Ewing, Todd; Marimuthu, Adhirai; Tantoy, Christina; Walters, Jason; Sanftner, Laura; Rezaei, Hamid; Nespi, Marika; Matusow, Bernice; Habets, Gaston; Ibrahim, Prabha; Zhang, Chao; Mathe, Ewy A.; Bollag, Gideon; Byrd, John C.; Lapalombella, Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-477</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins are key regulators of gene expression in cancer.  Herein, we utilize BRD4 profiling to identify crit. pathways involved in pathogenesis of chronic lymphocytic leukemia (CLL).  BRD4 is overexpressed in CLL and is enriched proximal to genes upregulated or de novo expressed in CLL with known functions in disease pathogenesis and progression.  These genes, including key members of the B-cell receptor (BCR) signaling pathway, provide a rationale for this therapeutic approach to identify new targets in alternative types of cancer.  Addnl., we describe PLX51107, a structurally distinct BET inhibitor with novel in vitro and in vivo pharmacol. properties that emulates or exceeds the efficacy of BCR signaling agents in preclin. models of CLL.  Herein, the discovery of the involvement of BRD4 in the core CLL transcriptional program provides a compelling rationale for clin. investigation of PLX51107 as epigenetic therapy in CLL and application of BRD4 profiling in other cancers.  Significance: To date, functional studies of BRD4 in CLL are lacking.  Through integrated genomic, functional, and pharmacol. analyses, we uncover the existence of BRD4-regulated core CLL transcriptional programs and present preclin. proof-of-concept studies validating BET inhibition as an epigenetic approach to target BCR signaling in CLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvzZtZKkb5rVg90H21EOLACvtfcHk0likJuW57Oj3xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D&md5=3715a65bb8cee2a5fe53994a3d106303</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-0902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-0902%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DH.%2BG.%26aulast%3DEl-Gamal%26aufirst%3DD.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHing%26aufirst%3DZ.%2BA.%26aulast%3DBlachly%26aufirst%3DJ.%2BS.%26aulast%3DHarrington%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DS.%26aulast%3DGrieselhuber%26aufirst%3DN.%2BR.%26aulast%3DWilliams%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DT.%2BH.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DBrinton%26aufirst%3DL.%26aulast%3DBaskin%26aufirst%3DE.%26aulast%3DCannon%26aufirst%3DM.%26aulast%3DBeaver%26aufirst%3DL.%26aulast%3DGoettl%26aufirst%3DV.%2BM.%26aulast%3DLucas%26aufirst%3DD.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DK.%26aulast%3DEwing%26aufirst%3DT.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DTantoy%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DJ.%26aulast%3DSanftner%26aufirst%3DL.%26aulast%3DRezaei%26aufirst%3DH.%26aulast%3DNespi%26aufirst%3DM.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMath%25C3%25A9%26aufirst%3DE.%2BA.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DLapalombella%26aufirst%3DR.%26atitle%3DBRD4%2520profiling%2520identifies%2520critical%2520chronic%2520lymphocytic%2520leukemia%2520oncogenic%2520circuits%2520and%2520reveals%2520sensitivity%2520to%2520PLX51107%252C%2520a%2520novel%2520structurally%2520distinct%2520BET%2520inhibitor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D458%26epage%3D477%26doi%3D10.1158%2F2159-8290.CD-17-0902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak bromo and extra terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>â <span class="NLM_lpage">9126</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+B.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+bromo+and+extra+terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0lhBKOuxLd9JKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCopley%26aufirst%3DR.%2BC.%2BB.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the bromodomain and extra terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>â <span class="NLM_lpage">9092</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+bromodomain+and+extra+terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0lhBKOuxLd9JKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-2-carboxamide (ABBV-744), a BET bromodomain inhibitor with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>â <span class="NLM_lpage">5623</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+J.+Kovarauthor=M.+Buiauthor=E.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+bromodomain+inhibitor+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0lhBKOuxLd9JKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520bromodomain%2520inhibitor%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>â <span class="NLM_lpage">4334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0lirkLBW1DfEeg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">WyspiaÅska, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a bromodomain and extraterminal domain (BET) inhibitor selective for the N-terminal bromodomains that retains anti-inflammatory and antiproliferative phenotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9020</span>â <span class="NLM_lpage">9044</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9020-9044&author=C.+R.+Wellawayauthor=P.+Bamboroughauthor=S.+G.+Bernardauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Cutlerauthor=E.+H.+Demontauthor=J.+P.+Evansauthor=L.+Gordonauthor=B.+Karamshiauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+J.+Mitchellauthor=I.+Riojaauthor=P.+E.+Sodenauthor=S.+Taylorauthor=R.+J.+Watsonauthor=R.+Willisauthor=J.+M.+Woolvenauthor=B.+S.+Wyspia%C5%84skaauthor=W.+J.+Kerrauthor=R.+K.+Prinjha&title=Structure-based+design+of+a+bromodomain+and+extraterminal+domain+%28BET%29+inhibitor+selective+for+the+N-terminal+bromodomains+that+retains+anti-inflammatory+and+antiproliferative+phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype</span></div><div class="casAuthors">Wellaway, Christopher R.; Bamborough, Paul; Bernard, Sharon G.; Chung, Chun-wa; Craggs, Peter D.; Cutler, Leanne; Demont, Emmanuel H.; Evans, John P.; Gordon, Laurie; Karamshi, Bhumika; Lewis, Antonia J.; Lindon, Matthew J.; Mitchell, Darren J.; Rioja, Inmaculada; Soden, Peter E.; Taylor, Simon; Watson, Robert J.; Willis, Rob; Woolven, James M.; Wyspianska, Beata S.; Kerr, William J.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9020-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) family of epigenetic regulators comprises four proteins (BRD2, BRD3, BRD4, BRDT), each contg. tandem bromodomains.  To date, small mol. inhibitors of these proteins typically bind all eight bromodomains of the family with similar affinity, resulting in a diverse range of biol. effects.  To enable further understanding of the broad phenotype characteristic of pan-BET inhibition, the development of inhibitors selective for individual, or sets of, bromodomains within the family is required.  In this regard, we report the discovery of a potent probe mol. possessing up to 150-fold selectivity for the N-terminal bromodomains (BD1s) over the C-terminal bromodomains (BD2s) of the BETs.  Guided by structural information, a specific amino acid difference between BD1 and BD2 domains was targeted for selective interaction with chem. functionality appended to the previously developed I-BET151 scaffold.  Data presented herein demonstrate that selective inhibition of BD1 domains is sufficient to drive anti-inflammatory and antiproliferative effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUREcHUhFfbVg90H21EOLACvtfcHk0lirkLBW1DfEeg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M&md5=52b2505bb0a36f526a08bb9bad1f2c26</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BG.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWillis%26aufirst%3DR.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DWyspia%25C5%2584ska%26aufirst%3DB.%2BS.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520inhibitor%2520selective%2520for%2520the%2520N-terminal%2520bromodomains%2520that%2520retains%2520anti-inflammatory%2520and%2520antiproliferative%2520phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9020%26epage%3D9044%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK789: A selective inhibitor of the first bromodomains (BD1) of the bromo and extra terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>â <span class="NLM_lpage">9069</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.+W.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=GSK789%3A+A+selective+inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+bromo+and+extra+terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0lgGGJSDQCEl7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK789%253A%2520A%2520selective%2520inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of orally bioavailable chromone derivatives as potent and selective BRD4 inhibitors: Scaffold hopping, optimization, and pharmacological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5242</span>â <span class="NLM_lpage">5256</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5242-5256&author=Z.+Liuauthor=H.+Chenauthor=P.+Wangauthor=Y.+Liauthor=E.+A.+Woldauthor=P.+G.+Leonardauthor=S.+Josephauthor=A.+R.+Brasierauthor=B.+Tianauthor=J.+Zhou&title=Discovery+of+orally+bioavailable+chromone+derivatives+as+potent+and+selective+BRD4+inhibitors%3A+Scaffold+hopping%2C+optimization%2C+and+pharmacological+evaluation&doi=10.1021%2Facs.jmedchem.0c00035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Bioavailable Chromone Derivatives as Potent and Selective BRD4 Inhibitors: Scaffold Hopping, Optimization, and Pharmacological Evaluation</span></div><div class="casAuthors">Liu Zhiqing; Chen Haiying; Wang Pingyuan; Li Yi; Wold Eric A; Zhou Jia; Leonard Paul G; Joseph Sarah; Brasier Allan R; Tian Bing; Tian Bing; Zhou Jia; Zhou Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5242-5256</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 4 (BRD4) represents a promising drug target for anti-inflammatory therapeutics.  Herein, we report the design, synthesis, and pharmacological evaluation of novel chromone derivatives via scaffold hopping to discover a new class of orally bioavailable BRD4-selective inhibitors.  Two potent BRD4 bromodomain 1 (BD1)-selective inhibitors 44 (ZL0513) and 45 (ZL0516) have been discovered with high binding affinity (IC50 values of 67-84 nM) and good selectivity over other BRD family proteins and distant BD-containing proteins.  Both compounds significantly inhibited the expression of Toll-like receptor-induced inflammatory genes in vitro and airway inflammation in murine models.  The cocrystal structure of 45 in complex with human BRD4 BD1 at a high resolution of 2.0 ÃA has been solved, offering a solid structural basis for its binding validation and further structure-based optimization.  These BRD4 BD1 inhibitors demonstrated impressive in vivo efficacy and overall promising pharmacokinetic properties, indicating their therapeutic potential for the treatment of inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR1QLaeKXMGxJ0d-O_J5C0AfW6udTcc2eZubuB7u5RD6bntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D&md5=d65e4298cbfff545c6c333713e4db210</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00035%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520orally%2520bioavailable%2520chromone%2520derivatives%2520as%2520potent%2520and%2520selective%2520BRD4%2520inhibitors%253A%2520Scaffold%2520hopping%252C%2520optimization%252C%2520and%2520pharmacological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5242%26epage%3D5256%26doi%3D10.1021%2Facs.jmedchem.0c00035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of 8-methyl-pyrrolo[1,2-<i>a</i>]pyrazin-1(2<i>H</i>)-one derivatives as highly potent and selective bromodomain and extra-terminal (BET) bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3956</span>â <span class="NLM_lpage">3975</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b01784</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01784" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXls1Cjtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3956-3975&author=Z.+Liauthor=S.+Xiaoauthor=Y.+Yangauthor=C.+Chenauthor=T.+Luauthor=Z.+Chenauthor=H.+Jiangauthor=S.+Chenauthor=C.+Luoauthor=B.+Zhou&title=Discovery+of+8-methyl-pyrrolo%5B1%2C2-a%5Dpyrazin-1%282H%29-one+derivatives+as+highly+potent+and+selective+bromodomain+and+extra-terminal+%28BET%29+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.9b01784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 8-Methyl-pyrrolo[1,2-a]pyrazin-1(2H)-one Derivatives as Highly Potent and Selective Bromodomain and Extra-Terminal (BET) Bromodomain Inhibitors</span></div><div class="casAuthors">Li, Zizhou; Xiao, Senhao; Yang, Yaxi; Chen, Chao; Lu, Tian; Chen, Zhifeng; Jiang, Hualiang; Chen, Shijie; Luo, Cheng; Zhou, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3956-3975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins have recently emerged as promising drug targets for cancer therapy.  In this study, identification of an 8-methyl-pyrrolo[1,2-a]pyrazin-1(2H)-one fragment (47) as a new binder to the BET bromodomains and the subsequent incorporation of fragment 47 to the scaffold of ABBV-075, which recently entered Phase I clin. trials, enabled the generation of a series of highly potent BET bromodomain inhibitors.  Further druggability optimization led to the discovery of compd. 38 as a potential preclin. candidate.  Significantly, compared with ABBV-075, which exhibits a 63-fold selectivity for BRD4(1) over EP300, compd. 38 demonstrates an excellent selectivity for the BET bromodomain family over other bromodomains, with an ~ 1500-fold selectivity for BRD4(1) over EP300.  Orally administered 38 achieves a complete inhibition of tumor growth with a tumor growth inhibition (TGI) of 99.7% accompanied by good tolerability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBnoe3zL-x2LVg90H21EOLACvtfcHk0lgGGJSDQCEl7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXls1Cjtb0%253D&md5=34cf8275969bd7a1ff214b0eb1e74ca9</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01784%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DB.%26atitle%3DDiscovery%2520of%25208-methyl-pyrrolo%255B1%252C2-a%255Dpyrazin-1%25282H%2529-one%2520derivatives%2520as%2520highly%2520potent%2520and%2520selective%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3956%26epage%3D3975%26doi%3D10.1021%2Facs.jmedchem.9b01784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[cd]indol-2(1<i>H</i>)-ones and pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-ones as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the first bromodomain with potential high efficiency against acute gouty arthritis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11080</span>â <span class="NLM_lpage">11107</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11080-11107&author=F.+Jiangauthor=Q.+Huauthor=Z.+Zhangauthor=H.+Liauthor=H.+Liauthor=D.+Zhangauthor=H.+Liauthor=Y.+Maauthor=J.+Xuauthor=H.+Chenauthor=Y.+Cuiauthor=Y.+Zhiauthor=Y.+Zhangauthor=J.+Xuauthor=J.+Zhuauthor=T.+Luauthor=Y.+Chen&title=Discovery+of+benzo%5Bcd%5Dindol-2%281H%29-ones+and+pyrrolo%5B4%2C3%2C2-de%5Dquinolin-2%281H%29-ones+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+first+bromodomain+with+potential+high+efficiency+against+acute+gouty+arthritis&doi=10.1021%2Facs.jmedchem.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></div><div class="casAuthors">Jiang, Fei; Hu, Qinghua; Zhang, Zhimin; Li, Hongmei; Li, Huili; Zhang, Dewei; Li, Hanwen; Ma, Yu; Xu, Jingjing; Chen, Haifang; Cui, Yong; Zhi, Yanle; Zhang, Yanmin; Xu, Junyu; Zhu, Jiapeng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11080-11107</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues.  Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2).  Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family.  Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1.  Through structure-based optimization, highly active and selective compds. are ultimately obtained.  The representative compds. are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2).  Among them, we further show that 68 (LT052) mediates BRD4/NF-ÎºB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.  Therefore, selective pharmacol. modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav2e3P7XQ1LVg90H21EOLACvtfcHk0ljM7tQsztkG0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E&md5=5cce669e137812d20f8ba2a9785f4666</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520and%2520pyrrolo%255B4%252C3%252C2-de%255Dquinolin-2%25281H%2529-ones%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520first%2520bromodomain%2520with%2520potential%2520high%2520efficiency%2520against%2520acute%2520gouty%2520arthritis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11080%26epage%3D11107%26doi%3D10.1021%2Facs.jmedchem.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Horst, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of novel covalent bromodomain and extra-terminal domain (BET) inhibitors targeting a methionine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8202</span>â <span class="NLM_lpage">8211</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8202-8211&author=O.+A.+Kharenkoauthor=R.+G.+Patelauthor=S.+D.+Brownauthor=C.+Calosingauthor=A.+Whiteauthor=D.+Lakshminarasimhanauthor=R.+K.+Sutoauthor=B.+C.+Duffyauthor=D.+B.+Kitchenauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=E.+H.+van+der%0AHorstauthor=P.+R.+Young&title=Design+and+characterization+of+novel+covalent+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+targeting+a+methionine&doi=10.1021%2Facs.jmedchem.8b00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Novel Covalent Bromodomain and Extra-Terminal Domain (BET) Inhibitors Targeting a Methionine</span></div><div class="casAuthors">Kharenko, Olesya A.; Patel, Reena G.; Brown, S. David; Calosing, Cyrus; White, Andre; Lakshminarasimhan, Damodharan; Suto, Robert K.; Duffy, Bryan C.; Kitchen, Douglas B.; McLure, Kevin G.; Hansen, Henrik C.; van der Horst, Edward H.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8202-8211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET proteins are key epigenetic regulators that regulate transcription through binding to acetylated lysine (AcLys) residues of histones and transcription factors through bromodomains (BDs).  The disruption of this interaction with small mol. bromodomain inhibitors is a promising approach to treat various diseases including cancer, autoimmune and cardiovascular diseases.  Covalent inhibitors can potentially offer a more durable target inhibition leading to improved in vivo pharmacol.  Here the authors describe the design of covalent inhibitors of BRD4(BD1) that target a methionine in the binding pocket by attaching an epoxide warhead to a suitably oriented noncovalent inhibitor.  Using thermal denaturation, MALDI-TOF mass spectrometry, and an x-ray crystal structure, the authors demonstrate that these inhibitors selectively form a covalent bond with Met 149 in BRD4(BD1) but not other bromodomains and provide durable transcriptional and antiproliferative activity in cell based assays.  Covalent targeting of methionine offers a novel approach to drug discovery for BET proteins and other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHccGRxf5ShrVg90H21EOLACvtfcHk0ljM7tQsztkG0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO&md5=1c3ae393a5f8a8c12ffb16e6f3798118</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00666%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520novel%2520covalent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520targeting%2520a%2520methionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8202%26epage%3D8211%26doi%3D10.1021%2Facs.jmedchem.8b00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alferiev, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinson, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breuer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. J.</span></span> <span> </span><span class="NLM_article-title">High reactivity of alkyl sulfides towards epoxides under conditions of collagen fixation--a convenient approach to 2-amino-4-butyrolactones</span>. <i>Biomaterials</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2501</span>â <span class="NLM_lpage">2506</span>, <span class="refDoi">Â DOI: 10.1016/S0142-9612(00)00440-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2FS0142-9612%2800%2900440-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=11516082" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlKksLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=2501-2506&author=I.+S.+Alferievauthor=J.+T.+Hinsonauthor=M.+Ogleauthor=E.+Breuerauthor=R.+J.+Levy&title=High+reactivity+of+alkyl+sulfides+towards+epoxides+under+conditions+of+collagen+fixation%2D%2Da+convenient+approach+to+2-amino-4-butyrolactones&doi=10.1016%2FS0142-9612%2800%2900440-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">High reactivity of alkyl sulfides towards epoxides under conditions of collagen fixation - a convenient approach to 2-amino-4-butyrolactones</span></div><div class="casAuthors">Alferiev, I. S.; Hinson, J. T.; Ogle, M.; Breuer, E.; Levy, R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Biomaterials</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2501-2506</span>CODEN:
                <span class="NLM_cas:coden">BIMADU</span>;
        ISSN:<span class="NLM_cas:issn">0142-9612</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Epoxy crosslinking agents have been investigated for use in the fabrication of bioprosthetic devices, such as heterograft heart valve prostheses.  It has been generally assumed that epoxy crosslinking takes place via amino-epoxy reactions.  The present study investigated the hypothesis that the reactions of methionine residues with epoxides also can occur in biomaterial crosslinking.  A series of model reactions were studied in which a mono-epoxide was combined with individual alkyl sulfides.  In the present studies epoxides rapidly alkylate aliph. sulfides, including methionine derivs., in buffered aq. solns. at room temp. and pH close to neutral, forming sulfonium compds., which are stable at pH 5-7 at temps. up to 50Â°C, except for cases in which methionine derivs. with non-protected carboxy groups are used.  The rate of reaction remains practically unchanged within the range of pH from 5 to 12, whereas in strongly alk. media the reverse reaction occurs.  This discovery can provide a better understanding of processes occurring in the fixation of bioprosthetic tissues with polyepoxides.  It can also develop into a site-specific method to label methionine residues in proteins.  The carboxy group-contg. sulfonium betaines derived from N-protected methionines undergo cyclization in unexpectedly mild conditions, which can be used as an efficient method for prepn. of N-protected 2-amino-4-butyrolactones with sensitive protective groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnU7-m8ArXsLVg90H21EOLACvtfcHk0lj2UnzugDbOmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlKksLo%253D&md5=e2226438d8f27afa4f6cf2cb5d1b4b91</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1016%2FS0142-9612%2800%2900440-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0142-9612%252800%252900440-3%26sid%3Dliteratum%253Aachs%26aulast%3DAlferiev%26aufirst%3DI.%2BS.%26aulast%3DHinson%26aufirst%3DJ.%2BT.%26aulast%3DOgle%26aufirst%3DM.%26aulast%3DBreuer%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BJ.%26atitle%3DHigh%2520reactivity%2520of%2520alkyl%2520sulfides%2520towards%2520epoxides%2520under%2520conditions%2520of%2520collagen%2520fixation--a%2520convenient%2520approach%2520to%25202-amino-4-butyrolactones%26jtitle%3DBiomaterials%26date%3D2001%26volume%3D22%26spage%3D2501%26epage%3D2506%26doi%3D10.1016%2FS0142-9612%2800%2900440-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanus, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantuck, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakhotia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abida, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alumkal, J. J.</span></span> <span> </span><span class="NLM_article-title">A phase Ib/IIa study of the pan-BET inhibitor ZEN-3694 in combination with enzalutamide in patients with metastatic castration-resistant prostate cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5338</span>â <span class="NLM_lpage">5347</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-20-1707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1078-0432.CCR-20-1707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32694156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFCku7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=5338-5347&author=R.+R.+Aggarwalauthor=M.+T.+Schweizerauthor=D.+M.+Nanusauthor=A.+J.+Pantuckauthor=E.+I.+Heathauthor=E.+Campeauauthor=S.+Attwellauthor=K.+Norekauthor=M.+Snyderauthor=L.+Baumanauthor=S.+Lakhotiaauthor=F.+Y.+Fengauthor=E.+J.+Smallauthor=W.+Abidaauthor=J.+J.+Alumkal&title=A+phase+Ib%2FIIa+study+of+the+pan-BET+inhibitor+ZEN-3694+in+combination+with+enzalutamide+in+patients+with+metastatic+castration-resistant+prostate+cancer&doi=10.1158%2F1078-0432.CCR-20-1707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">A phase Ib/lla study of the Pan-BET inhibitor ZEN-3694 in combination with enzalutamide in patients with metastatic castration-resistant prostate cancer</span></div><div class="casAuthors">Aggarwal, Rahul R.; Schweizer, Michael T.; Nanus, David M.; Pantuck, Allan J.; Heath, Elisabeth I.; Campeau, Eric; Attwell, Sarah; Norek, Karen; Snyder, Margo; Bauman, Lisa; Lakhotia, Sanjay; Feng, Felix Y.; Small, Eric J.; Abida, Wassim; Alumkal, Joshi J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5338-5347</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: ZEN-3694 is a bromodomain extraterminal inhibitor (BETi) with activity in androgen-signaling inhibitor (ASI)-resistant models.  The safety and efficacy of ZEN-3694 plus enzalutamide was evaluated in a phase Ib/IIa study in metastatic castration-resistant prostate cancer (mCRPC).  Patients and Methods: Patients had progressive mCRPC with prior resistance to abiraterone and/or enzalutanride. 3+3 dose escalation was followed by dose expansion in parallel cohorts (ZEN-3694 at 48 and 96 mg orally once daily, resp.).  Results: Seventy-five patients were enrolled (N = 26 and 14 in dose expansion at low- and high-dose ZEN-3694, resp.).  Thirty (40.0%) patients were resistant to abiraterone, 34 (45.3%) to enzalutanride, and 11 (14.7%) to both.  ZEN-3694 dosing ranged from 36 to 144 mg daily without reaching an MTD.  Fourteen patients (18.7%) experienced grade >3 toxicides, including three patients with grade 3 thrombocytopenia (4%).  An exposuredependent decrease in whole-blood RNA expression of BETi targets was obsd. (up to fourfold mean difference at 4 h post-ZEN-3694 dose; P 0.0001).  The median radiog. progression- free survival (rPFS) was 9.0 mo [95% confidence interval (CI), 4.6-12.9] and composite median radiog. or clin. progression-free survival (PFS) was 5.5 mo (95% CI, 4.0-7.8).  Median duration of treatment was 3.5 mo (range, 0-34.7+).  Lower androgen receptor (AR) transcriptional activity in baseline tumor biopsies was assocd. with longer rPFS (median rPFS 10.4 vs. 4.3 mo).  Conclusions: ZEN-3694 plus er.zalutamide demonstrated acceptable tolerability and potential efficacy in patients with ASIresistant mCRPC.  Further prospective study is warranted including in mCRPC harboring low AR transcriptional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry2ba_lIdCq7Vg90H21EOLACvtfcHk0lj2UnzugDbOmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFCku7rP&md5=5bda2dace9b6c9fc3d3b2d3ad8718450</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-20-1707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-20-1707%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DR.%2BR.%26aulast%3DSchweizer%26aufirst%3DM.%2BT.%26aulast%3DNanus%26aufirst%3DD.%2BM.%26aulast%3DPantuck%26aufirst%3DA.%2BJ.%26aulast%3DHeath%26aufirst%3DE.%2BI.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DSnyder%26aufirst%3DM.%26aulast%3DBauman%26aufirst%3DL.%26aulast%3DLakhotia%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DSmall%26aufirst%3DE.%2BJ.%26aulast%3DAbida%26aufirst%3DW.%26aulast%3DAlumkal%26aufirst%3DJ.%2BJ.%26atitle%3DA%2520phase%2520Ib%252FIIa%2520study%2520of%2520the%2520pan-BET%2520inhibitor%2520ZEN-3694%2520in%2520combination%2520with%2520enzalutamide%2520in%2520patients%2520with%2520metastatic%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2020%26volume%3D26%26spage%3D5338%26epage%3D5347%26doi%3D10.1158%2F1078-0432.CCR-20-1707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carceller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">Advances in the development of histone lysine demethylase inhibitors</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">52</span>â <span class="NLM_lpage">60</span>, <span class="refDoi">Â DOI: 10.1016/j.coph.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.coph.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26057211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=52-60&author=T.+Maesauthor=E.+Carcellerauthor=J.+Salasauthor=A.+Ortegaauthor=C.+Buesa&title=Advances+in+the+development+of+histone+lysine+demethylase+inhibitors&doi=10.1016%2Fj.coph.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of histone lysine demethylase inhibitors</span></div><div class="casAuthors">Maes, Tamara; Carceller, Elena; Salas, Jordi; Ortega, Alberto; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The covalent modification of histones is closely assocd. with regulation of gene transcription.  Chromatin modifications have been suggested to represent an epigenetic code that is dynamically 'written' and 'erased' by specialized proteins, and 'read', or interpreted, by proteins that translate the code into gene expression changes.  Initially thought to be an irreversible process, histone methylation is now known to be reversed by demethylases, FAD dependent amineoxidases and by iron(II)-alpha-ketoglutarate dependent deoxygenases of the Jumonji family.  Altered histone demethylase activities have been assocd. with human disease, including cancer.  The first wave of novel investigational drugs directed against KDM1A has recently entered the clinic, and the first specific inhibitor targeting a Jumonji KDM is advancing in preclin. regulatory studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCr97IlCYFrVg90H21EOLACvtfcHk0lj2UnzugDbOmw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D&md5=02e8a5f8d426743be1eb209c329bd3ed</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DSalas%26aufirst%3DJ.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.coph.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>â <span class="NLM_lpage">953</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lggsY3O84TSHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): A computational study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e25444</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0025444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1371%2Fjournal.pone.0025444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21984927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=X.+Kongauthor=S.+Ouyangauthor=Z.+Liangauthor=J.+Luauthor=L.+Chenauthor=B.+Shenauthor=D.+Liauthor=M.+Zhengauthor=K.+K.+Liauthor=C.+Luoauthor=H.+Jiang&title=Catalytic+mechanism+investigation+of+lysine-specific+demethylase+1+%28LSD1%29%3A+A+computational+study&doi=10.1371%2Fjournal.pone.0025444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span></div><div class="casAuthors">Kong, Xiangqian; Ouyang, Sisheng; Liang, Zhongjie; Lu, Junyan; Chen, Liang; Shen, Bairong; Li, Donghai; Zheng, Mingyue; Li, Keqin Kathy; Luo, Cheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25444</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), the first identified histone demethylase, is a flavin-dependent amine oxidase which specifically demethylates mono- or di-methylated H3K4 and H3K9 via a redox process.  It participates in a broad spectrum of biol. processes and is of high importance in cell proliferation, adipogenesis, spermatogenesis, chromosome segregation and embryonic development.  To date, as a potential drug target for discovering anti-tumor drugs, the medical significance of LSD1 has been greatly appreciated.  However, the catalytic mechanism for the rate-limiting reductive half-reaction in demethylation remains controversial.  By employing a combined computational approach including mol. modeling, mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns., the catalytic mechanism of dimethylated H3K4 demethylation by LSD1 was characterized in details.  The three-dimensional (3D) model of the complex was composed of LSD1, CoREST, and histone substrate.  A 30-ns MD simulation of the model highlights the pivotal role of the conserved Tyr761 and lysine-water-flavin motif in properly orienting FAD with respect to substrate.  The synergy of the two factors effectively stabilizes the catalytic environment and facilitated the demethylation reaction.  On the basis of the reasonable consistence between simulation results and available mutagenesis data, QM/MM strategy was further employed to probe the catalytic mechanism of the reductive half-reaction in demethylation.  The characteristics of the demethylation pathway detd. by the potential energy surface and charge distribution anal. indicates that this reaction belongs to the direct hydride transfer mechanism.  Our study provides insights into the LSD1 mechanism of reductive half-reaction in demethylation and has important implications for the discovery of regulators against LSD1 enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIm_wJ2koFrVg90H21EOLACvtfcHk0lggsY3O84TSHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP&md5=7ccdcdcf6e8770ea0d77c1cc80f79da8</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025444%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCatalytic%2520mechanism%2520investigation%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%253A%2520A%2520computational%2520study%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0025444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>437</i></span>,  <span class="NLM_fpage">432</span>â <span class="NLM_lpage">435</span>, <span class="refDoi">Â DOI: 10.1038/nature04021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature04021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16079794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation&doi=10.1038%2Fnature04021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Cooch, Neil; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">432-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors have previously described a multiprotein complex termed the BHC or BRAF-HDAC complex, which is required for the repression of neuronal-specific genes.  The authors have shown that the BHC complex is recruited by a neuronal silencer, REST (RE1-silencing transcription factor), and mediates the repression of REST-responsive genes.  BHC is a multiprotein complex consisting of two enzymic activities: a histone deacetylase (HDAC1 or 2) and a recently described histone demethylase (BHC110, also known as LSD1 or AOF2).  Here the authors show that BHC110-contg. complexes show a nearly fivefold increase in demethylation of histone H3 lysine 4 (H3K4) compared to recombinant BHC110.  Furthermore, recombinant BHC110 is unable to demethylate H3K4 on nucleosomes, but BHC110-contg. complexes readily demethylate nucleosomes.  In vitro reconstitution of the BHC complex using recombinant subunits reveals an essential role for the REST corepressor CoREST, not only in stimulating demethylation on core histones but also promoting demethylation of nucleosomal substrates.  The authors find that nucleosomal demethylation is the result of CoREST enhancing the assocn. between BHC110 and nucleosomes.  Depletion of CoREST in in vivo cell culture results in de-repression of REST-responsive gene expression and increased methylation of H3K4.  Together, these results highlight an essential role for CoREST in demethylation of H3K4 both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmTdZApayNhrVg90H21EOLACvtfcHk0lggsY3O84TSHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D&md5=535a6d800bd6672192be133516b4dee7</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1038%2Fnature04021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04021%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435%26doi%3D10.1038%2Fnature04021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¼nther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼le, R.</span></span> <span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>437</i></span>,  <span class="NLM_fpage">436</span>â <span class="NLM_lpage">439</span>, <span class="refDoi">Â DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0lhfZbBe7DH2VQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perillo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ombra, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiariotti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malorni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbondanza, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avvedimento, E. V.</span></span> <span> </span><span class="NLM_article-title">DNA oxidation as triggered by H3K9me2 demethylation drives estrogen-induced gene expression</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">202</span>â <span class="NLM_lpage">206</span>, <span class="refDoi">Â DOI: 10.1126/science.1147674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1126%2Fscience.1147674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=18187655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXis1amug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2008&pages=202-206&author=B.+Perilloauthor=M.+N.+Ombraauthor=A.+Bertoniauthor=C.+Cuozzoauthor=S.+Sacchettiauthor=A.+Sassoauthor=L.+Chiariottiauthor=A.+Malorniauthor=C.+Abbondanzaauthor=E.+V.+Avvedimento&title=DNA+oxidation+as+triggered+by+H3K9me2+demethylation+drives+estrogen-induced+gene+expression&doi=10.1126%2Fscience.1147674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Oxidation as Triggered by H3K9me2 Demethylation Drives Estrogen-Induced Gene Expression</span></div><div class="casAuthors">Perillo, Bruno; Ombra, Maria Neve; Bertoni, Alessandra; Cuozzo, Concetta; Sacchetti, Silvana; Sasso, Annarita; Chiariotti, Lorenzo; Malorni, Antonio; Abbondanza, Ciro; Avvedimento, Enrico V.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">5860</span>),
    <span class="NLM_cas:pages">202-206</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Modifications at the N-terminal tails of nucleosomal histones are required for efficient transcription in vivo.  We analyzed how H3 histone methylation and demethylation control expression of estrogen-responsive genes and show that a DNA-bound estrogen receptor directs transcription by participating in bending chromatin to contact the RNA polymerase II recruited to the promoter.  This process is driven by receptor-targeted demethylation of H3 lysine 9 at both enhancer and promoter sites and is achieved by activation of resident LSD1 demethylase.  Localized demethylation produces hydrogen peroxide, which modifies the surrounding DNA and recruits 8-oxoguanine-DNA glycosylase 1 and topoisomerase IIÎ², triggering chromatin and DNA conformational changes that are essential for estrogen-induced transcription.  Our data show a strategy that uses controlled DNA damage and repair to guide productive transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JvSAT1JwG7Vg90H21EOLACvtfcHk0lhfZbBe7DH2VQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXis1amug%253D%253D&md5=1208f5a6e8572df8deb2ef0588b6c102</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1126%2Fscience.1147674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1147674%26sid%3Dliteratum%253Aachs%26aulast%3DPerillo%26aufirst%3DB.%26aulast%3DOmbra%26aufirst%3DM.%2BN.%26aulast%3DBertoni%26aufirst%3DA.%26aulast%3DCuozzo%26aufirst%3DC.%26aulast%3DSacchetti%26aufirst%3DS.%26aulast%3DSasso%26aufirst%3DA.%26aulast%3DChiariotti%26aufirst%3DL.%26aulast%3DMalorni%26aufirst%3DA.%26aulast%3DAbbondanza%26aufirst%3DC.%26aulast%3DAvvedimento%26aufirst%3DE.%2BV.%26atitle%3DDNA%2520oxidation%2520as%2520triggered%2520by%2520H3K9me2%2520demethylation%2520drives%2520estrogen-induced%2520gene%2520expression%26jtitle%3DScience%26date%3D2008%26volume%3D319%26spage%3D202%26epage%3D206%26doi%3D10.1126%2Fscience.1147674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">A systematic review of histone lysine-specific demethylase 1 and its inhibitors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1032</span>â <span class="NLM_lpage">1071</span>, <span class="refDoi">Â DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+systematic+review+of+histone+lysine-specific+demethylase+1+and+its+inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0lhfZbBe7DH2VQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520systematic%2520review%2520of%2520histone%2520lysine-specific%2520demethylase%25201%2520and%2520its%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosseini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span> <span> </span><span class="NLM_article-title">A comprehensive review of lysine-specific demethylase 1 and its roles in cancer</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1123</span>â <span class="NLM_lpage">1142</span>, <span class="refDoi">Â DOI: 10.2217/epi-2017-0022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2217%2Fepi-2017-0022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28699367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVWnu7jP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1123-1142&author=A.+Hosseiniauthor=S.+Minucci&title=A+comprehensive+review+of+lysine-specific+demethylase+1+and+its+roles+in+cancer&doi=10.2217%2Fepi-2017-0022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive review of lysine-specific demethylase 1 and its roles in cancer</span></div><div class="casAuthors">Hosseini, Amir; Minucci, Saverio</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1123-1142</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone methylation plays a key role in the regulation of chromatin structure, and its dynamics regulates important cellular processes.  The investigation of the role of alterations in histone methylation in cancer has led to the identification of histone methyltransferases and demethylases as promising novel targets for therapy.  Lysine-specific demethylase 1(LSD1, also known as KDM1A) is the first discovered histone lysine demethylase, with the ability to demethylase H3K4me1/2 and H3K9me1/2 at target loci in a context-dependent manner.  LSD1 regulates the balance between self-renewal and differentiation of stem cells, and is highly expressed in various cancers, playing an important role in differentiation and self-renewal of tumor cells.  In this review, we summarize recent studies about the LSD1, its role in normal and tumor cells, and the potential use of small mol. LSD1 inhibitors in therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8i-cNutwmhbVg90H21EOLACvtfcHk0liwkFftVTnMVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWnu7jP&md5=3e64834c4968dd8545dcce30c2bc5c45</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.2217%2Fepi-2017-0022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2017-0022%26sid%3Dliteratum%253Aachs%26aulast%3DHosseini%26aufirst%3DA.%26aulast%3DMinucci%26aufirst%3DS.%26atitle%3DA%2520comprehensive%2520review%2520of%2520lysine-specific%2520demethylase%25201%2520and%2520its%2520roles%2520in%2520cancer%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26spage%3D1123%26epage%3D1142%26doi%3D10.2217%2Fepi-2017-0022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Tranylcypromine based lysine-specific demethylase 1 inhibitor: Summary and perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14197</span>â <span class="NLM_lpage">14215</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14197-14215&author=X.+J.+Daiauthor=Y.+Liuauthor=X.+P.+Xiongauthor=L.+P.+Xueauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Tranylcypromine+based+lysine-specific+demethylase+1+inhibitor%3A+Summary+and+perspective&doi=10.1021%2Facs.jmedchem.0c00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective</span></div><div class="casAuthors">Dai, Xing-Jie; Liu, Ying; Xiong, Xiao-Peng; Xue, Lei-Peng; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14197-14215</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histone lysine-specific demethylase 1 (LSD1/KDM1A) has become an important and promising anticancer target since it was first identified in 2004 and specially demethylates lysine residues of histone H3K4me1/2 and H3K9me1/2.  LSD1 is ubiquitously overexpressed in diverse cancers, and abrogation of LSD1 results in inhibition of proliferation, invasion, and migration in cancer cells.  Over the past decade, a no. of biol. active small-mol. LSD1 inhibitors have been developed.  To date, six trans-2-phenylcyclopropylamine (TCP)-based LSD1 inhibitors (including TCP, ORY-1001, GSK-2879552, INCB059872, IMG-7289, and ORY-2001) that covalently bind to the FAD (FAD) within the LSD1 catalytic cavity have already entered into clin. trials.  Here, we provide an overview about the structures, activities, and structure-activity relationship (SAR) of TCP-based LSD1 inhibitors that mainly covers the literature from 2008 to date.  The opportunities, challenges, and future research directions in this emerging and promising field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqasvgaOVZu7Vg90H21EOLACvtfcHk0liwkFftVTnMVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO&md5=88638bc10c8c090b3df3a0d2f2c020a8</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DX.%2BP.%26aulast%3DXue%26aufirst%3DL.%2BP.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTranylcypromine%2520based%2520lysine-specific%2520demethylase%25201%2520inhibitor%253A%2520Summary%2520and%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14197%26epage%3D14215%26doi%3D10.1021%2Facs.jmedchem.0c00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">LSD1/KDM1A inhibitors in clinical trials: Advances and prospects</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">129</span>, <span class="refDoi">Â DOI: 10.1186/s13045-019-0811-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1186%2Fs13045-019-0811-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31801559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=129&author=Y.+Fangauthor=G.+Liaoauthor=B.+Yu&title=LSD1%2FKDM1A+inhibitors+in+clinical+trials%3A+Advances+and+prospects&doi=10.1186%2Fs13045-019-0811-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1/KDM1A inhibitors in clinical trials: advances and prospects</span></div><div class="casAuthors">Fang Yuan; Liao Guochao; Yu Bin; Yu Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone demethylase LSD1 plays key roles during carcinogenesis, targeting LSD1 is becoming an emerging option for the treatment of cancers.  Numerous LSD1 inhibitors have been reported to date, some of them such as TCP, ORY-1001, GSK-2879552, IMG-7289, INCB059872, CC-90011, and ORY-2001 currently undergo clinical assessment for cancer therapy, particularly for small lung cancer cells (SCLC) and acute myeloid leukemia (AML).  This review is to provide a comprehensive overview of LSD1 inhibitors in clinical trials including molecular mechanistic studies, clinical efficacy, adverse drug reactions, and PD/PK studies and offer prospects in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbk3-5OqEjmWlSCSgBhwJFfW6udTcc2eZ_po8zQPsw5Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D&md5=112299f15281667341a55dea57f6c83b</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1186%2Fs13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DLSD1%252FKDM1A%2520inhibitors%2520in%2520clinical%2520trials%253A%2520Advances%2520and%2520prospects%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2019%26volume%3D12%26spage%3D129%26doi%3D10.1186%2Fs13045-019-0811-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">Advances toward LSD1 inhibitors for cancer therapy</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1227</span>â <span class="NLM_lpage">1242</span>, <span class="refDoi">Â DOI: 10.4155/fmc-2017-0068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.4155%2Ffmc-2017-0068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28722477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1227-1242&author=X.+Fuauthor=P.+Zhangauthor=B.+Yu&title=Advances+toward+LSD1+inhibitors+for+cancer+therapy&doi=10.4155%2Ffmc-2017-0068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Advances toward LSD1 inhibitors for cancer therapy</span></div><div class="casAuthors">Fu, Xiaoli; Zhang, Peng; Yu, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1227-1242</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  LSD1 has become an important biol. validated epigenetic target for cancer therapy since its identification in 2004.  LSD1 mediates many cellular signaling pathways and is involved in the initiation and development of cancers.  Aberrant overexpression of LSD1 has been obsd. in different types of cancers, and inactivation by small mols. suppresses cancer cell differentiation, proliferation, invasion and migration.  To date, a large no. of LSD1 inhibitors have been reported, RG6016, GSK-2879552, INCB059872, IMG-7289 and CC-90011 are currently undergoing clin. assessment for the treatment of acute myeloid leukemia, small-cell lung cancer, etc.  In this review, we briefly highlight recent advances of LSD1 inhibitors mainly covering the literatures from 2015 to 2017 and tentatively propose our perspectives on the design of new LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlJAzt51S4crVg90H21EOLACvtfcHk0liufWzSJRwHZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK&md5=182f10eb7a2aea7b5c13f50f84f44c20</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DAdvances%2520toward%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1227%26epage%3D1242%26doi%3D10.4155%2Ffmc-2017-0068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Magid, A. F.</span></span> <span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) inhibitors as potential treatment for different types of cancers</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1134</span>â <span class="NLM_lpage">1135</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.7b00426</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslaisLzI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1134-1135&author=A.+F.+Abdel-Magid&title=Lysine-specific+demethylase+1+%28LSD1%29+inhibitors+as+potential+treatment+for+different+types+of+cancers&doi=10.1021%2Facsmedchemlett.7b00426"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Specific Demethylase 1 (LSD1) Inhibitors as Potential Treatment for Different Types of Cancers</span></div><div class="casAuthors">Abdel-Magid, Ahmed F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1134-1135</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The invention in this patent application relates to 5-cyano indole derivs.  These compd. are lysine-specific demethylase 1(LSD1) inhibitors and maybe useful for the treatment of LSD1-mediated diseases or disorders including, but are not limited to, B cell lymphoma, acute myeloid leukemia, gastric cancer, hepatocellular carcinoma, prostate cancer, breast carcinoma, neuroblastoma, glioblastoma, nasopharyngeal carcinoma, colon cancer, gallbladder cancer, esophageal cancer, head and neck cancer, lung cancer, ovarian cancer, pancreatic cancer, endometrial carcinoma, and soft tissue sarcomas such as rhabdomyosarcoma (RMS), chondrosarcoma, osteosarcoma, Ewing's sarcoma, liver fibrosis, and sickle cell disease.  Studies have shown that LSD1 is involved in cell proliferation, epithelial-mesenchymal transition, stem cell biol., malignant transformation of cells, and cell differentiation.  Dysfunction in the activities of LSD1 is believed to be responsible for a no. of myeloproliferative and lymphoproliferative diseases such as acute myeloid leukemia (AML) and acute lymphoblastic leukemia (ALL).  Therefore, the inhibition of LSD1 provides a promising pharmacol. target to design novel treatments for cancer and other disorders that assoc. with LSD1's activity.  In particular, the need exists for novel small mols. that inhibit the activity of LSD1, such as the compds. of formula I of this patent application, which includes both irreversible and reversible inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohWPHUFQw3O7Vg90H21EOLACvtfcHk0liufWzSJRwHZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslaisLzI&md5=860645e21019b91e1f2bf6d6893be4d8</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00426%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520inhibitors%2520as%2520potential%2520treatment%2520for%2520different%2520types%2520of%2520cancers%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1134%26epage%3D1135%26doi%3D10.1021%2Facsmedchemlett.7b00426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willeit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Praschak-Rieder, N.</span></span> <span> </span><span class="NLM_article-title">Imaging the effects of genetic polymorphisms on radioligand binding in the living human brain: A review on genetic neuroreceptor imaging of monoaminergic systems in psychiatry</span>. <i>NeuroImage</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">878</span>â <span class="NLM_lpage">892</span>, <span class="refDoi">Â DOI: 10.1016/j.neuroimage.2010.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.neuroimage.2010.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20399868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWqtrvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=878-892&author=M.+Willeitauthor=N.+Praschak-Rieder&title=Imaging+the+effects+of+genetic+polymorphisms+on+radioligand+binding+in+the+living+human+brain%3A+A+review+on+genetic+neuroreceptor+imaging+of+monoaminergic+systems+in+psychiatry&doi=10.1016%2Fj.neuroimage.2010.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Imaging the effects of genetic polymorphisms on radioligand binding in the living human brain: A review on genetic neuroreceptor imaging of monoaminergic systems in psychiatry</span></div><div class="casAuthors">Willeit, Matthaeus; Praschak-Rieder, Nicole</div><div class="citationInfo"><span class="NLM_cas:title">NeuroImage</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">878-892</span>CODEN:
                <span class="NLM_cas:coden">NEIMEF</span>;
        ISSN:<span class="NLM_cas:issn">1053-8119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Imaging genetics is a research field that describes the impact of genetic risk variants on brain structure and function.  While magnetic resonance based imaging techniques are able to provide complex information on a system level, positron emission tomog. (PET) and single photon emission computer tomog. (SPECT) allow for detn. of distribution and d. of single receptor mols. in the human brain.  Major psychiatric disorders are highly heritable, and have been assocd. with a dysregulation in brain dopamine and serotonin systems.  Understanding the role of genetic polymorphisms within these neurotransmitter systems on brain phenotype is essential.  This review tries to cover the literature on the impact of gene variants implicated in psychiatric disorders on serotonin, dopamine, and MAO-A radioligand binding in living humans.  The majority of PET and SPECT studies investigated the role of polymorphisms within genes coding for the serotonin and dopamine transporters, the serotonin 1A receptor, and the dopamine D2 receptor on G protein coupled receptors or transporter proteins critically involved in serotonin or dopamine neurotransmission.  Other studies investigated the impact of variants in genes for monoamine oxidase-A (MAO-A) or brain derived neurotrophic factor on monoamine transporters, receptors, or MAO-A activity.  Two main findings in healthy subjects emerge from the current literature: one is an increased binding of the selective ligand [11C]DASB to serotonin transporters in subjects homozygous for the triallelic 5-HTTLPR LA allele.  The other one is decreased binding of the radioligand [11C]raclopride to dopamine D2 receptors in D2 Taq1 A1 allele carriers.  Other findings reported are highly interesting but require independent replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuC-oqxTXqrVg90H21EOLACvtfcHk0lhbW_Gcd7ZJHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWqtrvO&md5=7a679dc2e01de0c3b53527f88185e31c</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroimage.2010.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroimage.2010.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DWilleit%26aufirst%3DM.%26aulast%3DPraschak-Rieder%26aufirst%3DN.%26atitle%3DImaging%2520the%2520effects%2520of%2520genetic%2520polymorphisms%2520on%2520radioligand%2520binding%2520in%2520the%2520living%2520human%2520brain%253A%2520A%2520review%2520on%2520genetic%2520neuroreceptor%2520imaging%2520of%2520monoaminergic%2520systems%2520in%2520psychiatry%26jtitle%3DNeuroImage%26date%3D2010%26volume%3D53%26spage%3D878%26epage%3D892%26doi%3D10.1016%2Fj.neuroimage.2010.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">Histone H3 lysine 4 demethylation is a target of nonselective antidepressive medications</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">563</span>â <span class="NLM_lpage">567</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+lysine+4+demethylation+is+a+target+of+nonselective+antidepressive+medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 Î¼M.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0lhbW_Gcd7ZJHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520lysine%25204%2520demethylation%2520is%2520a%2520target%2520of%2520nonselective%2520antidepressive%2520medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, D. G.</span></span> <span> </span><span class="NLM_article-title">Trans-2-phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4408</span>â <span class="NLM_lpage">4416</span>, <span class="refDoi">Â DOI: 10.1021/bi0618621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0618621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4408-4416&author=D.+M.+Schmidtauthor=D.+G.+McCafferty&title=Trans-2-phenylcyclopropylamine+is+a+mechanism-based+inactivator+of+the+histone+demethylase+LSD1&doi=10.1021%2Fbi0618621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">trans-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span></div><div class="casAuthors">Schmidt, Dawn M. Z.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4408-4416</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic domain of the flavin-dependent human histone demethylase, lysine-specific demethylase 1 (LSD1), belongs to the family of amine oxidases including polyamine oxidase and monoamine oxidase (MAO).  The authors previously assessed MAO inhibitors (MAOIs) for their ability to inhibit the reaction catalyzed by LSD1, demonstrating that trans-2-phenylcyclopropylamine (2-PCPA; tranylcypromine, Parnate) was the most potent inhibitor with respect to LSD1.  Here, the authors showed that 2-PCPA was a time-dependent, mechanism-based irreversible inhibitor of LSD1 with a Ki of 242 Î¼M and a kinact of 0.0106 s-1.  2-PCPA showed limited selectivity for human MAOs vs. LSD1, with kinact/Ki values only 16-fold and 2.4-fold higher for MAO B and MAO A, resp.  Profiles of LSD1 activity and inactivation by 2-PCPA as a function of pH were consistent with a mechanism of inactivation dependent upon enzyme catalysis.  Mass spectrometry supported a role for FAD as the site of covalent modification by 2-PCPA.  These results will provide a foundation for the design of cyclopropylamine-based inhibitors that are selective for LSD1 to probe its role in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVK59U7IsBvbVg90H21EOLACvtfcHk0lhbW_Gcd7ZJHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D&md5=02925342b86c7dbfb5f1a99fa5264adc</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1021%2Fbi0618621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0618621%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DTrans-2-phenylcyclopropylamine%2520is%2520a%2520mechanism-based%2520inactivator%2520of%2520the%2520histone%2520demethylase%2520LSD1%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D4408%26epage%3D4416%26doi%3D10.1021%2Fbi0618621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culhane, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczuk, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalili, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the inhibition of the LSD1 histone demethylase by the antidepressant trans-2-phenylcyclopropylamine</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">8058</span>â <span class="NLM_lpage">8065</span>, <span class="refDoi">Â DOI: 10.1021/bi700664y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi700664y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=8058-8065&author=M.+Yangauthor=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=P.+Jaliliauthor=H.+L.+Ballauthor=M.+Machiusauthor=P.+A.+Coleauthor=H.+Yu&title=Structural+basis+for+the+inhibition+of+the+LSD1+histone+demethylase+by+the+antidepressant+trans-2-phenylcyclopropylamine&doi=10.1021%2Fbi700664y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Inhibition of the LSD1 Histone Demethylase by the Antidepressant trans-2-Phenylcyclopropylamine</span></div><div class="casAuthors">Yang, Maojun; Culhane, Jeffrey C.; Szewczuk, Lawrence M.; Jalili, Pegah; Ball, Haydn L.; Machius, Mischa; Cole, Philip A.; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8058-8065</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone modifications, such as acetylation and methylation, are important epigenetic marks that regulate diverse biol. processes that use chromatin as the template, including transcription.  Dysregulation of histone acetylation and methylation leads to the silencing of tumor suppressor genes and contributes to cancer progression.  Inhibitors of enzymes that catalyze the addn. and removal of these epigenetic marks thus have therapeutic potential for treating cancer.  Lysine-specific demethylase 1 (LSD1) is the first discovered histone lysine demethylase and with the help of its cofactor CoREST, specifically demethylates mono- and dimethylated histone H3 lysine 4 (H3-K4), thus repressing transcription.  Because LSD1 belongs to the family of FAD (FAD)-dependent amine oxidases, certain inhibitors of monoamine oxidases (MAOs), including the clin. used antidepressant trans-2-phenylcyclopropylamine (PCPA; tranylcypromine; Parnate), are also capable of inhibiting LSD1.  In this study, we have further measured the kinetic parameters of the inhibition of LSD1 by PCPA and detd. the crystal structure of LSD1-CoREST in the presence of PCPA.  Our structural and mass spectrometry analyses are consistent with PCPA forming a covalent adduct with FAD in LSD1 that is distinct from the FAD-PCPA adduct of MAO B.  The structure also reveals that the Ph ring of the FAD-PCPA adduct in LSD1 does not form extensive interactions with active-site residues.  This study thus provides the basis for designing more potent inhibitors of LSD1 that contain substitutions on the Ph ring of PCPA to fully engage neighboring residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqcLmzpB1S7Vg90H21EOLACvtfcHk0lhbW_Gcd7ZJHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D&md5=e11fe93832d46962b6c4848f682042d2</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1021%2Fbi700664y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi700664y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DJalili%26aufirst%3DP.%26aulast%3DBall%26aufirst%3DH.%2BL.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520the%2520LSD1%2520histone%2520demethylase%2520by%2520the%2520antidepressant%2520trans-2-phenylcyclopropylamine%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D8058%26epage%3D8065%26doi%3D10.1021%2Fbi700664y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span> <span> </span><span class="NLM_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">340</span>â <span class="NLM_lpage">352</span>, <span class="refDoi">Â DOI: 10.1080/15592294.2017.1290032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F15592294.2017.1290032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28277979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=340-352&author=H.+Niwaauthor=T.+Umehara&title=Structural+insight+into+inhibitors+of+flavin+adenine+dinucleotide-dependent+lysine+demethylases&doi=10.1080%2F15592294.2017.1290032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span></div><div class="casAuthors">Niwa Hideaki; Umehara Takashi; Umehara Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-352</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Until 2004, many researchers believed that protein methylation in eukaryotic cells was an irreversible reaction.  However, the discovery of lysine-specific demethylase 1 in 2004 drastically changed this view and the concept of chromatin regulation.  Since then, the enzymes responsible for lysine demethylation and their cellular substrates, biological significance, and selective regulation have become major research topics in epigenetics and chromatin biology.  Many cell-permeable inhibitors for lysine demethylases have been developed, including both target-specific and nonspecific inhibitors.  Structural understanding of how these inhibitors bind to lysine demethylases is crucial both for validation of the inhibitors as chemical probes and for the rational design of more potent, target-specific inhibitors.  This review focuses on published small-molecule inhibitors targeted at the two flavin adenine dinucleotide-dependent lysine demethylases, lysine-specific demethylases 1 and 2, and how the inhibitors interact with the tertiary structures of the enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyKG85Hfra9xjv2lLSLOxFfW6udTcc2eY5ohHbusq2ybntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D&md5=31f93c2cc7909c4197f5892f7f04af9d</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1080%2F15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DUmehara%26aufirst%3DT.%26atitle%3DStructural%2520insight%2520into%2520inhibitors%2520of%2520flavin%2520adenine%2520dinucleotide-dependent%2520lysine%2520demethylases%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D340%26epage%3D352%26doi%3D10.1080%2F15592294.2017.1290032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MascarÃ³, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirapu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lufino, M. M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willekens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valls, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fyfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro-Palomino, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-1001, a potent and selective covalent KDM1A inhibitor, for the treatment of acute leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">495</span>â <span class="NLM_lpage">511</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29502954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=495-511&author=T.+Maesauthor=C.+Mascar%C3%B3author=I.+Tirapuauthor=A.+Estiarteauthor=F.+Ciceriauthor=S.+Lunardiauthor=N.+Guibourtauthor=A.+Perdonesauthor=M.+M.+P.+Lufinoauthor=T.+C.+P.+Somervailleauthor=D.+H.+Wisemanauthor=C.+Duyauthor=A.+Melnickauthor=C.+Willekensauthor=A.+Ortegaauthor=M.+Martinellauthor=N.+Vallsauthor=G.+Kurzauthor=M.+Fyfeauthor=J.+C.+Castro-Palominoauthor=C.+Buesa&title=ORY-1001%2C+a+potent+and+selective+covalent+KDM1A+inhibitor%2C+for+the+treatment+of+acute+leukemia&doi=10.1016%2Fj.ccell.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">ORY-1001, a Potent and Selective Covalent KDM1A Inhibitor, for the Treatment of Acute Leukemia</span></div><div class="casAuthors">Maes, Tamara; Mascaro, Cristina; Tirapu, Inigo; Estiarte, Angels; Ciceri, Filippo; Lunardi, Serena; Guibourt, Nathalie; Perdones, Alvaro; Lufino, Michele M. P.; Somervaille, Tim C. P.; Wiseman, Dan H.; Duy, Cihangir; Melnick, Ari; Willekens, Christophe; Ortega, Alberto; Martinell, Marc; Valls, Nuria; Kurz, Guido; Fyfe, Matthew; Castro-Palomino, Julio Cesar; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-511.e12</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lysine-specific demethylase KDM1A is a key regulator of stem cell potential in acute myeloid leukemia (AML).  ORY-1001 is a highly potent and selective KDM1A inhibitor that induces H3K4me2 accumulation on KDM1A target genes, blast differentiation, and redn. of leukemic stem cell capacity in AML.  ORY-1001 exhibits potent synergy with std.-of-care drugs and selective epigenetic inhibitors, reduces growth of an AML xenograft model, and extends survival in a mouse PDX (patient-derived xenograft) model of T cell acute leukemia.  Surrogate pharmacodynamic biomarkers developed based on expression changes in leukemia cell lines were translated to samples from patients treated with ORY-1001.  ORY-1001 is a selective KDM1A inhibitor in clin. trials and is currently being evaluated in patients with leukemia and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2QFk3s8-eLVg90H21EOLACvtfcHk0lhkpWRWJDjSJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D&md5=68f69ade1b0110f3e0adf0173326e9b7</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascar%25C3%25B3%26aufirst%3DC.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DLunardi%26aufirst%3DS.%26aulast%3DGuibourt%26aufirst%3DN.%26aulast%3DPerdones%26aufirst%3DA.%26aulast%3DLufino%26aufirst%3DM.%2BM.%2BP.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DWillekens%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMartinell%26aufirst%3DM.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DKurz%26aufirst%3DG.%26aulast%3DFyfe%26aufirst%3DM.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%2BC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-1001%252C%2520a%2520potent%2520and%2520selective%2520covalent%2520KDM1A%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520acute%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D495%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degenhardt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDevitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crouthamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span> <span> </span><span class="NLM_article-title">A DNA hypomethylation signature predicts antitumor activity of LSD1 inhibitors in SCLC</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">57</span>â <span class="NLM_lpage">69</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+hypomethylation+signature+predicts+antitumor+activity+of+LSD1+inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lgpg2G80ZQeXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520hypomethylation%2520signature%2520predicts%2520antitumor%2520activity%2520of%2520LSD1%2520inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MascarÃ³, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotllant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GriÃ±an-FerrÃ©, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PallÃ s, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span> <span> </span><span class="NLM_article-title">The dual LSD1/MAOB inhibitor ORY 2001 prevents the development of the memory deficit in samp8 mice through induction of neuronal plasticity and reduction of neuroinflammation</span>. <i>Alzheimer's Dementia</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">P905</span>, <span class="refDoi">Â DOI: 10.1016/j.jalz.2015.08.146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.jalz.2015.08.146" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=P905&author=C.+Buesaauthor=C.+Mascar%C3%B3author=D.+Rotllantauthor=C.+Gri%C3%B1an-Ferr%C3%A9author=M.+Pall%C3%A0sauthor=T.+Maes&title=The+dual+LSD1%2FMAOB+inhibitor+ORY+2001+prevents+the+development+of+the+memory+deficit+in+samp8+mice+through+induction+of+neuronal+plasticity+and+reduction+of+neuroinflammation&doi=10.1016%2Fj.jalz.2015.08.146"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1016%2Fj.jalz.2015.08.146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jalz.2015.08.146%26sid%3Dliteratum%253Aachs%26aulast%3DBuesa%26aufirst%3DC.%26aulast%3DMascar%25C3%25B3%26aufirst%3DC.%26aulast%3DRotllant%26aufirst%3DD.%26aulast%3DGri%25C3%25B1an-Ferr%25C3%25A9%26aufirst%3DC.%26aulast%3DPall%25C3%25A0s%26aufirst%3DM.%26aulast%3DMaes%26aufirst%3DT.%26atitle%3DThe%2520dual%2520LSD1%252FMAOB%2520inhibitor%2520ORY%25202001%2520prevents%2520the%2520development%2520of%2520the%2520memory%2520deficit%2520in%2520samp8%2520mice%2520through%2520induction%2520of%2520neuronal%2520plasticity%2520and%2520reduction%2520of%2520neuroinflammation%26jtitle%3DAlzheimer%2527s%2520Dementia%26date%3D2015%26volume%3D11%26spage%3DP905%26doi%3D10.1016%2Fj.jalz.2015.08.146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MascarÃ³, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotllant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalcanti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carceller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-2001: An epigenetic drug for the treatment of cognition defects in alzheimerâs disease and other neurodegenerative disorders</span>. <i>Alzheimerâs Dementia</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">P1192</span>, <span class="refDoi">Â DOI: 10.1016/j.jalz.2016.07.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.jalz.2016.07.149" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=P1192&author=T.+Maesauthor=C.+Mascar%C3%B3author=D.+Rotllantauthor=F.+Cavalcantiauthor=E.+Carcellerauthor=A.+Ortegaauthor=C.+Molineroauthor=C.+Buesa&title=ORY-2001%3A+An+epigenetic+drug+for+the+treatment+of+cognition+defects+in+alzheimer%E2%80%99s+disease+and+other+neurodegenerative+disorders&doi=10.1016%2Fj.jalz.2016.07.149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1016%2Fj.jalz.2016.07.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jalz.2016.07.149%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascar%25C3%25B3%26aufirst%3DC.%26aulast%3DRotllant%26aufirst%3DD.%26aulast%3DCavalcanti%26aufirst%3DF.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMolinero%26aufirst%3DC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-2001%253A%2520An%2520epigenetic%2520drug%2520for%2520the%2520treatment%2520of%2520cognition%2520defects%2520in%2520alzheimer%25E2%2580%2599s%2520disease%2520and%2520other%2520neurodegenerative%2520disorders%26jtitle%3DAlzheimer%25E2%2580%2599s%2520Dementia%26date%3D2016%26volume%3D12%26spage%3DP1192%26doi%3D10.1016%2Fj.jalz.2016.07.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, O. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">4364</span>â <span class="NLM_lpage">4397</span>, <span class="refDoi">Â DOI: 10.1039/c0cs00203h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-4397&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases&doi=10.1039%2Fc0cs00203h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0lgpg2G80ZQeXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D4397%26doi%3D10.1039%2Fc0cs00203h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoeller, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of a linked jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">2631</span>â <span class="NLM_lpage">2646</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M115.698449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.M115.698449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26645689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=2631-2646&author=J.+R.+Hortonauthor=A.+Engstromauthor=E.+L.+Zoellerauthor=X.+Liuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=M.+A.+Johnsauthor=P.+M.+Vertinoauthor=H.+Fuauthor=X.+Cheng&title=Characterization+of+a+linked+jumonji+domain+of+the+KDM5%2FJARID1+family+of+histone+H3+lysine+4+demethylases&doi=10.1074%2Fjbc.M115.698449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a Linked Jumonji Domain of the KDM5/JARID1 Family of Histone H3 Lysine 4 Demethylases</span></div><div class="casAuthors">Horton, John R.; Engstrom, Amanda; Zoeller, Elizabeth L.; Liu, Xu; Shanks, John R.; Zhang, Xing; Johns, Margaret A.; Vertino, Paula M.; Fu, Haian; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2631-2646</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and Î±-ketoglutarate-dependent demethylases remove Me groups from tri- and dimethylated lysine 4 of histone H3.  Accumulating evidence from primary tumors and model systems supports a role for KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) as oncogenic drivers.  The KDM5 family is unique among the Jumonji domain-contg. histone demethylases in that there is an atypical insertion of a DNA-binding ARID domain and a histone-binding PHD domain into the Jumonji domain, which separates the catalytic domain into two fragments (JmjN and JmjC).  Here we demonstrate that internal deletion of the ARID and PHD1 domains has a negligible effect on in vitro enzymic kinetics of the KDM5 family of enzymes.  We present a crystal structure of the linked JmjN-JmjC domain from KDM5A, which reveals that the linked domain fully reconstitutes the cofactor (metal ion and Î±-ketoglutarate) binding characteristics of other structurally characterized Jumonji domain demethylases.  Docking studies with GSK-J1, a selective inhibitor of the KDM6/KDM5 subfamilies, identify crit. residues for binding of the inhibitor to the reconstituted KDM5 Jumonji domain.  Further, we found that GSK-J1 inhibited the demethylase activity of KDM5C with 8.5-fold increased potency compared with that of KDM5B at 1 mm Î±-ketoglutarate.  In contrast, JIB-04 (a pan-inhibitor of the Jumonji demethylase superfamily) had the opposite effect and was â¼8-fold more potent against KDM5B than against KDM5C.  Interestingly, the relative selectivity of JIB-04 toward KDM5B over KDM5C in vitro translates to a â¼10-50-fold greater growth-inhibitory activity against breast cancer cell lines.  These data define the minimal requirements for enzymic activity of the KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical zinc-binding domain and provide structural characterization of the linked JmjN-JmjC domain for the KDM5 family, which should prove useful in the design of KDM5 demethylase inhibitors with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa4qqOk0zHF7Vg90H21EOLACvtfcHk0lhhpwK0AYO2Ng"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D&md5=772a106c3c7f0a45c1b3d2ee81cc9487</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.698449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.698449%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DEngstrom%26aufirst%3DA.%26aulast%3DZoeller%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520linked%2520jumonji%2520domain%2520of%2520the%2520KDM5%252FJARID1%2520family%2520of%2520histone%2520H3%2520lysine%25204%2520demethylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D2631%26epage%3D2646%26doi%3D10.1074%2Fjbc.M115.698449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staller, P.</span></span> <span> </span><span class="NLM_article-title">The KDM5 family of histone demethylases as targets in oncology drug discovery</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>â <span class="NLM_lpage">286</span>, <span class="refDoi">Â DOI: 10.2217/epi.14.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2217%2Fepi.14.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25111482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=277-286&author=P.+B.+Rasmussenauthor=P.+Staller&title=The+KDM5+family+of+histone+demethylases+as+targets+in+oncology+drug+discovery&doi=10.2217%2Fepi.14.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">The KDM5 family of histone demethylases as targets in oncology drug discovery</span></div><div class="casAuthors">Rasmussen, Peter Birk; Staller, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-286</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing evidence for a causal role of the KDM5 family of histone demethylases in human cancer.  In particular, KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) contribute to cancer cell proliferation, reduce the expression of tumor suppressor genes, promote the development of drug tolerance and maintain tumor-initiating cells.  KDM5 enzymes remove tri- and di-methylations of lysine 4 of histone H3 - modifications that occur at the start site of transcription in actively transcribed genes.  However, the importance of the histone demethylase activity of KDM5 proteins for cancer cells has not been resolved so far.  The currently available approaches suppress or remove the targeted proteins and thereby affect their putative functions as structural components and recruitment factors for other chromatin-assocd. proteins.  Therefore, the development of specific enzymic inhibitors for KDM5 will promote our understanding of the biol. role of their catalytic activity and yield potential novel anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPJyGCVkwViLVg90H21EOLACvtfcHk0ljYTbUpkzLT4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K&md5=0b8dc67f4fde6ce24457dce092dbdd83</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.14%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DStaller%26aufirst%3DP.%26atitle%3DThe%2520KDM5%2520family%2520of%2520histone%2520demethylases%2520as%2520targets%2520in%2520oncology%2520drug%2520discovery%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D277%26epage%3D286%26doi%3D10.2217%2Fepi.14.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1383</span>â <span class="NLM_lpage">1404</span>, <span class="refDoi">Â DOI: 10.3390/cancers3011383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.3390%2Fcancers3011383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21544224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1383-1404&author=L.+P.+Blairauthor=J.+Caoauthor=M.+R.+Zouauthor=J.+Sayeghauthor=Q.+Yan&title=Epigenetic+regulation+by+lysine+demethylase+5+%28KDM5%29+enzymes+in+cancer&doi=10.3390%2Fcancers3011383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span></div><div class="casAuthors">Blair, Lauren P.; Cao, Jian; Zou, Mike Ran; Sayegh, Joyce; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1383-1404</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Similar to genetic alterations, epigenetic aberrations contribute significantly to tumor initiation and progression.  In many cases, these changes are caused by activation or inactivation of the regulators that maintain epigenetic states.  Here we review our current knowledge on the KDM5/JARID1 family of histone demethylases.  This family of enzymes contains a JmjC domain and is capable of removing tri- and di- Me marks from lysine 4 on histone H3.  Among these proteins, RBP2 mediates drug resistance while JARID1B is required for melanoma maintenance.  Preclin. studies suggest inhibition of these enzymes can suppress tumorigenesis and provide strong rationale for development of their inhibitors for use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8lTt6GjrOJrVg90H21EOLACvtfcHk0ljYTbUpkzLT4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D&md5=426fa61d08aa818b45781b57e1aba5f3</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.3390%2Fcancers3011383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3011383%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DEpigenetic%2520regulation%2520by%2520lysine%2520demethylase%25205%2520%2528KDM5%2529%2520enzymes%2520in%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D1383%26epage%3D1404%26doi%3D10.3390%2Fcancers3011383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shokri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fathi-Roudsari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouhkan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanati, M. H.</span></span> <span> </span><span class="NLM_article-title">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">105</span>â <span class="NLM_lpage">111</span>, <span class="refDoi">Â DOI: 10.1016/j.leukres.2018.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.leukres.2018.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29602065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=105-111&author=G.+Shokriauthor=S.+Doudiauthor=M.+Fathi-Roudsariauthor=F.+Kouhkanauthor=M.+H.+Sanati&title=Targeting+histone+demethylases+KDM5A+and+KDM5B+in+AML+cancer+cells%3A+A+comparative+view&doi=10.1016%2Fj.leukres.2018.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span></div><div class="casAuthors">Shokri, Gelareh; Doudi, Shaghayegh; Fathi-Roudsari, Mehrnoosh; Kouhkan, Fatemeh; Sanati, Mohammad-Hossein</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-111</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Epigenetic modifications play an important role in initiation and progression of cancers including acute myeloid leukemia.  Among different epigenetic modifiers, lysine specific demethylases have been noticed as potential therapeutic targets.  KDM5 family of histone demethylases which removes Me marks from lysine residues of H3, are frequently found in the promoter region of transcriptionally active genes resulting in repression of expression.  Here we have compared the effects of KDM5A and KDM5B downregulation on HL-60 cell line behavior.  KDM5A/5B knockdown resulted in lower viability of HL-60 cells in addn. to modified cell cycle distribution and sub-G1 accumulation.  Induction of apoptosis was obsd. in both knockdown cells.  But in spite of similarity in their role, downregulation of KDM5A showed more efficient anti-leukemic effects in comparison to KDM5B.  Cells showed higher accumulation in sub-G1 and apoptosis occurred significantly higher and also earlier after KDM5A redn.  Expression anal. confirmed almost 5 and 4 fold increased expression for bax and caspase-3 after downregulation of KDM5A in comparison to KDM5B.  Due to the present study we propose KDM5A as a potential target for therapeutic aspects of acute myeloid leukemia although further investigations are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIUHbiIE13rVg90H21EOLACvtfcHk0ljYTbUpkzLT4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D&md5=22a34070de67e600b8164273c1112405</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2018.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2018.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DShokri%26aufirst%3DG.%26aulast%3DDoudi%26aufirst%3DS.%26aulast%3DFathi-Roudsari%26aufirst%3DM.%26aulast%3DKouhkan%26aufirst%3DF.%26aulast%3DSanati%26aufirst%3DM.%2BH.%26atitle%3DTargeting%2520histone%2520demethylases%2520KDM5A%2520and%2520KDM5B%2520in%2520AML%2520cancer%2520cells%253A%2520A%2520comparative%2520view%26jtitle%3DLeuk.%2520Res.%26date%3D2018%26volume%3D68%26spage%3D105%26epage%3D111%26doi%3D10.1016%2Fj.leukres.2018.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalgliesh, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bignell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edkins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barthorpe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beare, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kok, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddison, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLay, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mironenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulderrig, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleasance, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajasingham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stebbings, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarpey, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykema, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoo, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petillo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wondergem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anema, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahnoski, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teh, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, P. A.</span></span> <span> </span><span class="NLM_article-title">Systematic sequencing of renal carcinoma reveals inactivation of histone modifying genes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">360</span>â <span class="NLM_lpage">363</span>, <span class="refDoi">Â DOI: 10.1038/nature08672</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature08672" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20054297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFeqtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=360-363&author=G.+L.+Dalglieshauthor=K.+Furgeauthor=C.+Greenmanauthor=L.+Chenauthor=G.+Bignellauthor=A.+Butlerauthor=H.+Daviesauthor=S.+Edkinsauthor=C.+Hardyauthor=C.+Latimerauthor=J.+Teagueauthor=J.+Andrewsauthor=S.+Barthorpeauthor=D.+Beareauthor=G.+Buckauthor=P.+J.+Campbellauthor=S.+Forbesauthor=M.+Jiaauthor=D.+Jonesauthor=H.+Knottauthor=C.+Y.+Kokauthor=K.+W.+Lauauthor=C.+Leroyauthor=M.+L.+Linauthor=D.+J.+McBrideauthor=M.+Maddisonauthor=S.+Maguireauthor=K.+McLayauthor=A.+Menziesauthor=T.+Mironenkoauthor=L.+Mulderrigauthor=L.+Mudieauthor=S.+O%E2%80%99Mearaauthor=E.+Pleasanceauthor=A.+Rajasinghamauthor=R.+Shepherdauthor=R.+Smithauthor=L.+Stebbingsauthor=P.+Stephensauthor=G.+Tangauthor=P.+S.+Tarpeyauthor=K.+Turrellauthor=K.+J.+Dykemaauthor=S.+K.+Khooauthor=D.+Petilloauthor=B.+Wondergemauthor=J.+Anemaauthor=R.+J.+Kahnoskiauthor=B.+T.+Tehauthor=M.+R.+Strattonauthor=P.+A.+Futreal&title=Systematic+sequencing+of+renal+carcinoma+reveals+inactivation+of+histone+modifying+genes&doi=10.1038%2Fnature08672"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic sequencing of renal carcinoma reveals inactivation of histone modifying genes</span></div><div class="casAuthors">Dalgliesh, Gillian L.; Furge, Kyle; Greenman, Chris; Chen, Lina; Bignell, Graham; Butler, Adam; Davies, Helen; Edkins, Sarah; Hardy, Claire; Latimer, Calli; Teague, Jon; Andrews, Jenny; Barthorpe, Syd; Beare, Dave; Buck, Gemma; Campbell, Peter J.; Forbes, Simon; Jia, Mingming; Jones, David; Knott, Henry; Kok, Chai Yin; Lau, King Wai; Leroy, Catherine; Lin, Meng-Lay; McBride, David J.; Maddison, Mark; Maguire, Simon; McLay, Kirsten; Menzies, Andrew; Mironenko, Tatiana; Mulderrig, Lee; Mudie, Laura; O'Meara, Sarah; Pleasance, Erin; Rajasingham, Arjunan; Shepherd, Rebecca; Smith, Raffaella; Stebbings, Lucy; Stephens, Philip; Tang, Gurpreet; Tarpey, Patrick S.; Turrell, Kelly; Dykema, Karl J.; Khoo, Sok Kean; Petillo, David; Wondergem, Bill; Anema, John; Kahnoski, Richard J.; Teh, Bin Tean; Stratton, Michael R.; Futreal, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7279</span>),
    <span class="NLM_cas:pages">360-363</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma (ccRCC) is the most common form of adult kidney cancer, characterized by the presence of inactivating mutations in the VHL gene in most cases, and by infrequent somatic mutations in known cancer genes.  To det. further the genetics of ccRCC, we have sequenced 101 cases through 3544 protein-coding genes.  Here we report the identification of inactivating mutations in two genes encoding enzymes involved in histone modification-SETD2, a histone H3 lysine 36 methyltransferase, and JARID1C (also known as KDM5C), a histone H3 lysine 4 demethylase-as well as mutations in the histone H3 lysine 27 demethylase, UTX (KMD6A), that we recently reported.  The results highlight the role of mutations in components of the chromatin modification machinery in human cancer.  Furthermore, NF2 mutations were found in non-VHL mutated ccRCC, and several other probable cancer genes were identified.  These results indicate that substantial genetic heterogeneity exists in a cancer type dominated by mutations in a single gene, and that systematic screens will be key to fully detg. the somatic genetic architecture of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuHtA6CxWXLVg90H21EOLACvtfcHk0liqZ104wpLyKA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFeqtw%253D%253D&md5=e08e7870bb8a78f1cf33b026d2d9c899</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1038%2Fnature08672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08672%26sid%3Dliteratum%253Aachs%26aulast%3DDalgliesh%26aufirst%3DG.%2BL.%26aulast%3DFurge%26aufirst%3DK.%26aulast%3DGreenman%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DEdkins%26aufirst%3DS.%26aulast%3DHardy%26aufirst%3DC.%26aulast%3DLatimer%26aufirst%3DC.%26aulast%3DTeague%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DJ.%26aulast%3DBarthorpe%26aufirst%3DS.%26aulast%3DBeare%26aufirst%3DD.%26aulast%3DBuck%26aufirst%3DG.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DForbes%26aufirst%3DS.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DKnott%26aufirst%3DH.%26aulast%3DKok%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DLeroy%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DM.%2BL.%26aulast%3DMcBride%26aufirst%3DD.%2BJ.%26aulast%3DMaddison%26aufirst%3DM.%26aulast%3DMaguire%26aufirst%3DS.%26aulast%3DMcLay%26aufirst%3DK.%26aulast%3DMenzies%26aufirst%3DA.%26aulast%3DMironenko%26aufirst%3DT.%26aulast%3DMulderrig%26aufirst%3DL.%26aulast%3DMudie%26aufirst%3DL.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DS.%26aulast%3DPleasance%26aufirst%3DE.%26aulast%3DRajasingham%26aufirst%3DA.%26aulast%3DShepherd%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DStebbings%26aufirst%3DL.%26aulast%3DStephens%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DG.%26aulast%3DTarpey%26aufirst%3DP.%2BS.%26aulast%3DTurrell%26aufirst%3DK.%26aulast%3DDykema%26aufirst%3DK.%2BJ.%26aulast%3DKhoo%26aufirst%3DS.%2BK.%26aulast%3DPetillo%26aufirst%3DD.%26aulast%3DWondergem%26aufirst%3DB.%26aulast%3DAnema%26aufirst%3DJ.%26aulast%3DKahnoski%26aufirst%3DR.%2BJ.%26aulast%3DTeh%26aufirst%3DB.%2BT.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26atitle%3DSystematic%2520sequencing%2520of%2520renal%2520carcinoma%2520reveals%2520inactivation%2520of%2520histone%2520modifying%2520genes%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D360%26epage%3D363%26doi%3D10.1038%2Fnature08672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinohara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantoff, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, C. J.</span></span> <span> </span><span class="NLM_article-title">Resistance to docetaxel in prostate cancer is associated with androgen receptor activation and loss of KDM5D expression</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">6259</span>â <span class="NLM_lpage">6264</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1600420113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1073%2Fpnas.1600420113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27185910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFeitrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=6259-6264&author=K.+Komuraauthor=S.+H.+Jeongauthor=K.+Hinoharaauthor=F.+Quauthor=X.+Wangauthor=M.+Hirakiauthor=H.+Azumaauthor=G.+S.+Leeauthor=P.+W.+Kantoffauthor=C.+J.+Sweeney&title=Resistance+to+docetaxel+in+prostate+cancer+is+associated+with+androgen+receptor+activation+and+loss+of+KDM5D+expression&doi=10.1073%2Fpnas.1600420113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to docetaxel in prostate cancer is associated with androgen receptor activation and loss of KDM5D expression</span></div><div class="casAuthors">Komura, Kazumasa; Jeong, Seong Ho; Hinohara, Kunihiko; Qu, Fangfang; Wang, Xiaodong; Hiraki, Masayuki; Azuma, Haruhito; Lee, Gwo-Shu Mary; Kantoff, Philip W.; Sweeney, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6259-6264</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The androgen receptor (AR) plays an essential role in prostate cancer, and suppression of its signaling with androgen deprivation therapy (ADT) has been the mainstay of treatment for metastatic hormone-sensitive prostate cancer for more than 70 y.  Chemotherapy has been reserved for metastatic castration-resistant prostate cancer (mCRPC).  The Eastern Cooperative Oncol. Group-led trial E3805: ChemoHormonal Therapy Vs. Androgen Ablation Randomized Trial for Extensive Disease in Prostate Cancer (CHAARTED) showed that the addn. of docetaxel to ADT prolonged overall survival compared with ADT alone in patients with metastatic hormone-sensitive prostate cancer.  This finding suggests that there is an interaction between AR signaling activity and docetaxel sensitivity.  Here we demonstrate that the prostate cancer cell lines LNCaP and LAPC4 display markedly different sensitivity to docetaxel with AR activation, and RNA-seq anal. of these cell lines identified KDM5D (lysine-specific demethylase 5D) encoded on the Y chromosome as a potential mediator of this sensitivity.  Knocking down KDM5D expression in LNCaP leads to docetaxel resistance in the presence of dihydrotestosterone.  KDM5D phys. interacts with AR in the nucleus, and regulates its transcriptional activity by demethylating H3K4me3 active transcriptional marks.  Attenuating KDM5D expression dysregulates AR signaling, resulting in docetaxel insensitivity.  KDM5D deletion was also obsd. in the LNCaP-derived CRPC cell line 104R2, which displayed docetaxel insensitivity with AR activation, unlike parental LNCaP.  Dataset anal. from the Oncomine database revealed significantly decreased KDM5D expression in CRPC and poorer prognosis with low KDM5D expression.  Taking these data together, this work indicates that KDM5D modulates the AR axis and that this is assocd. with altered docetaxel sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLVeWTWOWNyrVg90H21EOLACvtfcHk0liqZ104wpLyKA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFeitrw%253D&md5=9ca5ed64490276833a0f9aeb6a276d3f</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1600420113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1600420113%26sid%3Dliteratum%253Aachs%26aulast%3DKomura%26aufirst%3DK.%26aulast%3DJeong%26aufirst%3DS.%2BH.%26aulast%3DHinohara%26aufirst%3DK.%26aulast%3DQu%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHiraki%26aufirst%3DM.%26aulast%3DAzuma%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DG.%2BS.%26aulast%3DKantoff%26aufirst%3DP.%2BW.%26aulast%3DSweeney%26aufirst%3DC.%2BJ.%26atitle%3DResistance%2520to%2520docetaxel%2520in%2520prostate%2520cancer%2520is%2520associated%2520with%2520androgen%2520receptor%2520activation%2520and%2520loss%2520of%2520KDM5D%2520expression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D6259%26epage%3D6264%26doi%3D10.1073%2Fpnas.1600420113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harmeyer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facompre, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, D.</span></span> <span> </span><span class="NLM_article-title">JARID1 histone demethylases: Emerging targets in cancer</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">713</span>â <span class="NLM_lpage">725</span>, <span class="refDoi">Â DOI: 10.1016/j.trecan.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.trecan.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28958389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=713-725&author=K.+M.+Harmeyerauthor=N.+D.+Facompreauthor=M.+Herlynauthor=D.+Basu&title=JARID1+histone+demethylases%3A+Emerging+targets+in+cancer&doi=10.1016%2Fj.trecan.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">JARID1 Histone Demethylases: Emerging Targets in Cancer</span></div><div class="casAuthors">Harmeyer, Kayla M.; Facompre, Nicole D.; Herlyn, Meenhard; Basu, Devraj</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">713-725</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  JARID1 proteins are histone demethylases that both regulate normal cell fates during development and contribute to the epigenetic plasticity that underlies malignant transformation.  This H3K4 demethylase family participates in multiple repressive transcriptional complexes at promoters and has broader regulatory effects on chromatin that remain ill-defined.  There is growing understanding of the oncogenic and tumor suppressive functions of JARID1 proteins, which are contingent on cell context and the protein isoform.  Their contributions to stem cell-like dedifferentiation, tumor aggressiveness, and therapy resistance in cancer have sustained interest in the development of JARID1 inhibitors.  Here we review the diverse and context-specific functions of the JARID1 proteins that may impact the utilization of emerging targeted inhibitors of this histone demethylase family in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYLbdIKcjybVg90H21EOLACvtfcHk0lhqwjWZn0cI_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D&md5=e728e47a8d60d5e83f29ab6d153a2be4</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DHarmeyer%26aufirst%3DK.%2BM.%26aulast%3DFacompre%26aufirst%3DN.%2BD.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBasu%26aufirst%3DD.%26atitle%3DJARID1%2520histone%2520demethylases%253A%2520Emerging%2520targets%2520in%2520cancer%26jtitle%3DTrends%2520Cancer%26date%3D2017%26volume%3D3%26spage%3D713%26epage%3D725%26doi%3D10.1016%2Fj.trecan.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Lysine demethylase 5B (KDM5B): A potential anti-cancer drug target</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">131</span>â <span class="NLM_lpage">140</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2018.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30343192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvF2mu7fL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2019&pages=131-140&author=Y.+C.+Zhengauthor=J.+Changauthor=L.+C.+Wangauthor=H.+M.+Renauthor=J.+R.+Pangauthor=H.+M.+Liu&title=Lysine+demethylase+5B+%28KDM5B%29%3A+A+potential+anti-cancer+drug+target&doi=10.1016%2Fj.ejmech.2018.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine demethylase 5B (KDM5B): A potential anti-cancer drug target</span></div><div class="casAuthors">Zheng, Yi-Chao; Chang, Jiao; Wang, Lin-Chen; Ren, Hong-Mei; Pang, Jing-Ru; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">131-140</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine demethylase 5B (KDM5B) is a histone demethylase identified in 2007, which is responsible for erasing H3K4me2/3 activation marker.  It participates in multiple repressive transcriptional complexes around target gene promoters and performs wide regulatory effects on chromatin structure.  Until now, there is growing evidence for the oncogenic function of KDM5B.  As the H3K4me2/3 residue represents the transcription initiation site of the active transcription gene, and demethylation of H3K4 is assocd. with transcriptional repression, making it a potential participant in inhibiting the expression of tumor suppressors.  Therefore, KDM5B is considered as a promising drug target for cancer therapy, and many medicinal chemists are trying to design and synthesize potent and selective KDM5B inhibitors with the aid of high-throughput screening, structure based drug design, and structure activity relationship studies.  This review focuses on the basic biochem. and physiol. function of KDM5B and its involved mechanisms in cancers, a comprehensive overview of KDM5B inhibitors is also introduced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL9Qm25PJ8e7Vg90H21EOLACvtfcHk0lhqwjWZn0cI_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvF2mu7fL&md5=881eb589a48c5b00c67ae58a67c0d0f5</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%2BC.%26aulast%3DRen%26aufirst%3DH.%2BM.%26aulast%3DPang%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DLysine%2520demethylase%25205B%2520%2528KDM5B%2529%253A%2520A%2520potential%2520anti-cancer%2520drug%2520target%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D161%26spage%3D131%26epage%3D140%26doi%3D10.1016%2Fj.ejmech.2018.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tindell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guler, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">531</span>â <span class="NLM_lpage">538</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnchembio.2085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27214401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=531-538&author=M.+Vinogradovaauthor=V.+S.+Gehlingauthor=A.+Gustafsonauthor=S.+Aroraauthor=C.+A.+Tindellauthor=C.+Wilsonauthor=K.+E.+Williamsonauthor=G.+D.+Gulerauthor=P.+Gangurdeauthor=W.+Manieriauthor=J.+Busbyauthor=E.+M.+Flynnauthor=F.+Lanauthor=H.+J.+Kimauthor=S.+Odateauthor=A.+G.+Cochranauthor=Y.+Liuauthor=M.+Wongchenkoauthor=Y.+Yangauthor=T.+K.+Cheungauthor=T.+M.+Maileauthor=T.+Lauauthor=M.+Costaauthor=G.+V.+Hegdeauthor=E.+Jacksonauthor=R.+Pittiauthor=D.+Arnottauthor=C.+Baileyauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=J.+R.+Kieferauthor=P.+Trojerauthor=M.+Classon&title=An+inhibitor+of+KDM5+demethylases+reduces+survival+of+drug-tolerant+cancer+cells&doi=10.1038%2Fnchembio.2085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span></div><div class="casAuthors">Vinogradova, Maia; Gehling, Victor S.; Gustafson, Amy; Arora, Shilpi; Tindell, Charles A.; Wilson, Catherine; Williamson, Kaylyn E.; Guler, Gulfem D.; Gangurde, Pranoti; Manieri, Wanda; Busby, Jennifer; Flynn, E. Megan; Lan, Fei; Kim, Hyo-jin; Odate, Shobu; Cochran, Andrea G.; Liu, Yichin; Wongchenko, Matthew; Yang, Yibin; Cheung, Tommy K.; Maile, Tobias M.; Lau, Ted; Costa, Michael; Hegde, Ganapati V.; Jackson, Erica; Pitti, Robert; Arnott, David; Bailey, Christopher; Bellon, Steve; Cummings, Richard T.; Albrecht, Brian K.; Harmange, Jean-Christophe; Kiefer, James R.; Trojer, Patrick; Classon, Marie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KDM5 family of histone demethylases catalyzes the demethylation of histone H3 on lysine 4 (H3K4) and is required for the survival of drug-tolerant persister cancer cells (DTPs).  Here the authors report the discovery and characterization of the specific KDM5 inhibitor CPI-455.  The crystal structure of KDM5A revealed the mechanism of inhibition of CPI-455 as well as the topol. arrangements of protein domains that influence substrate binding.  CPI-455 mediated KDM5 inhibition, elevated global levels of H3K4 trimethylation (H3K4me3) and decreased the no. of DTPs in multiple cancer cell line models treated with std. chemotherapy or targeted agents.  These findings show that pretreatment of cancer cells with a KDM5-specific inhibitor results in the ablation of a subpopulation of cancer cells that can serve as the founders for therapeutic relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxyW7L2YsV1rVg90H21EOLACvtfcHk0lhqwjWZn0cI_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D&md5=41699f5c17c912b39cdda81f4dfaa31e</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2085%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DTindell%26aufirst%3DC.%2BA.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DGuler%26aufirst%3DG.%2BD.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWongchenko%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCheung%26aufirst%3DT.%2BK.%26aulast%3DMaile%26aufirst%3DT.%2BM.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DCosta%26aufirst%3DM.%26aulast%3DHegde%26aufirst%3DG.%2BV.%26aulast%3DJackson%26aufirst%3DE.%26aulast%3DPitti%26aufirst%3DR.%26aulast%3DArnott%26aufirst%3DD.%26aulast%3DBailey%26aufirst%3DC.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DClasson%26aufirst%3DM.%26atitle%3DAn%2520inhibitor%2520of%2520KDM5%2520demethylases%2520reduces%2520survival%2520of%2520drug-tolerant%2520cancer%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fnchembio.2085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Porten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span> <span> </span><span class="NLM_article-title">Lead optimization of a pyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one scaffold to identify potent, selective and orally bioavailable KDM5 inhibitors suitable for in vivo biological studies</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4036</span>â <span class="NLM_lpage">4041</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.06.078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.bmcl.2016.06.078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27406798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCrtrfO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4036-4041&author=J.+Liangauthor=B.+Zhangauthor=S.+Labadieauthor=D.+F.+Ortwineauthor=M.+Vinogradovaauthor=J.+R.+Kieferauthor=V.+S.+Gehlingauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=T.+Laiauthor=J.+Liaoauthor=X.+Zhengauthor=Y.+Liuauthor=A.+Gustafsonauthor=E.+Van+der+Portenauthor=W.+Maoauthor=B.+M.+Liedererauthor=G.+Deshmukhauthor=M.+Classonauthor=P.+Trojerauthor=P.+S.+Dragovichauthor=L.+Murray&title=Lead+optimization+of+a+pyrazolo%5B1%2C5-a%5Dpyrimidin-7%284H%29-one+scaffold+to+identify+potent%2C+selective+and+orally+bioavailable+KDM5+inhibitors+suitable+for+in+vivo+biological+studies&doi=10.1016%2Fj.bmcl.2016.06.078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Lead optimization of a pyrazolo[1,5-a]pyrimidin-7(4H)-one scaffold to identify potent, selective and orally bioavailable KDM5 inhibitors suitable for in vivo biological studies</span></div><div class="casAuthors">Liang, Jun; Zhang, Birong; Labadie, Sharada; Ortwine, Daniel F.; Vinogradova, Maia; Kiefer, James R.; Gehling, Victor S.; Harmange, Jean-Christophe; Cummings, Richard; Lai, Tommy; Liao, Jiangpeng; Zheng, Xiaoping; Liu, Yichin; Gustafson, Amy; Van der Porten, Erica; Mao, Weifeng; Liederer, Bianca M.; Deshmukh, Gauri; Classon, Marie; Trojer, Patrick; Dragovich, Peter S.; Murray, Lesley</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4036-4041</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Starting with a lead [1,5-a]pyrimidin-7(4H)-one-contg. mol., the authors generated potent, selective and orally bioavailable KDM5 inhibitors.  Using structure- and property-based approaches, the authors designed I with improved cell potency (PC9 H3K4Me3 EC50 = 0.34 Î¼M).  Furthermore, I maintained suitable physiochem. properties and displayed an excellent pharmacokinetic (PK) profile in mice.  When dosed orally in mice at 50 mg/kg twice a day (BID), I showed an unbound maximal plasma concn. (Cmax) >15-fold over its cell EC50, thereby providing a robust chem. probe for studying KDM5 biol. functions in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4w4DV2WzWGLVg90H21EOLACvtfcHk0lj2uzAPgwgzUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCrtrfO&md5=64cde81ab69441e83148851121df73a5</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.06.078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.06.078%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DVan%2Bder%2BPorten%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DMurray%26aufirst%3DL.%26atitle%3DLead%2520optimization%2520of%2520a%2520pyrazolo%255B1%252C5-a%255Dpyrimidin-7%25284H%2529-one%2520scaffold%2520to%2520identify%2520potent%252C%2520selective%2520and%2520orally%2520bioavailable%2520KDM5%2520inhibitors%2520suitable%2520for%2520in%2520vivo%2520biological%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4036%26epage%3D4041%26doi%3D10.1016%2Fj.bmcl.2016.06.078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanigan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atrash, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bihan, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dempster, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxall, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sejberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scozzafava, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span> <span> </span><span class="NLM_article-title">8-substituted pyrido[3,4-<i>d</i>]pyrimidin-4(3<i>H</i>)-one derivatives as potent, cell permeable, KDM4 (JMJD2) and KDM5 (JARID1) histone lysine demethylase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1388</span>â <span class="NLM_lpage">1409</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGmsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1388-1409&author=V.+Bavetsiasauthor=R.+M.+Laniganauthor=G.+F.+Rudaauthor=B.+Atrashauthor=M.+G.+McLaughlinauthor=A.+Tumberauthor=N.+Y.+Mokauthor=Y.+V.+Le+Bihanauthor=S.+Dempsterauthor=K.+J.+Boxallauthor=F.+Jeganathanauthor=S.+B.+Hatchauthor=P.+Savitskyauthor=S.+Velupillaiauthor=T.+Krojerauthor=K.+S.+Englandauthor=J.+Sejbergauthor=C.+Thaiauthor=A.+Donovanauthor=A.+Palauthor=G.+Scozzafavaauthor=J.+M.+Bennettauthor=A.+Kawamuraauthor=C.+Johanssonauthor=A.+Szykowskaauthor=C.+Gileadiauthor=N.+A.+Burgess-Brownauthor=F.+von+Delftauthor=U.+Oppermannauthor=Z.+Waltersauthor=J.+Shipleyauthor=F.+I.+Raynaudauthor=S.+M.+Westawayauthor=R.+K.+Prinjhaauthor=O.+Fedorovauthor=R.+Burkeauthor=C.+J.+Schofieldauthor=I.+M.+Westwoodauthor=C.+Bountraauthor=S.+M%C3%BCllerauthor=R.+L.+van+Montfortauthor=P.+E.+Brennanauthor=J.+Blagg&title=8-substituted+pyrido%5B3%2C4-d%5Dpyrimidin-4%283H%29-one+derivatives+as+potent%2C+cell+permeable%2C+KDM4+%28JMJD2%29+and+KDM5+%28JARID1%29+histone+lysine+demethylase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">8-Substituted Pyrido[3,4-d]pyrimidin-4(3H)-one Derivatives As Potent, Cell Permeable, KDM4 (JMJD2) and KDM5 (JARID1) Histone Lysine Demethylase Inhibitors</span></div><div class="casAuthors">Bavetsias, Vassilios; Lanigan, Rachel M.; Ruda, Gian Filippo; Atrash, Butrus; McLaughlin, Mark G.; Tumber, Anthony; Mok, N. Yi; Le Bihan, Yann-Vai; Dempster, Sally; Boxall, Katherine J.; Jeganathan, Fiona; Hatch, Stephanie B.; Savitsky, Pavel; Velupillai, Srikannathasan; Krojer, Tobias; England, Katherine S.; Sejberg, Jimmy; Thai, Ching; Donovan, Adam; Pal, Akos; Scozzafava, Giuseppe; Bennett, James M.; Kawamura, Akane; Johansson, Catrine; Szykowska, Aleksandra; Gileadi, Carina; Burgess-Brown, Nicola A.; von Delft, Frank; Oppermann, Udo; Walters, Zoe; Shipley, Janet; Raynaud, Florence I.; Westaway, Susan M.; Prinjha, Rab K.; Fedorov, Oleg; Burke, Rosemary; Schofield, Christopher J.; Westwood, Isaac M.; Bountra, Chas; Muller, Susanne; van Montfort, Rob L. M.; Brennan, Paul E.; Blagg, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1388-1409</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the discovery of N-substituted 4-(pyridin-2-yl)thiazole-2-amine derivs. and their subsequent optimization, guided by structure-based design, to give 8-(1H-pyrazol-3-yl)pyrido[3,4-d]pyrimidin-4(3H)-ones, a series of potent JmjC histone N-Me lysine demethylase (KDM) inhibitors which bind to Fe(II) in the active site.  Substitution from C4 of the pyrazole moiety allows access to the histone peptide substrate binding site; incorporation of a conformationally constrained 4-phenylpiperidine linker gives derivs. such as 54j and 54k which demonstrate equipotent activity vs. the KDM4 (JMJD2) and KDM5 (JARID1) subfamily demethylases, selectivity over representative exemplars of the KDM2, KDM3, and KDM6 subfamilies, cellular permeability in the Caco-2 assay, and, for 54k, inhibition of H3K9Me3 and H3K4Me3 demethylation in a cell-based assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgIlVNGDiOQbVg90H21EOLACvtfcHk0lj2uzAPgwgzUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGmsrY%253D&md5=a7ff8a630f2119f9d7be5a8348cb56c3</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01635%26sid%3Dliteratum%253Aachs%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DLanigan%26aufirst%3DR.%2BM.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DMcLaughlin%26aufirst%3DM.%2BG.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DMok%26aufirst%3DN.%2BY.%26aulast%3DLe%2BBihan%26aufirst%3DY.%2BV.%26aulast%3DDempster%26aufirst%3DS.%26aulast%3DBoxall%26aufirst%3DK.%2BJ.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DSejberg%26aufirst%3DJ.%26aulast%3DThai%26aufirst%3DC.%26aulast%3DDonovan%26aufirst%3DA.%26aulast%3DPal%26aufirst%3DA.%26aulast%3DScozzafava%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DWalters%26aufirst%3DZ.%26aulast%3DShipley%26aufirst%3DJ.%26aulast%3DRaynaud%26aufirst%3DF.%2BI.%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DWestwood%26aufirst%3DI.%2BM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBlagg%26aufirst%3DJ.%26atitle%3D8-substituted%2520pyrido%255B3%252C4-d%255Dpyrimidin-4%25283H%2529-one%2520derivatives%2520as%2520potent%252C%2520cell%2520permeable%252C%2520KDM4%2520%2528JMJD2%2529%2520and%2520KDM5%2520%2528JARID1%2529%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1388%26epage%3D1409%26doi%3D10.1021%2Facs.jmedchem.5b01635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Insights into the action of inhibitor enantiomers against histone lysine demethylase 5A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3193</span>â <span class="NLM_lpage">3208</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00261</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00261" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3193-3208&author=J.+R.+Hortonauthor=X.+Liuauthor=L.+Wuauthor=K.+Zhangauthor=J.+Shanksauthor=X.+Zhangauthor=G.+Raiauthor=B.+T.+Mottauthor=D.+J.+Jansenauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=K.+Pohidaauthor=Y.+Fangauthor=X.+Huauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=M.+D.+Hallauthor=A.+Simeonovauthor=H.+Fuauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Insights+into+the+action+of+inhibitor+enantiomers+against+histone+lysine+demethylase+5A&doi=10.1021%2Facs.jmedchem.8b00261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Wu, Lizhen; Zhang, Kai; Shanks, John; Zhang, Xing; Rai, Ganesha; Mott, Bryan T.; Jansen, Daniel J.; Kales, Stephen C.; Henderson, Mark J.; Pohida, Katherine; Fang, Yuhong; Hu, Xin; Jadhav, Ajit; Maloney, David J.; Hall, Matthew D.; Simeonov, Anton; Fu, Haian; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3193-3208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isomers of chiral drugs can exhibit marked differences in biol. activities.  We studied the binding and inhibitory activities of 12 compds. against KDM5A.  Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (R)- and (S)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1H-pyrrolo[3,2-b]pyridine-7-carboxylic acid (compds. N51 and N52) and (R)- and (S)-N-(1-(3-isopropyl-1H-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compds. N54 and N55).  In vitro, the S enantiomer of the N51/N52 pair (N52) and the R enantiomer of the N54/N55 pair (N54) exhibited about 4- to 5-fold greater binding affinity.  The more potent enzyme inhibition of KDM5A by the R-isoform for the cell-permeable N54/N55 pair translated to differences in growth inhibitory activity.  We detd. structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity.  These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovFfXyPcUszbVg90H21EOLACvtfcHk0ljLr3750FKmBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D&md5=13ecac6902afd69d4ce7ad05803ca4d9</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00261%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DInsights%2520into%2520the%2520action%2520of%2520inhibitor%2520enantiomers%2520against%2520histone%2520lysine%2520demethylase%25205A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3193%26epage%3D3208%26doi%3D10.1021%2Facs.jmedchem.8b00261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective KDM5 inhibitor stops cellular demethylation of H3K4me3 at transcription start sites and proliferation of MM1S myeloma cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">371</span>â <span class="NLM_lpage">380</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.chembiol.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28262558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=371-380&author=A.+Tumberauthor=A.+Nuzziauthor=E.+S.+Hookwayauthor=S.+B.+Hatchauthor=S.+Velupillaiauthor=C.+Johanssonauthor=A.+Kawamuraauthor=P.+Savitskyauthor=C.+Yappauthor=A.+Szykowskaauthor=N.+Wuauthor=C.+Bountraauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=G.+F.+Rudaauthor=O.+Fedorovauthor=S.+Munroauthor=K.+S.+Englandauthor=R.+P.+Nowakauthor=C.+J.+Schofieldauthor=N.+B.+La+Thangueauthor=C.+Pawlynauthor=F.+Daviesauthor=G.+Morganauthor=N.+Athanasouauthor=S.+M%C3%BCllerauthor=U.+Oppermannauthor=P.+E.+Brennan&title=Potent+and+selective+KDM5+inhibitor+stops+cellular+demethylation+of+H3K4me3+at+transcription+start+sites+and+proliferation+of+MM1S+myeloma+cells&doi=10.1016%2Fj.chembiol.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective KDM5 Inhibitor Stops Cellular Demethylation of H3K4me3 at Transcription Start Sites and Proliferation of MM1S Myeloma Cells</span></div><div class="casAuthors">Tumber, Anthony; Nuzzi, Andrea; Hookway, Edward S.; Hatch, Stephanie B.; Velupillai, Srikannathasan; Johansson, Catrine; Kawamura, Akane; Savitsky, Pavel; Yapp, Clarence; Szykowska, Aleksandra; Wu, Na; Bountra, Chas; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Ruda, Gian Filippo; Fedorov, Oleg; Munro, Shonagh; England, Katherine S.; Nowak, Radoslaw P.; Schofield, Christopher J.; La Thangue, Nicholas B.; Pawlyn, Charlotte; Davies, Faith; Morgan, Gareth; Athanasou, Nick; Muller, Susanne; Oppermann, Udo; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-380</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of lysine residues on histone tail is a dynamic epigenetic modification that plays a key role in chromatin structure and gene regulation.  Members of the KDM5 (also known as JARID1) sub-family are 2-oxoglutarate (2-OG) and Fe2+-dependent oxygenases acting as histone 3 lysine 4 tri-Me (H3K4me3) demethylases, regulating proliferation, stem cell self-renewal, and differentiation.  Here we present the characterization of KDOAM-25, an inhibitor of KDM5 enzymes.  KDOAM-25 shows biochem. half maximal inhibitory concn. values of <100 nM for KDM5A-D in vitro, high selectivity toward other 2-OG oxygenases sub-families, and no off-target activity on a panel of 55 receptors and enzymes.  In human cell assay systems, KDOAM-25 has a half maximal effective concn. of â¼50 Î¼M and good selectivity toward other demethylases.  KDM5B is overexpressed in multiple myeloma and neg. correlated with the overall survival.  Multiple myeloma MM1S cells treated with KDOAM-25 show increased global H3K4 methylation at transcriptional start sites and impaired proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UzNQI-MiYbVg90H21EOLACvtfcHk0ljLr3750FKmBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D&md5=0fc70179bda8b4fd38db1054c7bb3277</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DPawlyn%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DF.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DPotent%2520and%2520selective%2520KDM5%2520inhibitor%2520stops%2520cellular%2520demethylation%2520of%2520H3K4me3%2520at%2520transcription%2520start%2520sites%2520and%2520proliferation%2520of%2520MM1S%2520myeloma%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D371%26epage%3D380%26doi%3D10.1016%2Fj.chembiol.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Structure-based engineering of irreversible inhibitors against histone lysine demethylase KDM5A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10588</span>â <span class="NLM_lpage">10601</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01219</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01219" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2rur3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10588-10601&author=J.+R.+Hortonauthor=C.+B.+Woodcockauthor=Q.+Chenauthor=X.+Liuauthor=X.+Zhangauthor=J.+Shanksauthor=G.+Raiauthor=B.+T.+Mottauthor=D.+J.+Jansenauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=M.+Cyrauthor=K.+Pohidaauthor=X.+Huauthor=P.+Shahauthor=X.+Xuauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=M.+D.+Hallauthor=A.+Simeonovauthor=H.+Fuauthor=P.+M.+Vertinoauthor=X.+Cheng&title=Structure-based+engineering+of+irreversible+inhibitors+against+histone+lysine+demethylase+KDM5A&doi=10.1021%2Facs.jmedchem.8b01219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A</span></div><div class="casAuthors">Horton, John R.; Woodcock, Clayton B.; Chen, Qin; Liu, Xu; Zhang, Xing; Shanks, John; Rai, Ganesha; Mott, Bryan T.; Jansen, Daniel J.; Kales, Stephen C.; Henderson, Mark J.; Cyr, Matthew; Pohida, Katherine; Hu, Xin; Shah, Pranav; Xu, Xin; Jadhav, Ajit; Maloney, David J.; Hall, Matthew D.; Simeonov, Anton; Fu, Haian; Vertino, Paula M.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10588-10601</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The active sites of hundreds of human Î±-ketoglutarate (Î±KG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors.  We identified a noncatalytic cysteine (Cys481 in KDM5A) near the active sites of KDM5 histone H3 lysine 4 demethylases, which is absent in other histone demethylase families, that could be explored for interaction with the cysteine-reactive electrophile acrylamide.  We synthesized analogs of a thienopyridine-based inhibitor chemotype, namely, 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-b]pyridine-7-carboxylic acid (N70) and a deriv. contg. a ((dimethylamino)but-2-enamido)phenyl moiety (N71) designed to form a covalent interaction with Cys481.  We characterized the inhibitory and binding activities against KDM5A and detd. the cocrystal structures of the catalytic domain of KDM5A in complex with N70 and N71.  Whereas the noncovalent inhibitor N70 displayed Î±KG-competitive inhibition that could be reversed after dialysis, inhibition by N71 was dependent on enzyme concn. and persisted even after dialysis, consistent with covalent modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo58lpkd4vS1LVg90H21EOLACvtfcHk0lioC2nk408Ntw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2rur3P&md5=5ab97cae54ccff28f65071fa41a70c0a</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01219%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DWoodcock%26aufirst%3DC.%2BB.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DCyr%26aufirst%3DM.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructure-based%2520engineering%2520of%2520irreversible%2520inhibitors%2520against%2520histone%2520lysine%2520demethylase%2520KDM5A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10588%26epage%3D10601%26doi%3D10.1021%2Facs.jmedchem.8b01219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Rodriguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cribbs, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunford, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thezenas, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and characterization of covalent KDM5 inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">515</span>â <span class="NLM_lpage">519</span>, <span class="refDoi">Â DOI: 10.1002/anie.201810179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201810179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVemtL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=515-519&author=S.+Vazquez-Rodriguezauthor=M.+Wrightauthor=C.+M.+Rogersauthor=A.+P.+Cribbsauthor=S.+Velupillaiauthor=M.+Philpottauthor=H.+Leeauthor=J.+E.+Dunfordauthor=K.+V.+M.+Huberauthor=M.+B.+Robersauthor=J.+D.+Vastaauthor=M.+L.+Thezenasauthor=S.+Bonhamauthor=B.+Kesslerauthor=J.+Bennettauthor=O.+Fedorovauthor=F.+Raynaudauthor=A.+Donovanauthor=J.+Blaggauthor=V.+Bavetsiasauthor=U.+Oppermannauthor=C.+Bountraauthor=A.+Kawamuraauthor=P.+E.+Brennan&title=Design%2C+synthesis+and+characterization+of+covalent+KDM5+inhibitors&doi=10.1002%2Fanie.201810179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis and Characterization of Covalent KDM5 Inhibitors</span></div><div class="casAuthors">Vazquez-Rodriguez, Saleta; Wright, Miranda; Rogers, Catherine M.; Cribbs, Adam P.; Velupillai, Srikannathasan; Philpott, Martin; Lee, Henry; Dunford, James E.; Huber, Kilian V. M.; Robers, Matthew B.; Vasta, James D.; Thezenas, Marie-Laetitia; Bonham, Sarah; Kessler, Benedikt; Bennett, James; Fedorov, Oleg; Raynaud, Florence; Donovan, Adam; Blagg, Julian; Bavetsias, Vassilios; Oppermann, Udo; Bountra, Chas; Kawamura, Akane; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">515-519</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Histone lysine demethylases (KDMs) are involved in the dynamic regulation of gene expression and they play a crit. role in several biol. processes.  Achieving selectivity over the different KDMs has been a major challenge for KDM inhibitor development.  Here we report potent and selective KDM5 covalent inhibitors designed to target cysteine residues only present in the KDM5 sub-family.  The covalent binding to the targeted proteins was confirmed by MS and time-dependent inhibition.  Addnl. competition assays show that compds. were non 2-OG competitive.  Target engagement and ChIP-seq anal. showed that the compds. inhibited the KDM5 members in cells at nano- to micromolar levels and induce a global increase of the H3K4me3 mark at transcriptional start sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnPw38R0sKE7Vg90H21EOLACvtfcHk0lioC2nk408Ntw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVemtL%252FI&md5=3b9f340a3eed3d5dad7aed79e0d7d931</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1002%2Fanie.201810179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201810179%26sid%3Dliteratum%253Aachs%26aulast%3DVazquez-Rodriguez%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DCribbs%26aufirst%3DA.%2BP.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DDunford%26aufirst%3DJ.%2BE.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DThezenas%26aufirst%3DM.%2BL.%26aulast%3DBonham%26aufirst%3DS.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DDonovan%26aufirst%3DA.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%252C%2520synthesis%2520and%2520characterization%2520of%2520covalent%2520KDM5%2520inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D515%26epage%3D519%26doi%3D10.1002%2Fanie.201810179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref173"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref173'); return false;" data-citation="" class="refNumLink">173</a></strong><div class="NLM_citation" id="cit173"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interaction with covalent small-molecule inhibitors</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1872</span>â <span class="NLM_lpage">1876</span>, <span class="refDoi">Â DOI: 10.2174/1568026619666191011163410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2174%2F1568026619666191011163410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31696799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVymtLvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=1872-1876&author=B.+Liauthor=D.+Rongauthor=Y.+Wang&title=Targeting+protein-protein+interaction+with+covalent+small-molecule+inhibitors&doi=10.2174%2F1568026619666191011163410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Protein-Protein Interaction with Covalent Small-Molecule Inhibitors</span></div><div class="casAuthors">Li, Bingbing; Rong, Deqin; Wang, Yuanxiang</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">1872-1876</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  PPIs are involved in diverse biochem. events and perform their functions through the formation of protein-protein complexes or PPI networks.  The large and flat interacting surfaces of PPIs make discovery of small-mol. modulators a challenging task.  New strategies and more effective chem. technologies are needed to facilitate the development of PPIs small-mol. inhibitors.  Covalent modification of a nucleophilic residue located proximally to the immediate vicinity of PPIs can overcome the disadvantages of large interacting surfaces and provides high-affinity inhibitors with increased duration of action and prolonged target modulation.  On the other hand, covalent inhibitors that target non-conserved protein residues demonstrate improved selectivity over related protein family members.  Herein, we highlight the latest progress of small-mol. covalent PPIs inhibitors and hope to shed light on future PPIs inhibitor design and development.  The relevant challenges and opportunities are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox4P1EKJ7ZrLVg90H21EOLACvtfcHk0ljs9idwIlbjww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVymtLvO&md5=5be818e234b25f3577c9b7f593868f6f</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.2174%2F1568026619666191011163410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026619666191011163410%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DRong%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DTargeting%2520protein-protein%2520interaction%2520with%2520covalent%2520small-molecule%2520inhibitors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2019%26volume%3D19%26spage%3D1872%26epage%3D1876%26doi%3D10.2174%2F1568026619666191011163410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref174"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref174'); return false;" data-citation="" class="refNumLink">174</a></strong><div class="NLM_citation" id="cit174"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Progress with covalent small-molecule kinase inhibitors</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">727</span>â <span class="NLM_lpage">735</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2018.01.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.drudis.2018.01.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29337202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=727-735&author=Z.+Zhaoauthor=P.+E.+Bourne&title=Progress+with+covalent+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.drudis.2018.01.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Progress with covalent small-molecule kinase inhibitors</span></div><div class="casAuthors">Zhao, Zheng; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">With reduced risk of toxicity and high selectivity, covalent small-mol. kinase inhibitors (CSKIs) have emerged rapidly.  Through the lens of structural system pharmacol., here we review this rapid progress by considering design strategies and the challenges and opportunities offered by current CSKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog6l6qMAopYLVg90H21EOLACvtfcHk0ljs9idwIlbjww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D&md5=f97a19f53a72044812a778ee9351fe94</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProgress%2520with%2520covalent%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D727%26epage%3D735%26doi%3D10.1016%2Fj.drudis.2018.01.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref175"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref175'); return false;" data-citation="" class="refNumLink">175</a></strong><div class="NLM_citation" id="cit175"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asgian, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekici, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, K. E.</span></span> <span> </span><span class="NLM_article-title">Irreversible inhibitors of serine, cysteine, and threonine proteases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4639</span>â <span class="NLM_lpage">4750</span>, <span class="refDoi">Â DOI: 10.1021/cr010182v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr010182v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4639-4750&author=J.+C.+Powersauthor=J.+L.+Asgianauthor=O.+D.+Ekiciauthor=K.+E.+James&title=Irreversible+inhibitors+of+serine%2C+cysteine%2C+and+threonine+proteases&doi=10.1021%2Fcr010182v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible inhibitors of serine, cysteine, and threonine proteases</span></div><div class="casAuthors">Powers, James C.; Asgian, Juliana L.; Ekici, Oezlem Dogan; James, Karen Ellis</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4639-4750</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Topics discussed include alkylating, acylating, phosphonylating, and sulfonylating agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorivaxGhmhirVg90H21EOLACvtfcHk0ljs9idwIlbjww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D&md5=eb21ff0297b87537a7d459c92f13c75d</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1021%2Fcr010182v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr010182v%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26aulast%3DAsgian%26aufirst%3DJ.%2BL.%26aulast%3DEkici%26aufirst%3DO.%2BD.%26aulast%3DJames%26aufirst%3DK.%2BE.%26atitle%3DIrreversible%2520inhibitors%2520of%2520serine%252C%2520cysteine%252C%2520and%2520threonine%2520proteases%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4639%26epage%3D4750%26doi%3D10.1021%2Fcr010182v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref176"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref176'); return false;" data-citation="" class="refNumLink">176</a></strong><div class="NLM_citation" id="cit176"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gambini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udompholkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jossart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salem, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors of protein-protein interactions targeting lysine, tyrosine, or histidine residues</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5616</span>â <span class="NLM_lpage">5627</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b00561</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00561" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVOkur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5616-5627&author=L.+Gambiniauthor=C.+Baggioauthor=P.+Udompholkulauthor=J.+Jossartauthor=A.+F.+Salemauthor=J.+Perryauthor=M.+Pellecchia&title=Covalent+inhibitors+of+protein-protein+interactions+targeting+lysine%2C+tyrosine%2C+or+histidine+residues&doi=10.1021%2Facs.jmedchem.9b00561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibitors of Protein-Protein Interactions Targeting Lysine, Tyrosine, or Histidine Residues</span></div><div class="casAuthors">Gambini, Luca; Baggio, Carlo; Udompholkul, Parima; Jossart, Jennifer; Salem, Ahmed F.; Perry, J. Jefferson P.; Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5616-5627</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have recently reported a series of Lys-covalent agents targeting the BIR3 domain of the X-linked inhibitor of apoptosis protein (XIAP) using a benzamide-sulfonyl fluoride warhead.  Using XIAP as a model system, we further investigated a variety of addnl. warheads that can be easily incorporated into binding peptides and analyzed their ability to form covalent adducts with lysine and other amino acids, including tyrosine, histidine, serine, and threonine, using biochem. and biophys. assays.  Moreover, we tested aq., plasma stability, cell permeability, and cellular efficacy of the most effective agents.  These studies identified aryl-fluoro sulfates as likely the most suitable electrophiles to effectively form covalent adducts with Lys, Tyr, and His residues, given that these agents were cell permeable and stable in aq. buffer and in plasma.  Our studies contain a no. of general findings that open new possible avenues for the design of potent covalent protein-protein interaction antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_gJvmpArk6LVg90H21EOLACvtfcHk0ljs9idwIlbjww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVOkur0%253D&md5=820cf16d34a3230e9629991a3bca240d</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00561%26sid%3Dliteratum%253Aachs%26aulast%3DGambini%26aufirst%3DL.%26aulast%3DBaggio%26aufirst%3DC.%26aulast%3DUdompholkul%26aufirst%3DP.%26aulast%3DJossart%26aufirst%3DJ.%26aulast%3DSalem%26aufirst%3DA.%2BF.%26aulast%3DPerry%26aufirst%3DJ.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DCovalent%2520inhibitors%2520of%2520protein-protein%2520interactions%2520targeting%2520lysine%252C%2520tyrosine%252C%2520or%2520histidine%2520residues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5616%26epage%3D5627%26doi%3D10.1021%2Facs.jmedchem.9b00561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref177"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref177'); return false;" data-citation="" class="refNumLink">177</a></strong><div class="NLM_citation" id="cit177"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span> <span> </span><span class="NLM_article-title">Emerging and re-emerging warheads for targeted covalent inhibitors: applications in medicinal chemistry and chemical biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5673</span>â <span class="NLM_lpage">5724</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5673-5724&author=M.+Gehringerauthor=S.+A.+Laufer&title=Emerging+and+re-emerging+warheads+for+targeted+covalent+inhibitors%3A+applications+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Facs.jmedchem.8b01153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Gehringer, Matthias; Laufer, Stefan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5673-5724</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors (TCIs) are designed to bind poorly conserved amino acids by means of reactive groups, the so-called warheads.  Currently, targeting noncatalytic cysteine residues with acrylamides and other Î±,Î²-unsatd. carbonyl compds. is the predominant strategy in TCI development.  The recent ascent of covalent drugs has stimulated considerable efforts to characterize alternative warheads for the covalent-reversible and irreversible engagement of noncatalytic cysteine residues as well as other amino acids.  This Perspective article provides an overview of warheads-beyond Î±,Î²-unsatd. amides-recently used in the design of targeted covalent ligands.  Promising reactive groups that have not yet demonstrated their utility in TCI development are also highlighted.  Special emphasis is placed on the discussion of reactivity and of case studies illustrating applications in medicinal chem. and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDhc4PWV293LVg90H21EOLACvtfcHk0lgkHKPorGuwnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL&md5=4623733558ff99aadebfc13950b816ad</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26aulast%3DGehringer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26atitle%3DEmerging%2520and%2520re-emerging%2520warheads%2520for%2520targeted%2520covalent%2520inhibitors%253A%2520applications%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5673%26epage%3D5724%26doi%3D10.1021%2Facs.jmedchem.8b01153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref178"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref178'); return false;" data-citation="" class="refNumLink">178</a></strong><div class="NLM_citation" id="cit178"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">CovalentInDB: a comprehensive database facilitating the discovery of covalent inhibitors</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">D1122</span>â <span class="NLM_lpage">D1129</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkaa876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1093%2Fnar%2Fgkaa876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=33068433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3MXntlaqtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2021&pages=D1122-D1129&author=H.+Duauthor=J.+Gaoauthor=G.+Wengauthor=J.+Dingauthor=X.+Chaiauthor=J.+Pangauthor=Y.+Kangauthor=D.+Liauthor=D.+Caoauthor=T.+Hou&title=CovalentInDB%3A+a+comprehensive+database+facilitating+the+discovery+of+covalent+inhibitors&doi=10.1093%2Fnar%2Fgkaa876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">CovalentInDB: a comprehensive database facilitating the discovery of covalent inhibitors</span></div><div class="casAuthors">Du, Hongyan; Gao, Junbo; Weng, Gaoqi; Ding, Junjie; Chai, Xin; Pang, Jinping; Kang, Yu; Li, Dan; Cao, Dongsheng; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1122-D1129</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Inhibitors that form covalent bonds with their targets have traditionally been considered highly adventurous due to their potential off-target effects and toxicity concerns.  However, with the clin. validation and approval of many covalent inhibitors during the past decade, design and discovery of novel covalent inhibitors have attracted increasing attention.  A large amt. of scattered exptl. data for covalent inhibitors have been reported, but a resource by integrating the exptl. information for covalent inhibitor discovery is still lacking.  In this study, we presented Covalent Inhibitor Database (CovalentInDB), the largest online database that provides the structural information and exptl. data for covalent inhibitors.  CovalentInDB contains 4511 covalent inhibitors (including 68 approved drugs) with 57 different reactive warheads for 280 protein targets.  The crystal structures of some of the proteins bound with a covalent inhibitor are provided to visualize the protein-ligand interactions around the binding site.  Each covalent inhibitor is annotated with the structure, warhead, exptl. bioactivity, physicochem. properties, etc.  Moreover, CovalentInDB provides the covalent reaction mechanism and the corresponding exptl. verification methods for each inhibitor towards its target.  High-quality datasets are downloadable for users to evaluate and develop computational methods for covalent drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkFzGN1-OPHLVg90H21EOLACvtfcHk0lgkHKPorGuwnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXntlaqtbw%253D&md5=c426c2ce5eeac717236d956bb0882d36</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkaa876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkaa876%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DWeng%26aufirst%3DG.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DChai%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DCovalentInDB%253A%2520a%2520comprehensive%2520database%2520facilitating%2520the%2520discovery%2520of%2520covalent%2520inhibitors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2021%26volume%3D49%26spage%3DD1122%26epage%3DD1129%26doi%3D10.1093%2Fnar%2Fgkaa876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref179"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref179'); return false;" data-citation="" class="refNumLink">179</a></strong><div class="NLM_citation" id="cit179"><span><span class="NLM_contrib-group"><span class="NLM_string-name">ÃbrÃ¡nyi-Balogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szijj, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarpino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrast, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MitroviÄ, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FonoviÄ, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">NÃ©meth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barreteau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">HorvÃ¡ti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IlaÅ¡, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobec, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KeserÅ±, G. M.</span></span> <span> </span><span class="NLM_article-title">A road map for prioritizing warheads for cysteine targeting covalent inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">94</span>â <span class="NLM_lpage">107</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2018.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30321804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFeqtrbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2018&pages=94-107&author=P.+%C3%81br%C3%A1nyi-Baloghauthor=L.+Petriauthor=T.+Imreauthor=P.+Szijjauthor=A.+Scarpinoauthor=M.+Hrastauthor=A.+Mitrovi%C4%87author=U.+P.+Fonovi%C4%8Dauthor=K.+N%C3%A9methauthor=H.+Barreteauauthor=D.+I.+Roperauthor=K.+Horv%C3%A1tiauthor=G.+G.+Ferenczyauthor=J.+Kosauthor=J.+Ila%C5%A1author=S.+Gobecauthor=G.+M.+Keser%C5%B1&title=A+road+map+for+prioritizing+warheads+for+cysteine+targeting+covalent+inhibitors&doi=10.1016%2Fj.ejmech.2018.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">A road map for prioritizing warheads for cysteine targeting covalent inhibitors</span></div><div class="casAuthors">Abranyi-Balogh, Peter; Petri, Laszlo; Imre, Timea; Szijj, Peter; Scarpino, Andrea; Hrast, Martina; Mitrovic, Ana; Fonovic, Ursa Pecar; Nemeth, Krisztina; Barreteau, Helene; Roper, David I.; Horvati, Kata; Ferenczy, Gyorgy G.; Kos, Janko; Ilas, Janez; Gobec, Stanislav; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">94-107</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors have become an integral part of a no. of therapeutic protocols and are the subject of intense research.  The mechanism of action of these compds. involves the formation of a covalent bond with protein nucleophiles, mostly cysteines.  Given the abundance of cysteines in the proteome, the specificity of the covalent inhibitors is of utmost importance and requires careful optimization of the applied warheads.  In most of the cysteine targeting covalent inhibitor programs the design strategy involves incorporating Michael acceptors into a ligand that is already known to bind noncovalently.  In contrast, the authors suggest that the reactive warhead itself should be tailored to the reactivity of the specific cysteine being targeted, and the authors describe a strategy to achieve this goal.  Here, the authors have extended and systematically explored the available org. chem. toolbox and characterized a large no. of warheads representing different chemistries.  The authors demonstrate that in addn. to the common Michael addn., there are other nucleophilic addn., addn.-elimination, nucleophilic substitution and oxidn. reactions suitable for specific covalent protein modification.  Importantly, the authors reveal that warheads for these chemistries impact the reactivity and specificity of covalent fragments at both protein and proteome levels.  By integrating surrogate reactivity and selectivity models and subsequent protein assays, the authors define a road map to help enable new or largely unexplored covalent chemistries for the optimization of cysteine targeting inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogAumXXtgqXbVg90H21EOLACvtfcHk0lgkHKPorGuwnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFeqtrbJ&md5=5649a7a5b95adc70756caddf765cf67c</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.010%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2581br%25C3%25A1nyi-Balogh%26aufirst%3DP.%26aulast%3DPetri%26aufirst%3DL.%26aulast%3DImre%26aufirst%3DT.%26aulast%3DSzijj%26aufirst%3DP.%26aulast%3DScarpino%26aufirst%3DA.%26aulast%3DHrast%26aufirst%3DM.%26aulast%3DMitrovi%25C4%2587%26aufirst%3DA.%26aulast%3DFonovi%25C4%258D%26aufirst%3DU.%2BP.%26aulast%3DN%25C3%25A9meth%26aufirst%3DK.%26aulast%3DBarreteau%26aufirst%3DH.%26aulast%3DRoper%26aufirst%3DD.%2BI.%26aulast%3DHorv%25C3%25A1ti%26aufirst%3DK.%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKos%26aufirst%3DJ.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DGobec%26aufirst%3DS.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DA%2520road%2520map%2520for%2520prioritizing%2520warheads%2520for%2520cysteine%2520targeting%2520covalent%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D160%26spage%3D94%26epage%3D107%26doi%3D10.1016%2Fj.ejmech.2018.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref180"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref180'); return false;" data-citation="" class="refNumLink">180</a></strong><div class="NLM_citation" id="cit180"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egyed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">HetÃ©nyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ÃbrÃ¡nyi-Balogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KeserÅ±, G. M.</span></span> <span> </span><span class="NLM_article-title">An electrophilic warhead library for mapping the reactivity and accessibility of tractable cysteines in protein kinases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>,  <span class="NLM_fpage">112836</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2020.112836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2020.112836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32971426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSisb3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&pages=112836&author=L.+Petriauthor=A.+Egyedauthor=D.+Bajuszauthor=T.+Imreauthor=A.+Het%C3%A9nyiauthor=T.+Martinekauthor=P.+%C3%81br%C3%A1nyi-Baloghauthor=G.+M.+Keser%C5%B1&title=An+electrophilic+warhead+library+for+mapping+the+reactivity+and+accessibility+of+tractable+cysteines+in+protein+kinases&doi=10.1016%2Fj.ejmech.2020.112836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">An electrophilic warhead library for mapping the reactivity and accessibility of tractable cysteines in protein kinases</span></div><div class="casAuthors">Petri, Laszlo; Egyed, Attila; Bajusz, David; Imre, Timea; Hetenyi, Anasztazia; Martinek, Tamas; Abranyi-Balogh, Peter; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112836</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors represent a viable strategy to block protein kinases involved in different disease pathologies.  Although a no. of computational protocols have been published for identifying druggable cysteines, exptl. approaches are limited for mapping the reactivity and accessibility of these residues.  Here, we present a ligand based approach using a toolbox of fragment-sized mols. with identical scaffold but equipped with diverse covalent warheads.  Our library represents a unique opportunity for the efficient integration of warhead-optimization and target-validation into the covalent drug development process.  Screening this probe kit against multiple kinases could exptl. characterize the accessibility and reactivity of the targeted cysteines and helped to identify suitable warheads for designed covalent inhibitors.  The usefulness of this approach has been confirmed retrospectively on Janus kinase 3 (JAK3).  Furthermore, representing a prospective validation, we identified Maternal embryonic leucine zipper kinase (MELK), as a tractable covalent target.  Covalently labeling and biochem. inhibition of MELK would suggest an alternative covalent strategy for MELK inhibitor programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxfhiyWH3hMrVg90H21EOLACvtfcHk0lgd3ZX7WcpQsg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSisb3E&md5=370848ac3d886667890171b53f89341e</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112836%26sid%3Dliteratum%253Aachs%26aulast%3DPetri%26aufirst%3DL.%26aulast%3DEgyed%26aufirst%3DA.%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DImre%26aufirst%3DT.%26aulast%3DHet%25C3%25A9nyi%26aufirst%3DA.%26aulast%3DMartinek%26aufirst%3DT.%26aulast%3D%25C3%2581br%25C3%25A1nyi-Balogh%26aufirst%3DP.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DAn%2520electrophilic%2520warhead%2520library%2520for%2520mapping%2520the%2520reactivity%2520and%2520accessibility%2520of%2520tractable%2520cysteines%2520in%2520protein%2520kinases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26spage%3D112836%26doi%3D10.1016%2Fj.ejmech.2020.112836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172'],'ref173':['cit173'],'ref174':['cit174'],'ref175':['cit175'],'ref176':['cit176'],'ref177':['cit177'],'ref178':['cit178'],'ref179':['cit179'],'ref180':['cit180']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Covalent small-molecule TKIs approved by the FDA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of compounds <b>1</b>â<b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of compounds <b>8</b>â<b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structures of meninâMLL inhibitors in complex with menin. (A) Cocrystal structure of <b>8</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG3">4OG3</a>). (B) Cocrystal structure of <b>9</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8">4OG8</a>). (C) Cocrystal structure of <b>12</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B41">6B41</a>). (D) Cocrystal structure of <b>14</b> in complex with menin (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WNH">6WNH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of compounds <b>15</b>â<b>23</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Cocrystal structure of <b>19</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W1Y">5W1Y</a>). (B) Cocrystal structure of <b>21</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T5G">5T5G</a>). (C) Cocrystal structure of <b>22</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH7">5TH7</a>). (D) Cocrystal structure of <b>23</b> in complex with SETD8 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOZ">6BOZ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures of compounds <b>24</b>â<b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Cocrystal structure of <b>26</b> in complex with PRMT5 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CKC">6CKC</a>). (B) Cocrystal structure of <b>29</b> in complex with PRMT5 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6K1S">6K1S</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of compounds <b>30</b>â<b>37</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Cocrystal structure of <b>30</b> in complex with PRMT6 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R">5E8R</a>). (B) Cocrystal structure of <b>37</b> in complex with PRMT6 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structures of compounds <b>38</b>â<b>45</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Trapping mechanism of azacytosine. (A) Scheme of DNA methylation. (B) Scheme of DNMT inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Structures of compounds <b>46</b>â<b>58</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. (A) Cocrystal structure of <b>56</b> in complex with BRD4 BD1 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZV">6CZV</a>). (B) Cocrystal structure of <b>57</b> in complex with BRD4 BD1 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZU">6CZU</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. FAD-dependent histone demethylation by LSD1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Structures of compounds <b>59</b>â<b>64</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Catalytic mechanisms for LSD1 and MAOB inhibition with <b>59</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. Structures of compounds <b>72</b>â<b>83</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/medium/jm0c02055_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. (A) Cocrystal structure of <b>79</b> in complex with KDM5A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA">6DQA</a>). (B) Cocrystal structure of <b>80</b> in complex with KDM5A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB">6DQB</a>). (C) Cocrystal structure of <b>75</b> in complex with KDM4A (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F3I">5F3I</a>). (D) Cocrystal structure of <b>82</b> in complex with KDM5B (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIN">6EIN</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02055/20210616/images/large/jm0c02055_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02055&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 180 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, E.</span></span> <span> </span><span class="NLM_article-title">Advances in epigenetics link genetics to the environment and disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>571</i></span>,  <span class="NLM_fpage">489</span>â <span class="NLM_lpage">499</span>, <span class="refDoi">Â DOI: 10.1038/s41586-019-1411-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fs41586-019-1411-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31341302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVeqs7rO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2019&pages=489-499&author=G.+Cavalliauthor=E.+Heard&title=Advances+in+epigenetics+link+genetics+to+the+environment+and+disease&doi=10.1038%2Fs41586-019-1411-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in epigenetics link genetics to the environment and disease</span></div><div class="casAuthors">Cavalli, Giacomo; Heard, Edith</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">571</span>
        (<span class="NLM_cas:issue">7766</span>),
    <span class="NLM_cas:pages">489-499</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Epigenetic research has accelerated rapidly in the twenty-first century, generating justified excitement and hope, but also a degree of hype.  Here we review how the field has evolved over the last few decades and reflect on some of the recent advances that are changing our understanding of biol.  We discuss the interplay between epigenetics and DNA sequence variation as well as the implications of epigenetics for cellular memory and plasticity.  We consider the effects of the environment and both intergenerational and transgenerational epigenetic inheritance on biol., disease and evolution.  Finally, we present some new frontiers in epigenetics with implications for human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGOerze2PSHLVg90H21EOLACvtfcHk0lgkZSzssoXEcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVeqs7rO&md5=726fa6d235e33685dbb7a7f350eff7ff</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1411-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1411-0%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DG.%26aulast%3DHeard%26aufirst%3DE.%26atitle%3DAdvances%2520in%2520epigenetics%2520link%2520genetics%2520to%2520the%2520environment%2520and%2520disease%26jtitle%3DNature%26date%3D2019%26volume%3D571%26spage%3D489%26epage%3D499%26doi%3D10.1038%2Fs41586-019-1411-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span> <i>Epigenetics</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span>, <span class="NLM_string-name">Jenuwein, T.</span>, <span class="NLM_string-name">Reinberg, D.</span></span>, Eds.; <span class="NLM_publisher-name">Cold Spring Harbor Laboratory Press</span>: <span class="NLM_publisher-loc">Cold Spring Harbor, NY</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=C.+D.+Allis&author=T.+Jenuwein&author=D.+Reinberg&title=Epigenetics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DEpigenetics%26aulast%3DAllis%26aufirst%3DC.%2BD.%26pub%3DCold%2520Spring%2520Harbor%2520Laboratory%2520Press%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koldobskiy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶ndÃ¶r, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic modulators, modifiers and mediators in cancer aetiology and progression</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">284</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1038/nrg.2016.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrg.2016.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26972587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVOksLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=284-299&author=A.+P.+Feinbergauthor=M.+A.+Koldobskiyauthor=A.+G%C3%B6nd%C3%B6r&title=Epigenetic+modulators%2C+modifiers+and+mediators+in+cancer+aetiology+and+progression&doi=10.1038%2Fnrg.2016.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Disease mechanisms Epigenetic modulators, modifiers and mediators in cancer aetiology and progression</span></div><div class="casAuthors">Feinberg, Andrew P.; Koldobskiy, Michael A.; Gondor, Anita</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">284-299</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This year is the tenth anniversary of the publication in this journal of a model suggesting the existence of 'tumor progenitor genes'.  These genes are epigenetically disrupted at the earliest stages of malignancies, even before mutations, and thus cause altered differentiation throughout tumor evolution.  The past decade of discovery in cancer epigenetics has revealed a no. of similarities between cancer genes and stem cell reprogramming genes, widespread mutations in epigenetic regulators, and the part played by chromatin structure in cellular plasticity in both development and cancer.  In the light of these discoveries, we suggest here a framework for cancer epigenetics involving three types of genes: 'epigenetic mediators', corresponding to the tumor progenitor genes suggested earlier; 'epigenetic modifiers' of the mediators, which are frequently mutated in cancer; and 'epigenetic modulators' upstream of the modifiers, which are responsive to changes in the cellular environment and often linked to the nuclear architecture.  We suggest that this classification is helpful in framing new diagnostic and therapeutic approaches to cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowr2rnZ1iYq7Vg90H21EOLACvtfcHk0lhtaOnb1t_ihA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVOksLk%253D&md5=9437a1c1cde1a6745504b979cb6ab60f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2016.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2016.13%26sid%3Dliteratum%253Aachs%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DKoldobskiy%26aufirst%3DM.%2BA.%26aulast%3DG%25C3%25B6nd%25C3%25B6r%26aufirst%3DA.%26atitle%3DEpigenetic%2520modulators%252C%2520modifiers%2520and%2520mediators%2520in%2520cancer%2520aetiology%2520and%2520progression%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2016%26volume%3D17%26spage%3D284%26epage%3D299%26doi%3D10.1038%2Fnrg.2016.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S.</span></span> <span> </span><span class="NLM_article-title">Targeting the cancer epigenome for therapy</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">630</span>â <span class="NLM_lpage">641</span>, <span class="refDoi">Â DOI: 10.1038/nrg.2016.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrg.2016.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27629931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiu77O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=630-641&author=P.+A.+Jonesauthor=J.+P.+Issaauthor=S.+Baylin&title=Targeting+the+cancer+epigenome+for+therapy&doi=10.1038%2Fnrg.2016.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer epigenome for therapy</span></div><div class="casAuthors">Jones, Peter A.; Issa, Jean-Pierre J.; Baylin, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">630-641</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Next-generation sequencing has revealed that more than 50% of human cancers harbor mutations in enzymes that are involved in chromatin organization.  Tumor cells not only are activated by genetic and epigenetic alterations, but also routinely use epigenetic processes to ensure their escape from chemotherapy and host immune surveillance.  Hence, a growing emphasis of recent drug discovery efforts has been on targeting the epigenome, including DNA methylation and histone modifications, with several new drugs being tested and some already approved by the US Food and Drug Administration (FDA).  The future will see the increasing success of combining epigenetic drugs with other therapies.  As epigenetic drugs target the epigenome as a whole, these true 'genomic medicines' lessen the need for precision approaches to individualized therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5pfbPOnZMtbVg90H21EOLACvtfcHk0lhtaOnb1t_ihA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiu77O&md5=22d98e58a4637d61af34a382e92adace</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2016.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2016.93%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26aulast%3DBaylin%26aufirst%3DS.%26atitle%3DTargeting%2520the%2520cancer%2520epigenome%2520for%2520therapy%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2016%26volume%3D17%26spage%3D630%26epage%3D641%26doi%3D10.1038%2Fnrg.2016.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetics in cancer</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">187</span>â <span class="NLM_lpage">207</span>, <span class="refDoi">Â DOI: 10.1146/annurev-pharmtox-010716-105106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1146%2Fannurev-pharmtox-010716-105106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28992434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1altrrE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=187-207&author=R.+L.+Bennettauthor=J.+D.+Licht&title=Targeting+epigenetics+in+cancer&doi=10.1146%2Fannurev-pharmtox-010716-105106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Epigenetics in Cancer</span></div><div class="casAuthors">Bennett, Richard L.; Licht, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">187-207</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Alterations of genes regulating epigenetic processes are frequently found as cancer drivers and may cause widespread alterations of DNA methylation, histone modification patterns, or chromatin structure that disrupt normal patterns of gene expression.  Because of the inherent reversibility of epigenetic changes, inhibitors targeting these processes are promising anticancer strategies.  Small mols. targeting epigenetic regulators have been developed recently, and clin. trials of these agents are under way for hematol. malignancies and solid tumors.  In this review, we describe how the writers, readers, and erasers of epigenetic marks are dysregulated in cancer and summarize the development of therapies targeting these mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBHGapO_R7HbVg90H21EOLACvtfcHk0lhtaOnb1t_ihA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1altrrE&md5=49208408fd3d3c29f9e256c52c933446</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010716-105106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010716-105106%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DR.%2BL.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26atitle%3DTargeting%2520epigenetics%2520in%2520cancer%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2018%26volume%3D58%26spage%3D187%26epage%3D207%26doi%3D10.1146%2Fannurev-pharmtox-010716-105106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ã.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1596</span>â <span class="NLM_lpage">1629</span>, <span class="refDoi">Â DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+%C3%9C.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lhAqkYf1QAuyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ã.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>â <span class="NLM_lpage">1068</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lhAqkYf1QAuyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">673</span>â <span class="NLM_lpage">691</span>, <span class="refDoi">Â DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+deacetylases+and+their+inhibitors+in+cancer%2C+neurological+diseases+and+immune+disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0lhAqkYf1QAuyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520deacetylases%2520and%2520their%2520inhibitors%2520in%2520cancer%252C%2520neurological%2520diseases%2520and%2520immune%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkinson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Recent progress in histone demethylase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1308</span>â <span class="NLM_lpage">1329</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1308-1329&author=T.+E.+McAllisterauthor=K.+S.+Englandauthor=R.+J.+Hopkinsonauthor=P.+E.+Brennanauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Recent+progress+in+histone+demethylase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in Histone Demethylase Inhibitors</span></div><div class="casAuthors">McAllister, Tom E.; England, Katherine S.; Hopkinson, Richard J.; Brennan, Paul E.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1308-1329</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is increasing interest in targeting histone N-methyl-lysine demethylases (KDMs) with small mols. both for the generation of probes for target exploration and for therapeutic purposes.  Here we update on previous reviews on the inhibition of the lysine-specific demethylases (LSDs or KDM1s) and JmjC families of N-methyl-lysine demethylases (JmjC KDMs, KDM2-7), focusing on the academic and patent literature from 2014 to date.  We also highlight recent biochem., biol., and structural studies which are relevant to KDM inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWx0bnBT9cDrVg90H21EOLACvtfcHk0lhAqkYf1QAuyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE&md5=f744b65af58dff765cdab8f9fea85310</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01758%26sid%3Dliteratum%253Aachs%26aulast%3DMcAllister%26aufirst%3DT.%2BE.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DRecent%2520progress%2520in%2520histone%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1308%26epage%3D1329%26doi%3D10.1021%2Facs.jmedchem.5b01758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Chemical and biochemical perspectives of protein lysine methylation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">6656</span>â <span class="NLM_lpage">6705</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.8b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=6656-6705&author=M.+Luo&title=Chemical+and+biochemical+perspectives+of+protein+lysine+methylation&doi=10.1021%2Facs.chemrev.8b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Biochemical Perspectives of Protein Lysine Methylation</span></div><div class="casAuthors">Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6656-6705</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein lysine methylation is a distinct post-translational modification that causes minimal changes in the size and electrostatic status of lysine residues.  Lysine methylation plays essential roles in regulating fates and functions of target proteins in an epigenetic manner.  As a result, substrates and degrees (free vs. mono/di/tri) of protein lysine methylation are orchestrated within cells by balanced activities of protein lysine methyltransferases (PKMTs) and demethylases (KDMs).  Their dysregulation is often assocd. with neurol. disorders, developmental abnormalities or cancer.  Methyllysine-contg. proteins can be recognized by downstream effector proteins, which contain methyllysine reader domains, to relay their biol. functions.  While numerous efforts have been made to annotate biol. roles of protein lysine methylation, limited work has been done to uncover mechanisms assocd. with this modification at a mol. or at. level.  Given distinct biophys. and biochem. properties of methyllysine, this review will focus on chem. and biochem. aspects in addn., recognition, and removal of this post-translational mark.  Chem. and biophys. methods to profile PKMT substrates will be discussed along with classification of PKMT inhibitors for accurate perturbation of methyltransferase activities.  Semisynthesis of methyllysine-contg. proteins will also be covered given the crit. need for these reagents to unambiguously define functional roles of protein lysine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbSRWg3jP5bLVg90H21EOLACvtfcHk0lieBM0ZmYwf3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO&md5=d846db925be49839c5954f5fc34f17d9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00008%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DChemical%2520and%2520biochemical%2520perspectives%2520of%2520protein%2520lysine%2520methylation%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D6656%26epage%3D6705%26doi%3D10.1021%2Facs.chemrev.8b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic regulators for cancer therapy: Mechanisms and advances in clinical trials</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">62</span>, <span class="refDoi">Â DOI: 10.1038/s41392-019-0095-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fs41392-019-0095-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31871779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3MbjsFCisQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=62&author=Y.+Chengauthor=C.+Heauthor=M.+Wangauthor=X.+Maauthor=F.+Moauthor=S.+Yangauthor=J.+Hanauthor=X.+Wei&title=Targeting+epigenetic+regulators+for+cancer+therapy%3A+Mechanisms+and+advances+in+clinical+trials&doi=10.1038%2Fs41392-019-0095-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic regulators for cancer therapy: mechanisms and advances in clinical trials</span></div><div class="casAuthors">Cheng Yuan; He Cai; Wang Manni; Ma Xuelei; Mo Fei; Yang Shengyong; Han Junhong; Wei Xiawei</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetic alternations concern heritable yet reversible changes in histone or DNA modifications that regulate gene activity beyond the underlying sequence.  Epigenetic dysregulation is often linked to human disease, notably cancer.  With the development of various drugs targeting epigenetic regulators, epigenetic-targeted therapy has been applied in the treatment of hematological malignancies and has exhibited viable therapeutic potential for solid tumors in preclinical and clinical trials.  In this review, we summarize the aberrant functions of enzymes in DNA methylation, histone acetylation and histone methylation during tumor progression and highlight the development of inhibitors of or drugs targeted at epigenetic enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRhk-ixsypfJ0Q0lWbKiCflfW6udTcc2eYxcha2Q8NGYrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbjsFCisQ%253D%253D&md5=b93a1408a20df29fea33c22487f4a074</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0095-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0095-0%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DX.%26atitle%3DTargeting%2520epigenetic%2520regulators%2520for%2520cancer%2520therapy%253A%2520Mechanisms%2520and%2520advances%2520in%2520clinical%2520trials%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D62%26doi%3D10.1038%2Fs41392-019-0095-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">107406</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2019.107406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.pharmthera.2019.107406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31521697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVOgsr3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2019&pages=107406&author=F.+Yeauthor=J.+Huangauthor=H.+Wangauthor=C.+Luoauthor=K.+Zhao&title=Targeting+epigenetic+machinery%3A+Emerging+novel+allosteric+inhibitors&doi=10.1016%2Fj.pharmthera.2019.107406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic machinery: Emerging novel allosteric inhibitors</span></div><div class="casAuthors">Ye, Fei; Huang, Jing; Wang, Hongbo; Luo, Cheng; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107406</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Epigenetics has emerged as an extremely exciting fast-growing area of biomedical research in post genome era.  Epigenetic dysfunction is tightly related with various diseases such as cancer and aging related degeneration, potentiating epigenetics modulators as important therapeutics targets.  Indeed, inhibitors of histone deacetylase and DNA methyltransferase have been approved for treating blood tumor malignancies, whereas inhibitors of histone methyltransferase and histone acetyl-lysine recognizer bromodomain are in clin. stage.  However, it remains a great challenge to discover potent and selective inhibitors by targeting catalytic site, as the same subfamily of epigenetic enzymes often share high sequence identity and very conserved catalytic core pocket.  It is well known that epigenetic modifications are usually carried out by multi-protein complexes, and activation of catalytic subunit is often tightly regulated by other interactive protein component, esp. in disease conditions.  Therefore, it is not unusual that epigenetic complex machinery may exhibit allosteric regulation site induced by protein-protein interactions.  Targeting allosteric site emerges as a compelling alternative strategy to develop epigenetic drugs with enhanced druggability and pharmacol. profiles.  In this review, we highlight recent progress in the development of allosteric inhibitors for epigenetic complexes through targeting protein-protein interactions.  We also summarized the status of clin. applications of those inhibitors.  Finally, we provide perspectives of future novel allosteric epigenetic machinery modulators emerging from otherwise undruggable single protein target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgODybjVaUN7Vg90H21EOLACvtfcHk0lieBM0ZmYwf3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVOgsr3I&md5=b799ad2dc59c390f7a15847a5d41ba56</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DTargeting%2520epigenetic%2520machinery%253A%2520Emerging%2520novel%2520allosteric%2520inhibitors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2019%26volume%3D204%26spage%3D107406%26doi%3D10.1016%2Fj.pharmthera.2019.107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: A success story for cofactor interference</span>. <i>Philos. Trans. R. Soc., B</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">20170069</span>, <span class="refDoi">Â DOI: 10.1098/rstb.2017.0069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1098%2Frstb.2017.0069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29685973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Cmsb7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2018&pages=20170069&author=A.+Ganesan&title=Epigenetic+drug+discovery%3A+A+success+story+for+cofactor+interference&doi=10.1098%2Frstb.2017.0069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: a success story for cofactor interference</span></div><div class="casAuthors">Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Philosophical Transactions of the Royal Society, B: Biological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">1748</span>),
    <span class="NLM_cas:pages">20170069/1-20170069/15</span>CODEN:
                <span class="NLM_cas:coden">PTRBAE</span>;
        ISSN:<span class="NLM_cas:issn">0962-8436</span>.
    
            (<span class="NLM_cas:orgname">Royal Society</span>)
        </div><div class="casAbstract">Within the past two decades, seven epigenetic drugs have received regulatory approval and numerous other candidates are currently in clin. trials.  Among the epigenetic targets are the writer and eraser enzymes that are, resp., responsible for the reversible introduction and removal of structural modifications in the nucleosome.  This reviewdiscusses the progress achieved in the design and development of inhibitors against the key writer and eraser pairs: DNA methyltransferases and Tet demethylases; lysine/arginine methyltransferases and lysine demethylases; and histone acetyltransferases and histone deacetylases.  A common theme for the successful inhibition of these enzymes in a potent and selective manner is the targeting of the cofactors present in the active site, namely zinc and iron cations, S-adenosylmethione, NAD, FAD and acetyl CoA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrohR1yq_2gzbVg90H21EOLACvtfcHk0lj-MSNc_axcBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Cmsb7M&md5=9c9e20f8b16a73a1835320e82698e68a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1098%2Frstb.2017.0069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frstb.2017.0069%26sid%3Dliteratum%253Aachs%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520A%2520success%2520story%2520for%2520cofactor%2520interference%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520B%26date%3D2018%26volume%3D373%26spage%3D20170069%26doi%3D10.1098%2Frstb.2017.0069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unnikrishnan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papaemmanuil, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrakanthan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoms, J. A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tursky, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulasekararaj, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobiasson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karimi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellagatti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramakrishna, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthur, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curnow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warburton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncolato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boultwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, S. E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mufti, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellstrom-Lindberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkins, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pimanda, J. E.</span></span> <span> </span><span class="NLM_article-title">Integrative genomics identifies the molecular basis of resistance to azacitidine therapy in myelodysplastic syndromes</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">572</span>â <span class="NLM_lpage">585</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2017.06.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.celrep.2017.06.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28723562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurzM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=572-585&author=A.+Unnikrishnanauthor=E.+Papaemmanuilauthor=D.+Beckauthor=N.+P.+Deshpandeauthor=A.+Vermaauthor=A.+Kumariauthor=P.+S.+Wollauthor=L.+A.+Richardsauthor=K.+Knezevicauthor=V.+Chandrakanthanauthor=J.+A.+I.+Thomsauthor=M.+L.+Turskyauthor=Y.+Huangauthor=Z.+Aliauthor=J.+Olivierauthor=S.+Galbraithauthor=A.+G.+Kulasekararajauthor=M.+Tobiassonauthor=M.+Karimiauthor=A.+Pellagattiauthor=S.+R.+Wilsonauthor=R.+Lindemanauthor=B.+Youngauthor=R.+Ramakrishnaauthor=C.+Arthurauthor=R.+Starkauthor=P.+Crispinauthor=J.+Curnowauthor=P.+Warburtonauthor=F.+Roncolatoauthor=J.+Boultwoodauthor=K.+Lynchauthor=S.+E.+W.+Jacobsenauthor=G.+J.+Muftiauthor=E.+Hellstrom-Lindbergauthor=M.+R.+Wilkinsauthor=K.+L.+MacKenzieauthor=J.+W.+H.+Wongauthor=P.+J.+Campbellauthor=J.+E.+Pimanda&title=Integrative+genomics+identifies+the+molecular+basis+of+resistance+to+azacitidine+therapy+in+myelodysplastic+syndromes&doi=10.1016%2Fj.celrep.2017.06.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative genomics identifies the molecular basis of resistance to azacitidine therapy in myelodysplastic syndromes</span></div><div class="casAuthors">Unnikrishnan, Ashwin; Papaemmanuil, Elli; Beck, Dominik; Deshpande, Nandan P.; Verma, Arjun; Kumari, Ashu; Woll, Petter S.; Richards, Laura A.; Knezevic, Kathy; Chandrakanthan, Vashe; Thoms, Julie A. I.; Tursky, Melinda L.; Huang, Yizhou; Ali, Zara; Olivier, Jake; Galbraith, Sally; Kulasekararaj, Austin G.; Tobiasson, Magnus; Karimi, Mohsen; Pellagatti, Andrea; Wilson, Susan R.; Lindeman, Robert; Young, Boris; Ramakrishna, Raj; Arthur, Christopher; Stark, Richard; Crispin, Philip; Curnow, Jennifer; Warburton, Pauline; Roncolato, Fernando; Boultwood, Jacqueline; Lynch, Kevin; Jacobsen, Sten Eirik W.; Mufti, Ghulam J.; Hellstrom-Lindberg, Eva; Wilkins, Marc R.; MacKenzie, Karen L.; Wong, Jason W. H.; Campbell, Peter J.; Pimanda, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">572-585</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Myelodysplastic syndromes and chronic myelomonocytic leukemia are blood disorders characterized by ineffective hematopoiesis and progressive marrow failure that can transform into acute leukemia.  The DNA methyltransferase inhibitor 5-azacytidine (AZA) is the most effective pharmacol. option, but only â¼50% of patients respond.  A response only manifests after many months of treatment and is transient.  The reasons underlying AZA resistance are unknown, and few alternatives exist for non-responders.  Here, we show that AZA responders have more hematopoietic progenitor cells (HPCs) in the cell cycle.  Non-responder HPC quiescence is mediated by integrin Î±5 (ITGA5) signaling and their hematopoietic potential improved by combining AZA with an ITGA5 inhibitor.  AZA response is assocd. with the induction of an inflammatory response in HPCs in vivo.  By mol. bar coding and tracking individual clones, we found that, although AZA alters the sub-clonal contribution to different lineages, founder clones are not eliminated and continue to drive hematopoiesis even in complete responders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi31PH3PgdpLVg90H21EOLACvtfcHk0lj-MSNc_axcBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurzM&md5=210500f9690bbf6683f02562b4ae0de3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.06.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.06.067%26sid%3Dliteratum%253Aachs%26aulast%3DUnnikrishnan%26aufirst%3DA.%26aulast%3DPapaemmanuil%26aufirst%3DE.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DDeshpande%26aufirst%3DN.%2BP.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DKumari%26aufirst%3DA.%26aulast%3DWoll%26aufirst%3DP.%2BS.%26aulast%3DRichards%26aufirst%3DL.%2BA.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DChandrakanthan%26aufirst%3DV.%26aulast%3DThoms%26aufirst%3DJ.%2BA.%2BI.%26aulast%3DTursky%26aufirst%3DM.%2BL.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DAli%26aufirst%3DZ.%26aulast%3DOlivier%26aufirst%3DJ.%26aulast%3DGalbraith%26aufirst%3DS.%26aulast%3DKulasekararaj%26aufirst%3DA.%2BG.%26aulast%3DTobiasson%26aufirst%3DM.%26aulast%3DKarimi%26aufirst%3DM.%26aulast%3DPellagatti%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DS.%2BR.%26aulast%3DLindeman%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DB.%26aulast%3DRamakrishna%26aufirst%3DR.%26aulast%3DArthur%26aufirst%3DC.%26aulast%3DStark%26aufirst%3DR.%26aulast%3DCrispin%26aufirst%3DP.%26aulast%3DCurnow%26aufirst%3DJ.%26aulast%3DWarburton%26aufirst%3DP.%26aulast%3DRoncolato%26aufirst%3DF.%26aulast%3DBoultwood%26aufirst%3DJ.%26aulast%3DLynch%26aufirst%3DK.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%2BW.%26aulast%3DMufti%26aufirst%3DG.%2BJ.%26aulast%3DHellstrom-Lindberg%26aufirst%3DE.%26aulast%3DWilkins%26aufirst%3DM.%2BR.%26aulast%3DMacKenzie%26aufirst%3DK.%2BL.%26aulast%3DWong%26aufirst%3DJ.%2BW.%2BH.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DPimanda%26aufirst%3DJ.%2BE.%26atitle%3DIntegrative%2520genomics%2520identifies%2520the%2520molecular%2520basis%2520of%2520resistance%2520to%2520azacitidine%2520therapy%2520in%2520myelodysplastic%2520syndromes%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D572%26epage%3D585%26doi%3D10.1016%2Fj.celrep.2017.06.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiotsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niimi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, S.</span></span> <span> </span><span class="NLM_article-title">Reduced expression of HDAC3 contributes to the resistance against HDAC inhibitor, Vorinostat (SAHA) in mature lymphoid malignancies</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">1342</span>, <span class="refDoi">Â DOI: 10.1182/blood.V120.21.1342.1342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1182%2Fblood.V120.21.1342.1342" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=1342&author=J.+Dingauthor=M.+Riauthor=T.+Naritaauthor=A.+Masakiauthor=F.+Moriauthor=A.+Itoauthor=S.+Kusumotoauthor=T.+Ishidaauthor=H.+Komatsuauthor=Y.+Shiotsuauthor=A.+Niimiauthor=S.+Iida&title=Reduced+expression+of+HDAC3+contributes+to+the+resistance+against+HDAC+inhibitor%2C+Vorinostat+%28SAHA%29+in+mature+lymphoid+malignancies&doi=10.1182%2Fblood.V120.21.1342.1342"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1182%2Fblood.V120.21.1342.1342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V120.21.1342.1342%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DRi%26aufirst%3DM.%26aulast%3DNarita%26aufirst%3DT.%26aulast%3DMasaki%26aufirst%3DA.%26aulast%3DMori%26aufirst%3DF.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DKusumoto%26aufirst%3DS.%26aulast%3DIshida%26aufirst%3DT.%26aulast%3DKomatsu%26aufirst%3DH.%26aulast%3DShiotsu%26aufirst%3DY.%26aulast%3DNiimi%26aufirst%3DA.%26aulast%3DIida%26aufirst%3DS.%26atitle%3DReduced%2520expression%2520of%2520HDAC3%2520contributes%2520to%2520the%2520resistance%2520against%2520HDAC%2520inhibitor%252C%2520Vorinostat%2520%2528SAHA%2529%2520in%2520mature%2520lymphoid%2520malignancies%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D1342%26doi%3D10.1182%2Fblood.V120.21.1342.1342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitors: The road ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>â <span class="NLM_lpage">377</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+inhibitors%3A+The+road+ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lhQwbBGebZukw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520inhibitors%253A%2520The%2520road%2520ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D377%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, A.</span></span> <span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: Balancing the benefits and risks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6243</span>â <span class="NLM_lpage">6262</span>, <span class="refDoi">Â DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+Balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0lhQwbBGebZukw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520Balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span> <span> </span><span class="NLM_article-title">Targeted covalent inhibitors for drug design</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">13408</span>â <span class="NLM_lpage">13421</span>, <span class="refDoi">Â DOI: 10.1002/anie.201601091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201601091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=13408-13421&author=T.+A.+Baillie&title=Targeted+covalent+inhibitors+for+drug+design&doi=10.1002%2Fanie.201601091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Covalent Inhibitors for Drug Design</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">13408-13421</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In contrast to the traditional mechanism of drug action that relies on the reversible, noncovalent interaction of a ligand with its biol. target, a targeted covalent inhibitor (TCI) is designed such that the initial, reversible assocn. is followed by the formation of a covalent bond between an electrophile on the ligand and a nucleophilic center in the protein.  Although this approach offers a variety of potential benefits (high potency and extended duration of action), concerns over the possible toxicol. consequences of protein haptenization have hindered the development of the TCI concept.  Recently, approaches to mitigate the risk of serious adverse reactions to this new class of agent have emerged, thus stimulating interest in the field and leading to authorization of the first cadre of TCIs to be marketed.  The covalent inhibitor approach is rapidly gaining acceptance as a valuable tool in drug discovery, and is poised to make a major impact on the design of enzyme inhibitors and receptor modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEbjdFrctvNrVg90H21EOLACvtfcHk0lhQwbBGebZukw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOrtb%252FN&md5=18bed2ae08dcb4245a42b3070abc6354</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fanie.201601091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201601091%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DTargeted%2520covalent%2520inhibitors%2520for%2520drug%2520design%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D13408%26epage%3D13421%26doi%3D10.1002%2Fanie.201601091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Cysteinome of protein kinases as a target in drug development</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4372</span>â <span class="NLM_lpage">4385</span>, <span class="refDoi">Â DOI: 10.1002/anie.201707875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201707875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4372-4385&author=A.+Chaikuadauthor=P.+Kochauthor=S.+A.+Lauferauthor=S.+Knapp&title=The+Cysteinome+of+protein+kinases+as+a+target+in+drug+development&doi=10.1002%2Fanie.201707875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The Cysteinome of Protein Kinases as a Target in Drug Development</span></div><div class="casAuthors">Chaikuad, Apirat; Koch, Pierre; Laufer, Stefan A.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4372-4385</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through covalent bond formation represent a considerable fraction of the authors' repository of effective medicines but safety concerns and the complexity of developing covalent inhibitors has rendered covalent targeting a less attractive strategy for rational drug design.  The recent approval of four covalent kinase inhibitors and the development of highly potent covalent kinase probes with exceptional selectivity has raised significant interest in industry and academic research and validated the concept of covalent kinase targeting for clin. applications.  The abundance of cysteines at diverse positions in and around the kinase active site suggests that a large fraction of kinases can be targeted by covalent inhibitors.  Herein, the authors review recent developments of this rapidly growing area in kinase drug development and highlight the unique opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphHGXsIr-dpbVg90H21EOLACvtfcHk0lhQwbBGebZukw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D&md5=311ba5ab53dc4a5dc1dc92ddd4b6d4c2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707875%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Cysteinome%2520of%2520protein%2520kinases%2520as%2520a%2520target%2520in%2520drug%2520development%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D4372%26epage%3D4385%26doi%3D10.1002%2Fanie.201707875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry strategies for the development of kinaseinhibitors targeting point mutations</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">10726</span>â <span class="NLM_lpage">10741</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00507</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00507" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpslymtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10726-10741&author=X.+Luauthor=J.+B.+Smaillauthor=K.+Ding&title=Medicinal+chemistry+strategies+for+the+development+of+kinaseinhibitors+targeting+point+mutations&doi=10.1021%2Facs.jmedchem.0c00507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal Chemistry Strategies for the Development of Kinase Inhibitors Targeting Point Mutations</span></div><div class="casAuthors">Lu, Xiaoyun; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10726-10741</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Clin. acquired resistance to small mol. kinase inhibitors (SMKIs) has become a major "unmet clin. need" in cancer therapy.  To date, there are six SMKIs to be approved for the treatment of cancer patients through targeting of clin. acquired resistance caused by on-target mutations.  These are mainly focused on the mutant kinases Bcr-Abl T315I, EGFR T790M, and ALK L1196M.  Herein, we summarize the major medicinal chem. strategies employed in the discovery of these representative SMKIs, such as avoiding steric hindrance, making addnl. interactions with mutated residues, and forming a covalent bond with an active site cysteine to override resistance obsd. for reversible inhibitors.  Addnl., we also briefly describe allosteric kinase inhibitors and proteolysis targeting chimera (PROTAC) as two other potential strategies while addressing future opportunities in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMa3W_5KO3W7Vg90H21EOLACvtfcHk0lgIt2WY9vzSIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpslymtr8%253D&md5=4e008f3c68b1442e89f79a5a1a2b3968</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00507%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DMedicinal%2520chemistry%2520strategies%2520for%2520the%2520development%2520of%2520kinaseinhibitors%2520targeting%2520point%2520mutations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D10726%26epage%3D10741%26doi%3D10.1021%2Facs.jmedchem.0c00507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">207</span>â <span class="NLM_lpage">218</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2005.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.cell.2005.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16239140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=207-218&author=A.+Yokoyamaauthor=T.+C.+Somervailleauthor=K.+S.+Smithauthor=O.+Rozenblatt-Rosenauthor=M.+Meyersonauthor=M.+L.+Cleary&title=The+menin+tumor+suppressor+protein+is+an+essential+oncogenic+cofactor+for+MLL-associated+leukemogenesis&doi=10.1016%2Fj.cell.2005.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span></div><div class="casAuthors">Yokoyama, Akihiko; Somervaille, Tim C. P.; Smith, Kevin S.; Rozenblatt-Rosen, Orit; Meyerson, Matthew; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-218</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Mixed-Lineage Leukemia (MLL) protein is a histone methyltransferase that is mutated in clin. and biol. distinctive subsets of acute leukemia.  MLL normally assocs. with a cohort of highly conserved cofactors to form a macromol. complex that includes menin, a product of the MEN1 tumor suppressor gene, which is mutated in heritable and sporadic endocrine tumors.  We demonstrate here that oncogenic MLL fusion proteins retain an ability to stably assoc. with menin through a high-affinity, amino-terminal, conserved binding motif and that this interaction is required for the initiation of MLL-mediated leukemogenesis.  Furthermore, menin is essential for maintenance of MLL-assocd. but not other oncogene induced myeloid transformation.  Acute genetic ablation of menin reverses aberrant Hox gene expression mediated by MLL-menin promoter-assocd. complexes, and specifically abrogates the differentiation arrest and oncogenic properties of MLL-transformed leukemic blasts.  These results demonstrate that a human oncoprotein is critically dependent on direct phys. interaction with a tumor suppressor protein for its oncogenic activity, validate a potential target for mol. therapy, and suggest central roles for menin in altered epigenetic functions underlying the pathogenesis of hematopoietic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MXmqsWp8BrVg90H21EOLACvtfcHk0lgIt2WY9vzSIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOqsbbP&md5=d582f614f98d25ebaf2282e2915fd4b6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26aulast%3DSmith%26aufirst%3DK.%2BS.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DThe%2520menin%2520tumor%2520suppressor%2520protein%2520is%2520an%2520essential%2520oncogenic%2520cofactor%2520for%2520MLL-associated%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D123%26spage%3D207%26epage%3D218%26doi%3D10.1016%2Fj.cell.2005.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">713</span>â <span class="NLM_lpage">719</span>, <span class="refDoi">Â DOI: 10.1038/nsmb1128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+methyltransferase+activity+by+its+core+components&doi=10.1038%2Fnsmb1128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0lgIt2WY9vzSIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520by%2520its%2520core%2520components%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719%26doi%3D10.1038%2Fnsmb1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">823</span>â <span class="NLM_lpage">833</span>, <span class="refDoi">Â DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0li71_egVnkp1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Challenges and opportunities in targeting the meninâMLL interaction</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">447</span>â <span class="NLM_lpage">462</span>, <span class="refDoi">Â DOI: 10.4155/fmc.13.214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.4155%2Ffmc.13.214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=24635524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=447-462&author=T.+Cierpickiauthor=J.+Grembecka&title=Challenges+and+opportunities+in+targeting+the+menin%E2%80%93MLL+interaction&doi=10.4155%2Ffmc.13.214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and opportunities in targeting the menin-MLL interaction</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-462</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Menin is an essential co-factor of oncogenic MLL fusion proteins and the menin-MLL interaction is crit. for development of acute leukemia in vivo.  Targeting the menin-MLL interaction with small mols. represents an attractive strategy to develop new anticancer agents.  Recent developments, including detn. of menin crystal structure and development of potent small mol. and peptidomimetic inhibitors, demonstrate the feasibility of targeting the menin-MLL interaction.  On the other hand, biochem. and structural studies revealed that MLL binds to menin in a complex bivalent mode engaging two MLL motifs, and therefore inhibition of this protein-protein interaction represents a challenge.  This review summarizes the most recent achievements in targeting the menin-MLL interaction as well as discusses potential benefits of blocking menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovLrSLXU5DNbVg90H21EOLACvtfcHk0li71_egVnkp1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D&md5=97777abe7b2fd8df3daf390d9b019721</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.214%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DChallenges%2520and%2520opportunities%2520in%2520targeting%2520the%2520menin%25E2%2580%2593MLL%2520interaction%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D447%26epage%3D462%26doi%3D10.4155%2Ffmc.13.214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of menin reveals binding site for mixed lineage leukemia (MLL) protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">31742</span>â <span class="NLM_lpage">31748</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.258186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.M111.258186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21757704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=31742-31748&author=M.+J.+Muraiauthor=M.+Chruszczauthor=G.+Reddyauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Crystal+structure+of+menin+reveals+binding+site+for+mixed+lineage+leukemia+%28MLL%29+protein&doi=10.1074%2Fjbc.M111.258186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Menin Reveals Binding Site for Mixed Lineage Leukemia (MLL) Protein</span></div><div class="casAuthors">Murai, Marcelo J.; Chruszcz, Maksymilian; Reddy, Gireesh; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">31742-31748, S31742/1-S31742/2</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Menin is a tumor suppressor protein that is encoded by the MEN1 (multiple endocrine neoplasia 1) gene and controls cell growth in endocrine tissues.  Importantly, menin also serves as a crit. oncogenic cofactor of MLL (mixed lineage leukemia) fusion proteins in acute leukemias.  Direct assocn. of menin with MLL fusion proteins is required for MLL fusion protein-mediated leukemogenesis in vivo, and this interaction has been validated as a new potential therapeutic target for development of novel anti-leukemia agents.  Here, we report the first crystal structure of menin homolog from Nematostella vectensis.  Due to a very high sequence similarity, the Nematostella menin is a close homolog of human menin, and these two proteins likely have very similar structures.  Menin is predominantly an Î±-helical protein with the protein core comprising three tetratricopeptide motifs that are flanked by two Î±-helical bundles and covered by a Î²-sheet motif.  A very interesting feature of menin structure is the presence of a large central cavity that is highly conserved between Nematostella and human menin.  By employing site-directed mutagenesis, we have demonstrated that this cavity constitutes the binding site for MLL.  Our data provide a structural basis for understanding the role of menin as a tumor suppressor protein and as an oncogenic co-factor of MLL fusion proteins.  It also provides essential structural information for development of inhibitors targeting the menin-MLL interaction as a novel therapeutic strategy in MLL-related leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZHi8naCLUvLVg90H21EOLACvtfcHk0li71_egVnkp1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur7P&md5=5c3e41d9d5c1fa4d65e7c8718638551a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.258186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.258186%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520menin%2520reveals%2520binding%2520site%2520for%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D31742%26epage%3D31748%26doi%3D10.1074%2Fjbc.M111.258186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of the mixed lineage leukemia-menin interaction: implications for targeting mixed lineage leukemias</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">40690</span>â <span class="NLM_lpage">40698</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M110.172783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.M110.172783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20961854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFynsLrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=40690-40698&author=J.+Grembeckaauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=T.+Cierpicki&title=Molecular+basis+of+the+mixed+lineage+leukemia-menin+interaction%3A+implications+for+targeting+mixed+lineage+leukemias&doi=10.1074%2Fjbc.M110.172783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of the mixed lineage leukemia-menin interaction: implications for targeting mixed lineage leukemias</span></div><div class="casAuthors">Grembecka, Jolanta; Belcher, Amalia M.; Hartley, Thomas; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">40690-40698</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chromosomal translocations targeting the mixed lineage leukemia (MLL) gene result in MLL fusion proteins that are found in aggressive human acute leukemias.  Disruption of MLL by such translocations leads to overexpression of Hox genes, resulting in a blockage of hematopoietic differentiation that ultimately leads to leukemia.  Menin, which directly binds MLL, has been identified as an essential oncogenic co-factor required for the leukemogenic activity of MLL fusion proteins.  Here, we characterize the mol. basis of the MLL-menin interaction.  Using 13C-detected NMR expts., we have mapped the residues within the intrinsically unstructured fragment of MLL that are required for binding to menin.  Interestingly, we found that MLL interacts with menin with a nanomolar affinity (Kd â¼ 10 nM) through two motifs, MBM1 and MBM2 (menin binding motifs 1 and 2).  These motifs are located within the N-terminal 43-amino acid fragment of MLL, and the MBM1 represents a high affinity binding motif.  Using alanine scanning mutagenesis of MBM1, we found that the hydrophobic residues Phe9, Pro10, and Pro13 are most crit. for binding.  Furthermore, based on exchange-transferred nuclear Overhauser effect measurements, we established that MBM1 binds to menin in an extended conformation.  In a series of competition expts. we showed that a peptide corresponding to MBM1 efficiently dissocs. the menin-MLL complex.  Altogether, our work establishes the mol. basis of the menin interaction with MLL and MLL fusion proteins and provides the necessary foundation for development of small mol. inhibitors targeting this interaction in leukemias with MLL translocations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO3pwLp7kEn7Vg90H21EOLACvtfcHk0lhiSJt8sVdFLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFynsLrL&md5=79d3199a45d392803791b89ff2dc8f7a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.172783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.172783%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMolecular%2520basis%2520of%2520the%2520mixed%2520lineage%2520leukemia-menin%2520interaction%253A%2520implications%2520for%2520targeting%2520mixed%2520lineage%2520leukemias%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D40690%26epage%3D40698%26doi%3D10.1074%2Fjbc.M110.172783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorenson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">277</span>â <span class="NLM_lpage">284</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnchembio.773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=22286128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=277-284&author=J.+Grembeckaauthor=S.+Heauthor=A.+Shiauthor=T.+Purohitauthor=A.+G.+Munteanauthor=R.+J.+Sorensonauthor=H.+D.+Showalterauthor=M.+J.+Muraiauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=J.+L.+Hessauthor=T.+Cierpicki&title=Menin-MLL+inhibitors+reverse+oncogenic+activity+of+MLL+fusion+proteins+in+leukemia&doi=10.1038%2Fnchembio.773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span></div><div class="casAuthors">Grembecka, Jolanta; He, Shihan; Shi, Aibin; Purohit, Trupta; Muntean, Andrew G.; Sorenson, Roderick J.; Showalter, Hollis D.; Murai, Marcelo J.; Belcher, Amalia M.; Hartley, Thomas; Hess, Jay L.; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Translocations involving the mixed lineage leukemia (MLL) gene result in human acute leukemias with very poor prognosis.  The leukemogenic activity of MLL fusion proteins is critically dependent on their direct interaction with menin, a product of the multiple endocrine neoplasia (MEN1) gene.  Here we present what are to our knowledge the first small-mol. inhibitors of the menin-MLL fusion protein interaction that specifically bind menin with nanomolar affinities.  These compds. effectively reverse MLL fusion protein-mediated leukemic transformation by downregulating the expression of target genes required for MLL fusion protein oncogenic activity.  They also selectively block proliferation and induce both apoptosis and differentiation of leukemia cells harboring MLL translocations.  Identification of these compds. provides a new tool for better understanding MLL-mediated leukemogenesis and represents a new approach for studying the role of menin as an oncogenic cofactor of MLL fusion proteins.  Our findings also highlight a new therapeutic strategy for aggressive leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_e6VYyS5l7Vg90H21EOLACvtfcHk0lhiSJt8sVdFLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D&md5=55eb9f353031dfef4c1a281b0ac28cba</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.773%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DA.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DSorenson%26aufirst%3DR.%2BJ.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMenin-MLL%2520inhibitors%2520reverse%2520oncogenic%2520activity%2520of%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D277%26epage%3D284%26doi%3D10.1038%2Fnchembio.773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of the bivalent Menin-MLL interaction by small molecules in leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4461</span>â <span class="NLM_lpage">4469</span>, <span class="refDoi">Â DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=22936661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&author=A.+Shiauthor=M.+J.+Muraiauthor=S.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+insights+into+inhibition+of+the+bivalent+Menin-MLL+interaction+by+small+molecules+in+leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span></div><div class="casAuthors">Shi, Aibin; Murai, Marcelo J.; He, Shihan; Lund, George; Hartley, Thomas; Purohit, Trupta; Reddy, Gireesh; Chruszcz, Maksymilian; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4461-4469</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Menin functions as a crit. oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in the development of acute leukemias, and inhibition of the menin interaction with MLL fusion proteins represents a very promising strategy to reverse their oncogenic activity.  MLL interacts with menin in a bivalent mode involving 2 N-terminal fragments of MLL.  In the present study, we reveal the first high-resoln. crystal structure of human menin in complex with a small-mol. inhibitor of the menin-MLL interaction, MI-2.  The structure shows that the compd. binds to the MLL pocket in menin and mimics the key interactions of MLL with menin.  Based on the menin-MI-2 structure, we developed MI-2-2, a compd. that binds to menin with low nanomolar affinity (Kd = 22nM) and very effectively disrupts the bivalent protein-protein interaction between menin and MLL.  MI-2-2 demonstrated specific and very pronounced activity in MLL leukemia cells, including inhibition of cell proliferation, down-regulation of Hoxa9 expression, and differentiation.  Our results provide the rational and essential structural basis to design next generation of inhibitors for effective targeting of the menin-MLL interaction in leukemia and demonstrate a proof of concept that inhibition of complex multivalent protein-protein interactions can be achieved by a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFC9mQS_4LVg90H21EOLACvtfcHk0lhiSJt8sVdFLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J&md5=c1ab14816cf5ea3590fe03b0ed405ea5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520the%2520bivalent%2520Menin-MLL%2520interaction%2520by%2520small%2520molecules%2520in%2520leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the Menin-MLL interaction blocks progression of MLL leukemia in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">589</span>â <span class="NLM_lpage">602</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2015.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2015.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25817203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=589-602&author=D.+Borkinauthor=S.+Heauthor=H.+Miaoauthor=K.+Kempinskaauthor=J.+Pollockauthor=J.+Chaseauthor=T.+Purohitauthor=B.+Malikauthor=T.+Zhaoauthor=J.+Wangauthor=B.+Wenauthor=H.+Zongauthor=M.+Jonesauthor=G.+Danet-Desnoyersauthor=M.+L.+Guzmanauthor=M.+Talpazauthor=D.+L.+Bixbyauthor=D.+Sunauthor=J.+L.+Hessauthor=A.+G.+Munteanauthor=I.+Maillardauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+Menin-MLL+interaction+blocks+progression+of+MLL+leukemia+in+vivo&doi=10.1016%2Fj.ccell.2015.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of the Menin-MLL Interaction Blocks Progression of MLL Leukemia In Vivo</span></div><div class="casAuthors">Borkin, Dmitry; He, Shihan; Miao, Hongzhi; Kempinska, Katarzyna; Pollock, Jonathan; Chase, Jennifer; Purohit, Trupta; Malik, Bhavna; Zhao, Ting; Wang, Jingya; Wen, Bo; Zong, Hongliang; Jones, Morgan; Danet-Desnoyers, Gwenn; Guzman, Monica L.; Talpaz, Moshe; Bixby, Dale L.; Sun, Duxin; Hess, Jay L.; Muntean, Andrew G.; Maillard, Ivan; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">589-602</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Chromosomal translocations affecting mixed lineage leukemia gene (MLL) result in acute leukemias resistant to therapy.  The leukemogenic activity of MLL fusion proteins is dependent on their interaction with menin, providing basis for therapeutic intervention.  Here we report the development of highly potent and orally bioavailable small-mol. inhibitors of the menin-MLL interaction, MI-463 and MI-503, and show their profound effects in MLL leukemia cells and substantial survival benefit in mouse models of MLL leukemia.  Finally, we demonstrate the efficacy of these compds. in primary samples derived from MLL leukemia patients.  Overall, we demonstrate that pharmacol. inhibition of the menin-MLL interaction represents an effective treatment for MLL leukemias in vivo and provide advanced mol. scaffold for clin. lead identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc2hESfSmR8bVg90H21EOLACvtfcHk0lgp0CAA6BdLhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOis7o%253D&md5=1369b1556d139e9006c3fa7d1add49a0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DChase%26aufirst%3DJ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZong%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DBixby%26aufirst%3DD.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DMaillard%26aufirst%3DI.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520Menin-MLL%2520interaction%2520blocks%2520progression%2520of%2520MLL%2520leukemia%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D589%26epage%3D602%26doi%3D10.1016%2Fj.ccell.2015.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malik, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asangani, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CieÅlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prensner, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escara-Wilke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stender, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niknafs, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palanisamy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunju, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yocum, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellacheruvu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nesvizhskii, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanasekaran, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the MLL complex in castration-resistant prostate cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">344</span>â <span class="NLM_lpage">352</span>, <span class="refDoi">Â DOI: 10.1038/nm.3830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnm.3830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25822367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=344-352&author=R.+Malikauthor=A.+P.+Khanauthor=I.+A.+Asanganiauthor=M.+Cie%C5%9Blikauthor=J.+R.+Prensnerauthor=X.+Wangauthor=M.+K.+Iyerauthor=X.+Jiangauthor=D.+Borkinauthor=J.+Escara-Wilkeauthor=R.+Stenderauthor=Y.+M.+Wuauthor=Y.+S.+Niknafsauthor=X.+Jingauthor=Y.+Qiaoauthor=N.+Palanisamyauthor=L.+P.+Kunjuauthor=P.+M.+Krishnamurthyauthor=A.+K.+Yocumauthor=D.+Mellacheruvuauthor=A.+I.+Nesvizhskiiauthor=X.+Caoauthor=S.+M.+Dhanasekaranauthor=F.+Y.+Fengauthor=J.+Grembeckaauthor=T.+Cierpickiauthor=A.+M.+Chinnaiyan&title=Targeting+the+MLL+complex+in+castration-resistant+prostate+cancer&doi=10.1038%2Fnm.3830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the MLL complex in castration-resistant prostate cancer</span></div><div class="casAuthors">Malik, Rohit; Khan, Amjad P.; Asangani, Irfan A.; Cieslik, Marcin; Prensner, John R.; Wang, Xiaoju; Iyer, Matthew K.; Jiang, Xia; Borkin, Dmitry; Escara-Wilke, June; Stender, Rachell; Wu, Yi-Mi; Niknafs, Yashar S.; Jing, Xiaojun; Qiao, Yuanyuan; Palanisamy, Nallasivam; Kunju, Lakshmi P.; Krishnamurthy, Pranathi M.; Yocum, Anastasia K.; Mellacheruvu, Dattatreya; Nesvizhskii, Alexey I.; Cao, Xuhong; Dhanasekaran, Saravana M.; Feng, Felix Y.; Grembecka, Jolanta; Cierpicki, Tomasz; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">344-352</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Resistance to androgen deprivation therapies and increased androgen receptor (AR) activity are major drivers of castration-resistant prostate cancer (CRPC).  Although prior work has focused on targeting AR directly, co-activators of AR signaling, which may represent new therapeutic targets, are relatively underexplored.  Here we demonstrate that the mixed-lineage leukemia protein (MLL) complex, a well-known driver of MLL fusion-pos. leukemia, acts as a co-activator of AR signaling.  AR directly interacts with the MLL complex via the menin-MLL subunit.  Menin expression is higher in CRPC than in both hormone-naive prostate cancer and benign prostate tissue, and high menin expression correlates with poor overall survival of individuals diagnosed with prostate cancer.  Treatment with a small-mol. inhibitor of menin-MLL interaction blocks AR signaling and inhibits the growth of castration-resistant tumors in vivo in mice.  Taken together, this work identifies the MLL complex as a crucial co-activator of AR and a potential therapeutic target in advanced prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmvjojfsVQ97Vg90H21EOLACvtfcHk0lgp0CAA6BdLhA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D&md5=31d39514cc6f6cf723146a70676f4021</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnm.3830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3830%26sid%3Dliteratum%253Aachs%26aulast%3DMalik%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DA.%2BP.%26aulast%3DAsangani%26aufirst%3DI.%2BA.%26aulast%3DCie%25C5%259Blik%26aufirst%3DM.%26aulast%3DPrensner%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DIyer%26aufirst%3DM.%2BK.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DEscara-Wilke%26aufirst%3DJ.%26aulast%3DStender%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DNiknafs%26aufirst%3DY.%2BS.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DPalanisamy%26aufirst%3DN.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DKrishnamurthy%26aufirst%3DP.%2BM.%26aulast%3DYocum%26aufirst%3DA.%2BK.%26aulast%3DMellacheruvu%26aufirst%3DD.%26aulast%3DNesvizhskii%26aufirst%3DA.%2BI.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DTargeting%2520the%2520MLL%2520complex%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D344%26epage%3D352%26doi%3D10.1038%2Fnm.3830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Property focused structure-based optimization of small molecule inhibitors of the protein-protein interaction between menin and mixed lineage leukemia (MLL)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">892</span>â <span class="NLM_lpage">913</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=892-913&author=D.+Borkinauthor=J.+Pollockauthor=K.+Kempinskaauthor=T.+Purohitauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=H.+Miaoauthor=S.+Shuklaauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Property+focused+structure-based+optimization+of+small+molecule+inhibitors+of+the+protein-protein+interaction+between+menin+and+mixed+lineage+leukemia+%28MLL%29&doi=10.1021%2Facs.jmedchem.5b01305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Property Focused Structure-Based Optimization of Small Molecule Inhibitors of the Protein-Protein Interaction between Menin and Mixed Lineage Leukemia (MLL)</span></div><div class="casAuthors">Borkin, Dmitry; Pollock, Jonathan; Kempinska, Katarzyna; Purohit, Trupta; Li, Xiaoqin; Wen, Bo; Zhao, Ting; Miao, Hongzhi; Shukla, Shirish; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-913</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of potent small mol. inhibitors of protein-protein interactions with optimized druglike properties represents a challenging task in lead optimization process.  Here, we report synthesis and structure-based optimization of new thienopyrimidine class of compds., which block the protein-protein interaction between menin and MLL fusion proteins that plays an important role in acute leukemias with MLL translocations.  We performed simultaneous optimization of both activity and druglike properties through systematic exploration of substituents introduced to the indole ring of lead compd. 1 (MI-136) to identify compds. suitable for in vivo studies in mice.  This work resulted in the identification of compd. 27 (MI-538), which showed significantly increased activity, selectivity, polarity, and pharmacokinetic profile over 1 and demonstrated a pronounced effect in a mouse model of MLL leukemia.  This study, which reports detailed structure-activity and structure-property relationships for the menin-MLL inhibitors, demonstrates challenges in optimizing inhibitors of protein-protein interactions for potential therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDD5t7XKKz_LVg90H21EOLACvtfcHk0ljf9mS8nGZatQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFersw%253D%253D&md5=f89b6eda72671f133a38181b3b23632d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01305%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DProperty%2520focused%2520structure-based%2520optimization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520protein-protein%2520interaction%2520between%2520menin%2520and%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D892%26epage%3D913%26doi%3D10.1021%2Facs.jmedchem.5b01305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klossowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linhares, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Complexity of blocking bivalent protein-protein interactions: Development of a highly Potent inhibitor of the menin-mixed-lineage leukemia interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4832</span>â <span class="NLM_lpage">4850</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4832-4850&author=D.+Borkinauthor=S.+Klossowskiauthor=J.+Pollockauthor=H.+Miaoauthor=B.+M.+Linharesauthor=K.+Kempinskaauthor=Z.+Jinauthor=T.+Purohitauthor=B.+Wenauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Complexity+of+blocking+bivalent+protein-protein+interactions%3A+Development+of+a+highly+Potent+inhibitor+of+the+menin-mixed-lineage+leukemia+interaction&doi=10.1021%2Facs.jmedchem.8b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Complexity of Blocking Bivalent Protein-Protein Interactions: Development of a Highly Potent Inhibitor of the Menin-Mixed-Lineage Leukemia Interaction</span></div><div class="casAuthors">Borkin, Dmitry; Klossowski, Szymon; Pollock, Jonathan; Miao, Hongzhi; Linhares, Brian M.; Kempinska, Katarzyna; Jin, Zhuang; Purohit, Trupta; Wen, Bo; He, Miao; Sun, Duxin; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4832-4850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors.  Here, we report the development of a new generation of menin-MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compds.  This work resulted in compd. I (MI-1481), which showed very potent inhibition of the menin-MLL1 interaction (IC50 = 3.6 nM), representing the most potent reversible menin-MLL1 inhibitor reported to date.  The crystal structure of the menin-I complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of I.  Compd. I also demonstrates pronounced activity in MLL leukemia cells and in vivo in MLL leukemia models.  Thus, I is a valuable menin-MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_eephZ2UHnLVg90H21EOLACvtfcHk0ljf9mS8nGZatQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1Wkurs%253D&md5=8d12fa816af3a5aa191694d4bfe48802</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00071%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DKlossowski%26aufirst%3DS.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DLinhares%26aufirst%3DB.%2BM.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DComplexity%2520of%2520blocking%2520bivalent%2520protein-protein%2520interactions%253A%2520Development%2520of%2520a%2520highly%2520Potent%2520inhibitor%2520of%2520the%2520menin-mixed-lineage%2520leukemia%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4832%26epage%3D4850%26doi%3D10.1021%2Facs.jmedchem.8b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadrey, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eschle, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uckelmann, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braytee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chacon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claremon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gifford, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teicher, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pimanda, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGeehan, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lock, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">A meninâMLL inhibitor induces specific chromatin changes and eradicates disease in models of MLL-rearranged leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">660</span>â <span class="NLM_lpage">673</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2019.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2019.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31821784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOmtLrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=660-673&author=A.+V.+Krivtsovauthor=K.+Evansauthor=J.+Y.+Gadreyauthor=B.+K.+Eschleauthor=C.+Hattonauthor=H.+J.+Uckelmannauthor=K.+N.+Rossauthor=F.+Pernerauthor=S.+N.+Olsenauthor=T.+Pritchardauthor=L.+McDermottauthor=C.+D.+Jonesauthor=D.+Jingauthor=A.+Brayteeauthor=D.+Chaconauthor=E.+Earleyauthor=B.+M.+McKeeverauthor=D.+Claremonauthor=A.+J.+Giffordauthor=H.+J.+Leeauthor=B.+A.+Teicherauthor=J.+E.+Pimandaauthor=D.+Beckauthor=J.+A.+Perryauthor=M.+A.+Smithauthor=G.+M.+McGeehanauthor=R.+B.+Lockauthor=S.+A.+Armstrong&title=A+menin%E2%80%93MLL+inhibitor+induces+specific+chromatin+changes+and+eradicates+disease+in+models+of+MLL-rearranged+leukemia&doi=10.1016%2Fj.ccell.2019.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Menin-MLL Inhibitor Induces Specific Chromatin Changes and Eradicates Disease in Models of MLL-Rearranged Leukemia</span></div><div class="casAuthors">Krivtsov, Andrei V.; Evans, Kathryn; Gadrey, Jayant Y.; Eschle, Benjamin K.; Hatton, Charlie; Uckelmann, Hannah J.; Ross, Kenneth N.; Perner, Florian; Olsen, Sarah N.; Pritchard, Tara; McDermott, Lisa; Jones, Connor D.; Jing, Duohui; Braytee, Ali; Chacon, Diego; Earley, Eric; McKeever, Brian M.; Claremon, David; Gifford, Andrew J.; Lee, Heather J.; Teicher, Beverly A.; Pimanda, John E.; Beck, Dominik; Perry, Jennifer A.; Smith, Malcolm A.; McGeehan, Gerard M.; Lock, Richard B.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">660-673.e11</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Inhibition of the Menin (MEN1) and MLL (MLL1, KMT2A) interaction is a potential therapeutic strategy for MLL-rearranged (MLL-r) leukemia.  Structure-based design yielded the potent, highly selective, and orally bioavailable small-mol. inhibitor VTP50469.  Cell lines carrying MLL rearrangements were selectively responsive to VTP50469.  VTP50469 displaced Menin from protein complexes and inhibited chromatin occupancy of MLL at select genes.  Loss of MLL binding led to changes in gene expression, differentiation, and apoptosis.  Patient-derived xenograft (PDX) models derived from patients with either MLL-r acute myeloid leukemia or MLL-r acute lymphoblastic leukemia (ALL) showed dramatic redns. of leukemia burden when treated with VTP50469.  Multiple mice engrafted with MLL-r ALL remained disease free for more than 1 yr after treatment.  These data support rapid translation of this approach to clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgvJuHyOgm4LVg90H21EOLACvtfcHk0ljf9mS8nGZatQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOmtLrN&md5=31046dca120487279d1b9bd66d154c97</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DEvans%26aufirst%3DK.%26aulast%3DGadrey%26aufirst%3DJ.%2BY.%26aulast%3DEschle%26aufirst%3DB.%2BK.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DUckelmann%26aufirst%3DH.%2BJ.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DPerner%26aufirst%3DF.%26aulast%3DOlsen%26aufirst%3DS.%2BN.%26aulast%3DPritchard%26aufirst%3DT.%26aulast%3DMcDermott%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DC.%2BD.%26aulast%3DJing%26aufirst%3DD.%26aulast%3DBraytee%26aufirst%3DA.%26aulast%3DChacon%26aufirst%3DD.%26aulast%3DEarley%26aufirst%3DE.%26aulast%3DMcKeever%26aufirst%3DB.%2BM.%26aulast%3DClaremon%26aufirst%3DD.%26aulast%3DGifford%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA.%26aulast%3DPimanda%26aufirst%3DJ.%2BE.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DMcGeehan%26aufirst%3DG.%2BM.%26aulast%3DLock%26aufirst%3DR.%2BB.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DA%2520menin%25E2%2580%2593MLL%2520inhibitor%2520induces%2520specific%2520chromatin%2520changes%2520and%2520eradicates%2520disease%2520in%2520models%2520of%2520MLL-rearranged%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D660%26epage%3D673%26doi%3D10.1016%2Fj.ccell.2019.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">High-affinity small-molecule inhibitors of the menin-mixed lineage leukemia (MLL) interaction closely mimic a natural protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1543</span>â <span class="NLM_lpage">1556</span>, <span class="refDoi">Â DOI: 10.1021/jm401868d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401868d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1543-1556&author=S.+Heauthor=T.+J.+Senterauthor=J.+Pollockauthor=C.+Hanauthor=S.+K.+Upadhyayauthor=T.+Purohitauthor=R.+D.+Gogliottiauthor=C.+W.+Lindsleyauthor=T.+Cierpickiauthor=S.+R.+Staufferauthor=J.+Grembecka&title=High-affinity+small-molecule+inhibitors+of+the+menin-mixed+lineage+leukemia+%28MLL%29+interaction+closely+mimic+a+natural+protein-protein+interaction&doi=10.1021%2Fjm401868d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Small-Molecule Inhibitors of the Menin-Mixed Lineage Leukemia (MLL) Interaction Closely Mimic a Natural Protein-Protein Interaction</span></div><div class="casAuthors">He, Shihan; Senter, Timothy J.; Pollock, Jonathan; Han, Changho; Upadhyay, Sunil Kumar; Purohit, Trupta; Gogliotti, Rocco D.; Lindsley, Craig W.; Cierpicki, Tomasz; Stauffer, Shaun R.; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1543-1556</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction (PPI) between menin and mixed lineage leukemia (MLL) plays a crit. role in acute leukemias, and inhibition of this interaction represents a new potential therapeutic strategy for MLL leukemias.  We report development of a novel class of small-mol. inhibitors of the menin-MLL interaction, the hydroxy- and aminomethylpiperidine compds., which originated from HTS of â¼288000 small mols.  We detd. menin-inhibitor co-crystal structures and found that these compds. closely mimic all key interactions of MLL with menin.  Extensive crystallog. studies combined with structure-based design were applied for optimization of these compds., resulting in MIV-6R, which inhibits the menin-MLL interaction with IC50 = 56 nM.  Treatment with MIV-6 demonstrated strong and selective effects in MLL leukemia cells, validating specific mechanism of action.  Our studies provide novel and attractive scaffold as a new potential therapeutic approach for MLL leukemias and demonstrate an example of PPI amenable to inhibition by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNC4yHqMSn87Vg90H21EOLACvtfcHk0ljJmQDCromDlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCmsrw%253D&md5=a580d1b2ea3f657c4abb31678f39548c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm401868d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401868d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSenter%26aufirst%3DT.%2BJ.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DS.%2BK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DHigh-affinity%2520small-molecule%2520inhibitors%2520of%2520the%2520menin-mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520interaction%2520closely%2520mimic%2520a%2520natural%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1543%26epage%3D1556%26doi%3D10.1021%2Fjm401868d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FernÃ¡ndez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the first-in-class, highly potent irreversible inhibitor targeting the Menin-MLL protein-protein interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1601</span>â <span class="NLM_lpage">1605</span>, <span class="refDoi">Â DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fern%C3%A1ndez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+first-in-class%2C+highly+potent+irreversible+inhibitor+targeting+the+Menin-MLL+protein-protein+interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Xu, Tianfeng; Zheng, Ke; Huang, Liyue; Stuckey, Jeanne; Chinnaswamy, Krishnapriya; Bernard, Denzil; Fernandez-Salas, Ester; Liu, Liu; Wang, Mi; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1605</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The structure-based design of M-525 (I) as the first-in-class, highly potent, irreversible small-mol. inhibitor of the menin-MLL interaction is presented.  M-525 targets cellular menin protein at sub-nanomolar concns. and achieves low nanomolar potencies in cell growth inhibition and in the suppression of MLL-regulated gene expression in MLL leukemia cells.  M-525 demonstrates high cellular specificity over non-MLL leukemia cells and is more than 30 times more potent than its corresponding reversible inhibitors.  Mass spectrometric anal. and co-crystal structure of M-525 in complex with menin firmly establish its mode of action.  A single administration of M-525 effectively suppresses MLL-regulated gene expression in tumor tissue.  An efficient procedure was developed to synthesize M-525.  This study demonstrates that irreversible inhibition of menin may be a promising therapeutic strategy for MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fbcCaeUFsLVg90H21EOLACvtfcHk0ljJmQDCromDlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D&md5=f7d189e2138b88a4d8c21c72586b7502</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFern%25C3%25A1ndez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520first-in-class%252C%2520highly%2520potent%2520irreversible%2520inhibitor%2520targeting%2520the%2520Menin-MLL%2520protein-protein%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawacki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of M-808 as a highly potent, covalent, small-molecule inhibitor of the Menin-MLL interaction with strong <i>in vivo</i> antitumor activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4997</span>â <span class="NLM_lpage">5010</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVClsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4997-5010&author=S.+Xuauthor=A.+Aguilarauthor=L.+Huangauthor=T.+Xuauthor=K.+Zhengauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=K.+Zawackiauthor=Z.+Liuauthor=K.+Chinnaswamyauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+M-808+as+a+highly+potent%2C+covalent%2C+small-molecule+inhibitor+of+the+Menin-MLL+interaction+with+strong+in+vivo+antitumor+activity&doi=10.1021%2Facs.jmedchem.0c00547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of M-808 as a Highly Potent, Covalent, Small-Molecule Inhibitor of the Menin-MLL Interaction with Strong In Vivo Antitumor Activity</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Huang, Liyue; Xu, Tianfeng; Zheng, Ke; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Zawacki, Kaitlin; Liu, Zhaomin; Chinnaswamy, Krishnapriya; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4997-5010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting the menin-MLL protein-protein interaction is a new therapeutic strategy for the treatment of acute leukemia carrying MLL fusion (MLL leukemia).  We describe herein the structure-based optimization of a class of covalent menin inhibitors, which led to the discovery of M-808 (16)(I) as a highly potent and efficacious covalent menin inhibitor.  M-808 effectively inhibits leukemia cell growth at low nanomolar concns. and is capable of achieving partial tumor regression in an MV4;11 xenograft tumor model in mice at a well-tolerated dose schedule.  Detn. of the co-crystal structure of M-808 in complex with menin provides a structural basis for their high-affinity, covalent interactions.  M-808 represents a promising, covalent menin inhibitor for further optimization and evaluation toward developing a new therapy for the treatment of MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCh0XqIp3vpLVg90H21EOLACvtfcHk0lgzcbo_LE4LRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVClsrg%253D&md5=168b2c21b9222ee72120f6680e7f60db</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00547%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DZawacki%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520M-808%2520as%2520a%2520highly%2520potent%252C%2520covalent%252C%2520small-molecule%2520inhibitor%2520of%2520the%2520Menin-MLL%2520interaction%2520with%2520strong%2520in%2520vivo%2520antitumor%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4997%26epage%3D5010%26doi%3D10.1021%2Facs.jmedchem.0c00547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congdon, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokomori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J. C.</span></span> <span> </span><span class="NLM_article-title">Dynamic regulation of the PR-Set7 histone methyltransferase is required for normal cell cycle progression</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2531</span>â <span class="NLM_lpage">2542</span>, <span class="refDoi">Â DOI: 10.1101/gad.1984210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1101%2Fgad.1984210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20966048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSgu77O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2531-2542&author=S.+Wuauthor=W.+Wangauthor=X.+Kongauthor=L.+M.+Congdonauthor=K.+Yokomoriauthor=M.+W.+Kirschnerauthor=J.+C.+Rice&title=Dynamic+regulation+of+the+PR-Set7+histone+methyltransferase+is+required+for+normal+cell+cycle+progression&doi=10.1101%2Fgad.1984210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic regulation of the PR-Set7 histone methyltransferase is required for normal cell cycle progression</span></div><div class="casAuthors">Wu, Shumin; Wang, Weiping; Kong, Xiangduo; Congdon, Lauren M.; Yokomori, Kyoko; Kirschner, Marc W.; Rice, Judd C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2531-2542</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Although the PR-Set7/Set8/KMT5a histone H4 Lys 20 monomethyltransferase (H4K20me1) plays an essential role in mammalian cell cycle progression, esp. during G2/M, it remained unknown how PR-Set7 itself was regulated.  In this study, we discovered the mechanisms that govern the dynamic regulation of PR-Set7 during mitosis, and that perturbation of these pathways results in defective mitotic progression.  First, we found that PR-Set7 is phosphorylated at Ser 29 (S29) specifically by the cyclin-dependent kinase 1 (cdk1)/cyclinB complex, primarily from prophase through early anaphase, subsequent to global accumulation of H4K20me1.  While S29 phosphorylation did not affect PR-Set7 methyltransferase activity, this event resulted in the removal of PR-Set7 from mitotic chromosomes.  S29 phosphorylation also functions to stabilize PR-Set7 by directly inhibiting its interaction with the anaphase-promoting complex (APC), an E3 ubiquitin ligase.  The dephosphorylation of S29 during late mitosis by the Cdc14 phosphatases was required for APCcdh1-mediated ubiquitination of PR-Set7 and subsequent proteolysis.  This event is important for proper mitotic progression, as constitutive phosphorylation of PR-Set7 resulted in a substantial delay between metaphase and anaphase.  Collectively, we elucidated the mol. mechanisms that control PR-Set7 protein levels during mitosis, and demonstrated that its orchestrated regulation is important for normal mitotic progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV3w-MmV6YIrVg90H21EOLACvtfcHk0lgzcbo_LE4LRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSgu77O&md5=4094cec7d8bdca00a92072ab4b99999d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1101%2Fgad.1984210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1984210%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DCongdon%26aufirst%3DL.%2BM.%26aulast%3DYokomori%26aufirst%3DK.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26atitle%3DDynamic%2520regulation%2520of%2520the%2520PR-Set7%2520histone%2520methyltransferase%2520is%2520required%2520for%2520normal%2520cell%2520cycle%2520progression%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2531%26epage%3D2542%26doi%3D10.1101%2Fgad.1984210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saloura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span> <span> </span><span class="NLM_article-title">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">110</span>â <span class="NLM_lpage">124</span>, <span class="refDoi">Â DOI: 10.1038/nrc3884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrc3884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25614009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=110-124&author=R.+Hamamotoauthor=V.+Salouraauthor=Y.+Nakamura&title=Critical+roles+of+non-histone+protein+lysine+methylation+in+human+tumorigenesis&doi=10.1038%2Fnrc3884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span></div><div class="casAuthors">Hamamoto, Ryuji; Saloura, Vassiliki; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-124</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several protein lysine methyltransferases and demethylases have been identified to have crit. roles in histone modification.  A large body of evidence has indicated that their dysregulation is involved in the development and progression of various diseases, including cancer, and these enzymes are now considered to be potential therapeutic targets.  Although most studies have focused on histone methylation, many reports have revealed that these enzymes also regulate the methylation dynamics of non-histone proteins such as p53, RB1 and STAT3 (signal transducer and activator of transcription 3), which have important roles in human tumorigenesis.  In this Review, we summarize the mol. functions of protein lysine methylation and its involvement in human cancer, with a particular focus on lysine methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0wn_Jk7jSGbVg90H21EOLACvtfcHk0lgzcbo_LE4LRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D&md5=9d998a4efb3c1a069e14747341ceca5e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrc3884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3884%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DSaloura%26aufirst%3DV.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DCritical%2520roles%2520of%2520non-histone%2520protein%2520lysine%2520methylation%2520in%2520human%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D110%26epage%3D124%26doi%3D10.1038%2Fnrc3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kachirskaia, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appella, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">Modulation of p53 function by SET8-mediated methylation at lysine 382</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">636</span>â <span class="NLM_lpage">646</span>, <span class="refDoi">Â DOI: 10.1016/j.molcel.2007.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.molcel.2007.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=17707234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=636-646&author=X.+Shiauthor=I.+Kachirskaiaauthor=H.+Yamaguchiauthor=L.+E.+Westauthor=H.+Wenauthor=E.+W.+Wangauthor=S.+Duttaauthor=E.+Appellaauthor=O.+Gozani&title=Modulation+of+p53+function+by+SET8-mediated+methylation+at+lysine+382&doi=10.1016%2Fj.molcel.2007.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of p53 function by SET8-mediated methylation at lysine 382</span></div><div class="casAuthors">Shi, Xiaobing; Kachirskaia, Ioulia; Yamaguchi, Hiroshi; West, Lisandra E.; Wen, Hong; Wang, Evelyn W.; Dutta, Sucharita; Appella, Ettore; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">636-646</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Reversible covalent methylation of lysine residues on histone proteins constitutes a principal mol. mechanism that links chromatin states to diverse biol. outcomes.  Recently, lysine methylation has been obsd. on non-histone proteins, suggesting broad cellular roles for the enzymes generating and removing Me moieties.  Here we report that the lysine methyltransferase enzyme SET8/PR-Set7 regulates the tumor suppressor protein p53.  We find that SET8 specifically monomethylates p53 at lysine 382 (p53K382me1).  This methylation event robustly suppresses p53-mediated transcription activation of highly responsive target genes but has little influence on weak targets.  Further, depletion of SET8 augments the proapoptotic and checkpoint activation functions of p53, and accordingly, SET8 expression is downregulated upon DNA damage.  Together, our study identifies SET8 as a p53-modifying enzyme, identifies p53K382me1 as a regulatory posttranslational modification of p53, and begins to dissect how methylation may contribute to a dynamic posttranslational code that modulates distinct p53 functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR93io05Whx7Vg90H21EOLACvtfcHk0li1bUeQwlye9g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtL3E&md5=df158100369f875f859a4fdf296ac2fc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DX.%26aulast%3DKachirskaia%26aufirst%3DI.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DE.%2BW.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DModulation%2520of%2520p53%2520function%2520by%2520SET8-mediated%2520methylation%2520at%2520lysine%2520382%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D636%26epage%3D646%26doi%3D10.1016%2Fj.molcel.2007.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyokawa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maejima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dohmae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Field, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsunoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akasu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnuma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponder, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span> <span> </span><span class="NLM_article-title">Histone lysine methyltransferase SETD8 promotes carcinogenesis by deregulating PCNA expression</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3217</span>â <span class="NLM_lpage">3227</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-11-3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F0008-5472.CAN-11-3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=22556262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsFKktbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=3217-3227&author=M.+Takawaauthor=H.+S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=M.+Kogureauthor=Y.+Yamaneauthor=Y.+Iwaiauthor=K.+Maejimaauthor=K.+Uedaauthor=A.+Masudaauthor=N.+Dohmaeauthor=H.+I.+Fieldauthor=T.+Tsunodaauthor=T.+Kobayashiauthor=T.+Akasuauthor=M.+Sugiyamaauthor=S.+Ohnumaauthor=Y.+Atomiauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Histone+lysine+methyltransferase+SETD8+promotes+carcinogenesis+by+deregulating+PCNA+expression&doi=10.1158%2F0008-5472.CAN-11-3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methyltransferase SETD8 Promotes Carcinogenesis by Deregulating PCNA Expression</span></div><div class="casAuthors">Takawa, Masashi; Cho, Hyun-Soo; Hayami, Shinya; Toyokawa, Gouji; Kogure, Masaharu; Yamane, Yuka; Iwai, Yukiko; Maejima, Kazuhiro; Ueda, Koji; Masuda, Akiko; Dohmae, Naoshi; Field, Helen I.; Tsunoda, Tatsuhiko; Kobayashi, Takaaki; Akasu, Takayuki; Sugiyama, Masanori; Ohnuma, Shin-ichi; Atomi, Yutaka; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3217-3227</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although the physiol. significance of lysine methylation of histones is well known, whether lysine methylation plays a role in the regulation of nonhistone proteins has not yet been examd.  The histone lysine methyltransferase SETD8 is overexpressed in various types of cancer and seems to play a crucial role in S-phase progression.  Here, we show that SETD8 regulates the function of proliferating cell nuclear antigen (PCNA) protein through lysine methylation.  We found that SETD8 methylated PCNA on lysine 248, and either depletion of SETD8 or substitution of lysine 248 destabilized PCNA expression.  Mechanistically, lysine methylation significantly enhanced the interaction between PCNA and the flap endonuclease FEN1.  Loss of PCNA methylation retarded the maturation of Okazaki fragments, slowed DNA replication, and induced DNA damage, and cells expressing a methylation-inactive PCNA mutant were more susceptible to DNA damage.  An increase of methylated PCNA was found in cancer cells, and the expression levels of SETD8 and PCNA were correlated in cancer tissue samples.  Together, our findings reveal a function for lysine methylation on a nonhistone protein and suggest that aberrant lysine methylation of PCNA may play a role in human carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor3s_ED_aVQLVg90H21EOLACvtfcHk0li1bUeQwlye9g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsFKktbo%253D&md5=d1d7e9969329b06ea655c76b3fb93013</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-3701%26sid%3Dliteratum%253Aachs%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DAkasu%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DOhnuma%26aufirst%3DS.%26aulast%3DAtomi%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DHistone%2520lysine%2520methyltransferase%2520SETD8%2520promotes%2520carcinogenesis%2520by%2520deregulating%2520PCNA%2520expression%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D3217%26epage%3D3227%26doi%3D10.1158%2F0008-5472.CAN-11-3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milite, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rescigno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Progress in the development of lysine methyltransferase SETD8 inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1680</span>â <span class="NLM_lpage">1685</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201600272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fcmdc.201600272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27411844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKktrzF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1680-1685&author=C.+Militeauthor=A.+Feoliauthor=M.+Vivianoauthor=D.+Rescignoauthor=A.+Maiauthor=S.+Castellanoauthor=G.+Sbardella&title=Progress+in+the+development+of+lysine+methyltransferase+SETD8+inhibitors&doi=10.1002%2Fcmdc.201600272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the development of lysine methyltransferase SETD8 inhibitors</span></div><div class="casAuthors">Milite, Ciro; Feoli, Alessandra; Viviano, Monica; Rescigno, Donatella; Mai, Antonello; Castellano, Sabrina; Sbardella, Gianluca</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1680-1685</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  SETD8/SET8/Pr-SET7/KMT5A is the only known lysine methyltransferase that monomethylates lysine 20 of histone H4 (H4K20) in vivo.  The methyltransferase activity of SETD8 has been implicated in many essential cellular processes, including DNA replication, DNA damage response, transcription modulation, and cell cycle regulation.  In addn. to H4K20, SETD8 monomethylates non-histone substrates including proliferating cell nuclear antigen and p53.  During the past decade, different structural classes of inhibitors targeting various lysine methyltransferases have been designed and developed.  However, the development of SETD8 inhibitors is still in its infancy.  This review covers the progress made to date in inhibiting the activity of SETD8 by small mols., with an emphasis on their discovery, selectivity over other methyltransferases, and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqDsZ5YVk8ybVg90H21EOLACvtfcHk0li1bUeQwlye9g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKktrzF&md5=0e952b0788e4df4c4bd3626a808a4f66</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600272%26sid%3Dliteratum%253Aachs%26aulast%3DMilite%26aufirst%3DC.%26aulast%3DFeoli%26aufirst%3DA.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DRescigno%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DProgress%2520in%2520the%2520development%2520of%2520lysine%2520methyltransferase%2520SETD8%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D1680%26epage%3D1685%26doi%3D10.1002%2Fcmdc.201600272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milite, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rescigno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzano, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">The emerging role of lysine methyltransferase SETD8 in human diseases</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">102</span>, <span class="refDoi">Â DOI: 10.1186/s13148-016-0268-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1186%2Fs13148-016-0268-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27688818" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=102&author=C.+Militeauthor=A.+Feoliauthor=M.+Vivianoauthor=D.+Rescignoauthor=A.+Cianciulliauthor=A.+L.+Balzanoauthor=A.+Maiauthor=S.+Castellanoauthor=G.+Sbardella&title=The+emerging+role+of+lysine+methyltransferase+SETD8+in+human+diseases&doi=10.1186%2Fs13148-016-0268-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1186%2Fs13148-016-0268-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-016-0268-4%26sid%3Dliteratum%253Aachs%26aulast%3DMilite%26aufirst%3DC.%26aulast%3DFeoli%26aufirst%3DA.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DRescigno%26aufirst%3DD.%26aulast%3DCianciulli%26aufirst%3DA.%26aulast%3DBalzano%26aufirst%3DA.%2BL.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DThe%2520emerging%2520role%2520of%2520lysine%2520methyltransferase%2520SETD8%2520in%2520human%2520diseases%26jtitle%3DClin.%2520Epigenet.%26date%3D2016%26volume%3D8%26spage%3D102%26doi%3D10.1186%2Fs13148-016-0268-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalisay, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amphlett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maneerat, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matainaho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, R. J.</span></span> <span> </span><span class="NLM_article-title">Nahuoic acid a produced by a streptomyces sp. isolated from a marine sediment is a selective SAM-competitive inhibitor of the histone methyltransferase SETD8</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">414</span>â <span class="NLM_lpage">417</span>, <span class="refDoi">Â DOI: 10.1021/ol303416k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol303416k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFej" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=414-417&author=D.+E.+Williamsauthor=D.+S.+Dalisayauthor=F.+Liauthor=J.+Amphlettauthor=W.+Maneeratauthor=M.+A.+Chavezauthor=Y.+A.+Wangauthor=T.+Matainahoauthor=W.+Yuauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=R.+J.+Andersen&title=Nahuoic+acid+a+produced+by+a+streptomyces+sp.+isolated+from+a+marine+sediment+is+a+selective+SAM-competitive+inhibitor+of+the+histone+methyltransferase+SETD8&doi=10.1021%2Fol303416k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Nahuoic acid A produced by a Streptomyces sp. isolated from a marine sediment is a selective SAM-competitive inhibitor of the histone methyltransferase SETD8</span></div><div class="casAuthors">Williams, David E.; Dalisay, Doralyn S.; Li, Fengling; Amphlett, James; Maneerat, Wisanu; Chavez, Miguel Angel Garcia; Wang, Yan Alexander; Matainaho, Teatulohi; Yu, Wenyu; Brown, Peter J.; Arrowsmith, Cheryl H.; Vedadi, Masoud; Andersen, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">414-417</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone lysine monomethyltransferase SETD8 is an epigenetic regulator of cell cycle progression.  Nahuoic acid A (I), a polyketide produced in culture by a Streptomyces sp. obtained from a tropical marine sediment, is the first known selective SAM-competitive inhibitor of SETD8.  The structure of I was elucidated by chem. transformation and detailed anal. of spectroscopic data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLuQ0zAEbeO7Vg90H21EOLACvtfcHk0li7N70dHdkarA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFej&md5=acf39ac06135084b15b7fbeedf23a202</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fol303416k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol303416k%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DDalisay%26aufirst%3DD.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DAmphlett%26aufirst%3DJ.%26aulast%3DManeerat%26aufirst%3DW.%26aulast%3DChavez%26aufirst%3DM.%2BA.%26aulast%3DWang%26aufirst%3DY.%2BA.%26aulast%3DMatainaho%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DR.%2BJ.%26atitle%3DNahuoic%2520acid%2520a%2520produced%2520by%2520a%2520streptomyces%2520sp.%2520isolated%2520from%2520a%2520marine%2520sediment%2520is%2520a%2520selective%2520SAM-competitive%2520inhibitor%2520of%2520the%2520histone%2520methyltransferase%2520SETD8%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26spage%3D414%26epage%3D417%26doi%3D10.1021%2Fol303416k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blum, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IbÃ¡Ã±ez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djaballah, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of SETD8 with cellular activity</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2471</span>â <span class="NLM_lpage">2478</span>, <span class="refDoi">Â DOI: 10.1021/cb500515r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500515r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2471-2478&author=G.+Blumauthor=G.+Ib%C3%A1%C3%B1ezauthor=X.+Raoauthor=D.+Shumauthor=C.+Raduauthor=H.+Djaballahauthor=J.+C.+Riceauthor=M.+Luo&title=Small-molecule+inhibitors+of+SETD8+with+cellular+activity&doi=10.1021%2Fcb500515r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of SETD8 with Cellular Activity</span></div><div class="casAuthors">Blum, Gil; Ibanez, Glorymar; Rao, Xiangjun; Shum, David; Radu, Constantin; Djaballah, Hakim; Rice, Judd C.; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2471-2478</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SETD8/SET8/Pr-SET7/KMT5A is the sole protein lysine methyltransferase (PKMT) known to monomethylate lysine 20 of histone H4 in vivo.  SETD8's methyltransferase activity has been implicated in many essential cellular processes including DNA replication, DNA damage response, transcription modulation, and cell cycle regulation.  Developing SETD8 inhibitors with cellular activity is a key step toward elucidating the diverse roles of SETD8 via convenient pharmacol. perturbation.  From the hits of a prior high throughput screen (HTS), SPS8I1-3 (NSC663284, BVT948, and ryuvidine) were validated as potent SETD8 inhibitors.  These compds. contain different structural motifs and inhibit SETD8 via distinct modes.  More importantly, these compds. show cellular activity by suppressing the H4K20me1 mark of SETD8 and recapitulate characteristic S/G2/M-phase cell cycle defects as obsd. for RNAi-mediated SETD8 knockdown.  The commonality of SPS8I1-3 against SETD8, together with their distinct structures and mechanisms for SETD8 inhibition, argues for the collective application of these compds. as SETD8 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquxqpOQLgFObVg90H21EOLACvtfcHk0li7N70dHdkarA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE&md5=87b7c415bf82577843fd5eeb09c2c746</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fcb500515r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500515r%26sid%3Dliteratum%253Aachs%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DIb%25C3%25A1%25C3%25B1ez%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DX.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DRadu%26aufirst%3DC.%26aulast%3DDjaballah%26aufirst%3DH.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520SETD8%2520with%2520cellular%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2471%26epage%3D2478%26doi%3D10.1021%2Fcb500515r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiewiora, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PittellaSilva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beauchamp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodera, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">The Dynamic Conformational Landscape of the Protein Methyltransferase SETD8</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e45403</span> <span class="refDoi">Â DOI: 10.7554/eLife.45403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.7554%2FeLife.45403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31081496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlals7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&author=S.+Chenauthor=R.+P.+Wiewioraauthor=F.+Mengauthor=N.+Babaultauthor=A.+Maauthor=W.+Yuauthor=K.+Qianauthor=H.+Huauthor=H.+Zouauthor=J.+Wangauthor=S.+Fanauthor=G.+Blumauthor=F.+PittellaSilvaauthor=K.+A.+Beauchampauthor=W.+Tempelauthor=H.+Jiangauthor=K.+Chenauthor=R.+Skeneauthor=Y.+G.+Zhengauthor=P.+J.+Brownauthor=J.+Jinauthor=C.+Luoauthor=J.+D.+Choderaauthor=M.+Luo&title=The+Dynamic+Conformational+Landscape+of+the+Protein+Methyltransferase+SETD8&doi=10.7554%2FeLife.45403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic conformational landscape of the protein methyltransferase SETD8</span></div><div class="casAuthors">Chen, Shi; Wiewiora, Rafal P.; Meng, Fanwang; Babault, Nicolas; Ma, Anqi; Yu, Wenyu; Qian, Kun; Hu, Hao; Zou, Hua; Wang, Junyi; Fan, Shijie; Blum, Gil; Pittella-Silva, Fabio; Beauchamp, Kyle A.; Tempel, Wolfram; Jiang, Hualiang; Chen, Kaixian; Skene, Robert J.; Zheng, Yujun George; Brown, Peter J.; Jin, Jian; Luo, Cheng; Chodera, John D.; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e45403/1-e45403/76</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Elucidating the conformational heterogeneity of proteins is essential for understanding protein function and developing exogenous ligands.  With the rapid development of exptl. and computational methods, it is of great interest to integrate these approaches to illuminate the conformational landscapes of target proteins.  SETD8 is a protein lysine methyltransferase (PKMT), which functions in vivo via the methylation of histone and nonhistone targets.  Utilizing covalent inhibitors and depleting native ligands to trap hidden conformational states, we obtained diverse X-ray structures of SETD8.  These structures were used to seed distributed atomistic mol. dynamics simulations that generated a total of six milliseconds of trajectory data.  Markov state models, built via an automated machine learning approach and corroborated exptl., reveal how slow conformational motions and conformational states are relevant to catalysis.  These findings provide mol. insight on enzymic catalysis and allosteric mechanisms of a PKMT via its detailed conformational landscape.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovPJl3tz4iu7Vg90H21EOLACvtfcHk0lhsxOrjsfZBVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlals7nL&md5=db09ad2c975d64615a594c391011a607</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.7554%2FeLife.45403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.45403%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWiewiora%26aufirst%3DR.%2BP.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DPittellaSilva%26aufirst%3DF.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DThe%2520Dynamic%2520Conformational%2520Landscape%2520of%2520the%2520Protein%2520Methyltransferase%2520SETD8%26jtitle%3DeLife%26date%3D2019%26volume%3D8%26doi%3D10.7554%2FeLife.45403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">PAINS in the assay: Chemical mechanisms of assay interference and promiscuous enzymatic inhibition observed during a sulfhydryl-scavenging HTS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2091</span>â <span class="NLM_lpage">2113</span>, <span class="refDoi">Â DOI: 10.1021/jm5019093</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5019093" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeisbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2091-2113&author=J.+L.+Dahlinauthor=J.+W.+Nissinkauthor=J.+M.+Strasserauthor=S.+Francisauthor=L.+Higginsauthor=H.+Zhouauthor=Z.+Zhangauthor=M.+A.+Walters&title=PAINS+in+the+assay%3A+Chemical+mechanisms+of+assay+interference+and+promiscuous+enzymatic+inhibition+observed+during+a+sulfhydryl-scavenging+HTS&doi=10.1021%2Fjm5019093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PAINS in the Assay: Chemical Mechanisms of Assay Interference and Promiscuous Enzymatic Inhibition Observed during a Sulfhydryl-Scavenging HTS</span></div><div class="casAuthors">Dahlin, Jayme L.; Nissink, J. Willem M.; Strasser, Jessica M.; Francis, Subhashree; Higgins, LeeAnn; Zhou, Hui; Zhang, Zhiguo; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2091-2113</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Significant resources in early drug discovery are spent unknowingly pursuing artifacts and promiscuous bioactive compds., while understanding the chem. basis for these adverse behaviors often goes unexplored in pursuit of lead compds.  Nearly all the hits from our recent sulfhydryl-scavenging high-throughput screen (HTS) targeting the histone acetyltransferase Rtt109 were such compds.  Herein, we characterize the chem. basis for assay interference and promiscuous enzymic inhibition for several prominent chemotypes identified by this HTS, including some pan-assay interference compds. (PAINS).  Protein mass spectrometry and ALARM NMR confirmed these compds. react covalently with cysteines on multiple proteins.  Unfortunately, compds. contg. these chemotypes have been published as screening actives in reputable journals and even touted as chem. probes or preclin. candidates.  Our detailed characterization and identification of such thiol-reactive chemotypes should accelerate triage of nuisance compds., guide screening library design, and prevent follow-up on undesirable chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4XTJapoezkrVg90H21EOLACvtfcHk0lhsxOrjsfZBVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeisbY%253D&md5=3f77702544cea2ec4c8184b60dd38b8a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm5019093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5019093%26sid%3Dliteratum%253Aachs%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DStrasser%26aufirst%3DJ.%2BM.%26aulast%3DFrancis%26aufirst%3DS.%26aulast%3DHiggins%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DPAINS%2520in%2520the%2520assay%253A%2520Chemical%2520mechanisms%2520of%2520assay%2520interference%2520and%2520promiscuous%2520enzymatic%2520inhibition%2520observed%2520during%2520a%2520sulfhydryl-scavenging%2520HTS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2091%26epage%3D2113%26doi%3D10.1021%2Fjm5019093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittella-Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a covalent inhibitor of the SET domain-containing protein 8 (SETD8) lysine methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9881</span>â <span class="NLM_lpage">9889</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01244</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01244" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9881-9889&author=K.+V.+Butlerauthor=A.+Maauthor=W.+Yuauthor=F.+Liauthor=W.+Tempelauthor=N.+Babaultauthor=F.+Pittella-Silvaauthor=J.+Shaoauthor=J.+Wangauthor=M.+Luoauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Structure-based+design+of+a+covalent+inhibitor+of+the+SET+domain-containing+protein+8+%28SETD8%29+lysine+methyltransferase&doi=10.1021%2Facs.jmedchem.6b01244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase</span></div><div class="casAuthors">Butler, Kyle V.; Ma, Anqi; Yu, Wenyu; Li, Fengling; Tempel, Wolfram; Babault, Nicolas; Pittella-Silva, Fabio; Shao, Jason; Wang, Junyi; Luo, Minkui; Vedadi, Masoud; Brown, Peter J.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9881-9889</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of protein lysine methyltransferases, including SET domain-contg. protein 8 (SETD8), are highly desired, as only a fraction of these enzymes are assocd. with high-quality inhibitors.  From our previously discovered SETD8 inhibitor, we developed a more potent analog and solved a cocrystal structure, which is the first crystal structure of SETD8 in complex with a small-mol. inhibitor.  This cocrystal structure allowed the design of a covalent inhibitor of SETD8 (MS453), which specifically modifies a cysteine residue near the inhibitor binding site, has an IC50 value of 804 nM, reacts with SETD8 with near-quant. yield, and is selective for SETD8 against 28 other methyltransferases.  We also solved the crystal structure of the covalent inhibitor in complex with SETD8.  This work provides at.-level perspective on the inhibition of SETD8 by small mols. and will help identify high-quality chem. probes of SETD8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjPurua1qCK7Vg90H21EOLACvtfcHk0lh_8aYqeLtMxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE&md5=b4d0b38dfe3a911633aad17642e78134</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DPittella-Silva%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DStructure-based%2520design%2520of%2520a%2520covalent%2520inhibitor%2520of%2520the%2520SET%2520domain-containing%2520protein%25208%2520%2528SETD8%2529%2520lysine%2520methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9881%26epage%3D9889%26doi%3D10.1021%2Facs.jmedchem.6b01244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleich, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective, substrate-competitive inhibitor of the lysine methyltransferase SETD8</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6822</span>â <span class="NLM_lpage">6833</span>, <span class="refDoi">Â DOI: 10.1021/jm500871s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500871s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6822-6833&author=A.+Maauthor=W.+Yuauthor=F.+Liauthor=R.+M.+Bleichauthor=J.+M.+Heroldauthor=K.+V.+Butlerauthor=J.+L.+Norrisauthor=V.+Korboukhauthor=A.+Tripathyauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=M.+Vedadiauthor=P.+J.+Brownauthor=J.+Jin&title=Discovery+of+a+selective%2C+substrate-competitive+inhibitor+of+the+lysine+methyltransferase+SETD8&doi=10.1021%2Fjm500871s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span></div><div class="casAuthors">Ma, Anqi; Yu, Wenyu; Li, Fengling; Bleich, Rachel M.; Herold, J. Martin; Butler, Kyle V.; Norris, Jacqueline L.; Korboukh, Victoria; Tripathy, Ashutosh; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Vedadi, Masoud; Brown, Peter J.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6822-6833</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20).  Monomethylation of H4K20 has been implicated in regulating diverse biol. processes including the DNA damage response.  In addn. to H4K20, SETD8 monomethylates nonhistone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression.  However, selective inhibitors of SETD8 are scarce.  The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor.  Here, the authors report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 I.  This small-mol. inhibitor is active in multiple biochem. assays.  Its affinity to SETD8 was confirmed by ITC (isothermal titrn. calorimetry) and SPR (surface plasmon resonance) studies.  Importantly, compd. I is selective for SETD8 over 15 other methyltransferases.  The authors also describe structure-activity relationships (SAR) of this series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR9y31FWBfY7Vg90H21EOLACvtfcHk0lh_8aYqeLtMxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI&md5=adca277f02033f96fe45a28087dca14e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500871s%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBleich%26aufirst%3DR.%2BM.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520selective%252C%2520substrate-competitive%2520inhibitor%2520of%2520the%2520lysine%2520methyltransferase%2520SETD8%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6822%26epage%3D6833%26doi%3D10.1021%2Fjm500871s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clancy, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Chemical biology of protein arginine modifications in epigenetic regulation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">5413</span>â <span class="NLM_lpage">5461</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.5b00003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=5413-5461&author=J.+Fuhrmannauthor=K.+W.+Clancyauthor=P.+R.+Thompson&title=Chemical+biology+of+protein+arginine+modifications+in+epigenetic+regulation&doi=10.1021%2Facs.chemrev.5b00003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology of Protein Arginine Modifications in Epigenetic Regulation</span></div><div class="casAuthors">Fuhrmann, Jakob; Clancy, Kathleen W.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5413-5461</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Here we aim to summarize the current knowledge surrounding the posttranslational modification of histone arginine residues, focusing on enzyme classes that catalyze the citrullination and methylation of arginine residues as well as noncanonical arginine modifications such as phosphorylation, ADP-ribosylation, and arginylation.  The major focus will be given to the protein arginine deiminases and protein arginine methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdzgTmC4C4fbVg90H21EOLACvtfcHk0lh_8aYqeLtMxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D&md5=a6df90a99a606e1d0a7296fa95625202</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00003%26sid%3Dliteratum%253Aachs%26aulast%3DFuhrmann%26aufirst%3DJ.%26aulast%3DClancy%26aufirst%3DK.%2BW.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DChemical%2520biology%2520of%2520protein%2520arginine%2520modifications%2520in%2520epigenetic%2520regulation%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D5413%26epage%3D5461%26doi%3D10.1021%2Facs.chemrev.5b00003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira de Freitas, R.</span></span> <span> </span><span class="NLM_article-title">Structural biology and chemistry of protein arginine methyltransferases</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1779</span>â <span class="NLM_lpage">1788</span>, <span class="refDoi">Â DOI: 10.1039/C4MD00269E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1039%2FC4MD00269E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26693001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSiu7%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1779-1788&author=M.+Schapiraauthor=R.+Ferreira+de+Freitas&title=Structural+biology+and+chemistry+of+protein+arginine+methyltransferases&doi=10.1039%2FC4MD00269E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology and chemistry of protein arginine methyltransferases</span></div><div class="casAuthors">Schapira, Matthieu; Ferreira de Freitas, Renato</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1779-1788</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs), an emerging target class in drug discovery, can methylate histones and other substrates, and can be divided into three subgroups, based on the methylation pattern of the reaction product (monomethylation, sym. or asym. dimethylation).  Here, we review the growing body of structural information characterizing this protein family, including structures in complex with substrate-competitive and allosteric inhibitors.  We outline structural differences between type I, II and III enzymes and propose a model underlying class-specificity.  We analyze the structural plasticity and diversity of the substrate, cofactor and allosteric binding sites, and propose that the conformational dynamics of PRMTs can be exploited towards the discovery of allosteric inhibitors that would antagonize conformationally active states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFIT9cSgrvLVg90H21EOLACvtfcHk0lgDB6VYnPNy4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSiu7%252FM&md5=4b092e8fd36283b625abb9738a496bad</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2FC4MD00269E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00269E%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26atitle%3DStructural%2520biology%2520and%2520chemistry%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1779%26epage%3D1788%26doi%3D10.1039%2FC4MD00269E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druzina, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheyi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungheim, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emtage, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the human PRMT5:MEP50 complex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17960</span>â <span class="NLM_lpage">17965</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1209814109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1073%2Fpnas.1209814109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23071334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17960-17965&author=S.+Antonysamyauthor=Z.+Bondayauthor=R.+M.+Campbellauthor=B.+Doyleauthor=Z.+Druzinaauthor=T.+Gheyiauthor=B.+Hanauthor=L.+N.+Jungheimauthor=Y.+Qianauthor=C.+Rauchauthor=M.+Russellauthor=J.+M.+Sauderauthor=S.+R.+Wassermanauthor=K.+Weichertauthor=F.+S.+Willardauthor=A.+Zhangauthor=S.+Emtage&title=Crystal+structure+of+the+human+PRMT5%3AMEP50+complex&doi=10.1073%2Fpnas.1209814109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human PRMT5:MEP50 complex</span></div><div class="casAuthors">Antonysamy, Stephen; Bonday, Zahid; Campbell, Robert M.; Doyle, Brandon; Druzina, Zhanna; Gheyi, Tarun; Han, Bomie; Jungheim, Louis N.; Qian, Yuewei; Rauch, Charles; Russell, Marijane; Sauder, J. Michael; Wasserman, Stephen R.; Wichert, Kenneth; Willard, Francis S.; Zhang, Aiping; Emtage, Spencer</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17960-17965, S17960/1-S17960/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play important roles in several cellular processes, including signaling, gene regulation, and transport of proteins and nucleic acids, to impact growth, differentiation, proliferation, and development.  PRMT5 sym. di-methylates the two-terminal Ï-guanidino nitrogens of arginine residues on substrate proteins.  PRMT5 acts as part of a multimeric complex in concert with a variety of partner proteins that regulate its function and specificity.  A core component of these complexes is the WD40 protein MEP50/WDR77/p44, which mediates interactions with binding partners and substrates.  We have detd. the crystal structure of human PRMT5 in complex with MEP50 (methylosome protein 50), bound to an S-adenosylmethionine analog and a peptide substrate derived from histone H4.  The structure of the surprising hetero-octameric complex reveals the close interaction between the seven-bladed Î²-propeller MEP50 and the N-terminal domain of PRMT5, and delineates the structural elements of substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcWBh5CVI7LVg90H21EOLACvtfcHk0lgDB6VYnPNy4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL&md5=757af18cfd0787a02c88c840273109aa</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209814109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209814109%26sid%3Dliteratum%253Aachs%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DDoyle%26aufirst%3DB.%26aulast%3DDruzina%26aufirst%3DZ.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DJungheim%26aufirst%3DL.%2BN.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DSauder%26aufirst%3DJ.%2BM.%26aulast%3DWasserman%26aufirst%3DS.%2BR.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DWillard%26aufirst%3DF.%2BS.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DEmtage%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17960%26epage%3D17965%26doi%3D10.1073%2Fpnas.1209814109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span> <span> </span><span class="NLM_article-title">Targeting protein arginine methyltransferase 5 in disease</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2081</span>â <span class="NLM_lpage">2098</span>, <span class="refDoi">Â DOI: 10.4155/fmc-2017-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.4155%2Ffmc-2017-0089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29076773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=2081-2098&author=A.+Richters&title=Targeting+protein+arginine+methyltransferase+5+in+disease&doi=10.4155%2Ffmc-2017-0089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein arginine methyltransferase 5 in disease</span></div><div class="casAuthors">Richters, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2081-2098</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">PRMT5 catalyzes the mono- and sym. dimethylation of the arginine N-guanidine group of a wide variety of target proteins including histones, transcriptional elongation factors, kinases and tumor suppressors by utilizing the essential co-factor S-adenosylmethionine as Me source.  PRMT5 overexpression has been linked to the progression of various diseases, including cancer, and is oftentimes assocd. with a poor prognosis.  Therefore, PRMT5 is promoted as a valuable target for drug discovery approaches and was a subject matter in recent endeavors aiming for the development of specific PRMT5 inhibitors.  This review will embrace the significance of PRMT5 as therapeutic target with respect to its mol. interdependencies in disease states as well as its implication in drug development approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqverMGPRhuJbVg90H21EOLACvtfcHk0lgDB6VYnPNy4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN&md5=6f8f3e27b9d7389e5b89ff084e6b691e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0089%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26atitle%3DTargeting%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520disease%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D2081%26epage%3D2098%26doi%3D10.4155%2Ffmc-2017-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">A patent review of arginine methyltransferase inhibitors (2010â2018)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">97</span>â <span class="NLM_lpage">114</span>, <span class="refDoi">Â DOI: 10.1080/13543776.2019.1567711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F13543776.2019.1567711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30640571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitF2murk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=97-114&author=X.+Liauthor=C.+Wangauthor=H.+Jiangauthor=C.+Luo&title=A+patent+review+of+arginine+methyltransferase+inhibitors+%282010%E2%80%932018%29&doi=10.1080%2F13543776.2019.1567711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A patent review of arginine methyltransferase inhibitors (2010-2018)</span></div><div class="casAuthors">Li, Xiao; Wang, Chen; Jiang, Hao; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-114</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are fundamental enzymes that specifically modify the arginine residues of versatile substrates in cells.  The aberrant expression and abnormal enzymic activity of PRMTs are assocd. with many human diseases, esp. cancer.  PRMTs are emerging as promising drug targets in both academia and industry.  This review summarizes the updated patented inhibitors targeting PRMTs from 2010 to 2018.  The authors illustrate the chem. structures, mol. mechanism of action, pharmacol. activities as well as the potential clin. application including combination therapy and biomarker-guided therapy.  PRMT inhibitors in clin. trials are also highlighted.  The authors provide a future perspective for further development of potent and selective PRMT inhibitors.  Although a no. of small mol. inhibitors of PRMTs with sufficient potency have been developed, the selectivity of most PRMT inhibitors remains to be improved.  Hence, novel approaches such as allosteric regulation need to be further studied to identify PRMT inhibitors.  So far, three PRMT inhibitors have entered clin. trials, including PRMT5 inhibitor GSK3326595 and JNJ-64619178 as well as PRMT1 inhibitor GSK3368715.  PRMT inhibitors with novel mechanism of action and good drug-like properties may shed new light on drug research and development progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPH62wM5Eh8bVg90H21EOLACvtfcHk0ljpnQF9y5ZOpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitF2murk%253D&md5=196b7ad8fa047aad498c984bd1b72b15</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1567711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1567711%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DA%2520patent%2520review%2520of%2520arginine%2520methyltransferase%2520inhibitors%2520%25282010%25E2%2580%25932018%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2019%26volume%3D29%26spage%3D97%26epage%3D114%26doi%3D10.1080%2F13543776.2019.1567711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindiapina, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Recent advances in targeting protein arginine methyltransferase enzymes in cancer therapy</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">527</span>â <span class="NLM_lpage">545</span>, <span class="refDoi">Â DOI: 10.1080/14728222.2018.1474203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F14728222.2018.1474203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29781349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BC1Mflt1GntQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=527-545&author=E.+Smithauthor=W.+Zhouauthor=P.+Shindiapinaauthor=S.+Sifauthor=C.+Liauthor=R.+A.+Baiocchi&title=Recent+advances+in+targeting+protein+arginine+methyltransferase+enzymes+in+cancer+therapy&doi=10.1080%2F14728222.2018.1474203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in targeting protein arginine methyltransferase enzymes in cancer therapy</span></div><div class="casAuthors">Smith Emily; Shindiapina Polina; Baiocchi Robert A; Zhou Wei; Li Chenglong; Sif Said; Li Chenglong</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on therapeutic targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">527-545</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Exploration in the field of epigenetics has revealed the diverse roles of the protein arginine methyltransferase (PRMT) family of proteins in multiple disease states.  These findings have led to the development of specific inhibitors and discovery of several new classes of drugs with potential to treat both benign and malignant conditions.  Areas covered: We provide an overview on the role of PRMT enzymes in healthy and malignant cells, highlighting the role of arginine methylation in specific pathways relevant to cancer pathogenesis.  Additionally, we describe structure and catalytic activity of PRMT and discuss the mechanisms of action of novel small molecule inhibitors of specific members of the arginine methyltransferase family.  Expert opinion: As the field of PRMT biology advances, it's becoming clear that this class of enzymes is highly relevant to maintaining normal physiologic processes as well and disease pathogenesis.  We discuss the potential impact of PRMT inhibitors as a broad class of drugs, including the pleiotropic effects, off target effects the need for more detailed PRMT-centric interactomes, and finally, the potential for targeting this class of enzymes in clinical development of experimental therapeutics for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkL15Wkj-iUfx2phytoz6_fW6udTcc2eakg5bgndd2k7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mflt1GntQ%253D%253D&md5=413ff9fbe878fc14fcc95f6a133918ba</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1474203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1474203%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShindiapina%26aufirst%3DP.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DRecent%2520advances%2520in%2520targeting%2520protein%2520arginine%2520methyltransferase%2520enzymes%2520in%2520cancer%2520therapy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D527%26epage%3D545%26doi%3D10.1080%2F14728222.2018.1474203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>â <span class="NLM_lpage">6304</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 Î¼M) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 Î¼M.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0ljpnQF9y5ZOpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-substituted-<i>N</i>-(3-(3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide as potent and selective protein arginine methyltransferases 5 inhibitors: Design, synthesis and biological evaluation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">317</span>â <span class="NLM_lpage">333</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.12.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2018.12.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30605830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtl2nsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2019&pages=317-333&author=J.+Shaoauthor=K.+Zhuauthor=D.+Duauthor=Y+Zhangauthor=H.+Taoauthor=Z.+Chenauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=W.+Duan&title=Discovery+of+2-substituted-N-%283-%283%2C4-dihydroisoquinolin-2%281H%29-yl%29-2-hydroxypropyl%29-1%2C2%2C3%2C4-tetrahydroisoquinoline-6-carboxamide+as+potent+and+selective+protein+arginine+methyltransferases+5+inhibitors%3A+Design%2C+synthesis+and+biological+evaluation&doi=10.1016%2Fj.ejmech.2018.12.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-substituted-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)-2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide as potent and selective protein arginine methyltransferases 5 inhibitors: Design, synthesis and biological evaluation</span></div><div class="casAuthors">Shao, Jingwei; Zhu, Kongkai; Du, Daohai; Zhang, Yuanyuan; Tao, Hongrui; Chen, Zhifeng; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Duan, Wenhu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">317-333</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)-2-hydroxypropyl)amide derivs. targeting PRMT5 were designed with structure-based approach and synthesized.  Among them, compd. (S)-2-(4-bromobenzoyl)-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide showed potent and selective PRMT5 inhibition activity with an IC50 of 8.5 nM, which was approx. equiv. with the phase I clin. trial PRMT5 inhibitor GSK-3326595 (IC50 = 5.5 nM).  Compd. (S)-2-(4-bromobenzoyl)-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide also displayed pronounced anti-proliferative activity in MV4-11 cells (GI50 = 18 nM) and antitumor activity in MV4-11 mouse xenografts model.  This mol. served as an excellent tool compd. for probing the biol. function of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8r5tDk63g07Vg90H21EOLACvtfcHk0lhN-fjKtLTrpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtl2nsw%253D%253D&md5=d04aaa30e120c8e89d0f9778130cb800</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.12.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.12.065%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DDu%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY%26aulast%3DTao%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DW.%26atitle%3DDiscovery%2520of%25202-substituted-N-%25283-%25283%252C4-dihydroisoquinolin-2%25281H%2529-yl%2529-2-hydroxypropyl%2529-1%252C2%252C3%252C4-tetrahydroisoquinoline-6-carboxamide%2520as%2520potent%2520and%2520selective%2520protein%2520arginine%2520methyltransferases%25205%2520inhibitors%253A%2520Design%252C%2520synthesis%2520and%2520biological%2520evaluation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D164%26spage%3D317%26epage%3D333%26doi%3D10.1016%2Fj.ejmech.2018.12.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 (PRMT5) as an anticancer target and its inhibitor discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9429</span>â <span class="NLM_lpage">9441</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00598</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00598" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOmsLvJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9429-9441&author=Y.+Wangauthor=W.+Huauthor=Y.+Yuan&title=Protein+arginine+methyltransferase+5+%28PRMT5%29+as+an+anticancer+target+and+its+inhibitor+discovery&doi=10.1021%2Facs.jmedchem.8b00598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery</span></div><div class="casAuthors">Wang, Yuanxiang; Hu, Wenhao; Yuan, Yanqiu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9429-9441</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRMT5 is a major enzyme responsible for sym. dimethylation of arginine residues on both histone and non-histone proteins, regulating many biol. pathways in mammalian cells.  PRMT5 has been suggested as a therapeutic target in a variety of diseases including infectious disease, heart disease, and cancer.  Many PRMT5 inhibitors have been discovered in the past 5 years, and one entered clin. trial in 2015 for the treatment of solid tumor and mantle cell lymphoma (MCL).  The aim of this review is to summarize the current understanding of the roles of PRMT5 in cancer and the discovery of PRMT5 enzymic inhibitors.  By reviewing the structure-activity relationship (SAR) of known inhibitors of PRMT5, we hope to provide guidance for future drug designs and inhibitor optimization.  Opportunities and limitations of PRMT5 inhibitors for the treatment of cancer are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG2MM_2r6cc7Vg90H21EOLACvtfcHk0lhN-fjKtLTrpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOmsLvJ&md5=1fffdd9a78a5edbe4c31357d3e08e2d8</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00598%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DYuan%26aufirst%3DY.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520as%2520an%2520anticancer%2520target%2520and%2520its%2520inhibitor%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9429%26epage%3D9441%26doi%3D10.1021%2Facs.jmedchem.8b00598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>â <span class="NLM_lpage">437</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lhN-fjKtLTrpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter
Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Structure and property guided design in the identification of PRMT5 tool compound EPZ015666</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">162</span>â <span class="NLM_lpage">166</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=162-166&author=K.+W.+Duncanauthor=N.+Riouxauthor=P.+A.+Boriack-Sjodinauthor=M.+J.+Munchhofauthor=L.+A.+Reiterauthor=C.+R.+Majerauthor=L.+Jinauthor=L.+D.+Johnstonauthor=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=M.+Porter%0AScottauthor=R.+M.+Pollockauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=Structure+and+property+guided+design+in+the+identification+of+PRMT5+tool+compound+EPZ015666&doi=10.1021%2Facsmedchemlett.5b00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Property Guided Design in the Identification of PRMT5 Tool Compound EPZ015666</span></div><div class="casAuthors">Duncan, Kenneth W.; Rioux, Nathalie; Boriack-Sjodin, P. Ann; Munchhof, Michael J.; Reiter, Lawrence A.; Majer, Christina R.; Jin, Lei; Johnston, L. Danielle; Chan-Penebre, Elayne; Kuplast, Kristy G.; Porter Scott, Margaret; Pollock, Roy M.; Waters, Nigel J.; Smith, Jesse J.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-166</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent publication of a potent and selective inhibitor of protein methyltransferase 5 (PRMT5) provides the scientific community with in vivo-active tool compd. I (EPZ015666, GSK3235025) to probe the underlying pharmacol. of this key enzyme.  Herein, the authors report the design and optimization strategies employed on an initial hit compd. with poor in vitro clearance to yield in vivo tool compd. I and an addnl. potent in vitro tool mol. II (GSK3203591).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V7LiQgVZjrVg90H21EOLACvtfcHk0ljpq4kK-YcYWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L&md5=e7ac92c2d985da4994e9bb959adededf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00380%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DReiter%26aufirst%3DL.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DStructure%2520and%2520property%2520guided%2520design%2520in%2520the%2520identification%2520of%2520PRMT5%2520tool%2520compound%2520EPZ015666%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D162%26epage%3D166%26doi%3D10.1021%2Facsmedchemlett.5b00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerhart, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellner, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montes de Oca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span> <span> </span><span class="NLM_article-title">Activation of the p53-MDM4 regulatory axis defines the anti-tumour response to PRMT5 inhibition through its role in regulating cellular splicing</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">9711</span>, <span class="refDoi">Â DOI: 10.1038/s41598-018-28002-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fs41598-018-28002-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29946150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FitFGmsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9711&author=S.+V.+Gerhartauthor=W.+A.+Kellnerauthor=C.+Thompsonauthor=M.+B.+Pappalardiauthor=X.+P.+Zhangauthor=R.+Montes+de+Ocaauthor=E.+Penebreauthor=K.+Duncanauthor=A.+Boriack-Sjodinauthor=B.+Leauthor=C.+Majerauthor=M.+T.+McCabeauthor=C.+Carpenterauthor=N.+Johnsonauthor=R.+G.+Krugerauthor=O.+Barbash&title=Activation+of+the+p53-MDM4+regulatory+axis+defines+the+anti-tumour+response+to+PRMT5+inhibition+through+its+role+in+regulating+cellular+splicing&doi=10.1038%2Fs41598-018-28002-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the p53-MDM4 regulatory axis defines the anti-tumour response to PRMT5 inhibition through its role in regulating cellular splicing</span></div><div class="casAuthors">Gerhart Sarah V; Kellner Wendy A; Thompson Christine; Pappalardi Melissa B; Zhang Xi-Ping; Montes de Oca Rocio; Le BaoChau; McCabe Michael T; Carpenter Chris; Johnson Neil; Kruger Ryan G; Barbash Olena; Penebre Elayne; Duncan Kenneth; Boriack-Sjodin Ann; Majer Christina; Carpenter Chris</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9711</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Evasion of the potent tumour suppressor activity of p53 is one of the hurdles that must be overcome for cancer cells to escape normal regulation of cellular proliferation and survival.  In addition to frequent loss of function mutations, p53 wild-type activity can also be suppressed post-translationally through several mechanisms, including the activity of PRMT5.  Here we describe broad anti-proliferative activity of potent, selective, reversible inhibitors of protein arginine methyltransferase 5 (PRMT5) including GSK3326595 in human cancer cell lines representing both hematologic and solid malignancies.  Interestingly, PRMT5 inhibition activates the p53 pathway via the induction of alternative splicing of MDM4.  The MDM4 isoform switch and subsequent p53 activation are critical determinants of the response to PRMT5 inhibition suggesting that the integrity of the p53-MDM4 regulatory axis defines a subset of patients that could benefit from treatment with GSK3326595.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBwMOdegb-Q-_Q9URG6Dx8fW6udTcc2eYuay7OYKrlHrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FitFGmsg%253D%253D&md5=f61573ec23278ba5a69df3da1531ec97</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-28002-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-28002-y%26sid%3Dliteratum%253Aachs%26aulast%3DGerhart%26aufirst%3DS.%2BV.%26aulast%3DKellner%26aufirst%3DW.%2BA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DZhang%26aufirst%3DX.%2BP.%26aulast%3DMontes%2Bde%2BOca%26aufirst%3DR.%26aulast%3DPenebre%26aufirst%3DE.%26aulast%3DDuncan%26aufirst%3DK.%26aulast%3DBoriack-Sjodin%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DMajer%26aufirst%3DC.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DN.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DBarbash%26aufirst%3DO.%26atitle%3DActivation%2520of%2520the%2520p53-MDM4%2520regulatory%2520axis%2520defines%2520the%2520anti-tumour%2520response%2520to%2520PRMT5%2520inhibition%2520through%2520its%2520role%2520in%2520regulating%2520cellular%2520splicing%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D9711%26doi%3D10.1038%2Fs41598-018-28002-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">612</span>â <span class="NLM_lpage">617</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 Â± 3 nM and 25 Â± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0lhxInzD3K2bWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Nucleoside protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1264</span>â <span class="NLM_lpage">1269</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2019.03.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.bmcl.2019.03.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30956011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmslOit70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1264-1269&author=H.+Linauthor=J.+I.+Luengo&title=Nucleoside+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1016%2Fj.bmcl.2019.03.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoside protein arginine methyltransferase 5 (PRMT5) inhibitors</span></div><div class="casAuthors">Lin, Hong; Luengo, Juan I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1264-1269</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein Arginine Methyltransferase 5 (PRMT5) is known to sym. dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 selective inhibitors, GSK3326595, a substrate competitive inhibitor, and JNJ64619178, a SAM (S-adenosyl-L-methionine) mimetic/competitive inhibitor, have entered clinic trials for multiple cancer types.  This review focuses on the recent developments in SAM mimetic nucleoside PRMT5 inhibitors, their SAR and structural insight based on published co-crystal structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWzpjRzaVPBLVg90H21EOLACvtfcHk0lhxInzD3K2bWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmslOit70%253D&md5=f5889f2c5e084dc231f865805a241da6</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.03.042%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DNucleoside%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1264%26epage%3D1269%26doi%3D10.1016%2Fj.bmcl.2019.03.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Brehmer, D.</span>; <span class="NLM_string-name">Beke, L.</span>; <span class="NLM_string-name">Boeckx, A.</span>; <span class="NLM_string-name">Diels, G. S. M.</span>; <span class="NLM_string-name">Gilissen, R. A. H. J.</span>; <span class="NLM_string-name">Lawson, E. C.</span>; <span class="NLM_string-name">Pande, V.</span>; <span class="NLM_string-name">Parade, M. C. B. C.</span>; <span class="NLM_string-name">Schepens, W. B. G.</span></span> <span> </span><span class="NLM_article-title">Novel 6â6 Bicyclic Aromatic Ring Substituted Nucleoside Analogues for Use as PRMT5 Inhibitors</span>. <span class="NLM_patent">WO 2017032840 A1</span>, August 25, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=T.+Wu&author=D.+Brehmer&author=L.+Beke&author=A.+Boeckx&author=G.+S.+M.+Diels&author=R.+A.+H.+J.+Gilissen&author=E.+C.+Lawson&author=V.+Pande&author=M.+C.+B.+C.+Parade&author=W.+B.+G.+Schepens&title=Novel+6%E2%80%936+Bicyclic+Aromatic+Ring+Substituted+Nucleoside+Analogues+for+Use+as+PRMT5+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26atitle%3DNovel%25206%25E2%2580%25936%2520Bicyclic%2520Aromatic%2520Ring%2520Substituted%2520Nucleoside%2520Analogues%2520for%2520Use%2520as%2520PRMT5%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J. H.</span>; <span class="NLM_string-name">McAlpine, I. J.</span>; <span class="NLM_string-name">Tran-Dube, M. B.</span>; <span class="NLM_string-name">Rui, E. Y.</span>; <span class="NLM_string-name">Wythes, M. J.</span>; <span class="NLM_string-name">Kumpf, R. A.</span>; <span class="NLM_string-name">McTigue, M. A.</span></span> <span> </span><span class="NLM_article-title">Substituted Nucleoside Derivatives Useful as Anticancer Agents</span>. <span class="NLM_patent">US 20160244475 A1</span>, February 17, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+H.+Tatlock&author=I.+J.+McAlpine&author=M.+B.+Tran-Dube&author=E.+Y.+Rui&author=M.+J.+Wythes&author=R.+A.+Kumpf&author=M.+A.+McTigue&title=Substituted+Nucleoside+Derivatives+Useful+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26atitle%3DSubstituted%2520Nucleoside%2520Derivatives%2520Useful%2520as%2520Anticancer%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1033</span>â <span class="NLM_lpage">1038</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.9b00074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1033-1038&author=H.+Linauthor=M.+Wangauthor=Y.+W.+Zhangauthor=S.+Tongauthor=R.+A.+Lealauthor=R.+Shettyauthor=K.+Vaddiauthor=J.+I.+Luengo&title=Discovery+of+potent+and+selective+covalent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1021%2Facsmedchemlett.9b00074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors</span></div><div class="casAuthors">Lin Hong; Wang Min; Zhang Yang W; Leal Raul A; Shetty Rupa; Vaddi Kris; Luengo Juan I; Tong Shuilong</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1033-1038</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is known to symmetrically dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 possesses a cysteine (C449) in the active site, unique to PRMT5.  Therefore, covalent PRMT5 inhibition is an attractive chemical approach.  Herein, we report an exciting discovery of a series of novel hemiaminals that under physiological conditions can be converted to aldehydes and react with C449 to form covalent adducts, which presumably undergo an unprecedented elimination to form the thiol-vinyl ethers, as indicated by electron density in the co-crystal structure of the PRMT5/MEP50 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdNDVSKB6iezW1zwg2yUK5fW6udTcc2eaY6XvJnNXopLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D&md5=4b5b08dc3e3615144ece564f70bf3bd0</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BW.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DLeal%26aufirst%3DR.%2BA.%26aulast%3DShetty%26aufirst%3DR.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1033%26epage%3D1038%26doi%3D10.1021%2Facsmedchemlett.9b00074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J.</span>; <span class="NLM_string-name">Vaddi, K.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibitors of Protein Arginine Methyltransferase 5 (PRMT5)</span>. <span class="NLM_patent">WO 2017218802 A1</span>, June 15, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+Luengo&author=K.+Vaddi&title=Selective+Inhibitors+of+Protein+Arginine+Methyltransferase+5+%28PRMT5%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLuengo%26aufirst%3DJ.%26atitle%3DSelective%2520Inhibitors%2520of%2520Protein%2520Arginine%2520Methyltransferase%25205%2520%2528PRMT5%2529%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschman, H. R.</span></span> <span> </span><span class="NLM_article-title">PRMT1 is the predominant type I protein arginine methyltransferase in mammalian cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">7723</span>â <span class="NLM_lpage">7730</span>, <span class="refDoi">Â DOI: 10.1074/jbc.275.11.7723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.275.11.7723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10713084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVylsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=7723-7730&author=J.+Tangauthor=A.+Frankelauthor=R.+J.+Cookauthor=S.+Kimauthor=W.+K.+Paikauthor=K.+R.+Williamsauthor=S.+Clarkeauthor=H.+R.+Herschman&title=PRMT1+is+the+predominant+type+I+protein+arginine+methyltransferase+in+mammalian+cells&doi=10.1074%2Fjbc.275.11.7723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT1 is the predominant type I protein arginine methyltransferase in mammalian cells</span></div><div class="casAuthors">Tang, Jie; Frankel, Adam; Cook, Robert J.; Kim, Sangduk; Paik, Woon Ki; Williams, Kenneth R.; Clarke, Steven; Herschman, Harvey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7723-7730</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Type I protein arginine methyltransferases catalyze the formation of asym. Ï-NG,NG-dimethylarginine residues by transferring Me groups from S-adenosyl-L-methionine to guanidino groups of arginine residues in a variety of eucaryotic proteins.  The predominant type I enzyme activity is found in mammalian cells as a high mol. wt. complex (300-400 kDa).  In a previous study, this protein arginine methyltransferase activity was identified as an addnl. activity of 10-formyltetrahydrofolate dehydrogenase (FDH) protein.  However, immunodepletion of FDH activity in RAT1 cells and in murine tissue exts. with antibody to FDH does not diminish type I methyltransferase activity toward the methyl-accepting substrates glutathione S-transferase fibrillarin glycine arginine domain fusion protein or heterogeneous nuclear ribonucleoprotein A1.  Similarly, immunodepletion with anti-FDH antibody does not remove the endogenous methylating activity for hypomethylated proteins present in exts. from adenosine dialdehyde-treated RAT1 cells.  In contrast, anti-PRMT1 antibody can remove PRMT1 activity from RAT1 exts., murine tissue exts., and purified rat liver FDH prepns.  Tissue exts. from FDH(+/+), FDH(+/-), and FDH(-/-) mice have similar protein arginine methyltransferase activities but high, intermediate, and undetectable FDH activities, resp.  Recombinant glutathione S-transferase-PRMT1, but not purified FDH, can be cross-linked to the methyl-donor substrate S-adenosyl-L-methionine.  We conclude that PRMT1 contributes the major type I protein arginine methyltransferase enzyme activity present in mammalian cells and tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR-pUS71H91bVg90H21EOLACvtfcHk0liyN0AOgtjh-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVylsL0%253D&md5=5ddc959b9ce7f7c48715ac1dba77aaa9</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1074%2Fjbc.275.11.7723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.275.11.7723%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DR.%2BJ.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DPaik%26aufirst%3DW.%2BK.%26aulast%3DWilliams%26aufirst%3DK.%2BR.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26atitle%3DPRMT1%2520is%2520the%2520predominant%2520type%2520I%2520protein%2520arginine%2520methyltransferase%2520in%2520mammalian%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D7723%26epage%3D7730%26doi%3D10.1074%2Fjbc.275.11.7723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>â <span class="NLM_lpage">50</span>, <span class="refDoi">Â DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0liyN0AOgtjh-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. Q.</span></span> <span> </span><span class="NLM_article-title">PRMT1 promoted HCC growth and metastasis in vitro and in vivo via activating the STAT3 signalling pathway</span>. <i>Cell. Physiol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1643</span>â <span class="NLM_lpage">1654</span>, <span class="refDoi">Â DOI: 10.1159/000490983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1159%2F000490983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29945155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtl2hu7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=1643-1654&author=X.+P.+Zhangauthor=Y.+B.+Jiangauthor=C.+Q.+Zhongauthor=N.+Maauthor=E.+B.+Zhangauthor=F.+Zhangauthor=J.+J.+Liauthor=Y.+Z.+Dengauthor=K.+Wangauthor=D.+Xieauthor=S.+Q.+Cheng&title=PRMT1+promoted+HCC+growth+and+metastasis+in+vitro+and+in+vivo+via+activating+the+STAT3+signalling+pathway&doi=10.1159%2F000490983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT1 Promoted HCC Growth and Metastasis In Vitro and In Vivo via Activating the STAT3 Signalling Pathway</span></div><div class="casAuthors">Zhang, Xiu-Ping; Jiang, Ya-Bo; Zhong, Cheng-Qian; Ma, Ning; Zhang, Er-Bin; Zhang, Fan; Li, Jing-Jing; Deng, Yue-Zhen; Wang, Kang; Xie, Dong; Cheng, Shu-Qun</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1643-1654</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">Background/Aims: Although it has been widely accepted that protein arginine methyltransferase 1 (PRMT1) is a cancer-promoting gene in various cancers, the mechanism of PRMT1 in hepatocellular carcinoma (HCC) requires more exploration.  This study aimed to investigate the role of PRMT1 in HCC growth and metastasis.  Methods: We compared PRMT1 expression and clinicopathol. characteristics using paired HCC and adjacent noncancerous liver tissues from 210 patients and immunohistochem. analyses.  Cell proliferation, colony formation and migration were detd. in HCC cell lines with PRMT1 overexpression or downregulation through MTT, crystal violet and Boyden chamber assays.  Tumor growth was monitored in a xenograft model, and intrahepatic metastasis models were established.  Results: PRMT1 expression was greatly increased in clin. HCC samples and strongly assocd. with poor prognosis and recurrence; PRMT1 expression was also pos. correlated with microvascular invasion (P = 0.024), tumor differentiation (P = 0.014), tumor size (P = 0.002), and portal vein tumor thrombus (PVTT) (P = 0.028).  Cell proliferation, colony formation and migration in vitro were enhanced by PRMT1 upregulation and decreased by PRMT1 downregulation in HCC cell lines.  Moreover, low PRMT1 expression resulted in slow tumor growth and decreased tumor wt. in vivo, as well as tumor metastasis.  These phenotypes were assocd. with STAT3 signalling pathway activation.  Cryptotanshinone, a STAT3 inhibitor, inhibited STAT3 phosphorylation and reversed the HCC phenotype of PRMT1 expression.  Conclusions: We revealed a significant role for PRMT1 in HCC progression and metastasis in vitro and in vivo via STAT3 signalling pathway activation.  PRMT1 may be a potential novel prognostic biomarker and new therapeutic target for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWErzsvDb2GrVg90H21EOLACvtfcHk0liyN0AOgtjh-A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2hu7rP&md5=324c21d3dd1a54d33402ae667b2cc8b6</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1159%2F000490983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000490983%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%2BP.%26aulast%3DJiang%26aufirst%3DY.%2BB.%26aulast%3DZhong%26aufirst%3DC.%2BQ.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DE.%2BB.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DDeng%26aufirst%3DY.%2BZ.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DS.%2BQ.%26atitle%3DPRMT1%2520promoted%2520HCC%2520growth%2520and%2520metastasis%2520in%2520vitro%2520and%2520in%2520vivo%2520via%2520activating%2520the%2520STAT3%2520signalling%2520pathway%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2018%26volume%3D47%26spage%3D1643%26epage%3D1654%26doi%3D10.1159%2F000490983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span> <span> </span><span class="NLM_article-title">The role of PRMT1 in EGFR methylation and signaling in MDA-MB-468 triple-negative breast cancer cells</span>. <i>Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">74</span>â <span class="NLM_lpage">80</span>, <span class="refDoi">Â DOI: 10.1007/s12282-017-0790-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1007%2Fs12282-017-0790-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28643125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BC1cjhtFamsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=74-80&author=K.+Nakaiauthor=W.+Xiaauthor=H.+W.+Liaoauthor=M.+Saitoauthor=M.+C.+Hungauthor=H.+Yamaguchi&title=The+role+of+PRMT1+in+EGFR+methylation+and+signaling+in+MDA-MB-468+triple-negative+breast+cancer+cells&doi=10.1007%2Fs12282-017-0790-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The role of PRMT1 in EGFR methylation and signaling in MDA-MB-468 triple-negative breast cancer cells</span></div><div class="casAuthors">Nakai Katsuya; Xia Weiya; Liao Hsin-Wei; Hung Mien-Chie; Yamaguchi Hirohito; Nakai Katsuya; Saito Mitsue</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer (Tokyo, Japan)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Epidermal growth factor receptor (EGFR) is often overexpressed in triple-negative breast cancer (TNBC).  However, clinical studies have shown that therapies against EGFR are not effective in patients with TNBC.  Recently, it has been reported that arginine 198/200 in EGFR extracellular domain is methylated by PRMT1 and that the methylation confers resistance to EGFR monoclonal antibody cetuximab in colorectal cancer cells.  To explore a potential mechanism underlying intrinsic resistance to anti-EGFR therapy in TNBC, we investigated the role of PRMT1 in EGFR methylation and signaling in MDA-MB-468 (468) TNBC cells.  METHODS:  We knocked down PRMT1 in 468 cells by shRNA, and subjected the cell lysates to Western blot analysis to examine EGFR activation and its downstream molecules.  We also evaluated cell proliferation and sphere formation of PRMT1-knockdown cells.  Finally, we examined the effects of pan-PRMT inhibitor, AMI-1, on cetuximab by colony formation and soft agar assays.  RESULTS:  EGFR methylation and activity was significantly reduced in PRMT1-knockdown cells compared to the parental cells.  Knockdown of PRMT1 also reduced cell proliferation and sphere formation.  Moreover, AMI-1 sensitized 468 cells to cetuximab.  CONCLUSION:  The results indicate that PRMT1 is critical for EGFR activity in 468 cells.  Our data also suggest that inhibition of PRMT1 sensitizes TNBC cells to cetuximab.  Thus, inhibition of PRMT1 may be an effective therapeutic strategy to overcome intrinsic resistance to cetuximab in TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSydnDN7IVXh0NHe_gFldTbfW6udTcc2eYM2PZLyj1zdbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjhtFamsQ%253D%253D&md5=d38db639657d3d981a01619c03b1d2dd</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1007%2Fs12282-017-0790-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12282-017-0790-z%26sid%3Dliteratum%253Aachs%26aulast%3DNakai%26aufirst%3DK.%26aulast%3DXia%26aufirst%3DW.%26aulast%3DLiao%26aufirst%3DH.%2BW.%26aulast%3DSaito%26aufirst%3DM.%26aulast%3DHung%26aufirst%3DM.%2BC.%26aulast%3DYamaguchi%26aufirst%3DH.%26atitle%3DThe%2520role%2520of%2520PRMT1%2520in%2520EGFR%2520methylation%2520and%2520signaling%2520in%2520MDA-MB-468%2520triple-negative%2520breast%2520cancer%2520cells%26jtitle%3DBreast%2520Cancer%26date%3D2018%26volume%3D25%26spage%3D74%26epage%3D80%26doi%3D10.1007%2Fs12282-017-0790-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Theoretical insights into catalytic mechanism of protein arginine methyltransferase 1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e72424</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0072424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1371%2Fjournal.pone.0072424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23977297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKmu7bK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=R.+Zhangauthor=X.+Liauthor=Z.+Liangauthor=K.+Zhuauthor=J.+Luauthor=X.+Kongauthor=S.+Ouyangauthor=L.+Liauthor=Y.+G.+Zhengauthor=C.+Luo&title=Theoretical+insights+into+catalytic+mechanism+of+protein+arginine+methyltransferase+1&doi=10.1371%2Fjournal.pone.0072424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Theoretical insights into catalytic mechanism of protein arginine methyltransferase 1</span></div><div class="casAuthors">Zhang, Ruihan; Li, Xin; Liang, Zhongjie; Zhu, Kongkai; Lu, Junyan; Kong, Xiangqian; Ouyang, Sisheng; Li, Lin; Zheng, Yujun George; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e72424</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 1 (PRMT1), the major arginine asym. dimethylation enzyme in mammals, is emerging as a potential drug target for cancer and cardiovascular disease.  Understanding the catalytic mechanism of PRMT1 will facilitate inhibitor design.  However, detailed mechanisms of the Me transfer process and substrate deprotonation of PRMT1 remain unclear.  Here, the authors present a theor. study on PRMT1-catalyzed arginine dimethylation by employing mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns.  Ternary complex models, composed of PRMT1, peptide substrate, and S-adenosyl-L-methionine (AdoMet) as cofactor, were constructed and verified by 30-ns MD simulation.  The snapshots selected from the MD trajectory were applied for the QM/MM calcn.  The typical SN2-favored transition states of the 1st and 2nd Me transfers were identified from the potential energy profile.  Deprotonation of the substrate, arginine, occurred immediately after Me transfer, and the carboxylate group of Glu-144 acted as proton acceptor.  Furthermore, natural bond orbital anal. and electrostatic potential calcn. showed that Glu-144 facilitated the charge redistribution during the reaction and reduced the energy barrier.  The authors propose the detailed mechanism of PRMT1-catalyzed asym. dimethylation, which increases insight on the small-mol. effector design, and enables further investigations into the physiol. function of this family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpznCqxjTezqrVg90H21EOLACvtfcHk0li0LKvHQ1D7oQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKmu7bK&md5=91656eee3ce77e35b56f01852261df98</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0072424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0072424%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DTheoretical%2520insights%2520into%2520catalytic%2520mechanism%2520of%2520protein%2520arginine%2520methyltransferase%25201%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0072424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela
Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A Potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">772</span>â <span class="NLM_lpage">781</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+Shenauthor=M.+M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+Liauthor=K.+V.+Butlerauthor=H.+U.+Kaniskanauthor=B.+A.+Speedauthor=C.+Dela%0ASenaauthor=A.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+Potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0li0LKvHQ1D7oQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520Potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of protein arginine methyltransferase 6 reduces reactive oxygen species production and attenuates aminoglycoside- and cisplatin-induced hair cell death</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">133</span>â <span class="NLM_lpage">150</span>, <span class="refDoi">Â DOI: 10.7150/thno.37362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.7150%2Fthno.37362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31903111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1OnsbzM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=133-150&author=Y.+Heauthor=W.+Liauthor=Z.+Zhengauthor=L.+Zhaoauthor=W.+Liauthor=Y.+Wangauthor=H.+Li&title=Inhibition+of+protein+arginine+methyltransferase+6+reduces+reactive+oxygen+species+production+and+attenuates+aminoglycoside-+and+cisplatin-induced+hair+cell+death&doi=10.7150%2Fthno.37362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of protein arginine methyltransferase 6 reduces reactive oxygen species production and attenuates aminoglycoside- and cisplatin-induced hair cell death</span></div><div class="casAuthors">He, Yingzi; Li, Wen; Zheng, Zhiwei; Zhao, Liping; Li, Wenyan; Wang, Yunfeng; Li, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-150</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Hair cells in the inner ear have been shown to be susceptible to ototoxicity from some beneficial pharmaceutical drugs, such as aminoglycosides and cisplatin.  Thus, there is great interest in discovering new targets or compds. that protect hair cells from these ototoxic drugs.  Epigenetic regulation is closely related to inner ear development; however, little is known about epigenetic regulation in the process of ototoxic drugs-induced hearing loss.  Methods: In this study, we investigated the role of protein arginine methyltransferase 6 (PRMT6) in aminoglycoside- and cisplatin-induced hair cell loss by using EPZ020411, a selective small mol. PRMT6 inhibitor, in vitro in neonatal mouse cochlear explants and in vivo in C57BL/6 mice.  We also took advantage of the HEI-OC1 cell line to evaluate the anti-apoptosis effects of PRMT6 knockdown on cisplatin-induced ototoxicity.  Apoptotic cells were identified using cleaved caspase-3 staining and TUNEL assay.  The levels of reactive oxygen species (ROS) were evaluated by DCFH-DA and cellROX green staining.  The mitochondrial membrane potential (ÎÎ¨m) were detd. by JC-1, TMRM, and rhodamine 123 staining.  Results: We found that EPZ020411 significantly alleviated neomycin- and cisplatin-induced cell apoptosis and increased hair cell survival.  Moreover, pretreatment with EPZ020411 could attenuate neomycin- and cisplatin-induced hearing loss in vivo.  Mechanistic studies revealed that inhibition of PRMT6 could reverse the increased expression of caspase-3 and cytochrome c translocation, mitochondrial dysfunction, increased accumulation of ROS, and activation of cell apoptosis after cisplatin injury.  Conclusions: Our findings suggested that PRMT6 might serve as a new therapeutic target to prevent hearing loss caused by aminoglycoside- and cisplatin-induced ototoxicity by preventing ROS formation and modulating the mitochondria-related damage and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofANKtj0EPF7Vg90H21EOLACvtfcHk0li4ejWffCoQjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1OnsbzM&md5=777177eb5f3895a123b4b23864e87b9d</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.7150%2Fthno.37362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.37362%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DInhibition%2520of%2520protein%2520arginine%2520methyltransferase%25206%2520reduces%2520reactive%2520oxygen%2520species%2520production%2520and%2520attenuates%2520aminoglycoside-%2520and%2520cisplatin-induced%2520hair%2520cell%2520death%26jtitle%3DTheranostics%26date%3D2020%26volume%3D10%26spage%3D133%26epage%3D150%26doi%3D10.7150%2Fthno.37362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">9665</span>â <span class="NLM_lpage">9673</span>, <span class="refDoi">Â DOI: 10.1039/C4OB01591F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1039%2FC4OB01591F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25348815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=9665-9673&author=Y.+Xieauthor=R.+Zhouauthor=F.+Lianauthor=Y.+Liuauthor=L.+Chenauthor=Z.+Shiauthor=N.+Zhangauthor=M.+Zhengauthor=B.+Shenauthor=H.+Jiangauthor=Z.+Liangauthor=C.+Luo&title=Virtual+screening+and+biological+evaluation+of+novel+small+molecular+inhibitors+against+protein+arginine+methyltransferase+1+%28PRMT1%29&doi=10.1039%2FC4OB01591F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span></div><div class="casAuthors">Xie, Yiqian; Zhou, Ran; Lian, Fulin; Liu, Yan; Chen, Limin; Shi, Zhe; Zhang, Naixia; Zheng, Mingyue; Shen, Bairong; Jiang, Hualiang; Liang, Zhongjie; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">9665-9673</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methylation is a common post-translational modification which is crucial for a variety of biol. processes.  Dysregulation of protein arginine methyltransferases (PRMTs) activity has been implicated in cancer and other serious diseases.  Thus, small mol. inhibitors against PRMT have great potential for therapeutic development.  Herein, through the combination of virtual screening and bioassays, six small mol. compds. were identified as PRMT1 inhibitors.  Amongst them, the binding affinity of compds. DCLX069 and DCLX078 with PRMT1 was further validated by T1Ï and satn. transfer difference (STD) NMR expts.  Most important of all, both compds. effectively blocked cell proliferation in breast cancer, liver cancer and acute myeloid leukemia cell lines.  The binding mode anal. from mol. docking simulations theor. indicated that both inhibitors occupied the SAM binding pocket to exert the inhibitory effect.  Taken together, our compds. enriched the structural scaffolds as PRMT1 inhibitors and afforded clues for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrinhNjy5ctKbVg90H21EOLACvtfcHk0li4ejWffCoQjA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F&md5=36cb4c315dad033dc9f10a1f21609ab4</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1039%2FC4OB01591F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01591F%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520novel%2520small%2520molecular%2520inhibitors%2520against%2520protein%2520arginine%2520methyltransferase%25201%2520%2528PRMT1%2529%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2014%26volume%3D12%26spage%3D9665%26epage%3D9673%26doi%3D10.1039%2FC4OB01591F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Development of potent type I protein arginine methyltransferase (PRMT) inhibitors of leukemia cell proliferation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8888</span>â <span class="NLM_lpage">8905</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01134</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01134" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8888-8905&author=C.+Wangauthor=H.+Jiangauthor=J.+Jinauthor=Y.+Xieauthor=Z.+Chenauthor=H.+Zhangauthor=F.+Lianauthor=Y.+C.+Liuauthor=C.+Zhangauthor=H.+Dingauthor=S.+Chenauthor=N.+Zhangauthor=Y.+Zhangauthor=H.+Jiangauthor=K.+Chenauthor=F.+Yeauthor=Z.+Yaoauthor=C.+Luo&title=Development+of+potent+type+I+protein+arginine+methyltransferase+%28PRMT%29+inhibitors+of+leukemia+cell+proliferation&doi=10.1021%2Facs.jmedchem.7b01134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Type I Protein Arginine Methyltransferase (PRMT) Inhibitors of Leukemia Cell Proliferation</span></div><div class="casAuthors">Wang, Chen; Jiang, Hao; Jin, Jia; Xie, Yiqian; Chen, Zhifeng; Zhang, Hao; Lian, Fulin; Liu, Yu-Chih; Zhang, Chenhua; Ding, Hong; Chen, Shijie; Zhang, Naixia; Zhang, Yuanyuan; Jiang, Hualiang; Chen, Kaixian; Ye, Fei; Yao, Zhiyi; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8888-8905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein Arginine Methyltransferases (PRMTs) are crucial players in diverse biol. processes, and dysregulation of PRMTs has been linked to various human diseases, esp. cancer.  Therefore, small mols. targeting PRMTs have profound impact for both academic functional studies and clin. disease treatment.  Here, the authors report the discovery of N1-(2-((2-chlorophenyl)thio)benzyl)-N1-methylethane-1,2-diamine (28d, DCPR049_12), a highly potent inhibitor of type I PRMTs that has good selectivity against a panel of other methyltransferases.  Compd. 28d effectively inhibits cell proliferation in several leukemia cell lines and reduces the cellular asym. arginine dimethylation levels.  Serving as an effective inhibitor, 28d demonstrates the mechanism of cell killing in both cell cycle arrest and apoptotic effect as well as downregulation of the pivotal mixed lineage leukemia (MLL) fusion target genes such as HOXA9 and MEIS1, which reflects the crit. roles of type I PRMTs in MLL leukemia.  These studies present 28d as a valuable inhibitor to investigate the role of type I PRMTs in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrddA_Xz0t9VLVg90H21EOLACvtfcHk0liQu1NY3h6CtQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1CisbfN&md5=40953f4e3f8d78075d444451c67ff6e0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01134%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520potent%2520type%2520I%2520protein%2520arginine%2520methyltransferase%2520%2528PRMT%2529%2520inhibitors%2520of%2520leukemia%2520cell%2520proliferation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8888%26epage%3D8905%26doi%3D10.1021%2Facs.jmedchem.7b01134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedoriw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapurkar, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhart, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laraio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zappacosta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques-OâHagan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span> <span> </span><span class="NLM_article-title">Anti-tumor activity of the type I PRMT inhibitor, GSK3368715, synergizes with PRMT5 inhibition through MTAP loss</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">100</span>â <span class="NLM_lpage">114</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2019.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2019.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31257072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1OjsrjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=100-114&author=A.+Fedoriwauthor=S.+R.+Rajapurkarauthor=S.+O%E2%80%99Brienauthor=S.+V.+Gerhartauthor=L.+H.+Mitchellauthor=N.+D.+Adamsauthor=N.+Riouxauthor=T.+Lingarajauthor=S.+A.+Ribichauthor=M.+B.+Pappalardiauthor=N.+Shahauthor=J.+Laraioauthor=Y.+Liuauthor=M.+Butticelloauthor=C.+L.+Carpenterauthor=C.+Creasyauthor=S.+Korenchukauthor=M.+T.+McCabeauthor=C.+F.+McHughauthor=R.+Nagarajanauthor=C.+Wagnerauthor=F.+Zappacostaauthor=R.+Annanauthor=N.+O.+Conchaauthor=R.+A.+Thomasauthor=T.+K.+Hartauthor=J.+J.+Smithauthor=R.+A.+Copelandauthor=M.+P.+Moyerauthor=J.+Campbellauthor=K.+Sticklandauthor=J.+Millsauthor=S.+Jacques-O%E2%80%99Haganauthor=C.+Allainauthor=D.+Johnstonauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=N.+Watersauthor=K.+Swingerauthor=A.+Boriack-Sjodinauthor=T.+Rieraauthor=G.+Shapiroauthor=R.+Chesworthauthor=R.+K.+Prinjhaauthor=R.+G.+Krugerauthor=O.+Barbashauthor=H.+P.+Mohammad&title=Anti-tumor+activity+of+the+type+I+PRMT+inhibitor%2C+GSK3368715%2C+synergizes+with+PRMT5+inhibition+through+MTAP+loss&doi=10.1016%2Fj.ccell.2019.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-tumor Activity of the Type I PRMT Inhibitor, GSK3368715, Synergizes with PRMT5 Inhibition through MTAP Loss</span></div><div class="casAuthors">Fedoriw, Andrew; Rajapurkar, Satyajit R.; O'Brien, Shane; Gerhart, Sarah V.; Mitchell, Lorna H.; Adams, Nicholas D.; Rioux, Nathalie; Lingaraj, Trupti; Ribich, Scott A.; Pappalardi, Melissa B.; Shah, Niyant; Laraio, Jenny; Liu, Yan; Butticello, Michael; Carpenter, Chris L.; Creasy, Caretha; Korenchuk, Susan; McCabe, Michael T.; McHugh, Charles F.; Nagarajan, Raman; Wagner, Craig; Zappacosta, Francesca; Annan, Roland; Concha, Nestor O.; Thomas, Roberta A.; Hart, Timothy K.; Smith, Jesse J.; Copeland, Robert A.; Moyer, Mikel P.; Campbell, John; Stickland, Kim; Mills, James; Jacques-O'Hagan, Suzanne; Allain, Christina; Johnston, Danielle; Raimondi, Alejandra; Porter Scott, Margaret; Waters, Nigel; Swinger, Kerren; Boriack-Sjodin, Ann; Riera, Tom; Shapiro, Gideon; Chesworth, Richard; Prinjha, Rabinder K.; Kruger, Ryan G.; Barbash, Olena; Mohammad, Helai P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-114.e25</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Type I protein arginine methyltransferases (PRMTs) catalyze asym. dimethylation of arginines on proteins.  Type I PRMTs and their substrates have been implicated in human cancers, suggesting inhibition of type I PRMTs may offer a therapeutic approach for oncol.  The current report describes GSK3368715 (EPZ019997), a potent, reversible type I PRMT inhibitor with anti-tumor effects in human cancer models.  Inhibition of PRMT5, the predominant type II PRMT, produces synergistic cancer cell growth inhibition when combined with GSK3368715.  Interestingly, deletion of the methylthioadenosine phosphorylase gene (MTAP) results in accumulation of the metabolite 2-methylthioadenosine, an endogenous inhibitor of PRMT5, and correlates with sensitivity to GSK3368715 in cell lines.  These data provide rationale to explore MTAP status as a biomarker strategy for patient selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqtv1XXQ84BbVg90H21EOLACvtfcHk0liQu1NY3h6CtQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1OjsrjM&md5=68faabf5206504d403441bff26531af6</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DFedoriw%26aufirst%3DA.%26aulast%3DRajapurkar%26aufirst%3DS.%2BR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DGerhart%26aufirst%3DS.%2BV.%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DAdams%26aufirst%3DN.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DShah%26aufirst%3DN.%26aulast%3DLaraio%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DCreasy%26aufirst%3DC.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DNagarajan%26aufirst%3DR.%26aulast%3DWagner%26aufirst%3DC.%26aulast%3DZappacosta%26aufirst%3DF.%26aulast%3DAnnan%26aufirst%3DR.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DThomas%26aufirst%3DR.%2BA.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCampbell%26aufirst%3DJ.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DMills%26aufirst%3DJ.%26aulast%3DJacques-O%25E2%2580%2599Hagan%26aufirst%3DS.%26aulast%3DAllain%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%26aulast%3DBoriack-Sjodin%26aufirst%3DA.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DShapiro%26aufirst%3DG.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26atitle%3DAnti-tumor%2520activity%2520of%2520the%2520type%2520I%2520PRMT%2520inhibitor%252C%2520GSK3368715%252C%2520synergizes%2520with%2520PRMT5%2520inhibition%2520through%2520MTAP%2520loss%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D100%26epage%3D114%26doi%3D10.1016%2Fj.ccell.2019.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K. A.</span></span> <span> </span><span class="NLM_article-title">Novel inhibitors for PRMT1 discovered by high-throughput screening using activity-based fluorescence polarization</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1198</span>â <span class="NLM_lpage">1204</span>, <span class="refDoi">Â DOI: 10.1021/cb300024c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300024c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFeltr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1198-1204&author=M.+B.+Dillonauthor=D.+A.+Bachovchinauthor=S.+J.+Brownauthor=M.+G.+Finnauthor=H.+Rosenauthor=B.+F.+Cravattauthor=K.+A.+Mowen&title=Novel+inhibitors+for+PRMT1+discovered+by+high-throughput+screening+using+activity-based+fluorescence+polarization&doi=10.1021%2Fcb300024c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitors for PRMT1 Discovered by High-Throughput Screening Using Activity-Based Fluorescence Polarization</span></div><div class="casAuthors">Dillon, Myles B. C.; Bachovchin, Daniel A.; Brown, Steven J.; Finn, M. G.; Rosen, Hugh; Cravatt, Benjamin F.; Mowen, Kerri A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1198-1204</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) catalyze the posttranslational methylation of arginine using S-adenosylmethionine (SAM) as a methyl-donor.  The PRMT family is widely expressed and has been implicated in biol. functions such as RNA splicing, transcriptional control, signal transduction, and DNA repair.  Therefore, specific inhibitors of individual PRMTs have potentially significant research and therapeutic value.  In particular, PRMT1 is responsible for >85% of arginine methyltransferase activity, but currently available inhibitors of PRMT1 lack specificity, efficacy, and bioavailability.  To address this limitation, we developed a high-throughput screening assay for PRMT1 that utilizes a hyper-reactive cysteine within the active site, which is lacking in almost all other PRMTs.  This assay, which monitors the kinetics of the fluorescence polarization signal increase upon PRMT1 labeling by a rhodamine-contg. cysteine-reactive probe, successfully identified two novel inhibitors selective for PRMT1 over other SAM-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyzeBUHtpySbVg90H21EOLACvtfcHk0ljjev3SJNuAKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFeltr4%253D&md5=a59bceb8643a97e72bd0508254695fea</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fcb300024c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300024c%26sid%3Dliteratum%253Aachs%26aulast%3DDillon%26aufirst%3DM.%2BB.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DMowen%26aufirst%3DK.%2BA.%26atitle%3DNovel%2520inhibitors%2520for%2520PRMT1%2520discovered%2520by%2520high-throughput%2520screening%2520using%2520activity-based%2520fluorescence%2520polarization%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1198%26epage%3D1204%26doi%3D10.1021%2Fcb300024c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesaroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuchlautz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ¼scher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span> <span> </span><span class="NLM_article-title">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">933</span>â <span class="NLM_lpage">937</span>, <span class="refDoi">Â DOI: 10.1038/nature06166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature06166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=17898714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=933-937&author=E.+Guccioneauthor=C.+Bassiauthor=F.+Casadioauthor=F.+Martinatoauthor=M.+Cesaroniauthor=H.+Schuchlautzauthor=B.+L%C3%BCscherauthor=B.+Amati&title=Methylation+of+histone+H3R2+by+PRMT6+and+H3K4+by+an+MLL+complex+are+mutually+exclusive&doi=10.1038%2Fnature06166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span></div><div class="casAuthors">Guccione, Ernesto; Bassi, Christian; Casadio, Fabio; Martinato, Francesca; Cesaroni, Matteo; Schuchlautz, Henning; Luescher, Bernhard; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">933-937</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Eukaryotic genomes are organized into active (euchromatic) and inactive (heterochromatic) chromatin domains.  Post-translational modifications of histones (or 'marks') are key in defining these functional states, particularly in promoter regions.  Mutual regulatory interactions between these marks-and the enzymes that catalyze them-contribute to the shaping of this epigenetic landscape, in a manner that remains to be fully elucidated.  We previously obsd. that asym. di-methylation of histone H3 arginine 2 (H3R2me2a) counter-correlates with di- and tri- methylation of H3 lysine 4 (H3K4me2, H3K4me3) on human promoters.  Here we show that the arginine methyltransferase PRMT6 catalyzes H3R2 di-methylation in vitro and controls global levels of H3R2me2a in vivo.  H3R2 methylation by PRMT6 was prevented by the presence of H3K4me3 on the H3 tail.  Conversely, the H3R2me2a mark prevented methylation of H3K4 as well as binding to the H3 tail by an ASH2/WDR5/MLL-family methyltransferase complex.  Chromatin immunopptn. showed that H3R2me2a was distributed within the body and at the 3' end of human genes, regardless of their transcriptional state, whereas it was selectively and locally depleted from active promoters, coincident with the presence of H3K4me3.  Hence, the mutual antagonism between H3R2 and H3K4 methylation, together with the assocn. of MLL-family complexes with the basal transcription machinery, may contribute to the localized patterns of H3K4 tri-methylation characteristic of transcriptionally poised or active promoters in mammalian genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWqE8pDpHmKLVg90H21EOLACvtfcHk0ljjev3SJNuAKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK&md5=f4cc9371491088af7b74b614c90d6bd4</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnature06166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06166%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DMartinato%26aufirst%3DF.%26aulast%3DCesaroni%26aufirst%3DM.%26aulast%3DSchuchlautz%26aufirst%3DH.%26aulast%3DL%25C3%25BCscher%26aufirst%3DB.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DMethylation%2520of%2520histone%2520H3R2%2520by%2520PRMT6%2520and%2520H3K4%2520by%2520an%2520MLL%2520complex%2520are%2520mutually%2520exclusive%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D933%26epage%3D937%26doi%3D10.1038%2Fnature06166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vhuiyan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of arginine asymmetrical dimethylation by PRMT6</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>473</i></span>,  <span class="NLM_fpage">3049</span>â <span class="NLM_lpage">3063</span>, <span class="refDoi">Â DOI: 10.1042/BCJ20160537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1042%2FBCJ20160537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27480107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2016&pages=3049-3063&author=H.+Wuauthor=W.+Zhengauthor=M.+S.+Eramauthor=M.+Vhuiyanauthor=A.+Dongauthor=H.+Zengauthor=H.+Heauthor=P.+Brownauthor=A.+Frankelauthor=M.+Vedadiauthor=M.+Luoauthor=J.+Min&title=Structural+basis+of+arginine+asymmetrical+dimethylation+by+PRMT6&doi=10.1042%2FBCJ20160537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of arginine asymmetrical dimethylation by PRMT6</span></div><div class="casAuthors">Wu, Hong; Zheng, Weihong; Eram, Mohammad S.; Vhuiyan, Mynol; Dong, Aiping; Zeng, Hong; He, Hao; Brown, Peter; Frankel, Adam; Vedadi, Masoud; Luo, Minkui; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3049-3063</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">PRMT6 is a type I protein arginine methyltransferase, generating the asym. dimethylarginine mark on proteins such as histone H3R2.  Asym. dimethylation of histone H3R2 by PRMT6 acts as a repressive mark that antagonizes trimethylation of H3 lysine 4 by the MLL histone H3K4 methyltransferase.  PRMT6 is overexpressed in several cancer types, including prostate, bladder and lung cancers; therefore, it is of great interest to develop potent and selective inhibitors for PRMT6.  Here, we report the synthesis of a potent bisubstrate inhibitor GMS [6'-methyleneamine sinefungin, an analog of sinefungin (SNF)], and the crystal structures of human PRMT6 in complex, resp., with S-adenosyl- L-homocysteine (SAH) and the bisubstrate inhibitor GMS that shed light on the significantly improved inhibition effect of GMS on methylation activity of PRMT6 compared with SAH and an S-adenosyl-L-methionine competitive methyltransferase inhibitor SNF.  In addn., we also crystd. PRMT6 in complex with SAH and a short arginine-contg. peptide.  Based on the structural information here and available in the PDB database, we proposed a mechanism that can rationalize the distinctive arginine methylation product specificity of different types of arginine methyltransferases and pinpoint the structural determinant of such a specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo061VtGpp_KLVg90H21EOLACvtfcHk0ljjev3SJNuAKg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D&md5=dcacb06a69eb1a4856bea9b0d8458112</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160537%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DVhuiyan%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520arginine%2520asymmetrical%2520dimethylation%2520by%2520PRMT6%26jtitle%3DBiochem.%2520J.%26date%3D2016%26volume%3D473%26spage%3D3049%26epage%3D3063%26doi%3D10.1042%2FBCJ20160537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowhan, D. H.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2201</span>â <span class="NLM_lpage">2216</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkp1203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1093%2Fnar%2Fgkp1203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20047962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=2201-2216&author=M.+J.+Harrisonauthor=Y.+H.+Tangauthor=D.+H.+Dowhan&title=Protein+arginine+methyltransferase+6+regulates+multiple+aspects+of+gene+expression&doi=10.1093%2Fnar%2Fgkp1203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span></div><div class="casAuthors">Harrison, Matthew J.; Tang, Yue Hang; Dowhan, Dennis H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2201-2216</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It is well established that transcription and alternative splicing events are functionally coupled during gene expression.  Here, we report that protein arginine N-methyltransferase 6 (PRMT6) may play a key role in this coupling process by functioning as a transcriptional coactivator that can also regulate alternative splicing.  PRMT6 coactivates the progesterone, glucocorticoid and estrogen receptors in luciferase reporter assays in a hormone-dependent manner.  In addn., small interfering RNA (siRNA) oligonucleotide duplex knockdown of PRMT6 disrupts estrogen-stimulated transcription of endogenous GREB1 and progesterone receptor in MCF-7 breast cancer cells, demonstrating the importance of PRMT6 in hormone-dependent transcription.  In contrast, the regulation of alternative splicing by PRMT6 is hormone independent.  SiRNA knockdown of PRMT6 increases the exon inclusion:skipping ratio of alternatively spliced exons in endogenous vascular endothelial growth factor and spleen tyrosine kinase RNA transcripts in both the presence and absence of estrogen.  These results demonstrate that PRMT6 has a dual role in regulating gene expression and that these two functions can occur independently of each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA8_-JQGR0vbVg90H21EOLACvtfcHk0lgpekqmOLgJjw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D&md5=471281cfc866a6db8be4a7f8c9a009e5</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1203%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DM.%2BJ.%26aulast%3DTang%26aufirst%3DY.%2BH.%26aulast%3DDowhan%26aufirst%3DD.%2BH.%26atitle%3DProtein%2520arginine%2520methyltransferase%25206%2520regulates%2520multiple%2520aspects%2520of%2520gene%2520expression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D2201%26epage%3D2216%26doi%3D10.1093%2Fnar%2Fgkp1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarrold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. C.</span></span> <span> </span><span class="NLM_article-title">PRMTs and arginine methylation: Cancerâs best-kept secret?</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">993</span>â <span class="NLM_lpage">1009</span>, <span class="refDoi">Â DOI: 10.1016/j.molmed.2019.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.molmed.2019.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31230909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyns7jL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=993-1009&author=J.+Jarroldauthor=C.+C.+Davies&title=PRMTs+and+arginine+methylation%3A+Cancer%E2%80%99s+best-kept+secret%3F&doi=10.1016%2Fj.molmed.2019.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">PRMTs and Arginine Methylation: Cancer's Best-Kept Secret?</span></div><div class="casAuthors">Jarrold, James; Davies, Clare C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">993-1009</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Post-translational modification (PTM) of proteins is vital for increasing proteome diversity and maintaining cellular homeostasis.  If the writing, reading, and removal of modifications are not controlled, cancer can develop.  Arginine methylation is an understudied modification that is increasingly assocd. with cancer progression.  Consequently protein arginine methyltransferases (PRMTs), the writers of arginine methylation, have rapidly gained interest as novel drug targets.  However, for clin. success a deep mechanistic understanding of the biol. of PRMTs is required.  In this review we focus on advances made regarding the role of PRMTs in stem cell biol., epigenetics, splicing, immune surveillance and the DNA damage response, and highlight the rapid rise of specific inhibitors that are now in clin. trials for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVzzJJzCLCjbVg90H21EOLACvtfcHk0lgpekqmOLgJjw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyns7jL&md5=49e14b56c1d3cb73e1776d0f09204bdb</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2019.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2019.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DJarrold%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DC.%2BC.%26atitle%3DPRMTs%2520and%2520arginine%2520methylation%253A%2520Cancer%25E2%2580%2599s%2520best-kept%2520secret%253F%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D993%26epage%3D1009%26doi%3D10.1016%2Fj.molmed.2019.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ã.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent, selective, and cell-active dual inhibitor of protein arginine methyltransferase 4 and protein arginine methyltransferase 6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9124</span>â <span class="NLM_lpage">9139</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9124-9139&author=Y.+Shenauthor=M.+M.+Szewczykauthor=M.+S.+Eramauthor=D.+Smilauthor=H.+%C3%9C.+Kaniskanauthor=R.+Ferreira+de+Freitasauthor=G.+Senisterraauthor=F.+Liauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+potent%2C+selective%2C+and+cell-active+dual+inhibitor+of+protein+arginine+methyltransferase+4+and+protein+arginine+methyltransferase+6&doi=10.1021%2Facs.jmedchem.6b01033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Shen, Yudao; Szewczyk, Magdalena M.; Eram, Mohammad S.; Smil, David; Kaniskan, H. Umit; Ferreira de Freitas, Renato; Senisterra, Guillermo; Li, Fengling; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9124-9139</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chem. tools for testing biol. and therapeutic hypotheses.  Based on the lead compd., a fragment-like inhibitor of type I PRMTs, the authors conducted structure-activity relationship (SAR) studies and explored three regions of this scaffold.  The studies led to the discovery of a potent, selective and cell-active dual inhibitor of PRMT4 and PRMT6, I (MS049).  As compared to the lead, I displayed much improved potency for PRMT4 and PRMT6 in both biochem. and cellular assays.  It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets.  The authors also developed II (MS049N), which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  Considering possible overlapping substrate specificity of PRMTs, I and II are valuable chem. tools for dissecting specific biol. functions and dysregulation of PRMT4 and PRMT6 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZENTB29LcELVg90H21EOLACvtfcHk0lgpekqmOLgJjw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O&md5=2dcdad732107d0a64f7e317a05be1284</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01033%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520dual%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25204%2520and%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9124%26epage%3D9139%26doi%3D10.1021%2Facs.jmedchem.6b01033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchcock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1176</span>â <span class="NLM_lpage">1183</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0lhbTQq7zjwbrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradei, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: Identification of the first PRMT6 tool compound</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">655</span>â <span class="NLM_lpage">659</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+Identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lhbTQq7zjwbrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520Identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabelian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class protein arginine methyltransferase 6 (PRMT6) covalent inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5477</span>â <span class="NLM_lpage">5487</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5477-5487&author=Y.+Shenauthor=F.+Liauthor=M.+M.+Szewczykauthor=L.+Halabelianauthor=K.+S.+Parkauthor=I.+Chauauthor=A.+Dongauthor=H.+Zengauthor=H.+Chenauthor=F.+Mengauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+first-in-class+protein+arginine+methyltransferase+6+%28PRMT6%29+covalent+inhibitor&doi=10.1021%2Facs.jmedchem.0c00406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor</span></div><div class="casAuthors">Shen, Yudao; Li, Fengling; Szewczyk, Magdalena M.; Halabelian, Levon; Park, Kwang-su; Chau, Irene; Dong, Aiping; Zeng, Hong; Chen, He; Meng, Fanye; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5477-5487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biol. processes assocd. with multiple cancers.  Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biol. and therapeutic hypotheses.  Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported.  Based on a cocrystal structure of PRMT6-MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, 4 (MS117)(I).  The covalent binding mode of compd. 4 to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure.  Compd. 4 did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases.  We also developed two structurally similar control compds., 5 (MS167) and 7 (MS168).  We provide these valuable chem. tools to the scientific community for further studying PRMT6 physiol. and pathophysiol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTU1W0CZjN17Vg90H21EOLACvtfcHk0lhbTQq7zjwbrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D&md5=cb244ab5a38d04320439c05048998b2f</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DHalabelian%26aufirst%3DL.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520protein%2520arginine%2520methyltransferase%25206%2520%2528PRMT6%2529%2520covalent%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5477%26epage%3D5487%26doi%3D10.1021%2Facs.jmedchem.0c00406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenthal, R. M.</span></span> <span> </span><span class="NLM_article-title">Mammalian DNA methyltransferases: A structural perspective</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">341</span>â <span class="NLM_lpage">350</span>, <span class="refDoi">Â DOI: 10.1016/j.str.2008.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.str.2008.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=18334209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=341-350&author=X.+Chengauthor=R.+M.+Blumenthal&title=Mammalian+DNA+methyltransferases%3A+A+structural+perspective&doi=10.1016%2Fj.str.2008.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian DNA Methyltransferases: A Structural Perspective</span></div><div class="casAuthors">Cheng, Xiaodong; Blumenthal, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">341-350</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The methylation of mammalian DNA, primarily at CpG dinucleotides, has long been recognized to play a major role in controlling gene expression, among other functions.  Given their importance, it is surprising how many basic questions remain to be answered about the proteins responsible for this methylation and for coordination with the parallel chromatin-marking system that operates at the level of histone modification.  This article reviews recent studies on, and discusses the resulting biochem. and structural insights into, the DNA nucleotide methyltransferase (Dnmt) proteins 1, 3a, 3a2, 3b, and 3L.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr80ixWBaUZh7Vg90H21EOLACvtfcHk0lhX2Oxhip1XQA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsr0%253D&md5=7c0f3ce1acf8c9baac4a33deff36493c</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DBlumenthal%26aufirst%3DR.%2BM.%26atitle%3DMammalian%2520DNA%2520methyltransferases%253A%2520A%2520structural%2520perspective%26jtitle%3DStructure%26date%3D2008%26volume%3D16%26spage%3D341%26epage%3D350%26doi%3D10.1016%2Fj.str.2008.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jurkowska, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurkowski, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span> <span> </span><span class="NLM_article-title">Structure and function of mammalian DNA methyltransferases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">206</span>â <span class="NLM_lpage">222</span>, <span class="refDoi">Â DOI: 10.1002/cbic.201000195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fcbic.201000195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21243710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFWisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=206-222&author=R.+Z.+Jurkowskaauthor=T.+P.+Jurkowskiauthor=A.+Jeltsch&title=Structure+and+function+of+mammalian+DNA+methyltransferases&doi=10.1002%2Fcbic.201000195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of mammalian DNA methyltransferases</span></div><div class="casAuthors">Jurkowska, Renata Zofia; Jurkowski, Tomasz Piotr; Jeltsch, Albert</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-222</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  DNA methylation plays an important role in epigenetic signaling, having an impact on gene regulation, chromatin structure, development, and disease.  Here, the authors review the structure and function of mammalian DNA methyltransferases Dnmt1, Dnmt3a and Dnmt3b, including their domain structures, catalytic mechanisms, localization, regulation, post-translational modifications, and interaction with chromatin and other proteins, summarizing data obtained in genetic, cell biol., and enzymic studies.  The authors focus on the question of how the mol. and enzymic properties of these enzymes are connected to the dynamics of DNA methylation patterns and to the roles the enzymes play in the processes of de novo and maintenance DNA methylation.  Recent enzymic and genome-wide methylome data have led to a new model of genomic DNA methylation patterns based on the preservation of av. levels of DNA methylation in certain regions, rather than the methylation states of individual CG sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-58zM-psVXrVg90H21EOLACvtfcHk0lhX2Oxhip1XQA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFWisw%253D%253D&md5=b1d52bb8bcc5efe6052aaeb0051bc1a6</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000195%26sid%3Dliteratum%253Aachs%26aulast%3DJurkowska%26aufirst%3DR.%2BZ.%26aulast%3DJurkowski%26aufirst%3DT.%2BP.%26aulast%3DJeltsch%26aufirst%3DA.%26atitle%3DStructure%2520and%2520function%2520of%2520mammalian%2520DNA%2520methyltransferases%26jtitle%3DChemBioChem%26date%3D2011%26volume%3D12%26spage%3D206%26epage%3D222%26doi%3D10.1002%2Fcbic.201000195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rondelet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wouters, J.</span></span> <span> </span><span class="NLM_article-title">Human DNA (cytosine-5)-methyltransferases: A functional and structural perspective for epigenetic cancer therapy</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">137</span>â <span class="NLM_lpage">147</span>, <span class="refDoi">Â DOI: 10.1016/j.biochi.2017.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.biochi.2017.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28600135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVantb7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=137-147&author=G.+Rondeletauthor=J.+Wouters&title=Human+DNA+%28cytosine-5%29-methyltransferases%3A+A+functional+and+structural+perspective+for+epigenetic+cancer+therapy&doi=10.1016%2Fj.biochi.2017.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Human DNA (cytosine-5)-methyltransferases: a functional and structural perspective for epigenetic cancer therapy</span></div><div class="casAuthors">Rondelet, Gregoire; Wouters, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137-147</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Epigenetic modifications modulate chromatin states to regulate gene expression.  Among them, DNA methylation and histone modifications play a crucial role in the establishment of the epigenome.  In cancer, these epigenetic events may act in concert to repress tumor suppressor genes or promote oncogenic pathways.  In the context of cancer initiation and progression, recruitment of DNA (cytosine-5)-methyltransferases to specific genomic regions is mainly mediated by histone epigenetic marks, transcription factors and co-regulators as part of a dynamic process.  Herein, we will review these mechanisms and present state-of-the-art of DNA methylation, treatment and development of epigenetic cancer therapies targeting this epigenetic modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4OMz-iVRrHrVg90H21EOLACvtfcHk0lhX2Oxhip1XQA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVantb7E&md5=8cf0fb684fa94f1e058cfcb2986773bc</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2017.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2017.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DRondelet%26aufirst%3DG.%26aulast%3DWouters%26aufirst%3DJ.%26atitle%3DHuman%2520DNA%2520%2528cytosine-5%2529-methyltransferases%253A%2520A%2520functional%2520and%2520structural%2520perspective%2520for%2520epigenetic%2520cancer%2520therapy%26jtitle%3DBiochimie%26date%3D2017%26volume%3D139%26spage%3D137%26epage%3D147%26doi%3D10.1016%2Fj.biochi.2017.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">DNA methyltransferases: Emerging targets for the discovery of inhibitors as potent anticancer drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2323</span>â <span class="NLM_lpage">2331</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2019.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.drudis.2019.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31494187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslKntbvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2323-2331&author=J.+Yuauthor=T.+Xieauthor=Z.+Wangauthor=X.+Wangauthor=S.+Zengauthor=Y.+Kangauthor=T.+Hou&title=DNA+methyltransferases%3A+Emerging+targets+for+the+discovery+of+inhibitors+as+potent+anticancer+drugs&doi=10.1016%2Fj.drudis.2019.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferases: emerging targets for the discovery of inhibitors as potent anticancer drugs</span></div><div class="casAuthors">Yu, Jie; Xie, Tianli; Wang, Zhe; Wang, Xuwen; Zeng, Su; Kang, Yu; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2323-2331</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA methyltransferases (DNMTs) are a conserved family of cytosine methylases with crucial roles in epigenetic regulation.  They have been considered as promising therapeutic targets for the epigenetic treatment of cancer.  Therefore, DNMT inhibitors (DNMTis) have attracted considerable interest in recent years for the modulation of the aberrant DNA methylation pattern in a reversible way.  In this review, it provide a structure-based overview of the therapeutic importance of DNMTs against different cancer types, and then summarize recently investigated DNMTis as well as their inhibitory mechanisms, focusing on recent advances in the development of DNMTis with specificity and/or selectivity using computational approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIGX9s11xzIbVg90H21EOLACvtfcHk0lgCsFpWbL9xig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslKntbvF&md5=b191f9753cd011fc2129d918b1dde3a5</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DDNA%2520methyltransferases%253A%2520Emerging%2520targets%2520for%2520the%2520discovery%2520of%2520inhibitors%2520as%2520potent%2520anticancer%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D2323%26epage%3D2331%26doi%3D10.1016%2Fj.drudis.2019.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span> <span> </span><span class="NLM_article-title">DNA methyltransferase inhibitors and their therapeutic potential</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2448</span>â <span class="NLM_lpage">2457</span>, <span class="refDoi">Â DOI: 10.2174/1568026619666181120150122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2174%2F1568026619666181120150122" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=2448-2457&author=Z.+Zhouauthor=H.+Q.+Liauthor=F.+Liu&title=DNA+methyltransferase+inhibitors+and+their+therapeutic+potential&doi=10.2174%2F1568026619666181120150122"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.2174%2F1568026619666181120150122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026619666181120150122%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%2BQ.%26aulast%3DLiu%26aufirst%3DF.%26atitle%3DDNA%2520methyltransferase%2520inhibitors%2520and%2520their%2520therapeutic%2520potential%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2019%26volume%3D18%26spage%3D2448%26epage%3D2457%26doi%3D10.2174%2F1568026619666181120150122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pechalrieu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Etievant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">DNA methyltransferase inhibitors in cancer: From pharmacology to translational studies</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">13</span>, <span class="refDoi">Â DOI: 10.1016/j.bcp.2016.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.bcp.2016.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27956110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOjs7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=1-13&author=D.+Pechalrieuauthor=C.+Etievantauthor=P.+B.+Arimondo&title=DNA+methyltransferase+inhibitors+in+cancer%3A+From+pharmacology+to+translational+studies&doi=10.1016%2Fj.bcp.2016.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferase inhibitors in cancer: From pharmacology to translational studies</span></div><div class="casAuthors">Pechalrieu, Dany; Etievant, Chantal; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA methylation is a mammalian epigenetic mark that participates to define where and when genes are expressed, both in normal cells and in the context of diseases.  Like other epigenetic marks, it is reversible and can be modulated by chem. agents.  Because it plays an important role in cancer by silencing certain genes, such as tumor suppressor genes, it is a promising therapeutic target.  Two compds. are already approved to treat hematol. cancers, and many efforts have been carried out to discover new mols. that inhibit DNA methyltransferases, the enzymes responsible for DNA methylation.  Here, we analyze the mol. mechanisms and cellular pharmacol. of these inhibitors, pointing out the necessity for new pharmacol. models and paradigms.  The parameters of pharmacol. responses need to be redefined: the aim is cellular reprogramming rather than general cytotoxicity.  Thus, "epigenetic" rather than cytotoxic dosages are defined.  Another issue is the delay of the response: cellular reprogramming can take several generations to produce observable phenotypes.  Is this compatible with lab. scale expts.  Finally, it is important to consider the specificity for cancer cells compared to normal cells and the appearance of resistance.  We also discuss different techniques that are used and the selection of pharmacol. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj2Zvsz9F1krVg90H21EOLACvtfcHk0lgCsFpWbL9xig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOjs7%252FP&md5=3a835b4c437050d8a09c35d8357c1c56</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DPechalrieu%26aufirst%3DD.%26aulast%3DEtievant%26aufirst%3DC.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DDNA%2520methyltransferase%2520inhibitors%2520in%2520cancer%253A%2520From%2520pharmacology%2520to%2520translational%2520studies%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D129%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.bcp.2016.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA methylation with small molecules: Whatâs next?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2569</span>â <span class="NLM_lpage">2583</span>, <span class="refDoi">Â DOI: 10.1021/jm500843d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500843d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2569-2583&author=A.+Erdmannauthor=L.+Halbyauthor=J.+Fahyauthor=P.+B.+Arimondo&title=Targeting+DNA+methylation+with+small+molecules%3A+What%E2%80%99s+next%3F&doi=10.1021%2Fjm500843d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA Methylation with Small Molecules: What's Next?</span></div><div class="casAuthors">Erdmann, Alexandre; Halby, Ludovic; Fahy, Jacques; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2569-2583</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA methylation is a mammalian epigenetic mark that is involved in defining where and when genes are expressed, both in normal cells and in the context of diseases.  Like other epigenetic marks, it is reversible and can be modulated by chem. agents.  Because it plays an important role in cancer by silencing certain genes, such as tumor suppressor genes, and by reactivating other regions, such as repeated elements, it is a promising therapeutic target.  Two compds. are already approved to treat hematol. cancers.  Many efforts have been carried out to discover new mols. that are able to efficiently inhibit DNA methylation in cancer cells.  We will briefly overview the foremost of these efforts by focusing on what we have learned to this point on non-nucleoside inhibitors and on what we consider to be the features of an ideal inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9LcXQUubTNbVg90H21EOLACvtfcHk0lgCsFpWbL9xig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L&md5=cc469a073e82dc2cbe2d822cd8c02cce</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1021%2Fjm500843d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500843d%26sid%3Dliteratum%253Aachs%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DFahy%26aufirst%3DJ.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DTargeting%2520DNA%2520methylation%2520with%2520small%2520molecules%253A%2520What%25E2%2580%2599s%2520next%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2569%26epage%3D2583%26doi%3D10.1021%2Fjm500843d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Derissen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijnen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H.</span></span> <span> </span><span class="NLM_article-title">Concise drug review: Azacitidine and decitabine</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">619</span>â <span class="NLM_lpage">624</span>, <span class="refDoi">Â DOI: 10.1634/theoncologist.2012-0465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1634%2Ftheoncologist.2012-0465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=23671007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXps1Cktbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=619-624&author=E.+J.+Derissenauthor=J.+H.+Beijnenauthor=J.+H.+Schellens&title=Concise+drug+review%3A+Azacitidine+and+decitabine&doi=10.1634%2Ftheoncologist.2012-0465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Concise drug review: azacitidine and decitabine</span></div><div class="casAuthors">Derissen, Ellen J. B.; Beijnen, Jos H.; Schellens, Jan H. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">619-624</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">This concise drug review summarizes the current state of treatment with the hypomethylating agents azacitidine and decitabine.  The hypomethylating agents azacitidine and decitabine (5-aza-2'-deoxycytidine) for the treatment of several specific forms of myelodysplastic syndromes (MDS), chronic myelomonocytic leukemia (CMML), and acute myeloid leukemia (AML) were discussed.  The introduction of the hypomethylating agents has been a major advancement in the treatment of patients with higher-risk MDS who are ineligible for allogeneic stem cell transplantation (allo-SCT).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo52eea6V6hobVg90H21EOLACvtfcHk0ljrmsoe5SIqig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXps1Cktbo%253D&md5=131d4a854d1fc6d3986da9b5ae21e144</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2012-0465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2012-0465%26sid%3Dliteratum%253Aachs%26aulast%3DDerissen%26aufirst%3DE.%2BJ.%26aulast%3DBeijnen%26aufirst%3DJ.%2BH.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%26atitle%3DConcise%2520drug%2520review%253A%2520Azacitidine%2520and%2520decitabine%26jtitle%3DOncologist%26date%3D2013%26volume%3D18%26spage%3D619%26epage%3D624%26doi%3D10.1634%2Ftheoncologist.2012-0465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stresemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyko, F.</span></span> <span> </span><span class="NLM_article-title">Modes of action of the DNA methyltransferase inhibitors azacytidine and decitabine</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">8</span>â <span class="NLM_lpage">13</span>, <span class="refDoi">Â DOI: 10.1002/ijc.23607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fijc.23607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=18425818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtlantrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2008&pages=8-13&author=C.+Stresemannauthor=F.+Lyko&title=Modes+of+action+of+the+DNA+methyltransferase+inhibitors+azacytidine+and+decitabine&doi=10.1002%2Fijc.23607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Modes of action of the DNA methyltransferase inhibitors azacytidine and decitabine</span></div><div class="casAuthors">Stresemann, Carlo; Lyko, Frank</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-13</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  The cytosine analogs 5-azacytosine (azacytidine) and 2'-deoxy-5-azacytidine (decitabine) are the currently most advanced drugs for epigenetic cancer therapies.  These compds. function as DNA methyltransferase inhibitors and have shown substantial potency in reactivating epigenetically silenced tumor suppressor genes in vitro.  However, it has been difficult to define the mode of action of these drugs in patients and it appears that clin. responses are influenced both by epigenetic alterations and by apoptosis induction.  To maximize the clin. efficacy of azacytidine and decitabine it will be important to understand the mol. changes induced by these drugs.  In this review, we examine the pharmacol. properties of azanucleosides and their interactions with various cellular pathways.  Because azacytidine and decitabine are prodrugs, an understanding of the cellular mechanisms mediating transmembrane transport and metabolic activation will be critically important for optimizing patient responses.  We also discuss the mechanism of DNA methyltransferase inhibition and emphasize the need for the identification of predictive biomarkers for the further advancement of epigenetic therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfTNmdOvDjg7Vg90H21EOLACvtfcHk0ljrmsoe5SIqig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtlantrg%253D&md5=25154a9967a675a3dd93bc3dd7f08191</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1002%2Fijc.23607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.23607%26sid%3Dliteratum%253Aachs%26aulast%3DStresemann%26aufirst%3DC.%26aulast%3DLyko%26aufirst%3DF.%26atitle%3DModes%2520of%2520action%2520of%2520the%2520DNA%2520methyltransferase%2520inhibitors%2520azacytidine%2520and%2520decitabine%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2008%26volume%3D123%26spage%3D8%26epage%3D13%26doi%3D10.1002%2Fijc.23607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsen, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquez, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selker, E. U.</span></span> <span> </span><span class="NLM_article-title">Inhibition of DNA methylation and reactivation of silenced genes by zebularine</span>. <i>J. Natl. Cancer Ins.t.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">399</span>â <span class="NLM_lpage">409</span>, <span class="refDoi">Â DOI: 10.1093/jnci/95.5.399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1093%2Fjnci%2F95.5.399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=12618505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVeqs74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2003&pages=399-409&author=J.+C.+Chengauthor=C.+B.+Matsenauthor=F.+A.+Gonzalesauthor=W.+Yeauthor=S.+Greerauthor=V.+E.+Marquezauthor=P.+A.+Jonesauthor=E.+U.+Selker&title=Inhibition+of+DNA+methylation+and+reactivation+of+silenced+genes+by+zebularine&doi=10.1093%2Fjnci%2F95.5.399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DNA methylation and reactivation of silenced genes by zebularine</span></div><div class="casAuthors">Cheng, Jonathan C.; Matsen, Cindy B.; Gonzales, Felicidad A.; Ye, Wei; Greer, Sheldon; Marquez, Victor E.; Jones, Peter A.; Selker, Eric U.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">399-409</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Gene silencing by abnormal methylation of promoter regions of regulatory genes is commonly assocd. with cancer.  Silenced tumor suppressor genes are obvious targets for reactivation by methylation inhibitors such as 5-azacytidine (5-Aza-CR) and 5-aza-2'-deoxycytidine (5-Aza-CdR).  However, both compds. are chem. unstable and toxic and neither can be given orally.  We characterized a new demethylating agent, zebularine [1-(beta-D-ribofuranosyl)-1,2-dihydropyrimidin-2-one], which is a chem. stable cytidine analog.  We tested the ability of zebularine to reactivate a silenced Neurospora crassa gene using a hygromycin gene reactivation assay.  We then analyzed the ability of zebularine to inhibit DNA methylation in C3H 10T1/2 Cl8 (10T1/2) mouse embryo cells as assayed by induction of a myogenic phenotype and in T24 human bladder carcinoma cells, using the methylation-sensitive single nucleotide primer extension (Ms-SNuPE) assay.  We also evaluated the effects of zebularine (administered orally or i.p.) on growth of EJ6 human bladder carcinoma cells grown in BALB/c nu/nu mice (five mice per group) and the in vivo reactivation of a methylated p16 gene in these cells.  All statistical tests were two-sided.  In N. crassa, zebularine inhibited DNA methylation and reactivated a gene previously silenced by methylation.  Zebularine induced the myogenic phenotype in 10T1/2 cells, which is a phenomenon unique to DNA methylation inhibitors.  Zebularine reactivated a silenced p16 gene and demethylated its promoter region in T24 bladder carcinoma cells in vitro and in tumors grown in mice.  Zebularine was only slightly cytotoxic to T24 cells in vitro (1 mM zebularine for 48 h decreased plating efficiency by 17% [95% confidence interval (CI) = 12.8% to 21.2%]) and to tumor-bearing mice (av. maximal wt. change in mice treated with 1000 mg/kg zebularine = 11% [95% CI = 4% to 19%]).  Compared with those in control mice, tumor vols. were statistically significantly reduced in mice treated with high-dose zebularine administered by i.p. injection (P<.001) or by oral gavage (P<.001).  Zebularine is a stable DNA demethylating agent and the first drug in its class able to reactivate an epigenetically silenced gene by oral administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuC5AqIpr-IbVg90H21EOLACvtfcHk0ljrmsoe5SIqig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVeqs74%253D&md5=3c4016cb5147926157077807f569060d</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F95.5.399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F95.5.399%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DJ.%2BC.%26aulast%3DMatsen%26aufirst%3DC.%2BB.%26aulast%3DGonzales%26aufirst%3DF.%2BA.%26aulast%3DYe%26aufirst%3DW.%26aulast%3DGreer%26aufirst%3DS.%26aulast%3DMarquez%26aufirst%3DV.%2BE.%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DSelker%26aufirst%3DE.%2BU.%26atitle%3DInhibition%2520of%2520DNA%2520methylation%2520and%2520reactivation%2520of%2520silenced%2520genes%2520by%2520zebularine%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Ins.t.%26date%3D2003%26volume%3D95%26spage%3D399%26epage%3D409%26doi%3D10.1093%2Fjnci%2F95.5.399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickman, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurd, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornby, D. P.</span></span> <span> </span><span class="NLM_article-title">Zebularine: A novel DNA methylation inhibitor that forms a covalent complex with DNA methyltransferases</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">591</span>â <span class="NLM_lpage">599</span>, <span class="refDoi">Â DOI: 10.1016/S0022-2836(02)00676-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2FS0022-2836%2802%2900676-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=12206775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wmtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2002&pages=591-599&author=L.+Zhouauthor=X.+Chengauthor=B.+A.+Connollyauthor=M.+J.+Dickmanauthor=P.+J.+Hurdauthor=D.+P.+Hornby&title=Zebularine%3A+A+novel+DNA+methylation+inhibitor+that+forms+a+covalent+complex+with+DNA+methyltransferases&doi=10.1016%2FS0022-2836%2802%2900676-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Zebularine: A Novel DNA Methylation Inhibitor that Forms a Covalent Complex with DNA Methyltransferases</span></div><div class="casAuthors">Zhou, L.; Cheng, X.; Connolly, B. A.; Dickman, M. J.; Hurd, P. J.; Hornby, D. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">591-599</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Mechanism-based inhibitors of enzymes, which mimic reactive intermediates in the reaction pathway, have been deployed extensively in the anal. of metabolic pathways and as candidate drugs.  The inhibition of cytosine-[C5]-specific DNA methyltransferases (C5 MTases) by oligodeoxynucleotides contg. 5-azadeoxycytidine (AzadC) and 5-fluorodeoxycytidine (FdC) provides a well-documented example of mechanism-based inhibition of enzymes central to nucleic acid metab.  Here, we describe the interaction between the C5 MTase from Haemophilus haemolyticus (M.HhaI) and an oligodeoxynucleotide duplex contg. 2-H pyrimidinone, an analog often referred to as zebularine and known to give rise to high-affinity complexes with MTases.  X-ray crystallog. has demonstrated the formation of a covalent bond between M.HhaI and the 2-H pyrimidinone-contg. oligodeoxynucleotide.  This observation enables a comparison between the mechanisms of action of 2-H pyrimidinone with other mechanism-based inhibitors such as FdC.  This novel complex provides a mol. explanation for the mechanism of action of the anti-cancer drug zebularine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTq4bolT8QM7Vg90H21EOLACvtfcHk0ljrmsoe5SIqig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wmtbs%253D&md5=aeced0fbc4a800b82d117a0d1265042e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900676-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900676-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DConnolly%26aufirst%3DB.%2BA.%26aulast%3DDickman%26aufirst%3DM.%2BJ.%26aulast%3DHurd%26aufirst%3DP.%2BJ.%26aulast%3DHornby%26aufirst%3DD.%2BP.%26atitle%3DZebularine%253A%2520A%2520novel%2520DNA%2520methylation%2520inhibitor%2520that%2520forms%2520a%2520covalent%2520complex%2520with%2520DNA%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D321%26spage%3D591%26epage%3D599%26doi%3D10.1016%2FS0022-2836%2802%2900676-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bearss, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span> <span> </span><span class="NLM_article-title">S110, a 5-Aza-2â²-deoxycytidine-containing dinucleotide, is an effective DNA methylation inhibitor in vivo and can reduce tumor growth</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1443</span>â <span class="NLM_lpage">1450</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-09-1048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1535-7163.MCT-09-1048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20442312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslektbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1443-1450&author=J.+C.+Chuangauthor=S.+L.+Warnerauthor=D.+Vollmerauthor=H.+Vankayalapatiauthor=S.+Redkarauthor=D.+J.+Bearssauthor=X.+Qiuauthor=C.+B.+Yooauthor=P.+A.+Jones&title=S110%2C+a+5-Aza-2%E2%80%B2-deoxycytidine-containing+dinucleotide%2C+is+an+effective+DNA+methylation+inhibitor+in+vivo+and+can+reduce+tumor+growth&doi=10.1158%2F1535-7163.MCT-09-1048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">S110, a 5-Aza-2'-Deoxycytidine-Containing Dinucleotide, Is an Effective DNA Methylation Inhibitor In vivo and Can Reduce Tumor Growth</span></div><div class="casAuthors">Chuang, Jody C.; Warner, Steven L.; Vollmer, David; Vankayalapati, Hariprasad; Redkar, Sanjeev; Bearss, David J.; Qiu, Xiangning; Yoo, Christine B.; Jones, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1443-1450</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Methylation of CpG islands in promoter regions is often assocd. with gene silencing and aberrant DNA methylation occurs in most cancers, leading to the silencing of some tumor suppressor genes.  Reversal of this abnormal hypermethylation by DNA methylation inhibitors is effective in reactivating methylation-silenced tumor suppressor genes both in vitro and in vivo.  Several DNA methylation inhibitors have been well studied; the most potent among them is 5-aza-2'-deoxycytidine (5-Aza-CdR), which can induce myelosuppression in patients.  S110 is a dinucleotide consisting of 5-Aza-CdR followed by a deoxyguanosine, which we previously showed to be effective in vitro as a DNA methylation inhibitor while being less prone to deamination by cytidine deaminase, making it a promising alternative to 5-Aza-CdR.  Here, we show that S110 is better tolerated than 5-Aza-CdR in mice and is as effective in vivo in inducing p16 expression, reducing DNA methylation at the p16 promoter region, and retarding tumor growth in human xenograft.  We also show that S110 is effective by both i.p. and s.c. deliveries.  S110 therefore is a promising new agent that acts similarly to 5-Aza-CdR and has better stability and less toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMMN8iuoWq47Vg90H21EOLACvtfcHk0lh_2mO76-tO1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslektbg%253D&md5=538700770fa7ae214fccf238726e8e7d</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-1048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-1048%26sid%3Dliteratum%253Aachs%26aulast%3DChuang%26aufirst%3DJ.%2BC.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DVollmer%26aufirst%3DD.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DYoo%26aufirst%3DC.%2BB.%26aulast%3DJones%26aufirst%3DP.%2BA.%26atitle%3DS110%252C%2520a%25205-Aza-2%25E2%2580%25B2-deoxycytidine-containing%2520dinucleotide%252C%2520is%2520an%2520effective%2520DNA%2520methylation%2520inhibitor%2520in%2520vivo%2520and%2520can%2520reduce%2520tumor%2520growth%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D1443%26epage%3D1450%26doi%3D10.1158%2F1535-7163.MCT-09-1048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daher-Reyes, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchan, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K. W. L.</span></span> <span> </span><span class="NLM_article-title">Guadecitabine (SGI-110): An investigational drug for the treatment of myelodysplastic syndrome and acute myeloid leukemia</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">835</span>â <span class="NLM_lpage">849</span>, <span class="refDoi">Â DOI: 10.1080/13543784.2019.1667331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F13543784.2019.1667331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31510809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVSgurfJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=835-849&author=G.+S.+Daher-Reyesauthor=B.+M.+Merchanauthor=K.+W.+L.+Yee&title=Guadecitabine+%28SGI-110%29%3A+An+investigational+drug+for+the+treatment+of+myelodysplastic+syndrome+and+acute+myeloid+leukemia&doi=10.1080%2F13543784.2019.1667331"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Guadecitabine (SGI-110): an investigational drug for the treatment of myelodysplastic syndrome and acute myeloid leukemia</span></div><div class="casAuthors">Daher-Reyes, Georgina S.; Merchan, Brayan M.; Yee, Karen W. L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">835-849</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : The incidence of acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) is increasing with the aging population.  Prognosis and overall survival (OS) remain poor in elderly patients and in those not eligible for intensive treatment.  Hypomethylating agents (HMAs) have played an important role in this group of patients but their efficacy is limited.: This article reviews the mechanism of action, pharmacol., safety profile and clin. efficacy of s.c. guadecitabine, a second-generation DNA methylation inhibitor in development for the treatment of AML and MDS.: Although guadecitabine did not yield improved complete remission (CR) rates and OS compared to the control arm in patients with treatment-naive AML who were ineligible for intensive chemotherapy, subgroup anal. in patients who received â¥4 cycles of therapy demonstrated superior outcomes in favor of guadecitabine.  Given its stability, ease of administration, safety profile and prolonged exposure time, guadecitabine would be the more appropriate HMA, replacing azacitidine and decitabine, to be used combination treatment regimens in patients with myeloid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS4NKimdcQ1bVg90H21EOLACvtfcHk0lh_2mO76-tO1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVSgurfJ&md5=09dcfa3ca3050d20aff1e69a6aa0dbc9</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1080%2F13543784.2019.1667331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543784.2019.1667331%26sid%3Dliteratum%253Aachs%26aulast%3DDaher-Reyes%26aufirst%3DG.%2BS.%26aulast%3DMerchan%26aufirst%3DB.%2BM.%26aulast%3DYee%26aufirst%3DK.%2BW.%2BL.%26atitle%3DGuadecitabine%2520%2528SGI-110%2529%253A%2520An%2520investigational%2520drug%2520for%2520the%2520treatment%2520of%2520myelodysplastic%2520syndrome%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2019%26volume%3D28%26spage%3D835%26epage%3D849%26doi%3D10.1080%2F13543784.2019.1667331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duchmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzykson, R.</span></span> <span> </span><span class="NLM_article-title">Clinical update on hypomethylating agents</span>. <i>Int. J. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">161</span>â <span class="NLM_lpage">169</span>, <span class="refDoi">Â DOI: 10.1007/s12185-019-02651-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1007%2Fs12185-019-02651-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31020568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FovFOmuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2019&pages=161-169&author=M.+Duchmannauthor=R.+Itzykson&title=Clinical+update+on+hypomethylating+agents&doi=10.1007%2Fs12185-019-02651-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical update on hypomethylating agents</span></div><div class="casAuthors">Duchmann Matthieu; Itzykson Raphael; Duchmann Matthieu; Itzykson Raphael</div><div class="citationInfo"><span class="NLM_cas:title">International journal of hematology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-169</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hypomethylating agents (HMAs), azacitidine and decitabine, are standards of care in higher-risk myelodysplastic syndromes and in acute myeloid leukemia patients ineligible for intensive therapy.  Over the last 10 years, research efforts have sought to better understand their mechanism of action, both at the molecular and cellular level.  These efforts have yet to robustly identify biomarkers for these agents.  The clinical activity of HMAs in myeloid neoplasms has been firmly established now but still remains of limited magnitude.  Besides optimized use at different stages of the disease, most of the expected clinical progress with HMAs will come from the development of second-generation compounds orally available and/or with improved pharmacokinetics, and from the search, so far mostly empirical, of HMA-based synergistic drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKdrpYE4wMEnthwrmtH4OBfW6udTcc2ebkYZsZykeaArntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FovFOmuw%253D%253D&md5=e1c1e7abf621defaf7af3caabc6871c4</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1007%2Fs12185-019-02651-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12185-019-02651-9%26sid%3Dliteratum%253Aachs%26aulast%3DDuchmann%26aufirst%3DM.%26aulast%3DItzykson%26aufirst%3DR.%26atitle%3DClinical%2520update%2520on%2520hypomethylating%2520agents%26jtitle%3DInt.%2520J.%2520Hematol.%26date%3D2019%26volume%3D110%26spage%3D161%26epage%3D169%26doi%3D10.1007%2Fs12185-019-02651-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghoshal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamage, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooke, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phiasivongsa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S. T.</span></span> <span> </span><span class="NLM_article-title">A new class of quinoline-based DNA hypomethylating agents reactivates tumor suppressor genes by blocking DNA methyltransferase 1 activity and inducing its degradation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">4277</span>â <span class="NLM_lpage">4285</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-08-3669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F0008-5472.CAN-08-3669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=19417133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFWltLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=4277-4285&author=J.+Dattaauthor=K.+Ghoshalauthor=W.+A.+Dennyauthor=S.+A.+Gamageauthor=D.+G.+Brookeauthor=P.+Phiasivongsaauthor=S.+Redkarauthor=S.+T.+Jacob&title=A+new+class+of+quinoline-based+DNA+hypomethylating+agents+reactivates+tumor+suppressor+genes+by+blocking+DNA+methyltransferase+1+activity+and+inducing+its+degradation&doi=10.1158%2F0008-5472.CAN-08-3669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">A New Class of Quinoline-Based DNA Hypomethylating Agents Reactivates Tumor Suppressor Genes by Blocking DNA Methyltransferase 1 Activity and Inducing Its Degradation</span></div><div class="casAuthors">Datta, Jharna; Ghoshal, Kalpana; Denny, William A.; Gamage, Swarna A.; Brooke, Darby G.; Phiasivongsa, Pasit; Redkar, Sanjeev; Jacob, Samson T.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4277-4285</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Reactivation of silenced tumor suppressor genes by 5-azacytidine (Vidaza) and its congener 5-aza-2'-deoxycytidine (decitabine) has provided an alternate approach to cancer therapy.  We have shown previously that these drugs selectively and rapidly induce degrdn. of the maintenance DNA methyltransferase (DNMT) 1 by a proteasomal pathway.  Because the toxicity of these compds. is largely due to their incorporation into DNA, it is crit. to explore novel, nonnucleoside compds. that can effectively reactivate the silenced genes.  Here, we report that a quinoline-based compd., designated SGI-1027, inhibits the activity of DNMT1, DNMT3A, and DNMT3B as well M. SssI with comparable IC50 (6-13 Î¼mol/L) by competing with S-adenosylmethionine in the methylation reaction.  Treatment of different cancer cell lines with SGI-1027 resulted in selective degrdn. of DNMT1 with minimal or no effects on DNMT3A and DNMT3B.  At a concn. of 2.5 to 5 Î¼mol/L (similar to that of decitabine), complete degrdn. of DNMT1 protein was achieved within 24 h without significantly affecting its mRNA level.  MG132 blocked SGI-1027-induced depletion of DNMT1, indicating the involvement of proteasomal pathway.  Prolonged treatment of RKO cells with SGI-1027 led to demethylation and reexpression of the silenced tumor suppressor genes P16, MLH1, and TIMP3.  Further, this compd. did not exhibit significant toxicity in a rat hepatoma (H4IIE) cell line.  This study provides a novel class of DNA hypomethylating agents that have the potential for use in epigenetic cancer therapy. [Cancer Res 2009;69(10):4277-85].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlNfytT2_KPbVg90H21EOLACvtfcHk0lh1pabnfw--Jg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFWltLs%253D&md5=41a6321d36d19b46bc48171f230357a8</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3669%26sid%3Dliteratum%253Aachs%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DGhoshal%26aufirst%3DK.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DGamage%26aufirst%3DS.%2BA.%26aulast%3DBrooke%26aufirst%3DD.%2BG.%26aulast%3DPhiasivongsa%26aufirst%3DP.%26aulast%3DRedkar%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%2BT.%26atitle%3DA%2520new%2520class%2520of%2520quinoline-based%2520DNA%2520hypomethylating%2520agents%2520reactivates%2520tumor%2520suppressor%2520genes%2520by%2520blocking%2520DNA%2520methyltransferase%25201%2520activity%2520and%2520inducing%2520its%2520degradation%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D4277%26epage%3D4285%26doi%3D10.1158%2F0008-5472.CAN-08-3669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asgatay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champion, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marloie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drujon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senamaud-Beaufort, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajavelu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schambel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lequin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoyan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guianvarcâh, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of analogues of <i>N</i>-phthaloyl-l-tryptophan (RG108) as inhibitors of DNA methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">421</span>â <span class="NLM_lpage">434</span>, <span class="refDoi">Â DOI: 10.1021/jm401419p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401419p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegsLjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=421-434&author=S.+Asgatayauthor=C.+Championauthor=G.+Marloieauthor=T.+Drujonauthor=C.+Senamaud-Beaufortauthor=A.+Ceccaldiauthor=A.+Erdmannauthor=A.+Rajaveluauthor=P.+Schambelauthor=A.+Jeltschauthor=O.+Lequinauthor=P.+Karoyanauthor=P.+B.+Arimondoauthor=D.+Guianvarc%E2%80%99h&title=Synthesis+and+evaluation+of+analogues+of+N-phthaloyl-l-tryptophan+%28RG108%29+as+inhibitors+of+DNA+methyltransferase+1&doi=10.1021%2Fjm401419p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Analogues of N-Phthaloyl-L-tryptophan (RG108) as Inhibitors of DNA Methyltransferase 1</span></div><div class="casAuthors">Asgatay, Saadia; Champion, Christine; Marloie, Gael; Drujon, Thierry; Senamaud-Beaufort, Catherine; Ceccaldi, Alexandre; Erdmann, Alexandre; Rajavelu, Arumugam; Schambel, Philippe; Jeltsch, Albert; Lequin, Olivier; Karoyan, Philippe; Arimondo, Paola B.; Guianvarc'h, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">421-434</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMT) are promising drug targets in cancer provided that new, more specific, and chem. stable inhibitors are discovered.  Among the non-nucleoside DNMT inhibitors, N-phthaloyl-L-tryptophan (RG108) was first identified as inhibitor of DNMT1.  Here, RG108 analogs were synthesized to understand its interaction with DNMT.  The indole, carboxylate, and phthalimide moieties were modified.  Homologated and conformationally constrained analogs were prepd.  The latter were synthesized from prolino-homotryptophan derivs. through a methodol. based amino-zinc-ene-enolate cyclization.  All compds. were tested for their ability to inhibit DNMT1 in vitro.  Among them, constrained compds. e.g., (2S,3R)-3-((1H-indol-3-yl)methyl)-1-(2-naphthoyl)pyrrolidine-2-carboxylic acid and NPys derivs. e.g., N2-[(3-nitro-2-pyridinyl)thio]-L-tryptophan were at least 10-fold more potent than the ref. compd.  The cytotoxicity on the tumor DU145 cell line of the most potent inhibitors was correlated to their inhibitory potency.  Finally, docking studies were conducted to understand their binding mode.  This study provides insights for the design of the next-generation of DNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxxOlrdGxBwrVg90H21EOLACvtfcHk0lh1pabnfw--Jg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegsLjM&md5=d35b180c01308c55c9f6df2f24334ec2</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fjm401419p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401419p%26sid%3Dliteratum%253Aachs%26aulast%3DAsgatay%26aufirst%3DS.%26aulast%3DChampion%26aufirst%3DC.%26aulast%3DMarloie%26aufirst%3DG.%26aulast%3DDrujon%26aufirst%3DT.%26aulast%3DSenamaud-Beaufort%26aufirst%3DC.%26aulast%3DCeccaldi%26aufirst%3DA.%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DRajavelu%26aufirst%3DA.%26aulast%3DSchambel%26aufirst%3DP.%26aulast%3DJeltsch%26aufirst%3DA.%26aulast%3DLequin%26aufirst%3DO.%26aulast%3DKaroyan%26aufirst%3DP.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DGuianvarc%25E2%2580%2599h%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520analogues%2520of%2520N-phthaloyl-l-tryptophan%2520%2528RG108%2529%2520as%2520inhibitors%2520of%2520DNA%2520methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D421%26epage%3D434%26doi%3D10.1021%2Fjm401419p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyko, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Constrained analogues of procaine as novel small molecule inhibitors of DNA methyltransferase-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2321</span>â <span class="NLM_lpage">2325</span>, <span class="refDoi">Â DOI: 10.1021/jm7015705</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7015705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlOjsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2321-2325&author=S.+Castellanoauthor=D.+Kuckauthor=M.+Salaauthor=E.+Novellinoauthor=F.+Lykoauthor=G.+Sbardella&title=Constrained+analogues+of+procaine+as+novel+small+molecule+inhibitors+of+DNA+methyltransferase-1&doi=10.1021%2Fjm7015705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Constrained Analogues of Procaine as Novel Small Molecule Inhibitors of DNA Methyltransferase-1</span></div><div class="casAuthors">Castellano, Sabrina; Kuck, Dirk; Sala, Marina; Novellino, Ettore; Lyko, Frank; Sbardella, Gianluca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2321-2325</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Constrained analogs of procaine were synthesized, and their inhibiting activity against DNMT1 was tested.  Among them, the most potent compd., deriv. 3b, was also able to induce a recognizable demethylation of chromosomal satellite repeats in HL60 human myeloid leukemia cells and thus represents a lead compd. for the development of a novel class of nonnucleoside DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsBswQtfKqrVg90H21EOLACvtfcHk0lgy6BnbKpXhww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlOjsL8%253D&md5=d922a4841968fa2c0c1efddf5102b5f4</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Fjm7015705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7015705%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DKuck%26aufirst%3DD.%26aulast%3DSala%26aufirst%3DM.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DLyko%26aufirst%3DF.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DConstrained%2520analogues%2520of%2520procaine%2520as%2520novel%2520small%2520molecule%2520inhibitors%2520of%2520DNA%2520methyltransferase-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2321%26epage%3D2325%26doi%3D10.1021%2Fjm7015705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identifying novel selective non-nucleoside DNA methyltransferase 1 inhibitors through docking-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9028</span>â <span class="NLM_lpage">9041</span>, <span class="refDoi">Â DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+Chenauthor=Y.+Wangauthor=W.+Zhouauthor=S.+Liauthor=J.+Pengauthor=Z.+Shiauthor=J.+Huauthor=Y.+C.+Liuauthor=H.+Dingauthor=Y.+Linauthor=L.+Liauthor=S.+Chengauthor=J.+Liuauthor=T.+Luauthor=H.+Jiangauthor=B.+Liuauthor=M.+Zhengauthor=C.+Luo&title=Identifying+novel+selective+non-nucleoside+DNA+methyltransferase+1+inhibitors+through+docking-based+virtual+screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0lgy6BnbKpXhww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520novel%2520selective%2520non-nucleoside%2520DNA%2520methyltransferase%25201%2520inhibitors%2520through%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ³pez-Vallejo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamborini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biochemical evaluation of Î´(2)-isoxazoline derivatives as DNA methyltransferase 1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7663</span>â <span class="NLM_lpage">7677</span>, <span class="refDoi">Â DOI: 10.1021/jm2010404</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2010404" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Orsb3K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7663-7677&author=S.+Castellanoauthor=D.+Kuckauthor=M.+Vivianoauthor=J.+Yooauthor=F.+L%C3%B3pez-Vallejoauthor=P.+Contiauthor=L.+Tamboriniauthor=A.+Pintoauthor=J.+L.+Medina-Francoauthor=G.+Sbardella&title=Synthesis+and+biochemical+evaluation+of+%CE%B4%282%29-isoxazoline+derivatives+as+DNA+methyltransferase+1+inhibitors&doi=10.1021%2Fjm2010404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biochemical Evaluation of Î2-Isoxazoline Derivatives as DNA Methyltransferase 1 Inhibitors</span></div><div class="casAuthors">Castellano, Sabrina; Kuck, Dirk; Viviano, Monica; Yoo, Jakyung; Lopez-Vallejo, Fabian; Conti, Paola; Tamborini, Lucia; Pinto, Andrea; Medina-Franco, Jose L.; Sbardella, Gianluca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7663-7677</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of Î2-isoxazoline constrained analogs of procaine/procainamide were prepd. and their inhibitory activity against DNA methyltransferase 1 (DNMT1) was tested.  Among them, compd. I is far more potent in vitro (IC50 = 150 Î¼M) than other non-nucleoside inhibitors and also exhibits a strong and dose-dependent antiproliferative effect against HCT116 human colon carcinoma cells.  The binding mode of I with the enzyme was also investigated by means of a simple competition assay as well as of docking simulations conducted using the recently published crystallog. structure of human DNMT1.  On the basis of the findings, we assessed that the mode of inhibition of I is consistent with a competition with the cofactor and propose it as a novel lead compd. for the development of non-nucleoside DNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFLFATuXaYW7Vg90H21EOLACvtfcHk0lgy6BnbKpXhww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Orsb3K&md5=6ea821ff418e46e0fe055776dfad904c</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fjm2010404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2010404%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DKuck%26aufirst%3DD.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DL%25C3%25B3pez-Vallejo%26aufirst%3DF.%26aulast%3DConti%26aufirst%3DP.%26aulast%3DTamborini%26aufirst%3DL.%26aulast%3DPinto%26aufirst%3DA.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DSynthesis%2520and%2520biochemical%2520evaluation%2520of%2520%25CE%25B4%25282%2529-isoxazoline%2520derivatives%2520as%2520DNA%2520methyltransferase%25201%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7663%26epage%3D7677%26doi%3D10.1021%2Fjm2010404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnekenburger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwergel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosconati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirsch, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miele, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferretti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">Selective non-nucleoside inhibitors of human DNA methyltransferases active in cancer including in cancer stem cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">701</span>â <span class="NLM_lpage">713</span>, <span class="refDoi">Â DOI: 10.1021/jm4012627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4012627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=701-713&author=S.+Valenteauthor=Y.+Liuauthor=M.+Schnekenburgerauthor=C.+Zwergelauthor=S.+Cosconatiauthor=C.+Grosauthor=M.+Tardugnoauthor=D.+Labellaauthor=C.+Floreanauthor=S.+Mindenauthor=H.+Hashimotoauthor=Y.+Changauthor=X.+Zhangauthor=G.+Kirschauthor=E.+Novellinoauthor=P.+B.+Arimondoauthor=E.+Mieleauthor=E.+Ferrettiauthor=A.+Gulinoauthor=M.+Diederichauthor=X.+Chengauthor=A.+Mai&title=Selective+non-nucleoside+inhibitors+of+human+DNA+methyltransferases+active+in+cancer+including+in+cancer+stem+cells&doi=10.1021%2Fjm4012627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Non-nucleoside Inhibitors of Human DNA Methyltransferases Active in Cancer Including in Cancer Stem Cells</span></div><div class="casAuthors">Valente, Sergio; Yiwei, Liu; Schnekenburger, Michael; Zwergel, Clemens; Cosconati, Sandro; Gros, Christina; Tardugno, Maria; Labella, Donatella; Florean, Cristina; Minden, Steven; Hashimoto, Hideharu; Chang, Yanqi; Zhang, Xing; Kirsch, Gilbert; Novellino, Ettore; Arimondo, Paola B.; Miele, Evelina; Ferretti, Elisabetta; Gulino, Alberto; Diederich, Marc; Cheng, Xiaodong; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">701-713</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMTs) are important enzymes involved in epigenetic control of gene expression and represent valuable targets in cancer chemotherapy.  A no. of nucleoside DNMT inhibitors (DNMTi) have been studied in cancer, including in cancer stem cells, and two of them (azacytidine and decitabine) have been approved for treatment of myelodysplastic syndromes.  However, only a few non-nucleoside DNMTi have been identified so far, and even fewer have been validated in cancer.  Through a process of hit-to-lead optimization, we report here the discovery of compd. 5 as a potent non-nucleoside DNMTi that is also selective toward other AdoMet-dependent protein methyltransferases.  Compd. 5 was potent at single-digit micromolar concns. against a panel of cancer cells and was less toxic in peripheral blood mononuclear cells than two other compds. tested.  In mouse medulloblastoma stem cells, 5 inhibited cell growth, whereas related compd. 2 showed high cell differentiation.  To the best of our knowledge, 2 and 5 are the first non-nucleoside DNMTi tested in a cancer stem cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxOcRR02xFkLVg90H21EOLACvtfcHk0liBpZExbusj7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D&md5=0749e33ad2324aa591342d7c7f23a955</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Fjm4012627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4012627%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSchnekenburger%26aufirst%3DM.%26aulast%3DZwergel%26aufirst%3DC.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DFlorean%26aufirst%3DC.%26aulast%3DMinden%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DKirsch%26aufirst%3DG.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DMiele%26aufirst%3DE.%26aulast%3DFerretti%26aufirst%3DE.%26aulast%3DGulino%26aufirst%3DA.%26aulast%3DDiederich%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DSelective%2520non-nucleoside%2520inhibitors%2520of%2520human%2520DNA%2520methyltransferases%2520active%2520in%2520cancer%2520including%2520in%2520cancer%2520stem%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D701%26epage%3D713%26doi%3D10.1021%2Fjm4012627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rilova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pechalrieu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhlel, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novosad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aussagues, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausseil, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleury, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guianvarcâh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferroud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Rational design of bisubstrate-type analogues as inhibitors of DNA methyltransferases in cancer cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4665</span>â <span class="NLM_lpage">4679</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntVGgtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4665-4679&author=L.+Halbyauthor=Y.+Menonauthor=E.+Rilovaauthor=D.+Pechalrieuauthor=V.+Massonauthor=C.+Fauxauthor=M.+A.+Bouhlelauthor=M.+H.+David-Cordonnierauthor=N.+Novosadauthor=Y.+Aussaguesauthor=A.+Samsonauthor=L.+Lacroixauthor=F.+Ausseilauthor=L.+Fleuryauthor=D.+Guianvarc%E2%80%99hauthor=C.+Ferroudauthor=P.+B.+Arimondo&title=Rational+design+of+bisubstrate-type+analogues+as+inhibitors+of+DNA+methyltransferases+in+cancer+cells&doi=10.1021%2Facs.jmedchem.7b00176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Bisubstrate-Type Analogues as Inhibitors of DNA Methyltransferases in Cancer Cells</span></div><div class="casAuthors">Halby, Ludovic; Menon, Yoann; Rilova, Elodie; Pechalrieu, Dany; Masson, Veronique; Faux, Celine; Bouhlel, Mohamed Amine; David-Cordonnier, Marie-Helene; Novosad, Natacha; Aussagues, Yannick; Samson, Arnaud; Lacroix, Laurent; Ausseil, Frederic; Fleury, Laurence; Guianvarc'h, Dominique; Ferroud, Clotilde; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4665-4679</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant DNA hypermethylation of promoter of tumor suppressor genes is commonly obsd. in cancer and its inhibition by small mols. is promising for their reactivation.  Here the authors designed bisubstrate analogs-based inhibitors, by mimicking each substrate, - the S-adenosyl-L-methionine and the deoxycytidine -, and linking them together.  This approach resulted in quinazoline-quinoline derivs. as potent inhibitors of DNMT3A and DNMT1, some showing certain isoform selectivity.  The authors highlighted the importance of (i) the nature and rigidity of the linker between the two moieties for inhibition, as (ii) the presence of the nitrogen on the quinoline group and (iii) of a hydrophobic group on the quinazoline.  The most potent inhibitors induced demethylation of CDKN2A promoter in colon carcinoma HCT116 cells and its reactivation after 7 days of treatment.  Furthermore, in a leukemia cells model system, the authors found a correlation between demethylation of the promoter induced by the treatment, chromatin opening at the promoter and the reactivation of a reporter gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpymGPl1Goj1bVg90H21EOLACvtfcHk0liBpZExbusj7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntVGgtLY%253D&md5=91cf15839ca568c0f64987ff079df5e5</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00176%26sid%3Dliteratum%253Aachs%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DMenon%26aufirst%3DY.%26aulast%3DRilova%26aufirst%3DE.%26aulast%3DPechalrieu%26aufirst%3DD.%26aulast%3DMasson%26aufirst%3DV.%26aulast%3DFaux%26aufirst%3DC.%26aulast%3DBouhlel%26aufirst%3DM.%2BA.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.%2BH.%26aulast%3DNovosad%26aufirst%3DN.%26aulast%3DAussagues%26aufirst%3DY.%26aulast%3DSamson%26aufirst%3DA.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DAusseil%26aufirst%3DF.%26aulast%3DFleury%26aufirst%3DL.%26aulast%3DGuianvarc%25E2%2580%2599h%26aufirst%3DD.%26aulast%3DFerroud%26aufirst%3DC.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DRational%2520design%2520of%2520bisubstrate-type%2520analogues%2520as%2520inhibitors%2520of%2520DNA%2520methyltransferases%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4665%26epage%3D4679%26doi%3D10.1021%2Facs.jmedchem.7b00176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span> <span> </span><span class="NLM_article-title">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1243</span>â <span class="NLM_lpage">1260</span>, <span class="refDoi">Â DOI: 10.15252/embj.201694058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.15252%2Fembj.201694058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28320739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1243-1260&author=X.+Wangauthor=W.+Caoauthor=J.+Zhangauthor=M.+Yanauthor=Q.+Xuauthor=X.+Wuauthor=L.+Wanauthor=Z.+Zhangauthor=C.+Zhangauthor=X.+Qinauthor=M.+Xiaoauthor=D.+Yeauthor=Y.+Liuauthor=Z.+Hanauthor=S.+Wangauthor=L.+Maoauthor=W.+Weiauthor=W.+Chen&title=A+covalently+bound+inhibitor+triggers+EZH2+degradation+through+CHIP-mediated+ubiquitination&doi=10.15252%2Fembj.201694058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span></div><div class="casAuthors">Wang, Xu; Cao, Wei; Zhang, Jianjun; Yan, Ming; Xu, Qin; Wu, Xiangbing; Wan, Lixin; Zhang, Zhiyuan; Zhang, Chenping; Qin, Xing; Xiao, Meng; Ye, Dongxia; Liu, Yuyang; Han, Zeguang; Wang, Shaomeng; Mao, Li; Wei, Wenyi; Chen, Wantao</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1243-1260</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Enhancer of zeste homolog 2 (EZH2) has been characterized as a crit. oncogene and a promising drug target in human malignant tumors.  The current EZH2 inhibitors strongly suppress the enhanced enzymic function of mutant EZH2 in some lymphomas.  However, the recent identification of a PRC2- and methyltransferase-independent role of EZH2 indicates that a complete suppression of all oncogenic functions of EZH2 is needed.  Here, we report a unique EZH2-targeting strategy by identifying a gambogenic acid (GNA) deriv. as a novel agent that specifically and covalently bound to Cys668 within the EZH2-SET domain, triggering EZH2 degrdn. through COOH terminus of Hsp70-interacting protein (CHIP)-mediated ubiquitination.  This class of inhibitors significantly suppressed H3K27Me3 and effectively reactivated polycomb repressor complex 2 (PRC2)-silenced tumor suppressor genes.  Moreover, the novel inhibitors significantly suppressed tumor growth in an EZH2-dependent manner, and tumors bearing a non-GNA-interacting C668S-EZH2 mutation exhibited resistance to the inhibitors.  Together, our results identify the inhibition of the signaling pathway that governs GNA-mediated destruction of EZH2 as a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRQknSdWy5T7Vg90H21EOLACvtfcHk0liBpZExbusj7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D&md5=b9b8a266537e2922648784d9cd4edf02</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.15252%2Fembj.201694058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembj.201694058%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26atitle%3DA%2520covalently%2520bound%2520inhibitor%2520triggers%2520EZH2%2520degradation%2520through%2520CHIP-mediated%2520ubiquitination%26jtitle%3DEMBO%2520J.%26date%3D2017%26volume%3D36%26spage%3D1243%26epage%3D1260%26doi%3D10.15252%2Fembj.201694058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiltz, G. E.</span></span> <span> </span><span class="NLM_article-title">Small molecule approaches for targeting the polycomb repressive complex 2 (PRC2) in cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">15344</span>â <span class="NLM_lpage">15370</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c01344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFSqu7rF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=15344-15370&author=M.+C.+Martinauthor=G.+Zengauthor=J.+Yuauthor=G.+E.+Schiltz&title=Small+molecule+approaches+for+targeting+the+polycomb+repressive+complex+2+%28PRC2%29+in+cancer&doi=10.1021%2Facs.jmedchem.0c01344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Approaches for Targeting the Polycomb Repressive Complex 2 (PRC2) in Cancer</span></div><div class="casAuthors">Martin, M. Cynthia; Zeng, Guihua; Yu, Jindan; Schiltz, Gary E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15344-15370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The polycomb repressive complex 2 (PRC2) is composed of three core subunits, enhancer of zeste 2 (EZH2), embryonic ectoderm development (EED), and suppressor of zeste 12 (SUZ12), along with a no. of accessory proteins.  It is the key enzymic protein complex that catalyzes histone H3 lysine 27 (H3K27) methylation to mediate epigenetic silencing of target genes.  PRC2 thus plays essential roles in maintaining embryonic stem cell identity and in controlling cellular differentiation.  Studies in the past decade have reported frequent overexpression or mutation of PRC2 in various cancers including prostate cancer and lymphoma.  Aberrant PRC2 function has been extensively studied and proven to contribute to a large no. of abnormal cellular processes, including those that lead to uncontrolled proliferation and tumorigenesis.  Significant efforts have recently been made to develop small mols. targeting PRC2 function for potential use as anticancer therapeutics.  In this review, we describe recent approaches to identify and develop small mols. that target PRC2.  These various strategies include the inhibition of the function of individual PRC2 core proteins, the disruption of PRC2 complex formation, and the degrdn. of its subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokmpM-lxiL4bVg90H21EOLACvtfcHk0ljwzSJFQsrTbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFSqu7rF&md5=788620c0bb1dde734c26057707d8d825</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01344%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BC.%26aulast%3DZeng%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DSchiltz%26aufirst%3DG.%2BE.%26atitle%3DSmall%2520molecule%2520approaches%2520for%2520targeting%2520the%2520polycomb%2520repressive%2520complex%25202%2520%2528PRC2%2529%2520in%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D15344%26epage%3D15370%26doi%3D10.1021%2Facs.jmedchem.0c01344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banik, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harsha, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoloi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalduhsaki
Sailo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arfuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunnumakkara, A. B.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of gambogic acid, a caged xanthone, to target cancer</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>416</i></span>,  <span class="NLM_fpage">75</span>â <span class="NLM_lpage">86</span>, <span class="refDoi">Â DOI: 10.1016/j.canlet.2017.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.canlet.2017.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29246645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVegs7jL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=416&publication_year=2018&pages=75-86&author=K.+Banikauthor=C.+Harshaauthor=D.+Bordoloiauthor=B.+Lalduhsaki%0ASailoauthor=G.+Sethiauthor=H.+C.+Leongauthor=F.+Arfusoauthor=S.+Mishraauthor=L.+Wangauthor=A.+P.+Kumarauthor=A.+B.+Kunnumakkara&title=Therapeutic+potential+of+gambogic+acid%2C+a+caged+xanthone%2C+to+target+cancer&doi=10.1016%2Fj.canlet.2017.12.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of gambogic acid, a caged xanthone, to target cancer</span></div><div class="casAuthors">Banik, Kishore; Harsha, Choudhary; Bordoloi, Devivasha; Lalduhsaki Sailo, Bethsebie; Sethi, Gautam; Leong, Hin Chong; Arfuso, Frank; Mishra, Srishti; Wang, Lingzhi; Kumar, Alan P.; Kunnumakkara, Ajaikumar B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">416</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-86</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Natural compds. have enormous biol. and clin. activity against dreadful diseases such as cancer, as well as cardiovascular and neurodegenerative disorders.  In spite of the widespread research carried out in the field of cancer therapeutics, cancer is one of the most prevalent diseases with no perfect treatment till date.  Adverse side effects and the development of chemoresistance are the imperative limiting factors assocd. with conventional chemotherapeutics.  For this reason, there is an urgent need to find compds. that are highly safe and efficacious for the prevention and treatment of cancer.  Gambogic acid (GA) is a xanthone structure extd. from the dry, brownish gamboge resin secreted from the Garcinia hanburyi tree in Southeast Asia and has inherent anti-cancer properties.  In this review, the mol. mechanisms underlying the targets of GA that are liable for its effective anti-cancer activity are discussed that reveal the potential of GA as a pertinent candidate that can be appropriately developed and designed into a capable anti-cancer drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpQe0ZqukkY7Vg90H21EOLACvtfcHk0ljwzSJFQsrTbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVegs7jL&md5=45a6d09c16c77e19d487d33bc006d734</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.12.014%26sid%3Dliteratum%253Aachs%26aulast%3DBanik%26aufirst%3DK.%26aulast%3DHarsha%26aufirst%3DC.%26aulast%3DBordoloi%26aufirst%3DD.%26aulast%3DLalduhsaki%2BSailo%26aufirst%3DB.%26aulast%3DSethi%26aufirst%3DG.%26aulast%3DLeong%26aufirst%3DH.%2BC.%26aulast%3DArfuso%26aufirst%3DF.%26aulast%3DMishra%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DA.%2BP.%26aulast%3DKunnumakkara%26aufirst%3DA.%2BB.%26atitle%3DTherapeutic%2520potential%2520of%2520gambogic%2520acid%252C%2520a%2520caged%2520xanthone%252C%2520to%2520target%2520cancer%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D416%26spage%3D75%26epage%3D86%26doi%3D10.1016%2Fj.canlet.2017.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cipriano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic reader domains by chemical biology</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">82</span>â <span class="NLM_lpage">94</span>, <span class="refDoi">Â DOI: 10.1016/j.cbpa.2020.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.cbpa.2020.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32739717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2020&pages=82-94&author=A.+Ciprianoauthor=G.+Sbardellaauthor=A.+Ciulli&title=Targeting+epigenetic+reader+domains+by+chemical+biology&doi=10.1016%2Fj.cbpa.2020.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic reader domains by chemical biology</span></div><div class="casAuthors">Cipriano, Alessandra; Sbardella, Gianluca; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-94</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Over the past years, growing interest toward post-translational modifications (PTMs) of histones and nonhistone proteins has prompted academia and industrial research groups to develop different approaches to better understand the link between PTMs and pathol. states.  Selective recognition of PTMs is carried out by reader modules, which mediate the biol. readout of epigenetic mechanisms.  Progress in medicinal chem. and chem. biol. has contributed to corroborate the role of reader domains in chromatin-binding proteins as potential therapeutic targets.  Here, we review the state-of-the-art of the most important small mols. developed to date, with a particular attention on contemporary chem. biol. approaches, including photoaffinity probes, cyclic peptides, bifunctional inhibitors, and PROTAC degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOYlHbr36l8rVg90H21EOLACvtfcHk0ljwzSJFQsrTbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbbP&md5=5a804beab676445c967726108a33b9aa</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DCipriano%26aufirst%3DA.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DTargeting%2520epigenetic%2520reader%2520domains%2520by%2520chemical%2520biology%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94%26doi%3D10.1016%2Fj.cbpa.2020.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spriano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">107631</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2020.107631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.pharmthera.2020.107631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32693114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2020&pages=107631&author=F.+Sprianoauthor=A.+Stathisauthor=F.+Bertoni&title=Targeting+BET+bromodomain+proteins+in+cancer%3A+The+example+of+lymphomas&doi=10.1016%2Fj.pharmthera.2020.107631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span></div><div class="casAuthors">Spriano, Filippo; Stathis, Anastasios; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107631</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Bromo- and Extra-Terminal domain (BET) family proteins act as ''readers'' of acetylated histones and they are important transcription regulators.  BRD2, BRD3, BRD4 and BRDT, part of the BET family, are important in different tumors, where upregulation or translocation often occurs.  The potential of targeting BET proteins as anti-cancer treatment originated with data obtained with a first series of compds., and there are now several data supporting BET inhibition in both solid tumors and hematol. malignancies.  Despite very pos. preclin. data in different tumor types, the clin. results have been so far moderate.  Using lymphoma as an example to review the data produced in the lab. and in the context of the early clin. trials, we discuss the modalities to make BET targeting more efficient both generating novel generation of compds. and by exploring the combination with small mols. affecting various signaling pathways, BCL2, or DNA damage response signaling, but also with addnl. epigenetic agents and with immunotherapy.  We also discuss the mechanisms of resistance and the toxicity profiles so far reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Jn61Q24_wrVg90H21EOLACvtfcHk0lhzASUHdHPKhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI&md5=cdb034e619849cb681f48c5b03f7cde8</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107631%26sid%3Dliteratum%253Aachs%26aulast%3DSpriano%26aufirst%3DF.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DTargeting%2520BET%2520bromodomain%2520proteins%2520in%2520cancer%253A%2520The%2520example%2520of%2520lymphomas%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D215%26spage%3D107631%26doi%3D10.1016%2Fj.pharmthera.2020.107631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: Epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>â <span class="NLM_lpage">356</span>, <span class="refDoi">Â DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+Epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhzASUHdHPKhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520Epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>â <span class="NLM_lpage">4558</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lhzASUHdHPKhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CÃ´tÃ©, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: Progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>â <span class="NLM_lpage">1298</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+Progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lgkWDnEbpEYkA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520Progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooike, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugahara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daibata, M.</span></span> <span> </span><span class="NLM_article-title">Abstract C244: Development of the BET bromodomain inhibitor OTX015</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">C244</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.TARG-13-C244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1535-7163.TARG-13-C244" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=C244&author=J.+K.+Noelauthor=K.+Iwataauthor=S.+Ooikeauthor=K.+Sugaharaauthor=H.+Nakamuraauthor=M.+Daibata&title=Abstract+C244%3A+Development+of+the+BET+bromodomain+inhibitor+OTX015&doi=10.1158%2F1535-7163.TARG-13-C244"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-13-C244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-13-C244%26sid%3Dliteratum%253Aachs%26aulast%3DNoel%26aufirst%3DJ.%2BK.%26aulast%3DIwata%26aufirst%3DK.%26aulast%3DOoike%26aufirst%3DS.%26aulast%3DSugahara%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DH.%26aulast%3DDaibata%26aufirst%3DM.%26atitle%3DAbstract%2520C244%253A%2520Development%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520OTX015%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26volume%3D12%26spage%3DC244%26doi%3D10.1158%2F1535-7163.TARG-13-C244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorusso, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagey, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, S. B.</span></span> <span> </span><span class="NLM_article-title">Abstract A49: Clinically efficacy of the BET bromodomain inhibitor TEN-010 in an open-label substudy with patients with documented NUT-midline carcinoma (NMC)</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">A49</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.TARG-15-A49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1535-7163.TARG-15-A49" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=A49&author=G.+I.+Shapiroauthor=A.+Dowlatiauthor=P.+M.+Lorussoauthor=J.+P.+Ederauthor=A.+Andersonauthor=K.+T.+Doauthor=M.+H.+Kageyauthor=C.+Sirardauthor=J.+E.+Bradnerauthor=S.+B.+Landau&title=Abstract+A49%3A+Clinically+efficacy+of+the+BET+bromodomain+inhibitor+TEN-010+in+an+open-label+substudy+with+patients+with+documented+NUT-midline+carcinoma+%28NMC%29&doi=10.1158%2F1535-7163.TARG-15-A49"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-A49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-A49%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DLorusso%26aufirst%3DP.%2BM.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DAnderson%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DK.%2BT.%26aulast%3DKagey%26aufirst%3DM.%2BH.%26aulast%3DSirard%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLandau%26aufirst%3DS.%2BB.%26atitle%3DAbstract%2520A49%253A%2520Clinically%2520efficacy%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520TEN-010%2520in%2520an%2520open-label%2520substudy%2520with%2520patients%2520with%2520documented%2520NUT-midline%2520carcinoma%2520%2528NMC%2529%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DA49%26doi%3D10.1158%2F1535-7163.TARG-15-A49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>â <span class="NLM_lpage">1123</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0ljKFmv1glrAfg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a benzoisoxazoloazepine inhibitor (CPI-0610) of the bromodomain and extra- terminal (BET) family as a candidate for human clinical trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>â <span class="NLM_lpage">1339</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+Coteauthor=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.+C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+benzoisoxazoloazepine+inhibitor+%28CPI-0610%29+of+the+bromodomain+and+extra-+terminal+%28BET%29+family+as+a+candidate+for+human+clinical+trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-ÎºB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0lhV40hh1bqriA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520benzoisoxazoloazepine%2520inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520bromodomain%2520and%2520extra-%2520terminal%2520%2528BET%2529%2520family%2520as%2520a%2520candidate%2520for%2520human%2520clinical%2520trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-(4-(2,4-nifluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a potent and orally available bromodomain and extraterminal domain (BET) family bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>â <span class="NLM_lpage">8384</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-nifluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfonamide+%28ABBV-075%2FMivebresib%29%2C+a+potent+and+orally+available+bromodomain+and+extraterminal+domain+%28BET%29+family+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lhV40hh1bqriA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-nifluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfonamide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520potent%2520and%2520orally%2520available%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">VÃ¡zquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civenni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zadic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbone, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catapano, C. V.</span></span> <span> </span><span class="NLM_article-title">Abstract 5080: BET inhibitors INCB054329 and INCB057643 display significant activity in androgen-independent prostate cancer models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">5080</span>, <span class="refDoi">Â DOI: 10.1158/1538-7445.AM2017-5080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1538-7445.AM2017-5080" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=5080&author=R.+V%C3%A1zquezauthor=G.+Civenniauthor=M.+Marchettiauthor=S.+Zadicauthor=P.+Liuauthor=B.+Ruggeriauthor=G.+M.+Carboneauthor=C.+V.+Catapano&title=Abstract+5080%3A+BET+inhibitors+INCB054329+and+INCB057643+display+significant+activity+in+androgen-independent+prostate+cancer+models&doi=10.1158%2F1538-7445.AM2017-5080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-5080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-5080%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez%26aufirst%3DR.%26aulast%3DCivenni%26aufirst%3DG.%26aulast%3DMarchetti%26aufirst%3DM.%26aulast%3DZadic%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DRuggeri%26aufirst%3DB.%26aulast%3DCarbone%26aufirst%3DG.%2BM.%26aulast%3DCatapano%26aufirst%3DC.%2BV.%26atitle%3DAbstract%25205080%253A%2520BET%2520inhibitors%2520INCB054329%2520and%2520INCB057643%2520display%2520significant%2520activity%2520in%2520androgen-independent%2520prostate%2520cancer%2520models%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D5080%26doi%3D10.1158%2F1538-7445.AM2017-5080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavai, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortolani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMalley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnelle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, V.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Abstract 5789: Discovery of clinical candidate BMS-986158, an oral BET inhibitor, for the treatment of cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">5789</span>, <span class="refDoi">Â DOI: 10.1158/1538-7445.AM2018-5789</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1538-7445.AM2018-5789" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=5789&author=A.+V.+Gavaiauthor=D.+Norrisauthor=D.+Tortolaniauthor=D.+O%E2%80%99Malleyauthor=Y.+Zhaoauthor=C.+Quesnelleauthor=P.+Gillauthor=W.+Vaccaroauthor=T.+Huynhauthor=V.+Ahuja&title=Abstract+5789%3A+Discovery+of+clinical+candidate+BMS-986158%2C+an+oral+BET+inhibitor%2C+for+the+treatment+of+cancer&doi=10.1158%2F1538-7445.AM2018-5789"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2018-5789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2018-5789%26sid%3Dliteratum%253Aachs%26aulast%3DGavai%26aufirst%3DA.%2BV.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DTortolani%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DQuesnelle%26aufirst%3DC.%26aulast%3DGill%26aufirst%3DP.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DAhuja%26aufirst%3DV.%26atitle%3DAbstract%25205789%253A%2520Discovery%2520of%2520clinical%2520candidate%2520BMS-986158%252C%2520an%2520oral%2520BET%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3D5789%26doi%3D10.1158%2F1538-7445.AM2018-5789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>â <span class="NLM_lpage">19759</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0li_doFs00raUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Gamal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hing, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blachly, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieselhuber, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goettl, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spevak, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carias, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantoy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanftner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nespi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MathÃ©, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span> <span> </span><span class="NLM_article-title">BRD4 profiling identifies critical chronic lymphocytic leukemia oncogenic circuits and reveals sensitivity to PLX51107, a novel structurally distinct BET inhibitor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">458</span>â <span class="NLM_lpage">477</span>, <span class="refDoi">Â DOI: 10.1158/2159-8290.CD-17-0902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F2159-8290.CD-17-0902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29386193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=458-477&author=H.+G.+Ozerauthor=D.+El-Gamalauthor=B.+Powellauthor=Z.+A.+Hingauthor=J.+S.+Blachlyauthor=B.+Harringtonauthor=S.+Mitchellauthor=N.+R.+Grieselhuberauthor=K.+Williamsauthor=T.+H.+Laiauthor=L.+Alinariauthor=R.+A.+Baiocchiauthor=L.+Brintonauthor=E.+Baskinauthor=M.+Cannonauthor=L.+Beaverauthor=V.+M.+Goettlauthor=D.+M.+Lucasauthor=J.+A.+Woyachauthor=D.+Sampathauthor=A.+M.+Lehmanauthor=L.+Yuauthor=J.+Zhangauthor=Y.+Maauthor=Y.+Zhangauthor=W.+Spevakauthor=S.+Shiauthor=P.+Seversonauthor=R.+Shellooeauthor=H.+Cariasauthor=G.+Tsangauthor=K.+Dongauthor=T.+Ewingauthor=A.+Marimuthuauthor=C.+Tantoyauthor=J.+Waltersauthor=L.+Sanftnerauthor=H.+Rezaeiauthor=M.+Nespiauthor=B.+Matusowauthor=G.+Habetsauthor=P.+Ibrahimauthor=C.+Zhangauthor=E.+A.+Math%C3%A9author=G.+Bollagauthor=J.+C.+Byrdauthor=R.+Lapalombella&title=BRD4+profiling+identifies+critical+chronic+lymphocytic+leukemia+oncogenic+circuits+and+reveals+sensitivity+to+PLX51107%2C+a+novel+structurally+distinct+BET+inhibitor&doi=10.1158%2F2159-8290.CD-17-0902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span></div><div class="casAuthors">Ozer, Hatice Gulcin; El-Gamal, Dalia; Powell, Ben; Hing, Zachary A.; Blachly, James S.; Harrington, Bonnie; Mitchell, Shaneice; Grieselhuber, Nicole R.; Williams, Katie; Lai, Tzung-Huei; Alinari, Lapo; Baiocchi, Robert A.; Brinton, Lindsey; Baskin, Elizabeth; Cannon, Matthew; Beaver, Larry; Goettl, Virginia M.; Lucas, David M.; Woyach, Jennifer A.; Sampath, Deepa; Lehman, Amy M.; Yu, Lianbo; Zhang, Jiazhong; Ma, Yan; Zhang, Ying; Spevak, Wayne; Shi, Songyuan; Severson, Paul; Shellooe, Rafe; Carias, Heidi; Tsang, Garson; Dong, Ken; Ewing, Todd; Marimuthu, Adhirai; Tantoy, Christina; Walters, Jason; Sanftner, Laura; Rezaei, Hamid; Nespi, Marika; Matusow, Bernice; Habets, Gaston; Ibrahim, Prabha; Zhang, Chao; Mathe, Ewy A.; Bollag, Gideon; Byrd, John C.; Lapalombella, Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-477</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins are key regulators of gene expression in cancer.  Herein, we utilize BRD4 profiling to identify crit. pathways involved in pathogenesis of chronic lymphocytic leukemia (CLL).  BRD4 is overexpressed in CLL and is enriched proximal to genes upregulated or de novo expressed in CLL with known functions in disease pathogenesis and progression.  These genes, including key members of the B-cell receptor (BCR) signaling pathway, provide a rationale for this therapeutic approach to identify new targets in alternative types of cancer.  Addnl., we describe PLX51107, a structurally distinct BET inhibitor with novel in vitro and in vivo pharmacol. properties that emulates or exceeds the efficacy of BCR signaling agents in preclin. models of CLL.  Herein, the discovery of the involvement of BRD4 in the core CLL transcriptional program provides a compelling rationale for clin. investigation of PLX51107 as epigenetic therapy in CLL and application of BRD4 profiling in other cancers.  Significance: To date, functional studies of BRD4 in CLL are lacking.  Through integrated genomic, functional, and pharmacol. analyses, we uncover the existence of BRD4-regulated core CLL transcriptional programs and present preclin. proof-of-concept studies validating BET inhibition as an epigenetic approach to target BCR signaling in CLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvzZtZKkb5rVg90H21EOLACvtfcHk0li_doFs00raUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D&md5=3715a65bb8cee2a5fe53994a3d106303</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-0902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-0902%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DH.%2BG.%26aulast%3DEl-Gamal%26aufirst%3DD.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHing%26aufirst%3DZ.%2BA.%26aulast%3DBlachly%26aufirst%3DJ.%2BS.%26aulast%3DHarrington%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DS.%26aulast%3DGrieselhuber%26aufirst%3DN.%2BR.%26aulast%3DWilliams%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DT.%2BH.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DBrinton%26aufirst%3DL.%26aulast%3DBaskin%26aufirst%3DE.%26aulast%3DCannon%26aufirst%3DM.%26aulast%3DBeaver%26aufirst%3DL.%26aulast%3DGoettl%26aufirst%3DV.%2BM.%26aulast%3DLucas%26aufirst%3DD.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DK.%26aulast%3DEwing%26aufirst%3DT.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DTantoy%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DJ.%26aulast%3DSanftner%26aufirst%3DL.%26aulast%3DRezaei%26aufirst%3DH.%26aulast%3DNespi%26aufirst%3DM.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMath%25C3%25A9%26aufirst%3DE.%2BA.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DLapalombella%26aufirst%3DR.%26atitle%3DBRD4%2520profiling%2520identifies%2520critical%2520chronic%2520lymphocytic%2520leukemia%2520oncogenic%2520circuits%2520and%2520reveals%2520sensitivity%2520to%2520PLX51107%252C%2520a%2520novel%2520structurally%2520distinct%2520BET%2520inhibitor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D458%26epage%3D477%26doi%3D10.1158%2F2159-8290.CD-17-0902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak bromo and extra terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>â <span class="NLM_lpage">9126</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+B.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+bromo+and+extra+terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0lhOHDLhPOXMrw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCopley%26aufirst%3DR.%2BC.%2BB.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the bromodomain and extra terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>â <span class="NLM_lpage">9092</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+bromodomain+and+extra+terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0lhOHDLhPOXMrw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-2-carboxamide (ABBV-744), a BET bromodomain inhibitor with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>â <span class="NLM_lpage">5623</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+J.+Kovarauthor=M.+Buiauthor=E.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+bromodomain+inhibitor+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0ljmAlXdlcAipw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520bromodomain%2520inhibitor%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>â <span class="NLM_lpage">4334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0ljmAlXdlcAipw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">WyspiaÅska, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a bromodomain and extraterminal domain (BET) inhibitor selective for the N-terminal bromodomains that retains anti-inflammatory and antiproliferative phenotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9020</span>â <span class="NLM_lpage">9044</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9020-9044&author=C.+R.+Wellawayauthor=P.+Bamboroughauthor=S.+G.+Bernardauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Cutlerauthor=E.+H.+Demontauthor=J.+P.+Evansauthor=L.+Gordonauthor=B.+Karamshiauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+J.+Mitchellauthor=I.+Riojaauthor=P.+E.+Sodenauthor=S.+Taylorauthor=R.+J.+Watsonauthor=R.+Willisauthor=J.+M.+Woolvenauthor=B.+S.+Wyspia%C5%84skaauthor=W.+J.+Kerrauthor=R.+K.+Prinjha&title=Structure-based+design+of+a+bromodomain+and+extraterminal+domain+%28BET%29+inhibitor+selective+for+the+N-terminal+bromodomains+that+retains+anti-inflammatory+and+antiproliferative+phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype</span></div><div class="casAuthors">Wellaway, Christopher R.; Bamborough, Paul; Bernard, Sharon G.; Chung, Chun-wa; Craggs, Peter D.; Cutler, Leanne; Demont, Emmanuel H.; Evans, John P.; Gordon, Laurie; Karamshi, Bhumika; Lewis, Antonia J.; Lindon, Matthew J.; Mitchell, Darren J.; Rioja, Inmaculada; Soden, Peter E.; Taylor, Simon; Watson, Robert J.; Willis, Rob; Woolven, James M.; Wyspianska, Beata S.; Kerr, William J.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9020-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) family of epigenetic regulators comprises four proteins (BRD2, BRD3, BRD4, BRDT), each contg. tandem bromodomains.  To date, small mol. inhibitors of these proteins typically bind all eight bromodomains of the family with similar affinity, resulting in a diverse range of biol. effects.  To enable further understanding of the broad phenotype characteristic of pan-BET inhibition, the development of inhibitors selective for individual, or sets of, bromodomains within the family is required.  In this regard, we report the discovery of a potent probe mol. possessing up to 150-fold selectivity for the N-terminal bromodomains (BD1s) over the C-terminal bromodomains (BD2s) of the BETs.  Guided by structural information, a specific amino acid difference between BD1 and BD2 domains was targeted for selective interaction with chem. functionality appended to the previously developed I-BET151 scaffold.  Data presented herein demonstrate that selective inhibition of BD1 domains is sufficient to drive anti-inflammatory and antiproliferative effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUREcHUhFfbVg90H21EOLACvtfcHk0ljF62BxbPpiLQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M&md5=52b2505bb0a36f526a08bb9bad1f2c26</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BG.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWillis%26aufirst%3DR.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DWyspia%25C5%2584ska%26aufirst%3DB.%2BS.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520inhibitor%2520selective%2520for%2520the%2520N-terminal%2520bromodomains%2520that%2520retains%2520anti-inflammatory%2520and%2520antiproliferative%2520phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9020%26epage%3D9044%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK789: A selective inhibitor of the first bromodomains (BD1) of the bromo and extra terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>â <span class="NLM_lpage">9069</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.+W.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=GSK789%3A+A+selective+inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+bromo+and+extra+terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0ljF62BxbPpiLQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK789%253A%2520A%2520selective%2520inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of orally bioavailable chromone derivatives as potent and selective BRD4 inhibitors: Scaffold hopping, optimization, and pharmacological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5242</span>â <span class="NLM_lpage">5256</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5242-5256&author=Z.+Liuauthor=H.+Chenauthor=P.+Wangauthor=Y.+Liauthor=E.+A.+Woldauthor=P.+G.+Leonardauthor=S.+Josephauthor=A.+R.+Brasierauthor=B.+Tianauthor=J.+Zhou&title=Discovery+of+orally+bioavailable+chromone+derivatives+as+potent+and+selective+BRD4+inhibitors%3A+Scaffold+hopping%2C+optimization%2C+and+pharmacological+evaluation&doi=10.1021%2Facs.jmedchem.0c00035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Bioavailable Chromone Derivatives as Potent and Selective BRD4 Inhibitors: Scaffold Hopping, Optimization, and Pharmacological Evaluation</span></div><div class="casAuthors">Liu Zhiqing; Chen Haiying; Wang Pingyuan; Li Yi; Wold Eric A; Zhou Jia; Leonard Paul G; Joseph Sarah; Brasier Allan R; Tian Bing; Tian Bing; Zhou Jia; Zhou Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5242-5256</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 4 (BRD4) represents a promising drug target for anti-inflammatory therapeutics.  Herein, we report the design, synthesis, and pharmacological evaluation of novel chromone derivatives via scaffold hopping to discover a new class of orally bioavailable BRD4-selective inhibitors.  Two potent BRD4 bromodomain 1 (BD1)-selective inhibitors 44 (ZL0513) and 45 (ZL0516) have been discovered with high binding affinity (IC50 values of 67-84 nM) and good selectivity over other BRD family proteins and distant BD-containing proteins.  Both compounds significantly inhibited the expression of Toll-like receptor-induced inflammatory genes in vitro and airway inflammation in murine models.  The cocrystal structure of 45 in complex with human BRD4 BD1 at a high resolution of 2.0 ÃA has been solved, offering a solid structural basis for its binding validation and further structure-based optimization.  These BRD4 BD1 inhibitors demonstrated impressive in vivo efficacy and overall promising pharmacokinetic properties, indicating their therapeutic potential for the treatment of inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR1QLaeKXMGxJ0d-O_J5C0AfW6udTcc2eZf-ex4IOPqC7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D&md5=d65e4298cbfff545c6c333713e4db210</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00035%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520orally%2520bioavailable%2520chromone%2520derivatives%2520as%2520potent%2520and%2520selective%2520BRD4%2520inhibitors%253A%2520Scaffold%2520hopping%252C%2520optimization%252C%2520and%2520pharmacological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5242%26epage%3D5256%26doi%3D10.1021%2Facs.jmedchem.0c00035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of 8-methyl-pyrrolo[1,2-<i>a</i>]pyrazin-1(2<i>H</i>)-one derivatives as highly potent and selective bromodomain and extra-terminal (BET) bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3956</span>â <span class="NLM_lpage">3975</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b01784</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01784" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXls1Cjtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3956-3975&author=Z.+Liauthor=S.+Xiaoauthor=Y.+Yangauthor=C.+Chenauthor=T.+Luauthor=Z.+Chenauthor=H.+Jiangauthor=S.+Chenauthor=C.+Luoauthor=B.+Zhou&title=Discovery+of+8-methyl-pyrrolo%5B1%2C2-a%5Dpyrazin-1%282H%29-one+derivatives+as+highly+potent+and+selective+bromodomain+and+extra-terminal+%28BET%29+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.9b01784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 8-Methyl-pyrrolo[1,2-a]pyrazin-1(2H)-one Derivatives as Highly Potent and Selective Bromodomain and Extra-Terminal (BET) Bromodomain Inhibitors</span></div><div class="casAuthors">Li, Zizhou; Xiao, Senhao; Yang, Yaxi; Chen, Chao; Lu, Tian; Chen, Zhifeng; Jiang, Hualiang; Chen, Shijie; Luo, Cheng; Zhou, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3956-3975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins have recently emerged as promising drug targets for cancer therapy.  In this study, identification of an 8-methyl-pyrrolo[1,2-a]pyrazin-1(2H)-one fragment (47) as a new binder to the BET bromodomains and the subsequent incorporation of fragment 47 to the scaffold of ABBV-075, which recently entered Phase I clin. trials, enabled the generation of a series of highly potent BET bromodomain inhibitors.  Further druggability optimization led to the discovery of compd. 38 as a potential preclin. candidate.  Significantly, compared with ABBV-075, which exhibits a 63-fold selectivity for BRD4(1) over EP300, compd. 38 demonstrates an excellent selectivity for the BET bromodomain family over other bromodomains, with an ~ 1500-fold selectivity for BRD4(1) over EP300.  Orally administered 38 achieves a complete inhibition of tumor growth with a tumor growth inhibition (TGI) of 99.7% accompanied by good tolerability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBnoe3zL-x2LVg90H21EOLACvtfcHk0lgLg6LFxzFSPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXls1Cjtb0%253D&md5=34cf8275969bd7a1ff214b0eb1e74ca9</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01784%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DB.%26atitle%3DDiscovery%2520of%25208-methyl-pyrrolo%255B1%252C2-a%255Dpyrazin-1%25282H%2529-one%2520derivatives%2520as%2520highly%2520potent%2520and%2520selective%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3956%26epage%3D3975%26doi%3D10.1021%2Facs.jmedchem.9b01784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[cd]indol-2(1<i>H</i>)-ones and pyrrolo[4,3,2-<i>de</i>]quinolin-2(1<i>H</i>)-ones as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the first bromodomain with potential high efficiency against acute gouty arthritis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11080</span>â <span class="NLM_lpage">11107</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11080-11107&author=F.+Jiangauthor=Q.+Huauthor=Z.+Zhangauthor=H.+Liauthor=H.+Liauthor=D.+Zhangauthor=H.+Liauthor=Y.+Maauthor=J.+Xuauthor=H.+Chenauthor=Y.+Cuiauthor=Y.+Zhiauthor=Y.+Zhangauthor=J.+Xuauthor=J.+Zhuauthor=T.+Luauthor=Y.+Chen&title=Discovery+of+benzo%5Bcd%5Dindol-2%281H%29-ones+and+pyrrolo%5B4%2C3%2C2-de%5Dquinolin-2%281H%29-ones+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+first+bromodomain+with+potential+high+efficiency+against+acute+gouty+arthritis&doi=10.1021%2Facs.jmedchem.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></div><div class="casAuthors">Jiang, Fei; Hu, Qinghua; Zhang, Zhimin; Li, Hongmei; Li, Huili; Zhang, Dewei; Li, Hanwen; Ma, Yu; Xu, Jingjing; Chen, Haifang; Cui, Yong; Zhi, Yanle; Zhang, Yanmin; Xu, Junyu; Zhu, Jiapeng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11080-11107</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues.  Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2).  Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family.  Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1.  Through structure-based optimization, highly active and selective compds. are ultimately obtained.  The representative compds. are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2).  Among them, we further show that 68 (LT052) mediates BRD4/NF-ÎºB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.  Therefore, selective pharmacol. modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav2e3P7XQ1LVg90H21EOLACvtfcHk0lgLg6LFxzFSPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E&md5=5cce669e137812d20f8ba2a9785f4666</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520and%2520pyrrolo%255B4%252C3%252C2-de%255Dquinolin-2%25281H%2529-ones%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520first%2520bromodomain%2520with%2520potential%2520high%2520efficiency%2520against%2520acute%2520gouty%2520arthritis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11080%26epage%3D11107%26doi%3D10.1021%2Facs.jmedchem.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Horst, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of novel covalent bromodomain and extra-terminal domain (BET) inhibitors targeting a methionine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8202</span>â <span class="NLM_lpage">8211</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8202-8211&author=O.+A.+Kharenkoauthor=R.+G.+Patelauthor=S.+D.+Brownauthor=C.+Calosingauthor=A.+Whiteauthor=D.+Lakshminarasimhanauthor=R.+K.+Sutoauthor=B.+C.+Duffyauthor=D.+B.+Kitchenauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=E.+H.+van+der%0AHorstauthor=P.+R.+Young&title=Design+and+characterization+of+novel+covalent+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+targeting+a+methionine&doi=10.1021%2Facs.jmedchem.8b00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Novel Covalent Bromodomain and Extra-Terminal Domain (BET) Inhibitors Targeting a Methionine</span></div><div class="casAuthors">Kharenko, Olesya A.; Patel, Reena G.; Brown, S. David; Calosing, Cyrus; White, Andre; Lakshminarasimhan, Damodharan; Suto, Robert K.; Duffy, Bryan C.; Kitchen, Douglas B.; McLure, Kevin G.; Hansen, Henrik C.; van der Horst, Edward H.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8202-8211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET proteins are key epigenetic regulators that regulate transcription through binding to acetylated lysine (AcLys) residues of histones and transcription factors through bromodomains (BDs).  The disruption of this interaction with small mol. bromodomain inhibitors is a promising approach to treat various diseases including cancer, autoimmune and cardiovascular diseases.  Covalent inhibitors can potentially offer a more durable target inhibition leading to improved in vivo pharmacol.  Here the authors describe the design of covalent inhibitors of BRD4(BD1) that target a methionine in the binding pocket by attaching an epoxide warhead to a suitably oriented noncovalent inhibitor.  Using thermal denaturation, MALDI-TOF mass spectrometry, and an x-ray crystal structure, the authors demonstrate that these inhibitors selectively form a covalent bond with Met 149 in BRD4(BD1) but not other bromodomains and provide durable transcriptional and antiproliferative activity in cell based assays.  Covalent targeting of methionine offers a novel approach to drug discovery for BET proteins and other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHccGRxf5ShrVg90H21EOLACvtfcHk0lhBjdrX8pP_Wg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO&md5=1c3ae393a5f8a8c12ffb16e6f3798118</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00666%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520novel%2520covalent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520targeting%2520a%2520methionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8202%26epage%3D8211%26doi%3D10.1021%2Facs.jmedchem.8b00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alferiev, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinson, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breuer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. J.</span></span> <span> </span><span class="NLM_article-title">High reactivity of alkyl sulfides towards epoxides under conditions of collagen fixation--a convenient approach to 2-amino-4-butyrolactones</span>. <i>Biomaterials</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2501</span>â <span class="NLM_lpage">2506</span>, <span class="refDoi">Â DOI: 10.1016/S0142-9612(00)00440-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2FS0142-9612%2800%2900440-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=11516082" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlKksLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=2501-2506&author=I.+S.+Alferievauthor=J.+T.+Hinsonauthor=M.+Ogleauthor=E.+Breuerauthor=R.+J.+Levy&title=High+reactivity+of+alkyl+sulfides+towards+epoxides+under+conditions+of+collagen+fixation%2D%2Da+convenient+approach+to+2-amino-4-butyrolactones&doi=10.1016%2FS0142-9612%2800%2900440-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">High reactivity of alkyl sulfides towards epoxides under conditions of collagen fixation - a convenient approach to 2-amino-4-butyrolactones</span></div><div class="casAuthors">Alferiev, I. S.; Hinson, J. T.; Ogle, M.; Breuer, E.; Levy, R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Biomaterials</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2501-2506</span>CODEN:
                <span class="NLM_cas:coden">BIMADU</span>;
        ISSN:<span class="NLM_cas:issn">0142-9612</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Epoxy crosslinking agents have been investigated for use in the fabrication of bioprosthetic devices, such as heterograft heart valve prostheses.  It has been generally assumed that epoxy crosslinking takes place via amino-epoxy reactions.  The present study investigated the hypothesis that the reactions of methionine residues with epoxides also can occur in biomaterial crosslinking.  A series of model reactions were studied in which a mono-epoxide was combined with individual alkyl sulfides.  In the present studies epoxides rapidly alkylate aliph. sulfides, including methionine derivs., in buffered aq. solns. at room temp. and pH close to neutral, forming sulfonium compds., which are stable at pH 5-7 at temps. up to 50Â°C, except for cases in which methionine derivs. with non-protected carboxy groups are used.  The rate of reaction remains practically unchanged within the range of pH from 5 to 12, whereas in strongly alk. media the reverse reaction occurs.  This discovery can provide a better understanding of processes occurring in the fixation of bioprosthetic tissues with polyepoxides.  It can also develop into a site-specific method to label methionine residues in proteins.  The carboxy group-contg. sulfonium betaines derived from N-protected methionines undergo cyclization in unexpectedly mild conditions, which can be used as an efficient method for prepn. of N-protected 2-amino-4-butyrolactones with sensitive protective groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnU7-m8ArXsLVg90H21EOLACvtfcHk0lhBjdrX8pP_Wg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlKksLo%253D&md5=e2226438d8f27afa4f6cf2cb5d1b4b91</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1016%2FS0142-9612%2800%2900440-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0142-9612%252800%252900440-3%26sid%3Dliteratum%253Aachs%26aulast%3DAlferiev%26aufirst%3DI.%2BS.%26aulast%3DHinson%26aufirst%3DJ.%2BT.%26aulast%3DOgle%26aufirst%3DM.%26aulast%3DBreuer%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BJ.%26atitle%3DHigh%2520reactivity%2520of%2520alkyl%2520sulfides%2520towards%2520epoxides%2520under%2520conditions%2520of%2520collagen%2520fixation--a%2520convenient%2520approach%2520to%25202-amino-4-butyrolactones%26jtitle%3DBiomaterials%26date%3D2001%26volume%3D22%26spage%3D2501%26epage%3D2506%26doi%3D10.1016%2FS0142-9612%2800%2900440-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanus, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantuck, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakhotia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abida, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alumkal, J. J.</span></span> <span> </span><span class="NLM_article-title">A phase Ib/IIa study of the pan-BET inhibitor ZEN-3694 in combination with enzalutamide in patients with metastatic castration-resistant prostate cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5338</span>â <span class="NLM_lpage">5347</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-20-1707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1158%2F1078-0432.CCR-20-1707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32694156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFCku7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=5338-5347&author=R.+R.+Aggarwalauthor=M.+T.+Schweizerauthor=D.+M.+Nanusauthor=A.+J.+Pantuckauthor=E.+I.+Heathauthor=E.+Campeauauthor=S.+Attwellauthor=K.+Norekauthor=M.+Snyderauthor=L.+Baumanauthor=S.+Lakhotiaauthor=F.+Y.+Fengauthor=E.+J.+Smallauthor=W.+Abidaauthor=J.+J.+Alumkal&title=A+phase+Ib%2FIIa+study+of+the+pan-BET+inhibitor+ZEN-3694+in+combination+with+enzalutamide+in+patients+with+metastatic+castration-resistant+prostate+cancer&doi=10.1158%2F1078-0432.CCR-20-1707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">A phase Ib/lla study of the Pan-BET inhibitor ZEN-3694 in combination with enzalutamide in patients with metastatic castration-resistant prostate cancer</span></div><div class="casAuthors">Aggarwal, Rahul R.; Schweizer, Michael T.; Nanus, David M.; Pantuck, Allan J.; Heath, Elisabeth I.; Campeau, Eric; Attwell, Sarah; Norek, Karen; Snyder, Margo; Bauman, Lisa; Lakhotia, Sanjay; Feng, Felix Y.; Small, Eric J.; Abida, Wassim; Alumkal, Joshi J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5338-5347</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: ZEN-3694 is a bromodomain extraterminal inhibitor (BETi) with activity in androgen-signaling inhibitor (ASI)-resistant models.  The safety and efficacy of ZEN-3694 plus enzalutamide was evaluated in a phase Ib/IIa study in metastatic castration-resistant prostate cancer (mCRPC).  Patients and Methods: Patients had progressive mCRPC with prior resistance to abiraterone and/or enzalutanride. 3+3 dose escalation was followed by dose expansion in parallel cohorts (ZEN-3694 at 48 and 96 mg orally once daily, resp.).  Results: Seventy-five patients were enrolled (N = 26 and 14 in dose expansion at low- and high-dose ZEN-3694, resp.).  Thirty (40.0%) patients were resistant to abiraterone, 34 (45.3%) to enzalutanride, and 11 (14.7%) to both.  ZEN-3694 dosing ranged from 36 to 144 mg daily without reaching an MTD.  Fourteen patients (18.7%) experienced grade >3 toxicides, including three patients with grade 3 thrombocytopenia (4%).  An exposuredependent decrease in whole-blood RNA expression of BETi targets was obsd. (up to fourfold mean difference at 4 h post-ZEN-3694 dose; P 0.0001).  The median radiog. progression- free survival (rPFS) was 9.0 mo [95% confidence interval (CI), 4.6-12.9] and composite median radiog. or clin. progression-free survival (PFS) was 5.5 mo (95% CI, 4.0-7.8).  Median duration of treatment was 3.5 mo (range, 0-34.7+).  Lower androgen receptor (AR) transcriptional activity in baseline tumor biopsies was assocd. with longer rPFS (median rPFS 10.4 vs. 4.3 mo).  Conclusions: ZEN-3694 plus er.zalutamide demonstrated acceptable tolerability and potential efficacy in patients with ASIresistant mCRPC.  Further prospective study is warranted including in mCRPC harboring low AR transcriptional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry2ba_lIdCq7Vg90H21EOLACvtfcHk0lhBjdrX8pP_Wg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFCku7rP&md5=5bda2dace9b6c9fc3d3b2d3ad8718450</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-20-1707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-20-1707%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DR.%2BR.%26aulast%3DSchweizer%26aufirst%3DM.%2BT.%26aulast%3DNanus%26aufirst%3DD.%2BM.%26aulast%3DPantuck%26aufirst%3DA.%2BJ.%26aulast%3DHeath%26aufirst%3DE.%2BI.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DSnyder%26aufirst%3DM.%26aulast%3DBauman%26aufirst%3DL.%26aulast%3DLakhotia%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DSmall%26aufirst%3DE.%2BJ.%26aulast%3DAbida%26aufirst%3DW.%26aulast%3DAlumkal%26aufirst%3DJ.%2BJ.%26atitle%3DA%2520phase%2520Ib%252FIIa%2520study%2520of%2520the%2520pan-BET%2520inhibitor%2520ZEN-3694%2520in%2520combination%2520with%2520enzalutamide%2520in%2520patients%2520with%2520metastatic%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2020%26volume%3D26%26spage%3D5338%26epage%3D5347%26doi%3D10.1158%2F1078-0432.CCR-20-1707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carceller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">Advances in the development of histone lysine demethylase inhibitors</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">52</span>â <span class="NLM_lpage">60</span>, <span class="refDoi">Â DOI: 10.1016/j.coph.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.coph.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26057211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=52-60&author=T.+Maesauthor=E.+Carcellerauthor=J.+Salasauthor=A.+Ortegaauthor=C.+Buesa&title=Advances+in+the+development+of+histone+lysine+demethylase+inhibitors&doi=10.1016%2Fj.coph.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of histone lysine demethylase inhibitors</span></div><div class="casAuthors">Maes, Tamara; Carceller, Elena; Salas, Jordi; Ortega, Alberto; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The covalent modification of histones is closely assocd. with regulation of gene transcription.  Chromatin modifications have been suggested to represent an epigenetic code that is dynamically 'written' and 'erased' by specialized proteins, and 'read', or interpreted, by proteins that translate the code into gene expression changes.  Initially thought to be an irreversible process, histone methylation is now known to be reversed by demethylases, FAD dependent amineoxidases and by iron(II)-alpha-ketoglutarate dependent deoxygenases of the Jumonji family.  Altered histone demethylase activities have been assocd. with human disease, including cancer.  The first wave of novel investigational drugs directed against KDM1A has recently entered the clinic, and the first specific inhibitor targeting a Jumonji KDM is advancing in preclin. regulatory studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCr97IlCYFrVg90H21EOLACvtfcHk0lj_ucDo2v2PFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D&md5=02e8a5f8d426743be1eb209c329bd3ed</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DSalas%26aufirst%3DJ.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.coph.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>â <span class="NLM_lpage">953</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lj_ucDo2v2PFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): A computational study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e25444</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0025444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1371%2Fjournal.pone.0025444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21984927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=X.+Kongauthor=S.+Ouyangauthor=Z.+Liangauthor=J.+Luauthor=L.+Chenauthor=B.+Shenauthor=D.+Liauthor=M.+Zhengauthor=K.+K.+Liauthor=C.+Luoauthor=H.+Jiang&title=Catalytic+mechanism+investigation+of+lysine-specific+demethylase+1+%28LSD1%29%3A+A+computational+study&doi=10.1371%2Fjournal.pone.0025444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span></div><div class="casAuthors">Kong, Xiangqian; Ouyang, Sisheng; Liang, Zhongjie; Lu, Junyan; Chen, Liang; Shen, Bairong; Li, Donghai; Zheng, Mingyue; Li, Keqin Kathy; Luo, Cheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25444</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), the first identified histone demethylase, is a flavin-dependent amine oxidase which specifically demethylates mono- or di-methylated H3K4 and H3K9 via a redox process.  It participates in a broad spectrum of biol. processes and is of high importance in cell proliferation, adipogenesis, spermatogenesis, chromosome segregation and embryonic development.  To date, as a potential drug target for discovering anti-tumor drugs, the medical significance of LSD1 has been greatly appreciated.  However, the catalytic mechanism for the rate-limiting reductive half-reaction in demethylation remains controversial.  By employing a combined computational approach including mol. modeling, mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns., the catalytic mechanism of dimethylated H3K4 demethylation by LSD1 was characterized in details.  The three-dimensional (3D) model of the complex was composed of LSD1, CoREST, and histone substrate.  A 30-ns MD simulation of the model highlights the pivotal role of the conserved Tyr761 and lysine-water-flavin motif in properly orienting FAD with respect to substrate.  The synergy of the two factors effectively stabilizes the catalytic environment and facilitated the demethylation reaction.  On the basis of the reasonable consistence between simulation results and available mutagenesis data, QM/MM strategy was further employed to probe the catalytic mechanism of the reductive half-reaction in demethylation.  The characteristics of the demethylation pathway detd. by the potential energy surface and charge distribution anal. indicates that this reaction belongs to the direct hydride transfer mechanism.  Our study provides insights into the LSD1 mechanism of reductive half-reaction in demethylation and has important implications for the discovery of regulators against LSD1 enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIm_wJ2koFrVg90H21EOLACvtfcHk0lj_ucDo2v2PFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP&md5=7ccdcdcf6e8770ea0d77c1cc80f79da8</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025444%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCatalytic%2520mechanism%2520investigation%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%253A%2520A%2520computational%2520study%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0025444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>437</i></span>,  <span class="NLM_fpage">432</span>â <span class="NLM_lpage">435</span>, <span class="refDoi">Â DOI: 10.1038/nature04021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature04021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16079794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation&doi=10.1038%2Fnature04021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Cooch, Neil; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">432-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors have previously described a multiprotein complex termed the BHC or BRAF-HDAC complex, which is required for the repression of neuronal-specific genes.  The authors have shown that the BHC complex is recruited by a neuronal silencer, REST (RE1-silencing transcription factor), and mediates the repression of REST-responsive genes.  BHC is a multiprotein complex consisting of two enzymic activities: a histone deacetylase (HDAC1 or 2) and a recently described histone demethylase (BHC110, also known as LSD1 or AOF2).  Here the authors show that BHC110-contg. complexes show a nearly fivefold increase in demethylation of histone H3 lysine 4 (H3K4) compared to recombinant BHC110.  Furthermore, recombinant BHC110 is unable to demethylate H3K4 on nucleosomes, but BHC110-contg. complexes readily demethylate nucleosomes.  In vitro reconstitution of the BHC complex using recombinant subunits reveals an essential role for the REST corepressor CoREST, not only in stimulating demethylation on core histones but also promoting demethylation of nucleosomal substrates.  The authors find that nucleosomal demethylation is the result of CoREST enhancing the assocn. between BHC110 and nucleosomes.  Depletion of CoREST in in vivo cell culture results in de-repression of REST-responsive gene expression and increased methylation of H3K4.  Together, these results highlight an essential role for CoREST in demethylation of H3K4 both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmTdZApayNhrVg90H21EOLACvtfcHk0lj_ucDo2v2PFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D&md5=535a6d800bd6672192be133516b4dee7</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1038%2Fnature04021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04021%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435%26doi%3D10.1038%2Fnature04021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¼nther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SchÃ¼le, R.</span></span> <span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>437</i></span>,  <span class="NLM_fpage">436</span>â <span class="NLM_lpage">439</span>, <span class="refDoi">Â DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0liEE7gKcGyxIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perillo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ombra, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiariotti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malorni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbondanza, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avvedimento, E. V.</span></span> <span> </span><span class="NLM_article-title">DNA oxidation as triggered by H3K9me2 demethylation drives estrogen-induced gene expression</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">202</span>â <span class="NLM_lpage">206</span>, <span class="refDoi">Â DOI: 10.1126/science.1147674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1126%2Fscience.1147674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=18187655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD1cXis1amug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2008&pages=202-206&author=B.+Perilloauthor=M.+N.+Ombraauthor=A.+Bertoniauthor=C.+Cuozzoauthor=S.+Sacchettiauthor=A.+Sassoauthor=L.+Chiariottiauthor=A.+Malorniauthor=C.+Abbondanzaauthor=E.+V.+Avvedimento&title=DNA+oxidation+as+triggered+by+H3K9me2+demethylation+drives+estrogen-induced+gene+expression&doi=10.1126%2Fscience.1147674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Oxidation as Triggered by H3K9me2 Demethylation Drives Estrogen-Induced Gene Expression</span></div><div class="casAuthors">Perillo, Bruno; Ombra, Maria Neve; Bertoni, Alessandra; Cuozzo, Concetta; Sacchetti, Silvana; Sasso, Annarita; Chiariotti, Lorenzo; Malorni, Antonio; Abbondanza, Ciro; Avvedimento, Enrico V.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">5860</span>),
    <span class="NLM_cas:pages">202-206</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Modifications at the N-terminal tails of nucleosomal histones are required for efficient transcription in vivo.  We analyzed how H3 histone methylation and demethylation control expression of estrogen-responsive genes and show that a DNA-bound estrogen receptor directs transcription by participating in bending chromatin to contact the RNA polymerase II recruited to the promoter.  This process is driven by receptor-targeted demethylation of H3 lysine 9 at both enhancer and promoter sites and is achieved by activation of resident LSD1 demethylase.  Localized demethylation produces hydrogen peroxide, which modifies the surrounding DNA and recruits 8-oxoguanine-DNA glycosylase 1 and topoisomerase IIÎ², triggering chromatin and DNA conformational changes that are essential for estrogen-induced transcription.  Our data show a strategy that uses controlled DNA damage and repair to guide productive transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JvSAT1JwG7Vg90H21EOLACvtfcHk0liEE7gKcGyxIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXis1amug%253D%253D&md5=1208f5a6e8572df8deb2ef0588b6c102</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1126%2Fscience.1147674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1147674%26sid%3Dliteratum%253Aachs%26aulast%3DPerillo%26aufirst%3DB.%26aulast%3DOmbra%26aufirst%3DM.%2BN.%26aulast%3DBertoni%26aufirst%3DA.%26aulast%3DCuozzo%26aufirst%3DC.%26aulast%3DSacchetti%26aufirst%3DS.%26aulast%3DSasso%26aufirst%3DA.%26aulast%3DChiariotti%26aufirst%3DL.%26aulast%3DMalorni%26aufirst%3DA.%26aulast%3DAbbondanza%26aufirst%3DC.%26aulast%3DAvvedimento%26aufirst%3DE.%2BV.%26atitle%3DDNA%2520oxidation%2520as%2520triggered%2520by%2520H3K9me2%2520demethylation%2520drives%2520estrogen-induced%2520gene%2520expression%26jtitle%3DScience%26date%3D2008%26volume%3D319%26spage%3D202%26epage%3D206%26doi%3D10.1126%2Fscience.1147674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">A systematic review of histone lysine-specific demethylase 1 and its inhibitors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1032</span>â <span class="NLM_lpage">1071</span>, <span class="refDoi">Â DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+systematic+review+of+histone+lysine-specific+demethylase+1+and+its+inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0liEE7gKcGyxIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520systematic%2520review%2520of%2520histone%2520lysine-specific%2520demethylase%25201%2520and%2520its%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosseini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span> <span> </span><span class="NLM_article-title">A comprehensive review of lysine-specific demethylase 1 and its roles in cancer</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1123</span>â <span class="NLM_lpage">1142</span>, <span class="refDoi">Â DOI: 10.2217/epi-2017-0022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2217%2Fepi-2017-0022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28699367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVWnu7jP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1123-1142&author=A.+Hosseiniauthor=S.+Minucci&title=A+comprehensive+review+of+lysine-specific+demethylase+1+and+its+roles+in+cancer&doi=10.2217%2Fepi-2017-0022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive review of lysine-specific demethylase 1 and its roles in cancer</span></div><div class="casAuthors">Hosseini, Amir; Minucci, Saverio</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1123-1142</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone methylation plays a key role in the regulation of chromatin structure, and its dynamics regulates important cellular processes.  The investigation of the role of alterations in histone methylation in cancer has led to the identification of histone methyltransferases and demethylases as promising novel targets for therapy.  Lysine-specific demethylase 1(LSD1, also known as KDM1A) is the first discovered histone lysine demethylase, with the ability to demethylase H3K4me1/2 and H3K9me1/2 at target loci in a context-dependent manner.  LSD1 regulates the balance between self-renewal and differentiation of stem cells, and is highly expressed in various cancers, playing an important role in differentiation and self-renewal of tumor cells.  In this review, we summarize recent studies about the LSD1, its role in normal and tumor cells, and the potential use of small mol. LSD1 inhibitors in therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8i-cNutwmhbVg90H21EOLACvtfcHk0lg45e7s2vKu_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWnu7jP&md5=3e64834c4968dd8545dcce30c2bc5c45</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.2217%2Fepi-2017-0022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2017-0022%26sid%3Dliteratum%253Aachs%26aulast%3DHosseini%26aufirst%3DA.%26aulast%3DMinucci%26aufirst%3DS.%26atitle%3DA%2520comprehensive%2520review%2520of%2520lysine-specific%2520demethylase%25201%2520and%2520its%2520roles%2520in%2520cancer%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26spage%3D1123%26epage%3D1142%26doi%3D10.2217%2Fepi-2017-0022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Tranylcypromine based lysine-specific demethylase 1 inhibitor: Summary and perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14197</span>â <span class="NLM_lpage">14215</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14197-14215&author=X.+J.+Daiauthor=Y.+Liuauthor=X.+P.+Xiongauthor=L.+P.+Xueauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Tranylcypromine+based+lysine-specific+demethylase+1+inhibitor%3A+Summary+and+perspective&doi=10.1021%2Facs.jmedchem.0c00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective</span></div><div class="casAuthors">Dai, Xing-Jie; Liu, Ying; Xiong, Xiao-Peng; Xue, Lei-Peng; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14197-14215</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histone lysine-specific demethylase 1 (LSD1/KDM1A) has become an important and promising anticancer target since it was first identified in 2004 and specially demethylates lysine residues of histone H3K4me1/2 and H3K9me1/2.  LSD1 is ubiquitously overexpressed in diverse cancers, and abrogation of LSD1 results in inhibition of proliferation, invasion, and migration in cancer cells.  Over the past decade, a no. of biol. active small-mol. LSD1 inhibitors have been developed.  To date, six trans-2-phenylcyclopropylamine (TCP)-based LSD1 inhibitors (including TCP, ORY-1001, GSK-2879552, INCB059872, IMG-7289, and ORY-2001) that covalently bind to the FAD (FAD) within the LSD1 catalytic cavity have already entered into clin. trials.  Here, we provide an overview about the structures, activities, and structure-activity relationship (SAR) of TCP-based LSD1 inhibitors that mainly covers the literature from 2008 to date.  The opportunities, challenges, and future research directions in this emerging and promising field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqasvgaOVZu7Vg90H21EOLACvtfcHk0lg45e7s2vKu_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO&md5=88638bc10c8c090b3df3a0d2f2c020a8</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DX.%2BP.%26aulast%3DXue%26aufirst%3DL.%2BP.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTranylcypromine%2520based%2520lysine-specific%2520demethylase%25201%2520inhibitor%253A%2520Summary%2520and%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14197%26epage%3D14215%26doi%3D10.1021%2Facs.jmedchem.0c00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">LSD1/KDM1A inhibitors in clinical trials: Advances and prospects</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">129</span>, <span class="refDoi">Â DOI: 10.1186/s13045-019-0811-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1186%2Fs13045-019-0811-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31801559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=129&author=Y.+Fangauthor=G.+Liaoauthor=B.+Yu&title=LSD1%2FKDM1A+inhibitors+in+clinical+trials%3A+Advances+and+prospects&doi=10.1186%2Fs13045-019-0811-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1/KDM1A inhibitors in clinical trials: advances and prospects</span></div><div class="casAuthors">Fang Yuan; Liao Guochao; Yu Bin; Yu Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone demethylase LSD1 plays key roles during carcinogenesis, targeting LSD1 is becoming an emerging option for the treatment of cancers.  Numerous LSD1 inhibitors have been reported to date, some of them such as TCP, ORY-1001, GSK-2879552, IMG-7289, INCB059872, CC-90011, and ORY-2001 currently undergo clinical assessment for cancer therapy, particularly for small lung cancer cells (SCLC) and acute myeloid leukemia (AML).  This review is to provide a comprehensive overview of LSD1 inhibitors in clinical trials including molecular mechanistic studies, clinical efficacy, adverse drug reactions, and PD/PK studies and offer prospects in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbk3-5OqEjmWlSCSgBhwJFfW6udTcc2eZcOP3skgAb67ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D&md5=112299f15281667341a55dea57f6c83b</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1186%2Fs13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DLSD1%252FKDM1A%2520inhibitors%2520in%2520clinical%2520trials%253A%2520Advances%2520and%2520prospects%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2019%26volume%3D12%26spage%3D129%26doi%3D10.1186%2Fs13045-019-0811-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">Advances toward LSD1 inhibitors for cancer therapy</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1227</span>â <span class="NLM_lpage">1242</span>, <span class="refDoi">Â DOI: 10.4155/fmc-2017-0068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.4155%2Ffmc-2017-0068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28722477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1227-1242&author=X.+Fuauthor=P.+Zhangauthor=B.+Yu&title=Advances+toward+LSD1+inhibitors+for+cancer+therapy&doi=10.4155%2Ffmc-2017-0068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Advances toward LSD1 inhibitors for cancer therapy</span></div><div class="casAuthors">Fu, Xiaoli; Zhang, Peng; Yu, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1227-1242</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  LSD1 has become an important biol. validated epigenetic target for cancer therapy since its identification in 2004.  LSD1 mediates many cellular signaling pathways and is involved in the initiation and development of cancers.  Aberrant overexpression of LSD1 has been obsd. in different types of cancers, and inactivation by small mols. suppresses cancer cell differentiation, proliferation, invasion and migration.  To date, a large no. of LSD1 inhibitors have been reported, RG6016, GSK-2879552, INCB059872, IMG-7289 and CC-90011 are currently undergoing clin. assessment for the treatment of acute myeloid leukemia, small-cell lung cancer, etc.  In this review, we briefly highlight recent advances of LSD1 inhibitors mainly covering the literatures from 2015 to 2017 and tentatively propose our perspectives on the design of new LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlJAzt51S4crVg90H21EOLACvtfcHk0lg0xKzfQOfvGg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK&md5=182f10eb7a2aea7b5c13f50f84f44c20</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DAdvances%2520toward%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1227%26epage%3D1242%26doi%3D10.4155%2Ffmc-2017-0068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Magid, A. F.</span></span> <span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) inhibitors as potential treatment for different types of cancers</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1134</span>â <span class="NLM_lpage">1135</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.7b00426</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslaisLzI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1134-1135&author=A.+F.+Abdel-Magid&title=Lysine-specific+demethylase+1+%28LSD1%29+inhibitors+as+potential+treatment+for+different+types+of+cancers&doi=10.1021%2Facsmedchemlett.7b00426"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Specific Demethylase 1 (LSD1) Inhibitors as Potential Treatment for Different Types of Cancers</span></div><div class="casAuthors">Abdel-Magid, Ahmed F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1134-1135</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The invention in this patent application relates to 5-cyano indole derivs.  These compd. are lysine-specific demethylase 1(LSD1) inhibitors and maybe useful for the treatment of LSD1-mediated diseases or disorders including, but are not limited to, B cell lymphoma, acute myeloid leukemia, gastric cancer, hepatocellular carcinoma, prostate cancer, breast carcinoma, neuroblastoma, glioblastoma, nasopharyngeal carcinoma, colon cancer, gallbladder cancer, esophageal cancer, head and neck cancer, lung cancer, ovarian cancer, pancreatic cancer, endometrial carcinoma, and soft tissue sarcomas such as rhabdomyosarcoma (RMS), chondrosarcoma, osteosarcoma, Ewing's sarcoma, liver fibrosis, and sickle cell disease.  Studies have shown that LSD1 is involved in cell proliferation, epithelial-mesenchymal transition, stem cell biol., malignant transformation of cells, and cell differentiation.  Dysfunction in the activities of LSD1 is believed to be responsible for a no. of myeloproliferative and lymphoproliferative diseases such as acute myeloid leukemia (AML) and acute lymphoblastic leukemia (ALL).  Therefore, the inhibition of LSD1 provides a promising pharmacol. target to design novel treatments for cancer and other disorders that assoc. with LSD1's activity.  In particular, the need exists for novel small mols. that inhibit the activity of LSD1, such as the compds. of formula I of this patent application, which includes both irreversible and reversible inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohWPHUFQw3O7Vg90H21EOLACvtfcHk0lg0xKzfQOfvGg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslaisLzI&md5=860645e21019b91e1f2bf6d6893be4d8</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00426%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520inhibitors%2520as%2520potential%2520treatment%2520for%2520different%2520types%2520of%2520cancers%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1134%26epage%3D1135%26doi%3D10.1021%2Facsmedchemlett.7b00426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willeit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Praschak-Rieder, N.</span></span> <span> </span><span class="NLM_article-title">Imaging the effects of genetic polymorphisms on radioligand binding in the living human brain: A review on genetic neuroreceptor imaging of monoaminergic systems in psychiatry</span>. <i>NeuroImage</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">878</span>â <span class="NLM_lpage">892</span>, <span class="refDoi">Â DOI: 10.1016/j.neuroimage.2010.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.neuroimage.2010.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20399868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWqtrvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=878-892&author=M.+Willeitauthor=N.+Praschak-Rieder&title=Imaging+the+effects+of+genetic+polymorphisms+on+radioligand+binding+in+the+living+human+brain%3A+A+review+on+genetic+neuroreceptor+imaging+of+monoaminergic+systems+in+psychiatry&doi=10.1016%2Fj.neuroimage.2010.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Imaging the effects of genetic polymorphisms on radioligand binding in the living human brain: A review on genetic neuroreceptor imaging of monoaminergic systems in psychiatry</span></div><div class="casAuthors">Willeit, Matthaeus; Praschak-Rieder, Nicole</div><div class="citationInfo"><span class="NLM_cas:title">NeuroImage</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">878-892</span>CODEN:
                <span class="NLM_cas:coden">NEIMEF</span>;
        ISSN:<span class="NLM_cas:issn">1053-8119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Imaging genetics is a research field that describes the impact of genetic risk variants on brain structure and function.  While magnetic resonance based imaging techniques are able to provide complex information on a system level, positron emission tomog. (PET) and single photon emission computer tomog. (SPECT) allow for detn. of distribution and d. of single receptor mols. in the human brain.  Major psychiatric disorders are highly heritable, and have been assocd. with a dysregulation in brain dopamine and serotonin systems.  Understanding the role of genetic polymorphisms within these neurotransmitter systems on brain phenotype is essential.  This review tries to cover the literature on the impact of gene variants implicated in psychiatric disorders on serotonin, dopamine, and MAO-A radioligand binding in living humans.  The majority of PET and SPECT studies investigated the role of polymorphisms within genes coding for the serotonin and dopamine transporters, the serotonin 1A receptor, and the dopamine D2 receptor on G protein coupled receptors or transporter proteins critically involved in serotonin or dopamine neurotransmission.  Other studies investigated the impact of variants in genes for monoamine oxidase-A (MAO-A) or brain derived neurotrophic factor on monoamine transporters, receptors, or MAO-A activity.  Two main findings in healthy subjects emerge from the current literature: one is an increased binding of the selective ligand [11C]DASB to serotonin transporters in subjects homozygous for the triallelic 5-HTTLPR LA allele.  The other one is decreased binding of the radioligand [11C]raclopride to dopamine D2 receptors in D2 Taq1 A1 allele carriers.  Other findings reported are highly interesting but require independent replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuC-oqxTXqrVg90H21EOLACvtfcHk0lg0xKzfQOfvGg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWqtrvO&md5=7a679dc2e01de0c3b53527f88185e31c</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroimage.2010.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroimage.2010.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DWilleit%26aufirst%3DM.%26aulast%3DPraschak-Rieder%26aufirst%3DN.%26atitle%3DImaging%2520the%2520effects%2520of%2520genetic%2520polymorphisms%2520on%2520radioligand%2520binding%2520in%2520the%2520living%2520human%2520brain%253A%2520A%2520review%2520on%2520genetic%2520neuroreceptor%2520imaging%2520of%2520monoaminergic%2520systems%2520in%2520psychiatry%26jtitle%3DNeuroImage%26date%3D2010%26volume%3D53%26spage%3D878%26epage%3D892%26doi%3D10.1016%2Fj.neuroimage.2010.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">Histone H3 lysine 4 demethylation is a target of nonselective antidepressive medications</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">563</span>â <span class="NLM_lpage">567</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+lysine+4+demethylation+is+a+target+of+nonselective+antidepressive+medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 Î¼M.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0lilehYemaMvZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520lysine%25204%2520demethylation%2520is%2520a%2520target%2520of%2520nonselective%2520antidepressive%2520medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, D. G.</span></span> <span> </span><span class="NLM_article-title">Trans-2-phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4408</span>â <span class="NLM_lpage">4416</span>, <span class="refDoi">Â DOI: 10.1021/bi0618621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0618621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4408-4416&author=D.+M.+Schmidtauthor=D.+G.+McCafferty&title=Trans-2-phenylcyclopropylamine+is+a+mechanism-based+inactivator+of+the+histone+demethylase+LSD1&doi=10.1021%2Fbi0618621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">trans-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span></div><div class="casAuthors">Schmidt, Dawn M. Z.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4408-4416</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic domain of the flavin-dependent human histone demethylase, lysine-specific demethylase 1 (LSD1), belongs to the family of amine oxidases including polyamine oxidase and monoamine oxidase (MAO).  The authors previously assessed MAO inhibitors (MAOIs) for their ability to inhibit the reaction catalyzed by LSD1, demonstrating that trans-2-phenylcyclopropylamine (2-PCPA; tranylcypromine, Parnate) was the most potent inhibitor with respect to LSD1.  Here, the authors showed that 2-PCPA was a time-dependent, mechanism-based irreversible inhibitor of LSD1 with a Ki of 242 Î¼M and a kinact of 0.0106 s-1.  2-PCPA showed limited selectivity for human MAOs vs. LSD1, with kinact/Ki values only 16-fold and 2.4-fold higher for MAO B and MAO A, resp.  Profiles of LSD1 activity and inactivation by 2-PCPA as a function of pH were consistent with a mechanism of inactivation dependent upon enzyme catalysis.  Mass spectrometry supported a role for FAD as the site of covalent modification by 2-PCPA.  These results will provide a foundation for the design of cyclopropylamine-based inhibitors that are selective for LSD1 to probe its role in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVK59U7IsBvbVg90H21EOLACvtfcHk0lilehYemaMvZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D&md5=02925342b86c7dbfb5f1a99fa5264adc</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1021%2Fbi0618621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0618621%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DTrans-2-phenylcyclopropylamine%2520is%2520a%2520mechanism-based%2520inactivator%2520of%2520the%2520histone%2520demethylase%2520LSD1%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D4408%26epage%3D4416%26doi%3D10.1021%2Fbi0618621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culhane, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczuk, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalili, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the inhibition of the LSD1 histone demethylase by the antidepressant trans-2-phenylcyclopropylamine</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">8058</span>â <span class="NLM_lpage">8065</span>, <span class="refDoi">Â DOI: 10.1021/bi700664y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi700664y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=8058-8065&author=M.+Yangauthor=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=P.+Jaliliauthor=H.+L.+Ballauthor=M.+Machiusauthor=P.+A.+Coleauthor=H.+Yu&title=Structural+basis+for+the+inhibition+of+the+LSD1+histone+demethylase+by+the+antidepressant+trans-2-phenylcyclopropylamine&doi=10.1021%2Fbi700664y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Inhibition of the LSD1 Histone Demethylase by the Antidepressant trans-2-Phenylcyclopropylamine</span></div><div class="casAuthors">Yang, Maojun; Culhane, Jeffrey C.; Szewczuk, Lawrence M.; Jalili, Pegah; Ball, Haydn L.; Machius, Mischa; Cole, Philip A.; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8058-8065</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone modifications, such as acetylation and methylation, are important epigenetic marks that regulate diverse biol. processes that use chromatin as the template, including transcription.  Dysregulation of histone acetylation and methylation leads to the silencing of tumor suppressor genes and contributes to cancer progression.  Inhibitors of enzymes that catalyze the addn. and removal of these epigenetic marks thus have therapeutic potential for treating cancer.  Lysine-specific demethylase 1 (LSD1) is the first discovered histone lysine demethylase and with the help of its cofactor CoREST, specifically demethylates mono- and dimethylated histone H3 lysine 4 (H3-K4), thus repressing transcription.  Because LSD1 belongs to the family of FAD (FAD)-dependent amine oxidases, certain inhibitors of monoamine oxidases (MAOs), including the clin. used antidepressant trans-2-phenylcyclopropylamine (PCPA; tranylcypromine; Parnate), are also capable of inhibiting LSD1.  In this study, we have further measured the kinetic parameters of the inhibition of LSD1 by PCPA and detd. the crystal structure of LSD1-CoREST in the presence of PCPA.  Our structural and mass spectrometry analyses are consistent with PCPA forming a covalent adduct with FAD in LSD1 that is distinct from the FAD-PCPA adduct of MAO B.  The structure also reveals that the Ph ring of the FAD-PCPA adduct in LSD1 does not form extensive interactions with active-site residues.  This study thus provides the basis for designing more potent inhibitors of LSD1 that contain substitutions on the Ph ring of PCPA to fully engage neighboring residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqcLmzpB1S7Vg90H21EOLACvtfcHk0lilehYemaMvZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsVKmsr8%253D&md5=e11fe93832d46962b6c4848f682042d2</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1021%2Fbi700664y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi700664y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DJalili%26aufirst%3DP.%26aulast%3DBall%26aufirst%3DH.%2BL.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520the%2520LSD1%2520histone%2520demethylase%2520by%2520the%2520antidepressant%2520trans-2-phenylcyclopropylamine%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D8058%26epage%3D8065%26doi%3D10.1021%2Fbi700664y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span> <span> </span><span class="NLM_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">340</span>â <span class="NLM_lpage">352</span>, <span class="refDoi">Â DOI: 10.1080/15592294.2017.1290032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1080%2F15592294.2017.1290032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28277979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=340-352&author=H.+Niwaauthor=T.+Umehara&title=Structural+insight+into+inhibitors+of+flavin+adenine+dinucleotide-dependent+lysine+demethylases&doi=10.1080%2F15592294.2017.1290032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span></div><div class="casAuthors">Niwa Hideaki; Umehara Takashi; Umehara Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-352</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Until 2004, many researchers believed that protein methylation in eukaryotic cells was an irreversible reaction.  However, the discovery of lysine-specific demethylase 1 in 2004 drastically changed this view and the concept of chromatin regulation.  Since then, the enzymes responsible for lysine demethylation and their cellular substrates, biological significance, and selective regulation have become major research topics in epigenetics and chromatin biology.  Many cell-permeable inhibitors for lysine demethylases have been developed, including both target-specific and nonspecific inhibitors.  Structural understanding of how these inhibitors bind to lysine demethylases is crucial both for validation of the inhibitors as chemical probes and for the rational design of more potent, target-specific inhibitors.  This review focuses on published small-molecule inhibitors targeted at the two flavin adenine dinucleotide-dependent lysine demethylases, lysine-specific demethylases 1 and 2, and how the inhibitors interact with the tertiary structures of the enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyKG85Hfra9xjv2lLSLOxFfW6udTcc2eYu8d3APS1h4Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D&md5=31f93c2cc7909c4197f5892f7f04af9d</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1080%2F15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DUmehara%26aufirst%3DT.%26atitle%3DStructural%2520insight%2520into%2520inhibitors%2520of%2520flavin%2520adenine%2520dinucleotide-dependent%2520lysine%2520demethylases%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D340%26epage%3D352%26doi%3D10.1080%2F15592294.2017.1290032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MascarÃ³, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirapu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lufino, M. M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willekens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valls, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fyfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro-Palomino, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-1001, a potent and selective covalent KDM1A inhibitor, for the treatment of acute leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">495</span>â <span class="NLM_lpage">511</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29502954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=495-511&author=T.+Maesauthor=C.+Mascar%C3%B3author=I.+Tirapuauthor=A.+Estiarteauthor=F.+Ciceriauthor=S.+Lunardiauthor=N.+Guibourtauthor=A.+Perdonesauthor=M.+M.+P.+Lufinoauthor=T.+C.+P.+Somervailleauthor=D.+H.+Wisemanauthor=C.+Duyauthor=A.+Melnickauthor=C.+Willekensauthor=A.+Ortegaauthor=M.+Martinellauthor=N.+Vallsauthor=G.+Kurzauthor=M.+Fyfeauthor=J.+C.+Castro-Palominoauthor=C.+Buesa&title=ORY-1001%2C+a+potent+and+selective+covalent+KDM1A+inhibitor%2C+for+the+treatment+of+acute+leukemia&doi=10.1016%2Fj.ccell.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">ORY-1001, a Potent and Selective Covalent KDM1A Inhibitor, for the Treatment of Acute Leukemia</span></div><div class="casAuthors">Maes, Tamara; Mascaro, Cristina; Tirapu, Inigo; Estiarte, Angels; Ciceri, Filippo; Lunardi, Serena; Guibourt, Nathalie; Perdones, Alvaro; Lufino, Michele M. P.; Somervaille, Tim C. P.; Wiseman, Dan H.; Duy, Cihangir; Melnick, Ari; Willekens, Christophe; Ortega, Alberto; Martinell, Marc; Valls, Nuria; Kurz, Guido; Fyfe, Matthew; Castro-Palomino, Julio Cesar; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-511.e12</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lysine-specific demethylase KDM1A is a key regulator of stem cell potential in acute myeloid leukemia (AML).  ORY-1001 is a highly potent and selective KDM1A inhibitor that induces H3K4me2 accumulation on KDM1A target genes, blast differentiation, and redn. of leukemic stem cell capacity in AML.  ORY-1001 exhibits potent synergy with std.-of-care drugs and selective epigenetic inhibitors, reduces growth of an AML xenograft model, and extends survival in a mouse PDX (patient-derived xenograft) model of T cell acute leukemia.  Surrogate pharmacodynamic biomarkers developed based on expression changes in leukemia cell lines were translated to samples from patients treated with ORY-1001.  ORY-1001 is a selective KDM1A inhibitor in clin. trials and is currently being evaluated in patients with leukemia and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2QFk3s8-eLVg90H21EOLACvtfcHk0lgHPk2qYD2ngQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D&md5=68f69ade1b0110f3e0adf0173326e9b7</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascar%25C3%25B3%26aufirst%3DC.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DLunardi%26aufirst%3DS.%26aulast%3DGuibourt%26aufirst%3DN.%26aulast%3DPerdones%26aufirst%3DA.%26aulast%3DLufino%26aufirst%3DM.%2BM.%2BP.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DWillekens%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMartinell%26aufirst%3DM.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DKurz%26aufirst%3DG.%26aulast%3DFyfe%26aufirst%3DM.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%2BC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-1001%252C%2520a%2520potent%2520and%2520selective%2520covalent%2520KDM1A%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520acute%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D495%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degenhardt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDevitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crouthamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span> <span> </span><span class="NLM_article-title">A DNA hypomethylation signature predicts antitumor activity of LSD1 inhibitors in SCLC</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">57</span>â <span class="NLM_lpage">69</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+hypomethylation+signature+predicts+antitumor+activity+of+LSD1+inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lgHPk2qYD2ngQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520hypomethylation%2520signature%2520predicts%2520antitumor%2520activity%2520of%2520LSD1%2520inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MascarÃ³, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotllant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GriÃ±an-FerrÃ©, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PallÃ s, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span> <span> </span><span class="NLM_article-title">The dual LSD1/MAOB inhibitor ORY 2001 prevents the development of the memory deficit in samp8 mice through induction of neuronal plasticity and reduction of neuroinflammation</span>. <i>Alzheimer's Dementia</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">P905</span>, <span class="refDoi">Â DOI: 10.1016/j.jalz.2015.08.146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.jalz.2015.08.146" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=P905&author=C.+Buesaauthor=C.+Mascar%C3%B3author=D.+Rotllantauthor=C.+Gri%C3%B1an-Ferr%C3%A9author=M.+Pall%C3%A0sauthor=T.+Maes&title=The+dual+LSD1%2FMAOB+inhibitor+ORY+2001+prevents+the+development+of+the+memory+deficit+in+samp8+mice+through+induction+of+neuronal+plasticity+and+reduction+of+neuroinflammation&doi=10.1016%2Fj.jalz.2015.08.146"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1016%2Fj.jalz.2015.08.146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jalz.2015.08.146%26sid%3Dliteratum%253Aachs%26aulast%3DBuesa%26aufirst%3DC.%26aulast%3DMascar%25C3%25B3%26aufirst%3DC.%26aulast%3DRotllant%26aufirst%3DD.%26aulast%3DGri%25C3%25B1an-Ferr%25C3%25A9%26aufirst%3DC.%26aulast%3DPall%25C3%25A0s%26aufirst%3DM.%26aulast%3DMaes%26aufirst%3DT.%26atitle%3DThe%2520dual%2520LSD1%252FMAOB%2520inhibitor%2520ORY%25202001%2520prevents%2520the%2520development%2520of%2520the%2520memory%2520deficit%2520in%2520samp8%2520mice%2520through%2520induction%2520of%2520neuronal%2520plasticity%2520and%2520reduction%2520of%2520neuroinflammation%26jtitle%3DAlzheimer%2527s%2520Dementia%26date%3D2015%26volume%3D11%26spage%3DP905%26doi%3D10.1016%2Fj.jalz.2015.08.146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MascarÃ³, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotllant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalcanti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carceller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-2001: An epigenetic drug for the treatment of cognition defects in alzheimerâs disease and other neurodegenerative disorders</span>. <i>Alzheimerâs Dementia</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">P1192</span>, <span class="refDoi">Â DOI: 10.1016/j.jalz.2016.07.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.jalz.2016.07.149" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=P1192&author=T.+Maesauthor=C.+Mascar%C3%B3author=D.+Rotllantauthor=F.+Cavalcantiauthor=E.+Carcellerauthor=A.+Ortegaauthor=C.+Molineroauthor=C.+Buesa&title=ORY-2001%3A+An+epigenetic+drug+for+the+treatment+of+cognition+defects+in+alzheimer%E2%80%99s+disease+and+other+neurodegenerative+disorders&doi=10.1016%2Fj.jalz.2016.07.149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1016%2Fj.jalz.2016.07.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jalz.2016.07.149%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascar%25C3%25B3%26aufirst%3DC.%26aulast%3DRotllant%26aufirst%3DD.%26aulast%3DCavalcanti%26aufirst%3DF.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMolinero%26aufirst%3DC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-2001%253A%2520An%2520epigenetic%2520drug%2520for%2520the%2520treatment%2520of%2520cognition%2520defects%2520in%2520alzheimer%25E2%2580%2599s%2520disease%2520and%2520other%2520neurodegenerative%2520disorders%26jtitle%3DAlzheimer%25E2%2580%2599s%2520Dementia%26date%3D2016%26volume%3D12%26spage%3DP1192%26doi%3D10.1016%2Fj.jalz.2016.07.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, O. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">4364</span>â <span class="NLM_lpage">4397</span>, <span class="refDoi">Â DOI: 10.1039/c0cs00203h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-4397&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases&doi=10.1039%2Fc0cs00203h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0li576ngH0Sxrw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D4397%26doi%3D10.1039%2Fc0cs00203h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoeller, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of a linked jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">2631</span>â <span class="NLM_lpage">2646</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M115.698449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1074%2Fjbc.M115.698449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=26645689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=2631-2646&author=J.+R.+Hortonauthor=A.+Engstromauthor=E.+L.+Zoellerauthor=X.+Liuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=M.+A.+Johnsauthor=P.+M.+Vertinoauthor=H.+Fuauthor=X.+Cheng&title=Characterization+of+a+linked+jumonji+domain+of+the+KDM5%2FJARID1+family+of+histone+H3+lysine+4+demethylases&doi=10.1074%2Fjbc.M115.698449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a Linked Jumonji Domain of the KDM5/JARID1 Family of Histone H3 Lysine 4 Demethylases</span></div><div class="casAuthors">Horton, John R.; Engstrom, Amanda; Zoeller, Elizabeth L.; Liu, Xu; Shanks, John R.; Zhang, Xing; Johns, Margaret A.; Vertino, Paula M.; Fu, Haian; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2631-2646</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and Î±-ketoglutarate-dependent demethylases remove Me groups from tri- and dimethylated lysine 4 of histone H3.  Accumulating evidence from primary tumors and model systems supports a role for KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) as oncogenic drivers.  The KDM5 family is unique among the Jumonji domain-contg. histone demethylases in that there is an atypical insertion of a DNA-binding ARID domain and a histone-binding PHD domain into the Jumonji domain, which separates the catalytic domain into two fragments (JmjN and JmjC).  Here we demonstrate that internal deletion of the ARID and PHD1 domains has a negligible effect on in vitro enzymic kinetics of the KDM5 family of enzymes.  We present a crystal structure of the linked JmjN-JmjC domain from KDM5A, which reveals that the linked domain fully reconstitutes the cofactor (metal ion and Î±-ketoglutarate) binding characteristics of other structurally characterized Jumonji domain demethylases.  Docking studies with GSK-J1, a selective inhibitor of the KDM6/KDM5 subfamilies, identify crit. residues for binding of the inhibitor to the reconstituted KDM5 Jumonji domain.  Further, we found that GSK-J1 inhibited the demethylase activity of KDM5C with 8.5-fold increased potency compared with that of KDM5B at 1 mm Î±-ketoglutarate.  In contrast, JIB-04 (a pan-inhibitor of the Jumonji demethylase superfamily) had the opposite effect and was â¼8-fold more potent against KDM5B than against KDM5C.  Interestingly, the relative selectivity of JIB-04 toward KDM5B over KDM5C in vitro translates to a â¼10-50-fold greater growth-inhibitory activity against breast cancer cell lines.  These data define the minimal requirements for enzymic activity of the KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical zinc-binding domain and provide structural characterization of the linked JmjN-JmjC domain for the KDM5 family, which should prove useful in the design of KDM5 demethylase inhibitors with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa4qqOk0zHF7Vg90H21EOLACvtfcHk0lgucZnywyT4jw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D&md5=772a106c3c7f0a45c1b3d2ee81cc9487</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.698449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.698449%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DEngstrom%26aufirst%3DA.%26aulast%3DZoeller%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520linked%2520jumonji%2520domain%2520of%2520the%2520KDM5%252FJARID1%2520family%2520of%2520histone%2520H3%2520lysine%25204%2520demethylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D2631%26epage%3D2646%26doi%3D10.1074%2Fjbc.M115.698449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staller, P.</span></span> <span> </span><span class="NLM_article-title">The KDM5 family of histone demethylases as targets in oncology drug discovery</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>â <span class="NLM_lpage">286</span>, <span class="refDoi">Â DOI: 10.2217/epi.14.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2217%2Fepi.14.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=25111482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=277-286&author=P.+B.+Rasmussenauthor=P.+Staller&title=The+KDM5+family+of+histone+demethylases+as+targets+in+oncology+drug+discovery&doi=10.2217%2Fepi.14.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">The KDM5 family of histone demethylases as targets in oncology drug discovery</span></div><div class="casAuthors">Rasmussen, Peter Birk; Staller, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-286</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing evidence for a causal role of the KDM5 family of histone demethylases in human cancer.  In particular, KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) contribute to cancer cell proliferation, reduce the expression of tumor suppressor genes, promote the development of drug tolerance and maintain tumor-initiating cells.  KDM5 enzymes remove tri- and di-methylations of lysine 4 of histone H3 - modifications that occur at the start site of transcription in actively transcribed genes.  However, the importance of the histone demethylase activity of KDM5 proteins for cancer cells has not been resolved so far.  The currently available approaches suppress or remove the targeted proteins and thereby affect their putative functions as structural components and recruitment factors for other chromatin-assocd. proteins.  Therefore, the development of specific enzymic inhibitors for KDM5 will promote our understanding of the biol. role of their catalytic activity and yield potential novel anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPJyGCVkwViLVg90H21EOLACvtfcHk0lgucZnywyT4jw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K&md5=0b8dc67f4fde6ce24457dce092dbdd83</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.14%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DStaller%26aufirst%3DP.%26atitle%3DThe%2520KDM5%2520family%2520of%2520histone%2520demethylases%2520as%2520targets%2520in%2520oncology%2520drug%2520discovery%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D277%26epage%3D286%26doi%3D10.2217%2Fepi.14.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1383</span>â <span class="NLM_lpage">1404</span>, <span class="refDoi">Â DOI: 10.3390/cancers3011383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.3390%2Fcancers3011383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=21544224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1383-1404&author=L.+P.+Blairauthor=J.+Caoauthor=M.+R.+Zouauthor=J.+Sayeghauthor=Q.+Yan&title=Epigenetic+regulation+by+lysine+demethylase+5+%28KDM5%29+enzymes+in+cancer&doi=10.3390%2Fcancers3011383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span></div><div class="casAuthors">Blair, Lauren P.; Cao, Jian; Zou, Mike Ran; Sayegh, Joyce; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1383-1404</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Similar to genetic alterations, epigenetic aberrations contribute significantly to tumor initiation and progression.  In many cases, these changes are caused by activation or inactivation of the regulators that maintain epigenetic states.  Here we review our current knowledge on the KDM5/JARID1 family of histone demethylases.  This family of enzymes contains a JmjC domain and is capable of removing tri- and di- Me marks from lysine 4 on histone H3.  Among these proteins, RBP2 mediates drug resistance while JARID1B is required for melanoma maintenance.  Preclin. studies suggest inhibition of these enzymes can suppress tumorigenesis and provide strong rationale for development of their inhibitors for use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8lTt6GjrOJrVg90H21EOLACvtfcHk0lhpFZvfcV_N9g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D&md5=426fa61d08aa818b45781b57e1aba5f3</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.3390%2Fcancers3011383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3011383%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DEpigenetic%2520regulation%2520by%2520lysine%2520demethylase%25205%2520%2528KDM5%2529%2520enzymes%2520in%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D1383%26epage%3D1404%26doi%3D10.3390%2Fcancers3011383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shokri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fathi-Roudsari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouhkan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanati, M. H.</span></span> <span> </span><span class="NLM_article-title">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">105</span>â <span class="NLM_lpage">111</span>, <span class="refDoi">Â DOI: 10.1016/j.leukres.2018.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.leukres.2018.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29602065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=105-111&author=G.+Shokriauthor=S.+Doudiauthor=M.+Fathi-Roudsariauthor=F.+Kouhkanauthor=M.+H.+Sanati&title=Targeting+histone+demethylases+KDM5A+and+KDM5B+in+AML+cancer+cells%3A+A+comparative+view&doi=10.1016%2Fj.leukres.2018.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span></div><div class="casAuthors">Shokri, Gelareh; Doudi, Shaghayegh; Fathi-Roudsari, Mehrnoosh; Kouhkan, Fatemeh; Sanati, Mohammad-Hossein</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-111</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Epigenetic modifications play an important role in initiation and progression of cancers including acute myeloid leukemia.  Among different epigenetic modifiers, lysine specific demethylases have been noticed as potential therapeutic targets.  KDM5 family of histone demethylases which removes Me marks from lysine residues of H3, are frequently found in the promoter region of transcriptionally active genes resulting in repression of expression.  Here we have compared the effects of KDM5A and KDM5B downregulation on HL-60 cell line behavior.  KDM5A/5B knockdown resulted in lower viability of HL-60 cells in addn. to modified cell cycle distribution and sub-G1 accumulation.  Induction of apoptosis was obsd. in both knockdown cells.  But in spite of similarity in their role, downregulation of KDM5A showed more efficient anti-leukemic effects in comparison to KDM5B.  Cells showed higher accumulation in sub-G1 and apoptosis occurred significantly higher and also earlier after KDM5A redn.  Expression anal. confirmed almost 5 and 4 fold increased expression for bax and caspase-3 after downregulation of KDM5A in comparison to KDM5B.  Due to the present study we propose KDM5A as a potential target for therapeutic aspects of acute myeloid leukemia although further investigations are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIUHbiIE13rVg90H21EOLACvtfcHk0lhpFZvfcV_N9g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D&md5=22a34070de67e600b8164273c1112405</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2018.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2018.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DShokri%26aufirst%3DG.%26aulast%3DDoudi%26aufirst%3DS.%26aulast%3DFathi-Roudsari%26aufirst%3DM.%26aulast%3DKouhkan%26aufirst%3DF.%26aulast%3DSanati%26aufirst%3DM.%2BH.%26atitle%3DTargeting%2520histone%2520demethylases%2520KDM5A%2520and%2520KDM5B%2520in%2520AML%2520cancer%2520cells%253A%2520A%2520comparative%2520view%26jtitle%3DLeuk.%2520Res.%26date%3D2018%26volume%3D68%26spage%3D105%26epage%3D111%26doi%3D10.1016%2Fj.leukres.2018.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalgliesh, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bignell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edkins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barthorpe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beare, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kok, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddison, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLay, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mironenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulderrig, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâMeara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleasance, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajasingham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stebbings, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarpey, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykema, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoo, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petillo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wondergem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anema, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahnoski, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teh, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, P. A.</span></span> <span> </span><span class="NLM_article-title">Systematic sequencing of renal carcinoma reveals inactivation of histone modifying genes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">360</span>â <span class="NLM_lpage">363</span>, <span class="refDoi">Â DOI: 10.1038/nature08672</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnature08672" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=20054297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFeqtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=360-363&author=G.+L.+Dalglieshauthor=K.+Furgeauthor=C.+Greenmanauthor=L.+Chenauthor=G.+Bignellauthor=A.+Butlerauthor=H.+Daviesauthor=S.+Edkinsauthor=C.+Hardyauthor=C.+Latimerauthor=J.+Teagueauthor=J.+Andrewsauthor=S.+Barthorpeauthor=D.+Beareauthor=G.+Buckauthor=P.+J.+Campbellauthor=S.+Forbesauthor=M.+Jiaauthor=D.+Jonesauthor=H.+Knottauthor=C.+Y.+Kokauthor=K.+W.+Lauauthor=C.+Leroyauthor=M.+L.+Linauthor=D.+J.+McBrideauthor=M.+Maddisonauthor=S.+Maguireauthor=K.+McLayauthor=A.+Menziesauthor=T.+Mironenkoauthor=L.+Mulderrigauthor=L.+Mudieauthor=S.+O%E2%80%99Mearaauthor=E.+Pleasanceauthor=A.+Rajasinghamauthor=R.+Shepherdauthor=R.+Smithauthor=L.+Stebbingsauthor=P.+Stephensauthor=G.+Tangauthor=P.+S.+Tarpeyauthor=K.+Turrellauthor=K.+J.+Dykemaauthor=S.+K.+Khooauthor=D.+Petilloauthor=B.+Wondergemauthor=J.+Anemaauthor=R.+J.+Kahnoskiauthor=B.+T.+Tehauthor=M.+R.+Strattonauthor=P.+A.+Futreal&title=Systematic+sequencing+of+renal+carcinoma+reveals+inactivation+of+histone+modifying+genes&doi=10.1038%2Fnature08672"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic sequencing of renal carcinoma reveals inactivation of histone modifying genes</span></div><div class="casAuthors">Dalgliesh, Gillian L.; Furge, Kyle; Greenman, Chris; Chen, Lina; Bignell, Graham; Butler, Adam; Davies, Helen; Edkins, Sarah; Hardy, Claire; Latimer, Calli; Teague, Jon; Andrews, Jenny; Barthorpe, Syd; Beare, Dave; Buck, Gemma; Campbell, Peter J.; Forbes, Simon; Jia, Mingming; Jones, David; Knott, Henry; Kok, Chai Yin; Lau, King Wai; Leroy, Catherine; Lin, Meng-Lay; McBride, David J.; Maddison, Mark; Maguire, Simon; McLay, Kirsten; Menzies, Andrew; Mironenko, Tatiana; Mulderrig, Lee; Mudie, Laura; O'Meara, Sarah; Pleasance, Erin; Rajasingham, Arjunan; Shepherd, Rebecca; Smith, Raffaella; Stebbings, Lucy; Stephens, Philip; Tang, Gurpreet; Tarpey, Patrick S.; Turrell, Kelly; Dykema, Karl J.; Khoo, Sok Kean; Petillo, David; Wondergem, Bill; Anema, John; Kahnoski, Richard J.; Teh, Bin Tean; Stratton, Michael R.; Futreal, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7279</span>),
    <span class="NLM_cas:pages">360-363</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma (ccRCC) is the most common form of adult kidney cancer, characterized by the presence of inactivating mutations in the VHL gene in most cases, and by infrequent somatic mutations in known cancer genes.  To det. further the genetics of ccRCC, we have sequenced 101 cases through 3544 protein-coding genes.  Here we report the identification of inactivating mutations in two genes encoding enzymes involved in histone modification-SETD2, a histone H3 lysine 36 methyltransferase, and JARID1C (also known as KDM5C), a histone H3 lysine 4 demethylase-as well as mutations in the histone H3 lysine 27 demethylase, UTX (KMD6A), that we recently reported.  The results highlight the role of mutations in components of the chromatin modification machinery in human cancer.  Furthermore, NF2 mutations were found in non-VHL mutated ccRCC, and several other probable cancer genes were identified.  These results indicate that substantial genetic heterogeneity exists in a cancer type dominated by mutations in a single gene, and that systematic screens will be key to fully detg. the somatic genetic architecture of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuHtA6CxWXLVg90H21EOLACvtfcHk0lhAhQbNg5LJew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFeqtw%253D%253D&md5=e08e7870bb8a78f1cf33b026d2d9c899</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1038%2Fnature08672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08672%26sid%3Dliteratum%253Aachs%26aulast%3DDalgliesh%26aufirst%3DG.%2BL.%26aulast%3DFurge%26aufirst%3DK.%26aulast%3DGreenman%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DEdkins%26aufirst%3DS.%26aulast%3DHardy%26aufirst%3DC.%26aulast%3DLatimer%26aufirst%3DC.%26aulast%3DTeague%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DJ.%26aulast%3DBarthorpe%26aufirst%3DS.%26aulast%3DBeare%26aufirst%3DD.%26aulast%3DBuck%26aufirst%3DG.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DForbes%26aufirst%3DS.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DKnott%26aufirst%3DH.%26aulast%3DKok%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DLeroy%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DM.%2BL.%26aulast%3DMcBride%26aufirst%3DD.%2BJ.%26aulast%3DMaddison%26aufirst%3DM.%26aulast%3DMaguire%26aufirst%3DS.%26aulast%3DMcLay%26aufirst%3DK.%26aulast%3DMenzies%26aufirst%3DA.%26aulast%3DMironenko%26aufirst%3DT.%26aulast%3DMulderrig%26aufirst%3DL.%26aulast%3DMudie%26aufirst%3DL.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DS.%26aulast%3DPleasance%26aufirst%3DE.%26aulast%3DRajasingham%26aufirst%3DA.%26aulast%3DShepherd%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DStebbings%26aufirst%3DL.%26aulast%3DStephens%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DG.%26aulast%3DTarpey%26aufirst%3DP.%2BS.%26aulast%3DTurrell%26aufirst%3DK.%26aulast%3DDykema%26aufirst%3DK.%2BJ.%26aulast%3DKhoo%26aufirst%3DS.%2BK.%26aulast%3DPetillo%26aufirst%3DD.%26aulast%3DWondergem%26aufirst%3DB.%26aulast%3DAnema%26aufirst%3DJ.%26aulast%3DKahnoski%26aufirst%3DR.%2BJ.%26aulast%3DTeh%26aufirst%3DB.%2BT.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26atitle%3DSystematic%2520sequencing%2520of%2520renal%2520carcinoma%2520reveals%2520inactivation%2520of%2520histone%2520modifying%2520genes%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D360%26epage%3D363%26doi%3D10.1038%2Fnature08672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinohara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantoff, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, C. J.</span></span> <span> </span><span class="NLM_article-title">Resistance to docetaxel in prostate cancer is associated with androgen receptor activation and loss of KDM5D expression</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">6259</span>â <span class="NLM_lpage">6264</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1600420113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1073%2Fpnas.1600420113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27185910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFeitrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=6259-6264&author=K.+Komuraauthor=S.+H.+Jeongauthor=K.+Hinoharaauthor=F.+Quauthor=X.+Wangauthor=M.+Hirakiauthor=H.+Azumaauthor=G.+S.+Leeauthor=P.+W.+Kantoffauthor=C.+J.+Sweeney&title=Resistance+to+docetaxel+in+prostate+cancer+is+associated+with+androgen+receptor+activation+and+loss+of+KDM5D+expression&doi=10.1073%2Fpnas.1600420113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to docetaxel in prostate cancer is associated with androgen receptor activation and loss of KDM5D expression</span></div><div class="casAuthors">Komura, Kazumasa; Jeong, Seong Ho; Hinohara, Kunihiko; Qu, Fangfang; Wang, Xiaodong; Hiraki, Masayuki; Azuma, Haruhito; Lee, Gwo-Shu Mary; Kantoff, Philip W.; Sweeney, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6259-6264</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The androgen receptor (AR) plays an essential role in prostate cancer, and suppression of its signaling with androgen deprivation therapy (ADT) has been the mainstay of treatment for metastatic hormone-sensitive prostate cancer for more than 70 y.  Chemotherapy has been reserved for metastatic castration-resistant prostate cancer (mCRPC).  The Eastern Cooperative Oncol. Group-led trial E3805: ChemoHormonal Therapy Vs. Androgen Ablation Randomized Trial for Extensive Disease in Prostate Cancer (CHAARTED) showed that the addn. of docetaxel to ADT prolonged overall survival compared with ADT alone in patients with metastatic hormone-sensitive prostate cancer.  This finding suggests that there is an interaction between AR signaling activity and docetaxel sensitivity.  Here we demonstrate that the prostate cancer cell lines LNCaP and LAPC4 display markedly different sensitivity to docetaxel with AR activation, and RNA-seq anal. of these cell lines identified KDM5D (lysine-specific demethylase 5D) encoded on the Y chromosome as a potential mediator of this sensitivity.  Knocking down KDM5D expression in LNCaP leads to docetaxel resistance in the presence of dihydrotestosterone.  KDM5D phys. interacts with AR in the nucleus, and regulates its transcriptional activity by demethylating H3K4me3 active transcriptional marks.  Attenuating KDM5D expression dysregulates AR signaling, resulting in docetaxel insensitivity.  KDM5D deletion was also obsd. in the LNCaP-derived CRPC cell line 104R2, which displayed docetaxel insensitivity with AR activation, unlike parental LNCaP.  Dataset anal. from the Oncomine database revealed significantly decreased KDM5D expression in CRPC and poorer prognosis with low KDM5D expression.  Taking these data together, this work indicates that KDM5D modulates the AR axis and that this is assocd. with altered docetaxel sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLVeWTWOWNyrVg90H21EOLACvtfcHk0lhAhQbNg5LJew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFeitrw%253D&md5=9ca5ed64490276833a0f9aeb6a276d3f</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1600420113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1600420113%26sid%3Dliteratum%253Aachs%26aulast%3DKomura%26aufirst%3DK.%26aulast%3DJeong%26aufirst%3DS.%2BH.%26aulast%3DHinohara%26aufirst%3DK.%26aulast%3DQu%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHiraki%26aufirst%3DM.%26aulast%3DAzuma%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DG.%2BS.%26aulast%3DKantoff%26aufirst%3DP.%2BW.%26aulast%3DSweeney%26aufirst%3DC.%2BJ.%26atitle%3DResistance%2520to%2520docetaxel%2520in%2520prostate%2520cancer%2520is%2520associated%2520with%2520androgen%2520receptor%2520activation%2520and%2520loss%2520of%2520KDM5D%2520expression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D6259%26epage%3D6264%26doi%3D10.1073%2Fpnas.1600420113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harmeyer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facompre, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, D.</span></span> <span> </span><span class="NLM_article-title">JARID1 histone demethylases: Emerging targets in cancer</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">713</span>â <span class="NLM_lpage">725</span>, <span class="refDoi">Â DOI: 10.1016/j.trecan.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.trecan.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28958389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=713-725&author=K.+M.+Harmeyerauthor=N.+D.+Facompreauthor=M.+Herlynauthor=D.+Basu&title=JARID1+histone+demethylases%3A+Emerging+targets+in+cancer&doi=10.1016%2Fj.trecan.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">JARID1 Histone Demethylases: Emerging Targets in Cancer</span></div><div class="casAuthors">Harmeyer, Kayla M.; Facompre, Nicole D.; Herlyn, Meenhard; Basu, Devraj</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">713-725</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  JARID1 proteins are histone demethylases that both regulate normal cell fates during development and contribute to the epigenetic plasticity that underlies malignant transformation.  This H3K4 demethylase family participates in multiple repressive transcriptional complexes at promoters and has broader regulatory effects on chromatin that remain ill-defined.  There is growing understanding of the oncogenic and tumor suppressive functions of JARID1 proteins, which are contingent on cell context and the protein isoform.  Their contributions to stem cell-like dedifferentiation, tumor aggressiveness, and therapy resistance in cancer have sustained interest in the development of JARID1 inhibitors.  Here we review the diverse and context-specific functions of the JARID1 proteins that may impact the utilization of emerging targeted inhibitors of this histone demethylase family in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYLbdIKcjybVg90H21EOLACvtfcHk0lhAhQbNg5LJew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D&md5=e728e47a8d60d5e83f29ab6d153a2be4</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DHarmeyer%26aufirst%3DK.%2BM.%26aulast%3DFacompre%26aufirst%3DN.%2BD.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBasu%26aufirst%3DD.%26atitle%3DJARID1%2520histone%2520demethylases%253A%2520Emerging%2520targets%2520in%2520cancer%26jtitle%3DTrends%2520Cancer%26date%3D2017%26volume%3D3%26spage%3D713%26epage%3D725%26doi%3D10.1016%2Fj.trecan.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Lysine demethylase 5B (KDM5B): A potential anti-cancer drug target</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">131</span>â <span class="NLM_lpage">140</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2018.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30343192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvF2mu7fL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2019&pages=131-140&author=Y.+C.+Zhengauthor=J.+Changauthor=L.+C.+Wangauthor=H.+M.+Renauthor=J.+R.+Pangauthor=H.+M.+Liu&title=Lysine+demethylase+5B+%28KDM5B%29%3A+A+potential+anti-cancer+drug+target&doi=10.1016%2Fj.ejmech.2018.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine demethylase 5B (KDM5B): A potential anti-cancer drug target</span></div><div class="casAuthors">Zheng, Yi-Chao; Chang, Jiao; Wang, Lin-Chen; Ren, Hong-Mei; Pang, Jing-Ru; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">131-140</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine demethylase 5B (KDM5B) is a histone demethylase identified in 2007, which is responsible for erasing H3K4me2/3 activation marker.  It participates in multiple repressive transcriptional complexes around target gene promoters and performs wide regulatory effects on chromatin structure.  Until now, there is growing evidence for the oncogenic function of KDM5B.  As the H3K4me2/3 residue represents the transcription initiation site of the active transcription gene, and demethylation of H3K4 is assocd. with transcriptional repression, making it a potential participant in inhibiting the expression of tumor suppressors.  Therefore, KDM5B is considered as a promising drug target for cancer therapy, and many medicinal chemists are trying to design and synthesize potent and selective KDM5B inhibitors with the aid of high-throughput screening, structure based drug design, and structure activity relationship studies.  This review focuses on the basic biochem. and physiol. function of KDM5B and its involved mechanisms in cancers, a comprehensive overview of KDM5B inhibitors is also introduced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL9Qm25PJ8e7Vg90H21EOLACvtfcHk0liaBsWoUgESMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvF2mu7fL&md5=881eb589a48c5b00c67ae58a67c0d0f5</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%2BC.%26aulast%3DRen%26aufirst%3DH.%2BM.%26aulast%3DPang%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DLysine%2520demethylase%25205B%2520%2528KDM5B%2529%253A%2520A%2520potential%2520anti-cancer%2520drug%2520target%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D161%26spage%3D131%26epage%3D140%26doi%3D10.1016%2Fj.ejmech.2018.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tindell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guler, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">531</span>â <span class="NLM_lpage">538</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1038%2Fnchembio.2085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27214401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=531-538&author=M.+Vinogradovaauthor=V.+S.+Gehlingauthor=A.+Gustafsonauthor=S.+Aroraauthor=C.+A.+Tindellauthor=C.+Wilsonauthor=K.+E.+Williamsonauthor=G.+D.+Gulerauthor=P.+Gangurdeauthor=W.+Manieriauthor=J.+Busbyauthor=E.+M.+Flynnauthor=F.+Lanauthor=H.+J.+Kimauthor=S.+Odateauthor=A.+G.+Cochranauthor=Y.+Liuauthor=M.+Wongchenkoauthor=Y.+Yangauthor=T.+K.+Cheungauthor=T.+M.+Maileauthor=T.+Lauauthor=M.+Costaauthor=G.+V.+Hegdeauthor=E.+Jacksonauthor=R.+Pittiauthor=D.+Arnottauthor=C.+Baileyauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=J.+R.+Kieferauthor=P.+Trojerauthor=M.+Classon&title=An+inhibitor+of+KDM5+demethylases+reduces+survival+of+drug-tolerant+cancer+cells&doi=10.1038%2Fnchembio.2085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span></div><div class="casAuthors">Vinogradova, Maia; Gehling, Victor S.; Gustafson, Amy; Arora, Shilpi; Tindell, Charles A.; Wilson, Catherine; Williamson, Kaylyn E.; Guler, Gulfem D.; Gangurde, Pranoti; Manieri, Wanda; Busby, Jennifer; Flynn, E. Megan; Lan, Fei; Kim, Hyo-jin; Odate, Shobu; Cochran, Andrea G.; Liu, Yichin; Wongchenko, Matthew; Yang, Yibin; Cheung, Tommy K.; Maile, Tobias M.; Lau, Ted; Costa, Michael; Hegde, Ganapati V.; Jackson, Erica; Pitti, Robert; Arnott, David; Bailey, Christopher; Bellon, Steve; Cummings, Richard T.; Albrecht, Brian K.; Harmange, Jean-Christophe; Kiefer, James R.; Trojer, Patrick; Classon, Marie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KDM5 family of histone demethylases catalyzes the demethylation of histone H3 on lysine 4 (H3K4) and is required for the survival of drug-tolerant persister cancer cells (DTPs).  Here the authors report the discovery and characterization of the specific KDM5 inhibitor CPI-455.  The crystal structure of KDM5A revealed the mechanism of inhibition of CPI-455 as well as the topol. arrangements of protein domains that influence substrate binding.  CPI-455 mediated KDM5 inhibition, elevated global levels of H3K4 trimethylation (H3K4me3) and decreased the no. of DTPs in multiple cancer cell line models treated with std. chemotherapy or targeted agents.  These findings show that pretreatment of cancer cells with a KDM5-specific inhibitor results in the ablation of a subpopulation of cancer cells that can serve as the founders for therapeutic relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxyW7L2YsV1rVg90H21EOLACvtfcHk0liaBsWoUgESMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D&md5=41699f5c17c912b39cdda81f4dfaa31e</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2085%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DTindell%26aufirst%3DC.%2BA.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DGuler%26aufirst%3DG.%2BD.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWongchenko%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCheung%26aufirst%3DT.%2BK.%26aulast%3DMaile%26aufirst%3DT.%2BM.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DCosta%26aufirst%3DM.%26aulast%3DHegde%26aufirst%3DG.%2BV.%26aulast%3DJackson%26aufirst%3DE.%26aulast%3DPitti%26aufirst%3DR.%26aulast%3DArnott%26aufirst%3DD.%26aulast%3DBailey%26aufirst%3DC.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DClasson%26aufirst%3DM.%26atitle%3DAn%2520inhibitor%2520of%2520KDM5%2520demethylases%2520reduces%2520survival%2520of%2520drug-tolerant%2520cancer%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fnchembio.2085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Porten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span> <span> </span><span class="NLM_article-title">Lead optimization of a pyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one scaffold to identify potent, selective and orally bioavailable KDM5 inhibitors suitable for in vivo biological studies</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4036</span>â <span class="NLM_lpage">4041</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.06.078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.bmcl.2016.06.078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=27406798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCrtrfO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4036-4041&author=J.+Liangauthor=B.+Zhangauthor=S.+Labadieauthor=D.+F.+Ortwineauthor=M.+Vinogradovaauthor=J.+R.+Kieferauthor=V.+S.+Gehlingauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=T.+Laiauthor=J.+Liaoauthor=X.+Zhengauthor=Y.+Liuauthor=A.+Gustafsonauthor=E.+Van+der+Portenauthor=W.+Maoauthor=B.+M.+Liedererauthor=G.+Deshmukhauthor=M.+Classonauthor=P.+Trojerauthor=P.+S.+Dragovichauthor=L.+Murray&title=Lead+optimization+of+a+pyrazolo%5B1%2C5-a%5Dpyrimidin-7%284H%29-one+scaffold+to+identify+potent%2C+selective+and+orally+bioavailable+KDM5+inhibitors+suitable+for+in+vivo+biological+studies&doi=10.1016%2Fj.bmcl.2016.06.078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Lead optimization of a pyrazolo[1,5-a]pyrimidin-7(4H)-one scaffold to identify potent, selective and orally bioavailable KDM5 inhibitors suitable for in vivo biological studies</span></div><div class="casAuthors">Liang, Jun; Zhang, Birong; Labadie, Sharada; Ortwine, Daniel F.; Vinogradova, Maia; Kiefer, James R.; Gehling, Victor S.; Harmange, Jean-Christophe; Cummings, Richard; Lai, Tommy; Liao, Jiangpeng; Zheng, Xiaoping; Liu, Yichin; Gustafson, Amy; Van der Porten, Erica; Mao, Weifeng; Liederer, Bianca M.; Deshmukh, Gauri; Classon, Marie; Trojer, Patrick; Dragovich, Peter S.; Murray, Lesley</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4036-4041</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Starting with a lead [1,5-a]pyrimidin-7(4H)-one-contg. mol., the authors generated potent, selective and orally bioavailable KDM5 inhibitors.  Using structure- and property-based approaches, the authors designed I with improved cell potency (PC9 H3K4Me3 EC50 = 0.34 Î¼M).  Furthermore, I maintained suitable physiochem. properties and displayed an excellent pharmacokinetic (PK) profile in mice.  When dosed orally in mice at 50 mg/kg twice a day (BID), I showed an unbound maximal plasma concn. (Cmax) >15-fold over its cell EC50, thereby providing a robust chem. probe for studying KDM5 biol. functions in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4w4DV2WzWGLVg90H21EOLACvtfcHk0ljOzhd0DdnbOg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCrtrfO&md5=64cde81ab69441e83148851121df73a5</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.06.078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.06.078%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DVan%2Bder%2BPorten%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DMurray%26aufirst%3DL.%26atitle%3DLead%2520optimization%2520of%2520a%2520pyrazolo%255B1%252C5-a%255Dpyrimidin-7%25284H%2529-one%2520scaffold%2520to%2520identify%2520potent%252C%2520selective%2520and%2520orally%2520bioavailable%2520KDM5%2520inhibitors%2520suitable%2520for%2520in%2520vivo%2520biological%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4036%26epage%3D4041%26doi%3D10.1016%2Fj.bmcl.2016.06.078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanigan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atrash, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bihan, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dempster, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxall, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sejberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scozzafava, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span> <span> </span><span class="NLM_article-title">8-substituted pyrido[3,4-<i>d</i>]pyrimidin-4(3<i>H</i>)-one derivatives as potent, cell permeable, KDM4 (JMJD2) and KDM5 (JARID1) histone lysine demethylase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1388</span>â <span class="NLM_lpage">1409</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGmsrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1388-1409&author=V.+Bavetsiasauthor=R.+M.+Laniganauthor=G.+F.+Rudaauthor=B.+Atrashauthor=M.+G.+McLaughlinauthor=A.+Tumberauthor=N.+Y.+Mokauthor=Y.+V.+Le+Bihanauthor=S.+Dempsterauthor=K.+J.+Boxallauthor=F.+Jeganathanauthor=S.+B.+Hatchauthor=P.+Savitskyauthor=S.+Velupillaiauthor=T.+Krojerauthor=K.+S.+Englandauthor=J.+Sejbergauthor=C.+Thaiauthor=A.+Donovanauthor=A.+Palauthor=G.+Scozzafavaauthor=J.+M.+Bennettauthor=A.+Kawamuraauthor=C.+Johanssonauthor=A.+Szykowskaauthor=C.+Gileadiauthor=N.+A.+Burgess-Brownauthor=F.+von+Delftauthor=U.+Oppermannauthor=Z.+Waltersauthor=J.+Shipleyauthor=F.+I.+Raynaudauthor=S.+M.+Westawayauthor=R.+K.+Prinjhaauthor=O.+Fedorovauthor=R.+Burkeauthor=C.+J.+Schofieldauthor=I.+M.+Westwoodauthor=C.+Bountraauthor=S.+M%C3%BCllerauthor=R.+L.+van+Montfortauthor=P.+E.+Brennanauthor=J.+Blagg&title=8-substituted+pyrido%5B3%2C4-d%5Dpyrimidin-4%283H%29-one+derivatives+as+potent%2C+cell+permeable%2C+KDM4+%28JMJD2%29+and+KDM5+%28JARID1%29+histone+lysine+demethylase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">8-Substituted Pyrido[3,4-d]pyrimidin-4(3H)-one Derivatives As Potent, Cell Permeable, KDM4 (JMJD2) and KDM5 (JARID1) Histone Lysine Demethylase Inhibitors</span></div><div class="casAuthors">Bavetsias, Vassilios; Lanigan, Rachel M.; Ruda, Gian Filippo; Atrash, Butrus; McLaughlin, Mark G.; Tumber, Anthony; Mok, N. Yi; Le Bihan, Yann-Vai; Dempster, Sally; Boxall, Katherine J.; Jeganathan, Fiona; Hatch, Stephanie B.; Savitsky, Pavel; Velupillai, Srikannathasan; Krojer, Tobias; England, Katherine S.; Sejberg, Jimmy; Thai, Ching; Donovan, Adam; Pal, Akos; Scozzafava, Giuseppe; Bennett, James M.; Kawamura, Akane; Johansson, Catrine; Szykowska, Aleksandra; Gileadi, Carina; Burgess-Brown, Nicola A.; von Delft, Frank; Oppermann, Udo; Walters, Zoe; Shipley, Janet; Raynaud, Florence I.; Westaway, Susan M.; Prinjha, Rab K.; Fedorov, Oleg; Burke, Rosemary; Schofield, Christopher J.; Westwood, Isaac M.; Bountra, Chas; Muller, Susanne; van Montfort, Rob L. M.; Brennan, Paul E.; Blagg, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1388-1409</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the discovery of N-substituted 4-(pyridin-2-yl)thiazole-2-amine derivs. and their subsequent optimization, guided by structure-based design, to give 8-(1H-pyrazol-3-yl)pyrido[3,4-d]pyrimidin-4(3H)-ones, a series of potent JmjC histone N-Me lysine demethylase (KDM) inhibitors which bind to Fe(II) in the active site.  Substitution from C4 of the pyrazole moiety allows access to the histone peptide substrate binding site; incorporation of a conformationally constrained 4-phenylpiperidine linker gives derivs. such as 54j and 54k which demonstrate equipotent activity vs. the KDM4 (JMJD2) and KDM5 (JARID1) subfamily demethylases, selectivity over representative exemplars of the KDM2, KDM3, and KDM6 subfamilies, cellular permeability in the Caco-2 assay, and, for 54k, inhibition of H3K9Me3 and H3K4Me3 demethylation in a cell-based assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgIlVNGDiOQbVg90H21EOLACvtfcHk0ljOzhd0DdnbOg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGmsrY%253D&md5=a7ff8a630f2119f9d7be5a8348cb56c3</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01635%26sid%3Dliteratum%253Aachs%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DLanigan%26aufirst%3DR.%2BM.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DMcLaughlin%26aufirst%3DM.%2BG.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DMok%26aufirst%3DN.%2BY.%26aulast%3DLe%2BBihan%26aufirst%3DY.%2BV.%26aulast%3DDempster%26aufirst%3DS.%26aulast%3DBoxall%26aufirst%3DK.%2BJ.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DSejberg%26aufirst%3DJ.%26aulast%3DThai%26aufirst%3DC.%26aulast%3DDonovan%26aufirst%3DA.%26aulast%3DPal%26aufirst%3DA.%26aulast%3DScozzafava%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DWalters%26aufirst%3DZ.%26aulast%3DShipley%26aufirst%3DJ.%26aulast%3DRaynaud%26aufirst%3DF.%2BI.%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DWestwood%26aufirst%3DI.%2BM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBlagg%26aufirst%3DJ.%26atitle%3D8-substituted%2520pyrido%255B3%252C4-d%255Dpyrimidin-4%25283H%2529-one%2520derivatives%2520as%2520potent%252C%2520cell%2520permeable%252C%2520KDM4%2520%2528JMJD2%2529%2520and%2520KDM5%2520%2528JARID1%2529%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1388%26epage%3D1409%26doi%3D10.1021%2Facs.jmedchem.5b01635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Insights into the action of inhibitor enantiomers against histone lysine demethylase 5A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3193</span>â <span class="NLM_lpage">3208</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00261</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00261" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3193-3208&author=J.+R.+Hortonauthor=X.+Liuauthor=L.+Wuauthor=K.+Zhangauthor=J.+Shanksauthor=X.+Zhangauthor=G.+Raiauthor=B.+T.+Mottauthor=D.+J.+Jansenauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=K.+Pohidaauthor=Y.+Fangauthor=X.+Huauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=M.+D.+Hallauthor=A.+Simeonovauthor=H.+Fuauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Insights+into+the+action+of+inhibitor+enantiomers+against+histone+lysine+demethylase+5A&doi=10.1021%2Facs.jmedchem.8b00261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Wu, Lizhen; Zhang, Kai; Shanks, John; Zhang, Xing; Rai, Ganesha; Mott, Bryan T.; Jansen, Daniel J.; Kales, Stephen C.; Henderson, Mark J.; Pohida, Katherine; Fang, Yuhong; Hu, Xin; Jadhav, Ajit; Maloney, David J.; Hall, Matthew D.; Simeonov, Anton; Fu, Haian; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3193-3208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isomers of chiral drugs can exhibit marked differences in biol. activities.  We studied the binding and inhibitory activities of 12 compds. against KDM5A.  Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (R)- and (S)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1H-pyrrolo[3,2-b]pyridine-7-carboxylic acid (compds. N51 and N52) and (R)- and (S)-N-(1-(3-isopropyl-1H-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compds. N54 and N55).  In vitro, the S enantiomer of the N51/N52 pair (N52) and the R enantiomer of the N54/N55 pair (N54) exhibited about 4- to 5-fold greater binding affinity.  The more potent enzyme inhibition of KDM5A by the R-isoform for the cell-permeable N54/N55 pair translated to differences in growth inhibitory activity.  We detd. structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity.  These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovFfXyPcUszbVg90H21EOLACvtfcHk0ljl3VX2n2Xp7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D&md5=13ecac6902afd69d4ce7ad05803ca4d9</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00261%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DInsights%2520into%2520the%2520action%2520of%2520inhibitor%2520enantiomers%2520against%2520histone%2520lysine%2520demethylase%25205A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3193%26epage%3D3208%26doi%3D10.1021%2Facs.jmedchem.8b00261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective KDM5 inhibitor stops cellular demethylation of H3K4me3 at transcription start sites and proliferation of MM1S myeloma cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">371</span>â <span class="NLM_lpage">380</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.chembiol.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=28262558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=371-380&author=A.+Tumberauthor=A.+Nuzziauthor=E.+S.+Hookwayauthor=S.+B.+Hatchauthor=S.+Velupillaiauthor=C.+Johanssonauthor=A.+Kawamuraauthor=P.+Savitskyauthor=C.+Yappauthor=A.+Szykowskaauthor=N.+Wuauthor=C.+Bountraauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=G.+F.+Rudaauthor=O.+Fedorovauthor=S.+Munroauthor=K.+S.+Englandauthor=R.+P.+Nowakauthor=C.+J.+Schofieldauthor=N.+B.+La+Thangueauthor=C.+Pawlynauthor=F.+Daviesauthor=G.+Morganauthor=N.+Athanasouauthor=S.+M%C3%BCllerauthor=U.+Oppermannauthor=P.+E.+Brennan&title=Potent+and+selective+KDM5+inhibitor+stops+cellular+demethylation+of+H3K4me3+at+transcription+start+sites+and+proliferation+of+MM1S+myeloma+cells&doi=10.1016%2Fj.chembiol.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective KDM5 Inhibitor Stops Cellular Demethylation of H3K4me3 at Transcription Start Sites and Proliferation of MM1S Myeloma Cells</span></div><div class="casAuthors">Tumber, Anthony; Nuzzi, Andrea; Hookway, Edward S.; Hatch, Stephanie B.; Velupillai, Srikannathasan; Johansson, Catrine; Kawamura, Akane; Savitsky, Pavel; Yapp, Clarence; Szykowska, Aleksandra; Wu, Na; Bountra, Chas; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Ruda, Gian Filippo; Fedorov, Oleg; Munro, Shonagh; England, Katherine S.; Nowak, Radoslaw P.; Schofield, Christopher J.; La Thangue, Nicholas B.; Pawlyn, Charlotte; Davies, Faith; Morgan, Gareth; Athanasou, Nick; Muller, Susanne; Oppermann, Udo; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-380</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of lysine residues on histone tail is a dynamic epigenetic modification that plays a key role in chromatin structure and gene regulation.  Members of the KDM5 (also known as JARID1) sub-family are 2-oxoglutarate (2-OG) and Fe2+-dependent oxygenases acting as histone 3 lysine 4 tri-Me (H3K4me3) demethylases, regulating proliferation, stem cell self-renewal, and differentiation.  Here we present the characterization of KDOAM-25, an inhibitor of KDM5 enzymes.  KDOAM-25 shows biochem. half maximal inhibitory concn. values of <100 nM for KDM5A-D in vitro, high selectivity toward other 2-OG oxygenases sub-families, and no off-target activity on a panel of 55 receptors and enzymes.  In human cell assay systems, KDOAM-25 has a half maximal effective concn. of â¼50 Î¼M and good selectivity toward other demethylases.  KDM5B is overexpressed in multiple myeloma and neg. correlated with the overall survival.  Multiple myeloma MM1S cells treated with KDOAM-25 show increased global H3K4 methylation at transcriptional start sites and impaired proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UzNQI-MiYbVg90H21EOLACvtfcHk0ljl3VX2n2Xp7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D&md5=0fc70179bda8b4fd38db1054c7bb3277</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DPawlyn%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DF.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DPotent%2520and%2520selective%2520KDM5%2520inhibitor%2520stops%2520cellular%2520demethylation%2520of%2520H3K4me3%2520at%2520transcription%2520start%2520sites%2520and%2520proliferation%2520of%2520MM1S%2520myeloma%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D371%26epage%3D380%26doi%3D10.1016%2Fj.chembiol.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Structure-based engineering of irreversible inhibitors against histone lysine demethylase KDM5A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10588</span>â <span class="NLM_lpage">10601</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01219</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01219" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2rur3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10588-10601&author=J.+R.+Hortonauthor=C.+B.+Woodcockauthor=Q.+Chenauthor=X.+Liuauthor=X.+Zhangauthor=J.+Shanksauthor=G.+Raiauthor=B.+T.+Mottauthor=D.+J.+Jansenauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=M.+Cyrauthor=K.+Pohidaauthor=X.+Huauthor=P.+Shahauthor=X.+Xuauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=M.+D.+Hallauthor=A.+Simeonovauthor=H.+Fuauthor=P.+M.+Vertinoauthor=X.+Cheng&title=Structure-based+engineering+of+irreversible+inhibitors+against+histone+lysine+demethylase+KDM5A&doi=10.1021%2Facs.jmedchem.8b01219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A</span></div><div class="casAuthors">Horton, John R.; Woodcock, Clayton B.; Chen, Qin; Liu, Xu; Zhang, Xing; Shanks, John; Rai, Ganesha; Mott, Bryan T.; Jansen, Daniel J.; Kales, Stephen C.; Henderson, Mark J.; Cyr, Matthew; Pohida, Katherine; Hu, Xin; Shah, Pranav; Xu, Xin; Jadhav, Ajit; Maloney, David J.; Hall, Matthew D.; Simeonov, Anton; Fu, Haian; Vertino, Paula M.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10588-10601</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The active sites of hundreds of human Î±-ketoglutarate (Î±KG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors.  We identified a noncatalytic cysteine (Cys481 in KDM5A) near the active sites of KDM5 histone H3 lysine 4 demethylases, which is absent in other histone demethylase families, that could be explored for interaction with the cysteine-reactive electrophile acrylamide.  We synthesized analogs of a thienopyridine-based inhibitor chemotype, namely, 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-b]pyridine-7-carboxylic acid (N70) and a deriv. contg. a ((dimethylamino)but-2-enamido)phenyl moiety (N71) designed to form a covalent interaction with Cys481.  We characterized the inhibitory and binding activities against KDM5A and detd. the cocrystal structures of the catalytic domain of KDM5A in complex with N70 and N71.  Whereas the noncovalent inhibitor N70 displayed Î±KG-competitive inhibition that could be reversed after dialysis, inhibition by N71 was dependent on enzyme concn. and persisted even after dialysis, consistent with covalent modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo58lpkd4vS1LVg90H21EOLACvtfcHk0ljl3VX2n2Xp7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2rur3P&md5=5ab97cae54ccff28f65071fa41a70c0a</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01219%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DWoodcock%26aufirst%3DC.%2BB.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DCyr%26aufirst%3DM.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructure-based%2520engineering%2520of%2520irreversible%2520inhibitors%2520against%2520histone%2520lysine%2520demethylase%2520KDM5A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10588%26epage%3D10601%26doi%3D10.1021%2Facs.jmedchem.8b01219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez-Rodriguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cribbs, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunford, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thezenas, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and characterization of covalent KDM5 inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">515</span>â <span class="NLM_lpage">519</span>, <span class="refDoi">Â DOI: 10.1002/anie.201810179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1002%2Fanie.201810179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVemtL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=515-519&author=S.+Vazquez-Rodriguezauthor=M.+Wrightauthor=C.+M.+Rogersauthor=A.+P.+Cribbsauthor=S.+Velupillaiauthor=M.+Philpottauthor=H.+Leeauthor=J.+E.+Dunfordauthor=K.+V.+M.+Huberauthor=M.+B.+Robersauthor=J.+D.+Vastaauthor=M.+L.+Thezenasauthor=S.+Bonhamauthor=B.+Kesslerauthor=J.+Bennettauthor=O.+Fedorovauthor=F.+Raynaudauthor=A.+Donovanauthor=J.+Blaggauthor=V.+Bavetsiasauthor=U.+Oppermannauthor=C.+Bountraauthor=A.+Kawamuraauthor=P.+E.+Brennan&title=Design%2C+synthesis+and+characterization+of+covalent+KDM5+inhibitors&doi=10.1002%2Fanie.201810179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis and Characterization of Covalent KDM5 Inhibitors</span></div><div class="casAuthors">Vazquez-Rodriguez, Saleta; Wright, Miranda; Rogers, Catherine M.; Cribbs, Adam P.; Velupillai, Srikannathasan; Philpott, Martin; Lee, Henry; Dunford, James E.; Huber, Kilian V. M.; Robers, Matthew B.; Vasta, James D.; Thezenas, Marie-Laetitia; Bonham, Sarah; Kessler, Benedikt; Bennett, James; Fedorov, Oleg; Raynaud, Florence; Donovan, Adam; Blagg, Julian; Bavetsias, Vassilios; Oppermann, Udo; Bountra, Chas; Kawamura, Akane; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">515-519</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Histone lysine demethylases (KDMs) are involved in the dynamic regulation of gene expression and they play a crit. role in several biol. processes.  Achieving selectivity over the different KDMs has been a major challenge for KDM inhibitor development.  Here we report potent and selective KDM5 covalent inhibitors designed to target cysteine residues only present in the KDM5 sub-family.  The covalent binding to the targeted proteins was confirmed by MS and time-dependent inhibition.  Addnl. competition assays show that compds. were non 2-OG competitive.  Target engagement and ChIP-seq anal. showed that the compds. inhibited the KDM5 members in cells at nano- to micromolar levels and induce a global increase of the H3K4me3 mark at transcriptional start sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnPw38R0sKE7Vg90H21EOLACvtfcHk0lhKwigJ3cBMTg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVemtL%252FI&md5=3b9f340a3eed3d5dad7aed79e0d7d931</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1002%2Fanie.201810179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201810179%26sid%3Dliteratum%253Aachs%26aulast%3DVazquez-Rodriguez%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DCribbs%26aufirst%3DA.%2BP.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DDunford%26aufirst%3DJ.%2BE.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DThezenas%26aufirst%3DM.%2BL.%26aulast%3DBonham%26aufirst%3DS.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DDonovan%26aufirst%3DA.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%252C%2520synthesis%2520and%2520characterization%2520of%2520covalent%2520KDM5%2520inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D515%26epage%3D519%26doi%3D10.1002%2Fanie.201810179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref173"><div class="reference"><strong class="refLabel"><a href="#ref173" class="rightTabRefNumLink">173</a></strong><div class="NLM_citation" id="rightTab-cit173"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interaction with covalent small-molecule inhibitors</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1872</span>â <span class="NLM_lpage">1876</span>, <span class="refDoi">Â DOI: 10.2174/1568026619666191011163410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.2174%2F1568026619666191011163410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=31696799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVymtLvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=1872-1876&author=B.+Liauthor=D.+Rongauthor=Y.+Wang&title=Targeting+protein-protein+interaction+with+covalent+small-molecule+inhibitors&doi=10.2174%2F1568026619666191011163410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Protein-Protein Interaction with Covalent Small-Molecule Inhibitors</span></div><div class="casAuthors">Li, Bingbing; Rong, Deqin; Wang, Yuanxiang</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">1872-1876</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  PPIs are involved in diverse biochem. events and perform their functions through the formation of protein-protein complexes or PPI networks.  The large and flat interacting surfaces of PPIs make discovery of small-mol. modulators a challenging task.  New strategies and more effective chem. technologies are needed to facilitate the development of PPIs small-mol. inhibitors.  Covalent modification of a nucleophilic residue located proximally to the immediate vicinity of PPIs can overcome the disadvantages of large interacting surfaces and provides high-affinity inhibitors with increased duration of action and prolonged target modulation.  On the other hand, covalent inhibitors that target non-conserved protein residues demonstrate improved selectivity over related protein family members.  Herein, we highlight the latest progress of small-mol. covalent PPIs inhibitors and hope to shed light on future PPIs inhibitor design and development.  The relevant challenges and opportunities are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox4P1EKJ7ZrLVg90H21EOLACvtfcHk0lhKwigJ3cBMTg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVymtLvO&md5=5be818e234b25f3577c9b7f593868f6f</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.2174%2F1568026619666191011163410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026619666191011163410%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DRong%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DTargeting%2520protein-protein%2520interaction%2520with%2520covalent%2520small-molecule%2520inhibitors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2019%26volume%3D19%26spage%3D1872%26epage%3D1876%26doi%3D10.2174%2F1568026619666191011163410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref174"><div class="reference"><strong class="refLabel"><a href="#ref174" class="rightTabRefNumLink">174</a></strong><div class="NLM_citation" id="rightTab-cit174"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Progress with covalent small-molecule kinase inhibitors</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">727</span>â <span class="NLM_lpage">735</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2018.01.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.drudis.2018.01.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=29337202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=727-735&author=Z.+Zhaoauthor=P.+E.+Bourne&title=Progress+with+covalent+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.drudis.2018.01.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Progress with covalent small-molecule kinase inhibitors</span></div><div class="casAuthors">Zhao, Zheng; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">With reduced risk of toxicity and high selectivity, covalent small-mol. kinase inhibitors (CSKIs) have emerged rapidly.  Through the lens of structural system pharmacol., here we review this rapid progress by considering design strategies and the challenges and opportunities offered by current CSKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog6l6qMAopYLVg90H21EOLACvtfcHk0lil73AQLCmkSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslShurg%253D&md5=f97a19f53a72044812a778ee9351fe94</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProgress%2520with%2520covalent%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D727%26epage%3D735%26doi%3D10.1016%2Fj.drudis.2018.01.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref175"><div class="reference"><strong class="refLabel"><a href="#ref175" class="rightTabRefNumLink">175</a></strong><div class="NLM_citation" id="rightTab-cit175"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asgian, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekici, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, K. E.</span></span> <span> </span><span class="NLM_article-title">Irreversible inhibitors of serine, cysteine, and threonine proteases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4639</span>â <span class="NLM_lpage">4750</span>, <span class="refDoi">Â DOI: 10.1021/cr010182v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr010182v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4639-4750&author=J.+C.+Powersauthor=J.+L.+Asgianauthor=O.+D.+Ekiciauthor=K.+E.+James&title=Irreversible+inhibitors+of+serine%2C+cysteine%2C+and+threonine+proteases&doi=10.1021%2Fcr010182v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible inhibitors of serine, cysteine, and threonine proteases</span></div><div class="casAuthors">Powers, James C.; Asgian, Juliana L.; Ekici, Oezlem Dogan; James, Karen Ellis</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4639-4750</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Topics discussed include alkylating, acylating, phosphonylating, and sulfonylating agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorivaxGhmhirVg90H21EOLACvtfcHk0lil73AQLCmkSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D&md5=eb21ff0297b87537a7d459c92f13c75d</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1021%2Fcr010182v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr010182v%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26aulast%3DAsgian%26aufirst%3DJ.%2BL.%26aulast%3DEkici%26aufirst%3DO.%2BD.%26aulast%3DJames%26aufirst%3DK.%2BE.%26atitle%3DIrreversible%2520inhibitors%2520of%2520serine%252C%2520cysteine%252C%2520and%2520threonine%2520proteases%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4639%26epage%3D4750%26doi%3D10.1021%2Fcr010182v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref176"><div class="reference"><strong class="refLabel"><a href="#ref176" class="rightTabRefNumLink">176</a></strong><div class="NLM_citation" id="rightTab-cit176"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gambini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udompholkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jossart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salem, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellecchia, M.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors of protein-protein interactions targeting lysine, tyrosine, or histidine residues</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5616</span>â <span class="NLM_lpage">5627</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b00561</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00561" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVOkur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5616-5627&author=L.+Gambiniauthor=C.+Baggioauthor=P.+Udompholkulauthor=J.+Jossartauthor=A.+F.+Salemauthor=J.+Perryauthor=M.+Pellecchia&title=Covalent+inhibitors+of+protein-protein+interactions+targeting+lysine%2C+tyrosine%2C+or+histidine+residues&doi=10.1021%2Facs.jmedchem.9b00561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibitors of Protein-Protein Interactions Targeting Lysine, Tyrosine, or Histidine Residues</span></div><div class="casAuthors">Gambini, Luca; Baggio, Carlo; Udompholkul, Parima; Jossart, Jennifer; Salem, Ahmed F.; Perry, J. Jefferson P.; Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5616-5627</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have recently reported a series of Lys-covalent agents targeting the BIR3 domain of the X-linked inhibitor of apoptosis protein (XIAP) using a benzamide-sulfonyl fluoride warhead.  Using XIAP as a model system, we further investigated a variety of addnl. warheads that can be easily incorporated into binding peptides and analyzed their ability to form covalent adducts with lysine and other amino acids, including tyrosine, histidine, serine, and threonine, using biochem. and biophys. assays.  Moreover, we tested aq., plasma stability, cell permeability, and cellular efficacy of the most effective agents.  These studies identified aryl-fluoro sulfates as likely the most suitable electrophiles to effectively form covalent adducts with Lys, Tyr, and His residues, given that these agents were cell permeable and stable in aq. buffer and in plasma.  Our studies contain a no. of general findings that open new possible avenues for the design of potent covalent protein-protein interaction antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_gJvmpArk6LVg90H21EOLACvtfcHk0lil73AQLCmkSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVOkur0%253D&md5=820cf16d34a3230e9629991a3bca240d</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00561%26sid%3Dliteratum%253Aachs%26aulast%3DGambini%26aufirst%3DL.%26aulast%3DBaggio%26aufirst%3DC.%26aulast%3DUdompholkul%26aufirst%3DP.%26aulast%3DJossart%26aufirst%3DJ.%26aulast%3DSalem%26aufirst%3DA.%2BF.%26aulast%3DPerry%26aufirst%3DJ.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DCovalent%2520inhibitors%2520of%2520protein-protein%2520interactions%2520targeting%2520lysine%252C%2520tyrosine%252C%2520or%2520histidine%2520residues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5616%26epage%3D5627%26doi%3D10.1021%2Facs.jmedchem.9b00561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref177"><div class="reference"><strong class="refLabel"><a href="#ref177" class="rightTabRefNumLink">177</a></strong><div class="NLM_citation" id="rightTab-cit177"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span> <span> </span><span class="NLM_article-title">Emerging and re-emerging warheads for targeted covalent inhibitors: applications in medicinal chemistry and chemical biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5673</span>â <span class="NLM_lpage">5724</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5673-5724&author=M.+Gehringerauthor=S.+A.+Laufer&title=Emerging+and+re-emerging+warheads+for+targeted+covalent+inhibitors%3A+applications+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Facs.jmedchem.8b01153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Gehringer, Matthias; Laufer, Stefan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5673-5724</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors (TCIs) are designed to bind poorly conserved amino acids by means of reactive groups, the so-called warheads.  Currently, targeting noncatalytic cysteine residues with acrylamides and other Î±,Î²-unsatd. carbonyl compds. is the predominant strategy in TCI development.  The recent ascent of covalent drugs has stimulated considerable efforts to characterize alternative warheads for the covalent-reversible and irreversible engagement of noncatalytic cysteine residues as well as other amino acids.  This Perspective article provides an overview of warheads-beyond Î±,Î²-unsatd. amides-recently used in the design of targeted covalent ligands.  Promising reactive groups that have not yet demonstrated their utility in TCI development are also highlighted.  Special emphasis is placed on the discussion of reactivity and of case studies illustrating applications in medicinal chem. and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDhc4PWV293LVg90H21EOLACvtfcHk0lil73AQLCmkSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL&md5=4623733558ff99aadebfc13950b816ad</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26aulast%3DGehringer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26atitle%3DEmerging%2520and%2520re-emerging%2520warheads%2520for%2520targeted%2520covalent%2520inhibitors%253A%2520applications%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5673%26epage%3D5724%26doi%3D10.1021%2Facs.jmedchem.8b01153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref178"><div class="reference"><strong class="refLabel"><a href="#ref178" class="rightTabRefNumLink">178</a></strong><div class="NLM_citation" id="rightTab-cit178"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">CovalentInDB: a comprehensive database facilitating the discovery of covalent inhibitors</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">D1122</span>â <span class="NLM_lpage">D1129</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkaa876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1093%2Fnar%2Fgkaa876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=33068433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3MXntlaqtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2021&pages=D1122-D1129&author=H.+Duauthor=J.+Gaoauthor=G.+Wengauthor=J.+Dingauthor=X.+Chaiauthor=J.+Pangauthor=Y.+Kangauthor=D.+Liauthor=D.+Caoauthor=T.+Hou&title=CovalentInDB%3A+a+comprehensive+database+facilitating+the+discovery+of+covalent+inhibitors&doi=10.1093%2Fnar%2Fgkaa876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">CovalentInDB: a comprehensive database facilitating the discovery of covalent inhibitors</span></div><div class="casAuthors">Du, Hongyan; Gao, Junbo; Weng, Gaoqi; Ding, Junjie; Chai, Xin; Pang, Jinping; Kang, Yu; Li, Dan; Cao, Dongsheng; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1122-D1129</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Inhibitors that form covalent bonds with their targets have traditionally been considered highly adventurous due to their potential off-target effects and toxicity concerns.  However, with the clin. validation and approval of many covalent inhibitors during the past decade, design and discovery of novel covalent inhibitors have attracted increasing attention.  A large amt. of scattered exptl. data for covalent inhibitors have been reported, but a resource by integrating the exptl. information for covalent inhibitor discovery is still lacking.  In this study, we presented Covalent Inhibitor Database (CovalentInDB), the largest online database that provides the structural information and exptl. data for covalent inhibitors.  CovalentInDB contains 4511 covalent inhibitors (including 68 approved drugs) with 57 different reactive warheads for 280 protein targets.  The crystal structures of some of the proteins bound with a covalent inhibitor are provided to visualize the protein-ligand interactions around the binding site.  Each covalent inhibitor is annotated with the structure, warhead, exptl. bioactivity, physicochem. properties, etc.  Moreover, CovalentInDB provides the covalent reaction mechanism and the corresponding exptl. verification methods for each inhibitor towards its target.  High-quality datasets are downloadable for users to evaluate and develop computational methods for covalent drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkFzGN1-OPHLVg90H21EOLACvtfcHk0lgJMWZyvQKkqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXntlaqtbw%253D&md5=c426c2ce5eeac717236d956bb0882d36</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkaa876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkaa876%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DWeng%26aufirst%3DG.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DChai%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DCovalentInDB%253A%2520a%2520comprehensive%2520database%2520facilitating%2520the%2520discovery%2520of%2520covalent%2520inhibitors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2021%26volume%3D49%26spage%3DD1122%26epage%3DD1129%26doi%3D10.1093%2Fnar%2Fgkaa876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref179"><div class="reference"><strong class="refLabel"><a href="#ref179" class="rightTabRefNumLink">179</a></strong><div class="NLM_citation" id="rightTab-cit179"><span><span class="NLM_contrib-group"><span class="NLM_string-name">ÃbrÃ¡nyi-Balogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szijj, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarpino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrast, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MitroviÄ, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FonoviÄ, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">NÃ©meth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barreteau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">HorvÃ¡ti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IlaÅ¡, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobec, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KeserÅ±, G. M.</span></span> <span> </span><span class="NLM_article-title">A road map for prioritizing warheads for cysteine targeting covalent inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">94</span>â <span class="NLM_lpage">107</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2018.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2018.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=30321804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFeqtrbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2018&pages=94-107&author=P.+%C3%81br%C3%A1nyi-Baloghauthor=L.+Petriauthor=T.+Imreauthor=P.+Szijjauthor=A.+Scarpinoauthor=M.+Hrastauthor=A.+Mitrovi%C4%87author=U.+P.+Fonovi%C4%8Dauthor=K.+N%C3%A9methauthor=H.+Barreteauauthor=D.+I.+Roperauthor=K.+Horv%C3%A1tiauthor=G.+G.+Ferenczyauthor=J.+Kosauthor=J.+Ila%C5%A1author=S.+Gobecauthor=G.+M.+Keser%C5%B1&title=A+road+map+for+prioritizing+warheads+for+cysteine+targeting+covalent+inhibitors&doi=10.1016%2Fj.ejmech.2018.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">A road map for prioritizing warheads for cysteine targeting covalent inhibitors</span></div><div class="casAuthors">Abranyi-Balogh, Peter; Petri, Laszlo; Imre, Timea; Szijj, Peter; Scarpino, Andrea; Hrast, Martina; Mitrovic, Ana; Fonovic, Ursa Pecar; Nemeth, Krisztina; Barreteau, Helene; Roper, David I.; Horvati, Kata; Ferenczy, Gyorgy G.; Kos, Janko; Ilas, Janez; Gobec, Stanislav; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">94-107</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors have become an integral part of a no. of therapeutic protocols and are the subject of intense research.  The mechanism of action of these compds. involves the formation of a covalent bond with protein nucleophiles, mostly cysteines.  Given the abundance of cysteines in the proteome, the specificity of the covalent inhibitors is of utmost importance and requires careful optimization of the applied warheads.  In most of the cysteine targeting covalent inhibitor programs the design strategy involves incorporating Michael acceptors into a ligand that is already known to bind noncovalently.  In contrast, the authors suggest that the reactive warhead itself should be tailored to the reactivity of the specific cysteine being targeted, and the authors describe a strategy to achieve this goal.  Here, the authors have extended and systematically explored the available org. chem. toolbox and characterized a large no. of warheads representing different chemistries.  The authors demonstrate that in addn. to the common Michael addn., there are other nucleophilic addn., addn.-elimination, nucleophilic substitution and oxidn. reactions suitable for specific covalent protein modification.  Importantly, the authors reveal that warheads for these chemistries impact the reactivity and specificity of covalent fragments at both protein and proteome levels.  By integrating surrogate reactivity and selectivity models and subsequent protein assays, the authors define a road map to help enable new or largely unexplored covalent chemistries for the optimization of cysteine targeting inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogAumXXtgqXbVg90H21EOLACvtfcHk0lgJMWZyvQKkqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFeqtrbJ&md5=5649a7a5b95adc70756caddf765cf67c</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.010%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2581br%25C3%25A1nyi-Balogh%26aufirst%3DP.%26aulast%3DPetri%26aufirst%3DL.%26aulast%3DImre%26aufirst%3DT.%26aulast%3DSzijj%26aufirst%3DP.%26aulast%3DScarpino%26aufirst%3DA.%26aulast%3DHrast%26aufirst%3DM.%26aulast%3DMitrovi%25C4%2587%26aufirst%3DA.%26aulast%3DFonovi%25C4%258D%26aufirst%3DU.%2BP.%26aulast%3DN%25C3%25A9meth%26aufirst%3DK.%26aulast%3DBarreteau%26aufirst%3DH.%26aulast%3DRoper%26aufirst%3DD.%2BI.%26aulast%3DHorv%25C3%25A1ti%26aufirst%3DK.%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKos%26aufirst%3DJ.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DGobec%26aufirst%3DS.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DA%2520road%2520map%2520for%2520prioritizing%2520warheads%2520for%2520cysteine%2520targeting%2520covalent%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D160%26spage%3D94%26epage%3D107%26doi%3D10.1016%2Fj.ejmech.2018.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref180"><div class="reference"><strong class="refLabel"><a href="#ref180" class="rightTabRefNumLink">180</a></strong><div class="NLM_citation" id="rightTab-cit180"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egyed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">HetÃ©nyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ÃbrÃ¡nyi-Balogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KeserÅ±, G. M.</span></span> <span> </span><span class="NLM_article-title">An electrophilic warhead library for mapping the reactivity and accessibility of tractable cysteines in protein kinases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>,  <span class="NLM_fpage">112836</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2020.112836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=10.1016%2Fj.ejmech.2020.112836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=32971426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSisb3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&pages=112836&author=L.+Petriauthor=A.+Egyedauthor=D.+Bajuszauthor=T.+Imreauthor=A.+Het%C3%A9nyiauthor=T.+Martinekauthor=P.+%C3%81br%C3%A1nyi-Baloghauthor=G.+M.+Keser%C5%B1&title=An+electrophilic+warhead+library+for+mapping+the+reactivity+and+accessibility+of+tractable+cysteines+in+protein+kinases&doi=10.1016%2Fj.ejmech.2020.112836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">An electrophilic warhead library for mapping the reactivity and accessibility of tractable cysteines in protein kinases</span></div><div class="casAuthors">Petri, Laszlo; Egyed, Attila; Bajusz, David; Imre, Timea; Hetenyi, Anasztazia; Martinek, Tamas; Abranyi-Balogh, Peter; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112836</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors represent a viable strategy to block protein kinases involved in different disease pathologies.  Although a no. of computational protocols have been published for identifying druggable cysteines, exptl. approaches are limited for mapping the reactivity and accessibility of these residues.  Here, we present a ligand based approach using a toolbox of fragment-sized mols. with identical scaffold but equipped with diverse covalent warheads.  Our library represents a unique opportunity for the efficient integration of warhead-optimization and target-validation into the covalent drug development process.  Screening this probe kit against multiple kinases could exptl. characterize the accessibility and reactivity of the targeted cysteines and helped to identify suitable warheads for designed covalent inhibitors.  The usefulness of this approach has been confirmed retrospectively on Janus kinase 3 (JAK3).  Furthermore, representing a prospective validation, we identified Maternal embryonic leucine zipper kinase (MELK), as a tractable covalent target.  Covalently labeling and biochem. inhibition of MELK would suggest an alternative covalent strategy for MELK inhibitor programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxfhiyWH3hMrVg90H21EOLACvtfcHk0lj_mpCGWO0h9Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSisb3E&md5=370848ac3d886667890171b53f89341e</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112836%26sid%3Dliteratum%253Aachs%26aulast%3DPetri%26aufirst%3DL.%26aulast%3DEgyed%26aufirst%3DA.%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DImre%26aufirst%3DT.%26aulast%3DHet%25C3%25A9nyi%26aufirst%3DA.%26aulast%3DMartinek%26aufirst%3DT.%26aulast%3D%25C3%2581br%25C3%25A1nyi-Balogh%26aufirst%3DP.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DAn%2520electrophilic%2520warhead%2520library%2520for%2520mapping%2520the%2520reactivity%2520and%2520accessibility%2520of%2520tractable%2520cysteines%2520in%2520protein%2520kinases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26spage%3D112836%26doi%3D10.1016%2Fj.ejmech.2020.112836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQ4','PDB','4GQ4'); return false;">PDB: 4GQ4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG3','PDB','4OG3'); return false;">PDB: 4OG3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OG8','PDB','4OG8'); return false;">PDB: 4OG8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B41" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B41','PDB','6B41'); return false;">PDB: 6B41</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WNH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WNH','PDB','6WNH'); return false;">PDB: 6WNH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W1Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W1Y','PDB','5W1Y'); return false;">PDB: 5W1Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T5G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T5G','PDB','5T5G'); return false;">PDB: 5T5G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH7','PDB','5TH7'); return false;">PDB: 5TH7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BOZ','PDB','6BOZ'); return false;">PDB: 6BOZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61','PDB','4X61'); return false;">PDB: 4X61</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CKC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CKC','PDB','6CKC'); return false;">PDB: 6CKC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6K1S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6K1S','PDB','6K1S'); return false;">PDB: 6K1S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R','PDB','5E8R'); return false;">PDB: 5E8R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I','PDB','6P7I'); return false;">PDB: 6P7I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZV','PDB','6CZV'); return false;">PDB: 6CZV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CZU','PDB','6CZU'); return false;">PDB: 6CZU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2UXX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2UXX','PDB','2UXX'); return false;">PDB: 2UXX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH','PDB','5CEH'); return false;">PDB: 5CEH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY','PDB','6BGY'); return false;">PDB: 6BGY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA','PDB','6DQA'); return false;">PDB: 6DQA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB','PDB','6DQB'); return false;">PDB: 6DQB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F3I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F3I','PDB','5F3I'); return false;">PDB: 5F3I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIN','PDB','6EIN'); return false;">PDB: 6EIN</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02055&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02055%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02055" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e103e57c7324bb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
