
<html lang="en"     class="pb-page"  data-request-id="26446683-6db5-47f9-8132-7240ff3f4823"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-4;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.5b01209;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent" /></meta><meta name="dc.Creator" content="Paola  Vianello" /></meta><meta name="dc.Creator" content="Oronza A.  Botrugno" /></meta><meta name="dc.Creator" content="Anna  Cappa" /></meta><meta name="dc.Creator" content="Roberto  Dal Zuffo" /></meta><meta name="dc.Creator" content="Paola  Dessanti" /></meta><meta name="dc.Creator" content="Antonello  Mai" /></meta><meta name="dc.Creator" content="Biagina  Marrocco" /></meta><meta name="dc.Creator" content="Andrea  Mattevi" /></meta><meta name="dc.Creator" content="Giuseppe  Meroni" /></meta><meta name="dc.Creator" content="Saverio  Minucci" /></meta><meta name="dc.Creator" content="Giulia  Stazi" /></meta><meta name="dc.Creator" content="Florian  Thaler" /></meta><meta name="dc.Creator" content="Paolo  Trifiró" /></meta><meta name="dc.Creator" content="Sergio  Valente" /></meta><meta name="dc.Creator" content="Manuela  Villa" /></meta><meta name="dc.Creator" content="Mario  Varasi" /></meta><meta name="dc.Creator" content="Ciro  Mercurio" /></meta><meta name="dc.Description" content="We report the stereoselective synthesis and biological activity of a novel series of tranylcypromine (TCPA) derivatives (14a–k, 15, 16), potent inhibitors of KDM1A. The new compounds strongly inhib..." /></meta><meta name="Description" content="We report the stereoselective synthesis and biological activity of a novel series of tranylcypromine (TCPA) derivatives (14a–k, 15, 16), potent inhibitors of KDM1A. The new compounds strongly inhib..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 7, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01209" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01209" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01209" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01209" /></link>
        
    
    

<title>Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01209" /></meta><meta property="og:title" content="Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0019.jpeg" /></meta><meta property="og:description" content="We report the stereoselective synthesis and biological activity of a novel series of tranylcypromine (TCPA) derivatives (14a–k, 15, 16), potent inhibitors of KDM1A. The new compounds strongly inhibit the clonogenic potential of acute leukemia cell lines. In particular three molecules (14d, 14e, and 14g) showing selectivity versus MAO A and remarkably inhibiting colony formation in THP-1 human leukemia cells, were assessed in mouse for their preliminary pharmacokinetic. 14d and 14e were further tested in vivo in a murine acute promyelocytic leukemia model, resulting 14d the most effective. Its two enantiomers were synthesized: the (1S,2R) enantiomer 15 showed higher activity than its (1R,2S) analogue 16, in both biochemical and cellular assays. Compound 15 exhibited in vivo efficacy after oral administration, determining a 62% increased survival in mouse leukemia model with evidence of KDM1A inhibition. The biological profile of compound 15 supports its further investigation as a cancer therapeutic." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01209"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01209">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01209&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01209&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01209&amp;href=/doi/10.1021/acs.jmedchem.5b01209" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1501-1517</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01135" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01267" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Vianello">Paola Vianello</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oronza+A.++Botrugno">Oronza A. Botrugno</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Cappa">Anna Cappa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Roberto++Dal+Zuffo">Roberto Dal Zuffo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Dessanti">Paola Dessanti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Antonello++Mai">Antonello Mai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Biagina++Marrocco">Biagina Marrocco</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Mattevi">Andrea Mattevi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Meroni">Giuseppe Meroni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Saverio++Minucci">Saverio Minucci</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giulia++Stazi">Giulia Stazi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Florian++Thaler">Florian Thaler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Trifir%C3%B3">Paolo Trifiró</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sergio++Valente">Sergio Valente</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Manuela++Villa">Manuela Villa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mario++Varasi">Mario Varasi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ciro++Mercurio">Ciro Mercurio</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Biology and Biotechnology, University of Pavia, Via Ferrata 1, 27100 Pavia, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Drug Chemistry and Technologies, Sapienza University of Rome, P.le A. Moro 5, 00185 Rome, Italy</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Pasteur Institute-Cenci Bolognetti Foundation, Sapienza University of Rome, P.le A. Moro 5, 00185 Rome, Italy</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Biosciences, University of Milan, Via Celoria, 26, 20133 Milan, Italy</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Genextra Group, DAC s.r.l., Via Adamello 16, 20139 Milan, Italy</span></div><div class="corresp-info"><strong>*</strong>P.V.: e-mail, <a href="/cdn-cgi/l/email-protection#2b5b4a44474a055d424a454e4747446b424e44054e5e"><span class="__cf_email__" data-cfemail="0373626c6f622d756a626d666f6f6c436a666c2d6676">[email protected]</span></a>; phone, +39 02 9437 5030.</div><div class="corresp-info"><strong>*</strong>C.M.: e-mail, <a href="/cdn-cgi/l/email-protection#8fece6fde0a1e2eafdecfafde6e0cfe6eae0a1eafa"><span class="__cf_email__" data-cfemail="dbb8b2a9b4f5b6bea9b8aea9b2b49bb2beb4f5beae">[email protected]</span></a>; phone, +39 02 9437 5031.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01209&amp;href=/doi/10.1021%2Facs.jmedchem.5b01209" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1501–1517</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 24, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 July 2015</li><li><span class="item_label"><b>Published</b> online</span>7 January 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 February 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01209" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01209</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1501%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaola%2BVianello%252C%2BOronza%2BA.%2BBotrugno%252C%2BAnna%2BCappa%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D4%26contentID%3Dacs.jmedchem.5b01209%26title%3DDiscovery%2Bof%2Ba%2BNovel%2BInhibitor%2Bof%2BHistone%2BLysine-Specific%2BDemethylase%2B1A%2B%2528KDM1A%252FLSD1%2529%2Bas%2BOrally%2BActive%2BAntitumor%2BAgent%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1517%26publicationDate%3DFebruary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01209"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3156</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">38</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01209" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Vianello&quot;},{&quot;first_name&quot;:&quot;Oronza&quot;,&quot;last_name&quot;:&quot;A. Botrugno&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Cappa&quot;},{&quot;first_name&quot;:&quot;Roberto&quot;,&quot;last_name&quot;:&quot;Dal Zuffo&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Dessanti&quot;},{&quot;first_name&quot;:&quot;Antonello&quot;,&quot;last_name&quot;:&quot;Mai&quot;},{&quot;first_name&quot;:&quot;Biagina&quot;,&quot;last_name&quot;:&quot;Marrocco&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Mattevi&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Meroni&quot;},{&quot;first_name&quot;:&quot;Saverio&quot;,&quot;last_name&quot;:&quot;Minucci&quot;},{&quot;first_name&quot;:&quot;Giulia&quot;,&quot;last_name&quot;:&quot;Stazi&quot;},{&quot;first_name&quot;:&quot;Florian&quot;,&quot;last_name&quot;:&quot;Thaler&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Trifiró&quot;},{&quot;first_name&quot;:&quot;Sergio&quot;,&quot;last_name&quot;:&quot;Valente&quot;},{&quot;first_name&quot;:&quot;Manuela&quot;,&quot;last_name&quot;:&quot;Villa&quot;},{&quot;first_name&quot;:&quot;Mario&quot;,&quot;last_name&quot;:&quot;Varasi&quot;},{&quot;first_name&quot;:&quot;Ciro&quot;,&quot;last_name&quot;:&quot;Mercurio&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;1501-1517&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01209&quot;},&quot;abstract&quot;:&quot;We report the stereoselective synthesis and biological activity of a novel series of tranylcypromine (TCPA) derivatives (14a–k, 15, 16), potent inhibitors of KDM1A. The new compounds strongly inhibit the clonogenic potential of acute leukemia cell lines. In particular three molecules (14d, 14e, and 14g) showing selectivity versus MAO A and remarkably inhibiting colony formation in THP-1 human leukemia cells, were assessed in mouse for their preliminary pharmacokinetic. 14d and 14e were further tested in vivo in a murine acute promyelocytic leukemia model, resulting 14d the most effective. Its two enantiomers were synthesized: the (1S,2R) enantiomer 15 showed higher activity than its (1R,2S) analogue 16, in both biochemical and cellular assays. Compound 15 exhibited in vivo efficacy after oral administration, determining a 62% increased survival in mouse leukemia model with evidence of KDM1A inhibition. The biological profile of compound 15 supports its further investigation as a cancer therapeutic.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01209&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01209" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01209&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01209" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01209&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01209" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01209&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01209&amp;href=/doi/10.1021/acs.jmedchem.5b01209" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01209" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01209" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01209%26sid%3Dliteratum%253Aachs%26pmid%3D26702542%26genre%3Darticle%26aulast%3DVianello%26date%3D2016%26atitle%3DDiscovery%2Bof%2Ba%2BNovel%2BInhibitor%2Bof%2BHistone%2BLysine-Specific%2BDemethylase%2B1A%2B%2528KDM1A%252FLSD1%2529%2Bas%2BOrally%2BActive%2BAntitumor%2BAgent%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D4%26spage%3D1501%26epage%3D1517%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291456" title="Hydrocarbons">Hydrocarbons</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/jmcmar.2016.59.issue-4/20160225/jmcmar.2016.59.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We report the stereoselective synthesis and biological activity of a novel series of tranylcypromine (TCPA) derivatives (<b>14a</b>–<b>k</b>, <b>15</b>, <b>16</b>), potent inhibitors of KDM1A. The new compounds strongly inhibit the clonogenic potential of acute leukemia cell lines. In particular three molecules (<b>14d</b>, <b>14e</b>, and <b>14g</b>) showing selectivity versus MAO A and remarkably inhibiting colony formation in THP-1 human leukemia cells, were assessed in mouse for their preliminary pharmacokinetic. <b>14d</b> and <b>14e</b> were further tested in vivo in a murine acute promyelocytic leukemia model, resulting <b>14d</b> the most effective. Its two enantiomers were synthesized: the (1<i>S</i>,2<i>R</i>) enantiomer <b>15</b> showed higher activity than its (1<i>R</i>,2<i>S</i>) analogue <b>16</b>, in both biochemical and cellular assays. Compound <b>15</b> exhibited in vivo efficacy after oral administration, determining a 62% increased survival in mouse leukemia model with evidence of KDM1A inhibition. The biological profile of compound <b>15</b> supports its further investigation as a cancer therapeutic.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/4"><issue-title>Epigenetics</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Over the past 2 decades, the relevance of epigenetic modifications for transcriptional control and normal development, as well as the role of such modifications in cancer development and progression, has attracted a great deal of attention.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Histone lysine methylation, a dynamic and accurately regulated mechanism,<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> represents one of the several post-translation modifications controlling chromatin conformation and gene transcription.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Two enzyme families, lysine methyltransferases (KMTs)<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and demethylases (KDMs),<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> control histone lysine methylation. KDMs encompass the Jumonji C (JmjC) domain-containing proteins<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and the lysine specific demethylase (LSD) family.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Two lysine specific demethylase proteins have been identified to date: KDM1A (LSD1), a flavin adenine dinucleotide (FAD) dependent amino oxidase, which was the first discovered histone lysine demethylase,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and its homologous KDM1B (LSD2).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> KDM1A is often found associated with protein complexes functioning as transcriptional repressors, such as CoRest,<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12-14)</a> and with the nucleosome remodeling and deacetylation complex Nurd.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> Consistently, KDM1A demethylates mono and dimethyl histone 3 lysine 4 (H3K4me1, H3K4me2), which are marks associated with active gene transcription.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p">In addition, KDM1A has been found in complexes involved in active gene transcription and reported to change its substrate specificity from H3K4me1/me2 to mono and dimethyl histone H3 lysine 9 (H3K9me1/me2), once associated with estrogen and androgen nuclear receptors.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> KDM1A has also been found in protein complexes involved in transcription elongation such as the ELL complex<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and the MLL supercomplex.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_p">The H3K4 demethylase activity of KDM1A can be influenced by histone post-translational modifications such as acetylation<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and phosphorylation.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In particular, both histone H3 lysine 9 acetylation (H3K9Ac) and histone H3 serine 10 phosphorylation (H3S10p)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> can strongly reduce or block KDM1A demethylase activity.</div><div class="NLM_p">Several non-histone substrates of KDM1A have been identified.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref28">(24-28)</a> For instance, KDM1A demethylates and stabilizes DNA methyl transferase 1 (DNMT1),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> E2F1,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and STAT3.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Furthermore, KDM1A demethylates the myosin phosphatase MYPT1, a phosphatase involved in Rb dephosphorylation,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> as well as repressing p53 function through the inhibition of its interaction with 53BP1.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">High KDM1A expression correlates with poor prognosis in several tumor types such as neuroblastoma,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> prostate cancer,<a onclick="showRef(event, 'ref18 ref30'); return false;" href="javascript:void(0);" class="ref ref18 ref30">(18, 30)</a> non-small-cell lung cancer,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> breast cancer,<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> esophageal squamous cancer,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and hepatocellular carcinoma.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Bladder cancer<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and colorectal cancer<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> are also characterized by elevated KDM1A expression. Notably, KDM1A was recently proposed to confer stem-cell-like characteristics to breast cancer cells<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> as well as maintain stem-cell-like tumor propagating cells in human glioblastoma.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In hematopoietic and lymphoid neoplasias, high expression of KDM1A has been observed in acute myeloid leukemia,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> myeloproliferative neoplasms, chronic myelomonocytic leukemia, and myelodysplastic syndromes.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> In addition, strong experimental evidence supports a role for KDM1A in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML) and, more specifically, in the promotion of the oncogenic gene program associated with MLL-AF9 leukemia.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><div class="NLM_p">The link between KDM1A and tumor pathogenesis, and the discovery that the monoamine oxidase (MAO) inhibitor tranylcypromine (TCPA) blocks KDM1A activity,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> represented the starting point for intensive drug discovery programs aimed at the development of irreversible TCPA-based KDM1A inhibitors.<a onclick="showRef(event, 'ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56">(43-56)</a> Two TCPA derivatives (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), <i>N</i>4-[(1<i>R</i>,2<i>S</i>)-2-phenylcyclopropyl]cyclohexane-1,4-diamine (<b>22</b>, ORY1001/RG-6016)<a onclick="showRef(event, 'ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref59">(57-59)</a> and 4-[[4-[[[(1<i>R</i>,2<i>S</i>)-2-phenylcyclopropyl]amino]methyl]-1-piperidyl]methyl]benzoic acid (<b>23</b>, GSK2879552),<a onclick="showRef(event, 'ref60 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref60 ref61 ref62">(60-62)</a> both characterized by nitrogen substitution at the core of TCPA, are currently under clinical investigation. In addition, several reversible inhibitors of KDM1A have been identified by different research groups.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KDM1A inhibitors in clinical studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a previous manuscript, we showed that substitutions on the phenyl moiety of TCPA allowed the identification of highly active KDM1A inhibitors.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Among these, compound <b>1</b> (4-benzamidocyclopropylamine; <b>13b</b> in ref <a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">64</a>) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>) was demonstrated to be a potent KDM1A inhibitor, while its ortho and meta regioisomers exhibited a lower activity against KDM1A.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The enantiomers of compound <b>1</b> displayed potent KDM1A inhibition, with IC<sub>50</sub> values of 13 and 26 nM, as well as anticlonogenic activity in primary murine acute promyelocytic leukemia blasts. These results prompted us to expand the scaffold with the aim to obtain compounds with a better selectivity profile toward MAO A and good activity in a human cellular model of MLL-AF9 leukemia (THP-1 cells). Compound <b>14d</b> was found to be the most promising derivative in this respect, and its biological properties encouraged the synthesis of its two enantiomers. We report herein the stereoselective synthesis of the two enantiomers (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) together with their in vitro and in vivo characterization.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) triethyl phosphonoacetate, BuLi, DME, 135 °C, microwave, 1 h; (b) LiOH, THF, water, EtOH, microwave, 115 °C, 50 min; (c) I<sub>2</sub>, KIO<sub>3</sub>, H<sub>2</sub>O, CH<sub>3</sub>COOH, H<sub>2</sub>SO<sub>4</sub>, 2 h, 0 °C, reflux; (d) DPPA, <i>t</i>-BuOH, TEA, 90 °C, 20 h; (e) ArCONH<sub>2</sub>, CuI, DMEDA, K<sub>2</sub>CO<sub>3</sub>, dioxane, 110 °C, 20 h; (f) HCl, MeOH, diethyl ether, 0 °C, 20 h.</p></p></figure><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Novel compounds synthesized.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31324" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31324" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Chemistry</h3><div class="NLM_p">The synthesis of the final compounds <b>14a</b>–<b>k</b>, <b>15</b>, and <b>16</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) was carried out following two routes. According to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> (method A), the oxirane <b>2</b> and its 2-<i>S</i> (<b>3</b>) and 2-<i>R</i> (<b>4</b>) analogues were added to a solution of triethyl phosphonoacetate and butyllithium in dimethoxyethane and the reaction was microwave irradiated to 130 °C, affording the ethyl esters of 2-phenylcyclopropanecarboxylic acids. Subsequent hydrolysis of the ethyl esters with LiOH provided the carboxylic acids <b>5</b>–<b>7</b>, which were treated with iodine in acetic acid yielding the 4-iodo derivatives <b>8</b>–<b>10</b>. The protected amino derivatives <b>11</b>–<b>13</b> were subsequently obtained by reaction of the iodo derivatives <b>8</b>–<b>10</b> with diphenyl phosphorazidate (DPPA) in the presence of triethylamine and in <i>tert</i>-butanol at 90 °C. Ullman reaction with primary amides and subsequent BOC deprotection with hydrochloric acid afforded the cyclopropylamines <b>14a</b>–<b>g</b>, <b>15</b>, and <b>16</b> as their hydrochloride salts.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Enantiomers of compound <b>14d</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">According to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> (method B), the appropriate acids <b>18h</b>–<b>k</b> were activated by benzotriazol-1-yloxytripyrrolidinophosphonium hexafluorophosphate (PyBOP) or 1-(3-dimethylaminopropyl)-3-ethylcarbodiimide (EDC) in the presence triethylamine and condensated with <i>tert</i>-butyl <i>N</i>-[2-(4-aminophenyl)cyclopropyl]carbamate <b>17</b>.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> The final compounds <b>14h</b>–<b>k</b> were smoothly obtained by removing Boc protection with hydrochloric acid in dioxane.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) PyBOP, TEA, DMF, rt, overnight; (b) THF, 4 M HCl in dioxane, rt, overnight.</p></p></figure><div class="NLM_p">The carboxylic acid or primary amide intermediates were prepared according to <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a> when not commercially available.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) morpholine, 4-methylpiperazine, or piperidine, K<sub>2</sub>CO<sub>3</sub>, DMF, 90 °C, 5 h; (b) 10% Pd–C, MeOH, Parr apparatus, 50 psi, 25 °C, 5 h; (c) benzyl chloroformate, TEA, THF, rt 1.5 h; (d) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, overnight.</p></p></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) thionyl chloride, DMF, CH<sub>2</sub>Cl<sub>2</sub>, reflux, 2 h; (b) NH<sub>3</sub> (28–30% in water), rt, 1 h.</p></p></figure><div class="NLM_p">The synthesis of bisubstituted acids <b>18h</b>–<b>k</b> is reported in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. <i>tert</i>-Butyl chloronitrobenzoates<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> were converted into <b>19h</b>–<b>k</b> by reaction with morpholine, 4-methylpiperazine, or piperidine in the presence of K<sub>2</sub>CO<sub>3</sub>. Reduction of the nitro group to the amine in compounds <b>20h</b>–<b>k</b> was achieved by conventional catalytic hydrogenation, e.g., in a Parr hydrogenation or H-Cube apparatus using a palladium catalyst. The carbamates <b>21h</b>–<b>k</b> were then obtained by reacting the free amines <b>20h</b>–<b>k</b> with benzyl chloroformate in the presence triethylamine at room temperature. Finally, the <i>tert</i>-butyl ester group was hydrolyzed with TFA (trifluoroacetic acid) in dichloromethane.</div><div class="NLM_p last">The benzamides <b>19e</b>,<b>f</b> were prepared according to <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The commercially available 3- or 4-(2-oxo-1,3-oxazolidin-3-yl)benzoic acids <b>18e</b> and <b>18f</b> were first activated with thionyl chloride and then converted into the amides <b>19e</b>,<b>f</b> with aqueous NH<sub>3</sub> in DMF.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Biological Evaluation and SAR</h3><div class="NLM_p">Benzamido derivatives of TCPA have been previously described, reporting that the para substituted is more potent than the ortho and meta isomers.<a onclick="showRef(event, 'ref56 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref56 ref64 ref65">(56, 64, 65)</a> Substituents on the phenyl ring of compound <b>1</b> were directed toward a major occupancy of the binding pocket, without impacting drug-like properties of the molecules, thus maintaining solubility at an acceptable level. For this purpose, both aliphatic (piperazine, morpholine, piperidine oxazolidin-2-one) and aromatic (furane) heterocycles were prioritized among the set of most frequently used ring systems present in small molecule drugs.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Alternatively or concurrently we introduced a carbobenzyloxy residue which could either impact the stereoelectronic features of the benzamide end side or branch the molecule to modulate selectivity versus MAOs.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> As summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> the final compounds <b>14a</b>–<b>k</b> as well as the reference compound <b>22</b> were profiled in vitro for their KDM1A, KDM1B, MAO A and B inhibitory activities. To this end human recombinant KDM1A/CoREST and KDM1B as well as MAO A and MAO B proteins were employed as enzymatic sources. The compounds were incubated with the enzymatic complex, and the results are expressed as IC<sub>50</sub> values.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical Profile of Compounds <b>1</b>, <b>14a</b>–<b>k</b>, and <b>22</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0013.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0014.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation. IC<sub>50</sub> curves of biochemical assay are supplied as <a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></div><div></div></div><div class="NLM_p">The introduction of different residues did not lead to further increase in KDM1A inhibitory activity, the most potent being the equipotent furan-bearing <b>14a</b> (IC<sub>50</sub> = 0.022 μM compared to 0.019 μM of compound <b>1</b>). However, all compounds showed IC<sub>50</sub> values equal to or less than 0.1 μM, with the exception of compound <b>14k</b> (IC<sub>50</sub> = 0.308 μM). Similarly, substitution in meta versus para did not influence the inhibitory potency of KDM1A (<b>14e</b> vs <b>14f</b>), whereas the use of carbobenzyloxy group gave a 2 fold less potent derivative <b>14b</b>, which slightly improved in its cyclized analogue meta oxazolidin-2-one <b>14e</b>. The para oxazolidin-2-one derivative <b>14f</b> was 2 times more potent than the morpholine analogue <b>14c</b>, which in turn was 3 fold more potent than piperazine <b>14d</b>. Furthermore, substitution of the benzamide, in particular by a 3-benzyloxycarbonylamino (3-CBZ) moiety, resulted in an increased KDM1B inhibitory activity. Specifically, compound <b>1</b> exhibited an IC<sub>50</sub> value of 58.3 μM, the 3-CBZ substituted derivative <b>14b</b> was to be roughly 5 times more active (IC<sub>50</sub> = 11.6 μM), while the disubstituted inhibitors <b>14h</b>, <b>14j</b>, and <b>14k</b> had IC<sub>50</sub> values of around 5 μM. With the exception of <b>14a</b>, having an IC<sub>50</sub> value of 0.09 μM, all other compounds were selective over MAO B: in particular, compounds <b>14b</b>, <b>14g</b>, and <b>14j</b> did not show any MAO B inhibitory activity under the employed experimental conditions. We hypothesized that the higher activity of <b>14a</b> versus MAO B can be related to the aromaticity of the furan, which finds an ideal environment in the highly hydrophobic active site of MAO B.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> MAO A IC<sub>50</sub> values ranged between 0.025 μM (<b>14b</b>) and 1.0 μM (<b>14j</b>), and among them <b>14d</b>, <b>14e</b>, and <b>14g</b>–<b>j</b> were more selective than compound <b>1</b> (MAO A/KDM1A ratio of <2). <b>14g</b> and <b>14j</b> were the most selective ones, with a selectivity ratio higher than 10 (MAO A/KDM1A).</div><div class="NLM_p">Compounds <b>14d</b>, <b>14e</b>, and <b>14g−j</b> having good biochemical activity and with an acceptable selectivity versus MAO A, were selected and further tested towards leukemia cell lines, to investigate their ability to inhibit KDM1A (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). In the first experiment, human promyelocytic leukemia NB4 cells were incubated with the inhibitors at a concentration equal to their corresponding biochemical IC<sub>50</sub> value. NB4 cells were selected on the basis of literature data<a onclick="showRef(event, 'ref64 ref69'); return false;" href="javascript:void(0);" class="ref ref64 ref69">(64, 69)</a> which indicate that the inhibition of KDM1A has a biological relevance both in human promyelocytic leukemia cell lines (NB4) and in murine primary promyelocytic blasts. After 24 h of incubation, GFI1B mRNA levels were assessed by quantitative RT-PCR to evaluate KDM1A inhibition in cells. GFI1B is a gene associated with hematopoietic differentiation; its mRNA expression increases following KDM1A down regulation (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">Figure S2 in Supporting Information</a>), and importantly the gene has been demonstrated to be a direct transcriptional target of KDM1A.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> As shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, all the tested compounds determined at least a 4-fold increase of the mRNA expression of GFI1B, compared to cells treated with the vehicle, proving their ability to block KDM1A activity in cellular systems. Moreover, considering that KDM1A inhibition should induce differentiation of human AML cells,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> we evaluated the effect of the compounds on the expression level of CD11b, a well-known granulocytic differentiation marker modulated by KDM1A down regulation (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">Figure S2 in Supporting Information</a>). As reported in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, all the tested compounds elicited, at their respective biochemical IC<sub>50</sub>, an increase of CD11b expression, which suggests granulocytic differentiation induction of the NB4 cells. Starting from experimental data,<a onclick="showRef(event, 'ref64 ref69'); return false;" href="javascript:void(0);" class="ref ref64 ref69">(64, 69)</a> which indicate that the KDM1A inhibitors are of potential relevance for treatment of promyelocytic leukemia, we further investigated the cellular activity of the compounds by measuring their ability to inhibit the colony formation of murine acute myeloid leukemia blasts (originated by a murine acute promyelocytic leukemia<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a>) in semisolid culture at a dose of 0.250 μM. All the selected compounds inhibited colony formation, with percentage of inhibition ranging from 84% to 57% (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). To confirm that the obtained biological data specifically correlated to KDM1A inhibition, the effect of KDM1A down regulation by short hairpin RNA, both in NB4 cells and in murine promyelocytic blast, was analyzed. As reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">Supporting Information</a>, the effects of KDM1A silencing are in agreement with the reported biological data obtained with the inhibitors in terms of transcriptional effects on the expression of GFI1B and CD11b as well as on the reduction of colony formation in mouse acute promyelocytic blast (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">Supporting Information Figures S2 and S3</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Cellular Characterization of Selected Derivativatives <b>1</b>, <b>14d</b>, <b>14e</b>, <b>14g</b>–<b>j</b>, and <b>22</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0015.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0016.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn2"><div class="footnote" id="t2fn2"><sup>Table a</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) at the biochemical IC<sub>50</sub>, measured after 24 h of compound treatment.</p></div><div class="footnote" id="t2fn3"><sup>Table b</sup><p class="last">Colonies were counted after 7 days. Maximal assay variability of 4%. Percentage of inhibition is referenced to the vehicle (DMSO) treated cells, and the data reported are the mean of several replicates plus the standard deviation (≥2).</p></div><div class="footnote" id="t2fn4"><sup>Table c</sup><p class="last">Colonies were counted after 13 days. Maximal assay variability of 6%. Percentage of inhibition is referenced to the vehicle (DMSO) treated cells, and the data reported are the mean of several replicates plus the standard deviation (≥2).</p></div><div class="footnote" id="t2fn6"><sup>Table d</sup><p class="last">Not determined.</p></div></div><div></div></div><div class="NLM_p">The most active compounds <b>14d</b>, <b>14e</b> and <b>14g</b>, inhibiting the murine promyelocytic blasts at 84% and 79%, respectively, the reference compound <b>1</b> (inhibition of 73%), and <b>22</b> (inhibition of 84%) were additionally tested in a human leukemia cell line. Since the relevance of KDM1A in sustaining the oncogenic potential of MLL-AF9 leukemia stem cells has been demonstrated and reported in the literature,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> we evaluated the anticlonogenic potential of <b>14d</b>, <b>14e</b>, <b>14g</b>, <b>1</b>, and <b>22</b> in a human leukemia cell line, characterized by MLL-AF9 translocation (THP-1). As reported in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, compounds <b>14d</b>, <b>14e</b>, and <b>14g</b> as well as <b>22</b> continued to show strong inhibitory efficacy with a percentage of inhibition of 63%, 51%, 65%, and 58%, respectively, at the fixed dose of 0.50 μM, whereas reference compound <b>1</b> unexpectedly only weakly inhibited the colony forming ability of the cells (8%). Of relevance, <b>14d</b>, <b>14e</b>, and <b>14g</b>, similar to <b>22</b>, induced a clear differentiation of THP-1 cells as demonstrated by altered morphology of the colonies and by examination of cytospin preparations of cells recovered at the end of semisolid culture (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <b>14d</b>, <b>14e</b>, <b>14g</b>, and <b>22</b> induce differentiation of THP-1 cells. (A) Representative images of THP-1 colonies after 13 days in semisolid culture with the indicated inhibitors. (B) Representative images of cytospin preparations of THP-1 cells recovered at the end of semisolid culture assay and May Grunwald-Giemsa stained. I: immature cells. M: macrophage.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We therefore advanced through our screening funnel only those compounds with good potency in the biochemical assay, showing some level of selectivity versus MAO A and demonstrating a clear cellular efficacy in the anticlonogenic potential assay. In view of their promising overall activity in both biochemical and cellular assays <b>14d</b>, <b>14e</b>, and <b>14g</b> were selected for preliminary pharmacokinetic assessment in mouse (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) while the parent compound <b>1</b> was not included in the analysis for its very limited anticlonogenic activity observed in THP-1 cells.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Pharmacokinetic Parameters of <b>14d</b>, <b>14e</b>, and <b>14g e</b> after Single iv and os Administration in CD-1 Mice</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter (unit)</th><th class="colsep0 rowsep0" align="center"><b>14d</b></th><th class="colsep0 rowsep0" align="center"><b>14e</b></th><th class="colsep0 rowsep0" align="center"><b>14g</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">iv, 5 mg/kg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.739 ± 0.126</td><td class="colsep0 rowsep0" align="left">3.65 ± 0.658</td><td class="colsep0 rowsep0" align="left">0.383 ± 0.0367</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">8.09 ± 1.02</td><td class="colsep0 rowsep0" align="left">1.92 ± 0.40</td><td class="colsep0 rowsep0" align="left">12.3 ± 4.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–∞</sub> (μM h)</td><td class="colsep0 rowsep0" align="left">2.16 ± 0.556</td><td class="colsep0 rowsep0" align="left">4.62 ± 1.13</td><td class="colsep0 rowsep0" align="left">2.45 ± 0.144</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">94.8 ± 21.3</td><td class="colsep0 rowsep0" align="left">49.7 ± 12.2</td><td class="colsep0 rowsep0" align="left">80.7 ± 4.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub><i>z</i></sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">67.5 ± 22.1</td><td class="colsep0 rowsep0" align="left">8.45 ± 3.75</td><td class="colsep0 rowsep0" align="left">85.7 ± 28.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">os, 15 mg/kg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left">12.0 ± 10.4</td><td class="colsep0 rowsep0" align="left">0.667 ± 0.289</td><td class="colsep0 rowsep0" align="left">18.0 ± 10.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.138 ± 0.0517</td><td class="colsep0 rowsep0" align="left">1.14 ± 0.16</td><td class="colsep0 rowsep0" align="left">0.0588 ± 0.00317</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–∞</sub> (μM h)</td><td class="colsep0 rowsep0" align="left">2.39 ± 0.770</td><td class="colsep0 rowsep0" align="left">7.12 ± 2.04</td><td class="colsep0 rowsep0" align="left">0.979 ± 0.131</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">55.8</td><td class="colsep0 rowsep0" align="left">15.4</td></tr></tbody></table></div></div><div class="NLM_p">The selected inhibitors were administered in single intravenously (iv) or orally (os) to CD1 mice at 5 mg/kg or 15 mg/kg, respectively. The derivatives were dissolved in 5% glucose solution containing 10% Tween 80 for the iv dose or in 5% glucose solution containing 40% PEG for the oral dose. The plasma PK parameters are summarized in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>. Following the iv administration, the three compounds showed medium to high clearance rates of 94.8 ± 21.3, 49.7 ± 12.2, and 80.7 ± 4.8 mL min<sup>–1</sup> kg<sup>–1</sup> for <b>14d</b>, <b>14e</b>, and <b>14g</b>, respectively. The elimination half-lives were significantly longer for <b>14d</b> (with ∼8 h) and for <b>14g</b> (∼12 h) than for compound <b>14e</b> (1.92 ± 0.40 h). Moreover, the estimated terminal volume of distribution (<i>V</i><sub><i>z</i></sub>) was high for all three compounds, which indicates an extensive tissue distribution of the molecules. In particular, <b>14d</b> and <b>14g</b> had <i>V</i><sub><i>z</i></sub> of 67 and 85 L/kg, respectively, much higher than <b>14e</b> (<i>V</i><sub><i>z</i></sub> = 8.45).</div><div class="NLM_p">Compounds <b>14d</b> and <b>14g</b> were slowly absorbed after oral administration (<i>t</i><sub>max</sub> was 12 and 18 h for <b>14d</b> and <b>14g</b>, respectively), whereas <b>14e</b> was taken up more rapidly (<i>t</i><sub>max</sub> of less than 1 h). <i>C</i><sub>max</sub> and AUC<sub>0–∞</sub> after oral administration were significantly different among the compounds. Specifically, compound <b>14e</b> exhibits a <i>C</i><sub>max</sub> of 1.14 ± 0.16 μM, which is around 20 times more than that obtained for compound <b>14g</b>. The AUC<sub>0–∞</sub> ranged from 0.979 ± 0.131 μM h (<b>14g</b>) to 7.12 ± 2.04 μM h (<b>14e</b>). The higher exposure data after oral administration of <b>14d</b> and <b>14e</b> compared to <b>14g</b>, in terms of <i>C</i><sub>max</sub> and AUC<sub>0–∞</sub>, and the higher oral bioavailability of compounds <b>14d</b> (40%) and <b>14e</b> (55.8%) compared to <b>14g</b> (15.4%) prompted us to select <b>14d</b> and <b>14e</b> for further in vivo characterization.</div><div class="NLM_p">To this aim, <b>14d</b> and <b>14e</b> were submitted to in vivo efficacy experiment in an established murine promyelocytic leukemia model.<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71, 72)</a> For the in vivo test, one million leukemic cells were injected iv into nonirradiated 129SvEv mice. Treatment with <b>14d</b> and <b>14e</b> started once blast cells were detected in the recipients’ peripheral blood (10 days after cell injection). To compare the efficacy between the two selected compounds, <b>14d</b> and <b>14e</b> were orally administered at the fixed dose of 27 mg/kg 5 days per week for 2 weeks. The survival of mice belonging to different experimental groups was analyzed and represented by a Kaplan–Meier survival plot. No significant body weight differences among the groups of mice were observed during the treatment (data not shown), and as reported in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, both compounds elicited an increase of survival of the treated mice compared to the vehicle group. Specifically, <b>14d</b> and <b>14e</b> determined an increase of survival respectively of 35% and 15% with respect to the vehicle group. At this point, to investigate different schedule of compound administration for the in vivo efficacy, we considered the irreversible mechanism toward which these derivatives interact with the enzyme and we postulated that an intermittent treatment could have been desirable. <b>14d</b> in fact was tested at a dose of 45 mg/kg for 3 days per week (Monday, Tuesday, and Wednesday) for 2 weeks that corresponds to the same weekly cumulative dose obtained with the administration of the compound at a dose of 27 mg/kg for 5 days per week. Interestingly, the in vivo efficacy evaluated in terms of survival improved significantly (from 35% to 58%) when we followed the intermittent administration (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). Moreover and importantly, <b>14d</b> showed a significantly better in vivo efficacy in the murine promyelocytic leukemia model if compared to <b>22</b> which, once administered according to schedule/dosage reported by Oryzon,<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> was able to increase the survival of leukemic mice of 14% with respect to the vehicle group (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C).</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. In vivo studies of compounds <b>14e</b>, <b>14d</b>, and <b>22</b>. (A) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>14e</b>. Kaplan–Meier survival curves of leukemic mice treated with compound <b>14e</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>14e</b> was orally administered at a dose of 27 mg/kg for 5 days per week for 2 weeks. (B) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>14d</b>. Kaplan–Meier survival curves of leukemic mice treated with compound <b>14d</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>14d</b> was orally administered at a dose of 27 mg/kg for 5 days per week for 2 weeks and at a dose of 45 mg/kg for 3 days per week for 2 weeks. (C) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>22</b>. Kaplan–Meier survival curves of leukemic mice treated with <b>22</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>22</b> was intraperitoneally administered at a dose of 0.0125 mg/kg for 5 days per week for 2 weeks. (D) Quantitative real-time PCR analysis of GFI1B mRNA expression in leukemic cells recovered from infiltrated spleen at 3 days after starting in vivo treatment with compound <b>14d</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, in order to correlate the observed in vivo efficacy with KDM1A inhibition, spleens of the leukemic mice treated with the vehicle and those with <b>14d</b> were recovered after 3 days of treatment and cells were collected for RNA analysis. mRNA levels of GFI1B were measured by quantitative RT-PCR using specific primers and normalized against TBP mRNA. Results are presented as mRNA expression, and as illustrated in the graph (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D), both schedules determined a clear increase of GFI1B expression, although not statistically significant, supporting the ability of compound to inhibit KDM1A in vivo after oral administration.</div><div class="NLM_p">Encouraged by the results on the racemic mixture of compound <b>14d</b>, we approached the enantioselective synthesis of <b>15</b> (1<i>S</i>,2<i>R</i>) and <b>16</b> (1<i>R</i>,2<i>S</i>) starting from the enantiomerically pure styrene epoxides. As evidenced in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>, compound <b>15</b> inhibited KDM1A with an IC<sub>50</sub> of 0.084 μM and was thus 3 times more potent than the enantiomer <b>16</b> (IC<sub>50</sub> = 0.23uM), which in turn was a weaker inhibitor with respect to the racemate (IC<sub>50</sub> = 0.188 μM). The activity of the racemate and enantiomers versus KDM1B could be considered comparable, being in the high micromolar range. Selectivity versus MAOs remained instead almost unaltered in term of IC<sub>50</sub> ratio for the (1<i>S</i>,2<i>R</i>) enantiomer (ratio MAO A/LSD1 = 3.5 for <b>15</b>, compared to 3.9 for <b>14d</b>), whereas it worsens for compound <b>16</b>. An even greater difference among the enantiomers was found in the cellular assays: specifically compound <b>15</b> was more active in reducing the colony forming ability in the two used cell models and, importantly, in inducing the differentiation of THP-1 cells compared to the 1<i>R</i>,2<i>S</i> analogue <b>16</b> (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a> and <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>).</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>15</b> and <b>16</b> induce differentiation of THP-1 cells. (A) Representative images of THP-1 colonies after 13 days in semisolid culture with the indicated inhibitors. (B) Representative images of cytospin preparations of THP-1 cells recovered at the end of semisolid culture assay and May Grunwald-Giemsa stained. I: immature cells. M: macrophage.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Biochemical Characterization of <b>15</b> and <b>16</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0017.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation. IC<sub>50</sub> curves of biochemical assay are supplied as <a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Cellular Characterization of <b>15</b> and <b>16</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0018.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>Table a</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) at the biochemical IC<sub>50</sub>, measured after 24 h of compound treatment.</p></div><div class="footnote" id="t5fn2"><sup>Table b</sup><p class="last">Colonies were counted after 7 days. Percentage of inhibition is referenced versus the vehicle (DMSO) treated cells, and the data reported are the mean of several replicates plus the standard deviation (≥2) with a maximal assay variability of 4%.</p></div><div class="footnote" id="t5fn3"><sup>Table c</sup><p class="last">Colonies were counted after 13 days. Percentage of inhibition is referenced versus the vehicle (DMSO) treated cells, and the data reported are the mean of several replicates plus the standard deviation (≥2) with a maximal assay variability of 6%.</p></div></div><div></div></div><div class="NLM_p">Compound <b>15</b> was then evaluated for its in vivo behavior. For this purpose, the pharmacokinetic profile in CD-1 mice was analyzed at the same conditions formerly described for inhibitors <b>14d</b>, <b>14e</b>, and <b>14g</b> (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). The compound exhibited a high clearance rate but somewhat lower than the racemic mixture <b>14d</b> (79.7 ± 15.0 mL min<sup>–1</sup> kg<sup>–1</sup> for <b>15</b> over 94.8 ± 21.3 mL min<sup>–1</sup> kg<sup>–1</sup> for <b>14d</b>). Furthermore, <b>15</b>, when orally administered, exhibited a high <i>V</i><sub><i>z</i></sub> of 58.4 ± 9.78 L/kg and a half-life of more than 8 h. Although the oral bioavailability was lower than that shown by <b>14d</b> (22.5% for <b>15</b> over 40% for <b>14d</b>), both <i>C</i><sub>max</sub> and AUC<sub>0–∞</sub> were comparable (0.147 ± 0.112 and 1.66 ± 0.889 of <b>15</b> over 0.138 ± 0.0517 and 2.39 ± 0.770 of <b>14d</b>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Pharmacokinetic Parameters of <b>15</b> after Single iv and os Administration in CD-1 Mice</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter (unit)</th><th class="colsep0 rowsep0" align="center"><b>15</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">iv, 5 mg/kg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μM)</td><td class="colsep0 rowsep0" align="left">1.92 ± 0.883</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">8.52 ± 1.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–∞</sub> (μM h)</td><td class="colsep0 rowsep0" align="left">2.29 ± 0.484</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">79.7 ± 15.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub><i>z</i></sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">58.4 ± 9.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">os, 15 mg/kg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.147 ± 0.112</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–∞</sub> (μM h)</td><td class="colsep0 rowsep0" align="left">1.66 ± 0.889</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">22.5</td></tr></tbody></table></div></div><div class="NLM_p">The single 1<i>S</i>,2<i>R</i> enantiomer <b>15</b> was tested in vivo in the murine promyelocytic leukemia model as previously described for the trans racemate <b>14d</b>. <b>15</b> was dissolved (40% PEG 400 in a 5% glucose solution) and orally administered 3 days per week (Monday, Tuesday, and Wednesday) for 3 weeks at the doses of 11.25 mg/kg and 22.5 mg/kg. The treatment started once blast cells were detected in the recipients’ peripheral blood (10 days after cell injection). As reported for the racemic mixture <b>14d</b>, the survival of mice of the different experimental groups was analyzed and represented by a Kaplan–Meier survival plot. No significant body weight differences among the groups of mice were observed during the treatment. A significant dose dependent increase of survival was obtained, and specifically, <b>15</b> determined an increase of 35% and 62% at the dose of 11.25 and 22.50 mg/kg, respectively (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). Remarkably the observed in vivo efficacy associated with a good tolerability profile of the compound <b>15</b> could be suggestive of its preferential activity on tumor cells with respect to normal cells. In order to correlate the observed in vivo efficacy of <b>15</b> with KDM1A inhibition, spleens of the leukemic mice treated with the vehicle or with compound at the highest tested dose of 22.5 mg/kg were recovered after 3 days of treatment and cells were collected for RNA analysis. A clear increase of GFI1B expression, although not statistically significant, was observed (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B), which supported the ability of <b>15</b> to inhibit KDM1A in vivo after oral administration.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0008.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo studies of compound <b>15</b>. (A) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>15</b>. Kaplan–Meier survival curves of leukemic mice treated with compound <b>15</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>15</b> was orally administered at doses of 11.25 and 22.5 mg/kg for 3 days per week for 3 weeks. (B) Quantitative real-time PCR analysis of GFI1B mRNA expression in leukemic cells recovered from infiltrated spleen at 3 days after starting in vivo treatment with compound <b>15</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A series of novel and potent TCPA-derived inhibitors of KDM1A has been identified starting from the previously disclosed compound <b>1</b>. Some of the compounds (<b>14d</b>, <b>14e</b>, and <b>14g</b>) demonstrated improved selectivity versus MAO A and a higher cellular activity in human THP-1 cells, showing 51–65% inhibition compared to the 8% inhibitory activity of compound <b>1</b>. Pharmacokinetic studies revealed a good oral bioavailability of compounds <b>14d</b> (40%) and <b>14e</b> (55.8%), in both cases significantly higher than that of derivative <b>14g</b> (15.4%). Compounds <b>14d</b> and <b>14e</b> were then studied for their in vivo antitumoral activity in a murine acute promyelocytic leukemia model. Oral treatment with compounds <b>14d</b> and <b>14e</b>, at the fixed dose of 27 mg/kg 5 days per week for 2 weeks, led to significant survival increase without signs of overt toxicity and with evidence of in vivo target modulation. Two enantiomers of <b>14d</b> were eventually prepared and profiled, leading to the identification of the (1<i>S</i>,2<i>R</i>) eutomer <b>15</b>, which showed higher potency in both biochemical and cellular assays in comparison to the (1<i>R</i>,2<i>S</i>) enantiomer <b>16</b>.</div><div class="NLM_p last">Compound <b>15</b>, orally admistered in vivo in the murine promyelocytic leukemia model, was well tolerated and significantly prolonged survival time of the treated mice (35% and 62% at the doses of 11.25 and 22.50 mg/kg, respectively). Collectively, these results have supported the progress of compound <b>15</b> into preclinical development as a potential oral anticancer agent.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry. General Procedures</h3><div class="NLM_p last">Reagents and solvents used, unless stated otherwise, were of commercially available reagent grade quality and were used without further purification. Flash chromatography purifications were performed on Merck silica gel 60 (0.04–0.063 mm). Nuclear magnetic resonance spectra (<sup>1</sup>H NMR and <sup>13</sup>C NMR) were recorded either on a Varian 500 MHz or on a Varian 400 MHz spectrometer at 300 K and are referenced in ppm (δ) relative to TMS. Coupling constants (<i>J</i>) are expressed in hertz (Hz). HPLC–MS experiments were performed on an Acquity UPLC apparatus, equipped with a diode array and a Micromass SQD single quadruple (Waters). Optical rotations were measured at the sodium D line, using a Krüss P3001/3002 RS automatic digital polarimeter and a 100 mm cell, and are reported as follows: concentration (<i>c</i> = g/100 mL), solvent. Purity was monitored at 220 nm, and the purities of the compounds used for biological tests were found to be at least 95%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Materials</h3><div class="NLM_p last"><i>tert</i>-Butyl <i>N</i>-[2-(4-aminophenyl)cyclopropyl]carbamate (48), <i>tert</i> butyl 3-chloro-4-nitrobenzoate,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> 4-(3-furyl)benzamide,<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> benzyl <i>N</i>-(3-carbamoylphenyl)carbamate<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> were prepared according to procedures described in the literature. <b>22</b> was prepared according to the procedure described by Oryzon.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> (1<i>R</i>,2<i>R</i>)<i>-</i>1-Ethyl 2-Phenylcyclopropanecarboxylate (<b>6</b>)</h3><div class="NLM_p">69 mL (173 mmol) of a 2.5 M solution of <i>n</i>-butyllithium in hexane was added at 25 °C dropwise over 5 min to a solution of 39 g (170 mmol) of triethyl 2-phosphonoacetate in 75 mL of dry DME. The reaction mixture was stirred at rt for 5 min, and then 16.0 g (133 mmol) of (<i>S</i>)-styrene oxide was added in one portion. The reaction was heated at 135 °C for 60 min under microwave. The orange reaction mixture was then cooled down to rt, saturated aqueous NH<sub>4</sub>Cl was added, and the product was extracted with Et<sub>2</sub>O. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and the solvent was removed in vacuo. The crude product was filtered on a silica gel pad, giving 21g (83%) of ethyl 2-phenylcyclopropane carboxylate as yellow oil.</div><div class="NLM_p">The ethyl ester was hydrolyzed according to the following procedure: 10.58 g (441.6 mmol) of LiOH in 200 mL of H<sub>2</sub>O was added to a solution of 21 g (110 mmol) of ethyl 2-phenylcyclopropanecarboxylate dissolved in 220 mL of an ethanol/THF mixture (10:1, v:v), and the solution was heated at 115 °C for 50 min under microwave irradiation. Then, the solution was concentrated, diluted with H<sub>2</sub>O, and quenched with 2 M HCl at 0 °C. The precipitate was filtered off, washed with H<sub>2</sub>O, and triturated with Et<sub>2</sub>O, providing 14.85 g (83%) of the (1<i>R</i>,2<i>R</i>) cyclopropanecarboxylic acid <b>6</b> as a white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 11.5 (br s, 1 H, -COO<i>H</i>), 7.34–7.20 (m, 3 H, aromatic protons), 7.16–7.06 (m, 2 H, aromatic protons), 2.66–2.57 (m, 1 H, -C<i>H</i> cyclopropane), 1.98–1.88 (m, 1 H, -C<i>H</i> cyclopropane), 1.73–1.61 (m, 1 H, -C<i>H</i>H cyclopropane), 1.47–1.37 (m, 1 H, -CH<i>H</i>cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 161 [M – H]<sup>−</sup>. [α]<sub>D</sub><sup>20</sup> −297° (<i>c</i> 0.753, CHCl<sub>3</sub>).</div><div class="NLM_p"><b>The <i>trans</i>-1-ethyl-2-phenylcyclopropanecarboxylate 5</b> was purchased from Sigma-Aldrich.</div><div class="NLM_p"><b>(1<i>S</i>,2<i>S</i>)-2-Phenylcyclopropanecarboxylic acid 7</b> was prepared according to the procedure described for example <b>6</b> starting from the appropriate (<i>R</i>)-styrene oxides and phosphonoacetates. Yield: 85%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 11.75 (br s, 1 H, -COO<i>H</i>), 7.35–7.08 (m, 5 H, aromatic protons), 2.67–2.56 (m, 1 H, -C<i>H</i> cyclopropane), 1.98–1.86 (m, 1 H, -C<i>H</i> cyclopropane), 1.74–1.63 (m, 1 H, -C<i>H</i>H cyclopropane), 1.48–1.37 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 161 [M – H]<sup>−</sup>. [α]<sub>D</sub><sup>20</sup> +294° (<i>c</i> 0.933, CHCl<sub>3</sub>).</div><div class="NLM_p last">Examination of ethyl esters of <b>5</b>, <b>6</b>, and <b>7</b> by NMR with the aid of the optically active shift reagent europium tris[3-(heptafluoropropylhydroxymethylene)-(+)-camphorate] showed <b>5</b> to be a mixture of 1 to 1 of the two enantiomers <b>6</b> and <b>7</b> to be optically pure.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> (1<i>R</i>,2<i>R</i>)-2-(4-Iodophenyl)cyclopropanecarboxylic Acid (<b>9</b>)</h3><div class="NLM_p">13 g (50 mmol) of iodine, 4.5 g (21 mmol) of potassium iodate, and 7.5 mL of concentrated H<sub>2</sub>SO<sub>4</sub> in 75 mL of H<sub>2</sub>O were added to a stirred solution of 14.7 g (90.6 mmol) of (1<i>R</i>,2<i>R</i>)-2-phenylcyclopropanecarboxylic acid in 300 mL of AcOH. The mixture was heated to reflux for about 2 h, and then the reaction was stopped by adding 700 mL of 1 M NaHSO<sub>4</sub>. The formed precipitate was filtered, dried, and the resulting solid was triturated with hexane, providing 19 g (73%) of the (1<i>R</i>,2<i>R</i>) carboxylic acid <b>9</b> as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.66–10.65 (m, 1 H, -COO<i>H</i>), 7.72–7.54 (m, 2 H, aromatic protons), 6.96–6.79 (m, 2 H, aromatic protons), 2.63–2.46 (m, 1 H, -C<i>H</i> cyclopropane), 1.97–1.80 (m, 1 H, -C<i>H</i> cyclopropane), 1.74–1.61 (m, 1 H, -C<i>H</i>H cyclopropane), 1.47–1.30 (m, 1 H, -CH<i>H</i>cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 287 [M – H]<sup>−</sup>.</div><div class="NLM_p last">The following carboxylic acids were prepared according to the procedure for compound <b>9</b>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> (<i>trans</i>)-2-(4-Iodophenyl)cyclopropanecarboxylic Acid (<b>8</b>)</h3><div class="NLM_p last">Yield: 84%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.36 (br s, 1 H, -COO<i>H</i>), 7.65–7.55 (m, 2 H, aromatic protons), 7.06–6.93 (m, 2 H, aromatic protons), 2.40–2.28 (m, 1 H, -C<i>H</i> cyclopropane), 1.85–1.73 (m, 1 H, -C<i>H</i> cyclopropane), 1.47–1.37 (m, 1 H, -C<i>H</i>H cyclopropane), 1.35–1.24 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 287 [M – H]<sup>−</sup></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> (1<i>S</i>,2<i>S</i>)-2-(4-Iodophenyl)cyclopropanecarboxylic Acid (<b>10</b>)</h3><div class="NLM_p last">Yield: 80%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.29 (br s, 1 H, -COO<i>H</i>), 7.60 (d, <i>J = </i>8.3 Hz, 2 H, aromatic protons), 6.99 (d, <i>J = </i>8.3 Hz, 2 H, aromatic protons), 2.41–2.27 (m, 1 H, -C<i>H</i> cyclopropane), 1.85–1.71 (m, 1 H, -C<i>H</i> cyclopropane), 1.46–1.36 (m, 1 H, -C<i>H</i>H cyclopropane), 1.35–1.24 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 287 [M – H]<sup>−</sup></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> <i>tert</i>-Butyl <i>N</i>-[<i>trans</i>-2-(4-Iodophenyl)cyclopropyl]carbamate (<b>11</b>)</h3><div class="NLM_p">7.4 g (27 mmol) of diphenylphosphoryl azide and 4.4 mL (32 mmol) of TEA were added to a solution of 7.05 g (24.5 mmol) of <i>trans</i>-2-(4-iodophenyl)cyclopropanecarboxylic acid in 150 mL of dry <i>t</i>-BuOH. After stirring at 90 °C for 20 h the solution was concentrated and the residue was partitioned between 10% aqueous Na<sub>2</sub>CO<sub>3</sub> and Et<sub>2</sub>O. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by flash chromatography (hexane/EtOAc 9:1, v:v), giving 5.6 g (62%) of the <i>tert</i>-butyl carbamate <b>11</b> as white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.67–7.51 (m, 2 H, aromatic protons), 7.25 (br s, 1 H, N<i>H</i>COO-<i>t</i>-Bu), 6.98–6.84 (m, 2 H, aromatic protons), 2.57 (br s, 1 H, -C<i>H</i> cyclopropane), 1.89–1.74 (m, 1 H, -C<i>H</i> cyclopropane), 1.37 (s, 9 H, -COO(C<i>H</i><sub>3</sub>)<sub>3</sub>), 1.18–0.95 (m, 2 H, -C<i>H</i><sub>2</sub> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 260 [MH – 100]<sup>+</sup>.</div><div class="NLM_p last">The following carbamates were prepared according to the procedure for compound <b>11</b>:</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>tert</i>-Butyl <i>N</i>-[(1<i>R</i>,2<i>S</i>)-2-(4-Iodophenyl)cyclopropyl]carbamate (<b>12</b>)</h3><div class="NLM_p last">Yield: 84%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.62–7.53 (m, 2 H, aromatic protons), 6.96–6.82 (m, 2 H, aromatic protons), 4.93–4.72 (m, 1 H, -N<i>H</i>COO<i>t</i>Bu), 2.76–2.60 (m, 1 H, -C<i>H</i> cyclopropane), 2.05–1.91 (m, 1 H, -C<i>H</i> cyclopropane), 1.46 (s, 9 H, -COO(C<i>H</i><sub>3</sub>)<sub>3</sub>), 1.21–1.07 (m, 2 H, -C<i>H</i><sub>2</sub> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 382 [MNa]<sup>+</sup>. [α]<sub>D</sub><sup>20</sup> −74° (<i>c</i> 0.688, MeOH).</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>tert</i>-Butyl <i>N</i>-[(1<i>S</i>,2<i>R</i>)-2-(4-Iodophenyl)cyclopropyl]carbamate (<b>13</b>)</h3><div class="NLM_p last">Yield: 81%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.32–7.22 (m, 2 H, aromatic protons), 7.21–7.09 (m, 3 H, aromatic protons), 4.97–4.65 (m, 1 H, -N<i>H</i>COO-<i>t</i>-Bu), 2.85–2.64 (m, 1 H, -C<i>H</i> cyclopropane), 2.09–1.99 (m, 1 H-C<i>H</i> cyclopropane), 1.47 (s, 9 H, -COO(C<i>H</i><sub>3</sub>)<sub>3</sub>), 1.23–1.10 (m, 2 H, -C<i>H</i><sub>2</sub> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 260 [MH – 100]<sup>+</sup>. [α]<sub>D</sub><sup>20</sup> +89.23° (<i>c</i> 1.003, MeOH).</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 3-(2-Oxooxazolidin-3-yl)benzamide (<b>19e</b>)</h3><div class="NLM_p last">A suspension of 0.56 g (2.7 mmol) of 3-(2-oxooxazolidin-3-yl)benzoic acid (<b>18e</b>) (Enamine Ltd.) in 15 mL of dry CH<sub>2</sub>Cl<sub>2</sub> was treated with 0.246 mL (3.38 mmol) of thionyl chloride and 2 drops of dry DMF. After stirring at reflux for 2 h, the mixture was cooled down to rt and poured into 4 mL of an aqueous solution of NH<sub>3</sub> (28–30% in water). After 1 h the resulting mixture was filtered off to afford a white solid that was washed with water. The aqueous phases were extracted with CH<sub>2</sub>Cl<sub>2</sub>, the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give 0.549 g (98%) of the benzamide <b>19e</b> as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.01 (br s, 1 H, -CON<i>H</i>H), 7.94–7.88 (m, 1 H, aromatic proton), 7.84–7.78 (m, 1 H, aromatic proton), 7.61 (d, <i>J = </i>7.8 Hz, 1 H, aromatic proton), 7.52–7.34 (m, 2 H, aromatic proton, -CONH<i>H</i>), 4.49–4.39 (m, 2 H, -C<i>H</i><sub>2</sub>O oxazolidinone), 4.14–3.98 (m, 2 H, -C<i>H</i><sub>2</sub>N oxazolidinone). MS (ESI), <i>m</i>/<i>z</i>: 207 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-(2-Oxooxazolidin-3-yl)benzamide (<b>19f</b>)</h3><div class="NLM_p last">4-(2-Oxooxazolidin-3-yl)benzamide (<b>19f</b>) was prepared according to the procedure for compound <b>19e</b> starting from 4-(2-oxo-1,3-oxazolidin-3-yl)benzoic acid (<b>18f</b>, Enamine Ltd.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.01–7.79 (m, 3 H, aromatic protons, -CON<i>H</i>H), 7.67–7.53 (m, 2 H, aromatic protons), 7.28 (br s, 1 H, -CONH<i>H</i>), 4.44 (t, <i>J = </i>7.6 Hz, 2 H, -C<i>H</i><sub>2</sub>O oxazolidinone), 4.08 (t, <i>J = </i>7.6 Hz, 2 H, -C<i>H</i><sub>2</sub>N oxazolidinone). MS (ESI), <i>m</i>/<i>z</i>: 207 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 4-(1-Methyl-4-piperidyl)benzamide (<b>19g</b>)</h3><div class="NLM_p last">0.34 g (5.1 mmol) of sodium cyanoborohydride and 0.75 mL of acetic acid were added to a mixture of 0.745 g (3.63 mmol) of 4-(1-methyl-4-piperidyl)benzoic acid (<b>18g</b>, Fluorochem Ltd.) and 0.54 mL of 37% aqueous formaldehyde in 7.5 mL of MeOH. The mixture was stirred for about 30 min, then quenched with water and concentrated. The crude reaction intermediate was taken up with dry CH<sub>2</sub>Cl<sub>2</sub>, and 0.331 mL of (4.54 mmol) thionyl chloride and 2 drops of dry DMF were added. The solution was stirred at reflux for about 2 h, then cooled down to rt, and a further portion of 331 μL (4.538 mmol) thionyl chloride was added. Stirring at reflux was then continued for additional 2 h. For a complete conversion a third portion of 0.331 mL (4.54 mmol) of thionyl chloride was added. After further 2 h at reflux, the mixture was cooled down to rt and poured onto an ice cold solution of 15 mL of aqueous NH<sub>3</sub> (28–30%), and the resulting mixture was partitioned between CH<sub>2</sub>Cl<sub>2</sub> and 10% solution of Na<sub>2</sub>CO<sub>3</sub> in water. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give 442 mg (55% from compound <b>18g</b>) of benzamide <b>19g</b> as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.80–7.58 (m, 3 H, -CON<i>H</i>H, aromatic protons), 7.11–6.85 (m, 3 H, -CONH<i>H</i>, aromatic protons), 3.27–3.15 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.46–2.34 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.21 (s, 3 H, -NC<i>H</i><sub>3</sub>). MS (ESI): <i>m</i>/<i>z</i>: 219 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>N</i>-[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]-4-(4-methylpiperazin-1-yl)benzamide Dihydrochloride (<b>14d</b>)</h3><div class="NLM_p">6 mg (0.03 mmol) of CuI, 220 mg (0.613 mmol) of the carbamate <b>11</b>, 0.15 mg (0.67 mmol) of 4-(4-methylpiperazin-1-yl)benzamide, and 0.17 g (1.23 mmol) of K<sub>2</sub>CO<sub>3</sub> were placed in a vial and charged with nitrogen. 5 mg (0.06 mmol) of <i>N</i>,<i>N</i>′-dimethylethane-1,2-diamine and 2 mL of dioxane were added with a syringe, and the vial was heated to 110 °C for 20 h. The resulting suspension was allowed to cool down to rt and was then filtered through a silica gel pad, eluting with 25 mL of 9:1 CH<sub>2</sub>Cl<sub>2</sub>/MeOH. The filtrate was concentrated and the residue was purified by chromatography (CH<sub>2</sub>Cl<sub>2</sub> to CH<sub>2</sub>Cl<sub>2</sub>/MeOH 95/5) to give 223 mg (81%) of <i>N</i>-[<i>trans</i>-2-[4-[[4-(4-methylpiperazin-1-yl)benzoyl]amino]phenyl]cyclopropyl]carbamate as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.86 (s, 1 H, -CON<i>H</i>), 7.92–7.77 (m, 2 H, aromatic protons), 7.69–7.57 (m, 2 H, aromatic protons), 7.22 (br s, 1 H, -N<i>H</i>COO<i>t</i>Bu), 7.08–6.90 (m, 4 H, aromatic protons), 3.31–3.15 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.62–2.53 (m, 1 H, -C<i>H</i> cyclopropane), 2.47–2.36 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.22 (s, 3 H, -NC<i>H</i><sub>3</sub>), 1.90–1.79 (m, 1 H, -C<i>H</i> cyclopropane), 1.38 (s, 9 H, -COO(C<i>H</i><sub>3</sub>)<sub>3</sub>), 1.13–1.02 (m, 2 H, -C<i>H</i><sub>2</sub> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 451 [M + H]<sup>+</sup>. Then, 2 mL of 2 M HCl in Et<sub>2</sub>O was added to a solution of 200 mg (0.44 mmol) of <i>N</i>-[<i>trans</i>-2-[4-[[4-(4-methylpiperazin-1-yl)benzoyl]amino]phenyl]cyclopropyl]carbamate in 3 mL of Et<sub>2</sub>O/MeOH (2:1, v:v) cooled down to 0 °C. After stirring at rt for 20 h the formed precipitate was filtered off and the light yellow solid washed with Et<sub>2</sub>O and dried at 40 °C under vacuum, giving 167 mg (89%) of <i>N</i>-[4-[<i>trans</i>-2-aminocyclopropyl]phenyl]-4-(4-methylpiperazin-1-yl)benzamide as its dihydrochloride salt. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.97 (s, 1 H, -N<i>H</i><sup>+</sup> CH<sub>3</sub> methyl piperazine), 9.75 (br s, 1 H, -CON<i>H</i>), 8.22 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 7.98–7.83 (m, 2 H, aromatic protons), 7.77–7.63 (m, 2 H, aromatic protons), 7.20–6.98 (m, 4 H, aromatic protons), 4.14–3.93 (m, 2 H, -C<i>H</i><sub>2</sub>N piperazine), 3.60–3.45 (m, 2 H, -C<i>H</i><sub>2</sub>N piperazine), 3.18–2.98 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.87 (s, 3 H, -NC<i>H</i><sub>3</sub>), 2.83–2.74 (m, 1 H, -C<i>H</i> cyclopropane), 2.27–2.17 (m, 1 H, -C<i>H</i> cyclopropane), 1.35–1.27 (m, 1 H, -C<i>H</i>H cyclopropane), 1.24–1.14 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 351 [M + H]<sup>+</sup>.</div><div class="NLM_p last">The following cyclopropylamines were prepared following the procedure for compound <b>14d</b> starting from the carbamate <b>11 (14a-c, 14e-g</b>), <b>12</b> (<b>15</b>) or <b>13</b> (<b>16</b>).</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i>-[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]-4-(3-furyl)benzamide Hydrochloride(<b>14a</b>)</h3><div class="NLM_p last">Yield: 40% from <b>11</b>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.21 (s, 1 H, -CON<i>H</i>), 8.40–8.24 (m, 4 H, -C<i>H</i> furan, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.01–7.94 (m, 2 H, aromatic protons), 7.81–7.75 (m, 3 H, aromatic protons), 7.74–7.68 (m, 2 H, aromatic protons), 7.17–7.11 (m, 2 H, aromatic protons), 7.09–7.05 (m, 1 H, -C<i>H</i> furan), 2.86–2.74 (m, 1 H, -C<i>H</i> cyclopropane), 2.32–2.21 (m, 1 H, -C<i>H</i> cyclopropane), 1.40–1.29 (m, 1 H, -C<i>H</i>H cyclopropane), 1.25–1.11 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 319 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Benzyl <i>N</i>-[3-[[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]carbamoyl]phenyl]carbamate Hydrochloride (<b>14b</b>)</h3><div class="NLM_p last">Yield: 48% from <b>11</b>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.24 (s, 1 H, -N<i>H</i>COO), 9.99 (s, 1 H, -CON<i>H</i>), 8.56 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.06<i>–</i>7.02 (m, 13 H, aromatic protons), 5.17 (s, 2 H, -C<i>H</i><sub>2</sub>O), 2.82–2.68 (m, 1 H, -C<i>H</i> cyclopropane), 2.38<i>–</i>2.26 (m, 1 H,) -C<i>H</i> cyclopropane, 1.45<i>–</i>1.10 (m, 2 H, -C<i>H</i><sub>2</sub> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 402 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-[4-[(<i>trans</i>-2-Aminocyclopropyl]phenyl]-4-morpholinobenzamide Hydrochloride (<b>14c</b>)</h3><div class="NLM_p last">Yield: 21% from <b>11</b>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.93 (s, 1 H, -CON<i>H</i>), 8.31 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 7.93<i>–</i>7.81 (m, 2 H, aromatic protons), 7.74<i>–</i>7.62 (m, 2 H, aromatic protons), 7.17<i>–</i>7.07 (m, 2 H, aromatic protons), 7.05<i>–</i>6.96 (m, 2 H, aromatic protons), 3.79<i>–</i>3.68 (m, 4 H, -C<i>H</i><sub>2</sub>O morpholine), 3.29<i>–</i>3.19 (m, 4 H, -C<i>H</i><sub>2</sub>N morpholine), 2.85<i>–</i>2.71 (m, 1 H, -C<i>H</i> cyclopropane), 2.31<i>–</i>2.19 (m, 1 H, -C<i>H</i> cyclopropane), 1.40<i>–</i>1.27 (m, 1 H, -C<i>H</i>H cyclopropane), 1.23<i>–</i>1.14 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 338 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]-3-(2-oxooxazolidin-3-yl)benzamide Hydrochloride (<b>14e</b>)</h3><div class="NLM_p last">Yield: 49% from <b>11</b>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.29 (s, 1 H, -CON<i>H</i>), 8.31 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.07<i>–</i>8.00 (m, 1 H, aromatic proton), 7.84<i>–</i>7.77 (m, 1 H, aromatic proton), 7.73<i>–</i>7.66 (m, 3 H, aromatic protons), 7.59<i>–</i>7.50 (m, 1 H, aromatic proton), 7.19<i>–</i>7.12 (m, 2 H, aromatic protons), 4.51<i>–</i>4.43 (m, 2 H, -C<i>H</i><sub>2</sub>O oxazolidinone), 4.20<i>–</i>4.08 (m, 2 H, -C<i>H</i><sub>2</sub>N oxazolidinone), 2.87<i>–</i>2.75 (m, 1 H, -C<i>H</i> cyclopropane), 2.32<i>–</i>2.21 (m, 1 H, -C<i>H</i> cyclopropane), 1.39<i>–</i>1.29 (m, 1 H, -C<i>H</i>H cyclopropane), 1.27<i>–</i>1.15 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 338 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i>-[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]-4-(2-oxooxazolidin-3-yl)benzamide Trifluoroacetate (<b>14f</b>)</h3><div class="NLM_p last">Yield: 10% from <b>11</b> and after purification of the final product by preparative HPLC. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.16 (s, 1 H, -CON<i>H</i>), 8.12 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.02<i>–</i>7.96 (m, 2 H, aromatic protons), 7.74<i>–</i>7.66 (m, 4 H, aromatic protons), 7.18<i>–</i>7.08 (m, 2 H, aromatic protons), 4.55<i>–</i>4.40 (m, 2 H, -C<i>H</i><sub>2</sub>O oxazolidinone), 4.20<i>–</i>4.02 (m, 2 H, -C<i>H</i><sub>2</sub>N oxazolidinone), 2.85<i>–</i>2.74 (m, 1 H, -C<i>H</i> cyclopropane), 2.31<i>–</i>2.15 (m, 1 H, -C<i>H</i>H cyclopropane), 1.36<i>–</i>1.13 (m, 2 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 338 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]-4-(1-methyl-4-piperidyl)benzamide Trifluoroacetate (<b>14g</b>)</h3><div class="NLM_p last">Yield: 10% from <b>11</b> and after purification of the final product by preparative HPLC. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.17 (s, 1 H, -CON<i>H</i>), 9.39 (br s, 1 H, -N<i>H</i><sup>+</sup>CH3 piperidine), 8.23 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 7.98<i>–</i>7.87 (m, 2 H, aromatic protons), 7.73<i>–</i>7.61 (m, 2 H, aromatic protons), 7.48<i>–</i>7.31 (m, 2 H, aromatic protons), 7.19<i>–</i>7.09 (m, 2 H, aromatic protons), 3.61<i>–</i>3.45 (m, 2 H, -C<i>H</i><sub>2</sub>N piperidine), 3.14<i>–</i>2.99 (m, 2 H, -C<i>H</i><sub>2</sub>N piperidine), 2.94<i>–</i>2.75 (m, 5 H, -NC<i>H</i><sub>3</sub>, -C<i>H</i> piperidine, -C<i>H</i> cyclopropane), 2.30<i>–</i>2.19 (m, 1 H, -C<i>H</i> cyclopropane), 2.12<i>–</i>1.98 (m, 2 H, -C<i>H</i><sub>2</sub> piperidine), 1.93<i>–</i>1.78 (m, 2 H, -C<i>H</i><sub>2</sub> piperidine), 1.36<i>–</i>1.28 (m, 1 H, -C<i>H</i>H cyclopropane), 1.23<i>–</i>1.14 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 350 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i>-[4-[(1<i>S</i>,2<i>R</i>)-2-Aminocyclopropyl]phenyl]-4-(4-methylpiperazin-1-yl)benzamide Dihydrochloride (<b>15</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.85 (br. s., 1 H, -N<i>H</i><sup>+</sup>CH3 piperazine), 10.00 (s, 1 H, -CON<i>H</i>), 8.44 (br s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 7.95–7.85 (m, 2 H, aromatic protons), 7.74–7.65 (m, 2 H, aromatic protons), 7.10 (dd, <i>J</i> = 9.0, 12.5 Hz, 4 H, aromatic protons), 4.07–3.96 (m, 2 H, -C<i>H</i><sub>2</sub>N piperazine), 3.54–3.43 (m, 2 H, -C<i>H</i><sub>2</sub>N piperazine), 3.25–3.07 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.86–2.70 (m, 4 H, -NC<i>H</i><sub>3</sub>, -C<i>H</i> cyclopropane), 2.33–2.23 (m, 1 H, -C<i>H</i> cyclopropane), 1.41–1.30 (m, 1 H, -C<i>H</i>H cyclopropane), 1.22–1.11 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 351 [M + H]<sup>+</sup>. [α]<sub>D</sub><sup>20</sup> −46.48° (<i>c</i> 0.00236, DMSO).</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-[4-[(1<i>R</i>,2<i>S</i>)-2-Aminocyclopropyl]phenyl]-4-(4-methylpiperazin-1-yl)benzamide Dihydrochloride (<b>16</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.00 (br s, 1 H, -N<i>H</i><sup>+</sup>CH<sub>3</sub> piperazine), 10.01 (s, 1 H, -CON<i>H</i>), 8.48 (br. s, 3 H, -N<i>H</i><sub>3</sub><sup>+</sup>), 7.97–7.85 (m, 2 H, aromatic protons), 7.74–7.63 (m, 2 H, aromatic protons), 7.17–7.01 (m, 4 H, aromatic protons), 4.08–3.95 (m, 2 H, -C<i>H</i><sub>2</sub>N piperazine), 3.54–3.42 (m, 2 H, -C<i>H</i><sub>2</sub>N piperazine), 3.27–3.03 (m, 4 H, -C<i>H</i><sub>2</sub>N piperazine), 2.87–2.67 (m, 4 H, -NC<i>H</i><sub>3</sub>, -C<i>H</i> cyclopropane), 2.36–2.23 (m, 1 H, -C<i>H</i> cyclopropane), 1.43–1.30 (m, 1 H, -C<i>H</i>H cyclopropane), 1.22–1.09 (m, 1 H, -CH<i>H</i> cyclopropane). MS (ESI), <i>m</i>/<i>z</i>: 351 [M + H]<sup>+</sup>. [α]<sub>D</sub><sup>20</sup> +41.21° (<i>c</i> 0.00236, DMSO).</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>tert</i>-Butyl 4-(4-Methylpiperazin-1-yl)-3-nitrobenzoate (<b>19j</b>)</h3><div class="NLM_p last">A suspension of 2.0 g (7.8 mmol) <i>tert</i> butyl 4-chloro-3-nitrobenzoate, 3.22 g (23.3 mmol) of dry K<sub>2</sub>CO<sub>3</sub>), and 2.58 mL (23.3 mmol) of <i>N</i>-methylpiperazine was stirred in 10 mL of dry DMF at 90 °C for 5 h in a sealed tube. Then, the reaction mixture was quenched with water and extracted with EtOAc. The combined organic phases were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated and the residue was purified on silica gel (eluent, EtOAc) and recrystallized from toluene to obtain 2.3 g (92%) of intermediate <b>19j</b> as a yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.37 (s, 1H, aromatic proton), 8.03–8.06 (d, 1H, aromatic proton), 7.07–7.10 (d, 1H, aromatic proton), 3.20 (t, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.58 (t, 4H, -CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.38 (s, 3H, -NC<i>H</i><sub>3</sub>), 1.61 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 164.9, 147.8, 137.2, 135.8, 134.0, 119.3, 115.8, 82.6, 57.7 (2C), 51.9 (2C), 47.0, 28.9 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 321.17 [M]<sup>+</sup>. Mp 138–140 °C. The following <i>tert</i>-butyl benzoates were prepared starting from <i>tert</i> butyl 3-chloro-4-nitrobenzoate and morpholine (<b>19i</b>) or <i>tert</i> butyl 4-chloro-3-nitrobenzoate and piperidine (<b>19k</b>) or morpholine (<b>19h</b>) according to the procedure for intermediate <b>19j</b>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>tert</i>-Butyl 4-Morpholino-3-nitrobenzoate (<b>19h</b>)</h3><div class="NLM_p last">Yield: 85%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.38–8.39 (d, 1H, aromatic proton), 8.06–8.08 (dd, 1H, aromatic proton), 7.08–7.10 (d, 1H, aromatic proton), 3.87 (t, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 3.16 (t, 4H, -N(CH<sub>2</sub>)<sub>2</sub>), 1.61 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 148.7, 134.9, 134.8, 131.0, 122.5, 120.1, 81.5, 66.9 (2C), 51.3 (2C), 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 308.14 [M]<sup>+</sup>. Mp 149–151 °C.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>tert</i>-Butyl 3-Morpholino-4-nitrobenzoate (<b>19i</b>)</h3><div class="NLM_p last">Yield: 87%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 7.79 (d, 1H, aromatic proton), 7.76 (d, 1H, aromatic proton), 7.65–7.67 (dd, 1H, aromatic proton), 3.86 (t, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 3.10 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 1.63 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 142.8, 138.4, 130.9, 127.1, 121.7, 120.1, 81.5, 66.7 (2C), 51.4 (2C), 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 308.14 [M]<sup>+</sup>. Mp 57–60 °C.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>tert</i>-Butyl 4-(1-Piperidyl)-3-nitrobenzoate (<b>19k</b>)</h3><div class="NLM_p last">Yield: 88%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.38 (d, 1H, aromatic proton), 7.98–8.00 (dd, 1H, aromatic proton), 7.07–7.08 (d, 1H, aromatic proton), 2.89 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 1.62–1.64 (m, 6H, piperidine protons), 1.54 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 148.7, 134.9, 134.8, 131.0, 122.5, 120.1, 81.5, 52.6 (2C), 28.2 (3C), 26.1 (2C), 24.0 ppm. MS (EI) <i>m</i>/<i>z</i>: 306.16 [M]<sup>+</sup>. Mp 144–146 °C.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>tert</i>-Butyl 3-Amino-4-(4-methylpiperazin-1-yl)benzoate (<b>20j</b>)</h3><div class="NLM_p">A suspension of 0.80 g (2.5 mmol) of the <i>tert</i>-butyl nitrobenzoate <b>19j</b> in 30 mL of MeOH and 0.13 g (0.12 mmol) of 10% palladium on carbon was placed in a Parr apparatus and was hydrogenated at 50 psi and 25 °C for 5 h. The palladium was then filtered off and the MeOH was evaporated to afford an oily residue that was first purified on silica gel (eluent, CHCl<sub>3</sub>/MeOH, 10:1, v:v) and then recrystallized from cyclohexane to provide 0.29 g (65%) of the <i>tert</i>-butyl 4-aminobenzoate <b>20j</b> as a yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 7.41–7.43 (dd, 1H, aromatic proton), 7.36 (d, 1H, aromatic proton), 6.99–7.00 (d, 1H, aromatic proton), 3.96 (s, 2H, Ph(N<i>H</i><sub>2</sub>)), 2.98 (t, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.61 (t, 4H, -CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.38 (s, 3H, -N(C<i>H</i><sub>3</sub>)), 1.58 (s, 9H,-COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 141.5, 137.7, 124.5 (2C), 119.9, 117.3, 81.5, 53.2 (2C), 50.0 (2C), 44.7, 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 291.19 [M]<sup>+</sup>. Mp 114–116 °C.</div><div class="NLM_p last">The following <i>tert</i>-butyl benzoates were prepared starting from the <i>tert</i>-butyl benzoate <b>19h</b>, <b>19i</b>, or <b>19k</b> according to the procedure for intermediate <b>20j</b>.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>tert</i>-Butyl 4-Morpholino-3-aminobenzoate (<b>20h</b>)</h3><div class="NLM_p last">Yield: 73%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 7.42–7.45 (d, 1H, aromatic proton), 7.38 (s, 1H, aromatic proton), 6.98–7.02 (d, 1H, aromatic proton), 4.0 (br s, 2H, -PhN<i>H</i><sub>2</sub>), 3.88 (t, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.98 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 1.61 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 164.7, 142.2, 134.4, 120.7, 120.5, 115.4, 115.0, 81.7, 66.3 (2C), 53.4 (2C), 28.8 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 278.16 [M]<sup>+</sup>. Mp 126–128 °C.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>tert</i>-Butyl 3-Morpholino-4-aminobenzoate (<b>20i</b>)</h3><div class="NLM_p last">Yield: 84%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 7.68 (d, 1H, aromatic proton), 7.63 (dd, 1H, aromatic proton), 6.69–6.71 (d, 2H, aromatic proton), 4.39 (br s, 2H, -PhN<i>H</i><sub>2</sub>), 3.87 (t, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.93(t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 1.59 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 140.1, 139.2, 123.8, 123.3, 119.2, 115.4, 81.5, 66.6 (2C), 50.5 (2C), 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 278.16. Mp 136–138 °C.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <i>tert</i>-Butyl 4-(1-Piperidyl)-3-aminobenzoate (<b>20k</b>)</h3><div class="NLM_p last">Yield: 67%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 7.11–7.13 (dd, 1H, aromatic proton), 6.81 (d, 1H, aromatic proton), 6.58–6.60 (d, 1H, aromatic proton), 4.35 (s, 2H, -N<i>H</i><sub>2</sub>), 2.96 (t, 4H, piperidine proton), 1.77 (m, 4H, piperidine proton), 1.63 (m, 2H, piperidine proton), 1.54 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 141.5, 137.7, 124.6, 124.5, 120.0, 117.3, 81.5, 51.0 (2C), 28.2 (3C), 25.6 (2C), 24.0 ppm. MS (EI) <i>m</i>/<i>z</i>: 276.18 [M]<sup>+</sup>. Mp 115–117 °C.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>tert</i>-Butyl 3-(Benzyloxycarbonylamino)-4-(4-methylpiperazin-1-yl)benzoate (<b>21j</b>)</h3><div class="NLM_p">Benzyl chloroformate (2.1 mmol, 0.3 mL) was slowly added at 0 °C to a solution of 0.50 g (1.72 mmol) of <i>tert</i>-butyl 4-amino-3-(4-methylpiperazin-1-yl)benzoate (<b>20j</b>) in 10 mL of THF and 0.29 mL (2.1 mmol) of TEA. After stirring at rt for 1.5 h the solution was then quenched with water (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The organic phases were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to afford a residue that was purified on silica gel (eluent EtOAc/CHCl<sub>3</sub>, 1:1, v:v) providing 0.53 g (65%) of the <i>tert</i>-butyl benzoate <b>21j</b>. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.69 (br s, 1H, -PhN<i>H</i>COO), 7.69–7.71 (m, 2H, aromatic proton), 7.37–7.47 (m, 5H, -C<i>H</i><sub>2</sub>Ph), 7.15–7.17 (d, 1H, aromatic proton), 5.28 (s, 2H, -COOC<i>H</i><sub>2</sub>Ph), 2.92 (t, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.61 (t, 4H, CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.20 (s, 3H, -N(C<i>H</i><sub>3</sub>)), 1.60 ((s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 155.9, 144.2, 136.3, 130.3, 128.3 (2C), 128.1 (2C), 127.9, 124.4, 124.0, 120.9, 118.1, 81.5, 67.0, 53.2 (2C), 50.0 (2C), 44.7, 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 425.23 [M]<sup>+</sup>.</div><div class="NLM_p last">The following <i>tert</i>-butyl benzoates were prepared starting from the <i>tert</i> butyl benzoate <b>20h</b>, <b>20i</b>, or <b>20k</b> according to the procedure for intermediate <b>21j</b>.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>tert</i>-Butyl 4-Morpholino-3-(benzyloxycarbonylamino)benzoate (<b>21h</b>)</h3><div class="NLM_p last">Yield: 71%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.71 (br s, 1H, -PhN<i>H</i>COO), 7.71–7.73 (m, 2H, aromatic proton), 7.38–7.47 (m, 5H, -C<i>H</i><sub>2</sub>Ph), 7.15–7.17 (d, 1H, aromatic proton), 5.27 (s, 2H, -COOC<i>H</i><sub>2</sub>Ph), 3.88 (t, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.89 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 1.60 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 155.9, 144.2, 136.3, 130.3, 128.3 (2C), 128.1 (2C), 128.0, 124.4, 124.0, 120.9, 118.1, 81.5, 67.0, 66.6 (2C), 50.5 (2C), 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 425.23 [M]<sup>+</sup>.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>tert</i>-Butyl 3-Morpholino-4-(benzyloxycarbonylamino)benzoate (<b>21i</b>)</h3><div class="NLM_p last">Yield: 73%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.20 (d, 1H, aromatic proton), 8.15 (br s, 1H, -Ph(NH)COO), 7.81–7.83 (m, 2H, aromatic protons), 7.38–7.45 (m, 5H, -CH<sub>2</sub>(Ph)), 5.26 (s, 2H, -COO(CH<sub>2</sub>)Ph), 3.88 (t, 4H, -O(CH<sub>2</sub>)<sub>2</sub>), 2.89 (t, 4H, -N(CH<sub>2</sub>)<sub>2</sub>), 1.61 (s, 9H, -COOC(CH<sub>3</sub>)<sub>3</sub>)ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 155.9, 138.8, 136.3, 128.8, 128.3 (2C), 128.1 (2C), 127.9, 125.1, 122.4, 120.8, 118.6, 81.5, 67.0, 66.6 (2C), 50.5 (2C), 28.2 (3C) ppm. MS (EI) <i>m</i>/<i>z</i>: 425.23 [M]<sup>+</sup>.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>tert</i>-Butyl 4-(1-Piperidyl)-3-(benzyloxycarbonylamino)benzoate (<b>21k</b>)</h3><div class="NLM_p last">Yield: 68%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 8.08–8.11 (dd, 1H, aromatic proton), 7.86–7.87 (d, 1H, aromatic proton), 7.33 (m, 5H, aromatic protons), 7.03 (d, 1H, aromatic proton), 6.80 (s, 1H, -CON<i>H</i>Ph), 5.10 (s, 2H, -COOC<i>H</i><sub>2</sub>Ph), 2.96 (t, 4H, piperidine protons), 1.77 (m, 4H, piperidine protons), 1.63 (m, 2H, piperidine protons), 1.54 (s, 9H, -COOC(C<i>H</i><sub>3</sub>)<sub>3</sub>) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 165.1, 155.8, 144.2, 136.3, 130.3, 128.3 (2C), 128.1 (2C), 127.9, 124.4, 124.0, 120.9, 118.1, 81.5, 67.0, 51.0 (2C), 28.2 (3C), 25.6 (2C), 23.9 ppm. MS (EI) <i>m</i>/<i>z</i>: 410.22 [M]<sup>+</sup>.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 3-(Benzyloxycarbonylamino)-4-(4-methylpiperazin-1-yl)benzoic Acid (<b>18j</b>)</h3><div class="NLM_p">A solution of 0.20 g (0.47 mmol) of <i>tert-</i>butyl benzoate <b>21j</b> and 0.72 mL (9.4 mmol) of TFA in 5 mL of dry CH<sub>2</sub>Cl<sub>2</sub> was stirred at rt overnight. The solvent was then removed and the resulting solid was first triturated with 10 mL of Et<sub>2</sub>O and then crystallized from acetonitrile to give 0.13 g (76%) of the benzoic acid <b>18j</b> as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 12.84 (br s, 1H, -COO<i>H</i>), 9.85 (br s, 1H, -PhN<i>H</i>COO), 8.36 (s, 1H, aromatic proton), 7.66–7.68 (d, 1H, aromatic proton), 7.40–7.46 (m, 3H, aromatic protons), 7.36–7.38 (d, 2H, aromatic protons), 7.22–7.25 (d, 1H, aromatic proton), 5.22 (s, 2H, -COOC<i>H</i><sub>2</sub>Ph), 3.20 (t, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 3.08 (t, 4H, CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.86 (s, 3H, -N(C<i>H</i><sub>3</sub>)) ppm. <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 100 MHz) δ 167.6, 155.8, 145.2, 136.3, 129.3, 128.3 (2C), 128.1 (2C), 127.9, 125.2, 125.1, 119.7, 118.2, 67.0, 53.2 (2C), 50.0 (2C), 44.7 ppm. MS (EI) <i>m</i>/<i>z</i>: 369.17 [M]<sup>+</sup>. Mp 205–207 °C.</div><div class="NLM_p last">The following <i>tert</i>-butyl benzoates were prepared starting from the <i>tert</i> butyl benzoate <b>21h</b>, <b>21i</b>, or <b>21k</b> according to the procedure for intermediate <b>18j</b>.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 4-Morpholino-3-(benzyloxycarbonylamino)benzoic Acid (<b>18h</b>)</h3><div class="NLM_p last">Yield: 83%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 12.88 (br s, 1H, -COO<i>H</i>), 9.72 (br s, 1H, -N<i>H</i><sup><i>+</i></sup>), 8.76 (br s, 1H, -N<i>H</i>COOCH<sub>2</sub>Ph), 8.04–8.06 (d, 1H, aromatic proton), 7.75–7.77 (d, 1H, aromatic proton), 7.70 (s, 1H, aromatic proton), 7.36–7.47 (m, 5H, -CH<sub>2</sub><i>Ph</i>), 5.23 (s, 2H, -NHCOOC<i>H</i><sub>2</sub>Ph), 3.46–3.48 (t, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.99–3.12 (m, 4H, -CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.87 (s, 3H, -NC<i>H</i><sub>3</sub>) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 166.8, 154.1, 142.7, 136.3, 128.8 (2C), 127.5, 127.3 (2C), 126.2, 124.6, 122.7, 120.4, 112.5, 66.9, 57.2 (2C), 52.4 (2C), 46.7 ppm. MS (EI) <i>m</i>/<i>z</i>: 356.14 [M]<sup>+</sup>. Mp 234–236 °C.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 3-Morpholino-4-(benzyloxycarbonylamino)benzoic Acid (<b>18i</b>)</h3><div class="NLM_p last">Yield: 88%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 12.81 (br s, 1H, -COO<i>H</i>), 8.61 (s, 1H, PhN<i>H</i>COO), 7.96–7.98 (d, 1H, aromatic proton), 7.71–7.73 (m, 2H, aromatic proton), 7.33–7.46 (m, 5H, -C<i>H</i><sub>2</sub>Ph), 5.22 (s, 2H, -COOC<i>H</i><sub>2</sub>Ph), 3.77 (t, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.81 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 100 MHz) δ 167.6, 155.8, 138.7, 136.3, 130.9, 128.3 (2C), 128.1 (2C), 127.9, 126.7, 122.14, 121.0, 116.2, 67.0, 66.6 (2C), 50.5 (2C) ppm. MS (EI) <i>m</i>/<i>z</i>: 369.16 [M]<sup>+</sup>. Mp 218–220 °C.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 4-(1-Piperidyl)-3-(benzyloxycarbonylamino)benzoic Acid (<b>18k</b>)</h3><div class="NLM_p last">Yield: 86%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 12.99 (s, 1 H, -COO<i>H</i>), 9.88 (s, 1H, -CON<i>H</i>Ph), 8.38 (d, 1H, aromatic proton), 8.15–8.17 (dd,1H, aromatic proton), 7.40–7.41 (d, 1H, aromatic proton), 7.29–7.36 (m, 5H, aromatic protons), 4.87 (s, 2H, -COOC<i>H</i><sub>2</sub>Ph), 2.99 (t, 4H, piperidine protons), 1.77 (m, 4H, piperidine protons), 1.59 (m, 2H, piperidine protons) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 100 MHz) δ 167.6, 155.9, 145.2, 136.3, 129.3, 128.3 (2C), 128.1 (2C), 127.9, 125.2, 125.1, 119.7, 118.2, 67.0, 51.0 (2C), 25.6 (2C), 24.0 ppm. MS (EI) <i>m</i>/<i>z</i>: 354.15 [M]<sup>+</sup>. Mp 214–216 °C.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Benzyl <i>N</i>-[5-[[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]carbamoyl]-2-(4-methylpiperazin-1-yl)phenyl]carbamate Dihydrochloride (<b>14j</b>)</h3><div class="NLM_p">0.15 mL (1.08 mmol) of TEA and 0.17 g (0.32 mmol) of PyBop were added to a solution of 0.10 g (0.27 mmol) of the benzoic acid <b>18j</b> in dry DMF under nitrogen atmosphere, and the resulting mixture was stirred at rt for 45 min. 0.067 g (2.71 mmol) of <i>tert</i>-butyl <i>N</i>-[<i>trans</i>-2-(4-aminophenyl)cyclopropyl]carbamate (<b>17</b>)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> was added under N<sub>2</sub> atmosphere and the reaction was continued overnight, then quenched with water (20 mL), and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organic phases were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to furnish a residue that was purified by chromatography on silica gel (eluent, EtOAc/CHCl<sub>3</sub>, 1:1, v:v) giving benzyl <i>N</i>-[5-[[4-[<i>trans</i>-2-(<i>tert</i>-butoxycarbonylamino)cyclopropyl]phenyl]carbamoyl]-2-(4-methylpiperazin-1-yl)phenyl]carbamate. Yield: 68%. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ 11.12 (br s, 1H, -N<i>H</i><sup><i>+</i></sup>), 10.27 (br s, 1H, -PhCON<i>H</i>), 8.74 (br s, 1H, -N<i>H</i>COOCH<sub>2</sub>Ph), 8.01–8.03 (d, 1H, aromatic proton), 7.79–7.86 (m, 2H, aromatic protons), 7.70–7.72 (d, 2H, aromatic protons), 7.34–7.48 (m, 5H, aromatic protons), 7.13–7.16 (d, 2H, aromatic protons), 5.23 (s, 2H, -NHCOOC<i>H</i><sub>2</sub>Ph), 4.89 (br s, 1H, -N<i>H</i>COOC(CH<sub>3</sub>)<sub>3</sub>), 3.32–3.38 (m, 4H, -NC<i>H</i><sub>3</sub> and -PhC<i>H</i>), 3.15 (m, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.80 (m, 4H, -CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.31 (m, 1H, -C<i>H</i>NH<sub>2</sub>), 1.50 (s, 9H, -C(C<i>H</i><sub>3</sub>)<sub>3</sub>), 1.37 (m, 1H, -C<i>H</i>H cyclopropane), 1.18 (m, 1H, -CH<i>H</i> cyclopropane) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ 164.9, 155.8, 153.7, 143.3, 137.5, 136.4, 134.5, 130.8, 129.3 (2C), 127.9, 127.6 (2C), 125.3 (2C), 124.5, 122.6, 121.6 (2C), 119.5, 110.3, 79.8, 66.5, 57.4 (2C), 52.3 (2C), 46.8, 32.7, 28.6 (3C), 23.1, 14.5 ppm. MS (EI) <i>m</i>/<i>z</i>: 599.31 [M]<sup>+</sup>. MS (EI) <i>m</i>/<i>z</i>: 599.31 [M]<sup>+</sup>. Then, the Boc protection group was removed by stirring a solution of 0.06 g (0.1 mmol) of benzyl <i>N</i>-[5-[[4-[<i>trans</i>-2-(<i>tert</i>-butoxycarbonylamino)cyclopropyl]phenyl]carbamoyl]-2-(4-methylpiperazin-1-yl)phenyl]carbamate in tetrahydrofuran (5 mL), and 0.5 mL of 4 M HCl in 1,4-dioxane was stirred at rt overnight. The formed solid was then filtered off and washed with Et<sub>2</sub>O (3 × 5 mL) to afford the cyclopropylamine <b>14j</b> as a colorless solid (79%). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 11.12 (br s, 1H, -N<i>H</i><sup>+</sup>), 10.27 (br s, 1H, -PhCON<i>H</i>), 8.74 (br s, 1H, -N<i>H</i>COOCH<sub>2</sub>Ph), 8.56 (br s, 3H, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.01–8.03 (d, 1H, aromatic proton), 7.79–7.86 (m, 2H, aromatic protons), 7.70–7.72 (d, 2H, aromatic protons), 7.30–7.48 (m, 5H, aromatic protons), 7.15–7.18 (d, 2H, aromatic protons), 5.23 (s, 2H, -NHCOOC<i>H</i><sub>2</sub>Ph), 3.33 (s, 3H, -NC<i>H</i><sub>3</sub>), 3.15 (m, 4H, -PhN(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.83 (t, 4H, -CH<sub>3</sub>N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.34 (m, 1H, -C<i>H</i>NH<sub>2</sub>), 1.39 (m, 1H, -CH<i>H</i> cyclopropane), 1.20 (m, 1H, C<i>H</i>H cyclopropane) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 100 MHz) δ 164.8, 153.8, 146.4, 137.3, 136.3, 134.5, 129.0 (2C), 127.9, 127.4 (2C), 125.9, 125.5 (2C), 123.9, 121.3 (2C), 121.0, 118.2, 114.7, 66.8, 57.3 (2C), 52.1 (2C), 46.7, 34.4, 25.6, 17.3 ppm. MS (ESI) <i>m</i>/<i>z</i>: 500 ([M + H]<sup>+</sup>).</div><div class="NLM_p last">The following <i>tert</i>-butyl benzoates were prepared starting from the <i>tert</i> butyl benzoate <b>18h</b>, <b>18i</b>, or <b>18k</b> according to the procedure for compound <b>14j</b>.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Benzyl <i>N</i>-[5-[[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]carbamoyl]-2-morpholinophenyl]carbamate Hydrochloride (<b>14h</b>)</h3><div class="NLM_p last">Yield: 77%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 11.13 (br s, 1H, -N<i>H</i><sup>+</sup>), 10.30 (br s, 1H, -PhCON<i>H</i>), 8.75 (br s, 1H, -N<i>H</i>COOCH<sub>2</sub>Ph), 8.58 (br s, 3H, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.03–8.05 (d, 1H, aromatic proton), 7.79–7.86 (m, 2H, aromatic protons), 7.71–7.73 (d, 2H, aromatic protons), 7.32–7.46 (m, 5H, aromatic protons), 7.15–7.18 (d, 2H, aromatic protons), 5.27 (s, 2H, -NHCOOC<i>H</i><sub>2</sub>Ph), 3.88 (m, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.89 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.33 (m, 1H, -C<i>H</i>NH<sub>2</sub>), 1.38 (m, 1H, -CH<i>H</i> cyclopropane), 1.21 (m, 1H, -C<i>H</i>H cyclopropane) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 100 MHz) δ 166.0, 155.8, 144.7, 140.3, 136.4, 136.3, 128.9, 128.6, 128.3 (2C), 128.1 (2C), 127.9, 126.3 (2C), 124.7, 122.4 (2C), 120.2, 117.5, 67.0, 66.6 (2C), 50.5 (2C), 36.7, 26.9, 19.5 ppm. MS (ESI) <i>m</i>/<i>z</i>: 487 ([M + H]<sup>+</sup>). Yield: 77%.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Benzyl <i>N</i>-[4-[[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]carbamoyl]-2-morpholinophenyl]carbamate Hydrochloride (<b>14i</b>)</h3><div class="NLM_p last">Yield: 78%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 10.15 (br s, 1H, -PhCON<i>H</i>), 8.61 (br s, 1H, -N<i>H</i>COOCH<sub>2</sub>Ph), 8.47 (br s, 3H, -N<i>H</i><sub>3</sub><sup>+</sup>), 7.94–8.02 (m, 2H, aromatic protons), 7.74–7.83 (m, 3H, aromatic protons), 7.69–7.71 (d, 1H, aromatic proton), 7.31–7.46 (m, 6H, aromatic protons), 7.14–7.16 (d, 1H, aromatic proton), 5.22 (s, 2H, -NHCOOC<i>H</i><sub>2</sub>Ph), 3.79 (m, 4H, -O(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.88 (t, 4H, -N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.33 (m, 1H, -C<i>H</i>NH<sub>2</sub>), 1.38 (m, 1H, -CH<i>H</i> cyclopropane), 1.22 (m, 1H, -C<i>H</i>H cyclopropane) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 100 MHz) δ 164.8, 153.8, 143.1, 137.3, 136.0, 134.3, 130.8, 128.8 (2C), 127.6, 127.3 (2C), 125.3 (2C), 124.2, 122.7, 121.1 (2C), 119.1, 110.2, 66.9, 66.5 (2C), 53.3 (2C), 34.4, 25.6, 17.4 ppm. MS (EI) <i>m</i>/<i>z</i>: 486.23 [M]<sup>+</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Benzyl <i>N</i>-[5-[[4-[<i>trans</i>-2-Aminocyclopropyl]phenyl]carbamoyl]-2-(1-piperidyl)phenyl]carbamate Hydrochloride (<b>14k</b>)</h3><div class="NLM_p last">Yield: 86%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 11.12 (br s, 1H, -N<i>H</i><sup>+</sup>), 10.30 (br s, 1H, -PhCON<i>H</i>), 8.74 (br s, 1H, -N<i>H</i>COOCH<sub>2</sub>Ph), 8.56 (br s, 3H, -N<i>H</i><sub>3</sub><sup>+</sup>), 8.02–8.04 (d, 1H, aromatic proton), 7.79–7.86 (m, 2H, aromatic protons), 7.68–7.70 (d, 2H, aromatic protons), 7.32–7.46 (m, 5H, aromatic protons), 7.14–7.17 (d, 2H, aromatic protons), 5.23 (s, 2H, -NHCOOC<i>H</i><sub>2</sub>Ph), 3.32–3.36 (m, 1H, -PhC<i>H</i>), 3.0 (t, 4H, --N(C<i>H</i><sub>2</sub>)<sub>2</sub>), 2.34 (m, 1H, -C<i>H</i>NH<sub>2</sub>), 1.52–1.59 (m, 6H, piperidine protons), 1.39 (m, 1H, -CH<i>H</i> cyclopropane), 1.20 (m, 1H, -C<i>H</i>H cyclopropane) ppm. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 100 MHz) δ 164.8, 153.9, 146.5, 137.3, 136.2, 134.5, 128.9 (2C), 127.7, 127.5 (2C), 125.9, 125.4 (2C), 124.0, 121.3 (2C), 120.8, 118.5, 114.8, 66.8, 54.7 (2C), 34.4, 25.7, 25.5 (2C), 24.5, 17.2 ppm. MS (ESI) <i>m</i>/<i>z</i>: 485 ([M + H]<sup>+</sup>).</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Biological Evaluation. KDM1A (LSD1) Enzyme Inhibition Assay</h3><div class="NLM_p last">The KDM1A (LSD1) enzyme inhibition assay was performed as described previously.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Bioluminescent-Coupled Assay for Monoamine Oxidases (MAO Glo Assay)</h3><div class="NLM_p last">The MAO Glo assay from Promega (catalog V1402, Promega, Madison, WI) was performed as described previously.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Cell Studies (Transcriptional Assay)</h3><div class="NLM_p last">The transcriptional assay was performed as described previously.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Cell Studies (Clonogenic Assay)</h3><div id="sec4_2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Murine Acute Myeloid Leukemia Blast</h4><div class="NLM_p">Murine acute myeloid leukemia blasts were recovered from spleen of acute promyelocytic leukemic mice sacrificed once that the spleen was completely blast infiltrated. Specifically, the model was previously characterized and obtained by injection in 129sv mice of lineage-depleted cells from 129sv mice purified and transduced with PML-RARA-expressing retroviral vector.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> One million of leukemic blast cells (from 129sv mice) injected intravenously into nonirradiated syngenic recipients are able to induce leukemia development leading to mice death 21 days after blast injection.</div><div class="NLM_p last">For the clonogenic assay, murine acute myeloid leukemic cells (20 000 cells) originated from spleen of leukemic mice were seeded in duplicate in methylcellulose medium (MethoCult GF M3434, Stem Cell Technology, Vancouver, BC, Canada) in the presence of the compounds at a fixed concentration of 0.25 μM. After 7 days, colonies were counted. Percentage of inhibition is referenced versus the vehicle (DMSO) treated cells.</div></div><div id="sec4_2_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> THP-1 Cells</h4><div class="NLM_p">Human THP-1 cells were grown in RPMI supplemented with 10% fetal bovine serum, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, and antibiotics and maintained in a humidified tissue culture incubator at 37 °C in 5% CO<sub>2</sub>.</div><div class="NLM_p last">For the clonogenic assay, THP-1 cells were plated at a density of 250 cells/plate and cultured with drugs at 0.5 μM in MethoCult H4435 Enriched (StemCell Technologies, Vancouver, BC) according with the manufacturer instructions for 13 days. After this time, colonies were counted. Percentage of inhibition is referenced versus the vehicle (DMSO) treated cells.</div></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Retroviral Constructs and Production of Retroviruses</h3><div class="NLM_p">shRNA constructs were prepared in the MSCV-based pLMP retroviral vector. The hairpins used in the study were #12 and #13 (scramble ShRNAs) and 1A#3 and 1A#5 (shRNAs against KDM1A):</div><div class="NLM_p">#12, AGTACGCGAAGAATACTATCGA</div><div class="NLM_p">#13, AGTACGTTCAGAATATCATCGAT</div><div class="NLM_p">1A#3, AAGTGATACTGTGCTTGTCCAC</div><div class="NLM_p">1A#5, ATCTCAGAAGATGAGTATTATT</div><div class="NLM_p last">Supernatants from transfected Phoenix packaging cells were collected at 48 and 72 h after transfection and immediately used for infections.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Retroviral Transduction of NB4 and Murine Myeloid Cells</h3><div class="NLM_p">For NB4 cells infection, cells were diluted in retroviral supernatants added with 8 μg/mL polybrene and seeded at 10<sup>6</sup> cells/well in 24-well plates. Spin infection was performed for 2 consecutive days (2 infections per day) at 1800 rpm for 45 min. Puromycin selection (3μg/mL) started 24 h after the last cycle of infection. Assays were carried out 8 days after beginning of infection.</div><div class="NLM_p last">For murine myeloid cells infection, cells were seeded at a density of 2 × 10<sup>6</sup> cells/well onto RetroNectin-coated (Takara Bio Inc., Japan) nontreated 24-well plates (Corning Costar). Spin infection was performed as for NB4 cells 24 h after the last cycle of infection cells were seeded as for clonogenic assay in the presence of 3 μg/mL of puromycin. After 7 days colonies were pooled and 20 000 cells replated in the absence of puromycin. Assays were carried out 7 days later.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Immunoblots and Antibodies</h3><div class="NLM_p last">Whole cell extracts were obtained by lysis in SDS buffer (50 mM Tris-HCl, 10% glycerol, 2% SDS). Proteins were separated by SDS–PAGE, blotted onto PVDF membrane and probed with the indicated antibodies. The antibodies used in the study were anti-LSD1(KDM1A) (no. 2139, Cell Signaling), anti-KDM1B (Sigma HPA031269), and anti-HDAC1 (ab7028),</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> In Vivo Activity</h3><div class="NLM_p">The in vivo activity was conducted on a mouse model as described by Minucci et al.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The model is characterized by the development of a leukemia, resembling the human acute promyelocytic leukemia, which is associated with a blast infiltration of several organs such as bone marrow, liver, and particularly the spleen.</div><div class="NLM_p">For the in vivo analysis one million leukemic cells (obtained from 129SvEv mice<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a>) were injected intravenously into nonirradiated syngenic recipients. Mice were randomized in experimental groups of at least 6 mice, and the treatment started once blast cells are detected in the recipients’ peripheral blood (generally 10 days after injection). Compounds <b>14d</b> and <b>14e</b> were dissolved in a vehicle comprising 40% PEG 400 in 5% glucose solution and orally administered either 5 days per week for 2 weeks at the doses of 27 mg/kg or 3 days per week for 2 weeks at the dose of 45 mg/kg. <b>22</b> was dissolved in a vehicle comprising 40% PEG 400 in 5% glucose solution and intraperitoneally administered 5 days per week for 2 weeks at the dose of 0.0125 mg/kg as reported by Oryzon.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div><div class="NLM_p">The survival of mice of the different experimental groups was analyzed and represented by a Kaplan–Meier survival plot. The pure 1<i>S</i>,2<i>R</i><b>-</b>enantiomer <b>15</b> was then dissolved in a vehicle comprising 40% PEG 400 in a 5% glucose solution and orally administered 3 days per week for 3 weeks at the doses of 11.25 mg/kg and 22.5 mg/kg. The survival of the mice of the different experimental groups was analyzed as in the case of compound <b>14d</b> and represented by a Kaplan–Meier survival plot.</div><div class="NLM_p last">All animal studies were carried out in compliance with Italian Legislative Decree No. 116 dated January 27, 1992, and the European Communities Council Directive No. 86/609/EEC concerning the protection of animals used for experimental or other scientific purposes and according to Institutional Policy Regarding the Care and Use of Laboratory Animals (project specific license number 11/2012).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01209">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11150" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11150" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01209" class="ext-link">10.1021/acs.jmedchem.5b01209</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures illustrating (1) IC<sub>50</sub> curves of biochemical assay, (2) effect of KDM1A down regulation on mRNA expression of GFI1B and CD11b in NB4 cells, and (3) effect of KDM1A down regulation on clonogenic potential of mouse promyelocytic blasts (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf">jm5b01209_si_001.pdf (1.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_002.csv">jm5b01209_si_002.csv (3.02 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01209" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Vianello</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b8c8d9d7d4d996ced1d9d6ddd4d4d7f8d1ddd796ddcd"><span class="__cf_email__" data-cfemail="a3d3c2cccfc28dd5cac2cdc6cfcfcce3cac6cc8dc6d6">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ciro Mercurio</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Genextra
Group, DAC s.r.l., Via
Adamello 16, 20139 Milan, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#73101a011c5d1e16011006011a1c331a161c5d1606"><span class="__cf_email__" data-cfemail="3f5c564d5011525a4d5c4a4d56507f565a50115a4a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oronza A. Botrugno</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Cappa</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberto Dal Zuffo</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Dessanti</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonello Mai</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.le
A. Moro 5, 00185 Rome, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Pasteur Institute-Cenci Bolognetti Foundation, Sapienza
University of Rome, P.le A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biagina Marrocco</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.le
A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Mattevi</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
1, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Meroni</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Saverio Minucci</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biosciences, University of Milan, Via Celoria, 26, 20133 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giulia Stazi</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.le
A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florian Thaler</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Trifiró</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sergio Valente</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.le
A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manuela Villa</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mario Varasi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milan, Italy</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d86e6093-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10884" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10884" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We express our thanks to Paola Dalton for revising the manuscript. This work was supported by Fondazione Cariplo (Grant 2010.0778), AIRC (Grant  15208), MIUR (Progetto Epigen and RF-2010-2318330), FIRB RBFR10ZJQT, Sapienza Ateneo Project 2013, IIT-Sapienza Project, FP7 (Projects BLUEPRINT/282510 and A-PARADDISE/602080), and Regione Lombardia, Fondo per la Promozione di Accordi Istituzionale, Bando di Invito di cui al Decreto No. 4779 del 14/05/2009, Progetto DiVA.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i66" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i66"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i67" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i67"> Abbreviations Used</h2><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">APL</td><td class="NLM_def"><p class="first last">acute promyelocytic leukemia</p></td></tr><tr><td class="NLM_term">br s</td><td class="NLM_def"><p class="first last">broad signal</p></td></tr><tr><td class="NLM_term"><i>n</i>-BuLi</td><td class="NLM_def"><p class="first last"><i>n</i>-butyllithium</p></td></tr><tr><td class="NLM_term">CH<sub>2</sub>Cl<sub>2</sub></td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphenylphosphoryl azide</p></td></tr><tr><td class="NLM_term">DMEDA</td><td class="NLM_def"><p class="first last">2-(dimethylamino)ethylamine</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">mp</td><td class="NLM_def"><p class="first last">melting point</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tripyrrolidinophosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>tert</i>-butanol</p></td></tr><tr><td class="NLM_term">TCPA</td><td class="NLM_def"><p class="first last">tranylcypromine</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68176" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68176" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 75 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Helin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span> </span><span class="NLM_article-title">Chromatin proteins and modifications as drug targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">480</span><span class="NLM_x">–</span> <span class="NLM_lpage">488</span><span class="refDoi"> DOI: 10.1038/nature12751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature12751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24153301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=480-488&author=K.+Helinauthor=D.+Dhanak&title=Chromatin+proteins+and+modifications+as+drug+targets&doi=10.1038%2Fnature12751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin proteins and modifications as drug targets</span></div><div class="casAuthors">Helin, Kristian; Dhanak, Dashyant</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">480-488</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A plethora of groundbreaking studies have demonstrated the importance of chromatin-assocd. proteins and post-translational modifications of histones, proteins and DNA (so-called epigenetic modifications) for transcriptional control and normal development.  Disruption of epigenetic control is a frequent event in disease, and the first epigenetic-based therapies for cancer treatment have been approved.  A generation of new classes of potent and specific inhibitors for several chromatin-assocd. proteins have shown promise in preclin. trials.  Although the biol. of epigenetic regulation is complex, new inhibitors such as these will hopefully be of clin. use in the coming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa6LKkNrgP9bVg90H21EOLACvtfcHk0lhwhRc9h_gVBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E&md5=ca3f12c6aaba31bb571095bacc02550c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12751%26sid%3Dliteratum%253Aachs%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DChromatin%2520proteins%2520and%2520modifications%2520as%2520drug%2520targets%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D480%26epage%3D488%26doi%3D10.1038%2Fnature12751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The epigenomics of cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer&doi=10.1016%2Fj.cell.2007.01.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0lhwhRc9h_gVBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692%26doi%3D10.1016%2Fj.cell.2007.01.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Rodriguez-Paredes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteller, M.</span><span> </span><span class="NLM_article-title">Cancer epigenetics reaches mainstream oncology</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1038/nm.2305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnm.2305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21386836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=330-339&author=M.+Rodriguez-Paredesauthor=M.+Esteller&title=Cancer+epigenetics+reaches+mainstream+oncology&doi=10.1038%2Fnm.2305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics reaches mainstream oncology</span></div><div class="casAuthors">Rodriguez-Paredes, Manuel; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">330-339</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetics is one of the most promising and expanding fields in the current biomedical research landscape.  Since the inception of epigenetics in the 1940s, the discoveries regarding its implications in normal and disease biol. have not stopped, compiling a vast amt. of knowledge in the past decade.  The field has moved from just one recognized marker, DNA methylation, to a variety of others, including a wide spectrum of histone modifications.  From the methodol. standpoint, the successful initial single gene candidate approaches have been complemented by the current comprehensive epigenomic approaches that allow the interrogation of genomes to search for translational applications in an unbiased manner.  Most important, the discovery of mutations in the epigenetic machinery and the approval of the first epigenetic drugs for the treatment of subtypes of leukemias and lymphomas has been an eye-opener for many biomedical scientists and clinicians.  Herein, we will summarize the progress in the field of cancer epigenetics research that has reached mainstream oncol. in the development of new biomarkers of the disease and new pharmacol. strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxyADVbZ4KHrVg90H21EOLACvtfcHk0liFTsBDTSEg_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D&md5=07b6faaddb8bce884261990dbc927331</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnm.2305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2305%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Paredes%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DCancer%2520epigenetics%2520reaches%2520mainstream%2520oncology%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D330%26epage%3D339%26doi%3D10.1038%2Fnm.2305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+lysine+methylation+dynamics%3A+establishment%2C+regulation%2C+and+biological+impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0liFTsBDTSEg_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520lysine%2520methylation%2520dynamics%253A%2520establishment%252C%2520regulation%252C%2520and%2520biological%2520impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0lgYJCkk6q5bxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabari, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span> </span><span class="NLM_article-title">SnapShot: histone modifications</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">458 e1</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2014.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25303536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=458-458+e1&author=H.+Huangauthor=B.+R.+Sabariauthor=B.+A.+Garciaauthor=C.+D.+Allisauthor=Y.+Zhao&title=SnapShot%3A+histone+modifications&doi=10.1016%2Fj.cell.2014.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SnapShot: Histone modifications</span></div><div class="casAuthors">Huang, He; Sabari, Benjamin R.; Garcia, Benjamin A.; Allis, C. David; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">458-458.e1</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Histones are decorated by a variety of protein post-translational modifications called histone marks that modulate chromatin structure and function, contributing to the cellular gene expression program.  This SnapShot summarizes the reported human, mouse, and rat histone marks, including recently identified Lys residue acylation marks.  It has been proposed that histone acylations provide a link between cellular metab. and epigenetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQmsTna8jsxrVg90H21EOLACvtfcHk0lgYJCkk6q5bxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO&md5=969e02ba1cbaa8f87bd5aae32857720f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DSabari%26aufirst%3DB.%2BR.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DSnapShot%253A%2520histone%2520modifications%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D458%26epage%3D458%2520e1%26doi%3D10.1016%2Fj.cell.2014.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span> </span><span class="NLM_article-title">Structural and sequence motifs of protein (histone) methylation enzymes</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">294</span><span class="refDoi"> DOI: 10.1146/annurev.biophys.34.040204.144452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1146%2Fannurev.biophys.34.040204.144452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=15869391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslCku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2005&pages=267-294&author=X.+Chengauthor=R.+E.+Collinsauthor=X.+Zhang&title=Structural+and+sequence+motifs+of+protein+%28histone%29+methylation+enzymes&doi=10.1146%2Fannurev.biophys.34.040204.144452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and sequence motifs of protein (histone) methylation enzymes</span></div><div class="casAuthors">Cheng, Xiaodong; Collins, Robert E.; Zhang, Xing</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">267-294</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  With genome sequencing nearing completion for model organisms used in biomedical research, there is a rapidly growing appreciation that proteomics, the study of covalent modification to proteins, and transcriptional regulation will likely dominate the research headlines in the next decade.  Protein methylation plays a central role in both of these fields, as several different residues (Arg, Lys, Gln) are methylated in cells and methylation plays a central role in the "histone code" that regulates chromatin structure and impacts transcription.  In some cases, a single lysine can be mono-, di-, or trimethylated, with different functional consequences for each of the 3 forms.  This review describes structural aspects of methylation of histone Lys residues by 2 enzyme families with entirely different structural scaffolding (the SET proteins and Dot1p) and methylation of protein Arg residues by protein (arginine) methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptbkTQSJ8bYLVg90H21EOLACvtfcHk0ljMMxEqNP9tmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslCku7s%253D&md5=04dc4de26a818c98cdacccfe74051e52</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.34.040204.144452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.34.040204.144452%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCollins%26aufirst%3DR.%2BE.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DStructural%2520and%2520sequence%2520motifs%2520of%2520protein%2520%2528histone%2529%2520methylation%2520enzymes%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2005%26volume%3D34%26spage%3D267%26epage%3D294%26doi%3D10.1146%2Fannurev.biophys.34.040204.144452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallin, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">JmjC-domain-containing proteins and histone demethylation</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">715</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span><span class="refDoi"> DOI: 10.1038/nrg1945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrg1945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16983801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=715-727&author=R.+J.+Kloseauthor=E.+M.+Kallinauthor=Y.+Zhang&title=JmjC-domain-containing+proteins+and+histone+demethylation&doi=10.1038%2Fnrg1945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">JmjC-domain-containing proteins and histone demethylation</span></div><div class="casAuthors">Klose, Robert J.; Kallin, Eric M.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">715-727</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation has important roles in regulating gene expression and forms part of the epigenetic memory system that regulates cell fate and identity.  Enzymes that directly remove Me marks from histones have recently been identified, revealing a new level of plasticity within this epigenetic modification system.  Here we analyze the evolutionary relationship between Jumonji C (JmjC)-domain-contg. proteins and discuss their cellular functions in relation to their potential enzymic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb9c8M0AJzLVg90H21EOLACvtfcHk0ljMMxEqNP9tmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D&md5=1ac8a66426993770ae33b78ea35f5879</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrg1945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1945%26sid%3Dliteratum%253Aachs%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DKallin%26aufirst%3DE.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DJmjC-domain-containing%2520proteins%2520and%2520histone%2520demethylation%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2006%26volume%3D7%26spage%3D715%26epage%3D727%26doi%3D10.1038%2Fnrg1945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kooistra, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Molecular mechanisms and potential functions of histone demethylases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1038/nrm3327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrm3327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22473470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=297-311&author=S.+M.+Kooistraauthor=K.+Helin&title=Molecular+mechanisms+and+potential+functions+of+histone+demethylases&doi=10.1038%2Fnrm3327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms and potential functions of histone demethylases</span></div><div class="casAuthors">Kooistra, Susanne Marije; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-311</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone modifications are thought to regulate chromatin structure, transcription and other nuclear processes.  Histone methylation was originally believed to be an irreversible modification that could only be removed by histone eviction or by diln. during DNA replication.  However, the isolation of two families of enzymes that can demethylate histones has changed this notion.  The biochem. activities of these histone demethylases towards specific Lys residues on histones, and in some cases non-histone substrates, have highlighted their importance in developmental control, cell-fate decisions and disease.  Their ability to be regulated through protein-targeting complexes and post-translational modifications is also beginning to shed light on how they provide dynamic control during transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolbbZ5rmYmrVg90H21EOLACvtfcHk0ljMMxEqNP9tmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D&md5=a5f5ffdac5c5658bc262d584faf5216f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrm3327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3327%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DS.%2BM.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DMolecular%2520mechanisms%2520and%2520potential%2520functions%2520of%2520histone%2520demethylases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D297%26epage%3D311%26doi%3D10.1038%2Fnrm3327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljkZvafGEqtrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Profumo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">A novel mammalian flavin-dependent histone demethylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">17775</span><span class="NLM_x">–</span> <span class="NLM_lpage">17782</span><span class="refDoi"> DOI: 10.1074/jbc.M109.003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1074%2Fjbc.M109.003087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19407342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17775-17782&author=A.+Karytinosauthor=F.+Fornerisauthor=A.+Profumoauthor=G.+Ciossaniauthor=E.+Battaglioliauthor=C.+Bindaauthor=A.+Mattevi&title=A+novel+mammalian+flavin-dependent+histone+demethylase&doi=10.1074%2Fjbc.M109.003087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mammalian Flavin-dependent Histone Demethylase</span></div><div class="casAuthors">Karytinos, Aristotele; Forneris, Federico; Profumo, Antonella; Ciossani, Giuseppe; Battaglioli, Elena; Binda, Claudia; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17775-17782</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of Lys residues on histone proteins is a well known and extensively characterized epigenetic mark.  The recent discovery of lysine-specific demethylase 1 (LSD1) demonstrated that lysine methylation can be dynamically controlled.  Among the histone demethylases so far identified, LSD1 has the unique feature of functioning through a flavin-dependent amine oxidn. reaction.  Data base anal. reveals that mammalian genomes contain a gene (AOF1, for amine-oxidase flavin-contg. domain 1) that is homologous to the LSD1-coding gene.  Here, we demonstrate that the protein encoded by AOF1 represents a second mammalian flavin-dependent histone demethylase, named LSD2.  The new demethylase is strictly specific for mono- and dimethylated Lys4 of histone H3, recognizes a long stretch of the H3 N-terminal tail, senses the presence of addnl. epigenetic marks on the histone substrate, and is covalently inhibited by tranylcypromine.  As opposed to LSD1, LSD2 does not form a biochem. stable complex with the C-terminal domain of the corepressor protein CoREST.  Furthermore, LSD2 contains a CW-type zinc finger motif with potential zinc-binding sites that are not present in LSD1.  We conclude that mammalian LSD2 represents a new flavin-dependent H3-Lys4 demethylase that features substrate specificity properties highly similar to those of LSD1 but is very likely to be part of chromatin-remodeling complexes that are distinct from those involving LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSPhtflZQU7Vg90H21EOLACvtfcHk0ljkZvafGEqtrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D&md5=4fbe32f2274de75915e2308781a214d1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.003087%26sid%3Dliteratum%253Aachs%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DProfumo%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520novel%2520mammalian%2520flavin-dependent%2520histone%2520demethylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17775%26epage%3D17782%26doi%3D10.1074%2Fjbc.M109.003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooch, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1038/nature04021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature04021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16079794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation&doi=10.1038%2Fnature04021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Cooch, Neil; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">432-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors have previously described a multiprotein complex termed the BHC or BRAF-HDAC complex, which is required for the repression of neuronal-specific genes.  The authors have shown that the BHC complex is recruited by a neuronal silencer, REST (RE1-silencing transcription factor), and mediates the repression of REST-responsive genes.  BHC is a multiprotein complex consisting of two enzymic activities: a histone deacetylase (HDAC1 or 2) and a recently described histone demethylase (BHC110, also known as LSD1 or AOF2).  Here the authors show that BHC110-contg. complexes show a nearly fivefold increase in demethylation of histone H3 lysine 4 (H3K4) compared to recombinant BHC110.  Furthermore, recombinant BHC110 is unable to demethylate H3K4 on nucleosomes, but BHC110-contg. complexes readily demethylate nucleosomes.  In vitro reconstitution of the BHC complex using recombinant subunits reveals an essential role for the REST corepressor CoREST, not only in stimulating demethylation on core histones but also promoting demethylation of nucleosomal substrates.  The authors find that nucleosomal demethylation is the result of CoREST enhancing the assocn. between BHC110 and nucleosomes.  Depletion of CoREST in in vivo cell culture results in de-repression of REST-responsive gene expression and increased methylation of H3K4.  Together, these results highlight an essential role for CoREST in demethylation of H3K4 both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmTdZApayNhrVg90H21EOLACvtfcHk0lh05pcT57T1oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D&md5=535a6d800bd6672192be133516b4dee7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature04021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04021%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435%26doi%3D10.1038%2Fnature04021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Shi, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwase, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Regulation of LSD1 histone demethylase activity by its associated factors</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">857</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span><span class="refDoi"> DOI: 10.1016/j.molcel.2005.08.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2005.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16140033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=857-864&author=Y.+J.+Shiauthor=C.+Matsonauthor=F.+Lanauthor=S.+Iwaseauthor=T.+Babaauthor=Y.+Shi&title=Regulation+of+LSD1+histone+demethylase+activity+by+its+associated+factors&doi=10.1016%2Fj.molcel.2005.08.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of LSD1 histone demethylase activity by its associated factors</span></div><div class="casAuthors">Shi, Yu-Jiang; Matson, Caitlin; Lan, Fei; Iwase, Shigeki; Baba, Tadashi; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">857-864</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a recently identified human lysine (K)-specific histone demethylase.  LSD1 is assocd. with HDAC1/2, with CoREST, a SANT domain-contg. corepressor, and with BHC80, a PHD domain-contg. protein, among others.  We show that CoREST confers on LSD1 the ability to demethylate nucleosomal substrates and that it protects LSD1 from proteasomal degrdn. in vivo.  We find hyperacetylated nucleosomes less susceptible to CoREST/LSD1-mediated demethylation, suggesting that hypoacetylated nucleosomes may be the preferred physiol. substrates.  This raises the possibility that histone deacetylases and LSD1 may collaborate to generate a repressive chromatin environment.  Consistent with this model, TSA treatment results in derepression of LSD1 target genes.  While CoREST pos. regulates LSD1 function, BHC80 inhibits CoREST/LSD1-mediated demethylation in vitro and may therefore confer neg. regulation.  Taken together, these findings suggest that LSD1-mediated histone demethylation is regulated dynamically in vivo.  This is expected to have profound effects on gene expression under both physiol. and pathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraharVnxcaMLVg90H21EOLACvtfcHk0lh05pcT57T1oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK&md5=34ba238112151c6596dd6f6c7caa043e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BJ.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DIwase%26aufirst%3DS.%26aulast%3DBaba%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DRegulation%2520of%2520LSD1%2520histone%2520demethylase%2520activity%2520by%2520its%2520associated%2520factors%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D857%26epage%3D864%26doi%3D10.1016%2Fj.molcel.2005.08.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">You, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grozinger, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">CoREST is an integral component of the CoREST- human histone deacetylase complex</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1454</span><span class="NLM_x">–</span> <span class="NLM_lpage">1458</span><span class="refDoi"> DOI: 10.1073/pnas.98.4.1454</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1073%2Fpnas.98.4.1454" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=11171972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsVWjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=1454-1458&author=A.+Youauthor=J.+K.+Tongauthor=C.+M.+Grozingerauthor=S.+L.+Schreiber&title=CoREST+is+an+integral+component+of+the+CoREST-+human+histone+deacetylase+complex&doi=10.1073%2Fpnas.98.4.1454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">CoREST is an integral component of the CoREST-human histone deacetylase complex</span></div><div class="casAuthors">You, Angie; Tong, Jeffrey K.; Grozinger, Christina M.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1454-1458</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here we describe the components of a histone deacetylase (HDAC) complex that we term the CoREST-HDAC complex.  CoREST-HDAC is composed of polypeptides distinct from previously characterized HDAC1/2-contg. complexes such as the mSin3 and nucleosome remodeling and deacetylating (NRD, also named NURD, NuRD) complex.  Interestingly, we do not observe RbAp46 and RbAp48 in this complex, although these proteins have been obsd. in all previously identified complexes and are thought to be part of an HDAC1/2 core.  We identify the transcriptional corepressor CoREST and a protein with homol. to polyamine oxidases as components of CoREST-HDAC.  The HDAC1/2-interacting region of CoREST is mapped to a 179-aa region contg. a SANT domain, a domain found in other HDAC1/2-interacting proteins such as NCoR, MTA1, and MTA2.  Furthermore, we demonstrate that the corepressor function of CoREST depends on this region.  Although CoREST initially was cloned as a corepressor to REST (RE1 silencing transcription factor/neural restrictive silencing factor), we find no evidence for the existence of the eight-zinc finger REST transcription factor as an interacting partner in this complex; however, we do find evidence for assocn. of the putative oncogene ZNF 217 that contains eight zinc fingers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1PVa02kLm-7Vg90H21EOLACvtfcHk0ljXS7SNWw-gqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsVWjt7k%253D&md5=f584da096c4382ccf885a2d480281fba</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.98.4.1454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.98.4.1454%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DJ.%2BK.%26aulast%3DGrozinger%26aufirst%3DC.%2BM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCoREST%2520is%2520an%2520integral%2520component%2520of%2520the%2520CoREST-%2520human%2520histone%2520deacetylase%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D1454%26epage%3D1458%26doi%3D10.1073%2Fpnas.98.4.1454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Y.</span><span> </span><span class="NLM_article-title">LSD1 is a subunit of the NuRD complex and targets the metastasis programs in breast cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">660</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.05.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2009.05.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19703393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2009&pages=660-672&author=Y.+Wangauthor=H.+Zhangauthor=Y.+Chenauthor=Y.+Sunauthor=F.+Yangauthor=W.+Yuauthor=J.+Liangauthor=L.+Sunauthor=X.+Yangauthor=L.+Shiauthor=R.+Liauthor=Y.+Liauthor=Y.+Zhangauthor=Q.+Liauthor=X.+Yiauthor=Y.+Shang&title=LSD1+is+a+subunit+of+the+NuRD+complex+and+targets+the+metastasis+programs+in+breast+cancer&doi=10.1016%2Fj.cell.2009.05.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 is a subunit of the NuRD complex and targets the metastasis programs in breast cancer</span></div><div class="casAuthors">Wang, Yan; Zhang, Hua; Chen, Yupeng; Sun, Yimin; Yang, Fen; Yu, Wenhua; Liang, Jing; Sun, Luyang; Yang, Xiaohan; Shi, Lei; Li, Ruifang; Li, Yanyan; Zhang, Yu; Li, Qian; Yi, Xia; Shang, Yongfeng</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) exerts pathway-specific activity in animal development and has been linked to several high-risk cancers.  Here, we report that LSD1 is an integral component of the Mi-2/nucleosome remodeling and deacetylase (NuRD) complex.  Transcriptional target anal. revealed that the LSD1/NuRD complexes regulate several cellular signaling pathways including TGFβ1 signaling pathway that are critically involved in cell proliferation, survival, and epithelial-to-mesenchymal transition.  We demonstrated that LSD1 inhibits the invasion of breast cancer cells in vitro and suppresses breast cancer metastatic potential in vivo.  We found that LSD1 is downregulated in breast carcinomas and that its level of expression is neg. correlated with that of TGFβ1.  Our data provide a mol. basis for the interplay of histone demethylation and deacetylation in chromatin remodeling.  By enlisting LSD1, the NuRD complex expands its chromatin remodeling capacity to include ATPase, histone deacetylase, and histone demethylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpngSfM5Ree9LVg90H21EOLACvtfcHk0ljXS7SNWw-gqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73F&md5=5b62ee7d574671b5cda39a2d173b8491</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.05.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.05.050%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DYi%26aufirst%3DX.%26aulast%3DShang%26aufirst%3DY.%26atitle%3DLSD1%2520is%2520a%2520subunit%2520of%2520the%2520NuRD%2520complex%2520and%2520targets%2520the%2520metastasis%2520programs%2520in%2520breast%2520cancer%26jtitle%3DCell%26date%3D2009%26volume%3D138%26spage%3D660%26epage%3D672%26doi%3D10.1016%2Fj.cell.2009.05.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Tong, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassig, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnitzler, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingston, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">Chromatin deacetylation by an ATP-dependent nucleosome remodelling complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">395</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">921</span><span class="refDoi"> DOI: 10.1038/27699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2F27699" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=1998&pages=917-921&author=J.+K.+Tongauthor=C.+A.+Hassigauthor=G.+R.+Schnitzlerauthor=R.+E.+Kingstonauthor=S.+L.+Schreiber&title=Chromatin+deacetylation+by+an+ATP-dependent+nucleosome+remodelling+complex&doi=10.1038%2F27699"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2F27699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F27699%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DJ.%2BK.%26aulast%3DHassig%26aufirst%3DC.%2BA.%26aulast%3DSchnitzler%26aufirst%3DG.%2BR.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DChromatin%2520deacetylation%2520by%2520an%2520ATP-dependent%2520nucleosome%2520remodelling%2520complex%26jtitle%3DNature%26date%3D1998%26volume%3D395%26spage%3D917%26epage%3D921%26doi%3D10.1038%2F27699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Barski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuddapah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schones, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chepelev, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">High-resolution profiling of histone methylations in the human genome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">837</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2007.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=17512414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=823-837&author=A.+Barskiauthor=S.+Cuddapahauthor=K.+Cuiauthor=T.+Y.+Rohauthor=D.+E.+Schonesauthor=Z.+Wangauthor=G.+Weiauthor=I.+Chepelevauthor=K.+Zhao&title=High-resolution+profiling+of+histone+methylations+in+the+human+genome&doi=10.1016%2Fj.cell.2007.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution profiling of histone methylations in the human genome</span></div><div class="casAuthors">Barski, Artem; Cuddapah, Suresh; Cui, Kairong; Roh, Tae-Young; Schones, Dustin E.; Wang, Zhibin; Wei, Gang; Chepelev, Iouri; Zhao, Keji</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-837</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone modifications are implicated in influencing gene expression.  We have generated high-resoln. maps for the genome-wide distribution of 20 histone lysine and arginine methylations as well as histone variant H2A.Z, RNA polymerase II, and the insulator binding protein CTCF across the human genome using the Solexa 1G sequencing technol.  Typical patterns of histone methylations exhibited at promoters, insulators, enhancers, and transcribed regions are identified.  The monomethylations of H3K27, H3K9, H4K20, H3K79, and H2BK5 are all linked to gene activation, whereas trimethylations of H3K27, H3K9, and H3K79 are linked to repression.  H2A.Z assocs. with functional regulatory elements, and CTCF marks boundaries of histone methylation domains.  Chromosome banding patterns are correlated with unique patterns of histone modifications.  Chromosome breakpoints detected in T cell cancers frequently reside in chromatin regions assocd. with H3K4 methylations.  Our data provide new insights into the function of histone methylation and chromatin organization in genome function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxkecQmWJMZbVg90H21EOLACvtfcHk0lixWv6nVnRUiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D&md5=8e37e2d641f10e3cdd3ed51d1da82c96</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBarski%26aufirst%3DA.%26aulast%3DCuddapah%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DK.%26aulast%3DRoh%26aufirst%3DT.%2BY.%26aulast%3DSchones%26aufirst%3DD.%2BE.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DChepelev%26aufirst%3DI.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DHigh-resolution%2520profiling%2520of%2520histone%2520methylations%2520in%2520the%2520human%2520genome%26jtitle%3DCell%26date%3D2007%26volume%3D129%26spage%3D823%26epage%3D837%26doi%3D10.1016%2Fj.cell.2007.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+Mullerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+Guntherauthor=R.+Buettnerauthor=R.+Schule&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0lixWv6nVnRUiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DGunther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Nair, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brann, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tekmal, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vadlamudi, R. K.</span><span> </span><span class="NLM_article-title">PELP1 is a reader of histone H3 methylation that facilitates oestrogen receptor-alpha target gene activation by regulating lysine demethylase 1 specificity</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span><span class="refDoi"> DOI: 10.1038/embor.2010.62</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fembor.2010.62" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=438-444&author=S.+S.+Nairauthor=B.+C.+Nairauthor=V.+Cortezauthor=D.+Chakravartyauthor=E.+Metzgerauthor=R.+Schuleauthor=D.+W.+Brannauthor=R.+R.+Tekmalauthor=R.+K.+Vadlamudi&title=PELP1+is+a+reader+of+histone+H3+methylation+that+facilitates+oestrogen+receptor-alpha+target+gene+activation+by+regulating+lysine+demethylase+1+specificity&doi=10.1038%2Fembor.2010.62"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fembor.2010.62&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2010.62%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DS.%2BS.%26aulast%3DNair%26aufirst%3DB.%2BC.%26aulast%3DCortez%26aufirst%3DV.%26aulast%3DChakravarty%26aufirst%3DD.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBrann%26aufirst%3DD.%2BW.%26aulast%3DTekmal%26aufirst%3DR.%2BR.%26aulast%3DVadlamudi%26aufirst%3DR.%2BK.%26atitle%3DPELP1%2520is%2520a%2520reader%2520of%2520histone%2520H3%2520methylation%2520that%2520facilitates%2520oestrogen%2520receptor-alpha%2520target%2520gene%2520activation%2520by%2520regulating%2520lysine%2520demethylase%25201%2520specificity%26jtitle%3DEMBO%2520Rep.%26date%3D2010%26volume%3D11%26spage%3D438%26epage%3D444%26doi%3D10.1038%2Fembor.2010.62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Biswas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basrur, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elenitoba-Johnson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Function of leukemogenic mixed lineage leukemia 1 (MLL) fusion proteins through distinct partner protein complexes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">15751</span><span class="NLM_x">–</span> <span class="NLM_lpage">15756</span><span class="refDoi"> DOI: 10.1073/pnas.1111498108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1073%2Fpnas.1111498108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=15751-15756&author=D.+Biswasauthor=T.+A.+Milneauthor=V.+Basrurauthor=J.+Kimauthor=K.+S.+Elenitoba-Johnsonauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Function+of+leukemogenic+mixed+lineage+leukemia+1+%28MLL%29+fusion+proteins+through+distinct+partner+protein+complexes&doi=10.1073%2Fpnas.1111498108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111498108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111498108%26sid%3Dliteratum%253Aachs%26aulast%3DBiswas%26aufirst%3DD.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DBasrur%26aufirst%3DV.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DFunction%2520of%2520leukemogenic%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL%2529%2520fusion%2520proteins%2520through%2520distinct%2520partner%2520protein%2520complexes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D15751%26epage%3D15756%26doi%3D10.1073%2Fpnas.1111498108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozovskaia, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croce, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canaani, E.</span><span> </span><span class="NLM_article-title">ALL-1 is a histone methyltransferase that assembles a supercomplex of proteins involved in transcriptional regulation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1128</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00740-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2FS1097-2765%2802%2900740-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1119-1128&author=T.+Nakamuraauthor=T.+Moriauthor=S.+Tadaauthor=W.+Krajewskiauthor=T.+Rozovskaiaauthor=R.+Wassellauthor=G.+Duboisauthor=A.+Mazoauthor=C.+M.+Croceauthor=E.+Canaani&title=ALL-1+is+a+histone+methyltransferase+that+assembles+a+supercomplex+of+proteins+involved+in+transcriptional+regulation&doi=10.1016%2FS1097-2765%2802%2900740-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900740-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900740-2%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DTada%26aufirst%3DS.%26aulast%3DKrajewski%26aufirst%3DW.%26aulast%3DRozovskaia%26aufirst%3DT.%26aulast%3DWassell%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DG.%26aulast%3DMazo%26aufirst%3DA.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCanaani%26aufirst%3DE.%26atitle%3DALL-1%2520is%2520a%2520histone%2520methyltransferase%2520that%2520assembles%2520a%2520supercomplex%2520of%2520proteins%2520involved%2520in%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1119%26epage%3D1128%26doi%3D10.1016%2FS1097-2765%2802%2900740-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">Histone H3 lysine 4 demethylation is a target of nonselective antidepressive medications</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+lysine+4+demethylation+is+a+target+of+nonselective+antidepressive+medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 μM.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0lgjKlSas6Y_Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520lysine%25204%2520demethylation%2520is%2520a%2520target%2520of%2520nonselective%2520antidepressive%2520medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanoni, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Human histone demethylase LSD1 reads the histone code</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">41360</span><span class="NLM_x">–</span> <span class="NLM_lpage">41365</span><span class="refDoi"> DOI: 10.1074/jbc.M509549200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1074%2Fjbc.M509549200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16223729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=41360-41365&author=F.+Fornerisauthor=C.+Bindaauthor=M.+A.+Vanoniauthor=E.+Battaglioliauthor=A.+Mattevi&title=Human+histone+demethylase+LSD1+reads+the+histone+code&doi=10.1074%2Fjbc.M509549200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Human Histone Demethylase LSD1 Reads the Histone Code</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Vanoni, Maria Antonietta; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41360-41365</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human histone demethylase LSD1 is a flavin-dependent amine oxidase that catalyzes the specific removal of Me groups from mono- and dimethylated Lys4 of histone H3.  The N-terminal tail of H3 is subject to various covalent modifications, and a fundamental question in LSD1 biol. is how these epigenetic marks affect the demethylase activity.  The authors show that LSD1 does not have a strong preference for mono- or dimethylated Lys4 of H3.  Substrate recognition is not confined to the residues neighboring Lys4, but it requires a sufficiently long peptide segment consisting of the N-terminal 20 amino acids of H3.  Electrostatic interactions are an important factor in protein-substrate recognition, as indicated by the high sensitivity of Km to ionic strength.  The authors have probed LSD1 for its ability to demethylate Lys4 in presence of a second modification on the same peptide substrate.  Methylation of Lys9 does not affect enzyme catalysis.  Conversely, Lys9 acetylation causes an almost 6-fold increase in the Km value, whereas phosphorylation of Ser10 totally abolishes activity.  LSD1 is inhibited by a demethylated peptide with an inhibition const. of 1.8 μM, suggesting that LSD1 can bind to H3 independently of Lys4 methylation.  LSD1 is a chromatin-modifying enzyme, which is able to read different epigenetic marks on the histone N-terminal tail and can serve as a docking module for the stabilization of the assocd. corepressor complex(es) on chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxu9eUWof47Vg90H21EOLACvtfcHk0lgjKlSas6Y_Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L&md5=f39f7c79f1f789fb06ca7dd8a98752fe</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M509549200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M509549200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DVanoni%26aufirst%3DM.%2BA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DHuman%2520histone%2520demethylase%2520LSD1%2520reads%2520the%2520histone%2520code%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D41360%26epage%3D41365%26doi%3D10.1074%2Fjbc.M509549200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Demethylation of RB regulator MYPT1 by histone demethylase LSD1 promotes cell cycle progression in cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span><span class="NLM_x">–</span> <span class="NLM_lpage">660</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-2446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1158%2F0008-5472.CAN-10-2446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21115810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=655-660&author=H.+S.+Choauthor=T.+Suzukiauthor=N.+Dohmaeauthor=S.+Hayamiauthor=M.+Unokiauthor=M.+Yoshimatsuauthor=G.+Toyokawaauthor=M.+Takawaauthor=T.+Chenauthor=J.+K.+Kurashauthor=H.+I.+Fieldauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Demethylation+of+RB+regulator+MYPT1+by+histone+demethylase+LSD1+promotes+cell+cycle+progression+in+cancer+cells&doi=10.1158%2F0008-5472.CAN-10-2446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Demethylation of RB Regulator MYPT1 by Histone Demethylase LSD1 Promotes Cell Cycle Progression in Cancer Cells</span></div><div class="casAuthors">Cho, Hyun-Soo; Suzuki, Takehiro; Dohmae, Naoshi; Hayami, Shinya; Unoki, Motoko; Yoshimatsu, Masanori; Toyokawa, Gouji; Takawa, Masashi; Chen, Taiping; Kurash, Julia K.; Field, Helen I.; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">655-660</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 (also known as KDM1 and AOF2) is active in various cancer cells, but its biol. significance in human carcinogenesis is unexplored.  In this study, we explored hypothesized interactions between LSD1 and MYPT1, a known regulator of RB1 phosphorylation.  We found that MYPT1 was methylated in vitro and in vivo by histone lysine methyltransferase SETD7 and demethylated by LSD1, identifying Lys 442 of MYPT1 as a target for methylation/demethylation by these enzymes.  LSD1 silencing increased MYPT1 protein levels, decreasing the steady state level of phosphorylated RB1 (Ser 807/811) and reducing E2F activity.  MYPT1 methylation status influenced the affinity of MYPT1 for the ubiquitin-proteasome pathway of protein turnover.  MYPT1 was unstable in murine cells deficient in SETD7, supporting the concept that MYPT1 protein stability is physiol. regulated by methylation status.  LSD1 overexpression could activate RB1 phosphorylation by inducing a destabilization of MYPT1 protein.  Taken together, our results comprise a novel cell cycle regulatory mechanism mediated by methylation/demethylation dynamics, and they reveal the significance of LSD1 overexpression in human carcinogenesis.  Cancer Res; 71(3); 655-60.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh4fxBs2QNSrVg90H21EOLACvtfcHk0ljM4n4wGB3lpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D&md5=4812d562b79453755b7c2eb8f74cc3eb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2446%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDemethylation%2520of%2520RB%2520regulator%2520MYPT1%2520by%2520histone%2520demethylase%2520LSD1%2520promotes%2520cell%2520cycle%2520progression%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D655%26epage%3D660%26doi%3D10.1158%2F0008-5472.CAN-10-2446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espejo, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">p53 is regulated by the lysine demethylase LSD1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.1038/nature06092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature06092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=17805299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFWrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=105-108&author=J.+Huangauthor=R.+Senguptaauthor=A.+B.+Espejoauthor=M.+G.+Leeauthor=J.+A.+Dorseyauthor=M.+Richterauthor=S.+Opravilauthor=R.+Shiekhattarauthor=M.+T.+Bedfordauthor=T.+Jenuweinauthor=S.+L.+Berger&title=p53+is+regulated+by+the+lysine+demethylase+LSD1&doi=10.1038%2Fnature06092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">p53 is regulated by the lysine demethylase LSD1</span></div><div class="casAuthors">Huang, Jing; Sengupta, Roopsha; Espejo, Alexsandra B.; Lee, Min Gyu; Dorsey, Jean A.; Richter, Mario; Opravil, Susanne; Shiekhattar, Ramin; Bedford, Mark T.; Jenuwein, Thomas; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7158</span>),
    <span class="NLM_cas:pages">105-108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P53, the tumor suppressor and transcriptional activator, is regulated by numerous post-translational modifications, including lysine methylation.  Histone lysine methylation has recently been shown to be reversible; however, it is not known whether non-histone proteins are substrates for demethylation.  Here we show that, in human cells, the histone lysine-specific demethylase LSD1 (refs. 3, 4) interacts with p53 to repress p53-mediated transcriptional activation and to inhibit the role of p53 in promoting apoptosis.  We find that, in vitro, LSD1 removes both monomethylation (K370me1) and dimethylation (K370me2) at K370, a previously identified Smyd2-dependent monomethylation site.  However, in vivo, LSD1 shows a strong preference to reverse K370me2, which is performed by a distinct, but unknown, methyltransferase.  Our results indicate that K370me2 has a different role in regulating p53 from that of K370me1: K370me1 represses p53 function, whereas K370me2 promotes assocn. with the coactivator 53BP1 (p53-binding protein 1) through tandem Tudor domains in 53BP1.  Further, LSD1 represses p53 function through the inhibition of interaction of p53 with 53BP1.  These observations show that p53 is dynamically regulated by lysine methylation and demethylation and that the methylation status at a single lysine residue confers distinct regulatory output.  Lysine methylation therefore provides similar regulatory complexity for non-histone proteins and for histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ggpbeqDqsbVg90H21EOLACvtfcHk0liJGJSf4aapng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFWrs7w%253D&md5=8bf23f859f97d21bf7008a6d6e1c4946</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature06092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06092%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DEspejo%26aufirst%3DA.%2BB.%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DShiekhattar%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3Dp53%2520is%2520regulated%2520by%2520the%2520lysine%2520demethylase%2520LSD1%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D105%26epage%3D108%26doi%3D10.1038%2Fnature06092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Kontaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talianidis, I.</span><span> </span><span class="NLM_article-title">Lysine methylation regulates E2F1-induced cell death</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span><span class="refDoi"> DOI: 10.1016/j.molcel.2010.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2010.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=20603083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=152-160&author=H.+Kontakiauthor=I.+Talianidis&title=Lysine+methylation+regulates+E2F1-induced+cell+death&doi=10.1016%2Fj.molcel.2010.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methylation regulates E2F1-induced cell death</span></div><div class="casAuthors">Kontaki, Haroula; Talianidis, Iannis</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">152-160</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone-modifying enzymes can regulate DNA damage-induced apoptosis through modulation of p53 function.  Here, we show that, in p53-deficient tumor cells, Set9 and LSD1 regulate DNA damage-induced cell death in a manner opposite to that obsd. in p53+/+ cells, via modulation of E2F1 stabilization.  Set9 methylates E2F1 at lysine-185, which prevents E2F1 accumulation during DNA damage and activation of its proapoptotic target gene p73.  This Me mark is removed by LSD1, which is required for E2F1 stabilization and apoptotic function.  The mol. mechanism involves cross-talks between Lys methylation and other covalent modifications that affect E2F1 stability.  Methylation at Lys-185 inhibits acetylation and phosphorylation at distant positions and, in parallel, stimulates ubiquitination and degrdn. of the protein.  The findings illustrate that the function of methyltransferases can have opposing biol. outcomes depending on the specificity of transcription factor targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXEj98lICfQ7Vg90H21EOLACvtfcHk0liJGJSf4aapng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D&md5=12b34f33cde33be3f37a53f8115aecde</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DKontaki%26aufirst%3DH.%26aulast%3DTalianidis%26aufirst%3DI.%26atitle%3DLysine%2520methylation%2520regulates%2520E2F1-induced%2520cell%2520death%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D152%26epage%3D160%26doi%3D10.1016%2Fj.molcel.2010.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0liJGJSf4aapng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">21499</span><span class="NLM_x">–</span> <span class="NLM_lpage">21504</span><span class="refDoi"> DOI: 10.1073/pnas.1016147107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1073%2Fpnas.1016147107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21098664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=21499-21504&author=J.+Yangauthor=J.+Huangauthor=M.+Dasguptaauthor=N.+Searsauthor=M.+Miyagiauthor=B.+Wangauthor=M.+R.+Chanceauthor=X.+Chenauthor=Y.+Duauthor=Y.+Wangauthor=L.+Anauthor=Q.+Wangauthor=T.+Luauthor=X.+Zhangauthor=Z.+Wangauthor=G.+R.+Stark&title=Reversible+methylation+of+promoter-bound+STAT3+by+histone-modifying+enzymes&doi=10.1073%2Fpnas.1016147107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span></div><div class="casAuthors">Yang, Jinbo; Huang, Jing; Dasgupta, Maupali; Sears, Nathan; Miyagi, Masaru; Wang, Benlian; Chance, Mark R.; Chen, Xing; Du, Yuping; Wang, Yuxin; An, Lizhe; Wang, Qin; Lu, Tao; Zhang, Xiaodong; Wang, Zhenghe; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">21499-21504, S21499/1-S21499/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Following its tyrosine phosphorylation, STAT3 is methylated on K140 by the histone Me transferase SET9 and demethylated by LSD1 when it is bound to a subset of the promoters that it activates.  Methylation of K140 is a neg. regulatory event, because its blockade greatly increases the steady-state amt. of activated STAT3 and the expression of many (i.e., SOCS3) but not all (i.e., CD14) STAT3 target genes.  Biol. relevance is shown by the observation that overexpression of SOCS3 when K140 cannot be methylated blocks the ability of cells to activate STAT3 in response to IL-6.  K140 methylation does not occur with mutants of STAT3 that do not enter nuclei or bind to DNA.  Following treatment with IL-6, events at the SOCS3 promoter occur in an ordered sequence, as shown by chromatin immunopptns.  Y705-phosphoryl-STAT3 binds first and S727 is then phosphorylated, followed by the coincident binding of SET9 and dimethylation of K140, and lastly by the binding of LSD1.  We conclude that the lysine methylation of promoter-bound STAT3 leads to biol. important down-regulation of the dependent responses and that SET9, which is known to help provide an activating methylation mark to H3K4, is recruited to the newly activated SOCS3 promoter by STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTNJmJCGOKwbVg90H21EOLACvtfcHk0lh2eb5MLKDvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM&md5=255afed29b2b72fde8e2baf3d09a85dd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016147107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016147107%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDasgupta%26aufirst%3DM.%26aulast%3DSears%26aufirst%3DN.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DReversible%2520methylation%2520of%2520promoter-bound%2520STAT3%2520by%2520histone-modifying%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D21499%26epage%3D21504%26doi%3D10.1073%2Fpnas.1016147107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Schuleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+is+strongly+expressed+in+poorly+differentiated+neuroblastoma%3A+implications+for+therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0lh8-4gJ01LrNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520strongly%2520expressed%2520in%2520poorly%2520differentiated%2520neuroblastoma%253A%2520implications%2520for%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen receptor coactivators lysine-specific histone demethylase 1 and four and a half LIM domain protein 2 predict risk of prostate cancer recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Schuleauthor=R.+Buettner&title=Androgen+receptor+coactivators+lysine-specific+histone+demethylase+1+and+four+and+a+half+LIM+domain+protein+2+predict+risk+of+prostate+cancer+recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520receptor%2520coactivators%2520lysine-specific%2520histone%2520demethylase%25201%2520and%2520four%2520and%2520a%2520half%2520LIM%2520domain%2520protein%25202%2520predict%2520risk%2520of%2520prostate%2520cancer%2520recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Lv, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Over-expression of LSD1 promotes proliferation, migration and invasion in non-small cell lung cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e35065</span><span class="refDoi"> DOI: 10.1371/journal.pone.0035065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1371%2Fjournal.pone.0035065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e35065&author=T.+Lvauthor=D.+Yuanauthor=X.+Miaoauthor=Y.+Lvauthor=P.+Zhanauthor=X.+Shenauthor=Y.+Song&title=Over-expression+of+LSD1+promotes+proliferation%2C+migration+and+invasion+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0035065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0035065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0035065%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DT.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DOver-expression%2520of%2520LSD1%2520promotes%2520proliferation%252C%2520migration%2520and%2520invasion%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De35065%26doi%3D10.1371%2Fjournal.pone.0035065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Derr, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoesel, A. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goossens-Beumer, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sajet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dekker-Ensink, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Kruijf, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastiaannet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, V. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Velde, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuppen, P. J.</span><span> </span><span class="NLM_article-title">High nuclear expression levels of histone-modifying enzymes LSD1, HDAC2 and SIRT1 in tumor cells correlate with decreased survival and increased relapse in breast cancer patients</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="refDoi"> DOI: 10.1186/1471-2407-14-604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1186%2F1471-2407-14-604" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=604&author=R.+S.+Derrauthor=A.+Q.+van+Hoeselauthor=A.+Benardauthor=I.+J.+Goossens-Beumerauthor=A.+Sajetauthor=N.+G.+Dekker-Ensinkauthor=E.+M.+de+Kruijfauthor=E.+Bastiaannetauthor=V.+T.+Smitauthor=C.+J.+van+de+Veldeauthor=P.+J.+Kuppen&title=High+nuclear+expression+levels+of+histone-modifying+enzymes+LSD1%2C+HDAC2+and+SIRT1+in+tumor+cells+correlate+with+decreased+survival+and+increased+relapse+in+breast+cancer+patients&doi=10.1186%2F1471-2407-14-604"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-604%26sid%3Dliteratum%253Aachs%26aulast%3DDerr%26aufirst%3DR.%2BS.%26aulast%3Dvan%2BHoesel%26aufirst%3DA.%2BQ.%26aulast%3DBenard%26aufirst%3DA.%26aulast%3DGoossens-Beumer%26aufirst%3DI.%2BJ.%26aulast%3DSajet%26aufirst%3DA.%26aulast%3DDekker-Ensink%26aufirst%3DN.%2BG.%26aulast%3Dde%2BKruijf%26aufirst%3DE.%2BM.%26aulast%3DBastiaannet%26aufirst%3DE.%26aulast%3DSmit%26aufirst%3DV.%2BT.%26aulast%3Dvan%2Bde%2BVelde%26aufirst%3DC.%2BJ.%26aulast%3DKuppen%26aufirst%3DP.%2BJ.%26atitle%3DHigh%2520nuclear%2520expression%2520levels%2520of%2520histone-modifying%2520enzymes%2520LSD1%252C%2520HDAC2%2520and%2520SIRT1%2520in%2520tumor%2520cells%2520correlate%2520with%2520decreased%2520survival%2520and%2520increased%2520relapse%2520in%2520breast%2520cancer%2520patients%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D604%26doi%3D10.1186%2F1471-2407-14-604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Schuleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+%28LSD1%29+is+highly+expressed+in+ER-negative+breast+cancers+and+a+biomarker+predicting+aggressive+biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0lhwFGqHISIQrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520is%2520highly%2520expressed%2520in%2520ER-negative%2520breast%2520cancers%2520and%2520a%2520biomarker%2520predicting%2520aggressive%2520biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Yu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span> </span><span class="NLM_article-title">High expression of lysine-specific demethylase 1 correlates with poor prognosis of patients with esophageal squamous cell carcinoma</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">198</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2013.05.123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bbrc.2013.05.123" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2013&pages=192-198&author=Y.+Yuauthor=B.+Wangauthor=K.+Zhangauthor=Z.+Leiauthor=Y.+Guoauthor=H.+Xiaoauthor=J.+Wangauthor=L.+Fanauthor=C.+Lanauthor=Y.+Weiauthor=Q.+Maauthor=L.+Linauthor=C.+Maoauthor=X.+Yangauthor=X.+Chenauthor=Y.+Liauthor=Y.+Baiauthor=D.+Chen&title=High+expression+of+lysine-specific+demethylase+1+correlates+with+poor+prognosis+of+patients+with+esophageal+squamous+cell+carcinoma&doi=10.1016%2Fj.bbrc.2013.05.123"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2013.05.123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2013.05.123%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLei%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DLan%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DD.%26atitle%3DHigh%2520expression%2520of%2520lysine-specific%2520demethylase%25201%2520correlates%2520with%2520poor%2520prognosis%2520of%2520patients%2520with%2520esophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2013%26volume%3D437%26spage%3D192%26epage%3D198%26doi%3D10.1016%2Fj.bbrc.2013.05.123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Zhao, Z. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. B.</span><span> </span><span class="NLM_article-title">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span> <span class="citation_source-journal">World J. Gastroenterol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6651</span><span class="NLM_x">–</span> <span class="NLM_lpage">6656</span><span class="refDoi"> DOI: 10.3748/wjg.v18.i45.6651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.3748%2Fwjg.v18.i45.6651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=23236241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6651-6656&author=Z.+K.+Zhaoauthor=H.+F.+Yuauthor=D.+R.+Wangauthor=P.+Dongauthor=L.+Chenauthor=W.+G.+Wuauthor=W.+J.+Dingauthor=Y.+B.+Liu&title=Overexpression+of+lysine+specific+demethylase+1+predicts+worse+prognosis+in+primary+hepatocellular+carcinoma+patients&doi=10.3748%2Fwjg.v18.i45.6651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span></div><div class="casAuthors">Zhao, Ze-Kun; Yu, Hai-Feng; Wang, Dao-Rong; Dong, Ping; Chen, Lei; Wu, Wen-Guang; Ding, Wen-Jun; Liu, Ying-Bin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">6651-6656</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Aim: To investigate the clinicopathol. features and prognostic value of lysine specific demethylase 1 (LSD1) in hepatocellular carcinoma (HCC).  Methods: We examd. LSD1 expression in 60 paired liver cancer tissues and adjacent noncancerous tissues by quant. real time polymerase chain reaction (qRT-PCR) and Western blotting.  In addn., we analyzed LSD1 expression in 198 HCC samples by immunohistochem.  The relationship between LSD1 expression, clinicopathol. features and patient survival was investigated.  Results: Immunohistochem., Western blotting, and qRT-PCR consistently confirmed LSD1 overexpression in HCC tissues compared to adjacent non-neoplastic tissues (P < 0.01).  Addnl., immunostaining showed more LSD1-pos. cells in the higher tumor stage (T3-4) and tumor grade (G3) than in the lower tumor stage (T1-2, P < 0.001) and tumor grade (G1-2, P < 0.001), resp.  Moreover, HCC patients with high LSD1 expression had significantly lower 5-yr overall survival rates (P < 0.001) and lower 5-yr disease-free survival rates (P < 0.001), resp.  A Cox proportional hazards model further demonstrated that LSD1 over-expression was an independent predictor of poor prognosis for both 5-yr disease-free survival [hazards ratio (HR) = 1.426, 95%CI: 0.672-2.146, P < 0.001] and 5-yr overall survival (HR = 2.456, 95%CI: 1.234-3.932, P < 0.001) in HCC.  Conclusion: Our data suggest for the first time that the overexpression of LSD1 protein in HCC tissues indicates tumor progression and predicts poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqX4-UuO3NnLVg90H21EOLACvtfcHk0liKk-jUzj36vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ&md5=c657f6b1e0b5e183c28fefbda4cc8745</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v18.i45.6651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v18.i45.6651%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%2BK.%26aulast%3DYu%26aufirst%3DH.%2BF.%26aulast%3DWang%26aufirst%3DD.%2BR.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DW.%2BG.%26aulast%3DDing%26aufirst%3DW.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%2BB.%26atitle%3DOverexpression%2520of%2520lysine%2520specific%2520demethylase%25201%2520predicts%2520worse%2520prognosis%2520in%2520primary%2520hepatocellular%2520carcinoma%2520patients%26jtitle%3DWorld%2520J.%2520Gastroenterol%26date%3D2012%26volume%3D18%26spage%3D6651%26epage%3D6656%26doi%3D10.3748%2Fwjg.v18.i45.6651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span><span class="refDoi"> DOI: 10.1002/ijc.25349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fijc.25349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=20333681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=574-586&author=S.+Hayamiauthor=J.+D.+Kellyauthor=H.+S.+Choauthor=M.+Yoshimatsuauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=D.+E.+Nealauthor=H.+Yamaueauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Overexpression+of+LSD1+contributes+to+human+carcinogenesis+through+chromatin+regulation+in+various+cancers&doi=10.1002%2Fijc.25349"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span></div><div class="casAuthors">Hayami Shinya; Kelly John D; Cho Hyun-Soo; Yoshimatsu Masanori; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Neal David E; Yamaue Hiroki; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A number of histone demethylases have been identified and biochemically characterized, but the pathological roles of their dysfunction in human disease like cancer have not been well understood.  Here, we demonstrate important roles of lysine-specific demethylase 1 (LSD1) in human carcinogenesis.  Expression levels of LSD1 are significantly elevated in human bladder carcinomas compared with nonneoplastic bladder tissues (p < 0.0001). cDNA microarray analysis also revealed its transactivation in lung and colorectal carcinomas.  LSD1-specific small interfering RNAs significantly knocked down its expression and resulted in suppression of proliferation of various bladder and lung cancer cell lines.  Concordantly, introduction of exogenous LSD1 expression promoted cell cycle progression of human embryonic kidney fibroblast cells.  Expression profile analysis showed that LSD1 could affect the expression of genes involved in various chromatin-modifying pathways such as chromatin remodeling at centromere, centromeric heterochromatin formation and chromatin assembly, indicating its essential roles in carcinogenesis through chromatin modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WuIHSmpohpd0fW6udTcc2eb5DRd0Sq0lUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D&md5=b15e7aa7b05c05dd8cfd781ed1a7e3bd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fijc.25349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25349%26sid%3Dliteratum%253Aachs%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520LSD1%2520contributes%2520to%2520human%2520carcinogenesis%2520through%2520chromatin%2520regulation%2520in%2520various%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D574%26epage%3D586%26doi%3D10.1002%2Fijc.25349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1/KDM1A) contributes to colorectal tumorigenesis via activation of the Wnt/beta-catenin pathway by down-regulating Dickkopf-1 (DKK1) [corrected]</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e70077</span><span class="refDoi"> DOI: 10.1371/journal.pone.0070077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1371%2Fjournal.pone.0070077" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e70077&author=Z.+Huangauthor=S.+Liauthor=W.+Songauthor=X.+Liauthor=Q.+Liauthor=Z.+Zhangauthor=Y.+Hanauthor=X.+Zhangauthor=S.+Miaoauthor=R.+Duauthor=L.+Wang&title=Lysine-specific+demethylase+1+%28LSD1%2FKDM1A%29+contributes+to+colorectal+tumorigenesis+via+activation+of+the+Wnt%2Fbeta-catenin+pathway+by+down-regulating+Dickkopf-1+%28DKK1%29+%5Bcorrected%5D&doi=10.1371%2Fjournal.pone.0070077"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0070077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0070077%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMiao%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%2520contributes%2520to%2520colorectal%2520tumorigenesis%2520via%2520activation%2520of%2520the%2520Wnt%252Fbeta-catenin%2520pathway%2520by%2520down-regulating%2520Dickkopf-1%2520%2528DKK1%2529%2520%255Bcorrected%255D%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De70077%26doi%3D10.1371%2Fjournal.pone.0070077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The deubiquitinase USP28 stabilizes LSD1 and confers stem-cell-like traits to breast cancer cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1016/j.celrep.2013.08.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.celrep.2013.08.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24075993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=224-236&author=Y.+Wuauthor=Y.+Wangauthor=X.+H.+Yangauthor=T.+Kangauthor=Y.+Zhaoauthor=C.+Wangauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+deubiquitinase+USP28+stabilizes+LSD1+and+confers+stem-cell-like+traits+to+breast+cancer+cells&doi=10.1016%2Fj.celrep.2013.08.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Deubiquitinase USP28 Stabilizes LSD1 and Confers Stem-Cell-like Traits to Breast Cancer Cells</span></div><div class="casAuthors">Wu, Yadi; Wang, Yifan; Yang, Xiuwei H.; Kang, Tiebang; Zhao, Yongxiang; Wang, Chi; Evers, B. Mark; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">224-236</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a crit. chromatin modulator that controls cellular pluripotency and differentiation through the demethylation of H3K4me1/2.  Overexpression of LSD1 has been obsd. in many types of tumors and is correlated with its oncogenic effects in tumorigenesis.  However, the mechanism leading to LSD1 upregulation in tumors remains unclear.  Using an unbiased siRNA screening against all the human deubiquitinases, we identified USP28 as a bona fide deubiquitinase of LSD1.  USP28 interacted with and stabilized LSD1 via deubiquitination.  USP28 overexpression correlated with LSD1 upregulation in multiple cancer cell lines and breast tumor samples.  Knockdown of USP28 resulted in LSD1 destabilization, leading to the suppression of cancer stem cell (CSC)-like characteristics in vitro and inhibition of tumorigenicity in vivo, which can be rescued by ectopic LSD1 expression.  Our study reveals a crit. mechanism underlying the epigenetic regulation by USP28 and provides another treatment approach against breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDEa93VRia7Vg90H21EOLACvtfcHk0liEEqzzSis4EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM&md5=802c1b0561ed2a8ffa926aa4adb82405</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2013.08.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2013.08.030%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%2BH.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520deubiquitinase%2520USP28%2520stabilizes%2520LSD1%2520and%2520confers%2520stem-cell-like%2520traits%2520to%2520breast%2520cancer%2520cells%26jtitle%3DCell%2520Rep.%26date%3D2013%26volume%3D5%26spage%3D224%26epage%3D236%26doi%3D10.1016%2Fj.celrep.2013.08.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Suva, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rheinbay, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakimoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabkin, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riggi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cahill, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nahed, B. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martuza, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossetti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirosh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalek, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozenblatt-Rosen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernstein, B. E.</span><span> </span><span class="NLM_article-title">Reconstructing and reprogramming the tumor-propagating potential of glioblastoma stem-like cells</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">594</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.02.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2014.02.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24726434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVClurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=580-594&author=M.+L.+Suvaauthor=E.+Rheinbayauthor=S.+M.+Gillespieauthor=A.+P.+Patelauthor=H.+Wakimotoauthor=S.+D.+Rabkinauthor=N.+Riggiauthor=A.+S.+Chiauthor=D.+P.+Cahillauthor=B.+V.+Nahedauthor=W.+T.+Curryauthor=R.+L.+Martuzaauthor=M.+N.+Riveraauthor=N.+Rossettiauthor=S.+Kasifauthor=S.+Beikauthor=S.+Kadriauthor=I.+Tiroshauthor=I.+Wortmanauthor=A.+K.+Shalekauthor=O.+Rozenblatt-Rosenauthor=A.+Regevauthor=D.+N.+Louisauthor=B.+E.+Bernstein&title=Reconstructing+and+reprogramming+the+tumor-propagating+potential+of+glioblastoma+stem-like+cells&doi=10.1016%2Fj.cell.2014.02.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Reconstructing and Reprogramming the Tumor-Propagating Potential of Glioblastoma Stem-like Cells</span></div><div class="casAuthors">Suva, Mario L.; Rheinbay, Esther; Gillespie, Shawn M.; Patel, Anoop P.; Wakimoto, Hiroaki; Rabkin, Samuel D.; Riggi, Nicolo; Chi, Andrew S.; Cahill, Daniel P.; Nahed, Brian V.; Curry, William T.; Martuza, Robert L.; Rivera, Miguel N.; Rossetti, Nikki; Kasif, Simon; Beik, Samantha; Kadri, Sabah; Tirosh, Itay; Wortman, Ivo; Shalek, Alex K.; Rozenblatt-Rosen, Orit; Regev, Aviv; Louis, David N.; Bernstein, Bradley E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Developmental fate decisions are dictated by master transcription factors (TFs) that interact with cis-regulatory elements to direct transcriptional programs.  Certain malignant tumors may also depend on cellular hierarchies reminiscent of normal development but superimposed on underlying genetic aberrations.  In glioblastoma (GBM), a subset of stem-like tumor-propagating cells (TPCs) appears to drive tumor progression and underlie therapeutic resistance yet remain poorly understood.  Here, we identify a core set of neurodevelopmental TFs (POU3F2, SOX2, SALL2, and OLIG2) essential for GBM propagation.  These TFs coordinately bind and activate TPC-specific regulatory elements and are sufficient to fully reprogram differentiated GBM cells to "induced" TPCs, recapitulating the epigenetic landscape and phenotype of native TPCs.  We reconstruct a network model that highlights crit. interactions and identifies candidate therapeutic targets for eliminating TPCs.  Our study establishes the epigenetic basis of a developmental hierarchy in GBM, provides detailed insight into underlying gene regulatory programs, and suggests attendant therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJSQYzaStTT7Vg90H21EOLACvtfcHk0li4y28aZSykgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVClurw%253D&md5=72938847b07ffdc9e0c01e0c3cccb2ff</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.02.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.02.030%26sid%3Dliteratum%253Aachs%26aulast%3DSuva%26aufirst%3DM.%2BL.%26aulast%3DRheinbay%26aufirst%3DE.%26aulast%3DGillespie%26aufirst%3DS.%2BM.%26aulast%3DPatel%26aufirst%3DA.%2BP.%26aulast%3DWakimoto%26aufirst%3DH.%26aulast%3DRabkin%26aufirst%3DS.%2BD.%26aulast%3DRiggi%26aufirst%3DN.%26aulast%3DChi%26aufirst%3DA.%2BS.%26aulast%3DCahill%26aufirst%3DD.%2BP.%26aulast%3DNahed%26aufirst%3DB.%2BV.%26aulast%3DCurry%26aufirst%3DW.%2BT.%26aulast%3DMartuza%26aufirst%3DR.%2BL.%26aulast%3DRivera%26aufirst%3DM.%2BN.%26aulast%3DRossetti%26aufirst%3DN.%26aulast%3DKasif%26aufirst%3DS.%26aulast%3DBeik%26aufirst%3DS.%26aulast%3DKadri%26aufirst%3DS.%26aulast%3DTirosh%26aufirst%3DI.%26aulast%3DWortman%26aufirst%3DI.%26aulast%3DShalek%26aufirst%3DA.%2BK.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DLouis%26aufirst%3DD.%2BN.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26atitle%3DReconstructing%2520and%2520reprogramming%2520the%2520tumor-propagating%2520potential%2520of%2520glioblastoma%2520stem-like%2520cells%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D580%26epage%3D594%26doi%3D10.1016%2Fj.cell.2014.02.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">LSD1 inhibition: a therapeutic strategy in cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+Somervaille&title=LSD1+inhibition%3A+a+therapeutic+strategy+in+cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0li4y28aZSykgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DLSD1%2520inhibition%253A%2520a%2520therapeutic%2520strategy%2520in%2520cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Niebel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majores, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutgemann, I.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) in hematopoietic and lymphoid neoplasms</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span><span class="refDoi"> DOI: 10.1182/blood-2014-04-569525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1182%2Fblood-2014-04-569525" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=151-152&author=D.+Niebelauthor=J.+Kirfelauthor=V.+Janzenauthor=T.+Hollerauthor=M.+Majoresauthor=I.+Gutgemann&title=Lysine-specific+demethylase+1+%28LSD1%29+in+hematopoietic+and+lymphoid+neoplasms&doi=10.1182%2Fblood-2014-04-569525"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-04-569525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-04-569525%26sid%3Dliteratum%253Aachs%26aulast%3DNiebel%26aufirst%3DD.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DV.%26aulast%3DHoller%26aufirst%3DT.%26aulast%3DMajores%26aufirst%3DM.%26aulast%3DGutgemann%26aufirst%3DI.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520in%2520hematopoietic%2520and%2520lymphoid%2520neoplasms%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D151%26epage%3D152%26doi%3D10.1182%2Fblood-2014-04-569525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">The histone demethylase KDM1A sustains the oncogenic potential of MLL-AF9 leukemia stem cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+Somervaille&title=The+histone+demethylase+KDM1A+sustains+the+oncogenic+potential+of+MLL-AF9+leukemia+stem+cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lhsQ5oAZfRW-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DThe%2520histone%2520demethylase%2520KDM1A%2520sustains%2520the%2520oncogenic%2520potential%2520of%2520MLL-AF9%2520leukemia%2520stem%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Benelkebir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgkinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duriez, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulleid, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">Enantioselective synthesis of tranylcypromine analogues as lysine demethylase (LSD1) inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3709</span><span class="NLM_x">–</span> <span class="NLM_lpage">3716</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bmc.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21382717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3709-3716&author=H.+Benelkebirauthor=C.+Hodgkinsonauthor=P.+J.+Duriezauthor=A.+L.+Haydenauthor=R.+A.+Bulleidauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Ganesan&title=Enantioselective+synthesis+of+tranylcypromine+analogues+as+lysine+demethylase+%28LSD1%29+inhibitors&doi=10.1016%2Fj.bmc.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective synthesis of tranylcypromine analogues as lysine demethylase (LSD1) inhibitors</span></div><div class="casAuthors">Benelkebir, Hanae; Hodgkinson, Christopher; Duriez, Patrick J.; Hayden, Annette L.; Bulleid, Rosemary A.; Crabb, Simon J.; Packham, Graham; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3709-3716</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Asym. cyclopropanation of styrenes by tert-Bu diazoacetate followed by ester hydrolysis and Curtius rearrangement gave a series of tranylcypromine analogs as single enantiomers.  The o-, m-, and p-bromo analogs were all more active than tranylcypromine in a LSD1 enzyme assay.  The m- and p-bromo analogs were micromolar growth inhibitors of the LNCaP prostate cancer cell line as were the corresponding biphenyl analogs prepd. from the bromide by Suzuki crosscoupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuHGveNGE6AbVg90H21EOLACvtfcHk0lhsQ5oAZfRW-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D&md5=a8517cf8753c76b1403b6354977fc2a2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DBenelkebir%26aufirst%3DH.%26aulast%3DHodgkinson%26aufirst%3DC.%26aulast%3DDuriez%26aufirst%3DP.%2BJ.%26aulast%3DHayden%26aufirst%3DA.%2BL.%26aulast%3DBulleid%26aufirst%3DR.%2BA.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEnantioselective%2520synthesis%2520of%2520tranylcypromine%2520analogues%2520as%2520lysine%2520demethylase%2520%2528LSD1%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3709%26epage%3D3716%26doi%3D10.1016%2Fj.bmc.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Gooden, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabadi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3047</span><span class="NLM_x">–</span> <span class="NLM_lpage">3051</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bmcl.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=18242989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3047-3051&author=D.+M.+Goodenauthor=D.+M.+Schmidtauthor=J.+A.+Pollockauthor=A.+M.+Kabadiauthor=D.+G.+McCafferty&title=Facile+synthesis+of+substituted+trans-2-arylcyclopropylamine+inhibitors+of+the+human+histone+demethylase+LSD1+and+monoamine+oxidases+A+and+B&doi=10.1016%2Fj.bmcl.2008.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span></div><div class="casAuthors">Gooden, David M.; Schmidt, Dawn M. Z.; Pollock, Julie A.; Kabadi, Ami M.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3047-3051</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A facile synthetic route to substituted trans-2-arylcyclopropylamines was developed to provide access to mechanism-based inhibitors of the human flavoenzyme oxidase lysine-specific histone demethylase LSD1 and related enzyme family members such as monoamine oxidases A and B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt7YjHKC3yrVg90H21EOLACvtfcHk0lgTP-b9r6dxtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D&md5=2459dcb4cde160186fc17c09258c5906</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGooden%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DPollock%26aufirst%3DJ.%2BA.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DFacile%2520synthesis%2520of%2520substituted%2520trans-2-arylcyclopropylamine%2520inhibitors%2520of%2520the%2520human%2520histone%2520demethylase%2520LSD1%2520and%2520monoamine%2520oxidases%2520A%2520and%2520B%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3047%26epage%3D3051%26doi%3D10.1016%2Fj.bmcl.2008.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Hojfeldt, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Histone lysine demethylases as targets for anticancer therapy</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">930</span><span class="refDoi"> DOI: 10.1038/nrd4154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrd4154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24232376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=917-930&author=J.+W.+Hojfeldtauthor=K.+Aggerauthor=K.+Helin&title=Histone+lysine+demethylases+as+targets+for+anticancer+therapy&doi=10.1038%2Fnrd4154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases as targets for anticancer therapy</span></div><div class="casAuthors">Hojfeldt, Jonas W.; Agger, Karl; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-930</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It has recently been demonstrated that the genes controlling the epigenetic programs that are required for maintaining chromatin structure and cell identity include genes that drive human cancer.  This observation has led to an increased awareness of chromatin-assocd. proteins as potentially interesting drug targets.  The successful introduction of DNA methylation and histone deacetylase (HDAC) inhibitors for the treatment of specific subtypes of cancer has paved the way for the use of epigenetic therapy.  Here, we highlight key biol. findings demonstrating the roles of members of the histone lysine demethylase class of enzymes in the development of cancers, discuss the potential and challenges of therapeutically targeting them, and highlight emerging small-mol. inhibitors of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThyggRtM7qrVg90H21EOLACvtfcHk0lgTP-b9r6dxtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP&md5=b120d43a0353078b016c8dc8f7cac644</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrd4154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4154%26sid%3Dliteratum%253Aachs%26aulast%3DHojfeldt%26aufirst%3DJ.%2BW.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DHistone%2520lysine%2520demethylases%2520as%2520targets%2520for%2520anticancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D917%26epage%3D930%26doi%3D10.1038%2Fnrd4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">An overview of phenylcyclopropylamine derivatives: biochemical and biological significance and recent developments</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">910</span><span class="refDoi"> DOI: 10.1002/med.21269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fmed.21269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22893613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=873-910&author=M.+N.+Khanauthor=T.+Suzukiauthor=N.+Miyata&title=An+overview+of+phenylcyclopropylamine+derivatives%3A+biochemical+and+biological+significance+and+recent+developments&doi=10.1002%2Fmed.21269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span></div><div class="casAuthors">Khan, Mohammed Naseer Ahmed; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-910</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Trans-2-Phencylcyclopropylamine (2-PCPA), a potent, clin. used antidepressant, affects monoamine neurotransmitter levels by inhibiting the main metabolizing enzymes, monoamine oxidases (MAOs).  However, the antidepressant action of this compd. was not fully explained by its effects on MAOs due to its wide variety of biol. effects. 2-PCPA also affects depression-assocd. pathophysiol. pathways, and linked with increased levels of trace amines in brain, upregulation of GABAB receptors (where GABA is gamma amino butyric acid), modulation of phospholipid metab., and interference with various cytochrome P 450 (CYP) enzymes.  Consequently, despite its adverse effects and limited clin. applicability, 2-PCPA has attracted interest as a structural scaffold for the development of mechanism-based inhibitors of various enzymes, including lysine-specific demethylase 1 (LSD1), which is a possible target for cancer chemotherapy.  In the recent years, many reports have appeared in the literature based on 2-PCPA scaffold and their potential medicinal implications.  This review mainly focuses on the medicinal chem. aspects including drug design, structure-activity relationships (SAR), biol. and biochem. properties, and mechanism of actions of 2-PCPA and its derivs.  Furthermore, we also highlight recent advance in this area and discuss their future applications for beneficial therapeutic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZJFRem7GlrVg90H21EOLACvtfcHk0lhpMKIeokO59w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D&md5=d257f3b954efb33ae6f824c254ae1c30</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fmed.21269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21269%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DAn%2520overview%2520of%2520phenylcyclopropylamine%2520derivatives%253A%2520biochemical%2520and%2520biological%2520significance%2520and%2520recent%2520developments%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D873%26epage%3D910%26doi%3D10.1002%2Fmed.21269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally designed trans-2-phenylcyclopropylamine derivatives potently inhibit histone demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span><span class="refDoi"> DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+designed+trans-2-phenylcyclopropylamine+derivatives+potently+inhibit+histone+demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lhpMKIeokO59w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520designed%2520trans-2-phenylcyclopropylamine%2520derivatives%2520potently%2520inhibit%2520histone%2520demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+Inhibitors+of+LSD1+as+Therapeutic+Agents+in+Acute+Myeloid+Leukemia%3A+Clinical+Significance+and+Progress+to+Date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0ljRBIJUAfvBcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520Inhibitors%2520of%2520LSD1%2520as%2520Therapeutic%2520Agents%2520in%2520Acute%2520Myeloid%2520Leukemia%253A%2520Clinical%2520Significance%2520and%2520Progress%2520to%2520Date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuhara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1-selective inactivators: protein-targeted drug delivery mechanism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">8620</span><span class="NLM_x">–</span> <span class="NLM_lpage">8624</span><span class="refDoi"> DOI: 10.1002/anie.201303999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fanie.201303999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=8620-8624&author=D.+Ogasawaraauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Kakizawaauthor=K.+Minoauthor=K.+Fukuharaauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Lysine-specific+demethylase+1-selective+inactivators%3A+protein-targeted+drug+delivery+mechanism&doi=10.1002%2Fanie.201303999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1-selective inactivators: Protein-targeted drug delivery mechanism</span></div><div class="casAuthors">Ogasawara, Daisuke; Itoh, Yukihiro; Tsumoto, Hiroki; Kakizawa, Taeko; Mino, Koshiki; Fukuhara, Kiyoshi; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">8620-8624</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of novel LSD1 (LSD1 = lysine-specific demethylase) inactivators based on new concept of LSD1-targeted PCPA (PCPA = trans-2-phenylcyclopropylamine) delivery have been designed.  This concept was tested with peptide 1a [1a = (ART-X-QTARKSTGGKAPRKQLA)] X = (I) and confirmed that 1a does indeed inhibit LSD1 selectively and efficiently by delivering PCPA directly to the LSD1 active site.  The enzyme is then inactivated by FAD-PCPA adduct formation, as in the case of PCPA itself.  This strategy could be applied to the design of nonpeptide, small-mol. LSD1 inactivators.  Mechanistic studies and biol. evaluations revealed that these small-mol. compds. are potent LSD1-selective inactivators (comparable to peptide 1a) in enzyme assays, and they also exhibit potent cell growth inhibitory activities against cancer cell lines.  This is the first time that a drug-carrier-type system has been employed to target a specific mol.  These novel LSD1-selective inactivators are considered to be candidates for anticancer agents, as well as bioprobes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQOLhLwmv9rVg90H21EOLACvtfcHk0ljRBIJUAfvBcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF&md5=71bd05a50aeb73431444f2588fba2dac</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303999%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DLysine-specific%2520demethylase%25201-selective%2520inactivators%253A%2520protein-targeted%2520drug%2520delivery%2520mechanism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D8620%26epage%3D8624%26doi%3D10.1002%2Fanie.201303999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Pieroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annunziato, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzali, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beato, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, G.</span><span> </span><span class="NLM_article-title">Further insights into the SAR of alpha-substituted cyclopropylamine derivatives as inhibitors of histone demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.12.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ejmech.2014.12.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25585008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2015&pages=377-386&author=M.+Pieroniauthor=G.+Annunziatoauthor=E.+Azzaliauthor=P.+Dessantiauthor=C.+Mercurioauthor=G.+Meroniauthor=P.+Trifiroauthor=P.+Vianelloauthor=M.+Villaauthor=C.+Beatoauthor=M.+Varasiauthor=G.+Costantino&title=Further+insights+into+the+SAR+of+alpha-substituted+cyclopropylamine+derivatives+as+inhibitors+of+histone+demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.12.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Further insights into the SAR of α-substituted cyclopropylamine derivatives as inhibitors of histone demethylase KDM1A</span></div><div class="casAuthors">Pieroni, Marco; Annunziato, Giannamaria; Azzali, Elisa; Dessanti, Paola; Mercurio, Ciro; Meroni, Giuseppe; Trifiro, Paolo; Vianello, Paola; Villa, Manuela; Beato, Claudia; Varasi, Mario; Costantino, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">377-386</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Epigenetics alterations including histone methylation and acetylation, and DNA methylation, are thought to play important roles in the onset and progression of cancer in numerous tumor cell lines.  Lysine-specific demethylase 1 (LSD1 or KDM1A) is highly expressed in different cancer types and inhibiting KDM1A activity seems to have high therapeutic potential in cancer treatment.  In the recent years, several inhibitors of KDM1A have been prepd. and disclosed.  The majority of these derivs. were designed based on the structure of tranylcypromine, as the cyclopropane core is responsible for the covalent interaction between the inhibitor and the catalytic domain of KDM proteins.  In this study, we have further extended the SAR regarding compds. 1a-e, which were recently found to inhibit KDM1A with good activity.  The decoration of the Ph ring at the β-position of the cyclopropane ring with small functional groups, mostly halogenated, and in particular at the meta position, led to a significant improvement of the inhibitory activity against KDM1A, as exemplified by compd. 44a, which has a potency in the low nanomolar range (31 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSQyHWyX2UZLVg90H21EOLACvtfcHk0liA26c7YTNn-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGnuw%253D%253D&md5=76f37a0eeb69d3962a82fd4aab95cc8f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.12.032%26sid%3Dliteratum%253Aachs%26aulast%3DPieroni%26aufirst%3DM.%26aulast%3DAnnunziato%26aufirst%3DG.%26aulast%3DAzzali%26aufirst%3DE.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DBeato%26aufirst%3DC.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DCostantino%26aufirst%3DG.%26atitle%3DFurther%2520insights%2520into%2520the%2520SAR%2520of%2520alpha-substituted%2520cyclopropylamine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D92%26spage%3D377%26epage%3D386%26doi%3D10.1016%2Fj.ejmech.2014.12.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Johnson, N. W.; Kasparec, J.; Miller, W. H.; Rouse, M.; Suarez, D.; Tian, X.</span><span> </span><span class="NLM_article-title">Cyclopropylamines as LSD1 Inhibitors</span>. WO2012/135113, Oct 4,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=N.+W.+Johnson&author=J.+Kasparec&author=W.+H.+Miller&author=M.+Rouse&author=D.+Suarez&author=X.+Tian&title=Cyclopropylamines+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26atitle%3DCyclopropylamines%2520as%2520LSD1%2520Inhibitors%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Maes, T.; Martinell Pedemonte, M.</span><span> </span><span class="NLM_article-title">Lysine Demethylase Inhibitors for Thrombosis and Cardiovascular Diseases</span>. WO2012/156537, Nov 22,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=T.+Maes&author=M.+Martinell+Pedemonte&title=Lysine+Demethylase+Inhibitors+for+Thrombosis+and+Cardiovascular+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26atitle%3DLysine%2520Demethylase%2520Inhibitors%2520for%2520Thrombosis%2520and%2520Cardiovascular%2520Diseases%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Maes, T.; Martinell Pedemonte, M.</span><span> </span><span class="NLM_article-title">Lysine Demethylase Inhibitors for Inflammatory Diseases or Conditions</span>. WO2012/156531, Nov 22,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=T.+Maes&author=M.+Martinell+Pedemonte&title=Lysine+Demethylase+Inhibitors+for+Inflammatory+Diseases+or+Conditions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26atitle%3DLysine%2520Demethylase%2520Inhibitors%2520for%2520Inflammatory%2520Diseases%2520or%2520Conditions%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Ortega Munoz, A.; Fyfe, M. C. T.; Estiarte Martinez, M. d. L. A.; Valls Vidal, N.; Kurz, G.; Castro Palomino, L. J. C.</span><span> </span><span class="NLM_article-title">(Hetero)Aryl Cyclopropylamine Compounds as LSD1 Inhibitors</span>. WO2013/057320, Apr 25,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Ortega+Munoz&author=M.+C.+T.+Fyfe&author=M.+d.+L.+A.+Estiarte+Martinez&author=N.+Valls+Vidal&author=G.+Kurz&author=L.+J.+C.+Castro+Palomino&title=%28Hetero%29Aryl+Cyclopropylamine+Compounds+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%2BMunoz%26aufirst%3DA.%26atitle%3D%2528Hetero%2529Aryl%2520Cyclopropylamine%2520Compounds%2520as%2520LSD1%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Ortega Munoz, A.; Fyfe, M. C. T.; Martinell Pedemonte, M.; Estiarte Martinez, M. d. L. A.; Valls Vidal, N.; Kurz, G.; Castro Palomino, L. J. C.</span><span> </span><span class="NLM_article-title">(Hetero)Aryl Cyclopropylamine Compounds as LSD1 Inhibitors</span>. WO2013/057322, Apr 25,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Ortega+Munoz&author=M.+C.+T.+Fyfe&author=M.+Martinell+Pedemonte&author=M.+d.+L.+A.+Estiarte+Martinez&author=N.+Valls+Vidal&author=G.+Kurz&author=L.+J.+C.+Castro+Palomino&title=%28Hetero%29Aryl+Cyclopropylamine+Compounds+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%2BMunoz%26aufirst%3DA.%26atitle%3D%2528Hetero%2529Aryl%2520Cyclopropylamine%2520Compounds%2520as%2520LSD1%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Tomita, N.; Shigeo, K.</span><span> </span><span class="NLM_article-title">Cyclopropaneamine Compound</span>. WO2013/022047, Feb 14,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=N.+Tomita&author=K.+Shigeo&title=Cyclopropaneamine+Compound"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTomita%26aufirst%3DN.%26atitle%3DCyclopropaneamine%2520Compound%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span>A phase I study of human pharmacokinetics and safety of ORY-1001, an LSD1 inhibitor, in relapsed or refractory acute leukaemia (AL). EudraCT number 2013-002447-29. <a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES" class="extLink">https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=A+phase+I+study+of+human+pharmacokinetics+and+safety+of+ORY-1001%2C+an+LSD1+inhibitor%2C+in+relapsed+or+refractory+acute+leukaemia+%28AL%29.+EudraCT+number+2013-002447-29.+https%3A%2F%2Fwww.clinicaltrialsregister.eu%2Fctr-search%2Ftrial%2F2013-002447-29%2FES."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Maes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carceller, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buesa, C.</span><span> </span><span class="NLM_article-title">Advances in the development of histone lysine demethylase inhibitors</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.coph.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.coph.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=26057211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=52-60&author=T.+Maesauthor=E.+Carcellerauthor=J.+Salasauthor=A.+Ortegaauthor=C.+Buesa&title=Advances+in+the+development+of+histone+lysine+demethylase+inhibitors&doi=10.1016%2Fj.coph.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of histone lysine demethylase inhibitors</span></div><div class="casAuthors">Maes, Tamara; Carceller, Elena; Salas, Jordi; Ortega, Alberto; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The covalent modification of histones is closely assocd. with regulation of gene transcription.  Chromatin modifications have been suggested to represent an epigenetic code that is dynamically 'written' and 'erased' by specialized proteins, and 'read', or interpreted, by proteins that translate the code into gene expression changes.  Initially thought to be an irreversible process, histone methylation is now known to be reversed by demethylases, FAD dependent amineoxidases and by iron(II)-alpha-ketoglutarate dependent deoxygenases of the Jumonji family.  Altered histone demethylase activities have been assocd. with human disease, including cancer.  The first wave of novel investigational drugs directed against KDM1A has recently entered the clinic, and the first specific inhibitor targeting a Jumonji KDM is advancing in preclin. regulatory studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCr97IlCYFrVg90H21EOLACvtfcHk0liTIOCZxklpPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D&md5=02e8a5f8d426743be1eb209c329bd3ed</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DSalas%26aufirst%3DJ.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.coph.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">McGrath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span> </span><span class="NLM_article-title">Targeting histone lysine methylation in cancer</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.pharmthera.2015.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25578037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=1-22&author=J.+McGrathauthor=P.+Trojer&title=Targeting+histone+lysine+methylation+in+cancer&doi=10.1016%2Fj.pharmthera.2015.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine methylation in cancer</span></div><div class="casAuthors">McGrath, John; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-22</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Within the vast landscape of histone modifications lysine methylation has gained increasing attention because of its profound regulatory potential.  The methylation of lysine residues on histone proteins modulates chromatin structure and thereby contributes to the regulation of DNA-based nuclear processes such as transcription, replication and repair.  Protein families with opposing catalytic activities, lysine methyltransferases (KMTs) and demethylases (KDMs), dynamically control levels of histone lysine methylation and individual enzymes within these families have become candidate oncol. targets in recent years.  A no. of high quality small mol. inhibitors of these enzymes have been identified.  Several of these compds. elicit selective cancer cell killing in vitro and robust efficacy in vivo, suggesting that targeting 'histone lysine methylation pathways' may be a relevant, emerging cancer therapeutic strategy.  Here, we discuss individual histone lysine methylation pathway targets, the properties of currently available small mol. inhibitors and their application in the context of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogqKUpTpHisrVg90H21EOLACvtfcHk0liTIOCZxklpPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D&md5=225ece90384f762b248d2d7981fa120e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcGrath%26aufirst%3DJ.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DTargeting%2520histone%2520lysine%2520methylation%2520in%2520cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22%26doi%3D10.1016%2Fj.pharmthera.2015.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span>Investigation of GSK2879552 in Subjects with Relapsed/Refractory Small Cell Lung Carcinoma. <a href="https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&rank=1</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Investigation+of+GSK2879552+in+Subjects+with+Relapsed%2FRefractory+Small+Cell+Lung+Carcinoma.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02034123%3Fterm%3DGSK2879552%26rank%3D1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span>Phase I Dose Escalation Study for GSK2879552 in Subjects with Acute Myeloid Leukemia (AML). ClinicalTrials.gov Identifier: NCT02177812. <a href="https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&amp;rank=2" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&rank=2</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Phase+I+Dose+Escalation+Study+for+GSK2879552+in+Subjects+with+Acute+Myeloid+Leukemia+%28AML%29.+ClinicalTrials.gov+Identifier%3A+NCT02177812.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02177812%3Fterm%3DGSK2879552%26rank%3D2."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Mohammad, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smitheman, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamat, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federowicz, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneck, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butticello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnette, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degenhardt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasparec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNulty, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouthamel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span> </span><span class="NLM_article-title">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+Hypomethylation+Signature+Predicts+Antitumor+Activity+of+LSD1+Inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lg26LmX1GeO4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520Hypomethylation%2520Signature%2520Predicts%2520Antitumor%2520Activity%2520of%2520LSD1%2520Inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible inhibitors of LSD1 as therapeutic agents in acute myeloid leukemia: clinical significance and progress to date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+inhibitors+of+LSD1+as+therapeutic+agents+in+acute+myeloid+leukemia%3A+clinical+significance+and+progress+to+date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lj2sBoyAfyzCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520inhibitors%2520of%2520LSD1%2520as%2520therapeutic%2520agents%2520in%2520acute%2520myeloid%2520leukemia%253A%2520clinical%2520significance%2520and%2520progress%2520to%2520date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lj2sBoyAfyzCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saponara, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrocco, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monaldi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruoppolo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilset, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Pure enantiomers of benzoylamino-tranylcypromine: LSD1 inhibition, gene modulation in human leukemia cells and effects on clonogenic potential of murine promyelocytic blasts</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">174</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2015.02.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ejmech.2015.02.060" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=163-174&author=S.+Valenteauthor=V.+Rodriguezauthor=C.+Mercurioauthor=P.+Vianelloauthor=B.+Saponaraauthor=R.+Cirilliauthor=G.+Ciossaniauthor=D.+Labellaauthor=B.+Marroccoauthor=D.+Monaldiauthor=G.+Ruoppoloauthor=M.+Tilsetauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=A.+Matteviauthor=M.+Varasiauthor=A.+Mai&title=Pure+enantiomers+of+benzoylamino-tranylcypromine%3A+LSD1+inhibition%2C+gene+modulation+in+human+leukemia+cells+and+effects+on+clonogenic+potential+of+murine+promyelocytic+blasts&doi=10.1016%2Fj.ejmech.2015.02.060"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.02.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.02.060%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRodriguez%26aufirst%3DV.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DSaponara%26aufirst%3DB.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DMonaldi%26aufirst%3DD.%26aulast%3DRuoppolo%26aufirst%3DG.%26aulast%3DTilset%26aufirst%3DM.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DPure%2520enantiomers%2520of%2520benzoylamino-tranylcypromine%253A%2520LSD1%2520inhibition%252C%2520gene%2520modulation%2520in%2520human%2520leukemia%2520cells%2520and%2520effects%2520on%2520clonogenic%2520potential%2520of%2520murine%2520promyelocytic%2520blasts%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D163%26epage%3D174%26doi%3D10.1016%2Fj.ejmech.2015.02.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Bujok, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwast, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cmoch, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wróbel, Z.</span><span> </span><span class="NLM_article-title">Novel approach to synthesis of substituted 3-aminoquinolines from nitroarenes and protected ethyl aminocrotonate</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">708</span><span class="refDoi"> DOI: 10.1016/j.tet.2009.11.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.tet.2009.11.060" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=698-708&author=R.+Bujokauthor=A.+Kwastauthor=P.+Cmochauthor=Z.+Wr%C3%B3bel&title=Novel+approach+to+synthesis+of+substituted+3-aminoquinolines+from+nitroarenes+and+protected+ethyl+aminocrotonate&doi=10.1016%2Fj.tet.2009.11.060"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2009.11.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2009.11.060%26sid%3Dliteratum%253Aachs%26aulast%3DBujok%26aufirst%3DR.%26aulast%3DKwast%26aufirst%3DA.%26aulast%3DCmoch%26aufirst%3DP.%26aulast%3DWr%25C3%25B3bel%26aufirst%3DZ.%26atitle%3DNovel%2520approach%2520to%2520synthesis%2520of%2520substituted%25203-aminoquinolines%2520from%2520nitroarenes%2520and%2520protected%2520ethyl%2520aminocrotonate%26jtitle%3DTetrahedron%26date%3D2010%26volume%3D66%26spage%3D698%26epage%3D708%26doi%3D10.1016%2Fj.tet.2009.11.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacCoss, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawson, A. D.</span><span> </span><span class="NLM_article-title">Rings in drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">5845</span><span class="NLM_x">–</span> <span class="NLM_lpage">5859</span><span class="refDoi"> DOI: 10.1021/jm4017625</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4017625" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5845-5859&author=R.+D.+Taylorauthor=M.+MacCossauthor=A.+D.+Lawson&title=Rings+in+drugs&doi=10.1021%2Fjm4017625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Rings in Drugs</span></div><div class="casAuthors">Taylor, Richard D.; MacCoss, Malcolm; Lawson, Alastair D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5845-5859</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  We have analyzed the rings, ring systems, and frameworks in drugs listed in the FDA Orange Book to understand the frequency, time-lines, mol. property space, and the application of these rings in different therapeutic areas and target classes.  This anal. shows that there are only 351 ring systems and 1197 frameworks in drugs that came onto the market before 2013.  Furthermore, on av. six new ring systems enter drug space each year and approx. 28% of new drugs contain a new ring system.  Moreover, it is very unusual for a drug to contain more than one new ring system and the majority of the most frequently used ring systems (83%) were first used in drugs developed prior to 1983.  These observations give insight into the chem. novelty of drugs and potentially efficient ways to assess compd. libraries and develop compds. from hit identification to lead optimization and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC_e_L4z4XCrVg90H21EOLACvtfcHk0lirGQojhFpIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D&md5=c84ce84b8698eb6fad38fa82b0ab4912</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm4017625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4017625%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DA.%2BD.%26atitle%3DRings%2520in%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5845%26epage%3D5859%26doi%3D10.1021%2Fjm4017625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newton-Vinson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubalek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structure of human monoamine oxidase B, a drug target for the treatment of neurological disorders</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1038/nsb732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnsb732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=11753429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD38XitV2rsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=22-26&author=C.+Bindaauthor=P.+Newton-Vinsonauthor=F.+Hubalekauthor=D.+E.+Edmondsonauthor=A.+Mattevi&title=Structure+of+human+monoamine+oxidase+B%2C+a+drug+target+for+the+treatment+of+neurological+disorders&doi=10.1038%2Fnsb732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human monoamine oxidase B, a drug target for the treatment of neurological disorders</span></div><div class="casAuthors">Binda, Claudia; Newton-Vinson, Paige; Hubalek, Frantisek; Edmondson, Dale E.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-26</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Monoamine oxidase B (MAO B) is a mitochondrial outer membrane flavoenzyme that is a well-known target for antidepressant and neuroprotective drugs.  Here, the authors detd. the structure of the human liver enzyme to 3 Å resoln.  Human liver MAO B was found to bind to the membrane through a C-terminal transmembrane helix and apolar loops located at various positions in the sequence.  The electron d. showed that pargyline, an analog of the clin. used MAO B inhibitor, deprenyl, binds covalently to the flavin N5 atom.  The active site of MAO B consisted of a 420-Å3 hydrophobic substrate cavity interconnected to an entrance cavity of 290 Å3.  The recognition site for the substrate amino group was an arom. cage formed by Tyr-398 and Tyr-435.  The structure provides a framework for probing the catalytic mechanism, understanding the differences between the MAO A and B isoforms and designing specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8p43h-vsPi7Vg90H21EOLACvtfcHk0lirGQojhFpIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitV2rsg%253D%253D&md5=261d3b1375d833bbfd0a23bdade49be6</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnsb732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb732%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DNewton-Vinson%26aufirst%3DP.%26aulast%3DHubalek%26aufirst%3DF.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructure%2520of%2520human%2520monoamine%2520oxidase%2520B%252C%2520a%2520drug%2520target%2520for%2520the%2520treatment%2520of%2520neurological%2520disorders%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D22%26epage%3D26%26doi%3D10.1038%2Fnsb732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span> </span><span class="NLM_article-title">Synthesis, biological activity and mechanistic insights of 1-substituted cyclopropylamine derivatives: a novel class of irreversible inhibitors of histone demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.08.068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ejmech.2014.08.068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25173853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=352-363&author=P.+Vianelloauthor=O.+A.+Botrugnoauthor=A.+Cappaauthor=G.+Ciossaniauthor=P.+Dessantiauthor=A.+Maiauthor=A.+Matteviauthor=G.+Meroniauthor=S.+Minucciauthor=F.+Thalerauthor=M.+Tortoriciauthor=P.+Trifiroauthor=S.+Valenteauthor=M.+Villaauthor=M.+Varasiauthor=C.+Mercurio&title=Synthesis%2C+biological+activity+and+mechanistic+insights+of+1-substituted+cyclopropylamine+derivatives%3A+a+novel+class+of+irreversible+inhibitors+of+histone+demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.08.068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological activity and mechanistic insights of 1-substituted cyclopropylamine derivatives: A novel class of irreversible inhibitors of histone demethylase KDM1A</span></div><div class="casAuthors">Vianello, Paola; Botrugno, Oronza A.; Cappa, Anna; Ciossani, Giuseppe; Dessanti, Paola; Mai, Antonello; Mattevi, Andrea; Meroni, Giuseppe; Minucci, Saverio; Thaler, Florian; Tortorici, Marcello; Trifiro, Paolo; Valente, Sergio; Villa, Manuela; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">352-363</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Histone demethylase KDM1A [i.e., gene LSD1, histone H3 lysine-4 demethylase LSD1] has become an attractive therapeutic target for the treatment of cancer as well as other disorders such as viral infections.  The authors report on the synthesis of compds. derived from the expansion of tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  These compds., which are substituted on the cyclopropyl core moiety, were evaluated for their ability to inhibit KDM1A in vitro as well as to function in cells by modulating the expression of Gfi-1b, a well recognized KDM1A target gene.  The mols. were all found to covalently inhibit KDM1A and to become increasingly selective against human monoamine oxidase MAO A and MAO B through the introduction of bulkier substituents on the cyclopropylamine ring.  Structural and biochem. anal. of selected trans-isomers showed that the two stereoisomers are endowed with similar inhibitory activities against KDM1A, but form different covalent adducts with FAD co-enzyme [i.e., riboflavin 5'-(trihydrogen diphosphate) P'→5'-ester with adenosine, FAD].  The synthesis of the target compds. was achieved using (2S)-2-(phenyl)oxirane, (2R)-2-(phenyl)oxirane, racemic (phenyl)oxirane as starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEJ-thHP_b6rVg90H21EOLACvtfcHk0lg-IWrandlGTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP&md5=6285108e113c25939e3aa0b1ee5390c6</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.08.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.08.068%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DSynthesis%252C%2520biological%2520activity%2520and%2520mechanistic%2520insights%2520of%25201-substituted%2520cyclopropylamine%2520derivatives%253A%2520a%2520novel%2520class%2520of%2520irreversible%2520inhibitors%2520of%2520histone%2520demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D352%26epage%3D363%26doi%3D10.1016%2Fj.ejmech.2014.08.068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Saleque, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooke, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Epigenetic regulation of hematopoietic differentiation by Gfi-1 and Gfi-1b is mediated by the cofactors CoREST and LSD1</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2007.06.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=562-572&author=S.+Salequeauthor=J.+Kimauthor=H.+M.+Rookeauthor=S.+H.+Orkin&title=Epigenetic+regulation+of+hematopoietic+differentiation+by+Gfi-1+and+Gfi-1b+is+mediated+by+the+cofactors+CoREST+and+LSD1&doi=10.1016%2Fj.molcel.2007.06.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.039%26sid%3Dliteratum%253Aachs%26aulast%3DSaleque%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DRooke%26aufirst%3DH.%2BM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DEpigenetic%2520regulation%2520of%2520hematopoietic%2520differentiation%2520by%2520Gfi-1%2520and%2520Gfi-1b%2520is%2520mediated%2520by%2520the%2520cofactors%2520CoREST%2520and%2520LSD1%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D562%26epage%3D572%26doi%3D10.1016%2Fj.molcel.2007.06.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monestiroli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giavara, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ronzoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchesi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insinga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diverio, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasparini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capillo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matteucci, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contegno, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo-Coco, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scanziani, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelicci, P. G.</span><span> </span><span class="NLM_article-title">PML-RAR induces promyelocytic leukemias with high efficiency following retroviral gene transfer into purified murine hematopoietic progenitors</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">2989</span><span class="NLM_x">–</span> <span class="NLM_lpage">2995</span><span class="refDoi"> DOI: 10.1182/blood-2001-11-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1182%2Fblood-2001-11-0089" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2002&pages=2989-2995&author=S.+Minucciauthor=S.+Monestiroliauthor=S.+Giavaraauthor=S.+Ronzoniauthor=F.+Marchesiauthor=A.+Insingaauthor=D.+Diverioauthor=P.+Gaspariniauthor=M.+Capilloauthor=E.+Colomboauthor=C.+Matteucciauthor=F.+Contegnoauthor=F.+Lo-Cocoauthor=E.+Scanzianiauthor=A.+Gobbiauthor=P.+G.+Pelicci&title=PML-RAR+induces+promyelocytic+leukemias+with+high+efficiency+following+retroviral+gene+transfer+into+purified+murine+hematopoietic+progenitors&doi=10.1182%2Fblood-2001-11-0089"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1182%2Fblood-2001-11-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2001-11-0089%26sid%3Dliteratum%253Aachs%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMonestiroli%26aufirst%3DS.%26aulast%3DGiavara%26aufirst%3DS.%26aulast%3DRonzoni%26aufirst%3DS.%26aulast%3DMarchesi%26aufirst%3DF.%26aulast%3DInsinga%26aufirst%3DA.%26aulast%3DDiverio%26aufirst%3DD.%26aulast%3DGasparini%26aufirst%3DP.%26aulast%3DCapillo%26aufirst%3DM.%26aulast%3DColombo%26aufirst%3DE.%26aulast%3DMatteucci%26aufirst%3DC.%26aulast%3DContegno%26aufirst%3DF.%26aulast%3DLo-Coco%26aufirst%3DF.%26aulast%3DScanziani%26aufirst%3DE.%26aulast%3DGobbi%26aufirst%3DA.%26aulast%3DPelicci%26aufirst%3DP.%2BG.%26atitle%3DPML-RAR%2520induces%2520promyelocytic%2520leukemias%2520with%2520high%2520efficiency%2520following%2520retroviral%2520gene%2520transfer%2520into%2520purified%2520murine%2520hematopoietic%2520progenitors%26jtitle%3DBlood%26date%3D2002%26volume%3D100%26spage%3D2989%26epage%3D2995%26doi%3D10.1182%2Fblood-2001-11-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Leiva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moretti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soilihi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallavicini, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peres, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Zuffo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de The, H.</span><span> </span><span class="NLM_article-title">Valproic acid induces differentiation and transient tumor regression, but spares leukemia-initiating activity in mouse models of APL</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1630</span><span class="NLM_x">–</span> <span class="NLM_lpage">1637</span><span class="refDoi"> DOI: 10.1038/leu.2012.39</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fleu.2012.39" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1630-1637&author=M.+Leivaauthor=S.+Morettiauthor=H.+Soilihiauthor=I.+Pallaviciniauthor=L.+Peresauthor=C.+Mercurioauthor=R.+Dal+Zuffoauthor=S.+Minucciauthor=H.+de+The&title=Valproic+acid+induces+differentiation+and+transient+tumor+regression%2C+but+spares+leukemia-initiating+activity+in+mouse+models+of+APL&doi=10.1038%2Fleu.2012.39"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fleu.2012.39&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2012.39%26sid%3Dliteratum%253Aachs%26aulast%3DLeiva%26aufirst%3DM.%26aulast%3DMoretti%26aufirst%3DS.%26aulast%3DSoilihi%26aufirst%3DH.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DPeres%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DDal%2BZuffo%26aufirst%3DR.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3Dde%2BThe%26aufirst%3DH.%26atitle%3DValproic%2520acid%2520induces%2520differentiation%2520and%2520transient%2520tumor%2520regression%252C%2520but%2520spares%2520leukemia-initiating%2520activity%2520in%2520mouse%2520models%2520of%2520APL%26jtitle%3DLeukemia%26date%3D2012%26volume%3D26%26spage%3D1630%26epage%3D1637%26doi%3D10.1038%2Fleu.2012.39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Maes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirapu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascarò, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiarte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valls, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro Palomino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arjol, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurz, G.</span><span> </span><span class="NLM_article-title">Preclinical characterization of a potent and selective inhibitor of the histone demethylase KDM1A for MLL leukemia</span> <span class="citation_source-journal">J. Clin Oncol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl.</span><span class="NLM_x">) </span> <span class="NLM_fpage">e13543</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=e13543&issue=Suppl.&author=T.+Maesauthor=I.+Tirapuauthor=C.+Mascar%C3%B2author=A.+Ortegaauthor=A.+Estiarteauthor=N.+Vallsauthor=J.+Castro+Palominoauthor=C.+B.+Arjolauthor=G.+Kurz&title=Preclinical+characterization+of+a+potent+and+selective+inhibitor+of+the+histone+demethylase+KDM1A+for+MLL+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DMascar%25C3%25B2%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DCastro%2BPalomino%26aufirst%3DJ.%26aulast%3DArjol%26aufirst%3DC.%2BB.%26aulast%3DKurz%26aufirst%3DG.%26atitle%3DPreclinical%2520characterization%2520of%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520histone%2520demethylase%2520KDM1A%2520for%2520MLL%2520leukemia%26jtitle%3DJ.%2520Clin%2520Oncol%26date%3D2013%26volume%3D31%26issue%3DSuppl%26spage%3De13543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Brucoli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thurston, D. E.</span><span> </span><span class="NLM_article-title">Efficient solid-phase synthesis of a library of distamycin analogs containing novel biaryl motifs on SynPhase Lanterns</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span><span class="refDoi"> DOI: 10.1021/cc900009r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc900009r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Ontrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=576-586&author=F.+Brucoliauthor=P.+W.+Howardauthor=D.+E.+Thurston&title=Efficient+solid-phase+synthesis+of+a+library+of+distamycin+analogs+containing+novel+biaryl+motifs+on+SynPhase+Lanterns&doi=10.1021%2Fcc900009r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Solid-Phase Synthesis of a Library of Distamycin Analogs Containing Novel Biaryl Motifs on SynPhase Lanterns</span></div><div class="casAuthors">Brucoli, Federico; Howard, Philip W.; Thurston, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Distamycin is a naturally occurring antibiotic that binds to AT-rich sequences in the minor groove of DNA in a noncovalent manner.  It continues to be of interest as a "building block" for more-complex small-mol. ligands capable of targeting specific DNA sequences for gene regulation purposes (i.e., transcription factor inhibitors).  Here, the authors report a convenient and efficient synthesis of a library of 72 novel analogs (3a-f) of the parent distamycin on SynPhase Lanterns.  To investigate structure-activity relationships including DNA-binding affinity and sequence-selectivity, two previously unexplored points of diversification have been introduced into the distamycin structure by replacing one of its pyrrole rings with novel biaryl motifs.  The key aspects of the synthetic approach include the development of an efficient protocol for prepn. of the heterocyclic polyamide chain, optimization of the Suzuki-Miyaura cross-coupling reaction and application of a split-and-mix technique based on radiofrequency encoding.  In addn., a series of biaryl carboxamide derivs. (4a-f) has been synthesized utilizing the title library diversity reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMv-1jhPjphLVg90H21EOLACvtfcHk0ljTGdgCN484Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Ontrg%253D&md5=a8f16be7e893bc0c1c444fe33d763c02</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fcc900009r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc900009r%26sid%3Dliteratum%253Aachs%26aulast%3DBrucoli%26aufirst%3DF.%26aulast%3DHoward%26aufirst%3DP.%2BW.%26aulast%3DThurston%26aufirst%3DD.%2BE.%26atitle%3DEfficient%2520solid-phase%2520synthesis%2520of%2520a%2520library%2520of%2520distamycin%2520analogs%2520containing%2520novel%2520biaryl%2520motifs%2520on%2520SynPhase%2520Lanterns%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2009%26volume%3D11%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fcc900009r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Shearer, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uehling, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deaton, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milliken, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faison, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, F.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of potent and selective beta3 adrenergic receptor agonists containing heterobiaryl carboxylic acids</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4670</span><span class="NLM_x">–</span> <span class="NLM_lpage">4677</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.05.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bmcl.2007.05.069" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4670-4677&author=B.+G.+Shearerauthor=E.+Y.+Chaoauthor=D.+E.+Uehlingauthor=D.+N.+Deatonauthor=C.+Cowanauthor=B.+W.+Shermanauthor=T.+Millikenauthor=W.+Faisonauthor=K.+Brownauthor=K.+K.+Adkisonauthor=F.+Lee&title=Synthesis+and+evaluation+of+potent+and+selective+beta3+adrenergic+receptor+agonists+containing+heterobiaryl+carboxylic+acids&doi=10.1016%2Fj.bmcl.2007.05.069"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.05.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.05.069%26sid%3Dliteratum%253Aachs%26aulast%3DShearer%26aufirst%3DB.%2BG.%26aulast%3DChao%26aufirst%3DE.%2BY.%26aulast%3DUehling%26aufirst%3DD.%2BE.%26aulast%3DDeaton%26aufirst%3DD.%2BN.%26aulast%3DCowan%26aufirst%3DC.%26aulast%3DSherman%26aufirst%3DB.%2BW.%26aulast%3DMilliken%26aufirst%3DT.%26aulast%3DFaison%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DK.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DLee%26aufirst%3DF.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520potent%2520and%2520selective%2520beta3%2520adrenergic%2520receptor%2520agonists%2520containing%2520heterobiaryl%2520carboxylic%2520acids%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4670%26epage%3D4677%26doi%3D10.1016%2Fj.bmcl.2007.05.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 38 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Xiao-Peng Xiong, Lei-Peng Xue, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14197-14215. <a href="https://doi.org/10.1021/acs.jmedchem.0c00919" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTranylcypromine%252BBased%252BLysine-Specific%252BDemethylase%252B1%252BInhibitor%25253A%252BSummary%252Band%252BPerspective%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30052020%26date%3D30092020%26date%3D15092020%26volume%3D63%26issue%3D23%26spage%3D14197%26epage%3D14215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alessia Romussi, Anna Cappa, Paola Vianello, Silvia Brambillasca, Maria Rosaria Cera, Roberto Dal Zuffo, Giovanni Fagà, Raimondo Fattori, Loris Moretti, Paolo Trifirò, Manuela Villa, Stefania Vultaggio, Valentina Cecatiello, Sebastiano Pasqualato, Giulio Dondio, Chi Wai Eric So, Saverio Minucci, Luca Sartori, Mario Varasi, <span class="NLM_string-name hlFld-ContribAuthor">Ciro Mercurio</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Reversible Inhibitors of KDM1A Efficacious in Acute Myeloid Leukemia Models. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 754-759. <a href="https://doi.org/10.1021/acsmedchemlett.9b00604" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00604</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00604%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BReversible%252BInhibitors%252Bof%252BKDM1A%252BEfficacious%252Bin%252BAcute%252BMyeloid%252BLeukemia%252BModels%26aulast%3DRomussi%26aufirst%3DAlessia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D16122019%26date%3D13022020%26date%3D20022020%26date%3D13022020%26volume%3D11%26issue%3D5%26spage%3D754%26epage%3D759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hayato Inoue, Nga Phan Thi Thanh, Ikuhide Fujisawa, <span class="NLM_string-name hlFld-ContribAuthor">Seiji Iwasa</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Forms of a Chiral Ruthenium Complex Containing a Ru–Colefin(sp2) Bond and Their Application to Catalytic Asymmetric Cyclopropanation Reactions. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (4)
                                     , 1475-1479. <a href="https://doi.org/10.1021/acs.orglett.0c00050" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c00050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c00050%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSynthesis%252Bof%252BForms%252Bof%252Ba%252BChiral%252BRuthenium%252BComplex%252BContaining%252Ba%252BRu%2525E2%252580%252593Colefin%252528sp2%252529%252BBond%252Band%252BTheir%252BApplication%252Bto%252BCatalytic%252BAsymmetric%252BCyclopropanation%252BReactions%26aulast%3DInoue%26aufirst%3DHayato%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06012020%26date%3D28012020%26volume%3D22%26issue%3D4%26spage%3D1475%26epage%3D1479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Xue-Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Peng-Fei  Geng</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Jin-Lian  Ma</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Tao-Qian  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhao-Qing  Zhou</span>, <span class="hlFld-ContribAuthor ">Chen-Xi  Huang</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, and <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of [1,2,3]Triazolo[4,5-d]pyrimidine Derivatives as Novel LSD1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (4)
                                     , 384-389. <a href="https://doi.org/10.1021/acsmedchemlett.6b00423" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00423</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00423%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252B%25255B1%25252C2%25252C3%25255DTriazolo%25255B4%25252C5-d%25255Dpyrimidine%252BDerivatives%252Bas%252BNovel%252BLSD1%252BInhibitors%26aulast%3DLi%26aufirst%3DZhong-Hua%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D25102016%26date%3D06032017%26date%3D08032017%26date%3D13042017%26date%3D06032017%26volume%3D8%26issue%3D4%26spage%3D384%26epage%3D389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Yang</span>, <span class="hlFld-ContribAuthor ">Wanhe  Wang</span>, <span class="hlFld-ContribAuthor ">Jia-Xin  Liang</span>, <span class="hlFld-ContribAuthor ">Guodong  Li</span>, <span class="hlFld-ContribAuthor ">Kasipandi  Vellaisamy</span>, <span class="hlFld-ContribAuthor ">Chun-Yuen  Wong</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>, and <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>  . </span><span class="cited-content_cbyCitation_article-title">A Rhodium(III)-Based Inhibitor of Lysine-Specific Histone Demethylase 1 as an Epigenetic Modulator in Prostate Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (6)
                                     , 2597-2603. <a href="https://doi.org/10.1021/acs.jmedchem.7b00133" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00133</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00133%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BRhodium%252528III%252529-Based%252BInhibitor%252Bof%252BLysine-Specific%252BHistone%252BDemethylase%252B1%252Bas%252Ban%252BEpigenetic%252BModulator%252Bin%252BProstate%252BCancer%252BCells%26aulast%3DYang%26aufirst%3DChao%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D25012017%26date%3D01032017%26date%3D23032017%26date%3D20022017%26volume%3D60%26issue%3D6%26spage%3D2597%26epage%3D2603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Sartori</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Federica  Amigoni</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Giovanni  Fagá</span>, <span class="hlFld-ContribAuthor ">Raimondo  Fattori</span>, <span class="hlFld-ContribAuthor ">Elena  Legnaghi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Meroni</span>, <span class="hlFld-ContribAuthor ">Loris  Moretti</span>, <span class="hlFld-ContribAuthor ">Valentina  Cecatiello</span>, <span class="hlFld-ContribAuthor ">Sebastiano  Pasqualato</span>, <span class="hlFld-ContribAuthor ">Alessia  Romussi</span>, <span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Paolo  Trifiró</span>, <span class="hlFld-ContribAuthor ">Manuela  Villa</span>, <span class="hlFld-ContribAuthor ">Stefania  Vultaggio</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Elisa  Zagarrí</span>, <span class="hlFld-ContribAuthor ">Daniele  Carettoni</span>, <span class="hlFld-ContribAuthor ">Lucia  Iuzzolino</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Paola  Vianello</span>  . </span><span class="cited-content_cbyCitation_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1673-1692. <a href="https://doi.org/10.1021/acs.jmedchem.6b01018" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%252Bas%252BNew%252BReversible%252BInhibitors%252Bof%252BHistone%252BLysine%252BDemethylase%252BKDM1A%25252FLSD1.%252BPart%252B1%25253A%252BHigh-Throughput%252BScreening%252Band%252BPreliminary%252BExploration%26aulast%3DSartori%26aufirst%3DLuca%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D25072016%26date%3D27022017%26date%3D09032017%26date%3D10022017%26volume%3D60%26issue%3D5%26spage%3D1673%26epage%3D1692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunal  Nepali</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in epigenetic cancer therapeutics: clinical advancement and emerging trends. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Science</span><span> <strong>2021,</strong> <em>28 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12929-021-00721-x" title="DOI URL">https://doi.org/10.1186/s12929-021-00721-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12929-021-00721-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12929-021-00721-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Science%26atitle%3DRecent%252Bdevelopments%252Bin%252Bepigenetic%252Bcancer%252Btherapeutics%25253A%252Bclinical%252Badvancement%252Band%252Bemerging%252Btrends%26aulast%3DNepali%26aufirst%3DKunal%26date%3D2021%26date%3D2021%26volume%3D28%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hailan  Huang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel benzofuran derivatives as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113501. <a href="https://doi.org/10.1016/j.ejmech.2021.113501" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113501</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113501%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bbenzofuran%252Bderivatives%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26volume%3D220%26spage%3D113501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yafei  Chu</span>, <span class="hlFld-ContribAuthor ">Zheng  Xiao</span>, <span class="hlFld-ContribAuthor ">Nan  Jing</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Yan</span>, <span class="hlFld-ContribAuthor ">Shanmei  Wang</span>, <span class="hlFld-ContribAuthor ">Bing  Ma</span>, <span class="hlFld-ContribAuthor ">Jiangfeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Yi  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Arborinine, a potential LSD1 inhibitor, inhibits epithelial-mesenchymal transition of SGC-7901 cells and adriamycin-resistant gastric cancer SGC-7901/ADR cells. </span><span class="cited-content_cbyCitation_journal-name">Investigational New Drugs</span><span> <strong>2021,</strong> <em>39 </em>
                                    (3)
                                     , 627-635. <a href="https://doi.org/10.1007/s10637-020-01016-y" title="DOI URL">https://doi.org/10.1007/s10637-020-01016-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10637-020-01016-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10637-020-01016-y%26sid%3Dliteratum%253Aachs%26jtitle%3DInvestigational%2520New%2520Drugs%26atitle%3DArborinine%25252C%252Ba%252Bpotential%252BLSD1%252Binhibitor%25252C%252Binhibits%252Bepithelial-mesenchymal%252Btransition%252Bof%252BSGC-7901%252Bcells%252Band%252Badriamycin-resistant%252Bgastric%252Bcancer%252BSGC-7901%25252FADR%252Bcells%26aulast%3DChu%26aufirst%3DYafei%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D3%26spage%3D627%26epage%3D635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria  Teresa Borrello</span>, <span class="hlFld-ContribAuthor ">Hanae  Benelkebir</span>, <span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">Chak  Hin Tam</span>, <span class="hlFld-ContribAuthor ">Manar  Shafat</span>, <span class="hlFld-ContribAuthor ">Stuart A.  Rushworth</span>, <span class="hlFld-ContribAuthor ">Kristian M.  Bowles</span>, <span class="hlFld-ContribAuthor ">Leon  Douglas</span>, <span class="hlFld-ContribAuthor ">Patrick J.  Duriez</span>, <span class="hlFld-ContribAuthor ">Sarah  Bailey</span>, <span class="hlFld-ContribAuthor ">Simon J.  Crabb</span>, <span class="hlFld-ContribAuthor ">Graham  Packham</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Carboxamide‐Containing Tranylcypromine Analogues as LSD1 (KDM1A) Inhibitors Targeting Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (8)
                                     , 1316-1324. <a href="https://doi.org/10.1002/cmdc.202000754" title="DOI URL">https://doi.org/10.1002/cmdc.202000754</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000754%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSynthesis%252Bof%252BCarboxamide%2525E2%252580%252590Containing%252BTranylcypromine%252BAnalogues%252Bas%252BLSD1%252B%252528KDM1A%252529%252BInhibitors%252BTargeting%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DTeresa%2BBorrello%26aufirst%3DMaria%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D8%26spage%3D1316%26epage%3D1324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking, 3D-QSAR, and molecular dynamics simulations of thieno[3,2-b]pyrrole derivatives against anticancer targets of KDM1A/LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>39 </em>
                                    (4)
                                     , 1189-1202. <a href="https://doi.org/10.1080/07391102.2020.1726819" title="DOI URL">https://doi.org/10.1080/07391102.2020.1726819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1726819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1726819%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DMolecular%252Bdocking%25252C%252B3D-QSAR%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bof%252Bthieno%25255B3%25252C2-b%25255Dpyrrole%252Bderivatives%252Bagainst%252Banticancer%252Btargets%252Bof%252BKDM1A%25252FLSD1%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1189%26epage%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samir  Mehndiratta</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine specific demethylase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 969-981. <a href="https://doi.org/10.1039/D0MD00141D" title="DOI URL">https://doi.org/10.1039/D0MD00141D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00141D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00141D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252Binhibitors%26aulast%3DMehndiratta%26aufirst%3DSamir%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D9%26spage%3D969%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyun  Liang</span>, <span class="hlFld-ContribAuthor ">Haiwen  Wang</span>, <span class="hlFld-ContribAuthor ">Yongliang  Du</span>, <span class="hlFld-ContribAuthor ">Bingling  Luo</span>, <span class="hlFld-ContribAuthor ">Ning  Meng</span>, <span class="hlFld-ContribAuthor ">Meifeng  Cen</span>, <span class="hlFld-ContribAuthor ">Peng  Huang</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>, <span class="hlFld-ContribAuthor ">Shijun  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">New tranylcypromine derivatives containing sulfonamide motif as potent LSD1 inhibitors to target acute myeloid leukemia: Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103808. <a href="https://doi.org/10.1016/j.bioorg.2020.103808" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103808%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNew%252Btranylcypromine%252Bderivatives%252Bcontaining%252Bsulfonamide%252Bmotif%252Bas%252Bpotent%252BLSD1%252Binhibitors%252Bto%252Btarget%252Bacute%252Bmyeloid%252Bleukemia%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DLiang%26aufirst%3DLiyun%26date%3D2020%26volume%3D99%26spage%3D103808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rossella  Fioravanti</span>, <span class="hlFld-ContribAuthor ">Annalisa  Romanelli</span>, <span class="hlFld-ContribAuthor ">Nicola  Mautone</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Di Bello</span>, <span class="hlFld-ContribAuthor ">Annarita  Rovere</span>, <span class="hlFld-ContribAuthor ">Davide  Corinti</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Stefania  Vultaggio</span>, <span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Claudia  Binda</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Tranylcypromine‐Based LSD1 Inhibitors: Structure‐Activity Relationships, Antiproliferative Effects in Leukemia, and Gene Target Modulation. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (7)
                                     , 643-658. <a href="https://doi.org/10.1002/cmdc.201900730" title="DOI URL">https://doi.org/10.1002/cmdc.201900730</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900730%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DTranylcypromine%2525E2%252580%252590Based%252BLSD1%252BInhibitors%25253A%252BStructure%2525E2%252580%252590Activity%252BRelationships%25252C%252BAntiproliferative%252BEffects%252Bin%252BLeukemia%25252C%252Band%252BGene%252BTarget%252BModulation%26aulast%3DFioravanti%26aufirst%3DRossella%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D7%26spage%3D643%26epage%3D658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roberto  Ravasio</span>, <span class="hlFld-ContribAuthor ">Elena  Ceccacci</span>, <span class="hlFld-ContribAuthor ">Luciano  Nicosia</span>, <span class="hlFld-ContribAuthor ">Amir  Hosseini</span>, <span class="hlFld-ContribAuthor ">Pier Luigi  Rossi</span>, <span class="hlFld-ContribAuthor ">Iros  Barozzi</span>, <span class="hlFld-ContribAuthor ">Lorenzo  Fornasari</span>, <span class="hlFld-ContribAuthor ">Roberto Dal  Zuffo</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Rossella  Fioravanti</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Giulio  Pavesi</span>, <span class="hlFld-ContribAuthor ">Tiziana  Bonaldi</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the scaffolding role of LSD1 (KDM1A) poises acute myeloid leukemia cells for retinoic acid–induced differentiation. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2020,</strong> <em>6 </em>
                                    (15)
                                     , eaax2746. <a href="https://doi.org/10.1126/sciadv.aax2746" title="DOI URL">https://doi.org/10.1126/sciadv.aax2746</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.aax2746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.aax2746%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DTargeting%252Bthe%252Bscaffolding%252Brole%252Bof%252BLSD1%252B%252528KDM1A%252529%252Bpoises%252Bacute%252Bmyeloid%252Bleukemia%252Bcells%252Bfor%252Bretinoic%252Bacid%2525E2%252580%252593induced%252Bdifferentiation%26aulast%3DRavasio%26aufirst%3DRoberto%26date%3D2020%26date%3D2020%26volume%3D6%26issue%3D15%26spage%3Deaax2746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic polypharmacology: A new frontier for epi‐drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (1)
                                     , 190-244. <a href="https://doi.org/10.1002/med.21600" title="DOI URL">https://doi.org/10.1002/med.21600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21600%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DEpigenetic%252Bpolypharmacology%25253A%252BA%252Bnew%252Bfrontier%252Bfor%252Bepi%2525E2%252580%252590drug%252Bdiscovery%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26date%3D2019%26volume%3D40%26issue%3D1%26spage%3D190%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caterina  Martin</span>, <span class="hlFld-ContribAuthor ">Claudia  Binda</span>, <span class="hlFld-ContribAuthor ">Marco W.  Fraaije</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>. </span><span class="cited-content_cbyCitation_article-title">The multipurpose family of flavoprotein oxidases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 63-86. <a href="https://doi.org/10.1016/bs.enz.2020.05.002" title="DOI URL">https://doi.org/10.1016/bs.enz.2020.05.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.enz.2020.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.enz.2020.05.002%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252Bmultipurpose%252Bfamily%252Bof%252Bflavoprotein%252Boxidases%26aulast%3DMartin%26aufirst%3DCaterina%26date%3D2020%26spage%3D63%26epage%3D86%26pub%3DElsevier%26atitle%3DFlavin-Dependent%252BEnzymes%25253A%252BMechanisms%25252C%252BStructures%252Band%252BApplications%26date%3D2020%26volume%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amal Kamal  Abdel-Aziz</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>. </span><span class="cited-content_cbyCitation_article-title">Comparing apples with oranges: Studying LSD1 inhibitors in cellular assays. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2019,</strong> <em>146 </em>, 104345. <a href="https://doi.org/10.1016/j.phrs.2019.104345" title="DOI URL">https://doi.org/10.1016/j.phrs.2019.104345</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2019.104345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2019.104345%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DComparing%252Bapples%252Bwith%252Boranges%25253A%252BStudying%252BLSD1%252Binhibitors%252Bin%252Bcellular%252Bassays%26aulast%3DAbdel-Aziz%26aufirst%3DAmal%2BKamal%26date%3D2019%26volume%3D146%26spage%3D104345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 388-401. <a href="https://doi.org/10.1016/j.ejmech.2019.02.039" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bstructure-activity%252Brelationship%252Bstudies%252Band%252Bbiological%252Bcharacterization%252Bof%252Bnew%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine-based%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DWang%26aufirst%3DShuai%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (Lysine‐Specific Demethylase): A Decade‐Long Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-Specific Histone Demethylases 1/2 (LSD1/2) and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 197-219. <a href="https://doi.org/10.1007/7355_2019_74" title="DOI URL">https://doi.org/10.1007/7355_2019_74</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_74%26sid%3Dliteratum%253Aachs%26atitle%3DLysine-Specific%252BHistone%252BDemethylases%252B1%25252F2%252B%252528LSD1%25252F2%252529%252Band%252BTheir%252BInhibitors%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2020%26spage%3D197%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xudong  Sun</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mode and unbinding mechanism of LSD1 inhibitors by combined computational methods. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (47)
                                     , 29833-29846. <a href="https://doi.org/10.1039/C8CP03090A" title="DOI URL">https://doi.org/10.1039/C8CP03090A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP03090A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP03090A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmode%252Band%252Bunbinding%252Bmechanism%252Bof%252BLSD1%252Binhibitors%252Bby%252Bcombined%252Bcomputational%252Bmethods%26aulast%3DSun%26aufirst%3DXudong%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D47%26spage%3D29833%26epage%3D29846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naoki  Umezawa</span>, <span class="hlFld-ContribAuthor ">Kasumi  Tsuji</span>, <span class="hlFld-ContribAuthor ">Shin  Sato</span>, <span class="hlFld-ContribAuthor ">Masaki  Kikuchi</span>, <span class="hlFld-ContribAuthor ">Hisami  Watanabe</span>, <span class="hlFld-ContribAuthor ">Yuhei  Horai</span>, <span class="hlFld-ContribAuthor ">Masashi  Yamaguchi</span>, <span class="hlFld-ContribAuthor ">Yosuke  Hisamatsu</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>, <span class="hlFld-ContribAuthor ">Tsunehiko  Higuchi</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of FAD-dependent lysine-specific demethylases by chiral polyamine analogues. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2018,</strong> <em>8 </em>
                                    (64)
                                     , 36895-36902. <a href="https://doi.org/10.1039/C8RA07879C" title="DOI URL">https://doi.org/10.1039/C8RA07879C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8RA07879C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8RA07879C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DInhibition%252Bof%252BFAD-dependent%252Blysine-specific%252Bdemethylases%252Bby%252Bchiral%252Bpolyamine%252Banalogues%26aulast%3DUmezawa%26aufirst%3DNaoki%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D64%26spage%3D36895%26epage%3D36902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan‐Ru  Yeh</span>, <span class="hlFld-ContribAuthor ">Yi‐Jia  Tzeng</span>, <span class="hlFld-ContribAuthor ">Shau‐Wei  Tsai</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative Improvements and Insights into CALB‐Catalyzed Resolution of
              trans
              ‐ and
              cis
              ‐2‐Phenylcyclopropyl Azolides. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2018,</strong> <em>3 </em>
                                    (19)
                                     , 5353-5360. <a href="https://doi.org/10.1002/slct.201800578" title="DOI URL">https://doi.org/10.1002/slct.201800578</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201800578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201800578%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DQuantitative%252BImprovements%252Band%252BInsights%252Binto%252BCALB%2525E2%252580%252590Catalyzed%252BResolution%252Bof%252Btrans%252B%2525E2%252580%252590%252Band%252Bcis%252B%2525E2%252580%2525902%2525E2%252580%252590Phenylcyclopropyl%252BAzolides%26aulast%3DYeh%26aufirst%3DYan%25E2%2580%2590Ru%26date%3D2018%26date%3D2018%26volume%3D3%26issue%3D19%26spage%3D5353%26epage%3D5360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Volker M.  Lauschke</span>, <span class="hlFld-ContribAuthor ">Isabel  Barragan</span>, <span class="hlFld-ContribAuthor ">Magnus  Ingelman-Sundberg</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacoepigenetics and Toxicoepigenetics: Novel Mechanistic Insights and Therapeutic Opportunities. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Pharmacology and Toxicology</span><span> <strong>2018,</strong> <em>58 </em>
                                    (1)
                                     , 161-185. <a href="https://doi.org/10.1146/annurev-pharmtox-010617-053021" title="DOI URL">https://doi.org/10.1146/annurev-pharmtox-010617-053021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-pharmtox-010617-053021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-pharmtox-010617-053021%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Pharmacology%2520and%2520Toxicology%26atitle%3DPharmacoepigenetics%252Band%252BToxicoepigenetics%25253A%252BNovel%252BMechanistic%252BInsights%252Band%252BTherapeutic%252BOpportunities%26aulast%3DLauschke%26aufirst%3DVolker%2BM.%26date%3D2018%26volume%3D58%26issue%3D1%26spage%3D161%26epage%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Johannes  Schulz-Fincke</span>, <span class="hlFld-ContribAuthor ">Mirjam  Hau</span>, <span class="hlFld-ContribAuthor ">Jessica  Barth</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Dominica  Willmann</span>, <span class="hlFld-ContribAuthor ">Andreas  Kürner</span>, <span class="hlFld-ContribAuthor ">Julian  Haas</span>, <span class="hlFld-ContribAuthor ">Gabriele  Greve</span>, <span class="hlFld-ContribAuthor ">Tinka  Haydn</span>, <span class="hlFld-ContribAuthor ">Simone  Fulda</span>, <span class="hlFld-ContribAuthor ">Michael  Lübbert</span>, <span class="hlFld-ContribAuthor ">Steffen  Lüdeke</span>, <span class="hlFld-ContribAuthor ">Tobias  Berg</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity studies on N -Substituted tranylcypromine derivatives lead to selective inhibitors of lysine specific demethylase 1 (LSD1) and potent inducers of leukemic cell differentiation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>144 </em>, 52-67. <a href="https://doi.org/10.1016/j.ejmech.2017.12.001" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.12.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.12.001%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Bstudies%252Bon%252BN%252B-Substituted%252Btranylcypromine%252Bderivatives%252Blead%252Bto%252Bselective%252Binhibitors%252Bof%252Blysine%252Bspecific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Band%252Bpotent%252Binducers%252Bof%252Bleukemic%252Bcell%252Bdifferentiation%26aulast%3DSchulz-Fincke%26aufirst%3DJohannes%26date%3D2018%26volume%3D144%26spage%3D52%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eva  Horáková</span>, <span class="hlFld-ContribAuthor ">Pavel  Drabina</span>, <span class="hlFld-ContribAuthor ">Lenka  Brůčková</span>, <span class="hlFld-ContribAuthor ">Šárka  Štěpánková</span>, <span class="hlFld-ContribAuthor ">Katarína  Vorčáková</span>, <span class="hlFld-ContribAuthor ">Miloš  Sedlák</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and in vitro evaluation of novel N-cycloalkylcarbamates as potential cholinesterase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Monatshefte für Chemie - Chemical Monthly</span><span> <strong>2017,</strong> <em>148 </em>
                                    (12)
                                     , 2143-2153. <a href="https://doi.org/10.1007/s00706-017-2026-5" title="DOI URL">https://doi.org/10.1007/s00706-017-2026-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00706-017-2026-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00706-017-2026-5%26sid%3Dliteratum%253Aachs%26jtitle%3DMonatshefte%2520f%25C3%25BCr%2520Chemie%2520-%2520Chemical%2520Monthly%26atitle%3DSynthesis%252Band%252Bin%252Bvitro%252Bevaluation%252Bof%252Bnovel%252BN-cycloalkylcarbamates%252Bas%252Bpotential%252Bcholinesterase%252Binhibitors%26aulast%3DHor%25C3%25A1kov%25C3%25A1%26aufirst%3DEva%26date%3D2017%26date%3D2017%26volume%3D148%26issue%3D12%26spage%3D2143%26epage%3D2153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue-Yang  Ji</span>, <span class="hlFld-ContribAuthor ">Sen-Dong  Lin</span>, <span class="hlFld-ContribAuthor ">Yu-Jie  Wang</span>, <span class="hlFld-ContribAuthor ">Ming-Bo  Su</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">Hendra  Gunosewoyo</span>, <span class="hlFld-ContribAuthor ">Fan  Yang</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Jie  Tang</span>, <span class="hlFld-ContribAuthor ">Yu-Bo  Zhou</span>, <span class="hlFld-ContribAuthor ">Li-Fang  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Tying up tranylcypromine: Novel selective histone lysine specific demethylase 1 (LSD1) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>141 </em>, 101-112. <a href="https://doi.org/10.1016/j.ejmech.2017.09.073" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.09.073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.09.073%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTying%252Bup%252Btranylcypromine%25253A%252BNovel%252Bselective%252Bhistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Binhibitors%26aulast%3DJi%26aufirst%3DYue-Yang%26date%3D2017%26volume%3D141%26spage%3D101%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paolo  Trifirò</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Silvia  Brambillasca</span>, <span class="hlFld-ContribAuthor ">Oronza A  Botrugno</span>, <span class="hlFld-ContribAuthor ">Maria Rosaria  Cera</span>, <span class="hlFld-ContribAuthor ">Roberto Dal  Zuffo</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Meroni</span>, <span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Manuela  Villa</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, <span class="hlFld-ContribAuthor ">Paola  Vianello</span>. </span><span class="cited-content_cbyCitation_article-title">Novel potent inhibitors of the histone demethylase KDM1A (LSD1), orally active in a murine promyelocitic leukemia model. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1161-1174. <a href="https://doi.org/10.4155/fmc-2017-0003" title="DOI URL">https://doi.org/10.4155/fmc-2017-0003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0003%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DNovel%252Bpotent%252Binhibitors%252Bof%252Bthe%252Bhistone%252Bdemethylase%252BKDM1A%252B%252528LSD1%252529%25252C%252Borally%252Bactive%252Bin%252Ba%252Bmurine%252Bpromyelocitic%252Bleukemia%252Bmodel%26aulast%3DTrifir%25C3%25B2%26aufirst%3DPaolo%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1161%26epage%3D1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoli  Fu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Advances toward LSD1 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1227-1242. <a href="https://doi.org/10.4155/fmc-2017-0068" title="DOI URL">https://doi.org/10.4155/fmc-2017-0068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Btoward%252BLSD1%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DFu%26aufirst%3DXiaoli%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1227%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 340-352. <a href="https://doi.org/10.1080/15592294.2017.1290032" title="DOI URL">https://doi.org/10.1080/15592294.2017.1290032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DStructural%252Binsight%252Binto%252Binhibitors%252Bof%252Bflavin%252Badenine%252Bdinucleotide-dependent%252Blysine%252Bdemethylases%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D340%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">Benjamin  Schinor</span>, <span class="hlFld-ContribAuthor ">Katharina  Bartels</span>, <span class="hlFld-ContribAuthor ">Hanae  Benelkebir</span>, <span class="hlFld-ContribAuthor ">Sara  Pereira</span>, <span class="hlFld-ContribAuthor ">Wafa T.  Al-Jamal</span>, <span class="hlFld-ContribAuthor ">Leon  Douglas</span>, <span class="hlFld-ContribAuthor ">Patrick J.  Duriez</span>, <span class="hlFld-ContribAuthor ">Graham  Packham</span>, <span class="hlFld-ContribAuthor ">Günter  Haufe</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorinated tranylcypromine analogues as inhibitors of lysine-specific demethylase 1 (LSD1, KDM1A). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (10)
                                     , 2099-2101. <a href="https://doi.org/10.1016/j.bmcl.2017.03.081" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.03.081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.03.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.03.081%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DFluorinated%252Btranylcypromine%252Banalogues%252Bas%252Binhibitors%252Bof%252Blysine-specific%252Bdemethylase%252B1%252B%252528LSD1%25252C%252BKDM1A%252529%26aulast%3DBorrello%26aufirst%3DMaria%2BTeresa%26date%3D2017%26volume%3D27%26issue%3D10%26spage%3D2099%26epage%3D2101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria E.  Ourailidou</span>, <span class="hlFld-ContribAuthor ">Alessia  Lenoci</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Frank J.  Dekker</span>. </span><span class="cited-content_cbyCitation_article-title">Towards the development of activity-based probes for detection of lysine-specific demethylase-1 activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (3)
                                     , 847-856. <a href="https://doi.org/10.1016/j.bmc.2016.11.043" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.11.043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.11.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.11.043%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTowards%252Bthe%252Bdevelopment%252Bof%252Bactivity-based%252Bprobes%252Bfor%252Bdetection%252Bof%252Blysine-specific%252Bdemethylase-1%252Bactivity%26aulast%3DOurailidou%26aufirst%3DMaria%2BE.%26date%3D2017%26volume%3D25%26issue%3D3%26spage%3D847%26epage%3D856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.M.  Woster</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutics Targeting Epigenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 297-328. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12325-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12325-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12325-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12325-X%26sid%3Dliteratum%253Aachs%26atitle%3DNovel%252BTherapeutics%252BTargeting%252BEpigenetics%26aulast%3DWoster%26aufirst%3DP.M.%26date%3D2017%26spage%3D297%26epage%3D328%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashwini  Jambhekar</span>, <span class="hlFld-ContribAuthor ">Jamie N.  Anastas</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Lysine Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     , a026484. <a href="https://doi.org/10.1101/cshperspect.a026484" title="DOI URL">https://doi.org/10.1101/cshperspect.a026484</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DHistone%252BLysine%252BDemethylase%252BInhibitors%26aulast%3DJambhekar%26aufirst%3DAshwini%26date%3D2017%26date%3D2016%26volume%3D7%26issue%3D1%26spage%3Da026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingxin  Feng</span>, <span class="hlFld-ContribAuthor ">Guiying  Xu</span>, <span class="hlFld-ContribAuthor ">Jiwei  Liu</span>, <span class="hlFld-ContribAuthor ">Na  Zhang</span>, <span class="hlFld-ContribAuthor ">Lili  Li</span>, <span class="hlFld-ContribAuthor ">Jiafei  Ji</span>, <span class="hlFld-ContribAuthor ">Jianchao  Zhang</span>, <span class="hlFld-ContribAuthor ">Lian  Zhang</span>, <span class="hlFld-ContribAuthor ">Guannan  Wang</span>, <span class="hlFld-ContribAuthor ">Xiuli  Wang</span>, <span class="hlFld-ContribAuthor ">Jiang  Tan</span>, <span class="hlFld-ContribAuthor ">Baiqu  Huang</span>, <span class="hlFld-ContribAuthor ">Jun  Lu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphorylation of LSD1 at Ser112 is crucial for its function in induction of EMT and metastasis in breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Breast Cancer Research and Treatment</span><span> <strong>2016,</strong> <em>159 </em>
                                    (3)
                                     , 443-456. <a href="https://doi.org/10.1007/s10549-016-3959-9" title="DOI URL">https://doi.org/10.1007/s10549-016-3959-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10549-016-3959-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10549-016-3959-9%26sid%3Dliteratum%253Aachs%26jtitle%3DBreast%2520Cancer%2520Research%2520and%2520Treatment%26atitle%3DPhosphorylation%252Bof%252BLSD1%252Bat%252BSer112%252Bis%252Bcrucial%252Bfor%252Bits%252Bfunction%252Bin%252Binduction%252Bof%252BEMT%252Band%252Bmetastasis%252Bin%252Bbreast%252Bcancer%26aulast%3DFeng%26aufirst%3DJingxin%26date%3D2016%26date%3D2016%26volume%3D159%26issue%3D3%26spage%3D443%26epage%3D456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Speranzini</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Simona  Pilotto</span>, <span class="hlFld-ContribAuthor ">Biagina  Marrocco</span>, <span class="hlFld-ContribAuthor ">Mariantonietta  Forgione</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Federico  Forneris</span>, <span class="hlFld-ContribAuthor ">Parinaz  Mehdipour</span>, <span class="hlFld-ContribAuthor ">Sameer  Velankar</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>. </span><span class="cited-content_cbyCitation_article-title">Polymyxins and quinazolines are LSD1/KDM1A inhibitors with unusual structural features. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2016,</strong> <em>2 </em>
                                    (9)
                                     , e1601017. <a href="https://doi.org/10.1126/sciadv.1601017" title="DOI URL">https://doi.org/10.1126/sciadv.1601017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.1601017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.1601017%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DPolymyxins%252Band%252Bquinazolines%252Bare%252BLSD1%25252FKDM1A%252Binhibitors%252Bwith%252Bunusual%252Bstructural%252Bfeatures%26aulast%3DSperanzini%26aufirst%3DValentina%26date%3D2016%26date%3D2016%26volume%3D2%26issue%3D9%26spage%3De1601017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shin  Miyamura</span>, <span class="hlFld-ContribAuthor ">Misaho  Araki</span>, <span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Shusuke  Yasuda</span>, <span class="hlFld-ContribAuthor ">Mitsuharu  Masuda</span>, <span class="hlFld-ContribAuthor ">Tomoyuki  Taniguchi</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Sowa</span>, <span class="hlFld-ContribAuthor ">Toshiyuki  Sakai</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>, <span class="hlFld-ContribAuthor ">Kenichiro  Itami</span>, <span class="hlFld-ContribAuthor ">Junichiro  Yamaguchi</span>. </span><span class="cited-content_cbyCitation_article-title">C–H activation enables a rapid structure–activity relationship study of arylcyclopropyl amines for potent and selective LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2016,</strong> <em>14 </em>
                                    (36)
                                     , 8576-8585. <a href="https://doi.org/10.1039/C6OB01483F" title="DOI URL">https://doi.org/10.1039/C6OB01483F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6OB01483F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6OB01483F%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DC%2525E2%252580%252593H%252Bactivation%252Benables%252Ba%252Brapid%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Bstudy%252Bof%252Barylcyclopropyl%252Bamines%252Bfor%252Bpotent%252Band%252Bselective%252BLSD1%252Binhibitors%26aulast%3DMiyamura%26aufirst%3DShin%26date%3D2016%26date%3D2016%26volume%3D14%26issue%3D36%26spage%3D8576%26epage%3D8585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KDM1A inhibitors in clinical studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) triethyl phosphonoacetate, BuLi, DME, 135 °C, microwave, 1 h; (b) LiOH, THF, water, EtOH, microwave, 115 °C, 50 min; (c) I<sub>2</sub>, KIO<sub>3</sub>, H<sub>2</sub>O, CH<sub>3</sub>COOH, H<sub>2</sub>SO<sub>4</sub>, 2 h, 0 °C, reflux; (d) DPPA, <i>t</i>-BuOH, TEA, 90 °C, 20 h; (e) ArCONH<sub>2</sub>, CuI, DMEDA, K<sub>2</sub>CO<sub>3</sub>, dioxane, 110 °C, 20 h; (f) HCl, MeOH, diethyl ether, 0 °C, 20 h.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Novel compounds synthesized.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Enantiomers of compound <b>14d</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) PyBOP, TEA, DMF, rt, overnight; (b) THF, 4 M HCl in dioxane, rt, overnight.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) morpholine, 4-methylpiperazine, or piperidine, K<sub>2</sub>CO<sub>3</sub>, DMF, 90 °C, 5 h; (b) 10% Pd–C, MeOH, Parr apparatus, 50 psi, 25 °C, 5 h; (c) benzyl chloroformate, TEA, THF, rt 1.5 h; (d) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, overnight.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) thionyl chloride, DMF, CH<sub>2</sub>Cl<sub>2</sub>, reflux, 2 h; (b) NH<sub>3</sub> (28–30% in water), rt, 1 h.</p></p></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <b>14d</b>, <b>14e</b>, <b>14g</b>, and <b>22</b> induce differentiation of THP-1 cells. (A) Representative images of THP-1 colonies after 13 days in semisolid culture with the indicated inhibitors. (B) Representative images of cytospin preparations of THP-1 cells recovered at the end of semisolid culture assay and May Grunwald-Giemsa stained. I: immature cells. M: macrophage.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. In vivo studies of compounds <b>14e</b>, <b>14d</b>, and <b>22</b>. (A) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>14e</b>. Kaplan–Meier survival curves of leukemic mice treated with compound <b>14e</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>14e</b> was orally administered at a dose of 27 mg/kg for 5 days per week for 2 weeks. (B) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>14d</b>. Kaplan–Meier survival curves of leukemic mice treated with compound <b>14d</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>14d</b> was orally administered at a dose of 27 mg/kg for 5 days per week for 2 weeks and at a dose of 45 mg/kg for 3 days per week for 2 weeks. (C) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>22</b>. Kaplan–Meier survival curves of leukemic mice treated with <b>22</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>22</b> was intraperitoneally administered at a dose of 0.0125 mg/kg for 5 days per week for 2 weeks. (D) Quantitative real-time PCR analysis of GFI1B mRNA expression in leukemic cells recovered from infiltrated spleen at 3 days after starting in vivo treatment with compound <b>14d</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>15</b> and <b>16</b> induce differentiation of THP-1 cells. (A) Representative images of THP-1 colonies after 13 days in semisolid culture with the indicated inhibitors. (B) Representative images of cytospin preparations of THP-1 cells recovered at the end of semisolid culture assay and May Grunwald-Giemsa stained. I: immature cells. M: macrophage.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/medium/jm-2015-01209t_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0008.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo studies of compound <b>15</b>. (A) In vivo efficacy experiment in an established murine promyelocytic leukemia model of <b>15</b>. Kaplan–Meier survival curves of leukemic mice treated with compound <b>15</b> and its respective vehicle. Treatment started once blast cells are detected in the recipients’ peripheral blood (10 days after cell injection). <b>15</b> was orally administered at doses of 11.25 and 22.5 mg/kg for 3 days per week for 3 weeks. (B) Quantitative real-time PCR analysis of GFI1B mRNA expression in leukemic cells recovered from infiltrated spleen at 3 days after starting in vivo treatment with compound <b>15</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01209/20160219/images/large/jm-2015-01209t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01209&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73651" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73651" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 75 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Helin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span> </span><span class="NLM_article-title">Chromatin proteins and modifications as drug targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">480</span><span class="NLM_x">–</span> <span class="NLM_lpage">488</span><span class="refDoi"> DOI: 10.1038/nature12751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature12751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24153301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=480-488&author=K.+Helinauthor=D.+Dhanak&title=Chromatin+proteins+and+modifications+as+drug+targets&doi=10.1038%2Fnature12751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin proteins and modifications as drug targets</span></div><div class="casAuthors">Helin, Kristian; Dhanak, Dashyant</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">480-488</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A plethora of groundbreaking studies have demonstrated the importance of chromatin-assocd. proteins and post-translational modifications of histones, proteins and DNA (so-called epigenetic modifications) for transcriptional control and normal development.  Disruption of epigenetic control is a frequent event in disease, and the first epigenetic-based therapies for cancer treatment have been approved.  A generation of new classes of potent and specific inhibitors for several chromatin-assocd. proteins have shown promise in preclin. trials.  Although the biol. of epigenetic regulation is complex, new inhibitors such as these will hopefully be of clin. use in the coming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa6LKkNrgP9bVg90H21EOLACvtfcHk0lhmhMthrGJxng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E&md5=ca3f12c6aaba31bb571095bacc02550c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12751%26sid%3Dliteratum%253Aachs%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DChromatin%2520proteins%2520and%2520modifications%2520as%2520drug%2520targets%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D480%26epage%3D488%26doi%3D10.1038%2Fnature12751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The epigenomics of cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer&doi=10.1016%2Fj.cell.2007.01.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0lh0ejhoQDFyGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692%26doi%3D10.1016%2Fj.cell.2007.01.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Rodriguez-Paredes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteller, M.</span><span> </span><span class="NLM_article-title">Cancer epigenetics reaches mainstream oncology</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1038/nm.2305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnm.2305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21386836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=330-339&author=M.+Rodriguez-Paredesauthor=M.+Esteller&title=Cancer+epigenetics+reaches+mainstream+oncology&doi=10.1038%2Fnm.2305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics reaches mainstream oncology</span></div><div class="casAuthors">Rodriguez-Paredes, Manuel; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">330-339</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetics is one of the most promising and expanding fields in the current biomedical research landscape.  Since the inception of epigenetics in the 1940s, the discoveries regarding its implications in normal and disease biol. have not stopped, compiling a vast amt. of knowledge in the past decade.  The field has moved from just one recognized marker, DNA methylation, to a variety of others, including a wide spectrum of histone modifications.  From the methodol. standpoint, the successful initial single gene candidate approaches have been complemented by the current comprehensive epigenomic approaches that allow the interrogation of genomes to search for translational applications in an unbiased manner.  Most important, the discovery of mutations in the epigenetic machinery and the approval of the first epigenetic drugs for the treatment of subtypes of leukemias and lymphomas has been an eye-opener for many biomedical scientists and clinicians.  Herein, we will summarize the progress in the field of cancer epigenetics research that has reached mainstream oncol. in the development of new biomarkers of the disease and new pharmacol. strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxyADVbZ4KHrVg90H21EOLACvtfcHk0lh0ejhoQDFyGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D&md5=07b6faaddb8bce884261990dbc927331</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnm.2305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2305%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Paredes%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DCancer%2520epigenetics%2520reaches%2520mainstream%2520oncology%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D330%26epage%3D339%26doi%3D10.1038%2Fnm.2305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+lysine+methylation+dynamics%3A+establishment%2C+regulation%2C+and+biological+impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0lh0ejhoQDFyGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520lysine%2520methylation%2520dynamics%253A%2520establishment%252C%2520regulation%252C%2520and%2520biological%2520impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0lhy9OY81prH0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabari, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span> </span><span class="NLM_article-title">SnapShot: histone modifications</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">458 e1</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2014.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25303536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=458-458+e1&author=H.+Huangauthor=B.+R.+Sabariauthor=B.+A.+Garciaauthor=C.+D.+Allisauthor=Y.+Zhao&title=SnapShot%3A+histone+modifications&doi=10.1016%2Fj.cell.2014.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SnapShot: Histone modifications</span></div><div class="casAuthors">Huang, He; Sabari, Benjamin R.; Garcia, Benjamin A.; Allis, C. David; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">458-458.e1</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Histones are decorated by a variety of protein post-translational modifications called histone marks that modulate chromatin structure and function, contributing to the cellular gene expression program.  This SnapShot summarizes the reported human, mouse, and rat histone marks, including recently identified Lys residue acylation marks.  It has been proposed that histone acylations provide a link between cellular metab. and epigenetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQmsTna8jsxrVg90H21EOLACvtfcHk0lhRuEyBO8S0AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO&md5=969e02ba1cbaa8f87bd5aae32857720f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DSabari%26aufirst%3DB.%2BR.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DSnapShot%253A%2520histone%2520modifications%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D458%26epage%3D458%2520e1%26doi%3D10.1016%2Fj.cell.2014.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span> </span><span class="NLM_article-title">Structural and sequence motifs of protein (histone) methylation enzymes</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">294</span><span class="refDoi"> DOI: 10.1146/annurev.biophys.34.040204.144452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1146%2Fannurev.biophys.34.040204.144452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=15869391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslCku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2005&pages=267-294&author=X.+Chengauthor=R.+E.+Collinsauthor=X.+Zhang&title=Structural+and+sequence+motifs+of+protein+%28histone%29+methylation+enzymes&doi=10.1146%2Fannurev.biophys.34.040204.144452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and sequence motifs of protein (histone) methylation enzymes</span></div><div class="casAuthors">Cheng, Xiaodong; Collins, Robert E.; Zhang, Xing</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">267-294</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  With genome sequencing nearing completion for model organisms used in biomedical research, there is a rapidly growing appreciation that proteomics, the study of covalent modification to proteins, and transcriptional regulation will likely dominate the research headlines in the next decade.  Protein methylation plays a central role in both of these fields, as several different residues (Arg, Lys, Gln) are methylated in cells and methylation plays a central role in the "histone code" that regulates chromatin structure and impacts transcription.  In some cases, a single lysine can be mono-, di-, or trimethylated, with different functional consequences for each of the 3 forms.  This review describes structural aspects of methylation of histone Lys residues by 2 enzyme families with entirely different structural scaffolding (the SET proteins and Dot1p) and methylation of protein Arg residues by protein (arginine) methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptbkTQSJ8bYLVg90H21EOLACvtfcHk0lhRuEyBO8S0AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslCku7s%253D&md5=04dc4de26a818c98cdacccfe74051e52</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.34.040204.144452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.34.040204.144452%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCollins%26aufirst%3DR.%2BE.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DStructural%2520and%2520sequence%2520motifs%2520of%2520protein%2520%2528histone%2529%2520methylation%2520enzymes%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2005%26volume%3D34%26spage%3D267%26epage%3D294%26doi%3D10.1146%2Fannurev.biophys.34.040204.144452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallin, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">JmjC-domain-containing proteins and histone demethylation</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">715</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span><span class="refDoi"> DOI: 10.1038/nrg1945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrg1945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16983801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=715-727&author=R.+J.+Kloseauthor=E.+M.+Kallinauthor=Y.+Zhang&title=JmjC-domain-containing+proteins+and+histone+demethylation&doi=10.1038%2Fnrg1945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">JmjC-domain-containing proteins and histone demethylation</span></div><div class="casAuthors">Klose, Robert J.; Kallin, Eric M.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">715-727</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation has important roles in regulating gene expression and forms part of the epigenetic memory system that regulates cell fate and identity.  Enzymes that directly remove Me marks from histones have recently been identified, revealing a new level of plasticity within this epigenetic modification system.  Here we analyze the evolutionary relationship between Jumonji C (JmjC)-domain-contg. proteins and discuss their cellular functions in relation to their potential enzymic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb9c8M0AJzLVg90H21EOLACvtfcHk0ljghG9T4XkdUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1Ggu74%253D&md5=1ac8a66426993770ae33b78ea35f5879</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrg1945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1945%26sid%3Dliteratum%253Aachs%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DKallin%26aufirst%3DE.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DJmjC-domain-containing%2520proteins%2520and%2520histone%2520demethylation%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2006%26volume%3D7%26spage%3D715%26epage%3D727%26doi%3D10.1038%2Fnrg1945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kooistra, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Molecular mechanisms and potential functions of histone demethylases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1038/nrm3327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrm3327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22473470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=297-311&author=S.+M.+Kooistraauthor=K.+Helin&title=Molecular+mechanisms+and+potential+functions+of+histone+demethylases&doi=10.1038%2Fnrm3327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms and potential functions of histone demethylases</span></div><div class="casAuthors">Kooistra, Susanne Marije; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-311</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone modifications are thought to regulate chromatin structure, transcription and other nuclear processes.  Histone methylation was originally believed to be an irreversible modification that could only be removed by histone eviction or by diln. during DNA replication.  However, the isolation of two families of enzymes that can demethylate histones has changed this notion.  The biochem. activities of these histone demethylases towards specific Lys residues on histones, and in some cases non-histone substrates, have highlighted their importance in developmental control, cell-fate decisions and disease.  Their ability to be regulated through protein-targeting complexes and post-translational modifications is also beginning to shed light on how they provide dynamic control during transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolbbZ5rmYmrVg90H21EOLACvtfcHk0ljghG9T4XkdUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D&md5=a5f5ffdac5c5658bc262d584faf5216f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrm3327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3327%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DS.%2BM.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DMolecular%2520mechanisms%2520and%2520potential%2520functions%2520of%2520histone%2520demethylases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D297%26epage%3D311%26doi%3D10.1038%2Fnrm3327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljghG9T4XkdUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Profumo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">A novel mammalian flavin-dependent histone demethylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">17775</span><span class="NLM_x">–</span> <span class="NLM_lpage">17782</span><span class="refDoi"> DOI: 10.1074/jbc.M109.003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1074%2Fjbc.M109.003087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19407342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17775-17782&author=A.+Karytinosauthor=F.+Fornerisauthor=A.+Profumoauthor=G.+Ciossaniauthor=E.+Battaglioliauthor=C.+Bindaauthor=A.+Mattevi&title=A+novel+mammalian+flavin-dependent+histone+demethylase&doi=10.1074%2Fjbc.M109.003087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mammalian Flavin-dependent Histone Demethylase</span></div><div class="casAuthors">Karytinos, Aristotele; Forneris, Federico; Profumo, Antonella; Ciossani, Giuseppe; Battaglioli, Elena; Binda, Claudia; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17775-17782</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of Lys residues on histone proteins is a well known and extensively characterized epigenetic mark.  The recent discovery of lysine-specific demethylase 1 (LSD1) demonstrated that lysine methylation can be dynamically controlled.  Among the histone demethylases so far identified, LSD1 has the unique feature of functioning through a flavin-dependent amine oxidn. reaction.  Data base anal. reveals that mammalian genomes contain a gene (AOF1, for amine-oxidase flavin-contg. domain 1) that is homologous to the LSD1-coding gene.  Here, we demonstrate that the protein encoded by AOF1 represents a second mammalian flavin-dependent histone demethylase, named LSD2.  The new demethylase is strictly specific for mono- and dimethylated Lys4 of histone H3, recognizes a long stretch of the H3 N-terminal tail, senses the presence of addnl. epigenetic marks on the histone substrate, and is covalently inhibited by tranylcypromine.  As opposed to LSD1, LSD2 does not form a biochem. stable complex with the C-terminal domain of the corepressor protein CoREST.  Furthermore, LSD2 contains a CW-type zinc finger motif with potential zinc-binding sites that are not present in LSD1.  We conclude that mammalian LSD2 represents a new flavin-dependent H3-Lys4 demethylase that features substrate specificity properties highly similar to those of LSD1 but is very likely to be part of chromatin-remodeling complexes that are distinct from those involving LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSPhtflZQU7Vg90H21EOLACvtfcHk0lj7O2xR7Z-6uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D&md5=4fbe32f2274de75915e2308781a214d1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.003087%26sid%3Dliteratum%253Aachs%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DProfumo%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520novel%2520mammalian%2520flavin-dependent%2520histone%2520demethylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17775%26epage%3D17782%26doi%3D10.1074%2Fjbc.M109.003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooch, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1038/nature04021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature04021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16079794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation&doi=10.1038%2Fnature04021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Cooch, Neil; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">432-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors have previously described a multiprotein complex termed the BHC or BRAF-HDAC complex, which is required for the repression of neuronal-specific genes.  The authors have shown that the BHC complex is recruited by a neuronal silencer, REST (RE1-silencing transcription factor), and mediates the repression of REST-responsive genes.  BHC is a multiprotein complex consisting of two enzymic activities: a histone deacetylase (HDAC1 or 2) and a recently described histone demethylase (BHC110, also known as LSD1 or AOF2).  Here the authors show that BHC110-contg. complexes show a nearly fivefold increase in demethylation of histone H3 lysine 4 (H3K4) compared to recombinant BHC110.  Furthermore, recombinant BHC110 is unable to demethylate H3K4 on nucleosomes, but BHC110-contg. complexes readily demethylate nucleosomes.  In vitro reconstitution of the BHC complex using recombinant subunits reveals an essential role for the REST corepressor CoREST, not only in stimulating demethylation on core histones but also promoting demethylation of nucleosomal substrates.  The authors find that nucleosomal demethylation is the result of CoREST enhancing the assocn. between BHC110 and nucleosomes.  Depletion of CoREST in in vivo cell culture results in de-repression of REST-responsive gene expression and increased methylation of H3K4.  Together, these results highlight an essential role for CoREST in demethylation of H3K4 both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmTdZApayNhrVg90H21EOLACvtfcHk0lj7O2xR7Z-6uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL8%253D&md5=535a6d800bd6672192be133516b4dee7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature04021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04021%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435%26doi%3D10.1038%2Fnature04021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Shi, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwase, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Regulation of LSD1 histone demethylase activity by its associated factors</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">857</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span><span class="refDoi"> DOI: 10.1016/j.molcel.2005.08.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2005.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16140033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=857-864&author=Y.+J.+Shiauthor=C.+Matsonauthor=F.+Lanauthor=S.+Iwaseauthor=T.+Babaauthor=Y.+Shi&title=Regulation+of+LSD1+histone+demethylase+activity+by+its+associated+factors&doi=10.1016%2Fj.molcel.2005.08.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of LSD1 histone demethylase activity by its associated factors</span></div><div class="casAuthors">Shi, Yu-Jiang; Matson, Caitlin; Lan, Fei; Iwase, Shigeki; Baba, Tadashi; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">857-864</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a recently identified human lysine (K)-specific histone demethylase.  LSD1 is assocd. with HDAC1/2, with CoREST, a SANT domain-contg. corepressor, and with BHC80, a PHD domain-contg. protein, among others.  We show that CoREST confers on LSD1 the ability to demethylate nucleosomal substrates and that it protects LSD1 from proteasomal degrdn. in vivo.  We find hyperacetylated nucleosomes less susceptible to CoREST/LSD1-mediated demethylation, suggesting that hypoacetylated nucleosomes may be the preferred physiol. substrates.  This raises the possibility that histone deacetylases and LSD1 may collaborate to generate a repressive chromatin environment.  Consistent with this model, TSA treatment results in derepression of LSD1 target genes.  While CoREST pos. regulates LSD1 function, BHC80 inhibits CoREST/LSD1-mediated demethylation in vitro and may therefore confer neg. regulation.  Taken together, these findings suggest that LSD1-mediated histone demethylation is regulated dynamically in vivo.  This is expected to have profound effects on gene expression under both physiol. and pathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraharVnxcaMLVg90H21EOLACvtfcHk0liBws8JMAr7ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK&md5=34ba238112151c6596dd6f6c7caa043e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BJ.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DIwase%26aufirst%3DS.%26aulast%3DBaba%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DRegulation%2520of%2520LSD1%2520histone%2520demethylase%2520activity%2520by%2520its%2520associated%2520factors%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D857%26epage%3D864%26doi%3D10.1016%2Fj.molcel.2005.08.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">You, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grozinger, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">CoREST is an integral component of the CoREST- human histone deacetylase complex</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1454</span><span class="NLM_x">–</span> <span class="NLM_lpage">1458</span><span class="refDoi"> DOI: 10.1073/pnas.98.4.1454</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1073%2Fpnas.98.4.1454" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=11171972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsVWjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=1454-1458&author=A.+Youauthor=J.+K.+Tongauthor=C.+M.+Grozingerauthor=S.+L.+Schreiber&title=CoREST+is+an+integral+component+of+the+CoREST-+human+histone+deacetylase+complex&doi=10.1073%2Fpnas.98.4.1454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">CoREST is an integral component of the CoREST-human histone deacetylase complex</span></div><div class="casAuthors">You, Angie; Tong, Jeffrey K.; Grozinger, Christina M.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1454-1458</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here we describe the components of a histone deacetylase (HDAC) complex that we term the CoREST-HDAC complex.  CoREST-HDAC is composed of polypeptides distinct from previously characterized HDAC1/2-contg. complexes such as the mSin3 and nucleosome remodeling and deacetylating (NRD, also named NURD, NuRD) complex.  Interestingly, we do not observe RbAp46 and RbAp48 in this complex, although these proteins have been obsd. in all previously identified complexes and are thought to be part of an HDAC1/2 core.  We identify the transcriptional corepressor CoREST and a protein with homol. to polyamine oxidases as components of CoREST-HDAC.  The HDAC1/2-interacting region of CoREST is mapped to a 179-aa region contg. a SANT domain, a domain found in other HDAC1/2-interacting proteins such as NCoR, MTA1, and MTA2.  Furthermore, we demonstrate that the corepressor function of CoREST depends on this region.  Although CoREST initially was cloned as a corepressor to REST (RE1 silencing transcription factor/neural restrictive silencing factor), we find no evidence for the existence of the eight-zinc finger REST transcription factor as an interacting partner in this complex; however, we do find evidence for assocn. of the putative oncogene ZNF 217 that contains eight zinc fingers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1PVa02kLm-7Vg90H21EOLACvtfcHk0liBws8JMAr7ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsVWjt7k%253D&md5=f584da096c4382ccf885a2d480281fba</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.98.4.1454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.98.4.1454%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DJ.%2BK.%26aulast%3DGrozinger%26aufirst%3DC.%2BM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCoREST%2520is%2520an%2520integral%2520component%2520of%2520the%2520CoREST-%2520human%2520histone%2520deacetylase%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D1454%26epage%3D1458%26doi%3D10.1073%2Fpnas.98.4.1454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Y.</span><span> </span><span class="NLM_article-title">LSD1 is a subunit of the NuRD complex and targets the metastasis programs in breast cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">660</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.05.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2009.05.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19703393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2009&pages=660-672&author=Y.+Wangauthor=H.+Zhangauthor=Y.+Chenauthor=Y.+Sunauthor=F.+Yangauthor=W.+Yuauthor=J.+Liangauthor=L.+Sunauthor=X.+Yangauthor=L.+Shiauthor=R.+Liauthor=Y.+Liauthor=Y.+Zhangauthor=Q.+Liauthor=X.+Yiauthor=Y.+Shang&title=LSD1+is+a+subunit+of+the+NuRD+complex+and+targets+the+metastasis+programs+in+breast+cancer&doi=10.1016%2Fj.cell.2009.05.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 is a subunit of the NuRD complex and targets the metastasis programs in breast cancer</span></div><div class="casAuthors">Wang, Yan; Zhang, Hua; Chen, Yupeng; Sun, Yimin; Yang, Fen; Yu, Wenhua; Liang, Jing; Sun, Luyang; Yang, Xiaohan; Shi, Lei; Li, Ruifang; Li, Yanyan; Zhang, Yu; Li, Qian; Yi, Xia; Shang, Yongfeng</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) exerts pathway-specific activity in animal development and has been linked to several high-risk cancers.  Here, we report that LSD1 is an integral component of the Mi-2/nucleosome remodeling and deacetylase (NuRD) complex.  Transcriptional target anal. revealed that the LSD1/NuRD complexes regulate several cellular signaling pathways including TGFβ1 signaling pathway that are critically involved in cell proliferation, survival, and epithelial-to-mesenchymal transition.  We demonstrated that LSD1 inhibits the invasion of breast cancer cells in vitro and suppresses breast cancer metastatic potential in vivo.  We found that LSD1 is downregulated in breast carcinomas and that its level of expression is neg. correlated with that of TGFβ1.  Our data provide a mol. basis for the interplay of histone demethylation and deacetylation in chromatin remodeling.  By enlisting LSD1, the NuRD complex expands its chromatin remodeling capacity to include ATPase, histone deacetylase, and histone demethylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpngSfM5Ree9LVg90H21EOLACvtfcHk0ljkGaqpIHEbNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73F&md5=5b62ee7d574671b5cda39a2d173b8491</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.05.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.05.050%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DYi%26aufirst%3DX.%26aulast%3DShang%26aufirst%3DY.%26atitle%3DLSD1%2520is%2520a%2520subunit%2520of%2520the%2520NuRD%2520complex%2520and%2520targets%2520the%2520metastasis%2520programs%2520in%2520breast%2520cancer%26jtitle%3DCell%26date%3D2009%26volume%3D138%26spage%3D660%26epage%3D672%26doi%3D10.1016%2Fj.cell.2009.05.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Tong, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassig, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnitzler, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingston, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">Chromatin deacetylation by an ATP-dependent nucleosome remodelling complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">395</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">921</span><span class="refDoi"> DOI: 10.1038/27699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2F27699" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=1998&pages=917-921&author=J.+K.+Tongauthor=C.+A.+Hassigauthor=G.+R.+Schnitzlerauthor=R.+E.+Kingstonauthor=S.+L.+Schreiber&title=Chromatin+deacetylation+by+an+ATP-dependent+nucleosome+remodelling+complex&doi=10.1038%2F27699"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2F27699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F27699%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DJ.%2BK.%26aulast%3DHassig%26aufirst%3DC.%2BA.%26aulast%3DSchnitzler%26aufirst%3DG.%2BR.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DChromatin%2520deacetylation%2520by%2520an%2520ATP-dependent%2520nucleosome%2520remodelling%2520complex%26jtitle%3DNature%26date%3D1998%26volume%3D395%26spage%3D917%26epage%3D921%26doi%3D10.1038%2F27699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Barski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuddapah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roh, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schones, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chepelev, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">High-resolution profiling of histone methylations in the human genome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">837</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2007.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=17512414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=823-837&author=A.+Barskiauthor=S.+Cuddapahauthor=K.+Cuiauthor=T.+Y.+Rohauthor=D.+E.+Schonesauthor=Z.+Wangauthor=G.+Weiauthor=I.+Chepelevauthor=K.+Zhao&title=High-resolution+profiling+of+histone+methylations+in+the+human+genome&doi=10.1016%2Fj.cell.2007.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution profiling of histone methylations in the human genome</span></div><div class="casAuthors">Barski, Artem; Cuddapah, Suresh; Cui, Kairong; Roh, Tae-Young; Schones, Dustin E.; Wang, Zhibin; Wei, Gang; Chepelev, Iouri; Zhao, Keji</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-837</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone modifications are implicated in influencing gene expression.  We have generated high-resoln. maps for the genome-wide distribution of 20 histone lysine and arginine methylations as well as histone variant H2A.Z, RNA polymerase II, and the insulator binding protein CTCF across the human genome using the Solexa 1G sequencing technol.  Typical patterns of histone methylations exhibited at promoters, insulators, enhancers, and transcribed regions are identified.  The monomethylations of H3K27, H3K9, H4K20, H3K79, and H2BK5 are all linked to gene activation, whereas trimethylations of H3K27, H3K9, and H3K79 are linked to repression.  H2A.Z assocs. with functional regulatory elements, and CTCF marks boundaries of histone methylation domains.  Chromosome banding patterns are correlated with unique patterns of histone modifications.  Chromosome breakpoints detected in T cell cancers frequently reside in chromatin regions assocd. with H3K4 methylations.  Our data provide new insights into the function of histone methylation and chromatin organization in genome function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxkecQmWJMZbVg90H21EOLACvtfcHk0li4LTWwHJ-8kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFKjsro%253D&md5=8e37e2d641f10e3cdd3ed51d1da82c96</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBarski%26aufirst%3DA.%26aulast%3DCuddapah%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DK.%26aulast%3DRoh%26aufirst%3DT.%2BY.%26aulast%3DSchones%26aufirst%3DD.%2BE.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DChepelev%26aufirst%3DI.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DHigh-resolution%2520profiling%2520of%2520histone%2520methylations%2520in%2520the%2520human%2520genome%26jtitle%3DCell%26date%3D2007%26volume%3D129%26spage%3D823%26epage%3D837%26doi%3D10.1016%2Fj.cell.2007.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+Mullerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+Guntherauthor=R.+Buettnerauthor=R.+Schule&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0li4LTWwHJ-8kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DGunther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Nair, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brann, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tekmal, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vadlamudi, R. K.</span><span> </span><span class="NLM_article-title">PELP1 is a reader of histone H3 methylation that facilitates oestrogen receptor-alpha target gene activation by regulating lysine demethylase 1 specificity</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span><span class="refDoi"> DOI: 10.1038/embor.2010.62</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fembor.2010.62" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=438-444&author=S.+S.+Nairauthor=B.+C.+Nairauthor=V.+Cortezauthor=D.+Chakravartyauthor=E.+Metzgerauthor=R.+Schuleauthor=D.+W.+Brannauthor=R.+R.+Tekmalauthor=R.+K.+Vadlamudi&title=PELP1+is+a+reader+of+histone+H3+methylation+that+facilitates+oestrogen+receptor-alpha+target+gene+activation+by+regulating+lysine+demethylase+1+specificity&doi=10.1038%2Fembor.2010.62"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fembor.2010.62&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2010.62%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DS.%2BS.%26aulast%3DNair%26aufirst%3DB.%2BC.%26aulast%3DCortez%26aufirst%3DV.%26aulast%3DChakravarty%26aufirst%3DD.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBrann%26aufirst%3DD.%2BW.%26aulast%3DTekmal%26aufirst%3DR.%2BR.%26aulast%3DVadlamudi%26aufirst%3DR.%2BK.%26atitle%3DPELP1%2520is%2520a%2520reader%2520of%2520histone%2520H3%2520methylation%2520that%2520facilitates%2520oestrogen%2520receptor-alpha%2520target%2520gene%2520activation%2520by%2520regulating%2520lysine%2520demethylase%25201%2520specificity%26jtitle%3DEMBO%2520Rep.%26date%3D2010%26volume%3D11%26spage%3D438%26epage%3D444%26doi%3D10.1038%2Fembor.2010.62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Biswas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basrur, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elenitoba-Johnson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Function of leukemogenic mixed lineage leukemia 1 (MLL) fusion proteins through distinct partner protein complexes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">15751</span><span class="NLM_x">–</span> <span class="NLM_lpage">15756</span><span class="refDoi"> DOI: 10.1073/pnas.1111498108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1073%2Fpnas.1111498108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=15751-15756&author=D.+Biswasauthor=T.+A.+Milneauthor=V.+Basrurauthor=J.+Kimauthor=K.+S.+Elenitoba-Johnsonauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Function+of+leukemogenic+mixed+lineage+leukemia+1+%28MLL%29+fusion+proteins+through+distinct+partner+protein+complexes&doi=10.1073%2Fpnas.1111498108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111498108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111498108%26sid%3Dliteratum%253Aachs%26aulast%3DBiswas%26aufirst%3DD.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DBasrur%26aufirst%3DV.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DFunction%2520of%2520leukemogenic%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL%2529%2520fusion%2520proteins%2520through%2520distinct%2520partner%2520protein%2520complexes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D15751%26epage%3D15756%26doi%3D10.1073%2Fpnas.1111498108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozovskaia, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croce, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canaani, E.</span><span> </span><span class="NLM_article-title">ALL-1 is a histone methyltransferase that assembles a supercomplex of proteins involved in transcriptional regulation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1128</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00740-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2FS1097-2765%2802%2900740-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1119-1128&author=T.+Nakamuraauthor=T.+Moriauthor=S.+Tadaauthor=W.+Krajewskiauthor=T.+Rozovskaiaauthor=R.+Wassellauthor=G.+Duboisauthor=A.+Mazoauthor=C.+M.+Croceauthor=E.+Canaani&title=ALL-1+is+a+histone+methyltransferase+that+assembles+a+supercomplex+of+proteins+involved+in+transcriptional+regulation&doi=10.1016%2FS1097-2765%2802%2900740-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900740-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900740-2%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DTada%26aufirst%3DS.%26aulast%3DKrajewski%26aufirst%3DW.%26aulast%3DRozovskaia%26aufirst%3DT.%26aulast%3DWassell%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DG.%26aulast%3DMazo%26aufirst%3DA.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCanaani%26aufirst%3DE.%26atitle%3DALL-1%2520is%2520a%2520histone%2520methyltransferase%2520that%2520assembles%2520a%2520supercomplex%2520of%2520proteins%2520involved%2520in%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1119%26epage%3D1128%26doi%3D10.1016%2FS1097-2765%2802%2900740-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">Histone H3 lysine 4 demethylation is a target of nonselective antidepressive medications</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+lysine+4+demethylation+is+a+target+of+nonselective+antidepressive+medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 μM.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0li7P1dwP3nVbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520lysine%25204%2520demethylation%2520is%2520a%2520target%2520of%2520nonselective%2520antidepressive%2520medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanoni, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Human histone demethylase LSD1 reads the histone code</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">41360</span><span class="NLM_x">–</span> <span class="NLM_lpage">41365</span><span class="refDoi"> DOI: 10.1074/jbc.M509549200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1074%2Fjbc.M509549200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=16223729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=41360-41365&author=F.+Fornerisauthor=C.+Bindaauthor=M.+A.+Vanoniauthor=E.+Battaglioliauthor=A.+Mattevi&title=Human+histone+demethylase+LSD1+reads+the+histone+code&doi=10.1074%2Fjbc.M509549200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Human Histone Demethylase LSD1 Reads the Histone Code</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Vanoni, Maria Antonietta; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41360-41365</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human histone demethylase LSD1 is a flavin-dependent amine oxidase that catalyzes the specific removal of Me groups from mono- and dimethylated Lys4 of histone H3.  The N-terminal tail of H3 is subject to various covalent modifications, and a fundamental question in LSD1 biol. is how these epigenetic marks affect the demethylase activity.  The authors show that LSD1 does not have a strong preference for mono- or dimethylated Lys4 of H3.  Substrate recognition is not confined to the residues neighboring Lys4, but it requires a sufficiently long peptide segment consisting of the N-terminal 20 amino acids of H3.  Electrostatic interactions are an important factor in protein-substrate recognition, as indicated by the high sensitivity of Km to ionic strength.  The authors have probed LSD1 for its ability to demethylate Lys4 in presence of a second modification on the same peptide substrate.  Methylation of Lys9 does not affect enzyme catalysis.  Conversely, Lys9 acetylation causes an almost 6-fold increase in the Km value, whereas phosphorylation of Ser10 totally abolishes activity.  LSD1 is inhibited by a demethylated peptide with an inhibition const. of 1.8 μM, suggesting that LSD1 can bind to H3 independently of Lys4 methylation.  LSD1 is a chromatin-modifying enzyme, which is able to read different epigenetic marks on the histone N-terminal tail and can serve as a docking module for the stabilization of the assocd. corepressor complex(es) on chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxu9eUWof47Vg90H21EOLACvtfcHk0liyu6FM5JEPsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L&md5=f39f7c79f1f789fb06ca7dd8a98752fe</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M509549200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M509549200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DVanoni%26aufirst%3DM.%2BA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DHuman%2520histone%2520demethylase%2520LSD1%2520reads%2520the%2520histone%2520code%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D41360%26epage%3D41365%26doi%3D10.1074%2Fjbc.M509549200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Demethylation of RB regulator MYPT1 by histone demethylase LSD1 promotes cell cycle progression in cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span><span class="NLM_x">–</span> <span class="NLM_lpage">660</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-2446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1158%2F0008-5472.CAN-10-2446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21115810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=655-660&author=H.+S.+Choauthor=T.+Suzukiauthor=N.+Dohmaeauthor=S.+Hayamiauthor=M.+Unokiauthor=M.+Yoshimatsuauthor=G.+Toyokawaauthor=M.+Takawaauthor=T.+Chenauthor=J.+K.+Kurashauthor=H.+I.+Fieldauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Demethylation+of+RB+regulator+MYPT1+by+histone+demethylase+LSD1+promotes+cell+cycle+progression+in+cancer+cells&doi=10.1158%2F0008-5472.CAN-10-2446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Demethylation of RB Regulator MYPT1 by Histone Demethylase LSD1 Promotes Cell Cycle Progression in Cancer Cells</span></div><div class="casAuthors">Cho, Hyun-Soo; Suzuki, Takehiro; Dohmae, Naoshi; Hayami, Shinya; Unoki, Motoko; Yoshimatsu, Masanori; Toyokawa, Gouji; Takawa, Masashi; Chen, Taiping; Kurash, Julia K.; Field, Helen I.; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">655-660</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 (also known as KDM1 and AOF2) is active in various cancer cells, but its biol. significance in human carcinogenesis is unexplored.  In this study, we explored hypothesized interactions between LSD1 and MYPT1, a known regulator of RB1 phosphorylation.  We found that MYPT1 was methylated in vitro and in vivo by histone lysine methyltransferase SETD7 and demethylated by LSD1, identifying Lys 442 of MYPT1 as a target for methylation/demethylation by these enzymes.  LSD1 silencing increased MYPT1 protein levels, decreasing the steady state level of phosphorylated RB1 (Ser 807/811) and reducing E2F activity.  MYPT1 methylation status influenced the affinity of MYPT1 for the ubiquitin-proteasome pathway of protein turnover.  MYPT1 was unstable in murine cells deficient in SETD7, supporting the concept that MYPT1 protein stability is physiol. regulated by methylation status.  LSD1 overexpression could activate RB1 phosphorylation by inducing a destabilization of MYPT1 protein.  Taken together, our results comprise a novel cell cycle regulatory mechanism mediated by methylation/demethylation dynamics, and they reveal the significance of LSD1 overexpression in human carcinogenesis.  Cancer Res; 71(3); 655-60.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh4fxBs2QNSrVg90H21EOLACvtfcHk0liyu6FM5JEPsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D&md5=4812d562b79453755b7c2eb8f74cc3eb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2446%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDemethylation%2520of%2520RB%2520regulator%2520MYPT1%2520by%2520histone%2520demethylase%2520LSD1%2520promotes%2520cell%2520cycle%2520progression%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D655%26epage%3D660%26doi%3D10.1158%2F0008-5472.CAN-10-2446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espejo, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">p53 is regulated by the lysine demethylase LSD1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.1038/nature06092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnature06092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=17805299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFWrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=105-108&author=J.+Huangauthor=R.+Senguptaauthor=A.+B.+Espejoauthor=M.+G.+Leeauthor=J.+A.+Dorseyauthor=M.+Richterauthor=S.+Opravilauthor=R.+Shiekhattarauthor=M.+T.+Bedfordauthor=T.+Jenuweinauthor=S.+L.+Berger&title=p53+is+regulated+by+the+lysine+demethylase+LSD1&doi=10.1038%2Fnature06092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">p53 is regulated by the lysine demethylase LSD1</span></div><div class="casAuthors">Huang, Jing; Sengupta, Roopsha; Espejo, Alexsandra B.; Lee, Min Gyu; Dorsey, Jean A.; Richter, Mario; Opravil, Susanne; Shiekhattar, Ramin; Bedford, Mark T.; Jenuwein, Thomas; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7158</span>),
    <span class="NLM_cas:pages">105-108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P53, the tumor suppressor and transcriptional activator, is regulated by numerous post-translational modifications, including lysine methylation.  Histone lysine methylation has recently been shown to be reversible; however, it is not known whether non-histone proteins are substrates for demethylation.  Here we show that, in human cells, the histone lysine-specific demethylase LSD1 (refs. 3, 4) interacts with p53 to repress p53-mediated transcriptional activation and to inhibit the role of p53 in promoting apoptosis.  We find that, in vitro, LSD1 removes both monomethylation (K370me1) and dimethylation (K370me2) at K370, a previously identified Smyd2-dependent monomethylation site.  However, in vivo, LSD1 shows a strong preference to reverse K370me2, which is performed by a distinct, but unknown, methyltransferase.  Our results indicate that K370me2 has a different role in regulating p53 from that of K370me1: K370me1 represses p53 function, whereas K370me2 promotes assocn. with the coactivator 53BP1 (p53-binding protein 1) through tandem Tudor domains in 53BP1.  Further, LSD1 represses p53 function through the inhibition of interaction of p53 with 53BP1.  These observations show that p53 is dynamically regulated by lysine methylation and demethylation and that the methylation status at a single lysine residue confers distinct regulatory output.  Lysine methylation therefore provides similar regulatory complexity for non-histone proteins and for histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ggpbeqDqsbVg90H21EOLACvtfcHk0ljSAMZ4NcERNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFWrs7w%253D&md5=8bf23f859f97d21bf7008a6d6e1c4946</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature06092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06092%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DEspejo%26aufirst%3DA.%2BB.%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DShiekhattar%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3Dp53%2520is%2520regulated%2520by%2520the%2520lysine%2520demethylase%2520LSD1%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D105%26epage%3D108%26doi%3D10.1038%2Fnature06092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Kontaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talianidis, I.</span><span> </span><span class="NLM_article-title">Lysine methylation regulates E2F1-induced cell death</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span><span class="refDoi"> DOI: 10.1016/j.molcel.2010.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2010.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=20603083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=152-160&author=H.+Kontakiauthor=I.+Talianidis&title=Lysine+methylation+regulates+E2F1-induced+cell+death&doi=10.1016%2Fj.molcel.2010.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methylation regulates E2F1-induced cell death</span></div><div class="casAuthors">Kontaki, Haroula; Talianidis, Iannis</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">152-160</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone-modifying enzymes can regulate DNA damage-induced apoptosis through modulation of p53 function.  Here, we show that, in p53-deficient tumor cells, Set9 and LSD1 regulate DNA damage-induced cell death in a manner opposite to that obsd. in p53+/+ cells, via modulation of E2F1 stabilization.  Set9 methylates E2F1 at lysine-185, which prevents E2F1 accumulation during DNA damage and activation of its proapoptotic target gene p73.  This Me mark is removed by LSD1, which is required for E2F1 stabilization and apoptotic function.  The mol. mechanism involves cross-talks between Lys methylation and other covalent modifications that affect E2F1 stability.  Methylation at Lys-185 inhibits acetylation and phosphorylation at distant positions and, in parallel, stimulates ubiquitination and degrdn. of the protein.  The findings illustrate that the function of methyltransferases can have opposing biol. outcomes depending on the specificity of transcription factor targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXEj98lICfQ7Vg90H21EOLACvtfcHk0lgyRFG4zv-FKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D&md5=12b34f33cde33be3f37a53f8115aecde</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DKontaki%26aufirst%3DH.%26aulast%3DTalianidis%26aufirst%3DI.%26atitle%3DLysine%2520methylation%2520regulates%2520E2F1-induced%2520cell%2520death%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D152%26epage%3D160%26doi%3D10.1016%2Fj.molcel.2010.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lgyRFG4zv-FKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">21499</span><span class="NLM_x">–</span> <span class="NLM_lpage">21504</span><span class="refDoi"> DOI: 10.1073/pnas.1016147107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1073%2Fpnas.1016147107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21098664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=21499-21504&author=J.+Yangauthor=J.+Huangauthor=M.+Dasguptaauthor=N.+Searsauthor=M.+Miyagiauthor=B.+Wangauthor=M.+R.+Chanceauthor=X.+Chenauthor=Y.+Duauthor=Y.+Wangauthor=L.+Anauthor=Q.+Wangauthor=T.+Luauthor=X.+Zhangauthor=Z.+Wangauthor=G.+R.+Stark&title=Reversible+methylation+of+promoter-bound+STAT3+by+histone-modifying+enzymes&doi=10.1073%2Fpnas.1016147107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span></div><div class="casAuthors">Yang, Jinbo; Huang, Jing; Dasgupta, Maupali; Sears, Nathan; Miyagi, Masaru; Wang, Benlian; Chance, Mark R.; Chen, Xing; Du, Yuping; Wang, Yuxin; An, Lizhe; Wang, Qin; Lu, Tao; Zhang, Xiaodong; Wang, Zhenghe; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">21499-21504, S21499/1-S21499/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Following its tyrosine phosphorylation, STAT3 is methylated on K140 by the histone Me transferase SET9 and demethylated by LSD1 when it is bound to a subset of the promoters that it activates.  Methylation of K140 is a neg. regulatory event, because its blockade greatly increases the steady-state amt. of activated STAT3 and the expression of many (i.e., SOCS3) but not all (i.e., CD14) STAT3 target genes.  Biol. relevance is shown by the observation that overexpression of SOCS3 when K140 cannot be methylated blocks the ability of cells to activate STAT3 in response to IL-6.  K140 methylation does not occur with mutants of STAT3 that do not enter nuclei or bind to DNA.  Following treatment with IL-6, events at the SOCS3 promoter occur in an ordered sequence, as shown by chromatin immunopptns.  Y705-phosphoryl-STAT3 binds first and S727 is then phosphorylated, followed by the coincident binding of SET9 and dimethylation of K140, and lastly by the binding of LSD1.  We conclude that the lysine methylation of promoter-bound STAT3 leads to biol. important down-regulation of the dependent responses and that SET9, which is known to help provide an activating methylation mark to H3K4, is recruited to the newly activated SOCS3 promoter by STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTNJmJCGOKwbVg90H21EOLACvtfcHk0lhd5bC6SMHCJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM&md5=255afed29b2b72fde8e2baf3d09a85dd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016147107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016147107%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDasgupta%26aufirst%3DM.%26aulast%3DSears%26aufirst%3DN.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DReversible%2520methylation%2520of%2520promoter-bound%2520STAT3%2520by%2520histone-modifying%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D21499%26epage%3D21504%26doi%3D10.1073%2Fpnas.1016147107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Schuleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+is+strongly+expressed+in+poorly+differentiated+neuroblastoma%3A+implications+for+therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0lhd5bC6SMHCJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520strongly%2520expressed%2520in%2520poorly%2520differentiated%2520neuroblastoma%253A%2520implications%2520for%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen receptor coactivators lysine-specific histone demethylase 1 and four and a half LIM domain protein 2 predict risk of prostate cancer recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Schuleauthor=R.+Buettner&title=Androgen+receptor+coactivators+lysine-specific+histone+demethylase+1+and+four+and+a+half+LIM+domain+protein+2+predict+risk+of+prostate+cancer+recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520receptor%2520coactivators%2520lysine-specific%2520histone%2520demethylase%25201%2520and%2520four%2520and%2520a%2520half%2520LIM%2520domain%2520protein%25202%2520predict%2520risk%2520of%2520prostate%2520cancer%2520recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Lv, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Over-expression of LSD1 promotes proliferation, migration and invasion in non-small cell lung cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e35065</span><span class="refDoi"> DOI: 10.1371/journal.pone.0035065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1371%2Fjournal.pone.0035065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e35065&author=T.+Lvauthor=D.+Yuanauthor=X.+Miaoauthor=Y.+Lvauthor=P.+Zhanauthor=X.+Shenauthor=Y.+Song&title=Over-expression+of+LSD1+promotes+proliferation%2C+migration+and+invasion+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0035065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0035065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0035065%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DT.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DOver-expression%2520of%2520LSD1%2520promotes%2520proliferation%252C%2520migration%2520and%2520invasion%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De35065%26doi%3D10.1371%2Fjournal.pone.0035065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Derr, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoesel, A. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goossens-Beumer, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sajet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dekker-Ensink, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Kruijf, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastiaannet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, V. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Velde, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuppen, P. J.</span><span> </span><span class="NLM_article-title">High nuclear expression levels of histone-modifying enzymes LSD1, HDAC2 and SIRT1 in tumor cells correlate with decreased survival and increased relapse in breast cancer patients</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="refDoi"> DOI: 10.1186/1471-2407-14-604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1186%2F1471-2407-14-604" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=604&author=R.+S.+Derrauthor=A.+Q.+van+Hoeselauthor=A.+Benardauthor=I.+J.+Goossens-Beumerauthor=A.+Sajetauthor=N.+G.+Dekker-Ensinkauthor=E.+M.+de+Kruijfauthor=E.+Bastiaannetauthor=V.+T.+Smitauthor=C.+J.+van+de+Veldeauthor=P.+J.+Kuppen&title=High+nuclear+expression+levels+of+histone-modifying+enzymes+LSD1%2C+HDAC2+and+SIRT1+in+tumor+cells+correlate+with+decreased+survival+and+increased+relapse+in+breast+cancer+patients&doi=10.1186%2F1471-2407-14-604"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-604%26sid%3Dliteratum%253Aachs%26aulast%3DDerr%26aufirst%3DR.%2BS.%26aulast%3Dvan%2BHoesel%26aufirst%3DA.%2BQ.%26aulast%3DBenard%26aufirst%3DA.%26aulast%3DGoossens-Beumer%26aufirst%3DI.%2BJ.%26aulast%3DSajet%26aufirst%3DA.%26aulast%3DDekker-Ensink%26aufirst%3DN.%2BG.%26aulast%3Dde%2BKruijf%26aufirst%3DE.%2BM.%26aulast%3DBastiaannet%26aufirst%3DE.%26aulast%3DSmit%26aufirst%3DV.%2BT.%26aulast%3Dvan%2Bde%2BVelde%26aufirst%3DC.%2BJ.%26aulast%3DKuppen%26aufirst%3DP.%2BJ.%26atitle%3DHigh%2520nuclear%2520expression%2520levels%2520of%2520histone-modifying%2520enzymes%2520LSD1%252C%2520HDAC2%2520and%2520SIRT1%2520in%2520tumor%2520cells%2520correlate%2520with%2520decreased%2520survival%2520and%2520increased%2520relapse%2520in%2520breast%2520cancer%2520patients%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D604%26doi%3D10.1186%2F1471-2407-14-604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Schuleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+%28LSD1%29+is+highly+expressed+in+ER-negative+breast+cancers+and+a+biomarker+predicting+aggressive+biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0lj8iAzzRWbd8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520is%2520highly%2520expressed%2520in%2520ER-negative%2520breast%2520cancers%2520and%2520a%2520biomarker%2520predicting%2520aggressive%2520biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Yu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span> </span><span class="NLM_article-title">High expression of lysine-specific demethylase 1 correlates with poor prognosis of patients with esophageal squamous cell carcinoma</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">198</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2013.05.123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bbrc.2013.05.123" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2013&pages=192-198&author=Y.+Yuauthor=B.+Wangauthor=K.+Zhangauthor=Z.+Leiauthor=Y.+Guoauthor=H.+Xiaoauthor=J.+Wangauthor=L.+Fanauthor=C.+Lanauthor=Y.+Weiauthor=Q.+Maauthor=L.+Linauthor=C.+Maoauthor=X.+Yangauthor=X.+Chenauthor=Y.+Liauthor=Y.+Baiauthor=D.+Chen&title=High+expression+of+lysine-specific+demethylase+1+correlates+with+poor+prognosis+of+patients+with+esophageal+squamous+cell+carcinoma&doi=10.1016%2Fj.bbrc.2013.05.123"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2013.05.123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2013.05.123%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLei%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DLan%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DD.%26atitle%3DHigh%2520expression%2520of%2520lysine-specific%2520demethylase%25201%2520correlates%2520with%2520poor%2520prognosis%2520of%2520patients%2520with%2520esophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2013%26volume%3D437%26spage%3D192%26epage%3D198%26doi%3D10.1016%2Fj.bbrc.2013.05.123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Zhao, Z. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. B.</span><span> </span><span class="NLM_article-title">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span> <span class="citation_source-journal">World J. Gastroenterol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6651</span><span class="NLM_x">–</span> <span class="NLM_lpage">6656</span><span class="refDoi"> DOI: 10.3748/wjg.v18.i45.6651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.3748%2Fwjg.v18.i45.6651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=23236241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6651-6656&author=Z.+K.+Zhaoauthor=H.+F.+Yuauthor=D.+R.+Wangauthor=P.+Dongauthor=L.+Chenauthor=W.+G.+Wuauthor=W.+J.+Dingauthor=Y.+B.+Liu&title=Overexpression+of+lysine+specific+demethylase+1+predicts+worse+prognosis+in+primary+hepatocellular+carcinoma+patients&doi=10.3748%2Fwjg.v18.i45.6651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span></div><div class="casAuthors">Zhao, Ze-Kun; Yu, Hai-Feng; Wang, Dao-Rong; Dong, Ping; Chen, Lei; Wu, Wen-Guang; Ding, Wen-Jun; Liu, Ying-Bin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">6651-6656</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Aim: To investigate the clinicopathol. features and prognostic value of lysine specific demethylase 1 (LSD1) in hepatocellular carcinoma (HCC).  Methods: We examd. LSD1 expression in 60 paired liver cancer tissues and adjacent noncancerous tissues by quant. real time polymerase chain reaction (qRT-PCR) and Western blotting.  In addn., we analyzed LSD1 expression in 198 HCC samples by immunohistochem.  The relationship between LSD1 expression, clinicopathol. features and patient survival was investigated.  Results: Immunohistochem., Western blotting, and qRT-PCR consistently confirmed LSD1 overexpression in HCC tissues compared to adjacent non-neoplastic tissues (P < 0.01).  Addnl., immunostaining showed more LSD1-pos. cells in the higher tumor stage (T3-4) and tumor grade (G3) than in the lower tumor stage (T1-2, P < 0.001) and tumor grade (G1-2, P < 0.001), resp.  Moreover, HCC patients with high LSD1 expression had significantly lower 5-yr overall survival rates (P < 0.001) and lower 5-yr disease-free survival rates (P < 0.001), resp.  A Cox proportional hazards model further demonstrated that LSD1 over-expression was an independent predictor of poor prognosis for both 5-yr disease-free survival [hazards ratio (HR) = 1.426, 95%CI: 0.672-2.146, P < 0.001] and 5-yr overall survival (HR = 2.456, 95%CI: 1.234-3.932, P < 0.001) in HCC.  Conclusion: Our data suggest for the first time that the overexpression of LSD1 protein in HCC tissues indicates tumor progression and predicts poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqX4-UuO3NnLVg90H21EOLACvtfcHk0liLHJAsoQKnBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ&md5=c657f6b1e0b5e183c28fefbda4cc8745</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v18.i45.6651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v18.i45.6651%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%2BK.%26aulast%3DYu%26aufirst%3DH.%2BF.%26aulast%3DWang%26aufirst%3DD.%2BR.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DW.%2BG.%26aulast%3DDing%26aufirst%3DW.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%2BB.%26atitle%3DOverexpression%2520of%2520lysine%2520specific%2520demethylase%25201%2520predicts%2520worse%2520prognosis%2520in%2520primary%2520hepatocellular%2520carcinoma%2520patients%26jtitle%3DWorld%2520J.%2520Gastroenterol%26date%3D2012%26volume%3D18%26spage%3D6651%26epage%3D6656%26doi%3D10.3748%2Fwjg.v18.i45.6651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span><span class="refDoi"> DOI: 10.1002/ijc.25349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fijc.25349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=20333681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=574-586&author=S.+Hayamiauthor=J.+D.+Kellyauthor=H.+S.+Choauthor=M.+Yoshimatsuauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=D.+E.+Nealauthor=H.+Yamaueauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Overexpression+of+LSD1+contributes+to+human+carcinogenesis+through+chromatin+regulation+in+various+cancers&doi=10.1002%2Fijc.25349"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span></div><div class="casAuthors">Hayami Shinya; Kelly John D; Cho Hyun-Soo; Yoshimatsu Masanori; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Neal David E; Yamaue Hiroki; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A number of histone demethylases have been identified and biochemically characterized, but the pathological roles of their dysfunction in human disease like cancer have not been well understood.  Here, we demonstrate important roles of lysine-specific demethylase 1 (LSD1) in human carcinogenesis.  Expression levels of LSD1 are significantly elevated in human bladder carcinomas compared with nonneoplastic bladder tissues (p < 0.0001). cDNA microarray analysis also revealed its transactivation in lung and colorectal carcinomas.  LSD1-specific small interfering RNAs significantly knocked down its expression and resulted in suppression of proliferation of various bladder and lung cancer cell lines.  Concordantly, introduction of exogenous LSD1 expression promoted cell cycle progression of human embryonic kidney fibroblast cells.  Expression profile analysis showed that LSD1 could affect the expression of genes involved in various chromatin-modifying pathways such as chromatin remodeling at centromere, centromeric heterochromatin formation and chromatin assembly, indicating its essential roles in carcinogenesis through chromatin modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WuIHSmpohpd0fW6udTcc2ebNq7lH_FgYH7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D&md5=b15e7aa7b05c05dd8cfd781ed1a7e3bd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fijc.25349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25349%26sid%3Dliteratum%253Aachs%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520LSD1%2520contributes%2520to%2520human%2520carcinogenesis%2520through%2520chromatin%2520regulation%2520in%2520various%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D574%26epage%3D586%26doi%3D10.1002%2Fijc.25349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1/KDM1A) contributes to colorectal tumorigenesis via activation of the Wnt/beta-catenin pathway by down-regulating Dickkopf-1 (DKK1) [corrected]</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e70077</span><span class="refDoi"> DOI: 10.1371/journal.pone.0070077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1371%2Fjournal.pone.0070077" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e70077&author=Z.+Huangauthor=S.+Liauthor=W.+Songauthor=X.+Liauthor=Q.+Liauthor=Z.+Zhangauthor=Y.+Hanauthor=X.+Zhangauthor=S.+Miaoauthor=R.+Duauthor=L.+Wang&title=Lysine-specific+demethylase+1+%28LSD1%2FKDM1A%29+contributes+to+colorectal+tumorigenesis+via+activation+of+the+Wnt%2Fbeta-catenin+pathway+by+down-regulating+Dickkopf-1+%28DKK1%29+%5Bcorrected%5D&doi=10.1371%2Fjournal.pone.0070077"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0070077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0070077%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMiao%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%2520contributes%2520to%2520colorectal%2520tumorigenesis%2520via%2520activation%2520of%2520the%2520Wnt%252Fbeta-catenin%2520pathway%2520by%2520down-regulating%2520Dickkopf-1%2520%2528DKK1%2529%2520%255Bcorrected%255D%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De70077%26doi%3D10.1371%2Fjournal.pone.0070077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The deubiquitinase USP28 stabilizes LSD1 and confers stem-cell-like traits to breast cancer cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1016/j.celrep.2013.08.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.celrep.2013.08.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24075993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=224-236&author=Y.+Wuauthor=Y.+Wangauthor=X.+H.+Yangauthor=T.+Kangauthor=Y.+Zhaoauthor=C.+Wangauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+deubiquitinase+USP28+stabilizes+LSD1+and+confers+stem-cell-like+traits+to+breast+cancer+cells&doi=10.1016%2Fj.celrep.2013.08.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Deubiquitinase USP28 Stabilizes LSD1 and Confers Stem-Cell-like Traits to Breast Cancer Cells</span></div><div class="casAuthors">Wu, Yadi; Wang, Yifan; Yang, Xiuwei H.; Kang, Tiebang; Zhao, Yongxiang; Wang, Chi; Evers, B. Mark; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">224-236</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a crit. chromatin modulator that controls cellular pluripotency and differentiation through the demethylation of H3K4me1/2.  Overexpression of LSD1 has been obsd. in many types of tumors and is correlated with its oncogenic effects in tumorigenesis.  However, the mechanism leading to LSD1 upregulation in tumors remains unclear.  Using an unbiased siRNA screening against all the human deubiquitinases, we identified USP28 as a bona fide deubiquitinase of LSD1.  USP28 interacted with and stabilized LSD1 via deubiquitination.  USP28 overexpression correlated with LSD1 upregulation in multiple cancer cell lines and breast tumor samples.  Knockdown of USP28 resulted in LSD1 destabilization, leading to the suppression of cancer stem cell (CSC)-like characteristics in vitro and inhibition of tumorigenicity in vivo, which can be rescued by ectopic LSD1 expression.  Our study reveals a crit. mechanism underlying the epigenetic regulation by USP28 and provides another treatment approach against breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDEa93VRia7Vg90H21EOLACvtfcHk0lgQ1O7msMWURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM&md5=802c1b0561ed2a8ffa926aa4adb82405</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2013.08.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2013.08.030%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%2BH.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520deubiquitinase%2520USP28%2520stabilizes%2520LSD1%2520and%2520confers%2520stem-cell-like%2520traits%2520to%2520breast%2520cancer%2520cells%26jtitle%3DCell%2520Rep.%26date%3D2013%26volume%3D5%26spage%3D224%26epage%3D236%26doi%3D10.1016%2Fj.celrep.2013.08.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Suva, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rheinbay, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakimoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabkin, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riggi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cahill, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nahed, B. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martuza, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossetti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirosh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalek, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozenblatt-Rosen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernstein, B. E.</span><span> </span><span class="NLM_article-title">Reconstructing and reprogramming the tumor-propagating potential of glioblastoma stem-like cells</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">594</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.02.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.cell.2014.02.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24726434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVClurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=580-594&author=M.+L.+Suvaauthor=E.+Rheinbayauthor=S.+M.+Gillespieauthor=A.+P.+Patelauthor=H.+Wakimotoauthor=S.+D.+Rabkinauthor=N.+Riggiauthor=A.+S.+Chiauthor=D.+P.+Cahillauthor=B.+V.+Nahedauthor=W.+T.+Curryauthor=R.+L.+Martuzaauthor=M.+N.+Riveraauthor=N.+Rossettiauthor=S.+Kasifauthor=S.+Beikauthor=S.+Kadriauthor=I.+Tiroshauthor=I.+Wortmanauthor=A.+K.+Shalekauthor=O.+Rozenblatt-Rosenauthor=A.+Regevauthor=D.+N.+Louisauthor=B.+E.+Bernstein&title=Reconstructing+and+reprogramming+the+tumor-propagating+potential+of+glioblastoma+stem-like+cells&doi=10.1016%2Fj.cell.2014.02.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Reconstructing and Reprogramming the Tumor-Propagating Potential of Glioblastoma Stem-like Cells</span></div><div class="casAuthors">Suva, Mario L.; Rheinbay, Esther; Gillespie, Shawn M.; Patel, Anoop P.; Wakimoto, Hiroaki; Rabkin, Samuel D.; Riggi, Nicolo; Chi, Andrew S.; Cahill, Daniel P.; Nahed, Brian V.; Curry, William T.; Martuza, Robert L.; Rivera, Miguel N.; Rossetti, Nikki; Kasif, Simon; Beik, Samantha; Kadri, Sabah; Tirosh, Itay; Wortman, Ivo; Shalek, Alex K.; Rozenblatt-Rosen, Orit; Regev, Aviv; Louis, David N.; Bernstein, Bradley E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Developmental fate decisions are dictated by master transcription factors (TFs) that interact with cis-regulatory elements to direct transcriptional programs.  Certain malignant tumors may also depend on cellular hierarchies reminiscent of normal development but superimposed on underlying genetic aberrations.  In glioblastoma (GBM), a subset of stem-like tumor-propagating cells (TPCs) appears to drive tumor progression and underlie therapeutic resistance yet remain poorly understood.  Here, we identify a core set of neurodevelopmental TFs (POU3F2, SOX2, SALL2, and OLIG2) essential for GBM propagation.  These TFs coordinately bind and activate TPC-specific regulatory elements and are sufficient to fully reprogram differentiated GBM cells to "induced" TPCs, recapitulating the epigenetic landscape and phenotype of native TPCs.  We reconstruct a network model that highlights crit. interactions and identifies candidate therapeutic targets for eliminating TPCs.  Our study establishes the epigenetic basis of a developmental hierarchy in GBM, provides detailed insight into underlying gene regulatory programs, and suggests attendant therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJSQYzaStTT7Vg90H21EOLACvtfcHk0lgQ1O7msMWURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVClurw%253D&md5=72938847b07ffdc9e0c01e0c3cccb2ff</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.02.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.02.030%26sid%3Dliteratum%253Aachs%26aulast%3DSuva%26aufirst%3DM.%2BL.%26aulast%3DRheinbay%26aufirst%3DE.%26aulast%3DGillespie%26aufirst%3DS.%2BM.%26aulast%3DPatel%26aufirst%3DA.%2BP.%26aulast%3DWakimoto%26aufirst%3DH.%26aulast%3DRabkin%26aufirst%3DS.%2BD.%26aulast%3DRiggi%26aufirst%3DN.%26aulast%3DChi%26aufirst%3DA.%2BS.%26aulast%3DCahill%26aufirst%3DD.%2BP.%26aulast%3DNahed%26aufirst%3DB.%2BV.%26aulast%3DCurry%26aufirst%3DW.%2BT.%26aulast%3DMartuza%26aufirst%3DR.%2BL.%26aulast%3DRivera%26aufirst%3DM.%2BN.%26aulast%3DRossetti%26aufirst%3DN.%26aulast%3DKasif%26aufirst%3DS.%26aulast%3DBeik%26aufirst%3DS.%26aulast%3DKadri%26aufirst%3DS.%26aulast%3DTirosh%26aufirst%3DI.%26aulast%3DWortman%26aufirst%3DI.%26aulast%3DShalek%26aufirst%3DA.%2BK.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DLouis%26aufirst%3DD.%2BN.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26atitle%3DReconstructing%2520and%2520reprogramming%2520the%2520tumor-propagating%2520potential%2520of%2520glioblastoma%2520stem-like%2520cells%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D580%26epage%3D594%26doi%3D10.1016%2Fj.cell.2014.02.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">LSD1 inhibition: a therapeutic strategy in cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+Somervaille&title=LSD1+inhibition%3A+a+therapeutic+strategy+in+cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0liUSR5WGjYMpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DLSD1%2520inhibition%253A%2520a%2520therapeutic%2520strategy%2520in%2520cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Niebel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majores, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutgemann, I.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) in hematopoietic and lymphoid neoplasms</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span><span class="refDoi"> DOI: 10.1182/blood-2014-04-569525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1182%2Fblood-2014-04-569525" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=151-152&author=D.+Niebelauthor=J.+Kirfelauthor=V.+Janzenauthor=T.+Hollerauthor=M.+Majoresauthor=I.+Gutgemann&title=Lysine-specific+demethylase+1+%28LSD1%29+in+hematopoietic+and+lymphoid+neoplasms&doi=10.1182%2Fblood-2014-04-569525"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-04-569525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-04-569525%26sid%3Dliteratum%253Aachs%26aulast%3DNiebel%26aufirst%3DD.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DV.%26aulast%3DHoller%26aufirst%3DT.%26aulast%3DMajores%26aufirst%3DM.%26aulast%3DGutgemann%26aufirst%3DI.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520in%2520hematopoietic%2520and%2520lymphoid%2520neoplasms%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D151%26epage%3D152%26doi%3D10.1182%2Fblood-2014-04-569525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">The histone demethylase KDM1A sustains the oncogenic potential of MLL-AF9 leukemia stem cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+Somervaille&title=The+histone+demethylase+KDM1A+sustains+the+oncogenic+potential+of+MLL-AF9+leukemia+stem+cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lg7_ZjDFecdmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DThe%2520histone%2520demethylase%2520KDM1A%2520sustains%2520the%2520oncogenic%2520potential%2520of%2520MLL-AF9%2520leukemia%2520stem%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Benelkebir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgkinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duriez, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulleid, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">Enantioselective synthesis of tranylcypromine analogues as lysine demethylase (LSD1) inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3709</span><span class="NLM_x">–</span> <span class="NLM_lpage">3716</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bmc.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=21382717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3709-3716&author=H.+Benelkebirauthor=C.+Hodgkinsonauthor=P.+J.+Duriezauthor=A.+L.+Haydenauthor=R.+A.+Bulleidauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Ganesan&title=Enantioselective+synthesis+of+tranylcypromine+analogues+as+lysine+demethylase+%28LSD1%29+inhibitors&doi=10.1016%2Fj.bmc.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective synthesis of tranylcypromine analogues as lysine demethylase (LSD1) inhibitors</span></div><div class="casAuthors">Benelkebir, Hanae; Hodgkinson, Christopher; Duriez, Patrick J.; Hayden, Annette L.; Bulleid, Rosemary A.; Crabb, Simon J.; Packham, Graham; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3709-3716</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Asym. cyclopropanation of styrenes by tert-Bu diazoacetate followed by ester hydrolysis and Curtius rearrangement gave a series of tranylcypromine analogs as single enantiomers.  The o-, m-, and p-bromo analogs were all more active than tranylcypromine in a LSD1 enzyme assay.  The m- and p-bromo analogs were micromolar growth inhibitors of the LNCaP prostate cancer cell line as were the corresponding biphenyl analogs prepd. from the bromide by Suzuki crosscoupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuHGveNGE6AbVg90H21EOLACvtfcHk0lg7_ZjDFecdmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D&md5=a8517cf8753c76b1403b6354977fc2a2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DBenelkebir%26aufirst%3DH.%26aulast%3DHodgkinson%26aufirst%3DC.%26aulast%3DDuriez%26aufirst%3DP.%2BJ.%26aulast%3DHayden%26aufirst%3DA.%2BL.%26aulast%3DBulleid%26aufirst%3DR.%2BA.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEnantioselective%2520synthesis%2520of%2520tranylcypromine%2520analogues%2520as%2520lysine%2520demethylase%2520%2528LSD1%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3709%26epage%3D3716%26doi%3D10.1016%2Fj.bmc.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Gooden, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabadi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3047</span><span class="NLM_x">–</span> <span class="NLM_lpage">3051</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bmcl.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=18242989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3047-3051&author=D.+M.+Goodenauthor=D.+M.+Schmidtauthor=J.+A.+Pollockauthor=A.+M.+Kabadiauthor=D.+G.+McCafferty&title=Facile+synthesis+of+substituted+trans-2-arylcyclopropylamine+inhibitors+of+the+human+histone+demethylase+LSD1+and+monoamine+oxidases+A+and+B&doi=10.1016%2Fj.bmcl.2008.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span></div><div class="casAuthors">Gooden, David M.; Schmidt, Dawn M. Z.; Pollock, Julie A.; Kabadi, Ami M.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3047-3051</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A facile synthetic route to substituted trans-2-arylcyclopropylamines was developed to provide access to mechanism-based inhibitors of the human flavoenzyme oxidase lysine-specific histone demethylase LSD1 and related enzyme family members such as monoamine oxidases A and B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt7YjHKC3yrVg90H21EOLACvtfcHk0liOPoWirJfJtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D&md5=2459dcb4cde160186fc17c09258c5906</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGooden%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DPollock%26aufirst%3DJ.%2BA.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DFacile%2520synthesis%2520of%2520substituted%2520trans-2-arylcyclopropylamine%2520inhibitors%2520of%2520the%2520human%2520histone%2520demethylase%2520LSD1%2520and%2520monoamine%2520oxidases%2520A%2520and%2520B%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3047%26epage%3D3051%26doi%3D10.1016%2Fj.bmcl.2008.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Hojfeldt, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Histone lysine demethylases as targets for anticancer therapy</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">930</span><span class="refDoi"> DOI: 10.1038/nrd4154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnrd4154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=24232376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=917-930&author=J.+W.+Hojfeldtauthor=K.+Aggerauthor=K.+Helin&title=Histone+lysine+demethylases+as+targets+for+anticancer+therapy&doi=10.1038%2Fnrd4154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases as targets for anticancer therapy</span></div><div class="casAuthors">Hojfeldt, Jonas W.; Agger, Karl; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-930</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It has recently been demonstrated that the genes controlling the epigenetic programs that are required for maintaining chromatin structure and cell identity include genes that drive human cancer.  This observation has led to an increased awareness of chromatin-assocd. proteins as potentially interesting drug targets.  The successful introduction of DNA methylation and histone deacetylase (HDAC) inhibitors for the treatment of specific subtypes of cancer has paved the way for the use of epigenetic therapy.  Here, we highlight key biol. findings demonstrating the roles of members of the histone lysine demethylase class of enzymes in the development of cancers, discuss the potential and challenges of therapeutically targeting them, and highlight emerging small-mol. inhibitors of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThyggRtM7qrVg90H21EOLACvtfcHk0liOPoWirJfJtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP&md5=b120d43a0353078b016c8dc8f7cac644</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrd4154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4154%26sid%3Dliteratum%253Aachs%26aulast%3DHojfeldt%26aufirst%3DJ.%2BW.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DHistone%2520lysine%2520demethylases%2520as%2520targets%2520for%2520anticancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D917%26epage%3D930%26doi%3D10.1038%2Fnrd4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">An overview of phenylcyclopropylamine derivatives: biochemical and biological significance and recent developments</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">910</span><span class="refDoi"> DOI: 10.1002/med.21269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fmed.21269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=22893613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=873-910&author=M.+N.+Khanauthor=T.+Suzukiauthor=N.+Miyata&title=An+overview+of+phenylcyclopropylamine+derivatives%3A+biochemical+and+biological+significance+and+recent+developments&doi=10.1002%2Fmed.21269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span></div><div class="casAuthors">Khan, Mohammed Naseer Ahmed; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-910</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Trans-2-Phencylcyclopropylamine (2-PCPA), a potent, clin. used antidepressant, affects monoamine neurotransmitter levels by inhibiting the main metabolizing enzymes, monoamine oxidases (MAOs).  However, the antidepressant action of this compd. was not fully explained by its effects on MAOs due to its wide variety of biol. effects. 2-PCPA also affects depression-assocd. pathophysiol. pathways, and linked with increased levels of trace amines in brain, upregulation of GABAB receptors (where GABA is gamma amino butyric acid), modulation of phospholipid metab., and interference with various cytochrome P 450 (CYP) enzymes.  Consequently, despite its adverse effects and limited clin. applicability, 2-PCPA has attracted interest as a structural scaffold for the development of mechanism-based inhibitors of various enzymes, including lysine-specific demethylase 1 (LSD1), which is a possible target for cancer chemotherapy.  In the recent years, many reports have appeared in the literature based on 2-PCPA scaffold and their potential medicinal implications.  This review mainly focuses on the medicinal chem. aspects including drug design, structure-activity relationships (SAR), biol. and biochem. properties, and mechanism of actions of 2-PCPA and its derivs.  Furthermore, we also highlight recent advance in this area and discuss their future applications for beneficial therapeutic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZJFRem7GlrVg90H21EOLACvtfcHk0lh5--pNQephcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D&md5=d257f3b954efb33ae6f824c254ae1c30</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fmed.21269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21269%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DAn%2520overview%2520of%2520phenylcyclopropylamine%2520derivatives%253A%2520biochemical%2520and%2520biological%2520significance%2520and%2520recent%2520developments%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D873%26epage%3D910%26doi%3D10.1002%2Fmed.21269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally designed trans-2-phenylcyclopropylamine derivatives potently inhibit histone demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span><span class="refDoi"> DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+designed+trans-2-phenylcyclopropylamine+derivatives+potently+inhibit+histone+demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lh5--pNQephcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520designed%2520trans-2-phenylcyclopropylamine%2520derivatives%2520potently%2520inhibit%2520histone%2520demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+Inhibitors+of+LSD1+as+Therapeutic+Agents+in+Acute+Myeloid+Leukemia%3A+Clinical+Significance+and+Progress+to+Date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lhdMY8TNSoPIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520Inhibitors%2520of%2520LSD1%2520as%2520Therapeutic%2520Agents%2520in%2520Acute%2520Myeloid%2520Leukemia%253A%2520Clinical%2520Significance%2520and%2520Progress%2520to%2520Date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuhara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1-selective inactivators: protein-targeted drug delivery mechanism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">8620</span><span class="NLM_x">–</span> <span class="NLM_lpage">8624</span><span class="refDoi"> DOI: 10.1002/anie.201303999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fanie.201303999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=8620-8624&author=D.+Ogasawaraauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Kakizawaauthor=K.+Minoauthor=K.+Fukuharaauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Lysine-specific+demethylase+1-selective+inactivators%3A+protein-targeted+drug+delivery+mechanism&doi=10.1002%2Fanie.201303999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1-selective inactivators: Protein-targeted drug delivery mechanism</span></div><div class="casAuthors">Ogasawara, Daisuke; Itoh, Yukihiro; Tsumoto, Hiroki; Kakizawa, Taeko; Mino, Koshiki; Fukuhara, Kiyoshi; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">8620-8624</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of novel LSD1 (LSD1 = lysine-specific demethylase) inactivators based on new concept of LSD1-targeted PCPA (PCPA = trans-2-phenylcyclopropylamine) delivery have been designed.  This concept was tested with peptide 1a [1a = (ART-X-QTARKSTGGKAPRKQLA)] X = (I) and confirmed that 1a does indeed inhibit LSD1 selectively and efficiently by delivering PCPA directly to the LSD1 active site.  The enzyme is then inactivated by FAD-PCPA adduct formation, as in the case of PCPA itself.  This strategy could be applied to the design of nonpeptide, small-mol. LSD1 inactivators.  Mechanistic studies and biol. evaluations revealed that these small-mol. compds. are potent LSD1-selective inactivators (comparable to peptide 1a) in enzyme assays, and they also exhibit potent cell growth inhibitory activities against cancer cell lines.  This is the first time that a drug-carrier-type system has been employed to target a specific mol.  These novel LSD1-selective inactivators are considered to be candidates for anticancer agents, as well as bioprobes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQOLhLwmv9rVg90H21EOLACvtfcHk0lhdMY8TNSoPIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF&md5=71bd05a50aeb73431444f2588fba2dac</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303999%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DLysine-specific%2520demethylase%25201-selective%2520inactivators%253A%2520protein-targeted%2520drug%2520delivery%2520mechanism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D8620%26epage%3D8624%26doi%3D10.1002%2Fanie.201303999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Pieroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annunziato, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzali, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beato, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, G.</span><span> </span><span class="NLM_article-title">Further insights into the SAR of alpha-substituted cyclopropylamine derivatives as inhibitors of histone demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.12.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ejmech.2014.12.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25585008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2015&pages=377-386&author=M.+Pieroniauthor=G.+Annunziatoauthor=E.+Azzaliauthor=P.+Dessantiauthor=C.+Mercurioauthor=G.+Meroniauthor=P.+Trifiroauthor=P.+Vianelloauthor=M.+Villaauthor=C.+Beatoauthor=M.+Varasiauthor=G.+Costantino&title=Further+insights+into+the+SAR+of+alpha-substituted+cyclopropylamine+derivatives+as+inhibitors+of+histone+demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.12.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Further insights into the SAR of α-substituted cyclopropylamine derivatives as inhibitors of histone demethylase KDM1A</span></div><div class="casAuthors">Pieroni, Marco; Annunziato, Giannamaria; Azzali, Elisa; Dessanti, Paola; Mercurio, Ciro; Meroni, Giuseppe; Trifiro, Paolo; Vianello, Paola; Villa, Manuela; Beato, Claudia; Varasi, Mario; Costantino, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">377-386</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Epigenetics alterations including histone methylation and acetylation, and DNA methylation, are thought to play important roles in the onset and progression of cancer in numerous tumor cell lines.  Lysine-specific demethylase 1 (LSD1 or KDM1A) is highly expressed in different cancer types and inhibiting KDM1A activity seems to have high therapeutic potential in cancer treatment.  In the recent years, several inhibitors of KDM1A have been prepd. and disclosed.  The majority of these derivs. were designed based on the structure of tranylcypromine, as the cyclopropane core is responsible for the covalent interaction between the inhibitor and the catalytic domain of KDM proteins.  In this study, we have further extended the SAR regarding compds. 1a-e, which were recently found to inhibit KDM1A with good activity.  The decoration of the Ph ring at the β-position of the cyclopropane ring with small functional groups, mostly halogenated, and in particular at the meta position, led to a significant improvement of the inhibitory activity against KDM1A, as exemplified by compd. 44a, which has a potency in the low nanomolar range (31 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSQyHWyX2UZLVg90H21EOLACvtfcHk0lh-16Ko79oUzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGnuw%253D%253D&md5=76f37a0eeb69d3962a82fd4aab95cc8f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.12.032%26sid%3Dliteratum%253Aachs%26aulast%3DPieroni%26aufirst%3DM.%26aulast%3DAnnunziato%26aufirst%3DG.%26aulast%3DAzzali%26aufirst%3DE.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DBeato%26aufirst%3DC.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DCostantino%26aufirst%3DG.%26atitle%3DFurther%2520insights%2520into%2520the%2520SAR%2520of%2520alpha-substituted%2520cyclopropylamine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D92%26spage%3D377%26epage%3D386%26doi%3D10.1016%2Fj.ejmech.2014.12.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Johnson, N. W.; Kasparec, J.; Miller, W. H.; Rouse, M.; Suarez, D.; Tian, X.</span><span> </span><span class="NLM_article-title">Cyclopropylamines as LSD1 Inhibitors</span>. WO2012/135113, Oct 4,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=N.+W.+Johnson&author=J.+Kasparec&author=W.+H.+Miller&author=M.+Rouse&author=D.+Suarez&author=X.+Tian&title=Cyclopropylamines+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26atitle%3DCyclopropylamines%2520as%2520LSD1%2520Inhibitors%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Maes, T.; Martinell Pedemonte, M.</span><span> </span><span class="NLM_article-title">Lysine Demethylase Inhibitors for Thrombosis and Cardiovascular Diseases</span>. WO2012/156537, Nov 22,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=T.+Maes&author=M.+Martinell+Pedemonte&title=Lysine+Demethylase+Inhibitors+for+Thrombosis+and+Cardiovascular+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26atitle%3DLysine%2520Demethylase%2520Inhibitors%2520for%2520Thrombosis%2520and%2520Cardiovascular%2520Diseases%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Maes, T.; Martinell Pedemonte, M.</span><span> </span><span class="NLM_article-title">Lysine Demethylase Inhibitors for Inflammatory Diseases or Conditions</span>. WO2012/156531, Nov 22,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=T.+Maes&author=M.+Martinell+Pedemonte&title=Lysine+Demethylase+Inhibitors+for+Inflammatory+Diseases+or+Conditions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26atitle%3DLysine%2520Demethylase%2520Inhibitors%2520for%2520Inflammatory%2520Diseases%2520or%2520Conditions%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Ortega Munoz, A.; Fyfe, M. C. T.; Estiarte Martinez, M. d. L. A.; Valls Vidal, N.; Kurz, G.; Castro Palomino, L. J. C.</span><span> </span><span class="NLM_article-title">(Hetero)Aryl Cyclopropylamine Compounds as LSD1 Inhibitors</span>. WO2013/057320, Apr 25,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Ortega+Munoz&author=M.+C.+T.+Fyfe&author=M.+d.+L.+A.+Estiarte+Martinez&author=N.+Valls+Vidal&author=G.+Kurz&author=L.+J.+C.+Castro+Palomino&title=%28Hetero%29Aryl+Cyclopropylamine+Compounds+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%2BMunoz%26aufirst%3DA.%26atitle%3D%2528Hetero%2529Aryl%2520Cyclopropylamine%2520Compounds%2520as%2520LSD1%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Ortega Munoz, A.; Fyfe, M. C. T.; Martinell Pedemonte, M.; Estiarte Martinez, M. d. L. A.; Valls Vidal, N.; Kurz, G.; Castro Palomino, L. J. C.</span><span> </span><span class="NLM_article-title">(Hetero)Aryl Cyclopropylamine Compounds as LSD1 Inhibitors</span>. WO2013/057322, Apr 25,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Ortega+Munoz&author=M.+C.+T.+Fyfe&author=M.+Martinell+Pedemonte&author=M.+d.+L.+A.+Estiarte+Martinez&author=N.+Valls+Vidal&author=G.+Kurz&author=L.+J.+C.+Castro+Palomino&title=%28Hetero%29Aryl+Cyclopropylamine+Compounds+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%2BMunoz%26aufirst%3DA.%26atitle%3D%2528Hetero%2529Aryl%2520Cyclopropylamine%2520Compounds%2520as%2520LSD1%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Tomita, N.; Shigeo, K.</span><span> </span><span class="NLM_article-title">Cyclopropaneamine Compound</span>. WO2013/022047, Feb 14,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=N.+Tomita&author=K.+Shigeo&title=Cyclopropaneamine+Compound"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTomita%26aufirst%3DN.%26atitle%3DCyclopropaneamine%2520Compound%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span>A phase I study of human pharmacokinetics and safety of ORY-1001, an LSD1 inhibitor, in relapsed or refractory acute leukaemia (AL). EudraCT number 2013-002447-29. <a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES" class="extLink">https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=A+phase+I+study+of+human+pharmacokinetics+and+safety+of+ORY-1001%2C+an+LSD1+inhibitor%2C+in+relapsed+or+refractory+acute+leukaemia+%28AL%29.+EudraCT+number+2013-002447-29.+https%3A%2F%2Fwww.clinicaltrialsregister.eu%2Fctr-search%2Ftrial%2F2013-002447-29%2FES."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Maes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carceller, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buesa, C.</span><span> </span><span class="NLM_article-title">Advances in the development of histone lysine demethylase inhibitors</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.coph.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.coph.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=26057211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=52-60&author=T.+Maesauthor=E.+Carcellerauthor=J.+Salasauthor=A.+Ortegaauthor=C.+Buesa&title=Advances+in+the+development+of+histone+lysine+demethylase+inhibitors&doi=10.1016%2Fj.coph.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of histone lysine demethylase inhibitors</span></div><div class="casAuthors">Maes, Tamara; Carceller, Elena; Salas, Jordi; Ortega, Alberto; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The covalent modification of histones is closely assocd. with regulation of gene transcription.  Chromatin modifications have been suggested to represent an epigenetic code that is dynamically 'written' and 'erased' by specialized proteins, and 'read', or interpreted, by proteins that translate the code into gene expression changes.  Initially thought to be an irreversible process, histone methylation is now known to be reversed by demethylases, FAD dependent amineoxidases and by iron(II)-alpha-ketoglutarate dependent deoxygenases of the Jumonji family.  Altered histone demethylase activities have been assocd. with human disease, including cancer.  The first wave of novel investigational drugs directed against KDM1A has recently entered the clinic, and the first specific inhibitor targeting a Jumonji KDM is advancing in preclin. regulatory studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCr97IlCYFrVg90H21EOLACvtfcHk0lhG1c7iaxsdJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D&md5=02e8a5f8d426743be1eb209c329bd3ed</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DSalas%26aufirst%3DJ.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.coph.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">McGrath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span> </span><span class="NLM_article-title">Targeting histone lysine methylation in cancer</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.pharmthera.2015.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25578037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=1-22&author=J.+McGrathauthor=P.+Trojer&title=Targeting+histone+lysine+methylation+in+cancer&doi=10.1016%2Fj.pharmthera.2015.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine methylation in cancer</span></div><div class="casAuthors">McGrath, John; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-22</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Within the vast landscape of histone modifications lysine methylation has gained increasing attention because of its profound regulatory potential.  The methylation of lysine residues on histone proteins modulates chromatin structure and thereby contributes to the regulation of DNA-based nuclear processes such as transcription, replication and repair.  Protein families with opposing catalytic activities, lysine methyltransferases (KMTs) and demethylases (KDMs), dynamically control levels of histone lysine methylation and individual enzymes within these families have become candidate oncol. targets in recent years.  A no. of high quality small mol. inhibitors of these enzymes have been identified.  Several of these compds. elicit selective cancer cell killing in vitro and robust efficacy in vivo, suggesting that targeting 'histone lysine methylation pathways' may be a relevant, emerging cancer therapeutic strategy.  Here, we discuss individual histone lysine methylation pathway targets, the properties of currently available small mol. inhibitors and their application in the context of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogqKUpTpHisrVg90H21EOLACvtfcHk0lhro_EZI1yRrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D&md5=225ece90384f762b248d2d7981fa120e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcGrath%26aufirst%3DJ.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DTargeting%2520histone%2520lysine%2520methylation%2520in%2520cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22%26doi%3D10.1016%2Fj.pharmthera.2015.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span>Investigation of GSK2879552 in Subjects with Relapsed/Refractory Small Cell Lung Carcinoma. <a href="https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&rank=1</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Investigation+of+GSK2879552+in+Subjects+with+Relapsed%2FRefractory+Small+Cell+Lung+Carcinoma.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02034123%3Fterm%3DGSK2879552%26rank%3D1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span>Phase I Dose Escalation Study for GSK2879552 in Subjects with Acute Myeloid Leukemia (AML). ClinicalTrials.gov Identifier: NCT02177812. <a href="https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&amp;rank=2" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&rank=2</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Phase+I+Dose+Escalation+Study+for+GSK2879552+in+Subjects+with+Acute+Myeloid+Leukemia+%28AML%29.+ClinicalTrials.gov+Identifier%3A+NCT02177812.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02177812%3Fterm%3DGSK2879552%26rank%3D2."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Mohammad, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smitheman, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamat, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federowicz, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneck, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butticello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnette, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degenhardt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasparec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNulty, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouthamel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span> </span><span class="NLM_article-title">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+Hypomethylation+Signature+Predicts+Antitumor+Activity+of+LSD1+Inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lgQXFIfoNSz0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520Hypomethylation%2520Signature%2520Predicts%2520Antitumor%2520Activity%2520of%2520LSD1%2520Inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible inhibitors of LSD1 as therapeutic agents in acute myeloid leukemia: clinical significance and progress to date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+inhibitors+of+LSD1+as+therapeutic+agents+in+acute+myeloid+leukemia%3A+clinical+significance+and+progress+to+date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lgQXFIfoNSz0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520inhibitors%2520of%2520LSD1%2520as%2520therapeutic%2520agents%2520in%2520acute%2520myeloid%2520leukemia%253A%2520clinical%2520significance%2520and%2520progress%2520to%2520date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0ljM_qUkc5WmuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saponara, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrocco, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monaldi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruoppolo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilset, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Pure enantiomers of benzoylamino-tranylcypromine: LSD1 inhibition, gene modulation in human leukemia cells and effects on clonogenic potential of murine promyelocytic blasts</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">174</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2015.02.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ejmech.2015.02.060" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=163-174&author=S.+Valenteauthor=V.+Rodriguezauthor=C.+Mercurioauthor=P.+Vianelloauthor=B.+Saponaraauthor=R.+Cirilliauthor=G.+Ciossaniauthor=D.+Labellaauthor=B.+Marroccoauthor=D.+Monaldiauthor=G.+Ruoppoloauthor=M.+Tilsetauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=A.+Matteviauthor=M.+Varasiauthor=A.+Mai&title=Pure+enantiomers+of+benzoylamino-tranylcypromine%3A+LSD1+inhibition%2C+gene+modulation+in+human+leukemia+cells+and+effects+on+clonogenic+potential+of+murine+promyelocytic+blasts&doi=10.1016%2Fj.ejmech.2015.02.060"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.02.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.02.060%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRodriguez%26aufirst%3DV.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DSaponara%26aufirst%3DB.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DMonaldi%26aufirst%3DD.%26aulast%3DRuoppolo%26aufirst%3DG.%26aulast%3DTilset%26aufirst%3DM.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DPure%2520enantiomers%2520of%2520benzoylamino-tranylcypromine%253A%2520LSD1%2520inhibition%252C%2520gene%2520modulation%2520in%2520human%2520leukemia%2520cells%2520and%2520effects%2520on%2520clonogenic%2520potential%2520of%2520murine%2520promyelocytic%2520blasts%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D163%26epage%3D174%26doi%3D10.1016%2Fj.ejmech.2015.02.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Bujok, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwast, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cmoch, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wróbel, Z.</span><span> </span><span class="NLM_article-title">Novel approach to synthesis of substituted 3-aminoquinolines from nitroarenes and protected ethyl aminocrotonate</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">708</span><span class="refDoi"> DOI: 10.1016/j.tet.2009.11.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.tet.2009.11.060" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=698-708&author=R.+Bujokauthor=A.+Kwastauthor=P.+Cmochauthor=Z.+Wr%C3%B3bel&title=Novel+approach+to+synthesis+of+substituted+3-aminoquinolines+from+nitroarenes+and+protected+ethyl+aminocrotonate&doi=10.1016%2Fj.tet.2009.11.060"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2009.11.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2009.11.060%26sid%3Dliteratum%253Aachs%26aulast%3DBujok%26aufirst%3DR.%26aulast%3DKwast%26aufirst%3DA.%26aulast%3DCmoch%26aufirst%3DP.%26aulast%3DWr%25C3%25B3bel%26aufirst%3DZ.%26atitle%3DNovel%2520approach%2520to%2520synthesis%2520of%2520substituted%25203-aminoquinolines%2520from%2520nitroarenes%2520and%2520protected%2520ethyl%2520aminocrotonate%26jtitle%3DTetrahedron%26date%3D2010%26volume%3D66%26spage%3D698%26epage%3D708%26doi%3D10.1016%2Fj.tet.2009.11.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacCoss, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawson, A. D.</span><span> </span><span class="NLM_article-title">Rings in drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">5845</span><span class="NLM_x">–</span> <span class="NLM_lpage">5859</span><span class="refDoi"> DOI: 10.1021/jm4017625</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4017625" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5845-5859&author=R.+D.+Taylorauthor=M.+MacCossauthor=A.+D.+Lawson&title=Rings+in+drugs&doi=10.1021%2Fjm4017625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Rings in Drugs</span></div><div class="casAuthors">Taylor, Richard D.; MacCoss, Malcolm; Lawson, Alastair D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5845-5859</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  We have analyzed the rings, ring systems, and frameworks in drugs listed in the FDA Orange Book to understand the frequency, time-lines, mol. property space, and the application of these rings in different therapeutic areas and target classes.  This anal. shows that there are only 351 ring systems and 1197 frameworks in drugs that came onto the market before 2013.  Furthermore, on av. six new ring systems enter drug space each year and approx. 28% of new drugs contain a new ring system.  Moreover, it is very unusual for a drug to contain more than one new ring system and the majority of the most frequently used ring systems (83%) were first used in drugs developed prior to 1983.  These observations give insight into the chem. novelty of drugs and potentially efficient ways to assess compd. libraries and develop compds. from hit identification to lead optimization and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC_e_L4z4XCrVg90H21EOLACvtfcHk0liPLfCnTLZWVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D&md5=c84ce84b8698eb6fad38fa82b0ab4912</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm4017625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4017625%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DA.%2BD.%26atitle%3DRings%2520in%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5845%26epage%3D5859%26doi%3D10.1021%2Fjm4017625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newton-Vinson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubalek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structure of human monoamine oxidase B, a drug target for the treatment of neurological disorders</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1038/nsb732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fnsb732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=11753429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD38XitV2rsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=22-26&author=C.+Bindaauthor=P.+Newton-Vinsonauthor=F.+Hubalekauthor=D.+E.+Edmondsonauthor=A.+Mattevi&title=Structure+of+human+monoamine+oxidase+B%2C+a+drug+target+for+the+treatment+of+neurological+disorders&doi=10.1038%2Fnsb732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human monoamine oxidase B, a drug target for the treatment of neurological disorders</span></div><div class="casAuthors">Binda, Claudia; Newton-Vinson, Paige; Hubalek, Frantisek; Edmondson, Dale E.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-26</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Monoamine oxidase B (MAO B) is a mitochondrial outer membrane flavoenzyme that is a well-known target for antidepressant and neuroprotective drugs.  Here, the authors detd. the structure of the human liver enzyme to 3 Å resoln.  Human liver MAO B was found to bind to the membrane through a C-terminal transmembrane helix and apolar loops located at various positions in the sequence.  The electron d. showed that pargyline, an analog of the clin. used MAO B inhibitor, deprenyl, binds covalently to the flavin N5 atom.  The active site of MAO B consisted of a 420-Å3 hydrophobic substrate cavity interconnected to an entrance cavity of 290 Å3.  The recognition site for the substrate amino group was an arom. cage formed by Tyr-398 and Tyr-435.  The structure provides a framework for probing the catalytic mechanism, understanding the differences between the MAO A and B isoforms and designing specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8p43h-vsPi7Vg90H21EOLACvtfcHk0lgswle7vZoUbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitV2rsg%253D%253D&md5=261d3b1375d833bbfd0a23bdade49be6</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnsb732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb732%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DNewton-Vinson%26aufirst%3DP.%26aulast%3DHubalek%26aufirst%3DF.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructure%2520of%2520human%2520monoamine%2520oxidase%2520B%252C%2520a%2520drug%2520target%2520for%2520the%2520treatment%2520of%2520neurological%2520disorders%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D22%26epage%3D26%26doi%3D10.1038%2Fnsb732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span> </span><span class="NLM_article-title">Synthesis, biological activity and mechanistic insights of 1-substituted cyclopropylamine derivatives: a novel class of irreversible inhibitors of histone demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.08.068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.ejmech.2014.08.068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=25173853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=352-363&author=P.+Vianelloauthor=O.+A.+Botrugnoauthor=A.+Cappaauthor=G.+Ciossaniauthor=P.+Dessantiauthor=A.+Maiauthor=A.+Matteviauthor=G.+Meroniauthor=S.+Minucciauthor=F.+Thalerauthor=M.+Tortoriciauthor=P.+Trifiroauthor=S.+Valenteauthor=M.+Villaauthor=M.+Varasiauthor=C.+Mercurio&title=Synthesis%2C+biological+activity+and+mechanistic+insights+of+1-substituted+cyclopropylamine+derivatives%3A+a+novel+class+of+irreversible+inhibitors+of+histone+demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.08.068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological activity and mechanistic insights of 1-substituted cyclopropylamine derivatives: A novel class of irreversible inhibitors of histone demethylase KDM1A</span></div><div class="casAuthors">Vianello, Paola; Botrugno, Oronza A.; Cappa, Anna; Ciossani, Giuseppe; Dessanti, Paola; Mai, Antonello; Mattevi, Andrea; Meroni, Giuseppe; Minucci, Saverio; Thaler, Florian; Tortorici, Marcello; Trifiro, Paolo; Valente, Sergio; Villa, Manuela; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">352-363</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Histone demethylase KDM1A [i.e., gene LSD1, histone H3 lysine-4 demethylase LSD1] has become an attractive therapeutic target for the treatment of cancer as well as other disorders such as viral infections.  The authors report on the synthesis of compds. derived from the expansion of tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  These compds., which are substituted on the cyclopropyl core moiety, were evaluated for their ability to inhibit KDM1A in vitro as well as to function in cells by modulating the expression of Gfi-1b, a well recognized KDM1A target gene.  The mols. were all found to covalently inhibit KDM1A and to become increasingly selective against human monoamine oxidase MAO A and MAO B through the introduction of bulkier substituents on the cyclopropylamine ring.  Structural and biochem. anal. of selected trans-isomers showed that the two stereoisomers are endowed with similar inhibitory activities against KDM1A, but form different covalent adducts with FAD co-enzyme [i.e., riboflavin 5'-(trihydrogen diphosphate) P'→5'-ester with adenosine, FAD].  The synthesis of the target compds. was achieved using (2S)-2-(phenyl)oxirane, (2R)-2-(phenyl)oxirane, racemic (phenyl)oxirane as starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEJ-thHP_b6rVg90H21EOLACvtfcHk0lgswle7vZoUbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP&md5=6285108e113c25939e3aa0b1ee5390c6</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.08.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.08.068%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DSynthesis%252C%2520biological%2520activity%2520and%2520mechanistic%2520insights%2520of%25201-substituted%2520cyclopropylamine%2520derivatives%253A%2520a%2520novel%2520class%2520of%2520irreversible%2520inhibitors%2520of%2520histone%2520demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D352%26epage%3D363%26doi%3D10.1016%2Fj.ejmech.2014.08.068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Saleque, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooke, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Epigenetic regulation of hematopoietic differentiation by Gfi-1 and Gfi-1b is mediated by the cofactors CoREST and LSD1</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.molcel.2007.06.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=562-572&author=S.+Salequeauthor=J.+Kimauthor=H.+M.+Rookeauthor=S.+H.+Orkin&title=Epigenetic+regulation+of+hematopoietic+differentiation+by+Gfi-1+and+Gfi-1b+is+mediated+by+the+cofactors+CoREST+and+LSD1&doi=10.1016%2Fj.molcel.2007.06.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.039%26sid%3Dliteratum%253Aachs%26aulast%3DSaleque%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DRooke%26aufirst%3DH.%2BM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DEpigenetic%2520regulation%2520of%2520hematopoietic%2520differentiation%2520by%2520Gfi-1%2520and%2520Gfi-1b%2520is%2520mediated%2520by%2520the%2520cofactors%2520CoREST%2520and%2520LSD1%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D562%26epage%3D572%26doi%3D10.1016%2Fj.molcel.2007.06.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monestiroli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giavara, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ronzoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchesi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insinga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diverio, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasparini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capillo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matteucci, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contegno, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo-Coco, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scanziani, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelicci, P. G.</span><span> </span><span class="NLM_article-title">PML-RAR induces promyelocytic leukemias with high efficiency following retroviral gene transfer into purified murine hematopoietic progenitors</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">2989</span><span class="NLM_x">–</span> <span class="NLM_lpage">2995</span><span class="refDoi"> DOI: 10.1182/blood-2001-11-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1182%2Fblood-2001-11-0089" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2002&pages=2989-2995&author=S.+Minucciauthor=S.+Monestiroliauthor=S.+Giavaraauthor=S.+Ronzoniauthor=F.+Marchesiauthor=A.+Insingaauthor=D.+Diverioauthor=P.+Gaspariniauthor=M.+Capilloauthor=E.+Colomboauthor=C.+Matteucciauthor=F.+Contegnoauthor=F.+Lo-Cocoauthor=E.+Scanzianiauthor=A.+Gobbiauthor=P.+G.+Pelicci&title=PML-RAR+induces+promyelocytic+leukemias+with+high+efficiency+following+retroviral+gene+transfer+into+purified+murine+hematopoietic+progenitors&doi=10.1182%2Fblood-2001-11-0089"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1182%2Fblood-2001-11-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2001-11-0089%26sid%3Dliteratum%253Aachs%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMonestiroli%26aufirst%3DS.%26aulast%3DGiavara%26aufirst%3DS.%26aulast%3DRonzoni%26aufirst%3DS.%26aulast%3DMarchesi%26aufirst%3DF.%26aulast%3DInsinga%26aufirst%3DA.%26aulast%3DDiverio%26aufirst%3DD.%26aulast%3DGasparini%26aufirst%3DP.%26aulast%3DCapillo%26aufirst%3DM.%26aulast%3DColombo%26aufirst%3DE.%26aulast%3DMatteucci%26aufirst%3DC.%26aulast%3DContegno%26aufirst%3DF.%26aulast%3DLo-Coco%26aufirst%3DF.%26aulast%3DScanziani%26aufirst%3DE.%26aulast%3DGobbi%26aufirst%3DA.%26aulast%3DPelicci%26aufirst%3DP.%2BG.%26atitle%3DPML-RAR%2520induces%2520promyelocytic%2520leukemias%2520with%2520high%2520efficiency%2520following%2520retroviral%2520gene%2520transfer%2520into%2520purified%2520murine%2520hematopoietic%2520progenitors%26jtitle%3DBlood%26date%3D2002%26volume%3D100%26spage%3D2989%26epage%3D2995%26doi%3D10.1182%2Fblood-2001-11-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Leiva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moretti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soilihi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallavicini, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peres, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Zuffo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de The, H.</span><span> </span><span class="NLM_article-title">Valproic acid induces differentiation and transient tumor regression, but spares leukemia-initiating activity in mouse models of APL</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1630</span><span class="NLM_x">–</span> <span class="NLM_lpage">1637</span><span class="refDoi"> DOI: 10.1038/leu.2012.39</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1038%2Fleu.2012.39" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1630-1637&author=M.+Leivaauthor=S.+Morettiauthor=H.+Soilihiauthor=I.+Pallaviciniauthor=L.+Peresauthor=C.+Mercurioauthor=R.+Dal+Zuffoauthor=S.+Minucciauthor=H.+de+The&title=Valproic+acid+induces+differentiation+and+transient+tumor+regression%2C+but+spares+leukemia-initiating+activity+in+mouse+models+of+APL&doi=10.1038%2Fleu.2012.39"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fleu.2012.39&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2012.39%26sid%3Dliteratum%253Aachs%26aulast%3DLeiva%26aufirst%3DM.%26aulast%3DMoretti%26aufirst%3DS.%26aulast%3DSoilihi%26aufirst%3DH.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DPeres%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DDal%2BZuffo%26aufirst%3DR.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3Dde%2BThe%26aufirst%3DH.%26atitle%3DValproic%2520acid%2520induces%2520differentiation%2520and%2520transient%2520tumor%2520regression%252C%2520but%2520spares%2520leukemia-initiating%2520activity%2520in%2520mouse%2520models%2520of%2520APL%26jtitle%3DLeukemia%26date%3D2012%26volume%3D26%26spage%3D1630%26epage%3D1637%26doi%3D10.1038%2Fleu.2012.39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Maes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirapu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascarò, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiarte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valls, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro Palomino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arjol, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurz, G.</span><span> </span><span class="NLM_article-title">Preclinical characterization of a potent and selective inhibitor of the histone demethylase KDM1A for MLL leukemia</span> <span class="citation_source-journal">J. Clin Oncol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl.</span><span class="NLM_x">) </span> <span class="NLM_fpage">e13543</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=e13543&issue=Suppl.&author=T.+Maesauthor=I.+Tirapuauthor=C.+Mascar%C3%B2author=A.+Ortegaauthor=A.+Estiarteauthor=N.+Vallsauthor=J.+Castro+Palominoauthor=C.+B.+Arjolauthor=G.+Kurz&title=Preclinical+characterization+of+a+potent+and+selective+inhibitor+of+the+histone+demethylase+KDM1A+for+MLL+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DMascar%25C3%25B2%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DCastro%2BPalomino%26aufirst%3DJ.%26aulast%3DArjol%26aufirst%3DC.%2BB.%26aulast%3DKurz%26aufirst%3DG.%26atitle%3DPreclinical%2520characterization%2520of%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520histone%2520demethylase%2520KDM1A%2520for%2520MLL%2520leukemia%26jtitle%3DJ.%2520Clin%2520Oncol%26date%3D2013%26volume%3D31%26issue%3DSuppl%26spage%3De13543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Brucoli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thurston, D. E.</span><span> </span><span class="NLM_article-title">Efficient solid-phase synthesis of a library of distamycin analogs containing novel biaryl motifs on SynPhase Lanterns</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span><span class="refDoi"> DOI: 10.1021/cc900009r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc900009r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Ontrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=576-586&author=F.+Brucoliauthor=P.+W.+Howardauthor=D.+E.+Thurston&title=Efficient+solid-phase+synthesis+of+a+library+of+distamycin+analogs+containing+novel+biaryl+motifs+on+SynPhase+Lanterns&doi=10.1021%2Fcc900009r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Solid-Phase Synthesis of a Library of Distamycin Analogs Containing Novel Biaryl Motifs on SynPhase Lanterns</span></div><div class="casAuthors">Brucoli, Federico; Howard, Philip W.; Thurston, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Distamycin is a naturally occurring antibiotic that binds to AT-rich sequences in the minor groove of DNA in a noncovalent manner.  It continues to be of interest as a "building block" for more-complex small-mol. ligands capable of targeting specific DNA sequences for gene regulation purposes (i.e., transcription factor inhibitors).  Here, the authors report a convenient and efficient synthesis of a library of 72 novel analogs (3a-f) of the parent distamycin on SynPhase Lanterns.  To investigate structure-activity relationships including DNA-binding affinity and sequence-selectivity, two previously unexplored points of diversification have been introduced into the distamycin structure by replacing one of its pyrrole rings with novel biaryl motifs.  The key aspects of the synthetic approach include the development of an efficient protocol for prepn. of the heterocyclic polyamide chain, optimization of the Suzuki-Miyaura cross-coupling reaction and application of a split-and-mix technique based on radiofrequency encoding.  In addn., a series of biaryl carboxamide derivs. (4a-f) has been synthesized utilizing the title library diversity reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMv-1jhPjphLVg90H21EOLACvtfcHk0lgy8JgN8Xfd6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Ontrg%253D&md5=a8f16be7e893bc0c1c444fe33d763c02</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fcc900009r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc900009r%26sid%3Dliteratum%253Aachs%26aulast%3DBrucoli%26aufirst%3DF.%26aulast%3DHoward%26aufirst%3DP.%2BW.%26aulast%3DThurston%26aufirst%3DD.%2BE.%26atitle%3DEfficient%2520solid-phase%2520synthesis%2520of%2520a%2520library%2520of%2520distamycin%2520analogs%2520containing%2520novel%2520biaryl%2520motifs%2520on%2520SynPhase%2520Lanterns%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2009%26volume%3D11%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fcc900009r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Shearer, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uehling, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deaton, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milliken, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faison, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, F.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of potent and selective beta3 adrenergic receptor agonists containing heterobiaryl carboxylic acids</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4670</span><span class="NLM_x">–</span> <span class="NLM_lpage">4677</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.05.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;key=10.1016%2Fj.bmcl.2007.05.069" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4670-4677&author=B.+G.+Shearerauthor=E.+Y.+Chaoauthor=D.+E.+Uehlingauthor=D.+N.+Deatonauthor=C.+Cowanauthor=B.+W.+Shermanauthor=T.+Millikenauthor=W.+Faisonauthor=K.+Brownauthor=K.+K.+Adkisonauthor=F.+Lee&title=Synthesis+and+evaluation+of+potent+and+selective+beta3+adrenergic+receptor+agonists+containing+heterobiaryl+carboxylic+acids&doi=10.1016%2Fj.bmcl.2007.05.069"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.05.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.05.069%26sid%3Dliteratum%253Aachs%26aulast%3DShearer%26aufirst%3DB.%2BG.%26aulast%3DChao%26aufirst%3DE.%2BY.%26aulast%3DUehling%26aufirst%3DD.%2BE.%26aulast%3DDeaton%26aufirst%3DD.%2BN.%26aulast%3DCowan%26aufirst%3DC.%26aulast%3DSherman%26aufirst%3DB.%2BW.%26aulast%3DMilliken%26aufirst%3DT.%26aulast%3DFaison%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DK.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DLee%26aufirst%3DF.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520potent%2520and%2520selective%2520beta3%2520adrenergic%2520receptor%2520agonists%2520containing%2520heterobiaryl%2520carboxylic%2520acids%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4670%26epage%3D4677%26doi%3D10.1016%2Fj.bmcl.2007.05.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01209">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28310"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01209">10.1021/acs.jmedchem.5b01209</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures illustrating (1) IC<sub>50</sub> curves of biochemical assay, (2) effect of KDM1A down regulation on mRNA expression of GFI1B and CD11b in NB4 cells, and (3) effect of KDM1A down regulation on clonogenic potential of mouse promyelocytic blasts (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_001.pdf">jm5b01209_si_001.pdf (1.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01209/suppl_file/jm5b01209_si_002.csv">jm5b01209_si_002.csv (3.02 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-4%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01209%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01209" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e105a60bb1d93a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
