
<html lang="en"     class="pb-page"  data-request-id="283ec4a8-78fe-4fc8-9828-82181edcfca8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-7;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b02178;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Thieno[2,3-d]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells" /></meta><meta name="dc.Creator" content="Zhaoping  Pan" /></meta><meta name="dc.Creator" content="Xiang  Li" /></meta><meta name="dc.Creator" content="Yujia  Wang" /></meta><meta name="dc.Creator" content="Qinglin  Jiang" /></meta><meta name="dc.Creator" content="Li  Jiang" /></meta><meta name="dc.Creator" content="Min  Zhang" /></meta><meta name="dc.Creator" content="Nan  Zhang" /></meta><meta name="dc.Creator" content="Fengbo  Wu" /></meta><meta name="dc.Creator" content="Bo  Liu" /></meta><meta name="dc.Creator" content="Gu  He" /></meta><meta name="dc.Description" content="Bromodomain-containing protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases. Based on the integrated fragment-based drug desi..." /></meta><meta name="Description" content="Bromodomain-containing protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases. Based on the integrated fragment-based drug desi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 10, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b02178" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02178" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b02178" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02178" /></link>
        
    
    

<title>Discovery of Thieno[2,3-d]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b02178" /></meta><meta property="og:title" content="Discovery of Thieno[2,3-d]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0016.jpeg" /></meta><meta property="og:description" content="Bromodomain-containing protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases. Based on the integrated fragment-based drug design, synthesis, and in vitro and in vivo evaluations, a series of novel thieno[2,3-d]pyrimidine-based hydroxamic acid derivatives are discovered as selective BRD4–HDAC dual inhibitors. Compound 17c is the most potent inhibitor for BRD4 and HDAC with IC50 values at nanomolar levels, as well as the expression level of c-Myc, and increases the acetylation of histone H3. Moreover, 17c presents inhibitory effects on the proliferation of colorectal carcinoma (CRC) cells via inducing autophagic cell death. It also has a good pharmacokinetic profile in rats and oral bioavailability of 40.5%. In the HCT-116 xenograft in vivo models, 17c displays potent inhibitory efficiency on tumor growth by inducing autophagic cell death and suppressing IL6–JAK–STAT signaling pathways. Our results suggest that the BRD4–HDAC dual inhibition might be an attractive therapeutic strategy for CRC." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b02178"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02178">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b02178&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b02178&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b02178&amp;href=/doi/10.1021/acs.jmedchem.9b02178" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3678-3700</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02163" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00012" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Thieno[2,3-<i>d</i>]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Zhaoping Pan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaoping Pan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaoping++Pan">Zhaoping Pan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiang Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiang Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiang++Li">Xiang Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yujia Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yujia Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yujia++Wang">Yujia Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qinglin Jiang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qinglin Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy and Sichuan Province College Key Laboratory of Structure-Specific Small Molecule Drugs, Chengdu Medical College, Chengdu 610500, PR China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#4e243f22112d232d0e7f787d602d2123"><span class="__cf_email__" data-cfemail="3e544f52615d535d7e0f080d105d5153">[email protected]</span></a>. Phone/Fax: 86-28-85503817.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qinglin++Jiang">Qinglin Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Li Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Li Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy and Sichuan Province College Key Laboratory of Structure-Specific Small Molecule Drugs, Chengdu Medical College, Chengdu 610500, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Li++Jiang">Li Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Min Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Min Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy and Sichuan Province College Key Laboratory of Structure-Specific Small Molecule Drugs, Chengdu Medical College, Chengdu 610500, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Min++Zhang">Min Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nan Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nan Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nan++Zhang">Nan Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengbo Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengbo Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengbo++Wu">Fengbo Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bo Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ed8184988f82dfd9ddddaddcdbdec38e8280"><span class="__cf_email__" data-cfemail="264a4f53444914121616661710150845494b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Liu">Bo Liu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Gu He</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gu He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Department of Urology, West China Hospital, Sichuan University and Collaborative Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#7d15181a083d0e1e0853181908531e13"><span class="__cf_email__" data-cfemail="6109040614211202144f0405144f020f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gu++He">Gu He</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1536-8882" title="Orcid link">http://orcid.org/0000-0002-1536-8882</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02178&amp;href=/doi/10.1021%2Facs.jmedchem.9b02178" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3678–3700</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 10, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 December 2019</li><li><span class="item_label"><b>Published</b> online</span>10 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 April 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b02178" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02178</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3678%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DZhaoping%2BPan%252C%2BXiang%2BLi%252C%2BYujia%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D7%26contentID%3Dacs.jmedchem.9b02178%26title%3DDiscovery%2Bof%2BThieno%255B2%252C3-d%255Dpyrimidine-Based%2BHydroxamic%2BAcid%2BDerivatives%2Bas%2BBromodomain-Containing%2BProtein%2B4%252FHistone%2BDeacetylase%2BDual%2BInhibitors%2BInduce%2BAutophagic%2BCell%2BDeath%2Bin%2BColorectal%2BCarcinoma%2BCells%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3700%26publicationDate%3DApril%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b02178"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1822</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">16</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b02178" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Thieno[2,3-d]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Zhaoping&quot;,&quot;last_name&quot;:&quot;Pan&quot;},{&quot;first_name&quot;:&quot;Xiang&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yujia&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Qinglin&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Min&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Nan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Fengbo&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Gu&quot;,&quot;last_name&quot;:&quot;He&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;3678-3700&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b02178&quot;},&quot;abstract&quot;:&quot;Bromodomain-containing protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases. Based on the integrated fragment-based drug design, synthesis, and in vitro and in vivo evaluations, a series of novel thieno[2,3-d]pyrimidine-based hydroxamic acid derivatives are discovered as selective BRD4–HDAC dual inhibitors. Compound 17c is the most potent inhibitor for BRD4 and HDAC with IC50 values at nanomolar levels, as well as the expression level of c-Myc, and increases the acetylation of histone H3. Moreover, 17c presents inhibitory effects on the proliferation of colorectal carcinoma (CRC) cells via inducing autophagic cell death. It also has a good pharmacokinetic profile in rats and oral bioavailability of 40.5%. In the HCT-116 xenograft in vivo models, 17c displays potent inhibitory efficiency on tumor growth by inducing autophagic cell death and suppressing IL6–JAK–STAT signaling pathways. Our results suggest that the BRD4–HDAC dual inhibition &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02178&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02178" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02178&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02178" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02178&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02178" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02178&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02178&amp;href=/doi/10.1021/acs.jmedchem.9b02178" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b02178" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b02178" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (14 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b02178%26sid%3Dliteratum%253Aachs%26pmid%3D32153186%26genre%3Darticle%26aulast%3DPan%26date%3D2020%26atitle%3DDiscovery%2Bof%2BThieno%255B2%252C3-d%255Dpyrimidine-Based%2BHydroxamic%2BAcid%2BDerivatives%2Bas%2BBromodomain-Containing%2BProtein%2B4%252FHistone%2BDeacetylase%2BDual%2BInhibitors%2BInduce%2BAutophagic%2BCell%2BDeath%2Bin%2BColorectal%2BCarcinoma%2BCells%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D7%26spage%3D3678%26epage%3D3700%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291149" title="Apoptosis">Apoptosis</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/jmcmar.2020.63.issue-7/20200409/jmcmar.2020.63.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Bromodomain-containing protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases. Based on the integrated fragment-based drug design, synthesis, and in vitro and in vivo evaluations, a series of novel thieno[2,3-<i>d</i>]pyrimidine-based hydroxamic acid derivatives are discovered as selective BRD4–HDAC dual inhibitors. Compound <b>17c</b> is the most potent inhibitor for BRD4 and HDAC with IC<sub>50</sub> values at nanomolar levels, as well as the expression level of c-Myc, and increases the acetylation of histone H3. Moreover, <b>17c</b> presents inhibitory effects on the proliferation of colorectal carcinoma (CRC) cells via inducing autophagic cell death. It also has a good pharmacokinetic profile in rats and oral bioavailability of 40.5%. In the HCT-116 xenograft in vivo models, <b>17c</b> displays potent inhibitory efficiency on tumor growth by inducing autophagic cell death and suppressing IL6–JAK–STAT signaling pathways. Our results suggest that the BRD4–HDAC dual inhibition might be an attractive therapeutic strategy for CRC.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18551" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18551" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Colorectal carcinoma (CRC) is one of the most common malignances worldwide and remains the second and third leading causes of cancer-related deaths in male and female adults, respectively.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Although improvements in diagnostic and effective therapeutic methods have been developed for CRC, novel therapeutics for the treatment of advanced CRC are still urgently needed.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> In addition, the clinically used molecular targeted agents of CRC, such as epidermal growth factor receptor (EGFR) inhibitors, cetuximab and bevacizumab, only appear to have limited activity in metastatic CRC (mCRC) patients, but they have dramatically increased therapeutic costs and have a heavy economic burden on the patient and medical system.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> The epigenetic alterations of CRC are considered to be an important biomarker of mCRC, and among the epigenetic targets, histone acetylation in tumor tissues is one of the most frequent changes in mCRC.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a> The histone acetylation level detected by immunohistochemical (IHC) staining is strongly correlated with the stages, metastasis, and prognosis of CRC.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">In the process of histone acetylation, two types of protein families are considered as potential drugable targets: bromodomain-containing protein 4 (BRD4) plays a role as a histone acetylated lysine “reader” and histone deacetylases (HDAC) act as an acetylated lysine “eraser”.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> BRD4 is a member of the bromodomain and extra-terminal (BET) protein family, and it acts as a central element in the recognition of histone or nonhistone substrates, which may regulate many molecular and cellular processes in epigenetic modification and gene transcription.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19">(15−19)</a> HDACs are actuators of deacetylation reactions of lysine residues, which have been recently deemed as potential therapeutic targets in human malignancies.<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24 ref25">(20−25)</a> There are four main subgroups containing 11 isoforms of HDAC that have been recently categorized: HDACs 1, 2, 3, and 8 are class I; HDACs 4, 5, 7, and 9 are class IIa; HDACs 6 and 10 are class IIb; and HDAC11 is class IV.<a onclick="showRef(event, 'ref13 ref26'); return false;" href="javascript:void(0);" class="ref ref13 ref26">(13,26)</a> In the past investigations on BRD4 and HDAC inhibition, a number of structurally diverse inhibitors have been reported, and some HDAC inhibitors were approved by the U.S. food and drug administration (FDA) or discovered in various clinical evaluation or preclinical stages for cancer therapy, for example, vorinostat,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> panobinostat,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> mocetinostat,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> trichostatin A,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> romidepsin,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> 9Z8,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and so forth. In addition, some BRD4 inhibitors were evaluated in preclinical or clinical evaluation stages, such as JQ-1,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> I-BET151,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> I-BET726,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> I-BET762,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> XD14,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> RVX-208,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> PFI-1,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> ZL0420,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Mivebresib (ABBV-075),<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Alobresib (GS-5829),<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> PLX51107,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> BIC1,<a onclick="showRef(event, 'ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46 ref47">(44−47)</a> and so forth (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of some known BRD4 and HDAC inhibitors: (A) selection of published BRD4 inhibitors with diverse scaffolds; (B) selection of published HDAC inhibitors with diverse zinc binding group (ZBG); and (C) structures of some reported BRD4–HDAC dual inhibitors based on known BRD4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">BRD4 and HDACs could synergistically downregulate c-Myc expression and inhibit the progress and metastasis of CRC. Moreover, Hu et al. reported that protein phosphatase 1 alpha (PP1α) and class I histone deacetylase signaling pathways are essential for releasing chromatin-bound BRD4 for positive transcriptional elongation factor b (P-TEFb) recruitment.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Mishra et al. reported novel insights into the action mechanisms of HDAC inhibition by eliciting BRD4 and c-Myc and inducing a subset of gene expression in pancreatic ductal adenocarcinoma.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Recently, Zeng et al. reported that the BRD4 inhibitor JQ1 could sensitize breast cancer cells to HDAC inhibition by the LIFR–JAK–STAT (leukemia inhibitory factor receptor–Janus kinase–signal transducer and activator of transcription) signaling pathways.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> There have been several reports about the combination of HDAC and BRD4 inhibitors as potential synergistic therapeutics for various cancers, for example, the suberoylanilide hydroxamic acid (SAHA) analogues of I-BET726 (DUAL946),<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> I-BET151 (Zhang et al.),<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and (+)-JQ1 (He et al.).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The previous reports of our laboratory suggested that the combination of RVX-208 and SAHA also resulted in enhanced antiproliferative effects in AML cells.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div><div class="NLM_p last">As a part of our continuous work on discovering histone epigenetic modification inhibitors as autophagic regulators,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> in the current study, we designed and synthesized a series of novel BRD4–HDAC dual inhibitors and performed a panel of subsequent in vitro and in vivo biological evaluations. Furthermore, compound <b>17c</b> induced apoptosis and autophagic cell death by downregulating c-Myc, deacetylation of histone H3, inhibiting apoptosis inhibitor protein Bcl-2, and interfering the formation of autophagolysosome in HCT-116 colorectal cancer cells. <b>17c</b> mainly inhibited HDAC 1, 2, 3, and 6, while both western blotting and IHC analyses revealed that <b>17c</b> potently inhibited the deacetylation of histone H3. In addition, <b>17c</b> displayed a good therapeutic efficiency on the HCT-116 colorectal cancer xenograft mouse and increased the cytotoxic T-cell infiltration via activation of IL6–JAK–STAT signaling pathways. In summary, we reported a series of novel small molecule BRD4–HDAC dual inhibitors, and <b>17c</b> exhibited a superior antiproliferative capacity on colorectal cancer compared to those of RVX-208 and vorinostat, suggesting that <b>17c</b> is a potential anticancer lead compound via selective BRD4 and HDAC inhibition and subsequent autophagic cell death.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Tissue Microarray and IHC Analyses</h3><div class="NLM_p">The expressions of BRD4 and HDAC2 were evaluated by three independent pathologists to avoid diagnostic bias. The HDAC2 was used as a representative biomarker to evaluate the expression level of HDACs. The relative expression levels of BRD4 and HDAC2 in both tumors and matched normal tissues of each case are illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. There were significant differences in both BRD4 and HDAC2 expression levels between tumor and matched normal tissues. Moreover, there was strong correlation between BRD4 and HDAC2 in tumor tissues. In the HDAC2 staining, 35.56% (16/45) of stage I–II CRC tumor tissues showed moderate to strong staining, while in stage III–IV tissues, the percentage of moderate to strong staining was up to 58.97% (23/39) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). Additionally, there were 48.88% (22/45) and 61.54% (24/39) of tumor tissues that showed moderate to strong BRD4 staining in stage I–II and III–IV tissues, respectively. In brief, the BRD4 and HDAC2 expressions were upregulated and correlated with the clinical stage and prognosis in CRC tissues, which suggested that BRD4 and HDAC may be potential targets for combined therapeutics.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Identification of HDAC2 and BRD4 are significantly upregulated in CRC clinical samples: (A) representative immunoreactivity intensities of HDAC2 and BRD4 in CRC tissues and adjacent normal tissues; (B) HDAC2 expression is upregulated in CRC tissues; (C) BRD4 expression is upregulated in CRC tissues; and (D) the upregulated HDAC2 and BRD4 expression are related to the CRC stages.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Discovery of Novel BRD4–HDAC Dual Inhibitors</h3><div class="NLM_p">A four-step strategy was performed for the discovery of small-molecule BRD4–HDAC dual inhibitor candidates: the fragment-based design of the novel BRD4 inhibitory scaffold, the synthesis of BRD4 inhibitors and in vitro enzymatic evaluation, the design and synthesis of a novel BRD4–HDAC inhibitor, and the systemic evaluation of this novel BRD4–HDAC dual inhibitor as an anticancer lead compound both in vitro and in vivo. The BRD4 protein has two bromodomains: an ubiquitin binding domain and a carboxyl-terminal domain. A canonical BRD4 protein contains four left-handed α helices and two loops (ZA and BC loops) to form the binding site of the acetylated lysine (Kac) residue. Most of the BRD4 inhibitors were designed to mimic the interaction modes of Kac, thus giving them the ability to disrupt the interaction between an acetylated histone and the Kac binding site within a BRD protein. First, the fragments generated from known BRD4 inhibitors were embedded into the fragment-like library of the ZINC database and then individually docked into the two bromodomains of the BRD4 protein by the Libdock program, a rigid docking method. The top 200 hit fragments were subsequently screened by the LigandFit program, a flexible docking method considering induced-fit effects. The top ten scoring fragments are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, and there were three structural characteristics in the screened fragments: the stable hydrogen bond interactions with the ZA-loop residues; the van der waals and hydrogen bond interactions in the Kac binding pocket; and a hydrophobic fragment to fulfill the steric requirement on the outside of the binding site. Second, based on these screened fragments, a focused library of twenty-four compounds was synthesized, and the BRD4 inhibitory activities of the compounds were characterized (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The general synthetic route of the target molecules <b>4a–t</b> and <b>9a–d</b> has been presented in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, respectively. Commercially available cyclohexanone, cyclohexanone, and N-substituted piperidine <b>1a–e</b> as the first starting material were reacted with ethyl cyanoacetate and sulfur to give <b>2a–e</b>, and then these five compounds were reacted with commercially available cyanobenzene and 3,4,5-substituted cyanobenzene (<b>3a–d</b>) to give <b>4a–t</b>. Similarly, <i>tert</i>-butyl 4-oxopiperidine-1-carboxylate was reacted with 2-cyanoacetamide and sulfur to give <b>5</b> by Gewald reaction, and then <b>6</b> was obtained through acylation. After the removal of protection, the ring-closure reaction was completed under the catalysis of strong alkali to give <b>8</b>. Finally, compounds <b>9a–d</b> were prepared by nucleophilic substitution.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Fragment-based drug design protocols for the identification of the novel BRD4–HDAC dual inhibitors (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1">4ZW1</a> for BRD4 BD1, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM">2YEM</a> for BRD4 BD2, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a> for HDAC2, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI">5WGI</a> for HDAC6, and PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0Z">3C0Z</a> for HDAC7).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCH<sub>2</sub>CO<sub>2</sub>Et, S<sub>8</sub>, Et<sub>3</sub>N, EtOH, reflux, 12 h; (b) HCl–1,4-dioxane, 25 °C, overnight.</p></p></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCH<sub>2</sub>CONH<sub>2</sub>, S<sub>8</sub>, Et3N, EtOH, reflux, 12 h; (b) 3,5-dimethylisoxazole-4-carbonyl chloride, pyridine, 3 h, 25 °C; (c) 4 M HCl/dioxane, 4 h, 25 °C; (d) NaOH, EtOH, 24 h, 100 °C; and (e) K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 25 °C or reflux, 6 h.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. In Vitro Enzymatic and HCT-116 Cell Proliferation Evaluations of Compounds <b>4a–t</b> and <b>9a–d</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">R<sub>4</sub></th><th class="colsep0 rowsep0" align="center">BRD4 IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">HCT-116 IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4a</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4b</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">9.28 ± 1.37</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4c</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4d</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">2.64 ± 0.54</td><td class="colsep0 rowsep0" align="left">4.85 ± 0.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4e</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4f</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">5.77 ± 0.63</td><td class="colsep0 rowsep0" align="left">9.73 ± 3.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4g</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4h</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">1.81 ± 0.23</td><td class="colsep0 rowsep0" align="left">5.41 ± 0.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4i</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4j</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">6.31 ± 1.08</td><td class="colsep0 rowsep0" align="left">8.68 ± 0.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4k</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4l</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">0.85 ± 0.16</td><td class="colsep0 rowsep0" align="left">1.33 ± 0.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4m</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4n</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4o</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4p</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">5.60 ± 1.92</td><td class="colsep0 rowsep0" align="left">8.91 ± 1.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4q</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4r</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4s</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4t</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2.41 ± 0.31</td><td class="colsep0 rowsep0" align="left">9.77 ± 1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9b</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>OOCCH<sub>2</sub></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">6.68 ± 1.08</td><td class="colsep0 rowsep0" align="left">16.21 ± 2.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9c</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>OOC(CH<sub>2</sub>)<sub>3</sub></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">5.93 ± 0.56</td><td class="colsep0 rowsep0" align="left">12.70 ± 1.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9d</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub>OOC(CH<sub>2</sub>)<sub>5</sub></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3.62 ± 0.53</td><td class="colsep0 rowsep0" align="left">17.03 ± 2.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>RVX-208</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2.45 ± 0.36</td><td class="colsep0 rowsep0" align="left">12.83 ± 2.40</td></tr></tbody></table></div></div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the enzymatic assay results of compounds <b>4a–t</b> demonstrated that the substitutes on the phenyl groups at the 2-position of thieno[2,3-<i>b</i>]pyrimidin-4-one played a key role in maintaining the BRD4 inhibitory activity. When the <i>para</i>-hydroxyl group was removed, the BRD4 inhibitory capacity was totally lost; moreover, methyl groups at the metaposition could obviously increase the potency of the inhibitor. The 3,5-dimethylisoxazole fragment displayed comparable BRD4 inhibitory capacity than that of 3,5-dimethyl-4-hydroxyl-phenyl fragment, the docked confirmations of <b>9a</b> were highly similar to that of <b>4l</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf" class="ext-link">Figure S1</a>). In addition, larger substitutes on the outside fragment could decrease the inhibitory activity.</div><div class="NLM_p">Subsequently, compound <b>4l</b> was chosen as a novel BRD4 inhibitor scaffold for dual inhibitor discovery. It was well known that a typical HDAC inhibitor contained three fragments: a cap fragment, a hydrophobic linker, and a ZBG group. In the design of BRD4–HDAC dual inhibitors, compound <b>4l</b> was used as the cap fragment. There were two linkage sites in compound <b>4</b>: the hydroxyl group on the 3,5-dimethyl-4-hydroxyl-phenyl group and the N atom in the piperidine ring. According to the previous report of our laboratory, the BRD4–HDAC dual inhibitor based on RVX-208, the ZBG group could be linked to the 3,5-dimethyl-4-hydroxyl-phenyl group. We chose the <i>N</i>-hydroxyhexanamide and phthalate fragments as model groups to explore the correct linking mode of the cap fragment, respectively. The model compounds were synthesized and characterized for their BRD4–HDAC inhibition as well as cell proliferation inhibitory activity in vitro (<a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The <i>N</i>-hydroxyhexanamide and phthalate fragments were also attached to the piperidine ring of compound <b>9</b> to afford potential BRD4–HDAC dual inhibitors <b>21</b> and <b>22</b>. It was surprising that compounds <b>20</b>, <b>21</b>, and <b>22</b> displayed no significant BRD4 inhibitory capacity up to 10 μM, and compound <b>17</b> was greatly superior to compound <b>20</b> in cell proliferation assays. In addition, the HDAC2 and HDAC6 inhibitory activities of compound <b>17c</b> were moderately superior to that of compound <b>20</b> (58–136 and 73–129 nM), respectively. Then, the different linkages and ZBG fragments were conjugated on the scaffold of compound <b>17</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The synthetic routes of these dual inhibitors are outlined in <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a>–<a class="ref internalNav" href="#sch5" aria-label="5">5</a>. Compounds <b>11a–f</b> were reacted with 3,5-dimethyl-4-hydroxybenzonitrile to give <b>12a–f</b>, which were then hydrolyzed with NaOH to give <b>13a–f</b>. These compounds were treated with phthalates or NH<sub>2</sub>OH to give <b>14a–c</b> or <b>15a–f</b>. Compounds <b>14a–c</b> were reacted with <b>2b</b> to give <b>16a–c</b>, and <b>15a–f</b> reacted with <b>2b</b> to afford <b>17a–f</b>. On the other hand, the co-crystallized structure of <b>4l</b> to BRD4 suggested another binding mode, the <i>N</i>-methylpiperidine moiety exposed out of the binding pocket, providing another feasible linkage site for ZBG. As depicted in <a class="ref internalNav" href="#sch4" aria-label="Schemes 4">Schemes 4</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>, commercially available piperidin-4-one as the first starting material was reacted with ethyl cyanoacetate and sulfur to give compound <b>18</b>, which was reacted with methyl 6-bromohexanoate to give compound <b>19</b>, and then 4-hydroxy-3,5-dimethylbenzonitrile was reacted with compound <b>19</b> in HCl/1,4-dioxane under room temperature for 48 h; the resultant was treated with hydroxylamine aqueous solution in the mixed solvent CH<sub>3</sub>OH and CH<sub>2</sub>Cl<sub>2</sub> to give compound <b>20</b>. As described before, compounds <b>21</b> and <b>22</b> were prepared by amination and amidation of <b>9b–d</b>, respectively.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HCl–1,4-dioxane, 25 °C, 48 h; (b) 1,4-dioxane, NaOH, 30 min; (c)DCC/HOBt, DCM, 24 h; (d) ethyl chloroformate, <i>N</i>-methylmorpholine, 1,4-dioxane, 15 min and then NH<sub>2</sub>OH, CH<sub>3</sub>OH, 20 min; and (e) compound <b>4l</b>, HCl–1,4-dioxane, 25 °C, overnight.</p></p></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCH<sub>2</sub>CO<sub>2</sub>Et, S<sub>8</sub>, Et<sub>3</sub>N, EtOH, Reflux, 12 h; (b) methyl 6-bromohexanoate, DBU, THF, reflux, overnight; and (c) 4-hydroxy-3,5-dimethylbenzonitrile, HCl–1,4-dioxane, rt, 48 h and then CH<sub>3</sub>OH, CH<sub>2</sub>Cl<sub>2</sub>, HONH<sub>2</sub>, 2 h.</p></p></figure><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOH/CH<sub>3</sub>OH, hydroxylamine aqueous solution (50:50, w/w), 4 M HCl/dioxane; (b) 2 N NaOH, 4 M HCl/dioxane, and thenbenzene-1,2-diamine, EDCI, HOBt, CH<sub>2</sub>Cl<sub>2</sub>, 12 h, 25 °C.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. In Vitro Enzymatic Assays and CRC Cell Proliferation Evaluations of Compounds <b>16a–c</b>, <b>17a–f</b>, <b>20</b>, <b>21a–c</b>, and <b>22a–b</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0015.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">ND: Not determined.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">The molar ratio of RVX-208 and SAHA was set to 1:1.</p></div></div><div></div></div><div class="NLM_p">The synthesized dual inhibitors were assayed for their inhibitory activities on BRD4 as well as HDAC2, HDAC6, and HDAC7 by using RVX-208 and SAHA as positive controls, respectively. Compounds <b>17a–f</b>, which used hydroxamic acid as the ZBG fragment and various types of linker group positions, had better HDAC inhibitory activities than those of compounds <b>16a–c</b> with <i>o</i>-phenylenediamine as the ZBG fragment. Moreover, the inhibitory potencies of all of the compounds on HDAC2 and HDAC6 were greatly superior to those on HDAC7; these results suggested that the novel dual inhibitors might selectively inhibit class I- and IIb-type HDACs rather than class IIa. In addition, compounds <b>16a–c</b> and <b>17a–f</b> also displayed good BRD4 inhibitory capacities at a sub-micromolar scale, which suggested that the ZBG and linker moiety only slightly affected the BRD4 inhibitory effects. Intriguingly, the linkage of the ZBG moiety on the piperidine ring resulted in the totally loss of BRD4 inhibitory activities in compounds <b>20–22</b>. The most potent compound <b>17c</b>, with IC<sub>50</sub> values of 58 and 73 nM on HDAC2 and HDAC6, respectively, displayed good selectivity for HDAC 2/6 versus HDAC7. Compound <b>17c</b> also displayed a moderate inhibitory capacity on BRD4, with an IC<sub>50</sub> value of 710 nM, which was better than that of the positive control RVX-208.</div><div class="NLM_p">Because compound <b>17c</b> demonstrated both potent enzymatic and antiproliferative activity in vitro, it was further tested for its isoform selectivity on the other isoforms of BRDs and HDACs. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, compound <b>17c</b> potently inhibited HDAC1-3 and HDAC6 with IC<sub>50</sub> values from 45 to 482 nM and at the sub-micromolar level against HDAC8 and HDAC10, respectively. There were almost no inhibitory effects of compound <b>17c</b> on class IIa and class IV HDAC isoforms up to a concentration of 10 μM. Moreover, it was notable that compound <b>17c</b> displayed better activities on BRD4 than on the other isoforms of BRD proteins by using the Alphascreen assays; compared to the positive control compound JQ-1, compound <b>17c</b> showed higher selectivity to BRD4 with moderate IC<sub>50</sub> values. We then evaluated the antiproliferative capacity of <b>17c</b> on human CRC HCT-116, SW620, and DLD1 cells, and <b>17c</b> displayed good activities with IC<sub>50</sub> values of 0.45, 1.78, and 2.11 μM on HCT-116, SW620, and DLD1 cells, respectively. In summary, the above results suggested that <b>17c</b> deserved further development with potent BRD4–HDAC dual inhibition.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. HDAC and BRD Inhibitory Profiling of Compound <b>17c</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center"><b>17c</b></th><th class="colsep0 rowsep0" align="center">JQ-1</th><th class="colsep0 rowsep0" align="center">SAHA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center"><b>Class I HDAC</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC1</b></td><td class="colsep0 rowsep0" align="left">0.046 ± 0.003</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.017 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC2</b></td><td class="colsep0 rowsep0" align="left">0.058 ± 0.004</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.035 ± 0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC3</b></td><td class="colsep0 rowsep0" align="left">0.075 ± 0.009</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.049 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC8</b></td><td class="colsep0 rowsep0" align="left">0.167 ± 0.029</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.195 ± 0.031</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center"><b>Class IIa HDAC</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC4</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC5</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC7</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC9</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center"><b>Class IIb HDAC</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC6</b></td><td class="colsep0 rowsep0" align="left">0.073 ± 0.005</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.047 ± 0.007</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC10</b></td><td class="colsep0 rowsep0" align="left">0.923 ± 0.005</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.278 ± 0.056</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center"><b>Class IV HDAC</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>HDAC11</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center"><b>BRDs</b><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRD2(BD1)</b></td><td class="colsep0 rowsep0" align="left">16.4 ± 1.13</td><td class="colsep0 rowsep0" align="left">0.155 ± 0.005</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRD2(BD2)</b></td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">0.017 ± 0.003</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRD3(BD1)</b></td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">0.064 ± 0.004</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRD3(BD2)</b></td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">0.039 ± 0.002</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRD4(BD1)</b></td><td class="colsep0 rowsep0" align="left">1.10 ± 0.14</td><td class="colsep0 rowsep0" align="left">0.022 ± 0.005</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRD4(BD2)</b></td><td class="colsep0 rowsep0" align="left">2.05 ± 0.71</td><td class="colsep0 rowsep0" align="left">0.069 ± 0.007</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>BRDT(BD1)</b></td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">0.134 ± 0.011</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">ND: Not determined.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">The BRD inhibitory profiling was performed by an Alphascreen-based method reported in our previous study and using JQ-1 as a positive control.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Binding Modes of <b>17c</b> to BRD4, HDAC2, and HDAC6</h3><div class="NLM_p">The compound <b>17c</b> was docked into the binding pocket of BRD4, HDAC2, and HDAC6 proteins. The docking results are illustrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. The hydroxamic group of compound <b>17c</b> stable chelated with the zinc atom in the binding sites of HDACs. Moreover, the thieno[2,3-<i>d</i>]pyrimidine fragment of compound <b>17c</b> fitted into the surface groove and occupied the cap region well. The carbonyl fragment of the hydroxamic group could form a hydrogen bond interaction with Tyr308 residues of HDAC2 and Tyr745 residues of HDAC6, respectively. The hydroxylamine fragment of <b>17c</b> interacted with His145/His146 residues of HDAC2 and His614 residue of HDAC6 via stable hydrogen bonds, respectively. In addition, the phenyl substituted at 2-position of thieno[2,3-<i>d</i>]pyrimidine formed a cation−π interaction with Arg275 residue of HDAC2. In addition, the 3-nitrogen atom and 4-carbonyl group of thieno[2,3-<i>d</i>]pyrimidine could form H-bond with Asn140 residues of BRD4. The <i>N</i>-methylpiperidine group was placed between the WPF stack (Trp81, Pro82, and Phe83). Furthermore, <b>17c</b> derived supplementary activities by deploying the long alkane chain and hydroxamic group into the ZA channel. In general, the molecular docking results displayed good agreement with the results of enzymatic assays.</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docked conformations and binding modes of <b>17c</b> to BRD4 (A, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1">4ZW1</a>), HDAC2 (B, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>), and HDAC6 (C, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI">5WGI</a>), respectively. The proteins were illustrated by a solvent-accessible surface with charges (blue for positive and red for negative); the binding site residues were highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> <b>17c</b> Induces Cell Cycle Arrest, Apoptosis, and Autophagic Cell Death and in CRC Cells</h3><div class="NLM_p">The cytotoxicity of <b>17c</b> was screened against a panel of CRC cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the IC<sub>50</sub> of <b>17c</b> was remarkably higher in CRC cells than in normal colorectal epithelial cells. Moreover, the cell cycles of HCT-116 cells after being treated with <b>17c</b> were determined by a flow cytometry method. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, the cell cycle results displayed obviously sub-G1 peaks in the 0.2 and 0.5 μM of compound <b>17c</b>-treated HCT-116 cells, which indicated that the apoptotic cell death occurred. The death subroutines of <b>17c</b>-treated HCT-116 cells were detected by flow cytometry analysis using the annexin V/PI staining kit (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D). The percentages of annexin V-positive apoptotic cells in the <b>17c</b> treatment groups were 33.2 ± 5.30% (0.2 μM) and 45.3 ± 6.8% (0.5 μM), which were significantly higher than in the control (6.3 ± 0.4%, <i>p</i> < 0.05) group. However, there were significant differences in apoptotic cells between the 0.2 and 0.5 μM of <b>17c</b>-treated groups. These results suggest that <b>17c</b> induced apoptotic death in a dose-dependent manner. The nuclei morphology after Hoechst 33258 staining (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E) in <b>17c</b>-treated HCT-116 cells revealed that apoptotic cell death was apparent after 0.2 and 0.5 μM <b>17c</b> treatments. The colony formation of HCT-116 cells was also potently suppressed after <b>17c</b> incubation for 24 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F).</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>17c</b> induced apoptotic cell death in HCT-116 cells: (A) cell proliferation IC<sub>50</sub> values were measured for compound <b>17c</b> on a panel of CRC cells by MTT assay; (B) cell cycle analysis of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated; (C,D) apoptosis assay of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated by annexin V/PI dual staining; (E) nuclei morphology of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated; and (F) colony formation assay of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To determine the effects of <b>17c</b>, RVX-208, and SAHA on the expression level of histone H3 as well as acetylated histone H3, the biomarkers of HDAC inhibition in HCT-116 cells were determined via the western blotting method. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, in agreement with their relative potencies in enzymatic assays, <b>17c</b> and SAHA upregulated Ac-H3 levels in a dosage-dependent manner, and the expression levels of histone H3 were not affected, while both H3 and Ac-H3 were not disturbed by RVX-208 (data not shown). Moreover, the expression levels of BRD4 were declined by the addition of <b>17c</b> or RVX-208, and <b>17c</b> treatment suppressed the downstream c-Myc proteins more efficiently than that of RVX-208. As expected, <b>17c</b> induced more cleavage of PARP, caspase-3, and caspase-9, which suggested more mitochondrial apoptosis was induced by BRD4–HDAC dual inhibitor.</div><figure id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>17c</b> induced mitochondrial apoptosis via BRD4–HDAC dual inhibition. HCT-116 cells were treated with 0.2 or 0.5 μM of compound <b>17c</b>, positive control RVX-208, or SAHA for 24 h, respectively. Then, the expression levels of BRD4, c-Myc, Histone H3, Ac-H3, Bim, Bcl-2, Fas, FasL, PARP, caspase-3, and caspase-9 were detected by Western blot (WB) analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There were several reports that suggested BRD4 protein was an autophagy suppressor in cancer cells; therefore inhibition of BRD4 might result in autophagy initiation.<a onclick="showRef(event, 'ref55 ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref55 ref57 ref58 ref59">(55,57−59)</a> In current study, we observed <b>17c</b> induced the formation of a cytoplasm autophagic vacuole in HCT-116 cells under a transmission electron microscope (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A), and the increased LC3 puncta in GFP-LC3 transfected HCT-116 cells was also observed after 0.5 μM of <b>17c</b> incubation for 24 h; these results suggested the initiation of autophagy in HCT-116 cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). The <b>17c</b>-induced autophagy was further confirmed by the WB analysis of autophagy protein markers, and we found that <b>17c</b> incubation resulted in the declined expression of p62, upregulation of beclin-1, and dosage-dependent cleavage of LC3-I to LC3-II in HCT-116 cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). To further validate the cytotoxic or cyto-protective effects of <b>17c</b>-induced autophagy, 3-MA (3-methyladenine), an inhibitor of type III phosphatidylinositol 3-kinases, was employed to block the initiation of autophagic flux. As depicted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D, the cytotoxicity of <b>17c</b> was potently reversed by the addition of 500 μM 3-MA (<i>p</i> < 0.01). In addition, an autophagolysosome inhibitor bafilomycin A1 (BafA1) was utilized to validate the effects of <b>17c</b> on autophagic flux (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E). Both LC3-II and p62 were accumulated after the combined treatment of BafA1 and <b>17c</b>. These results suggest that compound <b>17c</b> induced autophagic cell death in HCT-116 CRC cells. In recent reports, the BRD4–AMPK–mTOR–ULK signaling axis was suggested as an upstream autophagy signaling pathway in cancer cells.<a onclick="showRef(event, 'ref55 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref55 ref59 ref60">(55,59,60)</a> To further determine whether <b>17c</b>-induced autophagy was dependent on ATG5, the ATG5 mRNA was interfered by a specific siRNA with or without the combination of <b>17c</b> incubation (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>F). In agreement with our previous report, the mRNA knockdown of ATG5 did not interfere the <b>17c</b>-induced AMPK activation; however, the cleavage of LC3 and degradation of p62 were obviously suppressed. Moreover, the autophagy induction capacities of <b>17c</b> and the corresponding BRD4i and HDACi fragments were detected by WB analysis (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>G). The high concentration of SAHA (10 μM) only led to marginal autophagy, and 0.5 μM <b>17c</b> incubation resulted in stronger autophagy than that of 2.0 μM <b>4l</b>. Taken together, above results indicated that <b>17c</b> induces autophagic cell death via the BRD4–AMPK signaling pathway.</div><figure id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>17c</b> induces autophagic cell death in CRC cells. (A) HCT-116 cells were incubated with 0.5 μM of <b>17c</b> for 24 h; then the cells were fixed and imaged under transmitted electron microscopy; scale bar = 1 μm. (B) HCT-116 cells were transfected with a GFP-LC3 plasmid, followed by treatment with 0.5 μM <b>17c</b>; the formation of autophagosomes were observed under a confocal fluorescence microscope, scale bar = 6 μm. (C) After <b>17c</b> incubation for 24 h, the protein expression of LC3, p62, and beclin-1 were detected by WB analysis. (D) HCT-116 cells were treated with 0.5 μM of <b>17c</b> and/or 500 μM of 3-MA; then the cell viability was determined by MTT assay. *** indicated <i>p</i> < 0.001 and ** indicated <i>p</i> < 0.01. (E) HCT-116 cells were treated with 0.5 μM of <b>17c</b> with or without bafilomycin-A1 for 24 h; the protein expression of LC3 and p62 were detected by WB analysis. (F) HCT-116 cells were transfected with ATG5 siRNA or scramble siRNA, respectively. After RNA silence, with 0.5 μM of <b>17c</b> or normal saline incubation, the protein expression of ATG5, AMPKα, p-AMPKα, p62, and LC3 were determined by WB analysis. (G) HCT-116 cells were treated with 0.5 μM <b>17c</b>, 2.0 μM <b>4l</b>, or 10 μM SAHA for 24 h. Then, the protein expressions of LC3 and p62 were detected by WB analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> <b>17c</b> Suppresses IL6–JAK–STAT Pathway in CRC Cells</h3><div class="NLM_p">Recent studies suggested that the activation of the IL6–JAK–STAT signaling pathway might be a vital factor of drug resistance to HDAC inhibitors in cancer cells.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The addition of BRD4 siRNA or inhibitor could reverse this drug resistance via suppressing the activation of IL6 family cytokines. In the current study, we also observed the transcriptional activation of four IL-6 family cytokines by RT-PCR, including interleukin 6, oncostatin M (OSM), leukemia inhibitory factor (LIF), and cardiotrophin 1 (CTF1), was stimulated by 10 μM SAHA incubation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). The <b>4l</b> treatment did not change the expression of these cytokines, <b>17c</b> treatment only slightly elevated these cytokines. The WB analysis of representative protein markers in IL6–JAK–STAT signaling pathways also indicated that SAHA could activate IL6 family cytokines and downstream JAK–STAT signaling axis, and these effects of HDAC inhibitors were not observed in the <b>17c</b>-treated group (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). These results suggested that these BRD4–HDAC dual inhibitors possibly overcome the drug resistance mechanism of single HDAC inhibition.</div><figure id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Change of IL6, OSM, LIF, and CTF1 mRNA levels in HCT-116 cells after incubation with 0.5 μM <b>17c</b>, 2.0 μM <b>4l,</b> or 10 μM SAHA for 24 h, respectively. (B) HCT-116 cells were treated with 0.5 μM <b>17c</b>, 5.0 μM <b>4l,</b> or 10 μM SAHA for 24 h, respectively. Then, the protein expressions of OSMR, LIFR, JAK1, STAT3, and phosphorylated STAT3 were detected by WB analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <b>17c</b> Inhibits Tumor Growth in Xenograft Models</h3><div class="NLM_p">We subsequently determined that <b>17c</b> bears potent inhibitory effects on the BRD4–HDAC pathways in vivo, and then, HCT-116 human CRC xenograft models were established to evaluate the efficacy of <b>17c</b>. The mean tumor data after 19 days of oral administration of SAHA or <b>17c</b> are presented in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. Both <b>17c</b> and SAHA exhibited potent antitumor activity at the dosage tested, and the average percentage of tumor growth inhibition in the <b>17c</b> groups was 68.8%. Moreover, the <b>17c</b>-treated groups displayed a dose-dependent decrease in tumor volume compared with the saline control (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B), with total treatment-to-control ratios (T/C) in the <b>17c</b> 15 and 30 mg/kg groups of 42.7 and 68.8%, respectively. As expected, there was a significant difference between the <b>17c</b> group and the SAHA group. Therefore, <b>17c</b> was effective in reducing the growth of HCT-116 tumors in a xenograft model in vivo. Furthermore, to confirm whether the molecular mechanism of <b>17c</b> was based on the BRD4 and HDAC pathways in vivo, the changes in the expression levels of Ac-H3 and c-Myc were determined by IHC methods as well as apoptosis marker TUNEL (terminal deoxynucleotidyl transferase dUTP nick end labeling) and Ki-67 staining. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D, the expression levels of Ac-H3 and c-Myc in HCT-116 xenograft tumor tissues in the vehicle- and <b>17c</b>-treated groups were measured by IHC analysis. The expression level of c-Myc was reduced in the <b>17c</b>-treated group, and Ac-H3 was elevated. Ki-67 is a proliferation marker with prognostic and predictive potential in CRC. The results of the IHC analysis and statistical analysis were also presented in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D. A significant reduction in the expression of Ki-67 was observed upon treatment with <b>17c</b>. Histological assessment of apoptosis by TUNEL staining revealed that TUNEL-positive apoptotic nuclei were significantly increased by <b>17c</b>. In agreement to the in vitro experiments, the autophagy marker LC3-II was elevated in <b>4l</b> and <b>17c</b>-treated groups. The protein expressions of LIFR and phosphorylated STAT3 were also activated in SAHA treated group, and these phenomena were not observed in the <b>17c</b>-treated group. In addition, there were no significant changes observed in the H&E (hematoxylin and eosin) staining sections of the main organs as well as body weights of mice in the <b>17c</b>-treated groups (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf" class="ext-link">Figure S2</a>).</div><figure id="fig9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. In vivo anticancer effect of compound <b>17c</b> on the HCT-116 xenograft models: (A) tumor volume changes with oral administration 15 or 30 mg/kg of compound <b>17c</b>, 30 mg/kg of <b>4l</b> or SAHA, **: <i>p</i> < 0.01 compared to the control group; (B) tumor weights analysis after the end of therapy; (C) body weight changes of mice; (D) representative images of IHC analysis of proliferative marker Ki-67, c-Myc, Ac-H3, BRD4, LC3-II, LIFR, p-STAT3, and immunofluorescent analysis of apoptosis marker TUNEL in different groups; scale bar = 50 μm; *: <i>p</i> < 0.05 compared to control group and **: <i>p</i> < 0.01 compared to control group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Oral Pharmacokinetics and Preliminary Safety Evaluation</h3><div class="NLM_p">Compound <b>17c</b> (encapsulated in HP-β-CD for intravenous administration and 0.5% CMC solution for oral administration) was administered to SD rats intravenously (iv) or orally at a 30 mg/kg dosage. The drug concentrations of <b>17c</b> in the plasma were analyzed on a Waters UPLC–MS system. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, the oral AUC value, in vivo half-life, and oral bioavailability of <b>17c</b> were 63,279.5 μg·L<sup>–1</sup>·h, 1.53 h, and 40.5%, respectively. <b>17c</b> could be enriched in the tumor tissues and can prolong its antitumor effects (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf" class="ext-link">Figure S3</a>). The predicted ADMET descriptors also suggested that the physicochemical properties of <b>17c</b> was located in a reasonable range (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf" class="ext-link">Figure S4</a>). These results indicated that compound <b>17c</b> possesses a good PK profile, which provides the foundation for further development.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pharmacokinetic Parameters Tested in Vivo</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><b>17c</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">route</th><th class="colsep0 rowsep0" align="center" char=".">iv</th><th class="colsep0 rowsep0" align="center" char=".">po</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>N</i><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dose (mg/kg)</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–inf</sub> (μ·g·L<sup>–1</sup>·h)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">156,245.5</td><td class="colsep0 rowsep0" align="char" char=".">63,279.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μ g·L<sup>–1</sup>)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">5361.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">0.12</td><td class="colsep0 rowsep0" align="char" char=".">1.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="char" char=".">2.48</td><td class="colsep0 rowsep0" align="char" char=".">4.80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">40.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Numbers of rats.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Area under the curve following intravenous dosing, integrated drug concentration with respect to time, and integrated drug concentration with respect to time following oral dosing.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Maximum plasma concentration after oral administration.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Time to archive maximum plasma concentration after oral administration.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Plasma half-life.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">Percentage of oral bioavailability.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31004" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31004" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To develop novel BRD4–HDAC dual inhibitors against CRC, a series of hydroxamic acid derivatives of thieno[2,3-<i>d</i>]pyrimidine were designed, synthesized, and evaluated for their activities in vitro and in vivo. The structure–activity relationship (SAR) analysis based on the enzymatic inhibition capacities on the class I and IIa isoform HDACs and BRDs of these synthesized compounds indicated that the main goal of finding novel small molecule BRD4–HDAC dual inhibitors was successfully achieved. The most potent compound <b>17c</b> demonstrated good inhibitory activity and selectivity on HDACs and BRDs. The cell proliferation results suggested that <b>17c</b> suppressed the growth of CRC cells with IC<sub>50</sub> values at sub-micromolar levels. Subsequently, the western blotting as well as flow cytometry analyses further confirmed the molecular mechanisms of <b>17c</b>. It also displays potent inhibitory efficiency on tumor growth by inducing autophagic cell death and suppressing IL6–JAK–STAT signaling pathways in the in vitro and in vivo CRC models. Moreover, compound <b>17c</b> could be orally administrated with an oral bioavailability of 40.5% in rats, and it displayed dose-dependent antitumor activity in xenograft CRC models. Taken together, these integrated design, synthesis, and in vitro and in vivo studies indicate that the small molecule BRD4–HDAC dual inhibitor <b>17c</b> is a potential lead compound for the novel chemotherapeutics of CRC.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78206" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78206" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Tissue Microarray and IHC Staining</h3><div class="NLM_p last">A human colorectal cancer tissue microarray (TMA) was purchased from Shanghai Outdo Biotech (OD-HColA180Su11). This TMA contained 180 tumors and matched adjacent normal tissues from 84 patients, including 70 adenocarcinomas, seven mucinous adenocarcinoma, and seven canalicular adenocarcinoma. The protocol of IHC staining was performed by using rabbit monoclonal antibodies against BRD4 (Abcam, USA, 1:50 dilution) and HDAC2 (Abcam, USA, 1:200 dilution) as in our previously described studies unless otherwise stated. The results were interpreted in a semi-quantitative manner, and the expressions of BRD4 and HDAC2 were identified as the percentage of positive tumor cells by three independent pathologists to avoid different thresholds of positively stained tumor cells.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Fragment-Based Molecular Design</h3><div class="NLM_p last">The coordinate files of BRD4 and HDACs were retrieved from the PDB database. Three crystallographic structures of HDACs, HDAC2 in class I (PDB no. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>), HDAC7 in class IIa (PDB no. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0Z">3C0Z</a>), and HDAC6 in class IIb (PDB no. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI">5WGI</a>), were utilized for the molecular docking studies. The three HDAC proteins complexed with SAHA were aligned and superposed by sequence similarity. The BRD4 protein contained two bromodomains, and the first bromodomain (PDB no. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1">4ZW1</a>) and the second bromodomain (PDB no. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM">2YEM</a>) were also aligned together by sequence similarity. All hydrogen atoms were minimized in the Accelrys Discovery Studio package by using the CHARMM27 force field. The fragments were generated from the known BRD4 inhibitors and then embedded into the ZINC fragment-like library. By using the MCSS (multiple copy simultaneous search) program in the Discovery Studio Package, the fragments were docked and scored into the binding pockets of BRD4 proteins. The top-scoring fragments were selected as novel scaffolds, merged, or optimized by the LUDI program in the Discovery Studio Package and in situ minimized by the Libdock program in the Discovery Studio Package. The 1000 top-ranked molecules according to docking scores were evaluated by Verber drug-likeness filters, the Lipinski’s “Rule-of-Five”, and the ADMET (absorption, distribution, metabolism, excretion and toxicity) filter developed by our library to remove improper molecules. The optimized molecules were synthesized, and their BRD4 binding potencies were initially evaluated on HTRF assays. The screened molecules were individually linked to the ZBG of the HDAC inhibitor and then docked into the binding pockets of three HDAC proteins.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Chemistry</h3><div class="NLM_p last">All of the chemicals were purchased as analytically pure and used without further purification. The <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were performed on a Bruker Avance III 400 MHz spectrometer (Bruker Co. Ltd., Germany). Chemical shifts were given in ppm (parts per million) with TMS as an internal standard. Mass spectra were recorded on a Waters Q-TOF mass spectrometer (Micromass, Manchester, U.K.). The purity of each compound (>95%) was determined on a Shimadzu LC-20 instrument with a GL-C18 reverse-phase column (4.6 mm × 150 mm, 5 μm) with methanol, acetonitrile, and water as the mobile phase.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedures of Method for the Synthesis of <b>4a–4t</b></h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Ethyl 2-Amino-5,6-dihydro-4<i>H</i>-cyclopenta[<i>b</i>]thiophene-3-carboxylate (<b>2a</b>)</h4><div class="NLM_p last">To a mixture of cyclopentanone (<b>1a</b>) (4.20 g, 50 mmol), ethyl cyanoacetate (5.66 g, 100.0 mmol), and sulfur (1.60 g, 50.0 mmol) in absolute ethanol (100 mL) was added triethylamine (10 mL) and refluxed for 12 h; the reaction mixture was concentrated, and the residue was partitioned between water and ethyl acetate. The organic layer was separated and concentrated, and the crude product was purified by silica gel column chromatography using a mixture solvent of petroleum ether/ethylacetate (8:1) to give <b>2a</b>. Yield 76%, white powder. <sup>1</sup> H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.18 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 4.13 (q, <i>J</i> = 7.2 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>3</sub>), 2.70 (t, <i>J</i> = 7.2 Hz, 2H, S–C–<b>CH</b><sub><b>2</b></sub>), 2.62 (t, <i>J</i> = 7.2 Hz, 2H, <b>CH</b><sub><b>2</b></sub>-thiophene), 2.22 (p, <i>J</i> = 7.2 Hz, 2H, CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 1.23 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Ethyl 2-Amino-4,5,6,7-tetrahydrobenzo[<i>b</i>]thiophene-3-carboxylate (<b>2b</b>)</h4><div class="NLM_p last">To a mixture of cyclohexanone (<b>1b</b>) (4.90 g, 50 mmol), ethyl cyanoacetate (5.66 g, 100.0 mmol), and sulfur (1.60 g, 50.0 mmol) in absolute ethanol (100 mL) was added triethylamine (10 mL) and refluxed for 12 h; the reaction mixture was concentrated, and the residue was partitioned between water and ethyl acetate. The organic layer was separated and concentrated, and the crude product was recrystallized with ethanol (100 mL) to give <b>2b</b>. Yield 72%, light yellow crystal. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.20 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 4.14 (q, <i>J</i> = 7.2 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>3</sub>), 2.59 (t, <i>J</i> = 5.6 Hz, 2H, S–C–<b>CH</b><sub><b>2</b></sub>), 2.41 (t, <i>J</i> = 5.6 Hz, 2H, <b>CH</b><sub><b>2</b></sub>-thiophene), 1.74–1.58 (m, 4H, CH<sub>2</sub><b>CH</b><sub><b>2</b></sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 1.24 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Ethyl 2-Amino-6-methyl-4,5,6,7-tetrahydrothieno[2,3-<i>c</i>]pyridine-3-carboxylate (<b>2c</b>)</h4><div class="NLM_p last">Compound <b>2c</b> was synthesized from <i>N</i>-methyl-4-piperidone (<b>1c</b>), in a manner similar to <b>2b</b>. Yield 79%, yellow powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.23 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 4.15 (q, <i>J</i> = 7.2 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>3</sub>), 3.22 (t, <i>J</i> = 2.0 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.66 (t, <i>J</i> = 5.2 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.53 (t, <i>J</i> = 6.0 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.29 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 1.24 (t, <i>J</i> = 7.2 Hz, 3H, CH<sub>2</sub><b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Ethyl 2-Amino-6-ethyl-4,5,6,7-tetrahydrothieno[2,3-<i>c</i>]pyridine-3-carboxylate (<b>2d</b>)</h4><div class="NLM_p last">Compound <b>2d</b> was synthesized from <i>N</i>-ethyl-4-piperidone (<b>1d</b>), in a manner similar to <b>2b</b>. Yield 72%, yellow powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.23 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 4.14 (q, <i>J</i> = 7.2 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>), 3.27 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.65 (t, <i>J</i> = 5.9 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.59 (t, <i>J</i> = 5.2 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.45 (q, <i>J</i> = 7.2 Hz, 2H, N<b>CH</b><sub><b>2</b></sub>), 1.24 (t, <i>J</i> = 7.2 Hz, 3H, OCH<sub>2</sub><b>CH</b><sub><b>3</b></sub>), 1.04 (t, <i>J</i> = 7.2 Hz, 3H, NCH<sub>2</sub><b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Ethyl 2-Amino-6-benzyl-4,5,6,7-tetrahydrothieno[2,3-<i>c</i>]pyridine-3-carboxylate (<b>2e</b>)</h4><div class="NLM_p last">Compound <b>2e</b> was synthesized from <i>N</i>-benzyl-4-piperidone (<b>1e</b>), in a manner similar to <b>2b</b>. Yield 69%, yellow powder. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.39–7.26 (m, 5H), 5.93 (s, 2H, Ar-<b>CH</b><sub><b>2</b></sub>), 4.25 (q, <i>J</i> = 7.2 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>3</sub>), 3.67 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.82 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.74 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.31 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 2-Phenyl-3,5,6,7-tetrahydro-4<i>H</i>-cyclopenta[4,5]thieno[2,3-<i>d</i>]pyrimidin-4-one (<b>4a</b>)</h4><div class="NLM_p last">Compound <b>2a</b> (105.50 mg, 0.5 mmol) and cyanobenzene (<b>3a</b>, 51.50 mg, 0.5 mmol) were placed in a 25 mL flask, and then 5 mL of 1,4-dioxane saturated by HCl was added. The reaction mixture was stirred for 48 h at room temperature; the reaction solvent was concentrated; the residues were added 10 mL H<sub>2</sub>O; this solvent was neutralized by NaHCO<sub>3</sub> to pH = 7 to give a suspension and centrifugated (4500 rounds/minute); and the resultant solids was dried and finally washed with ethylacetate (5 mL) and ethanol (5 mL) to give <b>4a</b>. Yield 58%, off-white powder. mp 284.4–286.9 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.26–8.15 (m, 2H, Ar-<b>H</b>), 7.51–7.36 (m, 3H, Ar-<b>H</b>), 2.96–2.82 (m, 4H, 2<b>CH</b><sub><b>2</b></sub>-thiophene), 2.35 (p, <i>J</i> = 7.2 Hz, 2H, CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.34, 155.13, 152.64, 145.20, 141.72, 136.42, 130.33, 127.49, 123.09, 119.22, 28.39, 27.59, 27.40. HRMS (ESI): calcd forC<sub>15</sub>H<sub>12</sub>N<sub>2</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 291.0563; found, 291.0566.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 2-(4-Hydroxyphenyl)-3,5,6,7-tetrahydro-4<i>H</i>-cyclopenta[4,5]thieno[2,3-<i>d</i>]pyrimidin-4-one (<b>4b</b>)</h4><div class="NLM_p last">Compound (<b>4b</b>) was synthesized from (<b>2a</b>) and 4-hydroxybenzonitrile (<b>3b</b>), in a manner similar to (<b>4a</b>). Yield 39%, gray powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.10–8.01 (m, 2H, Ar-<b>H</b>), 6.84–6.73 (m, 2H, Ar-<b>H</b>), 2.94–2.81 (m, 4H, 2<b>CH</b><sub><b>2</b></sub>-thiophene), 2.35 (p, <i>J</i> = 7.2 Hz, 2H, CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 162.94, 157.67, 154.27, 152.97, 144.47, 141.69, 130.32, 124.90, 117.66, 115.14, 28.41, 27.66, 27.50. HRMS (ESI): calcd for C<sub>15</sub>H<sub>12</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 307.0512; found, 307.0516.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 2-(3,5-Dimethylphenyl)-3,5,6,7-tetrahydro-4<i>H</i>-cyclopenta[4,5]thieno[2,3-<i>d</i>]pyrimidin-4-one (<b>4c</b>)</h4><div class="NLM_p last">Compound (<b>4c</b>) was synthesized from (<b>2a</b>) and 3,5-dimethylbenzonitrile (<b>3c</b>), in a manner similar to (<b>4a</b>). Yield 42%, off-white powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.86 (s, 2H, Ar-<b>H</b>), 7.02 (s, 1H, Ar-<b>H</b>), 2.92–2.81 (m, 4H, 2<b>CH</b><sub><b>2</b></sub>-thiophene), 2.35 (p, <i>J</i> = 7.2 Hz, 2H, CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.32 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.53, 155.29, 152.76, 144.91, 141.73, 141.64, 138.32, 124.92, 122.86, 119.05, 28.36, 27.57, 27.40, 19.13. HRMS (ESI): calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 319.0876; found, 319.0880.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-(4-Hydroxy-3,5-dimethylphenyl)-3,5,6,7-tetrahydro-4<i>H</i>-cyclopenta[4,5]thieno[2,3-<i>d</i>]pyrimidin-4-one (<b>4d</b>)</h4><div class="NLM_p last">Compound (<b>2d</b>) was synthesized from (<b>2a</b>) and 4-hydroxy-3,5-dimethylbenzonitrile (<b>3d</b>), in a manner similar to (<b>4a</b>). Yield 45%, off-white powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.82 (s, 2H, Ar-<b>H</b>), 2.94–2.84 (m, 4H, 2<b>CH</b><sub><b>2</b></sub>-thiophene), 2.36 (p, <i>J</i> = 7.2 Hz, 2H, CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.21 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 160.44, 154.40, 153.13, 144.09, 141.68, 128.53, 127.31, 124.94, 118.43, 114.15, 28.43, 27.69, 27.56, 14.10. HRMS (ESI): calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 335.0825; found, 335.0828.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-Phenyl-5,6,7,8-tetrahydrobenzo[4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4e</b>)</h4><div class="NLM_p last">Compound (<b>4e</b>) was synthesized from (<b>2b</b>) and cyanobenzene (<b>3a</b>), in a manner similar to (<b>4a</b>). Yield 55%, off-white powder. mp 279.6–282.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.31–8.22 (m, 2H, Ar-<b>H</b>), 7.43–7.29 (m, 3H, Ar-<b>H</b>), 2.91 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 2.68 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 1.85–1.68 (m, 4H, CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub><b>–CH</b><sub><b>2</b></sub><b>–</b>CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.46, 155.20, 148.63, 139.78, 136.51, 133.05, 130.38, 127.52, 123.05, 122.25, 24.36, 24.23, 21.59, 20.64. HRMS (ESI): calcd for C<sub>16</sub>H<sub>14</sub>N<sub>2</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 305.0719; found, 305.0725.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 2-(4-Hydroxyphenyl)-5,6,7,8-tetrahydrobenzo[4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4f</b>)</h4><div class="NLM_p last">Compound (<b>4f</b>) was synthesized from (<b>2b</b>) and 4-hydroxybenzonitrile (<b>3b</b>), in a manner similar to (<b>4a</b>). Yield 45%, dark gray powder. mp 270.2–272.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06–7.99 (m, 2H, Ar-<b>H</b>), 6.85–6.78 (m, 2H, Ar-<b>H</b>), 2.90 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 2.76–2.67 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 1.90–1.65 (m, 4H, 4H, CH<sub>2</sub><b>–CH</b><sub><b>2</b></sub><b>–CH</b><sub><b>2</b></sub><b>–</b>CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 162.95, 157.67, 154.24, 148.88, 138.97, 132.92, 130.30, 121.53, 117.62, 115.03, 24.40, 24.19, 21.65, 20.71. HRMS (ESI): calcd for C<sub>16</sub>H<sub>14</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 321.0668; found, 321.0675.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-(3,5-Dimethylphenyl)-5,6,7,8-tetrahydrobenzo[4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4g</b>)</h4><div class="NLM_p last">Compound (<b>4g</b>) was synthesized from (<b>2b</b>) and 3,5-dimethylbenzonitrile (<b>3c</b>), in a manner similar to (<b>4a</b>). Yield 49%, off-white powder. mp 289.5–290.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.36 (s, 1H, pyrimidine-<b>H</b>), 7.76 (s, 2H, Ar-<b>H</b>), 7.20 (s, 1H, Ar-<b>H</b>), 2.91 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 2.76 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 2.35 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.87–1.73 (m, 4H, CH<sub>2</sub><b>–CH</b><sub><b>2</b></sub><b>–CH</b><sub><b>2</b></sub><b>–</b>CH<sub>2</sub>). <sup>13</sup>C NMR (100 MH, CF<sub>3</sub>COOD): δ 157.58, 155.25, 148.68, 141.69, 139.39, 138.30, 132.88, 124.89, 122.77, 121.99, 24.29, 24.13, 21.53, 20.59, 19.12. HRMS (ESI): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 333.1032; found, 333.1036.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-(4-Hydroxy-3,5-dimethylphenyl)-5,6,7,8-tetrahydrobenzo[4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4h</b>)</h4><div class="NLM_p last">Compound (<b>4h</b>) was synthesized from (<b>2b</b>) and 4-hydroxy-3,5-dimethylbenzonitrile (<b>3d</b>), in a manner similar to (<b>4a</b>). Yield 41%, off-white powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.12 (s, 1H, pyrimidine-<b>H</b>), 8.97 (s, 1H, <b>OH</b>), 7.80 (s, 2H, Ar-<b>H</b>), 2.93–2.85 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 2.79–2.69 (t, <i>J</i> = 5.6 Hz, 2H, thiophene-<b>CH</b><sub><b>2</b></sub>), 2.22 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.88–1.69 (m, 4H, CH<sub>2</sub><b>–CH</b><sub><b>2</b></sub><b>–CH</b><sub><b>2</b></sub><b>–</b>CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 160.23, 157.80, 154.18, 148.99, 138.52, 132.77, 128.42, 127.13, 121.20, 114.00, 24.33, 24.09, 21.61, 20.65, 13.98. HRMS (ESI): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 349.0981; found, 349.0986.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-Phenyl-7-methyl-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4i</b>)</h4><div class="NLM_p last">Compound (<b>4i</b>) was synthesized from (<b>2c</b>) and cyanobenzene (<b>3a</b>), in a manner similar to (<b>4a</b>). Yield 65%, light yellow powder. mp 266.4–268.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.26–8.19 (m, 2H, Ar-H), 7.44–7.39 (m, 3H, Ar-H), 3.51 (s, 2H, N–CH<sub>2</sub>-thiophene), 2.97 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.64 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.37 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.58, 156.99, 151.47, 137.01, 130.35, 128.95, 127.82, 126.77, 122.80, 120.63, 51.83, 51.59, 42.84, 22.00. HRMS (ESI) calcd for C<sub>16</sub>H<sub>15</sub>N<sub>3</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 320.0828; found, 320.0836.</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 2-(4-Hydroxyphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4j</b>)</h4><div class="NLM_p last">Compound (<b>4j</b>) was synthesized from (<b>2c</b>) and 4-hydroxybenzonitrile (<b>3b</b>), in a manner similar to (<b>4a</b>). Yield 43%, yellow powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.09–8.00 (m, 2H, Ar-<b>H</b>), 6.84–6.76 (m, 2H, Ar-<b>H</b>), 3.51 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.95 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.64 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.36 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 163.95, 157.19, 156.35, 151.36, 130.73, 128.90, 125.92, 119.79, 117.70, 114.22, 51.73, 51.58, 42.78, 21.93. HRMS (ESI): calcd for C<sub>16</sub>H<sub>16</sub>N<sub>3</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 314.0958; found, 314.0961.</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 2-(3,5-Dimethylphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4k</b>)</h4><div class="NLM_p last">Compound (<b>4k</b>) was synthesized from (<b>2c</b>) and 3,5-dimethylbenzonitrile (<b>3c</b>), in a manner similar to (<b>4a</b>). Yield 41%, yellow powder. mp 292.3–293.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.92 (s, 2H, Ar-<b>H</b>), 6.96 (s, 1H, Ar-<b>H</b>), 3.47 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.96 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.62 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.36 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.32 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.78, 157.20, 151.22, 142.00, 139.22, 129.08, 126.81, 125.44, 122.49, 120.66, 51.82, 51.62, 42.83, 22.10, 19.19. HRMS (ESI): calcd for C<sub>18</sub>H<sub>19</sub>N<sub>3</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 348.1141; found, 348.1149.</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 2-(4-Hydroxy-3,5-dimethylphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4′,3′: 4,5]thieno [2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4l</b>)</h4><div class="NLM_p last">Compound (<b>4l</b>) was synthesized from (<b>2c</b>) and 4-hydroxy-3,5-dimethylbenzonitrile (<b>3d</b>), in a manner similar to (<b>4a</b>). Yield 47%, yellow powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (s, 2H, Ar-<b>H</b>), 3.53 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.95 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.66 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.37 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.22 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.52, 156.36, 151.50, 129.18, 129.05, 127.45, 125.95, 124.90, 119.76, 113.34, 51.93, 51.74, 42.92, 22.20, 14.08. HRMS (ESI): calcd for C<sub>18</sub>H<sub>19</sub>N<sub>3</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 364.1090; found, 364.1095.</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 7-Ethyl-2-phenyl-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4m</b>)</h4><div class="NLM_p last">Compound (<b>4m</b>) was synthesized from (<b>2d</b>) and cyanobenzene (<b>3a</b>), in a manner similar to (<b>4a</b>). Yield 61%, light yellow powder. mp 257.3–259.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.58 (br, 1H, pyrimidine-<b>H</b>), 8.20–8.13 (m, 2H, Ar-<b>H</b>), 7.54–7.45 (m, 3H, Ar-<b>H</b>), 3.61 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.96 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.72 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.55 (q, <i>J</i> = 7.2 Hz, 2H, CH<sub>3</sub>–<b>CH</b><sub><b>2</b></sub>–N), 1.09 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>–CH<sub>2</sub>–N). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 158.50, 157.77, 152.32, 137.90, 131.25, 130.17, 128.71, 127.79, 123.71, 121.49, 53.87, 50.42, 50.27, 22.86, 8.84. HRMS (ESI): calcd for C<sub>17</sub>H<sub>18</sub>N<sub>3</sub>OS<sup>+</sup> [M + H]<sup>+</sup>, 312.1165; found, 312.1173.</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 7-Ethyl-2-(4-hydroxyphenyl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4n</b>)</h4><div class="NLM_p last">Compound (<b>4n</b>) was synthesized from (<b>2d</b>) and 4-hydroxybenzonitrile (<b>3b</b>), in a manner similar to (<b>4a</b>). Yield 35%, yellow powder. mp 307.3–308.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.10–8.03 (m, 2H, Ar-<b>H</b>), 6.84–6.78 (m, 2H, Ar-<b>H</b>), 3.56 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.94 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.70 (t, <i>J</i> = 5.6 Hz, 2H. N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.58–2.52 (q, <i>J</i> = 7.2 Hz, 2H, CH<sub>3</sub>–<b>CH</b><sub><b>2</b></sub>–N), 1.09 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>–CH<sub>2</sub>–N). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 163.96, 157.22, 156.30, 151.35, 130.87, 129.32, 126.34, 119.89, 117.75, 114.27, 53.03, 49.61, 49.42, 22.07, 8.02. HRMS (ESI): calcd for C<sub>17</sub>H<sub>17</sub>N<sub>3</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 350.0934; found, 350.0938.</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 7-Ethyl-2-(3,5-dimethylphenyl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4o</b>)</h4><div class="NLM_p last">Compound (<b>4o</b>) was synthesized from (<b>2d</b>) and 3,5-dimethylbenzonitrile (<b>3c</b>), in a manner similar to (<b>4a</b>). Yield 48%, yellow powder. mp 261.7–263.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.89 (s, 2H, Ar-<b>H</b>), 6.98 (s, 1H, Ar-<b>H</b>), 3.53 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.94 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.68 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.53 (q, <i>J</i> = 7.2 Hz 2H, CH<sub>3</sub>–<b>CH</b><sub><b>2</b></sub>–N), 2.32 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.09 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>–CH<sub>2</sub>–N). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.71, 157.12, 151.20, 141.91, 139.12, 129.33, 126.87, 125.33, 122.44, 120.52, 53.07, 49.64, 49.48, 22.06, 19.11, 8.01. HRMS (ESI): calcd for C<sub>19</sub>H<sub>21</sub>N<sub>3</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 362.1298; found, 362.1304.</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 7-Ethyl-2-(4-hydroxy-3,5-dimethylphenyl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5] thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4p</b>)</h4><div class="NLM_p last">Compound (<b>4p</b>) was synthesized from (<b>2d</b>) and 4-hydroxy-3,5-dimethylbenzonitrile (<b>3d</b>), in a manner similar to (<b>4a</b>). Yield 43%, yellow powder. mp 260.5–263.7 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.83 (s, 2H, Ar-<b>H</b>), 3.57 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.93 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.70 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.58–2.52 (q, <i>J</i> = 7.2 Hz, 2H, CH<sub>3</sub>–<b>CH</b><sub><b>2</b></sub>–N), 2.21 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.09 (t, <i>J</i> = 7.2 Hz, 3H, <b>CH</b><sub><b>3</b></sub>–CH<sub>2</sub>–N). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.50, 156.30, 151.46, 129.35, 129.13, 127.41, 126.11, 127.87, 119.69, 113.31, 53.08, 49.70, 49.50, 22.16, 14.04, 8.09. HRMS (ESI): calcd for C<sub>19</sub>H<sub>21</sub>N<sub>3</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 378.1247; found, 378.1251.</div></div><div id="sec4_4_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 7-Benzyl-2-phenyl-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4q</b>)</h4><div class="NLM_p last">Compound (<b>4q</b>) was synthesized from (<b>2e</b>) and cyanobenzene (<b>3a</b>), in a manner similar to (<b>4a</b>). Yield 55%, light yellow powder. mp 253.7–254.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.31–8.20 (m, 2H, Ar-<b>H</b>), 7.43–7.23 (m, 8H, Ar-<b>H</b>), 3.69 (s, 2H, Ar-<b>CH</b><sub><b>2</b></sub>), 3.55 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.96 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.74 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.53, 157.01, 151.43, 137.02, 131.16, 130.37, 130.13, 129.56, 129.23, 127.78, 127.05, 126.01, 122.82, 120.57, 61.53, 49.59, 49.21, 22.02. HRMS (ESI): calcd for C<sub>22</sub>H<sub>19</sub>N<sub>3</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 396.1141; found, 396.1145.</div></div><div id="sec4_4_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 7-Benzyl-2-(4-hydroxyphenyl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4r</b>)</h4><div class="NLM_p last">Compound (<b>4r</b>) was synthesized from (<b>2e</b>) and 4-hydroxybenzonitrile (<b>3b</b>), in a manner similar to (<b>4a</b>). Yield 49%, yellow powder. mp 259.1–262.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.15–8.05 (m, 2H, Ar-<b>H</b>), 7.40–7.31 (m, 4H, Ar-<b>H</b>), 7.30–7.24 (m, 1H, Ar-<b>H</b>), 6.80–6.71 (m, 2H, Ar-<b>H</b>), 3.68 (s, 2H, Ar-<b>CH</b><sub><b>2</b></sub>), 3.53 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.96 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.73 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 164.02, 157.30, 156.31, 151.37, 131.27, 130.93, 130.38, 129.66, 129.37, 126.60, 126.18, 119.96, 117.82, 114.35, 61.62, 49.68, 49.32, 22.23.HRMS (ESI): calcd for C<sub>22</sub>H<sub>19</sub>N<sub>3</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 412.1090; found, 412.1098.</div></div><div id="sec4_4_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 7-Benzyl-2-(3,5-dimethylphenyl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4s</b>)</h4><div class="NLM_p last">Compound (<b>4s</b>) was synthesized from (<b>2e</b>) and 3,5-dimethylbenzonitrile (<b>3c</b>), in a manner similar to (<b>4a</b>). Yield 42%, yellow powder. mp 265.8–267.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.87 (s, 2H, Ar-<b>H</b>), 7.39–7.32 (m, 4H, Ar-<b>H</b>), 7.30–7.25 (m, 1H, Ar-<b>H</b>), 6.99 (s, 1H, Ar-<b>H</b>), 3.68 (s, 2H, Ar-<b>CH</b><sub><b>2</b></sub>), 3.53 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.94 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.73 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.31 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.63, 157.11, 151.14, 141.85, 139.05, 131.12, 130.11, 129.53, 129.22, 126.86, 126.02, 125.22, 122.40, 120.41, 61.50, 49.57, 49.19, 22.00, 19.07. HRMS (ESI): calcd for C<sub>24</sub>H<sub>23</sub>N<sub>3</sub>NaOS<sup>+</sup> [M + Na]<sup>+</sup>, 424.1454; found, 424.1458.</div></div><div id="sec4_4_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 7-Benzyl-2-(4-hydroxy-3,5-dimethylphenyl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5] thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>4t</b>)</h4><div class="NLM_p last">Compound (<b>4t</b>) was synthesized from (<b>2e</b>) and 4-hydroxy-3,5-dimethylbenzonitrile (<b>3d</b>), in a manner similar to (<b>4a</b>). Yield 46%, yellow powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.19 (s, 1H, pyrimidine-<b>H</b>), 8.99 (s, 1H, <b>OH</b>), 7.81 (s, 2H, Ar-<b>H</b>), 7.40–7.32 (m, 4H, Ar-<b>H</b>), 7.31–7.25 (m, 1H, Ar-<b>H</b>), 3.72 (s, 2H, Ar-<b>CH</b><sub><b>2</b></sub>), 3.61 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.95 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.79 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.22 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 157.47, 156.31, 151.45, 131.26, 130.26, 129.67, 129.28, 129.02, 127.43, 126.18, 126.12, 126.07, 119.57, 113.32, 61.65, 49.74, 49.34, 22.16, 14.00. HRMS (ESI): calcd for C<sub>24</sub>H<sub>23</sub>N<sub>3</sub>NaO<sub>2</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 440.1403; found, 440.1408.</div></div><div id="sec4_4_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>tert</i>-Butyl 2-Amino-3-carbamoyl-4,7-dihydrothieno[2,3-<i>c</i>]pyridine-6(5<i>H</i>)-carboxylate (<b>5</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-oxopiperidine-1-carboxylate (3.98 g, 20.0 mmol), 2-cyanoacetamide (1.68 g, 20.0 mmol), and sulfur (641 mg, 20.0 mmol) in absolute ethanol (25 mL) and triethylamine (4 mL) was added, and then the mixture was refluxed for 12 h; the reaction mixture was concentrated, and the residue was washed with cool methanol (20 mL) to give <b>5</b>. Yield 82%, white powder. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 6.17 (s, 2H, CO<b>NH</b><sub><b>2</b></sub>), 5.36 (s, 2H, thiophene-<b>NH</b><sub><b>2</b></sub>), 4.38 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.68 (t, <i>J</i> = 5.2 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.74 (t, <i>J</i> = 5.2 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.48 (s, 9H, Boc-<b>H</b>).</div></div><div id="sec4_4_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>tert</i>-Butyl 3-Carbamoyl-2-(3,5-dimethylisoxazole-4-carboxamido)-4,7-dihydrothieno[2,3-<i>c</i>]pyridine-6(5<i>H</i>)-carboxylate (<b>6</b>)</h4><div class="NLM_p last">Compound <b>5</b> (594 mg, 2 mmol) and pyridine (193 μL, 2.4 mmol) was dissolved in 1,4-dioxane (10 mL), and then 3,5-dimethylisoxazole-4-carbonyl chloride (381 mg, 2.4 mmol) was added into the above solution. The mixture was stirred for 3 h at room temperature. After the reaction was complete, the mixture were removed in vacuo, and the residue was washed with methanol (8 mL) to give <b>6</b>. Yield 86.9%, off-white powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.22 (s, 1H, CO<b>NH</b>), 7.40 (br, 2H, CO<b>NH</b><sub><b>2</b></sub>), 4.49 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.57 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.83 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.70 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.47 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 1.43 (s, 9H, Boc-<b>H</b>).</div></div><div id="sec4_4_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-(3-Carbamoyl-4,5,6,7-tetrahydrothieno[2,3-<i>c</i>]pyridin-2-yl)-3,5-dimethylisoxazole-4-carboxamide (<b>7</b>)</h4><div class="NLM_p last">Compound <b>6</b> (2.10 g, 5 mL mmol) was added into 4 M HC1/dioxane (10 mL, 40 mmol) at 0 °C, and the reaction mixture was stirred for 4 h at room temperature. The reaction solution was added with appropriate water and neutralized with NaHCO<sub>3</sub>. The precipitated product was collected by vacuum filtration and washed with additional water (30 mL) and dichloromethane (20 mL) to give <b>7</b>. Yield 83.3%, yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.13 (br, 2H, CO<b>NH</b><sub><b>2</b></sub>), 4.00 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.16 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.97 (s, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.63 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.41 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_4_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-(3,5-Dimethylisoxazol-4-yl)-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>8</b>)</h4><div class="NLM_p last">Compound <b>7</b> (300 mg, 0.94 mmol) and NaOH (600 mg, 15 mmol) was dissolved in absolute ethanol (5 mL). The mixture was stirred at 100 °C for 24 h. The reaction mixture was concentrated, and the residue was purified by silica gel column chromatography using a mixture of dichloromethane:methanol (15:1, v/v) to give <b>8</b>. Yield 57.2%, brown powder. mp 255.9–257.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 3.90 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.97 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.87 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.54 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.34 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.27, 163.43, 159.71, 158.92, 147.21, 132.12, 130.21, 121.30, 111.77, 44.50, 42.65, 26.92, 12.55, 10.95. HRMS (ESI): calcd for C<sub>14</sub>H<sub>15</sub>N<sub>4</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 303.0910; found, 303.0912.</div></div><div id="sec4_4_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-(3,5-Dimethylisoxazol-4-yl)-7-methyl-5,6,7,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-4(3<i>H</i>)-one (<b>9a</b>)</h4><div class="NLM_p last">Compound <b>8</b> (90 mg, 0.30 mmol) and K<sub>2</sub>CO<sub>3</sub> (83 mg, 0.60 mmol) were suspended in 1,4-dioxane (5 mL), and then iodomethane (42 mg, 0.30 mmol) was added into the suspension. The mixture was stirred for 4 h at room temperature. The suspension was concentrated, and the residue was purified by thin layer chromatography using a mixture of dichloromethane:methanol (20:1) to give <b>9a</b>. Yield 26.5%, yellow solid. mp 227.5–230.2 °C. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 11.93 (s, 1H, pyrimidine-<b>H</b>), 3.66 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.06 (t, <i>J</i> = 5.8 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.78 (t, <i>J</i> = 5.8 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.67 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.52 (s, 3H, N<b>CH</b><sub><b>3</b></sub>), 2.50 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 170.39, 164.89, 160.40, 158.85, 158.44, 145.66, 139.66, 131.54, 129.29, 120.49, 110.83, 53.73, 51.88, 45.55, 25.89, 12.69, 11.08. HRMS (ESI): calcd for C<sub>15</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 317.1067; found, 317.1062.</div></div><div id="sec4_4_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Methyl 2-(2-(3,5-Dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)acetate (<b>9b</b>)</h4><div class="NLM_p last">Compound <b>8</b> (90 mg, 0.30 mmol) and methyl 2-bromoacetate (55 mg, 0.36 mmol) were dissolved in 1,4-dioxane (5 mL), and then K<sub>2</sub>CO<sub>3</sub> (83 mg, 0.60 mmol) was added into the solution; the mixture was refluxed for 6 h. The reaction mixture was concentrated, and the residue was purified by silica gel column chromatography using a mixture of dichloromethane:methanol (80:1) to give <b>9b</b>. Yield 47.6%, yellow solid. mp 174.5–180.2 °C. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 11.17 (s, 1H, pyrimidine-<b>H</b>), 3.91 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.77 (s, 3H, O<b>Me</b>), 3.50 (s, 2H, MeCOO<b>CH</b><sub><b>2</b></sub>N), 3.09 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 3.00 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.68 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.50 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 170.69, 170.47, 164.55, 159.85, 158.29, 145.61, 131.01, 129.38, 119.98, 110.67, 58.04, 51.90, 51.13, 49.68, 25.55, 12.79, 11.20. HRMS (ESI) calcd for C<sub>17</sub>H<sub>19</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 375.1122; found, 375.1118.</div></div><div id="sec4_4_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Methyl 4-(2-(3,5-dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)butanoate (<b>9c</b>)</h4><div class="NLM_p last">Compound <b>9c</b> was synthesized from compound <b>8</b> and methyl 4-bromobutanoate, in a manner similar to <b>9b</b>. Yield 28.5%, yellow solid. mp 57.8–60.2 °C. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 11.78 (s, 1H, pyrimidine-<b>H</b>), 3.70 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.67 (s, 3H, O<b>Me</b>), 3.02 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.83 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.67 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.61 (t, <i>J</i> = 7.2 Hz, 2H, MeCOO<b>CH</b><sub><b>2</b></sub>), 2.50 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.45–2.39 (m, 2H, MeCOOCH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.98–1.89 (m, 2H, MeCOOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 173.94, 170.39, 164.87, 160.29, 158.42, 145.58, 131.78, 129.70, 120.50, 110.81, 56.69, 51.76, 51.58, 49.92, 31.86, 25.77, 22.56, 12.71, 11.09. HRMS (ESI) calcd for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 403.1435; found, 403.1435.</div></div><div id="sec4_4_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Methyl 6-(2-(3,5-Dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)hexanoate (<b>9d</b>)</h4><div class="NLM_p last">Compound <b>9d</b> was synthesized from compound <b>8</b> and 6-bromohexanoate, in a manner similar to <b>9b</b>. Yield 43.2%, yellow solid. mp 121.5–123.8 °C. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 11.99 (s, 1H, pyrimidine-<b>H</b>), 3.73 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.68 (s, 3H, O<b>Me</b>), 3.05 (t, <i>J</i> = 5.8 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.86 (t, <i>J</i> = 5.8 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.67 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.61 (t, <i>J</i> = 7.6 Hz, 2H, MeCOO<b>CH</b><sub><b>2</b></sub>), 2.49 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.35 (t, <i>J</i> = 7.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>–CH<sub>2</sub>), 1.73–1.61 (m, 4H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.44–1.37 (m, 2H, N–CH<sub>2</sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 174.10, 170.42, 165.00, 160.39, 158.46, 158.45, 145.73, 129.61, 120.44, 110.83, 57.40, 51.81, 51.52, 49.88, 33.97, 29.70, 26.98, 26.85, 24.81, 12.70, 11.07. HRMS (ESI): calcd for C<sub>21</sub>H<sub>27</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 431.1748; found, 431.1748.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> General Procedures of Method for the Synthesis of <b>16a–c</b> and <b>17a–f</b></h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Methyl 3-((4-Cyano-2,6-dimethylphenoxy)methyl)benzoate (<b>12a</b>)</h4><div class="NLM_p last">Compound (<b>12a</b>) was synthesized from 4-hydroxy-3,5-dimethylbenzonitrile (<b>10</b>) and methyl 3-(bromomethyl) benzoate (<b>11a</b>), in a manner similar to (<b>12b</b>). Yield 40%, white powder.mp: 68.6–69.7 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.09 (t, <i>J</i> = 1.6 Hz, 1H, Ar-<b>H</b>), 7.97 (dt, <i>J</i> = 7.8, 1.6 Hz, 1H, Ar-<b>H</b>), 7.78 (dt, <i>J</i> = 7.8, 1.6 Hz, 1H, Ar-<b>H</b>), 7.59 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 7.58 (s, 2H, Ar-<b>H</b>), 4.98 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 3.88 (s, 3H, O<b>CH</b><sub><b>3</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 166.50, 159.68, 138.20, 133.22 (5C), 130.29, 129.50, 129.32, 128.91, 119.31, 107.04, 73.29, 52.71, 16.35. HRMS (ESI): calcd for C<sub>18</sub>H<sub>17</sub>NNaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 318.1101; found, 318.1104.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Methyl 4-((4-Cyano-2,6-dimethylphenoxy)methyl)benzoate (<b>12b</b>)</h4><div class="NLM_p last">Compound (<b>12b</b>) was synthesized from 4-hydroxy-3,5-dimethylbenzonitrile (<b>10</b>) and methyl 4-(bromomethyl) benzoate (<b>11b</b>), in a manner similar to (<b>12d</b>), but the crude product was recrystallized by a mixture solvent of ethyl acetate and ethanol (50 mL, v/v = 1:1). Yield 53%, white crystal. mp 118.4–120.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.02 (d, <i>J</i> = 8.4 Hz, 2H, Ar-<b>H</b>), 7.64 (d, <i>J</i> = 8.0 Hz, 2H, Ar-<b>H</b>), 7.58 (s, 2H, Ar-<b>H</b>), 4.98 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 3.88 (s, 3H, O<b>CH</b><sub><b>3</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 166.44, 159.65, 142.76, 133.21 (4C), 129.80, 129.69, 128.35, 119.28, 107.09, 73.23, 52.62, 16.32. HRMS (ESI): calcd for C<sub>18</sub>H<sub>17</sub>NNaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 318.1101; found, 318.1107.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Methyl 6-(4-Cyano-2,6-dimethylphenoxy)hexanoate (<b>17c</b>)</h4><div class="NLM_p last">Compound (<b>17c</b>) was synthesized from4-hydroxy-3,5-dimethylbenzonitrile (<b>10</b>) and methyl 6-bromohexanoate (<b>11c</b>), in a manner similar to (<b>12d</b>). Yield 68%, light yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.53 (s, 2H, Ar-<b>H</b>), 3.79 (t, <i>J</i> = 6.4 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 3.59 (s, 3H, O<b>CH</b><sub><b>3</b></sub>), 2.34 (t, <i>J</i> = 7.3 Hz, 2H, <b>CH</b><sub><b>2</b></sub>COOCH<sub>3</sub>), 2.24 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.74 (p, <i>J</i> = 6.6 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.61 (p, <i>J</i> = 7.2 Hz, 2H, C═OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.52–1.42 (m, 2H, OCH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>).</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Methyl 4-(4-Cyano-2,6-dimethylphenoxy)butanoate (<b>12d</b>)</h4><div class="NLM_p last">4-Hydroxy-3,5-dimethyl benzonitrile (<b>10</b>) (1.47 g, 10 mmol) and methyl 4-bromobutanoate (<b>11d</b>) (2.17 g, 12 mmol) were dissolved in THF (100 mL), and then DBU (3.04 g, 20 mmol) was added into the mixed solution; the mixture was refluxed for 24 h. The reaction mixture was concentrated, and the residue was partitioned between water and ethyl acetate. The organic layer was separated and concentrated, and the crude product was purified by silica gel column chromatography using a mixture solvent of petroleum ether/ethylacetate (12:1) to give <b>12d</b>. Yield 70%, light yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.53 (s, 2H, Ar-<b>H</b>), 3.82 (t, <i>J</i> = 6.4 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>), 3.62 (s, 3H, O<b>CH</b><sub><b>3</b></sub>), 2.55 (t, <i>J</i> = 7.4 Hz, 2H, <b>CH</b><sub><b>2</b></sub>COOCH<sub>3</sub>), 2.23 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 2.01 (p, <i>J</i> = 6.8 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>).</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Methyl (<i>E</i>)-3-(4-((4-Cyano-2,6-dimethylphenoxy) methyl)phenyl)acrylate (<b>12e</b>)</h4><div class="NLM_p last">Compound (<b>12e</b>) was synthesized from 4-hydroxy-3,5-dimethylbenzonitrile (<b>10</b>) and methyl (<i>E</i>)-3-(4-(bromomethyl) phenyl)acrylate (<b>11e</b>), in a manner similar to (<b>12b</b>). Yield 47%, white powder. mp 152.3–153.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.77 (d, <i>J</i> = 8.2 Hz, 2H, Ar-<b>H</b>), 7.69 (d, <i>J</i> = 16.0 Hz, 1H, <b>CH</b>═CH–C═O), 7.58 (s, 2H, Ar-<b>H</b>), 7.53 (d, <i>J</i> = 8.0 Hz, 2H, Ar-<b>H</b>), 6.68 (d, <i>J</i> = 16.0 Hz, 1H, CH═<b>CH</b>–C═O), 4.92 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 3.74 (s, 3H, O<b>CH</b><sub><b>3</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.10, 159.77, 144.54, 139.75, 134.28, 133.23, 133.20, 128.95, 128.91, 119.32, 118.53, 106.97, 73.55, 51.97, 16.38. HRMS (ESI): calcd for C<sub>20</sub>H<sub>19</sub>NNaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 344.1257; found, 344.1261.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Methyl (<i>E</i>)-3-(4-((2-(4-Cyano-2,6-dimethylphenoxy)ethoxy)methyl)phenyl)acrylate (<b>12f</b>)</h4><div class="NLM_p last">Compound (<b>12f</b>) was synthesized from 4-hydroxy-3,5-dimethylbenzonitrile (<b>10</b>) and methyl (<i>E</i>)-3-(4-((2-bromoethoxy) methyl) phenyl)acrylate (<b>11f</b>), in a manner similar to (<b>12b</b>). Yield 42%, white powder. mp 75.1–77.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.71 (d, <i>J</i> = 8.0 Hz, 2H, Ar-<b>H</b>), 7.66 (d, <i>J</i> = 16.0 Hz, 1H, Ar-<b>CH</b>═CH), 7.53 (s, 2H, Ar-<b>H</b>), 7.38 (d, <i>J</i> = 7.9 Hz, 2H, Ar-<b>H</b>), 6.64 (d, <i>J</i> = 16.0 Hz, 1H, Ar-CH═<b>CH</b>), 4.60 (s, 2H, O<b>CH</b><sub><b>2</b></sub>Ar), 4.07–3.99 (m, 2H, ArO<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 3.81–3.74 (m, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 3.73 (s, 3H, O<b>CH</b><sub><b>3</b></sub>), 2.26 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.13, 160.03, 144.70, 141.42, 133.60, 133.08, 132.99, 128.79, 128.13, 119.36, 118.06, 106.62, 72.09, 72.07, 69.71, 51.91, 16.22. HRMS (ESI): calcd for C<sub>22</sub>H<sub>23</sub>NNaO<sub>4</sub><sup>+</sup> [M + Na]<sup>+</sup>, 388.1519; found, 388.1522.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 3-((4-Cyano-2,6-dimethylphenoxy)methyl)benzoicacid (<b>13a</b>)</h4><div class="NLM_p last">Compound (<b>13a</b>) was synthesized from (<b>12a</b>), in a manner similar to (<b>13b</b>). Yield 72%, white powder. mp 162.6–164.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.07 (s, 1H, <b>COOH</b>), 8.07 (t, <i>J</i> = 1.6 Hz, 1H, Ar-<b>H</b>), 7.95 (dt, <i>J</i> = 7.8, 1.4 Hz, 1H, Ar-<b>H</b>), 7.75 (dt, <i>J</i> = 7.8, 1.4 Hz, 1H, Ar-<b>H</b>), 7.58 (s, 2H, Ar-<b>H</b>), 7.56 (t, <i>J</i> = 7.8 Hz, 1H, Ar-<b>H</b>), 4.96 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 2.28 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.57, 159.70, 137.93, 133.24, 133.21, 132.85, 131.45, 129.48, 129.29, 129.20, 119.32, 107.01, 73.43, 16.35. HRMS (ESI): calcd for C<sub>17</sub>H<sub>15</sub>NNaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 304.0944; found, 304.0949.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-((4-Cyano-2,6-dimethylphenoxy)methyl)benzoic Acid (<b>13b</b>)</h4><div class="NLM_p last">To a solution of <b>12b</b> (1.48 g, 5 mmol) in CH<sub>3</sub>OH (30 mL) and 1.4-dioxane (30 mL) was added 1 M NaOH (30 mL), and the reaction mixture was stirred for 30 min at room temperature. At the end of the reaction, the resulting mixture was acidized by hydrochloric acid to pH = 3; the white precipitate was filtered and washed with CH<sub>3</sub>OH (10 mL) to give (<b>13b</b>). Yield 77%, white powder. mp 208.4–209.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.00 (s, 1H, COOH), 8.00 (d, <i>J</i> = 8.2 Hz, 2H, Ar-<b>H</b>), 7.62 (d, <i>J</i> = 8.2 Hz, 2H, Ar-<b>H</b>), 7.58 (s, 2H, Ar-<b>H</b>), 4.97 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.53, 159.71, 142.27, 133.23 (4C), 130.88, 129.95, 128.27, 119.31, 107.05, 73.34, 16.34. HRMS (ESI): calcd for C<sub>17</sub>H<sub>16</sub>NO<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>, 282.1125; found 282.1132.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 6-(4-Cyano-2,6-dimethylphenoxy)Hexanoic Acid (<b>13c</b>)</h4><div class="NLM_p last">Compound (<b>13c</b>) was synthesized from (<b>17c</b>), in a manner similar to (<b>13d</b>). Yield 80%, white powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.00 (s, 1H, <b>COOH</b>), 7.53 (s, 2H, Ar-<b>H</b>), 3.79 (t, <i>J</i> = 6.4 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 2.24 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 2.24 (t, <i>J</i> = 7.2 Hz, 2H, HOOC<b>CH</b><sub><b>2</b></sub>), 1.74 (p, <i>J</i> = 6.6 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.58 (p, <i>J</i> = 7.2 Hz, 2H, C═OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.52–1.43 (m, 2H, OCH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>).</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 4-(4-Cyano-2,6-dimethylphenoxy)butanoic Acid (<b>13d</b>)</h4><div class="NLM_p last">To a solution of <b>12d</b> (1.24 g, 5 mmol) in CH<sub>3</sub>OH (30 mL) was added 1 M NaOH (30 mL), and the reaction mixture was stirred for 30 min at room temperature. At the end of the reaction, the resulting mixture was acidized by hydrochloric acid to pH = 3; the white precipitate was filtered and wash with CH<sub>3</sub>OH to give (<b>13d</b>). Yield 76%, white powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.19 (br s, 1H, <b>COOH</b>), 7.54 (s, 2H, Ar-<b>H</b>), 3.81 (t, <i>J</i> = 6.4 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>), 2.45 (t, <i>J</i> = 7.2 Hz, 2H, <b>CH</b><sub><b>2</b></sub>COOH), 2.24 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.97 (p, <i>J</i> = 6.8 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>).</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (<i>E</i>)-3-(4-((4-Cyano-2,6-dimethylphenoxy)methyl)phenyl)acrylic Acid (<b>13e</b>)</h4><div class="NLM_p last">Compound <b>13e</b> was synthesized from <b>12e</b>, in a manner similar to <b>13c</b>. Yield 70%, white powder. mp 201.3–203.4 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.43 (s, 1H, <b>COOH</b>), 7.74 (d, <i>J</i> = 8.2 Hz, 2H, Ar-<b>H</b>), 7.62 (d, <i>J</i> = 16.0 Hz, 1H, <b>CH</b>═CH–C═O), 7.58 (s, 2H, Ar-<b>H</b>), 7.53 (d, <i>J</i> = 8.0 Hz, 2H, Ar-<b>H</b>), 6.56 (d, <i>J</i> = 16.0 Hz, 1H, CH═<b>CH</b>–C═O), 4.91 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.99, 159.78, 143.91, 139.45, 134.52, 133.24, 133.20, 128.93, 128.78, 119.95, 119.33, 106.96, 73.59, 16.39. HRMS (ESI): calcd for C<sub>19</sub>H<sub>17</sub>NNaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 330.1101; found, 330.1108.</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>E</i>)-3-(4-((2-(4-Cyano-2,6-dimethylphenoxy)ethoxy)methyl)phenyl)acrylic Acid (<b>13f</b>)</h4><div class="NLM_p last">Compound <b>13f</b> was synthesized from <b>12f</b>, in a manner similar to <b>13c</b>. Yield 68%, white powder. mp 183.2–185.4 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ7.62–7.53 (m, 4H, Ar-<b>H</b>), 7.43 (d, <i>J</i> = 15.8 Hz, 1H, Ar-<b>CH</b>═CH), 7.36 (d, <i>J</i> = 7.8 Hz, 2H, Ar-<b>H</b>), 6.48 (d, <i>J</i> = 15.8 Hz, 1H, Ar-CH═<b>CH</b>), 4.60 (s, 2H, O<b>CH</b><sub><b>2</b></sub>Ar), 4.07–4.01 (m, 2H, ArO<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 3.83–3.76 (m, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 2.25 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.25, 161.01, 140.45, 138.25, 135.01, 133.05, 132.81, 128.50, 127.76, 119.57, 119.28, 106.79, 72.10, 72.20, 69.45, 16.25. HRMS (ESI): calcd for C<sub>21</sub>H<sub>21</sub>NNaO<sub>4</sub><sup>+</sup> [M + Na]<sup>+</sup>, 374.1363; found, 374.1369.</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-(2-Aminophenyl)-3-((4-cyano-2,6-dimethylphenoxy) methyl)benzamide (<b>14a</b>)</h4><div class="NLM_p last">Compound <b>14a</b> was synthesized from <b>8a</b> and benzene-1,2-diamine, in a manner similar to <b>14b</b>. Yield 65%, white powder. mp 150.9–152.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.74 (s, 1H, <b>NH</b>), 8.11 (s, 1H, Ar-<b>H</b>), 8.01 (d, <i>J</i> = 7.8 Hz, 1H, Ar-<b>H</b>), 7.70 (d, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 7.59 (s, 2H, Ar-<b>H</b>), 7.58 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 7.18 (d, <i>J</i> = 7.8 Hz, 1H, Ar-<b>H</b>), 6.99 (td, <i>J</i> = 7.6, 1.5 Hz, 1H, Ar-<b>H</b>), 6.80 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H, Ar-<b>H</b>), 6.61 (td, <i>J</i> = 7.6, 1.4 Hz, 1H, Ar-<b>H</b>), 4.97 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 4.91 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 2.31 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.56, 159.79, 143.68, 137.63, 135.34, 133.28, 133.23 (4C), 131.39, 128.94, 127.95, 127.23, 127.03, 123.70, 119.34, 116.73, 116.60, 107.02, 73.70, 16.42. HRMS (ESI): calcd for C<sub>23</sub>H<sub>21</sub>N<sub>3</sub>NaO<sub>2</sub><sup>+</sup> [M + Na]<sup>+</sup>, 394.1526; found, 394.1532.</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-(2-Aminophenyl)-4-((4-cyano-2,6-dimethylphenoxy)methyl)benzamide (<b>14b</b>)</h4><div class="NLM_p last">To a solution of <b>13b</b> (281 mg, 1 mmol) and benzene-1,2-diamine (432 mg, 4 mmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (15 mL) was added EDCI (192 mg, 1 mmol) and HOBt (135 mg, 1 mmol), and the reaction mixture was stirred for 24 h at room temperature. The reaction mixture was partitioned between water and dichloromethane, the organic layer was separated and concentrated, and the crude product was purified by silica gel column chromatography using a mixture solvent of petroleum ether/ethylacetate (5:1) to give 14b. Yield 67%, white powder. mp 194.3–195.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.69 (br s, 1H, <b>NH</b>), 8.03 (d, <i>J</i> = 7.8 Hz, 2H, Ar-<b>H</b>), 7.62 (d, <i>J</i> = 8.2 Hz, 2H, Ar-<b>H</b>), 7.59 (s, 2H, Ar-<b>H</b>), 7.18 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H, Ar-<b>H</b>), 6.79 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H, Ar-<b>H</b>), 6.61 (td, <i>J</i> = 7.2, 1.4 Hz, 1H, Ar-<b>H</b>), 4.98 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 4.91 (br s, 2H, <b>NH2</b>), 2.29 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.47, 159.82, 143.64, 140.69, 134.80, 133.23 (4C), 128.44, 128.18, 127.19, 126.99, 123.71, 119.34, 116.71, 116.59, 106.99, 73.53, 16.43. HRMS (ESI): calcd for C<sub>23</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 372.1707; found, 372.1711.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-(2-Aminophenyl)-6-(4-cyano-2,6-dimethylphenoxy)hexanamide (<b>14c</b>)</h4><div class="NLM_p last">Compound <b>14c</b> was synthesized from <b>13c</b> and benzene-1,2-diamine, in a manner similar to <b>14b</b>. Yield 75%, white powder. mp 111.2–112.5 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.10 (s, 1H, <b>NH</b>), 7.53 (s, 2H, Ar-<b>H</b>), 7.15 (dd, <i>J</i> = 7.9, 1.6 Hz, 1H, Ar-<b>H</b>), 6.89 (td, <i>J</i> = 7.6, 1.6 Hz, 1H, Ar-<b>H</b>), 6.71 (dd, <i>J</i> = 8.0, 1.4 Hz, 1H, Ar-<b>H</b>), 6.53 (td, <i>J</i> = 7.6, 1.4 Hz, 1H, Ar-<b>H</b>), 4.81 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 3.81 (t, <i>J</i> = 6.4 Hz, 2H, ArO<b>CH</b><sub><b>2</b></sub>), 2.36 (t, <i>J</i> = 7.2 Hz, 2H, C═O–<b>CH</b><sub><b>2</b></sub>), 2.25 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.78 (p, <i>J</i> = 6.5 Hz, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.68 (p, <i>J</i> = 7.4 Hz, 2H, C═O–CH<sub>2</sub><b>CH2</b>), 1.57–1.48 (m, 2H, ArOCH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.50, 160.23, 142.35, 133.12, 132.98, 126.15, 125.73, 124.01, 119.39, 116.61, 116.33, 106.52, 72.45, 36.15, 30.11, 25.67, 25.59, 16.22. HRMS (ESI): calcd for C<sub>21</sub>H<sub>26</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 352.2020; found, 352.2023.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 3-((4-Cyano-2,6-dimethylphenoxy)methyl)-<i>N</i>-hydroxybenzamide (<b>15a</b>)</h4><div class="NLM_p last">Compound <b>15a</b> was synthesized from <b>13a</b> and hydroxylamine hydrochloride, in a manner similar to <b>15d</b>. Yield 56%, off-white powder mp 171.0–173.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.29 (s, 1H, OH), 9.07 (s, 1H, <b>NH</b>), 7.90 (t, <i>J</i> = 1.7 Hz, 1H, Ar-<b>H</b>), 7.75 (dt, <i>J</i> = 7.6, 1.4 Hz, 1H, Ar-<b>H</b>), 7.64 (dt, <i>J</i> = 7.6, 1.4 Hz, 1H, Ar-<b>H</b>), 7.59 (s, 2H, Ar-<b>H</b>), 7.50 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 4.92 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 2.28 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.40, 159.72, 137.69, 133.47, 133.27, 133.22, 131.22, 129.02, 127.17, 126.97, 119.33, 106.99, 73.64, 16.37. HRMS (ESI): calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 319.1053; found, 319.1061.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 4-((4-Cyano-2,6-dimethylphenoxy)methyl)-<i>N</i>-hydroxybenzamide (<b>15b</b>)</h4><div class="NLM_p last">Compound <b>15b</b> was synthesized from <b>13b</b> and hydroxylamine hydrochloride, in a manner similar to <b>15d</b>. Yield 58%, off-white powder. mp 191.2–193.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.26 (s, 1H, <b>OH</b>), 9.06 (s, 1H, <b>NH</b>), 7.80 (d, <i>J</i> = 8.2 Hz, 2H, Ar-<b>H</b>), 7.60–7.54 (m, 4H, Ar-<b>H</b>), 4.93 (s, 2H, CH<sub>2</sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.38, 159.75, 140.43, 133.23, 133.21, 132.96, 128.28, 127.53, 119.32, 107.00, 73.45, 16.37. HRMS (ESI): calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 319.1053; found, 319.1058.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 6-(4-Cyano-2,6-dimethylphenoxy)-<i>N</i>-hydroxyhexanamide (<b>15c</b>)</h4><div class="NLM_p last">Compound <b>15c</b> was synthesized from <b>13c</b> and hydroxylamine hydrochloride, in a manner similar to <b>15d</b>. Yield 63%, off-white powder. mp 93.3–95.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.36 (br s, 1H, <b>OH</b>), 8.67 (br s, 1H, <b>NH</b>), 7.53 (Ar-<b>H</b>), 3.78 (t, <i>J</i> = 6.4 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 2.24 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.99 (t, <i>J</i> = 7.2 Hz, 2H, C═O–<b>CH</b><sub><b>2</b></sub>), 1.73 (p, <i>J</i> = 6.6 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.57 (p, <i>J</i> = 7.2 Hz, 2H, C═OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.39–1.49 (m, 2H, OCH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.45, 160.20, 133.11, 132.96, 119.38, 106.52, 72.38, 32.65, 30.01, 25.55, 25.40, 16.20. HRMS (ESI): calcd for C<sub>15</sub>H<sub>20</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 299.1366; found, 299.1371.</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 4-(4-Cyano-2,6-dimethylphenoxy)-<i>N</i>-hydroxybutanamide (<b>15d</b>)</h4><div class="NLM_p last">Hydroxyl-amine hydrochloride (345 mg, 5 mmol) was added to a stirred solution of sodium methoxide (270 mg, 5 mmol) in methanol (10 mL). The mixture was stirred for 30 min at the same temperature, and the precipitated sodium chloride was filtered, and the filtrate was used as such. Meanwhile, to a solution of <b>13d</b> (233 mg, 1 mmol) in 1.4-dioxane (10 mL) ethyl chloroformate (130 mg, 1.2 mmol) and <i>N</i>-methylmorpholine (130 mg, 1.3 mmol) were added, and the mixture was stirred for 15 min. The solid was filtered, and the filtrate was added to above prepared hydroxylamine (165 mg, 5 mmol) in methanol. The reaction mixture was stirred at room temperature for 20 min. The solvent was evaporated, and the residue was washed by ether and purified by silica gel column chromatography using a mixture solvent of dichloromethane:acetone (5:1) to give <b>15d</b>. Yield 60%, off-white powder. mp 137.9–138.6 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.44 (br s, 1H, <b>OH</b>), 8.75 (br s, 1H, <b>NH</b>), 7.53 (s, 2H, Ar-<b>H</b>), 3.80 (t, <i>J</i> = 6.4 Hz, 2H, O–<b>CH</b><sub><b>2</b></sub>), 2.24 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 2.20 (t, <i>J</i> = 7.6 Hz, 2H, C═O–<b>CH</b><sub><b>2</b></sub>), 1.97 (p, <i>J</i> = 6.8 Hz, 2H, CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.02, 160.04, 133.13, 132.98, 119.35, 106.63, 71.81, 29.12, 26.38, 16.21. HRMS (ESI): calcd for C<sub>13</sub>H<sub>16</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 271.1053; found, 271.1060.</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (<i>E</i>)-3-(4-((4-Cyano-2,6-dimethylphenoxy)methyl)phenyl)-<i>N</i>-hydroxyacrylamide (<b>15e</b>)</h4><div class="NLM_p last">Compound <b>15e</b> was synthesized from <b>13e</b> and hydroxylamine hydrochloride, in a manner similar to <b>15d</b>. Yield 52%, off-white powder. mp 186.2–187.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.80 (s, 1H, <b>OH</b>), 9.08 (s, 1H, <b>NH</b>), 7.67–7.43 (m, 7H, Ar-<b>H</b>, Ar-<b>CH</b>═CH), 6.51 (d, <i>J</i> = 15.8 Hz, 1H, Ar-CH═<b>CH</b>), 4.90 (s, 2H, <b>CH</b><sub><b>2</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.14, 159.78, 138.62, 138.36, 135.14, 133.24, 133.19, 129.06, 128.05, 119.85, 119.33, 106.94, 73.66, 16.39. HRMS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 345.1210; found, 345.1213.</div></div><div id="sec4_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (<i>E</i>)-3-(4-((2-(4-Cyano-2,6-dimethylphenoxy)ethoxy)methyl)phenyl)-<i>N</i>-hydroxyacrylamide (<b>15f</b>)</h4><div class="NLM_p last">Compound <b>15f</b> was synthesized from <b>13f</b> and hydroxylamine hydrochloride, in a manner similar to <b>15d</b>. Yield 49%, brown oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.75 (s, 1H, <b>OH</b>), 9.04 (s, 1H, <b>NH</b>), 7.60–7.51 (m, 4H, Ar-<b>H</b>), 7.45 (d, <i>J</i> = 15.8 Hz, 1H, Ar-<b>CH</b>═CH), 7.37 (d, <i>J</i> = 7.8 Hz, 2H, Ar-<b>H</b>), 6.46 (d, <i>J</i> = 15.8 Hz, 1H, Ar-CH═<b>CH</b>), 4.59 (s, 2H, O<b>CH</b><sub><b>2</b></sub>Ar), 4.05–4.00 (m, 2H, ArO<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 3.80–3.74 (m, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 2.26 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.20, 160.07, 140.25, 138.50, 134.45, 133.10, 133.01, 128.30, 127.90, 119.39, 119.38, 106.59, 72.16, 72.09, 69.67, 16.24. HRMS (ESI): calcd for C<sub>21</sub>H<sub>22</sub>N<sub>2</sub>NaO<sub>4</sub><sup>+</sup> [M + Na]<sup>+</sup>, 389.1472; found, 389.1478.</div></div><div id="sec4_5_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N</i>-(2-Aminophenyl)-3-((2,6-dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydro pyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)methyl)benzamide (<b>16a</b>)</h4><div class="NLM_p last">Compound <b>16a</b> was synthesized from <b>14a</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 35%, yellow powder. mp 243.7–245.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.39 (br s, 1H, pyrimidine-<b>H</b>), 9.80 (s, 1H, <b>CONH</b>), 8.15 (s, 1H, Ar-<b>H</b>), 8.02 (d, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 7.92 (s, 2H, Ar-<b>H</b>), 7.72 (d, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 7.57 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 7.19 (d, <i>J</i> = 7.8 Hz, 1H, Ar-<b>H</b>), 6.98 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 6.80 (d, <i>J</i> = 8.0 Hz, 1H, Ar-<b>H</b>), 6.61 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 4.95 (s, 4H, O–<b>CH</b><sub><b>2</b></sub>, <b>NH</b><sub><b>2</b></sub>), 3.56 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.97 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.67 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.38 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.35 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.61, 164.26, 160.45, 158.33, 153.33, 143.66, 137.97, 135.27, 131.45, 131.28, 129.40, 129.30, 128.88, 128.81, 128.58, 127.93, 127.89, 127.24, 126.93, 123.82, 120.46, 116.68, 116.65, 73.60, 53.56, 51.79, 45.56, 26.33, 16.77. HRMS (ESI): calcd for C<sub>32</sub>H<sub>32</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 566.2220; found, 566.2224.</div></div><div id="sec4_5_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>-(2-Aminophenyl)-4-((2,6-dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydro pyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)methyl)benzamide (<b>16b</b>)</h4><div class="NLM_p last">Compound (<b>16b</b>) was synthesized from <b>14b</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 38%, yellow powder. mp 239.8–241.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.38 (br s, 1H, pyrimidine-<b>H</b>), 9.71 (s, 1H, <b>CONH</b>), 8.04 (s, 2H, Ar-<b>H</b>), 7.91 (s, 2H, Ar-<b>H</b>),7.64 (s, 2H, Ar-<b>H</b>), 7.19 (s, 1H, Ar-<b>H</b>), 6.98 (s, 1H, Ar-<b>H</b>), 6.80 (s, 1H, Ar-<b>H</b>), 6.61 (s, 1H, Ar-<b>H</b>), 4.97 (s, 2H, O–<b>CH</b><sub><b>2</b></sub>), 4.92 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 3.56 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.97 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.67 (s, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.38 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.33 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.50, 164.01, 159.23, 158.57, 152.52, 143.64, 141.05, 134.69, 131.60, 130.16, 129.37, 128.81, 128.42, 128.09, 127.72, 127.19, 126.97, 123.75, 120.62, 116.71, 116.59, 73.44, 53.50, 51.73, 45.53, 26.24, 16.76. HRMS (ESI): calcd for C<sub>32</sub>H<sub>32</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 566.2220; found, 566.2224.</div></div><div id="sec4_5_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>N</i>-(2-Aminophenyl)-6-(2,6-dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydro pyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)hexanamide (<b>16c</b>)</h4><div class="NLM_p last">Compound <b>16c</b> was synthesized from <b>14c</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 37%, yellow powder. mp 222.2–223.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.34 (br s, 1H, pyrimidine-<b>H</b>), 9.13 (s, 1H, <b>CONH</b>), 7.87 (s, 2H, Ar-<b>H</b>), 7.16 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H, Ar-<b>H</b>), 6.89 (td, <i>J</i> = 7.6, 1.6 Hz, 1H, Ar-<b>H</b>), 6.71 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H, Ar-<b>H</b>), 6.54 (td, <i>J</i> = 7.6, 1.6 Hz, 1H, Ar-<b>H</b>), 4.82 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 3.80 (t, <i>J</i> = 6.4 Hz, 2H, O–<b>CH</b><sub><b>2</b></sub>), 3.55 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.95 (d, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.66 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.39–2.34 (m, 5H, N–<b>CH</b><sub><b>3</b></sub>, <b>CH</b><sub><b>2</b></sub>CONH), 2.28 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.80 (p, <i>J</i> = 6.8 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>), 1.69 (p, <i>J</i> = 7.2 Hz, 2H, OCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 1.54 (m, 1.60–1.50, 2H, OCH<sub>2</sub>CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, CF<sub>3</sub>COOD): δ 178.18, 163.31, 156.55, 150.17, 149.69, 135.10, 129.59, 129.27, 128.99, 128.86, 128.31, 126.98, 126.52, 126.35, 118.03, 114.38, 111.56, 73.12, 51.71, 51.56, 42.75, 35.50, 29.00, 25.40, 24.95, 21.93, 14.82. HRMS (ESI): calcd for C<sub>30</sub>H<sub>36</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 546.2533; found, 546.2540.</div></div><div id="sec4_5_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 3-((2,6-Dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydropyrido[4′,3′:4,5] thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)methyl)-<i>N</i>-hydroxybenzamide (<b>17a</b>)</h4><div class="NLM_p last">Compound <b>17a</b> was synthesized from <b>15a</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 41%, yellow powder. mp 184.1–186.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.39 (br s, 1H, pyrimidine-<b>H</b>), 11.29 (br s, 1H, <b>OH</b>), 9.07 (br s, 1H, C═O<b>NH</b>), 7.93 (s, 1H, Ar-<b>H</b>), 7.90 (s, 2H, Ar-<b>H</b>), 7.75 (d, <i>J</i> = 7.5 Hz, 1H, Ar-<b>H</b>), 7.65 (d, <i>J</i> = 7.4 Hz, 1H, Ar-<b>H</b>), 7.51 (t, <i>J</i> = 7.6 Hz, 1H, Ar-<b>H</b>), 4.91 (s, 2H, O<b>CH</b><sub><b>2</b></sub>), 3.58 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.96 (t, <i>J</i> = 5.7 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.68 (t, <i>J</i> = 5.7 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.39 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.32 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.02, 159.19, 158.50, 152.52, 138.03, 133.46, 132.99, 131.63, 131.12, 130.08, 129.35, 128.99, 128.81, 127.69, 127.07, 126.86, 120.60, 73.57, 53.46, 51.70, 45.48, 26.20, 16.72. HRMS (ESI): calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 491.1748; found, 491.1752.</div></div><div id="sec4_5_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 4-((2,6-Dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydropyrido [4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)methyl)-<i>N</i>-hydroxybenzamide (<b>17b</b>)</h4><div class="NLM_p last">Compound <b>17b</b> was synthesized from <b>15b</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 43%, yellow powder. mp 213.0–215.3 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.93 (s, 2H, Ar-<b>H</b>), 7.81 (s, 2H, Ar-<b>H</b>), 7.58 (s, 2H, Ar-<b>H</b>), 4.90 (s, 2H, O<b>CH</b><sub><b>2</b></sub>), 3.53 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.97 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.66 (s, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.37 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.30 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.58, 131.22, 131.09, 129.27, 128.80, 128.76, 128.16, 120.31, 73.37, 53.68, 51.95, 45.63, 16.75. HRMS (ESI): calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 491.1748; found, 491.1750.</div></div><div id="sec4_5_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 6-(2,6-Dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydropyrido [4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)-<i>N</i>-hydroxyhexanamide (<b>17c</b>)</h4><div class="NLM_p last">Compound <b>17c</b> was synthesized from <b>15c</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 38%, yellow powder. mp 173.6–176.8 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.42 (br s, 1H, <b>OH</b>), 8.75 (s, 1H, <b>NH</b>), 7.90 (s, 2H, Ar-<b>H</b>), 3.75 (t, <i>J</i> = 6.4 Hz, 2H, ArO<b>CH</b><sub><b>2</b></sub>), 3.51 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.96 (t, <i>J</i> = 5.8 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.64 (t, <i>J</i> = 5.8 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.36 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.27 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 2.03–1.96 (m, 2H, <b>CH</b><sub><b>2</b></sub>C═O), 1.74 (p, <i>J</i> = 7.2 Hz, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.58 (p, <i>J</i> = 7.2 Hz, 2H, <b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>C═O), 1.51–1.40 (m, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.45, 165.12, 157.84, 130.51, 129.11, 128.81, 128.70, 128.66, 126.49, 119.87, 72.10, 53.79, 52.10, 45.68, 32.69, 30.12, 26.67, 25.69, 25.48, 16.59. HRMS (ESI): calcd for C<sub>24</sub>H<sub>31</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 471.2061; found, 471.2057.</div></div><div id="sec4_5_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 4-(2,6-Dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydropyrido [4′,3′:4,5] thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)-<i>N</i>-hydroxybutanamide (<b>17d</b>)</h4><div class="NLM_p last">Compound <b>17d</b> was synthesized from <b>15d</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 35%, yellow powder. mp 190.9–192.1 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.17 (br s, pyrimidine-<b>H</b>), 10.45 (s, 1H, <b>OH</b>), 7.85 (s, 2H, Ar-<b>H</b>), 3.79 (t, <i>J</i> = 6.3 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>), 3.55 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.95 (t, <i>J</i> = 5.7 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.67 (t, <i>J</i> = 5.7 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 2.37 (s, 3H, N–<b>CH</b><sub><b>3</b></sub>), 2.28 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 2.21 (t, <i>J</i> = 7.4 Hz, 2H, <b>CH</b><sub><b>2</b></sub>C═O), 1.99 (q, <i>J</i> = 6.8 Hz, 2H, OCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.06, 164.03, 159.22, 158.77, 152.56, 131.35, 130.05, 129.33, 128.72, 127.37, 120.54, 71.65, 53.48, 51.71, 45.51, 29.22, 26.47, 26.23, 16.57. HRMS (ESI): calcd for C<sub>22</sub>H<sub>27</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 443.1748; found, 443.1750.</div></div><div id="sec4_5_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (<i>E</i>)-3-(4-((2,6-Dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydro pyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)methyl)phenyl)-<i>N</i>-hydroxy Acrylamide <b>17e</b></h4><div class="NLM_p last">Compound 17e was synthesized from <b>15e</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 37%, yellow powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.03 (d, <i>J</i> = 8.7 Hz, 2H, Ar-<b>H</b>), 7.91 (s, 2H, Ar-<b>H</b>), 7.73 (d, <i>J</i> = 20.6 Hz, 2H, Ar-<b>H</b>), 5.18 (s, 2H, O<b>CH</b><sub><b>2</b></sub>Ar), 4.20–3.54 (m, 4H, Piperidline-<b>H</b>), 3.36 (d, <i>J</i> = 9.6 Hz, 2H, Piperidline-<b>H</b>), 2.56–2.36 (m, 9H, 2Ar-<b>CH</b><sub><b>3</b></sub>, N–<b>CH</b><sub><b>3</b></sub>). HRMS (ESI): calcd for C<sub>28</sub>H<sub>29</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 517.1904; found, 517.1912.</div></div><div id="sec4_5_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (<i>E</i>)-3-(4-((2-(2,6-Dimethyl-4-(7-methyl-4-oxo-3,4,5,6,7,8-hexahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-2-yl)phenoxy)ethoxy)methyl)phenyl)-<i>N</i>-hydroxy Acrylamide (<b>17f</b>)</h4><div class="NLM_p last">Compound <b>17f</b> was synthesized from <b>15f</b> and <b>2c</b>, in a manner similar to <b>4a</b>. Yield 33%, yellow powder. mp >320 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.90–7.74 (m, 3H, Ar-<b>H</b>, Ar-<b>CH</b>═CH), 7.56 (s, 2H, Ar-<b>H</b>), 7.41 (s, 2H, Ar-<b>H</b>), 6.48 (s, 1H, Ar-CH═<b>CH</b>), 4.61 (s, 2H, O<b>CH</b><sub><b>2</b></sub>Ar), 4.01 (s, 2H, ArO<b>CH</b><sub><b>2</b></sub>CH<sub>2</sub>), 3.78 (s, 2H, ArOCH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 3.56 (s, 2H, Piperidline-<b>H</b>), 2.96 (s, 2H, Piperidline-<b>H</b>), 2.67 (s, 2H, Piperidline-<b>H</b>), 2.42–2.22 (m, 9H, 2Ar-<b>CH</b><sub><b>3</b></sub>, N–<b>CH</b><sub><b>3</b></sub>). HRMS (ESI): calcd for C<sub>30</sub>H<sub>33</sub>N<sub>4</sub>O<sub>5</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 583.1986; found, 583.1995.</div></div><div id="sec4_5_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Ethyl 2-Amino-4,5,6,7-tetrahydrothieno[2,3-<i>c</i>]pyridine-3-carboxylate (<b>18</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.22 (s, 2H, <b>NH</b><sub><b>2</b></sub>), 4.14 (q, <i>J</i> = 7.1 Hz, 2H, O<b>CH</b><sub><b>2</b></sub>CH<sub>3</sub>), 3.56 (t, <i>J</i> = 2.0 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.85 (t, <i>J</i> = 5.8 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>), 2.56 (d, <i>J</i> = 5.4 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.24 (t, <i>J</i> = 7.1 Hz, 3H, OCH<sub>2</sub><b>CH</b><sub><b>3</b></sub>).</div></div><div id="sec4_5_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Ethyl 2-Amino-6-(6-methoxy-6-oxohexyl)-4,5,6,7-tetrahydrothieno[2,3-<i>c</i>]pyridine-3-carboxylate (<b>19</b>)</h4><div class="NLM_p last">Yellow powder. mp 281 °C, dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.82 (s, 2H, Ar-<b>H</b>), 3.58 (s, 3H, O<b>Me</b>), 3.53 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.91 (t, <i>J</i> = 5.5 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.68 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.45 (t, <i>J</i> = 6.9 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>), 2.31 (t, <i>J</i> = 7.4 Hz, 2H, MeCOO<b>CH</b><sub><b>2</b></sub>), 2.20 (s, 6H, 2Ar-<b>CH</b><sub><b>3</b></sub>), 1.60–1.47 (m, 4H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.35–1.29 (m, 2H, N–CH<sub>2</sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>).</div></div><div id="sec4_5_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>N</i>-Hydroxy-6-(2-(4-hydroxy-3,5-dimethylphenyl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)hexanamide (<b>20</b>)</h4><div class="NLM_p last">Yellow powder. mp 318 °C dec. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.16 (s, 1H, pyrimidine-<b>H</b>), 10.34 (s, 1H, NH<b>OH</b>), 9.00 (s, 1H, Ar-<b>OH</b>), 8.65 (s, 1H, <b>NH</b>OH), 7.81 (s, 2H, Ar-<b>H</b>), 3.77 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.99 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.87 (s, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.61 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>), 2.23 (s, 8H, 2Ar-<b>CH</b><sub><b>3</b></sub>, CO<b>CH</b><sub><b>2</b></sub>), 1.62–1.50 (m, 4H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub><b>CH</b><sub><b>2</b></sub>), 1.36–1.28 (m, 2H, N–CH<sub>2</sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.92, 169.50, 164.62, 159.22, 157.15, 153.13, 129.45, 128.47, 124.80, 122.58, 119.88, 56.93, 51.27, 49.82, 34.08, 32.68, 26.76, 25.46, 24.84, 17.08. HRMS (ESI): calcd for C<sub>23</sub>H<sub>28</sub>N<sub>4</sub>NaO<sub>4</sub>S<sup>+</sup> [M + Na]<sup>+</sup>, 479.1723; found, 479.1720.</div></div><div id="sec4_5_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 2-(2-(3,5-Dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)-<i>N</i>-hydroxyacetamide (<b>21a</b>)</h4><div class="NLM_p last">Compound <b>9b</b> (86 mg, 0.20 mmol) and NaOH (40 mg, 1.0 mmol) were dissolved in methanol (5 mL), and then hydroxylamine (330 mg, 10 mmol, 50% in water) was added into the solution. The mixture was stirred for 15 min at room temperature and then acidified with 4 M HC1/dioxane to pH 2–3, followed by filtration and concentration, and the crude product was purified by thin layer chromatography using a mixture of dichloromethane:methanol (5:1) to give <b>21a</b>. Yield 76.6%, yellow solid. mp 205.9–207.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.50 (s, 1H, NH<b>OH</b>), 8.79 (s, 1H, <b>NH</b>OH), 3.74 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.11 (s, 2H, CO<b>CH</b><sub><b>2</b></sub>N), 2.94 (t, <i>J</i> = 5.2 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.80 (t, <i>J</i> = 5.2 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.51 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.31 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.46, 166.25, 163.95, 163.65, 159.07, 158.89, 130.30, 129.54, 121.01, 111.45, 58.14, 51.38, 49.69, 25.88, 12.50, 10.83. HRMS (ESI): calcd for C<sub>16</sub>H<sub>18</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 376.1074; found, 376.1073.</div></div><div id="sec4_5_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 4-(2-(3,5-Dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)-<i>N</i>-hydroxybutanamide (<b>21b</b>)</h4><div class="NLM_p last">Compound <b>21b</b> was synthesized from compound <b>9c</b> and hydroxylamine, in a manner similar to <b>21a</b>. Yield 73.1%, yellow solid. mp 215.2–217.7 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.36 (s, 1H, NH<b>OH</b>), 8.68 (s, 1H, <b>NH</b>OH), 3.63 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.95 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.73 (t, <i>J</i> = 5.2 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.54 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.47 (t, <i>J</i> = 7.2 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>), 2.34 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.02 (t, <i>J</i> = 7.2 Hz, 2H, CO<b>CH</b><sub><b>2</b></sub>), 1.81–1.67 (m, 2H, N–CH<sub>2</sub>–<b>CH2</b>–CH<sub>2</sub>). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.40, 169.53, 163.66, 159.23, 158.89, 146.95, 130.63, 129.75, 120.96, 111.54, 56.53, 51.56, 49.75, 30.60, 26.06, 23.22, 12.51, 10.86. HRMS (ESI): calcd for C<sub>18</sub>H<sub>22</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>,404.1387; found, 404.1391.</div></div><div id="sec4_5_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 6-(2-(3,5-Dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5] thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)-<i>N</i>-hydroxyhexanamide (<b>21c</b>)</h4><div class="NLM_p last">Compound <b>21c</b> was synthesized from compound <b>29d</b> and hydroxylamine, in a manner similar to <b>21a</b>. Yield 72.5%, yellow solid. mp 183.6–185.4 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.37 (s, 1H, pyrimidine-<b>H</b>), 10.33 (s, 1H, NH<b>OH</b>), 8.65 (s, 1H, <b>NH</b>OH), 3.62 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.94 (t, <i>J</i> = 4.8 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.73 (t, <i>J</i> = 4.8 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.54 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.47 (t, <i>J</i> = 7.2 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>), 2.34 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 1.96 (t, <i>J</i> = 7.2 Hz, 2H, CO<b>CH</b><sub><b>2</b></sub>), 1.58–1.44 (m, 4H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.32–1.23 (m, 2H, N–CH<sub>2</sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.15, 174.29, 168.38, 163.96, 163.64, 151.68, 135.45, 134.49, 125.73, 116.29, 61.98, 56.38, 54.67, 37.48, 31.70, 31.53, 30.91, 30.29, 17.26, 15.61. HRMS (ESI): calcd for C<sub>20</sub>H<sub>26</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 432.1700; found, 432.1702.</div></div><div id="sec4_5_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> <i>N</i>-(2-Aminophenyl)-2-(2-(3,5-dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)acetamide (<b>22a</b>)</h4><div class="NLM_p last">Compound <b>9b</b> (86 mg, 0.20 mmol) was added into the mixed solution of THF (3 mL) and 2 N NaOH (1 mL). The solution was stirred for 4 h at room temperature. The mixture was acidified with 4 M HC1/dioxane to pH 2–3, followed by concentration to give a crude product. The above product, benzene-1,2-diamine (108 mg, 1.0 mmol) EDCI (76.8 mg, 0.40 mmol) and HOBt (54 mg, 0.4 mmol) was dissolved in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (10 mL), and the reaction mixture was stirred for 12 h at room temperature, followed by filtration and concentration; the crude product was purified by thin layer chromatography using a mixture of dichloromethane:methanol (20:1) to give <b>22a</b>. Yield 28.3%, yellow solid. mp 118.6–120.2 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.39 (s, 1H, pyrimidine-<b>H</b>), 9.19 (s, 1H, <b>NH</b>CO), 7.30 (dd, <i>J</i> = 7.9, 1.2 Hz, 1H, Ar-<b>H</b>), 6.94–6.88 (m, 1H, Ar-<b>H</b>), 6.76 (dd, <i>J</i> = 7.9, 1.2 Hz, 1H, Ar-<b>H</b>), 6.62–6.56 (m, 1H, Ar-<b>H</b>), 4.78 (s, 2H, Ar-<b>NH</b><sub><b>2</b></sub>), 3.88 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.38 (s, 2H, CO–<b>CH</b><sub><b>2</b></sub>–N), 3.03 (t, <i>J</i> = 5.6 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.94 (t, <i>J</i> = 5.6 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.54 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.34 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.46, 168.66, 163.72, 159.11, 158.89, 146.91, 142.07, 130.32, 129.60, 126.19, 125.27, 124.20, 121.03, 117.14, 116.80, 111.47, 60.56, 51.49, 49.84, 26.01, 12.50, 10.84. HRMS (ESI): calcd for C<sub>22</sub>H<sub>23</sub>N<sub>6</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 451.1547; found, 451.1540.</div></div><div id="sec4_5_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> <i>N</i>-(2-Aminophenyl)-6-(2-(3,5-dimethylisoxazol-4-yl)-4-oxo-3,5,6,8-tetrahydropyrido[4′,3′:4,5]thieno[2,3-<i>d</i>]pyrimidin-7(4<i>H</i>)-yl)hexanamide (<b>22b</b>)</h4><div class="NLM_p last">Compound <b>22b</b> was synthesized from compound <b>9d</b> and benzene-1,2-diamine, in a manner similar to <b>22a</b>. Yield 24.5%, yellow solid. mp 88.6–92.2 °C. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.46 (s, 1H, Ar-<b>H</b>), 7.13 (d, <i>J</i> = 7.7 Hz, 1H, Ar-<b>H</b>), 7.02 (t, <i>J</i> = 7.1 Hz, 1H, Ar-<b>H</b>), 6.76–6.72 (m, 2H, Ar-<b>H</b>, <b>NH</b>CO), 3.69 (s, 2H, N–<b>CH</b><sub><b>2</b></sub>-thiophene), 3.02 (t, <i>J</i> = 5.4 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>–CH<sub>2</sub>-thiophene), 2.82 (t, <i>J</i> = 5.4 Hz, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>-thiophene), 2.65 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.59 (t, <i>J</i> = 7.2 Hz, 2H, N–<b>CH</b><sub><b>2</b></sub>), 2.48 (s, 3H, isoxazole-<b>CH</b><sub><b>3</b></sub>), 2.43 (t, <i>J</i> = 7.2 Hz, 2H, CO–<b>CH</b><sub><b>2</b></sub>), 1.80 (p, <i>J</i> = 7.2 Hz, 2H, CO–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.71–1.63 (m, 2H, N–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>), 1.52–1.45 (m, 2H, N–CH<sub>2</sub>–CH<sub>2</sub>–<b>CH</b><sub><b>2</b></sub>). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 171.78, 170.40, 164.92, 160.51, 158.53, 145.67, 140.90, 131.65, 129.65, 127.10, 125.33, 124.35, 120.49, 119.46, 118.14, 110.90, 57.30, 51.99, 49.86, 36.82, 29.70, 26.94, 25.52, 14.12, 12.65, 11.03. HRMS (ESI): calcd for C<sub>26</sub>H<sub>31</sub>N<sub>6</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 507.2173; found, 507.2176.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> BRD4 and HDACs Enzymatic Assays</h3><div class="NLM_p last">The in vitro BRD4 and HDACs assays were performed according to the previous reports and manufacturer’s protocols. For the BRD4 assay, the HTRF assay used a GST-tagged bromodomain of the BRD4 protein, and then, the biotinylated Kac peptide of histone H4 (1–21) and two HTRF detection reagents were added. The HTRF signal was proportional to the amount of interaction between GST-tagged BRD4 protein and biotinylated Kac peptide. For the HDACs assays, the HTRF assay used the biotinylated Kac peptide of histone H3 (1–21), an Eu<sup>3+</sup> cryptate-labeled anti-H3K9 monoclonal antibody, and fluorescence-conjugated streptavidin. The HTRF signal was proportional to the concentration of deacetylated H3 (1–21) peptide. The BRD enzymatic profiling assays were performed by the Alphascreen method, and the detailed experimental procedures were reported previously.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Cell Proliferation and Apoptosis Assays</h3><div class="NLM_p last">The CRC cell lines SW620 and HCT-116 were obtained from the ATCC (American Type Culture Collection) and cultured in the state key laboratory of biotherapy, west china hospital, Sichuan University. The SW620 and HCT-116 cells were cultured in DMEM supplemented with 10% fetal bovine serum and maintained at 37 °C with 5% CO<sub>2</sub> in atmosphere. The cell proliferation assay was measured by using the MTT method. Cells treated with DMSO were set as negative control. In brief, about 5 × 10<sup>3</sup> cells were incubated with test compounds in a 96-well plate for 24–48 h; then 10 μL MTT (5 mg/mL) was added and incubated for an additional 4 h; the extraction buffer was added and incubated overnight; and the absorbance values were measured at 570 nm on a microplate reader (Thermo Scientific Multiskan, Finland). The annexin V-propidium iodide (PI) dual-staining method was used as apoptosis assay. HCT-116 cells with or without compounds incubated were harvested and washed twice with cold phosphate-buffered saline (PBS). The early and late apoptosis cells were identified on a flowcytometry instrument (BD Biosciences, San Jose, CA, USA) by staining with a FITC-conjugated annexin V-PI kit according to the manufacturer’s instructions (Keygen, Nanjing, China). In addition, HCT-116 cells were plated in six-well plates, the cells were grown and adhered for 24 h and then incubated with 1.5, 2, or 3 μM of <b>17c</b> for an additional 12 h followed by Hoechst 33258 addition. The morphology of nuclei was visualized under an Olympus fluorescence microscope.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Western Blot Analysis</h3><div class="NLM_p last">The WB analysis was performed according to our previously reports. In brief, the equivalent concentrations of total proteins in cell lysate were separated by SDS-PAGE and then transferred to PVDF (poly-vinylidene difluoride membrane, Millipore, MA, USA). After blocked by 0.5–1% of BSA under 4 °C overnight, the PVDF membranes were incubated by corresponding primary antibodies at 4 °C overnight or room temperature for 2 h. The PVDF membranes were washed twice by TBST solution and then incubated with HRP-conjugated second antibodies. The immunoblotting slides were collected by using the ECL (enhanced chemiluminescence) method according to the manufacturer’s instruction.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> In Vivo Pharmacokinetics Study</h3><div class="NLM_p last">The in vivo antitumor activity, oral pharmacokinetics, and preliminary safety of <b>17c</b> were carried out according to the Guidelines for the Care and Use of Laboratory Animals that were approved by the Institutional Animal Care and Use Committee and Committee of Ethics of Animal Experimentation of Sichuan University. Compound <b>17c</b> was encapsulated in hydroxypropyl-β-cyclodextrin aqueous solution for intravenous administration and 0.5% sodium carboxyl methyl cellulose (CMC-Na) aqueous solution for oral administration. The Sprague-Dawley rats with 200–250 g body weights were obtained from Beijing HFK Bioscience Co. Ltd. The compound <b>17c</b> was intravenously administered to a group of eight rats (four male and four female rats), by a bolus injection (30 mg/kg dose) to the tail vein or oral administration. At time points 0 (prior to dosing), 5, 15, 30, 45, 60, 120, 240, 360, 480, 600, 720, and 1440 min after dosing, the blood sample was collected from each animal, and the plasma was separated by refrigerated centrifugation and then stored in a freezer under liquid nitrogen. All samples were separated and determined the plasma concentration compound <b>17c</b> by LC–MS/MS (Waters Acquity UPLC-Q-TOF instrument).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Xenograft Tumor Models and Primary Safety Evaluation</h3><div class="NLM_p last">The six to eight weeks old SPF (specific pathogen-free) male nude mice (Balb/cAnNCrl) were purchased from Beijing Vital River Biotechnology Co., Ltd and were randomly grouped by weight and coded (<i>n</i> = 8 per group). After adaptation for five to eight days, the mice were subcutaneously grafted into the dorsal flank with 0.1 mL of phosphate buffer containing 2 × 10<sup>6</sup> HCT-116 cells. When the tumors grew to a size of approximate leg diameter of 6 mm, the mice were initially treated and orally administered with <b>17c</b> or saline on days 1–19 and were monitored on a daily basis during treatment (tumor volume and body weights). On the last day, the animals were sacrificed; tumors and main organs were isolated and weighed. The tumor and various organs tissues were sectioned, then fixed in 4% paraformaldehyde in PBS for immunohistochemistry analysis, and stained with TUNEL, Ki67, Ac-H3, c-Myc, BRD4, LC3-II, LIFR, and p-STAT3 antibodies, as stated by previous studies.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98"><a href="/doi/suppl/10.1021/acs.jmedchem.9b02178" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44583" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44583" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b02178?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02178</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Additional figures containing additional figures related biological studies; NMR spectra of final compounds and HPLC chromatogram of final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_001.csv">jm9b02178_si_001.csv (3.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf">jm9b02178_si_002.pdf (8.27 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b02178" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68446" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68446" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qinglin Jiang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy and Sichuan Province College Key Laboratory of Structure-Specific
Small Molecule Drugs, Chengdu Medical College, Chengdu 610500, PR China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9bf1eaf7c4f8f6f8dbaaada8b5f8f4f6"><span class="__cf_email__" data-cfemail="8ce6fde0d3efe1efccbdbabfa2efe3e1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Liu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#224e4b57404d10161212621314110c414d4f"><span class="__cf_email__" data-cfemail="107c7965727f22242020502126233e737f7d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gu He</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1536-8882" title="Orcid link">http://orcid.org/0000-0002-1536-8882</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d8b0bdbfad98abbbadf6bdbcadf6bbb6"><span class="__cf_email__" data-cfemail="3159545644714252441f5455441f525f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaoping Pan</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiang Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yujia Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Jiang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy and Sichuan Province College Key Laboratory of Structure-Specific
Small Molecule Drugs, Chengdu Medical College, Chengdu 610500, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Min Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy and Sichuan Province College Key Laboratory of Structure-Specific
Small Molecule Drugs, Chengdu Medical College, Chengdu 610500, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nan Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengbo Wu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Department of Urology, West China
Hospital, Sichuan University and Collaborative
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, PR China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Z.P. and X.L. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from National Natural Science Foundation of China (grant nos. 21772131, 81603065 and 81673455), China Postdoctoral Science Foundation (nos 2016M602696 and 2016M592679), National Science and Technology Major Project of the Ministry of Science and Technology of China (No. 2018ZX09735005) and the Fundamental Research Funds for the Science & Technology department of Sichuan Province (2019YFSY0004).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylases</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal carcinoma</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">IHC</td><td class="NLM_def"><p class="first last">immunohistochemical</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">U.S. food and drug administration</p></td></tr><tr><td class="NLM_term">PP1α</td><td class="NLM_def"><p class="first last">protein phosphatase 1 alpha</p></td></tr><tr><td class="NLM_term">P-TEFb</td><td class="NLM_def"><p class="first last">positive transcriptional elongation factor b</p></td></tr><tr><td class="NLM_term">LIFR</td><td class="NLM_def"><p class="first last">leukemia inhibitory factor receptor</p></td></tr><tr><td class="NLM_term">JAK</td><td class="NLM_def"><p class="first last">Janus kinase</p></td></tr><tr><td class="NLM_term">STAT</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription</p></td></tr><tr><td class="NLM_term">SAHA</td><td class="NLM_def"><p class="first last">suberoylanilide hydroxamic acid</p></td></tr><tr><td class="NLM_term">TMA</td><td class="NLM_def"><p class="first last">tissue microarray</p></td></tr><tr><td class="NLM_term">Kac</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">ZINC</td><td class="NLM_def"><p class="first last">ZINC is not commercial</p></td></tr><tr><td class="NLM_term">ZBG</td><td class="NLM_def"><p class="first last">zinc binding group</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence energy transfer</p></td></tr><tr><td class="NLM_term">MRT</td><td class="NLM_def"><p class="first last">mean residence time</p></td></tr><tr><td class="NLM_term">T/C</td><td class="NLM_def"><p class="first last">treatment to control ratios</p></td></tr><tr><td class="NLM_term">TUNEL</td><td class="NLM_def"><p class="first last">terminal deoxynucleotidyl transferase dUTP nick end labeling</p></td></tr><tr><td class="NLM_term">H&E</td><td class="NLM_def"><p class="first last">hematoxylin and eosin stain</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">MCSS</td><td class="NLM_def"><p class="first last">multiple copy simultaneous search</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption; distribution; metabolism; excretion and toxicity</p></td></tr><tr><td class="NLM_term">ECL</td><td class="NLM_def"><p class="first last">enhanced chemiluminescence</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76718" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76718" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedewa, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahnen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meester, R. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Colorectal Cancer Statistics, 2017</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.3322/caac.21395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.3322%2Fcaac.21395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28248415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC1czhslSisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=177-193&author=R.+L.+Siegelauthor=K.+D.+Millerauthor=S.+A.+Fedewaauthor=D.+J.+Ahnenauthor=R.+G.+S.+Meesterauthor=A.+Barziauthor=A.+Jemal&title=Colorectal+Cancer+Statistics%2C+2017&doi=10.3322%2Fcaac.21395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Colorectal cancer statistics, 2017</span></div><div class="casAuthors">Siegel Rebecca L; Miller Kimberly D; Fedewa Stacey A; Ahnen Dennis J; Meester Reinier G S; Barzi Afsaneh; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">177-193</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Colorectal cancer (CRC) is one of the most common malignancies in the United States.  Every 3 years, the American Cancer Society provides an update of CRC incidence, survival, and mortality rates and trends.  Incidence data through 2013 were provided by the Surveillance, Epidemiology, and End Results program, the National Program of Cancer Registries, and the North American Association of Central Cancer Registries.  Mortality data through 2014 were provided by the National Center for Health Statistics.  CRC incidence rates are highest in Alaska Natives and blacks and lowest in Asian/Pacific Islanders, and they are 30% to 40% higher in men than in women.  Recent temporal patterns are generally similar by race and sex, but differ by age.  Between 2000 and 2013, incidence rates in adults aged ≥50 years declined by 32%, with the drop largest for distal tumors in people aged ≥65 years (incidence rate ratio [IRR], 0.50; 95% confidence interval [95% CI], 0.48-0.52) and smallest for rectal tumors in ages 50 to 64 years (male IRR, 0.91; 95% CI, 0.85-0.96; female IRR, 1.00; 95% CI, 0.93-1.08).  Overall CRC incidence in individuals ages ≥50 years declined from 2009 to 2013 in every state except Arkansas, with the decrease exceeding 5% annually in 7 states; however, rectal tumor incidence in those ages 50 to 64 years was stable in most states.  Among adults aged <50 years, CRC incidence rates increased by 22% from 2000 to 2013, driven solely by tumors in the distal colon (IRR, 1.24; 95% CI, 1.13-1.35) and rectum (IRR, 1.22; 95% CI, 1.13-1.31).  Similar to incidence patterns, CRC death rates decreased by 34% among individuals aged ≥50 years during 2000 through 2014, but increased by 13% in those aged <50 years.  Progress against CRC can be accelerated by increasing initiation of screening at age 50 years (average risk) or earlier (eg, family history of CRC/advanced adenomas) and eliminating disparities in high-quality treatment.  In addition, research is needed to elucidate causes for increasing CRC in young adults.  CA Cancer J Clin 2017. © 2017 American Cancer Society.  CA Cancer J Clin 2017;67:177-193. © 2017 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZtSA02_IoCDAyhZbj3lKdfW6udTcc2eaV4KRiWOPSbbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czhslSisA%253D%253D&md5=928172926da7d9ba249bc33406087076</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21395%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DK.%2BD.%26aulast%3DFedewa%26aufirst%3DS.%2BA.%26aulast%3DAhnen%26aufirst%3DD.%2BJ.%26aulast%3DMeester%26aufirst%3DR.%2BG.%2BS.%26aulast%3DBarzi%26aufirst%3DA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DColorectal%2520Cancer%2520Statistics%252C%25202017%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2017%26volume%3D67%26spage%3D177%26epage%3D193%26doi%3D10.3322%2Fcaac.21395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Cancer Statistics, 2017</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.3322/caac.21387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.3322%2Fcaac.21387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28055103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC1c7isFWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=7-30&author=R.+L.+Siegelauthor=K.+D.+Millerauthor=A.+Jemal&title=Cancer+Statistics%2C+2017&doi=10.3322%2Fcaac.21387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Statistics, 2017</span></div><div class="casAuthors">Siegel Rebecca L; Miller Kimberly D; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-30</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Each year, the American Cancer Society estimates the numbers of new cancer cases and deaths that will occur in the United States in the current year and compiles the most recent data on cancer incidence, mortality, and survival.  Incidence data were collected by the Surveillance, Epidemiology, and End Results Program; the National Program of Cancer Registries; and the North American Association of Central Cancer Registries.  Mortality data were collected by the National Center for Health Statistics.  In 2017, 1,688,780 new cancer cases and 600,920 cancer deaths are projected to occur in the United States.  For all sites combined, the cancer incidence rate is 20% higher in men than in women, while the cancer death rate is 40% higher.  However, sex disparities vary by cancer type.  For example, thyroid cancer incidence rates are 3-fold higher in women than in men (21 vs 7 per 100,000 population), despite equivalent death rates (0.5 per 100,000 population), largely reflecting sex differences in the "epidemic of diagnosis." Over the past decade of available data, the overall cancer incidence rate (2004-2013) was stable in women and declined by approximately 2% annually in men, while the cancer death rate (2005-2014) declined by about 1.5% annually in both men and women.  From 1991 to 2014, the overall cancer death rate dropped 25%, translating to approximately 2,143,200 fewer cancer deaths than would have been expected if death rates had remained at their peak.  Although the cancer death rate was 15% higher in blacks than in whites in 2014, increasing access to care as a result of the Patient Protection and Affordable Care Act may expedite the narrowing racial gap; from 2010 to 2015, the proportion of blacks who were uninsured halved, from 21% to 11%, as it did for Hispanics (31% to 16%).  Gains in coverage for traditionally underserved Americans will facilitate the broader application of existing cancer control knowledge across every segment of the population.  CA Cancer J Clin 2017;67:7-30. © 2017 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSB4oj7vNRTIb5oR8RchjrrfW6udTcc2eaepKaRTWRWYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7isFWgtA%253D%253D&md5=e439f3424d5a773e4ae289c392236ad6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21387%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DK.%2BD.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DCancer%2520Statistics%252C%25202017%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2017%26volume%3D67%26spage%3D7%26epage%3D30%26doi%3D10.3322%2Fcaac.21387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span> <span> </span><span class="NLM_article-title">Enhanced Antitumor Activity and Mechanism of Biodegradable Polymeric Micelles-Encapsulated Chetomin in Both Transgenic Zebrafish and Mouse Models</span>. <i>Nanoscale</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">11940</span>– <span class="NLM_lpage">11952</span>, <span class="refDoi"> DOI: 10.1039/c4nr02978j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1039%2Fc4nr02978j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25175172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlGmur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=11940-11952&author=Q.+Wuauthor=G.+Liauthor=S.+Dengauthor=L.+Ouyangauthor=L.+Liauthor=L.+Liuauthor=N.+Luoauthor=X.+Songauthor=G.+Heauthor=C.+Gongauthor=Y.+Wei&title=Enhanced+Antitumor+Activity+and+Mechanism+of+Biodegradable+Polymeric+Micelles-Encapsulated+Chetomin+in+Both+Transgenic+Zebrafish+and+Mouse+Models&doi=10.1039%2Fc4nr02978j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced antitumor activity and mechanism of biodegradable polymeric micelles-encapsulated chetomin in both transgenic zebrafish and mouse models</span></div><div class="casAuthors">Wu, Qinjie; Li, Guoyou; Deng, Senyi; Ouyang, Liang; Li, Ling; Liu, Lei; Luo, Na; Song, Xiangrong; He, Gu; Gong, Changyang; Wei, Yuquan</div><div class="citationInfo"><span class="NLM_cas:title">Nanoscale</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11940-11952</span>CODEN:
                <span class="NLM_cas:coden">NANOHL</span>;
        ISSN:<span class="NLM_cas:issn">2040-3372</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Chetomin is a promising mol. with anti-tumor activities in the epipolythiodioxopiperazine family of fungal secondary metabolites; however, strong hydrophobicity has limited its further applications.  In this work, chetomin was encapsulated into polymeric micelles to obtain an aq. formulation, and the chetomin loaded micelles (Che-M) exhibited small particle size and high encapsulation efficiency.  When the concn. of copolymer was higher than the crit. gelation concn., the Che-M could form a thermosensitive hydrogel (Che-H), which was free-flowing sol at ambient temp. and converted into a non-flowing gel at body temp.  The mol. modeling study has indicated that chetomin interacted with PCL as a core, which was embraced by PEG as a shell.  Che-M showed equal cytotoxicity with free chetomin, but the apoptosis inducing effects of Che-M were more significant.  Besides, Che-M could increase the GSSG level, decrease the GSH level, and increase the ROS in CT26 cells.  Furthermore, stronger inhibitory effects of Che-M were obsd. on embryonic angiogenesis, tumor-induced angiogenesis and tumor growth in transgenic zebrafish models.  In addn., Che-M was effective in inhibiting tumor growth and prolonging survival in a s.c. CT26 tumor model.  In a colorectal peritoneal carcinomatosis model, both Che-M and Che-H showed excellent therapeutic effects, but Che-H was more effective.  In conclusion, Che-M and Che-H may serve as candidates for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9tkCAHB9X9rVg90H21EOLACvtfcHk0lgBIrSls605zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGmur%252FI&md5=01842011099d90eebc7eddb89c1ea6bd</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1039%2Fc4nr02978j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4nr02978j%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DN.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DGong%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DY.%26atitle%3DEnhanced%2520Antitumor%2520Activity%2520and%2520Mechanism%2520of%2520Biodegradable%2520Polymeric%2520Micelles-Encapsulated%2520Chetomin%2520in%2520Both%2520Transgenic%2520Zebrafish%2520and%2520Mouse%2520Models%26jtitle%3DNanoscale%26date%3D2014%26volume%3D6%26spage%3D11940%26epage%3D11952%26doi%3D10.1039%2Fc4nr02978j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span> <span> </span><span class="NLM_article-title">Targeting Programmed Cell Death Using Small-Molecule Compounds to Improve Potential Cancer Therapy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">983</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1002/med.21398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1002%2Fmed.21398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27357603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2s7ltVamsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=983-1035&author=B.+Keauthor=M.+Tianauthor=J.+Liauthor=B.+Liuauthor=G.+He&title=Targeting+Programmed+Cell+Death+Using+Small-Molecule+Compounds+to+Improve+Potential+Cancer+Therapy&doi=10.1002%2Fmed.21398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Programmed Cell Death Using Small-Molecule Compounds to Improve Potential Cancer Therapy</span></div><div class="casAuthors">Ke Bowen; Tian Mao; Li Jingjing; Liu Bo; He Gu</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal research reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">983-1035</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Evasion of cell death is one of the hallmarks of cancer cells, beginning with long-established apoptosis and extending to other new forms of cell death.  An elaboration of cell death pathways thus will contribute to a better understanding of cancer pathogenesis and therapeutics.  With the recent substantial biochemical and genetic explorations of cell death subroutines, their classification has switched from primarily morphological to more molecular definitions.  According to their measurable biochemical features and intricate mechanisms, cell death subroutines can be divided into apoptosis, autophagic cell death, mitotic catastrophe, necroptosis, parthanatos, ferroptosis, pyroptosis, pyronecrosis, anoikis, cornification, entosis, and NETosis.  Supportive evidence has gradually revealed the prime molecular mechanisms of each subroutine and thus providing series of possible targets in cancer therapy, while the intricate relationships between different cell death subroutines still remain to be clarified.  Over the past decades, cancer drug discovery has significantly benefited from the use of small-molecule compounds to target classical modalities of cell death such as apoptosis, while newly identified cell death subroutines has also emerging their potential for cancer drug discovery in recent years.  In this review, we comprehensively focus on summarizing 12 cell death subroutines and discussing their corresponding small-molecule compounds in potential cancer therapy.  Together, these inspiring findings may provide more evidence to fill in the gaps between cell death subroutines and small-molecule compounds to better develop novel cancer therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhPsvIEAjUsGBAiYWy6n3SfW6udTcc2ebRtp6xA-Fud7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7ltVamsA%253D%253D&md5=3fc5de00cb2c9a65b378d0358dfd3f5e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fmed.21398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21398%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DB.%26aulast%3DTian%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DG.%26atitle%3DTargeting%2520Programmed%2520Cell%2520Death%2520Using%2520Small-Molecule%2520Compounds%2520to%2520Improve%2520Potential%2520Cancer%2520Therapy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D983%26epage%3D1035%26doi%3D10.1002%2Fmed.21398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span> <span> </span><span class="NLM_article-title">Approved and Experimental Small-Molecule Oncology Kinase Inhibitor Drugs: A Mid-2016 Overview</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1002/med.21409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1002%2Fmed.21409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27775829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2srls1aqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=314-367&author=P.+M.+Fischer&title=Approved+and+Experimental+Small-Molecule+Oncology+Kinase+Inhibitor+Drugs%3A+A+Mid-2016+Overview&doi=10.1002%2Fmed.21409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Approved and Experimental Small-Molecule Oncology Kinase Inhibitor Drugs: A Mid-2016 Overview</span></div><div class="casAuthors">Fischer Peter M</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal research reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">314-367</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kinase inhibitor research is a comparatively recent branch of medicinal chemistry and pharmacology and the first small-molecule kinase inhibitor, imatinib, was approved for clinical use only 15 years ago.  Since then, 33 more kinase inhibitor drugs have received regulatory approval for the treatment of a variety of cancers and the volume of reports on the discovery and development of kinase inhibitors has increased to an extent where it is now difficult-even for those working in the field-easily to keep an overview of the compounds that are being developed, as currently there are 231 such compounds, targeting 38 different protein and lipid kinases (not counting isoforms), in clinical use or under clinical investigation.  The purpose of this review is thus to provide an overview of the biomedical rationales for the kinases being targeted on the one hand, and the design principles, as well as chemical, pharmacological, pharmaceutical, and toxicological kinase inhibitor properties, on the other hand.  Two issues that are especially important in kinase inhibitor research, target selectivity and drug resistance, as well as the underlying structural concepts, are discussed in general terms and in the context of relevant kinases and their inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQqoId3h33O7qe0F5ixvZgXfW6udTcc2eaH1Hc9LEuxn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srls1aqsw%253D%253D&md5=28842df0fe00c1a57c096d70ad7009ec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fmed.21409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21409%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DApproved%2520and%2520Experimental%2520Small-Molecule%2520Oncology%2520Kinase%2520Inhibitor%2520Drugs%253A%2520A%2520Mid-2016%2520Overview%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2017%26volume%3D37%26spage%3D314%26epage%3D367%26doi%3D10.1002%2Fmed.21409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Hsa-Mir-875-5p Exerts Tumor Suppressor Function through Down-Regulation of Egfr in Colorectal Carcinoma (Crc)</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42225</span>– <span class="NLM_lpage">42240</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.9944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27302926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2s7gtFeqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=42225-42240&author=T.+Zhangauthor=X.+Caiauthor=Q.+Liauthor=P.+Xueauthor=Z.+X.+Chenauthor=X.+Dongauthor=Y.+Xue&title=Hsa-Mir-875-5p+Exerts+Tumor+Suppressor+Function+through+Down-Regulation+of+Egfr+in+Colorectal+Carcinoma+%28Crc%29&doi=10.18632%2Foncotarget.9944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Hsa-miR-875-5p exerts tumor suppressor function through down-regulation of EGFR in colorectal carcinoma (CRC)</span></div><div class="casAuthors">Zhang Tiening; Cai Xun; Li Qi; Xue Peng; Chen Zhixiao; Dong Xiao; Xue Ying</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">42225-42240</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hsa-miRNA-875-5p (miR-875-5p) has recently been discovered to have anticancer efficacy in different organs.  However, the role of miR-875-5p on colorectal carcinoma (CRC) is still ambiguous.  In this study, we investigated the role of miR-875-5p on the development of CRC.  The results indicated that miR-875-5p was significantly down-regulated in primary tumor tissues and very low levels were found in CRC cell lines.  Ectopic expression of miR-875-5p in CRC cell lines significantly suppressed cell growth as evidenced by cell viability assay, colony formation assay and BrdU staining, through inhibition of cyclin D1, cyclin D2, CDK4 and up-regulation of p57(Kip2) and p21(Waf1/Cip1).  In addition, miR-875-5p induced apoptosis, as indicated by concomitantly with up-regulation of key apoptosis protein cleaved caspase-3, and down-regulation of anti-apoptosis protein Bcl2.  Moreover, miR-875-5p inhibited cellular migration and invasiveness through inhibition of matrix metalloproteinases (MMP)-7 and MMP-9.  Further, oncogene EGFR was revealed to be a putative target of miR-875-5p, which was inversely correlated with miR-875-5p expression in CRC.  Taken together, our results demonstrated that miR-875-5p played a pivotal role on CRC through inhibiting cell proliferation, migration, invasion, and promoting apoptosis by targeting oncogenic EGFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRC799yCbWd75oR8RchjrrfW6udTcc2eaH1Hc9LEuxn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7gtFeqsA%253D%253D&md5=d42ab34f353e73257e0f0edeeca36b79</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9944%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DXue%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DZ.%2BX.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DHsa-Mir-875-5p%2520Exerts%2520Tumor%2520Suppressor%2520Function%2520through%2520Down-Regulation%2520of%2520Egfr%2520in%2520Colorectal%2520Carcinoma%2520%2528Crc%2529%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D42225%26epage%3D42240%26doi%3D10.18632%2Foncotarget.9944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubiolo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra-Varela, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, S. B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabezas-Sainz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botana, L. M.</span></span> <span> </span><span class="NLM_article-title">Marine Guanidine Alkaloids Crambescidins Inhibit Tumor Growth and Activate Intrinsic Apoptotic Signaling Inducing Tumor Regression in a Colorectal Carcinoma Zebrafish Xenograft Model</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">83071</span>– <span class="NLM_lpage">83087</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.13068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27825113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2snktF2mug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=83071-83087&author=M.+Roelauthor=J.+A.+Rubioloauthor=J.+Guerra-Varelaauthor=S.+B.+L.+Silvaauthor=O.+P.+Thomasauthor=P.+Cabezas-Sainzauthor=L.+Sanchezauthor=R.+Lopezauthor=L.+M.+Botana&title=Marine+Guanidine+Alkaloids+Crambescidins+Inhibit+Tumor+Growth+and+Activate+Intrinsic+Apoptotic+Signaling+Inducing+Tumor+Regression+in+a+Colorectal+Carcinoma+Zebrafish+Xenograft+Model&doi=10.18632%2Foncotarget.13068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Marine guanidine alkaloids crambescidins inhibit tumor growth and activate intrinsic apoptotic signaling inducing tumor regression in a colorectal carcinoma zebrafish xenograft model</span></div><div class="casAuthors">Roel Maria; Rubiolo Juan A; Botana Luis M; Guerra-Varela Jorge; Cabezas-Sainz Pablo; Sanchez Laura; Silva Siguara B L; Thomas Olivier P; Silva Siguara B L; Thomas Olivier P; Lopez Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">83071-83087</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The marine environment constitutes an extraordinary resource for the discovery of new therapeutic agents.  In the present manuscript we studied the effect of 3 different sponge derived guanidine alkaloids, crambescidine-816, -830, and -800.  We show that these compounds strongly inhibit tumor cell proliferation by down-regulating cyclin-dependent kinases 2/6 and cyclins D/A expression while up-regulating the cell cyclin-dependent kinase inhibitors -2A, -2D and -1A.  We also show that these guanidine compounds disrupt tumor cell adhesion and cytoskeletal integrity promoting the activation of the intrinsic apoptotic signaling, resulting in loss of mitochondrial membrane potential and concomitant caspase-3 cleavage and activation.  The crambescidin 816 anti-tumor effect was fnally assayed in a zebrafish xenotransplantation model confirming its potent antitumor activity against colorectal carcinoma in vivo.Considering these results crambescidins could represent promising natural anticancer agents and therapeutic tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEd_cNSVKnvVMGsG_HTXYFfW6udTcc2ebKtjR60bTCerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snktF2mug%253D%253D&md5=3d650df90587b9d0976f86b32dbdb3fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13068%26sid%3Dliteratum%253Aachs%26aulast%3DRoel%26aufirst%3DM.%26aulast%3DRubiolo%26aufirst%3DJ.%2BA.%26aulast%3DGuerra-Varela%26aufirst%3DJ.%26aulast%3DSilva%26aufirst%3DS.%2BB.%2BL.%26aulast%3DThomas%26aufirst%3DO.%2BP.%26aulast%3DCabezas-Sainz%26aufirst%3DP.%26aulast%3DSanchez%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DBotana%26aufirst%3DL.%2BM.%26atitle%3DMarine%2520Guanidine%2520Alkaloids%2520Crambescidins%2520Inhibit%2520Tumor%2520Growth%2520and%2520Activate%2520Intrinsic%2520Apoptotic%2520Signaling%2520Inducing%2520Tumor%2520Regression%2520in%2520a%2520Colorectal%2520Carcinoma%2520Zebrafish%2520Xenograft%2520Model%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D83071%26epage%3D83087%26doi%3D10.18632%2Foncotarget.13068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. P.</span></span> <span> </span><span class="NLM_article-title">Synchronous Occurrence of Gastrointestinal Stromal Tumors and Other Digestive Tract Malignancies in the Elderly</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8397</span>– <span class="NLM_lpage">8406</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.3108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25826075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2MnpsVejug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8397-8406&author=C.+Shenauthor=H.+N.+Chenauthor=Y.+Yinauthor=J.+J.+Chenauthor=L.+Y.+Hanauthor=B.+Zhangauthor=Z.+X.+Chenauthor=J.+P.+Chen&title=Synchronous+Occurrence+of+Gastrointestinal+Stromal+Tumors+and+Other+Digestive+Tract+Malignancies+in+the+Elderly&doi=10.18632%2Foncotarget.3108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Synchronous occurrence of gastrointestinal stromal tumors and other digestive tract malignancies in the elderly</span></div><div class="casAuthors">Shen Chaoyong; Chen Haining; Yin Yuan; Chen Jiaju; Zhang Bo; Chen Zhixin; Chen Jiaping; Han Luyin</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">8397-406</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND/AIMS:  Elderly patients with gastrointestinal stromal tumors (GISTs) synchronous with other digestive tract malignancies have been rarely reported.  In this study, clinicopathological characteristics were evaluated among elderly patients with GISTs with or without coexisting digestive tract malignancies.  METHODS:  A total of 161 patients (≥65 years) were retrospectively reviewed at the West China Hospital, Sichuan University from January 2009 to June 2014.  RESULTS:  Sixty-one patients were diagnosed with synchronous digestive tract malignancies (synchronous group), whereas 100 patients were diagnosed with no synchronous condition (no-synchronous group).  The synchronous group exhibited a higher percentage of males (70.49% vs. 53.00%, P = 0.028) and poorer Eastern Cooperative Oncology Group performance status than the no-synchronous group (P = 0.029).  The three-year overall survival (OS) rate was significantly lower among patients with synchronous digestive tract malignancies than that among patients without synchronous condition (64.5% vs. 84.0%, P = 0.003).  Multivariate analysis showed that the presence of synchronous digestive tract malignancies (P = 0.002), co-morbidity (P = 0.004), and mitotic count ≥10 mitoses/50 high power fields (P = 0.012) were associated with poor OS.  CONCLUSIONS:  A synchronous condition with other digestive tract malignancies is common in elderly patients with GISTs.  OS primarily depends on synchronous digestive tract malignancies, mitotic count, and co-morbidity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQQhPf3aT5hM7GbPKh4T9eAfW6udTcc2ebKtjR60bTCerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnpsVejug%253D%253D&md5=299550a9d6f7af90287eb1fe57cfb2ed</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3108%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DH.%2BN.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DHan%26aufirst%3DL.%2BY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DZ.%2BX.%26aulast%3DChen%26aufirst%3DJ.%2BP.%26atitle%3DSynchronous%2520Occurrence%2520of%2520Gastrointestinal%2520Stromal%2520Tumors%2520and%2520Other%2520Digestive%2520Tract%2520Malignancies%2520in%2520the%2520Elderly%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D8397%26epage%3D8406%26doi%3D10.18632%2Foncotarget.3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.</span></span> <span> </span><span class="NLM_article-title">Brain-Derived Neurotrophic Factor Involved Epigenetic Repression of Ugt2b7 in Colorectal Carcinoma: A Mechanism to Alter Morphine Glucuronidation in Tumor</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">29138</span>– <span class="NLM_lpage">29150</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.16251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.16251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28418861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC1cvoslGrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=29138-29150&author=Z.+Z.+Yangauthor=L.+Liauthor=M.+C.+Xuauthor=H.+X.+Juauthor=M.+Haoauthor=J.+K.+Guauthor=Z.+J.+J.+Wangauthor=H.+D.+Jiangauthor=L.+S.+Yuauthor=S.+Zeng&title=Brain-Derived+Neurotrophic+Factor+Involved+Epigenetic+Repression+of+Ugt2b7+in+Colorectal+Carcinoma%3A+A+Mechanism+to+Alter+Morphine+Glucuronidation+in+Tumor&doi=10.18632%2Foncotarget.16251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Brain-derived neurotrophic factor involved epigenetic repression of UGT2B7 in colorectal carcinoma: A mechanism to alter morphine glucuronidation in tumor</span></div><div class="casAuthors">Yang Zi-Zhao; Xu Ming-Cheng; Jiang Hui-Di; Yu Lu-Shan; Zeng Su; Li Li; Ju Hai-Xing; Hao Miao; Gu Jing-Kai; Jim Wang Zai-Jie</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">29138-29150</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Uridine diphosphate-glucuronosyltransferase (UGT) 2B7, as one of significant drug enzymes, is responsible on the glucuronidation of abundant endobiotics or xenobiotics.  We here report that it is markedly repressed in the tumor tissues of colorectal carcinoma (CRC) patients.  Accordingly, morphine in CRC cells will stimulate the expression of its main metabolic enzyme, UGT2B7 during tolerance generation by activating the positive signals in histone 3, especially for trimethylated lysine 27 (H3K4Me3) and acetylated lysine 4 (H3K27Ac).  Further study reveals that brain-derived neutrophilic factor (BDNF), a secretory neurotrophin, enriched in CRC can interact and inhibit UGT2B7 by primarily blocking the positive signals of H3K4Me3 as well as activating H3K27Ac on the promoter region of UGT2B7.  Meanwhile, BDNF repression attributes to the sensitizations of main core factors in poly-comb repressive complex (PRC) 1 rather than PRC2 as the reason of the depression of SUZ12 in the later complex.  Besides that, the productions of two main morphine glucuronides are both increased in the BDNF deficient or TSA and BIX-01294 treated morphine tolerance-like HCT-116 cells.  On the same condition, active metabolite, morphine-6-glucuronide (M6G) was accumulated more than inactive M3G.  Our findings imply that enzymatic activity enhancement and substrate regioselective catalysis alteration of UGT2B7 may release morphine tolerance under the cure of tumor-induced pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjN6Q_QjxwZsULxIEhxUjPfW6udTcc2ea5FyXvdb9A6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvoslGrsg%253D%253D&md5=b76a88b93ba7d9940ec1b38bcdb5c61a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.16251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.16251%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%2BZ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DM.%2BC.%26aulast%3DJu%26aufirst%3DH.%2BX.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DJ.%2BK.%26aulast%3DWang%26aufirst%3DZ.%2BJ.%2BJ.%26aulast%3DJiang%26aufirst%3DH.%2BD.%26aulast%3DYu%26aufirst%3DL.%2BS.%26aulast%3DZeng%26aufirst%3DS.%26atitle%3DBrain-Derived%2520Neurotrophic%2520Factor%2520Involved%2520Epigenetic%2520Repression%2520of%2520Ugt2b7%2520in%2520Colorectal%2520Carcinoma%253A%2520A%2520Mechanism%2520to%2520Alter%2520Morphine%2520Glucuronidation%2520in%2520Tumor%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D29138%26epage%3D29150%26doi%3D10.18632%2Foncotarget.16251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y. S.</span></span> <span> </span><span class="NLM_article-title">Mecp2-Mediated Epigenetic Silencing of Mir-137 Contributes to Colorectal Adenoma-Carcinoma Sequence and Tumor Progression Via Relieving the Suppression of C-Met</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">44543</span>, <span class="refDoi"> DOI: 10.1038/srep44543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1038%2Fsrep44543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28291253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=44543&author=T.+Chenauthor=S.+L.+Caiauthor=J.+Liauthor=Z.+P.+Qiauthor=X.+Q.+Liauthor=L.+C.+Yeauthor=X.+F.+Xieauthor=Y.+Y.+Houauthor=L.+Q.+Yaoauthor=M.+D.+Xuauthor=P.+H.+Zhouauthor=J.+M.+Xuauthor=Y.+S.+Zhong&title=Mecp2-Mediated+Epigenetic+Silencing+of+Mir-137+Contributes+to+Colorectal+Adenoma-Carcinoma+Sequence+and+Tumor+Progression+Via+Relieving+the+Suppression+of+C-Met&doi=10.1038%2Fsrep44543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mecp2-mediated Epigenetic Silencing of miR-137 Contributes to Colorectal Adenoma-Carcinoma Sequence and Tumor Progression via Relieving the Suppression of c-Met</span></div><div class="casAuthors">Chen, Tao; Cai, Shi-Lun; Li, Jian; Qi, Zhi-Peng; Li, Xu-Quan; Ye, Le-Chi; Xie, Xiao-Feng; Hou, Ying-Yong; Yao, Li-Qing; Xu, Mei-Dong; Zhou, Ping-Hong; Xu, Jian-Min; Zhong, Yun-Shi</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">44543</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mol. mechanisms underlying colorectal cancer (CRC) development remain elusive.  In this study, we examd. the miRNA and mRNA expressions in the adenoma-carcinoma sequence (ACS), a crit. neoplastic progression in CRC development.  We found that miR-137 was down-regulated in all adenoma and carcinoma tissues.  Low miR-137 levels were correlated neg. with tumor progression and metastasis.  Then we identified the inhibition effect of the miR-137 in CRC development, both in CRC cell lines and mouse models.  MiR-137 was shown to control CRC cell proliferation, colony formation, migration and invasion and to control tumor growth and metastasis.  We further confirmed the neg. assocn. between miR-137 and c-Met expression and thus validated this important oncogene as the target of miR-137 in CRC.  In addn., we found a DNA methyl-CpG-binding protein, Mecp2, was up-regulated in ACS tissues via mRNA sequencing.  Further expt. showed that miR-137 expression in CRC was subjected to epigenetic regulation mediated by Mecp2.  We also confirmed c-Met expression can be up-regulated by silencing of miR-137 and suppressed by coexpression of Mecp2 and miR-137.  These findings highlight the crit. role of miR-137-c-Met nexus in CRC development and reveal Mecp2-regulated epigenetic silence causes the downregulation of miR-137 in colorectal adenoma and carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0nCo6ULP0ebVg90H21EOLACvtfcHk0ljNQsI2sgNKrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKrtLY%253D&md5=ecc8e70fbfdba3d6a0f9ec91b43bd4db</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsrep44543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep44543%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DCai%26aufirst%3DS.%2BL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DZ.%2BP.%26aulast%3DLi%26aufirst%3DX.%2BQ.%26aulast%3DYe%26aufirst%3DL.%2BC.%26aulast%3DXie%26aufirst%3DX.%2BF.%26aulast%3DHou%26aufirst%3DY.%2BY.%26aulast%3DYao%26aufirst%3DL.%2BQ.%26aulast%3DXu%26aufirst%3DM.%2BD.%26aulast%3DZhou%26aufirst%3DP.%2BH.%26aulast%3DXu%26aufirst%3DJ.%2BM.%26aulast%3DZhong%26aufirst%3DY.%2BS.%26atitle%3DMecp2-Mediated%2520Epigenetic%2520Silencing%2520of%2520Mir-137%2520Contributes%2520to%2520Colorectal%2520Adenoma-Carcinoma%2520Sequence%2520and%2520Tumor%2520Progression%2520Via%2520Relieving%2520the%2520Suppression%2520of%2520C-Met%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D44543%26doi%3D10.1038%2Fsrep44543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mrazek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmical, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajjar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellmich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, C.</span></span> <span> </span><span class="NLM_article-title">Genetic and Epigenetic Changes in Carcinoma-Associated Fibroblasts Derived from Human Colorectal Cancers</span>. <i>J. Am. Coll. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>217</i></span>,  <span class="NLM_fpage">S133</span>, <span class="refDoi"> DOI: 10.1016/j.jamcollsurg.2013.07.310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.jamcollsurg.2013.07.310" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2013&pages=S133&author=A.+Mrazekauthor=L.+Porroauthor=J.+R.+Carmicalauthor=G.+Gomezauthor=A.+Gajjarauthor=M.+R.+Hellmichauthor=C.+Chao&title=Genetic+and+Epigenetic+Changes+in+Carcinoma-Associated+Fibroblasts+Derived+from+Human+Colorectal+Cancers&doi=10.1016%2Fj.jamcollsurg.2013.07.310"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jamcollsurg.2013.07.310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jamcollsurg.2013.07.310%26sid%3Dliteratum%253Aachs%26aulast%3DMrazek%26aufirst%3DA.%26aulast%3DPorro%26aufirst%3DL.%26aulast%3DCarmical%26aufirst%3DJ.%2BR.%26aulast%3DGomez%26aufirst%3DG.%26aulast%3DGajjar%26aufirst%3DA.%26aulast%3DHellmich%26aufirst%3DM.%2BR.%26aulast%3DChao%26aufirst%3DC.%26atitle%3DGenetic%2520and%2520Epigenetic%2520Changes%2520in%2520Carcinoma-Associated%2520Fibroblasts%2520Derived%2520from%2520Human%2520Colorectal%2520Cancers%26jtitle%3DJ.%2520Am.%2520Coll.%2520Surg.%26date%3D2013%26volume%3D217%26spage%3DS133%26doi%3D10.1016%2Fj.jamcollsurg.2013.07.310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slyskova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenkova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochazka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vodickova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneiderova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liska, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holubec, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naccarati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vodicka, P.</span></span> <span> </span><span class="NLM_article-title">Functional, Genetic, and Epigenetic Aspects of Base and Nucleotide Excision Repair in Colorectal Carcinomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5878</span>– <span class="NLM_lpage">5887</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-12-1380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1078-0432.ccr-12-1380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=22966016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghtr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5878-5887&author=J.+Slyskovaauthor=V.+Korenkovaauthor=A.+R.+Collinsauthor=P.+Prochazkaauthor=L.+Vodickovaauthor=J.+Svecauthor=L.+Lipskaauthor=M.+Levyauthor=M.+Schneiderovaauthor=V.+Liskaauthor=L.+Holubecauthor=R.+Kumarauthor=P.+Soucekauthor=A.+Naccaratiauthor=P.+Vodicka&title=Functional%2C+Genetic%2C+and+Epigenetic+Aspects+of+Base+and+Nucleotide+Excision+Repair+in+Colorectal+Carcinomas&doi=10.1158%2F1078-0432.ccr-12-1380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Functional, Genetic, and Epigenetic Aspects of Base and Nucleotide Excision Repair in Colorectal Carcinomas</span></div><div class="casAuthors">Slyskova, Jana; Korenkova, Vlasta; Collins, Andrew R.; Prochazka, Pavel; Vodickova, Ludmila; Svec, Jiri; Lipska, Ludmila; Levy, Miroslav; Schneiderova, Michaela; Liska, Vaclav; Holubec, Lubos; Kumar, Rajiv; Soucek, Pavel; Naccarati, Alessio; Vodicka, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5878-5887</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: DNA repair capacity (DRC) is a determinant not only of cancer development but also of individual response to therapy.  Previously, altered base and nucleotide excision repair (BER and NER) have been described in lymphocytes of patients with sporadic colorectal cancer.  We, for the first time, evaluate both excision repair capacities in human colon biopsies to study their participation in colorectal tumorigenesis.  Exptl. design: Seventy pairs of tumor and adjacent healthy tissues were analyzed for BER- and NER-specific DRC by a comet repair assay.  Tissue pairs were further compared for expression levels of a panel of 25 BER and NER genes complemented by their promoter methylation status.  Results: We obsd. a moderate increase of NER-DRC (P = 0.019), but not of BER-DRC in tumors.  There was a strong correlation between both tissues for all investigated parameters (P < 0.001).  However, 4 NER (CSB, CCNH, XPA, XPD) and 4 BER (NEIL1, APEX1, OGG1, PARP1) genes showed a 1.08- to 1.28-fold change difference in expression in tumors (P < 0.05).  Individual gene expression levels did not correlate with overall DRC, and we did not detect any aberrant methylation of the investigated genes.  Conclusions: Our complex anal. showed that tumor cells are not deficient in BER and NER, but rather follow patterns characteristic for each individual and are comparable with adjacent tissue.  Alteration of excision repair pathways is not a pronounced event in colorectal carcinogenesis.  This study shows the feasibility of DRC evaluation in human solid tissues, representing a complex marker of multigene DNA repair processes.  Clin Cancer Res; 18(21); 5878-87. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeM1XBtSpXO7Vg90H21EOLACvtfcHk0ljNQsI2sgNKrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghtr3O&md5=e4d1210d02aa548cc61938c82d771f53</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-12-1380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-12-1380%26sid%3Dliteratum%253Aachs%26aulast%3DSlyskova%26aufirst%3DJ.%26aulast%3DKorenkova%26aufirst%3DV.%26aulast%3DCollins%26aufirst%3DA.%2BR.%26aulast%3DProchazka%26aufirst%3DP.%26aulast%3DVodickova%26aufirst%3DL.%26aulast%3DSvec%26aufirst%3DJ.%26aulast%3DLipska%26aufirst%3DL.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DSchneiderova%26aufirst%3DM.%26aulast%3DLiska%26aufirst%3DV.%26aulast%3DHolubec%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DSoucek%26aufirst%3DP.%26aulast%3DNaccarati%26aufirst%3DA.%26aulast%3DVodicka%26aufirst%3DP.%26atitle%3DFunctional%252C%2520Genetic%252C%2520and%2520Epigenetic%2520Aspects%2520of%2520Base%2520and%2520Nucleotide%2520Excision%2520Repair%2520in%2520Colorectal%2520Carcinomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5878%26epage%3D5887%26doi%3D10.1158%2F1078-0432.ccr-12-1380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lutz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzner, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahrens, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makowiec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopt, U. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogatyreva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauschke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassmann, S.</span></span> <span> </span><span class="NLM_article-title">Histone Modifiers and Marks Define Heterogeneous Groups of Colorectal Carcinomas and Affect Responses to Hdac Inhibitors in Vitro</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">664</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27152243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvFOhurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=664&author=L.+Lutzauthor=I.+C.+Fitznerauthor=T.+Ahrensauthor=A.+L.+Geisslerauthor=F.+Makowiecauthor=U.+T.+Hoptauthor=L.+Bogatyrevaauthor=D.+Hauschkeauthor=M.+Wernerauthor=S.+Lassmann&title=Histone+Modifiers+and+Marks+Define+Heterogeneous+Groups+of+Colorectal+Carcinomas+and+Affect+Responses+to+Hdac+Inhibitors+in+Vitro"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Histone modifiers and marks define heterogeneous groups of colorectal carcinomas and affect responses to HDAC inhibitors in vitro</span></div><div class="casAuthors">Lutz, Lisa; Fitzner, Ingrid Coutino; Ahrens, Theresa; Geissler, Anna-Lena; Makowiec, Frank; Hopt, Ulrich T.; Bogatyreva, Lioudmila; Hauschke, Dieter; Werner, Martin; Lassmann, Silke</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">664-676</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Little is known about histone modifiers and histone marks in colorectal cancers (CRC).  The present study therefore addressed the role of histone acetylation and histone deacetylases (HDAC) in CRCs in situ and in vitro.  Immunohistochem. of primary CRCs (n=47) revealed that selected histone marks were frequently present (H3K4me3: 100%; H3K9me3: 77%; H3K9ac: 75%), partially displayed intratumoral heterogeneity (H3K9me3; H3K9ac) and were significantly linked to higher pT category (H3K9me3: p=0.023; H3K9ac: p=0.028).  Furthermore, also HDAC1 (62%), HDAC2 (100%) and HDAC3 (72%) expression was frequent, revealing four CRC types: cases expressing 1) HDAC1, HDAC2 and HDAC3 (49%), 2) HDAC2 and HDAC3 (30%), 3) HDAC1 and HDAC2 (10.5%) and 4) exclusively HDAC2 (10.5%).  Correlation to clinico-pathol. parameters (pT, pN, G, MSI status) revealed that heterogeneous HDAC1 expression correlated with lymph node status (p=0.012).  HDAC expression in situ was partially reflected by six CRC cell lines, with similar expression of all three HDACs (DLD1, LS174T), preferential HDAC2 and HDAC3 expression (SW480, Caco2) or lower HDAC2 and HDAC3 expression (HCT116, HT29).  HDAC activity was variably higher in HCT116, HT29, DLD1 and SW480 compared to LS174T and Caco2 cells.  Treatment with broad (SAHA) and specific (MS-275; FK228) HDAC inhibitors (HDACi) caused loss of cell viability in predominantly MSIpos. CRC cells (HCT116, LS174T, DLD1; SAHA, MS-275 and in part FK228).  In contrast, MSI-neg. CRC cells (Caco2, HT29, SW480) were resistant, except for high doses of FK228 (Caco2, HT29).  Cell viability patterns were not linked to different efficacies of HDACi on redn. of HDAC activity or histone acetylation, p21 expression and/ or induction of DNA damage (γH2A-X levels).  In summary, this study reveals inter- and intra-tumoral heterogeneity of histone marks and HDAC expression in CRCs.  This is reflected by diverse HDACi responses in vitro, which do not follow known modes of action.  Together, this implies further exploitation of histone alterations in CRC for mol. classification and/or novel treatment options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEDxqeHVagGrVg90H21EOLACvtfcHk0lhLF2gAYPWIsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvFOhurc%253D&md5=c5ca6bcefff7c3f6c74a178072ab7366</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLutz%26aufirst%3DL.%26aulast%3DFitzner%26aufirst%3DI.%2BC.%26aulast%3DAhrens%26aufirst%3DT.%26aulast%3DGeissler%26aufirst%3DA.%2BL.%26aulast%3DMakowiec%26aufirst%3DF.%26aulast%3DHopt%26aufirst%3DU.%2BT.%26aulast%3DBogatyreva%26aufirst%3DL.%26aulast%3DHauschke%26aufirst%3DD.%26aulast%3DWerner%26aufirst%3DM.%26aulast%3DLassmann%26aufirst%3DS.%26atitle%3DHistone%2520Modifiers%2520and%2520Marks%2520Define%2520Heterogeneous%2520Groups%2520of%2520Colorectal%2520Carcinomas%2520and%2520Affect%2520Responses%2520to%2520Hdac%2520Inhibitors%2520in%2520Vitro%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2016%26volume%3D6%26spage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakakibara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecke, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, A. A.</span></span> <span> </span><span class="NLM_article-title">Brd4 Is Displaced from Hpv Replication Factories as They Expand and Amplify Viral DNA</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003777</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1003777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1371%2Fjournal.ppat.1003777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24278023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtV2nsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=N.+Sakakibaraauthor=D.+Chenauthor=M.+K.+Jangauthor=D.+W.+Kangauthor=H.+F.+Lueckeauthor=S.+Y.+Wuauthor=C.+M.+Chiangauthor=A.+A.+McBride&title=Brd4+Is+Displaced+from+Hpv+Replication+Factories+as+They+Expand+and+Amplify+Viral+DNA&doi=10.1371%2Fjournal.ppat.1003777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 is displaced from HPV replication factories as they expand and amplify viral DNA</span></div><div class="casAuthors">Sakakibara, Nozomi; Chen, Dan; Jang, Moon Kyoo; Kang, Dong Wook; Luecke, Hans F.; Wu, Shwu-Yuan; Chiang, Cheng-Ming; McBride, Alison A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e1003777/1-e1003777/23, 23 pp.</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Replication foci are generated by many viruses to conc. and localize viral DNA synthesis to specific regions of the cell.  Expression of the HPV16 E1 and E2 replication proteins in keratinocytes results in nuclear foci that recruit proteins assocd. with the host DNA damage response.  We show that the Brd4 protein localizes to these foci and is essential for their formation.  However, when E1 and E2 begin amplifying viral DNA, Brd4 is displaced from the foci and cellular factors assocd. with DNA synthesis and homologous recombination are recruited.  Differentiated HPV-infected keratinocytes form similar nuclear foci that contain amplifying viral DNA.  We compare the different foci and show that, while they have many characteristics in common, there is a switch between early Brd4-dependent foci and mature Brd4-independent replication foci.  However, HPV genomes encoding mutated E2 proteins that are unable to bind Brd4 can replicate and amplify the viral genome.  We propose that, while E1, E2 and Brd4 might bind host chromatin at early stages of infection, there is a temporal and functional switch at later stages and increased E1 and E2 levels promote viral DNA amplification, displacement of Brd4 and growth of a replication factory.  The concomitant DNA damage response recruits proteins required for DNA synthesis and repair, which could then be utilized for viral DNA replication.  Hence, while Brd4 can enhance replication by concg. viral processes in specific regions of the host nucleus, this interaction is not absolutely essential for HPV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFxSI-Fqa2XLVg90H21EOLACvtfcHk0lhLF2gAYPWIsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtV2nsr0%253D&md5=3538d6798e52d681e962ac4f080a6241</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003777%26sid%3Dliteratum%253Aachs%26aulast%3DSakakibara%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DKang%26aufirst%3DD.%2BW.%26aulast%3DLuecke%26aufirst%3DH.%2BF.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DMcBride%26aufirst%3DA.%2BA.%26atitle%3DBrd4%2520Is%2520Displaced%2520from%2520Hpv%2520Replication%2520Factories%2520as%2520They%2520Expand%2520and%2520Amplify%2520Viral%2520DNA%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.ppat.1003777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hossan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indenbirken, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipaleenmaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Batalla, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loges, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesse, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundhoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Protein Brd4 Is Required for Estrogen Receptor-Dependent Enhancer Activation and Gene Transcription</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.celrep.2014.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25017071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=460-469&author=S.+Nagarajanauthor=T.+Hossanauthor=M.+Alawiauthor=Z.+Najafovaauthor=D.+Indenbirkenauthor=U.+Bediauthor=H.+Taipaleenmakiauthor=I.+Ben-Batallaauthor=M.+Schellerauthor=S.+Logesauthor=S.+Knappauthor=E.+Hesseauthor=C.+M.+Chiangauthor=A.+Grundhoffauthor=S.+A.+Johnsen&title=Bromodomain+Protein+Brd4+Is+Required+for+Estrogen+Receptor-Dependent+Enhancer+Activation+and+Gene+Transcription&doi=10.1016%2Fj.celrep.2014.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain Protein BRD4 Is Required for Estrogen Receptor-Dependent Enhancer Activation and Gene Transcription</span></div><div class="casAuthors">Nagarajan, Sankari; Hossan, Tareq; Alawi, Malik; Najafova, Zeynab; Indenbirken, Daniela; Bedi, Upasana; Taipaleenmaeki, Hanna; Ben-Batalla, Isabel; Scheller, Marina; Loges, Sonja; Knapp, Stefan; Hesse, Eric; Chiang, Cheng-Ming; Grundhoff, Adam; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">460-469</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The estrogen receptor α (ERα) controls cell proliferation and tumorigenesis by recruiting various cofactors to estrogen response elements (EREs) to control gene transcription.  A deeper understanding of these transcriptional mechanisms may uncover therapeutic targets for ERα-dependent cancers.  We show that BRD4 regulates ERα-induced gene expression by affecting elongation-assocd. phosphorylation of RNA polymerase II (RNAPII) and histone H2B monoubiquitination.  Consistently, BRD4 activity is required for proliferation of ER+ breast and endometrial cancer cells and uterine growth in mice.  Genome-wide studies revealed an enrichment of BRD4 on transcriptional start sites of active genes and a requirement of BRD4 for H2B monoubiquitination in the transcribed region of estrogen-responsive genes.  Importantly, we demonstrate that BRD4 occupancy on distal EREs enriched for H3K27ac is required for recruitment and elongation of RNAPII on EREs and the prodn. of ERα-dependent enhancer RNAs.  These results uncover BRD4 as a central regulator of ERα function and potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0fNk5Ueff1bVg90H21EOLACvtfcHk0lgRVfqcA8xqng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM&md5=301a96567713aa777911a158736adb6d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DHossan%26aufirst%3DT.%26aulast%3DAlawi%26aufirst%3DM.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DIndenbirken%26aufirst%3DD.%26aulast%3DBedi%26aufirst%3DU.%26aulast%3DTaipaleenmaki%26aufirst%3DH.%26aulast%3DBen-Batalla%26aufirst%3DI.%26aulast%3DScheller%26aufirst%3DM.%26aulast%3DLoges%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHesse%26aufirst%3DE.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGrundhoff%26aufirst%3DA.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DBromodomain%2520Protein%2520Brd4%2520Is%2520Required%2520for%2520Estrogen%2520Receptor-Dependent%2520Enhancer%2520Activation%2520and%2520Gene%2520Transcription%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D460%26epage%3D469%26doi%3D10.1016%2Fj.celrep.2014.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simanski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span> <span> </span><span class="NLM_article-title">Brd4 Phosphorylation Regulates Hpv E2-Mediated Viral Transcription, Origin Replication, and Cellular Mmp-9 Expression</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1733</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.celrep.2016.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27477287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1733-1748&author=S.-Y.+Wuauthor=D.+S.+Ninauthor=A.-Y.+Leeauthor=S.+Simanskiauthor=T.+Kodadekauthor=C.-M.+Chiang&title=Brd4+Phosphorylation+Regulates+Hpv+E2-Mediated+Viral+Transcription%2C+Origin+Replication%2C+and+Cellular+Mmp-9+Expression&doi=10.1016%2Fj.celrep.2016.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Phosphorylation Regulates HPV E2-Mediated Viral Transcription, Origin Replication, and Cellular MMP-9 Expression</span></div><div class="casAuthors">Wu, Shwu-Yuan; Nin, Dawn Sijin; Lee, A-Young; Simanski, Scott; Kodadek, Thomas; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1733-1748</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Post-translational modification can modulate protein conformation and alter binding partner recruitment within gene regulatory regions.  Here, we report that bromodomain-contg. protein 4 (BRD4), a transcription co-factor and chromatin regulator, uses a phosphorylation-induced switch mechanism to recruit E2 protein encoded by cancer-assocd. human papillomavirus (HPV) to viral early gene and cellular matrix metalloproteinase-9 (MMP-9) promoters.  Enhanced MMP-9 expression, induced upon keratinocyte differentiation, occurs via BRD4-dependent recruitment of active AP-1 and NF-κB to their target sequences.  This is triggered by replacement of AP-1 family members JunB and JunD by c-Jun and by re-localization of NF-κB from the cytoplasm to the nucleus.  In addn., BRD4 phosphorylation is crit. for E2- and origin-dependent HPV DNA replication.  A class of phospho-BRD4-targeting compds., distinct from the BET bromodomain inhibitors, effectively blocks BRD4 phosphorylation-specific functions in transcription and factor recruitment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomvZx5RmFIRLVg90H21EOLACvtfcHk0lgRVfqcA8xqng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO&md5=10bde9118396d8a5a1f4614fb64a566c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DNin%26aufirst%3DD.%2BS.%26aulast%3DLee%26aufirst%3DA.-Y.%26aulast%3DSimanski%26aufirst%3DS.%26aulast%3DKodadek%26aufirst%3DT.%26aulast%3DChiang%26aufirst%3DC.-M.%26atitle%3DBrd4%2520Phosphorylation%2520Regulates%2520Hpv%2520E2-Mediated%2520Viral%2520Transcription%252C%2520Origin%2520Replication%252C%2520and%2520Cellular%2520Mmp-9%2520Expression%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D16%26spage%3D1733%26epage%3D1748%26doi%3D10.1016%2Fj.celrep.2016.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamdan, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indenbirken, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundhoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheel, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegwitz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Brd4 Promotes P63 and Grhl3 Expression Downstream of Foxo in Mammary Epithelial Cells</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">3130</span>– <span class="NLM_lpage">3145</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1093%2Fnar%2Fgkw1276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27980063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1WmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=3130-3145&author=S.+Nagarajanauthor=U.+Bediauthor=A.+Budidaauthor=F.+H.+Hamdanauthor=V.+K.+Mishraauthor=Z.+Najafovaauthor=W.+H.+Xieauthor=M.+Alawiauthor=D.+Indenbirkenauthor=S.+Knappauthor=C.+M.+Chiangauthor=A.+Grundhoffauthor=V.+Kariauthor=C.+H.+Scheelauthor=F.+Wegwitzauthor=S.+A.+Johnsen&title=Brd4+Promotes+P63+and+Grhl3+Expression+Downstream+of+Foxo+in+Mammary+Epithelial+Cells&doi=10.1093%2Fnar%2Fgkw1276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 promotes p63 and GRHL3 expression downstream of FOXO in mammary epithelial cells</span></div><div class="casAuthors">Nagarajan, Sankari; Bedi, Upasana; Budida, Anusha; Hamdan, Feda H.; Mishra, Vivek Kumar; Najafova, Zeynab; Xie, Wanhua; Alawi, Malik; Indenbirken, Daniela; Knapp, Stefan; Chiang, Cheng-Ming; Grundhoff, Adam; Kari, Vijayalakshmi; Scheel, Christina H.; Wegwitz, Florian; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3130-3145</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) is a member of the bromo- and extraterminal (BET) domain-contg. family of epigenetic readers which is under intensive investigation as a target for anti-tumor therapy.  BRD4 plays a central role in promoting the expression of select subsets of genes including many driven by oncogenic transcription factors and signaling pathways.  However, the role of BRD4 and the effects of BET inhibitors in non-transformed cells remain mostly unclear.  We demonstrate that BRD4 is required for the maintenance of a basal epithelial phenotype by regulating the expression of epithelial-specific genes including TP63 and Grainy Head-like transcription factor-3 (GRHL3) in non-transformed basal-like mammary epithelial cells.  Moreover, BRD4 occupancy correlates with enhancer activity and enhancer RNA (eRNA) transcription.  Motif analyses of cell context-specific BRD4-enriched regions predicted the involvement of FOXO transcription factors.  Consistently, activation of FOXO1 function via inhibition of EGFR-AKT signaling promoted the expression of TP63 and GRHL3.  Moreover, activation of Src kinase signaling and FOXO1 inhibition decreased the expression of FOXO/BRD4 target genes.  Together, our findings support a function for BRD4 in promoting basal mammary cell epithelial differentiation, at least in part, by regulating FOXO factor function on enhancers to activate TP63 and GRHL3 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8CeR-xfSherVg90H21EOLACvtfcHk0lgRNIXBO10cRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1WmtLs%253D&md5=051ed118e27e9ae51f2cfb7058c31d21</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1276%26sid%3Dliteratum%253Aachs%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DBedi%26aufirst%3DU.%26aulast%3DBudida%26aufirst%3DA.%26aulast%3DHamdan%26aufirst%3DF.%2BH.%26aulast%3DMishra%26aufirst%3DV.%2BK.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DW.%2BH.%26aulast%3DAlawi%26aufirst%3DM.%26aulast%3DIndenbirken%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGrundhoff%26aufirst%3DA.%26aulast%3DKari%26aufirst%3DV.%26aulast%3DScheel%26aufirst%3DC.%2BH.%26aulast%3DWegwitz%26aufirst%3DF.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DBrd4%2520Promotes%2520P63%2520and%2520Grhl3%2520Expression%2520Downstream%2520of%2520Foxo%2520in%2520Mammary%2520Epithelial%2520Cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D3130%26epage%3D3145%26doi%3D10.1093%2Fnar%2Fgkw1276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delcuratolo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fertey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuetz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiprecht, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudjetz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubenrauch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iftner, T.</span></span> <span> </span><span class="NLM_article-title">Papillomavirus-Associated Tumor Formation Critically Depends on C-Fos Expression Induced by Viral Protein E2 and Bromodomain Protein Brd4</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005366</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1371%2Fjournal.ppat.1005366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26727473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSrur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=M.+Delcuratoloauthor=J.+Ferteyauthor=M.+Schneiderauthor=J.+Schuetzauthor=N.+Leiprechtauthor=B.+Hudjetzauthor=S.+Brodbeckauthor=S.+Corallauthor=M.+Dreerauthor=R.+M.+Schwabauthor=M.+Grimmauthor=S.-Y.+Wuauthor=F.+Stubenrauchauthor=C.-M.+Chiangauthor=T.+Iftner&title=Papillomavirus-Associated+Tumor+Formation+Critically+Depends+on+C-Fos+Expression+Induced+by+Viral+Protein+E2+and+Bromodomain+Protein+Brd4&doi=10.1371%2Fjournal.ppat.1005366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Papillomavirus-associated tumor formation critically depends on c-Fos expression induced by viral protein E2 and bromodomain protein Brd4</span></div><div class="casAuthors">Delcuratolo, Maria; Fertey, Jasmin; Schneider, Markus; Schuetz, Johanna; Leiprecht, Natalie; Hudjetz, Benjamin; Brodbeck, Stephan; Corall, Silke; Dreer, Marcel; Schwab, Roxana Michaela; Grimm, Martin; Wu, Shwu-Yuan; Stubenrauch, Frank; Chiang, Cheng-Ming; Iftner, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e1005366/1-e1005366/22</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">We investigated the mechanism of how the papillomavirus E2 transcription factor can activate promoters through activator protein (AP)1 binding sites.  Using an unbiased approach with an inducible cell line expressing the viral transcription factor E2 and transcriptome anal., we found that E2 induces the expression of the two AP1 components c-Fos and FosB in a Brd4-dependent manner.  In vitro RNA interference confirmed that c-Fos is one of the AP1 members driving the expression of viral oncogenes E6/E7.  Mutation anal. and in vivo RNA interference identified an essential role for c-Fos/AP1 and also for the bromodomain protein Brd4 for papillomavirus-induced tumorigenesis.  Lastly, chromatin immunopptn. anal. demonstrated that E2 binds together with Brd4 to a canonical E2 binding site (E2BS) in the promoter of c-Fos, thus activating c-Fos expression.  Thus, we identified a novel way how E2 activates the viral oncogene promoter and show that E2 may act as a viral oncogene by direct activation of c-Fos involved in skin tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6HcuABW6FLVg90H21EOLACvtfcHk0lhH4UDlCuBbJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSrur4%253D&md5=3a32f66d3560dabdd61bb8f39ec4c978</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005366%26sid%3Dliteratum%253Aachs%26aulast%3DDelcuratolo%26aufirst%3DM.%26aulast%3DFertey%26aufirst%3DJ.%26aulast%3DSchneider%26aufirst%3DM.%26aulast%3DSchuetz%26aufirst%3DJ.%26aulast%3DLeiprecht%26aufirst%3DN.%26aulast%3DHudjetz%26aufirst%3DB.%26aulast%3DBrodbeck%26aufirst%3DS.%26aulast%3DCorall%26aufirst%3DS.%26aulast%3DDreer%26aufirst%3DM.%26aulast%3DSchwab%26aufirst%3DR.%2BM.%26aulast%3DGrimm%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DStubenrauch%26aufirst%3DF.%26aulast%3DChiang%26aufirst%3DC.-M.%26aulast%3DIftner%26aufirst%3DT.%26atitle%3DPapillomavirus-Associated%2520Tumor%2520Formation%2520Critically%2520Depends%2520on%2520C-Fos%2520Expression%2520Induced%2520by%2520Viral%2520Protein%2520E2%2520and%2520Bromodomain%2520Protein%2520Brd4%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iftner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haedicke-Jarboui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span> <span> </span><span class="NLM_article-title">Involvement of Brd4 in Different Steps of the Papillomavirus Life Cycle</span>. <i>Virus Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>231</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.virusres.2016.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.virusres.2016.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27965149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWnt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=231&publication_year=2017&pages=76-82&author=T.+Iftnerauthor=J.+Haedicke-Jarbouiauthor=S.-Y.+Wuauthor=C.-M.+Chiang&title=Involvement+of+Brd4+in+Different+Steps+of+the+Papillomavirus+Life+Cycle&doi=10.1016%2Fj.virusres.2016.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Involvement of Brd4 in different steps of the papillomavirus life cycle</span></div><div class="casAuthors">Iftner, Thomas; Haedicke-Jarboui, Juliane; Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">231</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-82</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Bromodomain-contg. protein 4 (Brd4) is a cellular chromatin-binding factor and transcriptional regulator that recruits sequence-specific transcription factors and chromatin modulators to control target gene transcription.  Papillomaviruses (PVs) have evolved to hijack Brd4's activity in order to create a facilitating environment for the viral life cycle.  Brd4, in assocn. with the major viral regulatory protein E2, is involved in multiple steps of the PV life cycle including replication initiation, viral gene transcription, and viral genome segregation and maintenance.  Phosphorylation of Brd4, regulated by casein kinase II (CK2) and protein phosphatase 2A (PP2A), is crit. for viral gene transcription as well as E1- and E2-dependent origin replication.  Thus, pharmacol. agents regulating Brd4 phosphorylation and inhibitors blocking phospho-Brd4 functions are promising candidates for therapeutic intervention in treating human papillomavirus (HPV) infections as well as assocd. disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp07jGfzmJg9rVg90H21EOLACvtfcHk0lhH4UDlCuBbJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWnt7rO&md5=2f9b41f45d836350dd1ce07981f0d0e9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.virusres.2016.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virusres.2016.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DIftner%26aufirst%3DT.%26aulast%3DHaedicke-Jarboui%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26atitle%3DInvolvement%2520of%2520Brd4%2520in%2520Different%2520Steps%2520of%2520the%2520Papillomavirus%2520Life%2520Cycle%26jtitle%3DVirus%2520Res.%26date%3D2017%26volume%3D231%26spage%3D76%26epage%3D82%26doi%3D10.1016%2Fj.virusres.2016.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Development of N-Hydroxycinnamamide-Based Hdac Inhibitors with Improved Hdac Inhibitory Activity and in Vitro Antitumor Activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2666</span>– <span class="NLM_lpage">2675</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmc.2016.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28336407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVSlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2666-2675&author=J.+Zangauthor=B.+Shiauthor=X.+Liangauthor=Q.+Gaoauthor=W.+Xuauthor=Y.+Zhang&title=Development+of+N-Hydroxycinnamamide-Based+Hdac+Inhibitors+with+Improved+Hdac+Inhibitory+Activity+and+in+Vitro+Antitumor+Activity&doi=10.1016%2Fj.bmc.2016.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Development of N-hydroxycinnamamide-based HDAC inhibitors with improved HDAC inhibitory activity and in vitro antitumor activity</span></div><div class="casAuthors">Zang, Jie; Shi, Baowen; Liang, Xuewu; Gao, Qianwen; Xu, Wenfang; Zhang, Yingjie</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2666-2675</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylase inhibitors (HDACIs) are promising in the treatment of various diseases, among which cancer treatment has achieved the most success.  We have previously developed series of HDACIs combining N-hydroxycinnamamide bioactive fragment and indole bioactive fragment, which showed moderate to potent antitumor activities.  Herein, further structural derivatization based on our previous structure-activity relationship (SAR) got 25 novel compds.  Most compds. showed much more potent histone deacetylases (HDACs) inhibitory activity than their parent compd. 1 and even the pos. control SAHA.  What's more, compared with the approved HDACs inhibitor SAHA, compds. 6i, 6k, 6q and 6t displayed better in vitro antiproliferation against multiple tumor cell lines.  It is worth noting that though the 4-hydroxycinnamic acid-based compd. 2 showed HDAC1/3 dual selectivity, its 4-hydroxy-3-methoxycinnamic acid-based analog 6t turned out to be a pan-HDACs inhibitor as SAHA, indicating that the 3-methoxy group on the N-hydroxycinnamamide fragment could dramatically influence the HDACs isoform selectivity of this series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolb8cpswLOMrVg90H21EOLACvtfcHk0lgxBn_Kp17L0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVSlur8%253D&md5=8ce20c262f96079fa20d9e48ddc02a3b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520N-Hydroxycinnamamide-Based%2520Hdac%2520Inhibitors%2520with%2520Improved%2520Hdac%2520Inhibitory%2520Activity%2520and%2520in%2520Vitro%2520Antitumor%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2666%26epage%3D2675%26doi%3D10.1016%2Fj.bmc.2016.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inks, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Selective Hdac Inhibitors with Potent Oral Activity against Leukemia and Colorectal Cancer: Design, Structure-Activity Relationship and Anti-Tumor Activity Study</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejmech.2017.03.069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28415009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFWnt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2017&pages=185-206&author=X.+Liauthor=Y.+Zhangauthor=Y.+Jiangauthor=J.+Wuauthor=E.+S.+Inksauthor=C.+J.+Chouauthor=S.+Gaoauthor=J.+Houauthor=Q.+Dingauthor=J.+Liauthor=X.+Wangauthor=Y.+Huangauthor=W.+Xu&title=Selective+Hdac+Inhibitors+with+Potent+Oral+Activity+against+Leukemia+and+Colorectal+Cancer%3A+Design%2C+Structure-Activity+Relationship+and+Anti-Tumor+Activity+Study&doi=10.1016%2Fj.ejmech.2017.03.069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selective HDAC inhibitors with potent oral activity against leukemia and colorectal cancer: Design, structure-activity relationship and anti-tumor activity study</span></div><div class="casAuthors">Li, Xiaoyang; Zhang, Yingjie; Jiang, Yuqi; Wu, Jingde; Inks, Elizabeth S.; Chou, C. James; Gao, Shuai; Hou, Jinning; Ding, Qinge; Li, Jingyao; Wang, Xue; Huang, Yongxue; Xu, Wenfang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-206</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Previously, we reported the discovery of a series of N-hydroxycinnamamide-based HDAC inhibitors, among which compd. 11y exhibited high HDAC1/3 selectivity.  In this current study, structural derivatization of 11y led to a new series of benzamide based HDAC inhibitors.  Most of the compds. exhibited high HDACs inhibitory potency.  Compd. 11a (with 4-methoxybenzoyl as N-substituent in the cap and 4-(aminomethyl) benzoyl as the linker group) exhibited selectivity against HDAC1 to some extent, and showed potent antiproliferative activity against several tumor cell lines.  In vivo studies revealed that compd. 11a displayed potent oral antitumor activity in both hematol. tumor cell U937 xenograft model and solid tumor cell HCT116 xenograft model with no obvious toxicity.  Further modification of benzamide 3, 11a and 19 afforded new thienyl and Ph compds. (50a, 50b, 63a, 63b and 63c) with dramatic HDAC1 and HDAC2 dual selectivity, and the fluorine contg. compd. 56, with moderate HDAC3 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvK_Gdgx31-LVg90H21EOLACvtfcHk0lgxBn_Kp17L0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFWnt70%253D&md5=66ad0ddda5b3e46ef07dfb40951b63db</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.069%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DInks%26aufirst%3DE.%2BS.%26aulast%3DChou%26aufirst%3DC.%2BJ.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DSelective%2520Hdac%2520Inhibitors%2520with%2520Potent%2520Oral%2520Activity%2520against%2520Leukemia%2520and%2520Colorectal%2520Cancer%253A%2520Design%252C%2520Structure-Activity%2520Relationship%2520and%2520Anti-Tumor%2520Activity%2520Study%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D134%26spage%3D185%26epage%3D206%26doi%3D10.1016%2Fj.ejmech.2017.03.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Biological Evaluation of Quinoline Derivatives as Hdac Class I Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejmech.2017.03.064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28371677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVCit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=11-23&author=C.+Chenauthor=X.+Houauthor=G.+Wangauthor=W.+Panauthor=X.+Yangauthor=Y.+Zhangauthor=H.+Fang&title=Design%2C+Synthesis+and+Biological+Evaluation+of+Quinoline+Derivatives+as+Hdac+Class+I+Inhibitors&doi=10.1016%2Fj.ejmech.2017.03.064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of quinoline derivatives as HDAC class I inhibitors</span></div><div class="casAuthors">Chen, Chen; Hou, Xuben; Wang, Guohua; Pan, Wenyan; Yang, Xinying; Zhang, Yingkai; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-23</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Based on pharmacophore model of HDAC inhibitors, a series of quinoline-based N-hydroxyacrylamides I [R1 = 4-N-hydroxyacrylamide, 3-N-hydroxyacrylamide; R2 = H, OMe, Cl, Br,; R3 = H, OMe, Cl; R4 = H, hexyl, Bn] and N-hydroxybenzamides II [R1 = H, Bn; R2 = H, OMe, Br; R3 = H, Cl] were designed and synthesized as potent HDAC inhibitors.  All target compds. were evaluated for their in vitro HDAC inhibitory activities and anti-proliferative activities and the best compd. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4= H] surpass Vorinostat in both enzymic inhibitory activity and cellular anti-proliferative activity.  In terms of HDAC isoforms selectivity, compd. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4= H] exhibited preferable inhibition for class I HDACs, esp. for HDAC8, the IC50 value (442 nM) was much lower than that of Vorinostat (7468 nM).  Subsequently, class I & IIa HDACs whole cell enzyme assay to evaluate inhibitory activity in whole cell context were performed.  Compds. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4 = H; R1 = 4-N-hydroxyacrylamide, R2, R4 = H, R3 = Cl] displayed much better cellular activity for class I HDACs than that for class IIa HDACs, which indicated that compds. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4 = H; R1 = 4-N-hydroxyacrylamide, R2, R4 = H, R3 = Cl] might be potent class I HDAC inhibitors.  Meanwhile, flow cytometry anal. showed that compds. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4 = H; R1 = 4-N-hydroxyacrylamide, R2, R4 = H, R3 = Cl] promoted cell apoptosis in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeuPe9e4YS9rVg90H21EOLACvtfcHk0li0ILolc5y0Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVCit7s%253D&md5=948aeb076d296895955ba4845240b596</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.064%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Quinoline%2520Derivatives%2520as%2520Hdac%2520Class%2520I%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D133%26spage%3D11%26epage%3D23%26doi%3D10.1016%2Fj.ejmech.2017.03.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Preliminary Bioactivity Evaluations of Substituted Quinoline Hydroxamic Acid Derivatives as Novel Histone Deacetylase (Hdac) Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4364</span>– <span class="NLM_lpage">4374</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmc.2015.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26149591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2itrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=4364-4374&author=L.+Wangauthor=X.+Houauthor=H.+Fuauthor=X.+Panauthor=W.+Xuauthor=W.+Tangauthor=H.+Fang&title=Design%2C+Synthesis+and+Preliminary+Bioactivity+Evaluations+of+Substituted+Quinoline+Hydroxamic+Acid+Derivatives+as+Novel+Histone+Deacetylase+%28Hdac%29+Inhibitors&doi=10.1016%2Fj.bmc.2015.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and preliminary bioactivity evaluations of substituted quinoline hydroxamic acid derivatives as novel histone deacetylase (HDAC) inhibitors</span></div><div class="casAuthors">Wang, Lei; Hou, Xuben; Fu, Huansheng; Pan, Xiaole; Xu, Wenfang; Tang, Weiping; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4364-4374</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Inhibition of HDACs activity has become a promising therapeutic strategy in clin. practice to reverse the abnormal epigenetic states of cancer and other diseases.  Therefore, HDAC inhibitors become a relatively new class of anti-cancer agent.  In the present study, we reported the design and synthesis of a series of novel HDAC inhibitors using various substituted quinoline rings as the cap group.  In vitro studies showed that some compds. have good inhibitory activities against HDACs and potent antiproliferative activities in some tumor cell lines.  Esp., compd. I (IC50 = 85 nM), exhibited better inhibitory effect compared with SAHA (IC50 = 161 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs4Se1Xd6Yp7Vg90H21EOLACvtfcHk0li0ILolc5y0Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2itrnI&md5=d0c9af14949d22036161a30c27288e24</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Preliminary%2520Bioactivity%2520Evaluations%2520of%2520Substituted%2520Quinoline%2520Hydroxamic%2520Acid%2520Derivatives%2520as%2520Novel%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D4364%26epage%3D4374%26doi%3D10.1016%2Fj.bmc.2015.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Preliminary Bioactivity Studies of Substituted Purine Hydroxamic Acid Derivatives as Novel Histone Deacetylase (Hdac) Inhibitors</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1887</span>– <span class="NLM_lpage">1891</span>, <span class="refDoi"> DOI: 10.1039/c4md00203b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1039%2Fc4md00203b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFeqsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1887-1891&author=J.+Wangauthor=F.+E.+Sunauthor=L.+Hanauthor=X.+Houauthor=X.+Panauthor=R.+Liuauthor=W.+Tangauthor=H.+Fang&title=Design%2C+Synthesis%2C+and+Preliminary+Bioactivity+Studies+of+Substituted+Purine+Hydroxamic+Acid+Derivatives+as+Novel+Histone+Deacetylase+%28Hdac%29+Inhibitors&doi=10.1039%2Fc4md00203b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and preliminary bioactivity studies of substituted purine hydroxamic acid derivatives as novel histone deacetylase (HDAC) inhibitors</span></div><div class="casAuthors">Wang, Junhua; Sun, Feng'e; Han, Leiqiang; Hou, Xuben; Pan, Xiaole; Liu, Renshuai; Tang, Weiping; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1887-1891</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) is a clin. validated target for anti-tumor therapy.  In order to increase HDAC inhibition and efficiency, we developed a series of novel substituted purine hydroxamic acids as potent HDAC inhibitors.  The biol. evaluation suggests that compd. 5r (IC50 = 0.075 μmol L-1) exhibits better HDAC1 and 2 inhibitory activity compared to the approved drug SAHA (IC50 = 0.14 μmol L-1).  Further biol. evaluation indicated that compds. 5r, 5w, and 5x have potent anti-proliferative activities against eight tumor cells, including MDA-MB231, KG1, PC3, U937 and so on.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrskovjc6x8l7Vg90H21EOLACvtfcHk0lgGlXsIQmAtHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFeqsrbL&md5=f8d472e0691edd7d0501880068c32720</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2Fc4md00203b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4md00203b%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DF.%2BE.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Preliminary%2520Bioactivity%2520Studies%2520of%2520Substituted%2520Purine%2520Hydroxamic%2520Acid%2520Derivatives%2520as%2520Novel%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Inhibitors%26jtitle%3DMedchemcomm%26date%3D2014%26volume%3D5%26spage%3D1887%26epage%3D1891%26doi%3D10.1039%2Fc4md00203b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Improved Antiproliferative Activity of 1,3,4-Thiadiazole-Containing Histone Deacetylase (Hdac) Inhibitors by Introduction of the Heteroaromatic Surface Recognition Motif</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5766</span>– <span class="NLM_lpage">5775</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2014.09.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmc.2014.09.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25311567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KltLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=5766-5775&author=P.+Guanauthor=L.+Wangauthor=X.+Houauthor=Y.+Wanauthor=W.+Xuauthor=W.+Tangauthor=H.+Fang&title=Improved+Antiproliferative+Activity+of+1%2C3%2C4-Thiadiazole-Containing+Histone+Deacetylase+%28Hdac%29+Inhibitors+by+Introduction+of+the+Heteroaromatic+Surface+Recognition+Motif&doi=10.1016%2Fj.bmc.2014.09.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Improved antiproliferative activity of 1,3,4-thiadiazole-containing histone deacetylase (HDAC) inhibitors by introduction of the heteroaromatic surface recognition motif</span></div><div class="casAuthors">Guan, Peng; Wang, Lei; Hou, Xuben; Wan, Yichao; Xu, Wenfang; Tang, Weiping; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5766-5775</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of 1,3,4-thiadiazole-contg. hydroxamic acids, in accord with the common pharmacophore of histone deacetylase (HDAC) inhibitors (a Zn2+ binding moiety-a linker-a surface recognition motif), was identified as submicromolar HDAC inhibitors by the authors' group.  In this study, the authors continued the efforts to develop 1,3,4-thiadiazole bearing hydroxamate analogs by modifying the surface recognition motif.  The authors found that 1,3,4-thiadiazoles having a heteroarom. substituent showed better HDAC inhibitory activity in enzymic assay and higher antiproliferative potency in cellular assay compared to SAHA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeTGSTrYDTiLVg90H21EOLACvtfcHk0lgGlXsIQmAtHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KltLfO&md5=41e9d828f2c5b03462245918bede4b46</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.09.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.09.039%26sid%3Dliteratum%253Aachs%26aulast%3DGuan%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DImproved%2520Antiproliferative%2520Activity%2520of%25201%252C3%252C4-Thiadiazole-Containing%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Inhibitors%2520by%2520Introduction%2520of%2520the%2520Heteroaromatic%2520Surface%2520Recognition%2520Motif%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D5766%26epage%3D5775%26doi%3D10.1016%2Fj.bmc.2014.09.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W. F.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Angiogenesis by Hdac Inhibitor Zyj-34c in Vitro</span>. <i>Acta Pharmacol. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">65</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=65&author=Y.+P.+Jiaauthor=D.+S.+Zhouauthor=Y.+J.+Zhangauthor=W.+F.+Xu&title=Inhibition+of+Angiogenesis+by+Hdac+Inhibitor+Zyj-34c+in+Vitro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%2BP.%26aulast%3DZhou%26aufirst%3DD.%2BS.%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DXu%26aufirst%3DW.%2BF.%26atitle%3DInhibition%2520of%2520Angiogenesis%2520by%2520Hdac%2520Inhibitor%2520Zyj-34c%2520in%2520Vitro%26jtitle%3DActa%2520Pharmacol.%2520Sin.%26date%3D2013%26volume%3D34%26spage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazarika, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiao, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricker, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, S. R.</span></span> <span> </span><span class="NLM_article-title">Phase 2 Trial of Oral Vorinostat (Suberoylanilide Hydroxamic Acid, Saha) for Refractory Cutaneous T-Cell Lymphoma (Ctcl)</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1182/blood-2006-06-025999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1182%2Fblood-2006-06-025999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=16960145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivVyrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2007&pages=31-39&author=M.+Duvicauthor=R.+Talpurauthor=X.+Niauthor=C.+Zhangauthor=P.+Hazarikaauthor=C.+Kellyauthor=J.+H.+Chiaoauthor=J.+F.+Reillyauthor=J.+L.+Rickerauthor=V.+M.+Richonauthor=S.+R.+Frankel&title=Phase+2+Trial+of+Oral+Vorinostat+%28Suberoylanilide+Hydroxamic+Acid%2C+Saha%29+for+Refractory+Cutaneous+T-Cell+Lymphoma+%28Ctcl%29&doi=10.1182%2Fblood-2006-06-025999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Phase 2 trial of oral vorinostat (suberoylanilide hydroxamic acid, SAHA) for refractory cutaneous T-cell lymphoma (CTCL)</span></div><div class="casAuthors">Duvic, Madeleine; Talpur, Rakshandra; Ni, Xiao; Zhang, Chunlei; Hazarika, Parul; Kelly, Cecilia; Chiao, Judy H.; Reilly, John F.; Ricker, Justin L.; Richon, Victoria M.; Frankel, Stanley R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-39</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The activity and safety of the histone deacetylase inhibitor vorinostat (suberoylanilide hydroxamic acid, SAHA) were evaluated in patients with refractory cutaneous T-cell lymphoma (CTCL).  Group 1 received vorinostat 400 mg daily, group 2 received vorinostat 300 mg twice daily for 3 days with 4 days rest, and group 3 received vorinostat 300 mg twice daily for 14 days with 7 days rest followed by 200 mg twice daily.  Treatment continued until disease progression or intolerable toxicity.  The primary objective was to det. the complete and partial response (PR) rate.  Time to response (TTR), time to progressive disease (TTP), response duration (DOR), pruritus relief, and safety were detd.  Thirty-three patients who had received a median of 5 prior therapies were enrolled.  Eight patients achieved a PR, including 7 with advanced disease and 4 with Sezary syndrome.  The median TTR, DOR, and TTP for responders were 11.9, 15.1, and 30.2 wk, resp.  Fourteen of 31 evaluable patients had pruritus relief.  The most common drug-related AEs were fatigue, thrombocytopenia, diarrhea, and nausea.  The most common grade 3 or 4 drug-related AEs were thrombocytopenia and dehydration.  Vorinostat demonstrated activity in heavily pretreated patients with CTCL.  The 400 mg daily regimen had the most favorable safety profile and is being further evaluated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo80_rNvPEEgLVg90H21EOLACvtfcHk0lhasZ4Nuqs8pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivVyrt7g%253D&md5=b4c8b3c20213eaf692bd9b44266d045f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1182%2Fblood-2006-06-025999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2006-06-025999%26sid%3Dliteratum%253Aachs%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DTalpur%26aufirst%3DR.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHazarika%26aufirst%3DP.%26aulast%3DKelly%26aufirst%3DC.%26aulast%3DChiao%26aufirst%3DJ.%2BH.%26aulast%3DReilly%26aufirst%3DJ.%2BF.%26aulast%3DRicker%26aufirst%3DJ.%2BL.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DFrankel%26aufirst%3DS.%2BR.%26atitle%3DPhase%25202%2520Trial%2520of%2520Oral%2520Vorinostat%2520%2528Suberoylanilide%2520Hydroxamic%2520Acid%252C%2520Saha%2529%2520for%2520Refractory%2520Cutaneous%2520T-Cell%2520Lymphoma%2520%2528Ctcl%2529%26jtitle%3DBlood%26date%3D2007%26volume%3D109%26spage%3D31%26epage%3D39%26doi%3D10.1182%2Fblood-2006-06-025999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span> <span> </span><span class="NLM_article-title">Development of the Pan-Dac Inhibitor Panobinostat (Lbh589): Successes and Challenges</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2009.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.canlet.2009.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=19344997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2009&pages=233-241&author=P.+Atadja&title=Development+of+the+Pan-Dac+Inhibitor+Panobinostat+%28Lbh589%29%3A+Successes+and+Challenges&doi=10.1016%2Fj.canlet.2009.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the pan-DAC inhibitor panobinostat (LBH589): Successes and challenges</span></div><div class="casAuthors">Atadja, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-241</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  The histone deacetylase (HDAC) inhibitors are emerging as a highly useful class of anticancer agents that inhibit the enzyme HDAC involved in the deacetylation of histone and non-histone cellular proteins.  The HDAC inhibitor, panobinostat (LBH589, Novartis Pharmaceuticals), achieves potent inhibition of all HDAC enzymes implicated in cancer and has demonstrated potent anti-tumor activity in preclin. models and promising clin. efficacy in cancer patients.  In this review we discuss the successes and challenges surrounding the development of panobinostat, focusing on its proposed mechanism of action, preclin. anti-tumor activity, and early clin. efficacy in hematol. and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSTYxNKUMt7bVg90H21EOLACvtfcHk0lhasZ4Nuqs8pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D&md5=4c487a2f8dc13f72cecf105d9fae97d9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2009.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2009.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DAtadja%26aufirst%3DP.%26atitle%3DDevelopment%2520of%2520the%2520Pan-Dac%2520Inhibitor%2520Panobinostat%2520%2528Lbh589%2529%253A%2520Successes%2520and%2520Challenges%26jtitle%3DCancer%2520Lett.%26date%3D2009%26volume%3D280%26spage%3D233%26epage%3D241%26doi%3D10.1016%2Fj.canlet.2009.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bociek, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neelapu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buglio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besterman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drouin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulus, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martell, R. E.</span></span> <span> </span><span class="NLM_article-title">Mocetinostat for Relapsed Classical Hodgkin’s Lymphoma: An Open-Label, Single-Arm, Phase 2 Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1222</span>– <span class="NLM_lpage">1228</span>, <span class="refDoi"> DOI: 10.1016/s1470-2045(11)70265-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fs1470-2045%2811%2970265-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=22033282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFGjt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=1222-1228&author=A.+Younesauthor=Y.+Okiauthor=R.+G.+Bociekauthor=J.+Kuruvillaauthor=M.+Fanaleauthor=S.+Neelapuauthor=A.+Copelandauthor=D.+Buglioauthor=A.+Galalauthor=J.+Bestermanauthor=Z.+Liauthor=M.+Drouinauthor=T.+Pattersonauthor=M.+R.+Wardauthor=J.+K.+Paulusauthor=Y.+Jiauthor=L.+J.+Medeirosauthor=R.+E.+Martell&title=Mocetinostat+for+Relapsed+Classical+Hodgkin%E2%80%99s+Lymphoma%3A+An+Open-Label%2C+Single-Arm%2C+Phase+2+Trial&doi=10.1016%2Fs1470-2045%2811%2970265-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Mocetinostat for relapsed classical Hodgkin's lymphoma: an open-label, single-arm, phase 2 trial</span></div><div class="casAuthors">Younes, Anas; Oki, Yasuhiro; Bociek, R. Gregory; Kuruvilla, John; Fanale, Michelle; Neelapu, Sattva; Copeland, Amanda; Buglio, Daniela; Galal, Ahmed; Besterman, Jeffrey; Li, Zuomei; Drouin, Michel; Patterson, Tracy; Ward, M. Renee; Paulus, Jessica K.; Ji, Yuan; Medeiros, L. Jeffrey; Martell, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1222-1228</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Summary: Background: The prognosis of patients with relapsed Hodgkin's lymphoma, esp. those who relapse after stem-cell transplantation, is poor, and the development of new agents for this patient population is an unmet medical need.  We tested the safety and efficacy of mocetinostat, an oral isotype-selective histone deacetylase inhibitor, in patients with relapsed classical Hodgkin's lymphoma.  Methods: Patients with relapsed or refractory classical Hodgkin's lymphoma aged 18 years or older were treated with mocetinostat administered orally three times per wk, in 28-day cycles.  Two doses were assessed (85 mg and 110 mg).  Patients were treated until disease progression or prohibitive toxicity.  The primary outcome was disease control rate, defined as complete response, partial response, or stable disease (for at least six cycles), analyzed by intention to treat.  This trial has been completed and is registered with ClinicalTrials.gov, no. NCT00358982.  Findings: 51 patients were enrolled.  Initially, 23 patients were enrolled in the 110 mg cohort.  Subsequently, because toxicity-related dose redns. were necessary in the 110 mg cohort, we treated 28 addnl. patients with a dose of 85 mg.  On the basis of intent-to-treat anal., the disease control rate was 35% (eight of 23 patients) in the 110 mg group and 25% (seven of 28) in the 85 mg group. 12 patients (24%) discontinued treatment because of adverse events, nine (32%) in the 85 mg cohort and three (13%) in the 110 mg cohort.  The most frequent treatment-related grade 3 and 4 adverse events were neutropenia (four patients [17%] in the 110 mg group, three [11%] in the 85 mg group); fatigue (five patients [22%] in the 110 mg group, three [11%] in the 85 mg group); and pneumonia (four patients [17%] in the 110 mg group, two [7%] in the 85 mg group).  Four patients, all in the 110 mg cohort, died during the study, of which two might have been related to treatment.  Interpretation: Mocetinostat, 85 mg three times per wk, has promising single-agent clin. activity with manageable toxicity in patients with relapsed classical Hodgkin's lymphoma.  Funding: MethylGene Inc, Montreal, Canada; Celgene Corporation, Summit, NJ, USA; Tufts Medical Center, Boston, MA, USA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3rbuk2PJOR7Vg90H21EOLACvtfcHk0lhS9PZmR0Bhvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFGjt77O&md5=c6f7748fe47bad8522fb16ef67195231</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fs1470-2045%2811%2970265-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1470-2045%252811%252970265-0%26sid%3Dliteratum%253Aachs%26aulast%3DYounes%26aufirst%3DA.%26aulast%3DOki%26aufirst%3DY.%26aulast%3DBociek%26aufirst%3DR.%2BG.%26aulast%3DKuruvilla%26aufirst%3DJ.%26aulast%3DFanale%26aufirst%3DM.%26aulast%3DNeelapu%26aufirst%3DS.%26aulast%3DCopeland%26aufirst%3DA.%26aulast%3DBuglio%26aufirst%3DD.%26aulast%3DGalal%26aufirst%3DA.%26aulast%3DBesterman%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DDrouin%26aufirst%3DM.%26aulast%3DPatterson%26aufirst%3DT.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DPaulus%26aufirst%3DJ.%2BK.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DMedeiros%26aufirst%3DL.%2BJ.%26aulast%3DMartell%26aufirst%3DR.%2BE.%26atitle%3DMocetinostat%2520for%2520Relapsed%2520Classical%2520Hodgkin%25E2%2580%2599s%2520Lymphoma%253A%2520An%2520Open-Label%252C%2520Single-Arm%252C%2520Phase%25202%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2011%26volume%3D12%26spage%3D1222%26epage%3D1228%26doi%3D10.1016%2Fs1470-2045%2811%2970265-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajji, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallenborg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlachos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyman, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermanson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, B.</span></span> <span> </span><span class="NLM_article-title">Combinatorial Action of the Hdac Inhibitor Trichostatin a and Etoposide Induces Caspase-Mediated Aif-Dependent Apoptotic Cell Death in Non-Small Cell Lung Carcinoma Cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3134</span>– <span class="NLM_lpage">3144</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1038%2Fsj.onc.1210976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=18071312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFSis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3134-3144&author=N.+Hajjiauthor=K.+Wallenborgauthor=P.+Vlachosauthor=U.+Nymanauthor=O.+Hermansonauthor=B.+Joseph&title=Combinatorial+Action+of+the+Hdac+Inhibitor+Trichostatin+a+and+Etoposide+Induces+Caspase-Mediated+Aif-Dependent+Apoptotic+Cell+Death+in+Non-Small+Cell+Lung+Carcinoma+Cells&doi=10.1038%2Fsj.onc.1210976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial action of the HDAC inhibitor trichostatin A and etoposide induces caspase-mediated AIF-dependent apoptotic cell death in non-small cell lung carcinoma cells</span></div><div class="casAuthors">Hajji, N.; Wallenborg, K.; Vlachos, P.; Nyman, U.; Hermanson, O.; Joseph, B.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3134-3144</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Commonly used regimens in cancer therapy rely on the induction of apoptotic cell death, and drug resistance can be attributed, at least in part, to a disabled apoptotic program.  Non-small cell lung carcinomas (NSCLC), exhibit an intrinsic resistance to chemotherapy.  Here, we show that co-treatment with etoposide (VP16) and the pan-histone deacetylase (HDAC) inhibitor trichostatin A (TSA), but not valproic acid (VPA), induced apoptotic cell death in drug-resistant NSCLC cells.  Co-treatment, but not single treatment, with VP16 and TSA induced apoptosis in a caspase-dependent manner accompanied by a crucial decrease in Bcl-xL expression allowing Bax activation and subsequent initiation of the apoptosis inducing factor (AIF)-dependent death pathway.  Importantly, AIF proved to be required for the effects of TSA/VP16 as RNA knockdown of AIF resulted in a complete abolishment of TSA/VP16-induced apoptotic cell death in drug-resistant NSCLC cells.  Our results thus provide evidence for the requirement of both caspase-dependent and caspase-independent apoptotic pathways in TSA/VP16-mediated death of drug-resistant NSCLC cells, and extend previous suggestions that HDAC inhibitors in combination with conventional chemotherapeutic drugs could be valuable in the treatment of NSCLC cancer and other malignancies in which Bcl-xL is overexpressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxjdh2HAy64LVg90H21EOLACvtfcHk0lhS9PZmR0Bhvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFSis7Y%253D&md5=1f21019a0c1b0cf7045831947c6e2144</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210976%26sid%3Dliteratum%253Aachs%26aulast%3DHajji%26aufirst%3DN.%26aulast%3DWallenborg%26aufirst%3DK.%26aulast%3DVlachos%26aufirst%3DP.%26aulast%3DNyman%26aufirst%3DU.%26aulast%3DHermanson%26aufirst%3DO.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DCombinatorial%2520Action%2520of%2520the%2520Hdac%2520Inhibitor%2520Trichostatin%2520a%2520and%2520Etoposide%2520Induces%2520Caspase-Mediated%2520Aif-Dependent%2520Apoptotic%2520Cell%2520Death%2520in%2520Non-Small%2520Cell%2520Lung%2520Carcinoma%2520Cells%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3134%26epage%3D3144%26doi%3D10.1038%2Fsj.onc.1210976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saijo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimodaira, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishioka, C.</span></span> <span> </span><span class="NLM_article-title">Biochemical, Biological and Structural Properties of Romidepsin (Fk228) and Its Analogs as Novel Hdac/Pi3k Dual Inhibitors</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1111/cas.12585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1111%2Fcas.12585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25492515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVyhurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2015&pages=208-215&author=K.+Saijoauthor=J.+Imamuraauthor=K.+Naritaauthor=A.+Odaauthor=H.+Shimodairaauthor=T.+Katohauthor=C.+Ishioka&title=Biochemical%2C+Biological+and+Structural+Properties+of+Romidepsin+%28Fk228%29+and+Its+Analogs+as+Novel+Hdac%2FPi3k+Dual+Inhibitors&doi=10.1111%2Fcas.12585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, biological and structural properties of romidepsin (FK228) and its analogs as novel HDAC/PI3K dual inhibitors</span></div><div class="casAuthors">Saijo, Ken; Imamura, Jin; Narita, Koichi; Oda, Akifumi; Shimodaira, Hideki; Katoh, Tadashi; Ishioka, Chikashi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-215</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Romidepsin (FK228, depsipeptide) is a potent histone deacetylase (HDAC) inhibitor that has FDA approval for the treatment of cutaneous and peripheral T-cell lymphomas.  We have previously reported that FK228 and its analogs have an addnl. activity as phosphatidylinositol 3-kinase (PI3K) inhibitors, and are defined as HDAC/PI3K dual inhibitors.  Because a combination of an HDAC inhibitor and a PI3K inhibitor induces apoptosis in human cancer cells in a synergistic manner, development of an HDAC/PI3K dual inhibitor will provide an attractive novel drug for cancer therapy.  Using structure-based optimization of the analogs, FK-A11 was identified as the most potent analog.  FK-A11 inhibited phosphorylation of AKT and accelerated histone acetylation at lower concns., resulting in stronger cytotoxic effects than FK228 and the other analogs in human cancer cells.  In this study, we have characterized the biochem., biol. and structural properties of FK228 analogs as PI3K inhibitors.  First, FK-A11 is an ATP competitive PI3K inhibitor.  Second, FK-A11 is a pan-p110 isoform inhibitor.  Third, FK-A11 selectively inhibits PI3K among 22 common cellular kinases.  Fourth, conformational changes of FK228 analogs by redn. of an internal disulfide bond have no effect on PI3K inhibitory activity, unlike HDAC inhibitory activity.  Finally, mol. modeling of PI3K-FK228 analogs and analyses of the binding affinities identified the structure that defines potency for PI3K inhibitory activity.  These results prove our concept that a series of FK228 analogs are HDAC/PI3K dual inhibitors.  These findings should help in the development of FK228 analogs as novel HDAC/PI3K dual inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Gsab8AjdsLVg90H21EOLACvtfcHk0ljeM1jMoyarFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVyhurw%253D&md5=cf8df800ee9fa639998bc13cc4a77c2b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1111%2Fcas.12585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12585%26sid%3Dliteratum%253Aachs%26aulast%3DSaijo%26aufirst%3DK.%26aulast%3DImamura%26aufirst%3DJ.%26aulast%3DNarita%26aufirst%3DK.%26aulast%3DOda%26aufirst%3DA.%26aulast%3DShimodaira%26aufirst%3DH.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DIshioka%26aufirst%3DC.%26atitle%3DBiochemical%252C%2520Biological%2520and%2520Structural%2520Properties%2520of%2520Romidepsin%2520%2528Fk228%2529%2520and%2520Its%2520Analogs%2520as%2520Novel%2520Hdac%252FPi3k%2520Dual%2520Inhibitors%26jtitle%3DCancer%2520Sci.%26date%3D2015%26volume%3D106%26spage%3D208%26epage%3D215%26doi%3D10.1111%2Fcas.12585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stolfa, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancelot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leproult, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melesina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavarelli, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for the Antiparasitic Activity of a Mercaptoacetamide Derivative That Inhibits Histone Deacetylase 8 (Hdac8) from the Human Pathogen Schistosoma Mansoni</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">3442</span>– <span class="NLM_lpage">3453</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2014.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.jmb.2014.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24657767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1ahu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3442-3453&author=D.+A.+Stolfaauthor=M.+Marekauthor=J.+Lancelotauthor=A.-T.+Hauserauthor=A.+Walterauthor=E.+Leproultauthor=J.+Melesinaauthor=T.+Rumpfauthor=J.-M.+Wurtzauthor=J.+Cavarelliauthor=W.+Sipplauthor=R.+J.+Pierceauthor=C.+Romierauthor=M.+Jung&title=Molecular+Basis+for+the+Antiparasitic+Activity+of+a+Mercaptoacetamide+Derivative+That+Inhibits+Histone+Deacetylase+8+%28Hdac8%29+from+the+Human+Pathogen+Schistosoma+Mansoni&doi=10.1016%2Fj.jmb.2014.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the Antiparasitic Activity of a Mercaptoacetamide Derivative That Inhibits Histone Deacetylase 8 (HDAC8) from the Human Pathogen Schistosoma mansoni</span></div><div class="casAuthors">Stolfa, Diana A.; Marek, Martin; Lancelot, Julien; Hauser, Alexander-Thomas; Walter, Alexandra; Leproult, Emeline; Melesina, Jelena; Rumpf, Tobias; Wurtz, Jean-Marie; Cavarelli, Jean; Sippl, Wolfgang; Pierce, Raymond J.; Romier, Christophe; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3442-3453</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Schistosomiasis, caused by the parasitic flatworm Schistosoma mansoni and related species, is a tropical disease that affects over 200 million people worldwide.  A new approach for targeting eukaryotic parasites is to tackle their dynamic epigenetic machinery that is necessary for the extensive phenotypic changes during the life cycle of the parasite.  Recently, the authors identified S. mansoni histone deacetylase 8 (smHDAC8) as a potential target for antiparasitic therapy.  Here, the authors present results on the investigations of a focused set of HDAC (histone deacetylase) inhibitors on smHDAC8.  Besides several active hydroxamates, the authors identified a thiol-based inhibitor that inhibited smHDAC8 activity in the micromolar range with unexpected selectivity over the human isotype, which has not been obsd. so far.  The crystal structure of smHDAC8 complexed with the thiol deriv. revealed that the inhibitor is accommodated in the catalytic pocket, where it interacts with both the catalytic zinc ion and the essential catalytic tyrosine (Y341) residue via its mercaptoacetamide warhead.  To the authors' knowledge, this is the first complex crystal structure of any HDAC inhibited by a mercaptoacetamide inhibitor, and therefore, this finding offers a rationale for further improvement.  Finally, an ester prodrug of the thiol HDAC inhibitor exhibited antiparasitic activity on cultured schistosomes in a dose-dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzjyeGmhcE27Vg90H21EOLACvtfcHk0ljeM1jMoyarFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1ahu7k%253D&md5=8732c934526a515aecbcacbe8a677066</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DStolfa%26aufirst%3DD.%2BA.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DLancelot%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DA.-T.%26aulast%3DWalter%26aufirst%3DA.%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DMelesina%26aufirst%3DJ.%26aulast%3DRumpf%26aufirst%3DT.%26aulast%3DWurtz%26aufirst%3DJ.-M.%26aulast%3DCavarelli%26aufirst%3DJ.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DMolecular%2520Basis%2520for%2520the%2520Antiparasitic%2520Activity%2520of%2520a%2520Mercaptoacetamide%2520Derivative%2520That%2520Inhibits%2520Histone%2520Deacetylase%25208%2520%2528Hdac8%2529%2520from%2520the%2520Human%2520Pathogen%2520Schistosoma%2520Mansoni%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3442%26epage%3D3453%26doi%3D10.1016%2Fj.jmb.2014.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">The Bet Bromodomain Inhibitor Jq1 Activates Hiv Latency through Antagonizing Brd4 Inhibition of Tat-Transactivation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1093/nar/gks976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1093%2Fnar%2Fgks976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=23087374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=277-287&author=Z.+Liauthor=J.+Guoauthor=Y.+Wuauthor=Q.+Zhou&title=The+Bet+Bromodomain+Inhibitor+Jq1+Activates+Hiv+Latency+through+Antagonizing+Brd4+Inhibition+of+Tat-Transactivation&doi=10.1093%2Fnar%2Fgks976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The BET bromodomain inhibitor JQ1 activates HIV latency through antagonizing Brd4 inhibition of Tat-transactivation</span></div><div class="casAuthors">Li, Zichong; Guo, Jia; Wu, Yuntao; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Latent HIV reservoirs are the primary hurdle to eradication of infection.  Identification of agents, pathways and mol. mechanisms that activate latent provirus may, in the presence of highly active antiretroviral therapy, permit clearance of infected cells by the immune system.  Promoter-proximal pausing of RNA polymerase (Pol) II is a major rate-limiting step in HIV gene expression.  The viral Tat protein recruits human Super Elongation Complex (SEC) to paused Pol II to overcome this limitation.  Here, we identify the bromodomain protein Brd4 and its inhibition of Tat-transactivation as a major impediment to latency reactivation.  Brd4 competitively blocks the Tat-SEC interaction on HIV promoter.  The BET bromodomain inhibitor JQ1 dissocs. Brd4 from the HIV promoter to allow Tat recruitment of SEC to stimulate HIV elongation.  JQ1 synergizes with another latency activator prostratin, which promotes Pol II loading onto the viral promoter.  Because JQ1 activates viral latency without inducing global T cell activation, this and other closely related compds. and their antagonization of Brd4 to promote Tat-SEC interaction merit further investigations as effective agents/strategies for eliminating latent HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod_cOKnnk8oLVg90H21EOLACvtfcHk0liPCrcDducAyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D&md5=87866d81d13af3d5051844ed6c17c98d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks976%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DThe%2520Bet%2520Bromodomain%2520Inhibitor%2520Jq1%2520Activates%2520Hiv%2520Latency%2520through%2520Antagonizing%2520Brd4%2520Inhibition%2520of%2520Tat-Transactivation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D277%26epage%3D287%26doi%3D10.1093%2Fnar%2Fgks976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Identification of a Novel Series of Bet Family Bromodomain Inhibitors: Binding Mode and Profile of I-Bet151 (Gsk1210151a)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2968</span>– <span class="NLM_lpage">2972</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.02.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.-W.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+Novel+Series+of+Bet+Family+Bromodomain+Inhibitors%3A+Binding+Mode+and+Profile+of+I-Bet151+%28Gsk1210151a%29&doi=10.1016%2Fj.bmcl.2012.02.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0liPCrcDducAyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520Novel%2520Series%2520of%2520Bet%2520Family%2520Bromodomain%2520Inhibitors%253A%2520Binding%2520Mode%2520and%2520Profile%2520of%2520I-Bet151%2520%2528Gsk1210151a%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972%26doi%3D10.1016%2Fj.bmcl.2012.02.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou-Eymard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">The Discovery of I-Bet726 (Gsk1324726a), a Potent Tetrahydroquinoline Apoa1 up-Regulator and Selective Bet Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8111</span>– <span class="NLM_lpage">8131</span>, <span class="refDoi"> DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.-M.+G.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.-w.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=The+Discovery+of+I-Bet726+%28Gsk1324726a%29%2C+a+Potent+Tetrahydroquinoline+Apoa1+up-Regulator+and+Selective+Bet+Bromodomain+Inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0lgGDY7LS0ahKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DThe%2520Discovery%2520of%2520I-Bet726%2520%2528Gsk1324726a%2529%252C%2520a%2520Potent%2520Tetrahydroquinoline%2520Apoa1%2520up-Regulator%2520and%2520Selective%2520Bet%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The Making of I-Bet762, a Bet Bromodomain Inhibitor Now in Clinical Development</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7498</span>– <span class="NLM_lpage">7500</span>, <span class="refDoi"> DOI: 10.1021/jm4014407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4014407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSmsrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7498-7500&author=Y.+Zhaoauthor=C.-Y.+Yangauthor=S.+Wang&title=The+Making+of+I-Bet762%2C+a+Bet+Bromodomain+Inhibitor+Now+in+Clinical+Development&doi=10.1021%2Fjm4014407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The Making of I-BET762, a BET Bromodomain Inhibitor Now in Clinical Development</span></div><div class="casAuthors">Zhao, Yujun; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7498-7500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) proteins belong to a class of proteins collectively called epigenetic "readers".  BET bromodomains have emerged as promising drug targets for treatment of cancers, inflammatory diseases, and other medical conditions.  GlaxoSmithKline scientists have successfully optimized a class of benzodiazepines as inhibitors of BET bromodomains, without any prior knowledge of identified mol. targets.  It thus is possible to hit a target without aiming at it.  The optimized lead compd. I-BET762 is currently being evaluated in a phase I clin. trial for treatment of human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN-T-2VhynyrVg90H21EOLACvtfcHk0lit_njYf_sa-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSmsrbF&md5=61aa569c1b1b129b54229b9f8401f46b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm4014407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4014407%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520Making%2520of%2520I-Bet762%252C%2520a%2520Bet%2520Bromodomain%2520Inhibitor%2520Now%2520in%2520Clinical%2520Development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7498%26epage%3D7500%26doi%3D10.1021%2Fjm4014407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kather, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlimpert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagies, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhardt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kammerer, B.</span></span> <span> </span><span class="NLM_article-title">Metabolic Response to Xd14 Treatment in Human Breast Cancer Cell Line Mcf-7</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1772</span>, <span class="refDoi"> DOI: 10.3390/ijms17101772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.3390%2Fijms17101772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1yksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1772&author=D.+Panauthor=M.+Katherauthor=L.+Willmannauthor=M.+Schlimpertauthor=C.+Bauerauthor=S.+Lagiesauthor=K.+Schmidtkunzauthor=S.+Eisenhardtauthor=M.+Jungauthor=S.+G%C3%BCntherauthor=B.+Kammerer&title=Metabolic+Response+to+Xd14+Treatment+in+Human+Breast+Cancer+Cell+Line+Mcf-7&doi=10.3390%2Fijms17101772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic response to XD14 treatment in human breast cancer cell line MCF-7</span></div><div class="casAuthors">Pan, Daqiang; Kather, Michel; Willmann, Lucas; Schlimpert, Manuel; Bauer, Christoph; Lagies, Simon; Schmidtkunz, Karin; Eisenhardt, Steffen U.; Jung, Manfred; Guenther, Stefan; Kammerer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1772/1-1772/15</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">XD14 is a 4-acyl pyrrole deriv., which was discovered by a high-throughput virtual screening expt.  XD14 inhibits bromodomain and extra-terminal domain (BET) proteins (BRD2, BRD3, BRD4 and BRDT) and consequently suppresses cell proliferation.  In this study, metabolic profiling reveals the mol. effects in the human breast cancer cell line MCF-7 (Michigan Cancer Foundation-7) treated by XD14.  A three-day time series expt. with two concns. of XD14 was performed.  Gas chromatog.-mass spectrometry (GC-MS) was applied for untargeted profiling of treated and non-treated MCF-7 cells.  The gained data sets were evaluated by several statistical methods: anal. of variance (ANOVA), clustering anal., principle component anal. (PCA), and partial least squares discriminant anal. (PLS-DA).  Cell proliferation was strongly inhibited by treatment with 50 μM XD14.  Samples could be discriminated by time and XD14 concn. using PLS-DA.  From the 117 identified metabolites, 67 were significantly altered after XD14 treatment.  These metabolites include amino acids, fatty acids, Krebs cycle and glycolysis intermediates, as well as compds. of purine and pyrimidine metab.  This massive intervention in energy metab. and the lack of available nucleotides could explain the decreased proliferation rate of the cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf1rN2g-3QFLVg90H21EOLACvtfcHk0lit_njYf_sa-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1yksg%253D%253D&md5=6e92165233c5cc213af50292dc373fa9</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.3390%2Fijms17101772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17101772%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DD.%26aulast%3DKather%26aufirst%3DM.%26aulast%3DWillmann%26aufirst%3DL.%26aulast%3DSchlimpert%26aufirst%3DM.%26aulast%3DBauer%26aufirst%3DC.%26aulast%3DLagies%26aufirst%3DS.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DEisenhardt%26aufirst%3DS.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DKammerer%26aufirst%3DB.%26atitle%3DMetabolic%2520Response%2520to%2520Xd14%2520Treatment%2520in%2520Human%2520Breast%2520Cancer%2520Cell%2520Line%2520Mcf-7%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D1772%26doi%3D10.3390%2Fijms17101772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafiane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiacchia, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimbou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genest, J.</span></span> <span> </span><span class="NLM_article-title">Rvx-208: A Small Molecule That Increases Apolipoprotein a-I and High-Density Lipoprotein Cholesterol in Vitro and in Vivo</span>. <i>J. Am. Coll. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2580</span>– <span class="NLM_lpage">2589</span>, <span class="refDoi"> DOI: 10.1016/j.jacc.2010.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.jacc.2010.02.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=20513599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2010&pages=2580-2589&author=D.+Baileyauthor=R.+Jahagirdarauthor=A.+Gordonauthor=A.+Hafianeauthor=S.+Campbellauthor=S.+Chaturauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=F.+S.+Chiacchiaauthor=J.+Johanssonauthor=L.+Krimbouauthor=N.+C.+W.+Wongauthor=J.+Genest&title=Rvx-208%3A+A+Small+Molecule+That+Increases+Apolipoprotein+a-I+and+High-Density+Lipoprotein+Cholesterol+in+Vitro+and+in+Vivo&doi=10.1016%2Fj.jacc.2010.02.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208. A small molecule that increases apolipoprotein A-I and high-density lipoprotein cholesterol in vitro and in vivo</span></div><div class="casAuthors">Bailey, Dana; Jahagirdar, Ravi; Gordon, Allan; Hafiane, Anouar; Campbell, Steven; Chatur, Safia; Wagner, Gregory S.; Hansen, Henrik C.; Chiacchia, Fabrizio S.; Johansson, Jan; Krimbou, Larbi; Wong, Norman C. W.; Genest, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American College of Cardiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2580-2589</span>CODEN:
                <span class="NLM_cas:coden">JACCDI</span>;
        ISSN:<span class="NLM_cas:issn">0735-1097</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Objectives: The aim of this study was to det. whether a novel small mol. RVX-208 affects apolipoprotein (apo)A-I and high-d. lipoprotein cholesterol (HDL-C) levels in vitro and in vivo.  Background: Increased apoA-I and HDL-C levels are potential therapeutic targets for reducing atherosclerotic disease.  Methods: HepG2 cells were treated with 0 to 60 μmol/l RVX-208 followed by assays for apoA-I and HDL-C prodn.  For in vivo studies, African green monkeys (AGMs) received 15 to 60 mg/kg/day RVX-208, and the serum was analyzed for lipoprotein levels, HDL-subparticle distribution, cholesterol efflux, and activity of lipid-modifying enzymes.  A phase I clin. trial was conducted in healthy volunteers (given 1 to 20 mg/kg/day of RVX-208) to assess safety, tolerability, and pharmacokinetics.  Results: The RVX-208 induced apoA-I mRNA and protein synthesis in HepG2 cells, leading to in- creased levels of pre-β-migrating and α-lipoprotein particles contg. apoA-I (LpA-I) in spent media.  Similarly, in AGMs, RVX-208 treatment for 63 days increased serum apoA-I and HDL-C levels (60% and 97%, resp.).  In addn., the levels of pre-β1-LpA-I and α1-LpA-I HDL-subparticles were increased as well as ATP binding cassette AI, ATP binding cassette G1, and scavenger receptor class B type I-dependent cholesterol efflux.  These changes were not mediated by cholesteryl-ester-transfer protein.  Treatment of humans for 1 wk with oral RVX-208 increased apoA-I, pre-β-HDL, and HDL functionality.  Conclusions: RVX-208 increases apoA-I and HDL-C in vitro and in vivo.  In AGMs, RVX-208 raises serum pre-β1-LpA-I and α-LpA-I levels and enhances cholesterol efflux.  Data in humans point to beneficial features of RVX-208 that might be useful for treating atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEGExcnx6drLVg90H21EOLACvtfcHk0lgkEZrwTgvUdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D&md5=aab9ac26e73704f1cd569e2ff47d0c51</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.jacc.2010.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacc.2010.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DD.%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DHafiane%26aufirst%3DA.%26aulast%3DCampbell%26aufirst%3DS.%26aulast%3DChatur%26aufirst%3DS.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DChiacchia%26aufirst%3DF.%2BS.%26aulast%3DJohansson%26aufirst%3DJ.%26aulast%3DKrimbou%26aufirst%3DL.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DGenest%26aufirst%3DJ.%26atitle%3DRvx-208%253A%2520A%2520Small%2520Molecule%2520That%2520Increases%2520Apolipoprotein%2520a-I%2520and%2520High-Density%2520Lipoprotein%2520Cholesterol%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Am.%2520Coll.%2520Cardiol.%26date%3D2010%26volume%3D55%26spage%3D2580%26epage%3D2589%26doi%3D10.1016%2Fj.jacc.2010.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Pfi-1, a Highly Selective Protein Interaction Inhibitor, Targeting Bet Bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3336</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F0008-5472.can-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+Mullerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=Pfi-1%2C+a+Highly+Selective+Protein+Interaction+Inhibitor%2C+Targeting+Bet+Bromodomains&doi=10.1158%2F0008-5472.can-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0li5l3HCpwF8-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPfi-1%252C%2520a%2520Highly%2520Selective%2520Protein%2520Interaction%2520Inhibitor%252C%2520Targeting%2520Bet%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346%26doi%3D10.1158%2F0008-5472.can-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent and Selective Brd4 Inhibitors Capable of Blocking Tlr3-Induced Acute Airway Inflammation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejmech.2018.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29649741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=450-461&author=Z.+Liuauthor=B.+Tianauthor=H.+Chenauthor=P.+Wangauthor=A.+R.+Brasierauthor=J.+Zhou&title=Discovery+of+Potent+and+Selective+Brd4+Inhibitors+Capable+of+Blocking+Tlr3-Induced+Acute+Airway+Inflammation&doi=10.1016%2Fj.ejmech.2018.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation</span></div><div class="casAuthors">Liu, Zhiqing; Tian, Bing; Chen, Haiying; Wang, Pingyuan; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of diverse small mols. have been designed and synthesized through structure-based drug design by taking advantage of fragment merging and elaboration approaches.  Compds. ZL0420 (28) and ZL0454 (35) were identified as potent and selective BRD4 inhibitors with nanomolar binding affinities to bromodomains (BDs) of BRD4.  Both of them can be well docked into the acetyl-lysine (KAc) binding pocket of BRD4, forming key interactions including the crit. hydrogen bonds with Asn140 directly and Tyr97 indirectly via a H2O mol.  Both compds. 28 and 35 exhibited submicromolar potency of inhibiting the TLR3-dependent innate immune gene program, including ISG54, ISG56, IL-8, and Groβ genes in cultured human small airway epithelial cells (hSAECs).  More importantly, they also demonstrated potent efficacy reducing airway inflammation in a mouse model with low toxicity, indicating a proof of concept that BRD4 inhibitors may offer the therapeutic potential to block the viral-induced airway inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLyejdKuiGi7Vg90H21EOLACvtfcHk0li5l3HCpwF8-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D&md5=564b088a350abef9964d97bbecbb52b1</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Potent%2520and%2520Selective%2520Brd4%2520Inhibitors%2520Capable%2520of%2520Blocking%2520Tlr3-Induced%2520Acute%2520Airway%2520Inflammation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ejmech.2018.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-Methyl-7-Oxo-6,7-Dihydro-1h-Pyrrolo[2,3-C]Pyridin -4-Yl)Phenyl)Ethanesulfonamide (Abbv-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (Bet) Family Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>– <span class="NLM_lpage">8384</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-Difluorophenoxy%29-3-%286-Methyl-7-Oxo-6%2C7-Dihydro-1h-Pyrrolo%5B2%2C3-C%5DPyridin+-4-Yl%29Phenyl%29Ethanesulfonamide+%28Abbv-075%2FMivebresib%29%2C+a+Potent+and+Orally+Available+Bromodomain+and+Extraterminal+Domain+%28Bet%29+Family+Bromodomain+Inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lh-p7F4Dae2bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-Difluorophenoxy%2529-3-%25286-Methyl-7-Oxo-6%252C7-Dihydro-1h-Pyrrolo%255B2%252C3-C%255DPyridin%2520-4-Yl%2529Phenyl%2529Ethanesulfonamide%2520%2528Abbv-075%252FMivebresib%2529%252C%2520a%2520Potent%2520and%2520Orally%2520Available%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528Bet%2529%2520Family%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonazzoli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Predolini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altwerger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menderes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zammataro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettinella, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manzano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buza, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litkouhi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silasi, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azodi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santin, A. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bet Bromodomain Proteins with Gs-5829 and Gs-626510 in Uterine Serous Carcinoma, a Biologically Aggressive Variant of Endometrial Cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4845</span>– <span class="NLM_lpage">4853</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-18-0864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1078-0432.ccr-18-0864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29941483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsFCksrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4845-4853&author=E.+Bonazzoliauthor=F.+Predoliniauthor=E.+Coccoauthor=S.+Belloneauthor=G.+Altwergerauthor=G.+Menderesauthor=L.+Zammataroauthor=A.+Bianchiauthor=F.+Pettinellaauthor=F.+Riccioauthor=C.+Hanauthor=G.+Yadavauthor=S.+Lopezauthor=A.+Manzanoauthor=P.+Manaraauthor=N.+Buzaauthor=P.+Huiauthor=S.+Wongauthor=B.+Litkouhiauthor=E.+Ratnerauthor=D.-A.+Silasiauthor=G.+S.+Huangauthor=M.+Azodiauthor=P.+E.+Schwartzauthor=J.+Schlessingerauthor=A.+D.+Santin&title=Inhibition+of+Bet+Bromodomain+Proteins+with+Gs-5829+and+Gs-626510+in+Uterine+Serous+Carcinoma%2C+a+Biologically+Aggressive+Variant+of+Endometrial+Cancer&doi=10.1158%2F1078-0432.ccr-18-0864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomain proteins with GS-5829 and GS-626510 in uterine serous carcinoma, a biologically aggressive variant of endometrial cancer</span></div><div class="casAuthors">Bonazzoli, Elena; Predolini, Federica; Cocco, Emiliano; Bellone, Stefania; Altwerger, Gary; Menderes, Gulden; Zammataro, Luca; Bianchi, Anna; Pettinella, Francesca; Riccio, Francesco; Han, Chanhee; Yadav, Ghanshyam; Lopez, Salvatore; Manzano, Aranzazu; Manara, Paola; Buza, Natalia; Hui, Pei; Wong, Serena; Litkouhi, Babak; Ratner, Elena; Silasi, Dan-Arin; Huang, Gloria S.; Azodi, Masoud; Schwartz, Peter E.; Schlessinger, Joseph; Santin, Alessandro D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4845-4853</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Uterine serous carcinoma (USC) is a rare and aggressive variant of endometrial cancer.  Whole-exome sequencing (WES) studies have recently reported c-Myc gene amplification in a large no. of USCs, suggesting c-Myc as a potential therapeutic target.  We investigated the activity of novel BET bromodomain inhibitors (GS-5829 and GS-626510, Gilead Sciences Inc.) and JQ1 against primary USC cultures and USC xenografts.  Exptl. Design: We evaluated c-Myc expression by qRT-PCR in a total of 45 USCs including fresh-frozen tumor tissues and primary USC cell lines.  We also performed IHC and Western blot expts. in 8 USC tumors.  USC cultures were evaluated for sensitivity to GS-5829, GS-626510, and JQ1 in vitro using proliferation, viability, and apoptosis assays.  Finally, the in vivo activity of GS-5829, GS-626510, and JQ1 was studied in USC-ARK1 and USC-ARK2 mouse xenografts.  Results: Fresh-frozen USC and primary USC cell lines overexpressed c-Myc when compared with normal tissues (P = 0.0009 and 0.0083, resp.).  High c-Myc expression was found in 7 of 8 of primary USC cell lines tested by qRT-PCR and 5 of 8 tested by IHC.  In vitro expts. demonstrated high sensitivity of USC cell lines to the exposure to GS-5829, GS-626510, and JQ1 with BET inhibitors causing a dose-dependent decrease in the phosphorylated levels of c-Myc and a dose-dependent increase in caspase activation (apoptosis).  In comparative in vivo expts., GS-5829 and/or GS-626510 were found more effective than JQ1 at the concns./doses used in decreasing tumor growth in both USC-ARK1 and USC-ARK2 mouse xenograft models.  Conclusions: GS-5829 and GS-626510 may represent novel, highly effective therapeutics agents against recurrent/chemotherapy-resistant USC-overexpressing c-Myc.  Clin. studies with GS-5829 in patients with USC harboring chemotherapy-resistant disease are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcTo-RvUJKTLVg90H21EOLACvtfcHk0lgPSG4A_cKsUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsFCksrg%253D&md5=7474c8f5fd4b6772f87b12cf114b47e8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-18-0864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-18-0864%26sid%3Dliteratum%253Aachs%26aulast%3DBonazzoli%26aufirst%3DE.%26aulast%3DPredolini%26aufirst%3DF.%26aulast%3DCocco%26aufirst%3DE.%26aulast%3DBellone%26aufirst%3DS.%26aulast%3DAltwerger%26aufirst%3DG.%26aulast%3DMenderes%26aufirst%3DG.%26aulast%3DZammataro%26aufirst%3DL.%26aulast%3DBianchi%26aufirst%3DA.%26aulast%3DPettinella%26aufirst%3DF.%26aulast%3DRiccio%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DYadav%26aufirst%3DG.%26aulast%3DLopez%26aufirst%3DS.%26aulast%3DManzano%26aufirst%3DA.%26aulast%3DManara%26aufirst%3DP.%26aulast%3DBuza%26aufirst%3DN.%26aulast%3DHui%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DLitkouhi%26aufirst%3DB.%26aulast%3DRatner%26aufirst%3DE.%26aulast%3DSilasi%26aufirst%3DD.-A.%26aulast%3DHuang%26aufirst%3DG.%2BS.%26aulast%3DAzodi%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DP.%2BE.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DSantin%26aufirst%3DA.%2BD.%26atitle%3DInhibition%2520of%2520Bet%2520Bromodomain%2520Proteins%2520with%2520Gs-5829%2520and%2520Gs-626510%2520in%2520Uterine%2520Serous%2520Carcinoma%252C%2520a%2520Biologically%2520Aggressive%2520Variant%2520of%2520Endometrial%2520Cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D24%26spage%3D4845%26epage%3D4853%26doi%3D10.1158%2F1078-0432.ccr-18-0864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Gamal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hing, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blachly, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieselhuber, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goettl, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spevak, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carias, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantoy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanftner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nespi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathé, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span> <span> </span><span class="NLM_article-title">Brd4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to Plx51107, a Novel Structurally Distinct Bet Inhibitor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-17-0902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F2159-8290.cd-17-0902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29386193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=458-477&author=H.+G.+Ozerauthor=D.+El-Gamalauthor=B.+Powellauthor=Z.+A.+Hingauthor=J.+S.+Blachlyauthor=B.+Harringtonauthor=S.+Mitchellauthor=N.+R.+Grieselhuberauthor=K.+Williamsauthor=T.-H.+Laiauthor=L.+Alinariauthor=R.+A.+Baiocchiauthor=L.+Brintonauthor=E.+Baskinauthor=M.+Cannonauthor=L.+Beaverauthor=V.+M.+Goettlauthor=D.+M.+Lucasauthor=J.+A.+Woyachauthor=D.+Sampathauthor=A.+M.+Lehmanauthor=L.+Yuauthor=J.+Zhangauthor=Y.+Maauthor=Y.+Zhangauthor=W.+Spevakauthor=S.+Shiauthor=P.+Seversonauthor=R.+Shellooeauthor=H.+Cariasauthor=G.+Tsangauthor=K.+Dongauthor=T.+Ewingauthor=A.+Marimuthuauthor=C.+Tantoyauthor=J.+Waltersauthor=L.+Sanftnerauthor=H.+Rezaeiauthor=M.+Nespiauthor=B.+Matusowauthor=G.+Habetsauthor=P.+Ibrahimauthor=C.+Zhangauthor=E.+A.+Math%C3%A9author=G.+Bollagauthor=J.+C.+Byrdauthor=R.+Lapalombella&title=Brd4+Profiling+Identifies+Critical+Chronic+Lymphocytic+Leukemia+Oncogenic+Circuits+and+Reveals+Sensitivity+to+Plx51107%2C+a+Novel+Structurally+Distinct+Bet+Inhibitor&doi=10.1158%2F2159-8290.cd-17-0902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span></div><div class="casAuthors">Ozer, Hatice Gulcin; El-Gamal, Dalia; Powell, Ben; Hing, Zachary A.; Blachly, James S.; Harrington, Bonnie; Mitchell, Shaneice; Grieselhuber, Nicole R.; Williams, Katie; Lai, Tzung-Huei; Alinari, Lapo; Baiocchi, Robert A.; Brinton, Lindsey; Baskin, Elizabeth; Cannon, Matthew; Beaver, Larry; Goettl, Virginia M.; Lucas, David M.; Woyach, Jennifer A.; Sampath, Deepa; Lehman, Amy M.; Yu, Lianbo; Zhang, Jiazhong; Ma, Yan; Zhang, Ying; Spevak, Wayne; Shi, Songyuan; Severson, Paul; Shellooe, Rafe; Carias, Heidi; Tsang, Garson; Dong, Ken; Ewing, Todd; Marimuthu, Adhirai; Tantoy, Christina; Walters, Jason; Sanftner, Laura; Rezaei, Hamid; Nespi, Marika; Matusow, Bernice; Habets, Gaston; Ibrahim, Prabha; Zhang, Chao; Mathe, Ewy A.; Bollag, Gideon; Byrd, John C.; Lapalombella, Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-477</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins are key regulators of gene expression in cancer.  Herein, we utilize BRD4 profiling to identify crit. pathways involved in pathogenesis of chronic lymphocytic leukemia (CLL).  BRD4 is overexpressed in CLL and is enriched proximal to genes upregulated or de novo expressed in CLL with known functions in disease pathogenesis and progression.  These genes, including key members of the B-cell receptor (BCR) signaling pathway, provide a rationale for this therapeutic approach to identify new targets in alternative types of cancer.  Addnl., we describe PLX51107, a structurally distinct BET inhibitor with novel in vitro and in vivo pharmacol. properties that emulates or exceeds the efficacy of BCR signaling agents in preclin. models of CLL.  Herein, the discovery of the involvement of BRD4 in the core CLL transcriptional program provides a compelling rationale for clin. investigation of PLX51107 as epigenetic therapy in CLL and application of BRD4 profiling in other cancers.  Significance: To date, functional studies of BRD4 in CLL are lacking.  Through integrated genomic, functional, and pharmacol. analyses, we uncover the existence of BRD4-regulated core CLL transcriptional programs and present preclin. proof-of-concept studies validating BET inhibition as an epigenetic approach to target BCR signaling in CLL.  Cancer Discov; 8(4); 458-77. ©2018 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvzZtZKkb5rVg90H21EOLACvtfcHk0lgPSG4A_cKsUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D&md5=3715a65bb8cee2a5fe53994a3d106303</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.cd-17-0902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.cd-17-0902%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DH.%2BG.%26aulast%3DEl-Gamal%26aufirst%3DD.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHing%26aufirst%3DZ.%2BA.%26aulast%3DBlachly%26aufirst%3DJ.%2BS.%26aulast%3DHarrington%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DS.%26aulast%3DGrieselhuber%26aufirst%3DN.%2BR.%26aulast%3DWilliams%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DT.-H.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DBrinton%26aufirst%3DL.%26aulast%3DBaskin%26aufirst%3DE.%26aulast%3DCannon%26aufirst%3DM.%26aulast%3DBeaver%26aufirst%3DL.%26aulast%3DGoettl%26aufirst%3DV.%2BM.%26aulast%3DLucas%26aufirst%3DD.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DK.%26aulast%3DEwing%26aufirst%3DT.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DTantoy%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DJ.%26aulast%3DSanftner%26aufirst%3DL.%26aulast%3DRezaei%26aufirst%3DH.%26aulast%3DNespi%26aufirst%3DM.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMath%25C3%25A9%26aufirst%3DE.%2BA.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DLapalombella%26aufirst%3DR.%26atitle%3DBrd4%2520Profiling%2520Identifies%2520Critical%2520Chronic%2520Lymphocytic%2520Leukemia%2520Oncogenic%2520Circuits%2520and%2520Reveals%2520Sensitivity%2520to%2520Plx51107%252C%2520a%2520Novel%2520Structurally%2520Distinct%2520Bet%2520Inhibitor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D458%26epage%3D477%26doi%3D10.1158%2F2159-8290.cd-17-0902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greer, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, I.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. H.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases Positively Regulate Transcription through the Elongation Machinery</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1444</span>– <span class="NLM_lpage">1455</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.celrep.2015.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26549458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=1444-1455&author=C.+B.+Greerauthor=Y.+Tanakaauthor=Y.+J.+Kimauthor=P.+Xieauthor=M.+Q.+Zhangauthor=I.-H.+Parkauthor=T.+H.+Kim&title=Histone+Deacetylases+Positively+Regulate+Transcription+through+the+Elongation+Machinery&doi=10.1016%2Fj.celrep.2015.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylases Positively Regulate Transcription through the Elongation Machinery</span></div><div class="casAuthors">Greer, Celeste B.; Tanaka, Yoshiaki; Kim, Yoon Jung; Xie, Peng; Zhang, Michael Q.; Park, In-Hyun; Kim, Tae Hoon</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1444-1455</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Transcription elongation regulates the expression of many genes, including oncogenes.  Histone deacetylase (HDAC) inhibitors (HDACIs) block elongation, suggesting that HDACs are involved in gene activation.  To understand this, we analyzed nascent transcription and elongation factor binding genome-wide after perturbation of elongation with small mol. inhibitors.  We found that HDACI-mediated repression requires heat shock protein 90 (HSP90) activity.  HDACIs promote the assocn. of RNA polymerase II (RNAP2) and neg. elongation factor (NELF), a complex stabilized by HSP90, at the same genomic sites.  Addnl., HDACIs redistribute bromodomain-contg. protein 4 (BRD4), a key elongation factor involved in enhancer activity.  BRD4 binds to newly acetylated sites, and its occupancy at promoters and enhancers is reduced.  Furthermore, HDACIs reduce enhancer activity, as measured by enhancer RNA prodn.  Therefore, HDACs are required for limiting acetylation in gene bodies and intergenic regions.  This facilitates the binding of elongation factors to properly acetylated promoters and enhancers for efficient elongation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqFzVFizqpAbVg90H21EOLACvtfcHk0liKhNA-25-Kpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7jK&md5=73a695b1a306db9eb042b28af3c7300d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DC.%2BB.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DXie%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%2BQ.%26aulast%3DPark%26aufirst%3DI.-H.%26aulast%3DKim%26aufirst%3DT.%2BH.%26atitle%3DHistone%2520Deacetylases%2520Positively%2520Regulate%2520Transcription%2520through%2520the%2520Elongation%2520Machinery%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D13%26spage%3D1444%26epage%3D1455%26doi%3D10.1016%2Fj.celrep.2015.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi-Yachide, T.</span></span> <span> </span><span class="NLM_article-title">Bet Bromodomain as a Target of Epigenetic Therapy</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1248/cpb.c16-00225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1248%2Fcpb.c16-00225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27250788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Omur7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2016&pages=540-547&author=T.+Noguchi-Yachide&title=Bet+Bromodomain+as+a+Target+of+Epigenetic+Therapy&doi=10.1248%2Fcpb.c16-00225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain as a Target of Epigenetic Therapy</span></div><div class="casAuthors">Noguchi-Yachide, Tomomi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-547</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Acetylation of histone is a key epigenetic modification, and contributes to many DNA-dependent cellular processes.  The bromodomain structure, which consists of approx. 110 amino acid residues, serves as a 'reader' that recognizes acetylated lysine in histones, leading to recruitment of pos. transcriptional elongation factor b (P-TEFb), and thereby promoting transcriptional activity and chromatin remodeling.  Among bromodomain-contg. proteins, members of the bromodomain and extra-terminal domain (BET) family contain tandem N-terminal bromodomains.  BET proteins, esp. BRD4, have attracted interest as candidate therapeutic targets due to their putative involvement in the pathogenesis of various diseases, including cancer and inflammatory diseases.  Several BET inhibitors are under clin. trial for treatment of various cancers.  Furthermore, polypharmacol. agents such as dual kinase/BET inhibitors and dual histone deacetylase (HDAC)/BET inhibitors have recently been developed, in addn. to agents that degrade BET family proteins, such as proteolysis-targeting chimeras (PROTACs).  This paper reviews recent progress in epigenetic therapy targeting the BET bromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxzWdAhkzHKbVg90H21EOLACvtfcHk0liKhNA-25-Kpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Omur7L&md5=a1787a330f39a6005daf2e077dc94b13</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c16-00225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c16-00225%26sid%3Dliteratum%253Aachs%26aulast%3DNoguchi-Yachide%26aufirst%3DT.%26atitle%3DBet%2520Bromodomain%2520as%2520a%2520Target%2520of%2520Epigenetic%2520Therapy%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2016%26volume%3D64%26spage%3D540%26epage%3D547%26doi%3D10.1248%2Fcpb.c16-00225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neele, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van den Bossche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeksema, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Winther, M. P. J.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Pathways in Macrophages Emerge as Novel Targets in Atherosclerosis</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>763</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.ejphar.2015.03.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejphar.2015.03.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26004034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovV2rtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=763&publication_year=2015&pages=79-89&author=A.+E.+Neeleauthor=J.+Van+den+Bosscheauthor=M.+A.+Hoeksemaauthor=M.+P.+J.+de+Winther&title=Epigenetic+Pathways+in+Macrophages+Emerge+as+Novel+Targets+in+Atherosclerosis&doi=10.1016%2Fj.ejphar.2015.03.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic pathways in macrophages emerge as novel targets in atherosclerosis</span></div><div class="casAuthors">Neele, Annette E.; Van den Bossche, Jan; Hoeksema, Marten A.; de Winther, Menno P. J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">763</span>
        (<span class="NLM_cas:issue">Part_A</span>),
    <span class="NLM_cas:pages">79-89</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Atherosclerosis is a lipid-driven chronic inflammatory disorder.  Monocytes and macrophages are key immune cells in the development of disease and clin. outcome.  It is becoming increasingly clear that epigenetic pathways govern many aspects of monocyte and macrophage differentiation and activation.  The dynamic regulation of epigenetic patterns provides opportunities to alter disease-assocd. epigenetic states.  Therefore, pharmaceutical companies have embraced the targeting of epigenetic processes as new approaches for interventions.  Particularly histone deacetylase (Hdac) inhibitors and DNA-methyltransferase inhibitors have long received attention and several of them have been approved for clin. use in relation to hematol. malignancies.  The key focus is still on oncol., but Alzheimer's disease, Huntington's disease and inflammatory disorders are coming in focus as well.  These developments raise opportunities for the epigenetic targeting in cardiovascular disease (CVD).  In this review we discuss the epigenetic regulation of the inflammatory pathways in relation to atherosclerosis with a specific attention to monocyte- and macrophage-related processes.  What are the opportunities for future therapy of atherosclerosis by epigenetic interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr-t4fA2Z1OLVg90H21EOLACvtfcHk0lgmumCjzdntpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovV2rtL0%253D&md5=804d86c0b66b3b3851b26dc41310812a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2015.03.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2015.03.101%26sid%3Dliteratum%253Aachs%26aulast%3DNeele%26aufirst%3DA.%2BE.%26aulast%3DVan%2Bden%2BBossche%26aufirst%3DJ.%26aulast%3DHoeksema%26aufirst%3DM.%2BA.%26aulast%3Dde%2BWinther%26aufirst%3DM.%2BP.%2BJ.%26atitle%3DEpigenetic%2520Pathways%2520in%2520Macrophages%2520Emerge%2520as%2520Novel%2520Targets%2520in%2520Atherosclerosis%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D763%26spage%3D79%26epage%3D89%26doi%3D10.1016%2Fj.ejphar.2015.03.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiskus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenta, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaub, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portier, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaraj, S. G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, K. N.</span></span> <span> </span><span class="NLM_article-title">Highly Active Combination of Brd4 Antagonist and Histone Deacetylase Inhibitor against Human Acute Myelogenous Leukemia Cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1142</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-13-0770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1535-7163.mct-13-0770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24435446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1142-1154&author=W.+Fiskusauthor=S.+Sharmaauthor=J.+Qiauthor=J.+A.+Valentaauthor=L.+J.+Schaubauthor=B.+Shahauthor=K.+Pethauthor=B.+P.+Portierauthor=M.+Rodriguezauthor=S.+G.+T.+Devarajauthor=M.+Zhanauthor=J.+Shengauthor=S.+P.+Iyerauthor=J.+E.+Bradnerauthor=K.+N.+Bhalla&title=Highly+Active+Combination+of+Brd4+Antagonist+and+Histone+Deacetylase+Inhibitor+against+Human+Acute+Myelogenous+Leukemia+Cells&doi=10.1158%2F1535-7163.mct-13-0770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Active Combination of BRD4 Antagonist and Histone Deacetylase Inhibitor against Human Acute Myelogenous Leukemia Cells</span></div><div class="casAuthors">Fiskus, Warren; Sharma, Sunil; Qi, Jun; Valenta, John A.; Schaub, Leasha J.; Shah, Bhavin; Peth, Karissa; Portier, Bryce P.; Rodriguez, Melissa; Devaraj, Santhana G. T.; Zhan, Ming; Sheng, Jianting; Iyer, Swaminathan P.; Bradner, James E.; Bhalla, Kapil N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1142-1154</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) protein family members, including BRD4, bind to acetylated lysines on histones and regulate the expression of important oncogenes, for example, c-MYC and BCL2.  Here, we demonstrate the sensitizing effects of the histone hyperacetylation-inducing pan-histone deacetylase (HDAC) inhibitor panobinostat on human acute myelogenous leukemia (AML) blast progenitor cells (BPC) to the BET protein antagonist JQ1.  Treatment with JQ1, but not its inactive enantiomer (R-JQ1), was highly lethal against AML BPCs expressing mutant NPM1c+ with or without coexpression of FLT3-ITD or AML expressing mixed lineage leukemia fusion oncoprotein.  JQ1 treatment reduced binding of BRD4 and RNA polymerase II to the DNA of c-MYC and BCL2 and reduced their levels in the AML cells.  Cotreatment with JQ1 and the HDAC inhibitor panobinostat synergistically induced apoptosis of the AML BPCs, but not of normal CD34+ hematopoietic progenitor cells.  This was assocd. with greater attenuation of c-MYC and BCL2, while increasing p21, BIM, and cleaved PARP levels in the AML BPCs.  Cotreatment with JQ1 and panobinostat significantly improved the survival of the NOD/SCID mice engrafted with OCI-AML3 or MOLM13 cells (P < 0.01).  These findings highlight cotreatment with a BRD4 antagonist and an HDAC inhibitor as a potentially efficacious therapy of AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDqEnKe8Ipo7Vg90H21EOLACvtfcHk0lgmumCjzdntpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFWjsbs%253D&md5=7e73581df913111b8bc3cc80c1a081f2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-13-0770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-13-0770%26sid%3Dliteratum%253Aachs%26aulast%3DFiskus%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DValenta%26aufirst%3DJ.%2BA.%26aulast%3DSchaub%26aufirst%3DL.%2BJ.%26aulast%3DShah%26aufirst%3DB.%26aulast%3DPeth%26aufirst%3DK.%26aulast%3DPortier%26aufirst%3DB.%2BP.%26aulast%3DRodriguez%26aufirst%3DM.%26aulast%3DDevaraj%26aufirst%3DS.%2BG.%2BT.%26aulast%3DZhan%26aufirst%3DM.%26aulast%3DSheng%26aufirst%3DJ.%26aulast%3DIyer%26aufirst%3DS.%2BP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DBhalla%26aufirst%3DK.%2BN.%26atitle%3DHighly%2520Active%2520Combination%2520of%2520Brd4%2520Antagonist%2520and%2520Histone%2520Deacetylase%2520Inhibitor%2520against%2520Human%2520Acute%2520Myelogenous%2520Leukemia%2520Cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D1142%26epage%3D1154%26doi%3D10.1158%2F1535-7163.mct-13-0770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span> <span> </span><span class="NLM_article-title">Histone Cross-Talk Connects Protein Phosphatase 1 Alpha (Pp1 Alpha) and Histone Deacetylase (Hdac) Pathways to Regulate the Functional Transition of Bromodomain-Containing 4 (Brd4) for Inducible Gene Expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">23154</span>– <span class="NLM_lpage">23167</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.570812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1074%2Fjbc.m114.570812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24939842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlGmtbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=23154-23167&author=X.+Huauthor=X.+Luauthor=R.+Liuauthor=N.+Aiauthor=Z.+Caoauthor=Y.+Liauthor=J.+Liuauthor=B.+Yuauthor=K.+Liuauthor=H.+Wangauthor=C.+Zhouauthor=Y.+Wangauthor=A.+Hanauthor=F.+Dingauthor=R.+Chen&title=Histone+Cross-Talk+Connects+Protein+Phosphatase+1+Alpha+%28Pp1+Alpha%29+and+Histone+Deacetylase+%28Hdac%29+Pathways+to+Regulate+the+Functional+Transition+of+Bromodomain-Containing+4+%28Brd4%29+for+Inducible+Gene+Expression&doi=10.1074%2Fjbc.m114.570812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Histone cross-talk connects protein phosphatase 1α (PP1α) and histone deacetylase (HDAC) pathways to regulate the functional transition of bromodomain-containing 4 (BRD4) for inducible gene expression</span></div><div class="casAuthors">Hu, Xiangming; Lu, Xiaodong; Liu, Runzhong; Ai, Nanping; Cao, Zhenhua; Li, Yannan; Liu, Jiangfang; Yu, Bin; Liu, Kai; Wang, Huiping; Zhou, Chao; Wang, Yu; Han, Aidong; Ding, Feng; Chen, Ruichuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">23154-23167</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcription elongation has been recognized as a rate-limiting step for the expression of signal-inducible genes.  Through recruitment of pos. transcription elongation factor P-TEFb, the bromodomain-contg. protein BRD4 plays crit. roles in regulating the transcription elongation of a vast array of inducible genes that are important for multiple cellular processes.  The diverse biol. roles of BRD4 have been proposed to rely on its functional transition between chromatin targeting and transcription regulation.  The signaling pathways and the mol. mechanism for regulating this transition process, however, are largely unknown.  Here, the authors report a novel role of phosphorylated Ser10 of histone H3 (H3S10ph) in governing the functional transition of BRD4.  The authors identified that the acetylated lysines 5 and 8 of nucleosomal histone H4 (H4K5ac/K8ac) is the BRD4 binding site, and the protein phosphatase PP1α and class I histone deacetylase (HDAC1/2/3) signaling pathways are essential for the stress-induced BRD4 release from chromatin.  In the unstressed state, phosphorylated H3S10 prevents the deacetylation of nucleosomal H4K5ac/K8ac by HDAC1/2/3, thereby locking up the majority of BRD4 onto chromatin.  Upon stress, PP1α-mediated dephosphorylation of H3S10ph allows the deacetylation of nucleosomal H4K5ac/K8ac by HDAC1/2/3, thereby leading to the release of chromatin-bound BRD4 for subsequent recruitment of P-TEFb to enhance the expression of inducible genes.  Therefore, this study revealed a novel mechanism that the histone cross-talk between H3S10ph and H4K5ac/K8ac connects PP1α and HDACs to govern the functional transition of BRD4.  Combined with previous studies on the regulation of P-TEFb activation, the intricate signaling network for the tight control of transcription elongation is established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLzn-a41szPbVg90H21EOLACvtfcHk0ljINJi09mLPFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGmtbjI&md5=b3b79f952cd57bffe39c6d563d564669</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m114.570812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m114.570812%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DAi%26aufirst%3DN.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DA.%26aulast%3DDing%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DR.%26atitle%3DHistone%2520Cross-Talk%2520Connects%2520Protein%2520Phosphatase%25201%2520Alpha%2520%2528Pp1%2520Alpha%2529%2520and%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Pathways%2520to%2520Regulate%2520the%2520Functional%2520Transition%2520of%2520Bromodomain-Containing%25204%2520%2528Brd4%2529%2520for%2520Inducible%2520Gene%2520Expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D23154%26epage%3D23167%26doi%3D10.1074%2Fjbc.m114.570812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegwitz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosinsky, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgartner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siveke, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schildhaus, H.-U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Class-I Inhibition Promotes Epithelial Gene Expression in Pancreatic Cancer Cells in a Brd4- and Myc-Dependent Manner</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">6334</span>– <span class="NLM_lpage">6349</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1093%2Fnar%2Fgkx212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28369619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1alt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=6334-6349&author=V.+K.+Mishraauthor=F.+Wegwitzauthor=R.+L.+Kosinskyauthor=M.+Senauthor=R.+Baumgartnerauthor=T.+Wulffauthor=J.+T.+Sivekeauthor=H.-U.+Schildhausauthor=Z.+Najafovaauthor=V.+Kariauthor=H.+Kohlhofauthor=E.+Hessmannauthor=S.+A.+Johnsen&title=Histone+Deacetylase+Class-I+Inhibition+Promotes+Epithelial+Gene+Expression+in+Pancreatic+Cancer+Cells+in+a+Brd4-+and+Myc-Dependent+Manner&doi=10.1093%2Fnar%2Fgkx212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase class-I inhibition promotes epithelial gene expression in pancreatic cancer cells in a BRD4- and MYC-dependent manner</span></div><div class="casAuthors">Mishra, Vivek Kumar; Wegwitz, Florian; Kosinsky, Robyn Laura; Sen, Madhobi; Baumgartner, Roland; Wulff, Tanja; Siveke, Jens T.; Schildhaus, Hans-Ulrich; Najafova, Zeynab; Kari, Vijayalakshmi; Kohlhof, Hella; Hessmann, Elisabeth; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6334-6349</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with a particularly dismal prognosis.  Histone deacetylases (HDAC) are epigenetic modulators whose activity is frequently deregulated in various cancers including PDAC.  In particular, class-I HDACs (HDAC 1, 2, 3 and 8) have been shown to play an important role in PDAC.  In this study, we investigated the effects of the class Ispecific HDAC inhibitor (HDACi) 4SC-202 in multiple PDAC cell lines in promoting tumor cell differentiation.  We show that 4SC-202 neg. affects TGF signaling and inhibits TGFβ-induced epithelial-mesenchymal transition (EMT).  Moreover, 4SC-202 markedly induced p21 (CDKN1A) expression and significantly attenuated cell proliferation.  Mechanistically, genome-wide studies revealed that 4SC-202- induced genes were enriched for bromodomaincontaining Protein-4 (BRD4) and MYC occupancy.  BRD4, a well-characterized acetyllysine reader, has been shown to play a major role in regulating transcription of selected subsets of genes.  Importantly, BRD4 and MYC are essential for the expression of a subgroup of genes induced by class-I HDACi.  Taken together, our study uncovers a previously unknown role of BRD4 and MYC in eliciting the HDACimediated induction of a subset of genes and provides mol. insight into the mechanisms of HDACi action in PDAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTaKoSDVpOerVg90H21EOLACvtfcHk0ljINJi09mLPFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1alt70%253D&md5=7852bb6d93a7791e99fd183cbebc3214</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx212%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DV.%2BK.%26aulast%3DWegwitz%26aufirst%3DF.%26aulast%3DKosinsky%26aufirst%3DR.%2BL.%26aulast%3DSen%26aufirst%3DM.%26aulast%3DBaumgartner%26aufirst%3DR.%26aulast%3DWulff%26aufirst%3DT.%26aulast%3DSiveke%26aufirst%3DJ.%2BT.%26aulast%3DSchildhaus%26aufirst%3DH.-U.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DKari%26aufirst%3DV.%26aulast%3DKohlhof%26aufirst%3DH.%26aulast%3DHessmann%26aufirst%3DE.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DHistone%2520Deacetylase%2520Class-I%2520Inhibition%2520Promotes%2520Epithelial%2520Gene%2520Expression%2520in%2520Pancreatic%2520Cancer%2520Cells%2520in%2520a%2520Brd4-%2520and%2520Myc-Dependent%2520Manner%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D6334%26epage%3D6349%26doi%3D10.1093%2Fnar%2Fgkx212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span> <span> </span><span class="NLM_article-title">Feedback Activation of Leukemia Inhibitory Factor Receptor Limits Response to Histone Deacetylase Inhibitors in Breast Cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ccell.2016.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27622335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=459-473&author=H.+Zengauthor=J.+Quauthor=N.+Jinauthor=J.+Xuauthor=C.+Linauthor=Y.+Chenauthor=X.+Yangauthor=X.+Heauthor=S.+Tangauthor=X.+Lanauthor=X.+Yangauthor=Z.+Chenauthor=M.+Huangauthor=J.+Dingauthor=M.+Geng&title=Feedback+Activation+of+Leukemia+Inhibitory+Factor+Receptor+Limits+Response+to+Histone+Deacetylase+Inhibitors+in+Breast+Cancer&doi=10.1016%2Fj.ccell.2016.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Feedback Activation of Leukemia Inhibitory Factor Receptor Limits Response to Histone Deacetylase Inhibitors in Breast Cancer</span></div><div class="casAuthors">Zeng, Hanlin; Qu, Jia; Jin, Nan; Xu, Jun; Lin, Chenchu; Chen, Yi; Yang, Xinying; He, Xiang; Tang, Shuai; Lan, Xiaojing; Yang, Xiaotong; Chen, Ziqi; Huang, Min; Ding, Jian; Geng, Meiyu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-473</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors have demonstrated clin. benefits in subtypes of hematol. malignancies.  However, the efficacy of HDAC inhibitors in solid tumors remains uncertain.  This study takes breast cancer as a model to understand mechanisms accounting for limited response of HDAC inhibitors in solid tumors and to seek combination solns.  We discover that feedback activation of leukemia inhibitory factor receptor (LIFR) signaling in breast cancer limits the response to HDAC inhibition.  Mechanistically, HDAC inhibition increases histone acetylation at the LIFR gene promoter, which recruits bromodomain protein BRD4, upregulates LIFR expression, and activates JAK1-STAT3 signaling.  Importantly, JAK1 or BRD4 inhibition sensitizes breast cancer to HDAC inhibitors, implicating combination inhibition of HDAC with JAK1 or BRD4 as potential therapies for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL24sYbhFYeLVg90H21EOLACvtfcHk0ljv_TU6EtT9FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI&md5=9ce620c7a19519a95edd714f8012518c</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DLan%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26atitle%3DFeedback%2520Activation%2520of%2520Leukemia%2520Inhibitory%2520Factor%2520Receptor%2520Limits%2520Response%2520to%2520Histone%2520Deacetylase%2520Inhibitors%2520in%2520Breast%2520Cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D459%26epage%3D473%26doi%3D10.1016%2Fj.ccell.2016.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bet Bromodomain Inhibitors and Their Role in Target Validation</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1039/c3md00291h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1039%2Fc3md00291h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=288-296&author=S.+M%C3%BCllerauthor=S.+Knapp&title=Discovery+of+Bet+Bromodomain+Inhibitors+and+Their+Role+in+Target+Validation&doi=10.1039%2Fc3md00291h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of BET bromodomain inhibitors and their role in target validation</span></div><div class="casAuthors">Mueller, S.; Knapp, S.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-296</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are protein interaction modules that selectively recognize .vepsiln.-N-acetylated lysine residues.  BRDs are present in diverse proteins that play key functions in chromatin organization and regulation of gene transcription.  Aberrant transcription is a hallmark of many diseases in particular cancer and inflammation.  The complexity of mol. processes regulating gene transcription identified transcriptional regulators as interesting targets for the development of specific chem. tool mols. (chem. probes) that help to understand the mol. mechanisms of transcription and to explore the potential of BRD mediated interactions as sites for pharmaceutical intervention.  Recently a no. of highly specific inhibitors have been developed against the BET (bromo and extra terminal) family of bromodomains.  The availability of selective BRD inhibitors had a significant impact on the validation of bromodomain-contg. proteins as targets for drug development and for our understanding of the biol. roles of these proteins.  In this review we will summarize the discovery of BET bromodomain inhibitors and their roles in target validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbUCy0a3RkObVg90H21EOLACvtfcHk0ljv_TU6EtT9FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymu70%253D&md5=88e7b8c3909462666a083cad35efb135</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2Fc3md00291h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00291h%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Bet%2520Bromodomain%2520Inhibitors%2520and%2520Their%2520Role%2520in%2520Target%2520Validation%26jtitle%3DMedchemcomm%26date%3D2014%26volume%3D5%26spage%3D288%26epage%3D296%26doi%3D10.1039%2Fc3md00291h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting Epigenetic Reader and Eraser: Rational Design, Synthesis and in Vitro Evaluation of Dimethylisoxazoles Derivatives as Brd4/Hdac Dual Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2931</span>– <span class="NLM_lpage">2935</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmcl.2016.04.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27142751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2931-2935&author=Z.+Zhangauthor=S.+Houauthor=H.+Chenauthor=T.+Ranauthor=F.+Jiangauthor=Y.+Bianauthor=D.+Zhangauthor=Y.+Zhiauthor=L.+Wangauthor=L.+Zhangauthor=H.+Liauthor=Y.+Zhangauthor=W.+Tangauthor=T.+Luauthor=Y.+Chen&title=Targeting+Epigenetic+Reader+and+Eraser%3A+Rational+Design%2C+Synthesis+and+in+Vitro+Evaluation+of+Dimethylisoxazoles+Derivatives+as+Brd4%2FHdac+Dual+Inhibitors&doi=10.1016%2Fj.bmcl.2016.04.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic reader and eraser: Rational design, synthesis and in vitro evaluation of dimethylisoxazoles derivatives as BRD4/HDAC dual inhibitors</span></div><div class="casAuthors">Zhang, Zhimin; Hou, Shaohua; Chen, Hongli; Ran, Ting; Jiang, Fei; Bian, Yuanyuan; Zhang, Dewei; Zhi, Yanle; Wang, Lu; Zhang, Li; Li, Hongmei; Zhang, Yanmin; Tang, Weifang; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2931-2935</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bromodomain protein module and histone deacetylase (HDAC), which recognize and remove acetylated lysine, resp., have emerged as important epigenetic therapeutic targets in cancer treatments.  Herein the authors presented a novel design approach for cancer drug development by combination of bromodomain and HDAC inhibitory activity in one mol.  The designed compds. were synthesized which showed inhibitory activity against bromodomain 4 and HDAC1.  The representative dual bromodomain/HDAC inhibitors, compds. N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N7-hydroxyheptanediamide and N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N8-hydroxyoctanediamide showed potent antiproliferative activities against human leukemia cell line K562 and MV4-11 in cellular assays.  This work may lay the foundation for developing dual bromodomain/HDAC inhibitors as potential anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriS0VB1G-QoLVg90H21EOLACvtfcHk0liiYrhdqPR0Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D&md5=a6ebdbd6402e3117a2f9d96ed071790c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.034%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHou%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DBian%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DTargeting%2520Epigenetic%2520Reader%2520and%2520Eraser%253A%2520Rational%2520Design%252C%2520Synthesis%2520and%2520in%2520Vitro%2520Evaluation%2520of%2520Dimethylisoxazoles%2520Derivatives%2520as%2520Brd4%252FHdac%2520Dual%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2931%26epage%3D2935%26doi%3D10.1016%2Fj.bmcl.2016.04.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Potent Dual Bet/Hdac Inhibitors for Efficient Treatment of Pancreatic Cancer</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3028</span>– <span class="NLM_lpage">3032</span>, <span class="refDoi"> DOI: 10.1002/anie.201915896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1002%2Fanie.201915896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=3028-3032&author=S.+Heauthor=G.+Dongauthor=Y.+Liauthor=S.+Wuauthor=W.+Wangauthor=C.+Sheng&title=Potent+Dual+Bet%2FHdac+Inhibitors+for+Efficient+Treatment+of+Pancreatic+Cancer&doi=10.1002%2Fanie.201915896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Dual BET/HDAC Inhibitors for Efficient Treatment of Pancreatic Cancer</span></div><div class="casAuthors">He, Shipeng; Dong, Guoqiang; Li, Yu; Wu, Shanchao; Wang, Wei; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3028-3032</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">As one of the most aggressive and lethal human malignancies with extremely poor prognosis, there is an urgent demand of more effective therapy for the treatment of pancreatic cancer.  Reported here is a new, effective therapeutic strategy and the design of small-mol. inhibitors that simultaneously target bromodomain and extra-terminal (BET) and histone deacetylase (HDAC), potentially serving as promising therapeutic agents for pancreatic cancer.  A highly potent dual inhibitor (13 a) is identified to possess excellent and balanced activities against BRD4 BD1 (IC50=11 nM) and HDAC1 (IC50=21 nM).  Notably, this compd. shows higher in vitro and in vivo antitumor potency than the BET inhibitor (+)-JQ1 and the HDAC inhibitor vorinostat, either alone or and in combination, highlighting the advantages of BET/HDAC dual inhibitors for more effective treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BFNcLfkA4bVg90H21EOLACvtfcHk0liiYrhdqPR0Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D&md5=78db62cb533611939d9abd0e654ff5f3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201915896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201915896%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DPotent%2520Dual%2520Bet%252FHdac%2520Inhibitors%2520for%2520Efficient%2520Treatment%2520of%2520Pancreatic%2520Cancer%26jtitle%3DAngew.%2520Chem.%26date%3D2020%26volume%3D59%26spage%3D3028%26epage%3D3032%26doi%3D10.1002%2Fanie.201915896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design, Synthesis and in Vitro Antiproliferative Effects Studies of Novel Dual Brd4/Hdac Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4051</span>– <span class="NLM_lpage">4055</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.07.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmcl.2017.07.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28765013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=4051-4055&author=M.+Shaoauthor=L.+Heauthor=L.+Zhengauthor=L.+Huangauthor=Y.+Zhouauthor=T.+Wangauthor=Y.+Chenauthor=M.+Shenauthor=F.+Wangauthor=Z.+Yangauthor=L.+Chen&title=Structure-Based+Design%2C+Synthesis+and+in+Vitro+Antiproliferative+Effects+Studies+of+Novel+Dual+Brd4%2FHdac+Inhibitors&doi=10.1016%2Fj.bmcl.2017.07.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design, synthesis and in vitro antiproliferative effects studies of novel dual BRD4/HDAC inhibitors</span></div><div class="casAuthors">Shao, Mingfeng; He, Linhong; Zheng, Li; Huang, Lingxiao; Zhou, Yuanyuan; Wang, Taijing; Chen, Yong; Shen, Mingsheng; Wang, Fang; Yang, Zhuang; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4051-4055</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone acetylation marks play important roles in controlling gene expressions and are removed by histone deacetylases (HDACs).  These marks are read by bromodomain and extra-terminal (BET) proteins, whose targeted inhibitors are under clin. investigation.  BET and HDAC inhibitors have been demonstrated to be synergistically killing in Myc-induced murine lymphoma.  Herein, the authors combine the inhibitory activities of BET and HDAC into one mol. through structure-based design method and evaluate its function.  The majority of these synthesized compds. showed inhibitory activity against second bromodomains(BRD) of BRD4 and HDAC1.  Among them, I presented antiproliferative effects against human acute myelogenous leukemia (AML) cell lines in vitro, and I is confirmed to reduce the expression of Myc by Western blot anal.  Those results indicated that I is a potent dual BRD4/HDAC inhibitor and deserves further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkMaHqVu_LM7Vg90H21EOLACvtfcHk0lgdXpLf-AaVOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP&md5=c525f1a2083699dd5f6f6a262f477f24</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.07.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.07.054%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DStructure-Based%2520Design%252C%2520Synthesis%2520and%2520in%2520Vitro%2520Antiproliferative%2520Effects%2520Studies%2520of%2520Novel%2520Dual%2520Brd4%252FHdac%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D4051%26epage%3D4055%26doi%3D10.1016%2Fj.bmcl.2017.07.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (Brd4) Inhibitor That Induces Amp-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9990</span>– <span class="NLM_lpage">10012</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00275</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00275" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9990-10012&author=L.+Ouyangauthor=L.+Zhangauthor=J.+Liuauthor=L.+Fuauthor=D.+Yaoauthor=Y.+Zhaoauthor=S.+Zhangauthor=G.+Wangauthor=G.+Heauthor=B.+Liu&title=Discovery+of+a+Small-Molecule+Bromodomain-Containing+Protein+4+%28Brd4%29+Inhibitor+That+Induces+Amp-Activated+Protein+Kinase-Modulated+Autophagy-Associated+Cell+Death+in+Breast+Cancer&doi=10.1021%2Facs.jmedchem.7b00275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (BRD4) Inhibitor That Induces AMP-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span></div><div class="casAuthors">Ouyang, Liang; Zhang, Lan; Liu, Jie; Fu, Leilei; Yao, Dahong; Zhao, Yuqian; Zhang, Shouyue; Wang, Guan; He, Gu; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9990-10012</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Upon the basis of The Cancer Genome Atlas (TCGA) data set, the authors identified that several autophagy-related proteins such as AMP-activated protein kinase (AMPK) were remarkably downregulated in breast cancer.  Combined with coimmunopptn. assay, the authors demonstrated that BRD4 might interact with AMPK.  After analyses of the pharmacophore and WPF interaction optimization, the authors designed a small-mol. inhibitor of BRD4, 9f (FL-411 (2-(4-hydroxy-3,5-dimethylphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4',3':4,5]thieno[2,3-d]pyrimidin-4(3H)-one)) which was validated by cocrystal structure with BD1 of BRD4.  Subsequently, 9f was discovered to induce ATG5-dependent autophagy-assocd. cell death (ACD) by blocking BRD4-AMPK interaction and thus activating AMPK-mTOR-ULK1-modulated autophagic pathway in breast cancer cells.  Interestingly, the iTRAQ-based proteomics analyses revealed that 9f induced ACD pathways involved in HMGB1, VDAC1/2, and eEF2.  Moreover, 9f displayed a therapeutic potential on both breast cancer xenograft mouse and zebrafish models.  Together, these results demonstrate that a novel small-mol. inhibitor of BRD4 induces BRD4-AMPK-modulated ACD in breast cancer, which may provide a candidate drug for future cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Ocrq2wyy6bVg90H21EOLACvtfcHk0lgdXpLf-AaVOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF&md5=ac15f7e108511ea207f2c2fcd299b4e5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00275%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Bromodomain-Containing%2520Protein%25204%2520%2528Brd4%2529%2520Inhibitor%2520That%2520Induces%2520Amp-Activated%2520Protein%2520Kinase-Modulated%2520Autophagy-Associated%2520Cell%2520Death%2520in%2520Breast%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9990%26epage%3D10012%26doi%3D10.1021%2Facs.jmedchem.7b00275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Small-Molecule Inhibitors of Brd4 Using Structure-Based Virtual Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8073</span>– <span class="NLM_lpage">8088</span>, <span class="refDoi"> DOI: 10.1021/jm4011302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+Novel+Small-Molecule+Inhibitors+of+Brd4+Using+Structure-Based+Virtual+Screening&doi=10.1021%2Fjm4011302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0lgAm4FxQGX26Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Small-Molecule%2520Inhibitors%2520of%2520Brd4%2520Using%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088%26doi%3D10.1021%2Fjm4011302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruckman, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herns, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spector, S. A.</span></span> <span> </span><span class="NLM_article-title">Induction of Autophagy by Pi3k/Mtor and Pi3k/Mtor/Brd4 Inhibitors Suppresses Hiv-1 Replication</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">5808</span>– <span class="NLM_lpage">5820</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.002353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1074%2Fjbc.ra118.002353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29475942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFSqu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=5808-5820&author=G.+R.+Campbellauthor=R.+S.+Bruckmanauthor=S.+D.+Hernsauthor=S.+Joshiauthor=D.+L.+Durdenauthor=S.+A.+Spector&title=Induction+of+Autophagy+by+Pi3k%2FMtor+and+Pi3k%2FMtor%2FBrd4+Inhibitors+Suppresses+Hiv-1+Replication&doi=10.1074%2Fjbc.ra118.002353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of autophagy by PI3K/MTOR and PI3K/MTOR/BRD4 inhibitors suppresses HIV-1 replication</span></div><div class="casAuthors">Campbell, Grant R.; Bruckman, Rachel S.; Herns, Shayna D.; Joshi, Shweta; Durden, Donald L.; Spector, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5808-5820</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In this study, we investigated the effects of the dual phosphatidylinositol 3-kinase/mechanistic target of rapamycin (PI3K/MTOR) inhibitor dactolisib (NVP-BEZ235), the PI3K/MTOR/bromodomain-contg. protein 4 (BRD4) inhibitor SF2523, and the bromodomain and extra terminal domain inhibitor JQ1 on the productive infection of primary macrophages with human immunodeficiency type-1 (HIV).  These inhibitors did not alter the initial susceptibility of macrophages to HIV infection.  However, dactolisib, JQ1, and SF2523 all decreased HIV replication in macrophages in a dose-dependent manner via degrdn. of intracellular HIV through autophagy.  Macrophages treated with dactolisib, JQ1, or SF2523 displayed an increase in LC3B lipidation combined with SQSTM1 degrdn. without inducing increased cell death.  LC3B-II levels were further increased in the presence of pepstatin A suggesting that these inhibitors induce autophagic flux.  RNA interference for ATG5 and ATG7 and pharmacol. inhibitors of autophagosome-lysosome fusion and of lysosomal hydrolases all blocked the inhibition of HIV.  Thus, we demonstrate that the mechanism of PI3K/MTOR and PI3K/MTOR/BRD4 inhibitor suppression of HIV requires the formation of autophagosomes, as well as their subsequent maturation into autolysosomes.  These data provide further evidence in support of a role for autophagy in the control of HIV infection and open new avenues for the use of this class of drugs in HIV therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8-D9wUfaylbVg90H21EOLACvtfcHk0lgAm4FxQGX26Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFSqu74%253D&md5=06a414ed7a31e34ed7cfd9bb28b48fdd</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1074%2Fjbc.ra118.002353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.ra118.002353%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DG.%2BR.%26aulast%3DBruckman%26aufirst%3DR.%2BS.%26aulast%3DHerns%26aufirst%3DS.%2BD.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26aulast%3DSpector%26aufirst%3DS.%2BA.%26atitle%3DInduction%2520of%2520Autophagy%2520by%2520Pi3k%252FMtor%2520and%2520Pi3k%252FMtor%252FBrd4%2520Inhibitors%2520Suppresses%2520Hiv-1%2520Replication%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D5808%26epage%3D5820%26doi%3D10.1074%2Fjbc.ra118.002353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamaki, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K. M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Regulation of Autophagy and Lysosomal Function by Bromodomain Protein Brd4</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2006</span>– <span class="NLM_lpage">2007</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1364822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1080%2F15548627.2017.1364822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28933601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=2006-2007&author=J.-I.+Sakamakiauthor=K.+M.+Ryan&title=Transcriptional+Regulation+of+Autophagy+and+Lysosomal+Function+by+Bromodomain+Protein+Brd4&doi=10.1080%2F15548627.2017.1364822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation of autophagy and lysosomal function by bromodomain protein BRD4</span></div><div class="casAuthors">Sakamaki, Jun-ichi; Ryan, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2006-2007</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Macroautophagy/autophagy is an intracellular recycling system that delivers cytoplasmic organelles and materials to lysosomes for degrdn.  This process is operated by autophagy-related (ATG) genes and tightly controlled by stress-responsive signaling pathways.  Our recent study revealed that autophagy programs are transcriptionally suppressed by the BET family protein BRD4.  This repression is alleviated during nutrient deprivation through the AMPK-SIRT1 pathway.  Our findings therefore provide new insights into the regulation of autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeGupyW9sug7Vg90H21EOLACvtfcHk0lgAm4FxQGX26Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbJ&md5=a5740d62bf9d4faaf0206c2e81e2d922</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1364822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1364822%26sid%3Dliteratum%253Aachs%26aulast%3DSakamaki%26aufirst%3DJ.-I.%26aulast%3DRyan%26aufirst%3DK.%2BM.%26atitle%3DTranscriptional%2520Regulation%2520of%2520Autophagy%2520and%2520Lysosomal%2520Function%2520by%2520Bromodomain%2520Protein%2520Brd4%26jtitle%3DAutophagy%26date%3D2017%26volume%3D13%26spage%3D2006%26epage%3D2007%26doi%3D10.1080%2F15548627.2017.1364822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Brd4 Is a Newly Characterized Transcriptional Regulator That Represses Autophagy and Lysosomal Function</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1801</span>– <span class="NLM_lpage">1803</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1364334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1080%2F15548627.2017.1364334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28837371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1801-1803&author=X.+Wenauthor=D.+J.+Klionsky&title=Brd4+Is+a+Newly+Characterized+Transcriptional+Regulator+That+Represses+Autophagy+and+Lysosomal+Function&doi=10.1080%2F15548627.2017.1364334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 is a newly characterized transcriptional regulator that represses autophagy and lysosomal function</span></div><div class="casAuthors">Wen, Xin; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1801-1803</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">As a member of the bromodomain and extraterminal (BET) family, BRD4 (bromodomain contg. 4) can bind to acetylated histones and transcription factors, and is also able to recruit various transcriptional regulators.  Previous studies have shown that BRD4 mainly plays a pos. role in cell growth and cell cycle progression.  In a recent study conducted by Sakamaki et al., the authors found that BRD4 acts as a transcriptional repressor in regulating macroautophagy/autophagy and lysosome gene expression, via binding to the histone lysine methyltransferase EHMT2/G9a.  The oncoprotein BRD4-NUTM1/NUT also helps block autophagy and lysosome functions.  A knockdown of BRD4 allows the retention of MTOR activity during starvation and significantly undermines starvation-induced cell death.  However, BRD4 repression only contributes to stimulus-dependent autophagy and aggrephagy, and is not involved in other types of selective autophagy.  Taken together, the newly reported repressive role of BRD4 in autophagy adds to our understanding of how autophagy and lysosome functions are regulated at the transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5yO5wGDB31LVg90H21EOLACvtfcHk0liKQgISzPIlWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbO&md5=39c20dc548ec31554a71f5a69b4a6361</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1364334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1364334%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DX.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DBrd4%2520Is%2520a%2520Newly%2520Characterized%2520Transcriptional%2520Regulator%2520That%2520Represses%2520Autophagy%2520and%2520Lysosomal%2520Function%26jtitle%3DAutophagy%26date%3D2017%26volume%3D13%26spage%3D1801%26epage%3D1803%26doi%3D10.1080%2F15548627.2017.1364334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eom, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeung, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y. H.</span></span> <span> </span><span class="NLM_article-title">Ampk-Ulk1-Mediated Autophagy Confers Resistance to Bet Inhibitor Jq1 in Acute Myeloid Leukemia Stem Cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2794</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-1903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1078-0432.ccr-16-1903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27864418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2781-2794&author=J.+E.+Jangauthor=J.-I.+Eomauthor=H.-K.+Jeungauthor=J.-W.+Cheongauthor=J.+Y.+Leeauthor=J.+S.+Kimauthor=Y.+H.+Min&title=Ampk-Ulk1-Mediated+Autophagy+Confers+Resistance+to+Bet+Inhibitor+Jq1+in+Acute+Myeloid+Leukemia+Stem+Cells&doi=10.1158%2F1078-0432.ccr-16-1903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AMPK-ULK1-Mediated Autophagy Confers Resistance to BET Inhibitor JQ1 in Acute Myeloid Leukemia Stem Cells</span></div><div class="casAuthors">Jang, Ji Eun; Eom, Ju-In; Jeung, Hoi-Kyung; Cheong, June-Won; Lee, Jung Yeon; Kim, Jin Seok; Min, Yoo Hong</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2781-2794</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Bromodomain and extraterminal domain (BET) inhibitors are promising epigenetic agents for the treatment of various subsets of acute myeloid leukemia (AML).  However, the resistance of leukemia stem cells (LSC) to BET inhibitors remains a major challenge.  In this study, we evaluated the mechanisms underlying LSC resistance to the BET inhibitor JQ1.  Exptl. Design: We evaluated the levels of apoptosis and autophagy induced by JQ1 in LSC-like leukemia cell lines and primary CD34+CD38- leukemic blasts obtained from AML cases with normal karyotype without recurrent mutations.  Results: JQ1 effectively induced apoptosis in a concn.-dependent manner in JQ1-sensitive AML cells.  However, in JQ1-resistant AML LSCs, JQ1 induced little apoptosis and led to upregulation of beclin-1, increased LC3-II lipidation, formation of autophagosomes, and downregulation of p62/SQSTM1.  Inhibition of autophagy by pharmacol. inhibitors or knockdown of beclin-1 using specific siRNA enhanced JQ1-induced apoptosis in resistant cells, indicating that prosurvival autophagy occurred in these cells.  Independent of mTOR signaling, activation of the AMPK (pThr172)/ULK1 (pSer555) pathway was found to be assocd. with JQ1-induced autophagy in resistant cells.  AMPK inhibition using the pharmacol. inhibitor compd. C or by knockdown of AMPKα suppressed autophagy and promoted JQ1-induced apoptosis in AML LSCs.  Conclusions: These findings revealed that prosurvival autophagy was one of the mechanisms involved in the resistance AML LSCs to JQ1.  Targeting the AMPK/ULK1 pathway or inhibition of autophagy could be an effective therapeutic strategy for combating resistance to BET inhibitors in AML and other types of cancer.  Clin Cancer Res; 23(11); 2781-94. 2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHOzsCo6fOvLVg90H21EOLACvtfcHk0liKQgISzPIlWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D&md5=f15df13e0addda969161b31973857176</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-16-1903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-16-1903%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BE.%26aulast%3DEom%26aufirst%3DJ.-I.%26aulast%3DJeung%26aufirst%3DH.-K.%26aulast%3DCheong%26aufirst%3DJ.-W.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DMin%26aufirst%3DY.%2BH.%26atitle%3DAmpk-Ulk1-Mediated%2520Autophagy%2520Confers%2520Resistance%2520to%2520Bet%2520Inhibitor%2520Jq1%2520in%2520Acute%2520Myeloid%2520Leukemia%2520Stem%2520Cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D2781%26epage%3D2794%26doi%3D10.1158%2F1078-0432.ccr-16-1903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 16 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yuhao Ren, Shanshan Li, Ren Zhu, Chengying Wan, Dongmei Song, Jiawen Zhu, Guiping Cai, Sihui Long, Lingyi Kong, <span class="NLM_string-name hlFld-ContribAuthor">Wenying Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of STAT3 and Histone Deacetylase (HDAC) Dual-Pathway Inhibitors for the Treatment of Solid Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7468-7482. <a href="https://doi.org/10.1021/acs.jmedchem.1c00136" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00136</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00136%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSTAT3%252Band%252BHistone%252BDeacetylase%252B%252528HDAC%252529%252BDual-Pathway%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BSolid%252BCancer%26aulast%3DRen%26aufirst%3DYuhao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D24012021%26date%3D27052021%26volume%3D64%26issue%3D11%26spage%3D7468%26epage%3D7482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tingting Liu, Yichao Wan, Yuliang Xiao, Chengcai Xia, <span class="NLM_string-name hlFld-ContribAuthor">Guiyun Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Dual-Target Inhibitors Based on HDACs: Novel Antitumor Agents for Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 8977-9002. <a href="https://doi.org/10.1021/acs.jmedchem.0c00491" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00491%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDual-Target%252BInhibitors%252BBased%252Bon%252BHDACs%25253A%252BNovel%252BAntitumor%252BAgents%252Bfor%252BCancer%252BTherapy%26aulast%3DLiu%26aufirst%3DTingting%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D23032020%26date%3D30042020%26date%3D22042020%26volume%3D63%26issue%3D17%26spage%3D8977%26epage%3D9002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Shao-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jing  Shi</span>, <span class="hlFld-ContribAuthor ">Lin-Feng  Jin</span>, <span class="hlFld-ContribAuthor ">Tong  Yu</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of dual inhibitors targeting crosstalk between histone epigenetic modulators for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113588. <a href="https://doi.org/10.1016/j.ejmech.2021.113588" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113588</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113588%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252Bdual%252Binhibitors%252Btargeting%252Bcrosstalk%252Bbetween%252Bhistone%252Bepigenetic%252Bmodulators%252Bfor%252Bcancer%252Btherapy%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2021%26volume%3D222%26spage%3D113588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ma  Su</span>, <span class="hlFld-ContribAuthor ">Xingyu  Gong</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">An update on the emerging approaches for histone deacetylase (HDAC) inhibitor drug discovery and future perspectives. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 745-761. <a href="https://doi.org/10.1080/17460441.2021.1877656" title="DOI URL">https://doi.org/10.1080/17460441.2021.1877656</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1877656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1877656%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAn%252Bupdate%252Bon%252Bthe%252Bemerging%252Bapproaches%252Bfor%252Bhistone%252Bdeacetylase%252B%252528HDAC%252529%252Binhibitor%252Bdrug%252Bdiscovery%252Band%252Bfuture%252Bperspectives%26aulast%3DSu%26aufirst%3DMa%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D7%26spage%3D745%26epage%3D761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai-Jiang  Liu</span>, <span class="hlFld-ContribAuthor ">Qian  Zhao</span>, <span class="hlFld-ContribAuthor ">Cheng  Peng</span>, <span class="hlFld-ContribAuthor ">Qing  Mao</span>, <span class="hlFld-ContribAuthor ">Fengbo  Wu</span>, <span class="hlFld-ContribAuthor ">Feng-Hua  Zhang</span>, <span class="hlFld-ContribAuthor ">Quan-Sheng  Feng</span>, <span class="hlFld-ContribAuthor ">Gu  He</span>, <span class="hlFld-ContribAuthor ">Bo  Han</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of nitroisoxazole-containing spiro[pyrrolidin-oxindole] derivatives as novel glutathione peroxidase 4/mouse double minute 2 dual inhibitors that inhibit breast adenocarcinoma cell proliferation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>217 </em>, 113359. <a href="https://doi.org/10.1016/j.ejmech.2021.113359" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113359%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252Bnitroisoxazole-containing%252Bspiro%25255Bpyrrolidin-oxindole%25255D%252Bderivatives%252Bas%252Bnovel%252Bglutathione%252Bperoxidase%252B4%25252Fmouse%252Bdouble%252Bminute%252B2%252Bdual%252Binhibitors%252Bthat%252Binhibit%252Bbreast%252Badenocarcinoma%252Bcell%252Bproliferation%26aulast%3DLiu%26aufirst%3DShuai-Jiang%26date%3D2021%26volume%3D217%26spage%3D113359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qinghua  Ren</span>, <span class="hlFld-ContribAuthor ">Wenqian  Gao</span>. </span><span class="cited-content_cbyCitation_article-title">Current status in the discovery of dual BET/HDAC inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>38 </em>, 127829. <a href="https://doi.org/10.1016/j.bmcl.2021.127829" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127829</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127829%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCurrent%252Bstatus%252Bin%252Bthe%252Bdiscovery%252Bof%252Bdual%252BBET%25252FHDAC%252Binhibitors%26aulast%3DRen%26aufirst%3DQinghua%26date%3D2021%26volume%3D38%26spage%3D127829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arshdeep  Singh</span>, <span class="hlFld-ContribAuthor ">Ting-Yu  Chang</span>, <span class="hlFld-ContribAuthor ">Navdeep  Kaur</span>, <span class="hlFld-ContribAuthor ">Kai-Cheng  Hsu</span>, <span class="hlFld-ContribAuthor ">Yun  Yen</span>, <span class="hlFld-ContribAuthor ">Tony Eight  Lin</span>, <span class="hlFld-ContribAuthor ">Mei-Jung  Lai</span>, <span class="hlFld-ContribAuthor ">Sung-Bau  Lee</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">CAP rigidification of MS-275 and chidamide leads to enhanced antiproliferative effects mediated through HDAC1, 2 and tubulin polymerization inhibition. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>215 </em>, 113169. <a href="https://doi.org/10.1016/j.ejmech.2021.113169" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113169</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113169%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCAP%252Brigidification%252Bof%252BMS-275%252Band%252Bchidamide%252Bleads%252Bto%252Benhanced%252Bantiproliferative%252Beffects%252Bmediated%252Bthrough%252BHDAC1%25252C%252B2%252Band%252Btubulin%252Bpolymerization%252Binhibition%26aulast%3DSingh%26aufirst%3DArshdeep%26date%3D2021%26volume%3D215%26spage%3D113169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jenna N.  Beyer</span>, <span class="hlFld-ContribAuthor ">Nicole R.  Raniszewski</span>, <span class="hlFld-ContribAuthor ">George M.  Burslem</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Opportunities in Epigenetic Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     , 17-42. <a href="https://doi.org/10.1002/cbic.202000459" title="DOI URL">https://doi.org/10.1002/cbic.202000459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000459%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DAdvances%252Band%252BOpportunities%252Bin%252BEpigenetic%252BChemical%252BBiology%26aulast%3DBeyer%26aufirst%3DJenna%2BN.%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D1%26spage%3D17%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qinghua  Ren</span>, <span class="hlFld-ContribAuthor ">Wenqian  Gao</span>. </span><span class="cited-content_cbyCitation_article-title">Current status in the discovery of dual BET/HDAC inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>31 </em>, 127671. <a href="https://doi.org/10.1016/j.bmcl.2020.127671" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127671%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCurrent%252Bstatus%252Bin%252Bthe%252Bdiscovery%252Bof%252Bdual%252BBET%25252FHDAC%252Binhibitors%26aulast%3DRen%26aufirst%3DQinghua%26date%3D2021%26volume%3D31%26spage%3D127671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingjing  Chen</span>, <span class="hlFld-ContribAuthor ">Yalei  Li</span>, <span class="hlFld-ContribAuthor ">Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Minmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Aihuan  Wei</span>, <span class="hlFld-ContribAuthor ">Hongchun  Liu</span>, <span class="hlFld-ContribAuthor ">Zhicheng  Xie</span>, <span class="hlFld-ContribAuthor ">Wenming  Ren</span>, <span class="hlFld-ContribAuthor ">Wenwen  Duan</span>, <span class="hlFld-ContribAuthor ">Zhuo  Zhang</span>, <span class="hlFld-ContribAuthor ">Aijun  Shen</span>, <span class="hlFld-ContribAuthor ">Youhong  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of selective HDAC/BRD4 dual inhibitors as epigenetic probes. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112868. <a href="https://doi.org/10.1016/j.ejmech.2020.112868" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112868</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112868%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bselective%252BHDAC%25252FBRD4%252Bdual%252Binhibitors%252Bas%252Bepigenetic%252Bprobes%26aulast%3DChen%26aufirst%3DJingjing%26date%3D2021%26volume%3D209%26spage%3D112868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amr K.A.  Bass</span>, <span class="hlFld-ContribAuthor ">Mona S.  El-Zoghbi</span>, <span class="hlFld-ContribAuthor ">El-Shimaa M.  Nageeb</span>, <span class="hlFld-ContribAuthor ">Mamdouh F.A.  Mohamed</span>, <span class="hlFld-ContribAuthor ">Mohamed  Badr</span>, <span class="hlFld-ContribAuthor ">Gamal El-Din A.  Abuo-Rahma</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive review for anticancer hybridized multitargeting HDAC inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112904. <a href="https://doi.org/10.1016/j.ejmech.2020.112904" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112904</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112904%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComprehensive%252Breview%252Bfor%252Banticancer%252Bhybridized%252Bmultitargeting%252BHDAC%252Binhibitors%26aulast%3DBass%26aufirst%3DAmr%2BK.A.%26date%3D2021%26volume%3D209%26spage%3D112904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ellen E.  Guest</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Pickett</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Hirst</span>. </span><span class="cited-content_cbyCitation_article-title">Structural variation of protein–ligand complexes of the first bromodomain of BRD4. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>1 </em><a href="https://doi.org/10.1039/D1OB00658D" title="DOI URL">https://doi.org/10.1039/D1OB00658D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00658D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00658D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DStructural%252Bvariation%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bcomplexes%252Bof%252Bthe%252Bfirst%252Bbromodomain%252Bof%252BBRD4%26aulast%3DGuest%26aufirst%3DEllen%2BE.%26date%3D2021%26date%3D2021%26volume%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Gao</span>, <span class="hlFld-ContribAuthor ">Hang  Zhang</span>, <span class="hlFld-ContribAuthor ">Frédéric  Lirussi</span>, <span class="hlFld-ContribAuthor ">Carmen  Garrido</span>, <span class="hlFld-ContribAuthor ">Xiang-Yang  Ye</span>, <span class="hlFld-ContribAuthor ">Tian  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Dual inhibitors of histone deacetylases and other cancer-related targets: A pharmacological perspective. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>182 </em>, 114224. <a href="https://doi.org/10.1016/j.bcp.2020.114224" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.114224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.114224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.114224%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DDual%252Binhibitors%252Bof%252Bhistone%252Bdeacetylases%252Band%252Bother%252Bcancer-related%252Btargets%25253A%252BA%252Bpharmacological%252Bperspective%26aulast%3DGao%26aufirst%3DYuan%26date%3D2020%26volume%3D182%26spage%3D114224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angel R.  de Lera</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Two-hit wonders: The expanding universe of multitargeting epigenetic agents. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 135-154. <a href="https://doi.org/10.1016/j.cbpa.2020.05.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTwo-hit%252Bwonders%25253A%252BThe%252Bexpanding%252Buniverse%252Bof%252Bmultitargeting%252Bepigenetic%252Bagents%26aulast%3Dde%2BLera%26aufirst%3DAngel%2BR.%26date%3D2020%26volume%3D57%26spage%3D135%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Yifan  Yu</span>, <span class="hlFld-ContribAuthor ">Zonghui  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Novel strategies targeting bromodomain-containing protein 4 (BRD4) for cancer drug discovery. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112426. <a href="https://doi.org/10.1016/j.ejmech.2020.112426" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112426%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252Bstrategies%252Btargeting%252Bbromodomain-containing%252Bprotein%252B4%252B%252528BRD4%252529%252Bfor%252Bcancer%252Bdrug%252Bdiscovery%26aulast%3DLiang%26aufirst%3DDailin%26date%3D2020%26volume%3D200%26spage%3D112426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of some known BRD4 and HDAC inhibitors: (A) selection of published BRD4 inhibitors with diverse scaffolds; (B) selection of published HDAC inhibitors with diverse zinc binding group (ZBG); and (C) structures of some reported BRD4–HDAC dual inhibitors based on known BRD4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Identification of HDAC2 and BRD4 are significantly upregulated in CRC clinical samples: (A) representative immunoreactivity intensities of HDAC2 and BRD4 in CRC tissues and adjacent normal tissues; (B) HDAC2 expression is upregulated in CRC tissues; (C) BRD4 expression is upregulated in CRC tissues; and (D) the upregulated HDAC2 and BRD4 expression are related to the CRC stages.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Fragment-based drug design protocols for the identification of the novel BRD4–HDAC dual inhibitors (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1">4ZW1</a> for BRD4 BD1, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM">2YEM</a> for BRD4 BD2, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a> for HDAC2, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI">5WGI</a> for HDAC6, and PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0Z">3C0Z</a> for HDAC7).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCH<sub>2</sub>CO<sub>2</sub>Et, S<sub>8</sub>, Et<sub>3</sub>N, EtOH, reflux, 12 h; (b) HCl–1,4-dioxane, 25 °C, overnight.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCH<sub>2</sub>CONH<sub>2</sub>, S<sub>8</sub>, Et3N, EtOH, reflux, 12 h; (b) 3,5-dimethylisoxazole-4-carbonyl chloride, pyridine, 3 h, 25 °C; (c) 4 M HCl/dioxane, 4 h, 25 °C; (d) NaOH, EtOH, 24 h, 100 °C; and (e) K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 25 °C or reflux, 6 h.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HCl–1,4-dioxane, 25 °C, 48 h; (b) 1,4-dioxane, NaOH, 30 min; (c)DCC/HOBt, DCM, 24 h; (d) ethyl chloroformate, <i>N</i>-methylmorpholine, 1,4-dioxane, 15 min and then NH<sub>2</sub>OH, CH<sub>3</sub>OH, 20 min; and (e) compound <b>4l</b>, HCl–1,4-dioxane, 25 °C, overnight.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCH<sub>2</sub>CO<sub>2</sub>Et, S<sub>8</sub>, Et<sub>3</sub>N, EtOH, Reflux, 12 h; (b) methyl 6-bromohexanoate, DBU, THF, reflux, overnight; and (c) 4-hydroxy-3,5-dimethylbenzonitrile, HCl–1,4-dioxane, rt, 48 h and then CH<sub>3</sub>OH, CH<sub>2</sub>Cl<sub>2</sub>, HONH<sub>2</sub>, 2 h.</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOH/CH<sub>3</sub>OH, hydroxylamine aqueous solution (50:50, w/w), 4 M HCl/dioxane; (b) 2 N NaOH, 4 M HCl/dioxane, and thenbenzene-1,2-diamine, EDCI, HOBt, CH<sub>2</sub>Cl<sub>2</sub>, 12 h, 25 °C.</p></p></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docked conformations and binding modes of <b>17c</b> to BRD4 (A, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1">4ZW1</a>), HDAC2 (B, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>), and HDAC6 (C, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI">5WGI</a>), respectively. The proteins were illustrated by a solvent-accessible surface with charges (blue for positive and red for negative); the binding site residues were highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>17c</b> induced apoptotic cell death in HCT-116 cells: (A) cell proliferation IC<sub>50</sub> values were measured for compound <b>17c</b> on a panel of CRC cells by MTT assay; (B) cell cycle analysis of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated; (C,D) apoptosis assay of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated by annexin V/PI dual staining; (E) nuclei morphology of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated; and (F) colony formation assay of HCT-116 cells after 0.2 and 0.5 μM of <b>17c</b> treated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>17c</b> induced mitochondrial apoptosis via BRD4–HDAC dual inhibition. HCT-116 cells were treated with 0.2 or 0.5 μM of compound <b>17c</b>, positive control RVX-208, or SAHA for 24 h, respectively. Then, the expression levels of BRD4, c-Myc, Histone H3, Ac-H3, Bim, Bcl-2, Fas, FasL, PARP, caspase-3, and caspase-9 were detected by Western blot (WB) analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>17c</b> induces autophagic cell death in CRC cells. (A) HCT-116 cells were incubated with 0.5 μM of <b>17c</b> for 24 h; then the cells were fixed and imaged under transmitted electron microscopy; scale bar = 1 μm. (B) HCT-116 cells were transfected with a GFP-LC3 plasmid, followed by treatment with 0.5 μM <b>17c</b>; the formation of autophagosomes were observed under a confocal fluorescence microscope, scale bar = 6 μm. (C) After <b>17c</b> incubation for 24 h, the protein expression of LC3, p62, and beclin-1 were detected by WB analysis. (D) HCT-116 cells were treated with 0.5 μM of <b>17c</b> and/or 500 μM of 3-MA; then the cell viability was determined by MTT assay. *** indicated <i>p</i> < 0.001 and ** indicated <i>p</i> < 0.01. (E) HCT-116 cells were treated with 0.5 μM of <b>17c</b> with or without bafilomycin-A1 for 24 h; the protein expression of LC3 and p62 were detected by WB analysis. (F) HCT-116 cells were transfected with ATG5 siRNA or scramble siRNA, respectively. After RNA silence, with 0.5 μM of <b>17c</b> or normal saline incubation, the protein expression of ATG5, AMPKα, p-AMPKα, p62, and LC3 were determined by WB analysis. (G) HCT-116 cells were treated with 0.5 μM <b>17c</b>, 2.0 μM <b>4l</b>, or 10 μM SAHA for 24 h. Then, the protein expressions of LC3 and p62 were detected by WB analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Change of IL6, OSM, LIF, and CTF1 mRNA levels in HCT-116 cells after incubation with 0.5 μM <b>17c</b>, 2.0 μM <b>4l,</b> or 10 μM SAHA for 24 h, respectively. (B) HCT-116 cells were treated with 0.5 μM <b>17c</b>, 5.0 μM <b>4l,</b> or 10 μM SAHA for 24 h, respectively. Then, the protein expressions of OSMR, LIFR, JAK1, STAT3, and phosphorylated STAT3 were detected by WB analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/medium/jm9b02178_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. In vivo anticancer effect of compound <b>17c</b> on the HCT-116 xenograft models: (A) tumor volume changes with oral administration 15 or 30 mg/kg of compound <b>17c</b>, 30 mg/kg of <b>4l</b> or SAHA, **: <i>p</i> < 0.01 compared to the control group; (B) tumor weights analysis after the end of therapy; (C) body weight changes of mice; (D) representative images of IHC analysis of proliferative marker Ki-67, c-Myc, Ac-H3, BRD4, LC3-II, LIFR, p-STAT3, and immunofluorescent analysis of apoptosis marker TUNEL in different groups; scale bar = 50 μm; *: <i>p</i> < 0.05 compared to control group and **: <i>p</i> < 0.01 compared to control group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-7/acs.jmedchem.9b02178/20200402/images/large/jm9b02178_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02178&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedewa, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahnen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meester, R. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Colorectal Cancer Statistics, 2017</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.3322/caac.21395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.3322%2Fcaac.21395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28248415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC1czhslSisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=177-193&author=R.+L.+Siegelauthor=K.+D.+Millerauthor=S.+A.+Fedewaauthor=D.+J.+Ahnenauthor=R.+G.+S.+Meesterauthor=A.+Barziauthor=A.+Jemal&title=Colorectal+Cancer+Statistics%2C+2017&doi=10.3322%2Fcaac.21395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Colorectal cancer statistics, 2017</span></div><div class="casAuthors">Siegel Rebecca L; Miller Kimberly D; Fedewa Stacey A; Ahnen Dennis J; Meester Reinier G S; Barzi Afsaneh; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">177-193</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Colorectal cancer (CRC) is one of the most common malignancies in the United States.  Every 3 years, the American Cancer Society provides an update of CRC incidence, survival, and mortality rates and trends.  Incidence data through 2013 were provided by the Surveillance, Epidemiology, and End Results program, the National Program of Cancer Registries, and the North American Association of Central Cancer Registries.  Mortality data through 2014 were provided by the National Center for Health Statistics.  CRC incidence rates are highest in Alaska Natives and blacks and lowest in Asian/Pacific Islanders, and they are 30% to 40% higher in men than in women.  Recent temporal patterns are generally similar by race and sex, but differ by age.  Between 2000 and 2013, incidence rates in adults aged ≥50 years declined by 32%, with the drop largest for distal tumors in people aged ≥65 years (incidence rate ratio [IRR], 0.50; 95% confidence interval [95% CI], 0.48-0.52) and smallest for rectal tumors in ages 50 to 64 years (male IRR, 0.91; 95% CI, 0.85-0.96; female IRR, 1.00; 95% CI, 0.93-1.08).  Overall CRC incidence in individuals ages ≥50 years declined from 2009 to 2013 in every state except Arkansas, with the decrease exceeding 5% annually in 7 states; however, rectal tumor incidence in those ages 50 to 64 years was stable in most states.  Among adults aged <50 years, CRC incidence rates increased by 22% from 2000 to 2013, driven solely by tumors in the distal colon (IRR, 1.24; 95% CI, 1.13-1.35) and rectum (IRR, 1.22; 95% CI, 1.13-1.31).  Similar to incidence patterns, CRC death rates decreased by 34% among individuals aged ≥50 years during 2000 through 2014, but increased by 13% in those aged <50 years.  Progress against CRC can be accelerated by increasing initiation of screening at age 50 years (average risk) or earlier (eg, family history of CRC/advanced adenomas) and eliminating disparities in high-quality treatment.  In addition, research is needed to elucidate causes for increasing CRC in young adults.  CA Cancer J Clin 2017. © 2017 American Cancer Society.  CA Cancer J Clin 2017;67:177-193. © 2017 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZtSA02_IoCDAyhZbj3lKdfW6udTcc2eZLJWdbaRAfL7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czhslSisA%253D%253D&md5=928172926da7d9ba249bc33406087076</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21395%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DK.%2BD.%26aulast%3DFedewa%26aufirst%3DS.%2BA.%26aulast%3DAhnen%26aufirst%3DD.%2BJ.%26aulast%3DMeester%26aufirst%3DR.%2BG.%2BS.%26aulast%3DBarzi%26aufirst%3DA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DColorectal%2520Cancer%2520Statistics%252C%25202017%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2017%26volume%3D67%26spage%3D177%26epage%3D193%26doi%3D10.3322%2Fcaac.21395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Cancer Statistics, 2017</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.3322/caac.21387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.3322%2Fcaac.21387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28055103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC1c7isFWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=7-30&author=R.+L.+Siegelauthor=K.+D.+Millerauthor=A.+Jemal&title=Cancer+Statistics%2C+2017&doi=10.3322%2Fcaac.21387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Statistics, 2017</span></div><div class="casAuthors">Siegel Rebecca L; Miller Kimberly D; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-30</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Each year, the American Cancer Society estimates the numbers of new cancer cases and deaths that will occur in the United States in the current year and compiles the most recent data on cancer incidence, mortality, and survival.  Incidence data were collected by the Surveillance, Epidemiology, and End Results Program; the National Program of Cancer Registries; and the North American Association of Central Cancer Registries.  Mortality data were collected by the National Center for Health Statistics.  In 2017, 1,688,780 new cancer cases and 600,920 cancer deaths are projected to occur in the United States.  For all sites combined, the cancer incidence rate is 20% higher in men than in women, while the cancer death rate is 40% higher.  However, sex disparities vary by cancer type.  For example, thyroid cancer incidence rates are 3-fold higher in women than in men (21 vs 7 per 100,000 population), despite equivalent death rates (0.5 per 100,000 population), largely reflecting sex differences in the "epidemic of diagnosis." Over the past decade of available data, the overall cancer incidence rate (2004-2013) was stable in women and declined by approximately 2% annually in men, while the cancer death rate (2005-2014) declined by about 1.5% annually in both men and women.  From 1991 to 2014, the overall cancer death rate dropped 25%, translating to approximately 2,143,200 fewer cancer deaths than would have been expected if death rates had remained at their peak.  Although the cancer death rate was 15% higher in blacks than in whites in 2014, increasing access to care as a result of the Patient Protection and Affordable Care Act may expedite the narrowing racial gap; from 2010 to 2015, the proportion of blacks who were uninsured halved, from 21% to 11%, as it did for Hispanics (31% to 16%).  Gains in coverage for traditionally underserved Americans will facilitate the broader application of existing cancer control knowledge across every segment of the population.  CA Cancer J Clin 2017;67:7-30. © 2017 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSB4oj7vNRTIb5oR8RchjrrfW6udTcc2ebwNgzzCMiicLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7isFWgtA%253D%253D&md5=e439f3424d5a773e4ae289c392236ad6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21387%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DK.%2BD.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DCancer%2520Statistics%252C%25202017%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2017%26volume%3D67%26spage%3D7%26epage%3D30%26doi%3D10.3322%2Fcaac.21387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span> <span> </span><span class="NLM_article-title">Enhanced Antitumor Activity and Mechanism of Biodegradable Polymeric Micelles-Encapsulated Chetomin in Both Transgenic Zebrafish and Mouse Models</span>. <i>Nanoscale</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">11940</span>– <span class="NLM_lpage">11952</span>, <span class="refDoi"> DOI: 10.1039/c4nr02978j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1039%2Fc4nr02978j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25175172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlGmur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=11940-11952&author=Q.+Wuauthor=G.+Liauthor=S.+Dengauthor=L.+Ouyangauthor=L.+Liauthor=L.+Liuauthor=N.+Luoauthor=X.+Songauthor=G.+Heauthor=C.+Gongauthor=Y.+Wei&title=Enhanced+Antitumor+Activity+and+Mechanism+of+Biodegradable+Polymeric+Micelles-Encapsulated+Chetomin+in+Both+Transgenic+Zebrafish+and+Mouse+Models&doi=10.1039%2Fc4nr02978j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced antitumor activity and mechanism of biodegradable polymeric micelles-encapsulated chetomin in both transgenic zebrafish and mouse models</span></div><div class="casAuthors">Wu, Qinjie; Li, Guoyou; Deng, Senyi; Ouyang, Liang; Li, Ling; Liu, Lei; Luo, Na; Song, Xiangrong; He, Gu; Gong, Changyang; Wei, Yuquan</div><div class="citationInfo"><span class="NLM_cas:title">Nanoscale</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11940-11952</span>CODEN:
                <span class="NLM_cas:coden">NANOHL</span>;
        ISSN:<span class="NLM_cas:issn">2040-3372</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Chetomin is a promising mol. with anti-tumor activities in the epipolythiodioxopiperazine family of fungal secondary metabolites; however, strong hydrophobicity has limited its further applications.  In this work, chetomin was encapsulated into polymeric micelles to obtain an aq. formulation, and the chetomin loaded micelles (Che-M) exhibited small particle size and high encapsulation efficiency.  When the concn. of copolymer was higher than the crit. gelation concn., the Che-M could form a thermosensitive hydrogel (Che-H), which was free-flowing sol at ambient temp. and converted into a non-flowing gel at body temp.  The mol. modeling study has indicated that chetomin interacted with PCL as a core, which was embraced by PEG as a shell.  Che-M showed equal cytotoxicity with free chetomin, but the apoptosis inducing effects of Che-M were more significant.  Besides, Che-M could increase the GSSG level, decrease the GSH level, and increase the ROS in CT26 cells.  Furthermore, stronger inhibitory effects of Che-M were obsd. on embryonic angiogenesis, tumor-induced angiogenesis and tumor growth in transgenic zebrafish models.  In addn., Che-M was effective in inhibiting tumor growth and prolonging survival in a s.c. CT26 tumor model.  In a colorectal peritoneal carcinomatosis model, both Che-M and Che-H showed excellent therapeutic effects, but Che-H was more effective.  In conclusion, Che-M and Che-H may serve as candidates for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9tkCAHB9X9rVg90H21EOLACvtfcHk0li17gpG7_SEpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGmur%252FI&md5=01842011099d90eebc7eddb89c1ea6bd</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1039%2Fc4nr02978j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4nr02978j%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DN.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DGong%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DY.%26atitle%3DEnhanced%2520Antitumor%2520Activity%2520and%2520Mechanism%2520of%2520Biodegradable%2520Polymeric%2520Micelles-Encapsulated%2520Chetomin%2520in%2520Both%2520Transgenic%2520Zebrafish%2520and%2520Mouse%2520Models%26jtitle%3DNanoscale%26date%3D2014%26volume%3D6%26spage%3D11940%26epage%3D11952%26doi%3D10.1039%2Fc4nr02978j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span> <span> </span><span class="NLM_article-title">Targeting Programmed Cell Death Using Small-Molecule Compounds to Improve Potential Cancer Therapy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">983</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1002/med.21398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1002%2Fmed.21398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27357603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2s7ltVamsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=983-1035&author=B.+Keauthor=M.+Tianauthor=J.+Liauthor=B.+Liuauthor=G.+He&title=Targeting+Programmed+Cell+Death+Using+Small-Molecule+Compounds+to+Improve+Potential+Cancer+Therapy&doi=10.1002%2Fmed.21398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Programmed Cell Death Using Small-Molecule Compounds to Improve Potential Cancer Therapy</span></div><div class="casAuthors">Ke Bowen; Tian Mao; Li Jingjing; Liu Bo; He Gu</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal research reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">983-1035</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Evasion of cell death is one of the hallmarks of cancer cells, beginning with long-established apoptosis and extending to other new forms of cell death.  An elaboration of cell death pathways thus will contribute to a better understanding of cancer pathogenesis and therapeutics.  With the recent substantial biochemical and genetic explorations of cell death subroutines, their classification has switched from primarily morphological to more molecular definitions.  According to their measurable biochemical features and intricate mechanisms, cell death subroutines can be divided into apoptosis, autophagic cell death, mitotic catastrophe, necroptosis, parthanatos, ferroptosis, pyroptosis, pyronecrosis, anoikis, cornification, entosis, and NETosis.  Supportive evidence has gradually revealed the prime molecular mechanisms of each subroutine and thus providing series of possible targets in cancer therapy, while the intricate relationships between different cell death subroutines still remain to be clarified.  Over the past decades, cancer drug discovery has significantly benefited from the use of small-molecule compounds to target classical modalities of cell death such as apoptosis, while newly identified cell death subroutines has also emerging their potential for cancer drug discovery in recent years.  In this review, we comprehensively focus on summarizing 12 cell death subroutines and discussing their corresponding small-molecule compounds in potential cancer therapy.  Together, these inspiring findings may provide more evidence to fill in the gaps between cell death subroutines and small-molecule compounds to better develop novel cancer therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhPsvIEAjUsGBAiYWy6n3SfW6udTcc2eaSlyG9yay6Ybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7ltVamsA%253D%253D&md5=3fc5de00cb2c9a65b378d0358dfd3f5e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fmed.21398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21398%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DB.%26aulast%3DTian%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DG.%26atitle%3DTargeting%2520Programmed%2520Cell%2520Death%2520Using%2520Small-Molecule%2520Compounds%2520to%2520Improve%2520Potential%2520Cancer%2520Therapy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D983%26epage%3D1035%26doi%3D10.1002%2Fmed.21398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span> <span> </span><span class="NLM_article-title">Approved and Experimental Small-Molecule Oncology Kinase Inhibitor Drugs: A Mid-2016 Overview</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1002/med.21409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1002%2Fmed.21409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27775829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2srls1aqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=314-367&author=P.+M.+Fischer&title=Approved+and+Experimental+Small-Molecule+Oncology+Kinase+Inhibitor+Drugs%3A+A+Mid-2016+Overview&doi=10.1002%2Fmed.21409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Approved and Experimental Small-Molecule Oncology Kinase Inhibitor Drugs: A Mid-2016 Overview</span></div><div class="casAuthors">Fischer Peter M</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal research reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">314-367</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kinase inhibitor research is a comparatively recent branch of medicinal chemistry and pharmacology and the first small-molecule kinase inhibitor, imatinib, was approved for clinical use only 15 years ago.  Since then, 33 more kinase inhibitor drugs have received regulatory approval for the treatment of a variety of cancers and the volume of reports on the discovery and development of kinase inhibitors has increased to an extent where it is now difficult-even for those working in the field-easily to keep an overview of the compounds that are being developed, as currently there are 231 such compounds, targeting 38 different protein and lipid kinases (not counting isoforms), in clinical use or under clinical investigation.  The purpose of this review is thus to provide an overview of the biomedical rationales for the kinases being targeted on the one hand, and the design principles, as well as chemical, pharmacological, pharmaceutical, and toxicological kinase inhibitor properties, on the other hand.  Two issues that are especially important in kinase inhibitor research, target selectivity and drug resistance, as well as the underlying structural concepts, are discussed in general terms and in the context of relevant kinases and their inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQqoId3h33O7qe0F5ixvZgXfW6udTcc2eaSlyG9yay6Ybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srls1aqsw%253D%253D&md5=28842df0fe00c1a57c096d70ad7009ec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fmed.21409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21409%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DApproved%2520and%2520Experimental%2520Small-Molecule%2520Oncology%2520Kinase%2520Inhibitor%2520Drugs%253A%2520A%2520Mid-2016%2520Overview%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2017%26volume%3D37%26spage%3D314%26epage%3D367%26doi%3D10.1002%2Fmed.21409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span> <span> </span><span class="NLM_article-title">Hsa-Mir-875-5p Exerts Tumor Suppressor Function through Down-Regulation of Egfr in Colorectal Carcinoma (Crc)</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42225</span>– <span class="NLM_lpage">42240</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.9944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27302926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2s7gtFeqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=42225-42240&author=T.+Zhangauthor=X.+Caiauthor=Q.+Liauthor=P.+Xueauthor=Z.+X.+Chenauthor=X.+Dongauthor=Y.+Xue&title=Hsa-Mir-875-5p+Exerts+Tumor+Suppressor+Function+through+Down-Regulation+of+Egfr+in+Colorectal+Carcinoma+%28Crc%29&doi=10.18632%2Foncotarget.9944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Hsa-miR-875-5p exerts tumor suppressor function through down-regulation of EGFR in colorectal carcinoma (CRC)</span></div><div class="casAuthors">Zhang Tiening; Cai Xun; Li Qi; Xue Peng; Chen Zhixiao; Dong Xiao; Xue Ying</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">42225-42240</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hsa-miRNA-875-5p (miR-875-5p) has recently been discovered to have anticancer efficacy in different organs.  However, the role of miR-875-5p on colorectal carcinoma (CRC) is still ambiguous.  In this study, we investigated the role of miR-875-5p on the development of CRC.  The results indicated that miR-875-5p was significantly down-regulated in primary tumor tissues and very low levels were found in CRC cell lines.  Ectopic expression of miR-875-5p in CRC cell lines significantly suppressed cell growth as evidenced by cell viability assay, colony formation assay and BrdU staining, through inhibition of cyclin D1, cyclin D2, CDK4 and up-regulation of p57(Kip2) and p21(Waf1/Cip1).  In addition, miR-875-5p induced apoptosis, as indicated by concomitantly with up-regulation of key apoptosis protein cleaved caspase-3, and down-regulation of anti-apoptosis protein Bcl2.  Moreover, miR-875-5p inhibited cellular migration and invasiveness through inhibition of matrix metalloproteinases (MMP)-7 and MMP-9.  Further, oncogene EGFR was revealed to be a putative target of miR-875-5p, which was inversely correlated with miR-875-5p expression in CRC.  Taken together, our results demonstrated that miR-875-5p played a pivotal role on CRC through inhibiting cell proliferation, migration, invasion, and promoting apoptosis by targeting oncogenic EGFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRC799yCbWd75oR8RchjrrfW6udTcc2ea-rlfeapWUyrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7gtFeqsA%253D%253D&md5=d42ab34f353e73257e0f0edeeca36b79</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9944%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DXue%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DZ.%2BX.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DXue%26aufirst%3DY.%26atitle%3DHsa-Mir-875-5p%2520Exerts%2520Tumor%2520Suppressor%2520Function%2520through%2520Down-Regulation%2520of%2520Egfr%2520in%2520Colorectal%2520Carcinoma%2520%2528Crc%2529%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D42225%26epage%3D42240%26doi%3D10.18632%2Foncotarget.9944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubiolo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra-Varela, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, S. B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabezas-Sainz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botana, L. M.</span></span> <span> </span><span class="NLM_article-title">Marine Guanidine Alkaloids Crambescidins Inhibit Tumor Growth and Activate Intrinsic Apoptotic Signaling Inducing Tumor Regression in a Colorectal Carcinoma Zebrafish Xenograft Model</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">83071</span>– <span class="NLM_lpage">83087</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.13068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27825113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2snktF2mug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=83071-83087&author=M.+Roelauthor=J.+A.+Rubioloauthor=J.+Guerra-Varelaauthor=S.+B.+L.+Silvaauthor=O.+P.+Thomasauthor=P.+Cabezas-Sainzauthor=L.+Sanchezauthor=R.+Lopezauthor=L.+M.+Botana&title=Marine+Guanidine+Alkaloids+Crambescidins+Inhibit+Tumor+Growth+and+Activate+Intrinsic+Apoptotic+Signaling+Inducing+Tumor+Regression+in+a+Colorectal+Carcinoma+Zebrafish+Xenograft+Model&doi=10.18632%2Foncotarget.13068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Marine guanidine alkaloids crambescidins inhibit tumor growth and activate intrinsic apoptotic signaling inducing tumor regression in a colorectal carcinoma zebrafish xenograft model</span></div><div class="casAuthors">Roel Maria; Rubiolo Juan A; Botana Luis M; Guerra-Varela Jorge; Cabezas-Sainz Pablo; Sanchez Laura; Silva Siguara B L; Thomas Olivier P; Silva Siguara B L; Thomas Olivier P; Lopez Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">83071-83087</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The marine environment constitutes an extraordinary resource for the discovery of new therapeutic agents.  In the present manuscript we studied the effect of 3 different sponge derived guanidine alkaloids, crambescidine-816, -830, and -800.  We show that these compounds strongly inhibit tumor cell proliferation by down-regulating cyclin-dependent kinases 2/6 and cyclins D/A expression while up-regulating the cell cyclin-dependent kinase inhibitors -2A, -2D and -1A.  We also show that these guanidine compounds disrupt tumor cell adhesion and cytoskeletal integrity promoting the activation of the intrinsic apoptotic signaling, resulting in loss of mitochondrial membrane potential and concomitant caspase-3 cleavage and activation.  The crambescidin 816 anti-tumor effect was fnally assayed in a zebrafish xenotransplantation model confirming its potent antitumor activity against colorectal carcinoma in vivo.Considering these results crambescidins could represent promising natural anticancer agents and therapeutic tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEd_cNSVKnvVMGsG_HTXYFfW6udTcc2ea-rlfeapWUyrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snktF2mug%253D%253D&md5=3d650df90587b9d0976f86b32dbdb3fd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13068%26sid%3Dliteratum%253Aachs%26aulast%3DRoel%26aufirst%3DM.%26aulast%3DRubiolo%26aufirst%3DJ.%2BA.%26aulast%3DGuerra-Varela%26aufirst%3DJ.%26aulast%3DSilva%26aufirst%3DS.%2BB.%2BL.%26aulast%3DThomas%26aufirst%3DO.%2BP.%26aulast%3DCabezas-Sainz%26aufirst%3DP.%26aulast%3DSanchez%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DBotana%26aufirst%3DL.%2BM.%26atitle%3DMarine%2520Guanidine%2520Alkaloids%2520Crambescidins%2520Inhibit%2520Tumor%2520Growth%2520and%2520Activate%2520Intrinsic%2520Apoptotic%2520Signaling%2520Inducing%2520Tumor%2520Regression%2520in%2520a%2520Colorectal%2520Carcinoma%2520Zebrafish%2520Xenograft%2520Model%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D83071%26epage%3D83087%26doi%3D10.18632%2Foncotarget.13068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. P.</span></span> <span> </span><span class="NLM_article-title">Synchronous Occurrence of Gastrointestinal Stromal Tumors and Other Digestive Tract Malignancies in the Elderly</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8397</span>– <span class="NLM_lpage">8406</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.3108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25826075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC2MnpsVejug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8397-8406&author=C.+Shenauthor=H.+N.+Chenauthor=Y.+Yinauthor=J.+J.+Chenauthor=L.+Y.+Hanauthor=B.+Zhangauthor=Z.+X.+Chenauthor=J.+P.+Chen&title=Synchronous+Occurrence+of+Gastrointestinal+Stromal+Tumors+and+Other+Digestive+Tract+Malignancies+in+the+Elderly&doi=10.18632%2Foncotarget.3108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Synchronous occurrence of gastrointestinal stromal tumors and other digestive tract malignancies in the elderly</span></div><div class="casAuthors">Shen Chaoyong; Chen Haining; Yin Yuan; Chen Jiaju; Zhang Bo; Chen Zhixin; Chen Jiaping; Han Luyin</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">8397-406</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND/AIMS:  Elderly patients with gastrointestinal stromal tumors (GISTs) synchronous with other digestive tract malignancies have been rarely reported.  In this study, clinicopathological characteristics were evaluated among elderly patients with GISTs with or without coexisting digestive tract malignancies.  METHODS:  A total of 161 patients (≥65 years) were retrospectively reviewed at the West China Hospital, Sichuan University from January 2009 to June 2014.  RESULTS:  Sixty-one patients were diagnosed with synchronous digestive tract malignancies (synchronous group), whereas 100 patients were diagnosed with no synchronous condition (no-synchronous group).  The synchronous group exhibited a higher percentage of males (70.49% vs. 53.00%, P = 0.028) and poorer Eastern Cooperative Oncology Group performance status than the no-synchronous group (P = 0.029).  The three-year overall survival (OS) rate was significantly lower among patients with synchronous digestive tract malignancies than that among patients without synchronous condition (64.5% vs. 84.0%, P = 0.003).  Multivariate analysis showed that the presence of synchronous digestive tract malignancies (P = 0.002), co-morbidity (P = 0.004), and mitotic count ≥10 mitoses/50 high power fields (P = 0.012) were associated with poor OS.  CONCLUSIONS:  A synchronous condition with other digestive tract malignancies is common in elderly patients with GISTs.  OS primarily depends on synchronous digestive tract malignancies, mitotic count, and co-morbidity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQQhPf3aT5hM7GbPKh4T9eAfW6udTcc2ebWDv6s2b_2jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnpsVejug%253D%253D&md5=299550a9d6f7af90287eb1fe57cfb2ed</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3108%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DH.%2BN.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DHan%26aufirst%3DL.%2BY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DZ.%2BX.%26aulast%3DChen%26aufirst%3DJ.%2BP.%26atitle%3DSynchronous%2520Occurrence%2520of%2520Gastrointestinal%2520Stromal%2520Tumors%2520and%2520Other%2520Digestive%2520Tract%2520Malignancies%2520in%2520the%2520Elderly%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D8397%26epage%3D8406%26doi%3D10.18632%2Foncotarget.3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.</span></span> <span> </span><span class="NLM_article-title">Brain-Derived Neurotrophic Factor Involved Epigenetic Repression of Ugt2b7 in Colorectal Carcinoma: A Mechanism to Alter Morphine Glucuronidation in Tumor</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">29138</span>– <span class="NLM_lpage">29150</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.16251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.18632%2Foncotarget.16251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28418861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A280%3ADC%252BC1cvoslGrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=29138-29150&author=Z.+Z.+Yangauthor=L.+Liauthor=M.+C.+Xuauthor=H.+X.+Juauthor=M.+Haoauthor=J.+K.+Guauthor=Z.+J.+J.+Wangauthor=H.+D.+Jiangauthor=L.+S.+Yuauthor=S.+Zeng&title=Brain-Derived+Neurotrophic+Factor+Involved+Epigenetic+Repression+of+Ugt2b7+in+Colorectal+Carcinoma%3A+A+Mechanism+to+Alter+Morphine+Glucuronidation+in+Tumor&doi=10.18632%2Foncotarget.16251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Brain-derived neurotrophic factor involved epigenetic repression of UGT2B7 in colorectal carcinoma: A mechanism to alter morphine glucuronidation in tumor</span></div><div class="casAuthors">Yang Zi-Zhao; Xu Ming-Cheng; Jiang Hui-Di; Yu Lu-Shan; Zeng Su; Li Li; Ju Hai-Xing; Hao Miao; Gu Jing-Kai; Jim Wang Zai-Jie</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">29138-29150</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Uridine diphosphate-glucuronosyltransferase (UGT) 2B7, as one of significant drug enzymes, is responsible on the glucuronidation of abundant endobiotics or xenobiotics.  We here report that it is markedly repressed in the tumor tissues of colorectal carcinoma (CRC) patients.  Accordingly, morphine in CRC cells will stimulate the expression of its main metabolic enzyme, UGT2B7 during tolerance generation by activating the positive signals in histone 3, especially for trimethylated lysine 27 (H3K4Me3) and acetylated lysine 4 (H3K27Ac).  Further study reveals that brain-derived neutrophilic factor (BDNF), a secretory neurotrophin, enriched in CRC can interact and inhibit UGT2B7 by primarily blocking the positive signals of H3K4Me3 as well as activating H3K27Ac on the promoter region of UGT2B7.  Meanwhile, BDNF repression attributes to the sensitizations of main core factors in poly-comb repressive complex (PRC) 1 rather than PRC2 as the reason of the depression of SUZ12 in the later complex.  Besides that, the productions of two main morphine glucuronides are both increased in the BDNF deficient or TSA and BIX-01294 treated morphine tolerance-like HCT-116 cells.  On the same condition, active metabolite, morphine-6-glucuronide (M6G) was accumulated more than inactive M3G.  Our findings imply that enzymatic activity enhancement and substrate regioselective catalysis alteration of UGT2B7 may release morphine tolerance under the cure of tumor-induced pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjN6Q_QjxwZsULxIEhxUjPfW6udTcc2ebWDv6s2b_2jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvoslGrsg%253D%253D&md5=b76a88b93ba7d9940ec1b38bcdb5c61a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.16251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.16251%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%2BZ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DM.%2BC.%26aulast%3DJu%26aufirst%3DH.%2BX.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DJ.%2BK.%26aulast%3DWang%26aufirst%3DZ.%2BJ.%2BJ.%26aulast%3DJiang%26aufirst%3DH.%2BD.%26aulast%3DYu%26aufirst%3DL.%2BS.%26aulast%3DZeng%26aufirst%3DS.%26atitle%3DBrain-Derived%2520Neurotrophic%2520Factor%2520Involved%2520Epigenetic%2520Repression%2520of%2520Ugt2b7%2520in%2520Colorectal%2520Carcinoma%253A%2520A%2520Mechanism%2520to%2520Alter%2520Morphine%2520Glucuronidation%2520in%2520Tumor%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D29138%26epage%3D29150%26doi%3D10.18632%2Foncotarget.16251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y. S.</span></span> <span> </span><span class="NLM_article-title">Mecp2-Mediated Epigenetic Silencing of Mir-137 Contributes to Colorectal Adenoma-Carcinoma Sequence and Tumor Progression Via Relieving the Suppression of C-Met</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">44543</span>, <span class="refDoi"> DOI: 10.1038/srep44543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1038%2Fsrep44543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28291253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=44543&author=T.+Chenauthor=S.+L.+Caiauthor=J.+Liauthor=Z.+P.+Qiauthor=X.+Q.+Liauthor=L.+C.+Yeauthor=X.+F.+Xieauthor=Y.+Y.+Houauthor=L.+Q.+Yaoauthor=M.+D.+Xuauthor=P.+H.+Zhouauthor=J.+M.+Xuauthor=Y.+S.+Zhong&title=Mecp2-Mediated+Epigenetic+Silencing+of+Mir-137+Contributes+to+Colorectal+Adenoma-Carcinoma+Sequence+and+Tumor+Progression+Via+Relieving+the+Suppression+of+C-Met&doi=10.1038%2Fsrep44543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mecp2-mediated Epigenetic Silencing of miR-137 Contributes to Colorectal Adenoma-Carcinoma Sequence and Tumor Progression via Relieving the Suppression of c-Met</span></div><div class="casAuthors">Chen, Tao; Cai, Shi-Lun; Li, Jian; Qi, Zhi-Peng; Li, Xu-Quan; Ye, Le-Chi; Xie, Xiao-Feng; Hou, Ying-Yong; Yao, Li-Qing; Xu, Mei-Dong; Zhou, Ping-Hong; Xu, Jian-Min; Zhong, Yun-Shi</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">44543</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mol. mechanisms underlying colorectal cancer (CRC) development remain elusive.  In this study, we examd. the miRNA and mRNA expressions in the adenoma-carcinoma sequence (ACS), a crit. neoplastic progression in CRC development.  We found that miR-137 was down-regulated in all adenoma and carcinoma tissues.  Low miR-137 levels were correlated neg. with tumor progression and metastasis.  Then we identified the inhibition effect of the miR-137 in CRC development, both in CRC cell lines and mouse models.  MiR-137 was shown to control CRC cell proliferation, colony formation, migration and invasion and to control tumor growth and metastasis.  We further confirmed the neg. assocn. between miR-137 and c-Met expression and thus validated this important oncogene as the target of miR-137 in CRC.  In addn., we found a DNA methyl-CpG-binding protein, Mecp2, was up-regulated in ACS tissues via mRNA sequencing.  Further expt. showed that miR-137 expression in CRC was subjected to epigenetic regulation mediated by Mecp2.  We also confirmed c-Met expression can be up-regulated by silencing of miR-137 and suppressed by coexpression of Mecp2 and miR-137.  These findings highlight the crit. role of miR-137-c-Met nexus in CRC development and reveal Mecp2-regulated epigenetic silence causes the downregulation of miR-137 in colorectal adenoma and carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0nCo6ULP0ebVg90H21EOLACvtfcHk0li4_OrzQKESPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKrtLY%253D&md5=ecc8e70fbfdba3d6a0f9ec91b43bd4db</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsrep44543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep44543%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DCai%26aufirst%3DS.%2BL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DZ.%2BP.%26aulast%3DLi%26aufirst%3DX.%2BQ.%26aulast%3DYe%26aufirst%3DL.%2BC.%26aulast%3DXie%26aufirst%3DX.%2BF.%26aulast%3DHou%26aufirst%3DY.%2BY.%26aulast%3DYao%26aufirst%3DL.%2BQ.%26aulast%3DXu%26aufirst%3DM.%2BD.%26aulast%3DZhou%26aufirst%3DP.%2BH.%26aulast%3DXu%26aufirst%3DJ.%2BM.%26aulast%3DZhong%26aufirst%3DY.%2BS.%26atitle%3DMecp2-Mediated%2520Epigenetic%2520Silencing%2520of%2520Mir-137%2520Contributes%2520to%2520Colorectal%2520Adenoma-Carcinoma%2520Sequence%2520and%2520Tumor%2520Progression%2520Via%2520Relieving%2520the%2520Suppression%2520of%2520C-Met%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D44543%26doi%3D10.1038%2Fsrep44543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mrazek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmical, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajjar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellmich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, C.</span></span> <span> </span><span class="NLM_article-title">Genetic and Epigenetic Changes in Carcinoma-Associated Fibroblasts Derived from Human Colorectal Cancers</span>. <i>J. Am. Coll. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>217</i></span>,  <span class="NLM_fpage">S133</span>, <span class="refDoi"> DOI: 10.1016/j.jamcollsurg.2013.07.310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.jamcollsurg.2013.07.310" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2013&pages=S133&author=A.+Mrazekauthor=L.+Porroauthor=J.+R.+Carmicalauthor=G.+Gomezauthor=A.+Gajjarauthor=M.+R.+Hellmichauthor=C.+Chao&title=Genetic+and+Epigenetic+Changes+in+Carcinoma-Associated+Fibroblasts+Derived+from+Human+Colorectal+Cancers&doi=10.1016%2Fj.jamcollsurg.2013.07.310"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jamcollsurg.2013.07.310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jamcollsurg.2013.07.310%26sid%3Dliteratum%253Aachs%26aulast%3DMrazek%26aufirst%3DA.%26aulast%3DPorro%26aufirst%3DL.%26aulast%3DCarmical%26aufirst%3DJ.%2BR.%26aulast%3DGomez%26aufirst%3DG.%26aulast%3DGajjar%26aufirst%3DA.%26aulast%3DHellmich%26aufirst%3DM.%2BR.%26aulast%3DChao%26aufirst%3DC.%26atitle%3DGenetic%2520and%2520Epigenetic%2520Changes%2520in%2520Carcinoma-Associated%2520Fibroblasts%2520Derived%2520from%2520Human%2520Colorectal%2520Cancers%26jtitle%3DJ.%2520Am.%2520Coll.%2520Surg.%26date%3D2013%26volume%3D217%26spage%3DS133%26doi%3D10.1016%2Fj.jamcollsurg.2013.07.310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slyskova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenkova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochazka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vodickova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneiderova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liska, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holubec, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naccarati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vodicka, P.</span></span> <span> </span><span class="NLM_article-title">Functional, Genetic, and Epigenetic Aspects of Base and Nucleotide Excision Repair in Colorectal Carcinomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5878</span>– <span class="NLM_lpage">5887</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-12-1380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1078-0432.ccr-12-1380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=22966016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghtr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5878-5887&author=J.+Slyskovaauthor=V.+Korenkovaauthor=A.+R.+Collinsauthor=P.+Prochazkaauthor=L.+Vodickovaauthor=J.+Svecauthor=L.+Lipskaauthor=M.+Levyauthor=M.+Schneiderovaauthor=V.+Liskaauthor=L.+Holubecauthor=R.+Kumarauthor=P.+Soucekauthor=A.+Naccaratiauthor=P.+Vodicka&title=Functional%2C+Genetic%2C+and+Epigenetic+Aspects+of+Base+and+Nucleotide+Excision+Repair+in+Colorectal+Carcinomas&doi=10.1158%2F1078-0432.ccr-12-1380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Functional, Genetic, and Epigenetic Aspects of Base and Nucleotide Excision Repair in Colorectal Carcinomas</span></div><div class="casAuthors">Slyskova, Jana; Korenkova, Vlasta; Collins, Andrew R.; Prochazka, Pavel; Vodickova, Ludmila; Svec, Jiri; Lipska, Ludmila; Levy, Miroslav; Schneiderova, Michaela; Liska, Vaclav; Holubec, Lubos; Kumar, Rajiv; Soucek, Pavel; Naccarati, Alessio; Vodicka, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5878-5887</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: DNA repair capacity (DRC) is a determinant not only of cancer development but also of individual response to therapy.  Previously, altered base and nucleotide excision repair (BER and NER) have been described in lymphocytes of patients with sporadic colorectal cancer.  We, for the first time, evaluate both excision repair capacities in human colon biopsies to study their participation in colorectal tumorigenesis.  Exptl. design: Seventy pairs of tumor and adjacent healthy tissues were analyzed for BER- and NER-specific DRC by a comet repair assay.  Tissue pairs were further compared for expression levels of a panel of 25 BER and NER genes complemented by their promoter methylation status.  Results: We obsd. a moderate increase of NER-DRC (P = 0.019), but not of BER-DRC in tumors.  There was a strong correlation between both tissues for all investigated parameters (P < 0.001).  However, 4 NER (CSB, CCNH, XPA, XPD) and 4 BER (NEIL1, APEX1, OGG1, PARP1) genes showed a 1.08- to 1.28-fold change difference in expression in tumors (P < 0.05).  Individual gene expression levels did not correlate with overall DRC, and we did not detect any aberrant methylation of the investigated genes.  Conclusions: Our complex anal. showed that tumor cells are not deficient in BER and NER, but rather follow patterns characteristic for each individual and are comparable with adjacent tissue.  Alteration of excision repair pathways is not a pronounced event in colorectal carcinogenesis.  This study shows the feasibility of DRC evaluation in human solid tissues, representing a complex marker of multigene DNA repair processes.  Clin Cancer Res; 18(21); 5878-87. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeM1XBtSpXO7Vg90H21EOLACvtfcHk0li4_OrzQKESPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ghtr3O&md5=e4d1210d02aa548cc61938c82d771f53</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-12-1380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-12-1380%26sid%3Dliteratum%253Aachs%26aulast%3DSlyskova%26aufirst%3DJ.%26aulast%3DKorenkova%26aufirst%3DV.%26aulast%3DCollins%26aufirst%3DA.%2BR.%26aulast%3DProchazka%26aufirst%3DP.%26aulast%3DVodickova%26aufirst%3DL.%26aulast%3DSvec%26aufirst%3DJ.%26aulast%3DLipska%26aufirst%3DL.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DSchneiderova%26aufirst%3DM.%26aulast%3DLiska%26aufirst%3DV.%26aulast%3DHolubec%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DSoucek%26aufirst%3DP.%26aulast%3DNaccarati%26aufirst%3DA.%26aulast%3DVodicka%26aufirst%3DP.%26atitle%3DFunctional%252C%2520Genetic%252C%2520and%2520Epigenetic%2520Aspects%2520of%2520Base%2520and%2520Nucleotide%2520Excision%2520Repair%2520in%2520Colorectal%2520Carcinomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5878%26epage%3D5887%26doi%3D10.1158%2F1078-0432.ccr-12-1380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lutz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzner, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahrens, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makowiec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopt, U. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogatyreva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauschke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassmann, S.</span></span> <span> </span><span class="NLM_article-title">Histone Modifiers and Marks Define Heterogeneous Groups of Colorectal Carcinomas and Affect Responses to Hdac Inhibitors in Vitro</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">664</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27152243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvFOhurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=664&author=L.+Lutzauthor=I.+C.+Fitznerauthor=T.+Ahrensauthor=A.+L.+Geisslerauthor=F.+Makowiecauthor=U.+T.+Hoptauthor=L.+Bogatyrevaauthor=D.+Hauschkeauthor=M.+Wernerauthor=S.+Lassmann&title=Histone+Modifiers+and+Marks+Define+Heterogeneous+Groups+of+Colorectal+Carcinomas+and+Affect+Responses+to+Hdac+Inhibitors+in+Vitro"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Histone modifiers and marks define heterogeneous groups of colorectal carcinomas and affect responses to HDAC inhibitors in vitro</span></div><div class="casAuthors">Lutz, Lisa; Fitzner, Ingrid Coutino; Ahrens, Theresa; Geissler, Anna-Lena; Makowiec, Frank; Hopt, Ulrich T.; Bogatyreva, Lioudmila; Hauschke, Dieter; Werner, Martin; Lassmann, Silke</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">664-676</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Little is known about histone modifiers and histone marks in colorectal cancers (CRC).  The present study therefore addressed the role of histone acetylation and histone deacetylases (HDAC) in CRCs in situ and in vitro.  Immunohistochem. of primary CRCs (n=47) revealed that selected histone marks were frequently present (H3K4me3: 100%; H3K9me3: 77%; H3K9ac: 75%), partially displayed intratumoral heterogeneity (H3K9me3; H3K9ac) and were significantly linked to higher pT category (H3K9me3: p=0.023; H3K9ac: p=0.028).  Furthermore, also HDAC1 (62%), HDAC2 (100%) and HDAC3 (72%) expression was frequent, revealing four CRC types: cases expressing 1) HDAC1, HDAC2 and HDAC3 (49%), 2) HDAC2 and HDAC3 (30%), 3) HDAC1 and HDAC2 (10.5%) and 4) exclusively HDAC2 (10.5%).  Correlation to clinico-pathol. parameters (pT, pN, G, MSI status) revealed that heterogeneous HDAC1 expression correlated with lymph node status (p=0.012).  HDAC expression in situ was partially reflected by six CRC cell lines, with similar expression of all three HDACs (DLD1, LS174T), preferential HDAC2 and HDAC3 expression (SW480, Caco2) or lower HDAC2 and HDAC3 expression (HCT116, HT29).  HDAC activity was variably higher in HCT116, HT29, DLD1 and SW480 compared to LS174T and Caco2 cells.  Treatment with broad (SAHA) and specific (MS-275; FK228) HDAC inhibitors (HDACi) caused loss of cell viability in predominantly MSIpos. CRC cells (HCT116, LS174T, DLD1; SAHA, MS-275 and in part FK228).  In contrast, MSI-neg. CRC cells (Caco2, HT29, SW480) were resistant, except for high doses of FK228 (Caco2, HT29).  Cell viability patterns were not linked to different efficacies of HDACi on redn. of HDAC activity or histone acetylation, p21 expression and/ or induction of DNA damage (γH2A-X levels).  In summary, this study reveals inter- and intra-tumoral heterogeneity of histone marks and HDAC expression in CRCs.  This is reflected by diverse HDACi responses in vitro, which do not follow known modes of action.  Together, this implies further exploitation of histone alterations in CRC for mol. classification and/or novel treatment options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEDxqeHVagGrVg90H21EOLACvtfcHk0liOoeBxylcJ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvFOhurc%253D&md5=c5ca6bcefff7c3f6c74a178072ab7366</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLutz%26aufirst%3DL.%26aulast%3DFitzner%26aufirst%3DI.%2BC.%26aulast%3DAhrens%26aufirst%3DT.%26aulast%3DGeissler%26aufirst%3DA.%2BL.%26aulast%3DMakowiec%26aufirst%3DF.%26aulast%3DHopt%26aufirst%3DU.%2BT.%26aulast%3DBogatyreva%26aufirst%3DL.%26aulast%3DHauschke%26aufirst%3DD.%26aulast%3DWerner%26aufirst%3DM.%26aulast%3DLassmann%26aufirst%3DS.%26atitle%3DHistone%2520Modifiers%2520and%2520Marks%2520Define%2520Heterogeneous%2520Groups%2520of%2520Colorectal%2520Carcinomas%2520and%2520Affect%2520Responses%2520to%2520Hdac%2520Inhibitors%2520in%2520Vitro%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2016%26volume%3D6%26spage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakakibara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecke, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, A. A.</span></span> <span> </span><span class="NLM_article-title">Brd4 Is Displaced from Hpv Replication Factories as They Expand and Amplify Viral DNA</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003777</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1003777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1371%2Fjournal.ppat.1003777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24278023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtV2nsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=N.+Sakakibaraauthor=D.+Chenauthor=M.+K.+Jangauthor=D.+W.+Kangauthor=H.+F.+Lueckeauthor=S.+Y.+Wuauthor=C.+M.+Chiangauthor=A.+A.+McBride&title=Brd4+Is+Displaced+from+Hpv+Replication+Factories+as+They+Expand+and+Amplify+Viral+DNA&doi=10.1371%2Fjournal.ppat.1003777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 is displaced from HPV replication factories as they expand and amplify viral DNA</span></div><div class="casAuthors">Sakakibara, Nozomi; Chen, Dan; Jang, Moon Kyoo; Kang, Dong Wook; Luecke, Hans F.; Wu, Shwu-Yuan; Chiang, Cheng-Ming; McBride, Alison A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e1003777/1-e1003777/23, 23 pp.</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Replication foci are generated by many viruses to conc. and localize viral DNA synthesis to specific regions of the cell.  Expression of the HPV16 E1 and E2 replication proteins in keratinocytes results in nuclear foci that recruit proteins assocd. with the host DNA damage response.  We show that the Brd4 protein localizes to these foci and is essential for their formation.  However, when E1 and E2 begin amplifying viral DNA, Brd4 is displaced from the foci and cellular factors assocd. with DNA synthesis and homologous recombination are recruited.  Differentiated HPV-infected keratinocytes form similar nuclear foci that contain amplifying viral DNA.  We compare the different foci and show that, while they have many characteristics in common, there is a switch between early Brd4-dependent foci and mature Brd4-independent replication foci.  However, HPV genomes encoding mutated E2 proteins that are unable to bind Brd4 can replicate and amplify the viral genome.  We propose that, while E1, E2 and Brd4 might bind host chromatin at early stages of infection, there is a temporal and functional switch at later stages and increased E1 and E2 levels promote viral DNA amplification, displacement of Brd4 and growth of a replication factory.  The concomitant DNA damage response recruits proteins required for DNA synthesis and repair, which could then be utilized for viral DNA replication.  Hence, while Brd4 can enhance replication by concg. viral processes in specific regions of the host nucleus, this interaction is not absolutely essential for HPV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFxSI-Fqa2XLVg90H21EOLACvtfcHk0liOoeBxylcJ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtV2nsr0%253D&md5=3538d6798e52d681e962ac4f080a6241</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003777%26sid%3Dliteratum%253Aachs%26aulast%3DSakakibara%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DKang%26aufirst%3DD.%2BW.%26aulast%3DLuecke%26aufirst%3DH.%2BF.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DMcBride%26aufirst%3DA.%2BA.%26atitle%3DBrd4%2520Is%2520Displaced%2520from%2520Hpv%2520Replication%2520Factories%2520as%2520They%2520Expand%2520and%2520Amplify%2520Viral%2520DNA%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.ppat.1003777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hossan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indenbirken, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipaleenmaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Batalla, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loges, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesse, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundhoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Protein Brd4 Is Required for Estrogen Receptor-Dependent Enhancer Activation and Gene Transcription</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.celrep.2014.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25017071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=460-469&author=S.+Nagarajanauthor=T.+Hossanauthor=M.+Alawiauthor=Z.+Najafovaauthor=D.+Indenbirkenauthor=U.+Bediauthor=H.+Taipaleenmakiauthor=I.+Ben-Batallaauthor=M.+Schellerauthor=S.+Logesauthor=S.+Knappauthor=E.+Hesseauthor=C.+M.+Chiangauthor=A.+Grundhoffauthor=S.+A.+Johnsen&title=Bromodomain+Protein+Brd4+Is+Required+for+Estrogen+Receptor-Dependent+Enhancer+Activation+and+Gene+Transcription&doi=10.1016%2Fj.celrep.2014.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain Protein BRD4 Is Required for Estrogen Receptor-Dependent Enhancer Activation and Gene Transcription</span></div><div class="casAuthors">Nagarajan, Sankari; Hossan, Tareq; Alawi, Malik; Najafova, Zeynab; Indenbirken, Daniela; Bedi, Upasana; Taipaleenmaeki, Hanna; Ben-Batalla, Isabel; Scheller, Marina; Loges, Sonja; Knapp, Stefan; Hesse, Eric; Chiang, Cheng-Ming; Grundhoff, Adam; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">460-469</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The estrogen receptor α (ERα) controls cell proliferation and tumorigenesis by recruiting various cofactors to estrogen response elements (EREs) to control gene transcription.  A deeper understanding of these transcriptional mechanisms may uncover therapeutic targets for ERα-dependent cancers.  We show that BRD4 regulates ERα-induced gene expression by affecting elongation-assocd. phosphorylation of RNA polymerase II (RNAPII) and histone H2B monoubiquitination.  Consistently, BRD4 activity is required for proliferation of ER+ breast and endometrial cancer cells and uterine growth in mice.  Genome-wide studies revealed an enrichment of BRD4 on transcriptional start sites of active genes and a requirement of BRD4 for H2B monoubiquitination in the transcribed region of estrogen-responsive genes.  Importantly, we demonstrate that BRD4 occupancy on distal EREs enriched for H3K27ac is required for recruitment and elongation of RNAPII on EREs and the prodn. of ERα-dependent enhancer RNAs.  These results uncover BRD4 as a central regulator of ERα function and potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0fNk5Ueff1bVg90H21EOLACvtfcHk0li8qRHZ5r-BvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM&md5=301a96567713aa777911a158736adb6d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DHossan%26aufirst%3DT.%26aulast%3DAlawi%26aufirst%3DM.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DIndenbirken%26aufirst%3DD.%26aulast%3DBedi%26aufirst%3DU.%26aulast%3DTaipaleenmaki%26aufirst%3DH.%26aulast%3DBen-Batalla%26aufirst%3DI.%26aulast%3DScheller%26aufirst%3DM.%26aulast%3DLoges%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHesse%26aufirst%3DE.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGrundhoff%26aufirst%3DA.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DBromodomain%2520Protein%2520Brd4%2520Is%2520Required%2520for%2520Estrogen%2520Receptor-Dependent%2520Enhancer%2520Activation%2520and%2520Gene%2520Transcription%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D460%26epage%3D469%26doi%3D10.1016%2Fj.celrep.2014.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simanski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span> <span> </span><span class="NLM_article-title">Brd4 Phosphorylation Regulates Hpv E2-Mediated Viral Transcription, Origin Replication, and Cellular Mmp-9 Expression</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1733</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.celrep.2016.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27477287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1733-1748&author=S.-Y.+Wuauthor=D.+S.+Ninauthor=A.-Y.+Leeauthor=S.+Simanskiauthor=T.+Kodadekauthor=C.-M.+Chiang&title=Brd4+Phosphorylation+Regulates+Hpv+E2-Mediated+Viral+Transcription%2C+Origin+Replication%2C+and+Cellular+Mmp-9+Expression&doi=10.1016%2Fj.celrep.2016.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Phosphorylation Regulates HPV E2-Mediated Viral Transcription, Origin Replication, and Cellular MMP-9 Expression</span></div><div class="casAuthors">Wu, Shwu-Yuan; Nin, Dawn Sijin; Lee, A-Young; Simanski, Scott; Kodadek, Thomas; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1733-1748</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Post-translational modification can modulate protein conformation and alter binding partner recruitment within gene regulatory regions.  Here, we report that bromodomain-contg. protein 4 (BRD4), a transcription co-factor and chromatin regulator, uses a phosphorylation-induced switch mechanism to recruit E2 protein encoded by cancer-assocd. human papillomavirus (HPV) to viral early gene and cellular matrix metalloproteinase-9 (MMP-9) promoters.  Enhanced MMP-9 expression, induced upon keratinocyte differentiation, occurs via BRD4-dependent recruitment of active AP-1 and NF-κB to their target sequences.  This is triggered by replacement of AP-1 family members JunB and JunD by c-Jun and by re-localization of NF-κB from the cytoplasm to the nucleus.  In addn., BRD4 phosphorylation is crit. for E2- and origin-dependent HPV DNA replication.  A class of phospho-BRD4-targeting compds., distinct from the BET bromodomain inhibitors, effectively blocks BRD4 phosphorylation-specific functions in transcription and factor recruitment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomvZx5RmFIRLVg90H21EOLACvtfcHk0li8qRHZ5r-BvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO&md5=10bde9118396d8a5a1f4614fb64a566c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DNin%26aufirst%3DD.%2BS.%26aulast%3DLee%26aufirst%3DA.-Y.%26aulast%3DSimanski%26aufirst%3DS.%26aulast%3DKodadek%26aufirst%3DT.%26aulast%3DChiang%26aufirst%3DC.-M.%26atitle%3DBrd4%2520Phosphorylation%2520Regulates%2520Hpv%2520E2-Mediated%2520Viral%2520Transcription%252C%2520Origin%2520Replication%252C%2520and%2520Cellular%2520Mmp-9%2520Expression%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D16%26spage%3D1733%26epage%3D1748%26doi%3D10.1016%2Fj.celrep.2016.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamdan, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indenbirken, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundhoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheel, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegwitz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Brd4 Promotes P63 and Grhl3 Expression Downstream of Foxo in Mammary Epithelial Cells</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">3130</span>– <span class="NLM_lpage">3145</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1093%2Fnar%2Fgkw1276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27980063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1WmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=3130-3145&author=S.+Nagarajanauthor=U.+Bediauthor=A.+Budidaauthor=F.+H.+Hamdanauthor=V.+K.+Mishraauthor=Z.+Najafovaauthor=W.+H.+Xieauthor=M.+Alawiauthor=D.+Indenbirkenauthor=S.+Knappauthor=C.+M.+Chiangauthor=A.+Grundhoffauthor=V.+Kariauthor=C.+H.+Scheelauthor=F.+Wegwitzauthor=S.+A.+Johnsen&title=Brd4+Promotes+P63+and+Grhl3+Expression+Downstream+of+Foxo+in+Mammary+Epithelial+Cells&doi=10.1093%2Fnar%2Fgkw1276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 promotes p63 and GRHL3 expression downstream of FOXO in mammary epithelial cells</span></div><div class="casAuthors">Nagarajan, Sankari; Bedi, Upasana; Budida, Anusha; Hamdan, Feda H.; Mishra, Vivek Kumar; Najafova, Zeynab; Xie, Wanhua; Alawi, Malik; Indenbirken, Daniela; Knapp, Stefan; Chiang, Cheng-Ming; Grundhoff, Adam; Kari, Vijayalakshmi; Scheel, Christina H.; Wegwitz, Florian; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3130-3145</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) is a member of the bromo- and extraterminal (BET) domain-contg. family of epigenetic readers which is under intensive investigation as a target for anti-tumor therapy.  BRD4 plays a central role in promoting the expression of select subsets of genes including many driven by oncogenic transcription factors and signaling pathways.  However, the role of BRD4 and the effects of BET inhibitors in non-transformed cells remain mostly unclear.  We demonstrate that BRD4 is required for the maintenance of a basal epithelial phenotype by regulating the expression of epithelial-specific genes including TP63 and Grainy Head-like transcription factor-3 (GRHL3) in non-transformed basal-like mammary epithelial cells.  Moreover, BRD4 occupancy correlates with enhancer activity and enhancer RNA (eRNA) transcription.  Motif analyses of cell context-specific BRD4-enriched regions predicted the involvement of FOXO transcription factors.  Consistently, activation of FOXO1 function via inhibition of EGFR-AKT signaling promoted the expression of TP63 and GRHL3.  Moreover, activation of Src kinase signaling and FOXO1 inhibition decreased the expression of FOXO/BRD4 target genes.  Together, our findings support a function for BRD4 in promoting basal mammary cell epithelial differentiation, at least in part, by regulating FOXO factor function on enhancers to activate TP63 and GRHL3 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8CeR-xfSherVg90H21EOLACvtfcHk0lhz7-fmOGKEew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1WmtLs%253D&md5=051ed118e27e9ae51f2cfb7058c31d21</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1276%26sid%3Dliteratum%253Aachs%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DBedi%26aufirst%3DU.%26aulast%3DBudida%26aufirst%3DA.%26aulast%3DHamdan%26aufirst%3DF.%2BH.%26aulast%3DMishra%26aufirst%3DV.%2BK.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DW.%2BH.%26aulast%3DAlawi%26aufirst%3DM.%26aulast%3DIndenbirken%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGrundhoff%26aufirst%3DA.%26aulast%3DKari%26aufirst%3DV.%26aulast%3DScheel%26aufirst%3DC.%2BH.%26aulast%3DWegwitz%26aufirst%3DF.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DBrd4%2520Promotes%2520P63%2520and%2520Grhl3%2520Expression%2520Downstream%2520of%2520Foxo%2520in%2520Mammary%2520Epithelial%2520Cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D3130%26epage%3D3145%26doi%3D10.1093%2Fnar%2Fgkw1276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delcuratolo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fertey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuetz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiprecht, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudjetz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubenrauch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iftner, T.</span></span> <span> </span><span class="NLM_article-title">Papillomavirus-Associated Tumor Formation Critically Depends on C-Fos Expression Induced by Viral Protein E2 and Bromodomain Protein Brd4</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005366</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1371%2Fjournal.ppat.1005366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26727473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSrur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=M.+Delcuratoloauthor=J.+Ferteyauthor=M.+Schneiderauthor=J.+Schuetzauthor=N.+Leiprechtauthor=B.+Hudjetzauthor=S.+Brodbeckauthor=S.+Corallauthor=M.+Dreerauthor=R.+M.+Schwabauthor=M.+Grimmauthor=S.-Y.+Wuauthor=F.+Stubenrauchauthor=C.-M.+Chiangauthor=T.+Iftner&title=Papillomavirus-Associated+Tumor+Formation+Critically+Depends+on+C-Fos+Expression+Induced+by+Viral+Protein+E2+and+Bromodomain+Protein+Brd4&doi=10.1371%2Fjournal.ppat.1005366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Papillomavirus-associated tumor formation critically depends on c-Fos expression induced by viral protein E2 and bromodomain protein Brd4</span></div><div class="casAuthors">Delcuratolo, Maria; Fertey, Jasmin; Schneider, Markus; Schuetz, Johanna; Leiprecht, Natalie; Hudjetz, Benjamin; Brodbeck, Stephan; Corall, Silke; Dreer, Marcel; Schwab, Roxana Michaela; Grimm, Martin; Wu, Shwu-Yuan; Stubenrauch, Frank; Chiang, Cheng-Ming; Iftner, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e1005366/1-e1005366/22</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">We investigated the mechanism of how the papillomavirus E2 transcription factor can activate promoters through activator protein (AP)1 binding sites.  Using an unbiased approach with an inducible cell line expressing the viral transcription factor E2 and transcriptome anal., we found that E2 induces the expression of the two AP1 components c-Fos and FosB in a Brd4-dependent manner.  In vitro RNA interference confirmed that c-Fos is one of the AP1 members driving the expression of viral oncogenes E6/E7.  Mutation anal. and in vivo RNA interference identified an essential role for c-Fos/AP1 and also for the bromodomain protein Brd4 for papillomavirus-induced tumorigenesis.  Lastly, chromatin immunopptn. anal. demonstrated that E2 binds together with Brd4 to a canonical E2 binding site (E2BS) in the promoter of c-Fos, thus activating c-Fos expression.  Thus, we identified a novel way how E2 activates the viral oncogene promoter and show that E2 may act as a viral oncogene by direct activation of c-Fos involved in skin tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6HcuABW6FLVg90H21EOLACvtfcHk0lhz7-fmOGKEew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSrur4%253D&md5=3a32f66d3560dabdd61bb8f39ec4c978</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005366%26sid%3Dliteratum%253Aachs%26aulast%3DDelcuratolo%26aufirst%3DM.%26aulast%3DFertey%26aufirst%3DJ.%26aulast%3DSchneider%26aufirst%3DM.%26aulast%3DSchuetz%26aufirst%3DJ.%26aulast%3DLeiprecht%26aufirst%3DN.%26aulast%3DHudjetz%26aufirst%3DB.%26aulast%3DBrodbeck%26aufirst%3DS.%26aulast%3DCorall%26aufirst%3DS.%26aulast%3DDreer%26aufirst%3DM.%26aulast%3DSchwab%26aufirst%3DR.%2BM.%26aulast%3DGrimm%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DStubenrauch%26aufirst%3DF.%26aulast%3DChiang%26aufirst%3DC.-M.%26aulast%3DIftner%26aufirst%3DT.%26atitle%3DPapillomavirus-Associated%2520Tumor%2520Formation%2520Critically%2520Depends%2520on%2520C-Fos%2520Expression%2520Induced%2520by%2520Viral%2520Protein%2520E2%2520and%2520Bromodomain%2520Protein%2520Brd4%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iftner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haedicke-Jarboui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span> <span> </span><span class="NLM_article-title">Involvement of Brd4 in Different Steps of the Papillomavirus Life Cycle</span>. <i>Virus Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>231</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.virusres.2016.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.virusres.2016.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27965149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWnt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=231&publication_year=2017&pages=76-82&author=T.+Iftnerauthor=J.+Haedicke-Jarbouiauthor=S.-Y.+Wuauthor=C.-M.+Chiang&title=Involvement+of+Brd4+in+Different+Steps+of+the+Papillomavirus+Life+Cycle&doi=10.1016%2Fj.virusres.2016.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Involvement of Brd4 in different steps of the papillomavirus life cycle</span></div><div class="casAuthors">Iftner, Thomas; Haedicke-Jarboui, Juliane; Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">231</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-82</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Bromodomain-contg. protein 4 (Brd4) is a cellular chromatin-binding factor and transcriptional regulator that recruits sequence-specific transcription factors and chromatin modulators to control target gene transcription.  Papillomaviruses (PVs) have evolved to hijack Brd4's activity in order to create a facilitating environment for the viral life cycle.  Brd4, in assocn. with the major viral regulatory protein E2, is involved in multiple steps of the PV life cycle including replication initiation, viral gene transcription, and viral genome segregation and maintenance.  Phosphorylation of Brd4, regulated by casein kinase II (CK2) and protein phosphatase 2A (PP2A), is crit. for viral gene transcription as well as E1- and E2-dependent origin replication.  Thus, pharmacol. agents regulating Brd4 phosphorylation and inhibitors blocking phospho-Brd4 functions are promising candidates for therapeutic intervention in treating human papillomavirus (HPV) infections as well as assocd. disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp07jGfzmJg9rVg90H21EOLACvtfcHk0lhcEvZJWbOsCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWnt7rO&md5=2f9b41f45d836350dd1ce07981f0d0e9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.virusres.2016.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virusres.2016.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DIftner%26aufirst%3DT.%26aulast%3DHaedicke-Jarboui%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26atitle%3DInvolvement%2520of%2520Brd4%2520in%2520Different%2520Steps%2520of%2520the%2520Papillomavirus%2520Life%2520Cycle%26jtitle%3DVirus%2520Res.%26date%3D2017%26volume%3D231%26spage%3D76%26epage%3D82%26doi%3D10.1016%2Fj.virusres.2016.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Development of N-Hydroxycinnamamide-Based Hdac Inhibitors with Improved Hdac Inhibitory Activity and in Vitro Antitumor Activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2666</span>– <span class="NLM_lpage">2675</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmc.2016.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28336407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVSlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2666-2675&author=J.+Zangauthor=B.+Shiauthor=X.+Liangauthor=Q.+Gaoauthor=W.+Xuauthor=Y.+Zhang&title=Development+of+N-Hydroxycinnamamide-Based+Hdac+Inhibitors+with+Improved+Hdac+Inhibitory+Activity+and+in+Vitro+Antitumor+Activity&doi=10.1016%2Fj.bmc.2016.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Development of N-hydroxycinnamamide-based HDAC inhibitors with improved HDAC inhibitory activity and in vitro antitumor activity</span></div><div class="casAuthors">Zang, Jie; Shi, Baowen; Liang, Xuewu; Gao, Qianwen; Xu, Wenfang; Zhang, Yingjie</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2666-2675</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylase inhibitors (HDACIs) are promising in the treatment of various diseases, among which cancer treatment has achieved the most success.  We have previously developed series of HDACIs combining N-hydroxycinnamamide bioactive fragment and indole bioactive fragment, which showed moderate to potent antitumor activities.  Herein, further structural derivatization based on our previous structure-activity relationship (SAR) got 25 novel compds.  Most compds. showed much more potent histone deacetylases (HDACs) inhibitory activity than their parent compd. 1 and even the pos. control SAHA.  What's more, compared with the approved HDACs inhibitor SAHA, compds. 6i, 6k, 6q and 6t displayed better in vitro antiproliferation against multiple tumor cell lines.  It is worth noting that though the 4-hydroxycinnamic acid-based compd. 2 showed HDAC1/3 dual selectivity, its 4-hydroxy-3-methoxycinnamic acid-based analog 6t turned out to be a pan-HDACs inhibitor as SAHA, indicating that the 3-methoxy group on the N-hydroxycinnamamide fragment could dramatically influence the HDACs isoform selectivity of this series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolb8cpswLOMrVg90H21EOLACvtfcHk0lhcEvZJWbOsCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVSlur8%253D&md5=8ce20c262f96079fa20d9e48ddc02a3b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520N-Hydroxycinnamamide-Based%2520Hdac%2520Inhibitors%2520with%2520Improved%2520Hdac%2520Inhibitory%2520Activity%2520and%2520in%2520Vitro%2520Antitumor%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2666%26epage%3D2675%26doi%3D10.1016%2Fj.bmc.2016.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inks, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Selective Hdac Inhibitors with Potent Oral Activity against Leukemia and Colorectal Cancer: Design, Structure-Activity Relationship and Anti-Tumor Activity Study</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejmech.2017.03.069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28415009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFWnt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2017&pages=185-206&author=X.+Liauthor=Y.+Zhangauthor=Y.+Jiangauthor=J.+Wuauthor=E.+S.+Inksauthor=C.+J.+Chouauthor=S.+Gaoauthor=J.+Houauthor=Q.+Dingauthor=J.+Liauthor=X.+Wangauthor=Y.+Huangauthor=W.+Xu&title=Selective+Hdac+Inhibitors+with+Potent+Oral+Activity+against+Leukemia+and+Colorectal+Cancer%3A+Design%2C+Structure-Activity+Relationship+and+Anti-Tumor+Activity+Study&doi=10.1016%2Fj.ejmech.2017.03.069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selective HDAC inhibitors with potent oral activity against leukemia and colorectal cancer: Design, structure-activity relationship and anti-tumor activity study</span></div><div class="casAuthors">Li, Xiaoyang; Zhang, Yingjie; Jiang, Yuqi; Wu, Jingde; Inks, Elizabeth S.; Chou, C. James; Gao, Shuai; Hou, Jinning; Ding, Qinge; Li, Jingyao; Wang, Xue; Huang, Yongxue; Xu, Wenfang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-206</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Previously, we reported the discovery of a series of N-hydroxycinnamamide-based HDAC inhibitors, among which compd. 11y exhibited high HDAC1/3 selectivity.  In this current study, structural derivatization of 11y led to a new series of benzamide based HDAC inhibitors.  Most of the compds. exhibited high HDACs inhibitory potency.  Compd. 11a (with 4-methoxybenzoyl as N-substituent in the cap and 4-(aminomethyl) benzoyl as the linker group) exhibited selectivity against HDAC1 to some extent, and showed potent antiproliferative activity against several tumor cell lines.  In vivo studies revealed that compd. 11a displayed potent oral antitumor activity in both hematol. tumor cell U937 xenograft model and solid tumor cell HCT116 xenograft model with no obvious toxicity.  Further modification of benzamide 3, 11a and 19 afforded new thienyl and Ph compds. (50a, 50b, 63a, 63b and 63c) with dramatic HDAC1 and HDAC2 dual selectivity, and the fluorine contg. compd. 56, with moderate HDAC3 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvK_Gdgx31-LVg90H21EOLACvtfcHk0liX7IqAv4YxgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFWnt70%253D&md5=66ad0ddda5b3e46ef07dfb40951b63db</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.069%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DInks%26aufirst%3DE.%2BS.%26aulast%3DChou%26aufirst%3DC.%2BJ.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DSelective%2520Hdac%2520Inhibitors%2520with%2520Potent%2520Oral%2520Activity%2520against%2520Leukemia%2520and%2520Colorectal%2520Cancer%253A%2520Design%252C%2520Structure-Activity%2520Relationship%2520and%2520Anti-Tumor%2520Activity%2520Study%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D134%26spage%3D185%26epage%3D206%26doi%3D10.1016%2Fj.ejmech.2017.03.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Biological Evaluation of Quinoline Derivatives as Hdac Class I Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejmech.2017.03.064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28371677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVCit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=11-23&author=C.+Chenauthor=X.+Houauthor=G.+Wangauthor=W.+Panauthor=X.+Yangauthor=Y.+Zhangauthor=H.+Fang&title=Design%2C+Synthesis+and+Biological+Evaluation+of+Quinoline+Derivatives+as+Hdac+Class+I+Inhibitors&doi=10.1016%2Fj.ejmech.2017.03.064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of quinoline derivatives as HDAC class I inhibitors</span></div><div class="casAuthors">Chen, Chen; Hou, Xuben; Wang, Guohua; Pan, Wenyan; Yang, Xinying; Zhang, Yingkai; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-23</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Based on pharmacophore model of HDAC inhibitors, a series of quinoline-based N-hydroxyacrylamides I [R1 = 4-N-hydroxyacrylamide, 3-N-hydroxyacrylamide; R2 = H, OMe, Cl, Br,; R3 = H, OMe, Cl; R4 = H, hexyl, Bn] and N-hydroxybenzamides II [R1 = H, Bn; R2 = H, OMe, Br; R3 = H, Cl] were designed and synthesized as potent HDAC inhibitors.  All target compds. were evaluated for their in vitro HDAC inhibitory activities and anti-proliferative activities and the best compd. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4= H] surpass Vorinostat in both enzymic inhibitory activity and cellular anti-proliferative activity.  In terms of HDAC isoforms selectivity, compd. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4= H] exhibited preferable inhibition for class I HDACs, esp. for HDAC8, the IC50 value (442 nM) was much lower than that of Vorinostat (7468 nM).  Subsequently, class I & IIa HDACs whole cell enzyme assay to evaluate inhibitory activity in whole cell context were performed.  Compds. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4 = H; R1 = 4-N-hydroxyacrylamide, R2, R4 = H, R3 = Cl] displayed much better cellular activity for class I HDACs than that for class IIa HDACs, which indicated that compds. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4 = H; R1 = 4-N-hydroxyacrylamide, R2, R4 = H, R3 = Cl] might be potent class I HDAC inhibitors.  Meanwhile, flow cytometry anal. showed that compds. I [R1 = 4-N-hydroxyacrylamide, R2, R3, R4 = H; R1 = 4-N-hydroxyacrylamide, R2, R4 = H, R3 = Cl] promoted cell apoptosis in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeuPe9e4YS9rVg90H21EOLACvtfcHk0liX7IqAv4YxgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVCit7s%253D&md5=948aeb076d296895955ba4845240b596</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.064%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Quinoline%2520Derivatives%2520as%2520Hdac%2520Class%2520I%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D133%26spage%3D11%26epage%3D23%26doi%3D10.1016%2Fj.ejmech.2017.03.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Preliminary Bioactivity Evaluations of Substituted Quinoline Hydroxamic Acid Derivatives as Novel Histone Deacetylase (Hdac) Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4364</span>– <span class="NLM_lpage">4374</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmc.2015.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26149591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2itrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=4364-4374&author=L.+Wangauthor=X.+Houauthor=H.+Fuauthor=X.+Panauthor=W.+Xuauthor=W.+Tangauthor=H.+Fang&title=Design%2C+Synthesis+and+Preliminary+Bioactivity+Evaluations+of+Substituted+Quinoline+Hydroxamic+Acid+Derivatives+as+Novel+Histone+Deacetylase+%28Hdac%29+Inhibitors&doi=10.1016%2Fj.bmc.2015.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and preliminary bioactivity evaluations of substituted quinoline hydroxamic acid derivatives as novel histone deacetylase (HDAC) inhibitors</span></div><div class="casAuthors">Wang, Lei; Hou, Xuben; Fu, Huansheng; Pan, Xiaole; Xu, Wenfang; Tang, Weiping; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4364-4374</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Inhibition of HDACs activity has become a promising therapeutic strategy in clin. practice to reverse the abnormal epigenetic states of cancer and other diseases.  Therefore, HDAC inhibitors become a relatively new class of anti-cancer agent.  In the present study, we reported the design and synthesis of a series of novel HDAC inhibitors using various substituted quinoline rings as the cap group.  In vitro studies showed that some compds. have good inhibitory activities against HDACs and potent antiproliferative activities in some tumor cell lines.  Esp., compd. I (IC50 = 85 nM), exhibited better inhibitory effect compared with SAHA (IC50 = 161 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs4Se1Xd6Yp7Vg90H21EOLACvtfcHk0liX7IqAv4YxgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2itrnI&md5=d0c9af14949d22036161a30c27288e24</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Preliminary%2520Bioactivity%2520Evaluations%2520of%2520Substituted%2520Quinoline%2520Hydroxamic%2520Acid%2520Derivatives%2520as%2520Novel%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D4364%26epage%3D4374%26doi%3D10.1016%2Fj.bmc.2015.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Preliminary Bioactivity Studies of Substituted Purine Hydroxamic Acid Derivatives as Novel Histone Deacetylase (Hdac) Inhibitors</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1887</span>– <span class="NLM_lpage">1891</span>, <span class="refDoi"> DOI: 10.1039/c4md00203b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1039%2Fc4md00203b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFeqsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1887-1891&author=J.+Wangauthor=F.+E.+Sunauthor=L.+Hanauthor=X.+Houauthor=X.+Panauthor=R.+Liuauthor=W.+Tangauthor=H.+Fang&title=Design%2C+Synthesis%2C+and+Preliminary+Bioactivity+Studies+of+Substituted+Purine+Hydroxamic+Acid+Derivatives+as+Novel+Histone+Deacetylase+%28Hdac%29+Inhibitors&doi=10.1039%2Fc4md00203b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and preliminary bioactivity studies of substituted purine hydroxamic acid derivatives as novel histone deacetylase (HDAC) inhibitors</span></div><div class="casAuthors">Wang, Junhua; Sun, Feng'e; Han, Leiqiang; Hou, Xuben; Pan, Xiaole; Liu, Renshuai; Tang, Weiping; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1887-1891</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) is a clin. validated target for anti-tumor therapy.  In order to increase HDAC inhibition and efficiency, we developed a series of novel substituted purine hydroxamic acids as potent HDAC inhibitors.  The biol. evaluation suggests that compd. 5r (IC50 = 0.075 μmol L-1) exhibits better HDAC1 and 2 inhibitory activity compared to the approved drug SAHA (IC50 = 0.14 μmol L-1).  Further biol. evaluation indicated that compds. 5r, 5w, and 5x have potent anti-proliferative activities against eight tumor cells, including MDA-MB231, KG1, PC3, U937 and so on.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrskovjc6x8l7Vg90H21EOLACvtfcHk0lh_8jDS2p56bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFeqsrbL&md5=f8d472e0691edd7d0501880068c32720</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2Fc4md00203b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4md00203b%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DF.%2BE.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Preliminary%2520Bioactivity%2520Studies%2520of%2520Substituted%2520Purine%2520Hydroxamic%2520Acid%2520Derivatives%2520as%2520Novel%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Inhibitors%26jtitle%3DMedchemcomm%26date%3D2014%26volume%3D5%26spage%3D1887%26epage%3D1891%26doi%3D10.1039%2Fc4md00203b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Improved Antiproliferative Activity of 1,3,4-Thiadiazole-Containing Histone Deacetylase (Hdac) Inhibitors by Introduction of the Heteroaromatic Surface Recognition Motif</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5766</span>– <span class="NLM_lpage">5775</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2014.09.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmc.2014.09.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25311567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KltLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=5766-5775&author=P.+Guanauthor=L.+Wangauthor=X.+Houauthor=Y.+Wanauthor=W.+Xuauthor=W.+Tangauthor=H.+Fang&title=Improved+Antiproliferative+Activity+of+1%2C3%2C4-Thiadiazole-Containing+Histone+Deacetylase+%28Hdac%29+Inhibitors+by+Introduction+of+the+Heteroaromatic+Surface+Recognition+Motif&doi=10.1016%2Fj.bmc.2014.09.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Improved antiproliferative activity of 1,3,4-thiadiazole-containing histone deacetylase (HDAC) inhibitors by introduction of the heteroaromatic surface recognition motif</span></div><div class="casAuthors">Guan, Peng; Wang, Lei; Hou, Xuben; Wan, Yichao; Xu, Wenfang; Tang, Weiping; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5766-5775</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of 1,3,4-thiadiazole-contg. hydroxamic acids, in accord with the common pharmacophore of histone deacetylase (HDAC) inhibitors (a Zn2+ binding moiety-a linker-a surface recognition motif), was identified as submicromolar HDAC inhibitors by the authors' group.  In this study, the authors continued the efforts to develop 1,3,4-thiadiazole bearing hydroxamate analogs by modifying the surface recognition motif.  The authors found that 1,3,4-thiadiazoles having a heteroarom. substituent showed better HDAC inhibitory activity in enzymic assay and higher antiproliferative potency in cellular assay compared to SAHA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeTGSTrYDTiLVg90H21EOLACvtfcHk0lh_8jDS2p56bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KltLfO&md5=41e9d828f2c5b03462245918bede4b46</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.09.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.09.039%26sid%3Dliteratum%253Aachs%26aulast%3DGuan%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DImproved%2520Antiproliferative%2520Activity%2520of%25201%252C3%252C4-Thiadiazole-Containing%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Inhibitors%2520by%2520Introduction%2520of%2520the%2520Heteroaromatic%2520Surface%2520Recognition%2520Motif%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D5766%26epage%3D5775%26doi%3D10.1016%2Fj.bmc.2014.09.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W. F.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Angiogenesis by Hdac Inhibitor Zyj-34c in Vitro</span>. <i>Acta Pharmacol. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">65</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=65&author=Y.+P.+Jiaauthor=D.+S.+Zhouauthor=Y.+J.+Zhangauthor=W.+F.+Xu&title=Inhibition+of+Angiogenesis+by+Hdac+Inhibitor+Zyj-34c+in+Vitro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%2BP.%26aulast%3DZhou%26aufirst%3DD.%2BS.%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DXu%26aufirst%3DW.%2BF.%26atitle%3DInhibition%2520of%2520Angiogenesis%2520by%2520Hdac%2520Inhibitor%2520Zyj-34c%2520in%2520Vitro%26jtitle%3DActa%2520Pharmacol.%2520Sin.%26date%3D2013%26volume%3D34%26spage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazarika, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiao, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricker, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, S. R.</span></span> <span> </span><span class="NLM_article-title">Phase 2 Trial of Oral Vorinostat (Suberoylanilide Hydroxamic Acid, Saha) for Refractory Cutaneous T-Cell Lymphoma (Ctcl)</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1182/blood-2006-06-025999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1182%2Fblood-2006-06-025999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=16960145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivVyrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2007&pages=31-39&author=M.+Duvicauthor=R.+Talpurauthor=X.+Niauthor=C.+Zhangauthor=P.+Hazarikaauthor=C.+Kellyauthor=J.+H.+Chiaoauthor=J.+F.+Reillyauthor=J.+L.+Rickerauthor=V.+M.+Richonauthor=S.+R.+Frankel&title=Phase+2+Trial+of+Oral+Vorinostat+%28Suberoylanilide+Hydroxamic+Acid%2C+Saha%29+for+Refractory+Cutaneous+T-Cell+Lymphoma+%28Ctcl%29&doi=10.1182%2Fblood-2006-06-025999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Phase 2 trial of oral vorinostat (suberoylanilide hydroxamic acid, SAHA) for refractory cutaneous T-cell lymphoma (CTCL)</span></div><div class="casAuthors">Duvic, Madeleine; Talpur, Rakshandra; Ni, Xiao; Zhang, Chunlei; Hazarika, Parul; Kelly, Cecilia; Chiao, Judy H.; Reilly, John F.; Ricker, Justin L.; Richon, Victoria M.; Frankel, Stanley R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-39</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The activity and safety of the histone deacetylase inhibitor vorinostat (suberoylanilide hydroxamic acid, SAHA) were evaluated in patients with refractory cutaneous T-cell lymphoma (CTCL).  Group 1 received vorinostat 400 mg daily, group 2 received vorinostat 300 mg twice daily for 3 days with 4 days rest, and group 3 received vorinostat 300 mg twice daily for 14 days with 7 days rest followed by 200 mg twice daily.  Treatment continued until disease progression or intolerable toxicity.  The primary objective was to det. the complete and partial response (PR) rate.  Time to response (TTR), time to progressive disease (TTP), response duration (DOR), pruritus relief, and safety were detd.  Thirty-three patients who had received a median of 5 prior therapies were enrolled.  Eight patients achieved a PR, including 7 with advanced disease and 4 with Sezary syndrome.  The median TTR, DOR, and TTP for responders were 11.9, 15.1, and 30.2 wk, resp.  Fourteen of 31 evaluable patients had pruritus relief.  The most common drug-related AEs were fatigue, thrombocytopenia, diarrhea, and nausea.  The most common grade 3 or 4 drug-related AEs were thrombocytopenia and dehydration.  Vorinostat demonstrated activity in heavily pretreated patients with CTCL.  The 400 mg daily regimen had the most favorable safety profile and is being further evaluated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo80_rNvPEEgLVg90H21EOLACvtfcHk0ligeeaCaJq5Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivVyrt7g%253D&md5=b4c8b3c20213eaf692bd9b44266d045f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1182%2Fblood-2006-06-025999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2006-06-025999%26sid%3Dliteratum%253Aachs%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DTalpur%26aufirst%3DR.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHazarika%26aufirst%3DP.%26aulast%3DKelly%26aufirst%3DC.%26aulast%3DChiao%26aufirst%3DJ.%2BH.%26aulast%3DReilly%26aufirst%3DJ.%2BF.%26aulast%3DRicker%26aufirst%3DJ.%2BL.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DFrankel%26aufirst%3DS.%2BR.%26atitle%3DPhase%25202%2520Trial%2520of%2520Oral%2520Vorinostat%2520%2528Suberoylanilide%2520Hydroxamic%2520Acid%252C%2520Saha%2529%2520for%2520Refractory%2520Cutaneous%2520T-Cell%2520Lymphoma%2520%2528Ctcl%2529%26jtitle%3DBlood%26date%3D2007%26volume%3D109%26spage%3D31%26epage%3D39%26doi%3D10.1182%2Fblood-2006-06-025999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span> <span> </span><span class="NLM_article-title">Development of the Pan-Dac Inhibitor Panobinostat (Lbh589): Successes and Challenges</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2009.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.canlet.2009.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=19344997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2009&pages=233-241&author=P.+Atadja&title=Development+of+the+Pan-Dac+Inhibitor+Panobinostat+%28Lbh589%29%3A+Successes+and+Challenges&doi=10.1016%2Fj.canlet.2009.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the pan-DAC inhibitor panobinostat (LBH589): Successes and challenges</span></div><div class="casAuthors">Atadja, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-241</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  The histone deacetylase (HDAC) inhibitors are emerging as a highly useful class of anticancer agents that inhibit the enzyme HDAC involved in the deacetylation of histone and non-histone cellular proteins.  The HDAC inhibitor, panobinostat (LBH589, Novartis Pharmaceuticals), achieves potent inhibition of all HDAC enzymes implicated in cancer and has demonstrated potent anti-tumor activity in preclin. models and promising clin. efficacy in cancer patients.  In this review we discuss the successes and challenges surrounding the development of panobinostat, focusing on its proposed mechanism of action, preclin. anti-tumor activity, and early clin. efficacy in hematol. and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSTYxNKUMt7bVg90H21EOLACvtfcHk0ligeeaCaJq5Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D&md5=4c487a2f8dc13f72cecf105d9fae97d9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2009.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2009.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DAtadja%26aufirst%3DP.%26atitle%3DDevelopment%2520of%2520the%2520Pan-Dac%2520Inhibitor%2520Panobinostat%2520%2528Lbh589%2529%253A%2520Successes%2520and%2520Challenges%26jtitle%3DCancer%2520Lett.%26date%3D2009%26volume%3D280%26spage%3D233%26epage%3D241%26doi%3D10.1016%2Fj.canlet.2009.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bociek, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neelapu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buglio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besterman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drouin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulus, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martell, R. E.</span></span> <span> </span><span class="NLM_article-title">Mocetinostat for Relapsed Classical Hodgkin’s Lymphoma: An Open-Label, Single-Arm, Phase 2 Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1222</span>– <span class="NLM_lpage">1228</span>, <span class="refDoi"> DOI: 10.1016/s1470-2045(11)70265-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fs1470-2045%2811%2970265-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=22033282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFGjt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=1222-1228&author=A.+Younesauthor=Y.+Okiauthor=R.+G.+Bociekauthor=J.+Kuruvillaauthor=M.+Fanaleauthor=S.+Neelapuauthor=A.+Copelandauthor=D.+Buglioauthor=A.+Galalauthor=J.+Bestermanauthor=Z.+Liauthor=M.+Drouinauthor=T.+Pattersonauthor=M.+R.+Wardauthor=J.+K.+Paulusauthor=Y.+Jiauthor=L.+J.+Medeirosauthor=R.+E.+Martell&title=Mocetinostat+for+Relapsed+Classical+Hodgkin%E2%80%99s+Lymphoma%3A+An+Open-Label%2C+Single-Arm%2C+Phase+2+Trial&doi=10.1016%2Fs1470-2045%2811%2970265-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Mocetinostat for relapsed classical Hodgkin's lymphoma: an open-label, single-arm, phase 2 trial</span></div><div class="casAuthors">Younes, Anas; Oki, Yasuhiro; Bociek, R. Gregory; Kuruvilla, John; Fanale, Michelle; Neelapu, Sattva; Copeland, Amanda; Buglio, Daniela; Galal, Ahmed; Besterman, Jeffrey; Li, Zuomei; Drouin, Michel; Patterson, Tracy; Ward, M. Renee; Paulus, Jessica K.; Ji, Yuan; Medeiros, L. Jeffrey; Martell, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1222-1228</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Summary: Background: The prognosis of patients with relapsed Hodgkin's lymphoma, esp. those who relapse after stem-cell transplantation, is poor, and the development of new agents for this patient population is an unmet medical need.  We tested the safety and efficacy of mocetinostat, an oral isotype-selective histone deacetylase inhibitor, in patients with relapsed classical Hodgkin's lymphoma.  Methods: Patients with relapsed or refractory classical Hodgkin's lymphoma aged 18 years or older were treated with mocetinostat administered orally three times per wk, in 28-day cycles.  Two doses were assessed (85 mg and 110 mg).  Patients were treated until disease progression or prohibitive toxicity.  The primary outcome was disease control rate, defined as complete response, partial response, or stable disease (for at least six cycles), analyzed by intention to treat.  This trial has been completed and is registered with ClinicalTrials.gov, no. NCT00358982.  Findings: 51 patients were enrolled.  Initially, 23 patients were enrolled in the 110 mg cohort.  Subsequently, because toxicity-related dose redns. were necessary in the 110 mg cohort, we treated 28 addnl. patients with a dose of 85 mg.  On the basis of intent-to-treat anal., the disease control rate was 35% (eight of 23 patients) in the 110 mg group and 25% (seven of 28) in the 85 mg group. 12 patients (24%) discontinued treatment because of adverse events, nine (32%) in the 85 mg cohort and three (13%) in the 110 mg cohort.  The most frequent treatment-related grade 3 and 4 adverse events were neutropenia (four patients [17%] in the 110 mg group, three [11%] in the 85 mg group); fatigue (five patients [22%] in the 110 mg group, three [11%] in the 85 mg group); and pneumonia (four patients [17%] in the 110 mg group, two [7%] in the 85 mg group).  Four patients, all in the 110 mg cohort, died during the study, of which two might have been related to treatment.  Interpretation: Mocetinostat, 85 mg three times per wk, has promising single-agent clin. activity with manageable toxicity in patients with relapsed classical Hodgkin's lymphoma.  Funding: MethylGene Inc, Montreal, Canada; Celgene Corporation, Summit, NJ, USA; Tufts Medical Center, Boston, MA, USA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3rbuk2PJOR7Vg90H21EOLACvtfcHk0lhtk3F5igk3kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFGjt77O&md5=c6f7748fe47bad8522fb16ef67195231</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fs1470-2045%2811%2970265-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1470-2045%252811%252970265-0%26sid%3Dliteratum%253Aachs%26aulast%3DYounes%26aufirst%3DA.%26aulast%3DOki%26aufirst%3DY.%26aulast%3DBociek%26aufirst%3DR.%2BG.%26aulast%3DKuruvilla%26aufirst%3DJ.%26aulast%3DFanale%26aufirst%3DM.%26aulast%3DNeelapu%26aufirst%3DS.%26aulast%3DCopeland%26aufirst%3DA.%26aulast%3DBuglio%26aufirst%3DD.%26aulast%3DGalal%26aufirst%3DA.%26aulast%3DBesterman%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DDrouin%26aufirst%3DM.%26aulast%3DPatterson%26aufirst%3DT.%26aulast%3DWard%26aufirst%3DM.%2BR.%26aulast%3DPaulus%26aufirst%3DJ.%2BK.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DMedeiros%26aufirst%3DL.%2BJ.%26aulast%3DMartell%26aufirst%3DR.%2BE.%26atitle%3DMocetinostat%2520for%2520Relapsed%2520Classical%2520Hodgkin%25E2%2580%2599s%2520Lymphoma%253A%2520An%2520Open-Label%252C%2520Single-Arm%252C%2520Phase%25202%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2011%26volume%3D12%26spage%3D1222%26epage%3D1228%26doi%3D10.1016%2Fs1470-2045%2811%2970265-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajji, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallenborg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlachos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyman, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermanson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, B.</span></span> <span> </span><span class="NLM_article-title">Combinatorial Action of the Hdac Inhibitor Trichostatin a and Etoposide Induces Caspase-Mediated Aif-Dependent Apoptotic Cell Death in Non-Small Cell Lung Carcinoma Cells</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3134</span>– <span class="NLM_lpage">3144</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1038%2Fsj.onc.1210976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=18071312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFSis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3134-3144&author=N.+Hajjiauthor=K.+Wallenborgauthor=P.+Vlachosauthor=U.+Nymanauthor=O.+Hermansonauthor=B.+Joseph&title=Combinatorial+Action+of+the+Hdac+Inhibitor+Trichostatin+a+and+Etoposide+Induces+Caspase-Mediated+Aif-Dependent+Apoptotic+Cell+Death+in+Non-Small+Cell+Lung+Carcinoma+Cells&doi=10.1038%2Fsj.onc.1210976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial action of the HDAC inhibitor trichostatin A and etoposide induces caspase-mediated AIF-dependent apoptotic cell death in non-small cell lung carcinoma cells</span></div><div class="casAuthors">Hajji, N.; Wallenborg, K.; Vlachos, P.; Nyman, U.; Hermanson, O.; Joseph, B.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3134-3144</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Commonly used regimens in cancer therapy rely on the induction of apoptotic cell death, and drug resistance can be attributed, at least in part, to a disabled apoptotic program.  Non-small cell lung carcinomas (NSCLC), exhibit an intrinsic resistance to chemotherapy.  Here, we show that co-treatment with etoposide (VP16) and the pan-histone deacetylase (HDAC) inhibitor trichostatin A (TSA), but not valproic acid (VPA), induced apoptotic cell death in drug-resistant NSCLC cells.  Co-treatment, but not single treatment, with VP16 and TSA induced apoptosis in a caspase-dependent manner accompanied by a crucial decrease in Bcl-xL expression allowing Bax activation and subsequent initiation of the apoptosis inducing factor (AIF)-dependent death pathway.  Importantly, AIF proved to be required for the effects of TSA/VP16 as RNA knockdown of AIF resulted in a complete abolishment of TSA/VP16-induced apoptotic cell death in drug-resistant NSCLC cells.  Our results thus provide evidence for the requirement of both caspase-dependent and caspase-independent apoptotic pathways in TSA/VP16-mediated death of drug-resistant NSCLC cells, and extend previous suggestions that HDAC inhibitors in combination with conventional chemotherapeutic drugs could be valuable in the treatment of NSCLC cancer and other malignancies in which Bcl-xL is overexpressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxjdh2HAy64LVg90H21EOLACvtfcHk0lhtk3F5igk3kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFSis7Y%253D&md5=1f21019a0c1b0cf7045831947c6e2144</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210976%26sid%3Dliteratum%253Aachs%26aulast%3DHajji%26aufirst%3DN.%26aulast%3DWallenborg%26aufirst%3DK.%26aulast%3DVlachos%26aufirst%3DP.%26aulast%3DNyman%26aufirst%3DU.%26aulast%3DHermanson%26aufirst%3DO.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DCombinatorial%2520Action%2520of%2520the%2520Hdac%2520Inhibitor%2520Trichostatin%2520a%2520and%2520Etoposide%2520Induces%2520Caspase-Mediated%2520Aif-Dependent%2520Apoptotic%2520Cell%2520Death%2520in%2520Non-Small%2520Cell%2520Lung%2520Carcinoma%2520Cells%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3134%26epage%3D3144%26doi%3D10.1038%2Fsj.onc.1210976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saijo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimodaira, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishioka, C.</span></span> <span> </span><span class="NLM_article-title">Biochemical, Biological and Structural Properties of Romidepsin (Fk228) and Its Analogs as Novel Hdac/Pi3k Dual Inhibitors</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1111/cas.12585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1111%2Fcas.12585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=25492515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVyhurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2015&pages=208-215&author=K.+Saijoauthor=J.+Imamuraauthor=K.+Naritaauthor=A.+Odaauthor=H.+Shimodairaauthor=T.+Katohauthor=C.+Ishioka&title=Biochemical%2C+Biological+and+Structural+Properties+of+Romidepsin+%28Fk228%29+and+Its+Analogs+as+Novel+Hdac%2FPi3k+Dual+Inhibitors&doi=10.1111%2Fcas.12585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, biological and structural properties of romidepsin (FK228) and its analogs as novel HDAC/PI3K dual inhibitors</span></div><div class="casAuthors">Saijo, Ken; Imamura, Jin; Narita, Koichi; Oda, Akifumi; Shimodaira, Hideki; Katoh, Tadashi; Ishioka, Chikashi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-215</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Romidepsin (FK228, depsipeptide) is a potent histone deacetylase (HDAC) inhibitor that has FDA approval for the treatment of cutaneous and peripheral T-cell lymphomas.  We have previously reported that FK228 and its analogs have an addnl. activity as phosphatidylinositol 3-kinase (PI3K) inhibitors, and are defined as HDAC/PI3K dual inhibitors.  Because a combination of an HDAC inhibitor and a PI3K inhibitor induces apoptosis in human cancer cells in a synergistic manner, development of an HDAC/PI3K dual inhibitor will provide an attractive novel drug for cancer therapy.  Using structure-based optimization of the analogs, FK-A11 was identified as the most potent analog.  FK-A11 inhibited phosphorylation of AKT and accelerated histone acetylation at lower concns., resulting in stronger cytotoxic effects than FK228 and the other analogs in human cancer cells.  In this study, we have characterized the biochem., biol. and structural properties of FK228 analogs as PI3K inhibitors.  First, FK-A11 is an ATP competitive PI3K inhibitor.  Second, FK-A11 is a pan-p110 isoform inhibitor.  Third, FK-A11 selectively inhibits PI3K among 22 common cellular kinases.  Fourth, conformational changes of FK228 analogs by redn. of an internal disulfide bond have no effect on PI3K inhibitory activity, unlike HDAC inhibitory activity.  Finally, mol. modeling of PI3K-FK228 analogs and analyses of the binding affinities identified the structure that defines potency for PI3K inhibitory activity.  These results prove our concept that a series of FK228 analogs are HDAC/PI3K dual inhibitors.  These findings should help in the development of FK228 analogs as novel HDAC/PI3K dual inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Gsab8AjdsLVg90H21EOLACvtfcHk0liDEBSASz90AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVyhurw%253D&md5=cf8df800ee9fa639998bc13cc4a77c2b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1111%2Fcas.12585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12585%26sid%3Dliteratum%253Aachs%26aulast%3DSaijo%26aufirst%3DK.%26aulast%3DImamura%26aufirst%3DJ.%26aulast%3DNarita%26aufirst%3DK.%26aulast%3DOda%26aufirst%3DA.%26aulast%3DShimodaira%26aufirst%3DH.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DIshioka%26aufirst%3DC.%26atitle%3DBiochemical%252C%2520Biological%2520and%2520Structural%2520Properties%2520of%2520Romidepsin%2520%2528Fk228%2529%2520and%2520Its%2520Analogs%2520as%2520Novel%2520Hdac%252FPi3k%2520Dual%2520Inhibitors%26jtitle%3DCancer%2520Sci.%26date%3D2015%26volume%3D106%26spage%3D208%26epage%3D215%26doi%3D10.1111%2Fcas.12585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stolfa, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancelot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leproult, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melesina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavarelli, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for the Antiparasitic Activity of a Mercaptoacetamide Derivative That Inhibits Histone Deacetylase 8 (Hdac8) from the Human Pathogen Schistosoma Mansoni</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">3442</span>– <span class="NLM_lpage">3453</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2014.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.jmb.2014.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24657767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1ahu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3442-3453&author=D.+A.+Stolfaauthor=M.+Marekauthor=J.+Lancelotauthor=A.-T.+Hauserauthor=A.+Walterauthor=E.+Leproultauthor=J.+Melesinaauthor=T.+Rumpfauthor=J.-M.+Wurtzauthor=J.+Cavarelliauthor=W.+Sipplauthor=R.+J.+Pierceauthor=C.+Romierauthor=M.+Jung&title=Molecular+Basis+for+the+Antiparasitic+Activity+of+a+Mercaptoacetamide+Derivative+That+Inhibits+Histone+Deacetylase+8+%28Hdac8%29+from+the+Human+Pathogen+Schistosoma+Mansoni&doi=10.1016%2Fj.jmb.2014.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the Antiparasitic Activity of a Mercaptoacetamide Derivative That Inhibits Histone Deacetylase 8 (HDAC8) from the Human Pathogen Schistosoma mansoni</span></div><div class="casAuthors">Stolfa, Diana A.; Marek, Martin; Lancelot, Julien; Hauser, Alexander-Thomas; Walter, Alexandra; Leproult, Emeline; Melesina, Jelena; Rumpf, Tobias; Wurtz, Jean-Marie; Cavarelli, Jean; Sippl, Wolfgang; Pierce, Raymond J.; Romier, Christophe; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3442-3453</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Schistosomiasis, caused by the parasitic flatworm Schistosoma mansoni and related species, is a tropical disease that affects over 200 million people worldwide.  A new approach for targeting eukaryotic parasites is to tackle their dynamic epigenetic machinery that is necessary for the extensive phenotypic changes during the life cycle of the parasite.  Recently, the authors identified S. mansoni histone deacetylase 8 (smHDAC8) as a potential target for antiparasitic therapy.  Here, the authors present results on the investigations of a focused set of HDAC (histone deacetylase) inhibitors on smHDAC8.  Besides several active hydroxamates, the authors identified a thiol-based inhibitor that inhibited smHDAC8 activity in the micromolar range with unexpected selectivity over the human isotype, which has not been obsd. so far.  The crystal structure of smHDAC8 complexed with the thiol deriv. revealed that the inhibitor is accommodated in the catalytic pocket, where it interacts with both the catalytic zinc ion and the essential catalytic tyrosine (Y341) residue via its mercaptoacetamide warhead.  To the authors' knowledge, this is the first complex crystal structure of any HDAC inhibited by a mercaptoacetamide inhibitor, and therefore, this finding offers a rationale for further improvement.  Finally, an ester prodrug of the thiol HDAC inhibitor exhibited antiparasitic activity on cultured schistosomes in a dose-dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzjyeGmhcE27Vg90H21EOLACvtfcHk0liDEBSASz90AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1ahu7k%253D&md5=8732c934526a515aecbcacbe8a677066</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DStolfa%26aufirst%3DD.%2BA.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DLancelot%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DA.-T.%26aulast%3DWalter%26aufirst%3DA.%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DMelesina%26aufirst%3DJ.%26aulast%3DRumpf%26aufirst%3DT.%26aulast%3DWurtz%26aufirst%3DJ.-M.%26aulast%3DCavarelli%26aufirst%3DJ.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DMolecular%2520Basis%2520for%2520the%2520Antiparasitic%2520Activity%2520of%2520a%2520Mercaptoacetamide%2520Derivative%2520That%2520Inhibits%2520Histone%2520Deacetylase%25208%2520%2528Hdac8%2529%2520from%2520the%2520Human%2520Pathogen%2520Schistosoma%2520Mansoni%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3442%26epage%3D3453%26doi%3D10.1016%2Fj.jmb.2014.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">The Bet Bromodomain Inhibitor Jq1 Activates Hiv Latency through Antagonizing Brd4 Inhibition of Tat-Transactivation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1093/nar/gks976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1093%2Fnar%2Fgks976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=23087374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=277-287&author=Z.+Liauthor=J.+Guoauthor=Y.+Wuauthor=Q.+Zhou&title=The+Bet+Bromodomain+Inhibitor+Jq1+Activates+Hiv+Latency+through+Antagonizing+Brd4+Inhibition+of+Tat-Transactivation&doi=10.1093%2Fnar%2Fgks976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The BET bromodomain inhibitor JQ1 activates HIV latency through antagonizing Brd4 inhibition of Tat-transactivation</span></div><div class="casAuthors">Li, Zichong; Guo, Jia; Wu, Yuntao; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Latent HIV reservoirs are the primary hurdle to eradication of infection.  Identification of agents, pathways and mol. mechanisms that activate latent provirus may, in the presence of highly active antiretroviral therapy, permit clearance of infected cells by the immune system.  Promoter-proximal pausing of RNA polymerase (Pol) II is a major rate-limiting step in HIV gene expression.  The viral Tat protein recruits human Super Elongation Complex (SEC) to paused Pol II to overcome this limitation.  Here, we identify the bromodomain protein Brd4 and its inhibition of Tat-transactivation as a major impediment to latency reactivation.  Brd4 competitively blocks the Tat-SEC interaction on HIV promoter.  The BET bromodomain inhibitor JQ1 dissocs. Brd4 from the HIV promoter to allow Tat recruitment of SEC to stimulate HIV elongation.  JQ1 synergizes with another latency activator prostratin, which promotes Pol II loading onto the viral promoter.  Because JQ1 activates viral latency without inducing global T cell activation, this and other closely related compds. and their antagonization of Brd4 to promote Tat-SEC interaction merit further investigations as effective agents/strategies for eliminating latent HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod_cOKnnk8oLVg90H21EOLACvtfcHk0ljjLYfsl3ZDXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D&md5=87866d81d13af3d5051844ed6c17c98d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks976%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DThe%2520Bet%2520Bromodomain%2520Inhibitor%2520Jq1%2520Activates%2520Hiv%2520Latency%2520through%2520Antagonizing%2520Brd4%2520Inhibition%2520of%2520Tat-Transactivation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D277%26epage%3D287%26doi%3D10.1093%2Fnar%2Fgks976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Identification of a Novel Series of Bet Family Bromodomain Inhibitors: Binding Mode and Profile of I-Bet151 (Gsk1210151a)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2968</span>– <span class="NLM_lpage">2972</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.02.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.-W.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+Novel+Series+of+Bet+Family+Bromodomain+Inhibitors%3A+Binding+Mode+and+Profile+of+I-Bet151+%28Gsk1210151a%29&doi=10.1016%2Fj.bmcl.2012.02.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0ljjLYfsl3ZDXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520Novel%2520Series%2520of%2520Bet%2520Family%2520Bromodomain%2520Inhibitors%253A%2520Binding%2520Mode%2520and%2520Profile%2520of%2520I-Bet151%2520%2528Gsk1210151a%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972%26doi%3D10.1016%2Fj.bmcl.2012.02.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou-Eymard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">The Discovery of I-Bet726 (Gsk1324726a), a Potent Tetrahydroquinoline Apoa1 up-Regulator and Selective Bet Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8111</span>– <span class="NLM_lpage">8131</span>, <span class="refDoi"> DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.-M.+G.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.-w.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=The+Discovery+of+I-Bet726+%28Gsk1324726a%29%2C+a+Potent+Tetrahydroquinoline+Apoa1+up-Regulator+and+Selective+Bet+Bromodomain+Inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0lgQNouz9iKqdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DThe%2520Discovery%2520of%2520I-Bet726%2520%2528Gsk1324726a%2529%252C%2520a%2520Potent%2520Tetrahydroquinoline%2520Apoa1%2520up-Regulator%2520and%2520Selective%2520Bet%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The Making of I-Bet762, a Bet Bromodomain Inhibitor Now in Clinical Development</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7498</span>– <span class="NLM_lpage">7500</span>, <span class="refDoi"> DOI: 10.1021/jm4014407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4014407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSmsrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7498-7500&author=Y.+Zhaoauthor=C.-Y.+Yangauthor=S.+Wang&title=The+Making+of+I-Bet762%2C+a+Bet+Bromodomain+Inhibitor+Now+in+Clinical+Development&doi=10.1021%2Fjm4014407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The Making of I-BET762, a BET Bromodomain Inhibitor Now in Clinical Development</span></div><div class="casAuthors">Zhao, Yujun; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7498-7500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) proteins belong to a class of proteins collectively called epigenetic "readers".  BET bromodomains have emerged as promising drug targets for treatment of cancers, inflammatory diseases, and other medical conditions.  GlaxoSmithKline scientists have successfully optimized a class of benzodiazepines as inhibitors of BET bromodomains, without any prior knowledge of identified mol. targets.  It thus is possible to hit a target without aiming at it.  The optimized lead compd. I-BET762 is currently being evaluated in a phase I clin. trial for treatment of human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN-T-2VhynyrVg90H21EOLACvtfcHk0lgQNouz9iKqdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSmsrbF&md5=61aa569c1b1b129b54229b9f8401f46b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm4014407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4014407%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520Making%2520of%2520I-Bet762%252C%2520a%2520Bet%2520Bromodomain%2520Inhibitor%2520Now%2520in%2520Clinical%2520Development%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7498%26epage%3D7500%26doi%3D10.1021%2Fjm4014407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kather, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlimpert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagies, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhardt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kammerer, B.</span></span> <span> </span><span class="NLM_article-title">Metabolic Response to Xd14 Treatment in Human Breast Cancer Cell Line Mcf-7</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1772</span>, <span class="refDoi"> DOI: 10.3390/ijms17101772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.3390%2Fijms17101772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1yksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1772&author=D.+Panauthor=M.+Katherauthor=L.+Willmannauthor=M.+Schlimpertauthor=C.+Bauerauthor=S.+Lagiesauthor=K.+Schmidtkunzauthor=S.+Eisenhardtauthor=M.+Jungauthor=S.+G%C3%BCntherauthor=B.+Kammerer&title=Metabolic+Response+to+Xd14+Treatment+in+Human+Breast+Cancer+Cell+Line+Mcf-7&doi=10.3390%2Fijms17101772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic response to XD14 treatment in human breast cancer cell line MCF-7</span></div><div class="casAuthors">Pan, Daqiang; Kather, Michel; Willmann, Lucas; Schlimpert, Manuel; Bauer, Christoph; Lagies, Simon; Schmidtkunz, Karin; Eisenhardt, Steffen U.; Jung, Manfred; Guenther, Stefan; Kammerer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1772/1-1772/15</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">XD14 is a 4-acyl pyrrole deriv., which was discovered by a high-throughput virtual screening expt.  XD14 inhibits bromodomain and extra-terminal domain (BET) proteins (BRD2, BRD3, BRD4 and BRDT) and consequently suppresses cell proliferation.  In this study, metabolic profiling reveals the mol. effects in the human breast cancer cell line MCF-7 (Michigan Cancer Foundation-7) treated by XD14.  A three-day time series expt. with two concns. of XD14 was performed.  Gas chromatog.-mass spectrometry (GC-MS) was applied for untargeted profiling of treated and non-treated MCF-7 cells.  The gained data sets were evaluated by several statistical methods: anal. of variance (ANOVA), clustering anal., principle component anal. (PCA), and partial least squares discriminant anal. (PLS-DA).  Cell proliferation was strongly inhibited by treatment with 50 μM XD14.  Samples could be discriminated by time and XD14 concn. using PLS-DA.  From the 117 identified metabolites, 67 were significantly altered after XD14 treatment.  These metabolites include amino acids, fatty acids, Krebs cycle and glycolysis intermediates, as well as compds. of purine and pyrimidine metab.  This massive intervention in energy metab. and the lack of available nucleotides could explain the decreased proliferation rate of the cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf1rN2g-3QFLVg90H21EOLACvtfcHk0lhkHx3XhCRulA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1yksg%253D%253D&md5=6e92165233c5cc213af50292dc373fa9</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.3390%2Fijms17101772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17101772%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DD.%26aulast%3DKather%26aufirst%3DM.%26aulast%3DWillmann%26aufirst%3DL.%26aulast%3DSchlimpert%26aufirst%3DM.%26aulast%3DBauer%26aufirst%3DC.%26aulast%3DLagies%26aufirst%3DS.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DEisenhardt%26aufirst%3DS.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DKammerer%26aufirst%3DB.%26atitle%3DMetabolic%2520Response%2520to%2520Xd14%2520Treatment%2520in%2520Human%2520Breast%2520Cancer%2520Cell%2520Line%2520Mcf-7%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D1772%26doi%3D10.3390%2Fijms17101772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafiane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiacchia, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimbou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genest, J.</span></span> <span> </span><span class="NLM_article-title">Rvx-208: A Small Molecule That Increases Apolipoprotein a-I and High-Density Lipoprotein Cholesterol in Vitro and in Vivo</span>. <i>J. Am. Coll. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2580</span>– <span class="NLM_lpage">2589</span>, <span class="refDoi"> DOI: 10.1016/j.jacc.2010.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.jacc.2010.02.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=20513599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2010&pages=2580-2589&author=D.+Baileyauthor=R.+Jahagirdarauthor=A.+Gordonauthor=A.+Hafianeauthor=S.+Campbellauthor=S.+Chaturauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=F.+S.+Chiacchiaauthor=J.+Johanssonauthor=L.+Krimbouauthor=N.+C.+W.+Wongauthor=J.+Genest&title=Rvx-208%3A+A+Small+Molecule+That+Increases+Apolipoprotein+a-I+and+High-Density+Lipoprotein+Cholesterol+in+Vitro+and+in+Vivo&doi=10.1016%2Fj.jacc.2010.02.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208. A small molecule that increases apolipoprotein A-I and high-density lipoprotein cholesterol in vitro and in vivo</span></div><div class="casAuthors">Bailey, Dana; Jahagirdar, Ravi; Gordon, Allan; Hafiane, Anouar; Campbell, Steven; Chatur, Safia; Wagner, Gregory S.; Hansen, Henrik C.; Chiacchia, Fabrizio S.; Johansson, Jan; Krimbou, Larbi; Wong, Norman C. W.; Genest, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American College of Cardiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2580-2589</span>CODEN:
                <span class="NLM_cas:coden">JACCDI</span>;
        ISSN:<span class="NLM_cas:issn">0735-1097</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Objectives: The aim of this study was to det. whether a novel small mol. RVX-208 affects apolipoprotein (apo)A-I and high-d. lipoprotein cholesterol (HDL-C) levels in vitro and in vivo.  Background: Increased apoA-I and HDL-C levels are potential therapeutic targets for reducing atherosclerotic disease.  Methods: HepG2 cells were treated with 0 to 60 μmol/l RVX-208 followed by assays for apoA-I and HDL-C prodn.  For in vivo studies, African green monkeys (AGMs) received 15 to 60 mg/kg/day RVX-208, and the serum was analyzed for lipoprotein levels, HDL-subparticle distribution, cholesterol efflux, and activity of lipid-modifying enzymes.  A phase I clin. trial was conducted in healthy volunteers (given 1 to 20 mg/kg/day of RVX-208) to assess safety, tolerability, and pharmacokinetics.  Results: The RVX-208 induced apoA-I mRNA and protein synthesis in HepG2 cells, leading to in- creased levels of pre-β-migrating and α-lipoprotein particles contg. apoA-I (LpA-I) in spent media.  Similarly, in AGMs, RVX-208 treatment for 63 days increased serum apoA-I and HDL-C levels (60% and 97%, resp.).  In addn., the levels of pre-β1-LpA-I and α1-LpA-I HDL-subparticles were increased as well as ATP binding cassette AI, ATP binding cassette G1, and scavenger receptor class B type I-dependent cholesterol efflux.  These changes were not mediated by cholesteryl-ester-transfer protein.  Treatment of humans for 1 wk with oral RVX-208 increased apoA-I, pre-β-HDL, and HDL functionality.  Conclusions: RVX-208 increases apoA-I and HDL-C in vitro and in vivo.  In AGMs, RVX-208 raises serum pre-β1-LpA-I and α-LpA-I levels and enhances cholesterol efflux.  Data in humans point to beneficial features of RVX-208 that might be useful for treating atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEGExcnx6drLVg90H21EOLACvtfcHk0lhkHx3XhCRulA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D&md5=aab9ac26e73704f1cd569e2ff47d0c51</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.jacc.2010.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacc.2010.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DD.%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DHafiane%26aufirst%3DA.%26aulast%3DCampbell%26aufirst%3DS.%26aulast%3DChatur%26aufirst%3DS.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DChiacchia%26aufirst%3DF.%2BS.%26aulast%3DJohansson%26aufirst%3DJ.%26aulast%3DKrimbou%26aufirst%3DL.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DGenest%26aufirst%3DJ.%26atitle%3DRvx-208%253A%2520A%2520Small%2520Molecule%2520That%2520Increases%2520Apolipoprotein%2520a-I%2520and%2520High-Density%2520Lipoprotein%2520Cholesterol%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Am.%2520Coll.%2520Cardiol.%26date%3D2010%26volume%3D55%26spage%3D2580%26epage%3D2589%26doi%3D10.1016%2Fj.jacc.2010.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Pfi-1, a Highly Selective Protein Interaction Inhibitor, Targeting Bet Bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3336</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F0008-5472.can-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+Mullerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=Pfi-1%2C+a+Highly+Selective+Protein+Interaction+Inhibitor%2C+Targeting+Bet+Bromodomains&doi=10.1158%2F0008-5472.can-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0lhkHx3XhCRulA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPfi-1%252C%2520a%2520Highly%2520Selective%2520Protein%2520Interaction%2520Inhibitor%252C%2520Targeting%2520Bet%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346%26doi%3D10.1158%2F0008-5472.can-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent and Selective Brd4 Inhibitors Capable of Blocking Tlr3-Induced Acute Airway Inflammation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejmech.2018.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29649741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=450-461&author=Z.+Liuauthor=B.+Tianauthor=H.+Chenauthor=P.+Wangauthor=A.+R.+Brasierauthor=J.+Zhou&title=Discovery+of+Potent+and+Selective+Brd4+Inhibitors+Capable+of+Blocking+Tlr3-Induced+Acute+Airway+Inflammation&doi=10.1016%2Fj.ejmech.2018.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation</span></div><div class="casAuthors">Liu, Zhiqing; Tian, Bing; Chen, Haiying; Wang, Pingyuan; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of diverse small mols. have been designed and synthesized through structure-based drug design by taking advantage of fragment merging and elaboration approaches.  Compds. ZL0420 (28) and ZL0454 (35) were identified as potent and selective BRD4 inhibitors with nanomolar binding affinities to bromodomains (BDs) of BRD4.  Both of them can be well docked into the acetyl-lysine (KAc) binding pocket of BRD4, forming key interactions including the crit. hydrogen bonds with Asn140 directly and Tyr97 indirectly via a H2O mol.  Both compds. 28 and 35 exhibited submicromolar potency of inhibiting the TLR3-dependent innate immune gene program, including ISG54, ISG56, IL-8, and Groβ genes in cultured human small airway epithelial cells (hSAECs).  More importantly, they also demonstrated potent efficacy reducing airway inflammation in a mouse model with low toxicity, indicating a proof of concept that BRD4 inhibitors may offer the therapeutic potential to block the viral-induced airway inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLyejdKuiGi7Vg90H21EOLACvtfcHk0lhRQfj342Nf4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D&md5=564b088a350abef9964d97bbecbb52b1</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Potent%2520and%2520Selective%2520Brd4%2520Inhibitors%2520Capable%2520of%2520Blocking%2520Tlr3-Induced%2520Acute%2520Airway%2520Inflammation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ejmech.2018.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-Methyl-7-Oxo-6,7-Dihydro-1h-Pyrrolo[2,3-C]Pyridin -4-Yl)Phenyl)Ethanesulfonamide (Abbv-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (Bet) Family Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>– <span class="NLM_lpage">8384</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-Difluorophenoxy%29-3-%286-Methyl-7-Oxo-6%2C7-Dihydro-1h-Pyrrolo%5B2%2C3-C%5DPyridin+-4-Yl%29Phenyl%29Ethanesulfonamide+%28Abbv-075%2FMivebresib%29%2C+a+Potent+and+Orally+Available+Bromodomain+and+Extraterminal+Domain+%28Bet%29+Family+Bromodomain+Inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lhsJ4SnEtmEdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-Difluorophenoxy%2529-3-%25286-Methyl-7-Oxo-6%252C7-Dihydro-1h-Pyrrolo%255B2%252C3-C%255DPyridin%2520-4-Yl%2529Phenyl%2529Ethanesulfonamide%2520%2528Abbv-075%252FMivebresib%2529%252C%2520a%2520Potent%2520and%2520Orally%2520Available%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528Bet%2529%2520Family%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonazzoli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Predolini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altwerger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menderes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zammataro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettinella, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manzano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buza, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litkouhi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silasi, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azodi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santin, A. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bet Bromodomain Proteins with Gs-5829 and Gs-626510 in Uterine Serous Carcinoma, a Biologically Aggressive Variant of Endometrial Cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4845</span>– <span class="NLM_lpage">4853</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-18-0864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1078-0432.ccr-18-0864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29941483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsFCksrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4845-4853&author=E.+Bonazzoliauthor=F.+Predoliniauthor=E.+Coccoauthor=S.+Belloneauthor=G.+Altwergerauthor=G.+Menderesauthor=L.+Zammataroauthor=A.+Bianchiauthor=F.+Pettinellaauthor=F.+Riccioauthor=C.+Hanauthor=G.+Yadavauthor=S.+Lopezauthor=A.+Manzanoauthor=P.+Manaraauthor=N.+Buzaauthor=P.+Huiauthor=S.+Wongauthor=B.+Litkouhiauthor=E.+Ratnerauthor=D.-A.+Silasiauthor=G.+S.+Huangauthor=M.+Azodiauthor=P.+E.+Schwartzauthor=J.+Schlessingerauthor=A.+D.+Santin&title=Inhibition+of+Bet+Bromodomain+Proteins+with+Gs-5829+and+Gs-626510+in+Uterine+Serous+Carcinoma%2C+a+Biologically+Aggressive+Variant+of+Endometrial+Cancer&doi=10.1158%2F1078-0432.ccr-18-0864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomain proteins with GS-5829 and GS-626510 in uterine serous carcinoma, a biologically aggressive variant of endometrial cancer</span></div><div class="casAuthors">Bonazzoli, Elena; Predolini, Federica; Cocco, Emiliano; Bellone, Stefania; Altwerger, Gary; Menderes, Gulden; Zammataro, Luca; Bianchi, Anna; Pettinella, Francesca; Riccio, Francesco; Han, Chanhee; Yadav, Ghanshyam; Lopez, Salvatore; Manzano, Aranzazu; Manara, Paola; Buza, Natalia; Hui, Pei; Wong, Serena; Litkouhi, Babak; Ratner, Elena; Silasi, Dan-Arin; Huang, Gloria S.; Azodi, Masoud; Schwartz, Peter E.; Schlessinger, Joseph; Santin, Alessandro D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4845-4853</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Uterine serous carcinoma (USC) is a rare and aggressive variant of endometrial cancer.  Whole-exome sequencing (WES) studies have recently reported c-Myc gene amplification in a large no. of USCs, suggesting c-Myc as a potential therapeutic target.  We investigated the activity of novel BET bromodomain inhibitors (GS-5829 and GS-626510, Gilead Sciences Inc.) and JQ1 against primary USC cultures and USC xenografts.  Exptl. Design: We evaluated c-Myc expression by qRT-PCR in a total of 45 USCs including fresh-frozen tumor tissues and primary USC cell lines.  We also performed IHC and Western blot expts. in 8 USC tumors.  USC cultures were evaluated for sensitivity to GS-5829, GS-626510, and JQ1 in vitro using proliferation, viability, and apoptosis assays.  Finally, the in vivo activity of GS-5829, GS-626510, and JQ1 was studied in USC-ARK1 and USC-ARK2 mouse xenografts.  Results: Fresh-frozen USC and primary USC cell lines overexpressed c-Myc when compared with normal tissues (P = 0.0009 and 0.0083, resp.).  High c-Myc expression was found in 7 of 8 of primary USC cell lines tested by qRT-PCR and 5 of 8 tested by IHC.  In vitro expts. demonstrated high sensitivity of USC cell lines to the exposure to GS-5829, GS-626510, and JQ1 with BET inhibitors causing a dose-dependent decrease in the phosphorylated levels of c-Myc and a dose-dependent increase in caspase activation (apoptosis).  In comparative in vivo expts., GS-5829 and/or GS-626510 were found more effective than JQ1 at the concns./doses used in decreasing tumor growth in both USC-ARK1 and USC-ARK2 mouse xenograft models.  Conclusions: GS-5829 and GS-626510 may represent novel, highly effective therapeutics agents against recurrent/chemotherapy-resistant USC-overexpressing c-Myc.  Clin. studies with GS-5829 in patients with USC harboring chemotherapy-resistant disease are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcTo-RvUJKTLVg90H21EOLACvtfcHk0lhsJ4SnEtmEdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsFCksrg%253D&md5=7474c8f5fd4b6772f87b12cf114b47e8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-18-0864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-18-0864%26sid%3Dliteratum%253Aachs%26aulast%3DBonazzoli%26aufirst%3DE.%26aulast%3DPredolini%26aufirst%3DF.%26aulast%3DCocco%26aufirst%3DE.%26aulast%3DBellone%26aufirst%3DS.%26aulast%3DAltwerger%26aufirst%3DG.%26aulast%3DMenderes%26aufirst%3DG.%26aulast%3DZammataro%26aufirst%3DL.%26aulast%3DBianchi%26aufirst%3DA.%26aulast%3DPettinella%26aufirst%3DF.%26aulast%3DRiccio%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DYadav%26aufirst%3DG.%26aulast%3DLopez%26aufirst%3DS.%26aulast%3DManzano%26aufirst%3DA.%26aulast%3DManara%26aufirst%3DP.%26aulast%3DBuza%26aufirst%3DN.%26aulast%3DHui%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DLitkouhi%26aufirst%3DB.%26aulast%3DRatner%26aufirst%3DE.%26aulast%3DSilasi%26aufirst%3DD.-A.%26aulast%3DHuang%26aufirst%3DG.%2BS.%26aulast%3DAzodi%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DP.%2BE.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DSantin%26aufirst%3DA.%2BD.%26atitle%3DInhibition%2520of%2520Bet%2520Bromodomain%2520Proteins%2520with%2520Gs-5829%2520and%2520Gs-626510%2520in%2520Uterine%2520Serous%2520Carcinoma%252C%2520a%2520Biologically%2520Aggressive%2520Variant%2520of%2520Endometrial%2520Cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D24%26spage%3D4845%26epage%3D4853%26doi%3D10.1158%2F1078-0432.ccr-18-0864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Gamal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hing, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blachly, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieselhuber, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goettl, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spevak, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carias, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantoy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanftner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nespi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathé, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span> <span> </span><span class="NLM_article-title">Brd4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to Plx51107, a Novel Structurally Distinct Bet Inhibitor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-17-0902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F2159-8290.cd-17-0902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29386193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=458-477&author=H.+G.+Ozerauthor=D.+El-Gamalauthor=B.+Powellauthor=Z.+A.+Hingauthor=J.+S.+Blachlyauthor=B.+Harringtonauthor=S.+Mitchellauthor=N.+R.+Grieselhuberauthor=K.+Williamsauthor=T.-H.+Laiauthor=L.+Alinariauthor=R.+A.+Baiocchiauthor=L.+Brintonauthor=E.+Baskinauthor=M.+Cannonauthor=L.+Beaverauthor=V.+M.+Goettlauthor=D.+M.+Lucasauthor=J.+A.+Woyachauthor=D.+Sampathauthor=A.+M.+Lehmanauthor=L.+Yuauthor=J.+Zhangauthor=Y.+Maauthor=Y.+Zhangauthor=W.+Spevakauthor=S.+Shiauthor=P.+Seversonauthor=R.+Shellooeauthor=H.+Cariasauthor=G.+Tsangauthor=K.+Dongauthor=T.+Ewingauthor=A.+Marimuthuauthor=C.+Tantoyauthor=J.+Waltersauthor=L.+Sanftnerauthor=H.+Rezaeiauthor=M.+Nespiauthor=B.+Matusowauthor=G.+Habetsauthor=P.+Ibrahimauthor=C.+Zhangauthor=E.+A.+Math%C3%A9author=G.+Bollagauthor=J.+C.+Byrdauthor=R.+Lapalombella&title=Brd4+Profiling+Identifies+Critical+Chronic+Lymphocytic+Leukemia+Oncogenic+Circuits+and+Reveals+Sensitivity+to+Plx51107%2C+a+Novel+Structurally+Distinct+Bet+Inhibitor&doi=10.1158%2F2159-8290.cd-17-0902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span></div><div class="casAuthors">Ozer, Hatice Gulcin; El-Gamal, Dalia; Powell, Ben; Hing, Zachary A.; Blachly, James S.; Harrington, Bonnie; Mitchell, Shaneice; Grieselhuber, Nicole R.; Williams, Katie; Lai, Tzung-Huei; Alinari, Lapo; Baiocchi, Robert A.; Brinton, Lindsey; Baskin, Elizabeth; Cannon, Matthew; Beaver, Larry; Goettl, Virginia M.; Lucas, David M.; Woyach, Jennifer A.; Sampath, Deepa; Lehman, Amy M.; Yu, Lianbo; Zhang, Jiazhong; Ma, Yan; Zhang, Ying; Spevak, Wayne; Shi, Songyuan; Severson, Paul; Shellooe, Rafe; Carias, Heidi; Tsang, Garson; Dong, Ken; Ewing, Todd; Marimuthu, Adhirai; Tantoy, Christina; Walters, Jason; Sanftner, Laura; Rezaei, Hamid; Nespi, Marika; Matusow, Bernice; Habets, Gaston; Ibrahim, Prabha; Zhang, Chao; Mathe, Ewy A.; Bollag, Gideon; Byrd, John C.; Lapalombella, Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-477</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins are key regulators of gene expression in cancer.  Herein, we utilize BRD4 profiling to identify crit. pathways involved in pathogenesis of chronic lymphocytic leukemia (CLL).  BRD4 is overexpressed in CLL and is enriched proximal to genes upregulated or de novo expressed in CLL with known functions in disease pathogenesis and progression.  These genes, including key members of the B-cell receptor (BCR) signaling pathway, provide a rationale for this therapeutic approach to identify new targets in alternative types of cancer.  Addnl., we describe PLX51107, a structurally distinct BET inhibitor with novel in vitro and in vivo pharmacol. properties that emulates or exceeds the efficacy of BCR signaling agents in preclin. models of CLL.  Herein, the discovery of the involvement of BRD4 in the core CLL transcriptional program provides a compelling rationale for clin. investigation of PLX51107 as epigenetic therapy in CLL and application of BRD4 profiling in other cancers.  Significance: To date, functional studies of BRD4 in CLL are lacking.  Through integrated genomic, functional, and pharmacol. analyses, we uncover the existence of BRD4-regulated core CLL transcriptional programs and present preclin. proof-of-concept studies validating BET inhibition as an epigenetic approach to target BCR signaling in CLL.  Cancer Discov; 8(4); 458-77. ©2018 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvzZtZKkb5rVg90H21EOLACvtfcHk0ljESIaPj7U75g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D&md5=3715a65bb8cee2a5fe53994a3d106303</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.cd-17-0902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.cd-17-0902%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DH.%2BG.%26aulast%3DEl-Gamal%26aufirst%3DD.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHing%26aufirst%3DZ.%2BA.%26aulast%3DBlachly%26aufirst%3DJ.%2BS.%26aulast%3DHarrington%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DS.%26aulast%3DGrieselhuber%26aufirst%3DN.%2BR.%26aulast%3DWilliams%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DT.-H.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DBrinton%26aufirst%3DL.%26aulast%3DBaskin%26aufirst%3DE.%26aulast%3DCannon%26aufirst%3DM.%26aulast%3DBeaver%26aufirst%3DL.%26aulast%3DGoettl%26aufirst%3DV.%2BM.%26aulast%3DLucas%26aufirst%3DD.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DK.%26aulast%3DEwing%26aufirst%3DT.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DTantoy%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DJ.%26aulast%3DSanftner%26aufirst%3DL.%26aulast%3DRezaei%26aufirst%3DH.%26aulast%3DNespi%26aufirst%3DM.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMath%25C3%25A9%26aufirst%3DE.%2BA.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DLapalombella%26aufirst%3DR.%26atitle%3DBrd4%2520Profiling%2520Identifies%2520Critical%2520Chronic%2520Lymphocytic%2520Leukemia%2520Oncogenic%2520Circuits%2520and%2520Reveals%2520Sensitivity%2520to%2520Plx51107%252C%2520a%2520Novel%2520Structurally%2520Distinct%2520Bet%2520Inhibitor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D458%26epage%3D477%26doi%3D10.1158%2F2159-8290.cd-17-0902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greer, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, I.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. H.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases Positively Regulate Transcription through the Elongation Machinery</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1444</span>– <span class="NLM_lpage">1455</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.celrep.2015.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26549458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=1444-1455&author=C.+B.+Greerauthor=Y.+Tanakaauthor=Y.+J.+Kimauthor=P.+Xieauthor=M.+Q.+Zhangauthor=I.-H.+Parkauthor=T.+H.+Kim&title=Histone+Deacetylases+Positively+Regulate+Transcription+through+the+Elongation+Machinery&doi=10.1016%2Fj.celrep.2015.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylases Positively Regulate Transcription through the Elongation Machinery</span></div><div class="casAuthors">Greer, Celeste B.; Tanaka, Yoshiaki; Kim, Yoon Jung; Xie, Peng; Zhang, Michael Q.; Park, In-Hyun; Kim, Tae Hoon</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1444-1455</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Transcription elongation regulates the expression of many genes, including oncogenes.  Histone deacetylase (HDAC) inhibitors (HDACIs) block elongation, suggesting that HDACs are involved in gene activation.  To understand this, we analyzed nascent transcription and elongation factor binding genome-wide after perturbation of elongation with small mol. inhibitors.  We found that HDACI-mediated repression requires heat shock protein 90 (HSP90) activity.  HDACIs promote the assocn. of RNA polymerase II (RNAP2) and neg. elongation factor (NELF), a complex stabilized by HSP90, at the same genomic sites.  Addnl., HDACIs redistribute bromodomain-contg. protein 4 (BRD4), a key elongation factor involved in enhancer activity.  BRD4 binds to newly acetylated sites, and its occupancy at promoters and enhancers is reduced.  Furthermore, HDACIs reduce enhancer activity, as measured by enhancer RNA prodn.  Therefore, HDACs are required for limiting acetylation in gene bodies and intergenic regions.  This facilitates the binding of elongation factors to properly acetylated promoters and enhancers for efficient elongation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqFzVFizqpAbVg90H21EOLACvtfcHk0lh_as8RmfNvcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2nu7jK&md5=73a695b1a306db9eb042b28af3c7300d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DC.%2BB.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DXie%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%2BQ.%26aulast%3DPark%26aufirst%3DI.-H.%26aulast%3DKim%26aufirst%3DT.%2BH.%26atitle%3DHistone%2520Deacetylases%2520Positively%2520Regulate%2520Transcription%2520through%2520the%2520Elongation%2520Machinery%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D13%26spage%3D1444%26epage%3D1455%26doi%3D10.1016%2Fj.celrep.2015.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi-Yachide, T.</span></span> <span> </span><span class="NLM_article-title">Bet Bromodomain as a Target of Epigenetic Therapy</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1248/cpb.c16-00225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1248%2Fcpb.c16-00225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27250788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Omur7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2016&pages=540-547&author=T.+Noguchi-Yachide&title=Bet+Bromodomain+as+a+Target+of+Epigenetic+Therapy&doi=10.1248%2Fcpb.c16-00225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain as a Target of Epigenetic Therapy</span></div><div class="casAuthors">Noguchi-Yachide, Tomomi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-547</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Acetylation of histone is a key epigenetic modification, and contributes to many DNA-dependent cellular processes.  The bromodomain structure, which consists of approx. 110 amino acid residues, serves as a 'reader' that recognizes acetylated lysine in histones, leading to recruitment of pos. transcriptional elongation factor b (P-TEFb), and thereby promoting transcriptional activity and chromatin remodeling.  Among bromodomain-contg. proteins, members of the bromodomain and extra-terminal domain (BET) family contain tandem N-terminal bromodomains.  BET proteins, esp. BRD4, have attracted interest as candidate therapeutic targets due to their putative involvement in the pathogenesis of various diseases, including cancer and inflammatory diseases.  Several BET inhibitors are under clin. trial for treatment of various cancers.  Furthermore, polypharmacol. agents such as dual kinase/BET inhibitors and dual histone deacetylase (HDAC)/BET inhibitors have recently been developed, in addn. to agents that degrade BET family proteins, such as proteolysis-targeting chimeras (PROTACs).  This paper reviews recent progress in epigenetic therapy targeting the BET bromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxzWdAhkzHKbVg90H21EOLACvtfcHk0lh_as8RmfNvcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Omur7L&md5=a1787a330f39a6005daf2e077dc94b13</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c16-00225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c16-00225%26sid%3Dliteratum%253Aachs%26aulast%3DNoguchi-Yachide%26aufirst%3DT.%26atitle%3DBet%2520Bromodomain%2520as%2520a%2520Target%2520of%2520Epigenetic%2520Therapy%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2016%26volume%3D64%26spage%3D540%26epage%3D547%26doi%3D10.1248%2Fcpb.c16-00225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neele, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van den Bossche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeksema, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Winther, M. P. J.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Pathways in Macrophages Emerge as Novel Targets in Atherosclerosis</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>763</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.ejphar.2015.03.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ejphar.2015.03.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=26004034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovV2rtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=763&publication_year=2015&pages=79-89&author=A.+E.+Neeleauthor=J.+Van+den+Bosscheauthor=M.+A.+Hoeksemaauthor=M.+P.+J.+de+Winther&title=Epigenetic+Pathways+in+Macrophages+Emerge+as+Novel+Targets+in+Atherosclerosis&doi=10.1016%2Fj.ejphar.2015.03.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic pathways in macrophages emerge as novel targets in atherosclerosis</span></div><div class="casAuthors">Neele, Annette E.; Van den Bossche, Jan; Hoeksema, Marten A.; de Winther, Menno P. J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">763</span>
        (<span class="NLM_cas:issue">Part_A</span>),
    <span class="NLM_cas:pages">79-89</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Atherosclerosis is a lipid-driven chronic inflammatory disorder.  Monocytes and macrophages are key immune cells in the development of disease and clin. outcome.  It is becoming increasingly clear that epigenetic pathways govern many aspects of monocyte and macrophage differentiation and activation.  The dynamic regulation of epigenetic patterns provides opportunities to alter disease-assocd. epigenetic states.  Therefore, pharmaceutical companies have embraced the targeting of epigenetic processes as new approaches for interventions.  Particularly histone deacetylase (Hdac) inhibitors and DNA-methyltransferase inhibitors have long received attention and several of them have been approved for clin. use in relation to hematol. malignancies.  The key focus is still on oncol., but Alzheimer's disease, Huntington's disease and inflammatory disorders are coming in focus as well.  These developments raise opportunities for the epigenetic targeting in cardiovascular disease (CVD).  In this review we discuss the epigenetic regulation of the inflammatory pathways in relation to atherosclerosis with a specific attention to monocyte- and macrophage-related processes.  What are the opportunities for future therapy of atherosclerosis by epigenetic interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr-t4fA2Z1OLVg90H21EOLACvtfcHk0lh_as8RmfNvcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovV2rtL0%253D&md5=804d86c0b66b3b3851b26dc41310812a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2015.03.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2015.03.101%26sid%3Dliteratum%253Aachs%26aulast%3DNeele%26aufirst%3DA.%2BE.%26aulast%3DVan%2Bden%2BBossche%26aufirst%3DJ.%26aulast%3DHoeksema%26aufirst%3DM.%2BA.%26aulast%3Dde%2BWinther%26aufirst%3DM.%2BP.%2BJ.%26atitle%3DEpigenetic%2520Pathways%2520in%2520Macrophages%2520Emerge%2520as%2520Novel%2520Targets%2520in%2520Atherosclerosis%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D763%26spage%3D79%26epage%3D89%26doi%3D10.1016%2Fj.ejphar.2015.03.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiskus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenta, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaub, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portier, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaraj, S. G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, K. N.</span></span> <span> </span><span class="NLM_article-title">Highly Active Combination of Brd4 Antagonist and Histone Deacetylase Inhibitor against Human Acute Myelogenous Leukemia Cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1142</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-13-0770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1535-7163.mct-13-0770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24435446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1142-1154&author=W.+Fiskusauthor=S.+Sharmaauthor=J.+Qiauthor=J.+A.+Valentaauthor=L.+J.+Schaubauthor=B.+Shahauthor=K.+Pethauthor=B.+P.+Portierauthor=M.+Rodriguezauthor=S.+G.+T.+Devarajauthor=M.+Zhanauthor=J.+Shengauthor=S.+P.+Iyerauthor=J.+E.+Bradnerauthor=K.+N.+Bhalla&title=Highly+Active+Combination+of+Brd4+Antagonist+and+Histone+Deacetylase+Inhibitor+against+Human+Acute+Myelogenous+Leukemia+Cells&doi=10.1158%2F1535-7163.mct-13-0770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Active Combination of BRD4 Antagonist and Histone Deacetylase Inhibitor against Human Acute Myelogenous Leukemia Cells</span></div><div class="casAuthors">Fiskus, Warren; Sharma, Sunil; Qi, Jun; Valenta, John A.; Schaub, Leasha J.; Shah, Bhavin; Peth, Karissa; Portier, Bryce P.; Rodriguez, Melissa; Devaraj, Santhana G. T.; Zhan, Ming; Sheng, Jianting; Iyer, Swaminathan P.; Bradner, James E.; Bhalla, Kapil N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1142-1154</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) protein family members, including BRD4, bind to acetylated lysines on histones and regulate the expression of important oncogenes, for example, c-MYC and BCL2.  Here, we demonstrate the sensitizing effects of the histone hyperacetylation-inducing pan-histone deacetylase (HDAC) inhibitor panobinostat on human acute myelogenous leukemia (AML) blast progenitor cells (BPC) to the BET protein antagonist JQ1.  Treatment with JQ1, but not its inactive enantiomer (R-JQ1), was highly lethal against AML BPCs expressing mutant NPM1c+ with or without coexpression of FLT3-ITD or AML expressing mixed lineage leukemia fusion oncoprotein.  JQ1 treatment reduced binding of BRD4 and RNA polymerase II to the DNA of c-MYC and BCL2 and reduced their levels in the AML cells.  Cotreatment with JQ1 and the HDAC inhibitor panobinostat synergistically induced apoptosis of the AML BPCs, but not of normal CD34+ hematopoietic progenitor cells.  This was assocd. with greater attenuation of c-MYC and BCL2, while increasing p21, BIM, and cleaved PARP levels in the AML BPCs.  Cotreatment with JQ1 and panobinostat significantly improved the survival of the NOD/SCID mice engrafted with OCI-AML3 or MOLM13 cells (P < 0.01).  These findings highlight cotreatment with a BRD4 antagonist and an HDAC inhibitor as a potentially efficacious therapy of AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDqEnKe8Ipo7Vg90H21EOLACvtfcHk0li4Dc2eaBxJrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFWjsbs%253D&md5=7e73581df913111b8bc3cc80c1a081f2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-13-0770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-13-0770%26sid%3Dliteratum%253Aachs%26aulast%3DFiskus%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DValenta%26aufirst%3DJ.%2BA.%26aulast%3DSchaub%26aufirst%3DL.%2BJ.%26aulast%3DShah%26aufirst%3DB.%26aulast%3DPeth%26aufirst%3DK.%26aulast%3DPortier%26aufirst%3DB.%2BP.%26aulast%3DRodriguez%26aufirst%3DM.%26aulast%3DDevaraj%26aufirst%3DS.%2BG.%2BT.%26aulast%3DZhan%26aufirst%3DM.%26aulast%3DSheng%26aufirst%3DJ.%26aulast%3DIyer%26aufirst%3DS.%2BP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DBhalla%26aufirst%3DK.%2BN.%26atitle%3DHighly%2520Active%2520Combination%2520of%2520Brd4%2520Antagonist%2520and%2520Histone%2520Deacetylase%2520Inhibitor%2520against%2520Human%2520Acute%2520Myelogenous%2520Leukemia%2520Cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D1142%26epage%3D1154%26doi%3D10.1158%2F1535-7163.mct-13-0770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span> <span> </span><span class="NLM_article-title">Histone Cross-Talk Connects Protein Phosphatase 1 Alpha (Pp1 Alpha) and Histone Deacetylase (Hdac) Pathways to Regulate the Functional Transition of Bromodomain-Containing 4 (Brd4) for Inducible Gene Expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">23154</span>– <span class="NLM_lpage">23167</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.570812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1074%2Fjbc.m114.570812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=24939842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlGmtbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=23154-23167&author=X.+Huauthor=X.+Luauthor=R.+Liuauthor=N.+Aiauthor=Z.+Caoauthor=Y.+Liauthor=J.+Liuauthor=B.+Yuauthor=K.+Liuauthor=H.+Wangauthor=C.+Zhouauthor=Y.+Wangauthor=A.+Hanauthor=F.+Dingauthor=R.+Chen&title=Histone+Cross-Talk+Connects+Protein+Phosphatase+1+Alpha+%28Pp1+Alpha%29+and+Histone+Deacetylase+%28Hdac%29+Pathways+to+Regulate+the+Functional+Transition+of+Bromodomain-Containing+4+%28Brd4%29+for+Inducible+Gene+Expression&doi=10.1074%2Fjbc.m114.570812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Histone cross-talk connects protein phosphatase 1α (PP1α) and histone deacetylase (HDAC) pathways to regulate the functional transition of bromodomain-containing 4 (BRD4) for inducible gene expression</span></div><div class="casAuthors">Hu, Xiangming; Lu, Xiaodong; Liu, Runzhong; Ai, Nanping; Cao, Zhenhua; Li, Yannan; Liu, Jiangfang; Yu, Bin; Liu, Kai; Wang, Huiping; Zhou, Chao; Wang, Yu; Han, Aidong; Ding, Feng; Chen, Ruichuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">23154-23167</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcription elongation has been recognized as a rate-limiting step for the expression of signal-inducible genes.  Through recruitment of pos. transcription elongation factor P-TEFb, the bromodomain-contg. protein BRD4 plays crit. roles in regulating the transcription elongation of a vast array of inducible genes that are important for multiple cellular processes.  The diverse biol. roles of BRD4 have been proposed to rely on its functional transition between chromatin targeting and transcription regulation.  The signaling pathways and the mol. mechanism for regulating this transition process, however, are largely unknown.  Here, the authors report a novel role of phosphorylated Ser10 of histone H3 (H3S10ph) in governing the functional transition of BRD4.  The authors identified that the acetylated lysines 5 and 8 of nucleosomal histone H4 (H4K5ac/K8ac) is the BRD4 binding site, and the protein phosphatase PP1α and class I histone deacetylase (HDAC1/2/3) signaling pathways are essential for the stress-induced BRD4 release from chromatin.  In the unstressed state, phosphorylated H3S10 prevents the deacetylation of nucleosomal H4K5ac/K8ac by HDAC1/2/3, thereby locking up the majority of BRD4 onto chromatin.  Upon stress, PP1α-mediated dephosphorylation of H3S10ph allows the deacetylation of nucleosomal H4K5ac/K8ac by HDAC1/2/3, thereby leading to the release of chromatin-bound BRD4 for subsequent recruitment of P-TEFb to enhance the expression of inducible genes.  Therefore, this study revealed a novel mechanism that the histone cross-talk between H3S10ph and H4K5ac/K8ac connects PP1α and HDACs to govern the functional transition of BRD4.  Combined with previous studies on the regulation of P-TEFb activation, the intricate signaling network for the tight control of transcription elongation is established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLzn-a41szPbVg90H21EOLACvtfcHk0li4Dc2eaBxJrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGmtbjI&md5=b3b79f952cd57bffe39c6d563d564669</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m114.570812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m114.570812%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DAi%26aufirst%3DN.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DA.%26aulast%3DDing%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DR.%26atitle%3DHistone%2520Cross-Talk%2520Connects%2520Protein%2520Phosphatase%25201%2520Alpha%2520%2528Pp1%2520Alpha%2529%2520and%2520Histone%2520Deacetylase%2520%2528Hdac%2529%2520Pathways%2520to%2520Regulate%2520the%2520Functional%2520Transition%2520of%2520Bromodomain-Containing%25204%2520%2528Brd4%2529%2520for%2520Inducible%2520Gene%2520Expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D23154%26epage%3D23167%26doi%3D10.1074%2Fjbc.m114.570812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegwitz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosinsky, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgartner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siveke, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schildhaus, H.-U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Class-I Inhibition Promotes Epithelial Gene Expression in Pancreatic Cancer Cells in a Brd4- and Myc-Dependent Manner</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">6334</span>– <span class="NLM_lpage">6349</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1093%2Fnar%2Fgkx212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28369619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1alt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=6334-6349&author=V.+K.+Mishraauthor=F.+Wegwitzauthor=R.+L.+Kosinskyauthor=M.+Senauthor=R.+Baumgartnerauthor=T.+Wulffauthor=J.+T.+Sivekeauthor=H.-U.+Schildhausauthor=Z.+Najafovaauthor=V.+Kariauthor=H.+Kohlhofauthor=E.+Hessmannauthor=S.+A.+Johnsen&title=Histone+Deacetylase+Class-I+Inhibition+Promotes+Epithelial+Gene+Expression+in+Pancreatic+Cancer+Cells+in+a+Brd4-+and+Myc-Dependent+Manner&doi=10.1093%2Fnar%2Fgkx212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase class-I inhibition promotes epithelial gene expression in pancreatic cancer cells in a BRD4- and MYC-dependent manner</span></div><div class="casAuthors">Mishra, Vivek Kumar; Wegwitz, Florian; Kosinsky, Robyn Laura; Sen, Madhobi; Baumgartner, Roland; Wulff, Tanja; Siveke, Jens T.; Schildhaus, Hans-Ulrich; Najafova, Zeynab; Kari, Vijayalakshmi; Kohlhof, Hella; Hessmann, Elisabeth; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6334-6349</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with a particularly dismal prognosis.  Histone deacetylases (HDAC) are epigenetic modulators whose activity is frequently deregulated in various cancers including PDAC.  In particular, class-I HDACs (HDAC 1, 2, 3 and 8) have been shown to play an important role in PDAC.  In this study, we investigated the effects of the class Ispecific HDAC inhibitor (HDACi) 4SC-202 in multiple PDAC cell lines in promoting tumor cell differentiation.  We show that 4SC-202 neg. affects TGF signaling and inhibits TGFβ-induced epithelial-mesenchymal transition (EMT).  Moreover, 4SC-202 markedly induced p21 (CDKN1A) expression and significantly attenuated cell proliferation.  Mechanistically, genome-wide studies revealed that 4SC-202- induced genes were enriched for bromodomaincontaining Protein-4 (BRD4) and MYC occupancy.  BRD4, a well-characterized acetyllysine reader, has been shown to play a major role in regulating transcription of selected subsets of genes.  Importantly, BRD4 and MYC are essential for the expression of a subgroup of genes induced by class-I HDACi.  Taken together, our study uncovers a previously unknown role of BRD4 and MYC in eliciting the HDACimediated induction of a subset of genes and provides mol. insight into the mechanisms of HDACi action in PDAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTaKoSDVpOerVg90H21EOLACvtfcHk0lhesAJ7SW3a-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1alt70%253D&md5=7852bb6d93a7791e99fd183cbebc3214</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx212%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DV.%2BK.%26aulast%3DWegwitz%26aufirst%3DF.%26aulast%3DKosinsky%26aufirst%3DR.%2BL.%26aulast%3DSen%26aufirst%3DM.%26aulast%3DBaumgartner%26aufirst%3DR.%26aulast%3DWulff%26aufirst%3DT.%26aulast%3DSiveke%26aufirst%3DJ.%2BT.%26aulast%3DSchildhaus%26aufirst%3DH.-U.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DKari%26aufirst%3DV.%26aulast%3DKohlhof%26aufirst%3DH.%26aulast%3DHessmann%26aufirst%3DE.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DHistone%2520Deacetylase%2520Class-I%2520Inhibition%2520Promotes%2520Epithelial%2520Gene%2520Expression%2520in%2520Pancreatic%2520Cancer%2520Cells%2520in%2520a%2520Brd4-%2520and%2520Myc-Dependent%2520Manner%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D6334%26epage%3D6349%26doi%3D10.1093%2Fnar%2Fgkx212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span> <span> </span><span class="NLM_article-title">Feedback Activation of Leukemia Inhibitory Factor Receptor Limits Response to Histone Deacetylase Inhibitors in Breast Cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.ccell.2016.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27622335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=459-473&author=H.+Zengauthor=J.+Quauthor=N.+Jinauthor=J.+Xuauthor=C.+Linauthor=Y.+Chenauthor=X.+Yangauthor=X.+Heauthor=S.+Tangauthor=X.+Lanauthor=X.+Yangauthor=Z.+Chenauthor=M.+Huangauthor=J.+Dingauthor=M.+Geng&title=Feedback+Activation+of+Leukemia+Inhibitory+Factor+Receptor+Limits+Response+to+Histone+Deacetylase+Inhibitors+in+Breast+Cancer&doi=10.1016%2Fj.ccell.2016.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Feedback Activation of Leukemia Inhibitory Factor Receptor Limits Response to Histone Deacetylase Inhibitors in Breast Cancer</span></div><div class="casAuthors">Zeng, Hanlin; Qu, Jia; Jin, Nan; Xu, Jun; Lin, Chenchu; Chen, Yi; Yang, Xinying; He, Xiang; Tang, Shuai; Lan, Xiaojing; Yang, Xiaotong; Chen, Ziqi; Huang, Min; Ding, Jian; Geng, Meiyu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-473</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors have demonstrated clin. benefits in subtypes of hematol. malignancies.  However, the efficacy of HDAC inhibitors in solid tumors remains uncertain.  This study takes breast cancer as a model to understand mechanisms accounting for limited response of HDAC inhibitors in solid tumors and to seek combination solns.  We discover that feedback activation of leukemia inhibitory factor receptor (LIFR) signaling in breast cancer limits the response to HDAC inhibition.  Mechanistically, HDAC inhibition increases histone acetylation at the LIFR gene promoter, which recruits bromodomain protein BRD4, upregulates LIFR expression, and activates JAK1-STAT3 signaling.  Importantly, JAK1 or BRD4 inhibition sensitizes breast cancer to HDAC inhibitors, implicating combination inhibition of HDAC with JAK1 or BRD4 as potential therapies for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL24sYbhFYeLVg90H21EOLACvtfcHk0lhesAJ7SW3a-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI&md5=9ce620c7a19519a95edd714f8012518c</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DLan%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26atitle%3DFeedback%2520Activation%2520of%2520Leukemia%2520Inhibitory%2520Factor%2520Receptor%2520Limits%2520Response%2520to%2520Histone%2520Deacetylase%2520Inhibitors%2520in%2520Breast%2520Cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D459%26epage%3D473%26doi%3D10.1016%2Fj.ccell.2016.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bet Bromodomain Inhibitors and Their Role in Target Validation</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1039/c3md00291h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1039%2Fc3md00291h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=288-296&author=S.+M%C3%BCllerauthor=S.+Knapp&title=Discovery+of+Bet+Bromodomain+Inhibitors+and+Their+Role+in+Target+Validation&doi=10.1039%2Fc3md00291h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of BET bromodomain inhibitors and their role in target validation</span></div><div class="casAuthors">Mueller, S.; Knapp, S.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-296</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are protein interaction modules that selectively recognize .vepsiln.-N-acetylated lysine residues.  BRDs are present in diverse proteins that play key functions in chromatin organization and regulation of gene transcription.  Aberrant transcription is a hallmark of many diseases in particular cancer and inflammation.  The complexity of mol. processes regulating gene transcription identified transcriptional regulators as interesting targets for the development of specific chem. tool mols. (chem. probes) that help to understand the mol. mechanisms of transcription and to explore the potential of BRD mediated interactions as sites for pharmaceutical intervention.  Recently a no. of highly specific inhibitors have been developed against the BET (bromo and extra terminal) family of bromodomains.  The availability of selective BRD inhibitors had a significant impact on the validation of bromodomain-contg. proteins as targets for drug development and for our understanding of the biol. roles of these proteins.  In this review we will summarize the discovery of BET bromodomain inhibitors and their roles in target validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbUCy0a3RkObVg90H21EOLACvtfcHk0ljZJLOqGmDHUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymu70%253D&md5=88e7b8c3909462666a083cad35efb135</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2Fc3md00291h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00291h%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Bet%2520Bromodomain%2520Inhibitors%2520and%2520Their%2520Role%2520in%2520Target%2520Validation%26jtitle%3DMedchemcomm%26date%3D2014%26volume%3D5%26spage%3D288%26epage%3D296%26doi%3D10.1039%2Fc3md00291h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting Epigenetic Reader and Eraser: Rational Design, Synthesis and in Vitro Evaluation of Dimethylisoxazoles Derivatives as Brd4/Hdac Dual Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2931</span>– <span class="NLM_lpage">2935</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmcl.2016.04.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27142751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2931-2935&author=Z.+Zhangauthor=S.+Houauthor=H.+Chenauthor=T.+Ranauthor=F.+Jiangauthor=Y.+Bianauthor=D.+Zhangauthor=Y.+Zhiauthor=L.+Wangauthor=L.+Zhangauthor=H.+Liauthor=Y.+Zhangauthor=W.+Tangauthor=T.+Luauthor=Y.+Chen&title=Targeting+Epigenetic+Reader+and+Eraser%3A+Rational+Design%2C+Synthesis+and+in+Vitro+Evaluation+of+Dimethylisoxazoles+Derivatives+as+Brd4%2FHdac+Dual+Inhibitors&doi=10.1016%2Fj.bmcl.2016.04.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic reader and eraser: Rational design, synthesis and in vitro evaluation of dimethylisoxazoles derivatives as BRD4/HDAC dual inhibitors</span></div><div class="casAuthors">Zhang, Zhimin; Hou, Shaohua; Chen, Hongli; Ran, Ting; Jiang, Fei; Bian, Yuanyuan; Zhang, Dewei; Zhi, Yanle; Wang, Lu; Zhang, Li; Li, Hongmei; Zhang, Yanmin; Tang, Weifang; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2931-2935</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bromodomain protein module and histone deacetylase (HDAC), which recognize and remove acetylated lysine, resp., have emerged as important epigenetic therapeutic targets in cancer treatments.  Herein the authors presented a novel design approach for cancer drug development by combination of bromodomain and HDAC inhibitory activity in one mol.  The designed compds. were synthesized which showed inhibitory activity against bromodomain 4 and HDAC1.  The representative dual bromodomain/HDAC inhibitors, compds. N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N7-hydroxyheptanediamide and N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N8-hydroxyoctanediamide showed potent antiproliferative activities against human leukemia cell line K562 and MV4-11 in cellular assays.  This work may lay the foundation for developing dual bromodomain/HDAC inhibitors as potential anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriS0VB1G-QoLVg90H21EOLACvtfcHk0ljZJLOqGmDHUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D&md5=a6ebdbd6402e3117a2f9d96ed071790c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.034%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHou%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DBian%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DTargeting%2520Epigenetic%2520Reader%2520and%2520Eraser%253A%2520Rational%2520Design%252C%2520Synthesis%2520and%2520in%2520Vitro%2520Evaluation%2520of%2520Dimethylisoxazoles%2520Derivatives%2520as%2520Brd4%252FHdac%2520Dual%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2931%26epage%3D2935%26doi%3D10.1016%2Fj.bmcl.2016.04.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Potent Dual Bet/Hdac Inhibitors for Efficient Treatment of Pancreatic Cancer</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3028</span>– <span class="NLM_lpage">3032</span>, <span class="refDoi"> DOI: 10.1002/anie.201915896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1002%2Fanie.201915896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=3028-3032&author=S.+Heauthor=G.+Dongauthor=Y.+Liauthor=S.+Wuauthor=W.+Wangauthor=C.+Sheng&title=Potent+Dual+Bet%2FHdac+Inhibitors+for+Efficient+Treatment+of+Pancreatic+Cancer&doi=10.1002%2Fanie.201915896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Dual BET/HDAC Inhibitors for Efficient Treatment of Pancreatic Cancer</span></div><div class="casAuthors">He, Shipeng; Dong, Guoqiang; Li, Yu; Wu, Shanchao; Wang, Wei; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3028-3032</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">As one of the most aggressive and lethal human malignancies with extremely poor prognosis, there is an urgent demand of more effective therapy for the treatment of pancreatic cancer.  Reported here is a new, effective therapeutic strategy and the design of small-mol. inhibitors that simultaneously target bromodomain and extra-terminal (BET) and histone deacetylase (HDAC), potentially serving as promising therapeutic agents for pancreatic cancer.  A highly potent dual inhibitor (13 a) is identified to possess excellent and balanced activities against BRD4 BD1 (IC50=11 nM) and HDAC1 (IC50=21 nM).  Notably, this compd. shows higher in vitro and in vivo antitumor potency than the BET inhibitor (+)-JQ1 and the HDAC inhibitor vorinostat, either alone or and in combination, highlighting the advantages of BET/HDAC dual inhibitors for more effective treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BFNcLfkA4bVg90H21EOLACvtfcHk0lg520PpbNJc5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D&md5=78db62cb533611939d9abd0e654ff5f3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201915896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201915896%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DPotent%2520Dual%2520Bet%252FHdac%2520Inhibitors%2520for%2520Efficient%2520Treatment%2520of%2520Pancreatic%2520Cancer%26jtitle%3DAngew.%2520Chem.%26date%3D2020%26volume%3D59%26spage%3D3028%26epage%3D3032%26doi%3D10.1002%2Fanie.201915896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design, Synthesis and in Vitro Antiproliferative Effects Studies of Novel Dual Brd4/Hdac Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4051</span>– <span class="NLM_lpage">4055</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.07.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1016%2Fj.bmcl.2017.07.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28765013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=4051-4055&author=M.+Shaoauthor=L.+Heauthor=L.+Zhengauthor=L.+Huangauthor=Y.+Zhouauthor=T.+Wangauthor=Y.+Chenauthor=M.+Shenauthor=F.+Wangauthor=Z.+Yangauthor=L.+Chen&title=Structure-Based+Design%2C+Synthesis+and+in+Vitro+Antiproliferative+Effects+Studies+of+Novel+Dual+Brd4%2FHdac+Inhibitors&doi=10.1016%2Fj.bmcl.2017.07.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design, synthesis and in vitro antiproliferative effects studies of novel dual BRD4/HDAC inhibitors</span></div><div class="casAuthors">Shao, Mingfeng; He, Linhong; Zheng, Li; Huang, Lingxiao; Zhou, Yuanyuan; Wang, Taijing; Chen, Yong; Shen, Mingsheng; Wang, Fang; Yang, Zhuang; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4051-4055</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone acetylation marks play important roles in controlling gene expressions and are removed by histone deacetylases (HDACs).  These marks are read by bromodomain and extra-terminal (BET) proteins, whose targeted inhibitors are under clin. investigation.  BET and HDAC inhibitors have been demonstrated to be synergistically killing in Myc-induced murine lymphoma.  Herein, the authors combine the inhibitory activities of BET and HDAC into one mol. through structure-based design method and evaluate its function.  The majority of these synthesized compds. showed inhibitory activity against second bromodomains(BRD) of BRD4 and HDAC1.  Among them, I presented antiproliferative effects against human acute myelogenous leukemia (AML) cell lines in vitro, and I is confirmed to reduce the expression of Myc by Western blot anal.  Those results indicated that I is a potent dual BRD4/HDAC inhibitor and deserves further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkMaHqVu_LM7Vg90H21EOLACvtfcHk0lg520PpbNJc5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP&md5=c525f1a2083699dd5f6f6a262f477f24</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.07.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.07.054%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DStructure-Based%2520Design%252C%2520Synthesis%2520and%2520in%2520Vitro%2520Antiproliferative%2520Effects%2520Studies%2520of%2520Novel%2520Dual%2520Brd4%252FHdac%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D4051%26epage%3D4055%26doi%3D10.1016%2Fj.bmcl.2017.07.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (Brd4) Inhibitor That Induces Amp-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9990</span>– <span class="NLM_lpage">10012</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00275</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00275" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9990-10012&author=L.+Ouyangauthor=L.+Zhangauthor=J.+Liuauthor=L.+Fuauthor=D.+Yaoauthor=Y.+Zhaoauthor=S.+Zhangauthor=G.+Wangauthor=G.+Heauthor=B.+Liu&title=Discovery+of+a+Small-Molecule+Bromodomain-Containing+Protein+4+%28Brd4%29+Inhibitor+That+Induces+Amp-Activated+Protein+Kinase-Modulated+Autophagy-Associated+Cell+Death+in+Breast+Cancer&doi=10.1021%2Facs.jmedchem.7b00275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (BRD4) Inhibitor That Induces AMP-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span></div><div class="casAuthors">Ouyang, Liang; Zhang, Lan; Liu, Jie; Fu, Leilei; Yao, Dahong; Zhao, Yuqian; Zhang, Shouyue; Wang, Guan; He, Gu; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9990-10012</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Upon the basis of The Cancer Genome Atlas (TCGA) data set, the authors identified that several autophagy-related proteins such as AMP-activated protein kinase (AMPK) were remarkably downregulated in breast cancer.  Combined with coimmunopptn. assay, the authors demonstrated that BRD4 might interact with AMPK.  After analyses of the pharmacophore and WPF interaction optimization, the authors designed a small-mol. inhibitor of BRD4, 9f (FL-411 (2-(4-hydroxy-3,5-dimethylphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4',3':4,5]thieno[2,3-d]pyrimidin-4(3H)-one)) which was validated by cocrystal structure with BD1 of BRD4.  Subsequently, 9f was discovered to induce ATG5-dependent autophagy-assocd. cell death (ACD) by blocking BRD4-AMPK interaction and thus activating AMPK-mTOR-ULK1-modulated autophagic pathway in breast cancer cells.  Interestingly, the iTRAQ-based proteomics analyses revealed that 9f induced ACD pathways involved in HMGB1, VDAC1/2, and eEF2.  Moreover, 9f displayed a therapeutic potential on both breast cancer xenograft mouse and zebrafish models.  Together, these results demonstrate that a novel small-mol. inhibitor of BRD4 induces BRD4-AMPK-modulated ACD in breast cancer, which may provide a candidate drug for future cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Ocrq2wyy6bVg90H21EOLACvtfcHk0lgag-xfjPIiEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF&md5=ac15f7e108511ea207f2c2fcd299b4e5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00275%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Bromodomain-Containing%2520Protein%25204%2520%2528Brd4%2529%2520Inhibitor%2520That%2520Induces%2520Amp-Activated%2520Protein%2520Kinase-Modulated%2520Autophagy-Associated%2520Cell%2520Death%2520in%2520Breast%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9990%26epage%3D10012%26doi%3D10.1021%2Facs.jmedchem.7b00275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Small-Molecule Inhibitors of Brd4 Using Structure-Based Virtual Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8073</span>– <span class="NLM_lpage">8088</span>, <span class="refDoi"> DOI: 10.1021/jm4011302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+Novel+Small-Molecule+Inhibitors+of+Brd4+Using+Structure-Based+Virtual+Screening&doi=10.1021%2Fjm4011302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0lgag-xfjPIiEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Small-Molecule%2520Inhibitors%2520of%2520Brd4%2520Using%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088%26doi%3D10.1021%2Fjm4011302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruckman, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herns, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spector, S. A.</span></span> <span> </span><span class="NLM_article-title">Induction of Autophagy by Pi3k/Mtor and Pi3k/Mtor/Brd4 Inhibitors Suppresses Hiv-1 Replication</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">5808</span>– <span class="NLM_lpage">5820</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.002353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1074%2Fjbc.ra118.002353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=29475942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFSqu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=5808-5820&author=G.+R.+Campbellauthor=R.+S.+Bruckmanauthor=S.+D.+Hernsauthor=S.+Joshiauthor=D.+L.+Durdenauthor=S.+A.+Spector&title=Induction+of+Autophagy+by+Pi3k%2FMtor+and+Pi3k%2FMtor%2FBrd4+Inhibitors+Suppresses+Hiv-1+Replication&doi=10.1074%2Fjbc.ra118.002353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of autophagy by PI3K/MTOR and PI3K/MTOR/BRD4 inhibitors suppresses HIV-1 replication</span></div><div class="casAuthors">Campbell, Grant R.; Bruckman, Rachel S.; Herns, Shayna D.; Joshi, Shweta; Durden, Donald L.; Spector, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5808-5820</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In this study, we investigated the effects of the dual phosphatidylinositol 3-kinase/mechanistic target of rapamycin (PI3K/MTOR) inhibitor dactolisib (NVP-BEZ235), the PI3K/MTOR/bromodomain-contg. protein 4 (BRD4) inhibitor SF2523, and the bromodomain and extra terminal domain inhibitor JQ1 on the productive infection of primary macrophages with human immunodeficiency type-1 (HIV).  These inhibitors did not alter the initial susceptibility of macrophages to HIV infection.  However, dactolisib, JQ1, and SF2523 all decreased HIV replication in macrophages in a dose-dependent manner via degrdn. of intracellular HIV through autophagy.  Macrophages treated with dactolisib, JQ1, or SF2523 displayed an increase in LC3B lipidation combined with SQSTM1 degrdn. without inducing increased cell death.  LC3B-II levels were further increased in the presence of pepstatin A suggesting that these inhibitors induce autophagic flux.  RNA interference for ATG5 and ATG7 and pharmacol. inhibitors of autophagosome-lysosome fusion and of lysosomal hydrolases all blocked the inhibition of HIV.  Thus, we demonstrate that the mechanism of PI3K/MTOR and PI3K/MTOR/BRD4 inhibitor suppression of HIV requires the formation of autophagosomes, as well as their subsequent maturation into autolysosomes.  These data provide further evidence in support of a role for autophagy in the control of HIV infection and open new avenues for the use of this class of drugs in HIV therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8-D9wUfaylbVg90H21EOLACvtfcHk0lgag-xfjPIiEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFSqu74%253D&md5=06a414ed7a31e34ed7cfd9bb28b48fdd</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1074%2Fjbc.ra118.002353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.ra118.002353%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DG.%2BR.%26aulast%3DBruckman%26aufirst%3DR.%2BS.%26aulast%3DHerns%26aufirst%3DS.%2BD.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26aulast%3DSpector%26aufirst%3DS.%2BA.%26atitle%3DInduction%2520of%2520Autophagy%2520by%2520Pi3k%252FMtor%2520and%2520Pi3k%252FMtor%252FBrd4%2520Inhibitors%2520Suppresses%2520Hiv-1%2520Replication%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D5808%26epage%3D5820%26doi%3D10.1074%2Fjbc.ra118.002353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamaki, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K. M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Regulation of Autophagy and Lysosomal Function by Bromodomain Protein Brd4</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2006</span>– <span class="NLM_lpage">2007</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1364822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1080%2F15548627.2017.1364822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28933601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=2006-2007&author=J.-I.+Sakamakiauthor=K.+M.+Ryan&title=Transcriptional+Regulation+of+Autophagy+and+Lysosomal+Function+by+Bromodomain+Protein+Brd4&doi=10.1080%2F15548627.2017.1364822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation of autophagy and lysosomal function by bromodomain protein BRD4</span></div><div class="casAuthors">Sakamaki, Jun-ichi; Ryan, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2006-2007</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Macroautophagy/autophagy is an intracellular recycling system that delivers cytoplasmic organelles and materials to lysosomes for degrdn.  This process is operated by autophagy-related (ATG) genes and tightly controlled by stress-responsive signaling pathways.  Our recent study revealed that autophagy programs are transcriptionally suppressed by the BET family protein BRD4.  This repression is alleviated during nutrient deprivation through the AMPK-SIRT1 pathway.  Our findings therefore provide new insights into the regulation of autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeGupyW9sug7Vg90H21EOLACvtfcHk0lg_t1wWhRPPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbJ&md5=a5740d62bf9d4faaf0206c2e81e2d922</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1364822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1364822%26sid%3Dliteratum%253Aachs%26aulast%3DSakamaki%26aufirst%3DJ.-I.%26aulast%3DRyan%26aufirst%3DK.%2BM.%26atitle%3DTranscriptional%2520Regulation%2520of%2520Autophagy%2520and%2520Lysosomal%2520Function%2520by%2520Bromodomain%2520Protein%2520Brd4%26jtitle%3DAutophagy%26date%3D2017%26volume%3D13%26spage%3D2006%26epage%3D2007%26doi%3D10.1080%2F15548627.2017.1364822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Brd4 Is a Newly Characterized Transcriptional Regulator That Represses Autophagy and Lysosomal Function</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1801</span>– <span class="NLM_lpage">1803</span>, <span class="refDoi"> DOI: 10.1080/15548627.2017.1364334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1080%2F15548627.2017.1364334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=28837371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1801-1803&author=X.+Wenauthor=D.+J.+Klionsky&title=Brd4+Is+a+Newly+Characterized+Transcriptional+Regulator+That+Represses+Autophagy+and+Lysosomal+Function&doi=10.1080%2F15548627.2017.1364334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 is a newly characterized transcriptional regulator that represses autophagy and lysosomal function</span></div><div class="casAuthors">Wen, Xin; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1801-1803</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">As a member of the bromodomain and extraterminal (BET) family, BRD4 (bromodomain contg. 4) can bind to acetylated histones and transcription factors, and is also able to recruit various transcriptional regulators.  Previous studies have shown that BRD4 mainly plays a pos. role in cell growth and cell cycle progression.  In a recent study conducted by Sakamaki et al., the authors found that BRD4 acts as a transcriptional repressor in regulating macroautophagy/autophagy and lysosome gene expression, via binding to the histone lysine methyltransferase EHMT2/G9a.  The oncoprotein BRD4-NUTM1/NUT also helps block autophagy and lysosome functions.  A knockdown of BRD4 allows the retention of MTOR activity during starvation and significantly undermines starvation-induced cell death.  However, BRD4 repression only contributes to stimulus-dependent autophagy and aggrephagy, and is not involved in other types of selective autophagy.  Taken together, the newly reported repressive role of BRD4 in autophagy adds to our understanding of how autophagy and lysosome functions are regulated at the transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5yO5wGDB31LVg90H21EOLACvtfcHk0lg_t1wWhRPPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFCqsrbO&md5=39c20dc548ec31554a71f5a69b4a6361</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1080%2F15548627.2017.1364334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2017.1364334%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DX.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DBrd4%2520Is%2520a%2520Newly%2520Characterized%2520Transcriptional%2520Regulator%2520That%2520Represses%2520Autophagy%2520and%2520Lysosomal%2520Function%26jtitle%3DAutophagy%26date%3D2017%26volume%3D13%26spage%3D1801%26epage%3D1803%26doi%3D10.1080%2F15548627.2017.1364334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eom, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeung, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y. H.</span></span> <span> </span><span class="NLM_article-title">Ampk-Ulk1-Mediated Autophagy Confers Resistance to Bet Inhibitor Jq1 in Acute Myeloid Leukemia Stem Cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2794</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-1903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=10.1158%2F1078-0432.ccr-16-1903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=27864418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2781-2794&author=J.+E.+Jangauthor=J.-I.+Eomauthor=H.-K.+Jeungauthor=J.-W.+Cheongauthor=J.+Y.+Leeauthor=J.+S.+Kimauthor=Y.+H.+Min&title=Ampk-Ulk1-Mediated+Autophagy+Confers+Resistance+to+Bet+Inhibitor+Jq1+in+Acute+Myeloid+Leukemia+Stem+Cells&doi=10.1158%2F1078-0432.ccr-16-1903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AMPK-ULK1-Mediated Autophagy Confers Resistance to BET Inhibitor JQ1 in Acute Myeloid Leukemia Stem Cells</span></div><div class="casAuthors">Jang, Ji Eun; Eom, Ju-In; Jeung, Hoi-Kyung; Cheong, June-Won; Lee, Jung Yeon; Kim, Jin Seok; Min, Yoo Hong</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2781-2794</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Bromodomain and extraterminal domain (BET) inhibitors are promising epigenetic agents for the treatment of various subsets of acute myeloid leukemia (AML).  However, the resistance of leukemia stem cells (LSC) to BET inhibitors remains a major challenge.  In this study, we evaluated the mechanisms underlying LSC resistance to the BET inhibitor JQ1.  Exptl. Design: We evaluated the levels of apoptosis and autophagy induced by JQ1 in LSC-like leukemia cell lines and primary CD34+CD38- leukemic blasts obtained from AML cases with normal karyotype without recurrent mutations.  Results: JQ1 effectively induced apoptosis in a concn.-dependent manner in JQ1-sensitive AML cells.  However, in JQ1-resistant AML LSCs, JQ1 induced little apoptosis and led to upregulation of beclin-1, increased LC3-II lipidation, formation of autophagosomes, and downregulation of p62/SQSTM1.  Inhibition of autophagy by pharmacol. inhibitors or knockdown of beclin-1 using specific siRNA enhanced JQ1-induced apoptosis in resistant cells, indicating that prosurvival autophagy occurred in these cells.  Independent of mTOR signaling, activation of the AMPK (pThr172)/ULK1 (pSer555) pathway was found to be assocd. with JQ1-induced autophagy in resistant cells.  AMPK inhibition using the pharmacol. inhibitor compd. C or by knockdown of AMPKα suppressed autophagy and promoted JQ1-induced apoptosis in AML LSCs.  Conclusions: These findings revealed that prosurvival autophagy was one of the mechanisms involved in the resistance AML LSCs to JQ1.  Targeting the AMPK/ULK1 pathway or inhibition of autophagy could be an effective therapeutic strategy for combating resistance to BET inhibitors in AML and other types of cancer.  Clin Cancer Res; 23(11); 2781-94. 2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHOzsCo6fOvLVg90H21EOLACvtfcHk0ljyh4_DOXyt7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D&md5=f15df13e0addda969161b31973857176</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-16-1903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-16-1903%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BE.%26aulast%3DEom%26aufirst%3DJ.-I.%26aulast%3DJeung%26aufirst%3DH.-K.%26aulast%3DCheong%26aufirst%3DJ.-W.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DMin%26aufirst%3DY.%2BH.%26atitle%3DAmpk-Ulk1-Mediated%2520Autophagy%2520Confers%2520Resistance%2520to%2520Bet%2520Inhibitor%2520Jq1%2520in%2520Acute%2520Myeloid%2520Leukemia%2520Stem%2520Cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D2781%26epage%3D2794%26doi%3D10.1158%2F1078-0432.ccr-16-1903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZW1','PDB','4ZW1'); return false;">PDB: 4ZW1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM','PDB','2YEM'); return false;">PDB: 2YEM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ','PDB','4LXZ'); return false;">PDB: 4LXZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WGI','PDB','5WGI'); return false;">PDB: 5WGI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C0Z','PDB','3C0Z'); return false;">PDB: 3C0Z</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i98"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02178">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42010"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b02178?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02178</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_001.csv">CSV</a>)</p></li><li><p class="inline">Additional figures containing additional figures related biological studies; NMR spectra of final compounds and HPLC chromatogram of final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_001.csv">jm9b02178_si_001.csv (3.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02178/suppl_file/jm9b02178_si_002.pdf">jm9b02178_si_002.pdf (8.27 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-7%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b02178%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b02178" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                14MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0b97a7b3232c1","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
