
<html lang="en"     class="pb-page"  data-request-id="8e668d45-8470-4d67-95f6-393b2003dfc9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-13;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00449;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of 4,6- and 5,7-Disubstituted Isoquinoline Derivatives as a Novel Class of Protein Kinase C ζ Inhibitors with Fragment-Merging Strategy" /></meta><meta name="dc.Creator" content="Masakazu  Atobe" /></meta><meta name="dc.Creator" content="Takayuki  Serizawa" /></meta><meta name="dc.Creator" content="Natsumi  Yamakawa" /></meta><meta name="dc.Creator" content="Kenichiro  Takaba" /></meta><meta name="dc.Creator" content="Yukiko  Nagano" /></meta><meta name="dc.Creator" content="Toshiaki  Yamaura" /></meta><meta name="dc.Creator" content="Eiichi  Tanaka" /></meta><meta name="dc.Creator" content="Atsutoshi  Tazumi" /></meta><meta name="dc.Creator" content="Shino  Bito" /></meta><meta name="dc.Creator" content="Masashi  Ishiguro" /></meta><meta name="dc.Creator" content="Masashi  Kawanishi" /></meta><meta name="dc.Description" content="Two chemical series of novel protein kinase C ζ (PKCζ) inhibitors, 4,6-disubstituted and 5,7-disubstituted isoquinolines, were rapidly identified using our fragment merging strategy. This methodolo..." /></meta><meta name="Description" content="Two chemical series of novel protein kinase C ζ (PKCζ) inhibitors, 4,6-disubstituted and 5,7-disubstituted isoquinolines, were rapidly identified using our fragment merging strategy. This methodolo..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 17, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00449" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00449" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00449" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00449" /></link>
        
    
    

<title>Discovery of 4,6- and 5,7-Disubstituted Isoquinoline Derivatives as a Novel Class of Protein Kinase C ζ Inhibitors with Fragment-Merging Strategy | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00449" /></meta><meta property="og:title" content="Discovery of 4,6- and 5,7-Disubstituted Isoquinoline Derivatives as a Novel Class of Protein Kinase C ζ Inhibitors with Fragment-Merging Strategy" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0024.jpeg" /></meta><meta property="og:description" content="Two chemical series of novel protein kinase C ζ (PKCζ) inhibitors, 4,6-disubstituted and 5,7-disubstituted isoquinolines, were rapidly identified using our fragment merging strategy. This methodology involves biochemical screening of a high concentration of a monosubstituted isoquinoline fragment library, then merging hit isoquinoline fragments into a single compound. Our strategy can be applied to the discovery of other challenging kinase inhibitors without protein–ligand structural information. Furthermore, our optimization effort identified the highly potent and orally available 5,7-isoquinoline 37 from the second chemical series. Compound 37 showed good efficacy in a mouse collagen-induced arthritis model. The in vivo studies suggest that PKCζ inhibition is a novel target for rheumatoid arthritis (RA) and that 5,7-disubstituted isoquinoline 37 has the potential to elucidate the biological consequences of PKCζ inhibition, specifically in terms of therapeutic intervention for RA." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00449"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00449">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00449&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00449&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00449&amp;href=/doi/10.1021/acs.jmedchem.0c00449" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 7143-7162</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00422" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00450" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of 4,6- and 5,7-Disubstituted Isoquinoline Derivatives as a Novel Class of Protein Kinase C ζ Inhibitors with Fragment-Merging Strategy</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Masakazu Atobe</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masakazu Atobe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +81-558-76-8493. E mail: <a href="/cdn-cgi/l/email-protection#2140554e43440f4c43614e4c0f40524049480c4a405244480f424e0f4b51"><span class="__cf_email__" data-cfemail="f79683989592d99a95b7989ad99684969f9eda9c9684929ed99498d99d87">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masakazu++Atobe">Masakazu Atobe</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7314-0237" title="Orcid link">http://orcid.org/0000-0002-7314-0237</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Takayuki Serizawa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takayuki Serizawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takayuki++Serizawa">Takayuki Serizawa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Natsumi Yamakawa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natsumi Yamakawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natsumi++Yamakawa">Natsumi Yamakawa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kenichiro Takaba</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenichiro Takaba</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenichiro++Takaba">Kenichiro Takaba</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yukiko Nagano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yukiko Nagano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research Coordination, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yukiko++Nagano">Yukiko Nagano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Toshiaki Yamaura</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Toshiaki Yamaura</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Drug Discovery, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Toshiaki++Yamaura">Toshiaki Yamaura</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eiichi Tanaka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eiichi Tanaka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Drug Discovery, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eiichi++Tanaka">Eiichi Tanaka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Atsutoshi Tazumi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Atsutoshi Tazumi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Pharmacology, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Atsutoshi++Tazumi">Atsutoshi Tazumi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shino Bito</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shino Bito</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Safety Assessment & ADME, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shino++Bito">Shino Bito</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masashi Ishiguro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masashi Ishiguro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Safety Assessment & ADME, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masashi++Ishiguro">Masashi Ishiguro</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Masashi Kawanishi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masashi Kawanishi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masashi++Kawanishi">Masashi Kawanishi</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00449&amp;href=/doi/10.1021%2Facs.jmedchem.0c00449" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 7143–7162</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 17, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 March 2020</li><li><span class="item_label"><b>Published</b> online</span>17 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00449" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00449</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7143%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMasakazu%2BAtobe%252C%2BTakayuki%2BSerizawa%252C%2BNatsumi%2BYamakawa%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D13%26contentID%3Dacs.jmedchem.0c00449%26title%3DDiscovery%2Bof%2B4%252C6-%2Band%2B5%252C7-Disubstituted%2BIsoquinoline%2BDerivatives%2Bas%2Ba%2BNovel%2BClass%2Bof%2BProtein%2BKinase%2BC%2B%25CE%25B6%2BInhibitors%2Bwith%2BFragment-Merging%2BStrategy%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7162%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00449"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1396</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00449" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of 4,6- and 5,7-Disubstituted Isoquinoline Derivatives as a Novel Class of Protein Kinase C ζ Inhibitors with Fragment-Merging Strategy&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Masakazu&quot;,&quot;last_name&quot;:&quot;Atobe&quot;},{&quot;first_name&quot;:&quot;Takayuki&quot;,&quot;last_name&quot;:&quot;Serizawa&quot;},{&quot;first_name&quot;:&quot;Natsumi&quot;,&quot;last_name&quot;:&quot;Yamakawa&quot;},{&quot;first_name&quot;:&quot;Kenichiro&quot;,&quot;last_name&quot;:&quot;Takaba&quot;},{&quot;first_name&quot;:&quot;Yukiko&quot;,&quot;last_name&quot;:&quot;Nagano&quot;},{&quot;first_name&quot;:&quot;Toshiaki&quot;,&quot;last_name&quot;:&quot;Yamaura&quot;},{&quot;first_name&quot;:&quot;Eiichi&quot;,&quot;last_name&quot;:&quot;Tanaka&quot;},{&quot;first_name&quot;:&quot;Atsutoshi&quot;,&quot;last_name&quot;:&quot;Tazumi&quot;},{&quot;first_name&quot;:&quot;Shino&quot;,&quot;last_name&quot;:&quot;Bito&quot;},{&quot;first_name&quot;:&quot;Masashi&quot;,&quot;last_name&quot;:&quot;Ishiguro&quot;},{&quot;first_name&quot;:&quot;Masashi&quot;,&quot;last_name&quot;:&quot;Kawanishi&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;7143-7162&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00449&quot;},&quot;abstract&quot;:&quot;Two chemical series of novel protein kinase C ζ (PKCζ) inhibitors, 4,6-disubstituted and 5,7-disubstituted isoquinolines, were rapidly identified using our fragment merging strategy. This methodology involves biochemical screening of a high concentration of a monosubstituted isoquinoline fragment library, then merging hit isoquinoline fragments into a single compound. Our strategy can be applied to the discovery of other challenging kinase inhibitors without protein–ligand structural information. Furthermore, our optimization effort identified the highly potent and orally available 5,7-isoquinoline 37 from the second chemical series. Compound 37 showed good efficacy in a mouse collagen-induced arthritis model. The in vivo studies suggest that PKCζ inhibition is a novel target for rheumatoid arthritis (RA) and that 5,7-disubstituted isoquinoline 37 has the potential to elucidate the biological consequences of PKCζ inhibition, specifically in terms of therapeutic intervention for RA.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00449&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00449" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00449&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00449" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00449&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00449" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00449&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00449&amp;href=/doi/10.1021/acs.jmedchem.0c00449" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00449" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00449" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00449%26sid%3Dliteratum%253Aachs%26pmid%3D32551607%26genre%3Darticle%26aulast%3DAtobe%26date%3D2020%26atitle%3DDiscovery%2Bof%2B4%252C6-%2Band%2B5%252C7-Disubstituted%2BIsoquinoline%2BDerivatives%2Bas%2Ba%2BNovel%2BClass%2Bof%2BProtein%2BKinase%2BC%2B%25CE%25B6%2BInhibitors%2Bwith%2BFragment-Merging%2BStrategy%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D13%26spage%3D7143%26epage%3D7162%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/jmcmar.2020.63.issue-13/20200709/jmcmar.2020.63.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0024.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Two chemical series of novel protein kinase C ζ (PKCζ) inhibitors, 4,6-disubstituted and 5,7-disubstituted isoquinolines, were rapidly identified using our fragment merging strategy. This methodology involves biochemical screening of a high concentration of a monosubstituted isoquinoline fragment library, then merging hit isoquinoline fragments into a single compound. Our strategy can be applied to the discovery of other challenging kinase inhibitors without protein–ligand structural information. Furthermore, our optimization effort identified the highly potent and orally available 5,7-isoquinoline <b>37</b> from the second chemical series. Compound <b>37</b> showed good efficacy in a mouse collagen-induced arthritis model. The in vivo studies suggest that PKCζ inhibition is a novel target for rheumatoid arthritis (RA) and that 5,7-disubstituted isoquinoline <b>37</b> has the potential to elucidate the biological consequences of PKCζ inhibition, specifically in terms of therapeutic intervention for RA.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02201" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02201" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein kinase C ζ (PKCζ) is a member of the PKC family of kinases, which comprises 11 members grouped into three categories based on their molecular structure and mode of activation. The 11 PKC isoforms can be subdivided into diacylglycerol- (DAG) and calcium-dependent classical isoforms (PKCs α, βI, βII, and γ), DAG-dependent novel isoforms (PKCs δ, ε, η, and θ), and DAG- and calcium-independent atypical isoforms (PKCs λ, ι, and ζ).<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a></div><div class="NLM_p">PKCζ is involved in multiple signal transduction pathways, for example, as an intermediary in the activation of the NF-κB<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and IL-4/Stat6 pathways, and in the regulation of cell polarity.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Several studies have shown that PKCζ is an attractive target for treating numerous diseases, such as diabetes,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> proliferative vitreoretinopathy,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> chronic inflammatory skin defects,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and breast cancer.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Our interest is in orthopedics, and thus we were interested that PKCζ effectively regulates tumor necrosis factor α (TNFα)-induced inflammatory response in arthritis.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The role of PKCζ in autoimmune and chronic inflammatory disease involves coactivation of the transcriptional activity of NF-κB, generated by the phosphorylation by PKCζ of Ser311 on the RelA subunit of NF-κB.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11−14)</a> Pfizer (formerly known as Wyeth) researchers concluded that PKCζ is an atypical PKC isoform that serves as a critical link to NF-κB activation by TNF and IL-1 in chondrocytes. Another key finding was that siRNA-gene silencing of PKCζ and ι inhibits the expression of collagenase, matrix metalloproteinase (MMP)-1 and MMP-13 and inhibits cartilage destruction, via STAT3 and ERK-dependent c-fos induction.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> PKCζ thus represents a potentially important disease-associated gene responsible for arthritis cartilage destruction and is an attractive therapeutic target. Orally active small molecule inhibitors of PKCζ could provide targeted and powerful therapeutic options for the treatment of chronic arthritis by blocking a common signaling component for multiple inflammatory cytokines, as well as interrupting arthritis progression that lie downstream of NF-κB.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Currently, however, few small molecules have been disclosed exhibiting either moderate activity or low selectivity against kinases. Representative examples are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative compounds that act as PKCξ inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The natural product spheciosterol sulfate C (<b>1</b>) inhibits NF-κB activation by inhibiting PKCζ.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Pfizer researchers identified 2-(6-phenyl-1<i>H</i>-indazole-3-yl)-1<i>H</i>-benzodimidazole (<b>2</b>) as a PKCζ inhibitor that binds to the ATP binding pocket of the kinase.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Additionally, cancer research investigators identified thieno[2,3-<i>d</i>]pyrimidine (<b>3</b>) as an atypical PKC inhibitor,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and Matius et al. discovered 1,3,5-trisubstituted <b>4</b> as a potent and selective example of a class of allosteric PKCζ inhibitors.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> All of these compounds most likely act on the 3-phosphoinositide-dependent protein kinase-interacting fragment (PIF)-pocket in the kinase catalytic domain, given their strongly reduced in vitro and cellular activities toward a PIF-pocket mutant of PKCζ. Several cell-based assays confirmed that PKCζ is also targeted in the cellular environment and provided evidence that these putative PIF-pocket-directed compounds affect phosphorylation of the activation loop in the catalytic domain. Although allosteric strategies have shown excellent selectivity against other kinases, their potencies remain low in enzymatic assays and cell studies. Our aim is to identify a small molecule with high potency in cell studies for target validation. To this end, we used the systemic merging strategy of our originated target-focused fragment library to identify a type 1 inhibitor with high potency in both kinase enzyme and cell assays.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of High Concentration Hit Fragments by Biochemical Screening</h3><div class="NLM_p">Fragment-based drug discovery (FBDD) encompasses a broad range of methodologies and strategies for finding small fragment molecules with affinity against a target protein. Fragment molecules are typically low molecular weight and are selected to be more polar and soluble than larger compounds used for high-through-put screening (HTS).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The hit rate of fragment screening is typically higher than conventional HTS screening, and thus hit compounds tend to be less complex and exhibit structurally broader variation. These characteristic features can aid the interpretation of structure–activity relationship (SAR) data and accelerate their optimization.<a onclick="showRef(event, 'ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23">(21−23)</a> Typically, the binding mode of a fragment is identified from the X-ray crystal structure of the fragment bound to its target protein, enabling a rational evolution of hit fragments. However, the crystal structure of PKCζ is currently unavailable, and thus an alternative method was required to identify binders to PKCζ. Our strategy is to biochemically screen PKCζ kinase with a high concentration of our kinase-focused fragment library, comprising approximately 5000 monosubstituted isoquinoline fragment compounds. The isoquinoline structure is a kinase hinge binder<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> potentially conducive to identifying selective type 1 kinase inhibitors. An example of the isoquinoline structure binding mode with kinase is shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. The nitrogen in isoquinoline (in this case, fasudil (<b>5</b>)) mediates hydrogen bonds to the kinase hinge region and the hydrophobic part of the isoquinoline ring occupies an ATP pocket, a typical binding mode. Each kinase ATP binding pocket has its own characteristics, in particular with respect to binding the hydrophobic part of the isoquinoline ring. Thus, we designed isoquinoline fragments monosubstituted on the 1-, 3-, 4-, 5-, 6-, 7-, or 8-positions of the isoquinoline ring. We anticipated that high concentration biochemical screening of monosubstituted isoquinoline fragments would provide many hit fragments. Furthermore, if we could identify two fragments substituted on different positions, they could be merged on the same isoquinoline ring, rapidly providing a highly potent molecule without the need for structural information. We should note that similar approach was previously performed by Plexxikon′s called as scaffold-based discovery.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In this approach, they built large libraries of target-focused libraries which were screened biochemically and successfully identified a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity (Vemurafenib).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Example structure of fasudil (<b>5</b>), its binding mode (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESM">2ESM</a>), and assessment of the isoquinoline structure as fragments. Red circle, polar functionality for protein binding; gray circle, synthetically precedented vectors for fragment optimization without disrupting the likely binding pharmacophore; blue arrows, other implicit growth vectors toward chemical space to afford high activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">But to return to our strategy, an additional advantage of using isoquinoline fragments is that the 1-, 3-, 4-, 5-, 6-, 7-, and 8-carbons on the isoquinoline ring can be substituted, potentially encompassing an enormous accessible chemical space.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> As mentioned above, this characteristic also can be utilized to easily construct a monosubstituted isoquinoline library while retaining structural diversity rather than other heterocycles included two or three heteroatoms, such as pyrido[3,4-<i>d</i>]pyrimidine, pyrido[3,2-<i>c</i>]pyridazine, and so on. We constructed a fragment library of approximately 5000 monosubstituted isoquinolines and conducted biochemical screening of PKCζ kinase using a fragment library concentration of 100 μM.</div><div class="NLM_p">Analysis of all the assay results identified the hit isoquinoline fragments. The representatives are drawn in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. Moreover, all fragment structures are shown in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Tables S1 and S2</a>. We looked for structural patterns in the hit fragments. Both fragments <b>7</b> and <b>8</b> have a saturated nitrogen-containing heterocycle, exhibit good potency, and have low molecular weight (Mw) (Mw of <b>7</b>, 227.31; Mw of <b>8</b>, 214.27). Comparing with the homopiperazine structure of fasudil (<b>5</b>), the structural features of <b>7</b> and <b>8</b> might result in similar interactions with acidic areas of PKCζ kinase. In contrast, fragments <b>6</b> and <b>9</b> have an aromatic ring and show lower PKCζ inhibitory activity but their aniline groups might interact with a specific amino acid residue in PKCζ. We thus designed a merged structure, with one saturated heterocycle and one aromatic ring on the isoquinoline ring, representative of merging fragments <b>6</b> and <b>8</b>, or fragments <b>7</b> and <b>9</b>. In practice, merging fragments <b>6</b> and <b>7</b> or fragments <b>8</b> and <b>9</b> could be synthetically challenging due to the close proximity of the two substituents and thus we designed and synthesized 4,6-disubstituted isoquinoline, the merged molecule of fragments <b>6</b> and <b>8</b>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Hit fragments from the high concentration biochemical assay with PKCζ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Optimization of 4,6-Disubstituted Isoquinoline</h3><div class="NLM_p">4,6-Disubstituted isoquinoline <b>10</b> was synthesized, then evaluated by biochemical assays (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The IC<sub>50</sub> of <b>10</b> indicated a significant increase in potency (0.033 μM) compared to fragment <b>8</b> (a 424-fold increase), prompting us to further evolve the 4- and 6-disubstituted isoquinoline compound from starting molecule <b>10</b>.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Result of merging fragments <b>6</b> and <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We investigated the SAR of aryl groups on the 4-position by preparing a subset library of the 4,6-isoquinoline derivatives <b>11</b>–<b>21</b>; the SAR results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The 4-amino-phenyl group substitution (<b>10</b>) showed good potency, whereas the 3-hydroxy (<b>11</b>) substitution exhibited less potency. We then replaced 4-Ar with other heterocycles and found that 2-thienyl (<b>12</b>), 3-thienyl (<b>13</b>), 3-furyl (<b>14</b>), 2-thiazolyl (<b>15</b>), and 2-pyridyl (<b>16</b>) exhibited decreased potency. Replacement of the Ar with bicyclic-heterocycles (<b>17</b>–<b>20</b>) did not improve potency compared to (<b>7</b>), whereas substitution with the 1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-5-yl group (<b>21</b>) achieved greatly improved potency, and thus we optimized the 6-position on the isoquinoline ring.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Ar Groups on the 4-Position of Isoquinoline</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0016.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">Ar</th><th class="colsep0 rowsep0" align="center" char=".">PKCζ IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">4-amino-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.033</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">3-hydroxy-phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.142</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">2-thienyl</td><td class="colsep0 rowsep0" align="char" char=".">0.147</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">3-thienyl</td><td class="colsep0 rowsep0" align="char" char=".">0.141</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">3-furyl</td><td class="colsep0 rowsep0" align="char" char=".">1.253</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">2-thiazolyl</td><td class="colsep0 rowsep0" align="char" char=".">3.638</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">2-pyridyl</td><td class="colsep0 rowsep0" align="char" char=".">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">3-quinolyl</td><td class="colsep0 rowsep0" align="char" char=".">1.074</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">2-naphthyl</td><td class="colsep0 rowsep0" align="char" char=".">0.356</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">2-benzoxazolyl</td><td class="colsep0 rowsep0" align="char" char=".">1.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">2-benzthiazolyl</td><td class="colsep0 rowsep0" align="char" char=".">0.631</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-5-yl</td><td class="colsep0 rowsep0" align="char" char=".">0.013</td></tr></tbody></table></div></div><div class="NLM_p">Extension of the carbon linker (<b>22</b>) and placing the nitrogen at a different position (<b>23</b>) did not improve potency. The piperidine structure (<b>24</b>) showed good potency, but again, extension of the linker (<b>25</b>) was not promising. We concluded that (<i>S</i>)-pyrrolidinol (<b>10</b>) is the optimal 6-position substituent and thus we further profiled <b>21</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the R<sup>1</sup> Group on the 6-Position of Isoquinoline</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0017.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0018.gif" alt="" id="fx3" /></img><div></div></div><div class="NLM_p">The kinase selectivity of <b>21</b> was evaluated using a Carnabio Science profiling kit. The results suggested excellent selectivity against 216 kinases, each present at 100 nM (for detailed results, see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Table S3</a>). In vivo mouse PK studies of <b>21</b> showed no significant detrimental biological effects (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. In Vivo Mouse PK Studies (2 mg/10 mL/kg, po <b>21</b>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">property</th><th class="colsep0 rowsep0" align="center" char=".">values</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (nM)</td><td class="colsep0 rowsep0" align="char" char=".">102</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC last (nM·h)</td><td class="colsep0 rowsep0" align="char" char=".">165.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDCK (cm/M)</td><td class="colsep0 rowsep0" align="char" char=".">11.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein binding (%)</td><td class="colsep0 rowsep0" align="char" char=".">83.1</td></tr></tbody></table></div></div><div class="NLM_p last">Finally, we conducted a cell assay for the production of TNFα using lipopolysaccharide (LPS)-stimulated THP-1 human acute monocytic leukemia cells and obtained an IC<sub>50</sub> of 8.0 μM, suggesting the inhibition of TNFα production. Animal arthritis model studies require more potent compounds in cell assays, and thus we investigated an alternative chemical series from our hit fragments.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Optimization of 5,7-Disubstituted Isoquinoline</h3><div class="NLM_p">FBDD can enable multiple chemical series from hit fragments, allowing the use of the same or different hit compounds. We merged fragments <b>7</b> and <b>9</b> and designed 5,7-disubstituted isoquinolines in a manner similar to 4,6-disubstituted isoquinoline <b>10</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The inhibitory activity of <b>26</b> was significantly increased (IC<sub>50</sub> = 0.5 nM) compared to <b>7</b>. However, the selectivity of <b>26</b> against PKA is not satisfied (10 nM, 20-fold of PKA/PKCζ), and in vivo pharmacokinetic studies of <b>26</b> showed a low PK profile (<i>T</i><sub>1/2</sub>, 3.4 h; <i>T</i><sub>max</sub>, 2 h; <i>C</i><sub>max</sub>, 30 nM; AUC, 122.18 nM·h at 2 mg/kg po) by the highly polar structure. Our hypothesis was that the alternative structure of aniline or the increase of the lipophilicity of <b>26</b> could improve the total PK profile to avoid rapid metabolism. Our optimization efforts of <b>26</b> focused next on achieving a balance between acceptable kinase selectivity and PK profile by seeking an alternative structure of aniline or increasing hydrophobicity.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Result of merging fragments <b>7</b> and <b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The aryl group on the 7-substituent of the isoquinoline ring was first optimized and studied by SAR (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). To avoid an outer charged structure, anilines at the meta or ortho positions on the phenyl group (<b>27</b> and <b>28</b>) were designed. However, both anilines led to decreased activity and worse selectivity against PKD2. Because those results were obtained. PKD2 selectivity of the synthesized compounds has been monitored by evaluating PKCζ activity. Phenol at the para position (<b>29</b>) showed good potency, but its selectivity against PKD2 was worse than <b>3</b>. An ortho phenol group (<b>30</b>) showed decreased potency and PKD2 selectivity, and a phenyl group (<b>31</b>) and acetamide (<b>32</b>) decreased potency as well. Therefore, the 4-aniline structure exhibited the best potency and selectivity against PKD2 in this SAR study. Next, we optimized the substituent on the 5-position with respect to both potency and PKD2 selectivity.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the R<sup>2</sup> Group on the 7-Position of Isoquinoline</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0019.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">PKCζ IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">PKD2 IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">PKD2/PKCζ (fold)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="center">4-NH<sub>2</sub>-phenyl-</td><td class="colsep0 rowsep0" align="char" char=".">0.0005</td><td class="colsep0 rowsep0" align="char" char=".">0.246</td><td class="colsep0 rowsep0" align="char" char=".">492</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="center">3-NH<sub>2</sub>-phenyl-</td><td class="colsep0 rowsep0" align="char" char=".">0.162</td><td class="colsep0 rowsep0" align="char" char=".">0.102</td><td class="colsep0 rowsep0" align="char" char=".">0.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="center">2-NH<sub>2</sub>-phenyl-</td><td class="colsep0 rowsep0" align="char" char=".">0.153</td><td class="colsep0 rowsep0" align="char" char=".">0.187</td><td class="colsep0 rowsep0" align="char" char=".">1.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="center">4-HO-phenyl-</td><td class="colsep0 rowsep0" align="char" char=".">0.0006</td><td class="colsep0 rowsep0" align="char" char=".">0.017</td><td class="colsep0 rowsep0" align="char" char=".">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="center">2-HO-phenyl-</td><td class="colsep0 rowsep0" align="char" char=".">0.054</td><td class="colsep0 rowsep0" align="char" char=".">0.263</td><td class="colsep0 rowsep0" align="char" char=".">4.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="center">phenyl</td><td class="colsep0 rowsep0" align="char" char=".">0.0064</td><td class="colsep0 rowsep0" align="char" char=".">0.261</td><td class="colsep0 rowsep0" align="char" char=".">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="center">4-acetyl-NH-phenyl-</td><td class="colsep0 rowsep0" align="char" char=".">0.013</td><td class="colsep0 rowsep0" align="char" char=".">0.762</td><td class="colsep0 rowsep0" align="char" char=".">59</td></tr></tbody></table></div></div><div class="NLM_p">A summary of the SAR results on the 5-position optimization studies is shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. We identified the stereochemistry of eutomer (<i>S</i>)-isomer <b>33</b>, which showed higher potency than (<i>R</i>)-isomer <b>34</b>. Next, we investigated the decomposed ring structure of amine <b>35</b> and confirmed the inevitability of the rigid structure for retaining its potency. The potency of the 7-membered ring compound (<b>36</b>) retained the potency of <b>3</b> but exhibited improved PKD2 selectivity and PK profile (<i>T</i><sub>1/2</sub>, 1.1 h; <i>T</i><sub>max</sub>, 0.25 h; <i>C</i><sub>max</sub>, 134 nM; AUC, 384.6 nM·h at 2 mg/kg po) by expanding the ring size to improve polarity compared to <b>3</b>, indicating that an azepane ring is optimal on the 5-position of isoquinoline. However, there was still the issue of the selectivity of PKA on <b>36</b> (10.4-fold of PKCζ). Thus, the further optimization of <b>36</b> was required.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of the R<sup>2</sup> Group on the 5-Position of Isoquinoline</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0020.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0021.gif" alt="" id="fx6" /></img><div></div></div><div class="NLM_p">As mentioned above, the azepane moiety is the important structure for the selectivity of PKD2 and PK profile. Therefore, the 7-aryl group was optimized while retaining the 5-azepane-isoquinoline structure, and the results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Introduction of a fluorine atom on the phenyl group next to isoquinoline (<b>37</b>) improved selectivity against PKA and PKCζ activity was unchanged. Difluorine substituent scanning showed that 2,3-difluorine (<b>38</b>) and 2,6-difluorine (<b>39</b>) had potencies similar to <b>37</b> but decreased selectivities against other kinases compared to monofluorine substituents. The phenyl group was revisited and <b>40</b> showed moderate activity, but introduction of the 2,6-difluoro substituent (<b>41</b>) was not effective. The selectivity of 2-aminopyridine (<b>42</b>) against PKA was improved, and potency toward PKCζ was maintained. 2-Amino-6-fluoropyridine (<b>43</b>) and 2-amino-4-fluoropyridine (<b>44</b>) showed only moderate PKCζ inhibitory activity but excellent selectivity against PKA. In comparison with <b>37</b>, <b>44</b> does not satisfy the requirements for PKCζ inhibitory activity and thus we further profiled <b>37</b>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Final Optimization Results</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0022.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0023.gif" alt="" id="fx8" /></img><div></div></div><div class="NLM_p">Next, we used <b>37</b> in THP-1/LPS TNFα cell assays (HTRF) and obtained an IC<sub>50</sub> of 0.045 μM for the inhibition of TNFα production. The kinase selectivity of <b>37</b> was evaluated in duplicate using a ProQuinase profiling kit at one concentration against 368 protein kinases and 13 lipid kinases. The results showed no other significant inhibition, except for 50% inhibition of proto-oncogene tyrosine-protein kinase (FYN) and 49% inhibition of tropomyosin receptor kinase B (TrkB) (for detailed results, see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Table S4</a>). In vivo mouse PK studies of <b>37</b> showed good PK profile and no significant detrimental biological effects (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). In retrospect, we suspect that the reason for this is an improvement in the PK profile. We thus chose <b>37</b> as a candidate as an in vivo mouse collagen-induced arthritis (CIA) model compound.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. In Vivo Mouse PK Studies (2 mg/10 mL/kg po and 1 mg/10 mL/kg, iv <b>37</b>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">property</th><th class="colsep0 rowsep0" align="center" char=".">values</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="char" char=".">1.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (nM)</td><td class="colsep0 rowsep0" align="char" char=".">261</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC last (nM·h)</td><td class="colsep0 rowsep0" align="char" char=".">395.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Vd (L/kg)</td><td class="colsep0 rowsep0" align="char" char=".">6.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL (L/h/kg)</td><td class="colsep0 rowsep0" align="char" char=".">3.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein binding (%)</td><td class="colsep0 rowsep0" align="char" char=".">92.5</td></tr></tbody></table></div></div><div class="NLM_p">We evaluated the anti-inflammatory effect of <b>37</b> by administering it to CIA mice, using BMS 345541,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> an inhibitor of nuclear factor kappa-B kinase subunit (IKK) α, and β inhibitor as positive controls in the CIA model. The administration of 3, 10, or 30 mg/kg of <b>37</b> dose-dependently ameliorated CIA progression, with reduced joint swelling in a dose-dependent manner, whereas the vehicle control mice showed severe swelling after booster immunization (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). Similar results were observed with 60 mg/kg of BMS 345541, suggesting that PKCζ inhibition is a novel therapeutic target of RA and that 5,7-disubstituted isoquinoline <b>37</b> has the potential to elucidate the biological consequences of PKCζ inhibition, specifically in terms of therapeutic intervention for RA. Additionally, mode of action was confirmed with the phosphorylation of p65Ser311 and IκB with Western blot. The p65 is a key factor for the phosphorylation of RelA because of the possible to form the complex of RelA/p65 and IκB is an upstream of PKCζ signal cascade.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref29'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref29">(11−14,29)</a> The results of Western blot showed compound <b>37</b> and BMS 345541 inhibited the phosphorylation on this site, and it agrees with in vivo results (for detailed results, see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Figures S1 and S2</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Changes in limb score following the administration of <b>37</b> to CIA model mice (3, 10, and 30 mg/kg, b.i.d., po).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0007.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Assessment of paw thickness on limb score following the administration of <b>37</b> (3, 10, and 30 mg/kg, b.i.d., po) or BMS345541 (60 mg/kg, b.i.d., po) to CIA model mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Homology Model Studies of Compounds <b>10</b> and <b>26</b> with PKCζ</h3><div class="NLM_p">Because the crystal structure of PKCζ is unknown, homology modeling was conducted to reveal the binding mode of the key compounds <b>10</b> and <b>26</b> with PKCζ. The crystal structure of PKCι (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DC2">4DC2</a>)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> was used as a template to build 100 PKCζ homology models, and docking studies were performed with the representative homology model (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for details). The docking poses were carefully inspected and suggested two possible binding modes (poses A and B) for each compound. These poses differed in the binding of the isoquinoline core (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Three distinct trials of 30 ns molecular dynamics (MD) simulations were conducted for each docking pose to assess the binding mode.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> MD trajectories of the last 20 ns were used to calculate the root-mean-square deviation (RMSD) of the ligand heavy atoms, which were measured from the averaged structure of the merged MD simulations (20 ns ×3 trials), as shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. The RMSD profiles of compound <b>10</b> showed different behavior between the two binding modes, and the RMSD value of pose A was lower than that of pose B. Furthermore, similar RMSD profiles were obtained for pose A, whereas pose B exhibited different profiles between the three simulation trials, suggesting that pose A is more stable than pose B. In contrast, the RMSD profiles for poses A and B of compound <b>26</b> were similar, but pose B exhibited slightly lower RMSD values than pose A, suggesting that pose B may be more stable. Interestingly, SAR studies of 4,6-isoquinoline showed that the bicyclic aryl groups on the 4-position had relatively high activity, whereas the bicyclic aryl groups on the 7-position of 5,7-isoquinoline were not acceptable. These results indicated that the 4-aniline of 4,6-isoquinoline and the 7-aniline of 5,7-isoquinoline are placed on different positions, and the 4-aniline of 4,6-isoquinoline in particular was more extended and flexible than the 7-aniline of 5,7-isoquinoline. We concluded that pose A is the most probable binding pose for both compounds <b>10</b> and <b>26</b> based on the in silico and SAR results. The representative binding modes of compounds <b>10</b> (yellow) and <b>26</b> (magenta) are shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> and indicate that the isoquinoline core of both compounds binds very similarly.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0008.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Predicted binding mode of compound <b>10</b> and <b>26</b> obtained by the docking study using the homology model of PKCζ. The black dashed lines indicate the hydrogen bond with the hinge residue V333.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0009.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ligand RMSD profiles of compound <b>10</b> and compound <b>26</b> measured from the averaged structure of the merged MD simulations. Trajectories from the last 20 ns were used to calculate the average structure and the RMSD profile. Three individual trials are shown for each binding pose.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0010.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Representative binding models of compound <b>10</b> (yellow) and <b>26</b> (magenta) regarding the in silico and SAR results. The black dashed lines indicate the hydrogen bond with the hinge residue V333.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we performed a fragment molecular orbital (FMO) analysis<a onclick="showRef(event, 'ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35">(32−35)</a> to better understand the nature of the protein–ligand interactions. This is a quantum mechanical computational approach in which interfragment interaction energy (IFIE) analysis<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> allows examination of protein–ligand interactions with quantum effects at reasonable computational cost. Thus, in pair interaction energy decomposition analysis (PIEDA),<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> the interaction energy can be deconstructed into four energy terms: electrostatic, exchange repulsion, dispersion, and charge transfer with higher-order mixed terms. These four terms provide detailed information regarding the protein–ligand interactions. For example, the FMO study suggested the formation of hydrogen bonds between the nitrogen at position 2 of the isoquinoline core and the backbone −NH of the hinge residue V333 in both compounds <b>10</b> and <b>26</b>, as expected (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Figure S3</a>). Furthermore, dispersion interactions (∼ −3.8 kcal/mol), namely CH/π interaction, were observed between the isoquinoline core and L383. We also observed an unexpected dispersion interaction (−6.2 ∼ −4.7 kcal/mol) between the isoquinoline core and Y332, which might be a horizontally displaced π–π interaction from a geometric perspective. These observations suggest that the isoquinoline core is a favorable hinge binder and can interact with the hinge via electrostatic and dispersion interactions. The amine parts of the pyrrolidine and piperidine of compounds <b>10</b> and <b>26</b> are situated in similar positions and form strong electrostatic interactions with D380. The FMO calculations were performed under in vacuo conditions. In aqueous solvent, the desolvation energy upon protein–ligand binding would likely compensate to some extent for the electrostatic energies, and thus the electrostatic interaction energy shown in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Figure S3</a>, does not fully contribute to the actual binding free energy.</div><div class="NLM_p last">The 4-aniline moiety of compound <b>10</b> extended into the solvent region and showed electrostatic interaction with D337 mediated by a bridging water molecule (not present in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). MD simulation analysis also suggested that the 4-aniline moiety can locate close to D337 and directly form a hydrogen bond. Both cases are reasonable because inhibitor activity decreased 10-fold when the 4-aniline moiety was replaced with a benzene group, due to the lack of interaction with D337 (data not shown). In contrast, the 4-aniline moiety of compound <b>26</b> is extended into the c-helix region, thereby gaining additional interactions with F304, Y393, and Y395. These interactions are absent in compound <b>10</b>, which may explain why the enzyme inhibitory activity of compound <b>26</b> (0.5 nM) was better than that for compound <b>10</b> (33 nM) and compound <b>26</b> (0.5 nM) (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">Figure S3</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41251" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41251" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of <b>10</b>–<b>21</b> is summarized in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Starting from commercially available 4-bromo-6-fluoro isoquinoline <b>45</b>, etherification of the 6-fluoro position with alcohol and NaH gave the desired product <b>46</b> in good yield. Suzuki coupling on 4-bromo isoquinoline with dicyclohexyl(2′,6′-dimethoxy-[1,1′-biphenyl]-2-yl)phosphine (SPHOS)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and boronate <b>47a</b>/boronic acids <b>47b</b>–<b>h</b>, followed by deprotection of the Boc group with (trifluoroacetic acid) TFA, gave 4,6-disubstituted isoquinolines <b>10</b>–<b>14</b> and <b>19</b>–<b>21</b>. Stille coupling with <b>48a</b> and <b>48b</b> and deprotection of the Boc group gave <b>15</b> and <b>16</b>. Replacing 5-azaindole boronic ester with C–H insertion with palladium catalyst<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and deprotection afforded <b>17</b> and <b>18</b>.</div><figure id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0011.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4,6-Isoquinoline Derivatives <b>10</b>–<b>21</b><sup>s</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>s</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl (<i>S</i>)-3-hydroxypyrrolidine-1-carboxylate, NaH, DMF, 80 °C, o.n. (78%); (b) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b>–<b>h</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 110 °C, o.n.; (c) 35% HCl<sub>(aq)</sub>, MeOH (2 steps, <b>10</b>, 52%; <b>11</b>, 60%; <b>12</b>, 52%; <b>13</b>, 62%; <b>14</b>, 56%; <b>15</b>, 71%; <b>16</b>, 63%; <b>17</b>, 72%; <b>18</b>, 78%; <b>19</b>, 70%; <b>20</b>, 36%; <b>21</b>, 72%); (d) Pd(PPh<sub>3</sub>)<sub>4</sub>, LiCl, <b>48a</b>,<b>b</b>, toluene, 110 °C, 18 h; (e) Pd(OAc)<sub>2</sub>, CuBr, <b>49a</b>,<b>b</b>, <sup><i>t</i></sup>Bu<sub>3</sub>P, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 150 °C, 18 h.</p></p></figure><div class="NLM_p">A summary of the synthesis of <b>22</b>–<b>25</b> is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The etherification of <b>45</b> with <b>50a</b>–<b>d</b> gave <b>51a</b>–<b>d</b>, and subsequent Suzuki coupling and deprotection provided <b>22</b>–<b>25</b> in good yield.</div><figure id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0012.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4,6-Isoquinoline Derivatives <b>22</b>–<b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>50a</b>–<b>d</b>, NaH, DMF, 80 °C, o.n. (78%); (b) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 110 °C, o.n.; (c) 35% HCl<sub>(aq)</sub>, MeOH (3 steps, <b>22</b>: 7%, <b>23</b>: 16%, <b>24</b>: 9%, <b>25</b>: 10%).</p></p></figure><div class="NLM_p">The synthesis of <b>26</b>–<b>32</b> is summarized in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Starting from commercially available 5-amino-7-chloroisoquinoline <b>52</b>, reductive amination afforded 3-amino piperidine. Subsequent Suzuki coupling<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and deprotection of the Boc group gave <b>53</b> in moderate yield. Using the same protocol, other aldehydes and ketones, and the amino-ethyl group or azepane structure, were introduced in place of 3-carbonyl-piperidine to provide <b>26</b>–<b>32</b>.</div><figure id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0013.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>26</b>–<b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl 3-oxopiperidine-1-carboxylate, NaBH(OAc)<sub>3</sub>, AcOH, rt (99%); (b) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b> or <b>47i</b>–<b>n</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 100 °C; (c) 35% HCl<sub>(aq)</sub>, MeOH (2 steps, <b>26</b>, 52%; <b>27</b>, 14%; <b>28</b>, 84%; <b>29</b>, 29%; <b>30</b>, 53%; <b>31</b>, 71%; <b>32</b>, 74%).</p></p></figure><div class="NLM_p">The synthesis of <b>33</b>–<b>36</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) involved an initial Suzuki coupling with <b>54</b> to afford <b>55</b> as the common intermediate, followed by reductive amination with <i>tert</i>-butyl (2-oxoethyl)-carbamate-carboxylate, then TFA, to afford <b>35</b> in good yield. Compounds <b>10</b>, <b>11</b>, and <b>13</b> that were synthesized using Sandmeyer conditions with <b>55</b> gave iodide derivative <b>56</b> in moderate yield. <i>N</i>-Arylation of <b>56</b> with Buchwald ligand 2-dicyclohexylphosphino-2′,4′,6′-triisopropylbiphenyl (XPHOS)<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> or 2-dicyclohexylphosphino-2′-(<i>N</i>,<i>N</i>-dimethylamino)biphenyl (DavePHOS),<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> followed by TFA, provided <b>33</b>, <b>34</b>, and <b>36</b>.</div><figure id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0014.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>33</b>–<b>36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 100 °C (56%); (b) <i>tert</i>-butyl (2-oxoethyl)-carbamate-carboxylate, NaBH(OAc)<sub>3</sub>, AcOH, rt; (c) 35% HCl<sub>(aq)</sub>, MeOH (2 steps, <b>33</b>, 28%; <b>34</b>, 17%; <b>35</b>, 33%; <b>36</b>, 19%); (d) <sup><i>t</i></sup>BuONO, I<sub>2</sub>, MeCN, 80 °C (97%); (e) Pd<sub>2</sub>dba<sub>3</sub>, Davephos/XPHOS, <b>57a</b>–<b>c</b>, <sup><i>t</i></sup>BuONa, dioxane 110 °C.</p></p></figure><div class="NLM_p">The derivatization of azepane <b>37</b>–<b>44</b> by <i>N</i>-arylation with 5-bromo-7-hydroxy-isoquinoline, followed by triflatation, provided the common intermediate <b>59</b>. Subsequent Suzuki coupling with the corresponding boronic acid/pinacol ester afforded <b>37</b>–<b>44</b> in good yield (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0015.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>37</b>–<b>44</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd<sub>2</sub>dba<sub>3</sub>, XPHOS, <i>tert</i>-butyl (<i>S</i>)-3-aminoazepane-1-carboxylate, <sup><i>t</i></sup>BuONa, dioxane 110 °C; (b) PhNTf<sub>2</sub>, Et<sub>3</sub>N, dioxane, 80 °C (2 steps, 85%); (c) PdCl<sub>2</sub>(dppf)<sub>2</sub>, 2 M Na<sub>2</sub>CO<sub>3(aq)</sub>, <b>47m</b> or <b>47o</b>–<b>u</b>, THF, reflux; (d) 35% HCl(aq), MeOH (2 steps, <b>37</b>, 31%; <b>38</b>, 59%; <b>39</b>, 66%; <b>40</b>, 37%; <b>41</b>, 40%; <b>42</b>, 46%; <b>43</b>, 31%; <b>44</b>, 32%).</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42346" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42346" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we identified two potent and selective chemical series of PKCζ inhibitors, <b>21</b> and <b>37</b>, using our systematical FBDD strategy. This strategy uses a unique isoquinoline fragment for biochemical screening against a high concentration of compounds, followed by merging the two fragments on the isoquinoline ring. This methodology can be applied to the discovery of other challenging kinase inhibitors. Our SAR-guided optimization effort rapidly identified potent 5,7-isoquinoline inhibitor <b>37</b>. We also demonstrated the good in vivo efficacy of <b>37</b> in a CIA model. Therefore, PKCζ inhibition is a novel therapeutic target of RA and compound <b>37</b> is an advanced chemical tool with which to further explore the efficacy, safety, and tolerability of PKCζ ligands.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14405" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14405" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Chemistry Information</h3><div class="NLM_p last">NMR spectra were run at 300 MHz on JEOL Co or 400 MHz on Brukar Co. As a solvent, deuterated chloroform was used, unless specifically described otherwise. For measurement of chemical shift, tetramethylsilane (TMS) was employed as an internal standard. Chemical shifts (δ) are expressed in ppm. Data for proton spectra are reported as follows: chemical shift (multiplicity [singlet (s), doublet (d), triplet (t), and multiplet (m)], coupling constants [Hz], integration). Carbon spectra were recorded with complete proton decoupling and the chemical shifts are reported in ppm. As a mass spectrometer, quadrupole type mass spectrometer, UPLC/SQD system manufactured by Waters Company was used and ionization was carried out based on an electrospray method (ESI) for the measurement. The liquid chromatography instrument used was Acquity Ultra Performance LC system. As a separation column, Acquity UPLC BEH (C18 2.1 mm × 50 mm 1.7 μM, Waters Company) was used. With respect to the examples and the reference examples in which specific descriptions are given for LC condition, the measurements were carried out by using any one of the above-described apparatuses and in accordance with the following solvent condition. In addition, <i>m</i>/<i>z</i> indicates mass spectrum data (M + H). Compound purity was determined using HPLC and is ≥95% for all test compounds.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Preparation of Compounds <b>10</b>–<b>44</b>, <b>46</b>, <b>53</b>, <b>55</b>, <b>56</b>, and <b>59</b></h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure A: S<sub>N</sub>Ar Reaction with NaH</h4><div class="NLM_p last">NaH (2.48 g, 65.61 mmol, 9.0 equiv) was added portionwise to a stirred solution of (<i>S</i>)-1-(<i>tert</i>-butoxycarbonyl)-3-pyrrolidinol (3.36 g, 17.9 mmol, 1.5 equiv) in DMA (40 mL) at 0 °C under air. After stirring for 30 min at 0 °C, 4-bromo-6-fluoro-isoquinoline <b>45</b> was added and stirred at 80 °C for 1.5 h. The reaction mixture was quenched with added H<sub>2</sub>O (50 mL), and the mixture was extracted with CHCl<sub>3</sub> (3 × 50 mL). The combined organic extracts were dried (MgSO<sub>4</sub>) and evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Procedure B: Suzuki–Miyaura Coupling with <b>47a</b>–<b>n</b></h4><div class="NLM_p last">Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv) was added dropwise to a stirred solution of <b>46</b>, <b>51a</b>–<b>d</b>, or <b>53</b> (40 mg, 0.105 mmol, 1.0 equiv), <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47a</b> (67 mg, 0.21 mmol, 2.0 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), and K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv) in <i>n</i>-BuOH (4 mL) at rt. The resulting solution was stirred and heated at 110 °C for 18 h. After filtering out the precipitation, the mixture was evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure C: Deprotection of Boc Group</h4><div class="NLM_p last">HCl<sub>(aq)</sub> (35%, 500 μL, 5 mmol, 56 equiv) was added to a stirred solution of carbamate (44.6 mg, 0.088 mmol, 1.0 equiv) in MeOH (1.0 mL) at rt. The resulting solution was stirred at rt for 4 h. The solvent was removed in vacuo. Then, NH<sub>4</sub>OH<sub>(aq)</sub> (5 mL) was added and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organic layers were dried (MgSO<sub>4</sub>) and evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> General Procedure D: Stille Coupling with <b>48a</b> and <b>48b</b></h4><div class="NLM_p last">Pd(PPh<sub>3</sub>)<sub>4</sub> (30 mg, 0.026 mmol, 0.2 equiv) was added dropwise to a stirred solution of <b>46</b> (50 mg, 0.127 mmol, 1.0 equiv), 2-tributylstannylthiazole <b>48a</b> (80 μL, 0.254 mmol, 2.0 equiv), and LiCl (11 mg, 0.254 mmol, 2.0 equiv) in toluene (1.3 mL) at rt. The resulting solution was stirred and heated at 110 °C for 18 h. After filtering out the precipitation, the mixture was evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> General Procedure E: C–H Insertion Reaction with <b>49a</b> and <b>49b</b></h4><div class="NLM_p last">Pd(OAc)<sub>2</sub> (5.7 mg, 0.025 mmol, 0.2 equiv) was added dropwise to a stirred solution of <b>46</b> (50 mg, 0.127 mmol, 1.0 equiv), benzoxazole <b>49a</b> (23 mg, 0.191 mmol, 1.5 equiv), CuBr (15 mg, 0.102 mmol, 0.8 equiv), Cs<sub>2</sub>CO<sub>3</sub> (42 mg, 0.127 mmol, 1.0 equiv), and <sup><i>t</i></sup>Bu<sub>3</sub>P (10 mg, 0.05 mmol, 0.4 equiv) in DMF (1 mL) at rt. The resulting solution was stirred and heated at 150 °C for 18 h. After filtering out the precipitation, the mixture was evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> General Procedure F: Reductive Amination with <b>52</b> or <b>55</b></h4><div class="NLM_p last"><i>tert</i>-butyl 3-oxopiperidine-1-carboxylate or <i>tert</i>-butyl (2-oxoethyl)-carbamate-carboxylate (0.298 mmol, 2 equiv) was added dropwise to a stirred solution of amine <b>52</b> or <b>55</b> (50 mg, 0.149 mmol, 1.0 equiv) in AcOH (1.5 mL) at rt and stirred at rt for 30 min. Then NaBH(OAc)<sub>3</sub> was added to a stirred reaction mixture and stirred at RT for 18 h. The mixture was evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> General Procedure G: <i>N</i>-Arylation with Buchwald’s Catalyst with <b>56</b> or <b>58</b></h4><div class="NLM_p last">Pd<sub>2</sub>dba<sub>3</sub> (110 mg, 0.12 mmol, 0.1 equiv) was added to a stirred solution of iodide <b>56</b> or bromide <b>58</b> (1.17 mmol, 1.0 equiv), amine <b>58a</b> (250 mg, 1.174 mmol, 1.0 equiv), Davephos or XPHOS (0.468 mmol, 0.4 equiv), and <sup><i>t</i></sup>BuONa (225 mg, 2.34 mmol, 2.0 equiv) in 1,4-dioxane (12 mL) at rt. The resulting solution was stirred and heated at 110 °C for 18 h. After filtering out the precipitation, the mixture was evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> General Procedure H: Suzuki–Miyaura Coupling with <b>47m</b> or <b>47o</b>–<b>u</b></h4><div class="NLM_p last">PdCl<sub>2</sub>(dppf)<sub>2</sub> (13.8 mg, 0.017 mmol, 0.2 equiv) was added dropwise to a stirred solution of triflate <b>59</b> (39.1 mg, 0.083 mmol, 1.0 equiv), boronic acid/boronate <b>47m</b> or <b>47o</b>–<b>u</b> (0.16 mmol, 2.0 equiv), and 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (320 μL, 0.16 mmol, 2.0 equiv) in THF (1 mL) at rt. The resulting solution was refluxed for 18 h. After filtering out the precipitation, the mixture was extracted with EtOAc, washed with Brine, dried over MgSO<sub>4</sub>, and evaporated under reduced pressure to give the crude product.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>S</i>)-4-(6-(Pyrrolidin-3-yloxy)isoquinolin-4-yl)aniline (<b>10</b>)</h4><div class="NLM_p last">Using general procedures B and C, <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47a</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>10</b> (22.8 mg, 52%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (br s, 1H, N<sup>+</sup>H), 9.74 (s, 1H, Ar), 9.69 (br s, 1H, N<sup>+</sup>H), 8.61 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.53 (s, 1H, Ar), 7.75 (dd, 1H, <i>J</i> = 10.0, 1.5 Hz, Ar), 7.62 (d, 1H, <i>J</i> = 9.0 Hz, Ar), 7.35 (d, 1H, <i>J</i> = 9.0 Hz, Ar), 7.30 (m, 1H, Ar), 5.41 (br s, 1H, OCH), 3.38–3.24 (m, 1H, NCH), 3.23–3.16 (m, 1H, NCH), 3.15–3.07 (m, 1H, NCH), 3.04–2.96 (m, 1H, NCH), 2.26–2.19 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.8, 145.3, 138.9, 135.3, 134.2, 132.0, 131.5, 123.5, 122.9, 120.8, 107.0, 77.3, 69.2, 52.6, 50.0, 33.5, 30.8. MS (ESI) <i>m</i>/<i>z</i> 306 [(M + H)<sup>+</sup>], RT 0.81 min. HRMS (ESI) calcd for C<sub>19</sub>H<sub>19</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 306.1606, found 306.1600.</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>S</i>)-3-(6-(Pyrrolidin-3-yloxy)isoquinolin-4-yl)phenol (<b>11</b>)</h4><div class="NLM_p last">Using general procedures B and C, 3-hydroxyphenylboronic acid <b>47b</b> (29 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL), followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL), gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>11</b> (24.6 mg, 60%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.74 (br s, 1H, N<sup>+</sup>H), 9.71 (br s, 1H, N<sup>+</sup>H), 9.70 (s, 1H, Ar), 8.57 (d, 1H, Ar), 7.72 (dd, 1H, <i>J</i> = 5.0, 3.0 Hz, Ar), 7.43 (dd, 1H, <i>J</i> = 8.0, 8.0 Hz, Ar), 7.31 (d, 1H, <i>J</i> = 2.0 Hz, Ar), 7.06–6.99 (m, 3H, Ar), 5.38 (br s, 1H, OCH), 3.51–3.42 (m, 2H, NCH), 3.38–3.21 (m, 2H, NCH), 2.26–2.18 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.4, 158.4, 146.2, 138.7, 1356, 135.4, 133.9, 133.2, 130.7, 123.5, 122.7, 120.8, 117.1, 116.7, 106.9, 77.3, 50.1, 43.8, 30.8. MS (ESI) <i>m</i>/<i>z</i> 307 [(M + H)<sup>+</sup>], RT 0.85 min. HRMS (ESI) calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 307.1446, found 307.1438.</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>S</i>)-6-(Pyrrolidin-3-yloxy)-4-(thiophen-2-yl)isoquinoline (<b>12</b>)</h4><div class="NLM_p last">Using general procedures B and C, thiophene-2-boronic acid <b>47c</b> (27 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL), followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL), gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>12</b> (21.7 mg, 52%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.78 (br s, 1H, N<sup>+</sup>H), 9.63 (br s, 1H, Ar), 9.61 (br s, 1H, N<sup>+</sup>H), 8.65 (s, 1H, Ar), 8.53 (br d, 1H, <i>J</i> = 10.0 Hz, Ar), 7.93 (d, 1H, <i>J</i> = 5.0 Hz, Ar), 7.69 (br d, 1H, <i>J</i> = 10.0, Ar), 7.64–7.61 (m, 3H, Ar), 7.35 (dd, 1H, <i>J</i> = 5.0, Ar), 5.44 (dd, 1H, <i>J</i> = 5.0, 3.0 Hz, Ar), 3.57–3.42 (m, 2H, NCH), 3.40–3.22 (m, 2H, NCH), 2.34–2.19 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.3, 147.2, 137.9, 135.1, 130.2, 129.5, 129.0, 127.9, 123.6, 122.7, 106.3, 77.2, 50.1, 43.9, 30.8. MS (ESI) <i>m</i>/<i>z</i> 297 [(M + H)<sup>+</sup>], RT 0.96 min. HRMS (ESI) calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>OS [M + H]<sup>+</sup> 297.1061, found 297.1053.</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>S</i>)-6-(Pyrrolidin-3-yloxy)-4-(thiophen-3-yl)isoquinoline (<b>13</b>)</h4><div class="NLM_p last">Using general procedures B and C, thiophene-3-boronic acid <b>47d</b> (27 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>13</b> (25.6 mg, 62%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (br s, 1H, N<sup>+</sup>H), 9.64 (s, 1H, Ar), 9.61 (br s, 1H, N<sup>+</sup>H), 8.60 (s, 1H, Ar), 8.53 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.06 (d, 1H, <i>J</i> = 2.0 Hz, Ar), 7.89 (dd, 1H, <i>J</i> = 5.0, 3.0 Hz, Ar), 7.69 (dd, 1H, <i>J</i> = 10.0, 3.0 Hz, Ar), 7.55 (dd, 1H, <i>J</i> = 5.0, 1.0 Hz, Ar), 7.44 (d, 1H, <i>J</i> = 3.0 Hz, Ar), 5.43 (br s, 1H, OCH), 3.62–3.40 (m, 2H, NCH), 3.40–3.20 (m, 2H, NCH), 2.30–2.19 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.3, 146.2, 138.4, 134.8, 133.7, 130.3, 129.2, 128.4, 127.4, 123.6, 122.8, 106.7, 77.2, 50.1, 43.8, 30.8. MS (ESI) <i>m</i>/<i>z</i> 297 [(M + H)<sup>+</sup>], RT 0.95 min. HRMS (ESI) calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>OS [M + H]<sup>+</sup> 297.1061, found 297.1054.</div></div><div id="sec5_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>S</i>)-4-(Furan-3-yl)-6-(pyrrolidin-3-yloxy)isoquinoline (<b>14</b>)</h4><div class="NLM_p last">Using general procedures B and C, 3-furanboronic acid <b>47e</b> (23 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>14</b> (22.3 mg, 56%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.62 (s, 1H, Ar), 9.62 (br s, 1H, N<sup>+</sup>H), 9.59 (br s, 1H, N<sup>+</sup>H), 8.61 (s, 1H, Ar), 8.51 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.34 (s, 1H, Ar), 8.01 (dd, 1H, <i>J</i> = 1.5, 1.5 Hz, Ar), 7.66 (dd, 1H, <i>J</i> = 10.0, 2.5 Hz, Ar), 7.53 (d, 1H, <i>J</i> = 2.5 Hz, Ar), 7.10–7.08 (m, 1H, Ar), 5.52 (br s, 2H, OCH), 3.57–3.45 (m, 2H, NCH), 3.43–3.27 (m, 2H, NCH), 2.32–2.23 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 146.5, 145.2, 143.0, 133.8, 133.6, 123.5, 123.1, 119.7, 111.9, 106.2, 77.2, 50.1, 43.9, 30.8. MS (ESI) <i>m</i>/<i>z</i> 281 [(M + H)<sup>+</sup>], RT 0.88 min. HRMS (ESI) calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 281.1290, found 281.1283.</div></div><div id="sec5_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>S</i>)-2-(6-(Pyrrolidin-3-yloxy)isoquinolin-4-yl)thiazole (<b>15</b>)</h4><div class="NLM_p last">Using general procedures D and C, Pd(PPh<sub>3</sub>)<sub>4</sub> (30 mg, 0.026 mmol, 0.2 equiv), <b>46</b> (50 mg, 0.127 mmol, 1.0 equiv), 2-tributylstannylthiazole <b>48a</b> (80 μL, 0.254 mmol, 2.0 equiv), and LiCl (11 mg, 0.254 mmol, 2.0 equiv) in toluene (1.3 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>15</b> (33.1 mg, 71%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.93 (br s, 1H, N<sup>+</sup>H), 9.84 (s, 1H, Ar), 9.76 (br s, 1H, N<sup>+</sup>H), 8.99 (s, 1H, Ar) 8.63 (d, 1H, <i>J</i> = 10.0, Ar), 8.51 (d, 1H, <i>J</i> = 2.5 Hz, Ar), 8.26 (d, 1H, <i>J</i> = 3.0 Hz, Ar), 8.15 (d, 1H, <i>J</i> = 3.0 Hz, Ar), 7.78 (dd, 1H, <i>J</i> = 10.0, 2.5 Hz, Ar), 5.47 (m, 1H, OCH), 3.67–3.54 (m, 1H, NCH), 3.52–3.41 (m, 1H, NCH), 3.41–3.24 (m, 2H, NCH), 2.41–2.27 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.8, 161.4, 147.2, 145.1, 137.2, 134.5, 133.6, 126.2, 123.8, 123.2, 107.5, 77.6, 49.9, 40.6, 30.9. MS (ESI) <i>m</i>/<i>z</i> 298 [(M + H)<sup>+</sup>], RT 0.80 min. HRMS (ESI) calcd for C<sub>16</sub>H<sub>15</sub>N<sub>3</sub>OS [M + H]<sup>+</sup> 298.1014, found 298.1005.</div></div><div id="sec5_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>S</i>)-4-(Pyridin-2-yl)-6-(pyrrolidin-3-yloxy)isoquinoline (<b>16</b>)</h4><div class="NLM_p last">Using general procedures D and C, Pd(PPh<sub>3</sub>)<sub>4</sub> (30 mg, 0.026 mmol, 0.2 equiv), <b>46</b> (50 mg, 0.127 mmol, 1.0 equiv), trimethyl(2-pyridyl)tin <b>48a</b> (61 μL, 0.254 mmol, 2.0 equiv), and LiCl (11 mg, 0.254 mmol, 2.0 equiv) in toluene (1.3 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>16</b> (29.4 mg, 63%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.94 (br s, 1H, N<sup>+</sup>H), 9.84 (s, 1H, Ar), 9.76 (br s, 1H, N<sup>+</sup>H), 8.92–8.88 (m, 1H, Ar), 8.79 (s, 1H, Ar), 8.63 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.19 (ddd, 1H, <i>J</i> = 8.0, 8.0, 1.5 Hz, Ar), 8.00 (d, 1H, <i>J</i> = 8.0 Hz, Ar), 7.80–7.68 (m, 3H, Ar), 5.42 (br s, 1H, OCH), 3.57–3.48 (m, 1H, NCH), 3.47–3.39 (m, 1H, NCH), 2.30–2.18 (m, 1H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1, 152.5, 149.7, 146.6, 139.4, 138.5, 134.2, 133.2 132.7 126.3, 125.0, 123.6, 123.3, 107.1, 77.4, 49.9, 43.8, 34.5, 30.8. MS (ESI) <i>m</i>/<i>z</i> 292 [(M + H)<sup>+</sup>], RT 0.75 min. HRMS (ESI) calcd for C<sub>16</sub>H<sub>17</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 292.1450, found 292.1443.</div></div><div id="sec5_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>S</i>)-2-(6-(Pyrrolidin-3-yloxy)isoquinolin-4-yl)benzo[<i>d</i>]oxazole (<b>17</b>)</h4><div class="NLM_p last">Using general procedures E and C, Pd(OAc)<sub>2</sub> (5.7 mg, 0.025 mmol, 0.2 equiv), <b>46</b> (50 mg, 0.127 mmol, 1.0 equiv), benzoxazole <b>49a</b> (23 mg, 0.191 mmol, 1.5 equiv), CuBr (15 mg, 0.102 mmol, 0.8 equiv), Cs<sub>2</sub>CO<sub>3</sub> (42 mg, 0.127 mmol, 1.0 equiv), and <sup><i>t</i></sup>Bu<sub>3</sub>P (10 mg, 0.05 mmol, 0.4 equiv) in DMF (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>17</b> (49.9 mg, 72%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.1 (br s, 1H, N<sup>+</sup>H), 10.2 (br s, 1H, Ar), 9.70 (br s, 1H, N<sup>+</sup>H), 8.53 (d, 1H, <i>J</i> = 9.0 Hz, Ar), 7.87 (s,1H, Ar), 7.73 (d, 1H, <i>J</i> = 8.0 Hz, Ar) 7.68 (d, 1H, <i>J</i> = 8.0 Hz, Ar), 7.12–7.04 (m, 3H, Ar), 6.88 (dd, 1H, <i>J</i> = 8.0, 8.0 Hz, Ar), 5.47 (br s, 1H, OCH), 3.66–3.53 (m, 1H, NCH), 3.53–3.43 (m, 1H, NCH), 3.43–3.21 (m, 2H NCH), 2.46–2.29 (m, 1H, CH), 2.29–2.14 (m, 1H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 150.7, 133.7, 127.0, 125.8, 125.4, 124.2, 123.6, 122.2, 119.3, 116.5, 109.9, 106.3, 77.3, 50.1, 43.9, 3.8, 29.1, 26.7. MS (ESI) <i>m</i>/<i>z</i> 350 [(M + H)<sup>+</sup>], RT 0.77 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> 332.1399, found 332.1396.</div></div><div id="sec5_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>S</i>)-2-(6-(Pyrrolidin-3-yloxy)isoquinolin-4-yl)benzo[d]thiazole (<b>18</b>)</h4><div class="NLM_p last">Using general procedures E and C, Pd(OAc)<sub>2</sub> (5.7 mg, 0.025 mmol, 0.2 equiv), <b>46</b> (50 mg, 0.127 mmol, 1.0 equiv), benzothiazole <b>49b</b> (26 mg, 0.191 mmol, 1.5 equiv), CuBr (15 mg, 0.102 mmol, 0.8 equiv), Cs<sub>2</sub>CO<sub>3</sub> (42 mg, 0.127 mmol, 1.0 equiv), and <sup><i>t</i></sup>Bu<sub>3</sub>P (10 mg, 0.05 mmol, 0.4 equiv) in DMF (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>18</b> (40.5 mg, 78%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (br s, 1H, N<sup>+</sup>H), (br s, 1H, N<sup>+</sup>H), 8.70 (d, 1H, <i>J</i> = 2.5 Hz, Ar), 8.46 (d, 1H, <i>J</i> = 9.0 H, Ar), 8.31–8.26 (m, 2H, Ar), 7.69–7.58 (m, 3H, Ar), 5.44 (br s, 1H, OCH), 3.69–3.58 (m, 1H, NC), 3.58–3.48 (m, 1H, NCH), 3.45–3.27 (m, 2H, NCH), 2.48–2.35 (m, 1H, CH), 2.35–2.25 (m, 1H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 153.9, 134.9, 127.4 126.8, 124.0, 122.9, 122.5, 77.2, 50.2, 44.0, 30.9, 26.7. MS (ESI) <i>m</i>/<i>z</i> 348 [(M + H)<sup>+</sup>], RT 1.09 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>OS [M + H]<sup>+</sup> 348.1170, found 348.1165.</div></div><div id="sec5_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>S</i>)-3-(6-(Pyrrolidin-3-yloxy)isoquinolin-4-yl)quinoline (<b>19</b>)</h4><div class="NLM_p last">Using general procedures B and C, 3-quinolineboronic acid <b>47f</b> (35.3 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>19</b> (32.7 mg, 70%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (br s, 1H, N<sup>+</sup>H), 9.88 (br s, 1H, Ar), 9.72 (br s, 1H, N<sup>+</sup>H), 9.38 (br s, 1H, Ar), 9.10 (br s, 1H, Ar), 8.82 (s, 1H, Ar), 8.66 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.40–8.26 (m, 2H, Ar), 8.12–8.00 (m, 1H, Ar), 7.93–7.84 (m, 1H, Ar), 7.75 (dd, 1H, <i>J</i> = 10.0, 2.0 Hz, Ar), 7.40 (br s, 1H, Ar), 5.42 (br s, 1H, OCH), 3.49–3.36 (m, 2H, NCH), 3.36–3.17 (m, 2H, NCH), 2.29–2.10 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.3, 139.2, 134.2, 129.6, 129.1, 128.3, 128.1, 123.8, 106.5, 77.4, 49.8, 43.7, 30.8. MS (ESI) <i>m</i>/<i>z</i> 342 [(M + H)<sup>+</sup>], RT 0.95 min. HRMS (ESI) calcd for C<sub>22</sub>H<sub>19</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 342.1606, found 342.1597.</div></div><div id="sec5_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>S</i>)-4-(Naphthalen-2-yl)-6-(pyrrolidin-3-yloxy)isoquinoline (<b>20</b>)</h4><div class="NLM_p last">Using general procedures B and C, 2-naphthaleneboronic acid <b>47g</b> (35.3 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>20</b> (22.5 mg, 36%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.74 (br m, 1H, Ar), 8.67 (s, 1H, Ar), 8.61–8.56 (m, 1H, Ar), 8.24 (br s, 1H, Ar), 8.18 (d, 1H, <i>J</i> = 8.5 Hz, Ar), 8.11–8.07 (m, 1H, Ar), 7.80–7.76 (m, 1H, Ar), 7.73–7.61 (m, 3H, Ar), 7.33 (br s, 1H, Ar), 5.33 (br s, 1H, OCH), 3.51–3.38 (m, 2H, NCH), 3.38–3.19 (m, 2H, NCH), 2.22–2.14 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.1, 147.0. 138.5, 135.0, 133.6, 133.5, 133.3, 132.3, 129.7, 129.0, 128.8, 128.3, 127.7, 127.6, 127.4, 123.7, 122.7, 106.7, 77.1, 50.1, 43.9, 30.6. MS (ESI) <i>m</i>/<i>z</i> 341 [(M + H)<sup>+</sup>], RT 1.20 min. HRMS (ESI) calcd for C<sub>23</sub>H<sub>20</sub>N<sub>2</sub>O [M + H]<sup>+</sup> 341.1654, found 341.1645.</div></div><div id="sec5_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>S</i>)-6-(Pyrrolidin-3-yloxy)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-5-yl)isoquinoline (<b>21</b>)</h4><div class="NLM_p last">Using general procedures B and C, 7-azaindole-5-boronic acid pinacol ester <b>47h</b> (52 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv), and <b>46</b> (40 mg, 0.105 mmol, 1.0 equiv) in <i>n</i>-BuOH (4 mL), followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>21</b> (49.9 mg, 72%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.2 (s, 1H, Ar), 9.84 (br s, 1H, N<sup>+</sup>H), 9.84 (s, 1H, Ar), 8.65 (br s, 1H, N<sup>+</sup>H), 8.66 (s, 1H, Ar), 8.48 (d, 1H, <i>J</i> = 2.0 Hz, Ar), 8.34 (d, 1H, <i>J</i> = 2.0 Hz, Ar), 7.76 (dd, 1H, <i>J</i> = 9.0, 2.0 Hz, Ar) 7.68 (dd, 1H, <i>J</i> = 3.0, 3.0 Hz, Ar), 7.35 (d, 1H, <i>J</i> = 3.0 Hz, Ar), 6.65 (dd, 1H, <i>J</i> = 3.0, 1.5 Hz, Ar), 5.39 (br s, 1H, OCH), 3.47–3.38 (m, 2H, NCH), 3.36–3.19 (m, 2H, NCH), 2.23–2.15 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1, 145.2, 142.0, 139.9, 134.3, 130.8, 128.5, 123.4, 121.9, 120.6, 107.0, 101.2, 77.3, 49.9, 43.8, 30.7. MS (ESI) <i>m</i>/<i>z</i> 331 [(M + H)<sup>+</sup>], RT 0.86 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>18</sub>N<sub>4</sub>O [M + H]<sup>+</sup> 331.1559, found 331.1552.</div></div><div id="sec5_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>S</i>)-4-(6-(Pyrrolidin-2-ylmethoxy)isoquinolin-4-yl)aniline (<b>22</b>)</h4><div class="NLM_p last">Using general procedures A, B, and C, 4-bromo-6-fluoro-isoquinoline <b>45</b> (50 mg, 0.222 mmol, 1.0 equiv), NaH (14 mg, 0.333 mmol, 1.5 equiv), <i>N</i>-(<i>t</i>-Butoxycarbonyl)-1-prolinol <b>50a</b> (67 mg, 0.333 mmol, 1.5 equiv) in DMA (1.5 mL), and then <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), and K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv) in <i>n</i>-BuOH (4 mL), followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>22</b> (6.5 mg, 7%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.93 (br s, 1H, N<sup>+</sup>H), 9.75 (d, 1H, <i>J</i> = 10.0 Hz, Ar) 9.51 (br s, 1H, N+H), 8.60 (dd, 1H, <i>J</i> = 10.0, 4.5 Hz, Ar), 8.53 (d, 1H, <i>J</i> = 3.5 Hz, Ar), 7.75 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 7.70–7.58 (m, 2H, Ar), 7.54–7.32 (m, 3H, Ar), 4.53–4.42 (m, 2H, OCH), 4.02–3.88 (m, 1H, NCH), 3.27–3.16 (m, 2H, NCH), 2.17–2.06 (m, 1H, CH), 2.06–1.94 (m, 1H, CH), 1.94–1.83 (m, 1H, CH), 1.83–1.69 (m, 1H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 145.3, 139.0, 135.3, 131.5, 123.5, 122.9, 105.7, 67.8, 57.9, 45.4, 26.7, 23.8. MS (ESI) <i>m</i>/<i>z</i> 320 [(M + H)<sup>+</sup>], RT 0.85 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 320.1763, found 320.1754.</div></div><div id="sec5_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-4-(6-(Pyrrolidin-3-ylmethoxy)isoquinolin-4-yl)aniline (<b>23</b>)</h4><div class="NLM_p last">Using general procedures A, B, and C, 4-bromo-6-fluoro-isoquinoline <b>45</b> (50 mg, 0.222 mmol, 1.0 equiv), NaH (14 mg, 0.333 mmol, 1.5 equiv), and (<i>S</i>)-<i>N</i>-Boc-pyrrolidine-3-methanol <b>50b</b> (67 mg, 0.333 mmol, 1.5 equiv) in DMA (1.5 mL), and then <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), and K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>23</b> (14.1 mg, 16%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.74 (s, 1H, Ar), 9.62 (br, 2H, N<sup>+</sup>H), 8.58 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.52 (s, 1H, Ar), 7.74 (dd, 1H, <i>J</i> = 10.0, 2.5, Ar), 7.68 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 7.46 (d, 1H, <i>J</i> = 10.0, Ar), 7.32 (s, 1H, Ar), 4.25 (d, 2H, <i>J</i> = 6.5 Hz, Ar), 3.38–3.21 (m, 2H, NCH), 3.21–3.09 (m, 1H, NCH), 3.08–2.96 (m, 1H, NCH), 2.86–2.74 (m, 1H, CH), 2.16–2.05 (m, 1H, CH), 1.84–1.72 (m, 1H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.9, 145.0, 139.2, 133.9, 131.8, 131.5, 123.33, 123.3, 122.1, 104.9, 66.7, 46.9, 44.5, 37.1, 27.1. MS (ESI) <i>m</i>/<i>z</i> 320 [(M + H)<sup>+</sup>], RT 0.86 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 320.1763, found 320.1755.</div></div><div id="sec5_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-(6-(Piperidin-4-yloxy)isoquinolin-4-yl)aniline (<b>24</b>)</h4><div class="NLM_p last">Using general procedures A, B, and C, 4-bromo-6-fluoro-isoquinoline <b>45</b> (50 mg, 0.222 mmol, 1.0 equiv), NaH (14 mg, 0.333 mmol, 1.5 equiv), and <i>t</i>-butyl 2-(hydroxymethyl)tetrahydro-(2<i>H</i>)-pyridinecarboxylate <b>50c</b> (72 mg, 0.333 mmol, 1.5 equiv) in DMA (1.5 mL), and then <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), and K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>24</b> (7.9 mg, 9%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.71 (br s, 1H, Ar), 9.40 (br s, 1H, N<sup>+</sup>H), 9.28 (br, 1H, N<sup>+</sup>H), 8.61 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.50 (s, 1H, Ar), 7.80 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 7.63–7.55 (m, 2H, Ar), 7.38–7.26 (m, 2H, Ar), 4.97 (brs, 1H, OCH), 3.21 (br s, 2H, NCH), 3.08 (br s, 2H, NCH), 2.23–2.11 (m, 2H, CH), 2.02–1.90 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.2, 145.1, 139.1, 135.2, 134.2, 131.7, 131.5, 123.26, 123.0, 120.6, 106.8, 70.7, 42.6, 27.0. MS (ESI) <i>m</i>/<i>z</i> 320 [(M + H)<sup>+</sup>], RT 0.85 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 320.1763, found 320.1757.</div></div><div id="sec5_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-(6-(Piperidin-4-ylmethoxy)isoquinolin-4-yl)aniline (<b>25</b>)</h4><div class="NLM_p last">Using general procedures A, B, and C, 4-bromo-6-fluoro-isoquinoline <b>45</b> (50 mg, 0.222 mmol, 1.0 equiv), NaH (14 mg, 0.333 mmol, 1.5 equiv), and <i>N</i>-Boc-4-piperidinemethanol <b>50d</b> (72 mg, 0.333 mmol, 1.5 equiv) in DMA (1.5 mL), and then <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (4.8 mg, 0.021 mmol, 0.2 equiv), SPHOS (16.4 mg, 0.044 mmol, 0.4 equiv), and K<sub>3</sub>PO<sub>4</sub> (46.8 mg, 0.21 mg, 2.0 equiv) in <i>n</i>-BuOH (4 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>25</b> (8.7 mg, 10%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.71 (s, 1H, Ar), 9.22 (br s, 1H, N<sup>+</sup>H), 8.96 (br s, 1H, N<sup>+</sup>H), 8.57 (d, 1H, <i>J</i> = 10.0 Hz, Ar), 8.50 (s, 1H, Ar), 7.70 (dd, 1H, <i>J</i> = 10.0, 2.0 Hz, Ar), 7.65–7.53 (m, 2H, Ar), 7.39–7.30 (m, 3H, Ar), 4.07 (d, 2H, <i>J</i> = 6.0 Hz, OCH), 3.27 (d, 2H, <i>J</i> = 12.0 Hz, NCH), 2.95–2.83 (m, 2H, NCH), 2.13 (br s, 1H, CH), 1.94–1.84 (m, 2H, CH), 1.63–1.47 (m, 2H, CH). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.9, 144.9, 139.2, 135.4, 133.9, 131.8, 131.4, 123.3, 123.0, 105.2, 72.4, 43.0, 33.4, 25.4. MS (ESI) <i>m</i>/<i>z</i> 334 [(M + H)<sup>+</sup>], RT 0.91 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>23</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 334.1919, found 334.1911.</div></div><div id="sec5_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 7-(4-Aminophenyl)-<i>N</i>-(piperidin-3-yl)isoquinolin-5-amine (<b>26</b>)</h4><div class="NLM_p last">Using general procedures B and C, <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47a</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (3.8 mg, 0.017 mmol, 0.2 equiv), SPHOS (14 mg, 0.034 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (36 mg, 0.17 mg, 2.0 equiv), and <b>53</b> (30 mg, 0.083 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>26</b> (22.8 mg, 52%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.8–9.5 (m, 2H), 9.45 (br s, 1H), 8.96 (d, 1H, <i>J</i> = 6.8 Hz), 8.59 (d, 1H, <i>J</i> = 6.6 Hz), 8.0–7.9 (m, 3H), 7.51 (s, 1H), 7.38 (d, 2H, <i>J</i> = 8.6 Hz), 7.14 (br s, 1H), 4.34 (br s, 2H), 4.11 (br s, 1H), 3.90 (br s, 1H), 3.8–3.5 (m, 1H), 3.5–3.4 (m, 1H), 3.4–3.2 (m, 1H), 3.0–2.8 (m, 2H), 2.1–1.9 (m, 2H), 1.9–1.7 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 146.7, 142.9, 142.3, 129.8, 129.0, 128.6, 127.0, 121.8, 120.3, 114.0, 110.0, 45.9, 42.8, 27.6, 20.5. MS (ESI) <i>m</i>/<i>z</i> 319 [(M + H)<sup>+</sup>], RT 0.58 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>N<sub>4</sub> [M + H]<sup>+</sup> 319.1922, found 319.1919.</div></div><div id="sec5_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 7-(3-Aminophenyl)-<i>N</i>-(piperidin-3-yl)isoquinolin-5-amine (<b>27</b>)</h4><div class="NLM_p last">Using general procedures B and C, <i>tert</i>-butyl <i>N</i>-[3-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47i</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (3.8 mg, 0.017 mmol, 0.2 equiv), SPHOS (14 mg, 0.034 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (36 mg, 0.17 mg, 2.0 equiv), and <b>53</b> (30 mg, 0.083 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>27</b> (6.5 mg, 14%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.63 (br s, 2H), 9.22 (br s, 3H), 8.75 (br s, 2H), 8.59 (d, 2H, <i>J</i> = 6.4 Hz), 7.84 (s, 2H), 7.41 (br s, 3H), 7.35 (s, 2H), 7.1–7.0 (m, 2H), 7.0–6.8 (m, 2H), 4.20 (br s, 2H), 3.3–3.1 (m, 3H), 2.88 (br s, 3H), 2.8–2.5 (m, 2H), 2.5–2.2 (m, 2H), 2.07 (br s, 2H), 1.99 (br s, 1H), 1.91 (s, 1H), 1.84 (br d, 2H, <i>J</i> = 9.8 Hz), 1.24 (s, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 160.5, 156.1, 153.2, 149.6, 142.7, 140.3, 134.3, 130.0, 129.1, 126.7, 122.6, 114.0, 46.1, 43.0, 31.6, 30.9, 27.6. MS (ESI) <i>m</i>/<i>z</i> 319 [(M + H)<sup>+</sup>], RT 0.89 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>N<sub>4</sub> [M + H]<sup>+</sup> 319.1922, found 319.1942.</div></div><div id="sec5_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 7-(2-Aminophenyl)-<i>N</i>-(piperidin-3-yl)isoquinolin-5-amine (<b>28</b>)</h4><div class="NLM_p last">Using general procedures B and C, <i>tert</i>-butyl <i>N</i>-[3-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47j</b> (67 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (3.8 mg, 0.017 mmol, 0.2 equiv), SPHOS (14 mg, 0.034 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (36 mg, 0.17 mg, 2.0 equiv), and <b>53</b> (30 mg, 0.083 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>28</b> (37.6 mg, 84%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.7–9.6 (m, 2H), 9.1–9.0 (m, 2H), 8.7–8.6 (m, 1H), 7.9–7.8 (m, 1H), 7.5–7.4 (m, 2H), 7.3–7.2 (m, 1H), 4.35 (br s, 1H), 4.28 (br s, 1H), 3.25 (br d, 1H, <i>J</i> = 12.2 Hz), 3.0–2.8 (m, 2H), 2.0–2.1 (m, 2H), 1.9–1.7 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 146.6, 142.8, 131.2, 130.3, 129.6, 128.7, 127.2, 120.4, 117.4, 112.5, 46.0, 42.9, 27.5, 20.5. MS (ESI) <i>m</i>/<i>z</i> 319 [(M + H)<sup>+</sup>], RT 0.95 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>N<sub>4</sub> [M + H]<sup>+</sup> 319.1922, found 319.1945.</div></div><div id="sec5_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-(5-(Piperidin-3-ylamino)isoquinolin-7-yl)phenol (<b>29</b>)</h4><div class="NLM_p last">Using general procedures B and C, 4-(tetrahydro-2<i>H</i>-pyran-2-yloxy)phenylboronic acid <b>47k</b> (62 mg, 0.276 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (6.2 mg, 0.028 mmol, 0.2 equiv), SPHOS (23 mg, 0.056 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (59 mg, 0.276 mmol, 2.0 equiv), and <b>53</b> (50 mg, 0.138 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (2 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>29</b> (11.9 mg, 29%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 10.0–9.7 (m, 1H), 9.60 (s, 1H), 9.5–9.2 (m, 2H), 8.83 (d, 1H, <i>J</i> = 6.4 Hz), 8.54 (d, 1H, <i>J</i> = 6.6 Hz), 7.87 (s, 1H), 7.8–7.7 (m, 2H), 7.41 (s, 1H), 7.0–6.8 (m, 3H), 3.7–3.4 (m, 3H), 3.4–3.2 (m, 2H), 3.0–2.8 (m, 2H), 2.7–2.5 (m, 1H), 2.2–1.9 (m, 2H), 1.9–1.7 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 158.7, 143.5, 143.1, 129.9, 129.7, 129.1, 126.9, 116.4, 113.6, 110.1, 46.5, 46.4, 43.4, 28.0, 20.9. MS (ESI) <i>m</i>/<i>z</i> 320 [(M + H)<sup>+</sup>], RT 0.87 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 320.1763, found 320.1752.</div></div><div id="sec5_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 2-(5-(Piperidin-3-ylamino)isoquinolin-7-yl)phenol (<b>30</b>)</h4><div class="NLM_p last">Using general procedures B and C, <i>o</i>-hydroxyphenylboronic acid <b>47l</b> (39 mg, 0.276 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (6.2 mg, 0.028 mmol, 0.2 equiv), SPHOS (23 mg, 0.056 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (59 mg, 0.276 mmol, 2.0 equiv), and <b>53</b> (50 mg, 0.138 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (2 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>30</b> (21.8 mg, 53%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.91 (br s, 1H), 9.70 (s, 1H), 9.56 (br s, 1H), 9.21 (br s, 1H), 8.95 (d, 1H, <i>J</i> = 6.6 Hz), 8.58 (d, 1H, <i>J</i> = 6.6 Hz), 7.85 (s, 1H), 7.42 (dd, 1H, <i>J</i> = 7.5, 1.5 Hz), 7.2–7.4 (m, 2H), 7.07 (d, 1H, <i>J</i> = 7.3 Hz), 7.0–6.9 (m, 2H), 4.08 (br s, 2H), 3.8–3.6 (m, 3H), 3.50 (br s, 1H), 3.5–3.4 (m, 1H), 3.3–3.1 (m, 2H), 3.1–2.8 (m, 3H), 2.03 (br s, 1H), 1.97 (br d, 1H, <i>J</i> = 5.6 Hz), 1.9–1.7 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 154.7, 146.8, 141.9, 141.7, 130.6, 129.5, 128.5, 126.7, 126.6, 119.5, 117.1, 116.3, 113.6, 46.0, 45.9, 42.9, 27.1, 20.0. MS (ESI) <i>m</i>/<i>z</i> 320 [(M + H)<sup>+</sup>], RT 0.94 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 320.1763, found 320.1749.</div></div><div id="sec5_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 7-Phenyl-<i>N</i>-(piperidin-3-yl)isoquinolin-5-amine (<b>31</b>)</h4><div class="NLM_p last">Using general procedures B and C, phenylboronic acid <b>47m</b> (21 mg, 0.17 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (3.8 mg, 0.017 mmol, 0.2 equiv), SPHOS (14 mg, 0.034 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (36 mg, 0.17 mg, 2.0 equiv), and <b>53</b> (30 mg, 0.083 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>31</b> (29.2 mg, 71%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.69 (s, 1H), 9.47 (br s, 2H), 8.94 (d, 1H, <i>J</i> = 6.6 Hz), 8.60 (d, 1H, <i>J</i> = 6.6 Hz), 7.98 (s, 1H), 7.91 (d, 2H, <i>J</i> = 7.6 Hz), 7.6–7.4 (m, 4H), 7.10 (br s, 1H), 4.30 (br s, 1H), 3.45 (br d, 1H, <i>J</i> = 10.3 Hz), 3.4–3.2 (m, 2H), 2.92 (br d, 2H, <i>J</i> = 7.1 Hz), 2.1–1.9 (m, 2H), 1.9–1.7 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 147.1, 143.0, 142.9, 139.0, 130.6, 129.1, 129.0, 128.6, 127.4, 127.0, 120.0, 114.4, 110.0, 46.0, 45.9, 42.9, 27.5, 20.4. MS (ESI) <i>m</i>/<i>z</i> 304 [(M + H)<sup>+</sup>], RT 1.09 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub> [M + H]<sup>+</sup> 304.1813, found 304.1799.</div></div><div id="sec5_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(4-(5-(Piperidin-3-ylamino)isoquinolin-7-yl)phenyl)acetamide (<b>32</b>)</h4><div class="NLM_p last">Using general procedures B and C, 4′-(4,4,5,5,-tetramethyl-1,3,2-dioxaborolan-2-yl)-acetanilide <b>47n</b> (38 mg, 0.21 mmol, 2.0 equiv), Pd(OAc)<sub>2</sub> (3.8 mg, 0.017 mmol, 0.2 equiv), SPHOS (14 mg, 0.034 mmol, 0.4 equiv), K<sub>3</sub>PO<sub>4</sub> (36 mg, 0.17 mg, 2.0 equiv), and <b>53</b> (30 mg, 0.083 mmol, 1.0 equiv) in <sup><i>n</i></sup>BuOH (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>32</b> (33.5 mg, 74%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.7–9.6 (m, 2H), 9.46 (br s, 1H), 8.97 (d, 1H, <i>J</i> = 6.8 Hz), 8.59 (d, 1H, <i>J</i> = 6.8 Hz), 7.97 (d, 4H, <i>J</i> = 8.8 Hz), 7.51 (s, 1H), 7.40 (d, 2H, <i>J</i> = 8.6 Hz), 7.17 (br d, 1H, <i>J</i> = 16.1 Hz), 4.34 (br s, 1H), 3.5–3.4 (m, 1H), 3.25 (br d, 1H, <i>J</i> = 12.2 Hz), 3.0–2.8 (m, 3H), 2.1–2.0 (m, 2H), 1.9–1.8 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 146.6, 143.0, 142.3, 129.8, 129.0, 128.6, 127.1, 122.1, 120.3, 114.1, 110.1, 45.9, 42.8, 27.6, 20.5. MS (ESI) <i>m</i>/<i>z</i> 361 [(M + H)<sup>+</sup>], RT 0.68 min.</div></div><div id="sec5_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>S</i>)-7-(4-Aminophenyl)-<i>N</i>-(piperidin-3-yl)isoquinolin-5-amine (<b>33</b>)</h4><div class="NLM_p last">Using general procedures G and C, (<i>S</i>)-1-Boc-3-aminopiperidine <b>57a</b> (16.2 mg, 0.081 mmol, 1.2 equiv), Pd<sub>2</sub>dba<sub>3</sub> (12.3 mg, 0.014 mmol, 0.2 equiv), DavePhos (11 mg, 0.027 mmol, 0.4 equiv), Cs<sub>2</sub>CO<sub>3</sub> (66 mg, 0.20 mmol, 3.0 equiv), and <b>56</b> (30 mg, 0.067 mmol, 1.0 equiv) in 1,4-dioxane (0.7 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>33</b> (9.0 mg, 28%) as a brown solid. NMR spectroscopic data consistent with <b>10</b>.</div></div><div id="sec5_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>R</i>)-7-(4-Aminophenyl)-<i>N</i>-(piperidin-3-yl)isoquinolin-5-amine (<b>34</b>)</h4><div class="NLM_p last">Using general procedures G and C, (<i>R</i>)-1-Boc-3-aminopiperidine <b>57b</b> (16.2 mg, 0.081 mmol, 1.2 equiv), Pd<sub>2</sub>dba<sub>3</sub> (12.3 mg, 0.014 mmol, 0.2 equiv), DavePhos (11 mg, 0.027 mmol, 0.4 equiv), Cs<sub>2</sub>CO<sub>3</sub> (66 mg, 0.20 mmol, 3.0 equiv), and <b>56</b> (30 mg, 0.067 mmol, 1.0 equiv) in 1,4-dioxane (0.7 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>34</b> (5.0 mg, 17%) as a brown solid. NMR spectroscopic data consistent with <b>10</b>.</div></div><div id="sec5_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i><sup>1</sup>-(7-(4-Aminophenyl)isoquinolin-5-yl)ethane-1,2-diamine (<b>35</b>)</h4><div class="NLM_p last">Using general procedures F and C, <b>55</b> (33.6 mg, 0.1 mmol, 1.0 equiv), <i>tert</i>-butyl <i>N</i>-(2-oxoethyl)carbamate (32 mg, 0.2 mmol, 2 equiv), NaBH(OAc)<sub>3</sub> (85 mg, 0.2 mmol, 2 equiv), and AcOH (1 mL) followed by 35% HCl<sub>(aq)</sub> (1 mL) in MeOH (2 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>35</b> (12.5 mg, 33%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.84–9.68 (m, 1H), 8.96 (d, <i>J</i> = 6.8 Hz, 1H), 8.63–8.46 (m, 4H), 8.44–8.22 (m, 1H), 8.02–7.91 (m, 2H), 7.61–7.46 (m, 6H), 7.43 (s, 4H), 7.39–7.25 (m, 5H), 3.62–3.53 (m, 1H), 3.24–3.07 (m, 2H), 2.90–2.75 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 146.4, 144.1, 142.3, 137.2, 129.6, 128.8, 128.6, 127.3, 123.1, 120.5, 114.1, 109.6, 57.5, 41.2, 40.4, 37.3. MS (ESI) <i>m</i>/<i>z</i> 279 [(M + H)<sup>+</sup>], RT 0.76 min.</div></div><div id="sec5_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 7-(4-Aminophenyl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>36</b>)</h4><div class="NLM_p last">Using general procedures G and C, 1-Boc-3-aminohomo-piperidine <b>57c</b> (17.3 mg, 0.081 mmol, 1.2 equiv), Pd<sub>2</sub>dba<sub>3</sub> (12.3 mg, 0.014 mmol, 0.2 equiv), DavePhos (11 mg, 0.027 mmol, 0.4 equiv), Cs<sub>2</sub>CO<sub>3</sub> (66 mg, 0.20 mmol, 3.0 equiv), and <b>56</b> (30 mg, 0.067 mmol, 1.0 equiv) in 1,4-dioxane (0.7 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>36</b> (6.5 mg, 19%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.90 (br s, 1H), 9.67 (s, 2H), 8.89 (br d, 1H, <i>J</i> = 6.4 Hz), 8.57 (d, 1H, <i>J</i> = 6.6 Hz), 8.0–7.9 (m, 3H), 7.43 (s, 1H), 7.32 (br d, 2H, <i>J</i> = 8.1 Hz), 7.13 (br s, 2H), 4.51 (br s, 1H), 3.4–3.2 (m, 4H), 3.16 (br s, 2H), 2.33 (br d, 1H, <i>J</i> = 6.1 Hz), 2.13 (br s, 1H), 2.0–1.8 (m, 5H), 1.8–1.5 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 146.7, 142.6, 129.1, 128.6, 127.0, 113.7, 110.1, 106.3, 103.7, 99.5, 49.0, 48.0, 45.9, 32.3, 24.8, 22.3. MS (ESI) <i>m</i>/<i>z</i> 333 [(M + H)<sup>+</sup>], RT 0.65 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>24</sub>N<sub>4</sub> [M + H]<sup>+</sup> 333.2079, found 333.2103.</div></div><div id="sec5_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>S</i>)-7-(4-Amino-2-fluorophenyl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>37</b>)</h4><div class="NLM_p last">Using general procedures H and C, boronate <b>47o</b> (1.06 g, 4.46 mmol, 1.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (1.09 g, 1.34 mmol, 0.3 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (19 mL, 4.5 mmol, 1.1 equiv), and <b>59</b> (2.18 g, 4.46 mmol, 1.0 equiv) in THF (36 mL), followed by 35% HCl<sub>(aq)</sub> (9 mL) in MeOH (18 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>37</b> (1.40 g, 31%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.72 (s, 1H), 9.63 (br s, 2H), 8.91 (d, 1H, <i>J</i> = 6.8 Hz), 8.59 (d, 1H, <i>J</i> = 6.8 Hz), 7.82 (s, 1H), 7.7–7.4 (m, 1H), 7.3–7.2 (m, 1H), 7.1–7.0 (m, 1H), 6.9–6.7 (m, 2H), 4.33 (br s, 2H), 3.4–3.2 (m, 3H), 3.2–3.1 (m, 1H), 2.2–2.1 (m, 1H), 2.0–1.8 (m, 4H), 1.8–1.5 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 161.3, 158.8, 146.4, 142.2, 138.8, 131.7, 131.7, 131.3, 129.4, 128.7, 126.9, 120.2, 116.0, 112.5, 49.1, 48.1, 46.0, 31.9, 25.0, 22.2. MS (ESI) <i>m</i>/<i>z</i> 351 [(M + H)<sup>+</sup>], RT 0.69 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>23</sub>FN<sub>4</sub> [M + H]<sup>+</sup> 351.1985, found 351.2007.</div></div><div id="sec5_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>S</i>)-7-(4-Amino-2,3-difluorophenyl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>38</b>)</h4><div class="NLM_p last">Using general procedures H and C, boronate <b>47p</b> (2.28 g, 8.92 mmol, 1.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (2.18 g, 2.68 mmol, 0.3 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (18 mL, 9.00 mmol, 1.1 equiv), and <b>59</b> (4.36 g, 8.92 mmol, 1.0 equiv) in THF (72 mL), followed by 35% HCl<sub>(aq)</sub> (18 mL) in MeOH (36 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>38</b> (2.51 g, 59%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.8–9.6 (m, 3H), 8.95 (d, 1H, <i>J</i> = 6.8 Hz), 8.6–8.5 (m, 1H), 7.83 (s, 1H), 7.5–7.4 (m, 1H), 7.3–7.2 (m, 1H), 6.9–6.7 (m, 2H), 4.38 (br dd, 3H, <i>J</i> = 4.6, 9.0 Hz), 3.3–3.2 (m, 2H), 3.13 (br s, 1H), 2.2–2.1 (m, 1H), 2.0–1.8 (m, 4H), 1.7–1.6 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 157.2, 154.8, 146.2, 146.1, 142.3, 138.3, 129.2, 128.7, 127.6, 126.9, 120.4, 116.4, 116.2, 112.3, 103.0, 49.0, 48.3, 46.1, 31.9, 25.1, 22.2. MS (ESI) <i>m</i>/<i>z</i> 369 [(M + H)<sup>+</sup>], RT 0.84 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>22</sub>F<sub>2</sub>N<sub>4</sub> [M + H]<sup>+</sup> 369.1891, found 369.1912.</div></div><div id="sec5_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>S</i>)-7-(4-Amino-2,5-difluorophenyl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>39</b>)</h4><div class="NLM_p last">Using general procedures H and C, boronate <b>47q</b> (1.14 g, 4.46 mmol, 1.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (1.09 g, 1.34 mmol, 0.3 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (19 mL, 4.5 mmol, 1.1 equiv), and <b>59</b> (2.18 g, 4.46 mmol, 1.0 equiv) in THF (36 mL) followed by 35% HCl<sub>(aq)</sub> (9 mL) in MeOH (18 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>39</b> (1.40 g, 66%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.80 (br s, 1H), 9.8–9.7 (m, 3H), 8.96 (d, 1H, <i>J</i> = 6.6 Hz), 8.7–8.6 (m, 1H), 8.6–8.6 (m, 1H), 7.82 (s, 1H), 7.49 (br s, 1H), 7.40 (s, 3H), 6.8–6.7 (m, 2H), 4.4–4.3 (m, 2H), 3.5–3.4 (m, 1H), 3.3–3.2 (m, 2H), 3.12 (br s, 2H), 2.1–2.2 (m, 2H), 1.9 (s, 7H), 1.7 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 146.2, 142.3, 138.3, 129.3, 128.8, 127.7, 127.5, 127.0, 125.2, 120.4, 115.9, 112.3, 111.8, 49.1, 48.2, 46.0, 31.9, 25.0, 22.2. MS (ESI) <i>m</i>/<i>z</i> 369 [(M + H)<sup>+</sup>], RT 0.82 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>22</sub>F<sub>2</sub>N<sub>4</sub> [M + H]<sup>+</sup> 369.1891, found 369.1886.</div></div><div id="sec5_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>S</i>)-<i>N</i>-(Azepan-3-yl)-7-phenylisoquinolin-5-amine (<b>40</b>)</h4><div class="NLM_p last">Using general procedures H and C, phenylboronic acid <b>47m</b> (19.5 mg, 0.16 mmol, 2.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (13.8 mg, 0.017 mmol, 0.2 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (320 μL, 0.16 mmol, 2.0 equiv), and <b>59</b> (39.1 mg, 0.083 mmol, 1.0 equiv) in THF (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>40</b> (12.0 mg, 37%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.75–9.56 (m, 3H), 8.85 (d, <i>J</i> = 6.6 Hz, 1H), 8.60 (d, <i>J</i> = 6.6 Hz, 1H), 7.96 (s, 1H), 7.90 (d, <i>J</i> = 7.6 Hz, 2H), 7.63–7.44 (m, 3H), 7.41 (s, 1H), 7.08 (br s, 1H), 4.47 (br s, 1H), 3.39–2.95 (m, 6H), 2.25–2.08 (m, 1H), 2.04–1.80 (m, 4H), 1.78–1.54 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 147.3, 143.0, 142.6, 139.1, 129.1, 128.6, 127.5, 127.0, 119.7, 114.3, 110.2, 49.0, 48.1, 46.0, 32.2, 24.9, 22.3. MS (ESI) <i>m</i>/<i>z</i> 319 [(M + H)<sup>+</sup>], RT 1.16 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>23</sub>N<sub>3</sub> [M + H]<sup>+</sup> 318.197, found 318.1967.</div></div><div id="sec5_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (<i>S</i>)-<i>N</i>-(Azepan-3-yl)-7-(2,6-difluorophenyl)isoquinolin-5-amine (<b>41</b>)</h4><div class="NLM_p last">Using general procedures H and C, 2,6-difluorophenylboronic acid <b>47r</b> (33 mg, 0.16 mmol, 2.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (13.8 mg, 0.017 mmol, 0.2 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (320 μL, 0.16 mmol, 2.0 equiv), and <b>59</b> (39.1 mg, 0.083 mmol, 1.0 equiv) in THF (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>41</b> (14.1 mg, 40%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.75 (s, 1H), 9.68 (br s, 1H), 9.62 (br s, 1H), 8.95 (d, <i>J</i> = 6.6 Hz, 1H), 8.67 (d, <i>J</i> = 6.8 Hz, 1H), 7.78 (s, 1H), 7.62–7.54 (m, 1H), 7.35–7.18 (m, 4H), 4.42–4.16 (m, 2H), 3.34–3.21 (m, 3H), 3.20–2.95 (m, 2H), 2.17–2.07 (m, 1H), 2.03–1.80 (m, 4H), 1.67–1.55 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 160.7, 160.6, 158.2, 158.1, 147.2, 142.2, 131.7, 128.4, 127.5, 120.1, 118.2, 113.2, 112.3, 112.1, 49.1, 48.2, 46.1, 31.8, 25.0, 22.3. MS (ESI) <i>m</i>/<i>z</i> 355 [(M + H)<sup>+</sup>], RT 0.85 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>21</sub>F<sub>2</sub>N<sub>3</sub> [M + H]<sup>+</sup> 354.1782, found 354.1773.</div></div><div id="sec5_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (<i>S</i>)-7-(6-Amino-pyridin-3-yl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>42</b>)</h4><div class="NLM_p last">Using general procedures H and C, 2-amino-5-(4,4,5,5-tetramethyl1,3,2-dioxaborolan)-2-ylpyridine <b>47s</b> (35.2 mg, 0.16 mmol, 2.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (13.8 mg, 0.017 mmol, 0.2 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (320 μL, 0.16 mmol, 2.0 equiv), and <b>59</b> (39.1 mg, 0.083 mmol, 1.0 equiv) in THF (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>42</b> (16.1 mg, 46%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 10.11 (br s, 1H), 9.71 (br s, 1H), 9.62 (s, 1H), 8.89 (d, <i>J</i> = 6.6 Hz, 1H), 8.71 (d, <i>J</i> = 2.0 Hz, 1H), 8.61–8.46 (m, 2H), 8.41 (br s, 1H), 7.97 (s, 1H), 7.47 (s, 1H), 7.29–7.07 (m, 2H), 4.56 (br s, 1H), 3.25–3.12 (m, 3H), 2.20–2.09 (m, 1H), 2.08–1.91 (m, 2H), 1.86 (br d, <i>J</i> = 7.8 Hz, 2H), 1.77–1.58 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 153.7, 146.5, 143.9, 142.8, 142.5, 137.4, 135.6, 134.3, 130.4, 128.9, 127.4, 122.8, 120.2, 113.7, 113.4, 113.3, 112.0, 108.8, 48.9, 48.0, 46.0, 32.6, 24.6, 22.4. MS (ESI) <i>m</i>/<i>z</i> 334 [(M + H)<sup>+</sup>], RT 0.52 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>23</sub>N<sub>5</sub> [M + H]<sup>+</sup> 334.2031, found 334.2025.</div></div><div id="sec5_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>S</i>)-7-(6-Amino-2-fluoropyridin-3-yl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>43</b>)</h4><div class="NLM_p last">Using general procedures H and C, boronate <b>47t</b> (38 mg, 0.16 mmol, 2.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (13.8 mg, 0.017 mmol, 0.2 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (320 μL, 0.16 mmol, 2.0 equiv), and <b>59</b> (39.1 mg, 0.083 mmol, 1.0 equiv) in THF (1 mL) followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then, 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>43</b> (11.3 mg, 31%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.77–9.61 (m, 3H), 8.91 (d, <i>J</i> = 6.6 Hz, 1H), 8.57 (d, <i>J</i> = 6.6 Hz, 1H), 7.95 (t, <i>J</i> = 9.3 Hz, 1H), 7.82 (s, 1H), 7.28 (s, 1H), 6.51 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 4.45–4.18 (m, 3H), 4.18–3.89 (m, 2H), 3.68 (br dd, <i>J</i> = 13.6, 5.5 Hz, 1H), 3.52–3.36 (m, 1H), 3.30–3.19 (m, 3H), 3.14 (br s, 1H), 2.20–2.06 (m, 1H), 2.04–1.81 (m, 4H), 1.73–1.58 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 160.5, 159.2, 159.0, 158.2, 146.3, 142.3, 142.1, 138.3, 138.2, 129.3, 128.8, 126.7, 120.3, 115.5, 112.0, 106.3, 106.0, 105.6, 49.1, 48.1, 46.0, 32.0, 25.0, 22.2. MS (ESI) <i>m</i>/<i>z</i> 352 [(M + H)<sup>+</sup>], RT 0.62 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>FN<sub>5</sub> [M + H]<sup>+</sup> 352.1937, found 352.1931.</div></div><div id="sec5_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>S</i>)-7-(6-Amino-4-fluoropyridin-3-yl)-<i>N</i>-(azepan-3-yl)isoquinolin-5-amine (<b>44</b>)</h4><div class="NLM_p last">Using general procedures E and C, boronate <b>47u</b> (38 mg, 0.16 mmol, 2.0 equiv), PdCl<sub>2</sub>(dppf)<sub>2</sub> (13.8 mg, 0.017 mmol, 0.2 equiv), 2 M Na<sub>2</sub>CO<sub>3(aq)</sub> (320 μL, 0.16 mmol, 2.0 equiv), and <b>59</b> (39.1 mg, 0.083 mmol, 1.0 equiv) in THF (1 mL), followed by 35% HCl<sub>(aq)</sub> (500 μL) in MeOH (1 mL) gave the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave the free salt. Then 1 M HCl in MeOH was added and evaporated to give the HCl salt of <b>44</b> (11.9 mg, 32%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.99 (br s, 1H), 9.75 (s, 1H), 9.67 (br s, 1H), 8.95 (d, <i>J</i> = 6.8 Hz, 1H), 8.66–8.38 (m, 3H), 7.83 (s, 1H), 7.51–7.31 (m, 1H), 7.25 (br s, 1H), 7.03 (d, <i>J</i> = 11.5 Hz, 1H), 4.39 (br s, 1H), 3.26–3.18 (m, 2H), 3.15 (br s, 1H), 2.33–2.08 (m, 1H), 2.06–1.81 (m, 4H), 1.76–1.56 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 169.6, 166.9, 156.7, 156.5, 146.9, 142.6, 140.2, 133.0, 130.8, 128.5, 127.5, 120.2, 116.6, 115.3, 115.1, 111.5, 98.4, 98.2, 49.0, 48.1, 46.1, 32.2, 24.6, 22.5. MS (ESI) <i>m</i>/<i>z</i> 352 [(M + H)<sup>+</sup>], RT 0.57 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>FN<sub>5</sub> [M + H]<sup>+</sup> 352.1937, found 352.1935.</div></div><div id="sec5_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>tert</i>-Butyl (<i>S</i>)-3-((4-Bromoisoquinolin-6-yl)oxy)pyrrolidine-1-carboxylate (<b>46</b>)</h4><div class="NLM_p last">Using general procedure A, (<i>S</i>)-1-(<i>tert</i>-butoxycarbonyl)-3-pyrrolidinol (3.36 g, 17.9 mmol, 1.5 equiv), NaH (720 mg, 17.9 mmol, 1.5 equiv), and isoquinoline <b>45</b> (2.69 g, 12.0 mmol, 1.0 equiv) in DMA (120 mL) gave the crude product. Purification by Yamazen automated chromatography with 70:30 hexane–EtOAc as eluent gave <b>46</b> (3.6 g, 78%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.16 (s, 1H, Ar), 8.65 (s, 1H, Ar), 8.15 (d, 1H, <i>J</i> = 10 Hz, Ar), 7.43 (dd, 1H, <i>J</i> = 10.0, 2.5 Hz, Ar), 7.29 (d, 1H, <i>J</i> = 2.5 Hz, Ar), 5.30 (br s, 1H, OCH), 3.75–3.62 (m, 1H, NCH), 3.55–3.44 (m, 2H, NCH), 3.43–3.30 (m, 1H, NCH), 2.33–2.08 (m, 2H, CH), 1.41 (s, 9H, <sup><i>t</i></sup>Bu). <sup>13</sup>C NMR (100.7 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.1, 154.0, 153.9, 151.7, 145.3, 136.7, 131.3, 125.6, 121.8, 118.0, 79.1, 77.5, 76.7, 51.8, 51.5, 44.3, 44.1, 31.0, 30.2, 28.6. MS (ESI) <i>m</i>/<i>z</i> 395 [(M + H)<sup>+</sup>], RT 1.76 min. HRMS (ESI) calcd for C<sub>18</sub>H<sub>22</sub>BrN<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup> 393.0814, found 393.0805.</div></div><div id="sec5_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>tert</i>-Butyl 3-((7-Chloroisoquinolin-5-yl)amino)piperidine-1-carboxylate (<b>53</b>)</h4><div class="NLM_p last">Using general procedure F, <b>52</b> (1.0 g, 5.6 mmol, 1.0 equiv), 1-Boc-3-piperidone (2.04 g, 11.2 mmol, 2 equiv), NaBH(OAc)<sub>3</sub> (2.38 g, 11.2 mmol, 2 equiv), and AcOH (38 mL) gave the crude product. Purification by Yamazen automated chromatography with 10:1 CHCl<sub>3</sub>–MeOH as eluent gave the <b>53</b> (2.02 g, 99%) as an orange amorphous. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) (1.5:1 of rotamer) δ 4.83 (d, <i>J</i> = 4.2 Hz, 1H), 3.77 (br s, 2H), 3.65–3.57 (m, 1H), 3.49–3.41 (m, 1H), 2.93–2.73 (m, 2H), 1.83–1.77 (m, 1H), 1.38 (s, 11H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 153.9, 151.3, 143.8, 141.7, 133.0, 129.6, 124.4, 115.1, 112.6, 106.2, 81.3, 78.4, 73.5, 65.0, 48.7, 32.9, 29.9, 28.0. MS (ESI) <i>m</i>/<i>z</i> 363 [(M + H)<sup>+</sup>], RT 1.70 min. HRMS (ESI) calcd for C<sub>19</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> 362.1635, found 362.1624.</div></div><div id="sec5_2_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>tert</i>-Butyl (4-(5-Aminoisoquinolin-7-yl)phenyl)carbamate (<b>55</b>)</h4><div class="NLM_p last">Using general procedure B, <b>54</b> (2.0 g, 11.2 mmol, 1.0 equiv), <i>tert</i>-butyl <i>N</i>-[4-(4,4,5,5-tetrametyl-1,3,2-dioxaborolane-2-yl)phenyl]carbamate <b>47a</b> (7.96 g, 33.6 mmol, 3.0 equiv), Pd(OAc)<sub>2</sub> (1.26 g, 5.6 mmol, 0.5 equiv), SPHOS (4.6 g, 11.2 mmol, 1.0 equiv), and K<sub>3</sub>PO<sub>4</sub> (7.13 g, 33.6 mmol, 3.0 equiv) in <sup><i>n</i></sup>BuOH (200 mL) gave the crude product. Purification by Yamazen automated chromatography with 30:70 hexane–EtOAc as eluent gave the <b>55</b> (1.59 g, 56%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 10.07 (s, 1H), 9.16 (s, 1H), 8.35 (d, <i>J</i> = 5.9 Hz, 1H), 7.95 (d, <i>J</i> = 6.1 Hz, 1H), 7.75–7.64 (m, 4H), 7.50 (s, 1H), 7.19 (d, <i>J</i> = 1.7 Hz, 1H), 6.07 (s, 2H), 2.11–2.05 (m, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 168.4, 152.5, 144.5, 140.8, 139.5, 139.0, 134.9, 129.8, 127.0, 124.1, 119.3, 115.4, 111.6, 109.2, 24.1. MS (ESI) <i>m</i>/<i>z</i> 278 [(M + H)<sup>+</sup>], RT 0.92 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup> 336.1712, found 336.1700.</div></div><div id="sec5_2_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>tert</i>-Butyl (4-(5-Iodoisoquinolin-7-yl)phenyl)carbamate (<b>56</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl nitrite (860 μL, 7.16 mmol, 1.5 equiv) was added to a stirred solution of <b>55</b> (1.6 g, 4.77 mmol, 1.0 equiv) and I<sub>2</sub> (1.82 g, 7.16 mmol, 1.5 equiv) in MeCN (60 mL) at rt and stirred at 80 °C for 1 h. The reaction mixture was quenched with saturated Na<sub>2</sub>S<sub>2</sub>O<sub>3(aq)</sub> (50 mL), extracted with EtOAc (100 mL × 3), washed with brine (100 mL), dried over MgSO<sub>4</sub>, and evaporated to give the crude product. Purification by Yamazen automated chromatography with 50:50 hexane–EtOAc as eluent gave the <b>56</b> (2.07 g, 97%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 9.55 (s, 1H), 9.29 (s, 1H), 8.66 (d, <i>J</i> = 1.7 Hz, 1H), 8.61 (d, <i>J</i> = 6.1 Hz, 1H), 8.46 (s, 1H), 7.84–7.73 (m, 3H), 7.67–7.58 (m, 2H), 1.80–1.55 (m, 1H), 1.51 (s, 9H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 153.5, 152.7, 144.6, 140.0, 140.0, 139.6, 135.5, 131.1, 129.3, 127.4, 124.8, 123.1, 118.5, 98.8, 79.3, 28.1. MS (ESI) <i>m</i>/<i>z</i> 447 [(M + H)<sup>+</sup>], RT 2.07 min. HRMS (ESI) calcd for C<sub>20</sub>H<sub>19</sub>IN<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 447.0569, found 447.0554. HRMS (ESI) calcd for C<sub>20</sub>H<sub>20</sub>IN<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup> 447.0569, found 447.0559.</div></div><div id="sec5_2_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>tert</i>-Butyl (<i>S</i>)-3-((7-(((Trifluoromethyl)sulfonyl)oxy)isoquinolin-5-yl)amino)azepane-1-carboxylate (<b>59</b>)</h4><div class="NLM_p last">Using general procedure G, (<i>S</i>)-1-Boc-3-aminohomo-piperidine (5.34 g, 24.9 mmol, 1.2 equiv), Pd<sub>2</sub>dba<sub>3</sub> (960 mg, 2.08 mmol, 0.2 equiv), DavePhos (11 mg, 0.027 mmol, 0.4 equiv), <sup><i>t</i></sup>BuONa (4.0 g, 20.8 mmol, 2.0 equiv), and <b>58</b> (4.65 g, 20.8 mmol, 1.0 equiv) in 1,4-dioxane (200 mL) gave the crude product. Subsequently, the crude product was dissolved to 1,4-dioxane (mL) and PhNTf<sub>2</sub> (15 g, 22.9 mmol, 1.1 equiv), Et<sub>3</sub>N (8.66 mL, 62.4 mmol, 3.0 equiv) were added to a stirred solution and stirred at 80 °C for 2 h. The reaction mixture was evaporated to give the crude product. Purification by Yamazen automated chromatography with 200:9:1 CH<sub>2</sub>Cl<sub>2</sub>–MeOH–NH<sub>4</sub>OH as eluent gave <b>59</b> (8.68 g, 85%) as a brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) (1.5:1 of rotamer) δ 9.24 (s, 1H), 8.4–8.6 (m, 1H), 8.3–8.1 (m, 1H), 7.4–7.3 (m, 1H), 7.3–7.1 (m, 1H), 6.8–6.6 (m, 2H), 4.0–3.8 (m, 1H), 3.72 (br dd, 1H, <i>J</i> = 14.0, 4.5 Hz), 3.56 (ddd, 2H, <i>J</i> = 14.0, 8.0, 6.0 Hz), 3.3–3.1 (m, 3H), 3.02 (br dd, 1H, <i>J</i> = 14.0, 8.0 Hz), 2.1–1.9 (m, 1H), 1.9–1.8 (m, 3H), 1.8–1.5 (m, 3H), 1.5 (s, 8H), 1.30 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz) δ 155.5, 152.4, 149.1, 145.0, 142.6, 142.5, 129.4, 128.8, 124.9, 122.9, 115.2, 115.0, 104.6, 99.9, 78.9, 78.6, 52.6, 52.0, 48.4, 47.1, 46.5, 33.1, 28.0, 27.9, 27.2, 22.3, 22.1. MS (ESI) <i>m</i>/<i>z</i> 490 [(M + H)<sup>+</sup>], RT 2.08 min. HRMS (ESI) calcd for C<sub>21</sub>H<sub>26</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub>S [M + H]<sup>+</sup> 490.1623, found 490.1653. HRMS (ESI) calcd for C<sub>21</sub>H<sub>27</sub>F3N<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup> 490.1624, found 490.1612.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Measurement of Kinase Inhibitory Activity against PKCζ</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Materials</h4><div class="NLM_p last">Purified recombinant human PKCζ protein was purchased from Invitrogen (P2273). The PKCζ peptide substrate (biotin-KKKKRFSFKKSFKC) was synthesized by SIGMA GENOSYS. The detection antibody was generated in-house using PKCζ peptide as an antigen. Streptavidin-XL665 was purchased from CisBio (610SAXLB). All other materials were purchased from commercial sources (e.g., Sigma-Aldrich and Wako).</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Methods</h4><div class="NLM_p">PKCζ full-length activity was measured by phosphorylation of the PKCζ peptide substrate with 10 μM ATP using the TR-FRET method. The enzyme reaction was conducted in 50 mM Tris-HCl (pH 7.5), 1 mM DTT, 5 mM MgCl<sub>2</sub>, 0.1% BSA, and 0.01% Triton X-100.</div><div class="NLM_p last">To determine inhibitor <i>K</i><sub>i</sub> values, test compounds were incubated for 30 min at room temperature in the presence of PKCζ enzyme, 0.5 μM peptide substrate, and 10 μM ATP. The reaction was stopped with EDTA (66.7 mM), detection mix was added (europium-labeled antibody (0.3 μg/mL), streptavidin-XL665 (3.3 μg/mL), 50 mM Tris-HCl (pH 7.5), 0.1% BSA, and 200 mM KF), the mixture was incubated for 60 min at room temperature, then fluorescence was measured at 615 and 665 nm.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Evaluation of Inhibitors on TNFα Production in THP-1 Cells Stimulated with LPS</h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Cell Culture</h4><div class="NLM_p last">THP-1 monocytic cell line was cultured in Roswell Park Memorial Institute (RPMI) 1640 medium containing 10% fetal bovine serum (FBS) at 37 °C. Before treatments, cells were seeded at 1 × 10<sup>5</sup> cells/well on 96-well plates.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Treatments of THP-1 Cells</h4><div class="NLM_p last">Test compounds (20 μL, 0.001, 0.01, 0.1, 1, 10, or 100 μM) containing 10% DMSO and 10% FBS/RPMI1640 were added to each well and preincubated for 60 min, then 20 μL of LPS (2–20 μg/mL of LPS in RPMI1640/10% FBS) was added, and the mixture was incubated for 4 h. After centrifugation for 2 min, 100 μL of the supernatant was removed and mixed with 5 μL of cell counting kit-8 solution and added to the remaining cell wells. Homogeneous time-resolved fluorescence (HTRF) from hTNFα was measured for each well at 450 nm.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> PK Study</h3><div class="NLM_p last">The PK activities of compounds <b>21</b> and <b>37</b> were studied using male C57BL/6N mice (Japan SLC, Inc.). Test compounds were administered intravenously at a dose of 1 mg/kg in saline or orally at a dose of 2 mg/kg in saline or water (dose volume 5 mL/kg) to mice (<i>n</i> = 2). Blood samples were collected via the subclavian vein at 0.5, 1, 2, 4, 6, and 8 h post dose. Plasma was separated by centrifugation and stored frozen at −20 °C until analysis. Plasma compound concentrations were determined using LC-MS/MS. PK parameters were calculated using the noncompartmental analysis tool provided with WinNonlin Enterprise software v6.4.0.768 (Certara).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Mouse CIA (Collagen-Induced Arthritis) Model</h3><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Preparation of Mouse CIA Model</h4><div class="NLM_p">DBA/1J female mice, 7 weeks old, were purchased from Japan SLC, Inc. and housed five per cage. After 1 week of acclimatization, the mice were injected into the base of the tail with 100 μL of 2 mg/mL bovine type II collagen (C2; MCK, K-41, Japan) emulsified 6:5 with 1 mg/mL Freund’s complete adjuvant (CFA; DIFCO). This C2/CFA emulsion was also injected into mice as a booster immunization 21 days after the first immunization.</div><div class="NLM_p">The development of arthritis was evaluated in a blinded manner. Disease severity was scored three times a week. The severity of arthritis in each paw was scored from 0 to 3.</div><div class="NLM_p last">No evidence of erythema and swelling received a score of 0, erythema and slight swelling confined to the toe, instep, or ankle joints received a score of 1, erythema and moderate swelling of the instep and ankle joints received a score of 2, and erythema and severe swelling extending from the toe to ankle joints received a score of 3. A total arthritis score was determined by summing the scores for all four paws.</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Evaluation of PKCζ Inhibitors</h4><div class="NLM_p last">We evaluated the anti-inflammatory effect of PKCζ inhibitors in mice by orally administering (p.o.) <b>37</b> (3, 10, 30 mg/kg) twice a day after booster immunization. The known PKCζ inhibitor BMS 345541 (60 mg/kg) was used as a positive control of PKCζ inhibition and was administrated (po) once a day. All inhibitors were dissolved in 0.5% aqueous methyl cellulose. The experimental protocols were approved by the experimental animal ethics committee at Asahi Kasei Pharma Corporation and were conducted in accordance with the guidelines concerning the management and handling of experimental animals.</div></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Modeling Studies</h3><div id="sec5_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Homology Modeling and Docking Simulation</h4><div class="NLM_p last">The crystal structure of PKCι (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DC2">4DC2</a>)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> was used as a template to build the homology models. The identity of the amino acid sequence with respect to PKCζ is 72% overall, rising to 84% in the catalytic domain. The kinase domain (residues 252–518) of PKCι was used to build 100 homology models (10 main chain models × 10 side chain models) of PKCζ. The lowest energy conformer was selected as the representative homology model of PKCζ and was subsequently subjected to docking studies with compounds <b>10</b> and <b>26</b>. The docking poses were carefully inspected and provided two possible binding modes for each compound (see text for details). Homology modeling and docking simulations were performed with MOE 2018.0101.</div></div><div id="sec5_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> MD Simulation</h4><div class="NLM_p last">The docking model candidates of compounds <b>10</b> and <b>26</b> were subjected to MD simulations to assess their predicted binding modes. In this study, MD simulations were performed using the GPU-accelerated version of AMBER14<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> using the Amber ff14SB force field<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> for proteins. The protonation states of the amino acid residues were treated in their general tautomer states under physiological conditions. Ligand force fields were parametrized using the general Amber force field (GAFF)<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> with AM1-BCC<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> charges. Rectangular simulation boxes were constructed with a margin of at least 10 Å between the protein and the periodic box boundaries solvated with TIP3P<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> water models. Electrostatic interactions were treated with the particle mesh Ewald method<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> in which Lennard-Jones interactions and real-space electrostatic interactions were smoothly switched at 9 Å. Each system was first minimized with harmonic positional restraints imposed on the protein backbone atoms with a force constant of 10 kcal mol<sup>–1</sup> Å<sup>–2</sup>. Subsequently, the system was subjected to a 100 ps MD simulation with the NVT ensemble at 300 K, and 200 ps of the NPT ensemble at 300 K and 1 bar, where the restraints were gradually reduced to zero. Finally, three distinct 30 ns simulations were performed, where the first 10 ns were used to further relax the system and the last 20 ns were used for the RMSD analysis described in the main text. Isothermal and isobaric conditions were realized using the Langevin thermostat,<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> with a friction constant of 2.0 ps<sup>–1</sup> and the Berendsen barostat<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> with a pressure relaxation time of 1.0 ps. The MD time step was 2 fs, with hydrogen bonds constrained using the SHAKE<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and SETTLE<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> algorithms.</div></div><div id="sec5_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> FMO Calculation</h4><div class="NLM_p last">Representative binding modes of compounds <b>10</b> and <b>26</b> were manually selected from the MD trajectories and subjected to FMO calculations to better understand the protein–ligand interactions. The protein–ligand system and the 20 nearest water molecules to the ligand were extracted from the MD trajectory. The complex system was minimized prior to FMO calculations using the default setting of the PrepWizard module implemented in Schrodinger 2018-4, ensuring that the protonation states were not changed. Then, the minimized structures, including the 20 nearest water molecules, were subjected to FMO calculations with FMO2-MP2/6-31G using the FMO code version 18 embedded in the General Atomic and Molecular Electronic Structure System (GAMESS).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00449" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21061" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21061" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00449?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00449</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Summary of biochemical assay and kinase panel data of <b>21</b> and <b>37</b>, mode of action studies with compound <b>37</b> and BMS 345541, FMO calculation, and HPLC traces of key compounds <b>21</b> and <b>37</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf">jm0c00449_si_001.pdf (560.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_002.csv">jm0c00449_si_002.csv (2.46 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00449" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32407" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32407" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masakazu Atobe</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7314-0237" title="Orcid link">http://orcid.org/0000-0002-7314-0237</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0362776c61662d6e61436c6e2d6270626b6a2e686270666a2d606c2d6973"><span class="__cf_email__" data-cfemail="6e0f1a010c0b40030c2e0103400f1d0f060743050f1d0b07400d0140041e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takayuki Serizawa</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natsumi Yamakawa</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span>; 
    <span>Present Address:
                        For N.Y.: Asahi Kasei Corporation, Olfactory Sensing Business Project, Marketing and Innovation, 1-3-1, Yako, Kawasaki, Kanagawa, 210-0863, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenichiro Takaba</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yukiko Nagano</span> - <span class="hlFld-Affiliation affiliation">Research
Coordination, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toshiaki Yamaura</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Drug Discovery, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eiichi Tanaka</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Drug Discovery, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atsutoshi Tazumi</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Pharmacology, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span>; 
    <span>Present Address:
                        For A.T.: Asahi Kasei Corporation, Regenerative Medicine Techhnology Department, Healthcare R&D Center, 632-1, Mifuku, Izunokuni, Shizuoka, 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shino Bito</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Safety Assessment & ADME, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni
Shizuoka 410-2321, Japan</span>; 
    <span>Present Address:
                        For S.B.: Asahi Kasei Pharma, Quality Assurance Department, Regulatory Affairs and Reliability Assurance Center, 632–1, Mifuku, Izunokuni, Shizuoka, 410–2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masashi Ishiguro</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Safety Assessment & ADME, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni
Shizuoka 410-2321, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masashi Kawanishi</span> - <span class="hlFld-Affiliation affiliation">Laboratory
for Medicinal Chemistry, Pharmaceutical Research Center, Asahi Kasei Pharma Corporation, 632-1 Mifuku, Izunokuni, Shizuoka 410-2321, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6710-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36838" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36838" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Hiromitsu Sakai, Dr. Kenji Takenuki, and Dr. Sachiko Obara for the chemistry support. Also, we thank Shinichi Tanaka, Takamichi Nagami, and Katsuhiko Sakurada for the RA model study support. Finally, we thank Rintaro Yamada, Hiromitsu Nagumo, and Dr. Akio Matsuda for the initiation and management of the project.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">aq</td><td class="NLM_def"><p class="first last">water</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">BSA</td><td class="NLM_def"><p class="first last">bovine serum albumin</p></td></tr><tr><td class="NLM_term">CFA</td><td class="NLM_def"><p class="first last">Freund’s complete adjuvant</p></td></tr><tr><td class="NLM_term">CIA</td><td class="NLM_def"><p class="first last">collagen-induced arthritis</p></td></tr><tr><td class="NLM_term">Cs<sub>2</sub>CO<sub>3</sub></td><td class="NLM_def"><p class="first last">cesium carbonate</p></td></tr><tr><td class="NLM_term">CuBr</td><td class="NLM_def"><p class="first last">copper(I) bromide</p></td></tr><tr><td class="NLM_term"><sup><i>t</i></sup>Bu<sub>3</sub>P</td><td class="NLM_def"><p class="first last">tri<i>tert</i>-butyl phosphine</p></td></tr><tr><td class="NLM_term">DAG</td><td class="NLM_def"><p class="first last">diacylglycerol</p></td></tr><tr><td class="NLM_term">DavePHOS</td><td class="NLM_def"><p class="first last">2-dicyclohexylphosphino-2′-(<i>N</i>,<i>N</i>-dimethylamino)-biphenyl</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiothreitol</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last">dimethylacetamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">EDTA</td><td class="NLM_def"><p class="first last">ethylenediaminetetraacetic acid</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray method</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug discovery</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">fragment molecular orbital</td><td class="NLM_def"><p class="first last">FMO</p></td></tr><tr><td class="NLM_term">GAFF</td><td class="NLM_def"><p class="first last">general Amber force field</p></td></tr><tr><td class="NLM_term">GAMESS</td><td class="NLM_def"><p class="first last">General Atomic and Molecular Electronic Structure System</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">I<sub>2</sub></td><td class="NLM_def"><p class="first last">iodine</p></td></tr><tr><td class="NLM_term">IFIE</td><td class="NLM_def"><p class="first last">interfragment interaction energy</p></td></tr><tr><td class="NLM_term">IKK</td><td class="NLM_def"><p class="first last">inhibitor of nuclear factor kappa-B kinase subunit</p></td></tr><tr><td class="NLM_term">K<sub>3</sub>PO<sub>4</sub></td><td class="NLM_def"><p class="first last">tripotassium phosphate</p></td></tr><tr><td class="NLM_term">LC/MS</td><td class="NLM_def"><p class="first last">liquid chromatography/mass spectrometry</p></td></tr><tr><td class="NLM_term">LPS</td><td class="NLM_def"><p class="first last">lipopolysaccharide</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MgSO<sub>4</sub></td><td class="NLM_def"><p class="first last">magnesium Sulfate</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matrix metalloprotease</p></td></tr><tr><td class="NLM_term">Mw</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">NaBH(OAc)<sub>3</sub></td><td class="NLM_def"><p class="first last">sodium triacetoxyborohydride</p></td></tr><tr><td class="NLM_term">NaH</td><td class="NLM_def"><p class="first last">sodium hydride</p></td></tr><tr><td class="NLM_term">Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub></td><td class="NLM_def"><p class="first last">sodium thiosulfate</p></td></tr><tr><td class="NLM_term"><sup><i>n</i></sup>BuOH</td><td class="NLM_def"><p class="first last">normal butanol</p></td></tr><tr><td class="NLM_term">ND</td><td class="NLM_def"><p class="first last">not determined</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">Na<sub>2</sub>CO<sub>3</sub></td><td class="NLM_def"><p class="first last">sodium carbonate</p></td></tr><tr><td class="NLM_term">PdCl<sub>2</sub>(dppf)<sub>2</sub></td><td class="NLM_def"><p class="first last">1,1′-Bis(diphenylphosphino)ferrocene]palladium(II) dichloride</p></td></tr><tr><td class="NLM_term">Pd<sub>2</sub>dba<sub>3</sub></td><td class="NLM_def"><p class="first last">Tris(dibenzylideneacetone)-dipalladium(0)</p></td></tr><tr><td class="NLM_term">Pd(OAc)<sub>2</sub></td><td class="NLM_def"><p class="first last">palladium acetate</p></td></tr><tr><td class="NLM_term">PhNTf<sub>2</sub></td><td class="NLM_def"><p class="first last"><i>N</i>-phenylbis(trifluoromethanesulfonimide)</p></td></tr><tr><td class="NLM_term">PIEDA</td><td class="NLM_def"><p class="first last">pair interaction energy decomposition analysis</p></td></tr><tr><td class="NLM_term">PIF</td><td class="NLM_def"><p class="first last">3-phosphoinositide-dependent protein kinase-interacting fragment</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PKA</td><td class="NLM_def"><p class="first last">protein kinase A</p></td></tr><tr><td class="NLM_term">PKC</td><td class="NLM_def"><p class="first last">protein kinase C</p></td></tr><tr><td class="NLM_term">PKD</td><td class="NLM_def"><p class="first last">protein kinase D</p></td></tr><tr><td class="NLM_term">p.o.</td><td class="NLM_def"><p class="first last">orally administration</p></td></tr><tr><td class="NLM_term">RA</td><td class="NLM_def"><p class="first last">rheumatoid arthritis</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean square deviation</p></td></tr><tr><td class="NLM_term">RPMI</td><td class="NLM_def"><p class="first last">Roswell Park Memorial Institute</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SPHOS</td><td class="NLM_def"><p class="first last">dicyclohexyl(2′,6′-dimethoxy-[1,1′-biphenyl]-2-yl)phosphine</p></td></tr><tr><td class="NLM_term"><sup><i>t</i></sup>BuONa</td><td class="NLM_def"><p class="first last">sodium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term"><sup><i>t</i></sup>BuONO</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl nitrite</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TMS</td><td class="NLM_def"><p class="first last">tetramethylsilane</p></td></tr><tr><td class="NLM_term">TNFα</td><td class="NLM_def"><p class="first last">tumor necrosis factor α</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultra performance liquid chromatography</p></td></tr><tr><td class="NLM_term">Vd</td><td class="NLM_def"><p class="first last">volume of distribution</p></td></tr><tr><td class="NLM_term">XPHOS</td><td class="NLM_def"><p class="first last">2-dicyclohexylphosphino-2′,4′,6′-triisopropylbiphenyl</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. C.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C: poised to signal</span>. <i>Am. J. Physiol. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">E395</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1152/ajpendo.00477.2009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1152%2Fajpendo.00477.2009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19934406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A280%3ADC%252BC3c7itVWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2010&pages=E395-402&author=A.+C.+Newton&title=Protein+kinase+C%3A+poised+to+signal&doi=10.1152%2Fajpendo.00477.2009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C: poised to signal</span></div><div class="casAuthors">Newton Alexandra C</div><div class="citationInfo"><span class="NLM_cas:title">American journal of physiology. Endocrinology and metabolism</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">E395-402</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nestled at the tip of a branch of the kinome, protein kinase C (PKC) family members are poised to transduce signals emanating from the cell surface.  Cell membranes provide the platform for PKC function, supporting the maturation of PKC through phosphorylation, its allosteric activation by binding specific lipids, and, ultimately, promoting the downregulation of the enzyme.  These regulatory mechanisms precisely control the level of signaling-competent PKC in the cell.  Disruption of this regulation results in pathophysiological states, most notably cancer, where PKC levels are often grossly altered.  This review introduces the PKC family and then focuses on recent advances in understanding the cellular regulation of its diacylglycerol-regulated members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKzc-AXy7rwI2y4bB4mnvYfW6udTcc2ea-1IarSTqT9Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7itVWrug%253D%253D&md5=4b8a3dfcc65370383ac6fffe0c54db00</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1152%2Fajpendo.00477.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpendo.00477.2009%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DA.%2BC.%26atitle%3DProtein%2520kinase%2520C%253A%2520poised%2520to%2520signal%26jtitle%3DAm.%2520J.%2520Physiol.%2520Endocrinol.%2520Metab.%26date%3D2010%26volume%3D298%26spage%3DE395%26epage%3D402%26doi%3D10.1152%2Fajpendo.00477.2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Meco, M. T.</span></span> <span> </span><span class="NLM_article-title">PKCζ at the crossroad of NF-κB and Jak1/Stat6 signaling pathways</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">702</span>, <span class="refDoi"> DOI: 10.1038/sj.cdd.4401823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1038%2Fsj.cdd.4401823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=16322752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFWisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=702&author=J.+Moscatauthor=P.+Rennertauthor=M.+T.+Diaz-Meco&title=PKC%CE%B6+at+the+crossroad+of+NF-%CE%BAB+and+Jak1%2FStat6+signaling+pathways&doi=10.1038%2Fsj.cdd.4401823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PKCζ at the crossroad of NF-κB and Jak1/Stat6 signaling pathways</span></div><div class="casAuthors">Moscat, J.; Rennert, P.; Diaz-Meco, M. T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death and Differentiation</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">702-711</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The atypical protein kinase C (PKC) isoforms (aPKC) have been implicated in the regulation of a no. of essential signaling events.  Early studies using dominant-neg. mutants suggested that they are important intermediaries in the activation of the canonical nuclear factor (NF)-κB pathway.  More recent data using knockout mice genetically demonstrate that in fact the PKCζ isoform is essential for the adequate activation of this cascade both upstream and downstream the IκB kinase complex.  In this review, we summarize the mechanistic details whereby the aPKC pathway regulates important cellular functions and how this is achieved by the ability of these kinases to interact with different protein regulators and adapters, as well as to impinge in NF-κB-independent signaling cascades such as the Janus kinase-1/signal transducer and activator of transcription 6 system, which plays a crit. role in T-cell-mediated hepatitis and asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptr6LpwzLTlLVg90H21EOLACvtfcHk0lhaA2lpc7qRGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFWisLk%253D&md5=54d1fc85832e875c800cb54180af1a48</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.cdd.4401823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.cdd.4401823%26sid%3Dliteratum%253Aachs%26aulast%3DMoscat%26aufirst%3DJ.%26aulast%3DRennert%26aufirst%3DP.%26aulast%3DDiaz-Meco%26aufirst%3DM.%2BT.%26atitle%3DPKC%25CE%25B6%2520at%2520the%2520crossroad%2520of%2520NF-%25CE%25BAB%2520and%2520Jak1%252FStat6%2520signaling%2520pathways%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2006%26volume%3D13%26spage%3D702%26doi%3D10.1038%2Fsj.cdd.4401823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Meco, M. T.</span></span> <span> </span><span class="NLM_article-title">The atypical PKCs in inflammation: NF-κB and beyond</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>246</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1111/j.1600-065X.2012.01093.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1111%2Fj.1600-065X.2012.01093.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=22435553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVyku73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=246&publication_year=2012&pages=154-167&author=J.+Moscatauthor=M.+T.+Diaz-Meco&title=The+atypical+PKCs+in+inflammation%3A+NF-%CE%BAB+and+beyond&doi=10.1111%2Fj.1600-065X.2012.01093.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The atypical PKCs in inflammation: NF-κB and beyond</span></div><div class="casAuthors">Diaz-Meco, Maria T.; Moscat, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">246</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">154-167, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  From the very early days of nuclear factor-κB (NF-κB) research, it was recognized that different protein kinase C (PKC) isoforms might be involved in the activation of NF-κB.  Pharmacol. tools and pseudosubstrate inhibitors suggested that these kinases play a role in this important inflammatory and survival pathway; however, it was the anal. of several genetic mouse knockout models that revealed the complexity and interrelations between the different components of the PB1 network in several cellular functions, including T-cell biol., bone homeostasis, inflammation assocd. with the metabolic syndrome, and cancer.  These studies unveiled, for example, the crit. role of PKCζ as a pos. regulator of NF-κB through the regulation of RelA but also its inflammatory suppressor activities through the regulation of the interleukin-4 signaling cascade.  This observation is of relevance in T cells, where p62, PKCζ, PKCλ/ι, and NBR1 establish a mesh of interactions that culminate in the regulation of T-cell effector responses through the modulation of T-cell polarity.  Many questions remain to be answered, not just from the point of view of the implication for NF-κB activation but also with regard to the in vivo interplay between these pathways in pathophysiol. processes like obesity and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUFcmnITPvHbVg90H21EOLACvtfcHk0lhaA2lpc7qRGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVyku73E&md5=9ee6c4f3a536bdab6f8de6e11002751a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-065X.2012.01093.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-065X.2012.01093.x%26sid%3Dliteratum%253Aachs%26aulast%3DMoscat%26aufirst%3DJ.%26aulast%3DDiaz-Meco%26aufirst%3DM.%2BT.%26atitle%3DThe%2520atypical%2520PKCs%2520in%2520inflammation%253A%2520NF-%25CE%25BAB%2520and%2520beyond%26jtitle%3DImmunol.%2520Rev.%26date%3D2012%26volume%3D246%26spage%3D154%26epage%3D167%26doi%3D10.1111%2Fj.1600-065X.2012.01093.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chida, K.</span></span> <span> </span><span class="NLM_article-title">Protein kinase Cζ (PKCζ): activation mechanisms and cellular functions</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1093/jb/mvg017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1093%2Fjb%2Fmvg017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=12761192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlOkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2003&pages=1-7&author=T.+Hiraiauthor=K.+Chida&title=Protein+kinase+C%CE%B6+%28PKC%CE%B6%29%3A+activation+mechanisms+and+cellular+functions&doi=10.1093%2Fjb%2Fmvg017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase Cξ (PKCξ): Activation mechanisms and cellular functions</span></div><div class="casAuthors">Hirai, Takaaki; Chida, Kazuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinase C isoform ξ (PKC-ξ) is a member of the atypical PKC subfamily and has been widely implicated in the regulation of cellular functions.  Increasing evidence from studies using in vitro and in vivo systems points to PKC-ξ as a key regulator of crit. intracellular signaling pathways induced by various extracellular stimuli.  The major activation pathway of PKC-ξ depends on phosphatidylinositol (PI)-3,4,5-trisphosphate (PIP3), which is mainly produced by PI-3 kinase.  3'-PI-dependent protein kinase 1, which binds with high affinity to PIP3, phosphorylates and activates PKC-ξ.  Many studies have demonstrated the involvement of PKC-ξ in the MAP kinase cascade, transcriptional factor NFκB activation, ribosomal protein S6 kinase signaling, and cell polarity.  An important mol. event in a cell is the assocn. of PKC-ξ with other signaling mols., as well as scaffold proteins, to form large complexes that regulate their pathways.  The understanding of the mechanisms underlying PKC-ξ-mediated control of intracellular signaling is beginning to provide important insights into the roles of PKC-ξ in various cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRc5gPsczr5rVg90H21EOLACvtfcHk0lgINDNo6R13Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlOkurY%253D&md5=fb6bf908b9d79feb37cfd1e8d21e2ea8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvg017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvg017%26sid%3Dliteratum%253Aachs%26aulast%3DHirai%26aufirst%3DT.%26aulast%3DChida%26aufirst%3DK.%26atitle%3DProtein%2520kinase%2520C%25CE%25B6%2520%2528PKC%25CE%25B6%2529%253A%2520activation%2520mechanisms%2520and%2520cellular%2520functions%26jtitle%3DJ.%2520Biochem.%26date%3D2003%26volume%3D133%26spage%3D1%26epage%3D7%26doi%3D10.1093%2Fjb%2Fmvg017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.-D.</span></span> <span> </span><span class="NLM_article-title">Effect of SNPs in protein kinase C ζ gene on gene expression in the reporter gene detection system</span>. <i>World. J. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2360</span>, <span class="refDoi"> DOI: 10.3748/wjg.v10.i16.2357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.3748%2Fwjg.v10.i16.2357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=15285019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2cXns1Gmtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=2357-2360&author=Z.+Liuauthor=H.-X.+Sunauthor=Y.-W.+Zhangauthor=Y.-F.+Liauthor=J.+Zuoauthor=Y.+Mengauthor=F.-D.+Fang&title=Effect+of+SNPs+in+protein+kinase+C+%CE%B6+gene+on+gene+expression+in+the+reporter+gene+detection+system&doi=10.3748%2Fwjg.v10.i16.2357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of SNPs in protein kinase Cz gene on gene expression in the reporter gene detection system</span></div><div class="casAuthors">Liu, Zhuo; Sun, Hong-Xia; Zhang, Yong-Wei; Li, Yun-Feng; Zuo, Jin; Meng, Yan; Fang, Fu-De</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2357-2360</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">World Journal of Gastroenterology</span>)
        </div><div class="casAbstract">The objective of this study was to investigated the effects of the SNPs (rs411021, rs436045, rs427811, rs385039 and rs809912) found in protein kinase Cζ gene on expression and further identify the susceptibility genes of type 2 diabetes.  Ten allele fragments (49 bp each) were synthesized according to the 5 SNPs mentioned above.  These fragments were cloned into luciferase reporter gene vector and then transfected into HepG2 cells.  The activity of the luciferase was assayed.  Effects of the SNPs on RNA splicing were analyzed by bioinformatics.  Rs427811T allele and rs809912G allele enhanced the activity of the reporter gene expression.  None of the 5 SNPs affected RNA splicing.  The authors concluded that SNPs in protein kinase Cζ (PKCZ) gene probably play a role in the susceptibility to type 2 diabetes by affecting the expression level of the relevant genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZXYiiJKugC7Vg90H21EOLACvtfcHk0lgINDNo6R13Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXns1Gmtb4%253D&md5=d139770198e6e8b1c6f9b1245dd836ea</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v10.i16.2357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v10.i16.2357%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DH.-X.%26aulast%3DZhang%26aufirst%3DY.-W.%26aulast%3DLi%26aufirst%3DY.-F.%26aulast%3DZuo%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DF.-D.%26atitle%3DEffect%2520of%2520SNPs%2520in%2520protein%2520kinase%2520C%2520%25CE%25B6%2520gene%2520on%2520gene%2520expression%2520in%2520the%2520reporter%2520gene%2520detection%2520system%26jtitle%3DWorld.%2520J.%2520Gastroenterol.%26date%3D2004%26volume%3D10%26spage%3D2357%26epage%3D2360%26doi%3D10.3748%2Fwjg.v10.i16.2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seto, K. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrulis, I. L.</span></span> <span> </span><span class="NLM_article-title">Atypical protein kinase C ζ: potential player in cell survival and cell migration of ovarian cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">e0123528</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0123528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1371%2Fjournal.pone.0123528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=25874946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OitL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0123528&issue=4&author=K.+K.+Y.+Setoauthor=I.+L.+Andrulis&title=Atypical+protein+kinase+C+%CE%B6%3A+potential+player+in+cell+survival+and+cell+migration+of+ovarian+cancer&doi=10.1371%2Fjournal.pone.0123528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Atypical protein kinase C Zeta: potential player in cell survival and cell migration of ovarian cancer</span></div><div class="casAuthors">Seto, Kelly K. Y.; Andrulis, Irene L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0123528/1-e0123528/1</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Ovarian cancer is one of the most aggressive gynaecol. cancers, thus understanding the different biol. pathways involved in ovarian cancer progression is important in identifying potential therapeutic targets for the disease.  The aim of this study was to investigate the potential roles of Protein Kinase C Zeta (PRKCZ) in ovarian cancer.  The atypical protein kinase C isoform, PRKCZ, is involved in the control of various signalling processes including cell proliferation, cell survival, and cell motility, all of which are important for cancer development and progression.  Herein, we observe a significant increase in cell survival upon PRKCZ over-expression in SKOV3 ovarian cancer cells; addnl., when the cells are treated with small interference RNA (siRNA) targeting PRKCZ, the motility of SKOV3 cells decreased.  Furthermore, we demonstrate that over-expression of PRKCZ results in gene and/or protein expression alterations of insulin-like growth factor 1 receptor (IGF1R) and integrin beta 3 (ITGB3) in SKOV3 and OVCAR3 cells.  Collectively, our study describes PRKCZ as a potential regulatory component of the IGF1R and ITGB3 pathways and suggests that it may play crit. roles in ovarian tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZycReUEmvQLVg90H21EOLACvtfcHk0lgINDNo6R13Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OitL%252FL&md5=79e27ecb2374dcddfa3f999a72818594</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0123528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0123528%26sid%3Dliteratum%253Aachs%26aulast%3DSeto%26aufirst%3DK.%2BK.%2BY.%26aulast%3DAndrulis%26aufirst%3DI.%2BL.%26atitle%3DAtypical%2520protein%2520kinase%2520C%2520%25CE%25B6%253A%2520potential%2520player%2520in%2520cell%2520survival%2520and%2520cell%2520migration%2520of%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D4%26spage%3De0123528%26doi%3D10.1371%2Fjournal.pone.0123528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leitges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camacho, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz Meco, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennert, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span> <span> </span><span class="NLM_article-title">Targeted disruption of the PKCζ gene results in the impairment of the NF-κB pathway</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)00361-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS1097-2765%2801%2900361-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=11684013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3MXotlagur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=771-780&author=M.+Leitgesauthor=L.+Sanzauthor=P.+Martinauthor=A.+Duranauthor=U.+Braunauthor=J.+F.+Garciaauthor=F.+Camachoauthor=M.+T.+Diaz+Mecoauthor=P.+D.+Rennertauthor=J.+Moscat&title=Targeted+disruption+of+the+PKC%CE%B6+gene+results+in+the+impairment+of+the+NF-%CE%BAB+pathway&doi=10.1016%2FS1097-2765%2801%2900361-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted disruption of the ζPKC gene results in the impairment of the NF-κB pathway</span></div><div class="casAuthors">Leitges, Michael; Sanz, Laura; Martin, Pilar; Duran, Angeles; Braun, Uschi; Garcia, Juan F.; Camacho, Fanny; Diaz-Meco, Maria T.; Rennert, Paul D.; Moscat, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">771-780</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Here the authors have addressed the role that ζPKC plays in NF-κB activation using mice in which this kinase was inactivated by homologous recombination.  These mice, although grossly normal, showed phenotypic alterations in secondary lymphoid organs reminiscent of those of the TNF receptor-1 and of the lymphotoxin-β receptor gene-deficient mice.  The lack of ζPKC in embryonic fibroblasts (EFs) severely impairs κB-dependent transcriptional activity as well as cytokine-induced phosphorylation of p65.  Also, a cytokine-inducible interaction of ζPKC with p65 was detected which requires the previous degrdn. of IκB.  Although in ζPKC-/- EFs this kinase is not necessary for IKK activation, in lung, which abundantly expresses ζPKC, IKK activation is inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod6cqcc6PgOrVg90H21EOLACvtfcHk0lhVg29eMUagpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXotlagur0%253D&md5=b8d6363ac857559db48a783ccdf6ea1e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900361-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900361-6%26sid%3Dliteratum%253Aachs%26aulast%3DLeitges%26aufirst%3DM.%26aulast%3DSanz%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DP.%26aulast%3DDuran%26aufirst%3DA.%26aulast%3DBraun%26aufirst%3DU.%26aulast%3DGarcia%26aufirst%3DJ.%2BF.%26aulast%3DCamacho%26aufirst%3DF.%26aulast%3DDiaz%2BMeco%26aufirst%3DM.%2BT.%26aulast%3DRennert%26aufirst%3DP.%2BD.%26aulast%3DMoscat%26aufirst%3DJ.%26atitle%3DTargeted%2520disruption%2520of%2520the%2520PKC%25CE%25B6%2520gene%2520results%2520in%2520the%2520impairment%2520of%2520the%2520NF-%25CE%25BAB%2520pathway%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D8%26spage%3D771%26epage%3D780%26doi%3D10.1016%2FS1097-2765%2801%2900361-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee-Rivera, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carranza-Perez, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Colome, A. M.</span></span> <span> </span><span class="NLM_article-title">PKC-ζ regulates thrombin-induced proliferation of human müller glial cells</span>. <i>Invest. Ophthalmol. Visual Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3769</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1167/iovs.16-19574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1167%2Fiovs.16-19574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=27421067" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFSnsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2016&pages=3769-3779&author=I.+Lee-Riveraauthor=E.+Lopezauthor=M.+G.+Carranza-Perezauthor=A.+M.+Lopez-Colome&title=PKC-%CE%B6+regulates+thrombin-induced+proliferation+of+human+m%C3%BCller+glial+cells&doi=10.1167%2Fiovs.16-19574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">PKC-ζ regulates thrombin-induced proliferation of human Muller glial cells</span></div><div class="casAuthors">Lee-Rivera, Irene; Lopez, Edith; Carranza-Perez, Miriam Gabriela; Lopez-Colome, Ana Maria</div><div class="citationInfo"><span class="NLM_cas:title">Investigative Ophthalmology & Visual Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3769-3779</span>CODEN:
                <span class="NLM_cas:coden">IOVSDA</span>;
        ISSN:<span class="NLM_cas:issn">1552-5783</span>.
    
            (<span class="NLM_cas:orgname">Association for Research in Vision and Ophthalmology</span>)
        </div><div class="casAbstract">PURPOSE. To investigate the effect of thrombin on the proliferation of human Muller glial cells (MCs) and define the possible signaling mechanisms involved in this process.  METHODS. Protease-activated receptor (PARs 1-4) expression was analyzed using RT-PCR and Western blot in the MIO-M1 Muller cell line (MC).  Muller cell proliferation was assessed by the MTS redn. method.  Wound healing and immunoreactivity to Ki67 antigen were used to dissoc. proliferation and migration.  Cell migration was examd. using transwell migration assays.  The involvement of extracellular signal-regulated kinase (ERK1/2) phosphorylation/activation in thrombin-induced human MC proliferation was detd. by Western blot.  Intracellular pathways involved in ERK1/2 activation were analyzed by pharmacol. inhibition.  RESULTS. We first demonstrated that human MCs express PARs 1 to 4.  Our results show that thrombin dose-dependently stimulates MC proliferation by 44%, with a calcd. Ec50 of 0.86 nM.  Muller cell maximal proliferation required sustained thrombin treatment for 72 h, in contrast to our previous findings in RPE cells showing maximal thrombin-induced proliferation at 24-h stimulation.  We demonstrate that thrombin induces MC cell proliferation through the Ras-independent activation of the Raf/MEK/ERK cascade, under the control of protein kinase C (PKC)-ζ.  CONCLUSIONS. The breakdown of blood-retina barrier (BRB) exposes MCs to thrombin contained in serum.  Our findings further strengthen the crit. involvement of thrombin in the development of proliferative retinopathies and may provide pharmacol. targets for the prevention or treatment of these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxhd3tSVgPm7Vg90H21EOLACvtfcHk0lhVg29eMUagpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFSnsLjE&md5=996f466b10c7850241a9d4c3cbe3ec83</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1167%2Fiovs.16-19574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1167%252Fiovs.16-19574%26sid%3Dliteratum%253Aachs%26aulast%3DLee-Rivera%26aufirst%3DI.%26aulast%3DLopez%26aufirst%3DE.%26aulast%3DCarranza-Perez%26aufirst%3DM.%2BG.%26aulast%3DLopez-Colome%26aufirst%3DA.%2BM.%26atitle%3DPKC-%25CE%25B6%2520regulates%2520thrombin-induced%2520proliferation%2520of%2520human%2520m%25C3%25BCller%2520glial%2520cells%26jtitle%3DInvest.%2520Ophthalmol.%2520Visual%2520Sci.%26date%3D2016%26volume%3D57%26spage%3D3769%26epage%3D3779%26doi%3D10.1167%2Fiovs.16-19574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D.</span></span> <span> </span><span class="NLM_article-title">PKCζ as a promising therapeutic target for TNFα-induced inflammatory disorders in chronic cutaneous wounds</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1335</span>– <span class="NLM_lpage">1346</span>, <span class="refDoi"> DOI: 10.3892/ijmm.2017.3144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.3892%2Fijmm.2017.3144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=28949382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOrurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2017&pages=1335-1346&author=J.+Zhangauthor=X.+Yangauthor=H.+Wangauthor=B.+Zhaoauthor=X.+Wuauthor=L.+Suauthor=S.+Xieauthor=Y.+Wangauthor=J.+Liauthor=J.+Liuauthor=M.+Liuauthor=F.+Hanauthor=T.+Heauthor=W.+Zhangauthor=K.+Taoauthor=D.+Hu&title=PKC%CE%B6+as+a+promising+therapeutic+target+for+TNF%CE%B1-induced+inflammatory+disorders+in+chronic+cutaneous+wounds&doi=10.3892%2Fijmm.2017.3144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">PKCζ as a promising therapeutic target for TNFα-induced inflammatory disorders in chronic cutaneous wounds</span></div><div class="casAuthors">Zhang, Jian; Yang, Xuekang; Wang, Hongtao; Zhao, Bin; Wu, Xue; Su, Linlin; Xie, Songtao; Wang, Yunchuan; Li, Jun; Liu, Jiaqi; Liu, Mengdong; Han, Fu; He, Ting; Zhang, Wei; Tao, Ke; Hu, Dahai</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1335-1346</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Protein kinase Cζ (PKCζ) is a member of the atypical protein kinase C family.  Its roles in macrophages or skin-resident keratinocytes have not been fully evaluated.  In this study, we provide evidence that PKCζ mediates lipopolysaccharide (LPS)-induced tumor necrosis factor α (TNFα) gene expression in the mouse macrophage cell line, RAW264.7.  TNFα has been proven to be one of the main culprits of chronic wounds and impaired acute wounds, which are characterized by excessive inflammation, enhanced proteolysis and reduced matrix deposition.  Among the multiple effects of TNFα on keratinocytes, the induction of chemokines which are indispensable factors involved in the massive infiltration of various inflammatory cells into skin lesions serves as a crucial mechanism.  In the present study, we found that PKCζ inhibitor or its specific siRNA inhibited the TNFα-induced upregulation in the levels of the chemokines, interleukin (IL)-8, monocyte chemotactic protein-1 (MCP-1) and intercellular cell adhesion mol.-1 (ICAM-1) in HaCaT keratinocytes.  Moreover, under a disrupted inflammatory environment, activated keratinocytes can synthesize large amts. of matrix metalloproteinases (MMP), which has a neg. effect on tissue remodeling.  We discovered that TNFα promoted the expression of MMP9 in a PKCζ-dependent manner.  Further expts. revealed that nuclear factor-κB (NF-κB) was a key downstream mol. of PKCζ.  In addn., as shown in vitro, PKCζ was not involved in the TNFα-induced decrease in HaCaT cell migration and proliferation.  In vivo expts. demonstrated that TNFα-induced wound closure impairment and inflammatory disorders were significantly attenuated in the PKCζ inhibitor group.  On the whole, our findings suggest that PKCζ is a crucial regulator in LPS- or TNFα-induced inflammatory responses in RAW264.7 cells and HaCaT keratinocytes, and that PKCζ/ NF-κB signaling may be a potential target for interventional therapy for TNFα-induced skin inflammatory injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYMbUGV_L5DrVg90H21EOLACvtfcHk0lhVg29eMUagpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOrurfK&md5=b65487ffba6929137a5364ce834da32f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2017.3144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2017.3144%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTao%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DD.%26atitle%3DPKC%25CE%25B6%2520as%2520a%2520promising%2520therapeutic%2520target%2520for%2520TNF%25CE%25B1-induced%2520inflammatory%2520disorders%2520in%2520chronic%2520cutaneous%2520wounds%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2017%26volume%3D40%26spage%3D1335%26epage%3D1346%26doi%3D10.3892%2Fijmm.2017.3144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anisul Islam, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolatos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolatos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acevedo-Duncan, M.</span></span> <span> </span><span class="NLM_article-title">Analysis of PKC-ζ protein levels in normal and malignant breast tissue subtypes</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1546</span>, <span class="refDoi"> DOI: 10.3892/ol.2018.9792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.3892%2Fol.2018.9792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=30675210" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1537-1546&author=T.+Smalleyauthor=S.+M.+Anisul+Islamauthor=C.+Apostolatosauthor=A.+Apostolatosauthor=M.+Acevedo-Duncan&title=Analysis+of+PKC-%CE%B6+protein+levels+in+normal+and+malignant+breast+tissue+subtypes&doi=10.3892%2Fol.2018.9792"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3892%2Fol.2018.9792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2018.9792%26sid%3Dliteratum%253Aachs%26aulast%3DSmalley%26aufirst%3DT.%26aulast%3DAnisul%2BIslam%26aufirst%3DS.%2BM.%26aulast%3DApostolatos%26aufirst%3DC.%26aulast%3DApostolatos%26aufirst%3DA.%26aulast%3DAcevedo-Duncan%26aufirst%3DM.%26atitle%3DAnalysis%2520of%2520PKC-%25CE%25B6%2520protein%2520levels%2520in%2520normal%2520and%2520malignant%2520breast%2520tissue%2520subtypes%26jtitle%3DOncol.%2520Lett.%26date%3D2018%26volume%3D17%26spage%3D1537%26epage%3D1546%26doi%3D10.3892%2Fol.2018.9792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anrather, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csizmadia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, H.</span></span> <span> </span><span class="NLM_article-title">Regulation of NF-κB RelA phosphorylation and transcriptional activity by p21(ras) and protein kinase C ζ in primary endothelial cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">13594</span>– <span class="NLM_lpage">13603</span>, <span class="refDoi"> DOI: 10.1074/jbc.274.19.13594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.274.19.13594" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10224130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFOlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=13594-13603&author=J.+Anratherauthor=V.+Csizmadiaauthor=M.+P.+Soaresauthor=H.+Winkler&title=Regulation+of+NF-%CE%BAB+RelA+phosphorylation+and+transcriptional+activity+by+p21%28ras%29+and+protein+kinase+C+%CE%B6+in+primary+endothelial+cells&doi=10.1074%2Fjbc.274.19.13594"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of NF-κB RelA phosphorylation and transcriptional activity by p21ras and protein kinase Cζ in primary endothelial cells</span></div><div class="casAuthors">Anrather, Josef; Csizmadia, Vilmos; Soares, Miguel P.; Winkler, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">13594-13603</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activity of the transcription factor NF-κB is thought to be regulated mainly through cytoplasmic retention by IκB mols.  Here we present evidence of a second mechanism of regulation acting on NF-κB after release from IκB.  In endothelial cells this mechanism involves phosphorylation of the RelA subunit of NF-κB through a pathway involving activation of protein kinase Cζ (PKCζ) and p21ras.  We show that transcriptional activity of RelA is dependent on phosphorylation of the N-terminal Rel homol. domain but not the C-terminal transactivation domain.  Inhibition of phosphorylation by dominant neg. mutants of PKCζ or p21ras results in loss of RelA transcriptional activity without interfering with DNA binding.  Raf/MEK, small GTPases, phosphatidylinositol 3-kinase, and stress-activated protein kinase pathways are not involved in this mechanism of regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp9UMgY6DG0bVg90H21EOLACvtfcHk0ljWvLXkwTJU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFOlu7c%253D&md5=f8aef80c60b9099eadbdaabc6b26cf0e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.19.13594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.19.13594%26sid%3Dliteratum%253Aachs%26aulast%3DAnrather%26aufirst%3DJ.%26aulast%3DCsizmadia%26aufirst%3DV.%26aulast%3DSoares%26aufirst%3DM.%2BP.%26aulast%3DWinkler%26aufirst%3DH.%26atitle%3DRegulation%2520of%2520NF-%25CE%25BAB%2520RelA%2520phosphorylation%2520and%2520transcriptional%2520activity%2520by%2520p21%2528ras%2529%2520and%2520protein%2520kinase%2520C%2520%25CE%25B6%2520in%2520primary%2520endothelial%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D13594%26epage%3D13603%26doi%3D10.1074%2Fjbc.274.19.13594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San-Antonio, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fresno, M.</span></span> <span> </span><span class="NLM_article-title">Regulation of nuclear factor κB transactivation. Implication of phosphatidylinositol 3-kinase and protein kinase C ζ in c-Rel activation by tumor necrosis factor α</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">15840</span>– <span class="NLM_lpage">15849</span>, <span class="refDoi"> DOI: 10.1074/jbc.M011313200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.M011313200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=11278885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFaks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=15840-15849&author=A.+G.+Martinauthor=B.+San-Antonioauthor=M.+Fresno&title=Regulation+of+nuclear+factor+%CE%BAB+transactivation.+Implication+of+phosphatidylinositol+3-kinase+and+protein+kinase+C+%CE%B6+in+c-Rel+activation+by+tumor+necrosis+factor+%CE%B1&doi=10.1074%2Fjbc.M011313200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of nuclear factor κB transactivation. Implication of phosphatidylinositol 3-kinase and protein kinase C ξ in c-Rel activation by tumor necrosis factor α</span></div><div class="casAuthors">Martin, Angel G.; San-Antonio, Belen; Fresno, Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">15840-15849</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transactivation by c-Rel (nuclear factor κB) was dependent on phosphorylation of several serines in the transactivation domain, indicating that it is a phosphorylation-dependent Ser-rich domain.  By Ser → Ala mutational and deletion anal., we have identified two regions in this domain: C-terminal region (amino acids 540-588), which is required for basal activity; and the 422-540 region, which responds to external stimuli as tumor necrosis factor (TNF) α or phorbol myristate acetate plus ionomycin.  Ser from 454 to 473 were shown to be required for TNFα-induced activation, whereas Ser between 492 and 519 were required for phorbol myristate acetate plus ionomycin activation.  Phosphatidylinositol 3-kinase (PI3K) and protein kinase C (PKC) ξ were identified as downstream signaling mols. of TNFα-activation of c-Rel transactivating activity.  Interestingly, dominant neg. forms of PI3K inhibited PKCξ activation and dominant neg. PKCξ inhibited PI3K-mediated activation of c-Rel transactivating activity, indicating a cross-talk between both enzymes.  We have identified the crit. role of different Ser for PKCξ- and PI3K-mediated responses.  Interestingly, those c-Rel mutants not only did not respond to TNFα but also acted as dominant neg. forms of nuclear factor κB activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU3sRdCE9CkrVg90H21EOLACvtfcHk0ljWvLXkwTJU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFaks7c%253D&md5=be9bde9f594ea5e63df27957a5f6a171</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M011313200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M011313200%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DA.%2BG.%26aulast%3DSan-Antonio%26aufirst%3DB.%26aulast%3DFresno%26aufirst%3DM.%26atitle%3DRegulation%2520of%2520nuclear%2520factor%2520%25CE%25BAB%2520transactivation.%2520Implication%2520of%2520phosphatidylinositol%25203-kinase%2520and%2520protein%2520kinase%2520C%2520%25CE%25B6%2520in%2520c-Rel%2520activation%2520by%2520tumor%2520necrosis%2520factor%2520%25CE%25B1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D15840%26epage%3D15849%26doi%3D10.1074%2Fjbc.M011313200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaVallie, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chockalingam, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins-Racie, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keohan, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, M.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C ζ is up-regulated in osteoarthritic cartilage and is required for activation of NF-κB by tumor necrosis factor and interleukin-1 in articular chondrocytes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">24124</span>– <span class="NLM_lpage">24137</span>, <span class="refDoi"> DOI: 10.1074/jbc.M601905200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.M601905200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=16798739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1eqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=24124-24137&author=E.+R.+LaVallieauthor=P.+S.+Chockalingamauthor=L.+A.+Collins-Racieauthor=B.+A.+Freemanauthor=C.+C.+Keohanauthor=M.+Leitgesauthor=A.+J.+Dornerauthor=E.+A.+Morrisauthor=M.+K.+Majumdarauthor=M.+Arai&title=Protein+kinase+C+%CE%B6+is+up-regulated+in+osteoarthritic+cartilage+and+is+required+for+activation+of+NF-%CE%BAB+by+tumor+necrosis+factor+and+interleukin-1+in+articular+chondrocytes&doi=10.1074%2Fjbc.M601905200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Cζ Is Up-regulated in Osteoarthritic Cartilage and Is Required for Activation of NF-κB by Tumor Necrosis Factor and Interleukin-1 in Articular Chondrocytes</span></div><div class="casAuthors">LaVallie, Edward R.; Chockalingam, Priya S.; Collins-Racie, Lisa A.; Freeman, Bethany A.; Keohan, Cristin C.; Leitges, Michael; Dorner, Andrew J.; Morris, Elisabeth A.; Majumdar, Manas K.; Arai, Maya</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">24124-24137</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein kinase Cζ (PKCζ) is an intracellular serine/threonine protein kinase that has been implicated in the signaling pathways for certain inflammatory cytokines, including interleukin-1 (IL-1) and tumor necrosis factor α (TNF-α), in some cell types.  A study of gene expression in articular chondrocytes from osteoarthritis (OA) patients revealed that PKCζ is transcriptionally up-regulated in human OA articular cartilage clin. samples.  This finding led to the hypothesis that PKCζ may be an important signaling component of cytokine-mediated cartilage matrix destruction in articular chondrocytes, believed to be an underlying factor in the pathophysiol. of OA.  IL-1 treatment of chondrocytes in culture resulted in rapidly increased phosphorylation of PKCζ, implicating PKCζ activation in the signaling pathway.  Chondrocyte cell-based assays were used to evaluate the contribution of PKCζ activity in NF-κB activation and extracellular matrix degrdn. mediated by IL-1, TNF, or sphingomyelinase.  In primary chondrocytes, IL-1 and TNF-α caused an increase in NF-κB activity resulting in induction of aggrecanase-1 and aggrecanase-2 expression, with consequent increased proteoglycan degrdn.  This effect was blocked by the pan-specific PKC inhibitors RO 31-8220 and bisindolylmaleimide I, partially blocked by Goe 6976, and was unaffected by the PKCζ-sparing inhibitor calphostin C.  A cell-permeable PKCζ pseudosubstrate peptide inhibitor was capable of blocking TNF and IL-1-mediated NF-κB activation and proteoglycan degrdn. in chondrocyte pellet cultures.  In addn., overexpression of a dominant neg. PKCζ protein effectively prevented cytokine-mediated NF-κB activation in primary chondrocytes.  These data implicate PKCζ as a necessary component of the IL-1 and TNF signaling pathways in chondrocytes that result in catabolic destruction of extracellular matrix proteins in osteoarthritic cartilage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNI_sLNyzO3rVg90H21EOLACvtfcHk0ljWvLXkwTJU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1eqsro%253D&md5=57ad065d1c9a5089772e9f67d89d0e0e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M601905200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M601905200%26sid%3Dliteratum%253Aachs%26aulast%3DLaVallie%26aufirst%3DE.%2BR.%26aulast%3DChockalingam%26aufirst%3DP.%2BS.%26aulast%3DCollins-Racie%26aufirst%3DL.%2BA.%26aulast%3DFreeman%26aufirst%3DB.%2BA.%26aulast%3DKeohan%26aufirst%3DC.%2BC.%26aulast%3DLeitges%26aufirst%3DM.%26aulast%3DDorner%26aufirst%3DA.%2BJ.%26aulast%3DMorris%26aufirst%3DE.%2BA.%26aulast%3DMajumdar%26aufirst%3DM.%2BK.%26aulast%3DArai%26aufirst%3DM.%26atitle%3DProtein%2520kinase%2520C%2520%25CE%25B6%2520is%2520up-regulated%2520in%2520osteoarthritic%2520cartilage%2520and%2520is%2520required%2520for%2520activation%2520of%2520NF-%25CE%25BAB%2520by%2520tumor%2520necrosis%2520factor%2520and%2520interleukin-1%2520in%2520articular%2520chondrocytes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D24124%26epage%3D24137%26doi%3D10.1074%2Fjbc.M601905200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Meco, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span> <span> </span><span class="NLM_article-title">Essential role of RelA Ser311 phosphorylation by PKC in NF-B transcriptional activation</span>. <i>EMBO. J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3910</span>– <span class="NLM_lpage">3918</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdg370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1093%2Femboj%2Fcdg370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=12881425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlsl2ksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=3910-3918&author=A.+Duranauthor=M.+T.+Diaz-Mecoauthor=J.+Moscat&title=Essential+role+of+RelA+Ser311+phosphorylation+by+PKC+in+NF-B+transcriptional+activation&doi=10.1093%2Femboj%2Fcdg370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of RelA Ser311 phosphorylation by ζPKC in NF-κB transcriptional activation</span></div><div class="casAuthors">Duran, Angeles; Diaz-Meco, Maria T.; Moscat, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3910-3918</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The activation of the transcription factor NF-κB is central to the control of the cellular response triggered by many stimuli.  Once released from the inhibitory mol. IκB, NF-κB is translocated to the nucleus, and it has to be phosphorylated to activate transcription.  In ζ protein kinase C (PKC)-deficient cells, NF-κB is transcriptionally inactive and the phosphorylation of the RelA subunit in response to tumor necrosis factor (TNF-α) is severely impaired.  In vitro assays showed that ζPKC directly phosphorylates RelA.  Here we demonstrate that Ser311 accounts for ζPKC phosphorylation of RelA and that this site is phosphorylated in vivo in response to TNF-α.  Also, an inactivating mutation of that residue severely impairs RelA transcriptional activity, blocks its anti-apoptotic function and abrogates the interaction of RelA with the co-activator CBP as well as its recruitment, and that of RNA polymerase II (Pol II) with the interleukin-6 (IL-6) promoter.  The interaction of endogenous CBP with endogenous RelA is inhibited in ζPKC-/- cells, as well as the binding of Pol II to the IL-6 promoter.  These results demonstrate the mechanism whereby ζPKC regulates NF-κB activation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqSrIUcVaTcrVg90H21EOLACvtfcHk0li8wiNx3yUblQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlsl2ksrY%253D&md5=215ff3d929588df807b83ead61724272</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdg370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdg370%26sid%3Dliteratum%253Aachs%26aulast%3DDuran%26aufirst%3DA.%26aulast%3DDiaz-Meco%26aufirst%3DM.%2BT.%26aulast%3DMoscat%26aufirst%3DJ.%26atitle%3DEssential%2520role%2520of%2520RelA%2520Ser311%2520phosphorylation%2520by%2520PKC%2520in%2520NF-B%2520transcriptional%2520activation%26jtitle%3DEMBO.%2520J.%26date%3D2003%26volume%3D22%26spage%3D3910%26epage%3D3918%26doi%3D10.1093%2Femboj%2Fcdg370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Litherland, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catterall, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, A. D.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C isoforms ζ and iota mediate collagenase expression and cartilage destruction via STAT3- and ERK-dependent c-fos induction</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">22414</span>– <span class="NLM_lpage">22425</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.120121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.M110.120121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=20463008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1Wnt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=22414-22425&author=G.+J.+Litherlandauthor=M.+S.+Eliasauthor=W.+Huiauthor=C.+D.+Macdonaldauthor=J.+B.+Catterallauthor=M.+J.+Barterauthor=M.+J.+Farrenauthor=M.+Jeffersonauthor=A.+D.+Rowan&title=Protein+kinase+C+isoforms+%CE%B6+and+iota+mediate+collagenase+expression+and+cartilage+destruction+via+STAT3-+and+ERK-dependent+c-fos+induction&doi=10.1074%2Fjbc.M110.120121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase C Isoforms ζ and ι Mediate Collagenase Expression and Cartilage Destruction via STAT3- and ERK-dependent c-fos Induction</span></div><div class="casAuthors">Litherland, Gary J.; Elias, Martina S.; Hui, Wang; MacDonald, Christopher D.; Catterall, Jonathon B.; Barter, Matt J.; Farren, Matthew J.; Jefferson, Matthew; Rowan, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">22414-22425</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein kinase C (PKC) signaling pathway is a major regulator of cellular functions and is implicated in pathologies involving extracellular matrix remodeling.  Inflammatory joint disease is characterized by excessive extracellular matrix catabolism, and here we assess the role of PKC in the induction of the collagenases, matrix metalloproteinase (MMP)-1 and MMP-13, in human chondrocytes by the potent cytokine stimulus interleukin-1 (IL-1) in combination with oncostatin M (OSM).  IL-1 + OSM-stimulated collagenolysis and gelatinase activity were ameliorated by pharmacol. PKC inhibition in bovine cartilage, as was collagenase gene induction in human chondrocytes.  Small interfering RNA-mediated silencing of PKC gene expression showed that both novel (nPKCδ, nPKCη) and atypical (aPKCζ, aPKCι) isoforms were involved in collagenase induction by IL-1.  However, MMP1 and MMP13 induction by IL-1 + OSM was inhibited only by aPKC silencing, suggesting that only atypical isoforms play a significant role in complex inflammatory milieus.  Silencing of either aPKC led to diminished IL-1 + OSM-dependent extracellular signal-regulated kinase (ERK) and signal transducer and activator of transcription (STAT) 3 phosphorylation, and c-fos expression.  STAT3 gene silencing or ERK pathway inhibition also resulted in loss of IL-1 + OSM-stimulated c-fos and collagenase expression.  Silencing of c-fos and c-jun expression was sufficient to abrogate IL-1 + OSM-stimulated collagenase gene induction, and overexpression of both c-fos and c-jun was sufficient to drive transcription from the MMP1 promoter in the absence of a stimulus.  Our data identify atypical PKC isoenzymes as STAT and ERK activators that mediate c-fos and collagenase expression during IL-1 + OSM synergy in human chondrocytes.  APKCs may constitute potential therapeutic targets for inflammatory joint diseases involving increased collagenase expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnN40-vsGhG7Vg90H21EOLACvtfcHk0li8wiNx3yUblQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1Wnt78%253D&md5=e80b96cafd40c4c158663bd8608cbdf7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.120121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.120121%26sid%3Dliteratum%253Aachs%26aulast%3DLitherland%26aufirst%3DG.%2BJ.%26aulast%3DElias%26aufirst%3DM.%2BS.%26aulast%3DHui%26aufirst%3DW.%26aulast%3DMacdonald%26aufirst%3DC.%2BD.%26aulast%3DCatterall%26aufirst%3DJ.%2BB.%26aulast%3DBarter%26aufirst%3DM.%2BJ.%26aulast%3DFarren%26aufirst%3DM.%2BJ.%26aulast%3DJefferson%26aufirst%3DM.%26aulast%3DRowan%26aufirst%3DA.%2BD.%26atitle%3DProtein%2520kinase%2520C%2520isoforms%2520%25CE%25B6%2520and%2520iota%2520mediate%2520collagenase%2520expression%2520and%2520cartilage%2520destruction%2520via%2520STAT3-%2520and%2520ERK-dependent%2520c-fos%2520induction%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D22414%26epage%3D22425%26doi%3D10.1074%2Fjbc.M110.120121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitson, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugni, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chockalingam, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concepcion, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, J. N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangalindan, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritacco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ireland, C. M.</span></span> <span> </span><span class="NLM_article-title">Spheciosterol sulfates, PKCζ inhibitors from a philippine sponge Spheciospongia sp</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi"> DOI: 10.1021/np8001628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np8001628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1cXntlaqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=1213-1217&author=E.+L.+Whitsonauthor=T.+S.+Bugniauthor=P.+S.+Chockalingamauthor=G.+P.+Concepcionauthor=M.+J.+Harperauthor=M.+Heauthor=J.+N.+A.+Hooperauthor=G.+C.+Mangalindanauthor=F.+Ritaccoauthor=C.+M.+Ireland&title=Spheciosterol+sulfates%2C+PKC%CE%B6+inhibitors+from+a+philippine+sponge+Spheciospongia+sp&doi=10.1021%2Fnp8001628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Spheciosterol Sulfates, PKCζ Inhibitors from a Philippine Sponge Spheciospongia sp.</span></div><div class="casAuthors">Whitson, Emily L.; Bugni, Tim S.; Chockalingam, Priya S.; Concepcion, Gisela P.; Harper, Mary Kay; He, Min; Hooper, John N. A.; Mangalindan, Gina C.; Ritacco, Frank; Ireland, Chris M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1217</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Three new sterol sulfates, spheciosterol sulfates A-C (I-III), and the known sterol sulfate topsentiasterol sulfate E (IV) have been isolated from the sponge Spheciospongia sp., collected in the Philippines.  Structures were assigned on the basis of extensive 1D and 2D NMR studies as well as anal. by HRESIMS.  Compds. I-IV inhibited PKCζ with IC50 values of 1.59, 0.53, 0.11, and 1.21 μM, resp.  In a cell-based assay, I-IV also inhibited NF-κB activation with EC50 values of 12-64 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8zpFO1Zuh2bVg90H21EOLACvtfcHk0ljBf6X9ESu1RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXntlaqsrc%253D&md5=369b82403363cb5e40c7f897a0cbbc99</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fnp8001628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp8001628%26sid%3Dliteratum%253Aachs%26aulast%3DWhitson%26aufirst%3DE.%2BL.%26aulast%3DBugni%26aufirst%3DT.%2BS.%26aulast%3DChockalingam%26aufirst%3DP.%2BS.%26aulast%3DConcepcion%26aufirst%3DG.%2BP.%26aulast%3DHarper%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHooper%26aufirst%3DJ.%2BN.%2BA.%26aulast%3DMangalindan%26aufirst%3DG.%2BC.%26aulast%3DRitacco%26aufirst%3DF.%26aulast%3DIreland%26aufirst%3DC.%2BM.%26atitle%3DSpheciosterol%2520sulfates%252C%2520PKC%25CE%25B6%2520inhibitors%2520from%2520a%2520philippine%2520sponge%2520Spheciospongia%2520sp%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2008%26volume%3D71%26spage%3D1213%26epage%3D1217%26doi%3D10.1021%2Fnp8001628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzmer, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reitz, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegeman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrightstone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christine, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Compton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span> <span> </span><span class="NLM_article-title">2-(6-Phenyl-1H-indazol-3-yl)-1H-benzo[d]imidazoles: design and synthesis of a potent and isoform selective PKC-ζ inhibitor</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">908</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.11.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.bmcl.2008.11.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19097791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtF2itL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=908-911&author=J.+I.+Trujilloauthor=J.+R.+Kieferauthor=W.+Huangauthor=A.+Thorarensenauthor=L.+Xingauthor=N.+L.+Caspersauthor=J.+E.+Dayauthor=K.+J.+Mathisauthor=K.+K.+Kretzmerauthor=B.+A.+Reitzauthor=R.+A.+Weinbergauthor=R.+A.+Stegemanauthor=A.+Wrightstoneauthor=L.+Christineauthor=R.+Comptonauthor=X.+Li&title=2-%286-Phenyl-1H-indazol-3-yl%29-1H-benzo%5Bd%5Dimidazoles%3A+design+and+synthesis+of+a+potent+and+isoform+selective+PKC-%CE%B6+inhibitor&doi=10.1016%2Fj.bmcl.2008.11.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">2-(6-Phenyl-1H-indazol-3-yl)-1H-benzo[d]imidazoles: Design and synthesis of a potent and isoform selective PKC-ζ inhibitor</span></div><div class="casAuthors">Trujillo, John I.; Kiefer, James R.; Huang, Wei; Thorarensen, Atli; Xing, Li; Caspers, Nicole L.; Day, Jacqueline E.; Mathis, Karl J.; Kretzmer, Kuniko K.; Reitz, Beverley A.; Weinberg, Robin A.; Stegeman, Roderick A.; Wrightstone, Ann; Christine, Lori; Compton, Robert; Li, Xiong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">908-911</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The inhibition of PKC-ζ has been proposed to be a potential drug target for immune and inflammatory diseases.  A series of 2-(6-phenyl-1H indazol-3-yl)-1H-benzo[d]imidazoles with initial high crossover to CDK-2 has been optimized to afford potent and selective inhibitors of protein kinase c-zeta (PKC-ζ).  The detn. of the crystal structures of key inhibitor:CDK-2 complexes informed the design and anal. of the series.  The most selective and potent analog was identified by variation of the aryl substituent at the 6-position of the indazole template to give a 4-NH2 deriv.  The analog displays good selectivity over other PKC isoforms (α, βII, γ, δ, ε, μ, θ, η and ι/λ) and CDK-2, however it displays marginal selectivity against a panel of other kinases (37 profiled).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYSeSuwVwvzbVg90H21EOLACvtfcHk0ljBf6X9ESu1RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtF2itL0%253D&md5=62c38c95188c1f42be3175d72a681c7d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.11.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.11.105%26sid%3Dliteratum%253Aachs%26aulast%3DTrujillo%26aufirst%3DJ.%2BI.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DXing%26aufirst%3DL.%26aulast%3DCaspers%26aufirst%3DN.%2BL.%26aulast%3DDay%26aufirst%3DJ.%2BE.%26aulast%3DMathis%26aufirst%3DK.%2BJ.%26aulast%3DKretzmer%26aufirst%3DK.%2BK.%26aulast%3DReitz%26aufirst%3DB.%2BA.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26aulast%3DStegeman%26aufirst%3DR.%2BA.%26aulast%3DWrightstone%26aufirst%3DA.%26aulast%3DChristine%26aufirst%3DL.%26aulast%3DCompton%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26atitle%3D2-%25286-Phenyl-1H-indazol-3-yl%2529-1H-benzo%255Bd%255Dimidazoles%253A%2520design%2520and%2520synthesis%2520of%2520a%2520potent%2520and%2520isoform%2520selective%2520PKC-%25CE%25B6%2520inhibitor%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D908%26epage%3D911%26doi%3D10.1016%2Fj.bmcl.2008.11.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjaer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostelecky, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray-Rust, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roffey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdonald, N. Q.</span></span> <span> </span><span class="NLM_article-title">Adenosine-binding motif mimicry and cellular effects of a thieno[2,3-d]pyrimidine-based chemical inhibitor of atypical protein kinase C isoenzymes</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.1042/BJ20121871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1042%2FBJ20121871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=23418854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2013&pages=329-342&author=S.+Kjaerauthor=M.+Linchauthor=A.+Purkissauthor=B.+Kosteleckyauthor=P.+P.+Knowlesauthor=C.+Rosseauthor=P.+Riouauthor=C.+Soudyauthor=S.+Kayeauthor=B.+Patelauthor=E.+Sorianoauthor=J.+Murray-Rustauthor=C.+Bartonauthor=C.+Dillonauthor=J.+Roffeyauthor=P.+J.+Parkerauthor=N.+Q.+Mcdonald&title=Adenosine-binding+motif+mimicry+and+cellular+effects+of+a+thieno%5B2%2C3-d%5Dpyrimidine-based+chemical+inhibitor+of+atypical+protein+kinase+C+isoenzymes&doi=10.1042%2FBJ20121871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine-binding motif mimicry and cellular effects of a thieno[2,3-d]pyrimidine-based chemical inhibitor of atypical protein kinase C isoenzymes</span></div><div class="casAuthors">Kjaer, Svend; Linch, Mark; Purkiss, Andrew; Kostelecky, Brenda; Knowles, Phillip P.; Rosse, Carine; Riou, Philippe; Soudy, Christelle; Kaye, Sarah; Patel, Bhavisha; Soriano, Erika; Murray-Rust, Judith; Barton, Caroline; Dillon, Christian; Roffey, Jon; Parker, Peter J.; McDonald, Neil Q.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">329-342</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The aPKC [atypical PKC (protein kinase C)] isoforms ι and ζ play crucial roles in the formation and maintenance of cell polarity and represent attractive anti-oncogenic drug targets in Ras-dependent tumors.  To date, few isoform-specific chem. biol. tools are available to inhibit aPKC catalytic activity.  In the present paper, we describe the identification and functional characterization of potent and selective thieno[2,3-d]pyrimidine-based chem. inhibitors of aPKCs.  A crystal structure of human PKCι kinase domain bound to a representative compd., CRT0066854, reveals the basis for potent and selective chem. inhibition.  Furthermore, CRT0066854 displaces a crucial Asn-Phe-Asp motif that is part of the adenosine-binding pocket and engages an acidic patch used by arginine-rich PKC substrates.  We show that CRT0066854 inhibits the LLGL2 (lethal giant larvae 2) phosphorylation in cell lines and exhibits phenotypic effects in a range of cell-based assays.  We conclude that this compd. can be used as a chem. tool to modulate aPKC activity in vitro and in vivo and may guide the search for further aPKC-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5lEC0hIc6q7Vg90H21EOLACvtfcHk0lgJhBw0EgOi4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7k%253D&md5=332ef49dabbbf6e2333a101ef338c3c5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1042%2FBJ20121871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121871%26sid%3Dliteratum%253Aachs%26aulast%3DKjaer%26aufirst%3DS.%26aulast%3DLinch%26aufirst%3DM.%26aulast%3DPurkiss%26aufirst%3DA.%26aulast%3DKostelecky%26aufirst%3DB.%26aulast%3DKnowles%26aufirst%3DP.%2BP.%26aulast%3DRosse%26aufirst%3DC.%26aulast%3DRiou%26aufirst%3DP.%26aulast%3DSoudy%26aufirst%3DC.%26aulast%3DKaye%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DB.%26aulast%3DSoriano%26aufirst%3DE.%26aulast%3DMurray-Rust%26aufirst%3DJ.%26aulast%3DBarton%26aufirst%3DC.%26aulast%3DDillon%26aufirst%3DC.%26aulast%3DRoffey%26aufirst%3DJ.%26aulast%3DParker%26aufirst%3DP.%2BJ.%26aulast%3DMcdonald%26aufirst%3DN.%2BQ.%26atitle%3DAdenosine-binding%2520motif%2520mimicry%2520and%2520cellular%2520effects%2520of%2520a%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520chemical%2520inhibitor%2520of%2520atypical%2520protein%2520kinase%2520C%2520isoenzymes%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D451%26spage%3D329%26epage%3D342%26doi%3D10.1042%2FBJ20121871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fröhner, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garcia, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neimanis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navratil, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">4-benzimidazolyl-3-phenylbutanoic acids as novel PIF-pocket-targeting allosteric inhibitors of protein kinase PKCζ</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6714</span>– <span class="NLM_lpage">6723</span>, <span class="refDoi"> DOI: 10.1021/jm2005892</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2005892" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A280%3ADC%252BC3MfoslOjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6714-6723&author=W.+Fr%C3%B6hnerauthor=L.+A.+Lopez-Garciaauthor=S.+Neimanisauthor=N.+Weberauthor=J.+Navratilauthor=F.+Maurerauthor=A.+Strobaauthor=H.+Zhangauthor=R.+M.+Biondiauthor=M.+Engel&title=4-benzimidazolyl-3-phenylbutanoic+acids+as+novel+PIF-pocket-targeting+allosteric+inhibitors+of+protein+kinase+PKC%CE%B6&doi=10.1021%2Fjm2005892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">4-benzimidazolyl-3-phenylbutanoic acids as novel PIF-pocket-targeting allosteric inhibitors of protein kinase PKCζ</span></div><div class="casAuthors">Frohner Wolfgang; Lopez-Garcia Laura A; Neimanis Sonja; Weber Nadja; Navratil Jeanette; Maurer Frauke; Stroba Adriana; Zhang Hua; Biondi Ricardo M; Engel Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6714-23</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein kinase inhibitors with an allosteric mode of action are expected to reach, in many cases, higher selectivity for the target enzyme than ATP-competitive compounds.  Therefore, basic research is aiming at identifying and establishing novel sites on the catalytic domain of protein kinases which might be targeted by allosteric inhibitors.  We previously published the first structure-activity relationships (SARs) for allosteric activators of protein kinase PDK1.  Here, we present the design, synthesis, and SAR data on a series of novel compounds, 4-benzimidazolyl-3-phenylbutanoic acids, that inhibit the atypical protein kinace C (PKC) ζ via binding to the PIF-pocket.  Key positions were identified in the compounds that can be modified to increase potency and selectivity.  Some congeners showed a high selectivity toward PKCζ, lacking inhibition of the most closely related isoform, PKCι, and of further AGC kinases.  Furthermore, evidence is provided that these compounds are also active toward cellular PKCζ without loss of potency compared to the cell-free assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBGw-v0ps7djQLBVDQn4bCfW6udTcc2eZu8Rlop1Q5Nrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfoslOjsg%253D%253D&md5=274fee7a71f82dbc9eb086a6ef81d96e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm2005892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2005892%26sid%3Dliteratum%253Aachs%26aulast%3DFr%25C3%25B6hner%26aufirst%3DW.%26aulast%3DLopez-Garcia%26aufirst%3DL.%2BA.%26aulast%3DNeimanis%26aufirst%3DS.%26aulast%3DWeber%26aufirst%3DN.%26aulast%3DNavratil%26aufirst%3DJ.%26aulast%3DMaurer%26aufirst%3DF.%26aulast%3DStroba%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26aulast%3DEngel%26aufirst%3DM.%26atitle%3D4-benzimidazolyl-3-phenylbutanoic%2520acids%2520as%2520novel%2520PIF-pocket-targeting%2520allosteric%2520inhibitors%2520of%2520protein%2520kinase%2520PKC%25CE%25B6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6714%26epage%3D6723%26doi%3D10.1021%2Fjm2005892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Halim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diesel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiemer, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abadi, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of 1,3,5-trisubstituted pyrazolines as potent and highly selective allosteric inhibitors of protein kinase C-ζ</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6513</span>– <span class="NLM_lpage">6530</span>, <span class="refDoi"> DOI: 10.1021/jm500521n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500521n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1WmsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6513-6530&author=M.+Abdel-Halimauthor=B.+Dieselauthor=A.+K.+Kiemerauthor=A.+H.+Abadiauthor=R.+W.+Hartmannauthor=M.+Engel&title=Discovery+and+optimization+of+1%2C3%2C5-trisubstituted+pyrazolines+as+potent+and+highly+selective+allosteric+inhibitors+of+protein+kinase+C-%CE%B6&doi=10.1021%2Fjm500521n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of 1,3,5-Trisubstituted Pyrazolines as Potent and Highly Selective Allosteric Inhibitors of Protein Kinase C-ζ</span></div><div class="casAuthors">Abdel-Halim, Mohammad; Diesel, Britta; Kiemer, Alexandra K.; Abadi, Ashraf H.; Hartmann, Rolf W.; Engel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6513-6530</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is increasing evidence that the atypical protein kinase C, PKCζ, might be a therapeutic target in pulmonary and hepatic inflammatory diseases.  However, targeting the highly conserved ATP-binding pocket in the catalytic domain held little promise to achieve selective inhibition.  In the present study, the authors introduce 1,3,5-trisubstituted pyrazolines, e.g. I [R1 = 4-HO, 4-F, 4-Cl, etc.; R2 = Me, Ph, t-Bu, etc.; R3 = H, 4-Cl, 4-Br, etc.; R4 = H, Me] as potent and selective allosteric PKCζ inhibitors.  The rigid scaffold offered many sites for modification, all acting as hot spots for improving activity, and gave rise to sharp structure-activity relationships.  Targeting of PKCζ in cells was confirmed by reporter gene assay, transfection assays, and Western blotting.  The strongly reduced cell-free and cellular activities toward a PIF-pocket mutant of PKCζ suggested that the inhibitors most likely bound to the PIF-pocket on the kinase catalytic domain.  Thus, using a rigidification strategy and by establishing and optimizing multiple mol. interactions with the binding site, the authors were able to significantly improve the potency of the previously reported PKCζ inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiEjZcMLasYrVg90H21EOLACvtfcHk0lgJhBw0EgOi4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1WmsrzJ&md5=95c3f2c59aabf54afb97632e8bd8ff7c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm500521n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500521n%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Halim%26aufirst%3DM.%26aulast%3DDiesel%26aufirst%3DB.%26aulast%3DKiemer%26aufirst%3DA.%2BK.%26aulast%3DAbadi%26aufirst%3DA.%2BH.%26aulast%3DHartmann%26aufirst%3DR.%2BW.%26aulast%3DEngel%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520optimization%2520of%25201%252C3%252C5-trisubstituted%2520pyrazolines%2520as%2520potent%2520and%2520highly%2520selective%2520allosteric%2520inhibitors%2520of%2520protein%2520kinase%2520C-%25CE%25B6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6513%26epage%3D6530%26doi%3D10.1021%2Fjm500521n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span> <span> </span><span class="NLM_article-title">Fragment-based lead discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1038/nrd1467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-based+lead+discovery&doi=10.1038%2Fnrd1467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lg1GGktPPxOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-based%2520lead%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D660%26epage%3D672%26doi%3D10.1038%2Fnrd1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Fragment Based Lead Discovery</span>.  <i>Handbook of Medicinal Chemistry: Principles and Practice</i>; <span class="NLM_publisher-name">Royal Society of Chemistry</span>, <span class="NLM_year">2015</span>; Chapter 5, pp  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">153</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=R.+E.+Hubbard&title=Fragment+Based+Lead+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DFragment%2520Based%2520Lead%2520Discovery%26jtitle%3DHandbook%2520of%2520Medicinal%2520Chemistry%253A%2520Principles%2520and%2520Practice%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2015%26spage%3D122%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Lessons for fragment library design: analysis of output from multiple screening campaigns</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9280-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1007%2Fs10822-009-9280-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19495994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=603-610&author=I.+J.+Chenauthor=R.+E.+Hubbard&title=Lessons+for+fragment+library+design%3A+analysis+of+output+from+multiple+screening+campaigns&doi=10.1007%2Fs10822-009-9280-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons for fragment library design: analysis of output from multiple screening campaigns</span></div><div class="casAuthors">Chen, I.-Jen; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-620</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past 8 years, we have developed, refined and applied a fragment based discovery approach to a range of protein targets.  Here we report computational analyses of various aspects of our fragment library and the results obtained for fragment screening.  We reinforce the finding of others that the exptl. obsd. hit rate for screening fragments can be related to a computationally defined druggability index for the target.  In general, the physicochem. properties of the fragment hits display the same profile as the library, as is expected for a truly diverse library which probes the relevant chem. space.  An anal. of the fragment hits against various protein classes has shown that the physicochem. properties of the fragments are complementary to the properties of the target binding site.  The effectiveness of some fragments appears to be achieved by an appropriate mix of pharmacophore features and enhanced aromaticity, with hydrophobic interactions playing an important role.  The anal. emphasizes that it is possible to identify small fragments that are specific for different binding sites.  To conclude, we discuss how the results could inform further development and improvement of our fragment library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo4yzAG3RSbVg90H21EOLACvtfcHk0lg1GGktPPxOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D&md5=f544180cae108c2d7365037388df0262</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9280-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9280-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLessons%2520for%2520fragment%2520library%2520design%253A%2520analysis%2520of%2520output%2520from%2520multiple%2520screening%2520campaigns%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D603%26epage%3D610%26doi%3D10.1007%2Fs10822-009-9280-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaibuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakoshima, T.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanism for the regulation of rho-kinase by dimerization and its inhibition by fasudil</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">600</span>, <span class="refDoi"> DOI: 10.1016/j.str.2005.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.str.2005.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=16531242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD28Xit1ylt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=589-600&author=H.+Yamaguchiauthor=M.+Kasaauthor=M.+Amanoauthor=K.+Kaibuchiauthor=T.+Hakoshima&title=Molecular+mechanism+for+the+regulation+of+rho-kinase+by+dimerization+and+its+inhibition+by+fasudil&doi=10.1016%2Fj.str.2005.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism for the Regulation of Rho-Kinase by Dimerization and Its Inhibition by Fasudil</span></div><div class="casAuthors">Yamaguchi, Hiroto; Kasa, Miyuki; Amano, Mutsuki; Kaibuchi, Kozo; Hakoshima, Toshio</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-600</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Rho-kinase is a key regulator of cytoskeletal events and a promising drug target in the treatment of vascular diseases and neurol. disorders.  Unlike other protein kinases, Rho-kinase requires both N- and C-terminal extension segments outside the kinase domain for activity, although the details of this requirement have been elusive.  The crystal structure of an active Rho-kinase fragment contg. the kinase domain and both the extensions revealed a head-to-head homodimer through the N-terminal extension forming a helix bundle that structurally integrates the C-terminal extension.  This structural organization enables binding of the C-terminal hydrophobic motif to the N-terminal lobe, which defines the correct disposition of helix αC that is important for the catalytic activity.  The bound inhibitor fasudil significantly alters the conformation and, consequently, the mode of interaction with the catalytic cleft that contains local structural changes.  Thus, both kinase and drug conformational pliability and stability confer selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKCe7VapDGkLVg90H21EOLACvtfcHk0lg1GGktPPxOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xit1ylt78%253D&md5=c00104ab0fe7731e51b68b3762ee894a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DKasa%26aufirst%3DM.%26aulast%3DAmano%26aufirst%3DM.%26aulast%3DKaibuchi%26aufirst%3DK.%26aulast%3DHakoshima%26aufirst%3DT.%26atitle%3DMolecular%2520mechanism%2520for%2520the%2520regulation%2520of%2520rho-kinase%2520by%2520dimerization%2520and%2520its%2520inhibition%2520by%2520fasudil%26jtitle%3DStructure%26date%3D2006%26volume%3D14%26spage%3D589%26epage%3D600%26doi%3D10.1016%2Fj.str.2005.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span>; <span class="NLM_string-name">Milburn, M. V.</span>; <span class="NLM_string-name">Artis, D. R.</span></span> <span> </span><span class="NLM_article-title">Scaffold-Based Drug Discovery</span>.  <i>Structure-Based Drug Discovery</i>; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2007</span>; Chapter 6, pp  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">153</span>, DOI: <span class="refDoi"> DOI: 10.1007/1-4020-4407-0_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1007%2F1-4020-4407-0_6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=129-153&author=K.+Y.+J.+Zhang&author=M.+V.+Milburn&author=D.+R.+Artis&title=Structure-Based+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2F1-4020-4407-0_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F1-4020-4407-0_6%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DScaffold-Based%2520Drug%2520Discovery%26btitle%3DStructure-Based%2520Drug%2520Discovery%26pub%3DSpringer%26date%3D2007%26spage%3D129%26epage%3D153%26doi%3D10.1007%2F1-4020-4407-0_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haass, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sproesser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settachatgul, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artis, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective inhibitor of oncologenic B-Raf kinase with potent antimelanoma activity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">3041</span>– <span class="NLM_lpage">3046</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711741105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1073%2Fpnas.0711741105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=18287029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=3041-3046&author=J.+Tsaiauthor=J.+T.+Leeauthor=W.+Wangauthor=J.+Zhangauthor=H.+Choauthor=S.+Mamoauthor=R.+Bremerauthor=S.+Gilletteauthor=J.+Kongauthor=N.+K.+Haassauthor=K.+Sproesserauthor=L.+Liauthor=K.+S.+M.+Smalleyauthor=D.+Fongauthor=Y.-L.+Zhuauthor=A.+Marimuthuauthor=H.+Nguyenauthor=B.+Lamauthor=J.+Liuauthor=I.+Cheungauthor=J.+Riceauthor=Y.+Suzukiauthor=C.+Luuauthor=C.+Settachatgulauthor=R.+Shellooeauthor=J.+Cantwellauthor=S.-H.+Kimauthor=J.+Schlessingerauthor=K.+Y.+J.+Zhangauthor=B.+L.+Westauthor=B.+Powellauthor=G.+Habetsauthor=C.+Zhangauthor=P.+N.+Ibrahimauthor=P.+Hirthauthor=D.+R.+Artisauthor=M.+Herlynauthor=G.+Bollag&title=Discovery+of+a+selective+inhibitor+of+oncologenic+B-Raf+kinase+with+potent+antimelanoma+activity&doi=10.1073%2Fpnas.0711741105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span></div><div class="casAuthors">Tsai, James; Lee, John T.; Wang, Weiru; Zhang, Jiazhong; Cho, Hanna; Mamo, Shumeye; Bremer, Ryan; Gillette, Sam; Kong, Jun; Haass, Nikolas K.; Sproesser, Katrin; Li, Ling; Smalley, Keiran S. M.; Fong, Daniel; Zhu, Yong-Liang; Marimuthu, Adhirai; Nguyen, Hoa; Lam, Billy; Liu, Jennifer; Cheung, Ivana; Rice, Julie; Suzuki, Yoshihisa; Luu, Catherine; Settachatgul, Calvin; Shellooe, Rafe; Cantwell, John; Kim, Sung-Hou; Schlessinger, Joseph; Zhang, Kam Y. J.; West, Brian L.; Powell, Ben; Habets, Gaston; Zhang, Chao; Ibrahim, Prabha N.; Hirth, Peter; Artis, Dean R.; Herlyn, Meenhard; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3041-3046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">BRAFV600E is the most frequent oncogenic protein kinase mutation known.  Furthermore, inhibitors targeting "active" protein kinases have demonstrated significant utility in the therapeutic repertoire against cancer.  Therefore, the authors pursued the development of specific kinase inhibitors targeting B-Raf, and the V600E allele in particular.  By using a structure-guided discovery approach, a potent and selective inhibitor of active B-Raf has been discovered.  PLX4720, a 7-azaindole deriv. that inhibits B-RafV600E with an IC50 of 13 nM, defines a class of kinase inhibitor with marked selectivity in both biochem. and cellular assays.  PLX4720 preferentially inhibits the active B-RafV600E kinase compared with a broad spectrum of other kinases, and potent cytotoxic effects are also exclusive to cells bearing the V600E allele.  Consistent with the high degree of selectivity, ERK phosphorylation is potently inhibited by PLX4720 in B-RafV600E-bearing tumor cell lines but not in cells lacking oncogenic B-Raf.  In melanoma models, PLX4720 induces cell cycle arrest and apoptosis exclusively in B-RafV600E-pos. cells.  In B-RafV600E-dependent tumor xenograft models, orally dosed PLX4720 causes significant tumor growth delays, including tumor regressions, without evidence of toxicity.  The work described here represents the entire discovery process, from initial identification through structural and biol. studies in animal models to a promising therapeutic for testing in cancer patients bearing B-RafV600E-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPX53IQCbDJ7Vg90H21EOLACvtfcHk0liH6uoHVYW1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D&md5=273b94202f99984e6eb24723d0c6141f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711741105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711741105%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DMamo%26aufirst%3DS.%26aulast%3DBremer%26aufirst%3DR.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHaass%26aufirst%3DN.%2BK.%26aulast%3DSproesser%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.-L.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DLuu%26aufirst%3DC.%26aulast%3DSettachatgul%26aufirst%3DC.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.-H.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520inhibitor%2520of%2520oncologenic%2520B-Raf%2520kinase%2520with%2520potent%2520antimelanoma%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D3041%26epage%3D3046%26doi%3D10.1073%2Fpnas.0711741105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Opportunity knocks: organic chemistry for fragment-based drug discovery (FBDD)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1002/anie.201506783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fanie.201506783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=488-492&author=C.+W.+Murrayauthor=D.+C.+Rees&title=Opportunity+knocks%3A+organic+chemistry+for+fragment-based+drug+discovery+%28FBDD%29&doi=10.1002%2Fanie.201506783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunity Knocks: Organic Chemistry for Fragment-Based Drug Discovery (FBDD)</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">488-492</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj8GEY4dDXxrVg90H21EOLACvtfcHk0liH6uoHVYW1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O&md5=97a92911033a2304da101888da0c3642</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.201506783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201506783%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DOpportunity%2520knocks%253A%2520organic%2520chemistry%2520for%2520fragment-based%2520drug%2520discovery%2520%2528FBDD%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D488%26epage%3D492%26doi%3D10.1002%2Fanie.201506783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuster, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillooly, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dambach, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattoli, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zusi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span> <span> </span><span class="NLM_article-title">A highly selective inhibitor of I kappa B kinase, BMS-345541, blocks both joint inflammation and destruction in collagen-induced arthritis in mice</span>. <i>Arthritis Rheum.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2652</span>– <span class="NLM_lpage">2659</span>, <span class="refDoi"> DOI: 10.1002/art.11131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fart.11131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=13130486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3sXot1Wjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2003&pages=2652-2659&author=K.+W.+McIntyreauthor=D.+J.+Shusterauthor=K.+M.+Gilloolyauthor=D.+M.+Dambachauthor=M.+A.+Pattoliauthor=P.+Luauthor=X.-D.+Zhouauthor=Y.+Qiuauthor=C.+Zusiauthor=J.+R.+Burke&title=A+highly+selective+inhibitor+of+I+kappa+B+kinase%2C+BMS-345541%2C+blocks+both+joint+inflammation+and+destruction+in+collagen-induced+arthritis+in+mice&doi=10.1002%2Fart.11131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A highly selective inhibitor of I.vkappa.B kinase, BMS-345541, blocks both joint inflammation and destruction in collagen-induced arthritis in mice</span></div><div class="casAuthors">McIntyre, Kim W.; Shuster, David J.; Gillooly, Kathleen M.; Dambach, Donna M.; Pattoli, Mark A.; Lu, Pin; Zhou, Xia-Di; Qiu, Yuping; Zusi, F. Christopher; Burke, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2652-2659</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The transcription of several cytokines, cell adhesion mols., and enzymes involved in the inflammatory and destructive mechanisms of rheumatoid arthritis is dependent on nuclear factor .vkappa.B (NF-.vkappa.B).  Because I.vkappa.B kinase (IKK) is crit. in transducing the signal-inducible activation of NF-.vkappa.B, we examd. whether the highly selective and orally bioavailable IKK inhibitor BMS-345541 is efficacious against collagen-induced arthritis (CIA) in mice.  Arthritis in DBA/1LacJ male mice was induced by s.c. immunization with, bovine type II collagen on day 0 and day 21.  BMS-345541 was administered perorally daily, either prophylactically (before disease onset) or therapeutically (after disease onset).  Clin. assessment of the incidence and severity of disease was conducted throughout the study, and histol. evaluation was performed at the time of study termination (day 42).  When administered prophylactically, BMS-345541 (in a dose range of 10-100 mg/kg) was effective, in a dose-dependent manner, in reducing the incidence of disease and inhibiting clin. signs of disease.  Histol. evaluation of the joints showed that both inflammation and joint destruction were blocked by the IKK inhibitor.  Message levels of interleukin-1β in the joints were also dose-dependently inhibited in the mice that received BMS-345541.  Dose-dependent efficacy in terms of both disease severity and histol. end points was obsd. with the therapeutic dosing regimen of BMS-345541, with use of the 100-mg/kg dose resulting in resoln. of disease.  IKK plays a key role in CIA in mice, and inhibitors of this enzyme represent a promising target for the development of novel agents to treat rheumatoid arthritis and other inflammatory diseases.  BMS-345541 represents the first example of an inhibitor of IKK that has antiinflammatory activity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrethUYjO8ZkLVg90H21EOLACvtfcHk0liH6uoHVYW1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXot1Wjt7Y%253D&md5=c560529ace35642a0700a8bc04ea8eae</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fart.11131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.11131%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntyre%26aufirst%3DK.%2BW.%26aulast%3DShuster%26aufirst%3DD.%2BJ.%26aulast%3DGillooly%26aufirst%3DK.%2BM.%26aulast%3DDambach%26aufirst%3DD.%2BM.%26aulast%3DPattoli%26aufirst%3DM.%2BA.%26aulast%3DLu%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DX.-D.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DZusi%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26atitle%3DA%2520highly%2520selective%2520inhibitor%2520of%2520I%2520kappa%2520B%2520kinase%252C%2520BMS-345541%252C%2520blocks%2520both%2520joint%2520inflammation%2520and%2520destruction%2520in%2520collagen-induced%2520arthritis%2520in%2520mice%26jtitle%3DArthritis%2520Rheum.%26date%3D2003%26volume%3D48%26spage%3D2652%26epage%3D2659%26doi%3D10.1002%2Fart.11131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerner, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimos, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z. K.</span></span> <span> </span><span class="NLM_article-title">An atypical protein kinase C (PKCζ) plays a critical role in lipopolysaccharide-activated NF-κB in human peripheral blood monocytes and macrophages</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">5810</span>– <span class="NLM_lpage">5815</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.0804073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.4049%2Fjimmunol.0804073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19380829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslKhsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2009&pages=5810-5815&author=X.+Huangauthor=L.-Y.+Chenauthor=A.+M.+Doernerauthor=W.+W.+Panauthor=L.+Smithauthor=S.+Huangauthor=T.+J.+Papadimosauthor=Z.+K.+Pan&title=An+atypical+protein+kinase+C+%28PKC%CE%B6%29+plays+a+critical+role+in+lipopolysaccharide-activated+NF-%CE%BAB+in+human+peripheral+blood+monocytes+and+macrophages&doi=10.4049%2Fjimmunol.0804073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">An Atypical Protein Kinase C (PKCζ) Plays a Critical Role in Lipopolysaccharide-Activated NF-κB in Human Peripheral Blood Monocytes and Macrophages</span></div><div class="casAuthors">Huang, XueSong; Chen, Ling-Yu; Doerner, Astrid M.; Pan, Warren W.; Smith, Laura; Huang, Shuang; Papadimos, Thomas J.; Pan, Zhixing K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5810-5815</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">The authors have reported that the bacterial LPS induces the activation of NF-κB and inflammatory cytokine gene expression and that this requires the activity of small GTPase, RhoA.  Here, they show that an atypical protein kinase C isoenzyme, PKCζ, assocs. functionally with RhoA and that PKCζ acts as a signaling component downstream of RhoA.  Stimulation of monocytes and macrophages with LPS resulted in PKCζ activation and inhibition of PKCζ activity blocked both LPS-stimulated activation of NF-κB and IL-1β gene expression.  The results also indicate that transforming growth factor β-activated kinase 1 (TAK1) acts as a signaling component downstream of PKCζ in cytokine gene transcription stimulated by LPS in human peripheral blood monocytes and macrophages.  The specificity of this response suggests an important role for the Rho GTPase/PKCζ/TAK1/NF-κB pathway in host defense and in proinflammatory cytokine synthesis induced by bacterial LPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQszyQcZ2kXLVg90H21EOLACvtfcHk0lgWuKX1Ozlgfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslKhsbY%253D&md5=c1dcc782d7737fe543e0f051d39a4448</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.0804073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.0804073%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.-Y.%26aulast%3DDoerner%26aufirst%3DA.%2BM.%26aulast%3DPan%26aufirst%3DW.%2BW.%26aulast%3DSmith%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DPapadimos%26aufirst%3DT.%2BJ.%26aulast%3DPan%26aufirst%3DZ.%2BK.%26atitle%3DAn%2520atypical%2520protein%2520kinase%2520C%2520%2528PKC%25CE%25B6%2529%2520plays%2520a%2520critical%2520role%2520in%2520lipopolysaccharide-activated%2520NF-%25CE%25BAB%2520in%2520human%2520peripheral%2520blood%2520monocytes%2520and%2520macrophages%26jtitle%3DJ.%2520Immunol.%26date%3D2009%26volume%3D182%26spage%3D5810%26epage%3D5815%26doi%3D10.4049%2Fjimmunol.0804073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span> <span> </span><span class="NLM_article-title">Substrate recognition mechanism of atypical protein kinase Cs revealed by the structure of PKCι in complex with a substrate peptide from Part-3</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">791</span>– <span class="NLM_lpage">801</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.str.2012.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=22579248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=791-801&author=C.+Wangauthor=Y.+Shangauthor=J.+Yuauthor=M.+Zhang&title=Substrate+recognition+mechanism+of+atypical+protein+kinase+Cs+revealed+by+the+structure+of+PKC%CE%B9+in+complex+with+a+substrate+peptide+from+Part-3&doi=10.1016%2Fj.str.2012.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Recognition Mechanism of Atypical Protein Kinase Cs Revealed by the Structure of PKCι in Complex with a Substrate Peptide from Par-3</span></div><div class="casAuthors">Wang, Chihao; Shang, Yuan; Yu, Jiang; Zhang, Mingjie</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">791-801</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein kinase C (PKC) play crit. roles in many cellular functions including differentiation, proliferation, growth, and survival.  However, the mol. bases governing PKC's substrate recognitions remain poorly understood.  Here we detd. the structure of PKCι in complex with a peptide from Par-3 at 2.4 Å.  PKCι in the complex adopts catalytically competent, closed conformation without phosphorylation of Thr402 in the activation loop.  The Par-3 peptide binds to an elongated groove formed by the N- and C-lobes of the kinase domain.  The PKCι/Par-3 complex structure, together with extensive biochem. studies, reveals a set of substrate recognition sites common to all PKC isoenzymes as well as a hydrophobic pocket unique to aPKC.  A consensus aPKC's substrate recognition sequence pattern can be readily identified based on the complex structure.  Finally, we demonstrate that the pseudosubstrate sequence of PKCι resembles its substrate sequence, directly binds to and inhibits the activity of the kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEP6WuO1OB2bVg90H21EOLACvtfcHk0lgWuKX1Ozlgfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahs78%253D&md5=4aa93528111174f5d6dfa7c508f54a03</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShang%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26atitle%3DSubstrate%2520recognition%2520mechanism%2520of%2520atypical%2520protein%2520kinase%2520Cs%2520revealed%2520by%2520the%2520structure%2520of%2520PKC%25CE%25B9%2520in%2520complex%2520with%2520a%2520substrate%2520peptide%2520from%2520Part-3%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D791%26epage%3D801%26doi%3D10.1016%2Fj.str.2012.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokubo, H.</span></span> <span> </span><span class="NLM_article-title">Exploring the stability of ligand binding modes to proteins by molecular dynamics simulations: a cross-docking study</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2514</span>– <span class="NLM_lpage">2522</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00412</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00412" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVOrsb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2514-2522&author=K.+Liuauthor=H.+Kokubo&title=Exploring+the+stability+of+ligand+binding+modes+to+proteins+by+molecular+dynamics+simulations%3A+a+cross-docking+study&doi=10.1021%2Facs.jcim.7b00412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Stability of Ligand Binding Modes to Proteins by Molecular Dynamics Simulations: A Cross-docking Study</span></div><div class="casAuthors">Liu, Kai; Kokubo, Hironori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2514-2522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking has become an indispensable approach in drug discovery research to predict the binding mode of a ligand.  One great challenge in docking is to efficiently refine the correct pose from various putative docking poses through score functions.  The authors recently examd. the stability of self-docking poses under mol. dynamics (MD) simulations and showed that equil. MD simulations have some capability to discriminate between correct and decoy poses.  Here, the authors have extended the previous work to cross-docking studies for practical applications.  Three target proteins (thrombin, heat shock protein 90-alpha, and cyclin-dependent kinase 2) of pharmaceutical interest were selected.  Three comparable poses (one correct pose and two decoys) for each ligand were then selected from the docking poses.  To obtain the docking poses for the three target proteins, the authors used three different protocols; namely, normal docking, induced fit docking (IFD), and IFD against the homol. model.  Finally, five parallel MD equil. runs were performed on each pose for the statistical anal.  The results showed that the correct poses were generally more stable than the decoy poses under MD.  The discrimination capability of MD depends on the strategy.  The safest way was to judge a pose as being stable if any one run among five parallel runs was stable under MD.  In this case, 95% of the correct poses were retained under MD, and about 25-44% of the decoys could be excluded by the simulations for all cases.  On the other hand, if the authors judge a pose as being stable when any two or three runs were stable, with the risk of incorrectly excluding some correct poses, approx. 31-53% or 39-56% of the two decoys could be excluded by MD, resp.  The results suggest that simple equil. simulations can serve as an effective filter to exclude decoy poses that cannot be distinguished by docking scores from the computationally expensive free-energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxgZngG4j75LVg90H21EOLACvtfcHk0lgWuKX1Ozlgfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVOrsb%252FI&md5=dfe3ab369e356abb2c27190b026792d2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00412%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DKokubo%26aufirst%3DH.%26atitle%3DExploring%2520the%2520stability%2520of%2520ligand%2520binding%2520modes%2520to%2520proteins%2520by%2520molecular%2520dynamics%2520simulations%253A%2520a%2520cross-docking%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2514%26epage%3D2522%26doi%3D10.1021%2Facs.jcim.7b00412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span> <span> </span><span class="NLM_article-title">Pair interaction molecular orbital method: an approximate computational method for molecular interactions</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>312</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(99)00937-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS0009-2614%2899%2900937-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ymtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=312&publication_year=1999&pages=319-324&author=K.+Kitauraauthor=T.+Sawaiauthor=T.+Asadaauthor=T.+Nakanoauthor=M.+Uebayasi&title=Pair+interaction+molecular+orbital+method%3A+an+approximate+computational+method+for+molecular+interactions&doi=10.1016%2FS0009-2614%2899%2900937-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pair interaction molecular orbital method: an approximate computational method for molecular interactions</span></div><div class="casAuthors">Kitaura, Kazuo; Sawai, Takuya; Asada, Toshio; Nakano, Tatsuya; Uebayasi, Masami</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">312</span>
        (<span class="NLM_cas:issue">2-4</span>),
    <span class="NLM_cas:pages">319-324</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We propose an approx. method for calcg. large mol. clusters using MO methods.  The method allows one to obtain the total energy of a mol. cluster by performing MO calcns. on the mols. and the mol. pairs in the system.  The computational time is greatly reduced for a system contg. a large no. of mols., since the supermol. calcn. of the whole system is avoided.  Our method considers important terms of many-body interactions within a pair approxn. and is expected to give results of reasonable accuracy even for mol. interactions having significant many-body effects.  The numerical calcns. of water trimers and tetramers revealed that our method included the major portion of the many-body energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWDa8nVDQk9LVg90H21EOLACvtfcHk0li0_jnxAcA7Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ymtLY%253D&md5=c748640d22f5194740c842a6aafc8531</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2900937-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252900937-9%26sid%3Dliteratum%253Aachs%26aulast%3DKitaura%26aufirst%3DK.%26aulast%3DSawai%26aufirst%3DT.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DUebayasi%26aufirst%3DM.%26atitle%3DPair%2520interaction%2520molecular%2520orbital%2520method%253A%2520an%2520approximate%2520computational%2520method%2520for%2520molecular%2520interactions%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D312%26spage%3D319%26epage%3D324%26doi%3D10.1016%2FS0009-2614%2899%2900937-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span> <span> </span><span class="NLM_article-title">Fragment molecular orbital method: an approximate computational method for large molecules</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(99)00874-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS0009-2614%2899%2900874-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFChu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=1999&pages=701-706&author=K.+Kitauraauthor=E.+Ikeoauthor=T.+Asadaauthor=T.+Nakanoauthor=M.+Uebayasi&title=Fragment+molecular+orbital+method%3A+an+approximate+computational+method+for+large+molecules&doi=10.1016%2FS0009-2614%2899%2900874-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment molecular orbital method: an approximate computational method for large molecules</span></div><div class="casAuthors">Kitaura, K.; Ikeo, E.; Asada, T.; Nakano, T.; Uebayasi, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">3,4</span>),
    <span class="NLM_cas:pages">701-706</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">An approx. MO method was proposed for calcg. large mols. such as proteins.  This method assigns the electrons of the mols. to fragments, and the MOs of fragments and fragment pairs are calcd. to obtain the total energy of the mol.  The method avoids the MO calcn. of the whole mol., and is expected to reduce the computational time drastically for large mols.  Numerical calcns. were performed on C3H8, PrOH and AcNHMe to show the accuracy of the method.  The optimized geometries and total energies were in good agreement with those from the ab initio MO calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwzZTYpLNVNbVg90H21EOLACvtfcHk0li0_jnxAcA7Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFChu7Y%253D&md5=7096f34748e293b1ebe2e488b271e2df</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2900874-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252900874-X%26sid%3Dliteratum%253Aachs%26aulast%3DKitaura%26aufirst%3DK.%26aulast%3DIkeo%26aufirst%3DE.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DUebayasi%26aufirst%3DM.%26atitle%3DFragment%2520molecular%2520orbital%2520method%253A%2520an%2520approximate%2520computational%2520method%2520for%2520large%2520molecules%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D313%26spage%3D701%26epage%3D706%26doi%3D10.1016%2FS0009-2614%2899%2900874-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuzawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Fragment molecular orbital method: use of approximate electrostatic potential</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">480</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(01)01416-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS0009-2614%2801%2901416-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsFCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2002&pages=475-480&author=T.+Nakanoauthor=T.+Kaminumaauthor=T.+Satoauthor=K.+Fukuzawaauthor=Y.+Akiyamaauthor=M.+Uebayasiauthor=K.+Kitaura&title=Fragment+molecular+orbital+method%3A+use+of+approximate+electrostatic+potential&doi=10.1016%2FS0009-2614%2801%2901416-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment molecular orbital method: use of approximate electrostatic potential</span></div><div class="casAuthors">Nakano, Tatsuya; Kaminuma, Tsuguchika; Sato, Toshiyuki; Fukuzawa, Kaori; Akiyama, Yutaka; Uebayasi, Masami; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">5,6</span>),
    <span class="NLM_cas:pages">475-480</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Recently, we have proposed the fragment MO (FMO) method, an approx. MO method for calcg. large mols. such as proteins.  The method has been shown to reproduce ab initio total energies and geometries of mols. in good accuracy.  The most time consuming part in the method, the calcns. of environmental electrostatic potentials, were speeded up by employing the Mulliken approxn. for two-electron integrals and a fractional point charge approxn.  Numerical calcns. on several polypeptides revealed that the approxns. brought no significant loss of accuracy in the total energy of mols. and were of practical use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-OUIE6YzWiLVg90H21EOLACvtfcHk0li0_jnxAcA7Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsFCkug%253D%253D&md5=e0bd6c422cacba5105c5a3d6f6059874</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2801%2901416-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252801%252901416-6%26sid%3Dliteratum%253Aachs%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DKaminuma%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DFukuzawa%26aufirst%3DK.%26aulast%3DAkiyama%26aufirst%3DY.%26aulast%3DUebayasi%26aufirst%3DM.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DFragment%2520molecular%2520orbital%2520method%253A%2520use%2520of%2520approximate%2520electrostatic%2520potential%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D2002%26volume%3D351%26spage%3D475%26epage%3D480%26doi%3D10.1016%2FS0009-2614%2801%2901416-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Pair interaction energy decomposition analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1002/jcc.20496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.20496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=17109433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=222-237&author=D.+G.+Fedorovauthor=K.+Kitaura&title=Pair+interaction+energy+decomposition+analysis&doi=10.1002%2Fjcc.20496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Pair interaction energy decomposition analysis</span></div><div class="casAuthors">Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">222-237</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The energy decompn. anal. (EDA) by Kitaura and Morokuma was redeveloped in the framework of the fragment MO method (FMO).  The proposed pair interaction energy decompn. anal. (PIEDA) can treat large mol. clusters and the systems in which fragments are connected by covalent bonds, such as proteins.  The interaction energy in PIEDA is divided into the same contributions as in EDA: the electrostatic, exchange-repulsion, and charge transfer energies, to which the correlation (dispersion) term was added.  The careful comparison to the ab initio EDA interaction energies for water clusters with 2-16 mols. revealed that PIEDA has the error of at most 1.2 kcal/mol (or about 1%).  The anal. was applied to (H2O)1024, the α helix, β turn, and β strand of polyalanine (ALA)10, as well as to the synthetic protein (PDB code 1L2Y) with 20 residues.  The comparative aspects of the polypeptide isomer stability are discussed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ89I3W8YX1LVg90H21EOLACvtfcHk0li0_jnxAcA7Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D&md5=759c711290fe1c44b7c86a28104ccef7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20496%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DPair%2520interaction%2520energy%2520decomposition%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D222%26epage%3D237%26doi%3D10.1002%2Fjcc.20496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barder, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Catalysts for Suzuki-Miyaura coupling processes: scope and studies of the effect of ligand structure</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">4685</span>– <span class="NLM_lpage">4696</span>, <span class="refDoi"> DOI: 10.1021/ja042491j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja042491j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVWrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=4685-4696&author=T.+E.+Barderauthor=S.+D.+Walkerauthor=J.+R.+Martinelliauthor=S.+L.+Buchwald&title=Catalysts+for+Suzuki-Miyaura+coupling+processes%3A+scope+and+studies+of+the+effect+of+ligand+structure&doi=10.1021%2Fja042491j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Catalysts for Suzuki-Miyaura Coupling Processes: Scope and Studies of the Effect of Ligand Structure</span></div><div class="casAuthors">Barder, Timothy E.; Walker, Shawn D.; Martinelli, Joseph R.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4685-4696</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Suzuki-Miyaura coupling reactions of aryl and heteroaryl halides with aryl-, heteroaryl- and vinylboronic acids proceed in very good to excellent yield with the use of 2-(2',6'-dimethoxybiphenyl)dicyclohexylphosphine, SPhos (1) (I).  This ligand confers unprecedented activity for these processes, allowing reactions to be performed at low catalyst levels, to prep. extremely hindered biaryls and to be carried out, in general, for reactions of aryl chlorides at room temp.  Addnl., structural studies of various 1·Pd complexes are presented along with computational data that help elucidate the efficacy that 1 imparts on Suzuki-Miyaura coupling processes.  Moreover, a comparison of the reactions with 1 and with 2-(2',4',6'-triisopropylbiphenyl)diphenylphosphine (2) (II) is presented that is informative in detg. the relative importance of ligand bulk and electron-donating ability in the high activity of catalysts derived from ligands of this type.  Further, when the aryl bromide becomes too hindered, an interesting C-H bond functionalization-cross-coupling sequence intervenes to provide product in high yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP20DMwGPU0LVg90H21EOLACvtfcHk0ljhMd0-UZi10g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVWrsbc%253D&md5=af3f56a57a2165183ddf23b73f099a2a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fja042491j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042491j%26sid%3Dliteratum%253Aachs%26aulast%3DBarder%26aufirst%3DT.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BD.%26aulast%3DMartinelli%26aufirst%3DJ.%2BR.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DCatalysts%2520for%2520Suzuki-Miyaura%2520coupling%2520processes%253A%2520scope%2520and%2520studies%2520of%2520the%2520effect%2520of%2520ligand%2520structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D4685%26epage%3D4696%26doi%3D10.1021%2Fja042491j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alagille, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamagnan, G. D.</span></span> <span> </span><span class="NLM_article-title">One-step synthesis of 2-arylbenzothiazole (′BTA′) and benzoxazole precursors for in vivo imaging of β-amyloid plaques</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1349</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2004.12.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.tetlet.2004.12.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2MXosF2rtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=1349-1351&author=D.+Alagilleauthor=R.+M.+Baldwinauthor=G.+D.+Tamagnan&title=One-step+synthesis+of+2-arylbenzothiazole+%28%E2%80%B2BTA%E2%80%B2%29+and+benzoxazole+precursors+for+in+vivo+imaging+of+%CE%B2-amyloid+plaques&doi=10.1016%2Fj.tetlet.2004.12.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">One-step synthesis of 2-arylbenzothiazole (BTA) and -benzoxazole precursors for in vivo imaging of β-amyloid plaques</span></div><div class="casAuthors">Alagille, David; Baldwin, Ronald M.; Tamagnan, Gilles D.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1349-1351</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The authors report the simple and efficient synthesis of 2-arylbenzothiazoles (BTA) and 2-arylbenzoxazoles by direct coupling of benzothiazoles or benzoxazoles with aryl bromides.  This method permits direct one-step access to precursors of radiolabeled BTA-1 and BTA-2 and their 6-methoxy analogs, used for in vivo imaging of β-amyloid plaques with positron emission tomog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovcoP2lcbxK7Vg90H21EOLACvtfcHk0ljhMd0-UZi10g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXosF2rtQ%253D%253D&md5=563b72aad86176693ab8e6a593d473c4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2004.12.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2004.12.111%26sid%3Dliteratum%253Aachs%26aulast%3DAlagille%26aufirst%3DD.%26aulast%3DBaldwin%26aufirst%3DR.%2BM.%26aulast%3DTamagnan%26aufirst%3DG.%2BD.%26atitle%3DOne-step%2520synthesis%2520of%25202-arylbenzothiazole%2520%2528%25E2%2580%25B2BTA%25E2%2580%25B2%2529%2520and%2520benzoxazole%2520precursors%2520for%2520in%2520vivo%2520imaging%2520of%2520%25CE%25B2-amyloid%2520plaques%26jtitle%3DTetrahedron%2520Lett.%26date%3D2005%26volume%3D46%26spage%3D1349%26epage%3D1351%26doi%3D10.1016%2Fj.tetlet.2004.12.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Billingsley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Highly efficient monophosphine-based catalyst for the palladium-catalyzed Suzuki-Miyaura reaction of heteroaryl halides and heteroaryl boronic acids and esters</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">3358</span>– <span class="NLM_lpage">3366</span>, <span class="refDoi"> DOI: 10.1021/ja068577p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja068577p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitFKrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=3358-3366&author=K.+Billingsleyauthor=S.+L.+Buchwald&title=Highly+efficient+monophosphine-based+catalyst+for+the+palladium-catalyzed+Suzuki-Miyaura+reaction+of+heteroaryl+halides+and+heteroaryl+boronic+acids+and+esters&doi=10.1021%2Fja068577p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Efficient Monophosphine-Based Catalyst for the Palladium-Catalyzed Suzuki-Miyaura Reaction of Heteroaryl Halides and Heteroaryl Boronic Acids and Esters</span></div><div class="casAuthors">Billingsley, Kelvin; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3358-3366</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly active and efficient catalyst system derived from a palladium precatalyst and monophosphine ligands I or II for the Suzuki-Miyaura cross-coupling reaction of heteroaryl boronic acids and esters has been developed.  This method allows for the prepn. of a wide variety of heterobiaryls in good to excellent yields and displays a high level of activity for the coupling of heteroaryl chlorides as well as hindered aryl and heteroaryl halides.  Specific factors that govern the efficacy of the transformation for certain heterocyclic motifs were also investigated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlSHWf5e2EBbVg90H21EOLACvtfcHk0ljhMd0-UZi10g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitFKrsrs%253D&md5=a9e65a6c356774a7340edd3b413080cf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja068577p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja068577p%26sid%3Dliteratum%253Aachs%26aulast%3DBillingsley%26aufirst%3DK.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DHighly%2520efficient%2520monophosphine-based%2520catalyst%2520for%2520the%2520palladium-catalyzed%2520Suzuki-Miyaura%2520reaction%2520of%2520heteroaryl%2520halides%2520and%2520heteroaryl%2520boronic%2520acids%2520and%2520esters%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D3358%26epage%3D3366%26doi%3D10.1021%2Fja068577p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Old, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">A highly active catalyst for palladium-catalyzed cross-coupling reactions: room-temperature Suzuki couplings and amination of unactivated aryl chlorides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">9722</span>– <span class="NLM_lpage">9723</span>, <span class="refDoi"> DOI: 10.1021/ja982250+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja982250%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1cXlslOhsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=9722-9723&author=D.+W.+Oldauthor=J.+P.+Wolfeauthor=S.+L.+Buchwald&title=A+highly+active+catalyst+for+palladium-catalyzed+cross-coupling+reactions%3A+room-temperature+Suzuki+couplings+and+amination+of+unactivated+aryl+chlorides&doi=10.1021%2Fja982250%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A Highly Active Catalyst for Palladium-Catalyzed Cross-Coupling Reactions: Room-Temperature Suzuki Couplings and Amination of Unactivated Aryl Chlorides</span></div><div class="casAuthors">Old, David W.; Wolfe, John P.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9722-9723</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pd2(dba)3 and ligand 2-(dicyclohexylphosphino)-2'-(dimethylamino)biphenyl catalyzed the amination of aryl chloride or bromides at room temp.  Also, Pd2(dba)3 or Pd(OAc)2 and ligand 2-(dicyclohexylphosphino)-2'-(dimethylamino)biphenyl catalyzed the Suzuki coupling of aryl chloride or bromides with boron reagents at room temp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsj7rB61O95LVg90H21EOLACvtfcHk0ljTcKuuPf_AEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlslOhsr8%253D&md5=6713de2eb0f19db80339a4fc2c94321e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja982250%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja982250%252B%26sid%3Dliteratum%253Aachs%26aulast%3DOld%26aufirst%3DD.%2BW.%26aulast%3DWolfe%26aufirst%3DJ.%2BP.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DA%2520highly%2520active%2520catalyst%2520for%2520palladium-catalyzed%2520cross-coupling%2520reactions%253A%2520room-temperature%2520Suzuki%2520couplings%2520and%2520amination%2520of%2520unactivated%2520aryl%2520chlorides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D9722%26epage%3D9723%26doi%3D10.1021%2Fja982250%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">An overview of the Amber biomolecular simulation package</span>. <i>WIREs Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1002/wcms.1121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fwcms.1121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtFaiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=198-210&author=R.+Salomon-Ferrerauthor=D.+A.+Caseauthor=R.+C.+Walker&title=An+overview+of+the+Amber+biomolecular+simulation+package&doi=10.1002%2Fwcms.1121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the amber biomolecular simulation package</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Case, David A.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: Computational Molecular Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">198-210</span>CODEN:
                <span class="NLM_cas:coden">WIRCAH</span>;
        ISSN:<span class="NLM_cas:issn">1759-0884</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics (MD) allows the study of biol. and chem. systems at the atomistic level on timescales from femtoseconds to milliseconds.  It complements expt. while also offering a way to follow processes difficult to discern with exptl. techniques.  Numerous software packages exist for conducting MD simulations of which one of the widest used is termed Amber.  Here, we outline the most recent developments, since version 9 was released in Apr. 2006, of the Amber and AmberTools MD software packages, referred to here as simply the Amber package.  The latest release represents six years of continued development, since version 9, by multiple research groups and the culmination of over 33 years of work beginning with the first version in 1979.  The latest release of the Amber package, version 12 released in Apr. 2012, includes a substantial no. of important developments in both the scientific and computer science arenas.  We present here a condensed vision of what Amber currently supports and where things are likely to head over the coming years.  Figure 1 shows the performance in ns/day of the Amber package version 12 on a single-core AMD FX-8120 8-Core 3.6GHz CPU, the Cray XT5 system, and a single GPU GTX680.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1JUOzTqkwZrVg90H21EOLACvtfcHk0ljTcKuuPf_AEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtFaiu70%253D&md5=baac68008e08e80bd825b2362e14fa1f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1121%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DAn%2520overview%2520of%2520the%2520Amber%2520biomolecular%2520simulation%2520package%26jtitle%3DWIREs%2520Comput.%2520Mol.%2520Sci.%26date%3D2013%26volume%3D3%26spage%3D198%26epage%3D210%26doi%3D10.1002%2Fwcms.1121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasavajhala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">ff14SB: improving the accuracy of protein side chain and backbone parameters from ff99SB</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3696</span>– <span class="NLM_lpage">3713</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0ljTcKuuPf_AEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0ljTcKuuPf_AEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C.</span></span> <span> </span><span class="NLM_article-title">I. Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(20000130)21:2<132::AID-JCC5>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+Bayly&title=I.+Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method&doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0lghPzfVjcUhgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%26atitle%3DI.%2520Fast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146%26doi%3D10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C.</span></span> <span> </span><span class="NLM_article-title">I. Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+Bayly&title=I.+Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lghPzfVjcUhgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%26atitle%3DI.%2520Fast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+Impeyauthor=M.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lghPzfVjcUhgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DM.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G.</span></span> <span> </span><span class="NLM_article-title">A smooth particle mesh Ewald method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0ljaa-fgkwHkrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loncharich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span> <span> </span><span class="NLM_article-title">Langevin dynamics of peptides: the frictional dependence of isomerization rates of <i>N</i>-acetylalanyl-<i>N</i>′-methylamide</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">535</span>, <span class="refDoi"> DOI: 10.1002/bip.360320508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fbip.360320508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1515543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK38XisFGqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1992&pages=523-535&author=R.+J.+Loncharichauthor=B.+R.+Brooksauthor=R.+W.+Pastor&title=Langevin+dynamics+of+peptides%3A+the+frictional+dependence+of+isomerization+rates+of+N-acetylalanyl-N%E2%80%B2-methylamide&doi=10.1002%2Fbip.360320508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Langevin dynamics of peptides:  the frictional dependence of isomerization rates of N-acetylalanyl-N'-methylamide</span></div><div class="casAuthors">Loncharich, Richard J.; Brooks, Bernard R.; Pastor, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">523-35</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">The rate const. for the transition between the equatorial and axial conformations of N-acetylalanyl-N'-methylamide has been detd. from Langevin dynamics (LD) simulations with no explicit solvent.  The isomerization rate is max. at collision frequency γ = 2 ps-1, shows diffusive character for γ ≥ 10 ps-1, but does not approach zero even at γ = 0.01 ps-1.  This behavior differs from that found for a one-dimensional bistable potential and indicates that both collisional energy transfer with solvent and vibrational energy transfer between internal modes are important in the dynamics of barrier crossing for this system.  It is suggested that conformational searches of peptides be carried out using LD with a collision frequency that maximizes the isomerization rate (i.e., γ ≈ 2 ps-1).  This method is expected to be more efficient than either mol. dynamics in vacuo (which corresponds to LD with γ = 0) or mol. dynamics in solvent (where dynamics is largely diffusive).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXONM8fktQybVg90H21EOLACvtfcHk0ljaa-fgkwHkrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XisFGqu7o%253D&md5=9209a0b3485915887d1b03fefcbadc35</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fbip.360320508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360320508%26sid%3Dliteratum%253Aachs%26aulast%3DLoncharich%26aufirst%3DR.%2BJ.%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26atitle%3DLangevin%2520dynamics%2520of%2520peptides%253A%2520the%2520frictional%2520dependence%2520of%2520isomerization%2520rates%2520of%2520N-acetylalanyl-N%25E2%2580%25B2-methylamide%26jtitle%3DBiopolymers%26date%3D1992%26volume%3D32%26spage%3D523%26epage%3D535%26doi%3D10.1002%2Fbip.360320508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, A.</span></span> <span> </span><span class="NLM_article-title">An analysis of the accuracy of Langevin and molecular dynamics algorithm</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1419</span>, <span class="refDoi"> DOI: 10.1080/00268978800101881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1080%2F00268978800101881" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1988&pages=1409-1419&author=R.+W.+Pastorauthor=B.+R.+Brooksauthor=A.+Szabo&title=An+analysis+of+the+accuracy+of+Langevin+and+molecular+dynamics+algorithm&doi=10.1080%2F00268978800101881"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1080%2F00268978800101881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268978800101881%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DR.%2BW.%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DSzabo%26aufirst%3DA.%26atitle%3DAn%2520analysis%2520of%2520the%2520accuracy%2520of%2520Langevin%2520and%2520molecular%2520dynamics%2520algorithm%26jtitle%3DMol.%2520Phys.%26date%3D1988%26volume%3D65%26spage%3D1409%26epage%3D1419%26doi%3D10.1080%2F00268978800101881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van
Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics with coupling to an external bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van%0AGunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+dynamics+with+coupling+to+an+external+bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0ljaa-fgkwHkrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520dynamics%2520with%2520coupling%2520to%2520an%2520external%2520bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.+P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0ljP3hvhj1CxEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.%2BP.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">Settle: an analytical version of the SHAKE and RATTLE algorithm for rigid water models</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1002/jcc.540130805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.540130805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK38Xlslykt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=952-962&author=S.+Miyamotoauthor=P.+A.+Kollman&title=Settle%3A+an+analytical+version+of+the+SHAKE+and+RATTLE+algorithm+for+rigid+water+models&doi=10.1002%2Fjcc.540130805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">SETTLE:  an analytical version of the SHAKE and RATTLE algorithm for rigid water models</span></div><div class="casAuthors">Miyamoto, Shuichi; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">952-62</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An anal. algorithm, called SETTLE, for resetting the positions and velocities to satisfy the holonomic constraints on the rigid water model is presented.  This method is based on the Cartesian coordinate system and can be used in place of SHAKE and RATTLE.  The authors implemented this algorithm in the SPASMS package of mol. mechanics and dynamics.  Several series of mol. dynamics simulations were carried out to examine the performance of the new algorithm in comparison with the original RATTLE method.  SETTLE is of higher accuracy and is faster than RATTLE with reasonable tolerances by three to nine times on a scalar machine.  The performance improvement ranged from factors of 26 to 98 on a vector machine since the method presented is not iterative.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAdRqj2ObejLVg90H21EOLACvtfcHk0ljP3hvhj1CxEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xlslykt7o%253D&md5=65da9d55c7905abeaf7708d91a09e6e4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130805%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DS.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DSettle%253A%2520an%2520analytical%2520version%2520of%2520the%2520SHAKE%2520and%2520RATTLE%2520algorithm%2520for%2520rigid%2520water%2520models%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D952%26epage%3D962%26doi%3D10.1002%2Fjcc.540130805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boatz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbert, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windus, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span> <span> </span><span class="NLM_article-title">General atomic and molecular electronic structure system</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1002/jcc.540141112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.540141112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=1347-1363&author=M.+W.+Schmidtauthor=K.+Baldridgeauthor=J.+Boatzauthor=S.+T.+Elbertauthor=M.+S.+Gordonauthor=J.+H.+Jensenauthor=S.+Kosekiauthor=N.+Matsunagaauthor=K.+A.+Nguyenauthor=S.+J.+Suauthor=T.+L.+Windusauthor=M.+Dupuisauthor=J.+Montgomery&title=General+atomic+and+molecular+electronic+structure+system&doi=10.1002%2Fjcc.540141112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">General atomic and molecular electronic structure system</span></div><div class="casAuthors">Schmidt, Michael W.; Baldridge, Kim K.; Boatz, Jerry A.; Elbert, Steven T.; Gordon, Mark S.; Jensen, Jan H.; Koseki, Shiro; Matsunaga, Nikita; Nguyen, Kiet A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1347-63</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A description of the ab initio quantum chem. package GAMESS is presented.  Chem. systems contg. atoms through Rn can be treated with wave functions ranging from the simplest closed-shell case up to a general MCSCF case, permitting calcns. at the necessary level of sophistication.  Emphasis is given to novel features of the program.  The parallelization strategy used in the RHF, ROHF, UHF, and GVB sections of the program is described, and detailed speedup results are given.  Parallel calcns. can be run on ordinary workstations as well as dedicated parallel machines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pHblAdXU4rVg90H21EOLACvtfcHk0ljP3hvhj1CxEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D&md5=4d9e37cd3ec67a92bdfb5b0c0084707c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540141112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540141112%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DBaldridge%26aufirst%3DK.%26aulast%3DBoatz%26aufirst%3DJ.%26aulast%3DElbert%26aufirst%3DS.%2BT.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKoseki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DK.%2BA.%26aulast%3DSu%26aufirst%3DS.%2BJ.%26aulast%3DWindus%26aufirst%3DT.%2BL.%26aulast%3DDupuis%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DJ.%26atitle%3DGeneral%2520atomic%2520and%2520molecular%2520electronic%2520structure%2520system%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26spage%3D1347%26epage%3D1363%26doi%3D10.1002%2Fjcc.540141112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0024.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative compounds that act as PKCξ inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Example structure of fasudil (<b>5</b>), its binding mode (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESM">2ESM</a>), and assessment of the isoquinoline structure as fragments. Red circle, polar functionality for protein binding; gray circle, synthetically precedented vectors for fragment optimization without disrupting the likely binding pharmacophore; blue arrows, other implicit growth vectors toward chemical space to afford high activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Hit fragments from the high concentration biochemical assay with PKCζ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Result of merging fragments <b>6</b> and <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Result of merging fragments <b>7</b> and <b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Changes in limb score following the administration of <b>37</b> to CIA model mice (3, 10, and 30 mg/kg, b.i.d., po).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0007.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Assessment of paw thickness on limb score following the administration of <b>37</b> (3, 10, and 30 mg/kg, b.i.d., po) or BMS345541 (60 mg/kg, b.i.d., po) to CIA model mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0008.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Predicted binding mode of compound <b>10</b> and <b>26</b> obtained by the docking study using the homology model of PKCζ. The black dashed lines indicate the hydrogen bond with the hinge residue V333.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0009.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ligand RMSD profiles of compound <b>10</b> and compound <b>26</b> measured from the averaged structure of the merged MD simulations. Trajectories from the last 20 ns were used to calculate the average structure and the RMSD profile. Three individual trials are shown for each binding pose.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0010.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Representative binding models of compound <b>10</b> (yellow) and <b>26</b> (magenta) regarding the in silico and SAR results. The black dashed lines indicate the hydrogen bond with the hinge residue V333.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0011.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4,6-Isoquinoline Derivatives <b>10</b>–<b>21</b><sup>s</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>s</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl (<i>S</i>)-3-hydroxypyrrolidine-1-carboxylate, NaH, DMF, 80 °C, o.n. (78%); (b) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b>–<b>h</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 110 °C, o.n.; (c) 35% HCl<sub>(aq)</sub>, MeOH (2 steps, <b>10</b>, 52%; <b>11</b>, 60%; <b>12</b>, 52%; <b>13</b>, 62%; <b>14</b>, 56%; <b>15</b>, 71%; <b>16</b>, 63%; <b>17</b>, 72%; <b>18</b>, 78%; <b>19</b>, 70%; <b>20</b>, 36%; <b>21</b>, 72%); (d) Pd(PPh<sub>3</sub>)<sub>4</sub>, LiCl, <b>48a</b>,<b>b</b>, toluene, 110 °C, 18 h; (e) Pd(OAc)<sub>2</sub>, CuBr, <b>49a</b>,<b>b</b>, <sup><i>t</i></sup>Bu<sub>3</sub>P, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 150 °C, 18 h.</p></p></figure><figure data-id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0012.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4,6-Isoquinoline Derivatives <b>22</b>–<b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>50a</b>–<b>d</b>, NaH, DMF, 80 °C, o.n. (78%); (b) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 110 °C, o.n.; (c) 35% HCl<sub>(aq)</sub>, MeOH (3 steps, <b>22</b>: 7%, <b>23</b>: 16%, <b>24</b>: 9%, <b>25</b>: 10%).</p></p></figure><figure data-id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0013.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>26</b>–<b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl 3-oxopiperidine-1-carboxylate, NaBH(OAc)<sub>3</sub>, AcOH, rt (99%); (b) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b> or <b>47i</b>–<b>n</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 100 °C; (c) 35% HCl<sub>(aq)</sub>, MeOH (2 steps, <b>26</b>, 52%; <b>27</b>, 14%; <b>28</b>, 84%; <b>29</b>, 29%; <b>30</b>, 53%; <b>31</b>, 71%; <b>32</b>, 74%).</p></p></figure><figure data-id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0014.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>33</b>–<b>36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(OAc)<sub>2</sub>, SPHOS, <b>47a</b>, K<sub>3</sub>PO<sub>4</sub>, <sup><i>n</i></sup>BuOH, 100 °C (56%); (b) <i>tert</i>-butyl (2-oxoethyl)-carbamate-carboxylate, NaBH(OAc)<sub>3</sub>, AcOH, rt; (c) 35% HCl<sub>(aq)</sub>, MeOH (2 steps, <b>33</b>, 28%; <b>34</b>, 17%; <b>35</b>, 33%; <b>36</b>, 19%); (d) <sup><i>t</i></sup>BuONO, I<sub>2</sub>, MeCN, 80 °C (97%); (e) Pd<sub>2</sub>dba<sub>3</sub>, Davephos/XPHOS, <b>57a</b>–<b>c</b>, <sup><i>t</i></sup>BuONa, dioxane 110 °C.</p></p></figure><figure data-id="sch5" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/medium/jm0c00449_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0015.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>37</b>–<b>44</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00449/20200701/images/large/jm0c00449_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00449&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd<sub>2</sub>dba<sub>3</sub>, XPHOS, <i>tert</i>-butyl (<i>S</i>)-3-aminoazepane-1-carboxylate, <sup><i>t</i></sup>BuONa, dioxane 110 °C; (b) PhNTf<sub>2</sub>, Et<sub>3</sub>N, dioxane, 80 °C (2 steps, 85%); (c) PdCl<sub>2</sub>(dppf)<sub>2</sub>, 2 M Na<sub>2</sub>CO<sub>3(aq)</sub>, <b>47m</b> or <b>47o</b>–<b>u</b>, THF, reflux; (d) 35% HCl(aq), MeOH (2 steps, <b>37</b>, 31%; <b>38</b>, 59%; <b>39</b>, 66%; <b>40</b>, 37%; <b>41</b>, 40%; <b>42</b>, 46%; <b>43</b>, 31%; <b>44</b>, 32%).</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i94">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. C.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C: poised to signal</span>. <i>Am. J. Physiol. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">E395</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1152/ajpendo.00477.2009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1152%2Fajpendo.00477.2009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19934406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A280%3ADC%252BC3c7itVWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2010&pages=E395-402&author=A.+C.+Newton&title=Protein+kinase+C%3A+poised+to+signal&doi=10.1152%2Fajpendo.00477.2009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C: poised to signal</span></div><div class="casAuthors">Newton Alexandra C</div><div class="citationInfo"><span class="NLM_cas:title">American journal of physiology. Endocrinology and metabolism</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">E395-402</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nestled at the tip of a branch of the kinome, protein kinase C (PKC) family members are poised to transduce signals emanating from the cell surface.  Cell membranes provide the platform for PKC function, supporting the maturation of PKC through phosphorylation, its allosteric activation by binding specific lipids, and, ultimately, promoting the downregulation of the enzyme.  These regulatory mechanisms precisely control the level of signaling-competent PKC in the cell.  Disruption of this regulation results in pathophysiological states, most notably cancer, where PKC levels are often grossly altered.  This review introduces the PKC family and then focuses on recent advances in understanding the cellular regulation of its diacylglycerol-regulated members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKzc-AXy7rwI2y4bB4mnvYfW6udTcc2eYBUIAF2ROPALntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7itVWrug%253D%253D&md5=4b8a3dfcc65370383ac6fffe0c54db00</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1152%2Fajpendo.00477.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpendo.00477.2009%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DA.%2BC.%26atitle%3DProtein%2520kinase%2520C%253A%2520poised%2520to%2520signal%26jtitle%3DAm.%2520J.%2520Physiol.%2520Endocrinol.%2520Metab.%26date%3D2010%26volume%3D298%26spage%3DE395%26epage%3D402%26doi%3D10.1152%2Fajpendo.00477.2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Meco, M. T.</span></span> <span> </span><span class="NLM_article-title">PKCζ at the crossroad of NF-κB and Jak1/Stat6 signaling pathways</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">702</span>, <span class="refDoi"> DOI: 10.1038/sj.cdd.4401823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1038%2Fsj.cdd.4401823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=16322752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFWisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=702&author=J.+Moscatauthor=P.+Rennertauthor=M.+T.+Diaz-Meco&title=PKC%CE%B6+at+the+crossroad+of+NF-%CE%BAB+and+Jak1%2FStat6+signaling+pathways&doi=10.1038%2Fsj.cdd.4401823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PKCζ at the crossroad of NF-κB and Jak1/Stat6 signaling pathways</span></div><div class="casAuthors">Moscat, J.; Rennert, P.; Diaz-Meco, M. T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death and Differentiation</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">702-711</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The atypical protein kinase C (PKC) isoforms (aPKC) have been implicated in the regulation of a no. of essential signaling events.  Early studies using dominant-neg. mutants suggested that they are important intermediaries in the activation of the canonical nuclear factor (NF)-κB pathway.  More recent data using knockout mice genetically demonstrate that in fact the PKCζ isoform is essential for the adequate activation of this cascade both upstream and downstream the IκB kinase complex.  In this review, we summarize the mechanistic details whereby the aPKC pathway regulates important cellular functions and how this is achieved by the ability of these kinases to interact with different protein regulators and adapters, as well as to impinge in NF-κB-independent signaling cascades such as the Janus kinase-1/signal transducer and activator of transcription 6 system, which plays a crit. role in T-cell-mediated hepatitis and asthma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptr6LpwzLTlLVg90H21EOLACvtfcHk0liYE3KHb8j7oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFWisLk%253D&md5=54d1fc85832e875c800cb54180af1a48</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.cdd.4401823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.cdd.4401823%26sid%3Dliteratum%253Aachs%26aulast%3DMoscat%26aufirst%3DJ.%26aulast%3DRennert%26aufirst%3DP.%26aulast%3DDiaz-Meco%26aufirst%3DM.%2BT.%26atitle%3DPKC%25CE%25B6%2520at%2520the%2520crossroad%2520of%2520NF-%25CE%25BAB%2520and%2520Jak1%252FStat6%2520signaling%2520pathways%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2006%26volume%3D13%26spage%3D702%26doi%3D10.1038%2Fsj.cdd.4401823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Meco, M. T.</span></span> <span> </span><span class="NLM_article-title">The atypical PKCs in inflammation: NF-κB and beyond</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>246</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1111/j.1600-065X.2012.01093.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1111%2Fj.1600-065X.2012.01093.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=22435553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVyku73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=246&publication_year=2012&pages=154-167&author=J.+Moscatauthor=M.+T.+Diaz-Meco&title=The+atypical+PKCs+in+inflammation%3A+NF-%CE%BAB+and+beyond&doi=10.1111%2Fj.1600-065X.2012.01093.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The atypical PKCs in inflammation: NF-κB and beyond</span></div><div class="casAuthors">Diaz-Meco, Maria T.; Moscat, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">246</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">154-167, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  From the very early days of nuclear factor-κB (NF-κB) research, it was recognized that different protein kinase C (PKC) isoforms might be involved in the activation of NF-κB.  Pharmacol. tools and pseudosubstrate inhibitors suggested that these kinases play a role in this important inflammatory and survival pathway; however, it was the anal. of several genetic mouse knockout models that revealed the complexity and interrelations between the different components of the PB1 network in several cellular functions, including T-cell biol., bone homeostasis, inflammation assocd. with the metabolic syndrome, and cancer.  These studies unveiled, for example, the crit. role of PKCζ as a pos. regulator of NF-κB through the regulation of RelA but also its inflammatory suppressor activities through the regulation of the interleukin-4 signaling cascade.  This observation is of relevance in T cells, where p62, PKCζ, PKCλ/ι, and NBR1 establish a mesh of interactions that culminate in the regulation of T-cell effector responses through the modulation of T-cell polarity.  Many questions remain to be answered, not just from the point of view of the implication for NF-κB activation but also with regard to the in vivo interplay between these pathways in pathophysiol. processes like obesity and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUFcmnITPvHbVg90H21EOLACvtfcHk0liYE3KHb8j7oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVyku73E&md5=9ee6c4f3a536bdab6f8de6e11002751a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-065X.2012.01093.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-065X.2012.01093.x%26sid%3Dliteratum%253Aachs%26aulast%3DMoscat%26aufirst%3DJ.%26aulast%3DDiaz-Meco%26aufirst%3DM.%2BT.%26atitle%3DThe%2520atypical%2520PKCs%2520in%2520inflammation%253A%2520NF-%25CE%25BAB%2520and%2520beyond%26jtitle%3DImmunol.%2520Rev.%26date%3D2012%26volume%3D246%26spage%3D154%26epage%3D167%26doi%3D10.1111%2Fj.1600-065X.2012.01093.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chida, K.</span></span> <span> </span><span class="NLM_article-title">Protein kinase Cζ (PKCζ): activation mechanisms and cellular functions</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1093/jb/mvg017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1093%2Fjb%2Fmvg017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=12761192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlOkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2003&pages=1-7&author=T.+Hiraiauthor=K.+Chida&title=Protein+kinase+C%CE%B6+%28PKC%CE%B6%29%3A+activation+mechanisms+and+cellular+functions&doi=10.1093%2Fjb%2Fmvg017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase Cξ (PKCξ): Activation mechanisms and cellular functions</span></div><div class="casAuthors">Hirai, Takaaki; Chida, Kazuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinase C isoform ξ (PKC-ξ) is a member of the atypical PKC subfamily and has been widely implicated in the regulation of cellular functions.  Increasing evidence from studies using in vitro and in vivo systems points to PKC-ξ as a key regulator of crit. intracellular signaling pathways induced by various extracellular stimuli.  The major activation pathway of PKC-ξ depends on phosphatidylinositol (PI)-3,4,5-trisphosphate (PIP3), which is mainly produced by PI-3 kinase.  3'-PI-dependent protein kinase 1, which binds with high affinity to PIP3, phosphorylates and activates PKC-ξ.  Many studies have demonstrated the involvement of PKC-ξ in the MAP kinase cascade, transcriptional factor NFκB activation, ribosomal protein S6 kinase signaling, and cell polarity.  An important mol. event in a cell is the assocn. of PKC-ξ with other signaling mols., as well as scaffold proteins, to form large complexes that regulate their pathways.  The understanding of the mechanisms underlying PKC-ξ-mediated control of intracellular signaling is beginning to provide important insights into the roles of PKC-ξ in various cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRc5gPsczr5rVg90H21EOLACvtfcHk0lgnbwDbnTc9Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlOkurY%253D&md5=fb6bf908b9d79feb37cfd1e8d21e2ea8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvg017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvg017%26sid%3Dliteratum%253Aachs%26aulast%3DHirai%26aufirst%3DT.%26aulast%3DChida%26aufirst%3DK.%26atitle%3DProtein%2520kinase%2520C%25CE%25B6%2520%2528PKC%25CE%25B6%2529%253A%2520activation%2520mechanisms%2520and%2520cellular%2520functions%26jtitle%3DJ.%2520Biochem.%26date%3D2003%26volume%3D133%26spage%3D1%26epage%3D7%26doi%3D10.1093%2Fjb%2Fmvg017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.-D.</span></span> <span> </span><span class="NLM_article-title">Effect of SNPs in protein kinase C ζ gene on gene expression in the reporter gene detection system</span>. <i>World. J. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2360</span>, <span class="refDoi"> DOI: 10.3748/wjg.v10.i16.2357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.3748%2Fwjg.v10.i16.2357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=15285019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2cXns1Gmtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=2357-2360&author=Z.+Liuauthor=H.-X.+Sunauthor=Y.-W.+Zhangauthor=Y.-F.+Liauthor=J.+Zuoauthor=Y.+Mengauthor=F.-D.+Fang&title=Effect+of+SNPs+in+protein+kinase+C+%CE%B6+gene+on+gene+expression+in+the+reporter+gene+detection+system&doi=10.3748%2Fwjg.v10.i16.2357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of SNPs in protein kinase Cz gene on gene expression in the reporter gene detection system</span></div><div class="casAuthors">Liu, Zhuo; Sun, Hong-Xia; Zhang, Yong-Wei; Li, Yun-Feng; Zuo, Jin; Meng, Yan; Fang, Fu-De</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2357-2360</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">World Journal of Gastroenterology</span>)
        </div><div class="casAbstract">The objective of this study was to investigated the effects of the SNPs (rs411021, rs436045, rs427811, rs385039 and rs809912) found in protein kinase Cζ gene on expression and further identify the susceptibility genes of type 2 diabetes.  Ten allele fragments (49 bp each) were synthesized according to the 5 SNPs mentioned above.  These fragments were cloned into luciferase reporter gene vector and then transfected into HepG2 cells.  The activity of the luciferase was assayed.  Effects of the SNPs on RNA splicing were analyzed by bioinformatics.  Rs427811T allele and rs809912G allele enhanced the activity of the reporter gene expression.  None of the 5 SNPs affected RNA splicing.  The authors concluded that SNPs in protein kinase Cζ (PKCZ) gene probably play a role in the susceptibility to type 2 diabetes by affecting the expression level of the relevant genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZXYiiJKugC7Vg90H21EOLACvtfcHk0lgnbwDbnTc9Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXns1Gmtb4%253D&md5=d139770198e6e8b1c6f9b1245dd836ea</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v10.i16.2357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v10.i16.2357%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DH.-X.%26aulast%3DZhang%26aufirst%3DY.-W.%26aulast%3DLi%26aufirst%3DY.-F.%26aulast%3DZuo%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DF.-D.%26atitle%3DEffect%2520of%2520SNPs%2520in%2520protein%2520kinase%2520C%2520%25CE%25B6%2520gene%2520on%2520gene%2520expression%2520in%2520the%2520reporter%2520gene%2520detection%2520system%26jtitle%3DWorld.%2520J.%2520Gastroenterol.%26date%3D2004%26volume%3D10%26spage%3D2357%26epage%3D2360%26doi%3D10.3748%2Fwjg.v10.i16.2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seto, K. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrulis, I. L.</span></span> <span> </span><span class="NLM_article-title">Atypical protein kinase C ζ: potential player in cell survival and cell migration of ovarian cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">e0123528</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0123528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1371%2Fjournal.pone.0123528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=25874946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OitL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0123528&issue=4&author=K.+K.+Y.+Setoauthor=I.+L.+Andrulis&title=Atypical+protein+kinase+C+%CE%B6%3A+potential+player+in+cell+survival+and+cell+migration+of+ovarian+cancer&doi=10.1371%2Fjournal.pone.0123528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Atypical protein kinase C Zeta: potential player in cell survival and cell migration of ovarian cancer</span></div><div class="casAuthors">Seto, Kelly K. Y.; Andrulis, Irene L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0123528/1-e0123528/1</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Ovarian cancer is one of the most aggressive gynaecol. cancers, thus understanding the different biol. pathways involved in ovarian cancer progression is important in identifying potential therapeutic targets for the disease.  The aim of this study was to investigate the potential roles of Protein Kinase C Zeta (PRKCZ) in ovarian cancer.  The atypical protein kinase C isoform, PRKCZ, is involved in the control of various signalling processes including cell proliferation, cell survival, and cell motility, all of which are important for cancer development and progression.  Herein, we observe a significant increase in cell survival upon PRKCZ over-expression in SKOV3 ovarian cancer cells; addnl., when the cells are treated with small interference RNA (siRNA) targeting PRKCZ, the motility of SKOV3 cells decreased.  Furthermore, we demonstrate that over-expression of PRKCZ results in gene and/or protein expression alterations of insulin-like growth factor 1 receptor (IGF1R) and integrin beta 3 (ITGB3) in SKOV3 and OVCAR3 cells.  Collectively, our study describes PRKCZ as a potential regulatory component of the IGF1R and ITGB3 pathways and suggests that it may play crit. roles in ovarian tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZycReUEmvQLVg90H21EOLACvtfcHk0lgnbwDbnTc9Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OitL%252FL&md5=79e27ecb2374dcddfa3f999a72818594</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0123528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0123528%26sid%3Dliteratum%253Aachs%26aulast%3DSeto%26aufirst%3DK.%2BK.%2BY.%26aulast%3DAndrulis%26aufirst%3DI.%2BL.%26atitle%3DAtypical%2520protein%2520kinase%2520C%2520%25CE%25B6%253A%2520potential%2520player%2520in%2520cell%2520survival%2520and%2520cell%2520migration%2520of%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D4%26spage%3De0123528%26doi%3D10.1371%2Fjournal.pone.0123528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leitges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camacho, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz Meco, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rennert, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span> <span> </span><span class="NLM_article-title">Targeted disruption of the PKCζ gene results in the impairment of the NF-κB pathway</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)00361-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS1097-2765%2801%2900361-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=11684013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3MXotlagur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=771-780&author=M.+Leitgesauthor=L.+Sanzauthor=P.+Martinauthor=A.+Duranauthor=U.+Braunauthor=J.+F.+Garciaauthor=F.+Camachoauthor=M.+T.+Diaz+Mecoauthor=P.+D.+Rennertauthor=J.+Moscat&title=Targeted+disruption+of+the+PKC%CE%B6+gene+results+in+the+impairment+of+the+NF-%CE%BAB+pathway&doi=10.1016%2FS1097-2765%2801%2900361-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted disruption of the ζPKC gene results in the impairment of the NF-κB pathway</span></div><div class="casAuthors">Leitges, Michael; Sanz, Laura; Martin, Pilar; Duran, Angeles; Braun, Uschi; Garcia, Juan F.; Camacho, Fanny; Diaz-Meco, Maria T.; Rennert, Paul D.; Moscat, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">771-780</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Here the authors have addressed the role that ζPKC plays in NF-κB activation using mice in which this kinase was inactivated by homologous recombination.  These mice, although grossly normal, showed phenotypic alterations in secondary lymphoid organs reminiscent of those of the TNF receptor-1 and of the lymphotoxin-β receptor gene-deficient mice.  The lack of ζPKC in embryonic fibroblasts (EFs) severely impairs κB-dependent transcriptional activity as well as cytokine-induced phosphorylation of p65.  Also, a cytokine-inducible interaction of ζPKC with p65 was detected which requires the previous degrdn. of IκB.  Although in ζPKC-/- EFs this kinase is not necessary for IKK activation, in lung, which abundantly expresses ζPKC, IKK activation is inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod6cqcc6PgOrVg90H21EOLACvtfcHk0lgnbwDbnTc9Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXotlagur0%253D&md5=b8d6363ac857559db48a783ccdf6ea1e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900361-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900361-6%26sid%3Dliteratum%253Aachs%26aulast%3DLeitges%26aufirst%3DM.%26aulast%3DSanz%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DP.%26aulast%3DDuran%26aufirst%3DA.%26aulast%3DBraun%26aufirst%3DU.%26aulast%3DGarcia%26aufirst%3DJ.%2BF.%26aulast%3DCamacho%26aufirst%3DF.%26aulast%3DDiaz%2BMeco%26aufirst%3DM.%2BT.%26aulast%3DRennert%26aufirst%3DP.%2BD.%26aulast%3DMoscat%26aufirst%3DJ.%26atitle%3DTargeted%2520disruption%2520of%2520the%2520PKC%25CE%25B6%2520gene%2520results%2520in%2520the%2520impairment%2520of%2520the%2520NF-%25CE%25BAB%2520pathway%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D8%26spage%3D771%26epage%3D780%26doi%3D10.1016%2FS1097-2765%2801%2900361-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee-Rivera, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carranza-Perez, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Colome, A. M.</span></span> <span> </span><span class="NLM_article-title">PKC-ζ regulates thrombin-induced proliferation of human müller glial cells</span>. <i>Invest. Ophthalmol. Visual Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3769</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1167/iovs.16-19574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1167%2Fiovs.16-19574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=27421067" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFSnsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2016&pages=3769-3779&author=I.+Lee-Riveraauthor=E.+Lopezauthor=M.+G.+Carranza-Perezauthor=A.+M.+Lopez-Colome&title=PKC-%CE%B6+regulates+thrombin-induced+proliferation+of+human+m%C3%BCller+glial+cells&doi=10.1167%2Fiovs.16-19574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">PKC-ζ regulates thrombin-induced proliferation of human Muller glial cells</span></div><div class="casAuthors">Lee-Rivera, Irene; Lopez, Edith; Carranza-Perez, Miriam Gabriela; Lopez-Colome, Ana Maria</div><div class="citationInfo"><span class="NLM_cas:title">Investigative Ophthalmology & Visual Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3769-3779</span>CODEN:
                <span class="NLM_cas:coden">IOVSDA</span>;
        ISSN:<span class="NLM_cas:issn">1552-5783</span>.
    
            (<span class="NLM_cas:orgname">Association for Research in Vision and Ophthalmology</span>)
        </div><div class="casAbstract">PURPOSE. To investigate the effect of thrombin on the proliferation of human Muller glial cells (MCs) and define the possible signaling mechanisms involved in this process.  METHODS. Protease-activated receptor (PARs 1-4) expression was analyzed using RT-PCR and Western blot in the MIO-M1 Muller cell line (MC).  Muller cell proliferation was assessed by the MTS redn. method.  Wound healing and immunoreactivity to Ki67 antigen were used to dissoc. proliferation and migration.  Cell migration was examd. using transwell migration assays.  The involvement of extracellular signal-regulated kinase (ERK1/2) phosphorylation/activation in thrombin-induced human MC proliferation was detd. by Western blot.  Intracellular pathways involved in ERK1/2 activation were analyzed by pharmacol. inhibition.  RESULTS. We first demonstrated that human MCs express PARs 1 to 4.  Our results show that thrombin dose-dependently stimulates MC proliferation by 44%, with a calcd. Ec50 of 0.86 nM.  Muller cell maximal proliferation required sustained thrombin treatment for 72 h, in contrast to our previous findings in RPE cells showing maximal thrombin-induced proliferation at 24-h stimulation.  We demonstrate that thrombin induces MC cell proliferation through the Ras-independent activation of the Raf/MEK/ERK cascade, under the control of protein kinase C (PKC)-ζ.  CONCLUSIONS. The breakdown of blood-retina barrier (BRB) exposes MCs to thrombin contained in serum.  Our findings further strengthen the crit. involvement of thrombin in the development of proliferative retinopathies and may provide pharmacol. targets for the prevention or treatment of these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxhd3tSVgPm7Vg90H21EOLACvtfcHk0lhtOxqTWwarMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFSnsLjE&md5=996f466b10c7850241a9d4c3cbe3ec83</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1167%2Fiovs.16-19574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1167%252Fiovs.16-19574%26sid%3Dliteratum%253Aachs%26aulast%3DLee-Rivera%26aufirst%3DI.%26aulast%3DLopez%26aufirst%3DE.%26aulast%3DCarranza-Perez%26aufirst%3DM.%2BG.%26aulast%3DLopez-Colome%26aufirst%3DA.%2BM.%26atitle%3DPKC-%25CE%25B6%2520regulates%2520thrombin-induced%2520proliferation%2520of%2520human%2520m%25C3%25BCller%2520glial%2520cells%26jtitle%3DInvest.%2520Ophthalmol.%2520Visual%2520Sci.%26date%3D2016%26volume%3D57%26spage%3D3769%26epage%3D3779%26doi%3D10.1167%2Fiovs.16-19574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D.</span></span> <span> </span><span class="NLM_article-title">PKCζ as a promising therapeutic target for TNFα-induced inflammatory disorders in chronic cutaneous wounds</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1335</span>– <span class="NLM_lpage">1346</span>, <span class="refDoi"> DOI: 10.3892/ijmm.2017.3144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.3892%2Fijmm.2017.3144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=28949382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOrurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2017&pages=1335-1346&author=J.+Zhangauthor=X.+Yangauthor=H.+Wangauthor=B.+Zhaoauthor=X.+Wuauthor=L.+Suauthor=S.+Xieauthor=Y.+Wangauthor=J.+Liauthor=J.+Liuauthor=M.+Liuauthor=F.+Hanauthor=T.+Heauthor=W.+Zhangauthor=K.+Taoauthor=D.+Hu&title=PKC%CE%B6+as+a+promising+therapeutic+target+for+TNF%CE%B1-induced+inflammatory+disorders+in+chronic+cutaneous+wounds&doi=10.3892%2Fijmm.2017.3144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">PKCζ as a promising therapeutic target for TNFα-induced inflammatory disorders in chronic cutaneous wounds</span></div><div class="casAuthors">Zhang, Jian; Yang, Xuekang; Wang, Hongtao; Zhao, Bin; Wu, Xue; Su, Linlin; Xie, Songtao; Wang, Yunchuan; Li, Jun; Liu, Jiaqi; Liu, Mengdong; Han, Fu; He, Ting; Zhang, Wei; Tao, Ke; Hu, Dahai</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1335-1346</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Protein kinase Cζ (PKCζ) is a member of the atypical protein kinase C family.  Its roles in macrophages or skin-resident keratinocytes have not been fully evaluated.  In this study, we provide evidence that PKCζ mediates lipopolysaccharide (LPS)-induced tumor necrosis factor α (TNFα) gene expression in the mouse macrophage cell line, RAW264.7.  TNFα has been proven to be one of the main culprits of chronic wounds and impaired acute wounds, which are characterized by excessive inflammation, enhanced proteolysis and reduced matrix deposition.  Among the multiple effects of TNFα on keratinocytes, the induction of chemokines which are indispensable factors involved in the massive infiltration of various inflammatory cells into skin lesions serves as a crucial mechanism.  In the present study, we found that PKCζ inhibitor or its specific siRNA inhibited the TNFα-induced upregulation in the levels of the chemokines, interleukin (IL)-8, monocyte chemotactic protein-1 (MCP-1) and intercellular cell adhesion mol.-1 (ICAM-1) in HaCaT keratinocytes.  Moreover, under a disrupted inflammatory environment, activated keratinocytes can synthesize large amts. of matrix metalloproteinases (MMP), which has a neg. effect on tissue remodeling.  We discovered that TNFα promoted the expression of MMP9 in a PKCζ-dependent manner.  Further expts. revealed that nuclear factor-κB (NF-κB) was a key downstream mol. of PKCζ.  In addn., as shown in vitro, PKCζ was not involved in the TNFα-induced decrease in HaCaT cell migration and proliferation.  In vivo expts. demonstrated that TNFα-induced wound closure impairment and inflammatory disorders were significantly attenuated in the PKCζ inhibitor group.  On the whole, our findings suggest that PKCζ is a crucial regulator in LPS- or TNFα-induced inflammatory responses in RAW264.7 cells and HaCaT keratinocytes, and that PKCζ/ NF-κB signaling may be a potential target for interventional therapy for TNFα-induced skin inflammatory injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYMbUGV_L5DrVg90H21EOLACvtfcHk0lhtOxqTWwarMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOrurfK&md5=b65487ffba6929137a5364ce834da32f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2017.3144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2017.3144%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTao%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DD.%26atitle%3DPKC%25CE%25B6%2520as%2520a%2520promising%2520therapeutic%2520target%2520for%2520TNF%25CE%25B1-induced%2520inflammatory%2520disorders%2520in%2520chronic%2520cutaneous%2520wounds%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2017%26volume%3D40%26spage%3D1335%26epage%3D1346%26doi%3D10.3892%2Fijmm.2017.3144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anisul Islam, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolatos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolatos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acevedo-Duncan, M.</span></span> <span> </span><span class="NLM_article-title">Analysis of PKC-ζ protein levels in normal and malignant breast tissue subtypes</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1546</span>, <span class="refDoi"> DOI: 10.3892/ol.2018.9792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.3892%2Fol.2018.9792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=30675210" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1537-1546&author=T.+Smalleyauthor=S.+M.+Anisul+Islamauthor=C.+Apostolatosauthor=A.+Apostolatosauthor=M.+Acevedo-Duncan&title=Analysis+of+PKC-%CE%B6+protein+levels+in+normal+and+malignant+breast+tissue+subtypes&doi=10.3892%2Fol.2018.9792"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3892%2Fol.2018.9792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2018.9792%26sid%3Dliteratum%253Aachs%26aulast%3DSmalley%26aufirst%3DT.%26aulast%3DAnisul%2BIslam%26aufirst%3DS.%2BM.%26aulast%3DApostolatos%26aufirst%3DC.%26aulast%3DApostolatos%26aufirst%3DA.%26aulast%3DAcevedo-Duncan%26aufirst%3DM.%26atitle%3DAnalysis%2520of%2520PKC-%25CE%25B6%2520protein%2520levels%2520in%2520normal%2520and%2520malignant%2520breast%2520tissue%2520subtypes%26jtitle%3DOncol.%2520Lett.%26date%3D2018%26volume%3D17%26spage%3D1537%26epage%3D1546%26doi%3D10.3892%2Fol.2018.9792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anrather, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csizmadia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, H.</span></span> <span> </span><span class="NLM_article-title">Regulation of NF-κB RelA phosphorylation and transcriptional activity by p21(ras) and protein kinase C ζ in primary endothelial cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">13594</span>– <span class="NLM_lpage">13603</span>, <span class="refDoi"> DOI: 10.1074/jbc.274.19.13594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.274.19.13594" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10224130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFOlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=13594-13603&author=J.+Anratherauthor=V.+Csizmadiaauthor=M.+P.+Soaresauthor=H.+Winkler&title=Regulation+of+NF-%CE%BAB+RelA+phosphorylation+and+transcriptional+activity+by+p21%28ras%29+and+protein+kinase+C+%CE%B6+in+primary+endothelial+cells&doi=10.1074%2Fjbc.274.19.13594"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of NF-κB RelA phosphorylation and transcriptional activity by p21ras and protein kinase Cζ in primary endothelial cells</span></div><div class="casAuthors">Anrather, Josef; Csizmadia, Vilmos; Soares, Miguel P.; Winkler, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">13594-13603</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activity of the transcription factor NF-κB is thought to be regulated mainly through cytoplasmic retention by IκB mols.  Here we present evidence of a second mechanism of regulation acting on NF-κB after release from IκB.  In endothelial cells this mechanism involves phosphorylation of the RelA subunit of NF-κB through a pathway involving activation of protein kinase Cζ (PKCζ) and p21ras.  We show that transcriptional activity of RelA is dependent on phosphorylation of the N-terminal Rel homol. domain but not the C-terminal transactivation domain.  Inhibition of phosphorylation by dominant neg. mutants of PKCζ or p21ras results in loss of RelA transcriptional activity without interfering with DNA binding.  Raf/MEK, small GTPases, phosphatidylinositol 3-kinase, and stress-activated protein kinase pathways are not involved in this mechanism of regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp9UMgY6DG0bVg90H21EOLACvtfcHk0lhIhabt9qq2kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFOlu7c%253D&md5=f8aef80c60b9099eadbdaabc6b26cf0e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.19.13594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.19.13594%26sid%3Dliteratum%253Aachs%26aulast%3DAnrather%26aufirst%3DJ.%26aulast%3DCsizmadia%26aufirst%3DV.%26aulast%3DSoares%26aufirst%3DM.%2BP.%26aulast%3DWinkler%26aufirst%3DH.%26atitle%3DRegulation%2520of%2520NF-%25CE%25BAB%2520RelA%2520phosphorylation%2520and%2520transcriptional%2520activity%2520by%2520p21%2528ras%2529%2520and%2520protein%2520kinase%2520C%2520%25CE%25B6%2520in%2520primary%2520endothelial%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D13594%26epage%3D13603%26doi%3D10.1074%2Fjbc.274.19.13594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San-Antonio, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fresno, M.</span></span> <span> </span><span class="NLM_article-title">Regulation of nuclear factor κB transactivation. Implication of phosphatidylinositol 3-kinase and protein kinase C ζ in c-Rel activation by tumor necrosis factor α</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">15840</span>– <span class="NLM_lpage">15849</span>, <span class="refDoi"> DOI: 10.1074/jbc.M011313200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.M011313200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=11278885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFaks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=15840-15849&author=A.+G.+Martinauthor=B.+San-Antonioauthor=M.+Fresno&title=Regulation+of+nuclear+factor+%CE%BAB+transactivation.+Implication+of+phosphatidylinositol+3-kinase+and+protein+kinase+C+%CE%B6+in+c-Rel+activation+by+tumor+necrosis+factor+%CE%B1&doi=10.1074%2Fjbc.M011313200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of nuclear factor κB transactivation. Implication of phosphatidylinositol 3-kinase and protein kinase C ξ in c-Rel activation by tumor necrosis factor α</span></div><div class="casAuthors">Martin, Angel G.; San-Antonio, Belen; Fresno, Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">15840-15849</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transactivation by c-Rel (nuclear factor κB) was dependent on phosphorylation of several serines in the transactivation domain, indicating that it is a phosphorylation-dependent Ser-rich domain.  By Ser → Ala mutational and deletion anal., we have identified two regions in this domain: C-terminal region (amino acids 540-588), which is required for basal activity; and the 422-540 region, which responds to external stimuli as tumor necrosis factor (TNF) α or phorbol myristate acetate plus ionomycin.  Ser from 454 to 473 were shown to be required for TNFα-induced activation, whereas Ser between 492 and 519 were required for phorbol myristate acetate plus ionomycin activation.  Phosphatidylinositol 3-kinase (PI3K) and protein kinase C (PKC) ξ were identified as downstream signaling mols. of TNFα-activation of c-Rel transactivating activity.  Interestingly, dominant neg. forms of PI3K inhibited PKCξ activation and dominant neg. PKCξ inhibited PI3K-mediated activation of c-Rel transactivating activity, indicating a cross-talk between both enzymes.  We have identified the crit. role of different Ser for PKCξ- and PI3K-mediated responses.  Interestingly, those c-Rel mutants not only did not respond to TNFα but also acted as dominant neg. forms of nuclear factor κB activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU3sRdCE9CkrVg90H21EOLACvtfcHk0lhIhabt9qq2kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFaks7c%253D&md5=be9bde9f594ea5e63df27957a5f6a171</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M011313200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M011313200%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DA.%2BG.%26aulast%3DSan-Antonio%26aufirst%3DB.%26aulast%3DFresno%26aufirst%3DM.%26atitle%3DRegulation%2520of%2520nuclear%2520factor%2520%25CE%25BAB%2520transactivation.%2520Implication%2520of%2520phosphatidylinositol%25203-kinase%2520and%2520protein%2520kinase%2520C%2520%25CE%25B6%2520in%2520c-Rel%2520activation%2520by%2520tumor%2520necrosis%2520factor%2520%25CE%25B1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D15840%26epage%3D15849%26doi%3D10.1074%2Fjbc.M011313200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaVallie, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chockalingam, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins-Racie, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keohan, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, M.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C ζ is up-regulated in osteoarthritic cartilage and is required for activation of NF-κB by tumor necrosis factor and interleukin-1 in articular chondrocytes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">24124</span>– <span class="NLM_lpage">24137</span>, <span class="refDoi"> DOI: 10.1074/jbc.M601905200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.M601905200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=16798739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1eqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=24124-24137&author=E.+R.+LaVallieauthor=P.+S.+Chockalingamauthor=L.+A.+Collins-Racieauthor=B.+A.+Freemanauthor=C.+C.+Keohanauthor=M.+Leitgesauthor=A.+J.+Dornerauthor=E.+A.+Morrisauthor=M.+K.+Majumdarauthor=M.+Arai&title=Protein+kinase+C+%CE%B6+is+up-regulated+in+osteoarthritic+cartilage+and+is+required+for+activation+of+NF-%CE%BAB+by+tumor+necrosis+factor+and+interleukin-1+in+articular+chondrocytes&doi=10.1074%2Fjbc.M601905200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Cζ Is Up-regulated in Osteoarthritic Cartilage and Is Required for Activation of NF-κB by Tumor Necrosis Factor and Interleukin-1 in Articular Chondrocytes</span></div><div class="casAuthors">LaVallie, Edward R.; Chockalingam, Priya S.; Collins-Racie, Lisa A.; Freeman, Bethany A.; Keohan, Cristin C.; Leitges, Michael; Dorner, Andrew J.; Morris, Elisabeth A.; Majumdar, Manas K.; Arai, Maya</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">24124-24137</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein kinase Cζ (PKCζ) is an intracellular serine/threonine protein kinase that has been implicated in the signaling pathways for certain inflammatory cytokines, including interleukin-1 (IL-1) and tumor necrosis factor α (TNF-α), in some cell types.  A study of gene expression in articular chondrocytes from osteoarthritis (OA) patients revealed that PKCζ is transcriptionally up-regulated in human OA articular cartilage clin. samples.  This finding led to the hypothesis that PKCζ may be an important signaling component of cytokine-mediated cartilage matrix destruction in articular chondrocytes, believed to be an underlying factor in the pathophysiol. of OA.  IL-1 treatment of chondrocytes in culture resulted in rapidly increased phosphorylation of PKCζ, implicating PKCζ activation in the signaling pathway.  Chondrocyte cell-based assays were used to evaluate the contribution of PKCζ activity in NF-κB activation and extracellular matrix degrdn. mediated by IL-1, TNF, or sphingomyelinase.  In primary chondrocytes, IL-1 and TNF-α caused an increase in NF-κB activity resulting in induction of aggrecanase-1 and aggrecanase-2 expression, with consequent increased proteoglycan degrdn.  This effect was blocked by the pan-specific PKC inhibitors RO 31-8220 and bisindolylmaleimide I, partially blocked by Goe 6976, and was unaffected by the PKCζ-sparing inhibitor calphostin C.  A cell-permeable PKCζ pseudosubstrate peptide inhibitor was capable of blocking TNF and IL-1-mediated NF-κB activation and proteoglycan degrdn. in chondrocyte pellet cultures.  In addn., overexpression of a dominant neg. PKCζ protein effectively prevented cytokine-mediated NF-κB activation in primary chondrocytes.  These data implicate PKCζ as a necessary component of the IL-1 and TNF signaling pathways in chondrocytes that result in catabolic destruction of extracellular matrix proteins in osteoarthritic cartilage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNI_sLNyzO3rVg90H21EOLACvtfcHk0liEQzlKHhnwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1eqsro%253D&md5=57ad065d1c9a5089772e9f67d89d0e0e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M601905200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M601905200%26sid%3Dliteratum%253Aachs%26aulast%3DLaVallie%26aufirst%3DE.%2BR.%26aulast%3DChockalingam%26aufirst%3DP.%2BS.%26aulast%3DCollins-Racie%26aufirst%3DL.%2BA.%26aulast%3DFreeman%26aufirst%3DB.%2BA.%26aulast%3DKeohan%26aufirst%3DC.%2BC.%26aulast%3DLeitges%26aufirst%3DM.%26aulast%3DDorner%26aufirst%3DA.%2BJ.%26aulast%3DMorris%26aufirst%3DE.%2BA.%26aulast%3DMajumdar%26aufirst%3DM.%2BK.%26aulast%3DArai%26aufirst%3DM.%26atitle%3DProtein%2520kinase%2520C%2520%25CE%25B6%2520is%2520up-regulated%2520in%2520osteoarthritic%2520cartilage%2520and%2520is%2520required%2520for%2520activation%2520of%2520NF-%25CE%25BAB%2520by%2520tumor%2520necrosis%2520factor%2520and%2520interleukin-1%2520in%2520articular%2520chondrocytes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D24124%26epage%3D24137%26doi%3D10.1074%2Fjbc.M601905200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Meco, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscat, J.</span></span> <span> </span><span class="NLM_article-title">Essential role of RelA Ser311 phosphorylation by PKC in NF-B transcriptional activation</span>. <i>EMBO. J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3910</span>– <span class="NLM_lpage">3918</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdg370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1093%2Femboj%2Fcdg370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=12881425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlsl2ksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=3910-3918&author=A.+Duranauthor=M.+T.+Diaz-Mecoauthor=J.+Moscat&title=Essential+role+of+RelA+Ser311+phosphorylation+by+PKC+in+NF-B+transcriptional+activation&doi=10.1093%2Femboj%2Fcdg370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of RelA Ser311 phosphorylation by ζPKC in NF-κB transcriptional activation</span></div><div class="casAuthors">Duran, Angeles; Diaz-Meco, Maria T.; Moscat, Jorge</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3910-3918</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The activation of the transcription factor NF-κB is central to the control of the cellular response triggered by many stimuli.  Once released from the inhibitory mol. IκB, NF-κB is translocated to the nucleus, and it has to be phosphorylated to activate transcription.  In ζ protein kinase C (PKC)-deficient cells, NF-κB is transcriptionally inactive and the phosphorylation of the RelA subunit in response to tumor necrosis factor (TNF-α) is severely impaired.  In vitro assays showed that ζPKC directly phosphorylates RelA.  Here we demonstrate that Ser311 accounts for ζPKC phosphorylation of RelA and that this site is phosphorylated in vivo in response to TNF-α.  Also, an inactivating mutation of that residue severely impairs RelA transcriptional activity, blocks its anti-apoptotic function and abrogates the interaction of RelA with the co-activator CBP as well as its recruitment, and that of RNA polymerase II (Pol II) with the interleukin-6 (IL-6) promoter.  The interaction of endogenous CBP with endogenous RelA is inhibited in ζPKC-/- cells, as well as the binding of Pol II to the IL-6 promoter.  These results demonstrate the mechanism whereby ζPKC regulates NF-κB activation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqSrIUcVaTcrVg90H21EOLACvtfcHk0liEQzlKHhnwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlsl2ksrY%253D&md5=215ff3d929588df807b83ead61724272</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdg370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdg370%26sid%3Dliteratum%253Aachs%26aulast%3DDuran%26aufirst%3DA.%26aulast%3DDiaz-Meco%26aufirst%3DM.%2BT.%26aulast%3DMoscat%26aufirst%3DJ.%26atitle%3DEssential%2520role%2520of%2520RelA%2520Ser311%2520phosphorylation%2520by%2520PKC%2520in%2520NF-B%2520transcriptional%2520activation%26jtitle%3DEMBO.%2520J.%26date%3D2003%26volume%3D22%26spage%3D3910%26epage%3D3918%26doi%3D10.1093%2Femboj%2Fcdg370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Litherland, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catterall, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, A. D.</span></span> <span> </span><span class="NLM_article-title">Protein kinase C isoforms ζ and iota mediate collagenase expression and cartilage destruction via STAT3- and ERK-dependent c-fos induction</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">22414</span>– <span class="NLM_lpage">22425</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.120121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1074%2Fjbc.M110.120121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=20463008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1Wnt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=22414-22425&author=G.+J.+Litherlandauthor=M.+S.+Eliasauthor=W.+Huiauthor=C.+D.+Macdonaldauthor=J.+B.+Catterallauthor=M.+J.+Barterauthor=M.+J.+Farrenauthor=M.+Jeffersonauthor=A.+D.+Rowan&title=Protein+kinase+C+isoforms+%CE%B6+and+iota+mediate+collagenase+expression+and+cartilage+destruction+via+STAT3-+and+ERK-dependent+c-fos+induction&doi=10.1074%2Fjbc.M110.120121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase C Isoforms ζ and ι Mediate Collagenase Expression and Cartilage Destruction via STAT3- and ERK-dependent c-fos Induction</span></div><div class="casAuthors">Litherland, Gary J.; Elias, Martina S.; Hui, Wang; MacDonald, Christopher D.; Catterall, Jonathon B.; Barter, Matt J.; Farren, Matthew J.; Jefferson, Matthew; Rowan, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">22414-22425</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein kinase C (PKC) signaling pathway is a major regulator of cellular functions and is implicated in pathologies involving extracellular matrix remodeling.  Inflammatory joint disease is characterized by excessive extracellular matrix catabolism, and here we assess the role of PKC in the induction of the collagenases, matrix metalloproteinase (MMP)-1 and MMP-13, in human chondrocytes by the potent cytokine stimulus interleukin-1 (IL-1) in combination with oncostatin M (OSM).  IL-1 + OSM-stimulated collagenolysis and gelatinase activity were ameliorated by pharmacol. PKC inhibition in bovine cartilage, as was collagenase gene induction in human chondrocytes.  Small interfering RNA-mediated silencing of PKC gene expression showed that both novel (nPKCδ, nPKCη) and atypical (aPKCζ, aPKCι) isoforms were involved in collagenase induction by IL-1.  However, MMP1 and MMP13 induction by IL-1 + OSM was inhibited only by aPKC silencing, suggesting that only atypical isoforms play a significant role in complex inflammatory milieus.  Silencing of either aPKC led to diminished IL-1 + OSM-dependent extracellular signal-regulated kinase (ERK) and signal transducer and activator of transcription (STAT) 3 phosphorylation, and c-fos expression.  STAT3 gene silencing or ERK pathway inhibition also resulted in loss of IL-1 + OSM-stimulated c-fos and collagenase expression.  Silencing of c-fos and c-jun expression was sufficient to abrogate IL-1 + OSM-stimulated collagenase gene induction, and overexpression of both c-fos and c-jun was sufficient to drive transcription from the MMP1 promoter in the absence of a stimulus.  Our data identify atypical PKC isoenzymes as STAT and ERK activators that mediate c-fos and collagenase expression during IL-1 + OSM synergy in human chondrocytes.  APKCs may constitute potential therapeutic targets for inflammatory joint diseases involving increased collagenase expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnN40-vsGhG7Vg90H21EOLACvtfcHk0liEQzlKHhnwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1Wnt78%253D&md5=e80b96cafd40c4c158663bd8608cbdf7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.120121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.120121%26sid%3Dliteratum%253Aachs%26aulast%3DLitherland%26aufirst%3DG.%2BJ.%26aulast%3DElias%26aufirst%3DM.%2BS.%26aulast%3DHui%26aufirst%3DW.%26aulast%3DMacdonald%26aufirst%3DC.%2BD.%26aulast%3DCatterall%26aufirst%3DJ.%2BB.%26aulast%3DBarter%26aufirst%3DM.%2BJ.%26aulast%3DFarren%26aufirst%3DM.%2BJ.%26aulast%3DJefferson%26aufirst%3DM.%26aulast%3DRowan%26aufirst%3DA.%2BD.%26atitle%3DProtein%2520kinase%2520C%2520isoforms%2520%25CE%25B6%2520and%2520iota%2520mediate%2520collagenase%2520expression%2520and%2520cartilage%2520destruction%2520via%2520STAT3-%2520and%2520ERK-dependent%2520c-fos%2520induction%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D22414%26epage%3D22425%26doi%3D10.1074%2Fjbc.M110.120121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitson, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugni, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chockalingam, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concepcion, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, J. N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangalindan, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritacco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ireland, C. M.</span></span> <span> </span><span class="NLM_article-title">Spheciosterol sulfates, PKCζ inhibitors from a philippine sponge Spheciospongia sp</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi"> DOI: 10.1021/np8001628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np8001628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1cXntlaqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=1213-1217&author=E.+L.+Whitsonauthor=T.+S.+Bugniauthor=P.+S.+Chockalingamauthor=G.+P.+Concepcionauthor=M.+J.+Harperauthor=M.+Heauthor=J.+N.+A.+Hooperauthor=G.+C.+Mangalindanauthor=F.+Ritaccoauthor=C.+M.+Ireland&title=Spheciosterol+sulfates%2C+PKC%CE%B6+inhibitors+from+a+philippine+sponge+Spheciospongia+sp&doi=10.1021%2Fnp8001628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Spheciosterol Sulfates, PKCζ Inhibitors from a Philippine Sponge Spheciospongia sp.</span></div><div class="casAuthors">Whitson, Emily L.; Bugni, Tim S.; Chockalingam, Priya S.; Concepcion, Gisela P.; Harper, Mary Kay; He, Min; Hooper, John N. A.; Mangalindan, Gina C.; Ritacco, Frank; Ireland, Chris M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1217</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Three new sterol sulfates, spheciosterol sulfates A-C (I-III), and the known sterol sulfate topsentiasterol sulfate E (IV) have been isolated from the sponge Spheciospongia sp., collected in the Philippines.  Structures were assigned on the basis of extensive 1D and 2D NMR studies as well as anal. by HRESIMS.  Compds. I-IV inhibited PKCζ with IC50 values of 1.59, 0.53, 0.11, and 1.21 μM, resp.  In a cell-based assay, I-IV also inhibited NF-κB activation with EC50 values of 12-64 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8zpFO1Zuh2bVg90H21EOLACvtfcHk0liPCkbt8pCCNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXntlaqsrc%253D&md5=369b82403363cb5e40c7f897a0cbbc99</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fnp8001628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp8001628%26sid%3Dliteratum%253Aachs%26aulast%3DWhitson%26aufirst%3DE.%2BL.%26aulast%3DBugni%26aufirst%3DT.%2BS.%26aulast%3DChockalingam%26aufirst%3DP.%2BS.%26aulast%3DConcepcion%26aufirst%3DG.%2BP.%26aulast%3DHarper%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHooper%26aufirst%3DJ.%2BN.%2BA.%26aulast%3DMangalindan%26aufirst%3DG.%2BC.%26aulast%3DRitacco%26aufirst%3DF.%26aulast%3DIreland%26aufirst%3DC.%2BM.%26atitle%3DSpheciosterol%2520sulfates%252C%2520PKC%25CE%25B6%2520inhibitors%2520from%2520a%2520philippine%2520sponge%2520Spheciospongia%2520sp%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2008%26volume%3D71%26spage%3D1213%26epage%3D1217%26doi%3D10.1021%2Fnp8001628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzmer, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reitz, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegeman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrightstone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christine, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Compton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span> <span> </span><span class="NLM_article-title">2-(6-Phenyl-1H-indazol-3-yl)-1H-benzo[d]imidazoles: design and synthesis of a potent and isoform selective PKC-ζ inhibitor</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">908</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.11.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.bmcl.2008.11.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19097791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtF2itL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=908-911&author=J.+I.+Trujilloauthor=J.+R.+Kieferauthor=W.+Huangauthor=A.+Thorarensenauthor=L.+Xingauthor=N.+L.+Caspersauthor=J.+E.+Dayauthor=K.+J.+Mathisauthor=K.+K.+Kretzmerauthor=B.+A.+Reitzauthor=R.+A.+Weinbergauthor=R.+A.+Stegemanauthor=A.+Wrightstoneauthor=L.+Christineauthor=R.+Comptonauthor=X.+Li&title=2-%286-Phenyl-1H-indazol-3-yl%29-1H-benzo%5Bd%5Dimidazoles%3A+design+and+synthesis+of+a+potent+and+isoform+selective+PKC-%CE%B6+inhibitor&doi=10.1016%2Fj.bmcl.2008.11.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">2-(6-Phenyl-1H-indazol-3-yl)-1H-benzo[d]imidazoles: Design and synthesis of a potent and isoform selective PKC-ζ inhibitor</span></div><div class="casAuthors">Trujillo, John I.; Kiefer, James R.; Huang, Wei; Thorarensen, Atli; Xing, Li; Caspers, Nicole L.; Day, Jacqueline E.; Mathis, Karl J.; Kretzmer, Kuniko K.; Reitz, Beverley A.; Weinberg, Robin A.; Stegeman, Roderick A.; Wrightstone, Ann; Christine, Lori; Compton, Robert; Li, Xiong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">908-911</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The inhibition of PKC-ζ has been proposed to be a potential drug target for immune and inflammatory diseases.  A series of 2-(6-phenyl-1H indazol-3-yl)-1H-benzo[d]imidazoles with initial high crossover to CDK-2 has been optimized to afford potent and selective inhibitors of protein kinase c-zeta (PKC-ζ).  The detn. of the crystal structures of key inhibitor:CDK-2 complexes informed the design and anal. of the series.  The most selective and potent analog was identified by variation of the aryl substituent at the 6-position of the indazole template to give a 4-NH2 deriv.  The analog displays good selectivity over other PKC isoforms (α, βII, γ, δ, ε, μ, θ, η and ι/λ) and CDK-2, however it displays marginal selectivity against a panel of other kinases (37 profiled).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYSeSuwVwvzbVg90H21EOLACvtfcHk0liPCkbt8pCCNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtF2itL0%253D&md5=62c38c95188c1f42be3175d72a681c7d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.11.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.11.105%26sid%3Dliteratum%253Aachs%26aulast%3DTrujillo%26aufirst%3DJ.%2BI.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DXing%26aufirst%3DL.%26aulast%3DCaspers%26aufirst%3DN.%2BL.%26aulast%3DDay%26aufirst%3DJ.%2BE.%26aulast%3DMathis%26aufirst%3DK.%2BJ.%26aulast%3DKretzmer%26aufirst%3DK.%2BK.%26aulast%3DReitz%26aufirst%3DB.%2BA.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26aulast%3DStegeman%26aufirst%3DR.%2BA.%26aulast%3DWrightstone%26aufirst%3DA.%26aulast%3DChristine%26aufirst%3DL.%26aulast%3DCompton%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26atitle%3D2-%25286-Phenyl-1H-indazol-3-yl%2529-1H-benzo%255Bd%255Dimidazoles%253A%2520design%2520and%2520synthesis%2520of%2520a%2520potent%2520and%2520isoform%2520selective%2520PKC-%25CE%25B6%2520inhibitor%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D908%26epage%3D911%26doi%3D10.1016%2Fj.bmcl.2008.11.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjaer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostelecky, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray-Rust, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roffey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdonald, N. Q.</span></span> <span> </span><span class="NLM_article-title">Adenosine-binding motif mimicry and cellular effects of a thieno[2,3-d]pyrimidine-based chemical inhibitor of atypical protein kinase C isoenzymes</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.1042/BJ20121871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1042%2FBJ20121871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=23418854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2013&pages=329-342&author=S.+Kjaerauthor=M.+Linchauthor=A.+Purkissauthor=B.+Kosteleckyauthor=P.+P.+Knowlesauthor=C.+Rosseauthor=P.+Riouauthor=C.+Soudyauthor=S.+Kayeauthor=B.+Patelauthor=E.+Sorianoauthor=J.+Murray-Rustauthor=C.+Bartonauthor=C.+Dillonauthor=J.+Roffeyauthor=P.+J.+Parkerauthor=N.+Q.+Mcdonald&title=Adenosine-binding+motif+mimicry+and+cellular+effects+of+a+thieno%5B2%2C3-d%5Dpyrimidine-based+chemical+inhibitor+of+atypical+protein+kinase+C+isoenzymes&doi=10.1042%2FBJ20121871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine-binding motif mimicry and cellular effects of a thieno[2,3-d]pyrimidine-based chemical inhibitor of atypical protein kinase C isoenzymes</span></div><div class="casAuthors">Kjaer, Svend; Linch, Mark; Purkiss, Andrew; Kostelecky, Brenda; Knowles, Phillip P.; Rosse, Carine; Riou, Philippe; Soudy, Christelle; Kaye, Sarah; Patel, Bhavisha; Soriano, Erika; Murray-Rust, Judith; Barton, Caroline; Dillon, Christian; Roffey, Jon; Parker, Peter J.; McDonald, Neil Q.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">329-342</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The aPKC [atypical PKC (protein kinase C)] isoforms ι and ζ play crucial roles in the formation and maintenance of cell polarity and represent attractive anti-oncogenic drug targets in Ras-dependent tumors.  To date, few isoform-specific chem. biol. tools are available to inhibit aPKC catalytic activity.  In the present paper, we describe the identification and functional characterization of potent and selective thieno[2,3-d]pyrimidine-based chem. inhibitors of aPKCs.  A crystal structure of human PKCι kinase domain bound to a representative compd., CRT0066854, reveals the basis for potent and selective chem. inhibition.  Furthermore, CRT0066854 displaces a crucial Asn-Phe-Asp motif that is part of the adenosine-binding pocket and engages an acidic patch used by arginine-rich PKC substrates.  We show that CRT0066854 inhibits the LLGL2 (lethal giant larvae 2) phosphorylation in cell lines and exhibits phenotypic effects in a range of cell-based assays.  We conclude that this compd. can be used as a chem. tool to modulate aPKC activity in vitro and in vivo and may guide the search for further aPKC-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5lEC0hIc6q7Vg90H21EOLACvtfcHk0liPCkbt8pCCNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7k%253D&md5=332ef49dabbbf6e2333a101ef338c3c5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1042%2FBJ20121871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121871%26sid%3Dliteratum%253Aachs%26aulast%3DKjaer%26aufirst%3DS.%26aulast%3DLinch%26aufirst%3DM.%26aulast%3DPurkiss%26aufirst%3DA.%26aulast%3DKostelecky%26aufirst%3DB.%26aulast%3DKnowles%26aufirst%3DP.%2BP.%26aulast%3DRosse%26aufirst%3DC.%26aulast%3DRiou%26aufirst%3DP.%26aulast%3DSoudy%26aufirst%3DC.%26aulast%3DKaye%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DB.%26aulast%3DSoriano%26aufirst%3DE.%26aulast%3DMurray-Rust%26aufirst%3DJ.%26aulast%3DBarton%26aufirst%3DC.%26aulast%3DDillon%26aufirst%3DC.%26aulast%3DRoffey%26aufirst%3DJ.%26aulast%3DParker%26aufirst%3DP.%2BJ.%26aulast%3DMcdonald%26aufirst%3DN.%2BQ.%26atitle%3DAdenosine-binding%2520motif%2520mimicry%2520and%2520cellular%2520effects%2520of%2520a%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520chemical%2520inhibitor%2520of%2520atypical%2520protein%2520kinase%2520C%2520isoenzymes%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D451%26spage%3D329%26epage%3D342%26doi%3D10.1042%2FBJ20121871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fröhner, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garcia, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neimanis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navratil, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">4-benzimidazolyl-3-phenylbutanoic acids as novel PIF-pocket-targeting allosteric inhibitors of protein kinase PKCζ</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6714</span>– <span class="NLM_lpage">6723</span>, <span class="refDoi"> DOI: 10.1021/jm2005892</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2005892" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A280%3ADC%252BC3MfoslOjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6714-6723&author=W.+Fr%C3%B6hnerauthor=L.+A.+Lopez-Garciaauthor=S.+Neimanisauthor=N.+Weberauthor=J.+Navratilauthor=F.+Maurerauthor=A.+Strobaauthor=H.+Zhangauthor=R.+M.+Biondiauthor=M.+Engel&title=4-benzimidazolyl-3-phenylbutanoic+acids+as+novel+PIF-pocket-targeting+allosteric+inhibitors+of+protein+kinase+PKC%CE%B6&doi=10.1021%2Fjm2005892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">4-benzimidazolyl-3-phenylbutanoic acids as novel PIF-pocket-targeting allosteric inhibitors of protein kinase PKCζ</span></div><div class="casAuthors">Frohner Wolfgang; Lopez-Garcia Laura A; Neimanis Sonja; Weber Nadja; Navratil Jeanette; Maurer Frauke; Stroba Adriana; Zhang Hua; Biondi Ricardo M; Engel Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6714-23</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein kinase inhibitors with an allosteric mode of action are expected to reach, in many cases, higher selectivity for the target enzyme than ATP-competitive compounds.  Therefore, basic research is aiming at identifying and establishing novel sites on the catalytic domain of protein kinases which might be targeted by allosteric inhibitors.  We previously published the first structure-activity relationships (SARs) for allosteric activators of protein kinase PDK1.  Here, we present the design, synthesis, and SAR data on a series of novel compounds, 4-benzimidazolyl-3-phenylbutanoic acids, that inhibit the atypical protein kinace C (PKC) ζ via binding to the PIF-pocket.  Key positions were identified in the compounds that can be modified to increase potency and selectivity.  Some congeners showed a high selectivity toward PKCζ, lacking inhibition of the most closely related isoform, PKCι, and of further AGC kinases.  Furthermore, evidence is provided that these compounds are also active toward cellular PKCζ without loss of potency compared to the cell-free assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBGw-v0ps7djQLBVDQn4bCfW6udTcc2eZpmJrpdxtUx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfoslOjsg%253D%253D&md5=274fee7a71f82dbc9eb086a6ef81d96e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm2005892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2005892%26sid%3Dliteratum%253Aachs%26aulast%3DFr%25C3%25B6hner%26aufirst%3DW.%26aulast%3DLopez-Garcia%26aufirst%3DL.%2BA.%26aulast%3DNeimanis%26aufirst%3DS.%26aulast%3DWeber%26aufirst%3DN.%26aulast%3DNavratil%26aufirst%3DJ.%26aulast%3DMaurer%26aufirst%3DF.%26aulast%3DStroba%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26aulast%3DEngel%26aufirst%3DM.%26atitle%3D4-benzimidazolyl-3-phenylbutanoic%2520acids%2520as%2520novel%2520PIF-pocket-targeting%2520allosteric%2520inhibitors%2520of%2520protein%2520kinase%2520PKC%25CE%25B6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6714%26epage%3D6723%26doi%3D10.1021%2Fjm2005892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Halim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diesel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiemer, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abadi, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of 1,3,5-trisubstituted pyrazolines as potent and highly selective allosteric inhibitors of protein kinase C-ζ</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6513</span>– <span class="NLM_lpage">6530</span>, <span class="refDoi"> DOI: 10.1021/jm500521n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500521n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1WmsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6513-6530&author=M.+Abdel-Halimauthor=B.+Dieselauthor=A.+K.+Kiemerauthor=A.+H.+Abadiauthor=R.+W.+Hartmannauthor=M.+Engel&title=Discovery+and+optimization+of+1%2C3%2C5-trisubstituted+pyrazolines+as+potent+and+highly+selective+allosteric+inhibitors+of+protein+kinase+C-%CE%B6&doi=10.1021%2Fjm500521n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of 1,3,5-Trisubstituted Pyrazolines as Potent and Highly Selective Allosteric Inhibitors of Protein Kinase C-ζ</span></div><div class="casAuthors">Abdel-Halim, Mohammad; Diesel, Britta; Kiemer, Alexandra K.; Abadi, Ashraf H.; Hartmann, Rolf W.; Engel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6513-6530</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is increasing evidence that the atypical protein kinase C, PKCζ, might be a therapeutic target in pulmonary and hepatic inflammatory diseases.  However, targeting the highly conserved ATP-binding pocket in the catalytic domain held little promise to achieve selective inhibition.  In the present study, the authors introduce 1,3,5-trisubstituted pyrazolines, e.g. I [R1 = 4-HO, 4-F, 4-Cl, etc.; R2 = Me, Ph, t-Bu, etc.; R3 = H, 4-Cl, 4-Br, etc.; R4 = H, Me] as potent and selective allosteric PKCζ inhibitors.  The rigid scaffold offered many sites for modification, all acting as hot spots for improving activity, and gave rise to sharp structure-activity relationships.  Targeting of PKCζ in cells was confirmed by reporter gene assay, transfection assays, and Western blotting.  The strongly reduced cell-free and cellular activities toward a PIF-pocket mutant of PKCζ suggested that the inhibitors most likely bound to the PIF-pocket on the kinase catalytic domain.  Thus, using a rigidification strategy and by establishing and optimizing multiple mol. interactions with the binding site, the authors were able to significantly improve the potency of the previously reported PKCζ inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiEjZcMLasYrVg90H21EOLACvtfcHk0lijovNKw40FqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1WmsrzJ&md5=95c3f2c59aabf54afb97632e8bd8ff7c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm500521n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500521n%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Halim%26aufirst%3DM.%26aulast%3DDiesel%26aufirst%3DB.%26aulast%3DKiemer%26aufirst%3DA.%2BK.%26aulast%3DAbadi%26aufirst%3DA.%2BH.%26aulast%3DHartmann%26aufirst%3DR.%2BW.%26aulast%3DEngel%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520optimization%2520of%25201%252C3%252C5-trisubstituted%2520pyrazolines%2520as%2520potent%2520and%2520highly%2520selective%2520allosteric%2520inhibitors%2520of%2520protein%2520kinase%2520C-%25CE%25B6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6513%26epage%3D6530%26doi%3D10.1021%2Fjm500521n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span> <span> </span><span class="NLM_article-title">Fragment-based lead discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1038/nrd1467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-based+lead+discovery&doi=10.1038%2Fnrd1467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lijovNKw40FqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-based%2520lead%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D660%26epage%3D672%26doi%3D10.1038%2Fnrd1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Fragment Based Lead Discovery</span>.  <i>Handbook of Medicinal Chemistry: Principles and Practice</i>; <span class="NLM_publisher-name">Royal Society of Chemistry</span>, <span class="NLM_year">2015</span>; Chapter 5, pp  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">153</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=R.+E.+Hubbard&title=Fragment+Based+Lead+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DFragment%2520Based%2520Lead%2520Discovery%26jtitle%3DHandbook%2520of%2520Medicinal%2520Chemistry%253A%2520Principles%2520and%2520Practice%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2015%26spage%3D122%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Lessons for fragment library design: analysis of output from multiple screening campaigns</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9280-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1007%2Fs10822-009-9280-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19495994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=603-610&author=I.+J.+Chenauthor=R.+E.+Hubbard&title=Lessons+for+fragment+library+design%3A+analysis+of+output+from+multiple+screening+campaigns&doi=10.1007%2Fs10822-009-9280-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons for fragment library design: analysis of output from multiple screening campaigns</span></div><div class="casAuthors">Chen, I.-Jen; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-620</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past 8 years, we have developed, refined and applied a fragment based discovery approach to a range of protein targets.  Here we report computational analyses of various aspects of our fragment library and the results obtained for fragment screening.  We reinforce the finding of others that the exptl. obsd. hit rate for screening fragments can be related to a computationally defined druggability index for the target.  In general, the physicochem. properties of the fragment hits display the same profile as the library, as is expected for a truly diverse library which probes the relevant chem. space.  An anal. of the fragment hits against various protein classes has shown that the physicochem. properties of the fragments are complementary to the properties of the target binding site.  The effectiveness of some fragments appears to be achieved by an appropriate mix of pharmacophore features and enhanced aromaticity, with hydrophobic interactions playing an important role.  The anal. emphasizes that it is possible to identify small fragments that are specific for different binding sites.  To conclude, we discuss how the results could inform further development and improvement of our fragment library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo4yzAG3RSbVg90H21EOLACvtfcHk0lhOyMdNPU91uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D&md5=f544180cae108c2d7365037388df0262</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9280-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9280-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLessons%2520for%2520fragment%2520library%2520design%253A%2520analysis%2520of%2520output%2520from%2520multiple%2520screening%2520campaigns%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D603%26epage%3D610%26doi%3D10.1007%2Fs10822-009-9280-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaibuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakoshima, T.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanism for the regulation of rho-kinase by dimerization and its inhibition by fasudil</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">600</span>, <span class="refDoi"> DOI: 10.1016/j.str.2005.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.str.2005.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=16531242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD28Xit1ylt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=589-600&author=H.+Yamaguchiauthor=M.+Kasaauthor=M.+Amanoauthor=K.+Kaibuchiauthor=T.+Hakoshima&title=Molecular+mechanism+for+the+regulation+of+rho-kinase+by+dimerization+and+its+inhibition+by+fasudil&doi=10.1016%2Fj.str.2005.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism for the Regulation of Rho-Kinase by Dimerization and Its Inhibition by Fasudil</span></div><div class="casAuthors">Yamaguchi, Hiroto; Kasa, Miyuki; Amano, Mutsuki; Kaibuchi, Kozo; Hakoshima, Toshio</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-600</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Rho-kinase is a key regulator of cytoskeletal events and a promising drug target in the treatment of vascular diseases and neurol. disorders.  Unlike other protein kinases, Rho-kinase requires both N- and C-terminal extension segments outside the kinase domain for activity, although the details of this requirement have been elusive.  The crystal structure of an active Rho-kinase fragment contg. the kinase domain and both the extensions revealed a head-to-head homodimer through the N-terminal extension forming a helix bundle that structurally integrates the C-terminal extension.  This structural organization enables binding of the C-terminal hydrophobic motif to the N-terminal lobe, which defines the correct disposition of helix αC that is important for the catalytic activity.  The bound inhibitor fasudil significantly alters the conformation and, consequently, the mode of interaction with the catalytic cleft that contains local structural changes.  Thus, both kinase and drug conformational pliability and stability confer selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKCe7VapDGkLVg90H21EOLACvtfcHk0lhOyMdNPU91uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xit1ylt78%253D&md5=c00104ab0fe7731e51b68b3762ee894a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DKasa%26aufirst%3DM.%26aulast%3DAmano%26aufirst%3DM.%26aulast%3DKaibuchi%26aufirst%3DK.%26aulast%3DHakoshima%26aufirst%3DT.%26atitle%3DMolecular%2520mechanism%2520for%2520the%2520regulation%2520of%2520rho-kinase%2520by%2520dimerization%2520and%2520its%2520inhibition%2520by%2520fasudil%26jtitle%3DStructure%26date%3D2006%26volume%3D14%26spage%3D589%26epage%3D600%26doi%3D10.1016%2Fj.str.2005.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span>; <span class="NLM_string-name">Milburn, M. V.</span>; <span class="NLM_string-name">Artis, D. R.</span></span> <span> </span><span class="NLM_article-title">Scaffold-Based Drug Discovery</span>.  <i>Structure-Based Drug Discovery</i>; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2007</span>; Chapter 6, pp  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">153</span>, DOI: <span class="refDoi"> DOI: 10.1007/1-4020-4407-0_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1007%2F1-4020-4407-0_6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=129-153&author=K.+Y.+J.+Zhang&author=M.+V.+Milburn&author=D.+R.+Artis&title=Structure-Based+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2F1-4020-4407-0_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F1-4020-4407-0_6%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DScaffold-Based%2520Drug%2520Discovery%26btitle%3DStructure-Based%2520Drug%2520Discovery%26pub%3DSpringer%26date%3D2007%26spage%3D129%26epage%3D153%26doi%3D10.1007%2F1-4020-4407-0_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haass, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sproesser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settachatgul, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artis, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective inhibitor of oncologenic B-Raf kinase with potent antimelanoma activity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">3041</span>– <span class="NLM_lpage">3046</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711741105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1073%2Fpnas.0711741105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=18287029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=3041-3046&author=J.+Tsaiauthor=J.+T.+Leeauthor=W.+Wangauthor=J.+Zhangauthor=H.+Choauthor=S.+Mamoauthor=R.+Bremerauthor=S.+Gilletteauthor=J.+Kongauthor=N.+K.+Haassauthor=K.+Sproesserauthor=L.+Liauthor=K.+S.+M.+Smalleyauthor=D.+Fongauthor=Y.-L.+Zhuauthor=A.+Marimuthuauthor=H.+Nguyenauthor=B.+Lamauthor=J.+Liuauthor=I.+Cheungauthor=J.+Riceauthor=Y.+Suzukiauthor=C.+Luuauthor=C.+Settachatgulauthor=R.+Shellooeauthor=J.+Cantwellauthor=S.-H.+Kimauthor=J.+Schlessingerauthor=K.+Y.+J.+Zhangauthor=B.+L.+Westauthor=B.+Powellauthor=G.+Habetsauthor=C.+Zhangauthor=P.+N.+Ibrahimauthor=P.+Hirthauthor=D.+R.+Artisauthor=M.+Herlynauthor=G.+Bollag&title=Discovery+of+a+selective+inhibitor+of+oncologenic+B-Raf+kinase+with+potent+antimelanoma+activity&doi=10.1073%2Fpnas.0711741105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span></div><div class="casAuthors">Tsai, James; Lee, John T.; Wang, Weiru; Zhang, Jiazhong; Cho, Hanna; Mamo, Shumeye; Bremer, Ryan; Gillette, Sam; Kong, Jun; Haass, Nikolas K.; Sproesser, Katrin; Li, Ling; Smalley, Keiran S. M.; Fong, Daniel; Zhu, Yong-Liang; Marimuthu, Adhirai; Nguyen, Hoa; Lam, Billy; Liu, Jennifer; Cheung, Ivana; Rice, Julie; Suzuki, Yoshihisa; Luu, Catherine; Settachatgul, Calvin; Shellooe, Rafe; Cantwell, John; Kim, Sung-Hou; Schlessinger, Joseph; Zhang, Kam Y. J.; West, Brian L.; Powell, Ben; Habets, Gaston; Zhang, Chao; Ibrahim, Prabha N.; Hirth, Peter; Artis, Dean R.; Herlyn, Meenhard; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3041-3046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">BRAFV600E is the most frequent oncogenic protein kinase mutation known.  Furthermore, inhibitors targeting "active" protein kinases have demonstrated significant utility in the therapeutic repertoire against cancer.  Therefore, the authors pursued the development of specific kinase inhibitors targeting B-Raf, and the V600E allele in particular.  By using a structure-guided discovery approach, a potent and selective inhibitor of active B-Raf has been discovered.  PLX4720, a 7-azaindole deriv. that inhibits B-RafV600E with an IC50 of 13 nM, defines a class of kinase inhibitor with marked selectivity in both biochem. and cellular assays.  PLX4720 preferentially inhibits the active B-RafV600E kinase compared with a broad spectrum of other kinases, and potent cytotoxic effects are also exclusive to cells bearing the V600E allele.  Consistent with the high degree of selectivity, ERK phosphorylation is potently inhibited by PLX4720 in B-RafV600E-bearing tumor cell lines but not in cells lacking oncogenic B-Raf.  In melanoma models, PLX4720 induces cell cycle arrest and apoptosis exclusively in B-RafV600E-pos. cells.  In B-RafV600E-dependent tumor xenograft models, orally dosed PLX4720 causes significant tumor growth delays, including tumor regressions, without evidence of toxicity.  The work described here represents the entire discovery process, from initial identification through structural and biol. studies in animal models to a promising therapeutic for testing in cancer patients bearing B-RafV600E-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPX53IQCbDJ7Vg90H21EOLACvtfcHk0lj8CxuaC0CmJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D&md5=273b94202f99984e6eb24723d0c6141f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711741105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711741105%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DMamo%26aufirst%3DS.%26aulast%3DBremer%26aufirst%3DR.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHaass%26aufirst%3DN.%2BK.%26aulast%3DSproesser%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.-L.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DLuu%26aufirst%3DC.%26aulast%3DSettachatgul%26aufirst%3DC.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.-H.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520inhibitor%2520of%2520oncologenic%2520B-Raf%2520kinase%2520with%2520potent%2520antimelanoma%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D3041%26epage%3D3046%26doi%3D10.1073%2Fpnas.0711741105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Opportunity knocks: organic chemistry for fragment-based drug discovery (FBDD)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1002/anie.201506783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fanie.201506783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=488-492&author=C.+W.+Murrayauthor=D.+C.+Rees&title=Opportunity+knocks%3A+organic+chemistry+for+fragment-based+drug+discovery+%28FBDD%29&doi=10.1002%2Fanie.201506783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunity Knocks: Organic Chemistry for Fragment-Based Drug Discovery (FBDD)</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">488-492</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj8GEY4dDXxrVg90H21EOLACvtfcHk0lj8CxuaC0CmJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O&md5=97a92911033a2304da101888da0c3642</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.201506783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201506783%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DOpportunity%2520knocks%253A%2520organic%2520chemistry%2520for%2520fragment-based%2520drug%2520discovery%2520%2528FBDD%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D488%26epage%3D492%26doi%3D10.1002%2Fanie.201506783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuster, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillooly, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dambach, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattoli, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zusi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span> <span> </span><span class="NLM_article-title">A highly selective inhibitor of I kappa B kinase, BMS-345541, blocks both joint inflammation and destruction in collagen-induced arthritis in mice</span>. <i>Arthritis Rheum.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2652</span>– <span class="NLM_lpage">2659</span>, <span class="refDoi"> DOI: 10.1002/art.11131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fart.11131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=13130486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3sXot1Wjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2003&pages=2652-2659&author=K.+W.+McIntyreauthor=D.+J.+Shusterauthor=K.+M.+Gilloolyauthor=D.+M.+Dambachauthor=M.+A.+Pattoliauthor=P.+Luauthor=X.-D.+Zhouauthor=Y.+Qiuauthor=C.+Zusiauthor=J.+R.+Burke&title=A+highly+selective+inhibitor+of+I+kappa+B+kinase%2C+BMS-345541%2C+blocks+both+joint+inflammation+and+destruction+in+collagen-induced+arthritis+in+mice&doi=10.1002%2Fart.11131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A highly selective inhibitor of I.vkappa.B kinase, BMS-345541, blocks both joint inflammation and destruction in collagen-induced arthritis in mice</span></div><div class="casAuthors">McIntyre, Kim W.; Shuster, David J.; Gillooly, Kathleen M.; Dambach, Donna M.; Pattoli, Mark A.; Lu, Pin; Zhou, Xia-Di; Qiu, Yuping; Zusi, F. Christopher; Burke, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2652-2659</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The transcription of several cytokines, cell adhesion mols., and enzymes involved in the inflammatory and destructive mechanisms of rheumatoid arthritis is dependent on nuclear factor .vkappa.B (NF-.vkappa.B).  Because I.vkappa.B kinase (IKK) is crit. in transducing the signal-inducible activation of NF-.vkappa.B, we examd. whether the highly selective and orally bioavailable IKK inhibitor BMS-345541 is efficacious against collagen-induced arthritis (CIA) in mice.  Arthritis in DBA/1LacJ male mice was induced by s.c. immunization with, bovine type II collagen on day 0 and day 21.  BMS-345541 was administered perorally daily, either prophylactically (before disease onset) or therapeutically (after disease onset).  Clin. assessment of the incidence and severity of disease was conducted throughout the study, and histol. evaluation was performed at the time of study termination (day 42).  When administered prophylactically, BMS-345541 (in a dose range of 10-100 mg/kg) was effective, in a dose-dependent manner, in reducing the incidence of disease and inhibiting clin. signs of disease.  Histol. evaluation of the joints showed that both inflammation and joint destruction were blocked by the IKK inhibitor.  Message levels of interleukin-1β in the joints were also dose-dependently inhibited in the mice that received BMS-345541.  Dose-dependent efficacy in terms of both disease severity and histol. end points was obsd. with the therapeutic dosing regimen of BMS-345541, with use of the 100-mg/kg dose resulting in resoln. of disease.  IKK plays a key role in CIA in mice, and inhibitors of this enzyme represent a promising target for the development of novel agents to treat rheumatoid arthritis and other inflammatory diseases.  BMS-345541 represents the first example of an inhibitor of IKK that has antiinflammatory activity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrethUYjO8ZkLVg90H21EOLACvtfcHk0lj8CxuaC0CmJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXot1Wjt7Y%253D&md5=c560529ace35642a0700a8bc04ea8eae</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fart.11131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.11131%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntyre%26aufirst%3DK.%2BW.%26aulast%3DShuster%26aufirst%3DD.%2BJ.%26aulast%3DGillooly%26aufirst%3DK.%2BM.%26aulast%3DDambach%26aufirst%3DD.%2BM.%26aulast%3DPattoli%26aufirst%3DM.%2BA.%26aulast%3DLu%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DX.-D.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DZusi%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26atitle%3DA%2520highly%2520selective%2520inhibitor%2520of%2520I%2520kappa%2520B%2520kinase%252C%2520BMS-345541%252C%2520blocks%2520both%2520joint%2520inflammation%2520and%2520destruction%2520in%2520collagen-induced%2520arthritis%2520in%2520mice%26jtitle%3DArthritis%2520Rheum.%26date%3D2003%26volume%3D48%26spage%3D2652%26epage%3D2659%26doi%3D10.1002%2Fart.11131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerner, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimos, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z. K.</span></span> <span> </span><span class="NLM_article-title">An atypical protein kinase C (PKCζ) plays a critical role in lipopolysaccharide-activated NF-κB in human peripheral blood monocytes and macrophages</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">5810</span>– <span class="NLM_lpage">5815</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.0804073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.4049%2Fjimmunol.0804073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=19380829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslKhsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2009&pages=5810-5815&author=X.+Huangauthor=L.-Y.+Chenauthor=A.+M.+Doernerauthor=W.+W.+Panauthor=L.+Smithauthor=S.+Huangauthor=T.+J.+Papadimosauthor=Z.+K.+Pan&title=An+atypical+protein+kinase+C+%28PKC%CE%B6%29+plays+a+critical+role+in+lipopolysaccharide-activated+NF-%CE%BAB+in+human+peripheral+blood+monocytes+and+macrophages&doi=10.4049%2Fjimmunol.0804073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">An Atypical Protein Kinase C (PKCζ) Plays a Critical Role in Lipopolysaccharide-Activated NF-κB in Human Peripheral Blood Monocytes and Macrophages</span></div><div class="casAuthors">Huang, XueSong; Chen, Ling-Yu; Doerner, Astrid M.; Pan, Warren W.; Smith, Laura; Huang, Shuang; Papadimos, Thomas J.; Pan, Zhixing K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5810-5815</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">The authors have reported that the bacterial LPS induces the activation of NF-κB and inflammatory cytokine gene expression and that this requires the activity of small GTPase, RhoA.  Here, they show that an atypical protein kinase C isoenzyme, PKCζ, assocs. functionally with RhoA and that PKCζ acts as a signaling component downstream of RhoA.  Stimulation of monocytes and macrophages with LPS resulted in PKCζ activation and inhibition of PKCζ activity blocked both LPS-stimulated activation of NF-κB and IL-1β gene expression.  The results also indicate that transforming growth factor β-activated kinase 1 (TAK1) acts as a signaling component downstream of PKCζ in cytokine gene transcription stimulated by LPS in human peripheral blood monocytes and macrophages.  The specificity of this response suggests an important role for the Rho GTPase/PKCζ/TAK1/NF-κB pathway in host defense and in proinflammatory cytokine synthesis induced by bacterial LPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQszyQcZ2kXLVg90H21EOLACvtfcHk0ljozbHLAh5uHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslKhsbY%253D&md5=c1dcc782d7737fe543e0f051d39a4448</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.0804073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.0804073%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.-Y.%26aulast%3DDoerner%26aufirst%3DA.%2BM.%26aulast%3DPan%26aufirst%3DW.%2BW.%26aulast%3DSmith%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DPapadimos%26aufirst%3DT.%2BJ.%26aulast%3DPan%26aufirst%3DZ.%2BK.%26atitle%3DAn%2520atypical%2520protein%2520kinase%2520C%2520%2528PKC%25CE%25B6%2529%2520plays%2520a%2520critical%2520role%2520in%2520lipopolysaccharide-activated%2520NF-%25CE%25BAB%2520in%2520human%2520peripheral%2520blood%2520monocytes%2520and%2520macrophages%26jtitle%3DJ.%2520Immunol.%26date%3D2009%26volume%3D182%26spage%3D5810%26epage%3D5815%26doi%3D10.4049%2Fjimmunol.0804073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span> <span> </span><span class="NLM_article-title">Substrate recognition mechanism of atypical protein kinase Cs revealed by the structure of PKCι in complex with a substrate peptide from Part-3</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">791</span>– <span class="NLM_lpage">801</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.str.2012.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=22579248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=791-801&author=C.+Wangauthor=Y.+Shangauthor=J.+Yuauthor=M.+Zhang&title=Substrate+recognition+mechanism+of+atypical+protein+kinase+Cs+revealed+by+the+structure+of+PKC%CE%B9+in+complex+with+a+substrate+peptide+from+Part-3&doi=10.1016%2Fj.str.2012.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Recognition Mechanism of Atypical Protein Kinase Cs Revealed by the Structure of PKCι in Complex with a Substrate Peptide from Par-3</span></div><div class="casAuthors">Wang, Chihao; Shang, Yuan; Yu, Jiang; Zhang, Mingjie</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">791-801</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein kinase C (PKC) play crit. roles in many cellular functions including differentiation, proliferation, growth, and survival.  However, the mol. bases governing PKC's substrate recognitions remain poorly understood.  Here we detd. the structure of PKCι in complex with a peptide from Par-3 at 2.4 Å.  PKCι in the complex adopts catalytically competent, closed conformation without phosphorylation of Thr402 in the activation loop.  The Par-3 peptide binds to an elongated groove formed by the N- and C-lobes of the kinase domain.  The PKCι/Par-3 complex structure, together with extensive biochem. studies, reveals a set of substrate recognition sites common to all PKC isoenzymes as well as a hydrophobic pocket unique to aPKC.  A consensus aPKC's substrate recognition sequence pattern can be readily identified based on the complex structure.  Finally, we demonstrate that the pseudosubstrate sequence of PKCι resembles its substrate sequence, directly binds to and inhibits the activity of the kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEP6WuO1OB2bVg90H21EOLACvtfcHk0ljozbHLAh5uHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahs78%253D&md5=4aa93528111174f5d6dfa7c508f54a03</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShang%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26atitle%3DSubstrate%2520recognition%2520mechanism%2520of%2520atypical%2520protein%2520kinase%2520Cs%2520revealed%2520by%2520the%2520structure%2520of%2520PKC%25CE%25B9%2520in%2520complex%2520with%2520a%2520substrate%2520peptide%2520from%2520Part-3%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D791%26epage%3D801%26doi%3D10.1016%2Fj.str.2012.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokubo, H.</span></span> <span> </span><span class="NLM_article-title">Exploring the stability of ligand binding modes to proteins by molecular dynamics simulations: a cross-docking study</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2514</span>– <span class="NLM_lpage">2522</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00412</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00412" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVOrsb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2514-2522&author=K.+Liuauthor=H.+Kokubo&title=Exploring+the+stability+of+ligand+binding+modes+to+proteins+by+molecular+dynamics+simulations%3A+a+cross-docking+study&doi=10.1021%2Facs.jcim.7b00412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Stability of Ligand Binding Modes to Proteins by Molecular Dynamics Simulations: A Cross-docking Study</span></div><div class="casAuthors">Liu, Kai; Kokubo, Hironori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2514-2522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking has become an indispensable approach in drug discovery research to predict the binding mode of a ligand.  One great challenge in docking is to efficiently refine the correct pose from various putative docking poses through score functions.  The authors recently examd. the stability of self-docking poses under mol. dynamics (MD) simulations and showed that equil. MD simulations have some capability to discriminate between correct and decoy poses.  Here, the authors have extended the previous work to cross-docking studies for practical applications.  Three target proteins (thrombin, heat shock protein 90-alpha, and cyclin-dependent kinase 2) of pharmaceutical interest were selected.  Three comparable poses (one correct pose and two decoys) for each ligand were then selected from the docking poses.  To obtain the docking poses for the three target proteins, the authors used three different protocols; namely, normal docking, induced fit docking (IFD), and IFD against the homol. model.  Finally, five parallel MD equil. runs were performed on each pose for the statistical anal.  The results showed that the correct poses were generally more stable than the decoy poses under MD.  The discrimination capability of MD depends on the strategy.  The safest way was to judge a pose as being stable if any one run among five parallel runs was stable under MD.  In this case, 95% of the correct poses were retained under MD, and about 25-44% of the decoys could be excluded by the simulations for all cases.  On the other hand, if the authors judge a pose as being stable when any two or three runs were stable, with the risk of incorrectly excluding some correct poses, approx. 31-53% or 39-56% of the two decoys could be excluded by MD, resp.  The results suggest that simple equil. simulations can serve as an effective filter to exclude decoy poses that cannot be distinguished by docking scores from the computationally expensive free-energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxgZngG4j75LVg90H21EOLACvtfcHk0ljozbHLAh5uHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVOrsb%252FI&md5=dfe3ab369e356abb2c27190b026792d2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00412%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DKokubo%26aufirst%3DH.%26atitle%3DExploring%2520the%2520stability%2520of%2520ligand%2520binding%2520modes%2520to%2520proteins%2520by%2520molecular%2520dynamics%2520simulations%253A%2520a%2520cross-docking%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2514%26epage%3D2522%26doi%3D10.1021%2Facs.jcim.7b00412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span> <span> </span><span class="NLM_article-title">Pair interaction molecular orbital method: an approximate computational method for molecular interactions</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>312</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(99)00937-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS0009-2614%2899%2900937-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ymtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=312&publication_year=1999&pages=319-324&author=K.+Kitauraauthor=T.+Sawaiauthor=T.+Asadaauthor=T.+Nakanoauthor=M.+Uebayasi&title=Pair+interaction+molecular+orbital+method%3A+an+approximate+computational+method+for+molecular+interactions&doi=10.1016%2FS0009-2614%2899%2900937-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pair interaction molecular orbital method: an approximate computational method for molecular interactions</span></div><div class="casAuthors">Kitaura, Kazuo; Sawai, Takuya; Asada, Toshio; Nakano, Tatsuya; Uebayasi, Masami</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">312</span>
        (<span class="NLM_cas:issue">2-4</span>),
    <span class="NLM_cas:pages">319-324</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We propose an approx. method for calcg. large mol. clusters using MO methods.  The method allows one to obtain the total energy of a mol. cluster by performing MO calcns. on the mols. and the mol. pairs in the system.  The computational time is greatly reduced for a system contg. a large no. of mols., since the supermol. calcn. of the whole system is avoided.  Our method considers important terms of many-body interactions within a pair approxn. and is expected to give results of reasonable accuracy even for mol. interactions having significant many-body effects.  The numerical calcns. of water trimers and tetramers revealed that our method included the major portion of the many-body energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWDa8nVDQk9LVg90H21EOLACvtfcHk0ljozbHLAh5uHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ymtLY%253D&md5=c748640d22f5194740c842a6aafc8531</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2900937-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252900937-9%26sid%3Dliteratum%253Aachs%26aulast%3DKitaura%26aufirst%3DK.%26aulast%3DSawai%26aufirst%3DT.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DUebayasi%26aufirst%3DM.%26atitle%3DPair%2520interaction%2520molecular%2520orbital%2520method%253A%2520an%2520approximate%2520computational%2520method%2520for%2520molecular%2520interactions%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D312%26spage%3D319%26epage%3D324%26doi%3D10.1016%2FS0009-2614%2899%2900937-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span> <span> </span><span class="NLM_article-title">Fragment molecular orbital method: an approximate computational method for large molecules</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(99)00874-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS0009-2614%2899%2900874-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFChu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=1999&pages=701-706&author=K.+Kitauraauthor=E.+Ikeoauthor=T.+Asadaauthor=T.+Nakanoauthor=M.+Uebayasi&title=Fragment+molecular+orbital+method%3A+an+approximate+computational+method+for+large+molecules&doi=10.1016%2FS0009-2614%2899%2900874-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment molecular orbital method: an approximate computational method for large molecules</span></div><div class="casAuthors">Kitaura, K.; Ikeo, E.; Asada, T.; Nakano, T.; Uebayasi, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">3,4</span>),
    <span class="NLM_cas:pages">701-706</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">An approx. MO method was proposed for calcg. large mols. such as proteins.  This method assigns the electrons of the mols. to fragments, and the MOs of fragments and fragment pairs are calcd. to obtain the total energy of the mol.  The method avoids the MO calcn. of the whole mol., and is expected to reduce the computational time drastically for large mols.  Numerical calcns. were performed on C3H8, PrOH and AcNHMe to show the accuracy of the method.  The optimized geometries and total energies were in good agreement with those from the ab initio MO calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwzZTYpLNVNbVg90H21EOLACvtfcHk0linlZy-2wPb0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFChu7Y%253D&md5=7096f34748e293b1ebe2e488b271e2df</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2900874-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252900874-X%26sid%3Dliteratum%253Aachs%26aulast%3DKitaura%26aufirst%3DK.%26aulast%3DIkeo%26aufirst%3DE.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DUebayasi%26aufirst%3DM.%26atitle%3DFragment%2520molecular%2520orbital%2520method%253A%2520an%2520approximate%2520computational%2520method%2520for%2520large%2520molecules%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D313%26spage%3D701%26epage%3D706%26doi%3D10.1016%2FS0009-2614%2899%2900874-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuzawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Fragment molecular orbital method: use of approximate electrostatic potential</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">480</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(01)01416-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2FS0009-2614%2801%2901416-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsFCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2002&pages=475-480&author=T.+Nakanoauthor=T.+Kaminumaauthor=T.+Satoauthor=K.+Fukuzawaauthor=Y.+Akiyamaauthor=M.+Uebayasiauthor=K.+Kitaura&title=Fragment+molecular+orbital+method%3A+use+of+approximate+electrostatic+potential&doi=10.1016%2FS0009-2614%2801%2901416-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment molecular orbital method: use of approximate electrostatic potential</span></div><div class="casAuthors">Nakano, Tatsuya; Kaminuma, Tsuguchika; Sato, Toshiyuki; Fukuzawa, Kaori; Akiyama, Yutaka; Uebayasi, Masami; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">5,6</span>),
    <span class="NLM_cas:pages">475-480</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Recently, we have proposed the fragment MO (FMO) method, an approx. MO method for calcg. large mols. such as proteins.  The method has been shown to reproduce ab initio total energies and geometries of mols. in good accuracy.  The most time consuming part in the method, the calcns. of environmental electrostatic potentials, were speeded up by employing the Mulliken approxn. for two-electron integrals and a fractional point charge approxn.  Numerical calcns. on several polypeptides revealed that the approxns. brought no significant loss of accuracy in the total energy of mols. and were of practical use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-OUIE6YzWiLVg90H21EOLACvtfcHk0linlZy-2wPb0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsFCkug%253D%253D&md5=e0bd6c422cacba5105c5a3d6f6059874</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2801%2901416-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252801%252901416-6%26sid%3Dliteratum%253Aachs%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DKaminuma%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DFukuzawa%26aufirst%3DK.%26aulast%3DAkiyama%26aufirst%3DY.%26aulast%3DUebayasi%26aufirst%3DM.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DFragment%2520molecular%2520orbital%2520method%253A%2520use%2520of%2520approximate%2520electrostatic%2520potential%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D2002%26volume%3D351%26spage%3D475%26epage%3D480%26doi%3D10.1016%2FS0009-2614%2801%2901416-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Pair interaction energy decomposition analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1002/jcc.20496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.20496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=17109433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=222-237&author=D.+G.+Fedorovauthor=K.+Kitaura&title=Pair+interaction+energy+decomposition+analysis&doi=10.1002%2Fjcc.20496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Pair interaction energy decomposition analysis</span></div><div class="casAuthors">Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">222-237</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The energy decompn. anal. (EDA) by Kitaura and Morokuma was redeveloped in the framework of the fragment MO method (FMO).  The proposed pair interaction energy decompn. anal. (PIEDA) can treat large mol. clusters and the systems in which fragments are connected by covalent bonds, such as proteins.  The interaction energy in PIEDA is divided into the same contributions as in EDA: the electrostatic, exchange-repulsion, and charge transfer energies, to which the correlation (dispersion) term was added.  The careful comparison to the ab initio EDA interaction energies for water clusters with 2-16 mols. revealed that PIEDA has the error of at most 1.2 kcal/mol (or about 1%).  The anal. was applied to (H2O)1024, the α helix, β turn, and β strand of polyalanine (ALA)10, as well as to the synthetic protein (PDB code 1L2Y) with 20 residues.  The comparative aspects of the polypeptide isomer stability are discussed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ89I3W8YX1LVg90H21EOLACvtfcHk0linlZy-2wPb0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D&md5=759c711290fe1c44b7c86a28104ccef7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20496%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DPair%2520interaction%2520energy%2520decomposition%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D222%26epage%3D237%26doi%3D10.1002%2Fjcc.20496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barder, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Catalysts for Suzuki-Miyaura coupling processes: scope and studies of the effect of ligand structure</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">4685</span>– <span class="NLM_lpage">4696</span>, <span class="refDoi"> DOI: 10.1021/ja042491j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja042491j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVWrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=4685-4696&author=T.+E.+Barderauthor=S.+D.+Walkerauthor=J.+R.+Martinelliauthor=S.+L.+Buchwald&title=Catalysts+for+Suzuki-Miyaura+coupling+processes%3A+scope+and+studies+of+the+effect+of+ligand+structure&doi=10.1021%2Fja042491j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Catalysts for Suzuki-Miyaura Coupling Processes: Scope and Studies of the Effect of Ligand Structure</span></div><div class="casAuthors">Barder, Timothy E.; Walker, Shawn D.; Martinelli, Joseph R.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4685-4696</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Suzuki-Miyaura coupling reactions of aryl and heteroaryl halides with aryl-, heteroaryl- and vinylboronic acids proceed in very good to excellent yield with the use of 2-(2',6'-dimethoxybiphenyl)dicyclohexylphosphine, SPhos (1) (I).  This ligand confers unprecedented activity for these processes, allowing reactions to be performed at low catalyst levels, to prep. extremely hindered biaryls and to be carried out, in general, for reactions of aryl chlorides at room temp.  Addnl., structural studies of various 1·Pd complexes are presented along with computational data that help elucidate the efficacy that 1 imparts on Suzuki-Miyaura coupling processes.  Moreover, a comparison of the reactions with 1 and with 2-(2',4',6'-triisopropylbiphenyl)diphenylphosphine (2) (II) is presented that is informative in detg. the relative importance of ligand bulk and electron-donating ability in the high activity of catalysts derived from ligands of this type.  Further, when the aryl bromide becomes too hindered, an interesting C-H bond functionalization-cross-coupling sequence intervenes to provide product in high yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP20DMwGPU0LVg90H21EOLACvtfcHk0linlZy-2wPb0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVWrsbc%253D&md5=af3f56a57a2165183ddf23b73f099a2a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fja042491j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042491j%26sid%3Dliteratum%253Aachs%26aulast%3DBarder%26aufirst%3DT.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BD.%26aulast%3DMartinelli%26aufirst%3DJ.%2BR.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DCatalysts%2520for%2520Suzuki-Miyaura%2520coupling%2520processes%253A%2520scope%2520and%2520studies%2520of%2520the%2520effect%2520of%2520ligand%2520structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D4685%26epage%3D4696%26doi%3D10.1021%2Fja042491j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alagille, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamagnan, G. D.</span></span> <span> </span><span class="NLM_article-title">One-step synthesis of 2-arylbenzothiazole (′BTA′) and benzoxazole precursors for in vivo imaging of β-amyloid plaques</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1349</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2004.12.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2Fj.tetlet.2004.12.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2MXosF2rtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=1349-1351&author=D.+Alagilleauthor=R.+M.+Baldwinauthor=G.+D.+Tamagnan&title=One-step+synthesis+of+2-arylbenzothiazole+%28%E2%80%B2BTA%E2%80%B2%29+and+benzoxazole+precursors+for+in+vivo+imaging+of+%CE%B2-amyloid+plaques&doi=10.1016%2Fj.tetlet.2004.12.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">One-step synthesis of 2-arylbenzothiazole (BTA) and -benzoxazole precursors for in vivo imaging of β-amyloid plaques</span></div><div class="casAuthors">Alagille, David; Baldwin, Ronald M.; Tamagnan, Gilles D.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1349-1351</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The authors report the simple and efficient synthesis of 2-arylbenzothiazoles (BTA) and 2-arylbenzoxazoles by direct coupling of benzothiazoles or benzoxazoles with aryl bromides.  This method permits direct one-step access to precursors of radiolabeled BTA-1 and BTA-2 and their 6-methoxy analogs, used for in vivo imaging of β-amyloid plaques with positron emission tomog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovcoP2lcbxK7Vg90H21EOLACvtfcHk0lj1xVSJhT9svg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXosF2rtQ%253D%253D&md5=563b72aad86176693ab8e6a593d473c4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2004.12.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2004.12.111%26sid%3Dliteratum%253Aachs%26aulast%3DAlagille%26aufirst%3DD.%26aulast%3DBaldwin%26aufirst%3DR.%2BM.%26aulast%3DTamagnan%26aufirst%3DG.%2BD.%26atitle%3DOne-step%2520synthesis%2520of%25202-arylbenzothiazole%2520%2528%25E2%2580%25B2BTA%25E2%2580%25B2%2529%2520and%2520benzoxazole%2520precursors%2520for%2520in%2520vivo%2520imaging%2520of%2520%25CE%25B2-amyloid%2520plaques%26jtitle%3DTetrahedron%2520Lett.%26date%3D2005%26volume%3D46%26spage%3D1349%26epage%3D1351%26doi%3D10.1016%2Fj.tetlet.2004.12.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Billingsley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Highly efficient monophosphine-based catalyst for the palladium-catalyzed Suzuki-Miyaura reaction of heteroaryl halides and heteroaryl boronic acids and esters</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">3358</span>– <span class="NLM_lpage">3366</span>, <span class="refDoi"> DOI: 10.1021/ja068577p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja068577p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitFKrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=3358-3366&author=K.+Billingsleyauthor=S.+L.+Buchwald&title=Highly+efficient+monophosphine-based+catalyst+for+the+palladium-catalyzed+Suzuki-Miyaura+reaction+of+heteroaryl+halides+and+heteroaryl+boronic+acids+and+esters&doi=10.1021%2Fja068577p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Efficient Monophosphine-Based Catalyst for the Palladium-Catalyzed Suzuki-Miyaura Reaction of Heteroaryl Halides and Heteroaryl Boronic Acids and Esters</span></div><div class="casAuthors">Billingsley, Kelvin; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3358-3366</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly active and efficient catalyst system derived from a palladium precatalyst and monophosphine ligands I or II for the Suzuki-Miyaura cross-coupling reaction of heteroaryl boronic acids and esters has been developed.  This method allows for the prepn. of a wide variety of heterobiaryls in good to excellent yields and displays a high level of activity for the coupling of heteroaryl chlorides as well as hindered aryl and heteroaryl halides.  Specific factors that govern the efficacy of the transformation for certain heterocyclic motifs were also investigated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlSHWf5e2EBbVg90H21EOLACvtfcHk0lj1xVSJhT9svg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitFKrsrs%253D&md5=a9e65a6c356774a7340edd3b413080cf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja068577p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja068577p%26sid%3Dliteratum%253Aachs%26aulast%3DBillingsley%26aufirst%3DK.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DHighly%2520efficient%2520monophosphine-based%2520catalyst%2520for%2520the%2520palladium-catalyzed%2520Suzuki-Miyaura%2520reaction%2520of%2520heteroaryl%2520halides%2520and%2520heteroaryl%2520boronic%2520acids%2520and%2520esters%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D3358%26epage%3D3366%26doi%3D10.1021%2Fja068577p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Old, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">A highly active catalyst for palladium-catalyzed cross-coupling reactions: room-temperature Suzuki couplings and amination of unactivated aryl chlorides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">9722</span>– <span class="NLM_lpage">9723</span>, <span class="refDoi"> DOI: 10.1021/ja982250+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja982250%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK1cXlslOhsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=9722-9723&author=D.+W.+Oldauthor=J.+P.+Wolfeauthor=S.+L.+Buchwald&title=A+highly+active+catalyst+for+palladium-catalyzed+cross-coupling+reactions%3A+room-temperature+Suzuki+couplings+and+amination+of+unactivated+aryl+chlorides&doi=10.1021%2Fja982250%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A Highly Active Catalyst for Palladium-Catalyzed Cross-Coupling Reactions: Room-Temperature Suzuki Couplings and Amination of Unactivated Aryl Chlorides</span></div><div class="casAuthors">Old, David W.; Wolfe, John P.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9722-9723</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pd2(dba)3 and ligand 2-(dicyclohexylphosphino)-2'-(dimethylamino)biphenyl catalyzed the amination of aryl chloride or bromides at room temp.  Also, Pd2(dba)3 or Pd(OAc)2 and ligand 2-(dicyclohexylphosphino)-2'-(dimethylamino)biphenyl catalyzed the Suzuki coupling of aryl chloride or bromides with boron reagents at room temp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsj7rB61O95LVg90H21EOLACvtfcHk0lj1xVSJhT9svg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlslOhsr8%253D&md5=6713de2eb0f19db80339a4fc2c94321e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja982250%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja982250%252B%26sid%3Dliteratum%253Aachs%26aulast%3DOld%26aufirst%3DD.%2BW.%26aulast%3DWolfe%26aufirst%3DJ.%2BP.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DA%2520highly%2520active%2520catalyst%2520for%2520palladium-catalyzed%2520cross-coupling%2520reactions%253A%2520room-temperature%2520Suzuki%2520couplings%2520and%2520amination%2520of%2520unactivated%2520aryl%2520chlorides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D9722%26epage%3D9723%26doi%3D10.1021%2Fja982250%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">An overview of the Amber biomolecular simulation package</span>. <i>WIREs Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1002/wcms.1121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fwcms.1121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtFaiu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=198-210&author=R.+Salomon-Ferrerauthor=D.+A.+Caseauthor=R.+C.+Walker&title=An+overview+of+the+Amber+biomolecular+simulation+package&doi=10.1002%2Fwcms.1121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the amber biomolecular simulation package</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Case, David A.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: Computational Molecular Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">198-210</span>CODEN:
                <span class="NLM_cas:coden">WIRCAH</span>;
        ISSN:<span class="NLM_cas:issn">1759-0884</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics (MD) allows the study of biol. and chem. systems at the atomistic level on timescales from femtoseconds to milliseconds.  It complements expt. while also offering a way to follow processes difficult to discern with exptl. techniques.  Numerous software packages exist for conducting MD simulations of which one of the widest used is termed Amber.  Here, we outline the most recent developments, since version 9 was released in Apr. 2006, of the Amber and AmberTools MD software packages, referred to here as simply the Amber package.  The latest release represents six years of continued development, since version 9, by multiple research groups and the culmination of over 33 years of work beginning with the first version in 1979.  The latest release of the Amber package, version 12 released in Apr. 2012, includes a substantial no. of important developments in both the scientific and computer science arenas.  We present here a condensed vision of what Amber currently supports and where things are likely to head over the coming years.  Figure 1 shows the performance in ns/day of the Amber package version 12 on a single-core AMD FX-8120 8-Core 3.6GHz CPU, the Cray XT5 system, and a single GPU GTX680.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1JUOzTqkwZrVg90H21EOLACvtfcHk0lgZwoouRGPxVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtFaiu70%253D&md5=baac68008e08e80bd825b2362e14fa1f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1121%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DAn%2520overview%2520of%2520the%2520Amber%2520biomolecular%2520simulation%2520package%26jtitle%3DWIREs%2520Comput.%2520Mol.%2520Sci.%26date%3D2013%26volume%3D3%26spage%3D198%26epage%3D210%26doi%3D10.1002%2Fwcms.1121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasavajhala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">ff14SB: improving the accuracy of protein side chain and backbone parameters from ff99SB</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3696</span>– <span class="NLM_lpage">3713</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0lgZwoouRGPxVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lgZwoouRGPxVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C.</span></span> <span> </span><span class="NLM_article-title">I. Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(20000130)21:2<132::AID-JCC5>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+Bayly&title=I.+Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method&doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0lgZwoouRGPxVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%26atitle%3DI.%2520Fast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146%26doi%3D10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C.</span></span> <span> </span><span class="NLM_article-title">I. Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+Bayly&title=I.+Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lh28yfi6raaig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%26atitle%3DI.%2520Fast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+Impeyauthor=M.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lh28yfi6raaig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DM.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G.</span></span> <span> </span><span class="NLM_article-title">A smooth particle mesh Ewald method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0lh28yfi6raaig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loncharich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span> <span> </span><span class="NLM_article-title">Langevin dynamics of peptides: the frictional dependence of isomerization rates of <i>N</i>-acetylalanyl-<i>N</i>′-methylamide</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">535</span>, <span class="refDoi"> DOI: 10.1002/bip.360320508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fbip.360320508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1515543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK38XisFGqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1992&pages=523-535&author=R.+J.+Loncharichauthor=B.+R.+Brooksauthor=R.+W.+Pastor&title=Langevin+dynamics+of+peptides%3A+the+frictional+dependence+of+isomerization+rates+of+N-acetylalanyl-N%E2%80%B2-methylamide&doi=10.1002%2Fbip.360320508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Langevin dynamics of peptides:  the frictional dependence of isomerization rates of N-acetylalanyl-N'-methylamide</span></div><div class="casAuthors">Loncharich, Richard J.; Brooks, Bernard R.; Pastor, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">523-35</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">The rate const. for the transition between the equatorial and axial conformations of N-acetylalanyl-N'-methylamide has been detd. from Langevin dynamics (LD) simulations with no explicit solvent.  The isomerization rate is max. at collision frequency γ = 2 ps-1, shows diffusive character for γ ≥ 10 ps-1, but does not approach zero even at γ = 0.01 ps-1.  This behavior differs from that found for a one-dimensional bistable potential and indicates that both collisional energy transfer with solvent and vibrational energy transfer between internal modes are important in the dynamics of barrier crossing for this system.  It is suggested that conformational searches of peptides be carried out using LD with a collision frequency that maximizes the isomerization rate (i.e., γ ≈ 2 ps-1).  This method is expected to be more efficient than either mol. dynamics in vacuo (which corresponds to LD with γ = 0) or mol. dynamics in solvent (where dynamics is largely diffusive).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXONM8fktQybVg90H21EOLACvtfcHk0lj_GPJB5E3dyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XisFGqu7o%253D&md5=9209a0b3485915887d1b03fefcbadc35</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fbip.360320508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360320508%26sid%3Dliteratum%253Aachs%26aulast%3DLoncharich%26aufirst%3DR.%2BJ.%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26atitle%3DLangevin%2520dynamics%2520of%2520peptides%253A%2520the%2520frictional%2520dependence%2520of%2520isomerization%2520rates%2520of%2520N-acetylalanyl-N%25E2%2580%25B2-methylamide%26jtitle%3DBiopolymers%26date%3D1992%26volume%3D32%26spage%3D523%26epage%3D535%26doi%3D10.1002%2Fbip.360320508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, A.</span></span> <span> </span><span class="NLM_article-title">An analysis of the accuracy of Langevin and molecular dynamics algorithm</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1419</span>, <span class="refDoi"> DOI: 10.1080/00268978800101881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1080%2F00268978800101881" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1988&pages=1409-1419&author=R.+W.+Pastorauthor=B.+R.+Brooksauthor=A.+Szabo&title=An+analysis+of+the+accuracy+of+Langevin+and+molecular+dynamics+algorithm&doi=10.1080%2F00268978800101881"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1080%2F00268978800101881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268978800101881%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DR.%2BW.%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DSzabo%26aufirst%3DA.%26atitle%3DAn%2520analysis%2520of%2520the%2520accuracy%2520of%2520Langevin%2520and%2520molecular%2520dynamics%2520algorithm%26jtitle%3DMol.%2520Phys.%26date%3D1988%26volume%3D65%26spage%3D1409%26epage%3D1419%26doi%3D10.1080%2F00268978800101881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van
Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics with coupling to an external bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van%0AGunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+dynamics+with+coupling+to+an+external+bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lj_GPJB5E3dyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520dynamics%2520with%2520coupling%2520to%2520an%2520external%2520bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.+P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lj_GPJB5E3dyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.%2BP.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">Settle: an analytical version of the SHAKE and RATTLE algorithm for rigid water models</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1002/jcc.540130805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.540130805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK38Xlslykt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=952-962&author=S.+Miyamotoauthor=P.+A.+Kollman&title=Settle%3A+an+analytical+version+of+the+SHAKE+and+RATTLE+algorithm+for+rigid+water+models&doi=10.1002%2Fjcc.540130805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">SETTLE:  an analytical version of the SHAKE and RATTLE algorithm for rigid water models</span></div><div class="casAuthors">Miyamoto, Shuichi; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">952-62</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An anal. algorithm, called SETTLE, for resetting the positions and velocities to satisfy the holonomic constraints on the rigid water model is presented.  This method is based on the Cartesian coordinate system and can be used in place of SHAKE and RATTLE.  The authors implemented this algorithm in the SPASMS package of mol. mechanics and dynamics.  Several series of mol. dynamics simulations were carried out to examine the performance of the new algorithm in comparison with the original RATTLE method.  SETTLE is of higher accuracy and is faster than RATTLE with reasonable tolerances by three to nine times on a scalar machine.  The performance improvement ranged from factors of 26 to 98 on a vector machine since the method presented is not iterative.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAdRqj2ObejLVg90H21EOLACvtfcHk0lj_GPJB5E3dyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xlslykt7o%253D&md5=65da9d55c7905abeaf7708d91a09e6e4</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130805%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DS.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DSettle%253A%2520an%2520analytical%2520version%2520of%2520the%2520SHAKE%2520and%2520RATTLE%2520algorithm%2520for%2520rigid%2520water%2520models%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D952%26epage%3D962%26doi%3D10.1002%2Fjcc.540130805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boatz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbert, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windus, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span> <span> </span><span class="NLM_article-title">General atomic and molecular electronic structure system</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1002/jcc.540141112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=10.1002%2Fjcc.540141112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=1347-1363&author=M.+W.+Schmidtauthor=K.+Baldridgeauthor=J.+Boatzauthor=S.+T.+Elbertauthor=M.+S.+Gordonauthor=J.+H.+Jensenauthor=S.+Kosekiauthor=N.+Matsunagaauthor=K.+A.+Nguyenauthor=S.+J.+Suauthor=T.+L.+Windusauthor=M.+Dupuisauthor=J.+Montgomery&title=General+atomic+and+molecular+electronic+structure+system&doi=10.1002%2Fjcc.540141112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">General atomic and molecular electronic structure system</span></div><div class="casAuthors">Schmidt, Michael W.; Baldridge, Kim K.; Boatz, Jerry A.; Elbert, Steven T.; Gordon, Mark S.; Jensen, Jan H.; Koseki, Shiro; Matsunaga, Nikita; Nguyen, Kiet A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1347-63</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A description of the ab initio quantum chem. package GAMESS is presented.  Chem. systems contg. atoms through Rn can be treated with wave functions ranging from the simplest closed-shell case up to a general MCSCF case, permitting calcns. at the necessary level of sophistication.  Emphasis is given to novel features of the program.  The parallelization strategy used in the RHF, ROHF, UHF, and GVB sections of the program is described, and detailed speedup results are given.  Parallel calcns. can be run on ordinary workstations as well as dedicated parallel machines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pHblAdXU4rVg90H21EOLACvtfcHk0lgDHDslK1J3kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D&md5=4d9e37cd3ec67a92bdfb5b0c0084707c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540141112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540141112%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DBaldridge%26aufirst%3DK.%26aulast%3DBoatz%26aufirst%3DJ.%26aulast%3DElbert%26aufirst%3DS.%2BT.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKoseki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DK.%2BA.%26aulast%3DSu%26aufirst%3DS.%2BJ.%26aulast%3DWindus%26aufirst%3DT.%2BL.%26aulast%3DDupuis%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DJ.%26atitle%3DGeneral%2520atomic%2520and%2520molecular%2520electronic%2520structure%2520system%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26spage%3D1347%26epage%3D1363%26doi%3D10.1002%2Fjcc.540141112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESM','PDB','2ESM'); return false;">PDB: 2ESM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DC2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DC2','PDB','4DC2'); return false;">PDB: 4DC2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i87"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00449">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20461"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00449?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00449</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Summary of biochemical assay and kinase panel data of <b>21</b> and <b>37</b>, mode of action studies with compound <b>37</b> and BMS 345541, FMO calculation, and HPLC traces of key compounds <b>21</b> and <b>37</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_001.pdf">jm0c00449_si_001.pdf (560.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00449/suppl_file/jm0c00449_si_002.csv">jm0c00449_si_002.csv (2.46 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00449&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-13%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00449%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00449" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e11bfcbca1199f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
