
<html lang="en"     class="pb-page"  data-request-id="f6d447ef-2eb3-4281-b838-57a88ff62ad3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-3;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.7b01558"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Binding-Site Compatible Fragment Growing Applied to the Design of β2-Adrenergic Receptor Ligands" /></meta><meta name="dc.Creator" content="Florent  Chevillard" /></meta><meta name="dc.Creator" content="Helena  Rimmer" /></meta><meta name="dc.Creator" content="Cecilia  Betti" /></meta><meta name="dc.Creator" content="Els  Pardon" /></meta><meta name="dc.Creator" content="Steven  Ballet" /></meta><meta name="dc.Creator" content="Niek  van Hilten" /></meta><meta name="dc.Creator" content="Jan  Steyaert" /></meta><meta name="dc.Creator" content="Wibke E.  Diederich" /></meta><meta name="dc.Creator" content="Peter  Kolb" /></meta><meta name="dc.Description" content="Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design. Although computers allow for the facile generation of ..." /></meta><meta name="Description" content="Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design. Although computers allow for the facile generation of ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 24, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01558" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01558" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01558" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01558" /></link>
        
    
    

<title>Binding-Site Compatible Fragment Growing Applied to the Design of β2-Adrenergic Receptor Ligands | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01558" /></meta><meta property="og:title" content="Binding-Site Compatible Fragment Growing Applied to the Design of β2-Adrenergic Receptor Ligands" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0015.jpeg" /></meta><meta property="og:description" content="Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design. Although computers allow for the facile generation of millions of suggestions, synthetic feasibility is however often neglected. In this study we computationally extended, chemically synthesized, and experimentally assayed new ligands for the β2-adrenergic receptor (β2AR) by growing fragment-sized ligands. In order to address the synthetic tractability issue, our in silico workflow aims at derivatized products based on robust organic reactions. The study started from the predicted binding modes of five fragments. We suggested a total of eight diverse extensions that were easily synthesized, and further assays showed that four products had an improved affinity (up to 40-fold) compared to their respective initial fragment. The described workflow, which we call “growing via merging” and for which the key tools are available online, can improve early fragment-based drug discovery projects, making it a useful creative tool for medicinal chemists during structure–activity relationship (SAR) studies." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01558"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01558">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01558&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01558&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01558&amp;href=/doi/10.1021/acs.jmedchem.7b01558" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1118-1129</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01547" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01598" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Binding-Site Compatible Fragment Growing Applied to the Design of β<sub>2</sub>-Adrenergic Receptor Ligands</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Florent++Chevillard">Florent Chevillard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Helena++Rimmer">Helena Rimmer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cecilia++Betti">Cecilia Betti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Els++Pardon">Els Pardon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven++Ballet">Steven Ballet</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niek++van+Hilten">Niek van Hilten</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Steyaert">Jan Steyaert</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wibke+E.++Diederich">Wibke E. Diederich</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5671-6575" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Kolb">Peter Kolb</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-4089-614X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmaceutical Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmaceutical Chemistry and Center for Tumor Biology and Immunology, Philipps-University Marburg, Hans-Meerwein-Straße 3, 35032 Marburg, Germany</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Research Group of Organic Chemistry, Departments of Chemistry and Bio-Engineering Sciences, Vrije Universiteit Brussel, Pleinlaan 2, 1050 Brussels, Belgium</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">VIB-VUB Center for Structural Biology, VIB, 1050 Brussels, Belgium</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Structural Biology Brussels, Vrije Universiteit Brussel, 1050 Brussels, Belgium</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#f9899c8d9c8bd79296959bb98c9790d494988b9b8c8b9ed79d9c"><span class="__cf_email__" data-cfemail="e9998c9d8c9bc78286858ba99c8780c484889b8b9c9b8ec78d8c">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01558&amp;href=/doi/10.1021%2Facs.jmedchem.7b01558" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1118–1129</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 24, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 October 2017</li><li><span class="item_label"><b>Published</b> online</span>24 January 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01558" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01558</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1118%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFlorent%2BChevillard%252C%2BHelena%2BRimmer%252C%2BCecilia%2BBetti%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D3%26contentID%3Dacs.jmedchem.7b01558%26title%3DBinding-Site%2BCompatible%2BFragment%2BGrowing%2BApplied%2Bto%2Bthe%2BDesign%2Bof%2B%25CE%25B22-Adrenergic%2BReceptor%2BLigands%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1129%26publicationDate%3DFebruary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01558"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2666</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">22</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01558" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Binding-Site Compatible Fragment Growing Applied to the Design of β2-Adrenergic Receptor Ligands&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Florent&quot;,&quot;last_name&quot;:&quot;Chevillard&quot;},{&quot;first_name&quot;:&quot;Helena&quot;,&quot;last_name&quot;:&quot;Rimmer&quot;},{&quot;first_name&quot;:&quot;Cecilia&quot;,&quot;last_name&quot;:&quot;Betti&quot;},{&quot;first_name&quot;:&quot;Els&quot;,&quot;last_name&quot;:&quot;Pardon&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;Ballet&quot;},{&quot;first_name&quot;:&quot;Niek&quot;,&quot;last_name&quot;:&quot;van Hilten&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Steyaert&quot;},{&quot;first_name&quot;:&quot;Wibke&quot;,&quot;last_name&quot;:&quot;E. Diederich&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Kolb&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;1118-1129&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01558&quot;},&quot;abstract&quot;:&quot;Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design. Although computers allow for the facile generation of millions of suggestions, synthetic feasibility is however often neglected. In this study we computationally extended, chemically synthesized, and experimentally assayed new ligands for the β2-adrenergic receptor (β2AR) by growing fragment-sized ligands. In order to address the synthetic tractability issue, our in silico workflow aims at derivatized products based on robust organic reactions. The study started from the predicted binding modes of five fragments. We suggested a total of eight diverse extensions that were easily synthesized, and further assays showed that four products had an improved affinity (up to 40-fold) compared to their respective initial fragment. The described workflow, which we call “growing via merging” and for which the key tools are available online, can improve early fragment-based drug di&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01558&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01558" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01558&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01558" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01558&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01558" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01558&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01558&amp;href=/doi/10.1021/acs.jmedchem.7b01558" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01558" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01558" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01558%26sid%3Dliteratum%253Aachs%26pmid%3D29364664%26genre%3Darticle%26aulast%3DChevillard%26date%3D2018%26atitle%3DBinding-Site%2BCompatible%2BFragment%2BGrowing%2BApplied%2Bto%2Bthe%2BDesign%2Bof%2B%25CE%25B22-Adrenergic%2BReceptor%2BLigands%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D3%26spage%3D1118%26epage%3D1129%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/jmcmar.2018.61.issue-3/20180208/jmcmar.2018.61.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Fragment-based drug discovery is intimately linked to fragment extension approaches that can be accelerated using software for de novo design. Although computers allow for the facile generation of millions of suggestions, synthetic feasibility is however often neglected. In this study we computationally extended, chemically synthesized, and experimentally assayed new ligands for the β<sub>2</sub>-adrenergic receptor (β<sub>2</sub>AR) by growing fragment-sized ligands. In order to address the synthetic tractability issue, our in silico workflow aims at derivatized products based on robust organic reactions. The study started from the predicted binding modes of five fragments. We suggested a total of eight diverse extensions that were easily synthesized, and further assays showed that four products had an improved affinity (up to 40-fold) compared to their respective initial fragment. The described workflow, which we call “growing via merging” and for which the key tools are available online, can improve early fragment-based drug discovery projects, making it a useful creative tool for medicinal chemists during structure–activity relationship (SAR) studies.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the past few years, fragment-based drug discovery (FBDD) has continuously gained in popularity and has become a dominant approach in order to explore novel chemical entities.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> The year 2011 witnessed an important success for this field, with the first drug (vemurafenib) originating from a hit found in a fragment-based screening, reaching FDA approval.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a></div><div class="NLM_p">Fragments are commonly regarded as attractive alternatives compared to standard compounds contained in the libraries used for high-throughput screening campaigns. They are small molecules (MW < 300 Da) and usually offer better solubility, making them favorable in terms of ADMET properties.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Moreover, it is easier to cover chemical space with fragment-sized molecules: screening a set of 1000 fragments is claimed to be equivalent to probing the chemical space of 1 000 000 drug-like molecules.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Because of their simplicity and few polar interaction points, they also have a higher chance to bind to a given protein with optimal interactions.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> However, because they are smaller, fragments usually bind with low affinity compared to drug-like molecules, resulting in the need for screening at high concentrations. Yet those that bind usually do so with high ligand efficiency (LE), defined as the ratio of Gibbs free energy of binding (Δ<i>G</i><sub>bind</sub>) and the number of heavy atoms.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Fragments are thus often considered as advantageous starting points for exploratory synthesis campaigns.</div><div class="NLM_p">Several biophysical techniques can deal with the low affinity and therefore high concentration of fragments and allow detection of binding to a protein. Among them, surface plasmon resonance (SPR), isothermal titration calorimetry (ITC), thermal shift assays, and mass spectrometry (MS) have been successfully implemented in fragment-based drug discovery pipelines.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> However, since the interactions remain weak, structural methods such as X-ray crystallography<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and NMR<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> might be preferable, as they lead to fewer false positives and negatives during screening campaigns.<a onclick="showRef(event, 'ref7 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref14">(7, 14)</a> These two structural techniques can in principle determine the precise binding mode(s) of a fragment within a protein target, thereby also laying the foundation for subsequent structure-based design approaches.</div><div class="NLM_p">When no experimental data are available, the binding mode of a fragment within an active site can be predicted using in silico methods, most notably docking. It has to be noted that because of the propensity of fragments to bind in more than one mode, fragment binding mode prediction remains challenging. Moreover, most of the docking programs were parametrized and optimized for drug-like molecules, which might lead to a relative imbalance of the individual terms in a scoring function.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Consequently, several programs have been developed more specifically for fragment docking, including MCSS,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> SuperStar,<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> and SEED.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a></div><div class="NLM_p">Once the binding mode of a fragment has been determined, either experimentally or computationally, it can be used as starting point for an FBDD project, aiming at expanding the fragment into a bigger molecule while concomitantly increasing its affinity.</div><div class="NLM_p">Three main approaches are conceivable in order to extend fragments: growing, linking, and merging (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). <i>Growing</i> is based on one core fragment and aims at extending it within the binding site, looking for additional interactions that could improve affinity or selectivity. <i>Growing</i> is the most frequently used approach because it is considered more straightforward and successful than the other two techniques.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> The <i>linking</i> strategy describes the process of joining two noncompetitive fragments (i.e., fragments that bind in two different regions of the binding site) by means of a linker. <i>Merging</i> concatenates two fragments that contain a common portion known to bind at the same position in the binding site of a given target. Our approach takes advantage of both growing and merging procedures.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fragment-based extension approaches in drug discovery: growing, linking, and merging.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Growing approaches have extensively been implemented in computer software, including LUDI,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> SPROUT,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> CONCERTS,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> ReCore,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Caveat,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> BREED,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> GANDI,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> BROOD,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and IADE.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Nonetheless, one of the main challenges of these softwares is to ensure the synthetic tractability of the designed molecules. Several rules can be implemented in order to assess whether or not a compound is synthetically feasible, among the most frequently used are BRICS,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> RECAP,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> or an estimation of the synthetic accessibility score.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> These approaches are based on knowledge of chemical reactions in order to analyze the molecules.</div><div class="NLM_p">Proceeding the other way around is also possible: concatenating two available building blocks using a compatible chemical reaction. Hartenfeller et al. developed a library of 58 robust organic reactions,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34, 35)</a> based on chemical reactions often-used in the pharmaceutical domain.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> These reactions are used in the PINGUI (Python in silico de novo growing utilities) toolbox and define the first step of the “growing via merging” (GVM) workflow presented in this study: creating a focused library consisting only of those building blocks that are compatible with the core fragment in terms of further synthesis.</div><div class="NLM_p">The second constraint of our approach for fragment growing is the assumption that each individual building block should engage in favorable interactions with the protein by itself. In an alternative approach, where extensions are attached to core fragments and the resulting compounds are docked, slightly unfavorable extensions might be compensated by the interaction of the core fragment and thus masked. Within the GVM workflow, we generate <i>maps</i> composed of a large number of favorable poses for each extension (i.e., building block). On the basis of the crystallographic or computed binding mode of the core fragment, building blocks are then chosen such that reactive groups are within geometrically feasible distances and angles, therefore allowing bond formation between the core and an already optimally placed building block to occur. Basically, this step is equivalent to <i>merging</i> two fragments at their reactive centers, which will be described in more detail below. In this way, fragment <i>chemical</i> space and <i>positional</i> space are pruned at an early stage, and only optimally positioned fragments with favorable interactions are retained. After computationally generating the products from the building blocks, we ensure that the initial assumption (i.e., that each component of a molecule is, when regarded individually, located in an optimal spot) is still valid by re-docking the entire product. A large deviation from the original (predicted) binding mode would violate this assumption and lead to the removal of the product from our GVM workflow. Hence, growing is guided by both the synthetic tractability and the protein structure. <a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">Figure S1 in the Supporting Information</a> shows a comprehensive scheme of our GVM workflow.</div><div class="NLM_p">In this manuscript, we describe the design of new ligands of the β<sub>2</sub>AR (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">Table S1</a>) by employing the PINGUI toolbox and the semiautomated GVM workflow. Five different fragments were grown. They have in common that they are predicted to bind in the orthosteric site with a reaction-compatible reactive feature (herein an amine) close to the secondary binding pocket (SBP; <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). Therefore, each GVM step was directed toward the SBP. Experimental results demonstrated that all eight suggested molecules could be successfully synthesized and half of them had a more favorable affinity than the initial fragment, with the best compound offering an almost 40-fold improvement.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Sliced surface side-view of the binding site of the receptor in its basal conformation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a>) in complex with carazolol (magenta carbons). The orthosteric site is colored in green and the SBP in blue. (b) BI167107 (green carbons) in complex with the active conformation of the receptor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE">4LDE</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42658" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> β<sub>2</sub>AR Binding Site</h3><div class="NLM_p">The β<sub>2</sub>AR binding pocket sits buried in the extracellular part of the receptor with only a small fraction accessible to solvent. For the sake of this study, we divided the binding site into two smaller cavities: the orthosteric site and the SBP (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). While most of the orthosteric site is hydrophobic,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> important polar residues shape its contour: Ser203<sup>5.42</sup>, Ser204<sup>5.43</sup>, and Ser207<sup>5.46</sup> in transmembrane helix 5 (TM5) define the bottom of the orthosteric site (superscript indices refer to Ballesteros–Weinstein numbering<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>) and are involved in receptor activation.<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39-41)</a> Residues Asp113<sup>3.32</sup> in TM3 and Asn312<sup>7.39</sup> and Tyr316<sup>7.43</sup> in TM7 offer strong electrostatic points that interact with the amine moieties of most known ligands. The SBP is defined by a small hydrophobic cavity (Trp109<sup>3.28</sup>, Phe193<sup>5.32</sup>, and Ile309<sup>7.36</sup>) and a number of more solvent exposed polar residues (Asp192<sup>ECL2</sup> and His93<sup>2.64</sup>) that agonists are known to interact with.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43">(39-43)</a></div><div class="NLM_p last">The difference in the orthosteric site between the basal and active receptor conformation is manifest mostly in His296<sup>6.58</sup> and the size of the orthosteric pocket.<a onclick="showRef(event, 'ref39 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref42">(39, 42)</a> In the active-like receptor conformation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE">4LDE</a>), His296<sup>6.58</sup> is part of a large polar network involving the catechol moiety of the ligand, a water molecule, Tyr308<sup>7.35</sup>, Ser204<sup>5.43</sup>, and Asn293<sup>6.55</sup>.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> This polar network is located close to the orthosteric site, making it smaller in the active conformation (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). In the basal conformation of the receptor, the aforementioned polar network is not so extended, resulting in an inward shift of His296<sup>6.58</sup>.<a onclick="showRef(event, 'ref39 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref42">(39, 42)</a> This shift results in a larger orthosteric site.</div></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Core Fragments</h3><div class="NLM_p">Five fragment-sized ligands with experimentally determined affinity were used as core fragments (i.e., starting points) in this growing study. Compounds <b>1</b>–<b>4</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) with favorable ligand efficiency (LE = 0.49, 0.39, 0.35, and 0.31, respectively, against the β<sub>2</sub>AR-Nb80) had emerged from an experimental screening with nanobody-stabilized receptors.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Compound <b>5</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) had initially been discovered in a docking screen against the β<sub>2</sub>AR in a basal conformation with the same docking setup as used by Kolb et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Compound <b>5</b> also showed favorable LE (0.34) against the β<sub>2</sub>AR-Nb80. Compounds <b>1</b>–<b>5</b> had a known preference for the receptor conformation stabilized in an active state by Nb80 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). We will refer to such molecules as “agonist candidate” (AC) to clearly distinguish them from assay-confirmed (partial) agonists.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Affinity of the Compounds for β<sub>2</sub>AR-Nb80 and β<sub>2</sub>AR-Nb69<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (Nb80)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (Nb69)</th><th class="colsep0 rowsep0" align="center">shift (Nb69/80)</th><th class="colsep0 rowsep0" align="center">profile</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">33.8 (0.8)</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>61</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">43.2 (1.3)</td><td class="colsep0 rowsep0" align="left">373 (13)</td><td class="colsep0 rowsep0" align="left">8.6</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">33.5 (1.6)</td><td class="colsep0 rowsep0" align="left">365 (14)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">78.1 (1.2)</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>91</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">20.4 (0.6)</td><td class="colsep0 rowsep0" align="left">36.2 (0.8)</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">52.3 (0.7)</td><td class="colsep0 rowsep0" align="left">393 (17)</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">95.4 (4.1)</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>51</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">4.39 (0.7)</td><td class="colsep0 rowsep0" align="left">268 (11)</td><td class="colsep0 rowsep0" align="left">61</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">7.89 (1.1)</td><td class="colsep0 rowsep0" align="left">74.0 (1.5)</td><td class="colsep0 rowsep0" align="left">9.4</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">0.52 (0.03)</td><td class="colsep0 rowsep0" align="left">3.9 (0.4)</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">10.4 (1.2)</td><td class="colsep0 rowsep0" align="left">25.0 (1.2)</td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">AC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">80.4 (3.0)</td><td class="colsep0 rowsep0" align="left">83.8 (3.2)</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">AntC</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">All reported <i>K</i><sub>i</sub> are in μM. Values in parentheses are the standard error of the mean.</p></div></div></div></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Prediction of the Binding Mode of the Core Fragments</h3><div class="NLM_p">We discuss compounds <b>1</b>–<b>4</b> and compound <b>5</b> separately, as they were docked to receptor structures in different activation states. Compounds <b>1</b>–<b>4</b> and compound <b>5</b> were placed in the binding site of the receptor active (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DLE">4DLE</a>) and basal (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a>) conformations, respectively, using DOCK 3.6.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The predicted binding modes were kept for further growing and visually examined for plausibility.</div><div id="sec2-3-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Compounds <b>1</b>–<b>4</b></h4><div class="NLM_p">The growing procedure started from docked poses of compounds <b>1</b>–<b>4</b>. We note that no direct interactions with the polar serines at the bottom of the orthosteric site were formed in the poses predicted in an active conformation of the β<sub>2</sub>AR for these compounds. Three of the core fragments contained an acceptor group, but they were too far away (>4 Å) and in unproductive geometries to form direct polar interactions with the serines in helix 5: the benzothiophene of compound <b>2</b>, the thiazole of compound <b>3</b>, and the pyrimidine moiety of compound <b>4</b>. All the charged amines of the core fragments were found to interact with Asp113<sup>3.32</sup> and Asn312<sup>7.39</sup>. The aromatic moieties of the core fragments made hydrophobic interactions with Phe289<sup>6.51</sup>. As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, all of these fragments were predicted to bind to the orthosteric site.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a–d) Prediction of the binding mode of the four AC core fragments within the β<sub>2</sub>AR active conformation. (e) Prediction of the binding mode for compound <b>5</b> after docking to the β<sub>2</sub>AR structure in a basal conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Five fragments with favorable LE determined in an experimental screen were chosen for further growing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-3-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Compound <b>5</b></h4><div class="NLM_p last">Because no reliable binding mode was found in the active conformation of the β<sub>2</sub>AR by our docking methods, the second growing procedure started from the docked pose of compound <b>5</b> in the basal conformation. The docking solution is consistent with a pose expected for a binder of the β<sub>2</sub>AR: the charged amine moiety interacted with Asp113<sup>3.32</sup> and Asn312<sup>7.39</sup>, the indole moiety made hydrophobic interactions with Phe289<sup>6.51</sup>, the methyl on the indole interacted with both Val114<sup>3.33</sup> and Val117<sup>3.36</sup>, and the ether moiety engaged Asn293<sup>6.55</sup> in a polar hydrogen bond. As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>e, we predicted compound <b>5</b> to bind to the orthosteric site with room to grow toward the SBP.</div></div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Chemical Derivatization by Reductive Amination</h3><div class="NLM_p last">All the fragments of this study share a common chemical feature: an amine that will be positively charged at physiological pH and thus likely interacts with Asp113<sup>3.32</sup>, according to the standard pharmacophore for β<sub>2</sub>AR ligands. This reactive group can also be harnessed to grow the fragment toward the SBP. The most straightforward reaction that can be used with such a functional group is reductive amination because the resulting amine can still be charged, and thus the charge–charge interaction with Asp113<sup>3.32</sup> will not be lost. This reaction involves a primary or secondary amine reacting with an aldehyde or a ketone in order to create a secondary or tertiary amine. The SMARTS code for the reaction was used as encoded by Hartenfeller et al.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Surrogate Creation</h3><div class="NLM_p">During the first docking stage with SEED,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> all synthetically compatible building blocks (i.e., all fragments amenable to being used as extension of the core fragment) were docked individually in order to find the ones that are predicted to fit the SBP and form favorable interactions. However, in their unreacted forms, the building blocks will usually possess atoms or atom groups that will not be present in the final product. Thus, docking a building block in its unreacted form would not reflect its interaction options once attached to the core fragment. In order to mimic the behavior of the final product and minimize the occurrence of unlikely interactions, we converted all building blocks to “surrogates”. We define these surrogates as the final product without the unreactive part of the core fragment. In the specific case presented here, the reaction to be applied to all cores was reductive amination. Surrogates are thus the reactants minus the carbonyl groups but including the amine (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). In this way, no additional interactions that could influence the placement of the reactant are available. 18 785 compatible building blocks (i.e., aldehydes or ketones) were extracted from the <i>fragments now</i> subset of the ZINC database<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and converted to a new set of 26 892 surrogates (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The increase in number resulted from the exhaustive enumeration of all possible stereoisomers.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Surrogate Derived from an Aldehyde Defined by the Replacement of the Carbonyl by an Amine Group<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>R<sup>1</sup> = alkyl.</p></p></figure><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Surrogate Derived from a Ketone Defined by the Replacement of the Carbonyl by an Amine Group<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>This transformation introduces a chiral center; therefore two different surrogates were generated (R and S). R<sup>1</sup> and R<sup>2</sup> = alkyl.</p></p></figure></div><div id="sec2-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Surrogate Docking, Filtering, and Ranking</h3><div class="NLM_p">Of the 26 892 surrogates, 15 702 were successfully docked with SEED to the SBP of the β<sub>2</sub>AR in an active conformation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE">4LDE</a>) yielding a total of 814 369 docking poses. The same docking procedure was applied to the basal conformation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a>) and 16 297 surrogates were successfully docked, yielding 1 011 929 poses. These numbers intuitively make sense, since in the basal conformation, the entrance of the orthosteric site (included in the SBP) is wider than in the active conformation, and therefore more molecules can be docked in clash-free conformations. Only those surrogate poses that did not overlap with the core fragment were kept for further processing, as illustrated in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a. Then, a filter was applied based on the distance between the amine of the surrogate and the predicted position of the amine of the core fragment (i.e., the docking pose). This cutoff was set to 2 Å. Every pose that passed both criteria was ranked according to the score calculated by SEED. Only the top 500 surrogates (without duplicates) were kept for further processing (i.e., generation of the products).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Illustration of the filtering process. Note that the number of poses shown is much smaller than the one actually used. (a) Typical poses (i.e., a map) after docking with SEED. (b) The surrogate poses (magenta carbons) were filtered so they do not overlap with the core scaffold (green carbons), and the charged amine is close to the one of the core fragment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Derivative Product Creation and Docking</h3><div class="NLM_p">For each core fragment, the top 500 surrogates were attached according to the reaction scheme for reductive amination which yielded equally many (500) derivative products. Then, the top 500 derivative products were docked into the binding site of the aforementioned β<sub>2</sub>AR conformation and visually inspected in order to narrow down the number of candidates for possible further synthesis. The selection of the best derivative products was based on several criteria: docking score, shape complementarity with the receptor, hydrogen bonds to the receptor, overlap of the products with the respective core fragment and surrogate (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>), and chemical diversity. Eight ligands not violating these criteria were selected and synthesized (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Example of a product keeping its orientation relative to the poses of the constitutive building blocks. Both the core fragment (compound <b>5</b> in green carbons) and the surrogate (magenta carbons) are predicted to overlap with the generated product (compound <b>11</b> in yellow carbons) after docking to the β<sub>2</sub>AR structure in a basal conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Synthesized products. Compounds <b>6</b>–<b>12</b> appeared as ACs in the experimental assay. Compound <b>13</b> appeared as AntC in the experimental assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Radioligand Displacement Assay</h3><div class="NLM_p">In lieu of performing cellular assays to determine the efficacy of compounds, we decided to use a “reverse pharmacology” assay as described in ref <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">44</a>. Since this assay uses radioligand displacement, it allows the precise determination of affinity, which is the quantity estimated in our docking calculations and should therefore be used to assess the validity of our predictions. At the same time, because of the way the assay is set up, we can also make a statement about the likely efficacy of each compound. As shown in the original manuscript,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> there is a high degree of correlation between this prediction and the actual cellular efficacy.</div><div class="NLM_p">The summary of the radioligand displacement assay for the eight products is illustrated in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. The IC<sub>50</sub> curves for compound <b>11</b>, which showed improved affinity, are depicted in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>, and the curves for remaining compounds are listed in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">Supporting Information (Figure S5)</a>.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Example of IC<sub>50</sub> curves recorded in the radioligand displacement assay for compound <b>11</b>. The green curve was obtained in the assay using the β<sub>2</sub>AR-Nb80, while the red curve corresponds to the β<sub>2</sub>AR-Nb69.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Analysis of the Derivative Products’ Binding Modes and Activities</h3><div class="NLM_p">In the following discussion, only the predicted interactions of the synthesized products in the SBP will be detailed, as the interactions in the orthosteric pocket remained constant in all cases, as required by our initial hypothesis. Compounds which are not detailed here are discussed in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">Supporting Information (Figure S3)</a>.</div><div id="sec2-9-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Compound <b>8</b></h4><div class="NLM_p">Compound <b>8</b> (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>) features a chiral center and was selected for synthesis because it contains a toluene moiety that can make favorable hydrophobic interactions with the apolar part of the SBP. The <i>R</i>-isomer was predicted to fit into the SBP, forming hydrophobic contacts with Trp109<sup>3.28</sup> and Ile309<sup>7.36</sup>. On the other hand, the <i>S</i>-isomer was docked such that we observed steric clashes in the SBP, thereby forcing the amine out of range from the key Asp113<sup>3.32</sup>. Thus, the <i>S</i>-isomer is left with two unmatched hydrogen bond donors, thus making it harder to compensate for the entropic penalty incurred upon removal of the water shell. The docking score also predicts the <i>R</i>-isomer to be more favorable than the <i>S</i>-isomer, which is in line with the aforementioned reasoning. Compound <b>8</b> was tested as a racemic mixture and displayed an improved affinity (<i>K</i><sub>i</sub> = 4.39 μM, LE = 0.27) compared to its initial core fragment compound <b>4</b> (<i>K</i><sub>i</sub> = 78.1 μM, LE = 0.31) against β<sub>2</sub>AR-Nb80. The selectivity shift between the two conformations was still high (61; cf. <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), thus classifying compound <b>8</b> as an AC molecule. We note, however, that the affinity of one of the enantiomers, presumably the <i>R</i>-isomer, could actually be higher than the one measured.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Fold Affinity Difference for the Derivatized Compounds <b>6</b>–<b>13</b> in Comparison to Their Respective Core Fragment</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">fold (Nb80)</th><th class="colsep0 rowsep0" align="center" char=".">fold (Nb69)</th><th class="colsep0 rowsep0" align="center" char=".">shift fold</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">17.8</td><td class="colsep0 rowsep0" align="char" char=".">26.6</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">28.1</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">0.04</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">38.9</td><td class="colsep0 rowsep0" align="char" char=".">9.2</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr></tbody></table></div></div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Predicted binding mode of the synthesized products using docking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2-9-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Compounds <b>11</b>–<b>13</b></h4><div class="NLM_p">With this series based on compound <b>5</b>, we wished to explore different substitutions on the benzene ring located in the SBP and the importance of the flexible chain between the two rigid scaffolds. Two of the compounds (compounds <b>11</b> and <b>12</b>) contain a flexible chain with six rotatable bonds, while compound <b>13</b> has four rotatable bonds.</div><div class="NLM_p">Compound <b>11</b> showed the highest improvement in our elaborated molecules. It bound with a <i>K</i><sub>i</sub> of 520 nM against β<sub>2</sub>AR-Nb80, corresponding to a nearly 40-fold improvement of the initial compound <b>5</b> while maintaining a decent LE (from 0.34 to 0.30). Interestingly, it showed a <i>K</i><sub>i</sub> of 4 μM against β<sub>2</sub>AR-Nb69 (9-fold improvement only). Thus, the extended compound showed a stronger AC profile with shift fold of 0.2 (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). One could speculate that Trp109<sup>3.28</sup> and Ile309<sup>7.36</sup>, which are predicted to interact with the chlorobenzene moiety, might be involved in the activation of the receptor.</div><div class="NLM_p">Compound <b>12</b> was tested and also had a more favorable affinity (<i>K</i><sub>i</sub> = 10.4 μM, LE = 0.20) against β<sub>2</sub>AR-Nb80 compared to its initial core fragment (<i>K</i><sub>i</sub> = 20.4 μM, LE = 0.34), but the LE was not maintained. It contains a trifluoromethyl moiety, rendering it bulkier than compound <b>11</b>, which only has a chlorine substituent. The docking pose suggests that compound <b>12</b> could furthermore entertain hydrophobic contacts with Trp109<sup>3.28</sup> and Ile309<sup>7.36</sup>.</div><div class="NLM_p">Upon testing of compound <b>13</b>, we found that it had a decreased affinity and LE (<i>K</i><sub>i</sub> = 80.4 μM, LE = 0.15) against β<sub>2</sub>AR-Nb80 compared to its initial core fragment (<i>K</i><sub>i</sub> = 20.4 μM, LE = 0.34). This small decrease in affinity (4-fold) can probably be explained by the size of the extension: it introduces more steric constraints in the SBP, thereby disrupting the hydrophobic interactions with Trp109<sup>3.28</sup>. Compound <b>13</b> has two rotatable bonds less than its two aforementioned analogs, and this could suggest that compound <b>13</b> has less flexibility for adapting to the SBP and therefore its interactions are less optimal.</div><div class="NLM_p last">Taken together, we elaborated each fragment with at least one extension. Moreover, we created a small set of three and two derivatives, respectively, for core fragments <b>3</b> and <b>5</b>. Four of the eight derivatives showed higher affinity, a 50% improvement rate.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Starting from a diverse set of fragments exhibiting affinity for the β<sub>2</sub>AR, we constructed extended derivatives guided by the protein structure. For 50% of the ligands, we were able to improve the initial affinities. The in silico workflow “growing via merging” we developed for this purpose tackles three challenges simultaneously. First, the fragments need to be grown toward a region offering enough space for the addition of another building block. Second, the suggested molecules should have a high likelihood of synthesizability. Third, the resulting molecules should have a high probability of binding to the target protein. The GVM workflow does this by focusing on a small number of candidates and pruning search space at an early stage.</div><div class="NLM_p">All the initial fragments in this study had two points in common: they were predicted to bind to the orthosteric site of the β<sub>2</sub>AR and they all contained a protonatable amine presumably interacting with Asp113<sup>3.32</sup>. Thus, an extension of the fragments in the direction of the SBP was feasible. In the interest of focusing on molecules that are likely to be synthesizable, we selected reductive amination from the library of 58 robust organic reactions.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Other approaches exist that can generate molecules with a high likelihood of synthesizability,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and it will depend on the specific protein and reaction which approach might be considered more appropriate by researchers who apply it. Importantly for usability, however, our tool is freely accessible online and does not require any software to be installed.</div><div class="NLM_p">The choice of reductive amination in this study was guided by the fact that the amine is the reaction center and is preserved in the reaction. Thus, all the generated products can extend past Asp113<sup>3.32</sup> to the SBP while maintaining the salt bridge with this residue. All eight molecules that we suggested were easily synthesized, highlighting that reductive amination can indeed be regarded as a robust reaction. Of note, despite using this standard reaction, all of our final compounds are well below a Tanimoto coefficient of 0.4 when compared to known ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">Table S1</a>). Such compounds are commonly regarded as chemically different indeed. Furthermore, our compounds are distinct from the pharmacophore of β<sub>2</sub>AR agonists (phenylethylamine with a hydroxy group in β position) as they do not have the β-hydroxy moiety. This emphasizes the power of FBDD: growing from diverse fragments yields chemically novel compounds. When compared to known adrenoreceptor ligands,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> compound <b>11</b> and labetolol share a similar extension past the amine (i.e., propylbenzene). This could suggest that this compound could prefer binding to the β<sub>2</sub>AR over the β<sub>3</sub>AR. Most importantly, however, this indicates that our workflow could also be applied to design selective ligands among receptor subtypes.</div><div class="NLM_p">In the interest of increasing the chance of our suggestions being active, our workflow prioritized extensions that entertain favorable interactions with the protein by themselves. We assume that the two-step docking of first the surrogate and then the resulting product leads to an increase in the robustness of our predictions. This has also recently been suggested in a different article focusing on the design of ligands for STAT3.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> In our approach, whenever a promising product deviated too much from its initially predicted position, it was not pursued further. We are aware that there are retrosynthetic experimental analyses, including X-ray crystallography of the resulting fragments, that show that such binding mode faithfulness is not always the case.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> However, one can afford to be more elitist in this in silico approach, as combinatorics works in our favor by generating such a large number of solutions that it needs to be pruned. Moreover, docking is notorious for producing many false positives,<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> and applying stringent selection criteria often improves true positive rates. In our hands, these considerations allowed us to significantly improve 50% of the original fragment binders. It is difficult to assess whether this is better or worse than what a human medicinal chemist might have achieved. However, when one takes a look at current literature examples, an improvement rate of 50% seems higher than what is usually reported.</div><div class="NLM_p">The presented workflow is not strictly automatized, because it relies on spatial reasoning abilities during the evaluation of the poses. Yet as most of the growing steps rely on computational approaches, we decided to make them freely available to the public. Thus, we created PINGUI, a toolbox that we hope will help medicinal chemists as an idea generator in fragment-based ligand discovery project. PINGUI is available online at <a href="http://www.kolblab.org/scubidoo/pingui" class="extLink">www.kolblab.org/scubidoo/pingui</a> and is implemented alongside the SCUBIDOO database.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> PINGUI features many options that were instrumental in this project, namely, displaying the compatible reaction for any given fragment, the creation of customized derivative libraries, a retrosynthesis module, or the option of filtering a library based on reactive chemical features. The addition of such functions should provide SCUBIDOO users with a full framework for virtual product generation. A description of the interface of PINGUI is available in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">Support Information</a>.</div><div class="NLM_p last">As with SCUBIDOO, we do not see the utility of this approach so much in the probability that it will yield a highly potent compound during an initial round of optimization but rather in providing medicinal chemists with creative suggestions. Hence, we believe that PINGUI is best employed at the very early stages, when SAR needs to be generated quickly and with limited synthetic efforts. In contrast to SCUBIDOO, which is limited by the number of fragments it was built on, PINGUI can be regarded as a flexible and basically unlimited alternative when navigating into the immensity that is chemical space. Indeed, PINGUI generates a tailored chemical space defined by the user (the fragment to grow but also the library to react with) while respecting the constraints imposed by the binding site of the target protein. This tremendously reduces the number of products to virtually screen to merely hundreds or thousands. We also think that it will enable researchers who are not dyed-in-the-wool organic chemists (such as ourselves) to develop and synthesize comparatively simple substances, thereby speeding up research and potentiating the availability of pharmacologically active chemical matter.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06334" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06334" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4-1-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Computational Details. Receptor X-ray Structures</h3><div class="NLM_p last">Docking calculations were performed with the basal conformation of the β<sub>2</sub>AR in complex with carazolol (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a>)<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54, 55)</a> and an active conformation in complex with the ligand BI167107 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE">4LDE</a>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> All ligands, solvent, lipid molecules as well as the T4-lysozyme insertion or the stabilizing nanobody Nb6B9 were removed. The hydrogens were placed and minimized using the HBUILD module in CHARMM.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> CHARMm22<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> atom types and MPEOE<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58, 59)</a> partial charges were assigned using the program Witnotp [Novartis Pharma AG, unpublished].</div></div><div id="sec4-1-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Surrogate Creation</h3><div class="NLM_p last">The <i>fragments now</i> data set was downloaded from the ZINC database<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and contained 504 074 building blocks. Every compatible building block was then converted into the corresponding surrogate by means of a Python script written using the RDKit library.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Reaction centers (the nitrogen of the introduced amine group) of the surrogates were flagged in order to facilitate filtering later on. In the case of ketones, where condensation with an amine results in the introduction of a chiral center, both stereoisomers for the surrogate were generated using flipper.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> In the context of docking, CHARMm22 atom types<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and MPEOE partial charges<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58, 59)</a> were assigned using the program Witnotp (Novartis Pharma AG, unpublished). Up to 100 conformers were generated for each surrogate using OMEGA.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div></div><div id="sec4-1-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Docking the Surrogates with SEED</h3><div class="NLM_p last">SEED<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> has been developed to exhaustively place small rigid fragments in the binding sites of proteins. The calculations also take into account the penalty that is incurred upon binding due to the removal of the water shells of the ligand and the protein, respectively. For each fragment, 10<sup>6</sup>–10<sup>7</sup> individual poses are generated and subsequently clustered based on geometric and energetic criteria. This yields on the order of hundreds of poses, clustered into groups of a maximum of five. These maps were used to select poses with favorable energy and geometrically appropriate orientation with respect to bond formation.</div></div><div id="sec4-1-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Docking the Derivative Products with DOCK</h3><div class="NLM_p last">For each fragment to be grown, the corresponding products were generated from the top 500 surrogates with favorable scores and appropriate geometry using PINGUI. Product conformations were generated using the <i>dbgen</i> pipeline described for ZINC.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Since DOCK does not evaluate intramolecular energy terms, some docked poses might show slightly unfavorable geometries. In order to ameliorate the clashes of such poses while keeping the overall binding mode found by DOCK, the poses were minimized using the force field including a Poisson–Boltzmann model for solvation as implemented in SZYBKI.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> All products were then reranked according to the score calculated by SZYBKI and kept for subsequent visual inspection.</div></div><div id="sec4-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Experimental Synthesis</h3><div class="NLM_p last">Reductive amination of the respective aldehyde or ketone derived from the surrogate library with fragments <b>1</b>–<b>5</b> rendered the proposed modulators. All commercially available reagents and solvents were used without further purification.</div></div><div id="sec4-2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Information</h3><div class="NLM_p last">Mass spectrometry (MS) was done on a Micromass Q-Tof Micro spectrometer with electrospray ionization (ESI). Data collection and spectrum analysis were done with Masslynx software. Analytical RP-HPLC was performed using an Agilent 1100 series system (Waldbronn, Germany) with a SUPELCO Discovery BIO Wide Pore (Bellefonte, PA, USA) RP C-18 column (15 cm × 2.1 mm, 3 μm) and UV detection at 215 nm. The mobile phase (water/acetonitrile) contained 0.1% TFA. The gradient consisted of a 20 min run from 3% to 97% acetonitrile at a flow rate of 0.3 mL/min. Preparative RP-HPLC purification was done on a Gilson (Middleton, WI) HPLC system with Gilson 322 pumps, controlled by the software package Unipoint, and a reversed phase C18 column (DiscoveryBIO SUPELCO Wide Pore C18 column, 25 cm × 2.21 cm, 5 μm) with the same conditions as the analytical RP-HPLC but with a flow rate of 20 mL/min. After purification, the purity of all compounds was evaluated as being more than 95% by analytical RP-HPLC. All fractions were lyophilized using a Flexy-Dry lyophilizer (FTS Systems, Warminster, PA). <sup>1</sup>H spectra were recorded at 250.53 MHz on a Bruker Avance 250 spectrometer (Bruker Corp, Billerica, MA). Trimethylsilane (TMS) or residual solvent signals are used as internal standard. The solvent used is mentioned in all cases, and the abbreviations used are as follows: s (singlet), d (doublet), dd (double doublet), t (triplet), and m (multiplet). Thin layer chromatography was performed on precoated plates (silica gel 60 F<sub>254</sub>, Merck). Flash column chromatography was performed on prepacked columns (PF-30SIHP-JP/12G; PF-30SIHP-JP/4G; Interchim) using a Büchi separation system. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on a Jeol ECA-500 and a Bruker AV II-300 spectrometer. Unless noted otherwise, spectra were recorded at 20 °C. Chemical shifts (δ) are given in ppm (parts per million). All NMR spectra were referenced to the residual solvent signal (CDCl<sub>3</sub>: 7.26 ppm [<sup>1</sup>H] and 77.16 ppm [<sup>13</sup>C]). Coupling constants are reported in hertz (Hz). Mass spectra were recorded on a double-focusing sector field spectrometer type AutoSpec (Micromass). Elemental combustion analyses were recorded on an Elementar vario MICRO instrument. Quantitative NMR (qNMR) measurements for compound <b>12</b> were recorded on a Jeol ECA-500 spectrometer<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> using maleic acid, purchased from Sigma-Aldrich, as internal reference standard.</div></div><div id="sec4-2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Derivative Products of Compound <b>1</b> (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>)</h3><div class="NLM_p">The appropriate primary amine R<sup>2</sup>NH<sub>2</sub> (0.269 mmol, 1.2 equiv) and triethylamine (1.344 mmol, 6 equiv) were dissolved in anhydrous Et<sub>2</sub>O (2.5 mL) in a two-neck round-bottom flask. The solution was cooled to 0 °C by means of an ice bath and flushed with argon. The 4-chloroacetophenone (0.224 mmol, 1 equiv) was added followed by dropwise addition of TiCl<sub>4</sub> (0.224, 1 equiv). The solution turned to a yellow suspension. The reaction mixture was warmed up to rt and stirred for 3.5 h. NaCNBH<sub>3</sub> (1.344 mmol, 6 equiv) was dissolved in MeOH (100 μL) and added dropwise to the reaction mixture. After overnight stirring the reaction was quenched with water. AcOEt was added, and the layers were separated. The aqueous phase was extracted 3 times with AcOEt, and the combined organic layers were dried over MgSO<sub>4</sub>, filtered off, and evaporated in vacuo. The crude mixtures were purified by semipreparative reverse-phase (RP) HPLC, and the desired secondary amines were isolated after lyophilization, as TFA salt. The structures of the desired products were confirmed by <sup>1</sup>H NMR and mass spectrometry (MS) analysis.</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Derivative products of compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4-2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Derivative Products of Compounds <b>2</b>, <b>3</b>, <b>4</b> (<a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>)</h3><div class="NLM_p">The aldehyde (0.155 mmol, 1 equiv) was dissolved in 500 μL of anhydrous DCM. MgSO<sub>4</sub> (150 mg) was suspended in the solution, and subsequently the appropriate amine (0.186–0.310, 1.2–2 equiv) was added. The reaction mixture was stirred at rt until the imine formation was completed (2 h overnight reaction). The conversion of the aldehyde to the imine was monitored by <sup>1</sup>H NMR. The MgSO<sub>4</sub> was filtered off, and the residue was dissolved in 1 mL of MeOH. NaBH<sub>4</sub> (0.465–0.755 mmol, 3–5 equiv) was added portionwise, and the reaction mixture was stirred at rt until complete conversion to the secondary amine, as analyzed by HPLC. Water and AcOEt were added, and the layers were separated. The aqueous phase was extracted 3 times with AcOEt, and the combined organic layers were dried over MgSO<sub>4</sub>, filtered off, and evaporated in vacuo. The crude mixtures were purified by semipreparative reverse-phase (RP) HPLC, and the desired secondary amines were isolated after lyophilization, as TFA salt. The structures of the desired products were confirmed by <sup>1</sup>H NMR and mass spectrometry (MS) analysis. When the primary amine used in the reaction was insoluble in DCM, the reductive amination was performed in MeOH, adding the NaBH<sub>4</sub> only after complete conversion to the imine. When the HCl salt of the primary amine was used, the pH of the reaction mixture was adjusted to 5 means of <i>N</i>-methylmorpholine. The pH was monitored immediately after the addition of the amine as well as during the imine formation reaction.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Derivative products of compounds <b>2</b>, <b>3</b>, <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4-2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Derivative Products of Compound <b>5</b> (<a class="ref internalNav" href="#fig12" aria-label="Figure 12">Figure 12</a>)</h3><div class="NLM_p">A solution of the respective aldehyde (1 equiv) and amine (1.06 equiv) in MeOH (0.1 mol/L) was stirred at room temperature under an argon atmosphere for 24 h. NaBH<sub>4</sub> (1.6 equiv) was added slowly, followed by further stirring for 10–15 min. The reaction mixture was quenched with 3 M NaOH solution and the product extracted with EtOAc. The organic layer was washed with water, saturated aqueous NaCl, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude reaction product was purified by flash column chromatography.</div><figure id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Derivative products of compound <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4-2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Analytical Data</h3><div class="NLM_p last">The purity of all compounds exceeds 95% and was determined by combustion analysis, quantitative <sup>1</sup>H NMR (qHNMR), or analytical RP-HPLC.</div></div><div id="sec4-2-5-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-(1,3-Thiazol-2-yl)phenylmethyl-((2<i>R</i>)-4,4,4-trifluorobutan-2-yl)amine <b>(</b>6)</h3><div class="NLM_p last">White powder. Yield: 10%. <i>t</i><sub>R</sub> (agilent, standard gradient): 11.007 min. ESI-MS: <i>m</i>/<i>z</i> 301.04 [M + H<sup>+</sup>] (calculated for C<sub>14</sub>H<sub>15</sub>H<sup>+</sup>F<sub>3</sub>N<sub>2</sub>S, 301.10). <sup>1</sup>H NMR, CDCl<sub>3</sub>, 250.53 MHz: δ (ppm) 7.93–7.90 (m, 3H), 7.44 (m, 2H), 7.36 (m, 1H), 3.94 (br s, 2H), 3.35 (m, 1H), 2.71 (m, 1H), 2.39 (m, 1H), 1.38 (d, 3H, <i>J</i> = 6 Hz), 0.87 (m, 1H).</div></div><div id="sec4-2-5-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-(Furan-2-yl)-2-([4-(1,3-thiazol-2-yl)phenyl]methylamino)ethanol (<b>7</b>)</h3><div class="NLM_p last">White powder. Yield: 22%. <i>t</i><sub>R</sub> (agilent, standard gradient): 10.327 min. ESI-MS: <i>m</i>/<i>z</i> 310.02 [M + H<sup>+</sup>] (calculated for C<sub>16</sub>H<sub>16</sub>H<sup>+</sup>N<sub>2</sub>O<sub>2</sub>S, 310.10). <sup>1</sup>H NMR, CDCl<sub>3</sub>, 250 MHz: δ (ppm) 7.94–7.91 (m, 3H), 7.50–7.41 (m, 4H), 6.58 (d, 1H, <i>J</i> = 3.18 Hz), 6.45–6.43 (m, 1H), 4.43 (s, 2H), 4.11–3.92 (m, 3H).</div></div><div id="sec4-2-5-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 1-(2-Methylphenyl)ethyl(4-(pyrimidin-2-yl)phenylmethyl)amine (<b>8</b>)</h3><div class="NLM_p last">White powder. Yield: 40%. <i>t</i><sub>R</sub> (agilent, standard gradient): 12.037 min. ESI-MS: <i>m</i>/<i>z</i> 304.11 [M + H<sup>+</sup>] (calculated for C<sub>20</sub>H<sub>21</sub>H<sup>+</sup>N<sub>3</sub>: 304.18). <sup>1</sup>H NMR, CDCl<sub>3</sub>, 250 MHz: δ (ppm) 8.75–8.68 (m, 2H), 8.31 (d, 2H, <i>J</i> = 7.34 Hz), 7.78 (d, 1H, <i>J</i> = 7.34 Hz), 7.40–7.29 (m, 3H), 7.17–7.10 (m, 3H), 4.53–4.44 (m, 1H), 3.78 (s, 2H), 2.11 (s, 3H), 1.41–1.35 (m, 3H).</div></div><div id="sec4-2-5-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-1-(4-Chlorophenyl)ethyl-3-(4-fluorophenyl)cyclobutan-1-amine (<b>9</b>)</h3><div class="NLM_p last">Waxy white solid. Yield: 24%. <i>t</i><sub>R</sub> (agilent, standard gradient): 14.059 min. ESI-MS: <i>m</i>/<i>z</i> 304, 05 [M + H<sup>+</sup>, <sup>35</sup>Cl], 306.05 [M + H<sup>+</sup>, <sup>37</sup>Cl], (calculated for C<sub>18</sub>H<sub>19</sub>H<sup>+</sup>ClFN, 304.17 (35Cl), 306.12 (37Cl) in expected ratio 3:1). <sup>1</sup>H NMR, CDCl<sub>3</sub>, 250 MHz: δ (ppm) 7.42 (d, 2H, <i>J</i> = 8.54 Hz), 7.37–7.30 (m, 2H), 7.13 (dd, 1H, <i>J</i> = 5.37, 8.60 Hz), 7.02–6.87 (m, 3H), 3.65–3.52 (m, 1H), 3.06–2.98 (m, 1H), 2.60–2.41 (m, 3H), 2.28–2.19 (m, 2H), 1.60 (d, 3H, <i>J</i> = 6.79 Hz).</div></div><div id="sec4-2-5-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i>-(1-Benzothiophen-2-ylmethyl)-3-methyloxetan-3-amine (<b>10</b>)</h3><div class="NLM_p last">White powder. Yield: 16%. <i>t</i><sub>R</sub> (agilent, standard gradient): 9.822 min. ESI-MS: <i>m</i>/<i>z</i> 146.98 [M + H<sup>+</sup>] fragment corresponding to benzothiophene-CH<sub>2</sub> (calculated for C<sub>13</sub>H<sub>15</sub>H<sup>+</sup>NOS, 234.10). <sup>1</sup>H NMR, CDCl<sub>3</sub>, 250 MHz: δ (ppm) 7.78–7.70 (m, 2H), 7.39 (s, 1H), 7.36–7.31 (m, 2H), 4.66 (d, 2H, <i>J</i> = 6.44 Hz), 4.24 (s, 2H), 4.13 (d, 2H, <i>J</i> = 6.44 Hz), 1.59 (s, 3H).</div></div><div id="sec4-2-5-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 3-(4-Chlorophenyl)-<i>N</i>-(2-(5-methoxy-1-methyl-1<i>H</i>-indol-3-yl)ethyl)propan-1-amine (<b>11</b>)</h3><div class="NLM_p last">The crude reaction product was purified by flash column chromatography using dichloromethane/methanol (gradient 0  →  5% over 18 min, +1% ammonia) as eluent; yellow-brown oil. Yield: 40%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 7.19–7.15 (m, 3H), 7.08 (d, <i>J</i> = 2.3 Hz, 1H), 7.03 (d, <i>J</i> = 8.4 Hz, 2H), 6.89 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 6.84 (s, 1H), 4.55 (brs, 1H), 3.85 (s, 3H), 3.70 (s, 3H), 3.08–2.96 (sm, 4H), 2.72 (t, <i>J</i> = 7.4 Hz, 2H), 2.59 (t, <i>J</i> = 7.7 Hz, 2H), 1.95–1.85 (sm, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ = 154.0, 140.0, 132.6, 131.7, 129.8, 128.6, 128.1, 127.6, 112.1, 110.9, 110.2, 100.9, 56.2, 49.7, 48.7, 32.9, 32.8, 30.4, 24.8. HRMS (EI+) calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>2</sub>O [M]<sup>+</sup> 356.165541; found 356.164252. Combustion analysis calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>2</sub>O·0.5H<sub>2</sub>O: C, 68.93; H, 7.16; N, 7.66. Found: C, 68.68; H, 6.93; N, 7.33.</div></div><div id="sec4-2-5-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(2-(5-Methoxy-1-methyl-1<i>H</i>-indol-3-yl)ethyl)-3-(3-(trifluoromethyl)phenyl)propan-1-amine (<b>12</b>)</h3><div class="NLM_p last">The crude reaction product was purified twice by flash column chromatography using dichloromethane/methanol (gradient 0  →  5% over 18 min, +1% ammonia) as eluent for the first and dichloromethane/methanol (99:1, +1% ammonia) as eluent for the second purification step; yellow solid. Yield: 35%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 7.45–7.29 (m, 4H), 7.19 (psd, <i>J</i> = 8.8 Hz, 1H), 7.07 (d, <i>J</i> = 2.3 Hz, 1H), 6.90 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 6.86 (s, 1H), 3.87 (s, 3H), 3.71 (s, 3H), 2.95 (s, 4H), 2.68 (pst, <i>J</i> = 7.4 Hz, 4H), 1.96 (brs, 1H), 1.88–1.78 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ = 153.9, 143.1, 132.7, 131.9, 130.8 (q, <sup>2</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 31.7 Hz), 128.8, 128.3, 127.5, 125.2 (q, <sup>3</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 3.7 Hz), 124.4 (q, <sup>1</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 270.5 Hz), 122.8 (q, <sup>3</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 3.9 Hz), 112.0, 111.9, 110.1, 101.1, 56.2, 50.2, 49.2, 33.5, 32.8, 31.4, 25.7. qNMR: 95.7% purity. HRMS (EI+) calcd for C<sub>22</sub>H<sub>25</sub>F<sub>3</sub>N<sub>2</sub>O [M]<sup>+</sup> 390.191898; found 390.188957.</div></div><div id="sec4-2-5-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(2-Fluoro-5-(trifluoromethoxy)benzyl)-2-(5-methoxy-1-methyl-1<i>H</i>-indol-3-yl)ethan-1-amine (<b>13</b>)</h3><div class="NLM_p last">The crude reaction product was purified by flash column chromatography using dichloromethane/methanol (99:1, + 1% ammonia). Yellow-brown oil. Yield: 64%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 7.29–7.27 (m, 1H), 7.20 (psd, <i>J</i> = 8.8 Hz, 1H), 7.11–6.99 (m, 3H), 6.91 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 6.87 (s, 1H), 3.87 (s, 5H), 3.72 (s, 3H), 2.98 (s, 4H), 1.72 (brs, 1H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ = 159.1 (d, <sup>1</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 245.4 Hz), 153.9, 145.2, 132.7, 129.5 (d, <sup>2</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 17.2 Hz), 128.3, 127.5, 122.8 (d, <sup>3</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 5.5 Hz), 121.0 (d, <sup>3</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 8.9 Hz), 120.6 (q, <sup>1</sup>J<sub><i>C</i>,<i>F</i></sub> = 255.1 Hz), 116.3 (d, <sup>2</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 24.6 Hz), 112.0, 111.8, 110.1, 101.0, 56.0, 49.6, 46.6 (d, <sup>3</sup><i>J</i><sub><i>C</i>,<i>F</i></sub> = 2.7 Hz), 32.8, 25.7. HRMS (EI+) calcd for C<sub>20</sub>H<sub>20</sub>F<sub>4</sub>N<sub>2</sub>O<sub>2</sub> [M]<sup>+</sup> 396.146091; found 396.144254. Combustion analysis calcd for C<sub>20</sub>H<sub>20</sub>F<sub>4</sub>N<sub>2</sub>O<sub>2</sub>: C, 60.60; H, 5.09; N, 7.07. Found: C, 60.79; H, 5.14; N, 7.24.</div></div><div id="sec4-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Radioligand Displacement Assay</h3><div class="NLM_p">Compounds were examined for their ability to inhibit the binding of [<sup>3</sup>H]-dihydroalprenolol ([<sup>3</sup>H]-DHA; 2 nM final concentration) to membranes of Sf9 cells expressing the β<sub>2</sub>AR. An amount of 5 μg of total protein was mixed with each compound in concentrations ranging from 10<sup>–10</sup> M to 10<sup>–3</sup> M. The reaction mixtures were incubated for 2 h at rt, and free radioligand was removed by filtering over a Whatman GF/C filter. Filters were washed six times, then dried, and 40 μL of scintillation fluid (MicroScintTM-O, PerkinElmer) was added. Radioactivity (counts per minutes [cpm]) retained on the filters was determined in a Wallac MicroBeta TriLux scintillation counter. The half-maximal inhibitory concentrations (IC<sub>50</sub>) for these compounds were calculated from normalized dose–response curves obtained using a one-site competition binding model (nonlinear regression analyses) of the GraphPad Prism software program.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Each assay was performed in triplicates.</div><div class="NLM_p last">For each compound to be tested, a comparative assay was performed. A first assay was with a β<sub>2</sub>-AR-nanobody fusion locked in an active state by a G-protein-mimicking nanobody called β<sub>2</sub>AR-Nb80.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> A second screen was conducted relative to the basal state of the same receptor in a fusion with an irrelevant nanobody called β<sub>2</sub>AR-Nb69.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Doing so allowed one to classify the candidate efficacy of each hit (i.e., agonist (AC), antagonist (AntC), or inverse-agonist candidates (IAC)).<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> An AC molecule is defined by a shift in affinity between β<sub>2</sub>AR-Nb69 and β<sub>2</sub>AR-Nb80 higher than 1.1 (i.e., the compound is more selective toward β<sub>2</sub>AR-Nb69). An IAC molecule is defined by a shift in affinity between β<sub>2</sub>AR-Nb69 and β<sub>2</sub>AR-Nb80 below 0.9. An AntC molecule is defined when no obvious selectivity is observed between the two receptors (shift of 1 ± 0.1). IC<sub>50</sub> values were transformed to <i>K</i><sub>i</sub> using the Cheng–Prusoff equation.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01558">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01558" class="ext-link">10.1021/acs.jmedchem.7b01558</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Description of the Web interface; workflow of the GVM procedure; similarity search and PAINS filter results; IC<sub>50</sub> values of the tested compounds; predicted binding mode and comparison to crystal structures; analysis of <sup>1</sup>H NMR, and MS of the tested compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and experimental data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB files of predicted binding modes of the 13 compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_003.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf">jm7b01558_si_001.pdf (4.93 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_002.csv">jm7b01558_si_002.csv (0.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_003.zip">jm7b01558_si_003.zip (236.37 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01558" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54235" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54235" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Kolb</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4089-614X" title="Orcid link">http://orcid.org/0000-0003-4089-614X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#19697c6d7c6b377276757b596c77703474786b7b6c6b7e377d7c"><span class="__cf_email__" data-cfemail="572732233225793c383b351722393e7a3a362535222530793332">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florent Chevillard</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helena Rimmer</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Chemistry and Center for
Tumor Biology and Immunology, Philipps-University
Marburg, Hans-Meerwein-Straße
3, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cecilia Betti</span> - <span class="hlFld-Affiliation affiliation">Research Group of Organic Chemistry, Departments
of Chemistry and Bio-Engineering Sciences, Vrije Universiteit Brussel, Pleinlaan 2, 1050 Brussels, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Els Pardon</span> - <span class="hlFld-Affiliation affiliation">VIB-VUB Center for
Structural Biology, VIB, 1050 Brussels, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Structural Biology Brussels, Vrije Universiteit Brussel, 1050 Brussels, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven Ballet</span> - <span class="hlFld-Affiliation affiliation">Research Group of Organic Chemistry, Departments
of Chemistry and Bio-Engineering Sciences, Vrije Universiteit Brussel, Pleinlaan 2, 1050 Brussels, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niek van Hilten</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Steyaert</span> - <span class="hlFld-Affiliation affiliation">VIB-VUB Center for
Structural Biology, VIB, 1050 Brussels, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Structural Biology Brussels, Vrije Universiteit Brussel, 1050 Brussels, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wibke E. Diederich</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Chemistry and Center for
Tumor Biology and Immunology, Philipps-University
Marburg, Hans-Meerwein-Straße
3, 35032 Marburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5671-6575" title="Orcid link">http://orcid.org/0000-0001-5671-6575</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43874" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43874" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">P.K. thanks the German Research Foundation DFG for Emmy Noether Fellowship KO4095/1-1 and Heisenberg Professorship KO4095/4-1. The authors acknowledge E. Beke for technical assistance. P.K. and J.S. participated in COST Action CM1207 “GLISTEN”. C.B and S.B. thank the Strategic Research Program (SRP) of the Vrije Universiteit Brussel for the provided financial support.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i43" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i43"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i44" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i44"> Abbreviations Used</h2><tr><td class="NLM_term">PINGUI</td><td class="NLM_def"><p class="first last">Python in silico de novo growing utilities</p></td></tr><tr><td class="NLM_term">GVM</td><td class="NLM_def"><p class="first last">growing via merging</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug discovery</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, toxicity</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">β<sub>2</sub>AR-Nb80</td><td class="NLM_def"><p class="first last">a β<sub>2</sub>-AR-nanobody fusion locked in an active state by a G-protein-mimicking nanobody</p></td></tr><tr><td class="NLM_term">β<sub>2</sub>AR-Nb69</td><td class="NLM_def"><p class="first last">a β<sub>2</sub>-AR-nanobody fusion locked in an active state by an irrelevant nanobody</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28434" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28434" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahnke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0ljVSgxT9CEdKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1038/nrd2220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned&doi=10.1038%2Fnrd2220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0lj5a6bw5KAZ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D211%26epage%3D219%26doi%3D10.1038%2Fnrd2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Experiences in fragment-based drug discovery</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">232</span><span class="refDoi"> DOI: 10.1016/j.tips.2012.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2Fj.tips.2012.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=22459076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=224-232&author=C.+W.+Murrayauthor=M.+L.+Verdonkauthor=D.+C.+Rees&title=Experiences+in+fragment-based+drug+discovery&doi=10.1016%2Fj.tips.2012.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">224-232</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) has become established in both industry and academia as an alternative approach to high-throughput screening for the generation of chem. leads for drug targets.  In FBDD, specialised detection methods are used to identify small chem. compds. (fragments) that bind to the drug target, and structural biol. is usually employed to establish their binding mode and to facilitate their optimization.  In this article, we present three recent and successful case histories in FBDD.  We then re-examine the key concepts and challenges of FBDD with particular emphasis on recent literature and our own experience from a substantial no. of FBDD applications.  Our opinion is that careful application of FBDD is living up to its promise of delivering high quality leads with good phys. properties and that in future many drug mols. will be derived from fragment-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUzcsuUdmx4bVg90H21EOLACvtfcHk0lj5a6bw5KAZ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D&md5=c97b6b84a4afeadb87ee7d46e6d2bb91</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2012.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2012.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DExperiences%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D224%26epage%3D232%26doi%3D10.1016%2Fj.tips.2012.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spevak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habets, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shellooe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marimuthu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K. Y J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Andrea, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koehler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stumm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grippo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puzanov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McArthur, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nathanson, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolop, K.</span><span> </span><span class="NLM_article-title">Clinical efficacy of a RAF inhibitor needs broad target blockade in BRAF-mutant melanoma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">596</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span><span class="refDoi"> DOI: 10.1038/nature09454</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature09454" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=20823850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1ajtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2010&pages=596-599&author=G.+Bollagauthor=P.+Hirthauthor=J.+Tsaiauthor=J.+Zhangauthor=P.+N.+Ibrahimauthor=H.+Choauthor=W.+Spevakauthor=C.+Zhangauthor=Y.+Zhangauthor=G.+Habetsauthor=E.+Burtonauthor=B.+Wongauthor=G.+Tsangauthor=B.+L.+Westauthor=B.+Powellauthor=R.+Shellooeauthor=A.+Marimuthuauthor=H.+Nguyenauthor=K.+Y+J.+Zhangauthor=D.+R.+Artisauthor=J.+Schlessingerauthor=F.+Suauthor=B.+Higginsauthor=R.+Iyerauthor=K.+D%E2%80%99Andreaauthor=A.+Koehlerauthor=M.+Stummauthor=P.+S.+Linauthor=R.+J.+Leeauthor=J.+Grippoauthor=I.+Puzanovauthor=K.+B.+Kimauthor=A.+Ribasauthor=G.+McArthurauthor=J.+Sosmanauthor=P.+B.+Chapmanauthor=K.+T.+Flahertyauthor=X.+Xuauthor=K.+L.+Nathansonauthor=K.+Nolop&title=Clinical+efficacy+of+a+RAF+inhibitor+needs+broad+target+blockade+in+BRAF-mutant+melanoma&doi=10.1038%2Fnature09454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical efficacy of a RAF inhibitor needs broad target blockade in BRAF-mutant melanoma</span></div><div class="casAuthors">Bollag, Gideon; Hirth, Peter; Tsai, James; Zhang, Jiazhong; Ibrahim, Prabha N.; Cho, Hanna; Spevak, Wayne; Zhang, Chao; Zhang, Ying; Habets, Gaston; Burton, Elizabeth A.; Wong, Bernice; Tsang, Garson; West, Brian L.; Powell, Ben; Shellooe, Rafe; Marimuthu, Adhirai; Nguyen, Hoa; Zhang, Kam Y. J.; Artis, Dean R.; Schlessinger, Joseph; Su, Fei; Higgins, Brian; Iyer, Raman; D'Andrea, Kurt; Koehler, Astrid; Stumm, Michael; Lin, Paul S.; Lee, Richard J.; Grippo, Joseph; Puzanov, Igor; Kim, Kevin B.; Ribas, Antoni; McArthur, Grant A.; Sosman, Jeffrey A.; Chapman, Paul B.; Flaherty, Keith T.; Xu, Xiaowei; Nathanson, Katherine L.; Nolop, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">7315</span>),
    <span class="NLM_cas:pages">596-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">B-RAF is the most frequently mutated protein kinase in human cancers.  The finding that oncogenic mutations in BRAF are common in melanoma, followed by the demonstration that these tumors are dependent on the RAF/MEK/ERK pathway, offered hope that inhibition of B-RAF kinase activity could benefit melanoma patients.  Herein, we describe the structure-guided discovery of PLX4032 (RG7204), a potent inhibitor of oncogenic B-RAF kinase activity.  Preclin. expts. demonstrated that PLX4032 selectively blocked the RAF/MEK/ERK pathway in BRAF mutant cells and caused regression of BRAF mutant xenografts.  Toxicol. studies confirmed a wide safety margin consistent with the high degree of selectivity, enabling Phase 1 clin. trials using a cryst. formulation of PLX4032 (ref. 5).  In a subset of melanoma patients, pathway inhibition was monitored in paired biopsy specimens collected before treatment initiation and following 2 wk of treatment.  This anal. revealed substantial inhibition of ERK phosphorylation, yet clin. evaluation did not show tumor regressions.  At higher drug exposures afforded by a new amorphous drug formulation, greater than 80% inhibition of ERK phosphorylation in the tumors of patients correlated with clin. response.  Indeed, the Phase 1 clin. data revealed a remarkably high 81% response rate in metastatic melanoma patients treated at an oral dose of 960 mg twice daily.  These data demonstrate that BRAF-mutant melanomas are highly dependent on B-RAF kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpYwlns1EfcLVg90H21EOLACvtfcHk0lj8Uu0cHCuEQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1ajtbfK&md5=8a26d0ab41970f39a109ddafff2f17b4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature09454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09454%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DBurton%26aufirst%3DE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DK.%2BY%2BJ.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DF.%26aulast%3DHiggins%26aufirst%3DB.%26aulast%3DIyer%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DK.%26aulast%3DKoehler%26aufirst%3DA.%26aulast%3DStumm%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DP.%2BS.%26aulast%3DLee%26aufirst%3DR.%2BJ.%26aulast%3DGrippo%26aufirst%3DJ.%26aulast%3DPuzanov%26aufirst%3DI.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DMcArthur%26aufirst%3DG.%26aulast%3DSosman%26aufirst%3DJ.%26aulast%3DChapman%26aufirst%3DP.%2BB.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DNathanson%26aufirst%3DK.%2BL.%26aulast%3DNolop%26aufirst%3DK.%26atitle%3DClinical%2520efficacy%2520of%2520a%2520RAF%2520inhibitor%2520needs%2520broad%2520target%2520blockade%2520in%2520BRAF-mutant%2520melanoma%26jtitle%3DNature%26date%3D2010%26volume%3D467%26spage%3D596%26epage%3D599%26doi%3D10.1038%2Fnature09454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, We.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mamo, Sh.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillette, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haass, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sproesser, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smalley, K. S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marimuthu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settachatgul, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shellooe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantwell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K. Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habets, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herlyn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span> </span><span class="NLM_article-title">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">3041</span><span class="NLM_x">–</span> <span class="NLM_lpage">3046</span><span class="refDoi"> DOI: 10.1073/pnas.0711741105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.0711741105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=18287029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=3041-3046&author=J.+Tsaiauthor=J.+T.+Leeauthor=We.+Wangauthor=J.+Zhangauthor=H.+Choauthor=Sh.+Mamoauthor=R.+Bremerauthor=S.+Gilletteauthor=J.+Kongauthor=N.+K.+Haassauthor=K.+Sproesserauthor=L.+Liauthor=K.+S.+M.+Smalleyauthor=D.+Fongauthor=Y.+Zhuauthor=A.+Marimuthuauthor=H.+Nguyenauthor=B.+Lamauthor=J.+Liuauthor=I.+Cheungauthor=J.+Riceauthor=Y.+Suzukiauthor=C.+Luuauthor=C.+Settachatgulauthor=R.+Shellooeauthor=J.+Cantwellauthor=S.+Kimauthor=J.+Schlessingerauthor=K.+Y.+J.+Zhangauthor=B.+L.+Westauthor=B.+Powellauthor=G.+Habetsauthor=C.+Zhangauthor=P.+N.+Ibrahimauthor=P.+Hirthauthor=D.+R.+Artisauthor=M.+Herlynauthor=G.+Bollag&title=Discovery+of+a+selective+inhibitor+of+oncogenic+B-Raf+kinase+with+potent+antimelanoma+activity&doi=10.1073%2Fpnas.0711741105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span></div><div class="casAuthors">Tsai, James; Lee, John T.; Wang, Weiru; Zhang, Jiazhong; Cho, Hanna; Mamo, Shumeye; Bremer, Ryan; Gillette, Sam; Kong, Jun; Haass, Nikolas K.; Sproesser, Katrin; Li, Ling; Smalley, Keiran S. M.; Fong, Daniel; Zhu, Yong-Liang; Marimuthu, Adhirai; Nguyen, Hoa; Lam, Billy; Liu, Jennifer; Cheung, Ivana; Rice, Julie; Suzuki, Yoshihisa; Luu, Catherine; Settachatgul, Calvin; Shellooe, Rafe; Cantwell, John; Kim, Sung-Hou; Schlessinger, Joseph; Zhang, Kam Y. J.; West, Brian L.; Powell, Ben; Habets, Gaston; Zhang, Chao; Ibrahim, Prabha N.; Hirth, Peter; Artis, Dean R.; Herlyn, Meenhard; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3041-3046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">BRAFV600E is the most frequent oncogenic protein kinase mutation known.  Furthermore, inhibitors targeting "active" protein kinases have demonstrated significant utility in the therapeutic repertoire against cancer.  Therefore, the authors pursued the development of specific kinase inhibitors targeting B-Raf, and the V600E allele in particular.  By using a structure-guided discovery approach, a potent and selective inhibitor of active B-Raf has been discovered.  PLX4720, a 7-azaindole deriv. that inhibits B-RafV600E with an IC50 of 13 nM, defines a class of kinase inhibitor with marked selectivity in both biochem. and cellular assays.  PLX4720 preferentially inhibits the active B-RafV600E kinase compared with a broad spectrum of other kinases, and potent cytotoxic effects are also exclusive to cells bearing the V600E allele.  Consistent with the high degree of selectivity, ERK phosphorylation is potently inhibited by PLX4720 in B-RafV600E-bearing tumor cell lines but not in cells lacking oncogenic B-Raf.  In melanoma models, PLX4720 induces cell cycle arrest and apoptosis exclusively in B-RafV600E-pos. cells.  In B-RafV600E-dependent tumor xenograft models, orally dosed PLX4720 causes significant tumor growth delays, including tumor regressions, without evidence of toxicity.  The work described here represents the entire discovery process, from initial identification through structural and biol. studies in animal models to a promising therapeutic for testing in cancer patients bearing B-RafV600E-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPX53IQCbDJ7Vg90H21EOLACvtfcHk0lj8Uu0cHCuEQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D&md5=273b94202f99984e6eb24723d0c6141f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711741105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711741105%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DWe.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DMamo%26aufirst%3DSh.%26aulast%3DBremer%26aufirst%3DR.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHaass%26aufirst%3DN.%2BK.%26aulast%3DSproesser%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DLuu%26aufirst%3DC.%26aulast%3DSettachatgul%26aufirst%3DC.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520inhibitor%2520of%2520oncogenic%2520B-Raf%2520kinase%2520with%2520potent%2520antimelanoma%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D3041%26epage%3D3046%26doi%3D10.1073%2Fpnas.0711741105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery and design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span><span class="refDoi"> DOI: 10.1021/ci400731w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400731w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=693-704&author=D.+Joseph-McCarthyauthor=A.+J.+Campbellauthor=G.+Kernauthor=D.+Moustakas&title=Fragment-based+lead+discovery+and+design&doi=10.1021%2Fci400731w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Lead Discovery and Design</span></div><div class="casAuthors">Joseph-McCarthy, Diane; Campbell, Arthur J.; Kern, Gunther; Moustakas, Demetri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">693-704</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery and design has and continues to show increasing promise in drug discovery.  In this article, the current state of the art in terms of hot-spot characterization, fragment screening techniques, and fragment-based design is discussed.  Three overall fragment-based lead generation strategies are explored and involve the chem. biol. characterization of biol. targets via fragment screening, fragment screening as a complementary approach to high-throughput screening of drug-like compds., and direct fragment-based drug discovery, resp.  The evolution and development of fragment libraries is described.  With an emphasis on computational approaches and the strategies applied at AstraZeneca, the review illustrates how integration of data from one regime can inform the design of expts. in the other, ultimately leading to the discovery of high quality chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCa5krM_w737Vg90H21EOLACvtfcHk0lhmDA35K1mzfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D&md5=31b72219dfd8cec7e3612de1b61f8150</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fci400731w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400731w%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DKern%26aufirst%3DG.%26aulast%3DMoustakas%26aufirst%3DD.%26atitle%3DFragment-based%2520lead%2520discovery%2520and%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D693%26epage%3D704%26doi%3D10.1021%2Fci400731w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ochiana, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollastri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">E2585</span><span class="NLM_x">–</span> <span class="NLM_lpage">E2594</span><span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=25918377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-E2594&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+deconstruction%3A+Why+some+fragment+binding+positions+are+conserved+and+others+are+not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Jehle, Sefan; Luo, Lingqi; Ochiana, Stefan O.; Jones, Elizabeth V.; Pollastri, Michael; Allen, Karen N.; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E2585-E2594</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) relies on the premise that the fragment binding mode will be conserved on subsequent expansion to a larger ligand.  However, no general condition has been established to explain when fragment binding modes will be conserved.  We show that a remarkably simple condition can be developed in terms of how fragments coincide with binding energy hot spots-regions of the protein where interactions with a ligand contribute substantial binding free energy-the locations of which can easily be detd. computationally.  Because a substantial fraction of the free energy of ligand binding comes from interacting with the residues in the energetically most important hot spot, a ligand moiety that sufficiently overlaps with this region will retain its location even when other parts of the ligand are removed.  This hypothesis is supported by eight case studies.  The condition helps identify whether a protein is suitable for FBDD, predicts the size of fragments required for screening, and dets. whether a fragment hit can be extended into a higher affinity ligand.  Our results show that ligand binding sites can usefully be thought of in terms of an anchor site, which is the top-ranked hot spot and dominates the free energy of binding, surrounded by a no. of weaker satellite sites that confer improved affinity and selectivity for a particular ligand and that it is the intrinsic binding potential of the protein surface that dets. whether it can serve as a robust binding site for a suitably optimized ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH6jbOut_iv7Vg90H21EOLACvtfcHk0lhmDA35K1mzfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D&md5=d56b18fb4e4befde24037b07e52d8b62</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520deconstruction%253A%2520Why%2520some%2520fragment%2520binding%2520positions%2520are%2520conserved%2520and%2520others%2520are%2520not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE2585%26epage%3DE2594%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span> </span><span class="NLM_article-title">Molecular complexity and its impact on the probability of finding leads for drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">856</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span><span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+complexity+and+its+impact+on+the+probability+of+finding+leads+for+drug+discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0lhmDA35K1mzfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520complexity%2520and%2520its%2520impact%2520on%2520the%2520probability%2520of%2520finding%2520leads%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eZAYyutcSeo7bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Rawlins, P.</span><span> </span><span class="NLM_article-title">Current trends in label-free technologies</span> <span class="citation_source-journal">Drug Discovery World</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=17-26&author=P.+Rawlins&title=Current+trends+in+label-free+technologies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRawlins%26aufirst%3DP.%26atitle%3DCurrent%2520trends%2520in%2520label-free%2520technologies%26jtitle%3DDrug%2520Discovery%2520World%26date%3D2010%26volume%3D11%26spage%3D17%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bartoli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fincham, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fattori, D.</span><span> </span><span class="NLM_article-title">Fragment-based drug design: combining philosophy with technology</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">429</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=422-429&author=S.+Bartoliauthor=C.+Finchamauthor=D.+Fattori&title=Fragment-based+drug+design%3A+combining+philosophy+with+technology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBartoli%26aufirst%3DS.%26aulast%3DFincham%26aufirst%3DC.%26aulast%3DFattori%26aufirst%3DD.%26atitle%3DFragment-based%2520drug%2520design%253A%2520combining%2520philosophy%2520with%2520technology%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2010%26volume%3D10%26spage%3D422%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: A fragment-based approach using biophysical methods</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4992</span><span class="NLM_x">–</span> <span class="NLM_lpage">5000</span><span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+A+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0ljrt9tV4kBi6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520A%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span><span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0ljrt9tV4kBi6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Schiebel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radeva, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krimmer, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stieler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrmann, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huschmann, F. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Six biophysical screening methods miss a large proportion of crystallographically discovered fragment hits: A case study</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1693</span><span class="NLM_x">–</span> <span class="NLM_lpage">1701</span><span class="refDoi"> DOI: 10.1021/acschembio.5b01034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1693-1701&author=J.+Schiebelauthor=N.+Radevaauthor=S.+G.+Krimmerauthor=X.+Wangauthor=M.+Stielerauthor=F.+R.+Ehrmannauthor=K.+Fuauthor=A.+Metzauthor=F.+U.+Huschmannauthor=M.+S.+Weissauthor=U.+Muellerauthor=A.+Heineauthor=G.+Klebe&title=Six+biophysical+screening+methods+miss+a+large+proportion+of+crystallographically+discovered+fragment+hits%3A+A+case+study&doi=10.1021%2Facschembio.5b01034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study</span></div><div class="casAuthors">Schiebel, Johannes; Radeva, Nedyalka; Krimmer, Stefan G.; Wang, Xiaojie; Stieler, Martin; Ehrmann, Frederik R.; Fu, Kan; Metz, Alexander; Huschmann, Franziska U.; Weiss, Manfred S.; Mueller, Uwe; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1693-1701</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based lead discovery (FBLD) has become a pillar in drug development.  Typical applications of this method comprise at least two biophys. screens as prefilter and a follow-up crystallog. expt. on a subset of fragments.  Clearly, structural information is pivotal in FBLD, but a key question is whether such a screening cascade strategy will retrieve the majority of fragment-bound structures.  We therefore set out to screen 361 fragments for binding to endothiapepsin, a representative of the challenging group of aspartic proteases, employing six screening techniques and crystallog. in parallel.  Crystallog. resulted in the very high no. of 71 structures.  Yet alarmingly, 44% of these hits were not detected by any biophys. screening approach.  Moreover, any screening cascade, building on the results from two or more screening methods, would have failed to predict at least 73% of these hits.  We thus conclude that, at least in the present case, the frequently applied biophys. prescreening filters deteriorate the no. of possible X-ray hits while only the immediate use of crystallog. enables exhaustive retrieval of a max. of fragment structures, which represent a rich source guiding hit-to-lead-to-drug evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgOpXVyCfeiLVg90H21EOLACvtfcHk0lhfCOitY6_mKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D&md5=30f16cf75361d0ca11e7b75ad796d207</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01034%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DStieler%26aufirst%3DM.%26aulast%3DEhrmann%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSix%2520biophysical%2520screening%2520methods%2520miss%2520a%2520large%2520proportion%2520of%2520crystallographically%2520discovered%2520fragment%2520hits%253A%2520A%2520case%2520study%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1693%26epage%3D1701%26doi%3D10.1021%2Facschembio.5b01034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Miranker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Functionality maps of binding sites: a multiple copy simultaneous search method</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1002/prot.340110104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fprot.340110104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1961699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=29-34&author=A.+Mirankerauthor=M.+Karplus&title=Functionality+maps+of+binding+sites%3A+a+multiple+copy+simultaneous+search+method&doi=10.1002%2Fprot.340110104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Functionality maps of binding sites:  a multiple copy simultaneous search method</span></div><div class="casAuthors">Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-34</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">A new method is proposed for detg. energetically favorable positions and orientations for functional groups on the surface of proteins with known 3-dimensional structure.  Some 1000-5000 copies of a functional group are randomly placed in the site and subjected to simultaneous energy minimization and/or quenched mol. dynamics.  The resulting functionality maps of a protein receptor site, which can take account of its flexibility, can be used for the anal. of protein ligand interactions and rational drug design.  Application of the method to the sialic acid binding site of the influenza coat protein, hemagglutinin, yields functional group min. that correspond with those of the ligand in a cocrystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5im8BqJfF37Vg90H21EOLACvtfcHk0lhfCOitY6_mKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D&md5=34e0a0c36899bfeabcbdfd4790f97a1e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fprot.340110104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340110104%26sid%3Dliteratum%253Aachs%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DFunctionality%2520maps%2520of%2520binding%2520sites%253A%2520a%2520multiple%2520copy%2520simultaneous%2520search%2520method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1991%26volume%3D11%26spage%3D29%26epage%3D34%26doi%3D10.1002%2Fprot.340110104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">SuperStar: a knowledge-based approach for identifying interaction sites in proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1006/jmbi.1999.2809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1006%2Fjmbi.1999.2809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10369784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=1999&pages=1093-1108&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=R.+Taylor&title=SuperStar%3A+a+knowledge-based+approach+for+identifying+interaction+sites+in+proteins&doi=10.1006%2Fjmbi.1999.2809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: A Knowledge-based Approach for Identifying Interaction Sites in Proteins</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1093-1108</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">An empirical method for identifying interaction sites in proteins is described and validated.  The method is based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures.  These data are used in the form of scatterplots that show the exptl. obsd. distribution of one functional group (the "contact group" or "probe") around another.  A template mol. (e.g. a protein binding site) is broken down into structure fragments and the scatterplots, showing the distribution of a chosen probe around these structure fragments, are superimposed on the corresponding parts of the template.  The scatterplots are then translated into a three-dimensional map that shows the propensity of the probe at different positions around the template mol.  The method is illustrated for L-arabinose-binding protein, complexed with L-arabinose and with D-fucose, and for dihydrofolate reductase complexed with methotrexate.  The method is validated on 122 x-ray structures of protein-ligand complexes.  For all the binding sites of these proteins, propensity maps are generated for four different probes: a charged NH+3 nitrogen, a carbonyl oxygen, a hydroxyl oxygen and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  For 74% of these ligand atoms (84% of the solvent-inaccessible ones) the calcd. propensity of the matching probe at the exptl. positions is higher than expected by chance.  For 68% of the atoms (82% of the solvent-inaccessible ones) the propensity of the matching probe is higher than that of the other three probes.  These results indicate that the approach generally gives good predictions for protein-ligand interactions.  The potential applications of the propensity maps range from an aid in manual docking and structure-based drug design to their use in pharmacophore development.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9QVkFGAwFt7Vg90H21EOLACvtfcHk0lhfCOitY6_mKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D&md5=72e09c54c9a631a8271fbe72f2f431b1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.2809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.2809%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DSuperStar%253A%2520a%2520knowledge-based%2520approach%2520for%2520identifying%2520interaction%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D289%26spage%3D1093%26epage%3D1108%26doi%3D10.1006%2Fjmbi.1999.2809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">SuperStar: Improved knowledge-based interaction fields for protein binding sites</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">307</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span><span class="refDoi"> DOI: 10.1006/jmbi.2001.4452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1006%2Fjmbi.2001.4452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=11273705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2001&pages=841-859&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=P.+Watsonauthor=V.+J.+Gilletauthor=P.+Willett&title=SuperStar%3A+Improved+knowledge-based+interaction+fields+for+protein+binding+sites&doi=10.1006%2Fjmbi.2001.4452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: Improved knowledge-based interaction fields for protein binding sites</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Watson, Paul; Gillet, Val; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">841-859</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">SuperStar is an empirical method for identifying interaction sites in proteins, based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures, taken from the IsoStar database.  We describe recent modifications and addns. to SuperStar, validating the results on a test set of 122 x-ray structures of protein-ligand complexes.  In this validation, propensity maps are generated for all the binding sites of these proteins, using four different probes: a charged NH+3 nitrogen atom, a carbonyl oxygen atom, a hydroxyl oxygen atom and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  A peak-searching algorithm is introduced that highlights potential interaction hot spots.  For the three hydrogen-bonding probes - NH+3 nitrogen atom, carbonyl oxygen atom and hydroxyl oxygen atom - the av. distance from the ligand atom to the nearest SuperStar peak is 1.0-1.2 Å (0.8-1.0 Å for solvent-inaccessible ligand atoms).  For the Me carbon atom probe, this distance is about 1.5 Å, probably because interactions to Me groups are much less directional. The most important addn. to SuperStar is the enabling of propensity maps around metal centers - Ca2+, Mg2+ and Zn2+ - in protein binding sites.  The results are validated on a test set of 24 protein-ligand complexes that have a metal ion in their binding site.  Coordination geometries are derived automatically, using only the protein atoms that coordinate to the metal ion.  The correct coordination geometry is derived in approx. 75 % of the cases.  If the derived geometry is assumed during the SuperStar calcn., the av. distance from a ligand atom coordinating to the metal ion to the nearest peak in the propensity map for an oxygen probe is 0.87(7) Å.  If the correct coordination geometry is imposed, this distance reduces to 0.59(7) Å.  This indicates that the SuperStar predictions around metal-binding sites are at least as good as those around other protein groups. Using clustering techniques, a non-redundant set of probes is selected from the set of probes available in the IsoStar database.  The performance in SuperStar of all these probes is tested on the test set of protein-ligand complexes.  With the exception of the "ether oxygen" probe and the "any NH+" probe, all new probes perform as well as the four probes introduced first.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCZ27rmWmxMbVg90H21EOLACvtfcHk0ljz2ZFeRmJLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D&md5=026cdc7beaec6446307bbcc36a7e813f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.4452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.4452%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DWatson%26aufirst%3DP.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSuperStar%253A%2520Improved%2520knowledge-based%2520interaction%2520fields%2520for%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D841%26epage%3D859%26doi%3D10.1006%2Fjmbi.2001.4452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarsi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apostolakis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrhardt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Exhaustive docking of molecular fragments with electrostatic solvation</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19991001)37:1<88::AID-PROT9>3.0.CO;2-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10451553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK1MXls12qsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1999&pages=88-105&author=N.+Majeuxauthor=M.+Scarsiauthor=J.+Apostolakisauthor=C.+Ehrhardtauthor=A.+Caflisch&title=Exhaustive+docking+of+molecular+fragments+with+electrostatic+solvation&doi=10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Exhaustive docking of molecular fragments with electrostatic solvation</span></div><div class="casAuthors">Majeux, Nicolas; Scarsi, Marco; Apostolakis, Joannis; Ehrhardt, Claus; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-105</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method is presented for docking mol. fragments to a rigid protein with evaluation of the binding energy.  Polar fragments are docked with at least one hydrogen bond with the protein while apolar fragments are positioned in the hydrophobic pockets.  The electrostatic contribution to the binding energy, which consists of screened intermol. energy and protein and fragment desolvation terms, is evaluated efficiently by a numerical approach based on the continuum dielec. approxn.  The latter is also used to predetermine the hydrophobic pockets of the protein by rolling a low dielec. sphere over the protein surface and calcg. the electrostatic desolvation of the protein and van der Waals interaction energy.  The method was implemented in the program SEED (solvation energy for exhaustive docking).  The SEED continuum electrostatic approach has been successfully validated by a comparison with finite difference solns. of the Poisson equation for more than 2,500 complexes of small mols. with thrombin and the monomer of HIV-1 aspartic proteinase.  The fragments docked by SEED in the active site of thrombin reproduce the structural features of the interaction patterns between known inhibitors and thrombin.  Moreover, the combinatorial connection of these fragments yields a no. of compds. that are very similar to potent inhibitors of thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOBGz0snZSsrVg90H21EOLACvtfcHk0ljz2ZFeRmJLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXls12qsL8%253D&md5=354f9e32fd0b2ff602f09fd0b81a057e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819991001%252937%253A1%253C88%253A%253AAID-PROT9%253E3.0.CO%253B2-O%26sid%3Dliteratum%253Aachs%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DScarsi%26aufirst%3DM.%26aulast%3DApostolakis%26aufirst%3DJ.%26aulast%3DEhrhardt%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DExhaustive%2520docking%2520of%2520molecular%2520fragments%2520with%2520electrostatic%2520solvation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1999%26volume%3D37%26spage%3D88%26epage%3D105%26doi%3D10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarsi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Efficient electrostatic solvation model for protein-fragment docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">256</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span><span class="refDoi"> DOI: 10.1002/1097-0134(20010201)42:2<256::AID-PROT130>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=11119650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvFyrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2001&pages=256-268&author=N.+Majeuxauthor=M.+Scarsiauthor=A.+Caflisch&title=Efficient+electrostatic+solvation+model+for+protein-fragment+docking&doi=10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient electrostatic solvation model for protein-fragment docking</span></div><div class="casAuthors">Majeux, Nicolas; Scarsi, Marco; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">256-268</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A method is presented for the fast evaluation of the binding energy of a protein-small mol. complex with electrostatic solvation.  It makes use of a fast preprocessing step based on the assumption that the main contribution to electrostatic desolvation upon ligand binding originates from the displacement of the first shell of water mols.  For a rigid protein, the precomputation of the energy contributions on a set of grids allows the estn. of the energy in soln. of about 300 protein-fragment binding modes per s on a personal computer.  The docking procedure is applied to five rigid binding sites whose size ranges from 17 residues to a whole protein of 107 amino acids.  Using a library of 70 mainly rigid mols., known micromolar inhibitors or close analogs are docked and prioritized correctly.  The docking based rank-ordering of the library requires about 5 h and is proposed as a complementary approach to structure-activity relationships by NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE2YlkKSp9MLVg90H21EOLACvtfcHk0ljz2ZFeRmJLvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvFyrug%253D%253D&md5=83f3ebb9884c081c5ebfef839ddabc54</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1097-0134%252820010201%252942%253A2%253C256%253A%253AAID-PROT130%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DScarsi%26aufirst%3DM.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DEfficient%2520electrostatic%2520solvation%2520model%2520for%2520protein-fragment%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2001%26volume%3D42%26spage%3D256%26epage%3D268%26doi%3D10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Hung, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silvestre, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C. I.</span><span> </span><span class="NLM_article-title">Application of fragment growing and fragment linking to the discovery of inhibitors of mycobacterium tuberculosis pantothenate synthetase</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8452</span><span class="NLM_x">–</span> <span class="NLM_lpage">8456</span><span class="refDoi"> DOI: 10.1002/anie.200903821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fanie.200903821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlWmur%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8452-8456&author=A.+W.+Hungauthor=H.+L.+Silvestreauthor=S.+Wenauthor=A.+Ciulliauthor=T.+L.+Blundellauthor=C.+I.+Abell&title=Application+of+fragment+growing+and+fragment+linking+to+the+discovery+of+inhibitors+of+mycobacterium+tuberculosis+pantothenate+synthetase&doi=10.1002%2Fanie.200903821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Fragment Growing and Fragment Linking to the Discovery of Inhibitors of Mycobacterium tuberculosis Pantothenate Synthetase</span></div><div class="casAuthors">Hung, Alvin W.; Silvestre, H. Leonardo; Wen, Shijun; Ciulli, Alessio; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">8452-8456, S8452/1-S8452/26</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Despite the fact that nearly 2 million people a year are killed by Mycobacterium tuberculosis (TB), we still lack a robust pipeline of potential therapies to combat this burden.  Fragment-based approaches have provided a new paradigm in the development of high-quality chem. ligands and have great potential to contribute as starting points in the development of drugs against TB.  To date, studies on pantothenate synthetase (PS) inhibition have focused on the synthesis of analogs of the pantoyl adenylate reaction intermediate or the identification of hits from high-throughput screening.  We describe a combination of fragment-growing and fragment-linking approaches that led to the discovery of a new series of inhibitors of M. tuberculosis PS for the further validation of PS as a potential drug target for TB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjhska_U-KXbVg90H21EOLACvtfcHk0lhNwr_7npHtNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlWmur%252FK&md5=821f86e9e1d7cbd848741962e3ac47dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.200903821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200903821%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DA.%2BW.%26aulast%3DSilvestre%26aufirst%3DH.%2BL.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%2BI.%26atitle%3DApplication%2520of%2520fragment%2520growing%2520and%2520fragment%2520linking%2520to%2520the%2520discovery%2520of%2520inhibitors%2520of%2520mycobacterium%2520tuberculosis%2520pantothenate%2520synthetase%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D8452%26epage%3D8456%26doi%3D10.1002%2Fanie.200903821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Chung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bianchet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amzel, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stivers, J. T.</span><span> </span><span class="NLM_article-title">Impact of linker strain and flexibility in the design of a fragment-based inhibitor</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1038/nchembio.163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnchembio.163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19396178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltValtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=407-413&author=S.+Chungauthor=J.+B.+Parkerauthor=M.+Bianchetauthor=L.+M.+Amzelauthor=J.+T.+Stivers&title=Impact+of+linker+strain+and+flexibility+in+the+design+of+a+fragment-based+inhibitor&doi=10.1038%2Fnchembio.163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of linker strain and flexibility in the design of a fragment-based inhibitor</span></div><div class="casAuthors">Chung, Suhman; Parker, Jared B.; Bianchet, Mario; Amzel, L. Mario; Stivers, James T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">407-413</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The linking together of mol. fragments that bind to adjacent sites on an enzyme can lead to high-affinity inhibitors.  Ideally, this strategy would use linkers that do not perturb the optimal binding geometries of the fragments and do not have excessive conformational flexibility that would increase the entropic penalty of binding.  In reality, these aims are seldom realized owing to limitations in linker chem.  Here the energetic and structural effects of rigid and flexible linkers on the binding of a fragment-based inhibitor of human uracil DNA glycosylase were systematically explored.  Anal. of the free energies of binding in combination with co-crystal structures shows that the flexibility and strain of a given linker can have a substantial impact on binding affinity even when the binding fragments are optimally positioned.  Such effects are not apparent from inspection of structures and underscore the importance of linker optimization in fragment-based drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo22ydPLyrVsrVg90H21EOLACvtfcHk0lhNwr_7npHtNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltValtLw%253D&md5=5fd9de328a39f6972d7ba376b5354851</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.163%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DS.%26aulast%3DParker%26aufirst%3DJ.%2BB.%26aulast%3DBianchet%26aufirst%3DM.%26aulast%3DAmzel%26aufirst%3DL.%2BM.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26atitle%3DImpact%2520of%2520linker%2520strain%2520and%2520flexibility%2520in%2520the%2520design%2520of%2520a%2520fragment-based%2520inhibitor%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D407%26epage%3D413%26doi%3D10.1038%2Fnchembio.163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Böhm, H. J.</span><span> </span><span class="NLM_article-title">LUDI: rule-based automatic design of new substituents for enzyme inhibitor leads</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">606</span><span class="refDoi"> DOI: 10.1007/BF00126217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1007%2FBF00126217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1291628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADyaK3s7nvF2jtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1992&pages=593-606&author=H.+J.+B%C3%B6hm&title=LUDI%3A+rule-based+automatic+design+of+new+substituents+for+enzyme+inhibitor+leads&doi=10.1007%2FBF00126217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">LUDI: rule-based automatic design of new substituents for enzyme inhibitor leads</span></div><div class="casAuthors">Bohm H J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">593-606</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">Recent advances in a new method for the de novo design of enzyme inhibitors are reported.  A new set of rules to define the possible nonbonded contacts between protein and ligand is presented.  This method was derived from published statistical analyses of nonbonded contacts in crystal packings of organic molecules and has been implemented in the recently described computer program LUDI.  Moreover, LUDI can now append a new substituent onto an already existing ligand.  Applications are reported for the design of inhibitors of HIV protease and dihydrofolate reductase.  The results demonstrate that LUDI is indeed capable of designing new ligands with improved binding when compared to the reference compound.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSuJ_iBRXZ5tz0O18WR3iAofW6udTcc2eZyBaK_OLLFYrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3s7nvF2jtg%253D%253D&md5=56422267240eb98f29e7314fdadd5bed</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2FBF00126217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00126217%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.%2BJ.%26atitle%3DLUDI%253A%2520rule-based%2520automatic%2520design%2520of%2520new%2520substituents%2520for%2520enzyme%2520inhibitor%2520leads%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1992%26volume%3D6%26spage%3D593%26epage%3D606%26doi%3D10.1007%2FBF00126217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myatt, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zsoldos, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, A. P.</span><span> </span><span class="NLM_article-title">SPROUT, HIPPO and CAESA: Tools for de novo structure generation and estimation of synthetic accessibility</span> <span class="citation_source-journal">Perspect. Drug Discovery Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1007/BF02174466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1007%2FBF02174466" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1995&pages=34-50&author=V.+J.+Gilletauthor=G.+Myattauthor=Z.+Zsoldosauthor=A.+P.+Johnson&title=SPROUT%2C+HIPPO+and+CAESA%3A+Tools+for+de+novo+structure+generation+and+estimation+of+synthetic+accessibility&doi=10.1007%2FBF02174466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2FBF02174466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02174466%26sid%3Dliteratum%253Aachs%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DMyatt%26aufirst%3DG.%26aulast%3DZsoldos%26aufirst%3DZ.%26aulast%3DJohnson%26aufirst%3DA.%2BP.%26atitle%3DSPROUT%252C%2520HIPPO%2520and%2520CAESA%253A%2520Tools%2520for%2520de%2520novo%2520structure%2520generation%2520and%2520estimation%2520of%2520synthetic%2520accessibility%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D1995%26volume%3D3%26spage%3D34%26epage%3D50%26doi%3D10.1007%2FBF02174466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Pearlman, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">CONCERTS: Dynamic connection of fragments as an approach to de novo ligand design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1651</span><span class="NLM_x">–</span> <span class="NLM_lpage">1663</span><span class="refDoi"> DOI: 10.1021/jm950792l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm950792l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK28XhslGqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=1651-1663&author=D.+A.+Pearlmanauthor=M.+A.+Murcko&title=CONCERTS%3A+Dynamic+connection+of+fragments+as+an+approach+to+de+novo+ligand+design&doi=10.1021%2Fjm950792l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">CONCERTS: Dynamic Connection of Fragments as an Approach to de Novo Ligand Design</span></div><div class="casAuthors">Pearlman, David A.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1651-63</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have implemented and tested a new approach to de novo ligand design, CONCERTS (creation of novel compds. by evaluation of residues at target sites).  In this method, each member of a user-defined set of fragments is allowed to move independently about a target active site during a mol. dynamics simulation.  This allows the fragments to sample various low-energy orientations.  When the geometry between proximal fragments is appropriate, bonds can be formed between the fragments.  In this fashion, larger mols. can be built.  The bonding arrangement can subsequently be changed-breaking bonds between chosen fragment pairs and forming them between other pairs-if the overall process creates lower energy mols.  We have tested this method with various mixes of fragments against the active sites of the FK506 binding protein (FKBP-12) and HIV-1 aspartyl protease.  In several cases, CONCERTS suggests ligands which are in surprisingly good agreement with known inhibitors of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzo_xUabLvOrVg90H21EOLACvtfcHk0lhBWg3YzTAx6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhslGqsr4%253D&md5=b7e36d919e4ebf403014a4d6ebf5cce9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm950792l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm950792l%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DCONCERTS%253A%2520Dynamic%2520connection%2520of%2520fragments%2520as%2520an%2520approach%2520to%2520de%2520novo%2520ligand%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D1651%26epage%3D1663%26doi%3D10.1021%2Fjm950792l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Maass, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Recore: A fast and versatile method for scaffold hopping based on small molecule crystal structure conformations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">390</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span><span class="refDoi"> DOI: 10.1021/ci060094h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci060094h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=390-399&author=P.+Maassauthor=T.+Schulz-Gaschauthor=M.+Stahlauthor=M.+Rarey&title=Recore%3A+A+fast+and+versatile+method+for+scaffold+hopping+based+on+small+molecule+crystal+structure+conformations&doi=10.1021%2Fci060094h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recore: A Fast and Versatile Method for Scaffold Hopping Based on Small Molecule Crystal Structure Conformations</span></div><div class="casAuthors">Maass, Patrick; Schulz-Gasch, Tanja; Stahl, Martin; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">390-399</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Replacing central elements of known active structures is a common procedure to enter new compd. classes.  Different computational methods have already been developed to help with this task, varying in the description of possible replacements, the query input, and the similarity measure used.  In this paper, a novel approach for scaffold replacement and a corresponding software tool, called Recore, is introduced.  In contrast to prior methods, the authors main objective was to combine the following three properties in one tool: to avoid structures with strained conformations, to enable the exploration of large search spaces, and to allow interactive use through short response times.  The authors introduce a new technique employing 3D fragments generated by combinatorial enumeration of cuts.  It allows focusing on fragments suitable for scaffold replacement while retaining conformational information of the corresponding crystal structures.  Based on this idea, the authors present an algorithm utilizing a geometric rank searching approach.  Given a geometric arrangement of two or three exit vectors and addnl. pharmacophore features, the algorithm finds fragments fulfilling all these constraints ordered by increasing deviation from the query constraints.  For the validation of the approach, three different drug design scenarios have been used.  The results obtained show that the authors approach is able to propose new valid scaffold topologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpphxap_w-jZbVg90H21EOLACvtfcHk0lhBWg3YzTAx6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D&md5=e9d4e277da49f4ffe2c21472990c2387</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci060094h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci060094h%26sid%3Dliteratum%253Aachs%26aulast%3DMaass%26aufirst%3DP.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DRecore%253A%2520A%2520fast%2520and%2520versatile%2520method%2520for%2520scaffold%2520hopping%2520based%2520on%2520small%2520molecule%2520crystal%2520structure%2520conformations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D390%26epage%3D399%26doi%3D10.1021%2Fci060094h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Lauri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartlett, P. A.</span><span> </span><span class="NLM_article-title">CAVEAT: A program to facilitate the design of organic molecules</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.1007/BF00124349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1007%2FBF00124349" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=51-66&author=G.+Lauriauthor=P.+A.+Bartlett&title=CAVEAT%3A+A+program+to+facilitate+the+design+of+organic+molecules&doi=10.1007%2FBF00124349"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2FBF00124349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124349%26sid%3Dliteratum%253Aachs%26aulast%3DLauri%26aufirst%3DG.%26aulast%3DBartlett%26aufirst%3DP.%2BA.%26atitle%3DCAVEAT%253A%2520A%2520program%2520to%2520facilitate%2520the%2520design%2520of%2520organic%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D51%26epage%3D66%26doi%3D10.1007%2FBF00124349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">BREED: Generating novel inhibitors through hybridization of known ligands. Application to CDK2, P38, and HIV protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2768</span><span class="NLM_x">–</span> <span class="NLM_lpage">2775</span><span class="refDoi"> DOI: 10.1021/jm030543u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030543u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2768-2775&author=A.+C.+Pierceauthor=G.+Raoauthor=G.+W.+Bemis&title=BREED%3A+Generating+novel+inhibitors+through+hybridization+of+known+ligands.+Application+to+CDK2%2C+P38%2C+and+HIV+protease&doi=10.1021%2Fjm030543u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">BREED: Generating Novel Inhibitors through Hybridization of Known Ligands. Application to CDK2, P38, and HIV Protease</span></div><div class="casAuthors">Pierce, Albert C.; Rao, Govinda; Bemis, Guy W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2768-2775</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this work we describe BREED, a method for the generation of novel inhibitors from structures of known ligands bound to a common target.  The method is essentially an automation of the common medicinal chem. practice of joining fragments of two known ligands to generate a new inhibitor.  The ligand-bound target structures are overlaid, all overlapping bonds in all pairs of ligands are found, and the fragments on each side of each matching bond are swapped to generate the new mols.  Since the method is automated, it can be applied recursively to generate all possible combinations of known ligands.  In an application of this method to HIV protease inhibitors and protein kinase inhibitors, hundreds of new mol. structures were generated.  These included known inhibitor scaffolds not included in the initial set, entirely novel scaffolds, and novel substituents on known scaffolds.  The method is fast, and since all of the ligand functional groups are known to bind the target in the precise position and orientation present in the novel ligand, the success rate of this method should be superior to more traditional de novo design techniques.  In an era of increasingly high-throughput structural biol., such methods for high-throughput utilization of structural information will become increasingly valuable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqskNA5h73CbbVg90H21EOLACvtfcHk0lhEWbRqA3ZBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D&md5=e0bec9082fab05b694742498b3f41549</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm030543u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030543u%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DRao%26aufirst%3DG.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DBREED%253A%2520Generating%2520novel%2520inhibitors%2520through%2520hybridization%2520of%2520known%2520ligands.%2520Application%2520to%2520CDK2%252C%2520P38%252C%2520and%2520HIV%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2768%26epage%3D2775%26doi%3D10.1021%2Fjm030543u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Dey, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Fragment-based de novo ligand design by multi-objective evolutionary optimization</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1021/ci700424b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700424b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFWltbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=679-690&author=F.+Deyauthor=A.+Caflisch&title=Fragment-based+de+novo+ligand+design+by+multi-objective+evolutionary+optimization&doi=10.1021%2Fci700424b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based de novo ligand design by multiobjective evolutionary optimization</span></div><div class="casAuthors">Dey, Fabian; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">679-690</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GANDI (Genetic Algorithm-based de Novo Design of Inhibitors) is a computational tool for automatic fragment-based design of mols. within a protein binding site of known structure.  A genetic algorithm and a tabu search act in concert to join predocked fragments with a user-supplied list of fragments.  A novel feature of GANDI is the simultaneous optimization of force field energy and a term enforcing 2D-similarity to known inhibitor(s) or 3D-overlap to known binding mode(s).  Scaffold hopping can be promoted by tuning the relative wts. of these terms.  The performance of GANDI is tested on cyclin-dependent kinase 2 (CDK2) using a library of about 14 000 fragments and the binding mode of a known oxindole inhibitor to bias the design.  Top ranking GANDI mols. are involved in one to three hydrogen bonds with the backbone polar groups in the hinge region of CDK2, an interaction pattern obsd. in potent kinase inhibitors.  Notably, a GANDI mol. with very favorable predicted binding affinity shares a 2-N-phenyl-1,3-thiazole-2,4-diamine moiety with a known nanomolar inhibitor of CDK2.  Importantly, mols. with a favorable GANDI score are synthetic accessible.  In fact, eight of the 1809 mols. designed by GANDI for CDK2 are found in the ZINC database of com. available compds. which also contains about 600 compds. with identical scaffolds as those in the top ranking GANDI mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrprXa_ftoRxrVg90H21EOLACvtfcHk0lhEWbRqA3ZBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFWltbc%253D&md5=fed0972e4a1c2aedcb2aadeae58c1d84</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci700424b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700424b%26sid%3Dliteratum%253Aachs%26aulast%3DDey%26aufirst%3DF.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DFragment-based%2520de%2520novo%2520ligand%2520design%2520by%2520multi-objective%2520evolutionary%2520optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D679%26epage%3D690%26doi%3D10.1021%2Fci700424b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span>BROOD, version <span class="NLM_edition">1.6.3.1</span>; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span> (accessed Nov 15, 2014).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BROOD%2C+version+1.6.3.1%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM+%28accessed+Nov+15%2C+2014%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DBROOD%26pub%3DOpenEye%2520Scientific%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Pirard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span> </span><span class="NLM_article-title">Evaluation of a semi-automated workflow for fragment growing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1021/ci5006355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5006355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=180-193&author=B.+Pirardauthor=P.+Ertl&title=Evaluation+of+a+semi-automated+workflow+for+fragment+growing&doi=10.1021%2Fci5006355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a Semi-Automated Workflow for Fragment Growing</span></div><div class="casAuthors">Pirard, Bernard; Ertl, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">180-193</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intelligent Automatic Design (IADE) is an expert system developed at Novartis to identify nonclassical bioisosteres.  In addn. to bioisostere searching, one could also use IADE to grow a fragment bound to a protein.  Here we report an evaluation of IADE as a tool for fragment growing.  Three examples from the literature served as test cases.  In all three cases, IADE generated close analogs of the published compds. and reproduced their crystallog. binding modes.  This exercise validated the use of the IADE system for fragment growing.  We have also gained experience in optimizing the performance of IADE for this type of application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu3eXbqlbVxbVg90H21EOLACvtfcHk0lgFXT7_hyVccg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7zJ&md5=607fc092518e799ba097242978cd31c5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci5006355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5006355%26sid%3Dliteratum%253Aachs%26aulast%3DPirard%26aufirst%3DB.%26aulast%3DErtl%26aufirst%3DP.%26atitle%3DEvaluation%2520of%2520a%2520semi-automated%2520workflow%2520for%2520fragment%2520growing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D180%26epage%3D193%26doi%3D10.1021%2Fci5006355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Degen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wegscheid-Gerlach, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaliani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">On the art of compiling and using ’drug-like’ chemical fragment spaces</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1503</span><span class="NLM_x">–</span> <span class="NLM_lpage">1507</span><span class="refDoi"> DOI: 10.1002/cmdc.200800178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fcmdc.200800178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=18792903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1503-1507&author=J.+Degenauthor=C.+Wegscheid-Gerlachauthor=A.+Zalianiauthor=M.+Rarey&title=On+the+art+of+compiling+and+using+%E2%80%99drug-like%E2%80%99+chemical+fragment+spaces&doi=10.1002%2Fcmdc.200800178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">On the art of compiling and using 'drug-like' chemical fragment spaces</span></div><div class="casAuthors">Degen, Joerg; Wegscheid-Gerlach, Christof; Zaliani, Andrea; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1503-1507</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">To improve existing approaches for the automatic decompn. of mols. into fragments, the authors compiled a new and more elaborate set of rules for the breaking of retrosynthetically interesting chem. substructures (BRICS) and used this for obtaining fragments from biol. active compds. and vendor catalog sources.  Based on this, the authors compiled corresponding fragment spaces by specifying a complementary set of rules for the recombination of the corresponding chem. motifs.  Furthermore, the authors put considerable effort into compiling a set of high-quality, high-performance, and in contrast to all other approaches, publicly available fragments that are meant to serve as a possible basis for various mol. design objectives and techniques.  Overall, this work led the authors to more comprehensive sets of fragments, and the corresponding fragment spaces show a significant increase in performance over existing methods.  Moreover, by incorporating fragments from vendor catalog sources, the performance can be increased even further.  The BRICS model consists of 16 chem. environments indicated by link atoms of different types.  A shredding example for nexavar (Sorafenib), showing the key steps in the generation of the fragments is shown as well as fragment prototypes used in BRICS.  A case study showing the best re-ranked solns. obtained with the four fragment spaces for the nexavar (Sorafenib) query is also shown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaVkm_-Qj5PrVg90H21EOLACvtfcHk0lgFXT7_hyVccg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt73M&md5=a927330dd3888352cc1528b1e3c6316d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800178%26sid%3Dliteratum%253Aachs%26aulast%3DDegen%26aufirst%3DJ.%26aulast%3DWegscheid-Gerlach%26aufirst%3DC.%26aulast%3DZaliani%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DOn%2520the%2520art%2520of%2520compiling%2520and%2520using%2520%25E2%2580%2599drug-like%25E2%2580%2599%2520chemical%2520fragment%2520spaces%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1503%26epage%3D1507%26doi%3D10.1002%2Fcmdc.200800178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Lewell, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span> </span><span class="NLM_article-title">RECAP retrosynthetic combinatorial analysis procedure: A powerful new technique for identifying privileged molecular fragments with useful applications in combinatorial chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1021/ci970429i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci970429i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=511-522&author=X.+Q.+Lewellauthor=D.+B.+Juddauthor=S.+P.+Watsonauthor=M.+M.+Hann&title=RECAP+retrosynthetic+combinatorial+analysis+procedure%3A+A+powerful+new+technique+for+identifying+privileged+molecular+fragments+with+useful+applications+in+combinatorial+chemistry&doi=10.1021%2Fci970429i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">RECAP-Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span></div><div class="casAuthors">Lewell, Xiao Qing; Judd, Duncan; Watson, Steve; Hann, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">511-522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of combinatorial chem. for the generation of new lead mols. is now a well established strategy in the drug discovery process.  Central to the use of combinatorial chem. is the design and availability of high quality building blocks which are likely to afford hits from the libraries that they generate.  Herein the authors describe "RECAP" (Retrosynthetic Combinatorial Anal. Procedure), a new computational technique designed to address this building block issue.  RECAP electronically fragments mols. based on chem. knowledge.  When applied to databases of biol. active mols., this allows the identification of building block fragments rich in biol. recognized elements and privileged motifs and structures.  This allows the design of building blocks and the synthesis of libraries rich in biol. motifs.  Application of RECAP to the Derwent World Drug Index (WDI) and the mol. fragments/building blocks that this generates are discussed.  The authors also describe a WDI fragment knowledge base which the authors have built which stores the drug motifs and mention its potential application in structure based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovNOpJj5LVU7Vg90H21EOLACvtfcHk0lhsWQuBvUm8wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D&md5=7445f76f162006c687116e1e539da8be</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci970429i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci970429i%26sid%3Dliteratum%253Aachs%26aulast%3DLewell%26aufirst%3DX.%2BQ.%26aulast%3DJudd%26aufirst%3DD.%2BB.%26aulast%3DWatson%26aufirst%3DS.%2BP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26atitle%3DRECAP%2520retrosynthetic%2520combinatorial%2520analysis%2520procedure%253A%2520A%2520powerful%2520new%2520technique%2520for%2520identifying%2520privileged%2520molecular%2520fragments%2520with%2520useful%2520applications%2520in%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D511%26epage%3D522%26doi%3D10.1021%2Fci970429i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span> </span><span class="NLM_article-title">Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">8</span><span class="refDoi"> DOI: 10.1186/1758-2946-1-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1186%2F1758-2946-1-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=8&author=P.+Ertlauthor=A.+Schuffenhauer&title=Estimation+of+synthetic+accessibility+score+of+drug-like+molecules+based+on+molecular+complexity+and+fragment+contributions&doi=10.1186%2F1758-2946-1-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions</span></div><div class="casAuthors">Ertl Peter; Schuffenhauer Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A method to estimate ease of synthesis (synthetic accessibility) of drug-like molecules is needed in many areas of the drug discovery process.  The development and validation of such a method that is able to characterize molecule synthetic accessibility as a score between 1 (easy to make) and 10 (very difficult to make) is described in this article.  RESULTS:  The method for estimation of the synthetic accessibility score (SAscore) described here is based on a combination of fragment contributions and a complexity penalty.  Fragment contributions have been calculated based on the analysis of one million representative molecules from PubChem and therefore one can say that they capture historical synthetic knowledge stored in this database.  The molecular complexity score takes into account the presence of non-standard structural features, such as large rings, non-standard ring fusions, stereocomplexity and molecule size.  The method has been validated by comparing calculated SAscores with ease of synthesis as estimated by experienced medicinal chemists for a set of 40 molecules.  The agreement between calculated and manually estimated synthetic accessibility is very good with r2 = 0.89.  CONCLUSION:  A novel method to estimate synthetic accessibility of molecules has been developed.  This method uses historical synthetic knowledge obtained by analyzing information from millions of already synthesized chemicals and considers also molecule complexity.  The method is sufficiently fast and provides results consistent with estimation of ease of synthesis by experienced medicinal chemists.  The calculated SAscore may be used to support various drug discovery processes where a large number of molecules needs to be ranked based on their synthetic accessibility, for example when purchasing samples for screening, selecting hits from high-throughput screening for follow-up, or ranking molecules generated by various de novo design approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQW_MJ6K-6QK9K0qsdY6Nc1fW6udTcc2eZ7aVhjjerE1rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D&md5=9ffc064a9d3952a583532411cb0f0673</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-8%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DEstimation%2520of%2520synthetic%2520accessibility%2520score%2520of%2520drug-like%2520molecules%2520based%2520on%2520molecular%2520complexity%2520and%2520fragment%2520contributions%26jtitle%3DJ.%2520Cheminf.%26date%3D2009%26volume%3D1%26spage%3D8%26doi%3D10.1186%2F1758-2946-1-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Hartenfeller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieto-Oberhuber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altmann, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span> </span><span class="NLM_article-title">A collection of robust organic synthesis reactions for in silico molecule design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3093</span><span class="NLM_x">–</span> <span class="NLM_lpage">3098</span><span class="refDoi"> DOI: 10.1021/ci200379p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200379p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCksLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=3093-3098&author=M.+Hartenfellerauthor=M.+Eberleauthor=P.+Meierauthor=C.+Nieto-Oberhuberauthor=K.-H.+Altmannauthor=G.+Schneiderauthor=E.+Jacobyauthor=S.+Renner&title=A+collection+of+robust+organic+synthesis+reactions+for+in+silico+molecule+design&doi=10.1021%2Fci200379p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A Collection of Robust Organic Synthesis Reactions for In Silico Molecule Design</span></div><div class="casAuthors">Hartenfeller, Markus; Eberle, Martin; Meier, Peter; Nieto-Oberhuber, Cristina; Altmann, Karl-Heinz; Schneider, Gisbert; Jacoby, Edgar; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3093-3098</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused collection of org. synthesis reactions for computer-based mol. construction is presented.  It is inspired by real-world chem. and has been compiled in close collaboration with medicinal chemists to achieve high practical relevance.  Virtual mols. assembled from existing starting material connected by these reactions are supposed to have an enhanced chance to be amenable to real chem. synthesis.  About 50% of the reactions in the dataset are ring-forming reactions, which fosters the assembly of novel ring systems and innovative chemotypes.  A comparison with a recent survey of the reactions used in early drug discovery revealed considerable overlaps with the collection presented here.  The dataset is available encoded as computer-readable Reaction SMARTS expressions from the Supporting Information presented for this paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhm-KV-_-dR7Vg90H21EOLACvtfcHk0li_oqE2eLJEGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCksLjP&md5=848e13095bc94dfb0ea96a5fe85463ec</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci200379p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200379p%26sid%3Dliteratum%253Aachs%26aulast%3DHartenfeller%26aufirst%3DM.%26aulast%3DEberle%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DP.%26aulast%3DNieto-Oberhuber%26aufirst%3DC.%26aulast%3DAltmann%26aufirst%3DK.-H.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DA%2520collection%2520of%2520robust%2520organic%2520synthesis%2520reactions%2520for%2520in%2520silico%2520molecule%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D3093%26epage%3D3098%26doi%3D10.1021%2Fci200379p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Hartenfeller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieto-Oberhuber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altmann, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span> </span><span class="NLM_article-title">Probing the bioactivity-relevant chemical space of robust reactions and common molecular building blocks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1167</span><span class="NLM_x">–</span> <span class="NLM_lpage">1178</span><span class="refDoi"> DOI: 10.1021/ci200618n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200618n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1167-1178&author=M.+Hartenfellerauthor=M.+Eberleauthor=P.+Meierauthor=C.+Nieto-Oberhuberauthor=K.+H.+Altmannauthor=G.+Schneiderauthor=E.+Jacobyauthor=S.+Renner&title=Probing+the+bioactivity-relevant+chemical+space+of+robust+reactions+and+common+molecular+building+blocks&doi=10.1021%2Fci200618n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Bioactivity-Relevant Chemical Space of Robust Reactions and Common Molecular Building Blocks</span></div><div class="casAuthors">Hartenfeller, Markus; Eberle, Martin; Meier, Peter; Nieto-Oberhuber, Cristina; Altmann, Karl-Heinz; Schneider, Gisbert; Jacoby, Edgar; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1167-1178</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the search for new bioactive compds., there is a trend toward increasingly complex compd. libraries aiming to target the demanding targets of the future.  In contrast, medicinal chem. and traditional library design rely mainly on a small set of highly established and robust reactions.  Here, we probe a set of 58 such reactions for their ability to sample the chem. space of known bioactive mols., and the potential to create new scaffolds.  Combined with ∼26 000 common available building blocks, the reactions retrieve around 9% of a scaffold-diverse set of compds. active on human target proteins covering all major pharmaceutical target classes.  Almost 80% of generated scaffolds from virtual one-step synthesis products are not present in a large set of known bioactive mols. for human targets, indicating potential for new discoveries.  The results suggest that established synthesis resources are well suited to cover the known bioactivity-relevant chem. space and that there are plenty of unexplored regions accessible by these reactions, possibly providing valuable "low-hanging fruit" for hit discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7dpgccjqzh7Vg90H21EOLACvtfcHk0li_oqE2eLJEGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOksLk%253D&md5=4505f35fa529e1f94a495aec2f6faff2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci200618n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200618n%26sid%3Dliteratum%253Aachs%26aulast%3DHartenfeller%26aufirst%3DM.%26aulast%3DEberle%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DP.%26aulast%3DNieto-Oberhuber%26aufirst%3DC.%26aulast%3DAltmann%26aufirst%3DK.%2BH.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DProbing%2520the%2520bioactivity-relevant%2520chemical%2520space%2520of%2520robust%2520reactions%2520and%2520common%2520molecular%2520building%2520blocks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1167%26epage%3D1178%26doi%3D10.1021%2Fci200618n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Roughley, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">The medicinal chemist’s toolbox: An analysis of reactions used in the pursuit of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3451</span><span class="NLM_x">–</span> <span class="NLM_lpage">3479</span><span class="refDoi"> DOI: 10.1021/jm200187y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200187y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3451-3479&author=S.+D.+Roughleyauthor=A.+M.+Jordan&title=The+medicinal+chemist%E2%80%99s+toolbox%3A+An+analysis+of+reactions+used+in+the+pursuit+of+drug+candidates&doi=10.1021%2Fjm200187y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The Medicinal Chemist's Toolbox: An Analysis of Reactions Used in the Pursuit of Drug Candidates</span></div><div class="casAuthors">Roughley, Stephen D.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3451-3479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A survey of selected medicinal chem. literature from 2008 was undertaken in order to det. the most common chem. reactions used in the prepn. of drug candidates.  In addn. to an examn. of the reactions, the frequency of occurrence of various functional groups in the drug targets was analyzed.  The Lipinski parameters and chirality of the surveyed compds. were also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTUrcluTkPBLVg90H21EOLACvtfcHk0li_oqE2eLJEGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D&md5=a94df26d82bbd58b269ed20aa3253f56</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm200187y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200187y%26sid%3Dliteratum%253Aachs%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520medicinal%2520chemist%25E2%2580%2599s%2520toolbox%253A%2520An%2520analysis%2520of%2520reactions%2520used%2520in%2520the%2520pursuit%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3451%26epage%3D3479%26doi%3D10.1021%2Fjm200187y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of β<sub>2</sub>-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span><span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+%CE%B22-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0lgw3f26m6KYKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520%25CE%25B22-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Ballesteros, J. A.; Weinstein, H.</span><span> </span><span class="NLM_article-title">Integrated Methods for the Construction of Three-dimensional Models and Computational Probing of Structure-Function Relations in G Protein-Coupled Receptors</span>. In  <span class="citation_source-book">Receptor Molecular Biology</span>; <span class="NLM_contrib-group">Sealfon, S. C.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">1995</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">25</span>, pp  <span class="NLM_fpage">366</span><span class="NLM_x">–</span> <span class="NLM_lpage">428</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&pages=366-428&author=J.+A.+Ballesteros&author=H.+Weinsteinauthor=S.+C.+Sealfon&title=Receptor+Molecular+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26atitle%3DIntegrated%2520Methods%2520for%2520the%2520Construction%2520of%2520Three-dimensional%2520Models%2520and%2520Computational%2520Probing%2520of%2520Structure-Function%2520Relations%2520in%2520G%2520Protein-Coupled%2520Receptors%26btitle%3DReceptor%2520Molecular%2520Biology%26aulast%3DSealfon%26aufirst%3DS.%2BC.%26pub%3DAcademic%2520Press%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casarosa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeVree, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnapp, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konetzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunahara, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pautsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyaert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Structure of a nanobody-stabilized active state of the beta(2) adrenoceptor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">469</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">180</span><span class="refDoi"> DOI: 10.1038/nature09648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature09648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=21228869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=175-180&author=S.+G.+F.+Rasmussenauthor=H.+Choiauthor=J.+Fungauthor=E.+Pardonauthor=P.+Casarosaauthor=P.+S.+Chaeauthor=B.+T.+DeVreeauthor=D.+M.+Rosenbaumauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=A.+Schnappauthor=I.+Konetzkiauthor=R.+K.+Sunaharaauthor=S.+H.+Gellmanauthor=A.+Pautschauthor=J.+Steyaertauthor=W.+I.+Weisauthor=B.+K.+Kobilka&title=Structure+of+a+nanobody-stabilized+active+state+of+the+beta%282%29+adrenoceptor&doi=10.1038%2Fnature09648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a nanobody-stabilized active state of the β2 adrenoceptor</span></div><div class="casAuthors">Rasmussen, Soren G. F.; Choi, Hee-Jung; Fung, Juan Jose; Pardon, Els; Casarosa, Paola; Chae, Pil Seok; DeVree, Brian T.; Rosenbaum, Daniel M.; Thian, Foon Sun; Kobilka, Tong Sun; Schnapp, Andreas; Konetzki, Ingo; Sunahara, Roger K.; Gellman, Samuel H.; Pautsch, Alexander; Steyaert, Jan; Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7329</span>),
    <span class="NLM_cas:pages">175-180</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein coupled receptors (GPCRs) exhibit a spectrum of functional behaviors in response to natural and synthetic ligands.  Recent crystal structures provide insights into inactive states of several GPCRs.  Efforts to obtain an agonist-bound active-state GPCR structure have proven difficult due to the inherent instability of this state in the absence of a G protein.  We generated a camelid antibody fragment (nanobody) to the human β2 adrenergic receptor (β2AR) that exhibits G protein-like behavior, and obtained an agonist-bound, active-state crystal structure of the receptor-nanobody complex.  Comparison with the inactive β2AR structure reveals subtle changes in the binding pocket; however, these small changes are assocd. with an 11 Å outward movement of the cytoplasmic end of transmembrane segment 6, and rearrangements of transmembrane segments 5 and 7 that are remarkably similar to those obsd. in opsin, an active form of rhodopsin.  This structure provides insights into the process of agonist binding and activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqITZxHz6Vi77Vg90H21EOLACvtfcHk0lgKtM0sZ0YgWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D&md5=3a1d3bac6d92c9d54cf1ddd14d56ab8c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature09648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09648%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DFung%26aufirst%3DJ.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCasarosa%26aufirst%3DP.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DSchnapp%26aufirst%3DA.%26aulast%3DKonetzki%26aufirst%3DI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DPautsch%26aufirst%3DA.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520of%2520a%2520nanobody-stabilized%2520active%2520state%2520of%2520the%2520beta%25282%2529%2520adrenoceptor%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D175%26epage%3D180%26doi%3D10.1038%2Fnature09648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeVree, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calinski, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathiesen, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyons, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyaert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skiniotis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunahara, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Crystal structure of the beta(2) adrenergic receptor-Gs protein complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">477</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">U311</span><span class="refDoi"> DOI: 10.1038/nature10361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature10361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=21772288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2011&pages=549-U311&author=S.+G.+F.+Rasmussenauthor=B.+T.+DeVreeauthor=Y.+Zouauthor=A.+C.+Kruseauthor=K.+Y.+Chungauthor=T.+S.+Kobilkaauthor=F.+S.+Thianauthor=P.+S.+Chaeauthor=E.+Pardonauthor=D.+Calinskiauthor=J.+M.+Mathiesenauthor=S.+T.+A.+Shahauthor=J.+A.+Lyonsauthor=M.+Caffreyauthor=S.+H.+Gellmanauthor=J.+Steyaertauthor=G.+Skiniotisauthor=W.+I.+Weisauthor=R.+K.+Sunaharaauthor=B.+K.+Kobilka&title=Crystal+structure+of+the+beta%282%29+adrenergic+receptor-Gs+protein+complex&doi=10.1038%2Fnature10361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the β2 adrenergic receptor-Gs protein complex</span></div><div class="casAuthors">Rasmussen, Soren G. F.; DeVree, Brian T.; Zou, Yao-Zhong; Kruse, Andrew C.; Chung, Ka-Young; Kobilka, Tong-Sun; Thian, Foon-Sun; Chae, Pil-Seok; Pardon, Els; Calinski, Diane; Mathiesen, Jesper M.; Shah, Syed T. A.; Lyons, Joseph A.; Caffrey, Martin; Gellman, Samuel H.; Steyaert, Jan; Skiniotis, Georgios; Weis, William I.; Sunahara, Roger K.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">7366</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are responsible for the majority of cellular responses to hormones and neurotransmitters as well as the senses of sight, olfaction and taste.  The paradigm of GPCR signalling is the activation of a heterotrimeric GTP binding protein (G protein) by an agonist-occupied receptor.  The β2 adrenergic receptor (β2AR) activation of Gs, the stimulatory G protein for adenylyl cyclase, has long been a model system for GPCR signalling.  Here we present the crystal structure of the active state ternary complex composed of agonist-occupied monomeric β2AR and nucleotide-free Gs heterotrimer.  The principal interactions between the β2AR and Gs involve the amino- and carboxy-terminal α-helixes of Gs, with conformational changes propagating to the nucleotide-binding pocket.  The largest conformational changes in the β2AR include a 14 Å outward movement at the cytoplasmic end of transmembrane segment 6 (TM6) and an α-helical extension of the cytoplasmic end of TM5.  The most surprising observation is a major displacement of the α-helical domain of Gαs relative to the Ras-like GTPase domain.  This crystal structure represents the first high-resoln. view of transmembrane signalling by a GPCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-PdYRDvolqrVg90H21EOLACvtfcHk0lgKtM0sZ0YgWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL&md5=d22a43dd677ac255d138b1aedff357d3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature10361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10361%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DChung%26aufirst%3DK.%2BY.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCalinski%26aufirst%3DD.%26aulast%3DMathiesen%26aufirst%3DJ.%2BM.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DLyons%26aufirst%3DJ.%2BA.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DSkiniotis%26aufirst%3DG.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCrystal%2520structure%2520of%2520the%2520beta%25282%2529%2520adrenergic%2520receptor-Gs%2520protein%2520complex%26jtitle%3DNature%26date%3D2011%26volume%3D477%26spage%3D549%26epage%3DU311%26doi%3D10.1038%2Fnature10361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyons, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holl, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aragao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlow, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeVree, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunahara, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmeiner, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Structure and function of an irreversible agonist-β2 adrenoceptor complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">469</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="NLM_x">–</span> <span class="NLM_lpage">240</span><span class="refDoi"> DOI: 10.1038/nature09665</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature09665" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=21228876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFehsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=236-240&author=D.+M.+Rosenbaumauthor=C.+Zhangauthor=J.+A.+Lyonsauthor=R.+Hollauthor=D.+Aragaoauthor=D.+H.+Arlowauthor=S.+G.+F.+Rasmussenauthor=H.+Choiauthor=B.+T.+DeVreeauthor=R.+K.+Sunaharaauthor=P.+S.+Chaeauthor=S.+H.+Gellmanauthor=R.+O.+Drorauthor=D.+E.+Shawauthor=W.+I.+Weisauthor=M.+Caffreyauthor=P.+Gmeinerauthor=B.+K.+Kobilka&title=Structure+and+function+of+an+irreversible+agonist-%CE%B22+adrenoceptor+complex&doi=10.1038%2Fnature09665"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of an irreversible agonist-β2 adrenoceptor complex</span></div><div class="casAuthors">Rosenbaum, Daniel M.; Zhang, Cheng; Lyons, Joseph A.; Holl, Ralph; Aragao, David; Arlow, Daniel H.; Rasmussen, Soren G. F.; Choi, Hee-Jung; DeVree, Brian T.; Sunahara, Roger K.; Chae, Pil Seok; Gellman, Samuel H.; Dror, Ron O.; Shaw, David E.; Weis, William I.; Caffrey, Martin; Gmeiner, Peter; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7329</span>),
    <span class="NLM_cas:pages">236-240</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are eukaryotic integral membrane proteins that modulate biol. function by initiating cellular signaling in response to chem. diverse agonists.  Despite recent progress in the structural biol. of GPCRs, the mol. basis for agonist binding and allosteric modulation of these proteins is poorly understood.  Structural knowledge of agonist-bound states is essential for deciphering the mechanism of receptor activation, and for structure-guided design and optimization of ligands.  However, the crystn. of agonist-bound GPCRs has been hampered by modest affinities and rapid off-rates of available agonists.  Using the inactive structure of the human β2 adrenergic receptor (β2AR) as a guide, we designed a β2AR agonist that can be covalently tethered to a specific site on the receptor through a disulfide bond.  The covalent β2AR-agonist complex forms efficiently, and is capable of activating a heterotrimeric G protein.  We crystd. a covalent agonist-bound β2AR-T4L fusion protein in lipid bilayers through the use of the lipidic mesophase method, and detd. its structure at 3.5 Å resoln.  A comparison to the inactive structure and an antibody-stabilized active structure (companion paper) shows how binding events at both the extracellular and intracellular surfaces are required to stabilize an active conformation of the receptor.  The structures are in agreement with long-timescale (up to 30 μs) mol. dynamics simulations showing that an agonist-bound active conformation spontaneously relaxes to an inactive-like conformation in the absence of a G protein or stabilizing antibody.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3cxTxwHig_rVg90H21EOLACvtfcHk0lgeghKrcr8Lng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFehsQ%253D%253D&md5=8629b93e43fb692395e2aa6f8bb011a9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature09665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09665%26sid%3Dliteratum%253Aachs%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLyons%26aufirst%3DJ.%2BA.%26aulast%3DHoll%26aufirst%3DR.%26aulast%3DAragao%26aufirst%3DD.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520and%2520function%2520of%2520an%2520irreversible%2520agonist-%25CE%25B22%2520adrenoceptor%2520complex%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D236%26epage%3D240%26doi%3D10.1038%2Fnature09665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Ring, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manglik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enos, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1038/nature12572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature12572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=24056936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=575-9&author=A.+M.+Ringauthor=A.+Manglikauthor=A.+C.+Kruseauthor=M.+D.+Enosauthor=W.+I.+Weisauthor=K.+C.+Garciaauthor=B.+K.+Kobilka&title=Adrenaline-activated+structure+of+%CE%B22-adrenoceptor+stabilized+by+an+engineered+nanobody&doi=10.1038%2Fnature12572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span></div><div class="casAuthors">Ring, Aaron M.; Manglik, Aashish; Kruse, Andrew C.; Enos, Michael D.; Weis, William I.; Garcia, K. Christopher; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">575-579</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are integral membrane proteins that play an essential role in human physiol., yet the mol. processes through which they bind to their endogenous agonists and activate effector proteins remain poorly understood.  So far, it has not been possible to capture an active-state GPCR bound to its native neurotransmitter.  Crystal structures of agonist-bound GPCRs have relied on the use of either exceptionally high-affinity agonists or receptor stabilization by mutagenesis.  Many natural agonists such as adrenaline, which activates the β2-adrenoceptor (β2AR), bind with relatively low affinity, and they are often chem. unstable.  Here, using directed evolution, the authors engineered a high-affinity conformationally selective single-domain camelid antibody fragment (nanobody; Nb6B9) that stabilized the active state of human β2AR, and used this to obtain crystal structures of the activated receptor bound to multiple ligands.  The authors present structures of the active-state human β2AR bound to 3 chem. distinct agonists: (1) the ultrahigh-affinity agonist BI167107; (2) the high-affinity catecholamine agonist, hydroxybenzylisoproterenol; and (3) the low-affinity endogenous agonist, adrenaline.  The crystal structures revealed a highly conserved overall ligand recognition and activation mode despite diverse ligand chem. structures and affinities that ranged from 100 nM to ∼80 pM.  Overall, the adrenaline-bound receptor structure was similar to the others, but it had substantial rearrangements in extracellular loop 3 and the extracellular tip of transmembrane helix 6.  These structures also revealed a water-mediated H-bond between 2 conserved Tyr residues, which appeared to stabilize the active state of the β2AR and related GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAp0VskflwWbVg90H21EOLACvtfcHk0lgeghKrcr8Lng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP&md5=c5b844a0a86ee0a74843588a4a363592</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature12572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12572%26sid%3Dliteratum%253Aachs%26aulast%3DRing%26aufirst%3DA.%2BM.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DEnos%26aufirst%3DM.%2BD.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DGarcia%26aufirst%3DK.%2BC.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DAdrenaline-activated%2520structure%2520of%2520%25CE%25B22-adrenoceptor%2520stabilized%2520by%2520an%2520engineered%2520nanobody%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D575%26epage%3D9%26doi%3D10.1038%2Fnature12572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Weichert, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manglik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiller, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hübner, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmeiner, P.</span><span> </span><span class="NLM_article-title">Covalent agonists for studying G protein-coupled receptor activation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">10744</span><span class="NLM_x">–</span> <span class="NLM_lpage">10748</span><span class="refDoi"> DOI: 10.1073/pnas.1410415111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.1410415111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=25006259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyqtbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=10744-10748&author=D.+Weichertauthor=A.+C.+Kruseauthor=A.+Manglikauthor=C.+Hillerauthor=C.+Zhangauthor=H.+H%C3%BCbnerauthor=B.+K.+Kobilkaauthor=P.+Gmeiner&title=Covalent+agonists+for+studying+G+protein-coupled+receptor+activation&doi=10.1073%2Fpnas.1410415111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent agonists for studying G protein-coupled receptor activation</span></div><div class="casAuthors">Weichert, Dietmar; Kruse, Andrew C.; Manglik, Aashish; Hiller, Christine; Zhang, Cheng; Huebner, Harald; Kobilka, Brian K.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">10744-10748</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Structural studies on G protein-coupled receptors (GPCRs) provide important insights into the architecture and function of these important drug targets.  However, the crystn. of GPCRs in active states is particularly challenging, requiring the formation of stable and conformationally homogeneous ligand-receptor complexes.  Native hormones, neurotransmitters, and synthetic agonists that bind with low affinity are ineffective at stabilizing an active state for crystallogenesis.  To promote structural studies on the pharmacol. highly relevant class of aminergic GPCRs, the authors here present the development of covalently binding mol. tools activating Gs-, Gi-, and Gq-coupled receptors.  The covalent agonists are derived from the monoamine neurotransmitters noradrenaline, dopamine, serotonin, and histamine, and they were accessed using a general and versatile synthetic strategy.  The authors demonstrate that the tool compds. presented herein display an efficient covalent binding mode and that the resp. covalent ligand-receptor complexes activate G proteins comparable to the natural neurotransmitters.  A crystal structure of the β2-adrenoreceptor in complex with a covalent noradrenaline analog and a conformationally selective antibody (nanobody) verified that these agonists can be used to facilitate crystallogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqbJHK2UMj1rVg90H21EOLACvtfcHk0li017TkVW8bbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyqtbnN&md5=a131bb7711d9a1826c77c3b598dee470</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1410415111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1410415111%26sid%3Dliteratum%253Aachs%26aulast%3DWeichert%26aufirst%3DD.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DHiller%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DCovalent%2520agonists%2520for%2520studying%2520G%2520protein-coupled%2520receptor%2520activation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D10744%26epage%3D10748%26doi%3D10.1073%2Fpnas.1410415111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Pardon, E.; Betti, C.; Laeremans, T.; Chevillard, F.; Guillemyn, K.; Kolb, P.; Ballet, S.; Steyaert, J.</span><span> </span><span class="NLM_article-title">Nanobody-enabled reverse pharmacology on GPCRs</span>.  <span class="citation_source-journal">Angew. Chem. Int. Ed</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2018</span><span class="NLM_x">, </span>submitted.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=E.+Pardon&author=C.+Betti&author=T.+Laeremans&author=F.+Chevillard&author=K.+Guillemyn&author=P.+Kolb&author=S.+Ballet&author=J.+Steyaert&title=Nanobody-enabled+reverse+pharmacology+on+GPCRs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPardon%26aufirst%3DE.%26atitle%3DNanobody-enabled%2520reverse%2520pharmacology%2520on%2520GPCRs%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span><span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0li017TkVW8bbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC – A free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+%E2%80%93+A+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2ea16LSh4O5TIbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520%25E2%2580%2593%2520A%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Pottel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moitessier, N.</span><span> </span><span class="NLM_article-title">Customizable generation of synthetically accessible, local chemical subspaces</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">454</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00648</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00648" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=454-467&author=J.+Pottelauthor=N.+Moitessier&title=Customizable+generation+of+synthetically+accessible%2C+local+chemical+subspaces&doi=10.1021%2Facs.jcim.6b00648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Customizable Generation of Synthetically Accessible, Local Chemical Subspaces</span></div><div class="casAuthors">Pottel, Joshua; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">454-467</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening large libraries of chems. has been an efficient strategy to discover bioactive compds.; however, a portion of the potential for success is limited to the available libraries.  Synergizing combinatorial and computational chemistries emerged as a time-efficient strategy to explore the chem. space more widely.  Ideally, streamlining the evaluation process for larger, feasible chem. libraries would become commonplace.  Thus, combinatorial tools and, for example, docking methods would be integrated to identify novel bioactive entities.  The idea is simple in nature, but much more complex in practice; combinatorial chem. is more than the coupling of chems. into products: synthetic feasibility includes chemoselectivity, stereoselectivity, protecting group chem. and chem. availability which must all be considered for combinatorial library design.  In addn., intuitive interfaces and simple user manipulation is key for optimal use of such tools by org. chemists-crucial for the integration of such software in medicinal chem. labs.  We present herein FINDERS and REACT2D-integrated into the VIRTUAL CHEMIST platform, a modular software suite.  This approach enhances virtual combinatorial chem. by identifying available chems. compatible with a user-defined chem. transformation and by carrying out the reaction leading to libraries of realistic, synthetically accessible chems.-all with a completely automated, black-box, and efficient design.  We demonstrate its utility by generating ∼40 million synthetically accessible, stereochem. accurate compds. from a single library of 100,000 purchasable mols. and 56 well-characterized chem. reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5D6CtvWNM4bVg90H21EOLACvtfcHk0lh_h2zu4rPpsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCntbk%253D&md5=5556a4e23ad84b3fbf594f613afb4841</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00648%26sid%3Dliteratum%253Aachs%26aulast%3DPottel%26aufirst%3DJ.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DCustomizable%2520generation%2520of%2520synthetically%2520accessible%252C%2520local%2520chemical%2520subspaces%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D454%26epage%3D467%26doi%3D10.1021%2Facs.jcim.6b00648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Baker, J. G.</span><span> </span><span class="NLM_article-title">The selectivity of β-adrenoceptor antagonists at the human β1, β2 and β3 adrenoceptors</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">144</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">322</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0706048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fsj.bjp.0706048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=15655528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2005&pages=317-322&author=J.+G.+Baker&title=The+selectivity+of+%CE%B2-adrenoceptor+antagonists+at+the+human+%CE%B21%2C+%CE%B22+and+%CE%B23+adrenoceptors&doi=10.1038%2Fsj.bjp.0706048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The selectivity of β-adrenoceptor antagonists at the human β1, β2 and β3 adrenoceptors</span></div><div class="casAuthors">Baker, Jillian G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">317-322</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">β-Adrenoceptor antagonists (β-blockers) are one of the most widely used classes of drugs in cardiovascular medicine (hypertension, ischemic heart disease and increasingly in heart failure) as well as in the management of anxiety, migraine and glaucoma.  Where known, the mode of action in cardiovascular disease is from antagonism of endogenous catecholamine responses in the heart (mainly at β1-adrenoceptors), while the worrisome side effects of bronchospasm result from airway β2-adrenoceptor blockade.  The aim of this study was to det. the selectivity of β-antagonists for the human β-adrenoceptor subtypes.  3H-CGP 12177 whole cell-binding studies were undertaken in CHO cell lines stably expressing either the human β1-, β2- or the β3-adrenoceptor to det. the affinity of ligands for each receptor subtype in the same cell background.  In this study, the selectivity of well-known subtype-selective ligands was clearly demonstrated: thus, the selective β1 antagonist CGP 20712A was 501-fold selective over β2 and 4169-fold selective over β3; the β2-selective antagonist ICI 118551 was 550- and 661-fold selective over β1 and β3, resp., and the selective β3 compd. CL 316243 was 10-fold selective over β2 and more than 129-fold selective over β1.  Those β2-adrenoceptor agonists used clin. for the treatment of asthma and COPD were β2 selective: 29-, 61- and 2818-fold for salbutamol, terbutaline and salmeterol over β1, resp.  There was little difference in the affinity of these ligands between β1 and β3 adrenoceptors.  The clin. used β-antagonists studied ranged from bisoprolol (14-fold β1-selective) to timolol (26-fold β2-selective).  However, the majority showed little selectivity for the β1- over the β2-adrenoceptor, with many actually being more β2-selective.  This study shows that the β1/β2 selectivity of most clin. used β-blockers is poor in intact cells, and that some compds. that are traditionally classed as β1-selective' actually have higher affinity for the β2-adrenoceptor.  There is therefore considerable potential for developing more selective β-antagonists for clin. use and thereby reducing the side-effect profile of β-blockers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqggxqVpB4tvrVg90H21EOLACvtfcHk0lgETLY5CDdmwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGms70%253D&md5=33cba45e9739901cf686d6cfbb096eed</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0706048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0706048%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DJ.%2BG.%26atitle%3DThe%2520selectivity%2520of%2520%25CE%25B2-adrenoceptor%2520antagonists%2520at%2520the%2520human%2520%25CE%25B21%252C%2520%25CE%25B22%2520and%2520%25CE%25B23%2520adrenoceptors%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2005%26volume%3D144%26spage%3D317%26epage%3D322%26doi%3D10.1038%2Fsj.bjp.0706048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.-y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, L.</span><span> </span><span class="NLM_article-title">Discovery of an orally selective inhibitor of signal transducer and activator of transcription 3 using advanced multiple ligand simultaneous docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2718</span><span class="NLM_x">–</span> <span class="NLM_lpage">2731</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01489</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01489" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2718-2731&author=W.+Yuauthor=C.+Liauthor=W.+Zhangauthor=Y.+Xiaauthor=S.+Liauthor=J.-y.+Linauthor=K.+Yuauthor=M.+Liuauthor=L.+Yangauthor=J.+Luoauthor=Y.+Chenauthor=H.+Sunauthor=L.+Kong&title=Discovery+of+an+orally+selective+inhibitor+of+signal+transducer+and+activator+of+transcription+3+using+advanced+multiple+ligand+simultaneous+docking&doi=10.1021%2Facs.jmedchem.6b01489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Orally Selective Inhibitor of Signal Transducer and Activator of Transcription 3 Using Advanced Multiple Ligand Simultaneous Docking</span></div><div class="casAuthors">Yu, Wenying; Li, Chenglong; Zhang, Wenda; Xia, Yuanzheng; Li, Shanshan; Lin, Jia-yuh; Yu, Keqin; Liu, Mu; Yang, Lei; Luo, Jianguang; Chen, Yijun; Sun, Hongbin; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2718-2731</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting signal transducer and activator of transcription 3 (STAT3) is a potential anticancer strategy.  However, STAT3 inhibitors with good selectivity and bioavailability are rare.  The aim of this study was to discover selective direct STAT3 inhibitors with good drug-likeness.  By advanced multiple ligand simultaneous docking (AMLSD) method, compd. 9 (5,8-dioxo-6-((2-(piperazin-1-yl)phenyl)amino)-5,8-dihydronaphthalene-1-sulfonamide) was designed as an orally-bioavailable STAT3 inhibitor, which presented superior druggability and selectivity to other representative STAT3 inhibitors.  Compd. 9 directly and selectively inhibited the pY705 site of STAT3 with an affinity (Ki) of 440nM.  The IC50 of 9 for breast cancer cell MDA-MB-231 was 184-fold lower than its IC50 for normal breast epithelial cell MCF-10A.  Compd. 9 in vivo induced significant antitumor responses better than Gefitinib and its therapeutic index should be over 100, indicating good safety of 9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiT8F_tCrky7Vg90H21EOLACvtfcHk0lgETLY5CDdmwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D&md5=d9554e49fd17a559e8c7171f36c53607</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01489%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.-y.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520selective%2520inhibitor%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520using%2520advanced%2520multiple%2520ligand%2520simultaneous%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2718%26epage%3D2731%26doi%3D10.1021%2Facs.jmedchem.6b01489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span><span class="refDoi"> DOI: 10.1038/nchembio831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0lgGtVG33B_5QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723%26doi%3D10.1038%2Fnchembio831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Docking screens: right for the right reasons?</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">770</span><span class="refDoi"> DOI: 10.2174/156802609789207091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.2174%2F156802609789207091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19754393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=755-770&author=P.+Kolbauthor=J.+J.+Irwin&title=Docking+screens%3A+right+for+the+right+reasons%3F&doi=10.2174%2F156802609789207091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Docking screens: right for the right reasons?</span></div><div class="casAuthors">Kolb, Peter; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">755-770</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Whereas docking screens have emerged as the most practical way to use protein structure for ligand discovery, an inconsistent track record raises questions about how well docking actually works.  In its favor, a growing no. of publications report the successful discovery of new ligands, often supported by exptl. affinity data and controls for artifacts.  Few reports, however, actually test the underlying structural hypotheses that docking makes.  To be successful and not just lucky, prospective docking must not only rank a true ligand among the top scoring compds., it must also correctly orient the ligand so the score it receives is biophys. sound.  If the correct binding pose is not predicted, a skeptic might well infer that the discovery was serendipitous.  Surveying over 15 years of the docking literature, we were surprised to discover how rarely sufficient evidence is presented to establish whether docking actually worked for the right reasons.  The paucity of exptl. tests of theor. predicted poses undermines confidence in a technique that has otherwise become widely accepted.  Of course, solving a crystal structure is not always possible, and even when it is, it can be a lot of work, and is not readily accessible to all groups.  Even when a structure can be detd., investigators may prefer to gloss over an erroneous structural prediction to better focus on their discovery.  Still, the absence of a direct test of theory by expt. is a loss for method developers seeking to understand and improve docking methods.  We hope this review will motivate investigators to solve structures and compare them with their predictions whenever possible, to advance the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXLK_ZZmntsrVg90H21EOLACvtfcHk0lgGtVG33B_5QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP&md5=3e6c0073795fae06c0d5a1175af8e8e5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.2174%2F156802609789207091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609789207091%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DDocking%2520screens%253A%2520right%2520for%2520the%2520right%2520reasons%253F%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D755%26epage%3D770%26doi%3D10.2174%2F156802609789207091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking & chemoinformatic screens for new ligands and targets</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">429</span><span class="NLM_x">–</span> <span class="NLM_lpage">436</span><span class="refDoi"> DOI: 10.1016/j.copbio.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2Fj.copbio.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19733475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=429-436&author=P.+Kolbauthor=R.+S.+Ferreiraauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Docking+%26+chemoinformatic+screens+for+new+ligands+and+targets&doi=10.1016%2Fj.copbio.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and chemoinformatic screens for new ligands and targets</span></div><div class="casAuthors">Kolb, Peter; Ferreira, Rafaela S.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-436</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computer-based docking screens are now widely used to discover new ligands for targets of known structure; in the last two years alone, the discovery of ligands for more than 20 proteins has been reported.  Recently, investigators have also turned to predicting new substrates for enzymes of unknown function, taking docking in a wholly new direction.  Increasingly, the hit rates, the true-positives, and the false-positives from the docking screens are being compared to those from empirical, high-throughput screens, revealing the strengths, weaknesses, and complementarities of both techniques.  The recent efflorescence of GPCR structures has made these quintessential drug targets available to structure-based approaches.  Consistent with their druggability', the docking screens have returned high hit rates and potent mols.  Finally, in the last several years, an approach almost exactly opposite to docking has also appeared; this pharmacol. network approach begins not with the structure of the target but rather those of drug mols. and asks, given a pattern of chem. in the ligands, what targets may a particular drug bind to.  This method, which returns to an older, pharmacol. logic, has been surprisingly successful in predicting new off-targets' for established drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryVNL3mMiQjbVg90H21EOLACvtfcHk0lgGtVG33B_5QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL&md5=4b6b2457a735e18be5780d9299ebffe6</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520%2526%2520chemoinformatic%2520screens%2520for%2520new%2520ligands%2520and%2520targets%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2009%26volume%3D20%26spage%3D429%26epage%3D436%26doi%3D10.1016%2Fj.copbio.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Chevillard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span> </span><span class="NLM_article-title">SCUBIDOO: A large yet screenable and easily searchable database of computationally created chemical compounds optimized toward high likelihood of synthetic tractability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1824</span><span class="NLM_x">–</span> <span class="NLM_lpage">1835</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCltbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1824-1835&author=F.+Chevillardauthor=P.+Kolb&title=SCUBIDOO%3A+A+large+yet+screenable+and+easily+searchable+database+of+computationally+created+chemical+compounds+optimized+toward+high+likelihood+of+synthetic+tractability&doi=10.1021%2Facs.jcim.5b00203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">SCUBIDOO: A Large yet Screenable and Easily Searchable Database of Computationally Created Chemical Compounds Optimized toward High Likelihood of Synthetic Tractability</span></div><div class="casAuthors">Chevillard, F.; Kolb, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1824-1835</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">De novo drug design is widely assisted by computational approaches that enable the generation of a tremendous amt. of new virtual mols. within a short time frame.  While the novelty of the computationally generated compds. can easily be assessed, such approaches often neglect the synthetic feasibility of the mols., thus creating a potential hurdle that can be a barrier to further investigation.  Therefore, we have developed SCUBIDOO, a freely accessible database concept that currently holds 21 million virtual products originating from a small library of building blocks and a collection of robust org. reactions.  This large data set was reduced to three representative and computationally tractable samples denoted as S, M, and L, contg. 9994, 99,977, and 999,794 products, resp.  These small sets are useful as starting points for ligand identification and optimization projects.  The generated products come with synthesis instructions and alerts of possible side reactions, and we show that they exhibit drug-like properties while still extending into unexplored quadrants of chem. space, thus suggesting novelty.  We show multiple examples that demonstrate how SCUBIDOO can facilitate the search around initial hits.  This database might be a useful idea generator for early ligand discovery projects since it allows a focus on those mols. that are likely to be synthetically feasible and can therefore be studied further.  Together with its modular building block construction principle, this database is also suitable for structure-activity relationship studies or fragment-growing strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJpliMmDkrU7Vg90H21EOLACvtfcHk0lhBTEvUuEx24w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCltbzL&md5=757ab5c75eeec61ae4f5811bee131f4c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00203%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DSCUBIDOO%253A%2520A%2520large%2520yet%2520screenable%2520and%2520easily%2520searchable%2520database%2520of%2520computationally%2520created%2520chemical%2520compounds%2520optimized%2520toward%2520high%2520likelihood%2520of%2520synthetic%2520tractability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D1824%26epage%3D1835%26doi%3D10.1021%2Facs.jcim.5b00203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">High-resolution crystal structure of an engineered human β<sub>2</sub>-adrenergic G protein-coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1258</span><span class="NLM_x">–</span> <span class="NLM_lpage">1265</span><span class="refDoi"> DOI: 10.1126/science.1150577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1126%2Fscience.1150577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17962520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1258-1265&author=V.+Cherezovauthor=D.+M.+Rosenbaumauthor=M.+A.+Hansonauthor=S.+G.+F.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.-J.+Choiauthor=P.+Kuhnauthor=W.+I.+Weisauthor=B.+K.+Kobilkaauthor=R.+C.+Stevens&title=High-resolution+crystal+structure+of+an+engineered+human+%CE%B22-adrenergic+G+protein-coupled+receptor&doi=10.1126%2Fscience.1150577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structure of an Engineered Human β2-Adrenergic G Protein-Coupled Receptor</span></div><div class="casAuthors">Cherezov, Vadim; Rosenbaum, Daniel M.; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Kuhn, Peter; Weis, William I.; Kobilka, Brian K.; Stevens, Raymond C.; Takeda, S.; Kadowaki, S.; Haga, T.; Takaesu, H.; Mitaku, S.; Fredriksson, R.; Lagerstrom, M. C.; Lundin, L. G.; Schioth, H. B.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Lefkowitz, R. J.; Shenoy, S. K.; Rosenbaum, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1258-1265</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Second extracellular loop, which in the β2-adrenergic receptor contains an unusual pair of disulfide bonds and an extra helix.  This loop and the absence Heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors constitute the largest family of eukaryotic signal transduction proteins that communicate across the membrane.  We report the crystal structure of a human β2-adrenergic receptor-T4 lysozyme fusion protein bound to the partial inverse agonist carazolol at 2.4 angstrom resoln.  The structure provides a high-resoln. view of a human G protein-coupled receptor bound to a diffusible ligand.  Ligand-binding site accessibility is enabled by the second extracellular loop, which is held out of the binding cavity by a pair of closely spaced disulfide bridges and a short helical segment within the loop.  Cholesterol, a necessary component for crystn., mediates an intriguing parallel assocn. of receptor mols. in the crystal lattice.  Although the location of carazolol in the β2-adrenergic receptor is very similar to that of retinal in rhodopsin, structural differences in the ligand-binding site and other regions highlight the challenges in using rhodopsin as a template model for this large receptor family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC-8wZ2k2GHLVg90H21EOLACvtfcHk0lhBTEvUuEx24w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I&md5=12c5bacb8464a4b243fe9341192b5b3b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150577%26sid%3Dliteratum%253Aachs%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520an%2520engineered%2520human%2520%25CE%25B22-adrenergic%2520G%2520protein-coupled%2520receptor%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1258%26epage%3D1265%26doi%3D10.1126%2Fscience.1150577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, X.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">GPCR engineering yields high-resolution structural insights into β<sub>2</sub>-adrenergic receptor function</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1266</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1126/science.1150609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1126%2Fscience.1150609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17962519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1266-1273&author=D.+M.+Rosenbaumauthor=V.+Cherezovauthor=M.+A.+Hansonauthor=S.+G.+F.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.-J.+Choiauthor=X.-J.+Yaoauthor=W.+I.+Weisauthor=R.+C.+Stevensauthor=B.+K.+Kobilka&title=GPCR+engineering+yields+high-resolution+structural+insights+into+%CE%B22-adrenergic+receptor+function&doi=10.1126%2Fscience.1150609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">GPCR Engineering Yields High-Resolution Structural Insights into β2-Adrenergic Receptor Function</span></div><div class="casAuthors">Rosenbaum, Daniel M.; Cherezov, Vadim; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Yao, Xiao-Jie; Weis, William I.; Stevens, Raymond C.; Kobilka, Brian K.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Kobilka, B. K.; Deupi, X.; Cherezov, V.; Rasmussen, S. G. F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1266-1273</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The β2-adrenergic receptor (β2AR) is a well-studied prototype for heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors (GPCRs) that respond to diffusible hormones and neurotransmitters.  To overcome the structural flexibility of the β2AR and to facilitate its crystn., we engineered a β2AR fusion protein in which T4 lysozyme (T4L) replaces most of the third intracellular loop of the GPCR ("β2AR-T4L") and showed that this protein retains near-native pharmacol. properties.  Anal. of adrenergic receptor ligand-binding mutants within the context of the reported high-resoln. structure of β2AR-T4L provides insights into inverse-agonist binding and the structural changes required to accommodate catecholamine agonists.  Amino acids known to regulate receptor function are linked through packing interactions and a network of hydrogen bonds, suggesting a conformational pathway from the ligand-binding pocket to regions that interact with G proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6gmrMhXnVl7Vg90H21EOLACvtfcHk0lijE1m1D863AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7J&md5=d422ff1e7a28719848c0e16fede045db</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150609%26sid%3Dliteratum%253Aachs%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DYao%26aufirst%3DX.-J.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DGPCR%2520engineering%2520yields%2520high-resolution%2520structural%2520insights%2520into%2520%25CE%25B22-adrenergic%2520receptor%2520function%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1266%26epage%3D1273%26doi%3D10.1126%2Fscience.1150609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Brooks, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruccoleri, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olafson, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">States, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">CHARMM: A program for macromolecular energy, minimization, and dynamics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span><span class="refDoi"> DOI: 10.1002/jcc.540040211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fjcc.540040211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1983&pages=187-217&author=B.+R.+Brooksauthor=R.+E.+Bruccoleriauthor=B.+D.+Olafsonauthor=D.+J.+Statesauthor=S.+Swaminathanauthor=M.+Karplus&title=CHARMM%3A+A+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations&doi=10.1002%2Fjcc.540040211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM:  a program for macromolecular energy, minimization, and dynamics calculations</span></div><div class="casAuthors">Brooks, Bernard R.; Bruccoleri, Robert E.; Olafson, Barry D.; States, David J.; Swaminathan, S.; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-217</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">CHARMM (Chem. at HARvard Macromol. Mechanics) is a highly flexible computer program which uses empirical energy functions to model macromol. systems.  The program can read or model build structures, energy minimize them by first- or second-deriv. techniques, perform a normal mode or mol. dynamics simulation, and analyze the structural, equil., and dynamic properties detd. in these calcns.  The operations that CHARMM can perform are described, and some implementation details are given.  A set of parameters for the empirical energy function and a sample run are included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSFuoT3w87LVg90H21EOLACvtfcHk0lijE1m1D863AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D&md5=bd639b4299ac9934f4497c1a9fe750d2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540040211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540040211%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DBruccoleri%26aufirst%3DR.%2BE.%26aulast%3DOlafson%26aufirst%3DB.%2BD.%26aulast%3DStates%26aufirst%3DD.%2BJ.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DCHARMM%253A%2520A%2520program%2520for%2520macromolecular%2520energy%252C%2520minimization%252C%2520and%2520dynamics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1983%26volume%3D4%26spage%3D187%26epage%3D217%26doi%3D10.1002%2Fjcc.540040211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Momany, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rone, R.</span><span> </span><span class="NLM_article-title">Validation of the general purpose QUANTA 3.2/CHARMm force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">888</span><span class="NLM_x">–</span> <span class="NLM_lpage">900</span><span class="refDoi"> DOI: 10.1002/jcc.540130714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fjcc.540130714" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=888-900&author=F.+A.+Momanyauthor=R.+Rone&title=Validation+of+the+general+purpose+QUANTA+3.2%2FCHARMm+force+field&doi=10.1002%2Fjcc.540130714"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130714%26sid%3Dliteratum%253Aachs%26aulast%3DMomany%26aufirst%3DF.%2BA.%26aulast%3DRone%26aufirst%3DR.%26atitle%3DValidation%2520of%2520the%2520general%2520purpose%2520QUANTA%25203.2%252FCHARMm%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D888%26epage%3D900%26doi%3D10.1002%2Fjcc.540130714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 1. Application to neutral molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4732</span><span class="NLM_x">–</span> <span class="NLM_lpage">4739</span><span class="refDoi"> DOI: 10.1021/j100374a066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100374a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4732-4739&author=K.+Noauthor=J.+Grantauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+1.+Application+to+neutral+molecules+as+models+for+polypeptides&doi=10.1021%2Fj100374a066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  1.  Application to neutral molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4732-9</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">Modified methods are proposed for detg. net at. charges, based on an existing scheme using the method of partial equalization of orbital electronegativity.  The new methods introduce (as constraints) exptl. dipole and, where possible, quadrupole moments of a set of mols. chosen to resemble the different at. environments found in proteins.  As a consequence, the at. charges reproduce accurately the exptl. values of mol. dipole and quadrupole moments.  The electrostatic potential surface, calcd. with the new net-at.-charge set, compares well to a surface obtained from a moderate-level ab initio calcn. (6-31G**).  However, the new charges are double those obtained from studies in which the charges are detd. from a least-squares fit to the calcd. ab initio electrostatic potential (potential-derived method).  The methodol. outlined leads to a direct calcn. of the permanent mol. charge distribution represented as a set of distributed monopoles, dependent only on the connectivity of the mol.  Hence, the method circumvents assumptions concerning the transferability of charges and can therefore approx. describe any redistribution of charge d. upon assembly of amino acid residues to form a polypeptide.  The method may be of use in providing an approx. representation of mol. charge distributions in extended systems, esp. large polypeptide mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5hUCBNRO0TLVg90H21EOLACvtfcHk0lisRlmsBL_O0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D&md5=1783d0a7d1e5c52cb5bf93d96f16d8c6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fj100374a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a066%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25201.%2520Application%2520to%2520neutral%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4732%26epage%3D4739%26doi%3D10.1021%2Fj100374a066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 2. Application to ionic and aromatic molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4740</span><span class="NLM_x">–</span> <span class="NLM_lpage">4746</span><span class="refDoi"> DOI: 10.1021/j100374a067</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100374a067" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4740-4746&author=K.+Noauthor=J.+Grantauthor=M.+Jhonauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+2.+Application+to+ionic+and+aromatic+molecules+as+models+for+polypeptides&doi=10.1021%2Fj100374a067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  2.  Application to ionic and aromatic molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Jhon, Mu Shik; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4740-6</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">To finalize a scheme intended to calc. directly approx. charge distributions for proteins, parameters for a modified title method (PEOE) are obtained for both ionic and arom. functional groups.  For ionic groups, the electrostatic potentials of several ionic mols. (computed with the 6-31G** basis set) are used as constraints; for arom. groups, the exptl. gas-phase dipole and quadrupole moments of several arom. mols. are used as constraints.  The electrostatic potentials calcd. with the modified PEOE point charges of ionic mols. are quite close to the ab initio electrostatic potential surfaces, and the dipole and quadrupole moments calcd. with the modified PEOE charges of arom. mols. agree well with exptl. data.  The method enables the charge distribution, described as a set of atom-centered monopoles, to be calcd. directly.  This avoids assumptions concerning transferability of charges from small model compds. and describes in an approx. fashion the redistribution of charge upon assembly of amino acid residues into polypeptide mols.  The computational requirements for the procedure are negligible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdOLr-aBqHiLVg90H21EOLACvtfcHk0lisRlmsBL_O0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D&md5=b819eb54db46af0ad66a22d12d052c4b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fj100374a067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a067%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DJhon%26aufirst%3DM.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25202.%2520Application%2520to%2520ionic%2520and%2520aromatic%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4740%26epage%3D4746%26doi%3D10.1021%2Fj100374a067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Landrum, G.</span><span> </span><span class="NLM_article-title">RDKit: Open-source chemoinformatics</span>. <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a> (accessed Nov 15,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Landrum&title=RDKit%3A+Open-source+chemoinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3DLandrum%26aufirst%3DG.%26atitle%3DRDKit%253A%2520Open-source%2520chemoinformatics%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span>OMEGA, version <span class="NLM_edition">2.5.1.4</span>; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>; <a href="https://www.eyesopen.com/omega" class="extLink">https://www.eyesopen.com/omega</a> (accessed Nov 15, 2014).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OMEGA%2C+version+2.5.1.4%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM%3B+https%3A%2F%2Fwww.eyesopen.com%2Fomega+%28accessed+Nov+15%2C+2014%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOMEGA%26pub%3DOpenEye%2520Scientific%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Hawkins, P. C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellingson, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M. T.</span><span> </span><span class="NLM_article-title">Conformer generation with OMEGA: algorithm and validation using high quality structures from the Protein Databank and Cambridge Structural Database</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">572</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span><span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+algorithm+and+validation+using+high+quality+structures+from+the+Protein+Databank+and+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lgZUCFK8Wv8Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520Protein%2520Databank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: A free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lgZUCFK8Wv8Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span>SZYBKI, version <span class="NLM_edition">1.8.0.2</span>; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>; <a href="https://www.eyesopen.com/oedocking" class="extLink">https://www.eyesopen.com/oedocking</a> (accessed Nov 15, 2014).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SZYBKI%2C+version+1.8.0.2%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM%3B+https%3A%2F%2Fwww.eyesopen.com%2Foedocking+%28accessed+Nov+15%2C+2014%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSZYBKI%26pub%3DOpenEye%2520Scientific%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span>GraphPad Prism, version <span class="NLM_edition">7.00</span> for Windows; <span class="NLM_publisher-name">GraphPad Software</span>: <span class="NLM_publisher-loc">La Jolla, CA</span>; <a href="https://www.graphpad.com/scientific-software/prism/" class="extLink">https://www.graphpad.com/scientific-software/prism/</a> (accessed Sep 9, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GraphPad+Prism%2C+version+7.00+for+Windows%3B+GraphPad+Software%3A+La+Jolla%2C+CA%3B+https%3A%2F%2Fwww.graphpad.com%2Fscientific-software%2Fprism%2F+%28accessed+Sep+9%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGraphPad%2520Prism%26pub%3DGraphPad%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Pauli, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lankin, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gödecke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaki, B. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McAlpine, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napolitano, J. G.</span><span> </span><span class="NLM_article-title">Importance of purity evaluation and the potential of quantitative 1H NMR as a purity assay: miniperspective</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9220</span><span class="NLM_x">–</span> <span class="NLM_lpage">9231</span><span class="refDoi"> DOI: 10.1021/jm500734a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500734a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslahsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9220-9231&author=G.+F.+Pauliauthor=S.-N.+Chenauthor=C.+Simmlerauthor=D.+C.+Lankinauthor=T.+G%C3%B6deckeauthor=B.+U.+Jakiauthor=J.+B.+Friesenauthor=J.+B.+McAlpineauthor=J.+G.+Napolitano&title=Importance+of+purity+evaluation+and+the+potential+of+quantitative+1H+NMR+as+a+purity+assay%3A+miniperspective&doi=10.1021%2Fjm500734a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Importance of Purity Evaluation and the Potential of Quantitative 1H NMR as a Purity Assay</span></div><div class="casAuthors">Pauli, Guido F.; Chen, Shao-Nong; Simmler, Charlotte; Lankin, David C.; Godecke, Tanja; Jaki, Birgit U.; Friesen, J. Brent; McAlpine, James B.; Napolitano, Jose G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9220-9231</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In any biomedical and chem. context, a truthful description of chem. constitution requires coverage of both structure and purity.  This qualification affects all drug mols., regardless of development stage (early discovery to approved drug) and source (natural product or synthetic).  Purity assessment is particularly crit. in discovery programs and whenever chem. is linked with biol. and/or therapeutic outcome.  Compared with chromatog. and elemental anal., quant. NMR (qNMR) uses nearly universal detection and provides a versatile and orthogonal means of purity evaluation.  Abs. qNMR with flexible calibration captures analytes that frequently escape detection (water, sorbents).  Widely accepted structural NMR workflows require minimal or no adjustments to become practical 1H qNMR (qHNMR) procedures with simultaneous qual. and (abs.) quant. capability.  This study reviews underlying concepts, provides a framework for std. qHNMR purity assays, and shows how adequate accuracy and precision are achieved for the intended use of the material.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqquFEu5lLVVrVg90H21EOLACvtfcHk0lj9vsv-ugQH3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslahsLzF&md5=78900d0b03e31739e50c04c63c55dba7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm500734a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500734a%26sid%3Dliteratum%253Aachs%26aulast%3DPauli%26aufirst%3DG.%2BF.%26aulast%3DChen%26aufirst%3DS.-N.%26aulast%3DSimmler%26aufirst%3DC.%26aulast%3DLankin%26aufirst%3DD.%2BC.%26aulast%3DG%25C3%25B6decke%26aufirst%3DT.%26aulast%3DJaki%26aufirst%3DB.%2BU.%26aulast%3DFriesen%26aufirst%3DJ.%2BB.%26aulast%3DMcAlpine%26aufirst%3DJ.%2BB.%26aulast%3DNapolitano%26aufirst%3DJ.%2BG.%26atitle%3DImportance%2520of%2520purity%2520evaluation%2520and%2520the%2520potential%2520of%2520quantitative%25201H%2520NMR%2520as%2520a%2520purity%2520assay%253A%2520miniperspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9220%26epage%3D9231%26doi%3D10.1021%2Fjm500734a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Steyaert, J.; Laeremans, T.; Pardon, E.</span><span> </span><span class="NLM_article-title">Novel chimeric polypeptides for screening and drug discovery purposes</span>. WO PCT Number PCT/EP2014/051845,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.+Steyaert&author=T.+Laeremans&author=E.+Pardon&title=Novel+chimeric+polypeptides+for+screening+and+drug+discovery+purposes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSteyaert%26aufirst%3DJ.%26atitle%3DNovel%2520chimeric%2520polypeptides%2520for%2520screening%2520and%2520drug%2520discovery%2520purposes%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Yung-Chi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (Ki) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28Ki%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528Ki%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 22 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jakub Gunera, Jillian G. Baker, Niek van Hilten, Daniel M. Rosenbaum, <span class="NLM_string-name hlFld-ContribAuthor">Peter Kolb</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery of Novel Ligands for the Orexin 2 Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 11045-11053. <a href="https://doi.org/10.1021/acs.jmedchem.0c00964" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00964</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00964%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDiscovery%252Bof%252BNovel%252BLigands%252Bfor%252Bthe%252BOrexin%252B2%252BReceptor%26aulast%3DGunera%26aufirst%3DJakub%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05062020%26date%3D26092020%26volume%3D63%26issue%3D19%26spage%3D11045%26epage%3D11053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, Christopher N. Johnson, <span class="NLM_string-name hlFld-ContribAuthor">Paul N. Mortenson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2018. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4430-4444. <a href="https://doi.org/10.1021/acs.jmedchem.9b01581" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01581%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2018%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D22092019%26date%3D22012020%26date%3D08012020%26volume%3D63%26issue%3D9%26spage%3D4430%26epage%3D4444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nathan Brown, Marco Fiscato, Marwin H.S. Segler, <span class="NLM_string-name hlFld-ContribAuthor">Alain C. Vaucher</span>. </span><span class="cited-content_cbyCitation_article-title">GuacaMol: Benchmarking Models for de Novo Molecular Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1096-1108. <a href="https://doi.org/10.1021/acs.jcim.8b00839" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00839</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00839%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DGuacaMol%25253A%252BBenchmarking%252BModels%252Bfor%252Bde%252BNovo%252BMolecular%252BDesign%26aulast%3DBrown%26aufirst%3DNathan%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D26112018%26date%3D19032019%26volume%3D59%26issue%3D3%26spage%3D1096%26epage%3D1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Niek van Hilten, Florent Chevillard, <span class="NLM_string-name hlFld-ContribAuthor">Peter Kolb</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Compound Libraries in Computer-Assisted Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (2)
                                     , 644-651. <a href="https://doi.org/10.1021/acs.jcim.8b00737" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00737</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00737%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DVirtual%252BCompound%252BLibraries%252Bin%252BComputer-Assisted%252BDrug%252BDiscovery%26aulast%3Dvan%2BHilten%26aufirst%3DNiek%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D22102018%26date%3D24012019%26date%3D09012019%26volume%3D59%26issue%3D2%26spage%3D644%26epage%3D651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miles  Congreve</span>, <span class="hlFld-ContribAuthor ">John A.  Christopher</span>, <span class="hlFld-ContribAuthor ">Chris  Graaf</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Based Drug Design for G Protein‐Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-59. <a href="https://doi.org/10.1002/0471266949.bmc269" title="DOI URL">https://doi.org/10.1002/0471266949.bmc269</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc269%26sid%3Dliteratum%253Aachs%26atitle%3DStructure%2525E2%252580%252590Based%252BDrug%252BDesign%252Bfor%252BG%252BProtein%2525E2%252580%252590Coupled%252BReceptors%26aulast%3DCongreve%26aufirst%3DMiles%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D59%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bruno Junior  Neves</span>, <span class="hlFld-ContribAuthor ">Melina  Mottin</span>, <span class="hlFld-ContribAuthor ">José Teofilo  Moreira-Filho</span>, <span class="hlFld-ContribAuthor ">Bruna Katiele de Paula  Sousa</span>, <span class="hlFld-ContribAuthor ">Sabrina Silva  Mendonca</span>, <span class="hlFld-ContribAuthor ">Carolina Horta  Andrade</span>. </span><span class="cited-content_cbyCitation_article-title">Best Practices for Docking-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 75-98. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00001-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00001-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00001-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00001-1%26sid%3Dliteratum%253Aachs%26atitle%3DBest%252BPractices%252Bfor%252BDocking-Based%252BVirtual%252BScreening%26aulast%3DNeves%26aufirst%3DBruno%2BJunior%26date%3D2021%26spage%3D75%26epage%3D98%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Suresh  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Online Resource and Tools for the Development of Drugs Against Novel Coronavirus. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1007/7653_2020_53" title="DOI URL">https://doi.org/10.1007/7653_2020_53</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7653_2020_53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7653_2020_53%26sid%3Dliteratum%253Aachs%26atitle%3DOnline%252BResource%252Band%252BTools%252Bfor%252Bthe%252BDevelopment%252Bof%252BDrugs%252BAgainst%252BNovel%252BCoronavirus%26aulast%3DKumar%26aufirst%3DSuresh%26date%3D2021%26date%3D2021%26pub%3DSpringer%2520US%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jan  Hellmann</span>, <span class="hlFld-ContribAuthor ">Matthäus  Drabek</span>, <span class="hlFld-ContribAuthor ">Jie  Yin</span>, <span class="hlFld-ContribAuthor ">Jakub  Gunera</span>, <span class="hlFld-ContribAuthor ">Theresa  Pröll</span>, <span class="hlFld-ContribAuthor ">Frank  Kraus</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Langmead</span>, <span class="hlFld-ContribAuthor ">Harald  Hübner</span>, <span class="hlFld-ContribAuthor ">Dorothee  Weikert</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Daniel M.  Rosenbaum</span>, <span class="hlFld-ContribAuthor ">Peter  Gmeiner</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based development of a subtype-selective orexin 1 receptor antagonist. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2020,</strong> <em>117 </em>
                                    (30)
                                     , 18059-18067. <a href="https://doi.org/10.1073/pnas.2002704117" title="DOI URL">https://doi.org/10.1073/pnas.2002704117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2002704117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2002704117%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DStructure-based%252Bdevelopment%252Bof%252Ba%252Bsubtype-selective%252Borexin%252B1%252Breceptor%252Bantagonist%26aulast%3DHellmann%26aufirst%3DJan%26date%3D2020%26date%3D2020%26volume%3D117%26issue%3D30%26spage%3D18059%26epage%3D18067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauro Ribeiro  de Souza Neto</span>, <span class="hlFld-ContribAuthor ">José Teófilo  Moreira-Filho</span>, <span class="hlFld-ContribAuthor ">Bruno Junior  Neves</span>, <span class="hlFld-ContribAuthor ">Rocío Lucía Beatriz Riveros  Maidana</span>, <span class="hlFld-ContribAuthor ">Ana Carolina Ramos  Guimarães</span>, <span class="hlFld-ContribAuthor ">Nicholas  Furnham</span>, <span class="hlFld-ContribAuthor ">Carolina Horta  Andrade</span>, <span class="hlFld-ContribAuthor ">Floriano Paes  Silva</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Strategies to Support Fragment-to-Lead Optimization in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.00093" title="DOI URL">https://doi.org/10.3389/fchem.2020.00093</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.00093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.00093%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BStrategies%252Bto%252BSupport%252BFragment-to-Lead%252BOptimization%252Bin%252BDrug%252BDiscovery%26aulast%3Dde%2BSouza%2BNeto%26aufirst%3DLauro%2BRibeiro%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vladimir  Bobkov</span>, <span class="hlFld-ContribAuthor ">Marta  Arimont</span>, <span class="hlFld-ContribAuthor ">Aurélien  Zarca</span>, <span class="hlFld-ContribAuthor ">Timo W.M.  De Groof</span>, <span class="hlFld-ContribAuthor ">Bas  van der Woning</span>, <span class="hlFld-ContribAuthor ">Hans  de Haard</span>, <span class="hlFld-ContribAuthor ">Martine J.  Smit</span>. </span><span class="cited-content_cbyCitation_article-title">Antibodies Targeting Chemokine Receptors CXCR4 and ACKR3. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmacology</span><span> <strong>2019,</strong> <em>96 </em>
                                    (6)
                                     , 753-764. <a href="https://doi.org/10.1124/mol.119.116954" title="DOI URL">https://doi.org/10.1124/mol.119.116954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/mol.119.116954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fmol.119.116954%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmacology%26atitle%3DAntibodies%252BTargeting%252BChemokine%252BReceptors%252BCXCR4%252Band%252BACKR3%26aulast%3DBobkov%26aufirst%3DVladimir%26date%3D2019%26date%3D2019%26volume%3D96%26issue%3D6%26spage%3D753%26epage%3D764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>, <span class="hlFld-ContribAuthor ">David E.  Clark</span>. </span><span class="cited-content_cbyCitation_article-title">Automated De Novo Drug Design: Are We Nearly There Yet?. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2019,</strong> <em>131 </em>
                                    (32)
                                     , 10906-10917. <a href="https://doi.org/10.1002/ange.201814681" title="DOI URL">https://doi.org/10.1002/ange.201814681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201814681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201814681%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DAutomated%252BDe%252BNovo%252BDrug%252BDesign%25253A%252BAre%252BWe%252BNearly%252BThere%252BYet%25253F%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2019%26date%3D2019%26volume%3D131%26issue%3D32%26spage%3D10906%26epage%3D10917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>, <span class="hlFld-ContribAuthor ">David E.  Clark</span>. </span><span class="cited-content_cbyCitation_article-title">Automated De Novo Drug Design: Are We Nearly There Yet?. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2019,</strong> <em>58 </em>
                                    (32)
                                     , 10792-10803. <a href="https://doi.org/10.1002/anie.201814681" title="DOI URL">https://doi.org/10.1002/anie.201814681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201814681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201814681%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DAutomated%252BDe%252BNovo%252BDrug%252BDesign%25253A%252BAre%252BWe%252BNearly%252BThere%252BYet%25253F%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2019%26volume%3D58%26issue%3D32%26spage%3D10792%26epage%3D10803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas  Fischer</span>, <span class="hlFld-ContribAuthor ">Silvia  Gazzola</span>, <span class="hlFld-ContribAuthor ">Rainer  Riedl</span>. </span><span class="cited-content_cbyCitation_article-title">Approaching Target Selectivity by De Novo Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (8)
                                     , 791-803. <a href="https://doi.org/10.1080/17460441.2019.1615435" title="DOI URL">https://doi.org/10.1080/17460441.2019.1615435</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1615435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1615435%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DApproaching%252BTarget%252BSelectivity%252Bby%252BDe%252BNovo%252BDrug%252BDesign%26aulast%3DFischer%26aufirst%3DThomas%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D8%26spage%3D791%26epage%3D803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Florent  Chevillard</span>, <span class="hlFld-ContribAuthor ">Silvia  Stotani</span>, <span class="hlFld-ContribAuthor ">Anna  Karawajczyk</span>, <span class="hlFld-ContribAuthor ">Stanimira  Hristeva</span>, <span class="hlFld-ContribAuthor ">Els  Pardon</span>, <span class="hlFld-ContribAuthor ">Jan  Steyaert</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Tzalis</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>. </span><span class="cited-content_cbyCitation_article-title">Interrogating dense ligand chemical space with a forward-synthetic library. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2019,</strong> <em>116 </em>
                                    (23)
                                     , 11496-11501. <a href="https://doi.org/10.1073/pnas.1818718116" title="DOI URL">https://doi.org/10.1073/pnas.1818718116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1818718116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1818718116%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DInterrogating%252Bdense%252Bligand%252Bchemical%252Bspace%252Bwith%252Ba%252Bforward-synthetic%252Blibrary%26aulast%3DChevillard%26aufirst%3DFlorent%26date%3D2019%26date%3D2019%26volume%3D116%26issue%3D23%26spage%3D11496%26epage%3D11501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raimond  Heukers</span>, <span class="hlFld-ContribAuthor ">Timo W.M.  De Groof</span>, <span class="hlFld-ContribAuthor ">Martine J.  Smit</span>. </span><span class="cited-content_cbyCitation_article-title">Nanobodies detecting and modulating GPCRs outside in and inside out. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Cell Biology</span><span> <strong>2019,</strong> <em>57 </em>, 115-122. <a href="https://doi.org/10.1016/j.ceb.2019.01.003" title="DOI URL">https://doi.org/10.1016/j.ceb.2019.01.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ceb.2019.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ceb.2019.01.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Cell%2520Biology%26atitle%3DNanobodies%252Bdetecting%252Band%252Bmodulating%252BGPCRs%252Boutside%252Bin%252Band%252Binside%252Bout%26aulast%3DHeukers%26aufirst%3DRaimond%26date%3D2019%26volume%3D57%26spage%3D115%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Timo W.M.  De Groof</span>, <span class="hlFld-ContribAuthor ">Vladimir  Bobkov</span>, <span class="hlFld-ContribAuthor ">Raimond  Heukers</span>, <span class="hlFld-ContribAuthor ">Martine J.  Smit</span>. </span><span class="cited-content_cbyCitation_article-title">Nanobodies: New avenues for imaging, stabilizing and modulating GPCRs. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Endocrinology</span><span> <strong>2019,</strong> <em>484 </em>, 15-24. <a href="https://doi.org/10.1016/j.mce.2019.01.021" title="DOI URL">https://doi.org/10.1016/j.mce.2019.01.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mce.2019.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mce.2019.01.021%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Endocrinology%26atitle%3DNanobodies%25253A%252BNew%252Bavenues%252Bfor%252Bimaging%25252C%252Bstabilizing%252Band%252Bmodulating%252BGPCRs%26aulast%3DDe%2BGroof%26aufirst%3DTimo%2BW.M.%26date%3D2019%26volume%3D484%26spage%3D15%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Sommer</span>, <span class="hlFld-ContribAuthor ">Florian  Flachsenberg</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">NAOMInext – Synthetically feasible fragment growing in a structure-based design context. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>163 </em>, 747-762. <a href="https://doi.org/10.1016/j.ejmech.2018.11.075" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.075%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNAOMInext%252B%2525E2%252580%252593%252BSynthetically%252Bfeasible%252Bfragment%252Bgrowing%252Bin%252Ba%252Bstructure-based%252Bdesign%252Bcontext%26aulast%3DSommer%26aufirst%3DKai%26date%3D2019%26volume%3D163%26spage%3D747%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Ben J.  Davis</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Jahnke</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery: Advancing Fragments in the Absence of Crystal Structures. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (1)
                                     , 9-15. <a href="https://doi.org/10.1016/j.chembiol.2018.10.001" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.10.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DFragment-Based%252BDrug%252BDiscovery%25253A%252BAdvancing%252BFragments%252Bin%252Bthe%252BAbsence%252Bof%252BCrystal%252BStructures%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D9%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laurent  Hoffer</span>, <span class="hlFld-ContribAuthor ">Christophe  Muller</span>, <span class="hlFld-ContribAuthor ">Philippe  Roche</span>, <span class="hlFld-ContribAuthor ">Xavier  Morelli</span>. </span><span class="cited-content_cbyCitation_article-title">Chemistry-driven Hit-to-lead Optimization Guided by Structure-based Approaches. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2018,</strong> <em>37 </em>
                                    (9-10)
                                     , 1800059. <a href="https://doi.org/10.1002/minf.201800059" title="DOI URL">https://doi.org/10.1002/minf.201800059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201800059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201800059%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DChemistry-driven%252BHit-to-lead%252BOptimization%252BGuided%252Bby%252BStructure-based%252BApproaches%26aulast%3DHoffer%26aufirst%3DLaurent%26date%3D2018%26date%3D2018%26volume%3D37%26issue%3D9-10%26spage%3D1800059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Rémy  Marchand</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">In silico fragment-based drug design with SEED. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>156 </em>, 907-917. <a href="https://doi.org/10.1016/j.ejmech.2018.07.042" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.07.042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.07.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.07.042%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIn%252Bsilico%252Bfragment-based%252Bdrug%252Bdesign%252Bwith%252BSEED%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2018%26volume%3D156%26spage%3D907%26epage%3D917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Els  Pardon</span>, <span class="hlFld-ContribAuthor ">Cecilia  Betti</span>, <span class="hlFld-ContribAuthor ">Toon  Laeremans</span>, <span class="hlFld-ContribAuthor ">Florent  Chevillard</span>, <span class="hlFld-ContribAuthor ">Karel  Guillemyn</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Steven  Ballet</span>, <span class="hlFld-ContribAuthor ">Jan  Steyaert</span>. </span><span class="cited-content_cbyCitation_article-title">Nanobody-Enabled Reverse Pharmacology on G-Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2018,</strong> <em>130 </em>
                                    (19)
                                     , 5390-5393. <a href="https://doi.org/10.1002/ange.201712581" title="DOI URL">https://doi.org/10.1002/ange.201712581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201712581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201712581%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DNanobody-Enabled%252BReverse%252BPharmacology%252Bon%252BG-Protein-Coupled%252BReceptors%26aulast%3DPardon%26aufirst%3DEls%26date%3D2018%26date%3D2018%26volume%3D130%26issue%3D19%26spage%3D5390%26epage%3D5393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Els  Pardon</span>, <span class="hlFld-ContribAuthor ">Cecilia  Betti</span>, <span class="hlFld-ContribAuthor ">Toon  Laeremans</span>, <span class="hlFld-ContribAuthor ">Florent  Chevillard</span>, <span class="hlFld-ContribAuthor ">Karel  Guillemyn</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Steven  Ballet</span>, <span class="hlFld-ContribAuthor ">Jan  Steyaert</span>. </span><span class="cited-content_cbyCitation_article-title">Nanobody-Enabled Reverse Pharmacology on G-Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2018,</strong> <em>57 </em>
                                    (19)
                                     , 5292-5295. <a href="https://doi.org/10.1002/anie.201712581" title="DOI URL">https://doi.org/10.1002/anie.201712581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201712581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201712581%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DNanobody-Enabled%252BReverse%252BPharmacology%252Bon%252BG-Protein-Coupled%252BReceptors%26aulast%3DPardon%26aufirst%3DEls%26date%3D2018%26date%3D2018%26volume%3D57%26issue%3D19%26spage%3D5292%26epage%3D5295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fragment-based extension approaches in drug discovery: growing, linking, and merging.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Sliced surface side-view of the binding site of the receptor in its basal conformation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a>) in complex with carazolol (magenta carbons). The orthosteric site is colored in green and the SBP in blue. (b) BI167107 (green carbons) in complex with the active conformation of the receptor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE">4LDE</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a–d) Prediction of the binding mode of the four AC core fragments within the β<sub>2</sub>AR active conformation. (e) Prediction of the binding mode for compound <b>5</b> after docking to the β<sub>2</sub>AR structure in a basal conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Five fragments with favorable LE determined in an experimental screen were chosen for further growing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Surrogate Derived from an Aldehyde Defined by the Replacement of the Carbonyl by an Amine Group<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>R<sup>1</sup> = alkyl.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Surrogate Derived from a Ketone Defined by the Replacement of the Carbonyl by an Amine Group<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>This transformation introduces a chiral center; therefore two different surrogates were generated (R and S). R<sup>1</sup> and R<sup>2</sup> = alkyl.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Illustration of the filtering process. Note that the number of poses shown is much smaller than the one actually used. (a) Typical poses (i.e., a map) after docking with SEED. (b) The surrogate poses (magenta carbons) were filtered so they do not overlap with the core scaffold (green carbons), and the charged amine is close to the one of the core fragment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Example of a product keeping its orientation relative to the poses of the constitutive building blocks. Both the core fragment (compound <b>5</b> in green carbons) and the surrogate (magenta carbons) are predicted to overlap with the generated product (compound <b>11</b> in yellow carbons) after docking to the β<sub>2</sub>AR structure in a basal conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Synthesized products. Compounds <b>6</b>–<b>12</b> appeared as ACs in the experimental assay. Compound <b>13</b> appeared as AntC in the experimental assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Example of IC<sub>50</sub> curves recorded in the radioligand displacement assay for compound <b>11</b>. The green curve was obtained in the assay using the β<sub>2</sub>AR-Nb80, while the red curve corresponds to the β<sub>2</sub>AR-Nb69.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Predicted binding mode of the synthesized products using docking.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Derivative products of compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Derivative products of compounds <b>2</b>, <b>3</b>, <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/medium/jm-2017-01558y_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Derivative products of compound <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01558/20180201/images/large/jm-2017-01558y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01558&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73073" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73073" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahnke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lg8Q6XeO_oIzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1038/nrd2220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned&doi=10.1038%2Fnrd2220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0lg8Q6XeO_oIzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D211%26epage%3D219%26doi%3D10.1038%2Fnrd2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Experiences in fragment-based drug discovery</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">232</span><span class="refDoi"> DOI: 10.1016/j.tips.2012.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2Fj.tips.2012.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=22459076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=224-232&author=C.+W.+Murrayauthor=M.+L.+Verdonkauthor=D.+C.+Rees&title=Experiences+in+fragment-based+drug+discovery&doi=10.1016%2Fj.tips.2012.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">224-232</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) has become established in both industry and academia as an alternative approach to high-throughput screening for the generation of chem. leads for drug targets.  In FBDD, specialised detection methods are used to identify small chem. compds. (fragments) that bind to the drug target, and structural biol. is usually employed to establish their binding mode and to facilitate their optimization.  In this article, we present three recent and successful case histories in FBDD.  We then re-examine the key concepts and challenges of FBDD with particular emphasis on recent literature and our own experience from a substantial no. of FBDD applications.  Our opinion is that careful application of FBDD is living up to its promise of delivering high quality leads with good phys. properties and that in future many drug mols. will be derived from fragment-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUzcsuUdmx4bVg90H21EOLACvtfcHk0lg8Q6XeO_oIzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D&md5=c97b6b84a4afeadb87ee7d46e6d2bb91</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2012.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2012.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DExperiences%2520in%2520fragment-based%2520drug%2520discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D224%26epage%3D232%26doi%3D10.1016%2Fj.tips.2012.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spevak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habets, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shellooe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marimuthu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K. Y J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Andrea, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koehler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stumm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grippo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puzanov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McArthur, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaherty, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nathanson, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolop, K.</span><span> </span><span class="NLM_article-title">Clinical efficacy of a RAF inhibitor needs broad target blockade in BRAF-mutant melanoma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">596</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span><span class="refDoi"> DOI: 10.1038/nature09454</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature09454" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=20823850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1ajtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2010&pages=596-599&author=G.+Bollagauthor=P.+Hirthauthor=J.+Tsaiauthor=J.+Zhangauthor=P.+N.+Ibrahimauthor=H.+Choauthor=W.+Spevakauthor=C.+Zhangauthor=Y.+Zhangauthor=G.+Habetsauthor=E.+Burtonauthor=B.+Wongauthor=G.+Tsangauthor=B.+L.+Westauthor=B.+Powellauthor=R.+Shellooeauthor=A.+Marimuthuauthor=H.+Nguyenauthor=K.+Y+J.+Zhangauthor=D.+R.+Artisauthor=J.+Schlessingerauthor=F.+Suauthor=B.+Higginsauthor=R.+Iyerauthor=K.+D%E2%80%99Andreaauthor=A.+Koehlerauthor=M.+Stummauthor=P.+S.+Linauthor=R.+J.+Leeauthor=J.+Grippoauthor=I.+Puzanovauthor=K.+B.+Kimauthor=A.+Ribasauthor=G.+McArthurauthor=J.+Sosmanauthor=P.+B.+Chapmanauthor=K.+T.+Flahertyauthor=X.+Xuauthor=K.+L.+Nathansonauthor=K.+Nolop&title=Clinical+efficacy+of+a+RAF+inhibitor+needs+broad+target+blockade+in+BRAF-mutant+melanoma&doi=10.1038%2Fnature09454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical efficacy of a RAF inhibitor needs broad target blockade in BRAF-mutant melanoma</span></div><div class="casAuthors">Bollag, Gideon; Hirth, Peter; Tsai, James; Zhang, Jiazhong; Ibrahim, Prabha N.; Cho, Hanna; Spevak, Wayne; Zhang, Chao; Zhang, Ying; Habets, Gaston; Burton, Elizabeth A.; Wong, Bernice; Tsang, Garson; West, Brian L.; Powell, Ben; Shellooe, Rafe; Marimuthu, Adhirai; Nguyen, Hoa; Zhang, Kam Y. J.; Artis, Dean R.; Schlessinger, Joseph; Su, Fei; Higgins, Brian; Iyer, Raman; D'Andrea, Kurt; Koehler, Astrid; Stumm, Michael; Lin, Paul S.; Lee, Richard J.; Grippo, Joseph; Puzanov, Igor; Kim, Kevin B.; Ribas, Antoni; McArthur, Grant A.; Sosman, Jeffrey A.; Chapman, Paul B.; Flaherty, Keith T.; Xu, Xiaowei; Nathanson, Katherine L.; Nolop, Keith</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">7315</span>),
    <span class="NLM_cas:pages">596-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">B-RAF is the most frequently mutated protein kinase in human cancers.  The finding that oncogenic mutations in BRAF are common in melanoma, followed by the demonstration that these tumors are dependent on the RAF/MEK/ERK pathway, offered hope that inhibition of B-RAF kinase activity could benefit melanoma patients.  Herein, we describe the structure-guided discovery of PLX4032 (RG7204), a potent inhibitor of oncogenic B-RAF kinase activity.  Preclin. expts. demonstrated that PLX4032 selectively blocked the RAF/MEK/ERK pathway in BRAF mutant cells and caused regression of BRAF mutant xenografts.  Toxicol. studies confirmed a wide safety margin consistent with the high degree of selectivity, enabling Phase 1 clin. trials using a cryst. formulation of PLX4032 (ref. 5).  In a subset of melanoma patients, pathway inhibition was monitored in paired biopsy specimens collected before treatment initiation and following 2 wk of treatment.  This anal. revealed substantial inhibition of ERK phosphorylation, yet clin. evaluation did not show tumor regressions.  At higher drug exposures afforded by a new amorphous drug formulation, greater than 80% inhibition of ERK phosphorylation in the tumors of patients correlated with clin. response.  Indeed, the Phase 1 clin. data revealed a remarkably high 81% response rate in metastatic melanoma patients treated at an oral dose of 960 mg twice daily.  These data demonstrate that BRAF-mutant melanomas are highly dependent on B-RAF kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpYwlns1EfcLVg90H21EOLACvtfcHk0liW6wNkmz1dnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1ajtbfK&md5=8a26d0ab41970f39a109ddafff2f17b4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature09454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09454%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DBurton%26aufirst%3DE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DK.%2BY%2BJ.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DF.%26aulast%3DHiggins%26aufirst%3DB.%26aulast%3DIyer%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DK.%26aulast%3DKoehler%26aufirst%3DA.%26aulast%3DStumm%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DP.%2BS.%26aulast%3DLee%26aufirst%3DR.%2BJ.%26aulast%3DGrippo%26aufirst%3DJ.%26aulast%3DPuzanov%26aufirst%3DI.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DMcArthur%26aufirst%3DG.%26aulast%3DSosman%26aufirst%3DJ.%26aulast%3DChapman%26aufirst%3DP.%2BB.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DNathanson%26aufirst%3DK.%2BL.%26aulast%3DNolop%26aufirst%3DK.%26atitle%3DClinical%2520efficacy%2520of%2520a%2520RAF%2520inhibitor%2520needs%2520broad%2520target%2520blockade%2520in%2520BRAF-mutant%2520melanoma%26jtitle%3DNature%26date%3D2010%26volume%3D467%26spage%3D596%26epage%3D599%26doi%3D10.1038%2Fnature09454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, We.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mamo, Sh.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bremer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillette, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haass, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sproesser, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smalley, K. S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marimuthu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settachatgul, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shellooe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantwell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K. Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habets, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibrahim, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artis, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herlyn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span> </span><span class="NLM_article-title">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">3041</span><span class="NLM_x">–</span> <span class="NLM_lpage">3046</span><span class="refDoi"> DOI: 10.1073/pnas.0711741105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.0711741105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=18287029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=3041-3046&author=J.+Tsaiauthor=J.+T.+Leeauthor=We.+Wangauthor=J.+Zhangauthor=H.+Choauthor=Sh.+Mamoauthor=R.+Bremerauthor=S.+Gilletteauthor=J.+Kongauthor=N.+K.+Haassauthor=K.+Sproesserauthor=L.+Liauthor=K.+S.+M.+Smalleyauthor=D.+Fongauthor=Y.+Zhuauthor=A.+Marimuthuauthor=H.+Nguyenauthor=B.+Lamauthor=J.+Liuauthor=I.+Cheungauthor=J.+Riceauthor=Y.+Suzukiauthor=C.+Luuauthor=C.+Settachatgulauthor=R.+Shellooeauthor=J.+Cantwellauthor=S.+Kimauthor=J.+Schlessingerauthor=K.+Y.+J.+Zhangauthor=B.+L.+Westauthor=B.+Powellauthor=G.+Habetsauthor=C.+Zhangauthor=P.+N.+Ibrahimauthor=P.+Hirthauthor=D.+R.+Artisauthor=M.+Herlynauthor=G.+Bollag&title=Discovery+of+a+selective+inhibitor+of+oncogenic+B-Raf+kinase+with+potent+antimelanoma+activity&doi=10.1073%2Fpnas.0711741105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span></div><div class="casAuthors">Tsai, James; Lee, John T.; Wang, Weiru; Zhang, Jiazhong; Cho, Hanna; Mamo, Shumeye; Bremer, Ryan; Gillette, Sam; Kong, Jun; Haass, Nikolas K.; Sproesser, Katrin; Li, Ling; Smalley, Keiran S. M.; Fong, Daniel; Zhu, Yong-Liang; Marimuthu, Adhirai; Nguyen, Hoa; Lam, Billy; Liu, Jennifer; Cheung, Ivana; Rice, Julie; Suzuki, Yoshihisa; Luu, Catherine; Settachatgul, Calvin; Shellooe, Rafe; Cantwell, John; Kim, Sung-Hou; Schlessinger, Joseph; Zhang, Kam Y. J.; West, Brian L.; Powell, Ben; Habets, Gaston; Zhang, Chao; Ibrahim, Prabha N.; Hirth, Peter; Artis, Dean R.; Herlyn, Meenhard; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3041-3046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">BRAFV600E is the most frequent oncogenic protein kinase mutation known.  Furthermore, inhibitors targeting "active" protein kinases have demonstrated significant utility in the therapeutic repertoire against cancer.  Therefore, the authors pursued the development of specific kinase inhibitors targeting B-Raf, and the V600E allele in particular.  By using a structure-guided discovery approach, a potent and selective inhibitor of active B-Raf has been discovered.  PLX4720, a 7-azaindole deriv. that inhibits B-RafV600E with an IC50 of 13 nM, defines a class of kinase inhibitor with marked selectivity in both biochem. and cellular assays.  PLX4720 preferentially inhibits the active B-RafV600E kinase compared with a broad spectrum of other kinases, and potent cytotoxic effects are also exclusive to cells bearing the V600E allele.  Consistent with the high degree of selectivity, ERK phosphorylation is potently inhibited by PLX4720 in B-RafV600E-bearing tumor cell lines but not in cells lacking oncogenic B-Raf.  In melanoma models, PLX4720 induces cell cycle arrest and apoptosis exclusively in B-RafV600E-pos. cells.  In B-RafV600E-dependent tumor xenograft models, orally dosed PLX4720 causes significant tumor growth delays, including tumor regressions, without evidence of toxicity.  The work described here represents the entire discovery process, from initial identification through structural and biol. studies in animal models to a promising therapeutic for testing in cancer patients bearing B-RafV600E-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPX53IQCbDJ7Vg90H21EOLACvtfcHk0lg-jDCM_fbzGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D&md5=273b94202f99984e6eb24723d0c6141f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711741105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711741105%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DWe.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DMamo%26aufirst%3DSh.%26aulast%3DBremer%26aufirst%3DR.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHaass%26aufirst%3DN.%2BK.%26aulast%3DSproesser%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DLuu%26aufirst%3DC.%26aulast%3DSettachatgul%26aufirst%3DC.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520inhibitor%2520of%2520oncogenic%2520B-Raf%2520kinase%2520with%2520potent%2520antimelanoma%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D3041%26epage%3D3046%26doi%3D10.1073%2Fpnas.0711741105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery and design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span><span class="refDoi"> DOI: 10.1021/ci400731w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400731w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=693-704&author=D.+Joseph-McCarthyauthor=A.+J.+Campbellauthor=G.+Kernauthor=D.+Moustakas&title=Fragment-based+lead+discovery+and+design&doi=10.1021%2Fci400731w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Lead Discovery and Design</span></div><div class="casAuthors">Joseph-McCarthy, Diane; Campbell, Arthur J.; Kern, Gunther; Moustakas, Demetri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">693-704</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery and design has and continues to show increasing promise in drug discovery.  In this article, the current state of the art in terms of hot-spot characterization, fragment screening techniques, and fragment-based design is discussed.  Three overall fragment-based lead generation strategies are explored and involve the chem. biol. characterization of biol. targets via fragment screening, fragment screening as a complementary approach to high-throughput screening of drug-like compds., and direct fragment-based drug discovery, resp.  The evolution and development of fragment libraries is described.  With an emphasis on computational approaches and the strategies applied at AstraZeneca, the review illustrates how integration of data from one regime can inform the design of expts. in the other, ultimately leading to the discovery of high quality chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCa5krM_w737Vg90H21EOLACvtfcHk0lg-jDCM_fbzGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Gns78%253D&md5=31b72219dfd8cec7e3612de1b61f8150</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fci400731w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400731w%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DCampbell%26aufirst%3DA.%2BJ.%26aulast%3DKern%26aufirst%3DG.%26aulast%3DMoustakas%26aufirst%3DD.%26atitle%3DFragment-based%2520lead%2520discovery%2520and%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D693%26epage%3D704%26doi%3D10.1021%2Fci400731w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ochiana, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollastri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">E2585</span><span class="NLM_x">–</span> <span class="NLM_lpage">E2594</span><span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=25918377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-E2594&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+deconstruction%3A+Why+some+fragment+binding+positions+are+conserved+and+others+are+not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Jehle, Sefan; Luo, Lingqi; Ochiana, Stefan O.; Jones, Elizabeth V.; Pollastri, Michael; Allen, Karen N.; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E2585-E2594</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) relies on the premise that the fragment binding mode will be conserved on subsequent expansion to a larger ligand.  However, no general condition has been established to explain when fragment binding modes will be conserved.  We show that a remarkably simple condition can be developed in terms of how fragments coincide with binding energy hot spots-regions of the protein where interactions with a ligand contribute substantial binding free energy-the locations of which can easily be detd. computationally.  Because a substantial fraction of the free energy of ligand binding comes from interacting with the residues in the energetically most important hot spot, a ligand moiety that sufficiently overlaps with this region will retain its location even when other parts of the ligand are removed.  This hypothesis is supported by eight case studies.  The condition helps identify whether a protein is suitable for FBDD, predicts the size of fragments required for screening, and dets. whether a fragment hit can be extended into a higher affinity ligand.  Our results show that ligand binding sites can usefully be thought of in terms of an anchor site, which is the top-ranked hot spot and dominates the free energy of binding, surrounded by a no. of weaker satellite sites that confer improved affinity and selectivity for a particular ligand and that it is the intrinsic binding potential of the protein surface that dets. whether it can serve as a robust binding site for a suitably optimized ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH6jbOut_iv7Vg90H21EOLACvtfcHk0lgJSGtiP-aQ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D&md5=d56b18fb4e4befde24037b07e52d8b62</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520deconstruction%253A%2520Why%2520some%2520fragment%2520binding%2520positions%2520are%2520conserved%2520and%2520others%2520are%2520not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE2585%26epage%3DE2594%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span> </span><span class="NLM_article-title">Molecular complexity and its impact on the probability of finding leads for drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">856</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span><span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+complexity+and+its+impact+on+the+probability+of+finding+leads+for+drug+discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0lgJSGtiP-aQ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520complexity%2520and%2520its%2520impact%2520on%2520the%2520probability%2520of%2520finding%2520leads%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebTkQdLJNytSrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Rawlins, P.</span><span> </span><span class="NLM_article-title">Current trends in label-free technologies</span> <span class="citation_source-journal">Drug Discovery World</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=17-26&author=P.+Rawlins&title=Current+trends+in+label-free+technologies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRawlins%26aufirst%3DP.%26atitle%3DCurrent%2520trends%2520in%2520label-free%2520technologies%26jtitle%3DDrug%2520Discovery%2520World%26date%3D2010%26volume%3D11%26spage%3D17%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bartoli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fincham, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fattori, D.</span><span> </span><span class="NLM_article-title">Fragment-based drug design: combining philosophy with technology</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">429</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=422-429&author=S.+Bartoliauthor=C.+Finchamauthor=D.+Fattori&title=Fragment-based+drug+design%3A+combining+philosophy+with+technology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBartoli%26aufirst%3DS.%26aulast%3DFincham%26aufirst%3DC.%26aulast%3DFattori%26aufirst%3DD.%26atitle%3DFragment-based%2520drug%2520design%253A%2520combining%2520philosophy%2520with%2520technology%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2010%26volume%3D10%26spage%3D422%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: A fragment-based approach using biophysical methods</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4992</span><span class="NLM_x">–</span> <span class="NLM_lpage">5000</span><span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+A+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0li0aF1BSFFyLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520A%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span><span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0li0aF1BSFFyLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Schiebel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radeva, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krimmer, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stieler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrmann, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huschmann, F. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Six biophysical screening methods miss a large proportion of crystallographically discovered fragment hits: A case study</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1693</span><span class="NLM_x">–</span> <span class="NLM_lpage">1701</span><span class="refDoi"> DOI: 10.1021/acschembio.5b01034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1693-1701&author=J.+Schiebelauthor=N.+Radevaauthor=S.+G.+Krimmerauthor=X.+Wangauthor=M.+Stielerauthor=F.+R.+Ehrmannauthor=K.+Fuauthor=A.+Metzauthor=F.+U.+Huschmannauthor=M.+S.+Weissauthor=U.+Muellerauthor=A.+Heineauthor=G.+Klebe&title=Six+biophysical+screening+methods+miss+a+large+proportion+of+crystallographically+discovered+fragment+hits%3A+A+case+study&doi=10.1021%2Facschembio.5b01034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study</span></div><div class="casAuthors">Schiebel, Johannes; Radeva, Nedyalka; Krimmer, Stefan G.; Wang, Xiaojie; Stieler, Martin; Ehrmann, Frederik R.; Fu, Kan; Metz, Alexander; Huschmann, Franziska U.; Weiss, Manfred S.; Mueller, Uwe; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1693-1701</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based lead discovery (FBLD) has become a pillar in drug development.  Typical applications of this method comprise at least two biophys. screens as prefilter and a follow-up crystallog. expt. on a subset of fragments.  Clearly, structural information is pivotal in FBLD, but a key question is whether such a screening cascade strategy will retrieve the majority of fragment-bound structures.  We therefore set out to screen 361 fragments for binding to endothiapepsin, a representative of the challenging group of aspartic proteases, employing six screening techniques and crystallog. in parallel.  Crystallog. resulted in the very high no. of 71 structures.  Yet alarmingly, 44% of these hits were not detected by any biophys. screening approach.  Moreover, any screening cascade, building on the results from two or more screening methods, would have failed to predict at least 73% of these hits.  We thus conclude that, at least in the present case, the frequently applied biophys. prescreening filters deteriorate the no. of possible X-ray hits while only the immediate use of crystallog. enables exhaustive retrieval of a max. of fragment structures, which represent a rich source guiding hit-to-lead-to-drug evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgOpXVyCfeiLVg90H21EOLACvtfcHk0liFIcsL23_nMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D&md5=30f16cf75361d0ca11e7b75ad796d207</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01034%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DStieler%26aufirst%3DM.%26aulast%3DEhrmann%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSix%2520biophysical%2520screening%2520methods%2520miss%2520a%2520large%2520proportion%2520of%2520crystallographically%2520discovered%2520fragment%2520hits%253A%2520A%2520case%2520study%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1693%26epage%3D1701%26doi%3D10.1021%2Facschembio.5b01034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Miranker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Functionality maps of binding sites: a multiple copy simultaneous search method</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1002/prot.340110104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fprot.340110104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1961699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=29-34&author=A.+Mirankerauthor=M.+Karplus&title=Functionality+maps+of+binding+sites%3A+a+multiple+copy+simultaneous+search+method&doi=10.1002%2Fprot.340110104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Functionality maps of binding sites:  a multiple copy simultaneous search method</span></div><div class="casAuthors">Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-34</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">A new method is proposed for detg. energetically favorable positions and orientations for functional groups on the surface of proteins with known 3-dimensional structure.  Some 1000-5000 copies of a functional group are randomly placed in the site and subjected to simultaneous energy minimization and/or quenched mol. dynamics.  The resulting functionality maps of a protein receptor site, which can take account of its flexibility, can be used for the anal. of protein ligand interactions and rational drug design.  Application of the method to the sialic acid binding site of the influenza coat protein, hemagglutinin, yields functional group min. that correspond with those of the ligand in a cocrystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5im8BqJfF37Vg90H21EOLACvtfcHk0liFIcsL23_nMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D&md5=34e0a0c36899bfeabcbdfd4790f97a1e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fprot.340110104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340110104%26sid%3Dliteratum%253Aachs%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DFunctionality%2520maps%2520of%2520binding%2520sites%253A%2520a%2520multiple%2520copy%2520simultaneous%2520search%2520method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1991%26volume%3D11%26spage%3D29%26epage%3D34%26doi%3D10.1002%2Fprot.340110104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">SuperStar: a knowledge-based approach for identifying interaction sites in proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1006/jmbi.1999.2809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1006%2Fjmbi.1999.2809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10369784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=1999&pages=1093-1108&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=R.+Taylor&title=SuperStar%3A+a+knowledge-based+approach+for+identifying+interaction+sites+in+proteins&doi=10.1006%2Fjmbi.1999.2809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: A Knowledge-based Approach for Identifying Interaction Sites in Proteins</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1093-1108</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">An empirical method for identifying interaction sites in proteins is described and validated.  The method is based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures.  These data are used in the form of scatterplots that show the exptl. obsd. distribution of one functional group (the "contact group" or "probe") around another.  A template mol. (e.g. a protein binding site) is broken down into structure fragments and the scatterplots, showing the distribution of a chosen probe around these structure fragments, are superimposed on the corresponding parts of the template.  The scatterplots are then translated into a three-dimensional map that shows the propensity of the probe at different positions around the template mol.  The method is illustrated for L-arabinose-binding protein, complexed with L-arabinose and with D-fucose, and for dihydrofolate reductase complexed with methotrexate.  The method is validated on 122 x-ray structures of protein-ligand complexes.  For all the binding sites of these proteins, propensity maps are generated for four different probes: a charged NH+3 nitrogen, a carbonyl oxygen, a hydroxyl oxygen and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  For 74% of these ligand atoms (84% of the solvent-inaccessible ones) the calcd. propensity of the matching probe at the exptl. positions is higher than expected by chance.  For 68% of the atoms (82% of the solvent-inaccessible ones) the propensity of the matching probe is higher than that of the other three probes.  These results indicate that the approach generally gives good predictions for protein-ligand interactions.  The potential applications of the propensity maps range from an aid in manual docking and structure-based drug design to their use in pharmacophore development.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9QVkFGAwFt7Vg90H21EOLACvtfcHk0liFIcsL23_nMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D&md5=72e09c54c9a631a8271fbe72f2f431b1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.2809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.2809%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DSuperStar%253A%2520a%2520knowledge-based%2520approach%2520for%2520identifying%2520interaction%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D289%26spage%3D1093%26epage%3D1108%26doi%3D10.1006%2Fjmbi.1999.2809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">SuperStar: Improved knowledge-based interaction fields for protein binding sites</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">307</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span><span class="refDoi"> DOI: 10.1006/jmbi.2001.4452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1006%2Fjmbi.2001.4452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=11273705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2001&pages=841-859&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=P.+Watsonauthor=V.+J.+Gilletauthor=P.+Willett&title=SuperStar%3A+Improved+knowledge-based+interaction+fields+for+protein+binding+sites&doi=10.1006%2Fjmbi.2001.4452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: Improved knowledge-based interaction fields for protein binding sites</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Watson, Paul; Gillet, Val; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">841-859</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">SuperStar is an empirical method for identifying interaction sites in proteins, based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures, taken from the IsoStar database.  We describe recent modifications and addns. to SuperStar, validating the results on a test set of 122 x-ray structures of protein-ligand complexes.  In this validation, propensity maps are generated for all the binding sites of these proteins, using four different probes: a charged NH+3 nitrogen atom, a carbonyl oxygen atom, a hydroxyl oxygen atom and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  A peak-searching algorithm is introduced that highlights potential interaction hot spots.  For the three hydrogen-bonding probes - NH+3 nitrogen atom, carbonyl oxygen atom and hydroxyl oxygen atom - the av. distance from the ligand atom to the nearest SuperStar peak is 1.0-1.2 Å (0.8-1.0 Å for solvent-inaccessible ligand atoms).  For the Me carbon atom probe, this distance is about 1.5 Å, probably because interactions to Me groups are much less directional. The most important addn. to SuperStar is the enabling of propensity maps around metal centers - Ca2+, Mg2+ and Zn2+ - in protein binding sites.  The results are validated on a test set of 24 protein-ligand complexes that have a metal ion in their binding site.  Coordination geometries are derived automatically, using only the protein atoms that coordinate to the metal ion.  The correct coordination geometry is derived in approx. 75 % of the cases.  If the derived geometry is assumed during the SuperStar calcn., the av. distance from a ligand atom coordinating to the metal ion to the nearest peak in the propensity map for an oxygen probe is 0.87(7) Å.  If the correct coordination geometry is imposed, this distance reduces to 0.59(7) Å.  This indicates that the SuperStar predictions around metal-binding sites are at least as good as those around other protein groups. Using clustering techniques, a non-redundant set of probes is selected from the set of probes available in the IsoStar database.  The performance in SuperStar of all these probes is tested on the test set of protein-ligand complexes.  With the exception of the "ether oxygen" probe and the "any NH+" probe, all new probes perform as well as the four probes introduced first.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCZ27rmWmxMbVg90H21EOLACvtfcHk0liJBO7ztsTjUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D&md5=026cdc7beaec6446307bbcc36a7e813f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.4452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.4452%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DWatson%26aufirst%3DP.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSuperStar%253A%2520Improved%2520knowledge-based%2520interaction%2520fields%2520for%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D841%26epage%3D859%26doi%3D10.1006%2Fjmbi.2001.4452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarsi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apostolakis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrhardt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Exhaustive docking of molecular fragments with electrostatic solvation</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19991001)37:1<88::AID-PROT9>3.0.CO;2-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10451553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK1MXls12qsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1999&pages=88-105&author=N.+Majeuxauthor=M.+Scarsiauthor=J.+Apostolakisauthor=C.+Ehrhardtauthor=A.+Caflisch&title=Exhaustive+docking+of+molecular+fragments+with+electrostatic+solvation&doi=10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Exhaustive docking of molecular fragments with electrostatic solvation</span></div><div class="casAuthors">Majeux, Nicolas; Scarsi, Marco; Apostolakis, Joannis; Ehrhardt, Claus; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-105</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method is presented for docking mol. fragments to a rigid protein with evaluation of the binding energy.  Polar fragments are docked with at least one hydrogen bond with the protein while apolar fragments are positioned in the hydrophobic pockets.  The electrostatic contribution to the binding energy, which consists of screened intermol. energy and protein and fragment desolvation terms, is evaluated efficiently by a numerical approach based on the continuum dielec. approxn.  The latter is also used to predetermine the hydrophobic pockets of the protein by rolling a low dielec. sphere over the protein surface and calcg. the electrostatic desolvation of the protein and van der Waals interaction energy.  The method was implemented in the program SEED (solvation energy for exhaustive docking).  The SEED continuum electrostatic approach has been successfully validated by a comparison with finite difference solns. of the Poisson equation for more than 2,500 complexes of small mols. with thrombin and the monomer of HIV-1 aspartic proteinase.  The fragments docked by SEED in the active site of thrombin reproduce the structural features of the interaction patterns between known inhibitors and thrombin.  Moreover, the combinatorial connection of these fragments yields a no. of compds. that are very similar to potent inhibitors of thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOBGz0snZSsrVg90H21EOLACvtfcHk0liJBO7ztsTjUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXls12qsL8%253D&md5=354f9e32fd0b2ff602f09fd0b81a057e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819991001%252937%253A1%253C88%253A%253AAID-PROT9%253E3.0.CO%253B2-O%26sid%3Dliteratum%253Aachs%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DScarsi%26aufirst%3DM.%26aulast%3DApostolakis%26aufirst%3DJ.%26aulast%3DEhrhardt%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DExhaustive%2520docking%2520of%2520molecular%2520fragments%2520with%2520electrostatic%2520solvation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1999%26volume%3D37%26spage%3D88%26epage%3D105%26doi%3D10.1002%2F%28SICI%291097-0134%2819991001%2937%3A1%3C88%3A%3AAID-PROT9%3E3.0.CO%3B2-O" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Majeux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarsi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Efficient electrostatic solvation model for protein-fragment docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">256</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span><span class="refDoi"> DOI: 10.1002/1097-0134(20010201)42:2<256::AID-PROT130>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=11119650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvFyrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2001&pages=256-268&author=N.+Majeuxauthor=M.+Scarsiauthor=A.+Caflisch&title=Efficient+electrostatic+solvation+model+for+protein-fragment+docking&doi=10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient electrostatic solvation model for protein-fragment docking</span></div><div class="casAuthors">Majeux, Nicolas; Scarsi, Marco; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">256-268</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A method is presented for the fast evaluation of the binding energy of a protein-small mol. complex with electrostatic solvation.  It makes use of a fast preprocessing step based on the assumption that the main contribution to electrostatic desolvation upon ligand binding originates from the displacement of the first shell of water mols.  For a rigid protein, the precomputation of the energy contributions on a set of grids allows the estn. of the energy in soln. of about 300 protein-fragment binding modes per s on a personal computer.  The docking procedure is applied to five rigid binding sites whose size ranges from 17 residues to a whole protein of 107 amino acids.  Using a library of 70 mainly rigid mols., known micromolar inhibitors or close analogs are docked and prioritized correctly.  The docking based rank-ordering of the library requires about 5 h and is proposed as a complementary approach to structure-activity relationships by NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE2YlkKSp9MLVg90H21EOLACvtfcHk0lhvLrr19L_lPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvFyrug%253D%253D&md5=83f3ebb9884c081c5ebfef839ddabc54</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1097-0134%252820010201%252942%253A2%253C256%253A%253AAID-PROT130%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DMajeux%26aufirst%3DN.%26aulast%3DScarsi%26aufirst%3DM.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DEfficient%2520electrostatic%2520solvation%2520model%2520for%2520protein-fragment%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2001%26volume%3D42%26spage%3D256%26epage%3D268%26doi%3D10.1002%2F1097-0134%2820010201%2942%3A2%3C256%3A%3AAID-PROT130%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Hung, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silvestre, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C. I.</span><span> </span><span class="NLM_article-title">Application of fragment growing and fragment linking to the discovery of inhibitors of mycobacterium tuberculosis pantothenate synthetase</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8452</span><span class="NLM_x">–</span> <span class="NLM_lpage">8456</span><span class="refDoi"> DOI: 10.1002/anie.200903821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fanie.200903821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlWmur%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8452-8456&author=A.+W.+Hungauthor=H.+L.+Silvestreauthor=S.+Wenauthor=A.+Ciulliauthor=T.+L.+Blundellauthor=C.+I.+Abell&title=Application+of+fragment+growing+and+fragment+linking+to+the+discovery+of+inhibitors+of+mycobacterium+tuberculosis+pantothenate+synthetase&doi=10.1002%2Fanie.200903821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Fragment Growing and Fragment Linking to the Discovery of Inhibitors of Mycobacterium tuberculosis Pantothenate Synthetase</span></div><div class="casAuthors">Hung, Alvin W.; Silvestre, H. Leonardo; Wen, Shijun; Ciulli, Alessio; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">8452-8456, S8452/1-S8452/26</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Despite the fact that nearly 2 million people a year are killed by Mycobacterium tuberculosis (TB), we still lack a robust pipeline of potential therapies to combat this burden.  Fragment-based approaches have provided a new paradigm in the development of high-quality chem. ligands and have great potential to contribute as starting points in the development of drugs against TB.  To date, studies on pantothenate synthetase (PS) inhibition have focused on the synthesis of analogs of the pantoyl adenylate reaction intermediate or the identification of hits from high-throughput screening.  We describe a combination of fragment-growing and fragment-linking approaches that led to the discovery of a new series of inhibitors of M. tuberculosis PS for the further validation of PS as a potential drug target for TB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjhska_U-KXbVg90H21EOLACvtfcHk0lhvLrr19L_lPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlWmur%252FK&md5=821f86e9e1d7cbd848741962e3ac47dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.200903821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200903821%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DA.%2BW.%26aulast%3DSilvestre%26aufirst%3DH.%2BL.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%2BI.%26atitle%3DApplication%2520of%2520fragment%2520growing%2520and%2520fragment%2520linking%2520to%2520the%2520discovery%2520of%2520inhibitors%2520of%2520mycobacterium%2520tuberculosis%2520pantothenate%2520synthetase%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D8452%26epage%3D8456%26doi%3D10.1002%2Fanie.200903821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Chung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bianchet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amzel, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stivers, J. T.</span><span> </span><span class="NLM_article-title">Impact of linker strain and flexibility in the design of a fragment-based inhibitor</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1038/nchembio.163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnchembio.163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19396178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltValtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=407-413&author=S.+Chungauthor=J.+B.+Parkerauthor=M.+Bianchetauthor=L.+M.+Amzelauthor=J.+T.+Stivers&title=Impact+of+linker+strain+and+flexibility+in+the+design+of+a+fragment-based+inhibitor&doi=10.1038%2Fnchembio.163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of linker strain and flexibility in the design of a fragment-based inhibitor</span></div><div class="casAuthors">Chung, Suhman; Parker, Jared B.; Bianchet, Mario; Amzel, L. Mario; Stivers, James T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">407-413</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The linking together of mol. fragments that bind to adjacent sites on an enzyme can lead to high-affinity inhibitors.  Ideally, this strategy would use linkers that do not perturb the optimal binding geometries of the fragments and do not have excessive conformational flexibility that would increase the entropic penalty of binding.  In reality, these aims are seldom realized owing to limitations in linker chem.  Here the energetic and structural effects of rigid and flexible linkers on the binding of a fragment-based inhibitor of human uracil DNA glycosylase were systematically explored.  Anal. of the free energies of binding in combination with co-crystal structures shows that the flexibility and strain of a given linker can have a substantial impact on binding affinity even when the binding fragments are optimally positioned.  Such effects are not apparent from inspection of structures and underscore the importance of linker optimization in fragment-based drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo22ydPLyrVsrVg90H21EOLACvtfcHk0lh1HNLDYdTr7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltValtLw%253D&md5=5fd9de328a39f6972d7ba376b5354851</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.163%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DS.%26aulast%3DParker%26aufirst%3DJ.%2BB.%26aulast%3DBianchet%26aufirst%3DM.%26aulast%3DAmzel%26aufirst%3DL.%2BM.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26atitle%3DImpact%2520of%2520linker%2520strain%2520and%2520flexibility%2520in%2520the%2520design%2520of%2520a%2520fragment-based%2520inhibitor%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D407%26epage%3D413%26doi%3D10.1038%2Fnchembio.163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Böhm, H. J.</span><span> </span><span class="NLM_article-title">LUDI: rule-based automatic design of new substituents for enzyme inhibitor leads</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">606</span><span class="refDoi"> DOI: 10.1007/BF00126217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1007%2FBF00126217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1291628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADyaK3s7nvF2jtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1992&pages=593-606&author=H.+J.+B%C3%B6hm&title=LUDI%3A+rule-based+automatic+design+of+new+substituents+for+enzyme+inhibitor+leads&doi=10.1007%2FBF00126217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">LUDI: rule-based automatic design of new substituents for enzyme inhibitor leads</span></div><div class="casAuthors">Bohm H J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">593-606</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">Recent advances in a new method for the de novo design of enzyme inhibitors are reported.  A new set of rules to define the possible nonbonded contacts between protein and ligand is presented.  This method was derived from published statistical analyses of nonbonded contacts in crystal packings of organic molecules and has been implemented in the recently described computer program LUDI.  Moreover, LUDI can now append a new substituent onto an already existing ligand.  Applications are reported for the design of inhibitors of HIV protease and dihydrofolate reductase.  The results demonstrate that LUDI is indeed capable of designing new ligands with improved binding when compared to the reference compound.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSuJ_iBRXZ5tz0O18WR3iAofW6udTcc2eZg8bejzfQPPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3s7nvF2jtg%253D%253D&md5=56422267240eb98f29e7314fdadd5bed</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2FBF00126217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00126217%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.%2BJ.%26atitle%3DLUDI%253A%2520rule-based%2520automatic%2520design%2520of%2520new%2520substituents%2520for%2520enzyme%2520inhibitor%2520leads%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1992%26volume%3D6%26spage%3D593%26epage%3D606%26doi%3D10.1007%2FBF00126217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myatt, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zsoldos, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, A. P.</span><span> </span><span class="NLM_article-title">SPROUT, HIPPO and CAESA: Tools for de novo structure generation and estimation of synthetic accessibility</span> <span class="citation_source-journal">Perspect. Drug Discovery Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1007/BF02174466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1007%2FBF02174466" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1995&pages=34-50&author=V.+J.+Gilletauthor=G.+Myattauthor=Z.+Zsoldosauthor=A.+P.+Johnson&title=SPROUT%2C+HIPPO+and+CAESA%3A+Tools+for+de+novo+structure+generation+and+estimation+of+synthetic+accessibility&doi=10.1007%2FBF02174466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2FBF02174466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02174466%26sid%3Dliteratum%253Aachs%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DMyatt%26aufirst%3DG.%26aulast%3DZsoldos%26aufirst%3DZ.%26aulast%3DJohnson%26aufirst%3DA.%2BP.%26atitle%3DSPROUT%252C%2520HIPPO%2520and%2520CAESA%253A%2520Tools%2520for%2520de%2520novo%2520structure%2520generation%2520and%2520estimation%2520of%2520synthetic%2520accessibility%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D1995%26volume%3D3%26spage%3D34%26epage%3D50%26doi%3D10.1007%2FBF02174466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Pearlman, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">CONCERTS: Dynamic connection of fragments as an approach to de novo ligand design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1651</span><span class="NLM_x">–</span> <span class="NLM_lpage">1663</span><span class="refDoi"> DOI: 10.1021/jm950792l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm950792l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK28XhslGqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=1651-1663&author=D.+A.+Pearlmanauthor=M.+A.+Murcko&title=CONCERTS%3A+Dynamic+connection+of+fragments+as+an+approach+to+de+novo+ligand+design&doi=10.1021%2Fjm950792l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">CONCERTS: Dynamic Connection of Fragments as an Approach to de Novo Ligand Design</span></div><div class="casAuthors">Pearlman, David A.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1651-63</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have implemented and tested a new approach to de novo ligand design, CONCERTS (creation of novel compds. by evaluation of residues at target sites).  In this method, each member of a user-defined set of fragments is allowed to move independently about a target active site during a mol. dynamics simulation.  This allows the fragments to sample various low-energy orientations.  When the geometry between proximal fragments is appropriate, bonds can be formed between the fragments.  In this fashion, larger mols. can be built.  The bonding arrangement can subsequently be changed-breaking bonds between chosen fragment pairs and forming them between other pairs-if the overall process creates lower energy mols.  We have tested this method with various mixes of fragments against the active sites of the FK506 binding protein (FKBP-12) and HIV-1 aspartyl protease.  In several cases, CONCERTS suggests ligands which are in surprisingly good agreement with known inhibitors of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzo_xUabLvOrVg90H21EOLACvtfcHk0ljVcgmNcN5tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhslGqsr4%253D&md5=b7e36d919e4ebf403014a4d6ebf5cce9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm950792l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm950792l%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DCONCERTS%253A%2520Dynamic%2520connection%2520of%2520fragments%2520as%2520an%2520approach%2520to%2520de%2520novo%2520ligand%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D1651%26epage%3D1663%26doi%3D10.1021%2Fjm950792l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Maass, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Recore: A fast and versatile method for scaffold hopping based on small molecule crystal structure conformations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">390</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span><span class="refDoi"> DOI: 10.1021/ci060094h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci060094h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=390-399&author=P.+Maassauthor=T.+Schulz-Gaschauthor=M.+Stahlauthor=M.+Rarey&title=Recore%3A+A+fast+and+versatile+method+for+scaffold+hopping+based+on+small+molecule+crystal+structure+conformations&doi=10.1021%2Fci060094h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recore: A Fast and Versatile Method for Scaffold Hopping Based on Small Molecule Crystal Structure Conformations</span></div><div class="casAuthors">Maass, Patrick; Schulz-Gasch, Tanja; Stahl, Martin; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">390-399</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Replacing central elements of known active structures is a common procedure to enter new compd. classes.  Different computational methods have already been developed to help with this task, varying in the description of possible replacements, the query input, and the similarity measure used.  In this paper, a novel approach for scaffold replacement and a corresponding software tool, called Recore, is introduced.  In contrast to prior methods, the authors main objective was to combine the following three properties in one tool: to avoid structures with strained conformations, to enable the exploration of large search spaces, and to allow interactive use through short response times.  The authors introduce a new technique employing 3D fragments generated by combinatorial enumeration of cuts.  It allows focusing on fragments suitable for scaffold replacement while retaining conformational information of the corresponding crystal structures.  Based on this idea, the authors present an algorithm utilizing a geometric rank searching approach.  Given a geometric arrangement of two or three exit vectors and addnl. pharmacophore features, the algorithm finds fragments fulfilling all these constraints ordered by increasing deviation from the query constraints.  For the validation of the approach, three different drug design scenarios have been used.  The results obtained show that the authors approach is able to propose new valid scaffold topologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpphxap_w-jZbVg90H21EOLACvtfcHk0ljVcgmNcN5tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D&md5=e9d4e277da49f4ffe2c21472990c2387</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci060094h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci060094h%26sid%3Dliteratum%253Aachs%26aulast%3DMaass%26aufirst%3DP.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DRecore%253A%2520A%2520fast%2520and%2520versatile%2520method%2520for%2520scaffold%2520hopping%2520based%2520on%2520small%2520molecule%2520crystal%2520structure%2520conformations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D390%26epage%3D399%26doi%3D10.1021%2Fci060094h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Lauri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartlett, P. A.</span><span> </span><span class="NLM_article-title">CAVEAT: A program to facilitate the design of organic molecules</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.1007/BF00124349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1007%2FBF00124349" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=51-66&author=G.+Lauriauthor=P.+A.+Bartlett&title=CAVEAT%3A+A+program+to+facilitate+the+design+of+organic+molecules&doi=10.1007%2FBF00124349"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2FBF00124349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124349%26sid%3Dliteratum%253Aachs%26aulast%3DLauri%26aufirst%3DG.%26aulast%3DBartlett%26aufirst%3DP.%2BA.%26atitle%3DCAVEAT%253A%2520A%2520program%2520to%2520facilitate%2520the%2520design%2520of%2520organic%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D51%26epage%3D66%26doi%3D10.1007%2FBF00124349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">BREED: Generating novel inhibitors through hybridization of known ligands. Application to CDK2, P38, and HIV protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2768</span><span class="NLM_x">–</span> <span class="NLM_lpage">2775</span><span class="refDoi"> DOI: 10.1021/jm030543u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030543u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2768-2775&author=A.+C.+Pierceauthor=G.+Raoauthor=G.+W.+Bemis&title=BREED%3A+Generating+novel+inhibitors+through+hybridization+of+known+ligands.+Application+to+CDK2%2C+P38%2C+and+HIV+protease&doi=10.1021%2Fjm030543u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">BREED: Generating Novel Inhibitors through Hybridization of Known Ligands. Application to CDK2, P38, and HIV Protease</span></div><div class="casAuthors">Pierce, Albert C.; Rao, Govinda; Bemis, Guy W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2768-2775</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this work we describe BREED, a method for the generation of novel inhibitors from structures of known ligands bound to a common target.  The method is essentially an automation of the common medicinal chem. practice of joining fragments of two known ligands to generate a new inhibitor.  The ligand-bound target structures are overlaid, all overlapping bonds in all pairs of ligands are found, and the fragments on each side of each matching bond are swapped to generate the new mols.  Since the method is automated, it can be applied recursively to generate all possible combinations of known ligands.  In an application of this method to HIV protease inhibitors and protein kinase inhibitors, hundreds of new mol. structures were generated.  These included known inhibitor scaffolds not included in the initial set, entirely novel scaffolds, and novel substituents on known scaffolds.  The method is fast, and since all of the ligand functional groups are known to bind the target in the precise position and orientation present in the novel ligand, the success rate of this method should be superior to more traditional de novo design techniques.  In an era of increasingly high-throughput structural biol., such methods for high-throughput utilization of structural information will become increasingly valuable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqskNA5h73CbbVg90H21EOLACvtfcHk0liet-ldLAPDYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjt1ajurk%253D&md5=e0bec9082fab05b694742498b3f41549</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm030543u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030543u%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DRao%26aufirst%3DG.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DBREED%253A%2520Generating%2520novel%2520inhibitors%2520through%2520hybridization%2520of%2520known%2520ligands.%2520Application%2520to%2520CDK2%252C%2520P38%252C%2520and%2520HIV%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2768%26epage%3D2775%26doi%3D10.1021%2Fjm030543u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Dey, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Fragment-based de novo ligand design by multi-objective evolutionary optimization</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1021/ci700424b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700424b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFWltbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=679-690&author=F.+Deyauthor=A.+Caflisch&title=Fragment-based+de+novo+ligand+design+by+multi-objective+evolutionary+optimization&doi=10.1021%2Fci700424b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based de novo ligand design by multiobjective evolutionary optimization</span></div><div class="casAuthors">Dey, Fabian; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">679-690</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GANDI (Genetic Algorithm-based de Novo Design of Inhibitors) is a computational tool for automatic fragment-based design of mols. within a protein binding site of known structure.  A genetic algorithm and a tabu search act in concert to join predocked fragments with a user-supplied list of fragments.  A novel feature of GANDI is the simultaneous optimization of force field energy and a term enforcing 2D-similarity to known inhibitor(s) or 3D-overlap to known binding mode(s).  Scaffold hopping can be promoted by tuning the relative wts. of these terms.  The performance of GANDI is tested on cyclin-dependent kinase 2 (CDK2) using a library of about 14 000 fragments and the binding mode of a known oxindole inhibitor to bias the design.  Top ranking GANDI mols. are involved in one to three hydrogen bonds with the backbone polar groups in the hinge region of CDK2, an interaction pattern obsd. in potent kinase inhibitors.  Notably, a GANDI mol. with very favorable predicted binding affinity shares a 2-N-phenyl-1,3-thiazole-2,4-diamine moiety with a known nanomolar inhibitor of CDK2.  Importantly, mols. with a favorable GANDI score are synthetic accessible.  In fact, eight of the 1809 mols. designed by GANDI for CDK2 are found in the ZINC database of com. available compds. which also contains about 600 compds. with identical scaffolds as those in the top ranking GANDI mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrprXa_ftoRxrVg90H21EOLACvtfcHk0liet-ldLAPDYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFWltbc%253D&md5=fed0972e4a1c2aedcb2aadeae58c1d84</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci700424b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700424b%26sid%3Dliteratum%253Aachs%26aulast%3DDey%26aufirst%3DF.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DFragment-based%2520de%2520novo%2520ligand%2520design%2520by%2520multi-objective%2520evolutionary%2520optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D679%26epage%3D690%26doi%3D10.1021%2Fci700424b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span>BROOD, version <span class="NLM_edition">1.6.3.1</span>; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span> (accessed Nov 15, 2014).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BROOD%2C+version+1.6.3.1%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM+%28accessed+Nov+15%2C+2014%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DBROOD%26pub%3DOpenEye%2520Scientific%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Pirard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span> </span><span class="NLM_article-title">Evaluation of a semi-automated workflow for fragment growing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">180</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1021/ci5006355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5006355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=180-193&author=B.+Pirardauthor=P.+Ertl&title=Evaluation+of+a+semi-automated+workflow+for+fragment+growing&doi=10.1021%2Fci5006355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a Semi-Automated Workflow for Fragment Growing</span></div><div class="casAuthors">Pirard, Bernard; Ertl, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">180-193</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intelligent Automatic Design (IADE) is an expert system developed at Novartis to identify nonclassical bioisosteres.  In addn. to bioisostere searching, one could also use IADE to grow a fragment bound to a protein.  Here we report an evaluation of IADE as a tool for fragment growing.  Three examples from the literature served as test cases.  In all three cases, IADE generated close analogs of the published compds. and reproduced their crystallog. binding modes.  This exercise validated the use of the IADE system for fragment growing.  We have also gained experience in optimizing the performance of IADE for this type of application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu3eXbqlbVxbVg90H21EOLACvtfcHk0lhWDbXn7vnYYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrs7zJ&md5=607fc092518e799ba097242978cd31c5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci5006355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5006355%26sid%3Dliteratum%253Aachs%26aulast%3DPirard%26aufirst%3DB.%26aulast%3DErtl%26aufirst%3DP.%26atitle%3DEvaluation%2520of%2520a%2520semi-automated%2520workflow%2520for%2520fragment%2520growing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D180%26epage%3D193%26doi%3D10.1021%2Fci5006355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Degen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wegscheid-Gerlach, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaliani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">On the art of compiling and using ’drug-like’ chemical fragment spaces</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1503</span><span class="NLM_x">–</span> <span class="NLM_lpage">1507</span><span class="refDoi"> DOI: 10.1002/cmdc.200800178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fcmdc.200800178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=18792903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1503-1507&author=J.+Degenauthor=C.+Wegscheid-Gerlachauthor=A.+Zalianiauthor=M.+Rarey&title=On+the+art+of+compiling+and+using+%E2%80%99drug-like%E2%80%99+chemical+fragment+spaces&doi=10.1002%2Fcmdc.200800178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">On the art of compiling and using 'drug-like' chemical fragment spaces</span></div><div class="casAuthors">Degen, Joerg; Wegscheid-Gerlach, Christof; Zaliani, Andrea; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1503-1507</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">To improve existing approaches for the automatic decompn. of mols. into fragments, the authors compiled a new and more elaborate set of rules for the breaking of retrosynthetically interesting chem. substructures (BRICS) and used this for obtaining fragments from biol. active compds. and vendor catalog sources.  Based on this, the authors compiled corresponding fragment spaces by specifying a complementary set of rules for the recombination of the corresponding chem. motifs.  Furthermore, the authors put considerable effort into compiling a set of high-quality, high-performance, and in contrast to all other approaches, publicly available fragments that are meant to serve as a possible basis for various mol. design objectives and techniques.  Overall, this work led the authors to more comprehensive sets of fragments, and the corresponding fragment spaces show a significant increase in performance over existing methods.  Moreover, by incorporating fragments from vendor catalog sources, the performance can be increased even further.  The BRICS model consists of 16 chem. environments indicated by link atoms of different types.  A shredding example for nexavar (Sorafenib), showing the key steps in the generation of the fragments is shown as well as fragment prototypes used in BRICS.  A case study showing the best re-ranked solns. obtained with the four fragment spaces for the nexavar (Sorafenib) query is also shown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaVkm_-Qj5PrVg90H21EOLACvtfcHk0lhWDbXn7vnYYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt73M&md5=a927330dd3888352cc1528b1e3c6316d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800178%26sid%3Dliteratum%253Aachs%26aulast%3DDegen%26aufirst%3DJ.%26aulast%3DWegscheid-Gerlach%26aufirst%3DC.%26aulast%3DZaliani%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DOn%2520the%2520art%2520of%2520compiling%2520and%2520using%2520%25E2%2580%2599drug-like%25E2%2580%2599%2520chemical%2520fragment%2520spaces%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1503%26epage%3D1507%26doi%3D10.1002%2Fcmdc.200800178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Lewell, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, M. M.</span><span> </span><span class="NLM_article-title">RECAP retrosynthetic combinatorial analysis procedure: A powerful new technique for identifying privileged molecular fragments with useful applications in combinatorial chemistry</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1021/ci970429i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci970429i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=511-522&author=X.+Q.+Lewellauthor=D.+B.+Juddauthor=S.+P.+Watsonauthor=M.+M.+Hann&title=RECAP+retrosynthetic+combinatorial+analysis+procedure%3A+A+powerful+new+technique+for+identifying+privileged+molecular+fragments+with+useful+applications+in+combinatorial+chemistry&doi=10.1021%2Fci970429i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">RECAP-Retrosynthetic Combinatorial Analysis Procedure: A Powerful New Technique for Identifying Privileged Molecular Fragments with Useful Applications in Combinatorial Chemistry</span></div><div class="casAuthors">Lewell, Xiao Qing; Judd, Duncan; Watson, Steve; Hann, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">511-522</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of combinatorial chem. for the generation of new lead mols. is now a well established strategy in the drug discovery process.  Central to the use of combinatorial chem. is the design and availability of high quality building blocks which are likely to afford hits from the libraries that they generate.  Herein the authors describe "RECAP" (Retrosynthetic Combinatorial Anal. Procedure), a new computational technique designed to address this building block issue.  RECAP electronically fragments mols. based on chem. knowledge.  When applied to databases of biol. active mols., this allows the identification of building block fragments rich in biol. recognized elements and privileged motifs and structures.  This allows the design of building blocks and the synthesis of libraries rich in biol. motifs.  Application of RECAP to the Derwent World Drug Index (WDI) and the mol. fragments/building blocks that this generates are discussed.  The authors also describe a WDI fragment knowledge base which the authors have built which stores the drug motifs and mention its potential application in structure based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovNOpJj5LVU7Vg90H21EOLACvtfcHk0lhWDbXn7vnYYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1yrtbg%253D&md5=7445f76f162006c687116e1e539da8be</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci970429i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci970429i%26sid%3Dliteratum%253Aachs%26aulast%3DLewell%26aufirst%3DX.%2BQ.%26aulast%3DJudd%26aufirst%3DD.%2BB.%26aulast%3DWatson%26aufirst%3DS.%2BP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26atitle%3DRECAP%2520retrosynthetic%2520combinatorial%2520analysis%2520procedure%253A%2520A%2520powerful%2520new%2520technique%2520for%2520identifying%2520privileged%2520molecular%2520fragments%2520with%2520useful%2520applications%2520in%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D511%26epage%3D522%26doi%3D10.1021%2Fci970429i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span> </span><span class="NLM_article-title">Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">8</span><span class="refDoi"> DOI: 10.1186/1758-2946-1-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1186%2F1758-2946-1-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=8&author=P.+Ertlauthor=A.+Schuffenhauer&title=Estimation+of+synthetic+accessibility+score+of+drug-like+molecules+based+on+molecular+complexity+and+fragment+contributions&doi=10.1186%2F1758-2946-1-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions</span></div><div class="casAuthors">Ertl Peter; Schuffenhauer Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A method to estimate ease of synthesis (synthetic accessibility) of drug-like molecules is needed in many areas of the drug discovery process.  The development and validation of such a method that is able to characterize molecule synthetic accessibility as a score between 1 (easy to make) and 10 (very difficult to make) is described in this article.  RESULTS:  The method for estimation of the synthetic accessibility score (SAscore) described here is based on a combination of fragment contributions and a complexity penalty.  Fragment contributions have been calculated based on the analysis of one million representative molecules from PubChem and therefore one can say that they capture historical synthetic knowledge stored in this database.  The molecular complexity score takes into account the presence of non-standard structural features, such as large rings, non-standard ring fusions, stereocomplexity and molecule size.  The method has been validated by comparing calculated SAscores with ease of synthesis as estimated by experienced medicinal chemists for a set of 40 molecules.  The agreement between calculated and manually estimated synthetic accessibility is very good with r2 = 0.89.  CONCLUSION:  A novel method to estimate synthetic accessibility of molecules has been developed.  This method uses historical synthetic knowledge obtained by analyzing information from millions of already synthesized chemicals and considers also molecule complexity.  The method is sufficiently fast and provides results consistent with estimation of ease of synthesis by experienced medicinal chemists.  The calculated SAscore may be used to support various drug discovery processes where a large number of molecules needs to be ranked based on their synthetic accessibility, for example when purchasing samples for screening, selecting hits from high-throughput screening for follow-up, or ranking molecules generated by various de novo design approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQW_MJ6K-6QK9K0qsdY6Nc1fW6udTcc2eZvavLaPZwIUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7psV2gtA%253D%253D&md5=9ffc064a9d3952a583532411cb0f0673</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-8%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DEstimation%2520of%2520synthetic%2520accessibility%2520score%2520of%2520drug-like%2520molecules%2520based%2520on%2520molecular%2520complexity%2520and%2520fragment%2520contributions%26jtitle%3DJ.%2520Cheminf.%26date%3D2009%26volume%3D1%26spage%3D8%26doi%3D10.1186%2F1758-2946-1-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Hartenfeller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieto-Oberhuber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altmann, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span> </span><span class="NLM_article-title">A collection of robust organic synthesis reactions for in silico molecule design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3093</span><span class="NLM_x">–</span> <span class="NLM_lpage">3098</span><span class="refDoi"> DOI: 10.1021/ci200379p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200379p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCksLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=3093-3098&author=M.+Hartenfellerauthor=M.+Eberleauthor=P.+Meierauthor=C.+Nieto-Oberhuberauthor=K.-H.+Altmannauthor=G.+Schneiderauthor=E.+Jacobyauthor=S.+Renner&title=A+collection+of+robust+organic+synthesis+reactions+for+in+silico+molecule+design&doi=10.1021%2Fci200379p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A Collection of Robust Organic Synthesis Reactions for In Silico Molecule Design</span></div><div class="casAuthors">Hartenfeller, Markus; Eberle, Martin; Meier, Peter; Nieto-Oberhuber, Cristina; Altmann, Karl-Heinz; Schneider, Gisbert; Jacoby, Edgar; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3093-3098</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused collection of org. synthesis reactions for computer-based mol. construction is presented.  It is inspired by real-world chem. and has been compiled in close collaboration with medicinal chemists to achieve high practical relevance.  Virtual mols. assembled from existing starting material connected by these reactions are supposed to have an enhanced chance to be amenable to real chem. synthesis.  About 50% of the reactions in the dataset are ring-forming reactions, which fosters the assembly of novel ring systems and innovative chemotypes.  A comparison with a recent survey of the reactions used in early drug discovery revealed considerable overlaps with the collection presented here.  The dataset is available encoded as computer-readable Reaction SMARTS expressions from the Supporting Information presented for this paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhm-KV-_-dR7Vg90H21EOLACvtfcHk0ljXIp8MKpBxxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCksLjP&md5=848e13095bc94dfb0ea96a5fe85463ec</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci200379p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200379p%26sid%3Dliteratum%253Aachs%26aulast%3DHartenfeller%26aufirst%3DM.%26aulast%3DEberle%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DP.%26aulast%3DNieto-Oberhuber%26aufirst%3DC.%26aulast%3DAltmann%26aufirst%3DK.-H.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DA%2520collection%2520of%2520robust%2520organic%2520synthesis%2520reactions%2520for%2520in%2520silico%2520molecule%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D3093%26epage%3D3098%26doi%3D10.1021%2Fci200379p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Hartenfeller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieto-Oberhuber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altmann, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span> </span><span class="NLM_article-title">Probing the bioactivity-relevant chemical space of robust reactions and common molecular building blocks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1167</span><span class="NLM_x">–</span> <span class="NLM_lpage">1178</span><span class="refDoi"> DOI: 10.1021/ci200618n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200618n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1167-1178&author=M.+Hartenfellerauthor=M.+Eberleauthor=P.+Meierauthor=C.+Nieto-Oberhuberauthor=K.+H.+Altmannauthor=G.+Schneiderauthor=E.+Jacobyauthor=S.+Renner&title=Probing+the+bioactivity-relevant+chemical+space+of+robust+reactions+and+common+molecular+building+blocks&doi=10.1021%2Fci200618n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Bioactivity-Relevant Chemical Space of Robust Reactions and Common Molecular Building Blocks</span></div><div class="casAuthors">Hartenfeller, Markus; Eberle, Martin; Meier, Peter; Nieto-Oberhuber, Cristina; Altmann, Karl-Heinz; Schneider, Gisbert; Jacoby, Edgar; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1167-1178</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the search for new bioactive compds., there is a trend toward increasingly complex compd. libraries aiming to target the demanding targets of the future.  In contrast, medicinal chem. and traditional library design rely mainly on a small set of highly established and robust reactions.  Here, we probe a set of 58 such reactions for their ability to sample the chem. space of known bioactive mols., and the potential to create new scaffolds.  Combined with ∼26 000 common available building blocks, the reactions retrieve around 9% of a scaffold-diverse set of compds. active on human target proteins covering all major pharmaceutical target classes.  Almost 80% of generated scaffolds from virtual one-step synthesis products are not present in a large set of known bioactive mols. for human targets, indicating potential for new discoveries.  The results suggest that established synthesis resources are well suited to cover the known bioactivity-relevant chem. space and that there are plenty of unexplored regions accessible by these reactions, possibly providing valuable "low-hanging fruit" for hit discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7dpgccjqzh7Vg90H21EOLACvtfcHk0liHkCnkvq6afg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOksLk%253D&md5=4505f35fa529e1f94a495aec2f6faff2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci200618n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200618n%26sid%3Dliteratum%253Aachs%26aulast%3DHartenfeller%26aufirst%3DM.%26aulast%3DEberle%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DP.%26aulast%3DNieto-Oberhuber%26aufirst%3DC.%26aulast%3DAltmann%26aufirst%3DK.%2BH.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DProbing%2520the%2520bioactivity-relevant%2520chemical%2520space%2520of%2520robust%2520reactions%2520and%2520common%2520molecular%2520building%2520blocks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1167%26epage%3D1178%26doi%3D10.1021%2Fci200618n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Roughley, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">The medicinal chemist’s toolbox: An analysis of reactions used in the pursuit of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3451</span><span class="NLM_x">–</span> <span class="NLM_lpage">3479</span><span class="refDoi"> DOI: 10.1021/jm200187y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200187y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3451-3479&author=S.+D.+Roughleyauthor=A.+M.+Jordan&title=The+medicinal+chemist%E2%80%99s+toolbox%3A+An+analysis+of+reactions+used+in+the+pursuit+of+drug+candidates&doi=10.1021%2Fjm200187y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The Medicinal Chemist's Toolbox: An Analysis of Reactions Used in the Pursuit of Drug Candidates</span></div><div class="casAuthors">Roughley, Stephen D.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3451-3479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A survey of selected medicinal chem. literature from 2008 was undertaken in order to det. the most common chem. reactions used in the prepn. of drug candidates.  In addn. to an examn. of the reactions, the frequency of occurrence of various functional groups in the drug targets was analyzed.  The Lipinski parameters and chirality of the surveyed compds. were also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTUrcluTkPBLVg90H21EOLACvtfcHk0liHkCnkvq6afg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVyhsrg%253D&md5=a94df26d82bbd58b269ed20aa3253f56</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm200187y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200187y%26sid%3Dliteratum%253Aachs%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520medicinal%2520chemist%25E2%2580%2599s%2520toolbox%253A%2520An%2520analysis%2520of%2520reactions%2520used%2520in%2520the%2520pursuit%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3451%26epage%3D3479%26doi%3D10.1021%2Fjm200187y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of β<sub>2</sub>-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span><span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+%CE%B22-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0liHkCnkvq6afg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520%25CE%25B22-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Ballesteros, J. A.; Weinstein, H.</span><span> </span><span class="NLM_article-title">Integrated Methods for the Construction of Three-dimensional Models and Computational Probing of Structure-Function Relations in G Protein-Coupled Receptors</span>. In  <span class="citation_source-book">Receptor Molecular Biology</span>; <span class="NLM_contrib-group">Sealfon, S. C.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">1995</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">25</span>, pp  <span class="NLM_fpage">366</span><span class="NLM_x">–</span> <span class="NLM_lpage">428</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&pages=366-428&author=J.+A.+Ballesteros&author=H.+Weinsteinauthor=S.+C.+Sealfon&title=Receptor+Molecular+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26atitle%3DIntegrated%2520Methods%2520for%2520the%2520Construction%2520of%2520Three-dimensional%2520Models%2520and%2520Computational%2520Probing%2520of%2520Structure-Function%2520Relations%2520in%2520G%2520Protein-Coupled%2520Receptors%26btitle%3DReceptor%2520Molecular%2520Biology%26aulast%3DSealfon%26aufirst%3DS.%2BC.%26pub%3DAcademic%2520Press%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casarosa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeVree, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnapp, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konetzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunahara, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pautsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyaert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Structure of a nanobody-stabilized active state of the beta(2) adrenoceptor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">469</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">180</span><span class="refDoi"> DOI: 10.1038/nature09648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature09648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=21228869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=175-180&author=S.+G.+F.+Rasmussenauthor=H.+Choiauthor=J.+Fungauthor=E.+Pardonauthor=P.+Casarosaauthor=P.+S.+Chaeauthor=B.+T.+DeVreeauthor=D.+M.+Rosenbaumauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=A.+Schnappauthor=I.+Konetzkiauthor=R.+K.+Sunaharaauthor=S.+H.+Gellmanauthor=A.+Pautschauthor=J.+Steyaertauthor=W.+I.+Weisauthor=B.+K.+Kobilka&title=Structure+of+a+nanobody-stabilized+active+state+of+the+beta%282%29+adrenoceptor&doi=10.1038%2Fnature09648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a nanobody-stabilized active state of the β2 adrenoceptor</span></div><div class="casAuthors">Rasmussen, Soren G. F.; Choi, Hee-Jung; Fung, Juan Jose; Pardon, Els; Casarosa, Paola; Chae, Pil Seok; DeVree, Brian T.; Rosenbaum, Daniel M.; Thian, Foon Sun; Kobilka, Tong Sun; Schnapp, Andreas; Konetzki, Ingo; Sunahara, Roger K.; Gellman, Samuel H.; Pautsch, Alexander; Steyaert, Jan; Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7329</span>),
    <span class="NLM_cas:pages">175-180</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein coupled receptors (GPCRs) exhibit a spectrum of functional behaviors in response to natural and synthetic ligands.  Recent crystal structures provide insights into inactive states of several GPCRs.  Efforts to obtain an agonist-bound active-state GPCR structure have proven difficult due to the inherent instability of this state in the absence of a G protein.  We generated a camelid antibody fragment (nanobody) to the human β2 adrenergic receptor (β2AR) that exhibits G protein-like behavior, and obtained an agonist-bound, active-state crystal structure of the receptor-nanobody complex.  Comparison with the inactive β2AR structure reveals subtle changes in the binding pocket; however, these small changes are assocd. with an 11 Å outward movement of the cytoplasmic end of transmembrane segment 6, and rearrangements of transmembrane segments 5 and 7 that are remarkably similar to those obsd. in opsin, an active form of rhodopsin.  This structure provides insights into the process of agonist binding and activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqITZxHz6Vi77Vg90H21EOLACvtfcHk0lgF5GRgBMDWRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D&md5=3a1d3bac6d92c9d54cf1ddd14d56ab8c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature09648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09648%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DFung%26aufirst%3DJ.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCasarosa%26aufirst%3DP.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DSchnapp%26aufirst%3DA.%26aulast%3DKonetzki%26aufirst%3DI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DPautsch%26aufirst%3DA.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520of%2520a%2520nanobody-stabilized%2520active%2520state%2520of%2520the%2520beta%25282%2529%2520adrenoceptor%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D175%26epage%3D180%26doi%3D10.1038%2Fnature09648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeVree, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calinski, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathiesen, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyons, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyaert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skiniotis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunahara, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Crystal structure of the beta(2) adrenergic receptor-Gs protein complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">477</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">U311</span><span class="refDoi"> DOI: 10.1038/nature10361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature10361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=21772288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2011&pages=549-U311&author=S.+G.+F.+Rasmussenauthor=B.+T.+DeVreeauthor=Y.+Zouauthor=A.+C.+Kruseauthor=K.+Y.+Chungauthor=T.+S.+Kobilkaauthor=F.+S.+Thianauthor=P.+S.+Chaeauthor=E.+Pardonauthor=D.+Calinskiauthor=J.+M.+Mathiesenauthor=S.+T.+A.+Shahauthor=J.+A.+Lyonsauthor=M.+Caffreyauthor=S.+H.+Gellmanauthor=J.+Steyaertauthor=G.+Skiniotisauthor=W.+I.+Weisauthor=R.+K.+Sunaharaauthor=B.+K.+Kobilka&title=Crystal+structure+of+the+beta%282%29+adrenergic+receptor-Gs+protein+complex&doi=10.1038%2Fnature10361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the β2 adrenergic receptor-Gs protein complex</span></div><div class="casAuthors">Rasmussen, Soren G. F.; DeVree, Brian T.; Zou, Yao-Zhong; Kruse, Andrew C.; Chung, Ka-Young; Kobilka, Tong-Sun; Thian, Foon-Sun; Chae, Pil-Seok; Pardon, Els; Calinski, Diane; Mathiesen, Jesper M.; Shah, Syed T. A.; Lyons, Joseph A.; Caffrey, Martin; Gellman, Samuel H.; Steyaert, Jan; Skiniotis, Georgios; Weis, William I.; Sunahara, Roger K.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">7366</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are responsible for the majority of cellular responses to hormones and neurotransmitters as well as the senses of sight, olfaction and taste.  The paradigm of GPCR signalling is the activation of a heterotrimeric GTP binding protein (G protein) by an agonist-occupied receptor.  The β2 adrenergic receptor (β2AR) activation of Gs, the stimulatory G protein for adenylyl cyclase, has long been a model system for GPCR signalling.  Here we present the crystal structure of the active state ternary complex composed of agonist-occupied monomeric β2AR and nucleotide-free Gs heterotrimer.  The principal interactions between the β2AR and Gs involve the amino- and carboxy-terminal α-helixes of Gs, with conformational changes propagating to the nucleotide-binding pocket.  The largest conformational changes in the β2AR include a 14 Å outward movement at the cytoplasmic end of transmembrane segment 6 (TM6) and an α-helical extension of the cytoplasmic end of TM5.  The most surprising observation is a major displacement of the α-helical domain of Gαs relative to the Ras-like GTPase domain.  This crystal structure represents the first high-resoln. view of transmembrane signalling by a GPCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-PdYRDvolqrVg90H21EOLACvtfcHk0lgCMhDCEDk63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL&md5=d22a43dd677ac255d138b1aedff357d3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature10361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10361%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DChung%26aufirst%3DK.%2BY.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCalinski%26aufirst%3DD.%26aulast%3DMathiesen%26aufirst%3DJ.%2BM.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DLyons%26aufirst%3DJ.%2BA.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DSkiniotis%26aufirst%3DG.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCrystal%2520structure%2520of%2520the%2520beta%25282%2529%2520adrenergic%2520receptor-Gs%2520protein%2520complex%26jtitle%3DNature%26date%3D2011%26volume%3D477%26spage%3D549%26epage%3DU311%26doi%3D10.1038%2Fnature10361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyons, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holl, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aragao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlow, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeVree, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunahara, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellman, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmeiner, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Structure and function of an irreversible agonist-β2 adrenoceptor complex</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">469</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="NLM_x">–</span> <span class="NLM_lpage">240</span><span class="refDoi"> DOI: 10.1038/nature09665</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature09665" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=21228876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFehsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=236-240&author=D.+M.+Rosenbaumauthor=C.+Zhangauthor=J.+A.+Lyonsauthor=R.+Hollauthor=D.+Aragaoauthor=D.+H.+Arlowauthor=S.+G.+F.+Rasmussenauthor=H.+Choiauthor=B.+T.+DeVreeauthor=R.+K.+Sunaharaauthor=P.+S.+Chaeauthor=S.+H.+Gellmanauthor=R.+O.+Drorauthor=D.+E.+Shawauthor=W.+I.+Weisauthor=M.+Caffreyauthor=P.+Gmeinerauthor=B.+K.+Kobilka&title=Structure+and+function+of+an+irreversible+agonist-%CE%B22+adrenoceptor+complex&doi=10.1038%2Fnature09665"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of an irreversible agonist-β2 adrenoceptor complex</span></div><div class="casAuthors">Rosenbaum, Daniel M.; Zhang, Cheng; Lyons, Joseph A.; Holl, Ralph; Aragao, David; Arlow, Daniel H.; Rasmussen, Soren G. F.; Choi, Hee-Jung; DeVree, Brian T.; Sunahara, Roger K.; Chae, Pil Seok; Gellman, Samuel H.; Dror, Ron O.; Shaw, David E.; Weis, William I.; Caffrey, Martin; Gmeiner, Peter; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7329</span>),
    <span class="NLM_cas:pages">236-240</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are eukaryotic integral membrane proteins that modulate biol. function by initiating cellular signaling in response to chem. diverse agonists.  Despite recent progress in the structural biol. of GPCRs, the mol. basis for agonist binding and allosteric modulation of these proteins is poorly understood.  Structural knowledge of agonist-bound states is essential for deciphering the mechanism of receptor activation, and for structure-guided design and optimization of ligands.  However, the crystn. of agonist-bound GPCRs has been hampered by modest affinities and rapid off-rates of available agonists.  Using the inactive structure of the human β2 adrenergic receptor (β2AR) as a guide, we designed a β2AR agonist that can be covalently tethered to a specific site on the receptor through a disulfide bond.  The covalent β2AR-agonist complex forms efficiently, and is capable of activating a heterotrimeric G protein.  We crystd. a covalent agonist-bound β2AR-T4L fusion protein in lipid bilayers through the use of the lipidic mesophase method, and detd. its structure at 3.5 Å resoln.  A comparison to the inactive structure and an antibody-stabilized active structure (companion paper) shows how binding events at both the extracellular and intracellular surfaces are required to stabilize an active conformation of the receptor.  The structures are in agreement with long-timescale (up to 30 μs) mol. dynamics simulations showing that an agonist-bound active conformation spontaneously relaxes to an inactive-like conformation in the absence of a G protein or stabilizing antibody.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3cxTxwHig_rVg90H21EOLACvtfcHk0lgCMhDCEDk63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFehsQ%253D%253D&md5=8629b93e43fb692395e2aa6f8bb011a9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature09665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09665%26sid%3Dliteratum%253Aachs%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLyons%26aufirst%3DJ.%2BA.%26aulast%3DHoll%26aufirst%3DR.%26aulast%3DAragao%26aufirst%3DD.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520and%2520function%2520of%2520an%2520irreversible%2520agonist-%25CE%25B22%2520adrenoceptor%2520complex%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D236%26epage%3D240%26doi%3D10.1038%2Fnature09665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Ring, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manglik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enos, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1038/nature12572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnature12572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=24056936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=575-9&author=A.+M.+Ringauthor=A.+Manglikauthor=A.+C.+Kruseauthor=M.+D.+Enosauthor=W.+I.+Weisauthor=K.+C.+Garciaauthor=B.+K.+Kobilka&title=Adrenaline-activated+structure+of+%CE%B22-adrenoceptor+stabilized+by+an+engineered+nanobody&doi=10.1038%2Fnature12572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Adrenaline-activated structure of β2-adrenoceptor stabilized by an engineered nanobody</span></div><div class="casAuthors">Ring, Aaron M.; Manglik, Aashish; Kruse, Andrew C.; Enos, Michael D.; Weis, William I.; Garcia, K. Christopher; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">575-579</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are integral membrane proteins that play an essential role in human physiol., yet the mol. processes through which they bind to their endogenous agonists and activate effector proteins remain poorly understood.  So far, it has not been possible to capture an active-state GPCR bound to its native neurotransmitter.  Crystal structures of agonist-bound GPCRs have relied on the use of either exceptionally high-affinity agonists or receptor stabilization by mutagenesis.  Many natural agonists such as adrenaline, which activates the β2-adrenoceptor (β2AR), bind with relatively low affinity, and they are often chem. unstable.  Here, using directed evolution, the authors engineered a high-affinity conformationally selective single-domain camelid antibody fragment (nanobody; Nb6B9) that stabilized the active state of human β2AR, and used this to obtain crystal structures of the activated receptor bound to multiple ligands.  The authors present structures of the active-state human β2AR bound to 3 chem. distinct agonists: (1) the ultrahigh-affinity agonist BI167107; (2) the high-affinity catecholamine agonist, hydroxybenzylisoproterenol; and (3) the low-affinity endogenous agonist, adrenaline.  The crystal structures revealed a highly conserved overall ligand recognition and activation mode despite diverse ligand chem. structures and affinities that ranged from 100 nM to ∼80 pM.  Overall, the adrenaline-bound receptor structure was similar to the others, but it had substantial rearrangements in extracellular loop 3 and the extracellular tip of transmembrane helix 6.  These structures also revealed a water-mediated H-bond between 2 conserved Tyr residues, which appeared to stabilize the active state of the β2AR and related GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAp0VskflwWbVg90H21EOLACvtfcHk0lh7ym1qu7J3gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSlsLzP&md5=c5b844a0a86ee0a74843588a4a363592</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature12572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12572%26sid%3Dliteratum%253Aachs%26aulast%3DRing%26aufirst%3DA.%2BM.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DEnos%26aufirst%3DM.%2BD.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DGarcia%26aufirst%3DK.%2BC.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DAdrenaline-activated%2520structure%2520of%2520%25CE%25B22-adrenoceptor%2520stabilized%2520by%2520an%2520engineered%2520nanobody%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D575%26epage%3D9%26doi%3D10.1038%2Fnature12572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Weichert, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manglik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiller, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hübner, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmeiner, P.</span><span> </span><span class="NLM_article-title">Covalent agonists for studying G protein-coupled receptor activation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">10744</span><span class="NLM_x">–</span> <span class="NLM_lpage">10748</span><span class="refDoi"> DOI: 10.1073/pnas.1410415111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1073%2Fpnas.1410415111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=25006259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyqtbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=10744-10748&author=D.+Weichertauthor=A.+C.+Kruseauthor=A.+Manglikauthor=C.+Hillerauthor=C.+Zhangauthor=H.+H%C3%BCbnerauthor=B.+K.+Kobilkaauthor=P.+Gmeiner&title=Covalent+agonists+for+studying+G+protein-coupled+receptor+activation&doi=10.1073%2Fpnas.1410415111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent agonists for studying G protein-coupled receptor activation</span></div><div class="casAuthors">Weichert, Dietmar; Kruse, Andrew C.; Manglik, Aashish; Hiller, Christine; Zhang, Cheng; Huebner, Harald; Kobilka, Brian K.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">10744-10748</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Structural studies on G protein-coupled receptors (GPCRs) provide important insights into the architecture and function of these important drug targets.  However, the crystn. of GPCRs in active states is particularly challenging, requiring the formation of stable and conformationally homogeneous ligand-receptor complexes.  Native hormones, neurotransmitters, and synthetic agonists that bind with low affinity are ineffective at stabilizing an active state for crystallogenesis.  To promote structural studies on the pharmacol. highly relevant class of aminergic GPCRs, the authors here present the development of covalently binding mol. tools activating Gs-, Gi-, and Gq-coupled receptors.  The covalent agonists are derived from the monoamine neurotransmitters noradrenaline, dopamine, serotonin, and histamine, and they were accessed using a general and versatile synthetic strategy.  The authors demonstrate that the tool compds. presented herein display an efficient covalent binding mode and that the resp. covalent ligand-receptor complexes activate G proteins comparable to the natural neurotransmitters.  A crystal structure of the β2-adrenoreceptor in complex with a covalent noradrenaline analog and a conformationally selective antibody (nanobody) verified that these agonists can be used to facilitate crystallogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqbJHK2UMj1rVg90H21EOLACvtfcHk0lh7ym1qu7J3gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyqtbnN&md5=a131bb7711d9a1826c77c3b598dee470</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1410415111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1410415111%26sid%3Dliteratum%253Aachs%26aulast%3DWeichert%26aufirst%3DD.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DHiller%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DCovalent%2520agonists%2520for%2520studying%2520G%2520protein-coupled%2520receptor%2520activation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D10744%26epage%3D10748%26doi%3D10.1073%2Fpnas.1410415111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Pardon, E.; Betti, C.; Laeremans, T.; Chevillard, F.; Guillemyn, K.; Kolb, P.; Ballet, S.; Steyaert, J.</span><span> </span><span class="NLM_article-title">Nanobody-enabled reverse pharmacology on GPCRs</span>.  <span class="citation_source-journal">Angew. Chem. Int. Ed</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2018</span><span class="NLM_x">, </span>submitted.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=E.+Pardon&author=C.+Betti&author=T.+Laeremans&author=F.+Chevillard&author=K.+Guillemyn&author=P.+Kolb&author=S.+Ballet&author=J.+Steyaert&title=Nanobody-enabled+reverse+pharmacology+on+GPCRs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPardon%26aufirst%3DE.%26atitle%3DNanobody-enabled%2520reverse%2520pharmacology%2520on%2520GPCRs%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span><span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0lgcozRit2C_1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC – A free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+%E2%80%93+A+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2ea3sq_fh3iSv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520%25E2%2580%2593%2520A%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Pottel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moitessier, N.</span><span> </span><span class="NLM_article-title">Customizable generation of synthetically accessible, local chemical subspaces</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">454</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00648</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00648" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=454-467&author=J.+Pottelauthor=N.+Moitessier&title=Customizable+generation+of+synthetically+accessible%2C+local+chemical+subspaces&doi=10.1021%2Facs.jcim.6b00648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Customizable Generation of Synthetically Accessible, Local Chemical Subspaces</span></div><div class="casAuthors">Pottel, Joshua; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">454-467</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening large libraries of chems. has been an efficient strategy to discover bioactive compds.; however, a portion of the potential for success is limited to the available libraries.  Synergizing combinatorial and computational chemistries emerged as a time-efficient strategy to explore the chem. space more widely.  Ideally, streamlining the evaluation process for larger, feasible chem. libraries would become commonplace.  Thus, combinatorial tools and, for example, docking methods would be integrated to identify novel bioactive entities.  The idea is simple in nature, but much more complex in practice; combinatorial chem. is more than the coupling of chems. into products: synthetic feasibility includes chemoselectivity, stereoselectivity, protecting group chem. and chem. availability which must all be considered for combinatorial library design.  In addn., intuitive interfaces and simple user manipulation is key for optimal use of such tools by org. chemists-crucial for the integration of such software in medicinal chem. labs.  We present herein FINDERS and REACT2D-integrated into the VIRTUAL CHEMIST platform, a modular software suite.  This approach enhances virtual combinatorial chem. by identifying available chems. compatible with a user-defined chem. transformation and by carrying out the reaction leading to libraries of realistic, synthetically accessible chems.-all with a completely automated, black-box, and efficient design.  We demonstrate its utility by generating ∼40 million synthetically accessible, stereochem. accurate compds. from a single library of 100,000 purchasable mols. and 56 well-characterized chem. reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5D6CtvWNM4bVg90H21EOLACvtfcHk0ljPYur2nlH-_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCntbk%253D&md5=5556a4e23ad84b3fbf594f613afb4841</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00648%26sid%3Dliteratum%253Aachs%26aulast%3DPottel%26aufirst%3DJ.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DCustomizable%2520generation%2520of%2520synthetically%2520accessible%252C%2520local%2520chemical%2520subspaces%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D454%26epage%3D467%26doi%3D10.1021%2Facs.jcim.6b00648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Baker, J. G.</span><span> </span><span class="NLM_article-title">The selectivity of β-adrenoceptor antagonists at the human β1, β2 and β3 adrenoceptors</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">144</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">322</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0706048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fsj.bjp.0706048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=15655528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2005&pages=317-322&author=J.+G.+Baker&title=The+selectivity+of+%CE%B2-adrenoceptor+antagonists+at+the+human+%CE%B21%2C+%CE%B22+and+%CE%B23+adrenoceptors&doi=10.1038%2Fsj.bjp.0706048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The selectivity of β-adrenoceptor antagonists at the human β1, β2 and β3 adrenoceptors</span></div><div class="casAuthors">Baker, Jillian G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">317-322</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">β-Adrenoceptor antagonists (β-blockers) are one of the most widely used classes of drugs in cardiovascular medicine (hypertension, ischemic heart disease and increasingly in heart failure) as well as in the management of anxiety, migraine and glaucoma.  Where known, the mode of action in cardiovascular disease is from antagonism of endogenous catecholamine responses in the heart (mainly at β1-adrenoceptors), while the worrisome side effects of bronchospasm result from airway β2-adrenoceptor blockade.  The aim of this study was to det. the selectivity of β-antagonists for the human β-adrenoceptor subtypes.  3H-CGP 12177 whole cell-binding studies were undertaken in CHO cell lines stably expressing either the human β1-, β2- or the β3-adrenoceptor to det. the affinity of ligands for each receptor subtype in the same cell background.  In this study, the selectivity of well-known subtype-selective ligands was clearly demonstrated: thus, the selective β1 antagonist CGP 20712A was 501-fold selective over β2 and 4169-fold selective over β3; the β2-selective antagonist ICI 118551 was 550- and 661-fold selective over β1 and β3, resp., and the selective β3 compd. CL 316243 was 10-fold selective over β2 and more than 129-fold selective over β1.  Those β2-adrenoceptor agonists used clin. for the treatment of asthma and COPD were β2 selective: 29-, 61- and 2818-fold for salbutamol, terbutaline and salmeterol over β1, resp.  There was little difference in the affinity of these ligands between β1 and β3 adrenoceptors.  The clin. used β-antagonists studied ranged from bisoprolol (14-fold β1-selective) to timolol (26-fold β2-selective).  However, the majority showed little selectivity for the β1- over the β2-adrenoceptor, with many actually being more β2-selective.  This study shows that the β1/β2 selectivity of most clin. used β-blockers is poor in intact cells, and that some compds. that are traditionally classed as β1-selective' actually have higher affinity for the β2-adrenoceptor.  There is therefore considerable potential for developing more selective β-antagonists for clin. use and thereby reducing the side-effect profile of β-blockers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqggxqVpB4tvrVg90H21EOLACvtfcHk0ljPYur2nlH-_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGms70%253D&md5=33cba45e9739901cf686d6cfbb096eed</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0706048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0706048%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DJ.%2BG.%26atitle%3DThe%2520selectivity%2520of%2520%25CE%25B2-adrenoceptor%2520antagonists%2520at%2520the%2520human%2520%25CE%25B21%252C%2520%25CE%25B22%2520and%2520%25CE%25B23%2520adrenoceptors%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2005%26volume%3D144%26spage%3D317%26epage%3D322%26doi%3D10.1038%2Fsj.bjp.0706048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.-y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, L.</span><span> </span><span class="NLM_article-title">Discovery of an orally selective inhibitor of signal transducer and activator of transcription 3 using advanced multiple ligand simultaneous docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2718</span><span class="NLM_x">–</span> <span class="NLM_lpage">2731</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01489</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01489" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2718-2731&author=W.+Yuauthor=C.+Liauthor=W.+Zhangauthor=Y.+Xiaauthor=S.+Liauthor=J.-y.+Linauthor=K.+Yuauthor=M.+Liuauthor=L.+Yangauthor=J.+Luoauthor=Y.+Chenauthor=H.+Sunauthor=L.+Kong&title=Discovery+of+an+orally+selective+inhibitor+of+signal+transducer+and+activator+of+transcription+3+using+advanced+multiple+ligand+simultaneous+docking&doi=10.1021%2Facs.jmedchem.6b01489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Orally Selective Inhibitor of Signal Transducer and Activator of Transcription 3 Using Advanced Multiple Ligand Simultaneous Docking</span></div><div class="casAuthors">Yu, Wenying; Li, Chenglong; Zhang, Wenda; Xia, Yuanzheng; Li, Shanshan; Lin, Jia-yuh; Yu, Keqin; Liu, Mu; Yang, Lei; Luo, Jianguang; Chen, Yijun; Sun, Hongbin; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2718-2731</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting signal transducer and activator of transcription 3 (STAT3) is a potential anticancer strategy.  However, STAT3 inhibitors with good selectivity and bioavailability are rare.  The aim of this study was to discover selective direct STAT3 inhibitors with good drug-likeness.  By advanced multiple ligand simultaneous docking (AMLSD) method, compd. 9 (5,8-dioxo-6-((2-(piperazin-1-yl)phenyl)amino)-5,8-dihydronaphthalene-1-sulfonamide) was designed as an orally-bioavailable STAT3 inhibitor, which presented superior druggability and selectivity to other representative STAT3 inhibitors.  Compd. 9 directly and selectively inhibited the pY705 site of STAT3 with an affinity (Ki) of 440nM.  The IC50 of 9 for breast cancer cell MDA-MB-231 was 184-fold lower than its IC50 for normal breast epithelial cell MCF-10A.  Compd. 9 in vivo induced significant antitumor responses better than Gefitinib and its therapeutic index should be over 100, indicating good safety of 9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiT8F_tCrky7Vg90H21EOLACvtfcHk0ljPYur2nlH-_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D&md5=d9554e49fd17a559e8c7171f36c53607</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01489%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.-y.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520selective%2520inhibitor%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520using%2520advanced%2520multiple%2520ligand%2520simultaneous%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2718%26epage%3D2731%26doi%3D10.1021%2Facs.jmedchem.6b01489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span><span class="refDoi"> DOI: 10.1038/nchembio831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0ljNDDcUcgV-SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723%26doi%3D10.1038%2Fnchembio831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Docking screens: right for the right reasons?</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">770</span><span class="refDoi"> DOI: 10.2174/156802609789207091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.2174%2F156802609789207091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19754393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=755-770&author=P.+Kolbauthor=J.+J.+Irwin&title=Docking+screens%3A+right+for+the+right+reasons%3F&doi=10.2174%2F156802609789207091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Docking screens: right for the right reasons?</span></div><div class="casAuthors">Kolb, Peter; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">755-770</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Whereas docking screens have emerged as the most practical way to use protein structure for ligand discovery, an inconsistent track record raises questions about how well docking actually works.  In its favor, a growing no. of publications report the successful discovery of new ligands, often supported by exptl. affinity data and controls for artifacts.  Few reports, however, actually test the underlying structural hypotheses that docking makes.  To be successful and not just lucky, prospective docking must not only rank a true ligand among the top scoring compds., it must also correctly orient the ligand so the score it receives is biophys. sound.  If the correct binding pose is not predicted, a skeptic might well infer that the discovery was serendipitous.  Surveying over 15 years of the docking literature, we were surprised to discover how rarely sufficient evidence is presented to establish whether docking actually worked for the right reasons.  The paucity of exptl. tests of theor. predicted poses undermines confidence in a technique that has otherwise become widely accepted.  Of course, solving a crystal structure is not always possible, and even when it is, it can be a lot of work, and is not readily accessible to all groups.  Even when a structure can be detd., investigators may prefer to gloss over an erroneous structural prediction to better focus on their discovery.  Still, the absence of a direct test of theory by expt. is a loss for method developers seeking to understand and improve docking methods.  We hope this review will motivate investigators to solve structures and compare them with their predictions whenever possible, to advance the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXLK_ZZmntsrVg90H21EOLACvtfcHk0ljNDDcUcgV-SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP&md5=3e6c0073795fae06c0d5a1175af8e8e5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.2174%2F156802609789207091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609789207091%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DDocking%2520screens%253A%2520right%2520for%2520the%2520right%2520reasons%253F%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D755%26epage%3D770%26doi%3D10.2174%2F156802609789207091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking & chemoinformatic screens for new ligands and targets</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">429</span><span class="NLM_x">–</span> <span class="NLM_lpage">436</span><span class="refDoi"> DOI: 10.1016/j.copbio.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2Fj.copbio.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=19733475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=429-436&author=P.+Kolbauthor=R.+S.+Ferreiraauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Docking+%26+chemoinformatic+screens+for+new+ligands+and+targets&doi=10.1016%2Fj.copbio.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and chemoinformatic screens for new ligands and targets</span></div><div class="casAuthors">Kolb, Peter; Ferreira, Rafaela S.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-436</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computer-based docking screens are now widely used to discover new ligands for targets of known structure; in the last two years alone, the discovery of ligands for more than 20 proteins has been reported.  Recently, investigators have also turned to predicting new substrates for enzymes of unknown function, taking docking in a wholly new direction.  Increasingly, the hit rates, the true-positives, and the false-positives from the docking screens are being compared to those from empirical, high-throughput screens, revealing the strengths, weaknesses, and complementarities of both techniques.  The recent efflorescence of GPCR structures has made these quintessential drug targets available to structure-based approaches.  Consistent with their druggability', the docking screens have returned high hit rates and potent mols.  Finally, in the last several years, an approach almost exactly opposite to docking has also appeared; this pharmacol. network approach begins not with the structure of the target but rather those of drug mols. and asks, given a pattern of chem. in the ligands, what targets may a particular drug bind to.  This method, which returns to an older, pharmacol. logic, has been surprisingly successful in predicting new off-targets' for established drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryVNL3mMiQjbVg90H21EOLACvtfcHk0lgvu9kIQxKN8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL&md5=4b6b2457a735e18be5780d9299ebffe6</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520%2526%2520chemoinformatic%2520screens%2520for%2520new%2520ligands%2520and%2520targets%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2009%26volume%3D20%26spage%3D429%26epage%3D436%26doi%3D10.1016%2Fj.copbio.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Chevillard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span> </span><span class="NLM_article-title">SCUBIDOO: A large yet screenable and easily searchable database of computationally created chemical compounds optimized toward high likelihood of synthetic tractability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1824</span><span class="NLM_x">–</span> <span class="NLM_lpage">1835</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCltbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1824-1835&author=F.+Chevillardauthor=P.+Kolb&title=SCUBIDOO%3A+A+large+yet+screenable+and+easily+searchable+database+of+computationally+created+chemical+compounds+optimized+toward+high+likelihood+of+synthetic+tractability&doi=10.1021%2Facs.jcim.5b00203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">SCUBIDOO: A Large yet Screenable and Easily Searchable Database of Computationally Created Chemical Compounds Optimized toward High Likelihood of Synthetic Tractability</span></div><div class="casAuthors">Chevillard, F.; Kolb, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1824-1835</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">De novo drug design is widely assisted by computational approaches that enable the generation of a tremendous amt. of new virtual mols. within a short time frame.  While the novelty of the computationally generated compds. can easily be assessed, such approaches often neglect the synthetic feasibility of the mols., thus creating a potential hurdle that can be a barrier to further investigation.  Therefore, we have developed SCUBIDOO, a freely accessible database concept that currently holds 21 million virtual products originating from a small library of building blocks and a collection of robust org. reactions.  This large data set was reduced to three representative and computationally tractable samples denoted as S, M, and L, contg. 9994, 99,977, and 999,794 products, resp.  These small sets are useful as starting points for ligand identification and optimization projects.  The generated products come with synthesis instructions and alerts of possible side reactions, and we show that they exhibit drug-like properties while still extending into unexplored quadrants of chem. space, thus suggesting novelty.  We show multiple examples that demonstrate how SCUBIDOO can facilitate the search around initial hits.  This database might be a useful idea generator for early ligand discovery projects since it allows a focus on those mols. that are likely to be synthetically feasible and can therefore be studied further.  Together with its modular building block construction principle, this database is also suitable for structure-activity relationship studies or fragment-growing strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJpliMmDkrU7Vg90H21EOLACvtfcHk0lgvu9kIQxKN8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCltbzL&md5=757ab5c75eeec61ae4f5811bee131f4c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00203%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DSCUBIDOO%253A%2520A%2520large%2520yet%2520screenable%2520and%2520easily%2520searchable%2520database%2520of%2520computationally%2520created%2520chemical%2520compounds%2520optimized%2520toward%2520high%2520likelihood%2520of%2520synthetic%2520tractability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D1824%26epage%3D1835%26doi%3D10.1021%2Facs.jcim.5b00203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">High-resolution crystal structure of an engineered human β<sub>2</sub>-adrenergic G protein-coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1258</span><span class="NLM_x">–</span> <span class="NLM_lpage">1265</span><span class="refDoi"> DOI: 10.1126/science.1150577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1126%2Fscience.1150577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17962520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1258-1265&author=V.+Cherezovauthor=D.+M.+Rosenbaumauthor=M.+A.+Hansonauthor=S.+G.+F.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.-J.+Choiauthor=P.+Kuhnauthor=W.+I.+Weisauthor=B.+K.+Kobilkaauthor=R.+C.+Stevens&title=High-resolution+crystal+structure+of+an+engineered+human+%CE%B22-adrenergic+G+protein-coupled+receptor&doi=10.1126%2Fscience.1150577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structure of an Engineered Human β2-Adrenergic G Protein-Coupled Receptor</span></div><div class="casAuthors">Cherezov, Vadim; Rosenbaum, Daniel M.; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Kuhn, Peter; Weis, William I.; Kobilka, Brian K.; Stevens, Raymond C.; Takeda, S.; Kadowaki, S.; Haga, T.; Takaesu, H.; Mitaku, S.; Fredriksson, R.; Lagerstrom, M. C.; Lundin, L. G.; Schioth, H. B.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Lefkowitz, R. J.; Shenoy, S. K.; Rosenbaum, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1258-1265</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Second extracellular loop, which in the β2-adrenergic receptor contains an unusual pair of disulfide bonds and an extra helix.  This loop and the absence Heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors constitute the largest family of eukaryotic signal transduction proteins that communicate across the membrane.  We report the crystal structure of a human β2-adrenergic receptor-T4 lysozyme fusion protein bound to the partial inverse agonist carazolol at 2.4 angstrom resoln.  The structure provides a high-resoln. view of a human G protein-coupled receptor bound to a diffusible ligand.  Ligand-binding site accessibility is enabled by the second extracellular loop, which is held out of the binding cavity by a pair of closely spaced disulfide bridges and a short helical segment within the loop.  Cholesterol, a necessary component for crystn., mediates an intriguing parallel assocn. of receptor mols. in the crystal lattice.  Although the location of carazolol in the β2-adrenergic receptor is very similar to that of retinal in rhodopsin, structural differences in the ligand-binding site and other regions highlight the challenges in using rhodopsin as a template model for this large receptor family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC-8wZ2k2GHLVg90H21EOLACvtfcHk0lgvu9kIQxKN8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I&md5=12c5bacb8464a4b243fe9341192b5b3b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150577%26sid%3Dliteratum%253Aachs%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520an%2520engineered%2520human%2520%25CE%25B22-adrenergic%2520G%2520protein-coupled%2520receptor%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1258%26epage%3D1265%26doi%3D10.1126%2Fscience.1150577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, X.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">GPCR engineering yields high-resolution structural insights into β<sub>2</sub>-adrenergic receptor function</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1266</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1126/science.1150609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1126%2Fscience.1150609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=17962519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1266-1273&author=D.+M.+Rosenbaumauthor=V.+Cherezovauthor=M.+A.+Hansonauthor=S.+G.+F.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.-J.+Choiauthor=X.-J.+Yaoauthor=W.+I.+Weisauthor=R.+C.+Stevensauthor=B.+K.+Kobilka&title=GPCR+engineering+yields+high-resolution+structural+insights+into+%CE%B22-adrenergic+receptor+function&doi=10.1126%2Fscience.1150609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">GPCR Engineering Yields High-Resolution Structural Insights into β2-Adrenergic Receptor Function</span></div><div class="casAuthors">Rosenbaum, Daniel M.; Cherezov, Vadim; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Yao, Xiao-Jie; Weis, William I.; Stevens, Raymond C.; Kobilka, Brian K.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Kobilka, B. K.; Deupi, X.; Cherezov, V.; Rasmussen, S. G. F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1266-1273</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The β2-adrenergic receptor (β2AR) is a well-studied prototype for heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors (GPCRs) that respond to diffusible hormones and neurotransmitters.  To overcome the structural flexibility of the β2AR and to facilitate its crystn., we engineered a β2AR fusion protein in which T4 lysozyme (T4L) replaces most of the third intracellular loop of the GPCR ("β2AR-T4L") and showed that this protein retains near-native pharmacol. properties.  Anal. of adrenergic receptor ligand-binding mutants within the context of the reported high-resoln. structure of β2AR-T4L provides insights into inverse-agonist binding and the structural changes required to accommodate catecholamine agonists.  Amino acids known to regulate receptor function are linked through packing interactions and a network of hydrogen bonds, suggesting a conformational pathway from the ligand-binding pocket to regions that interact with G proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6gmrMhXnVl7Vg90H21EOLACvtfcHk0lj-kuNVdWtGmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7J&md5=d422ff1e7a28719848c0e16fede045db</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150609%26sid%3Dliteratum%253Aachs%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DYao%26aufirst%3DX.-J.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DGPCR%2520engineering%2520yields%2520high-resolution%2520structural%2520insights%2520into%2520%25CE%25B22-adrenergic%2520receptor%2520function%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1266%26epage%3D1273%26doi%3D10.1126%2Fscience.1150609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Brooks, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruccoleri, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olafson, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">States, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">CHARMM: A program for macromolecular energy, minimization, and dynamics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span><span class="refDoi"> DOI: 10.1002/jcc.540040211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fjcc.540040211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1983&pages=187-217&author=B.+R.+Brooksauthor=R.+E.+Bruccoleriauthor=B.+D.+Olafsonauthor=D.+J.+Statesauthor=S.+Swaminathanauthor=M.+Karplus&title=CHARMM%3A+A+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations&doi=10.1002%2Fjcc.540040211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM:  a program for macromolecular energy, minimization, and dynamics calculations</span></div><div class="casAuthors">Brooks, Bernard R.; Bruccoleri, Robert E.; Olafson, Barry D.; States, David J.; Swaminathan, S.; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-217</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">CHARMM (Chem. at HARvard Macromol. Mechanics) is a highly flexible computer program which uses empirical energy functions to model macromol. systems.  The program can read or model build structures, energy minimize them by first- or second-deriv. techniques, perform a normal mode or mol. dynamics simulation, and analyze the structural, equil., and dynamic properties detd. in these calcns.  The operations that CHARMM can perform are described, and some implementation details are given.  A set of parameters for the empirical energy function and a sample run are included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSFuoT3w87LVg90H21EOLACvtfcHk0lj-kuNVdWtGmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXit1aiu7w%253D&md5=bd639b4299ac9934f4497c1a9fe750d2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540040211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540040211%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DBruccoleri%26aufirst%3DR.%2BE.%26aulast%3DOlafson%26aufirst%3DB.%2BD.%26aulast%3DStates%26aufirst%3DD.%2BJ.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DCHARMM%253A%2520A%2520program%2520for%2520macromolecular%2520energy%252C%2520minimization%252C%2520and%2520dynamics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1983%26volume%3D4%26spage%3D187%26epage%3D217%26doi%3D10.1002%2Fjcc.540040211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Momany, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rone, R.</span><span> </span><span class="NLM_article-title">Validation of the general purpose QUANTA 3.2/CHARMm force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">888</span><span class="NLM_x">–</span> <span class="NLM_lpage">900</span><span class="refDoi"> DOI: 10.1002/jcc.540130714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1002%2Fjcc.540130714" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=888-900&author=F.+A.+Momanyauthor=R.+Rone&title=Validation+of+the+general+purpose+QUANTA+3.2%2FCHARMm+force+field&doi=10.1002%2Fjcc.540130714"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130714%26sid%3Dliteratum%253Aachs%26aulast%3DMomany%26aufirst%3DF.%2BA.%26aulast%3DRone%26aufirst%3DR.%26atitle%3DValidation%2520of%2520the%2520general%2520purpose%2520QUANTA%25203.2%252FCHARMm%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D888%26epage%3D900%26doi%3D10.1002%2Fjcc.540130714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 1. Application to neutral molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4732</span><span class="NLM_x">–</span> <span class="NLM_lpage">4739</span><span class="refDoi"> DOI: 10.1021/j100374a066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100374a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4732-4739&author=K.+Noauthor=J.+Grantauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+1.+Application+to+neutral+molecules+as+models+for+polypeptides&doi=10.1021%2Fj100374a066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  1.  Application to neutral molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4732-9</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">Modified methods are proposed for detg. net at. charges, based on an existing scheme using the method of partial equalization of orbital electronegativity.  The new methods introduce (as constraints) exptl. dipole and, where possible, quadrupole moments of a set of mols. chosen to resemble the different at. environments found in proteins.  As a consequence, the at. charges reproduce accurately the exptl. values of mol. dipole and quadrupole moments.  The electrostatic potential surface, calcd. with the new net-at.-charge set, compares well to a surface obtained from a moderate-level ab initio calcn. (6-31G**).  However, the new charges are double those obtained from studies in which the charges are detd. from a least-squares fit to the calcd. ab initio electrostatic potential (potential-derived method).  The methodol. outlined leads to a direct calcn. of the permanent mol. charge distribution represented as a set of distributed monopoles, dependent only on the connectivity of the mol.  Hence, the method circumvents assumptions concerning the transferability of charges and can therefore approx. describe any redistribution of charge d. upon assembly of amino acid residues to form a polypeptide.  The method may be of use in providing an approx. representation of mol. charge distributions in extended systems, esp. large polypeptide mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5hUCBNRO0TLVg90H21EOLACvtfcHk0lj-kuNVdWtGmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktVGgtbw%253D&md5=1783d0a7d1e5c52cb5bf93d96f16d8c6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fj100374a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a066%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25201.%2520Application%2520to%2520neutral%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4732%26epage%3D4739%26doi%3D10.1021%2Fj100374a066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">No, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheraga, H.</span><span> </span><span class="NLM_article-title">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method. 2. Application to ionic and aromatic molecules as models for polypeptides</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">4740</span><span class="NLM_x">–</span> <span class="NLM_lpage">4746</span><span class="refDoi"> DOI: 10.1021/j100374a067</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100374a067" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1990&pages=4740-4746&author=K.+Noauthor=J.+Grantauthor=M.+Jhonauthor=H.+Scheraga&title=Determination+of+net+atomic+charges+using+a+modified+partial+equalization+of+orbital+electronegativity+method.+2.+Application+to+ionic+and+aromatic+molecules+as+models+for+polypeptides&doi=10.1021%2Fj100374a067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of net atomic charges using a modified partial equalization of orbital electronegativity method.  2.  Application to ionic and aromatic molecules as models for polypeptides</span></div><div class="casAuthors">No, Kyoung Tai; Grant, J. Andrew; Jhon, Mu Shik; Scheraga, Harold A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4740-6</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">To finalize a scheme intended to calc. directly approx. charge distributions for proteins, parameters for a modified title method (PEOE) are obtained for both ionic and arom. functional groups.  For ionic groups, the electrostatic potentials of several ionic mols. (computed with the 6-31G** basis set) are used as constraints; for arom. groups, the exptl. gas-phase dipole and quadrupole moments of several arom. mols. are used as constraints.  The electrostatic potentials calcd. with the modified PEOE point charges of ionic mols. are quite close to the ab initio electrostatic potential surfaces, and the dipole and quadrupole moments calcd. with the modified PEOE charges of arom. mols. agree well with exptl. data.  The method enables the charge distribution, described as a set of atom-centered monopoles, to be calcd. directly.  This avoids assumptions concerning transferability of charges from small model compds. and describes in an approx. fashion the redistribution of charge upon assembly of amino acid residues into polypeptide mols.  The computational requirements for the procedure are negligible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdOLr-aBqHiLVg90H21EOLACvtfcHk0lgg52Dpv0LOTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitlCmsL8%253D&md5=b819eb54db46af0ad66a22d12d052c4b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fj100374a067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100374a067%26sid%3Dliteratum%253Aachs%26aulast%3DNo%26aufirst%3DK.%26aulast%3DGrant%26aufirst%3DJ.%26aulast%3DJhon%26aufirst%3DM.%26aulast%3DScheraga%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520net%2520atomic%2520charges%2520using%2520a%2520modified%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%2520method.%25202.%2520Application%2520to%2520ionic%2520and%2520aromatic%2520molecules%2520as%2520models%2520for%2520polypeptides%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1990%26volume%3D94%26spage%3D4740%26epage%3D4746%26doi%3D10.1021%2Fj100374a067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Landrum, G.</span><span> </span><span class="NLM_article-title">RDKit: Open-source chemoinformatics</span>. <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a> (accessed Nov 15,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Landrum&title=RDKit%3A+Open-source+chemoinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3DLandrum%26aufirst%3DG.%26atitle%3DRDKit%253A%2520Open-source%2520chemoinformatics%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span>OMEGA, version <span class="NLM_edition">2.5.1.4</span>; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>; <a href="https://www.eyesopen.com/omega" class="extLink">https://www.eyesopen.com/omega</a> (accessed Nov 15, 2014).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=OMEGA%2C+version+2.5.1.4%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM%3B+https%3A%2F%2Fwww.eyesopen.com%2Fomega+%28accessed+Nov+15%2C+2014%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOMEGA%26pub%3DOpenEye%2520Scientific%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Hawkins, P. C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellingson, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M. T.</span><span> </span><span class="NLM_article-title">Conformer generation with OMEGA: algorithm and validation using high quality structures from the Protein Databank and Cambridge Structural Database</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">572</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span><span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+algorithm+and+validation+using+high+quality+structures+from+the+Protein+Databank+and+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lgg52Dpv0LOTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520Protein%2520Databank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: A free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0ljixWCoSvYuUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span>SZYBKI, version <span class="NLM_edition">1.8.0.2</span>; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>; <a href="https://www.eyesopen.com/oedocking" class="extLink">https://www.eyesopen.com/oedocking</a> (accessed Nov 15, 2014).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SZYBKI%2C+version+1.8.0.2%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM%3B+https%3A%2F%2Fwww.eyesopen.com%2Foedocking+%28accessed+Nov+15%2C+2014%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSZYBKI%26pub%3DOpenEye%2520Scientific%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span>GraphPad Prism, version <span class="NLM_edition">7.00</span> for Windows; <span class="NLM_publisher-name">GraphPad Software</span>: <span class="NLM_publisher-loc">La Jolla, CA</span>; <a href="https://www.graphpad.com/scientific-software/prism/" class="extLink">https://www.graphpad.com/scientific-software/prism/</a> (accessed Sep 9, 2017).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GraphPad+Prism%2C+version+7.00+for+Windows%3B+GraphPad+Software%3A+La+Jolla%2C+CA%3B+https%3A%2F%2Fwww.graphpad.com%2Fscientific-software%2Fprism%2F+%28accessed+Sep+9%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGraphPad%2520Prism%26pub%3DGraphPad%2520Software" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Pauli, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lankin, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gödecke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaki, B. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McAlpine, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napolitano, J. G.</span><span> </span><span class="NLM_article-title">Importance of purity evaluation and the potential of quantitative 1H NMR as a purity assay: miniperspective</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9220</span><span class="NLM_x">–</span> <span class="NLM_lpage">9231</span><span class="refDoi"> DOI: 10.1021/jm500734a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500734a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslahsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9220-9231&author=G.+F.+Pauliauthor=S.-N.+Chenauthor=C.+Simmlerauthor=D.+C.+Lankinauthor=T.+G%C3%B6deckeauthor=B.+U.+Jakiauthor=J.+B.+Friesenauthor=J.+B.+McAlpineauthor=J.+G.+Napolitano&title=Importance+of+purity+evaluation+and+the+potential+of+quantitative+1H+NMR+as+a+purity+assay%3A+miniperspective&doi=10.1021%2Fjm500734a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Importance of Purity Evaluation and the Potential of Quantitative 1H NMR as a Purity Assay</span></div><div class="casAuthors">Pauli, Guido F.; Chen, Shao-Nong; Simmler, Charlotte; Lankin, David C.; Godecke, Tanja; Jaki, Birgit U.; Friesen, J. Brent; McAlpine, James B.; Napolitano, Jose G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9220-9231</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In any biomedical and chem. context, a truthful description of chem. constitution requires coverage of both structure and purity.  This qualification affects all drug mols., regardless of development stage (early discovery to approved drug) and source (natural product or synthetic).  Purity assessment is particularly crit. in discovery programs and whenever chem. is linked with biol. and/or therapeutic outcome.  Compared with chromatog. and elemental anal., quant. NMR (qNMR) uses nearly universal detection and provides a versatile and orthogonal means of purity evaluation.  Abs. qNMR with flexible calibration captures analytes that frequently escape detection (water, sorbents).  Widely accepted structural NMR workflows require minimal or no adjustments to become practical 1H qNMR (qHNMR) procedures with simultaneous qual. and (abs.) quant. capability.  This study reviews underlying concepts, provides a framework for std. qHNMR purity assays, and shows how adequate accuracy and precision are achieved for the intended use of the material.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqquFEu5lLVVrVg90H21EOLACvtfcHk0ljixWCoSvYuUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslahsLzF&md5=78900d0b03e31739e50c04c63c55dba7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm500734a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500734a%26sid%3Dliteratum%253Aachs%26aulast%3DPauli%26aufirst%3DG.%2BF.%26aulast%3DChen%26aufirst%3DS.-N.%26aulast%3DSimmler%26aufirst%3DC.%26aulast%3DLankin%26aufirst%3DD.%2BC.%26aulast%3DG%25C3%25B6decke%26aufirst%3DT.%26aulast%3DJaki%26aufirst%3DB.%2BU.%26aulast%3DFriesen%26aufirst%3DJ.%2BB.%26aulast%3DMcAlpine%26aufirst%3DJ.%2BB.%26aulast%3DNapolitano%26aufirst%3DJ.%2BG.%26atitle%3DImportance%2520of%2520purity%2520evaluation%2520and%2520the%2520potential%2520of%2520quantitative%25201H%2520NMR%2520as%2520a%2520purity%2520assay%253A%2520miniperspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9220%26epage%3D9231%26doi%3D10.1021%2Fjm500734a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Steyaert, J.; Laeremans, T.; Pardon, E.</span><span> </span><span class="NLM_article-title">Novel chimeric polypeptides for screening and drug discovery purposes</span>. WO PCT Number PCT/EP2014/051845,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.+Steyaert&author=T.+Laeremans&author=E.+Pardon&title=Novel+chimeric+polypeptides+for+screening+and+drug+discovery+purposes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSteyaert%26aufirst%3DJ.%26atitle%3DNovel%2520chimeric%2520polypeptides%2520for%2520screening%2520and%2520drug%2520discovery%2520purposes%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Yung-Chi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (Ki) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28Ki%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528Ki%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1','PDB','2RH1'); return false;">PDB: 2RH1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LDE','PDB','4LDE'); return false;">PDB: 4LDE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DLE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DLE','PDB','4DLE'); return false;">PDB: 4DLE</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01558">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81235"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01558">10.1021/acs.jmedchem.7b01558</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Description of the Web interface; workflow of the GVM procedure; similarity search and PAINS filter results; IC<sub>50</sub> values of the tested compounds; predicted binding mode and comparison to crystal structures; analysis of <sup>1</sup>H NMR, and MS of the tested compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and experimental data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_002.csv">CSV</a>)</p></li><li><p class="inline">PDB files of predicted binding modes of the 13 compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_003.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_001.pdf">jm7b01558_si_001.pdf (4.93 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_002.csv">jm7b01558_si_002.csv (0.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01558/suppl_file/jm7b01558_si_003.zip">jm7b01558_si_003.zip (236.37 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01558&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-3%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01558" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01558" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0f7906cb81995","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
