
<html lang="en"     class="pb-page"  data-request-id="e62036ab-ff45-43d2-abe2-bc36a56878ce"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-24;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01530;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein" /></meta><meta name="dc.Creator" content="Haibin  Zhou" /></meta><meta name="dc.Creator" content="Longchuan  Bai" /></meta><meta name="dc.Creator" content="Renqi  Xu" /></meta><meta name="dc.Creator" content="Yujun  Zhao" /></meta><meta name="dc.Creator" content="Jianyong  Chen" /></meta><meta name="dc.Creator" content="Donna  McEachern" /></meta><meta name="dc.Creator" content="Krishnapriya  Chinnaswamy" /></meta><meta name="dc.Creator" content="Bo  Wen" /></meta><meta name="dc.Creator" content="Lipeng  Dai" /></meta><meta name="dc.Creator" content="Praveen  Kumar" /></meta><meta name="dc.Creator" content="Chao-Yie  Yang" /></meta><meta name="dc.Creator" content="Zhaomin  Liu" /></meta><meta name="dc.Creator" content="Mi  Wang" /></meta><meta name="dc.Creator" content="Liu  Liu" /></meta><meta name="dc.Creator" content="Jennifer L.  Meagher" /></meta><meta name="dc.Creator" content="Han  Yi" /></meta><meta name="dc.Creator" content="Duxin  Sun" /></meta><meta name="dc.Creator" content="Jeanne A.  Stuckey" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases. Despite 20 years of persistent researc..." /></meta><meta name="Description" content="Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases. Despite 20 years of persistent researc..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 20, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01530" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01530" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01530" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01530" /></link>
        
    
    

<title>Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01530" /></meta><meta property="og:title" content="Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0027.jpeg" /></meta><meta property="og:description" content="Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases. Despite 20 years of persistent research efforts, targeting STAT3 has been very challenging. We report herein the structure-based discovery of potent small-molecule STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept. We first designed SI-109 as a potent, small-molecule inhibitor of the STAT3 SH2 domain. Employing ligands for cereblon/cullin 4A E3 ligase and SI-109, we obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36. SD-36 induces rapid STAT3 degradation at low nanomolar concentrations in cells and fails to degrade other STAT proteins. SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3. A single dose of SD-36 results in complete STAT3 protein degradation in xenograft tumor tissue and normal mouse tissues. SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules. SD-36 is a potent, selective, and efficacious STAT3 degrader." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01530"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01530">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/acsodf/largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/ancham/largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/cmatex/largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/jceda8/largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/jctcce/largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/langd5/largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/mamobx/largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01530&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01530&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01530&amp;href=/doi/10.1021/acs.jmedchem.9b01530" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 11280-11300</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01489" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01580" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Haibin Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haibin Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haibin++Zhou">Haibin Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Longchuan Bai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Longchuan Bai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Longchuan++Bai">Longchuan Bai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renqi Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renqi Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renqi++Xu">Renqi Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yujun Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yujun Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yujun++Zhao">Yujun Zhao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1035-2272" title="Orcid link">http://orcid.org/0000-0003-1035-2272</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianyong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianyong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianyong++Chen">Jianyong Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Donna McEachern</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Donna McEachern</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Donna++McEachern">Donna McEachern</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Krishnapriya Chinnaswamy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krishnapriya Chinnaswamy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krishnapriya++Chinnaswamy">Krishnapriya Chinnaswamy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bo Wen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Wen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Wen">Bo Wen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lipeng Dai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lipeng Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lipeng++Dai">Lipeng Dai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Praveen Kumar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Praveen Kumar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Praveen++Kumar">Praveen Kumar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao-Yie Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao-Yie Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao-Yie++Yang">Chao-Yie Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhaomin Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaomin Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaomin++Liu">Zhaomin Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mi Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mi Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mi++Wang">Mi Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liu Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liu Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liu++Liu">Liu Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer L. Meagher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer L. Meagher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+L.++Meagher">Jennifer L. Meagher</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Han Yi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Han Yi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Han++Yi">Han Yi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Duxin Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duxin Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duxin++Sun">Duxin Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeanne A. Stuckey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeanne A. Stuckey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeanne+A.++Stuckey">Jeanne A. Stuckey</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center,  Department of Internal Medicine,  Department of Pharmacology, Medical School,  Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4b38232a24262e252c0b3e26222823652e2f3e"><span class="__cf_email__" data-cfemail="67140f06080a02090027120a0e040f49020312">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01530&amp;href=/doi/10.1021%2Facs.jmedchem.9b01530" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 11280–11300</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 20, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 September 2019</li><li><span class="item_label"><b>Published</b> online</span>20 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01530" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01530</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11280%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHaibin%2BZhou%252C%2BLongchuan%2BBai%252C%2BRenqi%2BXu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D24%26contentID%3Dacs.jmedchem.9b01530%26title%3DStructure-Based%2BDiscovery%2Bof%2BSD-36%2Bas%2Ba%2BPotent%252C%2BSelective%252C%2Band%2BEfficacious%2BPROTAC%2BDegrader%2Bof%2BSTAT3%2BProtein%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11300%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01530"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4383</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">40</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01530" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Haibin&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Longchuan&quot;,&quot;last_name&quot;:&quot;Bai&quot;},{&quot;first_name&quot;:&quot;Renqi&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Yujun&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Jianyong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Donna&quot;,&quot;last_name&quot;:&quot;McEachern&quot;},{&quot;first_name&quot;:&quot;Krishnapriya&quot;,&quot;last_name&quot;:&quot;Chinnaswamy&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Wen&quot;},{&quot;first_name&quot;:&quot;Lipeng&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Praveen&quot;,&quot;last_name&quot;:&quot;Kumar&quot;},{&quot;first_name&quot;:&quot;Chao-Yie&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Zhaomin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Mi&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Liu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;L. Meagher&quot;},{&quot;first_name&quot;:&quot;Han&quot;,&quot;last_name&quot;:&quot;Yi&quot;},{&quot;first_name&quot;:&quot;Duxin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Jeanne&quot;,&quot;last_name&quot;:&quot;A. Stuckey&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;11280-11300&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01530&quot;},&quot;abstract&quot;:&quot;Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases. Despite 20 years of persistent research efforts, targeting STAT3 has been very challenging. We report herein the structure-based discovery of potent small-molecule STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept. We first designed SI-109 as a potent, small-molecule inhibitor of the STAT3 SH2 domain. Employing ligands for cereblon/cullin 4A E3 ligase and SI-109, we obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36. SD-36 induces rapid STAT3 degradation at low nanomolar concentrations in cells and fails to degrade other STAT proteins. SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3. A single dose of SD-36 results in complete STAT3 protein degradation in xenograft tumor tissue and normal mouse tissues. SD-36 achieves co&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01530&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01530" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01530&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01530" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01530&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01530" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01530&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01530&amp;href=/doi/10.1021/acs.jmedchem.9b01530" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01530" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01530" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26pmid%3D31747516%26genre%3Darticle%26aulast%3DZhou%26date%3D2019%26atitle%3DStructure-Based%2BDiscovery%2Bof%2BSD-36%2Bas%2Ba%2BPotent%252C%2BSelective%252C%2Band%2BEfficacious%2BPROTAC%2BDegrader%2Bof%2BSTAT3%2BProtein%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D24%26spage%3D11280%26epage%3D11300%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/jmcmar.2019.62.issue-24/20191226/jmcmar.2019.62.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases. Despite 20 years of persistent research efforts, targeting STAT3 has been very challenging. We report herein the structure-based discovery of potent small-molecule STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept. We first designed SI-109 as a potent, small-molecule inhibitor of the STAT3 SH2 domain. Employing ligands for cereblon/cullin 4A E3 ligase and SI-109, we obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36. SD-36 induces rapid STAT3 degradation at low nanomolar concentrations in cells and fails to degrade other STAT proteins. SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3. A single dose of SD-36 results in complete STAT3 protein degradation in xenograft tumor tissue and normal mouse tissues. SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules. SD-36 is a potent, selective, and efficacious STAT3 degrader.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24756" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24756" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Signal transducer and activator of transcription 3 (STAT3) is a member of the STAT family of transcription factors that transmits signals from cell surface receptors to the nucleus.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Persistent activation of STAT3 is often associated with poor prognosis of human cancers since activated STAT3 signaling not only provides advantages of growth, survival, and metastasis to tumor cells but also suppresses the antitumor immune response.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> STAT3 is an attractive therapeutic target for human cancer and other human diseases.</div><div class="NLM_p">In the canonical mechanism, STAT3 is recruited from the cytosol to the cell membrane where it interacts with different cytokine receptors through its Src homology 2 (SH2) domain. This results in phosphorylation of its Tyr705 residue (Y705) and its subsequent dimerization and translocation into the nucleus for gene transcription.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It has been proposed that dimerization of STAT3 is required for its translocation from cytosol into the nucleus and for its transcriptional activity, and pharmacological inhibition of STAT3 SH2 domain could effectively block the dimerization and transcriptional activity of STAT3.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Consequently, considerable efforts have been made in the past 2 decades to develop small molecular inhibitors of the STAT3 SH2 domain.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a></div><div class="NLM_p">Because STAT3 homodimerization is mediated by a phosphotyrosine-containing peptide from one monomer and a well-defined binding pocket in the SH2 domain of the second monomer, peptidomimetic inhibitors have been designed based upon STAT3 phosphotyrosine-containing peptides. Although previous peptidomimetic inhibitors have been reported to achieve high binding affinities to STAT3 in biochemical binding assays, they are generally not cell-permeable due to the presence of their negatively charged phosphate group and their peptidic characteristics.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Non-peptide small-molecule inhibitors of the STAT3 SH2 domain have been reported, but they often have weak to moderate binding affinities to STAT3, making it difficult to establish firmly that STAT3 is the primary cellular target.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Furthermore, because STAT3 shares a high sequence homology of its SH2 domain with other STAT members, it has been challenging to design SH2 domain inhibitors for STAT3 that are highly selective over other STAT proteins. Significantly, in contrast to initial hypotheses, it has now become clear that the monomeric STAT3 protein can translocate from cytosol into the nucleus and is transcriptionally active.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Consequently, even if a potent, cell-permeable, and highly selective inhibitor of the STAT3 SH2 domain is ultimately discovered, it can be predicted that it would only be able to partially block the gene transcriptional activity of STAT3. Accordingly, there is a clear need to develop a new therapeutic approach to targeting STAT3.</div><div class="NLM_p">Therefore, in a therapeutic setting, specific down-regulation or knock-down of STAT3 may be much more effective than inhibition of its dimerization. To support this idea, AZD9150,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> which was designed as a STAT3 antisense oligonucleotide (ASO), has shown promising preclinical and clinical activity, despite some major limitations associated with ASO molecules.<a onclick="showRef(event, 'ref3 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref7">(3,7)</a></div><div class="NLM_p last">Recently, the proteolysis targeting chimeras (PROTAC) strategy has gained momentum for its promise in the discovery and development of an entirely new type of therapeutics for the treatment of human diseases.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8−11)</a> A PROTAC molecule is a bifunctional small molecule designed to recruit cellular degradation machinery to induce targeted protein degradation.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> We therefore proposed to design PROTAC degraders of STAT3 as a new strategy to effectively target STAT3. In the present study, we report our structure-based discovery of SD-36 as a potent, exceptionally selective, and highly efficacious PROTAC degrader of the STAT3 protein.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18864" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18864" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of Small Molecule STAT3 SH2 Domain Inhibitors Containing a Novel Phosphotyrosine Mimetic</h3><div class="NLM_p">Our laboratory has previously reported the design of peptidomimetic compounds with high binding affinities to the STAT3 SH2 domain, such as CJ-887 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>),<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> which binds to the human recombinant STAT3 SH2 protein with <i>K</i><sub>i</sub> = 47 nM in a fluorescence polarization (FP) assay (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S1 and Table S1</a>). Despite this high binding affinity to STAT3, CJ-887 is ineffective in inhibition of STAT3 Y705 phosphorylation in cells, suggesting its poor cell permeability. We hypothesized that the poor cell-permeability of CJ-887 could be largely attributed to its doubly ionized natural phosphate group.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> A second major issue associated with CJ-887 is its expected susceptibility to degradation by protein tyrosine phosphatases (PTPs).<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> To overcome these major limitations, we have performed extensive optimization of CJ-887, with the key steps outlined in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0001.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of potent and cell-permeable small molecule inhibitors of the STAT3 SH2 domain: chemical structures and binding affinities of our previously reported compound CJ-887 and newly designed and synthesized inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Cyclization of the amino group with the phenyl group in CJ-887 forms an indole (<b>2</b>), which converts CJ-887, a phosphotyrosine into a conformationally constrained, non-amino acid phosphate.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The resulting compound <b>2</b> binds to STAT3 with a <i>K</i><sub>i</sub> value of 25 nM and is thus twice as potent as CJ-887 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Table S1</a>). To address the potential susceptibility of the phosphate ester bond in <b>2</b> to hydrolysis by PTPs, we replaced the -OPO<sub>3</sub>H<sub>2</sub> group with a non-hydrolyzable -CH<sub>2</sub>-PO<sub>3</sub>H<sub>2</sub> group,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> yielding compound <b>3</b>. Compound <b>3</b> binds to STAT3 with a <i>K</i><sub>i</sub> value of 761 nM and is thus 30 times weaker than <b>2</b>. We reasoned that the reduced binding affinity of <b>3</b> is partially attributable to the weaker acidity of the -CH<sub>2</sub>PO<sub>3</sub>H<sub>2</sub> group compared to that of the -OPO<sub>3</sub>H<sub>2</sub> group. To test this idea, we replaced the -CH<sub>2</sub>PO<sub>3</sub>H<sub>2</sub> group with -CF<sub>2</sub>PO<sub>3</sub>H<sub>2</sub>, which has been used previously in the design of phosphotyrosine mimetics.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17">(13−17)</a> Compound <b>4</b>, containing the -CF<sub>2</sub>PO<sub>3</sub>H<sub>2</sub> group binds to STAT3 with a very high affinity (<i>K</i><sub>i</sub> = 7 nM)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and is >100 times more potent than <b>3</b> and 7 times more potent than the initial peptidomimetic <b>1</b> (CJ-887).</div><div class="NLM_p">Our predicted binding model of compound <b>4</b> in a complex with the STAT3 SH2 domain (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S2</a>) suggested that part of the eight-membered ring in <b>4</b> is exposed to the solvent environment. To facilitate the synthesis of STAT3 degraders, we replaced one of the carbon atoms in the eight-membered ring with a nitrogen atom, producing <b>5</b> with a methyl substituent,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and <b>6</b> with an acetyl substituent. Both <b>5</b> and <b>6</b> bind to STAT3 with the same high affinity, achieving <i>K</i><sub>i</sub> values of 14 nM. Next we installed a second phenyl group at the α position of the benzyl group in <b>5</b> and <b>6</b> to restrict the conformation of the original phenyl group, yielding <b>7</b> and SI-109 (<b>8</b>),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> respectively. Compounds <b>7</b> and SI-109 bind to STAT3 with <i>K</i><sub>i</sub> values of 12 nM and 14 nM, respectively.</div><div class="NLM_p">To directly compare the phosphotyrosine mimetic used in compounds <b>4</b>–<b>8</b> with other phosphotyrosine mimetics, we used SI-109 as the template molecule and synthesized <b>9</b>, <b>10</b>, and <b>11</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Compound <b>9</b> in which the -CF<sub>2</sub>PO<sub>3</sub>H<sub>2</sub> moiety in SI-109 is replaced with a phosphate (-OPO<sub>3</sub>H<sub>2</sub>) is 2 times less potent than SI-109. Compound <b>10</b> in which the -CF<sub>2</sub>PO<sub>3</sub>H<sub>2</sub> moiety in SI-109 is replaced with -CH<sub>2</sub>PO<sub>3</sub>H<sub>2</sub> is 14 times less potent than SI-109. Compound <b>11</b> containing 4-phosphonodifluoromethylcinnamate,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> binds to STAT3 with <i>K</i><sub>i</sub> value of 164 nM and is >10 times less potent than SI-109. These binding data show that the combination of the bicyclic indole with the difluoromethylphosphate moiety has yielded an excellent phosphotyrosine mimetic, which is used in compounds <b>4</b>–<b>8</b>.</div><div class="NLM_p last">Hence, optimization of <b>1</b> (CJ-887) has yielded a series of high-affinity, small-molecule ligands for the STAT3 SH2 domain, including SI-109 (<b>8</b>), which was employed in our subsequent design of PROTAC STAT3 degraders.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Determination of the Cocrystal Structure of SI-109 in a Complex with STAT3</h3><div class="NLM_p">To understand the structural basis for the high binding affinity of our designed STAT3 inhibitors, we determined the cocrystal structure of SI-109 in a complex with the STAT3 SH2 domain at a resolution of 3.15 Å (PDB access code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ">6NUQ</a>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Supporting Information, Table S2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0002.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of SI-109 with STAT3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ">6NUQ</a>). (A) The <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density generated from an omit map contoured at 3σ shows placement of atoms of SI-109. (B) Detailed interactions of STAT3 with SI-109. SI-109 (cyan) and side chains of interacting residues of STAT3 with SI-109 are shown in stick form. Hydrogen bonds are depicted as dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Comparison of the cocrystal structure of SI-109 in a complex with STAT3 and the STAT3 SH2 dimeric crystal structure showed that SI-109 captures all the critical interactions with STAT3 that are observed in the STAT3 dimeric complex. The difluoromethylphosphate in SI-109 forms extensive hydrogen bonding interactions with the side chains of Arg609, Ser611, and Ser613 and the backbone amide nitrogen atoms of Glu612 and Ser613. An additional hydrogen bond is formed between the pro-(<i>S</i>)-F atom of the difluoromethyl group with Arg609. The indole moiety in SI-109 sits in a shallow cleft formed between Pro639 and Ser636 of STAT3. The glutamine group in SI-109 forms two hydrogen bonds with the side chain of Gln644. The backbone carbonyl moiety of the glutamine group in SI-109 forms two water-mediated hydrogen bonds with the hydroxyl group of Tyr640 and the amino group of Lys658. The pro-(<i>S</i>)-phenyl group of the diphenylmethyl group in SI-109 interacts with Leu666, Met660, and Ile659, whereas a pro-(<i>R</i>)-phenyl group has no specific interactions with any residue in STAT3 and is fully exposed to solvent environment. Although no specific interactions are observed between the [8,5]-bicyclic ring system in SI-109 and the STAT3 protein, optimal hydrogen bonds are formed between the amide groups in the proximity of the [8,5]-bicyclic ring and the protein. Specifically, the amino group tethered to the eight-membered ring forms an optimal hydrogen bond with the backbone carbonyl group of Ser636 and the carbonyl group attached to the five-membered ring forms a hydrogen bond with the hydroxyl group of Tyr657 and additionally a water-mediated hydrogen bond with the backbone carbonyl group of Lys658. Hence, the bicyclic ring system functions as a scaffold which ensures that the difluoromethylphosphonic acid and the glutamine moieties in SI-109 are ideally positioned and oriented to achieve optimal interactions with STAT3.</div><div class="NLM_p last">This cocrystal structure of SI-109 in a complex with STAT3 provides the structural basis for its high binding affinity to STAT3 and facilitated our design of STAT3 degraders.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Design of PROTAC STAT3 Degraders</h3><div class="NLM_p">VHL/cullin 2 and cereblon (CRBN)/cullin 4A E3 ligase complexes have been successfully employed in the design of PROTAC degraders for different proteins.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8−11)</a> For the VHL/cullin 2 E3 ligase complex, high-affinity peptidomimetic ligands have been discovered<a onclick="showRef(event, 'ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21">(18−21)</a> and have been successfully used for the design of highly potent PROTAC degraders for different proteins.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8−11)</a> For the cereblon (CRBN)/cullin 4A E3 ligase complex, ligands for cereblon, such as lenalidomide and pomalidomide, are small-molecule drugs with molecular weights between 250 and 260, which are ideal for the design of PROTAC degraders with excellent physiochemical properties. Accordingly, we have decided to employ lenalidomide and pomalidomide (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) for the design of STAT3 degraders.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0003.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of putative STAT3 PROTAC degraders based upon cereblon ligands and our STAT3 SH2 domain inhibitor SI-109.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Our cocrystal structure of SI-109 in a complex with the STAT3 SH2 domain shows that the amino group of the eight-membered ring in SI-109 is exposed to the solvent (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) and is a suitable site at which to tether a ligand for an E3 ligase complex in the design of PROTAC STAT3 degraders. We have designed and synthesized a series of potential STAT3 degraders linking this site in SI-109 with a cereblon ligand through a linker (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Determination of the Optimal Linker Length</h3><div class="NLM_p">Our previous studies demonstrated that the length of the linker plays a key role in protein degradation potencies in our BET and MDM2 PROTAC degraders.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25">(22−25)</a> We synthesized and evaluated a series of putative STAT3 degraders with different linker lengths (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. STAT3 Degraders with Various Linker Lengths</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0017.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0018.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_p">An acute myeloid leukemia (AML) cell line, Molm-16 expresses high levels of phosphorylated STAT3 at the Tyr705 residue (pSTAT3<sup>Y705</sup>) protein and total (phosphorylated and unphosphorylated) STAT3 protein.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We thus used the Molm-16 cell line to evaluate our putative STAT3 degraders for their ability to degrade both the total STAT3 protein and pSTAT3<sup>Y705</sup> protein. Because PROTAC degraders can induce protein degradation rapidly, we treated the Molm-16 cells for 4 h with our putative STAT3 degraders and obtained the data shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S3</a>.</div><div class="NLM_p">In this study, we examined the degradation of both the total STAT3 protein and pSTAT3<sup>Y705</sup> protein for our designed STAT3 degraders and found that our STAT3 degraders induce degradation of the total STAT3 protein and pSTAT3<sup>Y705</sup> protein with very similar potencies. Hence, we determined the DC<sub>50</sub> value, the concentration required to reduce the protein level by 50% of the total STAT3 protein for each compound, and used this value to establish the structure–activity relationships.</div><div class="NLM_p">Western blotting data show that the STAT3 inhibitor SI-109 has no effect on the level of total STAT3 protein and only modestly decreases the level of the pSTAT3<sup>Y705</sup> protein at concentrations up to 1 μM. Compound <b>12</b>, with a total of six non-hydrogen atoms in its linker between SI-109 and the cereblon ligand, has no effect on the level of total STAT3 protein up to 1 μM and reduces pSTAT3<sup>Y705</sup> protein only minimally at 1 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S3</a>). However, with higher concentrations, <b>12</b> dose-dependently reduces the levels of both the total STAT3 and pSTAT3<sup>Y705</sup> proteins and has an DC<sub>50</sub> value of 2.63 μM for the total STAT3 protein (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S3</a>). Increasing the length of the linker in <b>12</b> by one carbon yielded <b>13</b>, which effectively reduces the levels of both the total STAT3 and pSTAT3<sup>Y705</sup> proteins and with DC<sub>50</sub> = 0.26 μM, is 10 times more potent than <b>12</b>. Increasing the linker length in <b>13</b> by one additional methylene group yielded SD-36 (<b>14</b>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> SD-36 is very effective and potent in reducing the levels of the total STAT3 and pSTAT3<sup>Y705</sup> proteins and achieves a DC<sub>50</sub> of 0.06 μM.</div><div class="NLM_p last">Increasing the linker length by introduction of one more methylene group in SD-36 resulted in <b>15</b>, which has a DC<sub>50</sub> of 0.09 μM and is thus similarly potent as SD-36. Increasing the linker length in <b>15</b> by two methylenes yielded <b>16</b>, which has DC<sub>50</sub> = 0.15 μM and is thus 2–3 times less potent than SD-36. Our data thus established that a linker length of 8–9 atoms, as in SD-36 or <b>15</b>, is optimal for these STAT3 degraders to reduce the levels of both the total STAT3 and pSTAT3<sup>Y705</sup> proteins.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Investigation of the Linker Composition</h3><div class="NLM_p">Our previous studies showed that the linker composition has a major effect on the ability of our MDM2 and BET PROTAC degraders to induce degradation of the target proteins.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> Accordingly, we investigated the linker composition using SD-36 as the template compound and obtained the data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S4</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. STAT3 Degraders with Various Linker Compositions</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0019.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0020.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">Changing the amide group in SD-36 to an amino group yielded <b>17</b>, which has DC<sub>50</sub> = 0.65 μM and is >10 times less potent than SD-36 in reducing the levels of total STAT3 and pSTAT3<sup>Y705</sup> proteins (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S4A</a>). Conversion of the amide group into a carbamate or urea group led to <b>18</b> and <b>19</b>, respectively. While <b>18</b> is about 4 times less potent than SD-36 in inducing STAT3 degradation, <b>19</b> is as potent as SD-36. Replacing a CH<sub>2</sub>-CH<sub>2</sub> linker moiety in SD-36 with a pyrazole ring in either side of the linker resulted in <b>20</b> and <b>21</b>, respectively, and both compounds are 3–4 times less potent than SD-36. Replacement of a CH<sub>2</sub>-CH<sub>2</sub>-CH<sub>2</sub> linker moiety in SD-36 with a piperidine ring yielded <b>22</b> or <b>23</b>. Compound <b>22</b> is 3 times less potent than SD-36, and <b>23</b> is 40 times less potent than SD-36 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S4B</a>).</div><div class="NLM_p">Because both <b>17</b> and <b>23</b> contain a linker with a positively charged group and are much less potent than SD-36, a positive charge in the linker of these degraders would appear to be detrimental to their ability to degrade STAT3 protein. To test this idea further, we changed the amino group in <b>23</b> to an amide, affording compound <b>24</b>. Although <b>24</b> is still 6 times less potent than SD-36, it is more potent than <b>23</b> in reducing the levels of the total STAT3 and the pSTAT3<sup>Y705</sup> proteins, supporting the hypothesis. Changing the ethynyl group in SD-36 to -CH<sub>2</sub>-CH<sub>2</sub>- and -NH-CH<sub>2</sub>- led to compounds <b>25</b> and <b>26</b>, respectively, both of which are 2–3 times less potent than SD-36 in reducing the levels of STAT3 and pSTAT3<sup>Y705</sup>.</div><div class="NLM_p last">Taken together, these data show that both the length and the composition of the linker play an important role in the ability of our designed STAT3 degraders to reduce the levels of STAT3 and pSTAT3<sup>Y705</sup>, and the optimal linkers are those used in SD-36 and compound <b>19</b>.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Examination of the CRBN Ligand Portion in SD-36</h3><div class="NLM_p">We next performed modifications of the CRBN ligand portion in SD-36 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. STAT3 Degraders with Modifications of the Cereblon Ligand Portion</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0021.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0022.gif" alt="" id="fx6" /></img><div></div></div><div class="NLM_p">Thalidomide and lenalidomide bind to CRBN with similar modes and affinities.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> We replaced the lenalidomide moiety in SD-36 with a thalidomide moiety, producing <b>27</b>. Surprisingly, <b>27</b> has DC<sub>50</sub> = 3.54 μM and is thus >50 times less potent than SD-36 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p last">The cocrystal structure of lenalidomide in a complex with CRBN suggests two additional linking positions in the phenyl ring of lenalidomide, and accordingly, we synthesized <b>28</b> and <b>29</b>. While these two compounds effectively degrade STAT3 and pSTAT3<sup>Y705</sup>, both are 5–10 times less potent than SD-36 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S5</a>).</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Modifications of the STAT3 Ligand Portion of SD-36</h3><div class="NLM_p">The cocrystal structure of SI-109 in a complex with STAT3 shows that one of the phenyl groups lacks specific interactions with STAT3 protein (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). We modified this part of the STAT3 ligand in SD-36 to investigate the effect on STAT3 degradation (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. STAT3 Degraders with Modified STAT3 Ligand Portion</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0023.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0024.gif" alt="" id="fx8" /></img><div></div></div><div class="NLM_p last">Removal of one phenyl group from SD-36 yielded compound <b>30</b>, which is 4 times less potent than SD-36 in reducing STAT3 protein (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S6</a>). Removal of both the phenyl groups from SD-36 resulted in <b>31</b>, which achieve DC<sub>50</sub> = 0.64 μM and is thus 10 times less potent than SD-36. These data indicate that the two phenyl groups in SD-36 are important for SD-36 to achieve its high potency in inducing degradation of STAT3.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Evaluation of Various Phosphotyrosine Mimetics</h3><div class="NLM_p">In SD-36, we employed 5-(difluoro(phosphoro)methyl)-1<i>H</i>-indole-2-carboxylic acid to mimic the phosphotyrosine group. To investigate the importance of this phosphotyrosine mimic in achieving potent and effective STAT3 degradation by SD-36, we synthesized <b>32</b>, <b>33</b>, and <b>34</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) with three different phosphotyrosine mimetics.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. STAT3 Degraders with Modifications of the STAT3 Ligand Portion</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0025.gif" alt="" id="fx9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0026.gif" alt="" id="fx10" /></img><div></div></div><div class="NLM_p last">Compounds <b>32</b> and <b>33</b> fail to degrade both the total STAT3 and pSTAT3<sup>Y705</sup> proteins at concentrations up to 10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S7</a>). Compound <b>34</b> reduces the levels of both the total STAT3 and pSTAT3<sup>Y705</sup> proteins with DC<sub>50</sub> = 4.45 μM and is >70-times less potent than SD-36 (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S7</a>). Hence, the phosphotyrosine mimic in SD-36 is critically important for achieving potent and effective degradation of the total STAT3 and pSTAT3<sup>Y705</sup> proteins.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Design and Synthesis of a Control Compound for SD-36</h3><div class="NLM_p">To facilitate our investigation of the cellular mechanism of action of our potent STAT3 degrader SD-36, we synthesized SD-36Me (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), in which a methyl group was installed to block the binding of SD-36 to cereblon.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0004.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design of SD-36Me as a control compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">SD-36Me is completely ineffective in inducing STAT3 degradation in Molm-16 cells at concentrations up to 10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S8</a>), clearly suggesting that the degradation of STAT3 by SD-36 is cereblon and cullin-4A ligase-dependent, consistent with the PROTAC mechanism. We employed SD-36Me and the STAT3 inhibitor SI-109 as control compounds for further investigation of the activity, specificity, and cellular molecular mechanism of SD-36.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Evaluation of Representative Degraders in the SU-DHL-1 Lymphoma Cell Line</h3><div class="NLM_p">The SU-DHL-1 anaplastic large cell lymphoma (ALCL) cell line has a high level of pSTAT3<sup>Y705</sup> protein,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and we evaluated the ability of several representative STAT3 degraders in the SU-DHL-1 cell line and obtained the data shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S9</a>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Degradation Potencies of STAT3 Protein and Cell Growth-Inhibitory Activities of STAT3 Degraders in the Molm-16 and SU-DHL-1 Cell Lines</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">Molm-16</th><th class="rowsep1 colsep0" colspan="2" align="center">SU-DHL-1</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">DC<sub>50</sub> (STAT3, μM), 4 h</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> ± SD (μM), 4 days</th><th class="colsep0 rowsep0" align="center">DC<sub>50</sub> (STAT3, μM), 16 h</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> ± SD (μM), 4 days</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SI-109 (<b>8</b>)</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">6.45 ± 4.16</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">2.63</td><td class="colsep0 rowsep0" align="left">0.95 ± 0.081</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">0.26</td><td class="colsep0 rowsep0" align="left">0.078 ± 0.029</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SD-36 (<b>14</b>)</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">0.013 ± 0.010</td><td class="colsep0 rowsep0" align="left">0.028</td><td class="colsep0 rowsep0" align="left">0.61 ± 0.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">0.09</td><td class="colsep0 rowsep0" align="left">0.025 ± 0.007</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">0.15</td><td class="colsep0 rowsep0" align="left">0.042 ± 0.004</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">0.65</td><td class="colsep0 rowsep0" align="left">0.147 ± 0.027</td><td class="colsep0 rowsep0" align="left">0.919</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.128 ± 0.050</td><td class="colsep0 rowsep0" align="left">0.049</td><td class="colsep0 rowsep0" align="left">2.37 ± 0.70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">0.05</td><td class="colsep0 rowsep0" align="left">0.091 ± 0.081</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">0.43 ± 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">0.18</td><td class="colsep0 rowsep0" align="left">0.241 ± 0.246</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.232 ± 0.175</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">0.18</td><td class="colsep0 rowsep0" align="left">0.099 ± 0.054</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">2.47</td><td class="colsep0 rowsep0" align="left">0.36 ± 0.056</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">0.38</td><td class="colsep0 rowsep0" align="left">0.067 ± 0.032</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">0.16</td><td class="colsep0 rowsep0" align="left">0.063 ± 0.016</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">0.09</td><td class="colsep0 rowsep0" align="left">0.059 ± 0.016</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">3.54</td><td class="colsep0 rowsep0" align="left">0.582 ± 0.654</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">0.41</td><td class="colsep0 rowsep0" align="left">0.104 ± 0.071</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">0.36</td><td class="colsep0 rowsep0" align="left">0.711 ± 0.262</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">0.210 ± 0.109</td><td class="colsep0 rowsep0" align="left">0.055</td><td class="colsep0 rowsep0" align="left">3.24 ± 0.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">0.64</td><td class="colsep0 rowsep0" align="left">0.226 ± 0.114</td><td class="colsep0 rowsep0" align="left">0.89</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>2</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>2</td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">nt<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">4.45</td><td class="colsep0 rowsep0" align="left">1.25 ± 0.71</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SD-36Me (<b>35</b>)</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">5.21 ± 6.04</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Not tested.</p></div></div></div><div class="NLM_p">SD-36 effectively and potently reduces the levels of STAT3 and pSTAT3<sup>Y705</sup> with DC<sub>50</sub> of 28 nM in the SU-DHL-1 cell line with a 16 h treatment time (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S9</a>). SI-109 and SD-36Me had no effect on the levels of STAT3 and activated pSTAT3<sup>Y705</sup> at concentrations up to 10 μM in SU-DHL-1 cells.</div><div class="NLM_p">Compound <b>19</b>, which potently induces degradation of total STAT3 and pSTAT3<sup>Y705</sup> proteins in Molm-16 cells with DC<sub>50</sub> = 50 nM, is also potent and effective in reducing STAT3 and pSTAT3<sup>Y705</sup> protein levels in SU-DHL-1 cells with a DC<sub>50</sub> value of 11 nM.</div><div class="NLM_p">Compounds <b>18</b> and <b>30</b> are 4 times less potent than SD-36 in degradation of STAT3 protein in Molm-16 cells, and both compounds are 2 times less potent than SD-36.</div><div class="NLM_p">Compounds <b>17</b> and <b>31</b>, which are >10 times less potent than SD-36 in Molm-16 cells, are 30 times less potent than SD-36 in SU-DHL-1 cells.</div><div class="NLM_p last">Compounds <b>27</b> and <b>34</b> have DC<sub>50</sub> values of 3–5 μM in the Molm-16 cell line, but in the SU-DHL-1 cell line, both compounds are ineffective in reducing the levels of STAT3 and pSTAT3<sup>Y705</sup> proteins in SU-DHL-1 cells up to 5 μM.</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Evaluation of Degradation Selectivity of SD-36 on Other STAT Proteins in Cells</h3><div class="NLM_p">There are seven STAT family members, STAT1, -2, -3, -4, -5A, -5B, and -6, which share a highly homologous SH2 domain. To investigate the degradation selectivity of SD-36, we evaluated SD-36 for its selectivity in both Molm-16 and SU-DHL-1 cells with SI-109 and SD-36Me included as controls.</div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, SD-36 is very effective in reducing the level of STAT3 protein at concentrations as low as 0.1 μM but it has little or no effect on STAT1, STAT2, STAT4, STAT5A/B, and STAT6 proteins in both Molm-16 cell and SU-DHL-1 cell lines up to 10 μM. Hence, SD-36 demonstrates >100-fold selectivity for degradation of STAT3 over other STAT members in both Molm-16 cell and SU-DHL-1 cell lines. The control compound SD-36Me has no effect on the levels of any STAT protein at 10 μM. Similarly, the STAT3 inhibitor SI-109 has no effect on the levels of any STAT protein.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0005.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western blotting analysis of STAT proteins in Molm-16 and SU-DHL-1 cells treated with SD-36 or control compounds SD-36Me and SI-109. (A) Molm-16 cells were treated with SD-36 at 0.01–10 μM or SD-36Me and SI-109 at 10 μM for 4 h. (B) SU-DHL-1 cells were treated with SD-36 at 0.01–10 μM or SD-36Me and SI-109 at 10 μM for 18 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our data therefore show that SD-36 achieves an exceptional degradation selectivity for STAT3 over other STATs (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><div class="NLM_p last">To gain a further insight into the degradation selectivity of SD-36 for STAT3 over other STAT proteins, we determined its binding affinities to different STAT proteins.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Our binding data showed that while SD-36 has a high affinity for STAT3 with a <i>K</i><sub>d</sub> value of 44 nM, it also displays a good affinity to STAT1 and STAT4 with <i>K</i><sub>d</sub> values of approximately 1 μM.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> SD-36 has a much weaker affinity to other STAT proteins tested.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Hence, although SD-36 binds to STAT1 and STAT4 with a good affinity, it fails to degrade these two STAT proteins, as well as other STAT proteins in cells. We posit that the high degradation selectivity for SD-36 is achieved through the formation of a stable and productive degradation ternary complex consisting of STAT3:SD-36:CRBN/cullin4A, as compared to much less stable and productive ternary complexes for other STAT proteins.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Further investigations are needed to fully understand the exceptional STAT3 degradation selectivity for SD-36.</div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Evaluation of STAT3 Degraders for Their Growth-Inhibitory Activities</h3><div class="NLM_p">As SD-36 is able to effectively and selectively degrade STAT3 protein in Molm-16 and SU-DHL-1 cells, we therefore tested all of our STAT3 degraders reported in this study for their ability to inhibit cell growth in the Molm-16 cell line and a number of representative STAT3 degraders in the SU-DHL-1 cell line. SI-109 and SD-36Me were included as control compounds in the cell growth experiments. The data are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>.</div><div class="NLM_p">In the Molm-16 cell line, SD-36 exerts a potent growth-inhibitory activity with IC<sub>50</sub> = 13 nM after a 4-day treatment. The control compound SD-36Me shows modest growth-inhibitory activity with IC<sub>50</sub> = 5.21 μM and is 400 times less potent than SD-36.</div><div class="NLM_p">There is an excellent correlation between the STAT3 degradation potencies and cell growth inhibitory activities for these STAT3 degraders in the Molm-16 cell line. All compounds that effectively and potently degrade STAT3 protein also potently inhibit cell growth in the Molm-16 cell line. Compounds such as <b>32</b> and <b>33</b>, which fail to degrade STAT3 protein up to 10 μM, also display very weak activity in inhibition of cell growth.</div><div class="NLM_p">In the SU-DHL-1 line, SD-36 effectively inhibits cell growth and achieves IC<sub>50</sub> values of 0.61 μM, while STAT3 inhibitors SI-109 and SD-36Me have no effect on cell viability up to 10 μM. Compound <b>19</b>, which effectively degrades STAT3 with DC<sub>50</sub> = 28 nM, displays an IC<sub>50</sub> value of 0.43 μM. Compounds <b>18</b> and <b>30</b>, which achieve DC<sub>50</sub> values of 49 nM and 55 nM, respectively, have IC<sub>50</sub> values of 2.37 μM and 3.24 μM, respectively, in the SU-DHL-1 line. Compounds <b>17</b> and <b>31</b>, which have DC<sub>50</sub> values of 0.92 μM and 0.89 μM, respectively, have minimal activity in inhibition of cell growth with IC<sub>50</sub> values of >10 μM. Compound <b>34</b>, which fails to degrade STAT3 up to 10 μM, is also ineffective in inhibition of cell growth (IC<sub>50</sub> > 10 μM). In general, STAT3 degraders show weaker cell growth inhibitory activity in the SU-DHL-1 line than in the Molm-16 cell line.</div><div class="NLM_p last">Hence, effective degradation of STAT3 results in robust growth inhibition in both Molm-16 and SU-DHL-1 cell lines, and there is a good correlation between STAT3 degradation and cell growth inhibition in both cell lines.</div></div><div id="sec2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Pharmacodynamics Evaluation of SD-36 in Native and Xenograft Tumor Tissues in Mice</h3><div class="NLM_p">Our cellular data demonstrate that SD-36 and <b>19</b> are two potent STAT3 degraders. We performed pharmacodynamics (PD) experiments to evaluate the ability of SD-36 and <b>19</b> to degrade STAT3 protein in mice.</div><div class="NLM_p">We first investigated SD-36 and <b>19</b> for their ability to degrade STAT3 protein in native mouse tissue. Western blotting analysis showed that a single intravenous dose of both SD-36 and <b>19</b> at 25 mg/kg is very effective in reducing STAT3 protein in native mouse spleen tissue with the effect persisting for more than 48 h, and SD-36 appears to be more effective than <b>19</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). We thus chose to further evaluate SD-36 <i>in vivo</i>.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0006.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Pharmacodynamics study of SD-36 and <b>19</b> in native mouse spleen tissue in CD-1 mice (A) and SD-36 in the Molm-16 xenograft tumors in SCID mice (B). CD-1 and SCID mice were treated intravenously with a single dose of SD-36 or <b>19</b> as indicated. Native spleen tissue or tumor lysates were analyzed by immunoblotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We evaluated SD-36 for its ability to degrade STAT3 in the Molm-16 xenograft tumor tissue in mice (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Our Western blotting data showed that a single dose of SD-36 at 50 mg/kg is very effective in reducing the levels of both total STAT3 and pSTAT3<sup>Y705</sup> proteins in xenograft tumor tissue. Both total STAT3 and pSTAT3<sup>Y705</sup> proteins were reduced by >95% after 1 h, indicating a rapid degradation of STAT3 <i>in vivo</i>. Even 72 h (3 days) after dosing, SD-36 still reduced the levels of total STAT3 and pSTAT3<sup>Y705</sup> proteins by >90% as compared to the control. Our PD data thus demonstrate that a single dose of SD-36 is highly effective in inducing near complete elimination of STAT3 protein in tumor tissue in mice with the effect persisting for >3 days.</div></div><div id="sec2_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Antitumor Activity of SD-36 in the Molm-16 Xenograft Model in Mice</h3><div class="NLM_p">Upon the basis of its impressive PD data in the Molm-16 xenograft tumor tissue, we evaluated SD-36 for its efficacy in the Molm-16 xenograft model in mice at different doses and schedules.</div><div class="NLM_p">Administered to mice at either 25 or 50 mg/kg three times per week for 3 weeks intravenously, SD-36 is capable of inducing complete tumor regression (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). In fact, complete tumor regression was achieved after dosing for only 2 weeks at either 25 or 50 mg/kg and persisted for more than 2 weeks after termination of the treatment.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0007.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <i>In vivo</i> antitumor activity of SD-36 in the Molm-16 xenograft model in mice: (A) tumor growth; (B) percentage of animal body weight change.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because a single dose of SD-36 at 50 mg/kg is capable of reducing STAT3 protein levels for >3 days, we evaluated its efficacy with a weekly dosing schedule. SD-36 at 50 mg/kg administered once weekly for 3 weeks is very effective in inhibition of tumor growth (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Furthermore, SD-36 at 100 mg/kg weekly dosing was capable of achieving complete tumor regression.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In the efficacy experiment, SD-36 caused no weight loss or other signs of toxicity (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).</div><div class="NLM_p last">Hence, SD-36 is highly efficacious <i>in vivo</i> and achieves complete and long-lasting tumor regression in the Molm-16 cancer xenograft model at well-tolerated dose-schedules.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32330" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32330" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> outlines the synthesis of the protected phosphotyrosine mimetic, 5-((di-<i>tert</i>-butoxyphosphoryl)methyl)-1<i>H</i>-indole-2-carboxylic acid (<b>41</b>). Boc protection of <b>36</b> followed by benzylic bromination provided <b>38</b>. Treatment of <b>38</b> with sodium di-<i>tert</i>-butyl phosphite yielded a mixture of <b>39</b> and <b>40</b> which were hydrolyzed to afford compound <b>41</b>.</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 5-((Di-<i>tert</i>-butoxyphosphoryl)methyl)-1<i>H</i>-indole-2-carboxylic Acid (<b>41</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, Boc<sub>2</sub>O, THF, 0 °C to rt; (b) NBS, Bz<sub>2</sub>O<sub>2</sub>, CCl<sub>4</sub>, reflux, 77% over 2 steps; (c) (<i>t</i>-BuO)<sub>2</sub>PONa 1.0 equiv of NaH 2.5 equiv, THF, then reflux for 3 h; (d) LiOH, MeOH–THF–H<sub>2</sub>O, 60 °C.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> shows the construction of the 5-((diethoxyphosphoryl)difluoromethyl)-1<i>H</i>-indole-2-carboxylic acid (<b>46</b>), an α,α-difluoro derivative of <b>41</b> as previously reported.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Compound <b>38</b> was heated in P(OEt)<sub>3</sub> at 100 °C for 12 h furnishing the phosphate (<b>42</b>), which was subject to transesterification, affording the benzyl ester (<b>43</b>). The indole nitrogen of <b>40</b> was protected with a Cbz group, giving <b>44</b>. Difluorination of the double-activated methylene of <b>44</b> provided <b>45</b>, and finally, hydrogenation of <b>45</b> removed two protecting groups, furnishing the carboxylic acid (<b>46</b>).</div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 3-((Diethoxyphosphoryl)difluoromethyl)-1<i>H</i>-indole-2-carboxylic Acid (<b>46</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) P(OEt)<sub>3</sub>, 100 °C, 84%; (b) Ti(O-<i>i</i>-Pr)<sub>4</sub>, BnOH, 100 °C, 83%; (c) NaH, Cbz-Cl, THF, 0 °C to rt, 88%; (d) NFBS, NaHMDS, THF, −78 °C to rt, 95%; (e) H<sub>2</sub>/Pd–C, THF, 94%.</p></p></figure><div class="NLM_p">The syntheses of compounds <b>2</b>–<b>4</b> are shown in <a class="ref internalNav" href="#sch2a" aria-label="Scheme 3">Scheme 3</a> as previously reported.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Amide condensation of the acid (<b>47</b>) with the amine (<b>48</b>) gave compound <b>49</b> removal of whose Boc group yielded compound <b>50</b>. Condensation of <b>50</b> with corresponding protected phosphotyrosine mimetics followed by hydrolysis of the dialkyl phosphate group afforded compounds <b>2</b>–<b>4</b>.</div><figure id="sch2a" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>2</b>–<b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch2a"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EDC–HCl 2.0 equiv, HOBt 2.0 equiv, EtN(<i>i</i>-Pr)<sub>2</sub>, DCM, rt, 3 h, 89% yield; (b) CF<sub>3</sub>CO<sub>2</sub>H 2 mL, Et<sub>3</sub>Si–H 0.1 mL, DCM 3 mL, rt, 1 h; (c) (1) protected phosphotyrosine mimetics, EDC–HCl, HOBt, EtN(<i>i</i>-Pr)<sub>2</sub>, DCM, rt, 3 h; (2) TFA–DCM, rt, 3 h; or TMSI, 0 °C, 1 h.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch3" aria-label="Scheme 4">Scheme 4</a> shows the syntheses of compounds <b>5</b> and <b>7</b> as previously reported.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> By use of the acid (<b>51</b>)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>as starting material, amide condensation reaction and subsequent selective removal of Boc group yielded an amine (<b>53</b>) which, coupled with <b>46</b>, gave <b>54</b>. The common intermediate (<b>56</b>) was prepared by deprotection of Cbz group in <b>51</b> and introduction of a methyl group followed by hydrolysis of the <i>tert</i>-butyl ester. The corresponding amines were coupled to <b>56</b> and the resulting diethyl phosphates were treated with TMSI to yield compounds <b>5</b> and <b>7</b>.</div><figure id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0011.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>5</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (2) TFA, DCM; (b) <b>46</b>, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (c) H<sub>2</sub>, Pd/C, MeOH; (d) (1) HCHO, sodium triacetoxyborohydride, DCE; (2) TFA, DCM; (e) R-NH<sub>2</sub>, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h.</p></p></figure><div class="NLM_p">The syntheses of compounds <b>8</b>–<b>11</b> are shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 5">Scheme 5</a>. The intermediate (<b>60</b>) was obtained by an amide condensation reaction followed by removal of the Boc protecting group under acidic conditions. Condensation of <b>60</b> with <b>51</b> led to <b>61</b>, which following removal of the Cbz and subsequent acetylation and Boc deprotection gave the common intermediate (<b>62</b>). Amide condensation of <b>62</b> with corresponding protected phosphotyrosine mimetics and hydrolysis of the diethyl phosphate group afforded compounds <b>8</b>–<b>11</b>.</div><figure id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0012.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>8</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (b) TFA, DCM; (c) <b>51</b>, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (d) (1) H<sub>2</sub>, Pd/C, MeOH; (2) Ac<sub>2</sub>O, EtN(<i>i</i>-Pr)<sub>2</sub>, DCM; (3) TFA, DCM; (e) (1) phosphotyrosine mimetics, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (2) TFA–DCM, rt, 3 h; or TMSI, 0 °C, 1 h.</p></p></figure><div class="NLM_p">The preparation of the representative STAT3 degrader SD-36 is outlined in <a class="ref internalNav" href="#sch5" aria-label="Scheme 6">Scheme 6</a> as previously reported.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Compound <b>63</b> was obtained by removal of the Boc group of intermediate <b>61</b> followed by amidation with <b>46</b>. Hydrogenation of <b>63</b> gave the amine (<b>64</b>) which was coupled with <b>68</b> to afford <b>65</b>, which was treated with TMSI to yield SD-36. The synthesis of intermediate <b>68</b> was achieved by Sonogashira coupling of commercially available starting material <b>66</b> and oct-7-ynoic acid (<b>67</b>). By utilization of compound <b>64</b> as a common intermediate, the final compounds (<b>13</b>–<b>29</b>) were prepared by a method similar to that used for SD-36.</div><figure id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0013.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of SD-36<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) TFA, DCM; (2) <b>46</b>, HATU, DIEA, DMF; (b) H<sub>2</sub>, Pd/C, MeOH; (c) <b>68</b>, HATU, DIEA, DMF; (d) TMSI, BSTFA, DCM; (e) Pd(PPh<sub>3</sub>)Cl<sub>2</sub>, CuI, DMF/TEA, 80 °C, 1 h.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>30</b> and <b>31</b> is shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 7">Scheme 7</a>. By use of compound <b>55</b> as starting material, the acid (<b>69</b>) was obtained by amide condensation followed deprotection of the <i>tert</i>-butyl ester. By employment of compound <b>66</b> as a common intermediate, compounds <b>30</b> and <b>31</b>were synthesized by condensation with corresponding amine and subsequent hydrolysis of the diethyl phosphate group.</div><figure id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0014.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>30</b> and <b>31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) <b>68</b>, HATU, DIEA, DMF; (2) TFA, DCM; (b) (1) R-NH<sub>2</sub>, HATU, DIEA, DMF; (2) TMSI, BSTFA, DCM.</p></p></figure><div class="NLM_p">Compounds <b>32</b>–<b>34</b> were prepared as shown in <a class="ref internalNav" href="#sch7" aria-label="Scheme 8">Scheme 8</a>. Intermediate <b>70</b> was prepared by deprotection of the Cbz group in <b>61</b> and subsequent amidation with <b>68</b> followed by removal of the Boc group. The corresponding protected phosphotyrosine mimetics were coupled to <b>70</b> followed by TMSI-promoted hydrolysis of diethyl phosphate group to afford compounds <b>32</b>–<b>34</b>.</div><figure id="sch7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0015.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>32</b>–<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) H<sub>2</sub>, Pd/C, MeOH; (2) <b>68</b>, HATU, DIEA, DMF; (3) TFA, DCM; (b) (1) phosphotyrosine mimetics, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (2) TFA–DCM, rt, 3 h; or TMSI, 0 °C, 1 h.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch8" aria-label="Scheme 9">Scheme 9</a> outlines the synthesis of fluorescent labeled compound <b>74</b> as previously reported.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Intermediate <b>73</b> was prepared by condensation of <b>71</b> and <b>72</b> with HATU as the coupling reagent, followed by subsequent removal of the Boc protecting group. Compound <b>74</b> was furnished by amide condensation reaction of compound <b>73</b> and <b>56</b> followed by TMSI-promoted hydrolysis of the diethyl phosphate group.</div><figure id="sch8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0016.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Fluorescent Labeled Compound <b>74</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HATU, DIEA, DMF; (2) TFA, DCM; (b) (1) HATU, DIEA, DMF; (2) TMSI, BSTFA, DCM.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76560" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76560" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this paper, we report our structure-based discovery of potent and selective PROTAC STAT3 degraders with SD-36 being the best compound. We first designed and obtained SI-109 as a potent inhibitor of the STAT3 SH2 domain, starting from our previously reported peptidomimetic STAT3 inhibitor. A key element in the successful discovery of SI-109 was the design of 5-(difluoro(phosphoro)methyl)-1<i>H</i>-indole-2-carboxylic acid as a novel phosphotyrosine mimetic. Determination of the cocrystal structure of SI-109 in a complex with STAT3 protein provided a structural basis for the design of STAT3 degraders.</div><div class="NLM_p">Employing SI-109 and ligands for cereblon/cullin 4A, we designed, synthesized, and tested a series of PROTAC STAT3 degraders. Our efforts have yielded a series of potent STAT3 degraders, with SD-36 being the best. SD-36 induces degradation of STAT3 protein in cells and demonstrates >100-fold selectivity over other STAT proteins.</div><div class="NLM_p last">SD-36 potently inhibits cell growth in the Molm-16 leukemia and SU-DHL-1 lymphoma cell lines. A single dose of SD-36 is capable of achieving complete degradation of STAT3 in native mouse spleen tissue and in the Molm-16 xenograft tumor tissue with the degradation effect persisting for >2–3 days. SD-36 is capable of achieving complete and persistent tumor regression in the Molm-16 xenograft model in mice at well tolerated dose-schedules. To our knowledge, SD-36 is a first, potent, selective, and highly efficacious degrader of STAT3 protein reported. Because SD-36 effectively and selectively degrades STAT3 in cells and in tissues, it can be employed as a powerful tool to study the role(s) of STAT3 in different biological processes. Furthermore, SD-36 warrants extensive investigation as a new class of anticancer therapy for the treatment of human cancers and other human diseases. Our present study demonstrates that the PROTAC technology can be successfully employed to target those traditionally difficult-to-drug or undruggable protein targets.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88565" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88565" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> General Information</h3><div class="NLM_p last">All commercial reagents and solvents were used as supplied without further purification with the following exceptions: THF was freshly distilled from sodium wire. The reactions were performed under an N<sub>2</sub> atmosphere in anhydrous solvents. The final products were purified by reverse phase HPLC (RP-HPLC) with solvent A (0.1% of TFA in water) and solvent B (0.1% of TFA in CH<sub>3</sub>CN) as eluents with a flow rate of 60 mL/min. The purity of compounds was determined by Waters ACQUITY UPLC, and all the final compounds were >95% pure. Proton nuclear magnetic resonance (<sup>1</sup>H NMR), carbon nuclear magnetic resonance (<sup>13</sup>C NMR), and phosphorus nuclear magnetic resonance (<sup>31</sup>P NMR) spectroscopies were performed on Bruker Advance 300/400 NMR spectrometers, and chemical shifts are reported in parts per million (ppm) relative to an internal standard. MS analysis was carried out with a Thermo-Scientific LCQ Fleet mass spectrometer or a Waters ultraperformance liquid chromatography (UPLC)–mass spectrometer.</div></div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-((3<i>S</i>,6<i>S</i>)-3-((6S)-5-Amino-1-(benzylamino)-1,5-dioxopentan-2-ylcarbamoyl)-5-oxodecahydropyrrolo[1,2-<i>a</i>]azocin-6-ylcarbamoyl)-1<i>H</i>-indol-5-yl Dihydrogen Phosphate (<b>2</b>)</h3><div class="NLM_p">The acid (<b>47</b>, 66 mg, 0.2 mmol), HOBt (56 mg, 0.4 mmol), and EDC-HCl (80 mg, 0.4 mmol) were placed in round-bottomed flask. EtN(<i>i</i>-Pr)<sub>2</sub> (0.2 mL, 1.0 mmol) and DCM were added via syringe, and the reaction mixture was stirred at rt for 5 min. Compound <b>48</b> was added as a solution in DCM. The reaction was stirred at rt for 3 h before being quenched with saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM, and the combined organic layers were washed with brine and dried over anhydrous sodium sulfate. The solvent was removed <i>in vacuo</i> and the residue was purified by flash column chromatography to yield <b>49</b> in 89% yield (0.14 g). <sup>1</sup>H NMR (300 MHz, MeOD-<i>d</i><sub>4</sub>): 8.52 (s, 1H), 8.40–8.20 (m, 2H), 7.50–7.00 (m, 20 H), 4.70–4.45 (m, 1H), 4.45–4.20 (m, 5H), 2.70–2.40 (m, 2H), 2.30–1.00 (m, 14H), 1.44 (s, 9H). <sup>13</sup>C NMR (75 MHz, MeOD-<i>d</i><sub>4</sub>): 174.60, 174.53, 173.96, 173.56, 157.57, 146.21, 139.92, 130.25, 129.69, 128.91, 128.68, 128.38, 127.93, 80.80, 71.74, 63.17, 61.63, 54.28, 52.14, 44.21, 37.46, 35.80, 33.60, 29.14, 28.95, 26.95, 23.53. ESI-MS calculated for C<sub>47</sub>H<sub>56</sub>N<sub>5</sub>O<sub>6</sub> [M + H]<sup>+</sup> = 786.42. Obtained: 786.92</div><div class="NLM_p last">The amine <b>49</b> (78 mg, 0.1 mmol) was dissolved in DCM (5 mL). Et<sub>3</sub>Si-H (0.1 mL) and CF<sub>3</sub>CO<sub>2</sub>H (2 mL) were added via syringe. The reaction was stirred at rt for 1 h. The volatile components were removed on a rotary evaporator. The residue compound (<b>50</b>) was used in the next step without further purification. Compound <b>50</b> (crude, 0.1 mmol) and 5-((di-<i>tert</i>-butoxyphosphoryl)oxy)-1<i>H</i>-indole-2-carboxylic acid<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (55 mg, 0.15 mmol) were coupled using peptide coupling conditions (3.0 equiv of HOBt, 3.0 equiv of EDC-HCl, 20 equiv of EtN(<i>i</i>-Pr)<sub>2</sub>, and DCM as solvent). After being purified by flash column chromatography, the obtained product was dissolved in DCM (5 mL) in a round bottomed flask equipped with a magnetic stirring bar. CF<sub>3</sub>CO<sub>2</sub>H (2 mL) and Et<sub>3</sub>Si-H (0.1 mL) were added to the reaction mixture, which was then stirred for 3 h. Solvent was removed <i>in vacuo</i> and the product was purified by reverse phase semipreparative HPLC to give compound <b>2</b> (25 mg, 36% yield). <sup>1</sup>H NMR (300 MHz, MeOD-<i>d</i><sub>4</sub>): 7.47 (s, 1H), 7.42 (d, <i>J</i> = 8.92 Hz, 1H), 7.38–7.20 (m, 5H), 7.17 (s, 1H), 7.13 (dd, <i>J</i> = 8.92, 1.67 Hz, 1H), 5.14–5.04 (m, 1H), 4.50–4.32 (m, 5H), 2.50–1.50 (m, 16H). <sup>31</sup>P NMR (121 MHz, MeOD-<i>d</i><sub>4</sub>): 4.4 (s). ESI-MS calculated for C<sub>32</sub>H<sub>40</sub>N<sub>6</sub>O<sub>9</sub>P [M + H]<sup>+</sup> = 683.26. Found: 683.20. Reverse phase analytical HPLC: >99% purity.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> (2-((3<i>S</i>,6<i>S</i>)-3-((<i>S</i>)-5-Amino-1-(benzylamino)-1,5-dioxopentan-2-ylcarbamoyl)-5-oxodecahydropyrrolo[1,2-<i>a</i>]azocin-6-ylcarbamoyl)-1<i>H</i>-indol-5-yl)methylphosphonic Acid (<b>3</b>)</h3><div class="NLM_p">NaH (2.2 g, 53 mmol, 2.0 equiv, 60% in mineral oil) and THF (300 mL) were mixed in a round bottomed flask equipped with a magnetic stirring bar. The suspension was cooled with an ice/water bath before addition of ethyl 5-methyl-1<i>H</i>-indole-2-carboxylate (<b>36</b>) (5.0 g, 26 mmol, 1.0 equiv) over 15 min. The solution was stirred at this temperature for 30 min, and the color of the solution turned red. Boc<sub>2</sub>O (8.1 g, 37 mmol, 1.4 equiv) was added to the solution in one portion. The reaction mixture was stirred at rt for another 24 h before quenching with ice–water. The aqueous layer was extracted with EtOAc (200 mL × 2), and the combined organic layers were washed with brine (50 mL × 2), dried over anhydrous sodium sulfate, and concentrated on a rotary evaporator. The residual crude product (<b>37</b>) was used directly for the next step without further purification. The crude <b>37</b>, (PhCO)<sub>2</sub>O<sub>2</sub> (242 mg, 1.0 mmol, 0.04 equiv), NBS (4,62 g, 26.0 mmol, 1.0 equiv), and anhydrous CCl<sub>4</sub> (150 mL) were added to a round bottomed flask equipped with a magnetic stirring bar. The reaction mixture was heated at reflux for 12 h. The precipitate was filtered off, and the solvent was removed on a rotary evaporator. The residual crude product was purified by flash column chromatography to afford the desired benzylic bromide (<b>38</b>) as colorless oil (7.6 g, 77% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 8.05 (d, <i>J</i> = 8.66 Hz, 1H), 7.61 9d, <i>J</i> = 1.39 Hz, 1H), 7.44 (dd, <i>J</i> = 8.66, 1.81 Hz, 1H), 7.06 (d, <i>J</i> = 0.65 Hz, 1H), 5.29 (s, 2H), 4.38 (q, <i>J</i> = 7.14 Hz, 2H), 1.62 (s, 9H), 1.40 (t, <i>J</i> = 7.14 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 161.8, 149.2, 146.9, 137.6, 133.0, 131.8, 128.0, 127.9, 122.7, 115.5, 114.5, 85.3, 85.0, 61.7, 34.1, 28.0, 14.4. ESI-MS calculated for C<sub>17</sub>H<sub>21</sub><sup>79</sup>BrNO<sub>4</sub> [M + H]<sup>+</sup> = 382.07. Found: 382.42. C<sub>17</sub>H<sub>21</sub><sup>81</sup>BrNO<sub>4</sub> [M + H]<sup>+</sup>: 384.06. Found: 384.08</div><div class="NLM_p">NaH 0.22 g (60% in mineral oil, 5.5 mmol, 2.5 equiv) and anhydrous THF (10 mL) were added to a round-bottom flask equipped with a magnetic stirring bar. (<i>t</i>-BuO)<sub>2</sub>POH (427 mg, 2.2 mmol, 1.0 equiv) was added dropwise as a THF solution, and the reaction mixture was stirred for 0.5 h at rt. The bromide (<b>38</b>) was then added as a THF solution, and the mixture was heated at 80 °C for 3 h before quenching with NH<sub>4</sub>Cl saturated solution at 0 °C. The aqueous layer was extracted with EtOAc (3 × 30 mL). The organic layers were combined and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed <i>in vacuo</i> and the product was purified by flash column chromatography to give a mixture of <b>39</b> and <b>40</b> (0.68 g, 1.4 mmol, 63% yield, <b>39</b>:<b>40</b> = 75:25, inseparable by flash column chromatography). The data of the major isomer (<b>39</b>) are shown below. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 7.99 (d, <i>J</i> = 8.60 Hz, 1H), 7.51–7.49 (m, 1H), 7.36–7.30 (m, 1H), 7.05 (s, 1H), 4.38 (q, <i>J</i> = 7.16 Hz, 2H), 3.12 (d, <i>J</i><sub>P–H</sub> = 21.08 Hz, 2H), 1.63 (s, 9H), 1.41 (s, 18H), 1.41–1.39 (m, 3H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 161.8, 149.2, 136.6 (d, <i>J</i><sub>P–C</sub> = 2.9 Hz), 130.9, 128.9 (d, <i>J</i><sub>P–H</sub> = 5.9 Hz), 128.51 (d, <i>J</i><sub>P–H</sub> = 9.6 Hz), 127.6 (d, <i>J</i><sub>P–C</sub> = 2.8 Hz), 123.0 (d, <i>J</i><sub>P–C</sub> = 7.2 Hz), 114.5, 114.4 (d, <i>J</i><sub>P–C</sub> = 2.4 Hz), 84.5, 82.0 (d, <i>J</i><sub>P–C</sub> = 9.0 Hz), 61.3, 37.6 (d, <i>J</i><sub>P–C</sub> = 142.9 Hz), 30.33 (d, <i>J</i><sub>P–C</sub> = 3.9 Hz), 27.8, 14.2. <sup>31</sup>P NMR (121 MHz, CDCl<sub>3</sub>): 18.10. ESI-MS calculated for C<sub>25</sub>H<sub>38</sub>NO<sub>7</sub>P [M + Na]<sup>+</sup> = 518.23. Found: 518.24</div><div class="NLM_p">The acid <b>41</b> was synthesized from <b>39</b> and <b>40</b> (90 mg, 2.0 equiv) using LiOH–H<sub>2</sub>O (16 mg, 4.0 equiv) as base and THF/H<sub>2</sub>O/MeOH (3 mL:3 mL:3 mL) as solvents. The reaction was heated at 60 °C. Upon completion of hydrolysis determined by TLC (∼2 h), the reaction mixture was acidified to pH = 2 with 0.1 N HCl. The aqueous layer was extracted with EtOAc (3 × 30 mL), and the organic layers were combined and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed <i>in vacuo</i>, and the crude product was coupled with <b>41</b> without further purification.</div><div class="NLM_p last">The freshly prepared amine <b>50</b> (0.05 mmol) and <b>41</b> (0.2 mmol) were coupled using peptide coupling conditions (6.0 equiv HOBt, 6.0 equiv of EDC-HCl, 40 equiv of EtN(<i>i</i>-Pr)<sub>2</sub>, and DCM as solvent). The reaction was stirred at rt for 3 h before being quenched with saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM, and the combined organic layers were washed with brine and dried over anhydrous sodium sulfate. The solvent was removed <i>in vacuo</i>, and the residue was purified by flash column chromatography. The obtained product was dissolved in DCM (5 mL), and CF<sub>3</sub>CO<sub>2</sub>H (3 mL) and Et<sub>3</sub>Si-H (0.1 mL) were added via syringe. The reaction mixture was stirred at rt for 3 h. The volatile components were removed on a rotary evaporator and the desired product was purified by reverse phase semipreparative HPLC to give compound <b>3</b> (20 mg, 58% yield). <sup>1</sup>H NMR (300 MHz, MeOD-<i>d</i><sub>4</sub>): 7.57 (s, 1H), 7.39 (d, <i>J</i> = 8.52 Hz, 1H), 7.36–7.24 (m, 5H), 7.23 (d, <i>J</i> = 8.52 Hz, 1H), 7.15 (s, 1H), 5.14–5.02 (m, 1H), 4.52–4.06 (m, 5H), 3.20 (d, <i>J</i><sub>P–H</sub> = 21.05 Hz, 2H), 2.50–1.50 (m, 16H). <sup>31</sup>P NMR (121 MHz, MeOD-<i>d</i><sub>4</sub>): 25.3 (s). ESI-MS calculated for C<sub>33</sub>H<sub>42</sub>N<sub>6</sub>O<sub>8</sub>P [M + H]<sup>+</sup> = 681.28. Found: 681.28. Reverse phase analytical HPLC: 98% purity.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (2-((3<i>S</i>,6<i>S</i>)-3-((<i>S</i>)-5-Amino-1-(benzylamino)-1,5-dioxopentan-2-ylcarbamoyl)-5-oxodecahydropyrrolo[1,2-<i>a</i>]azocin-6-ylcarbamoyl)-1<i>H</i>-indol-5-yl)difluoromethylphosphonic Acid (<b>4</b>)</h3><div class="NLM_p">Compound <b>38</b> (3 g, 7.9 mmol, 1.0 equiv) and (EtO)<sub>3</sub>P (1.72 mL, 10.0 mmol, 1.2 equiv) were placed in a round bottomed flask equipped with a magnetic stirring bar. The reaction mixture was heated at 100 °C for 12 h and then was loaded directly onto a silica gel column and purified by flash column chromatography to afford the desired phosphate (<b>42</b>) as a colorless oil (2.9 g, 84%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 8.02 (d, <i>J</i> = 8.62 Hz, 1H), 7.53 (s, 1H), 7.35 (d, <i>J</i> = 8.63 Hz, 1H), 7.05 (s, 1H), 4.38 (q, <i>J</i> = 7.13 Hz, 2H), 4.07–3.92 (m, 4H), 3.23 (d, <i>J</i><sub>P–H</sub> = 21.24 Hz, 2H), 1.63 (s, 9H), 1.39 (t, <i>J</i> = 7.13 Hz, 3H), 1.23 (t, <i>J</i> = 7.06 Hz, 6H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 161.7, 149.1, 136.7 (d, <i>J</i><sub>P–C</sub> = 2.88 Hz), 131.1, 128.5 (d, <i>J</i><sub>P–C</sub> = 5.88 Hz), 127.7 (d, <i>J</i><sub>P–C</sub> = 2.81), 126.5 (d, <i>J</i><sub>P–C</sub> = 9.12 Hz), 122.9 (d, <i>J</i><sub>P–C</sub> = 7.15 Hz), 114.8 (d, <i>J</i><sub>P–C</sub> = 2.50 Hz), 114.3, 84.5, 62.0 (d, <i>J</i><sub>P–C</sub> = 6.79 Hz), 61.3, 33.3 (d, <i>J</i><sub>P–C</sub> = 128.4), 27.7, 16.3 (d, <i>J</i><sub>P–C</sub> = 5.96 Hz), 14.1. <sup>31</sup>P NMR (121 M Hz, CDCl<sub>3</sub>): 26.3 (s). ESI-MS calculated for C<sub>21</sub>H<sub>31</sub>NO<sub>7</sub>P [M + H]<sup>+</sup> = 440.18. Found: 440.67</div><div class="NLM_p">Compound <b>42</b> (2.9 g, 6.6 mmol, 1.0 equiv), BnOH (14 mL, 132 mmol, 20 equiv), and Ti(O<i>i</i>-Pr)<sub>4</sub> (0.32 mL, 1.6 mmol, 0.25 equiv) were placed in a round bottomed flask equipped with a magnetic stirring bar. The reaction mixture was heated at 100 °C for 12 h. The reaction mixture was cooled down to 35 °C and quenched with 1 N HCl (20 mL). The aqueous layer was extracted with EtOAc (200 mL × 2), and the combined organic extracts were washed with brine (50 mL × 2), dried over anhydrous sodium sulfate, filtered, and concentrated <i>in vacuo</i>. The residual crude product was purified by flash column chromatography to afford the desired benzyl carboxylate (<b>43</b>) as a colorless oil (2.25 g, 83% yield). 80% purity (determined by <sup>31</sup>P NMR): 10% ethyl carboxylate, 10% unknown. <sup>1</sup>H NMR (300 MHz, MeOD-<i>d</i><sub>4</sub>): 7.65 (s, 1H), 7.60–7.38 (m, 6H), 7.31 (dt, <i>J</i> = 8.57, 1.72 Hz, 1H), 7.24 (s, 1H), 5.43 (s, 2H), 4.15–4.00 (m, 4H), 3.35 (d, <i>J</i><sub>P–H</sub> = 21.03 Hz, 2H), 1.30 (t, <i>J</i> = 7.06 Hz, 6H). <sup>13</sup>C NMR (75 MHz, MeOD-<i>d</i><sub>4</sub>): 163.0, 138.2 (d, <i>J</i><sub>P–C</sub> = 2.19 Hz), 137.6, 129.6, 129.3, 129.2, 129.1, 128.8 (d, <i>J</i><sub>P–C</sub> = 2.76 Hz), 128.2 (d, <i>J</i><sub>P–C</sub> = 5.33 Hz), 124.3 (d, <i>J</i><sub>P–C</sub> = 7.95 Hz), 124.1 (d, <i>J</i><sub>P–C</sub> = 9.42 Hz), 113.4 (d, <i>J</i><sub>P–C</sub> = 2.37 Hz), 109.3, 67.4, 63.6 (d, <i>J</i><sub>P–C</sub> = 6.96 Hz), 33.6 (d, <i>J</i><sub>P–C</sub> = 138.3 Hz), 16.7 (d, <i>J</i><sub>P–C</sub> = 5.92 Hz). <sup>31</sup>P NMR (121 M Hz, MeOD-<i>d</i><sub>4</sub>): 28.3 (s), 26.4 (s). ESI-MS calculated for C<sub>21</sub>H<sub>25</sub>NO<sub>5</sub>P [M + H]<sup>+</sup> = 402.15. Found: 402.50</div><div class="NLM_p">NaH (0.6 g, 15 mmol, 3.0 equiv, 60% in mineral oil) and THF (100 mL) were placed in a round bottomed flask equipped with a magnetic stirring bar. The suspension was cooled with ice/water bath before addition of compound <b>43</b> (2.25 g in THF, 5.5 mmol, 1.0 equiv) over 5 min. The solution was stirred at this temperature for 10 min before addition of Cbz-Cl (1.12 mL, 8 mmol, 1.5 equiv) via a syringe. The reaction mixture was stirred at rt for another 12 h before quenching with ice–water. The aqueous layer was extracted with EtOAc (200 mL × 2), and the combined organic extracts were washed with brine (50 mL × 2), dried over anhydrous sodium sulfate, and concentrated <i>in vacuo</i>. The residual crude product was purified by flash column chromatography to afford the desired <b>41</b> as a colorless oil (2.6 g, 88% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 8.00 (d, <i>J</i> = 8.63 Hz, 1H), 7.52 (s, 1H), 7.46–7.26 (m, 11H), 7.11 (s, 1H), 5.33 (s, 2H), 5.20 (s, 2H), 4.10–3.90 (m, 4H), 3.22 (d, <i>J</i><sub>P–H</sub> = 21.30 Hz, 2H), 1.21 (t, <i>J</i> = 7.05 Hz, 6H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 161.3, 150.5, 136.6 (d, <i>J</i><sub>P–C</sub> = 2.97 Hz), 135.3, 134.4, 130.6, 129.0 (d, <i>J</i><sub>P–C</sub> = 5.88 Hz) 128.7, 128.6, 128.6, 128.5, 128.3, 128.2, 127.8 (d, <i>J</i><sub>P–C</sub> = 2.82 Hz), 127.0 (d, <i>J</i><sub>P–C</sub> = 9.10 Hz), 123.1 (d, <i>J</i><sub>P–C</sub> = 7.08 Hz), 115.6, 115.0 (d, <i>J</i><sub>P–C</sub> = 2.25 Hz), 69.5, 67.1, 62.1 (d, <i>J</i><sub>P–C</sub> = 6.78 Hz), 33.4 (d, <i>J</i><sub>P–C</sub> = 138.49 Hz), 16.3 (d, <i>J</i><sub>P–C</sub> = 5.87 Hz). <sup>31</sup>P NMR (121 M Hz, CDCl<sub>3</sub>): 26.3 (s). ESI-MS calculated for C<sub>29</sub>H<sub>30</sub>NO<sub>7</sub>P [M + Na]<sup>+</sup> = 558.17. Found: 558.08</div><div class="NLM_p">Compound <b>44</b> (9.17 g, 17 mmol, 1.0 equiv), (PhSO<sub>2</sub>)<sub>2</sub>NF (NFSB, 16 g, 51 mmol, 3.0 equiv), and THF (300 mL) were added to a round bottomed flask equipped with a magnetic stirring bar. The reaction mixture was cooled to −78 °C with the aid of an ethanol/dry ice bath. To this solution, NaHMDS (51 mL, 1.0 M in THF, 3.0 equiv) was added over 10 min. The reaction mixture was stirred at this temperature for 2 h before warming up to rt over 3–4 h. The reaction was quenched with saturated NH<sub>4</sub>Cl aqueous solution (100 mL). The aqueous layer was extracted with EtOAc (200 mL × 2), and the combined organic extracts were washed with brine (50 mL × 2), dried over anhydrous sodium sulfate, and concentrated <i>in vacuo</i>. The residual crude product was purified by flash column chromatography to afford the desired product (<b>45</b>) as a colorless oil (9.6 g, 95% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 8.13 (d, <i>J</i> = 8.70 Hz, 1H), 7.88 (s, 1H), 7.65 (d, <i>J</i> = 8.90 Hz, 1H), 7.50–7.28 (m, 10H), 7.17 (s, 1H), 5.33 (s, 2H), 5.20 (s, 2H), 4.30–4.00 (m, 4H), 1.27 (t, <i>J</i> = 6.85 Hz, 6H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 161.2, 150.3, 138.6, 135.2, 134.2, 131.5, 129.0, 128.8, 128.7, 128.6, 128.5, 128.4, 128.4–127.6 (m), 127.4, 125.2–124.4 (m), 121.0–120.6 (m), 120.5–119.5 (m), 115.5, 115.2, 70.0, 67.3, 64.9 (d, <i>J</i><sub>P–C</sub> = 6.76 Hz), 16.3 (d, <i>J</i><sub>P–C</sub> = 5.49 Hz). <sup>31</sup>P NMR (121 M Hz, CDCl<sub>3</sub>): 6.3 (t, <i>J</i><sub>P–F</sub> = 117 Hz). ESI-MS calculated for C<sub>29</sub>H<sub>29</sub>F<sub>2</sub>NO<sub>7</sub>P [M + H]<sup>+</sup> = 572.17. Found: 572.25</div><div class="NLM_p">Compound <b>45</b> (1 g, 1.7 mmol, 1.0 equiv) and THF (300 mL) were placed in a round bottomed flask equipped with a magnetic stirring bar. The oxygen was removed with the aid of a vacuum line and a nitrogen balloon, and then 10% Pd/C (0.1 g, 0.1 mmol, 0.05 equiv) was added to the reaction mixture. The reaction was stirred at rt for 12 h under an H<sub>2</sub> atmosphere (1 atm H<sub>2</sub> balloon). The Pd/C was removed by filtration, and the solvent was removed <i>in vacuo</i>. The residual crude product was purified by flash column chromatography to afford the desired compound (<b>46</b>) as a pale green solid (0.56 g, 94% yield). Higher purity can be achieved by recrystallization from CHCl<sub>3</sub>. <sup>1</sup>H NMR (300 MHz, MeOD-<i>d</i><sub>4</sub>): 11.6 (s, 1H), 7.94 (s, 1H), 7.58 (d, <i>J</i> = 8.75 Hz, 1H), 7.48 (d, <i>J</i> = 8.75 Hz, 1H), 7.27 (s, 1H), 4.30–4.05 (m, 4H), 1.30 (td, <i>J</i> = 7.04 Hz, <i>J</i><sub>P–H</sub> = 0.49 Hz, 6H). <sup>13</sup>C NMR (75 MHz, MeOD-<i>d</i><sub>4</sub>): 164.5, 139.7, 131.2, 128.1, 126.0–124.0 (m), 123.4–123.0 (m), 122.4–122.0 (m), 119.0–118.1 (m), 113.5, 109.6, 66.3 (d, <i>J</i><sub>P–C</sub> = 7.09 Hz), 16.6 (d, <i>J</i><sub>P–C</sub> = 5.34 Hz). <sup>31</sup>P NMR (121 M Hz, MeOD-<i>d</i><sub>4</sub>): 6.6 (t, <i>J</i><sub>P–F</sub> = 123 Hz). ESI-MS calculated for C<sub>14</sub>H<sub>17</sub>F<sub>2</sub>NO<sub>5</sub>P [M + H]<sup>+</sup> = 348.08. Found: 348.42.</div><div class="NLM_p last">The freshly prepared amine (<b>50</b>) (0.2 mmol) and the acid (<b>46</b>) (0.2 mmol) were coupled using peptide coupling conditions (3.0 equiv of HOBt. 3.0 equiv of EDC–HCl, and 4.0 equiv of EtN(<i>i</i>-Pr)<sub>2</sub>, DCM as solvent). The reaction was stirred at rt for 3 h before being quenched with saturated NaHCO<sub>3</sub>. The aqueous layer was extracted with DCM, and the combined organic layers were washed with brine and dried over anhydrous sodium sulfate. The solvent was removed <i>in vacuo</i>, and the residue was purified by flash column chromatography. The obtained product was dissolved in DCM (5 mL) in a round bottomed flask equipped with a magnetic stirring bar, and TMSI (0.7 mmol, 0.1 mL) was added at 0 °C. The reaction mixture was stirred for 2 h at 0 °C. Volatile components were removed on a rotary evaporator, and a mixture of MeCN–H<sub>2</sub>O–AcOH (20 mL, 8:1:1) was added. The mixture was stirred at rt for 1 h. The volatile components were removed on a rotary evaporator and the desired product was purified by reverse phase semipreparative HPLC to give compound <b>4</b> (38 mg, 26% yield). <sup>1</sup>H NMR (300 MHz, MeOD-<i>d</i><sub>4</sub>): 7.89 (s, 1H), 7.50 (d, <i>J</i> = 8.74 Hz, 1H), 7.45 (d, <i>J</i> = 8.74 Hz, 1H), 7.34–7.16 (m, 6H), 5.10–5.00 (m, 1H), 4.48–4.30 (m, 5H), 2.50–1.50 (m, 16H). <sup>31</sup>P NMR (121 MHz, MeOD-<i>d</i><sub>4</sub>): 5.6 (t, <i>J</i><sub>P–F</sub> = 119 Hz). ESI-MS calculated for C<sub>33</sub>H<sub>40</sub>F<sub>2</sub>N<sub>6</sub>O<sub>8</sub>P [M + H]<sup>+</sup> = 717.26. Found: 717.33. Reverse phase analytical HPLC: >95%.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-methyl-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>5</b>)</h3><div class="NLM_p">HATU (418 mg, 1.1 mmol, 1.1 equiv) was added to a solution of <b>51</b> (462 mg, 1 mmol, 1 equiv), <i>tert</i>-butyl <span class="smallcaps smallerCapital">l</span>-glutaminate <b>52</b> (202 mg, 1 mmol, 1 equiv), and DIEA (0.52 mL, 3 mmol, 3 equiv) in DMF (10 mL), and the resulting mixture was stirred at rt for 1 h. The solution was diluted with EtOAc and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was dissolved in DCM (50 mL) and TFA (2.5 mL) was added. The resulting reaction mixture was stirred for 1 day, and the solvent was removed under vacuum. The residual was purified by HPLC to afford <b>53</b> (279 mg 51% over two steps). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.44–7.33 (m, 5H), 5.33–5.14 (m, 2H), 4.53 (t, <i>J</i> = 8.6 Hz, 1H), 4.45–4.29 (m, 2H), 4.18–4.11 (m, 2H), 3.83–3.63 (m, 1H), 3.55–3.36 (m, 2H), 2.38 (tt, <i>J</i> = 7.3, 6.0 Hz, 3H), 2.28–2.03 (m, 3H), 2.01–1.72 (m, 4H), 1.50–1.44 m, 9H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>27</sub>H<sub>40</sub>N<sub>5</sub>O<sub>7</sub><sup>+</sup> 546.29, found [M + H]<sup>+</sup> 546.47.</div><div class="NLM_p">HATU (153 g, 0.40 mmol, 1.1 equiv) was added to a solution of compounds <b>53</b> (200 mg, 0.37 mmol, 1 equiv), <b>46</b> (127 mg, 0.37 mmol, 1 equiv), and DIEA (0.19 mL, 1.1 mmoL, 3 equiv) in DMF (5 mL). The resulting mixture was stirred at rt for 1 h and diluted with EtOAc, washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine, and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by flash chromatography on silica gel to afford <b>54</b> (237 mg, 74%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.90 (s, 1H), 7.61–7.18 (m, 8H), 5.29–5.06 (m, 3H), 4.49 (t, <i>J</i> = 8.5 Hz, 1H), 4.39–4.30 (m, 2H), 4.26–4.08 (m, 4H), 4.01–3.44 (m, 4H), 2.52–1.62 (m, 10H), 1.50–1.43 (m, 9H), 1.29 (t, <i>J</i> = 7.1 Hz, 6H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>37</sub>H<sub>46</sub>F<sub>2</sub>N<sub>6</sub>O<sub>11</sub>P<sup>+</sup> [M – CH<sub>2</sub>═C(CH<sub>3</sub>)<sub>2</sub> + H]<sup>+</sup> 819.29, found [M – CH<sub>2</sub>═C(CH<sub>3</sub>)<sub>2</sub> + H]<sup>+</sup> 819.59.</div><div class="NLM_p">10% Pd–C (50 mg) was added to a solution of compound <b>54</b> (150 mg, 0.17 mmol) in MeOH (10 mL). The solution was stirred under 1 atm of H<sub>2</sub> at rt for 3 h before filtering through Celite and being concentrated. The resulting amine was purified by HPLC to afford compound <b>55</b> (103 mg, 81%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.91 (s, 1H), 7.56 (d, <i>J</i> = 8.7 Hz, 1H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 7.33 (d, <i>J</i> = 0.6 Hz, 1H), 5.64 (dd, <i>J</i> = 12.1, 5.6 Hz, 1H), 4.84–4.67 (m, 2H), 4.39 (dd, <i>J</i> = 9.2, 5.3 Hz, 1H), 4.29–4.04 (m, 4H), 3.79–3.50 (m, 3H), 3.43 (t, <i>J</i> = 12.4 Hz, 1H), 2.53–2.41 (m, 1H), 2.37 (t, <i>J</i> = 7.4 Hz, 2H), 2.28–1.80 (m, 7H), 1.48 (s, 9H), 1.36–1.23 (m, 6H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>33</sub>H<sub>48</sub>F<sub>2</sub>N<sub>6</sub>O<sub>9</sub>P<sup>+</sup> 741.32, found [M + H]<sup>+</sup> 741.44.</div><div class="NLM_p">Compound <b>55</b> (103 g, 0.14 mmol, 1.0 equiv), HCHO (37%, 0.06 mL, 0.70 mmol, 5.0 equiv), DCE (5 mL), and NaBH(OAc)<sub>3</sub> (59 mg, 0.28 mmol, 2.0 equiv) were added to a 25 mL round bottomed flask equipped with a magnetic stirring bar. The solution was stirred rt for 2 h until LC–MS detected the reaction was complete. Water (5 mL) was added to quench the reaction, and the reaction mixture was then extracted with DCM (10 mL × 3) and dried with anhydrous sodium sulfate. The mixture was filtered, and the solvent was removed <i>in vacuo</i>. The residual crude product was dissolved in DCM (5 mL), and TFA (2 mL) was added. The resulting mixture was stirred at rt until UPLC–MS detected it was finished. Most of the organic solvent was removed by evaporation, then the residue was purified by HPLC to afford <b>56</b> (60 mg, 64% yield over two steps). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.93 (s, 1H), 7.59 (d, <i>J</i> = 8.7 Hz, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.34 (d, <i>J</i> = 0.6 Hz, 1H), 5.63 (dd, <i>J</i> = 11.9, 5.3 Hz, 1H), 4.84 (t, <i>J</i> = 8.9 Hz, 1H), 4.75–4.71 (m, 1H), 4.56 (dd, <i>J</i> = 9.9, 4.3 Hz, 1H), 4.30–4.09 (m, 4H), 4.01 (t, <i>J</i> = 12.7 Hz, 1H), 3.80 (dd, <i>J</i> = 12.1, 5.4 Hz, 1H), 3.60 (d, <i>J</i> = 14.3 Hz, 1H), 3.43 (t, <i>J</i> = 12.0 Hz, 1H), 3.13 (s, 3H), 2.56–2.49 (m, 1H), 2.45–2.16 (m, 5H), 2.14–2.02 (m, 1H), 2.00–1.82 (m, 3H), 1.31 (t, <i>J</i> = 7.1 Hz, 6H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>30</sub>H<sub>42</sub>F<sub>2</sub>N<sub>6</sub>O<sub>9</sub>P<sup>+</sup> 699.27, found [M + H]<sup>+</sup> 699.05.</div><div class="NLM_p last">HATU (21 mg, 0.055 mmol, 1.1 equiv) was added to a solution of the acid <b>56</b> (35 mg, 0.05 mmol, 1 equiv), benzylamine (5.4 mg, 0.05 mmol, 1 equiv), and DIEA (46 μL, 0.15 mmoL, 3 equiv) in DMF (3 mL). The resulting mixture was stirred at rt for 10 min and purified by HPLC. The product obtained (32 mg, 0.04 mmol, 1 equiv) was dissolved in DCM (4 mL) and cooled to 0 °C before adding CF<sub>3</sub>CON(TMS)<sub>2</sub> (63 mg, 0.24 mmol, 6.0 equiv) and 1 M TMS-I in DCM (0.16 mL, 0.16 mmol, 4.0 equiv). The reaction mixture was stirred at 0 °C for 10 min, and the solvent was removed <i>in vacuo</i> at 0 °C. The residue was dissolved in a mixed solvent of CH<sub>3</sub>CN (1 mL), water (2 mL), and TFA (0.1 mL) and purified by HPLC to yield compound <b>5</b> (24 mg, 65% over two steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.82 (s, 1H), 7.45–7.39 (m, 2H), 7.31–7.23 (m, 2H), 7.19 (dd, <i>J</i> = 6.0, 3.6 Hz, 4H), 5.50–5.28 (m, 1H), 4.67–4.48 (m, 2H), 4.25 (s, 2H), 4.22–4.18 (m, 1H), 3.82–3.73 (m, 1H), 3.62–3.51 (m, 1H), 3.44–3.37 (m, 1H), 3.24 (t, <i>J</i> = 12.2 Hz, 1H), 2.88 (s, 3H), 2.38–2.30 (m, 1H), 2.22 (t, <i>J</i> = 7.6 Hz, 2H), 2.14–1.92 (m, 3H), 1.87–1.63 (m, 4H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>33</sub>H<sub>41</sub>F<sub>2</sub>N<sub>7</sub>O<sub>8</sub>P<sup>+</sup> 732.27, found [M + H]<sup>+</sup> 732.44.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-3-Acetyl-8-(((<i>S</i>)-5-amino-1-(benzylamino)-1,5-dioxopentan-2-yl)carbamoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>6</b>)</h3><div class="NLM_p last">Compound <b>6</b> was synthesized by employing the similar method as for compound <b>5</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.82 (s, 1H), 7.47–7.43 (m, 1H), 7.39–7.37 (m, 1H), 7.28–7.20 (m, 2H), 7.18–7.15 (m, 4H), 5.06–4.91 (m, 1H), 4.37–4.17 (m, 5H), 3.87–3.70 (m, 1H), 3.66–3.29 (m, 3H), 2.28–2.10 (m, 3H), 2.09–1.91 (m, 5H), 1.87–1.59 (m, 5H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>34</sub>H<sub>41</sub>F<sub>2</sub>N<sub>7</sub>O<sub>9</sub>P<sup>+</sup> 760.27, found [M + H]<sup>+</sup> 760.38.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-methyl-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>7</b>)</h3><div class="NLM_p last">Compound <b>7</b> was prepared from <b>56</b> by a procedure similar to that used for compound <b>5</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.91 (s, 1H), 7.62–7.43 (m, 2H), 7.42–7.14 (m, 11H), 6.10 (s, 1H), 5.48–5.46 (m, 1H), 4.76–4.57 (m, 2H), 4.42–4.39 (m, 1H), 3.92–3.84 (m, 1H), 3.67–3.65 (m, 1H), 3.49–3.46 (m, 1H), 3.36–3.30 (m, 1H), 2.96 (s, 3H), 2.51–2.23 (m, 3H), 2.17–2.04 (m, 3H), 1.92–1.87 (m, 2H), 1.80–1.72 (m, 2H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>39</sub>H<sub>45</sub>F<sub>2</sub>N<sub>7</sub>O<sub>8</sub>P<sup>+</sup> 808.30, found [M + H]<sup>+</sup> 808.46.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-3-Acetyl-8-(((S)-5-amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (SI-109, <b>8</b>)</h3><div class="NLM_p">HATU (8.5 g, 22.3 mmol, 1.1 equiv) was added to a solution of the Boc-Gln-OH <b>57</b> (5.0 g, 20.3 mmol, 1 equiv), aminodiphenylmethane hydrochloride <b>58</b> (4.5 g, 20.3 mmol, 1 equiv), and DIEA (10.6 mL, 60.9 mmol, 3 equiv) in DMF (60 mL), and the resulting mixture was stirred at rt for 1 h. The solution was diluted with EtOAc and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by flash chromatography on silica gel to afford <b>59</b> (7.3 g 87%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.95 (d, <i>J</i> = 6.5 Hz, 1H), 7.35–7.23 (m, 10H), 6.36 (s, 1H), 6.21 (d, <i>J</i> = 8.2 Hz, 1H), 5.89 (d, <i>J</i> = 5.8 Hz, 1H), 5.74 (s, 1H), 4.24 (s, 1H), 2.33–2.26 (m, 1H), 2.21–2.11 (m, 1H), 2.09–2.00 (m, 1H), 1.92–1.87 (m, 1H), 1.43 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 175.42, 171.02, 156.26, 141.50, 141.32, 128.63, 127.44, 80.10, 56.92, 53.75, 31.91, 29.10, 28.31.</div><div class="NLM_p">TFA (5 mL) was added slowly to a solution of <b>59</b> (3 g) in DCM (50 mL), and the resulting reaction solution was stirred at rt for 6 h and then evaporated. The residue, consisting of <b>60</b> was used directly in the next step without further purification.</div><div class="NLM_p">HATU (1.59 g, 4.2 mmol, 1.1 equiv) was added to a solution of <b>51</b> (1.75 g, 3.8 mmol, 1 equiv), <b>60</b> (1.61 g, 3.8 mmol, 1 equiv), and DIEA (1.98 mL, 11.4 mmol, 3 equiv) in DMF (15 mL), and the resulting mixture was stirred at rt for 1 h. The solution was diluted with EtOAc and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution and brine and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by flash chromatography on silica gel to afford <b>61</b> (2.4 g, 84%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.46–7.39 (m, 2H), 7.38–7.19 (m, 13H), 6.16–6.15 (m, 1H), 5.20–5.18 (m, 2H), 4.74–4.57 (m, 1H), 4.56–4.37 (m, 2H), 4.25–4.23 (m, 1H), 3.87–3.35 (m, 4H), 2.55–2.28 (m, 2H), 2.27–1.58 (m, 8H), 1.46 (s, 9H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>41</sub>H<sub>51</sub>N<sub>6</sub>O<sub>8</sub>P<sup>+</sup> 755.38, found [M + H]<sup>+</sup> 755.52.</div><div class="NLM_p">10% Pd–C (150 mg) was added to a solution of compound <b>61</b> (0.5 g, 0.66 mmol) in MeOH (30 mL). The solution was stirred under 1 atm of H<sub>2</sub> at rt for 2 h before filtration through Celite and being concentrated. The resulting amine was dissolved in DCM (30 mL), and Ac<sub>2</sub>O (135 mg, 1.3 mmoL, 2 equiv) and DIEA (0.35 mL, 2.0 mmol, 3 equiv) were added. The resulting reaction mixture was stirred for 30 min and was then evaporated. The residue was treated with TFA (3 mL) in DCM (30 mL) at rt, and the resulting reaction solution was stirred at rt for 6 h and then evaporated. The residue was purified by HPLC to afford compound <b>62</b> (264 mg, 71% over three steps). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.40–7.21 (m, 10H), 6.17 (s, 1H), 4.66–4.36 (m, 3H), 4.30–3.95 (m, 2H), 3.83–3.54 (m, 3H), 2.46–1.75 (m, 13H).</div><div class="NLM_p last">HATU (22 mg, 0.06 mmol, 1.1 equiv) was added to a solution of compound <b>62</b> (30 mg, 0.05 mmol, 1 equiv), <b>46</b> (19 mg, 0.05 mmol, 1 equiv), and DIEA (28 μL, 0.16 mmoL, 3 equiv) in DMF (2 mL). The resulting mixture was stirred at rt for 1 h and diluted with EtOAc, then washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution and brine, and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by flash chromatography on silica gel. The obtained product was dissolved in DCM (2 mL) and cooled to 0 °C before adding CF<sub>3</sub>CON(TMS)<sub>2</sub> (82 mg, 0.32 mmol, 6.0 equiv) and 1 M of TMS-I in DCM (0.21 mL, 0.21 mmol, 4.0 equiv). The reaction mixture was stirred at 0 °C for 10 min, and the solvent was removed under vacuum at 0 °C. The residue was dissolved in a mixed solvent of CH<sub>3</sub>CN (1 mL), water (2 mL), and TFA (0.1 mL) and purified by HPLC to yield SI-109 (<b>8</b>, 28 mg, 63% over two steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.90 (s, 1H), 7.56–7.52 (m, 1H), 7.48–7.46 (m, 1H), 7.35–7.21 (m, 11H), 6.07–6.06 (m, 1H), 5.11–5.02 (m, 1H), 4.42–4.23 (m, 3H), 3.95–3.80 (m, 1H), 3.77–3.53 (m, 2H), 3.49–3.37 (m, 1H), 2.32–2.26 (m, 2H), 2.19–2.14 (m, 4H), 2.11–2.01 (m, 2H), 1.96–1.58 (m, 5H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>40</sub>H<sub>45</sub>F<sub>2</sub>N<sub>7</sub>O<sub>9</sub>P<sup>+</sup> 836.30, found [M + H]<sup>+</sup> 836.43.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-3-Acetyl-8-(((<i>S</i>)-5-amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl Dihydrogen Phosphate (<b>9</b>)</h3><div class="NLM_p last">Compound <b>9</b> was prepared from <b>62</b> by a similar procedure as was used for compound <b>2</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.39–7.35 (m, 2H), 7.27–7.17 (m, 10H), 7.06–7.03 (m, 2H), 5.99–5.98 (m, 1H), 5.03–4.94 (m, 1H), 4.20–4.23 (m, 3H), 3.88–3.46 (m, 3H), 3.43–3.23 (m, 1H), 2.24–2.19 (m, 2H), 2.16–1.94 (m, 6H), 1.89–1.56 (m, 5H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>39</sub>H<sub>45</sub>N<sub>7</sub>O<sub>10</sub>P<sup>+</sup> 802.30, found [M + H]<sup>+</sup> 802.41.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-3-Acetyl-8-(((<i>S</i>)-5-amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)methyl)phosphonic Acid (<b>10</b>)</h3><div class="NLM_p last">Compound <b>10</b> was prepared from <b>62</b> by a similar procedure as was used for compound <b>3</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.48 (s, 1H), 7.38–7.36 (m, 1H), 7.33–7.09 (m, 11H), 7.06–7.02 (m, 1H), 6.00–5.99 (m, 1H), 5.07–4.92 (m, 1H), 4.40–4.22 (m, 3H), 3.80–3.59 (m, 3H), 3.41–3.29 (m, 1H), 3.13 (d, <i>J</i> = 21.1 Hz, 2H), 2.25–2.20 (m, 2H), 2.17–1.96 (m, 6H), 1.89–1.54 (m, 5H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>40</sub>H<sub>47</sub>N<sub>7</sub>O<sub>9</sub>P <sup>+</sup> 800.32, found [M + H]<sup>+</sup> 800.45.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> ((4-((<i>E</i>)-4-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-3-Acetyl-8-(((<i>S</i>)-5-amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)amino)-4-oxobut-2-en-2-yl)phenyl)difluoromethyl)phosphonic Acid (<b>11</b>)</h3><div class="NLM_p last">Compound <b>11</b> was prepared from <b>62</b> and the previously reported protected phosphotyrosine mimetic, (<i>E</i>)-3-(4-((diethoxyphosphoryl)difluoromethyl)phenyl)but-2-enoic acid,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> by a similar procedure to that used for compound <b>8</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.56–7.40 (m, 4H), 7.28–7.07 (m, 10H), 6.19–6.17 (m, 1H), 5.94 (s, 1H), 4.94–4.71 (m, 1H), 4.29–4.17 (m, 3H), 3.78–3.03 (m, 4H), 2.32 (s, 3H), 2.19–2.16 (m, 2H), 2.08–1.88 (m, 6H), 1.85–1.48 (m, 5H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>41</sub>H<sub>48</sub>F<sub>2</sub>N<sub>6</sub>O<sub>9</sub>P<sup>+</sup> 837.32, found [M + H]<sup>+</sup> 837.57.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(6-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)hex-5-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>12</b>)</h3><div class="NLM_p last">Compound <b>12</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.99–7.79 (m, 1H), 7.78–7.43 (m, 4H), 7.42–6.93 (m, 12H), 6.09–5.98 (m, 1H), 5.11–4.89 (m, 2H), 4.50–4.24 (m, 5H), 3.91–3.15 (m, 4H), 2.86–2.47 (m, 6H), 2.42–1.99 (m, 8H), 1.93–1.45 (m, 6H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>57</sub>H<sub>60</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>2+</sup> 565.71, found [M + H]<sup>2+</sup> 565.96.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(7-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)hept-6-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>13</b>)</h3><div class="NLM_p last">Compound <b>13</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.99–7.81 (m, 1H), 7.76–7.43 (m, 4H), 7.43–7.00 (m, 12H), 6.06–6.03 (m, 1H), 5.19–4.88 (m, 2H), 4.45–4.18 (m, 5H), 3.95–3.65 (m, 3H), 3.37–3.28 (m, 1H), 3.02–1.99 (m, 14H), 1.93–1.51 (m, 8H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>58</sub>H<sub>61</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>+</sup> 1144.41, found [M + H]<sup>+</sup> 1144.45.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (SD-36, <b>14</b>)</h3><div class="NLM_p">TFA (3 mL) was added slowly to a solution of <b>61</b> (2g, 2.65 mmol) in DCM (30 mL) at rt, and the resulting solution was stirred at rt for 6 h and then evaporated. The crude product was directly used in the next step without purification. HATU (1.1 g, 2.91 mmol, 1.1 equiv) was added to a solution of the crude product of the previous step (2.65 mmol, 1 equiv), <b>46</b> (0.92 g, 2.65 mmol, 1 equiv), and DIEA (1.39 mL, 7.95 mmoL, 3 equiv) in DMF (15 mL). The resulting mixture was stirred at rt for 1 h and diluted with EtOAc and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by flash chromatography on silica gel to afford compound <b>63</b>. 10% Pd–C (200 mg) was added to a solution of the obtained <b>63</b> in MeOH (60 mL). The solution was stirred under 1 atm of H<sub>2</sub> at rt for 3 h before being filtered through Celite and concentrated. The resulting amine was purified by HPLC to afford compound <b>64</b> (1.73 g, 77% over three steps). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.92 (s, 1H), 7.57 (d, <i>J</i> = 8.7 Hz, 1H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.31 (m, 5H), 7.30–7.24 (m, 6H), 6.19 (s, 1H), 5.61 (dd, <i>J</i> = 12.1, 5.6 Hz, 1H), 4.78 (t, <i>J</i> = 8.9 Hz, 1H), 4.70–4.66 (m, 1H), 4.59 (dd, <i>J</i> = 9.5, 5.0 Hz, 1H), 4.29–4.05 (m, 4H), 3.72–3.61 (m, 2H), 3.54 (t, <i>J</i> = 12.6 Hz, 1H), 3.42 (t, <i>J</i> = 12.4 Hz, 1H), 2.45–2.27 (m, 3H), 2.24–2.01 (m, 3H), 2.00–1.70 (m, 4H), 1.29 (td, <i>J</i> = 7.1, 3.2 Hz, 6H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>42</sub>H<sub>51</sub>F<sub>2</sub>N<sub>7</sub>O<sub>8</sub>P<sup>+</sup> 850.35, found [M + H]<sup>+</sup> 850.37.</div><div class="NLM_p">Trimethylamine (10 mL) was added to a mixture of compound <b>66</b> (1.3 g, 4.0 mmol, 1 equiv), oct-7-ynoic acid <b>67</b> (0.56 g, 4.0 mmol, 1 equiv), CuI (154 mg, 0.8 mmol, 0.2 equiv), and Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (282 mg, 0.4 mmol, 0.1 equiv) in DMF (10 mL). The resulting mixture was purged and refilled with argon three times and then stirred at 70–80 °C for 3 h under argon. The reaction mixture was then cooled to rt and evaporated to remove the solvent. The residue was purified by HPLC to yield compound <b>68</b> (1.18 g, 76%). <sup>1</sup>H NMR (400 MHz, DMSO) δ 11.99 (s, 1H), 10.99 (s, 1H), 7.77–7.68 (m, 1H), 7.68–7.59 (m, 1H), 7.52 (t, <i>J</i> = 7.6 Hz, 1H), 5.15 (dd, <i>J</i> = 13.3, 5.0 Hz, 1H), 4.46 (d, <i>J</i> = 17.7 Hz, 1H), 4.33 (d, <i>J</i> = 17.7 Hz, 1H), 2.97–2.88 (m, 1H), 2.63–2.59 (m, 1H), 2.53–2.47 (m, 3H), 2.24 (t, <i>J</i> = 7.2 Hz, 2H), 2.13–1.94 (m, 1H), 1.78–1.27 (m, 6H). <sup>13</sup>C NMR (101 MHz, DMSO) δ 174.90, 173.32, 171.45, 168.14, 144.23, 134.52, 132.46, 129.05, 123.05, 119.32, 96.73, 76.91, 52.14, 47.47, 34.07, 31.68, 28.32, 28.25, 24.50, 22.83, 19.14. UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>21</sub>H<sub>23</sub>N<sub>2</sub>O<sub>5</sub><sup>+</sup> 383.16, found [M + H]<sup>+</sup> 383.28.</div><div class="NLM_p">HATU (295 mg, 0.77 mmol, 1.1 equiv) was added to a solution of <b>64</b> (600 mg, 0.71 mmol, 1 equiv), <b>68</b> (270 g, 0.71 mmol, 1 equiv), and DIEA (0.37 mL, 2.12 mmol, 3 equiv) in DMF (10 mL), and the resulting mixture was stirred at rt for 1 h. The solution was diluted with EtOAc and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine and dried over sodium sulfate. After removal of the solvent <i>in vacuo</i>, the residue was purified by flash chromatography on silica gel to afford <b>65</b> (702 mg 82%). <sup>1</sup>H NMR (400 MHz, MeOD:CDCl<sub>3</sub> = 1:1) δ 7.90 (s, 1H), 7.76–7.66 (m, 1H), 7.57–7.51 (m, 2H), 7.47–7.35 (m, 2H), 7.33–7.19 (m, 11H), 6.27–6.06 (m, 1H), 5.33–4.87 (m, 2H), 4.55–4.37 (m, 4H), 4.26–3.75 (m, 7H), 3.71–3.26 (m, 2H), 2.90–2.71 (m, 2H), 2.67–1.85 (m, 14H), 1.83–1.47 (m, 8H), 1.29 (t, <i>J</i> = 7.1 Hz, 6H).</div><div class="NLM_p last">Compound <b>65</b> (500 mg, 0.41 mmol, 1.0 equiv) and DCM (40 mL) were placed in a round bottomed flask. The solution was cooled to 0 °C before adding CF<sub>3</sub>CON(TMS)<sub>2</sub> (635 mg, 2.5 mmol, 6.0 equiv) and a 1 M solution of TMS-I in DCM (1.65 mL, 1.65 mmol, 4.0 equiv). The reaction mixture was stirred at 0 °C for 10 min, and then the solvent was removed under vacuum at 0 °C. The residue was dissolved in a mixture of CH<sub>3</sub>CN (5 mL), H<sub>2</sub>O (5 mL), and TFA (0.3 mL) and purified by HPLC to yield SD-36 (<b>14</b>, 420 mg, 88%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ δ 7.90–7.85 (m, 1H), 7.73–7.59 (m, 1H), 7.56–7.36 (m, 4H), 7.35–7.05 (m, 11H), 6.06–6.03 (m, 1H), 5.12–4.87 (m, 2H), 4.48–4.19 (m, 5H), 3.98–3.49 (m, 3H), 3.36–3.24 (m, 1H), 2.88–2.64 (m, 2H), 2.61–2.23 (m, 7H), 2.21–1.99 (m, 4H), 1.94–1.35 (m, 11H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>59</sub>H<sub>63</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>+</sup> 1158.43, found [M + H]<sup>+</sup> 1158.60.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(9-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)non-8-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>15</b>)</h3><div class="NLM_p last">Compound <b>15</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.99–7.87 (m, 1H), 7.73–7.41 (m, 5H), 7.40–6.99 (m, 11H), 6.07–6.04 (m, 1H), 5.12–4.91 (m, 2H), 4.46–4.21 (m, 5H), 3.98–3.53 (m, 3H), 3.34–3.26 (m, 1H), 2.88–2.68 (m, 2H), 2.57–2.06 (m, 10H), 1.90–1.21 (m, 14H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>60</sub>H<sub>66</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>2+</sup> 586.73, found [M + 2H]<sup>2+</sup> 586.95.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(11-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)undec-10-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>16</b>)</h3><div class="NLM_p last">Compound <b>16</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.89–7.88 (m, 1H), 7.72–7.67 (m, 1H), 7.60–7.39 (m, 4H), 7.37–7.13 (m, 11H), 6.06–6.03 (m, 1H), 5.17–4.86 (m, 2H), 4.47–4.19 (m, 5H), 3.95–3.53 (m, 3H), 3.38–3.27 (m, 1H), 2.90–2.67 (m, 2H), 2.57–2.01 (m, 11H), 1.93–1.20 (m, 17H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>62</sub>H<sub>70</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>2+</sup> 600.74, found [M + 2H]<sup>2+</sup> 600.88.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-yn-1-yl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>17</b>)</h3><div class="NLM_p last">Compound <b>17</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.86 (s, 1H), 7.67–7.64 (m, 1H), 7.48–7.37 (m, 4H), 7.29–7.14 (m, 11H), 6.01 (s, 1H), 5.45 (br, 1H), 5.02–4.96 (m, 1H), 4.68–4.53 (m, 2H), 4.32–4.21 (m, 3H), 3.73–3.70 (m, 1H), 3.54–3.49 (m, 2H), 3.29 (t, <i>J</i> = 1.2 Hz, 1H), 3.14–3.08 (m, 2H), 2.82–2.64 (m, 2H), 2.38–2.15 (m, 6H), 2.14–1.95 (m, 3H), 1.93–1.23 (m, 13H). UPLC–MS calculated for C<sub>59</sub>H<sub>64</sub>F<sub>2</sub>N<sub>8</sub>O<sub>11</sub>PS [M + H]<sup>+</sup>: 1144.45, found 1144.50.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(((6-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)hex-5-yn-1-yl)oxy)carbonyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>18</b>)</h3><div class="NLM_p last">Compound <b>18</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.90 (s, 1H), 7.72 (d, <i>J</i> = 7.5 Hz, 1H), 7.65–7.39 (m, 4H), 7.38–6.99 (m, 11H), 6.05–6.03 (m, 1H), 5.12–4.90 (m, 2H), 4.46–4.16 (m, 6H), 4.07–3.97 (m, 1H), 3.68–3.31 (m, 4H), 2.86–2.69 (m, 2H), 2.56–1.98 (m, 10H), 1.90–1.43 (m, 8H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>58</sub>H<sub>63</sub>F<sub>2</sub>N<sub>9</sub>O<sub>13</sub>P<sup>2+</sup> 580.71, found [M + H]<sup>2+</sup> 581.11</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-((6-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)hex-5-yn-1-yl)carbamoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>19</b>)</h3><div class="NLM_p last">Compound <b>19</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.84 (d, <i>J</i> = 4.8 Hz, 1H), 7.63–7.55 (m, 1H), 7.53–7.42 (m, 3H), 7.41–7.01 (m, 12H), 6.07 (s, 1H), 5.04–4.92 (m, 1H), 4.78 (d, <i>J</i> = 8.1 Hz, 1H), 4.45–4.32 (m, 4H), 4.09–4.04 (m, 1H), 3.87 (d, <i>J</i> = 13.5 Hz, 2H), 3.43–3.22 (m, 2H), 3.15–3.09 (m, 2H), 2.88–2.63 (m, 2H), 2.51 (t, <i>J</i> = 6.4 Hz, 2H), 2.39–2.16 (m, 4H), 2.12–2.02 (m, 4H), 1.96–1.90 (m, 1H), 1.83–1.43 (m, 7H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>58</sub>H<sub>62</sub>F<sub>2</sub>N<sub>10</sub>O<sub>12</sub>P<sup>+</sup> 1159.42, found [M + H]<sup>+</sup> 1159.44.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(1-(5-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pent-4-yn-1-yl)-1<i>H</i>-pyrazole-4-carbonyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>20</b>)</h3><div class="NLM_p last">Compound <b>20</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 8.14–8.03 (m, 1H), 7.86–7.60 (m, 3H), 7.51–7.10 (m, 15H), 6.00–5.99 (m, 1H), 5.15–4.95 (m, 2H), 4.27–4.18 (m, 7H), 3.82–3.48 (m, 3H), 2.74–2.66 (m, 2H), 2.38–1.95 (m, 11H), 1.92–1.54 (m, 5H). UPLC–MS calculated for C<sub>60</sub>H<sub>61</sub>F<sub>2</sub>N<sub>11</sub>O<sub>12</sub>P [M + H]<sup>+</sup>: 1196.42, found 1196.28.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(4-(4-((2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)ethynyl)-1<i>H</i>-pyrazol-1-yl)butanoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>21</b>)</h3><div class="NLM_p last">Compound <b>21</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.89–7.78 (m, 2H), 7.71–7.60 (m, 2H), 7.50–7.34 (m, 4H), 7.27–7.13 (m, 11H), 6.01–5.96 (m, 1H), 5.05–4.88 (m, 2H), 4.37–4.21 (m, 7H), 3.75–3.29 (m, 4H), 2.82–2.70 (m, 2H), 2.65–1.94 (m, 11H), 1.93–1.65 (m, 5H). UPLC–MS calculated for C<sub>60</sub>H<sub>61</sub>F<sub>2</sub>N<sub>11</sub>O<sub>12</sub>P [M + H]<sup>+</sup>: 1196.42, found 1196.44.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(4-(4-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)but-3-yn-1-yl)piperidine-1-carbonyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>22</b>)</h3><div class="NLM_p last">Compound <b>22</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.84 (s, 1H), 7.70–7.68 (m, 1H), 7.55–7.41 (m, 4H), 7.29–7.13 (m, 11H), 6.02 (s, 1H), 5.14–5.02 (m, 2H), 4.44–4.13 (m, 6H), 3.64–3.52 (m, 4H), 3.27–3.15 (m, 2H), 2.83–2.61 (m, 4H), 2.45–2.02 (m, 9H), 1.92–1.49 (m, 9H), 1.20–1.05 (m, 2H). UPLC–MS calculated for C<sub>61</sub>H<sub>66</sub>F<sub>2</sub>N<sub>10</sub>O<sub>12</sub>P [M + H]<sup>+</sup>: 1199.46, found 1199.48.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(3-(4-((2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)ethynyl)piperidin-1-yl)propanoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>23</b>)</h3><div class="NLM_p last">Compound <b>23</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.88–7.85 (m, 1H), 7.74–7.70 (m, 1H), 7.64–7.57 (m, 1H), 7.50–7.42 (m, 3H), 7.29–7.13 (m, 11H), 6.01–5.97 (m, 1H), 5.20–5.00 (m, 2H), 4.43–4.18 (m, 5H), 3.76–3.32 (m, 8H), 3.21–2.72 (m, 7H), 2.39–2.04 (m, 9H), 1.85–1.65 (m, 7H). UPLC–MS calculated for C<sub>61</sub>H<sub>66</sub>F<sub>2</sub>N<sub>10</sub>O<sub>12</sub>P [M + H]<sup>+</sup>: 1199.46, found 1199.68.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(3-(4-((2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)ethynyl)piperidin-1-yl)-3-oxopropanoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>24</b>)</h3><div class="NLM_p last">Compound <b>24</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.87–7.67 (m, 2H), 7.51–7.38 (m, 3H), 7.30–7.13 (m, 12H), 6.03–5.97 (m, 1H), 5.21–5.01 (m, 2H), 4.35–4.19 (m, 5H), 3.85–3.62 (m, 5H), 3.36–3.32 (m, 4H), 2.92–2.65 (m, 4H), 2.35–2.04 (m, 9H), 1.88–1.69 (m, 7H). UPLC–MS calculated for C<sub>61</sub>H<sub>64</sub>F<sub>2</sub>N<sub>10</sub>O<sub>13</sub>P [M + H]<sup>+</sup>: 1213.44, found 1213.56.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)octanoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>25</b>)</h3><div class="NLM_p last">Compound <b>25</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.90–7.88 (m, 1H), 7.69–7.41 (m, 4H), 7.41–7.07 (m, 12H), 6.07–6.03 (m, 1H), 5.13–4.90 (m, 2H), 4.51–4.18 (m, 5H), 3.99–3.55 (m, 3H), 3.36–3.27 (m, 1H), 3.01–2.02 (m, 13H), 1.91–1.09 (m, 15H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>59</sub>H<sub>67</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>+</sup> 1162.46, found [M + H]<sup>+</sup> 1162.44.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(7-((2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)amino)heptanoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>26</b>)</h3><div class="NLM_p last">Compound <b>26</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) 7.89–7.87 (m, 1H), 7.61–7.41 (m, 4H), 7.38–7.08 (m, 12H), 6.06–6.03 (m, 1H), 5.13–4.90 (m, 2H), 4.50–4.18 (m, 5H), 3.97–3.50 (m, 3H), 3.41–3.33 (m, 1H), 3.26–3.30 (m, 2H), 2.90–2.65 (m, 2H), 2.56–2.00 (m, 9H), 1.92–1.84 (m, 2H), 1.73–1.55 (m, 7H), 1.42–1.21 (m, 4H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>57</sub>H<sub>60</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>2+</sup> 565.71, found [M + H]<sup>2+</sup> 582.65</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>27</b>)</h3><div class="NLM_p last">Compound <b>27</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.89–7.85 (m, 1H), 7.76–7.65 (m, 3H), 7.57–7.41 (m, 2H), 7.38–7.08 (m, 11H), 6.06–6.03 (m, 1H), 5.13–4.84 (m, 2H), 4.35–4.21 (m, 3H), 3.3.95–3.65 (m, 3H), 3.38–3.26 (m, 1H), 2.87–2.68 (m, 2H), 2.65–2.36 (m, 5H), 2.33–2.00 (m, 6H), 1.96–1.36 (m, 11H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>59</sub>H<sub>61</sub>F<sub>2</sub>N<sub>9</sub>O<sub>13</sub>P<sup>+</sup> 1172.41, found [M + H]<sup>+</sup> 1172.29</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-5-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>28</b>)</h3><div class="NLM_p last">Compound <b>28</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.93–7.83 (m, 1H), 7.65–7.54 (m, 1H), 7.47–7.10 (m, 15H), 6.03–5.99 (m, 1H), 5.00–4.90 (m, 2H), 4.30–4.18 (m, 6H), 3.91–3.63 (m, 3H), 3.29–3.26 (m, 1H), 2.76–2.57 (m, 3H), 2.45–1.94 (m, 10H), 1.93–1.44 (m, 10H). UPLC–MS calculated for C<sub>59</sub>H<sub>64</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>2+</sup>: 579.72, found 580.24</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-3-oxoisoindolin-5-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>29</b>)</h3><div class="NLM_p last">Compound <b>29</b> was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.95–7.82 (m, 1H), 7.68–7.61 (m, 1H), 7.53–7.09 (m, 15H), 6.03–5.99 (m, 1H), 4.99–4.90 (m, 2H), 4.31–4.15 (m, 6H), 3.93–3.64 (m, 3H), 3.29–3.28 (m, 1H), 2.78–2.69 (m, 2H), 2.57–2.13 (m, 7H), 2.03–1.94 (m, 4H), 1.92–1.47 (m, 10H). UPLC–MS calculated for C<sub>59</sub>H<sub>63</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P [M + H]<sup>+</sup>: 1158.43, found 1158.45.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>30</b>)</h3><div class="NLM_p">Compound <b>69</b> was prepared from <b>55</b> by a procedure similar to that used for compound <b>56</b>. <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.91 (s, 1H), 7.76–7.67 (m, 1H), 7.64–7.53 (m, 2H), 7.51–7.40 (m, 2H), 7.33–7.24 (m, 1H), 5.27–5.07 (m, 2H), 4.59–4.40 (m, 4H), 4.35–4.25 (m, 1H), 4.25–4.09 (m, 4H), 4.02–3.99 (m, 1H), 3.82–3.54 (m, 3H), 2.95–2.47 (m, 7H), 2.45–1.85 (m, 10H), 1.83–1.49 (m, 7H), 1.30 (t, <i>J</i> = 7.1 Hz, 6H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>50</sub>H<sub>60</sub>F<sub>2</sub>N<sub>8</sub>O<sub>13</sub>P<sup>+</sup> 1049.40, found [M + H]<sup>+</sup> 1049.53.</div><div class="NLM_p last">Compound <b>30</b> was prepared from <b>69</b> by a procedure similar to that used for compound <b>5</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.98–7.87 (m, 1H), 7.79–7.45 (m, 4H), 7.44–7.35 (m, 1H), 7.35–6.92 (m, 6H), 5.12–4.93 (m, 2H), 4.46–4.21 (m, 6H), 3.94–3.28 (m, 5H), 2.81–2.73 (m, 2H), 2.63–1.99 (m, 12H), 1.92–1.38 (m, 10H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>53</sub>H<sub>59</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>+</sup> 1082.40, found [M + H]<sup>+</sup> 1082.42.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(methylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>31</b>)</h3><div class="NLM_p last">Compound <b>31</b> was prepared from <b>69</b> by a procedure similar to that used for compound <b>5</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.97–7.82 (m, 1H), 7.81–6.96 (m, 6H), 5.16–4.93 (m, 2H), 4.49–4.18 (m, 5H), 3.95–3.18 (m, 4H), 3.00–1.99 (m, 14H), 1.95–1.08 (m, 13H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>47</sub>H<sub>55</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>+</sup> 1006.37, found [M + H]<sup>+</sup> 1006.49.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl Dihydrogen Phosphate (<b>32</b>)</h3><div class="NLM_p">Compound <b>70</b> was prepared from <b>61</b> by a procedure similar to that used for compound <b>62</b>. <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.74 (d, <i>J</i> = 7.6 Hz, 1H), 7.63–7.56 (m, 1H), 7.49 (t, <i>J</i> = 7.6 Hz, 1H), 7.36–7.22 (m, 10H), 6.16 (s, 1H), 5.26–5.11 (m, 1H), 4.65–4.34 (m, 5H), 4.20–4.04 (m, 2H), 3.88–3.40 (m, 3H), 2.93–2.74 (m, 2H), 2.67–2.23 (m, 8H), 2.19–1.52 (m, 14H).</div><div class="NLM_p last">Compound <b>32</b> was prepared from <b>70</b> by a procedure similar to that used for compound <b>2</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.66–7.54 (m, 1H), 7.52–7.09 (m, 14H), 7.07–6.92 (m, 2H), 6.00–5.96 (m, 1H), 5.05–4.82 (m, 2H), 4.47–4.22 (m, 5H), 3.89–3.46 (m, 3H), 3.26–3.19 (m, 1H), 2.76–2.62 (m, 2H), 2.57–1.97 (m, 12H), 1.85–1.35 (m, 10H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>58</sub>H<sub>64</sub>N<sub>9</sub>O<sub>13</sub>P<sup>2+</sup> 562.72, found [M + 2H]<sup>2+</sup> 563.17.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)methyl)phosphonic Acid (<b>33</b>)</h3><div class="NLM_p last">Compound <b>33</b> was prepared from <b>70</b> by a procedure similar to that used for compound <b>3</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.67–7.54 (m, 1H), 7.54–6.82 (m, 16H), 6.01–5.96 (m, 1H), 5.05–4.82 (m, 2H), 4.41–4.22 (m, 5H), 3.88–3.53 (m, 3H), 3.27–3.07 (m, 3H), 2.77–2.66 (m, 2H), 2.56–1.94 (m, 12H), 1.87–1.35 (m, 10H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>59</sub>H<sub>66</sub>N<sub>9</sub>O<sub>12</sub>P<sup>2+</sup> 561.73, found [M + 2H]<sup>2+</sup> 561.97.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> ((4-((<i>E</i>)-4-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)amino)-4-oxobut-2-en-2-yl)phenyl)difluoromethyl)phosphonic Acid (<b>34</b>)</h3><div class="NLM_p last">Compound <b>34</b> was prepared from <b>70</b> by a procedure similar to that used for compound <b>4</b>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.66–7.56 (m, 1H), 7.52–7.29 (m, 6H), 7.28–7.02 (m, 10H), 6.19 (s, 1H), 5.97–5.93 (m, 1H), 5.00–4.72 (m, 2H), 4.45–4.17 (m, 5H), 4.11–4.00 (m, 1H), 3.85–3.30 (m, 3H), 3.20–3.05 (m, 1H), 2.77–2.61 (m, 2H), 2.52–1.91 (m, 14H), 1.87–1.30 (m, 11H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>60</sub>H<sub>66</sub>F<sub>2</sub>N<sub>8</sub>O<sub>12</sub>P <sup>+</sup> 1159.45, found [M + H]<sup>+</sup> 1159.64.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(benzhydrylamino)-1,5-dioxopentan-2-yl)carbamoyl)-3-(8-(2-(1-methyl-2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)oct-7-ynoyl)-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>35</b>, SD-36Me)</h3><div class="NLM_p last">SD-36Me was prepared from <b>64</b> by a procedure similar to that used for SD-36. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN:D<sub>2</sub>O = 1:1) δ 7.88 (d, <i>J</i> = 16.1 Hz, 1H), 7.72–7.59 (m, 1H), 7.56–7.36 (m, 4H), 7.35–7.06 (m, 11H), 6.06–6.03 (m, 1H), 5.12–4.87 (m, 2H), 4.45–4.20 (m, 5H), 3.98–3.50 (m, 3H), 3.37–3.22 (m, 1H), 3.08–2.94 (m, 3H), 2.88–2.73 (m, 2H), 2.61–2.37 (m, 4H), 2.34–1.99 (m, 7H), 1.95–1.81 (m, 2H), 1.79–1.40 (m, 9H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>60</sub>H<sub>65</sub>F<sub>2</sub>N<sub>9</sub>O<sub>12</sub>P<sup>+</sup> 1172.45, found [M + H]<sup>+</sup> 1172.45.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> ((2-(((5<i>S</i>,8<i>S</i>,10a<i>R</i>)-8-(((<i>S</i>)-5-Amino-1-(((S)-2-((8-(3′,6′-dihydroxy-3-oxo-3<i>H</i>-spiro[isobenzofuran-1,9′-xanthene]-5-carboxamido)octyl)amino)-2-oxo-1-phenylethyl)amino)-1,5-dioxopentan-2-yl)carbamoyl)-3-methyl-6-oxodecahydropyrrolo[1,2-<i>a</i>][1,5]diazocin-5-yl)carbamoyl)-1<i>H</i>-indol-5-yl)difluoromethyl)phosphonic Acid (<b>74</b>)</h3><div class="NLM_p last">HATU (42 mg, 0.11 mmol, 1.1 equiv) was added to a solution of <b>71</b> (50 mg, 0.1 mmol, 1 equiv), (<i>S</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)-2-phenylacetic acid <b>72</b> (25 mg, 0.1 mmol, 1 equiv), and DIEA (0.05 mL, 0.3 mmoL, 3 equiv) in DMF (3 mL). The resulting mixture was stirred at rt for 1 h, then diluted with EtOAc, and washed with H<sub>2</sub>O, saturated sodium bicarbonate aqueous solution, and brine, and dried over sodium sulfate. After removal of the solvent under vacuum, the residue was dissolved in DCM (5 mL), and TFA (1 mL) was added. The resulting mixture was stirred at rt for 3 h. After removal of the organic solvent, the residual was purified by HPLC to afford compound <b>73</b> (32 mg, 50% over two steps). HATU (21 g, 0.055 mmol, 1.1 equiv) was added to a solution of <b>73</b> (32 mg, 0.05 mmol, 1 equiv), <b>56</b> (35 mg, 0.05 mmol, 1 equiv), and DIEA (0.025 mL, 0.15 mmoL, 3 equiv) in DMF (3 mL). The resulting mixture was stirred at rt for 1 h and directly purified by HPLC. Compound <b>74</b> was achieved using a similar method as for SD-36 from <b>65</b>, and the yield was 65% over two steps. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ 8.30 (s, 1H), 8.06 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 7.82 (s, 1H), 7.45–7.39 (m, 2H), 7.34–7.08 (m, 7H), 6.75–6.72 (m, 2H), 6.65–6.63 (m, 2H), 6.55–6.53 (m, 2H), 5.37 (s, 1H), 5.24 (s, 1H), 4.63–4.51 (m, 2H), 4.34–4.28 (m, 1H), 3.79–3.55 (m, 2H), 3.45–3.19 (m, 4H), 3.13–3.06 (m, 1H), 2.98–2.91 (m, 1H), 2.87 (s, 3H), 2.32–2.21(m, 3H), 2.11–1.93 (m, 3H), 1.86–1.65 (m, 4H), 1.56–1.39 (m, 2H), 1.33–1.01 (m, 10H). UPLC–MS (ESI-MS) <i>m</i>/<i>z</i>: calculated for C<sub>63</sub>H<sub>70</sub>F<sub>2</sub>N<sub>9</sub>O<sub>15</sub>P<sup>2+</sup> 630.73, found [M + 2H]<sup>2+</sup> 630.92.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Expression and Purification of STAT3</h3><div class="NLM_p last">STAT3 (residues 127–688) was cloned into a pHIS-SUMO vector, transformed into Rosetta DE3 cells, and grown in LB media overnight at 20 °C. HIS-SUMO-STAT3 was purified from the cleared cell lysate using Ni-NTA resin (Qiagen), then loaded onto a Superdex 200 column (GE Healthcare) in buffer containing 50 mM Tris, pH 8.0, 150 mM NaCl, 0.1% NP-40, 1 mM DTT, and 10% glycerol. For crystallography, the HIS-SUMO tag was cleaved overnight with SUMO protease. The cleaved protein was then applied to a Ni-NTA column, concentrated, and loaded onto a Superdex 200 column pre-equilibrated with 50 mM Tris, pH 8.5, 150 mM NaCl, and 1 mM DTT.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Fluorescence Polarization (FP) Assay</h3><div class="NLM_p last">FP assay was performed as described previously.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> To determine the dissociation constant (<i>K</i><sub>d</sub>) for the interaction between STAT3 SH2 domain binder <b>74</b> and STAT3, 5 nM <b>74</b> was incubated with serially diluted STAT3 proteins in FP assay buffer (50 mM NaCl, 10 mM Hepes, pH 7.5, 1 mM EDTA pH 8.0, 0.01% Triton X-100, 2 mM DTT). FP was measured 1 h after incubation on a Tecan Infinite microplate reader. <i>K</i><sub>d</sub> values were determined from the binding isotherm derived from curves of mP vs protein concentrations. For the competitive binding assay, STAT3 recombinant protein was first incubated with tracer <b>74</b> for 5 min, then added to the serially diluted compounds for 1 h. IC<sub>50</sub> values of <b>74</b> displacement were calculated by nonlinear regression analysis using GraphPad Prism software. The <i>K</i><sub>i</sub> values of competitive inhibitors were calculated as described previously.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Molecular Modeling</h3><div class="NLM_p last">The binding pose of compound <b>4</b> with STAT3 protein was modeled based on the crystal structure of STAT3 (pY705) with DNA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1">1BG1</a><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>). First, chain A of STAT3 (pY705) was extracted from the crystal structure and protons were added using the “protonate 3D” module in MOE<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> by setting the pH value at 7.0. Then, compound <b>4</b> was geometrically optimized using MOE. The GOLD program (version 5.2)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> was used for the docking calculations. In the docking simulation, the center of the binding site in STAT3 was set at V637 and the radius at 13 Å. Additional parameters in the genetic algorithm were the same as in our previous report.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> GoldScore was used as the fitness function to rank the docked poses. The top ranked pose was considered as the binding model and displayed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">Figure S2</a>. The figures were prepared using PyMOL.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Western Blot Analysis</h3><div class="NLM_p last">Western blotting was performed as described previously.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Cells were lysed in cell lysis buffer (Cell Signaling Technology, no. 9803), separated by SDS–PAGE NuPAGE gel (Thermo Fisher Scientific), and transferred to a PVDF membrane (Millipore). PVDF membranes were first blocked for 1 h using 5% blotting-grade blocker (no. 1706404, Bio-Rad) in Tris-buffered saline with Tween 20 (TBST, Pierce) and then were incubated with the primary and subsequently their corresponding secondary antibodies. SuperSignal West Pico Plus chemiluminescent substrate (Thermo Fisher Scientific) was used for signal development.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Cell Growth Assay</h3><div class="NLM_p last">Cell viability analysis was performed as described previously.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Cells grown in 384-well white plates (Corning Costar) were incubated with serially diluted compounds for 4 days. Cell viability was determined using the CellTiter-Glo luminescent cell viability assay (Promega) following the manufacturer’s instructions.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Crystallization and Structure Determination of STAT3 Bound to SI-109</h3><div class="NLM_p last">1.2-fold molar excess of SI-109 was incubated with STAT3 (6.25 mg/mL) for 3 h at 4 °C, and the crystals were grown by sitting drop vapor diffusion against well solution containing 10% PEG 3350 and 100–150 mM ammonium nitrate at 4 °C. Prior to flash freezing in liquid nitrogen, crystals were cryoprotected in mother liquor containing 20% ethylene glycol. Data were collected at the Life Sciences Collaborative Access Team beamline 21-ID-D at the Advanced Photon Source, Argonne National Laboratory. HKL2000<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was used to process data, and the structures were solved by molecular replacement using Phaser<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CWG">3CWG</a> as a search model. The structures were iteratively refined with Buster<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and fit with COOT.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Coordinates and geometric restraints for the compounds were generated in GRADE.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The complexes crystallized in space group <i>P</i>41212 with one molecule in the asymmetric unit. Residues 136–687 were modeled with the exception of the following missing loop regions: 181–191, 372–378, and 419–429. Prior to deposition, the structures were validated with Molprobity<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and the PDB validation server (wwPDB validation server).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> <i>In Vivo</i> PK/PD and Efficacy</h3><div class="NLM_p last">All the <i>in vivo</i> studies were carried out under the animal protocol PRO00007499, which has been approved by the Institutional Animal Care & Use Committee (IACUC) of the University of Michigan and is in accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. For PD studies in native mouse tissues, CD-1 female mice were treated with SD-36 or <b>19</b> or vehicle control (10% PEG400 + 90% PBS) as indicated in the figures. Mice were euthanized at the indicated time points in the figures, and mouse organs were harvested for Western blotting analysis. For efficacy and PD studies, CB.17 SCID mice were injected subcutaneously with 3 × 10<sup>6</sup> Molm-16 cells in 5 mg/mL Matrigel (Corning) for tumor growth. When the tumors reached an average volume of 150 mm<sup>3</sup>, mice were randomly assigned to different experimental groups for treatment. Drugs or vehicle control (10% PEG400 + 90% PBS) were given at the indicated dose schedules in the figures. Tumor sizes and animal weights were measured 2–3 times per week. Tumor volumes were calculated as tumor volume (mm<sup>3</sup>) = (length × width<sup>2</sup>)/2. For PD studies, mice were euthanized at the indicated time points in the figures, and tumor tissues or mouse organs were harvested for Western blotting analysis.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01530" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46385" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46385" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01530?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01530</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figures of FP binding experiments, computational model, and Western blotting analysis of total STAT3 and pSTAT3<sup>Y705</sup> proteins; tables of binding affinities and crystallography data collection and refinement statistics; <sup>1</sup>H NMR and UPLC–MS spectra for representative compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf">jm9b01530_si_001.pdf (6.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_002.csv">jm9b01530_si_002.csv (6.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_003.pdb">jm9b01530_si_003.pdb (250.59 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates for cocrystal structure SI-109 in complex with STAT3 have been deposited in the PDB under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ">6NUQ</a> and will be released upon publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01530" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    Department of Pharmacology, Medical
School, 
    Medicinal Chemistry, College of Pharmacy, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8af9e2ebe5e7efe4edcaffe7e3e9e2a4efeeff"><span class="__cf_email__" data-cfemail="ea99828b85878f848daa9f87838982c48f8e9f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haibin Zhou</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Longchuan Bai</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renqi Xu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yujun Zhao</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1035-2272" title="Orcid link">http://orcid.org/0000-0003-1035-2272</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianyong Chen</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donna McEachern</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krishnapriya Chinnaswamy</span> - <span class="hlFld-Affiliation affiliation">Life Sciences Institute,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Wen</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Sciences, College of Pharmacy,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lipeng Dai</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Sciences, College of Pharmacy,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Praveen Kumar</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Sciences, College of Pharmacy,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao-Yie Yang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaomin Liu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mi Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liu Liu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer L. Meagher</span> - <span class="hlFld-Affiliation affiliation">Life Sciences Institute,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Han Yi</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, 
    Department of Internal Medicine, 
    ,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duxin Sun</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Sciences, College of Pharmacy,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanne A. Stuckey</span> - <span class="hlFld-Affiliation affiliation">Life Sciences Institute,  and , , , and , University
of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>H.Z., L.B., R.X., Y.Z., and J.C. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The University of Michigan has filed patent applications on SD-36 and its analogues, for which S. Wang, H. Zhou, R. Xu, L. Bai, D. McEachern, C.-Y. Yang, and J. Stuckey are co-inventors. These patents have been licensed by Oncopia Therapeutics Inc., of which S. Wang is a co-founder, a paid consultant, and owns stock. The University of Michigan also owns stock in Oncopia. This study is supported in part by a research contract from Oncopia.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6974-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i82">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10877" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10877" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study is supported in part by a research contract from Oncopia Therapeutics Inc. S.W. and J.S. are grateful for the support of this work from Grant NIH P30 CA046592, the University of Michigan Comprehensive Cancer Center (now the Rogel Cancer Center) Core grant from the National Cancer Institute (NCI), NIH. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor (Grant 085P1000817).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SH2</td><td class="NLM_def"><p class="first last">Src homology 2</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotide</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">PTP</td><td class="NLM_def"><p class="first last">protein tyrosine phosphatase</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">MDM2</td><td class="NLM_def"><p class="first last">human murine double minute 2</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">ALCL</td><td class="NLM_def"><p class="first last">anaplastic large cell lymphoma</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DCE</td><td class="NLM_def"><p class="first last">1,2-dichloroethane</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TMSI</td><td class="NLM_def"><p class="first last">iodotrimethylsilane</p></td></tr><tr><td class="NLM_term">Pd/C</td><td class="NLM_def"><p class="first last">palladium on carbon</p></td></tr><tr><td class="NLM_term">NBS</td><td class="NLM_def"><p class="first last"><i>N</i>-bromosuccinimide</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole hydrate</p></td></tr><tr><td class="NLM_term">BSTFA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>O</i>-bis(trimethylsilyl)trifluoroacetamide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70037" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70037" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 43 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.</span></span> <span> </span><span class="NLM_article-title">STATs: Transcriptional control and biological impact</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1038/nrm909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrm909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=12209125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmslansrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=651-662&issue=9&author=D.+E.+Levyauthor=J.+E.+Darnell&title=STATs%3A+Transcriptional+control+and+biological+impact&doi=10.1038%2Fnrm909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Stats: transcriptional control and biological impact</span></div><div class="casAuthors">Levy, David E.; Darnell, J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">651-662</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Extracellular proteins bound to cell-surface receptors can change nuclear gene expression patterns in minutes, with far-reaching consequences for development, cell growth, and homeostasis.  The signal transducer and activator of transcription (STAT) proteins are among the most well studied of the latent cytoplasmic signal-dependent transcription factor pathways.  In addn. to several roles in normal cell decisions, dysregulation of STAT function contributes to human disease, making the study of these proteins an important topic of current research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGqYF32j262LVg90H21EOLACvtfcHk0lihOL1foCVTQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmslansrw%253D&md5=4a04359c59e54b12d8a67efb6ef42e4c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm909%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DD.%2BE.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DSTATs%253A%2520Transcriptional%2520control%2520and%2520biological%2520impact%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2002%26volume%3D3%26issue%3D9%26spage%3D651%26epage%3D662%26doi%3D10.1038%2Fnrm909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">The stats of cancer - new molecular targets come of age</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/nrc1275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrc1275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=14964307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=97-105&issue=2&author=H.+Yuauthor=R.+Jove&title=The+stats+of+cancer+-+new+molecular+targets+come+of+age&doi=10.1038%2Fnrc1275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The STATs of cancer - new molecular targets come of age</span></div><div class="casAuthors">Yu, Hua; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-105</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Tumor cells acquire the ability to proliferate uncontrollably, resist apoptosis, sustain angiogenesis and evade immune surveillance.  STAT proteins - esp. STAT3 and STAT5 - regulate all of these processes and are persistently activated in a surprisingly large no. of human cancers.  Consequently, STAT proteins are emerging - unexpectedly - as ideal targets for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdnLoVoH4d1rVg90H21EOLACvtfcHk0lihOL1foCVTQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D&md5=3e342db4856310cac2465240e36d3226</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc1275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1275%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DThe%2520stats%2520of%2520cancer%2520-%2520new%2520molecular%2520targets%2520come%2520of%2520age%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26issue%3D2%26spage%3D97%26epage%3D105%26doi%3D10.1038%2Fnrc1275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandis, J. R.</span></span> <span> </span><span class="NLM_article-title">Targeting the IL-6/JAK/STAT3 signalling axis in cancer</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">248</span>, <span class="refDoi"> DOI: 10.1038/nrclinonc.2018.8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrclinonc.2018.8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29405201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12nsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=234-248&issue=4&author=D.+E.+Johnsonauthor=R.+A.+O%E2%80%99Keefeauthor=J.+R.+Grandis&title=Targeting+the+IL-6%2FJAK%2FSTAT3+signalling+axis+in+cancer&doi=10.1038%2Fnrclinonc.2018.8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the IL-6/JAK/STAT3 signalling axis in cancer</span></div><div class="casAuthors">Johnson, Daniel E.; O'Keefe, Rachel A.; Grandis, Jennifer R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">234-248</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The IL-6/JAK/STAT3 pathway is aberrantly hyperactivated in many types of cancer, and such hyperactivation is generally assocd. with a poor clin. prognosis.  In the tumor microenvironment, IL-6/JAK/STAT3 signalling acts to drive the proliferation, survival, invasiveness, and metastasis of tumor cells, while strongly suppressing the antitumor immune response.  Thus, treatments that target the IL-6/JAK/STAT3 pathway in patients with cancer are poised to provide therapeutic benefit by directly inhibiting tumor cell growth and by stimulating antitumor immunity.  Agents targeting IL-6, the IL-6 receptor, or JAKs have already received FDA approval for the treatment of inflammatory conditions or myeloproliferative neoplasms and for the management of certain adverse effects of chimeric antigen receptor T cells, and are being further evaluated in patients with haematopoietic malignancies and in those with solid tumors.  Novel inhibitors of the IL-6/JAK/STAT3 pathway, including STAT3-selective inhibitors, are currently in development.  Herein, we review the role of IL-6/JAK/STAT3 signalling in the tumor microenvironment and the status of preclin. and clin. investigations of agents targeting this pathway.  We also discuss the potential of combining IL-6/JAK/STAT3 inhibitors with currently approved therapeutic agents directed against immune-checkpoint inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocclyreQ67DbVg90H21EOLACvtfcHk0lihOL1foCVTQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12nsbc%253D&md5=91390fe680e391d628b53fdda5324a1b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2018.8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2018.8%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BE.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DR.%2BA.%26aulast%3DGrandis%26aufirst%3DJ.%2BR.%26atitle%3DTargeting%2520the%2520IL-6%252FJAK%252FSTAT3%2520signalling%2520axis%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D15%26issue%3D4%26spage%3D234%26epage%3D248%26doi%3D10.1038%2Fnrclinonc.2018.8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. T.</span></span> <span> </span><span class="NLM_article-title">Two decades of research in discovery of anticancer drugs targeting STAT3, how close are we?</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2018.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.pharmthera.2018.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29933035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2018&pages=74-91&author=J.+D.+Beebeauthor=J.+Y.+Liuauthor=J.+T.+Zhang&title=Two+decades+of+research+in+discovery+of+anticancer+drugs+targeting+STAT3%2C+how+close+are+we%3F&doi=10.1016%2Fj.pharmthera.2018.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Two decades of research in discovery of anticancer drugs targeting STAT3, how close are we?</span></div><div class="casAuthors">Beebe, Jenny D.; Liu, Jing-Yuan; Zhang, Jian-Ting</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">74-91</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) controls many biol. processes including differentiation, survival, proliferation, and angiogenesis.  In normal healthy cells, STAT3 is tightly regulated to maintain a momentary active state.  However, aberrant or constitutively activated STAT3 has been obsd. in many different cancers and constitutively activated STAT3 has been shown to assoc. with poor prognosis and tumor progression.  For this reason, STAT3 has been studied as a possible target in the treatment of many different types of cancers.  However, despite decades of research, a FDA-approved STAT3 inhibitor has yet to emerge.  In this review, we will analyze past studies targeting STAT3 for drug discovery, understand possible causes of failure in these studies, and provide potential insights for future efforts to overcome these roadblocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSWDPJOIaRKrVg90H21EOLACvtfcHk0lihOL1foCVTQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO&md5=048e7d95608bdab18335c55d084d62ab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2018.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2018.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DBeebe%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%2BY.%26aulast%3DZhang%26aufirst%3DJ.%2BT.%26atitle%3DTwo%2520decades%2520of%2520research%2520in%2520discovery%2520of%2520anticancer%2520drugs%2520targeting%2520STAT3%252C%2520how%2520close%2520are%2520we%253F%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D191%26spage%3D74%26epage%3D91%26doi%3D10.1016%2Fj.pharmthera.2018.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, G. R.</span></span> <span> </span><span class="NLM_article-title">Roles of unphosphorylated STATs in signaling</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">443</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1038/cr.2008.41</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fcr.2008.41" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=18364677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFCks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=443-451&issue=4&author=J.+Yangauthor=G.+R.+Stark&title=Roles+of+unphosphorylated+STATs+in+signaling&doi=10.1038%2Fcr.2008.41"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of unphosphorylated STATs in signaling</span></div><div class="casAuthors">Yang, Jinbo; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">443-451</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The seven members of the signal transducer and activator of transcription (STAT) family of transcription factors are activated in response to many different cytokines and growth factors by phosphorylation of specific tyrosine residues.  The STAT1 and STAT3 genes are specific targets of activated STATs 1 and 3, resp., resulting in large increases in the levels of these unphosphorylated STATs (U-STATs) in response to the interferons (STAT1) or ligands that active gp130, such as IL-6 (STAT3).  U-STATs drive gene expression by novel mechanisms distinct from those used by phosphorylated STAT (P-STAT) dimers.  In this review, we discuss the roles of U-STATs in transcription and regulation of gene expression.  Cell Research (2008) 18:443-451. doi: 10.1038/cr.2008.41: published online 25 March 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqynLvLsNqMOrVg90H21EOLACvtfcHk0lhI2FhDWCbGrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFCks74%253D&md5=ce421809dffce50716f7ebec75d2ad80</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcr.2008.41&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2008.41%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DRoles%2520of%2520unphosphorylated%2520STATs%2520in%2520signaling%26jtitle%3DCell%2520Res.%26date%3D2008%26volume%3D18%26issue%3D4%26spage%3D443%26epage%3D451%26doi%3D10.1038%2Fcr.2008.41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzrock, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woessner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemunaitis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fayad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piha-Paul, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadella, M. V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohseni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakey, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLeod, A. R.</span></span> <span> </span><span class="NLM_article-title">AZD9150, a next-generation antisense oligonucleotide inhibitor of STAT3 with early evidence of clinical activity in lymphoma and lung cancer</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">314</span>),  <span class="NLM_fpage">314ra185</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aac5272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1126%2Fscitranslmed.aac5272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=26582900" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=314ra185&issue=314&author=D.+Hongauthor=R.+Kurzrockauthor=Y.+Kimauthor=R.+Woessnerauthor=A.+Younesauthor=J.+Nemunaitisauthor=N.+Fowlerauthor=T.+Y.+Zhouauthor=J.+Schmidtauthor=M.+J.+Joauthor=S.+J.+Leeauthor=M.+Yamashitaauthor=S.+G.+Hughesauthor=L.+Fayadauthor=S.+Piha-Paulauthor=M.+V.+P.+Nadellaauthor=M.+Mohseniauthor=D.+Lawsonauthor=C.+Reimerauthor=D.+C.+Blakeyauthor=X.+K.+Xiaoauthor=J.+Hsuauthor=A.+Revenkoauthor=B.+P.+Moniaauthor=A.+R.+MacLeod&title=AZD9150%2C+a+next-generation+antisense+oligonucleotide+inhibitor+of+STAT3+with+early+evidence+of+clinical+activity+in+lymphoma+and+lung+cancer&doi=10.1126%2Fscitranslmed.aac5272"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aac5272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aac5272%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DD.%26aulast%3DKurzrock%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DWoessner%26aufirst%3DR.%26aulast%3DYounes%26aufirst%3DA.%26aulast%3DNemunaitis%26aufirst%3DJ.%26aulast%3DFowler%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DT.%2BY.%26aulast%3DSchmidt%26aufirst%3DJ.%26aulast%3DJo%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DYamashita%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DS.%2BG.%26aulast%3DFayad%26aufirst%3DL.%26aulast%3DPiha-Paul%26aufirst%3DS.%26aulast%3DNadella%26aufirst%3DM.%2BV.%2BP.%26aulast%3DMohseni%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DBlakey%26aufirst%3DD.%2BC.%26aulast%3DXiao%26aufirst%3DX.%2BK.%26aulast%3DHsu%26aufirst%3DJ.%26aulast%3DRevenko%26aufirst%3DA.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DMacLeod%26aufirst%3DA.%2BR.%26atitle%3DAZD9150%252C%2520a%2520next-generation%2520antisense%2520oligonucleotide%2520inhibitor%2520of%2520STAT3%2520with%2520early%2520evidence%2520of%2520clinical%2520activity%2520in%2520lymphoma%2520and%2520lung%2520cancer%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26issue%3D314%26spage%3D314ra185%26doi%3D10.1126%2Fscitranslmed.aac5272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. J. A.</span></span> <span> </span><span class="NLM_article-title">Antisense oligonucleotides: the next frontier for treatment of neurological disorders</span>. <i>Nat. Rev. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1038/nrneurol.2017.148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrneurol.2017.148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29192260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCrs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=9-21&issue=1&author=C.+Rinaldiauthor=M.+J.+A.+Wood&title=Antisense+oligonucleotides%3A+the+next+frontier+for+treatment+of+neurological+disorders&doi=10.1038%2Fnrneurol.2017.148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Antisense oligonucleotides: the next frontier for treatment of neurological disorders</span></div><div class="casAuthors">Rinaldi, Carlo; Wood, Matthew J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-21</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Antisense oligonucleotides (ASOs) were first discovered to influence RNA processing and modulate protein expression over two decades ago; however, progress translating these agents into the clinic has been hampered by inadequate target engagement, insufficient biol. activity, and off-target toxic effects.  Over the years, novel chem. modifications of ASOs have been employed to address these issues.  These modifications, in combination with elucidation of the mechanism of action of ASOs and improved clin. trial design, have provided momentum for the translation of ASO-based strategies into therapies.  Many neurol. conditions lack an effective treatment; however, as research progressively disentangles the pathogenic mechanisms of these diseases, they provide an ideal platform to test ASO-based strategies.  This steady progress reached a pinnacle in the past few years with approvals of ASOs for the treatment of spinal muscular atrophy and Duchenne muscular dystrophy, which represent landmarks in a field in which disease-modifying therapies were virtually non-existent.  With the rapid development of improved next-generation ASOs toward clin. application, this technol. now holds the potential to have a dramatic effect on the treatment of many neurol. conditions in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlk6GFxML247Vg90H21EOLACvtfcHk0ljVpwTBV4XGow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCrs7jL&md5=a9256a0257a13c8c331e1229aecfd4aa</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2017.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2017.148%26sid%3Dliteratum%253Aachs%26aulast%3DRinaldi%26aufirst%3DC.%26aulast%3DWood%26aufirst%3DM.%2BJ.%2BA.%26atitle%3DAntisense%2520oligonucleotides%253A%2520the%2520next%2520frontier%2520for%2520treatment%2520of%2520neurological%2520disorders%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2018%26volume%3D14%26issue%3D1%26spage%3D9%26epage%3D21%26doi%3D10.1038%2Fnrneurol.2017.148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&issue=6&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljVpwTBV4XGow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26issue%3D6%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&issue=2&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljVpwTBV4XGow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26issue%3D2%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: elements of PROTAC design</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.cbpa.2019.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=31004963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=111-119&author=S.+L.+Paivaauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+elements+of+PROTAC+design&doi=10.1016%2Fj.cbpa.2019.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: elements of PROTAC design</span></div><div class="casAuthors">Paiva, Stacey-Lynn; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn. using Proteolysis Targeting Chimeras (PROTACs) has emerged as a novel therapeutic modality in drug discovery.  PROTACs mediate the degrdn. of select proteins of interest (POIs) by hijacking the activity of E3 ubiquitin ligases for POI ubiquitination and subsequent degrdn. by the 26S proteasome.  This hijacking mechanism has been used to degrade various types of disease-relevant POIs.  In this review, we aim to highlight the recent advances in targeted protein degrdn. and describe the challenges that need to be addressed in order to efficiently develop potent PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol6Nk0VkUdzLVg90H21EOLACvtfcHk0lgX3bqoG7-5WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D&md5=5783311db47da378180c14d67526a398</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DPaiva%26aufirst%3DS.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520elements%2520of%2520PROTAC%2520design%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D111%26epage%3D119%26doi%3D10.1016%2Fj.cbpa.2019.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation by small molecules</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">123</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1146%2Fannurev-pharmtox-010715-103507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=27732798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=107-123&author=D.+P.+Bondesonauthor=C.+M.+Crews&title=Targeted+protein+degradation+by+small+molecules&doi=10.1146%2Fannurev-pharmtox-010715-103507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation by Small Molecules</span></div><div class="casAuthors">Bondeson, Daniel P.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107-123</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Protein homeostasis networks are highly regulated systems responsible for maintaining the health and productivity of cells.  Whereas therapeutics have been developed to disrupt protein homeostasis, more recently identified techniques have been used to repurpose homeostatic networks to effect degrdn. of disease-relevant proteins.  Here, we review recent advances in the use of small mols. to degrade proteins in a selective manner.  First, we highlight all-small-mol. techniques with direct clin. application.  Second, we describe techniques that may find broader acceptance in the biomedical research community that require little or no synthetic chem.  In addn. to serving as innovative research tools, these new approaches to control intracellular protein levels offer the potential to develop novel therapeutics targeting proteins that are not currently pharmaceutically vulnerable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6k9y8vt0uPbVg90H21EOLACvtfcHk0lgX3bqoG7-5WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt73L&md5=a9d7b9fc821e6dca7fbe1615187a2e87</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103507%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%2520by%2520small%2520molecules%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2017%26volume%3D57%26spage%3D107%26epage%3D123%26doi%3D10.1146%2Fannurev-pharmtox-010715-103507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of conformationally constrained, cell-permeable STAT3 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1021/ml100010j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100010j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=85-89&issue=2&author=J.+Chenauthor=L.+Baiauthor=D.+Bernardauthor=Z.+Nikolovska-Coleskaauthor=C.+Gomezauthor=J.+Zhangauthor=H.+Yiauthor=S.+Wang&title=Structure-based+design+of+conformationally+constrained%2C+cell-permeable+STAT3+inhibitors&doi=10.1021%2Fml100010j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Conformationally Constrained, Cell-Permeable STAT3 Inhibitors</span></div><div class="casAuthors">Chen, Jianyong; Bai, Longchuan; Bernard, Denzil; Nikolovska-Coleska, Zaneta; Gomez, Cindy; Zhang, Jian; Yi, Han; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-89</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the structure-based design of a class of conformationally constrained, potent, cell-permeable small-mol. inhibitors to target the SH2 domain in STAT3.  Compd. 11 (CJ-1383) binds to STAT3 with a Ki value of 0.95 μM, dose-dependently inhibits cellular STAT3 signaling and cancer cell growth, and induces apoptosis in the MDA-MB-468 cancer cell line with constitutively activated STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMrK-vDy7ISbVg90H21EOLACvtfcHk0liNYV3fiSk9Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D&md5=87a0b1a823441940f75fc2af960e0359</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fml100010j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100010j%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DGomez%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520conformationally%2520constrained%252C%2520cell-permeable%2520STAT3%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26issue%3D2%26spage%3D85%26epage%3D89%26doi%3D10.1021%2Fml100010j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burke, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span> <span> </span><span class="NLM_article-title">Phosphoryltyrosyl mimetics in the design of peptide-based signal transduction inhibitors</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1002/1097-0282(2001)60:1<32::AID-BIP1002>3.0.CO;2-I</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=11376431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Cqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2001&pages=32-44&issue=1&author=T.+R.+Burkeauthor=Z.+J.+Yaoauthor=D.+G.+Liuauthor=J.+Voigtauthor=Y.+Gao&title=Phosphoryltyrosyl+mimetics+in+the+design+of+peptide-based+signal+transduction+inhibitors&doi=10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoryltyrosyl mimetics in the design of peptide-based signal transduction inhibitors</span></div><div class="casAuthors">Burke, Terrence R., Jr.; Yao, Zhu-Jun; Liu, Ding-Guo; Voigt, Johannes; Gao, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-44</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review, with 83 refs.  The central roles played by protein-tyrosine kinase (PTK)-dependent signal transduction in normal cellular regulation and homeostasis have made inappropriate or aberrant functions of certain of these pathways contributing factors to a variety of diseases, including several cancers.  For this reason, development of PTK signaling inhibitors has evolved into an important approach toward new therapeutics.  Since in these pathways phosphotyrosyl (pTyr) residues provide unique and defining functions either by (1) their creation under the catalysis of PTKs, (2) their recognition and binding by protein modules such as SH2 and phosphotyrosyl binding (PTB) domains, or (3) their destruction by protein-tyrosine phosphatases (PTPs), pTyr mimetics provide useful general starting points for inhibitor design.  Important considerations in the development of such pTyr mimetics include enzymic stability (particularly toward PTPs), high affinity recognition by target pTyr binding proteins, and good cellular bioavailability.  Although small mol. nonpeptide inhibitors may represent the ultimate objectives of inhibitor development, peptides frequently serve as display platforms for pTyr mimetics, which afford useful and conceptually straightforward starting points in the development process.  Reported herein is a limited overview of pTyr mimetic development as it relates to peptide-based agents.  Of particular interest are recent findings that highlight potential limitations of peptides as display platforms for the identification of small mol. leads.  One conclusion that results from this work is that while peptide-based approaches toward small mol. inhibitor design are often intellectually satisfying from a structure-based perspective, extrapolation of neg. findings to small mol., nonpeptide contexts should be undertaken with extreme caution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcIBlemRR8r7Vg90H21EOLACvtfcHk0liNYV3fiSk9Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Cqsbk%253D&md5=76bb70bd5e8cab68f76f8d9edcc67fbd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1097-0282%25282001%252960%253A1%253C32%253A%253AAID-BIP1002%253E3.0.CO%253B2-I%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DYao%26aufirst%3DZ.%2BJ.%26aulast%3DLiu%26aufirst%3DD.%2BG.%26aulast%3DVoigt%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DY.%26atitle%3DPhosphoryltyrosyl%2520mimetics%2520in%2520the%2520design%2520of%2520peptide-based%2520signal%2520transduction%2520inhibitors%26jtitle%3DBiopolymers%26date%3D2001%26volume%3D60%26issue%3D1%26spage%3D32%26epage%3D44%26doi%3D10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burke, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Phosphotyrosyl mimetics in the development of signal transduction inhibitors</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">426</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1021/ar020127o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar020127o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslCmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2003&pages=426-433&issue=6&author=T.+R.+Burkeauthor=K.+Lee&title=Phosphotyrosyl+mimetics+in+the+development+of+signal+transduction+inhibitors&doi=10.1021%2Far020127o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphotyrosyl Mimetics in the Development of Signal Transduction Inhibitors</span></div><div class="casAuthors">Burke, Terrence R., Jr.; Lee, Kyeong</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">426-433</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Phosphotyrosyl (pTyr) residues play important roles in cellular signal transduction by facilitating recognition and binding necessary for crit. protein-protein interactions, and for this reason pTyr motifs represent attractive starting points in the development of signaling antagonists.  Although the pTyr phosphoryl moiety is central in these phenomena, its incorporation into signaling inhibitors is contraindicated due to enzymic lability and limited bioavailability assocd. with phosphate esters.  To address these limitations, an entire field of study has arisen devoted to the design and utilization of pTyr mimetics.  This Account provides a perspective on the roles of pTyr residues in signal transduction and approaches to pTyr mimetic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqecqbwygKL7bVg90H21EOLACvtfcHk0lhRayud2__2Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslCmtLs%253D&md5=84c9610cea90bfcb3ce91a0ede387e77</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Far020127o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far020127o%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPhosphotyrosyl%2520mimetics%2520in%2520the%2520development%2520of%2520signal%2520transduction%2520inhibitors%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2003%26volume%3D36%26issue%3D6%26spage%3D426%26epage%3D433%26doi%3D10.1021%2Far020127o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of stat3</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3394</span>– <span class="NLM_lpage">3397</span>, <span class="refDoi"> DOI: 10.1021/ol9012662</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol9012662" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=3394-3397&issue=15&author=P.+K.+Mandalauthor=W.+S.+L.+Liaoauthor=J.+S.+McMurray&title=Synthesis+of+phosphatase-stable%2C+cell-permeable+peptidomimetic+prodrugs+that+target+the+SH2+domain+of+stat3&doi=10.1021%2Fol9012662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3</span></div><div class="casAuthors">Mandal, Pijus K.; Liao, Warren S.-L.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3394-3397</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of prodrugs targeted to the SH2 domain of Stat3 is reported.  Using a convergent strategy, the pivaloyloxymethyl phosphonodiester of pentachlorophenyl 4-phosphonodifluoromethylcinnamate, a phosphotyrosine surrogate, was synthesized and used to acylate peptidomimetic fragments that were prepd. on solid supports.  Two prodrugs described here inhibited the phosphorylation of Stat3 in breast tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqokZ-epNTxprVg90H21EOLACvtfcHk0liyzsdmxiz1QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D&md5=c935049f107f5ab89bf38cd445c845f5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fol9012662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol9012662%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLiao%26aufirst%3DW.%2BS.%2BL.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520phosphatase-stable%252C%2520cell-permeable%2520peptidomimetic%2520prodrugs%2520that%2520target%2520the%2520SH2%2520domain%2520of%2520stat3%26jtitle%3DOrg.%2520Lett.%26date%3D2009%26volume%3D11%26issue%3D15%26spage%3D3394%26epage%3D3397%26doi%3D10.1021%2Fol9012662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limbrick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtwistle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Conformationally constrained peptidomimetic inhibitors of signal transducer and activator of transcription 3: evaluation and molecular modeling</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2429</span>– <span class="NLM_lpage">2442</span>, <span class="refDoi"> DOI: 10.1021/jm801491w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801491w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFKrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2429-2442&issue=8&author=P.+K.+Mandalauthor=D.+Limbrickauthor=D.+R.+Colemanauthor=G.+A.+Dyerauthor=Z.+Y.+Renauthor=J.+S.+Birtwistleauthor=C.+Y.+Xiongauthor=X.+M.+Chenauthor=J.+M.+Briggsauthor=J.+S.+McMurray&title=Conformationally+constrained+peptidomimetic+inhibitors+of+signal+transducer+and+activator+of+transcription+3%3A+evaluation+and+molecular+modeling&doi=10.1021%2Fjm801491w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conformationally Constrained Peptidomimetic Inhibitors of Signal Transducer and Activator of Transcription 3: Evaluation and Molecular Modeling</span></div><div class="casAuthors">Mandal, Pijus K.; Limbrick, Donald; Coleman, David R.; Dyer, Garrett A.; Ren, Zhiyong; Birtwistle, J. Sanderson; Xiong, Chiyi; Chen, Xiaomin; Briggs, James M.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2429-2442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (Stat3) is involved in aberrant growth and survival signals in malignant tumor cells and is a validated target for anticancer drug design.  We are targeting its SH2 domain to prevent docking to cytokine and growth factor receptors and subsequent signaling.  The amino acids of our lead phosphopeptide, Ac-pTyr-Leu-Pro-Gln-Thr-Val-NH2, were replaced with conformationally constrained mimics.  Structure-affinity studies led to the peptidomimetic, pCinn-Haic-Gln-NHBn, which had an IC50 of 162 nM (fluorescence polarization), compared to 290 nM for the lead phosphopeptide (pCinn = 4-phosphoryloxycinnamate, Haic = (2S,5S)-5-amino-1,2,4,5,6,7-hexahydro-4-oxo-azepino[3,2,1-hi]indole-2-carboxylic acid).  PCinn-Haic-Gln-OH was docked to the SH2 domain (AUTODOCK), and the two highest populated clusters were subjected to mol. dynamics simulations.  Both converged to a common peptide conformation.  The complex exhibits unique hydrogen bonding between Haic and Gln and Stat3 as well as hydrophobic interactions between the protein and pCinn and Haic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorsyN8dGgwIrVg90H21EOLACvtfcHk0liyzsdmxiz1QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFKrsL8%253D&md5=e1e59cdd54551d4d167394034ecdb7ee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm801491w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801491w%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLimbrick%26aufirst%3DD.%26aulast%3DColeman%26aufirst%3DD.%2BR.%26aulast%3DDyer%26aufirst%3DG.%2BA.%26aulast%3DRen%26aufirst%3DZ.%2BY.%26aulast%3DBirtwistle%26aufirst%3DJ.%2BS.%26aulast%3DXiong%26aufirst%3DC.%2BY.%26aulast%3DChen%26aufirst%3DX.%2BM.%26aulast%3DBriggs%26aufirst%3DJ.%2BM.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DConformationally%2520constrained%2520peptidomimetic%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%253A%2520evaluation%2520and%2520molecular%2520modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D8%26spage%3D2429%26epage%3D2442%26doi%3D10.1021%2Fjm801491w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtwistle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaluarachchi, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Potent and selective phosphopeptide mimetic prodrugs targeted to the src homology 2 (SH2) domain of signal transducer and activator of transcription 3</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3549</span>– <span class="NLM_lpage">3563</span>, <span class="refDoi"> DOI: 10.1021/jm2000882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2000882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlt1Cnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3549-3563&issue=10&author=P.+K.+Mandalauthor=F.+Q.+Gaoauthor=Z.+Luauthor=Z.+Y.+Renauthor=R.+Rameshauthor=J.+S.+Birtwistleauthor=K.+K.+Kaluarachchiauthor=X.+M.+Chenauthor=R.+C.+Bastauthor=W.+S.+Liaoauthor=J.+S.+McMurray&title=Potent+and+selective+phosphopeptide+mimetic+prodrugs+targeted+to+the+src+homology+2+%28SH2%29+domain+of+signal+transducer+and+activator+of+transcription+3&doi=10.1021%2Fjm2000882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Phosphopeptide Mimetic Prodrugs Targeted to the Src Homology 2 (SH2) Domain of Signal Transducer and Activator of Transcription 3</span></div><div class="casAuthors">Mandal, Pijus K.; Gao, Fengqin; Lu, Zhen; Ren, Zhiyong; Ramesh, Rajagopal; Birtwistle, J. Sanderson; Kaluarachchi, Kumaralal K.; Chen, Xiaomin; Bast, Robert C.; Liao, Warren S.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3549-3563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (Stat3), a target for anticancer drug design, is activated by recruitment to phosphotyrosine residues on growth factor and cytokine receptors via its SH2 domain.  We report here structure-activity relation studies on phosphopeptide mimics targeted to the SH2 domain of Stat3.  Inclusion of a Me group on the β-position of the pTyr mimic 4-phosphocinnamide enhanced affinity 2- to 3-fold.  Bis-pivaloyloxymethyl prodrugs contg. β-methylcinnamide, dipeptide scaffolds Haic and Nle-cis-3,4-methanoproline, and glutamine surrogates were highly potent, completely inhibiting phosphorylation of Stat3 Tyr705 at 0.5-1 μM in a variety of cancer cell lines.  The inhibitors were selective for Stat3 over Stat1, Stat5, Src, and p85 of PI3K, indicating ability to discriminate individual SH2 domains in intact cells.  At concns. that completely inhibited Stat3 phosphorylation, the prodrugs were not cytotoxic to a panel of tumor cells, thereby showing clear distinction between cytotoxicity and effects downstream of activated Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqVZisU6d4iLVg90H21EOLACvtfcHk0liwspRw9MDbSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlt1Cnsbk%253D&md5=f0a68b4edcbcfc621cb50ab29af3bd45</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm2000882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2000882%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DGao%26aufirst%3DF.%2BQ.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DRen%26aufirst%3DZ.%2BY.%26aulast%3DRamesh%26aufirst%3DR.%26aulast%3DBirtwistle%26aufirst%3DJ.%2BS.%26aulast%3DKaluarachchi%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DX.%2BM.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DLiao%26aufirst%3DW.%2BS.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DPotent%2520and%2520selective%2520phosphopeptide%2520mimetic%2520prodrugs%2520targeted%2520to%2520the%2520src%2520homology%25202%2520%2528SH2%2529%2520domain%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D10%26spage%3D3549%26epage%3D3563%26doi%3D10.1021%2Fjm2000882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the von hippel-lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1 alpha interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&issue=10&author=D.+L.+Buckleyauthor=I.+Van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+von+hippel-lindau+E3+ubiquitin+ligase+using+small+molecules+to+disrupt+the+VHL%2FHIF-1+alpha+interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0liwspRw9MDbSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520von%2520hippel-lindau%2520E3%2520ubiquitin%2520ligase%2520using%2520small%2520molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1%2520alpha%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26issue%3D10%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein protein interaction between the von hippel- lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&issue=20&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein+protein+interaction+between+the+von+hippel-+lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lh-RDJXkWnIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein%2520protein%2520interaction%2520between%2520the%2520von%2520hippel-%2520lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D20%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-based optimization of potent and cell-active inhibitors of the von hippel-lindau (VHL) E3 ubiquitin ligase: structure-activity relationships leading to the chemical probe (2S,4R)-1-((S)-2-(1-cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyI)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyppyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=599-618&issue=2&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-based+optimization+of+potent+and+cell-active+inhibitors+of+the+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase%3A+structure-activity+relationships+leading+to+the+chemical+probe+%282S%2C4R%29-1-%28%28S%29-2-%281-cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyI%29-4-hydroxy-N-%284-%284-methylthiazol-5-yl%29benzyppyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase: Structure-Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span></div><div class="casAuthors">Soares, Pedro; Gadd, Morgan S.; Frost, Julianty; Galdeano, Carles; Ellis, Lucy; Epemolu, Ola; Rocha, Sonia; Read, Kevin D.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">599-618</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau tumor suppressor protein is the substrate binding subunit of the VHL E3 ubiquitin ligase, which targets hydroxylated α subunit of hypoxia inducible factors (HIFs) for ubiquitination and subsequent proteasomal degrdn.  VHL is a potential target for treating anemia and ischemic diseases, motivating the development of inhibitors of the VHL:HIF-α protein-protein interaction.  Addnl., bifunctional proteolysis targeting chimeras (PROTACs) contg. a VHL ligand can hijack the E3 ligase activity to induce degrdn. of target proteins.  We report the structure-guided design and group-based optimization of a series of VHL inhibitors with low nanomolar potencies and improved cellular permeability.  Structure-activity relationships led to the discovery of potent inhibitors 10 and chem. probe VH298, with dissocn. consts. <100 nM, which induced marked HIF-1α intracellular stabilization.  Our study provides new chem. tools to probe the VHL-HIF pathways and new VHL ligands for next-generation PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQfJe2l2WgX7Vg90H21EOLACvtfcHk0lh-RDJXkWnIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P&md5=1c1e7c976c2de9a9681773ff216fdfd7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-based%2520optimization%2520of%2520potent%2520and%2520cell-active%2520inhibitors%2520of%2520the%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%253A%2520structure-activity%2520relationships%2520leading%2520to%2520the%2520chemical%2520probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyI%2529-4-hydroxy-N-%25284-%25284-methylthiazol-5-yl%2529benzyppyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D2%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Surface probing by fragment-based screening and computational methods identifies ligandable pockets on the von hippel-lindau (VHL) E3 ubiquitin ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">7387</span>– <span class="NLM_lpage">7393</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00842</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00842" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyktLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7387-7393&issue=16&author=X.+Lucasauthor=I.+Van+Molleauthor=A.+Ciulli&title=Surface+probing+by+fragment-based+screening+and+computational+methods+identifies+ligandable+pockets+on+the+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase&doi=10.1021%2Facs.jmedchem.8b00842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Probing by Fragment-Based Screening and Computational Methods Identifies Ligandable Pockets on the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Lucas, Xavier; Van Molle, Inge; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7387-7393</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Beyond the targeting of E3 ubiquitin ligases to inhibit protein homeostasis, E3 ligase binders can be repurposed as targeted protein degraders (PROTACs or mol. glues).  The authors sought to identify new binders of the VHL E3 ligase by biophys. fragment-based screening followed by x-ray crystallog. soaking.  The authors identified fragments binding at the Elongin C:Cullin 2 interface and a new cryptic pocket in VHL, along with other potential ligandable sites predicted computationally and found to bind solvent mols. in crystal structures.  The elucidated interactions provide starting points for future ligand development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZdhy5IVXNgrVg90H21EOLACvtfcHk0ljfmVPYHDsHSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyktLbE&md5=c081e7cf26807f59e646a60a0ec74361</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00842%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSurface%2520probing%2520by%2520fragment-based%2520screening%2520and%2520computational%2520methods%2520identifies%2520ligandable%2520pockets%2520on%2520the%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D16%26spage%3D7387%26epage%3D7393%26doi%3D10.1021%2Facs.jmedchem.8b00842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BET proteins in triple-negative breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2476</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1158%2F0008-5472.CAN-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476-2487&issue=9&author=L.+C.+Baiauthor=B.+Zhouauthor=C.+Y.+Yangauthor=J.+Jiauthor=D.+McEachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+T.+Huauthor=F.+M.+Xuauthor=Y.+J.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+L.+Douauthor=B.+Wenauthor=D.+X.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Q.+Liauthor=M.+J.+Ellisauthor=S.+M.+Wang&title=Targeted+degradation+of+BET+proteins+in+triple-negative+breast+cancer&doi=10.1158%2F0008-5472.CAN-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0ljfmVPYHDsHSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%2BC.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%2BT.%26aulast%3DXu%26aufirst%3DF.%2BM.%26aulast%3DZhao%26aufirst%3DY.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%2BL.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%2BX.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%2BQ.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DTargeted%2520degradation%2520of%2520BET%2520proteins%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26issue%3D9%26spage%3D2476%26epage%3D2487%26doi%3D10.1158%2F0008-5472.CAN-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+T.+Huauthor=F.+M.+Xuauthor=Z.+Chenauthor=L.+C.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+J.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+X.+Sunauthor=S.+M.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0ljCAr5-IU1wIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%2BT.%26aulast%3DXu%26aufirst%3DF.%2BM.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%2BC.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%2BJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%2BX.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&issue=15&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+C.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+L.+Yangauthor=D.+N.+Ziazadehauthor=F.+M.+Xuauthor=J.+T.+Huauthor=W.+G.+Xiangauthor=L.+Y.+Huangauthor=S.+W.+Liauthor=B.+Wenauthor=D.+X.+Sunauthor=S.+M.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0ljCAr5-IU1wIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%2BC.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%2BL.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%2BM.%26aulast%3DHu%26aufirst%3DJ.%2BT.%26aulast%3DXiang%26aufirst%3DW.%2BG.%26aulast%3DHuang%26aufirst%3DL.%2BY.%26aulast%3DLi%26aufirst%3DS.%2BW.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%2BX.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D15%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera murine double minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&issue=2&author=Y.+B.+Liauthor=J.+L.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.+Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+M.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+murine+double+minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0lhVyFJh2PafPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%2BB.%26aulast%3DYang%26aufirst%3DJ.%2BL.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520murine%2520double%2520minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D2%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent and selective small-molecule degrader of STAT3 achieves complete tumor regression In vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511&issue=5&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+potent+and+selective+small-molecule+degrader+of+STAT3+achieves+complete+tumor+regression+In+vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0lhVyFJh2PafPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520and%2520selective%2520small-molecule%2520degrader%2520of%2520STAT3%2520achieves%2520complete%2520tumor%2520regression%2520In%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26issue%3D5%26spage%3D498%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDBI-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span> (<span class="NLM_issue">7512</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&issue=7512&author=E.+S.+Fischerauthor=K.+Bohmauthor=J.+R.+Lydeardauthor=H.+D.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thoma&title=Structure+of+the+DDBI-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lhVyFJh2PafPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBohm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%2BD.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDBI-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26issue%3D7512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&issue=6&author=J.+Luauthor=Y.+M.+Qianauthor=M.+Altieriauthor=H.+Q.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhF4tuRTGIf5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%2BM.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%2BQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26issue%3D6%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&issue=1&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lhF4tuRTGIf5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a 1,5-diazabicyclo[6,3,0] dodecane amino acid derivative as a novel dipeptide reverse-turn mimetic</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">4769</span>– <span class="NLM_lpage">4770</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2006.04.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.tetlet.2006.04.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsVyktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=4769-4770&issue=27&author=Y.+F.+Pengauthor=H.+Y.+Sunauthor=S.+M.+Wang&title=Design+and+synthesis+of+a+1%2C5-diazabicyclo%5B6%2C3%2C0%5D+dodecane+amino+acid+derivative+as+a+novel+dipeptide+reverse-turn+mimetic&doi=10.1016%2Fj.tetlet.2006.04.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of a 1,5-diazabicyclo[6.3.0]dodecane amino acid derivative as a novel dipeptide reverse-turn mimetic</span></div><div class="casAuthors">Peng, Yuefeng; Sun, Haiying; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">4769-4770</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">N-Boc-1,5-diazabicyclo[6.3.0]dodecane amino acid tert-Bu ester I, designed as a novel dipeptide reverse-turn mimetic, was synthesized in a concise method from the known compd. II in an overall yield of 47%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyiUhd6L-vHLVg90H21EOLACvtfcHk0liJf4_0oyPJoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsVyktrg%253D&md5=1aadc23cec2bd840e08a333a951ed1f1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2006.04.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2006.04.053%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%2BF.%26aulast%3DSun%26aufirst%3DH.%2BY.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%25201%252C5-diazabicyclo%255B6%252C3%252C0%255D%2520dodecane%2520amino%2520acid%2520derivative%2520as%2520a%2520novel%2520dipeptide%2520reverse-turn%2520mimetic%26jtitle%3DTetrahedron%2520Lett.%26date%3D2006%26volume%3D47%26issue%3D27%26spage%3D4769%26epage%3D4770%26doi%3D10.1016%2Fj.tetlet.2006.04.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morlacchi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Link, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurieva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavraki, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corry, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Targeting the src homology 2 (SH2) domain of signal transducer and activator of transcription 6 (STAT6) with cell-permeable, phosphatase-stable phosphopeptide mimics potently inhibits tyr641 phosphorylation and transcriptional activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">8970</span>– <span class="NLM_lpage">8984</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01321</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01321" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegt7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8970-8984&issue=22&author=P.+K.+Mandalauthor=P.+Morlacchiauthor=J.+M.+Knightauthor=T.+M.+Linkauthor=G.+R.+Leeauthor=R.+Nurievaauthor=D.+Singhauthor=A.+Dhanikauthor=L.+Kavrakiauthor=D.+B.+Corryauthor=J.+E.+Ladburyauthor=J.+S.+McMurray&title=Targeting+the+src+homology+2+%28SH2%29+domain+of+signal+transducer+and+activator+of+transcription+6+%28STAT6%29+with+cell-permeable%2C+phosphatase-stable+phosphopeptide+mimics+potently+inhibits+tyr641+phosphorylation+and+transcriptional+activity&doi=10.1021%2Facs.jmedchem.5b01321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Src Homology 2 (SH2) Domain of Signal Transducer and Activator of Transcription 6 (STAT6) with Cell-Permeable, Phosphatase-Stable Phosphopeptide Mimics Potently Inhibits Tyr641 Phosphorylation and Transcriptional Activity</span></div><div class="casAuthors">Mandal, Pijus K.; Morlacchi, Pietro; Knight, J. Morgan; Link, Todd M.; Lee, Gilbert R.; Nurieva, Roza; Singh, Divyendu; Dhanik, Ankur; Kavraki, Lydia; Corry, David B.; Ladbury, John E.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8970-8984</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 6 (STAT6) transmits signals from cytokines IL-4 and IL-13 and is activated in allergic airway disease.  The authors are developing phosphopeptide mimetics targeting the SH2 domain of STAT6 to block recruitment to phosphotyrosine residues on IL-4 or IL-13 receptors and subsequent Tyr 641 phosphorylation to inhibit the expression of genes contributing to asthma.  Structure-affinity relationship studies showed that phosphopeptides based on Tyr 631 from IL-4Rα bind with weak affinity to STAT6, whereas replacing the pY+3 residue with simple aryl and alkyl amides resulted in affinities in the mid to low nM range.  A set of phosphatase-stable, cell-permeable prodrug analogs inhibited cytokine-stimulated STAT6 phosphorylation in both Beas-2B human airway cells and primary mouse T-lymphocytes at concns. as low as 100 nM.  IL-13-stimulated expression of CCL26 (eotaxin-3) was inhibited in a dose-dependent manner, demonstrating that targeting the SH2 domain blocks both phosphorylation and transcriptional activity of STAT6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquqyZb9Mhgg7Vg90H21EOLACvtfcHk0liJf4_0oyPJoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegt7zE&md5=699ec8930ac7325966a21b937d4edab9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01321%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DMorlacchi%26aufirst%3DP.%26aulast%3DKnight%26aufirst%3DJ.%2BM.%26aulast%3DLink%26aufirst%3DT.%2BM.%26aulast%3DLee%26aufirst%3DG.%2BR.%26aulast%3DNurieva%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DD.%26aulast%3DDhanik%26aufirst%3DA.%26aulast%3DKavraki%26aufirst%3DL.%26aulast%3DCorry%26aufirst%3DD.%2BB.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DTargeting%2520the%2520src%2520homology%25202%2520%2528SH2%2529%2520domain%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25206%2520%2528STAT6%2529%2520with%2520cell-permeable%252C%2520phosphatase-stable%2520phosphopeptide%2520mimics%2520potently%2520inhibits%2520tyr641%2520phosphorylation%2520and%2520transcriptional%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D22%26spage%3D8970%26epage%3D8984%26doi%3D10.1021%2Facs.jmedchem.5b01321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span>; <span class="NLM_string-name">Mandal, P. K.</span>; <span class="NLM_string-name">Liao, W. S.</span>; <span class="NLM_string-name">Ren, Z.</span>; <span class="NLM_string-name">Chen, X.</span>; <span class="NLM_string-name">Rajaopal, R.</span>; <span class="NLM_string-name">Robertson, F.</span></span> <span> </span><span class="NLM_article-title">Preparation of Phosphopeptide Inhibitors of STAT3</span>. <span class="NLM_patent">WO 2010118309</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=J.+S.+McMurray&author=P.+K.+Mandal&author=W.+S.+Liao&author=Z.+Ren&author=X.+Chen&author=R.+Rajaopal&author=F.+Robertson&title=Preparation+of+Phosphopeptide+Inhibitors+of+STAT3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DPreparation%2520of%2520Phosphopeptide%2520Inhibitors%2520of%2520STAT3%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cer, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudunuri, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeda, F. J.</span></span> <span> </span><span class="NLM_article-title">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">Web Server</span>),  <span class="NLM_fpage">W441</span>– <span class="NLM_lpage">W445</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1093%2Fnar%2Fgkp253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=19395593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=W441-W445&issue=Web+Server&author=R.+Z.+Cerauthor=U.+Mudunuriauthor=R.+Stephensauthor=F.+J.+Lebeda&title=IC50-to-Ki%3A+a+web-based+tool+for+converting+IC50+to+Ki+values+for+inhibitors+of+enzyme+activity+and+ligand+binding&doi=10.1093%2Fnar%2Fgkp253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span></div><div class="casAuthors">Cer, R. Z.; Mudunuri, U.; Stephens, R.; Lebeda, F. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W441-W445</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A new web-server tool ests. Ki values from exptl. detd. IC50 values for inhibitors of enzymes and of binding reactions between macromols. (e.g., proteins, nucleic acids) and ligands.  This converter was developed to enable end-users to help gauge the quality of the underlying assumptions used in these calcns. which depend on the type of mechanism of inhibitor action and the concns. of the interacting mol. species.  Addnl. calcns. are performed for nonclassical, tightly bound inhibitors of enzyme-substrate or of macromol.-ligand systems in which free, rather than total concns. of the reacting species are required.  Required user-defined input values include the total enzyme (or another target mol.) and substrate (or ligand) concns., the Km of the enzyme-substrate (or the Kd of the target-ligand) reaction, and the IC50 value.  Assumptions and caveats for these calcns. are discussed along with examples taken from the literature.  The host database for this converter contains kinetic consts. and other data for inhibitors of the proteolytic clostridial neurotoxins (http://botdb.abcc.ncifcrf.gov/toxin/kiConverter.jsp).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9gPXTwGaXwLVg90H21EOLACvtfcHk0liJf4_0oyPJoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D&md5=6b0ffe268d709ad8f80da6df29a9565d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp253%26sid%3Dliteratum%253Aachs%26aulast%3DCer%26aufirst%3DR.%2BZ.%26aulast%3DMudunuri%26aufirst%3DU.%26aulast%3DStephens%26aufirst%3DR.%26aulast%3DLebeda%26aufirst%3DF.%2BJ.%26atitle%3DIC50-to-Ki%253A%2520a%2520web-based%2520tool%2520for%2520converting%2520IC50%2520to%2520Ki%2520values%2520for%2520inhibitors%2520of%2520enzyme%2520activity%2520and%2520ligand%2520binding%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3DWeb%2520Server%26spage%3DW441%26epage%3DW445%26doi%3D10.1093%2Fnar%2Fgkp253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, C. W.</span></span> <span> </span><span class="NLM_article-title">Three-dimensional structure of the Stat3 beta homodimer bound to DNA</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>394</i></span> (<span class="NLM_issue">6689</span>),  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1038/28101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2F28101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=9671298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADyaK1cXksFynsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=1998&pages=145-151&issue=6689&author=S.+Beckerauthor=B.+Gronerauthor=C.+W.+Muller&title=Three-dimensional+structure+of+the+Stat3+beta+homodimer+bound+to+DNA&doi=10.1038%2F28101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Three-dimensional structure of the Stat3β homodimer bound to DNA</span></div><div class="casAuthors">Becker, Stefan; Groner, Bernd; Muller, Christoph W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">6689</span>),
    <span class="NLM_cas:pages">145-151</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">STAT proteins are a family of eukaryotic transcription factors that mediate the response to a large no. of cytokines and growth factors.  Upon activation by cell-surface receptors of their assocd. kinases, STAT proteins dimerize, translocate to the nucleus and bind to specific promoter sequences on their target genes.  Here we report the first crystal structure of a STAT protein bound to its DNA recognition site at 2.25 Å resoln.  The structure provides insight into the various steps by which STAT proteins deliver a response signal directly from the cell membrane to their target genes in the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5xUbCQWRy_7Vg90H21EOLACvtfcHk0ljrRx-3ep-ZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksFynsr4%253D&md5=8f6609364a2e7366cff0c95eda84a64a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F28101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F28101%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DGroner%26aufirst%3DB.%26aulast%3DMuller%26aufirst%3DC.%2BW.%26atitle%3DThree-dimensional%2520structure%2520of%2520the%2520Stat3%2520beta%2520homodimer%2520bound%2520to%2520DNA%26jtitle%3DNature%26date%3D1998%26volume%3D394%26issue%3D6689%26spage%3D145%26epage%3D151%26doi%3D10.1038%2F28101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vilar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozza, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moro, S.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry and the molecular operating environment (MOE): application of QSAR and molecular docking to drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.2174/156802608786786624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.2174%2F156802608786786624" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=19075767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=1555-1572&issue=18&author=S.+Vilarauthor=G.+Cozzaauthor=S.+Moro&title=Medicinal+chemistry+and+the+molecular+operating+environment+%28MOE%29%3A+application+of+QSAR+and+molecular+docking+to+drug+discovery&doi=10.2174%2F156802608786786624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry and the Molecular Operating Environment (MOE): application of QSAR and molecular docking to drug discovery</span></div><div class="casAuthors">Vilar, Santiago; Cozza, Giorgio; Moro, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1555-1572</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The search for new compds. with a given biol. activity requires enormous effort in terms of manpower and cost.  This effort arises from the large no. of compds. that need to be synthesized and subsequently biol. evaluated.  For this reason the pharmaceutical industry has shown great interest in theor. methods that enable the rational design of pharmaceutical agents.  In the last years bioinformatics has experienced a great evolution due to the development of specialized software and to the increasing computer power.  The codification of the structural information of mols. through mol. descriptors and the subsequent data anal. allow establishing QSAR models (Quant. Structure-Activity Relationship) that can be applied to the design and the virtual screening of new drugs.  The development of sophisticated Docking methodologies also allows a more accurate predict of the biol. activity of mols.  Moreover, through this type of computational techniques and theor. approaches, it is possible to develop explanatory hypothesis on the mechanism of action of drugs.  This work provides a brief description of a series of studies implemented in the software MOE (Mol. Operating Environment) with particular attention to the medicinal chem. aspects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYzUP1J-30y7Vg90H21EOLACvtfcHk0ljrRx-3ep-ZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlCgsb8%253D&md5=81af5c6fc8cb6e65b34cc62732bbad31</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2174%2F156802608786786624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802608786786624%26sid%3Dliteratum%253Aachs%26aulast%3DVilar%26aufirst%3DS.%26aulast%3DCozza%26aufirst%3DG.%26aulast%3DMoro%26aufirst%3DS.%26atitle%3DMedicinal%2520chemistry%2520and%2520the%2520molecular%2520operating%2520environment%2520%2528MOE%2529%253A%2520application%2520of%2520QSAR%2520and%2520molecular%2520docking%2520to%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2008%26volume%3D8%26issue%3D18%26spage%3D1555%26epage%3D1572%26doi%3D10.2174%2F156802608786786624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&issue=3&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0ljrRx-3ep-ZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26issue%3D3%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">High-affinity peptidomimetic inhibitors of the DCN1-UBC12 protein protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1934</span>– <span class="NLM_lpage">1950</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1934-1950&issue=5&author=H.+B.+Zhouauthor=W.+H.+Zhouauthor=B.+Zhouauthor=L.+Liuauthor=T.+R.+Chernauthor=K.+Chinnaswamyauthor=J.+F.+Luauthor=D.+Bernardauthor=C.+Y.+Yangauthor=S.+S.+Liauthor=M.+Wangauthor=J.+Stuckeyauthor=Y.+Sunauthor=S.+M.+Wang&title=High-affinity+peptidomimetic+inhibitors+of+the+DCN1-UBC12+protein+protein+interaction&doi=10.1021%2Facs.jmedchem.7b01455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity peptidomimetic inhibitors of the DCN1-UBC12 protein-protein interaction</span></div><div class="casAuthors">Zhou, Haibin; Zhou, Weihua; Zhou, Bing; Liu, Liu; Chern, Ting-Rong; Chinnaswamy, Krishnapriya; Lu, Jianfeng; Bernard, Denzil; Yang, Chao-Yie; Li, Shasha; Wang, Mi; Stuckey, Jeanne; Sun, Yi; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1934-1950</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Cullin-RING ligases (CRLs) regulate the turnover of approx. 20% of the proteins in mammalian cells and are emerging therapeutic targets in human diseases.  The activation of CRLs requires the neddylation of their cullin subunit, which is controlled by an activation complex consisting of Cullin-RBX1-UBC12-NEDD8-DCN1.  Herein, we describe the design, synthesis, and evaluation of peptidomimetics targeting the DCN1-UBC12 protein-protein interaction.  Starting from a 12-residue UBC12 peptide, we have successfully obtained a series of peptidomimetic compds. that bind to DCN1 protein with KD values of <10 nM.  Detn. of a cocrystal structure of a potent peptidomimetic inhibitor complexed with DCN1 provides the structural basis for their high-affinity interaction.  Cellular investigation of one potent DCN1 inhibitor, compd. (I) (DI-404), reveals that it effectively and selectively inhibits the neddylation of cullin 3 over other cullin members.  Further optimization of DI-404 may yield a new class of therapeutics for the treatment of human diseases in which cullin 3 CRL plays a key role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfHxzNV467rVg90H21EOLACvtfcHk0liCreq3uO5FkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D&md5=6489af488ea0608aed7348662e9b261f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01455%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%2BB.%26aulast%3DZhou%26aufirst%3DW.%2BH.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChern%26aufirst%3DT.%2BR.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%2BF.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLi%26aufirst%3DS.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DHigh-affinity%2520peptidomimetic%2520inhibitors%2520of%2520the%2520DCN1-UBC12%2520protein%2520protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D5%26spage%3D1934%26epage%3D1950%26doi%3D10.1021%2Facs.jmedchem.7b01455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0liCreq3uO5FkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">Part 4</span>),  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&issue=Part+4&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgk2RdoReN6iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26issue%3DPart%25204%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version, <span class="NLM_edition">2.10.3</span>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">Part 4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=Part+4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgk2RdoReN6iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3DPart%25204%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Womack, T. O.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Bricogne, G.</span></span> <i>GRADE</i>, version <span class="NLM_edition">1.2.13</span>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=O.+S.+Smart&author=T.+O.+Womack&author=A.+Sharff&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=C.+Vonrhein&author=G.+Bricogne&title=GRADE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26jtitle%3DGRADE%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">Part 1</span>),  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&issue=Part+1&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljTw7Ssnstf1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3DPart%25201%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 40 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jinyun Dong, Xiang-Dong Cheng, Wei-Dong Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Jiang-Jiang Qin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Update on Development of Small-Molecule STAT3 Inhibitors for Cancer Therapy: From Phosphorylation Inhibition to Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 8884-8915. <a href="https://doi.org/10.1021/acs.jmedchem.1c00629" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00629%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BUpdate%252Bon%252BDevelopment%252Bof%252BSmall-Molecule%252BSTAT3%252BInhibitors%252Bfor%252BCancer%252BTherapy%25253A%252BFrom%252BPhosphorylation%252BInhibition%252Bto%252BProtein%252BDegradation%26aulast%3DDong%26aufirst%3DJinyun%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06042021%26date%3D25062021%26volume%3D64%26issue%3D13%26spage%3D8884%26epage%3D8915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Troy A. Bemis, James J. La Clair, <span class="NLM_string-name hlFld-ContribAuthor">Michael D. Burkart</span>. </span><span class="cited-content_cbyCitation_article-title">Unraveling the Role of Linker Design in Proteolysis Targeting Chimeras. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8042-8052. <a href="https://doi.org/10.1021/acs.jmedchem.1c00482" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00482</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00482%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUnraveling%252Bthe%252BRole%252Bof%252BLinker%252BDesign%252Bin%252BProteolysis%252BTargeting%252BChimeras%26aulast%3DBemis%26aufirst%3DTroy%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D16032021%26date%3D09062021%26volume%3D64%26issue%3D12%26spage%3D8042%26epage%3D8052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Haibin Zhou, Longchuan Bai, Renqi Xu, Donna McEachern, Krishnapriya Chinnaswamy, Ruiting Li, Bo Wen, Mi Wang, Chao-Yie Yang, Jennifer L. Meagher, Duxin Sun, Jeanne A. Stuckey, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">SD-91 as A Potent and Selective STAT3 Degrader Capable of Achieving Complete and Long-Lasting Tumor Regression. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 996-1004. <a href="https://doi.org/10.1021/acsmedchemlett.1c00155" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00155</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00155%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSD-91%252Bas%252BA%252BPotent%252Band%252BSelective%252BSTAT3%252BDegrader%252BCapable%252Bof%252BAchieving%252BComplete%252Band%252BLong-Lasting%252BTumor%252BRegression%26aulast%3DZhou%26aufirst%3DHaibin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D16032021%26date%3D05052021%26date%3D10052021%26volume%3D12%26issue%3D6%26spage%3D996%26epage%3D1004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rao Song, Wenliang Qiao, Jun He, Jiasheng Huang, Youfu Luo, <span class="NLM_string-name hlFld-ContribAuthor">Tao Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Proteases and Their Modulators in Cancer Therapy: Challenges and Opportunities. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 2851-2877. <a href="https://doi.org/10.1021/acs.jmedchem.0c01640" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01640</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01640%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteases%252Band%252BTheir%252BModulators%252Bin%252BCancer%252BTherapy%25253A%252BChallenges%252Band%252BOpportunities%26aulast%3DSong%26aufirst%3DRao%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D19092020%26date%3D03032021%26volume%3D64%26issue%3D6%26spage%3D2851%26epage%3D2877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maha Hanafi, Xinde Chen, <span class="NLM_string-name hlFld-ContribAuthor">Nouri Neamati</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1626-1648. <a href="https://doi.org/10.1021/acs.jmedchem.0c01897" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01897</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01897%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNapabucasin%252BPROTAC%252Bas%252Ban%252BEffective%252BDegrader%252Bof%252Bthe%252BE3%252BLigase%252BZFP91%26aulast%3DHanafi%26aufirst%3DMaha%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D03112020%26date%3D28012021%26volume%3D64%26issue%3D3%26spage%3D1626%26epage%3D1648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christine Brotherton-Pleiss, Peibin Yue, Yinsong Zhu, Kayo Nakamura, Weiliang Chen, Wenzhen Fu, Casie Kubota, Jasmine Chen, Felix Alonso-Valenteen, Simoun Mikhael, Lali Medina-Kauwe, Marcus A. Tius, Francisco Lopez-Tapia, <span class="NLM_string-name hlFld-ContribAuthor">James Turkson</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Azetidine Amides as Potent Small-Molecule STAT3 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (1)
                                     , 695-710. <a href="https://doi.org/10.1021/acs.jmedchem.0c01705" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01705%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BAzetidine%252BAmides%252Bas%252BPotent%252BSmall-Molecule%252BSTAT3%252BInhibitors%26aulast%3DBrotherton-Pleiss%26aufirst%3DChristine%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D28092020%26date%3D22122020%26volume%3D64%26issue%3D1%26spage%3D695%26epage%3D710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Cyrille S. Kounde, <span class="NLM_string-name hlFld-ContribAuthor">Edward W. Tate</span>. </span><span class="cited-content_cbyCitation_article-title">Photoactive Bifunctional Degraders: Precision Tools To Regulate Protein Stability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15483-15493. <a href="https://doi.org/10.1021/acs.jmedchem.0c01542" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01542</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01542%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPhotoactive%252BBifunctional%252BDegraders%25253A%252BPrecision%252BTools%252BTo%252BRegulate%252BProtein%252BStability%26aulast%3DKounde%26aufirst%3DCyrille%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03092020%26date%3D23112020%26volume%3D63%26issue%3D24%26spage%3D15483%26epage%3D15493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mingliang Wang, Jianfeng Lu, Mi Wang, Chao-Yie Yang, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (14)
                                     , 7510-7528. <a href="https://doi.org/10.1021/acs.jmedchem.0c00471" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00471%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSHP2-D26%252Bas%252Ba%252BFirst%25252C%252BPotent%25252C%252Band%252BEffective%252BPROTAC%252BDegrader%252Bof%252BSHP2%252BProtein%26aulast%3DWang%26aufirst%3DMingliang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20032020%26date%3D21052020%26volume%3D63%26issue%3D14%26spage%3D7510%26epage%3D7528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sanaz Attarha, Anja Reithmeier, Sander Busker, Matthieu Desroses, <span class="NLM_string-name hlFld-ContribAuthor">Brent D. G. Page</span>. </span><span class="cited-content_cbyCitation_article-title">Validating Signal Transducer and Activator of Transcription (STAT) Protein–Inhibitor Interactions Using Biochemical and Cellular Thermal Shift Assays. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (7)
                                     , 1842-1851. <a href="https://doi.org/10.1021/acschembio.0c00046" title="DOI URL">https://doi.org/10.1021/acschembio.0c00046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00046%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DValidating%252BSignal%252BTransducer%252Band%252BActivator%252Bof%252BTranscription%252B%252528STAT%252529%252BProtein%2525E2%252580%252593Inhibitor%252BInteractions%252BUsing%252BBiochemical%252Band%252BCellular%252BThermal%252BShift%252BAssays%26aulast%3DAttarha%26aufirst%3DSanaz%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19012020%26date%3D15052020%26date%3D02072020%26date%3D15052020%26volume%3D15%26issue%3D7%26spage%3D1842%26epage%3D1851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Stuart J. Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Bifunctional Molecules beyond PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (6)
                                     , 2802-2806. <a href="https://doi.org/10.1021/acs.jmedchem.0c00293" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00293</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00293%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBifunctional%252BMolecules%252Bbeyond%252BPROTACs%26aulast%3DConway%26aufirst%3DStuart%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19022020%26date%3D11032020%26volume%3D63%26issue%3D6%26spage%3D2802%26epage%3D2806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiuyao  Huang</span>, <span class="hlFld-ContribAuthor ">Yan  Zhong</span>, <span class="hlFld-ContribAuthor ">Bingbing  Li</span>, <span class="hlFld-ContribAuthor ">Shumin  Ouyang</span>, <span class="hlFld-ContribAuthor ">Lin  Deng</span>, <span class="hlFld-ContribAuthor ">Jianshan  Mo</span>, <span class="hlFld-ContribAuthor ">Shuo  Shi</span>, <span class="hlFld-ContribAuthor ">Nan  Lv</span>, <span class="hlFld-ContribAuthor ">Ruibo  Wu</span>, <span class="hlFld-ContribAuthor ">Peiqing  Liu</span>, <span class="hlFld-ContribAuthor ">Wenhao  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaolei  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuanxiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based discovery of potent and selective small-molecule inhibitors targeting signal transducer and activator of transcription 3 (STAT3). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>221 </em>, 113525. <a href="https://doi.org/10.1016/j.ejmech.2021.113525" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113525%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bdiscovery%252Bof%252Bpotent%252Band%252Bselective%252Bsmall-molecule%252Binhibitors%252Btargeting%252Bsignal%252Btransducer%252Band%252Bactivator%252Bof%252Btranscription%252B3%252B%252528STAT3%252529%26aulast%3DHuang%26aufirst%3DQiuyao%26date%3D2021%26volume%3D221%26spage%3D113525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangbo  Yang</span>, <span class="hlFld-ContribAuthor ">Zhijia  Wang</span>, <span class="hlFld-ContribAuthor ">Yuan  Pei</span>, <span class="hlFld-ContribAuthor ">Ning  Song</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Bo  Feng</span>, <span class="hlFld-ContribAuthor ">Hanlin  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Xiaobei  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Qiu</span>, <span class="hlFld-ContribAuthor ">Huijin  Feng</span>, <span class="hlFld-ContribAuthor ">Yaxi  Yang</span>, <span class="hlFld-ContribAuthor ">Yubo  Zhou</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of thalidomide-based PROTAC small molecules as the highly efficient SHP2 degraders. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113341. <a href="https://doi.org/10.1016/j.ejmech.2021.113341" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113341</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113341%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bthalidomide-based%252BPROTAC%252Bsmall%252Bmolecules%252Bas%252Bthe%252Bhighly%252Befficient%252BSHP2%252Bdegraders%26aulast%3DYang%26aufirst%3DXiangbo%26date%3D2021%26volume%3D218%26spage%3D113341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Niamh  Coleman</span>, <span class="hlFld-ContribAuthor ">Jordi  Rodon</span>. </span><span class="cited-content_cbyCitation_article-title">Taking Aim at the Undruggable. </span><span class="cited-content_cbyCitation_journal-name">American Society of Clinical Oncology Educational Book</span><span> <strong>2021,</strong> <em>61 </em>
                                    (41)
                                     , e145-e152. <a href="https://doi.org/10.1200/EDBK_325885" title="DOI URL">https://doi.org/10.1200/EDBK_325885</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1200/EDBK_325885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1200%2FEDBK_325885%26sid%3Dliteratum%253Aachs%26jtitle%3DAmerican%2520Society%2520of%2520Clinical%2520Oncology%2520Educational%2520Book%26atitle%3DTaking%252BAim%252Bat%252Bthe%252BUndruggable%26aulast%3DColeman%26aufirst%3DNiamh%26date%3D2021%26volume%3D61%26issue%3D41%26spage%3De145%26epage%3De152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Donna D.  Zhang</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting NRF2 to treat cancer. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2021,</strong> <em>19 </em><a href="https://doi.org/10.1016/j.semcancer.2021.06.003" title="DOI URL">https://doi.org/10.1016/j.semcancer.2021.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2021.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2021.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DTargeting%252BNRF2%252Bto%252Btreat%252Bcancer%26aulast%3DSivinski%26aufirst%3DJared%26date%3D2021%26volume%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriel  LaPlante</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Ubiquitin-Proteasome System for Cancer Therapeutics by Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 3079. <a href="https://doi.org/10.3390/cancers13123079" title="DOI URL">https://doi.org/10.3390/cancers13123079</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13123079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13123079%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTargeting%252Bthe%252BUbiquitin-Proteasome%252BSystem%252Bfor%252BCancer%252BTherapeutics%252Bby%252BSmall-Molecule%252BInhibitors%26aulast%3DLaPlante%26aufirst%3DGabriel%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D3079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Si-Min  Qi</span>, <span class="hlFld-ContribAuthor ">Jinyun  Dong</span>, <span class="hlFld-ContribAuthor ">Zhi-Yuan  Xu</span>, <span class="hlFld-ContribAuthor ">Xiang-Dong  Cheng</span>, <span class="hlFld-ContribAuthor ">Wei-Dong  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiang-Jiang  Qin</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC: An Effective Targeted Protein Degradation Strategy for Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.692574" title="DOI URL">https://doi.org/10.3389/fphar.2021.692574</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.692574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.692574%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DPROTAC%25253A%252BAn%252BEffective%252BTargeted%252BProtein%252BDegradation%252BStrategy%252Bfor%252BCancer%252BTherapy%26aulast%3DQi%26aufirst%3DSi-Min%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephanie  Gleicher</span>, <span class="hlFld-ContribAuthor ">Baylee A.  Porter</span>, <span class="hlFld-ContribAuthor ">Disharee  Nath</span>, <span class="hlFld-ContribAuthor ">Guanqun  Li</span>, <span class="hlFld-ContribAuthor ">Rakesh  Khanna</span>, <span class="hlFld-ContribAuthor ">Hanan  Goldberg</span>, <span class="hlFld-ContribAuthor ">Marcin  Kortylewski</span>, <span class="hlFld-ContribAuthor ">Gennady  Bratslavsky</span>, <span class="hlFld-ContribAuthor ">Leszek  Kotula</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Target Opportunities in Non-Metastatic Castrate Resistant Prostate Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (10)
                                     , 2426. <a href="https://doi.org/10.3390/cancers13102426" title="DOI URL">https://doi.org/10.3390/cancers13102426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13102426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13102426%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DNovel%252BTarget%252BOpportunities%252Bin%252BNon-Metastatic%252BCastrate%252BResistant%252BProstate%252BCancer%26aulast%3DGleicher%26aufirst%3DStephanie%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D10%26spage%3D2426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoshun  Luo</span>, <span class="hlFld-ContribAuthor ">Zhenbang  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Xinyu  Li</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Kun  Xi</span>, <span class="hlFld-ContribAuthor ">Maoxu  Xiao</span>, <span class="hlFld-ContribAuthor ">Hanlin  Wei</span>, <span class="hlFld-ContribAuthor ">Lizhe  Zhu</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of an orally active VHL-recruiting PROTAC that achieves robust HMGCR degradation and potent hypolipidemic activity in vivo. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 1300-1314. <a href="https://doi.org/10.1016/j.apsb.2020.11.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.11.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.11.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ban%252Borally%252Bactive%252BVHL-recruiting%252BPROTAC%252Bthat%252Bachieves%252Brobust%252BHMGCR%252Bdegradation%252Band%252Bpotent%252Bhypolipidemic%252Bactivity%252Bin%2525C2%2525A0vivo%26aulast%3DLuo%26aufirst%3DGuoshun%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D1300%26epage%3D1314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Camille Amandine  Dupont</span>, <span class="hlFld-ContribAuthor ">Kristina  Riegel</span>, <span class="hlFld-ContribAuthor ">Malvika  Pompaiah</span>, <span class="hlFld-ContribAuthor ">Hartmut  Juhl</span>, <span class="hlFld-ContribAuthor ">Krishnaraj  Rajalingam</span>. </span><span class="cited-content_cbyCitation_article-title">Druggable genome and precision medicine in cancer: current challenges. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.1111/febs.15788" title="DOI URL">https://doi.org/10.1111/febs.15788</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.15788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.15788%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DDruggable%252Bgenome%252Band%252Bprecision%252Bmedicine%252Bin%252Bcancer%25253A%252Bcurrent%252Bchallenges%26aulast%3DDupont%26aufirst%3DCamille%2BAmandine%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Hongjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Photopharmacology of Proteolysis-Targeting Chimeras: A New Frontier for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.639176" title="DOI URL">https://doi.org/10.3389/fchem.2021.639176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.639176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.639176%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DPhotopharmacology%252Bof%252BProteolysis-Targeting%252BChimeras%25253A%252BA%252BNew%252BFrontier%252Bfor%252BDrug%252BDiscovery%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaohe  Yang</span>, <span class="hlFld-ContribAuthor ">Yali  Wang</span>, <span class="hlFld-ContribAuthor ">Yueying  Yang</span>, <span class="hlFld-ContribAuthor ">Dejuan  Sun</span>, <span class="hlFld-ContribAuthor ">Hua  Li</span>, <span class="hlFld-ContribAuthor ">Lixia  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting SHP2 as a therapeutic strategy for inflammatory diseases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113264. <a href="https://doi.org/10.1016/j.ejmech.2021.113264" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113264</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113264%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BSHP2%252Bas%252Ba%252Btherapeutic%252Bstrategy%252Bfor%252Binflammatory%252Bdiseases%26aulast%3DLiu%26aufirst%3DYang%26date%3D2021%26volume%3D214%26spage%3D113264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nikolai  Makukhin</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in synthetic and medicinal chemistry of phosphotyrosine and phosphonate-based phosphotyrosine analogues. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 8-23. <a href="https://doi.org/10.1039/D0MD00272K" title="DOI URL">https://doi.org/10.1039/D0MD00272K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00272K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00272K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bsynthetic%252Band%252Bmedicinal%252Bchemistry%252Bof%252Bphosphotyrosine%252Band%252Bphosphonate-based%252Bphosphotyrosine%252Banalogues%26aulast%3DMakukhin%26aufirst%3DNikolai%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1%26spage%3D8%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoyi  Yan</span>, <span class="hlFld-ContribAuthor ">Xinxin  Zhong</span>, <span class="hlFld-ContribAuthor ">Lin  Yue</span>, <span class="hlFld-ContribAuthor ">Chunlan  Pu</span>, <span class="hlFld-ContribAuthor ">Huifang  Shan</span>, <span class="hlFld-ContribAuthor ">Suke  Lan</span>, <span class="hlFld-ContribAuthor ">Meng  Zhou</span>, <span class="hlFld-ContribAuthor ">Xueyan  Hou</span>, <span class="hlFld-ContribAuthor ">Jie  Yang</span>, <span class="hlFld-ContribAuthor ">Rui  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a PROTAC targeting ALK with in vivo activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>212 </em>, 113150. <a href="https://doi.org/10.1016/j.ejmech.2020.113150" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113150%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPROTAC%252Btargeting%252BALK%252Bwith%252Bin%2525C2%2525A0vivo%252Bactivity%26aulast%3DYan%26aufirst%3DGuoyi%26date%3D2021%26volume%3D212%26spage%3D113150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jamie V.  Shiah</span>, <span class="hlFld-ContribAuthor ">Jennifer R.  Grandis</span>, <span class="hlFld-ContribAuthor ">Daniel E.  Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting STAT3 with Proteolysis Targeting Chimeras and Next-Generation Antisense Oligonucleotides. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2021,</strong> <em>20 </em>
                                    (2)
                                     , 219-228. <a href="https://doi.org/10.1158/1535-7163.MCT-20-0599" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-20-0599</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-20-0599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-20-0599%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DTargeting%252BSTAT3%252Bwith%252BProteolysis%252BTargeting%252BChimeras%252Band%252BNext-Generation%252BAntisense%252BOligonucleotides%26aulast%3DShiah%26aufirst%3DJamie%2BV.%26date%3D2021%26date%3D2020%26volume%3D20%26issue%3D2%26spage%3D219%26epage%3D228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahul S.  Bhansali</span>, <span class="hlFld-ContribAuthor ">Malini  Rammohan</span>, <span class="hlFld-ContribAuthor ">Paul  Lee</span>, <span class="hlFld-ContribAuthor ">Anouchka P.  Laurent</span>, <span class="hlFld-ContribAuthor ">Qiang  Wen</span>, <span class="hlFld-ContribAuthor ">Praveen  Suraneni</span>, <span class="hlFld-ContribAuthor ">Bon Ham  Yip</span>, <span class="hlFld-ContribAuthor ">Yi-Chien  Tsai</span>, <span class="hlFld-ContribAuthor ">Silvia  Jenni</span>, <span class="hlFld-ContribAuthor ">Beat  Bornhauser</span>, <span class="hlFld-ContribAuthor ">Aurélie  Siret</span>, <span class="hlFld-ContribAuthor ">Corinne  Fruit</span>, <span class="hlFld-ContribAuthor ">Alexandra  Pacheco-Benichou</span>, <span class="hlFld-ContribAuthor ">Ethan  Harris</span>, <span class="hlFld-ContribAuthor ">Thierry  Besson</span>, <span class="hlFld-ContribAuthor ">Benjamin J.  Thompson</span>, <span class="hlFld-ContribAuthor ">Young Ah  Goo</span>, <span class="hlFld-ContribAuthor ">Nobuko  Hijiya</span>, <span class="hlFld-ContribAuthor ">Maria  Vilenchik</span>, <span class="hlFld-ContribAuthor ">Shai  Izraeli</span>, <span class="hlFld-ContribAuthor ">Jean-Pierre  Bourquin</span>, <span class="hlFld-ContribAuthor ">Sébastien  Malinge</span>, <span class="hlFld-ContribAuthor ">John D.  Crispino</span>. </span><span class="cited-content_cbyCitation_article-title">DYRK1A regulates B cell acute lymphoblastic leukemia through phosphorylation of FOXO1 and STAT3. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Investigation</span><span> <strong>2021,</strong> <em>131 </em>
                                    (1)
                                     <a href="https://doi.org/10.1172/JCI135937" title="DOI URL">https://doi.org/10.1172/JCI135937</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1172/JCI135937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1172%2FJCI135937%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Investigation%26atitle%3DDYRK1A%252Bregulates%252BB%252Bcell%252Bacute%252Blymphoblastic%252Bleukemia%252Bthrough%252Bphosphorylation%252Bof%252BFOXO1%252Band%252BSTAT3%26aulast%3DBhansali%26aufirst%3DRahul%2BS.%26date%3D2021%26date%3D2021%26volume%3D131%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro  Martín-Acosta</span>, <span class="hlFld-ContribAuthor ">Xiangshu  Xiao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs to address the challenges facing small molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112993. <a href="https://doi.org/10.1016/j.ejmech.2020.112993" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112993</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112993%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPROTACs%252Bto%252Baddress%252Bthe%252Bchallenges%252Bfacing%252Bsmall%252Bmolecule%252Binhibitors%26aulast%3DMart%25C3%25ADn-Acosta%26aufirst%3DPedro%26date%3D2021%26volume%3D210%26spage%3D112993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuqing  Liang</span>, <span class="hlFld-ContribAuthor ">Kutty Selva  Nandakumar</span>, <span class="hlFld-ContribAuthor ">Kui  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design and pharmaceutical applications of proteolysis-targeting chimeric molecules. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>182 </em>, 114211. <a href="https://doi.org/10.1016/j.bcp.2020.114211" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.114211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.114211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.114211%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DDesign%252Band%252Bpharmaceutical%252Bapplications%252Bof%252Bproteolysis-targeting%252Bchimeric%252Bmolecules%26aulast%3DLiang%26aufirst%3DYuqing%26date%3D2020%26volume%3D182%26spage%3D114211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yonghan  He</span>, <span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Zhiguang  Huo</span>, <span class="hlFld-ContribAuthor ">Dongwen  Lv</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Robert  Hromas</span>, <span class="hlFld-ContribAuthor ">Mingjiang  Xu</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimeras (PROTACs) are emerging therapeutics for hematologic malignancies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-020-00924-z" title="DOI URL">https://doi.org/10.1186/s13045-020-00924-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-020-00924-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-020-00924-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DProteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bare%252Bemerging%252Btherapeutics%252Bfor%252Bhematologic%252Bmalignancies%26aulast%3DHe%26aufirst%3DYonghan%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-020-00885-3" title="DOI URL">https://doi.org/10.1186/s13045-020-00885-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-020-00885-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-020-00885-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DProteolysis-targeting%252Bchimera%252B%252528PROTAC%252529%252Bfor%252Btargeted%252Bprotein%252Bdegradation%252Band%252Bcancer%252Btherapy%26aulast%3DLi%26aufirst%3DXin%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Jieqing  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC: A Novel Technology for Drug Development**. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (42)
                                     , 13232-13247. <a href="https://doi.org/10.1002/slct.202003162" title="DOI URL">https://doi.org/10.1002/slct.202003162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202003162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202003162%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DPROTAC%25253A%252BA%252BNovel%252BTechnology%252Bfor%252BDrug%252BDevelopment%252A%252A%26aulast%3DLi%26aufirst%3DJie%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D42%26spage%3D13232%26epage%3D13247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lina  Yin</span>, <span class="hlFld-ContribAuthor ">Qingzhong  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Chimera induced protein degradation: PROTACs and beyond. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112494. <a href="https://doi.org/10.1016/j.ejmech.2020.112494" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112494</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112494%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChimera%252Binduced%252Bprotein%252Bdegradation%25253A%252BPROTACs%252Band%252Bbeyond%26aulast%3DYin%26aufirst%3DLina%26date%3D2020%26volume%3D206%26spage%3D112494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenhao  Dai</span>, <span class="hlFld-ContribAuthor ">Chunpu  Li</span>, <span class="hlFld-ContribAuthor ">Yichu  Liu</span>, <span class="hlFld-ContribAuthor ">Xu  Han</span>, <span class="hlFld-ContribAuthor ">Xingjun  Li</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-catalyzed [4 + 3] dearomatizing cycloaddition reaction of
              N
              -iminoquinolinium ylides. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2020,</strong> <em>7 </em>
                                    (18)
                                     , 2612-2617. <a href="https://doi.org/10.1039/D0QO00320D" title="DOI URL">https://doi.org/10.1039/D0QO00320D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0QO00320D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0QO00320D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DPalladium-catalyzed%252B%25255B4%252B%25252B%252B3%25255D%252Bdearomatizing%252Bcycloaddition%252Breaction%252Bof%252BN%252B-iminoquinolinium%252Bylides%26aulast%3DDai%26aufirst%3DWenhao%26date%3D2020%26date%3D2020%26volume%3D7%26issue%3D18%26spage%3D2612%26epage%3D2617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai-Rui  Feng</span>, <span class="hlFld-ContribAuthor ">Feng  Wang</span>, <span class="hlFld-ContribAuthor ">Xin-Wei  Shi</span>, <span class="hlFld-ContribAuthor ">Yun-Xuan  Tan</span>, <span class="hlFld-ContribAuthor ">Jia-Ying  Zhao</span>, <span class="hlFld-ContribAuthor ">Jian-Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">Qing-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Guo-Qiang  Lin</span>, <span class="hlFld-ContribAuthor ">Dingding  Gao</span>, <span class="hlFld-ContribAuthor ">Ping  Tian</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel potent STAT3 inhibitors based on BBI608 for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>201 </em>, 112428. <a href="https://doi.org/10.1016/j.ejmech.2020.112428" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112428%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bpotent%252BSTAT3%252Binhibitors%252Bbased%252Bon%252BBBI608%252Bfor%252Bcancer%252Btherapy%26aulast%3DFeng%26aufirst%3DKai-Rui%26date%3D2020%26volume%3D201%26spage%3D112428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elvin D.  de Araujo</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>, <span class="hlFld-ContribAuthor ">Patrick T.  Gunning</span>, <span class="hlFld-ContribAuthor ">Richard  Moriggl</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting STAT3 and STAT5 in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (8)
                                     , 2002. <a href="https://doi.org/10.3390/cancers12082002" title="DOI URL">https://doi.org/10.3390/cancers12082002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12082002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12082002%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTargeting%252BSTAT3%252Band%252BSTAT5%252Bin%252BCancer%26aulast%3Dde%2BAraujo%26aufirst%3DElvin%2BD.%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D8%26spage%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yilin  Chen</span>, <span class="hlFld-ContribAuthor ">Jianping  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">The application of ubiquitin ligases in the PROTAC drug design. </span><span class="cited-content_cbyCitation_journal-name">Acta Biochimica et Biophysica Sinica</span><span> <strong>2020,</strong> <em>52 </em>
                                    (7)
                                     , 776-790. <a href="https://doi.org/10.1093/abbs/gmaa053" title="DOI URL">https://doi.org/10.1093/abbs/gmaa053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abbs/gmaa053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabbs%2Fgmaa053%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Biochimica%2520et%2520Biophysica%2520Sinica%26atitle%3DThe%252Bapplication%252Bof%252Bubiquitin%252Bligases%252Bin%252Bthe%252BPROTAC%252Bdrug%252Bdesign%26aulast%3DChen%26aufirst%3DYilin%26date%3D2020%26date%3D2020%26volume%3D52%26issue%3D7%26spage%3D776%26epage%3D790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Yonghan  He</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Shaoyan  Pu</span>, <span class="hlFld-ContribAuthor ">Qingpeng  Kong</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">PROteolysis TArgeting Chimeras (PROTACs) as emerging anticancer therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2020,</strong> <em>39 </em>
                                    (26)
                                     , 4909-4924. <a href="https://doi.org/10.1038/s41388-020-1336-y" title="DOI URL">https://doi.org/10.1038/s41388-020-1336-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-020-1336-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-020-1336-y%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DPROteolysis%252BTArgeting%252BChimeras%252B%252528PROTACs%252529%252Bas%252Bemerging%252Banticancer%252Btherapeutics%26aulast%3DKhan%26aufirst%3DSajid%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D26%26spage%3D4909%26epage%3D4924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Mohrherr</span>, <span class="hlFld-ContribAuthor ">Iris Z.  Uras</span>, <span class="hlFld-ContribAuthor ">Herwig P.  Moll</span>, <span class="hlFld-ContribAuthor ">Emilio  Casanova</span>. </span><span class="cited-content_cbyCitation_article-title">STAT3: Versatile Functions in Non-Small Cell Lung Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (5)
                                     , 1107. <a href="https://doi.org/10.3390/cancers12051107" title="DOI URL">https://doi.org/10.3390/cancers12051107</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12051107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12051107%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DSTAT3%25253A%252BVersatile%252BFunctions%252Bin%252BNon-Small%252BCell%252BLung%252BCancer%26aulast%3DMohrherr%26aufirst%3DJulian%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D5%26spage%3D1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Dinesh  Thummuri</span>, <span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Wanyi  Hu</span>, <span class="hlFld-ContribAuthor ">Peiyi  Zhang</span>, <span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of PROTAC BCL-XL degraders as potent anticancer agents with low on-target platelet toxicity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112186. <a href="https://doi.org/10.1016/j.ejmech.2020.112186" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112186%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPROTAC%252BBCL-XL%252Bdegraders%252Bas%252Bpotent%252Banticancer%252Bagents%252Bwith%252Blow%252Bon-target%252Bplatelet%252Btoxicity%26aulast%3DZhang%26aufirst%3DXuan%26date%3D2020%26volume%3D192%26spage%3D112186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0001.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of potent and cell-permeable small molecule inhibitors of the STAT3 SH2 domain: chemical structures and binding affinities of our previously reported compound CJ-887 and newly designed and synthesized inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0002.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of SI-109 with STAT3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ">6NUQ</a>). (A) The <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density generated from an omit map contoured at 3σ shows placement of atoms of SI-109. (B) Detailed interactions of STAT3 with SI-109. SI-109 (cyan) and side chains of interacting residues of STAT3 with SI-109 are shown in stick form. Hydrogen bonds are depicted as dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0003.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of putative STAT3 PROTAC degraders based upon cereblon ligands and our STAT3 SH2 domain inhibitor SI-109.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0004.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design of SD-36Me as a control compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0005.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western blotting analysis of STAT proteins in Molm-16 and SU-DHL-1 cells treated with SD-36 or control compounds SD-36Me and SI-109. (A) Molm-16 cells were treated with SD-36 at 0.01–10 μM or SD-36Me and SI-109 at 10 μM for 4 h. (B) SU-DHL-1 cells were treated with SD-36 at 0.01–10 μM or SD-36Me and SI-109 at 10 μM for 18 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0006.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Pharmacodynamics study of SD-36 and <b>19</b> in native mouse spleen tissue in CD-1 mice (A) and SD-36 in the Molm-16 xenograft tumors in SCID mice (B). CD-1 and SCID mice were treated intravenously with a single dose of SD-36 or <b>19</b> as indicated. Native spleen tissue or tumor lysates were analyzed by immunoblotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0007.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <i>In vivo</i> antitumor activity of SD-36 in the Molm-16 xenograft model in mice: (A) tumor growth; (B) percentage of animal body weight change.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 5-((Di-<i>tert</i>-butoxyphosphoryl)methyl)-1<i>H</i>-indole-2-carboxylic Acid (<b>41</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, Boc<sub>2</sub>O, THF, 0 °C to rt; (b) NBS, Bz<sub>2</sub>O<sub>2</sub>, CCl<sub>4</sub>, reflux, 77% over 2 steps; (c) (<i>t</i>-BuO)<sub>2</sub>PONa 1.0 equiv of NaH 2.5 equiv, THF, then reflux for 3 h; (d) LiOH, MeOH–THF–H<sub>2</sub>O, 60 °C.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 3-((Diethoxyphosphoryl)difluoromethyl)-1<i>H</i>-indole-2-carboxylic Acid (<b>46</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) P(OEt)<sub>3</sub>, 100 °C, 84%; (b) Ti(O-<i>i</i>-Pr)<sub>4</sub>, BnOH, 100 °C, 83%; (c) NaH, Cbz-Cl, THF, 0 °C to rt, 88%; (d) NFBS, NaHMDS, THF, −78 °C to rt, 95%; (e) H<sub>2</sub>/Pd–C, THF, 94%.</p></p></figure><figure data-id="sch2a" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>2</b>–<b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch2a"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EDC–HCl 2.0 equiv, HOBt 2.0 equiv, EtN(<i>i</i>-Pr)<sub>2</sub>, DCM, rt, 3 h, 89% yield; (b) CF<sub>3</sub>CO<sub>2</sub>H 2 mL, Et<sub>3</sub>Si–H 0.1 mL, DCM 3 mL, rt, 1 h; (c) (1) protected phosphotyrosine mimetics, EDC–HCl, HOBt, EtN(<i>i</i>-Pr)<sub>2</sub>, DCM, rt, 3 h; (2) TFA–DCM, rt, 3 h; or TMSI, 0 °C, 1 h.</p></p></figure><figure data-id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0011.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>5</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (2) TFA, DCM; (b) <b>46</b>, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (c) H<sub>2</sub>, Pd/C, MeOH; (d) (1) HCHO, sodium triacetoxyborohydride, DCE; (2) TFA, DCM; (e) R-NH<sub>2</sub>, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h.</p></p></figure><figure data-id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0012.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>8</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (b) TFA, DCM; (c) <b>51</b>, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (d) (1) H<sub>2</sub>, Pd/C, MeOH; (2) Ac<sub>2</sub>O, EtN(<i>i</i>-Pr)<sub>2</sub>, DCM; (3) TFA, DCM; (e) (1) phosphotyrosine mimetics, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (2) TFA–DCM, rt, 3 h; or TMSI, 0 °C, 1 h.</p></p></figure><figure data-id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0013.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of SD-36<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) TFA, DCM; (2) <b>46</b>, HATU, DIEA, DMF; (b) H<sub>2</sub>, Pd/C, MeOH; (c) <b>68</b>, HATU, DIEA, DMF; (d) TMSI, BSTFA, DCM; (e) Pd(PPh<sub>3</sub>)Cl<sub>2</sub>, CuI, DMF/TEA, 80 °C, 1 h.</p></p></figure><figure data-id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0014.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>30</b> and <b>31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) <b>68</b>, HATU, DIEA, DMF; (2) TFA, DCM; (b) (1) R-NH<sub>2</sub>, HATU, DIEA, DMF; (2) TMSI, BSTFA, DCM.</p></p></figure><figure data-id="sch7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0015.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>32</b>–<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) H<sub>2</sub>, Pd/C, MeOH; (2) <b>68</b>, HATU, DIEA, DMF; (3) TFA, DCM; (b) (1) phosphotyrosine mimetics, HATU, EtN(<i>i</i>-Pr)<sub>2</sub>, DMF, rt, 1 h; (2) TFA–DCM, rt, 3 h; or TMSI, 0 °C, 1 h.</p></p></figure><figure data-id="sch8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/medium/jm9b01530_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0016.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Fluorescent Labeled Compound <b>74</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b01530/20191219/images/large/jm9b01530_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01530&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HATU, DIEA, DMF; (2) TFA, DCM; (b) (1) HATU, DIEA, DMF; (2) TMSI, BSTFA, DCM.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i84">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14001" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14001" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 43 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.</span></span> <span> </span><span class="NLM_article-title">STATs: Transcriptional control and biological impact</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1038/nrm909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrm909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=12209125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmslansrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=651-662&issue=9&author=D.+E.+Levyauthor=J.+E.+Darnell&title=STATs%3A+Transcriptional+control+and+biological+impact&doi=10.1038%2Fnrm909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Stats: transcriptional control and biological impact</span></div><div class="casAuthors">Levy, David E.; Darnell, J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">651-662</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Extracellular proteins bound to cell-surface receptors can change nuclear gene expression patterns in minutes, with far-reaching consequences for development, cell growth, and homeostasis.  The signal transducer and activator of transcription (STAT) proteins are among the most well studied of the latent cytoplasmic signal-dependent transcription factor pathways.  In addn. to several roles in normal cell decisions, dysregulation of STAT function contributes to human disease, making the study of these proteins an important topic of current research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGqYF32j262LVg90H21EOLACvtfcHk0lh3qneTaywKeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmslansrw%253D&md5=4a04359c59e54b12d8a67efb6ef42e4c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm909%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DD.%2BE.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DSTATs%253A%2520Transcriptional%2520control%2520and%2520biological%2520impact%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2002%26volume%3D3%26issue%3D9%26spage%3D651%26epage%3D662%26doi%3D10.1038%2Fnrm909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">The stats of cancer - new molecular targets come of age</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1038/nrc1275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrc1275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=14964307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=97-105&issue=2&author=H.+Yuauthor=R.+Jove&title=The+stats+of+cancer+-+new+molecular+targets+come+of+age&doi=10.1038%2Fnrc1275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The STATs of cancer - new molecular targets come of age</span></div><div class="casAuthors">Yu, Hua; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-105</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Tumor cells acquire the ability to proliferate uncontrollably, resist apoptosis, sustain angiogenesis and evade immune surveillance.  STAT proteins - esp. STAT3 and STAT5 - regulate all of these processes and are persistently activated in a surprisingly large no. of human cancers.  Consequently, STAT proteins are emerging - unexpectedly - as ideal targets for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdnLoVoH4d1rVg90H21EOLACvtfcHk0lh3qneTaywKeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtlaqtb0%253D&md5=3e342db4856310cac2465240e36d3226</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc1275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1275%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DThe%2520stats%2520of%2520cancer%2520-%2520new%2520molecular%2520targets%2520come%2520of%2520age%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26issue%3D2%26spage%3D97%26epage%3D105%26doi%3D10.1038%2Fnrc1275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandis, J. R.</span></span> <span> </span><span class="NLM_article-title">Targeting the IL-6/JAK/STAT3 signalling axis in cancer</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">248</span>, <span class="refDoi"> DOI: 10.1038/nrclinonc.2018.8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrclinonc.2018.8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29405201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12nsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=234-248&issue=4&author=D.+E.+Johnsonauthor=R.+A.+O%E2%80%99Keefeauthor=J.+R.+Grandis&title=Targeting+the+IL-6%2FJAK%2FSTAT3+signalling+axis+in+cancer&doi=10.1038%2Fnrclinonc.2018.8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the IL-6/JAK/STAT3 signalling axis in cancer</span></div><div class="casAuthors">Johnson, Daniel E.; O'Keefe, Rachel A.; Grandis, Jennifer R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">234-248</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The IL-6/JAK/STAT3 pathway is aberrantly hyperactivated in many types of cancer, and such hyperactivation is generally assocd. with a poor clin. prognosis.  In the tumor microenvironment, IL-6/JAK/STAT3 signalling acts to drive the proliferation, survival, invasiveness, and metastasis of tumor cells, while strongly suppressing the antitumor immune response.  Thus, treatments that target the IL-6/JAK/STAT3 pathway in patients with cancer are poised to provide therapeutic benefit by directly inhibiting tumor cell growth and by stimulating antitumor immunity.  Agents targeting IL-6, the IL-6 receptor, or JAKs have already received FDA approval for the treatment of inflammatory conditions or myeloproliferative neoplasms and for the management of certain adverse effects of chimeric antigen receptor T cells, and are being further evaluated in patients with haematopoietic malignancies and in those with solid tumors.  Novel inhibitors of the IL-6/JAK/STAT3 pathway, including STAT3-selective inhibitors, are currently in development.  Herein, we review the role of IL-6/JAK/STAT3 signalling in the tumor microenvironment and the status of preclin. and clin. investigations of agents targeting this pathway.  We also discuss the potential of combining IL-6/JAK/STAT3 inhibitors with currently approved therapeutic agents directed against immune-checkpoint inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocclyreQ67DbVg90H21EOLACvtfcHk0lg214ru-7sciA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12nsbc%253D&md5=91390fe680e391d628b53fdda5324a1b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2018.8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2018.8%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BE.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DR.%2BA.%26aulast%3DGrandis%26aufirst%3DJ.%2BR.%26atitle%3DTargeting%2520the%2520IL-6%252FJAK%252FSTAT3%2520signalling%2520axis%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D15%26issue%3D4%26spage%3D234%26epage%3D248%26doi%3D10.1038%2Fnrclinonc.2018.8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. T.</span></span> <span> </span><span class="NLM_article-title">Two decades of research in discovery of anticancer drugs targeting STAT3, how close are we?</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2018.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.pharmthera.2018.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29933035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2018&pages=74-91&author=J.+D.+Beebeauthor=J.+Y.+Liuauthor=J.+T.+Zhang&title=Two+decades+of+research+in+discovery+of+anticancer+drugs+targeting+STAT3%2C+how+close+are+we%3F&doi=10.1016%2Fj.pharmthera.2018.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Two decades of research in discovery of anticancer drugs targeting STAT3, how close are we?</span></div><div class="casAuthors">Beebe, Jenny D.; Liu, Jing-Yuan; Zhang, Jian-Ting</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">74-91</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) controls many biol. processes including differentiation, survival, proliferation, and angiogenesis.  In normal healthy cells, STAT3 is tightly regulated to maintain a momentary active state.  However, aberrant or constitutively activated STAT3 has been obsd. in many different cancers and constitutively activated STAT3 has been shown to assoc. with poor prognosis and tumor progression.  For this reason, STAT3 has been studied as a possible target in the treatment of many different types of cancers.  However, despite decades of research, a FDA-approved STAT3 inhibitor has yet to emerge.  In this review, we will analyze past studies targeting STAT3 for drug discovery, understand possible causes of failure in these studies, and provide potential insights for future efforts to overcome these roadblocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSWDPJOIaRKrVg90H21EOLACvtfcHk0lg214ru-7sciA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO&md5=048e7d95608bdab18335c55d084d62ab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2018.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2018.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DBeebe%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%2BY.%26aulast%3DZhang%26aufirst%3DJ.%2BT.%26atitle%3DTwo%2520decades%2520of%2520research%2520in%2520discovery%2520of%2520anticancer%2520drugs%2520targeting%2520STAT3%252C%2520how%2520close%2520are%2520we%253F%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D191%26spage%3D74%26epage%3D91%26doi%3D10.1016%2Fj.pharmthera.2018.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, G. R.</span></span> <span> </span><span class="NLM_article-title">Roles of unphosphorylated STATs in signaling</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">443</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1038/cr.2008.41</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fcr.2008.41" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=18364677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFCks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=443-451&issue=4&author=J.+Yangauthor=G.+R.+Stark&title=Roles+of+unphosphorylated+STATs+in+signaling&doi=10.1038%2Fcr.2008.41"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of unphosphorylated STATs in signaling</span></div><div class="casAuthors">Yang, Jinbo; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">443-451</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The seven members of the signal transducer and activator of transcription (STAT) family of transcription factors are activated in response to many different cytokines and growth factors by phosphorylation of specific tyrosine residues.  The STAT1 and STAT3 genes are specific targets of activated STATs 1 and 3, resp., resulting in large increases in the levels of these unphosphorylated STATs (U-STATs) in response to the interferons (STAT1) or ligands that active gp130, such as IL-6 (STAT3).  U-STATs drive gene expression by novel mechanisms distinct from those used by phosphorylated STAT (P-STAT) dimers.  In this review, we discuss the roles of U-STATs in transcription and regulation of gene expression.  Cell Research (2008) 18:443-451. doi: 10.1038/cr.2008.41: published online 25 March 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqynLvLsNqMOrVg90H21EOLACvtfcHk0lg214ru-7sciA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFCks74%253D&md5=ce421809dffce50716f7ebec75d2ad80</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcr.2008.41&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2008.41%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DRoles%2520of%2520unphosphorylated%2520STATs%2520in%2520signaling%26jtitle%3DCell%2520Res.%26date%3D2008%26volume%3D18%26issue%3D4%26spage%3D443%26epage%3D451%26doi%3D10.1038%2Fcr.2008.41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzrock, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woessner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemunaitis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fayad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piha-Paul, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadella, M. V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohseni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakey, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLeod, A. R.</span></span> <span> </span><span class="NLM_article-title">AZD9150, a next-generation antisense oligonucleotide inhibitor of STAT3 with early evidence of clinical activity in lymphoma and lung cancer</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">314</span>),  <span class="NLM_fpage">314ra185</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aac5272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1126%2Fscitranslmed.aac5272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=26582900" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=314ra185&issue=314&author=D.+Hongauthor=R.+Kurzrockauthor=Y.+Kimauthor=R.+Woessnerauthor=A.+Younesauthor=J.+Nemunaitisauthor=N.+Fowlerauthor=T.+Y.+Zhouauthor=J.+Schmidtauthor=M.+J.+Joauthor=S.+J.+Leeauthor=M.+Yamashitaauthor=S.+G.+Hughesauthor=L.+Fayadauthor=S.+Piha-Paulauthor=M.+V.+P.+Nadellaauthor=M.+Mohseniauthor=D.+Lawsonauthor=C.+Reimerauthor=D.+C.+Blakeyauthor=X.+K.+Xiaoauthor=J.+Hsuauthor=A.+Revenkoauthor=B.+P.+Moniaauthor=A.+R.+MacLeod&title=AZD9150%2C+a+next-generation+antisense+oligonucleotide+inhibitor+of+STAT3+with+early+evidence+of+clinical+activity+in+lymphoma+and+lung+cancer&doi=10.1126%2Fscitranslmed.aac5272"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aac5272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aac5272%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DD.%26aulast%3DKurzrock%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DWoessner%26aufirst%3DR.%26aulast%3DYounes%26aufirst%3DA.%26aulast%3DNemunaitis%26aufirst%3DJ.%26aulast%3DFowler%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DT.%2BY.%26aulast%3DSchmidt%26aufirst%3DJ.%26aulast%3DJo%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DYamashita%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DS.%2BG.%26aulast%3DFayad%26aufirst%3DL.%26aulast%3DPiha-Paul%26aufirst%3DS.%26aulast%3DNadella%26aufirst%3DM.%2BV.%2BP.%26aulast%3DMohseni%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DBlakey%26aufirst%3DD.%2BC.%26aulast%3DXiao%26aufirst%3DX.%2BK.%26aulast%3DHsu%26aufirst%3DJ.%26aulast%3DRevenko%26aufirst%3DA.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DMacLeod%26aufirst%3DA.%2BR.%26atitle%3DAZD9150%252C%2520a%2520next-generation%2520antisense%2520oligonucleotide%2520inhibitor%2520of%2520STAT3%2520with%2520early%2520evidence%2520of%2520clinical%2520activity%2520in%2520lymphoma%2520and%2520lung%2520cancer%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26issue%3D314%26spage%3D314ra185%26doi%3D10.1126%2Fscitranslmed.aac5272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. J. A.</span></span> <span> </span><span class="NLM_article-title">Antisense oligonucleotides: the next frontier for treatment of neurological disorders</span>. <i>Nat. Rev. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1038/nrneurol.2017.148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrneurol.2017.148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29192260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCrs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=9-21&issue=1&author=C.+Rinaldiauthor=M.+J.+A.+Wood&title=Antisense+oligonucleotides%3A+the+next+frontier+for+treatment+of+neurological+disorders&doi=10.1038%2Fnrneurol.2017.148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Antisense oligonucleotides: the next frontier for treatment of neurological disorders</span></div><div class="casAuthors">Rinaldi, Carlo; Wood, Matthew J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-21</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Antisense oligonucleotides (ASOs) were first discovered to influence RNA processing and modulate protein expression over two decades ago; however, progress translating these agents into the clinic has been hampered by inadequate target engagement, insufficient biol. activity, and off-target toxic effects.  Over the years, novel chem. modifications of ASOs have been employed to address these issues.  These modifications, in combination with elucidation of the mechanism of action of ASOs and improved clin. trial design, have provided momentum for the translation of ASO-based strategies into therapies.  Many neurol. conditions lack an effective treatment; however, as research progressively disentangles the pathogenic mechanisms of these diseases, they provide an ideal platform to test ASO-based strategies.  This steady progress reached a pinnacle in the past few years with approvals of ASOs for the treatment of spinal muscular atrophy and Duchenne muscular dystrophy, which represent landmarks in a field in which disease-modifying therapies were virtually non-existent.  With the rapid development of improved next-generation ASOs toward clin. application, this technol. now holds the potential to have a dramatic effect on the treatment of many neurol. conditions in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlk6GFxML247Vg90H21EOLACvtfcHk0lhyhz00bfVCfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCrs7jL&md5=a9256a0257a13c8c331e1229aecfd4aa</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2017.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2017.148%26sid%3Dliteratum%253Aachs%26aulast%3DRinaldi%26aufirst%3DC.%26aulast%3DWood%26aufirst%3DM.%2BJ.%2BA.%26atitle%3DAntisense%2520oligonucleotides%253A%2520the%2520next%2520frontier%2520for%2520treatment%2520of%2520neurological%2520disorders%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2018%26volume%3D14%26issue%3D1%26spage%3D9%26epage%3D21%26doi%3D10.1038%2Fnrneurol.2017.148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&issue=6&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0liy37gXkDLJpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26issue%3D6%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&issue=2&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0liy37gXkDLJpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26issue%3D2%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: elements of PROTAC design</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.cbpa.2019.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=31004963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=111-119&author=S.+L.+Paivaauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+elements+of+PROTAC+design&doi=10.1016%2Fj.cbpa.2019.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: elements of PROTAC design</span></div><div class="casAuthors">Paiva, Stacey-Lynn; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn. using Proteolysis Targeting Chimeras (PROTACs) has emerged as a novel therapeutic modality in drug discovery.  PROTACs mediate the degrdn. of select proteins of interest (POIs) by hijacking the activity of E3 ubiquitin ligases for POI ubiquitination and subsequent degrdn. by the 26S proteasome.  This hijacking mechanism has been used to degrade various types of disease-relevant POIs.  In this review, we aim to highlight the recent advances in targeted protein degrdn. and describe the challenges that need to be addressed in order to efficiently develop potent PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol6Nk0VkUdzLVg90H21EOLACvtfcHk0liy37gXkDLJpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D&md5=5783311db47da378180c14d67526a398</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DPaiva%26aufirst%3DS.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520elements%2520of%2520PROTAC%2520design%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D111%26epage%3D119%26doi%3D10.1016%2Fj.cbpa.2019.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation by small molecules</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">123</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1146%2Fannurev-pharmtox-010715-103507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=27732798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=107-123&author=D.+P.+Bondesonauthor=C.+M.+Crews&title=Targeted+protein+degradation+by+small+molecules&doi=10.1146%2Fannurev-pharmtox-010715-103507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation by Small Molecules</span></div><div class="casAuthors">Bondeson, Daniel P.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107-123</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Protein homeostasis networks are highly regulated systems responsible for maintaining the health and productivity of cells.  Whereas therapeutics have been developed to disrupt protein homeostasis, more recently identified techniques have been used to repurpose homeostatic networks to effect degrdn. of disease-relevant proteins.  Here, we review recent advances in the use of small mols. to degrade proteins in a selective manner.  First, we highlight all-small-mol. techniques with direct clin. application.  Second, we describe techniques that may find broader acceptance in the biomedical research community that require little or no synthetic chem.  In addn. to serving as innovative research tools, these new approaches to control intracellular protein levels offer the potential to develop novel therapeutics targeting proteins that are not currently pharmaceutically vulnerable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6k9y8vt0uPbVg90H21EOLACvtfcHk0lgY0VbegoGyvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt73L&md5=a9d7b9fc821e6dca7fbe1615187a2e87</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103507%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%2520by%2520small%2520molecules%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2017%26volume%3D57%26spage%3D107%26epage%3D123%26doi%3D10.1146%2Fannurev-pharmtox-010715-103507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of conformationally constrained, cell-permeable STAT3 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1021/ml100010j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100010j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=85-89&issue=2&author=J.+Chenauthor=L.+Baiauthor=D.+Bernardauthor=Z.+Nikolovska-Coleskaauthor=C.+Gomezauthor=J.+Zhangauthor=H.+Yiauthor=S.+Wang&title=Structure-based+design+of+conformationally+constrained%2C+cell-permeable+STAT3+inhibitors&doi=10.1021%2Fml100010j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Conformationally Constrained, Cell-Permeable STAT3 Inhibitors</span></div><div class="casAuthors">Chen, Jianyong; Bai, Longchuan; Bernard, Denzil; Nikolovska-Coleska, Zaneta; Gomez, Cindy; Zhang, Jian; Yi, Han; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-89</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the structure-based design of a class of conformationally constrained, potent, cell-permeable small-mol. inhibitors to target the SH2 domain in STAT3.  Compd. 11 (CJ-1383) binds to STAT3 with a Ki value of 0.95 μM, dose-dependently inhibits cellular STAT3 signaling and cancer cell growth, and induces apoptosis in the MDA-MB-468 cancer cell line with constitutively activated STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMrK-vDy7ISbVg90H21EOLACvtfcHk0lgY0VbegoGyvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGiurs%253D&md5=87a0b1a823441940f75fc2af960e0359</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fml100010j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100010j%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DGomez%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520of%2520conformationally%2520constrained%252C%2520cell-permeable%2520STAT3%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26issue%3D2%26spage%3D85%26epage%3D89%26doi%3D10.1021%2Fml100010j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burke, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span> <span> </span><span class="NLM_article-title">Phosphoryltyrosyl mimetics in the design of peptide-based signal transduction inhibitors</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1002/1097-0282(2001)60:1<32::AID-BIP1002>3.0.CO;2-I</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=11376431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Cqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2001&pages=32-44&issue=1&author=T.+R.+Burkeauthor=Z.+J.+Yaoauthor=D.+G.+Liuauthor=J.+Voigtauthor=Y.+Gao&title=Phosphoryltyrosyl+mimetics+in+the+design+of+peptide-based+signal+transduction+inhibitors&doi=10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoryltyrosyl mimetics in the design of peptide-based signal transduction inhibitors</span></div><div class="casAuthors">Burke, Terrence R., Jr.; Yao, Zhu-Jun; Liu, Ding-Guo; Voigt, Johannes; Gao, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-44</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review, with 83 refs.  The central roles played by protein-tyrosine kinase (PTK)-dependent signal transduction in normal cellular regulation and homeostasis have made inappropriate or aberrant functions of certain of these pathways contributing factors to a variety of diseases, including several cancers.  For this reason, development of PTK signaling inhibitors has evolved into an important approach toward new therapeutics.  Since in these pathways phosphotyrosyl (pTyr) residues provide unique and defining functions either by (1) their creation under the catalysis of PTKs, (2) their recognition and binding by protein modules such as SH2 and phosphotyrosyl binding (PTB) domains, or (3) their destruction by protein-tyrosine phosphatases (PTPs), pTyr mimetics provide useful general starting points for inhibitor design.  Important considerations in the development of such pTyr mimetics include enzymic stability (particularly toward PTPs), high affinity recognition by target pTyr binding proteins, and good cellular bioavailability.  Although small mol. nonpeptide inhibitors may represent the ultimate objectives of inhibitor development, peptides frequently serve as display platforms for pTyr mimetics, which afford useful and conceptually straightforward starting points in the development process.  Reported herein is a limited overview of pTyr mimetic development as it relates to peptide-based agents.  Of particular interest are recent findings that highlight potential limitations of peptides as display platforms for the identification of small mol. leads.  One conclusion that results from this work is that while peptide-based approaches toward small mol. inhibitor design are often intellectually satisfying from a structure-based perspective, extrapolation of neg. findings to small mol., nonpeptide contexts should be undertaken with extreme caution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcIBlemRR8r7Vg90H21EOLACvtfcHk0lg2zyoBzY3eMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Cqsbk%253D&md5=76bb70bd5e8cab68f76f8d9edcc67fbd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1097-0282%25282001%252960%253A1%253C32%253A%253AAID-BIP1002%253E3.0.CO%253B2-I%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DYao%26aufirst%3DZ.%2BJ.%26aulast%3DLiu%26aufirst%3DD.%2BG.%26aulast%3DVoigt%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DY.%26atitle%3DPhosphoryltyrosyl%2520mimetics%2520in%2520the%2520design%2520of%2520peptide-based%2520signal%2520transduction%2520inhibitors%26jtitle%3DBiopolymers%26date%3D2001%26volume%3D60%26issue%3D1%26spage%3D32%26epage%3D44%26doi%3D10.1002%2F1097-0282%282001%2960%3A1%3C32%3A%3AAID-BIP1002%3E3.0.CO%3B2-I" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burke, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Phosphotyrosyl mimetics in the development of signal transduction inhibitors</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">426</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1021/ar020127o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar020127o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslCmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2003&pages=426-433&issue=6&author=T.+R.+Burkeauthor=K.+Lee&title=Phosphotyrosyl+mimetics+in+the+development+of+signal+transduction+inhibitors&doi=10.1021%2Far020127o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphotyrosyl Mimetics in the Development of Signal Transduction Inhibitors</span></div><div class="casAuthors">Burke, Terrence R., Jr.; Lee, Kyeong</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">426-433</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Phosphotyrosyl (pTyr) residues play important roles in cellular signal transduction by facilitating recognition and binding necessary for crit. protein-protein interactions, and for this reason pTyr motifs represent attractive starting points in the development of signaling antagonists.  Although the pTyr phosphoryl moiety is central in these phenomena, its incorporation into signaling inhibitors is contraindicated due to enzymic lability and limited bioavailability assocd. with phosphate esters.  To address these limitations, an entire field of study has arisen devoted to the design and utilization of pTyr mimetics.  This Account provides a perspective on the roles of pTyr residues in signal transduction and approaches to pTyr mimetic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqecqbwygKL7bVg90H21EOLACvtfcHk0lg2zyoBzY3eMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslCmtLs%253D&md5=84c9610cea90bfcb3ce91a0ede387e77</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Far020127o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far020127o%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPhosphotyrosyl%2520mimetics%2520in%2520the%2520development%2520of%2520signal%2520transduction%2520inhibitors%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2003%26volume%3D36%26issue%3D6%26spage%3D426%26epage%3D433%26doi%3D10.1021%2Far020127o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of stat3</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3394</span>– <span class="NLM_lpage">3397</span>, <span class="refDoi"> DOI: 10.1021/ol9012662</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol9012662" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=3394-3397&issue=15&author=P.+K.+Mandalauthor=W.+S.+L.+Liaoauthor=J.+S.+McMurray&title=Synthesis+of+phosphatase-stable%2C+cell-permeable+peptidomimetic+prodrugs+that+target+the+SH2+domain+of+stat3&doi=10.1021%2Fol9012662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of phosphatase-stable, cell-permeable peptidomimetic prodrugs that target the SH2 domain of Stat3</span></div><div class="casAuthors">Mandal, Pijus K.; Liao, Warren S.-L.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3394-3397</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of prodrugs targeted to the SH2 domain of Stat3 is reported.  Using a convergent strategy, the pivaloyloxymethyl phosphonodiester of pentachlorophenyl 4-phosphonodifluoromethylcinnamate, a phosphotyrosine surrogate, was synthesized and used to acylate peptidomimetic fragments that were prepd. on solid supports.  Two prodrugs described here inhibited the phosphorylation of Stat3 in breast tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqokZ-epNTxprVg90H21EOLACvtfcHk0lg2zyoBzY3eMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotl2ht78%253D&md5=c935049f107f5ab89bf38cd445c845f5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fol9012662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol9012662%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLiao%26aufirst%3DW.%2BS.%2BL.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520phosphatase-stable%252C%2520cell-permeable%2520peptidomimetic%2520prodrugs%2520that%2520target%2520the%2520SH2%2520domain%2520of%2520stat3%26jtitle%3DOrg.%2520Lett.%26date%3D2009%26volume%3D11%26issue%3D15%26spage%3D3394%26epage%3D3397%26doi%3D10.1021%2Fol9012662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limbrick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtwistle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Conformationally constrained peptidomimetic inhibitors of signal transducer and activator of transcription 3: evaluation and molecular modeling</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2429</span>– <span class="NLM_lpage">2442</span>, <span class="refDoi"> DOI: 10.1021/jm801491w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801491w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFKrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2429-2442&issue=8&author=P.+K.+Mandalauthor=D.+Limbrickauthor=D.+R.+Colemanauthor=G.+A.+Dyerauthor=Z.+Y.+Renauthor=J.+S.+Birtwistleauthor=C.+Y.+Xiongauthor=X.+M.+Chenauthor=J.+M.+Briggsauthor=J.+S.+McMurray&title=Conformationally+constrained+peptidomimetic+inhibitors+of+signal+transducer+and+activator+of+transcription+3%3A+evaluation+and+molecular+modeling&doi=10.1021%2Fjm801491w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conformationally Constrained Peptidomimetic Inhibitors of Signal Transducer and Activator of Transcription 3: Evaluation and Molecular Modeling</span></div><div class="casAuthors">Mandal, Pijus K.; Limbrick, Donald; Coleman, David R.; Dyer, Garrett A.; Ren, Zhiyong; Birtwistle, J. Sanderson; Xiong, Chiyi; Chen, Xiaomin; Briggs, James M.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2429-2442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (Stat3) is involved in aberrant growth and survival signals in malignant tumor cells and is a validated target for anticancer drug design.  We are targeting its SH2 domain to prevent docking to cytokine and growth factor receptors and subsequent signaling.  The amino acids of our lead phosphopeptide, Ac-pTyr-Leu-Pro-Gln-Thr-Val-NH2, were replaced with conformationally constrained mimics.  Structure-affinity studies led to the peptidomimetic, pCinn-Haic-Gln-NHBn, which had an IC50 of 162 nM (fluorescence polarization), compared to 290 nM for the lead phosphopeptide (pCinn = 4-phosphoryloxycinnamate, Haic = (2S,5S)-5-amino-1,2,4,5,6,7-hexahydro-4-oxo-azepino[3,2,1-hi]indole-2-carboxylic acid).  PCinn-Haic-Gln-OH was docked to the SH2 domain (AUTODOCK), and the two highest populated clusters were subjected to mol. dynamics simulations.  Both converged to a common peptide conformation.  The complex exhibits unique hydrogen bonding between Haic and Gln and Stat3 as well as hydrophobic interactions between the protein and pCinn and Haic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorsyN8dGgwIrVg90H21EOLACvtfcHk0lgZYCsKROEtDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFKrsL8%253D&md5=e1e59cdd54551d4d167394034ecdb7ee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm801491w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801491w%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DLimbrick%26aufirst%3DD.%26aulast%3DColeman%26aufirst%3DD.%2BR.%26aulast%3DDyer%26aufirst%3DG.%2BA.%26aulast%3DRen%26aufirst%3DZ.%2BY.%26aulast%3DBirtwistle%26aufirst%3DJ.%2BS.%26aulast%3DXiong%26aufirst%3DC.%2BY.%26aulast%3DChen%26aufirst%3DX.%2BM.%26aulast%3DBriggs%26aufirst%3DJ.%2BM.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DConformationally%2520constrained%2520peptidomimetic%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%253A%2520evaluation%2520and%2520molecular%2520modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D8%26spage%3D2429%26epage%3D2442%26doi%3D10.1021%2Fjm801491w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtwistle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaluarachchi, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Potent and selective phosphopeptide mimetic prodrugs targeted to the src homology 2 (SH2) domain of signal transducer and activator of transcription 3</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3549</span>– <span class="NLM_lpage">3563</span>, <span class="refDoi"> DOI: 10.1021/jm2000882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2000882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlt1Cnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3549-3563&issue=10&author=P.+K.+Mandalauthor=F.+Q.+Gaoauthor=Z.+Luauthor=Z.+Y.+Renauthor=R.+Rameshauthor=J.+S.+Birtwistleauthor=K.+K.+Kaluarachchiauthor=X.+M.+Chenauthor=R.+C.+Bastauthor=W.+S.+Liaoauthor=J.+S.+McMurray&title=Potent+and+selective+phosphopeptide+mimetic+prodrugs+targeted+to+the+src+homology+2+%28SH2%29+domain+of+signal+transducer+and+activator+of+transcription+3&doi=10.1021%2Fjm2000882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Phosphopeptide Mimetic Prodrugs Targeted to the Src Homology 2 (SH2) Domain of Signal Transducer and Activator of Transcription 3</span></div><div class="casAuthors">Mandal, Pijus K.; Gao, Fengqin; Lu, Zhen; Ren, Zhiyong; Ramesh, Rajagopal; Birtwistle, J. Sanderson; Kaluarachchi, Kumaralal K.; Chen, Xiaomin; Bast, Robert C.; Liao, Warren S.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3549-3563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (Stat3), a target for anticancer drug design, is activated by recruitment to phosphotyrosine residues on growth factor and cytokine receptors via its SH2 domain.  We report here structure-activity relation studies on phosphopeptide mimics targeted to the SH2 domain of Stat3.  Inclusion of a Me group on the β-position of the pTyr mimic 4-phosphocinnamide enhanced affinity 2- to 3-fold.  Bis-pivaloyloxymethyl prodrugs contg. β-methylcinnamide, dipeptide scaffolds Haic and Nle-cis-3,4-methanoproline, and glutamine surrogates were highly potent, completely inhibiting phosphorylation of Stat3 Tyr705 at 0.5-1 μM in a variety of cancer cell lines.  The inhibitors were selective for Stat3 over Stat1, Stat5, Src, and p85 of PI3K, indicating ability to discriminate individual SH2 domains in intact cells.  At concns. that completely inhibited Stat3 phosphorylation, the prodrugs were not cytotoxic to a panel of tumor cells, thereby showing clear distinction between cytotoxicity and effects downstream of activated Stat3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqVZisU6d4iLVg90H21EOLACvtfcHk0lgZYCsKROEtDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlt1Cnsbk%253D&md5=f0a68b4edcbcfc621cb50ab29af3bd45</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm2000882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2000882%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DGao%26aufirst%3DF.%2BQ.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DRen%26aufirst%3DZ.%2BY.%26aulast%3DRamesh%26aufirst%3DR.%26aulast%3DBirtwistle%26aufirst%3DJ.%2BS.%26aulast%3DKaluarachchi%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DX.%2BM.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DLiao%26aufirst%3DW.%2BS.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DPotent%2520and%2520selective%2520phosphopeptide%2520mimetic%2520prodrugs%2520targeted%2520to%2520the%2520src%2520homology%25202%2520%2528SH2%2529%2520domain%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D10%26spage%3D3549%26epage%3D3563%26doi%3D10.1021%2Fjm2000882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the von hippel-lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1 alpha interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&issue=10&author=D.+L.+Buckleyauthor=I.+Van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+von+hippel-lindau+E3+ubiquitin+ligase+using+small+molecules+to+disrupt+the+VHL%2FHIF-1+alpha+interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0lhGTmp9uHk5RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520von%2520hippel-lindau%2520E3%2520ubiquitin%2520ligase%2520using%2520small%2520molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1%2520alpha%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26issue%3D10%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein protein interaction between the von hippel- lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&issue=20&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein+protein+interaction+between+the+von+hippel-+lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhGTmp9uHk5RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein%2520protein%2520interaction%2520between%2520the%2520von%2520hippel-%2520lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D20%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-based optimization of potent and cell-active inhibitors of the von hippel-lindau (VHL) E3 ubiquitin ligase: structure-activity relationships leading to the chemical probe (2S,4R)-1-((S)-2-(1-cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyI)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyppyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=599-618&issue=2&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-based+optimization+of+potent+and+cell-active+inhibitors+of+the+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase%3A+structure-activity+relationships+leading+to+the+chemical+probe+%282S%2C4R%29-1-%28%28S%29-2-%281-cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyI%29-4-hydroxy-N-%284-%284-methylthiazol-5-yl%29benzyppyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase: Structure-Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span></div><div class="casAuthors">Soares, Pedro; Gadd, Morgan S.; Frost, Julianty; Galdeano, Carles; Ellis, Lucy; Epemolu, Ola; Rocha, Sonia; Read, Kevin D.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">599-618</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau tumor suppressor protein is the substrate binding subunit of the VHL E3 ubiquitin ligase, which targets hydroxylated α subunit of hypoxia inducible factors (HIFs) for ubiquitination and subsequent proteasomal degrdn.  VHL is a potential target for treating anemia and ischemic diseases, motivating the development of inhibitors of the VHL:HIF-α protein-protein interaction.  Addnl., bifunctional proteolysis targeting chimeras (PROTACs) contg. a VHL ligand can hijack the E3 ligase activity to induce degrdn. of target proteins.  We report the structure-guided design and group-based optimization of a series of VHL inhibitors with low nanomolar potencies and improved cellular permeability.  Structure-activity relationships led to the discovery of potent inhibitors 10 and chem. probe VH298, with dissocn. consts. <100 nM, which induced marked HIF-1α intracellular stabilization.  Our study provides new chem. tools to probe the VHL-HIF pathways and new VHL ligands for next-generation PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQfJe2l2WgX7Vg90H21EOLACvtfcHk0ljM99HAECnjtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P&md5=1c1e7c976c2de9a9681773ff216fdfd7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-based%2520optimization%2520of%2520potent%2520and%2520cell-active%2520inhibitors%2520of%2520the%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%253A%2520structure-activity%2520relationships%2520leading%2520to%2520the%2520chemical%2520probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyI%2529-4-hydroxy-N-%25284-%25284-methylthiazol-5-yl%2529benzyppyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D2%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Surface probing by fragment-based screening and computational methods identifies ligandable pockets on the von hippel-lindau (VHL) E3 ubiquitin ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">7387</span>– <span class="NLM_lpage">7393</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00842</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00842" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyktLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7387-7393&issue=16&author=X.+Lucasauthor=I.+Van+Molleauthor=A.+Ciulli&title=Surface+probing+by+fragment-based+screening+and+computational+methods+identifies+ligandable+pockets+on+the+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase&doi=10.1021%2Facs.jmedchem.8b00842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Probing by Fragment-Based Screening and Computational Methods Identifies Ligandable Pockets on the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Lucas, Xavier; Van Molle, Inge; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7387-7393</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Beyond the targeting of E3 ubiquitin ligases to inhibit protein homeostasis, E3 ligase binders can be repurposed as targeted protein degraders (PROTACs or mol. glues).  The authors sought to identify new binders of the VHL E3 ligase by biophys. fragment-based screening followed by x-ray crystallog. soaking.  The authors identified fragments binding at the Elongin C:Cullin 2 interface and a new cryptic pocket in VHL, along with other potential ligandable sites predicted computationally and found to bind solvent mols. in crystal structures.  The elucidated interactions provide starting points for future ligand development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZdhy5IVXNgrVg90H21EOLACvtfcHk0ljM99HAECnjtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyktLbE&md5=c081e7cf26807f59e646a60a0ec74361</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00842%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSurface%2520probing%2520by%2520fragment-based%2520screening%2520and%2520computational%2520methods%2520identifies%2520ligandable%2520pockets%2520on%2520the%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D16%26spage%3D7387%26epage%3D7393%26doi%3D10.1021%2Facs.jmedchem.8b00842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BET proteins in triple-negative breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2476</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1158%2F0008-5472.CAN-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476-2487&issue=9&author=L.+C.+Baiauthor=B.+Zhouauthor=C.+Y.+Yangauthor=J.+Jiauthor=D.+McEachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+T.+Huauthor=F.+M.+Xuauthor=Y.+J.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+L.+Douauthor=B.+Wenauthor=D.+X.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Q.+Liauthor=M.+J.+Ellisauthor=S.+M.+Wang&title=Targeted+degradation+of+BET+proteins+in+triple-negative+breast+cancer&doi=10.1158%2F0008-5472.CAN-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0lhZFhBZ3Gz2Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%2BC.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%2BT.%26aulast%3DXu%26aufirst%3DF.%2BM.%26aulast%3DZhao%26aufirst%3DY.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%2BL.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%2BX.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%2BQ.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DTargeted%2520degradation%2520of%2520BET%2520proteins%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26issue%3D9%26spage%3D2476%26epage%3D2487%26doi%3D10.1158%2F0008-5472.CAN-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+T.+Huauthor=F.+M.+Xuauthor=Z.+Chenauthor=L.+C.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+J.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+X.+Sunauthor=S.+M.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lgvGtT7ZRd9dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%2BT.%26aulast%3DXu%26aufirst%3DF.%2BM.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%2BC.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%2BJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%2BX.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&issue=15&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+C.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+L.+Yangauthor=D.+N.+Ziazadehauthor=F.+M.+Xuauthor=J.+T.+Huauthor=W.+G.+Xiangauthor=L.+Y.+Huangauthor=S.+W.+Liauthor=B.+Wenauthor=D.+X.+Sunauthor=S.+M.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lgvGtT7ZRd9dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%2BC.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%2BL.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%2BM.%26aulast%3DHu%26aufirst%3DJ.%2BT.%26aulast%3DXiang%26aufirst%3DW.%2BG.%26aulast%3DHuang%26aufirst%3DL.%2BY.%26aulast%3DLi%26aufirst%3DS.%2BW.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%2BX.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D15%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera murine double minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&issue=2&author=Y.+B.+Liauthor=J.+L.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.+Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+M.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+murine+double+minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0liDb7wuIDmIVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%2BB.%26aulast%3DYang%26aufirst%3DJ.%2BL.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520murine%2520double%2520minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D2%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent and selective small-molecule degrader of STAT3 achieves complete tumor regression In vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511&issue=5&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+potent+and+selective+small-molecule+degrader+of+STAT3+achieves+complete+tumor+regression+In+vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0liDb7wuIDmIVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520and%2520selective%2520small-molecule%2520degrader%2520of%2520STAT3%2520achieves%2520complete%2520tumor%2520regression%2520In%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26issue%3D5%26spage%3D498%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDBI-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span> (<span class="NLM_issue">7512</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&issue=7512&author=E.+S.+Fischerauthor=K.+Bohmauthor=J.+R.+Lydeardauthor=H.+D.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thoma&title=Structure+of+the+DDBI-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0ljJCJ_TPAxJLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBohm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%2BD.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDBI-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26issue%3D7512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&issue=6&author=J.+Luauthor=Y.+M.+Qianauthor=M.+Altieriauthor=H.+Q.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0ljJCJ_TPAxJLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%2BM.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%2BQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26issue%3D6%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&issue=1&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lg_xdABnDhcfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26issue%3D1%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a 1,5-diazabicyclo[6,3,0] dodecane amino acid derivative as a novel dipeptide reverse-turn mimetic</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">4769</span>– <span class="NLM_lpage">4770</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2006.04.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2Fj.tetlet.2006.04.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsVyktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=4769-4770&issue=27&author=Y.+F.+Pengauthor=H.+Y.+Sunauthor=S.+M.+Wang&title=Design+and+synthesis+of+a+1%2C5-diazabicyclo%5B6%2C3%2C0%5D+dodecane+amino+acid+derivative+as+a+novel+dipeptide+reverse-turn+mimetic&doi=10.1016%2Fj.tetlet.2006.04.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of a 1,5-diazabicyclo[6.3.0]dodecane amino acid derivative as a novel dipeptide reverse-turn mimetic</span></div><div class="casAuthors">Peng, Yuefeng; Sun, Haiying; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">4769-4770</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">N-Boc-1,5-diazabicyclo[6.3.0]dodecane amino acid tert-Bu ester I, designed as a novel dipeptide reverse-turn mimetic, was synthesized in a concise method from the known compd. II in an overall yield of 47%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyiUhd6L-vHLVg90H21EOLACvtfcHk0lgD1vjcyrDhSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsVyktrg%253D&md5=1aadc23cec2bd840e08a333a951ed1f1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2006.04.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2006.04.053%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%2BF.%26aulast%3DSun%26aufirst%3DH.%2BY.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%25201%252C5-diazabicyclo%255B6%252C3%252C0%255D%2520dodecane%2520amino%2520acid%2520derivative%2520as%2520a%2520novel%2520dipeptide%2520reverse-turn%2520mimetic%26jtitle%3DTetrahedron%2520Lett.%26date%3D2006%26volume%3D47%26issue%3D27%26spage%3D4769%26epage%3D4770%26doi%3D10.1016%2Fj.tetlet.2006.04.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morlacchi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Link, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurieva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavraki, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corry, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span></span> <span> </span><span class="NLM_article-title">Targeting the src homology 2 (SH2) domain of signal transducer and activator of transcription 6 (STAT6) with cell-permeable, phosphatase-stable phosphopeptide mimics potently inhibits tyr641 phosphorylation and transcriptional activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">8970</span>– <span class="NLM_lpage">8984</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01321</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01321" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegt7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8970-8984&issue=22&author=P.+K.+Mandalauthor=P.+Morlacchiauthor=J.+M.+Knightauthor=T.+M.+Linkauthor=G.+R.+Leeauthor=R.+Nurievaauthor=D.+Singhauthor=A.+Dhanikauthor=L.+Kavrakiauthor=D.+B.+Corryauthor=J.+E.+Ladburyauthor=J.+S.+McMurray&title=Targeting+the+src+homology+2+%28SH2%29+domain+of+signal+transducer+and+activator+of+transcription+6+%28STAT6%29+with+cell-permeable%2C+phosphatase-stable+phosphopeptide+mimics+potently+inhibits+tyr641+phosphorylation+and+transcriptional+activity&doi=10.1021%2Facs.jmedchem.5b01321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Src Homology 2 (SH2) Domain of Signal Transducer and Activator of Transcription 6 (STAT6) with Cell-Permeable, Phosphatase-Stable Phosphopeptide Mimics Potently Inhibits Tyr641 Phosphorylation and Transcriptional Activity</span></div><div class="casAuthors">Mandal, Pijus K.; Morlacchi, Pietro; Knight, J. Morgan; Link, Todd M.; Lee, Gilbert R.; Nurieva, Roza; Singh, Divyendu; Dhanik, Ankur; Kavraki, Lydia; Corry, David B.; Ladbury, John E.; McMurray, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8970-8984</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 6 (STAT6) transmits signals from cytokines IL-4 and IL-13 and is activated in allergic airway disease.  The authors are developing phosphopeptide mimetics targeting the SH2 domain of STAT6 to block recruitment to phosphotyrosine residues on IL-4 or IL-13 receptors and subsequent Tyr 641 phosphorylation to inhibit the expression of genes contributing to asthma.  Structure-affinity relationship studies showed that phosphopeptides based on Tyr 631 from IL-4Rα bind with weak affinity to STAT6, whereas replacing the pY+3 residue with simple aryl and alkyl amides resulted in affinities in the mid to low nM range.  A set of phosphatase-stable, cell-permeable prodrug analogs inhibited cytokine-stimulated STAT6 phosphorylation in both Beas-2B human airway cells and primary mouse T-lymphocytes at concns. as low as 100 nM.  IL-13-stimulated expression of CCL26 (eotaxin-3) was inhibited in a dose-dependent manner, demonstrating that targeting the SH2 domain blocks both phosphorylation and transcriptional activity of STAT6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquqyZb9Mhgg7Vg90H21EOLACvtfcHk0lgD1vjcyrDhSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegt7zE&md5=699ec8930ac7325966a21b937d4edab9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01321%26sid%3Dliteratum%253Aachs%26aulast%3DMandal%26aufirst%3DP.%2BK.%26aulast%3DMorlacchi%26aufirst%3DP.%26aulast%3DKnight%26aufirst%3DJ.%2BM.%26aulast%3DLink%26aufirst%3DT.%2BM.%26aulast%3DLee%26aufirst%3DG.%2BR.%26aulast%3DNurieva%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DD.%26aulast%3DDhanik%26aufirst%3DA.%26aulast%3DKavraki%26aufirst%3DL.%26aulast%3DCorry%26aufirst%3DD.%2BB.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DTargeting%2520the%2520src%2520homology%25202%2520%2528SH2%2529%2520domain%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25206%2520%2528STAT6%2529%2520with%2520cell-permeable%252C%2520phosphatase-stable%2520phosphopeptide%2520mimics%2520potently%2520inhibits%2520tyr641%2520phosphorylation%2520and%2520transcriptional%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D22%26spage%3D8970%26epage%3D8984%26doi%3D10.1021%2Facs.jmedchem.5b01321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. S.</span>; <span class="NLM_string-name">Mandal, P. K.</span>; <span class="NLM_string-name">Liao, W. S.</span>; <span class="NLM_string-name">Ren, Z.</span>; <span class="NLM_string-name">Chen, X.</span>; <span class="NLM_string-name">Rajaopal, R.</span>; <span class="NLM_string-name">Robertson, F.</span></span> <span> </span><span class="NLM_article-title">Preparation of Phosphopeptide Inhibitors of STAT3</span>. <span class="NLM_patent">WO 2010118309</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=J.+S.+McMurray&author=P.+K.+Mandal&author=W.+S.+Liao&author=Z.+Ren&author=X.+Chen&author=R.+Rajaopal&author=F.+Robertson&title=Preparation+of+Phosphopeptide+Inhibitors+of+STAT3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcMurray%26aufirst%3DJ.%2BS.%26atitle%3DPreparation%2520of%2520Phosphopeptide%2520Inhibitors%2520of%2520STAT3%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cer, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudunuri, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeda, F. J.</span></span> <span> </span><span class="NLM_article-title">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">Web Server</span>),  <span class="NLM_fpage">W441</span>– <span class="NLM_lpage">W445</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1093%2Fnar%2Fgkp253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=19395593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=W441-W445&issue=Web+Server&author=R.+Z.+Cerauthor=U.+Mudunuriauthor=R.+Stephensauthor=F.+J.+Lebeda&title=IC50-to-Ki%3A+a+web-based+tool+for+converting+IC50+to+Ki+values+for+inhibitors+of+enzyme+activity+and+ligand+binding&doi=10.1093%2Fnar%2Fgkp253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span></div><div class="casAuthors">Cer, R. Z.; Mudunuri, U.; Stephens, R.; Lebeda, F. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W441-W445</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A new web-server tool ests. Ki values from exptl. detd. IC50 values for inhibitors of enzymes and of binding reactions between macromols. (e.g., proteins, nucleic acids) and ligands.  This converter was developed to enable end-users to help gauge the quality of the underlying assumptions used in these calcns. which depend on the type of mechanism of inhibitor action and the concns. of the interacting mol. species.  Addnl. calcns. are performed for nonclassical, tightly bound inhibitors of enzyme-substrate or of macromol.-ligand systems in which free, rather than total concns. of the reacting species are required.  Required user-defined input values include the total enzyme (or another target mol.) and substrate (or ligand) concns., the Km of the enzyme-substrate (or the Kd of the target-ligand) reaction, and the IC50 value.  Assumptions and caveats for these calcns. are discussed along with examples taken from the literature.  The host database for this converter contains kinetic consts. and other data for inhibitors of the proteolytic clostridial neurotoxins (http://botdb.abcc.ncifcrf.gov/toxin/kiConverter.jsp).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9gPXTwGaXwLVg90H21EOLACvtfcHk0lgUlzTIL5QDeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D&md5=6b0ffe268d709ad8f80da6df29a9565d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp253%26sid%3Dliteratum%253Aachs%26aulast%3DCer%26aufirst%3DR.%2BZ.%26aulast%3DMudunuri%26aufirst%3DU.%26aulast%3DStephens%26aufirst%3DR.%26aulast%3DLebeda%26aufirst%3DF.%2BJ.%26atitle%3DIC50-to-Ki%253A%2520a%2520web-based%2520tool%2520for%2520converting%2520IC50%2520to%2520Ki%2520values%2520for%2520inhibitors%2520of%2520enzyme%2520activity%2520and%2520ligand%2520binding%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3DWeb%2520Server%26spage%3DW441%26epage%3DW445%26doi%3D10.1093%2Fnar%2Fgkp253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, C. W.</span></span> <span> </span><span class="NLM_article-title">Three-dimensional structure of the Stat3 beta homodimer bound to DNA</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>394</i></span> (<span class="NLM_issue">6689</span>),  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1038/28101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1038%2F28101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=9671298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADyaK1cXksFynsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=1998&pages=145-151&issue=6689&author=S.+Beckerauthor=B.+Gronerauthor=C.+W.+Muller&title=Three-dimensional+structure+of+the+Stat3+beta+homodimer+bound+to+DNA&doi=10.1038%2F28101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Three-dimensional structure of the Stat3β homodimer bound to DNA</span></div><div class="casAuthors">Becker, Stefan; Groner, Bernd; Muller, Christoph W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">6689</span>),
    <span class="NLM_cas:pages">145-151</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">STAT proteins are a family of eukaryotic transcription factors that mediate the response to a large no. of cytokines and growth factors.  Upon activation by cell-surface receptors of their assocd. kinases, STAT proteins dimerize, translocate to the nucleus and bind to specific promoter sequences on their target genes.  Here we report the first crystal structure of a STAT protein bound to its DNA recognition site at 2.25 Å resoln.  The structure provides insight into the various steps by which STAT proteins deliver a response signal directly from the cell membrane to their target genes in the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5xUbCQWRy_7Vg90H21EOLACvtfcHk0lgUlzTIL5QDeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksFynsr4%253D&md5=8f6609364a2e7366cff0c95eda84a64a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F28101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F28101%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DGroner%26aufirst%3DB.%26aulast%3DMuller%26aufirst%3DC.%2BW.%26atitle%3DThree-dimensional%2520structure%2520of%2520the%2520Stat3%2520beta%2520homodimer%2520bound%2520to%2520DNA%26jtitle%3DNature%26date%3D1998%26volume%3D394%26issue%3D6689%26spage%3D145%26epage%3D151%26doi%3D10.1038%2F28101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vilar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozza, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moro, S.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry and the molecular operating environment (MOE): application of QSAR and molecular docking to drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.2174/156802608786786624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.2174%2F156802608786786624" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=19075767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=1555-1572&issue=18&author=S.+Vilarauthor=G.+Cozzaauthor=S.+Moro&title=Medicinal+chemistry+and+the+molecular+operating+environment+%28MOE%29%3A+application+of+QSAR+and+molecular+docking+to+drug+discovery&doi=10.2174%2F156802608786786624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry and the Molecular Operating Environment (MOE): application of QSAR and molecular docking to drug discovery</span></div><div class="casAuthors">Vilar, Santiago; Cozza, Giorgio; Moro, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1555-1572</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The search for new compds. with a given biol. activity requires enormous effort in terms of manpower and cost.  This effort arises from the large no. of compds. that need to be synthesized and subsequently biol. evaluated.  For this reason the pharmaceutical industry has shown great interest in theor. methods that enable the rational design of pharmaceutical agents.  In the last years bioinformatics has experienced a great evolution due to the development of specialized software and to the increasing computer power.  The codification of the structural information of mols. through mol. descriptors and the subsequent data anal. allow establishing QSAR models (Quant. Structure-Activity Relationship) that can be applied to the design and the virtual screening of new drugs.  The development of sophisticated Docking methodologies also allows a more accurate predict of the biol. activity of mols.  Moreover, through this type of computational techniques and theor. approaches, it is possible to develop explanatory hypothesis on the mechanism of action of drugs.  This work provides a brief description of a series of studies implemented in the software MOE (Mol. Operating Environment) with particular attention to the medicinal chem. aspects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYzUP1J-30y7Vg90H21EOLACvtfcHk0lif2wxnMEzG0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlCgsb8%253D&md5=81af5c6fc8cb6e65b34cc62732bbad31</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2174%2F156802608786786624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802608786786624%26sid%3Dliteratum%253Aachs%26aulast%3DVilar%26aufirst%3DS.%26aulast%3DCozza%26aufirst%3DG.%26aulast%3DMoro%26aufirst%3DS.%26atitle%3DMedicinal%2520chemistry%2520and%2520the%2520molecular%2520operating%2520environment%2520%2528MOE%2529%253A%2520application%2520of%2520QSAR%2520and%2520molecular%2520docking%2520to%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2008%26volume%3D8%26issue%3D18%26spage%3D1555%26epage%3D1572%26doi%3D10.2174%2F156802608786786624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&issue=3&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lif2wxnMEzG0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26issue%3D3%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">High-affinity peptidomimetic inhibitors of the DCN1-UBC12 protein protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1934</span>– <span class="NLM_lpage">1950</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1934-1950&issue=5&author=H.+B.+Zhouauthor=W.+H.+Zhouauthor=B.+Zhouauthor=L.+Liuauthor=T.+R.+Chernauthor=K.+Chinnaswamyauthor=J.+F.+Luauthor=D.+Bernardauthor=C.+Y.+Yangauthor=S.+S.+Liauthor=M.+Wangauthor=J.+Stuckeyauthor=Y.+Sunauthor=S.+M.+Wang&title=High-affinity+peptidomimetic+inhibitors+of+the+DCN1-UBC12+protein+protein+interaction&doi=10.1021%2Facs.jmedchem.7b01455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity peptidomimetic inhibitors of the DCN1-UBC12 protein-protein interaction</span></div><div class="casAuthors">Zhou, Haibin; Zhou, Weihua; Zhou, Bing; Liu, Liu; Chern, Ting-Rong; Chinnaswamy, Krishnapriya; Lu, Jianfeng; Bernard, Denzil; Yang, Chao-Yie; Li, Shasha; Wang, Mi; Stuckey, Jeanne; Sun, Yi; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1934-1950</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Cullin-RING ligases (CRLs) regulate the turnover of approx. 20% of the proteins in mammalian cells and are emerging therapeutic targets in human diseases.  The activation of CRLs requires the neddylation of their cullin subunit, which is controlled by an activation complex consisting of Cullin-RBX1-UBC12-NEDD8-DCN1.  Herein, we describe the design, synthesis, and evaluation of peptidomimetics targeting the DCN1-UBC12 protein-protein interaction.  Starting from a 12-residue UBC12 peptide, we have successfully obtained a series of peptidomimetic compds. that bind to DCN1 protein with KD values of <10 nM.  Detn. of a cocrystal structure of a potent peptidomimetic inhibitor complexed with DCN1 provides the structural basis for their high-affinity interaction.  Cellular investigation of one potent DCN1 inhibitor, compd. (I) (DI-404), reveals that it effectively and selectively inhibits the neddylation of cullin 3 over other cullin members.  Further optimization of DI-404 may yield a new class of therapeutics for the treatment of human diseases in which cullin 3 CRL plays a key role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfHxzNV467rVg90H21EOLACvtfcHk0lif2wxnMEzG0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Kqsr0%253D&md5=6489af488ea0608aed7348662e9b261f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01455%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%2BB.%26aulast%3DZhou%26aufirst%3DW.%2BH.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChern%26aufirst%3DT.%2BR.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%2BF.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLi%26aufirst%3DS.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DHigh-affinity%2520peptidomimetic%2520inhibitors%2520of%2520the%2520DCN1-UBC12%2520protein%2520protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D5%26spage%3D1934%26epage%3D1950%26doi%3D10.1021%2Facs.jmedchem.7b01455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhdglz2ZMx2XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">Part 4</span>),  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&issue=Part+4&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhdglz2ZMx2XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26issue%3DPart%25204%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version, <span class="NLM_edition">2.10.3</span>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">Part 4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=Part+4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgIjScqEUhGnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3DPart%25204%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Womack, T. O.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Bricogne, G.</span></span> <i>GRADE</i>, version <span class="NLM_edition">1.2.13</span>; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=O.+S.+Smart&author=T.+O.+Womack&author=A.+Sharff&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=C.+Vonrhein&author=G.+Bricogne&title=GRADE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26jtitle%3DGRADE%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">Part 1</span>),  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&issue=Part+1&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0li3N5lk7zKtKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3DPart%25201%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ','PDB','6NUQ'); return false;">PDB: 6NUQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1','PDB','1BG1'); return false;">PDB: 1BG1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CWG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CWG','PDB','3CWG'); return false;">PDB: 3CWG</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i79"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01530">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42464"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01530?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01530</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figures of FP binding experiments, computational model, and Western blotting analysis of total STAT3 and pSTAT3<sup>Y705</sup> proteins; tables of binding affinities and crystallography data collection and refinement statistics; <sup>1</sup>H NMR and UPLC–MS spectra for representative compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_002.csv">CSV</a>)</p></li><li><p class="inline">Structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_001.pdf">jm9b01530_si_001.pdf (6.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_002.csv">jm9b01530_si_002.csv (6.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01530/suppl_file/jm9b01530_si_003.pdb">jm9b01530_si_003.pdb (250.59 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates for cocrystal structure SI-109 in complex with STAT3 have been deposited in the PDB under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NUQ">6NUQ</a> and will be released upon publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01530%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01530" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0a6185a93d1df","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
